
<html lang="en"     class="pb-page"  data-request-id="763faa06-770e-4a01-bfc5-e9e56591e5e4"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2020.63.issue-17;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.0c00973"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Elucidating the Structural Requirement of Uridylpeptide Antibiotics for Antibacterial Activity" /></meta><meta name="dc.Creator" content="Yuma  Terasawa" /></meta><meta name="dc.Creator" content="Chisato  Sataka" /></meta><meta name="dc.Creator" content="Toyotaka  Sato" /></meta><meta name="dc.Creator" content="Kazuki  Yamamoto" /></meta><meta name="dc.Creator" content="Yukari  Fukushima" /></meta><meta name="dc.Creator" content="Chie  Nakajima" /></meta><meta name="dc.Creator" content="Yasuhiko  Suzuki" /></meta><meta name="dc.Creator" content="Akira  Katsuyama" /></meta><meta name="dc.Creator" content="Takanori  Matsumaru" /></meta><meta name="dc.Creator" content="Fumika  Yakushiji" /></meta><meta name="dc.Creator" content="Shin-ichi  Yokota" /></meta><meta name="dc.Creator" content="Satoshi  Ichikawa" /></meta><meta name="dc.Description" content="The synthesis and biological evaluation of analogues of uridylpeptide antibiotics were described, and the molecular interaction between the 3′-hydroxy analogue of mureidomycin A (3′-hydroxymureidom..." /></meta><meta name="Description" content="The synthesis and biological evaluation of analogues of uridylpeptide antibiotics were described, and the molecular interaction between the 3′-hydroxy analogue of mureidomycin A (3′-hydroxymureidom..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 28, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00973" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00973" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00973" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00973" /></link>
        
    
    

<title>Elucidating the Structural Requirement of Uridylpeptide Antibiotics for Antibacterial Activity | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00973" /></meta><meta property="og:title" content="Elucidating the Structural Requirement of Uridylpeptide Antibiotics for Antibacterial Activity" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0016.jpeg" /></meta><meta property="og:description" content="The synthesis and biological evaluation of analogues of uridylpeptide antibiotics were described, and the molecular interaction between the 3′-hydroxy analogue of mureidomycin A (3′-hydroxymureidomycin A) and its target enzyme, phospho-MurNAc-pentapeptide transferase (MraY), was analyzed in detail. The structure–activity relationship (SAR) involving MraY inhibition suggests that the side chain at the urea-dipeptide moiety does not affect the MraY inhibition. However, the anti-Pseudomonas aeruginosa activity is in great contrast and the urea-dipeptide motif is a key contributor. It is also suggested that the nucleoside peptide permease NppA1A2BCD is responsible for the transport of 3′-hydroxymureidomycin A into the cytoplasm. A systematic SAR analysis of the urea-dipeptide moiety of 3′-hydroxymureidomycin A was further conducted and the antibacterial activity was determined. This study provides a guide for the rational design of analogues based on uridylpeptide antibiotics." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00973"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00973">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00973&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00973&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00973&amp;href=/doi/10.1021/acs.jmedchem.0c00973" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 9803-9827</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00962" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00983" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Elucidating the Structural Requirement of Uridylpeptide Antibiotics for Antibacterial Activity</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yuma Terasawa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuma Terasawa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuma++Terasawa">Yuma Terasawa</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chisato Sataka</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chisato Sataka</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chisato++Sataka">Chisato Sataka</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Toyotaka Sato</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Toyotaka Sato</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Microbiology, Sapporo Medical University School of Medicine, South-1, West-17, Chuo-ku, Sapporo 060-8556, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Toyotaka++Sato">Toyotaka Sato</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kazuki Yamamoto</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kazuki Yamamoto</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kazuki++Yamamoto">Kazuki Yamamoto</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yukari Fukushima</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yukari Fukushima</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Bioresources, Hokkaido University Research Center for Zoonosis Control, Sapporo 001-0020, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yukari++Fukushima">Yukari Fukushima</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chie Nakajima</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chie Nakajima</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Bioresources, Hokkaido University Research Center for Zoonosis Control, Sapporo 001-0020, Japan</div><div class="loa-info-affiliations-info">Global Station for Zoonosis Control, Global Institution for Collaborative Research and Education (GI-CoRE), Hokkaido University, Kita-20, Nishi-10, Kita-ku, Sapporo 001-0020, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chie++Nakajima">Chie Nakajima</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yasuhiko Suzuki</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yasuhiko Suzuki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Bioresources, Hokkaido University Research Center for Zoonosis Control, Sapporo 001-0020, Japan</div><div class="loa-info-affiliations-info">Global Station for Zoonosis Control, Global Institution for Collaborative Research and Education (GI-CoRE), Hokkaido University, Kita-20, Nishi-10, Kita-ku, Sapporo 001-0020, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yasuhiko++Suzuki">Yasuhiko Suzuki</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Akira Katsuyama</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Akira Katsuyama</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</div><div class="loa-info-affiliations-info">Center for Research and Education on Drug Discovery, Faculty of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Akira++Katsuyama">Akira Katsuyama</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4062-5561" title="Orcid link">http://orcid.org/0000-0002-4062-5561</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Takanori Matsumaru</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Takanori Matsumaru</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</div><div class="loa-info-affiliations-info">Center for Research and Education on Drug Discovery, Faculty of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Takanori++Matsumaru">Takanori Matsumaru</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fumika Yakushiji</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fumika Yakushiji</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</div><div class="loa-info-affiliations-info">Center for Research and Education on Drug Discovery, Faculty of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fumika++Yakushiji">Fumika Yakushiji</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9003-3767" title="Orcid link">http://orcid.org/0000-0002-9003-3767</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shin-ichi Yokota</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shin-ichi Yokota</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Microbiology, Sapporo Medical University School of Medicine, South-1, West-17, Chuo-ku, Sapporo 060-8556, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shin-ichi++Yokota">Shin-ichi Yokota</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Satoshi Ichikawa</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Satoshi Ichikawa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</div><div class="loa-info-affiliations-info">Center for Research and Education on Drug Discovery, Faculty of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</div><div class="loa-info-affiliations-info">Global Station for Biosurfaces and Drug Discovery, Global Institution for Collaborative Research and Education (GI-CoRE), Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#93faf0fbfaf8f2e4f2d3e3fbf2e1febdfbfcf8e6f7f2fabdf2f0bdf9e3"><span class="__cf_email__" data-cfemail="9bf2f8f3f2f0faecfadbebf3fae9f6b5f3f4f0eefffaf2b5faf8b5f1eb">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Satoshi++Ichikawa">Satoshi Ichikawa</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5345-5007" title="Orcid link">http://orcid.org/0000-0001-5345-5007</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00973&amp;href=/doi/10.1021%2Facs.jmedchem.0c00973" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 9803–9827</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 28, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>7 June 2020</li><li><span class="item_label"><b>Published</b> online</span>28 July 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 September 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00973" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00973</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9803%26pageCount%3D25%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYuma%2BTerasawa%252C%2BChisato%2BSataka%252C%2BToyotaka%2BSato%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D17%26contentID%3Dacs.jmedchem.0c00973%26title%3DElucidating%2Bthe%2BStructural%2BRequirement%2Bof%2BUridylpeptide%2BAntibiotics%2Bfor%2BAntibacterial%2BActivity%26numPages%3D25%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9827%26publicationDate%3DSeptember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00973"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">926</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00973" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Elucidating the Structural Requirement of Uridylpeptide Antibiotics for Antibacterial Activity&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yuma&quot;,&quot;last_name&quot;:&quot;Terasawa&quot;},{&quot;first_name&quot;:&quot;Chisato&quot;,&quot;last_name&quot;:&quot;Sataka&quot;},{&quot;first_name&quot;:&quot;Toyotaka&quot;,&quot;last_name&quot;:&quot;Sato&quot;},{&quot;first_name&quot;:&quot;Kazuki&quot;,&quot;last_name&quot;:&quot;Yamamoto&quot;},{&quot;first_name&quot;:&quot;Yukari&quot;,&quot;last_name&quot;:&quot;Fukushima&quot;},{&quot;first_name&quot;:&quot;Chie&quot;,&quot;last_name&quot;:&quot;Nakajima&quot;},{&quot;first_name&quot;:&quot;Yasuhiko&quot;,&quot;last_name&quot;:&quot;Suzuki&quot;},{&quot;first_name&quot;:&quot;Akira&quot;,&quot;last_name&quot;:&quot;Katsuyama&quot;},{&quot;first_name&quot;:&quot;Takanori&quot;,&quot;last_name&quot;:&quot;Matsumaru&quot;},{&quot;first_name&quot;:&quot;Fumika&quot;,&quot;last_name&quot;:&quot;Yakushiji&quot;},{&quot;first_name&quot;:&quot;Shin-ichi&quot;,&quot;last_name&quot;:&quot;Yokota&quot;},{&quot;first_name&quot;:&quot;Satoshi&quot;,&quot;last_name&quot;:&quot;Ichikawa&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;28&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;9803-9827&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00973&quot;},&quot;abstract&quot;:&quot;The synthesis and biological evaluation of analogues of uridylpeptide antibiotics were described, and the molecular interaction between the 3′-hydroxy analogue of mureidomycin A (3′-hydroxymureidomycin A) and its target enzyme, phospho-MurNAc-pentapeptide transferase (MraY), was analyzed in detail. The structure–activity relationship (SAR) involving MraY inhibition suggests that the side chain at the urea-dipeptide moiety does not affect the MraY inhibition. However, the anti-Pseudomonas aeruginosa activity is in great contrast and the urea-dipeptide motif is a key contributor. It is also suggested that the nucleoside peptide permease NppA1A2BCD is responsible for the transport of 3′-hydroxymureidomycin A into the cytoplasm. A systematic SAR analysis of the urea-dipeptide moiety of 3′-hydroxymureidomycin A was further conducted and the antibacterial activity was determined. This study provides a guide for the rational design of analogues based on uridylpeptide antibiotics.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00973&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00973" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00973&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00973" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00973&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00973" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00973&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00973&amp;href=/doi/10.1021/acs.jmedchem.0c00973" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00973" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00973" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00973&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00973%26sid%3Dliteratum%253Aachs%26pmid%3D32787111%26genre%3Darticle%26aulast%3DTerasawa%26date%3D2020%26atitle%3DElucidating%2Bthe%2BStructural%2BRequirement%2Bof%2BUridylpeptide%2BAntibiotics%2Bfor%2BAntibacterial%2BActivity%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D17%26spage%3D9803%26epage%3D9827%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291362" title="Antimicrobial agents">Antimicrobial agents</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291676" title="Antibacterial activity">Antibacterial activity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/jmcmar.2020.63.issue-17/20200910/jmcmar.2020.63.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The synthesis and biological evaluation of analogues of uridylpeptide antibiotics were described, and the molecular interaction between the 3′-hydroxy analogue of mureidomycin A (3′-hydroxymureidomycin A) and its target enzyme, phospho-MurNAc-pentapeptide transferase (MraY), was analyzed in detail. The structure–activity relationship (SAR) involving MraY inhibition suggests that the side chain at the urea-dipeptide moiety does not affect the MraY inhibition. However, the anti-<i>Pseudomonas aeruginosa</i> activity is in great contrast and the urea-dipeptide motif is a key contributor. It is also suggested that the nucleoside peptide permease NppA1A2BCD is responsible for the transport of 3′-hydroxymureidomycin A into the cytoplasm. A systematic SAR analysis of the urea-dipeptide moiety of 3′-hydroxymureidomycin A was further conducted and the antibacterial activity was determined. This study provides a guide for the rational design of analogues based on uridylpeptide antibiotics.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57838" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57838" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><i>Pseudomonas aeruginosa</i>, which is classified as a Gram-negative bacterium, is a common nosocomial pathogen; healthy individuals are rarely infected by this pathogen because of its low pathogenicity. However, opportunistic and nosocomial infections in newborns and patients with weakened immunity have become problematic. <i>P. aeruginosa</i> also locally infects the urethra, airways, and wounds following medical procedures such as surgery and catheterization, in addition to burns, trauma, and purulent sites; moreover, infection of patients with cystic fibrosis and chronic respiratory diseases by <i>P. aeruginosa</i> can cause increased patient mortality.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1−4)</a> In addition, local infection is followed by systemic infections, such as pneumonia and peritonitis, and it is known that the fatality rate of sepsis due to <i>P. aeruginosa</i> in the blood is as high as 80%. Thus, <i>P. aeruginosa</i> is among pathogens that must be carefully treated because it worsens the prognosis and mortality of patients. Compared to Gram-positive pathogens, <i>P. aeruginosa</i> is inherently less susceptible to antibacterial drugs currently used in the clinic because it possesses an outer membrane that acts as a barrier to prevent their permeation. Furthermore, multidrug-resistant <i>P. aeruginosa</i>, which is resistant to all carbapenems, aminoglycosides, and fluoroquinolones used for the treatment of <i>P. aeruginosa</i> infection, has emerged in the clinic. The fatality rate caused by its infection is high because of the very limited number of effective drugs and it is the cause of a severe global health problem. Consequently, development of new drugs for multidrug-resistant <i>P. aeruginosa</i> is an urgent task for worldwide health care.<a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5−7)</a> In choosing novel antibacterial agents, the target must be essential for their growth, and the agent’s mechanism of action should be different from those of existing drugs.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11 ref12">(8−12)</a></div><div class="NLM_p">Uridylpeptide antibiotics are nucleoside natural products sharing a common structural feature; a 3′-deoxyuridine with a <i>Z</i>-enamide linkage is attached to a tetrapeptide moiety with a urea linkage reversing the direction of the <i>N</i>- and <i>C</i>-termini, with α,β-diaminobutylic acid playing a pivotal role by connecting the <i>N</i>-terminal amino acid, the urea-dipeptide, and the 3′-deoxyuridine moieties (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> The class includes mureidomycins (MRDs),<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> napsamycins,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> pacidamycins (PCDs),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and sansanmycins.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Uridylpeptide antibiotics are a growing class of nucleoside natural products, and their chemical structures vary with various amino acid residues. Among the class of uridylpeptide antibiotics, the MRDs, isolated from <i>Streptomyces flavidoviridens</i> SANK 60486, have been reported to show the most potent antibacterial activity against strains of <i>P. aeruginosa</i> [minimum inhibitory concentration (MIC) 1.5–12.5 μg/mL] <i>in vitro</i> and can also protect mice against <i>P. aeruginosa</i> infection.<a onclick="showRef(event, 'cit15b cit15c'); return false;" href="javascript:void(0);" class="ref cit15b cit15c">(15b,c)</a> The MRDs and PCDs have been reported to inhibit the formation of lipid II and peptidoglycan and are strong inhibitors of the phospho-MurNAc-pentapeptide transferase (MraY),<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> which is responsible for the formation of lipid I in the peptidoglycan biosynthesis pathway.<a onclick="showRef(event, 'ref19 ref20 ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22 ref23 ref24">(19−24)</a> As MraY is an essential enzyme in bacteria, it is a potential target for the development of general antibacterial agents.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> A narrow antibacterial spectrum is characteristic of uridylpeptide antibiotics.<a onclick="showRef(event, 'cit15b cit15c'); return false;" href="javascript:void(0);" class="ref cit15b cit15c">(15b,c)</a> They exhibit highly selective antibacterial activity against <i>P. aeruginosa</i> but not against other Gram-negative and Gram-positive bacterial pathogens. Different from Gram-positive bacteria, Gram-negative bacteria possess an outer membrane, which acts as a physiological barrier preventing antibacterial drugs from reaching the periplasm. Only limited nutrients and ions necessary for bacterial growth can be incorporated through specific transporters. The observed narrow spectrum of uridylpeptides indicates that they utilize a transporter specifically expressed on <i>P. aeruginosa</i>.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> We have recently investigated the MraY inhibitory and anti-<i>P. aeruginosa</i> activities of synthetic PCD D and its 3′-hydroxy analogue <b>1</b> (3′-hPCD D, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> They exhibited potent MraY inhibitory activity with IC<sub>50</sub> values of 22 nM for PCD D and 42 nM for <b>1</b> and moderate antibacterial activity against <i>P. aeruginosa</i>. The existence of a hydroxyl group at the 3′-position of the uridine moiety does not interfere with either the MraY inhibition or the antibacterial activity. The 3′-hydroxy analogues of uridylpeptide antibiotics are a good starting point to investigate a structure–activity relationship (SAR) of uridylpeptide antibiotics as the 3′-hydroxy analogues are more accessible from a chemical synthesis point of view. Judging from the data reported in the literature, among the uridylpeptide antibiotics shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, those possessing the <span class="smallcaps smallerCapital">l</span>-<i>m</i>-tyrosine (<span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr) on both <i>N</i>- and <i>C</i>-termini, namely, MRDs, seem to exhibit the strongest antibacterial activity against <i>P. aeruginosa</i>. Although SAR studies of antibacterial activity of this class of natural products have independently been conducted,<a onclick="showRef(event, 'ref29 ref30 ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39">(29−39)</a> the strains tested were not uniform. Therefore, a side-by-side comparison of the impact of each amino acid residue on both MraY inhibition and antibacterial activity must be performed to understand the molecular basis of uridylpeptide antibiotics. Very recently, we reported a complex structure of 3′-hydroxymureidomycin A (3′-hMRD A, <b>2</b>, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) bound to MraY from <i>Aquifex aeolicus</i> (MraY<sub>AA</sub>).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Here, we describe the detailed synthesis, biological evaluation, and further SAR of <b>2</b>. In addition, a transporter responsible for uptake of <b>2</b> is also proposed. The aim of this study is to elucidate the structural requirements of <b>2</b> for both MraY inhibition and antibacterial activity against <i>P. aeruginosa</i> to gain insight into future drug design based on uridylpeptide antibiotics.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of uridylpeptide natural products.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of 3′-hydroxyuridylpeptide analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21632" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21632" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Synthesis of 3′-hMRD A</h3><div class="NLM_p">We have previously accomplished the total synthesis of PCD D and <b>1</b> featuring a stereoselective construction of the <i>Z</i>-oxy-acyl enamide architecture by copper-catalyzed <i>C</i>–<i>N</i> cross-coupling.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> We have also developed the second-generation synthesis of <b>1</b><i>via</i> a Ugi four-component reaction.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> This second-generation synthesis is a convergent method to provide a range of analogues. However, it also provided an undesired and inactive diastereomer at the 2,3-diaminobutanoic acid residue. Therefore, we developed a more efficient third-generation synthesis to investigate the systematic SAR of uridylpeptide antibiotics. Our retrosynthetic analysis of <b>2</b> is shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The 2,3-diaminobutanoic acid residue linking to the uridine moiety with a <i>Z</i>-enamide linkage plays a role as a branching scaffold and both the urea-dipeptide and the amino acid are linked to two nitrogen atoms by forming primary and secondary carboxamide linkages. Thereby, <b>2</b> was disconnected from these components, <b>3–5</b>. Thus, a range of analogues suitable for investigating the SAR regarding the urea-dipeptide and the amino acids on both termini could be attained in the last stage of the synthesis. A uridine derivative connected to diaminobutanoic acid <i>via</i> an enamide linkage <b>3</b> was disconnected from the 2,3-diaminobutancarboxamide <b>6</b> and the <i>Z</i>-vinyl iodide <b>7</b> by copper-catalyzed <i>C</i>–<i>N</i> cross-coupling.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Retrosynthetic Analysis of <b>2</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Suitably protected <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr derivatives <b>5</b> and <b>10</b> were prepared starting from commercially available <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Solution phase synthesis has thus far been used for synthesis of urea-dipeptide; however, it sometimes causes a decrease in the yield of the desired urea-dipeptide because of homodimer formation. Therefore, solid phase synthesis was applied, in which an asymmetric urea-dipeptide can be selectively and easily synthesized by a simple purification procedure (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>).<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Fmoc-<span class="smallcaps smallerCapital">l</span>-methionine immobilized on the 2-chlorotrityl resin was treated with piperidine, and the resulting amine was converted to the corresponding carbonylimidazolidate, which was reacted with <b>10</b> in the presence of <i>i</i>-Pr<sub>2</sub>NEt to yield urea <b>12</b>. Finally, removal of the urea-dipeptide from the resin by treatment with 1% TFA in CH<sub>2</sub>Cl<sub>2</sub> cleanly provided <b>4</b> with a 78% yield over four steps from <b>11</b>. This method is suitable for the comprehensive preparation of a variety of urea-dipeptides.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Preparation of Suitably Protected <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr Derivatives <b>5</b> and <b>10</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of an Asymmetric Urea-Dipeptide 4</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">With <b>4</b> and <b>5</b> in hand, synthesis of <b>2</b> was investigated as shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. The carboxylic acid <b>13</b>, which was readily prepared from <span class="smallcaps smallerCapital">l</span>-threonine in six steps in large quantity by using a previous procedure,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> was converted to the carboxamide <b>14</b> in 81% yield [NH<sub>4</sub>Cl, 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate (HATU), <i>i</i>-Pr<sub>2</sub>NEt, dimethylformamide (DMF)]. After cleavage of the <i>N</i>–<i>O</i> bond of <b>14</b> by catalytic hydrogenolysis [H<sub>2</sub>, Pd(OH)<sub>2</sub>, MeOH], the liberated amine <b>6</b> was directly coupled with the iodide <b>7</b><a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> under the conditions using CuI, MeNHCH<sub>2</sub>CH<sub>2</sub>NHMe, and Cs<sub>2</sub>CO<sub>3</sub> in tetrahydrofuran (THF) at 70 °C to give the desired <i>Z</i>-enamide <b>15</b> in 67% yield. Cross coupling of <b>7</b> and <b>14</b> also proceeded to produce the corresponding <i>Z</i>-enamide in good yield, however the following <i>N</i>–<i>O</i> bond cleavage was unsuccessful. After protection of the <i>N</i>-methylamine of the carboxamide <b>6</b> with an allyloxycarbonyl (Alloc) group, the corresponding <i>N</i>-Alloc carboxamide was used for the coupling. In this case, an unknown byproduct was mainly generated, and the desired <b>16</b> was not obtained at all. This problem was solved by simply changing the order of the cross coupling and Alloc protection reactions. After the <i>N</i>-methylamino group of <b>15</b> was protected by the Alloc group to give <b>16</b> in 93% yield, simultaneous deprotection of the benzyloxymethyl and Boc groups was achieved with BCl<sub>3</sub> in CH<sub>2</sub>Cl<sub>2</sub> at −78 °C to afford <b>3</b> in 53% yield. The primary amine <b>3</b> was condensed with the urea-dipeptide <b>4</b> [(2-(1H-benzotriazol-1-yl)-1,1,3,3-tetramethyluronium hexafluorophosphate (HBTU) and <i>i</i>-Pr<sub>2</sub>NEt in DMF] to give <b>17</b> in 84% yield. Removal of the Alloc group at the <i>N</i>-terminus of <b>17</b> was conducted by the use of conventional conditions [Pd(PPh<sub>3</sub>)<sub>4</sub>, morphorine, THF], and the resulting <i>N</i>-methylamine was condensed with <b>5</b> in the presence of HATU and <i>i</i>-Pr<sub>2</sub>NEt in DMF to afford the fully protected 3′-hMRD A <b>18</b> in 44% yield over two steps. Finally, deprotection of the <i>tert</i>-butyldimethylsilyl groups (3HF·Et<sub>3</sub>N, MeCN) followed by the Boc and <i>t</i>-Bu groups (80% <i>aq.</i> TFA) successfully provided 3′-hMRD A (<b>2</b>) in 45% yield over two steps. During 80% <i>aq.</i> TFA treatment, the methylthio group was oxidized, and the corresponding sulfoxide <b>19</b> was also obtained in 35% yield, which was easily separated by reverse-phase column chromatography.</div><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0014.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>2</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Biological Activity of <b>2</b></h3><div class="NLM_p">The MraY inhibition of <b>2</b> was evaluated with fluorescence-based MraY inhibitory assay<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> using the purified MraY enzyme (<i>Staphylococcus aureus</i>, MraY<sub>SA</sub>) and dansylated Park’s nucleotide that was prepared by dansylation of Park’s nucleotide chemically synthesized with our method.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> The data were obtained from three independent experiments. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <b>2</b> exhibits a strong MraY<sub>SA</sub> inhibitory activity, with an IC<sub>50</sub> value of 1.3 ± 0.2 nM and is 34-fold more potent than <b>1</b>. The antibacterial activity of <b>2</b> against a range of <i>P. aeruginosa</i> strains, including a variety of clinical isolates and serotypes, was then investigated. The data were obtained from three independent experiments, and MIC values for each compound are within 4-fold, which are described as a range of these values. 3′-Hydroxymureidomycin A (<b>2</b>) exhibits a potent antibacterial activity over a wide range of <i>P. aeruginosa</i> strains with MIC values ranging from 4 to 128 μg/mL, which are lower than those of <b>1</b>. It is noteworthy that <b>2</b> is effective against multidrug-resistant <i>P. aeruginosa</i> (MDRP62) with an MIC value of 16–32 μg/mL. Improvement of the MraY<sub>SA</sub> inhibitory activity of <b>2</b> over <b>1</b> is correlated with the anti-<i>P. aeruginosa</i> activity, and it was revealed that <b>2</b> possesses superior properties in terms of both MraY<sub>SA</sub> inhibitory activity and anti-<i>P. aeruginosa</i> activity. Overall, the replacement of amino acid resides at both <i>C</i>- and <i>N</i>-termini with <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyrs improved the biological activity, confirming <b>2</b> as a superior candidate, as expected from the literature review.<a onclick="showRef(event, 'ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20">(13−20)</a> Selective toxicity is an important issue in chemotherapy for infectious disease. Accordingly, the cytotoxicity of <b>2</b> against human hepatocellular carcinoma (HepG2) cells was then evaluated. Tunicamycins were also tested as a positive control and showed evidence of cytotoxicity, with an IC<sub>50</sub> of 38.6 μM. Under these conditions, <b>2</b> exhibited no cytotoxicity (IC<sub>50</sub> > 100 μM). This result indicates that <b>2</b> showed selective toxicity against bacterial strains. The observed high therapeutic index is a desirable property for antibacterial agents. Thus, <b>2</b> was confirmed to be a selective anti-<i>P. aeruginosa</i> agent.</div><figure id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. MraY inhibitory activity of analogues. <sup>a</sup>The inhibitory activities of the compounds against purified MraY<sub>SA</sub>. Reaction was conducted with 50 mM Tris-HCl (pH 7.6), 50 mM KCl, 25 mM MgCl<sub>2</sub>, 0.2% Triton X-100, 8% glycerol, 50 μM C55-P, 10 μM UDP-MurNAc-dansylpentapeptide, and MraY enzyme (55 ng/5 μL/well). After 3–4 h of incubation at room temperature, the formation of dansylated lipid I was monitored by fluorescence enhancement (excitation at 355 nm, emission at 535 nm). The data were obtained from three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Biological Activity of <b>1</b> and <b>2</b><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>1</b></th><th class="colsep0 rowsep0" align="center"><b>2</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> for MraY<sub>SA</sub> (nM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">44 ± 4</td><td class="colsep0 rowsep0" align="left">1.3 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> for HepG2 (μM)<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" align="center">strains</th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">MIC (μg/mL)<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAO1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">16–32</td><td class="colsep0 rowsep0" align="left">4–16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ATCC27853</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">32–128</td><td class="colsep0 rowsep0" align="left">4–16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MDRP62</td><td class="colsep0 rowsep0" align="left">multidrug-resistant</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">16–32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1483</td><td class="colsep0 rowsep0" align="left">clinical isolate</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">8–32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1485</td><td class="colsep0 rowsep0" align="left">clinical isolate</td><td class="colsep0 rowsep0" align="left">16–64</td><td class="colsep0 rowsep0" align="left">4–8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2867</td><td class="colsep0 rowsep0" align="left">clinical isolate</td><td class="colsep0 rowsep0" align="left">32–128</td><td class="colsep0 rowsep0" align="left">8–32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2931</td><td class="colsep0 rowsep0" align="left">clinical isolate</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9728b</td><td class="colsep0 rowsep0" align="left">clinical isolate</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID1001</td><td class="colsep0 rowsep0" align="left">serotype A</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">64–128</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID1002</td><td class="colsep0 rowsep0" align="left">serotype B</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID1007</td><td class="colsep0 rowsep0" align="left">serotype B</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">32–128</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID1013</td><td class="colsep0 rowsep0" align="left">serotype B</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID5004</td><td class="colsep0 rowsep0" align="left">serotype B</td><td class="colsep0 rowsep0" align="left">64–128</td><td class="colsep0 rowsep0" align="left">8–16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID1021</td><td class="colsep0 rowsep0" align="left">serotype C</td><td class="colsep0 rowsep0" align="left">16–64</td><td class="colsep0 rowsep0" align="left">8–32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID1004</td><td class="colsep0 rowsep0" align="left">serotype D</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">8–16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID1130</td><td class="colsep0 rowsep0" align="left">serotype E</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">16–32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID1006</td><td class="colsep0 rowsep0" align="left">serotype F</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID1020</td><td class="colsep0 rowsep0" align="left">serotype G</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">8–16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID1009</td><td class="colsep0 rowsep0" align="left">serotype H</td><td class="colsep0 rowsep0" align="left">32–128</td><td class="colsep0 rowsep0" align="left">8–32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID1010</td><td class="colsep0 rowsep0" align="left">serotype I</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">32–128</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID1011</td><td class="colsep0 rowsep0" align="left">serotype J</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">16–32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID1012</td><td class="colsep0 rowsep0" align="left">serotype K</td><td class="colsep0 rowsep0" align="left">64–128</td><td class="colsep0 rowsep0" align="left">16–64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID5141</td><td class="colsep0 rowsep0" align="left">serotype L</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">4–16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID1015</td><td class="colsep0 rowsep0" align="left">serotype M</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">16–32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IID5018</td><td class="colsep0 rowsep0" align="left">serotype M</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The inhibitory activities of the compounds against purified MraY<sub>SA</sub> were determined in a manner similar to the experiments shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">HepG2 cells were seeded in 96-well tissue culture plates at 1 × 10<sup>5</sup> cells/well. After 24 h, the cells were treated with varying concentrations of compounds for 24 h. After the treatment, the cell viability was measured by WST-8 assay. In these experiments, tunicamycin exhibited cytotoxicity with the IC<sub>50</sub> value of 38.6 μM.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">MICs were determined by a microdilution broth method as recommended by the CLSI. The data were obtained from three independent experiments described as a range of values.</p></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Elucidating the Interaction between MraY and <b>2</b></h3><div class="NLM_p">The X-ray crystal structure of a complex of muraymycin D2 (MRY D2), other class of nucleoside natural products and a strong MraY inhibitor, bound to MraY<sub>AA</sub> has been elucidated (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CKR">5CKR</a>, 2.95 Å).<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> It is also revealed that MraY<sub>AA</sub> undergoes a large conformational change upon binding to MRY D2 by comparing the structure with the apoMraY<sub>AA</sub> structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4J72">4J72</a>) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b,d,f).<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> With <b>2</b> possessing better properties, the X-ray crystal structure of a complex of <b>2</b> bound to MraY<sub>AA</sub> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OZ6">6OZ6</a>, 3.832–3.70 Å) was solved by our collaborative study to understand the molecular basis of the interaction of <b>2</b> and MraY<sub>AA</sub> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a,c,e).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The complex of <b>2</b> bound to MraY<sub>AA</sub> is similar to that of MRY D2. Using the alignment of the sequence of MraY<sub>SA</sub> and MraY from <i>P. aeruginosa</i> (MraY<sub>PA</sub>) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a><named-content content-type="anchor" rid="fig6" type="simple"></named-content>, for larger image, see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00973/suppl_file/jm0c00973_si_001.pdf" class="ext-link">Supporting Information</a>) on the crystal structure of MraY<sub>AA</sub> bound to <b>2</b> as a template, in this study, we developed homology models of MraY<sub>SA</sub> and MraY<sub>PA</sub> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>) using the SWISS-MODEL<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> to predict the molecular basis of <b>2</b> interacting with MraY<sub>SA</sub>, which is used for enzymatic assays, and MraY<sub>PA</sub> expressed in <i>P. aeruginosa</i>, the target bacteria. These models reveal that overall, the three-dimensional structures of the MraY<sub>SA</sub> and MraY<sub>PA</sub> models are quite similar to that of MraY<sub>AA</sub>, and the amino acid residues and their interacting modes are almost identical among MraY<sub>AA</sub>, MraY<sub>SA</sub>, and MraY<sub>PA</sub> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>a,b). Conserved residues in Loop C of the cytoplasmic faces of both MraY<sub>SA</sub> and MraY<sub>PA</sub> form a hydrogen-bonding network around the uracil moiety of <b>2</b>, which is further stabilized by a π–π stacking interaction with Phe226 in MraY<sub>SA</sub> and Phe264 in MraY<sub>PA</sub> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a,b). A similar interaction with Phe262 was observed in the MraY<sub>AA</sub>/MRY D2 complex (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). The <i>N</i>-terminal <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr residue of <b>2</b> interacts with the conserved residues Thr49 in MraY<sub>SA</sub>, Thr73 in MraY<sub>PA</sub> (Thr75 in MraY<sub>AA</sub>), and Asn172 in MraY<sub>SA</sub> and Asn195 in MraY<sub>PA</sub> (Asn190 in MraY<sub>AA</sub>), which create a shallow pocket adjacent to the uracil binding pocket (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>c,d). In the MraY<sub>AA</sub>/MRY D2 complex, this interaction is formed by the 5′-aminoribose moiety (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>d). In detail, the amino group at the <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr residue of <b>2</b> forms a hydrogen bond with Asp229 in MraY<sub>SA</sub> and Asp267 in MraY<sub>PA</sub> (Asp265 in MraY<sub>AA</sub>), among the most conserved and essential residues in the enzymatic activity of MraY. Of note is the interaction of the hydroxy group at the <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr residue. The hydroxy group interacts with Thr49 in MraY<sub>SA</sub> and Thr73 in MraY<sub>PA</sub> (Thr75 in MraY<sub>AA</sub>) as in the hydroxy group at the aminoribose moiety of MRY D2. 3′-Hydroxypacidamycin <b>1</b> has an <span class="smallcaps smallerCapital">l</span>-alanine residue at the corresponding site, which lacks the hydroxy group. This structural analysis revealed the reason for the improved MraY<sub>SA</sub> inhibitory activity of <b>2</b> over <b>1</b>. Observations that the 5′-amino-4′,5′-didehydro-5′-deoxyribofuranose moiety does not directly interact with MraY and the 3′-hydroxy group of <b>2</b> is exposed to a solvent-accessible area are in good accordance with our previous results where existence of the hydroxyl group at the 3′-position of the uridine moiety of <b>1</b> does not interfere with MraY inhibition.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray crystal structures of MraY<sub>AA</sub> bound to <b>2</b> and MRY D2. Carbons of <b>2</b> and MRY D2 are shown in green and orange, respectively. Hydrogen bond and salt bridge are indicated by cyan and magenta dashed lines, respectively. (a) Overall view of the complex bound to <b>2</b>. (b) Overall view of the complex bound to MRY D2. (c) Zoomed-in view of the uridine-adjacent pocket bound to <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr residue of <b>2</b>. (d) Zoomed-in view of the uridine-adjacent pocket bound to the aminoribose moiety of MRD D2. (e) Zoomed-in view of the TM9b/Loop E sites interacting with the urea-dipeptide motif of <b>2</b>. (f) Zoomed-in view of the TM9b/Loop E sites interacting with the urea-dipeptide motif of MRY D2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Sequence alignment of MraY. For detail, see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00973/suppl_file/jm0c00973_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Docking model structures of MraY<sub>SA</sub> and MraY<sub>PA</sub> bound to <b>2</b>. Carbons of <b>2</b> are shown in green. Hydrogen bond and salt bridge are indicated by cyan and magenta dashed lines, respectively. (a) Overall view of the complex of a homology modeled for MraY<sub>SA</sub> docked to <b>2</b>. (b) Overall view of the complex of a homology modeled for MraY<sub>PA</sub> docked to <b>2</b>. (c) Zoomed-in view of the uridine-adjacent pocket of MraY<sub>SA</sub> bound to the <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr residue of <b>2</b>. (d) Zoomed-in view of the uridine-adjacent pocket bound to the aminoribose moiety of MraY<sub>PA</sub> bound to the <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr residue of <b>2</b>. (e) Zoomed-in view of the TM9b/Loop E sites of MraY<sub>SA</sub> interacting with the urea-dipeptide motif of <b>2</b>. (f) Zoomed-in view of the TM9b/Loop E sites of MraY<sub>PA</sub> interacting with the urea-dipeptide motif of <b>2</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">3′-hMRD A (<b>2</b>) and MRY D2 share a urea-dipeptide moiety in their chemical structure though the amino acid residues constituting the urea-dipeptide moiety are different between <b>2</b> and MRY D2. The moieties of each extend toward transmembrane (TM) 9b and Loop E, forming an interaction with the essential residue His288 in MraY<sub>SA</sub> and His326 in MraY<sub>PA</sub> (His324 in MraY<sub>AA</sub>) in the Loop E helix (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>e,f). One of the two hydrogens in the urea-motif of <b>2</b> forms a hydrogen bond with the carbonyl oxygen of the Ala321 in the MraY<sub>AA</sub> crystal structure (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>f). However, both hydrogens in the urea-motif of <b>2</b> are suggested to form hydrogen bonds with the carbonyl oxygen of the Ser284 in the MraY<sub>SA</sub> model and the Ala322 in the MraY<sub>PA</sub> model upon minimization of the complex structures (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>e,f). In all cases, the carboxylate in the <i>C</i>-terminus of <b>2</b> interacts with Gln269 in MraY<sub>SA</sub> and Gln307 in MraY<sub>PA</sub> (Gln305 in MraY<sub>AA</sub>). Contrary to the interaction observed for the urea-motif of <b>2</b> and MRY D2, none of the side chains in the urea-dipeptide moiety interacts with any MraY (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>e,f and <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>e,f). Our previous SAR study of MRY D2 regarding the urea-dipeptide moiety indicated that the observed interaction between the urea-dipeptide moiety and TM9b and Loop E is not necessary for MraY<sub>SA</sub> inhibition.<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> Namely, truncation of the urea-dipeptide moiety of the lipophilic analogue <b>20</b>, which is a membrane-permeable analogue of MRY D2, did not diminish the inhibitory activity at all and the truncated analogue <b>21</b> still showed a potent MraY inhibitory activity comparable to that of MRY D2 and <b>20</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). It is suggested that the binding affinity of MRYs is mainly contributed by the aminoribosyluridine moiety, which is superimposable to the uridine and <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr moieties in <b>2</b>, and the urea-dipeptide moiety plays a role as an accessory motif. To see the impact of the urea-peptide moiety in the case of <b>2</b>, several truncated analogues <b>22–24</b> (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>), which lack the urea-motif, were synthesized (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00973/suppl_file/jm0c00973_si_001.pdf" class="ext-link">Schemes S1 and S2</a>), and their MraY<sub>SA</sub> inhibitory activity was evaluated (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). As a result, truncation of the urea-motif of <b>2</b> led to a complete loss of inhibitory activity, in great contrast to the results observed for MRY<sub>SA</sub>. These results clearly indicate that the urea-dipeptide motif is also a key contributor in <b>2</b>, in addition to the uridine and <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr moieties. The fact that <b>23</b> has a carboxamide functionality with only one hydrogen suggests that both hydrogens in the urea-motif likely form hydrogen bonds with the carbonyl oxygen of the Ser284 in the MraY<sub>SA</sub> and the Ala322 in the MraY<sub>PA</sub>, as predicted by modeling (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). However, it is also true that only the <i>C</i>-terminal carboxylic acid moiety could be important for MraY inhibition. Such kinds of analogues have never been tested and the possibility remains to be elucidated. Brandish <i>et al.</i>, through kinetic analysis of the enzymatic reaction, revealed that MRD A inhibits the function of MraY in a concentration- and time-dependent manner and thus is a slow-binding inhibitor.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Initially, this was proposed to be a reaction with an unusual enamide functional group although the enamide was later found not to be an essential pharmacophore.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Presumably, the urea-dipeptide motif, uridine and <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr moieties cooperatively interact with MraY during the course of the conformational change, allowing the urea-dipeptide moiety of <b>2</b> to bind to the side of TM9b and Loop E to form a more stable conformation, although more detailed analyses will be necessary to support this hypothesis.</div><figure id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structures and biological activity of MRY D2 and its lipophilic analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Structures of truncated and lipophilic analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Biological Activity of Analogues</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."><b>19</b></th><th class="colsep0 rowsep0" align="center"><b>22</b></th><th class="colsep0 rowsep0" align="center"><b>23</b></th><th class="colsep0 rowsep0" align="center"><b>24</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">4.7</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MIC (μg/mL)<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="left">n.a.</td><td class="colsep0 rowsep0" align="left">n.a.</td><td class="colsep0 rowsep0" align="left">n.a.</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">The inhibitory activities of the compounds against purified MraY<sub>SA</sub> were determined in a manner similar to the experiments shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">MICs for <i>P. aeruginosa</i> were determined by a microdilution broth method as recommended by the CLSI. n.a.: not analyzed.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Identification of the Protein Associated with 3′-hMRD A Resistance</h3><div class="NLM_p">Our initial SAR study with <b>1</b>, <b>2</b>, and <b>22–24</b> suggests that their amino acid composition has a great impact on antibacterial activity. The active site of MraY faces the cytoplasm and these inhibitors must be incorporated into the cytoplasm to reach the active site. It is unlikely that they pass through the outer and inner membranes by passive diffusion, considering the hydrophilic chemical structures of uridylpeptide antibiotics and the analogues investigated in this study. In conjunction with the narrow spectrum of uridylpeptide antibiotics, it is suggested that they utilize a transporter specifically expressed on <i>P. aeruginosa</i>; therefore, it is very important to understand how they are taken up by <i>P. aeruginosa</i>. It is known that both <i>Escherichia coli</i> and <i>P. aeruginosa</i> show intrinsic resistance to uridylpeptide antibiotics. Gotoh <i>et al.</i> reported that the resistance of <i>E. coli</i> to MRDs A and C is attributed to efflux by extrusion <i>via</i> the AcrAB-TolC pump.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Recently, Mistry <i>et al.</i> and Pletzer <i>et al.</i> independently found that the resistance of <i>P. aeruginosa</i> to PCD 4 and D is primarily due to a loss of function of uptake into the cell mediated by peptide ABC transporters belonging to the nucleoside peptide permease (Npp) family, NppA1A2BCD, which is encoded in the <i>P. aeruginosa</i> genome and was previously designated the oligopeptide transport system.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The latter reports are in good accordance with the selective antibacterial activity of uridylpeptide antibiotics only against <i>P. aeruginosa</i>. Considering these reports, it is presumed that the transporter recognizes chemical structures and/or sequence compositions of the amino acid residues involved in uridylpeptide antibiotics. We briefly investigated whether the synthetic analogue <b>2</b>, which possesses the extra hydroxy group at the 3′-position of the parent MRD A, behaves similarly. Exposure of <i>P. aeruginosa</i> PAO1 to <b>2</b> resulted in emergence of resistant mutants with an 8-fold increase of the MIC values within a day. Among 30 resistant mutants obtained in this study, and the full genome sequences of the three mutants, MRD1, 2, and 30 were determined by next-generation genome sequencing, and the sequences obtained were compared with those of wild-type <i>P. aeruginosa</i> PAO1 (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Several nucleotide mutations were found in the genes encoding NppA1A2BCD (<b>PA1810</b>) as reported previously, and five nucleotide insertions attributed to functional disruption of NppA1A2BCD were commonly detected in MRD1, 2, and 30. These results support previous reports<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> that NppA1A2BCD is responsible for transport of <b>2</b> into the cytoplasm. To confirm whether the NppA1A2BC transporter of <i>P. aeruginosa</i> PAO1 is truly required for uptake of <b>2</b> across the inner membrane, antimicrobial susceptibility of two transposonmutants <i>P. aeruginosa</i> PW4182 and PW4181 (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>), which have functional disruptions of NppA1A2BCD <i>via</i> transposon insertion into the encoding gene, was investigated. As expected, two mutants were totally resistant to <b>2</b> (MIC > 128 μg/mL), and therefore, the NppA1A2BCD transporter is responsible for uptake of <b>2</b> in <i>P. aeruginosa</i> PAO1. As NppA1A2BCD transports peptides, the observed difference in the antibacterial activity of analogues synthesized in this study (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) may be attributed to the mode of recognition of the amino acid sequence of analogues by NppA1A2BCD.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Properties of <i>P. aeruginosa</i> Mutants Resistant to <b>2</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">mutants</th><th class="colsep0 rowsep0" align="center">MIC of <b>2</b> (μg/mL)</th><th class="colsep0 rowsep0" align="center">Products</th><th class="colsep0 rowsep0" align="center">coding region change</th><th class="colsep0 rowsep0" align="center">amino acid change</th><th class="colsep0 rowsep0" align="center">CDS<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">gene<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MRD1</td><td class="colsep0 rowsep0" align="left">>64</td><td class="colsep0 rowsep0" align="left">ABC transporter</td><td class="colsep0 rowsep0" align="left">insertion of GAAGG in 1164_1165</td><td class="colsep0 rowsep0" align="left">Asp389 frame shift</td><td class="colsep0 rowsep0" align="left"><a href="https://NP_250501.1" class="ext-link">NP_250501.1</a></td><td class="colsep0 rowsep0" align="left"><b>PA1810</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">ABC transporter</td><td class="colsep0 rowsep0" align="left">G → T of conversion in 1534</td><td class="colsep0 rowsep0" align="left">Gly512Cys</td><td class="colsep0 rowsep0" align="left"><a href="https://NP_250501.1" class="ext-link">NP_250501.1</a></td><td class="colsep0 rowsep0" align="left"><b>PA1810</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Aminomethyltransferase</td><td class="colsep0 rowsep0" align="left">G → T of conversion in 569</td><td class="colsep0 rowsep0" align="left">Gly190Val</td><td class="colsep0 rowsep0" align="left"><a href="https://NP_251132.1" class="ext-link">NP_251132.1</a></td><td class="colsep0 rowsep0" align="left"><i>gcvT2</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MRD2</td><td class="colsep0 rowsep0" align="left">>64</td><td class="colsep0 rowsep0" align="left">ABC transporter</td><td class="colsep0 rowsep0" align="left">insertion of GAAGG in 1164_1165</td><td class="colsep0 rowsep0" align="left">Asp389 frame shift</td><td class="colsep0 rowsep0" align="left"><a href="https://NP_250501.1" class="ext-link">NP_250501.1</a></td><td class="colsep0 rowsep0" align="left"><b>PA1810</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">ABC transporter</td><td class="colsep0 rowsep0" align="left">G → T of conversion in 1534</td><td class="colsep0 rowsep0" align="left">Gly512Cys</td><td class="colsep0 rowsep0" align="left"><a href="https://NP_250501.1" class="ext-link">NP_250501.1</a></td><td class="colsep0 rowsep0" align="left"><b>PA1810</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MRD30</td><td class="colsep0 rowsep0" align="left">>64</td><td class="colsep0 rowsep0" align="left">bifunctional diguanylate cyclase/phosphodiesterase</td><td class="colsep0 rowsep0" align="left">A → C of conversion in 806</td><td class="colsep0 rowsep0" align="left">Asp269Ala</td><td class="colsep0 rowsep0" align="left"><a href="https://NP_253704.1" class="ext-link">NP_253704.1</a></td><td class="colsep0 rowsep0" align="left">PA5017</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">ABC transporter</td><td class="colsep0 rowsep0" align="left">insertion of GAAGG in 1164_1165</td><td class="colsep0 rowsep0" align="left">Asp389 frame shift</td><td class="colsep0 rowsep0" align="left"><a href="https://NP_250501.1" class="ext-link">NP_250501.1</a></td><td class="colsep0 rowsep0" align="left"><b>PA1810</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">phage coat protein A</td><td class="colsep0 rowsep0" align="left">A → G of conversion in 569</td><td class="colsep0 rowsep0" align="left">Asp190Gly</td><td class="colsep0 rowsep0" align="left"><a href="https://NP_250501.1" class="ext-link">NP_250501.1</a></td><td class="colsep0 rowsep0" align="left">PA0724</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">phage coat protein A</td><td class="colsep0 rowsep0" align="left">G → A of conversion in 559</td><td class="colsep0 rowsep0" align="left">Gly187Ser</td><td class="colsep0 rowsep0" align="left"><a href="https://NP_250501.1" class="ext-link">NP_250501.1</a></td><td class="colsep0 rowsep0" align="left">PA0724</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">phage coat protein A</td><td class="colsep0 rowsep0" align="left">insertion of GCG in 634_635</td><td class="colsep0 rowsep0" align="left">Gly insertion in Gly211_Asp212</td><td class="colsep0 rowsep0" align="left"><a href="https://NP_250501.1" class="ext-link">NP_250501.1</a></td><td class="colsep0 rowsep0" align="left">PA0724</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">CDS and Genes were annotated by <i>P. aeruginosa</i> PAO1 genome (accession number, <a href="https://NC_002516.2" class="ext-link">NC_002516.2</a>).</p></div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Properties of Transposonmutants <i>P. aeruginosa</i> Mutants</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">mutants</th><th class="colsep0 rowsep0" align="center">MIC of <b>2</b> (μg/mL)</th><th class="colsep0 rowsep0" align="center">ORF</th><th class="colsep0 rowsep0" align="center">putative ORF function</th><th class="colsep0 rowsep0" align="center">position in ORF</th><th class="colsep0 rowsep0" align="center">gene position</th><th class="colsep0 rowsep0" align="center">transposon</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PW4181</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left"><b>PA1810</b></td><td class="colsep0 rowsep0" align="left">probable binding protein component of ABC transporter</td><td class="colsep0 rowsep0" align="left">747 (1848)</td><td class="colsep0 rowsep0" align="left">1966872</td><td class="colsep0 rowsep0" align="left">ISlacZ/hah</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PW4182</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left"><b>PA1810</b></td><td class="colsep0 rowsep0" align="left">probable binding protein component of ABC transporter</td><td class="colsep0 rowsep0" align="left">799 (1848)</td><td class="colsep0 rowsep0" align="left">1966820</td><td class="colsep0 rowsep0" align="left">ISphoA/hah</td></tr></tbody></table></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Detailed SAR Regarding the Urea-Dipeptide Moiety</h3><div class="NLM_p">Uridylpeptide antibiotics have a dual nature as antibacterial agents: they must be a substrate of a transporter, one of the candidates of which is suggested to be NppA1A2BCD, as well as an inhibitor of MraY. Therefore, it has proved difficult to design analogues of these antibiotics that retain both MraY inhibition and antimicrobial activity.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> NppA1A2BCD transports peptides with a certain selectivity for the length and sequence; however, the details of the selectivity and sequence preference remain unclear.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> It was revealed that the <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr at the <i>N</i>-terminus of <b>2</b> largely contributes to MraY inhibition and none of the side chains in the urea-dipeptide moiety of <b>2</b> interacts with MraY<sub>AA</sub> in the X-ray crystal structure or with MraY<sub>SA</sub> and MraY<sub>PA</sub> constructed by homology modeling. These results prompted us to see the impact of the side chains on MraY inhibition and antibacterial activity through a systematic SAR of the urea-dipeptide moiety. In a different study, Bugg <i>et al.</i> proposed a structural link between two aromatic residues (<span class="smallcaps smallerCapital">l</span>-Trp or <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr at the C-terminus and <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr at the <i>N</i>-terminus) in the MDR/PCD structures, and an Arg-Trp-x-x-Trp motif on the lysis protein E of bacteriophage ϕX174,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> which inhibits MraY <i>via</i> a protein–protein interaction remote from the active site<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(55)</a> and suggested that two aromatic residues in the MDR/PCD are important for uptake into the cell <i>via</i> a hydrophobic channel present in the structure of MraY. Therefore, the impact of the internal amino acid residue at the urea-dipeptide moiety on antibacterial activity was investigated as the index of NppA1A2BCD transport by using a set of analogues <b>25a–t</b>. <span class="smallcaps smallerCapital">l</span>-Tryptophan was chosen for the <i>C</i>-terminal amino acid instead of <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). These analogues were prepared by a modified synthetic route for <b>2</b>. Namely, <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr at the <i>N</i>-terminus was introduced to <b>15</b> to give <b>27</b> prior to connecting the urea-dipeptide moiety <b>29a–t</b> to reduce the synthetic steps (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>). The attempt to prepare a free primary amine obtained by deprotection of <b>27</b> by BCl<sub>3</sub> failed because the free amine underwent decomposition, resulting in removal of the Fmoc group at the <i>N</i>-terminus. Therefore, the free amine was converted to the corresponding hydrochloride salt <b>28</b>, which is stable and can be stored, immediately after deprotection of the protecting groups. Except for <b>29t</b>, solid-phase parallel synthesis of urea-dipeptides is suitable for rapid access to the urea-dipeptide carboxylic acids <b>29a–s</b>. Condensation of <b>28</b> and <b>29a–t</b> followed by deprotection afforded <b>25a–t</b>.</div><figure id="fig9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Structures of analogues modified at the urea-dipeptide moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch5" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0015.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Analogues <b>29a–t</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The IC<sub>50</sub> values for the MraY<sub>SA</sub> inhibitory activities of <b>25a–s</b> were essentially within a similar range (0.22 ± 0.07 to 7.6 ± 1.6 nM (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>)) and a variety of amino acid residues with various physicochemical properties including negative or positive charges are acceptable for MraY inhibition. Analogues <b>1</b> and <b>25b</b> share the same urea-dipeptide moiety, with a structural difference of a single amino acid replacement at the <i>N</i>-terminus (<span class="smallcaps smallerCapital">l</span>-Ala <i>vs</i><span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr). The fact that this replacement resulted in a significant, 32-fold, increase in MraY<sub>SA</sub> inhibition supported the hypothesis that the <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr at the <i>N</i>-terminus largely contributes to the inhibitory activity, as was expected in the interactions in a uridine-adjacent pocket (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>c,d). Analogues possessing aromatic residues tend to exhibit a stronger inhibitory activity among <b>25a–s</b>, and the <span class="smallcaps smallerCapital">l</span>-Tyr analogue <b>25i</b> showed the strongest MraY<sub>SA</sub> inhibitory activity with an IC<sub>50</sub> value of 0.22 ± 0.07 nM. One exception is the <span class="smallcaps smallerCapital">l</span>-Pro analogue <b>25t</b>, which largely decreased the inhibitory activity with an IC<sub>50</sub> value of >500 nM. Replacement with <span class="smallcaps smallerCapital">l</span>-Pro consequently results in a deficit of one of the two hydrogens in the urea-motif, and the decrease in inhibitory activity of <b>25t</b> is consistent with the SAR for the truncated analogues <b>22</b>–<b>24</b>. Overall, the SAR of <b>25a–t</b> regarding MraY inhibition is in good accordance with the structural features observed for the X-ray crystal structure of the complex of <b>2</b> bound to MraY<sub>AA</sub>. The antibacterial activities of analogues against <i>P. aeruginosa</i> PAO1, which may reflect the selectivity for their cellular uptake by NppA1A2BCD, were next investigated. Different from MraY<sub>SA</sub> inhibitory activity, it was found that the side chains in the urea-dipeptide largely affected the antibacterial activity. Except for the <span class="smallcaps smallerCapital">l</span>-Leu analogue <b>25c</b>, analogues possessing hydrophobic residues such as <span class="smallcaps smallerCapital">l</span>-Ile, <span class="smallcaps smallerCapital">l</span>-Val, and <span class="smallcaps smallerCapital">l</span>-Met (<b>25d</b>, <b>25e</b>, <b>25g</b>, respectively) and aromatic residues such as <span class="smallcaps smallerCapital">l</span>-Phe, <span class="smallcaps smallerCapital">l</span>-Tyr, and <span class="smallcaps smallerCapital">l</span>-Trp (<b>25h</b>, <b>25i</b>, and <b>25j</b>, respectively) exhibited a comparable antibacterial activity against <i>P. aeruginosa</i> PAO1 to <b>2</b> with MICs ranging from 16 to 128 μg/mL. The <span class="smallcaps smallerCapital">l</span>-Val analogue is the best in terms of antibacterial activity against <i>P. aeruginosa</i> among those tested in this study (MIC 16 μg/mL for <b>25e</b>). Analogues containing less hydrophobic residues such as <span class="smallcaps smallerCapital">l</span>-Gly (<b>25a</b>), charged residues, such <span class="smallcaps smallerCapital">l</span>-Asp, <span class="smallcaps smallerCapital">l</span>-Glu, <span class="smallcaps smallerCapital">l</span>-Lys, and <span class="smallcaps smallerCapital">l</span>-Arg (<b>25l</b>, <b>25m</b>, <b>25n</b>, and <b>25o</b>, respectively), and polar residues such as <span class="smallcaps smallerCapital">l</span>-His, <span class="smallcaps smallerCapital">l</span>-Ser, <span class="smallcaps smallerCapital">l</span>-Asn, and <span class="smallcaps smallerCapital">l</span>-Gln (<b>25k</b>, <b>25p</b>, <b>25r</b>, and <b>25s</b>, respectively), which can form a hydrogen bonding, decreased the antibacterial activity to a large extent. These results suggest that polar amino acid residues at the internal amino acid residue of the urea-dipeptide moiety are not preferred but hydrophobic residues are preferred for incorporation. Of interest is the antibacterial activity of <b>25d</b>, <b>25e</b>, and <b>25q</b>. Namely, the analogues containing β-branched amino acid residues such as <span class="smallcaps smallerCapital">l</span>-Ile, <span class="smallcaps smallerCapital">l</span>-Val, and <span class="smallcaps smallerCapital">l</span>-Thr tend to exhibit better antibacterial activity.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Biological Activity of Analogues <b>25a–t</b><a class="ref internalNav" href="#t5fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="5" align="center">Hydrophobic</th><th class="rowsep1 colsep0" colspan="5" align="center">aromatic</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>25a</b></th><th class="colsep0 rowsep0" align="center"><b>25b</b></th><th class="colsep0 rowsep0" align="center"><b>25c</b></th><th class="colsep0 rowsep0" align="center"><b>25d</b></th><th class="colsep0 rowsep0" align="center"><b>25e</b></th><th class="colsep0 rowsep0" align="center"><b>25f</b></th><th class="colsep0 rowsep0" align="center"><b>25g</b></th><th class="colsep0 rowsep0" align="center"><b>25h</b></th><th class="colsep0 rowsep0" align="center"><b>25i</b></th><th class="colsep0 rowsep0" align="center"><b>25j</b></th><th class="colsep0 rowsep0" align="center"><b>25k</b></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">Gly</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Ala</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Leu</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Ile</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Val</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Cys</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Met</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Phe</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Tyr</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Trp</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-His</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">4.2 ± 1.2</td><td class="colsep0 rowsep0" align="left">1.3 ± 0.2</td><td class="colsep0 rowsep0" align="left">4.0 ± 1.3</td><td class="colsep0 rowsep0" align="left">1.4 ± 0.4</td><td class="colsep0 rowsep0" align="left">1.5 ± 0.2</td><td class="colsep0 rowsep0" align="left">1.6 ± 0.2</td><td class="colsep0 rowsep0" align="left">1.0 ± 0.2</td><td class="colsep0 rowsep0" align="left">0.60 ± 0.15</td><td class="colsep0 rowsep0" align="left">0.22 ± 0.07</td><td class="colsep0 rowsep0" align="left">0.44 ± 0.10</td><td class="colsep0 rowsep0" align="left">2.8 ± 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MIC (μg/mL)<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">64</td><td class="colsep0 rowsep0" align="left">32–128</td><td class="colsep0 rowsep0" align="left">16–32</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">32–64</td><td class="colsep0 rowsep0" align="left">16–32</td><td class="colsep0 rowsep0" align="left">16–32</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">>128</td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">acidic</th><th class="rowsep1 colsep0" colspan="2" align="center">basic</th><th class="rowsep1 colsep0" colspan="4" align="center">hydrogen bonding</th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>25l</b></th><th class="colsep0 rowsep0" align="center"><b>25m</b></th><th class="colsep0 rowsep0" align="center"><b>25n</b></th><th class="colsep0 rowsep0" align="center"><b>25o</b></th><th class="colsep0 rowsep0" align="center"><b>25p</b></th><th class="colsep0 rowsep0" align="center"><b>25q</b></th><th class="colsep0 rowsep0" align="center"><b>25r</b></th><th class="colsep0 rowsep0" align="center"><b>25s</b></th><th class="colsep0 rowsep0" align="center"><b>25t</b></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Asp</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Glu</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Lys</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Arg</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Ser</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Thr</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Asn</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Gln</th><th class="colsep0 rowsep0" align="center"><span class="smallcaps smallerCapital">l</span>-Pro</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">7.6 ± 1.6</td><td class="colsep0 rowsep0" align="left">3.3 ± 0.6</td><td class="colsep0 rowsep0" align="left">1.8 ± 0.5</td><td class="colsep0 rowsep0" align="left">3.4 ± 0.5</td><td class="colsep0 rowsep0" align="left">1.8 ± 0.4</td><td class="colsep0 rowsep0" align="left">1.8 ± 0.4</td><td class="colsep0 rowsep0" align="left">2.1 ± 0.4</td><td class="colsep0 rowsep0" align="left">1.3 ± 0.2</td><td class="colsep0 rowsep0" align="left">>500</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MIC (μg/mL)<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">32–64</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">>128</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">The inhibitory activities of the compounds against purified MraY<sub>SA</sub> were determined in a manner similar to the experiments shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">MICs for <i>P. aeruginosa</i> were determined by a microdilution broth method as recommended by the CLSI. The data were obtained from three independent experiments described as a range of the values.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36527" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36527" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Development of a novel lead effective against multidrug-resistant <i>P. aeruginosa</i> is urgently needed worldwide. Intrigued by the specific antibacterial activity and potent MraY inhibitory activity of uridylpeptide antibiotics, the SAR of the 3′-hydroxy derivative of mureidomycin A (<b>2</b>) was conducted to elucidate its structural requirement for both MraY inhibition and antibacterial activities against <i>P. aeruginosa</i>. First, a synthetic method capable of providing a range of analogues suitable for investigating the SAR was established as the third-generation synthesis, where the urea-dipeptide moiety and <i>N</i>-terminal amino acid are introduced in the last stage of the synthesis. Design and synthesis of analogues guided by X-ray crystal structure analysis of <b>2</b> bound to MraY<sub>AA</sub> was conducted to elucidate the mode of MraY inhibition. The SAR of analogues <b>25a–t</b> suggests that hydrophobic residues at the internal amino acid residue of the urea-dipeptide moiety are a better substrate for bacterial incorporation. It was suggested that the interaction of the urea-motif is essential for the strong MraY inhibitory activity and that the amino acid residues in the motif did not contribute to the inhibition. The SAR regarding anti-<i>P. aeruginosa</i> activity is in great contrast to that for MraY inhibition, as anti-<i>P. aeruginosa</i> activity is greatly affected by the amino acid residues. As shown in this study, a side-by-side comparison of the impact of each amino acid residue on both MraY inhibition and antibacterial activity enabled us to evaluate a more reliable SAR of uridylpeptide antibiotics and their analogues. A summary of the SAR of uridylpeptide antibiotics revealed by this study includes the following: (1) <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr at the <i>N</i>-terminus is favored to increase the MraY inhibitory activity, (2) the interaction of the urea-dipeptide motif is indispensable for the MraY inhibitory activity, (3) the side chain at the urea-dipeptide moiety does not affect the MraY inhibition, and (4) the antibacterial activity of uridylpeptide antibiotics depends on their amino acid composition, presumably because of the selectivity for cellular uptake by NppA1A2BCD. As for antibacterial activity, analogues possessing hydrophobic residues and aromatic residues exhibited a comparable antibacterial activity against <i>P. aeruginosa</i> PAO1 to <b>2</b>. There are parallel trends between these data and those reported in previous studies.<a onclick="showRef(event, 'ref33 ref39'); return false;" href="javascript:void(0);" class="ref ref33 ref39">(33,39)</a> With this SAR information in hand, future optimization of the amino acid residues would improve the antibacterial activity against <i>P. aeruginosa</i>. Thus, it will be necessary to understand the structural requirements of uridylpeptide antibiotics in NppA1A2BCD recognition in much more detail with future SAR analyses. Uridylpeptide antibiotics have a dual nature as an antibacterial agent and it is challenging to design analogues that retain both MraY inhibition and antimicrobial activity. This study provides a guide for the rational design of analogues based on uridylpeptide antibiotics.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77753" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77753" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> General Experimental Methods</h3><div class="NLM_p last">NMR spectra were reported in parts per million (δ) relative to tetramethylsilane (0.00 ppm) as the internal standard unless otherwise noted. Coupling constant (<i>J</i>) was reported in hertz (Hz). Abbreviations of multiplicity were as follows: s: singlet, d; doublet, t: triplet, q: quartet, m: multiplet, br: broad. Data were presented as follows; chemical shift (multiplicity, integration, coupling constant). Assignment was based on <sup>1</sup>H–<sup>1</sup>H COSY, HMBC, and HMQC NMR spectra. Purity of all the compounds tested for biological evaluation was confirmed to be >95% by liquid chromatography–mass spectrometry (LC–MS) analysis as shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00973/suppl_file/jm0c00973_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> (<i>S</i>)-2-[(<i>tert</i>-Butoxycarbonyl)amino]-3-(3-hydroxyphenyl)propanoic Acid (<b>5</b>)</h3><div class="NLM_p last">A mixture of H-<span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr-OH (<b>8</b>) (1.50 g, 8.28 mmol) and NaHCO<sub>3</sub> (1.04 g, 12.4 mmol) in THF (40 mL) and H<sub>2</sub>O (40 mL) was treated with Boc<sub>2</sub>O (1.99 g, 9.11 mmol) at 0 °C, and the mixture was stirred at room temperature for 17 h. The mixture was partitioned between AcOEt and 1 M aq HCl. The organic phase was washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i> to afford a crude carboxylic acid. A solution of the crude carboxylic acid and imidazole (3.38 g, 49.7 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (83 mL) was treated with TIPSCl (5.26 mL, 24.8 mmol) at 0 °C, and the mixture was stirred at room temperature for 44 h. The mixture was partitioned between AcOEt and 1 M aq HCl, and the organic phase was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered and concentrated <i>in vacuo</i>. A solution of the residue in THF/MeOH (2/1, 83 mL) and H<sub>2</sub>O (13 mL) was treated with K<sub>2</sub>CO<sub>3</sub> (1.60 g, 11.6 mmol) at room temperature for 2 h. The mixture was partitioned between AcOEt and 1 M aq HCl, and the organic phase was washed with water and brine, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The residue was purified by silica gel column chromatography (50% AcOEt/hexane) to afford <b>5</b> (3.30 g, 7.54 mmol, 91%) as a colorless oil. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 7.15 (dd, 1H, H-5-<i>m</i>Tyr, <i>J</i><sub>5-<i>m</i>Tyr,4-<i>m</i>Tyr</sub> = <i>J</i><sub>5-<i>m</i>Tyr,6-<i>m</i>Tyr</sub> = 7.7 Hz), 6.80–6.68 (m, 3H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-6-<i>m</i>Tyr), 4.89 (br d, 1H, NH-<i>m</i>Tyr, <i>J</i><sub>NH-<i>m</i>Tyr,α-<i>m</i>Tyr</sub> = 7.4 Hz), 4.57 (m, 1H, H-α-<i>m</i>Tyr), 3.12 (dd, 1H, H-β-<i>m</i>Tyr, <i>J</i><sub>β-<i>m</i>Tyr,α-<i>m</i>Tyr</sub> = 4.6, <i>J</i><sub>gem</sub> = 13.7 Hz), 3.03 (dd, 1H, H-β-<i>m</i>Tyr, <i>J</i><sub>β-<i>m</i>Tyr,α-<i>m</i>Tyr</sub> = 6.3, <i>J</i><sub>gem</sub> = 13.7 Hz), 1.42 (s, 9H, <sup><i>t</i></sup>Bu), 1.24 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.10 (s, 12H, Si[CH(C<i>H</i><sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.08 (s, 6H, Si[CH(C<i>H</i><sub>3</sub>)<sub>2</sub>]<sub>3</sub>); <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz): δ 176.8, 156.3, 137.3, 129.7, 122.1, 121.2, 118.8, 80.4, 54.3, 37.6, 28.4, 18.1, 12.8; ESIMS-LR <i>m</i>/<i>z</i>: 438 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>39</sub>NO<sub>5</sub>SiNa, 460.2490; found, 460.2503; [α]<sub class="stack">D</sub><sup class="stack">24</sup> +13.7 (<i>c</i> 0.44, MeOH).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> <i>tert</i>-Butyl (<i>S</i>)-2-[(<i>tert</i>-Butoxycarbonyl)amino]-3-{3-[(triisopropylsilyl)oxy]phenyl}-propanoate (<b>9</b>)</h3><div class="NLM_p last">A solution of <b>5</b> (2.51 g, 5.74 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (57 mL) was treated with <i>O</i>-<i>tert</i>-butyl 2,2,2-trichloroacetimidate (5.14 mL, 28.7 mmol) at room temperature. The mixture was then warmed to 45 °C and stirred for 2 h. Additional <i>O</i>-<i>tert</i>-butyl 2,2,2-trichloroacetimidate (5.14 mL, 28.7 mmol) was added to the mixture, which was stirred for 2 h. The mixture was concentrated <i>in vacuo</i>, and the residue was purified by silica gel column chromatography (3% AcOEt/hexane) to afford <b>9</b> (2.70 g, 5.47 mmol, 95%) as a colorless oil. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 7.11 (dd, 1H, H-5-<i>m</i>Tyr, <i>J</i><sub>5-<i>m</i>Tyr,4-<i>m</i>Tyr</sub> = <i>J</i><sub>5-<i>m</i>Tyr,6-<i>m</i>Tyr</sub> = 8.0 Hz), 6.75–6.70 (m, 3H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-6-<i>m</i>Tyr), 4.96 (br d, 1H, NH-<i>m</i>Tyr, <i>J</i><sub>NH-<i>m</i>Tyr,α-<i>m</i>Tyr</sub> = 8.2 Hz), 4.43 (dd, 1H, H-α-<i>m</i>Tyr, <i>J</i><sub>α-<i>m</i>Tyr,N<i>H</i></sub> = 8.2, <i>J</i><sub>α-<i>m</i>Tyr,β-<i>m</i>Tyr</sub> = 6.0 Hz), 3.00 (d, 2H, H-β-<i>m</i>Tyr, <i>J</i><sub>β-<i>m</i>Tyr,α-<i>m</i>Tyr</sub> = 6.0 Hz), 1.42 (s, 9H, Boc-<sup><i>t</i></sup>Bu), 1.41 (s, 9H, <sup><i>t</i></sup>Bu), 1.29–1.19 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.10 (d, 18H, <i>J</i> = 6.9 Hz, H-Si[CH(C<i>H</i><sub>3</sub>)<sub>2</sub>]<sub>3</sub>); <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz): δ 171.0, 156.1, 155.2, 137.8, 129.3, 122.4, 121.4, 118.3, 82.1, 79.7, 54.8, 38.3, 28.5, 28.1, 18.1, 12.8; ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>48</sub>NO<sub>5</sub>Si, 494.33; found, 494.00.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> <i>tert</i>-Butyl (<i>S</i>)-2-Amino-3-[3-(triisopropylsilyloxy)phenyl]propanoate Trifluoro Acetate Salt (<b>10</b>)</h3><div class="NLM_p last">Compound <b>9</b> (2.60 g, 5.27 mmol) was treated with 10% TFA/CH<sub>2</sub>Cl<sub>2</sub> (53 mL) at room temperature for 1.5 h. The mixture was concentrated <i>in vacuo</i>, and the residue was purified by silica gel column chromatography (3% MeOH/CHCl<sub>3</sub> containing 0.1% TFA) to afford <b>10</b> (2.60 g, 5.30 mmol, 97%) as a colorless oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.24 (dd, 1H, H-5-<i>m</i>Tyr, <i>J</i><sub>5-<i>m</i>Tyr,4-<i>m</i>Tyr</sub> = <i>J</i><sub>5-<i>m</i>Tyr,6-<i>m</i>Tyr</sub> = 7.8 Hz), 6.88–6.80 (m, 3H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-6-<i>m</i>Tyr), 4.12 (dd, 1H, H-α-<i>m</i>Tyr, <i>J</i><sub>α-<i>m</i>Tyr,β-<i>m</i>Tyr</sub> = 7.8, <i>J</i><sub>α-<i>m</i>Tyr,β-<i>m</i>Tyr</sub> = 6.9 Hz), 3.11 (m, 2H, H-β-<i>m</i>Tyr), 1.42 (s, 9H, <sup><i>t</i></sup>Bu), 1.29 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.13 (s, 12H, Si[CH(C<i>H</i><sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.11 (s, 6H, Si[CH(C<i>H</i><sub>3</sub>)<sub>2</sub>]<sub>3</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>22</sub>H<sub>40</sub>NO<sub>3</sub>Si, 394.28; found, 394.0.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> (<i>S</i>)-1-(<i>tert</i>-Butoxy)-1-oxo-3-{[3-(triisopropylsilyloxy)phenyl]propan-2-yl-carbamoyl}-<span class="smallcaps smallerCapital">l</span>-methionine (<b>4</b>)</h3><div class="NLM_p last">2-Chlorotrityl chloride resin (1.6 mmol/g, 200 mg) was placed in a 10 mL polypropylene syringe fitted with a polyethylene filter disc. The resin was washed with CH<sub>2</sub>Cl<sub>2</sub> (2 mL), and a solution of Fmoc-Met-OH (356 mg, 960 μmol) and <sup><i>i</i></sup>Pr<sub>2</sub>NEt (334 μL, 1.92 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) was then added. After the mixture was agitated at room temperature for 3 h, the solvent and reagents were removed by suction. The resin was subjected to the following washing treatments with CH<sub>2</sub>Cl<sub>2</sub>/MeOH/<sup><i>i</i></sup>Pr<sub>2</sub>NEt (17/2/1, 2 mL × 3), DMF (2 mL × 2), and CH<sub>2</sub>Cl<sub>2</sub> (2 mL × 2). The resin was treated with piperidine/DMF (1/4, 5 min, then 1/9, 15 min) to remove the Fmoc group at room temperature and washed with DMF (2 mL × 3) and CH<sub>2</sub>Cl<sub>2</sub> (2 mL × 3). A solution of 1,1′-carbonyldiimidazole (156 mg, 960 μmol) and <i>N</i>-methylmorpholine (211 mg, 1.92 μmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) was added to each resin. After the mixture was agitated at room temperature for 1 min, the solvent and reagents were removed by suction. This procedure was repeated 5 times. The resin was washed with CH<sub>2</sub>Cl<sub>2</sub> (2 mL × 3). Then, a solution of <b>10</b> (189 mg, 480 μmol) and <sup><i>i</i></sup>Pr<sub>2</sub>NEt (167 μL, 960 μmol) in DMF (2 mL) was added to the resins. The mixture was agitated at room temperature for 3 h, and the solvent and reagents were removed by suction. The resin was washed with DMF (2 mL × 3) and CH<sub>2</sub>Cl<sub>2</sub> (2 mL × 3). The resin was treated with 1% TFA/CH<sub>2</sub>Cl<sub>2</sub> to cleave the urea-dipeptide from the resin, and the filtrate was partitioned between AcOEt and H<sub>2</sub>O. The organic phase was washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i> to afford <b>4</b> (141 mg, 248 μmol, 78%) as a colorless oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.14 (dd, 1H, H-5-<i>m</i>Tyr, <i>J</i><sub>5-<i>m</i>Tyr,4-<i>m</i>Tyr</sub> = <i>J</i><sub>5-<i>m</i>Tyr,6-<i>m</i>Tyr</sub> = 7.9 Hz), 6.82 (d, 1H, H-6-<i>m</i>Tyr, <i>J</i><sub>6-<i>m</i>Tyr,5-<i>m</i>Tyr</sub> = 7.9 Hz), 6.76–6.73 (m, 2H, H-4-<i>m</i>Tyr, H-2-<i>m</i>Tyr), 4.42–4.38 (m, 2H, H-α-<i>m</i>Tyr, H-α-Met), 2.95 (d, 2H, H-β-<i>m</i>Tyr, <i>J</i><sub>β-<i>m</i>Tyr,α-<i>m</i>Tyr</sub> = 6.8 Hz), 2.56–2.52 (m, 2H, H-γ-Met), 2.12–2.08 (m, 1H, H-β-Met), 2.08 (s, 3H, SC<i>H</i><sub>3</sub>), 1.94–1.84 (m, 1H, H-β-Met), 1.40 (s, 9H, <sup><i>t</i></sup>Bu), 1.28–1.19 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.11 (d, 18H, Si[CH(C<i>H</i><sub>3</sub>)<sub>2</sub>]<sub>3</sub>, <i>J</i> = 7.3 Hz); <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz): δ 176.8, 173.1, 159.7, 157.3, 139.7, 130.3, 123.6, 122.1, 119.1, 82.8, 56.2, 53.2, 39.3, 33.4, 31.0, 28.3, 18.4, 15.3, 13.9; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>49</sub>N<sub>2</sub>O<sub>6</sub>SSi<sub>2</sub>, 569.3075; found, 569.3064.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> (2<i>S</i>,3<i>S</i>)-2-[(<i>tert</i>-Butoxycarbonyl)amino]-3-[(<i>N</i>-benzyloxy-<i>N</i>-methyl)amino]-butanamide (<b>14</b>)</h3><div class="NLM_p last">A mixture of (2<i>S</i>,3<i>S</i>)-2-[(<i>tert</i>-butoxycarbonyl)amino]-3-[(<i>N</i>-benzyloxy-<i>N</i>-methyl)amino]butanoic acid (<b>13</b>, 7.6 g, 22.3 mmol), NH<sub>4</sub>Cl (11.9 g, 223 mmol), HATU (11.0 g, 29.0 mmol), and <i>N</i>-methylmorpholine (31.9 mL, 290 mmol) in DMF (220 mL) was stirred at 0 °C for 1 h. The reaction mixture was diluted with H<sub>2</sub>O and extracted with AcOEt. The combined organic layers were washed with 0.2 M aq HCl and brine, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The residue was crystallized from AcOEt to afford <b>14</b> (6.1 g, 18.1 mmol, 81%) as a white solid. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.38–7.26 (m, 5H, OCH<sub>2</sub><i>C</i><sub>6</sub><i>H</i><sub>5</sub>), 4.66 (d, 1H, O<i>CH</i><sub>2</sub>Ph, <i>J</i><sub>gem</sub> = 10.7 Hz), 4.63 (d, 1H, O<i>CH</i><sub>2</sub>Ph, <i>J</i><sub>gem</sub> = 10.7 Hz), 4.49 (br s, 1H, H-α), 3.16 (qd, 1H, H-β, <i>J</i><sub>β,γ</sub> = <i>J</i><sub>β,α</sub> = 6.8 Hz), 2.61 (s, 3H, H-NC<i>H</i><sub>3</sub>), 1.46 (s, 9H, Boc), 1.07 (d, H-γ, <i>J</i><sub>γ,β</sub> = 6.8 Hz); <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz): δ 176.8, 158.0, 138.6, 130.1, 129.3, 128.9, 80.8, 75.6, 64.4, 57.6, 42.3, 28.7, 11.1; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>17</sub>H<sub>28</sub>N<sub>3</sub>O<sub>4</sub>, 338.2074; found, 338.2078; [α]<sub class="stack">D</sub><sup class="stack">25</sup> −8.5 (<i>c</i> 0.87, MeOH).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Compound <b>15</b></h3><div class="NLM_p last">A mixture of <b>14</b> (2.55 g, 6.78 mmol) and Pd(OH)<sub>2</sub> (1.1 g) in MeOH (68 mL) was vigorously stirred at room temperature for 3 h under H<sub>2</sub> atmosphere. The catalyst was filtered off through a Celite pad, and the filtrate was concentrated <i>in vacuo</i> to afford a crude carboxamide. The crude carboxamide, <b>7</b> (4.75 g, 6.78 mmol), CuI (1.25 g, 8.14 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (13.1 g, 10.2 mmol) were added to a flask. Dried THF (68 mL) and <i>N</i>,<i>N</i>-dimethyl-1,2-ethylendiamine (2.8 mL, 16.3 mmol) were added to the mixture, which was heated at 70 °C for 7 h. The reaction mixture was diluted with AcOEt, the insoluble substances were filtered off through a Celite pad, and the filtrate was washed with H<sub>2</sub>O and brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated <i>in vacuo</i>. The residue was purified by silica gel column chromatography (CHCl<sub>3</sub>/MeOH = 6–10%) to afford <b>15</b> (3.65 g, 4.53 mmol, 67%) as a white foam. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 9.15 (br s, 1H, NH), 7.38–7.27 (m, 5H, NCH<sub>2</sub>OCH<sub>2</sub>C<sub>6</sub><i>H</i><sub>5</sub>), 7.17 (d, 1H, H-6-uridine, <i>J</i><sub>6,5</sub> = 8.2 Hz), 6.30 (s, 1H, H-5′-uracil), 6.29 (d, 1H, H-1′-uridine, <i>J</i><sub>H-1′-uridine,H-2′-uridine</sub> = 6.0 Hz), 5.86 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 8.2 Hz), 5.50 (d, 2H, NC<i>H</i><sub>2</sub>OBn, <i>J</i><sub>gem</sub> = 6.4 Hz), 4.69 (s, 2H, NCH<sub>2</sub>OC<i>H</i><sub>2</sub>Ph), 4.40 (d, 1H, H-3′-uridine, <i>J</i><sub>H-3′-uridine,H-2′-uridine</sub> = 4.1 Hz), 4.17 (dd, 1H, H-2′-uridine, <i>J</i><sub>H-2′-uridine,H-1′-uridine</sub> = 6.0, <i>J</i><sub>H-2′-uridine,H-3′-uridine</sub> = 4.1 Hz), 4.05 (dd, 1H, H-α-DABA, <i>J</i><sub>H-α-DABA,H-β-DABA</sub> = 6.9 Hz), 2.74 (dq, 1H, H-β-DABA, <i>J</i><sub>H-β-DABA,H-α-DABA</sub> = 6.9, <i>J</i><sub>H-β-DABA,H-γ-DABA</sub> = 6.4 Hz), 2.38 (s, 3H, NC<i>H</i><sub>3</sub>), 1.41 (s, 9H, Boc-<sup><i>t</i></sup>Bu), 1.13 (d, 3H, H-γ-DABA, <i>J</i><sub>H-γ-DABA,H-β-DABA</sub> = 6.4 Hz), 0.90 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.83 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.11 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.09 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.03 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.03 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz): δ 168.4, 162.4, 156.5, 151.2, 141.4, 138.3, 137.8, 128.4, 127.8, 103.2, 100.9, 89.6, 75.8, 72.2, 71.2, 70.5, 33.8, 28.4, 25.9, 25.7, 18.3, 18.0, 16.2, −4.1, −4.4, −4.9; ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>39</sub>H<sub>66</sub>N<sub>5</sub>O<sub>9</sub>Si<sub>2</sub>, 804.44; found, 805.0; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>39</sub>H<sub>66</sub>N<sub>5</sub>O<sub>9</sub>Si<sub>2</sub>, 804.4394; found, 804.4399; [α]<sub class="stack">D</sub><sup class="stack">19</sup> −28.1 (<i>c</i> 0.75, CHCl<sub>3</sub>).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Compound <b>16</b></h3><div class="NLM_p last">A mixture of <b>15</b> (2.62 g, 3.25 mmol) and AllocCl (414.8 μmol, 3.90 mmol) in THF/sat. aq NaHCO<sub>3</sub> (1/1, 40 mL) was vigorously stirred at 0 °C for 5 h. The reaction was quenched with MeOH, and the organic solvents were removed under reduced pressure. The resulting aqueous phase was extracted with AcOEt, which was washed with H<sub>2</sub>O and brine, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The residue was purified by silica gel column chromatography (ϕ 5 × 10 cm, hexane/AcOEt = 10–30%) to afford <b>16</b> (2.7 mg, 3.0 μmol, 92%) as a white foam. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 500 MHz, 1:1 mixture of rotamers): δ 7.70 (d, 1H, H-6-uridine, <i>J</i><sub>H-6-uridine,H-5-uridine</sub> = 8.0 Hz), 7.68 (d, 1H, H-6-uridine, <i>J</i><sub>H-6-uridine,H-5-uridine</sub> = 8.0 Hz), 7.33–7.28 (m, 5H, NCH<sub>2</sub>OCH<sub>2</sub>C<sub>6</sub><i>H</i><sub>5</sub>), 6.40 (d, 1H, H-1′-uridine, <i>J</i><sub>H-1′-uridine,H-2′-uridine</sub> = 6.3 Hz), 6.35 (d, 1H, H-1′-uridine, <i>J</i><sub>H-1′-uridine,H-2′-uridine</sub> = 6.9 Hz), 6.25 (s, 1H, H-5′-uridine), 5.97–5.88 (m, 2H, H-5-uridine, C<i>H</i>-Alloc), 5.48 (s, 5H, C<i>H</i><sub>2</sub>OBn), 5.31, 5.29 (each d, 1H, C<i>H</i>-Alloc, <i>J</i> = 17.2, 16.6 Hz), 5.18 (d, 1H, C<i>H</i>-Alloc, <i>J</i> = 10.3 Hz), 4.65 (s, 2H, NCH<sub>2</sub>OC<i>H</i><sub>2</sub>Ph), 4.58–4.51 (m, 4H, H-2′-uridine, H-3′-uridine, C<i>H</i><sub>2</sub>-Alloc), 4.44–4.28 (m, 1H, H-α-DABA), 4.37 (m, 1H, H-β-DABA), 2.81 (s, 3H, NC<i>H</i><sub>3</sub>), 1.42 (s, 9H, Boc-<sup><i>t</i></sup>Bu), 1.21 (m, 3H, H-γ-DABA), 0.94 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.85 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.17 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.14 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.08 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.02 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); <sup>13</sup>C NMR (CD<sub>3</sub>OD, 125 MHz, 1:1 mixture of rotamers): δ 169.7, 164.2, 157.6, 152.8, 144.0, 141.4, 141.2, 139.1, 134.2, 129.3, 128.8, 128.7, 117.9, 117.6, 1103.7, 103.5, 101.4, 101.3, 91.0, 90.3, 81.0, 76.5, 76.1, 73.0, 72.7, 71.5, 67.6, 67.3, 58.4, 54.5, 30.4, 28.7, 26.4, 26.2, 19.1, 18.8, 15.2, 14.3, −3.9. −4.2, −4.3, −4.8; ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>43</sub>H<sub>70</sub>N<sub>5</sub>O<sub>11</sub>Si<sub>2</sub>, 888.46; found, 888.4; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>43</sub>H<sub>70</sub>N<sub>5</sub>O<sub>11</sub>Si<sub>2</sub>, 888.4605; found, 888.4598; [α]<sub class="stack">D</sub><sup class="stack">24</sup> −0.65 (<i>c</i> 1.06, CHCl<sub>3</sub>).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Compound <b>17</b></h3><div class="NLM_p last">A solution of <b>16</b> (444 mg, 500 μmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) was treated with 1 M BCl<sub>3</sub> in CH<sub>2</sub>Cl<sub>2</sub> (5 mL, 5 mmol) at −78 °C for 30 min and then at 0 °C for 15 min. After sat. aq NaHCO<sub>3</sub> was added to the mixture, the resulting biphasic mixture was vigorously stirred for 15 min. The mixture was extracted with CH<sub>2</sub>Cl<sub>2</sub>, which was washed with sat. aq NaHCO<sub>3</sub>, H<sub>2</sub>O, and brine. The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The residue was semipurified by silica gel column chromatography (ϕ 2 × 10 cm, CHCl<sub>3</sub>/MeOH = 0–1–2–5%) to afford amine (177.9 mg, 266.3 μmol, 53%) as a white foam. This amine was immediately used in the next reaction. A mixture of the amine (47.3 mg, 70.7 μmol), urea-dipeptide <b>4</b> (60.4 mg, 106.1 μmol), HOBt (14.3 mg, 106.1 μmol), and <sup><i>i</i></sup>Pr<sub>2</sub>NEt (37.0 μL, 212.2 μmol) in CH<sub>2</sub>Cl<sub>2</sub> (1 mL) was treated with EDCI (20.4 mg, 106.1 mmol) at room temperature for 12 h. The reaction mixture was diluted with AcOEt and washed with 0.1 M aq HCl, sat. aq NaHCO<sub>3</sub>, H<sub>2</sub>O, and brine. The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The residue was purified by flash silica gel column chromatography (ϕ 1.5 × 10 cm, CHCl<sub>3</sub>/MeOH = 0–1–2%) to afford <b>17</b> (64.2 mg, 53 μmol, 75%) as a colorless amorphous solid. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 500 MHz, 1:1 mixture of rotamers): δ 7.68, 7.61 (each d, 1H, H-6, <i>J</i><sub>6,5</sub> = 8.0, 8.0 Hz), 7.14 (dd, 1H, H-5-<i>m</i>Tyr, <i>J</i><sub>5-<i>m</i>Tyr,4-<i>m</i>Tyr</sub> = <i>J</i><sub>5-<i>m</i>Tyr,6-<i>m</i>Tyr</sub> = 7.6 Hz), 6.80 (d, 1H, H-6-<i>m</i>Tyr, <i>J</i><sub>6-<i>m</i>Tyr,5-<i>m</i>Tyr</sub> = 7.6 Hz), 6.74 (s, 1H, H-2-<i>m</i>Tyr), 6.74 (d, 1H, H-4-<i>m</i>Tyr, <i>J</i><sub>4-<i>m</i>Tyr,5-<i>m</i>Tyr</sub> = 7.6 Hz), 6.38 (d, 1H, H-1′-uridine, <i>J</i><sub>H-1′-uridine,H-2′-uridine</sub> = 6.9 Hz), 6.33 (d, 1H, H-1′-uridine, <i>J</i><sub>H-1′-uridine,H-2′-uridine</sub> = 6.9 Hz), 6.26 (s, 1H, H-5′-uridine), 6.24 (s, 1H, H-5′-uridine), 5.94 (ddt, 1H, C<i>H</i>-Alloc, <i>J</i> = 16.6, 10.9, 5.7 Hz), 5.86 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 8.0 Hz), 5.80 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 8.0 Hz), 5.31 (d, 1H, C<i>H</i>-Alloc, <i>J</i> = 16.6 Hz), 5.28 (d, 1H, C<i>H</i>-Alloc, <i>J</i> = 16.6 Hz), 5.18 (d, 1H, C<i>H</i>-Alloc, <i>J</i> = 10.9 Hz), 4.71 (d, 1H, H-α-DABA, <i>J</i><sub>H-α-DABA,H-β-DABA</sub> = 8.0 Hz), 4.63 (d, 1H, H-α-DABA, <i>J</i><sub>H-α-DABA,H-β-DABA</sub> = 10.3 Hz), 4.58–4.52 (m, 3H, H-3′-uridine, C<i>H</i><sub>2</sub>-Alloc), 4.48–4.37 (m, 2H, H-2′-uridine, H-β-DABA), 4.41 (t, 1H, H-α-<i>m</i>Tyr, <i>J</i><sub>α-<i>m</i>Tyr,β-<i>m</i>Tyr</sub> = 6.9 Hz), 4.35 (m, 1H, H-α-Met), 2.92 (d, 2H, H-β-<i>m</i>Tyr, <i>J</i><sub>β-<i>m</i>Tyr,α-<i>m</i>Tyr</sub> = 6.9 Hz), 2.81 (s, 3H, NC<i>H</i><sub>3</sub>), 2.80 (s, 3H, H-NC<i>H</i><sub>3</sub>), 2.47 (m, 2H, H-γ-Met), 2.04 (s, 3H, SC<i>H</i><sub>3</sub>-Met), 1.98 (dddd, 1H, H-β-Met, <i>J</i><sub>gem</sub> = <i>J</i><sub>β-Met,α-Met</sub> = 14.2, <i>J</i><sub>β-Met,γ-Met</sub> = 6.9, 2.3 Hz), 1.83 (ddd, 1H, H-β-Met, <i>J</i><sub>gem</sub> = <i>J</i><sub>β-Met,α-Met</sub> = 14.2, <i>J</i><sub>β-Met,γ-Met</sub> = 8.0 Hz), 1.39 (s, 9H, <sup><i>t</i></sup>Bu), 1.29–1.24 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.21 (d, 3H, H-γ-DABA, <i>J</i><sub>H-γ-DABA,H-β-DABA</sub> = 6.9 Hz), 1.10 (d, 18H, Si[CH(C<i>H</i><sub>3</sub>)<sub>2</sub>]<sub>3</sub>, <i>J</i> = 7.5 Hz), 0.92 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.88 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.15 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.12 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.08 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); <sup>13</sup>C NMR (CD<sub>3</sub>OD, 125 MHz, 1:1 mixture of rotamers): δ 174.6, 173.0, 169.1, 168.9, 165.5, 159.4, 157.3, 152.3, 152.3, 144.1, 144.0, 142.1, 139.7, 134.3, 130.3, 123.5, 122.1, 119.0, 118.0, 117.6, 104.0, 101.3, 89.3, 82.8, 76.6, 76.3, 72.7, 67.6, 67.3, 56.7, 56.2, 54.6, 54.1, 39.4, 33.7, 30.9, 30.5, 28.3, 27.9, 26.3, 26.2, 19.0, 18.8, 18.5, 15.4, 14.5, 13.9, −3.9, −4.2, −4.3, −4.9; ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>58</sub>H<sub>100</sub>N<sub>7</sub>O<sub>13</sub>SSi<sub>3</sub>, 1218.64; found, 1218.7; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>58</sub>H<sub>100</sub>N<sub>7</sub>O<sub>13</sub>SSi<sub>3</sub>, 1218.6402; found, 1218.6417; [α]<sub class="stack">D</sub><sup class="stack">23</sup> +15.5 (<i>c</i> 0.89, CHCl<sub>3</sub>).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Compound <b>18</b></h3><div class="NLM_p last">A solution of <b>17</b> (44.6 mg, 36.6 μmol) and morpholine (12.6 μL, 146.4 μmol) in THF (400 μL) was treated with Pd(PPh<sub>3</sub>)<sub>4</sub> (21.1 mg, 18.3 μmol) at room temperature for 2 h. After SH silica gel was added to the mixture, the whole was stirred for 1 h. The SH silica gel was filtered off, and the filtrate was concentrated <i>in vacuo</i>. A mixture of the crude amine, Boc-<i>m</i>Tyr(TIPS)-OH (<b>5</b>) (32.0 mg, 73.2 μmol), and <sup><i>i</i></sup>Pr<sub>2</sub>NEt (25.6 μL, 146.4 μmol) in DMF (400 μL) was treated with HATU (27.8 mg, 73.2 μmol) at room temperature for 5 h. The reaction mixture was diluted with AcOEt/hexane (4/1), which was washed with 0.1 M aq HCl, sat. aq NaHCO<sub>3</sub>, H<sub>2</sub>O, and brine. The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The residue was purified by flash silica gel column chromatography (ϕ 1 × 10 cm, CHCl<sub>3</sub>/MeOH = 0–1–2–5%, then hexane/AcOEt = 50%) to afford <b>18</b> (24.8 mg, 16.0 μmol, 44%) as a colorless amorphous solid. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 500 MHz, 3:2 mixture of rotamers, selected data for the major rotamer): δ 7.38 (d, 1H, H-6-uridine, <i>J</i><sub>H-6-uridine,H-5-uridine</sub> = 8.0 Hz), 7.18–7.11 (m, 2H, H-5-<i>m</i>Tyr), 6.88 (d, 1H, H-4-<i>m</i>Tyr, <i>J</i><sub>4-<i>m</i>Tyr,5-<i>m</i>Tyr</sub> = 7.5 Hz), 6.83 (br s, 1H, H-2-<i>m</i>Tyr), 6.81–6.68 (m, 4H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-6-<i>m</i>Tyr), 6.49 (d, 1H, NH-<i>m</i>Tyr, <i>J</i><sub>NH-<i>m</i>Tyr,α-<i>m</i>Tyr</sub> = 8.6 Hz), 6.24 (s, 1H, H-5′-uridine), 6.19 (d, 1H, H-1′-uridine, <i>J</i><sub>H-1′-uridine,H-2′-uridine</sub> = 7.5 Hz), 5.73 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 8.0 Hz), 4.91–4.84 (m, 1H, H-β-DABA), 4.71 (d, 1H, H-α-DABA, <i>J</i><sub>H-α-DABA,H-β-DABA</sub> = 8.0 Hz), 4.68 (ddd, 1H, H-α-<i>m</i>Tyr, <i>J</i><sub>α-<i>m</i>Tyr,β-<i>m</i>Tyr</sub> = 10.3, <i>J</i><sub>α-<i>m</i>Tyr,NH-<i>m</i>Tyr</sub> = 8.6, <i>J</i><sub>α-<i>m</i>Tyr,β-<i>m</i>Tyr</sub> = 3.4 Hz), 4.49 (d, 1H, H-3′-uridine, <i>J</i><sub>H-3′-uridine,H-2′-uridine</sub> = 4.0 Hz), 4.42–4.47 (m, 2H, H-2′-uridine, H-α-<i>m</i>Tyr), 4.37–4.30 (m, 1H, H-α-Met), 3.07 (s, 3H, NC<i>H</i><sub>3</sub>), 2.92 (d, 1H, H-β-<i>m</i>Tyr, <i>J</i><sub>β-<i>m</i>Tyr,α-<i>m</i>Tyr</sub> = 6.9 Hz), 2.90–2.85 (m, 2H, H-β-<i>m</i>Tyr, H-β′-<i>m</i>Tyr), 2.59 (dd, 1H, H-β-<i>m</i>Tyr, <i>J</i><sub>gem</sub> = 14.3, <i>J</i><sub>β-<i>m</i>Tyr,α-<i>m</i>Tyr</sub> = 10.3 Hz), 2.51–2.45 (m, 2H, H-γ-Met), 2.05 (s, 3H, SC<i>H</i><sub>3</sub>-Met), 2.02–1.95 (m, 1H, H-β-Met), 1.87–1.80 (m, 1H, H-β-Met), 1.39 (s, 9H, <sup><i>t</i></sup>Bu), 1.34 (s, 9H, Boc-<sup><i>t</i></sup>Bu), 1.31–1.20 (m, 9H, H-γ-DABA, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.12–1.10 (d, 36H, Si[CH(C<i>H</i><sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 0.90 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.85 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.12 (s, 3H, <sup><i>t</i></sup>BuCSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.10 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.05 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); <sup>13</sup>C NMR (CD<sub>3</sub>OD, 125 MHz, 1:1 mixture of rotamers): δ 174.8, 174.7, 174.4, 174.0, 173.1, 173.0, 168.7, 165.6, 165.5, 159.5, 159.4, 157.6, 157.5, 157.4, 157.3, 115.2, 143.9, 143.7, 142.5, 140.4, 139.7, 139.7, 139.4, 130.7, 130.5, 130.4, 123.6, 123.4, 123.2, 122.2, 122.1, 122.1, 104.0, 101.5, 101.4, 91.0, 82.8, 80.9, 80.5, 76.5, 75.9, 72.9, 72.6, 56.9, 56.6, 56.2, 56.1, 55.8, 54.3, 53.7, 53.0, 52.3, 40.3, 39.5, 39.3, 38.4, 33.5, 33.3, 31.0, 30.8, 28.9, 28.7, 28.6, 28.3, 26.4, 26.3, 26.2, 19.1, 18.8, 18.5, 18.5, 18.2, 18.5, 15.4, 14.3, 14.2, 13.9, 13.7, −3.8, −4.1, −4.2, −4.9, −4.9; ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>77</sub>H<sub>133</sub>N<sub>8</sub>O<sub>15</sub>SSi<sub>2</sub>, 1553.87; found, 1554.6; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>77</sub>H<sub>133</sub>N<sub>8</sub>O<sub>15</sub>SSi<sub>2</sub>, 1553.8683; found, 1553.8624; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +26.1 (<i>c</i> 0.70, CHCl<sub>3</sub>).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 3′-hMRD A (<b>2</b>) and Sulfoxide Analogue <b>19</b></h3><div class="NLM_p last">A solution of <b>18</b> (17.0 mg, 10.9 μmol) in MeCN (1 mL) was treated with 3HF·Et<sub>3</sub>N (71.3 μL, 437.5 μmol) at room temperature for 50 h. The reaction mixture was concentrated, and the residue was diluted with AcOEt. The organic layer was washed with sat. aq NaHCO<sub>3</sub>, H<sub>2</sub>O, and brine, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The resulting residue was treated with 80% aq TFA at room temperature for 2 h. After toluene was added, the mixture was concentrated <i>in vacuo</i>. The residue was triturated with Et<sub>2</sub>O, and the precipitate was collected and purified by ODS column chromatography (H<sub>2</sub>O/MeCN = 0–30%, containing 0.1% TFA) to afford 3′-hMRD A (<b>2</b>) (4.2 mg, 4.9 μmol, 45% over two steps) and sulfoxide analogue <b>19</b> (3.3 mg, 3.8 μmol, 35% over two steps) as a white powder after lyophilization. <sup>1</sup>H NMR (D<sub>2</sub>O, 500 MHz, 3:1 mixture of rotamers, selected data for the major rotamer): δ 7.30 (1H, H-5-<i>m</i>Tyr, H-5′-<i>m</i>Tyr, <i>J</i><sub>5-<i>m</i>Tyr,6-<i>m</i>Tyr</sub> = <i>J</i><sub>5′-<i>m</i>Tyr,6′-<i>m</i>Tyr</sub> = <i>J</i><sub>5-<i>m</i>Tyr,4-<i>m</i>Tyr</sub> = <i>J</i><sub>5-<i>m</i>Tyr,4′-<i>m</i>Tyr</sub> = 7.5 Hz), 7.22 (1H, H-5-<i>m</i>Tyr, H-5′-<i>m</i>Tyr, <i>J</i><sub>5-<i>m</i>Tyr,6-<i>m</i>Tyr</sub> = <i>J</i><sub>5′-<i>m</i>Tyr,6′-<i>m</i>Tyr</sub> = <i>J</i><sub>5-<i>m</i>Tyr,4-<i>m</i>Tyr</sub> = <i>J</i><sub>5-<i>m</i>Tyr,4′-<i>m</i>Tyr</sub> = 7.5 Hz), 7.10 (d, 1H, H-6-uridine, <i>J</i><sub>H-6-uridine,H-5-uridine</sub> = 8.0 Hz), 6.87–6.73 (m, 6H, H-6-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-2-<i>m</i>Tyr), 6.23 (s, 1H, H-5′-uridine), 6.06 (d, 1H, H-1′-uridine, <i>J</i><sub>H-1′-uridine,H-2′-uridine</sub> = 5.2 Hz), 5.65 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 8.0 Hz), 4.95 (dq, 1H, H-β-DABA, <i>J</i><sub>H-β-DABA,H-α-DABA</sub> = 9.7, <i>J</i><sub>H-β-DABA,H-γ-DABA</sub> = 6.9 Hz), 4.72 (d, 1H, H-3′-uridine, <i>J</i><sub>H-3′-uridine,H-2′-uridine</sub> = 5.2 Hz), 4.63 (d, 1H, H-α-DABA, <i>J</i><sub>H-α-DABA,H-β-DABA</sub> = 9.2 Hz), 4.58 (dd, 1H, H-α-<i>m</i>Tyr, <i>J</i><sub>α-<i>m</i>Tyr,β-<i>m</i>Tyr</sub> = 10.0, <i>J</i><sub>α-<i>m</i>Tyr,β-<i>m</i>Tyr</sub> = 4.0 Hz), 4.36 (dd, 1H, H-α-<i>m</i>Tyr, <i>J</i><sub>α-<i>m</i>Tyr,β-<i>m</i>Tyr</sub> = 8.3, <i>J</i><sub>α-<i>m</i>Tyr,β-<i>m</i>Tyr</sub> = 5.7 Hz), 4.27 (dd, 1H, H-α-Met, <i>J</i><sub>H-α-Met,H-β-Met</sub> = 8.0, <i>J</i><sub>H-α-Met,H-β-Met</sub> = 5.7 Hz), 4.24 (dd, 1H, H-2′-uridine, <i>J</i><sub>H-2′-uridine,H-3′-uridine</sub> = <i>J</i><sub>H-2′-uridine,H-1′-uridine</sub> = 5.2 Hz), 3.06 (s, 3H, H-NC<i>H</i><sub>3</sub>), 3.10–3.05 (m, 1H, H-β-<i>m</i>Tyr), 3.02 (dd, 1H, H-β-<i>m</i>Tyr, <i>J</i><sub>gem</sub> = 14.3, <i>J</i><sub>β-<i>m</i>Tyr,α-<i>m</i>Tyr</sub> = 5.7 Hz), 2.88 (dd, 1H, H-β-<i>m</i>Tyr, <i>J</i><sub>gem</sub> = 14.2, <i>J</i><sub>β-<i>m</i>Tyr,α-<i>m</i>Tyr</sub> = 8.3 Hz), 2.73–2.66 (m, 1H, H-β-<i>m</i>Tyr), 2.55–2.42 (m, 2H, H-γ-Met), 2.06 (s, 3H, SC<i>H</i><sub>3</sub>-Met), 2.00–1.95 (m, 2H, H-β-Met), 1.23 (d, 3H, H-γ-DABA, <i>J</i><sub>H-γ-DABA,H-β-DABA</sub> = 6.9 Hz); ESIMS-LR <i>m</i>/<i>z</i>: 857.1 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>38</sub>H<sub>49</sub>N<sub>8</sub>O<sub>13</sub>S, 857.3134; found, 857.3122; [α]<sub class="stack">D</sub><sup class="stack">19</sup> −13.4 (<i>c</i> 0.15, H<sub>2</sub>O). <sup>1</sup>H NMR (D<sub>2</sub>O, 500 MHz, a mixture of diastereomers): δ 7.27 (1H, H-5-<i>m</i>Tyr), 7.20 (1H, H-5-<i>m</i>Tyr), 7.02 (m, 1H, H-6), 6.82–6.70 (m, 6H, H-6-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-2-<i>m</i>Tyr), 6.19 (s, 1H, H-5′-uridine), 6.00 (m, 1H, H-1′-uridine), 5.58 (m, 1H, H-5-uridine), 4.91 (m, 1H, H-β-DABA), 4.72 (m, 1H, H-3′-uridine), 4.55 (m, 2H, H-α-DABA, H-α-<i>m</i>Tyr), 4.38–4.15 (m, 3H, H-α-<i>m</i>Tyr, H-α-Met, H-2′-uridine), 3.08–3.05 (m, 1H, H-β-<i>m</i>Tyr), 3.05 (s, 3H, NC<i>H</i><sub>3</sub>), 2.94–2.80 (m, 4H, H-β-<i>m</i>Tyr, H-γ-Met), 2.71–2.65 (m, 1H, H-β-<i>m</i>Tyr), 2.65 (s, 3H, SOC<i>H</i><sub>3</sub>-Met), 2.13–1.97 (m, 2H, H-β-Met), 1.18 (m, 3H, H-γ-DABA); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>38</sub>H<sub>49</sub>N<sub>8</sub>O<sub>14</sub>S, 873.31; found, 873.1 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>38</sub>H<sub>49</sub>N<sub>8</sub>O<sub>14</sub>S, 873.3083; found, 873.3077.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Compound <b>27</b></h3><div class="NLM_p last">A mixture of <b>15</b> (2.50 g, 3.1 mmol), (<i>S</i>)-2-[(fluorenylmethyl)amino]-3-(3-hydroxyphenyl)propanoic acid (<b>26</b>, 2.09 g, 4.0 mmol), and <sup><i>i</i></sup>Pr<sub>2</sub>NEt (1.35 mL, 7.44 mol) in DMF (30 mL) was treated with HATU (981 mg, 1.56 mmol) at 0 °C for 2 h. The reaction mixture was diluted with AcOEt and washed with 0.1 M aq HCl, sat. aq NaHCO<sub>3</sub>, H<sub>2</sub>O, and brine. The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The residue was purified by flash silica gel column chromatography (ϕ 5 × 15 cm, AcOEt/hexane = 33–55%) to afford <b>27</b> (3.59 g, 2.67 mol, 86%) as a colorless amorphous solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 9.07 (d, 1H, NH-5′-uridine, <i>J</i><sub>NH-5′-uridine,H-5′-uridine</sub> = 10.1 Hz), 7.89–7.84 (m, 4H, H-Fmoc), 7.79–7.69 (m, 2H, H-6-uridine, NH-mTyr), 7.61 (t, 1H, H-Ph, <i>J</i><sub>H-Ph,H-Phe</sub> = 8.5 Hz), 7.42–7.23 (m, 9H, H-Ph, H-Fmoc, NH-DABA), 7.14 (dd, 1H, H-5-<i>m</i>Tyr, <i>J</i><sub>5-<i>m</i>Tyr,4-<i>m</i>Tyr</sub> = 8.3 Hz, <i>J</i><sub>5-<i>m</i>Tyr,6-<i>m</i>Tyr</sub> = 8.3 Hz), 6.91–6.84 (m, 2H, H-2-<i>m</i>Tyr, H-6-<i>m</i>Tyr), 6.69 (d, 1H, H-4-<i>m</i>Tyr, <i>J</i><sub>4-<i>m</i>Tyr,5-<i>m</i>Tyr</sub> = 8.3 Hz), 6.23 (d, 1H, H-1′-uridine, <i>J</i><sub>H-1′-uridine,H-2′-uridine</sub> = 7.2 Hz), 6.20 (d, 1H, H-5′-uridine, <i>J</i><sub>H-5′-uridine,NH-5′-uridine</sub> = 10.1 Hz), 5.87 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 8.7 Hz), 5.33 (m, 2H, H-NC<i>H</i><sub>2</sub>OBn), 4.93–4.81 (m, 1H, H-β-DABA), 4.58–4.40 (m, 6H, H-2′-uridine, H-3′-uridine, H-Fmoc, H-α-mTyr), 4.56 (s, 2H, H-C<i>H</i><sub>2</sub>Ph), 4.25–4.17 (m, 1H, H-α-DABA), 2.98 (s, 3H, NC<i>H</i><sub>3</sub>), 2.71–2.62 (m, 2H, H-β-<i>m</i>Tyr), 1.34 (s, 9H, <sup><i>t</i></sup>Bu), 1.26–1.14 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.07–0.97 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.86 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.76 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.07 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.04 (m, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.01 (m, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.11 (m, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>72</sub>H<sub>104</sub>N<sub>6</sub>NaO<sub>13</sub>Si<sub>3</sub>, 1367.69; found, 1368.5; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +17.4 (<i>c</i> 1.00, CHCl<sub>3</sub>).</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Compound <b>28</b></h3><div class="NLM_p last">A solution of <b>27</b> (2.63 g, 2.00 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (120 mL) was treated with 1 M BCl<sub>3</sub> in CH<sub>2</sub>Cl<sub>2</sub> (20 mL, 20 mmol) at −78 °C for 30 min and then at 0 °C for 15 min. After sat. aq NaHCO<sub>3</sub> was added to the mixture, the resulting biphasic mixture was vigorously stirred for 15 min. The organic layer was separated and washed with sat. aq NaHCO<sub>3</sub>, H<sub>2</sub>O, and brine. The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>) and filtered. The residue was treated with a solution of HCl in AcOEt (4 M, 1.1 mL, 4.4 mmol) for 30 min. Hexane (200 mL) was added to the solution, which was concentrated <i>in vacuo</i> to one-third in volume. Hexane (100 mL) was added to the solution and was concentrated <i>in vacuo</i>. The precipitates were collected by filtration, washed with ether, and dried to afford <b>28</b> (1.31 g, quant.) as a pale brown solid. This compound was directly used in the next reaction without further purification. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 10.04 (s, 1H, NH-5′-uridine), 7.87 (d, 2H, NH-DABA, <i>J</i><sub>NH-DABA,H-α-DABA</sub> = 9.6 Hz), 7.67–6.37 (m, 13H, Fmoc, H-2, 4, 5, 6-<i>m</i>Tyr, NH-DABA), 7.60 (m, 1H, NH-<i>m</i>Tyr), 6.19 (m, 1H, H-1′-uridine), 5.20 (m, 1H, H-5-uridine), 4.92 (m, 1H, H-β-DABA), 4.57 (m, 1H, H-α-DABA), 4.10 (m, 7H, Fmoc, H-3′, 4′, 5′, H-α-<i>m</i>Tyr), 2.95 (s, 3H, NCH<sub>3</sub>), 2.66 (m, 2H, H-β-<i>m</i>Tyr), 1.24 (m, 3H, H-γ-DABA), 1.23–0.77 (m, 39H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, Si[CH(C<i>H</i><sub>3</sub>)<sub>2</sub>]<sub>3</sub>, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.07 (m, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.04 (m, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.06 (m, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.19 (m, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: 1125.6 [(M + Na)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>59</sub>H<sub>89</sub>N<sub>6</sub>O<sub>10</sub>Si<sub>3</sub>, 1125.5942; found, 1125.5940; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +8.6 (<i>c</i> 1.00, CHCl<sub>3</sub>).</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> (<i>S</i>)-{[1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}glycine (<b>29a</b>)</h3><div class="NLM_p last">2-Chlorotrityl chloride resin (300 mg, 0.48 mmol) was placed in a 10 mL polypropylene syringe fitted with a polyethylene filter disc. The resin was washed with CH<sub>2</sub>Cl<sub>2</sub> (2 mL), and a solution of Fmoc-Gly (428 mg, 1.4 mmol) and <sup><i>i</i></sup>Pr<sub>2</sub>NEt (505 μL, 2.9 mmol) in CH<sub>2</sub>Cl<sub>2</sub> was then added. After the mixture was agitated at room temperature for 2 h, the solvent and reagents were removed by suction. The resin bound to the Fmoc-amino acid was subjected to the following washing treatments with CH<sub>2</sub>Cl<sub>2</sub>/MeOH/<sup><i>i</i></sup>Pr<sub>2</sub>NEt (17/2/1, 2 mL × 3), DMF (2 mL × 3), and CH<sub>2</sub>Cl<sub>2</sub> (2 mL × 3). The Fmoc group was removed with piperidine/DMF (1/4, 5 min, then 1/9, 15 min). The resin was washed with DMF (2 mL × 3) and CH<sub>2</sub>Cl<sub>2</sub> (2 mL × 3). A solution of 1,1′-carbonyldiimidazole (234 mg, 1.4 mmol) and <i>N</i>-methylmorpholine (319 μL, 2.9 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) was added to the resin. After the mixture was agitated at room temperature for 1 min, the solvent and reagents were removed by suction. This procedure was repeated 5 times. The resin was washed with CH<sub>2</sub>Cl<sub>2</sub> (2 mL × 3). Then, a solution of <span class="smallcaps smallerCapital">l</span>-Trp-O<sup><i>t</i></sup>Bu·HCl (142 mg, 0.48 mmol) and <sup><i>i</i></sup>Pr<sub>2</sub>NEt (251 μL, 1.4 mmol) in CH<sub>2</sub>Cl<sub>2</sub> was added, and the mixture was agitated at room temperature for 3 h. The resin was washed with DMF (2 mL × 3) and CH<sub>2</sub>Cl<sub>2</sub> (2 mL × 3). Completion of the reaction was confirmed by a Kaiser test. The resin was treated with 5% TFA/CH<sub>2</sub>Cl<sub>2</sub> to cleave the urea-dipeptide, and the filtrate was partitioned between CH<sub>2</sub>Cl<sub>2</sub> and H<sub>2</sub>O. The organic phase was washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i> to afford <b>29a</b> (43 mg, 0.11 mmol, 23%) as a white solid. This material was used for the next reaction without further purification. ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>18</sub>H<sub>23</sub>N<sub>3</sub>NaO<sub>5</sub>, 384.15; found, 383.7.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> {[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-<span class="smallcaps smallerCapital">l</span>-alanine (<b>29b</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29b</b> (123 mg, 0.33 mmol, 69%) was obtained as a white solid using <span class="smallcaps smallerCapital">l</span>-Trp-O<sup><i>t</i></sup>Bu·HCl (142 mg, 0.48 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>19</sub>H<sub>25</sub>N<sub>3</sub>NaO<sub>5</sub>, 398.17; found, 397.9.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> {[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-<span class="smallcaps smallerCapital">l</span>-leucine (<b>29c</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29c</b> (92 mg, 0.22 mmol, 46%) was obtained as a white solid using Fmoc-Leu (509 mg, 1.4 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>22</sub>H<sub>31</sub>N<sub>3</sub>NaO<sub>5</sub>, 440.22; found, 439.8.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> {[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-<span class="smallcaps smallerCapital">l</span>-isoleucine (<b>29d</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29d</b> (54 mg, 0.13 mmol, 54%) was obtained as a white solid using Fmoc-Ile (254 mg, 0.72 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>22</sub>H<sub>31</sub>N<sub>3</sub>NaO<sub>5</sub>, 440.22; found, 440.1.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> {[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-<span class="smallcaps smallerCapital">l</span>-valine (<b>29e</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29e</b> (58 mg, 0.14 mmol, 58%) was obtained as a white solid using Fmoc-Val (244 mg, 0.72 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>21</sub>H<sub>29</sub>N<sub>3</sub>NaO<sub>5</sub>, 426.20; found, 426.0.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> <i>N</i>-{[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-<i>S</i>-trityl-<span class="smallcaps smallerCapital">l</span>-cysteine (<b>29f</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29f</b> (251 mg, 0.38 mmol, 80%) was obtained as a white solid using Fmoc-Cys(Trt) (843 mg, 1.4 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>38</sub>H<sub>39</sub>N<sub>3</sub>NaO<sub>5</sub>S, 672.25; found, 671.6.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> {[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-<span class="smallcaps smallerCapital">l</span>-methionine (<b>29g</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29g</b> (66 mg, 0.15 mmol, 63%) was obtained as a white solid using Fmoc-Met (267 mg, 0.72 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>21</sub>H<sub>29</sub>N<sub>3</sub>NaO<sub>5</sub>S, 458.17; found, 458.1.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> {[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-<span class="smallcaps smallerCapital">l</span>-phenyl-alanine (<b>29h</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29h</b> (72 mg, 0.16 mmol, 67%) was obtained as a white solid using Fmoc-Phe (279 mg, 0.72 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>25</sub>H<sub>29</sub>N<sub>3</sub>NaO<sub>5</sub>, 474.20; found, 474.1.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> (<i>S</i>)-2-{3-[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]ureido}-3-[4-(<i>tert</i>-butoxy)phenyl]propanoic Acid (<b>29i</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29i</b> (67 mg, 0.13 mmol, 54%) was obtained as a white solid using Fmoc-Tyr(O<sup><i>t</i></sup>Bu) (331 mg, 0.72 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>29</sub>H<sub>37</sub>N<sub>3</sub>NaO<sub>6</sub>, 546.26; found, 546.0.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> <i>N</i>α-{[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-1-(<i>tert</i>-butoxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-tryptophan (<b>29j</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29j</b> (69 mg, 0.12 mmol, 50%) was obtained as a white solid using Fmoc-Trp(Boc) (379 mg, 0.72 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>32</sub>H<sub>38</sub>N<sub>4</sub>NaO<sub>7</sub>, 474.20; found, 613.0.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> <i>N</i>α-{[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-<i>N</i>-trityl-<span class="smallcaps smallerCapital">l</span>-histidine (<b>29k</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29k</b> (113 mg, 0.16 mmol, 33%) was obtained as a white solid using Fmoc-His(Trt) (892 mg, 1.4 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>41</sub>H<sub>42</sub>N<sub>5</sub>O<sub>5</sub>, 684.32; found, 683.9.</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> (<i>S</i>)-4-(<i>tert</i>-Butoxy)-2-{3-[(<i>S</i>)-1-(<i>tert</i>-butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]-ureido}-4-oxobutanoic Acid (<b>29l</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29l</b> (145 mg, 0.22 mmol, 46%) was obtained as a white solid using Fmoc-Asp(O<sup><i>t</i></sup>Bu) (592 mg, 1.4 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>33</sub>N<sub>3</sub>NaO<sub>7</sub>, 498.22; found, 498.1.</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> (<i>S</i>)-5-(<i>tert</i>-Butoxy)-2-{3-[(<i>S</i>)-1-(<i>tert</i>-butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]ureido}-5-oxopentanoic Acid (<b>29m</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29m</b> (144 mg, 0.29 mmol, 60%) was obtained as a white solid using Fmoc-Glu(O<sup><i>t</i></sup>Bu) (638 mg, 1.4 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>25</sub>H<sub>35</sub>N<sub>3</sub>NaO<sub>7</sub>, 512.24; found, 511.8.</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> <i>N</i>2-{[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-<i>N</i>6-(<i>tert</i>-butoxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-lysine (<b>29n</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29n</b> (74 mg, 0.14 mmol, 58%) was obtained as a white solid using Fmoc-Lys(Boc) (337 mg, 0.72 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>40</sub>N<sub>4</sub>NaO<sub>7</sub>, 555.28; found, 555.1.</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> <i>N</i>2-{[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-<i>N</i>ω-[(2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-yl)sulfonyl]-<span class="smallcaps smallerCapital">l</span>-arginine (<b>29o</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29o</b> (133 mg, 0.19 mmol, 40%) was obtained as a white solid using Fmoc-Arg(Pbf) (934 mg, 1.4 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>35</sub>H<sub>49</sub>N<sub>6</sub>O<sub>8</sub>S, 713.33; found, 713.0.</div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> <i>N</i>-{[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-<i>O</i>-<i>tert</i>-butyl-<span class="smallcaps smallerCapital">l</span>-serine (<b>29p</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29p</b> (64 mg, 0.14 mmol, 58%) was obtained as a white solid using Fmoc-Ser(O<sup><i>t</i></sup>Bu) (276 mg, 0.72 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>33</sub>N<sub>3</sub>NaO<sub>6</sub>, 470.23; found, 469.9.</div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> {[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-<i>O</i>-<i>tert</i>-butyl-<span class="smallcaps smallerCapital">l</span>-threonine (<b>29q</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29q</b> (63 mg, 0.14 mmol, 57%) was obtained as a white solid using Fmoc-Thr(O<sup><i>t</i></sup>Bu) (286 mg, 0.72 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>35</sub>N<sub>3</sub>NaO<sub>6</sub>, 484.24; found, 484.0.</div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> <i>N</i>2-{[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-<i>N</i>4-trityl-<span class="smallcaps smallerCapital">l</span>-asparagine (<b>29r</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29r</b> (145 mg, 0.22 mmol, 46%) was obtained as a white solid using Fmoc-Asn(Trt) (859 mg, 1.4 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>39</sub>H<sub>40</sub>N<sub>4</sub>NaO<sub>6</sub>, 683.28; found, 683.2.</div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> <i>N</i>2-{[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-<i>N</i>5-trityl-<span class="smallcaps smallerCapital">l</span>-glutamine (<b>29s</b>)</h3><div class="NLM_p last">In a manner similar to the synthesis of <b>29a</b>, <b>29s</b> (66 mg, 0.09 mmol, 19%) was obtained as a white solid using Fmoc-Gln(Trt) (880 mg, 1.4 mmol). ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>40</sub>H<sub>42</sub>N<sub>4</sub>NaO<sub>6</sub>, 697.30; found, 696.8.</div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> {[(<i>S</i>)-1-(<i>tert</i>-Butoxy)-3-(1<i>H</i>-indol-3-yl)-1-oxopropan-2-yl]carbamoyl}-<span class="smallcaps smallerCapital">l</span>-proline (<b>29t</b>)</h3><div class="NLM_p last">A solution of <span class="smallcaps smallerCapital">l</span>-Trp(Boc)·HCl (100 mg, 0.33 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3.3 mL) was added dropwise to a solution of triphosgene (76 mg, 0.25 mmol) and Et<sub>3</sub>N (460 μL, 3.3 mmol) at −78 °C, and the mixture was stirred for 7 h. A solution of <span class="smallcaps smallerCapital">l</span>-Pro-OH (37 mg, 0.66 mmol) in DMF (mL) was added to the mixture, which was stirred for 17 h. After H<sub>2</sub>O was added, the whole mixture was partitioned between AcOEt and 1 M aq HCl. The organic phase was washed with H<sub>2</sub>O and brine, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The residue was purified by silica gel chromatography to afford <b>29t</b> (20 mg, 0.05 mmol, 15%) as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 10.82 (s, 1H, NH-δ1-Trp), 7.53 (d, 1H, H-ε3-Trp, <i>J</i><sub>H-ε3-Trp,H-ζ2-Trp</sub> = 9.0 Hz), 7.33 (d, 1H, H-ζ1-Trp, <i>J</i><sub>H-ζ1-Trp,H-η-Trp</sub> = 8.1 Hz), 7.15 (s, 1H, H-δ1-Trp), 7.06 (dd, 1H, H-η-Trp, <i>J</i><sub>H-η-Trp,H-ζ1-Trp</sub> = <i>J</i><sub>H-η-Trp,H-ζ2-Trp</sub> = 8.1 Hz), 6.97 (dd, 1H, H-ζ2-Trp, <i>J</i><sub>H-ζ2-Trp,H-η-Trp</sub> = 8.1, <i>J</i><sub>H-ζ2-Trp,H-ε3-Trp</sub> = 9.0 Hz), 4.30–3.98 (m, 2H, H-α-Pro, H-α-Trp), 3.17–2.96 (m, 4H, H-β-Trp, H-δ-Pro), 1.93–1.74 (m, 4H, H-β-Pro, H-γ-Pro), 1.27 (s, 9H, <sup><i>t</i></sup>Bu); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>21</sub>H<sub>27</sub>N<sub>3</sub>NaO<sub>5</sub>, 424.18; found, 424.0.</div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Compound <b>30a</b></h3><div class="NLM_p last">A solution of <b>29a</b> (68 mg, 0.10 mmol), <b>28</b> (56 mg, 0.05 mmol), and <sup><i>i</i></sup>Pr<sub>2</sub>NEt (58 μL, 0.33 mmol) in DMF (1 mL) was treated with HBTU (42 mg, 0.11 mmol) at room temperature for 2 h. The mixture was partitioned between AcOEt and 1 M aq HCl and brine. The organic phase was dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. The residue was purified by silica gel column chromatography to afford <b>30a</b> (16 mg, 0.01 mmol, 10%) as a pale yellow syrup. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.41 (s, 1H, H-N3-uridine), 10.76 (s, 1H, NH-δ1-Trp), 9.40–9.31 (m, 1H, NH-5′-uridine), 8.17–8.04 (m, 1H, DABA-NH), 7.90–7.79 (m, 2H, H-Fmoc), 7.71–7.45 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.43–7.20 (m, 5H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp), 7.19–6.60 (m, 6H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.42 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 8.1 Hz), 6.42–6.29 (m, 1H, NH-Gly), 6.22–6.08 (m, 2H, H-1′-uridine, H-5′-uridine), 5.67 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 8.2 Hz), 5.06–4.71 (m, 1H, H-β-DABA), 4.68–4.58 (m, 1H, H-α-DABA), 4.57–4.02 (m, 7H, H-Fmoc, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.68 (br s, 2H, H-α-Gly), 3.08–2.84 (m, 2H, H-β-Trp), 3.02 (s, 3H, NC<i>H</i><sub>3</sub>), 2.81–2.58 (m, 2H, H-β-<i>m</i>Tyr), 1.27 (s, 9H, <sup><i>t</i></sup>Bu), 1.31–1.14 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.07–0.97 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.86 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.80 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.07 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.04 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.10 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>77</sub>H<sub>109</sub>N<sub>9</sub>NaO<sub>14</sub>Si<sub>3</sub>, 1492.73; found, 1490.8; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −2.0 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Compound <b>30b</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30b</b> (28 mg, 0.018 mmol, 38%) was obtained as a pale yellow syrup using <b>29b</b> (20 mg, 0.05 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.44 (s, 1H, H-N3-uridine), 10.77 (s, 1H, NH-δ1-Trp), 9.02 (d, 1H, NH-5′-uridine, <i>J</i><sub>NH-5-uridine,H-5′-uridine</sub> = 8.7 Hz), 8.02 (d, 1H, NH-DABA, <i>J</i><sub>NH-DABA,H-α-DABA</sub> = 9.6 Hz), 7.90–7.80 (m, 2H, H-Fmoc), 7.71–7.45 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.43–7.20 (m, 5H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp), 7.20–6.62 (m, 6H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.39 (d, 1H, NH-Ala, <i>J</i><sub>NH-Ala,H-α-Ala</sub> = 6.7 Hz), 6.27 (d, 1H, NH-Trp, <i>J</i><sub>NH-Trp,α-Trp</sub> = 7.6 Hz), 6.22–6.05 (m, 2H, H-1′-uridine, H-5′-uridine), 5.64 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 7.6 Hz), 4.64–4.05 (m, 9H, H-Fmoc, H-α-DABA, H-α-Ala, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.08–2.92 (m, 5H, H-β-Trp, NC<i>H</i><sub>3</sub>), 2.80–2.58 (m, 2H, H-β-<i>m</i>Tyr), 1.28 (s, 9H, <sup><i>t</i></sup>Bu), 1.32–1.14 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.12 (d, 3H, H-β-Ala, <i>J</i><sub>H-β-Ala,H-α-Ala</sub> = 6.7 Hz), 1.07–0.94 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.86 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.80 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.07 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.04 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.00 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.08 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>78</sub>H<sub>111</sub>N<sub>9</sub>NaO<sub>14</sub>Si<sub>3</sub>, 1504.75; found, 1504.8; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −2.6 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Compound <b>30c</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30c</b> (26 mg, 0.017 mmol, 17%) was obtained as a pale yellow syrup using <b>29c</b> (42 mg, 0.10 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.39 (s, 1H, H-N3-uridine), 10.76 (s, 1H, NH-δ1-Trp), 9.21–9.08 (m, 1H, NH-5′-uridine), 8.30–8.24 (m, 1H, DABA-NH), 7.90–7.79 (m, 2H, H-Fmoc), 7.70–7.47 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.47–7.19 (m, 5H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp), 7.19–6.59 (m, 6H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.38–6.30 (m, 1H, NH-α-Trp), 6.25–6.04 (m, 2H, H-1′-uridine, H-5′-uridine, NH-Leu), 5.66–5.58 (m, 1H, H-5-uridine), 5.06–4.85 (m, 1H, H-β-DABA), 4.57–4.02 (m, 8H, H-α-DABA, H-Fmoc, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.08–2.84 (m, 5H, H-β-Trp, NC<i>H</i><sub>3</sub>), 2.79–2.60 (m, 2H, H-β-<i>m</i>Tyr), 1.71–1.12 (m, 6H, H-β-Leu, H-γ-Leu, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.27 (s, 9H, <sup><i>t</i></sup>Bu), 1.01–0.92 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.92–0.66 (m, 24H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>, H-δ-Leu), 0.80 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.07 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.04 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.08 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>81</sub>H<sub>117</sub>N<sub>9</sub>NaO<sub>14</sub>Si<sub>3</sub>, 1546.79; found, 1547.8; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −6.5 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Compound <b>30d</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30d</b> (27 mg, 0.017 mmol, 18%) was obtained as a pale yellow syrup using <b>29d</b> (42 mg, 0.10 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 10.75 (s, 1H, NH-δ1-Trp), 9.09–9.02 (m, 1H, NH-5′-uridine), 8.37–8.25 (m, 1H, DABA-NH), 7.91–7.78 (m, 2H, H-Fmoc), 7.71–7.46 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.44–7.22 (m, 5H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp), 7.20–6.62 (m, 6H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.38–6.28 (m, 1H, NH-Ile), 6.28–6.07 (m, 3H, NH-α-Trp, H-1′-uridine, H-5′-uridine), 5.67–5.57 (m, 1H, H-5-uridine), 5.91–4.87 (m, 1H, H-β-DABA), 4.63–3.94 (m, 9H, H-α-DABA, H-Fmoc, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine, H-α-Ile), 3.06–2.84 (m, 5H, H-β-Trp, NC<i>H</i><sub>3</sub>), 2.77–2.60 (m, 2H, H-β-<i>m</i>Tyr), 1.68–1.52 (m, 1H, H-β-Ile), 1.68–1.52 (m, 1H, H-γ-Ile), 1.51–1.33 (m, 1H, H-γ-Ile), 1.32–1.14 (m, 7H, H-γ-Ile, H-δ-Ile, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.27 (s, 9H, <sup><i>t</i></sup>Bu), 1.07–0.97 (m, 24H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA, H-β-Me-Ile), 0.88–0.71 (m, 21H, H-γ-Me-Ile, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.07 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.03 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.08 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>81</sub>H<sub>118</sub>N<sub>9</sub>O<sub>14</sub>Si<sub>3</sub>, 1524.81; found, 1525.0; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −10.7 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Compound <b>30e</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30e</b> (88 mg, 0.06 mmol, 58%) was obtained as a pale yellow syrup using <b>29e</b> (40 mg, 0.10 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.44 (s, 1H, H-N3-uridine), 10.76 (s, 1H, NH-δ1-Trp), 9.03 (d, 1H, NH-5′-uridine, <i>J</i><sub>NH-5′-uridine,H-5′-uridine</sub> = 9.9 Hz), 8.30–8.25 (m, 1H, DABA-NH), 7.90–7.81 (m, 2H, H-Fmoc), 7.70–7.47 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.46–7.21 (m, 5H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp), 7.16–6.62 (m, 6H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.34 (d, 1H, NH-Val, <i>J</i><sub>NH-Val,H-α-Val</sub> = 9.4 Hz), 6.30 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 7.6 Hz), 6.18 (d, 2H, H-1′-uridine, <i>J</i><sub>H-1′-uridine,H-2′-uridine</sub> = 7.2 Hz), 6.18 (d, 2H, H-5′-uridine, <i>J</i><sub>H-5′-uridine,NH-5′-uridine</sub> = 9.9 Hz), 5.65 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 7.2 Hz), 5.01–4.89 (m, 1H, H-β-DABA), 4.64–4.55 (m, 1H, H-α-DABA), 4.55–4.04 (m, 7H, H-Fmoc, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 4.04–3.94 (m, 1H, H-α-Val), 3.04–2.91 (m, 2H, H-β-Trp), 2.98 (s, 3H, H-NC<i>H</i><sub>3</sub>), 2.80–2.55 (m, 2H, H-β-<i>m</i>Tyr), 1.29 (s, 9H, <sup><i>t</i></sup>Bu), 1.30–1.14 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.07–0.97 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.91–0.71 (s, 24H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>, H-γ-Val), 0.07 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.04 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.08 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>80</sub>H<sub>115</sub>N<sub>9</sub>NaO<sub>14</sub>Si<sub>3</sub>, 1532.78; found, 1531.7; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −10.2 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Compound <b>30f</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30f</b> (14 mg, 0.008 mmol, 16%) was obtained as a pale yellow syrup using <b>29f</b> (65 mg, 0.10 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.44 (s, 1H, H-N3-uridine), 10.74 (s, 1H, NH-δ1-Trp), 9.58 (d, 1H, NH-5′-uridine, <i>J</i><sub>NH-5′-uridine,H-5′-uridine</sub> = 10.3 Hz), 8.18 (m, 1H, DABA-NH, <i>J</i><sub>NH-DABA,H-α-DABA</sub> = 9.6 Hz), 7.90–7.82 (m, 2H, H-Fmoc), 7.71–7.45 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.43–7.61 (m, 26H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp, Trt), 6.54 (d, 1H, NH-Cys, <i>J</i><sub>NH-Cys,H-α-Cys</sub> = 8.1 Hz), 6.46 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 8.1 Hz), 6.23–6.14 (m, 2H, H-1′-uridine, H-5′-uridine), 5.55 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 9.0 Hz), 4.97–4.82 (m, 1H, H-β-DABA), 4.74–4.65 (m, 1H, H-α-DABA), 4.58–4.05 (m, 8H, H-Fmoc, H-α-Cys, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.08–2.93 (m, 2H, H-β-Trp), 2.96 (s, 3H, NC<i>H</i><sub>3</sub>), 2.81–2.61 (m, 2H, H-β-<i>m</i>Tyr), 2.36–2.28 (m, 2H, H-β-Cys), 1.26 (s, 9H, <sup><i>t</i></sup>Bu), 1.31–1.14 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.08–0.94 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.82 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.79 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.06 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.01-–0.04 (m, 6H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.10 (m, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>97</sub>H<sub>125</sub>N<sub>9</sub>NaO<sub>14</sub>SSi<sub>3</sub>, 1778.83; found, 1779.6; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −3.9 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Compound <b>30g</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30g</b> (92 mg, 0.06 mmol, 59%) was obtained as a pale yellow syrup using <b>29g</b> (44 mg, 0.10 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.39 (s, 1H, H-N3-uridine), 10.72 (s, 1H, NH-δ1-Trp), 9.25–9.17 (m, 1H, NH-5′-uridine), 8.28–8.17 (m, 1H, DABA-NH), 7.86–7.75 (m, 2H, H-Fmoc), 7.67–7.41 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.40–7.16 (m, 5H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp), 7.15–6.57 (m, 6H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.40 (d, 1H, NH-Met, <i>J</i><sub>NH-α-Met,H-α-Met</sub> = 7.6 Hz), 6.21 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 7.6 Hz), 6.16–6.05 (m, 2H, H-1′-uridine, H-5′-uridine), 5.61 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 7.2 Hz), 4.97–4.83 (m, 1H, H-β-DABA), 4.61–4.51 (m, 1H, H-α-DABA), 4.49–4.02 (m, 8H, H-Fmoc, H-α-Met, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.03–2.88 (m, 5H, H-β-Trp, NC<i>H</i><sub>3</sub>), 2.74–2.55 (m, 2H, H-β-<i>m</i>Tyr), 2.40–2.25 (m, 2H, H-γ-Met), 1.95 (s, 3H, H-SC<i>H</i><sub>3</sub>), 1.88–1.71 (m, 1H, H-β-Met), 1.70–1.57 (m, 1H, H-β-Met), 1.24 (s, 9H, <sup><i>t</i></sup>Bu), 1.28–1.10 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.06–0.90 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.81 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.76 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.07 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.04 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.10 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>80</sub>H<sub>115</sub>N<sub>9</sub>NaO<sub>14</sub>SSi<sub>3</sub>, 1564.75; found, 1564.5; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −5.2 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Compound <b>30h</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30h</b> (80 mg, 0.05 mmol, 51%) was obtained as a pale yellow syrup using <b>29h</b> (45 mg, 0.10 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.34 (s, 1H, H-N3-uridine), 10.75 (s, 1H, NH-δ1-Trp), 9.31–9.24 (m, 1H, NH-5′-uridine), 8.38–8.30 (m, 1H, DABA-NH), 7.90–7.80 (m, 2H, H-Fmoc), 7.69–7.51 (m, 1H, H-ε3-Trp), 7.49 (d, 1H, NH-<i>m</i>Tyr, <i>J</i><sub>NH-α-<i>m</i>Tyr,H-α-<i>m</i>Tyr</sub> = 8.1 Hz), 7.43–7.20 (m, 5H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp), 7.19–6.60 (m, 11H, H-δ-Phe, H-ε-Phe, H-ζ-Phe, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.41–6.29 (m, 2H, NH-α-Trp, NH-Phe), 6.22–6.12 (m, 2H, H-1′-uridine, H-5′-uridine), 5.44 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 7.2 Hz), 5.01–4.89 (m, 1H, H-β-DABA), 4.67–4.55 (m, 1H, H-α-DABA), 4.54–4.06 (m, 8H, H-Fmoc, H-α-Phe, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.04–2.86 (m, 7H, H-β-Phe, H-β-Trp, NC<i>H</i><sub>3</sub>), 2.80–2.62 (m, 2H, H-β-<i>m</i>Tyr), 1.26 (s, 9H, <sup><i>t</i></sup>Bu), 1.29–1.15 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.07–0.96 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.86 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.79 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.08 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.06 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.11 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>84</sub>H<sub>115</sub>N<sub>9</sub>NaO<sub>14</sub>Si<sub>3</sub>, 1580.78; found, 1580.6; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +10.6 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Compound <b>30i</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30i</b> (37 mg, 0.02 mmol, 22%) was obtained as a pale yellow syrup using <b>29i</b> (52 mg, 0.10 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.38 (s, 1H, H-N3-uridine), 10.75 (s, 1H, NH-δ1-Trp), 9.33–9.26 (m, 1H, NH-5′-uridine), 8.37–8.25 (m, 1H, DABA-NH), 7.90–7.81 (m, 2H, H-Fmoc), 7.68–7.45 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.45–6.60 (m, 15H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp, H-δ-Tyr, H-ε-Tyr), 6.47–6.29 (m, 2H, NH-α-Trp, NH-Tyr), 6.23–6.13 (m, 2H, H-1′-uridine, H-5′-uridine), 5.53–5.44 (m, 1H, H-5-uridine), 5.03–4.89 (m, 1H, H-β-DABA), 4.68–4.55 (m, 1H, H-α-DABA), 4.55–4.02 (m, 8H, H-Fmoc, H-α-Tyr, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.06–2.82 (m, 7H, H-β-Trp, H-β-Tyr, NC<i>H</i><sub>3</sub>), 2.77–2.60 (m, 2H, H-β-<i>m</i>Tyr), 1.26 (s, 9H, <sup><i>t</i></sup>Bu), 1.26 (s, 9H, <sup><i>t</i></sup>Bu), 1.33–1.13 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.11–0.95 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.86 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.79 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.08 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.05 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.11 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>88</sub>H<sub>123</sub>N<sub>9</sub>NaO<sub>15</sub>Si<sub>3</sub>, 1652.834; found, 1652.2; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +1.5 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Compound <b>30j</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30j</b> (62 mg, 0.03 mmol, 36%) was obtained as a pale yellow syrup using <b>29j</b> (59 mg, 0.10 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.24 (s, 1H, H-N3-uridine), 10.74 (s, 1H, NH-δ1-Trp), 9.35–9.25 (m, 1H, NH-5′-uridine), 8.44–8.34 (m, 1H, DABA-NH), 7.90–7.78 (m, 2H, H-Fmoc), 7.71–6.60 (m, 18H, NH-<i>m</i>Tyr, H-ε3-Trp, H-Fmoc, H-6-uridine, H-δ1-Trp, H-ζ2-Trp, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr), 6.43 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 8.1 Hz), 6.33 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 8.1 Hz), 6.22–6.12 (m, 2H, H-1′-uridine, H-5′-uridine), 5.36–5.29 (m, 1H, H-5-uridine), 5.02–4.88 (m, 1H, H-β-DABA), 4.67–4.06 (m, 9H, H-α-DABA, H-Fmoc, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.09–2.80 (m, 7H, H-β-Trp, NC<i>H</i><sub>3</sub>), 2.81–2.57 (m, 2H, H-β-<i>m</i>Tyr), 1.62 (s, 9H, Boc-<sup><i>t</i></sup>Bu), 1.21 (s, 9H, <sup><i>t</i></sup>Bu), 1.28–1.15 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.07–0.93 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.85 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.79 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.07 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.04 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.11 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>91</sub>H<sub>124</sub>N<sub>10</sub>NaO<sub>16</sub>Si<sub>3</sub>, 1719.84; found, 1719.6; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +4.7 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> Compound <b>30k</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30k</b> (23 mg, 0.012 mmol, 12%) was obtained as a pale yellow syrup using <b>29k</b> (68 mg, 0.10 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.60 (s, 1H, H-N3-uridine), 10.74 (s, 1H, NH-δ1-Trp), 9.22 (d, 1H, NH-5′-uridine, <i>J</i><sub>NH-5-uridine,H-5′-uridine</sub> = 9.4 Hz), 8.23 (4, 1H, DABA-NH, <i>J</i><sub>DABA-NH,H-α-DABA</sub> = 9.4 Hz), 7.92–7.72 (m, 2H, H-Fmoc), 7.71–6.57 (m, 30H, NH-<i>m</i>Tyr, H-ε3-Trp, H-Fmoc, H-6-uridine, H-δ1-Trp, H-ζ2-Trp, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ-His, H-ε-His, Trt), 6.51 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 7.6 Hz), 6.30 (d, 1H, NH-α-His, <i>J</i><sub>NH-α-His,H-α-His</sub> = 7.6 Hz), 6.20–6.08 (m, 2H, H-1′-uridine, H-5′-uridine), 5.54 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 7.6 Hz), 5.02–4.86 (m, 1H, H-β-DABA), 4.64–4.01 (m, 9H, H-α-DABA, H-Fmoc, H-α-His, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.07–2.56 (m, 9H, H-β-Trp, H-β-<i>m</i>Tyr, H-β-His, NC<i>H</i><sub>3</sub>), 1.22 (s, 9H, <sup><i>t</i></sup>Bu), 1.29–1.13 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.09–0.96 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.83 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.79 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.04 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.00 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.02 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.11 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>100</sub>H<sub>128</sub>N<sub>11</sub>O<sub>14</sub>Si<sub>3</sub>, 1490.89; found, 1792.1; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −0.5 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Compound <b>30l</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30l</b> (15 mg, 0.009 mmol, 18%) was obtained as a pale yellow syrup using <b>29l</b> (47 mg, 0.10 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.42 (s, 1H, H-N3-uridine), 10.75 (s, 1H, NH-δ1-Trp), 9.38 (d, 1H, NH-5′-uridine, <i>J</i><sub>NH-5-uridine,H-5′-uridine</sub> = 9.4 Hz), 8.12 (d, 1H, DABA-NH, <i>J</i><sub>DABA-NH,H-α-DABA</sub> = 9.0 Hz), 7.90–7.79 (m, 2H, H-Fmoc), 7.68–7.46 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.43–7.21 (m, 5H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp), 7.19–6.62 (m, 6H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.58–6.49 (m, 1H, NH-Asp), 6.45 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 8.1 Hz), 6.18 (d, 1H, H-1′-uridine, <i>J</i><sub>H-1′-uridine,H-2′-uridine</sub> = 7.2 Hz), 6.14 (d, 1H, H-5′-uridine, <i>J</i><sub>H-5′-uridine,NH-5-uridine</sub> = 9.4 Hz), 5.63 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 8.1 Hz), 4.99–4.85 (m, 1H, H-β-DABA), 4.66–3.93 (m, 9H, H-α-DABA, H-Fmoc, H-α-Asp, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.08–2.92 (m, 5H, H-β-Trp, NC<i>H</i><sub>3</sub>), 2.80–2.39 (m, 4H, H-β-<i>m</i>Tyr, H-β-Asp), 1.35 (s, 9H, <sup><i>t</i></sup>Bu), 1.26 (s, 9H, <sup><i>t</i></sup>Bu), 1.30–1.13 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.08–0.93 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.85 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.80 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.06 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.03 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.00 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.09 (s, 3H, H-<sup><i>t</i></sup>BuCSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>83</sub>H<sub>119</sub>N<sub>9</sub>NaO<sub>16</sub>Si<sub>3</sub>, 1604.80; found, 1605.9; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −0.3 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Compound <b>30m</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30m</b> (26 mg, 0.016 mmol, 16%) was obtained as a pale yellow syrup using <b>29m</b> (50 mg, 0.10 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.42 (s, 1H, H-N3-uridine), 10.77 (s, 1H, NH-δ1-Trp), 9.36 (d, 1H, NH-5′-uridine, <i>J</i><sub>NH-5-uridine,H-5′-uridine</sub> = 10.3 Hz), 8.27 (d, 1H, DABA-NH, <i>J</i><sub>DABA-NH,H-α-DABA</sub> = 8.5 Hz), 7.92–7.76 (m, 2H, H-Fmoc), 7.71–7.45 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.45–7.20 (m, 5H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp), 7.20–6.57 (m, 6H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.58–6.49 (d, 1H, NH-Glu, <i>J</i><sub>NH-α-Glu,H-α-Glu</sub> = 8.5 Hz), 6.28 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 8.1 Hz), 6.21–6.12 (m, 2H, H-1′-uridine, H-5′-uridine), 5.64 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 7.6 Hz), 5.00–4.87 (m, 1H, H-β-DABA), 4.67–3.93 (m, 9H, H-α-DABA, H-Fmoc, H-α-Glu, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.08–2.91 (m, 5H, H-β-Trp, NC<i>H</i><sub>3</sub>), 2.80–2.58 (m, 2H, H-β-<i>m</i>Tyr), 2.24–2.08 (m, 2H, H-γ-Glu), 1.92–1.75 (m, 2H, H-β-Glu), 1.71–1.58 (m, 2H, H-β-Glu), 1.38 (s, 9H, <sup><i>t</i></sup>Bu), 1.28 (s, 9H, <sup><i>t</i></sup>Bu), 1.31–1.13 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.09–0.92 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.85 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.80 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.06 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.03 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.09 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>83</sub>H<sub>119</sub>N<sub>9</sub>NaO<sub>16</sub>Si<sub>3</sub>, 1618.81; found, 1619.5; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −2.3 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Compound <b>30n</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30n</b> (25 mg, 0.015 mmol, 15%) was obtained as a pale yellow syrup using <b>29n</b> (53 mg, 0.10 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.43 (s, 1H, H-N3-uridine), 10.76 (s, 1H, NH-δ1-Trp), 9.31–9.20 (m, 1H, NH-5′-uridine), 8.24–8.17 (m, 1H, DABA-NH), 7.90–7.80 (m, 2H, H-Fmoc), 7.70–7.48 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.43–7.19 (m, 6H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp, NH-ε-Lys), 7.19–6.63 (m, 6H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.36 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 8.5 Hz), 6.23 (d, 1H, NH-α-Lys, <i>J</i><sub>NH-α-Lys,H-α-Lys</sub> = 7.6 Hz), 6.20–6.12 (m, 2H, H-1′-uridine, H-5′-uridine), 5.64 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 8.5 Hz), 4.97–4.87 (m, 1H, H-β-DABA), 4.65–4.56 (m, 1H, H-α-DABA), 4.54–4.03 (m, 8H, H-Fmoc, H-α-Lys, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.04–2.92 (m, 5H, H-β-Trp, NC<i>H</i><sub>3</sub>), 2.90–2.80 (m, 2H, H-ε-Lys), 2.77–2.61 (m, 2H, H-β-<i>m</i>Tyr), 1.62–1.4 (m, 1H, H-β-Lys), 1.36 (s, 9H, Boc), 1.27 (s, 9H, <sup><i>t</i></sup>Bu), 1.43–1.13 (m, 8H, H-β-Lys, H-γ-Lys, H-δ-Lys, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.08–0.92 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.85 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.80 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.07 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.03 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.09 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>86</sub>H<sub>126</sub>N<sub>10</sub>NaO<sub>16</sub>Si<sub>3</sub>, 1661.86; found, 1661.1; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −0.2 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> Compound <b>30o</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30o</b> (55 mg, 0.03 mmol, 60%) was obtained as a pale yellow syrup using <b>29o</b> (36 mg, 0.05 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.45 (s, 1H, H-N3-uridine), 10.75 (s, 1H, NH-δ1-Trp), 9.41–9.32 (m, 1H, NH-5′-uridine), 8.26–8.16 (m, 1H, DABA-NH), 7.91–7.78 (m, 2H, H-Fmoc), 7.69–7.45 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.45–7.20 (m, 5H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp), 7.19–6.59 (m, 6H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.56–6.35 (m, NH-α-Arg), 6.27 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 7.6 Hz), 6.18–6.09 (m, 2H, H-1′-uridine, H-5′-uridine), 5.66–5.59 (m, 1H, H-5-uridine), 4.97–4.86 (m, 1H, H-β-DABA), 4.68–4.57 (m, 1H, H-α-DABA), 4.57–4.05 (m, 8H, H-Fmoc, H-α-Arg, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.08–2.86 (m, 9H, H-δ-Arg, H-β-Trp, NC<i>H</i><sub>3</sub>, Pbf-C<i>H</i><sub>2</sub>), 2.78–2.61 (m, 2H, H-β-<i>m</i>Tyr), 2.43 (m, 3H, Pbf-C<i>H</i><sub>3</sub>), 2.01 (m, 3H, Pbf-C<i>H</i><sub>3</sub>), 1.68–1.49 (m, 1H, H-β-Arg), 1.41 (s, 3H, Pbf-C<i>H</i><sub>3</sub>), 1.28 (s, 9H, <sup><i>t</i></sup>Bu), 1.48–1.14 (m, 6H, β-Arg, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.07–0.95 (m, 24H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA, Pbf-C<i>H</i><sub>3</sub>), 0.83 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.79 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.05 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.10 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>94</sub>H<sub>134</sub>N<sub>12</sub>NaO<sub>17</sub>Si<sub>3</sub>, 1841.89; found, 1842.7; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −2.4 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Compound <b>30p</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30p</b> (84 mg, 0.054 mmol, 54%) was obtained as a pale yellow syrup using <b>29p</b> (45 mg, 0.10 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.43 (s, 1H, H-N3-uridine), 10.75 (s, 1H, NH-δ1-Trp), 9.58–9.47 (m, 1H, NH-5′-uridine), 8.06–7.97 (m, 1H, DABA-NH), 7.90–7.79 (m, 2H, H-Fmoc), 7.69–7.57 (m, 1H, H-ε3-Trp), 7.51 (m, 2H, NH-<i>m</i>Tyr, <i>J</i><sub>NH-<i>m</i>Trp,H-α-<i>m</i>Tyr</sub> = 8.1 Hz), 7.43–7.18 (m, 5H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp), 7.17–6.63 (m, 6H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.58 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 7.6 Hz), 6.35 (d, 1H, NH-Ser, 7.2 Hz), 6.20–6.08 (m, 2H, H-1′-uridine, H-5′-uridine), 5.61 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 8.1 Hz), 4.97–4.86 (m, 1H, H-β-DABA), 4.80–4.69 (m, 1H, H-α-DABA), 4.60–4.05 (m, 8H, H-Fmoc, H-α-Ser, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.56–3.46 (m, 1H, H-β-Ser), 3.39–3.31 (m, 1H, H-β-Ser), 3.03–2.93 (m, 5H, H-β-Trp, NC<i>H</i><sub>3</sub>), 2.80–2.57 (m, 2H, H-β-<i>m</i>Tyr), 1.26 (s, 9H, <sup><i>t</i></sup>Bu), 1.29–1.13 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.08–0.96 (m, 30H, <sup><i>t</i></sup>Bu, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.84 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.79 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.06 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.02 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.11 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>82</sub>H<sub>119</sub>N<sub>9</sub>NaO<sub>15</sub>Si<sub>3</sub>, 1576.80; found, 1576.7; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +7.2 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_50" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Compound <b>30q</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>34q</b> (66 mg, 0.042 mmol, 42%) was obtained as a pale yellow syrup using <b>29q</b> (46 mg, 0.10 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.43 (s, 1H, H-N3-uridine), 10.75 (s, 1H, NH-δ1-Trp), 9.36–9.25 (m, 1H, NH-5′-uridine), 7.99–7.91 (m, 1H, DABA-NH), 7.91–7.75 (m, 2H, H-Fmoc), 7.71–7.45 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.44–7.20 (m, 5H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp), 7.19–6.60 (m, 6H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.30–6.05 (d, 4H, NH-α-Trp, NH-Thr, H-1′-uridine, H-5′-uridine), 5.61 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 8.1 Hz), 4.98–4.82 (m, 1H, H-β-DABA), 4.74–4.62 (m, 1H, H-α-DABA), 4.60–4.04 (m, 8H, H-Fmoc, H-α-Thr, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.96–3.84 (m, 1H, H-β-Thr), 3.08–2.84 (m, 5H, H-β-Trp, NC<i>H</i><sub>3</sub>), 2.82–2.59 (m, 2H, H-β-<i>m</i>Tyr), 1.27 (s, 9H, <sup><i>t</i></sup>Bu), 1.31–1.14 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.12–0.97 (m, 30H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA, <sup><i>t</i></sup>Bu), 0.94 (d, 3H, H-γ-Thr, <i>J</i><sub>H-γ-Thr,H-β-Thr</sub> = 6.3 Hz), 0.85 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.80 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.08 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.04 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.00 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.09 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>83</sub>H<sub>122</sub>N<sub>9</sub>O<sub>15</sub>Si<sub>3</sub>, 1568.83; found, 1568.9; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −8.0 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_51" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> Compound <b>30r</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30r</b> (63 mg, 0.036 mmol, 71%) was obtained as a pale yellow syrup using <b>29r</b> (33 mg, 0.05 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.36 (s, 1H, H-N3-uridine), 10.74 (s, 1H, NH-δ1-Trp), 9.60–9.39 (m, 1H, NH-5′-uridine), 8.51 (s, 1H, NH-γ-Asn), 8.04–7.92 (m, 1H, DABA-NH), 7.92–7.73 (m, 2H, H-Fmoc), 7.71–7.46 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.43–6.44 (m, 27H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp, NH-α-Trp, NH-α-Asn, Trt, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 7.19–6.60 (m, 6H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.42 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 8.1 Hz), 6.25–6.03 (m, 2H, H-1′-uridine, H-5′-uridine), 5.59–5.48 (m, 1H, H-5-uridine), 4.98–4.75 (m, 1H, H-β-DABA), 4.71–4.57 (m, 1H, H-α-DABA), 4.57–3.90 (m, 7H, H-Fmoc, H-α-Asn, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.09–2.81 (m, 5H, H-β-Trp, NC<i>H</i><sub>3</sub>), 2.80–2.28 (m, 4H, H-β-<i>m</i>Tyr, H-β-Asn), 1.26 (s, 9H, <sup><i>t</i></sup>Bu), 1.30–1.13 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.08–0.92 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.89–0.73 (m, 18H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.08 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.12 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>98</sub>H<sub>126</sub>N<sub>10</sub>NaO<sub>15</sub>Si<sub>3</sub>, 1789.86; found, 1790.7; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +2.7 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_52" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> Compound <b>30s</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30s</b> (26 mg, 0.014 mmol, 29%) was obtained as a pale yellow syrup using <b>29s</b> (66 mg, 0.10 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.61 (s, 1H, H-N3-uridine), 10.75 (s, 1H, NH-δ1-Trp), 9.23 (d, 1H, NH-5′-uridine, <i>J</i><sub>NH-5-uridine,H-5′-uridine</sub> = 10.3 Hz), 8.24 (d, 1H, DABA-NH, <i>J</i><sub>DABA-NH,H-α-DABA</sub> = 9.0 Hz), 7.92–7.77 (m, 2H, H-Fmoc), 7.70–7.43 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.43–7.20 (m, 5H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp), 7.19–6.59 (m, 22H, NH-ε-Gln, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp, Trt), 6.51 (d, 1H, NH-a-Gln, <i>J</i><sub>NH-α-Gln,H-α-Gln</sub> = 7.6 Hz), 6.31 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 7.6 Hz), 6.20–6.08 (m, 2H, H-1′-uridine, H-5′-uridine), 5.55 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 7.6 Hz), 5.03–4.87 (m, 1H, H-β-DABA), 4.68–4.58 (m, 1H, H-α-DABA), 4.65–3.93 (m, 9H, H-α-DABA, H-α-Gln, H-Fmoc, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.07–2.56 (m, 11H, H-β-Trp, NC<i>H</i><sub>3</sub>, H-β-<i>m</i>Tyr, H-β-Gln, H-γ-Gln), 1.22 (s, 9H, <sup><i>t</i></sup>Bu), 1.29–1.13 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.11–0.96 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.83 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.80 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.05 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.01 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.11 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>99</sub>H<sub>128</sub>N<sub>10</sub>NaO<sub>15</sub>Si<sub>3</sub>, 1803.88; found, 1804.7; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −1.7 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_53" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> Compound <b>30t</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>30a</b>, <b>30t</b> (40 mg, 0.026 mmol, 53%) was obtained as a pale yellow syrup using <b>29t</b> (20 mg, 0.05 mmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C): δ 11.44 (s, 1H, H-N3-uridine), 10.73 (s, 1H, NH-δ1-Trp), 9.06–8.93 (m, 1H, NH-5′-uridine), 8.23–8.14 (m, 1H, DABA-NH), 7.90–7.81 (m, 2H, H-Fmoc), 7.67–7.47 (m, 2H, NH-<i>m</i>Tyr, H-ε3-Trp), 7.43–7.21 (m, 5H, H-Fmoc, H-6-uridine, H-ζ1-Trp, H-η-Trp), 7.17–6.62 (m, 6H, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.22–6.11 (m, 2H, H-1′-uridine, H-5′-uridine), 5.97–5.88 (m, 1H, NH-α-Trp), 5.67 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 7.2 Hz), 4.98–4.86 (m, 1H, H-β-DABA), 4.58–4.05 (m, 9H, H-α-DABA, H-Fmoc, H-α-Pro, H-α-Trp, H-α-<i>m</i>Tyr, H-2′-uridine, H-3′-uridine), 3.14–2.57 (m, 9H, H-β-Trp, H-δ-Pro, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 1.98–1.70 (m, 4H, H-β-Pro, H-γ-Pro), 1.29 (s, 9H, <sup><i>t</i></sup>Bu), 1.31–1.15 (m, 3H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>), 1.07–0.98 (m, 21H, Si[C<i>H</i>(CH<sub>3</sub>)<sub>2</sub>]<sub>3</sub>, H-γ-DABA), 0.86 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.80 (s, 9H, SiC(<u class="uu">C</u><i><u class="uu">H</u></i><sub>3</sub>)<sub>3</sub>), 0.07 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.04 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), 0.00 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>), −0.09 (s, 3H, <sup><i>t</i></sup>BuSi(C<i>H</i><sub>3</sub>)<sub>2</sub>); ESIMS-LR <i>m</i>/<i>z</i>: calcd for C<sub>80</sub>H<sub>113</sub>N<sub>9</sub>NaO<sub>14</sub>Si<sub>3</sub>, 1530.76; found, 1530.5; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −22.7 (<i>c</i> 1.00, MeOH).</div></div><div id="sec4_54" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> Compound <b>25a</b></h3><div class="NLM_p last">A solution of <b>30a</b> (10 mg, 6.8 μmol) in MeCN (136 μL) was treated with 3HF·Et<sub>3</sub>N (44 μL, 270 mmol) at room temperature for 3 d. Trimethylsilanol (70 μL, 630 μmol) was added to the reaction mixture, which was further stirred for 1 h. The mixture was concentrated <i>in vacuo</i>, and the residue was treated with 20% Et<sub>2</sub>NH/MeCN (250 μL) at room temperature for 1 h. The mixture was concentrated <i>in vacuo</i>, and the residue was triturated with Et<sub>2</sub>O. The solvent was removed by decantation, and the resulting precipitate was treated with 80% aq TFA (250 μL) at room temperature for 3 h. After toluene was added, the mixture was concentrated <i>in vacuo</i>. The residue was triturate with Et<sub>2</sub>O and the solvent was removed by decantation. The precipitate was purified by ODS column chromatography (MeCN/H<sub>2</sub>O = 10–30%, containing 0.1% TFA) to afford <b>25a</b> (1.5 mg, 1.6 μmol, 24% over three steps) as a yellow powder. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.37 (s, 1H, H-N3-uridine), 10.77 (s, 1H, NH-δ1-Tr), 9.58–9.44 (m, 1H, NH-5′-uridine), 9.45–9.26 (m, 2H, NH-<i>m</i>Tyr), 8.63–8.54 (m, 1H, DABA-NH), 7.62 (d, 1H, H-ε3-Trp, <i>J</i><sub>H-ε3-Trp,H-ζ2-Trp</sub> = 7.8 Hz), 7.42–6.71 (m, 9H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.47–6.32 (m, 2H, NH-Gly, NH-α-Trp), 6.17–6.01 (m, 1H, H-1′-uridine), 5.74–5.64 (m, 1H, H-5′-uridine), 5.47 (d, 1H, H-5-uridine, <i>J</i><sub>H-5-uridine,H-6-uridine</sub> = 7.8 Hz), 4.97–4.84 (m, 1H, H-β-DABA), 4.76–4.56 (m, 1H, H-α-DABA), 4.46–4.27 (m, 2H, H-α-Tyr, H-α-Trp), 4.41–3.91 (m, 2H, H-2′-uridine, H-3′-uridine), 3.74–3.63 (m, 2H, H-α-Gly), 3.17–2.79 (m, 4H, H-β-Trp, H-β-<i>m</i>Tyr), 2.90 (s, 3H, NC<i>H</i><sub>3</sub>), 1.22–1.15 (m, 3H, H-γ-DABA); ESIMS-LR <i>m</i>/<i>z</i>: 806.3 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>37</sub>H<sub>44</sub>N<sub>9</sub>O<sub>12</sub>, 806.3104; found, 806.3048; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +3.4 (<i>c</i> 0.12, MeOH).</div></div><div id="sec4_55" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> Compound <b>25b</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25b</b> (0.5 mg, 0.5 μmol, 7.4% over three steps) was obtained as a pale yellow powder using <b>30b</b> (10 mg, 6.7 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.34 (s, 1H, H-N3-uridine), 10.77 (s, 1H, NH-δ1-Trp), 9.43–9.30 (m, 2H, NH-5′-uridine, NH-<i>m</i>Tyr), 8.29–8.18 (m, 1H, DABA-NH), 7.55–7.47 (m, 1H, H-ε3-Trp), 7.37–6.56 (m, 9H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.47–6.36 (m, 1H, NH-Ala), 6.26–6.16 (m, 1H, NH-α-Trp), 6.15–6.04 (m, 1H, H-1′-uridine), 5.59–5.46 (m, 1H, H-5′-uridine), 5.39–5.32 (m, 1H, H-5-uridine), 4.98–4.88 (m, 1H, H-β-DABA), 4.81–4.67 (m, 1H, H-α-DABA), 4.66–4.28 (m, 2H, H-α-<i>m</i>Tyr, H-α-Trp), 4.22–3.87 (m, 3H, H-α-Ala, H-2′-uridine, H-3′-uridine), 3.13–2.78 (m, 7H, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 1.33–1.00 (m, 6H, H-γ-DABA, H-β-Ala); ESIMS-LR <i>m</i>/<i>z</i>: 820.4 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>38</sub>H<sub>46</sub>N<sub>9</sub>O<sub>12</sub>, 820.3260; found, 820.3256; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −1.7 (<i>c</i> 0.02, MeOH).</div></div><div id="sec4_56" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> Compound <b>25c</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25c</b> (1.8 mg, 1.8 μmol, 27% over three steps) was obtained as a pale yellow powder using <b>30c</b> (10 mg, 6.6 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.38 (s, 1H, H-N3-uridine), 10.77 (s, 1H, NH-δ1-Trp), 9.44–9.14 (m, 3H, NH-5′-uridine, NH-<i>m</i>Tyr), 8.08–7.85 (m, 1H, DABA-NH), 7.70–7.59 (m, 1H, H-ε3-Trp), 7.56–6.53 (m, 9H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.43–6.31 (m, 1H, NH-Leu), 6.21–6.02 (m, 2H, NH-α-Trp, H-1′-uridine), 5.72–5.66 (m, 1H, H-5′-uridine), 5.53–5.48 (m, 1H, H-5-uridine), 4.98–4.88 (m, 1H, H-β-DABA), 4.80–4.73 (m, 1H, H-α-DABA), 4.70–4.24 (m, 3H, H-α-<i>m</i>Tyr, H-α-Trp, H-α-Leu), 4.21–3.96 (m, 2H, H-2′-uridine, H-3′-uridine), 3.12–2.82 (m, 7H, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 1.70–1.14 (m, 3H, H-β-Leu, H-γ-Leu), 1.14–1.00 (m, 3H, H-γ-DABA), 0.95–0.71 (m, 6H, H-δ-Leu); ESIMS-LR <i>m</i>/<i>z</i>: 862.5 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>41</sub>H<sub>52</sub>N<sub>9</sub>O<sub>12</sub>, 862.3730; found, 862.3748; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −1.7 (<i>c</i> 0.15, MeOH).</div></div><div id="sec4_57" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> Compound <b>25d</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25d</b> (1.0 mg, 1.0 μmol, 15% over three steps) was obtained as a pale yellow powder using <b>30d</b> (10 mg, 6.6 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.40 (s, 1H, H-N3-uridine), 10.77 (s, 1H, NH-δ1-Trp), 9.47–9.27 (m, 2H, NH-<i>m</i>Tyr), 9.14–9.05 (m, 1H, NH-5′-uridine), 8.32–8.21 (m, 1H, DABA-NH), 8.13–7.85 (m, 1H, H-ε3-Trp), 7.55–6.52 (m, 9H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.41–6.30 (m, 1H, NH-Ile) 6.29–6.17 (m, 1H, NH-Trp), 6.15–6.03 (m, 1H, H-1′-uridine), 5.73–5.67 (m, 1H, H-5′-uridine), 5.56–5.49 (m, 1H, H-5-uridine), 5.00–4.87 (m, 1H, H-β-DABA), 4.81–4.69 (m, 1H, H-α-DABA), 4.68–4.29 (m, 3H, H-α-<i>m</i>Tyr, H-α-Trp, H-α-Ile), 4.12–3.95 (m, 2H, H-2′-uridine, H-3′-uridine), 3.15–2.80 (m, 7H, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 1.63–1.53 (m, 1H, H-β-Ile), 1.50–1.35 (m, 2H, H-γ-Ile), 1.14–0.98 (m, 3H, H-γ-DABA), 0.92–0.67 (m, 6H, H-γ-Ile, H-δ-Ile); ESIMS-LR <i>m</i>/<i>z</i>: 862.6 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>41</sub>H<sub>52</sub>N<sub>9</sub>O<sub>12</sub>, 862.3730; found, 862.3748; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +1.9 (<i>c</i> 0.07, MeOH).</div></div><div id="sec4_58" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> Compound <b>25e</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25e</b> (2.2 mg, 2.3 μmol, 18% over three steps) was obtained as a pale yellow powder using <b>30e</b> (20 mg, 13 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.40 (s, 1H, H-N3-uridine), 10.77 (s, 1H, NH-δ1-Trp), 9.53–9.22 (m, 2H, NH-<i>m</i>Tyr), 9.10 (m, 1H, NH-5′-uridine, <i>J</i><sub>NH-5′-uridine,H-5′-uridine</sub> = 10.3 Hz), 8.23–8.20 (m, 1H, DABA-NH), 7.63 (d, 1H, H-ε3-Trp, <i>J</i><sub>H-ε3-Trp,H-ζ2-Trp</sub> = 7.8 Hz), 7.56–6.52 (m, 9H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.41–6.22 (m, 2H, NH-Trp, NH-Val), 6.16–6.04 (d, 1H, H-1-uridine, <i>J</i><sub>1,2</sub> = 9.6 Hz), 5.74–5.67 (m, 1H, H-5′-uridine, <i>J</i><sub>5′,4′</sub> = 7.6 Hz), 5.53–5.49 (m, 1H, H-5-uridine), 5.00–4.89 (m, 1H, H-β-DABA), 4.80–4.71 (m, 1H, H-α-DABA), 4.69–4.27 (m, 3H, H-α-<i>m</i>Tyr, H-α-Val, H-α-Trp), 4.10–3.95 (m, 2H, H-2′-uridine, H-3′-uridine), 3.10–2.83 (m, 7H, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 1.97–1.76 (m, 1H, H-β-Val), 1.10 (d, 3H, H-γ-DABA, <i>J</i><sub>H-γ-DABA,H-β-DABA</sub> = 6.9 Hz), 1.00–0.66 (m, 6H, Val-CH<sub>3</sub>); ESIMS-LR <i>m</i>/<i>z</i>: 848.5 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>40</sub>H<sub>50</sub>N<sub>9</sub>O<sub>12</sub>, 848.3573; found, 848.3611; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −4.1 (<i>c</i> 0.19, MeOH).</div></div><div id="sec4_59" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Compound <b>25f</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25f</b> (0.5 mg, 0.5 μmol, 8.7% over three steps) was obtained as a pale yellow powder using <b>30f</b> (10 mg, 5.7 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.37 (s, 1H, H-N3-uridine), 10.77 (s, 1H, NH-δ1-Trp), 9.49–9.22 (m, 3H, NH-5′-uridine, NH-<i>m</i>Tyr), 8.36–8.31 (m, 1H, NH-DABA), 7.51 (d, 1H, H-ε3-Trp, <i>J</i><sub>H-ε3-Trp,H-ζ2-Trp</sub> = 8.1 Hz), 7.35–6.52 (m, 10H, H-2, 4, 5, 6, 7-Trp, H-2, 4, 5, 6-<i>m</i>Tyr, NH-Cys), 6.44–6.36 (m, 1H, NH-α-Trp), 6.16–6.05 (m, 1H, H-1′-uridine), 5.73–5.66 (d, 1H, H-5′-uridine), 5.52–5.47 (m, 1H, H-5-uridine), 5.00–4.91 (m, 1H, H-β-DABA), 4.84–4.72 (m, 1H, H-α-DABA), 4.70–4.26 (m, 3H, H-α-<i>m</i>Tyr, H-α-Trp, H-α-Cys), 4.06–3.97 (m, 2H, H-2′-uridine, H-3′-uridine), 3.11–2.85 (m, 7H, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 2.72–2.63 (m, 2H, H-β-Cys), 1.12–1.06 (m, 3H, H-γ-DABA); ESIMS-LR <i>m</i>/<i>z</i>: 852.4 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>38</sub>H<sub>46</sub>N<sub>9</sub>O<sub>12</sub>S, 852.2981; found, 852.3046; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +8.7 (<i>c</i> 0.02, MeOH).</div></div><div id="sec4_60" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> Compound <b>25g</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25g</b> (2.3 mg, 2.3 μmol, 18% over three steps) was obtained as a pale yellow powder using <b>30g</b> (20 mg, 13 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.37 (s, 1H, H-N3-uridine), 10.78 (m, 1H, NH-δ1-Trp), 9.50–9.24 (m, 3H, NH-5′-uridine, NH-<i>m</i>Tyr), 8.36–8.23 (m, 1H, DABA-NH), 7.51 (d, 1H, H-ε3-Trp, <i>J</i><sub>H-ε3-Trp,H-ζ2-Trp</sub> = 8.1 Hz), 7.36–6.56 (m, 9H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.51–6.42 (m, 1H, NH-Met), 6.25–6.16 (m, 1H, NH-α-Trp), 6.15–6.03 (m, 1H, H-1′-uridine), 5.73–5.68 (m, 1H, H-5′-uridine), 5.52–5.47 (m, 1H, H-5-uridine), 4.99–4.88 (m, 1H, H-β-DABA), 4.82–4.72 (m, 1H, H-α-DABA), 4.70–4.27 (m, 3H, H-α-<i>m</i>Tyr, H-α-Trp, H-α-Met), 4.25–4.18 (m, 2H, H-2′-uridine, H-3′-uridine), 3.15–2.82 (m, 9H, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>, H-γ-Met), 2.00 (s, 3H, SCH<sub>3</sub>), 1.90–1.61 (m, 2H, H-β-Met), 1.14–1.02 (m, 3H, H-γ-DABA); ESIMS-LR <i>m</i>/<i>z</i>: 880.4 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>40</sub>H<sub>50</sub>N<sub>9</sub>O<sub>12</sub>S, 880.3294; found, 880.3288; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −1.1 (<i>c</i> 0.21, MeOH).</div></div><div id="sec4_61" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> Compound <b>25h</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25h</b> (3.4 mg, 3.4 μmol, 26% over three steps) was obtained as a pale yellow powder using <b>30h</b> (20 mg, 13 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.32 (s, 1H, H-N3-uridine), 10.76 (s, 1H, NH-δ1-Trp), 9.60 (d, 1H, NH-5′-uridine, <i>J</i><sub>NH-5′-uridine,H-5′-uridine</sub> = 9.0 Hz), 9.49–9.24 (m, 2H, NH-<i>m</i>Tyr), 8.36 (d, 1H, DABA-NH, <i>J</i><sub>DABA-NH,H-α-DABA</sub> = 9.0 Hz), 7.50 (d, 1H, H-ε3-Trp, <i>J</i><sub>H-ε3-Trp,H-ζ2-Trp</sub> = 7.6 Hz), 7.35–6.54 (m, 14H, H-6-uridine, H-δ-Phe, H-ε-Phe, H-ζ-Phe, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.43–6.29 (m, 2H, NH-Phe, NH-α-Trp), 6.18–6.08 (m, 1H, H-1′-uridine), 5.71–5.66 (m, 1H, H-5′-uridine), 5.42–5.37 (m, 1H, H-5-uridine), 4.99–4.89 (m, 1H, H-β-DABA), 4.83–4.83 (m, 1H, H-α-DABA), 4.72–4.30 (m, 3H, H-α-<i>m</i>Tyr, H-α-Trp, H-α-Phe), 4.08–3.90 (m, 2H, H-2′-uridine, H-3′-uridine), 3.07–2.80 (m, 9H, H-β-Trp, H-β-Phe, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 1.10 (d, 3H, H-γ-DABA, <i>J</i><sub>H-γ-DABA,H-β-DABA</sub> = 6.7 Hz); ESIMS-LR <i>m</i>/<i>z</i>: 896.4 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>44</sub>H<sub>50</sub>N<sub>9</sub>O<sub>12</sub>, 896.3573; found, 896.3593; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +0.52 (<i>c</i> 0.31, MeOH).</div></div><div id="sec4_62" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> Compound <b>25i</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25i</b> (2.6 mg, 2.5 μmol, 21% over three steps) was obtained as a pale yellow powder using <b>30i</b> (20 mg, 12 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.34 (s, 1H, H-N3-uridine), 10.75 (s, 1H, NH-δ1-Trp), 9.62 (d, 1H, NH-5′-uridine, <i>J</i><sub>NH-5′-uridine,H-5′-uridine</sub> = 10.3 Hz), 9.45–9.29 (m, 2H, NH-<i>m</i>Tyr), 8.29 (d, 1H, NH-DABA, <i>J</i><sub>DABA-NH,H-α-DABA</sub> = 10.3 Hz), 7.50 (d, 1H, H-ε3-Trp, <i>J</i><sub>H-ε3-Trp,H-ζ2-Trp</sub> = 7.6 Hz), 7.34–7.54 (m, 13H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp, H-δ-Tyr, H-ε-Tyr), 6.40–6.26 (m, 2H, NH-α-Trp, NH-Tyr), 6.17–6.07 (m, 1H, H-1′-uridine), 5.71–5.67 (m, 1H, H-5′-uridine), 5.46–5.41 (m, 1H, H-5-uridine), 4.98–4.88 (m, 1H, H-β-DABA), 4.82–4.75 (m, 1H, H-α-DABA), 4.70–4.22 (m, 3H, H-α-<i>m</i>Tyr, H-α-Tyr, H-α-Trp), 4.01–3.90 (m, 2H, H-2′-uridine, H-3′-uridine), 3.10–2.54 (m, 7H, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>, H-β-Tyr), 1.09 (d, 3H, H-γ-DABA, <i>J</i><sub>H-γ-DABA,H-β-DABA</sub> = 7.2 Hz); ESIMS-LR <i>m</i>/<i>z</i>: 912.5 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>44</sub>H<sub>50</sub>N<sub>9</sub>O<sub>13</sub>, 912.3523; found, 912.3546; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +3.6 (<i>c</i> 0.23, MeOH).</div></div><div id="sec4_63" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> Compound <b>25j</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25j</b> (3.2 mg, 3.1 μmol, 26% over three steps) was obtained as a pale yellow powder using <b>30j</b> (20 mg, 12 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.32 (s, 1H, H-N3-uridine), 10.75 (m, 1H, NH-δ1-Trp), 9.68 (d, 1H, NH-5′-uridine, <i>J</i><sub>NH-5′-uridine,H-5′-uridine</sub> = 9.9 Hz), 9.54–9.18 (m, 2H, NH-<i>m</i>Tyr), 8.38–8.25 (m, 1H, DABA-NH), 7.61 (d, 1H, H-ε3-Trp, <i>J</i><sub>H-ε3-Trp,H-ζ2-Trp</sub> = 8.1 Hz), 7.55–6.54 (m, 14H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.48–6.27 (m, 2H, NH-α-Trp), 6.20–6.07 (m, 2H, NH-α-Trp), 5.96–5.90 (m, 1H, H-1′-uridine), 5.72–5.66 (m, 1H, H-5′-uridine), 5.36–5.31 (m, 1H, H-5-uridine), 4.99–4.89 (m, 1H, H-β-DABA), 4.83–4.75 (m, 1H, H-α-DABA), 4.74–4.30 (m, 3H, H-α-<i>m</i>Tyr, H-α-Trp), 4.01–3.92 (m, 2H, H-2′-uridine, H-3′-uridine), 3.14–2.80 (m, 9H, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 1.07 (d, 3H, H-γ-DABA, <i>J</i><sub>H-γ-DABA,H-β-DABA</sub> = 7.2 Hz); ESIMS-LR <i>m</i>/<i>z</i>: 935.5 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>46</sub>H<sub>51</sub>N<sub>10</sub>O<sub>12</sub>, 935.3882; found, 935.3696; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +9.6 (<i>c</i> 0.28, MeOH).</div></div><div id="sec4_64" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> Compound <b>25k</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25k</b> (2.4 mg, 2.4 μmol, 43% over three steps) was obtained as a pale yellow powder using <b>30k</b> (10 mg, 5.6 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.37 (s, 1H, H-N3-uridine), 10.78 (s, 1H, NH-δ1-Trp), 9.57–9.25 (m, 1H, NH-5′-uridine), 8.96–8.78 (m, 2H, NH-<i>m</i>Tyr), 8.44–7.77 (m, 2H, DABA-NH, H-ε-His), 7.77–6.53 (m, 13H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp, H-ε3-Trp, H-δ-His, NH-α-His, NH-ε-His), 6.31 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 7.6 Hz), 6.15–6.03 (m, 1H, H-1′-uridine), 5.63–5.58 (m, 1H, H-5′-uridine), 5.49–5.40 (m, 1H, H-5-uridine), 4.98–4.82 (m, 1H, H-β-DABA), 4.80–4.10 (m, 4H, H-α-DABA, H-α-<i>m</i>Tyr, H-α-Trp, H-α-His), 4.10–3.82 (m, 2H, H-2′-uridine, H-3′-uridine), 3.13–2.57 (m, 9H, H-β-Trp, H-β-<i>m</i>Tyr, H-β-His, NC<i>H</i><sub>3</sub>), 1.21–1.13 (m, 3H, H-γ-DABA); ESIMS-LR <i>m</i>/<i>z</i>: 886.3 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>41</sub>H<sub>48</sub>N<sub>11</sub>O<sub>12</sub>, 886.3478; found, 886.3484; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −0.9 (<i>c</i> 0.21, MeOH).</div></div><div id="sec4_65" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> Compound <b>25l</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25l</b> (1.5 mg, 1.5 μmol, 24% over three steps) was obtained as a pale yellow powder using <b>30l</b> (10 mg, 6.3 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.36 (s, 1H, H-N3-uridine), 10.75 (s, 1H, NH-δ1-Trp), 9.58–9.24 (m, 3H, NH-5′-uridine, NH-<i>m</i>Tyr), 8.16–7.83 (m, 1H, DABA-NH), 7.54–7.47 (m, 1H, H-ε3-Trp), 7.36–6.53 (m, 10H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp, NH-Asp), 6.44–6.37 (m, 1H, NH-α-Trp), 6.14–6.03 (m, 1H, H-1′-uridine), 5.73–5.67 (m, 1H, H-5′-uridine), 5.51–5.45 (m, 1H, H-5-uridine), 4.96–4.27 (m, 5H, H-β-DABA, H-α-DABA, H-α-<i>m</i>Tyr, H-α-Asp, H-α-Trp), 4.08–3.94 (m, 2H, H-2′-uridine, H-3′-uridine), 3.15–2.80 (m, 7H, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 2.70–2.62 (m, 2H, H-β-Asp), 1.21–1.14 (m, 3H, H-γ-DABA); ESIMS-LR <i>m</i>/<i>z</i>: 864.4 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>39</sub>H<sub>46</sub>N<sub>9</sub>O<sub>14</sub>, 864.3159; found, 864.3103; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −1.7 (<i>c</i> 0.12, MeOH).</div></div><div id="sec4_66" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> Compound <b>25m</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25m</b> (1.3 mg, 1.3 μmol, 21% over three steps) was obtained as a pale yellow powder using <b>30m</b> (10 mg, 6.3 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.36 (s, 1H, H-N3-uridine), 10.77 (s, 1H, NH-δ1-Trp), 9.52–9.21 (m, 3H, NH-5′-uridine, NH-<i>m</i>Tyr), 8.18–7.82 (m, 1H, DABA-NH), 7.51 (d, 1H, H-ε3-Trp, <i>J</i><sub>H-ε3-Trp,H-ζ2-Trp</sub> = 7.6 Hz), 7.36–6.51 (m, 9H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.50–6.35 (m, 1H, NH-Glu), 6.28–6.19 (m, 1H, NH-α-Trp), 6.13–6.04 (m, 1H, H-1′-uridine), 5.73–5.61 (m, 1H, H-5′-uridine), 5.50–5.42 (m, 1H, H-5-uridine), 4.99–4.83 (m, 1H, H-β-DABA), 4.82–4.28 (m, 4H, H-α-DABA, H-α-<i>m</i>Tyr, H-α-Trp, H-α-Glu), 4.20–4.07 (m, 2H, H-2′-uridine, H-3′-uridine), 3.14–2.82 ((m, 7H, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 2.25–2.13 (m, 2H, H-β-Gln), 1.89–1.72 (m, 2H, H-γ-Gln), 1.71–1.57 (m, 2H, H-γ-Gln), 1.13–0.98 (m, 3H, H-γ-DABA); ESIMS-LR <i>m</i>/<i>z</i>: 878.3 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>40</sub>H<sub>48</sub>N<sub>9</sub>O<sub>14</sub>, 878.3315; found, 878.3329; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +1.7 (<i>c</i> 0.10, MeOH).</div></div><div id="sec4_67" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> Compound <b>25n</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25n</b> (5.5 mg, 5.5 μmol, 47% over three steps) was obtained as a pale yellow powder using <b>30n</b> (20 mg, 12 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.39 (s, 1H, H-N3-uridine), 10.79 (s, 1H, NH-δ1-Trp), 9.58–9.18 (m, 3H, NH-5′-uridine, NH-<i>m</i>Tyr), 8.15–7.86 (m, 1H, DABA-NH), 7.78–7.57 (m, 1H, H-ε3-Trp), 7.38–6.52 (m, 9H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.48–6.39 (m, 1H, NH-α-Lys), 6.26–6.19 (m, 1H, NH-α-Trp), 6.10–6.03 (m, 1H, H-1′-uridine), 5.75–5.70 (m, 1H, H-5′-uridine), 5.52–5.45 (m, 1H, H-5-uridine), 4.98–4.84 (m, 1H, H-β-DABA), 4.82–4.74 (m, 1H, H-α-DABA), 4.71–4.30 (m, 3H, H-α-<i>m</i>Tyr, H-α-Trp, H-α-Lys), 4.18–4.07 (m, 2H, H-2′-uridine, H-3′-uridine), 3.12–2.82 (m, 7H, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 2.80–2.63 (m, 2H, H-ε-Lys), 1.63–1.20 (m, 6H, H-β-Lys, H-γ-Lys, H-δ-Lys), 1.42 (m, 2H, NH<sub>2</sub>-Lys), 1.28 (m, 2H, H-γ-Lys), 1.09 (d, 3H, H-γ-DABA, <i>J</i><sub>H-γ-DABA,H-β-DABA</sub> = 6.7 Hz); ESIMS-LR <i>m</i>/<i>z</i>: 877.5 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>41</sub>H<sub>53</sub>N<sub>10</sub>O<sub>12</sub>, 877.3839; found, 877.3905; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +0.1 (<i>c</i> 0.52, MeOH).</div></div><div id="sec4_68" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Compound <b>25o</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25o</b> (1.4 mg, 1.4 μmol, 25% over three steps) was obtained as a pale yellow powder using <b>30o</b> (10 mg, 5.5 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.39 (s, 1H, H-N3-uridine), 10.77 (s, 1H, NH-δ1-Trp), 9.61–9.20 (m, 3H, NH-5′-uridine, NH-<i>m</i>Tyr), 8.07–7.86 (m, 1H, DABA-NH), 7.57–7.41 (m, 1H, H-ε3-Trp), 7.38–6.55 (m, 9H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.52–6.40 (m, 1H, NH-α-Arg), 6.32–6.26 (m, 1H, NH-α-Trp), 6.10–6.05 (m, 1H, H-1′-uridine), 5.75–5.69 (m, 1H, H-5′-uridine), 5.49–5.42 (m, 1H, H-5-uridine), 4.97–4.74 (m, 1H, H-β-DABA), 4.74–4.58 (m, 1H, H-α-DABA), 4.51–4.11 (m, 3H, H-α-mTyr, H-α-Trp, H-α-Arg), 4.00–3.88 (m, 2H, H-2′-uridine, H-3′-uridine), 3.15–2.81 (m, 7H, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 2.70–2.65 (m, 2H, H-δ-Arg), 1.66–1.35 (m, 4H, H-β-Aργ, H-γ-Arg), 1.11–1.00 (m, 3H, H-γ-DABA); ESIMS-LR <i>m</i>/<i>z</i>: 905.5 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>41</sub>H<sub>53</sub>N<sub>12</sub>O<sub>12</sub>, 905.3900; found, 905.3150; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +3.4 (<i>c</i> 0.11, MeOH).</div></div><div id="sec4_69" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> Compound <b>25p</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25p</b> (1.8 mg, 1.9 μmol, 15% over three steps) was obtained as a pale yellow powder using <b>30p</b> (20 mg, 13 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.36 (s, 1H, H-N3-uridine), 10.76 (s, 1H, NH-δ1-Trp), 9.46–9.21 (m, 3H, NH-5′-uridine, NH-<i>m</i>Tyr), 8.23–7.83 (m, 1H, DABA-NH), 7.57–7.40 (m, 1H, H-ε3-Trp), 7.37–6.52 (m, 9H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp), 6.49–6.33 (m, 2H, NH-Ser, NH-α-Trp), 6.15–6.02 (m, 1H, H-1′-uridine), 5.77–5.60 (m, 1H, H-5′-uridine), 5.55–5.42 (m, 1H, H-5-uridine), 5.07–3.84 (m, 7H, H-β-DABA, H-α-DABA, H-α-Ser, H-α-<i>m</i>Tyr, H-α-Trp, H-2′-uridine, H-3′-uridine), 3.07–2.79 (m, 9H, H-β-Trp, H-β-Ser, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 1.30–0.99 (m, 3H, H-γ-DABA); ESIMS-LR <i>m</i>/<i>z</i>: 836.2 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>38</sub>H<sub>46</sub>N<sub>9</sub>O<sub>13</sub>, 836.3210; found, 836.3187; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +1.3 (<i>c</i> 0.15, MeOH).</div></div><div id="sec4_70" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Compound <b>25q</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25q</b> (2.9 mg, 3.0 μmol, 27% over three steps) was obtained as a pale yellow powder using <b>30q</b> (20 mg, 11 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.37 (s, 1H, H-N3-uridine), 10.75 (s, 1H, NH-δ1-Trp), 9.48–9.20 (m, 3H, NH-5′-uridine, NH-<i>m</i>Tyr), 8.15–7.81 (m, 1H, DABA-NH), 7.52 (d, 1H, H-ε3-Trp, <i>J</i><sub>H-ε3-Trp,H-ζ2-Trp</sub> = 8.1 Hz), 7.36–6.48 (m, 10H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp, NH-Thr), 6.54 (d, 1H, NH-Thr, <i>J</i><sub>NH-Thr,α-Thr</sub> = 7.8 Hz), 6.38 (d, 1H, NH-α-Trp, <i>J</i><sub>NH-α-Trp,H-α-Trp</sub> = 8.1 Hz), 6.11–6.04 (m, 1H, H-1′-uridine), 5.72–5.67 (m, 1H, H-5′-uridine), 5.51–5.46 (m, 1H, H-5-uridine), 4.98–4.87 (m, 1H, H-β-DABA), 4.82–4.74 (m, 1H, H-α-DABA), 4.73–4.27 (m, 3H, H-α-<i>m</i>Tyr, H-α-Trp, H-α-Thr), 4.14–3.74 (m, 3H, H-β-Thr, H-2′-uridine, H-3′-uridine), 3.13–2.80 (m, 7H, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 1.11 (d, 3H, H-γ-Thr, <i>J</i><sub>H-γ-Thr,H-β-Thr</sub> = 7.2 Hz), 1.01–0.95 (m, 3H, H-γ-DABA); ESIMS-LR <i>m</i>/<i>z</i>: 850.5 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>39</sub>H<sub>48</sub>N<sub>9</sub>O<sub>13</sub>, 850.3366; found, 850.3347; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −1.9 (<i>c</i> 0.26, MeOH).</div></div><div id="sec4_71" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Compound <b>25r</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25r</b> (0.5 mg, 0.5 μmol, 10% over three steps) was obtained as a pale yellow powder using <b>30r</b> (10 mg, 5.6 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.47 (s, 1H, H-N3-uridine), 10.81 (s, 1H, NH-δ1-Trp), 9.65–9.39 (m, 3H, NH-5′-uridine, NH-<i>m</i>Tyr), 8.21–7.87 (m, 3H, NH-DABA, NH-γ-Asn), 7.55–7.46 (m, 1H, H-ε3-Trp), 7.36–6.40 (m, 9H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp, NH-α-Asn, NH-α-Trp), 6.17–6.00 (m, 1H, H-1′-uridine), 5.75–5.56 (m, 1H, H-5′-uridine), 5.50–5.38 (m, 1H, H-5-uridine), 4.99–4.84 (m, 1H, H-β-DABA), 4.81–4.22 (m, 4H, H-α-DABA, H-α-<i>m</i>Tyr, H-α-Asn, H-α-Trp), 4.06–3.86 (m, 2H, H-2′-uridine, H-3′-uridine), 3.17–2.26 (m, 7H, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>, H-β-Asn), 1.11–0.96 (m, 3H, H-γ-DABA); ESIMS-LR <i>m</i>/<i>z</i>: 863.4 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>39</sub>H<sub>47</sub>N<sub>10</sub>O<sub>13</sub>, 863.3319; found, 863.3333; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +2.1 (<i>c</i> 0.02, MeOH).</div></div><div id="sec4_72" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> Compound <b>25s</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25s</b> (1.2 mg, 1.2 μmol, 21% over three steps) was obtained as a pale yellow powder using <b>30s</b> (10 mg, 5.6 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.37 (s, 1H, H-N3-uridine), 10.77 (s, 1H, NH-δ1-Trp), 9.55–9.23 (m, 3H, NH-5′-uridine, NH-<i>m</i>Tyr), 8.17–7.83 (m, 1H, DABA-NH), 7.54–7.44 (m, 1H, H-ε3-Trp), 7.37–6.51 (m, 11H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp, NH-δ-Gln), 6.49–6.37 (m, 1H, NH-α-Gln), 6.29–6.22 (m, 1H, NH-α-Trp), 6.11–6.05 (m, 1H, H-1′-uridine), 5.73–5.67 (m, 1H, H-5′-uridine), 5.50–5.44 (m, 1H, H-5-uridine), 5.16–4.28 (m, 5H, H-β-DABA, H-α-DABA, H-α-<i>m</i>Tyr, H-α-Trp, H-α-Gln), 4.19–4.07 (m, 2H, H-2′-uridine, H-3′-uridine), 3.13–2.81 (m, 7H, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 2.17–1.99 (m, 2H, H-γ-Gln), 1.86–1.72 (m, 1H, H-β-Gln), 1.70–1.55 (m, 1H, H-β-Gln), 1.13–1.00 (m, 3H, H-γ-DABA); ESIMS-LR <i>m</i>/<i>z</i>: 877.4 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>40</sub>H<sub>49</sub>N<sub>10</sub>O<sub>13</sub>, 877.3475; found, 877.3601; [α]<sub class="stack">D</sub><sup class="stack">20</sup> +10.7 (<i>c</i> 0.12, MeOH).</div></div><div id="sec4_73" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> Compound <b>25t</b></h3><div class="NLM_p last">In a manner similar to the synthesis of <b>25a</b>, <b>25t</b> (2.2 mg, 2.3 μmol, 18% over three steps) was obtained as a pale yellow powder using <b>30t</b> (20 mg, 13 μmol). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, <i>T</i> = 50 °C, 3:2 mixture of rotamers, selected data for the major rotamer): δ 11.39 (s, 1H, H-N3-uridine), 10.73 (s, 1H, NH-δ1-Trp), 9.54–9.23 (m, 3H, NH-5′-uridine, NH-<i>m</i>Tyr), 8.10–7.82 (m, 1H, DABA-NH), 7.59–7.48 (m, 1H, H-ε3-Trp), 7.37–6.47 (m, 9H, H-6-uridine, H-ζ1-Trp, H-η-Trp, H-2-<i>m</i>Tyr, H-4-<i>m</i>Tyr, H-5-<i>m</i>Tyr, H-6-<i>m</i>Tyr, H-δ1-Trp, H-ζ2-Trp, NH-α-Trp), 6.17–6.02 (m, 1H, H-1′-uridine), 5.73–5.65 (m, 1H, H-5′-uridine), 5.57–5.48 (m, 1H, H-5-uridine), 4.96–4.85 (m, 1H, H-β-DABA), 4.78–4.20 (m, 4H, H-α-DABA, H-α-<i>m</i>Tyr, H-α-Trp, H-α-Pro), 4.09–3.91 (m, 2H, H-2′-uridine, H-3′-uridine), 3.20–2.78 (m, 9H, H-δ-Pro, H-β-Trp, H-β-<i>m</i>Tyr, NC<i>H</i><sub>3</sub>), 2.05–1.66 (m, 4H, H-β-Pro, H-γ-Pro), 1.13–0.99 (m, 3H, H-γ-DABA); ESIMS-LR <i>m</i>/<i>z</i>: 846.4 [(M + H)<sup>+</sup>]; ESIMS-HR <i>m</i>/<i>z</i>: calcd for C<sub>40</sub>H<sub>48</sub>N<sub>9</sub>O<sub>12</sub>, 846.3417; found, 846.3422; [α]<sub class="stack">D</sub><sup class="stack">20</sup> −6.1 (<i>c</i> 0.19, MeOH).</div></div><div id="sec4_74" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> Fluorescence-Based MraY Assay</h3><div class="NLM_p last">Reactions were carried out in a 384-well microplate. A solution containing 10 μM of UDP-MurNAc-dansylpentapeptide in 20 μL of an assay buffer [50 mM Tris-HCl (pH 7.6), 50 mM KCl, 25 mM MgCl<sub>2</sub>, 0.2% Triton X-100, 8% glycerol, 50 μM C55-P] was prepared. The reaction was initiated by the addition of <i>S. aureus</i> MraY enzyme<a onclick="showRef(event, 'ref5400'); return false;" href="javascript:void(0);" class="ref ref5400">(54)</a> (55 ng/5 μL/well). After 3 h of incubation at room temperature, the formation of dansylated lipid I was monitored by fluorescence enhancement (excitation at 355 nm, emission at 535 nm) by using an Infinite M200 microplate reader (Tecan). The inhibitory effects of each compound were determined in the MraY assays described above. The mixtures contained 2% dimethyl sulfoxide in order to increase the solubility of the compounds.</div></div><div id="sec4_75" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> Evaluation of Antibacterial Activity</h3><div class="NLM_p last">MICs were determined by a microdilution broth method as recommended by the CLSI<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(55)</a> with cation-adjusted Mueller–Hinton broth (MHB). Serial twofold dilutions of each compound were made in appropriate broth, and the strains were inoculated with 5 × 10<sup>5</sup> cfu/mL in 96-well plates (each 0.1 mL/well). The plates were incubated at 37 °C for 18 h and then MICs were determined.</div></div><div id="sec4_76" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> Evaluation of Cytotoxicity</h3><div class="NLM_p last">Cytotoxic activity of the compounds against HepG2 cells was measured using cell counting kit-8 according to manufacturer’s protocol. Briefly, HepG2 cells (1 × 10<sup>5</sup> cells/well) in a 96-well plate were cultured in D-MEM (low glucose) medium containing 10% fetal bovine serum in the presence of test compounds at 37 °C for 24 h under 5% CO<sub>2</sub> atmosphere. A solution of cell counting kit-8 reagent in medium (1:10) was added. The plates were incubated at 37 °C for 2 h under 5% CO<sub>2</sub> atmosphere and then 450 nm absorbance was measured.</div></div><div id="sec4_77" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97"> Modeling of Complex Structures of <b>2</b> Bound to MraY<sub>SA</sub> and MraY<sub>PA</sub></h3><div class="NLM_p last">The target amino acid sequences of MraY<sub>SA</sub> and MraY<sub>PA</sub> were retrieved from the UniProtKB database [accession number: <a href="https://Q6GHQ3" class="ext-link">Q6GHQ3</a> (MraY<sub>SA</sub>), <a href="https://Q9HVZ8" class="ext-link">Q9HVZ8</a> (MraY<sub>PA</sub>)]. The X-ray crystal structure of MraY<sub>AA</sub> in complex with MRY D2 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CKR">5CKR</a>) was used as a template structure. The three-dimensional structures of MraY<sub>SA</sub> and MraY<sub>PA</sub> were generated by using the SWISS-MODEL server, and each generated structure was superimposed on the template structure. The template structure was separated into protein and ligand structures; then, the ligand structures were combined with the modeled structures to build initial complex structures. Finally, each complex structure was energy-minimized using MacroModel with a GB/SA water solvent model to generate the complex structure shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>. All calculations were performed using an OPLS3 force field, and heavy atoms of the side chain of Phe262, Lys70, Asp196, Asn255, and the uracil moiety of the ligand were restrained during the calculation. Coordinates have been included in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00973/suppl_file/jm0c00973_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_78" class="NLM_sec NLM_sec_level_2"><div id="ac_i98" class="anchor-spacer"></div><h3 class="article-section__title" id="_i98"> Continuous <i>in Vitro</i> Cultivation with <b>2</b> in <i>P. aeruginosa</i></h3><div class="NLM_p last">Continuous cultivation with MRD was performed using MHB-II (Becton, Dickinson and Company, USA) containing sub-MIC of MRD (4 mg/L). The parent strain was used <i>P. aeruginosa</i> PAO1 (day 0) and cultivated in MHB-II containing <b>2</b> overnight at 37 °C. Each 1 μL (approximately 1 × 10<sup>6</sup> cfu) of cell culture was inoculated into 1 mL of fresh MHB-II (1:1000, vol/vol) containing sub-MIC of MRD and incubated overnight at 37 °C. This step was continued for 30 days, and strains derived from each day were determined MIC of MRD with reference to CLSI guidelines.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(56)</a></div></div><div id="sec4_79" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> Whole Genome Sequencing</h3><div class="NLM_p last">The experiment was conducted in a manner similar to a procedure previously described. The genomic DNA of <i>P. aeruginosa</i> PAO1 (day 0) and the MRD-resistant mutants derived at days 1, 2, and 30 during continuous cultivation with sub-MIC of MRD (4 mg/L) were isolated using a DNeasy Blood and Tissue Kit (Qiagen, Hilden, Germany). The isolated DNA was sequenced by MiSeq (Illumina, San Diego, CA) using a Nextera XT DNA Library Prep Kit (Illumian). For the assembly, the contig was mapped with 300-bp paired-end reads to the genome of the parent strain followed by polishing using a CLC Genomic Workbench (Qiagen). Genome annotations were using DFAST based on PROKKA.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(57)</a> A comprehensive genetic comparison between <i>P. aeruginosa</i> PAO1 (day 0) and the MRD-resistant mutants was performed, and non-synonymous mutations were detected by a CLC Genomic Workbench by sorting the obtained contig count frequency, which was more than 70%.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i100"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00973" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44382" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44382" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00973?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00973</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Preparation of compounds, <sup>1</sup>H, <sup>13</sup>C NMR spectrum of compounds, chromatograms of LC–MS analysis, data of cell viability assay, data of MraY inhibitory assay, coordinates of MraY<sub>SA</sub> and MraY<sub>PA</sub> docked with <b>2</b>, and structures with SMILES codes (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00973/suppl_file/jm0c00973_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Biological activity of <b>2</b>, <b>19</b>, <b>22</b>, <b>23</b>, <b>24</b>, and <b>25a</b>–<b>t</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00973/suppl_file/jm0c00973_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00973/suppl_file/jm0c00973_si_001.pdf">jm0c00973_si_001.pdf (10.19 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00973/suppl_file/jm0c00973_si_003.csv">jm0c00973_si_003.csv (4.73 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00973" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43408" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43408" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Satoshi Ichikawa</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</span>; 
    <span class="hlFld-Affiliation affiliation">Center
for Research and Education on Drug Discovery, Faculty of Pharmaceutical
Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</span>; 
    <span class="hlFld-Affiliation affiliation">Global Station
for Biosurfaces and Drug Discovery, Global Institution for Collaborative
Research and Education (GI-CoRE), Hokkaido
University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5345-5007" title="Orcid link">http://orcid.org/0000-0001-5345-5007</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#4c252f2425272d3b2d0c3c242d3e216224232739282d25622d2f62263c"><span class="__cf_email__" data-cfemail="c7aea4afaeaca6b0a687b7afa6b5aae9afa8acb2a3a6aee9a6a4e9adb7">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuma Terasawa</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chisato Sataka</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Toyotaka Sato</span> - <span class="hlFld-Affiliation affiliation">Department
of Microbiology, Sapporo Medical University
School of Medicine, South-1,
West-17, Chuo-ku, Sapporo 060-8556, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kazuki Yamamoto</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yukari Fukushima</span> - <span class="hlFld-Affiliation affiliation">Division
of Bioresources, Hokkaido University Research
Center for Zoonosis Control, Sapporo 001-0020, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chie Nakajima</span> - <span class="hlFld-Affiliation affiliation">Division
of Bioresources, Hokkaido University Research
Center for Zoonosis Control, Sapporo 001-0020, Japan</span>; 
    <span class="hlFld-Affiliation affiliation">Global
Station for Zoonosis Control, Global Institution for Collaborative
Research and Education (GI-CoRE), Hokkaido
University, Kita-20, Nishi-10, Kita-ku, Sapporo 001-0020, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yasuhiko Suzuki</span> - <span class="hlFld-Affiliation affiliation">Division
of Bioresources, Hokkaido University Research
Center for Zoonosis Control, Sapporo 001-0020, Japan</span>; 
    <span class="hlFld-Affiliation affiliation">Global
Station for Zoonosis Control, Global Institution for Collaborative
Research and Education (GI-CoRE), Hokkaido
University, Kita-20, Nishi-10, Kita-ku, Sapporo 001-0020, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Akira Katsuyama</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</span>; 
    <span class="hlFld-Affiliation affiliation">Center
for Research and Education on Drug Discovery, Faculty of Pharmaceutical
Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4062-5561" title="Orcid link">http://orcid.org/0000-0002-4062-5561</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Takanori Matsumaru</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</span>; 
    <span class="hlFld-Affiliation affiliation">Center
for Research and Education on Drug Discovery, Faculty of Pharmaceutical
Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</span>; 
    <span>Present Address:
                        Department of Chemistry, Faculty of Science and Technology, Keio University, Hiyoshi 3-14-1, Minatokita-ku, Yokohama, 223-8522, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fumika Yakushiji</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Pharmaceutical Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</span>; 
    <span class="hlFld-Affiliation affiliation">Center
for Research and Education on Drug Discovery, Faculty of Pharmaceutical
Sciences, Hokkaido University, Kita-12, Nishi-6, Kita-ku, Sapporo 060-0812, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9003-3767" title="Orcid link">http://orcid.org/0000-0002-9003-3767</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shin-ichi Yokota</span> - <span class="hlFld-Affiliation affiliation">Department
of Microbiology, Sapporo Medical University
School of Medicine, South-1,
West-17, Chuo-ku, Sapporo 060-8556, Japan</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i103">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28216" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28216" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Chihiro Inagaki, Kousuke Nakamura, and Naoto Tokoro for initial investigation for the synthesis of <b>2</b>. We are also thankful Dr. K. Nishiguchi and Dr. S. Arioka (Shionogi Co., Ltd.) for MraY expression and purification. This research was supported in part by JSPS KAKENHI Grant-in-Aid for Scientific Research (B) (grant numbers 16H05097 and 19H03345 to S.I.), Grant-in Aid for Scientific Research on Innovative Areas “Frontier Research on Chemical Communications” (nos. 18H04599 and 20H04757 to S.I.), JSPS KAKENHI Grant-in-Aid for Research for Young Scientist (grant number JP19K16648 to T.S.), Takeda Foundation, The Tokyo Biomedical Research Foundation and was partly supported by Hokkaido University, Global Facility Center (GFC), Pharma Science Open Unit (PSOU), funded by MEXT under “Support Program for Implementation of New Equipment Sharing System”, Platform Project for Supporting Drug Discovery and Life Science Research [Basis for Supporting Innovative Drug Discovery and Life Science Research (BINDS)] from AMED under grant number JP18am0101093j0002, AMED under grant number JP19ak0101118h0001, Japan Initiative for Global Research Network on Infectious Diseases (J-GRID) from the Ministry of Education, Culture, Sport, Science, and Technology in Japan, and MEXT for the Joint Research Program of the Research Center for Zoonosis Control, Hokkaido University.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">Alloc</td><td class="NLM_def"><p class="first last">allyloxycarbonyl</p></td></tr><tr><td class="NLM_term">BOM</td><td class="NLM_def"><p class="first last">benzyloxymethyl</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate</p></td></tr><tr><td class="NLM_term">HepG2</td><td class="NLM_def"><p class="first last">human hepatocellular carcinoma</p></td></tr><tr><td class="NLM_term">MIC</td><td class="NLM_def"><p class="first last">minimum inhibitory concentration</p></td></tr><tr><td class="NLM_term">MraY</td><td class="NLM_def"><p class="first last">phospho-MurNAc-pentapeptide transferase</p></td></tr><tr><td class="NLM_term">MRD</td><td class="NLM_def"><p class="first last">mureidomycin</p></td></tr><tr><td class="NLM_term">MRY</td><td class="NLM_def"><p class="first last">muraymycin</p></td></tr><tr><td class="NLM_term"><i>m</i>Tyr</td><td class="NLM_def"><p class="first last"><i>m</i>-tyrosine</p></td></tr><tr><td class="NLM_term">NppA1A2BCD</td><td class="NLM_def"><p class="first last">nucleoside peptide permease</p></td></tr><tr><td class="NLM_term">PCD</td><td class="NLM_def"><p class="first last">pacidamycin</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i105">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21617" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21617" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 57 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagiya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takimoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomono, K.</span></span> <span> </span><span class="NLM_article-title">Non-nosocomial healthcare-associated left-sided <i>Pseudomonas aeruginosa</i> endocarditis: a case report and literature review</span>. <i>BMC Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1186/s12879-016-1757-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1186%2Fs12879-016-1757-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=27543116" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BC2szivFKktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=431-437&author=H.+Hagiyaauthor=T.+Tanakaauthor=K.+Takimotoauthor=H.+Yoshidaauthor=N.+Yamamotoauthor=Y.+Akedaauthor=K.+Tomono&title=Non-nosocomial+healthcare-associated+left-sided+Pseudomonas+aeruginosa+endocarditis%3A+a+case+report+and+literature+review&doi=10.1186%2Fs12879-016-1757-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Non-nosocomial healthcare-associated left-sided Pseudomonas aeruginosa endocarditis: a case report and literature review</span></div><div class="casAuthors">Hagiya Hideharu; Yoshida Hisao; Yamamoto Norihisa; Akeda Yukihiro; Tomono Kazunori; Tanaka Takeshi; Takimoto Kohei</div><div class="citationInfo"><span class="NLM_cas:title">BMC infectious diseases</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">431</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  With the development of invasive medical procedures, an increasing number of healthcare-associated infective endocarditis cases have been reported.  In particular, non-nosocomial healthcare-associated infective endocarditis in outpatients with recent medical intervention has been increasingly identified.  CASE PRESENTATION:  A 66-year-old man with diabetes mellitus and a recent history of intermittent urethral self-catheterization was admitted due to a high fever.  Repeated blood cultures identified Pseudomonas aeruginosa, and transesophageal echocardiography uncovered a new-onset severe aortic regurgitation along with a vegetative valvular structure.  The patient underwent emergency aortic valve replacement surgery and was successfully treated with 6 weeks of high-dose meropenem and tobramycin.  Historically, most cases of P. aeruginosa endocarditis have occurred in the right side of the heart and in outpatients with a history of intravenous drug abuse.  In the case presented, the repeated manipulations of the urethra may have triggered the infection.  Our literature review for left-sided P. aeruginosa endocarditis showed that non-nosocomial infection accounted for nearly half of the cases and resulted in fatal outcomes as often as nosocomial cases.  A combination therapy with anti-pseudomonal beta-lactams or carbapenems and aminoglycosides may be the preferable treatment.  Medical treatment alone may be effective, and surgical treatment should be carefully considered.  CONCLUSIONS:  We presented a rare case of native aortic valve endocarditis caused by P. aeruginosa.  This case illustrates the importance of identifying the causative pathogen(s), especially for outpatients with a recent history of medical procedures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRlDlluXidKJdLIORRPd5lbfW6udTcc2eaz7AkEivLexbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szivFKktQ%253D%253D&md5=bf76c3c3de79b2e105fd1e5568e440c2</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1186%2Fs12879-016-1757-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12879-016-1757-y%26sid%3Dliteratum%253Aachs%26aulast%3DHagiya%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DTakimoto%26aufirst%3DK.%26aulast%3DYoshida%26aufirst%3DH.%26aulast%3DYamamoto%26aufirst%3DN.%26aulast%3DAkeda%26aufirst%3DY.%26aulast%3DTomono%26aufirst%3DK.%26atitle%3DNon-nosocomial%2520healthcare-associated%2520left-sided%2520Pseudomonas%2520aeruginosa%2520endocarditis%253A%2520a%2520case%2520report%2520and%2520literature%2520review%26jtitle%3DBMC%2520Infect.%2520Dis.%26date%3D2016%26volume%3D16%26spage%3D431%26epage%3D437%26doi%3D10.1186%2Fs12879-016-1757-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marcus, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashkenazi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samra, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livni, G.</span></span> <span> </span><span class="NLM_article-title">Community-acquired <i>Pseudomonas aeruginosa</i> urinary tract infections in children hospitalized in a tertiary center: relative frequency, risk factors, antimicrobial resistance and treatment</span>. <i>Infection</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">421</span>– <span class="NLM_lpage">426</span>, <span class="refDoi"> DOI: 10.1007/s15010-008-7328-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1007%2Fs15010-008-7328-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=18795227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BD1cjksFOjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=421-426&author=N.+Marcusauthor=S.+Ashkenaziauthor=Z.+Samraauthor=A.+Cohenauthor=G.+Livni&title=Community-acquired+Pseudomonas+aeruginosa+urinary+tract+infections+in+children+hospitalized+in+a+tertiary+center%3A+relative+frequency%2C+risk+factors%2C+antimicrobial+resistance+and+treatment&doi=10.1007%2Fs15010-008-7328-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Community-acquired Pseudomonas aeruginosa urinary tract infections in children hospitalized in a tertiary center: relative frequency, risk factors, antimicrobial resistance and treatment</span></div><div class="casAuthors">Marcus N; Ashkenazi S; Samra Z; Cohen A; Livni G</div><div class="citationInfo"><span class="NLM_cas:title">Infection</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">421-6</span>
        ISSN:<span class="NLM_cas:issn">0300-8126</span>.
    </div><div class="casAbstract">BACKGROUND:  The practice of antibiotic prophylaxis against recurrent urinary tract infection (UTI), with hospitalization reserved for severe or complicated cases, has led to changes in the nature and culprit uropathogens of community-acquired (CA), hospital-treated UTI.  Characterization of subgroups that need special considerations is crucial.  OBJECTIVES:  To elucidate the trends and characteristics of CA Pseudomonas UTI in hospitalized children; define the antibiotic susceptibility; determine the appropriateness of the empiric antibiotics used; compare to other causes of UTI in this population; and thereby define predictors for Pseudomonas UTI.  METHODS:  A prospective clinical and laboratory study from 2001 through 2005.  Children with P. aeruginosa UTI were characterized and compared with non-Pseudomonas UTI.  RESULTS:  Of 351 episodes of culture-proven CA UTI, 28 (8%) were caused by Pseudomonas, representing a 2.8-fold increase from our previous study.  Pseudomonas UTI was more common in children > 5 years (p < 0.01), with urinary abnormalities (p < 0.01) and with previous antibiotic use in the previous month (p < 0.001).  Pseudomonas UTI was often resistant to antibiotics usually recommended for empiric therapy; 25% was initially treated with inappropriate IV antibiotics (4.6% in the non-Pseudomonas group, p < 0.001) with 1.3 days longer IV antibiotics.  On multivariate analysis, risk factors for Pseudomonas UTI were previous antibiotic therapy and underlying urinary pathology.  CONCLUSIONS:  Pseudomonas UTI seems to increase in CA, hospital-treated children and is often treated inappropriately according to current treatment protocols.  Awareness of this trend and knowledge of the defined risk factors of Pseudomonas UTI might improve the empiric antibiotic therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQYTFUnWY3yh1HFYivNCdPRfW6udTcc2eZSqByVO8qp07ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cjksFOjug%253D%253D&md5=fde47dc9ec042d5292fc56b6a5166b08</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2Fs15010-008-7328-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs15010-008-7328-4%26sid%3Dliteratum%253Aachs%26aulast%3DMarcus%26aufirst%3DN.%26aulast%3DAshkenazi%26aufirst%3DS.%26aulast%3DSamra%26aufirst%3DZ.%26aulast%3DCohen%26aufirst%3DA.%26aulast%3DLivni%26aufirst%3DG.%26atitle%3DCommunity-acquired%2520Pseudomonas%2520aeruginosa%2520urinary%2520tract%2520infections%2520in%2520children%2520hospitalized%2520in%2520a%2520tertiary%2520center%253A%2520relative%2520frequency%252C%2520risk%2520factors%252C%2520antimicrobial%2520resistance%2520and%2520treatment%26jtitle%3DInfection%26date%3D2008%26volume%3D36%26spage%3D421%26epage%3D426%26doi%3D10.1007%2Fs15010-008-7328-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obritsch, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacLaren, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, R.</span></span> <span> </span><span class="NLM_article-title">Nosocomial infections due to multidrug-resistant <i>Pseudomonas aeruginosa</i>: epidemiology and treatment options</span>. <i>Pharmacotherapy</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1353</span>– <span class="NLM_lpage">1364</span>, <span class="refDoi"> DOI: 10.1592/phco.2005.25.10.1353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1592%2Fphco.2005.25.10.1353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=16185180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFGitbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2005&pages=1353-1364&author=M.+D.+Obritschauthor=D.+N.+Fishauthor=R.+MacLarenauthor=R.+Jung&title=Nosocomial+infections+due+to+multidrug-resistant+Pseudomonas+aeruginosa%3A+epidemiology+and+treatment+options&doi=10.1592%2Fphco.2005.25.10.1353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Nosocomial infections due to multidrug-resistant Pseudomonas aeruginosa: Epidemiology and treatment options</span></div><div class="casAuthors">Obritsch, Marilee D.; Fish, Douglas N.; MacLaren, Robert; Jung, Rose</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacotherapy</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1353-1364</span>CODEN:
                <span class="NLM_cas:coden">PHPYDQ</span>;
        ISSN:<span class="NLM_cas:issn">0277-0008</span>.
    
            (<span class="NLM_cas:orgname">Pharmacotherapy Publications</span>)
        </div><div class="casAbstract">A review.  Pseudomonas aeruginosa is one of the leading gram-neg. organisms assocd. with nosocomial infections.  The increasing frequency of multidrug-resistant Pseudomonas aeruginosa (MDRPA) strains is concerning as efficacious antimicrobial options are severely limited.  By searching MEDLINE from Jan. 1966-Feb. 2005 and relevant journals for abstrs., we reviewed the frequency, risk factors, and patient outcomes of MDRPA nosocomial infections in critically ill patients, detd. the available antimicrobial therapies, and then provided recommendations for clinicians.  The definition of MDRPA was established as isolates intermediate or resistant to at least three drugs in the following classes: β-lactams, carbapenems, aminoglycosides, and fluoroquinolones.  Reported rates of MDRPA varied from 0.6-.32% according to geog. location and type of surveillance study.  Risk factors for MDRPA infection included prolonged hospitalization, exposure to antimicrobial therapy, and immunocompromised states such as human immunodeficiency virus infection.  Emergence of MDRPA isolates during therapy was reported in 27-72% of patients with initially susceptible P. aeruginosa isolates.  Patients with severe MDRPA infections should be treated with combination therapy, consisting of an antipseudomonal β-lactam with an aminoglycoside or fluoroquinolone rather than aminoglycoside and fluoroquinolone combinations, to provide adequate therapy and improve patient outcomes.  Synergy has been obsd. when resistant antipseudomonal drugs were combined in vitro against MDRPA with successful clin. application reported in two centers.  Colistin with adjunctive therapy, such as a β-lactam or rifampin, may be a useful agent in MDRPA when antimicrobial options are limited, but patients should be monitored closely for toxicities assocd. with this agent.  Standardization of terminol. for MDRPA isolates is needed for consistency and comparability of surveillance and institutional reports.  Clin. studies are needed to identify risk factors for MDRPA development and to det. the economic impact of these infections, as well as to det. the most efficacious antimicrobial regimens and duration of therapy to maximize outcomes in the treatment of MDRPA infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW44ZKjwzU0LVg90H21EOLACvtfcHk0liD-ktckfPMgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFGitbzE&md5=09a8ef3955486ab6279f37148a364de5</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1592%2Fphco.2005.25.10.1353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1592%252Fphco.2005.25.10.1353%26sid%3Dliteratum%253Aachs%26aulast%3DObritsch%26aufirst%3DM.%2BD.%26aulast%3DFish%26aufirst%3DD.%2BN.%26aulast%3DMacLaren%26aufirst%3DR.%26aulast%3DJung%26aufirst%3DR.%26atitle%3DNosocomial%2520infections%2520due%2520to%2520multidrug-resistant%2520Pseudomonas%2520aeruginosa%253A%2520epidemiology%2520and%2520treatment%2520options%26jtitle%3DPharmacotherapy%26date%3D2005%26volume%3D25%26spage%3D1353%26epage%3D1364%26doi%3D10.1592%2Fphco.2005.25.10.1353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sonmezer, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertem, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdinc, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaya Kilic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tulek, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adiloglu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatipoglu, C.</span></span> <span> </span><span class="NLM_article-title">Evaluation of risk factors for antibiotic resistance in patients with nosocomial infections caused by. <i>Pseudomonas aeruginosa</i></span>. <i>Can. J. Infect Dis. Med. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2016</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1155/2016/1321487</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1155%2F2016%2F1321487" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2016&publication_year=2016&pages=1-9&author=M.+C.+Sonmezerauthor=G.+Ertemauthor=F.+S.+Erdincauthor=E.+Kaya+Kilicauthor=N.+Tulekauthor=A.+Adilogluauthor=C.+Hatipoglu&title=Evaluation+of+risk+factors+for+antibiotic+resistance+in+patients+with+nosocomial+infections+caused+by.+Pseudomonas+aeruginosa&doi=10.1155%2F2016%2F1321487"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1155%2F2016%2F1321487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2016%252F1321487%26sid%3Dliteratum%253Aachs%26aulast%3DSonmezer%26aufirst%3DM.%2BC.%26aulast%3DErtem%26aufirst%3DG.%26aulast%3DErdinc%26aufirst%3DF.%2BS.%26aulast%3DKaya%2BKilic%26aufirst%3DE.%26aulast%3DTulek%26aufirst%3DN.%26aulast%3DAdiloglu%26aufirst%3DA.%26aulast%3DHatipoglu%26aufirst%3DC.%26atitle%3DEvaluation%2520of%2520risk%2520factors%2520for%2520antibiotic%2520resistance%2520in%2520patients%2520with%2520nosocomial%2520infections%2520caused%2520by.%2520Pseudomonas%2520aeruginosa%26jtitle%3DCan.%2520J.%2520Infect%2520Dis.%2520Med.%2520Microbiol.%26date%3D2016%26volume%3D2016%26spage%3D1%26epage%3D9%26doi%3D10.1155%2F2016%2F1321487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tam, V. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdelraouf, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brioso, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ameka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCaskey, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weston, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caeiro, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garey, K. W.</span></span> <span> </span><span class="NLM_article-title">Prevalence, resistance mechanisms, and susceptibility of multidrug-resistant bloodstream isolates of <i>Pseudomonas aeruginosa</i></span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1164</span>, <span class="refDoi"> DOI: 10.1128/aac.01446-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1128%2FAAC.01446-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=20086165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFWqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2010&pages=1160-1164&author=V.+H.+Tamauthor=K.-T.+Changauthor=K.+Abdelraoufauthor=C.+G.+Briosoauthor=M.+Amekaauthor=L.+A.+McCaskeyauthor=J.+S.+Westonauthor=J.-P.+Caeiroauthor=K.+W.+Garey&title=Prevalence%2C+resistance+mechanisms%2C+and+susceptibility+of+multidrug-resistant+bloodstream+isolates+of+Pseudomonas+aeruginosa&doi=10.1128%2Faac.01446-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence, resistance mechanisms, and susceptibility of multidrug-resistant bloodstream isolates of Pseudomonas aeruginosa</span></div><div class="casAuthors">Tam, Vincent H.; Chang, Kai-Tai; Abdelraouf, Kamilia; Brioso, Cristina G.; Ameka, Magdalene; McCaskey, Laurie A.; Weston, Jaye S.; Caeiro, Juan-Pablo; Garey, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1160-1164</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa is an important pathogen commonly implicated in nosocomial infections.  The occurrence of multidrug-resistant (MDR) P. aeruginosa strains is increasing worldwide and limiting our therapeutic options.  The MDR phenotype can be mediated by a variety of resistance mechanisms, and the corresponding relative biofitness is not well established.  We examd. the prevalence, resistance mechanisms, and susceptibility of MDR P. aeruginosa isolates (resistant to ≥3 classes of antipseudomonal agents [penicillins/cephalosporins, carbapenems, quinolones, and aminoglycosides]) obtained from a large, university-affiliated hospital.  Among 235 nonrepeat bloodstream isolates screened between 2005 and 2007, 33 isolates (from 20 unique patients) were found to be MDR (crude prevalence rate, 14%).  All isolates were resistant to carbapenems and quinolones, 91% were resistant to penicillins/cephalosporins, and 21% were resistant to the aminoglycosides.  By using the first available isolate for each bacteremia episode (n = 18), 13 distinct clones were revealed by repetitive-element-based PCR.  Western blotting revealed eight isolates (44%) to have MexB overexpression.  Prodn. of a carbapenemase (VIM-2) was found in one isolate, and mutations in gyrA (T83I) and parC (S87L) were commonly found.  Growth rates of most MDR isolates were similar to that of the wild type, and two isolates (11%) were found to be hypermutable.  All available isolates were susceptible to polymyxin B, and only one isolate was nonsusceptible to colistin (MIC, 3 mg/L), but all isolates were nonsusceptible to doripenem (MIC, >2 mg/L).  Understanding and continuous monitoring of the prevalence and resistance mechanisms of MDR P. aeruginosa would enable us to formulate rational treatment strategies to combat nosocomial infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYWXf_nYRWLLVg90H21EOLACvtfcHk0lhzHs4bw9j1rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFWqsLs%253D&md5=949cd3e6fe2dd51c0288766a705bc570</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FAAC.01446-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01446-09%26sid%3Dliteratum%253Aachs%26aulast%3DTam%26aufirst%3DV.%2BH.%26aulast%3DChang%26aufirst%3DK.-T.%26aulast%3DAbdelraouf%26aufirst%3DK.%26aulast%3DBrioso%26aufirst%3DC.%2BG.%26aulast%3DAmeka%26aufirst%3DM.%26aulast%3DMcCaskey%26aufirst%3DL.%2BA.%26aulast%3DWeston%26aufirst%3DJ.%2BS.%26aulast%3DCaeiro%26aufirst%3DJ.-P.%26aulast%3DGarey%26aufirst%3DK.%2BW.%26atitle%3DPrevalence%252C%2520resistance%2520mechanisms%252C%2520and%2520susceptibility%2520of%2520multidrug-resistant%2520bloodstream%2520isolates%2520of%2520Pseudomonas%2520aeruginosa%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2010%26volume%3D54%26spage%3D1160%26epage%3D1164%26doi%3D10.1128%2Faac.01446-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Page, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heim, J.</span></span> <span> </span><span class="NLM_article-title">Prospects for the next anti-Pseudomonas drug</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">558</span>– <span class="NLM_lpage">565</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2009.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2Fj.coph.2009.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=19748829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1WntLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=558-565&author=M.+G.+Pageauthor=J.+Heim&title=Prospects+for+the+next+anti-Pseudomonas+drug&doi=10.1016%2Fj.coph.2009.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Prospects for the next anti-Pseudomonas drug</span></div><div class="casAuthors">Page, Malcolm G. P.; Heim, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">558-565</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Pseudomonas aeruginosa is one of the most dreaded Gram-neg. bacterial pathogens in hospitals.  Not only it is among the most frequently isolated Gram-neg. organisms in bloodstream and wound infections, pneumonia, intra-abdominal-sepsis and urogenital-sepsis, but also it is frequently found in patients with comorbid illnesses and compromised by in-dwelling catheters, tubes and surgery where mortality rates of more than 60% have been reported.  Besides its intrinsic resistance to a no. of widely used antibiotics, Pseudomonas also managed to acquire resistance via addnl. mechanisms, including target mutations, increased expression of efflux pumps and of antibiotic-degrading enzymes.  Taken together, the increased incidence in certain types of infections, the increased use of invasive devices in the hospital as well as the increased frequency of multi-resistant Pseudomonas strains, have clearly led to a shortage of treatment options for nosocomial Pseudomonas infections.  Even the recommended combination therapy of an antibiotic of the β-lactam class together with an aminoglycoside or a fluoroquinolone, is no longer always successful and sometimes a polymyxin has to be given as last resort.  Despite growing concerns of clinicians and medical societies about the very limited no. of novel drugs in the pipeline to fight multi-resistant Pseudomonas strains, only a very small no. of novel anti-Pseudomonas drugs are currently in late stage of pre-clin. or clin. development.  However, and possibly as a reflection of the magnitude of the problem, quite a variety of approaches are being pursued.  Among these are next-generation analogs of successful antibiotic classes (e.g. novel β-lactams and combinations of novel β-lactamase inhibitors with known penicillins or cephalosporins), antibodies, phages and selective peptides.  It is to be hoped that a no. of these novel drugs will show clin. utility and reach the market over the next 6-10 years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxOCyGCdsTGbVg90H21EOLACvtfcHk0lhzHs4bw9j1rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1WntLnP&md5=d4022458c9a09f2c1f4a243ba7a7d61b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2009.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2009.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DPage%26aufirst%3DM.%2BG.%26aulast%3DHeim%26aufirst%3DJ.%26atitle%3DProspects%2520for%2520the%2520next%2520anti-Pseudomonas%2520drug%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2009%26volume%3D9%26spage%3D558%26epage%3D565%26doi%3D10.1016%2Fj.coph.2009.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boucher, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talbot, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheld, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spellberg, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, J.</span></span> <span> </span><span class="NLM_article-title">Bad Bugs, No drugs: no ESKAPE! An update from the Infectious Diseases Society of America</span>. <i>Clin. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.1086/595011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1086%2F595011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=19035777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BD1M%252Fit1Krsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=1-12&author=H.+W.+Boucherauthor=G.+H.+Talbotauthor=J.+S.+Bradleyauthor=J.+E.+Edwardsauthor=D.+Gilbertauthor=L.+B.+Riceauthor=M.+Scheldauthor=B.+Spellbergauthor=J.+Bartlett&title=Bad+Bugs%2C+No+drugs%3A+no+ESKAPE%21+An+update+from+the+Infectious+Diseases+Society+of+America&doi=10.1086%2F595011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Bad bugs, no drugs: no ESKAPE! An update from the Infectious Diseases Society of America</span></div><div class="casAuthors">Boucher Helen W; Talbot George H; Bradley John S; Edwards John E; Gilbert David; Rice Louis B; Scheld Michael; Spellberg Brad; Bartlett John</div><div class="citationInfo"><span class="NLM_cas:title">Clinical infectious diseases : an official publication of the Infectious Diseases Society of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-12</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The Infectious Diseases Society of America (IDSA) continues to view with concern the lean pipeline for novel therapeutics to treat drug-resistant infections, especially those caused by gram-negative pathogens.  Infections now occur that are resistant to all current antibacterial options.  Although the IDSA is encouraged by the prospect of success for some agents currently in preclinical development, there is an urgent, immediate need for new agents with activity against these panresistant organisms.  There is no evidence that this need will be met in the foreseeable future.  Furthermore, we remain concerned that the infrastructure for discovering and developing new antibacterials continues to stagnate, thereby risking the future pipeline of antibacterial drugs.  The IDSA proposed solutions in its 2004 policy report, "Bad Bugs, No Drugs: As Antibiotic R&D Stagnates, a Public Health Crisis Brews," and recently issued a "Call to Action" to provide an update on the scope of the problem and the proposed solutions.  A primary objective of these periodic reports is to encourage a community and legislative response to establish greater financial parity between the antimicrobial development and the development of other drugs.  Although recent actions of the Food and Drug Administration and the 110th US Congress present a glimmer of hope, significant uncertainly remains.  Now, more than ever, it is essential to create a robust and sustainable antibacterial research and development infrastructure--one that can respond to current antibacterial resistance now and anticipate evolving resistance.  This challenge requires that industry, academia, the National Institutes of Health, the Food and Drug Administration, the Centers for Disease Control and Prevention, the US Department of Defense, and the new Biomedical Advanced Research and Development Authority at the Department of Health and Human Services work productively together.  This report provides an update on potentially effective antibacterial drugs in the late-stage development pipeline, in the hope of encouraging such collaborative action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKtuWEUmAWleWhZTslLrfffW6udTcc2eZtlunOxs82Sbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M%252Fit1Krsg%253D%253D&md5=a3f71406b891f90f4f0f77b5f1ebc4f8</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1086%2F595011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F595011%26sid%3Dliteratum%253Aachs%26aulast%3DBoucher%26aufirst%3DH.%2BW.%26aulast%3DTalbot%26aufirst%3DG.%2BH.%26aulast%3DBradley%26aufirst%3DJ.%2BS.%26aulast%3DEdwards%26aufirst%3DJ.%2BE.%26aulast%3DGilbert%26aufirst%3DD.%26aulast%3DRice%26aufirst%3DL.%2BB.%26aulast%3DScheld%26aufirst%3DM.%26aulast%3DSpellberg%26aufirst%3DB.%26aulast%3DBartlett%26aufirst%3DJ.%26atitle%3DBad%2520Bugs%252C%2520No%2520drugs%253A%2520no%2520ESKAPE%2521%2520An%2520update%2520from%2520the%2520Infectious%2520Diseases%2520Society%2520of%2520America%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2009%26volume%3D48%26spage%3D1%26epage%3D12%26doi%3D10.1086%2F595011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Payne, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwynn, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompliano, D. L.</span></span> <span> </span><span class="NLM_article-title">Drugs for bad bugs: confronting the challenges of antibacterial discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1038/nrd2201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1038%2Fnrd2201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=17159923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=29-40&author=D.+J.+Payneauthor=M.+N.+Gwynnauthor=D.+J.+Holmesauthor=D.+L.+Pompliano&title=Drugs+for+bad+bugs%3A+confronting+the+challenges+of+antibacterial+discovery&doi=10.1038%2Fnrd2201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Drugs for bad bugs: confronting the challenges of antibacterial discovery</span></div><div class="casAuthors">Payne, David J.; Gwynn, Michael N.; Holmes, David J.; Pompliano, David L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-40</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The sequencing of the first complete bacterial genome in 1995 heralded a new era of hope for antibacterial drug discoverers, who now had the tools to search entire genomes for new antibacterial targets.  Several companies, including GlaxoSmithKline, moved back into the antibacterials area and embraced a genomics-derived, target-based approach to screen for new classes of drugs with novel modes of action.  Here, we share our experience of evaluating more than 300 genes and 70 high-throughput screening campaigns over a period of 7 years, and look at what we learned and how that has influenced GlaxoSmithKline's antibacterials strategy going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH1tM2BSZbWLVg90H21EOLACvtfcHk0livUQbfnUJNRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfM&md5=18100253f9868af578ea621ab98a5c86</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd2201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2201%26sid%3Dliteratum%253Aachs%26aulast%3DPayne%26aufirst%3DD.%2BJ.%26aulast%3DGwynn%26aufirst%3DM.%2BN.%26aulast%3DHolmes%26aufirst%3DD.%2BJ.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26atitle%3DDrugs%2520for%2520bad%2520bugs%253A%2520confronting%2520the%2520challenges%2520of%2520antibacterial%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D29%26epage%3D40%26doi%3D10.1038%2Fnrd2201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Talbot, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. E.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheld, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, J. G.</span></span> <span> </span><span class="NLM_article-title">Bad Bugs Need Drugs: An update on the development pipeline from the antimicrobial availability task force of the Infectious Diseases Society of America</span>. <i>Clin. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">657</span>– <span class="NLM_lpage">668</span>, <span class="refDoi"> DOI: 10.1086/499819</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1086%2F499819" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=16447111" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BD28%252FmtFGqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2006&pages=657-668&author=G.+H.+Talbotauthor=J.+Bradleyauthor=J.+E.+Edwardsauthor=D.+Gilbertauthor=M.+Scheldauthor=J.+G.+Bartlett&title=Bad+Bugs+Need+Drugs%3A+An+update+on+the+development+pipeline+from+the+antimicrobial+availability+task+force+of+the+Infectious+Diseases+Society+of+America&doi=10.1086%2F499819"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Bad bugs need drugs: an update on the development pipeline from the Antimicrobial Availability Task Force of the Infectious Diseases Society of America</span></div><div class="casAuthors">Talbot George H; Bradley John; Edwards John E Jr; Gilbert David; Scheld Michael; Bartlett John G</div><div class="citationInfo"><span class="NLM_cas:title">Clinical infectious diseases : an official publication of the Infectious Diseases Society of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">657-68</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The Antimicrobial Availability Task Force (AATF) of the Infectious Diseases Society of America (IDSA) has viewed with concern the decreasing investment by major pharmaceutical companies in antimicrobial research and development.  Although smaller companies are stepping forward to address this gap, their success is uncertain.  The IDSA proposed legislative and other federal solutions to this emerging public health problem in its July 2004 policy report "Bad Bugs, No Drugs: As Antibiotic R&D Stagnates, a Public Health Crisis Brews." At this time, the legislative response cannot be predicted.  To emphasize further the urgency of the problem for the benefit of legislators and policy makers and to capture the ongoing frustration our clinician colleagues experience in their frequent return to an inadequate medicine cabinet, the AATF has prepared this review to highlight pathogens that are frequently resistant to licensed antimicrobials and for which few, if any, potentially effective drugs are identifiable in the late-stage development pipeline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTcyYffQlmsoVQKX3ooHxWjfW6udTcc2eZg8JssTlOdybntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28%252FmtFGqsQ%253D%253D&md5=fd193c8654a73888c84ba22299c93cc8</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1086%2F499819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F499819%26sid%3Dliteratum%253Aachs%26aulast%3DTalbot%26aufirst%3DG.%2BH.%26aulast%3DBradley%26aufirst%3DJ.%26aulast%3DEdwards%26aufirst%3DJ.%2BE.%26aulast%3DGilbert%26aufirst%3DD.%26aulast%3DScheld%26aufirst%3DM.%26aulast%3DBartlett%26aufirst%3DJ.%2BG.%26atitle%3DBad%2520Bugs%2520Need%2520Drugs%253A%2520An%2520update%2520on%2520the%2520development%2520pipeline%2520from%2520the%2520antimicrobial%2520availability%2520task%2520force%2520of%2520the%2520Infectious%2520Diseases%2520Society%2520of%2520America%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2006%26volume%3D42%26spage%3D657%26epage%3D668%26doi%3D10.1086%2F499819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Overbye, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J.</span></span> <span> </span><span class="NLM_article-title">Antibiotics: where did we go wrong?</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1016/s1359-6446(04)03285-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2FS1359-6446%2804%2903285-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=15676298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FkvFOhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=45-52&author=K.+Overbyeauthor=J.+Barrett&title=Antibiotics%3A+where+did+we+go+wrong%3F&doi=10.1016%2Fs1359-6446%2804%2903285-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotics: where did we go wrong?</span></div><div class="casAuthors">Overbye Karen M; Barrett John F</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-52</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">In the late 1960s, the medical need for new antibiotics began to be questioned, and the pharmaceutical industry shifted its emphasis of antibacterials from that of a therapeutic leader to a low-priority research area.  Although infectious diseases, in particular those caused by bacterial infections, are still among the top causes of mortality in the world, industrial support continues to wane.  The shift from this important area of antimicrobial research has been attributed to a combination of science, medical, marketing and business reasons.  This decline in antibacterial drug discovery, coupled with increasing risk as a result of infections caused by drug-resistant bacterial pathogens, represents a clear public health threat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS--fo71XGN1X6CEoQgWY10fW6udTcc2eZg8JssTlOdybntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FkvFOhtA%253D%253D&md5=9d48b7cd2b497d8db69f0fae744f7baf</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903285-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903285-4%26sid%3Dliteratum%253Aachs%26aulast%3DOverbye%26aufirst%3DK.%26aulast%3DBarrett%26aufirst%3DJ.%26atitle%3DAntibiotics%253A%2520where%2520did%2520we%2520go%2520wrong%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D45%26epage%3D52%26doi%3D10.1016%2Fs1359-6446%2804%2903285-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monaghan, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. F.</span></span> <span> </span><span class="NLM_article-title">Antibacterial drug discovery-then, now and the genomics future</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">901</span>– <span class="NLM_lpage">909</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2005.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2Fj.bcp.2005.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=16494849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD28XitVKjurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2006&pages=901-909&author=R.+L.+Monaghanauthor=J.+F.+Barrett&title=Antibacterial+drug+discovery-then%2C+now+and+the+genomics+future&doi=10.1016%2Fj.bcp.2005.11.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial drug discovery-Then, now and the genomics future</span></div><div class="casAuthors">Monaghan, Richard L.; Barrett, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">901-909</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Drug discovery research in the area of infectious diseases, in particular that dealing with antibacterial/antibiotic susceptibility and resistance, is in a process of continuing evolution.  Steeped in the history of the highly successful intervention with chemotherapeutic agents to treat human infections, the emergence of drug-resistant pathogens worldwide presents a serious unmet medical need, if not a pending catastrophe.  Research in both academia and industry over the past 30 years using mol. biol., genetics and more recently - bacterial genomics - has assembled key enabling technologies to increase productivity and success rates in the discovery and development of novel antibacterial agents.  However genomics is not limited only to antibacterial target selection but provides the opportunity to further understand key interactions in the use of antibacterial compds. as therapeutic agents (such as resistance emergence, susceptibility, efflux, interactions between compd. and pathogen, etc.).  Genomics also offers the potential for insights into: bacterial niche adaptation, host susceptibility, treatment regimens, antibiotic resistance, pharmacokinetics (e.g., host metab. differences), safety and the microbial genesis of chronic diseases (e.g., gastric ulceration).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4sY_8A5Ry07Vg90H21EOLACvtfcHk0ljqKAwwjNiJpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XitVKjurg%253D&md5=a3c65f493cb5a82cff2f736931a56248</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2005.11.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2005.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DMonaghan%26aufirst%3DR.%2BL.%26aulast%3DBarrett%26aufirst%3DJ.%2BF.%26atitle%3DAntibacterial%2520drug%2520discovery-then%252C%2520now%2520and%2520the%2520genomics%2520future%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2006%26volume%3D71%26spage%3D901%26epage%3D909%26doi%3D10.1016%2Fj.bcp.2005.11.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, C.</span></span> <span> </span><span class="NLM_article-title">Where will new antibiotics come from?</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">65</span>– <span class="NLM_lpage">70</span>, <span class="refDoi"> DOI: 10.1038/nrmicro727</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1038%2Fnrmicro727" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=15040181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptFWlu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2003&pages=65-70&author=C.+Walsh&title=Where+will+new+antibiotics+come+from%3F&doi=10.1038%2Fnrmicro727"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Where will new antibiotics come from?</span></div><div class="casAuthors">Walsh, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-70</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There is a const. need for new antibacterial drugs owing to the inevitable development of resistance that follows the introduction of antibiotics to the clinic.  When a new class of antibiotic is introduced, it is effective at first, but will eventually select for survival of the small fraction of bacterial populations that have an intrinsic or acquired resistance mechanism.  Pathogens that are resistant to multiple drugs emerge around the globe, so how robust are antibiotic discovery processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWf-wNvjdudbVg90H21EOLACvtfcHk0ljqKAwwjNiJpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptFWlu7s%253D&md5=be8a897b57e796094c5b54652152d533</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro727&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro727%26sid%3Dliteratum%253Aachs%26aulast%3DWalsh%26aufirst%3DC.%26atitle%3DWhere%2520will%2520new%2520antibiotics%2520come%2520from%253F%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2003%26volume%3D1%26spage%3D65%26epage%3D70%26doi%3D10.1038%2Fnrmicro727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goss, R. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, K.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial nucleoside antibiotics targeting cell wall assembly: Recent advances in structure–function studies and nucleoside biosynthesis</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">279</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1039/b816215h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2FB816215H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=20111805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Sntb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2010&pages=279-304&author=M.+Winnauthor=R.+J.+M.+Gossauthor=K.-I.+Kimuraauthor=T.+D.+H.+Bugg&title=Antimicrobial+nucleoside+antibiotics+targeting+cell+wall+assembly%3A+Recent+advances+in+structure%E2%80%93function+studies+and+nucleoside+biosynthesis&doi=10.1039%2Fb816215h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial nucleoside antibiotics targeting cell wall assembly: Recent advances in structure-function studies and nucleoside biosynthesis</span></div><div class="casAuthors">Winn, Michael; Goss, Rebecca J. M.; Kimura, Ken-ichi; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">279-304</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The quest for new antibiotics, esp. those with activity against Gram-neg. bacteria, is urgent; however, very few new antibiotics have been marketed in the last 40 years, with this limited no. falling into only four new structural classes.  Several nucleoside natural product antibiotics target bacterial translocase MraY, involved in the lipid-linked cycle of peptidoglycan biosynthesis, and fungal chitin synthase.  Biosynthetic studies on the nikkomycin, caprazamycin and pacidamycin/mureidomycin families are also reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8Wne70VFcV7Vg90H21EOLACvtfcHk0ljJ4LC4S-i0hA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Sntb8%253D&md5=49a27386210ce2250334db085a7bfc67</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2FB816215H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FB816215H%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%26aulast%3DGoss%26aufirst%3DR.%2BJ.%2BM.%26aulast%3DKimura%26aufirst%3DK.-I.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DAntimicrobial%2520nucleoside%2520antibiotics%2520targeting%2520cell%2520wall%2520assembly%253A%2520Recent%2520advances%2520in%2520structure%25E2%2580%2593function%2520studies%2520and%2520nucleoside%2520biosynthesis%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2010%26volume%3D27%26spage%3D279%26epage%3D304%26doi%3D10.1039%2Fb816215h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, K.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Recent advances in antimicrobial nucleoside antibiotics targeting cell wall biosynthesis</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1039/b202149h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2Fb202149h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=12735700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslCqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2003&pages=252-273&author=K.-I.+Kimuraauthor=T.+D.+H.+Bugg&title=Recent+advances+in+antimicrobial+nucleoside+antibiotics+targeting+cell+wall+biosynthesis&doi=10.1039%2Fb202149h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in antimicrobial nucleoside antibiotics targeting cell wall biosynthesis</span></div><div class="casAuthors">Kimura, Ken-Ichi; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">252-273</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The biosynthesis of peptidoglycan is essential for all bacteria and has no counterpart in eukaryotic cells.  It is one of the prime targets for antibiotic chemotherapy, esp. phospho-MurNAc-pentapeptide translocase (translocase I) is a fascinating target in which there is no com. antibiotic.  In this review we will describe three nucleoside translocase I inhibitors, mureidomycin, tunicamycin and liposidomycin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrksNstyeT_iLVg90H21EOLACvtfcHk0ljJ4LC4S-i0hA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslCqtr4%253D&md5=8c6823d378a4754c368a6179b5bfa057</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1039%2Fb202149h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb202149h%26sid%3Dliteratum%253Aachs%26aulast%3DKimura%26aufirst%3DK.-I.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DRecent%2520advances%2520in%2520antimicrobial%2520nucleoside%2520antibiotics%2520targeting%2520cell%2520wall%2520biosynthesis%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2003%26volume%3D20%26spage%3D252%26epage%3D273%26doi%3D10.1039%2Fb202149h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enokita, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaida, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haneishi, T.</span></span> <span> </span><span class="NLM_article-title">Mureidomycins A-D, novel peptidylnucleoside antibiotics with spheroplast forming activity. I. Taxonomy, fermentation, isolation and physico-chemical properties</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">662</span>– <span class="NLM_lpage">666</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.42.662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.42.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=2664250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaL1MXkslCntrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1989&pages=662-666&author=M.+Inukaiauthor=F.+Isonoauthor=S.+Takahashiauthor=R.+Enokitaauthor=Y.+Sakaidaauthor=T.+Haneishi&title=Mureidomycins+A-D%2C+novel+peptidylnucleoside+antibiotics+with+spheroplast+forming+activity.+I.+Taxonomy%2C+fermentation%2C+isolation+and+physico-chemical+properties&doi=10.7164%2Fantibiotics.42.662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15aR"><div class="casContent"><span class="casTitleNuber">15a</span><div class="casTitle"><span class="NLM_cas:atitle">Mureidomycins A-D, novel peptidyl nucleoside antibiotics with spheroplast forming activity. I. Taxonomy, fermentation, isolation and physicochemical properties</span></div><div class="casAuthors">Inukai, Masatoshi; Isono, Fujio; Takahashi, Shuji; Enokita, Ryuzo; Sakaida, Yoshiharu; Haneishi, Tatsuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">662-6</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">A strain of actinomycetes identified as Streptomyces flavidovirens produced new antibiotics, mureidomycins (MRD's) A-D, specifically active against Pseudomonas aeruginosa.  They were isolated from the culture filtrate by successive column chromatogs. such as Amberlite XAD-2 and CG-50, Whatman DE-52 and Toyoperal HW-40.  They were amphoteric white powders, sol. in MeOH and H2O.  Their mol. wts. and mol. formula, in parentheses were 840 (C38H48N8O12S), 842 (C38H50N8O12S), 897 (C40H51N9O13S), and 899 (C40H53N9O13S), resp.  m-Tyrosine and two unknown substances were detected by amino acid analyses as their common constituents.  MRD's A and C contained uracil but MRD's B and D dihydrouracil instead of uracil.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrheyBZJOz2o7Vg90H21EOLACvtfcHk0ljJ4LC4S-i0hA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXkslCntrc%253D&md5=bb9351de6d81f2e5d84869fb209f0eac</span></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.42.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.42.662%26sid%3Dliteratum%253Aachs%26aulast%3DInukai%26aufirst%3DM.%26aulast%3DIsono%26aufirst%3DF.%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DEnokita%26aufirst%3DR.%26aulast%3DSakaida%26aufirst%3DY.%26aulast%3DHaneishi%26aufirst%3DT.%26atitle%3DMureidomycins%2520A-D%252C%2520novel%2520peptidylnucleoside%2520antibiotics%2520with%2520spheroplast%2520forming%2520activity.%2520I.%2520Taxonomy%252C%2520fermentation%252C%2520isolation%2520and%2520physico-chemical%2520properties%26jtitle%3DJ.%2520Antibiot.%26date%3D1989%26volume%3D42%26spage%3D662%26epage%3D666%26doi%3D10.7164%2Fantibiotics.42.662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit15b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Isono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katayama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haneishi, T.</span></span> <span> </span><span class="NLM_article-title">Mureidomycins A-D, novel peptidylnucleoside antibiotics with spheroplast forming activity. III. Biological properties</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">674</span>– <span class="NLM_lpage">679</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.42.674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.42.674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=2498275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaL1MXksFCjt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1989&pages=674-679&author=F.+Isonoauthor=T.+Katayamaauthor=M.+Inukaiauthor=T.+Haneishi&title=Mureidomycins+A-D%2C+novel+peptidylnucleoside+antibiotics+with+spheroplast+forming+activity.+III.+Biological+properties&doi=10.7164%2Fantibiotics.42.674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15bR"><div class="casContent"><span class="casTitleNuber">15b</span><div class="casTitle"><span class="NLM_cas:atitle">Mureidomycins A∼D, novel peptidylnucleoside antibiotics with spheroplast forming activity.  III.  Biological properties</span></div><div class="casAuthors">Isono, Fujio; Katayama, Toshiaki; Inukai, Masatoshi; Haneishi, Tatsuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">674-9</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Mureidomycins (MRDs) A-D were specifically active against Pseudomonas aeruginosa.  Among them, MRD C was most active, with MICs of 0.1-3.13 μg/mL against many strains of the target organism.  Its activity was comparable to that of cefoperazone, ceftazidime, and cefsulodin.  MRD C-resistant mutants of P. aeruginosa appeared spontaneously at a high frequency when cultured in the presence of the antibiotic.  No cross-resistance was obsd. with β-lactam antibiotics.  A rapid decrease of turbidity along with spheroplast formation and cell lysis was obsd. when cells of P. aeruginosa were grown in the presence of MRD C.  The compds. exhibited low toxicity and protected mice from exptl. infection with P. aeruginosa.  The urinary and fecal recoveries of MRD C given s.c. were 5 and 18%, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn4ZvK5VmHgrVg90H21EOLACvtfcHk0lh22mzI0eA8-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXksFCjt7o%253D&md5=eb690f68ff7f669c6bdef656625ffb9d</span></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.42.674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.42.674%26sid%3Dliteratum%253Aachs%26aulast%3DIsono%26aufirst%3DF.%26aulast%3DKatayama%26aufirst%3DT.%26aulast%3DInukai%26aufirst%3DM.%26aulast%3DHaneishi%26aufirst%3DT.%26atitle%3DMureidomycins%2520A-D%252C%2520novel%2520peptidylnucleoside%2520antibiotics%2520with%2520spheroplast%2520forming%2520activity.%2520III.%2520Biological%2520properties%26jtitle%3DJ.%2520Antibiot.%26date%3D1989%26volume%3D42%26spage%3D674%26epage%3D679%26doi%3D10.7164%2Fantibiotics.42.674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit15c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Isono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haneishi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinoshita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuwano, H.</span></span> <span> </span><span class="NLM_article-title">Mureidomycins A-D, novel peptidylnucleoside antibiotics with spheroplast forming activity. II. Structural elucidation</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">673</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.42.667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.42.667" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=2498274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaL1MXltlertbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1989&pages=667-673&author=F.+Isonoauthor=M.+Inukaiauthor=S.+Takahashiauthor=T.+Haneishiauthor=T.+Kinoshitaauthor=H.+Kuwano&title=Mureidomycins+A-D%2C+novel+peptidylnucleoside+antibiotics+with+spheroplast+forming+activity.+II.+Structural+elucidation&doi=10.7164%2Fantibiotics.42.667"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15cR"><div class="casContent"><span class="casTitleNuber">15c</span><div class="casTitle"><span class="NLM_cas:atitle">Mureidomycins A-D, novel peptidyl nucleoside antibiotics with spheroplast forming activity. II. Structural elucidation</span></div><div class="casAuthors">Isono, Fujio; Inukai, Masatoshi; Takahashi, Shuji; Haneishi, Tatsuo; Kinoshita, Takeshi; Kuwano, Harumitsu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">667-73</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Structures of new antibiotics, mureidomycins (MRD's) A-D (I-IV), were deduced from spectroscopic analyses and degrdn. studies.  Two residues of m-tyrosine, one residue of 2-amino-3-N-methylaminobutyric acid (AMBA) and methionine are present in all components of the complex.  Uracil is contained in MRD's A and C, while dihydrouracil in MRD's B and D.  Methionine and m-tyrosine are connected through an ureido bond,and uracil or dihydrouracil is linked to AMBA via an enamine sugar moiety.  In addn., MRD's C and D contain a glycine residue at the N-terminal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh5gV1ov1yurVg90H21EOLACvtfcHk0lh22mzI0eA8-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXltlertbw%253D&md5=ac87a58b5e06562179cdacca9f183013</span></div><a href="/servlet/linkout?suffix=cit15c&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.42.667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.42.667%26sid%3Dliteratum%253Aachs%26aulast%3DIsono%26aufirst%3DF.%26aulast%3DInukai%26aufirst%3DM.%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DHaneishi%26aufirst%3DT.%26aulast%3DKinoshita%26aufirst%3DT.%26aulast%3DKuwano%26aufirst%3DH.%26atitle%3DMureidomycins%2520A-D%252C%2520novel%2520peptidylnucleoside%2520antibiotics%2520with%2520spheroplast%2520forming%2520activity.%2520II.%2520Structural%2520elucidation%26jtitle%3DJ.%2520Antibiot.%26date%3D1989%26volume%3D42%26spage%3D667%26epage%3D673%26doi%3D10.7164%2Fantibiotics.42.667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit15d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Isono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaida, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinoshita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span> <span> </span><span class="NLM_article-title">Mureidomycins E and F, minor components of mureidomycins</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1203</span>– <span class="NLM_lpage">1207</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.46.1203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.46.1203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=8407581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK2cXjtVGrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=1993&pages=1203-1207&author=F.+Isonoauthor=Y.+Sakaidaauthor=S.+Takahashiauthor=T.+Kinoshitaauthor=T.+Nakamuraauthor=M.+Inukai&title=Mureidomycins+E+and+F%2C+minor+components+of+mureidomycins&doi=10.7164%2Fantibiotics.46.1203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15dR"><div class="casContent"><span class="casTitleNuber">15d</span><div class="casTitle"><span class="NLM_cas:atitle">Mureidomycins E and F, minor components of mureidomycins</span></div><div class="casAuthors">Isono, Fujio; Sakaida, Yoshiharu; Takahashi, Shuji; Kinoshita, Takeshi; Nakamura, Takemichi; Inukai, Masatoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1203-7</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Mureidomycins (MRDs) E and F I (R1 = OH, R2 = H; R1 = H, R2 = OH, resp.) were isolated from a culture filtrate of Streptomyces flavidovirens SNAK 60486 which produces MRDs A-D.  They possessed the same mol. formulas, C39H48N8O12S and very similar UV, IR and NMR spectra, but differed clearly from each other in HPLC profile.  In the hydrolyzates of MRDs E and F, 8- and 6-hydroxy-1,2,3,4-tetrahydro-3-isoquinolinecarboxylic acid were detected, resp., which were not detected in those of MRDs A-D.  They showed strong anti-pseudomonal activity but less active than MRD A.  MRDs E and F were synthesized from MRD A and formaldehyde through Pictet-Spengler reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiih125mLEILVg90H21EOLACvtfcHk0ljkGvAilWYhWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXjtVGrtg%253D%253D&md5=621fee20a2786169666e17016d2b1a12</span></div><a href="/servlet/linkout?suffix=cit15d&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.46.1203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.46.1203%26sid%3Dliteratum%253Aachs%26aulast%3DIsono%26aufirst%3DF.%26aulast%3DSakaida%26aufirst%3DY.%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DKinoshita%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DInukai%26aufirst%3DM.%26atitle%3DMureidomycins%2520E%2520and%2520F%252C%2520minor%2520components%2520of%2520mureidomycins%26jtitle%3DJ.%2520Antibiot.%26date%3D1993%26volume%3D46%26spage%3D1203%26epage%3D1207%26doi%3D10.7164%2Fantibiotics.46.1203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadkarni, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijayakumar, E. K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangul, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fehlhaber, H.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertesy, L.</span></span> <span> </span><span class="NLM_article-title">Napsamycins, new Pseudomonas active antibiotics of the mureidomycin family from Streptomyces sp. HIL Y-82, 11372</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">595</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.47.595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.47.595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=8072167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK2cXlt12msrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1994&pages=595-598&author=S.+Chatterjeeauthor=S.+R.+Nadkarniauthor=E.+K.+S.+Vijayakumarauthor=M.+V.+Patelauthor=B.+N.+Gangulauthor=H.-W.+Fehlhaberauthor=L.+Vertesy&title=Napsamycins%2C+new+Pseudomonas+active+antibiotics+of+the+mureidomycin+family+from+Streptomyces+sp.+HIL+Y-82%2C+11372&doi=10.7164%2Fantibiotics.47.595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Napsamycins, new Pseudomonas active antibiotics of the mureidomycin family from Streptomyces sp. HIL Y-82,11372</span></div><div class="casAuthors">Chatterjee, Sugata; Nadkarni, S. R.; Vijayakumar, E. K. S.; Patel, M. V.; Ganguli, B. N.; Fehlhaber, H. W.; Vertesy, L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">595-8</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Four antibiotics specifically active against Pseudomonas were discovered in a screening program.  These antibiotics were named napsamycins A (I), B (II), C (III), and D (IV).  The fermn., isolation, structural detn. and biol. activity of these antibiotics are reported here.  The napsamycins are peptidyl nucleosides which are specific for Pseudomonas inhibition; poor activity is obsd. against other gram-neg. and -pos. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTLo0fJgIHNbVg90H21EOLACvtfcHk0ljkGvAilWYhWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlt12msrg%253D&md5=872ed505e07fd46af4cb149647ddd72c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.47.595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.47.595%26sid%3Dliteratum%253Aachs%26aulast%3DChatterjee%26aufirst%3DS.%26aulast%3DNadkarni%26aufirst%3DS.%2BR.%26aulast%3DVijayakumar%26aufirst%3DE.%2BK.%2BS.%26aulast%3DPatel%26aufirst%3DM.%2BV.%26aulast%3DGangul%26aufirst%3DB.%2BN.%26aulast%3DFehlhaber%26aufirst%3DH.-W.%26aulast%3DVertesy%26aufirst%3DL.%26atitle%3DNapsamycins%252C%2520new%2520Pseudomonas%2520active%2520antibiotics%2520of%2520the%2520mureidomycin%2520family%2520from%2520Streptomyces%2520sp.%2520HIL%2520Y-82%252C%252011372%26jtitle%3DJ.%2520Antibiot.%26date%3D1994%26volume%3D47%26spage%3D595%26epage%3D598%26doi%3D10.7164%2Fantibiotics.47.595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Karwowski, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theriault, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barlow, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maus, M. L.</span></span> <span> </span><span class="NLM_article-title">Pacidamycins, a novel series of antibiotics with anti-<i>Pseudomonas aeruginosa</i> activity. I. Taxonomy of the producing organism and fermentation</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">506</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.42.506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.42.506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=2498263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaL1MXkt1aisro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1989&pages=506-511&author=J.+P.+Karwowskiauthor=M.+Jacksonauthor=R.+J.+Theriaultauthor=R.+H.+Chenauthor=G.+J.+Barlowauthor=M.+L.+Maus&title=Pacidamycins%2C+a+novel+series+of+antibiotics+with+anti-Pseudomonas+aeruginosa+activity.+I.+Taxonomy+of+the+producing+organism+and+fermentation&doi=10.7164%2Fantibiotics.42.506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17aR"><div class="casContent"><span class="casTitleNuber">17a</span><div class="casTitle"><span class="NLM_cas:atitle">Pacidamycins, a novel series of antibiotics with anti-Pseudomonas aeruginosa activity.  I.  Taxonomy of the producing organism and fermentation</span></div><div class="casAuthors">Karwowski, J. P.; Jackson, M.; Theriault, R. J.; Chen, R. H.; Barlow, G. J.; Maus, M. L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">506-11</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">The pacidamycins are a new complex of nucleosidyl peptide antibiotics with highly specific activity against P. aeruginosa.  They are produced by Streptomyces coeruleorubidus AB 1183F-64 which was isolated from a soil sample collected at Offenburg in the FRG.  The mature spore masses of the producing organism are greenish gray to blue, and the spore chains are arranged in spirals.  After the structures of the pacidamycins were detd., the fermn. medium was supplemented with component amino acids.  This resulted in the directed biosynthesis of several members of the complex.  The overall antibiotic recovered was increased from 1∼2 mg/L to >100 mg/L through a combination of strain selection, medium manipulation, and amino acid feeding expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9SL4rQa2e67Vg90H21EOLACvtfcHk0ljkGvAilWYhWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXkt1aisro%253D&md5=c86dc18e58c0d6f6660ee40f7e8ca9af</span></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.42.506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.42.506%26sid%3Dliteratum%253Aachs%26aulast%3DKarwowski%26aufirst%3DJ.%2BP.%26aulast%3DJackson%26aufirst%3DM.%26aulast%3DTheriault%26aufirst%3DR.%2BJ.%26aulast%3DChen%26aufirst%3DR.%2BH.%26aulast%3DBarlow%26aufirst%3DG.%2BJ.%26aulast%3DMaus%26aufirst%3DM.%2BL.%26atitle%3DPacidamycins%252C%2520a%2520novel%2520series%2520of%2520antibiotics%2520with%2520anti-Pseudomonas%2520aeruginosa%2520activity.%2520I.%2520Taxonomy%2520of%2520the%2520producing%2520organism%2520and%2520fermentation%26jtitle%3DJ.%2520Antibiot.%26date%3D1989%26volume%3D42%26spage%3D506%26epage%3D511%26doi%3D10.7164%2Fantibiotics.42.506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit17b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buko, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittern, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAlpine, J. B.</span></span> <span> </span><span class="NLM_article-title">Pacidamycins, a novel series of antibiotics with anti-<i>Pseudomonas aeruginosa</i> activity. II. Isolation and structural elucidation</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">512</span>– <span class="NLM_lpage">520</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.42.512</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.42.512" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=2526259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaL1MXlsFSiurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1989&pages=512-520&author=R.+H.+Chenauthor=A.+M.+Bukoauthor=D.+N.+Whitternauthor=J.+B.+McAlpine&title=Pacidamycins%2C+a+novel+series+of+antibiotics+with+anti-Pseudomonas+aeruginosa+activity.+II.+Isolation+and+structural+elucidation&doi=10.7164%2Fantibiotics.42.512"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17bR"><div class="casContent"><span class="casTitleNuber">17b</span><div class="casTitle"><span class="NLM_cas:atitle">Pacidamycins, a novel series of antibiotics with anti-Pseudomonas aeruginosa activity.  II.  Isolation and structural elucidation</span></div><div class="casAuthors">Chen, Randal H.; Buko, Alexander M.; Whittern, David N.; McAlpine, James B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">512-20</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">A novel class of antibiotics (named pacidamycins) was isolated from cultures of Streptomyces coeruleorubidus strain AB 1183F-64.  The antimicrobial activity of the pacidamycins 1-7 (I-VIII) were selective against Pseudomonas aeruginosa.  The various congeners are nucleoside peptides which differ in the terminal amino acid residues.  The structures were detd. using MS-MS and 2D NMR techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1PlXMET3hurVg90H21EOLACvtfcHk0lhmxVXQEmbXqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXlsFSiurk%253D&md5=dac46a68eceaff4fdc7c451f8fde1da2</span></div><a href="/servlet/linkout?suffix=cit17b&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.42.512&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.42.512%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DR.%2BH.%26aulast%3DBuko%26aufirst%3DA.%2BM.%26aulast%3DWhittern%26aufirst%3DD.%2BN.%26aulast%3DMcAlpine%26aufirst%3DJ.%2BB.%26atitle%3DPacidamycins%252C%2520a%2520novel%2520series%2520of%2520antibiotics%2520with%2520anti-Pseudomonas%2520aeruginosa%2520activity.%2520II.%2520Isolation%2520and%2520structural%2520elucidation%26jtitle%3DJ.%2520Antibiot.%26date%3D1989%26volume%3D42%26spage%3D512%26epage%3D520%26doi%3D10.7164%2Fantibiotics.42.512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit17c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R. H.</span></span> <span> </span><span class="NLM_article-title">Pacidamycins, a novel series of antibiotics with anti-<i>Pseudomonas aeruginosa</i> activity. III. Microbiologic profile</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">526</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.42.521</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.42.521" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=2498265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaL1MXlsFSnsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1989&pages=521-526&author=P.+B.+Fernandesauthor=R.+N.+Swansonauthor=D.+J.+Hardyauthor=C.+W.+Hansonauthor=L.+Coenauthor=R.+R.+Rasmussenauthor=R.+H.+Chen&title=Pacidamycins%2C+a+novel+series+of+antibiotics+with+anti-Pseudomonas+aeruginosa+activity.+III.+Microbiologic+profile&doi=10.7164%2Fantibiotics.42.521"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17cR"><div class="casContent"><span class="casTitleNuber">17c</span><div class="casTitle"><span class="NLM_cas:atitle">Pacidamycins, a novel series of antibiotics with anti-Pseudomonas aeruginosa activity.  III.  Microbiologic profile</span></div><div class="casAuthors">Fernandes, Prabhavathi B.; Swanson, Robert N.; Hardy, Dwight J.; Hanson, Charles W.; Coen, Lyle; Rasmussen, Ronald R.; Chen, Randal H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">521-6</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Pacidamycins are nucleosidyl-peptide antibiotics which have activity only against Pseudomonas aeruginosa.  Their MICs for other organisms such as Enterobacteriaceae, Staphylococcus aereus, most Streptococci and other Pseudomonas species are >100 μg/mL.  These compds. had no activity against erythromycin-susceptible Streptococci.  The MICs for Streptococcus pyogenes with constitutive- and inducible-type of macrolide-lincosamide-streptogramin resistance were 12.5 and 25 μg/mL, resp.  The MICs against P. aeruginosa ranged from 8 to 64 μg/mL.  The activity of these compds. was 1-2-fold less in serum than broth.  Time-kill curves were performed using 4 and 8 times the MIC of pacidamycin 1.  It was bactericidal against P. aeruginosa (3 log10 decrease in 4-6 h).  At 24 h, resistant mutants were found in the cultures.  The MICs of piperacillin and gentamicin for these mutants were the same as for the parent strain.  The frequency of resistance to these compds. was <3.5 × 10-6.  The resistant mutants were stable after 10 transfers in antibiotic-free medium.  The pacidamycins were inactive against P. aeruginosa in mouse protection tests.  After a single s.c. injection of 25 mg/kg of pacidamycin 1, the Cmax (peak serum concn.) was approx. 50 μg/mL and the serum half-life was 0.5 h.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnLwV7-wQ7PLVg90H21EOLACvtfcHk0lhmxVXQEmbXqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXlsFSnsLg%253D&md5=f0e403c9a17619b4650b7b7c603d94d4</span></div><a href="/servlet/linkout?suffix=cit17c&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.42.521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.42.521%26sid%3Dliteratum%253Aachs%26aulast%3DFernandes%26aufirst%3DP.%2BB.%26aulast%3DSwanson%26aufirst%3DR.%2BN.%26aulast%3DHardy%26aufirst%3DD.%2BJ.%26aulast%3DHanson%26aufirst%3DC.%2BW.%26aulast%3DCoen%26aufirst%3DL.%26aulast%3DRasmussen%26aufirst%3DR.%2BR.%26aulast%3DChen%26aufirst%3DR.%2BH.%26atitle%3DPacidamycins%252C%2520a%2520novel%2520series%2520of%2520antibiotics%2520with%2520anti-Pseudomonas%2520aeruginosa%2520activity.%2520III.%2520Microbiologic%2520profile%26jtitle%3DJ.%2520Antibiot.%26date%3D1989%26volume%3D42%26spage%3D521%26epage%3D526%26doi%3D10.7164%2Fantibiotics.42.521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit17d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Fronko, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galazzo, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberland, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malouin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. D.</span></span> <span> </span><span class="NLM_article-title">New pacidamycins produced by <i>Streptomyces coeruleorubidus</i>, NRRL 18370</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1405</span>– <span class="NLM_lpage">1410</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.53.1405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.53.1405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=11217807" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3MXosFGh" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2000&pages=1405-1410&author=R.+M.+Fronkoauthor=J.+C.+Leeauthor=J.+G.+Galazzoauthor=S.+Chamberlandauthor=F.+Malouinauthor=M.+D.+Lee&title=New+pacidamycins+produced+by+Streptomyces+coeruleorubidus%2C+NRRL+18370&doi=10.7164%2Fantibiotics.53.1405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17dR"><div class="casContent"><span class="casTitleNuber">17d</span><div class="casTitle"><span class="NLM_cas:atitle">New pacidamycins produced by Streptomyces coeruleorubidus, NRRL 18370</span></div><div class="casAuthors">Fronko, R. M.; Lee, J. C.; Galazzo, J. G.; Chamberland, S.; Malouin, F.; Lee, M. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1405-1410</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Japan Antibiotics Research Association</span>)
        </div><div class="casAbstract">Three new pacidamycins were isolated from a fermn. of Streptomyces coeruleorubidus.  The new antibiotics, pacidamycin -D, -4N, and -5T along with pacidamycins -3, -4 and -1 were isolated from fermn. broth by anion exchange chromatog. followed by cation exchange chromatog.  The structures, mol. formulas and mol. wts. of the new pacidamycins were detd. as well as their antibacterial activities.  It was found that all three new pacidamycins were active against Pseudomonas aeruginosa, but not active against gram positives and other gram neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpklF9U0J7M2bVg90H21EOLACvtfcHk0ljk2TXJjw_z3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXosFGh&md5=72098a6c1b4fb6a30687f1c0a853a62d</span></div><a href="/servlet/linkout?suffix=cit17d&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.53.1405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.53.1405%26sid%3Dliteratum%253Aachs%26aulast%3DFronko%26aufirst%3DR.%2BM.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DGalazzo%26aufirst%3DJ.%2BG.%26aulast%3DChamberland%26aufirst%3DS.%26aulast%3DMalouin%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DM.%2BD.%26atitle%3DNew%2520pacidamycins%2520produced%2520by%2520Streptomyces%2520coeruleorubidus%252C%2520NRRL%252018370%26jtitle%3DJ.%2520Antibiot.%26date%3D2000%26volume%3D53%26spage%3D1405%26epage%3D1410%26doi%3D10.7164%2Fantibiotics.53.1405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span> <span> </span><span class="NLM_article-title">A new nucleosidyl-peptide antibiotic, sansanmycin</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">158</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1038/ja.2007.16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1038%2Fja.2007.16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=17420567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjt1GktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2007&pages=158-161&author=Y.+Xieauthor=R.+Chenauthor=S.+Siauthor=C.+Sunauthor=H.+Xu&title=A+new+nucleosidyl-peptide+antibiotic%2C+sansanmycin&doi=10.1038%2Fja.2007.16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18aR"><div class="casContent"><span class="casTitleNuber">18a</span><div class="casTitle"><span class="NLM_cas:atitle">A new nucleosidyl-peptide antibiotic, sansanmycin</span></div><div class="casAuthors">Xie, Yunying; Chen, Ruxian; Si, Shuyi; Sun, Chenghang; Xu, Hongzhang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">158-161</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Japan Antibiotics Research Association</span>)
        </div><div class="casAbstract">A new nucleosidyl-peptide antibiotic, sansanmycin, was isolated from an unidentified Streptomyces sp SS.  The structure of sansanmycin was elucidated by analyses of its alk. hydrolyzate and spectroscopic analyses.  Sansanmycin exhibits antibacterial activity against Mycobacterium tuberculosis H37Ra and Pseudomonas aeruginosa with MIC values of 10 and 12.5 μg/mL, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvkPLMVwy1aLVg90H21EOLACvtfcHk0ljk2TXJjw_z3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjt1GktLs%253D&md5=2f5b20c12e958a2641ac4a71c205331f</span></div><a href="/servlet/linkout?suffix=cit18a&amp;dbid=16384&amp;doi=10.1038%2Fja.2007.16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fja.2007.16%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DSi%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DH.%26atitle%3DA%2520new%2520nucleosidyl-peptide%2520antibiotic%252C%2520sansanmycin%26jtitle%3DJ.%2520Antibiot.%26date%3D2007%26volume%3D60%26spage%3D158%26epage%3D161%26doi%3D10.1038%2Fja.2007.16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit18b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span> <span> </span><span class="NLM_article-title">Sansanmycins B and C, new components of sansanmycins</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1038/ja.2008.34</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1038%2Fja.2008.34" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=18503203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmt1Sms7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2008&pages=237-240&author=Y.+Xieauthor=H.+Xuauthor=S.+Siauthor=C.+Sunauthor=R.+Chen&title=Sansanmycins+B+and+C%2C+new+components+of+sansanmycins&doi=10.1038%2Fja.2008.34"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18bR"><div class="casContent"><span class="casTitleNuber">18b</span><div class="casTitle"><span class="NLM_cas:atitle">Sansanmycins B and C, new components of sansanmycins</span></div><div class="casAuthors">Xie, Yunying; Xu, Hongzhang; Si, Shuyi; Sun, Chenghang; Chen, Ruxian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">237-240</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Japan Antibiotics Research Association</span>)
        </div><div class="casAbstract">Two new antibiotics, sansanmycins B and C, were isolated from Streptomyces sp. SS.  Their structures were elucidated by extensive 1D and 2D NMR and MS spectral analyses.  Sansanmycins B and C exhibited inhibitory activity against our test strain of Pseudomonas aeruginosa with MIC values of 8.0 and 16 μg/mL, resp.  Sansanmycin B also exhibited inhibitory activity against Mycobacterium tuberculosis H37Rv and multi-drug-resistant Mycobacterium tuberculosis, with the MIC values ranging from 8.0 to 20 μg/mL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreqWRRIRHh7rVg90H21EOLACvtfcHk0ljk2TXJjw_z3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmt1Sms7Y%253D&md5=c4ceb4a51cc62eabc39926591086b750</span></div><a href="/servlet/linkout?suffix=cit18b&amp;dbid=16384&amp;doi=10.1038%2Fja.2008.34&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fja.2008.34%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DSi%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DR.%26atitle%3DSansanmycins%2520B%2520and%2520C%252C%2520new%2520components%2520of%2520sansanmycins%26jtitle%3DJ.%2520Antibiot.%26date%3D2008%26volume%3D61%26spage%3D237%26epage%3D240%26doi%3D10.1038%2Fja.2008.34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Isono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span> <span> </span><span class="NLM_article-title">Mureidomycin A, a new inhibitor of bacterial peptidoglycan synthesis</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">236</span>, <span class="refDoi"> DOI: 10.1128/aac.35.2.234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1128%2FAAC.35.2.234" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1902646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK3MXhs1SmurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1991&pages=234-236&author=F.+Isonoauthor=M.+Inukai&title=Mureidomycin+A%2C+a+new+inhibitor+of+bacterial+peptidoglycan+synthesis&doi=10.1128%2Faac.35.2.234"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Mureidomycin A, a new inhibitor of bacterial peptidoglycan synthesis</span></div><div class="casAuthors">Isono, Fujio; Inukai, Masatoshi</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">234-6</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    </div><div class="casAbstract">Mureidomycin A (MRD), a novel peptidylnucleoside antibiotic with antipseudomonal activity, inhibited not only peptidoglycan synthesis but also lipid-intermediate formation from UDP-N-acetylmuramyl (MurNAc)-pentapeptide and UDP-N-acetylglucosamine in an in vitro peptidoglycan-synthesizing system, using ether-treated cells of Pseudomonas aeruginosa.  Both types of inhibition by MRD disappeared when UDP-MurNAc-pentapeptide was preincubated with ether-treated cells.  Moreover, MRD completely inhibited lipid-intermediate I (undecaprenyl-p-p-MurNAc-pentapeptide) formation at a concn. below the MIC.  Apparently, the real target of MRD's action was translocase, which catalyzes lipid-intermediate I formation from UDP-MurNAc-pentapeptide and a lipid carrier.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnzLcZpNOh8LVg90H21EOLACvtfcHk0lgJNwsz25hH1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXhs1SmurY%253D&md5=b26e05bb4b6f1f83cd3acbd09547f6bc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FAAC.35.2.234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.35.2.234%26sid%3Dliteratum%253Aachs%26aulast%3DIsono%26aufirst%3DF.%26aulast%3DInukai%26aufirst%3DM.%26atitle%3DMureidomycin%2520A%252C%2520a%2520new%2520inhibitor%2520of%2520bacterial%2520peptidoglycan%2520synthesis%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1991%26volume%3D35%26spage%3D234%26epage%3D236%26doi%3D10.1128%2Faac.35.2.234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takatsuki, A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of the bacterial translocase reaction in peptidoglycan synthesis by mureidomycins</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">980</span>– <span class="NLM_lpage">983</span>, <span class="refDoi"> DOI: 10.1128/aac.37.5.980</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1128%2FAAC.37.5.980" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=8517724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK3sXkt1Sjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1993&pages=980-983&author=M.+Inukaiauthor=F.+Isonoauthor=A.+Takatsuki&title=Selective+inhibition+of+the+bacterial+translocase+reaction+in+peptidoglycan+synthesis+by+mureidomycins&doi=10.1128%2Faac.37.5.980"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of the bacterial translocase reaction in peptidoglycan synthesis by mureidomycins</span></div><div class="casAuthors">Inukai, Masatoshi; Isono, Fujio; Takatsuki, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">980-3</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    </div><div class="casAbstract">Mureidomycins (MRDs) A and C inhibited strongly the formation of undecaprenyl pyrophosphoryl N-acetylmuramyl-pentapeptide (lipid intermediate I), which is an intermediate in bacterial peptidoglycan synthesis (50% inhibitory concn. [IC50] of MRD A, 0.05 μg/mL).  However, they did not inhibit the formation of dolichyl pyrophosphoryl N-acetylglucosamine (Dol-p-p-GlcNAc), dolichyl phosphoryl glucose, or Bacillus subtilis of lipid-linked N-acetylglucosamine for teichoic acid synthesis (IC40s, > 100 μg/mL).  In contrast, tunicamycin (TCM) inhibited strongly the formation of Dol-p-p-GlcNAc (IC50, 0.03 μg/mL) but inhibited weakly the formation of bacterial lipid intermediate I (IC50, 44 μg/mL).  When the effects of MRDs A and C and TCM on the growth of BALB/3T3 cells at 10 μg/mL.  On the basis of these results, it was concluded that MRDs are the first specific and potent inhibitors of the translocase reaction in bacterial peptidoglycan synthesis, showing a high level of toxicity against bacteria and a low level of toxicity against mammalian cells.  A specific inhibitor of translocase could be a potent antibiotic with highly selective toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZQqVu8KkaC7Vg90H21EOLACvtfcHk0lgJNwsz25hH1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkt1Sjs7c%253D&md5=f299a807a59e67dc9560c7a82a86b63f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1128%2FAAC.37.5.980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.37.5.980%26sid%3Dliteratum%253Aachs%26aulast%3DInukai%26aufirst%3DM.%26aulast%3DIsono%26aufirst%3DF.%26aulast%3DTakatsuki%26aufirst%3DA.%26atitle%3DSelective%2520inhibition%2520of%2520the%2520bacterial%2520translocase%2520reaction%2520in%2520peptidoglycan%2520synthesis%2520by%2520mureidomycins%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1993%26volume%3D37%26spage%3D980%26epage%3D983%26doi%3D10.1128%2Faac.37.5.980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, D.</span></span> <span> </span><span class="NLM_article-title">Phospho-MurNAc-pentapeptide translocase (MraY) as a target for antibacterial agents and antibacterial proteins</span>. <i>Infect. Disord.: Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.2174/187152606784112128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.2174%2F187152606784112128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=16789873" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtFers7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=85-106&author=T.+Buggauthor=A.+Lloydauthor=D.+Roper&title=Phospho-MurNAc-pentapeptide+translocase+%28MraY%29+as+a+target+for+antibacterial+agents+and+antibacterial+proteins&doi=10.2174%2F187152606784112128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21aR"><div class="casContent"><span class="casTitleNuber">21a</span><div class="casTitle"><span class="NLM_cas:atitle">Phospho-MurNAc-pentapeptide translocase (MraY) as a target for antibacterial agents and antibacterial proteins</span></div><div class="casAuthors">Bugg, Timothy D. H.; Lloyd, Adrian J.; Roper, David I.</div><div class="citationInfo"><span class="NLM_cas:title">Infectious Disorders: Drug Targets</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-106</span>CODEN:
                <span class="NLM_cas:coden">IDDTAD</span>;
        ISSN:<span class="NLM_cas:issn">1871-5265</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Phospho-MurNAc-pentapeptide translocase (MraY, translocase I) catalyzes the first step of the lipid-linked cycle of reactions of bacterial peptidoglycan biosynthesis.  MraY is the target for five families of nucleoside antibacterial natural products: the tunicamycins, the mureidomycins (also pacidamycins, napsamycins), the liposidomycins, the muraymycins, and the capuramycins.  Recent structure-activity studies on these families have led to the identification of active pharmacophores, and insight into their mechanisms of action.  This step of peptidoglycan biosynthesis is also the target for the bacteriolytic E protein from bacteriophage .vphi.X174, and for cyclic peptides of the amphomycin family which complex the undecaprenyl phosphate cosubstrate.  The mechanisms of enzyme inhibition by these agents are discussed, and the state of knowledge regarding the transmembrane structure, active site, and catalytic mechanism of MraY.  The availability of high throughput assays and prospects of MraY as an antibacterial target are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA2V6DpkrljrVg90H21EOLACvtfcHk0lgigTXApbuQzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtFers7Y%253D&md5=b209c55dbd91c009a292e861eebecf44</span></div><a href="/servlet/linkout?suffix=cit21a&amp;dbid=16384&amp;doi=10.2174%2F187152606784112128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F187152606784112128%26sid%3Dliteratum%253Aachs%26aulast%3DBugg%26aufirst%3DT.%26aulast%3DLloyd%26aufirst%3DA.%26aulast%3DRoper%26aufirst%3DD.%26atitle%3DPhospho-MurNAc-pentapeptide%2520translocase%2520%2528MraY%2529%2520as%2520a%2520target%2520for%2520antibacterial%2520agents%2520and%2520antibacterial%2520proteins%26jtitle%3DInfect.%2520Disord.%253A%2520Drug%2520Targets%26date%3D2006%26volume%3D6%26spage%3D85%26epage%3D106%26doi%3D10.2174%2F187152606784112128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit21b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, K.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Recent advances in antimicrobial nucleoside antibiotics targeting cell wall biosynthesis</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1039/b202149h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2Fb202149h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=12735700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslCqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2003&pages=252-273&author=K.-i.+Kimuraauthor=T.+D.+H.+Bugg&title=Recent+advances+in+antimicrobial+nucleoside+antibiotics+targeting+cell+wall+biosynthesis&doi=10.1039%2Fb202149h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21bR"><div class="casContent"><span class="casTitleNuber">21b</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in antimicrobial nucleoside antibiotics targeting cell wall biosynthesis</span></div><div class="casAuthors">Kimura, Ken-Ichi; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">252-273</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The biosynthesis of peptidoglycan is essential for all bacteria and has no counterpart in eukaryotic cells.  It is one of the prime targets for antibiotic chemotherapy, esp. phospho-MurNAc-pentapeptide translocase (translocase I) is a fascinating target in which there is no com. antibiotic.  In this review we will describe three nucleoside translocase I inhibitors, mureidomycin, tunicamycin and liposidomycin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrksNstyeT_iLVg90H21EOLACvtfcHk0lgEcFc2fyeUcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslCqtr4%253D&md5=8c6823d378a4754c368a6179b5bfa057</span></div><a href="/servlet/linkout?suffix=cit21b&amp;dbid=16384&amp;doi=10.1039%2Fb202149h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb202149h%26sid%3Dliteratum%253Aachs%26aulast%3DKimura%26aufirst%3DK.-i.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DRecent%2520advances%2520in%2520antimicrobial%2520nucleoside%2520antibiotics%2520targeting%2520cell%2520wall%2520biosynthesis%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2003%26volume%3D20%26spage%3D252%26epage%3D273%26doi%3D10.1039%2Fb202149h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bouhss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mengin-Lecreulx, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Beller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Heijenoort, J.</span></span> <span> </span><span class="NLM_article-title">Topological analysis of the MraY protein catalysing the first membrane step of peptidoglycan synthesis</span>. <i>Mol. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">576</span>– <span class="NLM_lpage">585</span>, <span class="refDoi"> DOI: 10.1046/j.1365-2958.1999.01623.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1046%2Fj.1365-2958.1999.01623.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10564498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK1MXntlGrtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=576-585&author=A.+Bouhssauthor=D.+Mengin-Lecreulxauthor=D.+Le+Bellerauthor=J.+Van+Heijenoort&title=Topological+analysis+of+the+MraY+protein+catalysing+the+first+membrane+step+of+peptidoglycan+synthesis&doi=10.1046%2Fj.1365-2958.1999.01623.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Topological analysis of the MraY protein catalyzing the first membrane step of peptidoglycan synthesis</span></div><div class="casAuthors">Bouhss, Ahmed; Mengin-Lecreulx, Dominique; Le Beller, Dominique; Van Heijenoort, Jean</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">576-585</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science Ltd.</span>)
        </div><div class="casAbstract">The two-dimensional membrane topol. of the Escherichia coli and Staphylococcus aureus MraY transferases, which catalyze the formation of the first lipid intermediate of peptidoglycan synthesis, was established using the β-lactamase fusion system.  All 28 constructed mraY-blaM fusions produced hybrid proteins.  Anal. of the ampicillin resistance of the strains with hybrids led to a common topol. model possessing 10 transmembrane segments, five cytoplasmic domains and six periplasmic domains including the N- and C-terminal ends.  The agreement between the topologies of E. coli and S. aureus, their agreement to a fair extent with predicted models and a no. of features arising from the comparative anal. of 25 orthologue sequences strongly suggested the validity of the model for all eubacterial MraYs.  The primary structure of the 10 transmembrane segments diverged among orthologues, but they retained their hydrophobicity, no. and size.  The similarity of the sequences and distribution of the five cytoplasmic domains in both models, as well as their conservation among the MraY orthologues, clearly suggested their possible involvement in substrate recognition and in the catalytic process.  Complementation tests showed that only fusions with untruncated mraY restored growth.  It was noteworthy that S. aureus MraY was functional in E. coli.  An increased MraY transferase activity was obsd. only with the untruncated hybrids from both organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqPNxoxJOkUbVg90H21EOLACvtfcHk0lgEcFc2fyeUcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntlGrtr4%253D&md5=2de5abca52068879991d91c9377f3113</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1046%2Fj.1365-2958.1999.01623.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1365-2958.1999.01623.x%26sid%3Dliteratum%253Aachs%26aulast%3DBouhss%26aufirst%3DA.%26aulast%3DMengin-Lecreulx%26aufirst%3DD.%26aulast%3DLe%2BBeller%26aufirst%3DD.%26aulast%3DVan%2BHeijenoort%26aufirst%3DJ.%26atitle%3DTopological%2520analysis%2520of%2520the%2520MraY%2520protein%2520catalysing%2520the%2520first%2520membrane%2520step%2520of%2520peptidoglycan%2520synthesis%26jtitle%3DMol.%2520Microbiol.%26date%3D1999%26volume%3D34%26spage%3D576%26epage%3D585%26doi%3D10.1046%2Fj.1365-2958.1999.01623.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bouhss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trunkfield, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mengin-Lecreulx, D.</span></span> <span> </span><span class="NLM_article-title">The biosynthesis of peptidoglycan lipid-linked intermediates</span>. <i>FEMS Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">208</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1111/j.1574-6976.2007.00089.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1111%2Fj.1574-6976.2007.00089.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=18081839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1GitL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=208-233&author=A.+Bouhssauthor=A.+E.+Trunkfieldauthor=T.+D.+H.+Buggauthor=D.+Mengin-Lecreulx&title=The+biosynthesis+of+peptidoglycan+lipid-linked+intermediates&doi=10.1111%2Fj.1574-6976.2007.00089.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The biosynthesis of peptidoglycan lipid-linked intermediates</span></div><div class="casAuthors">Bouhss, Ahmed; Trunkfield, Amy E.; Bugg, Timothy D. H.; Mengin-Lecreulx, Dominique</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">208-233</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  The biosynthesis of bacterial cell wall peptidoglycan is a complex process involving many different steps taking place in the cytoplasm (synthesis of the nucleotide precursors) and on the inner and outer sides of the cytoplasmic membrane (assembly and polymn. of the disaccharide-peptide monomer unit, resp.).  Here, the authors summarize current knowledge on the membrane steps leading to the formation of the lipid II intermediate, i.e., the substrate of the polymn. reactions.  It makes the point on past and recent data that have significantly contributed to the understanding of the biosynthesis of undecaprenyl phosphate, the carrier lipid required for the anchoring of the peptidoglycan hydrophilic units in the membrane, and to the characterization of the MraY and MurG enzymes which catalyze the successive transfers of the N-acetylmuramoylpeptide and N-acetylglucosamine moieties onto the carrier lipid, resp.  Enzyme inhibitors and antibacterial compds. interfering with these essential metabolic steps and interesting targets are also presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMhOvpdR85NrVg90H21EOLACvtfcHk0ljTsBGpnSBRrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1GitL8%253D&md5=587da9b2738c4751350c35a235c1f3f5</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6976.2007.00089.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6976.2007.00089.x%26sid%3Dliteratum%253Aachs%26aulast%3DBouhss%26aufirst%3DA.%26aulast%3DTrunkfield%26aufirst%3DA.%2BE.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26aulast%3DMengin-Lecreulx%26aufirst%3DD.%26atitle%3DThe%2520biosynthesis%2520of%2520peptidoglycan%2520lipid-linked%2520intermediates%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2008%26volume%3D32%26spage%3D208%26epage%3D233%26doi%3D10.1111%2Fj.1574-6976.2007.00089.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Al-Dabbagh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Ghachi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanot, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parquet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mengin-Lecreulx, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouhss, A.</span></span> <span> </span><span class="NLM_article-title">Active site mapping of MraY, a member of the polyprenyl-phosphate <i>N</i>-acetylhexosamine 1-phosphate transferase superfamily, catalyzing the first membrane step of peptidoglycan biosynthesis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">8919</span>– <span class="NLM_lpage">8928</span>, <span class="refDoi"> DOI: 10.1021/bi8006274</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi8006274" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptlaqur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=8919-8928&author=B.+Al-Dabbaghauthor=X.+Henryauthor=M.+El+Ghachiauthor=G.+Augerauthor=D.+Blanotauthor=C.+Parquetauthor=D.+Mengin-Lecreulxauthor=A.+Bouhss&title=Active+site+mapping+of+MraY%2C+a+member+of+the+polyprenyl-phosphate+N-acetylhexosamine+1-phosphate+transferase+superfamily%2C+catalyzing+the+first+membrane+step+of+peptidoglycan+biosynthesis&doi=10.1021%2Fbi8006274"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Active Site Mapping of MraY, a Member of the Polyprenyl-phosphate N-Acetylhexosamine 1-Phosphate Transferase Superfamily, Catalyzing the First Membrane Step of Peptidoglycan Biosynthesis</span></div><div class="casAuthors">Al-Dabbagh, Bayan; Henry, Xavier; El Ghachi, Meriem; Auger, Genevieve; Blanot, Didier; Parquet, Claudine; Mengin-Lecreulx, Dominique; Bouhss, Ahmed</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">8919-8928</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The MraY transferase is an integral membrane protein that catalyzes an essential step of peptidoglycan biosynthesis, namely the transfer of the phospho-N-acetylmuramoyl-pentapeptide motif onto the undecaprenyl phosphate carrier lipid.  It belongs to a large superfamily of eukaryotic and prokaryotic prenyl sugar transferases.  No 3D structure has been reported for any member of this superfamily, and to date MraY is the only protein that has been successfully purified to homogeneity.  Nineteen polar residues located in the five cytoplasmic segments of MraY appeared as invariants in the sequences of MraY orthologues.  A certain no. of these invariant residues were found to be conserved in the whole superfamily.  To assess the importance of these residues in the catalytic process, site-directed mutagenesis was performed using the Bacillus subtilis MraY as a model.  Fourteen residues were shown to be essential for MraY activity by an in vivo functional complementation assay using a constructed conditional mraY mutant strain.  The corresponding mutant proteins were purified and biochem. characterized.  None of these mutations did significantly affect the binding of the nucleotidic and lipidic substrates, but the kcat was dramatically reduced in almost all cases.  The important residues for activity therefore appeared to be distributed in all the cytoplasmic segments, indicating that these five regions contribute to the structure of the catalytic site.  Our data show that the D98 residue that is invariant in the whole superfamily should be involved in the deprotonation of the lipid substrate during the catalytic process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRVoho_-T00bVg90H21EOLACvtfcHk0ljTsBGpnSBRrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptlaqur8%253D&md5=bb0d877ecbefa5f3c323e9582db02b1e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fbi8006274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi8006274%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Dabbagh%26aufirst%3DB.%26aulast%3DHenry%26aufirst%3DX.%26aulast%3DEl%2BGhachi%26aufirst%3DM.%26aulast%3DAuger%26aufirst%3DG.%26aulast%3DBlanot%26aufirst%3DD.%26aulast%3DParquet%26aufirst%3DC.%26aulast%3DMengin-Lecreulx%26aufirst%3DD.%26aulast%3DBouhss%26aufirst%3DA.%26atitle%3DActive%2520site%2520mapping%2520of%2520MraY%252C%2520a%2520member%2520of%2520the%2520polyprenyl-phosphate%2520N-acetylhexosamine%25201-phosphate%2520transferase%2520superfamily%252C%2520catalyzing%2520the%2520first%2520membrane%2520step%2520of%2520peptidoglycan%2520biosynthesis%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D8919%26epage%3D8928%26doi%3D10.1021%2Fbi8006274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="note"><p class="first last">Review of MraY inhibitors;</p></div><div class="NLM_citation" id="cit25a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Fer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Corre, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietrancosta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evrard-Todeschi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olatunji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouhss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvet-Vitale, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gravier-Pelletier, C.</span></span> <span> </span><span class="NLM_article-title">Bacterial transferase MraY, a source of inspiration towards new antibiotics</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">6013</span>– <span class="NLM_lpage">6029</span>, <span class="refDoi"> DOI: 10.2174/0929867325666180330095154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.2174%2F0929867325666180330095154" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=6013-6029&author=M.+J.+Ferauthor=L.+Le+Correauthor=N.+Pietrancostaauthor=N.+Evrard-Todeschiauthor=S.+Olatunjiauthor=A.+Bouhssauthor=S.+Calvet-Vitaleauthor=C.+Gravier-Pelletier&title=Bacterial+transferase+MraY%2C+a+source+of+inspiration+towards+new+antibiotics&doi=10.2174%2F0929867325666180330095154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25a&amp;dbid=16384&amp;doi=10.2174%2F0929867325666180330095154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867325666180330095154%26sid%3Dliteratum%253Aachs%26aulast%3DFer%26aufirst%3DM.%2BJ.%26aulast%3DLe%2BCorre%26aufirst%3DL.%26aulast%3DPietrancosta%26aufirst%3DN.%26aulast%3DEvrard-Todeschi%26aufirst%3DN.%26aulast%3DOlatunji%26aufirst%3DS.%26aulast%3DBouhss%26aufirst%3DA.%26aulast%3DCalvet-Vitale%26aufirst%3DS.%26aulast%3DGravier-Pelletier%26aufirst%3DC.%26atitle%3DBacterial%2520transferase%2520MraY%252C%2520a%2520source%2520of%2520inspiration%2520towards%2520new%2520antibiotics%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2019%26volume%3D25%26spage%3D6013%26epage%3D6029%26doi%3D10.2174%2F0929867325666180330095154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit25b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Katsuyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Medicinal Chemistry of Muraymycins, Nucleoside Antibiotics</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">123</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1248/cpb.c17-00684</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1248%2Fcpb.c17-00684" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=29386462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVKrtrfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2018&pages=123-131&author=A.+Katsuyamaauthor=S.+Ichikawa&title=Synthesis+and+Medicinal+Chemistry+of+Muraymycins%2C+Nucleoside+Antibiotics&doi=10.1248%2Fcpb.c17-00684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25bR"><div class="casContent"><span class="casTitleNuber">25b</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and medicinal chemistry of muraymycins, nucleoside antibiotics</span></div><div class="casAuthors">Katsuyama, Akira; Ichikawa, Satoshi</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">123-131</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">A review with refs.  Muraymycins, isolated from a culture broth of Streptomyces sp., are members of a class of naturally occurring nucleoside antibiotics.  They are strong inhibitors of the phospho-MurNAc-pentapeptide translocase (MraY), which is responsible for the peptidoglycan biosynthesis.  Since MraY is an essential enzyme among bacteria, muraymycins are expected to be a novel antibacterial agent.  In this review, our efforts to synthesize muraymycin D2, simplify the chem. structure, improve antibacterial spectrum, and solve the X-ray crystal structure of the muraymycin D2/MraY complex are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo3FXON1XjXrVg90H21EOLACvtfcHk0ljwLkwyGdmFGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVKrtrfI&md5=90da55a7dc818c3b5f8da2100aed2724</span></div><a href="/servlet/linkout?suffix=cit25b&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c17-00684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c17-00684%26sid%3Dliteratum%253Aachs%26aulast%3DKatsuyama%26aufirst%3DA.%26aulast%3DIchikawa%26aufirst%3DS.%26atitle%3DSynthesis%2520and%2520Medicinal%2520Chemistry%2520of%2520Muraymycins%252C%2520Nucleoside%2520Antibiotics%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2018%26volume%3D66%26spage%3D123%26epage%3D131%26doi%3D10.1248%2Fcpb.c17-00684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit25c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wiegmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koppermann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyerer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ducho, C.</span></span> <span> </span><span class="NLM_article-title">Muraymycin nucleoside-peptide antibiotics: uridine-derived natural products as lead structures for the development of novel antibacterial agents</span>. <i>Beilstein J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">795</span>, <span class="refDoi"> DOI: 10.3762/bjoc.12.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.3762%2Fbjoc.12.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=27340469" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWhsb7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=769-795&author=D.+Wiegmannauthor=S.+Koppermannauthor=M.+Wirthauthor=G.+Niroauthor=K.+Leyererauthor=C.+Ducho&title=Muraymycin+nucleoside-peptide+antibiotics%3A+uridine-derived+natural+products+as+lead+structures+for+the+development+of+novel+antibacterial+agents&doi=10.3762%2Fbjoc.12.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25cR"><div class="casContent"><span class="casTitleNuber">25c</span><div class="casTitle"><span class="NLM_cas:atitle">Muraymycin nucleoside-peptide antibiotics: uridine-derived natural products as lead structures for the development of novel antibacterial agents</span></div><div class="casAuthors">Wiegmann, Daniel; Koppermann, Stefan; Wirth, Marius; Niro, Giuliana; Leyerer, Kristin; Ducho, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Beilstein Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">769-795</span>CODEN:
                <span class="NLM_cas:coden">BJOCBH</span>;
        ISSN:<span class="NLM_cas:issn">1860-5397</span>.
    
            (<span class="NLM_cas:orgname">Beilstein-Institut zur Foerderung der Chemischen Wissenschaften</span>)
        </div><div class="casAbstract">Muraymycins are a promising class of antimicrobial natural products.  These uridine-derived nucleoside-peptide antibiotics inhibit the bacterial membrane protein translocase I (MraY), a key enzyme in the intracellular part of peptidoglycan biosynthesis.  This review describes the structures of naturally occurring muraymycins, their mode of action, synthetic access to muraymycins and their analogs, some structure-activity relationship (SAR) studies and first insights into muraymycin biosynthesis.  It therefore provides an overview on the current state of research, as well as an outlook on possible future developments in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXk9Xt6Q4WWrVg90H21EOLACvtfcHk0linsQNQ1vylHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWhsb7M&md5=a4fea89e777b397ce7cd1edb34cc6e81</span></div><a href="/servlet/linkout?suffix=cit25c&amp;dbid=16384&amp;doi=10.3762%2Fbjoc.12.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3762%252Fbjoc.12.77%26sid%3Dliteratum%253Aachs%26aulast%3DWiegmann%26aufirst%3DD.%26aulast%3DKoppermann%26aufirst%3DS.%26aulast%3DWirth%26aufirst%3DM.%26aulast%3DNiro%26aufirst%3DG.%26aulast%3DLeyerer%26aufirst%3DK.%26aulast%3DDucho%26aufirst%3DC.%26atitle%3DMuraymycin%2520nucleoside-peptide%2520antibiotics%253A%2520uridine-derived%2520natural%2520products%2520as%2520lead%2520structures%2520for%2520the%2520development%2520of%2520novel%2520antibacterial%2520agents%26jtitle%3DBeilstein%2520J.%2520Org.%2520Chem.%26date%3D2016%26volume%3D12%26spage%3D769%26epage%3D795%26doi%3D10.3762%2Fbjoc.12.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit25d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodolis, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mihalyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamshidi, S.</span></span> <span> </span><span class="NLM_article-title">Inhibition of phospho-MurNAc-pentapeptide translocase (MraY) by nucleoside natural product antibiotics, bacteriophage ϕX174 lysis protein E, and cationic antibacterial peptides</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">6340</span>– <span class="NLM_lpage">6347</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.03.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2Fj.bmc.2016.03.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=27021004" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC28XkvFCqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=6340-6347&author=T.+D.+H.+Buggauthor=M.+T.+Rodolisauthor=A.+Mihalyiauthor=S.+Jamshidi&title=Inhibition+of+phospho-MurNAc-pentapeptide+translocase+%28MraY%29+by+nucleoside+natural+product+antibiotics%2C+bacteriophage+%CF%95X174+lysis+protein+E%2C+and+cationic+antibacterial+peptides&doi=10.1016%2Fj.bmc.2016.03.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25dR"><div class="casContent"><span class="casTitleNuber">25d</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of phospho-MurNAc-pentapeptide translocase (MraY) by nucleoside natural product antibiotics, bacteriophage .vphi.X174 lysis protein E, and cationic antibacterial peptides</span></div><div class="casAuthors">Bugg, Timothy D. H.; Rodolis, Maria T.; Mihalyi, Agnes; Jamshidi, Shirin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6340-6347</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This review covers recent developments in the inhibition of translocase MraY and related phospho-GlcNAc transferases WecA and TagO, and insight into the inhibition and catalytic mechanism of this class of integral membrane proteins from the structure of Aquifex aeolicus MraY.  Recent studies have also identified a protein-protein interaction site in Escherichia coli MraY, that is targeted by bacteriophage .vphi.X174 lysis protein E, and also by cationic antimicrobial peptides contg. Arg-Trp close to their N- or C-termini.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRDrMytnSGmLVg90H21EOLACvtfcHk0linsQNQ1vylHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XkvFCqtb8%253D&md5=31257f2d6c764ce79aa39a926e336fd7</span></div><a href="/servlet/linkout?suffix=cit25d&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.03.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.03.018%26sid%3Dliteratum%253Aachs%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26aulast%3DRodolis%26aufirst%3DM.%2BT.%26aulast%3DMihalyi%26aufirst%3DA.%26aulast%3DJamshidi%26aufirst%3DS.%26atitle%3DInhibition%2520of%2520phospho-MurNAc-pentapeptide%2520translocase%2520%2528MraY%2529%2520by%2520nucleoside%2520natural%2520product%2520antibiotics%252C%2520bacteriophage%2520%25CF%2595X174%2520lysis%2520protein%2520E%252C%2520and%2520cationic%2520antibacterial%2520peptides%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D6340%26epage%3D6347%26doi%3D10.1016%2Fj.bmc.2016.03.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit25e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span> <span> </span><span class="NLM_article-title">Function-oriented synthesis: how to design simplified analogues of antibacterial nucleoside natural products?</span>. <i>Chem. Rec.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1106</span>– <span class="NLM_lpage">1115</span>, <span class="refDoi"> DOI: 10.1002/tcr.201500247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1002%2Ftcr.201500247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=27027613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1OmtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=1106-1115&author=S.+Ichikawa&title=Function-oriented+synthesis%3A+how+to+design+simplified+analogues+of+antibacterial+nucleoside+natural+products%3F&doi=10.1002%2Ftcr.201500247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25eR"><div class="casContent"><span class="casTitleNuber">25e</span><div class="casTitle"><span class="NLM_cas:atitle">Function-oriented synthesis: How to design simplified analogues of antibacterial nucleoside natural products?</span></div><div class="casAuthors">Ichikawa, Satoshi</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Record</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1106-1115</span>CODEN:
                <span class="NLM_cas:coden">CRHEAK</span>;
        ISSN:<span class="NLM_cas:issn">1528-0691</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">It is important to pursue function-oriented synthesis (FOS), a strategy for the design of less structurally complex targets with comparable or superior activity that can be made in a practical manner, because compared to synthetic drugs, many biol. relevant natural products possess large and complex chem. structures that may restrict chem. modifications in a structure-activity relationship study.  In this account, we describe recent efforts to simplify complex nucleoside natural products including caprazamycines.  Considering the structure-activity relationship study with several truncated analogs, three types of simplified derivs., namely, oxazolidine, isoxazolidine, and lactam-fused isoxazolidine-contg. uridine derivs., were designed and efficiently synthesized.  These simplified derivs. have exhibited promising antibacterial activities.  A significant feature of our studies is the rational and drastic simplification of the mol. architecture of caprazamycins.  This study provides a novel strategy for the development of a new type of antibacterial agent effective against drug-resistant bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGbH1STCoqfbVg90H21EOLACvtfcHk0lgjrSHTr-c9wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1OmtLo%253D&md5=0550fc70208bad8899ab47ed9558652a</span></div><a href="/servlet/linkout?suffix=cit25e&amp;dbid=16384&amp;doi=10.1002%2Ftcr.201500247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Ftcr.201500247%26sid%3Dliteratum%253Aachs%26aulast%3DIchikawa%26aufirst%3DS.%26atitle%3DFunction-oriented%2520synthesis%253A%2520how%2520to%2520design%2520simplified%2520analogues%2520of%2520antibacterial%2520nucleoside%2520natural%2520products%253F%26jtitle%3DChem.%2520Rec.%26date%3D2016%26volume%3D16%26spage%3D1106%26epage%3D1115%26doi%3D10.1002%2Ftcr.201500247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit25f"><span><span class="NLM_label">(f) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Antibacterial Nucleoside Natural Products Inhibiting Phospho-MurNAc-Pentapeptide Translocase; Chemistry and Structure-Activity Relationship</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3951</span>– <span class="NLM_lpage">3979</span>, <span class="refDoi"> DOI: 10.2174/0929867322666150818103502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.2174%2F0929867322666150818103502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=26282943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGit7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=3951-3979&author=S.+Ichikawaauthor=M.+Yamaguchiauthor=A.+Matsuda&title=Antibacterial+Nucleoside+Natural+Products+Inhibiting+Phospho-MurNAc-Pentapeptide+Translocase%3B+Chemistry+and+Structure-Activity+Relationship&doi=10.2174%2F0929867322666150818103502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25fR"><div class="casContent"><span class="casTitleNuber">25f</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial Nucleoside Natural Products Inhibiting Phospho-MurNAc-Pentapeptide Translocase; Chemistry and Structure-Activity Relationship.</span></div><div class="casAuthors">Ichikawa, S.; Yamaguchi, M.; Matsuda, A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">3951-3979</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The continued emergence of drug-resistance to existing antibacterial agents represents a severe and ongoing public health concern, which demands the discovery of new antibiotics.  However the no. of novel classes of antibacterial drugs launched in the clinic has been remarkably slow since the 1960s, and it is urgent to develop novel antibacterial agents to fight against drug-resistant bacterial pathogens.  Peptidoglycan is a component of the bacterial cell wall, which consists of a repeated N-acetylmuramic acid (MurNAc) and Nacetylglucosamine (GluNAc) polymer cross-linked with polypeptides, and is a good target for antibacterial drug discovery.  Among enzymes responsible for its biosynthesis, phospho-MurNAc-pentapeptide translocase (MraY) is a novel and promising target.  Many nucleoside natural products, which strongly inhibit MraY, have been found in nature.  This review will summarize the synthesis and biol. properties of selected MraY inhibitory nucleoside natural products and their analogs synthesized in our lab. and by others.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrObgC5iVVElLVg90H21EOLACvtfcHk0lgjrSHTr-c9wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGit7nL&md5=bb572201c43b94cc3461a8809f3d164b</span></div><a href="/servlet/linkout?suffix=cit25f&amp;dbid=16384&amp;doi=10.2174%2F0929867322666150818103502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867322666150818103502%26sid%3Dliteratum%253Aachs%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DYamaguchi%26aufirst%3DM.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DAntibacterial%2520Nucleoside%2520Natural%2520Products%2520Inhibiting%2520Phospho-MurNAc-Pentapeptide%2520Translocase%253B%2520Chemistry%2520and%2520Structure-Activity%2520Relationship%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2015%26volume%3D22%26spage%3D3951%26epage%3D3979%26doi%3D10.2174%2F0929867322666150818103502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit25g"><span><span class="NLM_label">(g) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goss, R. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, K.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial nucleoside antibiotics targeting cell wall assembly: Recent advances in structure–function studies and nucleoside biosynthesis</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">279</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1039/b816215h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25g&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2FB816215H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25g&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=20111805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Sntb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2010&pages=279-304&author=M.+Winnauthor=R.+J.+M.+Gossauthor=K.-i.+Kimuraauthor=T.+D.+H.+Bugg&title=Antimicrobial+nucleoside+antibiotics+targeting+cell+wall+assembly%3A+Recent+advances+in+structure%E2%80%93function+studies+and+nucleoside+biosynthesis&doi=10.1039%2Fb816215h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25gR"><div class="casContent"><span class="casTitleNuber">25g</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial nucleoside antibiotics targeting cell wall assembly: Recent advances in structure-function studies and nucleoside biosynthesis</span></div><div class="casAuthors">Winn, Michael; Goss, Rebecca J. M.; Kimura, Ken-ichi; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">279-304</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The quest for new antibiotics, esp. those with activity against Gram-neg. bacteria, is urgent; however, very few new antibiotics have been marketed in the last 40 years, with this limited no. falling into only four new structural classes.  Several nucleoside natural product antibiotics target bacterial translocase MraY, involved in the lipid-linked cycle of peptidoglycan biosynthesis, and fungal chitin synthase.  Biosynthetic studies on the nikkomycin, caprazamycin and pacidamycin/mureidomycin families are also reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8Wne70VFcV7Vg90H21EOLACvtfcHk0li_Kpj-Lj_QUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Sntb8%253D&md5=49a27386210ce2250334db085a7bfc67</span></div><a href="/servlet/linkout?suffix=cit25g&amp;dbid=16384&amp;doi=10.1039%2FB816215H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FB816215H%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%26aulast%3DGoss%26aufirst%3DR.%2BJ.%2BM.%26aulast%3DKimura%26aufirst%3DK.-i.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DAntimicrobial%2520nucleoside%2520antibiotics%2520targeting%2520cell%2520wall%2520assembly%253A%2520Recent%2520advances%2520in%2520structure%25E2%2580%2593function%2520studies%2520and%2520nucleoside%2520biosynthesis%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2010%26volume%3D27%26spage%3D279%26epage%3D304%26doi%3D10.1039%2Fb816215h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit25h"><span><span class="NLM_label">(h) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Nucleoside natural products and related analogs with potential therapeutic properties as antibacterial and antiviral agents</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">487</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1517/13543776.17.5.487</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25h&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1517%2F13543776.17.5.487" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25h&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksl2nt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=487-498&author=S.+Ichikawaauthor=A.+Matsuda&title=Nucleoside+natural+products+and+related+analogs+with+potential+therapeutic+properties+as+antibacterial+and+antiviral+agents&doi=10.1517%2F13543776.17.5.487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25hR"><div class="casContent"><span class="casTitleNuber">25h</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleoside natural products and related analogs with potential therapeutic properties as antibacterial and antiviral agents</span></div><div class="casAuthors">Ichikawa, Satoshi; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">487-498</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Nucleoside natural products possess a variety of interesting biol. activities, including antibacterial, antiviral and antitumor properties, and are therefore expected to be potential candidates for developing drugs.  Complex nucleoside natural products exhibiting antibacterial activity by specific inhibition of bacterial cell wall peptidoglycan biosynthesis are described.  In addn. to the class of antibacterial nucleoside natural products, the newest members of nucleoside natural products exhibiting interesting antiviral activity are briefly described.  In spite of promising properties, no nucleoside natural products and their analogs are presently used in the clinic.  A global structure-activity relationship of these classes of nucleoside natural products clearly indicates that it is very important to: i) modulate the cell entry ability; and ii) simplify hydrophilic core structures in order to reduce the size of mols. and stabilize the chem. labile structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsFnkD2P4u1bVg90H21EOLACvtfcHk0li_Kpj-Lj_QUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksl2nt74%253D&md5=1d17b1075e0539aa05fb188f9645812b</span></div><a href="/servlet/linkout?suffix=cit25h&amp;dbid=16384&amp;doi=10.1517%2F13543776.17.5.487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.17.5.487%26sid%3Dliteratum%253Aachs%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DNucleoside%2520natural%2520products%2520and%2520related%2520analogs%2520with%2520potential%2520therapeutic%2520properties%2520as%2520antibacterial%2520and%2520antiviral%2520agents%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2007%26volume%3D17%26spage%3D487%26epage%3D498%26doi%3D10.1517%2F13543776.17.5.487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit25i"><span><span class="NLM_label">(i) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, D.</span></span> <span> </span><span class="NLM_article-title">Phospho-MurNAc-pentapeptide translocase (MraY) as a target for antibacterial agents and antibacterial proteins</span>. <i>Infect. Disord.: Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.2174/187152606784112128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25i&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.2174%2F187152606784112128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25i&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=16789873" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25i&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtFers7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=85-106&author=T.+Buggauthor=A.+Lloydauthor=D.+Roper&title=Phospho-MurNAc-pentapeptide+translocase+%28MraY%29+as+a+target+for+antibacterial+agents+and+antibacterial+proteins&doi=10.2174%2F187152606784112128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25iR"><div class="casContent"><span class="casTitleNuber">25i</span><div class="casTitle"><span class="NLM_cas:atitle">Phospho-MurNAc-pentapeptide translocase (MraY) as a target for antibacterial agents and antibacterial proteins</span></div><div class="casAuthors">Bugg, Timothy D. H.; Lloyd, Adrian J.; Roper, David I.</div><div class="citationInfo"><span class="NLM_cas:title">Infectious Disorders: Drug Targets</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-106</span>CODEN:
                <span class="NLM_cas:coden">IDDTAD</span>;
        ISSN:<span class="NLM_cas:issn">1871-5265</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Phospho-MurNAc-pentapeptide translocase (MraY, translocase I) catalyzes the first step of the lipid-linked cycle of reactions of bacterial peptidoglycan biosynthesis.  MraY is the target for five families of nucleoside antibacterial natural products: the tunicamycins, the mureidomycins (also pacidamycins, napsamycins), the liposidomycins, the muraymycins, and the capuramycins.  Recent structure-activity studies on these families have led to the identification of active pharmacophores, and insight into their mechanisms of action.  This step of peptidoglycan biosynthesis is also the target for the bacteriolytic E protein from bacteriophage .vphi.X174, and for cyclic peptides of the amphomycin family which complex the undecaprenyl phosphate cosubstrate.  The mechanisms of enzyme inhibition by these agents are discussed, and the state of knowledge regarding the transmembrane structure, active site, and catalytic mechanism of MraY.  The availability of high throughput assays and prospects of MraY as an antibacterial target are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA2V6DpkrljrVg90H21EOLACvtfcHk0lhY2xsvH-rrwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtFers7Y%253D&md5=b209c55dbd91c009a292e861eebecf44</span></div><a href="/servlet/linkout?suffix=cit25i&amp;dbid=16384&amp;doi=10.2174%2F187152606784112128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F187152606784112128%26sid%3Dliteratum%253Aachs%26aulast%3DBugg%26aufirst%3DT.%26aulast%3DLloyd%26aufirst%3DA.%26aulast%3DRoper%26aufirst%3DD.%26atitle%3DPhospho-MurNAc-pentapeptide%2520translocase%2520%2528MraY%2529%2520as%2520a%2520target%2520for%2520antibacterial%2520agents%2520and%2520antibacterial%2520proteins%26jtitle%3DInfect.%2520Disord.%253A%2520Drug%2520Targets%26date%3D2006%26volume%3D6%26spage%3D85%26epage%3D106%26doi%3D10.2174%2F187152606784112128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit25j"><span><span class="NLM_label">(j) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, K.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Recent advances in antimicrobial nucleoside antibiotics targeting cell wall biosynthesis</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1039/b202149h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25j&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2Fb202149h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25j&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=12735700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25j&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslCqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2003&pages=252-273&author=K.-i.+Kimuraauthor=T.+D.+H.+Bugg&title=Recent+advances+in+antimicrobial+nucleoside+antibiotics+targeting+cell+wall+biosynthesis&doi=10.1039%2Fb202149h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25jR"><div class="casContent"><span class="casTitleNuber">25j</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in antimicrobial nucleoside antibiotics targeting cell wall biosynthesis</span></div><div class="casAuthors">Kimura, Ken-Ichi; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">252-273</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The biosynthesis of peptidoglycan is essential for all bacteria and has no counterpart in eukaryotic cells.  It is one of the prime targets for antibiotic chemotherapy, esp. phospho-MurNAc-pentapeptide translocase (translocase I) is a fascinating target in which there is no com. antibiotic.  In this review we will describe three nucleoside translocase I inhibitors, mureidomycin, tunicamycin and liposidomycin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrksNstyeT_iLVg90H21EOLACvtfcHk0lhY2xsvH-rrwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslCqtr4%253D&md5=8c6823d378a4754c368a6179b5bfa057</span></div><a href="/servlet/linkout?suffix=cit25j&amp;dbid=16384&amp;doi=10.1039%2Fb202149h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb202149h%26sid%3Dliteratum%253Aachs%26aulast%3DKimura%26aufirst%3DK.-i.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DRecent%2520advances%2520in%2520antimicrobial%2520nucleoside%2520antibiotics%2520targeting%2520cell%2520wall%2520biosynthesis%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2003%26volume%3D20%26spage%3D252%26epage%3D273%26doi%3D10.1039%2Fb202149h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusick, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karkhoff-Schweizer, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomovskaya, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweizer, H. P.</span></span> <span> </span><span class="NLM_article-title">High-level pacidamycin resistance in <i>Pseudomonas aeruginosa</i> is mediated by an Opp oligopeptide permease encoded by the <i>opp-fabI</i> operon</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">5565</span>– <span class="NLM_lpage">5571</span>, <span class="refDoi"> DOI: 10.1128/aac.01198-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1128%2FAAC.01198-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=23979749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Kjtr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2013&pages=5565-5571&author=A.+Mistryauthor=M.+S.+Warrenauthor=J.+K.+Cusickauthor=R.+R.+Karkhoff-Schweizerauthor=O.+Lomovskayaauthor=H.+P.+Schweizer&title=High-level+pacidamycin+resistance+in+Pseudomonas+aeruginosa+is+mediated+by+an+Opp+oligopeptide+permease+encoded+by+the+opp-fabI+operon&doi=10.1128%2Faac.01198-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26aR"><div class="casContent"><span class="casTitleNuber">26a</span><div class="casTitle"><span class="NLM_cas:atitle">High-level pacidamycin resistance in Pseudomonas aeruginosa is mediated by an Opp oligopeptide permease encoded by the opp-fabI operon</span></div><div class="casAuthors">Mistry, Anita; Warren, Mark S.; Cusick, John K.; Karkhoff-Schweizer, RoxAnn R.; Lomovskaya, Olga; Schweizer, Herbert P.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5565-5571</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Pacidamycins (or uridyl peptide antibiotics) possess selective in vivo activity against Pseudomonas aeruginosa.  An important limitation for the therapeutic use of pacidamycins with P. aeruginosa is the high frequency (10-6 to 10-7) at which resistant mutants emerge.  To elucidate the mechanism(s) of this resistance, pacidamycin-resistant P. aeruginosa mutants were isolated.  Two types of mutants were obtained.  Type 1, or high-level resistance mutants with a pacidamycin MIC of 512 μg/mL, were more abundant, with a frequency of ∼2 × 10-6, and did not show cross-resistance with other antibiotics.  Type 2, low-level resistance mutants, were isolated with a frequency of ∼10-8 and had a pacidamycin MIC of 64 μg/mL (the MIC for the wild-type strain was 4 to 16 μg/mL).  These mutants were cross-resistant to levofloxacin, tetracycline, and erythromycin and were shown to overexpress either the MexAB-OprM or MexCD-OprJ multidrug resistance efflux pumps.  High-level resistant mutants were isolated by transposon mutagenesis and one insertion was localized to oppB, one of two periplasmic binding protein components of an oligopeptide transport system which is encoded by the opp-fabI operon.  The Opp system is required for uptake of pacidamycin across the inner membrane, since various opp, but not fabI, mutants were resistant to high levels of pacidamycin.  Both of the two putative Opp periplasmic binding proteins, OppA and OppB, were required for pacidamycin uptake.  Although both impaired uptake into and efflux from the cell can cause pacidamycin resistance in P. aeruginosa, our data suggest that impaired uptake is the primary reason for the high-frequency and high-level pacidamycin resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfDLaUEW4Ib7Vg90H21EOLACvtfcHk0lg_i2ewtcqVYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Kjtr7L&md5=5f45df1642af5825a9545b0ac6e4d228</span></div><a href="/servlet/linkout?suffix=cit26a&amp;dbid=16384&amp;doi=10.1128%2FAAC.01198-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01198-13%26sid%3Dliteratum%253Aachs%26aulast%3DMistry%26aufirst%3DA.%26aulast%3DWarren%26aufirst%3DM.%2BS.%26aulast%3DCusick%26aufirst%3DJ.%2BK.%26aulast%3DKarkhoff-Schweizer%26aufirst%3DR.%2BR.%26aulast%3DLomovskaya%26aufirst%3DO.%26aulast%3DSchweizer%26aufirst%3DH.%2BP.%26atitle%3DHigh-level%2520pacidamycin%2520resistance%2520in%2520Pseudomonas%2520aeruginosa%2520is%2520mediated%2520by%2520an%2520Opp%2520oligopeptide%2520permease%2520encoded%2520by%2520the%2520opp-fabI%2520operon%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2013%26volume%3D57%26spage%3D5565%26epage%3D5571%26doi%3D10.1128%2Faac.01198-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit26b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Pletzer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubiley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köhler, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, M. G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severinov, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weingart, H.</span></span> <span> </span><span class="NLM_article-title">The <i>Pseudomonas aeruginosa</i> PA14 ABC transporter NppA1A2BCD is required for uptake of peptidyl nucleoside antibiotics</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>197</i></span>,  <span class="NLM_fpage">2217</span>– <span class="NLM_lpage">2228</span>, <span class="refDoi"> DOI: 10.1128/jb.00234-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1128%2FJB.00234-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=25917903" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVGgtL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=197&publication_year=2015&pages=2217-2228&author=D.+Pletzerauthor=Y.+Braunauthor=S.+Dubileyauthor=C.+Lafonauthor=T.+K%C3%B6hlerauthor=M.+G.+P.+Pageauthor=M.+Mourezauthor=K.+Severinovauthor=H.+Weingart&title=The+Pseudomonas+aeruginosa+PA14+ABC+transporter+NppA1A2BCD+is+required+for+uptake+of+peptidyl+nucleoside+antibiotics&doi=10.1128%2Fjb.00234-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26bR"><div class="casContent"><span class="casTitleNuber">26b</span><div class="casTitle"><span class="NLM_cas:atitle">The Pseudomonas aeruginosa PA14 ABC transporter NppA1A2BCD is required for uptake of peptidyl nucleoside antibiotics</span></div><div class="casAuthors">Pletzer, Daniel; Braun, Yvonne; Dubiley, Svetlana; Lafon, Corinne; Koehler, Thilo; Page, Malcolm G. P.; Mourez, Michael; Severinov, Konstantin; Weingart, Helge</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">197</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2217-2228</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">1098-5530</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Anal. of the genome sequence of Pseudomonas aeruginosa PA14 revealed the presence of an operon encoding an ABC-type transporter (NppA1A2BCD) showing homol. to the Yej transporter of Escherichia coli.  The Yej transporter is involved in the uptake of the peptide-nucleotide antibiotic microcin C, a translation inhibitor that targets the enzyme aspartyl-tRNA synthetase.  Furthermore, it was recently shown that the Opp transporter from P. aeruginosa PAO1, which is identical to Npp, is required for uptake of the uridyl peptide antibiotic pacidamycin, which targets the enzyme translocase I (MraY), which is involved in peptidoglycan synthesis.  The authors used several approaches to further explore the substrate specificity of the Npp transporter.  Assays of growth in defined minimal medium contg. peptides of various lengths and amino acid compns. as sole nitrogen sources, as well as Biolog Phenotype MicroArrays, showed that the Npp transporter is not required for di-, tri-, and oligopeptide uptake.  Overexpression of the npp operon increased susceptibility not just to pacidamycin but also to nickel chloride and the peptidyl nucleoside antibiotic blasticidin S.  Furthermore, heterologous expression of the npp operon in a yej-deficient mutant of E. coli resulted in increased susceptibility to albomycin, a naturally occurring sideromycin with a peptidyl nucleoside antibiotic.  Addnl., heterologous expression showed that microcin C is recognized by the P. aeruginosa Npp system.  Thus, these results suggest that the NppA1A2BCD transporter is involved in the uptake of peptidyl nucleoside antibiotics by P. aeruginosa PA14.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbpXQyntxmhLVg90H21EOLACvtfcHk0lg_i2ewtcqVYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVGgtL3J&md5=122c04b5145b955316d2ab5674a17b4d</span></div><a href="/servlet/linkout?suffix=cit26b&amp;dbid=16384&amp;doi=10.1128%2FJB.00234-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.00234-15%26sid%3Dliteratum%253Aachs%26aulast%3DPletzer%26aufirst%3DD.%26aulast%3DBraun%26aufirst%3DY.%26aulast%3DDubiley%26aufirst%3DS.%26aulast%3DLafon%26aufirst%3DC.%26aulast%3DK%25C3%25B6hler%26aufirst%3DT.%26aulast%3DPage%26aufirst%3DM.%2BG.%2BP.%26aulast%3DMourez%26aufirst%3DM.%26aulast%3DSeverinov%26aufirst%3DK.%26aulast%3DWeingart%26aufirst%3DH.%26atitle%3DThe%2520Pseudomonas%2520aeruginosa%2520PA14%2520ABC%2520transporter%2520NppA1A2BCD%2520is%2520required%2520for%2520uptake%2520of%2520peptidyl%2520nucleoside%2520antibiotics%26jtitle%3DJ.%2520Bacteriol.%26date%3D2015%26volume%3D197%26spage%3D2217%26epage%3D2228%26doi%3D10.1128%2Fjb.00234-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakagami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togame, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Total Synthesis of Pacidamycin D by Cu(I)-Catalyzed Oxy Enamide Formation</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">5240</span>– <span class="NLM_lpage">5243</span>, <span class="refDoi"> DOI: 10.1021/ol202124b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol202124b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCgsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=5240-5243&author=K.+Okamotoauthor=M.+Sakagamiauthor=F.+Fengauthor=H.+Togameauthor=H.+Takemotoauthor=S.+Ichikawaauthor=A.+Matsuda&title=Total+Synthesis+of+Pacidamycin+D+by+Cu%28I%29-Catalyzed+Oxy+Enamide+Formation&doi=10.1021%2Fol202124b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Total Synthesis of Pacidamycin D by Cu(I)-Catalyzed Oxy Enamide Formation</span></div><div class="casAuthors">Okamoto, Kazuya; Sakagami, Masahiro; Feng, Fei; Togame, Hiroko; Takemoto, Hiroshi; Ichikawa, Satoshi; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5240-5243</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The first total synthesis of pacidamycin D, which is expected to be a good candidate as an antibacterial agent against P. aeruginosa, is described.  The key elements of this synthetic approach feature an efficient and stereocontrolled construction of the Z-oxyvinyl iodide and copper-catalyzed cross-coupling with the tetrapeptide carboxamide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyYNL8uT0xTLVg90H21EOLACvtfcHk0lipl5XlihGalA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCgsLrJ&md5=1fea3a123b0d5f46fcd4a556f28207ad</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fol202124b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol202124b%26sid%3Dliteratum%253Aachs%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DSakagami%26aufirst%3DM.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DTogame%26aufirst%3DH.%26aulast%3DTakemoto%26aufirst%3DH.%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DTotal%2520Synthesis%2520of%2520Pacidamycin%2520D%2520by%2520Cu%2528I%2529-Catalyzed%2520Oxy%2520Enamide%2520Formation%26jtitle%3DOrg.%2520Lett.%26date%3D2011%26volume%3D13%26spage%3D5240%26epage%3D5243%26doi%3D10.1021%2Fol202124b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakagami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togame, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Total Synthesis and Biological Evaluation of Pacidamycin D and Its 3′-Hydroxy Analogue</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">1367</span>– <span class="NLM_lpage">1377</span>, <span class="refDoi"> DOI: 10.1021/jo202159q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo202159q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1GlsL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2012&pages=1367-1377&author=K.+Okamotoauthor=M.+Sakagamiauthor=F.+Fengauthor=H.+Togameauthor=H.+Takemotoauthor=S.+Ichikawaauthor=A.+Matsuda&title=Total+Synthesis+and+Biological+Evaluation+of+Pacidamycin+D+and+Its+3%E2%80%B2-Hydroxy+Analogue&doi=10.1021%2Fjo202159q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Total Synthesis and Biological Evaluation of Pacidamycin D and Its 3'-Hydroxy Analogue</span></div><div class="casAuthors">Okamoto, Kazuya; Sakagami, Masahiro; Feng, Fei; Togame, Hiroko; Takemoto, Hiroshi; Ichikawa, Satoshi; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1367-1377</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">(compd. structures identified in this abstr. correspond to Roman numerals in graphic.).  Full details of the total synthesis of pacidamycin D (4) and its 3'-hydroxy analog 32 are described.  The chem. labile Z-oxyacyl enamide moiety is the most challenging chem. structure found in uridylpeptide natural products.  Key elements of our approach to the synthesis of 4 include the efficient and stereocontrolled construction of the Z-oxyvinyl halides 6 and 7 and their copper-catalyzed cross-coupling with the tetrapeptide carboxamide 5, a thermally unstable compd. contg. a no. of potentially reactive functional groups.  This synthetic route also allowed us to easily prep. 3'-hydroxy analog 32.  The assemblage by cross-coupling of the Z-oxyvinyl halide 6 and the carboxamide 5 at a late stage of the synthesis provided ready access to a range of uridylpeptide antibiotics and their analogs, despite their inherent labile nature with potential epimerization, simply by altering the tetrapeptide moiety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLrAm5Px3svLVg90H21EOLACvtfcHk0lj_0eOaRK10sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1GlsL%252FE&md5=4de3744da72fad4ea807629e8d51e645</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjo202159q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo202159q%26sid%3Dliteratum%253Aachs%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DSakagami%26aufirst%3DM.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DTogame%26aufirst%3DH.%26aulast%3DTakemoto%26aufirst%3DH.%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DTotal%2520Synthesis%2520and%2520Biological%2520Evaluation%2520of%2520Pacidamycin%2520D%2520and%2520Its%25203%25E2%2580%25B2-Hydroxy%2520Analogue%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2012%26volume%3D77%26spage%3D1367%26epage%3D1377%26doi%3D10.1021%2Fjo202159q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gentle, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Structure-function studies on nucleoside antibiotic mureidomycin A: synthesis of 5′-functionalised uridine models</span>. <i>J. Chem. Soc., Perkin Trans. 1</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_fpage">1287</span>– <span class="NLM_lpage">1294</span>, <span class="refDoi"> DOI: 10.1039/a901287g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2Fa901287g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK1MXjtVegtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1999&pages=1287-1294&author=C.+A.+Gentleauthor=S.+A.+Harrisonauthor=M.+Inukaiauthor=T.+D.+H.+Bugg&title=Structure-function+studies+on+nucleoside+antibiotic+mureidomycin+A%3A+synthesis+of+5%E2%80%B2-functionalised+uridine+models&doi=10.1039%2Fa901287g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-function studies on nucleoside antibiotic mureidomycin A: synthesis of 5'-functionalized uridine models</span></div><div class="casAuthors">Gentle, Caragh A.; Harrison, Stephen A.; Inukai, Masatoshi; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Perkin Transactions 1: Organic and Bio-Organic Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1287-1294</span>CODEN:
                <span class="NLM_cas:coden">JCPRB4</span>;
        ISSN:<span class="NLM_cas:issn">0300-922X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The importance of functional groups in nucleoside antibiotic mureidomycin A (MRD A) for biol. activity has been examd. by derivatization of samples of the natural product, by synthesis of uridine-contg. analogs.  N-Succinyl and di- and tri-acetyl derivs. MRD A have been prepd., and were found to have reduced activity as inhibitors of E. coli translocase I.  The enamide alkene of MRD A was found to be extremely resistant towards hydrogenation by a variety of reagents.  Several 5'-functionalized uridine derivs. were synthesized from N 3-p-methoxybenzyl-2',3'-isopropylideneuridine.  A series of 5'-aminoacyl derivs. were prepd., and the 3-aminopropionyl (IC50 260 μM) and 7-aminoheptanoyl (IC50 1.5 mM) derivs. were found to act as reversible inhibitors.  An analog mimicking the carboxy terminus of MRD A was synthesized, and also acted as an inhibitor of translocase I (IC50 1.9 mM).  A phosphonate analog designed as a possible suicide inhibitor showed modest inhibition (IC50 3.7 mM), which was shown to be irreversible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ11wcWMYYq7Vg90H21EOLACvtfcHk0lj_0eOaRK10sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtVegtLk%253D&md5=640e78e2d719d3f5246c8383405bc470</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2Fa901287g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fa901287g%26sid%3Dliteratum%253Aachs%26aulast%3DGentle%26aufirst%3DC.%2BA.%26aulast%3DHarrison%26aufirst%3DS.%2BA.%26aulast%3DInukai%26aufirst%3DM.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DStructure-function%2520studies%2520on%2520nucleoside%2520antibiotic%2520mureidomycin%2520A%253A%2520synthesis%2520of%25205%25E2%2580%25B2-functionalised%2520uridine%2520models%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25201%26date%3D1999%26spage%3D1287%26epage%3D1294%26doi%3D10.1039%2Fa901287g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bozzoli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazmierski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasquarello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pecunioso, A.</span></span> <span> </span><span class="NLM_article-title">A solid-phase approach to analogues of the antibiotic mureidomycin</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2759</span>– <span class="NLM_lpage">2763</span>, <span class="refDoi"> DOI: 10.1016/s0960-894x(00)00560-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2FS0960-894X%2800%2900560-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=11133085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosFeltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=2759-2763&author=A.+Bozzoliauthor=W.+Kazmierskiauthor=G.+Kennedyauthor=A.+Pasquarelloauthor=A.+Pecunioso&title=A+solid-phase+approach+to+analogues+of+the+antibiotic+mureidomycin&doi=10.1016%2Fs0960-894x%2800%2900560-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A solid-phase approach to analogues of the antibiotic mureidomycin</span></div><div class="casAuthors">Bozzoli, Andrea; Kazmierski, Wieslaw; Kennedy, Gordon; Pasquarello, Alessandra; Pecunioso, Angelo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">2759-2763</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A library of 80 analogs of the antibacterial family of mureidomycins [(I); R = H, mureidomycin A; R = Gly, mureidomycin C], was prepd. using solid-phase chem. techniques.  Analog fragments (2,3-diaminopropionic acid, methionine, p-tyrosine, m-tyrosine, as enantiomers, and tert-Bu 2-Ph malonate, as a racemate), were used in the synthesis.  A selection of ten of the compds. with best a/a values (by LC-MS anal.) was presented.  No inhibitory activity on the growth of S. aureus 853, E coli 1952, Saccharomyces cerevisiae NCY81, P. aeruginosa and P aeruginosa 2033 were detected for any library member.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfJCQoDsV6VbVg90H21EOLACvtfcHk0ljzIAyG_i-TZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosFeltro%253D&md5=72a44c2c88635f3baf5f0b07132305be</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2800%2900560-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252800%252900560-6%26sid%3Dliteratum%253Aachs%26aulast%3DBozzoli%26aufirst%3DA.%26aulast%3DKazmierski%26aufirst%3DW.%26aulast%3DKennedy%26aufirst%3DG.%26aulast%3DPasquarello%26aufirst%3DA.%26aulast%3DPecunioso%26aufirst%3DA.%26atitle%3DA%2520solid-phase%2520approach%2520to%2520analogues%2520of%2520the%2520antibiotic%2520mureidomycin%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2000%26volume%3D10%26spage%3D2759%26epage%3D2763%26doi%3D10.1016%2Fs0960-894x%2800%2900560-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boojamra, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemoine, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Léger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernier, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomovskaya, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberland, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecker, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V. J.</span></span> <span> </span><span class="NLM_article-title">Stereochemical elucidation and total synthesis of dihydropacidamycin D, a semisynthetic pacidamycin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">870</span>– <span class="NLM_lpage">874</span>, <span class="refDoi"> DOI: 10.1021/ja003292c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja003292c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivVWlug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2001&pages=870-874&author=C.+G.+Boojamraauthor=R.+C.+Lemoineauthor=J.+C.+Leeauthor=R.+L%C3%A9gerauthor=K.+A.+Steinauthor=N.+G.+Vernierauthor=A.+Magonauthor=O.+Lomovskayaauthor=P.+K.+Martinauthor=S.+Chamberlandauthor=M.+D.+Leeauthor=S.+J.+Heckerauthor=V.+J.+Lee&title=Stereochemical+elucidation+and+total+synthesis+of+dihydropacidamycin+D%2C+a+semisynthetic+pacidamycin&doi=10.1021%2Fja003292c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Stereochemical elucidation and total synthesis of dihydropacidamycin D, a semisynthetic pacidamycin</span></div><div class="casAuthors">Boojamra, Constantine G.; Lemoine, Remy C.; Lee, Julie C.; Leger, Roger; Stein, Karin A.; Vernier, Nicole G.; Magon, Angela; Lomovskaya, Olga; Martin, Patrick K.; Chamberland, Suzanne; Lee, May D.; Hecker, Scott J.; Lee, Ving J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">870-874</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydrogenation of the C(4') exocyclic olefin of the pacidamycins has been shown to produce a series of semisynthetic compds., the dihydropacidamycins, with antimicrobial activity similar to that of the natural products.  Elucidation of stereochem. in the pacidamycins has been completed through a campaign of natural product degrdn. expts. in combination with the total synthesis of the lowest-mol. wt. dihydropacidamycin, dihydropacidamycin D.  The stereochem. identities of the tryptophan and two alanine residues contained in pacidamycin D have been shown to be of the natural (S) configuration, and the unique 3-methylamino-2-aminobutyric acid contained in this series of antibiotics has been shown to be of the (2S,3S) configuration.  Finally, the stereochem. obtained by hydrogenation of the C(4')-C(5') exocyclic olefin has been shown to be (R) at the C(4') nucleoside site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAex6jWLlofrVg90H21EOLACvtfcHk0ljzIAyG_i-TZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivVWlug%253D%253D&md5=0c2e28c233f558d99a2f7affa9108bd8</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fja003292c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja003292c%26sid%3Dliteratum%253Aachs%26aulast%3DBoojamra%26aufirst%3DC.%2BG.%26aulast%3DLemoine%26aufirst%3DR.%2BC.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DL%25C3%25A9ger%26aufirst%3DR.%26aulast%3DStein%26aufirst%3DK.%2BA.%26aulast%3DVernier%26aufirst%3DN.%2BG.%26aulast%3DMagon%26aufirst%3DA.%26aulast%3DLomovskaya%26aufirst%3DO.%26aulast%3DMartin%26aufirst%3DP.%2BK.%26aulast%3DChamberland%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DM.%2BD.%26aulast%3DHecker%26aufirst%3DS.%2BJ.%26aulast%3DLee%26aufirst%3DV.%2BJ.%26atitle%3DStereochemical%2520elucidation%2520and%2520total%2520synthesis%2520of%2520dihydropacidamycin%2520D%252C%2520a%2520semisynthetic%2520pacidamycin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2001%26volume%3D123%26spage%3D870%26epage%3D874%26doi%3D10.1021%2Fja003292c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lemoine, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecker, S. J.</span></span> <span> </span><span class="NLM_article-title">Synthesis of base-modified dihydropacidamycins</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1121</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1016/s0960-894x(02)00100-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2FS0960-894X%2802%2900100-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=11909731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD38XitFWhtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=1121-1123&author=R.+C.+Lemoineauthor=A.+Magonauthor=S.+J.+Hecker&title=Synthesis+of+base-modified+dihydropacidamycins&doi=10.1016%2Fs0960-894x%2802%2900100-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of base-modified dihydropacidamycins</span></div><div class="casAuthors">Lemoine, Remy C.; Magon, Angela; Hecker, Scott J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1121-1123</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The authors describe the synthesis of 1,2-di-O-acetyl-5-azido-3,5-dideoxy-α,β-L-arabinofuranose, a carbohydrate donor that was used for the synthesis of 1-(5'-amino-3',5'-dideoxy-α-L-arabinofuranosyl)uracil, the nucleoside found in dihydropacidamycin D.  The carbohydrate donor was also used for the synthesis of a set of new nucleosides that were introduced in new dihydropacidamycins.  These compds. were tested for biol. activity, and the results showed that uracil is the only base recognized by MraY.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm_0n48wLNS7Vg90H21EOLACvtfcHk0liAMz3nFsfS1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitFWhtrs%253D&md5=7695a61500331abb4fecca6941e9f513</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2802%2900100-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252802%252900100-2%26sid%3Dliteratum%253Aachs%26aulast%3DLemoine%26aufirst%3DR.%2BC.%26aulast%3DMagon%26aufirst%3DA.%26aulast%3DHecker%26aufirst%3DS.%2BJ.%26atitle%3DSynthesis%2520of%2520base-modified%2520dihydropacidamycins%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2002%26volume%3D12%26spage%3D1121%26epage%3D1123%26doi%3D10.1016%2Fs0960-894x%2802%2900100-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boojamra, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemoine, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blais, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernier, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberland, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecker, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V. J.</span></span> <span> </span><span class="NLM_article-title">Synthetic dihydropacidamycin antibiotics: a modified spectrum of activity for the pacidamycin class</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3305</span>– <span class="NLM_lpage">3309</span>, <span class="refDoi"> DOI: 10.1016/s0960-894x(03)00682-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2FS0960-894X%2803%2900682-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=12951115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvV2ltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=3305-3309&author=C.+G.+Boojamraauthor=R.+C.+Lemoineauthor=J.+Blaisauthor=N.+G.+Vernierauthor=K.+A.+Steinauthor=A.+Magonauthor=S.+Chamberlandauthor=S.+J.+Heckerauthor=V.+J.+Lee&title=Synthetic+dihydropacidamycin+antibiotics%3A+a+modified+spectrum+of+activity+for+the+pacidamycin+class&doi=10.1016%2Fs0960-894x%2803%2900682-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic dihydropacidamycin antibiotics: a modified spectrum of activity for the pacidamycin class</span></div><div class="casAuthors">Boojamra, Constantine G.; Lemoine, Remy C.; Blais, Johanne; Vernier, Nicole G.; Stein, Karin A.; Magon, Angela; Chamberland, Suzanne; Hecker, Scott J.; Lee, Ving J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3305-3309</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Dihydropacidamycins having an antibacterial spectrum modified from that of the natural product pacidamycins and mureidomycins have been synthesized.  Synthetic dihydropacidamycins with noteworthy antibacterial activity against wild-type and resistant Escherichia coli have been identified (MIC = 4-8 μg/mL).  Some dihydropacidamycins are shown to have activity against multi-resistant clin. strains of Mycobacterium tuberculosis.  Compds. of this class are inhibitors of the cell wall biosynthetic enzyme, MraY.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2XZsZvatUTLVg90H21EOLACvtfcHk0lis3RXZzQXXBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvV2ltL0%253D&md5=5fe29abd53588760ae84603f168f06e2</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2803%2900682-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252803%252900682-6%26sid%3Dliteratum%253Aachs%26aulast%3DBoojamra%26aufirst%3DC.%2BG.%26aulast%3DLemoine%26aufirst%3DR.%2BC.%26aulast%3DBlais%26aufirst%3DJ.%26aulast%3DVernier%26aufirst%3DN.%2BG.%26aulast%3DStein%26aufirst%3DK.%2BA.%26aulast%3DMagon%26aufirst%3DA.%26aulast%3DChamberland%26aufirst%3DS.%26aulast%3DHecker%26aufirst%3DS.%2BJ.%26aulast%3DLee%26aufirst%3DV.%2BJ.%26atitle%3DSynthetic%2520dihydropacidamycin%2520antibiotics%253A%2520a%2520modified%2520spectrum%2520of%2520activity%2520for%2520the%2520pacidamycin%2520class%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D3305%26epage%3D3309%26doi%3D10.1016%2Fs0960-894x%2803%2900682-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Howard, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Synthesis and activity of 5′-Uridinyl dipeptide analogues mimicking the amino terminal peptide chain of nucleoside antibiotic mureidomycin A</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3083</span>– <span class="NLM_lpage">3099</span>, <span class="refDoi"> DOI: 10.1016/s0968-0896(03)00270-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2FS0968-0896%2803%2900270-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=12818671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3sXks1amt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=3083-3099&author=N.+I.+Howardauthor=T.+D.+H.+Bugg&title=Synthesis+and+activity+of+5%E2%80%B2-Uridinyl+dipeptide+analogues+mimicking+the+amino+terminal+peptide+chain+of+nucleoside+antibiotic+mureidomycin+A&doi=10.1016%2Fs0968-0896%2803%2900270-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and activity of 5'-Uridinyl dipeptide analogues mimicking the amino terminal peptide chain of nucleoside antibiotic mureidomycin A</span></div><div class="casAuthors">Howard, Nigel I.; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3083-3099</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">This work reports the synthesis of a series of 5'-uridinyl dipeptides, which mimic the amino terminal chain of nucleoside antibiotic mureidomycin A.  For example, 5'-uridinyl dipeptides connected by amide linkage, I (AA = L-Phe, D-Phe, L-Tyr, Gly, β-Ala, L-Ala; R = H, Me), and dipeptides connected by ester linkage, II, were prepd. and their biol. activity as inhibitors of E. coli phospho-MurNAc-pentapeptide translocase (MraY) was evaluated.  The most active inhibitor of E. coli MraY was II (AA = Ala, R = Me), which showed 97% enzyme inhibition at 2.35 mM concn., and showed antibacterial activity at 100 μg/mL concn. against Pseudomonas putida.  Both the central N-Me amide linkage and a 5'-uridine ester linkage were required for highest biol. activity.  Enzyme inhibition was shown to be competitive with Mg2+.  It is proposed that the primary amino terminus of the inhibitor binds in place of the Mg2+ cofactor at the MraY active site, positioned via a cis-N-Me amide linkage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDF2zsLzTUQbVg90H21EOLACvtfcHk0lis3RXZzQXXBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXks1amt7w%253D&md5=f1a35252a3da6d3eca43569a18db8171</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2803%2900270-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252803%252900270-0%26sid%3Dliteratum%253Aachs%26aulast%3DHoward%26aufirst%3DN.%2BI.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DSynthesis%2520and%2520activity%2520of%25205%25E2%2580%25B2-Uridinyl%2520dipeptide%2520analogues%2520mimicking%2520the%2520amino%2520terminal%2520peptide%2520chain%2520of%2520nucleoside%2520antibiotic%2520mureidomycin%2520A%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2003%26volume%3D11%26spage%3D3083%26epage%3D3099%26doi%3D10.1016%2Fs0968-0896%2803%2900270-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, R. E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherman, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeil, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, R. E.</span></span> <span> </span><span class="NLM_article-title">Solid-phase synthesis and biological evaluation of a uridinyl branched peptide urea library</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">6899</span>– <span class="NLM_lpage">6904</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2007.09.097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2Fj.bmcl.2007.09.097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=17962016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlWjtLzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=6899-6904&author=D.+Sunauthor=V.+Jonesauthor=E.+I.+Carsonauthor=R.+E.+B.+Leeauthor=M.+S.+Schermanauthor=M.+R.+McNeilauthor=R.+E.+Lee&title=Solid-phase+synthesis+and+biological+evaluation+of+a+uridinyl+branched+peptide+urea+library&doi=10.1016%2Fj.bmcl.2007.09.097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Solid-phase synthesis and biological evaluation of a uridinyl branched peptide urea library</span></div><div class="casAuthors">Sun, Dianqing; Jones, Victoria; Carson, Elizabeth I.; Lee, Robin E. B.; Scherman, Michael S.; McNeil, Michael R.; Lee, Richard E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6899-6904</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A 1000-member uridinyl branched peptide library was synthesized on PS-DES support using IRORI technol.  High-throughput screening of this library for anti-tuberculosis activity identified several members with a MIC90 = 12.5 μg/mL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrksRV-IXa4dbVg90H21EOLACvtfcHk0lj0VLl9Xluv0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlWjtLzN&md5=33e30357d583e02da8058949556b1251</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.09.097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.09.097%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DD.%26aulast%3DJones%26aufirst%3DV.%26aulast%3DCarson%26aufirst%3DE.%2BI.%26aulast%3DLee%26aufirst%3DR.%2BE.%2BB.%26aulast%3DScherman%26aufirst%3DM.%2BS.%26aulast%3DMcNeil%26aufirst%3DM.%2BR.%26aulast%3DLee%26aufirst%3DR.%2BE.%26atitle%3DSolid-phase%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520a%2520uridinyl%2520branched%2520peptide%2520urea%2520library%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D6899%26epage%3D6904%26doi%3D10.1016%2Fj.bmcl.2007.09.097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grüschow, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rackham, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newill, P. L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goss, R. J. M.</span></span> <span> </span><span class="NLM_article-title">New pacidamycin antibiotics through precursor-directed biosynthesis</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">360</span>, <span class="refDoi"> DOI: 10.1002/cbic.200800575</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1002%2Fcbic.200800575" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=19090518" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvVymtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=355-360&author=S.+Gr%C3%BCschowauthor=E.+J.+Rackhamauthor=B.+Elkinsauthor=P.+L.+A.+Newillauthor=L.+M.+Hillauthor=R.+J.+M.+Goss&title=New+pacidamycin+antibiotics+through+precursor-directed+biosynthesis&doi=10.1002%2Fcbic.200800575"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">New pacidamycin antibiotics through precursor-directed biosynthesis</span></div><div class="casAuthors">Gruschow, Sabine; Rackham, Emma J.; Elkins, Benjamin; Newill, Philip L. A.; Hill, Lionel M.; Goss, Rebecca J. M.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">355-360</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Pacidamycins, mureidomycins and napsamycins are structurally related uridyl peptide antibiotics that inhibit translocase I, an as yet clin. unexploited target.  This potentially important bioactivity coupled to the biosynthetically intriguing structure of pacidamycin make this natural product a fascinating subject for study.  A precursor-directed biosynthesis approach was employed in order to access new pacidamycin derivs.  Strikingly, the biosynthetic machinery exhibited highly relaxed substrate specificity with the majority of the tryptophan analogs that were administered; this resulted in the prodn. of new pacidamycin derivs.  Remarkably, 2-methyl-, 7-methyl-, 7-chloro- and 7-bromotryptophans produced their corresponding pacidamycin analogs in larger amts. than the natural pacidamycin.  Low levels or no incorporation was obsd. for tryptophans substituted at positions 4, 5 and 6.  The ability to generate bromo- and chloropacidamycins opens up the possibility of further functionalising these compds. through chem. cross-coupling in order to access a much larger family of derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpogTLu9TvpLrVg90H21EOLACvtfcHk0lj0VLl9Xluv0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvVymtr8%253D&md5=f386fd4b1b9de046b5ed1f342e121fb1</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200800575&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200800575%26sid%3Dliteratum%253Aachs%26aulast%3DGr%25C3%25BCschow%26aufirst%3DS.%26aulast%3DRackham%26aufirst%3DE.%2BJ.%26aulast%3DElkins%26aufirst%3DB.%26aulast%3DNewill%26aufirst%3DP.%2BL.%2BA.%26aulast%3DHill%26aufirst%3DL.%2BM.%26aulast%3DGoss%26aufirst%3DR.%2BJ.%2BM.%26atitle%3DNew%2520pacidamycin%2520antibiotics%2520through%2520precursor-directed%2520biosynthesis%26jtitle%3DChemBioChem%26date%3D2009%26volume%3D10%26spage%3D355%26epage%3D360%26doi%3D10.1002%2Fcbic.200800575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roy, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grüschow, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cairns, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goss, R. J. M.</span></span> <span> </span><span class="NLM_article-title">Gene expression enabling synthetic diversification of natural products: chemogenetic generation of pacidamycin analogs</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">12243</span>– <span class="NLM_lpage">12245</span>, <span class="refDoi"> DOI: 10.1021/ja1060406</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja1060406" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVaqs7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=12243-12245&author=A.+D.+Royauthor=S.+Gr%C3%BCschowauthor=N.+Cairnsauthor=R.+J.+M.+Goss&title=Gene+expression+enabling+synthetic+diversification+of+natural+products%3A+chemogenetic+generation+of+pacidamycin+analogs&doi=10.1021%2Fja1060406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Gene Expression Enabling Synthetic Diversification of Natural Products: Chemogenetic Generation of Pacidamycin Analogs</span></div><div class="casAuthors">Roy, Abhijeet Deb; Gruschow, Sabine; Cairns, Nickiwe; Goss, Rebecca J. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12243-12245</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Introduction of prnA, the halogenase gene from pyrrolnitrin biosynthesis, into Streptomyces coeruleorubidus resulted in efficient in situ chlorination of the uridyl peptide antibiotic pacidamycin.  The installed chlorine provided a selectably functionalizable handle enabling synthetic modification of the natural product using mild cross-coupling conditions in crude aq. exts. of the culture broth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBddbriQJpwbVg90H21EOLACvtfcHk0li5Z__qwfvTjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVaqs7%252FK&md5=75bc048ba945187979110b021a964b2f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fja1060406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja1060406%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DA.%2BD.%26aulast%3DGr%25C3%25BCschow%26aufirst%3DS.%26aulast%3DCairns%26aufirst%3DN.%26aulast%3DGoss%26aufirst%3DR.%2BJ.%2BM.%26atitle%3DGene%2520expression%2520enabling%2520synthetic%2520diversification%2520of%2520natural%2520products%253A%2520chemogenetic%2520generation%2520of%2520pacidamycin%2520analogs%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D12243%26epage%3D12245%26doi%3D10.1021%2Fja1060406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakagami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uotani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togame, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Synthesis of pacidamycin analogues via an Ugi-multicomponent reaction</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">4810</span>– <span class="NLM_lpage">4815</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.05.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2Fj.bmcl.2012.05.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=22677318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC38XotFKmu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=4810-4815&author=K.+Okamotoauthor=M.+Sakagamiauthor=F.+Fengauthor=F.+Takahashiauthor=K.+Uotaniauthor=H.+Togameauthor=H.+Takemotoauthor=S.+Ichikawaauthor=A.+Matsuda&title=Synthesis+of+pacidamycin+analogues+via+an+Ugi-multicomponent+reaction&doi=10.1016%2Fj.bmcl.2012.05.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of pacidamycin analogues via an Ugi-multicomponent reaction</span></div><div class="casAuthors">Okamoto, Kazuya; Sakagami, Masahiro; Feng, Fei; Takahashi, Fumiyo; Uotani, Kouichi; Togame, Hiroko; Takemoto, Hiroshi; Ichikawa, Satoshi; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4810-4815</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The second-generation synthesis of 3'-hydroxypacidamycin D has been accomplished via an Ugi-four component reaction at a late stage of the synthesis.  This approach provided ready access to a range of analogs including diastereomers of the diaminobutyric acid residue and hybrid-type analogs of mureidomycins.  Biol. evaluations of these analogs indicated that the stereochem. at the diaminobutyric acid residue has a crucial impact on both the MraY biochem. inhibition and whole-cell antibacterial activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHRTHdECquz7Vg90H21EOLACvtfcHk0lh6QWK7lK30Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XotFKmu7c%253D&md5=d8f0cbed9b9c369769d09497a2166327</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.05.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.05.050%26sid%3Dliteratum%253Aachs%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DSakagami%26aufirst%3DM.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DTakahashi%26aufirst%3DF.%26aulast%3DUotani%26aufirst%3DK.%26aulast%3DTogame%26aufirst%3DH.%26aulast%3DTakemoto%26aufirst%3DH.%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DSynthesis%2520of%2520pacidamycin%2520analogues%2520via%2520an%2520Ugi-multicomponent%2520reaction%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D4810%26epage%3D4815%26doi%3D10.1016%2Fj.bmcl.2012.05.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tran, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pujari, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conroy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowman, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giltrap, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahapatra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tod, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowson, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crick, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Britton, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, R. J.</span></span> <span> </span><span class="NLM_article-title">Sansanmycin natural product analogues as potent and selective anti-mycobacterials that inhibit lipid I biosynthesis</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">14414</span>, <span class="refDoi"> DOI: 10.1038/ncomms14414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1038%2Fncomms14414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=28248311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BC1czhs12jug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=14414&author=A.+T.+Tranauthor=E.+E.+Watsonauthor=V.+Pujariauthor=T.+Conroyauthor=L.+J.+Dowmanauthor=A.+M.+Giltrapauthor=A.+Pangauthor=W.+R.+Wongauthor=R.+G.+Liningtonauthor=S.+Mahapatraauthor=J.+S.+Saundersauthor=A.+Charmanauthor=N.+P.+Westauthor=T.+D.+H.+Buggauthor=J.+Todauthor=C.+G.+Dowsonauthor=D.+I.+Roperauthor=D.+C.+Crickauthor=W.+J.+Brittonauthor=R.+J.+Payne&title=Sansanmycin+natural+product+analogues+as+potent+and+selective+anti-mycobacterials+that+inhibit+lipid+I+biosynthesis&doi=10.1038%2Fncomms14414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Sansanmycin natural product analogues as potent and selective anti-mycobacterials that inhibit lipid I biosynthesis</span></div><div class="casAuthors">Tran Anh T; Watson Emma E; Conroy Trent; Dowman Luke J; Giltrap Andrew M; Payne Richard J; Pujari Venugopal; Mahapatra Sebabrata; Crick Dean C; Pang Angel; Britton Warwick J; Wong Weng Ruh; Linington Roger G; Linington Roger G; Saunders Jessica; Charman Susan A; West Nicholas P; Bugg Timothy D H; Tod Julie; Dowson Christopher G; Roper David I</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14414</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Tuberculosis (TB) is responsible for enormous global morbidity and mortality, and current treatment regimens rely on the use of drugs that have been in use for more than 40 years.  Owing to widespread resistance to these therapies, new drugs are desperately needed to control the TB disease burden.  Herein, we describe the rapid synthesis of analogues of the sansanmycin uridylpeptide natural products that represent promising new TB drug leads.  The compounds exhibit potent and selective inhibition of Mycobacterium tuberculosis, the etiological agent of TB, both in vitro and intracellularly.  The natural product analogues are nanomolar inhibitors of Mtb phospho-MurNAc-pentapeptide translocase, the enzyme responsible for the synthesis of lipid I in mycobacteria.  This work lays the foundation for the development of uridylpeptide natural product analogues as new TB drug candidates that operate through the inhibition of peptidoglycan biosynthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSZtSA02_IoCDToTC3aBaesfW6udTcc2eaaVu1cUMHl9rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czhs12jug%253D%253D&md5=afbd88ba3b02293592ef01d0ae75e8db</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fncomms14414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms14414%26sid%3Dliteratum%253Aachs%26aulast%3DTran%26aufirst%3DA.%2BT.%26aulast%3DWatson%26aufirst%3DE.%2BE.%26aulast%3DPujari%26aufirst%3DV.%26aulast%3DConroy%26aufirst%3DT.%26aulast%3DDowman%26aufirst%3DL.%2BJ.%26aulast%3DGiltrap%26aufirst%3DA.%2BM.%26aulast%3DPang%26aufirst%3DA.%26aulast%3DWong%26aufirst%3DW.%2BR.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26aulast%3DMahapatra%26aufirst%3DS.%26aulast%3DSaunders%26aufirst%3DJ.%2BS.%26aulast%3DCharman%26aufirst%3DA.%26aulast%3DWest%26aufirst%3DN.%2BP.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26aulast%3DTod%26aufirst%3DJ.%26aulast%3DDowson%26aufirst%3DC.%2BG.%26aulast%3DRoper%26aufirst%3DD.%2BI.%26aulast%3DCrick%26aufirst%3DD.%2BC.%26aulast%3DBritton%26aufirst%3DW.%2BJ.%26aulast%3DPayne%26aufirst%3DR.%2BJ.%26atitle%3DSansanmycin%2520natural%2520product%2520analogues%2520as%2520potent%2520and%2520selective%2520anti-mycobacterials%2520that%2520inhibit%2520lipid%2520I%2520biosynthesis%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D14414%26doi%3D10.1038%2Fncomms14414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mashalidis, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaeser, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terasawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katsuyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, D.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-Y.</span></span> <span> </span><span class="NLM_article-title">Chemical logic of MraY inhibition by antibacterial nucleoside natural products</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2917</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-10957-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1038%2Fs41467-019-10957-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=31266949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BB3MzjsFSmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=2917&author=E.+H.+Mashalidisauthor=B.+Kaeserauthor=Y.+Terasawaauthor=A.+Katsuyamaauthor=D.-Y.+Kwonauthor=K.+Leeauthor=J.+Hongauthor=S.+Ichikawaauthor=S.-Y.+Lee&title=Chemical+logic+of+MraY+inhibition+by+antibacterial+nucleoside+natural+products&doi=10.1038%2Fs41467-019-10957-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical logic of MraY inhibition by antibacterial nucleoside natural products</span></div><div class="casAuthors">Mashalidis Ellene H; Kaeser Benjamin; Lee Seok-Yong; Terasawa Yuma; Katsuyama Akira; Ichikawa Satoshi; Kwon Do-Yeon; Hong Jiyong; Lee Kiyoun</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2917</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Novel antibacterial agents are needed to address the emergence of global antibiotic resistance.  MraY is a promising candidate for antibiotic development because it is the target of five classes of naturally occurring nucleoside inhibitors with potent antibacterial activity.  Although these natural products share a common uridine moiety, their core structures vary substantially and they exhibit different activity profiles.  An incomplete understanding of the structural and mechanistic basis of MraY inhibition has hindered the translation of these compounds to the clinic.  Here we present crystal structures of MraY in complex with representative members of the liposidomycin/caprazamycin, capuramycin, and mureidomycin classes of nucleoside inhibitors.  Our structures reveal cryptic druggable hot spots in the shallow inhibitor binding site of MraY that were not previously appreciated.  Structural analyses of nucleoside inhibitor binding provide insights into the chemical logic of MraY inhibition, which can guide novel approaches to MraY-targeted antibiotic design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTttHveiMTXK77NkxioyDXTfW6udTcc2eaaVu1cUMHl9rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzjsFSmuw%253D%253D&md5=78fbcd5ef6013cc5ebbc5797799e57b2</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-10957-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-10957-9%26sid%3Dliteratum%253Aachs%26aulast%3DMashalidis%26aufirst%3DE.%2BH.%26aulast%3DKaeser%26aufirst%3DB.%26aulast%3DTerasawa%26aufirst%3DY.%26aulast%3DKatsuyama%26aufirst%3DA.%26aulast%3DKwon%26aufirst%3DD.-Y.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHong%26aufirst%3DJ.%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DS.-Y.%26atitle%3DChemical%2520logic%2520of%2520MraY%2520inhibition%2520by%2520antibacterial%2520nucleoside%2520natural%2520products%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D2917%26doi%3D10.1038%2Fs41467-019-10957-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosono, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span> <span> </span><span class="NLM_article-title">Total Synthesis of Syringolin A and Improvement of Its Biological Activity</span>. <i>Angew. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">4936</span>– <span class="NLM_lpage">4939</span>, <span class="refDoi"> DOI: 10.1002/ange.201402428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1002%2Fange.201402428" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2014&pages=4936-4939&author=T.+Chibaauthor=H.+Hosonoauthor=K.+Nakagawaauthor=M.+Asakaauthor=H.+Takedaauthor=A.+Matsudaauthor=S.+Ichikawa&title=Total+Synthesis+of+Syringolin+A+and+Improvement+of+Its+Biological+Activity&doi=10.1002%2Fange.201402428"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fange.201402428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fange.201402428%26sid%3Dliteratum%253Aachs%26aulast%3DChiba%26aufirst%3DT.%26aulast%3DHosono%26aufirst%3DH.%26aulast%3DNakagawa%26aufirst%3DK.%26aulast%3DAsaka%26aufirst%3DM.%26aulast%3DTakeda%26aufirst%3DH.%26aulast%3DMatsuda%26aufirst%3DA.%26aulast%3DIchikawa%26aufirst%3DS.%26atitle%3DTotal%2520Synthesis%2520of%2520Syringolin%2520A%2520and%2520Improvement%2520of%2520Its%2520Biological%2520Activity%26jtitle%3DAngew.%2520Chem.%26date%3D2014%26volume%3D126%26spage%3D4936%26epage%3D4939%26doi%3D10.1002%2Fange.201402428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stachyra, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouhss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Heijenoort, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mengin-Lecreulx, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanot, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Beller, D.</span></span> <span> </span><span class="NLM_article-title">Fluorescence detection-based functional assay for high-throughput screening for MraY</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">897</span>– <span class="NLM_lpage">902</span>, <span class="refDoi"> DOI: 10.1128/aac.48.3.897-902.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1128%2FAAC.48.3.897-902.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=14982781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitVGht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2004&pages=897-902&author=T.+Stachyraauthor=C.+Diniauthor=P.+Ferrariauthor=A.+Bouhssauthor=J.+van+Heijenoortauthor=D.+Mengin-Lecreulxauthor=D.+Blanotauthor=J.+Bitonauthor=D.+Le+Beller&title=Fluorescence+detection-based+functional+assay+for+high-throughput+screening+for+MraY&doi=10.1128%2Faac.48.3.897-902.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence detection-based functional assay for high-throughput screening for MraY</span></div><div class="casAuthors">Stachyra, Therese; Dini, Christophe; Ferrari, Paul; Bouhss, Ahmed; Van Heijenoort, Jean; Mengin-Lecreulx, Dominique; Blanot, Didier; Biton, Jacques; Le Beller, Dominique</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">897-902</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">We have developed a novel assay specific to MraY, which catalyzes the first membrane step in the biosynthesis of bacterial cell wall peptidoglycan.  This was accomplished by using UDP-MurNAc-Nε-dansylpentapeptide, a fluorescent deriv. of the MraY nucleotide substrate, and a partially purified prepn. of MraY solubilized from membranes of an Escherichia coli overproducing strain.  Two versions of the assay were developed, one consisting of the high-pressure liq. chromatog. sepn. of the substrate and product (dansylated lipid I) and the other, without sepn. and adapted to the high-throughput format, taking advantage of the different fluorescence properties of the nucleotide and lipid I in the reaction medium.  The latter assay was validated with a set of natural and synthetic MraY inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_YmdYNHFMubVg90H21EOLACvtfcHk0lhao4s2yALZ3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitVGht7o%253D&md5=232a3c03a8e1ea8711e53ffad2279ab3</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1128%2FAAC.48.3.897-902.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.48.3.897-902.2004%26sid%3Dliteratum%253Aachs%26aulast%3DStachyra%26aufirst%3DT.%26aulast%3DDini%26aufirst%3DC.%26aulast%3DFerrari%26aufirst%3DP.%26aulast%3DBouhss%26aufirst%3DA.%26aulast%3Dvan%2BHeijenoort%26aufirst%3DJ.%26aulast%3DMengin-Lecreulx%26aufirst%3DD.%26aulast%3DBlanot%26aufirst%3DD.%26aulast%3DBiton%26aufirst%3DJ.%26aulast%3DLe%2BBeller%26aufirst%3DD.%26atitle%3DFluorescence%2520detection-based%2520functional%2520assay%2520for%2520high-throughput%2520screening%2520for%2520MraY%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2004%26volume%3D48%26spage%3D897%26epage%3D902%26doi%3D10.1128%2Faac.48.3.897-902.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katsuyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yakushiji, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumaru, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span> <span> </span><span class="NLM_article-title">Solid-Phase Modular Synthesis of Park Nucleotide and Lipids I and II Analogues</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">95</span>, <span class="refDoi"> DOI: 10.1248/cpb.c17-00828</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1248%2Fcpb.c17-00828" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=29311516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslOntr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2018&pages=84-95&author=A.+Katsuyamaauthor=K.+Satoauthor=F.+Yakushijiauthor=T.+Matsumaruauthor=S.+Ichikawa&title=Solid-Phase+Modular+Synthesis+of+Park+Nucleotide+and+Lipids+I+and+II+Analogues&doi=10.1248%2Fcpb.c17-00828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Solid-phase modular synthesis of park nucleotide and lipids I and II analogues</span></div><div class="casAuthors">Katsuyama, Akira; Sato, Kousuke; Yakushiji, Fumika; Matsumaru, Takanori; Ichikawa, Satoshi</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">84-95</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">A solid-phase synthesis of Park nucleotide as well as lipids I and II analogs, which is applicable to the synthesis of a range of analogs, is described in this work.  This technique allows highly functionalized macromols. to he modularly labeled.  Multiple steps are used in a short time (4d) with a single purifn. step to synthesize the mols. by solid-phase synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs4edJm7YrbrVg90H21EOLACvtfcHk0liX7rE_osmZrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslOntr%252FL&md5=f3c950215cf73490dbd2d2049eafd6a1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c17-00828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c17-00828%26sid%3Dliteratum%253Aachs%26aulast%3DKatsuyama%26aufirst%3DA.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DYakushiji%26aufirst%3DF.%26aulast%3DMatsumaru%26aufirst%3DT.%26aulast%3DIchikawa%26aufirst%3DS.%26atitle%3DSolid-Phase%2520Modular%2520Synthesis%2520of%2520Park%2520Nucleotide%2520and%2520Lipids%2520I%2520and%2520II%2520Analogues%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2018%26volume%3D66%26spage%3D84%26epage%3D95%26doi%3D10.1248%2Fcpb.c17-00828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashalidis, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-Y.</span></span> <span> </span><span class="NLM_article-title">Structural insights into inhibition of lipid I production in bacterial cell wall synthesis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>533</i></span>,  <span class="NLM_fpage">557</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1038/nature17636</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1038%2Fnature17636" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=27088606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmt1eku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=533&publication_year=2016&pages=557-560&author=B.+C.+Chungauthor=E.+H.+Mashalidisauthor=T.+Taninoauthor=M.+Kimauthor=A.+Matsudaauthor=J.+Hongauthor=S.+Ichikawaauthor=S.-Y.+Lee&title=Structural+insights+into+inhibition+of+lipid+I+production+in+bacterial+cell+wall+synthesis&doi=10.1038%2Fnature17636"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into inhibition of lipid I production in bacterial cell wall synthesis</span></div><div class="casAuthors">Chung, Ben C.; Mashalidis, Ellene H.; Tanino, Tetsuya; Kim, Mijung; Matsuda, Akira; Hong, Jiyong; Ichikawa, Satoshi; Lee, Seok-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">533</span>
        (<span class="NLM_cas:issue">7604</span>),
    <span class="NLM_cas:pages">557-560</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Antibiotic-resistant bacterial infection is a serious threat to public health.  Peptidoglycan biosynthesis is a well-established target for antibiotic development.  MraY (phospho-MurNAc-pentapeptide translocase) catalyzes the first and an essential membrane step of peptidoglycan biosynthesis.  It is considered a very promising target for the development of new antibiotics, as many naturally occurring nucleoside inhibitors with antibacterial activity target this enzyme.  However, antibiotics targeting MraY have not been developed for clin. use, mainly owing to a lack of structural insight into inhibition of this enzyme.  Here, the authors present the crystal structure of MraY from Aquifex aeolicus (MraYAA) in complex with its naturally occurring inhibitor, muraymycin D2 (MD2).  They show that after binding MD2, MraYAA undergoes remarkably large conformational rearrangements near the active site, which lead to the formation of a nucleoside-binding pocket and a peptide-binding site.  MD2 binds the nucleoside-binding pocket like a two-pronged plug inserting into a socket.  Further interactions it makes in the adjacent peptide-binding site anchor MD2 to and enhance its affinity for MraYAA.  Surprisingly, MD2 does not interact with three acidic residues or the Mg2+ cofactor required for catalysis, suggesting that MD2 binds to MraYAA in a manner that overlaps with, but is distinct from, its natural substrate, UDP-MurNAc-pentapeptide.  The authors have detd. the principles of MD2 binding to MraYAA, including how it avoids the need for pyrophosphate and sugar moieties, which are essential features for substrate binding.  The conformational plasticity of MraY could be the reason that it is the target of many structurally distinct inhibitors.  These findings can inform the design of new inhibitors targeting MraY as well as its paralogs, WecA and TarO.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosfgI8BNe27LVg90H21EOLACvtfcHk0liX7rE_osmZrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmt1eku70%253D&md5=4379cae27f228869c9439ecd9696f123</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature17636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17636%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DB.%2BC.%26aulast%3DMashalidis%26aufirst%3DE.%2BH.%26aulast%3DTanino%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DMatsuda%26aufirst%3DA.%26aulast%3DHong%26aufirst%3DJ.%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DS.-Y.%26atitle%3DStructural%2520insights%2520into%2520inhibition%2520of%2520lipid%2520I%2520production%2520in%2520bacterial%2520cell%2520wall%2520synthesis%26jtitle%3DNature%26date%3D2016%26volume%3D533%26spage%3D557%26epage%3D560%26doi%3D10.1038%2Fnature17636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillespie, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, D.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-Y.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of MraY, an essential membrane enzyme for bacterial cell wall synthesis</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">1012</span>– <span class="NLM_lpage">1016</span>, <span class="refDoi"> DOI: 10.1126/science.1236501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1126%2Fscience.1236501" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=23990562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlaktbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=1012-1016&author=B.+C.+Chungauthor=J.+Zhaoauthor=R.+A.+Gillespieauthor=D.-Y.+Kwonauthor=Z.+Guanauthor=J.+Hongauthor=P.+Zhouauthor=S.-Y.+Lee&title=Crystal+structure+of+MraY%2C+an+essential+membrane+enzyme+for+bacterial+cell+wall+synthesis&doi=10.1126%2Fscience.1236501"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of MraY, an Essential Membrane Enzyme for Bacterial Cell Wall Synthesis</span></div><div class="casAuthors">Chung, Ben C.; Zhao, Jinshi; Gillespie, Robert A.; Kwon, Do-Yeon; Guan, Ziqiang; Hong, Jiyong; Zhou, Pei; Lee, Seok-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6149</span>),
    <span class="NLM_cas:pages">1012-1016</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">MraY (phospho-MurNAc-pentapeptide translocase) is an integral membrane enzyme that catalyzes an essential step of bacterial cell wall biosynthesis: the transfer of the peptidoglycan precursor phospho-MurNAc-pentapeptide to the lipid carrier undecaprenyl phosphate.  MraY has long been considered a promising target for the development of antibiotics, but the lack of a structure has hindered mechanistic understanding of this crit. enzyme and the enzyme superfamily in general.  The superfamily includes enzymes involved in bacterial lipopolysaccharide/teichoic acid formation and eukaryotic N-linked glycosylation, modifications that are central in many biol. processes.  We present the crystal structure of MraY from Aquifex aeolicus (MraYAA) at 3.3 Å resoln., which allows us to visualize the overall architecture, locate Mg2+ within the active site, and provide a structural basis of catalysis for this class of enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry8oc1FCGbcbVg90H21EOLACvtfcHk0ljUcEiI7fzN2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlaktbnE&md5=3c9d143b126c25b0430e1cf70e5f960a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fscience.1236501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1236501%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DB.%2BC.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DGillespie%26aufirst%3DR.%2BA.%26aulast%3DKwon%26aufirst%3DD.-Y.%26aulast%3DGuan%26aufirst%3DZ.%26aulast%3DHong%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DS.-Y.%26atitle%3DCrystal%2520structure%2520of%2520MraY%252C%2520an%2520essential%2520membrane%2520enzyme%2520for%2520bacterial%2520cell%2520wall%2520synthesis%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D1012%26epage%3D1016%26doi%3D10.1126%2Fscience.1236501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waterhouse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bienert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Studer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tauriello, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gumienny, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heer, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Beer, T. A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rempfer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordoli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepore, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwede, T.</span></span> <span> </span><span class="NLM_article-title">SWISS-MODEL: homology modelling of protein structures and complexes</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">W296</span>– <span class="NLM_lpage">W303</span>, <span class="refDoi"> DOI: 10.1093/nar/gky427</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1093%2Fnar%2Fgky427" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=29788355" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVyru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=W296-W303&author=A.+Waterhouseauthor=M.+Bertoniauthor=S.+Bienertauthor=G.+Studerauthor=G.+Taurielloauthor=R.+Gumiennyauthor=F.+T.+Heerauthor=T.+A.+P.+de+Beerauthor=C.+Rempferauthor=L.+Bordoliauthor=R.+Leporeauthor=T.+Schwede&title=SWISS-MODEL%3A+homology+modelling+of+protein+structures+and+complexes&doi=10.1093%2Fnar%2Fgky427"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">SWISS-MODEL: homology modelling of protein structures and complexes</span></div><div class="casAuthors">Waterhouse, Andrew; Bertoni, Martino; Bienert, Stefan; Studer, Gabriel; Tauriello, Gerardo; Gumienny, Rafal; Heer, Florian T.; de Beer, Tjaart A. P.; Rempfer, Christine; Bordoli, Lorenza; Lepore, Rosalba; Schwede, Torsten</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W296-W303</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Homol. modeling has matured into an important technique in structural biol., significantly contributing to narrowing the gap between known protein sequences and exptl. detd. structures.  Fully automated workflows and servers simplify and streamline the homol. modeling process, also allowing users without a specific computational expertise to generate reliable protein models and have easy access to modeling results, their visualization and interpretation.  Here, we present an update to the SWISS-MODEL server, which pioneered the field of automated modeling 25 years ago and been continuously further developed.  Recently, its functionality has been extended to the modeling of homo- and heteromeric complexes.  Starting from the amino acid sequences of the interacting proteins, both the stoichiometry and the overall structure of the complex are inferred by homol. modeling.  Other major improvements include the implementation of a new modeling engine, ProMod3 and the introduction a new local model quality estn. method, QMEANDisCo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf9ZhQAzPVe7Vg90H21EOLACvtfcHk0lg65PXqeqqjPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVyru70%253D&md5=651b55ade2d4f354f5d6699f318424d0</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky427%26sid%3Dliteratum%253Aachs%26aulast%3DWaterhouse%26aufirst%3DA.%26aulast%3DBertoni%26aufirst%3DM.%26aulast%3DBienert%26aufirst%3DS.%26aulast%3DStuder%26aufirst%3DG.%26aulast%3DTauriello%26aufirst%3DG.%26aulast%3DGumienny%26aufirst%3DR.%26aulast%3DHeer%26aufirst%3DF.%2BT.%26aulast%3Dde%2BBeer%26aufirst%3DT.%2BA.%2BP.%26aulast%3DRempfer%26aufirst%3DC.%26aulast%3DBordoli%26aufirst%3DL.%26aulast%3DLepore%26aufirst%3DR.%26aulast%3DSchwede%26aufirst%3DT.%26atitle%3DSWISS-MODEL%253A%2520homology%2520modelling%2520of%2520protein%2520structures%2520and%2520complexes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3DW296%26epage%3DW303%26doi%3D10.1093%2Fnar%2Fgky427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takeoka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekiguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonezawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakagami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togame, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Expansion of antibacterial spectrum of muraymycins toward <i>Pseudomonas aeruginosa</i></span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">556</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1021/ml5000096</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml5000096" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjs1GqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=556-560&author=Y.+Takeokaauthor=T.+Taninoauthor=M.+Sekiguchiauthor=S.+Yonezawaauthor=M.+Sakagamiauthor=F.+Takahashiauthor=H.+Togameauthor=Y.+Tanakaauthor=H.+Takemotoauthor=S.+Ichikawaauthor=A.+Matsuda&title=Expansion+of+antibacterial+spectrum+of+muraymycins+toward+Pseudomonas+aeruginosa&doi=10.1021%2Fml5000096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Expansion of antibacterial spectrum of muraymycins toward Pseudomonas aeruginosa</span></div><div class="casAuthors">Takeoka, Yusuke; Tanino, Tetsuya; Sekiguchi, Mitsuaki; Yonezawa, Shuji; Sakagami, Masahiro; Takahashi, Fumiyo; Togame, Hiroko; Tanaka, Yoshikazu; Takemoto, Hiroshi; Ichikawa, Satoshi; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">556-560</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">It is urgent to develop novel anti-Pseudomonas agents that should also be active against multidrug-resistant P. aeruginosa.  Expanding the antibacterial spectrum of muraymycins toward P. aeruginosa was investigated by the systematic structure-activity relationship study.  It was revealed that two functional groups, a lipophilic side chain and a guanidino group, at the accessory moiety of muraymycins were important for the anti-Pseudomonas activity, and analog 29 exhibited antibacterial activity against a range of P. aeruginosa strains with the min. inhibitory concn. values of 4-8 μg/mL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPDfz8OY4rRbVg90H21EOLACvtfcHk0lg65PXqeqqjPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjs1GqsL8%253D&md5=58a7f2a7d0368e0bf56691c1e5026397</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fml5000096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5000096%26sid%3Dliteratum%253Aachs%26aulast%3DTakeoka%26aufirst%3DY.%26aulast%3DTanino%26aufirst%3DT.%26aulast%3DSekiguchi%26aufirst%3DM.%26aulast%3DYonezawa%26aufirst%3DS.%26aulast%3DSakagami%26aufirst%3DM.%26aulast%3DTakahashi%26aufirst%3DF.%26aulast%3DTogame%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DTakemoto%26aufirst%3DH.%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DExpansion%2520of%2520antibacterial%2520spectrum%2520of%2520muraymycins%2520toward%2520Pseudomonas%2520aeruginosa%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D556%26epage%3D560%26doi%3D10.1021%2Fml5000096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Dabbagh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mengin-Lecreulx, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouhss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Mechanistic analysis of muraymycin analogues: a guide to the design of MraY inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8421</span>– <span class="NLM_lpage">8439</span>, <span class="refDoi"> DOI: 10.1021/jm200906r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200906r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOlt73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8421-8439&author=T.+Taninoauthor=B.+Al-Dabbaghauthor=D.+Mengin-Lecreulxauthor=A.+Bouhssauthor=H.+Oyamaauthor=S.+Ichikawaauthor=A.+Matsuda&title=Mechanistic+analysis+of+muraymycin+analogues%3A+a+guide+to+the+design+of+MraY+inhibitors&doi=10.1021%2Fjm200906r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic Analysis of Muraymycin Analogues: A Guide to the Design of MraY Inhibitors</span></div><div class="casAuthors">Tanino, Tetsuya; Al-Dabbagh, Bayan; Mengin-Lecreulx, Dominique; Bouhss, Ahmed; Oyama, Hiroshi; Ichikawa, Satoshi; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8421-8439</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The systematic structure-activity relationship (SAR) of the muraymycins (MRYs) using an Ugi four-component reaction (U4CR) was investigated.  The impact of the lipophilic substituent on antibacterial activity was significant, and the analogs 8 and 9 having a lipophilic side chain exhibited good activity against a range of Gram-pos. bacterial pathogens, including MRSA and VRE.  Further investigation of compds. 8 and 9 revealed these analogs to be selective inhibitors of the MraY transferase and nontoxic to HepG2 cells.  The SAR of the accessory urea-peptide moiety indicated that it could be simplified.  Our SAR study of the MRYs suggests a probable mechanism for inhibition of the MraY, where the inner moiety of the urea-dipeptide motif interacts with the carbohydrate recognition domain in the cytoplasmic loop 5.  The predicted binding model would provide further direction toward the design of potent MraY inhibitors.  This study has set the stage for the generation of novel antibacterial "lead" compds. based on MRYs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwPj7zgQroc7Vg90H21EOLACvtfcHk0lgxXfv3085-Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOlt73O&md5=82a6f8ac626297cc8df1a72c05325902</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm200906r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200906r%26sid%3Dliteratum%253Aachs%26aulast%3DTanino%26aufirst%3DT.%26aulast%3DAl-Dabbagh%26aufirst%3DB.%26aulast%3DMengin-Lecreulx%26aufirst%3DD.%26aulast%3DBouhss%26aufirst%3DA.%26aulast%3DOyama%26aufirst%3DH.%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DMechanistic%2520analysis%2520of%2520muraymycin%2520analogues%253A%2520a%2520guide%2520to%2520the%2520design%2520of%2520MraY%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8421%26epage%3D8439%26doi%3D10.1021%2Fjm200906r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Brandish, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnham, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lonsdale, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southgate, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Slow binding inhibition of phospho-<i>N</i>-acetylmuramyl-pentapeptide-translocase (<i>Escherichia coli</i>) by mureidomycin A</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>271</i></span>,  <span class="NLM_fpage">7609</span>– <span class="NLM_lpage">7614</span>, <span class="refDoi"> DOI: 10.1074/jbc.271.13.7609</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1074%2Fjbc.271.13.7609" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=8631795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK28XhvFyksbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=7609-7614&author=P.+E.+Brandishauthor=M.+K.+Burnhamauthor=J.+T.+Lonsdaleauthor=R.+Southgateauthor=M.+Inukaiauthor=T.+D.+H.+Bugg&title=Slow+binding+inhibition+of+phospho-N-acetylmuramyl-pentapeptide-translocase+%28Escherichia+coli%29+by+mureidomycin+A&doi=10.1074%2Fjbc.271.13.7609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49aR"><div class="casContent"><span class="casTitleNuber">49a</span><div class="casTitle"><span class="NLM_cas:atitle">Slow binding inhibition of phospho-N-acetylmuramyl-pentapeptide-translocase (Escherichia coli) by mureidomycin A</span></div><div class="casAuthors">Brandish, Philip E.; Burnham, Martin K.; Lonsdale, John T.; Southgate, Robert; Inukai, Masatoshi; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">271</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">7609-14</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Enzymes of the membrane cycle of reactions in bacterial peptidoglycan biosynthesis remain as unexploited potential targets for antibacterial agents.  The first of these enzymes, phospho-N-acetylmuramyl-pentapeptide-translocase (EC 2.7.8.13), has been overexpressed in Escherichia coli and solubilized from particulate fractions.  The work of W. A. Weppner and F. C. Neuhaus ((1977) J. Biol. Chem. 252, 2296-2303) has been extended to establish a usable routine fluorescence-based continuous assay for solubilized prepns.  This assay has been used in the characterization of the natural product, mureidomycin A as a potent slow binding inhibitor of the enzyme with Ki and Ki* of 36 nM and 2 nM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJBHjL8B2BZrVg90H21EOLACvtfcHk0lgxXfv3085-Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvFyksbo%253D&md5=899edaad9b1d4af57dd63d5ac91fa696</span></div><a href="/servlet/linkout?suffix=cit49a&amp;dbid=16384&amp;doi=10.1074%2Fjbc.271.13.7609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.271.13.7609%26sid%3Dliteratum%253Aachs%26aulast%3DBrandish%26aufirst%3DP.%2BE.%26aulast%3DBurnham%26aufirst%3DM.%2BK.%26aulast%3DLonsdale%26aufirst%3DJ.%2BT.%26aulast%3DSouthgate%26aufirst%3DR.%26aulast%3DInukai%26aufirst%3DM.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DSlow%2520binding%2520inhibition%2520of%2520phospho-N-acetylmuramyl-pentapeptide-translocase%2520%2528Escherichia%2520coli%2529%2520by%2520mureidomycin%2520A%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26spage%3D7609%26epage%3D7614%26doi%3D10.1074%2Fjbc.271.13.7609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit49b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Brandish, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southgate, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lonsdale, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D.</span></span> <span> </span><span class="NLM_article-title">Modes of action of tunicamycin, liposidomycin B, and mureidomycin A: inhibition of phospho-<i>N</i>-acetylmuramyl-pentapeptide translocase from <i>Escherichia coli</i></span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1640</span>– <span class="NLM_lpage">1644</span>, <span class="refDoi"> DOI: 10.1128/aac.40.7.1640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1128%2FAAC.40.7.1640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=8807054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK28XktVaksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1996&pages=1640-1644&author=P.+E.+Brandishauthor=K.+I.+Kimuraauthor=M.+Inukaiauthor=R.+Southgateauthor=J.+T.+Lonsdaleauthor=T.+D.+Bugg&title=Modes+of+action+of+tunicamycin%2C+liposidomycin+B%2C+and+mureidomycin+A%3A+inhibition+of+phospho-N-acetylmuramyl-pentapeptide+translocase+from+Escherichia+coli&doi=10.1128%2Faac.40.7.1640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49bR"><div class="casContent"><span class="casTitleNuber">49b</span><div class="casTitle"><span class="NLM_cas:atitle">Modes of action of tunicamycin, liposidomycin B, and mureidomycin A: inhibition of phospho-N-acetylmuramyl-pentapeptide translocase from Escherichia coli</span></div><div class="casAuthors">Brandish, Philip E.; Kimura, Ken-Ichi; Inukai, Masatoshi; Southgate, Robert; Lonsdale, John T.; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1640-1644</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Using a continuous fluorescence-based enzyme assay, the authors characterized the antibacterial agents, tunicamycin and liposidomycin B, as inhibitors of solubilized E. coli phospho-N-acetylmuramyl-pentapeptide translocase.  Tunicamycin exhibited reversible inhibition (Ki = 0.55 μM) which was noncompetitive with respect to the lipid acceptor substrate, dodecaprenyl phosphate (I), and competitive with respect to the fluorescent substrate analog, dansyl-UDPMurNAc-pentapeptide (II).  Liposidomycin B exhibited slow-binding inhibition (Ki = 80 nM) which was competitive with respect to I and noncompetitive with respect to II.  Mureidomycin A exhibited slow-binding inhibition which was competitive with respect to both I and II.  The results provided insight into the mol. mechanisms of action of these classes of nucleoside antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonrQ_H-_u_OLVg90H21EOLACvtfcHk0ljhT5WV7ABA2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XktVaksLw%253D&md5=b6172f4c155922ee6389c39265987b1a</span></div><a href="/servlet/linkout?suffix=cit49b&amp;dbid=16384&amp;doi=10.1128%2FAAC.40.7.1640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.40.7.1640%26sid%3Dliteratum%253Aachs%26aulast%3DBrandish%26aufirst%3DP.%2BE.%26aulast%3DKimura%26aufirst%3DK.%2BI.%26aulast%3DInukai%26aufirst%3DM.%26aulast%3DSouthgate%26aufirst%3DR.%26aulast%3DLonsdale%26aufirst%3DJ.%2BT.%26aulast%3DBugg%26aufirst%3DT.%2BD.%26atitle%3DModes%2520of%2520action%2520of%2520tunicamycin%252C%2520liposidomycin%2520B%252C%2520and%2520mureidomycin%2520A%253A%2520inhibition%2520of%2520phospho-N-acetylmuramyl-pentapeptide%2520translocase%2520from%2520Escherichia%2520coli%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1996%26volume%3D40%26spage%3D1640%26epage%3D1644%26doi%3D10.1128%2Faac.40.7.1640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gentle, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Role of the enamide linkage of nucleoside antibiotic mureidomycin A: synthesis and reactivity of enamide-containing analogues</span>. <i>J. Chem. Soc., Perkin Trans. 1</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_fpage">1279</span>– <span class="NLM_lpage">1286</span>, <span class="refDoi"> DOI: 10.1039/a901279f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2Fa901279f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK1MXjtVegtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1999&pages=1279-1286&author=C.+A.+Gentleauthor=T.+D.+H.+Bugg&title=Role+of+the+enamide+linkage+of+nucleoside+antibiotic+mureidomycin+A%3A+synthesis+and+reactivity+of+enamide-containing+analogues&doi=10.1039%2Fa901279f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the enamide linkage of nucleoside antibiotic mureidomycin A: synthesis and reactivity of enamide-containing analogues</span></div><div class="casAuthors">Gentle, Caragh A.; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Perkin Transactions 1: Organic and Bio-Organic Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1279-1286</span>CODEN:
                <span class="NLM_cas:coden">JCPRB4</span>;
        ISSN:<span class="NLM_cas:issn">0300-922X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The reactivity of an unusual enamide functional group in the nucleoside antibiotic mureidomycin A (MRD A) has been investigated by synthesis of enamide-contg. analogs.  Enamides based on 2-methoxyethylamine and tetrahydrofurfurylamine were found to be unstable and reactive, whereas a uridine-based analog showed high stability and low reactivity.  Samples of mureidomycin A and the uridine-based analog were found to be stable towards acid-catalyzed tautomerization and nucleophilic attack.  The lack of reactivity and lack of enzyme inhibition shown by the uridine-based analog are not consistent with the involvement of the enamide group in slow-binding inhibition of translocase I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0iG8FEyWJ57Vg90H21EOLACvtfcHk0ljhT5WV7ABA2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtVegtLg%253D&md5=cb9c92b222070a89fde6852d52106ca4</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1039%2Fa901279f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fa901279f%26sid%3Dliteratum%253Aachs%26aulast%3DGentle%26aufirst%3DC.%2BA.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DRole%2520of%2520the%2520enamide%2520linkage%2520of%2520nucleoside%2520antibiotic%2520mureidomycin%2520A%253A%2520synthesis%2520and%2520reactivity%2520of%2520enamide-containing%2520analogues%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25201%26date%3D1999%26spage%3D1279%26epage%3D1286%26doi%3D10.1039%2Fa901279f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gotoh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotoh, N.</span></span> <span> </span><span class="NLM_article-title">Intrinsic resistance of <i>Escherichia coli</i> to mureidomycin A and C due to expression of the multidrug efflux system AcrAB-TolC: comparison with the efflux systems of mureidomycin-susceptible <i>Pseudomonas aeruginosa</i></span>. <i>J. Infect. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">103</span>, <span class="refDoi"> DOI: 10.1007/s10156-002-0205-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1007%2Fs10156-002-0205-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=12673418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BD3s7lslehsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2003&pages=101-103&author=N.+Gotohauthor=T.+Murataauthor=T.+Ozakiauthor=T.+Kimuraauthor=A.+Kondoauthor=T.+Nishinoauthor=N.+Gotoh&title=Intrinsic+resistance+of+Escherichia+coli+to+mureidomycin+A+and+C+due+to+expression+of+the+multidrug+efflux+system+AcrAB-TolC%3A+comparison+with+the+efflux+systems+of+mureidomycin-susceptible+Pseudomonas+aeruginosa&doi=10.1007%2Fs10156-002-0205-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Intrinsic resistance of Escherichia coli to mureidomycin A and C due to expression of the multidrug efflux system AcrAB-TolC: comparison with the efflux systems of mureidomycin-susceptible Pseudomonas aeruginosa</span></div><div class="casAuthors">Gotoh Naomasa; Murata Takeshi; Ozaki Toru; Kimura Tadashi; Kondo Akiko; Nishino Takeshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of infection and chemotherapy : official journal of the Japan Society of Chemotherapy</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">101-3</span>
        ISSN:<span class="NLM_cas:issn">1341-321X</span>.
    </div><div class="casAbstract">Intrinsic resistance to mureidomycin is shown in Escherichia coli.  This is in contrast to Pseudomonas aeruginosa, which generally displays intrinsic resistance to a variety of antimicrobial agents, but not to mureidomycin.  Isogenic efflux system mutants from both species were subjected to antibiotic susceptibility tests.  These studies showed that the differences regarding the susceptibility of E. coli and P. aeruginosa to mureidomycin A and C may be explained by the expression of efflux systems that mediate resistance to mureidomycin A and C.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTQeUH2h_3phkmhnSqMnvzDfW6udTcc2ea37JB1bDlTcLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3s7lslehsA%253D%253D&md5=2f466331e28f4023a830b27edd717468</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2Fs10156-002-0205-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10156-002-0205-2%26sid%3Dliteratum%253Aachs%26aulast%3DGotoh%26aufirst%3DN.%26aulast%3DMurata%26aufirst%3DT.%26aulast%3DOzaki%26aufirst%3DT.%26aulast%3DKimura%26aufirst%3DT.%26aulast%3DKondo%26aufirst%3DA.%26aulast%3DNishino%26aufirst%3DT.%26aulast%3DGotoh%26aufirst%3DN.%26atitle%3DIntrinsic%2520resistance%2520of%2520Escherichia%2520coli%2520to%2520mureidomycin%2520A%2520and%2520C%2520due%2520to%2520expression%2520of%2520the%2520multidrug%2520efflux%2520system%2520AcrAB-TolC%253A%2520comparison%2520with%2520the%2520efflux%2520systems%2520of%2520mureidomycin-susceptible%2520Pseudomonas%2520aeruginosa%26jtitle%3DJ.%2520Infect.%2520Chemother.%26date%3D2003%26volume%3D9%26spage%3D101%26epage%3D103%26doi%3D10.1007%2Fs10156-002-0205-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodolis, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mihalyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ducho, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eitel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gust, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gos, R. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Mechanism of action of the uridyl peptide antibiotics: an unexpected link to a protein-protein interaction site in translocase MraY</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">13023</span>– <span class="NLM_lpage">13025</span>, <span class="refDoi"> DOI: 10.1039/c4cc06516f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2FC4CC06516F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=25222373" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFWjurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=13023-13025&author=M.+T.+Rodolisauthor=A.+Mihalyiauthor=C.+Duchoauthor=K.+Eitelauthor=B.+Gustauthor=R.+J.+M.+Gosauthor=T.+D.+H.+Bugg&title=Mechanism+of+action+of+the+uridyl+peptide+antibiotics%3A+an+unexpected+link+to+a+protein-protein+interaction+site+in+translocase+MraY&doi=10.1039%2Fc4cc06516f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of action of the uridyl peptide antibiotics: an unexpected link to a protein-protein interaction site in translocase MraY</span></div><div class="casAuthors">Rodolis, Maria T.; Mihalyi, Agnes; Ducho, Christian; Eitel, Kornelia; Gust, Bertolt; Goss, Rebecca J. M.; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">86</span>),
    <span class="NLM_cas:pages">13023-13025</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The pacidamycin and muraymycin uridyl peptide antibiotics show some structural resemblance to an Arg-Trp-x-x-Trp (RWxxW) sequence motif in bacteriophage .vphi.X174 protein E that mediates a protein-protein interaction between protein E and E. coli translocase MraY.  Members of the uridyl peptide antibiotic (UPA) class, and a synthetic uridine-peptide analog, were found to show reduced levels of inhibition to F288L or E287A mutant MraY enzymes, implying that the UPAs interact at this extracellular site as part of the enzyme inhibition mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoftpmB1S2-KbVg90H21EOLACvtfcHk0lgs8STBvSkumw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFWjurzN&md5=6bf1f9e17e5c2629af8aa1321cef8265</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1039%2FC4CC06516F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4CC06516F%26sid%3Dliteratum%253Aachs%26aulast%3DRodolis%26aufirst%3DM.%2BT.%26aulast%3DMihalyi%26aufirst%3DA.%26aulast%3DDucho%26aufirst%3DC.%26aulast%3DEitel%26aufirst%3DK.%26aulast%3DGust%26aufirst%3DB.%26aulast%3DGos%26aufirst%3DR.%2BJ.%2BM.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DMechanism%2520of%2520action%2520of%2520the%2520uridyl%2520peptide%2520antibiotics%253A%2520an%2520unexpected%2520link%2520to%2520a%2520protein-protein%2520interaction%2520site%2520in%2520translocase%2520MraY%26jtitle%3DChem.%2520Commun.%26date%3D2014%26volume%3D50%26spage%3D13023%26epage%3D13025%26doi%3D10.1039%2Fc4cc06516f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roof, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R.</span></span> <span> </span><span class="NLM_article-title">Genetic evidence that the bacteriophage ϕX174 lysis protein inhibits cell wall synthesis</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">4297</span>– <span class="NLM_lpage">4302</span>, <span class="refDoi"> DOI: 10.1073/pnas.97.8.4297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1073%2Fpnas.97.8.4297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10760296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3cXislSnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=4297-4302&author=T.+G.+Bernhardtauthor=W.+D.+Roofauthor=R.+Young&title=Genetic+evidence+that+the+bacteriophage+%CF%95X174+lysis+protein+inhibits+cell+wall+synthesis&doi=10.1073%2Fpnas.97.8.4297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic evidence that the bacteriophage φX174 lysis protein inhibits cell wall synthesis</span></div><div class="casAuthors">Bernhardt, Thomas G.; Roof, William D.; Young, Ry</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4297-4302</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein E, a 91-residue membrane protein of φX174, causes lysis of the host in a growth-dependent manner reminiscent of cell wall antibiotics, suggesting E acts by inhibiting peptidoglycan synthesis.  A search for the cellular target of E previously isolated recessive mutations in the host gene slyD (sensitivity to lysis) that block the lytic effects of E.  The role of slyD, which encodes a FK506 binding protein-type peptidyl-prolyl cis-trans isomerase, is not fully understood.  However, E mutants referred to as Epos (plates on slyD) lack a slyD requirement, indicating that slyD is not crucial for lysis.  To identify the gene encoding the cellular target, survivors of Epos were selected.  This study describes the isolation of dominant mutations in the essential host gene mraY that result in a general lysis-defective phenotype.  Gene mraY encodes translocase I, which catalyzes the formation of the 1st lipid-linked intermediate in cell wall biosynthesis.  The isolation of these lysis-defective mutants supports a model in which translocase I is the cellular target of E and that inhibition of cell wall synthesis is the mechanism of lysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiesl_-CBo6LVg90H21EOLACvtfcHk0lgs8STBvSkumw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXislSnsrg%253D&md5=ad25239127b702f58106839b08df5db6</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1073%2Fpnas.97.8.4297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.97.8.4297%26sid%3Dliteratum%253Aachs%26aulast%3DBernhardt%26aufirst%3DT.%2BG.%26aulast%3DRoof%26aufirst%3DW.%2BD.%26aulast%3DYoung%26aufirst%3DR.%26atitle%3DGenetic%2520evidence%2520that%2520the%2520bacteriophage%2520%25CF%2595X174%2520lysis%2520protein%2520inhibits%2520cell%2520wall%2520synthesis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2000%26volume%3D97%26spage%3D4297%26epage%3D4302%26doi%3D10.1073%2Fpnas.97.8.4297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5400"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5400'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit5400"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsuan, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Carbacaprazamycins: chemically stable analogues of the caprazamycin nucleoside antibiotics</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">151</span>– <span class="NLM_lpage">156</span>, <span class="refDoi"> DOI: 10.1021/id5000376</span> </span><div class="citationLinks">[<a href="/doi/10.1021/id5000376" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5400/cit5400&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=151-156&author=S.+Ichikawaauthor=M.+Yamaguchiauthor=L.+S.+Hsuanauthor=Y.+Katoauthor=A.+Matsuda&title=Carbacaprazamycins%3A+chemically+stable+analogues+of+the+caprazamycin+nucleoside+antibiotics&doi=10.1021%2Fid5000376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5400R"><div class="casContent"><span class="casTitleNuber">5400</span><div class="casTitle"><span class="NLM_cas:atitle">Carbacaprazamycins: Chemically Stable Analogues of the Caprazamycin Nucleoside Antibiotics</span></div><div class="casAuthors">Ichikawa, Satoshi; Yamaguchi, Mayumi; Hsuan, Lee Shang; Kato, Yuta; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">151-156</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Carbacaprazamycins, which are chem. stable analogs of caprazamycins, were designed and synthesized.  These analogs (I) were active against drug-resistant bacterial pathogens such as methicillin-resistant Staphylococcus aureus and vancomycin-resistant enterococci, and their activities were comparable to those of the parent caprazamycins.  The effect of treatment with carbacaprazamycin on morphol. changes in S. aureus indicated that the mode of action was completely different from those of existing peptidoglycan inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8wPaC80lQfbVg90H21EOLACvtfcHk0lhNByKxUOtnBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVahsbg%253D&md5=9caed0fec927f4f10a0e1435660eb746</span></div><a href="/servlet/linkout?suffix=cit5400&amp;dbid=16384&amp;doi=10.1021%2Fid5000376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fid5000376%26sid%3Dliteratum%253Aachs%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DYamaguchi%26aufirst%3DM.%26aulast%3DHsuan%26aufirst%3DL.%2BS.%26aulast%3DKato%26aufirst%3DY.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DCarbacaprazamycins%253A%2520chemically%2520stable%2520analogues%2520of%2520the%2520caprazamycin%2520nucleoside%2520antibiotics%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2015%26volume%3D1%26spage%3D151%26epage%3D156%26doi%3D10.1021%2Fid5000376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struck, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R.</span></span> <span> </span><span class="NLM_article-title">The lysis protein E of ϕX174 is a specific inhibitor of the MraY-catalyzed step in peptidoglycan synthesis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">6093</span>– <span class="NLM_lpage">6097</span>, <span class="refDoi"> DOI: 10.1074/jbc.m007638200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1074%2Fjbc.M007638200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=11078734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhslShsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=6093-6097&author=T.+G.+Bernhardtauthor=D.+K.+Struckauthor=R.+Young&title=The+lysis+protein+E+of+%CF%95X174+is+a+specific+inhibitor+of+the+MraY-catalyzed+step+in+peptidoglycan+synthesis&doi=10.1074%2Fjbc.m007638200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The lysis protein E of φX174 is a specific inhibitor of the MraY-catalyzed step in peptidoglycan synthesis</span></div><div class="casAuthors">Bernhardt, Thomas G.; Struck, Douglas K.; Young, Ry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">6093-6097</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Coliphage φX174 encodes a single lysis protein, E, a 91-amino acid membrane protein.  Dominant mutations have been isolated in the host gene mraY that confer E resistance.  MraY encodes translocase I, which catalyzes the formation of the first lipid intermediate in bacterial cell wall synthesis, suggesting a model in which E inhibits MraY and promotes cell lysis in a manner analogous to cell wall synthesis inhibitors like penicillin.  To test this model biochem., the authors monitored the effect of E on cell wall synthesis in vivo and in vitro.  The authors find that expression of Emyc, encoding an epitope-tagged E protein, from a multicopy plasmid inhibits the incorporation of [3H]diaminopimelic acid into cell wall and leads to a profile of labeled precursors consistent with MraY inhibition.  Moreover, the authors find that membranes isolated after Emyc expression are drastically reduced in MraY activity, whereas the activity of Rfe, an enzyme in the same superfamily, was unaffected.  The authors therefore conclude that E is indeed a cell wall synthesis inhibitor and that this inhibition results from a specific block at the MraY-catalyzed step in the pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr0ez8EiSHrrVg90H21EOLACvtfcHk0lhNByKxUOtnBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhslShsLw%253D&md5=35248473fdbc463b0550188d7c5c76c8</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M007638200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M007638200%26sid%3Dliteratum%253Aachs%26aulast%3DBernhardt%26aufirst%3DT.%2BG.%26aulast%3DStruck%26aufirst%3DD.%2BK.%26aulast%3DYoung%26aufirst%3DR.%26atitle%3DThe%2520lysis%2520protein%2520E%2520of%2520%25CF%2595X174%2520is%2520a%2520specific%2520inhibitor%2520of%2520the%2520MraY-catalyzed%2520step%2520in%2520peptidoglycan%2520synthesis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D6093%26epage%3D6097%26doi%3D10.1074%2Fjbc.m007638200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Clinical and Laboratory Standards
Institute</span>.  <i>Performance Standards for
Antimicrobial Susceptibility Testing; 27th Informational Supplement.
CLSI M100-S27</i>; <span class="NLM_publisher-name">Clinical and Laboratory Standards
Institute</span>: <span class="NLM_publisher-loc">Wayne, PA</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=Clinical+and+Laboratory+Standards%0AInstitute&title=Performance+Standards+for%0AAntimicrobial+Susceptibility+Testing%3B+27th+Informational+Supplement.%0ACLSI+M100-S27"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DPerformance%2520Standards%2520for%250AAntimicrobial%2520Susceptibility%2520Testing%253B%252027th%2520Informational%2520Supplement.%250ACLSI%2520M100-S27%26pub%3DClinical%2520and%2520Laboratory%2520Standards%250AInstitute%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanizawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminuma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arita, M.</span></span> <span> </span><span class="NLM_article-title">DFAST and DAGA: web-based integrated genome annotation tools and resources</span>. <i>Biosci. Microbiota, Food Health</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.12938/bmfh.16-003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.12938%2Fbmfh.16-003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=27867804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpsF2msrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=173-184&author=Y.+Tanizawaauthor=T.+Fujisawaauthor=E.+Kaminumaauthor=Y.+Nakamuraauthor=M.+Arita&title=DFAST+and+DAGA%3A+web-based+integrated+genome+annotation+tools+and+resources&doi=10.12938%2Fbmfh.16-003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">DFAST and DAGA: web-based integrated genome annotation tools and resources</span></div><div class="casAuthors">Tanizawa, Yasuhiro; Fujisawa, Takatomo; Kaminuma, Eli; Nakamura, Yasukazu; Arita, Masanori</div><div class="citationInfo"><span class="NLM_cas:title">Bioscience of Microbiota, Food and Health</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">173-184</span>CODEN:
                <span class="NLM_cas:coden">BMFHB8</span>;
        ISSN:<span class="NLM_cas:issn">2186-3342</span>.
    
            (<span class="NLM_cas:orgname">BMFH Press</span>)
        </div><div class="casAbstract">Quality assurance and correct taxonomic affiliation of data submitted to public sequence databases have been an everlasting problem.  The DDBJ Fast Annotation and Submission Tool (DFAST) is a newly developed genome annotation pipeline with quality and taxonomy assessment tools.  To enable annotation of ready-to-submit quality, we also constructed curated ref. protein databases tailored for lactic acid bacteria.  DFAST was developed so that all the procedures required for DDBJ submission could be done seamlessly online.  The online workspace would be esp. useful for users not familiar with bioinformatics skills.  In addn., we have developed a genome repository, DFAST Archive of Genome Annotation (DAGA), which currently includes 1,421 genomes covering 179 species and 18 subspecies of two genera, Lactobacillus and Pediococcus, obtained from both DDBJ/ENA/GenBank and Sequence Read Archive (SRA).  All the genomes deposited in DAGA were annotated consistently and assessed using DFAST.  To assess the taxonomic position based on genomic sequence information, we used the av. nucleotide identity (ANI), which showed high discriminative power to det. whether two given genomes belong to the same species.  We cor. mislabeled or misidentified genomes in the public database and deposited the curated information in DAGA.  The repository will improve the accessibility and reusability of genome resources for lactic acid bacteria.  By exploiting the data deposited in DAGA, we found intraspecific subgroups in Lactobacillus gasseri and Lactobacillus jensenii, whose variation between subgroups is larger than the well-accepted ANI threshold of 95% to differentiate species.  DFAST and DAGA are freely accessible at https://dfast.nig.ac.jp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGothR2Wl4cs-rVg90H21EOLACvtfcHk0lhNByKxUOtnBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpsF2msrc%253D&md5=bb55f302221e3bf2cf4c7a61f6e63b11</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.12938%2Fbmfh.16-003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12938%252Fbmfh.16-003%26sid%3Dliteratum%253Aachs%26aulast%3DTanizawa%26aufirst%3DY.%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DKaminuma%26aufirst%3DE.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DArita%26aufirst%3DM.%26atitle%3DDFAST%2520and%2520DAGA%253A%2520web-based%2520integrated%2520genome%2520annotation%2520tools%2520and%2520resources%26jtitle%3DBiosci.%2520Microbiota%252C%2520Food%2520Health%26date%3D2016%26volume%3D35%26spage%3D173%26epage%3D184%26doi%3D10.12938%2Fbmfh.16-003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15a','cit15b','cit15c','cit15d'],'ref16':['cit16'],'ref17':['cit17a','cit17b','cit17c','cit17d'],'ref18':['cit18a','cit18b'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21a','cit21b'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25a','cit25b','cit25c','cit25d','cit25e','cit25f','cit25g','cit25h','cit25i','cit25j'],'ref26':['cit26a','cit26b'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49a','cit49b'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref5400':['cit5400'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giuliana  Niro</span>, <span class="hlFld-ContribAuthor ">Stefanie C.  Weck</span>, <span class="hlFld-ContribAuthor ">Christian  Ducho</span>. </span><span class="cited-content_cbyCitation_article-title">Merging Natural Products: Muraymycin–Sansanmycin Hybrid Structures as Novel Scaffolds for Potential Antibacterial Agents. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2020,</strong> <em>26 </em>
                                    (70)
                                     , 16875-16887. <a href="https://doi.org/10.1002/chem.202003387" title="DOI URL">https://doi.org/10.1002/chem.202003387</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.202003387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.202003387%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DMerging%252BNatural%252BProducts%25253A%252BMuraymycin%2525E2%252580%252593Sansanmycin%252BHybrid%252BStructures%252Bas%252BNovel%252BScaffolds%252Bfor%252BPotential%252BAntibacterial%252BAgents%26aulast%3DNiro%26aufirst%3DGiuliana%26date%3D2020%26date%3D2020%26volume%3D26%26issue%3D70%26spage%3D16875%26epage%3D16887" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of uridylpeptide natural products.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of 3′-hydroxyuridylpeptide analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Retrosynthetic Analysis of <b>2</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Preparation of Suitably Protected <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr Derivatives <b>5</b> and <b>10</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of an Asymmetric Urea-Dipeptide 4</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0014.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>2</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. MraY inhibitory activity of analogues. <sup>a</sup>The inhibitory activities of the compounds against purified MraY<sub>SA</sub>. Reaction was conducted with 50 mM Tris-HCl (pH 7.6), 50 mM KCl, 25 mM MgCl<sub>2</sub>, 0.2% Triton X-100, 8% glycerol, 50 μM C55-P, 10 μM UDP-MurNAc-dansylpentapeptide, and MraY enzyme (55 ng/5 μL/well). After 3–4 h of incubation at room temperature, the formation of dansylated lipid I was monitored by fluorescence enhancement (excitation at 355 nm, emission at 535 nm). The data were obtained from three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray crystal structures of MraY<sub>AA</sub> bound to <b>2</b> and MRY D2. Carbons of <b>2</b> and MRY D2 are shown in green and orange, respectively. Hydrogen bond and salt bridge are indicated by cyan and magenta dashed lines, respectively. (a) Overall view of the complex bound to <b>2</b>. (b) Overall view of the complex bound to MRY D2. (c) Zoomed-in view of the uridine-adjacent pocket bound to <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr residue of <b>2</b>. (d) Zoomed-in view of the uridine-adjacent pocket bound to the aminoribose moiety of MRD D2. (e) Zoomed-in view of the TM9b/Loop E sites interacting with the urea-dipeptide motif of <b>2</b>. (f) Zoomed-in view of the TM9b/Loop E sites interacting with the urea-dipeptide motif of MRY D2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Sequence alignment of MraY. For detail, see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00973/suppl_file/jm0c00973_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Docking model structures of MraY<sub>SA</sub> and MraY<sub>PA</sub> bound to <b>2</b>. Carbons of <b>2</b> are shown in green. Hydrogen bond and salt bridge are indicated by cyan and magenta dashed lines, respectively. (a) Overall view of the complex of a homology modeled for MraY<sub>SA</sub> docked to <b>2</b>. (b) Overall view of the complex of a homology modeled for MraY<sub>PA</sub> docked to <b>2</b>. (c) Zoomed-in view of the uridine-adjacent pocket of MraY<sub>SA</sub> bound to the <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr residue of <b>2</b>. (d) Zoomed-in view of the uridine-adjacent pocket bound to the aminoribose moiety of MraY<sub>PA</sub> bound to the <span class="smallcaps smallerCapital">l</span>-<i>m</i>Tyr residue of <b>2</b>. (e) Zoomed-in view of the TM9b/Loop E sites of MraY<sub>SA</sub> interacting with the urea-dipeptide motif of <b>2</b>. (f) Zoomed-in view of the TM9b/Loop E sites of MraY<sub>PA</sub> interacting with the urea-dipeptide motif of <b>2</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structures and biological activity of MRY D2 and its lipophilic analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Structures of truncated and lipophilic analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Structures of analogues modified at the urea-dipeptide moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch5" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/medium/jm0c00973_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0015.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Analogues <b>29a–t</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00973/20200903/images/large/jm0c00973_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00973&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i105">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12261" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12261" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 57 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagiya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takimoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomono, K.</span></span> <span> </span><span class="NLM_article-title">Non-nosocomial healthcare-associated left-sided <i>Pseudomonas aeruginosa</i> endocarditis: a case report and literature review</span>. <i>BMC Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1186/s12879-016-1757-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1186%2Fs12879-016-1757-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=27543116" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BC2szivFKktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=431-437&author=H.+Hagiyaauthor=T.+Tanakaauthor=K.+Takimotoauthor=H.+Yoshidaauthor=N.+Yamamotoauthor=Y.+Akedaauthor=K.+Tomono&title=Non-nosocomial+healthcare-associated+left-sided+Pseudomonas+aeruginosa+endocarditis%3A+a+case+report+and+literature+review&doi=10.1186%2Fs12879-016-1757-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Non-nosocomial healthcare-associated left-sided Pseudomonas aeruginosa endocarditis: a case report and literature review</span></div><div class="casAuthors">Hagiya Hideharu; Yoshida Hisao; Yamamoto Norihisa; Akeda Yukihiro; Tomono Kazunori; Tanaka Takeshi; Takimoto Kohei</div><div class="citationInfo"><span class="NLM_cas:title">BMC infectious diseases</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">431</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  With the development of invasive medical procedures, an increasing number of healthcare-associated infective endocarditis cases have been reported.  In particular, non-nosocomial healthcare-associated infective endocarditis in outpatients with recent medical intervention has been increasingly identified.  CASE PRESENTATION:  A 66-year-old man with diabetes mellitus and a recent history of intermittent urethral self-catheterization was admitted due to a high fever.  Repeated blood cultures identified Pseudomonas aeruginosa, and transesophageal echocardiography uncovered a new-onset severe aortic regurgitation along with a vegetative valvular structure.  The patient underwent emergency aortic valve replacement surgery and was successfully treated with 6 weeks of high-dose meropenem and tobramycin.  Historically, most cases of P. aeruginosa endocarditis have occurred in the right side of the heart and in outpatients with a history of intravenous drug abuse.  In the case presented, the repeated manipulations of the urethra may have triggered the infection.  Our literature review for left-sided P. aeruginosa endocarditis showed that non-nosocomial infection accounted for nearly half of the cases and resulted in fatal outcomes as often as nosocomial cases.  A combination therapy with anti-pseudomonal beta-lactams or carbapenems and aminoglycosides may be the preferable treatment.  Medical treatment alone may be effective, and surgical treatment should be carefully considered.  CONCLUSIONS:  We presented a rare case of native aortic valve endocarditis caused by P. aeruginosa.  This case illustrates the importance of identifying the causative pathogen(s), especially for outpatients with a recent history of medical procedures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRlDlluXidKJdLIORRPd5lbfW6udTcc2eZzsfzzVGMVcbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szivFKktQ%253D%253D&md5=bf76c3c3de79b2e105fd1e5568e440c2</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1186%2Fs12879-016-1757-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12879-016-1757-y%26sid%3Dliteratum%253Aachs%26aulast%3DHagiya%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DTakimoto%26aufirst%3DK.%26aulast%3DYoshida%26aufirst%3DH.%26aulast%3DYamamoto%26aufirst%3DN.%26aulast%3DAkeda%26aufirst%3DY.%26aulast%3DTomono%26aufirst%3DK.%26atitle%3DNon-nosocomial%2520healthcare-associated%2520left-sided%2520Pseudomonas%2520aeruginosa%2520endocarditis%253A%2520a%2520case%2520report%2520and%2520literature%2520review%26jtitle%3DBMC%2520Infect.%2520Dis.%26date%3D2016%26volume%3D16%26spage%3D431%26epage%3D437%26doi%3D10.1186%2Fs12879-016-1757-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marcus, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashkenazi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samra, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livni, G.</span></span> <span> </span><span class="NLM_article-title">Community-acquired <i>Pseudomonas aeruginosa</i> urinary tract infections in children hospitalized in a tertiary center: relative frequency, risk factors, antimicrobial resistance and treatment</span>. <i>Infection</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">421</span>– <span class="NLM_lpage">426</span>, <span class="refDoi"> DOI: 10.1007/s15010-008-7328-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1007%2Fs15010-008-7328-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=18795227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BD1cjksFOjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=421-426&author=N.+Marcusauthor=S.+Ashkenaziauthor=Z.+Samraauthor=A.+Cohenauthor=G.+Livni&title=Community-acquired+Pseudomonas+aeruginosa+urinary+tract+infections+in+children+hospitalized+in+a+tertiary+center%3A+relative+frequency%2C+risk+factors%2C+antimicrobial+resistance+and+treatment&doi=10.1007%2Fs15010-008-7328-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Community-acquired Pseudomonas aeruginosa urinary tract infections in children hospitalized in a tertiary center: relative frequency, risk factors, antimicrobial resistance and treatment</span></div><div class="casAuthors">Marcus N; Ashkenazi S; Samra Z; Cohen A; Livni G</div><div class="citationInfo"><span class="NLM_cas:title">Infection</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">421-6</span>
        ISSN:<span class="NLM_cas:issn">0300-8126</span>.
    </div><div class="casAbstract">BACKGROUND:  The practice of antibiotic prophylaxis against recurrent urinary tract infection (UTI), with hospitalization reserved for severe or complicated cases, has led to changes in the nature and culprit uropathogens of community-acquired (CA), hospital-treated UTI.  Characterization of subgroups that need special considerations is crucial.  OBJECTIVES:  To elucidate the trends and characteristics of CA Pseudomonas UTI in hospitalized children; define the antibiotic susceptibility; determine the appropriateness of the empiric antibiotics used; compare to other causes of UTI in this population; and thereby define predictors for Pseudomonas UTI.  METHODS:  A prospective clinical and laboratory study from 2001 through 2005.  Children with P. aeruginosa UTI were characterized and compared with non-Pseudomonas UTI.  RESULTS:  Of 351 episodes of culture-proven CA UTI, 28 (8%) were caused by Pseudomonas, representing a 2.8-fold increase from our previous study.  Pseudomonas UTI was more common in children > 5 years (p < 0.01), with urinary abnormalities (p < 0.01) and with previous antibiotic use in the previous month (p < 0.001).  Pseudomonas UTI was often resistant to antibiotics usually recommended for empiric therapy; 25% was initially treated with inappropriate IV antibiotics (4.6% in the non-Pseudomonas group, p < 0.001) with 1.3 days longer IV antibiotics.  On multivariate analysis, risk factors for Pseudomonas UTI were previous antibiotic therapy and underlying urinary pathology.  CONCLUSIONS:  Pseudomonas UTI seems to increase in CA, hospital-treated children and is often treated inappropriately according to current treatment protocols.  Awareness of this trend and knowledge of the defined risk factors of Pseudomonas UTI might improve the empiric antibiotic therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQYTFUnWY3yh1HFYivNCdPRfW6udTcc2eZzsfzzVGMVcbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cjksFOjug%253D%253D&md5=fde47dc9ec042d5292fc56b6a5166b08</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2Fs15010-008-7328-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs15010-008-7328-4%26sid%3Dliteratum%253Aachs%26aulast%3DMarcus%26aufirst%3DN.%26aulast%3DAshkenazi%26aufirst%3DS.%26aulast%3DSamra%26aufirst%3DZ.%26aulast%3DCohen%26aufirst%3DA.%26aulast%3DLivni%26aufirst%3DG.%26atitle%3DCommunity-acquired%2520Pseudomonas%2520aeruginosa%2520urinary%2520tract%2520infections%2520in%2520children%2520hospitalized%2520in%2520a%2520tertiary%2520center%253A%2520relative%2520frequency%252C%2520risk%2520factors%252C%2520antimicrobial%2520resistance%2520and%2520treatment%26jtitle%3DInfection%26date%3D2008%26volume%3D36%26spage%3D421%26epage%3D426%26doi%3D10.1007%2Fs15010-008-7328-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obritsch, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacLaren, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, R.</span></span> <span> </span><span class="NLM_article-title">Nosocomial infections due to multidrug-resistant <i>Pseudomonas aeruginosa</i>: epidemiology and treatment options</span>. <i>Pharmacotherapy</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1353</span>– <span class="NLM_lpage">1364</span>, <span class="refDoi"> DOI: 10.1592/phco.2005.25.10.1353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1592%2Fphco.2005.25.10.1353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=16185180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFGitbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2005&pages=1353-1364&author=M.+D.+Obritschauthor=D.+N.+Fishauthor=R.+MacLarenauthor=R.+Jung&title=Nosocomial+infections+due+to+multidrug-resistant+Pseudomonas+aeruginosa%3A+epidemiology+and+treatment+options&doi=10.1592%2Fphco.2005.25.10.1353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Nosocomial infections due to multidrug-resistant Pseudomonas aeruginosa: Epidemiology and treatment options</span></div><div class="casAuthors">Obritsch, Marilee D.; Fish, Douglas N.; MacLaren, Robert; Jung, Rose</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacotherapy</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1353-1364</span>CODEN:
                <span class="NLM_cas:coden">PHPYDQ</span>;
        ISSN:<span class="NLM_cas:issn">0277-0008</span>.
    
            (<span class="NLM_cas:orgname">Pharmacotherapy Publications</span>)
        </div><div class="casAbstract">A review.  Pseudomonas aeruginosa is one of the leading gram-neg. organisms assocd. with nosocomial infections.  The increasing frequency of multidrug-resistant Pseudomonas aeruginosa (MDRPA) strains is concerning as efficacious antimicrobial options are severely limited.  By searching MEDLINE from Jan. 1966-Feb. 2005 and relevant journals for abstrs., we reviewed the frequency, risk factors, and patient outcomes of MDRPA nosocomial infections in critically ill patients, detd. the available antimicrobial therapies, and then provided recommendations for clinicians.  The definition of MDRPA was established as isolates intermediate or resistant to at least three drugs in the following classes: β-lactams, carbapenems, aminoglycosides, and fluoroquinolones.  Reported rates of MDRPA varied from 0.6-.32% according to geog. location and type of surveillance study.  Risk factors for MDRPA infection included prolonged hospitalization, exposure to antimicrobial therapy, and immunocompromised states such as human immunodeficiency virus infection.  Emergence of MDRPA isolates during therapy was reported in 27-72% of patients with initially susceptible P. aeruginosa isolates.  Patients with severe MDRPA infections should be treated with combination therapy, consisting of an antipseudomonal β-lactam with an aminoglycoside or fluoroquinolone rather than aminoglycoside and fluoroquinolone combinations, to provide adequate therapy and improve patient outcomes.  Synergy has been obsd. when resistant antipseudomonal drugs were combined in vitro against MDRPA with successful clin. application reported in two centers.  Colistin with adjunctive therapy, such as a β-lactam or rifampin, may be a useful agent in MDRPA when antimicrobial options are limited, but patients should be monitored closely for toxicities assocd. with this agent.  Standardization of terminol. for MDRPA isolates is needed for consistency and comparability of surveillance and institutional reports.  Clin. studies are needed to identify risk factors for MDRPA development and to det. the economic impact of these infections, as well as to det. the most efficacious antimicrobial regimens and duration of therapy to maximize outcomes in the treatment of MDRPA infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW44ZKjwzU0LVg90H21EOLACvtfcHk0lic5h__M5oZog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFGitbzE&md5=09a8ef3955486ab6279f37148a364de5</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1592%2Fphco.2005.25.10.1353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1592%252Fphco.2005.25.10.1353%26sid%3Dliteratum%253Aachs%26aulast%3DObritsch%26aufirst%3DM.%2BD.%26aulast%3DFish%26aufirst%3DD.%2BN.%26aulast%3DMacLaren%26aufirst%3DR.%26aulast%3DJung%26aufirst%3DR.%26atitle%3DNosocomial%2520infections%2520due%2520to%2520multidrug-resistant%2520Pseudomonas%2520aeruginosa%253A%2520epidemiology%2520and%2520treatment%2520options%26jtitle%3DPharmacotherapy%26date%3D2005%26volume%3D25%26spage%3D1353%26epage%3D1364%26doi%3D10.1592%2Fphco.2005.25.10.1353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sonmezer, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertem, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdinc, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaya Kilic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tulek, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adiloglu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatipoglu, C.</span></span> <span> </span><span class="NLM_article-title">Evaluation of risk factors for antibiotic resistance in patients with nosocomial infections caused by. <i>Pseudomonas aeruginosa</i></span>. <i>Can. J. Infect Dis. Med. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2016</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1155/2016/1321487</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1155%2F2016%2F1321487" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2016&publication_year=2016&pages=1-9&author=M.+C.+Sonmezerauthor=G.+Ertemauthor=F.+S.+Erdincauthor=E.+Kaya+Kilicauthor=N.+Tulekauthor=A.+Adilogluauthor=C.+Hatipoglu&title=Evaluation+of+risk+factors+for+antibiotic+resistance+in+patients+with+nosocomial+infections+caused+by.+Pseudomonas+aeruginosa&doi=10.1155%2F2016%2F1321487"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1155%2F2016%2F1321487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2016%252F1321487%26sid%3Dliteratum%253Aachs%26aulast%3DSonmezer%26aufirst%3DM.%2BC.%26aulast%3DErtem%26aufirst%3DG.%26aulast%3DErdinc%26aufirst%3DF.%2BS.%26aulast%3DKaya%2BKilic%26aufirst%3DE.%26aulast%3DTulek%26aufirst%3DN.%26aulast%3DAdiloglu%26aufirst%3DA.%26aulast%3DHatipoglu%26aufirst%3DC.%26atitle%3DEvaluation%2520of%2520risk%2520factors%2520for%2520antibiotic%2520resistance%2520in%2520patients%2520with%2520nosocomial%2520infections%2520caused%2520by.%2520Pseudomonas%2520aeruginosa%26jtitle%3DCan.%2520J.%2520Infect%2520Dis.%2520Med.%2520Microbiol.%26date%3D2016%26volume%3D2016%26spage%3D1%26epage%3D9%26doi%3D10.1155%2F2016%2F1321487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tam, V. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdelraouf, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brioso, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ameka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCaskey, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weston, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caeiro, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garey, K. W.</span></span> <span> </span><span class="NLM_article-title">Prevalence, resistance mechanisms, and susceptibility of multidrug-resistant bloodstream isolates of <i>Pseudomonas aeruginosa</i></span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1164</span>, <span class="refDoi"> DOI: 10.1128/aac.01446-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1128%2FAAC.01446-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=20086165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFWqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2010&pages=1160-1164&author=V.+H.+Tamauthor=K.-T.+Changauthor=K.+Abdelraoufauthor=C.+G.+Briosoauthor=M.+Amekaauthor=L.+A.+McCaskeyauthor=J.+S.+Westonauthor=J.-P.+Caeiroauthor=K.+W.+Garey&title=Prevalence%2C+resistance+mechanisms%2C+and+susceptibility+of+multidrug-resistant+bloodstream+isolates+of+Pseudomonas+aeruginosa&doi=10.1128%2Faac.01446-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence, resistance mechanisms, and susceptibility of multidrug-resistant bloodstream isolates of Pseudomonas aeruginosa</span></div><div class="casAuthors">Tam, Vincent H.; Chang, Kai-Tai; Abdelraouf, Kamilia; Brioso, Cristina G.; Ameka, Magdalene; McCaskey, Laurie A.; Weston, Jaye S.; Caeiro, Juan-Pablo; Garey, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1160-1164</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa is an important pathogen commonly implicated in nosocomial infections.  The occurrence of multidrug-resistant (MDR) P. aeruginosa strains is increasing worldwide and limiting our therapeutic options.  The MDR phenotype can be mediated by a variety of resistance mechanisms, and the corresponding relative biofitness is not well established.  We examd. the prevalence, resistance mechanisms, and susceptibility of MDR P. aeruginosa isolates (resistant to ≥3 classes of antipseudomonal agents [penicillins/cephalosporins, carbapenems, quinolones, and aminoglycosides]) obtained from a large, university-affiliated hospital.  Among 235 nonrepeat bloodstream isolates screened between 2005 and 2007, 33 isolates (from 20 unique patients) were found to be MDR (crude prevalence rate, 14%).  All isolates were resistant to carbapenems and quinolones, 91% were resistant to penicillins/cephalosporins, and 21% were resistant to the aminoglycosides.  By using the first available isolate for each bacteremia episode (n = 18), 13 distinct clones were revealed by repetitive-element-based PCR.  Western blotting revealed eight isolates (44%) to have MexB overexpression.  Prodn. of a carbapenemase (VIM-2) was found in one isolate, and mutations in gyrA (T83I) and parC (S87L) were commonly found.  Growth rates of most MDR isolates were similar to that of the wild type, and two isolates (11%) were found to be hypermutable.  All available isolates were susceptible to polymyxin B, and only one isolate was nonsusceptible to colistin (MIC, 3 mg/L), but all isolates were nonsusceptible to doripenem (MIC, >2 mg/L).  Understanding and continuous monitoring of the prevalence and resistance mechanisms of MDR P. aeruginosa would enable us to formulate rational treatment strategies to combat nosocomial infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYWXf_nYRWLLVg90H21EOLACvtfcHk0lgt4CJDLipGJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFWqsLs%253D&md5=949cd3e6fe2dd51c0288766a705bc570</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FAAC.01446-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01446-09%26sid%3Dliteratum%253Aachs%26aulast%3DTam%26aufirst%3DV.%2BH.%26aulast%3DChang%26aufirst%3DK.-T.%26aulast%3DAbdelraouf%26aufirst%3DK.%26aulast%3DBrioso%26aufirst%3DC.%2BG.%26aulast%3DAmeka%26aufirst%3DM.%26aulast%3DMcCaskey%26aufirst%3DL.%2BA.%26aulast%3DWeston%26aufirst%3DJ.%2BS.%26aulast%3DCaeiro%26aufirst%3DJ.-P.%26aulast%3DGarey%26aufirst%3DK.%2BW.%26atitle%3DPrevalence%252C%2520resistance%2520mechanisms%252C%2520and%2520susceptibility%2520of%2520multidrug-resistant%2520bloodstream%2520isolates%2520of%2520Pseudomonas%2520aeruginosa%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2010%26volume%3D54%26spage%3D1160%26epage%3D1164%26doi%3D10.1128%2Faac.01446-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Page, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heim, J.</span></span> <span> </span><span class="NLM_article-title">Prospects for the next anti-Pseudomonas drug</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">558</span>– <span class="NLM_lpage">565</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2009.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2Fj.coph.2009.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=19748829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1WntLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=558-565&author=M.+G.+Pageauthor=J.+Heim&title=Prospects+for+the+next+anti-Pseudomonas+drug&doi=10.1016%2Fj.coph.2009.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Prospects for the next anti-Pseudomonas drug</span></div><div class="casAuthors">Page, Malcolm G. P.; Heim, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">558-565</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Pseudomonas aeruginosa is one of the most dreaded Gram-neg. bacterial pathogens in hospitals.  Not only it is among the most frequently isolated Gram-neg. organisms in bloodstream and wound infections, pneumonia, intra-abdominal-sepsis and urogenital-sepsis, but also it is frequently found in patients with comorbid illnesses and compromised by in-dwelling catheters, tubes and surgery where mortality rates of more than 60% have been reported.  Besides its intrinsic resistance to a no. of widely used antibiotics, Pseudomonas also managed to acquire resistance via addnl. mechanisms, including target mutations, increased expression of efflux pumps and of antibiotic-degrading enzymes.  Taken together, the increased incidence in certain types of infections, the increased use of invasive devices in the hospital as well as the increased frequency of multi-resistant Pseudomonas strains, have clearly led to a shortage of treatment options for nosocomial Pseudomonas infections.  Even the recommended combination therapy of an antibiotic of the β-lactam class together with an aminoglycoside or a fluoroquinolone, is no longer always successful and sometimes a polymyxin has to be given as last resort.  Despite growing concerns of clinicians and medical societies about the very limited no. of novel drugs in the pipeline to fight multi-resistant Pseudomonas strains, only a very small no. of novel anti-Pseudomonas drugs are currently in late stage of pre-clin. or clin. development.  However, and possibly as a reflection of the magnitude of the problem, quite a variety of approaches are being pursued.  Among these are next-generation analogs of successful antibiotic classes (e.g. novel β-lactams and combinations of novel β-lactamase inhibitors with known penicillins or cephalosporins), antibodies, phages and selective peptides.  It is to be hoped that a no. of these novel drugs will show clin. utility and reach the market over the next 6-10 years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxOCyGCdsTGbVg90H21EOLACvtfcHk0lj9KAE8DIfMLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1WntLnP&md5=d4022458c9a09f2c1f4a243ba7a7d61b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2009.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2009.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DPage%26aufirst%3DM.%2BG.%26aulast%3DHeim%26aufirst%3DJ.%26atitle%3DProspects%2520for%2520the%2520next%2520anti-Pseudomonas%2520drug%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2009%26volume%3D9%26spage%3D558%26epage%3D565%26doi%3D10.1016%2Fj.coph.2009.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boucher, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talbot, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheld, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spellberg, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, J.</span></span> <span> </span><span class="NLM_article-title">Bad Bugs, No drugs: no ESKAPE! An update from the Infectious Diseases Society of America</span>. <i>Clin. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.1086/595011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1086%2F595011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=19035777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BD1M%252Fit1Krsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=1-12&author=H.+W.+Boucherauthor=G.+H.+Talbotauthor=J.+S.+Bradleyauthor=J.+E.+Edwardsauthor=D.+Gilbertauthor=L.+B.+Riceauthor=M.+Scheldauthor=B.+Spellbergauthor=J.+Bartlett&title=Bad+Bugs%2C+No+drugs%3A+no+ESKAPE%21+An+update+from+the+Infectious+Diseases+Society+of+America&doi=10.1086%2F595011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Bad bugs, no drugs: no ESKAPE! An update from the Infectious Diseases Society of America</span></div><div class="casAuthors">Boucher Helen W; Talbot George H; Bradley John S; Edwards John E; Gilbert David; Rice Louis B; Scheld Michael; Spellberg Brad; Bartlett John</div><div class="citationInfo"><span class="NLM_cas:title">Clinical infectious diseases : an official publication of the Infectious Diseases Society of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-12</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The Infectious Diseases Society of America (IDSA) continues to view with concern the lean pipeline for novel therapeutics to treat drug-resistant infections, especially those caused by gram-negative pathogens.  Infections now occur that are resistant to all current antibacterial options.  Although the IDSA is encouraged by the prospect of success for some agents currently in preclinical development, there is an urgent, immediate need for new agents with activity against these panresistant organisms.  There is no evidence that this need will be met in the foreseeable future.  Furthermore, we remain concerned that the infrastructure for discovering and developing new antibacterials continues to stagnate, thereby risking the future pipeline of antibacterial drugs.  The IDSA proposed solutions in its 2004 policy report, "Bad Bugs, No Drugs: As Antibiotic R&D Stagnates, a Public Health Crisis Brews," and recently issued a "Call to Action" to provide an update on the scope of the problem and the proposed solutions.  A primary objective of these periodic reports is to encourage a community and legislative response to establish greater financial parity between the antimicrobial development and the development of other drugs.  Although recent actions of the Food and Drug Administration and the 110th US Congress present a glimmer of hope, significant uncertainly remains.  Now, more than ever, it is essential to create a robust and sustainable antibacterial research and development infrastructure--one that can respond to current antibacterial resistance now and anticipate evolving resistance.  This challenge requires that industry, academia, the National Institutes of Health, the Food and Drug Administration, the Centers for Disease Control and Prevention, the US Department of Defense, and the new Biomedical Advanced Research and Development Authority at the Department of Health and Human Services work productively together.  This report provides an update on potentially effective antibacterial drugs in the late-stage development pipeline, in the hope of encouraging such collaborative action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKtuWEUmAWleWhZTslLrfffW6udTcc2eZcp1MGYdmiw7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M%252Fit1Krsg%253D%253D&md5=a3f71406b891f90f4f0f77b5f1ebc4f8</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1086%2F595011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F595011%26sid%3Dliteratum%253Aachs%26aulast%3DBoucher%26aufirst%3DH.%2BW.%26aulast%3DTalbot%26aufirst%3DG.%2BH.%26aulast%3DBradley%26aufirst%3DJ.%2BS.%26aulast%3DEdwards%26aufirst%3DJ.%2BE.%26aulast%3DGilbert%26aufirst%3DD.%26aulast%3DRice%26aufirst%3DL.%2BB.%26aulast%3DScheld%26aufirst%3DM.%26aulast%3DSpellberg%26aufirst%3DB.%26aulast%3DBartlett%26aufirst%3DJ.%26atitle%3DBad%2520Bugs%252C%2520No%2520drugs%253A%2520no%2520ESKAPE%2521%2520An%2520update%2520from%2520the%2520Infectious%2520Diseases%2520Society%2520of%2520America%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2009%26volume%3D48%26spage%3D1%26epage%3D12%26doi%3D10.1086%2F595011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Payne, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwynn, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompliano, D. L.</span></span> <span> </span><span class="NLM_article-title">Drugs for bad bugs: confronting the challenges of antibacterial discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1038/nrd2201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1038%2Fnrd2201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=17159923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=29-40&author=D.+J.+Payneauthor=M.+N.+Gwynnauthor=D.+J.+Holmesauthor=D.+L.+Pompliano&title=Drugs+for+bad+bugs%3A+confronting+the+challenges+of+antibacterial+discovery&doi=10.1038%2Fnrd2201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Drugs for bad bugs: confronting the challenges of antibacterial discovery</span></div><div class="casAuthors">Payne, David J.; Gwynn, Michael N.; Holmes, David J.; Pompliano, David L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-40</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The sequencing of the first complete bacterial genome in 1995 heralded a new era of hope for antibacterial drug discoverers, who now had the tools to search entire genomes for new antibacterial targets.  Several companies, including GlaxoSmithKline, moved back into the antibacterials area and embraced a genomics-derived, target-based approach to screen for new classes of drugs with novel modes of action.  Here, we share our experience of evaluating more than 300 genes and 70 high-throughput screening campaigns over a period of 7 years, and look at what we learned and how that has influenced GlaxoSmithKline's antibacterials strategy going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH1tM2BSZbWLVg90H21EOLACvtfcHk0lhMpeoEh9pwZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfM&md5=18100253f9868af578ea621ab98a5c86</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd2201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2201%26sid%3Dliteratum%253Aachs%26aulast%3DPayne%26aufirst%3DD.%2BJ.%26aulast%3DGwynn%26aufirst%3DM.%2BN.%26aulast%3DHolmes%26aufirst%3DD.%2BJ.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26atitle%3DDrugs%2520for%2520bad%2520bugs%253A%2520confronting%2520the%2520challenges%2520of%2520antibacterial%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D29%26epage%3D40%26doi%3D10.1038%2Fnrd2201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Talbot, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. E.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheld, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, J. G.</span></span> <span> </span><span class="NLM_article-title">Bad Bugs Need Drugs: An update on the development pipeline from the antimicrobial availability task force of the Infectious Diseases Society of America</span>. <i>Clin. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">657</span>– <span class="NLM_lpage">668</span>, <span class="refDoi"> DOI: 10.1086/499819</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1086%2F499819" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=16447111" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BD28%252FmtFGqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2006&pages=657-668&author=G.+H.+Talbotauthor=J.+Bradleyauthor=J.+E.+Edwardsauthor=D.+Gilbertauthor=M.+Scheldauthor=J.+G.+Bartlett&title=Bad+Bugs+Need+Drugs%3A+An+update+on+the+development+pipeline+from+the+antimicrobial+availability+task+force+of+the+Infectious+Diseases+Society+of+America&doi=10.1086%2F499819"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Bad bugs need drugs: an update on the development pipeline from the Antimicrobial Availability Task Force of the Infectious Diseases Society of America</span></div><div class="casAuthors">Talbot George H; Bradley John; Edwards John E Jr; Gilbert David; Scheld Michael; Bartlett John G</div><div class="citationInfo"><span class="NLM_cas:title">Clinical infectious diseases : an official publication of the Infectious Diseases Society of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">657-68</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The Antimicrobial Availability Task Force (AATF) of the Infectious Diseases Society of America (IDSA) has viewed with concern the decreasing investment by major pharmaceutical companies in antimicrobial research and development.  Although smaller companies are stepping forward to address this gap, their success is uncertain.  The IDSA proposed legislative and other federal solutions to this emerging public health problem in its July 2004 policy report "Bad Bugs, No Drugs: As Antibiotic R&D Stagnates, a Public Health Crisis Brews." At this time, the legislative response cannot be predicted.  To emphasize further the urgency of the problem for the benefit of legislators and policy makers and to capture the ongoing frustration our clinician colleagues experience in their frequent return to an inadequate medicine cabinet, the AATF has prepared this review to highlight pathogens that are frequently resistant to licensed antimicrobials and for which few, if any, potentially effective drugs are identifiable in the late-stage development pipeline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTcyYffQlmsoVQKX3ooHxWjfW6udTcc2eZKkW3RZI_OmLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28%252FmtFGqsQ%253D%253D&md5=fd193c8654a73888c84ba22299c93cc8</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1086%2F499819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F499819%26sid%3Dliteratum%253Aachs%26aulast%3DTalbot%26aufirst%3DG.%2BH.%26aulast%3DBradley%26aufirst%3DJ.%26aulast%3DEdwards%26aufirst%3DJ.%2BE.%26aulast%3DGilbert%26aufirst%3DD.%26aulast%3DScheld%26aufirst%3DM.%26aulast%3DBartlett%26aufirst%3DJ.%2BG.%26atitle%3DBad%2520Bugs%2520Need%2520Drugs%253A%2520An%2520update%2520on%2520the%2520development%2520pipeline%2520from%2520the%2520antimicrobial%2520availability%2520task%2520force%2520of%2520the%2520Infectious%2520Diseases%2520Society%2520of%2520America%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2006%26volume%3D42%26spage%3D657%26epage%3D668%26doi%3D10.1086%2F499819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Overbye, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J.</span></span> <span> </span><span class="NLM_article-title">Antibiotics: where did we go wrong?</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1016/s1359-6446(04)03285-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2FS1359-6446%2804%2903285-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=15676298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FkvFOhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=45-52&author=K.+Overbyeauthor=J.+Barrett&title=Antibiotics%3A+where+did+we+go+wrong%3F&doi=10.1016%2Fs1359-6446%2804%2903285-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotics: where did we go wrong?</span></div><div class="casAuthors">Overbye Karen M; Barrett John F</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-52</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">In the late 1960s, the medical need for new antibiotics began to be questioned, and the pharmaceutical industry shifted its emphasis of antibacterials from that of a therapeutic leader to a low-priority research area.  Although infectious diseases, in particular those caused by bacterial infections, are still among the top causes of mortality in the world, industrial support continues to wane.  The shift from this important area of antimicrobial research has been attributed to a combination of science, medical, marketing and business reasons.  This decline in antibacterial drug discovery, coupled with increasing risk as a result of infections caused by drug-resistant bacterial pathogens, represents a clear public health threat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS--fo71XGN1X6CEoQgWY10fW6udTcc2ebPNz37MsO7M7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FkvFOhtA%253D%253D&md5=9d48b7cd2b497d8db69f0fae744f7baf</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903285-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903285-4%26sid%3Dliteratum%253Aachs%26aulast%3DOverbye%26aufirst%3DK.%26aulast%3DBarrett%26aufirst%3DJ.%26atitle%3DAntibiotics%253A%2520where%2520did%2520we%2520go%2520wrong%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D45%26epage%3D52%26doi%3D10.1016%2Fs1359-6446%2804%2903285-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monaghan, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. F.</span></span> <span> </span><span class="NLM_article-title">Antibacterial drug discovery-then, now and the genomics future</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">901</span>– <span class="NLM_lpage">909</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2005.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2Fj.bcp.2005.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=16494849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD28XitVKjurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2006&pages=901-909&author=R.+L.+Monaghanauthor=J.+F.+Barrett&title=Antibacterial+drug+discovery-then%2C+now+and+the+genomics+future&doi=10.1016%2Fj.bcp.2005.11.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial drug discovery-Then, now and the genomics future</span></div><div class="casAuthors">Monaghan, Richard L.; Barrett, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">901-909</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Drug discovery research in the area of infectious diseases, in particular that dealing with antibacterial/antibiotic susceptibility and resistance, is in a process of continuing evolution.  Steeped in the history of the highly successful intervention with chemotherapeutic agents to treat human infections, the emergence of drug-resistant pathogens worldwide presents a serious unmet medical need, if not a pending catastrophe.  Research in both academia and industry over the past 30 years using mol. biol., genetics and more recently - bacterial genomics - has assembled key enabling technologies to increase productivity and success rates in the discovery and development of novel antibacterial agents.  However genomics is not limited only to antibacterial target selection but provides the opportunity to further understand key interactions in the use of antibacterial compds. as therapeutic agents (such as resistance emergence, susceptibility, efflux, interactions between compd. and pathogen, etc.).  Genomics also offers the potential for insights into: bacterial niche adaptation, host susceptibility, treatment regimens, antibiotic resistance, pharmacokinetics (e.g., host metab. differences), safety and the microbial genesis of chronic diseases (e.g., gastric ulceration).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4sY_8A5Ry07Vg90H21EOLACvtfcHk0ljvI8Q1sWotsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XitVKjurg%253D&md5=a3c65f493cb5a82cff2f736931a56248</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2005.11.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2005.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DMonaghan%26aufirst%3DR.%2BL.%26aulast%3DBarrett%26aufirst%3DJ.%2BF.%26atitle%3DAntibacterial%2520drug%2520discovery-then%252C%2520now%2520and%2520the%2520genomics%2520future%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2006%26volume%3D71%26spage%3D901%26epage%3D909%26doi%3D10.1016%2Fj.bcp.2005.11.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, C.</span></span> <span> </span><span class="NLM_article-title">Where will new antibiotics come from?</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">65</span>– <span class="NLM_lpage">70</span>, <span class="refDoi"> DOI: 10.1038/nrmicro727</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1038%2Fnrmicro727" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=15040181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptFWlu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2003&pages=65-70&author=C.+Walsh&title=Where+will+new+antibiotics+come+from%3F&doi=10.1038%2Fnrmicro727"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Where will new antibiotics come from?</span></div><div class="casAuthors">Walsh, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-70</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There is a const. need for new antibacterial drugs owing to the inevitable development of resistance that follows the introduction of antibiotics to the clinic.  When a new class of antibiotic is introduced, it is effective at first, but will eventually select for survival of the small fraction of bacterial populations that have an intrinsic or acquired resistance mechanism.  Pathogens that are resistant to multiple drugs emerge around the globe, so how robust are antibiotic discovery processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWf-wNvjdudbVg90H21EOLACvtfcHk0liD3ZGOLDNwRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptFWlu7s%253D&md5=be8a897b57e796094c5b54652152d533</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro727&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro727%26sid%3Dliteratum%253Aachs%26aulast%3DWalsh%26aufirst%3DC.%26atitle%3DWhere%2520will%2520new%2520antibiotics%2520come%2520from%253F%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2003%26volume%3D1%26spage%3D65%26epage%3D70%26doi%3D10.1038%2Fnrmicro727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goss, R. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, K.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial nucleoside antibiotics targeting cell wall assembly: Recent advances in structure–function studies and nucleoside biosynthesis</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">279</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1039/b816215h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2FB816215H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=20111805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Sntb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2010&pages=279-304&author=M.+Winnauthor=R.+J.+M.+Gossauthor=K.-I.+Kimuraauthor=T.+D.+H.+Bugg&title=Antimicrobial+nucleoside+antibiotics+targeting+cell+wall+assembly%3A+Recent+advances+in+structure%E2%80%93function+studies+and+nucleoside+biosynthesis&doi=10.1039%2Fb816215h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial nucleoside antibiotics targeting cell wall assembly: Recent advances in structure-function studies and nucleoside biosynthesis</span></div><div class="casAuthors">Winn, Michael; Goss, Rebecca J. M.; Kimura, Ken-ichi; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">279-304</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The quest for new antibiotics, esp. those with activity against Gram-neg. bacteria, is urgent; however, very few new antibiotics have been marketed in the last 40 years, with this limited no. falling into only four new structural classes.  Several nucleoside natural product antibiotics target bacterial translocase MraY, involved in the lipid-linked cycle of peptidoglycan biosynthesis, and fungal chitin synthase.  Biosynthetic studies on the nikkomycin, caprazamycin and pacidamycin/mureidomycin families are also reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8Wne70VFcV7Vg90H21EOLACvtfcHk0liD3ZGOLDNwRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Sntb8%253D&md5=49a27386210ce2250334db085a7bfc67</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2FB816215H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FB816215H%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%26aulast%3DGoss%26aufirst%3DR.%2BJ.%2BM.%26aulast%3DKimura%26aufirst%3DK.-I.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DAntimicrobial%2520nucleoside%2520antibiotics%2520targeting%2520cell%2520wall%2520assembly%253A%2520Recent%2520advances%2520in%2520structure%25E2%2580%2593function%2520studies%2520and%2520nucleoside%2520biosynthesis%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2010%26volume%3D27%26spage%3D279%26epage%3D304%26doi%3D10.1039%2Fb816215h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, K.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Recent advances in antimicrobial nucleoside antibiotics targeting cell wall biosynthesis</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1039/b202149h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2Fb202149h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=12735700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslCqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2003&pages=252-273&author=K.-I.+Kimuraauthor=T.+D.+H.+Bugg&title=Recent+advances+in+antimicrobial+nucleoside+antibiotics+targeting+cell+wall+biosynthesis&doi=10.1039%2Fb202149h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in antimicrobial nucleoside antibiotics targeting cell wall biosynthesis</span></div><div class="casAuthors">Kimura, Ken-Ichi; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">252-273</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The biosynthesis of peptidoglycan is essential for all bacteria and has no counterpart in eukaryotic cells.  It is one of the prime targets for antibiotic chemotherapy, esp. phospho-MurNAc-pentapeptide translocase (translocase I) is a fascinating target in which there is no com. antibiotic.  In this review we will describe three nucleoside translocase I inhibitors, mureidomycin, tunicamycin and liposidomycin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrksNstyeT_iLVg90H21EOLACvtfcHk0lgRTvubxmjEcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslCqtr4%253D&md5=8c6823d378a4754c368a6179b5bfa057</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1039%2Fb202149h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb202149h%26sid%3Dliteratum%253Aachs%26aulast%3DKimura%26aufirst%3DK.-I.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DRecent%2520advances%2520in%2520antimicrobial%2520nucleoside%2520antibiotics%2520targeting%2520cell%2520wall%2520biosynthesis%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2003%26volume%3D20%26spage%3D252%26epage%3D273%26doi%3D10.1039%2Fb202149h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enokita, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaida, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haneishi, T.</span></span> <span> </span><span class="NLM_article-title">Mureidomycins A-D, novel peptidylnucleoside antibiotics with spheroplast forming activity. I. Taxonomy, fermentation, isolation and physico-chemical properties</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">662</span>– <span class="NLM_lpage">666</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.42.662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.42.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=2664250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaL1MXkslCntrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1989&pages=662-666&author=M.+Inukaiauthor=F.+Isonoauthor=S.+Takahashiauthor=R.+Enokitaauthor=Y.+Sakaidaauthor=T.+Haneishi&title=Mureidomycins+A-D%2C+novel+peptidylnucleoside+antibiotics+with+spheroplast+forming+activity.+I.+Taxonomy%2C+fermentation%2C+isolation+and+physico-chemical+properties&doi=10.7164%2Fantibiotics.42.662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15aR"><div class="casContent"><span class="casTitleNuber">15a</span><div class="casTitle"><span class="NLM_cas:atitle">Mureidomycins A-D, novel peptidyl nucleoside antibiotics with spheroplast forming activity. I. Taxonomy, fermentation, isolation and physicochemical properties</span></div><div class="casAuthors">Inukai, Masatoshi; Isono, Fujio; Takahashi, Shuji; Enokita, Ryuzo; Sakaida, Yoshiharu; Haneishi, Tatsuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">662-6</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">A strain of actinomycetes identified as Streptomyces flavidovirens produced new antibiotics, mureidomycins (MRD's) A-D, specifically active against Pseudomonas aeruginosa.  They were isolated from the culture filtrate by successive column chromatogs. such as Amberlite XAD-2 and CG-50, Whatman DE-52 and Toyoperal HW-40.  They were amphoteric white powders, sol. in MeOH and H2O.  Their mol. wts. and mol. formula, in parentheses were 840 (C38H48N8O12S), 842 (C38H50N8O12S), 897 (C40H51N9O13S), and 899 (C40H53N9O13S), resp.  m-Tyrosine and two unknown substances were detected by amino acid analyses as their common constituents.  MRD's A and C contained uracil but MRD's B and D dihydrouracil instead of uracil.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrheyBZJOz2o7Vg90H21EOLACvtfcHk0lgRTvubxmjEcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXkslCntrc%253D&md5=bb9351de6d81f2e5d84869fb209f0eac</span></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.42.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.42.662%26sid%3Dliteratum%253Aachs%26aulast%3DInukai%26aufirst%3DM.%26aulast%3DIsono%26aufirst%3DF.%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DEnokita%26aufirst%3DR.%26aulast%3DSakaida%26aufirst%3DY.%26aulast%3DHaneishi%26aufirst%3DT.%26atitle%3DMureidomycins%2520A-D%252C%2520novel%2520peptidylnucleoside%2520antibiotics%2520with%2520spheroplast%2520forming%2520activity.%2520I.%2520Taxonomy%252C%2520fermentation%252C%2520isolation%2520and%2520physico-chemical%2520properties%26jtitle%3DJ.%2520Antibiot.%26date%3D1989%26volume%3D42%26spage%3D662%26epage%3D666%26doi%3D10.7164%2Fantibiotics.42.662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit15b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Isono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katayama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haneishi, T.</span></span> <span> </span><span class="NLM_article-title">Mureidomycins A-D, novel peptidylnucleoside antibiotics with spheroplast forming activity. III. Biological properties</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">674</span>– <span class="NLM_lpage">679</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.42.674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.42.674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=2498275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaL1MXksFCjt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1989&pages=674-679&author=F.+Isonoauthor=T.+Katayamaauthor=M.+Inukaiauthor=T.+Haneishi&title=Mureidomycins+A-D%2C+novel+peptidylnucleoside+antibiotics+with+spheroplast+forming+activity.+III.+Biological+properties&doi=10.7164%2Fantibiotics.42.674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15bR"><div class="casContent"><span class="casTitleNuber">15b</span><div class="casTitle"><span class="NLM_cas:atitle">Mureidomycins A∼D, novel peptidylnucleoside antibiotics with spheroplast forming activity.  III.  Biological properties</span></div><div class="casAuthors">Isono, Fujio; Katayama, Toshiaki; Inukai, Masatoshi; Haneishi, Tatsuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">674-9</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Mureidomycins (MRDs) A-D were specifically active against Pseudomonas aeruginosa.  Among them, MRD C was most active, with MICs of 0.1-3.13 μg/mL against many strains of the target organism.  Its activity was comparable to that of cefoperazone, ceftazidime, and cefsulodin.  MRD C-resistant mutants of P. aeruginosa appeared spontaneously at a high frequency when cultured in the presence of the antibiotic.  No cross-resistance was obsd. with β-lactam antibiotics.  A rapid decrease of turbidity along with spheroplast formation and cell lysis was obsd. when cells of P. aeruginosa were grown in the presence of MRD C.  The compds. exhibited low toxicity and protected mice from exptl. infection with P. aeruginosa.  The urinary and fecal recoveries of MRD C given s.c. were 5 and 18%, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn4ZvK5VmHgrVg90H21EOLACvtfcHk0lgSoaCnHJYwEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXksFCjt7o%253D&md5=eb690f68ff7f669c6bdef656625ffb9d</span></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.42.674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.42.674%26sid%3Dliteratum%253Aachs%26aulast%3DIsono%26aufirst%3DF.%26aulast%3DKatayama%26aufirst%3DT.%26aulast%3DInukai%26aufirst%3DM.%26aulast%3DHaneishi%26aufirst%3DT.%26atitle%3DMureidomycins%2520A-D%252C%2520novel%2520peptidylnucleoside%2520antibiotics%2520with%2520spheroplast%2520forming%2520activity.%2520III.%2520Biological%2520properties%26jtitle%3DJ.%2520Antibiot.%26date%3D1989%26volume%3D42%26spage%3D674%26epage%3D679%26doi%3D10.7164%2Fantibiotics.42.674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit15c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Isono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haneishi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinoshita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuwano, H.</span></span> <span> </span><span class="NLM_article-title">Mureidomycins A-D, novel peptidylnucleoside antibiotics with spheroplast forming activity. II. Structural elucidation</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">673</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.42.667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.42.667" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=2498274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaL1MXltlertbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1989&pages=667-673&author=F.+Isonoauthor=M.+Inukaiauthor=S.+Takahashiauthor=T.+Haneishiauthor=T.+Kinoshitaauthor=H.+Kuwano&title=Mureidomycins+A-D%2C+novel+peptidylnucleoside+antibiotics+with+spheroplast+forming+activity.+II.+Structural+elucidation&doi=10.7164%2Fantibiotics.42.667"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15cR"><div class="casContent"><span class="casTitleNuber">15c</span><div class="casTitle"><span class="NLM_cas:atitle">Mureidomycins A-D, novel peptidyl nucleoside antibiotics with spheroplast forming activity. II. Structural elucidation</span></div><div class="casAuthors">Isono, Fujio; Inukai, Masatoshi; Takahashi, Shuji; Haneishi, Tatsuo; Kinoshita, Takeshi; Kuwano, Harumitsu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">667-73</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Structures of new antibiotics, mureidomycins (MRD's) A-D (I-IV), were deduced from spectroscopic analyses and degrdn. studies.  Two residues of m-tyrosine, one residue of 2-amino-3-N-methylaminobutyric acid (AMBA) and methionine are present in all components of the complex.  Uracil is contained in MRD's A and C, while dihydrouracil in MRD's B and D.  Methionine and m-tyrosine are connected through an ureido bond,and uracil or dihydrouracil is linked to AMBA via an enamine sugar moiety.  In addn., MRD's C and D contain a glycine residue at the N-terminal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh5gV1ov1yurVg90H21EOLACvtfcHk0lgSoaCnHJYwEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXltlertbw%253D&md5=ac87a58b5e06562179cdacca9f183013</span></div><a href="/servlet/linkout?suffix=cit15c&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.42.667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.42.667%26sid%3Dliteratum%253Aachs%26aulast%3DIsono%26aufirst%3DF.%26aulast%3DInukai%26aufirst%3DM.%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DHaneishi%26aufirst%3DT.%26aulast%3DKinoshita%26aufirst%3DT.%26aulast%3DKuwano%26aufirst%3DH.%26atitle%3DMureidomycins%2520A-D%252C%2520novel%2520peptidylnucleoside%2520antibiotics%2520with%2520spheroplast%2520forming%2520activity.%2520II.%2520Structural%2520elucidation%26jtitle%3DJ.%2520Antibiot.%26date%3D1989%26volume%3D42%26spage%3D667%26epage%3D673%26doi%3D10.7164%2Fantibiotics.42.667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit15d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Isono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaida, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinoshita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span> <span> </span><span class="NLM_article-title">Mureidomycins E and F, minor components of mureidomycins</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1203</span>– <span class="NLM_lpage">1207</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.46.1203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.46.1203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=8407581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK2cXjtVGrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=1993&pages=1203-1207&author=F.+Isonoauthor=Y.+Sakaidaauthor=S.+Takahashiauthor=T.+Kinoshitaauthor=T.+Nakamuraauthor=M.+Inukai&title=Mureidomycins+E+and+F%2C+minor+components+of+mureidomycins&doi=10.7164%2Fantibiotics.46.1203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15dR"><div class="casContent"><span class="casTitleNuber">15d</span><div class="casTitle"><span class="NLM_cas:atitle">Mureidomycins E and F, minor components of mureidomycins</span></div><div class="casAuthors">Isono, Fujio; Sakaida, Yoshiharu; Takahashi, Shuji; Kinoshita, Takeshi; Nakamura, Takemichi; Inukai, Masatoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1203-7</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Mureidomycins (MRDs) E and F I (R1 = OH, R2 = H; R1 = H, R2 = OH, resp.) were isolated from a culture filtrate of Streptomyces flavidovirens SNAK 60486 which produces MRDs A-D.  They possessed the same mol. formulas, C39H48N8O12S and very similar UV, IR and NMR spectra, but differed clearly from each other in HPLC profile.  In the hydrolyzates of MRDs E and F, 8- and 6-hydroxy-1,2,3,4-tetrahydro-3-isoquinolinecarboxylic acid were detected, resp., which were not detected in those of MRDs A-D.  They showed strong anti-pseudomonal activity but less active than MRD A.  MRDs E and F were synthesized from MRD A and formaldehyde through Pictet-Spengler reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiih125mLEILVg90H21EOLACvtfcHk0lhebrPhWwbloA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXjtVGrtg%253D%253D&md5=621fee20a2786169666e17016d2b1a12</span></div><a href="/servlet/linkout?suffix=cit15d&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.46.1203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.46.1203%26sid%3Dliteratum%253Aachs%26aulast%3DIsono%26aufirst%3DF.%26aulast%3DSakaida%26aufirst%3DY.%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DKinoshita%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DInukai%26aufirst%3DM.%26atitle%3DMureidomycins%2520E%2520and%2520F%252C%2520minor%2520components%2520of%2520mureidomycins%26jtitle%3DJ.%2520Antibiot.%26date%3D1993%26volume%3D46%26spage%3D1203%26epage%3D1207%26doi%3D10.7164%2Fantibiotics.46.1203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadkarni, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijayakumar, E. K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangul, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fehlhaber, H.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertesy, L.</span></span> <span> </span><span class="NLM_article-title">Napsamycins, new Pseudomonas active antibiotics of the mureidomycin family from Streptomyces sp. HIL Y-82, 11372</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">595</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.47.595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.47.595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=8072167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK2cXlt12msrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1994&pages=595-598&author=S.+Chatterjeeauthor=S.+R.+Nadkarniauthor=E.+K.+S.+Vijayakumarauthor=M.+V.+Patelauthor=B.+N.+Gangulauthor=H.-W.+Fehlhaberauthor=L.+Vertesy&title=Napsamycins%2C+new+Pseudomonas+active+antibiotics+of+the+mureidomycin+family+from+Streptomyces+sp.+HIL+Y-82%2C+11372&doi=10.7164%2Fantibiotics.47.595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Napsamycins, new Pseudomonas active antibiotics of the mureidomycin family from Streptomyces sp. HIL Y-82,11372</span></div><div class="casAuthors">Chatterjee, Sugata; Nadkarni, S. R.; Vijayakumar, E. K. S.; Patel, M. V.; Ganguli, B. N.; Fehlhaber, H. W.; Vertesy, L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">595-8</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Four antibiotics specifically active against Pseudomonas were discovered in a screening program.  These antibiotics were named napsamycins A (I), B (II), C (III), and D (IV).  The fermn., isolation, structural detn. and biol. activity of these antibiotics are reported here.  The napsamycins are peptidyl nucleosides which are specific for Pseudomonas inhibition; poor activity is obsd. against other gram-neg. and -pos. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTLo0fJgIHNbVg90H21EOLACvtfcHk0lhebrPhWwbloA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlt12msrg%253D&md5=872ed505e07fd46af4cb149647ddd72c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.47.595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.47.595%26sid%3Dliteratum%253Aachs%26aulast%3DChatterjee%26aufirst%3DS.%26aulast%3DNadkarni%26aufirst%3DS.%2BR.%26aulast%3DVijayakumar%26aufirst%3DE.%2BK.%2BS.%26aulast%3DPatel%26aufirst%3DM.%2BV.%26aulast%3DGangul%26aufirst%3DB.%2BN.%26aulast%3DFehlhaber%26aufirst%3DH.-W.%26aulast%3DVertesy%26aufirst%3DL.%26atitle%3DNapsamycins%252C%2520new%2520Pseudomonas%2520active%2520antibiotics%2520of%2520the%2520mureidomycin%2520family%2520from%2520Streptomyces%2520sp.%2520HIL%2520Y-82%252C%252011372%26jtitle%3DJ.%2520Antibiot.%26date%3D1994%26volume%3D47%26spage%3D595%26epage%3D598%26doi%3D10.7164%2Fantibiotics.47.595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Karwowski, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theriault, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barlow, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maus, M. L.</span></span> <span> </span><span class="NLM_article-title">Pacidamycins, a novel series of antibiotics with anti-<i>Pseudomonas aeruginosa</i> activity. I. Taxonomy of the producing organism and fermentation</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">506</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.42.506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.42.506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=2498263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaL1MXkt1aisro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1989&pages=506-511&author=J.+P.+Karwowskiauthor=M.+Jacksonauthor=R.+J.+Theriaultauthor=R.+H.+Chenauthor=G.+J.+Barlowauthor=M.+L.+Maus&title=Pacidamycins%2C+a+novel+series+of+antibiotics+with+anti-Pseudomonas+aeruginosa+activity.+I.+Taxonomy+of+the+producing+organism+and+fermentation&doi=10.7164%2Fantibiotics.42.506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17aR"><div class="casContent"><span class="casTitleNuber">17a</span><div class="casTitle"><span class="NLM_cas:atitle">Pacidamycins, a novel series of antibiotics with anti-Pseudomonas aeruginosa activity.  I.  Taxonomy of the producing organism and fermentation</span></div><div class="casAuthors">Karwowski, J. P.; Jackson, M.; Theriault, R. J.; Chen, R. H.; Barlow, G. J.; Maus, M. L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">506-11</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">The pacidamycins are a new complex of nucleosidyl peptide antibiotics with highly specific activity against P. aeruginosa.  They are produced by Streptomyces coeruleorubidus AB 1183F-64 which was isolated from a soil sample collected at Offenburg in the FRG.  The mature spore masses of the producing organism are greenish gray to blue, and the spore chains are arranged in spirals.  After the structures of the pacidamycins were detd., the fermn. medium was supplemented with component amino acids.  This resulted in the directed biosynthesis of several members of the complex.  The overall antibiotic recovered was increased from 1∼2 mg/L to >100 mg/L through a combination of strain selection, medium manipulation, and amino acid feeding expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9SL4rQa2e67Vg90H21EOLACvtfcHk0li_BalCs_V5rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXkt1aisro%253D&md5=c86dc18e58c0d6f6660ee40f7e8ca9af</span></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.42.506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.42.506%26sid%3Dliteratum%253Aachs%26aulast%3DKarwowski%26aufirst%3DJ.%2BP.%26aulast%3DJackson%26aufirst%3DM.%26aulast%3DTheriault%26aufirst%3DR.%2BJ.%26aulast%3DChen%26aufirst%3DR.%2BH.%26aulast%3DBarlow%26aufirst%3DG.%2BJ.%26aulast%3DMaus%26aufirst%3DM.%2BL.%26atitle%3DPacidamycins%252C%2520a%2520novel%2520series%2520of%2520antibiotics%2520with%2520anti-Pseudomonas%2520aeruginosa%2520activity.%2520I.%2520Taxonomy%2520of%2520the%2520producing%2520organism%2520and%2520fermentation%26jtitle%3DJ.%2520Antibiot.%26date%3D1989%26volume%3D42%26spage%3D506%26epage%3D511%26doi%3D10.7164%2Fantibiotics.42.506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit17b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buko, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittern, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAlpine, J. B.</span></span> <span> </span><span class="NLM_article-title">Pacidamycins, a novel series of antibiotics with anti-<i>Pseudomonas aeruginosa</i> activity. II. Isolation and structural elucidation</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">512</span>– <span class="NLM_lpage">520</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.42.512</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.42.512" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=2526259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaL1MXlsFSiurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1989&pages=512-520&author=R.+H.+Chenauthor=A.+M.+Bukoauthor=D.+N.+Whitternauthor=J.+B.+McAlpine&title=Pacidamycins%2C+a+novel+series+of+antibiotics+with+anti-Pseudomonas+aeruginosa+activity.+II.+Isolation+and+structural+elucidation&doi=10.7164%2Fantibiotics.42.512"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17bR"><div class="casContent"><span class="casTitleNuber">17b</span><div class="casTitle"><span class="NLM_cas:atitle">Pacidamycins, a novel series of antibiotics with anti-Pseudomonas aeruginosa activity.  II.  Isolation and structural elucidation</span></div><div class="casAuthors">Chen, Randal H.; Buko, Alexander M.; Whittern, David N.; McAlpine, James B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">512-20</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">A novel class of antibiotics (named pacidamycins) was isolated from cultures of Streptomyces coeruleorubidus strain AB 1183F-64.  The antimicrobial activity of the pacidamycins 1-7 (I-VIII) were selective against Pseudomonas aeruginosa.  The various congeners are nucleoside peptides which differ in the terminal amino acid residues.  The structures were detd. using MS-MS and 2D NMR techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1PlXMET3hurVg90H21EOLACvtfcHk0li_BalCs_V5rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXlsFSiurk%253D&md5=dac46a68eceaff4fdc7c451f8fde1da2</span></div><a href="/servlet/linkout?suffix=cit17b&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.42.512&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.42.512%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DR.%2BH.%26aulast%3DBuko%26aufirst%3DA.%2BM.%26aulast%3DWhittern%26aufirst%3DD.%2BN.%26aulast%3DMcAlpine%26aufirst%3DJ.%2BB.%26atitle%3DPacidamycins%252C%2520a%2520novel%2520series%2520of%2520antibiotics%2520with%2520anti-Pseudomonas%2520aeruginosa%2520activity.%2520II.%2520Isolation%2520and%2520structural%2520elucidation%26jtitle%3DJ.%2520Antibiot.%26date%3D1989%26volume%3D42%26spage%3D512%26epage%3D520%26doi%3D10.7164%2Fantibiotics.42.512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit17c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R. H.</span></span> <span> </span><span class="NLM_article-title">Pacidamycins, a novel series of antibiotics with anti-<i>Pseudomonas aeruginosa</i> activity. III. Microbiologic profile</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">526</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.42.521</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.42.521" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=2498265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaL1MXlsFSnsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1989&pages=521-526&author=P.+B.+Fernandesauthor=R.+N.+Swansonauthor=D.+J.+Hardyauthor=C.+W.+Hansonauthor=L.+Coenauthor=R.+R.+Rasmussenauthor=R.+H.+Chen&title=Pacidamycins%2C+a+novel+series+of+antibiotics+with+anti-Pseudomonas+aeruginosa+activity.+III.+Microbiologic+profile&doi=10.7164%2Fantibiotics.42.521"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17cR"><div class="casContent"><span class="casTitleNuber">17c</span><div class="casTitle"><span class="NLM_cas:atitle">Pacidamycins, a novel series of antibiotics with anti-Pseudomonas aeruginosa activity.  III.  Microbiologic profile</span></div><div class="casAuthors">Fernandes, Prabhavathi B.; Swanson, Robert N.; Hardy, Dwight J.; Hanson, Charles W.; Coen, Lyle; Rasmussen, Ronald R.; Chen, Randal H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">521-6</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Pacidamycins are nucleosidyl-peptide antibiotics which have activity only against Pseudomonas aeruginosa.  Their MICs for other organisms such as Enterobacteriaceae, Staphylococcus aereus, most Streptococci and other Pseudomonas species are >100 μg/mL.  These compds. had no activity against erythromycin-susceptible Streptococci.  The MICs for Streptococcus pyogenes with constitutive- and inducible-type of macrolide-lincosamide-streptogramin resistance were 12.5 and 25 μg/mL, resp.  The MICs against P. aeruginosa ranged from 8 to 64 μg/mL.  The activity of these compds. was 1-2-fold less in serum than broth.  Time-kill curves were performed using 4 and 8 times the MIC of pacidamycin 1.  It was bactericidal against P. aeruginosa (3 log10 decrease in 4-6 h).  At 24 h, resistant mutants were found in the cultures.  The MICs of piperacillin and gentamicin for these mutants were the same as for the parent strain.  The frequency of resistance to these compds. was <3.5 × 10-6.  The resistant mutants were stable after 10 transfers in antibiotic-free medium.  The pacidamycins were inactive against P. aeruginosa in mouse protection tests.  After a single s.c. injection of 25 mg/kg of pacidamycin 1, the Cmax (peak serum concn.) was approx. 50 μg/mL and the serum half-life was 0.5 h.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnLwV7-wQ7PLVg90H21EOLACvtfcHk0lgRL0RgZPXl8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXlsFSnsLg%253D&md5=f0e403c9a17619b4650b7b7c603d94d4</span></div><a href="/servlet/linkout?suffix=cit17c&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.42.521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.42.521%26sid%3Dliteratum%253Aachs%26aulast%3DFernandes%26aufirst%3DP.%2BB.%26aulast%3DSwanson%26aufirst%3DR.%2BN.%26aulast%3DHardy%26aufirst%3DD.%2BJ.%26aulast%3DHanson%26aufirst%3DC.%2BW.%26aulast%3DCoen%26aufirst%3DL.%26aulast%3DRasmussen%26aufirst%3DR.%2BR.%26aulast%3DChen%26aufirst%3DR.%2BH.%26atitle%3DPacidamycins%252C%2520a%2520novel%2520series%2520of%2520antibiotics%2520with%2520anti-Pseudomonas%2520aeruginosa%2520activity.%2520III.%2520Microbiologic%2520profile%26jtitle%3DJ.%2520Antibiot.%26date%3D1989%26volume%3D42%26spage%3D521%26epage%3D526%26doi%3D10.7164%2Fantibiotics.42.521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit17d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Fronko, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galazzo, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberland, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malouin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. D.</span></span> <span> </span><span class="NLM_article-title">New pacidamycins produced by <i>Streptomyces coeruleorubidus</i>, NRRL 18370</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1405</span>– <span class="NLM_lpage">1410</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.53.1405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.7164%2Fantibiotics.53.1405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=11217807" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3MXosFGh" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2000&pages=1405-1410&author=R.+M.+Fronkoauthor=J.+C.+Leeauthor=J.+G.+Galazzoauthor=S.+Chamberlandauthor=F.+Malouinauthor=M.+D.+Lee&title=New+pacidamycins+produced+by+Streptomyces+coeruleorubidus%2C+NRRL+18370&doi=10.7164%2Fantibiotics.53.1405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17dR"><div class="casContent"><span class="casTitleNuber">17d</span><div class="casTitle"><span class="NLM_cas:atitle">New pacidamycins produced by Streptomyces coeruleorubidus, NRRL 18370</span></div><div class="casAuthors">Fronko, R. M.; Lee, J. C.; Galazzo, J. G.; Chamberland, S.; Malouin, F.; Lee, M. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1405-1410</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Japan Antibiotics Research Association</span>)
        </div><div class="casAbstract">Three new pacidamycins were isolated from a fermn. of Streptomyces coeruleorubidus.  The new antibiotics, pacidamycin -D, -4N, and -5T along with pacidamycins -3, -4 and -1 were isolated from fermn. broth by anion exchange chromatog. followed by cation exchange chromatog.  The structures, mol. formulas and mol. wts. of the new pacidamycins were detd. as well as their antibacterial activities.  It was found that all three new pacidamycins were active against Pseudomonas aeruginosa, but not active against gram positives and other gram neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpklF9U0J7M2bVg90H21EOLACvtfcHk0lgRL0RgZPXl8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXosFGh&md5=72098a6c1b4fb6a30687f1c0a853a62d</span></div><a href="/servlet/linkout?suffix=cit17d&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.53.1405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.53.1405%26sid%3Dliteratum%253Aachs%26aulast%3DFronko%26aufirst%3DR.%2BM.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DGalazzo%26aufirst%3DJ.%2BG.%26aulast%3DChamberland%26aufirst%3DS.%26aulast%3DMalouin%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DM.%2BD.%26atitle%3DNew%2520pacidamycins%2520produced%2520by%2520Streptomyces%2520coeruleorubidus%252C%2520NRRL%252018370%26jtitle%3DJ.%2520Antibiot.%26date%3D2000%26volume%3D53%26spage%3D1405%26epage%3D1410%26doi%3D10.7164%2Fantibiotics.53.1405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span> <span> </span><span class="NLM_article-title">A new nucleosidyl-peptide antibiotic, sansanmycin</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">158</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1038/ja.2007.16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1038%2Fja.2007.16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=17420567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjt1GktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2007&pages=158-161&author=Y.+Xieauthor=R.+Chenauthor=S.+Siauthor=C.+Sunauthor=H.+Xu&title=A+new+nucleosidyl-peptide+antibiotic%2C+sansanmycin&doi=10.1038%2Fja.2007.16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18aR"><div class="casContent"><span class="casTitleNuber">18a</span><div class="casTitle"><span class="NLM_cas:atitle">A new nucleosidyl-peptide antibiotic, sansanmycin</span></div><div class="casAuthors">Xie, Yunying; Chen, Ruxian; Si, Shuyi; Sun, Chenghang; Xu, Hongzhang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">158-161</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Japan Antibiotics Research Association</span>)
        </div><div class="casAbstract">A new nucleosidyl-peptide antibiotic, sansanmycin, was isolated from an unidentified Streptomyces sp SS.  The structure of sansanmycin was elucidated by analyses of its alk. hydrolyzate and spectroscopic analyses.  Sansanmycin exhibits antibacterial activity against Mycobacterium tuberculosis H37Ra and Pseudomonas aeruginosa with MIC values of 10 and 12.5 μg/mL, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvkPLMVwy1aLVg90H21EOLACvtfcHk0lgPNHO8JPZWKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjt1GktLs%253D&md5=2f5b20c12e958a2641ac4a71c205331f</span></div><a href="/servlet/linkout?suffix=cit18a&amp;dbid=16384&amp;doi=10.1038%2Fja.2007.16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fja.2007.16%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DSi%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DH.%26atitle%3DA%2520new%2520nucleosidyl-peptide%2520antibiotic%252C%2520sansanmycin%26jtitle%3DJ.%2520Antibiot.%26date%3D2007%26volume%3D60%26spage%3D158%26epage%3D161%26doi%3D10.1038%2Fja.2007.16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit18b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span> <span> </span><span class="NLM_article-title">Sansanmycins B and C, new components of sansanmycins</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1038/ja.2008.34</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1038%2Fja.2008.34" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=18503203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmt1Sms7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2008&pages=237-240&author=Y.+Xieauthor=H.+Xuauthor=S.+Siauthor=C.+Sunauthor=R.+Chen&title=Sansanmycins+B+and+C%2C+new+components+of+sansanmycins&doi=10.1038%2Fja.2008.34"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18bR"><div class="casContent"><span class="casTitleNuber">18b</span><div class="casTitle"><span class="NLM_cas:atitle">Sansanmycins B and C, new components of sansanmycins</span></div><div class="casAuthors">Xie, Yunying; Xu, Hongzhang; Si, Shuyi; Sun, Chenghang; Chen, Ruxian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">237-240</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Japan Antibiotics Research Association</span>)
        </div><div class="casAbstract">Two new antibiotics, sansanmycins B and C, were isolated from Streptomyces sp. SS.  Their structures were elucidated by extensive 1D and 2D NMR and MS spectral analyses.  Sansanmycins B and C exhibited inhibitory activity against our test strain of Pseudomonas aeruginosa with MIC values of 8.0 and 16 μg/mL, resp.  Sansanmycin B also exhibited inhibitory activity against Mycobacterium tuberculosis H37Rv and multi-drug-resistant Mycobacterium tuberculosis, with the MIC values ranging from 8.0 to 20 μg/mL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreqWRRIRHh7rVg90H21EOLACvtfcHk0lg8-bwoNQkvNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmt1Sms7Y%253D&md5=c4ceb4a51cc62eabc39926591086b750</span></div><a href="/servlet/linkout?suffix=cit18b&amp;dbid=16384&amp;doi=10.1038%2Fja.2008.34&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fja.2008.34%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DSi%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DR.%26atitle%3DSansanmycins%2520B%2520and%2520C%252C%2520new%2520components%2520of%2520sansanmycins%26jtitle%3DJ.%2520Antibiot.%26date%3D2008%26volume%3D61%26spage%3D237%26epage%3D240%26doi%3D10.1038%2Fja.2008.34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Isono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span> <span> </span><span class="NLM_article-title">Mureidomycin A, a new inhibitor of bacterial peptidoglycan synthesis</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">236</span>, <span class="refDoi"> DOI: 10.1128/aac.35.2.234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1128%2FAAC.35.2.234" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1902646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK3MXhs1SmurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1991&pages=234-236&author=F.+Isonoauthor=M.+Inukai&title=Mureidomycin+A%2C+a+new+inhibitor+of+bacterial+peptidoglycan+synthesis&doi=10.1128%2Faac.35.2.234"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Mureidomycin A, a new inhibitor of bacterial peptidoglycan synthesis</span></div><div class="casAuthors">Isono, Fujio; Inukai, Masatoshi</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">234-6</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    </div><div class="casAbstract">Mureidomycin A (MRD), a novel peptidylnucleoside antibiotic with antipseudomonal activity, inhibited not only peptidoglycan synthesis but also lipid-intermediate formation from UDP-N-acetylmuramyl (MurNAc)-pentapeptide and UDP-N-acetylglucosamine in an in vitro peptidoglycan-synthesizing system, using ether-treated cells of Pseudomonas aeruginosa.  Both types of inhibition by MRD disappeared when UDP-MurNAc-pentapeptide was preincubated with ether-treated cells.  Moreover, MRD completely inhibited lipid-intermediate I (undecaprenyl-p-p-MurNAc-pentapeptide) formation at a concn. below the MIC.  Apparently, the real target of MRD's action was translocase, which catalyzes lipid-intermediate I formation from UDP-MurNAc-pentapeptide and a lipid carrier.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnzLcZpNOh8LVg90H21EOLACvtfcHk0lg8-bwoNQkvNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXhs1SmurY%253D&md5=b26e05bb4b6f1f83cd3acbd09547f6bc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FAAC.35.2.234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.35.2.234%26sid%3Dliteratum%253Aachs%26aulast%3DIsono%26aufirst%3DF.%26aulast%3DInukai%26aufirst%3DM.%26atitle%3DMureidomycin%2520A%252C%2520a%2520new%2520inhibitor%2520of%2520bacterial%2520peptidoglycan%2520synthesis%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1991%26volume%3D35%26spage%3D234%26epage%3D236%26doi%3D10.1128%2Faac.35.2.234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takatsuki, A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of the bacterial translocase reaction in peptidoglycan synthesis by mureidomycins</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">980</span>– <span class="NLM_lpage">983</span>, <span class="refDoi"> DOI: 10.1128/aac.37.5.980</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1128%2FAAC.37.5.980" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=8517724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK3sXkt1Sjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1993&pages=980-983&author=M.+Inukaiauthor=F.+Isonoauthor=A.+Takatsuki&title=Selective+inhibition+of+the+bacterial+translocase+reaction+in+peptidoglycan+synthesis+by+mureidomycins&doi=10.1128%2Faac.37.5.980"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of the bacterial translocase reaction in peptidoglycan synthesis by mureidomycins</span></div><div class="casAuthors">Inukai, Masatoshi; Isono, Fujio; Takatsuki, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">980-3</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    </div><div class="casAbstract">Mureidomycins (MRDs) A and C inhibited strongly the formation of undecaprenyl pyrophosphoryl N-acetylmuramyl-pentapeptide (lipid intermediate I), which is an intermediate in bacterial peptidoglycan synthesis (50% inhibitory concn. [IC50] of MRD A, 0.05 μg/mL).  However, they did not inhibit the formation of dolichyl pyrophosphoryl N-acetylglucosamine (Dol-p-p-GlcNAc), dolichyl phosphoryl glucose, or Bacillus subtilis of lipid-linked N-acetylglucosamine for teichoic acid synthesis (IC40s, > 100 μg/mL).  In contrast, tunicamycin (TCM) inhibited strongly the formation of Dol-p-p-GlcNAc (IC50, 0.03 μg/mL) but inhibited weakly the formation of bacterial lipid intermediate I (IC50, 44 μg/mL).  When the effects of MRDs A and C and TCM on the growth of BALB/3T3 cells at 10 μg/mL.  On the basis of these results, it was concluded that MRDs are the first specific and potent inhibitors of the translocase reaction in bacterial peptidoglycan synthesis, showing a high level of toxicity against bacteria and a low level of toxicity against mammalian cells.  A specific inhibitor of translocase could be a potent antibiotic with highly selective toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZQqVu8KkaC7Vg90H21EOLACvtfcHk0lg8-bwoNQkvNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkt1Sjs7c%253D&md5=f299a807a59e67dc9560c7a82a86b63f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1128%2FAAC.37.5.980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.37.5.980%26sid%3Dliteratum%253Aachs%26aulast%3DInukai%26aufirst%3DM.%26aulast%3DIsono%26aufirst%3DF.%26aulast%3DTakatsuki%26aufirst%3DA.%26atitle%3DSelective%2520inhibition%2520of%2520the%2520bacterial%2520translocase%2520reaction%2520in%2520peptidoglycan%2520synthesis%2520by%2520mureidomycins%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1993%26volume%3D37%26spage%3D980%26epage%3D983%26doi%3D10.1128%2Faac.37.5.980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, D.</span></span> <span> </span><span class="NLM_article-title">Phospho-MurNAc-pentapeptide translocase (MraY) as a target for antibacterial agents and antibacterial proteins</span>. <i>Infect. Disord.: Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.2174/187152606784112128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.2174%2F187152606784112128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=16789873" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtFers7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=85-106&author=T.+Buggauthor=A.+Lloydauthor=D.+Roper&title=Phospho-MurNAc-pentapeptide+translocase+%28MraY%29+as+a+target+for+antibacterial+agents+and+antibacterial+proteins&doi=10.2174%2F187152606784112128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21aR"><div class="casContent"><span class="casTitleNuber">21a</span><div class="casTitle"><span class="NLM_cas:atitle">Phospho-MurNAc-pentapeptide translocase (MraY) as a target for antibacterial agents and antibacterial proteins</span></div><div class="casAuthors">Bugg, Timothy D. H.; Lloyd, Adrian J.; Roper, David I.</div><div class="citationInfo"><span class="NLM_cas:title">Infectious Disorders: Drug Targets</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-106</span>CODEN:
                <span class="NLM_cas:coden">IDDTAD</span>;
        ISSN:<span class="NLM_cas:issn">1871-5265</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Phospho-MurNAc-pentapeptide translocase (MraY, translocase I) catalyzes the first step of the lipid-linked cycle of reactions of bacterial peptidoglycan biosynthesis.  MraY is the target for five families of nucleoside antibacterial natural products: the tunicamycins, the mureidomycins (also pacidamycins, napsamycins), the liposidomycins, the muraymycins, and the capuramycins.  Recent structure-activity studies on these families have led to the identification of active pharmacophores, and insight into their mechanisms of action.  This step of peptidoglycan biosynthesis is also the target for the bacteriolytic E protein from bacteriophage .vphi.X174, and for cyclic peptides of the amphomycin family which complex the undecaprenyl phosphate cosubstrate.  The mechanisms of enzyme inhibition by these agents are discussed, and the state of knowledge regarding the transmembrane structure, active site, and catalytic mechanism of MraY.  The availability of high throughput assays and prospects of MraY as an antibacterial target are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA2V6DpkrljrVg90H21EOLACvtfcHk0lj73Rq6mCXZlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtFers7Y%253D&md5=b209c55dbd91c009a292e861eebecf44</span></div><a href="/servlet/linkout?suffix=cit21a&amp;dbid=16384&amp;doi=10.2174%2F187152606784112128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F187152606784112128%26sid%3Dliteratum%253Aachs%26aulast%3DBugg%26aufirst%3DT.%26aulast%3DLloyd%26aufirst%3DA.%26aulast%3DRoper%26aufirst%3DD.%26atitle%3DPhospho-MurNAc-pentapeptide%2520translocase%2520%2528MraY%2529%2520as%2520a%2520target%2520for%2520antibacterial%2520agents%2520and%2520antibacterial%2520proteins%26jtitle%3DInfect.%2520Disord.%253A%2520Drug%2520Targets%26date%3D2006%26volume%3D6%26spage%3D85%26epage%3D106%26doi%3D10.2174%2F187152606784112128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit21b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, K.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Recent advances in antimicrobial nucleoside antibiotics targeting cell wall biosynthesis</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1039/b202149h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2Fb202149h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=12735700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslCqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2003&pages=252-273&author=K.-i.+Kimuraauthor=T.+D.+H.+Bugg&title=Recent+advances+in+antimicrobial+nucleoside+antibiotics+targeting+cell+wall+biosynthesis&doi=10.1039%2Fb202149h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21bR"><div class="casContent"><span class="casTitleNuber">21b</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in antimicrobial nucleoside antibiotics targeting cell wall biosynthesis</span></div><div class="casAuthors">Kimura, Ken-Ichi; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">252-273</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The biosynthesis of peptidoglycan is essential for all bacteria and has no counterpart in eukaryotic cells.  It is one of the prime targets for antibiotic chemotherapy, esp. phospho-MurNAc-pentapeptide translocase (translocase I) is a fascinating target in which there is no com. antibiotic.  In this review we will describe three nucleoside translocase I inhibitors, mureidomycin, tunicamycin and liposidomycin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrksNstyeT_iLVg90H21EOLACvtfcHk0lj73Rq6mCXZlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslCqtr4%253D&md5=8c6823d378a4754c368a6179b5bfa057</span></div><a href="/servlet/linkout?suffix=cit21b&amp;dbid=16384&amp;doi=10.1039%2Fb202149h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb202149h%26sid%3Dliteratum%253Aachs%26aulast%3DKimura%26aufirst%3DK.-i.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DRecent%2520advances%2520in%2520antimicrobial%2520nucleoside%2520antibiotics%2520targeting%2520cell%2520wall%2520biosynthesis%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2003%26volume%3D20%26spage%3D252%26epage%3D273%26doi%3D10.1039%2Fb202149h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bouhss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mengin-Lecreulx, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Beller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Heijenoort, J.</span></span> <span> </span><span class="NLM_article-title">Topological analysis of the MraY protein catalysing the first membrane step of peptidoglycan synthesis</span>. <i>Mol. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">576</span>– <span class="NLM_lpage">585</span>, <span class="refDoi"> DOI: 10.1046/j.1365-2958.1999.01623.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1046%2Fj.1365-2958.1999.01623.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10564498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK1MXntlGrtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=576-585&author=A.+Bouhssauthor=D.+Mengin-Lecreulxauthor=D.+Le+Bellerauthor=J.+Van+Heijenoort&title=Topological+analysis+of+the+MraY+protein+catalysing+the+first+membrane+step+of+peptidoglycan+synthesis&doi=10.1046%2Fj.1365-2958.1999.01623.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Topological analysis of the MraY protein catalyzing the first membrane step of peptidoglycan synthesis</span></div><div class="casAuthors">Bouhss, Ahmed; Mengin-Lecreulx, Dominique; Le Beller, Dominique; Van Heijenoort, Jean</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">576-585</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science Ltd.</span>)
        </div><div class="casAbstract">The two-dimensional membrane topol. of the Escherichia coli and Staphylococcus aureus MraY transferases, which catalyze the formation of the first lipid intermediate of peptidoglycan synthesis, was established using the β-lactamase fusion system.  All 28 constructed mraY-blaM fusions produced hybrid proteins.  Anal. of the ampicillin resistance of the strains with hybrids led to a common topol. model possessing 10 transmembrane segments, five cytoplasmic domains and six periplasmic domains including the N- and C-terminal ends.  The agreement between the topologies of E. coli and S. aureus, their agreement to a fair extent with predicted models and a no. of features arising from the comparative anal. of 25 orthologue sequences strongly suggested the validity of the model for all eubacterial MraYs.  The primary structure of the 10 transmembrane segments diverged among orthologues, but they retained their hydrophobicity, no. and size.  The similarity of the sequences and distribution of the five cytoplasmic domains in both models, as well as their conservation among the MraY orthologues, clearly suggested their possible involvement in substrate recognition and in the catalytic process.  Complementation tests showed that only fusions with untruncated mraY restored growth.  It was noteworthy that S. aureus MraY was functional in E. coli.  An increased MraY transferase activity was obsd. only with the untruncated hybrids from both organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqPNxoxJOkUbVg90H21EOLACvtfcHk0lhfSI6p7ooL0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntlGrtr4%253D&md5=2de5abca52068879991d91c9377f3113</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1046%2Fj.1365-2958.1999.01623.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1365-2958.1999.01623.x%26sid%3Dliteratum%253Aachs%26aulast%3DBouhss%26aufirst%3DA.%26aulast%3DMengin-Lecreulx%26aufirst%3DD.%26aulast%3DLe%2BBeller%26aufirst%3DD.%26aulast%3DVan%2BHeijenoort%26aufirst%3DJ.%26atitle%3DTopological%2520analysis%2520of%2520the%2520MraY%2520protein%2520catalysing%2520the%2520first%2520membrane%2520step%2520of%2520peptidoglycan%2520synthesis%26jtitle%3DMol.%2520Microbiol.%26date%3D1999%26volume%3D34%26spage%3D576%26epage%3D585%26doi%3D10.1046%2Fj.1365-2958.1999.01623.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bouhss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trunkfield, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mengin-Lecreulx, D.</span></span> <span> </span><span class="NLM_article-title">The biosynthesis of peptidoglycan lipid-linked intermediates</span>. <i>FEMS Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">208</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1111/j.1574-6976.2007.00089.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1111%2Fj.1574-6976.2007.00089.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=18081839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1GitL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=208-233&author=A.+Bouhssauthor=A.+E.+Trunkfieldauthor=T.+D.+H.+Buggauthor=D.+Mengin-Lecreulx&title=The+biosynthesis+of+peptidoglycan+lipid-linked+intermediates&doi=10.1111%2Fj.1574-6976.2007.00089.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The biosynthesis of peptidoglycan lipid-linked intermediates</span></div><div class="casAuthors">Bouhss, Ahmed; Trunkfield, Amy E.; Bugg, Timothy D. H.; Mengin-Lecreulx, Dominique</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">208-233</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  The biosynthesis of bacterial cell wall peptidoglycan is a complex process involving many different steps taking place in the cytoplasm (synthesis of the nucleotide precursors) and on the inner and outer sides of the cytoplasmic membrane (assembly and polymn. of the disaccharide-peptide monomer unit, resp.).  Here, the authors summarize current knowledge on the membrane steps leading to the formation of the lipid II intermediate, i.e., the substrate of the polymn. reactions.  It makes the point on past and recent data that have significantly contributed to the understanding of the biosynthesis of undecaprenyl phosphate, the carrier lipid required for the anchoring of the peptidoglycan hydrophilic units in the membrane, and to the characterization of the MraY and MurG enzymes which catalyze the successive transfers of the N-acetylmuramoylpeptide and N-acetylglucosamine moieties onto the carrier lipid, resp.  Enzyme inhibitors and antibacterial compds. interfering with these essential metabolic steps and interesting targets are also presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMhOvpdR85NrVg90H21EOLACvtfcHk0lhfSI6p7ooL0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1GitL8%253D&md5=587da9b2738c4751350c35a235c1f3f5</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6976.2007.00089.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6976.2007.00089.x%26sid%3Dliteratum%253Aachs%26aulast%3DBouhss%26aufirst%3DA.%26aulast%3DTrunkfield%26aufirst%3DA.%2BE.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26aulast%3DMengin-Lecreulx%26aufirst%3DD.%26atitle%3DThe%2520biosynthesis%2520of%2520peptidoglycan%2520lipid-linked%2520intermediates%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2008%26volume%3D32%26spage%3D208%26epage%3D233%26doi%3D10.1111%2Fj.1574-6976.2007.00089.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Al-Dabbagh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Ghachi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanot, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parquet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mengin-Lecreulx, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouhss, A.</span></span> <span> </span><span class="NLM_article-title">Active site mapping of MraY, a member of the polyprenyl-phosphate <i>N</i>-acetylhexosamine 1-phosphate transferase superfamily, catalyzing the first membrane step of peptidoglycan biosynthesis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">8919</span>– <span class="NLM_lpage">8928</span>, <span class="refDoi"> DOI: 10.1021/bi8006274</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi8006274" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptlaqur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=8919-8928&author=B.+Al-Dabbaghauthor=X.+Henryauthor=M.+El+Ghachiauthor=G.+Augerauthor=D.+Blanotauthor=C.+Parquetauthor=D.+Mengin-Lecreulxauthor=A.+Bouhss&title=Active+site+mapping+of+MraY%2C+a+member+of+the+polyprenyl-phosphate+N-acetylhexosamine+1-phosphate+transferase+superfamily%2C+catalyzing+the+first+membrane+step+of+peptidoglycan+biosynthesis&doi=10.1021%2Fbi8006274"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Active Site Mapping of MraY, a Member of the Polyprenyl-phosphate N-Acetylhexosamine 1-Phosphate Transferase Superfamily, Catalyzing the First Membrane Step of Peptidoglycan Biosynthesis</span></div><div class="casAuthors">Al-Dabbagh, Bayan; Henry, Xavier; El Ghachi, Meriem; Auger, Genevieve; Blanot, Didier; Parquet, Claudine; Mengin-Lecreulx, Dominique; Bouhss, Ahmed</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">8919-8928</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The MraY transferase is an integral membrane protein that catalyzes an essential step of peptidoglycan biosynthesis, namely the transfer of the phospho-N-acetylmuramoyl-pentapeptide motif onto the undecaprenyl phosphate carrier lipid.  It belongs to a large superfamily of eukaryotic and prokaryotic prenyl sugar transferases.  No 3D structure has been reported for any member of this superfamily, and to date MraY is the only protein that has been successfully purified to homogeneity.  Nineteen polar residues located in the five cytoplasmic segments of MraY appeared as invariants in the sequences of MraY orthologues.  A certain no. of these invariant residues were found to be conserved in the whole superfamily.  To assess the importance of these residues in the catalytic process, site-directed mutagenesis was performed using the Bacillus subtilis MraY as a model.  Fourteen residues were shown to be essential for MraY activity by an in vivo functional complementation assay using a constructed conditional mraY mutant strain.  The corresponding mutant proteins were purified and biochem. characterized.  None of these mutations did significantly affect the binding of the nucleotidic and lipidic substrates, but the kcat was dramatically reduced in almost all cases.  The important residues for activity therefore appeared to be distributed in all the cytoplasmic segments, indicating that these five regions contribute to the structure of the catalytic site.  Our data show that the D98 residue that is invariant in the whole superfamily should be involved in the deprotonation of the lipid substrate during the catalytic process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRVoho_-T00bVg90H21EOLACvtfcHk0lgSQbiHdLT8cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptlaqur8%253D&md5=bb0d877ecbefa5f3c323e9582db02b1e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fbi8006274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi8006274%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Dabbagh%26aufirst%3DB.%26aulast%3DHenry%26aufirst%3DX.%26aulast%3DEl%2BGhachi%26aufirst%3DM.%26aulast%3DAuger%26aufirst%3DG.%26aulast%3DBlanot%26aufirst%3DD.%26aulast%3DParquet%26aufirst%3DC.%26aulast%3DMengin-Lecreulx%26aufirst%3DD.%26aulast%3DBouhss%26aufirst%3DA.%26atitle%3DActive%2520site%2520mapping%2520of%2520MraY%252C%2520a%2520member%2520of%2520the%2520polyprenyl-phosphate%2520N-acetylhexosamine%25201-phosphate%2520transferase%2520superfamily%252C%2520catalyzing%2520the%2520first%2520membrane%2520step%2520of%2520peptidoglycan%2520biosynthesis%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D8919%26epage%3D8928%26doi%3D10.1021%2Fbi8006274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="note"><p class="first last">Review of MraY inhibitors;</p></div><div class="NLM_citation" id="rightTab-cit25a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Fer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Corre, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietrancosta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evrard-Todeschi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olatunji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouhss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvet-Vitale, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gravier-Pelletier, C.</span></span> <span> </span><span class="NLM_article-title">Bacterial transferase MraY, a source of inspiration towards new antibiotics</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">6013</span>– <span class="NLM_lpage">6029</span>, <span class="refDoi"> DOI: 10.2174/0929867325666180330095154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.2174%2F0929867325666180330095154" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=6013-6029&author=M.+J.+Ferauthor=L.+Le+Correauthor=N.+Pietrancostaauthor=N.+Evrard-Todeschiauthor=S.+Olatunjiauthor=A.+Bouhssauthor=S.+Calvet-Vitaleauthor=C.+Gravier-Pelletier&title=Bacterial+transferase+MraY%2C+a+source+of+inspiration+towards+new+antibiotics&doi=10.2174%2F0929867325666180330095154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25a&amp;dbid=16384&amp;doi=10.2174%2F0929867325666180330095154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867325666180330095154%26sid%3Dliteratum%253Aachs%26aulast%3DFer%26aufirst%3DM.%2BJ.%26aulast%3DLe%2BCorre%26aufirst%3DL.%26aulast%3DPietrancosta%26aufirst%3DN.%26aulast%3DEvrard-Todeschi%26aufirst%3DN.%26aulast%3DOlatunji%26aufirst%3DS.%26aulast%3DBouhss%26aufirst%3DA.%26aulast%3DCalvet-Vitale%26aufirst%3DS.%26aulast%3DGravier-Pelletier%26aufirst%3DC.%26atitle%3DBacterial%2520transferase%2520MraY%252C%2520a%2520source%2520of%2520inspiration%2520towards%2520new%2520antibiotics%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2019%26volume%3D25%26spage%3D6013%26epage%3D6029%26doi%3D10.2174%2F0929867325666180330095154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit25b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Katsuyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Medicinal Chemistry of Muraymycins, Nucleoside Antibiotics</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">123</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1248/cpb.c17-00684</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1248%2Fcpb.c17-00684" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=29386462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVKrtrfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2018&pages=123-131&author=A.+Katsuyamaauthor=S.+Ichikawa&title=Synthesis+and+Medicinal+Chemistry+of+Muraymycins%2C+Nucleoside+Antibiotics&doi=10.1248%2Fcpb.c17-00684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25bR"><div class="casContent"><span class="casTitleNuber">25b</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and medicinal chemistry of muraymycins, nucleoside antibiotics</span></div><div class="casAuthors">Katsuyama, Akira; Ichikawa, Satoshi</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">123-131</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">A review with refs.  Muraymycins, isolated from a culture broth of Streptomyces sp., are members of a class of naturally occurring nucleoside antibiotics.  They are strong inhibitors of the phospho-MurNAc-pentapeptide translocase (MraY), which is responsible for the peptidoglycan biosynthesis.  Since MraY is an essential enzyme among bacteria, muraymycins are expected to be a novel antibacterial agent.  In this review, our efforts to synthesize muraymycin D2, simplify the chem. structure, improve antibacterial spectrum, and solve the X-ray crystal structure of the muraymycin D2/MraY complex are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo3FXON1XjXrVg90H21EOLACvtfcHk0lgzv9uWWb3Izw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVKrtrfI&md5=90da55a7dc818c3b5f8da2100aed2724</span></div><a href="/servlet/linkout?suffix=cit25b&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c17-00684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c17-00684%26sid%3Dliteratum%253Aachs%26aulast%3DKatsuyama%26aufirst%3DA.%26aulast%3DIchikawa%26aufirst%3DS.%26atitle%3DSynthesis%2520and%2520Medicinal%2520Chemistry%2520of%2520Muraymycins%252C%2520Nucleoside%2520Antibiotics%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2018%26volume%3D66%26spage%3D123%26epage%3D131%26doi%3D10.1248%2Fcpb.c17-00684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit25c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wiegmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koppermann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyerer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ducho, C.</span></span> <span> </span><span class="NLM_article-title">Muraymycin nucleoside-peptide antibiotics: uridine-derived natural products as lead structures for the development of novel antibacterial agents</span>. <i>Beilstein J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">795</span>, <span class="refDoi"> DOI: 10.3762/bjoc.12.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.3762%2Fbjoc.12.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=27340469" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWhsb7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=769-795&author=D.+Wiegmannauthor=S.+Koppermannauthor=M.+Wirthauthor=G.+Niroauthor=K.+Leyererauthor=C.+Ducho&title=Muraymycin+nucleoside-peptide+antibiotics%3A+uridine-derived+natural+products+as+lead+structures+for+the+development+of+novel+antibacterial+agents&doi=10.3762%2Fbjoc.12.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25cR"><div class="casContent"><span class="casTitleNuber">25c</span><div class="casTitle"><span class="NLM_cas:atitle">Muraymycin nucleoside-peptide antibiotics: uridine-derived natural products as lead structures for the development of novel antibacterial agents</span></div><div class="casAuthors">Wiegmann, Daniel; Koppermann, Stefan; Wirth, Marius; Niro, Giuliana; Leyerer, Kristin; Ducho, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Beilstein Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">769-795</span>CODEN:
                <span class="NLM_cas:coden">BJOCBH</span>;
        ISSN:<span class="NLM_cas:issn">1860-5397</span>.
    
            (<span class="NLM_cas:orgname">Beilstein-Institut zur Foerderung der Chemischen Wissenschaften</span>)
        </div><div class="casAbstract">Muraymycins are a promising class of antimicrobial natural products.  These uridine-derived nucleoside-peptide antibiotics inhibit the bacterial membrane protein translocase I (MraY), a key enzyme in the intracellular part of peptidoglycan biosynthesis.  This review describes the structures of naturally occurring muraymycins, their mode of action, synthetic access to muraymycins and their analogs, some structure-activity relationship (SAR) studies and first insights into muraymycin biosynthesis.  It therefore provides an overview on the current state of research, as well as an outlook on possible future developments in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXk9Xt6Q4WWrVg90H21EOLACvtfcHk0lgzv9uWWb3Izw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWhsb7M&md5=a4fea89e777b397ce7cd1edb34cc6e81</span></div><a href="/servlet/linkout?suffix=cit25c&amp;dbid=16384&amp;doi=10.3762%2Fbjoc.12.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3762%252Fbjoc.12.77%26sid%3Dliteratum%253Aachs%26aulast%3DWiegmann%26aufirst%3DD.%26aulast%3DKoppermann%26aufirst%3DS.%26aulast%3DWirth%26aufirst%3DM.%26aulast%3DNiro%26aufirst%3DG.%26aulast%3DLeyerer%26aufirst%3DK.%26aulast%3DDucho%26aufirst%3DC.%26atitle%3DMuraymycin%2520nucleoside-peptide%2520antibiotics%253A%2520uridine-derived%2520natural%2520products%2520as%2520lead%2520structures%2520for%2520the%2520development%2520of%2520novel%2520antibacterial%2520agents%26jtitle%3DBeilstein%2520J.%2520Org.%2520Chem.%26date%3D2016%26volume%3D12%26spage%3D769%26epage%3D795%26doi%3D10.3762%2Fbjoc.12.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit25d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodolis, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mihalyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamshidi, S.</span></span> <span> </span><span class="NLM_article-title">Inhibition of phospho-MurNAc-pentapeptide translocase (MraY) by nucleoside natural product antibiotics, bacteriophage ϕX174 lysis protein E, and cationic antibacterial peptides</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">6340</span>– <span class="NLM_lpage">6347</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.03.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2Fj.bmc.2016.03.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=27021004" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC28XkvFCqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=6340-6347&author=T.+D.+H.+Buggauthor=M.+T.+Rodolisauthor=A.+Mihalyiauthor=S.+Jamshidi&title=Inhibition+of+phospho-MurNAc-pentapeptide+translocase+%28MraY%29+by+nucleoside+natural+product+antibiotics%2C+bacteriophage+%CF%95X174+lysis+protein+E%2C+and+cationic+antibacterial+peptides&doi=10.1016%2Fj.bmc.2016.03.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25dR"><div class="casContent"><span class="casTitleNuber">25d</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of phospho-MurNAc-pentapeptide translocase (MraY) by nucleoside natural product antibiotics, bacteriophage .vphi.X174 lysis protein E, and cationic antibacterial peptides</span></div><div class="casAuthors">Bugg, Timothy D. H.; Rodolis, Maria T.; Mihalyi, Agnes; Jamshidi, Shirin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6340-6347</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This review covers recent developments in the inhibition of translocase MraY and related phospho-GlcNAc transferases WecA and TagO, and insight into the inhibition and catalytic mechanism of this class of integral membrane proteins from the structure of Aquifex aeolicus MraY.  Recent studies have also identified a protein-protein interaction site in Escherichia coli MraY, that is targeted by bacteriophage .vphi.X174 lysis protein E, and also by cationic antimicrobial peptides contg. Arg-Trp close to their N- or C-termini.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRDrMytnSGmLVg90H21EOLACvtfcHk0lgzv9uWWb3Izw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XkvFCqtb8%253D&md5=31257f2d6c764ce79aa39a926e336fd7</span></div><a href="/servlet/linkout?suffix=cit25d&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.03.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.03.018%26sid%3Dliteratum%253Aachs%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26aulast%3DRodolis%26aufirst%3DM.%2BT.%26aulast%3DMihalyi%26aufirst%3DA.%26aulast%3DJamshidi%26aufirst%3DS.%26atitle%3DInhibition%2520of%2520phospho-MurNAc-pentapeptide%2520translocase%2520%2528MraY%2529%2520by%2520nucleoside%2520natural%2520product%2520antibiotics%252C%2520bacteriophage%2520%25CF%2595X174%2520lysis%2520protein%2520E%252C%2520and%2520cationic%2520antibacterial%2520peptides%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D6340%26epage%3D6347%26doi%3D10.1016%2Fj.bmc.2016.03.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit25e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span> <span> </span><span class="NLM_article-title">Function-oriented synthesis: how to design simplified analogues of antibacterial nucleoside natural products?</span>. <i>Chem. Rec.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1106</span>– <span class="NLM_lpage">1115</span>, <span class="refDoi"> DOI: 10.1002/tcr.201500247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1002%2Ftcr.201500247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=27027613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1OmtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=1106-1115&author=S.+Ichikawa&title=Function-oriented+synthesis%3A+how+to+design+simplified+analogues+of+antibacterial+nucleoside+natural+products%3F&doi=10.1002%2Ftcr.201500247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25eR"><div class="casContent"><span class="casTitleNuber">25e</span><div class="casTitle"><span class="NLM_cas:atitle">Function-oriented synthesis: How to design simplified analogues of antibacterial nucleoside natural products?</span></div><div class="casAuthors">Ichikawa, Satoshi</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Record</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1106-1115</span>CODEN:
                <span class="NLM_cas:coden">CRHEAK</span>;
        ISSN:<span class="NLM_cas:issn">1528-0691</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">It is important to pursue function-oriented synthesis (FOS), a strategy for the design of less structurally complex targets with comparable or superior activity that can be made in a practical manner, because compared to synthetic drugs, many biol. relevant natural products possess large and complex chem. structures that may restrict chem. modifications in a structure-activity relationship study.  In this account, we describe recent efforts to simplify complex nucleoside natural products including caprazamycines.  Considering the structure-activity relationship study with several truncated analogs, three types of simplified derivs., namely, oxazolidine, isoxazolidine, and lactam-fused isoxazolidine-contg. uridine derivs., were designed and efficiently synthesized.  These simplified derivs. have exhibited promising antibacterial activities.  A significant feature of our studies is the rational and drastic simplification of the mol. architecture of caprazamycins.  This study provides a novel strategy for the development of a new type of antibacterial agent effective against drug-resistant bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGbH1STCoqfbVg90H21EOLACvtfcHk0lii1fO-5PcR2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1OmtLo%253D&md5=0550fc70208bad8899ab47ed9558652a</span></div><a href="/servlet/linkout?suffix=cit25e&amp;dbid=16384&amp;doi=10.1002%2Ftcr.201500247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Ftcr.201500247%26sid%3Dliteratum%253Aachs%26aulast%3DIchikawa%26aufirst%3DS.%26atitle%3DFunction-oriented%2520synthesis%253A%2520how%2520to%2520design%2520simplified%2520analogues%2520of%2520antibacterial%2520nucleoside%2520natural%2520products%253F%26jtitle%3DChem.%2520Rec.%26date%3D2016%26volume%3D16%26spage%3D1106%26epage%3D1115%26doi%3D10.1002%2Ftcr.201500247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit25f"><span><span class="NLM_label">(f) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Antibacterial Nucleoside Natural Products Inhibiting Phospho-MurNAc-Pentapeptide Translocase; Chemistry and Structure-Activity Relationship</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3951</span>– <span class="NLM_lpage">3979</span>, <span class="refDoi"> DOI: 10.2174/0929867322666150818103502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.2174%2F0929867322666150818103502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=26282943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGit7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=3951-3979&author=S.+Ichikawaauthor=M.+Yamaguchiauthor=A.+Matsuda&title=Antibacterial+Nucleoside+Natural+Products+Inhibiting+Phospho-MurNAc-Pentapeptide+Translocase%3B+Chemistry+and+Structure-Activity+Relationship&doi=10.2174%2F0929867322666150818103502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25fR"><div class="casContent"><span class="casTitleNuber">25f</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial Nucleoside Natural Products Inhibiting Phospho-MurNAc-Pentapeptide Translocase; Chemistry and Structure-Activity Relationship.</span></div><div class="casAuthors">Ichikawa, S.; Yamaguchi, M.; Matsuda, A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">3951-3979</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The continued emergence of drug-resistance to existing antibacterial agents represents a severe and ongoing public health concern, which demands the discovery of new antibiotics.  However the no. of novel classes of antibacterial drugs launched in the clinic has been remarkably slow since the 1960s, and it is urgent to develop novel antibacterial agents to fight against drug-resistant bacterial pathogens.  Peptidoglycan is a component of the bacterial cell wall, which consists of a repeated N-acetylmuramic acid (MurNAc) and Nacetylglucosamine (GluNAc) polymer cross-linked with polypeptides, and is a good target for antibacterial drug discovery.  Among enzymes responsible for its biosynthesis, phospho-MurNAc-pentapeptide translocase (MraY) is a novel and promising target.  Many nucleoside natural products, which strongly inhibit MraY, have been found in nature.  This review will summarize the synthesis and biol. properties of selected MraY inhibitory nucleoside natural products and their analogs synthesized in our lab. and by others.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrObgC5iVVElLVg90H21EOLACvtfcHk0lii1fO-5PcR2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGit7nL&md5=bb572201c43b94cc3461a8809f3d164b</span></div><a href="/servlet/linkout?suffix=cit25f&amp;dbid=16384&amp;doi=10.2174%2F0929867322666150818103502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867322666150818103502%26sid%3Dliteratum%253Aachs%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DYamaguchi%26aufirst%3DM.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DAntibacterial%2520Nucleoside%2520Natural%2520Products%2520Inhibiting%2520Phospho-MurNAc-Pentapeptide%2520Translocase%253B%2520Chemistry%2520and%2520Structure-Activity%2520Relationship%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2015%26volume%3D22%26spage%3D3951%26epage%3D3979%26doi%3D10.2174%2F0929867322666150818103502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit25g"><span><span class="NLM_label">(g) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goss, R. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, K.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial nucleoside antibiotics targeting cell wall assembly: Recent advances in structure–function studies and nucleoside biosynthesis</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">279</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1039/b816215h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25g&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2FB816215H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25g&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=20111805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Sntb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2010&pages=279-304&author=M.+Winnauthor=R.+J.+M.+Gossauthor=K.-i.+Kimuraauthor=T.+D.+H.+Bugg&title=Antimicrobial+nucleoside+antibiotics+targeting+cell+wall+assembly%3A+Recent+advances+in+structure%E2%80%93function+studies+and+nucleoside+biosynthesis&doi=10.1039%2Fb816215h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25gR"><div class="casContent"><span class="casTitleNuber">25g</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial nucleoside antibiotics targeting cell wall assembly: Recent advances in structure-function studies and nucleoside biosynthesis</span></div><div class="casAuthors">Winn, Michael; Goss, Rebecca J. M.; Kimura, Ken-ichi; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">279-304</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The quest for new antibiotics, esp. those with activity against Gram-neg. bacteria, is urgent; however, very few new antibiotics have been marketed in the last 40 years, with this limited no. falling into only four new structural classes.  Several nucleoside natural product antibiotics target bacterial translocase MraY, involved in the lipid-linked cycle of peptidoglycan biosynthesis, and fungal chitin synthase.  Biosynthetic studies on the nikkomycin, caprazamycin and pacidamycin/mureidomycin families are also reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8Wne70VFcV7Vg90H21EOLACvtfcHk0liMxuOHmLid3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Sntb8%253D&md5=49a27386210ce2250334db085a7bfc67</span></div><a href="/servlet/linkout?suffix=cit25g&amp;dbid=16384&amp;doi=10.1039%2FB816215H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FB816215H%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%26aulast%3DGoss%26aufirst%3DR.%2BJ.%2BM.%26aulast%3DKimura%26aufirst%3DK.-i.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DAntimicrobial%2520nucleoside%2520antibiotics%2520targeting%2520cell%2520wall%2520assembly%253A%2520Recent%2520advances%2520in%2520structure%25E2%2580%2593function%2520studies%2520and%2520nucleoside%2520biosynthesis%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2010%26volume%3D27%26spage%3D279%26epage%3D304%26doi%3D10.1039%2Fb816215h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit25h"><span><span class="NLM_label">(h) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Nucleoside natural products and related analogs with potential therapeutic properties as antibacterial and antiviral agents</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">487</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1517/13543776.17.5.487</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25h&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1517%2F13543776.17.5.487" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25h&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksl2nt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=487-498&author=S.+Ichikawaauthor=A.+Matsuda&title=Nucleoside+natural+products+and+related+analogs+with+potential+therapeutic+properties+as+antibacterial+and+antiviral+agents&doi=10.1517%2F13543776.17.5.487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25hR"><div class="casContent"><span class="casTitleNuber">25h</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleoside natural products and related analogs with potential therapeutic properties as antibacterial and antiviral agents</span></div><div class="casAuthors">Ichikawa, Satoshi; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">487-498</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Nucleoside natural products possess a variety of interesting biol. activities, including antibacterial, antiviral and antitumor properties, and are therefore expected to be potential candidates for developing drugs.  Complex nucleoside natural products exhibiting antibacterial activity by specific inhibition of bacterial cell wall peptidoglycan biosynthesis are described.  In addn. to the class of antibacterial nucleoside natural products, the newest members of nucleoside natural products exhibiting interesting antiviral activity are briefly described.  In spite of promising properties, no nucleoside natural products and their analogs are presently used in the clinic.  A global structure-activity relationship of these classes of nucleoside natural products clearly indicates that it is very important to: i) modulate the cell entry ability; and ii) simplify hydrophilic core structures in order to reduce the size of mols. and stabilize the chem. labile structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsFnkD2P4u1bVg90H21EOLACvtfcHk0liMxuOHmLid3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksl2nt74%253D&md5=1d17b1075e0539aa05fb188f9645812b</span></div><a href="/servlet/linkout?suffix=cit25h&amp;dbid=16384&amp;doi=10.1517%2F13543776.17.5.487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.17.5.487%26sid%3Dliteratum%253Aachs%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DNucleoside%2520natural%2520products%2520and%2520related%2520analogs%2520with%2520potential%2520therapeutic%2520properties%2520as%2520antibacterial%2520and%2520antiviral%2520agents%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2007%26volume%3D17%26spage%3D487%26epage%3D498%26doi%3D10.1517%2F13543776.17.5.487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit25i"><span><span class="NLM_label">(i) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, D.</span></span> <span> </span><span class="NLM_article-title">Phospho-MurNAc-pentapeptide translocase (MraY) as a target for antibacterial agents and antibacterial proteins</span>. <i>Infect. Disord.: Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.2174/187152606784112128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25i&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.2174%2F187152606784112128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25i&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=16789873" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25i&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtFers7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=85-106&author=T.+Buggauthor=A.+Lloydauthor=D.+Roper&title=Phospho-MurNAc-pentapeptide+translocase+%28MraY%29+as+a+target+for+antibacterial+agents+and+antibacterial+proteins&doi=10.2174%2F187152606784112128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25iR"><div class="casContent"><span class="casTitleNuber">25i</span><div class="casTitle"><span class="NLM_cas:atitle">Phospho-MurNAc-pentapeptide translocase (MraY) as a target for antibacterial agents and antibacterial proteins</span></div><div class="casAuthors">Bugg, Timothy D. H.; Lloyd, Adrian J.; Roper, David I.</div><div class="citationInfo"><span class="NLM_cas:title">Infectious Disorders: Drug Targets</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-106</span>CODEN:
                <span class="NLM_cas:coden">IDDTAD</span>;
        ISSN:<span class="NLM_cas:issn">1871-5265</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Phospho-MurNAc-pentapeptide translocase (MraY, translocase I) catalyzes the first step of the lipid-linked cycle of reactions of bacterial peptidoglycan biosynthesis.  MraY is the target for five families of nucleoside antibacterial natural products: the tunicamycins, the mureidomycins (also pacidamycins, napsamycins), the liposidomycins, the muraymycins, and the capuramycins.  Recent structure-activity studies on these families have led to the identification of active pharmacophores, and insight into their mechanisms of action.  This step of peptidoglycan biosynthesis is also the target for the bacteriolytic E protein from bacteriophage .vphi.X174, and for cyclic peptides of the amphomycin family which complex the undecaprenyl phosphate cosubstrate.  The mechanisms of enzyme inhibition by these agents are discussed, and the state of knowledge regarding the transmembrane structure, active site, and catalytic mechanism of MraY.  The availability of high throughput assays and prospects of MraY as an antibacterial target are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA2V6DpkrljrVg90H21EOLACvtfcHk0liosc8eoOqfAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtFers7Y%253D&md5=b209c55dbd91c009a292e861eebecf44</span></div><a href="/servlet/linkout?suffix=cit25i&amp;dbid=16384&amp;doi=10.2174%2F187152606784112128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F187152606784112128%26sid%3Dliteratum%253Aachs%26aulast%3DBugg%26aufirst%3DT.%26aulast%3DLloyd%26aufirst%3DA.%26aulast%3DRoper%26aufirst%3DD.%26atitle%3DPhospho-MurNAc-pentapeptide%2520translocase%2520%2528MraY%2529%2520as%2520a%2520target%2520for%2520antibacterial%2520agents%2520and%2520antibacterial%2520proteins%26jtitle%3DInfect.%2520Disord.%253A%2520Drug%2520Targets%26date%3D2006%26volume%3D6%26spage%3D85%26epage%3D106%26doi%3D10.2174%2F187152606784112128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit25j"><span><span class="NLM_label">(j) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, K.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Recent advances in antimicrobial nucleoside antibiotics targeting cell wall biosynthesis</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1039/b202149h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25j&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2Fb202149h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25j&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=12735700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25j&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslCqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2003&pages=252-273&author=K.-i.+Kimuraauthor=T.+D.+H.+Bugg&title=Recent+advances+in+antimicrobial+nucleoside+antibiotics+targeting+cell+wall+biosynthesis&doi=10.1039%2Fb202149h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25jR"><div class="casContent"><span class="casTitleNuber">25j</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in antimicrobial nucleoside antibiotics targeting cell wall biosynthesis</span></div><div class="casAuthors">Kimura, Ken-Ichi; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">252-273</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The biosynthesis of peptidoglycan is essential for all bacteria and has no counterpart in eukaryotic cells.  It is one of the prime targets for antibiotic chemotherapy, esp. phospho-MurNAc-pentapeptide translocase (translocase I) is a fascinating target in which there is no com. antibiotic.  In this review we will describe three nucleoside translocase I inhibitors, mureidomycin, tunicamycin and liposidomycin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrksNstyeT_iLVg90H21EOLACvtfcHk0liosc8eoOqfAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslCqtr4%253D&md5=8c6823d378a4754c368a6179b5bfa057</span></div><a href="/servlet/linkout?suffix=cit25j&amp;dbid=16384&amp;doi=10.1039%2Fb202149h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb202149h%26sid%3Dliteratum%253Aachs%26aulast%3DKimura%26aufirst%3DK.-i.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DRecent%2520advances%2520in%2520antimicrobial%2520nucleoside%2520antibiotics%2520targeting%2520cell%2520wall%2520biosynthesis%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2003%26volume%3D20%26spage%3D252%26epage%3D273%26doi%3D10.1039%2Fb202149h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusick, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karkhoff-Schweizer, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomovskaya, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweizer, H. P.</span></span> <span> </span><span class="NLM_article-title">High-level pacidamycin resistance in <i>Pseudomonas aeruginosa</i> is mediated by an Opp oligopeptide permease encoded by the <i>opp-fabI</i> operon</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">5565</span>– <span class="NLM_lpage">5571</span>, <span class="refDoi"> DOI: 10.1128/aac.01198-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1128%2FAAC.01198-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=23979749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Kjtr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2013&pages=5565-5571&author=A.+Mistryauthor=M.+S.+Warrenauthor=J.+K.+Cusickauthor=R.+R.+Karkhoff-Schweizerauthor=O.+Lomovskayaauthor=H.+P.+Schweizer&title=High-level+pacidamycin+resistance+in+Pseudomonas+aeruginosa+is+mediated+by+an+Opp+oligopeptide+permease+encoded+by+the+opp-fabI+operon&doi=10.1128%2Faac.01198-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26aR"><div class="casContent"><span class="casTitleNuber">26a</span><div class="casTitle"><span class="NLM_cas:atitle">High-level pacidamycin resistance in Pseudomonas aeruginosa is mediated by an Opp oligopeptide permease encoded by the opp-fabI operon</span></div><div class="casAuthors">Mistry, Anita; Warren, Mark S.; Cusick, John K.; Karkhoff-Schweizer, RoxAnn R.; Lomovskaya, Olga; Schweizer, Herbert P.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5565-5571</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Pacidamycins (or uridyl peptide antibiotics) possess selective in vivo activity against Pseudomonas aeruginosa.  An important limitation for the therapeutic use of pacidamycins with P. aeruginosa is the high frequency (10-6 to 10-7) at which resistant mutants emerge.  To elucidate the mechanism(s) of this resistance, pacidamycin-resistant P. aeruginosa mutants were isolated.  Two types of mutants were obtained.  Type 1, or high-level resistance mutants with a pacidamycin MIC of 512 μg/mL, were more abundant, with a frequency of ∼2 × 10-6, and did not show cross-resistance with other antibiotics.  Type 2, low-level resistance mutants, were isolated with a frequency of ∼10-8 and had a pacidamycin MIC of 64 μg/mL (the MIC for the wild-type strain was 4 to 16 μg/mL).  These mutants were cross-resistant to levofloxacin, tetracycline, and erythromycin and were shown to overexpress either the MexAB-OprM or MexCD-OprJ multidrug resistance efflux pumps.  High-level resistant mutants were isolated by transposon mutagenesis and one insertion was localized to oppB, one of two periplasmic binding protein components of an oligopeptide transport system which is encoded by the opp-fabI operon.  The Opp system is required for uptake of pacidamycin across the inner membrane, since various opp, but not fabI, mutants were resistant to high levels of pacidamycin.  Both of the two putative Opp periplasmic binding proteins, OppA and OppB, were required for pacidamycin uptake.  Although both impaired uptake into and efflux from the cell can cause pacidamycin resistance in P. aeruginosa, our data suggest that impaired uptake is the primary reason for the high-frequency and high-level pacidamycin resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfDLaUEW4Ib7Vg90H21EOLACvtfcHk0livGECHtGFUzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Kjtr7L&md5=5f45df1642af5825a9545b0ac6e4d228</span></div><a href="/servlet/linkout?suffix=cit26a&amp;dbid=16384&amp;doi=10.1128%2FAAC.01198-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01198-13%26sid%3Dliteratum%253Aachs%26aulast%3DMistry%26aufirst%3DA.%26aulast%3DWarren%26aufirst%3DM.%2BS.%26aulast%3DCusick%26aufirst%3DJ.%2BK.%26aulast%3DKarkhoff-Schweizer%26aufirst%3DR.%2BR.%26aulast%3DLomovskaya%26aufirst%3DO.%26aulast%3DSchweizer%26aufirst%3DH.%2BP.%26atitle%3DHigh-level%2520pacidamycin%2520resistance%2520in%2520Pseudomonas%2520aeruginosa%2520is%2520mediated%2520by%2520an%2520Opp%2520oligopeptide%2520permease%2520encoded%2520by%2520the%2520opp-fabI%2520operon%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2013%26volume%3D57%26spage%3D5565%26epage%3D5571%26doi%3D10.1128%2Faac.01198-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit26b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Pletzer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubiley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köhler, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, M. G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severinov, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weingart, H.</span></span> <span> </span><span class="NLM_article-title">The <i>Pseudomonas aeruginosa</i> PA14 ABC transporter NppA1A2BCD is required for uptake of peptidyl nucleoside antibiotics</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>197</i></span>,  <span class="NLM_fpage">2217</span>– <span class="NLM_lpage">2228</span>, <span class="refDoi"> DOI: 10.1128/jb.00234-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1128%2FJB.00234-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=25917903" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVGgtL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=197&publication_year=2015&pages=2217-2228&author=D.+Pletzerauthor=Y.+Braunauthor=S.+Dubileyauthor=C.+Lafonauthor=T.+K%C3%B6hlerauthor=M.+G.+P.+Pageauthor=M.+Mourezauthor=K.+Severinovauthor=H.+Weingart&title=The+Pseudomonas+aeruginosa+PA14+ABC+transporter+NppA1A2BCD+is+required+for+uptake+of+peptidyl+nucleoside+antibiotics&doi=10.1128%2Fjb.00234-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26bR"><div class="casContent"><span class="casTitleNuber">26b</span><div class="casTitle"><span class="NLM_cas:atitle">The Pseudomonas aeruginosa PA14 ABC transporter NppA1A2BCD is required for uptake of peptidyl nucleoside antibiotics</span></div><div class="casAuthors">Pletzer, Daniel; Braun, Yvonne; Dubiley, Svetlana; Lafon, Corinne; Koehler, Thilo; Page, Malcolm G. P.; Mourez, Michael; Severinov, Konstantin; Weingart, Helge</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">197</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2217-2228</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">1098-5530</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Anal. of the genome sequence of Pseudomonas aeruginosa PA14 revealed the presence of an operon encoding an ABC-type transporter (NppA1A2BCD) showing homol. to the Yej transporter of Escherichia coli.  The Yej transporter is involved in the uptake of the peptide-nucleotide antibiotic microcin C, a translation inhibitor that targets the enzyme aspartyl-tRNA synthetase.  Furthermore, it was recently shown that the Opp transporter from P. aeruginosa PAO1, which is identical to Npp, is required for uptake of the uridyl peptide antibiotic pacidamycin, which targets the enzyme translocase I (MraY), which is involved in peptidoglycan synthesis.  The authors used several approaches to further explore the substrate specificity of the Npp transporter.  Assays of growth in defined minimal medium contg. peptides of various lengths and amino acid compns. as sole nitrogen sources, as well as Biolog Phenotype MicroArrays, showed that the Npp transporter is not required for di-, tri-, and oligopeptide uptake.  Overexpression of the npp operon increased susceptibility not just to pacidamycin but also to nickel chloride and the peptidyl nucleoside antibiotic blasticidin S.  Furthermore, heterologous expression of the npp operon in a yej-deficient mutant of E. coli resulted in increased susceptibility to albomycin, a naturally occurring sideromycin with a peptidyl nucleoside antibiotic.  Addnl., heterologous expression showed that microcin C is recognized by the P. aeruginosa Npp system.  Thus, these results suggest that the NppA1A2BCD transporter is involved in the uptake of peptidyl nucleoside antibiotics by P. aeruginosa PA14.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbpXQyntxmhLVg90H21EOLACvtfcHk0livGECHtGFUzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVGgtL3J&md5=122c04b5145b955316d2ab5674a17b4d</span></div><a href="/servlet/linkout?suffix=cit26b&amp;dbid=16384&amp;doi=10.1128%2FJB.00234-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.00234-15%26sid%3Dliteratum%253Aachs%26aulast%3DPletzer%26aufirst%3DD.%26aulast%3DBraun%26aufirst%3DY.%26aulast%3DDubiley%26aufirst%3DS.%26aulast%3DLafon%26aufirst%3DC.%26aulast%3DK%25C3%25B6hler%26aufirst%3DT.%26aulast%3DPage%26aufirst%3DM.%2BG.%2BP.%26aulast%3DMourez%26aufirst%3DM.%26aulast%3DSeverinov%26aufirst%3DK.%26aulast%3DWeingart%26aufirst%3DH.%26atitle%3DThe%2520Pseudomonas%2520aeruginosa%2520PA14%2520ABC%2520transporter%2520NppA1A2BCD%2520is%2520required%2520for%2520uptake%2520of%2520peptidyl%2520nucleoside%2520antibiotics%26jtitle%3DJ.%2520Bacteriol.%26date%3D2015%26volume%3D197%26spage%3D2217%26epage%3D2228%26doi%3D10.1128%2Fjb.00234-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakagami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togame, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Total Synthesis of Pacidamycin D by Cu(I)-Catalyzed Oxy Enamide Formation</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">5240</span>– <span class="NLM_lpage">5243</span>, <span class="refDoi"> DOI: 10.1021/ol202124b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol202124b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCgsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=5240-5243&author=K.+Okamotoauthor=M.+Sakagamiauthor=F.+Fengauthor=H.+Togameauthor=H.+Takemotoauthor=S.+Ichikawaauthor=A.+Matsuda&title=Total+Synthesis+of+Pacidamycin+D+by+Cu%28I%29-Catalyzed+Oxy+Enamide+Formation&doi=10.1021%2Fol202124b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Total Synthesis of Pacidamycin D by Cu(I)-Catalyzed Oxy Enamide Formation</span></div><div class="casAuthors">Okamoto, Kazuya; Sakagami, Masahiro; Feng, Fei; Togame, Hiroko; Takemoto, Hiroshi; Ichikawa, Satoshi; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5240-5243</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The first total synthesis of pacidamycin D, which is expected to be a good candidate as an antibacterial agent against P. aeruginosa, is described.  The key elements of this synthetic approach feature an efficient and stereocontrolled construction of the Z-oxyvinyl iodide and copper-catalyzed cross-coupling with the tetrapeptide carboxamide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyYNL8uT0xTLVg90H21EOLACvtfcHk0ljkb99ToYP5NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCgsLrJ&md5=1fea3a123b0d5f46fcd4a556f28207ad</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fol202124b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol202124b%26sid%3Dliteratum%253Aachs%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DSakagami%26aufirst%3DM.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DTogame%26aufirst%3DH.%26aulast%3DTakemoto%26aufirst%3DH.%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DTotal%2520Synthesis%2520of%2520Pacidamycin%2520D%2520by%2520Cu%2528I%2529-Catalyzed%2520Oxy%2520Enamide%2520Formation%26jtitle%3DOrg.%2520Lett.%26date%3D2011%26volume%3D13%26spage%3D5240%26epage%3D5243%26doi%3D10.1021%2Fol202124b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakagami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togame, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Total Synthesis and Biological Evaluation of Pacidamycin D and Its 3′-Hydroxy Analogue</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">1367</span>– <span class="NLM_lpage">1377</span>, <span class="refDoi"> DOI: 10.1021/jo202159q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo202159q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1GlsL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2012&pages=1367-1377&author=K.+Okamotoauthor=M.+Sakagamiauthor=F.+Fengauthor=H.+Togameauthor=H.+Takemotoauthor=S.+Ichikawaauthor=A.+Matsuda&title=Total+Synthesis+and+Biological+Evaluation+of+Pacidamycin+D+and+Its+3%E2%80%B2-Hydroxy+Analogue&doi=10.1021%2Fjo202159q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Total Synthesis and Biological Evaluation of Pacidamycin D and Its 3'-Hydroxy Analogue</span></div><div class="casAuthors">Okamoto, Kazuya; Sakagami, Masahiro; Feng, Fei; Togame, Hiroko; Takemoto, Hiroshi; Ichikawa, Satoshi; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1367-1377</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">(compd. structures identified in this abstr. correspond to Roman numerals in graphic.).  Full details of the total synthesis of pacidamycin D (4) and its 3'-hydroxy analog 32 are described.  The chem. labile Z-oxyacyl enamide moiety is the most challenging chem. structure found in uridylpeptide natural products.  Key elements of our approach to the synthesis of 4 include the efficient and stereocontrolled construction of the Z-oxyvinyl halides 6 and 7 and their copper-catalyzed cross-coupling with the tetrapeptide carboxamide 5, a thermally unstable compd. contg. a no. of potentially reactive functional groups.  This synthetic route also allowed us to easily prep. 3'-hydroxy analog 32.  The assemblage by cross-coupling of the Z-oxyvinyl halide 6 and the carboxamide 5 at a late stage of the synthesis provided ready access to a range of uridylpeptide antibiotics and their analogs, despite their inherent labile nature with potential epimerization, simply by altering the tetrapeptide moiety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLrAm5Px3svLVg90H21EOLACvtfcHk0ljkb99ToYP5NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1GlsL%252FE&md5=4de3744da72fad4ea807629e8d51e645</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjo202159q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo202159q%26sid%3Dliteratum%253Aachs%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DSakagami%26aufirst%3DM.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DTogame%26aufirst%3DH.%26aulast%3DTakemoto%26aufirst%3DH.%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DTotal%2520Synthesis%2520and%2520Biological%2520Evaluation%2520of%2520Pacidamycin%2520D%2520and%2520Its%25203%25E2%2580%25B2-Hydroxy%2520Analogue%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2012%26volume%3D77%26spage%3D1367%26epage%3D1377%26doi%3D10.1021%2Fjo202159q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gentle, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Structure-function studies on nucleoside antibiotic mureidomycin A: synthesis of 5′-functionalised uridine models</span>. <i>J. Chem. Soc., Perkin Trans. 1</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_fpage">1287</span>– <span class="NLM_lpage">1294</span>, <span class="refDoi"> DOI: 10.1039/a901287g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2Fa901287g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK1MXjtVegtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1999&pages=1287-1294&author=C.+A.+Gentleauthor=S.+A.+Harrisonauthor=M.+Inukaiauthor=T.+D.+H.+Bugg&title=Structure-function+studies+on+nucleoside+antibiotic+mureidomycin+A%3A+synthesis+of+5%E2%80%B2-functionalised+uridine+models&doi=10.1039%2Fa901287g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-function studies on nucleoside antibiotic mureidomycin A: synthesis of 5'-functionalized uridine models</span></div><div class="casAuthors">Gentle, Caragh A.; Harrison, Stephen A.; Inukai, Masatoshi; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Perkin Transactions 1: Organic and Bio-Organic Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1287-1294</span>CODEN:
                <span class="NLM_cas:coden">JCPRB4</span>;
        ISSN:<span class="NLM_cas:issn">0300-922X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The importance of functional groups in nucleoside antibiotic mureidomycin A (MRD A) for biol. activity has been examd. by derivatization of samples of the natural product, by synthesis of uridine-contg. analogs.  N-Succinyl and di- and tri-acetyl derivs. MRD A have been prepd., and were found to have reduced activity as inhibitors of E. coli translocase I.  The enamide alkene of MRD A was found to be extremely resistant towards hydrogenation by a variety of reagents.  Several 5'-functionalized uridine derivs. were synthesized from N 3-p-methoxybenzyl-2',3'-isopropylideneuridine.  A series of 5'-aminoacyl derivs. were prepd., and the 3-aminopropionyl (IC50 260 μM) and 7-aminoheptanoyl (IC50 1.5 mM) derivs. were found to act as reversible inhibitors.  An analog mimicking the carboxy terminus of MRD A was synthesized, and also acted as an inhibitor of translocase I (IC50 1.9 mM).  A phosphonate analog designed as a possible suicide inhibitor showed modest inhibition (IC50 3.7 mM), which was shown to be irreversible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ11wcWMYYq7Vg90H21EOLACvtfcHk0lhL1Yzu-GaGew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtVegtLk%253D&md5=640e78e2d719d3f5246c8383405bc470</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2Fa901287g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fa901287g%26sid%3Dliteratum%253Aachs%26aulast%3DGentle%26aufirst%3DC.%2BA.%26aulast%3DHarrison%26aufirst%3DS.%2BA.%26aulast%3DInukai%26aufirst%3DM.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DStructure-function%2520studies%2520on%2520nucleoside%2520antibiotic%2520mureidomycin%2520A%253A%2520synthesis%2520of%25205%25E2%2580%25B2-functionalised%2520uridine%2520models%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25201%26date%3D1999%26spage%3D1287%26epage%3D1294%26doi%3D10.1039%2Fa901287g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bozzoli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazmierski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasquarello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pecunioso, A.</span></span> <span> </span><span class="NLM_article-title">A solid-phase approach to analogues of the antibiotic mureidomycin</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2759</span>– <span class="NLM_lpage">2763</span>, <span class="refDoi"> DOI: 10.1016/s0960-894x(00)00560-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2FS0960-894X%2800%2900560-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=11133085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosFeltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=2759-2763&author=A.+Bozzoliauthor=W.+Kazmierskiauthor=G.+Kennedyauthor=A.+Pasquarelloauthor=A.+Pecunioso&title=A+solid-phase+approach+to+analogues+of+the+antibiotic+mureidomycin&doi=10.1016%2Fs0960-894x%2800%2900560-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A solid-phase approach to analogues of the antibiotic mureidomycin</span></div><div class="casAuthors">Bozzoli, Andrea; Kazmierski, Wieslaw; Kennedy, Gordon; Pasquarello, Alessandra; Pecunioso, Angelo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">2759-2763</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A library of 80 analogs of the antibacterial family of mureidomycins [(I); R = H, mureidomycin A; R = Gly, mureidomycin C], was prepd. using solid-phase chem. techniques.  Analog fragments (2,3-diaminopropionic acid, methionine, p-tyrosine, m-tyrosine, as enantiomers, and tert-Bu 2-Ph malonate, as a racemate), were used in the synthesis.  A selection of ten of the compds. with best a/a values (by LC-MS anal.) was presented.  No inhibitory activity on the growth of S. aureus 853, E coli 1952, Saccharomyces cerevisiae NCY81, P. aeruginosa and P aeruginosa 2033 were detected for any library member.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfJCQoDsV6VbVg90H21EOLACvtfcHk0lhL1Yzu-GaGew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosFeltro%253D&md5=72a44c2c88635f3baf5f0b07132305be</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2800%2900560-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252800%252900560-6%26sid%3Dliteratum%253Aachs%26aulast%3DBozzoli%26aufirst%3DA.%26aulast%3DKazmierski%26aufirst%3DW.%26aulast%3DKennedy%26aufirst%3DG.%26aulast%3DPasquarello%26aufirst%3DA.%26aulast%3DPecunioso%26aufirst%3DA.%26atitle%3DA%2520solid-phase%2520approach%2520to%2520analogues%2520of%2520the%2520antibiotic%2520mureidomycin%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2000%26volume%3D10%26spage%3D2759%26epage%3D2763%26doi%3D10.1016%2Fs0960-894x%2800%2900560-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boojamra, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemoine, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Léger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernier, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomovskaya, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberland, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecker, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V. J.</span></span> <span> </span><span class="NLM_article-title">Stereochemical elucidation and total synthesis of dihydropacidamycin D, a semisynthetic pacidamycin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">870</span>– <span class="NLM_lpage">874</span>, <span class="refDoi"> DOI: 10.1021/ja003292c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja003292c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivVWlug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2001&pages=870-874&author=C.+G.+Boojamraauthor=R.+C.+Lemoineauthor=J.+C.+Leeauthor=R.+L%C3%A9gerauthor=K.+A.+Steinauthor=N.+G.+Vernierauthor=A.+Magonauthor=O.+Lomovskayaauthor=P.+K.+Martinauthor=S.+Chamberlandauthor=M.+D.+Leeauthor=S.+J.+Heckerauthor=V.+J.+Lee&title=Stereochemical+elucidation+and+total+synthesis+of+dihydropacidamycin+D%2C+a+semisynthetic+pacidamycin&doi=10.1021%2Fja003292c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Stereochemical elucidation and total synthesis of dihydropacidamycin D, a semisynthetic pacidamycin</span></div><div class="casAuthors">Boojamra, Constantine G.; Lemoine, Remy C.; Lee, Julie C.; Leger, Roger; Stein, Karin A.; Vernier, Nicole G.; Magon, Angela; Lomovskaya, Olga; Martin, Patrick K.; Chamberland, Suzanne; Lee, May D.; Hecker, Scott J.; Lee, Ving J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">870-874</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydrogenation of the C(4') exocyclic olefin of the pacidamycins has been shown to produce a series of semisynthetic compds., the dihydropacidamycins, with antimicrobial activity similar to that of the natural products.  Elucidation of stereochem. in the pacidamycins has been completed through a campaign of natural product degrdn. expts. in combination with the total synthesis of the lowest-mol. wt. dihydropacidamycin, dihydropacidamycin D.  The stereochem. identities of the tryptophan and two alanine residues contained in pacidamycin D have been shown to be of the natural (S) configuration, and the unique 3-methylamino-2-aminobutyric acid contained in this series of antibiotics has been shown to be of the (2S,3S) configuration.  Finally, the stereochem. obtained by hydrogenation of the C(4')-C(5') exocyclic olefin has been shown to be (R) at the C(4') nucleoside site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAex6jWLlofrVg90H21EOLACvtfcHk0lhyLZcV8PS5ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivVWlug%253D%253D&md5=0c2e28c233f558d99a2f7affa9108bd8</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fja003292c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja003292c%26sid%3Dliteratum%253Aachs%26aulast%3DBoojamra%26aufirst%3DC.%2BG.%26aulast%3DLemoine%26aufirst%3DR.%2BC.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DL%25C3%25A9ger%26aufirst%3DR.%26aulast%3DStein%26aufirst%3DK.%2BA.%26aulast%3DVernier%26aufirst%3DN.%2BG.%26aulast%3DMagon%26aufirst%3DA.%26aulast%3DLomovskaya%26aufirst%3DO.%26aulast%3DMartin%26aufirst%3DP.%2BK.%26aulast%3DChamberland%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DM.%2BD.%26aulast%3DHecker%26aufirst%3DS.%2BJ.%26aulast%3DLee%26aufirst%3DV.%2BJ.%26atitle%3DStereochemical%2520elucidation%2520and%2520total%2520synthesis%2520of%2520dihydropacidamycin%2520D%252C%2520a%2520semisynthetic%2520pacidamycin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2001%26volume%3D123%26spage%3D870%26epage%3D874%26doi%3D10.1021%2Fja003292c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lemoine, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecker, S. J.</span></span> <span> </span><span class="NLM_article-title">Synthesis of base-modified dihydropacidamycins</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1121</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1016/s0960-894x(02)00100-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2FS0960-894X%2802%2900100-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=11909731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD38XitFWhtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=1121-1123&author=R.+C.+Lemoineauthor=A.+Magonauthor=S.+J.+Hecker&title=Synthesis+of+base-modified+dihydropacidamycins&doi=10.1016%2Fs0960-894x%2802%2900100-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of base-modified dihydropacidamycins</span></div><div class="casAuthors">Lemoine, Remy C.; Magon, Angela; Hecker, Scott J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1121-1123</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The authors describe the synthesis of 1,2-di-O-acetyl-5-azido-3,5-dideoxy-α,β-L-arabinofuranose, a carbohydrate donor that was used for the synthesis of 1-(5'-amino-3',5'-dideoxy-α-L-arabinofuranosyl)uracil, the nucleoside found in dihydropacidamycin D.  The carbohydrate donor was also used for the synthesis of a set of new nucleosides that were introduced in new dihydropacidamycins.  These compds. were tested for biol. activity, and the results showed that uracil is the only base recognized by MraY.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm_0n48wLNS7Vg90H21EOLACvtfcHk0lhyLZcV8PS5ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitFWhtrs%253D&md5=7695a61500331abb4fecca6941e9f513</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2802%2900100-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252802%252900100-2%26sid%3Dliteratum%253Aachs%26aulast%3DLemoine%26aufirst%3DR.%2BC.%26aulast%3DMagon%26aufirst%3DA.%26aulast%3DHecker%26aufirst%3DS.%2BJ.%26atitle%3DSynthesis%2520of%2520base-modified%2520dihydropacidamycins%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2002%26volume%3D12%26spage%3D1121%26epage%3D1123%26doi%3D10.1016%2Fs0960-894x%2802%2900100-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boojamra, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemoine, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blais, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernier, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberland, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecker, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V. J.</span></span> <span> </span><span class="NLM_article-title">Synthetic dihydropacidamycin antibiotics: a modified spectrum of activity for the pacidamycin class</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3305</span>– <span class="NLM_lpage">3309</span>, <span class="refDoi"> DOI: 10.1016/s0960-894x(03)00682-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2FS0960-894X%2803%2900682-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=12951115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvV2ltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=3305-3309&author=C.+G.+Boojamraauthor=R.+C.+Lemoineauthor=J.+Blaisauthor=N.+G.+Vernierauthor=K.+A.+Steinauthor=A.+Magonauthor=S.+Chamberlandauthor=S.+J.+Heckerauthor=V.+J.+Lee&title=Synthetic+dihydropacidamycin+antibiotics%3A+a+modified+spectrum+of+activity+for+the+pacidamycin+class&doi=10.1016%2Fs0960-894x%2803%2900682-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic dihydropacidamycin antibiotics: a modified spectrum of activity for the pacidamycin class</span></div><div class="casAuthors">Boojamra, Constantine G.; Lemoine, Remy C.; Blais, Johanne; Vernier, Nicole G.; Stein, Karin A.; Magon, Angela; Chamberland, Suzanne; Hecker, Scott J.; Lee, Ving J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3305-3309</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Dihydropacidamycins having an antibacterial spectrum modified from that of the natural product pacidamycins and mureidomycins have been synthesized.  Synthetic dihydropacidamycins with noteworthy antibacterial activity against wild-type and resistant Escherichia coli have been identified (MIC = 4-8 μg/mL).  Some dihydropacidamycins are shown to have activity against multi-resistant clin. strains of Mycobacterium tuberculosis.  Compds. of this class are inhibitors of the cell wall biosynthetic enzyme, MraY.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2XZsZvatUTLVg90H21EOLACvtfcHk0ljT9eDDmyw8Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvV2ltL0%253D&md5=5fe29abd53588760ae84603f168f06e2</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2803%2900682-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252803%252900682-6%26sid%3Dliteratum%253Aachs%26aulast%3DBoojamra%26aufirst%3DC.%2BG.%26aulast%3DLemoine%26aufirst%3DR.%2BC.%26aulast%3DBlais%26aufirst%3DJ.%26aulast%3DVernier%26aufirst%3DN.%2BG.%26aulast%3DStein%26aufirst%3DK.%2BA.%26aulast%3DMagon%26aufirst%3DA.%26aulast%3DChamberland%26aufirst%3DS.%26aulast%3DHecker%26aufirst%3DS.%2BJ.%26aulast%3DLee%26aufirst%3DV.%2BJ.%26atitle%3DSynthetic%2520dihydropacidamycin%2520antibiotics%253A%2520a%2520modified%2520spectrum%2520of%2520activity%2520for%2520the%2520pacidamycin%2520class%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D3305%26epage%3D3309%26doi%3D10.1016%2Fs0960-894x%2803%2900682-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Howard, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Synthesis and activity of 5′-Uridinyl dipeptide analogues mimicking the amino terminal peptide chain of nucleoside antibiotic mureidomycin A</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3083</span>– <span class="NLM_lpage">3099</span>, <span class="refDoi"> DOI: 10.1016/s0968-0896(03)00270-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2FS0968-0896%2803%2900270-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=12818671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3sXks1amt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=3083-3099&author=N.+I.+Howardauthor=T.+D.+H.+Bugg&title=Synthesis+and+activity+of+5%E2%80%B2-Uridinyl+dipeptide+analogues+mimicking+the+amino+terminal+peptide+chain+of+nucleoside+antibiotic+mureidomycin+A&doi=10.1016%2Fs0968-0896%2803%2900270-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and activity of 5'-Uridinyl dipeptide analogues mimicking the amino terminal peptide chain of nucleoside antibiotic mureidomycin A</span></div><div class="casAuthors">Howard, Nigel I.; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3083-3099</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">This work reports the synthesis of a series of 5'-uridinyl dipeptides, which mimic the amino terminal chain of nucleoside antibiotic mureidomycin A.  For example, 5'-uridinyl dipeptides connected by amide linkage, I (AA = L-Phe, D-Phe, L-Tyr, Gly, β-Ala, L-Ala; R = H, Me), and dipeptides connected by ester linkage, II, were prepd. and their biol. activity as inhibitors of E. coli phospho-MurNAc-pentapeptide translocase (MraY) was evaluated.  The most active inhibitor of E. coli MraY was II (AA = Ala, R = Me), which showed 97% enzyme inhibition at 2.35 mM concn., and showed antibacterial activity at 100 μg/mL concn. against Pseudomonas putida.  Both the central N-Me amide linkage and a 5'-uridine ester linkage were required for highest biol. activity.  Enzyme inhibition was shown to be competitive with Mg2+.  It is proposed that the primary amino terminus of the inhibitor binds in place of the Mg2+ cofactor at the MraY active site, positioned via a cis-N-Me amide linkage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDF2zsLzTUQbVg90H21EOLACvtfcHk0ljT9eDDmyw8Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXks1amt7w%253D&md5=f1a35252a3da6d3eca43569a18db8171</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2803%2900270-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252803%252900270-0%26sid%3Dliteratum%253Aachs%26aulast%3DHoward%26aufirst%3DN.%2BI.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DSynthesis%2520and%2520activity%2520of%25205%25E2%2580%25B2-Uridinyl%2520dipeptide%2520analogues%2520mimicking%2520the%2520amino%2520terminal%2520peptide%2520chain%2520of%2520nucleoside%2520antibiotic%2520mureidomycin%2520A%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2003%26volume%3D11%26spage%3D3083%26epage%3D3099%26doi%3D10.1016%2Fs0968-0896%2803%2900270-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, R. E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherman, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeil, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, R. E.</span></span> <span> </span><span class="NLM_article-title">Solid-phase synthesis and biological evaluation of a uridinyl branched peptide urea library</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">6899</span>– <span class="NLM_lpage">6904</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2007.09.097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2Fj.bmcl.2007.09.097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=17962016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlWjtLzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=6899-6904&author=D.+Sunauthor=V.+Jonesauthor=E.+I.+Carsonauthor=R.+E.+B.+Leeauthor=M.+S.+Schermanauthor=M.+R.+McNeilauthor=R.+E.+Lee&title=Solid-phase+synthesis+and+biological+evaluation+of+a+uridinyl+branched+peptide+urea+library&doi=10.1016%2Fj.bmcl.2007.09.097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Solid-phase synthesis and biological evaluation of a uridinyl branched peptide urea library</span></div><div class="casAuthors">Sun, Dianqing; Jones, Victoria; Carson, Elizabeth I.; Lee, Robin E. B.; Scherman, Michael S.; McNeil, Michael R.; Lee, Richard E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6899-6904</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A 1000-member uridinyl branched peptide library was synthesized on PS-DES support using IRORI technol.  High-throughput screening of this library for anti-tuberculosis activity identified several members with a MIC90 = 12.5 μg/mL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrksRV-IXa4dbVg90H21EOLACvtfcHk0lgA85h60mGG7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlWjtLzN&md5=33e30357d583e02da8058949556b1251</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.09.097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.09.097%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DD.%26aulast%3DJones%26aufirst%3DV.%26aulast%3DCarson%26aufirst%3DE.%2BI.%26aulast%3DLee%26aufirst%3DR.%2BE.%2BB.%26aulast%3DScherman%26aufirst%3DM.%2BS.%26aulast%3DMcNeil%26aufirst%3DM.%2BR.%26aulast%3DLee%26aufirst%3DR.%2BE.%26atitle%3DSolid-phase%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520a%2520uridinyl%2520branched%2520peptide%2520urea%2520library%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D6899%26epage%3D6904%26doi%3D10.1016%2Fj.bmcl.2007.09.097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grüschow, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rackham, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newill, P. L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goss, R. J. M.</span></span> <span> </span><span class="NLM_article-title">New pacidamycin antibiotics through precursor-directed biosynthesis</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">360</span>, <span class="refDoi"> DOI: 10.1002/cbic.200800575</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1002%2Fcbic.200800575" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=19090518" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvVymtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=355-360&author=S.+Gr%C3%BCschowauthor=E.+J.+Rackhamauthor=B.+Elkinsauthor=P.+L.+A.+Newillauthor=L.+M.+Hillauthor=R.+J.+M.+Goss&title=New+pacidamycin+antibiotics+through+precursor-directed+biosynthesis&doi=10.1002%2Fcbic.200800575"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">New pacidamycin antibiotics through precursor-directed biosynthesis</span></div><div class="casAuthors">Gruschow, Sabine; Rackham, Emma J.; Elkins, Benjamin; Newill, Philip L. A.; Hill, Lionel M.; Goss, Rebecca J. M.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">355-360</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Pacidamycins, mureidomycins and napsamycins are structurally related uridyl peptide antibiotics that inhibit translocase I, an as yet clin. unexploited target.  This potentially important bioactivity coupled to the biosynthetically intriguing structure of pacidamycin make this natural product a fascinating subject for study.  A precursor-directed biosynthesis approach was employed in order to access new pacidamycin derivs.  Strikingly, the biosynthetic machinery exhibited highly relaxed substrate specificity with the majority of the tryptophan analogs that were administered; this resulted in the prodn. of new pacidamycin derivs.  Remarkably, 2-methyl-, 7-methyl-, 7-chloro- and 7-bromotryptophans produced their corresponding pacidamycin analogs in larger amts. than the natural pacidamycin.  Low levels or no incorporation was obsd. for tryptophans substituted at positions 4, 5 and 6.  The ability to generate bromo- and chloropacidamycins opens up the possibility of further functionalising these compds. through chem. cross-coupling in order to access a much larger family of derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpogTLu9TvpLrVg90H21EOLACvtfcHk0lgA85h60mGG7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvVymtr8%253D&md5=f386fd4b1b9de046b5ed1f342e121fb1</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200800575&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200800575%26sid%3Dliteratum%253Aachs%26aulast%3DGr%25C3%25BCschow%26aufirst%3DS.%26aulast%3DRackham%26aufirst%3DE.%2BJ.%26aulast%3DElkins%26aufirst%3DB.%26aulast%3DNewill%26aufirst%3DP.%2BL.%2BA.%26aulast%3DHill%26aufirst%3DL.%2BM.%26aulast%3DGoss%26aufirst%3DR.%2BJ.%2BM.%26atitle%3DNew%2520pacidamycin%2520antibiotics%2520through%2520precursor-directed%2520biosynthesis%26jtitle%3DChemBioChem%26date%3D2009%26volume%3D10%26spage%3D355%26epage%3D360%26doi%3D10.1002%2Fcbic.200800575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roy, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grüschow, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cairns, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goss, R. J. M.</span></span> <span> </span><span class="NLM_article-title">Gene expression enabling synthetic diversification of natural products: chemogenetic generation of pacidamycin analogs</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">12243</span>– <span class="NLM_lpage">12245</span>, <span class="refDoi"> DOI: 10.1021/ja1060406</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja1060406" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVaqs7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=12243-12245&author=A.+D.+Royauthor=S.+Gr%C3%BCschowauthor=N.+Cairnsauthor=R.+J.+M.+Goss&title=Gene+expression+enabling+synthetic+diversification+of+natural+products%3A+chemogenetic+generation+of+pacidamycin+analogs&doi=10.1021%2Fja1060406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Gene Expression Enabling Synthetic Diversification of Natural Products: Chemogenetic Generation of Pacidamycin Analogs</span></div><div class="casAuthors">Roy, Abhijeet Deb; Gruschow, Sabine; Cairns, Nickiwe; Goss, Rebecca J. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12243-12245</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Introduction of prnA, the halogenase gene from pyrrolnitrin biosynthesis, into Streptomyces coeruleorubidus resulted in efficient in situ chlorination of the uridyl peptide antibiotic pacidamycin.  The installed chlorine provided a selectably functionalizable handle enabling synthetic modification of the natural product using mild cross-coupling conditions in crude aq. exts. of the culture broth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBddbriQJpwbVg90H21EOLACvtfcHk0lgA85h60mGG7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVaqs7%252FK&md5=75bc048ba945187979110b021a964b2f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fja1060406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja1060406%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DA.%2BD.%26aulast%3DGr%25C3%25BCschow%26aufirst%3DS.%26aulast%3DCairns%26aufirst%3DN.%26aulast%3DGoss%26aufirst%3DR.%2BJ.%2BM.%26atitle%3DGene%2520expression%2520enabling%2520synthetic%2520diversification%2520of%2520natural%2520products%253A%2520chemogenetic%2520generation%2520of%2520pacidamycin%2520analogs%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D12243%26epage%3D12245%26doi%3D10.1021%2Fja1060406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakagami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uotani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togame, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Synthesis of pacidamycin analogues via an Ugi-multicomponent reaction</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">4810</span>– <span class="NLM_lpage">4815</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.05.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1016%2Fj.bmcl.2012.05.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=22677318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC38XotFKmu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=4810-4815&author=K.+Okamotoauthor=M.+Sakagamiauthor=F.+Fengauthor=F.+Takahashiauthor=K.+Uotaniauthor=H.+Togameauthor=H.+Takemotoauthor=S.+Ichikawaauthor=A.+Matsuda&title=Synthesis+of+pacidamycin+analogues+via+an+Ugi-multicomponent+reaction&doi=10.1016%2Fj.bmcl.2012.05.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of pacidamycin analogues via an Ugi-multicomponent reaction</span></div><div class="casAuthors">Okamoto, Kazuya; Sakagami, Masahiro; Feng, Fei; Takahashi, Fumiyo; Uotani, Kouichi; Togame, Hiroko; Takemoto, Hiroshi; Ichikawa, Satoshi; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4810-4815</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The second-generation synthesis of 3'-hydroxypacidamycin D has been accomplished via an Ugi-four component reaction at a late stage of the synthesis.  This approach provided ready access to a range of analogs including diastereomers of the diaminobutyric acid residue and hybrid-type analogs of mureidomycins.  Biol. evaluations of these analogs indicated that the stereochem. at the diaminobutyric acid residue has a crucial impact on both the MraY biochem. inhibition and whole-cell antibacterial activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHRTHdECquz7Vg90H21EOLACvtfcHk0ljat_Gi9xtong"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XotFKmu7c%253D&md5=d8f0cbed9b9c369769d09497a2166327</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.05.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.05.050%26sid%3Dliteratum%253Aachs%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DSakagami%26aufirst%3DM.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DTakahashi%26aufirst%3DF.%26aulast%3DUotani%26aufirst%3DK.%26aulast%3DTogame%26aufirst%3DH.%26aulast%3DTakemoto%26aufirst%3DH.%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DSynthesis%2520of%2520pacidamycin%2520analogues%2520via%2520an%2520Ugi-multicomponent%2520reaction%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D4810%26epage%3D4815%26doi%3D10.1016%2Fj.bmcl.2012.05.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tran, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pujari, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conroy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowman, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giltrap, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahapatra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tod, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowson, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crick, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Britton, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, R. J.</span></span> <span> </span><span class="NLM_article-title">Sansanmycin natural product analogues as potent and selective anti-mycobacterials that inhibit lipid I biosynthesis</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">14414</span>, <span class="refDoi"> DOI: 10.1038/ncomms14414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1038%2Fncomms14414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=28248311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BC1czhs12jug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=14414&author=A.+T.+Tranauthor=E.+E.+Watsonauthor=V.+Pujariauthor=T.+Conroyauthor=L.+J.+Dowmanauthor=A.+M.+Giltrapauthor=A.+Pangauthor=W.+R.+Wongauthor=R.+G.+Liningtonauthor=S.+Mahapatraauthor=J.+S.+Saundersauthor=A.+Charmanauthor=N.+P.+Westauthor=T.+D.+H.+Buggauthor=J.+Todauthor=C.+G.+Dowsonauthor=D.+I.+Roperauthor=D.+C.+Crickauthor=W.+J.+Brittonauthor=R.+J.+Payne&title=Sansanmycin+natural+product+analogues+as+potent+and+selective+anti-mycobacterials+that+inhibit+lipid+I+biosynthesis&doi=10.1038%2Fncomms14414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Sansanmycin natural product analogues as potent and selective anti-mycobacterials that inhibit lipid I biosynthesis</span></div><div class="casAuthors">Tran Anh T; Watson Emma E; Conroy Trent; Dowman Luke J; Giltrap Andrew M; Payne Richard J; Pujari Venugopal; Mahapatra Sebabrata; Crick Dean C; Pang Angel; Britton Warwick J; Wong Weng Ruh; Linington Roger G; Linington Roger G; Saunders Jessica; Charman Susan A; West Nicholas P; Bugg Timothy D H; Tod Julie; Dowson Christopher G; Roper David I</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14414</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Tuberculosis (TB) is responsible for enormous global morbidity and mortality, and current treatment regimens rely on the use of drugs that have been in use for more than 40 years.  Owing to widespread resistance to these therapies, new drugs are desperately needed to control the TB disease burden.  Herein, we describe the rapid synthesis of analogues of the sansanmycin uridylpeptide natural products that represent promising new TB drug leads.  The compounds exhibit potent and selective inhibition of Mycobacterium tuberculosis, the etiological agent of TB, both in vitro and intracellularly.  The natural product analogues are nanomolar inhibitors of Mtb phospho-MurNAc-pentapeptide translocase, the enzyme responsible for the synthesis of lipid I in mycobacteria.  This work lays the foundation for the development of uridylpeptide natural product analogues as new TB drug candidates that operate through the inhibition of peptidoglycan biosynthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSZtSA02_IoCDToTC3aBaesfW6udTcc2eaSgo-9WRTf5bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czhs12jug%253D%253D&md5=afbd88ba3b02293592ef01d0ae75e8db</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fncomms14414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms14414%26sid%3Dliteratum%253Aachs%26aulast%3DTran%26aufirst%3DA.%2BT.%26aulast%3DWatson%26aufirst%3DE.%2BE.%26aulast%3DPujari%26aufirst%3DV.%26aulast%3DConroy%26aufirst%3DT.%26aulast%3DDowman%26aufirst%3DL.%2BJ.%26aulast%3DGiltrap%26aufirst%3DA.%2BM.%26aulast%3DPang%26aufirst%3DA.%26aulast%3DWong%26aufirst%3DW.%2BR.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26aulast%3DMahapatra%26aufirst%3DS.%26aulast%3DSaunders%26aufirst%3DJ.%2BS.%26aulast%3DCharman%26aufirst%3DA.%26aulast%3DWest%26aufirst%3DN.%2BP.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26aulast%3DTod%26aufirst%3DJ.%26aulast%3DDowson%26aufirst%3DC.%2BG.%26aulast%3DRoper%26aufirst%3DD.%2BI.%26aulast%3DCrick%26aufirst%3DD.%2BC.%26aulast%3DBritton%26aufirst%3DW.%2BJ.%26aulast%3DPayne%26aufirst%3DR.%2BJ.%26atitle%3DSansanmycin%2520natural%2520product%2520analogues%2520as%2520potent%2520and%2520selective%2520anti-mycobacterials%2520that%2520inhibit%2520lipid%2520I%2520biosynthesis%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D14414%26doi%3D10.1038%2Fncomms14414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mashalidis, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaeser, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terasawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katsuyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, D.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-Y.</span></span> <span> </span><span class="NLM_article-title">Chemical logic of MraY inhibition by antibacterial nucleoside natural products</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2917</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-10957-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1038%2Fs41467-019-10957-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=31266949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BB3MzjsFSmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=2917&author=E.+H.+Mashalidisauthor=B.+Kaeserauthor=Y.+Terasawaauthor=A.+Katsuyamaauthor=D.-Y.+Kwonauthor=K.+Leeauthor=J.+Hongauthor=S.+Ichikawaauthor=S.-Y.+Lee&title=Chemical+logic+of+MraY+inhibition+by+antibacterial+nucleoside+natural+products&doi=10.1038%2Fs41467-019-10957-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical logic of MraY inhibition by antibacterial nucleoside natural products</span></div><div class="casAuthors">Mashalidis Ellene H; Kaeser Benjamin; Lee Seok-Yong; Terasawa Yuma; Katsuyama Akira; Ichikawa Satoshi; Kwon Do-Yeon; Hong Jiyong; Lee Kiyoun</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2917</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Novel antibacterial agents are needed to address the emergence of global antibiotic resistance.  MraY is a promising candidate for antibiotic development because it is the target of five classes of naturally occurring nucleoside inhibitors with potent antibacterial activity.  Although these natural products share a common uridine moiety, their core structures vary substantially and they exhibit different activity profiles.  An incomplete understanding of the structural and mechanistic basis of MraY inhibition has hindered the translation of these compounds to the clinic.  Here we present crystal structures of MraY in complex with representative members of the liposidomycin/caprazamycin, capuramycin, and mureidomycin classes of nucleoside inhibitors.  Our structures reveal cryptic druggable hot spots in the shallow inhibitor binding site of MraY that were not previously appreciated.  Structural analyses of nucleoside inhibitor binding provide insights into the chemical logic of MraY inhibition, which can guide novel approaches to MraY-targeted antibiotic design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTttHveiMTXK77NkxioyDXTfW6udTcc2eYau_PSL9Ed1rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzjsFSmuw%253D%253D&md5=78fbcd5ef6013cc5ebbc5797799e57b2</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-10957-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-10957-9%26sid%3Dliteratum%253Aachs%26aulast%3DMashalidis%26aufirst%3DE.%2BH.%26aulast%3DKaeser%26aufirst%3DB.%26aulast%3DTerasawa%26aufirst%3DY.%26aulast%3DKatsuyama%26aufirst%3DA.%26aulast%3DKwon%26aufirst%3DD.-Y.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHong%26aufirst%3DJ.%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DS.-Y.%26atitle%3DChemical%2520logic%2520of%2520MraY%2520inhibition%2520by%2520antibacterial%2520nucleoside%2520natural%2520products%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D2917%26doi%3D10.1038%2Fs41467-019-10957-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosono, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span> <span> </span><span class="NLM_article-title">Total Synthesis of Syringolin A and Improvement of Its Biological Activity</span>. <i>Angew. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">4936</span>– <span class="NLM_lpage">4939</span>, <span class="refDoi"> DOI: 10.1002/ange.201402428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1002%2Fange.201402428" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2014&pages=4936-4939&author=T.+Chibaauthor=H.+Hosonoauthor=K.+Nakagawaauthor=M.+Asakaauthor=H.+Takedaauthor=A.+Matsudaauthor=S.+Ichikawa&title=Total+Synthesis+of+Syringolin+A+and+Improvement+of+Its+Biological+Activity&doi=10.1002%2Fange.201402428"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fange.201402428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fange.201402428%26sid%3Dliteratum%253Aachs%26aulast%3DChiba%26aufirst%3DT.%26aulast%3DHosono%26aufirst%3DH.%26aulast%3DNakagawa%26aufirst%3DK.%26aulast%3DAsaka%26aufirst%3DM.%26aulast%3DTakeda%26aufirst%3DH.%26aulast%3DMatsuda%26aufirst%3DA.%26aulast%3DIchikawa%26aufirst%3DS.%26atitle%3DTotal%2520Synthesis%2520of%2520Syringolin%2520A%2520and%2520Improvement%2520of%2520Its%2520Biological%2520Activity%26jtitle%3DAngew.%2520Chem.%26date%3D2014%26volume%3D126%26spage%3D4936%26epage%3D4939%26doi%3D10.1002%2Fange.201402428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stachyra, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouhss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Heijenoort, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mengin-Lecreulx, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanot, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Beller, D.</span></span> <span> </span><span class="NLM_article-title">Fluorescence detection-based functional assay for high-throughput screening for MraY</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">897</span>– <span class="NLM_lpage">902</span>, <span class="refDoi"> DOI: 10.1128/aac.48.3.897-902.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1128%2FAAC.48.3.897-902.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=14982781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitVGht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2004&pages=897-902&author=T.+Stachyraauthor=C.+Diniauthor=P.+Ferrariauthor=A.+Bouhssauthor=J.+van+Heijenoortauthor=D.+Mengin-Lecreulxauthor=D.+Blanotauthor=J.+Bitonauthor=D.+Le+Beller&title=Fluorescence+detection-based+functional+assay+for+high-throughput+screening+for+MraY&doi=10.1128%2Faac.48.3.897-902.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence detection-based functional assay for high-throughput screening for MraY</span></div><div class="casAuthors">Stachyra, Therese; Dini, Christophe; Ferrari, Paul; Bouhss, Ahmed; Van Heijenoort, Jean; Mengin-Lecreulx, Dominique; Blanot, Didier; Biton, Jacques; Le Beller, Dominique</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">897-902</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">We have developed a novel assay specific to MraY, which catalyzes the first membrane step in the biosynthesis of bacterial cell wall peptidoglycan.  This was accomplished by using UDP-MurNAc-Nε-dansylpentapeptide, a fluorescent deriv. of the MraY nucleotide substrate, and a partially purified prepn. of MraY solubilized from membranes of an Escherichia coli overproducing strain.  Two versions of the assay were developed, one consisting of the high-pressure liq. chromatog. sepn. of the substrate and product (dansylated lipid I) and the other, without sepn. and adapted to the high-throughput format, taking advantage of the different fluorescence properties of the nucleotide and lipid I in the reaction medium.  The latter assay was validated with a set of natural and synthetic MraY inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_YmdYNHFMubVg90H21EOLACvtfcHk0litVu54F_GSBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitVGht7o%253D&md5=232a3c03a8e1ea8711e53ffad2279ab3</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1128%2FAAC.48.3.897-902.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.48.3.897-902.2004%26sid%3Dliteratum%253Aachs%26aulast%3DStachyra%26aufirst%3DT.%26aulast%3DDini%26aufirst%3DC.%26aulast%3DFerrari%26aufirst%3DP.%26aulast%3DBouhss%26aufirst%3DA.%26aulast%3Dvan%2BHeijenoort%26aufirst%3DJ.%26aulast%3DMengin-Lecreulx%26aufirst%3DD.%26aulast%3DBlanot%26aufirst%3DD.%26aulast%3DBiton%26aufirst%3DJ.%26aulast%3DLe%2BBeller%26aufirst%3DD.%26atitle%3DFluorescence%2520detection-based%2520functional%2520assay%2520for%2520high-throughput%2520screening%2520for%2520MraY%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2004%26volume%3D48%26spage%3D897%26epage%3D902%26doi%3D10.1128%2Faac.48.3.897-902.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katsuyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yakushiji, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumaru, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span> <span> </span><span class="NLM_article-title">Solid-Phase Modular Synthesis of Park Nucleotide and Lipids I and II Analogues</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">95</span>, <span class="refDoi"> DOI: 10.1248/cpb.c17-00828</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1248%2Fcpb.c17-00828" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=29311516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslOntr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2018&pages=84-95&author=A.+Katsuyamaauthor=K.+Satoauthor=F.+Yakushijiauthor=T.+Matsumaruauthor=S.+Ichikawa&title=Solid-Phase+Modular+Synthesis+of+Park+Nucleotide+and+Lipids+I+and+II+Analogues&doi=10.1248%2Fcpb.c17-00828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Solid-phase modular synthesis of park nucleotide and lipids I and II analogues</span></div><div class="casAuthors">Katsuyama, Akira; Sato, Kousuke; Yakushiji, Fumika; Matsumaru, Takanori; Ichikawa, Satoshi</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">84-95</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">A solid-phase synthesis of Park nucleotide as well as lipids I and II analogs, which is applicable to the synthesis of a range of analogs, is described in this work.  This technique allows highly functionalized macromols. to he modularly labeled.  Multiple steps are used in a short time (4d) with a single purifn. step to synthesize the mols. by solid-phase synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs4edJm7YrbrVg90H21EOLACvtfcHk0litVu54F_GSBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslOntr%252FL&md5=f3c950215cf73490dbd2d2049eafd6a1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c17-00828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c17-00828%26sid%3Dliteratum%253Aachs%26aulast%3DKatsuyama%26aufirst%3DA.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DYakushiji%26aufirst%3DF.%26aulast%3DMatsumaru%26aufirst%3DT.%26aulast%3DIchikawa%26aufirst%3DS.%26atitle%3DSolid-Phase%2520Modular%2520Synthesis%2520of%2520Park%2520Nucleotide%2520and%2520Lipids%2520I%2520and%2520II%2520Analogues%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2018%26volume%3D66%26spage%3D84%26epage%3D95%26doi%3D10.1248%2Fcpb.c17-00828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashalidis, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-Y.</span></span> <span> </span><span class="NLM_article-title">Structural insights into inhibition of lipid I production in bacterial cell wall synthesis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>533</i></span>,  <span class="NLM_fpage">557</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1038/nature17636</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1038%2Fnature17636" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=27088606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmt1eku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=533&publication_year=2016&pages=557-560&author=B.+C.+Chungauthor=E.+H.+Mashalidisauthor=T.+Taninoauthor=M.+Kimauthor=A.+Matsudaauthor=J.+Hongauthor=S.+Ichikawaauthor=S.-Y.+Lee&title=Structural+insights+into+inhibition+of+lipid+I+production+in+bacterial+cell+wall+synthesis&doi=10.1038%2Fnature17636"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into inhibition of lipid I production in bacterial cell wall synthesis</span></div><div class="casAuthors">Chung, Ben C.; Mashalidis, Ellene H.; Tanino, Tetsuya; Kim, Mijung; Matsuda, Akira; Hong, Jiyong; Ichikawa, Satoshi; Lee, Seok-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">533</span>
        (<span class="NLM_cas:issue">7604</span>),
    <span class="NLM_cas:pages">557-560</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Antibiotic-resistant bacterial infection is a serious threat to public health.  Peptidoglycan biosynthesis is a well-established target for antibiotic development.  MraY (phospho-MurNAc-pentapeptide translocase) catalyzes the first and an essential membrane step of peptidoglycan biosynthesis.  It is considered a very promising target for the development of new antibiotics, as many naturally occurring nucleoside inhibitors with antibacterial activity target this enzyme.  However, antibiotics targeting MraY have not been developed for clin. use, mainly owing to a lack of structural insight into inhibition of this enzyme.  Here, the authors present the crystal structure of MraY from Aquifex aeolicus (MraYAA) in complex with its naturally occurring inhibitor, muraymycin D2 (MD2).  They show that after binding MD2, MraYAA undergoes remarkably large conformational rearrangements near the active site, which lead to the formation of a nucleoside-binding pocket and a peptide-binding site.  MD2 binds the nucleoside-binding pocket like a two-pronged plug inserting into a socket.  Further interactions it makes in the adjacent peptide-binding site anchor MD2 to and enhance its affinity for MraYAA.  Surprisingly, MD2 does not interact with three acidic residues or the Mg2+ cofactor required for catalysis, suggesting that MD2 binds to MraYAA in a manner that overlaps with, but is distinct from, its natural substrate, UDP-MurNAc-pentapeptide.  The authors have detd. the principles of MD2 binding to MraYAA, including how it avoids the need for pyrophosphate and sugar moieties, which are essential features for substrate binding.  The conformational plasticity of MraY could be the reason that it is the target of many structurally distinct inhibitors.  These findings can inform the design of new inhibitors targeting MraY as well as its paralogs, WecA and TarO.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosfgI8BNe27LVg90H21EOLACvtfcHk0lgKlE26-dqRHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmt1eku70%253D&md5=4379cae27f228869c9439ecd9696f123</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature17636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17636%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DB.%2BC.%26aulast%3DMashalidis%26aufirst%3DE.%2BH.%26aulast%3DTanino%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DMatsuda%26aufirst%3DA.%26aulast%3DHong%26aufirst%3DJ.%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DS.-Y.%26atitle%3DStructural%2520insights%2520into%2520inhibition%2520of%2520lipid%2520I%2520production%2520in%2520bacterial%2520cell%2520wall%2520synthesis%26jtitle%3DNature%26date%3D2016%26volume%3D533%26spage%3D557%26epage%3D560%26doi%3D10.1038%2Fnature17636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillespie, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, D.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-Y.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of MraY, an essential membrane enzyme for bacterial cell wall synthesis</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">1012</span>– <span class="NLM_lpage">1016</span>, <span class="refDoi"> DOI: 10.1126/science.1236501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1126%2Fscience.1236501" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=23990562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlaktbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=1012-1016&author=B.+C.+Chungauthor=J.+Zhaoauthor=R.+A.+Gillespieauthor=D.-Y.+Kwonauthor=Z.+Guanauthor=J.+Hongauthor=P.+Zhouauthor=S.-Y.+Lee&title=Crystal+structure+of+MraY%2C+an+essential+membrane+enzyme+for+bacterial+cell+wall+synthesis&doi=10.1126%2Fscience.1236501"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of MraY, an Essential Membrane Enzyme for Bacterial Cell Wall Synthesis</span></div><div class="casAuthors">Chung, Ben C.; Zhao, Jinshi; Gillespie, Robert A.; Kwon, Do-Yeon; Guan, Ziqiang; Hong, Jiyong; Zhou, Pei; Lee, Seok-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6149</span>),
    <span class="NLM_cas:pages">1012-1016</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">MraY (phospho-MurNAc-pentapeptide translocase) is an integral membrane enzyme that catalyzes an essential step of bacterial cell wall biosynthesis: the transfer of the peptidoglycan precursor phospho-MurNAc-pentapeptide to the lipid carrier undecaprenyl phosphate.  MraY has long been considered a promising target for the development of antibiotics, but the lack of a structure has hindered mechanistic understanding of this crit. enzyme and the enzyme superfamily in general.  The superfamily includes enzymes involved in bacterial lipopolysaccharide/teichoic acid formation and eukaryotic N-linked glycosylation, modifications that are central in many biol. processes.  We present the crystal structure of MraY from Aquifex aeolicus (MraYAA) at 3.3 Å resoln., which allows us to visualize the overall architecture, locate Mg2+ within the active site, and provide a structural basis of catalysis for this class of enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry8oc1FCGbcbVg90H21EOLACvtfcHk0lgKlE26-dqRHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlaktbnE&md5=3c9d143b126c25b0430e1cf70e5f960a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fscience.1236501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1236501%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DB.%2BC.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DGillespie%26aufirst%3DR.%2BA.%26aulast%3DKwon%26aufirst%3DD.-Y.%26aulast%3DGuan%26aufirst%3DZ.%26aulast%3DHong%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DS.-Y.%26atitle%3DCrystal%2520structure%2520of%2520MraY%252C%2520an%2520essential%2520membrane%2520enzyme%2520for%2520bacterial%2520cell%2520wall%2520synthesis%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D1012%26epage%3D1016%26doi%3D10.1126%2Fscience.1236501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waterhouse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bienert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Studer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tauriello, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gumienny, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heer, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Beer, T. A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rempfer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordoli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepore, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwede, T.</span></span> <span> </span><span class="NLM_article-title">SWISS-MODEL: homology modelling of protein structures and complexes</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">W296</span>– <span class="NLM_lpage">W303</span>, <span class="refDoi"> DOI: 10.1093/nar/gky427</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1093%2Fnar%2Fgky427" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=29788355" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVyru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=W296-W303&author=A.+Waterhouseauthor=M.+Bertoniauthor=S.+Bienertauthor=G.+Studerauthor=G.+Taurielloauthor=R.+Gumiennyauthor=F.+T.+Heerauthor=T.+A.+P.+de+Beerauthor=C.+Rempferauthor=L.+Bordoliauthor=R.+Leporeauthor=T.+Schwede&title=SWISS-MODEL%3A+homology+modelling+of+protein+structures+and+complexes&doi=10.1093%2Fnar%2Fgky427"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">SWISS-MODEL: homology modelling of protein structures and complexes</span></div><div class="casAuthors">Waterhouse, Andrew; Bertoni, Martino; Bienert, Stefan; Studer, Gabriel; Tauriello, Gerardo; Gumienny, Rafal; Heer, Florian T.; de Beer, Tjaart A. P.; Rempfer, Christine; Bordoli, Lorenza; Lepore, Rosalba; Schwede, Torsten</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W296-W303</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Homol. modeling has matured into an important technique in structural biol., significantly contributing to narrowing the gap between known protein sequences and exptl. detd. structures.  Fully automated workflows and servers simplify and streamline the homol. modeling process, also allowing users without a specific computational expertise to generate reliable protein models and have easy access to modeling results, their visualization and interpretation.  Here, we present an update to the SWISS-MODEL server, which pioneered the field of automated modeling 25 years ago and been continuously further developed.  Recently, its functionality has been extended to the modeling of homo- and heteromeric complexes.  Starting from the amino acid sequences of the interacting proteins, both the stoichiometry and the overall structure of the complex are inferred by homol. modeling.  Other major improvements include the implementation of a new modeling engine, ProMod3 and the introduction a new local model quality estn. method, QMEANDisCo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf9ZhQAzPVe7Vg90H21EOLACvtfcHk0ljHpvtIbPlOCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVyru70%253D&md5=651b55ade2d4f354f5d6699f318424d0</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky427%26sid%3Dliteratum%253Aachs%26aulast%3DWaterhouse%26aufirst%3DA.%26aulast%3DBertoni%26aufirst%3DM.%26aulast%3DBienert%26aufirst%3DS.%26aulast%3DStuder%26aufirst%3DG.%26aulast%3DTauriello%26aufirst%3DG.%26aulast%3DGumienny%26aufirst%3DR.%26aulast%3DHeer%26aufirst%3DF.%2BT.%26aulast%3Dde%2BBeer%26aufirst%3DT.%2BA.%2BP.%26aulast%3DRempfer%26aufirst%3DC.%26aulast%3DBordoli%26aufirst%3DL.%26aulast%3DLepore%26aufirst%3DR.%26aulast%3DSchwede%26aufirst%3DT.%26atitle%3DSWISS-MODEL%253A%2520homology%2520modelling%2520of%2520protein%2520structures%2520and%2520complexes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3DW296%26epage%3DW303%26doi%3D10.1093%2Fnar%2Fgky427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takeoka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekiguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonezawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakagami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togame, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Expansion of antibacterial spectrum of muraymycins toward <i>Pseudomonas aeruginosa</i></span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">556</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1021/ml5000096</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml5000096" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjs1GqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=556-560&author=Y.+Takeokaauthor=T.+Taninoauthor=M.+Sekiguchiauthor=S.+Yonezawaauthor=M.+Sakagamiauthor=F.+Takahashiauthor=H.+Togameauthor=Y.+Tanakaauthor=H.+Takemotoauthor=S.+Ichikawaauthor=A.+Matsuda&title=Expansion+of+antibacterial+spectrum+of+muraymycins+toward+Pseudomonas+aeruginosa&doi=10.1021%2Fml5000096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Expansion of antibacterial spectrum of muraymycins toward Pseudomonas aeruginosa</span></div><div class="casAuthors">Takeoka, Yusuke; Tanino, Tetsuya; Sekiguchi, Mitsuaki; Yonezawa, Shuji; Sakagami, Masahiro; Takahashi, Fumiyo; Togame, Hiroko; Tanaka, Yoshikazu; Takemoto, Hiroshi; Ichikawa, Satoshi; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">556-560</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">It is urgent to develop novel anti-Pseudomonas agents that should also be active against multidrug-resistant P. aeruginosa.  Expanding the antibacterial spectrum of muraymycins toward P. aeruginosa was investigated by the systematic structure-activity relationship study.  It was revealed that two functional groups, a lipophilic side chain and a guanidino group, at the accessory moiety of muraymycins were important for the anti-Pseudomonas activity, and analog 29 exhibited antibacterial activity against a range of P. aeruginosa strains with the min. inhibitory concn. values of 4-8 μg/mL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPDfz8OY4rRbVg90H21EOLACvtfcHk0ljHpvtIbPlOCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjs1GqsL8%253D&md5=58a7f2a7d0368e0bf56691c1e5026397</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fml5000096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5000096%26sid%3Dliteratum%253Aachs%26aulast%3DTakeoka%26aufirst%3DY.%26aulast%3DTanino%26aufirst%3DT.%26aulast%3DSekiguchi%26aufirst%3DM.%26aulast%3DYonezawa%26aufirst%3DS.%26aulast%3DSakagami%26aufirst%3DM.%26aulast%3DTakahashi%26aufirst%3DF.%26aulast%3DTogame%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DTakemoto%26aufirst%3DH.%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DExpansion%2520of%2520antibacterial%2520spectrum%2520of%2520muraymycins%2520toward%2520Pseudomonas%2520aeruginosa%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D556%26epage%3D560%26doi%3D10.1021%2Fml5000096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Dabbagh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mengin-Lecreulx, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouhss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Mechanistic analysis of muraymycin analogues: a guide to the design of MraY inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8421</span>– <span class="NLM_lpage">8439</span>, <span class="refDoi"> DOI: 10.1021/jm200906r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200906r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOlt73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8421-8439&author=T.+Taninoauthor=B.+Al-Dabbaghauthor=D.+Mengin-Lecreulxauthor=A.+Bouhssauthor=H.+Oyamaauthor=S.+Ichikawaauthor=A.+Matsuda&title=Mechanistic+analysis+of+muraymycin+analogues%3A+a+guide+to+the+design+of+MraY+inhibitors&doi=10.1021%2Fjm200906r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic Analysis of Muraymycin Analogues: A Guide to the Design of MraY Inhibitors</span></div><div class="casAuthors">Tanino, Tetsuya; Al-Dabbagh, Bayan; Mengin-Lecreulx, Dominique; Bouhss, Ahmed; Oyama, Hiroshi; Ichikawa, Satoshi; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8421-8439</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The systematic structure-activity relationship (SAR) of the muraymycins (MRYs) using an Ugi four-component reaction (U4CR) was investigated.  The impact of the lipophilic substituent on antibacterial activity was significant, and the analogs 8 and 9 having a lipophilic side chain exhibited good activity against a range of Gram-pos. bacterial pathogens, including MRSA and VRE.  Further investigation of compds. 8 and 9 revealed these analogs to be selective inhibitors of the MraY transferase and nontoxic to HepG2 cells.  The SAR of the accessory urea-peptide moiety indicated that it could be simplified.  Our SAR study of the MRYs suggests a probable mechanism for inhibition of the MraY, where the inner moiety of the urea-dipeptide motif interacts with the carbohydrate recognition domain in the cytoplasmic loop 5.  The predicted binding model would provide further direction toward the design of potent MraY inhibitors.  This study has set the stage for the generation of novel antibacterial "lead" compds. based on MRYs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwPj7zgQroc7Vg90H21EOLACvtfcHk0lgFdO3SPJa_eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOlt73O&md5=82a6f8ac626297cc8df1a72c05325902</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm200906r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200906r%26sid%3Dliteratum%253Aachs%26aulast%3DTanino%26aufirst%3DT.%26aulast%3DAl-Dabbagh%26aufirst%3DB.%26aulast%3DMengin-Lecreulx%26aufirst%3DD.%26aulast%3DBouhss%26aufirst%3DA.%26aulast%3DOyama%26aufirst%3DH.%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DMechanistic%2520analysis%2520of%2520muraymycin%2520analogues%253A%2520a%2520guide%2520to%2520the%2520design%2520of%2520MraY%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8421%26epage%3D8439%26doi%3D10.1021%2Fjm200906r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Brandish, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnham, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lonsdale, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southgate, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Slow binding inhibition of phospho-<i>N</i>-acetylmuramyl-pentapeptide-translocase (<i>Escherichia coli</i>) by mureidomycin A</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>271</i></span>,  <span class="NLM_fpage">7609</span>– <span class="NLM_lpage">7614</span>, <span class="refDoi"> DOI: 10.1074/jbc.271.13.7609</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1074%2Fjbc.271.13.7609" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=8631795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK28XhvFyksbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=7609-7614&author=P.+E.+Brandishauthor=M.+K.+Burnhamauthor=J.+T.+Lonsdaleauthor=R.+Southgateauthor=M.+Inukaiauthor=T.+D.+H.+Bugg&title=Slow+binding+inhibition+of+phospho-N-acetylmuramyl-pentapeptide-translocase+%28Escherichia+coli%29+by+mureidomycin+A&doi=10.1074%2Fjbc.271.13.7609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49aR"><div class="casContent"><span class="casTitleNuber">49a</span><div class="casTitle"><span class="NLM_cas:atitle">Slow binding inhibition of phospho-N-acetylmuramyl-pentapeptide-translocase (Escherichia coli) by mureidomycin A</span></div><div class="casAuthors">Brandish, Philip E.; Burnham, Martin K.; Lonsdale, John T.; Southgate, Robert; Inukai, Masatoshi; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">271</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">7609-14</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Enzymes of the membrane cycle of reactions in bacterial peptidoglycan biosynthesis remain as unexploited potential targets for antibacterial agents.  The first of these enzymes, phospho-N-acetylmuramyl-pentapeptide-translocase (EC 2.7.8.13), has been overexpressed in Escherichia coli and solubilized from particulate fractions.  The work of W. A. Weppner and F. C. Neuhaus ((1977) J. Biol. Chem. 252, 2296-2303) has been extended to establish a usable routine fluorescence-based continuous assay for solubilized prepns.  This assay has been used in the characterization of the natural product, mureidomycin A as a potent slow binding inhibitor of the enzyme with Ki and Ki* of 36 nM and 2 nM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJBHjL8B2BZrVg90H21EOLACvtfcHk0lhLIW6BUUbURQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvFyksbo%253D&md5=899edaad9b1d4af57dd63d5ac91fa696</span></div><a href="/servlet/linkout?suffix=cit49a&amp;dbid=16384&amp;doi=10.1074%2Fjbc.271.13.7609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.271.13.7609%26sid%3Dliteratum%253Aachs%26aulast%3DBrandish%26aufirst%3DP.%2BE.%26aulast%3DBurnham%26aufirst%3DM.%2BK.%26aulast%3DLonsdale%26aufirst%3DJ.%2BT.%26aulast%3DSouthgate%26aufirst%3DR.%26aulast%3DInukai%26aufirst%3DM.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DSlow%2520binding%2520inhibition%2520of%2520phospho-N-acetylmuramyl-pentapeptide-translocase%2520%2528Escherichia%2520coli%2529%2520by%2520mureidomycin%2520A%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26spage%3D7609%26epage%3D7614%26doi%3D10.1074%2Fjbc.271.13.7609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit49b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Brandish, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inukai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southgate, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lonsdale, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D.</span></span> <span> </span><span class="NLM_article-title">Modes of action of tunicamycin, liposidomycin B, and mureidomycin A: inhibition of phospho-<i>N</i>-acetylmuramyl-pentapeptide translocase from <i>Escherichia coli</i></span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1640</span>– <span class="NLM_lpage">1644</span>, <span class="refDoi"> DOI: 10.1128/aac.40.7.1640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1128%2FAAC.40.7.1640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=8807054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK28XktVaksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1996&pages=1640-1644&author=P.+E.+Brandishauthor=K.+I.+Kimuraauthor=M.+Inukaiauthor=R.+Southgateauthor=J.+T.+Lonsdaleauthor=T.+D.+Bugg&title=Modes+of+action+of+tunicamycin%2C+liposidomycin+B%2C+and+mureidomycin+A%3A+inhibition+of+phospho-N-acetylmuramyl-pentapeptide+translocase+from+Escherichia+coli&doi=10.1128%2Faac.40.7.1640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49bR"><div class="casContent"><span class="casTitleNuber">49b</span><div class="casTitle"><span class="NLM_cas:atitle">Modes of action of tunicamycin, liposidomycin B, and mureidomycin A: inhibition of phospho-N-acetylmuramyl-pentapeptide translocase from Escherichia coli</span></div><div class="casAuthors">Brandish, Philip E.; Kimura, Ken-Ichi; Inukai, Masatoshi; Southgate, Robert; Lonsdale, John T.; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1640-1644</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Using a continuous fluorescence-based enzyme assay, the authors characterized the antibacterial agents, tunicamycin and liposidomycin B, as inhibitors of solubilized E. coli phospho-N-acetylmuramyl-pentapeptide translocase.  Tunicamycin exhibited reversible inhibition (Ki = 0.55 μM) which was noncompetitive with respect to the lipid acceptor substrate, dodecaprenyl phosphate (I), and competitive with respect to the fluorescent substrate analog, dansyl-UDPMurNAc-pentapeptide (II).  Liposidomycin B exhibited slow-binding inhibition (Ki = 80 nM) which was competitive with respect to I and noncompetitive with respect to II.  Mureidomycin A exhibited slow-binding inhibition which was competitive with respect to both I and II.  The results provided insight into the mol. mechanisms of action of these classes of nucleoside antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonrQ_H-_u_OLVg90H21EOLACvtfcHk0lhLIW6BUUbURQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XktVaksLw%253D&md5=b6172f4c155922ee6389c39265987b1a</span></div><a href="/servlet/linkout?suffix=cit49b&amp;dbid=16384&amp;doi=10.1128%2FAAC.40.7.1640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.40.7.1640%26sid%3Dliteratum%253Aachs%26aulast%3DBrandish%26aufirst%3DP.%2BE.%26aulast%3DKimura%26aufirst%3DK.%2BI.%26aulast%3DInukai%26aufirst%3DM.%26aulast%3DSouthgate%26aufirst%3DR.%26aulast%3DLonsdale%26aufirst%3DJ.%2BT.%26aulast%3DBugg%26aufirst%3DT.%2BD.%26atitle%3DModes%2520of%2520action%2520of%2520tunicamycin%252C%2520liposidomycin%2520B%252C%2520and%2520mureidomycin%2520A%253A%2520inhibition%2520of%2520phospho-N-acetylmuramyl-pentapeptide%2520translocase%2520from%2520Escherichia%2520coli%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1996%26volume%3D40%26spage%3D1640%26epage%3D1644%26doi%3D10.1128%2Faac.40.7.1640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gentle, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Role of the enamide linkage of nucleoside antibiotic mureidomycin A: synthesis and reactivity of enamide-containing analogues</span>. <i>J. Chem. Soc., Perkin Trans. 1</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_fpage">1279</span>– <span class="NLM_lpage">1286</span>, <span class="refDoi"> DOI: 10.1039/a901279f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2Fa901279f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADyaK1MXjtVegtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1999&pages=1279-1286&author=C.+A.+Gentleauthor=T.+D.+H.+Bugg&title=Role+of+the+enamide+linkage+of+nucleoside+antibiotic+mureidomycin+A%3A+synthesis+and+reactivity+of+enamide-containing+analogues&doi=10.1039%2Fa901279f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the enamide linkage of nucleoside antibiotic mureidomycin A: synthesis and reactivity of enamide-containing analogues</span></div><div class="casAuthors">Gentle, Caragh A.; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Perkin Transactions 1: Organic and Bio-Organic Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1279-1286</span>CODEN:
                <span class="NLM_cas:coden">JCPRB4</span>;
        ISSN:<span class="NLM_cas:issn">0300-922X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The reactivity of an unusual enamide functional group in the nucleoside antibiotic mureidomycin A (MRD A) has been investigated by synthesis of enamide-contg. analogs.  Enamides based on 2-methoxyethylamine and tetrahydrofurfurylamine were found to be unstable and reactive, whereas a uridine-based analog showed high stability and low reactivity.  Samples of mureidomycin A and the uridine-based analog were found to be stable towards acid-catalyzed tautomerization and nucleophilic attack.  The lack of reactivity and lack of enzyme inhibition shown by the uridine-based analog are not consistent with the involvement of the enamide group in slow-binding inhibition of translocase I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0iG8FEyWJ57Vg90H21EOLACvtfcHk0lhHhD9c5b-t8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtVegtLg%253D&md5=cb9c92b222070a89fde6852d52106ca4</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1039%2Fa901279f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fa901279f%26sid%3Dliteratum%253Aachs%26aulast%3DGentle%26aufirst%3DC.%2BA.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DRole%2520of%2520the%2520enamide%2520linkage%2520of%2520nucleoside%2520antibiotic%2520mureidomycin%2520A%253A%2520synthesis%2520and%2520reactivity%2520of%2520enamide-containing%2520analogues%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25201%26date%3D1999%26spage%3D1279%26epage%3D1286%26doi%3D10.1039%2Fa901279f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gotoh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotoh, N.</span></span> <span> </span><span class="NLM_article-title">Intrinsic resistance of <i>Escherichia coli</i> to mureidomycin A and C due to expression of the multidrug efflux system AcrAB-TolC: comparison with the efflux systems of mureidomycin-susceptible <i>Pseudomonas aeruginosa</i></span>. <i>J. Infect. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">103</span>, <span class="refDoi"> DOI: 10.1007/s10156-002-0205-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1007%2Fs10156-002-0205-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=12673418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A280%3ADC%252BD3s7lslehsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2003&pages=101-103&author=N.+Gotohauthor=T.+Murataauthor=T.+Ozakiauthor=T.+Kimuraauthor=A.+Kondoauthor=T.+Nishinoauthor=N.+Gotoh&title=Intrinsic+resistance+of+Escherichia+coli+to+mureidomycin+A+and+C+due+to+expression+of+the+multidrug+efflux+system+AcrAB-TolC%3A+comparison+with+the+efflux+systems+of+mureidomycin-susceptible+Pseudomonas+aeruginosa&doi=10.1007%2Fs10156-002-0205-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Intrinsic resistance of Escherichia coli to mureidomycin A and C due to expression of the multidrug efflux system AcrAB-TolC: comparison with the efflux systems of mureidomycin-susceptible Pseudomonas aeruginosa</span></div><div class="casAuthors">Gotoh Naomasa; Murata Takeshi; Ozaki Toru; Kimura Tadashi; Kondo Akiko; Nishino Takeshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of infection and chemotherapy : official journal of the Japan Society of Chemotherapy</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">101-3</span>
        ISSN:<span class="NLM_cas:issn">1341-321X</span>.
    </div><div class="casAbstract">Intrinsic resistance to mureidomycin is shown in Escherichia coli.  This is in contrast to Pseudomonas aeruginosa, which generally displays intrinsic resistance to a variety of antimicrobial agents, but not to mureidomycin.  Isogenic efflux system mutants from both species were subjected to antibiotic susceptibility tests.  These studies showed that the differences regarding the susceptibility of E. coli and P. aeruginosa to mureidomycin A and C may be explained by the expression of efflux systems that mediate resistance to mureidomycin A and C.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTQeUH2h_3phkmhnSqMnvzDfW6udTcc2eY7G3PnbjTR8bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3s7lslehsA%253D%253D&md5=2f466331e28f4023a830b27edd717468</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2Fs10156-002-0205-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10156-002-0205-2%26sid%3Dliteratum%253Aachs%26aulast%3DGotoh%26aufirst%3DN.%26aulast%3DMurata%26aufirst%3DT.%26aulast%3DOzaki%26aufirst%3DT.%26aulast%3DKimura%26aufirst%3DT.%26aulast%3DKondo%26aufirst%3DA.%26aulast%3DNishino%26aufirst%3DT.%26aulast%3DGotoh%26aufirst%3DN.%26atitle%3DIntrinsic%2520resistance%2520of%2520Escherichia%2520coli%2520to%2520mureidomycin%2520A%2520and%2520C%2520due%2520to%2520expression%2520of%2520the%2520multidrug%2520efflux%2520system%2520AcrAB-TolC%253A%2520comparison%2520with%2520the%2520efflux%2520systems%2520of%2520mureidomycin-susceptible%2520Pseudomonas%2520aeruginosa%26jtitle%3DJ.%2520Infect.%2520Chemother.%26date%3D2003%26volume%3D9%26spage%3D101%26epage%3D103%26doi%3D10.1007%2Fs10156-002-0205-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodolis, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mihalyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ducho, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eitel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gust, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gos, R. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugg, T. D. H.</span></span> <span> </span><span class="NLM_article-title">Mechanism of action of the uridyl peptide antibiotics: an unexpected link to a protein-protein interaction site in translocase MraY</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">13023</span>– <span class="NLM_lpage">13025</span>, <span class="refDoi"> DOI: 10.1039/c4cc06516f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1039%2FC4CC06516F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=25222373" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFWjurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=13023-13025&author=M.+T.+Rodolisauthor=A.+Mihalyiauthor=C.+Duchoauthor=K.+Eitelauthor=B.+Gustauthor=R.+J.+M.+Gosauthor=T.+D.+H.+Bugg&title=Mechanism+of+action+of+the+uridyl+peptide+antibiotics%3A+an+unexpected+link+to+a+protein-protein+interaction+site+in+translocase+MraY&doi=10.1039%2Fc4cc06516f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of action of the uridyl peptide antibiotics: an unexpected link to a protein-protein interaction site in translocase MraY</span></div><div class="casAuthors">Rodolis, Maria T.; Mihalyi, Agnes; Ducho, Christian; Eitel, Kornelia; Gust, Bertolt; Goss, Rebecca J. M.; Bugg, Timothy D. H.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">86</span>),
    <span class="NLM_cas:pages">13023-13025</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The pacidamycin and muraymycin uridyl peptide antibiotics show some structural resemblance to an Arg-Trp-x-x-Trp (RWxxW) sequence motif in bacteriophage .vphi.X174 protein E that mediates a protein-protein interaction between protein E and E. coli translocase MraY.  Members of the uridyl peptide antibiotic (UPA) class, and a synthetic uridine-peptide analog, were found to show reduced levels of inhibition to F288L or E287A mutant MraY enzymes, implying that the UPAs interact at this extracellular site as part of the enzyme inhibition mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoftpmB1S2-KbVg90H21EOLACvtfcHk0ljAx1GAawFdtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFWjurzN&md5=6bf1f9e17e5c2629af8aa1321cef8265</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1039%2FC4CC06516F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4CC06516F%26sid%3Dliteratum%253Aachs%26aulast%3DRodolis%26aufirst%3DM.%2BT.%26aulast%3DMihalyi%26aufirst%3DA.%26aulast%3DDucho%26aufirst%3DC.%26aulast%3DEitel%26aufirst%3DK.%26aulast%3DGust%26aufirst%3DB.%26aulast%3DGos%26aufirst%3DR.%2BJ.%2BM.%26aulast%3DBugg%26aufirst%3DT.%2BD.%2BH.%26atitle%3DMechanism%2520of%2520action%2520of%2520the%2520uridyl%2520peptide%2520antibiotics%253A%2520an%2520unexpected%2520link%2520to%2520a%2520protein-protein%2520interaction%2520site%2520in%2520translocase%2520MraY%26jtitle%3DChem.%2520Commun.%26date%3D2014%26volume%3D50%26spage%3D13023%26epage%3D13025%26doi%3D10.1039%2Fc4cc06516f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roof, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R.</span></span> <span> </span><span class="NLM_article-title">Genetic evidence that the bacteriophage ϕX174 lysis protein inhibits cell wall synthesis</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">4297</span>– <span class="NLM_lpage">4302</span>, <span class="refDoi"> DOI: 10.1073/pnas.97.8.4297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1073%2Fpnas.97.8.4297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10760296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3cXislSnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=4297-4302&author=T.+G.+Bernhardtauthor=W.+D.+Roofauthor=R.+Young&title=Genetic+evidence+that+the+bacteriophage+%CF%95X174+lysis+protein+inhibits+cell+wall+synthesis&doi=10.1073%2Fpnas.97.8.4297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic evidence that the bacteriophage φX174 lysis protein inhibits cell wall synthesis</span></div><div class="casAuthors">Bernhardt, Thomas G.; Roof, William D.; Young, Ry</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4297-4302</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein E, a 91-residue membrane protein of φX174, causes lysis of the host in a growth-dependent manner reminiscent of cell wall antibiotics, suggesting E acts by inhibiting peptidoglycan synthesis.  A search for the cellular target of E previously isolated recessive mutations in the host gene slyD (sensitivity to lysis) that block the lytic effects of E.  The role of slyD, which encodes a FK506 binding protein-type peptidyl-prolyl cis-trans isomerase, is not fully understood.  However, E mutants referred to as Epos (plates on slyD) lack a slyD requirement, indicating that slyD is not crucial for lysis.  To identify the gene encoding the cellular target, survivors of Epos were selected.  This study describes the isolation of dominant mutations in the essential host gene mraY that result in a general lysis-defective phenotype.  Gene mraY encodes translocase I, which catalyzes the formation of the 1st lipid-linked intermediate in cell wall biosynthesis.  The isolation of these lysis-defective mutants supports a model in which translocase I is the cellular target of E and that inhibition of cell wall synthesis is the mechanism of lysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiesl_-CBo6LVg90H21EOLACvtfcHk0ljAx1GAawFdtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXislSnsrg%253D&md5=ad25239127b702f58106839b08df5db6</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1073%2Fpnas.97.8.4297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.97.8.4297%26sid%3Dliteratum%253Aachs%26aulast%3DBernhardt%26aufirst%3DT.%2BG.%26aulast%3DRoof%26aufirst%3DW.%2BD.%26aulast%3DYoung%26aufirst%3DR.%26atitle%3DGenetic%2520evidence%2520that%2520the%2520bacteriophage%2520%25CF%2595X174%2520lysis%2520protein%2520inhibits%2520cell%2520wall%2520synthesis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2000%26volume%3D97%26spage%3D4297%26epage%3D4302%26doi%3D10.1073%2Fpnas.97.8.4297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5400"><div class="reference"><strong class="refLabel"><a href="#ref5400" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit5400"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsuan, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Carbacaprazamycins: chemically stable analogues of the caprazamycin nucleoside antibiotics</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">151</span>– <span class="NLM_lpage">156</span>, <span class="refDoi"> DOI: 10.1021/id5000376</span> </span><div class="citationLinks">[<a href="/doi/10.1021/id5000376" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5400/cit5400&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=151-156&author=S.+Ichikawaauthor=M.+Yamaguchiauthor=L.+S.+Hsuanauthor=Y.+Katoauthor=A.+Matsuda&title=Carbacaprazamycins%3A+chemically+stable+analogues+of+the+caprazamycin+nucleoside+antibiotics&doi=10.1021%2Fid5000376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5400R"><div class="casContent"><span class="casTitleNuber">5400</span><div class="casTitle"><span class="NLM_cas:atitle">Carbacaprazamycins: Chemically Stable Analogues of the Caprazamycin Nucleoside Antibiotics</span></div><div class="casAuthors">Ichikawa, Satoshi; Yamaguchi, Mayumi; Hsuan, Lee Shang; Kato, Yuta; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">151-156</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Carbacaprazamycins, which are chem. stable analogs of caprazamycins, were designed and synthesized.  These analogs (I) were active against drug-resistant bacterial pathogens such as methicillin-resistant Staphylococcus aureus and vancomycin-resistant enterococci, and their activities were comparable to those of the parent caprazamycins.  The effect of treatment with carbacaprazamycin on morphol. changes in S. aureus indicated that the mode of action was completely different from those of existing peptidoglycan inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8wPaC80lQfbVg90H21EOLACvtfcHk0lhVrWk2MXcbog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVahsbg%253D&md5=9caed0fec927f4f10a0e1435660eb746</span></div><a href="/servlet/linkout?suffix=cit5400&amp;dbid=16384&amp;doi=10.1021%2Fid5000376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fid5000376%26sid%3Dliteratum%253Aachs%26aulast%3DIchikawa%26aufirst%3DS.%26aulast%3DYamaguchi%26aufirst%3DM.%26aulast%3DHsuan%26aufirst%3DL.%2BS.%26aulast%3DKato%26aufirst%3DY.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DCarbacaprazamycins%253A%2520chemically%2520stable%2520analogues%2520of%2520the%2520caprazamycin%2520nucleoside%2520antibiotics%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2015%26volume%3D1%26spage%3D151%26epage%3D156%26doi%3D10.1021%2Fid5000376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struck, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R.</span></span> <span> </span><span class="NLM_article-title">The lysis protein E of ϕX174 is a specific inhibitor of the MraY-catalyzed step in peptidoglycan synthesis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">6093</span>– <span class="NLM_lpage">6097</span>, <span class="refDoi"> DOI: 10.1074/jbc.m007638200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.1074%2Fjbc.M007638200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=11078734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhslShsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=6093-6097&author=T.+G.+Bernhardtauthor=D.+K.+Struckauthor=R.+Young&title=The+lysis+protein+E+of+%CF%95X174+is+a+specific+inhibitor+of+the+MraY-catalyzed+step+in+peptidoglycan+synthesis&doi=10.1074%2Fjbc.m007638200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The lysis protein E of φX174 is a specific inhibitor of the MraY-catalyzed step in peptidoglycan synthesis</span></div><div class="casAuthors">Bernhardt, Thomas G.; Struck, Douglas K.; Young, Ry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">6093-6097</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Coliphage φX174 encodes a single lysis protein, E, a 91-amino acid membrane protein.  Dominant mutations have been isolated in the host gene mraY that confer E resistance.  MraY encodes translocase I, which catalyzes the formation of the first lipid intermediate in bacterial cell wall synthesis, suggesting a model in which E inhibits MraY and promotes cell lysis in a manner analogous to cell wall synthesis inhibitors like penicillin.  To test this model biochem., the authors monitored the effect of E on cell wall synthesis in vivo and in vitro.  The authors find that expression of Emyc, encoding an epitope-tagged E protein, from a multicopy plasmid inhibits the incorporation of [3H]diaminopimelic acid into cell wall and leads to a profile of labeled precursors consistent with MraY inhibition.  Moreover, the authors find that membranes isolated after Emyc expression are drastically reduced in MraY activity, whereas the activity of Rfe, an enzyme in the same superfamily, was unaffected.  The authors therefore conclude that E is indeed a cell wall synthesis inhibitor and that this inhibition results from a specific block at the MraY-catalyzed step in the pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr0ez8EiSHrrVg90H21EOLACvtfcHk0lhVrWk2MXcbog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhslShsLw%253D&md5=35248473fdbc463b0550188d7c5c76c8</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M007638200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M007638200%26sid%3Dliteratum%253Aachs%26aulast%3DBernhardt%26aufirst%3DT.%2BG.%26aulast%3DStruck%26aufirst%3DD.%2BK.%26aulast%3DYoung%26aufirst%3DR.%26atitle%3DThe%2520lysis%2520protein%2520E%2520of%2520%25CF%2595X174%2520is%2520a%2520specific%2520inhibitor%2520of%2520the%2520MraY-catalyzed%2520step%2520in%2520peptidoglycan%2520synthesis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D6093%26epage%3D6097%26doi%3D10.1074%2Fjbc.m007638200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Clinical and Laboratory Standards
Institute</span>.  <i>Performance Standards for
Antimicrobial Susceptibility Testing; 27th Informational Supplement.
CLSI M100-S27</i>; <span class="NLM_publisher-name">Clinical and Laboratory Standards
Institute</span>: <span class="NLM_publisher-loc">Wayne, PA</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=Clinical+and+Laboratory+Standards%0AInstitute&title=Performance+Standards+for%0AAntimicrobial+Susceptibility+Testing%3B+27th+Informational+Supplement.%0ACLSI+M100-S27"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DPerformance%2520Standards%2520for%250AAntimicrobial%2520Susceptibility%2520Testing%253B%252027th%2520Informational%2520Supplement.%250ACLSI%2520M100-S27%26pub%3DClinical%2520and%2520Laboratory%2520Standards%250AInstitute%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanizawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminuma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arita, M.</span></span> <span> </span><span class="NLM_article-title">DFAST and DAGA: web-based integrated genome annotation tools and resources</span>. <i>Biosci. Microbiota, Food Health</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.12938/bmfh.16-003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=10.12938%2Fbmfh.16-003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=27867804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpsF2msrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=173-184&author=Y.+Tanizawaauthor=T.+Fujisawaauthor=E.+Kaminumaauthor=Y.+Nakamuraauthor=M.+Arita&title=DFAST+and+DAGA%3A+web-based+integrated+genome+annotation+tools+and+resources&doi=10.12938%2Fbmfh.16-003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">DFAST and DAGA: web-based integrated genome annotation tools and resources</span></div><div class="casAuthors">Tanizawa, Yasuhiro; Fujisawa, Takatomo; Kaminuma, Eli; Nakamura, Yasukazu; Arita, Masanori</div><div class="citationInfo"><span class="NLM_cas:title">Bioscience of Microbiota, Food and Health</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">173-184</span>CODEN:
                <span class="NLM_cas:coden">BMFHB8</span>;
        ISSN:<span class="NLM_cas:issn">2186-3342</span>.
    
            (<span class="NLM_cas:orgname">BMFH Press</span>)
        </div><div class="casAbstract">Quality assurance and correct taxonomic affiliation of data submitted to public sequence databases have been an everlasting problem.  The DDBJ Fast Annotation and Submission Tool (DFAST) is a newly developed genome annotation pipeline with quality and taxonomy assessment tools.  To enable annotation of ready-to-submit quality, we also constructed curated ref. protein databases tailored for lactic acid bacteria.  DFAST was developed so that all the procedures required for DDBJ submission could be done seamlessly online.  The online workspace would be esp. useful for users not familiar with bioinformatics skills.  In addn., we have developed a genome repository, DFAST Archive of Genome Annotation (DAGA), which currently includes 1,421 genomes covering 179 species and 18 subspecies of two genera, Lactobacillus and Pediococcus, obtained from both DDBJ/ENA/GenBank and Sequence Read Archive (SRA).  All the genomes deposited in DAGA were annotated consistently and assessed using DFAST.  To assess the taxonomic position based on genomic sequence information, we used the av. nucleotide identity (ANI), which showed high discriminative power to det. whether two given genomes belong to the same species.  We cor. mislabeled or misidentified genomes in the public database and deposited the curated information in DAGA.  The repository will improve the accessibility and reusability of genome resources for lactic acid bacteria.  By exploiting the data deposited in DAGA, we found intraspecific subgroups in Lactobacillus gasseri and Lactobacillus jensenii, whose variation between subgroups is larger than the well-accepted ANI threshold of 95% to differentiate species.  DFAST and DAGA are freely accessible at https://dfast.nig.ac.jp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGothR2Wl4cs-rVg90H21EOLACvtfcHk0lgRtbrJ5RBkOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpsF2msrc%253D&md5=bb55f302221e3bf2cf4c7a61f6e63b11</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.12938%2Fbmfh.16-003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12938%252Fbmfh.16-003%26sid%3Dliteratum%253Aachs%26aulast%3DTanizawa%26aufirst%3DY.%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DKaminuma%26aufirst%3DE.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DArita%26aufirst%3DM.%26atitle%3DDFAST%2520and%2520DAGA%253A%2520web-based%2520integrated%2520genome%2520annotation%2520tools%2520and%2520resources%26jtitle%3DBiosci.%2520Microbiota%252C%2520Food%2520Health%26date%3D2016%26volume%3D35%26spage%3D173%26epage%3D184%26doi%3D10.12938%2Fbmfh.16-003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CKR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CKR','PDB','5CKR'); return false;">PDB: 5CKR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4J72" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4J72','PDB','4J72'); return false;">PDB: 4J72</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OZ6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OZ6','PDB','6OZ6'); return false;">PDB: 6OZ6</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i100"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00973">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_57028"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00973?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00973</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Preparation of compounds, <sup>1</sup>H, <sup>13</sup>C NMR spectrum of compounds, chromatograms of LC–MS analysis, data of cell viability assay, data of MraY inhibitory assay, coordinates of MraY<sub>SA</sub> and MraY<sub>PA</sub> docked with <b>2</b>, and structures with SMILES codes (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00973/suppl_file/jm0c00973_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Biological activity of <b>2</b>, <b>19</b>, <b>22</b>, <b>23</b>, <b>24</b>, and <b>25a</b>–<b>t</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00973/suppl_file/jm0c00973_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00973/suppl_file/jm0c00973_si_001.pdf">jm0c00973_si_001.pdf (10.19 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00973/suppl_file/jm0c00973_si_003.csv">jm0c00973_si_003.csv (4.73 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00973&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-17%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00973" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00973" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799cb50791dd94a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
