
<html lang="en"     class="pb-page"  data-request-id="559da08b-8f91-49dc-8018-8333cfcc2de7"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.1c00777;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Design, Optimization, and Structural Characterization of an Apoptosis-Inducing Factor Peptide Targeting Human Cyclophilin A to Inhibit Apoptosis Inducing Factor-Mediated Cell Death" /></meta><meta name="dc.Creator" content="Luigi  Russo" /></meta><meta name="dc.Creator" content="Fabiola  Mascanzoni" /></meta><meta name="dc.Creator" content="Biancamaria  Farina" /></meta><meta name="dc.Creator" content="Amalia Mihaela  Dolga" /></meta><meta name="dc.Creator" content="Alessandra  Monti" /></meta><meta name="dc.Creator" content="Andrea  Caporale" /></meta><meta name="dc.Creator" content="Carsten  Culmsee" /></meta><meta name="dc.Creator" content="Roberto  Fattorusso" /></meta><meta name="dc.Creator" content="Menotti  Ruvo" /></meta><meta name="dc.Creator" content="Nunzianna  Doti" /></meta><meta name="dc.Description" content="Blocking the interaction between the apoptosis-inducing factor (AIF) and cyclophilin A (CypA) by the AIF fragment AIF(370–394) is protective against glutamate-induced neuronal cell death and brain ..." /></meta><meta name="Description" content="Blocking the interaction between the apoptosis-inducing factor (AIF) and cyclophilin A (CypA) by the AIF fragment AIF(370–394) is protective against glutamate-induced neuronal cell death and brain ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 2, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00777" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00777" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00777" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00777" /></link>
        
    
    

<title>Design, Optimization, and Structural Characterization of an Apoptosis-Inducing Factor Peptide Targeting Human Cyclophilin A to Inhibit Apoptosis Inducing Factor-Mediated Cell Death | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00777" /></meta><meta property="og:title" content="Design, Optimization, and Structural Characterization of an Apoptosis-Inducing Factor Peptide Targeting Human Cyclophilin A to Inhibit Apoptosis Inducing Factor-Mediated Cell Death" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0010.jpeg" /></meta><meta property="og:description" content="Blocking the interaction between the apoptosis-inducing factor (AIF) and cyclophilin A (CypA) by the AIF fragment AIF(370–394) is protective against glutamate-induced neuronal cell death and brain injury in mice. Starting from AIF(370–394), we report the generation of the disulfide-bridged and shorter variant AIF(381–389) and its structural characterization by nuclear magnetic resonance (NMR) in the free and CypA-bound state. AIF(381–389) in both the free and bound states assumes a β-hairpin conformation similar to that of the fragment in the AIF protein and shows a highly reduced conformational flexibility. This peptide displays a similar in vitro affinity for CypA, an improved antiapoptotic activity in cells and an enhanced proteolytic stability compared to the parent peptide. The NMR-based 3D model of the AIF(381–389)/CypA complex provides a better understanding of the binding hot spots on both the peptide and the protein and can be exploited to design AIF/CypA inhibitors with improved pharmacokinetic and pharmacodynamics features." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00777"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00777">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00777&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00777&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00777&amp;href=/doi/10.1021/acs.jmedchem.1c00777" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, XXXX</span><span class="cit-fg-issue">, XXX</span><span class="cit-fg-pageRange">, XXX-XXX</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/0/0" title="Return to Articles ASAP" id="returnToASAP" class="content-navigation__btn--return"><span>RETURN TO ARTICLES ASAP</span></a><a href="/doi/10.1021/acs.jmedchem.1c00663" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c01268" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design, Optimization, and Structural Characterization of an Apoptosis-Inducing Factor Peptide Targeting Human Cyclophilin A to Inhibit Apoptosis Inducing Factor-Mediated Cell Death</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Luigi Russo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Luigi Russo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Environmental, Biological and Pharmaceutical Sciences and Technologies, University of Campania “L. Vanvitelli”, Via Vivaldi 43, 81100 Caserta, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Luigi++Russo">Luigi Russo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fabiola Mascanzoni</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fabiola Mascanzoni</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biostructures and Bioimaging (IBB)-CNR, Via Mezzocannone, 16, 80134 Napoli, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fabiola++Mascanzoni">Fabiola Mascanzoni</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Biancamaria Farina</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Biancamaria Farina</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biostructures and Bioimaging (IBB)-CNR, Via Mezzocannone, 16, 80134 Napoli, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Biancamaria++Farina">Biancamaria Farina</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Amalia Mihaela Dolga</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Amalia Mihaela Dolga</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmacology and Clinical Pharmacy, University of Marburg, 35043 Marburg, Germany</div><div class="loa-info-affiliations-info">Faculty of Science and Engineering, Groningen Research Institute of Pharmacy (GRIP), Research School of Behavioural and Cognitive Neurosciences (BCN), Department of Molecular Pharmacology, University of Groningen, 9713 AV Groningen, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Amalia+Mihaela++Dolga">Amalia Mihaela Dolga</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5400-5614" title="Orcid link">https://orcid.org/0000-0001-5400-5614</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alessandra Monti</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alessandra Monti</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biostructures and Bioimaging (IBB)-CNR, Via Mezzocannone, 16, 80134 Napoli, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alessandra++Monti">Alessandra Monti</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrea Caporale</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrea Caporale</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biostructures and Bioimaging (IBB)-CNR, Via Mezzocannone, 16, 80134 Napoli, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Caporale">Andrea Caporale</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carsten Culmsee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carsten Culmsee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmacology and Clinical Pharmacy, University of Marburg, 35043 Marburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carsten++Culmsee">Carsten Culmsee</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Roberto Fattorusso</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Roberto Fattorusso</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Environmental, Biological and Pharmaceutical Sciences and Technologies, University of Campania “L. Vanvitelli”, Via Vivaldi 43, 81100 Caserta, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Roberto++Fattorusso">Roberto Fattorusso</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-3539-5343" title="Orcid link">https://orcid.org/0000-0002-3539-5343</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Menotti Ruvo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Menotti Ruvo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biostructures and Bioimaging (IBB)-CNR, Via Mezzocannone, 16, 80134 Napoli, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Menotti++Ruvo">Menotti Ruvo</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5997-756X" title="Orcid link">https://orcid.org/0000-0001-5997-756X</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Nunzianna Doti</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nunzianna Doti</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biostructures and Bioimaging (IBB)-CNR, Via Mezzocannone, 16, 80134 Napoli, Italy</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#91ffe4ffebf8f0fffff0bff5fee5f8d1f2ffe3bff8e5"><span class="__cf_email__" data-cfemail="2e405b4054474f40404f004a415a476e4d405c00475a">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nunzianna++Doti">Nunzianna Doti</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2952-6658" title="Orcid link">https://orcid.org/0000-0003-2952-6658</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00777&amp;href=/doi/10.1021%2Facs.jmedchem.1c00777" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, XXXX</span><span class="cit-issue">, XXX</span><span class="cit-pageRange">, XXX-XXX</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 2, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>29 April 2021</li><li><span class="item_label"><b>Published</b> online</span>2 August 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00777" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00777</a></div><div class="article_header-article-copyright"><strong>© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DLuigi%2BRusso%252C%2BFabiola%2BMascanzoni%252C%2BBiancamaria%2BFarina%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D0%26issueNum%3D0%26contentID%3Dacs.jmedchem.1c00777%26title%3DDesign%252C%2BOptimization%252C%2Band%2BStructural%2BCharacterization%2Bof%2Ban%2BApoptosis-Inducing%2BFactor%2BPeptide%2BTargeting%2BHuman%2BCyclophilin%2BA%2Bto%2BInhibit%2BApoptosis%2BInducing%2BFactor-Mediated%2BCell%2BDeath%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D15%26publicationDate%3DAugust%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00777"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">-</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00777" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design, Optimization, and Structural Characterization of an Apoptosis-Inducing Factor Peptide Targeting Human Cyclophilin A to Inhibit Apoptosis Inducing Factor-Mediated Cell Death&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Luigi&quot;,&quot;last_name&quot;:&quot;Russo&quot;},{&quot;first_name&quot;:&quot;Fabiola&quot;,&quot;last_name&quot;:&quot;Mascanzoni&quot;},{&quot;first_name&quot;:&quot;Biancamaria&quot;,&quot;last_name&quot;:&quot;Farina&quot;},{&quot;first_name&quot;:&quot;Amalia&quot;,&quot;last_name&quot;:&quot;Mihaela Dolga&quot;},{&quot;first_name&quot;:&quot;Alessandra&quot;,&quot;last_name&quot;:&quot;Monti&quot;},{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Caporale&quot;},{&quot;first_name&quot;:&quot;Carsten&quot;,&quot;last_name&quot;:&quot;Culmsee&quot;},{&quot;first_name&quot;:&quot;Roberto&quot;,&quot;last_name&quot;:&quot;Fattorusso&quot;},{&quot;first_name&quot;:&quot;Menotti&quot;,&quot;last_name&quot;:&quot;Ruvo&quot;},{&quot;first_name&quot;:&quot;Nunzianna&quot;,&quot;last_name&quot;:&quot;Doti&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;02&quot;,&quot;issue&quot;:&quot;0&quot;,&quot;volume&quot;:&quot;0&quot;,&quot;pages&quot;:&quot;&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00777&quot;},&quot;abstract&quot;:&quot;Blocking the interaction between the apoptosis-inducing factor (AIF) and cyclophilin A (CypA) by the AIF fragment AIF(370–394) is protective against glutamate-induced neuronal cell death and brain injury in mice. Starting from AIF(370–394), we report the generation of the disulfide-bridged and shorter variant AIF(381–389) and its structural characterization by nuclear magnetic resonance (NMR) in the free and CypA-bound state. AIF(381–389) in both the free and bound states assumes a β-hairpin conformation similar to that of the fragment in the AIF protein and shows a highly reduced conformational flexibility. This peptide displays a similar in vitro affinity for CypA, an improved antiapoptotic activity in cells and an enhanced proteolytic stability compared to the parent peptide. The NMR-based 3D model of the AIF(381–389)/CypA complex provides a better understanding of the binding hot spots on both the peptide and the protein and can be exploited to design AIF/CypA inhibitors with improved pharmacokinetic and &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00777&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00777" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00777&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00777" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00777&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00777" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00777&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00777&amp;href=/doi/10.1021/acs.jmedchem.1c00777" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00777" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00777" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00777%26sid%3Dliteratum%253Aachs%26pmid%3D34338510%26genre%3Darticle%26aulast%3DRusso%26date%3D2021%26atitle%3DDesign%252C%2BOptimization%252C%2Band%2BStructural%2BCharacterization%2Bof%2Ban%2BApoptosis-Inducing%2BFactor%2BPeptide%2BTargeting%2BHuman%2BCyclophilin%2BA%2Bto%2BInhibit%2BApoptosis%2BInducing%2BFactor-Mediated%2BCell%2BDeath%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D0%26issue%3D0%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/0/0" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Blocking the interaction between the apoptosis-inducing factor (AIF) and cyclophilin A (CypA) by the AIF fragment AIF(370–394) is protective against glutamate-induced neuronal cell death and brain injury in mice. Starting from AIF(370–394), we report the generation of the disulfide-bridged and shorter variant AIF(381–389) and its structural characterization by nuclear magnetic resonance (NMR) in the free and CypA-bound state. AIF(381–389) in both the free and bound states assumes a β-hairpin conformation similar to that of the fragment in the AIF protein and shows a highly reduced conformational flexibility. This peptide displays a similar <i>in vitro</i> affinity for CypA, an improved antiapoptotic activity in cells and an enhanced proteolytic stability compared to the parent peptide. The NMR-based 3D model of the AIF(381–389)/CypA complex provides a better understanding of the binding hot spots on both the peptide and the protein and can be exploited to design AIF/CypA inhibitors with improved pharmacokinetic and pharmacodynamics features.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20130" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20130" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The interaction of the apoptosis-inducing factor (AIF) with cyclophilin A (CypA) plays a key role <i>in vitro</i> and <i>in vivo</i> in pathways of programed cell death.</div><div class="NLM_p">The AIF is a flavoprotein with NAD(P)H oxidase and monodehydroascorbate reductase activities localized in the inner mitochondrial membrane.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Genetic downregulation of the AIF, as in Hq mice, causes progressive loss of terminally differentiated cerebellar and retinal neurons, resulting in ataxia and blindness, respectively. Such progressive neurodegeneration is associated with complex I deficiencies.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3−5)</a> These findings indicate that the AIF has a vital role in mitochondrial bioenergetics and cellular redox metabolism.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">However, many reports also provide evidence of a lethal function of the AIF.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Under pathological conditions, the AIF is released from the mitochondria in its proapoptotic form [AIF(Δ1-121)] and translocates into the nuclei, where it induces the chromatin condensation and large-scale DNA fragmentation, ultimately leading to caspase-independent cell death.<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8−10)</a> Selective targeting of AIF mediates protection against excitotoxic cell death, oxidative stress, and ischemic injury of the nervous system.<a onclick="showRef(event, 'ref11 ref12 ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14 ref15">(11−15)</a> As an example, the overexpression of Hsp70 family members prevents AIF nuclear translocation and provides cytoprotection in primary cortical neurons and in SH-SY5Y neuronal cells.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Also, the microinjection of neutralizing antibodies against AIF blocks <i>N</i>-methyl-<span class="smallcaps smallerCapital">d</span>-aspartate/poly(ADP-ribose)polymerase-1 (PARP-1)-dependent excitotoxicity in neurons and PARP-1-dependent cytotoxicity in fibroblasts.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Moreover, siRNA-mediated AIF gene silencing or its genetic downregulation in Hq mice provides neuroprotective effects <i>in vitro</i> and against brain damage after focal cerebral hypoxia/ischemia damage <i>in vivo</i>.<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(17−19)</a></div><div class="NLM_p">Overall, these findings demonstrated the importance of having tools for the selective regulation of the lethal function of the AIF without altering its vital activity. In this regard, several <i>in vitro</i> and <i>in vivo</i> models of programmed cell death have shown that AIF(Δ1-121) cooperates with the peptidyl prolyl <i>cis</i>–<i>trans</i> isomerase CypA to mediate cell death.<a onclick="showRef(event, 'ref5 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref5 ref20 ref21 ref22">(5,20−22)</a> Disruption of the CypA homologue CPR1 in yeast cells inhibits cell death induced by overexpression of AIF.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Jurkat cells lacking CypA expression are relatively resistant to AIF-induced cell death and this effect is reversed by reintroduction of the CypA gene.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Downregulation of CypA reduces the sensitivity of AIF-induced cell death during glutamate-mediated oxidative stress.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> CypA knockout (CypA–/−) confers significant neuroprotection in neurons following hypoxia–ischemia in the neonatal rat brain.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Noteworthily, the detrimental cooperation between the AIF and CypA does not rely on the CypA peptidyl-prolyl <i>cis</i>–<i>trans</i> isomerase activity,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and unlike Hq mutant mice, CypA–/– mice are phenotypically normal, showing that the selective targeting of CypA has a high potential for limiting AIF-mediated damage. The amino acid region of AIF(Δ1-121) mainly involved in the interaction with CypA has been identified.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> A peptide mimetic of this region (henceforth called AIF(370–394)) binds CypA in the low micromolar range (KD = 5 μM) and is able to compete with the AIF(Δ1-121) protein for the binding to CypA. The transfection of AIF(370–394) in HT-22 neuronal cells blocks the AIF/CypA complex formation and, like CypA downregulation, significantly reduces the oxidative stress and ameliorates the survival of cells treated with high doses of glutamate.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Moreover, the prior intranasal delivery of AIF(370–394) to pups subjected to hypoxia–ischemia damage significantly reduces brain injury for both the gray matter and the subcortical white matter within 72 h after the damage.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Notably, a very recent seminal work has shown that AIF(370–394) can efficiently reduce myocardial cell death and has the potential to prevent sudden cardiac death in patients with arrhythmogenic cardiomyopathy, a disease that provokes approximately 400,000–460,000 death every year only in the United States.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> This finding greatly expands the importance of AIF as a therapeutic target and urges the development of inhibitors for medical intervention.</div><div class="NLM_p">The structural characterization of the AIF(Δ1-121)/CypA and AIF(370–394)/CypA complexes has demonstrated that the region on CypA interacting with AIF(370–394) is included in that used to bind the AIF, in line with the ability of the peptide to inhibit AIF(Δ1-121)/CypA complex formation.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> Moreover, the interacting surface of AIF(370–394) is located on the rim of the catalytic site of CypA, which agrees with the inability of the peptide, as well as of the AIF protein, to inhibit the catalytic activity of the enzyme.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> Taken together, the data show that the AIF(370–394) represents a simplified but effective ligand model to study the AIF/CypA interaction and to design selective inhibitors of the lethal complex. Nuclear magnetic resonance (NMR) studies of the AIF(370–394) peptide, which adopts a β-hairpin conformation in the AIF protein,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> have shown that its interaction with CypA induces a more organized structure, suggesting that the stabilization of the secondary structure elements could increase the affinity of the peptide for CypA. To further expand this working hypothesis, we designed and tested a set of constrained AIF(370–394) analogues identifying a stapled bicyclic peptide with an affinity for CypA in the nanomolar range.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The structural data coupled with molecular modeling analyses indicated that this improved affinity derived from a balance between an increased conformational stability and a partially retained flexibility that likely favors an induced-fit mechanism of recognition.</div><div class="NLM_p">Here, we report the generation of a disulfide-bridged and shorter variant of AIF(370–394) and its structural characterization by NMR in the free and CypA-bound state.</div><div class="NLM_p last">In particular, we have designed a 11-mer stapled peptide [hereafter, AIF(381–389)], which displays a similar <i>in vitro</i> affinity for CypA, an improved antiapoptotic activity in cells, and an enhanced proteolytic stability compared to the parent peptide. A 3D molecular model of the AIF(381–389)/CypA complex has been built using NMR data, providing an atomistic view of the binding hot spots on both the peptide and the protein. These data can be exploited to design with high confidence new inhibitors of the CypA/AIF complex with improved pharmacokinetic and pharmacodynamic features and to develop drug candidates for therapeutic purposes.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15670" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15670" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design, Synthesis, and Characterization of a Size-Reduced Version of AIF(370–394)</h3><div class="NLM_p">Starting from previous structural and biochemical information on AIF(370–394) and its stapled analogues,<a onclick="showRef(event, 'ref26 ref27 ref29'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref29">(26,27,29)</a> we tried to improve the peptide structural features by reducing the size and addressing its potency and stability. In particular, we removed the amino- and carboxy-terminal portions, which turned out to be less involved in the binding with CypA.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> We thereby focused our attention on the AIF residues encompassing the region from Leu<sup>380</sup> to Glu<sup>390</sup> (single letter sequence: LIKLKDGRKVE; Table S1, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>) and started to investigate the corresponding synthetic peptide. Since this region assumes in the native AIF protein a β-hairpin conformation, two cysteine residues (Cys) were added at the amino- and carboxy-termini of the peptide to potentially stabilize the β-hairpin structure through the formation of a disulfide bridge.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The linear (reduced) and oxidized peptides (hereafter called AIF(380–390)red and AIF(380–390)ox, respectively) were evaluated for their ability to bind CypA using a label-free Corning Epic technology, and the results are reported as KD values (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). As shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A,B, both peptides bind CypA in a dose-dependent and saturable manner. The fitting of the binding curves with a nonlinear regression algorithm showed a 1-log improvement in potency of the cyclic peptide compared to the linear variant (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). To evaluate the role of the individual side chains of AIF(380–390)ox in the binding, Ala-scan variants were prepared (Table S1, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>) and tested at two concentrations (10 and 20 μM) against CypA (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). Binding experiments showed that peptides with alanine in place of K<sup>382</sup>, K<sup>384</sup>, R<sup>387</sup>, and K<sup>388</sup> had lower affinity for CypA compared to the parent peptide (AIF(380–390)ox). On the contrary, peptides with alanine replacing L<sup>380</sup>, I<sup>381</sup>, L<sup>383</sup>, and V<sup>389</sup> showed substantially unaltered binding affinity compared to AIF(380–390)ox. Remarkably, peptides where D<sup>385</sup>, G<sup>386</sup>, and E<sup>390</sup> were replaced by alanine had higher affinity for CypA than the parent peptide. These findings suggested that AIF(380–390)ox binds CypA with an affinity similar to that of the parent peptide AIF(370–394) and that the basic residues K<sup>382</sup>, K<sup>384</sup>, R<sup>387</sup>, and K<sup>388</sup> are mainly involved in the interaction with the target, in line with previous results.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding curves of AIF(380–390)red (A) and AIF(380–390)ox (B) to CypA. (C) Binding assay of the Ala-scan peptides at two different concentrations (10 and 20 μM) to CypA. (D) Binding curve of the cyclic peptide AIF(381–389) to CypA. Experiments were carried out using the label-free Corning Epic technology.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. KD Values of AIF Peptides Underlying the Binding with CypA<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">peptide</th><th class="colsep0 rowsep0" align="center" char="±">KD (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AIF(380–390)red</td><td class="colsep0 rowsep0" align="char" char="±">30.0 ± 2.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AIF(380–390)ox</td><td class="colsep0 rowsep0" align="char" char="±">3.2 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AIF(381–389)</td><td class="colsep0 rowsep0" align="char" char="±">2.4 ± 0.2</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">KD values were obtained by concentration-dependent binding experiments for each peptide.</p></div></div></div><div class="NLM_p last">Since L<sup>380</sup> only negligibly affected the binding to CypA and the replacement of E<sup>390</sup> with alanine increased the affinity, we replaced them with cysteines (Cys<sup>N-ter</sup> and Cys<sup>C-ter</sup>), thus keeping residues 381–389 connected by a disulfide bridge that could potentially stabilize a β-hairpin structure (Table S1, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>). The new cyclic peptide [henceforth called AIF(381–389)] was tested in binding experiments against CypA, as described above. As shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the peptide AIF(381–389)-bound CypA with a KD in the low micromolar range, demonstrating that, in agreement with the alanine scanning experiments, the deletion of the two residues L<sup>380</sup> and E<sup>390</sup> does not affect the affinity for CypA of the resulting peptide.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Resonance Assignment and Chemical Shift Analysis of the AIF(381–389) Peptide</h3><div class="NLM_p">In order to evaluate the conformational determinants governing the protein–peptide recognition mechanism between the cyclic peptide AIF(381–389) and the protein CypA, we first investigated the peptide in the free form by natural-abundance NMR spectroscopy.</div><div class="NLM_p">A nearly complete <sup>1</sup>H, <sup>15</sup>N, and <sup>13</sup>C chemical shift assignment at 298 K (Tables S2 and S3, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>) was obtained using a previously published strategy,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> in which, by exploring the natural isotopic abundance, heteronuclear 2D [<sup>1</sup>H–<sup>15</sup>N] HSQC and 2D [<sup>1</sup>H–<sup>13</sup>C] HSQC were analyzed using the <sup>1</sup>H resonances assignment obtained by inspection of homonuclear 2D [<sup>1</sup>H–<sup>1</sup>H] TOCSY and 2D [<sup>1</sup>H–<sup>1</sup>H] ROESY spectra (Figure S1, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>).</div><div class="NLM_p">First, the <sup>1</sup>H–<sup>15</sup>N HSQC spectrum showed a good dispersion of the backbone N–H<sub>N</sub> cross-peaks in both proton and nitrogen dimension, suggesting that AIF(381–389) has a preference for structured conformations.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> In addition, as illustrated in the <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A–C, the experimental Hα, Cα, and Cβ chemical shifts significantly deviate from random coil values predicted by using several methodologies,<a onclick="showRef(event, 'ref34 ref35 ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref34 ref35 ref36 ref37 ref38">(34−38)</a> indicating that the peptide adopts an ordered secondary structure. In particular, the positive deviations of Hα and Cβ, as well as the negative Cα secondary chemical shifts (δ<sub>obs</sub>–δ<sub>coil</sub>) suggest that residues included in the Ile<sup>381</sup>–Leu<sup>383</sup> and Lys<sup>388</sup>–Cys<sup>C-ter</sup> regions have a significant β-strand propensity. In line with this assumption, the differences between Cα and Cβ secondary chemical shifts (ΔδCα–ΔδCβ)<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> for the residues located at the N- and C-terminal regions are lower than 2 ppm (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D). Noteworthily, the profiles of the secondary chemical shifts obtained for Hα protons and for Cα and Cβ carbons fit well with those expected for a β-hairpin structure: two stretches of positive ΔδHα and ΔδCβ and negative ΔδCα corresponding to β-strands are connected by some residues having negative ΔδHα and ΔδCβ and positive ΔδCα. Among this central amino acid region, Lys<sup>384</sup> exhibits a negative ΔδCβ and highly positive ΔδCα values, the well-known feature for the i + 1 residue of type I′ β-turns.<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> The presence of a β-hairpin conformation was also indicated by medium- and long-range ROE connectivities detected in the 2D [<sup>1</sup>H–<sup>1</sup>H] ROESY spectrum. In particular, H<sub>N</sub> Asp<sup>385</sup> shows a strong ROE correlation with the H<sub>N</sub> of the previous (Lys<sup>384</sup>) and the successive (Gly<sup>386</sup>) residue. The amide proton of the Gly<sup>386</sup> displays a ROE cross-peak with Lys<sup>384</sup> Hα, whereas Ile<sup>381</sup>, Lys<sup>382</sup>, and Leu<sup>383</sup> show strong long-range ROE connectivities with Arg<sup>387</sup>, Lys<sup>388</sup>, and Val<sup>389</sup> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E). Similar analyses performed on the linear variant of the peptide showed that it does not show any preferential conformation (Figure S2, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>), suggesting that the β-hairpin conformation is stabilized by the presence of the disulfide bridge (Table S4, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. NMR conformational analysis of AIF(381–389) in the free form. Secondary chemical shifts (Δδ) (δ<sub>obs</sub> – δ<sub>coil</sub>) of (A) Hα, (B) Cα, (C) Cβ, and (D) ΔδCα – ΔδCβ differences of the peptide AIF(381–389). The chemical analysis was performed using the random coil shifts calculated as reported in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. Dashed lines indicate the cutoff values for the identification of secondary structure elements defined by Wishart and Sykes<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> (panels A–C) and by Marsh et al.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> (panel D). (E) Portions of the 2D [<sup>1</sup>H–<sup>1</sup>H] ROESY spectrum of AIF(381–389). Some of the inter-residue ROEs observed for the highlighted residues (black) are labeled in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Structural Characterization of AIF(381–389)</h3><div class="NLM_p">To get deeper insights into the β-hairpin formed in solution by AIF(381–389), we determined the three-dimensional structure of the peptide by using NMR spectroscopy (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A–E). The structure calculation was performed using ROE-driven distances (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), hydrogen bonds, and backbone dihedral angles (ϕ,ψ) as conformational restraints (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). Additionally, secondary structure elements of AIF(381–389) were evaluated by vicinal <sup>3</sup>JH<sub>N</sub>Hα scalar couplings analysis. According to the chemical shift index (CSI) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A) and amide proton temperature coefficient (ΔδH<sub>N</sub>/Δ<i>T</i>), the <sup>3</sup>JH<sub>N</sub>Hα coupling values are >8 Hz for the residues located in two terminal regions (Cys<sup>N-ter</sup>-Leu<sup>383</sup> and Lys<sup>388</sup>-Cys<sup>C-ter</sup>) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C and Table S5, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>), confirming the presence of β-strand conformations. As reported in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D, the superposition of the 20 lowest energy structures of the AIF(381–389) indicates that the backbone of the resulting NMR conformational ensemble is well defined, especially at the turn region (Leu<sup>383</sup>–Gly<sup>386</sup>). In particular, considering all residues from Cys<sup>N-ter</sup> to Cys<sup>C-ter</sup> the pair-wise root-mean-square deviation (rmsd) value for the backbone atoms of the 20 selected structures is 0.037 Å, suggesting that most of the residues are characterized by a reduced conformational flexibility (Table S6, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>). On the contrary, side chains are less ordered (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E), as reflected by the increase in the pair-wise rmsd value (0.97 Å) (Table S6, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>) when all heavy atoms are considered.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure and dynamics of the AIF(381–389) peptide. (A) (Upper) Secondary structure elements of AIF(381–389) in dependence of the peptide sequence were derived by the CSI based on Cα and Hα resonance assignments. (Lower) Schematic diagram showing the magnitude of ROE connectivities observed in the 2D [<sup>1</sup>H–<sup>1</sup>H] ROESY spectrum. (B) Plot of the distribution of distance restraints as a function of the residue number. The distances are classified according to their range (<i>R</i>): white intraresidual (<i>R</i> = 0), light-gray sequential (<i>R</i> = 1), dark-gray medium-range (<i>R</i> < 5), and black long-range (<i>R</i> ≥ 5). (C) <sup>3</sup>JH<sub>N</sub>Hα scalar coupling (upper) and amide temperature coefficient (lower) analysis. The open bars indicate the <sup>3</sup>JH<sub>N</sub>Hα reference values in the random coil conformation.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> (D,E) Overlay of the 20 lowest energy NMR structures. In panel E, the backbone and side chains are illustrated as gold neon and blue sticks, respectively. The disulfide-bond between Cys<sup>N-ter</sup>–Cys<sup>C-ter</sup> is reported as a light-green stick. (F) Mapping of the residues for which the <sup>1</sup>H–<sup>15</sup>N cross peaks are detectable in the T2-filter <sup>1</sup>H–<sup>15</sup>N HSQC experiment acquired using the filter delay of 500 ms; the H<sub>N</sub>/N intensities are classified as strong (dark blue) or weak (light blue). In red are reported the residues for which <sup>1</sup>H–<sup>15</sup>N signals are not observable. (G) Model-free order parameter (<i>S</i><sup>2</sup>) mapped onto the representative NMR structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Notably, the overall stability of the β-hairpin structure is supported by (i) the disulfide bridge between the Cys<sup>N-ter</sup> and Cys<sup>C-ter</sup>, (ii) the Ile<sup>381</sup>–Val<sup>389</sup> and Leu<sup>383</sup>–Val<sup>389</sup> hydrophobic interactions, and by (iii) an extensive network of hydrogen bonds involving the backbone atoms. In more detail, the carboxyl oxygen of the Ile<sup>381</sup> interacts through a hydrogen bond with the amide proton of the Val<sup>389</sup>. The backbone amide of the Leu<sup>383</sup> is involved in the hydrogen bond with the carboxyl oxygen of the Arg<sup>387</sup>, whereas the carboxyl oxygen of the Leu<sup>383</sup> interacts through a hydrogen bond with the amide proton of Asp<sup>385</sup> and Gly<sup>386</sup>, respectively. The carboxyl oxygen of the Lys<sup>384</sup> is hydrogen-bonded with the H<sub>N</sub> of the Gly<sup>386</sup>. In addition, the carboxyl oxygens of the Asp<sup>385</sup> and Gly<sup>386</sup> are involved in hydrogen bonds with the H<sub>N</sub> of the Arg<sup>387</sup> and Lys<sup>388</sup>, respectively.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> AIF(381–389) Backbone Dynamics</h3><div class="NLM_p last">Further insights on the structural properties of AIF(381–389) were provided by the investigation of the <sup>15</sup>N backbone motions of the peptide in the microsecond to millisecond (μs–ms) and in the picosecond to nanosecond (ps–ns) timescales by natural abundance NMR spectroscopy. For the evaluation of peptide dynamics on different timescales, a combined approach was used, in which the dynamical information contained in the backbone chemical shifts was integrated with the data obtained by measuring a couple of <sup>1</sup>H–<sup>15</sup>N HSQC spectra filtered with different relaxation-compensated CPMG (Carr–Purcell–Meiboom–Gill) periods (125 and 500 ms)<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> (Figure S3A,B, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>). In fact, in this HSQC-based experiment, also called T2-filter <sup>1</sup>H–<sup>15</sup>N HSQC, only the <sup>1</sup>H–<sup>15</sup>N cross-peaks of the residues with a <sup>15</sup>N transversal relaxation time (T2) longer than the applied filter delay are detectable. As reported in Figure S3A (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>), the T2-filter <sup>1</sup>H–<sup>15</sup>N HSQC spectrum of AIF(381–389), acquired using a filter delay of 125 ms (R2 = 1/T2 = 8 Hz), contains the 11 expected <sup>1</sup>H–<sup>15</sup>N cross-peaks with a comparable intensity, indicating that all residues are characterized by a <sup>15</sup>N R2 self-relaxation rate lower than 8 Hz. On the contrary, the T2-filter <sup>1</sup>H–<sup>15</sup>N HSQC experiments measured, using a filter delay of 500 ms (R2 = 2 Hz; Figure S3B, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>), do not contain the <sup>1</sup>H–<sup>15</sup>N cross-peaks (<sup>15</sup>N R2 faster than 2 Hz) of Ile<sup>381</sup>, K<sup>382</sup>, K<sup>388</sup> Val<sup>389</sup>, Cys<sup>C-ter</sup>, and Asp<sup>385</sup>, whereas the NMR signals of Cys<sup>N-ter</sup>, L<sup>383</sup>, K<sup>384</sup>, G<sup>386</sup>, and R<sup>387</sup> show a significant intensity reduction (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). These results thus demonstrate that the regions encompassing the two β-strands (β1 = Ile<sup>381</sup>–K<sup>382</sup>; β2 = Lys<sup>388</sup>–Val<sup>389</sup>), together with the Asp<sup>385</sup> placed in the middle of the β-turn, are slightly more rigid than the rest of the peptide in the μs–ms timescale. To explore the conformational dynamics of the peptide in the ps–ns timescale, the model-free order parameter <i>S</i><sup>2</sup> was estimated. The <i>S</i><sup>2</sup> value represents a measure of the partial restriction of the H–N bond in the ps–ns timescale in a molecule-fixed frame and assumes values between 0 (large motion) and 1 (no motion). The order parameter <i>S</i><sup>2</sup> values obtained in the range between 0.51 and 0.72 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>G) indicate that AIF(381–389) adopts a β-hairpin structure in solution characterized by a reduced conformational flexibility also in the ps–ns time scale.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Mapping of the Binding Site of CypA on AIF(381–389)</h3><div class="NLM_p">The AIF(381–389)–CypA interaction was first described by investigating the chemical shift perturbation (CSP) and intensity ratio of the peptide resonances upon binding to CypA. In particular, <sup>1</sup>H–<sup>15</sup>N HSQC and <sup>1</sup>H–<sup>13</sup>C HSQC spectra of unlabeled AIF(381–389) in the presence of CypA (peptide/protein 150:1 M ratio) were acquired. Notably, a subset of AIF(381–389) resonances in the <sup>1</sup>H–<sup>15</sup>N HSQC spectrum (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A), as well as in the <sup>1</sup>H–<sup>13</sup>C HSQC experiment (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B), showed significant changes upon addition of CypA, indicating that several AIF(381–389) residues were involved in the binding mechanism. The <sup>1</sup>H, <sup>15</sup>N, and <sup>13</sup>C shift variations were evaluated by dividing the chemical shifts into three groups: backbone (Δ<sub>H,N,C</sub><sup>backbone</sup>), side chains (Δ<sub>H,N,C</sub><sup>side-chain</sup>), and all together (Δ<sub>H,N,C</sub><sup>all-together</sup>) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). Significant backbone and side-chain CSPs, as well as the Δ<sub>H,N,C</sub><sup>all-together</sup> values, were observed for Lys<sup>382</sup>, Asp<sup>385</sup>, Gly<sup>386</sup>, Arg<sup>387</sup>, and Lys<sup>388</sup>. Additionally, the comparison of the H<sub>N</sub>/N and Hα/Cα cross-peak intensities of AIF(381–389) in the absence and presence of CypA showed a significant signal reduction for Lys<sup>382</sup>, Asp<sup>385</sup>, and Lys<sup>388</sup> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,B), confirming that these residues play a major role in the recognition mechanism.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. NMR analysis of AIF(381–389) binding to CypA. (A,B Upper) H<sub>N</sub>/N (A) and Hα/Cα (B) signal intensity ratios of AIF(381–389) upon addition of CypA (I<sub><i>b</i>ound</sub>) and in the free form (<i>I</i><sub>free</sub>) as a function of the amino acid sequence. The orange line indicates the average <i>I</i><sub>bound</sub>/<i>I</i><sub>free</sub> value. (A,B Lower) Overlay of the <sup>1</sup>H–<sup>15</sup>N HSQC (A) and <sup>1</sup>H–<sup>13</sup>C HSQC (B) spectra of unlabeled AIF(381–389) in the absence (blue) or in the presence (red) of CypA. (C) Mapping of weighted combined <sup>1</sup>H, <sup>15</sup>N, and <sup>13</sup>C CSPs onto the representative conformer of the AIF(381–389) NMR ensemble obtained for backbone (left), side chains (middle), and all-together atoms (right), respectively. The residues of AIF(381–389) showing significant CSP are labeled using the color code reported on the right side, which indicates the chemical shift variation observed for the peptide upon addition of CypA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Taken together, the CSP data and the intensity profile analysis indicate that the recognition of CypA by AIF(381–389) is principally mediated by Lys<sup>382</sup> and Lys<sup>388</sup>, located in β1 and β2 strands, respectively, and by Arg<sup>387</sup> and Asp<sup>385</sup> situated in the β-turn. These results well agree with the Ala-scanning experiments reported above and with previous results obtained on AIF(370–394) in complex with CypA.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Indeed, it was shown that the strongest interactions are established by the peptide-charged groups on lysine and aspartate side chains, explaining the preserved CypA binding affinity of the new peptide compared to the parent peptide.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Mapping of the Interaction Site of AIF(381–389) on CypA</h3><div class="NLM_p">To identify the binding site of AIF(381–389) on CypA, we acquired the <sup>1</sup>H–<sup>15</sup>N HSQC spectrum of <sup>15</sup>N-labeled CypA in the presence and absence of the peptide (molar ratio protein/peptide 10:1) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A,B). Largest CSPs on CypA were observed for Tyr48 and Cys52–Arg55 (β3), Gly59 (type II turn between β3 and β4); Thr68, Arg69, Gly74, Gly<sup>75</sup>, Lys<sup>76</sup>, and Tyr<sup>79</sup> (turn between β4 and β5); Ser<sup>99</sup>–Ala<sup>103</sup> (β5); Gln<sup>111</sup> and Thr<sup>116</sup> (β6); Thr<sup>119</sup> (the 3<sub>10</sub>-helix); and Val<sup>127</sup>, Val<sup>128</sup> (β7), and Ser<sup>147</sup> (type I turn) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C,D and S4, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>). We also analyzed the NMR signal intensity variations of CypA upon addition of AIF(381–389) and the relative ASA (accessible surface area) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E). These data were combined with CSPs in order to identify the CypA residues directly involved in the binding to the peptide and discarding the residues for which the chemical shift variations are related to indirect binding effects (i.e., allosteric change). As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F and S4 in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>, the recognition mechanism of AIF by CypA is mainly mediated by Thr68, Arg<sup>69</sup>, Gly<sup>72</sup>, Gly<sup>75</sup>, Asn<sup>102</sup>, and Ala<sup>103</sup>. These residues, located in a shallow hydrophobic pocket on the CypA surface, flanking the catalytic site, were used as ambiguous interaction restraints (AIRs) in the molecular docking studies reported in the next paragraph.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Binding of CypA to the AIF(381–389) peptide by NMR. (A,B) Superposition of the <sup>1</sup>H–<sup>15</sup>N HSQC spectra of <sup>15</sup>N-CypA alone (blue) and after the addition of AIF(381–389) (molar ratio 10:1). The insets show portions of the overlapped <sup>1</sup>H–<sup>15</sup>N HSQC spectra. (C) <sup>1</sup>H/<sup>15</sup>N CSPs plotted against the CypA sequence. The yellow line indicates the average CSP (CSPavg), whereas the orange line reports the CSPavg + SD (standard deviation) value. (D) Mapping of the residues showing CSP > CSPavg (yellow) and ≥ CSPavg + SD (orange) onto the CypA X-ray structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CPL">2CPL</a>) reported as ribbon drawing (left) and surface (right) representation. The dotted line circle indicates the catalytic site of CypA. (E) Ratios between NMR signal intensity observed for CypA in the presence (<i>I</i><sub>bound</sub>) and absence (<i>I</i><sub>free</sub>) of AIF(381–389) (values normalized to Asn<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a>) and ASA plotted <i>vs</i> the residue number. (F) CypA residues used as active residues in the docking studies. (G) CypA electrostatic surface potential depicted from electropositive (blue; 10 kcal/mole) to electronegative (red; −10 kcal/mole) in two orientations rotated 180° around the <i>z</i>-axis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Structure of the AIF(381–389)/CypA Complex</h3><div class="NLM_p">To provide an accurate description of the molecular determinants driving the AIF(381–389) binding to CypA, a structural model of the AIF(381–389)/CypA was calculated using the data-driven docking program HADDOCK.<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> To take into account the molecular motions and include this information in the determination of the AIF(381–389)/CypA structural model, we used the whole NMR ensemble (20 conformers) of the AIF-derived peptide in the docking protocol calculated as reported above. In the case of CypA, we generated a conformational ensemble (51 conformers) by molecular dynamics simulations and cluster analysis techniques (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>) using the X-ray crystal structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CPL">2CPL</a>) of the protein as the template (Figures S5 and S6, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>). Additionally, in the docking protocol, we defined the AIRs between AIF(381–389) and residues of CypA based on the integrated analysis of NMR (i.e., CSPs and intensity profile analysis) and structural data (i.e., ASA; see the previous paragraph and the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>) (Figure S4, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>). The docking calculation generated 200 solutions that were next clustered using a 5 Å rmsd as the cutoff based on the pairwise backbone rmsd. This procedure resulted in 16 clusters, which were then ranked according to the HADDOCK scores calculated by evaluating the intermolecular energy. In addition, the docking structures were also clustered considering the <i>Z</i>-score values that indicate the standard deviation from the average of a particular cluster in terms of the HADDOCK score (the more negative the better). The top two clusters (cluster1 = 41 structures; cluster2 = 11 structures) had HADDOCK scores of −53.1 ± 12.7 and −40.3 ± 3.4 and <i>Z</i>-scores of −2.3 and −0.7, respectively (Figure S6, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>). Therefore, the representative structure from cluster1, which had the lowest HADDOCK and <i>Z</i> scores of any cluster and was more in line with the NMR structural data, was accepted as the reference structure of the complex (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A–F). The structural model of AIF(381–389)/CypA clearly indicates that the AIF-binding surface of CypA is mainly composed residues located in the loop region included between the β4 and β5 strands (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A and S7, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>). As illustrated in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A–C, AIF(381–389) binds CypA into a shallow groove of the protein surface with specific contacts flanking its catalytic site. In detail, the four basic residues Lys<sup>382</sup>, Lys<sup>384</sup>, Arg<sup>387</sup>, and Lys<sup>388</sup> of AIF(381–389) form the major contacts with CypA (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D–F): Lys<sup>382</sup> and Lys<sup>388</sup> side chains point in the same direction interacting through hydrogen bonds with the carboxyl oxygen atoms of the Glu<sup>81</sup> of CypA (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D); Lys<sup>384</sup> interacts with Lys<sup>76</sup> of the protein, and the Arg<sup>387</sup> guanidinium group is hydrogen-bonded with the carboxyl oxygen atoms of Asn<sup>71</sup> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>F). Additionally, the recognition of CypA by AIF(381–389) is further stabilized by the interaction of the Asp<sup>385</sup> of AIF(381–389), which forms a hydrogen bond with Lys<sup>76</sup> of CypA.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structural model of the AIF(381–389)/CypA complex. (A) Ribbon drawing representation of the AIF(381–389)/CypA model in which the side chains of AIF(381–389) and CypA residues, involved in the complex formation, are shown as light-green and light-blue sticks, respectively. (B) ConSurf bioinformatics analysis of CypA. The conserved and variable residues are depicted as teal and dark magenta, respectively. (C) Mapping of the CypA residues (light green) forming the AIF(381–389) binding surface onto the complex structure. The CypA is reported as a light-gray surface representation, whereas the AIF(381–389) peptide is shown as a light-blue ribbon diagram. The two main residues of the catalytic site (Arg<sup>55</sup> and Trp<sup>121</sup>) are depicted in orange (D–F). Close-up views of the AIF(381–389)/CypA complex. The side chains of the residues playing a crucial role in the recognition mechanism are illustrated as light-blue and light-green sticks for AIF(381–389) and CypA, respectively. The 2D interaction scheme is also reported. The dashed lines reported in magenta and dark blue denote hydrogen bonds and ionic interactions, respectively. The crucial residues of the binding surface of CypA are labeled.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The validation of the selected structure of the complex AIF(381–389)/CypA was performed through <i>in silico</i> alanine-scanning mutagenesis experiments.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Data obtained, in line with the experimental data previously described, predicted that Lys<sup>382</sup>, Asp<sup>385</sup>, Arg<sup>387</sup>, and Lys<sup>388</sup> are the key residues in the binding to CypA (Figure S8, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>), supporting the reliability of the AIF(381–389)/CypA structural model obtained.</div><div class="NLM_p last">Furthermore, as revealed by the ConSurf bioinformatics analysis (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B), the residues of CypA recognized by the peptide AIF(381–389) are situated in a region of the protein that shows a high degree of conservation with homologous proteins, suggesting a mechanism of ligand recognition conserved among the different variants and involving the residues surrounding the catalytic pocket.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Serum Stability of AIF(381–389)</h3><div class="NLM_p">Proteolytic degradation of peptides is a major weakness of this class of molecules, largely limiting the systemic administration.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> We thereby evaluated the susceptibility of the oxidized AIF(381–389) peptide to degradation in the presence of human serum as compared to the linear precursor AIF(370–394). The disappearance of the intact peptides incubated in human serum at 37 °C was monitored by RP-high-performance liquid chromatography (HPLC) for 12 h. As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, the linear peptide AIF(370–394) was quickly degraded with a half-life of less than 4 h and almost disappeared after 6 h. In contrast, the cyclic AIF(381–389) was remarkably resilient toward the degradation. In particular, a slight decrease was observed after 2 h, but the process did not proceed further over the next 6 h. After 12 h, more than 75% peptide remained in solution.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Serum stability profiles of AIF(381–389) and AIF(370–394). Relative peptide concentrations were determined by integrating RP-HPLC peaks from chromatograms at 215 nm. Assays were performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Biological Evaluation</h3><div class="NLM_p">To investigate the biological activity of AIF(381–389), we assessed the peptide ability in protecting the mouse hippocampal (HT-22) neuronal cells from oxidative glutamate toxicity. In HT-22 cells, the prolonged exposure to high concentrations of glutamate induced cell death by a process involving AIF/CypA nuclear translocation.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Previously, we showed that the delivery of the peptide AIF(370–394) in HT-22 cells significantly reduced neuronal damage induced by glutamate at a level similar to that observed by the siRNA downregulation of CypA.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Here, HT-22 cells were transfected with 50 μM of both AIF(381–389) and AIF(370–394) using the Pro-Ject protein transfection kit (for details see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). Then, 24 h after transfection, glutamate was added to the cells at three different concentrations (5, 6, and 7 mM), and the cell viability was detected by the MTT assay upon 16 h of glutamate exposure. First, we assessed if the peptide provided cytotoxic effects. As shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, AIF(381–389), similar to AIF(370–394), does not exhibit cytotoxic effects on cultured neuronal cells under the tested conditions (cell viability above 90%). Moreover, the MTT assay not only shows that the AIF(381–389) is able to attenuate the glutamate-mediated toxicity, as noted for the AIF(370–394), but also provides a greater neuroprotective effect compared to the parent peptide. Indeed, HT-22 cells treated with AIF(381–389) exhibit an increased survival (up to 10–20%) compared to those treated with AIF(370–394) and compared to the control cells (up to 20–30%) (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). These preliminary findings show that the new peptide can provide enhanced biological activity in living cells, as compared to the parent peptide.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Relative cell viability of HT-22 neuronal cells transfected with AIF(381–389) and AIF(370–394), as determined by the MTT assay. HT-22 cells transfected with AIF(381–389) and AIF(370–394) were treated with glutamate at 5, 6, and 7 mM for 24 h. Results obtained are shown as percentage of controls set at 100%. Data are representative of the mean ± S.D. of at least four technical replicates. Statistical significance (<i>p</i> < 0.05) compared to the respective control is indicated by the asterisk symbol (*).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70140" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70140" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The AIF/CypA complex is becoming an ever-important target for treating several diseases, and peptide-based inhibitors are showing a promising efficacy as potential therapeutics.<a onclick="showRef(event, 'ref20 ref21 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref24 ref25">(20,21,24,25)</a> We have here described a potential new lead peptide and determined the structure of the complex it forms with CypA by NMR. The molecule has been identified through a process of size reduction and sequence optimization of the previously reported peptide AIF(370–394) and its stapled analogues, which bind CypA and are selective inhibitors of the complex.<a onclick="showRef(event, 'ref21 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref21 ref24 ref25">(21,24,25)</a> The new peptide spans residues 381–389 of AIF and is cyclized through a disulfide bond connecting two cysteines added at the C- and N-terminal ends of the chain.</div><div class="NLM_p">AIF(381–389), despite its small dimensions, less than half the size of the precursor, shows an affinity <i>in vitro</i> comparable to that of AIF(370–394). However, it should be noted that AIF(381–389) has a reduced potency compared to the previously reported AIF(370–394) stapled analogue, which shows a KD in the very low micromolar range (KD = 0.85 μM).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> These results indicate that the interaction of AIF(370–394) with CypA involves residues from the N- and C-terminal ends of the peptide as already reported in previous studies<a onclick="showRef(event, 'ref27 ref29'); return false;" href="javascript:void(0);" class="ref ref27 ref29">(27,29)</a> and also that the loss of affinity is likely compensated by the conformational constriction of AIF(381–389) in its native conformation. The cyclization significantly restricts the conformational space explored by AIF(381–389), resulting in the formation of intramolecular hydrophobic interactions between the residues Ile<sup>381</sup>–Val<sup>389</sup> and Leu<sup>383</sup>–Val<sup>389</sup> and an extensive network of hydrogen bonds involving the backbone atoms, which significantly stabilize the native β-hairpin structure adopted by the fragment free in solution. Noteworthily, the improved conformational rigidity of the peptide enabled further structural investigations to elucidate the molecular basis of the interaction between AIF and CypA by NMR in depth, as compared to that obtained with AIF(370–394) and stapled AIF(370–394). In fact, the high flexibility of the linear peptide and the low solubility of the stapled AIF(370–394) precluded a detailed structural characterization by NMR analysis either free in solution or in complex with CypA.<a onclick="showRef(event, 'ref27 ref29'); return false;" href="javascript:void(0);" class="ref ref27 ref29">(27,29)</a></div><div class="NLM_p">Data here obtained show that, similar to what was suggested for AIF mimetic peptides in previous 3D molecular models, AIF(381–389) binds CypA into a shallow groove flanking the protein catalytic site and establishes specific contacts mediated by the AIF residues Lys<sup>382</sup>, Arg<sup>387</sup>, Lys<sup>388</sup>, and Asp<sup>385</sup>, demonstrating that it retains high selectivity for the binding site of the target. The new peptide shows a remarkably higher resistance to degradation in human serum and an improved ability to prevent cell death in a cellular model of glutamate-induced neurotoxicity compared to the AIF(370–394) parent peptide. Considering the instability of disulfide bonds in the cytoplasm due to the reducing conditions, the improved biological activity observed with AIF(381–389) is probably even underestimated. Further improvements of the biological activity are expected by replacing the unstable disulfide bridges with isosteric more stable bonds (e.g., triazole bridges or thioethers).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div><div class="NLM_p last">Despite the relevant role played by the AIF/CypA complex in pathways critical for programed cell death, no clinical inhibitors of the complex have been developed to date. This is certainly due to the lack of reliable structural data; therefore, we strongly believe that the identification of a new small and compact peptide inhibitor of the AIF/CypA complex and the detailed understanding of its recognition mechanism with CypA are the crucial steps toward the generation of drug candidates for therapeutic intervention in neurodegenerative and cardiac diseases underpinning the nonphysiological activity of the AIF. The structural data here provided are thereby very useful for drug development programs based on structure–activity relationship studies and on computational approaches for improving the druggability of the peptide and to predict new highly effective, target selective, stable, and cell-permeable inhibitors.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51157" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51157" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Materials</h3><div class="NLM_p last">Protected amino acids, coupling reagents (HATU), and Fmoc-Rink Amide AM Resin used for peptide synthesis were obtained from GL Biochem (Shanghai, PRC). The solvents, including CH<sub>3</sub>CN, DMF, trifluoroacetyl (TFA), CH<sub>3</sub>OH, sym-collidine, DIPEA, and piperidine were obtained from Merck (Milan, Italy). Analytical HPLC analyses were performed on an Alliance HT Waters 2795 system, equipped with a PDA Waters detector 2996. Preparative purifications were carried out on a Waters 2545 preparative system (Waters, Milan, Italy) fitted with a Waters 2489 UV/visible detector. LC–MS analyses were performed using an ESI Ion Trap HCT ETD IIHC Ultra PTM Discovery system Bruker mass spectrometer coupled with an HPLC System Alliance e2695 separation module fitted with a 2998 PDA detector (Waters, Milan, Italy). An EnSpire Multimode Plate Reader (PerkinElmer, Monza, Italy) was used to perform label-free assays.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Peptide and Protein Preparation</h3><div class="NLM_p">The peptides were synthesized following the Fmoc methodology as previously described in the literature.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The peptides were cyclized by dissolving the linear peptides with cysteines at N- and C-termini at a concentration of 10<sup>–3</sup> to 10<sup>–4</sup> M in 0.015 M ammonium carbonate pH 8.0 for 24 h under agitation at room temperature. The reactions were monitored by analytical HPLC, and at the end, after acidification with TFA, the mixtures were lyophilized and directly used for preparative HPLC purification. The linear peptide was similarly obtained, skipping the cyclization step. Peptide purification was carried out using an Onyx monolithic semi-PREP C18 column, operating at a flow rate of 15 mL/min using H<sub>2</sub>O + 0.1% TFA (solvent A) and CH<sub>3</sub>CN + 0.1% TFA (solvent B) as solvents. The linear gradient used for the elution was: 0–70% of solvent B for 20 min. Purity and identity of peptides were assessed by analytical RP-HPLC and LC–MS (Figure S9, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Supporting Information</a>).<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The yields of the target peptides, calculated as [(experimental weight of pure peptide)/(theoretical weight) × 100], where the theoretical weight was calculated on the basis of the synthesis scale used, were estimated to be around 60%. The relative purity of peptides was calculated as the ratio of the peak area of the target peptide and the sum of areas of all detected peaks from the UV chromatograms at 210 nm. The purity of all peptides was >95%.</div><div class="NLM_p last">Recombinant His6-tagged-CypA, <sup>15</sup>N-labeled CypA, was efficiently expressed in BL21(DE3) <i>Escherichia coli</i> cells, purified to homogeneity and characterized as previously reported in the literature.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> The protein concentration was determined by reading the absorbance at 280 nm using a NanoDrop2000c UV–vis spectrophotometer (Thermo Scientific, USA). The concentration of the peptides, lacking tryptophan and tyrosine residues, was determined via the Scopes method,<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> in which the absorbance of the peptide bond was monitored at 205 nm by a NanoDrop200c UV–vis spectrophotometer.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Corning Epic Label-Free Technology</h3><div class="NLM_p last">Binding assays were performed using the Corning Epic label-free technology on the EnSpire multimode plate reader, as previously reported (PerkinElmer, Rodgau, Germany).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> In brief, CypA immobilization on the optical biosensors was accomplished by adding 200 μg/mL protein in 20 mM sodium acetate, pH 5.5, using a 12-channel Thermo Scientific matrix multichannel equalizer pipette followed by overnight incubation at 4 °C. The microplate was subsequently washed three times with phosphate-buffered saline (PBS, pH 7.4) buffer. After the washing steps, the samples coated on wells were equilibrated in the assay buffer (PBS) for 3 h (30 μL) and subsequently, a baseline was recorded. After another washing step, 15 μL of peptides were dispensed in the plate wells. Peptides were diluted with the assay buffer (PBS) at a working concentration of 300 μM (150 μM final concentration in the plate) and then further diluted in the assay buffer directly in a 384-well polypropylene plate for a total of 12 different concentrations. The final readings were taken over a period of 1 h. The difference between the last baseline measurements and the maximum signal was used to determine the KD value. Graphs were generated using GraphPad PrismR V-5.01 (GraphPad Software, La Jolla, CA). The results are means of triplicates.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Serum Stability Assay</h3><div class="NLM_p last">The peptides were mixed with 10% FBS to a final concentration of 2.0 mg/mL and incubated at 37 °C. Aliquots of 50 μL taken after 0, 30, 60, 120, 240, 360, and 720 min were mixed with 100 μL of 90% ethanol and incubated on ice for 15 min. After centrifugation (12,000<i>g</i>, 2 min), the supernatant was diluted with 0.1% TFA in H<sub>2</sub>O (1:1). The samples were analyzed by RP-HPLC on a C18 Onyx column (2.0 × 50 mm, 5 μm), applying a linear gradient from 5 to 70% of solvent B (0.1% TFA in CH<sub>3</sub>CN) over solvent A (0.1% TFA in water) in 20 min at a flow rate of 0.6 mL/min. All stability tests were performed at least in triplicate.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Cell Culture of HT-22 and Transfection</h3><div class="NLM_p last">HT-22 cells were cultured at a density of 10,000 cells/well in a 96-well plate (Greiner Bio-One, Frickenhausen, Germany) at 37 °C and 5% CO<sub>2</sub> in Dulbecco’s modified Eagle medium (Invitrogen, Karlsruhe, Germany) with 10% heat-inactivated calf serum and 100 U/mL penicillin, 100 μg/mL streptomycin, and 2 mM glutamine. The transient transfection of the peptides was performed using the cationic lipid mixture Pro-Ject protein transfection reagent kit according to the manufacturer’s instructions (Pierce, Thermo Scientific, Munich, Germany, Prod no. 89850).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Cell Viability Assay</h3><div class="NLM_p last">The quantification of cell viability in HT-22 cells was performed in standard 96-well plates by the method of MTT (Sigma-Aldrich) reduction. Then, 24 h after peptide transfection, the glutamate-mediated oxidative stress<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> was induced by adding to HT-22 fresh medium containing high concentrations of glutamate (5, 6, and 7 mM) (Sigma-Aldrich, Munich, Germany). Therefore, the cell viability was evaluated between 14 and 16 h later. The reagent was used at 0.25 mg/mL for 1 h. After terminating the reaction by removing the media and freezing the plate at −80 °C for at least 1 h, the MTT dye was dissolved in dimethyl sulfoxide (DMSO) and absorbance was determined at 590 nm with the blank obtained at 630 nm (FluoStar OPTIMA; BMG Labtech, Offenburg, Germany). The cell viability levels were determined by comparing absorption of treated cell samples with that of untreated control cell samples (100% cell viability). The DMSO control was used as the solvent control. The results obtained are shown as a percentage of controls set at 100%. The data are representative of the mean ± S.D. of at least four technical replicates.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Nuclear Magnetic Resonance Measurements</h3><div class="NLM_p">NMR experiments were performed at 298 K by using a Bruker AVIII HD 600 MHz spectrometer, equipped with a triple resonance Prodigy N2 cryoprobe with a <i>z</i>-axis pulse field gradient, at the NMR laboratory of the Department of Environmental, Biological and Pharmaceutical Science and Technology (University of Campania “Luigi Vanvitelli”). The AIF(381–389) samples were prepared by dissolving 0.75 mg of the unlabeled peptide in 200 μL of PBS pH 7.4 with 10% D<sub>2</sub>O. The pH was finally adjusted to 5.8. <sup>1</sup>H, <sup>15</sup>N, and <sup>13</sup>C chemical shift assignments of the unlabeled oxidized AIF(381–389) peptide in the free form and upon addition of CypA (molar ratio peptide/protein 150:1) were carried out by exploring the natural isotopic abundance through the use of NMR pulse sequences optimized, as reported in a previous publication.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> In brief, the following spectra were acquired and analyzed: (i) homonuclear 2D [<sup>1</sup>H–<sup>1</sup>H] TOCSY,<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> 2D [<sup>1</sup>H–<sup>1</sup>H] NOESY<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> and 2D [<sup>1</sup>H–<sup>1</sup>H] ROESY<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> were acquired using 32 (64 for ROESY) for scans per t1 increment, a spectral width (SW) of 7211.54 Hz along both dimensions, a 3.0 s relaxation delay and 2048 × 256 complex points in t2 and t1, respectively. In particular, 2D [<sup>1</sup>H–<sup>1</sup>H] TOCSY spectra were acquired using a homonuclear Hartman–Hahn transfer through a MLEV<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> sequence using a mixing time of 70 ms and 10 KHz spin-lock field strength; 2D [<sup>1</sup>H–<sup>1</sup>H] NOESY experiments were reordered with a mixing time of 250 ms; 2D [<sup>1</sup>H–<sup>1</sup>H] ROESY spectra were measured with a CW spin-lock field strength of 4 kHz, a mixing time of 250 ms, and 5.0 s relaxation delay. In all homonuclear 2D experiments, water suppression was achieved by means of water-gate pulse sequence with gradients using double echo. 2D [<sup>1</sup>H–<sup>1</sup>H] spectra were apodized with a square cosine window function and zero fill to a matrix of size 4096 × 4096 before Fourier transform and baseline correction; (ii) heteronuclear 2D [<sup>1</sup>H–<sup>15</sup>N] HSQC experiments were acquired using 880 scans per t1 increment, a SW of 1581.26 Hz along t1 and 7211.54 Hz along t2, 2048 × 128 complex points in t2 and t1, respectively, and 1.0 s relaxation delay. [<sup>1</sup>H–<sup>15</sup>N] HSQC spectra were apodized with a square cosine window function and a zero filling to a matrix of size 4096 × 4096 before Fourier transform and baseline correction. Heteronuclear 2D [<sup>1</sup>H–<sup>13</sup>C] HSQC constant-time version ([<sup>1</sup>H–<sup>13</sup>C] CT HSQC) spectra were measured with 800 scans per t1 increment, a SW of 5131.08 Hz along t1 and 7211.54 Hz along t2, 2048 × 200 complex points in t2 and t1, respectively, and 1.0 s relaxation delay. 2D [<sup>1</sup>H–<sup>13</sup>C] CT HSQC spectra were performed with a coupling constant JXH = 145 Hz, constant time period of 13.3 ms, and were apodized with a square cosine window function and zero filled to a matrix of size 4096 × 4096 before Fourier transform and baseline correction. Microsecond-to-millisecond timescale backbone dynamics characterization of the unlabeled AIF(381–389) peptide was performed by evaluating the <sup>15</sup>N R2 relaxation rate through natural abundance T2-filter [<sup>1</sup>H–<sup>15</sup>N] HSQC measurements acquired with the following acquisition parameters: 880 scans per t1 increment, SW of 1581.26 Hz along t1 and 7211.54 Hz along t2, 2048 × 128 complex points in t2 and t1, respectively, and 1.0 s relaxation delay. 2D T2-filter [<sup>1</sup>H–<sup>15</sup>N] HSQC experiments<sup>31</sup> were carried out by using two different relaxation-compensated CPMG (Carr–Purcell–Meiboom–Gill) periods (125 and 500 ms) and were apodized with a square cosine window function and a zero filling to a matrix of size 4096 × 4096 before Fourier transform and baseline correction.</div><div class="NLM_p">Amide temperature coefficients (Δδ<sub>NH</sub>/Δ<i>T</i> (ppb/K)) and 3JHNHα couplings constants for the free AIF(381–389) peptide were obtained by measuring one-dimensional (1D) 1H spectra with the following acquisition parameters: SW of 7211.54 Hz, relaxation delay 5.0 s, 128 K data points for acquisition, and 256 K for transformation. Δδ<sub>NH</sub>/Δ<i>T</i> values were obtained from a series of 1D 1H experiments measured at 298, 300, 302, 304, 306, 308, and 310 K. The AIF(381–389) residues with Δδ<sub>NH</sub>/Δ<i>T</i> larger than −4.6 ppb/K were considered to be involved in hydrogen bonding<a onclick="showRef(event, 'ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref55 ref56">(55,56)</a> in the structure calculation (see below).</div><div class="NLM_p last">To define the AIF(381–389) binding site on CypA by CSP studies, the 2D [<sup>1</sup>H–<sup>15</sup>N] HSQC spectra were recorded on His-tagged <sup>15</sup>N-labeled CypA (80 μM) in the presence of increasing concentrations of AIF(381–389) peptide ranging from 0 to 800 μM in PBS buffer pH 5.8. 2D [<sup>1</sup>H–<sup>15</sup>N] HSQC spectra were measured using eight scans per t1 increment, a SW of 2160.76 Hz along t1 and 7201.56 Hz along t2, 1024 × 128 complex points in t2 and t1, respectively, and 1.0 s relaxation delay. [<sup>1</sup>H–<sup>15</sup>N] HSQC spectra were apodized with a square cosine window function and a zero filling to a matrix of size 2048 × 1024 before Fourier transform and baseline correction. The spectra were processed using NMRpipe<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> and analyzed using SPARKY<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> and CARA<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> software. <sup>1</sup>H, <sup>13</sup>C, and <sup>15</sup>N chemical shifts were calibrated indirectly by external sodium 2,2-dimethyl-2-silapentane-5-sulfonate (DSS) references.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Chemical Shift Analysis</h3><div class="NLM_p">For the AIF(381–389) peptide, the secondary chemical shifts (Δδ) of Hα, Cα and Cβ nuclei were calculated as the difference between the observed chemical shifts (δ<sub>obs</sub>) and the random coil shifts (δ<sub>coil</sub>). These latter values were predicted from the AIF(381–389) amino-acid sequence by using different methodologies defined by Wishart and Sykes,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> De Simone and Sanz-Hernández,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Kjaergaard et al.,<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> and Tamiola and Mulder.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> In the last two cases, the chemical shifts of the Cys<sup>N-ter</sup> and Cys<sup>C-ter</sup> were not included in the data analysis. Additionally, the structural investigation of AIF(381–389) was performed by analyzing the difference between Cα and Cβ secondary chemical shifts (ΔδCα–ΔδCβ).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> In particular, consecutive positive values of ΔδCα–ΔδCβ (+2 ppm) indicate α-helix structure and negative values (−2 ppm) indicate the β-strand or extended structure.</div><div class="NLM_p last">The prediction of the redox state of Cys<sup>N-ter</sup> and Cys<sup>C-ter</sup> contained in the AIF(381–389) peptide was performed by comparing the cysteine Cα and Cβ chemical shifts with the reference values in accordance with the procedure proposed by Rajarathnam and co-workers.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> The structural rearrangements of AIF(381–389) induced by CypA binding were estimated by applying a weighted combined <sup>1</sup>H (ΔδH), <sup>15</sup>N (ΔδN) and <sup>13</sup>C (ΔδC) CSPs based on the following equation<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_m001.gif" alt="" id="_i24" /></img></span>where <i>W</i><sub>H</sub>, <i>W</i><sub>N</sub>, and <i>W</i><sub>C</sub> are the weighing factors for the <sup>1</sup>H, <sup>15</sup>N, and <sup>13</sup>C shifts defined as <i>W</i><sub>H</sub> = |γ<sub>H</sub>/γ<sub>H</sub>| = 1; <i>W</i><sub>N</sub> = |γ<sub>N</sub>/γ<sub>H</sub>| = 0.101; <i>W</i><sub>C</sub> = |γ<sub>C</sub>/γ<sub>H</sub>| = 0.251. Δδ<sub>H</sub>, Δδ<sub>N</sub>, and Δδ<sub>C</sub> are the chemical shift differences in parts per million between the compared protein states for <sup>1</sup>H, <sup>15</sup>N, and <sup>13</sup>C, respectively; γ<sub>H</sub>, γ<sub>N</sub>, and γ<sub>C</sub> are the gyromagnetic ratios. The AIF(381–389) CSPs were evaluated by dividing the chemical shifts in three groups: backbone (H<sub>N</sub>, Hα, Cα, and N), side chains, and all together. In the case of CypA, the CSP analysis was performed using the backbone H<sub>N</sub> and N shifts. The model-free order parameter <i>S</i><sup>2</sup> for the AIF(381–389) backbone amide group were predicted from H<sub>N</sub>, Hα, Cα, N, and Cβ chemical shifts using a random coil index approach in accordance with the chemical shift-based methodology developed by Berjanskii and Wishart.<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60,61)</a></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Structure Determination of AIF(381–389)</h3><div class="NLM_p">The NMR structure of AIF(381–389) was calculated with the program CYANA 3.0<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> using ROE-derived distance constraints, backbone dihedral angles, and hydrogen bonds. In particular, backbone dihedral angles were obtained using NMR chemical shifts by TALOS+;<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> whereas the hydrogen bonds were defined for residues showing amide temperature coefficients larger than −4.6 ppb/K. The input restraints for the final structure are reported in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Table S5</a>. In particular, the NMR conformational ensemble of AIF(381–389) was calculated using (i) a total of 151 distance constraints (30 sequential and 23 nonsequential) from the ROEs (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Table S6</a>), (ii) 3 hydrogen bonds identified from the amide proton temperature coefficients (ΔδH<sub>N</sub>/Δ<i>T</i>) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">Table S5</a>), and (iii) 12 TALOS + dihedral angle restraints derived from Hα, H<sub>N</sub>, N, Cα, and Cβ chemical shifts.</div><div class="NLM_p last">A total of 100 structures were calculated, and the 20 conformers having the lowest CYANA target function were selected to generate the final NMR conformational ensemble. The absence of residual constraint violations demonstrates that the input data represent a self-consistent set and the restraints are well satisfied in the calculated structures. The NMR structures were visualized and evaluated by using the programs MOLMOL,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> CHIMERA,<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> and PROCHECK-NMR.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Generation of the CypA Conformational Ensemble</h3><div class="NLM_p last">In order to provide a rigorous description of the conformational space sampled by the CypA protein, we used a conformational sampling approach to generate a pool of CypA conformers. The normal mode-based simulation (NMSim) methodology has been demonstrated to be a computationally efficient alternative to molecular dynamics simulations for conformational sampling of proteins.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> Therefore, an ensemble of 500 conformers was generated with the software NMSim, starting from the CypA reference structure that was generated by applying a mild minimization to the protonated X-ray structure (PDB ID code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CPL">2CPL</a>) (pH 5.8), in which explicit hydrogens were added using the software H<sup>2+</sup>.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> In particular, the minimization was performed by the software SPDB,<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> whereas the conformational ensemble was generated using the default NMSim parameters suitable for large motions. Then, the conformational ensemble was clustered by using the software NMRCLUST<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> implemented in the program CHIMERA.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> A total of 51 clusters were found, and for each of them, the representative structure was considered to obtain the final MD ensemble. The adaptive Poisson–Boltzmann solver<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> was used to calculate spatial distributions of CypA electrostatic potentials using the linearized Poisson–Boltzmann equation and parameters from the PQR files of the CypA X-ray structure obtained using the PDB2PQR server.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> The CypA electrostatic map was generated with CHIMERA.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Molecular Docking Calculation</h3><div class="NLM_p last">The software HADDOCK<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> was used to dock the peptide AIF(381–389) to the protein CypA using AIRs identified according to the NMR data (i.e., CSP and intensity ratio analysis) and solvent-accessible CypA residues defined by using the ProtSA approach.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> Six residues (Thr68, Arg<sup>69</sup>, Gly<sup>72</sup>, Gly<sup>75</sup>, Asn<sup>102</sup>, and Ala<sup>103</sup>) of CypA, showing significant chemical shift, and NMR signal intensity variations upon binding to AIF(381–389) with >40% solvent accessibility were defined as “active” residues, whereas for the AIF(381–389) peptide, four residues (Lys<sup>382</sup>, Asp<sup>385</sup>, Arg<sup>387</sup>, and Lys<sup>388</sup>) were selected as “active” according to the CSPs and H<sub>N</sub>/N signal intensity profile. In both cases, the “passive” amino acids were automatically selected by the program HADDOCK. The starting structures of AIF(381–389) for docking were the 20 conformers of the NMR ensemble calculated as reported below, while for CypA, the MD ensemble (51 structures) was generated by molecular dynamics simulations and cluster analysis (as described above). The structural model of the AIF(381–389)/CypA complex was performed following the HADDOCK 2.2 protocol using optimal setting parameters for docking of peptides (i.e., number of MD steps for rigid-body high temperature TAD = 2000; number of MD steps during the first rigid body cooling stage = 2000; number of MD steps during the second cooling stage with flexible side chains at the interface = 4000; number of MD steps during the third cooling stage with fully flexible interface = 4000).<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> In particular, during rigid-body energy minimization, 2000 docking conformers were calculated. The best 200 structures in terms of intermolecular energies were then used for the semiflexible simulating annealing, followed by explicit water refinement. Then, the obtained structures were clustered using a 5 Å rmsd as the cutoff based on the pairwise backbone rmsd. The representative structure from the lowest energy cluster (cluster 1), containing 41 structures, which had the lowest HADDOCK/<i>Z</i>-score and was more in line with the NMR experimental data, was accepted as reference structural model of the complex. The selected AIF(381–389)/CypA structure was analyzed and visualized using the software CHIMERA. The 2D interaction scheme of the AIF(381–389)/CypA complex was obtained and visualized using PIC<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> and Schrodinger Maestro software. The sequence conservation profile of CypA was calculated using the ConSurf server (<a href="https://consurf.tau.ac.il" class="extLink">https://consurf.tau.ac.il</a>). The <i>in silico</i> alanine scanning for the AIF(381–389) residues was performed by the DrugScore web server (<a href="https://cpclab.uni-duesseldorf.de/index.php/Web-Server" class="extLink">https://cpclab.uni-duesseldorf.de/index.php/Web-Server</a>) using the selected PDB file of the AIF(381–389)/CypA complex as input.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00777" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11255" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11255" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00777?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00777</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Amino acid sequences, SMILES code formula and theoretical and experimental masses of AIF-derived peptides; <sup>1</sup>H and <sup>15</sup>N chemical shift assignments of AIF(381–389); <sup>13</sup>C chemical shift assignments of AIF(381–389) peptide; cysteine Cα and Cβ chemical shifts analysis of AIF(381–389) peptide; amide proton temperature coefficients (ΔδNH/Δ<i>T</i>, 298–310 K), Scalar couplings (3JH<sub>N</sub>Hα) and model free order parameters (S2) obtained for AIF(381–389) peptide in the free form; NMR structural statistics of the AIF(381–389) peptide; 2D [<sup>1</sup>H–<sup>1</sup>H] TOCSY (green) and 2D [<sup>1</sup>H–<sup>1</sup>H] ROESY (blue) spectra used for the identification of spin systems and the chemical shifts assignment of the AIF(381–389); chemical shift deviation from random coil values of HN and Hα backbone atoms plotted as a function of residue number; T2-filter <sup>1</sup>H–<sup>15</sup>N HSQC spectrum of AIF(381–389); secondary structure elements of CypA; flow chart of the NMR-based strategy used to identify the CypA residues involved in the recognition process of the AIF(381–389) peptide; CypA conformational ensemble generated by molecular dynamics simulations before and after cluster analysis; flow chart of the molecular docking protocol; <i>in silico</i> alanine scanning analysis of the AIF(381–389) peptide; and MS profiles of purified peptides (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">NMR structure of AIF(381-389) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Structure of the complex AIF(381-389)/CypA  (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_004.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf">jm1c00777_si_001.pdf (1.37 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_002.pdb">jm1c00777_si_002.pdb (112.97 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_003.pdb">jm1c00777_si_003.pdb (130.6 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_004.csv">jm1c00777_si_004.csv (5.42 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00777" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63847" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63847" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nunzianna Doti</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biostructures and Bioimaging (IBB)-CNR, Via Mezzocannone, 16, 80134 Napoli, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2952-6658" title="Orcid link">https://orcid.org/0000-0003-2952-6658</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#325c475c485b535c5c531c565d465b72515c401c5b46"><span class="__cf_email__" data-cfemail="ff918a9185969e91919ed19b908b96bf9c918dd1968b">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luigi Russo</span> - <span class="hlFld-Affiliation affiliation">Department
of Environmental, Biological and Pharmaceutical Sciences and Technologies, University of Campania “L. Vanvitelli”, Via Vivaldi 43, 81100 Caserta, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fabiola Mascanzoni</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biostructures and Bioimaging (IBB)-CNR, Via Mezzocannone, 16, 80134 Napoli, Italy</span>; 
    <span>Present Address:
                        Institute of Biochemistry and Cell Biology (IBBC), National Research Council (CNR), Via Pietro Castellino 111, 80131, Napoli, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Biancamaria Farina</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biostructures and Bioimaging (IBB)-CNR, Via Mezzocannone, 16, 80134 Napoli, Italy</span>; 
    <span>Present Address:
                        Advanced Accelerator Applications, a Novartis Company; Via Vivaldi 43, 81100, Caserta, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Amalia Mihaela Dolga</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmacology and Clinical Pharmacy, University
of Marburg, 35043 Marburg, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Faculty
of Science and Engineering, Groningen Research Institute of Pharmacy
(GRIP), Research School of Behavioural and Cognitive Neurosciences
(BCN), Department of Molecular Pharmacology, University of Groningen, 9713 AV Groningen, The Netherlands</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5400-5614" title="Orcid link">https://orcid.org/0000-0001-5400-5614</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alessandra Monti</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biostructures and Bioimaging (IBB)-CNR, Via Mezzocannone, 16, 80134 Napoli, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Caporale</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biostructures and Bioimaging (IBB)-CNR, Via Mezzocannone, 16, 80134 Napoli, Italy</span>; 
    <span>Present Address:
                        Institute of crystallization (IC), National Research Council (CNR), c/o AREA SCIENCE PARKS.S. 14 Km 163.5, 3414-Basovizza, Trieste, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carsten Culmsee</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmacology and Clinical Pharmacy, University
of Marburg, 35043 Marburg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roberto Fattorusso</span> - <span class="hlFld-Affiliation affiliation">Department
of Environmental, Biological and Pharmaceutical Sciences and Technologies, University of Campania “L. Vanvitelli”, Via Vivaldi 43, 81100 Caserta, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-3539-5343" title="Orcid link">https://orcid.org/0000-0002-3539-5343</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Menotti Ruvo</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biostructures and Bioimaging (IBB)-CNR, Via Mezzocannone, 16, 80134 Napoli, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5997-756X" title="Orcid link">https://orcid.org/0000-0001-5997-756X</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>L.R. and F.M. contributed equally. We confirm that all authors have approved the submission of the final version of this manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AIF</td><td class="NLM_def"><p class="first last">apoptosis-inducing factor</p></td></tr><tr><td class="NLM_term">CH<sub>3</sub>CN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">CypA</td><td class="NLM_def"><p class="first last">cyclophilin A</p></td></tr><tr><td class="NLM_term">CH<sub>3</sub>OH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">CSI</td><td class="NLM_def"><p class="first last">chemical shift index</p></td></tr><tr><td class="NLM_term">CSP</td><td class="NLM_def"><p class="first last">chemical shift perturbation</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">D<sub>2</sub>O</td><td class="NLM_def"><p class="first last">water-d2</p></td></tr><tr><td class="NLM_term">FCC</td><td class="NLM_def"><p class="first last">fraction of common contact</p></td></tr><tr><td class="NLM_term">Fmoc</td><td class="NLM_def"><p class="first last">9-fluorenylmethoxycarbonyl</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">HSQC</td><td class="NLM_def"><p class="first last">heteronuclear single quantum coherence spectroscopy</p></td></tr><tr><td class="NLM_term">HT-22</td><td class="NLM_def"><p class="first last">mouse hippocampal neuronal cell line</p></td></tr><tr><td class="NLM_term">KD</td><td class="NLM_def"><p class="first last">dissociation constant</p></td></tr><tr><td class="NLM_term">LC–MS</td><td class="NLM_def"><p class="first last">liquid chromatography–mass spectrometry</p></td></tr><tr><td class="NLM_term">MTT</td><td class="NLM_def"><p class="first last">3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">NOESY</td><td class="NLM_def"><p class="first last">nuclear Overhauser effect spectroscopy</p></td></tr><tr><td class="NLM_term">NOE</td><td class="NLM_def"><p class="first last">nuclear Overhauser effect</p></td></tr><tr><td class="NLM_term">PBS</td><td class="NLM_def"><p class="first last">phosphate-buffered saline</p></td></tr><tr><td class="NLM_term">RP-HPLC</td><td class="NLM_def"><p class="first last">reversed-phase high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">ROESY</td><td class="NLM_def"><p class="first last">rotating frame Overhauser effect spectroscopy</p></td></tr><tr><td class="NLM_term">ROE</td><td class="NLM_def"><p class="first last">rotating frame Overhauser effect</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">TOCSY</td><td class="NLM_def"><p class="first last">total correlated spectroscopy</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27104" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27104" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 74 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maté, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz-Lombardía, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boitel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haouz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tello, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Susin, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penninger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alzari, P. M.</span></span> <span> </span><span class="NLM_article-title">The crystal structure of the mouse apoptosis-inducing factor AIF</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">442</span>– <span class="NLM_lpage">446</span>, <span class="refDoi"> DOI: 10.1038/nsb793</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Fnsb793" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=11967568" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvFOms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=442-446&author=M.+J.+Mat%C3%A9author=M.+Ortiz-Lombard%C3%ADaauthor=B.+Boitelauthor=A.+Haouzauthor=D.+Telloauthor=S.+A.+Susinauthor=J.+Penningerauthor=G.+Kroemerauthor=P.+M.+Alzari&title=The+crystal+structure+of+the+mouse+apoptosis-inducing+factor+AIF&doi=10.1038%2Fnsb793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The crystal structure of the mouse apoptosis-inducing factor AIF</span></div><div class="casAuthors">Mate, Maria J.; Ortiz-Lombardia, Miguel; Boitel, Brigitte; Haouz, Ahmed; Tello, Diana; Susin, Santos A.; Penninger, Josef; Kroemer, Guido; Alzari, Pedro M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">442-446</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Mitochondria play a key role in apoptosis due to their capacity to release potentially lethal proteins.  One of these latent death factors is cytochrome c, which can stimulate the proteolytic activation of caspase zymogens.  Another important protein is apoptosis-inducing factor (AIF), a 57-kDa electron-transferring flavoprotein oxidoreductase, which can stimulate a caspase-independent cell-death pathway required for early embryonic morphogenesis.  Here, the authors report the crystal structure of mouse mitochondrial AIF at 2.0 Å.  Its active site structure and redox properties suggest that AIF functions as an electron transferase with a mechanism similar to that of the bacterial ferredoxin reductases, its closest evolutionary homologs.  However, AIF structurally differs from these proteins in some essential features, including a long insertion in a C-terminal β-hairpin loop, which may be related to its apoptogenic functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBoQsgWDBjzbVg90H21EOLACvtfcHk0lh8kQt76kzjYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvFOms78%253D&md5=ab8fec73c8964c16f5282ae87a5c87ac</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnsb793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb793%26sid%3Dliteratum%253Aachs%26aulast%3DMat%25C3%25A9%26aufirst%3DM.%2BJ.%26aulast%3DOrtiz-Lombard%25C3%25ADa%26aufirst%3DM.%26aulast%3DBoitel%26aufirst%3DB.%26aulast%3DHaouz%26aufirst%3DA.%26aulast%3DTello%26aufirst%3DD.%26aulast%3DSusin%26aufirst%3DS.%2BA.%26aulast%3DPenninger%26aufirst%3DJ.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DAlzari%26aufirst%3DP.%2BM.%26atitle%3DThe%2520crystal%2520structure%2520of%2520the%2520mouse%2520apoptosis-inducing%2520factor%2520AIF%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D9%26spage%3D442%26epage%3D446%26doi%3D10.1038%2Fnsb793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sevrioukova, I. F.</span></span> <span> </span><span class="NLM_article-title">Apoptosis-inducing factor: structure, function, and redox regulation</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2545</span>– <span class="NLM_lpage">2579</span>, <span class="refDoi"> DOI: 10.1089/ars.2010.3445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1089%2Fars.2010.3445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=20868295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtl2lu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2011&pages=2545-2579&author=I.+F.+Sevrioukova&title=Apoptosis-inducing+factor%3A+structure%2C+function%2C+and+redox+regulation&doi=10.1089%2Fars.2010.3445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Apoptosis-inducing factor: structure, function, and redox regulation</span></div><div class="casAuthors">Sevrioukova, Irina F.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2545-2579</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Apoptosis-inducing factor (AIF) is a FAD-contg., NADH-dependent oxidoreductase residing in the mitochondrial intermembrane space whose specific enzymic activity remains unknown.  Upon an apoptotic insult, AIF undergoes proteolysis and translocates to the nucleus, where it triggers chromatin condensation and large-scale DNA degrdn. in a caspase-independent manner.  In addn. to playing a key role in the execution of caspase-independent cell death, AIF has emerged as a protein crit. for cell survival.  Anal. of in vivo phenotypes assocd. with AIF deficiency and defects, and identification of its mitochondrial, cytoplasmic, and nuclear partners has revealed the complexity and multi-level regulation of AIF-mediated signal transduction and has suggested an important role of AIF in the maintenance of mitochondrial morphol. and energy metab.  The redox activity of AIF is essential for optimal oxidative phosphorylation.  Addnl., the protein is proposed to regulate the respiratory chain indirectly, through assembly and/or stabilization of complexes I and III.  Here, the author discusses accumulated data with respect to AIF structure and outlines evidence that supports the prevalent mechanistic view on the apoptogenic actions of the flavoprotein, as well as the emerging concept of AIF as a redox sensor capable of linking NAD(H)-dependent metabolic pathways to apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9o4PRjq8nbLVg90H21EOLACvtfcHk0lgBeqeeqYnNtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtl2lu7g%253D&md5=05dad64022bf3bd44ec55c0139f7f420</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1089%2Fars.2010.3445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2010.3445%26sid%3Dliteratum%253Aachs%26aulast%3DSevrioukova%26aufirst%3DI.%2BF.%26atitle%3DApoptosis-inducing%2520factor%253A%2520structure%252C%2520function%252C%2520and%2520redox%2520regulation%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2011%26volume%3D14%26spage%3D2545%26epage%3D2579%26doi%3D10.1089%2Fars.2010.3445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackerman, S. L.</span></span> <span> </span><span class="NLM_article-title">Oxidative stress, cell cycle, and neurodegeneration</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">785</span>– <span class="NLM_lpage">793</span>, <span class="refDoi"> DOI: 10.1172/jci200318182</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1172%2FJCI200318182" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12639981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlOjsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2003&pages=785-793&author=J.+A.+Kleinauthor=S.+L.+Ackerman&title=Oxidative+stress%2C+cell+cycle%2C+and+neurodegeneration&doi=10.1172%2Fjci200318182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Oxidative stress, cell cycle, and neurodegeneration</span></div><div class="casAuthors">Klein, Jeffrey A.; Ackerman, Susan L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">785-793</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">A review discusses the influence of oxidative stress on neurodegeneration and the possible connections between oxidative stress and unscheduled cell cycle reentry.  The misregulation of the cell cycle in many cell types can lead to unchecked proliferation and neoplastic disease.  Some of the potential pathways by which the repression of the cell cycle in neurons may be modulated by reactive oxygen species are presented.  Specific types of neurons may responds to different signaling pathways, or need activation of multiple signaling pathways.  The testing of the roles of these and other potential pathways in neurodegeneration will require the development and anal. of the addnl. animals models, in addn. to further correlative human studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6guQbmgpEXbVg90H21EOLACvtfcHk0lgBeqeeqYnNtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlOjsbw%253D&md5=051d0f2d1745075cf73d56e8c4d19261</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1172%2FJCI200318182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI200318182%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DJ.%2BA.%26aulast%3DAckerman%26aufirst%3DS.%2BL.%26atitle%3DOxidative%2520stress%252C%2520cell%2520cycle%252C%2520and%2520neurodegeneration%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2003%26volume%3D111%26spage%3D785%26epage%3D793%26doi%3D10.1172%2Fjci200318182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longo-Guess, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossmann, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seburn, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurd, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackerman, S. L.</span></span> <span> </span><span class="NLM_article-title">The harlequin mouse mutation downregulates apoptosis-inducing factor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>419</i></span>,  <span class="NLM_fpage">367</span>– <span class="NLM_lpage">374</span>, <span class="refDoi"> DOI: 10.1038/nature01034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Fnature01034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12353028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD38XntlGgsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=419&publication_year=2002&pages=367-374&author=J.+A.+Kleinauthor=C.+M.+Longo-Guessauthor=M.+P.+Rossmannauthor=K.+L.+Seburnauthor=R.+E.+Hurdauthor=W.+N.+Frankelauthor=R.+T.+Bronsonauthor=S.+L.+Ackerman&title=The+harlequin+mouse+mutation+downregulates+apoptosis-inducing+factor&doi=10.1038%2Fnature01034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The harlequin mouse mutation downregulates apoptosis-inducing factor</span></div><div class="casAuthors">Klein, Jeffrey A.; Longo-Guess, Chantal M.; Rossmann, Marlies P.; Seburn, Kevin L.; Hurd, Ronald E.; Frankel, Wayne N.; Bronson, Roderick T.; Ackerman, Susan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">419</span>
        (<span class="NLM_cas:issue">6905</span>),
    <span class="NLM_cas:pages">367-374</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Harlequin (Hq) mutant mice have progressive degeneration of terminally differentiated cerebellar and retinal neurons.  The authors have identified the Hq mutation as a proviral insertion in the apoptosis-inducing factor (Aif) gene, causing about an 80% redn. in AIF expression.  Mutant cerebellar granule cells are susceptible to exogenous and endogenous peroxide-mediated apoptosis, but can be rescued by AIF expression.  Overexpression of AIF in wild-type granule cells further decreases peroxide-mediated cell death, suggesting that AIF serves as a free radical scavenger.  In agreement, dying neurons in aged Hq mutant mice show oxidative stress.  In addn., neurons damaged by oxidative stress in both the cerebellum and retina of Hq mutant mice re-enter the cell cycle before undergoing apoptosis.  Our results provide a genetic model of oxidative stress-mediated neurodegeneration and demonstrate a direct connection between cell cycle re-entry and oxidative stress in the ageing central nervous system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrae1zYgtmzcrVg90H21EOLACvtfcHk0lgBeqeeqYnNtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntlGgsb8%253D&md5=5c229f0747c0c6e975cc6c695ca98367</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature01034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01034%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DJ.%2BA.%26aulast%3DLongo-Guess%26aufirst%3DC.%2BM.%26aulast%3DRossmann%26aufirst%3DM.%2BP.%26aulast%3DSeburn%26aufirst%3DK.%2BL.%26aulast%3DHurd%26aufirst%3DR.%2BE.%26aulast%3DFrankel%26aufirst%3DW.%2BN.%26aulast%3DBronson%26aufirst%3DR.%2BT.%26aulast%3DAckerman%26aufirst%3DS.%2BL.%26atitle%3DThe%2520harlequin%2520mouse%2520mutation%2520downregulates%2520apoptosis-inducing%2520factor%26jtitle%3DNature%26date%3D2002%26volume%3D419%26spage%3D367%26epage%3D374%26doi%3D10.1038%2Fnature01034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deinum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modjtahedi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neagu, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eriksson, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luban, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomgren, K.</span></span> <span> </span><span class="NLM_article-title">Cyclophilin A participates in the nuclear translocation of apoptosis-inducing factor in neurons after cerebral hypoxia-ischemia</span>. <i>J. Exp. Med.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>204</i></span>,  <span class="NLM_fpage">1741</span>– <span class="NLM_lpage">1748</span>, <span class="refDoi"> DOI: 10.1084/jem.20070193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1084%2Fjem.20070193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=17635954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2sXptVyjs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=204&publication_year=2007&pages=1741-1748&author=C.+Zhuauthor=X.+Wangauthor=J.+Deinumauthor=Z.+Huangauthor=J.+Gaoauthor=N.+Modjtahediauthor=M.+R.+Neaguauthor=M.+Nilssonauthor=P.+S.+Erikssonauthor=H.+Hagbergauthor=J.+Lubanauthor=G.+Kroemerauthor=K.+Blomgren&title=Cyclophilin+A+participates+in+the+nuclear+translocation+of+apoptosis-inducing+factor+in+neurons+after+cerebral+hypoxia-ischemia&doi=10.1084%2Fjem.20070193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclophilin A participates in the nuclear translocation of apoptosis-inducing factor in neurons after cerebral hypoxia-ischemia</span></div><div class="casAuthors">Zhu, Changlian; Wang, Xiaoyang; Deinum, Johanna; Huang, Zhiheng; Gao, Jianfeng; Modjtahedi, Nazanine; Neagu, Martha R.; Nilsson, Michael; Eriksson, Peter S.; Hagberg, Henrik; Luban, Jeremy; Kroemer, Guido; Blomgren, Klas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Medicine</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">204</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1741-1748</span>CODEN:
                <span class="NLM_cas:coden">JEMEAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-1007</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Upon cerebral hypoxia-ischemia (HI), apoptosis-inducing factor (AIF) can move from mitochondria to nuclei, participate in chromatinolysis, and contribute to the execution of cell death.  Previous work performed in vitro suggests that AIF must interact with cyclophilin A (CypA) to form a proapoptotic DNA degrdn. complex.  The authors addressed the question as to whether elimination of CypA may afford neuroprotection in vivo.  9-D-old wild-type (WT), CypA+/-, or CypA-/- mice were subjected to unilateral cerebral HI.  The infarct vol. after HI was reduced by 47% in CypA-/- mice compared with their WT littermates.  Importantly, CypA-/- neurons failed to manifest the HI-induced nuclear translocation of AIF that was obsd. in WT neurons.  Conversely, CypA accumulated within the nuclei of damaged neurons after HI, and this nuclear translocation of CypA was suppressed in AIF-deficient harlequin mice.  Immunopptn. of AIF revealed copptn. of CypA, but only in injured, ischemic tissue.  Surface plasmon resonance revealed direct mol. interactions between recombinant AIF and CypA.  These data indicate that the lethal translocation of AIF to the nucleus requires interaction with CypA, suggesting a model in which two proteins that normally reside in sep. cytoplasmic compartments acquire novel properties when moving together to the nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ0LbSrzcFfbVg90H21EOLACvtfcHk0liRq6YKczYbDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXptVyjs7s%253D&md5=cc2086d63cd3d6d3200b09605f82af32</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1084%2Fjem.20070193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20070193%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DDeinum%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DModjtahedi%26aufirst%3DN.%26aulast%3DNeagu%26aufirst%3DM.%2BR.%26aulast%3DNilsson%26aufirst%3DM.%26aulast%3DEriksson%26aufirst%3DP.%2BS.%26aulast%3DHagberg%26aufirst%3DH.%26aulast%3DLuban%26aufirst%3DJ.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DBlomgren%26aufirst%3DK.%26atitle%3DCyclophilin%2520A%2520participates%2520in%2520the%2520nuclear%2520translocation%2520of%2520apoptosis-inducing%2520factor%2520in%2520neurons%2520after%2520cerebral%2520hypoxia-ischemia%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2007%26volume%3D204%26spage%3D1741%26epage%3D1748%26doi%3D10.1084%2Fjem.20070193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ekert, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaux, D. L.</span></span> <span> </span><span class="NLM_article-title">The mitochondrial death squad: hardened killers or innocent bystanders?</span>. <i>Curr. Opin. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">626</span>– <span class="NLM_lpage">630</span>, <span class="refDoi"> DOI: 10.1016/j.ceb.2005.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2Fj.ceb.2005.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=16219456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFyksL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2005&pages=626-630&author=P.+G.+Ekertauthor=D.+L.+Vaux&title=The+mitochondrial+death+squad%3A+hardened+killers+or+innocent+bystanders%3F&doi=10.1016%2Fj.ceb.2005.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The mitochondrial death squad: hardened killers or innocent bystanders?</span></div><div class="casAuthors">Ekert, Paul G.; Vaux, David L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">626-630</span>CODEN:
                <span class="NLM_cas:coden">COCBE3</span>;
        ISSN:<span class="NLM_cas:issn">0955-0674</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Since the discovery that formation of the apoptosome in mammalian cells is triggered by cytochrome c released from the mitochondria, many other mitochondrial proteins have been suspected to be part of a conspiracy to cause cell death.  Proteins AIF, EndoG, ANT, cyclophilin D, Bit1, p53AIP, GRIM-19, DAP3, Nur77/TR3/NGFB-1, HtrA2/Omi and Smac/Diablo have all been convicted as killers, but new genetic technol. is raising questions about their guilt.  Gene knockout expts. suggest that many were wrongly convicted on circumstantial evidence, and just happened to be in the wrong place at the wrong time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6t5fPD8wACbVg90H21EOLACvtfcHk0liRq6YKczYbDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFyksL%252FP&md5=5a574244aa43acdbba38ab88dd9b55db</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.ceb.2005.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ceb.2005.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DEkert%26aufirst%3DP.%2BG.%26aulast%3DVaux%26aufirst%3DD.%2BL.%26atitle%3DThe%2520mitochondrial%2520death%2520squad%253A%2520hardened%2520killers%2520or%2520innocent%2520bystanders%253F%26jtitle%3DCurr.%2520Opin.%2520Cell%2520Biol.%26date%3D2005%26volume%3D17%26spage%3D626%26epage%3D630%26doi%3D10.1016%2Fj.ceb.2005.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Modjtahedi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madeo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">Apoptosis-inducing factor: vital and lethal</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1016/j.tcb.2006.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2Fj.tcb.2006.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=16621561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD28XkvVOrsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=264-272&author=N.+Modjtahediauthor=F.+Giordanettoauthor=F.+Madeoauthor=G.+Kroemer&title=Apoptosis-inducing+factor%3A+vital+and+lethal&doi=10.1016%2Fj.tcb.2006.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Apoptosis-inducing factor: vital and lethal</span></div><div class="casAuthors">Modjtahedi, Nazanine; Giordanetto, Fabrizio; Madeo, Frank; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">264-272</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Apoptosis-inducing factor (AIF) is a NADH oxidase with a local redox function that is essential for optimal oxidative phosphorylation and for an efficient anti-oxidant defense.  The absence of AIF can cause neurodegeneration, skeleton muscle atrophy and dilated cardiomyopathy.  In many models of apoptosis, AIF translocates to the nucleus, where it induces chromatin condensation and DNA degrdn.  The nuclear localization of AIF can be inhibited by blocking upstream signals of apoptosis.  The contribution of AIF to cell death depends on the cell type and apoptotic insult and is only seen when caspases are inhibited or not activated.  It is unknown to what extent and through which mechanisms AIF contributes to the induction of cell death.  Here, we discuss recent progress in the quest to understand the contribution of AIF to life and death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQv5SK7lySk7Vg90H21EOLACvtfcHk0liRq6YKczYbDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkvVOrsr4%253D&md5=0af7dd5ce7279497c13262fd4af1c46a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2006.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2006.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DModjtahedi%26aufirst%3DN.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DMadeo%26aufirst%3DF.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DApoptosis-inducing%2520factor%253A%2520vital%2520and%2520lethal%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2006%26volume%3D16%26spage%3D264%26epage%3D272%26doi%3D10.1016%2Fj.tcb.2006.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Candé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecconi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dessen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">Apoptosis-inducing factor (AIF): key to the conserved caspase-independent pathways of cell death?</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">4727</span>– <span class="NLM_lpage">4734</span>, <span class="refDoi"> DOI: 10.1242/jcs.00210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1242%2Fjcs.00210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12432061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3sXis1Kkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2002&pages=4727-4734&author=C.+Cand%C3%A9author=F.+Cecconiauthor=P.+Dessenauthor=G.+Kroemer&title=Apoptosis-inducing+factor+%28AIF%29%3A+key+to+the+conserved+caspase-independent+pathways+of+cell+death%3F&doi=10.1242%2Fjcs.00210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Apoptosis-inducing factor (AIF): key to the conserved caspase-independent pathways of cell death?</span></div><div class="casAuthors">Cande, Celine; Cecconi, Francesco; Dessen, Philippe; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4727-4734</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review.  Numerous pro-apoptotic signal transducing mols. act on mitochondria and provoke the permeabilization of the outer mitochondrial membrane, thereby triggering the release of potentially toxic mitochondrial proteins.  One of these proteins, apoptosis-inducing factor (AIF), is a phylogenetically old flavoprotein which, in healthy cells, is confined to the mitochondrial intermembrane space.  Upon lethal signaling, AIF translocates, via the cytosol, to the nucleus where it binds to DNA and provokes caspase-independent chromatin condensation.  The crystal structures of both human and mouse AIF have been detd., and the fine mechanisms accounting for its oxidoreductase activity and its electrostatic interaction with double-stranded DNA have been elucidated.  Importantly, the apoptogenic and oxidoreductase functions of AIF can be dissocd.  Thus, mutations that abolish the AIF-DNA interaction suppress AIF-induced chromatin condensation, yet have no effect on the NADH oxidase activity.  Recent studies suggest AIF to be a major factor detg. caspase-independent neuronal death, emphasizing the central role of mitochondria in the control of physiol. and pathol. cell demise.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO8zyBaGTMJrVg90H21EOLACvtfcHk0lh9QSkzr5JLVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXis1Kkuw%253D%253D&md5=71dba7dd46a1d90ce9c94b5899b47ea2</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1242%2Fjcs.00210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.00210%26sid%3Dliteratum%253Aachs%26aulast%3DCand%25C3%25A9%26aufirst%3DC.%26aulast%3DCecconi%26aufirst%3DF.%26aulast%3DDessen%26aufirst%3DP.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DApoptosis-inducing%2520factor%2520%2528AIF%2529%253A%2520key%2520to%2520the%2520conserved%2520caspase-independent%2520pathways%2520of%2520cell%2520death%253F%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2002%26volume%3D115%26spage%3D4727%26epage%3D4734%26doi%3D10.1242%2Fjcs.00210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cregan, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacLaurin, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callaghan, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecconi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slack, R. S.</span></span> <span> </span><span class="NLM_article-title">Apoptosis-inducing factor is involved in the regulation of caspase-independent neuronal cell death</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>158</i></span>,  <span class="NLM_fpage">507</span>– <span class="NLM_lpage">517</span>, <span class="refDoi"> DOI: 10.1083/jcb.200202130</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1083%2Fjcb.200202130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12147675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD38XmtVeks7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2002&pages=507-517&author=S.+P.+Creganauthor=A.+Fortinauthor=J.+G.+MacLaurinauthor=S.+M.+Callaghanauthor=F.+Cecconiauthor=S.-W.+Yuauthor=T.+M.+Dawsonauthor=V.+L.+Dawsonauthor=D.+S.+Parkauthor=G.+Kroemerauthor=R.+S.+Slack&title=Apoptosis-inducing+factor+is+involved+in+the+regulation+of+caspase-independent+neuronal+cell+death&doi=10.1083%2Fjcb.200202130"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Apoptosis-inducing factor is involved in the regulation of caspase-independent neuronal cell death</span></div><div class="casAuthors">Cregan, Sean P.; Fortin, Andre; MacLaurin, Jason G.; Callaghan, Steven M.; Cecconi, Francesco; Yu, Seong-Woon; Dawson, Ted M.; Dawson, Valina L.; Park, David S.; Kroemer, Guido; Slack, Ruth S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">507-517</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9525</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Caspase-independent death mechanisms have been shown to execute apoptosis in many types of neuronal injury.  P53 has been identified as a key regulator of neuronal cell death after acute injury such as DNA damage, ischemia, and excitotoxicity.  Here, we demonstrate that p53 can induce neuronal cell death via a caspase-mediated process activated by apoptotic activating factor-1 (Apaf1) and via a delayed onset caspase-independent mechanism.  In contrast to wild-type cells, Apaf1-deficient neurons exhibit delayed DNA fragmentation and only peripheral chromatin condensation.  More importantly, we demonstrate that apoptosis-inducing factor (AIF) is an important factor involved in the regulation of this caspase-independent neuronal cell death.  Immunofluorescence studies demonstrate that AIF is released from the mitochondria by a mechanism distinct from that of cytochrome-c in neurons undergoing p53-mediated cell death.  The Bcl-2 family regulates this release of AIF and subsequent caspase-independent cell death.  In addn., we show that enforced expression of AIF can induce neuronal cell death in a Bax- and caspase-independent manner.  Microinjection of neutralizing antibodies against AIF significantly decreased injury-induced neuronal cell death in Apaf1-deficient neurons, indicating its importance in caspase-independent apoptosis.  Taken together, our results suggest that AIF may be an important therapeutic target for the treatment of neuronal injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPDGvLxICd4bVg90H21EOLACvtfcHk0lh9QSkzr5JLVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmtVeks7Y%253D&md5=35c504769247a1a0f591925e3dc78f5b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200202130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200202130%26sid%3Dliteratum%253Aachs%26aulast%3DCregan%26aufirst%3DS.%2BP.%26aulast%3DFortin%26aufirst%3DA.%26aulast%3DMacLaurin%26aufirst%3DJ.%2BG.%26aulast%3DCallaghan%26aufirst%3DS.%2BM.%26aulast%3DCecconi%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DS.-W.%26aulast%3DDawson%26aufirst%3DT.%2BM.%26aulast%3DDawson%26aufirst%3DV.%2BL.%26aulast%3DPark%26aufirst%3DD.%2BS.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DSlack%26aufirst%3DR.%2BS.%26atitle%3DApoptosis-inducing%2520factor%2520is%2520involved%2520in%2520the%2520regulation%2520of%2520caspase-independent%2520neuronal%2520cell%2520death%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2002%26volume%3D158%26spage%3D507%26epage%3D517%26doi%3D10.1083%2Fjcb.200202130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Susin, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzo, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamzami, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snow, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangion, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacotot, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costantini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loeffler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larochette, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodlett, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aebersold, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siderovski, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penninger, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">Molecular characterization of mitochondrial apoptosis-inducing factor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>397</i></span>,  <span class="NLM_fpage">441</span>– <span class="NLM_lpage">446</span>, <span class="refDoi"> DOI: 10.1038/17135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2F17135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=9989411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK1MXhtFOhu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=397&publication_year=1999&pages=441-446&author=S.+A.+Susinauthor=H.+K.+Lorenzoauthor=N.+Zamzamiauthor=I.+Marzoauthor=B.+E.+Snowauthor=G.+M.+Brothersauthor=J.+Mangionauthor=E.+Jacototauthor=P.+Costantiniauthor=M.+Loefflerauthor=N.+Larochetteauthor=D.+R.+Goodlettauthor=R.+Aebersoldauthor=D.+P.+Siderovskiauthor=J.+M.+Penningerauthor=G.+Kroemer&title=Molecular+characterization+of+mitochondrial+apoptosis-inducing+factor&doi=10.1038%2F17135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular characterization of mitochondrial apoptosis-inducing factor</span></div><div class="casAuthors">Susin, Santos A.; Lorenzo, Hans K.; Zamzami, Naoufal; Marzo, Isabel; Snow, Bryan E.; Brothers, Greg M.; Mangion, Joan; Jacotot, Etienne; Costantini, Paola; Loeffler, Markus; Larochette, Nathanael; Goodlett, David R.; Aebersold, Ruedi; Siderovski, David P.; Penninger, Josef M.; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">397</span>
        (<span class="NLM_cas:issue">6718</span>),
    <span class="NLM_cas:pages">441-446</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Mitochondria play a key part in the regulation of apoptosis (cell death).  Their intermembrane space contains several proteins that are liberated through the outer membrane in order to participate in the degrdn. phase of apoptosis.  Here we report the identification and cloning of an apoptosis-inducing factor, AIF5, which is sufficient to induce apoptosis of isolated nuclei.  AIF is a flavoprotein of relative mol. mass 57,000 which shares homol. with the bacterial oxidoreductases; it is normally confined to mitochondria but translocates to the nucleus when apoptosis is induced.  Recombinant AIF causes chromatin condensation in isolated nuclei and large-scale fragmentation of DNA.  It induces purified mitochondria to release the apoptogenic proteins cytochrome c and caspase-9.  Microinjection of AIF into the cytoplasm of intact cells induces condensation of chromatin, dissipation of the mitochondrial transmembrane potential, and exposure of phosphatidylserine in the plasma membrane.  None of these effects is prevented by the wide-ranging caspase inhibitor known as Z-VAD.fmk.  Overexpression of Bcl-2, which controls the opening of mitochondrial permeability transition pores, prevents the release of AIF from the mitochondrion but does not affect its apoptogenic activity.  These results indicate that AIF is a mitochondrial effector of apoptotic cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8KHCoQLJgaLVg90H21EOLACvtfcHk0lh9QSkzr5JLVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhtFOhu78%253D&md5=e02254e6bf0898bc9218950620b08196</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2F17135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F17135%26sid%3Dliteratum%253Aachs%26aulast%3DSusin%26aufirst%3DS.%2BA.%26aulast%3DLorenzo%26aufirst%3DH.%2BK.%26aulast%3DZamzami%26aufirst%3DN.%26aulast%3DMarzo%26aufirst%3DI.%26aulast%3DSnow%26aufirst%3DB.%2BE.%26aulast%3DBrothers%26aufirst%3DG.%2BM.%26aulast%3DMangion%26aufirst%3DJ.%26aulast%3DJacotot%26aufirst%3DE.%26aulast%3DCostantini%26aufirst%3DP.%26aulast%3DLoeffler%26aufirst%3DM.%26aulast%3DLarochette%26aufirst%3DN.%26aulast%3DGoodlett%26aufirst%3DD.%2BR.%26aulast%3DAebersold%26aufirst%3DR.%26aulast%3DSiderovski%26aufirst%3DD.%2BP.%26aulast%3DPenninger%26aufirst%3DJ.%2BM.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DMolecular%2520characterization%2520of%2520mitochondrial%2520apoptosis-inducing%2520factor%26jtitle%3DNature%26date%3D1999%26volume%3D397%26spage%3D441%26epage%3D446%26doi%3D10.1038%2F17135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S. J.</span>; <span class="NLM_string-name">Dawson, T. M.</span>; <span class="NLM_string-name">Dawson, V. L.</span></span> <span> </span><span class="NLM_article-title">PARP and the Release of Apoptosis-Inducing Factor from Mitochondria</span>.  <i>Madame Curie Bioscience Database [Internet]</i>; <span class="NLM_publisher-name">Landes Bioscience</span>: <span class="NLM_publisher-loc">Austin (TX)</span>, <span class="NLM_year">2000–2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000%E2%80%932013&author=S.+J.+Hong&author=T.+M.+Dawson&author=V.+L.+Dawson&title=Madame+Curie+Bioscience+Database+%5BInternet%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DS.%2BJ.%26atitle%3DPARP%2520and%2520the%2520Release%2520of%2520Apoptosis-Inducing%2520Factor%2520from%2520Mitochondria%26btitle%3DMadame%2520Curie%2520Bioscience%2520Database%2520%255BInternet%255D%26pub%3DLandes%2520Bioscience%26date%3D2000%25E2%2580%25932013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S. J.</span></span> <span> </span><span class="NLM_article-title">AIF translocates to the nucleus in the spinal motor neurons in a mouse model of ALS</span>. <i>Neurosci. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>406</i></span>,  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1016/j.neulet.2006.07.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2Fj.neulet.2006.07.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=16916579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsVOms70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=406&publication_year=2006&pages=205-210&author=Y.+K.+Ohauthor=K.+S.+Shinauthor=S.+J.+Kang&title=AIF+translocates+to+the+nucleus+in+the+spinal+motor+neurons+in+a+mouse+model+of+ALS&doi=10.1016%2Fj.neulet.2006.07.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">AIF translocates to the nucleus in the spinal motor neurons in a mouse model of ALS</span></div><div class="casAuthors">Oh, Yeo Kyoung; Shin, Ki Soon; Kang, Shin Jung</div><div class="citationInfo"><span class="NLM_cas:title">Neuroscience Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">205-210</span>CODEN:
                <span class="NLM_cas:coden">NELED5</span>;
        ISSN:<span class="NLM_cas:issn">0304-3940</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Amyotrophic lateral sclerosis (ALS) is a fatal neurodegenerative disease characterized by selective loss of motor neurons in the brain stem and the spinal cords.  One of the causes for the familial ALS has been attributed to the mutations in copper-zinc superoxide dismutase (SOD1).  Although the toxic function of the mutant enzyme has not been fully understood, the final cell death pathway has been suggested as caspase-dependent.  In the present study, we present evidence that the activation of apoptosis inducing factor (AIF) may play a role to induce motor neuron death during ALS pathogenesis.  In the spinal cord of SOD1 G93A transgenic mice, expression of AIF was detected in the motor neurons and astrocytes.  The level of AIF expression increased as the disease progressed.  In the symptomatic SOD1 G93A transgenic mice, AIF released from the mitochondria and translocated into the nucleus in the motor neurons as evidenced by confocal microscopy and biochem. anal.  These results suggest that AIF may play a role to induce motor neuron death in a mouse model of ALS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxEibnYSSM47Vg90H21EOLACvtfcHk0lgtG6rm-py1qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsVOms70%253D&md5=e4ad1a7a13b17826df70ba0bde044915</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.neulet.2006.07.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neulet.2006.07.044%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DY.%2BK.%26aulast%3DShin%26aufirst%3DK.%2BS.%26aulast%3DKang%26aufirst%3DS.%2BJ.%26atitle%3DAIF%2520translocates%2520to%2520the%2520nucleus%2520in%2520the%2520spinal%2520motor%2520neurons%2520in%2520a%2520mouse%2520model%2520of%2520ALS%26jtitle%3DNeurosci.%2520Lett.%26date%3D2006%26volume%3D406%26spage%3D205%26epage%3D210%26doi%3D10.1016%2Fj.neulet.2006.07.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Plesnila, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culmsee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gröger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moskowitz, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomgren, K.</span></span> <span> </span><span class="NLM_article-title">Nuclear translocation of apoptosis-inducing factor after focal cerebral ischemia</span>. <i>J. Cereb. Blood Flow Metab. Suppl.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">458</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1097/00004647-200404000-00011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1097%2F00004647-200404000-00011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=15087715" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=458-466&author=N.+Plesnilaauthor=C.+Zhuauthor=C.+Culmseeauthor=M.+Gr%C3%B6gerauthor=M.+A.+Moskowitzauthor=K.+Blomgren&title=Nuclear+translocation+of+apoptosis-inducing+factor+after+focal+cerebral+ischemia&doi=10.1097%2F00004647-200404000-00011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1097%2F00004647-200404000-00011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00004647-200404000-00011%26sid%3Dliteratum%253Aachs%26aulast%3DPlesnila%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DCulmsee%26aufirst%3DC.%26aulast%3DGr%25C3%25B6ger%26aufirst%3DM.%26aulast%3DMoskowitz%26aufirst%3DM.%2BA.%26aulast%3DBlomgren%26aufirst%3DK.%26atitle%3DNuclear%2520translocation%2520of%2520apoptosis-inducing%2520factor%2520after%2520focal%2520cerebral%2520ischemia%26jtitle%3DJ.%2520Cereb.%2520Blood%2520Flow%2520Metab.%2520Suppl.%26date%3D2004%26volume%3D24%26spage%3D458%26epage%3D466%26doi%3D10.1097%2F00004647-200404000-00011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poitras, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coombs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federoff, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, V. L.</span></span> <span> </span><span class="NLM_article-title">Mediation of poly(ADP-ribose) polymerase-1-dependent cell death by apoptosis-inducing factor</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>297</i></span>,  <span class="NLM_fpage">259</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1126/science.1072221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1126%2Fscience.1072221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12114629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsVCntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=297&publication_year=2002&pages=259-263&author=S.-W.+Yuauthor=H.+Wangauthor=M.+F.+Poitrasauthor=C.+Coombsauthor=W.+J.+Bowersauthor=H.+J.+Federoffauthor=G.+G.+Poirierauthor=T.+M.+Dawsonauthor=V.+L.+Dawson&title=Mediation+of+poly%28ADP-ribose%29+polymerase-1-dependent+cell+death+by+apoptosis-inducing+factor&doi=10.1126%2Fscience.1072221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Mediation of poly(ADP-ribose) polymerase-1-dependent cell death by apoptosis-inducing factor</span></div><div class="casAuthors">Yu, Seong-Woon; Wang, Hongmin; Poitras, Marc F.; Coombs, Carmen; Bowers, William J.; Federoff, Howard J.; Poirier, Guy G.; Dawson, Ted M.; Dawson, Valina L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">297</span>
        (<span class="NLM_cas:issue">5579</span>),
    <span class="NLM_cas:pages">259-263</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase-1 (PARP-1) protects the genome by functioning in the DNA damage surveillance network.  PARP- 1 is also a mediator of cell death after ischemia-reperfusion injury, glutamate excitotoxicity, and various inflammatory processes.  We show that PARP-1 activation is required for translocation of apoptosis-inducing factor (AIF) from the mitochondria to the nucleus and that AIF is necessary for PARP-1-dependent cell death.  N-methyl-N'-nitro-N-nitrosoguanidine, H2O2, and N-methyl-D-aspartate induce AIF translocation and cell death, which is prevented by PARP inhibitors or genetic knockout of PARP-1, but is caspase independent.  Microinjection of an antibody to AIF protects against PARP-1-dependent cytotoxicity.  These data support a model in which PARP-1 activation signals AIF release from mitochondria, resulting in a caspase-independent pathway of programmed cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRtfSpfY4287Vg90H21EOLACvtfcHk0lgtG6rm-py1qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsVCntr0%253D&md5=5e5b38d90e9bfbcbc149cdf43dad0d4e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1126%2Fscience.1072221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1072221%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DS.-W.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DPoitras%26aufirst%3DM.%2BF.%26aulast%3DCoombs%26aufirst%3DC.%26aulast%3DBowers%26aufirst%3DW.%2BJ.%26aulast%3DFederoff%26aufirst%3DH.%2BJ.%26aulast%3DPoirier%26aufirst%3DG.%2BG.%26aulast%3DDawson%26aufirst%3DT.%2BM.%26aulast%3DDawson%26aufirst%3DV.%2BL.%26atitle%3DMediation%2520of%2520poly%2528ADP-ribose%2529%2520polymerase-1-dependent%2520cell%2520death%2520by%2520apoptosis-inducing%2520factor%26jtitle%3DScience%26date%3D2002%26volume%3D297%26spage%3D259%26epage%3D263%26doi%3D10.1126%2Fscience.1072221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kochanek, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draviam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nathaniel, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, R. S. B.</span></span> <span> </span><span class="NLM_article-title">Intranuclear localization of apoptosis-inducing factor (AIF) and large scale DNA fragmentation after traumatic brain injury in rats and in neuronal cultures exposed to peroxynitrite</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.1046/j.1471-4159.2002.00975.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1046%2Fj.1471-4159.2002.00975.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12091479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsV2jtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2002&pages=181-191&author=X.+Zhangauthor=J.+Chenauthor=S.+H.+Grahamauthor=L.+Duauthor=P.+M.+Kochanekauthor=R.+Draviamauthor=F.+Guoauthor=P.+D.+Nathanielauthor=C.+Szaboauthor=S.+C.+Watkinsauthor=R.+S.+B.+Clark&title=Intranuclear+localization+of+apoptosis-inducing+factor+%28AIF%29+and+large+scale+DNA+fragmentation+after+traumatic+brain+injury+in+rats+and+in+neuronal+cultures+exposed+to+peroxynitrite&doi=10.1046%2Fj.1471-4159.2002.00975.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Intranuclear localization of apoptosis-inducing factor (AIF) and large scale DNA fragmentation after traumatic brain injury in rats and in neuronal cultures exposed to peroxynitrite</span></div><div class="casAuthors">Zhang, Xiaopeng; Chen, Jun; Graham, Steven H.; Du, Lina; Kochanek, Patrick M.; Draviam, Romesh; Guo, Fengli; Nathaniel, Paula D.; Szabo, Csaba; Watkins, Simon C.; Clark, Robert S. B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">181-191</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science Ltd.</span>)
        </div><div class="casAbstract">Programmed cell death occurs after ischemic, excitotoxic, and traumatic brain injury (TBI).  Recently, a caspase-independent pathway involving intranuclear translocation of mitochondrial apoptosis-inducing factor (AIF) has been reported in vitro; but whether this occurs after acute brain injury was unknown.  To address this question adult rats were sacrificed at various times after TBI.  Western blot anal. on subcellular protein fractions demonstrated intranuclear localization of AIF in ipsilateral cortex and hippocampus at 2-72 h.  Immunocytochem. anal. showed AIF labeling in neuronal nuclei with DNA fragmentation in the ipsilateral cortex and hippocampus.  Immunoelectronmicroscopy verified intranuclear localization of AIF in hippocampal neurons after TBI, primarily in regions of euchromatin.  Large-scale DNA fragmentation (∼50 kbp), a signature event in AIF-mediated cell death, was detected in ipsilateral cortex and hippocampi by 6 h.  Neuron-enriched cultures exposed to peroxynitrite also demonstrated intranuclear AIF and large-scale DNA fragmentation concurrent with impaired mitochondrial respiration and cell death, events that are inhibited by treatment with a peroxynitrite decompn. catalyst.  Intranuclear localization of AIF and large-scale DNA fragmentation occurs after TBI and in neurons under conditions of oxidative/nitrosative stress, providing the first evidence of this alternative mechanism by which programmed cell death may proceed in neurons after brain injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLmABPAgQYZbVg90H21EOLACvtfcHk0lgoTeNE1rOQFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsV2jtrs%253D&md5=0a448a90b84fd3fa3ecbf5602f1798a9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1046%2Fj.1471-4159.2002.00975.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1471-4159.2002.00975.x%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DGraham%26aufirst%3DS.%2BH.%26aulast%3DDu%26aufirst%3DL.%26aulast%3DKochanek%26aufirst%3DP.%2BM.%26aulast%3DDraviam%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DF.%26aulast%3DNathaniel%26aufirst%3DP.%2BD.%26aulast%3DSzabo%26aufirst%3DC.%26aulast%3DWatkins%26aufirst%3DS.%2BC.%26aulast%3DClark%26aufirst%3DR.%2BS.%2BB.%26atitle%3DIntranuclear%2520localization%2520of%2520apoptosis-inducing%2520factor%2520%2528AIF%2529%2520and%2520large%2520scale%2520DNA%2520fragmentation%2520after%2520traumatic%2520brain%2520injury%2520in%2520rats%2520and%2520in%2520neuronal%2520cultures%2520exposed%2520to%2520peroxynitrite%26jtitle%3DJ.%2520Neurochem.%26date%3D2002%26volume%3D82%26spage%3D181%26epage%3D191%26doi%3D10.1046%2Fj.1471-4159.2002.00975.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sabirzhanov, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoica, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanscom, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piao, C.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faden, A. I.</span></span> <span> </span><span class="NLM_article-title">Over-expression of HSP70 attenuates caspase-dependent and caspase-independent pathways and inhibits neuronal apoptosis</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">554</span>, <span class="refDoi"> DOI: 10.1111/j.1471-4159.2012.07927.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1111%2Fj.1471-4159.2012.07927.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=22909049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFSnsb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2012&pages=542-554&author=B.+Sabirzhanovauthor=B.+A.+Stoicaauthor=M.+Hanscomauthor=C.-S.+Piaoauthor=A.+I.+Faden&title=Over-expression+of+HSP70+attenuates+caspase-dependent+and+caspase-independent+pathways+and+inhibits+neuronal+apoptosis&doi=10.1111%2Fj.1471-4159.2012.07927.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Over-expression of HSP70 attenuates caspase-dependent and caspase-independent pathways and inhibits neuronal apoptosis</span></div><div class="casAuthors">Sabirzhanov, Boris; Stoica, Bogdan A.; Hanscom, Marie; Piao, Chun-Shu; Faden, Alan I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">3&4</span>),
    <span class="NLM_cas:pages">542-554</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">HSP70 is a member of the family of heat-shock proteins that are known to be up-regulated in neurons following injury and/or stress.  HSP70 over-expression has been linked to neuroprotection in multiple models, including neurodegenerative disorders.  In contrast, less is known about the neuroprotective effects of HSP70 in neuronal apoptosis and with regard to modulation of programmed cell death (PCD) mechanisms in neurons.  The authors examd. the effects of HSP70 over-expression by transfection with HSP70-expression plasmids in primary cortical neurons and the SH-SY5Y neuronal cell line using four independent models of apoptosis: etoposide, staurosporine, C2-ceramide, and β-Amyloid.  In these apoptotic models, neurons transfected with the HSP70 construct showed significantly reduced induction of nuclear apoptotic markers and/or cell death.  Furthermore, the authors demonstrated that HSP70 binds and potentially inactivates Apoptotic protease-activating factor 1, as well as apoptosis-inducing factor, key mols. involved in development of caspase-dependent and caspase-independent PCD, resp.  Markers of caspase-dependent PCD, including active caspase-3, caspase-9, and cleaved PARP were attenuated in neurons over-expressing HSP70.  These data indicate that HSP70 protects against neuronal apoptosis and suggest that these effects reflect, at least in part, to inhibition of both caspase-dependent and caspase-independent PCD pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4WvC0W71ZZbVg90H21EOLACvtfcHk0lgoTeNE1rOQFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFSnsb%252FF&md5=437751f997c5e44d935b615b6920f48c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1111%2Fj.1471-4159.2012.07927.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1471-4159.2012.07927.x%26sid%3Dliteratum%253Aachs%26aulast%3DSabirzhanov%26aufirst%3DB.%26aulast%3DStoica%26aufirst%3DB.%2BA.%26aulast%3DHanscom%26aufirst%3DM.%26aulast%3DPiao%26aufirst%3DC.-S.%26aulast%3DFaden%26aufirst%3DA.%2BI.%26atitle%3DOver-expression%2520of%2520HSP70%2520attenuates%2520caspase-dependent%2520and%2520caspase-independent%2520pathways%2520and%2520inhibits%2520neuronal%2520apoptosis%26jtitle%3DJ.%2520Neurochem.%26date%3D2012%26volume%3D123%26spage%3D542%26epage%3D554%26doi%3D10.1111%2Fj.1471-4159.2012.07927.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Öxler, E.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culmsee, C.</span></span> <span> </span><span class="NLM_article-title">AIF depletion provides neuroprotection through a preconditioning effect</span>. <i>Apoptosis</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1027</span>– <span class="NLM_lpage">1038</span>, <span class="refDoi"> DOI: 10.1007/s10495-012-0748-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2Fs10495-012-0748-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=22865232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A280%3ADC%252BC38fls1Gktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=1027-1038&author=E.-M.+%C3%96xlerauthor=A.+Dolgaauthor=C.+Culmsee&title=AIF+depletion+provides+neuroprotection+through+a+preconditioning+effect&doi=10.1007%2Fs10495-012-0748-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">AIF depletion provides neuroprotection through a preconditioning effect</span></div><div class="casAuthors">Oxler Eva-Maria; Dolga Amalia; Culmsee C</div><div class="citationInfo"><span class="NLM_cas:title">Apoptosis : an international journal on programmed cell death</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1027-38</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Previous studies established a major role for apoptosis inducing factor (AIF) in neuronal cell death after acute brain injury.  For example, AIF translocation from mitochondria to the nucleus determined delayed neuronal death, whereas reduced AIF expression provided neuroprotective effects in models of cerebral ischemia or brain trauma.  The question remains, however, why reduced AIF levels are sufficient to mediate neuroprotection, since only very little AIF translocation to the nucleus is required for induction of cell death.  Thus, the present study addresses the question, whether AIF gene silencing affects intrinsic death pathways upstream of nuclear translocation at the level of the mitochondria.  Using MTT assays and real-time cell impedance measurements we confirmed the protective effect of AIF siRNA against glutamate toxicity in immortalized mouse hippocampal HT-22 neurons.  Further, AIF siRNA prevented glutamate-induced mitochondrial fragmentation and loss of mitochondrial membrane potential.  The protection of mitochondrial integrity was associated with preserved ATP levels, attenuated increases in lipid peroxidation and reduced complex I expression levels.  Notably, low concentrations of the complex I inhibitor rotenone (20 nM), provided similar protective effects against glutamate toxicity at the mitochondrial level.  These results expose a preconditioning effect as a mechanism for neuroprotection mediated by AIF depletion.  In particular, they point out an association between mitochondrial complex I and AIF, which regulate each other's stability in mitochondria.  Overall, these findings postulate that AIF depletion mediates a preconditioning effect protecting neuronal cells from subsequent glutamate toxicity through reduced levels of complex I protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTbRQBo_3oNhtbHMBt0mMIXfW6udTcc2eanx6rwvceX6Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38fls1Gktw%253D%253D&md5=0c90a3506b4c8b0a2192c0aee8501808</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs10495-012-0748-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10495-012-0748-8%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2596xler%26aufirst%3DE.-M.%26aulast%3DDolga%26aufirst%3DA.%26aulast%3DCulmsee%26aufirst%3DC.%26atitle%3DAIF%2520depletion%2520provides%2520neuroprotection%2520through%2520a%2520preconditioning%2520effect%26jtitle%3DApoptosis%26date%3D2012%26volume%3D17%26spage%3D1027%26epage%3D1038%26doi%3D10.1007%2Fs10495-012-0748-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tretter, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sipos, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam-Vizi, V.</span></span> <span> </span><span class="NLM_article-title">Initiation of neuronal damage by complex I deficiency and oxidative stress in Parkinson’s disease</span>. <i>Neurochem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">569</span>– <span class="NLM_lpage">577</span>, <span class="refDoi"> DOI: 10.1023/b:nere.0000014827.94562.4b</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1023%2FB%3ANERE.0000014827.94562.4b" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=15038604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpslWitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2004&pages=569-577&author=L.+Tretterauthor=I.+Siposauthor=V.+Adam-Vizi&title=Initiation+of+neuronal+damage+by+complex+I+deficiency+and+oxidative+stress+in+Parkinson%E2%80%99s+disease&doi=10.1023%2Fb%3Anere.0000014827.94562.4b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Initiation of neuronal damage by complex I deficiency and oxidative stress in Parkinson's Disease</span></div><div class="casAuthors">Tretter, Laszlo; Sipos, Ildiko; Adam-Vizi, Vera</div><div class="citationInfo"><span class="NLM_cas:title">Neurochemical Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">569-577</span>CODEN:
                <span class="NLM_cas:coden">NEREDZ</span>;
        ISSN:<span class="NLM_cas:issn">0364-3190</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic/Plenum Publishers</span>)
        </div><div class="casAbstract">A review.  Oxidative stress and partial deficiencies of mitochondrial complex I appear to be key factors in the pathogenesis of Parkinson's disease.  They are interconnected; complex I inhibition results in an enhanced prodn. of reactive oxygen species (ROS), which in turn will inhibit complex I.  Partial inhibition of complex I in nerve terminals is sufficient for in situ mitochondria to generate more ROS.  H2O2 plays a major role in inhibiting complex I as well as a key metabolic enzyme, α-ketoglutarate dehydrogenase.  The vicious cycle resulting from partial inhibition of complex I and/or an inherently higher ROS prodn. in dopaminergic neurons leads over time to excessive oxidative stress and ATP deficit that eventually will result in cell death in the nigro-striatal pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnUGJVRLCHLVg90H21EOLACvtfcHk0lg8Tv45NjBkaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpslWitA%253D%253D&md5=582748bd96a6ffa6edcc55c8bc99c576</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1023%2FB%3ANERE.0000014827.94562.4b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FB%253ANERE.0000014827.94562.4b%26sid%3Dliteratum%253Aachs%26aulast%3DTretter%26aufirst%3DL.%26aulast%3DSipos%26aufirst%3DI.%26aulast%3DAdam-Vizi%26aufirst%3DV.%26atitle%3DInitiation%2520of%2520neuronal%2520damage%2520by%2520complex%2520I%2520deficiency%2520and%2520oxidative%2520stress%2520in%2520Parkinson%25E2%2580%2599s%2520disease%26jtitle%3DNeurochem.%2520Res.%26date%3D2004%26volume%3D29%26spage%3D569%26epage%3D577%26doi%3D10.1023%2Fb%3Anere.0000014827.94562.4b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallin, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Candé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomgren, K.</span></span> <span> </span><span class="NLM_article-title">Involvement of apoptosis-inducing factor in neuronal death after hypoxia-ischemia in the neonatal rat brain</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1046/j.1471-4159.2003.01832.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1046%2Fj.1471-4159.2003.01832.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12871572" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlslGhsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2003&pages=306-317&author=C.+Zhuauthor=L.+Qiuauthor=X.+Wangauthor=U.+Hallinauthor=C.+Cand%C3%A9author=G.+Kroemerauthor=H.+Hagbergauthor=K.+Blomgren&title=Involvement+of+apoptosis-inducing+factor+in+neuronal+death+after+hypoxia-ischemia+in+the+neonatal+rat+brain&doi=10.1046%2Fj.1471-4159.2003.01832.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Involvement of apoptosis-inducing factor in neuronal death after hypoxia-ischemia in the neonatal rat brain</span></div><div class="casAuthors">Zhu, Changlian; Qiu, Lin; Wang, Xiaoyang; Hallin, Ulrika; Cande, Celine; Kroemer, Guido; Hagberg, Henrik; Blomgren, Klas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">306-317</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Apoptosis-inducing factor (AIF) triggers apoptosis in a caspase-independent manner.  Here the authors report for the first time involvement of AIF in neuronal death induced by cerebral ischemia.  Unilateral cerebral hypoxia-ischemia (HI) was induced in 7-day-old rats by ligation of the left carotid artery and hypoxia (7.7% O2) for 55 min.  AIF release from mitochondria and AIF translocation to nuclei was detected immediately after HI, and only in damaged areas, as judged by the concurrent loss of MAP-2.  AIF release was detected earlier than that of cytochrome c.  Cells with AIF-pos. nuclei displayed nuclear condensation and signs of DNA damage.  The no. of AIF-pos. nuclei showed a pos. correlation with the infarct vol. 72 h post-HI, and this was not changed by treating the animals with boc-Asp-fmk (BAF), a multicaspase inhibitor.  BAF treatment reduced the activity of caspase-3, -2 and -9 (78, 73 and 33%, resp.), and prevented caspase-dependent fodrin cleavage in vivo, but did not affect AIF release from mitochondria or the frequency of pos. nuclear AIF or DNA damage 72 h post-HI, indicating that these processes occurred in a caspase-independent fashion.  In summary, AIF-mediated cell death may be an important mechanism of HI-induced neuronal loss in the immature brain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox7tE9HIjSC7Vg90H21EOLACvtfcHk0lg8Tv45NjBkaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlslGhsbg%253D&md5=6576f5b468020f839aef71179329cbf3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1046%2Fj.1471-4159.2003.01832.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1471-4159.2003.01832.x%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DQiu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHallin%26aufirst%3DU.%26aulast%3DCand%25C3%25A9%26aufirst%3DC.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DHagberg%26aufirst%3DH.%26aulast%3DBlomgren%26aufirst%3DK.%26atitle%3DInvolvement%2520of%2520apoptosis-inducing%2520factor%2520in%2520neuronal%2520death%2520after%2520hypoxia-ischemia%2520in%2520the%2520neonatal%2520rat%2520brain%26jtitle%3DJ.%2520Neurochem.%26date%3D2003%26volume%3D86%26spage%3D306%26epage%3D317%26doi%3D10.1046%2Fj.1471-4159.2003.01832.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Candé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vahsen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouranti, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spahr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luban, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrido, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penninger, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">AIF and cyclophilin A cooperate in apoptosis-associated chromatinolysis</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1514</span>– <span class="NLM_lpage">1521</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1207279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Fsj.onc.1207279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=14716299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFOht7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=1514-1521&author=C.+Cand%C3%A9author=N.+Vahsenauthor=I.+Kourantiauthor=E.+Schmittauthor=E.+Daugasauthor=C.+Spahrauthor=J.+Lubanauthor=R.+T.+Kroemerauthor=F.+Giordanettoauthor=C.+Garridoauthor=J.+M.+Penningerauthor=G.+Kroemer&title=AIF+and+cyclophilin+A+cooperate+in+apoptosis-associated+chromatinolysis&doi=10.1038%2Fsj.onc.1207279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">AIF and cyclophilin A cooperate in apoptosis-associated chromatinolysis</span></div><div class="casAuthors">Cande, Celine; Vahsen, Nicola; Kouranti, Ilektra; Schmitt, Elise; Daugas, Eric; Spahr, Chris; Luban, Jeremy; Kroemer, Romano T.; Giordanetto, Fabrizio; Garrido, Carmen; Penninger, Josef M.; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1514-1521</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cyclophilin A (CypA) was detd. to interact with apoptosis-inducing factor (AIF) by mass spectroscopy, coimmunopptn., pull-down assays, and mol. modeling.  During the initial, caspase-independent stage of chromatin condensation that accompanies apoptosis, AIF and CypA were found to coimmunolocalize in the nucleus.  Recombinant AIF and CypA proteins synergized in vitro in the degrdn. of plasmid DNA, as well as in the capacity to induce DNA loss in purified nuclei.  The apoptogenic cooperation between AIF and CypA did not rely on the CypA peptidyl-prolyl cis-trans isomerase activity.  In Cyp-expressing cells, AIF overexpression augmented apoptotic chromatinolysis.  The AIF-dependent large-scale DNA fragmentation was less pronounced in CypA knockout cells as compared to controls.  AIF mutants lacking the CypA-binding domain were inefficient apoptosis sensitizers in transfection expts.  Moreover, AIF failed to sensitize CypA knockout cells to apoptosis induction, and this defect in the AIF response was reversed by reintroduction of the CypA gene into CypA-deficient cells.  In summary, AIF and CypA collaborate in chromatinolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7ACyaVMkTobVg90H21EOLACvtfcHk0lg8Tv45NjBkaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFOht7w%253D&md5=86e306afe590b4454aacf11bb911e52e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1207279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1207279%26sid%3Dliteratum%253Aachs%26aulast%3DCand%25C3%25A9%26aufirst%3DC.%26aulast%3DVahsen%26aufirst%3DN.%26aulast%3DKouranti%26aufirst%3DI.%26aulast%3DSchmitt%26aufirst%3DE.%26aulast%3DDaugas%26aufirst%3DE.%26aulast%3DSpahr%26aufirst%3DC.%26aulast%3DLuban%26aufirst%3DJ.%26aulast%3DKroemer%26aufirst%3DR.%2BT.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DGarrido%26aufirst%3DC.%26aulast%3DPenninger%26aufirst%3DJ.%2BM.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DAIF%2520and%2520cyclophilin%2520A%2520cooperate%2520in%2520apoptosis-associated%2520chromatinolysis%26jtitle%3DOncogene%26date%3D2004%26volume%3D23%26spage%3D1514%26epage%3D1521%26doi%3D10.1038%2Fsj.onc.1207279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reuther, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scognamiglio, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolga, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plesnila, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culmsee, C.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the AIF/CypA complex protects against intrinsic death pathways induced by oxidative stress</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e993</span> <span class="refDoi"> DOI: 10.1038/cddis.2013.518</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Fcddis.2013.518" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=24434516" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&author=N.+Dotiauthor=C.+Reutherauthor=P.+L.+Scognamiglioauthor=A.+M.+Dolgaauthor=N.+Plesnilaauthor=M.+Ruvoauthor=C.+Culmsee&title=Inhibition+of+the+AIF%2FCypA+complex+protects+against+intrinsic+death+pathways+induced+by+oxidative+stress&doi=10.1038%2Fcddis.2013.518"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2013.518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2013.518%26sid%3Dliteratum%253Aachs%26aulast%3DDoti%26aufirst%3DN.%26aulast%3DReuther%26aufirst%3DC.%26aulast%3DScognamiglio%26aufirst%3DP.%2BL.%26aulast%3DDolga%26aufirst%3DA.%2BM.%26aulast%3DPlesnila%26aufirst%3DN.%26aulast%3DRuvo%26aufirst%3DM.%26aulast%3DCulmsee%26aufirst%3DC.%26atitle%3DInhibition%2520of%2520the%2520AIF%252FCypA%2520complex%2520protects%2520against%2520intrinsic%2520death%2520pathways%2520induced%2520by%2520oxidative%2520stress%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2014%26volume%3D5%26doi%3D10.1038%2Fcddis.2013.518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Artus, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boujrad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouharrour, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunelle, M.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuste, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenormand, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousselle, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namane, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">England, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzo, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Susin, S. A.</span></span> <span> </span><span class="NLM_article-title">AIF promotes chromatinolysis and caspase-independent programmed necrosis by interacting with histone H2AX</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1585</span>– <span class="NLM_lpage">1599</span>, <span class="refDoi"> DOI: 10.1038/emboj.2010.43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Femboj.2010.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=20360685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvVGlu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=1585-1599&author=C.+Artusauthor=H.+Boujradauthor=A.+Bouharrourauthor=M.-N.+Brunelleauthor=S.+Hoosauthor=V.+J.+Yusteauthor=P.+Lenormandauthor=J.-C.+Rousselleauthor=A.+Namaneauthor=P.+Englandauthor=H.+K.+Lorenzoauthor=S.+A.+Susin&title=AIF+promotes+chromatinolysis+and+caspase-independent+programmed+necrosis+by+interacting+with+histone+H2AX&doi=10.1038%2Femboj.2010.43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">AIF promotes chromatinolysis and caspase-independent programmed necrosis by interacting with histone H2AX</span></div><div class="casAuthors">Artus, Cedric; Boujrad, Hanan; Bouharrour, Aida; Brunelle, Marie-Noelle; Hoos, Sylviane; Yuste, Victor J.; Lenormand, Pascal; Rousselle, Jean-Claude; Namane, Abdelkader; England, Patrick; Lorenzo, Hans K.; Susin, Santos A.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1585-1599</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Programmed necrosis induced by DNA alkylating agents, such as MNNG, is a caspase-independent mode of cell death mediated by apoptosis-inducing factor (AIF).  After poly(ADP-ribose) polymerase 1, calpain, and Bax activation, AIF moves from the mitochondria to the nucleus where it induces chromatinolysis and cell death.  The mechanisms underlying the nuclear action of AIF are, however, largely unknown.  We show here that, through its C-terminal proline-rich binding domain (PBD, residues 543-559), AIF assocs. in the nucleus with histone H2AX.  This interaction regulates chromatinolysis and programmed necrosis by generating an active DNA-degrading complex with cyclophilin A (CypA).  Deletion or directed mutagenesis in the AIF C-terminal PBD abolishes AIF/H2AX interaction and AIF-mediated chromatinolysis.  H2AX genetic ablation or CypA downregulation confers resistance to programmed necrosis.  AIF fails to induce chromatinolysis in H2AX or CypA-deficient nuclei.  We also establish that H2AX is phosphorylated at Ser139 after MNNG treatment and that this phosphorylation is crit. for caspase-independent programmed necrosis.  Overall, our data shed new light in the mechanisms regulating programmed necrosis, elucidate a key nuclear partner of AIF, and uncover an AIF apoptogenic motif.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ8ffTkggQzrVg90H21EOLACvtfcHk0li5EMunryvVbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvVGlu7c%253D&md5=5788f2abc9a49fe557591dbb3c019899</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Femboj.2010.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2010.43%26sid%3Dliteratum%253Aachs%26aulast%3DArtus%26aufirst%3DC.%26aulast%3DBoujrad%26aufirst%3DH.%26aulast%3DBouharrour%26aufirst%3DA.%26aulast%3DBrunelle%26aufirst%3DM.-N.%26aulast%3DHoos%26aufirst%3DS.%26aulast%3DYuste%26aufirst%3DV.%2BJ.%26aulast%3DLenormand%26aufirst%3DP.%26aulast%3DRousselle%26aufirst%3DJ.-C.%26aulast%3DNamane%26aufirst%3DA.%26aulast%3DEngland%26aufirst%3DP.%26aulast%3DLorenzo%26aufirst%3DH.%2BK.%26aulast%3DSusin%26aufirst%3DS.%2BA.%26atitle%3DAIF%2520promotes%2520chromatinolysis%2520and%2520caspase-independent%2520programmed%2520necrosis%2520by%2520interacting%2520with%2520histone%2520H2AX%26jtitle%3DEMBO%2520J.%26date%3D2010%26volume%3D29%26spage%3D1585%26epage%3D1599%26doi%3D10.1038%2Femboj.2010.43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wissing, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludovico, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herker, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Büttner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelhardt, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Link, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proksch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corte-Real, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fröhlich, K.-U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manns, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Candé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigrist, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madeo, F.</span></span> <span> </span><span class="NLM_article-title">An AIF orthologue regulates apoptosis in yeast</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>166</i></span>,  <span class="NLM_fpage">969</span>– <span class="NLM_lpage">974</span>, <span class="refDoi"> DOI: 10.1083/jcb.200404138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1083%2Fjcb.200404138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=15381687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotVehtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2004&pages=969-974&author=S.+Wissingauthor=P.+Ludovicoauthor=E.+Herkerauthor=S.+B%C3%BCttnerauthor=S.+M.+Engelhardtauthor=T.+Deckerauthor=A.+Linkauthor=A.+Prokschauthor=F.+Rodriguesauthor=M.+Corte-Realauthor=K.-U.+Fr%C3%B6hlichauthor=J.+Mannsauthor=C.+Cand%C3%A9author=S.+J.+Sigristauthor=G.+Kroemerauthor=F.+Madeo&title=An+AIF+orthologue+regulates+apoptosis+in+yeast&doi=10.1083%2Fjcb.200404138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">An AIF orthologue regulates apoptosis in yeast</span></div><div class="casAuthors">Wissing, Silke; Ludovico, Paula; Herker, Eva; Buettner, Sabrina; Engelhardt, Silvia M.; Decker, Thorsten; Link, Alexander; Proksch, Astrid; Rodrigues, Fernando; Corte-Real, Manuela; Froehlich, Kai-Uwe; Manns, Joachim; Cande, Celine; Sigrist, Stephan J.; Kroemer, Guido; Madeo, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">969-974</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9525</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Apoptosis-inducing factor (AIF), a key regulator of cell death, is essential for normal mammalian development and participates in pathol. apoptosis.  The proapoptotic nature of AIF and its mode of action are controversial.  Here, we show that the yeast AIF homolog Ynr074cp controls yeast apoptosis.  Similar to mammalian AIF, Ynr074cp is located in mitochondria and translocates to the nucleus of yeast cells in response to apoptotic stimuli.  Purified Ynr074cp degrades yeast nuclei and plasmid DNA.  YNR074C disruption rescues yeast cells from oxygen stress and delays age-induced apoptosis.  Conversely, overexpression of Ynr074cp strongly stimulates apoptotic cell death induced by hydrogen peroxide and this effect is attenuated by disruption of cyclophilin A or the yeast caspase YCA1.  We conclude that Ynr074cp is a cell death effector in yeast and rename it AIF-1 (Aif1p, gene AIF1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPaxPrMBVeT7Vg90H21EOLACvtfcHk0li5EMunryvVbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotVehtL4%253D&md5=a526d50e744e440f77eebc4ebd124ba5</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200404138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200404138%26sid%3Dliteratum%253Aachs%26aulast%3DWissing%26aufirst%3DS.%26aulast%3DLudovico%26aufirst%3DP.%26aulast%3DHerker%26aufirst%3DE.%26aulast%3DB%25C3%25BCttner%26aufirst%3DS.%26aulast%3DEngelhardt%26aufirst%3DS.%2BM.%26aulast%3DDecker%26aufirst%3DT.%26aulast%3DLink%26aufirst%3DA.%26aulast%3DProksch%26aufirst%3DA.%26aulast%3DRodrigues%26aufirst%3DF.%26aulast%3DCorte-Real%26aufirst%3DM.%26aulast%3DFr%25C3%25B6hlich%26aufirst%3DK.-U.%26aulast%3DManns%26aufirst%3DJ.%26aulast%3DCand%25C3%25A9%26aufirst%3DC.%26aulast%3DSigrist%26aufirst%3DS.%2BJ.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DMadeo%26aufirst%3DF.%26atitle%3DAn%2520AIF%2520orthologue%2520regulates%2520apoptosis%2520in%2520yeast%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2004%26volume%3D166%26spage%3D969%26epage%3D974%26doi%3D10.1083%2Fjcb.200404138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span> <span> </span><span class="NLM_article-title">Inhibiting the interaction between apoptosis-inducing factor and cyclophilin A prevents brain injury in neonatal mice after hypoxia-ischemia</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">108088</span>, <span class="refDoi"> DOI: 10.1016/j.neuropharm.2020.108088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2Fj.neuropharm.2020.108088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=32277944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BB3cXos1SitL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2020&pages=108088&author=J.+Rodriguezauthor=C.+Xieauthor=T.+Liauthor=Y.+Sunauthor=Y.+Wangauthor=Y.+Xuauthor=K.+Liauthor=S.+Zhangauthor=K.+Zhouauthor=Y.+Wangauthor=C.+Mallardauthor=H.+Hagbergauthor=N.+Dotiauthor=X.+Wangauthor=C.+Zhu&title=Inhibiting+the+interaction+between+apoptosis-inducing+factor+and+cyclophilin+A+prevents+brain+injury+in+neonatal+mice+after+hypoxia-ischemia&doi=10.1016%2Fj.neuropharm.2020.108088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting the interaction between apoptosis-inducing factor and cyclophilin A prevents brain injury in neonatal mice after hypoxia-ischemia</span></div><div class="casAuthors">Rodriguez, Juan; Xie, Cuicui; Li, Tao; Sun, Yanyan; Wang, Yafeng; Xu, Yiran; Li, Kenan; Zhang, Shan; Zhou, Kai; Wang, Yong; Mallard, Carina; Hagberg, Henrik; Doti, Nunzianna; Wang, Xiaoyang; Zhu, Changlian</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">108088</span>CODEN:
                <span class="NLM_cas:coden">NEPHBW</span>;
        ISSN:<span class="NLM_cas:issn">0028-3908</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Blocking the AIF/CypA interaction protects against glutamate-induced neuronal cell death in vitro, and the purpose of this study was to det. the in vivo effect of an AIF/CypA interaction blocking peptide (AIF(370-394)-TAT) on neonatal mouse brain injury after HI.  The pups were treated with AIF (370-394)-TAT peptide intranasally prior to HI.  Brain injury was significantly reduced at 72 h after HI in AIF(370-394)-TAT peptide treatment group compared to vehicle-only treatment for both the gray matter and the subcortical white matter, and the neuroprotection was more pronounced in males than in females.  Neuronal cell death was evaluated in males at 8 h and 24 h post-HI, and it was decreased significantly in the CA1 region of the hippocampus and the nucleus habenularis region after AIF(370-394)-TAT treatment.  Caspase-independent apoptosis was decreased in the cortex, striatum, and nucleus habenularis after AIF(370-394)-TAT treatment, but no significant change was found on caspase-dependent apoptosis as indicated by the no. of active caspase-3-labeled cells.  Further anal. showed that both AIF and CypA nuclear accumulation were decreased after treatment with the AIF(370-394)-TAT peptide.  These results suggest that AIF(370-394)-TAT inhibited AIF/CypA translocation to the nucleus and reduced HI-induced caspase-independent apoptosis and brain injury in young male mice, suggesting that blocking AIF/CypA might be potential therapeutic target for neonatal brain injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmR2RcyHMdpLVg90H21EOLACvtfcHk0li5EMunryvVbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXos1SitL4%253D&md5=a4104264354bbbaaf1d6f7d55afce584</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2020.108088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2020.108088%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMallard%26aufirst%3DC.%26aulast%3DHagberg%26aufirst%3DH.%26aulast%3DDoti%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DC.%26atitle%3DInhibiting%2520the%2520interaction%2520between%2520apoptosis-inducing%2520factor%2520and%2520cyclophilin%2520A%2520prevents%2520brain%2520injury%2520in%2520neonatal%2520mice%2520after%2520hypoxia-ischemia%26jtitle%3DNeuropharmacology%26date%3D2020%26volume%3D171%26spage%3D108088%26doi%3D10.1016%2Fj.neuropharm.2020.108088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chelko, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keceli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrimi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asimaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beti, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amat-Codina, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedja, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichnell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calkins, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Rourke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halushka, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melloni, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffitz, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Judge, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitsis, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Lisa, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paolocci, N.</span></span> <span> </span><span class="NLM_article-title">Exercise triggers CAPN1-mediated AIF truncation, inducing myocyte cell death in arrhythmogenic cardiomyopathy</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">eabf0891</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.abf0891</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1126%2Fscitranslmed.abf0891" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=33597260" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2021&author=S.+P.+Chelkoauthor=G.+Keceliauthor=A.+Carpiauthor=N.+Dotiauthor=J.+Agrimiauthor=A.+Asimakiauthor=C.+B.+Betiauthor=M.+Miyamotoauthor=N.+Amat-Codinaauthor=D.+Bedjaauthor=A.-C.+Weiauthor=B.+Murrayauthor=C.+Tichnellauthor=C.+Kwonauthor=H.+Calkinsauthor=C.+A.+Jamesauthor=B.+O%E2%80%99Rourkeauthor=M.+K.+Halushkaauthor=E.+Melloniauthor=J.+E.+Saffitzauthor=D.+P.+Judgeauthor=M.+Ruvoauthor=R.+N.+Kitsisauthor=P.+Andersenauthor=F.+Di+Lisaauthor=N.+Paolocci&title=Exercise+triggers+CAPN1-mediated+AIF+truncation%2C+inducing+myocyte+cell+death+in+arrhythmogenic+cardiomyopathy&doi=10.1126%2Fscitranslmed.abf0891"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.abf0891&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.abf0891%26sid%3Dliteratum%253Aachs%26aulast%3DChelko%26aufirst%3DS.%2BP.%26aulast%3DKeceli%26aufirst%3DG.%26aulast%3DCarpi%26aufirst%3DA.%26aulast%3DDoti%26aufirst%3DN.%26aulast%3DAgrimi%26aufirst%3DJ.%26aulast%3DAsimaki%26aufirst%3DA.%26aulast%3DBeti%26aufirst%3DC.%2BB.%26aulast%3DMiyamoto%26aufirst%3DM.%26aulast%3DAmat-Codina%26aufirst%3DN.%26aulast%3DBedja%26aufirst%3DD.%26aulast%3DWei%26aufirst%3DA.-C.%26aulast%3DMurray%26aufirst%3DB.%26aulast%3DTichnell%26aufirst%3DC.%26aulast%3DKwon%26aufirst%3DC.%26aulast%3DCalkins%26aufirst%3DH.%26aulast%3DJames%26aufirst%3DC.%2BA.%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DB.%26aulast%3DHalushka%26aufirst%3DM.%2BK.%26aulast%3DMelloni%26aufirst%3DE.%26aulast%3DSaffitz%26aufirst%3DJ.%2BE.%26aulast%3DJudge%26aufirst%3DD.%2BP.%26aulast%3DRuvo%26aufirst%3DM.%26aulast%3DKitsis%26aufirst%3DR.%2BN.%26aulast%3DAndersen%26aufirst%3DP.%26aulast%3DDi%2BLisa%26aufirst%3DF.%26aulast%3DPaolocci%26aufirst%3DN.%26atitle%3DExercise%2520triggers%2520CAPN1-mediated%2520AIF%2520truncation%252C%2520inducing%2520myocyte%2520cell%2520death%2520in%2520arrhythmogenic%2520cardiomyopathy%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2021%26volume%3D13%26doi%3D10.1126%2Fscitranslmed.abf0891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farina, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Sorbo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambery, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caporale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leoni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mascanzoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattorusso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span> <span> </span><span class="NLM_article-title">Structural and biochemical insights of CypA and AIF interaction</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1138</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-01337-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Fs41598-017-01337-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=28442737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A280%3ADC%252BC1crhtFCjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=1138&author=B.+Farinaauthor=G.+Di+Sorboauthor=A.+Chamberyauthor=A.+Caporaleauthor=G.+Leoniauthor=R.+Russoauthor=F.+Mascanzoniauthor=D.+Raimondoauthor=R.+Fattorussoauthor=M.+Ruvoauthor=N.+Doti&title=Structural+and+biochemical+insights+of+CypA+and+AIF+interaction&doi=10.1038%2Fs41598-017-01337-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and biochemical insights of CypA and AIF interaction</span></div><div class="casAuthors">Farina Biancamaria; Di Sorbo Gianluigi; Caporale Andrea; Mascanzoni Fabiola; Ruvo Menotti; Doti Nunzianna; Di Sorbo Gianluigi; Chambery Angela; Russo Rosita; Fattorusso Roberto; Leoni Guido; Raimondo Domenico</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1138</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The Cyclophilin A (CypA)/Apoptosis Inducing Factor (AIF) complex is implicated in the DNA degradation in response to various cellular stress conditions, such as oxidative stress, cerebral hypoxia-ischemia and traumatic brain injury.  The pro-apoptotic form of AIF (AIF(Δ1-121)) mainly interacts with CypA through the amino acid region 370-394.  The AIF(370-394) synthetic peptide inhibits complex formation in vitro by binding to CypA and exerts neuroprotection in a model of glutamate-mediated oxidative stress.  Here, the binding site of AIF(Δ1-121) and AIF(370-394) on CypA has been mapped by NMR spectroscopy and biochemical studies, and a molecular model of the complex has been proposed.  We show that AIF(370-394) interacts with CypA on the same surface recognized by AIF(Δ1-121) protein and that the region is very close to the CypA catalytic pocket.  Such region partially overlaps with the binding site of cyclosporin A (CsA), the strongest catalytic inhibitor of CypA.  Our data point toward distinct CypA structural determinants governing the inhibitor selectivity and the differential biological effects of AIF and CsA, and provide new structural insights for designing CypA/AIF selective inhibitors with therapeutic relevance in neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSCGXnCF2P3vJQHPYJuLg8TfW6udTcc2eYFkk1od69IwLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crhtFCjsA%253D%253D&md5=d036ad6254cef5a4b232f3c074bfec66</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-01337-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-01337-8%26sid%3Dliteratum%253Aachs%26aulast%3DFarina%26aufirst%3DB.%26aulast%3DDi%2BSorbo%26aufirst%3DG.%26aulast%3DChambery%26aufirst%3DA.%26aulast%3DCaporale%26aufirst%3DA.%26aulast%3DLeoni%26aufirst%3DG.%26aulast%3DRusso%26aufirst%3DR.%26aulast%3DMascanzoni%26aufirst%3DF.%26aulast%3DRaimondo%26aufirst%3DD.%26aulast%3DFattorusso%26aufirst%3DR.%26aulast%3DRuvo%26aufirst%3DM.%26aulast%3DDoti%26aufirst%3DN.%26atitle%3DStructural%2520and%2520biochemical%2520insights%2520of%2520CypA%2520and%2520AIF%2520interaction%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D1138%26doi%3D10.1038%2Fs41598-017-01337-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farina, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturlese, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mascanzoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caporale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Sorbo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattorusso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span> <span> </span><span class="NLM_article-title">Binding mode of AIF(370-394) peptide to CypA: insights from NMR, label-free and molecular docking studies</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>475</i></span>,  <span class="NLM_fpage">2377</span>– <span class="NLM_lpage">2393</span>, <span class="refDoi"> DOI: 10.1042/bcj20180177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1042%2FBCJ20180177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=29891613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVGjtLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=475&publication_year=2018&pages=2377-2393&author=B.+Farinaauthor=M.+Sturleseauthor=F.+Mascanzoniauthor=A.+Caporaleauthor=A.+Montiauthor=G.+Di+Sorboauthor=R.+Fattorussoauthor=M.+Ruvoauthor=N.+Doti&title=Binding+mode+of+AIF%28370-394%29+peptide+to+CypA%3A+insights+from+NMR%2C+label-free+and+molecular+docking+studies&doi=10.1042%2Fbcj20180177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Binding mode of AIF(370-394) peptide to CypA: insights from NMR, label-free and molecular docking studies</span></div><div class="casAuthors">Farina, Biancamaria; Sturlese, Mattia; Mascanzoni, Fabiola; Caporale, Andrea; Monti, Alessandra; Sorbo, Gianluigi Di; Fattorusso, Roberto; Ruvo, Menotti; Doti, Nunzianna</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">475</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2377-2393</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The complex formation between the proteins apoptosis-inducing factor (AIF) and cyclophilin A (CypA) following oxidative stress in neuronal cells has been suggested as a main target for reverting ischemia-stroke damage.  Recently, a peptide encompassing AIF residues 370-394 has been developed to target the AIF-binding site on CypA, to prevent the assocn. between the two proteins and suppress glutamate-induced cell death in neuronal cells.  Using a combined approach based on NMR spectroscopy, synthesis and in vitro testing of all Ala-scan mutants of the peptide and mol. docking/mol. dynamics, we have generated a detailed model of the AIF (370-394)/CypA complex.  The model suggests us that the central region of the peptide spanning residues V374-K384 mostly interacts with the protein and that for efficient complex inhibition and preservation of CypA activity, it is bent around amino acids F46-G75 of the protein.  The model is consistent with exptl. data also from previous works and supports the concept that the peptide does not interfere with other CypA activities unrelated to AIF activation; therefore, it may serve as an ideal template for generating future non-peptidic antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl93TSrK4VArVg90H21EOLACvtfcHk0lgV_I2WM2W6Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVGjtLfI&md5=5cf337043a3a440de781333ae404a385</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1042%2FBCJ20180177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20180177%26sid%3Dliteratum%253Aachs%26aulast%3DFarina%26aufirst%3DB.%26aulast%3DSturlese%26aufirst%3DM.%26aulast%3DMascanzoni%26aufirst%3DF.%26aulast%3DCaporale%26aufirst%3DA.%26aulast%3DMonti%26aufirst%3DA.%26aulast%3DDi%2BSorbo%26aufirst%3DG.%26aulast%3DFattorusso%26aufirst%3DR.%26aulast%3DRuvo%26aufirst%3DM.%26aulast%3DDoti%26aufirst%3DN.%26atitle%3DBinding%2520mode%2520of%2520AIF%2528370-394%2529%2520peptide%2520to%2520CypA%253A%2520insights%2520from%2520NMR%252C%2520label-free%2520and%2520molecular%2520docking%2520studies%26jtitle%3DBiochem.%2520J.%26date%3D2018%26volume%3D475%26spage%3D2377%26epage%3D2393%26doi%3D10.1042%2Fbcj20180177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cande, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephanou, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurbuxani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larochette, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrido, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span> <span> </span><span class="NLM_article-title">DNA binding is required for the apoptogenic action of apoptosis inducing factor</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">680</span>– <span class="NLM_lpage">684</span>, <span class="refDoi"> DOI: 10.1038/nsb836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Fnsb836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12198487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1ajur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=680-684&author=H.+Yeauthor=C.+Candeauthor=N.+C.+Stephanouauthor=S.+Jiangauthor=S.+Gurbuxaniauthor=N.+Larochetteauthor=E.+Daugasauthor=C.+Garridoauthor=G.+Kroemerauthor=H.+Wu&title=DNA+binding+is+required+for+the+apoptogenic+action+of+apoptosis+inducing+factor&doi=10.1038%2Fnsb836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">DNA binding is required for the apoptogenic action of apoptosis inducing factor</span></div><div class="casAuthors">Ye, Hong; Cande, Celine; Stephanou, Nicolas C.; Jiang, Sulin; Gurbuxani, Sundeep; Larochette, Nathanael; Daugas, Eric; Garrido, Carmen; Kroemer, Guido; Wu, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">680-684</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The execution of apoptosis or programmed cell death comprises both caspase-dependent and caspase-independent processes.  Apoptosis inducing factor (AIF) was identified as a major player in caspase-independent cell death.  It induces chromatin condensation and initial DNA cleavage via an unknown mol. mechanism.  Here we report the crystal structure of human AIF at 1.8 Å resoln.  The structure reveals the presence of a strong pos. electrostatic potential at the AIF surface, although the calcd. isoelec. point for the entire protein is neutral.  We show that recombinant AIF interacts with DNA in a sequence-independent manner.  In addn., in cells treated with an apoptotic stimulus, endogenous AIF becomes co-localized with DNA at an early stage of nuclear morphol. changes.  Structure-based mutagenesis shows that DNA-binding defective mutants of AIF fail to induce cell death while retaining nuclear translocation.  The potential DNA-binding site identified from mutagenesis also coincides with computational docking of a DNA duplex.  These observations suggest that AIF-induced nuclear apoptosis requires a direct interaction with DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnsbZhi6U0ALVg90H21EOLACvtfcHk0liqlr7dIvaPsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1ajur8%253D&md5=419d4eb7d4feb72f72614c6b3f72969c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnsb836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb836%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DH.%26aulast%3DCande%26aufirst%3DC.%26aulast%3DStephanou%26aufirst%3DN.%2BC.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DGurbuxani%26aufirst%3DS.%26aulast%3DLarochette%26aufirst%3DN.%26aulast%3DDaugas%26aufirst%3DE.%26aulast%3DGarrido%26aufirst%3DC.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26atitle%3DDNA%2520binding%2520is%2520required%2520for%2520the%2520apoptogenic%2520action%2520of%2520apoptosis%2520inducing%2520factor%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D9%26spage%3D680%26epage%3D684%26doi%3D10.1038%2Fnsb836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturlese, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caporale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roger, J. D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mascanzoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, structural analysis and biochemical studies of stapled AIF(370-394) analogues as ligand of CypA</span>. <i>BBA, Biochim. Biophys. Acta, Gen. Subj.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>1864</i></span>,  <span class="NLM_fpage">129717</span>, <span class="refDoi"> DOI: 10.1016/j.bbagen.2020.129717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2Fj.bbagen.2020.129717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=32861757" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1864&publication_year=2020&pages=129717&author=A.+Montiauthor=M.+Sturleseauthor=A.+Caporaleauthor=J.+D.+A.+Rogerauthor=F.+Mascanzoniauthor=M.+Ruvoauthor=N.+Doti&title=Design%2C+synthesis%2C+structural+analysis+and+biochemical+studies+of+stapled+AIF%28370-394%29+analogues+as+ligand+of+CypA&doi=10.1016%2Fj.bbagen.2020.129717"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2020.129717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2020.129717%26sid%3Dliteratum%253Aachs%26aulast%3DMonti%26aufirst%3DA.%26aulast%3DSturlese%26aufirst%3DM.%26aulast%3DCaporale%26aufirst%3DA.%26aulast%3DRoger%26aufirst%3DJ.%2BD.%2BA.%26aulast%3DMascanzoni%26aufirst%3DF.%26aulast%3DRuvo%26aufirst%3DM.%26aulast%3DDoti%26aufirst%3DN.%26atitle%3DDesign%252C%2520synthesis%252C%2520structural%2520analysis%2520and%2520biochemical%2520studies%2520of%2520stapled%2520AIF%2528370-394%2529%2520analogues%2520as%2520ligand%2520of%2520CypA%26jtitle%3DBBA%252C%2520Biochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2020%26volume%3D1864%26spage%3D129717%26doi%3D10.1016%2Fj.bbagen.2020.129717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starovasnik, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDowell, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theaker, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skelton, N. J.</span></span> <span> </span><span class="NLM_article-title">A Minimal Peptide Scaffold for β-Turn Display: Optimizing a Strand Position in Disulfide-Cyclized β-Hairpins</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">625</span>– <span class="NLM_lpage">632</span>, <span class="refDoi"> DOI: 10.1021/ja003369x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja003369x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3cXptVyjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2001&pages=625-632&author=A.+G.+Cochranauthor=R.+T.+Tongauthor=M.+A.+Starovasnikauthor=E.+J.+Parkauthor=R.+S.+McDowellauthor=J.+E.+Theakerauthor=N.+J.+Skelton&title=A+Minimal+Peptide+Scaffold+for+%CE%B2-Turn+Display%3A+Optimizing+a+Strand+Position+in+Disulfide-Cyclized+%CE%B2-Hairpins&doi=10.1021%2Fja003369x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A Minimal Peptide Scaffold for β-Turn Display: Optimizing a Strand Position in Disulfide-Cyclized β-Hairpins</span></div><div class="casAuthors">Cochran, Andrea G.; Tong, Ricky T.; Starovasnik, Melissa A.; Park, Eleanor J.; McDowell, Robert S.; Theaker, J. E.; Skelton, Nicholas J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">625-632</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phage display of peptide libraries has become a powerful tool for the evolution of novel ligands that bind virtually any protein target.  However, the rules governing conformational preferences in natural peptides are poorly understood, and consequently, structure-activity relationships in these mols. can be difficult to define.  In an effort to simplify this process, we have investigated the structural stability of 10-residue, disulfide-constrained β-hairpins and assessed their suitability as scaffolds for β-turn display.  Using disulfide formation as a probe, relative free energies of folding were measured for 19 peptides that differ at a one strand position.  A tryptophan substitution promotes folding to a remarkable degree.  NMR anal. confirms that the measured energies correlate well with the degree of β-hairpin structure in the disulfide-cyclized peptides.  Reexamn. of a subset of the strand substitutions in peptides with different turn sequences reveals linear free energy relationships, indicating that turns and strand-strand interactions make independent, additive contributions to hairpin stability.  Significantly, the tryptophan strand substitution is highly stabilizing with all turns tested, and peptides that display model turns or the less stable C'-C'' turn of CD4 on this tryptophan "stem" are highly structured β-hairpins in water.  Thus, we have developed a small, structured β-turn scaffold, contg. only natural L-amino acids, that may be used to display peptide libraries of limited conformational diversity on phage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr76zn4qfmZE7Vg90H21EOLACvtfcHk0liqlr7dIvaPsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXptVyjsLg%253D&md5=e422fa4f1204a9573f2f6266c59c30ec</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja003369x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja003369x%26sid%3Dliteratum%253Aachs%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DTong%26aufirst%3DR.%2BT.%26aulast%3DStarovasnik%26aufirst%3DM.%2BA.%26aulast%3DPark%26aufirst%3DE.%2BJ.%26aulast%3DMcDowell%26aufirst%3DR.%2BS.%26aulast%3DTheaker%26aufirst%3DJ.%2BE.%26aulast%3DSkelton%26aufirst%3DN.%2BJ.%26atitle%3DA%2520Minimal%2520Peptide%2520Scaffold%2520for%2520%25CE%25B2-Turn%2520Display%253A%2520Optimizing%2520a%2520Strand%2520Position%2520in%2520Disulfide-Cyclized%2520%25CE%25B2-Hairpins%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2001%26volume%3D123%26spage%3D625%26epage%3D632%26doi%3D10.1021%2Fja003369x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farina, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Gatto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comegna, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Gaetano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capasso, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isernia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattorusso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaccaro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, L.</span></span> <span> </span><span class="NLM_article-title">Conformational studies of RGDechi peptide by natural-abundance NMR spectroscopy</span>. <i>J. Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>, <span class="NLM_elocation-id">e3166</span> <span class="refDoi"> DOI: 10.1002/psc.3166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1002%2Fpsc.3166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=30884005" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&author=B.+Farinaauthor=A.+Del+Gattoauthor=D.+Comegnaauthor=S.+Di+Gaetanoauthor=D.+Capassoauthor=C.+Iserniaauthor=M.+Savianoauthor=R.+Fattorussoauthor=L.+Zaccaroauthor=L.+Russo&title=Conformational+studies+of+RGDechi+peptide+by+natural-abundance+NMR+spectroscopy&doi=10.1002%2Fpsc.3166"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fpsc.3166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsc.3166%26sid%3Dliteratum%253Aachs%26aulast%3DFarina%26aufirst%3DB.%26aulast%3DDel%2BGatto%26aufirst%3DA.%26aulast%3DComegna%26aufirst%3DD.%26aulast%3DDi%2BGaetano%26aufirst%3DS.%26aulast%3DCapasso%26aufirst%3DD.%26aulast%3DIsernia%26aufirst%3DC.%26aulast%3DSaviano%26aufirst%3DM.%26aulast%3DFattorusso%26aufirst%3DR.%26aulast%3DZaccaro%26aufirst%3DL.%26aulast%3DRusso%26aufirst%3DL.%26atitle%3DConformational%2520studies%2520of%2520RGDechi%2520peptide%2520by%2520natural-abundance%2520NMR%2520spectroscopy%26jtitle%3DJ.%2520Pept.%2520Sci.%26date%3D2019%26volume%3D25%26doi%3D10.1002%2Fpsc.3166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sykes, B. D.</span></span> <span> </span><span class="NLM_article-title">The 13C chemical-shift index: a simple method for the identification of protein secondary structure using 13C chemical-shift data</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1007/BF00175245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2FBF00175245" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=8019132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK2cXjtFymtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1994&pages=171-180&author=D.+S.+Wishartauthor=B.+D.+Sykes&title=The+13C+chemical-shift+index%3A+a+simple+method+for+the+identification+of+protein+secondary+structure+using+13C+chemical-shift+data&doi=10.1007%2FBF00175245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The 13C chemical-shift index: a simple method for the identification of protein secondary structure using 13C chemical-shift data</span></div><div class="casAuthors">Wishart, David S.; Sykes, Brian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">171-80</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    </div><div class="casAbstract">A simple technique for identifying protein secondary structures through the anal. of backbone 13C chem. shifts is described.  It is based on the Chem.-Shift Index [Wishart et al. (1992)] which was originally developed for the anal. of 1Hα chem. shifts.  By extending the Chem.-Shift Index to include 13Cα, 13Cβ and carbonyl 13C chem. shifts, it is now possible to use four independent chem.-shift measurements to identify and locate protein secondary structures.  It is shown that by combining both 1H and 13C chem.-shift indexes to produce a 'consensus' est. of secondary structure, it is possible to achieve a predictive accuracy in excess of 92%.  This suggests that the secondary structure of peptides and proteins can be accurately obtained from 1H and 13C chem. shifts, without recourse to NOE measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDXGadKPmScrVg90H21EOLACvtfcHk0lgfT4Xn9bPmwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXjtFymtLw%253D&md5=a248adaea4793a6b67ff69e33f6248e8</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1007%2FBF00175245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00175245%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DSykes%26aufirst%3DB.%2BD.%26atitle%3DThe%252013C%2520chemical-shift%2520index%253A%2520a%2520simple%2520method%2520for%2520the%2520identification%2520of%2520protein%2520secondary%2520structure%2520using%252013C%2520chemical-shift%2520data%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1994%26volume%3D4%26spage%3D171%26epage%3D180%26doi%3D10.1007%2FBF00175245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Use of chemical shifts in macromolecular structure determination</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>338</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1016/s0076-6879(02)38214-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2FS0076-6879%2802%2938214-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=11460554" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1eltrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=338&publication_year=2001&pages=3-34&author=D.+S.+Wishartauthor=D.+A.+Case&title=Use+of+chemical+shifts+in+macromolecular+structure+determination&doi=10.1016%2Fs0076-6879%2802%2938214-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Use of chemical shifts in macromolecular structure determination</span></div><div class="casAuthors">Wishart, David S.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">338</span>
        (<span class="NLM_cas:issue">Nuclear Magnetic Resonance of Biological Macromolecules, Part A</span>),
    <span class="NLM_cas:pages">3-34</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A review discusses the use of chem. shifts as an aid in understanding, generating or refining macromol. structures.  It also covers the random coil shifts, solid-state NMR structure generation, and chem. shift refinement.  The quantum mech. and more classical approaches to calcg. chem. shifts from structure are also discussed.  Chem. shifts are among the most precisely measurable but least accurately measured spectral parameters in all of NMR spectroscopy.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp875NU8XRsL7Vg90H21EOLACvtfcHk0lgfT4Xn9bPmwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1eltrg%253D&md5=04ad26a71d01536439c8e35fafb94a7f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2802%2938214-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252802%252938214-4%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DUse%2520of%2520chemical%2520shifts%2520in%2520macromolecular%2520structure%2520determination%26jtitle%3DMethods%2520Enzymol.%26date%3D2001%26volume%3D338%26spage%3D3%26epage%3D34%26doi%3D10.1016%2Fs0076-6879%2802%2938214-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kjaergaard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brander, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulsen, F. M.</span></span> <span> </span><span class="NLM_article-title">Random coil chemical shift for intrinsically disordered proteins: effects of temperature and pH</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">139</span>– <span class="NLM_lpage">149</span>, <span class="refDoi"> DOI: 10.1007/s10858-011-9472-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2Fs10858-011-9472-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=21234644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1GrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2011&pages=139-149&author=M.+Kjaergaardauthor=S.+Branderauthor=F.+M.+Poulsen&title=Random+coil+chemical+shift+for+intrinsically+disordered+proteins%3A+effects+of+temperature+and+pH&doi=10.1007%2Fs10858-011-9472-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Random coil chemical shift for intrinsically disordered proteins: effects of temperature and pH</span></div><div class="casAuthors">Kjaergaard, Magnus; Brander, Soren; Poulsen, Flemming M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">139-149</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Secondary chem. shift anal. is the main NMR method for detection of transiently formed secondary structure in intrinsically disordered proteins.  The quality of the secondary chem. shifts is dependent on an appropriate choice of random coil chem. shifts.  We report random coil chem. shifts and sequence correction factors detd. for a GGXGG peptide series following the approach of Schwarzinger et al. (J Am Chem Soc 123(13):2970-2978, 2001).  The chem. shifts are detd. at neutral pH in order to match the conditions of most studies of intrinsically disordered proteins.  Temp. has a non-negligible effect on the 13C random coil chem. shifts, so temp. coeffs. are reported for the random coil chem. shifts to allow extrapolation to other temps.  The pH dependence of the histidine random coil chem. shifts is investigated in a titrn. series, which allows the accurate random coil chem. shifts to be obtained at any pH.  By correcting the random coil chem. shifts for the effects of temp. and pH, systematic biases of the secondary chem. shifts are minimized, which will improve the reliability of detection of transient secondary structure in disordered proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7DYWlBZnyaLVg90H21EOLACvtfcHk0ljgTnf2m9uvIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1GrsLw%253D&md5=fcc9873b54dcf445a35f70e0e9ac6a2e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1007%2Fs10858-011-9472-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10858-011-9472-x%26sid%3Dliteratum%253Aachs%26aulast%3DKjaergaard%26aufirst%3DM.%26aulast%3DBrander%26aufirst%3DS.%26aulast%3DPoulsen%26aufirst%3DF.%2BM.%26atitle%3DRandom%2520coil%2520chemical%2520shift%2520for%2520intrinsically%2520disordered%2520proteins%253A%2520effects%2520of%2520temperature%2520and%2520pH%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2011%26volume%3D49%26spage%3D139%26epage%3D149%26doi%3D10.1007%2Fs10858-011-9472-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kjaergaard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulsen, F. M.</span></span> <span> </span><span class="NLM_article-title">Sequence correction of random coil chemical shifts: correlation between neighbor correction factors and changes in the Ramachandran distribution</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">165</span>, <span class="refDoi"> DOI: 10.1007/s10858-011-9508-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2Fs10858-011-9508-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=21604143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntlahu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=157-165&author=M.+Kjaergaardauthor=F.+M.+Poulsen&title=Sequence+correction+of+random+coil+chemical+shifts%3A+correlation+between+neighbor+correction+factors+and+changes+in+the+Ramachandran+distribution&doi=10.1007%2Fs10858-011-9508-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence correction of random coil chemical shifts: correlation between neighbor correction factors and changes in the Ramachandran distribution</span></div><div class="casAuthors">Kjaergaard, Magnus; Poulsen, Flemming M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-165</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Random coil chem. shifts are necessary for secondary chem. shift anal., which is the main NMR method for identification of secondary structure in proteins.  One of the largest challenges in the detn. of random coil chem. shifts is accounting for the effect of neighboring residues.  The contributions from the neighboring residues are typically removed by using neighbor correction factors detd. based on each residue's effect on glycine chem. shifts.  Due to its unusual conformational freedom, glycine may be particularly unrepresentative for the remaining residue types.  In this study, we use random coil peptides contg. glutamine instead of glycine to det. the random coil chem. shifts and the neighbor correction factors.  The resulting correction factors correlate to changes in the populations of the major wells in the Ramachandran plot, which demonstrates that changes in the conformational ensemble are an important source of neighbor effects in disordered proteins.  Glutamine derived random coil chem. shifts and correction factors modestly improve our ability to predict 13C chem. shifts of intrinsically disordered proteins compared to existing datasets, and may thus improve the identification of small populations of transient structure in disordered proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTS4ANLXd1bbVg90H21EOLACvtfcHk0ljgTnf2m9uvIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntlahu70%253D&md5=f8ea3e2e00a827e4836b0f5ed66becb6</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2Fs10858-011-9508-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10858-011-9508-2%26sid%3Dliteratum%253Aachs%26aulast%3DKjaergaard%26aufirst%3DM.%26aulast%3DPoulsen%26aufirst%3DF.%2BM.%26atitle%3DSequence%2520correction%2520of%2520random%2520coil%2520chemical%2520shifts%253A%2520correlation%2520between%2520neighbor%2520correction%2520factors%2520and%2520changes%2520in%2520the%2520Ramachandran%2520distribution%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2011%26volume%3D50%26spage%3D157%26epage%3D165%26doi%3D10.1007%2Fs10858-011-9508-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanz-Hernández, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Simone, A.</span></span> <span> </span><span class="NLM_article-title">The PROSECCO server for chemical shift predictions in ordered and disordered proteins</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">147</span>– <span class="NLM_lpage">156</span>, <span class="refDoi"> DOI: 10.1007/s10858-017-0145-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2Fs10858-017-0145-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=29119515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2jurzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2017&pages=147-156&author=M.+Sanz-Hern%C3%A1ndezauthor=A.+De+Simone&title=The+PROSECCO+server+for+chemical+shift+predictions+in+ordered+and+disordered+proteins&doi=10.1007%2Fs10858-017-0145-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The PROSECCO server for chemical shift predictions in ordered and disordered proteins</span></div><div class="casAuthors">Sanz-Hernandez, Maximo; De Simone, Alfonso</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">147-156</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The chem. shifts measured in soln.-state and solid-state NMR (NMR) are powerful probes of the structure and dynamics of protein mols.  The exploitation of chem. shifts requires methods to correlate these data with the protein structures and sequences.  We present here an approach to calc. accurate chem. shifts in both ordered and disordered proteins using exclusively the information contained in their sequences.  Our sequence-based approach, protein sequences and chem. shift correlations (PROSECCO), achieves the accuracy of the most advanced structure-based methods in the characterization of chem. shifts of folded proteins and improves the state of the art in the study of disordered proteins.  Our analyses revealed fundamental insights on the structural information carried by NMR chem. shifts of structured and unstructured protein states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSDm6QomKYULVg90H21EOLACvtfcHk0ljgTnf2m9uvIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2jurzE&md5=8008a7dcc2025e87fdd5fa09629787cc</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1007%2Fs10858-017-0145-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10858-017-0145-2%26sid%3Dliteratum%253Aachs%26aulast%3DSanz-Hern%25C3%25A1ndez%26aufirst%3DM.%26aulast%3DDe%2BSimone%26aufirst%3DA.%26atitle%3DThe%2520PROSECCO%2520server%2520for%2520chemical%2520shift%2520predictions%2520in%2520ordered%2520and%2520disordered%2520proteins%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2017%26volume%3D69%26spage%3D147%26epage%3D156%26doi%3D10.1007%2Fs10858-017-0145-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tamiola, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulder, F. A. A.</span></span> <span> </span><span class="NLM_article-title">Using NMR chemical shifts to calculate the propensity for structural order and disorder in proteins</span>. <i>Biochem. Soc. Trans.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1014</span>– <span class="NLM_lpage">1020</span>, <span class="refDoi"> DOI: 10.1042/bst20120171</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1042%2FBST20120171" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=22988857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlOjur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=1014-1020&author=K.+Tamiolaauthor=F.+A.+A.+Mulder&title=Using+NMR+chemical+shifts+to+calculate+the+propensity+for+structural+order+and+disorder+in+proteins&doi=10.1042%2Fbst20120171"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Using NMR chemical shifts to calculate the propensity for structural order and disorder in proteins</span></div><div class="casAuthors">Tamiola, Kamil; Mulder, Frans A. A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Society Transactions</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1014-1020</span>CODEN:
                <span class="NLM_cas:coden">BCSTB5</span>;
        ISSN:<span class="NLM_cas:issn">0300-5127</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  NMR spectroscopy offers the unique possibility to relate the structural propensities of disordered proteins and loop segments of folded peptides to biol. function and aggregation behavior.  Backbone chem. shifts are ideally suited for this task, provided that appropriate ref. data are available and idiosyncratic sensitivity of backbone chem. shifts to structural information is treated in a sensible manner.  In the present paper, we describe methods to detect structural protein changes from chem. shifts, and present an online tool [ncSPC (neighbor-cor. Structural Propensity Calculator)], which unites aspects of several current approaches.  Examples of structural propensity calcns. are given for two well-characterized systems, namely the binding of α-synuclein to micelles and light activation of photoactive yellow protein.  These examples spotlight the great power of NMR chem. shift anal. for the quant. assessment of protein disorder at the at. level, and further our understanding of biol. important problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6Faz_S9Sm2rVg90H21EOLACvtfcHk0ljgTnf2m9uvIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlOjur7I&md5=5a028e85b8450c0ee2b46a3656d17916</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1042%2FBST20120171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBST20120171%26sid%3Dliteratum%253Aachs%26aulast%3DTamiola%26aufirst%3DK.%26aulast%3DMulder%26aufirst%3DF.%2BA.%2BA.%26atitle%3DUsing%2520NMR%2520chemical%2520shifts%2520to%2520calculate%2520the%2520propensity%2520for%2520structural%2520order%2520and%2520disorder%2520in%2520proteins%26jtitle%3DBiochem.%2520Soc.%2520Trans.%26date%3D2012%26volume%3D40%26spage%3D1014%26epage%3D1020%26doi%3D10.1042%2Fbst20120171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span> <span> </span><span class="NLM_article-title">Interpreting protein chemical shift data</span>. <i>Prog. Nucl. Magn. Reson. Spectrosc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.pnmrs.2010.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2Fj.pnmrs.2010.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=21241884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVKqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2011&pages=62-87&author=D.+S.+Wishart&title=Interpreting+protein+chemical+shift+data&doi=10.1016%2Fj.pnmrs.2010.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Interpreting protein chemical shift data</span></div><div class="casAuthors">Wishart, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Nuclear Magnetic Resonance Spectroscopy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">62-87</span>CODEN:
                <span class="NLM_cas:coden">PNMRAT</span>;
        ISSN:<span class="NLM_cas:issn">0079-6565</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopFqH4zvjnwLVg90H21EOLACvtfcHk0libMDbHrat1Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVKqtg%253D%253D&md5=88c1ea81ee5bc5748b2e7215d946c792</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.pnmrs.2010.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pnmrs.2010.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26atitle%3DInterpreting%2520protein%2520chemical%2520shift%2520data%26jtitle%3DProg.%2520Nucl.%2520Magn.%2520Reson.%2520Spectrosc.%26date%3D2011%26volume%3D58%26spage%3D62%26epage%3D87%26doi%3D10.1016%2Fj.pnmrs.2010.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman-Kay, J. D.</span></span> <span> </span><span class="NLM_article-title">Sensitivity of secondary structure propensities to sequence differences between α- and γ-synuclein: Implications for fibrillation</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2795</span>– <span class="NLM_lpage">2804</span>, <span class="refDoi"> DOI: 10.1110/ps.062465306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1110%2Fps.062465306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=17088319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlSmtbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2006&pages=2795-2804&author=J.+A.+Marshauthor=V.+K.+Singhauthor=Z.+Jiaauthor=J.+D.+Forman-Kay&title=Sensitivity+of+secondary+structure+propensities+to+sequence+differences+between+%CE%B1-+and+%CE%B3-synuclein%3A+Implications+for+fibrillation&doi=10.1110%2Fps.062465306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Sensitivity of secondary structure propensities to sequence differences between α- and γ-synuclein: implications for fibrillation</span></div><div class="casAuthors">Marsh, Joseph A.; Singh, Vinay K.; Jia, Zongchao; Forman-Kay, Julie D.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2795-2804</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The synucleins are a family of intrinsically disordered proteins involved in various human diseases.  α-Synuclein has been extensively characterized due to its role in Parkinson's disease where it forms intracellular aggregates, while γ-synuclein is overexpressed in a majority of late-stage breast cancers.  Despite fairly strong sequence similarity between the amyloid-forming regions of α- and γ-synuclein, γ-synuclein has only a weak propensity to form amyloid fibrils.  The authors hypothesize that the different fibrillation tendencies of α- and γ-synuclein may be related to differences in structural propensities.  Here the authors have measured chem. shifts for γ-synuclein and compared them to previously published shifts for α-synuclein.  To facilitate direct comparison, the authors have implemented a simple new technique for re-referencing chem. shifts that the authors have highly effective for both disordered and folded proteins.  In addn., the authors have developed a new method that combines different chem. shifts into a single residue-specific secondary structure propensity (SSP) score.  The authors observe significant differences between α- and γ-synuclein secondary structure propensities.  Most interestingly, γ-synuclein has an increased α-helical propensity in the amyloid-forming region that is crit. for α-synuclein fibrillation, suggesting that increased structural stability in this region may protect against γ-synuclein aggregation.  This comparison of residue-specific secondary structure propensities between intrinsically disordered homologs highlights the sensitivity of transient structure to sequence changes, which the authors suggest may have been exploited as an evolutionary mechanism for fast modulation of protein structure and, hence, function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoObrK2Iobl67Vg90H21EOLACvtfcHk0libMDbHrat1Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlSmtbzL&md5=9e78ad92366e7fe40c3523ba781f80dc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1110%2Fps.062465306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.062465306%26sid%3Dliteratum%253Aachs%26aulast%3DMarsh%26aufirst%3DJ.%2BA.%26aulast%3DSingh%26aufirst%3DV.%2BK.%26aulast%3DJia%26aufirst%3DZ.%26aulast%3DForman-Kay%26aufirst%3DJ.%2BD.%26atitle%3DSensitivity%2520of%2520secondary%2520structure%2520propensities%2520to%2520sequence%2520differences%2520between%2520%25CE%25B1-%2520and%2520%25CE%25B3-synuclein%253A%2520Implications%2520for%2520fibrillation%26jtitle%3DProtein%2520Sci.%26date%3D2006%26volume%3D15%26spage%3D2795%26epage%3D2804%26doi%3D10.1110%2Fps.062465306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fesinmeyer, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, G. W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Euser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, N. H.</span></span> <span> </span><span class="NLM_article-title">Chemical Shifts Provide Fold Populations and Register of β Hairpins and β Sheets</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1007/s10858-005-3731-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2Fs10858-005-3731-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=16341751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSjtLbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=213-231&author=R.+M.+Fesinmeyerauthor=F.+M.+Hudsonauthor=K.+A.+Olsenauthor=G.+W.+N.+Whiteauthor=A.+Euserauthor=N.+H.+Andersen&title=Chemical+Shifts+Provide+Fold+Populations+and+Register+of+%CE%B2+Hairpins+and+%CE%B2+Sheets&doi=10.1007%2Fs10858-005-3731-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Shifts Provide Fold Populations and Register of β Hairpins and β Sheets</span></div><div class="casAuthors">Fesinmeyer, R. Matthew; Hudson, F. Michael; Olsen, Katherine A.; White, George W. N.; Euser, Anna; Andersen, Niels H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">213-231</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A detailed anal. of peptide backbone amide (HN) and Hα chem. shifts reveals a consistent pattern for β hairpins and three-stranded β sheets.  The Hα's at non-hydrogen-bonded strand positions are inwardly directed and shifted downfield ∼1 ppm due largely to an anisotropy contribution from the cross-strand amide function.  The secondary structure assocd. Hα shift deviations for the H-bonded strand positions are also pos. but much smaller (0.1-0.3 ppm) and the turn residues display neg. Hα chem. shift deviations (CSDs).  The pattern of (HN) shift deviations is an even better indicator of both hairpin formation and register, with the cross-strand H-bonded sites shifted downfield (also by ∼1 ppm) and with diagnostic values for the first turn residue and the first strand position following the turn.  These empirical observations, initially made for [2:2]/[2:4]-type-I' and -II' hairpins, are rationalized and can be extended to the anal. of other turns, hairpin classes ([3:5], [4:4]/[4:6]), and three-stranded peptide β-sheet models.  The Hα's at non-hydrogen-bonded sites and (HN)'s in the intervening H-bonded sites provide the largest and most dependable measures of hairpin structuring and can be used for melting studies; however the intrinsic temp. dependence of (HN) shifts deviations needs to reflect the extent of solvent sequestration in the folded state.  Several observations made in the course of this study provide insights into β-sheet folding mechanisms: (1) The magnitude of the (HN) shifts suggests that the cross-strand H-bonds in peptide hairpins are as short as those in protein β sheets. (2) Even L-Pro-Gly turns, which are frequently used in unfolded controls for hairpin peptides, can support hairpin populations in aq. fluoroalc. media. (3) The good correlation between hairpin population ests. from cross-strand H-bonded (HN) shift deviations, Hα shift deviations, and structuring shifts at the turn locus implies that hairpin folding transitions approx. two-state behavior.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_LkJ8Hx5IgrVg90H21EOLACvtfcHk0libMDbHrat1Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSjtLbP&md5=78c92d18825a4cb8f7e9f30ca8e79fdb</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1007%2Fs10858-005-3731-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10858-005-3731-7%26sid%3Dliteratum%253Aachs%26aulast%3DFesinmeyer%26aufirst%3DR.%2BM.%26aulast%3DHudson%26aufirst%3DF.%2BM.%26aulast%3DOlsen%26aufirst%3DK.%2BA.%26aulast%3DWhite%26aufirst%3DG.%2BW.%2BN.%26aulast%3DEuser%26aufirst%3DA.%26aulast%3DAndersen%26aufirst%3DN.%2BH.%26atitle%3DChemical%2520Shifts%2520Provide%2520Fold%2520Populations%2520and%2520Register%2520of%2520%25CE%25B2%2520Hairpins%2520and%2520%25CE%25B2%2520Sheets%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2005%26volume%3D33%26spage%3D213%26epage%3D231%26doi%3D10.1007%2Fs10858-005-3731-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santiveri, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pantoja-Uceda, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rico, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiménez, M. A.</span></span> <span> </span><span class="NLM_article-title">β-Hairpin formation in aqueous solution and in the presence of trifluoroethanol: A1H and13C nuclear magnetic resonance conformational study of designed peptides</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">150</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1002/bip.20345</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1002%2Fbip.20345" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=16078190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVyhtbjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2005&pages=150-162&author=C.+M.+Santiveriauthor=D.+Pantoja-Ucedaauthor=M.+Ricoauthor=M.+A.+Jim%C3%A9nez&title=%CE%B2-Hairpin+formation+in+aqueous+solution+and+in+the+presence+of+trifluoroethanol%3A+A1H+and13C+nuclear+magnetic+resonance+conformational+study+of+designed+peptides&doi=10.1002%2Fbip.20345"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">β-hairpin formation in aqueous solution and in the presence of trifluoroethanol: A 1H and 13C nuclear magnetic resonance conformational study of designed peptides</span></div><div class="casAuthors">Santiveri, Clara M.; Pantoja-Uceda, David; Rico, Manuel; Jimenez, M. Angeles</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">150-162</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In order to check our current knowledge on the principles involved in β-hairpin formation, we have modified the sequence of a 3:5 β-hairpin forming peptide with two different purposes, first to increase the stability of the formed 3:5 β-hairpin, and second to convert the 3:5 β-hairpin into a 2:2 β-hairpin.  The conformational behavior of the designed peptides was investigated in aq. soln. and in 30% trifluoroethanol (TFE) by anal. of the following NMR parameters: nuclear Overhauser effect (NOE) data, and CαH, 13Cα, and 13Cβ conformational shifts.  From the differences in the ability to adopt β-hairpin structures in these peptides, we have arrived to the following conclusions: (i) β-Hairpin population increases with the statistical propensity of residues to occupy each turn position.  The loop length, and in turn, the β-hairpin type, can be modified as a function of the type of turn favored by the loop sequence.  These two conclusions reinforce previous results about the importance of β-turn sequence in β-hairpin folding.  Side-chain packing on each face of the β-sheet may play a major role in β-hairpin stability; hence simplified anal. in terms of isolated pair interactions and intrinsic β-sheet propensities is insufficient.  Contributions to β-hairpin stability of turn and strand sequences are not completely independent.  The burial of hydrophobic surface upon β-hairpin formation that, in turn, depends on side-chain packing also contributes to β-hairpin stability.  As previously obsd., TFE stabilizes β-hairpin structures, but the extent of the contribution of different factors to β-hairpin formation is sometimes different in aq. soln. and in 30% TFE.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor1sjAzZ7MOLVg90H21EOLACvtfcHk0liF561qQb_cXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVyhtbjM&md5=7c051b5f3d9df01fcd284c89cfa504ab</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fbip.20345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.20345%26sid%3Dliteratum%253Aachs%26aulast%3DSantiveri%26aufirst%3DC.%2BM.%26aulast%3DPantoja-Uceda%26aufirst%3DD.%26aulast%3DRico%26aufirst%3DM.%26aulast%3DJim%25C3%25A9nez%26aufirst%3DM.%2BA.%26atitle%3D%25CE%25B2-Hairpin%2520formation%2520in%2520aqueous%2520solution%2520and%2520in%2520the%2520presence%2520of%2520trifluoroethanol%253A%2520A1H%2520and13C%2520nuclear%2520magnetic%2520resonance%2520conformational%2520study%2520of%2520designed%2520peptides%26jtitle%3DBiopolymers%26date%3D2005%26volume%3D79%26spage%3D150%26epage%3D162%26doi%3D10.1002%2Fbip.20345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajarathnam, K.</span></span> <span> </span><span class="NLM_article-title">13C NMR chemical shifts can predict disulfide bond formation</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">165</span>– <span class="NLM_lpage">171</span>, <span class="refDoi"> DOI: 10.1023/a:1008398416292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1023%2FA%3A1008398416292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=11101221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosVWgsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2000&pages=165-171&author=D.+Sharmaauthor=K.+Rajarathnam&title=13C+NMR+chemical+shifts+can+predict+disulfide+bond+formation&doi=10.1023%2Fa%3A1008398416292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">13C NMR chemical shifts can predict disulfide bond formation</span></div><div class="casAuthors">Sharma, Deepak; Rajarathnam, Krishna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">165-171</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">The presence of disulfide bonds can be detected unambiguously only by x-ray crystallog., and otherwise must be inferred by chem. methods.  In this study we demonstrate that 13C NMR chem. shifts are diagnostic of disulfide bond formation, and can discriminate between cysteine in the reduced (free) and oxidized (disulfide bonded) state.  A database of cysteine 13CCα and Cβ chem. shifts was constructed from the BioMagResBank (BMRB) and Sheffield databases, and published journals.  Statistical anal. indicated that the Cβ shift is extremely sensitive to the redox state, and can predict the disulfide-bonded state.  Further, chem. shifts in both states occupy distinct clusters as a function of secondary structure in the Cα/Cβ chem. shift map.  On the basis of these results, we provide simple ground rules for predicting the redox state of cysteines; these rules could be used effectively in NMR structure detn., predicting new folds, and in protein folding studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD37laLLTfO7Vg90H21EOLACvtfcHk0liF561qQb_cXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosVWgsL8%253D&md5=248353c6611b9b52e614a399260b4fd1</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008398416292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008398416292%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DD.%26aulast%3DRajarathnam%26aufirst%3DK.%26atitle%3D13C%2520NMR%2520chemical%2520shifts%2520can%2520predict%2520disulfide%2520bond%2520formation%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2000%26volume%3D18%26spage%3D165%26epage%3D171%26doi%3D10.1023%2Fa%3A1008398416292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolin, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwalbe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacArthur, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, C. M.</span></span> <span> </span><span class="NLM_article-title">Analysis of main chain torsion angles in proteins: prediction of NMR coupling constants for native and random coil conformations</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>255</i></span>,  <span class="NLM_fpage">494</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1006%2Fjmbi.1996.0041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=8568893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK28XnvVSjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=255&publication_year=1996&pages=494-506&author=L.+J.+Smithauthor=K.+A.+Bolinauthor=H.+Schwalbeauthor=M.+W.+MacArthurauthor=J.+M.+Thorntonauthor=C.+M.+Dobson&title=Analysis+of+main+chain+torsion+angles+in+proteins%3A+prediction+of+NMR+coupling+constants+for+native+and+random+coil+conformations&doi=10.1006%2Fjmbi.1996.0041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of main chain torsion angles in proteins: prediction of NMR coupling constants for native and random coil conformations</span></div><div class="casAuthors">Smith, Lorna J.; Bolin, Kimberly A.; Schwalbe, Harald; MacArthur, Malcolm W.; Thornton, Janet M.; Dobson, Christopher M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">255</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">494-506</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Using a database of 85 high resoln. protein crystal structures, the distributions of main chain torsion angles, both in secondary structure and in coil regions where no secondary structure is present, have been analyzed.  These torsion angle distributions have been used to predict NMR homonuclear and heteronuclear coupling consts. for residues in the secondary structure using known Karplus relationships.  For α helixes, 310 helixes and β strands mean predicted 3JHNα coupling consts. are 4.8, 5.6 and 8.5 Hz, resp.  These values differ significantly from those expected for the ideal .vphi. angles (3.9, 3.0 and 8.9 Hz; .vphi. = -57°, -49°, -139° for α and 310 helixes and β strands (antiparallel), resp.) in regular secondary structure, but agree well with available exptl. NMR data for nine proteins.  The crystallog. data set has also been used to provide a basis for interpreting coupling consts. measured for peptides and denatured proteins.  Using a model for a random coil, in which all residues adopt distributions of .vphi., ψ angles equiv. to those seen for residues in the coil regions of native folded proteins, predicted 3JHNα values for different residue types have been found to range from 5.9 Hz and 6.1 Hz for glycine and alanine, resp., to 7.7 Hz for valine.  A good correlation has been found between the predicted 3JHNα coupling consts. for this model and exptl. values for a set of peptides that other evidence suggests are highly unstructured.  For other peptides, however, deviations from the predictions of the model are clear and provided evidence for addnl. interactions within otherwise disordered states.  The values of homonuclear and heteronuclear coupling consts. derived from the protein database listed here therefore provide a basis not only for analyzing the secondary structure of native proteins in soln.but for assessing and interpreting the extent of structure present in peptides and non-native states of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5kHZ5tujTGbVg90H21EOLACvtfcHk0liF561qQb_cXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XnvVSjsw%253D%253D&md5=03a9c166a532053820da51ee2c564b95</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0041%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DL.%2BJ.%26aulast%3DBolin%26aufirst%3DK.%2BA.%26aulast%3DSchwalbe%26aufirst%3DH.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26aulast%3DDobson%26aufirst%3DC.%2BM.%26atitle%3DAnalysis%2520of%2520main%2520chain%2520torsion%2520angles%2520in%2520proteins%253A%2520prediction%2520of%2520NMR%2520coupling%2520constants%2520for%2520native%2520and%2520random%2520coil%2520conformations%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1996%26volume%3D255%26spage%3D494%26epage%3D506%26doi%3D10.1006%2Fjmbi.1996.0041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dominguez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boelens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonvin, A. M. J. J.</span></span> <span> </span><span class="NLM_article-title">HADDOCK: A Protein–Protein Docking Approach Based on Biochemical or Biophysical Information</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">1731</span>– <span class="NLM_lpage">1737</span>, <span class="refDoi"> DOI: 10.1021/ja026939x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja026939x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvFGquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=1731-1737&author=C.+Dominguezauthor=R.+Boelensauthor=A.+M.+J.+J.+Bonvin&title=HADDOCK%3A+A+Protein%E2%80%93Protein+Docking+Approach+Based+on+Biochemical+or+Biophysical+Information&doi=10.1021%2Fja026939x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">HADDOCK: A protein-protein docking approach based on biochemical or biophysical information</span></div><div class="casAuthors">Dominguez, Cyril; Boelens, Rolf; Bonvin, Alexandre M. J. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1731-1737</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure detn. of protein-protein complexes is a rather tedious and lengthy process, by both NMR and x-ray crystallog.  Several methods based on docking to study protein complexes have also been well developed over the past few years.  Most of these approaches are not driven by exptl. data but are based on a combination of energetics and shape complementarity. Here, we present an approach called HADDOCK (High Ambiguity Driven protein-protein Docking) that makes use of biochem. and/or biophys. interaction data such as chem. shift perturbation data resulting from NMR titrn. expts. or mutagenesis data.  This information is introduced as Ambiguous Interaction Restraints (AIRs) to drive the docking process.  An AIR is defined as an ambiguous distance between all residues shown to be involved in the interaction.  The accuracy of our approach is demonstrated with three mol. complexes.  For two of these complexes, for which both the complex and the free protein structures have been solved, NMR titrn. data were available.  Mutagenesis data were used in the last example.  In all cases, the best structures generated by HADDOCK, i.e., the structures with the lowest intermol. energies, were the closest to the published structure of the resp. complexes (within 2.0 Å backbone RMSD).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2GoXS6PbQz7Vg90H21EOLACvtfcHk0lgzRu1eJbqpLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvFGquw%253D%253D&md5=9cf6fd9c690afb0181579485ae263197</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fja026939x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja026939x%26sid%3Dliteratum%253Aachs%26aulast%3DDominguez%26aufirst%3DC.%26aulast%3DBoelens%26aufirst%3DR.%26aulast%3DBonvin%26aufirst%3DA.%2BM.%2BJ.%2BJ.%26atitle%3DHADDOCK%253A%2520A%2520Protein%25E2%2580%2593Protein%2520Docking%2520Approach%2520Based%2520on%2520Biochemical%2520or%2520Biophysical%2520Information%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D1731%26epage%3D1737%26doi%3D10.1021%2Fja026939x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Vries, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Dijk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonvin, A. M. J. J.</span></span> <span> </span><span class="NLM_article-title">The HADDOCK web server for data-driven biomolecular docking</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">883</span>– <span class="NLM_lpage">897</span>, <span class="refDoi"> DOI: 10.1038/nprot.2010.32</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Fnprot.2010.32" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=20431534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltl2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=883-897&author=S.+J.+de%0AVriesauthor=M.+van+Dijkauthor=A.+M.+J.+J.+Bonvin&title=The+HADDOCK+web+server+for+data-driven+biomolecular+docking&doi=10.1038%2Fnprot.2010.32"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The HADDOCK web server for data-driven biomolecular docking</span></div><div class="casAuthors">de Vries, Sjoerd J.; van Dijk, Marc; Bonvin, Alexandre M. J. J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">883-897</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Computational docking is the prediction or modeling of the three-dimensional structure of a biomol. complex, starting from the structures of the individual mols. in their free, unbound form.  HADDOCK is a popular docking program that takes a data-driven approach to docking, with support for a wide range of exptl. data.  Here the authors present the HADDOCK web server protocol, facilitating the modeling of biomol. complexes for a wide community.  The main web interface is user-friendly, requiring only the structures of the individual components and a list of interacting residues as input.  Addnl. web interfaces allow the more advanced user to exploit the full range of exptl. data supported by HADDOCK and to customize the docking process.  The HADDOCK server has access to the resources of a dedicated cluster and of the e-NMR GRID infrastructure.  Therefore, a typical docking run takes only a few minutes to prep. and a few hours to complete.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCBZ3XxzZgL7Vg90H21EOLACvtfcHk0lgzRu1eJbqpLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltl2qsLw%253D&md5=dbf801d99065e8d2c3aedc016209838c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2010.32&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2010.32%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BVries%26aufirst%3DS.%2BJ.%26aulast%3Dvan%2BDijk%26aufirst%3DM.%26aulast%3DBonvin%26aufirst%3DA.%2BM.%2BJ.%2BJ.%26atitle%3DThe%2520HADDOCK%2520web%2520server%2520for%2520data-driven%2520biomolecular%2520docking%26jtitle%3DNat.%2520Protoc.%26date%3D2010%26volume%3D5%26spage%3D883%26epage%3D897%26doi%3D10.1038%2Fnprot.2010.32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krüger, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span> <span> </span><span class="NLM_article-title">DrugScorePPI webserver: fast and accurate in silico alanine scanning for scoring protein-protein interactions</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">W480</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1093%2Fnar%2Fgkq471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=20511591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVSqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=W480&author=D.+M.+Kr%C3%BCgerauthor=H.+Gohlke&title=DrugScorePPI+webserver%3A+fast+and+accurate+in+silico+alanine+scanning+for+scoring+protein-protein+interactions&doi=10.1093%2Fnar%2Fgkq471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">DrugScorePPI webserver: fast and accurate in silico alanine scanning for scoring protein-protein interactions</span></div><div class="casAuthors">Krueger, Dennis M.; Gohlke, Holger</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W480-W486</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Protein-protein complexes play key roles in all cellular signal transduction processes.  We have developed a fast and accurate computational approach to predict changes in the binding free energy upon alanine mutations in protein-protein interfaces.  The approach is based on a knowledge-based scoring function, DrugScorePPI, for which pair potentials were derived from 851 complex structures and adapted against 309 exptl. alanine scanning results.  Based on this approach, we developed the DrugScorePPI webserver.  The input consists of a protein-protein complex structure; the output is a summary table and bar plot of binding free energy differences for wild-type residue-to-Ala mutations.  The results of the anal. are mapped on the protein-protein complex structure and visualized using J mol.  A single interface can be analyzed within a few minutes.  Our approach has been successfully validated by application to an external test set of 22 alanine mutations in the interface of Ras/RalGDS.  The DrugScorePPI webserver is primarily intended for identifying hot-spot residues in protein-protein interfaces, which provides valuable information for guiding biol. expts. and in the development of protein-protein interaction modulators.  The DrugScorePPI Webserver, accessible at http://cpclab.uni-duesseldorf.de/dsppi, is free and open to all users with no login requirement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmnID8v_JPHLVg90H21EOLACvtfcHk0lgzRu1eJbqpLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVSqt7g%253D&md5=5c55172fecbec2710fe0e53b45a28a1c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq471%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25BCger%26aufirst%3DD.%2BM.%26aulast%3DGohlke%26aufirst%3DH.%26atitle%3DDrugScorePPI%2520webserver%253A%2520fast%2520and%2520accurate%2520in%2520silico%2520alanine%2520scanning%2520for%2520scoring%2520protein-protein%2520interactions%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3DW480%26doi%3D10.1093%2Fnar%2Fgkq471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Böttger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knappe, D.</span></span> <span> </span><span class="NLM_article-title">Differential stability of therapeutic peptides with different proteolytic cleavage sites in blood, plasma and serum</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0178943</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0178943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1371%2Fjournal.pone.0178943" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=28575099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1Cjtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=R.+B%C3%B6ttgerauthor=R.+Hoffmannauthor=D.+Knappe&title=Differential+stability+of+therapeutic+peptides+with+different+proteolytic+cleavage+sites+in+blood%2C+plasma+and+serum&doi=10.1371%2Fjournal.pone.0178943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Differential stability of therapeutic peptides with different proteolytic cleavage sites in blood, plasma and serum</span></div><div class="casAuthors">Bottger, Roland; Hoffmann, Ralf; Knappe, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e0178943/1-e0178943/15</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Proteolytic degrdn. of peptide-based drugs is often considered as major weakness limiting systemic therapeutic applications.  Therefore, huge efforts are typically devoted to stabilize sequences against proteases present in serum or plasma, obtained as supernatants after complete blood coagulation or centrifugation of blood supplemented with anticoagulants, resp.  Plasma and serum are reproducibly obtained from animals and humans allowing consistent for clin. analyses and research applications.  However, the spectrum of active or activated proteases appears to vary depending on the activation of proteases and cofactors during coagulation (serum) or inhibition of such enzymes by anticoagulants (plasma), such as EDTA (metallo- and Ca2+-dependent proteases) and heparin (e.g. thrombin, factor Xa).  Here, we studied the presumed effects on peptide degrdn. by taking blood via cardiac puncture of CD-1 mice using a syringe contg. a peptide soln.  Due to absence of coagulation activators (e.g. glass surfaces and damaged cells), visible blood clotting was prevented allowing to study peptide degrdn. for one hour.  The remaining peptide was quantified and the degrdn. products were identified using mass spectrometry.  When the degrdn. rates (half-life times) were compared to serum derived freshly from the same animal and com. serum and plasma samples, peptides of three different families showed indeed considerably different stabilities.  Generally, peptides were faster degraded in serum than in plasma, but surprisingly all peptides were more stable in fresh blood and the order of degrdn. rates among the peptides varied among the six different incubation expts.  This indicates, that proteolytic degrdn. of peptide-based therapeutics may often be misleading stimulating efforts to stabilize peptides at degrdn. sites relevant only in vitro, i.e., for serum or plasma stability assays, but of lower importance in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT8Seu6LHCzLVg90H21EOLACvtfcHk0lgzRu1eJbqpLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1Cjtrg%253D&md5=cbf5555f8b11a6fedf08a240297365da</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0178943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0178943%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6ttger%26aufirst%3DR.%26aulast%3DHoffmann%26aufirst%3DR.%26aulast%3DKnappe%26aufirst%3DD.%26atitle%3DDifferential%2520stability%2520of%2520therapeutic%2520peptides%2520with%2520different%2520proteolytic%2520cleavage%2520sites%2520in%2520blood%252C%2520plasma%2520and%2520serum%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0178943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aneja, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaiken, I.</span></span> <span> </span><span class="NLM_article-title">Click chemistry in peptide-based drug design</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">9797</span>– <span class="NLM_lpage">9817</span>, <span class="refDoi"> DOI: 10.3390/molecules18089797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.3390%2Fmolecules18089797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=23959192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlKlsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=9797-9817&author=H.+Liauthor=R.+Anejaauthor=I.+Chaiken&title=Click+chemistry+in+peptide-based+drug+design&doi=10.3390%2Fmolecules18089797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Click chemistry in peptide-based drug design</span></div><div class="casAuthors">Li, Huiyuan; Aneja, Rachna; Chaiken, Irwin</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">9797-9817</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Click chem. is an efficient and chemoselective synthetic method for coupling mol. fragments under mild reaction conditions.  Since the advent in 2001 of methods to improve stereochem. conservation, the click chem. approach has been broadly used to construct diverse chemotypes in both chem. and biol. fields.  In this review, we discuss the application of click chem. in peptide-based drug design.  We highlight how triazoles formed by click reactions have been used for mimicking peptide and disulfide bonds, building secondary structural components of peptides, linking functional groups together, and bioconjugation.  The progress made in this field opens the way for synthetic approaches to convert peptides with promising functional leads into structure-minimized and more stable forms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo3QhPT6JMbrVg90H21EOLACvtfcHk0li4QKno3ckVfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlKlsrbN&md5=cc7cfed7b6249b9ce98a7051def886ee</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.3390%2Fmolecules18089797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules18089797%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DAneja%26aufirst%3DR.%26aulast%3DChaiken%26aufirst%3DI.%26atitle%3DClick%2520chemistry%2520in%2520peptide-based%2520drug%2520design%26jtitle%3DMolecules%26date%3D2013%26volume%3D18%26spage%3D9797%26epage%3D9817%26doi%3D10.3390%2Fmolecules18089797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caporale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandomenico, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvo, M.</span></span> <span> </span><span class="NLM_article-title">Automatic procedures for the synthesis of difficult peptides using oxyma as activating reagent: A comparative study on the use of bases and on different deprotection and agitation conditions</span>. <i>Peptides</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/j.peptides.2018.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2Fj.peptides.2018.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=29486214" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjvVClsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2018&pages=38-46&author=A.+Caporaleauthor=N.+Dotiauthor=A.+Montiauthor=A.+Sandomenicoauthor=M.+Ruvo&title=Automatic+procedures+for+the+synthesis+of+difficult+peptides+using+oxyma+as+activating+reagent%3A+A+comparative+study+on+the+use+of+bases+and+on+different+deprotection+and+agitation+conditions&doi=10.1016%2Fj.peptides.2018.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic procedures for the synthesis of difficult peptides using oxyma as activating reagent: A comparative study on the use of bases and on different deprotection and agitation conditions</span></div><div class="casAuthors">Caporale, A.; Doti, N.; Monti, A.; Sandomenico, A.; Ruvo, M.</div><div class="citationInfo"><span class="NLM_cas:title">Peptides (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">38-46</span>CODEN:
                <span class="NLM_cas:coden">PPTDD5</span>;
        ISSN:<span class="NLM_cas:issn">0196-9781</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Solid-Phase Peptide Synthesis (SPPS) is a rapid and efficient methodol. for the chem. synthesis of peptides and small proteins.  However, the assembly of peptide sequences classified as "difficult" poses severe synthetic problems in SPPS for the occurrence of extensive aggregation of growing peptide chains which often leads to synthesis failure.  In this framework, we have investigated the impact of different synthetic procedures on the yield and final purity of three well-known "difficult peptides" prepd. using oxyma as additive for the coupling steps.  In particular, we have comparatively investigated the use of piperidine and morpholine/DBU as deprotection reagents, the addn. of DIPEA, collidine and N-methylmorpholine as bases to the coupling reagent.  Moreover, the effect of different agitation modalities during the acylation reactions has been investigated.  Data obtained represent a step forward in optimizing strategies for the synthesis of "difficult peptides".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot8kFiXgO1VLVg90H21EOLACvtfcHk0li4QKno3ckVfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjvVClsbw%253D&md5=e62fa59f680ab328407d6992db13b802</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.peptides.2018.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.peptides.2018.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DCaporale%26aufirst%3DA.%26aulast%3DDoti%26aufirst%3DN.%26aulast%3DMonti%26aufirst%3DA.%26aulast%3DSandomenico%26aufirst%3DA.%26aulast%3DRuvo%26aufirst%3DM.%26atitle%3DAutomatic%2520procedures%2520for%2520the%2520synthesis%2520of%2520difficult%2520peptides%2520using%2520oxyma%2520as%2520activating%2520reagent%253A%2520A%2520comparative%2520study%2520on%2520the%2520use%2520of%2520bases%2520and%2520on%2520different%2520deprotection%2520and%2520agitation%2520conditions%26jtitle%3DPeptides%26date%3D2018%26volume%3D102%26spage%3D38%26epage%3D46%26doi%3D10.1016%2Fj.peptides.2018.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caporale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandomenico, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvo, M.</span></span> <span> </span><span class="NLM_article-title">Evaluation of combined use of Oxyma and HATU in aggregating peptide sequences</span>. <i>J. Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">272</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1002/psc.2977</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1002%2Fpsc.2977" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=28139012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslegtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=272-281&author=A.+Caporaleauthor=N.+Dotiauthor=A.+Sandomenicoauthor=M.+Ruvo&title=Evaluation+of+combined+use+of+Oxyma+and+HATU+in+aggregating+peptide+sequences&doi=10.1002%2Fpsc.2977"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of combined use of Oxyma and HATU in aggregating peptide sequences</span></div><div class="casAuthors">Caporale, Andrea; Doti, Nunzianna; Sandomenico, Annamaria; Ruvo, Menotti</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Peptide Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">272-281</span>CODEN:
                <span class="NLM_cas:coden">JPSIEI</span>;
        ISSN:<span class="NLM_cas:issn">1075-2617</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Polypeptides are finding increasing applications as therapeutics because of their specificity that often translates into excellent safety, tolerability, and efficacy profiles in humans.  New synthetic methodologies for their prepn. are thereby continuously sought to reduce the costs assocd. to chain assembly and purifn.  Although solid-phase peptide synthesis has become one of the most advanced synthetic procedures at both lab. and industrial scale, the process is often complicated by aggregation phenomena originating from the combined occurrence of intermol. and intramol. hydrogen bonding, hydrophobic interactions, or other effects.  Altogether, these effects cause accumulation of many side products and synthetic mixts. extremely hard to sep. and purify, strongly affecting the costs of the final material.  In the attempt to optimize the coupling steps of some well-known aggregating or otherwise difficult to obtain peptides, we have comparatively investigated the use of Oxyma/DIC and HATU/Sym-collidine as second coupling reagents in double coupling settings for the prepn. of some model peptides.  Comparative anal. data obtained on the unpurified products with the two different protocols clearly show that the use of Oxyma/DIC largely improves the content of the target mols. in the final crude materials, making the synthesis more convenient and cost-effective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotT_SozIStk7Vg90H21EOLACvtfcHk0li4QKno3ckVfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslegtb4%253D&md5=99bf6ed623303d83a15d147bb38087f6</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fpsc.2977&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsc.2977%26sid%3Dliteratum%253Aachs%26aulast%3DCaporale%26aufirst%3DA.%26aulast%3DDoti%26aufirst%3DN.%26aulast%3DSandomenico%26aufirst%3DA.%26aulast%3DRuvo%26aufirst%3DM.%26atitle%3DEvaluation%2520of%2520combined%2520use%2520of%2520Oxyma%2520and%2520HATU%2520in%2520aggregating%2520peptide%2520sequences%26jtitle%3DJ.%2520Pept.%2520Sci.%26date%3D2017%26volume%3D23%26spage%3D272%26epage%3D281%26doi%3D10.1002%2Fpsc.2977" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scopes, R. K.</span></span> <span> </span><span class="NLM_article-title">Measurement of protein by spectrophotometry at 205 nm</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1974</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">282</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(74)90034-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2F0003-2697%2874%2990034-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=4407487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaE2cXksVOjsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=1974&pages=277-282&author=R.+K.+Scopes&title=Measurement+of+protein+by+spectrophotometry+at+205+nm&doi=10.1016%2F0003-2697%2874%2990034-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Measurement of protein by spectrophotometry at 205nm</span></div><div class="casAuthors">Scopes, R. K.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1974</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">277-82</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A method was described for the measurement of protein concn. by using the peptide bond absorption at 205 nm.  The extinction coeff. (ε) at 205 nm was estd., allowing for the absorption due to tryptophan and tyrosine residues, by measuring the absorbance at 280 nm as well as at 205 nm.  The estd. ε205 was compared with the actual ε205 for a no. of proteins, the mean error being <2%.  This was about 3 times better than using an av. ε2051mg/ml of 31 and approached the range of exptl. error inherent in any method of protein estn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFO6zpXSyQvrVg90H21EOLACvtfcHk0lgCoXNF82QegQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXksVOjsb0%253D&md5=ff53a6ad935bde82fc8c8a60bb606760</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2874%2990034-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252874%252990034-7%26sid%3Dliteratum%253Aachs%26aulast%3DScopes%26aufirst%3DR.%2BK.%26atitle%3DMeasurement%2520of%2520protein%2520by%2520spectrophotometry%2520at%2520205%2520nm%26jtitle%3DAnal.%2520Biochem.%26date%3D1974%26volume%3D59%26spage%3D277%26epage%3D282%26doi%3D10.1016%2F0003-2697%2874%2990034-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Braunschweiler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, R. R.</span></span> <span> </span><span class="NLM_article-title">Coherence transfer by isotropic mixing: Application to proton correlation spectroscopy</span>. <i>J. Magn. Reson.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">528</span>, <span class="refDoi"> DOI: 10.1016/0022-2364(83)90226-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2F0022-2364%2883%2990226-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaL3sXks1eju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1983&pages=521-528&author=R.+Braunschweilerauthor=R.+R.+Ernst&title=Coherence+transfer+by+isotropic+mixing%3A+Application+to+proton+correlation+spectroscopy&doi=10.1016%2F0022-2364%2883%2990226-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Coherence transfer by isotropic mixing:  application to proton correlation spectroscopy</span></div><div class="casAuthors">Braunschweiler, L.; Ernst, R. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Magnetic Resonance (1969-1992)</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">521-8</span>CODEN:
                <span class="NLM_cas:coden">JOMRA4</span>;
        ISSN:<span class="NLM_cas:issn">0022-2364</span>.
    </div><div class="casAbstract">A new mechanism of coherence transfer is proposed which is useful for 2-dimensional correlation spectroscopy in liqs.  Cross-peaks are generated between all members of a coupled spin network leading to 4 most complete correlation diagram.  The scheme produces net coherence transfer and can be arranged to create pure absorption mode spectra with peaks of pos. intensity.  A new type of 2-dimensional correlation NMR expt. is illustrated.  After a prepn. pulse the spin system is allowed to evolve freely under the (assumed) weak coupling Hamiltonian in terms of single spin modes.  For the mixing process the Hamiltonian is modified to a form which is dominated by the full isotropic coupling terms.  The coherence of the spins is periodically exchanged during the mixing time and is obsd. during the detection period again under the unperturbed Hamiltonian.  Illustrative examples of 2-dimensional correlated spectra with and without relayed peaks are shown.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo25_GwCJe3XrVg90H21EOLACvtfcHk0lgCoXNF82QegQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXks1eju7c%253D&md5=693eaaee37b0224796b7d9e3403759c6</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2F0022-2364%2883%2990226-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2364%252883%252990226-3%26sid%3Dliteratum%253Aachs%26aulast%3DBraunschweiler%26aufirst%3DR.%26aulast%3DErnst%26aufirst%3DR.%2BR.%26atitle%3DCoherence%2520transfer%2520by%2520isotropic%2520mixing%253A%2520Application%2520to%2520proton%2520correlation%2520spectroscopy%26jtitle%3DJ.%2520Magn.%2520Reson.%26date%3D1983%26volume%3D53%26spage%3D521%26epage%3D528%26doi%3D10.1016%2F0022-2364%2883%2990226-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wüthrich, K.</span></span> <span> </span><span class="NLM_article-title">A two-dimensional nuclear Overhauser enhancement (2D NOE) experiment for the elucidation of complete proton-proton cross-relaxation networks in biological macromolecules</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">6</span>, <span class="refDoi"> DOI: 10.1016/0006-291x(80)90695-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2F0006-291X%2880%2990695-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=7417242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaL3cXlsVGlu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1980&pages=1-6&author=A.+Kumarauthor=R.+R.+Ernstauthor=K.+W%C3%BCthrich&title=A+two-dimensional+nuclear+Overhauser+enhancement+%282D+NOE%29+experiment+for+the+elucidation+of+complete+proton-proton+cross-relaxation+networks+in+biological+macromolecules&doi=10.1016%2F0006-291x%2880%2990695-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">A two-dimensional nuclear Overhauser enhancement (2D NOE) experiment for the elucidation of complete proton-proton cross-relaxation networks in biological macromolecules</span></div><div class="casAuthors">Kumar, Anil; Ernst, R. R.; Wuethrich, K.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-6</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    </div><div class="casAbstract">The recently developed technique of 2D cross-relaxation spectroscopy was used for systematic measurements of selective NOE in the high-resoln. 1H NMR spectra of biol. macromols. in soln.  Compared to conventional 1-dimensional NOE studies, the 2D NOE expt. has the principal advantage that it avoids detrimental effects arising from the limited selectivity of preirradn. in crowded spectral regions.  Furthermore, it yields, with a single instrument setting, a complete network of NOEs between all the protons in the macromol.  The resulting information on intramol. proton-proton distances provides a new avenue for studies of the spatial structures of biopolymers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiH0XrevG7UbVg90H21EOLACvtfcHk0lgCoXNF82QegQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXlsVGlu7Y%253D&md5=fe460f61af8426f88ba1aaec0efd6bfa</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2F0006-291X%2880%2990695-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-291X%252880%252990695-6%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DErnst%26aufirst%3DR.%2BR.%26aulast%3DW%25C3%25BCthrich%26aufirst%3DK.%26atitle%3DA%2520two-dimensional%2520nuclear%2520Overhauser%2520enhancement%2520%25282D%2520NOE%2529%2520experiment%2520for%2520the%2520elucidation%2520of%2520complete%2520proton-proton%2520cross-relaxation%2520networks%2520in%2520biological%2520macromolecules%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1980%26volume%3D95%26spage%3D1%26epage%3D6%26doi%3D10.1016%2F0006-291x%2880%2990695-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Griesinger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, R. R.</span></span> <span> </span><span class="NLM_article-title">Frequency Offset Effects and Their Elimination in NMR Rotating-Frame Cross-Relaxation Spectroscopy</span>. <i>J. Magn. Reson.</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">271</span>, <span class="refDoi"> DOI: 10.1016/0022-2364(87)90035-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2F0022-2364%2887%2990035-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaL1cXos1GksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=1987&pages=261-271&author=C.+Griesingerauthor=R.+R.+Ernst&title=Frequency+Offset+Effects+and+Their+Elimination+in+NMR+Rotating-Frame+Cross-Relaxation+Spectroscopy&doi=10.1016%2F0022-2364%2887%2990035-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Frequency offset effects and their elimination in NMR rotating-frame cross-relaxation spectroscopy</span></div><div class="casAuthors">Griesinger, C.; Ernst, R. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Magnetic Resonance (1969-1992)</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-71</span>CODEN:
                <span class="NLM_cas:coden">JOMRA4</span>;
        ISSN:<span class="NLM_cas:issn">0022-2364</span>.
    </div><div class="casAbstract">The effects of frequency offsets in NMR rotating-frame cross-relaxation expts. are analyzed.  A quant. explanation is given for the offset dependence of peak intensities and multiplet structure in 2-dimensional rotating-frame Overhauser effect spectroscopy.  A modified pulse sequence is proposed to eliminate offset effects and to improve the signal-to-noise ratio.  The spectra of cyclo(-D-Ala1-Ala4-) is then given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryswO43JPv2bVg90H21EOLACvtfcHk0lgCoXNF82QegQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXos1GksA%253D%253D&md5=276b634ddc995061f0d8931081b18949</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2F0022-2364%2887%2990035-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2364%252887%252990035-7%26sid%3Dliteratum%253Aachs%26aulast%3DGriesinger%26aufirst%3DC.%26aulast%3DErnst%26aufirst%3DR.%2BR.%26atitle%3DFrequency%2520Offset%2520Effects%2520and%2520Their%2520Elimination%2520in%2520NMR%2520Rotating-Frame%2520Cross-Relaxation%2520Spectroscopy%26jtitle%3DJ.%2520Magn.%2520Reson.%26date%3D1987%26volume%3D75%26spage%3D261%26epage%3D271%26doi%3D10.1016%2F0022-2364%2887%2990035-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otlewski, J.</span></span> <span> </span><span class="NLM_article-title">Amide proton temperature coefficients as hydrogen bond indicators in proteins</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">261</span>, <span class="refDoi"> DOI: 10.1023/a:1012911329730</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1023%2FA%3A1012911329730" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=11775741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3MXptlKrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2001&pages=249-261&author=T.+Cierpickiauthor=J.+Otlewski&title=Amide+proton+temperature+coefficients+as+hydrogen+bond+indicators+in+proteins&doi=10.1023%2Fa%3A1012911329730"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Amide proton temperature coefficients as hydrogen bond indicators in proteins</span></div><div class="casAuthors">Cierpicki, Tomasz; Otlewski, Jacek</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">249-261</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">Correlations between amide proton temp. coeffs. (ΔσHN/ΔT) and hydrogen bonds were investigated for a data set of 793 amides derived from 14 proteins.  For amide protons showing temp. gradients more pos. than -4.6 ppb/K there is a hydrogen bond predictivity value exceeding 85%.  It increases to over 93% for amides within the range between -4 and -1 ppb/K.  Detailed anal. shows an inverse proportionality between amide proton temp. coeffs. and hydrogen bond lengths.  Furthermore, for hydrogen bonds of similar bond lengths, values of temp. gradients in α-helixes are on av. 1 ppb/K more neg. than in β-sheets.  In consequence, a no. of amide protons in α-helixes involved in hydrogen bonds shorter than 2 Å show ΔσHN/ΔT < -4.6 ppb/K.  Due to longer hydrogen bonds, 90% of amides in 310 helixes and 98% in β-turns have temp. coeffs. more pos. than -4.6ppb/K.  Ring current effect also significantly influences temp. coeffs. of amide protons.  In seven out of eight cases non-hydrogen bonded amides strongly deshielded by neighboring arom. rings show temp. coeffs. more pos. than -2 ppb/K.  In general, amide proton temp. gradients do not change with pH unless they correspond to conformational changes.  Three examples of pH dependent equil. showing hydrogen bond formation at higher pH were found.  In conclusion, amide proton temp. coeffs. offer an attractive and simple way to confirm existence of hydrogen bonds in NMR detd. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-yCsJE1GrlbVg90H21EOLACvtfcHk0litSdkzrfyYwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXptlKrs7g%253D&md5=d0662943f22587f95e2be3074503b33d</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1023%2FA%3A1012911329730&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1012911329730%26sid%3Dliteratum%253Aachs%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DOtlewski%26aufirst%3DJ.%26atitle%3DAmide%2520proton%2520temperature%2520coefficients%2520as%2520hydrogen%2520bond%2520indicators%2520in%2520proteins%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2001%26volume%3D21%26spage%3D249%26epage%3D261%26doi%3D10.1023%2Fa%3A1012911329730" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhukov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otlewski, J.</span></span> <span> </span><span class="NLM_article-title">Hydrogen bonds in human ubiquitin reflected in temperature coefficients of amide protons</span>. <i>J. Magn. Reson.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">178</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1006/jmre.2002.2597</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1006%2Fjmre.2002.2597" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12323135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD38XntFShtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2002&pages=178-180&author=T.+Cierpickiauthor=I.+Zhukovauthor=R.+A.+Byrdauthor=J.+Otlewski&title=Hydrogen+bonds+in+human+ubiquitin+reflected+in+temperature+coefficients+of+amide+protons&doi=10.1006%2Fjmre.2002.2597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen Bonds in Human Ubiquitin Reflected in Temperature Coefficients of Amide Protons</span></div><div class="casAuthors">Cierpicki, Tomasz; Zhukov, Igor; Byrd, R. Andrew; Otlewski, Jacek</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Magnetic Resonance</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">178-180</span>CODEN:
                <span class="NLM_cas:coden">JMARF3</span>;
        ISSN:<span class="NLM_cas:issn">1090-7807</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">Anal. of amide proton temp. coeffs. (ΔσHN/ΔT) in human ubiquitin shows their usefulness in indicating hydrogen bonds.  The availability of a very accurate soln. structure of ubiquitin enables the precise detn. of hydrogen bonds and increases the reliability of the anal. of chem. shift temp. gradients.  Values of ΔσHN/ΔT more pos. than -4.6 ppb/K are very good indicators of hydrogen bonds.  Addnl., a weak temp. dependence of non-hydrogen-bonded amides was obsd. for amide protons that are significantly shifted upfield.  We obsd. that temp. gradients of amide protons involved in short hydrogen bonds are related to donor-acceptor distances.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2qiprxXj5ubVg90H21EOLACvtfcHk0litSdkzrfyYwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntFShtLs%253D&md5=02801090d9f21616c8741d7e6dcca29a</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1006%2Fjmre.2002.2597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmre.2002.2597%26sid%3Dliteratum%253Aachs%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DZhukov%26aufirst%3DI.%26aulast%3DByrd%26aufirst%3DR.%2BA.%26aulast%3DOtlewski%26aufirst%3DJ.%26atitle%3DHydrogen%2520bonds%2520in%2520human%2520ubiquitin%2520reflected%2520in%2520temperature%2520coefficients%2520of%2520amide%2520protons%26jtitle%3DJ.%2520Magn.%2520Reson.%26date%3D2002%26volume%3D157%26spage%3D178%26epage%3D180%26doi%3D10.1006%2Fjmre.2002.2597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delaglio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzesiek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuister, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeifer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, A.</span></span> <span> </span><span class="NLM_article-title">NMRPipe: a multidimensional spectral processing system based on UNIX pipes</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1007/BF00197809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2FBF00197809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=8520220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1995&pages=277-293&author=F.+Delaglioauthor=S.+Grzesiekauthor=G.+W.+Vuisterauthor=G.+Zhuauthor=J.+Pfeiferauthor=A.+Bax&title=NMRPipe%3A+a+multidimensional+spectral+processing+system+based+on+UNIX+pipes&doi=10.1007%2FBF00197809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">NMRPipe: a multidimensional spectral processing system based on UNIX pipes</span></div><div class="casAuthors">Delaglio, Frank; Grzesiek, Stephan; Vuister, Geerten W.; Zhu, Guang; Pfeifer, John; Bax, Ad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-93</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">The NMRPipe system is a UNIX software environment of processing, graphics, and anal. tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments.  The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control.  In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction.  Complete multidimensional processing schemes are constructed as simple UNIX shell scripts.  The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing.  The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSIUdXrdcEgbVg90H21EOLACvtfcHk0litSdkzrfyYwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK&md5=a670fca5b164083e2178fafd2fb951ff</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2FBF00197809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00197809%26sid%3Dliteratum%253Aachs%26aulast%3DDelaglio%26aufirst%3DF.%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DVuister%26aufirst%3DG.%2BW.%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DPfeifer%26aufirst%3DJ.%26aulast%3DBax%26aufirst%3DA.%26atitle%3DNMRPipe%253A%2520a%2520multidimensional%2520spectral%2520processing%2520system%2520based%2520on%2520UNIX%2520pipes%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1995%26volume%3D6%26spage%3D277%26epage%3D293%26doi%3D10.1007%2FBF00197809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span>; <span class="NLM_string-name">Kneller, D. G.</span></span> <i>Sparky 3</i>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>, <span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=T.+D.+Goddard&author=D.+G.+Kneller&title=Sparky+3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26btitle%3DSparky%25203%26pub%3DUniversity%2520of%2520California%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, R. L. J.</span></span> <i>Optimizing the Process of Nuclear Magnetic Resonance Spectrum Analysis and Computer Aided Resonance Assignment Dissertation</i>; <span class="NLM_publisher-name">Swiss Federal Institute of Technology</span>: <span class="NLM_publisher-loc">Zürich</span>, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=R.+L.+J.+Keller&title=Optimizing+the+Process+of+Nuclear+Magnetic+Resonance+Spectrum+Analysis+and+Computer+Aided+Resonance+Assignment+Dissertation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DR.%2BL.%2BJ.%26btitle%3DOptimizing%2520the%2520Process%2520of%2520Nuclear%2520Magnetic%2520Resonance%2520Spectrum%2520Analysis%2520and%2520Computer%2520Aided%2520Resonance%2520Assignment%2520Dissertation%26pub%3DSwiss%2520Federal%2520Institute%2520of%2520Technology%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berjanskii, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span> <span> </span><span class="NLM_article-title">A simple method to predict protein flexibility using secondary chemical shifts</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">14970</span>– <span class="NLM_lpage">14971</span>, <span class="refDoi"> DOI: 10.1021/ja054842f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja054842f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKhsb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=14970-14971&author=M.+V.+Berjanskiiauthor=D.+S.+Wishart&title=A+simple+method+to+predict+protein+flexibility+using+secondary+chemical+shifts&doi=10.1021%2Fja054842f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">A Simple Method To Predict Protein Flexibility Using Secondary Chemical Shifts</span></div><div class="casAuthors">Berjanskii, Mark V.; Wishart, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">14970-14971</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein motions play a crit. role in many biol. processes, such as enzyme catalysis, allosteric regulation, antigen-antibody interactions, and protein-DNA binding.  NMR spectroscopy occupies a unique place among methods for investigating protein dynamics due to its ability to provide site-specific information about protein motions over a large range of time scales.  However, most NMR methods require a detailed knowledge of the 3D structure and/or the collection of addnl. exptl. data (NOEs, T1, T2, etc.) to accurately measure protein dynamics.  Here the authors present a simple method based on chem. shift data that allows accurate, quant., site-specific mapping of protein backbone mobility without the need of a three-dimensional structure or the collection and anal. of NMR relaxation data.  Further, the authors show that this chem. shift method is able to quant. predict per-residue RMSD values (from both MD simulations and NMR structural ensembles) as well as model-free backbone order parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjDPJ5T3Wtx7Vg90H21EOLACvtfcHk0lgO7JraMlA_mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKhsb7J&md5=26b0104fc58b338a3c01f1b97aaadf1c</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fja054842f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja054842f%26sid%3Dliteratum%253Aachs%26aulast%3DBerjanskii%26aufirst%3DM.%2BV.%26aulast%3DWishart%26aufirst%3DD.%2BS.%26atitle%3DA%2520simple%2520method%2520to%2520predict%2520protein%2520flexibility%2520using%2520secondary%2520chemical%2520shifts%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D14970%26epage%3D14971%26doi%3D10.1021%2Fja054842f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berjanskii, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span> <span> </span><span class="NLM_article-title">The RCI server: rapid and accurate calculation of protein flexibility using chemical shifts</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">W531</span>– <span class="NLM_lpage">W537</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1093%2Fnar%2Fgkm328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=17485469" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W531-W537&author=M.+V.+Berjanskiiauthor=D.+S.+Wishart&title=The+RCI+server%3A+rapid+and+accurate+calculation+of+protein+flexibility+using+chemical+shifts&doi=10.1093%2Fnar%2Fgkm328"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm328%26sid%3Dliteratum%253Aachs%26aulast%3DBerjanskii%26aufirst%3DM.%2BV.%26aulast%3DWishart%26aufirst%3DD.%2BS.%26atitle%3DThe%2520RCI%2520server%253A%2520rapid%2520and%2520accurate%2520calculation%2520of%2520protein%2520flexibility%2520using%2520chemical%2520shifts%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW531%26epage%3DW537%26doi%3D10.1093%2Fnar%2Fgkm328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Güntert, P.</span></span> <span> </span><span class="NLM_article-title">Automated structure determination from NMR spectra</span>. <i>Eur. Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">143</span>, <span class="refDoi"> DOI: 10.1007/s00249-008-0367-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2Fs00249-008-0367-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=18807026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotVelug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2009&pages=129-143&author=P.+G%C3%BCntert&title=Automated+structure+determination+from+NMR+spectra&doi=10.1007%2Fs00249-008-0367-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Automated structure determination from NMR spectra</span></div><div class="casAuthors">Guentert, Peter</div><div class="citationInfo"><span class="NLM_cas:title">European Biophysics Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">129-143</span>CODEN:
                <span class="NLM_cas:coden">EBJOE8</span>;
        ISSN:<span class="NLM_cas:issn">0175-7571</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Automated methods for protein structure detn. by NMR have increasingly gained acceptance and are now widely used for the automated assignment of distance restraints and the calcn. of three-dimensional structures.  This review gives an overview of the techniques for automated protein structure anal. by NMR, including both NOE-based approaches and methods relying on other exptl. data such as residual dipolar couplings and chem. shifts, and presents the FLYA algorithm for the fully automated NMR structure detn. of proteins that is suitable to substitute all manual spectra anal. and thus overcomes a major efficiency limitation of the NMR method for protein structure detn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOooOL4oujYrVg90H21EOLACvtfcHk0lgO7JraMlA_mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotVelug%253D%253D&md5=93eaa09a8420d2a1fb583806c896c3d2</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1007%2Fs00249-008-0367-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00249-008-0367-z%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25BCntert%26aufirst%3DP.%26atitle%3DAutomated%2520structure%2520determination%2520from%2520NMR%2520spectra%26jtitle%3DEur.%2520Biophys.%2520J.%26date%3D2009%26volume%3D38%26spage%3D129%26epage%3D143%26doi%3D10.1007%2Fs00249-008-0367-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaglio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornilescu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, A.</span></span> <span> </span><span class="NLM_article-title">TALOS+: a hybrid method for predicting protein backbone torsion angles from NMR chemical shifts</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">223</span>, <span class="refDoi"> DOI: 10.1007/s10858-009-9333-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2Fs10858-009-9333-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=19548092" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD1MXovFajtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2009&pages=213-223&author=Y.+Shenauthor=F.+Delaglioauthor=G.+Cornilescuauthor=A.+Bax&title=TALOS%2B%3A+a+hybrid+method+for+predicting+protein+backbone+torsion+angles+from+NMR+chemical+shifts&doi=10.1007%2Fs10858-009-9333-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">TALOS+: a hybrid method for predicting protein backbone torsion angles from NMR chemical shifts</span></div><div class="casAuthors">Shen, Yang; Delaglio, Frank; Cornilescu, Gabriel; Bax, Ad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">213-223</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">NMR chem. shifts in proteins depend strongly on local structure.  The program TALOS establishes an empirical relation between 13C, 15N and 1H chem. shifts and backbone torsion angles .vphi. and ψ.  Extension of the original 20-protein database to 200 proteins increased the fraction of residues for which backbone angles could be predicted from 65 to 74%, while reducing the error rate from 3 to 2.5%.  Addn. of a two-layer neural network filter to the database fragment selection process forms the basis for a new program, TALOS+, which further enhances the prediction rate to 88.5%, without increasing the error rate.  Excluding the 2.5% of residues for which TALOS+ makes predictions that strongly differ from those obsd. in the cryst. state, the accuracy of predicted .vphi. and ψ angles, equals ±13°.  Large discrepancies between predictions and crystal structures are primarily limited to loop regions, and for the few cases where multiple X-ray structures are available such residues are often found in different states in the different structures.  The TALOS+ output includes predictions for individual residues with missing chem. shifts, and the neural network component of the program also predicts secondary structure with good accuracy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJZxR7vg_rarVg90H21EOLACvtfcHk0lgO7JraMlA_mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXovFajtr8%253D&md5=ee3f8647a02e19937988056fd7af04b2</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2Fs10858-009-9333-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10858-009-9333-z%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DDelaglio%26aufirst%3DF.%26aulast%3DCornilescu%26aufirst%3DG.%26aulast%3DBax%26aufirst%3DA.%26atitle%3DTALOS%252B%253A%2520a%2520hybrid%2520method%2520for%2520predicting%2520protein%2520backbone%2520torsion%2520angles%2520from%2520NMR%2520chemical%2520shifts%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2009%26volume%3D44%26spage%3D213%26epage%3D223%26doi%3D10.1007%2Fs10858-009-9333-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koradi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billeter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wüthrich, K.</span></span> <span> </span><span class="NLM_article-title">MOLMOL: a program for display and analysis of macromolecular structures</span>. <i>J. Mol. Graph.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1016/0263-7855(96)00009-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2F0263-7855%2896%2900009-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=8744573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK28Xis12nsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=51-55&author=R.+Koradiauthor=M.+Billeterauthor=K.+W%C3%BCthrich&title=MOLMOL%3A+a+program+for+display+and+analysis+of+macromolecular+structures&doi=10.1016%2F0263-7855%2896%2900009-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">MOLMOL: a program for display and analysis of macromolecular structures</span></div><div class="casAuthors">Koradi, Reto; Billeter, Martin; Wuethrich, Kurt</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-5, plates, 29-32</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">MOLMOL is a mol. graphics program for display, anal., and manipulation of three-dimensional structures of biol. macromols., with special emphasis on NMR soln. structures of proteins and nucleic acids.  MOLMOL has a graphical user interface with menus, dialog boxes, and online help.  The display possibilities include conventional presentation, as well as novel schematic drawings, with the option of combining different presentations in one view of a mol.  Covalent mol. structures can be modified by addn. or removal of individual atoms and bonds, and three-dimensional structures can be manipulated by interactive rotation about individual bonds.  Special efforts were made to allow for appropriate display and anal. of the sets of typically 20-40 conformers that are conventionally used to represent the result of an NMR structure detn., using functions for superimposing sets of conformers, calcn. of root mean square distance (RMSD) values, identification of hydrogen bonds, checking and displaying violations of NMR constraints, and identification and listing of short distances between pairs of hydrogen atoms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDF50yz_iOsbVg90H21EOLACvtfcHk0liW8sAjGP70-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis12nsrc%253D&md5=dab669ff481b19cde8fbe2e4234711fa</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2F0263-7855%2896%2900009-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0263-7855%252896%252900009-4%26sid%3Dliteratum%253Aachs%26aulast%3DKoradi%26aufirst%3DR.%26aulast%3DBilleter%26aufirst%3DM.%26aulast%3DW%25C3%25BCthrich%26aufirst%3DK.%26atitle%3DMOLMOL%253A%2520a%2520program%2520for%2520display%2520and%2520analysis%2520of%2520macromolecular%2520structures%26jtitle%3DJ.%2520Mol.%2520Graph.%26date%3D1996%26volume%3D14%26spage%3D51%26epage%3D55%26doi%3D10.1016%2F0263-7855%2896%2900009-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera—A visualization system for exploratory research and analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1605</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%E2%80%94A+visualization+system+for+exploratory+research+and+analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0liW8sAjGP70-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%25E2%2580%2594A%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rullmannn, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacArthur, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaptein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">AQUA and PROCHECK-NMR: programs for checking the quality of protein structures solved by NMR</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">486</span>, <span class="refDoi"> DOI: 10.1007/BF00228148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2FBF00228148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=9008363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK2sXlvVSnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1996&pages=477-486&author=R.+A.+Laskowskiauthor=J.+A.+Rullmannnauthor=M.+W.+MacArthurauthor=R.+Kapteinauthor=J.+M.+Thornton&title=AQUA+and+PROCHECK-NMR%3A+programs+for+checking+the+quality+of+protein+structures+solved+by+NMR&doi=10.1007%2FBF00228148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">AQUA and PROCHECK-NMR: programs for checking the quality of protein structures solved by NMR</span></div><div class="casAuthors">Laskowski, Roman A.; Antoon, J.; Rullmann, C.; Macarthur, Malcolm W.; Kaptein, Robert; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">477-486</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">The AQUA and PROCHECK-NMR programs provide a means of validating the geometry and restraint violations of an ensemble of protein structures solved by soln. NMR.  The outputs include a detailed breakdown of the restraint violations, a no. of plots in PostScript format and summary statistics.  These various analyses indicate both the degree of agreement of the model structures with the exptl. data, and the quality of their geometrical properties.  They are intended to be of use both to support ongoing NMR structure detn. and in the validation of the final results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsafj3zezC3rVg90H21EOLACvtfcHk0liW8sAjGP70-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvVSnsw%253D%253D&md5=030956fb35cbfdaab78801c61ba5a3e1</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1007%2FBF00228148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00228148%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DRullmannn%26aufirst%3DJ.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DKaptein%26aufirst%3DR.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DAQUA%2520and%2520PROCHECK-NMR%253A%2520programs%2520for%2520checking%2520the%2520quality%2520of%2520protein%2520structures%2520solved%2520by%2520NMR%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1996%26volume%3D8%26spage%3D477%26epage%3D486%26doi%3D10.1007%2FBF00228148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krüger, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span> <span> </span><span class="NLM_article-title">NMSim web server: integrated approach for normal mode-based geometric simulations of biologically relevant conformational transitions in proteins</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">W310</span>– <span class="NLM_lpage">W316</span>, <span class="refDoi"> DOI: 10.1093/nar/gks478</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1093%2Fnar%2Fgks478" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=22669906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVCqsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=W310-W316&author=D.+M.+Kr%C3%BCgerauthor=A.+Ahmedauthor=H.+Gohlke&title=NMSim+web+server%3A+integrated+approach+for+normal+mode-based+geometric+simulations+of+biologically+relevant+conformational+transitions+in+proteins&doi=10.1093%2Fnar%2Fgks478"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">NMSim Web Server: integrated approach for normal mode-based geometric simulations of biologically relevant conformational transitions in proteins</span></div><div class="casAuthors">Krueger, Dennis M.; Ahmed, Aqeel; Gohlke, Holger</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W310-W316</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The NMSim web server implements a three-step approach for multiscale modeling of protein conformational changes.  First, the protein structure is coarse-grained using the FIRST software.  Second, a rigid cluster normal-mode anal. provides low-frequency normal modes.  Third, these modes are used to extend the recently introduced idea of constrained geometric simulations by biasing backbone motions of the protein, whereas side chain motions are biased toward favorable rotamer states (NMSim).  The generated structures are iteratively cor. regarding steric clashes and stereochem. constraint violations.  The approach allows performing three simulation types: unbiased exploration of conformational space; pathway generation by a targeted simulation; and radius of gyration-guided simulation.  On a data set of proteins with exptl. obsd. conformational changes, the NMSim approach has been shown to be a computationally efficient alternative to mol. dynamics simulations for conformational sampling of proteins.  The generated conformations and pathways of conformational transitions can serve as input to docking approaches or more sophisticated sampling techniques.  The web server output is a trajectory of generated conformations, Jmol representations of the coarse-graining and a subset of the trajectory and data plots of structural analyses.  The NMSim webserver, accessible at http://www.nmsim.de, is free and open to all users with no login requirement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSHVzQuJQm3bVg90H21EOLACvtfcHk0lh4x55csg8CpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVCqsbc%253D&md5=8d17e24709fa81d55316c63a5b260ecb</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks478%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25BCger%26aufirst%3DD.%2BM.%26aulast%3DAhmed%26aufirst%3DA.%26aulast%3DGohlke%26aufirst%3DH.%26atitle%3DNMSim%2520web%2520server%253A%2520integrated%2520approach%2520for%2520normal%2520mode-based%2520geometric%2520simulations%2520of%2520biologically%2520relevant%2520conformational%2520transitions%2520in%2520proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DW310%26epage%3DW316%26doi%3D10.1093%2Fnar%2Fgks478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heath, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onufriev, A.</span></span> <span> </span><span class="NLM_article-title">H++: a server for estimating pKas and adding missing hydrogens to macromolecules</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">W368</span>– <span class="NLM_lpage">W371</span>, <span class="refDoi"> DOI: 10.1093/nar/gki464</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1093%2Fnar%2Fgki464" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=15980491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlslyqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=W368-W371&author=J.+C.+Gordonauthor=J.+B.+Myersauthor=T.+Foltaauthor=V.+Shojaauthor=L.+S.+Heathauthor=A.+Onufriev&title=H%2B%2B%3A+a+server+for+estimating+pKas+and+adding+missing+hydrogens+to+macromolecules&doi=10.1093%2Fnar%2Fgki464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">H++: a server for estimating pKas and adding missing hydrogens to macromolecules</span></div><div class="casAuthors">Gordon, John C.; Myers, Jonathan B.; Folta, Timothy; Shoja, Valia; Heath, Lenwood S.; Onufriev, Alexey</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W368-W371</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The structure and function of macromols. depend critically on the ionization (protonation) states of their acidic and basic groups.  A no. of existing practical methods predict protonation equil. pK consts. of macromols. based upon their at. resoln. Protein Data Bank (PDB) structures; the calcns. are often performed within the framework of the continuum electrostatics model.  Unfortunately, these methodologies are complex, involve multiple steps and require considerable investment of effort.  Our web server http://biophysics.cs.vt.edu/H++ provides access to a tool that automates this process, allowing both experts and novices to quickly obtain ests. of pKs as well as other related characteristics of biomols. such as isoelec. points, titrn. curves and energies of protonation microstates.  Protons are added to the input structure according to the calcd. ionization states of its titratable groups at the user-specified pH; the output is in the PQR (PDB + charges + radii) format.  In addn., corresponding coordinate and topol. files are generated in the format supported by the mol. modeling package AMBER.  The server is intended for a broad community of biochemists, mol. modelers, structural biologists and drug designers; it can also be used as an educational tool in biochem. courses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaTJ1ZbNsl1LVg90H21EOLACvtfcHk0lh4x55csg8CpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlslyqt7k%253D&md5=a9751f67f129a45a87e12187627d3a2b</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki464%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DJ.%2BC.%26aulast%3DMyers%26aufirst%3DJ.%2BB.%26aulast%3DFolta%26aufirst%3DT.%26aulast%3DShoja%26aufirst%3DV.%26aulast%3DHeath%26aufirst%3DL.%2BS.%26aulast%3DOnufriev%26aufirst%3DA.%26atitle%3DH%252B%252B%253A%2520a%2520server%2520for%2520estimating%2520pKas%2520and%2520adding%2520missing%2520hydrogens%2520to%2520macromolecules%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26spage%3DW368%26epage%3DW371%26doi%3D10.1093%2Fnar%2Fgki464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guex, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peitsch, M. C.</span></span> <span> </span><span class="NLM_article-title">SWISS-MODEL and the Swiss-Pdb Viewer: An environment for comparative protein modeling</span>. <i>Electrophoresis</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2714</span>– <span class="NLM_lpage">2723</span>, <span class="refDoi"> DOI: 10.1002/elps.1150181505</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1002%2Felps.1150181505" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=9504803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK1cXhvFaks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1997&pages=2714-2723&author=N.+Guexauthor=M.+C.+Peitsch&title=SWISS-MODEL+and+the+Swiss-Pdb+Viewer%3A+An+environment+for+comparative+protein+modeling&doi=10.1002%2Felps.1150181505"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">SWISS-MODEL and the Swiss-PdbViewer. An environment for comparative protein modeling</span></div><div class="casAuthors">Guex, Nicolas; Peitsch, Manuel C.</div><div class="citationInfo"><span class="NLM_cas:title">Electrophoresis</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2714-2723</span>CODEN:
                <span class="NLM_cas:coden">ELCTDN</span>;
        ISSN:<span class="NLM_cas:issn">0173-0835</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Comparative protein modeling is increasingly gaining interest since it is of great assistance during the rational design of mutagenesis expts.  The availability of this method, and the resulting models, has however been restricted by the availability of expensive computer hardware and software.  To overcome these limitations, the authors have developed an environment for comparative protein modeling that consists of SWISS-MODEL, a server for automated comparative protein modeling and of the SWISS-PdbViewer, a sequence to structure workbench.  The Swiss-PdbViewer not only acts as a client for SWISS-MODEL, but also provides a large selection of structure anal. and display tools.  In addn., the authors provide the SWISS-MODEL Repository, a database contg. more than 3500 automatically generated protein models.  By making such tools freely available to the scientific community, the authors hope to increase the use of protein structures and models in the process of expt. design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQsoTgcUeb6bVg90H21EOLACvtfcHk0lh4x55csg8CpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhvFaks70%253D&md5=f97197dfdb1cf196099b0576f150d14b</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1002%2Felps.1150181505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Felps.1150181505%26sid%3Dliteratum%253Aachs%26aulast%3DGuex%26aufirst%3DN.%26aulast%3DPeitsch%26aufirst%3DM.%2BC.%26atitle%3DSWISS-MODEL%2520and%2520the%2520Swiss-Pdb%2520Viewer%253A%2520An%2520environment%2520for%2520comparative%2520protein%2520modeling%26jtitle%3DElectrophoresis%26date%3D1997%26volume%3D18%26spage%3D2714%26epage%3D2723%26doi%3D10.1002%2Felps.1150181505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutcliffe, M. J.</span></span> <span> </span><span class="NLM_article-title">An automated approach for clustering an ensemble of NMR-derived protein structures into conformationally related subfamilies</span>. <i>Protein Eng.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1063</span>– <span class="NLM_lpage">1065</span>, <span class="refDoi"> DOI: 10.1093/protein/9.11.1063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1093%2Fprotein%2F9.11.1063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=8961360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK28XnsFamt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1996&pages=1063-1065&author=L.+A.+Kelleyauthor=S.+P.+Gardnerauthor=M.+J.+Sutcliffe&title=An+automated+approach+for+clustering+an+ensemble+of+NMR-derived+protein+structures+into+conformationally+related+subfamilies&doi=10.1093%2Fprotein%2F9.11.1063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">An automated approach for clustering an ensemble of NMR-derived protein structures into conformationally related subfamilies</span></div><div class="casAuthors">Kelley, Lawrence A.; Gardner, Stephen P.; Sutcliffe, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1063-1065</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A computer program has been developed that automatically, systematically, and rapidly clusters an ensemble of structures into a set of conformationally related subfamilies, and selects a representative structure from each cluster.  The program uses the method of av. linkage to define how clusters are built up, followed by the application of a penalty function that seeks to minimize simultaneously the no. of clusters and the spread across each cluster.  This program, known as NMRCLUST, is available via the World Wide Web (URL:http://neon.chem.le.ac.uk/) and by anonymous ftp from ftp.oxmol.co.uk.  Although developed for the anal. of NMR-derived structures, the program can be used to automatically cluster any data set.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1EjchxnvGGrVg90H21EOLACvtfcHk0ligrlS8ajEtyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XnsFamt74%253D&md5=6940b36a9dcbba4dd8352df595737cb2</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F9.11.1063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F9.11.1063%26sid%3Dliteratum%253Aachs%26aulast%3DKelley%26aufirst%3DL.%2BA.%26aulast%3DGardner%26aufirst%3DS.%2BP.%26aulast%3DSutcliffe%26aufirst%3DM.%2BJ.%26atitle%3DAn%2520automated%2520approach%2520for%2520clustering%2520an%2520ensemble%2520of%2520NMR-derived%2520protein%2520structures%2520into%2520conformationally%2520related%2520subfamilies%26jtitle%3DProtein%2520Eng.%26date%3D1996%26volume%3D9%26spage%3D1063%26epage%3D1065%26doi%3D10.1093%2Fprotein%2F9.11.1063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baker, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sept, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holst, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span> <span> </span><span class="NLM_article-title">Electrostatics of nanosystems: application to microtubules and the ribosome</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10037</span>– <span class="NLM_lpage">10041</span>, <span class="refDoi"> DOI: 10.1073/pnas.181342398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1073%2Fpnas.181342398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=11517324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFWisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=10037-10041&author=N.+A.+Bakerauthor=D.+Septauthor=S.+Josephauthor=M.+J.+Holstauthor=J.+A.+McCammon&title=Electrostatics+of+nanosystems%3A+application+to+microtubules+and+the+ribosome&doi=10.1073%2Fpnas.181342398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatics of nanosystems: application to microtubules and the ribosome</span></div><div class="casAuthors">Baker, Nathan A.; Sept, David; Joseph, Simpson; Holst, Michael J.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10037-10041</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Evaluation of the electrostatic properties of biomols. has become a std. practice in mol. biophysics.  Foremost among the models used to elucidate the electrostatic potential is the Poisson-Boltzmann equation; however, existing methods for solving this equation have limited the scope of accurate electrostatic calcns. to relatively small biomol. systems.  Here we present the application of numerical methods to enable the trivially parallel soln. of the Poisson-Boltzmann equation for supramol. structures that are orders of magnitude larger in size.  As a demonstration of this methodol., electrostatic potentials have been calcd. for large microtubule and ribosome structures.  The results point to the likely role of electrostatics in a variety of activities of these structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA_CAlhE0uX7Vg90H21EOLACvtfcHk0ligrlS8ajEtyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFWisbc%253D&md5=1b861999ef12c6972e82e8ada0f387cb</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1073%2Fpnas.181342398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.181342398%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DN.%2BA.%26aulast%3DSept%26aufirst%3DD.%26aulast%3DJoseph%26aufirst%3DS.%26aulast%3DHolst%26aufirst%3DM.%2BJ.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DElectrostatics%2520of%2520nanosystems%253A%2520application%2520to%2520microtubules%2520and%2520the%2520ribosome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D10037%26epage%3D10041%26doi%3D10.1073%2Fpnas.181342398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dolinsky, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, N. A.</span></span> <span> </span><span class="NLM_article-title">PDB2PQR: an automated pipeline for the setup of Poisson-Boltzmann electrostatics calculations</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">W665</span>– <span class="NLM_lpage">W667</span>, <span class="refDoi"> DOI: 10.1093/nar/gkh381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1093%2Fnar%2Fgkh381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=15215472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=W665-W667&author=T.+J.+Dolinskyauthor=J.+E.+Nielsenauthor=J.+A.+McCammonauthor=N.+A.+Baker&title=PDB2PQR%3A+an+automated+pipeline+for+the+setup+of+Poisson-Boltzmann+electrostatics+calculations&doi=10.1093%2Fnar%2Fgkh381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">PDB2PQR: An automated pipeline for the setup of Poisson-Boltzmann electrostatics calculations</span></div><div class="casAuthors">Dolinsky, Todd J.; Nielsen, Jens E.; McCammon, J. Andrew; Baker, Nathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W665-W667</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Continuum solvation models, such as Poisson-Boltzmann and Generalized Born methods, have become increasingly popular tools for investigating the influence of electrostatics on biomol. structure, energetics and dynamics.  However, the use of such methods requires accurate and complete structural data as well as force field parameters such as at. charges and radii.  Unfortunately, the limiting step in continuum electrostatics calcns. is often the addn. of missing at. coordinates to mol. structures from the Protein Data Bank and the assignment of parameters to biomol. structures.  To address this problem, we have developed the PDB2PQR web service (http://agave.wustl.edu/pdb2pqr/).  This server automates many of the common tasks of prepg. structures for continuum electrostatics calcns., including adding a limited no. of missing heavy atoms to biomol. structures, estg. titrn. states and protonating biomols. in a manner consistent with favorable hydrogen bonding, assigning charge and radius parameters from a variety of force fields, and finally generating "PQR" output compatible with several popular computational biol. packages.  This service is intended to facilitate the setup and execution of electrostatics calcns. for both experts and non-experts and thereby broaden the accessibility to the biol. community of continuum electrostatics analyses of biomol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-GebFAmzzkrVg90H21EOLACvtfcHk0ligrlS8ajEtyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsbY%253D&md5=678d2d58f5cb2c1dd862e2d71cf9dbe1</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh381%26sid%3Dliteratum%253Aachs%26aulast%3DDolinsky%26aufirst%3DT.%2BJ.%26aulast%3DNielsen%26aufirst%3DJ.%2BE.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DBaker%26aufirst%3DN.%2BA.%26atitle%3DPDB2PQR%253A%2520an%2520automated%2520pipeline%2520for%2520the%2520setup%2520of%2520Poisson-Boltzmann%2520electrostatics%2520calculations%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26spage%3DW665%26epage%3DW667%26doi%3D10.1093%2Fnar%2Fgkh381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Estrada, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernadó, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackledge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sancho, J.</span></span> <span> </span><span class="NLM_article-title">ProtSA: a web application for calculating sequence specific protein solvent accessibilities in the unfolded ensemble</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">104</span>, <span class="refDoi"> DOI: 10.1186/1471-2105-10-104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1186%2F1471-2105-10-104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=19356231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A280%3ADC%252BD1MzgsFWnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=104&author=J.+Estradaauthor=P.+Bernad%C3%B3author=M.+Blackledgeauthor=J.+Sancho&title=ProtSA%3A+a+web+application+for+calculating+sequence+specific+protein+solvent+accessibilities+in+the+unfolded+ensemble&doi=10.1186%2F1471-2105-10-104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">ProtSA: a web application for calculating sequence specific protein solvent accessibilities in the unfolded ensemble</span></div><div class="casAuthors">Estrada Jorge; Bernado Pau; Blackledge Martin; Sancho Javier</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The stability of proteins is governed by the heat capacity, enthalpy and entropy changes of folding, which are strongly correlated to the change in solvent accessible surface area experienced by the polypeptide.  While the surface exposed in the folded state can be easily determined, accessibilities for the unfolded state at the atomic level cannot be obtained experimentally and are typically estimated using simplistic models of the unfolded ensemble.  A web application providing realistic accessibilities of the unfolded ensemble of a given protein at the atomic level will prove useful.  RESULTS:  ProtSA, a web application that calculates sequence-specific solvent accessibilities of the unfolded state ensembles of proteins has been developed and made freely available to the scientific community.  The input is the amino acid sequence of the protein of interest.  ProtSA follows a previously published calculation protocol which uses the Flexible-Meccano algorithm to generate unfolded conformations representative of the unfolded ensemble of the protein, and uses the exact analytical software ALPHASURF to calculate atom solvent accessibilities, which are averaged on the ensemble.  CONCLUSION:  ProtSA is a novel tool for the researcher investigating protein folding energetics.  The sequence specific atom accessibilities provided by ProtSA will allow obtaining better estimates of the contribution of the hydrophobic effect to the free energy of folding, will help to refine existing parameterizations of protein folding energetics, and will be useful to understand the influence of point mutations on protein stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQdQW7AV3-BvhWfpMXQLRstfW6udTcc2eZbbWt5WpDpg7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzgsFWnsg%253D%253D&md5=52df84f04dacb028ec27924eaec7a286</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-10-104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-10-104%26sid%3Dliteratum%253Aachs%26aulast%3DEstrada%26aufirst%3DJ.%26aulast%3DBernad%25C3%25B3%26aufirst%3DP.%26aulast%3DBlackledge%26aufirst%3DM.%26aulast%3DSancho%26aufirst%3DJ.%26atitle%3DProtSA%253A%2520a%2520web%2520application%2520for%2520calculating%2520sequence%2520specific%2520protein%2520solvent%2520accessibilities%2520in%2520the%2520unfolded%2520ensemble%26jtitle%3DBMC%2520Bioinf.%26date%3D2009%26volume%3D10%26spage%3D104%26doi%3D10.1186%2F1471-2105-10-104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tina, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhadra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, N.</span></span> <span> </span><span class="NLM_article-title">PIC: Protein Interactions Calculator</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">W473</span>– <span class="NLM_lpage">W476</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1093%2Fnar%2Fgkm423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=17584791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVOltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W473-W476&author=K.+G.+Tinaauthor=R.+Bhadraauthor=N.+Srinivasan&title=PIC%3A+Protein+Interactions+Calculator&doi=10.1093%2Fnar%2Fgkm423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">PIC: Protein Interactions Calculator</span></div><div class="casAuthors">Tina K G; Bhadra R; Srinivasan N</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W473-6</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Interactions within a protein structure and interactions between proteins in an assembly are essential considerations in understanding molecular basis of stability and functions of proteins and their complexes.  There are several weak and strong interactions that render stability to a protein structure or an assembly.  Protein Interactions Calculator (PIC) is a server which, given the coordinate set of 3D structure of a protein or an assembly, computes various interactions such as disulphide bonds, interactions between hydrophobic residues, ionic interactions, hydrogen bonds, aromatic-aromatic interactions, aromatic-sulphur interactions and cation-pi interactions within a protein or between proteins in a complex.  Interactions are calculated on the basis of standard, published criteria.  The identified interactions between residues can be visualized using a RasMol and Jmol interface.  The advantage with PIC server is the easy availability of inter-residue interaction calculations in a single site.  It also determines the accessible surface area and residue-depth, which is the distance of a residue from the surface of the protein.  User can also recognize specific kind of interactions, such as apolar-apolar residue interactions or ionic interactions, that are formed between buried or exposed residues or near the surface or deep inside.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKn5zO1OOtqL5XHngOXiV_fW6udTcc2eZbbWt5WpDpg7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVOltQ%253D%253D&md5=6c59648eeba47a6175754ed6d7499c14</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm423%26sid%3Dliteratum%253Aachs%26aulast%3DTina%26aufirst%3DK.%2BG.%26aulast%3DBhadra%26aufirst%3DR.%26aulast%3DSrinivasan%26aufirst%3DN.%26atitle%3DPIC%253A%2520Protein%2520Interactions%2520Calculator%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW473%26epage%3DW476%26doi%3D10.1093%2Fnar%2Fgkm423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding curves of AIF(380–390)red (A) and AIF(380–390)ox (B) to CypA. (C) Binding assay of the Ala-scan peptides at two different concentrations (10 and 20 μM) to CypA. (D) Binding curve of the cyclic peptide AIF(381–389) to CypA. Experiments were carried out using the label-free Corning Epic technology.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. NMR conformational analysis of AIF(381–389) in the free form. Secondary chemical shifts (Δδ) (δ<sub>obs</sub> – δ<sub>coil</sub>) of (A) Hα, (B) Cα, (C) Cβ, and (D) ΔδCα – ΔδCβ differences of the peptide AIF(381–389). The chemical analysis was performed using the random coil shifts calculated as reported in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. Dashed lines indicate the cutoff values for the identification of secondary structure elements defined by Wishart and Sykes<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> (panels A–C) and by Marsh et al.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> (panel D). (E) Portions of the 2D [<sup>1</sup>H–<sup>1</sup>H] ROESY spectrum of AIF(381–389). Some of the inter-residue ROEs observed for the highlighted residues (black) are labeled in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure and dynamics of the AIF(381–389) peptide. (A) (Upper) Secondary structure elements of AIF(381–389) in dependence of the peptide sequence were derived by the CSI based on Cα and Hα resonance assignments. (Lower) Schematic diagram showing the magnitude of ROE connectivities observed in the 2D [<sup>1</sup>H–<sup>1</sup>H] ROESY spectrum. (B) Plot of the distribution of distance restraints as a function of the residue number. The distances are classified according to their range (<i>R</i>): white intraresidual (<i>R</i> = 0), light-gray sequential (<i>R</i> = 1), dark-gray medium-range (<i>R</i> < 5), and black long-range (<i>R</i> ≥ 5). (C) <sup>3</sup>JH<sub>N</sub>Hα scalar coupling (upper) and amide temperature coefficient (lower) analysis. The open bars indicate the <sup>3</sup>JH<sub>N</sub>Hα reference values in the random coil conformation.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> (D,E) Overlay of the 20 lowest energy NMR structures. In panel E, the backbone and side chains are illustrated as gold neon and blue sticks, respectively. The disulfide-bond between Cys<sup>N-ter</sup>–Cys<sup>C-ter</sup> is reported as a light-green stick. (F) Mapping of the residues for which the <sup>1</sup>H–<sup>15</sup>N cross peaks are detectable in the T2-filter <sup>1</sup>H–<sup>15</sup>N HSQC experiment acquired using the filter delay of 500 ms; the H<sub>N</sub>/N intensities are classified as strong (dark blue) or weak (light blue). In red are reported the residues for which <sup>1</sup>H–<sup>15</sup>N signals are not observable. (G) Model-free order parameter (<i>S</i><sup>2</sup>) mapped onto the representative NMR structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. NMR analysis of AIF(381–389) binding to CypA. (A,B Upper) H<sub>N</sub>/N (A) and Hα/Cα (B) signal intensity ratios of AIF(381–389) upon addition of CypA (I<sub><i>b</i>ound</sub>) and in the free form (<i>I</i><sub>free</sub>) as a function of the amino acid sequence. The orange line indicates the average <i>I</i><sub>bound</sub>/<i>I</i><sub>free</sub> value. (A,B Lower) Overlay of the <sup>1</sup>H–<sup>15</sup>N HSQC (A) and <sup>1</sup>H–<sup>13</sup>C HSQC (B) spectra of unlabeled AIF(381–389) in the absence (blue) or in the presence (red) of CypA. (C) Mapping of weighted combined <sup>1</sup>H, <sup>15</sup>N, and <sup>13</sup>C CSPs onto the representative conformer of the AIF(381–389) NMR ensemble obtained for backbone (left), side chains (middle), and all-together atoms (right), respectively. The residues of AIF(381–389) showing significant CSP are labeled using the color code reported on the right side, which indicates the chemical shift variation observed for the peptide upon addition of CypA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Binding of CypA to the AIF(381–389) peptide by NMR. (A,B) Superposition of the <sup>1</sup>H–<sup>15</sup>N HSQC spectra of <sup>15</sup>N-CypA alone (blue) and after the addition of AIF(381–389) (molar ratio 10:1). The insets show portions of the overlapped <sup>1</sup>H–<sup>15</sup>N HSQC spectra. (C) <sup>1</sup>H/<sup>15</sup>N CSPs plotted against the CypA sequence. The yellow line indicates the average CSP (CSPavg), whereas the orange line reports the CSPavg + SD (standard deviation) value. (D) Mapping of the residues showing CSP > CSPavg (yellow) and ≥ CSPavg + SD (orange) onto the CypA X-ray structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CPL">2CPL</a>) reported as ribbon drawing (left) and surface (right) representation. The dotted line circle indicates the catalytic site of CypA. (E) Ratios between NMR signal intensity observed for CypA in the presence (<i>I</i><sub>bound</sub>) and absence (<i>I</i><sub>free</sub>) of AIF(381–389) (values normalized to Asn<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a>) and ASA plotted <i>vs</i> the residue number. (F) CypA residues used as active residues in the docking studies. (G) CypA electrostatic surface potential depicted from electropositive (blue; 10 kcal/mole) to electronegative (red; −10 kcal/mole) in two orientations rotated 180° around the <i>z</i>-axis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structural model of the AIF(381–389)/CypA complex. (A) Ribbon drawing representation of the AIF(381–389)/CypA model in which the side chains of AIF(381–389) and CypA residues, involved in the complex formation, are shown as light-green and light-blue sticks, respectively. (B) ConSurf bioinformatics analysis of CypA. The conserved and variable residues are depicted as teal and dark magenta, respectively. (C) Mapping of the CypA residues (light green) forming the AIF(381–389) binding surface onto the complex structure. The CypA is reported as a light-gray surface representation, whereas the AIF(381–389) peptide is shown as a light-blue ribbon diagram. The two main residues of the catalytic site (Arg<sup>55</sup> and Trp<sup>121</sup>) are depicted in orange (D–F). Close-up views of the AIF(381–389)/CypA complex. The side chains of the residues playing a crucial role in the recognition mechanism are illustrated as light-blue and light-green sticks for AIF(381–389) and CypA, respectively. The 2D interaction scheme is also reported. The dashed lines reported in magenta and dark blue denote hydrogen bonds and ionic interactions, respectively. The crucial residues of the binding surface of CypA are labeled.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Serum stability profiles of AIF(381–389) and AIF(370–394). Relative peptide concentrations were determined by integrating RP-HPLC peaks from chromatograms at 215 nm. Assays were performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/medium/jm1c00777_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Relative cell viability of HT-22 neuronal cells transfected with AIF(381–389) and AIF(370–394), as determined by the MTT assay. HT-22 cells transfected with AIF(381–389) and AIF(370–394) were treated with glutamate at 5, 6, and 7 mM for 24 h. Results obtained are shown as percentage of controls set at 100%. Data are representative of the mean ± S.D. of at least four technical replicates. Statistical significance (<i>p</i> < 0.05) compared to the respective control is indicated by the asterisk symbol (*).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00777/20210802/images/large/jm1c00777_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00777&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i34">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83608" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83608" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 74 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maté, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz-Lombardía, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boitel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haouz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tello, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Susin, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penninger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alzari, P. M.</span></span> <span> </span><span class="NLM_article-title">The crystal structure of the mouse apoptosis-inducing factor AIF</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">442</span>– <span class="NLM_lpage">446</span>, <span class="refDoi"> DOI: 10.1038/nsb793</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Fnsb793" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=11967568" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvFOms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=442-446&author=M.+J.+Mat%C3%A9author=M.+Ortiz-Lombard%C3%ADaauthor=B.+Boitelauthor=A.+Haouzauthor=D.+Telloauthor=S.+A.+Susinauthor=J.+Penningerauthor=G.+Kroemerauthor=P.+M.+Alzari&title=The+crystal+structure+of+the+mouse+apoptosis-inducing+factor+AIF&doi=10.1038%2Fnsb793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The crystal structure of the mouse apoptosis-inducing factor AIF</span></div><div class="casAuthors">Mate, Maria J.; Ortiz-Lombardia, Miguel; Boitel, Brigitte; Haouz, Ahmed; Tello, Diana; Susin, Santos A.; Penninger, Josef; Kroemer, Guido; Alzari, Pedro M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">442-446</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Mitochondria play a key role in apoptosis due to their capacity to release potentially lethal proteins.  One of these latent death factors is cytochrome c, which can stimulate the proteolytic activation of caspase zymogens.  Another important protein is apoptosis-inducing factor (AIF), a 57-kDa electron-transferring flavoprotein oxidoreductase, which can stimulate a caspase-independent cell-death pathway required for early embryonic morphogenesis.  Here, the authors report the crystal structure of mouse mitochondrial AIF at 2.0 Å.  Its active site structure and redox properties suggest that AIF functions as an electron transferase with a mechanism similar to that of the bacterial ferredoxin reductases, its closest evolutionary homologs.  However, AIF structurally differs from these proteins in some essential features, including a long insertion in a C-terminal β-hairpin loop, which may be related to its apoptogenic functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBoQsgWDBjzbVg90H21EOLACvtfcHk0ljxHGTYTGSVIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvFOms78%253D&md5=ab8fec73c8964c16f5282ae87a5c87ac</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnsb793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb793%26sid%3Dliteratum%253Aachs%26aulast%3DMat%25C3%25A9%26aufirst%3DM.%2BJ.%26aulast%3DOrtiz-Lombard%25C3%25ADa%26aufirst%3DM.%26aulast%3DBoitel%26aufirst%3DB.%26aulast%3DHaouz%26aufirst%3DA.%26aulast%3DTello%26aufirst%3DD.%26aulast%3DSusin%26aufirst%3DS.%2BA.%26aulast%3DPenninger%26aufirst%3DJ.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DAlzari%26aufirst%3DP.%2BM.%26atitle%3DThe%2520crystal%2520structure%2520of%2520the%2520mouse%2520apoptosis-inducing%2520factor%2520AIF%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D9%26spage%3D442%26epage%3D446%26doi%3D10.1038%2Fnsb793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sevrioukova, I. F.</span></span> <span> </span><span class="NLM_article-title">Apoptosis-inducing factor: structure, function, and redox regulation</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2545</span>– <span class="NLM_lpage">2579</span>, <span class="refDoi"> DOI: 10.1089/ars.2010.3445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1089%2Fars.2010.3445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=20868295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtl2lu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2011&pages=2545-2579&author=I.+F.+Sevrioukova&title=Apoptosis-inducing+factor%3A+structure%2C+function%2C+and+redox+regulation&doi=10.1089%2Fars.2010.3445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Apoptosis-inducing factor: structure, function, and redox regulation</span></div><div class="casAuthors">Sevrioukova, Irina F.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2545-2579</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Apoptosis-inducing factor (AIF) is a FAD-contg., NADH-dependent oxidoreductase residing in the mitochondrial intermembrane space whose specific enzymic activity remains unknown.  Upon an apoptotic insult, AIF undergoes proteolysis and translocates to the nucleus, where it triggers chromatin condensation and large-scale DNA degrdn. in a caspase-independent manner.  In addn. to playing a key role in the execution of caspase-independent cell death, AIF has emerged as a protein crit. for cell survival.  Anal. of in vivo phenotypes assocd. with AIF deficiency and defects, and identification of its mitochondrial, cytoplasmic, and nuclear partners has revealed the complexity and multi-level regulation of AIF-mediated signal transduction and has suggested an important role of AIF in the maintenance of mitochondrial morphol. and energy metab.  The redox activity of AIF is essential for optimal oxidative phosphorylation.  Addnl., the protein is proposed to regulate the respiratory chain indirectly, through assembly and/or stabilization of complexes I and III.  Here, the author discusses accumulated data with respect to AIF structure and outlines evidence that supports the prevalent mechanistic view on the apoptogenic actions of the flavoprotein, as well as the emerging concept of AIF as a redox sensor capable of linking NAD(H)-dependent metabolic pathways to apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9o4PRjq8nbLVg90H21EOLACvtfcHk0ljxHGTYTGSVIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtl2lu7g%253D&md5=05dad64022bf3bd44ec55c0139f7f420</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1089%2Fars.2010.3445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2010.3445%26sid%3Dliteratum%253Aachs%26aulast%3DSevrioukova%26aufirst%3DI.%2BF.%26atitle%3DApoptosis-inducing%2520factor%253A%2520structure%252C%2520function%252C%2520and%2520redox%2520regulation%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2011%26volume%3D14%26spage%3D2545%26epage%3D2579%26doi%3D10.1089%2Fars.2010.3445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackerman, S. L.</span></span> <span> </span><span class="NLM_article-title">Oxidative stress, cell cycle, and neurodegeneration</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">785</span>– <span class="NLM_lpage">793</span>, <span class="refDoi"> DOI: 10.1172/jci200318182</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1172%2FJCI200318182" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12639981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlOjsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2003&pages=785-793&author=J.+A.+Kleinauthor=S.+L.+Ackerman&title=Oxidative+stress%2C+cell+cycle%2C+and+neurodegeneration&doi=10.1172%2Fjci200318182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Oxidative stress, cell cycle, and neurodegeneration</span></div><div class="casAuthors">Klein, Jeffrey A.; Ackerman, Susan L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">785-793</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">A review discusses the influence of oxidative stress on neurodegeneration and the possible connections between oxidative stress and unscheduled cell cycle reentry.  The misregulation of the cell cycle in many cell types can lead to unchecked proliferation and neoplastic disease.  Some of the potential pathways by which the repression of the cell cycle in neurons may be modulated by reactive oxygen species are presented.  Specific types of neurons may responds to different signaling pathways, or need activation of multiple signaling pathways.  The testing of the roles of these and other potential pathways in neurodegeneration will require the development and anal. of the addnl. animals models, in addn. to further correlative human studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6guQbmgpEXbVg90H21EOLACvtfcHk0ljxHGTYTGSVIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlOjsbw%253D&md5=051d0f2d1745075cf73d56e8c4d19261</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1172%2FJCI200318182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI200318182%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DJ.%2BA.%26aulast%3DAckerman%26aufirst%3DS.%2BL.%26atitle%3DOxidative%2520stress%252C%2520cell%2520cycle%252C%2520and%2520neurodegeneration%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2003%26volume%3D111%26spage%3D785%26epage%3D793%26doi%3D10.1172%2Fjci200318182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longo-Guess, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossmann, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seburn, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurd, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackerman, S. L.</span></span> <span> </span><span class="NLM_article-title">The harlequin mouse mutation downregulates apoptosis-inducing factor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>419</i></span>,  <span class="NLM_fpage">367</span>– <span class="NLM_lpage">374</span>, <span class="refDoi"> DOI: 10.1038/nature01034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Fnature01034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12353028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD38XntlGgsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=419&publication_year=2002&pages=367-374&author=J.+A.+Kleinauthor=C.+M.+Longo-Guessauthor=M.+P.+Rossmannauthor=K.+L.+Seburnauthor=R.+E.+Hurdauthor=W.+N.+Frankelauthor=R.+T.+Bronsonauthor=S.+L.+Ackerman&title=The+harlequin+mouse+mutation+downregulates+apoptosis-inducing+factor&doi=10.1038%2Fnature01034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The harlequin mouse mutation downregulates apoptosis-inducing factor</span></div><div class="casAuthors">Klein, Jeffrey A.; Longo-Guess, Chantal M.; Rossmann, Marlies P.; Seburn, Kevin L.; Hurd, Ronald E.; Frankel, Wayne N.; Bronson, Roderick T.; Ackerman, Susan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">419</span>
        (<span class="NLM_cas:issue">6905</span>),
    <span class="NLM_cas:pages">367-374</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Harlequin (Hq) mutant mice have progressive degeneration of terminally differentiated cerebellar and retinal neurons.  The authors have identified the Hq mutation as a proviral insertion in the apoptosis-inducing factor (Aif) gene, causing about an 80% redn. in AIF expression.  Mutant cerebellar granule cells are susceptible to exogenous and endogenous peroxide-mediated apoptosis, but can be rescued by AIF expression.  Overexpression of AIF in wild-type granule cells further decreases peroxide-mediated cell death, suggesting that AIF serves as a free radical scavenger.  In agreement, dying neurons in aged Hq mutant mice show oxidative stress.  In addn., neurons damaged by oxidative stress in both the cerebellum and retina of Hq mutant mice re-enter the cell cycle before undergoing apoptosis.  Our results provide a genetic model of oxidative stress-mediated neurodegeneration and demonstrate a direct connection between cell cycle re-entry and oxidative stress in the ageing central nervous system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrae1zYgtmzcrVg90H21EOLACvtfcHk0lg7Ey0vED9vnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntlGgsb8%253D&md5=5c229f0747c0c6e975cc6c695ca98367</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature01034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01034%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DJ.%2BA.%26aulast%3DLongo-Guess%26aufirst%3DC.%2BM.%26aulast%3DRossmann%26aufirst%3DM.%2BP.%26aulast%3DSeburn%26aufirst%3DK.%2BL.%26aulast%3DHurd%26aufirst%3DR.%2BE.%26aulast%3DFrankel%26aufirst%3DW.%2BN.%26aulast%3DBronson%26aufirst%3DR.%2BT.%26aulast%3DAckerman%26aufirst%3DS.%2BL.%26atitle%3DThe%2520harlequin%2520mouse%2520mutation%2520downregulates%2520apoptosis-inducing%2520factor%26jtitle%3DNature%26date%3D2002%26volume%3D419%26spage%3D367%26epage%3D374%26doi%3D10.1038%2Fnature01034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deinum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modjtahedi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neagu, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eriksson, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luban, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomgren, K.</span></span> <span> </span><span class="NLM_article-title">Cyclophilin A participates in the nuclear translocation of apoptosis-inducing factor in neurons after cerebral hypoxia-ischemia</span>. <i>J. Exp. Med.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>204</i></span>,  <span class="NLM_fpage">1741</span>– <span class="NLM_lpage">1748</span>, <span class="refDoi"> DOI: 10.1084/jem.20070193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1084%2Fjem.20070193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=17635954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2sXptVyjs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=204&publication_year=2007&pages=1741-1748&author=C.+Zhuauthor=X.+Wangauthor=J.+Deinumauthor=Z.+Huangauthor=J.+Gaoauthor=N.+Modjtahediauthor=M.+R.+Neaguauthor=M.+Nilssonauthor=P.+S.+Erikssonauthor=H.+Hagbergauthor=J.+Lubanauthor=G.+Kroemerauthor=K.+Blomgren&title=Cyclophilin+A+participates+in+the+nuclear+translocation+of+apoptosis-inducing+factor+in+neurons+after+cerebral+hypoxia-ischemia&doi=10.1084%2Fjem.20070193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclophilin A participates in the nuclear translocation of apoptosis-inducing factor in neurons after cerebral hypoxia-ischemia</span></div><div class="casAuthors">Zhu, Changlian; Wang, Xiaoyang; Deinum, Johanna; Huang, Zhiheng; Gao, Jianfeng; Modjtahedi, Nazanine; Neagu, Martha R.; Nilsson, Michael; Eriksson, Peter S.; Hagberg, Henrik; Luban, Jeremy; Kroemer, Guido; Blomgren, Klas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Medicine</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">204</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1741-1748</span>CODEN:
                <span class="NLM_cas:coden">JEMEAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-1007</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Upon cerebral hypoxia-ischemia (HI), apoptosis-inducing factor (AIF) can move from mitochondria to nuclei, participate in chromatinolysis, and contribute to the execution of cell death.  Previous work performed in vitro suggests that AIF must interact with cyclophilin A (CypA) to form a proapoptotic DNA degrdn. complex.  The authors addressed the question as to whether elimination of CypA may afford neuroprotection in vivo.  9-D-old wild-type (WT), CypA+/-, or CypA-/- mice were subjected to unilateral cerebral HI.  The infarct vol. after HI was reduced by 47% in CypA-/- mice compared with their WT littermates.  Importantly, CypA-/- neurons failed to manifest the HI-induced nuclear translocation of AIF that was obsd. in WT neurons.  Conversely, CypA accumulated within the nuclei of damaged neurons after HI, and this nuclear translocation of CypA was suppressed in AIF-deficient harlequin mice.  Immunopptn. of AIF revealed copptn. of CypA, but only in injured, ischemic tissue.  Surface plasmon resonance revealed direct mol. interactions between recombinant AIF and CypA.  These data indicate that the lethal translocation of AIF to the nucleus requires interaction with CypA, suggesting a model in which two proteins that normally reside in sep. cytoplasmic compartments acquire novel properties when moving together to the nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ0LbSrzcFfbVg90H21EOLACvtfcHk0lg7Ey0vED9vnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXptVyjs7s%253D&md5=cc2086d63cd3d6d3200b09605f82af32</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1084%2Fjem.20070193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20070193%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DDeinum%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DModjtahedi%26aufirst%3DN.%26aulast%3DNeagu%26aufirst%3DM.%2BR.%26aulast%3DNilsson%26aufirst%3DM.%26aulast%3DEriksson%26aufirst%3DP.%2BS.%26aulast%3DHagberg%26aufirst%3DH.%26aulast%3DLuban%26aufirst%3DJ.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DBlomgren%26aufirst%3DK.%26atitle%3DCyclophilin%2520A%2520participates%2520in%2520the%2520nuclear%2520translocation%2520of%2520apoptosis-inducing%2520factor%2520in%2520neurons%2520after%2520cerebral%2520hypoxia-ischemia%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2007%26volume%3D204%26spage%3D1741%26epage%3D1748%26doi%3D10.1084%2Fjem.20070193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ekert, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaux, D. L.</span></span> <span> </span><span class="NLM_article-title">The mitochondrial death squad: hardened killers or innocent bystanders?</span>. <i>Curr. Opin. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">626</span>– <span class="NLM_lpage">630</span>, <span class="refDoi"> DOI: 10.1016/j.ceb.2005.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2Fj.ceb.2005.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=16219456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFyksL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2005&pages=626-630&author=P.+G.+Ekertauthor=D.+L.+Vaux&title=The+mitochondrial+death+squad%3A+hardened+killers+or+innocent+bystanders%3F&doi=10.1016%2Fj.ceb.2005.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The mitochondrial death squad: hardened killers or innocent bystanders?</span></div><div class="casAuthors">Ekert, Paul G.; Vaux, David L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">626-630</span>CODEN:
                <span class="NLM_cas:coden">COCBE3</span>;
        ISSN:<span class="NLM_cas:issn">0955-0674</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Since the discovery that formation of the apoptosome in mammalian cells is triggered by cytochrome c released from the mitochondria, many other mitochondrial proteins have been suspected to be part of a conspiracy to cause cell death.  Proteins AIF, EndoG, ANT, cyclophilin D, Bit1, p53AIP, GRIM-19, DAP3, Nur77/TR3/NGFB-1, HtrA2/Omi and Smac/Diablo have all been convicted as killers, but new genetic technol. is raising questions about their guilt.  Gene knockout expts. suggest that many were wrongly convicted on circumstantial evidence, and just happened to be in the wrong place at the wrong time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6t5fPD8wACbVg90H21EOLACvtfcHk0li-FjuITUMALg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFyksL%252FP&md5=5a574244aa43acdbba38ab88dd9b55db</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.ceb.2005.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ceb.2005.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DEkert%26aufirst%3DP.%2BG.%26aulast%3DVaux%26aufirst%3DD.%2BL.%26atitle%3DThe%2520mitochondrial%2520death%2520squad%253A%2520hardened%2520killers%2520or%2520innocent%2520bystanders%253F%26jtitle%3DCurr.%2520Opin.%2520Cell%2520Biol.%26date%3D2005%26volume%3D17%26spage%3D626%26epage%3D630%26doi%3D10.1016%2Fj.ceb.2005.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Modjtahedi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madeo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">Apoptosis-inducing factor: vital and lethal</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1016/j.tcb.2006.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2Fj.tcb.2006.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=16621561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD28XkvVOrsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=264-272&author=N.+Modjtahediauthor=F.+Giordanettoauthor=F.+Madeoauthor=G.+Kroemer&title=Apoptosis-inducing+factor%3A+vital+and+lethal&doi=10.1016%2Fj.tcb.2006.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Apoptosis-inducing factor: vital and lethal</span></div><div class="casAuthors">Modjtahedi, Nazanine; Giordanetto, Fabrizio; Madeo, Frank; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">264-272</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Apoptosis-inducing factor (AIF) is a NADH oxidase with a local redox function that is essential for optimal oxidative phosphorylation and for an efficient anti-oxidant defense.  The absence of AIF can cause neurodegeneration, skeleton muscle atrophy and dilated cardiomyopathy.  In many models of apoptosis, AIF translocates to the nucleus, where it induces chromatin condensation and DNA degrdn.  The nuclear localization of AIF can be inhibited by blocking upstream signals of apoptosis.  The contribution of AIF to cell death depends on the cell type and apoptotic insult and is only seen when caspases are inhibited or not activated.  It is unknown to what extent and through which mechanisms AIF contributes to the induction of cell death.  Here, we discuss recent progress in the quest to understand the contribution of AIF to life and death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQv5SK7lySk7Vg90H21EOLACvtfcHk0li-FjuITUMALg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkvVOrsr4%253D&md5=0af7dd5ce7279497c13262fd4af1c46a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2006.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2006.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DModjtahedi%26aufirst%3DN.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DMadeo%26aufirst%3DF.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DApoptosis-inducing%2520factor%253A%2520vital%2520and%2520lethal%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2006%26volume%3D16%26spage%3D264%26epage%3D272%26doi%3D10.1016%2Fj.tcb.2006.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Candé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecconi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dessen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">Apoptosis-inducing factor (AIF): key to the conserved caspase-independent pathways of cell death?</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">4727</span>– <span class="NLM_lpage">4734</span>, <span class="refDoi"> DOI: 10.1242/jcs.00210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1242%2Fjcs.00210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12432061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3sXis1Kkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2002&pages=4727-4734&author=C.+Cand%C3%A9author=F.+Cecconiauthor=P.+Dessenauthor=G.+Kroemer&title=Apoptosis-inducing+factor+%28AIF%29%3A+key+to+the+conserved+caspase-independent+pathways+of+cell+death%3F&doi=10.1242%2Fjcs.00210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Apoptosis-inducing factor (AIF): key to the conserved caspase-independent pathways of cell death?</span></div><div class="casAuthors">Cande, Celine; Cecconi, Francesco; Dessen, Philippe; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4727-4734</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review.  Numerous pro-apoptotic signal transducing mols. act on mitochondria and provoke the permeabilization of the outer mitochondrial membrane, thereby triggering the release of potentially toxic mitochondrial proteins.  One of these proteins, apoptosis-inducing factor (AIF), is a phylogenetically old flavoprotein which, in healthy cells, is confined to the mitochondrial intermembrane space.  Upon lethal signaling, AIF translocates, via the cytosol, to the nucleus where it binds to DNA and provokes caspase-independent chromatin condensation.  The crystal structures of both human and mouse AIF have been detd., and the fine mechanisms accounting for its oxidoreductase activity and its electrostatic interaction with double-stranded DNA have been elucidated.  Importantly, the apoptogenic and oxidoreductase functions of AIF can be dissocd.  Thus, mutations that abolish the AIF-DNA interaction suppress AIF-induced chromatin condensation, yet have no effect on the NADH oxidase activity.  Recent studies suggest AIF to be a major factor detg. caspase-independent neuronal death, emphasizing the central role of mitochondria in the control of physiol. and pathol. cell demise.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO8zyBaGTMJrVg90H21EOLACvtfcHk0li-FjuITUMALg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXis1Kkuw%253D%253D&md5=71dba7dd46a1d90ce9c94b5899b47ea2</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1242%2Fjcs.00210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.00210%26sid%3Dliteratum%253Aachs%26aulast%3DCand%25C3%25A9%26aufirst%3DC.%26aulast%3DCecconi%26aufirst%3DF.%26aulast%3DDessen%26aufirst%3DP.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DApoptosis-inducing%2520factor%2520%2528AIF%2529%253A%2520key%2520to%2520the%2520conserved%2520caspase-independent%2520pathways%2520of%2520cell%2520death%253F%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2002%26volume%3D115%26spage%3D4727%26epage%3D4734%26doi%3D10.1242%2Fjcs.00210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cregan, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacLaurin, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callaghan, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecconi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slack, R. S.</span></span> <span> </span><span class="NLM_article-title">Apoptosis-inducing factor is involved in the regulation of caspase-independent neuronal cell death</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>158</i></span>,  <span class="NLM_fpage">507</span>– <span class="NLM_lpage">517</span>, <span class="refDoi"> DOI: 10.1083/jcb.200202130</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1083%2Fjcb.200202130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12147675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD38XmtVeks7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2002&pages=507-517&author=S.+P.+Creganauthor=A.+Fortinauthor=J.+G.+MacLaurinauthor=S.+M.+Callaghanauthor=F.+Cecconiauthor=S.-W.+Yuauthor=T.+M.+Dawsonauthor=V.+L.+Dawsonauthor=D.+S.+Parkauthor=G.+Kroemerauthor=R.+S.+Slack&title=Apoptosis-inducing+factor+is+involved+in+the+regulation+of+caspase-independent+neuronal+cell+death&doi=10.1083%2Fjcb.200202130"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Apoptosis-inducing factor is involved in the regulation of caspase-independent neuronal cell death</span></div><div class="casAuthors">Cregan, Sean P.; Fortin, Andre; MacLaurin, Jason G.; Callaghan, Steven M.; Cecconi, Francesco; Yu, Seong-Woon; Dawson, Ted M.; Dawson, Valina L.; Park, David S.; Kroemer, Guido; Slack, Ruth S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">507-517</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9525</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Caspase-independent death mechanisms have been shown to execute apoptosis in many types of neuronal injury.  P53 has been identified as a key regulator of neuronal cell death after acute injury such as DNA damage, ischemia, and excitotoxicity.  Here, we demonstrate that p53 can induce neuronal cell death via a caspase-mediated process activated by apoptotic activating factor-1 (Apaf1) and via a delayed onset caspase-independent mechanism.  In contrast to wild-type cells, Apaf1-deficient neurons exhibit delayed DNA fragmentation and only peripheral chromatin condensation.  More importantly, we demonstrate that apoptosis-inducing factor (AIF) is an important factor involved in the regulation of this caspase-independent neuronal cell death.  Immunofluorescence studies demonstrate that AIF is released from the mitochondria by a mechanism distinct from that of cytochrome-c in neurons undergoing p53-mediated cell death.  The Bcl-2 family regulates this release of AIF and subsequent caspase-independent cell death.  In addn., we show that enforced expression of AIF can induce neuronal cell death in a Bax- and caspase-independent manner.  Microinjection of neutralizing antibodies against AIF significantly decreased injury-induced neuronal cell death in Apaf1-deficient neurons, indicating its importance in caspase-independent apoptosis.  Taken together, our results suggest that AIF may be an important therapeutic target for the treatment of neuronal injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPDGvLxICd4bVg90H21EOLACvtfcHk0li-FjuITUMALg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmtVeks7Y%253D&md5=35c504769247a1a0f591925e3dc78f5b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200202130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200202130%26sid%3Dliteratum%253Aachs%26aulast%3DCregan%26aufirst%3DS.%2BP.%26aulast%3DFortin%26aufirst%3DA.%26aulast%3DMacLaurin%26aufirst%3DJ.%2BG.%26aulast%3DCallaghan%26aufirst%3DS.%2BM.%26aulast%3DCecconi%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DS.-W.%26aulast%3DDawson%26aufirst%3DT.%2BM.%26aulast%3DDawson%26aufirst%3DV.%2BL.%26aulast%3DPark%26aufirst%3DD.%2BS.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DSlack%26aufirst%3DR.%2BS.%26atitle%3DApoptosis-inducing%2520factor%2520is%2520involved%2520in%2520the%2520regulation%2520of%2520caspase-independent%2520neuronal%2520cell%2520death%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2002%26volume%3D158%26spage%3D507%26epage%3D517%26doi%3D10.1083%2Fjcb.200202130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Susin, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzo, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamzami, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snow, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangion, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacotot, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costantini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loeffler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larochette, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodlett, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aebersold, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siderovski, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penninger, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">Molecular characterization of mitochondrial apoptosis-inducing factor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>397</i></span>,  <span class="NLM_fpage">441</span>– <span class="NLM_lpage">446</span>, <span class="refDoi"> DOI: 10.1038/17135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2F17135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=9989411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK1MXhtFOhu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=397&publication_year=1999&pages=441-446&author=S.+A.+Susinauthor=H.+K.+Lorenzoauthor=N.+Zamzamiauthor=I.+Marzoauthor=B.+E.+Snowauthor=G.+M.+Brothersauthor=J.+Mangionauthor=E.+Jacototauthor=P.+Costantiniauthor=M.+Loefflerauthor=N.+Larochetteauthor=D.+R.+Goodlettauthor=R.+Aebersoldauthor=D.+P.+Siderovskiauthor=J.+M.+Penningerauthor=G.+Kroemer&title=Molecular+characterization+of+mitochondrial+apoptosis-inducing+factor&doi=10.1038%2F17135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular characterization of mitochondrial apoptosis-inducing factor</span></div><div class="casAuthors">Susin, Santos A.; Lorenzo, Hans K.; Zamzami, Naoufal; Marzo, Isabel; Snow, Bryan E.; Brothers, Greg M.; Mangion, Joan; Jacotot, Etienne; Costantini, Paola; Loeffler, Markus; Larochette, Nathanael; Goodlett, David R.; Aebersold, Ruedi; Siderovski, David P.; Penninger, Josef M.; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">397</span>
        (<span class="NLM_cas:issue">6718</span>),
    <span class="NLM_cas:pages">441-446</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Mitochondria play a key part in the regulation of apoptosis (cell death).  Their intermembrane space contains several proteins that are liberated through the outer membrane in order to participate in the degrdn. phase of apoptosis.  Here we report the identification and cloning of an apoptosis-inducing factor, AIF5, which is sufficient to induce apoptosis of isolated nuclei.  AIF is a flavoprotein of relative mol. mass 57,000 which shares homol. with the bacterial oxidoreductases; it is normally confined to mitochondria but translocates to the nucleus when apoptosis is induced.  Recombinant AIF causes chromatin condensation in isolated nuclei and large-scale fragmentation of DNA.  It induces purified mitochondria to release the apoptogenic proteins cytochrome c and caspase-9.  Microinjection of AIF into the cytoplasm of intact cells induces condensation of chromatin, dissipation of the mitochondrial transmembrane potential, and exposure of phosphatidylserine in the plasma membrane.  None of these effects is prevented by the wide-ranging caspase inhibitor known as Z-VAD.fmk.  Overexpression of Bcl-2, which controls the opening of mitochondrial permeability transition pores, prevents the release of AIF from the mitochondrion but does not affect its apoptogenic activity.  These results indicate that AIF is a mitochondrial effector of apoptotic cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8KHCoQLJgaLVg90H21EOLACvtfcHk0lhR8651-D43yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhtFOhu78%253D&md5=e02254e6bf0898bc9218950620b08196</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2F17135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F17135%26sid%3Dliteratum%253Aachs%26aulast%3DSusin%26aufirst%3DS.%2BA.%26aulast%3DLorenzo%26aufirst%3DH.%2BK.%26aulast%3DZamzami%26aufirst%3DN.%26aulast%3DMarzo%26aufirst%3DI.%26aulast%3DSnow%26aufirst%3DB.%2BE.%26aulast%3DBrothers%26aufirst%3DG.%2BM.%26aulast%3DMangion%26aufirst%3DJ.%26aulast%3DJacotot%26aufirst%3DE.%26aulast%3DCostantini%26aufirst%3DP.%26aulast%3DLoeffler%26aufirst%3DM.%26aulast%3DLarochette%26aufirst%3DN.%26aulast%3DGoodlett%26aufirst%3DD.%2BR.%26aulast%3DAebersold%26aufirst%3DR.%26aulast%3DSiderovski%26aufirst%3DD.%2BP.%26aulast%3DPenninger%26aufirst%3DJ.%2BM.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DMolecular%2520characterization%2520of%2520mitochondrial%2520apoptosis-inducing%2520factor%26jtitle%3DNature%26date%3D1999%26volume%3D397%26spage%3D441%26epage%3D446%26doi%3D10.1038%2F17135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S. J.</span>; <span class="NLM_string-name">Dawson, T. M.</span>; <span class="NLM_string-name">Dawson, V. L.</span></span> <span> </span><span class="NLM_article-title">PARP and the Release of Apoptosis-Inducing Factor from Mitochondria</span>.  <i>Madame Curie Bioscience Database [Internet]</i>; <span class="NLM_publisher-name">Landes Bioscience</span>: <span class="NLM_publisher-loc">Austin (TX)</span>, <span class="NLM_year">2000–2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000%E2%80%932013&author=S.+J.+Hong&author=T.+M.+Dawson&author=V.+L.+Dawson&title=Madame+Curie+Bioscience+Database+%5BInternet%5D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DS.%2BJ.%26atitle%3DPARP%2520and%2520the%2520Release%2520of%2520Apoptosis-Inducing%2520Factor%2520from%2520Mitochondria%26btitle%3DMadame%2520Curie%2520Bioscience%2520Database%2520%255BInternet%255D%26pub%3DLandes%2520Bioscience%26date%3D2000%25E2%2580%25932013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S. J.</span></span> <span> </span><span class="NLM_article-title">AIF translocates to the nucleus in the spinal motor neurons in a mouse model of ALS</span>. <i>Neurosci. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>406</i></span>,  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1016/j.neulet.2006.07.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2Fj.neulet.2006.07.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=16916579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsVOms70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=406&publication_year=2006&pages=205-210&author=Y.+K.+Ohauthor=K.+S.+Shinauthor=S.+J.+Kang&title=AIF+translocates+to+the+nucleus+in+the+spinal+motor+neurons+in+a+mouse+model+of+ALS&doi=10.1016%2Fj.neulet.2006.07.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">AIF translocates to the nucleus in the spinal motor neurons in a mouse model of ALS</span></div><div class="casAuthors">Oh, Yeo Kyoung; Shin, Ki Soon; Kang, Shin Jung</div><div class="citationInfo"><span class="NLM_cas:title">Neuroscience Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">205-210</span>CODEN:
                <span class="NLM_cas:coden">NELED5</span>;
        ISSN:<span class="NLM_cas:issn">0304-3940</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Amyotrophic lateral sclerosis (ALS) is a fatal neurodegenerative disease characterized by selective loss of motor neurons in the brain stem and the spinal cords.  One of the causes for the familial ALS has been attributed to the mutations in copper-zinc superoxide dismutase (SOD1).  Although the toxic function of the mutant enzyme has not been fully understood, the final cell death pathway has been suggested as caspase-dependent.  In the present study, we present evidence that the activation of apoptosis inducing factor (AIF) may play a role to induce motor neuron death during ALS pathogenesis.  In the spinal cord of SOD1 G93A transgenic mice, expression of AIF was detected in the motor neurons and astrocytes.  The level of AIF expression increased as the disease progressed.  In the symptomatic SOD1 G93A transgenic mice, AIF released from the mitochondria and translocated into the nucleus in the motor neurons as evidenced by confocal microscopy and biochem. anal.  These results suggest that AIF may play a role to induce motor neuron death in a mouse model of ALS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxEibnYSSM47Vg90H21EOLACvtfcHk0lhR8651-D43yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsVOms70%253D&md5=e4ad1a7a13b17826df70ba0bde044915</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.neulet.2006.07.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neulet.2006.07.044%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DY.%2BK.%26aulast%3DShin%26aufirst%3DK.%2BS.%26aulast%3DKang%26aufirst%3DS.%2BJ.%26atitle%3DAIF%2520translocates%2520to%2520the%2520nucleus%2520in%2520the%2520spinal%2520motor%2520neurons%2520in%2520a%2520mouse%2520model%2520of%2520ALS%26jtitle%3DNeurosci.%2520Lett.%26date%3D2006%26volume%3D406%26spage%3D205%26epage%3D210%26doi%3D10.1016%2Fj.neulet.2006.07.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Plesnila, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culmsee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gröger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moskowitz, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomgren, K.</span></span> <span> </span><span class="NLM_article-title">Nuclear translocation of apoptosis-inducing factor after focal cerebral ischemia</span>. <i>J. Cereb. Blood Flow Metab. Suppl.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">458</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1097/00004647-200404000-00011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1097%2F00004647-200404000-00011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=15087715" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=458-466&author=N.+Plesnilaauthor=C.+Zhuauthor=C.+Culmseeauthor=M.+Gr%C3%B6gerauthor=M.+A.+Moskowitzauthor=K.+Blomgren&title=Nuclear+translocation+of+apoptosis-inducing+factor+after+focal+cerebral+ischemia&doi=10.1097%2F00004647-200404000-00011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1097%2F00004647-200404000-00011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00004647-200404000-00011%26sid%3Dliteratum%253Aachs%26aulast%3DPlesnila%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DCulmsee%26aufirst%3DC.%26aulast%3DGr%25C3%25B6ger%26aufirst%3DM.%26aulast%3DMoskowitz%26aufirst%3DM.%2BA.%26aulast%3DBlomgren%26aufirst%3DK.%26atitle%3DNuclear%2520translocation%2520of%2520apoptosis-inducing%2520factor%2520after%2520focal%2520cerebral%2520ischemia%26jtitle%3DJ.%2520Cereb.%2520Blood%2520Flow%2520Metab.%2520Suppl.%26date%3D2004%26volume%3D24%26spage%3D458%26epage%3D466%26doi%3D10.1097%2F00004647-200404000-00011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poitras, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coombs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federoff, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, V. L.</span></span> <span> </span><span class="NLM_article-title">Mediation of poly(ADP-ribose) polymerase-1-dependent cell death by apoptosis-inducing factor</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>297</i></span>,  <span class="NLM_fpage">259</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1126/science.1072221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1126%2Fscience.1072221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12114629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsVCntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=297&publication_year=2002&pages=259-263&author=S.-W.+Yuauthor=H.+Wangauthor=M.+F.+Poitrasauthor=C.+Coombsauthor=W.+J.+Bowersauthor=H.+J.+Federoffauthor=G.+G.+Poirierauthor=T.+M.+Dawsonauthor=V.+L.+Dawson&title=Mediation+of+poly%28ADP-ribose%29+polymerase-1-dependent+cell+death+by+apoptosis-inducing+factor&doi=10.1126%2Fscience.1072221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Mediation of poly(ADP-ribose) polymerase-1-dependent cell death by apoptosis-inducing factor</span></div><div class="casAuthors">Yu, Seong-Woon; Wang, Hongmin; Poitras, Marc F.; Coombs, Carmen; Bowers, William J.; Federoff, Howard J.; Poirier, Guy G.; Dawson, Ted M.; Dawson, Valina L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">297</span>
        (<span class="NLM_cas:issue">5579</span>),
    <span class="NLM_cas:pages">259-263</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase-1 (PARP-1) protects the genome by functioning in the DNA damage surveillance network.  PARP- 1 is also a mediator of cell death after ischemia-reperfusion injury, glutamate excitotoxicity, and various inflammatory processes.  We show that PARP-1 activation is required for translocation of apoptosis-inducing factor (AIF) from the mitochondria to the nucleus and that AIF is necessary for PARP-1-dependent cell death.  N-methyl-N'-nitro-N-nitrosoguanidine, H2O2, and N-methyl-D-aspartate induce AIF translocation and cell death, which is prevented by PARP inhibitors or genetic knockout of PARP-1, but is caspase independent.  Microinjection of an antibody to AIF protects against PARP-1-dependent cytotoxicity.  These data support a model in which PARP-1 activation signals AIF release from mitochondria, resulting in a caspase-independent pathway of programmed cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRtfSpfY4287Vg90H21EOLACvtfcHk0lhiFfXgLdXLMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsVCntr0%253D&md5=5e5b38d90e9bfbcbc149cdf43dad0d4e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1126%2Fscience.1072221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1072221%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DS.-W.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DPoitras%26aufirst%3DM.%2BF.%26aulast%3DCoombs%26aufirst%3DC.%26aulast%3DBowers%26aufirst%3DW.%2BJ.%26aulast%3DFederoff%26aufirst%3DH.%2BJ.%26aulast%3DPoirier%26aufirst%3DG.%2BG.%26aulast%3DDawson%26aufirst%3DT.%2BM.%26aulast%3DDawson%26aufirst%3DV.%2BL.%26atitle%3DMediation%2520of%2520poly%2528ADP-ribose%2529%2520polymerase-1-dependent%2520cell%2520death%2520by%2520apoptosis-inducing%2520factor%26jtitle%3DScience%26date%3D2002%26volume%3D297%26spage%3D259%26epage%3D263%26doi%3D10.1126%2Fscience.1072221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kochanek, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draviam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nathaniel, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, R. S. B.</span></span> <span> </span><span class="NLM_article-title">Intranuclear localization of apoptosis-inducing factor (AIF) and large scale DNA fragmentation after traumatic brain injury in rats and in neuronal cultures exposed to peroxynitrite</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.1046/j.1471-4159.2002.00975.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1046%2Fj.1471-4159.2002.00975.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12091479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsV2jtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2002&pages=181-191&author=X.+Zhangauthor=J.+Chenauthor=S.+H.+Grahamauthor=L.+Duauthor=P.+M.+Kochanekauthor=R.+Draviamauthor=F.+Guoauthor=P.+D.+Nathanielauthor=C.+Szaboauthor=S.+C.+Watkinsauthor=R.+S.+B.+Clark&title=Intranuclear+localization+of+apoptosis-inducing+factor+%28AIF%29+and+large+scale+DNA+fragmentation+after+traumatic+brain+injury+in+rats+and+in+neuronal+cultures+exposed+to+peroxynitrite&doi=10.1046%2Fj.1471-4159.2002.00975.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Intranuclear localization of apoptosis-inducing factor (AIF) and large scale DNA fragmentation after traumatic brain injury in rats and in neuronal cultures exposed to peroxynitrite</span></div><div class="casAuthors">Zhang, Xiaopeng; Chen, Jun; Graham, Steven H.; Du, Lina; Kochanek, Patrick M.; Draviam, Romesh; Guo, Fengli; Nathaniel, Paula D.; Szabo, Csaba; Watkins, Simon C.; Clark, Robert S. B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">181-191</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science Ltd.</span>)
        </div><div class="casAbstract">Programmed cell death occurs after ischemic, excitotoxic, and traumatic brain injury (TBI).  Recently, a caspase-independent pathway involving intranuclear translocation of mitochondrial apoptosis-inducing factor (AIF) has been reported in vitro; but whether this occurs after acute brain injury was unknown.  To address this question adult rats were sacrificed at various times after TBI.  Western blot anal. on subcellular protein fractions demonstrated intranuclear localization of AIF in ipsilateral cortex and hippocampus at 2-72 h.  Immunocytochem. anal. showed AIF labeling in neuronal nuclei with DNA fragmentation in the ipsilateral cortex and hippocampus.  Immunoelectronmicroscopy verified intranuclear localization of AIF in hippocampal neurons after TBI, primarily in regions of euchromatin.  Large-scale DNA fragmentation (∼50 kbp), a signature event in AIF-mediated cell death, was detected in ipsilateral cortex and hippocampi by 6 h.  Neuron-enriched cultures exposed to peroxynitrite also demonstrated intranuclear AIF and large-scale DNA fragmentation concurrent with impaired mitochondrial respiration and cell death, events that are inhibited by treatment with a peroxynitrite decompn. catalyst.  Intranuclear localization of AIF and large-scale DNA fragmentation occurs after TBI and in neurons under conditions of oxidative/nitrosative stress, providing the first evidence of this alternative mechanism by which programmed cell death may proceed in neurons after brain injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLmABPAgQYZbVg90H21EOLACvtfcHk0lhiFfXgLdXLMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsV2jtrs%253D&md5=0a448a90b84fd3fa3ecbf5602f1798a9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1046%2Fj.1471-4159.2002.00975.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1471-4159.2002.00975.x%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DGraham%26aufirst%3DS.%2BH.%26aulast%3DDu%26aufirst%3DL.%26aulast%3DKochanek%26aufirst%3DP.%2BM.%26aulast%3DDraviam%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DF.%26aulast%3DNathaniel%26aufirst%3DP.%2BD.%26aulast%3DSzabo%26aufirst%3DC.%26aulast%3DWatkins%26aufirst%3DS.%2BC.%26aulast%3DClark%26aufirst%3DR.%2BS.%2BB.%26atitle%3DIntranuclear%2520localization%2520of%2520apoptosis-inducing%2520factor%2520%2528AIF%2529%2520and%2520large%2520scale%2520DNA%2520fragmentation%2520after%2520traumatic%2520brain%2520injury%2520in%2520rats%2520and%2520in%2520neuronal%2520cultures%2520exposed%2520to%2520peroxynitrite%26jtitle%3DJ.%2520Neurochem.%26date%3D2002%26volume%3D82%26spage%3D181%26epage%3D191%26doi%3D10.1046%2Fj.1471-4159.2002.00975.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sabirzhanov, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoica, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanscom, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piao, C.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faden, A. I.</span></span> <span> </span><span class="NLM_article-title">Over-expression of HSP70 attenuates caspase-dependent and caspase-independent pathways and inhibits neuronal apoptosis</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">554</span>, <span class="refDoi"> DOI: 10.1111/j.1471-4159.2012.07927.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1111%2Fj.1471-4159.2012.07927.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=22909049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFSnsb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2012&pages=542-554&author=B.+Sabirzhanovauthor=B.+A.+Stoicaauthor=M.+Hanscomauthor=C.-S.+Piaoauthor=A.+I.+Faden&title=Over-expression+of+HSP70+attenuates+caspase-dependent+and+caspase-independent+pathways+and+inhibits+neuronal+apoptosis&doi=10.1111%2Fj.1471-4159.2012.07927.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Over-expression of HSP70 attenuates caspase-dependent and caspase-independent pathways and inhibits neuronal apoptosis</span></div><div class="casAuthors">Sabirzhanov, Boris; Stoica, Bogdan A.; Hanscom, Marie; Piao, Chun-Shu; Faden, Alan I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">3&4</span>),
    <span class="NLM_cas:pages">542-554</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">HSP70 is a member of the family of heat-shock proteins that are known to be up-regulated in neurons following injury and/or stress.  HSP70 over-expression has been linked to neuroprotection in multiple models, including neurodegenerative disorders.  In contrast, less is known about the neuroprotective effects of HSP70 in neuronal apoptosis and with regard to modulation of programmed cell death (PCD) mechanisms in neurons.  The authors examd. the effects of HSP70 over-expression by transfection with HSP70-expression plasmids in primary cortical neurons and the SH-SY5Y neuronal cell line using four independent models of apoptosis: etoposide, staurosporine, C2-ceramide, and β-Amyloid.  In these apoptotic models, neurons transfected with the HSP70 construct showed significantly reduced induction of nuclear apoptotic markers and/or cell death.  Furthermore, the authors demonstrated that HSP70 binds and potentially inactivates Apoptotic protease-activating factor 1, as well as apoptosis-inducing factor, key mols. involved in development of caspase-dependent and caspase-independent PCD, resp.  Markers of caspase-dependent PCD, including active caspase-3, caspase-9, and cleaved PARP were attenuated in neurons over-expressing HSP70.  These data indicate that HSP70 protects against neuronal apoptosis and suggest that these effects reflect, at least in part, to inhibition of both caspase-dependent and caspase-independent PCD pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4WvC0W71ZZbVg90H21EOLACvtfcHk0lhiFfXgLdXLMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFSnsb%252FF&md5=437751f997c5e44d935b615b6920f48c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1111%2Fj.1471-4159.2012.07927.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1471-4159.2012.07927.x%26sid%3Dliteratum%253Aachs%26aulast%3DSabirzhanov%26aufirst%3DB.%26aulast%3DStoica%26aufirst%3DB.%2BA.%26aulast%3DHanscom%26aufirst%3DM.%26aulast%3DPiao%26aufirst%3DC.-S.%26aulast%3DFaden%26aufirst%3DA.%2BI.%26atitle%3DOver-expression%2520of%2520HSP70%2520attenuates%2520caspase-dependent%2520and%2520caspase-independent%2520pathways%2520and%2520inhibits%2520neuronal%2520apoptosis%26jtitle%3DJ.%2520Neurochem.%26date%3D2012%26volume%3D123%26spage%3D542%26epage%3D554%26doi%3D10.1111%2Fj.1471-4159.2012.07927.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Öxler, E.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culmsee, C.</span></span> <span> </span><span class="NLM_article-title">AIF depletion provides neuroprotection through a preconditioning effect</span>. <i>Apoptosis</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1027</span>– <span class="NLM_lpage">1038</span>, <span class="refDoi"> DOI: 10.1007/s10495-012-0748-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2Fs10495-012-0748-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=22865232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A280%3ADC%252BC38fls1Gktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=1027-1038&author=E.-M.+%C3%96xlerauthor=A.+Dolgaauthor=C.+Culmsee&title=AIF+depletion+provides+neuroprotection+through+a+preconditioning+effect&doi=10.1007%2Fs10495-012-0748-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">AIF depletion provides neuroprotection through a preconditioning effect</span></div><div class="casAuthors">Oxler Eva-Maria; Dolga Amalia; Culmsee C</div><div class="citationInfo"><span class="NLM_cas:title">Apoptosis : an international journal on programmed cell death</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1027-38</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Previous studies established a major role for apoptosis inducing factor (AIF) in neuronal cell death after acute brain injury.  For example, AIF translocation from mitochondria to the nucleus determined delayed neuronal death, whereas reduced AIF expression provided neuroprotective effects in models of cerebral ischemia or brain trauma.  The question remains, however, why reduced AIF levels are sufficient to mediate neuroprotection, since only very little AIF translocation to the nucleus is required for induction of cell death.  Thus, the present study addresses the question, whether AIF gene silencing affects intrinsic death pathways upstream of nuclear translocation at the level of the mitochondria.  Using MTT assays and real-time cell impedance measurements we confirmed the protective effect of AIF siRNA against glutamate toxicity in immortalized mouse hippocampal HT-22 neurons.  Further, AIF siRNA prevented glutamate-induced mitochondrial fragmentation and loss of mitochondrial membrane potential.  The protection of mitochondrial integrity was associated with preserved ATP levels, attenuated increases in lipid peroxidation and reduced complex I expression levels.  Notably, low concentrations of the complex I inhibitor rotenone (20 nM), provided similar protective effects against glutamate toxicity at the mitochondrial level.  These results expose a preconditioning effect as a mechanism for neuroprotection mediated by AIF depletion.  In particular, they point out an association between mitochondrial complex I and AIF, which regulate each other's stability in mitochondria.  Overall, these findings postulate that AIF depletion mediates a preconditioning effect protecting neuronal cells from subsequent glutamate toxicity through reduced levels of complex I protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTbRQBo_3oNhtbHMBt0mMIXfW6udTcc2eZkIfIhhev-srntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38fls1Gktw%253D%253D&md5=0c90a3506b4c8b0a2192c0aee8501808</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs10495-012-0748-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10495-012-0748-8%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2596xler%26aufirst%3DE.-M.%26aulast%3DDolga%26aufirst%3DA.%26aulast%3DCulmsee%26aufirst%3DC.%26atitle%3DAIF%2520depletion%2520provides%2520neuroprotection%2520through%2520a%2520preconditioning%2520effect%26jtitle%3DApoptosis%26date%3D2012%26volume%3D17%26spage%3D1027%26epage%3D1038%26doi%3D10.1007%2Fs10495-012-0748-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tretter, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sipos, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam-Vizi, V.</span></span> <span> </span><span class="NLM_article-title">Initiation of neuronal damage by complex I deficiency and oxidative stress in Parkinson’s disease</span>. <i>Neurochem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">569</span>– <span class="NLM_lpage">577</span>, <span class="refDoi"> DOI: 10.1023/b:nere.0000014827.94562.4b</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1023%2FB%3ANERE.0000014827.94562.4b" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=15038604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpslWitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2004&pages=569-577&author=L.+Tretterauthor=I.+Siposauthor=V.+Adam-Vizi&title=Initiation+of+neuronal+damage+by+complex+I+deficiency+and+oxidative+stress+in+Parkinson%E2%80%99s+disease&doi=10.1023%2Fb%3Anere.0000014827.94562.4b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Initiation of neuronal damage by complex I deficiency and oxidative stress in Parkinson's Disease</span></div><div class="casAuthors">Tretter, Laszlo; Sipos, Ildiko; Adam-Vizi, Vera</div><div class="citationInfo"><span class="NLM_cas:title">Neurochemical Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">569-577</span>CODEN:
                <span class="NLM_cas:coden">NEREDZ</span>;
        ISSN:<span class="NLM_cas:issn">0364-3190</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic/Plenum Publishers</span>)
        </div><div class="casAbstract">A review.  Oxidative stress and partial deficiencies of mitochondrial complex I appear to be key factors in the pathogenesis of Parkinson's disease.  They are interconnected; complex I inhibition results in an enhanced prodn. of reactive oxygen species (ROS), which in turn will inhibit complex I.  Partial inhibition of complex I in nerve terminals is sufficient for in situ mitochondria to generate more ROS.  H2O2 plays a major role in inhibiting complex I as well as a key metabolic enzyme, α-ketoglutarate dehydrogenase.  The vicious cycle resulting from partial inhibition of complex I and/or an inherently higher ROS prodn. in dopaminergic neurons leads over time to excessive oxidative stress and ATP deficit that eventually will result in cell death in the nigro-striatal pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnUGJVRLCHLVg90H21EOLACvtfcHk0ljAH8ZgS9C8Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpslWitA%253D%253D&md5=582748bd96a6ffa6edcc55c8bc99c576</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1023%2FB%3ANERE.0000014827.94562.4b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FB%253ANERE.0000014827.94562.4b%26sid%3Dliteratum%253Aachs%26aulast%3DTretter%26aufirst%3DL.%26aulast%3DSipos%26aufirst%3DI.%26aulast%3DAdam-Vizi%26aufirst%3DV.%26atitle%3DInitiation%2520of%2520neuronal%2520damage%2520by%2520complex%2520I%2520deficiency%2520and%2520oxidative%2520stress%2520in%2520Parkinson%25E2%2580%2599s%2520disease%26jtitle%3DNeurochem.%2520Res.%26date%3D2004%26volume%3D29%26spage%3D569%26epage%3D577%26doi%3D10.1023%2Fb%3Anere.0000014827.94562.4b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallin, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Candé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomgren, K.</span></span> <span> </span><span class="NLM_article-title">Involvement of apoptosis-inducing factor in neuronal death after hypoxia-ischemia in the neonatal rat brain</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1046/j.1471-4159.2003.01832.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1046%2Fj.1471-4159.2003.01832.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12871572" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlslGhsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2003&pages=306-317&author=C.+Zhuauthor=L.+Qiuauthor=X.+Wangauthor=U.+Hallinauthor=C.+Cand%C3%A9author=G.+Kroemerauthor=H.+Hagbergauthor=K.+Blomgren&title=Involvement+of+apoptosis-inducing+factor+in+neuronal+death+after+hypoxia-ischemia+in+the+neonatal+rat+brain&doi=10.1046%2Fj.1471-4159.2003.01832.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Involvement of apoptosis-inducing factor in neuronal death after hypoxia-ischemia in the neonatal rat brain</span></div><div class="casAuthors">Zhu, Changlian; Qiu, Lin; Wang, Xiaoyang; Hallin, Ulrika; Cande, Celine; Kroemer, Guido; Hagberg, Henrik; Blomgren, Klas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">306-317</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Apoptosis-inducing factor (AIF) triggers apoptosis in a caspase-independent manner.  Here the authors report for the first time involvement of AIF in neuronal death induced by cerebral ischemia.  Unilateral cerebral hypoxia-ischemia (HI) was induced in 7-day-old rats by ligation of the left carotid artery and hypoxia (7.7% O2) for 55 min.  AIF release from mitochondria and AIF translocation to nuclei was detected immediately after HI, and only in damaged areas, as judged by the concurrent loss of MAP-2.  AIF release was detected earlier than that of cytochrome c.  Cells with AIF-pos. nuclei displayed nuclear condensation and signs of DNA damage.  The no. of AIF-pos. nuclei showed a pos. correlation with the infarct vol. 72 h post-HI, and this was not changed by treating the animals with boc-Asp-fmk (BAF), a multicaspase inhibitor.  BAF treatment reduced the activity of caspase-3, -2 and -9 (78, 73 and 33%, resp.), and prevented caspase-dependent fodrin cleavage in vivo, but did not affect AIF release from mitochondria or the frequency of pos. nuclear AIF or DNA damage 72 h post-HI, indicating that these processes occurred in a caspase-independent fashion.  In summary, AIF-mediated cell death may be an important mechanism of HI-induced neuronal loss in the immature brain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox7tE9HIjSC7Vg90H21EOLACvtfcHk0ljAH8ZgS9C8Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlslGhsbg%253D&md5=6576f5b468020f839aef71179329cbf3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1046%2Fj.1471-4159.2003.01832.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1471-4159.2003.01832.x%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DQiu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHallin%26aufirst%3DU.%26aulast%3DCand%25C3%25A9%26aufirst%3DC.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DHagberg%26aufirst%3DH.%26aulast%3DBlomgren%26aufirst%3DK.%26atitle%3DInvolvement%2520of%2520apoptosis-inducing%2520factor%2520in%2520neuronal%2520death%2520after%2520hypoxia-ischemia%2520in%2520the%2520neonatal%2520rat%2520brain%26jtitle%3DJ.%2520Neurochem.%26date%3D2003%26volume%3D86%26spage%3D306%26epage%3D317%26doi%3D10.1046%2Fj.1471-4159.2003.01832.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Candé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vahsen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouranti, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spahr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luban, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrido, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penninger, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">AIF and cyclophilin A cooperate in apoptosis-associated chromatinolysis</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1514</span>– <span class="NLM_lpage">1521</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1207279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Fsj.onc.1207279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=14716299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFOht7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=1514-1521&author=C.+Cand%C3%A9author=N.+Vahsenauthor=I.+Kourantiauthor=E.+Schmittauthor=E.+Daugasauthor=C.+Spahrauthor=J.+Lubanauthor=R.+T.+Kroemerauthor=F.+Giordanettoauthor=C.+Garridoauthor=J.+M.+Penningerauthor=G.+Kroemer&title=AIF+and+cyclophilin+A+cooperate+in+apoptosis-associated+chromatinolysis&doi=10.1038%2Fsj.onc.1207279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">AIF and cyclophilin A cooperate in apoptosis-associated chromatinolysis</span></div><div class="casAuthors">Cande, Celine; Vahsen, Nicola; Kouranti, Ilektra; Schmitt, Elise; Daugas, Eric; Spahr, Chris; Luban, Jeremy; Kroemer, Romano T.; Giordanetto, Fabrizio; Garrido, Carmen; Penninger, Josef M.; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1514-1521</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cyclophilin A (CypA) was detd. to interact with apoptosis-inducing factor (AIF) by mass spectroscopy, coimmunopptn., pull-down assays, and mol. modeling.  During the initial, caspase-independent stage of chromatin condensation that accompanies apoptosis, AIF and CypA were found to coimmunolocalize in the nucleus.  Recombinant AIF and CypA proteins synergized in vitro in the degrdn. of plasmid DNA, as well as in the capacity to induce DNA loss in purified nuclei.  The apoptogenic cooperation between AIF and CypA did not rely on the CypA peptidyl-prolyl cis-trans isomerase activity.  In Cyp-expressing cells, AIF overexpression augmented apoptotic chromatinolysis.  The AIF-dependent large-scale DNA fragmentation was less pronounced in CypA knockout cells as compared to controls.  AIF mutants lacking the CypA-binding domain were inefficient apoptosis sensitizers in transfection expts.  Moreover, AIF failed to sensitize CypA knockout cells to apoptosis induction, and this defect in the AIF response was reversed by reintroduction of the CypA gene into CypA-deficient cells.  In summary, AIF and CypA collaborate in chromatinolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7ACyaVMkTobVg90H21EOLACvtfcHk0ljAH8ZgS9C8Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFOht7w%253D&md5=86e306afe590b4454aacf11bb911e52e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1207279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1207279%26sid%3Dliteratum%253Aachs%26aulast%3DCand%25C3%25A9%26aufirst%3DC.%26aulast%3DVahsen%26aufirst%3DN.%26aulast%3DKouranti%26aufirst%3DI.%26aulast%3DSchmitt%26aufirst%3DE.%26aulast%3DDaugas%26aufirst%3DE.%26aulast%3DSpahr%26aufirst%3DC.%26aulast%3DLuban%26aufirst%3DJ.%26aulast%3DKroemer%26aufirst%3DR.%2BT.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DGarrido%26aufirst%3DC.%26aulast%3DPenninger%26aufirst%3DJ.%2BM.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DAIF%2520and%2520cyclophilin%2520A%2520cooperate%2520in%2520apoptosis-associated%2520chromatinolysis%26jtitle%3DOncogene%26date%3D2004%26volume%3D23%26spage%3D1514%26epage%3D1521%26doi%3D10.1038%2Fsj.onc.1207279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reuther, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scognamiglio, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolga, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plesnila, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culmsee, C.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the AIF/CypA complex protects against intrinsic death pathways induced by oxidative stress</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e993</span> <span class="refDoi"> DOI: 10.1038/cddis.2013.518</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Fcddis.2013.518" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=24434516" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&author=N.+Dotiauthor=C.+Reutherauthor=P.+L.+Scognamiglioauthor=A.+M.+Dolgaauthor=N.+Plesnilaauthor=M.+Ruvoauthor=C.+Culmsee&title=Inhibition+of+the+AIF%2FCypA+complex+protects+against+intrinsic+death+pathways+induced+by+oxidative+stress&doi=10.1038%2Fcddis.2013.518"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2013.518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2013.518%26sid%3Dliteratum%253Aachs%26aulast%3DDoti%26aufirst%3DN.%26aulast%3DReuther%26aufirst%3DC.%26aulast%3DScognamiglio%26aufirst%3DP.%2BL.%26aulast%3DDolga%26aufirst%3DA.%2BM.%26aulast%3DPlesnila%26aufirst%3DN.%26aulast%3DRuvo%26aufirst%3DM.%26aulast%3DCulmsee%26aufirst%3DC.%26atitle%3DInhibition%2520of%2520the%2520AIF%252FCypA%2520complex%2520protects%2520against%2520intrinsic%2520death%2520pathways%2520induced%2520by%2520oxidative%2520stress%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2014%26volume%3D5%26doi%3D10.1038%2Fcddis.2013.518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Artus, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boujrad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouharrour, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunelle, M.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuste, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenormand, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousselle, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namane, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">England, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzo, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Susin, S. A.</span></span> <span> </span><span class="NLM_article-title">AIF promotes chromatinolysis and caspase-independent programmed necrosis by interacting with histone H2AX</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1585</span>– <span class="NLM_lpage">1599</span>, <span class="refDoi"> DOI: 10.1038/emboj.2010.43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Femboj.2010.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=20360685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvVGlu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=1585-1599&author=C.+Artusauthor=H.+Boujradauthor=A.+Bouharrourauthor=M.-N.+Brunelleauthor=S.+Hoosauthor=V.+J.+Yusteauthor=P.+Lenormandauthor=J.-C.+Rousselleauthor=A.+Namaneauthor=P.+Englandauthor=H.+K.+Lorenzoauthor=S.+A.+Susin&title=AIF+promotes+chromatinolysis+and+caspase-independent+programmed+necrosis+by+interacting+with+histone+H2AX&doi=10.1038%2Femboj.2010.43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">AIF promotes chromatinolysis and caspase-independent programmed necrosis by interacting with histone H2AX</span></div><div class="casAuthors">Artus, Cedric; Boujrad, Hanan; Bouharrour, Aida; Brunelle, Marie-Noelle; Hoos, Sylviane; Yuste, Victor J.; Lenormand, Pascal; Rousselle, Jean-Claude; Namane, Abdelkader; England, Patrick; Lorenzo, Hans K.; Susin, Santos A.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1585-1599</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Programmed necrosis induced by DNA alkylating agents, such as MNNG, is a caspase-independent mode of cell death mediated by apoptosis-inducing factor (AIF).  After poly(ADP-ribose) polymerase 1, calpain, and Bax activation, AIF moves from the mitochondria to the nucleus where it induces chromatinolysis and cell death.  The mechanisms underlying the nuclear action of AIF are, however, largely unknown.  We show here that, through its C-terminal proline-rich binding domain (PBD, residues 543-559), AIF assocs. in the nucleus with histone H2AX.  This interaction regulates chromatinolysis and programmed necrosis by generating an active DNA-degrading complex with cyclophilin A (CypA).  Deletion or directed mutagenesis in the AIF C-terminal PBD abolishes AIF/H2AX interaction and AIF-mediated chromatinolysis.  H2AX genetic ablation or CypA downregulation confers resistance to programmed necrosis.  AIF fails to induce chromatinolysis in H2AX or CypA-deficient nuclei.  We also establish that H2AX is phosphorylated at Ser139 after MNNG treatment and that this phosphorylation is crit. for caspase-independent programmed necrosis.  Overall, our data shed new light in the mechanisms regulating programmed necrosis, elucidate a key nuclear partner of AIF, and uncover an AIF apoptogenic motif.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ8ffTkggQzrVg90H21EOLACvtfcHk0liuXFnM1Muc_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvVGlu7c%253D&md5=5788f2abc9a49fe557591dbb3c019899</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Femboj.2010.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2010.43%26sid%3Dliteratum%253Aachs%26aulast%3DArtus%26aufirst%3DC.%26aulast%3DBoujrad%26aufirst%3DH.%26aulast%3DBouharrour%26aufirst%3DA.%26aulast%3DBrunelle%26aufirst%3DM.-N.%26aulast%3DHoos%26aufirst%3DS.%26aulast%3DYuste%26aufirst%3DV.%2BJ.%26aulast%3DLenormand%26aufirst%3DP.%26aulast%3DRousselle%26aufirst%3DJ.-C.%26aulast%3DNamane%26aufirst%3DA.%26aulast%3DEngland%26aufirst%3DP.%26aulast%3DLorenzo%26aufirst%3DH.%2BK.%26aulast%3DSusin%26aufirst%3DS.%2BA.%26atitle%3DAIF%2520promotes%2520chromatinolysis%2520and%2520caspase-independent%2520programmed%2520necrosis%2520by%2520interacting%2520with%2520histone%2520H2AX%26jtitle%3DEMBO%2520J.%26date%3D2010%26volume%3D29%26spage%3D1585%26epage%3D1599%26doi%3D10.1038%2Femboj.2010.43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wissing, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludovico, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herker, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Büttner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelhardt, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Link, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proksch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corte-Real, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fröhlich, K.-U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manns, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Candé, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigrist, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madeo, F.</span></span> <span> </span><span class="NLM_article-title">An AIF orthologue regulates apoptosis in yeast</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>166</i></span>,  <span class="NLM_fpage">969</span>– <span class="NLM_lpage">974</span>, <span class="refDoi"> DOI: 10.1083/jcb.200404138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1083%2Fjcb.200404138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=15381687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotVehtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2004&pages=969-974&author=S.+Wissingauthor=P.+Ludovicoauthor=E.+Herkerauthor=S.+B%C3%BCttnerauthor=S.+M.+Engelhardtauthor=T.+Deckerauthor=A.+Linkauthor=A.+Prokschauthor=F.+Rodriguesauthor=M.+Corte-Realauthor=K.-U.+Fr%C3%B6hlichauthor=J.+Mannsauthor=C.+Cand%C3%A9author=S.+J.+Sigristauthor=G.+Kroemerauthor=F.+Madeo&title=An+AIF+orthologue+regulates+apoptosis+in+yeast&doi=10.1083%2Fjcb.200404138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">An AIF orthologue regulates apoptosis in yeast</span></div><div class="casAuthors">Wissing, Silke; Ludovico, Paula; Herker, Eva; Buettner, Sabrina; Engelhardt, Silvia M.; Decker, Thorsten; Link, Alexander; Proksch, Astrid; Rodrigues, Fernando; Corte-Real, Manuela; Froehlich, Kai-Uwe; Manns, Joachim; Cande, Celine; Sigrist, Stephan J.; Kroemer, Guido; Madeo, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">969-974</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9525</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Apoptosis-inducing factor (AIF), a key regulator of cell death, is essential for normal mammalian development and participates in pathol. apoptosis.  The proapoptotic nature of AIF and its mode of action are controversial.  Here, we show that the yeast AIF homolog Ynr074cp controls yeast apoptosis.  Similar to mammalian AIF, Ynr074cp is located in mitochondria and translocates to the nucleus of yeast cells in response to apoptotic stimuli.  Purified Ynr074cp degrades yeast nuclei and plasmid DNA.  YNR074C disruption rescues yeast cells from oxygen stress and delays age-induced apoptosis.  Conversely, overexpression of Ynr074cp strongly stimulates apoptotic cell death induced by hydrogen peroxide and this effect is attenuated by disruption of cyclophilin A or the yeast caspase YCA1.  We conclude that Ynr074cp is a cell death effector in yeast and rename it AIF-1 (Aif1p, gene AIF1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPaxPrMBVeT7Vg90H21EOLACvtfcHk0liuXFnM1Muc_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotVehtL4%253D&md5=a526d50e744e440f77eebc4ebd124ba5</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200404138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200404138%26sid%3Dliteratum%253Aachs%26aulast%3DWissing%26aufirst%3DS.%26aulast%3DLudovico%26aufirst%3DP.%26aulast%3DHerker%26aufirst%3DE.%26aulast%3DB%25C3%25BCttner%26aufirst%3DS.%26aulast%3DEngelhardt%26aufirst%3DS.%2BM.%26aulast%3DDecker%26aufirst%3DT.%26aulast%3DLink%26aufirst%3DA.%26aulast%3DProksch%26aufirst%3DA.%26aulast%3DRodrigues%26aufirst%3DF.%26aulast%3DCorte-Real%26aufirst%3DM.%26aulast%3DFr%25C3%25B6hlich%26aufirst%3DK.-U.%26aulast%3DManns%26aufirst%3DJ.%26aulast%3DCand%25C3%25A9%26aufirst%3DC.%26aulast%3DSigrist%26aufirst%3DS.%2BJ.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DMadeo%26aufirst%3DF.%26atitle%3DAn%2520AIF%2520orthologue%2520regulates%2520apoptosis%2520in%2520yeast%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2004%26volume%3D166%26spage%3D969%26epage%3D974%26doi%3D10.1083%2Fjcb.200404138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span> <span> </span><span class="NLM_article-title">Inhibiting the interaction between apoptosis-inducing factor and cyclophilin A prevents brain injury in neonatal mice after hypoxia-ischemia</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">108088</span>, <span class="refDoi"> DOI: 10.1016/j.neuropharm.2020.108088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2Fj.neuropharm.2020.108088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=32277944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BB3cXos1SitL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2020&pages=108088&author=J.+Rodriguezauthor=C.+Xieauthor=T.+Liauthor=Y.+Sunauthor=Y.+Wangauthor=Y.+Xuauthor=K.+Liauthor=S.+Zhangauthor=K.+Zhouauthor=Y.+Wangauthor=C.+Mallardauthor=H.+Hagbergauthor=N.+Dotiauthor=X.+Wangauthor=C.+Zhu&title=Inhibiting+the+interaction+between+apoptosis-inducing+factor+and+cyclophilin+A+prevents+brain+injury+in+neonatal+mice+after+hypoxia-ischemia&doi=10.1016%2Fj.neuropharm.2020.108088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting the interaction between apoptosis-inducing factor and cyclophilin A prevents brain injury in neonatal mice after hypoxia-ischemia</span></div><div class="casAuthors">Rodriguez, Juan; Xie, Cuicui; Li, Tao; Sun, Yanyan; Wang, Yafeng; Xu, Yiran; Li, Kenan; Zhang, Shan; Zhou, Kai; Wang, Yong; Mallard, Carina; Hagberg, Henrik; Doti, Nunzianna; Wang, Xiaoyang; Zhu, Changlian</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">108088</span>CODEN:
                <span class="NLM_cas:coden">NEPHBW</span>;
        ISSN:<span class="NLM_cas:issn">0028-3908</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Blocking the AIF/CypA interaction protects against glutamate-induced neuronal cell death in vitro, and the purpose of this study was to det. the in vivo effect of an AIF/CypA interaction blocking peptide (AIF(370-394)-TAT) on neonatal mouse brain injury after HI.  The pups were treated with AIF (370-394)-TAT peptide intranasally prior to HI.  Brain injury was significantly reduced at 72 h after HI in AIF(370-394)-TAT peptide treatment group compared to vehicle-only treatment for both the gray matter and the subcortical white matter, and the neuroprotection was more pronounced in males than in females.  Neuronal cell death was evaluated in males at 8 h and 24 h post-HI, and it was decreased significantly in the CA1 region of the hippocampus and the nucleus habenularis region after AIF(370-394)-TAT treatment.  Caspase-independent apoptosis was decreased in the cortex, striatum, and nucleus habenularis after AIF(370-394)-TAT treatment, but no significant change was found on caspase-dependent apoptosis as indicated by the no. of active caspase-3-labeled cells.  Further anal. showed that both AIF and CypA nuclear accumulation were decreased after treatment with the AIF(370-394)-TAT peptide.  These results suggest that AIF(370-394)-TAT inhibited AIF/CypA translocation to the nucleus and reduced HI-induced caspase-independent apoptosis and brain injury in young male mice, suggesting that blocking AIF/CypA might be potential therapeutic target for neonatal brain injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmR2RcyHMdpLVg90H21EOLACvtfcHk0lh_NejmkrAyEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXos1SitL4%253D&md5=a4104264354bbbaaf1d6f7d55afce584</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2020.108088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2020.108088%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMallard%26aufirst%3DC.%26aulast%3DHagberg%26aufirst%3DH.%26aulast%3DDoti%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DC.%26atitle%3DInhibiting%2520the%2520interaction%2520between%2520apoptosis-inducing%2520factor%2520and%2520cyclophilin%2520A%2520prevents%2520brain%2520injury%2520in%2520neonatal%2520mice%2520after%2520hypoxia-ischemia%26jtitle%3DNeuropharmacology%26date%3D2020%26volume%3D171%26spage%3D108088%26doi%3D10.1016%2Fj.neuropharm.2020.108088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chelko, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keceli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrimi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asimaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beti, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amat-Codina, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedja, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichnell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calkins, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Rourke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halushka, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melloni, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffitz, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Judge, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitsis, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Lisa, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paolocci, N.</span></span> <span> </span><span class="NLM_article-title">Exercise triggers CAPN1-mediated AIF truncation, inducing myocyte cell death in arrhythmogenic cardiomyopathy</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">eabf0891</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.abf0891</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1126%2Fscitranslmed.abf0891" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=33597260" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2021&author=S.+P.+Chelkoauthor=G.+Keceliauthor=A.+Carpiauthor=N.+Dotiauthor=J.+Agrimiauthor=A.+Asimakiauthor=C.+B.+Betiauthor=M.+Miyamotoauthor=N.+Amat-Codinaauthor=D.+Bedjaauthor=A.-C.+Weiauthor=B.+Murrayauthor=C.+Tichnellauthor=C.+Kwonauthor=H.+Calkinsauthor=C.+A.+Jamesauthor=B.+O%E2%80%99Rourkeauthor=M.+K.+Halushkaauthor=E.+Melloniauthor=J.+E.+Saffitzauthor=D.+P.+Judgeauthor=M.+Ruvoauthor=R.+N.+Kitsisauthor=P.+Andersenauthor=F.+Di+Lisaauthor=N.+Paolocci&title=Exercise+triggers+CAPN1-mediated+AIF+truncation%2C+inducing+myocyte+cell+death+in+arrhythmogenic+cardiomyopathy&doi=10.1126%2Fscitranslmed.abf0891"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.abf0891&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.abf0891%26sid%3Dliteratum%253Aachs%26aulast%3DChelko%26aufirst%3DS.%2BP.%26aulast%3DKeceli%26aufirst%3DG.%26aulast%3DCarpi%26aufirst%3DA.%26aulast%3DDoti%26aufirst%3DN.%26aulast%3DAgrimi%26aufirst%3DJ.%26aulast%3DAsimaki%26aufirst%3DA.%26aulast%3DBeti%26aufirst%3DC.%2BB.%26aulast%3DMiyamoto%26aufirst%3DM.%26aulast%3DAmat-Codina%26aufirst%3DN.%26aulast%3DBedja%26aufirst%3DD.%26aulast%3DWei%26aufirst%3DA.-C.%26aulast%3DMurray%26aufirst%3DB.%26aulast%3DTichnell%26aufirst%3DC.%26aulast%3DKwon%26aufirst%3DC.%26aulast%3DCalkins%26aufirst%3DH.%26aulast%3DJames%26aufirst%3DC.%2BA.%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DB.%26aulast%3DHalushka%26aufirst%3DM.%2BK.%26aulast%3DMelloni%26aufirst%3DE.%26aulast%3DSaffitz%26aufirst%3DJ.%2BE.%26aulast%3DJudge%26aufirst%3DD.%2BP.%26aulast%3DRuvo%26aufirst%3DM.%26aulast%3DKitsis%26aufirst%3DR.%2BN.%26aulast%3DAndersen%26aufirst%3DP.%26aulast%3DDi%2BLisa%26aufirst%3DF.%26aulast%3DPaolocci%26aufirst%3DN.%26atitle%3DExercise%2520triggers%2520CAPN1-mediated%2520AIF%2520truncation%252C%2520inducing%2520myocyte%2520cell%2520death%2520in%2520arrhythmogenic%2520cardiomyopathy%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2021%26volume%3D13%26doi%3D10.1126%2Fscitranslmed.abf0891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farina, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Sorbo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambery, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caporale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leoni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mascanzoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattorusso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span> <span> </span><span class="NLM_article-title">Structural and biochemical insights of CypA and AIF interaction</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1138</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-01337-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Fs41598-017-01337-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=28442737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A280%3ADC%252BC1crhtFCjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=1138&author=B.+Farinaauthor=G.+Di+Sorboauthor=A.+Chamberyauthor=A.+Caporaleauthor=G.+Leoniauthor=R.+Russoauthor=F.+Mascanzoniauthor=D.+Raimondoauthor=R.+Fattorussoauthor=M.+Ruvoauthor=N.+Doti&title=Structural+and+biochemical+insights+of+CypA+and+AIF+interaction&doi=10.1038%2Fs41598-017-01337-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and biochemical insights of CypA and AIF interaction</span></div><div class="casAuthors">Farina Biancamaria; Di Sorbo Gianluigi; Caporale Andrea; Mascanzoni Fabiola; Ruvo Menotti; Doti Nunzianna; Di Sorbo Gianluigi; Chambery Angela; Russo Rosita; Fattorusso Roberto; Leoni Guido; Raimondo Domenico</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1138</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The Cyclophilin A (CypA)/Apoptosis Inducing Factor (AIF) complex is implicated in the DNA degradation in response to various cellular stress conditions, such as oxidative stress, cerebral hypoxia-ischemia and traumatic brain injury.  The pro-apoptotic form of AIF (AIF(Δ1-121)) mainly interacts with CypA through the amino acid region 370-394.  The AIF(370-394) synthetic peptide inhibits complex formation in vitro by binding to CypA and exerts neuroprotection in a model of glutamate-mediated oxidative stress.  Here, the binding site of AIF(Δ1-121) and AIF(370-394) on CypA has been mapped by NMR spectroscopy and biochemical studies, and a molecular model of the complex has been proposed.  We show that AIF(370-394) interacts with CypA on the same surface recognized by AIF(Δ1-121) protein and that the region is very close to the CypA catalytic pocket.  Such region partially overlaps with the binding site of cyclosporin A (CsA), the strongest catalytic inhibitor of CypA.  Our data point toward distinct CypA structural determinants governing the inhibitor selectivity and the differential biological effects of AIF and CsA, and provide new structural insights for designing CypA/AIF selective inhibitors with therapeutic relevance in neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSCGXnCF2P3vJQHPYJuLg8TfW6udTcc2eYl4mS947HyUbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crhtFCjsA%253D%253D&md5=d036ad6254cef5a4b232f3c074bfec66</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-01337-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-01337-8%26sid%3Dliteratum%253Aachs%26aulast%3DFarina%26aufirst%3DB.%26aulast%3DDi%2BSorbo%26aufirst%3DG.%26aulast%3DChambery%26aufirst%3DA.%26aulast%3DCaporale%26aufirst%3DA.%26aulast%3DLeoni%26aufirst%3DG.%26aulast%3DRusso%26aufirst%3DR.%26aulast%3DMascanzoni%26aufirst%3DF.%26aulast%3DRaimondo%26aufirst%3DD.%26aulast%3DFattorusso%26aufirst%3DR.%26aulast%3DRuvo%26aufirst%3DM.%26aulast%3DDoti%26aufirst%3DN.%26atitle%3DStructural%2520and%2520biochemical%2520insights%2520of%2520CypA%2520and%2520AIF%2520interaction%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D1138%26doi%3D10.1038%2Fs41598-017-01337-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farina, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturlese, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mascanzoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caporale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Sorbo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattorusso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span> <span> </span><span class="NLM_article-title">Binding mode of AIF(370-394) peptide to CypA: insights from NMR, label-free and molecular docking studies</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>475</i></span>,  <span class="NLM_fpage">2377</span>– <span class="NLM_lpage">2393</span>, <span class="refDoi"> DOI: 10.1042/bcj20180177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1042%2FBCJ20180177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=29891613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVGjtLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=475&publication_year=2018&pages=2377-2393&author=B.+Farinaauthor=M.+Sturleseauthor=F.+Mascanzoniauthor=A.+Caporaleauthor=A.+Montiauthor=G.+Di+Sorboauthor=R.+Fattorussoauthor=M.+Ruvoauthor=N.+Doti&title=Binding+mode+of+AIF%28370-394%29+peptide+to+CypA%3A+insights+from+NMR%2C+label-free+and+molecular+docking+studies&doi=10.1042%2Fbcj20180177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Binding mode of AIF(370-394) peptide to CypA: insights from NMR, label-free and molecular docking studies</span></div><div class="casAuthors">Farina, Biancamaria; Sturlese, Mattia; Mascanzoni, Fabiola; Caporale, Andrea; Monti, Alessandra; Sorbo, Gianluigi Di; Fattorusso, Roberto; Ruvo, Menotti; Doti, Nunzianna</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">475</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2377-2393</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The complex formation between the proteins apoptosis-inducing factor (AIF) and cyclophilin A (CypA) following oxidative stress in neuronal cells has been suggested as a main target for reverting ischemia-stroke damage.  Recently, a peptide encompassing AIF residues 370-394 has been developed to target the AIF-binding site on CypA, to prevent the assocn. between the two proteins and suppress glutamate-induced cell death in neuronal cells.  Using a combined approach based on NMR spectroscopy, synthesis and in vitro testing of all Ala-scan mutants of the peptide and mol. docking/mol. dynamics, we have generated a detailed model of the AIF (370-394)/CypA complex.  The model suggests us that the central region of the peptide spanning residues V374-K384 mostly interacts with the protein and that for efficient complex inhibition and preservation of CypA activity, it is bent around amino acids F46-G75 of the protein.  The model is consistent with exptl. data also from previous works and supports the concept that the peptide does not interfere with other CypA activities unrelated to AIF activation; therefore, it may serve as an ideal template for generating future non-peptidic antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl93TSrK4VArVg90H21EOLACvtfcHk0lgWvApkxpGGhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVGjtLfI&md5=5cf337043a3a440de781333ae404a385</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1042%2FBCJ20180177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20180177%26sid%3Dliteratum%253Aachs%26aulast%3DFarina%26aufirst%3DB.%26aulast%3DSturlese%26aufirst%3DM.%26aulast%3DMascanzoni%26aufirst%3DF.%26aulast%3DCaporale%26aufirst%3DA.%26aulast%3DMonti%26aufirst%3DA.%26aulast%3DDi%2BSorbo%26aufirst%3DG.%26aulast%3DFattorusso%26aufirst%3DR.%26aulast%3DRuvo%26aufirst%3DM.%26aulast%3DDoti%26aufirst%3DN.%26atitle%3DBinding%2520mode%2520of%2520AIF%2528370-394%2529%2520peptide%2520to%2520CypA%253A%2520insights%2520from%2520NMR%252C%2520label-free%2520and%2520molecular%2520docking%2520studies%26jtitle%3DBiochem.%2520J.%26date%3D2018%26volume%3D475%26spage%3D2377%26epage%3D2393%26doi%3D10.1042%2Fbcj20180177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cande, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephanou, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurbuxani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larochette, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrido, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span> <span> </span><span class="NLM_article-title">DNA binding is required for the apoptogenic action of apoptosis inducing factor</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">680</span>– <span class="NLM_lpage">684</span>, <span class="refDoi"> DOI: 10.1038/nsb836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Fnsb836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12198487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1ajur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=680-684&author=H.+Yeauthor=C.+Candeauthor=N.+C.+Stephanouauthor=S.+Jiangauthor=S.+Gurbuxaniauthor=N.+Larochetteauthor=E.+Daugasauthor=C.+Garridoauthor=G.+Kroemerauthor=H.+Wu&title=DNA+binding+is+required+for+the+apoptogenic+action+of+apoptosis+inducing+factor&doi=10.1038%2Fnsb836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">DNA binding is required for the apoptogenic action of apoptosis inducing factor</span></div><div class="casAuthors">Ye, Hong; Cande, Celine; Stephanou, Nicolas C.; Jiang, Sulin; Gurbuxani, Sundeep; Larochette, Nathanael; Daugas, Eric; Garrido, Carmen; Kroemer, Guido; Wu, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">680-684</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The execution of apoptosis or programmed cell death comprises both caspase-dependent and caspase-independent processes.  Apoptosis inducing factor (AIF) was identified as a major player in caspase-independent cell death.  It induces chromatin condensation and initial DNA cleavage via an unknown mol. mechanism.  Here we report the crystal structure of human AIF at 1.8 Å resoln.  The structure reveals the presence of a strong pos. electrostatic potential at the AIF surface, although the calcd. isoelec. point for the entire protein is neutral.  We show that recombinant AIF interacts with DNA in a sequence-independent manner.  In addn., in cells treated with an apoptotic stimulus, endogenous AIF becomes co-localized with DNA at an early stage of nuclear morphol. changes.  Structure-based mutagenesis shows that DNA-binding defective mutants of AIF fail to induce cell death while retaining nuclear translocation.  The potential DNA-binding site identified from mutagenesis also coincides with computational docking of a DNA duplex.  These observations suggest that AIF-induced nuclear apoptosis requires a direct interaction with DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnsbZhi6U0ALVg90H21EOLACvtfcHk0lgWvApkxpGGhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1ajur8%253D&md5=419d4eb7d4feb72f72614c6b3f72969c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnsb836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb836%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DH.%26aulast%3DCande%26aufirst%3DC.%26aulast%3DStephanou%26aufirst%3DN.%2BC.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DGurbuxani%26aufirst%3DS.%26aulast%3DLarochette%26aufirst%3DN.%26aulast%3DDaugas%26aufirst%3DE.%26aulast%3DGarrido%26aufirst%3DC.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26atitle%3DDNA%2520binding%2520is%2520required%2520for%2520the%2520apoptogenic%2520action%2520of%2520apoptosis%2520inducing%2520factor%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D9%26spage%3D680%26epage%3D684%26doi%3D10.1038%2Fnsb836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturlese, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caporale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roger, J. D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mascanzoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, structural analysis and biochemical studies of stapled AIF(370-394) analogues as ligand of CypA</span>. <i>BBA, Biochim. Biophys. Acta, Gen. Subj.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>1864</i></span>,  <span class="NLM_fpage">129717</span>, <span class="refDoi"> DOI: 10.1016/j.bbagen.2020.129717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2Fj.bbagen.2020.129717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=32861757" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1864&publication_year=2020&pages=129717&author=A.+Montiauthor=M.+Sturleseauthor=A.+Caporaleauthor=J.+D.+A.+Rogerauthor=F.+Mascanzoniauthor=M.+Ruvoauthor=N.+Doti&title=Design%2C+synthesis%2C+structural+analysis+and+biochemical+studies+of+stapled+AIF%28370-394%29+analogues+as+ligand+of+CypA&doi=10.1016%2Fj.bbagen.2020.129717"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2020.129717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2020.129717%26sid%3Dliteratum%253Aachs%26aulast%3DMonti%26aufirst%3DA.%26aulast%3DSturlese%26aufirst%3DM.%26aulast%3DCaporale%26aufirst%3DA.%26aulast%3DRoger%26aufirst%3DJ.%2BD.%2BA.%26aulast%3DMascanzoni%26aufirst%3DF.%26aulast%3DRuvo%26aufirst%3DM.%26aulast%3DDoti%26aufirst%3DN.%26atitle%3DDesign%252C%2520synthesis%252C%2520structural%2520analysis%2520and%2520biochemical%2520studies%2520of%2520stapled%2520AIF%2528370-394%2529%2520analogues%2520as%2520ligand%2520of%2520CypA%26jtitle%3DBBA%252C%2520Biochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2020%26volume%3D1864%26spage%3D129717%26doi%3D10.1016%2Fj.bbagen.2020.129717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starovasnik, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDowell, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theaker, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skelton, N. J.</span></span> <span> </span><span class="NLM_article-title">A Minimal Peptide Scaffold for β-Turn Display: Optimizing a Strand Position in Disulfide-Cyclized β-Hairpins</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">625</span>– <span class="NLM_lpage">632</span>, <span class="refDoi"> DOI: 10.1021/ja003369x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja003369x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3cXptVyjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2001&pages=625-632&author=A.+G.+Cochranauthor=R.+T.+Tongauthor=M.+A.+Starovasnikauthor=E.+J.+Parkauthor=R.+S.+McDowellauthor=J.+E.+Theakerauthor=N.+J.+Skelton&title=A+Minimal+Peptide+Scaffold+for+%CE%B2-Turn+Display%3A+Optimizing+a+Strand+Position+in+Disulfide-Cyclized+%CE%B2-Hairpins&doi=10.1021%2Fja003369x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A Minimal Peptide Scaffold for β-Turn Display: Optimizing a Strand Position in Disulfide-Cyclized β-Hairpins</span></div><div class="casAuthors">Cochran, Andrea G.; Tong, Ricky T.; Starovasnik, Melissa A.; Park, Eleanor J.; McDowell, Robert S.; Theaker, J. E.; Skelton, Nicholas J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">625-632</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phage display of peptide libraries has become a powerful tool for the evolution of novel ligands that bind virtually any protein target.  However, the rules governing conformational preferences in natural peptides are poorly understood, and consequently, structure-activity relationships in these mols. can be difficult to define.  In an effort to simplify this process, we have investigated the structural stability of 10-residue, disulfide-constrained β-hairpins and assessed their suitability as scaffolds for β-turn display.  Using disulfide formation as a probe, relative free energies of folding were measured for 19 peptides that differ at a one strand position.  A tryptophan substitution promotes folding to a remarkable degree.  NMR anal. confirms that the measured energies correlate well with the degree of β-hairpin structure in the disulfide-cyclized peptides.  Reexamn. of a subset of the strand substitutions in peptides with different turn sequences reveals linear free energy relationships, indicating that turns and strand-strand interactions make independent, additive contributions to hairpin stability.  Significantly, the tryptophan strand substitution is highly stabilizing with all turns tested, and peptides that display model turns or the less stable C'-C'' turn of CD4 on this tryptophan "stem" are highly structured β-hairpins in water.  Thus, we have developed a small, structured β-turn scaffold, contg. only natural L-amino acids, that may be used to display peptide libraries of limited conformational diversity on phage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr76zn4qfmZE7Vg90H21EOLACvtfcHk0lgWvApkxpGGhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXptVyjsLg%253D&md5=e422fa4f1204a9573f2f6266c59c30ec</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja003369x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja003369x%26sid%3Dliteratum%253Aachs%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DTong%26aufirst%3DR.%2BT.%26aulast%3DStarovasnik%26aufirst%3DM.%2BA.%26aulast%3DPark%26aufirst%3DE.%2BJ.%26aulast%3DMcDowell%26aufirst%3DR.%2BS.%26aulast%3DTheaker%26aufirst%3DJ.%2BE.%26aulast%3DSkelton%26aufirst%3DN.%2BJ.%26atitle%3DA%2520Minimal%2520Peptide%2520Scaffold%2520for%2520%25CE%25B2-Turn%2520Display%253A%2520Optimizing%2520a%2520Strand%2520Position%2520in%2520Disulfide-Cyclized%2520%25CE%25B2-Hairpins%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2001%26volume%3D123%26spage%3D625%26epage%3D632%26doi%3D10.1021%2Fja003369x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farina, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Gatto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comegna, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Gaetano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capasso, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isernia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattorusso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaccaro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, L.</span></span> <span> </span><span class="NLM_article-title">Conformational studies of RGDechi peptide by natural-abundance NMR spectroscopy</span>. <i>J. Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>, <span class="NLM_elocation-id">e3166</span> <span class="refDoi"> DOI: 10.1002/psc.3166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1002%2Fpsc.3166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=30884005" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&author=B.+Farinaauthor=A.+Del+Gattoauthor=D.+Comegnaauthor=S.+Di+Gaetanoauthor=D.+Capassoauthor=C.+Iserniaauthor=M.+Savianoauthor=R.+Fattorussoauthor=L.+Zaccaroauthor=L.+Russo&title=Conformational+studies+of+RGDechi+peptide+by+natural-abundance+NMR+spectroscopy&doi=10.1002%2Fpsc.3166"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fpsc.3166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsc.3166%26sid%3Dliteratum%253Aachs%26aulast%3DFarina%26aufirst%3DB.%26aulast%3DDel%2BGatto%26aufirst%3DA.%26aulast%3DComegna%26aufirst%3DD.%26aulast%3DDi%2BGaetano%26aufirst%3DS.%26aulast%3DCapasso%26aufirst%3DD.%26aulast%3DIsernia%26aufirst%3DC.%26aulast%3DSaviano%26aufirst%3DM.%26aulast%3DFattorusso%26aufirst%3DR.%26aulast%3DZaccaro%26aufirst%3DL.%26aulast%3DRusso%26aufirst%3DL.%26atitle%3DConformational%2520studies%2520of%2520RGDechi%2520peptide%2520by%2520natural-abundance%2520NMR%2520spectroscopy%26jtitle%3DJ.%2520Pept.%2520Sci.%26date%3D2019%26volume%3D25%26doi%3D10.1002%2Fpsc.3166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sykes, B. D.</span></span> <span> </span><span class="NLM_article-title">The 13C chemical-shift index: a simple method for the identification of protein secondary structure using 13C chemical-shift data</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1007/BF00175245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2FBF00175245" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=8019132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK2cXjtFymtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1994&pages=171-180&author=D.+S.+Wishartauthor=B.+D.+Sykes&title=The+13C+chemical-shift+index%3A+a+simple+method+for+the+identification+of+protein+secondary+structure+using+13C+chemical-shift+data&doi=10.1007%2FBF00175245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The 13C chemical-shift index: a simple method for the identification of protein secondary structure using 13C chemical-shift data</span></div><div class="casAuthors">Wishart, David S.; Sykes, Brian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">171-80</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    </div><div class="casAbstract">A simple technique for identifying protein secondary structures through the anal. of backbone 13C chem. shifts is described.  It is based on the Chem.-Shift Index [Wishart et al. (1992)] which was originally developed for the anal. of 1Hα chem. shifts.  By extending the Chem.-Shift Index to include 13Cα, 13Cβ and carbonyl 13C chem. shifts, it is now possible to use four independent chem.-shift measurements to identify and locate protein secondary structures.  It is shown that by combining both 1H and 13C chem.-shift indexes to produce a 'consensus' est. of secondary structure, it is possible to achieve a predictive accuracy in excess of 92%.  This suggests that the secondary structure of peptides and proteins can be accurately obtained from 1H and 13C chem. shifts, without recourse to NOE measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDXGadKPmScrVg90H21EOLACvtfcHk0limZHPASZoj7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXjtFymtLw%253D&md5=a248adaea4793a6b67ff69e33f6248e8</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1007%2FBF00175245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00175245%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DSykes%26aufirst%3DB.%2BD.%26atitle%3DThe%252013C%2520chemical-shift%2520index%253A%2520a%2520simple%2520method%2520for%2520the%2520identification%2520of%2520protein%2520secondary%2520structure%2520using%252013C%2520chemical-shift%2520data%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1994%26volume%3D4%26spage%3D171%26epage%3D180%26doi%3D10.1007%2FBF00175245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Use of chemical shifts in macromolecular structure determination</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>338</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1016/s0076-6879(02)38214-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2FS0076-6879%2802%2938214-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=11460554" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1eltrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=338&publication_year=2001&pages=3-34&author=D.+S.+Wishartauthor=D.+A.+Case&title=Use+of+chemical+shifts+in+macromolecular+structure+determination&doi=10.1016%2Fs0076-6879%2802%2938214-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Use of chemical shifts in macromolecular structure determination</span></div><div class="casAuthors">Wishart, David S.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">338</span>
        (<span class="NLM_cas:issue">Nuclear Magnetic Resonance of Biological Macromolecules, Part A</span>),
    <span class="NLM_cas:pages">3-34</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A review discusses the use of chem. shifts as an aid in understanding, generating or refining macromol. structures.  It also covers the random coil shifts, solid-state NMR structure generation, and chem. shift refinement.  The quantum mech. and more classical approaches to calcg. chem. shifts from structure are also discussed.  Chem. shifts are among the most precisely measurable but least accurately measured spectral parameters in all of NMR spectroscopy.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp875NU8XRsL7Vg90H21EOLACvtfcHk0limZHPASZoj7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1eltrg%253D&md5=04ad26a71d01536439c8e35fafb94a7f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2802%2938214-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252802%252938214-4%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DUse%2520of%2520chemical%2520shifts%2520in%2520macromolecular%2520structure%2520determination%26jtitle%3DMethods%2520Enzymol.%26date%3D2001%26volume%3D338%26spage%3D3%26epage%3D34%26doi%3D10.1016%2Fs0076-6879%2802%2938214-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kjaergaard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brander, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulsen, F. M.</span></span> <span> </span><span class="NLM_article-title">Random coil chemical shift for intrinsically disordered proteins: effects of temperature and pH</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">139</span>– <span class="NLM_lpage">149</span>, <span class="refDoi"> DOI: 10.1007/s10858-011-9472-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2Fs10858-011-9472-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=21234644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1GrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2011&pages=139-149&author=M.+Kjaergaardauthor=S.+Branderauthor=F.+M.+Poulsen&title=Random+coil+chemical+shift+for+intrinsically+disordered+proteins%3A+effects+of+temperature+and+pH&doi=10.1007%2Fs10858-011-9472-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Random coil chemical shift for intrinsically disordered proteins: effects of temperature and pH</span></div><div class="casAuthors">Kjaergaard, Magnus; Brander, Soren; Poulsen, Flemming M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">139-149</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Secondary chem. shift anal. is the main NMR method for detection of transiently formed secondary structure in intrinsically disordered proteins.  The quality of the secondary chem. shifts is dependent on an appropriate choice of random coil chem. shifts.  We report random coil chem. shifts and sequence correction factors detd. for a GGXGG peptide series following the approach of Schwarzinger et al. (J Am Chem Soc 123(13):2970-2978, 2001).  The chem. shifts are detd. at neutral pH in order to match the conditions of most studies of intrinsically disordered proteins.  Temp. has a non-negligible effect on the 13C random coil chem. shifts, so temp. coeffs. are reported for the random coil chem. shifts to allow extrapolation to other temps.  The pH dependence of the histidine random coil chem. shifts is investigated in a titrn. series, which allows the accurate random coil chem. shifts to be obtained at any pH.  By correcting the random coil chem. shifts for the effects of temp. and pH, systematic biases of the secondary chem. shifts are minimized, which will improve the reliability of detection of transient secondary structure in disordered proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7DYWlBZnyaLVg90H21EOLACvtfcHk0limZHPASZoj7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1GrsLw%253D&md5=fcc9873b54dcf445a35f70e0e9ac6a2e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1007%2Fs10858-011-9472-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10858-011-9472-x%26sid%3Dliteratum%253Aachs%26aulast%3DKjaergaard%26aufirst%3DM.%26aulast%3DBrander%26aufirst%3DS.%26aulast%3DPoulsen%26aufirst%3DF.%2BM.%26atitle%3DRandom%2520coil%2520chemical%2520shift%2520for%2520intrinsically%2520disordered%2520proteins%253A%2520effects%2520of%2520temperature%2520and%2520pH%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2011%26volume%3D49%26spage%3D139%26epage%3D149%26doi%3D10.1007%2Fs10858-011-9472-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kjaergaard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulsen, F. M.</span></span> <span> </span><span class="NLM_article-title">Sequence correction of random coil chemical shifts: correlation between neighbor correction factors and changes in the Ramachandran distribution</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">165</span>, <span class="refDoi"> DOI: 10.1007/s10858-011-9508-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2Fs10858-011-9508-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=21604143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntlahu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=157-165&author=M.+Kjaergaardauthor=F.+M.+Poulsen&title=Sequence+correction+of+random+coil+chemical+shifts%3A+correlation+between+neighbor+correction+factors+and+changes+in+the+Ramachandran+distribution&doi=10.1007%2Fs10858-011-9508-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence correction of random coil chemical shifts: correlation between neighbor correction factors and changes in the Ramachandran distribution</span></div><div class="casAuthors">Kjaergaard, Magnus; Poulsen, Flemming M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-165</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Random coil chem. shifts are necessary for secondary chem. shift anal., which is the main NMR method for identification of secondary structure in proteins.  One of the largest challenges in the detn. of random coil chem. shifts is accounting for the effect of neighboring residues.  The contributions from the neighboring residues are typically removed by using neighbor correction factors detd. based on each residue's effect on glycine chem. shifts.  Due to its unusual conformational freedom, glycine may be particularly unrepresentative for the remaining residue types.  In this study, we use random coil peptides contg. glutamine instead of glycine to det. the random coil chem. shifts and the neighbor correction factors.  The resulting correction factors correlate to changes in the populations of the major wells in the Ramachandran plot, which demonstrates that changes in the conformational ensemble are an important source of neighbor effects in disordered proteins.  Glutamine derived random coil chem. shifts and correction factors modestly improve our ability to predict 13C chem. shifts of intrinsically disordered proteins compared to existing datasets, and may thus improve the identification of small populations of transient structure in disordered proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTS4ANLXd1bbVg90H21EOLACvtfcHk0lgPgbrBAZm5Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntlahu70%253D&md5=f8ea3e2e00a827e4836b0f5ed66becb6</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2Fs10858-011-9508-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10858-011-9508-2%26sid%3Dliteratum%253Aachs%26aulast%3DKjaergaard%26aufirst%3DM.%26aulast%3DPoulsen%26aufirst%3DF.%2BM.%26atitle%3DSequence%2520correction%2520of%2520random%2520coil%2520chemical%2520shifts%253A%2520correlation%2520between%2520neighbor%2520correction%2520factors%2520and%2520changes%2520in%2520the%2520Ramachandran%2520distribution%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2011%26volume%3D50%26spage%3D157%26epage%3D165%26doi%3D10.1007%2Fs10858-011-9508-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanz-Hernández, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Simone, A.</span></span> <span> </span><span class="NLM_article-title">The PROSECCO server for chemical shift predictions in ordered and disordered proteins</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">147</span>– <span class="NLM_lpage">156</span>, <span class="refDoi"> DOI: 10.1007/s10858-017-0145-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2Fs10858-017-0145-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=29119515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2jurzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2017&pages=147-156&author=M.+Sanz-Hern%C3%A1ndezauthor=A.+De+Simone&title=The+PROSECCO+server+for+chemical+shift+predictions+in+ordered+and+disordered+proteins&doi=10.1007%2Fs10858-017-0145-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The PROSECCO server for chemical shift predictions in ordered and disordered proteins</span></div><div class="casAuthors">Sanz-Hernandez, Maximo; De Simone, Alfonso</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">147-156</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The chem. shifts measured in soln.-state and solid-state NMR (NMR) are powerful probes of the structure and dynamics of protein mols.  The exploitation of chem. shifts requires methods to correlate these data with the protein structures and sequences.  We present here an approach to calc. accurate chem. shifts in both ordered and disordered proteins using exclusively the information contained in their sequences.  Our sequence-based approach, protein sequences and chem. shift correlations (PROSECCO), achieves the accuracy of the most advanced structure-based methods in the characterization of chem. shifts of folded proteins and improves the state of the art in the study of disordered proteins.  Our analyses revealed fundamental insights on the structural information carried by NMR chem. shifts of structured and unstructured protein states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSDm6QomKYULVg90H21EOLACvtfcHk0lgPgbrBAZm5Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2jurzE&md5=8008a7dcc2025e87fdd5fa09629787cc</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1007%2Fs10858-017-0145-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10858-017-0145-2%26sid%3Dliteratum%253Aachs%26aulast%3DSanz-Hern%25C3%25A1ndez%26aufirst%3DM.%26aulast%3DDe%2BSimone%26aufirst%3DA.%26atitle%3DThe%2520PROSECCO%2520server%2520for%2520chemical%2520shift%2520predictions%2520in%2520ordered%2520and%2520disordered%2520proteins%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2017%26volume%3D69%26spage%3D147%26epage%3D156%26doi%3D10.1007%2Fs10858-017-0145-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tamiola, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulder, F. A. A.</span></span> <span> </span><span class="NLM_article-title">Using NMR chemical shifts to calculate the propensity for structural order and disorder in proteins</span>. <i>Biochem. Soc. Trans.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1014</span>– <span class="NLM_lpage">1020</span>, <span class="refDoi"> DOI: 10.1042/bst20120171</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1042%2FBST20120171" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=22988857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlOjur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=1014-1020&author=K.+Tamiolaauthor=F.+A.+A.+Mulder&title=Using+NMR+chemical+shifts+to+calculate+the+propensity+for+structural+order+and+disorder+in+proteins&doi=10.1042%2Fbst20120171"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Using NMR chemical shifts to calculate the propensity for structural order and disorder in proteins</span></div><div class="casAuthors">Tamiola, Kamil; Mulder, Frans A. A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Society Transactions</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1014-1020</span>CODEN:
                <span class="NLM_cas:coden">BCSTB5</span>;
        ISSN:<span class="NLM_cas:issn">0300-5127</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  NMR spectroscopy offers the unique possibility to relate the structural propensities of disordered proteins and loop segments of folded peptides to biol. function and aggregation behavior.  Backbone chem. shifts are ideally suited for this task, provided that appropriate ref. data are available and idiosyncratic sensitivity of backbone chem. shifts to structural information is treated in a sensible manner.  In the present paper, we describe methods to detect structural protein changes from chem. shifts, and present an online tool [ncSPC (neighbor-cor. Structural Propensity Calculator)], which unites aspects of several current approaches.  Examples of structural propensity calcns. are given for two well-characterized systems, namely the binding of α-synuclein to micelles and light activation of photoactive yellow protein.  These examples spotlight the great power of NMR chem. shift anal. for the quant. assessment of protein disorder at the at. level, and further our understanding of biol. important problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6Faz_S9Sm2rVg90H21EOLACvtfcHk0lgPgbrBAZm5Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlOjur7I&md5=5a028e85b8450c0ee2b46a3656d17916</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1042%2FBST20120171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBST20120171%26sid%3Dliteratum%253Aachs%26aulast%3DTamiola%26aufirst%3DK.%26aulast%3DMulder%26aufirst%3DF.%2BA.%2BA.%26atitle%3DUsing%2520NMR%2520chemical%2520shifts%2520to%2520calculate%2520the%2520propensity%2520for%2520structural%2520order%2520and%2520disorder%2520in%2520proteins%26jtitle%3DBiochem.%2520Soc.%2520Trans.%26date%3D2012%26volume%3D40%26spage%3D1014%26epage%3D1020%26doi%3D10.1042%2Fbst20120171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span> <span> </span><span class="NLM_article-title">Interpreting protein chemical shift data</span>. <i>Prog. Nucl. Magn. Reson. Spectrosc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.pnmrs.2010.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2Fj.pnmrs.2010.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=21241884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVKqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2011&pages=62-87&author=D.+S.+Wishart&title=Interpreting+protein+chemical+shift+data&doi=10.1016%2Fj.pnmrs.2010.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Interpreting protein chemical shift data</span></div><div class="casAuthors">Wishart, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Nuclear Magnetic Resonance Spectroscopy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">62-87</span>CODEN:
                <span class="NLM_cas:coden">PNMRAT</span>;
        ISSN:<span class="NLM_cas:issn">0079-6565</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopFqH4zvjnwLVg90H21EOLACvtfcHk0lgPgbrBAZm5Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVKqtg%253D%253D&md5=88c1ea81ee5bc5748b2e7215d946c792</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.pnmrs.2010.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pnmrs.2010.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26atitle%3DInterpreting%2520protein%2520chemical%2520shift%2520data%26jtitle%3DProg.%2520Nucl.%2520Magn.%2520Reson.%2520Spectrosc.%26date%3D2011%26volume%3D58%26spage%3D62%26epage%3D87%26doi%3D10.1016%2Fj.pnmrs.2010.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman-Kay, J. D.</span></span> <span> </span><span class="NLM_article-title">Sensitivity of secondary structure propensities to sequence differences between α- and γ-synuclein: Implications for fibrillation</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2795</span>– <span class="NLM_lpage">2804</span>, <span class="refDoi"> DOI: 10.1110/ps.062465306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1110%2Fps.062465306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=17088319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlSmtbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2006&pages=2795-2804&author=J.+A.+Marshauthor=V.+K.+Singhauthor=Z.+Jiaauthor=J.+D.+Forman-Kay&title=Sensitivity+of+secondary+structure+propensities+to+sequence+differences+between+%CE%B1-+and+%CE%B3-synuclein%3A+Implications+for+fibrillation&doi=10.1110%2Fps.062465306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Sensitivity of secondary structure propensities to sequence differences between α- and γ-synuclein: implications for fibrillation</span></div><div class="casAuthors">Marsh, Joseph A.; Singh, Vinay K.; Jia, Zongchao; Forman-Kay, Julie D.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2795-2804</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The synucleins are a family of intrinsically disordered proteins involved in various human diseases.  α-Synuclein has been extensively characterized due to its role in Parkinson's disease where it forms intracellular aggregates, while γ-synuclein is overexpressed in a majority of late-stage breast cancers.  Despite fairly strong sequence similarity between the amyloid-forming regions of α- and γ-synuclein, γ-synuclein has only a weak propensity to form amyloid fibrils.  The authors hypothesize that the different fibrillation tendencies of α- and γ-synuclein may be related to differences in structural propensities.  Here the authors have measured chem. shifts for γ-synuclein and compared them to previously published shifts for α-synuclein.  To facilitate direct comparison, the authors have implemented a simple new technique for re-referencing chem. shifts that the authors have highly effective for both disordered and folded proteins.  In addn., the authors have developed a new method that combines different chem. shifts into a single residue-specific secondary structure propensity (SSP) score.  The authors observe significant differences between α- and γ-synuclein secondary structure propensities.  Most interestingly, γ-synuclein has an increased α-helical propensity in the amyloid-forming region that is crit. for α-synuclein fibrillation, suggesting that increased structural stability in this region may protect against γ-synuclein aggregation.  This comparison of residue-specific secondary structure propensities between intrinsically disordered homologs highlights the sensitivity of transient structure to sequence changes, which the authors suggest may have been exploited as an evolutionary mechanism for fast modulation of protein structure and, hence, function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoObrK2Iobl67Vg90H21EOLACvtfcHk0ljMW_Q7Trebwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlSmtbzL&md5=9e78ad92366e7fe40c3523ba781f80dc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1110%2Fps.062465306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.062465306%26sid%3Dliteratum%253Aachs%26aulast%3DMarsh%26aufirst%3DJ.%2BA.%26aulast%3DSingh%26aufirst%3DV.%2BK.%26aulast%3DJia%26aufirst%3DZ.%26aulast%3DForman-Kay%26aufirst%3DJ.%2BD.%26atitle%3DSensitivity%2520of%2520secondary%2520structure%2520propensities%2520to%2520sequence%2520differences%2520between%2520%25CE%25B1-%2520and%2520%25CE%25B3-synuclein%253A%2520Implications%2520for%2520fibrillation%26jtitle%3DProtein%2520Sci.%26date%3D2006%26volume%3D15%26spage%3D2795%26epage%3D2804%26doi%3D10.1110%2Fps.062465306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fesinmeyer, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, G. W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Euser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, N. H.</span></span> <span> </span><span class="NLM_article-title">Chemical Shifts Provide Fold Populations and Register of β Hairpins and β Sheets</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1007/s10858-005-3731-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2Fs10858-005-3731-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=16341751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSjtLbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=213-231&author=R.+M.+Fesinmeyerauthor=F.+M.+Hudsonauthor=K.+A.+Olsenauthor=G.+W.+N.+Whiteauthor=A.+Euserauthor=N.+H.+Andersen&title=Chemical+Shifts+Provide+Fold+Populations+and+Register+of+%CE%B2+Hairpins+and+%CE%B2+Sheets&doi=10.1007%2Fs10858-005-3731-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Shifts Provide Fold Populations and Register of β Hairpins and β Sheets</span></div><div class="casAuthors">Fesinmeyer, R. Matthew; Hudson, F. Michael; Olsen, Katherine A.; White, George W. N.; Euser, Anna; Andersen, Niels H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">213-231</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A detailed anal. of peptide backbone amide (HN) and Hα chem. shifts reveals a consistent pattern for β hairpins and three-stranded β sheets.  The Hα's at non-hydrogen-bonded strand positions are inwardly directed and shifted downfield ∼1 ppm due largely to an anisotropy contribution from the cross-strand amide function.  The secondary structure assocd. Hα shift deviations for the H-bonded strand positions are also pos. but much smaller (0.1-0.3 ppm) and the turn residues display neg. Hα chem. shift deviations (CSDs).  The pattern of (HN) shift deviations is an even better indicator of both hairpin formation and register, with the cross-strand H-bonded sites shifted downfield (also by ∼1 ppm) and with diagnostic values for the first turn residue and the first strand position following the turn.  These empirical observations, initially made for [2:2]/[2:4]-type-I' and -II' hairpins, are rationalized and can be extended to the anal. of other turns, hairpin classes ([3:5], [4:4]/[4:6]), and three-stranded peptide β-sheet models.  The Hα's at non-hydrogen-bonded sites and (HN)'s in the intervening H-bonded sites provide the largest and most dependable measures of hairpin structuring and can be used for melting studies; however the intrinsic temp. dependence of (HN) shifts deviations needs to reflect the extent of solvent sequestration in the folded state.  Several observations made in the course of this study provide insights into β-sheet folding mechanisms: (1) The magnitude of the (HN) shifts suggests that the cross-strand H-bonds in peptide hairpins are as short as those in protein β sheets. (2) Even L-Pro-Gly turns, which are frequently used in unfolded controls for hairpin peptides, can support hairpin populations in aq. fluoroalc. media. (3) The good correlation between hairpin population ests. from cross-strand H-bonded (HN) shift deviations, Hα shift deviations, and structuring shifts at the turn locus implies that hairpin folding transitions approx. two-state behavior.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_LkJ8Hx5IgrVg90H21EOLACvtfcHk0ljMW_Q7Trebwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSjtLbP&md5=78c92d18825a4cb8f7e9f30ca8e79fdb</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1007%2Fs10858-005-3731-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10858-005-3731-7%26sid%3Dliteratum%253Aachs%26aulast%3DFesinmeyer%26aufirst%3DR.%2BM.%26aulast%3DHudson%26aufirst%3DF.%2BM.%26aulast%3DOlsen%26aufirst%3DK.%2BA.%26aulast%3DWhite%26aufirst%3DG.%2BW.%2BN.%26aulast%3DEuser%26aufirst%3DA.%26aulast%3DAndersen%26aufirst%3DN.%2BH.%26atitle%3DChemical%2520Shifts%2520Provide%2520Fold%2520Populations%2520and%2520Register%2520of%2520%25CE%25B2%2520Hairpins%2520and%2520%25CE%25B2%2520Sheets%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2005%26volume%3D33%26spage%3D213%26epage%3D231%26doi%3D10.1007%2Fs10858-005-3731-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santiveri, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pantoja-Uceda, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rico, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiménez, M. A.</span></span> <span> </span><span class="NLM_article-title">β-Hairpin formation in aqueous solution and in the presence of trifluoroethanol: A1H and13C nuclear magnetic resonance conformational study of designed peptides</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">150</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1002/bip.20345</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1002%2Fbip.20345" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=16078190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVyhtbjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2005&pages=150-162&author=C.+M.+Santiveriauthor=D.+Pantoja-Ucedaauthor=M.+Ricoauthor=M.+A.+Jim%C3%A9nez&title=%CE%B2-Hairpin+formation+in+aqueous+solution+and+in+the+presence+of+trifluoroethanol%3A+A1H+and13C+nuclear+magnetic+resonance+conformational+study+of+designed+peptides&doi=10.1002%2Fbip.20345"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">β-hairpin formation in aqueous solution and in the presence of trifluoroethanol: A 1H and 13C nuclear magnetic resonance conformational study of designed peptides</span></div><div class="casAuthors">Santiveri, Clara M.; Pantoja-Uceda, David; Rico, Manuel; Jimenez, M. Angeles</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">150-162</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In order to check our current knowledge on the principles involved in β-hairpin formation, we have modified the sequence of a 3:5 β-hairpin forming peptide with two different purposes, first to increase the stability of the formed 3:5 β-hairpin, and second to convert the 3:5 β-hairpin into a 2:2 β-hairpin.  The conformational behavior of the designed peptides was investigated in aq. soln. and in 30% trifluoroethanol (TFE) by anal. of the following NMR parameters: nuclear Overhauser effect (NOE) data, and CαH, 13Cα, and 13Cβ conformational shifts.  From the differences in the ability to adopt β-hairpin structures in these peptides, we have arrived to the following conclusions: (i) β-Hairpin population increases with the statistical propensity of residues to occupy each turn position.  The loop length, and in turn, the β-hairpin type, can be modified as a function of the type of turn favored by the loop sequence.  These two conclusions reinforce previous results about the importance of β-turn sequence in β-hairpin folding.  Side-chain packing on each face of the β-sheet may play a major role in β-hairpin stability; hence simplified anal. in terms of isolated pair interactions and intrinsic β-sheet propensities is insufficient.  Contributions to β-hairpin stability of turn and strand sequences are not completely independent.  The burial of hydrophobic surface upon β-hairpin formation that, in turn, depends on side-chain packing also contributes to β-hairpin stability.  As previously obsd., TFE stabilizes β-hairpin structures, but the extent of the contribution of different factors to β-hairpin formation is sometimes different in aq. soln. and in 30% TFE.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor1sjAzZ7MOLVg90H21EOLACvtfcHk0ljMW_Q7Trebwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVyhtbjM&md5=7c051b5f3d9df01fcd284c89cfa504ab</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fbip.20345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.20345%26sid%3Dliteratum%253Aachs%26aulast%3DSantiveri%26aufirst%3DC.%2BM.%26aulast%3DPantoja-Uceda%26aufirst%3DD.%26aulast%3DRico%26aufirst%3DM.%26aulast%3DJim%25C3%25A9nez%26aufirst%3DM.%2BA.%26atitle%3D%25CE%25B2-Hairpin%2520formation%2520in%2520aqueous%2520solution%2520and%2520in%2520the%2520presence%2520of%2520trifluoroethanol%253A%2520A1H%2520and13C%2520nuclear%2520magnetic%2520resonance%2520conformational%2520study%2520of%2520designed%2520peptides%26jtitle%3DBiopolymers%26date%3D2005%26volume%3D79%26spage%3D150%26epage%3D162%26doi%3D10.1002%2Fbip.20345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajarathnam, K.</span></span> <span> </span><span class="NLM_article-title">13C NMR chemical shifts can predict disulfide bond formation</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">165</span>– <span class="NLM_lpage">171</span>, <span class="refDoi"> DOI: 10.1023/a:1008398416292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1023%2FA%3A1008398416292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=11101221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosVWgsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2000&pages=165-171&author=D.+Sharmaauthor=K.+Rajarathnam&title=13C+NMR+chemical+shifts+can+predict+disulfide+bond+formation&doi=10.1023%2Fa%3A1008398416292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">13C NMR chemical shifts can predict disulfide bond formation</span></div><div class="casAuthors">Sharma, Deepak; Rajarathnam, Krishna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">165-171</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">The presence of disulfide bonds can be detected unambiguously only by x-ray crystallog., and otherwise must be inferred by chem. methods.  In this study we demonstrate that 13C NMR chem. shifts are diagnostic of disulfide bond formation, and can discriminate between cysteine in the reduced (free) and oxidized (disulfide bonded) state.  A database of cysteine 13CCα and Cβ chem. shifts was constructed from the BioMagResBank (BMRB) and Sheffield databases, and published journals.  Statistical anal. indicated that the Cβ shift is extremely sensitive to the redox state, and can predict the disulfide-bonded state.  Further, chem. shifts in both states occupy distinct clusters as a function of secondary structure in the Cα/Cβ chem. shift map.  On the basis of these results, we provide simple ground rules for predicting the redox state of cysteines; these rules could be used effectively in NMR structure detn., predicting new folds, and in protein folding studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD37laLLTfO7Vg90H21EOLACvtfcHk0ljWPoDNa2s1CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosVWgsL8%253D&md5=248353c6611b9b52e614a399260b4fd1</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008398416292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008398416292%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DD.%26aulast%3DRajarathnam%26aufirst%3DK.%26atitle%3D13C%2520NMR%2520chemical%2520shifts%2520can%2520predict%2520disulfide%2520bond%2520formation%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2000%26volume%3D18%26spage%3D165%26epage%3D171%26doi%3D10.1023%2Fa%3A1008398416292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolin, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwalbe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacArthur, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, C. M.</span></span> <span> </span><span class="NLM_article-title">Analysis of main chain torsion angles in proteins: prediction of NMR coupling constants for native and random coil conformations</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>255</i></span>,  <span class="NLM_fpage">494</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1006%2Fjmbi.1996.0041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=8568893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK28XnvVSjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=255&publication_year=1996&pages=494-506&author=L.+J.+Smithauthor=K.+A.+Bolinauthor=H.+Schwalbeauthor=M.+W.+MacArthurauthor=J.+M.+Thorntonauthor=C.+M.+Dobson&title=Analysis+of+main+chain+torsion+angles+in+proteins%3A+prediction+of+NMR+coupling+constants+for+native+and+random+coil+conformations&doi=10.1006%2Fjmbi.1996.0041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of main chain torsion angles in proteins: prediction of NMR coupling constants for native and random coil conformations</span></div><div class="casAuthors">Smith, Lorna J.; Bolin, Kimberly A.; Schwalbe, Harald; MacArthur, Malcolm W.; Thornton, Janet M.; Dobson, Christopher M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">255</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">494-506</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Using a database of 85 high resoln. protein crystal structures, the distributions of main chain torsion angles, both in secondary structure and in coil regions where no secondary structure is present, have been analyzed.  These torsion angle distributions have been used to predict NMR homonuclear and heteronuclear coupling consts. for residues in the secondary structure using known Karplus relationships.  For α helixes, 310 helixes and β strands mean predicted 3JHNα coupling consts. are 4.8, 5.6 and 8.5 Hz, resp.  These values differ significantly from those expected for the ideal .vphi. angles (3.9, 3.0 and 8.9 Hz; .vphi. = -57°, -49°, -139° for α and 310 helixes and β strands (antiparallel), resp.) in regular secondary structure, but agree well with available exptl. NMR data for nine proteins.  The crystallog. data set has also been used to provide a basis for interpreting coupling consts. measured for peptides and denatured proteins.  Using a model for a random coil, in which all residues adopt distributions of .vphi., ψ angles equiv. to those seen for residues in the coil regions of native folded proteins, predicted 3JHNα values for different residue types have been found to range from 5.9 Hz and 6.1 Hz for glycine and alanine, resp., to 7.7 Hz for valine.  A good correlation has been found between the predicted 3JHNα coupling consts. for this model and exptl. values for a set of peptides that other evidence suggests are highly unstructured.  For other peptides, however, deviations from the predictions of the model are clear and provided evidence for addnl. interactions within otherwise disordered states.  The values of homonuclear and heteronuclear coupling consts. derived from the protein database listed here therefore provide a basis not only for analyzing the secondary structure of native proteins in soln.but for assessing and interpreting the extent of structure present in peptides and non-native states of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5kHZ5tujTGbVg90H21EOLACvtfcHk0ljWPoDNa2s1CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XnvVSjsw%253D%253D&md5=03a9c166a532053820da51ee2c564b95</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0041%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DL.%2BJ.%26aulast%3DBolin%26aufirst%3DK.%2BA.%26aulast%3DSchwalbe%26aufirst%3DH.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26aulast%3DDobson%26aufirst%3DC.%2BM.%26atitle%3DAnalysis%2520of%2520main%2520chain%2520torsion%2520angles%2520in%2520proteins%253A%2520prediction%2520of%2520NMR%2520coupling%2520constants%2520for%2520native%2520and%2520random%2520coil%2520conformations%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1996%26volume%3D255%26spage%3D494%26epage%3D506%26doi%3D10.1006%2Fjmbi.1996.0041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dominguez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boelens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonvin, A. M. J. J.</span></span> <span> </span><span class="NLM_article-title">HADDOCK: A Protein–Protein Docking Approach Based on Biochemical or Biophysical Information</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">1731</span>– <span class="NLM_lpage">1737</span>, <span class="refDoi"> DOI: 10.1021/ja026939x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja026939x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvFGquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=1731-1737&author=C.+Dominguezauthor=R.+Boelensauthor=A.+M.+J.+J.+Bonvin&title=HADDOCK%3A+A+Protein%E2%80%93Protein+Docking+Approach+Based+on+Biochemical+or+Biophysical+Information&doi=10.1021%2Fja026939x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">HADDOCK: A protein-protein docking approach based on biochemical or biophysical information</span></div><div class="casAuthors">Dominguez, Cyril; Boelens, Rolf; Bonvin, Alexandre M. J. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1731-1737</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure detn. of protein-protein complexes is a rather tedious and lengthy process, by both NMR and x-ray crystallog.  Several methods based on docking to study protein complexes have also been well developed over the past few years.  Most of these approaches are not driven by exptl. data but are based on a combination of energetics and shape complementarity. Here, we present an approach called HADDOCK (High Ambiguity Driven protein-protein Docking) that makes use of biochem. and/or biophys. interaction data such as chem. shift perturbation data resulting from NMR titrn. expts. or mutagenesis data.  This information is introduced as Ambiguous Interaction Restraints (AIRs) to drive the docking process.  An AIR is defined as an ambiguous distance between all residues shown to be involved in the interaction.  The accuracy of our approach is demonstrated with three mol. complexes.  For two of these complexes, for which both the complex and the free protein structures have been solved, NMR titrn. data were available.  Mutagenesis data were used in the last example.  In all cases, the best structures generated by HADDOCK, i.e., the structures with the lowest intermol. energies, were the closest to the published structure of the resp. complexes (within 2.0 Å backbone RMSD).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2GoXS6PbQz7Vg90H21EOLACvtfcHk0ljWPoDNa2s1CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvFGquw%253D%253D&md5=9cf6fd9c690afb0181579485ae263197</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fja026939x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja026939x%26sid%3Dliteratum%253Aachs%26aulast%3DDominguez%26aufirst%3DC.%26aulast%3DBoelens%26aufirst%3DR.%26aulast%3DBonvin%26aufirst%3DA.%2BM.%2BJ.%2BJ.%26atitle%3DHADDOCK%253A%2520A%2520Protein%25E2%2580%2593Protein%2520Docking%2520Approach%2520Based%2520on%2520Biochemical%2520or%2520Biophysical%2520Information%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D1731%26epage%3D1737%26doi%3D10.1021%2Fja026939x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Vries, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Dijk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonvin, A. M. J. J.</span></span> <span> </span><span class="NLM_article-title">The HADDOCK web server for data-driven biomolecular docking</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">883</span>– <span class="NLM_lpage">897</span>, <span class="refDoi"> DOI: 10.1038/nprot.2010.32</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1038%2Fnprot.2010.32" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=20431534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltl2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=883-897&author=S.+J.+de%0AVriesauthor=M.+van+Dijkauthor=A.+M.+J.+J.+Bonvin&title=The+HADDOCK+web+server+for+data-driven+biomolecular+docking&doi=10.1038%2Fnprot.2010.32"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The HADDOCK web server for data-driven biomolecular docking</span></div><div class="casAuthors">de Vries, Sjoerd J.; van Dijk, Marc; Bonvin, Alexandre M. J. J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">883-897</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Computational docking is the prediction or modeling of the three-dimensional structure of a biomol. complex, starting from the structures of the individual mols. in their free, unbound form.  HADDOCK is a popular docking program that takes a data-driven approach to docking, with support for a wide range of exptl. data.  Here the authors present the HADDOCK web server protocol, facilitating the modeling of biomol. complexes for a wide community.  The main web interface is user-friendly, requiring only the structures of the individual components and a list of interacting residues as input.  Addnl. web interfaces allow the more advanced user to exploit the full range of exptl. data supported by HADDOCK and to customize the docking process.  The HADDOCK server has access to the resources of a dedicated cluster and of the e-NMR GRID infrastructure.  Therefore, a typical docking run takes only a few minutes to prep. and a few hours to complete.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCBZ3XxzZgL7Vg90H21EOLACvtfcHk0ljWPoDNa2s1CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltl2qsLw%253D&md5=dbf801d99065e8d2c3aedc016209838c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2010.32&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2010.32%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BVries%26aufirst%3DS.%2BJ.%26aulast%3Dvan%2BDijk%26aufirst%3DM.%26aulast%3DBonvin%26aufirst%3DA.%2BM.%2BJ.%2BJ.%26atitle%3DThe%2520HADDOCK%2520web%2520server%2520for%2520data-driven%2520biomolecular%2520docking%26jtitle%3DNat.%2520Protoc.%26date%3D2010%26volume%3D5%26spage%3D883%26epage%3D897%26doi%3D10.1038%2Fnprot.2010.32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krüger, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span> <span> </span><span class="NLM_article-title">DrugScorePPI webserver: fast and accurate in silico alanine scanning for scoring protein-protein interactions</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">W480</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1093%2Fnar%2Fgkq471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=20511591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVSqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=W480&author=D.+M.+Kr%C3%BCgerauthor=H.+Gohlke&title=DrugScorePPI+webserver%3A+fast+and+accurate+in+silico+alanine+scanning+for+scoring+protein-protein+interactions&doi=10.1093%2Fnar%2Fgkq471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">DrugScorePPI webserver: fast and accurate in silico alanine scanning for scoring protein-protein interactions</span></div><div class="casAuthors">Krueger, Dennis M.; Gohlke, Holger</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W480-W486</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Protein-protein complexes play key roles in all cellular signal transduction processes.  We have developed a fast and accurate computational approach to predict changes in the binding free energy upon alanine mutations in protein-protein interfaces.  The approach is based on a knowledge-based scoring function, DrugScorePPI, for which pair potentials were derived from 851 complex structures and adapted against 309 exptl. alanine scanning results.  Based on this approach, we developed the DrugScorePPI webserver.  The input consists of a protein-protein complex structure; the output is a summary table and bar plot of binding free energy differences for wild-type residue-to-Ala mutations.  The results of the anal. are mapped on the protein-protein complex structure and visualized using J mol.  A single interface can be analyzed within a few minutes.  Our approach has been successfully validated by application to an external test set of 22 alanine mutations in the interface of Ras/RalGDS.  The DrugScorePPI webserver is primarily intended for identifying hot-spot residues in protein-protein interfaces, which provides valuable information for guiding biol. expts. and in the development of protein-protein interaction modulators.  The DrugScorePPI Webserver, accessible at http://cpclab.uni-duesseldorf.de/dsppi, is free and open to all users with no login requirement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmnID8v_JPHLVg90H21EOLACvtfcHk0ljrmK82RcyJig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVSqt7g%253D&md5=5c55172fecbec2710fe0e53b45a28a1c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq471%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25BCger%26aufirst%3DD.%2BM.%26aulast%3DGohlke%26aufirst%3DH.%26atitle%3DDrugScorePPI%2520webserver%253A%2520fast%2520and%2520accurate%2520in%2520silico%2520alanine%2520scanning%2520for%2520scoring%2520protein-protein%2520interactions%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3DW480%26doi%3D10.1093%2Fnar%2Fgkq471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Böttger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knappe, D.</span></span> <span> </span><span class="NLM_article-title">Differential stability of therapeutic peptides with different proteolytic cleavage sites in blood, plasma and serum</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0178943</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0178943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1371%2Fjournal.pone.0178943" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=28575099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1Cjtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=R.+B%C3%B6ttgerauthor=R.+Hoffmannauthor=D.+Knappe&title=Differential+stability+of+therapeutic+peptides+with+different+proteolytic+cleavage+sites+in+blood%2C+plasma+and+serum&doi=10.1371%2Fjournal.pone.0178943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Differential stability of therapeutic peptides with different proteolytic cleavage sites in blood, plasma and serum</span></div><div class="casAuthors">Bottger, Roland; Hoffmann, Ralf; Knappe, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e0178943/1-e0178943/15</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Proteolytic degrdn. of peptide-based drugs is often considered as major weakness limiting systemic therapeutic applications.  Therefore, huge efforts are typically devoted to stabilize sequences against proteases present in serum or plasma, obtained as supernatants after complete blood coagulation or centrifugation of blood supplemented with anticoagulants, resp.  Plasma and serum are reproducibly obtained from animals and humans allowing consistent for clin. analyses and research applications.  However, the spectrum of active or activated proteases appears to vary depending on the activation of proteases and cofactors during coagulation (serum) or inhibition of such enzymes by anticoagulants (plasma), such as EDTA (metallo- and Ca2+-dependent proteases) and heparin (e.g. thrombin, factor Xa).  Here, we studied the presumed effects on peptide degrdn. by taking blood via cardiac puncture of CD-1 mice using a syringe contg. a peptide soln.  Due to absence of coagulation activators (e.g. glass surfaces and damaged cells), visible blood clotting was prevented allowing to study peptide degrdn. for one hour.  The remaining peptide was quantified and the degrdn. products were identified using mass spectrometry.  When the degrdn. rates (half-life times) were compared to serum derived freshly from the same animal and com. serum and plasma samples, peptides of three different families showed indeed considerably different stabilities.  Generally, peptides were faster degraded in serum than in plasma, but surprisingly all peptides were more stable in fresh blood and the order of degrdn. rates among the peptides varied among the six different incubation expts.  This indicates, that proteolytic degrdn. of peptide-based therapeutics may often be misleading stimulating efforts to stabilize peptides at degrdn. sites relevant only in vitro, i.e., for serum or plasma stability assays, but of lower importance in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT8Seu6LHCzLVg90H21EOLACvtfcHk0ljrmK82RcyJig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1Cjtrg%253D&md5=cbf5555f8b11a6fedf08a240297365da</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0178943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0178943%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6ttger%26aufirst%3DR.%26aulast%3DHoffmann%26aufirst%3DR.%26aulast%3DKnappe%26aufirst%3DD.%26atitle%3DDifferential%2520stability%2520of%2520therapeutic%2520peptides%2520with%2520different%2520proteolytic%2520cleavage%2520sites%2520in%2520blood%252C%2520plasma%2520and%2520serum%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0178943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aneja, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaiken, I.</span></span> <span> </span><span class="NLM_article-title">Click chemistry in peptide-based drug design</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">9797</span>– <span class="NLM_lpage">9817</span>, <span class="refDoi"> DOI: 10.3390/molecules18089797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.3390%2Fmolecules18089797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=23959192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlKlsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=9797-9817&author=H.+Liauthor=R.+Anejaauthor=I.+Chaiken&title=Click+chemistry+in+peptide-based+drug+design&doi=10.3390%2Fmolecules18089797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Click chemistry in peptide-based drug design</span></div><div class="casAuthors">Li, Huiyuan; Aneja, Rachna; Chaiken, Irwin</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">9797-9817</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Click chem. is an efficient and chemoselective synthetic method for coupling mol. fragments under mild reaction conditions.  Since the advent in 2001 of methods to improve stereochem. conservation, the click chem. approach has been broadly used to construct diverse chemotypes in both chem. and biol. fields.  In this review, we discuss the application of click chem. in peptide-based drug design.  We highlight how triazoles formed by click reactions have been used for mimicking peptide and disulfide bonds, building secondary structural components of peptides, linking functional groups together, and bioconjugation.  The progress made in this field opens the way for synthetic approaches to convert peptides with promising functional leads into structure-minimized and more stable forms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo3QhPT6JMbrVg90H21EOLACvtfcHk0ljrmK82RcyJig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlKlsrbN&md5=cc7cfed7b6249b9ce98a7051def886ee</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.3390%2Fmolecules18089797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules18089797%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DAneja%26aufirst%3DR.%26aulast%3DChaiken%26aufirst%3DI.%26atitle%3DClick%2520chemistry%2520in%2520peptide-based%2520drug%2520design%26jtitle%3DMolecules%26date%3D2013%26volume%3D18%26spage%3D9797%26epage%3D9817%26doi%3D10.3390%2Fmolecules18089797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caporale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandomenico, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvo, M.</span></span> <span> </span><span class="NLM_article-title">Automatic procedures for the synthesis of difficult peptides using oxyma as activating reagent: A comparative study on the use of bases and on different deprotection and agitation conditions</span>. <i>Peptides</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/j.peptides.2018.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2Fj.peptides.2018.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=29486214" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjvVClsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2018&pages=38-46&author=A.+Caporaleauthor=N.+Dotiauthor=A.+Montiauthor=A.+Sandomenicoauthor=M.+Ruvo&title=Automatic+procedures+for+the+synthesis+of+difficult+peptides+using+oxyma+as+activating+reagent%3A+A+comparative+study+on+the+use+of+bases+and+on+different+deprotection+and+agitation+conditions&doi=10.1016%2Fj.peptides.2018.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic procedures for the synthesis of difficult peptides using oxyma as activating reagent: A comparative study on the use of bases and on different deprotection and agitation conditions</span></div><div class="casAuthors">Caporale, A.; Doti, N.; Monti, A.; Sandomenico, A.; Ruvo, M.</div><div class="citationInfo"><span class="NLM_cas:title">Peptides (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">38-46</span>CODEN:
                <span class="NLM_cas:coden">PPTDD5</span>;
        ISSN:<span class="NLM_cas:issn">0196-9781</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Solid-Phase Peptide Synthesis (SPPS) is a rapid and efficient methodol. for the chem. synthesis of peptides and small proteins.  However, the assembly of peptide sequences classified as "difficult" poses severe synthetic problems in SPPS for the occurrence of extensive aggregation of growing peptide chains which often leads to synthesis failure.  In this framework, we have investigated the impact of different synthetic procedures on the yield and final purity of three well-known "difficult peptides" prepd. using oxyma as additive for the coupling steps.  In particular, we have comparatively investigated the use of piperidine and morpholine/DBU as deprotection reagents, the addn. of DIPEA, collidine and N-methylmorpholine as bases to the coupling reagent.  Moreover, the effect of different agitation modalities during the acylation reactions has been investigated.  Data obtained represent a step forward in optimizing strategies for the synthesis of "difficult peptides".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot8kFiXgO1VLVg90H21EOLACvtfcHk0ljC6qWJsheNOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjvVClsbw%253D&md5=e62fa59f680ab328407d6992db13b802</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.peptides.2018.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.peptides.2018.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DCaporale%26aufirst%3DA.%26aulast%3DDoti%26aufirst%3DN.%26aulast%3DMonti%26aufirst%3DA.%26aulast%3DSandomenico%26aufirst%3DA.%26aulast%3DRuvo%26aufirst%3DM.%26atitle%3DAutomatic%2520procedures%2520for%2520the%2520synthesis%2520of%2520difficult%2520peptides%2520using%2520oxyma%2520as%2520activating%2520reagent%253A%2520A%2520comparative%2520study%2520on%2520the%2520use%2520of%2520bases%2520and%2520on%2520different%2520deprotection%2520and%2520agitation%2520conditions%26jtitle%3DPeptides%26date%3D2018%26volume%3D102%26spage%3D38%26epage%3D46%26doi%3D10.1016%2Fj.peptides.2018.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caporale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandomenico, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruvo, M.</span></span> <span> </span><span class="NLM_article-title">Evaluation of combined use of Oxyma and HATU in aggregating peptide sequences</span>. <i>J. Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">272</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1002/psc.2977</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1002%2Fpsc.2977" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=28139012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslegtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=272-281&author=A.+Caporaleauthor=N.+Dotiauthor=A.+Sandomenicoauthor=M.+Ruvo&title=Evaluation+of+combined+use+of+Oxyma+and+HATU+in+aggregating+peptide+sequences&doi=10.1002%2Fpsc.2977"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of combined use of Oxyma and HATU in aggregating peptide sequences</span></div><div class="casAuthors">Caporale, Andrea; Doti, Nunzianna; Sandomenico, Annamaria; Ruvo, Menotti</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Peptide Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">272-281</span>CODEN:
                <span class="NLM_cas:coden">JPSIEI</span>;
        ISSN:<span class="NLM_cas:issn">1075-2617</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Polypeptides are finding increasing applications as therapeutics because of their specificity that often translates into excellent safety, tolerability, and efficacy profiles in humans.  New synthetic methodologies for their prepn. are thereby continuously sought to reduce the costs assocd. to chain assembly and purifn.  Although solid-phase peptide synthesis has become one of the most advanced synthetic procedures at both lab. and industrial scale, the process is often complicated by aggregation phenomena originating from the combined occurrence of intermol. and intramol. hydrogen bonding, hydrophobic interactions, or other effects.  Altogether, these effects cause accumulation of many side products and synthetic mixts. extremely hard to sep. and purify, strongly affecting the costs of the final material.  In the attempt to optimize the coupling steps of some well-known aggregating or otherwise difficult to obtain peptides, we have comparatively investigated the use of Oxyma/DIC and HATU/Sym-collidine as second coupling reagents in double coupling settings for the prepn. of some model peptides.  Comparative anal. data obtained on the unpurified products with the two different protocols clearly show that the use of Oxyma/DIC largely improves the content of the target mols. in the final crude materials, making the synthesis more convenient and cost-effective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotT_SozIStk7Vg90H21EOLACvtfcHk0ljC6qWJsheNOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslegtb4%253D&md5=99bf6ed623303d83a15d147bb38087f6</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fpsc.2977&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsc.2977%26sid%3Dliteratum%253Aachs%26aulast%3DCaporale%26aufirst%3DA.%26aulast%3DDoti%26aufirst%3DN.%26aulast%3DSandomenico%26aufirst%3DA.%26aulast%3DRuvo%26aufirst%3DM.%26atitle%3DEvaluation%2520of%2520combined%2520use%2520of%2520Oxyma%2520and%2520HATU%2520in%2520aggregating%2520peptide%2520sequences%26jtitle%3DJ.%2520Pept.%2520Sci.%26date%3D2017%26volume%3D23%26spage%3D272%26epage%3D281%26doi%3D10.1002%2Fpsc.2977" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scopes, R. K.</span></span> <span> </span><span class="NLM_article-title">Measurement of protein by spectrophotometry at 205 nm</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1974</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">282</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(74)90034-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2F0003-2697%2874%2990034-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=4407487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaE2cXksVOjsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=1974&pages=277-282&author=R.+K.+Scopes&title=Measurement+of+protein+by+spectrophotometry+at+205+nm&doi=10.1016%2F0003-2697%2874%2990034-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Measurement of protein by spectrophotometry at 205nm</span></div><div class="casAuthors">Scopes, R. K.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1974</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">277-82</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A method was described for the measurement of protein concn. by using the peptide bond absorption at 205 nm.  The extinction coeff. (ε) at 205 nm was estd., allowing for the absorption due to tryptophan and tyrosine residues, by measuring the absorbance at 280 nm as well as at 205 nm.  The estd. ε205 was compared with the actual ε205 for a no. of proteins, the mean error being <2%.  This was about 3 times better than using an av. ε2051mg/ml of 31 and approached the range of exptl. error inherent in any method of protein estn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFO6zpXSyQvrVg90H21EOLACvtfcHk0ljC6qWJsheNOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXksVOjsb0%253D&md5=ff53a6ad935bde82fc8c8a60bb606760</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2874%2990034-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252874%252990034-7%26sid%3Dliteratum%253Aachs%26aulast%3DScopes%26aufirst%3DR.%2BK.%26atitle%3DMeasurement%2520of%2520protein%2520by%2520spectrophotometry%2520at%2520205%2520nm%26jtitle%3DAnal.%2520Biochem.%26date%3D1974%26volume%3D59%26spage%3D277%26epage%3D282%26doi%3D10.1016%2F0003-2697%2874%2990034-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Braunschweiler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, R. R.</span></span> <span> </span><span class="NLM_article-title">Coherence transfer by isotropic mixing: Application to proton correlation spectroscopy</span>. <i>J. Magn. Reson.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">528</span>, <span class="refDoi"> DOI: 10.1016/0022-2364(83)90226-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2F0022-2364%2883%2990226-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaL3sXks1eju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1983&pages=521-528&author=R.+Braunschweilerauthor=R.+R.+Ernst&title=Coherence+transfer+by+isotropic+mixing%3A+Application+to+proton+correlation+spectroscopy&doi=10.1016%2F0022-2364%2883%2990226-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Coherence transfer by isotropic mixing:  application to proton correlation spectroscopy</span></div><div class="casAuthors">Braunschweiler, L.; Ernst, R. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Magnetic Resonance (1969-1992)</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">521-8</span>CODEN:
                <span class="NLM_cas:coden">JOMRA4</span>;
        ISSN:<span class="NLM_cas:issn">0022-2364</span>.
    </div><div class="casAbstract">A new mechanism of coherence transfer is proposed which is useful for 2-dimensional correlation spectroscopy in liqs.  Cross-peaks are generated between all members of a coupled spin network leading to 4 most complete correlation diagram.  The scheme produces net coherence transfer and can be arranged to create pure absorption mode spectra with peaks of pos. intensity.  A new type of 2-dimensional correlation NMR expt. is illustrated.  After a prepn. pulse the spin system is allowed to evolve freely under the (assumed) weak coupling Hamiltonian in terms of single spin modes.  For the mixing process the Hamiltonian is modified to a form which is dominated by the full isotropic coupling terms.  The coherence of the spins is periodically exchanged during the mixing time and is obsd. during the detection period again under the unperturbed Hamiltonian.  Illustrative examples of 2-dimensional correlated spectra with and without relayed peaks are shown.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo25_GwCJe3XrVg90H21EOLACvtfcHk0ljC6qWJsheNOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXks1eju7c%253D&md5=693eaaee37b0224796b7d9e3403759c6</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2F0022-2364%2883%2990226-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2364%252883%252990226-3%26sid%3Dliteratum%253Aachs%26aulast%3DBraunschweiler%26aufirst%3DR.%26aulast%3DErnst%26aufirst%3DR.%2BR.%26atitle%3DCoherence%2520transfer%2520by%2520isotropic%2520mixing%253A%2520Application%2520to%2520proton%2520correlation%2520spectroscopy%26jtitle%3DJ.%2520Magn.%2520Reson.%26date%3D1983%26volume%3D53%26spage%3D521%26epage%3D528%26doi%3D10.1016%2F0022-2364%2883%2990226-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wüthrich, K.</span></span> <span> </span><span class="NLM_article-title">A two-dimensional nuclear Overhauser enhancement (2D NOE) experiment for the elucidation of complete proton-proton cross-relaxation networks in biological macromolecules</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">6</span>, <span class="refDoi"> DOI: 10.1016/0006-291x(80)90695-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2F0006-291X%2880%2990695-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=7417242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaL3cXlsVGlu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1980&pages=1-6&author=A.+Kumarauthor=R.+R.+Ernstauthor=K.+W%C3%BCthrich&title=A+two-dimensional+nuclear+Overhauser+enhancement+%282D+NOE%29+experiment+for+the+elucidation+of+complete+proton-proton+cross-relaxation+networks+in+biological+macromolecules&doi=10.1016%2F0006-291x%2880%2990695-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">A two-dimensional nuclear Overhauser enhancement (2D NOE) experiment for the elucidation of complete proton-proton cross-relaxation networks in biological macromolecules</span></div><div class="casAuthors">Kumar, Anil; Ernst, R. R.; Wuethrich, K.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-6</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    </div><div class="casAbstract">The recently developed technique of 2D cross-relaxation spectroscopy was used for systematic measurements of selective NOE in the high-resoln. 1H NMR spectra of biol. macromols. in soln.  Compared to conventional 1-dimensional NOE studies, the 2D NOE expt. has the principal advantage that it avoids detrimental effects arising from the limited selectivity of preirradn. in crowded spectral regions.  Furthermore, it yields, with a single instrument setting, a complete network of NOEs between all the protons in the macromol.  The resulting information on intramol. proton-proton distances provides a new avenue for studies of the spatial structures of biopolymers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiH0XrevG7UbVg90H21EOLACvtfcHk0lhR-olhK-JFXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXlsVGlu7Y%253D&md5=fe460f61af8426f88ba1aaec0efd6bfa</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2F0006-291X%2880%2990695-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-291X%252880%252990695-6%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DErnst%26aufirst%3DR.%2BR.%26aulast%3DW%25C3%25BCthrich%26aufirst%3DK.%26atitle%3DA%2520two-dimensional%2520nuclear%2520Overhauser%2520enhancement%2520%25282D%2520NOE%2529%2520experiment%2520for%2520the%2520elucidation%2520of%2520complete%2520proton-proton%2520cross-relaxation%2520networks%2520in%2520biological%2520macromolecules%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1980%26volume%3D95%26spage%3D1%26epage%3D6%26doi%3D10.1016%2F0006-291x%2880%2990695-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Griesinger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, R. R.</span></span> <span> </span><span class="NLM_article-title">Frequency Offset Effects and Their Elimination in NMR Rotating-Frame Cross-Relaxation Spectroscopy</span>. <i>J. Magn. Reson.</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">271</span>, <span class="refDoi"> DOI: 10.1016/0022-2364(87)90035-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2F0022-2364%2887%2990035-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaL1cXos1GksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=1987&pages=261-271&author=C.+Griesingerauthor=R.+R.+Ernst&title=Frequency+Offset+Effects+and+Their+Elimination+in+NMR+Rotating-Frame+Cross-Relaxation+Spectroscopy&doi=10.1016%2F0022-2364%2887%2990035-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Frequency offset effects and their elimination in NMR rotating-frame cross-relaxation spectroscopy</span></div><div class="casAuthors">Griesinger, C.; Ernst, R. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Magnetic Resonance (1969-1992)</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-71</span>CODEN:
                <span class="NLM_cas:coden">JOMRA4</span>;
        ISSN:<span class="NLM_cas:issn">0022-2364</span>.
    </div><div class="casAbstract">The effects of frequency offsets in NMR rotating-frame cross-relaxation expts. are analyzed.  A quant. explanation is given for the offset dependence of peak intensities and multiplet structure in 2-dimensional rotating-frame Overhauser effect spectroscopy.  A modified pulse sequence is proposed to eliminate offset effects and to improve the signal-to-noise ratio.  The spectra of cyclo(-D-Ala1-Ala4-) is then given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryswO43JPv2bVg90H21EOLACvtfcHk0lhR-olhK-JFXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXos1GksA%253D%253D&md5=276b634ddc995061f0d8931081b18949</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2F0022-2364%2887%2990035-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2364%252887%252990035-7%26sid%3Dliteratum%253Aachs%26aulast%3DGriesinger%26aufirst%3DC.%26aulast%3DErnst%26aufirst%3DR.%2BR.%26atitle%3DFrequency%2520Offset%2520Effects%2520and%2520Their%2520Elimination%2520in%2520NMR%2520Rotating-Frame%2520Cross-Relaxation%2520Spectroscopy%26jtitle%3DJ.%2520Magn.%2520Reson.%26date%3D1987%26volume%3D75%26spage%3D261%26epage%3D271%26doi%3D10.1016%2F0022-2364%2887%2990035-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otlewski, J.</span></span> <span> </span><span class="NLM_article-title">Amide proton temperature coefficients as hydrogen bond indicators in proteins</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">261</span>, <span class="refDoi"> DOI: 10.1023/a:1012911329730</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1023%2FA%3A1012911329730" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=11775741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3MXptlKrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2001&pages=249-261&author=T.+Cierpickiauthor=J.+Otlewski&title=Amide+proton+temperature+coefficients+as+hydrogen+bond+indicators+in+proteins&doi=10.1023%2Fa%3A1012911329730"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Amide proton temperature coefficients as hydrogen bond indicators in proteins</span></div><div class="casAuthors">Cierpicki, Tomasz; Otlewski, Jacek</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">249-261</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">Correlations between amide proton temp. coeffs. (ΔσHN/ΔT) and hydrogen bonds were investigated for a data set of 793 amides derived from 14 proteins.  For amide protons showing temp. gradients more pos. than -4.6 ppb/K there is a hydrogen bond predictivity value exceeding 85%.  It increases to over 93% for amides within the range between -4 and -1 ppb/K.  Detailed anal. shows an inverse proportionality between amide proton temp. coeffs. and hydrogen bond lengths.  Furthermore, for hydrogen bonds of similar bond lengths, values of temp. gradients in α-helixes are on av. 1 ppb/K more neg. than in β-sheets.  In consequence, a no. of amide protons in α-helixes involved in hydrogen bonds shorter than 2 Å show ΔσHN/ΔT < -4.6 ppb/K.  Due to longer hydrogen bonds, 90% of amides in 310 helixes and 98% in β-turns have temp. coeffs. more pos. than -4.6ppb/K.  Ring current effect also significantly influences temp. coeffs. of amide protons.  In seven out of eight cases non-hydrogen bonded amides strongly deshielded by neighboring arom. rings show temp. coeffs. more pos. than -2 ppb/K.  In general, amide proton temp. gradients do not change with pH unless they correspond to conformational changes.  Three examples of pH dependent equil. showing hydrogen bond formation at higher pH were found.  In conclusion, amide proton temp. coeffs. offer an attractive and simple way to confirm existence of hydrogen bonds in NMR detd. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-yCsJE1GrlbVg90H21EOLACvtfcHk0lhR-olhK-JFXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXptlKrs7g%253D&md5=d0662943f22587f95e2be3074503b33d</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1023%2FA%3A1012911329730&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1012911329730%26sid%3Dliteratum%253Aachs%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DOtlewski%26aufirst%3DJ.%26atitle%3DAmide%2520proton%2520temperature%2520coefficients%2520as%2520hydrogen%2520bond%2520indicators%2520in%2520proteins%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2001%26volume%3D21%26spage%3D249%26epage%3D261%26doi%3D10.1023%2Fa%3A1012911329730" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhukov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otlewski, J.</span></span> <span> </span><span class="NLM_article-title">Hydrogen bonds in human ubiquitin reflected in temperature coefficients of amide protons</span>. <i>J. Magn. Reson.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">178</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1006/jmre.2002.2597</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1006%2Fjmre.2002.2597" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=12323135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD38XntFShtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2002&pages=178-180&author=T.+Cierpickiauthor=I.+Zhukovauthor=R.+A.+Byrdauthor=J.+Otlewski&title=Hydrogen+bonds+in+human+ubiquitin+reflected+in+temperature+coefficients+of+amide+protons&doi=10.1006%2Fjmre.2002.2597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen Bonds in Human Ubiquitin Reflected in Temperature Coefficients of Amide Protons</span></div><div class="casAuthors">Cierpicki, Tomasz; Zhukov, Igor; Byrd, R. Andrew; Otlewski, Jacek</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Magnetic Resonance</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">178-180</span>CODEN:
                <span class="NLM_cas:coden">JMARF3</span>;
        ISSN:<span class="NLM_cas:issn">1090-7807</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">Anal. of amide proton temp. coeffs. (ΔσHN/ΔT) in human ubiquitin shows their usefulness in indicating hydrogen bonds.  The availability of a very accurate soln. structure of ubiquitin enables the precise detn. of hydrogen bonds and increases the reliability of the anal. of chem. shift temp. gradients.  Values of ΔσHN/ΔT more pos. than -4.6 ppb/K are very good indicators of hydrogen bonds.  Addnl., a weak temp. dependence of non-hydrogen-bonded amides was obsd. for amide protons that are significantly shifted upfield.  We obsd. that temp. gradients of amide protons involved in short hydrogen bonds are related to donor-acceptor distances.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2qiprxXj5ubVg90H21EOLACvtfcHk0lhR-olhK-JFXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntFShtLs%253D&md5=02801090d9f21616c8741d7e6dcca29a</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1006%2Fjmre.2002.2597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmre.2002.2597%26sid%3Dliteratum%253Aachs%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DZhukov%26aufirst%3DI.%26aulast%3DByrd%26aufirst%3DR.%2BA.%26aulast%3DOtlewski%26aufirst%3DJ.%26atitle%3DHydrogen%2520bonds%2520in%2520human%2520ubiquitin%2520reflected%2520in%2520temperature%2520coefficients%2520of%2520amide%2520protons%26jtitle%3DJ.%2520Magn.%2520Reson.%26date%3D2002%26volume%3D157%26spage%3D178%26epage%3D180%26doi%3D10.1006%2Fjmre.2002.2597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delaglio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzesiek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuister, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeifer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, A.</span></span> <span> </span><span class="NLM_article-title">NMRPipe: a multidimensional spectral processing system based on UNIX pipes</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1007/BF00197809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2FBF00197809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=8520220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1995&pages=277-293&author=F.+Delaglioauthor=S.+Grzesiekauthor=G.+W.+Vuisterauthor=G.+Zhuauthor=J.+Pfeiferauthor=A.+Bax&title=NMRPipe%3A+a+multidimensional+spectral+processing+system+based+on+UNIX+pipes&doi=10.1007%2FBF00197809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">NMRPipe: a multidimensional spectral processing system based on UNIX pipes</span></div><div class="casAuthors">Delaglio, Frank; Grzesiek, Stephan; Vuister, Geerten W.; Zhu, Guang; Pfeifer, John; Bax, Ad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-93</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">The NMRPipe system is a UNIX software environment of processing, graphics, and anal. tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments.  The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control.  In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction.  Complete multidimensional processing schemes are constructed as simple UNIX shell scripts.  The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing.  The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSIUdXrdcEgbVg90H21EOLACvtfcHk0lhlkhbsRPIJEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK&md5=a670fca5b164083e2178fafd2fb951ff</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2FBF00197809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00197809%26sid%3Dliteratum%253Aachs%26aulast%3DDelaglio%26aufirst%3DF.%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DVuister%26aufirst%3DG.%2BW.%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DPfeifer%26aufirst%3DJ.%26aulast%3DBax%26aufirst%3DA.%26atitle%3DNMRPipe%253A%2520a%2520multidimensional%2520spectral%2520processing%2520system%2520based%2520on%2520UNIX%2520pipes%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1995%26volume%3D6%26spage%3D277%26epage%3D293%26doi%3D10.1007%2FBF00197809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span>; <span class="NLM_string-name">Kneller, D. G.</span></span> <i>Sparky 3</i>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>, <span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=T.+D.+Goddard&author=D.+G.+Kneller&title=Sparky+3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26btitle%3DSparky%25203%26pub%3DUniversity%2520of%2520California%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, R. L. J.</span></span> <i>Optimizing the Process of Nuclear Magnetic Resonance Spectrum Analysis and Computer Aided Resonance Assignment Dissertation</i>; <span class="NLM_publisher-name">Swiss Federal Institute of Technology</span>: <span class="NLM_publisher-loc">Zürich</span>, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=R.+L.+J.+Keller&title=Optimizing+the+Process+of+Nuclear+Magnetic+Resonance+Spectrum+Analysis+and+Computer+Aided+Resonance+Assignment+Dissertation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DR.%2BL.%2BJ.%26btitle%3DOptimizing%2520the%2520Process%2520of%2520Nuclear%2520Magnetic%2520Resonance%2520Spectrum%2520Analysis%2520and%2520Computer%2520Aided%2520Resonance%2520Assignment%2520Dissertation%26pub%3DSwiss%2520Federal%2520Institute%2520of%2520Technology%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berjanskii, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span> <span> </span><span class="NLM_article-title">A simple method to predict protein flexibility using secondary chemical shifts</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">14970</span>– <span class="NLM_lpage">14971</span>, <span class="refDoi"> DOI: 10.1021/ja054842f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja054842f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKhsb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=14970-14971&author=M.+V.+Berjanskiiauthor=D.+S.+Wishart&title=A+simple+method+to+predict+protein+flexibility+using+secondary+chemical+shifts&doi=10.1021%2Fja054842f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">A Simple Method To Predict Protein Flexibility Using Secondary Chemical Shifts</span></div><div class="casAuthors">Berjanskii, Mark V.; Wishart, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">14970-14971</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein motions play a crit. role in many biol. processes, such as enzyme catalysis, allosteric regulation, antigen-antibody interactions, and protein-DNA binding.  NMR spectroscopy occupies a unique place among methods for investigating protein dynamics due to its ability to provide site-specific information about protein motions over a large range of time scales.  However, most NMR methods require a detailed knowledge of the 3D structure and/or the collection of addnl. exptl. data (NOEs, T1, T2, etc.) to accurately measure protein dynamics.  Here the authors present a simple method based on chem. shift data that allows accurate, quant., site-specific mapping of protein backbone mobility without the need of a three-dimensional structure or the collection and anal. of NMR relaxation data.  Further, the authors show that this chem. shift method is able to quant. predict per-residue RMSD values (from both MD simulations and NMR structural ensembles) as well as model-free backbone order parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjDPJ5T3Wtx7Vg90H21EOLACvtfcHk0lhlkhbsRPIJEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKhsb7J&md5=26b0104fc58b338a3c01f1b97aaadf1c</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fja054842f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja054842f%26sid%3Dliteratum%253Aachs%26aulast%3DBerjanskii%26aufirst%3DM.%2BV.%26aulast%3DWishart%26aufirst%3DD.%2BS.%26atitle%3DA%2520simple%2520method%2520to%2520predict%2520protein%2520flexibility%2520using%2520secondary%2520chemical%2520shifts%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D14970%26epage%3D14971%26doi%3D10.1021%2Fja054842f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berjanskii, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span> <span> </span><span class="NLM_article-title">The RCI server: rapid and accurate calculation of protein flexibility using chemical shifts</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">W531</span>– <span class="NLM_lpage">W537</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1093%2Fnar%2Fgkm328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=17485469" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W531-W537&author=M.+V.+Berjanskiiauthor=D.+S.+Wishart&title=The+RCI+server%3A+rapid+and+accurate+calculation+of+protein+flexibility+using+chemical+shifts&doi=10.1093%2Fnar%2Fgkm328"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm328%26sid%3Dliteratum%253Aachs%26aulast%3DBerjanskii%26aufirst%3DM.%2BV.%26aulast%3DWishart%26aufirst%3DD.%2BS.%26atitle%3DThe%2520RCI%2520server%253A%2520rapid%2520and%2520accurate%2520calculation%2520of%2520protein%2520flexibility%2520using%2520chemical%2520shifts%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW531%26epage%3DW537%26doi%3D10.1093%2Fnar%2Fgkm328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Güntert, P.</span></span> <span> </span><span class="NLM_article-title">Automated structure determination from NMR spectra</span>. <i>Eur. Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">143</span>, <span class="refDoi"> DOI: 10.1007/s00249-008-0367-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2Fs00249-008-0367-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=18807026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotVelug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2009&pages=129-143&author=P.+G%C3%BCntert&title=Automated+structure+determination+from+NMR+spectra&doi=10.1007%2Fs00249-008-0367-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Automated structure determination from NMR spectra</span></div><div class="casAuthors">Guentert, Peter</div><div class="citationInfo"><span class="NLM_cas:title">European Biophysics Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">129-143</span>CODEN:
                <span class="NLM_cas:coden">EBJOE8</span>;
        ISSN:<span class="NLM_cas:issn">0175-7571</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Automated methods for protein structure detn. by NMR have increasingly gained acceptance and are now widely used for the automated assignment of distance restraints and the calcn. of three-dimensional structures.  This review gives an overview of the techniques for automated protein structure anal. by NMR, including both NOE-based approaches and methods relying on other exptl. data such as residual dipolar couplings and chem. shifts, and presents the FLYA algorithm for the fully automated NMR structure detn. of proteins that is suitable to substitute all manual spectra anal. and thus overcomes a major efficiency limitation of the NMR method for protein structure detn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOooOL4oujYrVg90H21EOLACvtfcHk0lhlkhbsRPIJEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotVelug%253D%253D&md5=93eaa09a8420d2a1fb583806c896c3d2</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1007%2Fs00249-008-0367-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00249-008-0367-z%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25BCntert%26aufirst%3DP.%26atitle%3DAutomated%2520structure%2520determination%2520from%2520NMR%2520spectra%26jtitle%3DEur.%2520Biophys.%2520J.%26date%3D2009%26volume%3D38%26spage%3D129%26epage%3D143%26doi%3D10.1007%2Fs00249-008-0367-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaglio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornilescu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, A.</span></span> <span> </span><span class="NLM_article-title">TALOS+: a hybrid method for predicting protein backbone torsion angles from NMR chemical shifts</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">223</span>, <span class="refDoi"> DOI: 10.1007/s10858-009-9333-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2Fs10858-009-9333-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=19548092" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD1MXovFajtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2009&pages=213-223&author=Y.+Shenauthor=F.+Delaglioauthor=G.+Cornilescuauthor=A.+Bax&title=TALOS%2B%3A+a+hybrid+method+for+predicting+protein+backbone+torsion+angles+from+NMR+chemical+shifts&doi=10.1007%2Fs10858-009-9333-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">TALOS+: a hybrid method for predicting protein backbone torsion angles from NMR chemical shifts</span></div><div class="casAuthors">Shen, Yang; Delaglio, Frank; Cornilescu, Gabriel; Bax, Ad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">213-223</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">NMR chem. shifts in proteins depend strongly on local structure.  The program TALOS establishes an empirical relation between 13C, 15N and 1H chem. shifts and backbone torsion angles .vphi. and ψ.  Extension of the original 20-protein database to 200 proteins increased the fraction of residues for which backbone angles could be predicted from 65 to 74%, while reducing the error rate from 3 to 2.5%.  Addn. of a two-layer neural network filter to the database fragment selection process forms the basis for a new program, TALOS+, which further enhances the prediction rate to 88.5%, without increasing the error rate.  Excluding the 2.5% of residues for which TALOS+ makes predictions that strongly differ from those obsd. in the cryst. state, the accuracy of predicted .vphi. and ψ angles, equals ±13°.  Large discrepancies between predictions and crystal structures are primarily limited to loop regions, and for the few cases where multiple X-ray structures are available such residues are often found in different states in the different structures.  The TALOS+ output includes predictions for individual residues with missing chem. shifts, and the neural network component of the program also predicts secondary structure with good accuracy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJZxR7vg_rarVg90H21EOLACvtfcHk0lh_sNroN6ysgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXovFajtr8%253D&md5=ee3f8647a02e19937988056fd7af04b2</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2Fs10858-009-9333-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10858-009-9333-z%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DDelaglio%26aufirst%3DF.%26aulast%3DCornilescu%26aufirst%3DG.%26aulast%3DBax%26aufirst%3DA.%26atitle%3DTALOS%252B%253A%2520a%2520hybrid%2520method%2520for%2520predicting%2520protein%2520backbone%2520torsion%2520angles%2520from%2520NMR%2520chemical%2520shifts%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2009%26volume%3D44%26spage%3D213%26epage%3D223%26doi%3D10.1007%2Fs10858-009-9333-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koradi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billeter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wüthrich, K.</span></span> <span> </span><span class="NLM_article-title">MOLMOL: a program for display and analysis of macromolecular structures</span>. <i>J. Mol. Graph.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1016/0263-7855(96)00009-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1016%2F0263-7855%2896%2900009-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=8744573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK28Xis12nsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=51-55&author=R.+Koradiauthor=M.+Billeterauthor=K.+W%C3%BCthrich&title=MOLMOL%3A+a+program+for+display+and+analysis+of+macromolecular+structures&doi=10.1016%2F0263-7855%2896%2900009-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">MOLMOL: a program for display and analysis of macromolecular structures</span></div><div class="casAuthors">Koradi, Reto; Billeter, Martin; Wuethrich, Kurt</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-5, plates, 29-32</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">MOLMOL is a mol. graphics program for display, anal., and manipulation of three-dimensional structures of biol. macromols., with special emphasis on NMR soln. structures of proteins and nucleic acids.  MOLMOL has a graphical user interface with menus, dialog boxes, and online help.  The display possibilities include conventional presentation, as well as novel schematic drawings, with the option of combining different presentations in one view of a mol.  Covalent mol. structures can be modified by addn. or removal of individual atoms and bonds, and three-dimensional structures can be manipulated by interactive rotation about individual bonds.  Special efforts were made to allow for appropriate display and anal. of the sets of typically 20-40 conformers that are conventionally used to represent the result of an NMR structure detn., using functions for superimposing sets of conformers, calcn. of root mean square distance (RMSD) values, identification of hydrogen bonds, checking and displaying violations of NMR constraints, and identification and listing of short distances between pairs of hydrogen atoms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDF50yz_iOsbVg90H21EOLACvtfcHk0lh_sNroN6ysgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis12nsrc%253D&md5=dab669ff481b19cde8fbe2e4234711fa</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2F0263-7855%2896%2900009-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0263-7855%252896%252900009-4%26sid%3Dliteratum%253Aachs%26aulast%3DKoradi%26aufirst%3DR.%26aulast%3DBilleter%26aufirst%3DM.%26aulast%3DW%25C3%25BCthrich%26aufirst%3DK.%26atitle%3DMOLMOL%253A%2520a%2520program%2520for%2520display%2520and%2520analysis%2520of%2520macromolecular%2520structures%26jtitle%3DJ.%2520Mol.%2520Graph.%26date%3D1996%26volume%3D14%26spage%3D51%26epage%3D55%26doi%3D10.1016%2F0263-7855%2896%2900009-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera—A visualization system for exploratory research and analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1605</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%E2%80%94A+visualization+system+for+exploratory+research+and+analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lh_sNroN6ysgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%25E2%2580%2594A%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rullmannn, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacArthur, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaptein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">AQUA and PROCHECK-NMR: programs for checking the quality of protein structures solved by NMR</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">486</span>, <span class="refDoi"> DOI: 10.1007/BF00228148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1007%2FBF00228148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=9008363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK2sXlvVSnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1996&pages=477-486&author=R.+A.+Laskowskiauthor=J.+A.+Rullmannnauthor=M.+W.+MacArthurauthor=R.+Kapteinauthor=J.+M.+Thornton&title=AQUA+and+PROCHECK-NMR%3A+programs+for+checking+the+quality+of+protein+structures+solved+by+NMR&doi=10.1007%2FBF00228148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">AQUA and PROCHECK-NMR: programs for checking the quality of protein structures solved by NMR</span></div><div class="casAuthors">Laskowski, Roman A.; Antoon, J.; Rullmann, C.; Macarthur, Malcolm W.; Kaptein, Robert; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">477-486</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">The AQUA and PROCHECK-NMR programs provide a means of validating the geometry and restraint violations of an ensemble of protein structures solved by soln. NMR.  The outputs include a detailed breakdown of the restraint violations, a no. of plots in PostScript format and summary statistics.  These various analyses indicate both the degree of agreement of the model structures with the exptl. data, and the quality of their geometrical properties.  They are intended to be of use both to support ongoing NMR structure detn. and in the validation of the final results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsafj3zezC3rVg90H21EOLACvtfcHk0lgAu5Jgldi4eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvVSnsw%253D%253D&md5=030956fb35cbfdaab78801c61ba5a3e1</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1007%2FBF00228148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00228148%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DRullmannn%26aufirst%3DJ.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DKaptein%26aufirst%3DR.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DAQUA%2520and%2520PROCHECK-NMR%253A%2520programs%2520for%2520checking%2520the%2520quality%2520of%2520protein%2520structures%2520solved%2520by%2520NMR%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1996%26volume%3D8%26spage%3D477%26epage%3D486%26doi%3D10.1007%2FBF00228148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krüger, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span> <span> </span><span class="NLM_article-title">NMSim web server: integrated approach for normal mode-based geometric simulations of biologically relevant conformational transitions in proteins</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">W310</span>– <span class="NLM_lpage">W316</span>, <span class="refDoi"> DOI: 10.1093/nar/gks478</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1093%2Fnar%2Fgks478" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=22669906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVCqsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=W310-W316&author=D.+M.+Kr%C3%BCgerauthor=A.+Ahmedauthor=H.+Gohlke&title=NMSim+web+server%3A+integrated+approach+for+normal+mode-based+geometric+simulations+of+biologically+relevant+conformational+transitions+in+proteins&doi=10.1093%2Fnar%2Fgks478"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">NMSim Web Server: integrated approach for normal mode-based geometric simulations of biologically relevant conformational transitions in proteins</span></div><div class="casAuthors">Krueger, Dennis M.; Ahmed, Aqeel; Gohlke, Holger</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W310-W316</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The NMSim web server implements a three-step approach for multiscale modeling of protein conformational changes.  First, the protein structure is coarse-grained using the FIRST software.  Second, a rigid cluster normal-mode anal. provides low-frequency normal modes.  Third, these modes are used to extend the recently introduced idea of constrained geometric simulations by biasing backbone motions of the protein, whereas side chain motions are biased toward favorable rotamer states (NMSim).  The generated structures are iteratively cor. regarding steric clashes and stereochem. constraint violations.  The approach allows performing three simulation types: unbiased exploration of conformational space; pathway generation by a targeted simulation; and radius of gyration-guided simulation.  On a data set of proteins with exptl. obsd. conformational changes, the NMSim approach has been shown to be a computationally efficient alternative to mol. dynamics simulations for conformational sampling of proteins.  The generated conformations and pathways of conformational transitions can serve as input to docking approaches or more sophisticated sampling techniques.  The web server output is a trajectory of generated conformations, Jmol representations of the coarse-graining and a subset of the trajectory and data plots of structural analyses.  The NMSim webserver, accessible at http://www.nmsim.de, is free and open to all users with no login requirement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSHVzQuJQm3bVg90H21EOLACvtfcHk0lgAu5Jgldi4eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVCqsbc%253D&md5=8d17e24709fa81d55316c63a5b260ecb</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks478%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25BCger%26aufirst%3DD.%2BM.%26aulast%3DAhmed%26aufirst%3DA.%26aulast%3DGohlke%26aufirst%3DH.%26atitle%3DNMSim%2520web%2520server%253A%2520integrated%2520approach%2520for%2520normal%2520mode-based%2520geometric%2520simulations%2520of%2520biologically%2520relevant%2520conformational%2520transitions%2520in%2520proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DW310%26epage%3DW316%26doi%3D10.1093%2Fnar%2Fgks478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heath, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onufriev, A.</span></span> <span> </span><span class="NLM_article-title">H++: a server for estimating pKas and adding missing hydrogens to macromolecules</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">W368</span>– <span class="NLM_lpage">W371</span>, <span class="refDoi"> DOI: 10.1093/nar/gki464</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1093%2Fnar%2Fgki464" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=15980491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlslyqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=W368-W371&author=J.+C.+Gordonauthor=J.+B.+Myersauthor=T.+Foltaauthor=V.+Shojaauthor=L.+S.+Heathauthor=A.+Onufriev&title=H%2B%2B%3A+a+server+for+estimating+pKas+and+adding+missing+hydrogens+to+macromolecules&doi=10.1093%2Fnar%2Fgki464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">H++: a server for estimating pKas and adding missing hydrogens to macromolecules</span></div><div class="casAuthors">Gordon, John C.; Myers, Jonathan B.; Folta, Timothy; Shoja, Valia; Heath, Lenwood S.; Onufriev, Alexey</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W368-W371</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The structure and function of macromols. depend critically on the ionization (protonation) states of their acidic and basic groups.  A no. of existing practical methods predict protonation equil. pK consts. of macromols. based upon their at. resoln. Protein Data Bank (PDB) structures; the calcns. are often performed within the framework of the continuum electrostatics model.  Unfortunately, these methodologies are complex, involve multiple steps and require considerable investment of effort.  Our web server http://biophysics.cs.vt.edu/H++ provides access to a tool that automates this process, allowing both experts and novices to quickly obtain ests. of pKs as well as other related characteristics of biomols. such as isoelec. points, titrn. curves and energies of protonation microstates.  Protons are added to the input structure according to the calcd. ionization states of its titratable groups at the user-specified pH; the output is in the PQR (PDB + charges + radii) format.  In addn., corresponding coordinate and topol. files are generated in the format supported by the mol. modeling package AMBER.  The server is intended for a broad community of biochemists, mol. modelers, structural biologists and drug designers; it can also be used as an educational tool in biochem. courses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaTJ1ZbNsl1LVg90H21EOLACvtfcHk0lgAu5Jgldi4eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlslyqt7k%253D&md5=a9751f67f129a45a87e12187627d3a2b</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki464%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DJ.%2BC.%26aulast%3DMyers%26aufirst%3DJ.%2BB.%26aulast%3DFolta%26aufirst%3DT.%26aulast%3DShoja%26aufirst%3DV.%26aulast%3DHeath%26aufirst%3DL.%2BS.%26aulast%3DOnufriev%26aufirst%3DA.%26atitle%3DH%252B%252B%253A%2520a%2520server%2520for%2520estimating%2520pKas%2520and%2520adding%2520missing%2520hydrogens%2520to%2520macromolecules%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26spage%3DW368%26epage%3DW371%26doi%3D10.1093%2Fnar%2Fgki464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guex, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peitsch, M. C.</span></span> <span> </span><span class="NLM_article-title">SWISS-MODEL and the Swiss-Pdb Viewer: An environment for comparative protein modeling</span>. <i>Electrophoresis</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2714</span>– <span class="NLM_lpage">2723</span>, <span class="refDoi"> DOI: 10.1002/elps.1150181505</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1002%2Felps.1150181505" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=9504803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK1cXhvFaks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1997&pages=2714-2723&author=N.+Guexauthor=M.+C.+Peitsch&title=SWISS-MODEL+and+the+Swiss-Pdb+Viewer%3A+An+environment+for+comparative+protein+modeling&doi=10.1002%2Felps.1150181505"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">SWISS-MODEL and the Swiss-PdbViewer. An environment for comparative protein modeling</span></div><div class="casAuthors">Guex, Nicolas; Peitsch, Manuel C.</div><div class="citationInfo"><span class="NLM_cas:title">Electrophoresis</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2714-2723</span>CODEN:
                <span class="NLM_cas:coden">ELCTDN</span>;
        ISSN:<span class="NLM_cas:issn">0173-0835</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Comparative protein modeling is increasingly gaining interest since it is of great assistance during the rational design of mutagenesis expts.  The availability of this method, and the resulting models, has however been restricted by the availability of expensive computer hardware and software.  To overcome these limitations, the authors have developed an environment for comparative protein modeling that consists of SWISS-MODEL, a server for automated comparative protein modeling and of the SWISS-PdbViewer, a sequence to structure workbench.  The Swiss-PdbViewer not only acts as a client for SWISS-MODEL, but also provides a large selection of structure anal. and display tools.  In addn., the authors provide the SWISS-MODEL Repository, a database contg. more than 3500 automatically generated protein models.  By making such tools freely available to the scientific community, the authors hope to increase the use of protein structures and models in the process of expt. design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQsoTgcUeb6bVg90H21EOLACvtfcHk0lir-QwUt1KXzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhvFaks70%253D&md5=f97197dfdb1cf196099b0576f150d14b</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1002%2Felps.1150181505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Felps.1150181505%26sid%3Dliteratum%253Aachs%26aulast%3DGuex%26aufirst%3DN.%26aulast%3DPeitsch%26aufirst%3DM.%2BC.%26atitle%3DSWISS-MODEL%2520and%2520the%2520Swiss-Pdb%2520Viewer%253A%2520An%2520environment%2520for%2520comparative%2520protein%2520modeling%26jtitle%3DElectrophoresis%26date%3D1997%26volume%3D18%26spage%3D2714%26epage%3D2723%26doi%3D10.1002%2Felps.1150181505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutcliffe, M. J.</span></span> <span> </span><span class="NLM_article-title">An automated approach for clustering an ensemble of NMR-derived protein structures into conformationally related subfamilies</span>. <i>Protein Eng.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1063</span>– <span class="NLM_lpage">1065</span>, <span class="refDoi"> DOI: 10.1093/protein/9.11.1063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1093%2Fprotein%2F9.11.1063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=8961360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADyaK28XnsFamt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1996&pages=1063-1065&author=L.+A.+Kelleyauthor=S.+P.+Gardnerauthor=M.+J.+Sutcliffe&title=An+automated+approach+for+clustering+an+ensemble+of+NMR-derived+protein+structures+into+conformationally+related+subfamilies&doi=10.1093%2Fprotein%2F9.11.1063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">An automated approach for clustering an ensemble of NMR-derived protein structures into conformationally related subfamilies</span></div><div class="casAuthors">Kelley, Lawrence A.; Gardner, Stephen P.; Sutcliffe, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1063-1065</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A computer program has been developed that automatically, systematically, and rapidly clusters an ensemble of structures into a set of conformationally related subfamilies, and selects a representative structure from each cluster.  The program uses the method of av. linkage to define how clusters are built up, followed by the application of a penalty function that seeks to minimize simultaneously the no. of clusters and the spread across each cluster.  This program, known as NMRCLUST, is available via the World Wide Web (URL:http://neon.chem.le.ac.uk/) and by anonymous ftp from ftp.oxmol.co.uk.  Although developed for the anal. of NMR-derived structures, the program can be used to automatically cluster any data set.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1EjchxnvGGrVg90H21EOLACvtfcHk0lir-QwUt1KXzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XnsFamt74%253D&md5=6940b36a9dcbba4dd8352df595737cb2</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F9.11.1063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F9.11.1063%26sid%3Dliteratum%253Aachs%26aulast%3DKelley%26aufirst%3DL.%2BA.%26aulast%3DGardner%26aufirst%3DS.%2BP.%26aulast%3DSutcliffe%26aufirst%3DM.%2BJ.%26atitle%3DAn%2520automated%2520approach%2520for%2520clustering%2520an%2520ensemble%2520of%2520NMR-derived%2520protein%2520structures%2520into%2520conformationally%2520related%2520subfamilies%26jtitle%3DProtein%2520Eng.%26date%3D1996%26volume%3D9%26spage%3D1063%26epage%3D1065%26doi%3D10.1093%2Fprotein%2F9.11.1063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baker, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sept, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holst, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span> <span> </span><span class="NLM_article-title">Electrostatics of nanosystems: application to microtubules and the ribosome</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10037</span>– <span class="NLM_lpage">10041</span>, <span class="refDoi"> DOI: 10.1073/pnas.181342398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1073%2Fpnas.181342398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=11517324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFWisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=10037-10041&author=N.+A.+Bakerauthor=D.+Septauthor=S.+Josephauthor=M.+J.+Holstauthor=J.+A.+McCammon&title=Electrostatics+of+nanosystems%3A+application+to+microtubules+and+the+ribosome&doi=10.1073%2Fpnas.181342398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatics of nanosystems: application to microtubules and the ribosome</span></div><div class="casAuthors">Baker, Nathan A.; Sept, David; Joseph, Simpson; Holst, Michael J.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10037-10041</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Evaluation of the electrostatic properties of biomols. has become a std. practice in mol. biophysics.  Foremost among the models used to elucidate the electrostatic potential is the Poisson-Boltzmann equation; however, existing methods for solving this equation have limited the scope of accurate electrostatic calcns. to relatively small biomol. systems.  Here we present the application of numerical methods to enable the trivially parallel soln. of the Poisson-Boltzmann equation for supramol. structures that are orders of magnitude larger in size.  As a demonstration of this methodol., electrostatic potentials have been calcd. for large microtubule and ribosome structures.  The results point to the likely role of electrostatics in a variety of activities of these structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA_CAlhE0uX7Vg90H21EOLACvtfcHk0lir-QwUt1KXzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFWisbc%253D&md5=1b861999ef12c6972e82e8ada0f387cb</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1073%2Fpnas.181342398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.181342398%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DN.%2BA.%26aulast%3DSept%26aufirst%3DD.%26aulast%3DJoseph%26aufirst%3DS.%26aulast%3DHolst%26aufirst%3DM.%2BJ.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DElectrostatics%2520of%2520nanosystems%253A%2520application%2520to%2520microtubules%2520and%2520the%2520ribosome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D10037%26epage%3D10041%26doi%3D10.1073%2Fpnas.181342398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dolinsky, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, N. A.</span></span> <span> </span><span class="NLM_article-title">PDB2PQR: an automated pipeline for the setup of Poisson-Boltzmann electrostatics calculations</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">W665</span>– <span class="NLM_lpage">W667</span>, <span class="refDoi"> DOI: 10.1093/nar/gkh381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1093%2Fnar%2Fgkh381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=15215472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=W665-W667&author=T.+J.+Dolinskyauthor=J.+E.+Nielsenauthor=J.+A.+McCammonauthor=N.+A.+Baker&title=PDB2PQR%3A+an+automated+pipeline+for+the+setup+of+Poisson-Boltzmann+electrostatics+calculations&doi=10.1093%2Fnar%2Fgkh381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">PDB2PQR: An automated pipeline for the setup of Poisson-Boltzmann electrostatics calculations</span></div><div class="casAuthors">Dolinsky, Todd J.; Nielsen, Jens E.; McCammon, J. Andrew; Baker, Nathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W665-W667</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Continuum solvation models, such as Poisson-Boltzmann and Generalized Born methods, have become increasingly popular tools for investigating the influence of electrostatics on biomol. structure, energetics and dynamics.  However, the use of such methods requires accurate and complete structural data as well as force field parameters such as at. charges and radii.  Unfortunately, the limiting step in continuum electrostatics calcns. is often the addn. of missing at. coordinates to mol. structures from the Protein Data Bank and the assignment of parameters to biomol. structures.  To address this problem, we have developed the PDB2PQR web service (http://agave.wustl.edu/pdb2pqr/).  This server automates many of the common tasks of prepg. structures for continuum electrostatics calcns., including adding a limited no. of missing heavy atoms to biomol. structures, estg. titrn. states and protonating biomols. in a manner consistent with favorable hydrogen bonding, assigning charge and radius parameters from a variety of force fields, and finally generating "PQR" output compatible with several popular computational biol. packages.  This service is intended to facilitate the setup and execution of electrostatics calcns. for both experts and non-experts and thereby broaden the accessibility to the biol. community of continuum electrostatics analyses of biomol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-GebFAmzzkrVg90H21EOLACvtfcHk0lj38QtD4Pznpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsbY%253D&md5=678d2d58f5cb2c1dd862e2d71cf9dbe1</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh381%26sid%3Dliteratum%253Aachs%26aulast%3DDolinsky%26aufirst%3DT.%2BJ.%26aulast%3DNielsen%26aufirst%3DJ.%2BE.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26aulast%3DBaker%26aufirst%3DN.%2BA.%26atitle%3DPDB2PQR%253A%2520an%2520automated%2520pipeline%2520for%2520the%2520setup%2520of%2520Poisson-Boltzmann%2520electrostatics%2520calculations%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26spage%3DW665%26epage%3DW667%26doi%3D10.1093%2Fnar%2Fgkh381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Estrada, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernadó, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackledge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sancho, J.</span></span> <span> </span><span class="NLM_article-title">ProtSA: a web application for calculating sequence specific protein solvent accessibilities in the unfolded ensemble</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">104</span>, <span class="refDoi"> DOI: 10.1186/1471-2105-10-104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1186%2F1471-2105-10-104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=19356231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A280%3ADC%252BD1MzgsFWnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=104&author=J.+Estradaauthor=P.+Bernad%C3%B3author=M.+Blackledgeauthor=J.+Sancho&title=ProtSA%3A+a+web+application+for+calculating+sequence+specific+protein+solvent+accessibilities+in+the+unfolded+ensemble&doi=10.1186%2F1471-2105-10-104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">ProtSA: a web application for calculating sequence specific protein solvent accessibilities in the unfolded ensemble</span></div><div class="casAuthors">Estrada Jorge; Bernado Pau; Blackledge Martin; Sancho Javier</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The stability of proteins is governed by the heat capacity, enthalpy and entropy changes of folding, which are strongly correlated to the change in solvent accessible surface area experienced by the polypeptide.  While the surface exposed in the folded state can be easily determined, accessibilities for the unfolded state at the atomic level cannot be obtained experimentally and are typically estimated using simplistic models of the unfolded ensemble.  A web application providing realistic accessibilities of the unfolded ensemble of a given protein at the atomic level will prove useful.  RESULTS:  ProtSA, a web application that calculates sequence-specific solvent accessibilities of the unfolded state ensembles of proteins has been developed and made freely available to the scientific community.  The input is the amino acid sequence of the protein of interest.  ProtSA follows a previously published calculation protocol which uses the Flexible-Meccano algorithm to generate unfolded conformations representative of the unfolded ensemble of the protein, and uses the exact analytical software ALPHASURF to calculate atom solvent accessibilities, which are averaged on the ensemble.  CONCLUSION:  ProtSA is a novel tool for the researcher investigating protein folding energetics.  The sequence specific atom accessibilities provided by ProtSA will allow obtaining better estimates of the contribution of the hydrophobic effect to the free energy of folding, will help to refine existing parameterizations of protein folding energetics, and will be useful to understand the influence of point mutations on protein stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQdQW7AV3-BvhWfpMXQLRstfW6udTcc2ebveuFekcMLB7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzgsFWnsg%253D%253D&md5=52df84f04dacb028ec27924eaec7a286</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-10-104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-10-104%26sid%3Dliteratum%253Aachs%26aulast%3DEstrada%26aufirst%3DJ.%26aulast%3DBernad%25C3%25B3%26aufirst%3DP.%26aulast%3DBlackledge%26aufirst%3DM.%26aulast%3DSancho%26aufirst%3DJ.%26atitle%3DProtSA%253A%2520a%2520web%2520application%2520for%2520calculating%2520sequence%2520specific%2520protein%2520solvent%2520accessibilities%2520in%2520the%2520unfolded%2520ensemble%26jtitle%3DBMC%2520Bioinf.%26date%3D2009%26volume%3D10%26spage%3D104%26doi%3D10.1186%2F1471-2105-10-104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tina, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhadra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, N.</span></span> <span> </span><span class="NLM_article-title">PIC: Protein Interactions Calculator</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">W473</span>– <span class="NLM_lpage">W476</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=10.1093%2Fnar%2Fgkm423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=17584791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVOltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W473-W476&author=K.+G.+Tinaauthor=R.+Bhadraauthor=N.+Srinivasan&title=PIC%3A+Protein+Interactions+Calculator&doi=10.1093%2Fnar%2Fgkm423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">PIC: Protein Interactions Calculator</span></div><div class="casAuthors">Tina K G; Bhadra R; Srinivasan N</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W473-6</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Interactions within a protein structure and interactions between proteins in an assembly are essential considerations in understanding molecular basis of stability and functions of proteins and their complexes.  There are several weak and strong interactions that render stability to a protein structure or an assembly.  Protein Interactions Calculator (PIC) is a server which, given the coordinate set of 3D structure of a protein or an assembly, computes various interactions such as disulphide bonds, interactions between hydrophobic residues, ionic interactions, hydrogen bonds, aromatic-aromatic interactions, aromatic-sulphur interactions and cation-pi interactions within a protein or between proteins in a complex.  Interactions are calculated on the basis of standard, published criteria.  The identified interactions between residues can be visualized using a RasMol and Jmol interface.  The advantage with PIC server is the easy availability of inter-residue interaction calculations in a single site.  It also determines the accessible surface area and residue-depth, which is the distance of a residue from the surface of the protein.  User can also recognize specific kind of interactions, such as apolar-apolar residue interactions or ionic interactions, that are formed between buried or exposed residues or near the surface or deep inside.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKn5zO1OOtqL5XHngOXiV_fW6udTcc2ebveuFekcMLB7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVOltQ%253D%253D&md5=6c59648eeba47a6175754ed6d7499c14</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm423%26sid%3Dliteratum%253Aachs%26aulast%3DTina%26aufirst%3DK.%2BG.%26aulast%3DBhadra%26aufirst%3DR.%26aulast%3DSrinivasan%26aufirst%3DN.%26atitle%3DPIC%253A%2520Protein%2520Interactions%2520Calculator%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW473%26epage%3DW476%26doi%3D10.1093%2Fnar%2Fgkm423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CPL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CPL','PDB','2CPL'); return false;">PDB: 2CPL</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i28"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00777">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_16413"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00777?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00777</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Amino acid sequences, SMILES code formula and theoretical and experimental masses of AIF-derived peptides; <sup>1</sup>H and <sup>15</sup>N chemical shift assignments of AIF(381–389); <sup>13</sup>C chemical shift assignments of AIF(381–389) peptide; cysteine Cα and Cβ chemical shifts analysis of AIF(381–389) peptide; amide proton temperature coefficients (ΔδNH/Δ<i>T</i>, 298–310 K), Scalar couplings (3JH<sub>N</sub>Hα) and model free order parameters (S2) obtained for AIF(381–389) peptide in the free form; NMR structural statistics of the AIF(381–389) peptide; 2D [<sup>1</sup>H–<sup>1</sup>H] TOCSY (green) and 2D [<sup>1</sup>H–<sup>1</sup>H] ROESY (blue) spectra used for the identification of spin systems and the chemical shifts assignment of the AIF(381–389); chemical shift deviation from random coil values of HN and Hα backbone atoms plotted as a function of residue number; T2-filter <sup>1</sup>H–<sup>15</sup>N HSQC spectrum of AIF(381–389); secondary structure elements of CypA; flow chart of the NMR-based strategy used to identify the CypA residues involved in the recognition process of the AIF(381–389) peptide; CypA conformational ensemble generated by molecular dynamics simulations before and after cluster analysis; flow chart of the molecular docking protocol; <i>in silico</i> alanine scanning analysis of the AIF(381–389) peptide; and MS profiles of purified peptides (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf">PDF</a>)</p></li><li><p class="inline">NMR structure of AIF(381-389) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_002.pdb">PDB</a>)</p></li><li><p class="inline">Structure of the complex AIF(381-389)/CypA  (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_004.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_001.pdf">jm1c00777_si_001.pdf (1.37 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_002.pdb">jm1c00777_si_002.pdb (112.97 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_003.pdb">jm1c00777_si_003.pdb (130.6 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00777/suppl_file/jm1c00777_si_004.csv">jm1c00777_si_004.csv (5.42 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00777&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00777%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00777" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67998a83da9ad1b7","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
