
<html lang="en"     class="pb-page"  data-request-id="570c3d50-8a22-407a-b962-e650160274f8"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.6b00972;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2016.59.issue-18"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="1,2,4-Triazolyl 5-Azaspiro[2.4]heptanes: Lead Identification and Early Lead Optimization of a New Series of Potent and Selective Dopamine D3 Receptor Antagonists" /></meta><meta name="dc.Creator" content="Fabrizio  Micheli" /></meta><meta name="dc.Creator" content="Alessia  Bacchi" /></meta><meta name="dc.Creator" content="Simone  Braggio" /></meta><meta name="dc.Creator" content="Laura  Castelletti" /></meta><meta name="dc.Creator" content="Palmina  Cavallini" /></meta><meta name="dc.Creator" content="Paolo  Cavanni" /></meta><meta name="dc.Creator" content="Susanna  Cremonesi" /></meta><meta name="dc.Creator" content="Michele  Dal Cin" /></meta><meta name="dc.Creator" content="Aldo  Feriani" /></meta><meta name="dc.Creator" content="Sylvie  Gehanne" /></meta><meta name="dc.Creator" content="Mahmud  Kajbaf" /></meta><meta name="dc.Creator" content="Luciano  Marchió" /></meta><meta name="dc.Creator" content="Selena  Nola" /></meta><meta name="dc.Creator" content="Beatrice  Oliosi" /></meta><meta name="dc.Creator" content="Annalisa  Pellacani" /></meta><meta name="dc.Creator" content="Elisabetta  Perdonà" /></meta><meta name="dc.Creator" content="Anna  Sava" /></meta><meta name="dc.Creator" content="Teresa  Semeraro" /></meta><meta name="dc.Creator" content="Luca  Tarsi" /></meta><meta name="dc.Creator" content="Silvia  Tomelleri" /></meta><meta name="dc.Creator" content="Andrea  Wong" /></meta><meta name="dc.Creator" content="Filippo  Visentini" /></meta><meta name="dc.Creator" content="Laura  Zonzini" /></meta><meta name="dc.Creator" content="Christian  Heidbreder" /></meta><meta name="dc.Description" content="A novel series of 1,2,4-triazolyl 5-azaspiro[2.4]heptanes with high affinity and selectivity at the dopamine (DA) D3 receptor (D3R) is described. Some of these compounds also have high selectivity ..." /></meta><meta name="Description" content="A novel series of 1,2,4-triazolyl 5-azaspiro[2.4]heptanes with high affinity and selectivity at the dopamine (DA) D3 receptor (D3R) is described. Some of these compounds also have high selectivity ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 9, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b00972" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00972" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b00972" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00972" /></link>
        
    
    

<title>1,2,4-Triazolyl 5-Azaspiro[2.4]heptanes: Lead Identification and Early Lead Optimization of a New Series of Potent and Selective Dopamine D3 Receptor Antagonists | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b00972" /></meta><meta property="og:title" content="1,2,4-Triazolyl 5-Azaspiro[2.4]heptanes: Lead Identification and Early Lead Optimization of a New Series of Potent and Selective Dopamine D3 Receptor Antagonists" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0014.jpeg" /></meta><meta property="og:description" content="A novel series of 1,2,4-triazolyl 5-azaspiro[2.4]heptanes with high affinity and selectivity at the dopamine (DA) D3 receptor (D3R) is described. Some of these compounds also have high selectivity over the hERG channel and were characterized with respect to their pharmacokinetic properties both in vitro and in vivo during lead identification and early lead optimization phases. A few derivatives with overall favorable developability characteristics were selected for further late lead optimization studies." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b00972"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b00972">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b00972&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b00972&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b00972&amp;href=/doi/10.1021/acs.jmedchem.6b00972" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 18</span><span class="cit-fg-pageRange">, 8549-8576</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/18" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b00989" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b01009" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">1,2,4-Triazolyl 5-Azaspiro[2.4]heptanes: Lead Identification and Early Lead Optimization of a New Series of Potent and Selective Dopamine D3 Receptor Antagonists</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fabrizio++Micheli">Fabrizio Micheli</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alessia++Bacchi">Alessia Bacchi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Simone++Braggio">Simone Braggio</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Laura++Castelletti">Laura Castelletti</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Palmina++Cavallini">Palmina Cavallini</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paolo++Cavanni">Paolo Cavanni</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Susanna++Cremonesi">Susanna Cremonesi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michele++Dal+Cin">Michele Dal Cin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Aldo++Feriani">Aldo Feriani</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sylvie++Gehanne">Sylvie Gehanne</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mahmud++Kajbaf">Mahmud Kajbaf</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Luciano++Marchi%C3%B3">Luciano Marchió</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Selena++Nola">Selena Nola</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Beatrice++Oliosi">Beatrice Oliosi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Annalisa++Pellacani">Annalisa Pellacani</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elisabetta++Perdon%C3%A0">Elisabetta Perdonà</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anna++Sava">Anna Sava</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Teresa++Semeraro">Teresa Semeraro</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Luca++Tarsi">Luca Tarsi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Silvia++Tomelleri">Silvia Tomelleri</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Wong">Andrea Wong</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Filippo++Visentini">Filippo Visentini</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Laura++Zonzini">Laura Zonzini</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christian++Heidbreder">Christian Heidbreder</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Indivior Inc., The Fairfax Building, 10710 Midlothian Turnpike, Suite 430, Richmond Virginia 23235, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Dipartimento di Chimica, Università di Parma, Viale delle Scienze, 17/A, Biopharmanet-tec, Viale delle Scienze, 27/A, Campus, I-43124 Parma, Italy</span></div><div class="corresp-info"><strong>*</strong>Phone: +39-045-8218515. Fax: +39-045-8219111. E-mail: <a href="/cdn-cgi/l/email-protection#1076717262796a797f3e7d797378757c79507160646579643e737f7d"><span class="__cf_email__" data-cfemail="c0a6a1a2b2a9baa9afeeada9a3a8a5aca980a1b0b4b5a9b4eea3afad">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00972&amp;href=/doi/10.1021%2Facs.jmedchem.6b00972" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 18</span><span class="cit-pageRange">, 8549–8576</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 26, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 July 2016</li><li><span class="item_label"><b>Published</b> online</span>9 September 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 September 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b00972" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00972</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8549%26pageCount%3D28%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFabrizio%2BMicheli%252C%2BAlessia%2BBacchi%252C%2BSimone%2BBraggio%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D18%26contentID%3Dacs.jmedchem.6b00972%26title%3D1%252C2%252C4-Triazolyl%2B5-Azaspiro%255B2.4%255Dheptanes%253A%2BLead%2BIdentification%2Band%2BEarly%2BLead%2BOptimization%2Bof%2Ba%2BNew%2BSeries%2Bof%2BPotent%2Band%2BSelective%2BDopamine%2BD3%2BReceptor%2BAntagonists%26numPages%3D28%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8576%26publicationDate%3DSeptember%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b00972"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1270</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">11</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b00972" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;1,2,4-Triazolyl 5-Azaspiro[2.4]heptanes: Lead Identification and Early Lead Optimization of a New Series of Potent and Selective Dopamine D3 Receptor Antagonists&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Fabrizio&quot;,&quot;last_name&quot;:&quot;Micheli&quot;},{&quot;first_name&quot;:&quot;Alessia&quot;,&quot;last_name&quot;:&quot;Bacchi&quot;},{&quot;first_name&quot;:&quot;Simone&quot;,&quot;last_name&quot;:&quot;Braggio&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;Castelletti&quot;},{&quot;first_name&quot;:&quot;Palmina&quot;,&quot;last_name&quot;:&quot;Cavallini&quot;},{&quot;first_name&quot;:&quot;Paolo&quot;,&quot;last_name&quot;:&quot;Cavanni&quot;},{&quot;first_name&quot;:&quot;Susanna&quot;,&quot;last_name&quot;:&quot;Cremonesi&quot;},{&quot;first_name&quot;:&quot;Michele&quot;,&quot;last_name&quot;:&quot;Dal Cin&quot;},{&quot;first_name&quot;:&quot;Aldo&quot;,&quot;last_name&quot;:&quot;Feriani&quot;},{&quot;first_name&quot;:&quot;Sylvie&quot;,&quot;last_name&quot;:&quot;Gehanne&quot;},{&quot;first_name&quot;:&quot;Mahmud&quot;,&quot;last_name&quot;:&quot;Kajbaf&quot;},{&quot;first_name&quot;:&quot;Luciano&quot;,&quot;last_name&quot;:&quot;Marchió&quot;},{&quot;first_name&quot;:&quot;Selena&quot;,&quot;last_name&quot;:&quot;Nola&quot;},{&quot;first_name&quot;:&quot;Beatrice&quot;,&quot;last_name&quot;:&quot;Oliosi&quot;},{&quot;first_name&quot;:&quot;Annalisa&quot;,&quot;last_name&quot;:&quot;Pellacani&quot;},{&quot;first_name&quot;:&quot;Elisabetta&quot;,&quot;last_name&quot;:&quot;Perdonà&quot;},{&quot;first_name&quot;:&quot;Anna&quot;,&quot;last_name&quot;:&quot;Sava&quot;},{&quot;first_name&quot;:&quot;Teresa&quot;,&quot;last_name&quot;:&quot;Semeraro&quot;},{&quot;first_name&quot;:&quot;Luca&quot;,&quot;last_name&quot;:&quot;Tarsi&quot;},{&quot;first_name&quot;:&quot;Silvia&quot;,&quot;last_name&quot;:&quot;Tomelleri&quot;},{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Wong&quot;},{&quot;first_name&quot;:&quot;Filippo&quot;,&quot;last_name&quot;:&quot;Visentini&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;Zonzini&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;Heidbreder&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;09&quot;,&quot;issue&quot;:&quot;18&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;8549-8576&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b00972&quot;},&quot;abstract&quot;:&quot;A novel series of 1,2,4-triazolyl 5-azaspiro[2.4]heptanes with high affinity and selectivity at the dopamine (DA) D3 receptor (D3R) is described. Some of these compounds also have high selectivity over the hERG channel and were characterized with respect to their pharmacokinetic properties both in vitro and in vivo during lead identification and early lead optimization phases. A few derivatives with overall favorable developability characteristics were selected for further late lead optimization studies.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00972&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00972" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00972&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00972" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00972&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00972" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00972&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00972&amp;href=/doi/10.1021/acs.jmedchem.6b00972" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b00972" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b00972" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b00972%26sid%3Dliteratum%253Aachs%26pmid%3D27564135%26genre%3Darticle%26aulast%3DMicheli%26date%3D2016%26atitle%3D1%252C2%252C4-Triazolyl%2B5-Azaspiro%255B2.4%255Dheptanes%253A%2BLead%2BIdentification%2Band%2BEarly%2BLead%2BOptimization%2Bof%2Ba%2BNew%2BSeries%2Bof%2BPotent%2Band%2BSelective%2BDopamine%2BD3%2BReceptor%2BAntagonists%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D18%26spage%3D8549%26epage%3D8576%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292588" title="Partition coefficient">Partition coefficient</a>,</li><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=292444" title="Molecular structure">Molecular structure</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/18" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/jmcmar.2016.59.issue-18/20160922/jmcmar.2016.59.issue-18.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00972&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A novel series of 1,2,4-triazolyl 5-azaspiro[2.4]heptanes with high affinity and selectivity at the dopamine (DA) D3 receptor (D3R) is described. Some of these compounds also have high selectivity over the hERG channel and were characterized with respect to their pharmacokinetic properties both in vitro and in vivo during lead identification and early lead optimization phases. A few derivatives with overall favorable developability characteristics were selected for further late lead optimization studies.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07583" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07583" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Long-lasting adaptations in the brains of individuals addicted to drugs support the concept of addiction as a chronically relapsing disorder that is characterized by the compulsion to seek and take the drug, the loss of control in limiting drug intake, and the emergence of a negative emotional state when access to the drug is withdrawn. The inherent complexity of these neuroadaptations makes the therapeutic management of substance use disorder (SUD) a major challenge for the discovery and development of safe and efficacious pharmacotherapeutic strategies ensuring satisfactory patient compliance and promoting drug abstinence and recovery.</div><div class="NLM_p">Half a century of research has pointed toward activation of the mesolimbic dopamine (DA) system as a common denominator in response to virtually all drugs of abuse that are voluntarily self-administered by laboratory animals and humans. It is thus quite natural that DA receptors were considered as reasonable targets for the development of SUD medications. Of particular interest is the restricted high-density expression of the DA D3 receptor (D3R) subtype into key elements of the mesolimbic DA system such as the ventral striatum, midbrain, and pallidum, which have all been associated with behaviors controlled by the presentation of drug-associated cues.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> The strategic localization of the D3R together with changes in its expression following acute, subchronic, or chronic exposure to drugs of abuse have triggered research efforts leading to the discovery, synthesis, and characterization of highly potent and selective D3R antagonists.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1-4)</a> Nonclinical evidence clearly supports the contention that selective D3R antagonists show promise for the treatment of SUD as reflected by their potential to (1) regulate the motivation to self-administer drugs under schedules of reinforcement that require an increase in work demand and (2) disrupt the responsiveness to drug-associated stimuli that play a key role in reinstatement of drug-seeking behavior triggered by re-exposure to the drug itself, re-exposure to environmental cues that had been previously associated with drug-taking behavior, or stress.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a></div><div class="NLM_p">GlaxoSmithKline (GSK) reported compounds<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5, 6)</a> with high in vitro and in vivo selectivity at the D3R, which can attenuate or block the effects of drugs of abuse in a wide range of animal models of addiction. Additional compounds were also disclosed by the pharmaceutical industry and academic centers.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_p">The scope of the present work was to identify potential replacements of the “amino portion” of the above-mentioned scaffolds. Two attempts of “amino portion” substitution were recently disclosed<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> through a mixed computational and medicinal chemistry strategy that identified both morpholines and octahydropyrrolo[2,3-<i>b</i>]pyrroles.</div><div class="NLM_p last">Using a similar approach, the present work describes a new series of 5-azaspiro[2.4]heptanes with a detailed structure–activity relationship (SAR) in lead identification and early lead optimization phases; a detailed in vitro and in vivo pharmacokinetic (PK) strategy to selected compounds for the subsequent late lead optimization phase is also presented.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63130" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63130" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Since the discovery of the D3R in 1990, both industrial and academic researchers have been trying to identify novel chemical entities (NCEs) that selectively modulate the D3R.<a onclick="showRef(event, 'ref4 ref7'); return false;" href="javascript:void(0);" class="ref ref4 ref7">(4, 7)</a> Among the most characterized derivatives, both in vitro and in vivo, SB-277011, <b>1</b>,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> represents one of the prototypical selective D3R antagonists. In 2010, GSK presented a full characterization of GSK598809A, <b>2</b>,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> which was progressed to clinical study in humans.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Examples of structurally unrelated templates are PG01037, <b>3</b>,<a onclick="showRef(event, 'cit12a'); return false;" href="javascript:void(0);" class="ref cit12a">(12a)</a> the tranylcypromine substituted <i>cis</i>-hydroxycyclobutylnaphthamides (CJ-1882, <b>4</b>)<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and YQA14 (<b>5</b>),<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> all shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>. The octahydropyrrolo[2,3-<i>b</i>]pyrrole derivatives series<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> (<b>6</b>) is also reported in the same figure.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of known and new D3R antagonists.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00972&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">All these structures can, at least theoretically, fit the general “pharmacophore model” for D3R antagonists originally described by Hackling and Stark.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>, <b>1</b> was pictorially positioned along such a model, which requires an aryl region, an H-bond acceptor, a spacer, the “amino” region, and another aryl moiety.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Schematic model of the D3R pharmacophore described in the early 2000 aligned with derivative <b>1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00972&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The tridimensional representation of this theoretical space in a D3R model clearly highlights the medicinal chemistry challenges associated with the identification of an appropriate “amino portion”. Such a scaffold must be compliant with specific criteria including appropriate basicity to interact with Asp<sup>3.32</sup> in the orthosteric binding site,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> contribution to the overall selectivity of the molecule vs the DA D2 receptor (D2R), and novelty from an intellectual property perspective. Last, but not least, new molecules must have the appropriate “developability”<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> profile to be progressed toward clinical trials.</div><div class="NLM_p">Mixed approaches based on the D3R crystal structure<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> were attempted for the identification of the previously mentioned NCEs.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> As described more than 10 years ago,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> “scaffold hopping” is also used by medicinal chemists to identify structurally novel molecules starting from known active compounds and modifying their central core structure.</div><div class="NLM_p">This approach can lead to completely different core structures that can then be characterized through screening cascades. In this article, our efforts led to the identification of the 5-azaspiro[2.4]heptane moiety as the novel “amino portion”.</div><div class="NLM_p">On the basis of our previous experience,<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> the new molecules identified in this exploratory process were initially characterized for their binding activity at the human D3R and D2R, while a few selected derivatives were also evaluated in functional assays to determine their agonist and antagonist properties at the same receptors. The aim of this approach was to identify potent D3R antagonists (p<i>K</i><sub>i</sub> ≥ 8), endowed with high selectivity (ideally ≥100-fold) vs the D2R, high (≥100-fold or hERG fp<i>K</i><sub>i</sub> < 6.0) selectivity vs the hERG ion channel (for preventing potential cardiovascular liability) as well as <i>a</i> ≥ 50-fold selectivity vs muscarinic M1–3 receptors (based on previous liability shown by some D3R antagonists reported in literature<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a>). Exceptions to these general rules were applied in case of need of additional SAR information both in vitro and in vivo.</div><div class="NLM_p">To ensure appropriate developability, all NCEs went through in vitro pharmacokinetic (PK) assays such as CYPEX bactosome P450 inhibition and rat and human in vitro clearance (rCl<sub>i</sub> and hCl<sub>i</sub>, respectively) in liver microsomes early in the screening cascade.</div><div class="NLM_p">The synthesis of the new scaffolds is described in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, while all the new compounds identified (<b>7</b>–<b>85a</b>) are reported in <a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a>–<a class="ref showTableEvent internalNav" data-ID="tbl11" href="#tbl11">11</a>. Compound <b>1</b> was used as internal standard for biological assays, while literature data<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> were reported for <b>2</b>.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0006.jpeg" id="GRAPHIC-d126e531-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Scheme<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00972&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(i) MnO<sub>2</sub>, dioxane, RT; (ii) 1M LiAlH<sub>4</sub> THF, reflux; (iii) N<sub>2</sub>, NH<sub>4</sub>COOH, Pd/C, MeOH, reflux; (iv) T3P in AcOEt, 4-methyl-3-thiosemicarbazide, NaOH; (v) K<sub>2</sub>CO<sub>3</sub>/acetone/MeOH, RT, alkylating agent; (vi) Na<sub>2</sub>CO<sub>3</sub>, NaI, DMF or CH<sub>3</sub>CN, 60 °C.</p></p></figure><div class="NLM_p">Because the 5-azaspiro[2.4]heptane template contains two stereogenic centers, the relative stereochemistry of the compounds can lead to the isolation of both “<i>cis</i>” and “<i>trans</i>” isomers. Early in the process, the racemate was submitted to testing and promising results resulted in each single enantiomer being tested after chromatographic separation. Later on in the exploration, only specific isomers were tested; the X-rays of a few key intermediates were obtained as well as described later on.</div><div class="NLM_p">The exploration of the newly designed “amino portion” started with an unsubstituted phenyl ring (see <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> for derivatives <b>7</b> and <b>8</b>). Derivative <b>8b</b>, which showed nanomolar affinity at the D3R, had about 140-fold selectivity vs the D2R and more than 500-fold selectivity vs the hERG channel. This compound was further profiled and proved to be fully inactive as an agonist at the D3R; it behaved as an antagonist (fp<i>K</i><sub>i</sub> = 8.18 ± 0.09) with 100-fold selectivity vs the D2R (fp<i>K</i><sub>i</sub> = 6.17 ± 0.16) when tested in a functional DA D2 assay (see <a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a> for a comparison between binding and functional data). In addition, derivative <b>8b</b> was completely inactive (fp<i>K</i><sub>i</sub> < 4.0) at the DA D1 receptor (D1R) and DA D4 receptor (D4R), whereas 100-fold selectivity was also observed at the M1 and M3 receptors (fp<i>K</i><sub>i</sub> = 6.13 ± 0.11 and 5.69 ± 0.25, respectively), suggesting that the new scaffold could be a promising starting point for exploration. Considering its in vitro PK properties, compound <b>8b</b> showed IC<sub>50</sub> values greater than 50 μM on all CYP P450 isoforms (namely CYP1A2, CYP2C9, CYP2C19, CYP3A4, DBOMF, and CYP3A4 7BQ) tested with the exception of CYP2D6 (IC<sub>50</sub> = 0.13 μM).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Affinity Data at the D3R and D2R and Potency Data at the hERG Ion Channel for Derivatives <b>1</b>, <b>2</b>, and <b>7</b>–<b>26</b><a class="ref internalNav" href="#t1fn2" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0007.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">hD3 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center">hD2 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center">hERG pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center">n, R1</th><th class="colsep0 rowsep0" align="center" char=".">cLogP</th><th class="colsep0 rowsep0" align="center" char=".">PSA</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="center">8.19 ± 0.02</td><td class="colsep0 rowsep0" align="center">6.31 ± 0.01</td><td class="colsep0 rowsep0" align="center">6.17 ± 0.03</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="char" char=".">69.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="center">8.9<a class="ref internalNav" href="#t1fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="center">6.2<a class="ref internalNav" href="#t1fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="center">6.1<a class="ref internalNav" href="#t1fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">H (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="center">7.60 ± 0.21</td><td class="colsep0 rowsep0" align="center">6.00 ± 0.17</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7a</b></td><td class="colsep0 rowsep0" align="left">H (<i>trans</i>, se 1)</td><td class="colsep0 rowsep0" align="center">7.55 ± 0.31</td><td class="colsep0 rowsep0" align="center">6.29 ± 0.11</td><td class="colsep0 rowsep0" align="center">5.40 ± 0.13</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="left">H (<i>trans</i>, se 2)</td><td class="colsep0 rowsep0" align="center">7.38 ± 0.20</td><td class="colsep0 rowsep0" align="center">6.15 ± 0.18</td><td class="colsep0 rowsep0" align="center">5.44 ± 0.06</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">H (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="center">7.75 ± 0.08</td><td class="colsep0 rowsep0" align="center">6.03 ± 0.07</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8a</b></td><td class="colsep0 rowsep0" align="left">H (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="center">6.10 ± 0.28</td><td class="colsep0 rowsep0" align="center">4.77 ± 0.18</td><td class="colsep0 rowsep0" align="center">5.33 ± 0.20</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8b</b></td><td class="colsep0 rowsep0" align="left">H (<i>cis</i>, se 2)</td><td class="colsep0 rowsep0" align="center">8.26 ± 0.20</td><td class="colsep0 rowsep0" align="center">6.10 ± 0.11</td><td class="colsep0 rowsep0" align="center">5.54 ± 0.02</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub> (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="center">8.04 ± 0.10</td><td class="colsep0 rowsep0" align="center">6.18 ± 0.01</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9a</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub> (<i>trans</i>, se 1)</td><td class="colsep0 rowsep0" align="center">8.40 ± 0.03</td><td class="colsep0 rowsep0" align="center">6.05 ± 0.08</td><td class="colsep0 rowsep0" align="center">6.72 ± 0.13</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9b</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub> (<i>trans</i>, se 2)</td><td class="colsep0 rowsep0" align="center">7.91 ± 0.15</td><td class="colsep0 rowsep0" align="center">6.20 ± 0.11</td><td class="colsep0 rowsep0" align="center">6.43 ± 0.04</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub> (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="center">8.78 ± 0.18</td><td class="colsep0 rowsep0" align="center">6.53 ± 0.05</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10a</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub> (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="center">9.20 ± 0.12</td><td class="colsep0 rowsep0" align="center">6.67 ± 0.04</td><td class="colsep0 rowsep0" align="center">6.39 ± 0.09</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10b</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub> (<i>cis</i>, se 2)</td><td class="colsep0 rowsep0" align="center">6.89 ± 0.20</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub> (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="center">7.56 ± 0.11</td><td class="colsep0 rowsep0" align="center">6.29 ± 0.09</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">0</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11a</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub> (<i>trans</i>, se 1)</td><td class="colsep0 rowsep0" align="center">7.11 ± 0.13</td><td class="colsep0 rowsep0" align="center">6.01 ± 0.10</td><td class="colsep0 rowsep0" align="center">6.50 ± 0.18</td><td class="colsep0 rowsep0" align="center">0</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11b</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub> (<i>trans</i>, se 2)</td><td class="colsep0 rowsep0" align="center">7.65 ± 0.05</td><td class="colsep0 rowsep0" align="center">6.10 ± 0.07</td><td class="colsep0 rowsep0" align="center">6.87 ± 0.06</td><td class="colsep0 rowsep0" align="center">0</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub> (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="center">7.02 ± 0.20</td><td class="colsep0 rowsep0" align="center">6.28 ± 0.04</td><td class="colsep0 rowsep0" align="center">6.28 ± 0.08</td><td class="colsep0 rowsep0" align="center">0</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub> (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="center">7.34 ± 0.04</td><td class="colsep0 rowsep0" align="center">6.01 ± 0.05</td><td class="colsep0 rowsep0" align="center">6.25 ± 0.06</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="char" char=".">3.9</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">2-CF<sub>3</sub> (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="center">8.82 ± 0.16</td><td class="colsep0 rowsep0" align="center">6.91 ± 0.14</td><td class="colsep0 rowsep0" align="center">5.71 ± 0.02</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14a</b></td><td class="colsep0 rowsep0" align="left">2-CF<sub>3</sub> (<i>trans</i>, se 1)</td><td class="colsep0 rowsep0" align="center">7.05 ± 0.15</td><td class="colsep0 rowsep0" align="center">6.14 ± 0.10</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14b</b></td><td class="colsep0 rowsep0" align="left">2-CF<sub>3</sub> (<i>trans</i>, se 2)</td><td class="colsep0 rowsep0" align="center">9.26 ± 0.05</td><td class="colsep0 rowsep0" align="center">7.20 ± 0.18</td><td class="colsep0 rowsep0" align="center">6.05 ± 0.01</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">2-CF<sub>3</sub> (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="center">7.58 ± 0.20</td><td class="colsep0 rowsep0" align="center">5.80 ± 0.07</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">4-F (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="center">7.60 ± 0.03</td><td class="colsep0 rowsep0" align="center">6.01 ± 0.04</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">60.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16a</b></td><td class="colsep0 rowsep0" align="left">4-F (<i>trans</i>, se 1)</td><td class="colsep0 rowsep0" align="center">7.38 ± 0.07</td><td class="colsep0 rowsep0" align="center">5.96 ± 0.01</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">60.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16b</b></td><td class="colsep0 rowsep0" align="left">4-F (<i>trans</i>, se 2)</td><td class="colsep0 rowsep0" align="center">7.54 ± 0.05</td><td class="colsep0 rowsep0" align="center">6.10 ± 0.02</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">60.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">4-F (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="center">8.49 ± 0.03</td><td class="colsep0 rowsep0" align="center">6.37 ± 0.08</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">60.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17a</b></td><td class="colsep0 rowsep0" align="left">4-F (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="center">6.54 ± 0.09</td><td class="colsep0 rowsep0" align="center">5.09 ± 0.18</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">60.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17b</b></td><td class="colsep0 rowsep0" align="left">4-F (<i>cis</i>, se 2)</td><td class="colsep0 rowsep0" align="center">9.04 ± 0.12</td><td class="colsep0 rowsep0" align="center">6.62 ± 0.01</td><td class="colsep0 rowsep0" align="center">6.22 ± 0.06</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">60.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">2,4-F (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="center">7.60 ± 0.12</td><td class="colsep0 rowsep0" align="center">6.27 ± 0.06</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">60.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18a</b></td><td class="colsep0 rowsep0" align="left">2,4-F (<i>trans</i>, se 1)</td><td class="colsep0 rowsep0" align="center">7.46 ± 0.08</td><td class="colsep0 rowsep0" align="center">6.17 ± 0.01</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">60.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18b</b></td><td class="colsep0 rowsep0" align="left">2,4-F (<i>trans</i>, se 2)</td><td class="colsep0 rowsep0" align="center">7.70 ± 0.12</td><td class="colsep0 rowsep0" align="center">6.02 ± 0.05</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">60.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">2,4-F (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="center">8.70 ± 0.01</td><td class="colsep0 rowsep0" align="center">6.53 ± 0.03</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">60.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19a</b></td><td class="colsep0 rowsep0" align="left">2,4-F (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="center">6.62 ± 0.01</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">60.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19b</b></td><td class="colsep0 rowsep0" align="left">2,4-F (<i>cis</i>, se 2)</td><td class="colsep0 rowsep0" align="center">9.09 ± 0.01</td><td class="colsep0 rowsep0" align="center">6.59 ± 0.01</td><td class="colsep0 rowsep0" align="center">6.12 ± 0.10</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">60.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">2-F, 4-CF<sub>3</sub> (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="center">8.03 ± 0.10</td><td class="colsep0 rowsep0" align="center">6.50 ± 0.07</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20a</b></td><td class="colsep0 rowsep0" align="left">2-F, 4-CF<sub>3</sub> (<i>trans</i>, se 1)</td><td class="colsep0 rowsep0" align="center">8.49 ± 0.15</td><td class="colsep0 rowsep0" align="center">6.51 ± 0.03</td><td class="colsep0 rowsep0" align="center">6.60 ± 0.18</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20b</b></td><td class="colsep0 rowsep0" align="left">2-F, 4-CF<sub>3</sub> (<i>trans</i>, se 2)</td><td class="colsep0 rowsep0" align="center">8.21 ± 0.22</td><td class="colsep0 rowsep0" align="center">6.01 ± 0.04</td><td class="colsep0 rowsep0" align="center">6.35 ± 0.14</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">2-F, 4-CF<sub>3</sub> (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="center">8.98 ± 0.22</td><td class="colsep0 rowsep0" align="center">6.48 ± 0.01</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21a</b></td><td class="colsep0 rowsep0" align="left">2-F, 4-CF<sub>3</sub> (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="center">7.14 ± 0.19</td><td class="colsep0 rowsep0" align="center">5.36 ± 0.02</td><td class="colsep0 rowsep0" align="center">6.16 ± 0.11</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21b</b></td><td class="colsep0 rowsep0" align="left">2-F, 4-CF<sub>3</sub> (<i>cis</i>, se 2)</td><td class="colsep0 rowsep0" align="center">9.38 ± 0.09</td><td class="colsep0 rowsep0" align="center">6.65 ± 0.04</td><td class="colsep0 rowsep0" align="center">6.28 ± 0.02</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22a</b></td><td class="colsep0 rowsep0" align="left">4-F, 2-CF<sub>3</sub> (<i>trans</i>, se 1)</td><td class="colsep0 rowsep0" align="center">8.98 ± 0.13</td><td class="colsep0 rowsep0" align="center">6.78 ± 0.02</td><td class="colsep0 rowsep0" align="center">5.92 ± 0.19</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22b</b></td><td class="colsep0 rowsep0" align="left">4-F, 2-CF<sub>3</sub> (<i>trans</i>, se 2)</td><td class="colsep0 rowsep0" align="center">7.12 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.66 ± 0.08</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23a</b></td><td class="colsep0 rowsep0" align="left">4-F, 2-CF<sub>3</sub> (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="center">7.05 ± 0.08</td><td class="colsep0 rowsep0" align="center">5.42 ± 0.01</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23b</b></td><td class="colsep0 rowsep0" align="left">4-F, 2-CF<sub>3</sub> (<i>cis</i>, se 2)</td><td class="colsep0 rowsep0" align="center">8.17 ± 0.05</td><td class="colsep0 rowsep0" align="center">5.89 ± 0.03</td><td class="colsep0 rowsep0" align="center">5.57 ± 0.24</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub> (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="center">7.70 ± 0.12</td><td class="colsep0 rowsep0" align="center">5.76 ± 0.01</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">1, OH</td><td class="colsep0 rowsep0" align="char" char=".">2.6</td><td class="colsep0 rowsep0" align="char" char=".">105.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">3,5-Cl (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="center">9.38 ± 0.12</td><td class="colsep0 rowsep0" align="center">7.18 ± 0.08</td><td class="colsep0 rowsep0" align="center">>7.0</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">3,5-Cl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="center">9.01 ± 0.02</td><td class="colsep0 rowsep0" align="center">7.21 ± 0.08</td><td class="colsep0 rowsep0" align="center">6.58 ± 0.11</td><td class="colsep0 rowsep0" align="center">1, H</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn2"><div class="footnote" id="t1fn2"><sup>a</sup><p class="last">rac = racemate; se 1 = single enantiomer 1; se 2 = single enantiomer 2; NT = not tested; NA = not applicable; values shown are the average of three replicates and standard deviation.</p></div><div class="footnote" id="t1fn1"><sup>b</sup><p class="last">Literature data as reported in ref <a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">5</a>. DA D3 and DA D2 values are fp<i>K</i><sub>i</sub>s.</p></div></div></div><div class="NLM_p">The introduction of a −CF<sub>3</sub> moiety on the <i>para</i> position of the phenyl ring (<b>9</b>, <b>10</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) of the “amino portion” increased the lipophilicity (cLogP)<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> of the overall molecule of about one unit. This increase led to an improvement in the affinity at the D3R of about one log unit on the best enantiomers both in the <i>trans</i> (<b>9a</b>) and in the <i>cis</i> (<b>10a</b>) configurations of the 5-azaspiro[2.4]heptane moiety. The difference in affinity between the two enantiomers was greater for the <i>cis</i> vs the <i>trans</i> isomer. The substitution had no effect on the functional activity of the compounds, which continued to behave as full antagonists (<a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>). No modification on the muscarinic profile of compound <b>10a</b> was observed (fp<i>K</i><sub>i</sub> = 6.51 ± 0.04 and 6.56 ± 0.11, respectively, for M1 and M3 receptors, <a class="ref internalNav" href="#tbl11" aria-label="Table 11">Table 11</a>), maintaining the selectivity at about 400-fold over these receptors. Selectivity vs the D2R was increased by about 350-fold, while selectivity vs the hERG channel was greater than 600-fold, even if the affinity for the channel was also slightly increased to 0.4 μM.</div><div class="NLM_p">Both compounds were also tested for in vitro PK properties (<a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>). Compound <b>9a</b> showed IC<sub>50</sub> values greater than 17 μM on all CYP P450 isoforms tested with the exception of CYP2D6 (IC<sub>50</sub> = 1.3 μM), while compound <b>10a</b> showed IC<sub>50</sub> values greater than 17 μM on all CYP P450 isoforms tested with the exception of CYP2D6 and CYP3A4 7BQ (IC<sub>50</sub> = 1.3 μM and 0.30 μM, respectively). Derivative <b>9a</b> showed relatively low human intrinsic clearance (hCl<sub>i</sub>) on microsomes (22 μL/min/mg protein) and high clearance in rat (rCl<sub>i</sub>) microsomes (161 μL/min/mg protein). However, derivative <b>10a</b> showed relatively high Cl<sub>i</sub> in both species (hCl<sub>i</sub> = 86 and rCl<sub>i</sub> = 83 μL/min/mg protein, respectively). The last in vitro parameter to be considered (based on the need for brain penetration)<a onclick="showRef(event, 'cit17e'); return false;" href="javascript:void(0);" class="ref cit17e">(17e)</a> was the blood and tissue (rodent) protein binding. Both derivatives had excellent fraction unbound (Fu) both in brain (Fu<sub>br</sub>) and blood (Fu<sub>bl</sub>). For <b>9a</b> the Fu<sub>br</sub> was 4.8%, while its Fu<sub>bl</sub> was 12%. Derivative <b>10a</b> showed Fu<sub>br</sub> equal to 9.5% and Fu<sub>bl</sub> equal to 10.2%. Altogether, these data suggested that these molecules are endowed with in vitro properties that are suitable for further optimization. Selected PK data are reported in <a class="ref internalNav" href="#tbl9" aria-label="Tables 9">Tables 9</a> and <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a>.</div><div class="NLM_p">In agreement with previously reported procedures,<a onclick="showRef(event, 'ref9 ref16'); return false;" href="javascript:void(0);" class="ref ref9 ref16">(9, 16)</a> derivative <b>10a</b> was fitted in a D3R model based on X-ray crystal structure using MOE.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> As clearly shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>, derivative <b>10a</b> (red) can adopt similar conformations to compound <b>2</b>. One may appreciate, with the obvious limitation of reproducing in two dimensions a 3D interaction, that the basic nitrogen of the new “amino portion” can interact with Asp<sup>3,32</sup>. Both derivative <b>10a</b> and derivative <b>2</b> (in yellow) formed a salt bridge with the aspartic acid in the receptor. Furthermore, the aromatic portion (the oxazole in both compounds) clearly pointed toward the extracellular loop (EL) of the receptor, in agreement with literature data.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Finally, the new template was able to direct the −CF<sub>3</sub> moiety of the phenyl ring of derivative <b>10a</b> in a similar hydrophobic region common with compound <b>2</b>.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Relevant interactions of derivative <b>2</b> (yellow) and <b>10a</b> (red) in docking to the D3R model. Portions of the backbone ribbon have been removed for clarity. The interaction of the basic nitrogen of the amine portion of the two compounds with Asp<sup>3,32</sup> is clearly visible. The region of the thiotriazole and of the oxazole portion clearly points toward the EL. The new 5-azaspiro[2.4]heptanes seem to direct the −CF<sub>3</sub> moiety of derivative <b>10a</b> in a similar hydrophobic region in common with compound <b>2</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00972&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Keeping constant the <i>p</i>-CF<sub>3</sub> substitution on the phenyl ring and shortening the linker chain (from C3 to C2, derivatives <b>11</b> and <b>12</b>) led to a reduction in affinity at the D3R; an increase in length (from C3 to C4) also had a detrimental effect on the affinity at the D3R (<b>13</b>). The change in the linker length (<b>12</b>, <b>13</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) did not modify the functional efficacy of the compounds as reported in <a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>.</div><div class="NLM_p">To verify whether the rotation of the phenyl group of the “amino portion”, i.e., whether a change in the amount of that dihedral angle had any effect on the affinity of the compounds both at the D3R receptor and at the hERG channel, the −CF<sub>3</sub> substituent was added in <i>ortho</i> position (without changing the cLogP, <b>14</b>, <b>15</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Although the impact on affinity was not obvious for the <i>trans</i> isomer, basically maintaining nanomolar affinity at the D3R, a more pronounced effect was observed with the <i>cis</i> isomer with a reduction in affinity of about 10-fold at the D3R. The affinity of the best compound at the hERG channel was not affected.</div><div class="NLM_p">To verify if the cLogP itself had a role in driving the affinity at the D3R, the −CF<sub>3</sub> moiety was replaced by a single −F atom (<b>16</b>, <b>17</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>; cLogP lowered from 3.4 to 2.7). Comparing the most potent compound in each series (i.e., <b>10a</b> vs <b>17b</b>), no major difference was observed in terms of affinity and selectivity vs the D2R and hERG targets. Furthermore, the functional profile (full antagonism, <a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>) remained unchanged. The in vitro PK profiling (<a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>) for <b>17b</b> showed IC<sub>50</sub> values greater than 17 μM on all CYP P450 isoforms tested with the exception of CYP2D6 and CYP3A4 7BQ (IC<sub>50</sub> = 0.7 μM and 3.4 μM, respectively). The compound showed relatively low hCl<sub>i</sub> on microsomes (37 μL/min/mg protein) and moderate rCl<sub>i</sub> (73 μL/min/mg protein). Finally, both Fu<sub>br</sub> and Fu<sub>bl</sub> were exceptionally high, with values of 27% and 38%, respectively, supporting a promising profile to interact with the D3R in the brain.</div><div class="NLM_p">The introduction of a second −F atom (<b>18</b>, <b>19</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) led to a slight change in the overall cLogP but had no major impact on the affinity at the D3R and the selectivity profile of the best enantiomer (<b>19b</b>). Furthermore, the functional profile remained unchanged as reported in <a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>.</div><div class="NLM_p">The in vitro PK profile (<a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>) of <b>19b</b> revealed IC<sub>50</sub> values greater than 17 μM on all CYP P450 isoforms tested with the exception of CYP2D6 (IC<sub>50</sub> = 1.2 μM). Although hCl<sub>i</sub> on microsomes was relatively high (53 μL/min/mg protein), rCl<sub>i</sub> was definitely higher (161 μL/min/mg protein). Both Fu<sub>br</sub> and Fu<sub>bl</sub> were high with values of 28% and 33%, respectively, suggesting again a promising profile to interact with the D3R in the brain.</div><div class="NLM_p">A slight increase in cLogP with respect to compound <b>10a</b> was achieved with the introduction of a single −F on the already −CF<sub>3</sub> substituted phenyl ring (cLogP from 3.4 to 3.6 in derivatives <b>20</b>–<b>23</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). The introduction of the −F in position 2, keeping the −CF<sub>3</sub> in position 4 of the phenyl ring, did not change significantly the affinity of the best enantiomer (<b>21b</b>) at the D3R and at the hERG channel, with still more than 500-fold selectivity vs the D2R while increasing selectivity vs the hERG channel to more than 1250-fold. The <i>cis</i> enantiomer showed greater affinity at the D3R compared to the <i>trans</i> enantiomer. As reported in <a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>, functional antagonism was confirmed for compounds <b>20b</b> and <b>21b</b>. A slightly lower selectivity window with high D2R functional potency was observed for <b>21b</b> (<a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>). With regards to in vitro PK properties (<a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>), <b>20b</b> showed an improved profile at CYP2D6 (IC<sub>50</sub> = 5.7 μM), with IC<sub>50</sub> values greater than 17 μM on all CYP P450 isoforms tested with the exception of CYP1A2 (IC<sub>50</sub>= 0.8 μM). Compound <b>21b</b> showed IC<sub>50</sub> values greater than 17 μM on all CYP P450 isoforms tested with the exception of CYP2D6 (IC<sub>50</sub> = 0.6 μM). Considering a marked difference in the Cl<sub>i</sub> of the two derivatives, with <b>20b</b> showing low hCl<sub>i</sub> and rCl<sub>i</sub> (20 and 23 μL/min/mg protein, respectively) and <b>21b</b> showing high hCl<sub>i</sub> and moderate rCl<sub>i</sub> (107 and 59 μL/min/mg protein, respectively), the two compounds were selected to validate in vitro data with rodent in vivo data.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22, 23)</a> Therefore, both compounds were tested in a portal vein-cannulated rat model.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Compound <b>20b</b> showed a fraction absorbed (<i>F</i><sub>a</sub>%) of 70% with moderate distribution volume (<i>V</i><sub>ss</sub> = 4.9 L/kg) and moderate clearance in blood as expected from the in vitro parameters (Cl<sub>b</sub> = 32.9 mL/min/kg), leading to moderate half-life (<i>T</i><sub>1/2</sub> = 2.3 h) and good bioavailability (<i>F</i>% = 49%) given the relatively low hepatic extraction (<i>E</i><sub>H</sub> = 0.31). Compound <b>21b</b> had a fraction absorbed (<i>F</i><sub>a</sub>%) of 8.6% with low distribution volume (<i>V</i><sub>ss</sub> = 0.49 L/kg), clearance in blood in relative agreement with the in vitro parameters (Cl<sub>b</sub> = 15.1 mL/min/kg) although lower than expected. The half-life was short (<i>T</i><sub>1/2</sub> = 0.41 h) and bioavailability was low (<i>F</i>% = 4%) because of the poor fraction absorbed and a doubled <i>E</i><sub>H</sub> (0.56). These data are reported in <a class="ref internalNav" href="#tbl10" aria-label="Table 10">Table 10</a>. This head-to-head comparison therefore demonstrated a relatively good reliability of the Cl<sub>i</sub> parameters but also highlighted a striking difference in terms of the in vivo PK properties of the diastereoisomers. Brain penetration was also assessed and a brain to blood (B/B) ratio was achieved using the area under the curve (AUC) values. Both compounds had good ratios, namely 1.6 and 0.8 for <b>20b</b> and <b>21b</b>, respectively (<a class="ref internalNav" href="#tbl10" aria-label="Table 10">Table 10</a>).</div><div class="NLM_p">Interestingly enough, when the −F was kept in position 4 of the phenyl ring and the −CF<sub>3</sub> moiety was in position 2, the most active enantiomer resulted in the <i>trans</i> derivative <b>22a</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>); neither the overall affinity of this compound at the D3R nor its selectivity vs off-targets changed significantly. The in vitro PK parameters of this derivative did not change either (<a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>). <b>22a</b> showed a Fu<sub>br</sub>= 8.9% and a Fu<sub>bl</sub> = 12.3%, while both hCl<sub>i</sub> and rCl<sub>i</sub> were moderate (65 and 51 μL/min/mg protein, respectively). In contrast, derivative <b>23b</b> showed Fu<sub>br</sub> and Fu<sub>bl</sub> values that were comparable to the other geometric isomer (8.6 and 13.5%, respectively), both hCl<sub>i</sub> and rCl<sub>i</sub> significantly increased (320 and 151 μL/min/mg protein, respectively), suggesting a role of the 2-CF<sub>3</sub> moiety in the relative conformation of the phenyl ring, potentially through a change in dihedral angle in the <i>cis</i> derivative to a more metabolically unstable situation.</div><div class="NLM_p">The introduction of a hydroxyl moiety in the side chain (C3 side chain, derivative <b>24</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) to look for additional interaction within the receptor and to reduce the cLogP ever further, was detrimental for the affinity profile, similarly to what was also reported for a different series by Newman et al.<a onclick="showRef(event, 'cit12b'); return false;" href="javascript:void(0);" class="ref cit12b">(12b)</a> Finally, the introduction of a 3,5 dichloro substitution on the phenyl (<b>25</b>, <b>26</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) had no major effect on the affinity at the D3R but was clearly detrimental for the selectivity vs the D2R and vs the hERG channel.</div><div class="NLM_p">On the basis of the preliminary results of this early exploration, it was decided to maintain the <i>p</i>-CF<sub>3</sub> phenyl moiety of the “amino portion” but to replace the methyl oxazole ring of <b>10</b> with different aromatic or heteroaromatic moieties.</div><div class="NLM_p">The results for 5-membered heteroaromatics are reported in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>. The introduction of a methyl thiazole (<b>27</b>, <b>28</b>) despite an increase in cLogP and in PSA<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> led to a slight decrease in affinity at the D3R when compared to the racemic <i>cis</i> derivatives (<b>28</b> vs <b>10</b>). The removal of the methyl group and a change in relative orientation of the heteroatoms (<b>29</b>, <b>30</b>) led to equipotent derivatives (<b>30</b> vs <b>10</b>) with about 300-fold selectivity over the D2R and the hERG channel but with more than one log unit increase in cLogP. The introduction of one extra nitrogen atom in the system (<b>31</b>, <b>32</b>), despite a significant increase in PSA, left the primary affinity almost unchanged but produced an unexpected drop in affinity at the hERG channel in the <i>cis</i> racemate (<b>32</b>), which did not match the significant increase in affinity observed for the <i>trans</i> derivative (<b>31</b>). The functional profile of <b>32</b> was unchanged (<a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>). Unfortunately, and in contrast with other compounds in this series, <b>32</b> was not progressed because it showed a significantly reduced selectivity vs the M1 receptor (fp<i>K</i><sub>i</sub> = 6.77 ± 0.18, <a class="ref internalNav" href="#tbl11" aria-label="Table 11">Table 11</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Affinity Data at the D3R and D2R and Potency Data at the hERG Ion Channel for Derivatives <b>27</b>–<b>39</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0008.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char="±">hD3 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="±">hD2 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center">hERG pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">cLogP</th><th class="colsep0 rowsep0" align="center" char=".">PSA</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">4-methyl-1,3-thiazol-5-yl (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">7.82 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">5.91 ± 0.08</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="char" char=".">4.2</td><td class="colsep0 rowsep0" align="char" char=".">100.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">4-methyl-1,3-thiazol-5-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.44 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="±">5.79 ± 0.01</td><td class="colsep0 rowsep0" align="center">6.10 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">4.2</td><td class="colsep0 rowsep0" align="char" char=".">100.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">1,3-thiazol-2-yl (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.00 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="±">6.33 ± 0.06</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">100.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">1,3-thiazol-2-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.86 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">6.37 ± 0.01</td><td class="colsep0 rowsep0" align="center">6.35 ± 0.17</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">100.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">1,2,3-thiadiazol-4-yl (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.56 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">6.93 ± 0.09</td><td class="colsep0 rowsep0" align="center">6.95 ± 0.08</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td><td class="colsep0 rowsep0" align="char" char=".">113.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">1,2,3-thiadiazol-4-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.50 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">6.89 ± 0.14</td><td class="colsep0 rowsep0" align="center"><5.0</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td><td class="colsep0 rowsep0" align="char" char=".">113.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">thiophen-2-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">9.22 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">6.46 ± 0.11</td><td class="colsep0 rowsep0" align="center">>7.0</td><td class="colsep0 rowsep0" align="char" char=".">5.2</td><td class="colsep0 rowsep0" align="char" char=".">72.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">thiophen-3-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">9.24 ± 0.17</td><td class="colsep0 rowsep0" align="char" char="±">6.48 ± 0.04</td><td class="colsep0 rowsep0" align="center">>7.0</td><td class="colsep0 rowsep0" align="char" char=".">5.2</td><td class="colsep0 rowsep0" align="char" char=".">72.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="left">furan-2-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.86 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">6.32 ± 0.08</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">72.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="left">furan-3-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.78 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">6.30 ± 0.11</td><td class="colsep0 rowsep0" align="center">6.82 ± 0.11</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">72.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="left">1-methyl-1<i>H</i>-pyrazol-4-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.62 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">6.27 ± 0.01</td><td class="colsep0 rowsep0" align="center">6.33 ± 0.15</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="char" char=".">77.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="left">1-methyl-1<i>H</i>-pyrazol-5-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.76 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">6.07 ± 0.05</td><td class="colsep0 rowsep0" align="center">6.21 ± 0.11</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="char" char=".">77.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="left">3-methyl-1,2-oxazol-5-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.57 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">6.28 ± 0.05</td><td class="colsep0 rowsep0" align="center">6.69 ± 0.07</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">rac = racemate; NT = not tested;; values shown are the average of three replicates and standard deviation.</p></div></div></div><div class="NLM_p">The introduction of a 2 thiophene (<b>33</b>, <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>) further increased lipophilicity while reducing PSA. This had a beneficial effect on the affinity at the D3R, leading to a subnanomolar derivative on the racemate, but this change also had a detrimental effect on the hERG affinity. Its regioisomers (<b>34</b>) was equivalent both in terms of potency and selectivity. The same behavior was also observed in the case of the furane moiety (<b>35</b>, <b>36</b>), but high affinity at the hERG channel was observed for derivative <b>36</b>. The introduction of a pyrazole moiety (<b>37</b>, <b>38</b>) led to compounds with high affinity at the D3R and high selectivity. Derivative <b>37</b> showed about 200-fold selectivity vs the D2R and the hERG channel, while <b>38</b> increased such window to more than 450-fold. To complete the 5-membered preliminary heterocyclic replacement, the isoxazole <b>39</b> was prepared. In this case, the affinity at the D3R was maintained but with a slight reduction of selectivity as well as an increase in the hERG affinity. None of these derivatives were tested for in vitro PK properties because of their racemic nature and their suboptimal profile for further progression through the screening cascade.</div><div class="NLM_p">The exploration of the oxazole replacement (<b>10</b>) was pursued with substituted phenyl rings so that some of the original oxazole polarity/charge distribution was present (see <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a> with only the best enantiomer of the derivatives with <i>cis</i> configuration).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Affinity Data at the D3R and D2R and Potency Data at the hERG Ion Channel for Derivatives <b>40a</b>–<b>48a</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0009.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char="±">hD3 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="±">hD2 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center">hERG pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">cLogP</th><th class="colsep0 rowsep0" align="center" char=".">PSA</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40a</b></td><td class="colsep0 rowsep0" align="left">4-benzamide (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">9.80 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">6.45 ± 0.01</td><td class="colsep0 rowsep0" align="center">6.09 ± 0.07</td><td class="colsep0 rowsep0" align="char" char=".">4.3</td><td class="colsep0 rowsep0" align="char" char=".">102.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41a</b></td><td class="colsep0 rowsep0" align="left">3-benzamide (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">9.33 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">6.20 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.55 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">4.3</td><td class="colsep0 rowsep0" align="char" char=".">102.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42a</b></td><td class="colsep0 rowsep0" align="left">2-benzamide (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">7.82 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">7.40 ± 0.01</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="char" char=".">4.3</td><td class="colsep0 rowsep0" align="char" char=".">102.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43a</b></td><td class="colsep0 rowsep0" align="left">4-phenyl-1-sulfonamide (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">9.50 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">6.66 ± 0.08</td><td class="colsep0 rowsep0" align="center">6.65 ± 0.10</td><td class="colsep0 rowsep0" align="char" char=".">4.0</td><td class="colsep0 rowsep0" align="char" char=".">127.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>44a</b></td><td class="colsep0 rowsep0" align="left">4-phenyl-ethan-1-one (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">9.68 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">6.35 ± 0.01</td><td class="colsep0 rowsep0" align="center">>7.0</td><td class="colsep0 rowsep0" align="char" char=".">5.0</td><td class="colsep0 rowsep0" align="char" char=".">76.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>45a</b></td><td class="colsep0 rowsep0" align="left">4-benzonitrile (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">9.58 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">6.46 ± 0.03</td><td class="colsep0 rowsep0" align="center">>7.0</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td><td class="colsep0 rowsep0" align="char" char=".">83.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46a</b></td><td class="colsep0 rowsep0" align="left">4-phenyl-1-acetamide (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">9.07 ± 0.16</td><td class="colsep0 rowsep0" align="char" char="±">6.33 ± 0.08</td><td class="colsep0 rowsep0" align="center">5.12 ± 0.03</td><td class="colsep0 rowsep0" align="char" char=".">4.2</td><td class="colsep0 rowsep0" align="char" char=".">102.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>47a</b></td><td class="colsep0 rowsep0" align="left">4-(1,3-oxazol-2-yl)-phenyl (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">10.17 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">6.44 ± 0.01</td><td class="colsep0 rowsep0" align="center">6.82 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48a</b></td><td class="colsep0 rowsep0" align="left">3-(1,3-oxazol-2-yl)-phenyl (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">9.74 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">6.30 ± 0.07</td><td class="colsep0 rowsep0" align="center">6.92 ± 0.09</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td><td class="colsep0 rowsep0" align="char" char=".">85.3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">se 1 = single enantiomer 1; NT = not tested; values shown are the average of three replicates and standard deviation.</p></div></div></div><div class="NLM_p">Derivative <b>40a</b> (<i>p</i>-CONH<sub>2</sub> phenyl, <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>) achieved a 2200-fold selectivity over the D2R and a 5000-fold selectivity over the hERG channel, thus supporting the potential of this subseries. The compounds showed no activity at the D1R and D4R (p<i>K</i><sub>i</sub> < 4.50), and a more than 2500-fold selectivity was also achieved vs M1 and M3 receptors (fp<i>K</i><sub>i</sub> = 6.34 ± 0.05 and 6.37 ± 0.08, respectively, <a class="ref internalNav" href="#tbl11" aria-label="Table 11">Table 11</a>). The in vitro PK profile (<a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>) revealed IC<sub>50</sub> values greater than 5 μM on all CYP P450 isoforms tested. hCl<sub>i</sub> on microsomes was high (155 μL/min/mg protein), rCl<sub>i</sub> was moderate (41 μL/min/mg protein), Fu<sub>br</sub> was equal to 4.4%, and Fu<sub>bl</sub> resulted 14.1%. The modeling studies performed on the compound (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>) suggested the possibility for the amide to positively interact with additional groups present in the secondary binding pocket (SBP)<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> of the D3R.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Derivatives <b>40a</b> (cyan), <b>41a</b> (mustard), and <b>42a</b> (red) docking to the D3R model. Portions of the backbone ribbon have been removed for clarity. The interaction of the basic nitrogen of the amine portion of the two compounds with Asp<sup>3,32</sup> is clearly visible. For derivative <b>40a</b>, the potential interaction with Cys<sup>3</sup><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> is also highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00972&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Moving the amide portion to position 3 of the aromatic ring (<i>m</i>-CONH<sub>2</sub> phenyl; <b>41a</b>, <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>) led to about half a log unit reduction in the affinity at the D3R but also led to a similar reduction in affinity at the other off-targets. Consequently, selectivity vs the D2R was greater than 1300-fold and selectivity over the hERG channel was 6000-fold; more importantly, the absolute hERG value was 3 μM compared with a 0.4 nM affinity at the D3R. The compound showed no affinity at the D1R and only low affinity at the D4R (fp<i>K</i><sub>i</sub>= 5.23 ± 0.03); affinity at the M1 and M3 receptors was also moderately low (fp<i>K</i><sub>i</sub>= 6.58 ± 0.16 and 6.42 ± 0.10, respectively, <a class="ref internalNav" href="#tbl11" aria-label="Table 11">Table 11</a>), leading to more than 550-fold selectivity at these receptors. When tested for in vitro PK properties (<a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>), <b>41a</b> showed IC<sub>50</sub> values greater than 3 μM on all CYP P450 isoforms tested. hCl<sub>i</sub> on microsomes was comparable to derivative <b>40a</b> (187 μL/min/mg protein), while rCl<sub>i</sub> was higher (92 μL/min/mg protein), suggesting an increased metabolic transformation of the now unprotected position 4 of the phenyl ring in rodents. Conversely, both Fu<sub>br</sub> and Fu<sub>bl</sub> were quite similar to the previous compound (3.7 and 6.7%, respectively). Both <b>40a</b> and <b>41a</b> were tested in a functional D3R assay, confirming an unchanged antagonist behavior (<a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>).</div><div class="NLM_p">The insertion of the amide moiety in position 2 of the phenyl ring (<i>o</i>-CONH<sub>2</sub> phenyl; <b>42a</b>, <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>) led to a significant decrease in affinity at the D3R (about 100-fold when compared to <b>40a</b>) as well as a decrease in selectivity vs the D2R. It was hypothesized that the change in the dihedral angle induced by the <i>o</i>-amide insertion disfavored the interactions with key residues within the D3R and the SBP, leaving the residues within the D2R unchanged. Alternatively, one may hypothesize that such a substituent may have generated steric clashes, preventing the best fitting within the binding pocket. To find additional hints which could help to explain better the observed SAR, all the three structures were modeled within the D3R model (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>) as previously reported,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and the GBVI/WSA dG scoring function as implemented in MOE<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> was used. Subsequently, the “lowest score” poses were submitted to conformational search using LowModeMD (MOE) and the lowest energy conformations obtained were further minimized. The ligand–protein affinity was recalculated and the analysis of the values observed (data in the experimental part), suggested a trend in line with the binding affinity of the derivatives (<b>40a</b>, <b>41a</b>, <b>42a</b>), even if the absolute difference in energy did not fully justify the differences observed in the binding affinities for <b>42a</b>.</div><div class="NLM_p">The introduction of a <i>p</i>-SO<sub>2</sub>NH<sub>2</sub> phenyl (<b>43a</b>, <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>) slightly decreased cLogP but greatly increased the PSA value; the affinity at the D3R was maintained and the selectivity over the D2R and the hERG channel remained high (about 700-fold). Unfortunately, the absolute affinity at the hERG channel was about 0.2 μM and the compound was not progressed. The replacement of the sulphonamide group with an acetyl moiety (<b>44a</b>) increased cLogP, reducing the PSA value. The affinity at the D3R was unaltered, leading to a selectivity vs the D2R greater than 2000-fold. Unfortunately, the fp<i>K</i><sub>i</sub> observed at the hERG channel was similarly high. The nitrile derivative <b>45a</b> behaved similarly, with very high affinity at the D3R and high selectivity vs the D2R but with a comparably high hERG affinity. In contrast, the introduction of a <i>p</i>-CH<sub>2</sub>CONH<sub>2</sub> (<b>46a</b>) group on the phenyl group led to a slight decrease in affinity at the D3R with respect to derivative <b>10a</b>. Selectivity vs the D2R remained greater than 500-fold, and the selectivity vs the hERG channel was increased to about 8900-fold with an absolute value of 7.5 μM, leading to one of the lowest hERG affinity up to that point of the exploration. Accordingly, in vitro PK data were generated (<a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>) and the compound showed IC<sub>50</sub> values greater than 7 μM on all CYP P450 isoforms tested; hCl<sub>i</sub> on microsomes was moderate (58 μL/min/mg protein), while rCl<sub>i</sub> was low (22 μL/min/mg protein). In terms of free fraction, both Fu<sub>br</sub> and Fu<sub>bl</sub> were relatively high (5.2 and 14.1%, respectively).</div><div class="NLM_p">In light of the promising results achieved with derivatives <b>40a</b> and <b>41a</b>, a minimal bioisosteric replacement of the primary amide was attempted and the oxazole derivatives <b>47a</b> and <b>48a</b> were prepared. Derivative <b>47a</b> reached subnanomolar affinity (0.07 nM) at the D3R, with a more than 5300-fold selectivity window vs the D2R and more than 2200-fold selectivity over the hERG channel. The compound showed no affinity at the D1R and D4R; the affinity at the M1 and M3 receptors was also moderately low (fp<i>K</i><sub>i</sub>= 6.17 ± 0.01 and 6.52 ± 0.11, respectively, <a class="ref internalNav" href="#tbl11" aria-label="Table 11">Table 11</a>), leading to about 4500-fold selectivity at these muscarinic receptors. The functional activity confirmed the antagonistic activity at the D3R (<a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>). Despite a relatively high cLogP, the in vitro PK profile (<a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>) of <b>47a</b> showed IC<sub>50</sub> values greater than 3 μM on all CYP P450 isoforms tested. In line with our expectations, however, this high value was also reflected in a reduction of free compound both in brain and blood (Fu<sub>br</sub> and Fu<sub>bl</sub> = 1.0 and 1.2%, respectively). In addition, Cl<sub>i</sub> values reflected the increased lipophilicity: hCl<sub>i</sub> and rCl<sub>i</sub> were increased to 352 and 276 (μL/min/mg protein), respectively. Comparable results were obtained with derivative <b>48a</b> (<a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>). No further bioisosteric replacement exercise was performed at that stage of the project, and this activity was postponed to a subsequent lead optimization phase. Functional data for <b>48a</b> are reported in <a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>.</div><div class="NLM_p">A rapid exploration of the tolerability of the scaffold to basic 6-membered heteroaromatics was performed using racemic mixtures (and consequently both <i>trans</i> and <i>cis</i> isomers were also assayed). The results of this exploration are reported in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>. For the pyridine moiety (<b>49</b>–<b>54</b>), the <i>cis</i> pyridin-2-yl substituent (<b>54</b>) showed the best affinity at the D3R and a 660-fold selectivity vs the D2R. The <i>cis</i> derivative <b>52</b> (pyridine-3-yl moiety) showed a more balanced profile with increased selectivity at the hERG up to 680-fold and a 1.2 μM absolute hERG value vs a 0.2 μM hERG value for derivative <b>54</b>. Although the introduction of a second nitrogen atom (<b>55</b>–<b>62</b>) in the six-membered ring system proved to be efficient for the reduction in overall lipophilicity, it failed to show any major advantage in terms of affinity at the D3R and overall selectivity. One of the most balanced derivatives was the pyridazine-4-yl (<b>62</b>), which showed (once again on the <i>cis</i> isomer) good affinity at the D3R and 390- and 630-fold selectivity over the D2R and the hERG channel, respectively (the absolute hERG value of 2.5 μM was observed for <b>62</b>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Affinity Data at the D3R and D2R and Potency Data at the hERG Ion Channel for Derivatives <b>49</b>–<b>62</b><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0010.gif" alt="" id="fx4" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char="±">hD3 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="±">hD2 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center">hERG pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">cLogP</th><th class="colsep0 rowsep0" align="center" char=".">PSA</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>49</b></td><td class="colsep0 rowsep0" align="left">pyridin-4-yl (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.10 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">6.06 ± 0.07</td><td class="colsep0 rowsep0" align="center">6.65 ± 0.03</td><td class="colsep0 rowsep0" align="char" char=".">4.2</td><td class="colsep0 rowsep0" align="char" char=".">72.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>50</b></td><td class="colsep0 rowsep0" align="left">pyridin-4-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.96 ± 0.18</td><td class="colsep0 rowsep0" align="char" char="±">6.21 ± 0.01</td><td class="colsep0 rowsep0" align="center">6.53 ± 0.06</td><td class="colsep0 rowsep0" align="char" char=".">4.2</td><td class="colsep0 rowsep0" align="char" char=".">72.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>51</b></td><td class="colsep0 rowsep0" align="left">pyridin-3-yl (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">7.70 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">5.87 ± 0.04</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="char" char=".">4.2</td><td class="colsep0 rowsep0" align="char" char=".">72.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>52</b></td><td class="colsep0 rowsep0" align="left">pyridin-3-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.74 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">6.05 ± 0.02</td><td class="colsep0 rowsep0" align="center">5.91 ± 0.03</td><td class="colsep0 rowsep0" align="char" char=".">4.2</td><td class="colsep0 rowsep0" align="char" char=".">72.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>53</b></td><td class="colsep0 rowsep0" align="left">pyridin-2-yl (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.11 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">6.20 ± 0.06</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="char" char=".">4.2</td><td class="colsep0 rowsep0" align="char" char=".">72.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>54</b></td><td class="colsep0 rowsep0" align="left">pyridin-2-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">9.10 ± 0.19</td><td class="colsep0 rowsep0" align="char" char="±">6.28 ± 0.02</td><td class="colsep0 rowsep0" align="center">6.68 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">4.2</td><td class="colsep0 rowsep0" align="char" char=".">72.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>55</b></td><td class="colsep0 rowsep0" align="left">pyrazin-2-yl (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">7.97 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">5.84 ± 0.21</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">85.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>56</b></td><td class="colsep0 rowsep0" align="left">pyrazin-2-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.90 ± 0.16</td><td class="colsep0 rowsep0" align="char" char="±">6.33 ± 0.07</td><td class="colsep0 rowsep0" align="center">6.59 ± 0.07</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">85.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>57</b></td><td class="colsep0 rowsep0" align="left">pyrimidin-4-yl (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.29 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">6.36 ± 0.12</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="char" char=".">3.9</td><td class="colsep0 rowsep0" align="char" char=".">85.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>58</b></td><td class="colsep0 rowsep0" align="left">pyrimidin-4-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.99 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">6.53 ± 0.02</td><td class="colsep0 rowsep0" align="center">6.84 ± 0.06</td><td class="colsep0 rowsep0" align="char" char=".">3.9</td><td class="colsep0 rowsep0" align="char" char=".">85.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>59</b></td><td class="colsep0 rowsep0" align="left">pyridazin-3-yl (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">7.01 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">5.75 ± 0.01</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">85.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>60</b></td><td class="colsep0 rowsep0" align="left">pyridazin-3-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">7.78 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">5.86 ± 0.01</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">85.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>61</b></td><td class="colsep0 rowsep0" align="left">pyridazin-4-yl (<i>trans</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">7.45 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">5.59 ± 0.04</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">85.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>62</b></td><td class="colsep0 rowsep0" align="left">pyridazin-4-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.40 ± 0.15</td><td class="colsep0 rowsep0" align="char" char="±">5.81 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.60 ± 0.22</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">85.0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">rac = racemate; NT = not tested; values shown are the average of three replicates and standard deviation.</p></div></div></div><div class="NLM_p">In light of the overall balanced profile of the <i>cis</i> 3-pyridinyl derivative <b>52</b>, the unsubstituted derivative <b>63</b> was prepared (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). The direct comparison between the two racemic compounds clearly show that the removal of the 4-CF<sub>3</sub> group in the phenyl derivative of the “amino portion” led to a decrease in affinity greater than one log unit. For compound <b>64b</b> (the best <i>cis</i> single enantiomer), such substitution was reintroduced adding an additional 2-F substitution. The result was a derivative showing almost 1.2 nM affinity at the D3R and a selectivity greater than 600-fold at the D2R and hERG channel. The in vitro PK (<a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>) revealed IC<sub>50</sub> values greater than 7 μM on all CYP P450 isoforms tested with the exception of CYP2D6, which was greater than 1 μM. Both Fu<sub>br</sub> and Fu<sub>bl</sub> were relatively high with 7.7% and 16.1%, respectively. As clearly expected for a nonsubstituted 3-pyridine, the in vitro clearance was characterized by high values (namely, hCl<sub>i</sub> and rCl<sub>i</sub> = 485 and 278 μL/min/mg protein, respectively).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Affinity Data at the D3R and D2R and Potency Data at the hERG Ion Channel for Derivatives <b>63</b>–<b>66b</b><a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0011.gif" alt="" id="fx5" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char="±">hD3 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="±">hD2 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center">hERG pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">cLogP</th><th class="colsep0 rowsep0" align="center" char=".">PSA</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>63</b></td><td class="colsep0 rowsep0" align="left">H (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">7.53 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">5.48 ± 0.04</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char=".">72.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>64b</b></td><td class="colsep0 rowsep0" align="left">2-F, 4-CF<sub>3</sub> (<i>cis</i>, se 2)</td><td class="colsep0 rowsep0" align="char" char="±">8.93 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">6.14 ± 0.01</td><td class="colsep0 rowsep0" align="center">6.03 ± 0.11</td><td class="colsep0 rowsep0" align="char" char=".">4.3</td><td class="colsep0 rowsep0" align="char" char=".">72.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>65b</b></td><td class="colsep0 rowsep0" align="left">2,4-F (<i>cis</i>, se 2)</td><td class="colsep0 rowsep0" align="char" char="±">8.89 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">6.28 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.56 ± 0.17</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">72.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>66b</b></td><td class="colsep0 rowsep0" align="left">4-F (<i>cis</i>, se 2)</td><td class="colsep0 rowsep0" align="char" char="±">8.70 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">6.18 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.68 ± 0.11</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">72.1</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">se 2 = single enantiomer 2; rac = racemate; NT = not tested; values shown are the average of three replicates and standard deviation.</p></div></div></div><div class="NLM_p">A 2,4-F disubstitution, i.e., the replacement of the 4-CF<sub>3</sub> group with a −F atom (<b>65b</b>, <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>) led to a slight reduction in the affinity at the D3R with the advantage of a more than half one log unit decrease in cLogP. While maintaining a 1.2 nM affinity at the D3R and a 400-fold selectivity vs the D2R, this compound increased selectivity vs hERG to over 2100-fold with an absolute 2.8 μM hERG affinity. Its functional behavior was also confirmed (<a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>). A slight improvement was observed with respect to the CYP P450 profile, while a large increase in the free fraction was seen both in brain and blood (Fu<sub>br</sub> and Fu<sub>bl</sub> = 25 and >50%, respectively, <a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>). Similar to the previous compound, Cl<sub>i</sub> data were high (namely, hCl<sub>i</sub> and rCl<sub>i</sub> = 258 and 253 μL/min/mg protein, respectively). Finally, the decision to leave a single 4-F substituent (<b>66b</b>, <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>) on the aromatic ring further reduced cLogP almost to the original value of the unsubstituted derivative. This reduction had no major impact on the affinity at the D3R, keeping the selectivity profile comparable to the previous derivatives. The fraction unbound (<a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>) of the compound was high (Fu<sub>br</sub> and Fu<sub>bl</sub> = 26 and >50%, respectively), and most probably because of the decrease in lipophilicity and despite the presence of an unsubstituted 3 pyridine, Cl<sub>i</sub> values were markedly reduced with respect to the previously analyzed compounds, leading to a hCl<sub>i</sub> value equal to 91 μL/min/mg protein.</div><div class="NLM_p">Given the results of this very rapid exploration and to maintain homogeneous comparison with the previously synthesized compounds, a further analysis of the aromatic pattern of substitution of this part of the molecule was postponed to the lead optimization phase. The 4-CF<sub>3</sub> group was kept as a reference while substituting the 3-pyridine with groups that, based on Cl<sub>i</sub> data, were hypothesized to prevent the metabolic degradation of the ring.</div><div class="NLM_p">The insertion of a methyl group in position 2 of the pyridine system (<b>67a</b>, <a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>) was slightly less potent than the corresponding unsubstituted racemate (<b>52</b>) but was also less active at off-targets, leading to 450- and 1900-fold selectivity vs the D2R and the hERG channel; derivative <b>67a</b> showed a 6.3 μM hERG activity. According to the working hypothesis, the methyl group should have increased the dihedral angle formed by the pyridine ring with the triazole moiety, hence reducing the conjugation of the two rings. Compound <b>67a</b> was inactive at the D1R and D4R and the activity at the M1 and M3 muscarinic receptors was quite reduced, leading to a selectivity greater than 350-fold at both receptors (fp<i>K</i><sub>i</sub> = 5.78 ± 0.14 and 5.91 ± 0.07, respectively, <a class="ref internalNav" href="#tbl11" aria-label="Table 11">Table 11</a>). The impact on the CYP P450 profile was positive, with IC<sub>50</sub> values greater than 7 μM on all CYP P450 isoforms tested with the exception of CYP3A4 7BQ, which was greater than 2 μM. Both Fu<sub>br</sub> and Fu<sub>bl</sub> were moderate with values of 6.2 and 18.6%, respectively. A clear reduction in Cl<sub>i</sub> was also observed with hCl<sub>i</sub> and rCl<sub>i</sub> equivalent to 80 and 103 μL/min/mg protein, respectively, <a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>).</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Affinity Data at the D3R and D2R and Potency Data at the hERG Ion Channel for Derivatives <b>67a</b>–<b>74a</b><a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0012.gif" alt="" id="fx6" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char="±">hD3 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="±">hD2 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center">hERG pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">cLogP</th><th class="colsep0 rowsep0" align="center" char=".">PSA</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>67a</b></td><td class="colsep0 rowsep0" align="left">2-methylpyridin-3-yl (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">8.48 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="±">5.83 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.20 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">4.3</td><td class="colsep0 rowsep0" align="char" char=".">72.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>68a</b></td><td class="colsep0 rowsep0" align="left">2-(trifluoromethyl)pyridin-3-yl (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">8.15 ± 0.17</td><td class="colsep0 rowsep0" align="char" char="±">6.02 ± 0.03</td><td class="colsep0 rowsep0" align="center">5.88 ± 0.23</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td><td class="colsep0 rowsep0" align="char" char=".">72.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>69a</b></td><td class="colsep0 rowsep0" align="left">2-methoxypyridin-3-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.59 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">5.94 ± 0.05</td><td class="colsep0 rowsep0" align="center">6.22 ± 0.09</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="char" char=".">81.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>70a</b></td><td class="colsep0 rowsep0" align="left">2-cyano pyridin-5-yl (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">8.84 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">6.16 ± 0.01</td><td class="colsep0 rowsep0" align="center">>7.0</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td><td class="colsep0 rowsep0" align="char" char=".">95.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>71a</b></td><td class="colsep0 rowsep0" align="left">2-carboxamide pyridin-5-yl (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">9.47 ± 0.18</td><td class="colsep0 rowsep0" align="char" char="±">6.26 ± 0.03</td><td class="colsep0 rowsep0" align="center">6.18 ± 0.13</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">115.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>72a</b></td><td class="colsep0 rowsep0" align="left">2,6-dimethylpyridin-3-yl (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">8.48 ± 0.18</td><td class="colsep0 rowsep0" align="char" char="±">5.91 ± 0.04</td><td class="colsep0 rowsep0" align="center">6.03 ± 0.12</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">72.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>73a</b></td><td class="colsep0 rowsep0" align="left">2-carboxamide-6-methyl- pyridin-5-yl (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">8.90 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">5.94 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.56 ± 0.17</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">115.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>74a</b></td><td class="colsep0 rowsep0" align="left">2-carboxy-6-methyl- pyridin-5-yl (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">7.44 ± 0.17</td><td class="colsep0 rowsep0" align="char" char="±">5.12 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.00 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td><td class="colsep0 rowsep0" align="char" char=".">109.4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup>a</sup><p class="last">se 1 = single enantiomer 1; rac = racemate; values shown are the average of three replicates and standard deviation.</p></div></div></div><div class="NLM_p">When this methyl was replaced by a trifluoromethyl group (<b>68a</b>, <a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>) cLogP was clearly increased; a decrease in affinity at the D3R and a reduced selectivity profile were also observed. In contrast, this manipulation had a positive effect on the CYP P450 profile; on all the CYP P450 isoforms tested, compound <b>68a</b> had IC<sub>50</sub> values greater than 6 μM. However, the increased lipophilicity had a detrimental effect on free fraction (Fu<sub>br</sub> and Fu<sub>bl</sub> = 2.2 and 7.5%, respectively, <a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>) as well as on intrinsic clearance; hCl<sub>i</sub> and rCl<sub>i</sub> reverted to 302 and 114 (μL/min/mg protein), respectively.</div><div class="NLM_p">A reduction in cLogP and a slight increase in PSA were calculated for the replacement of the −CF<sub>3</sub> with an electron-donating group such as the methoxy one (<b>69a</b>, <a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>). Potency and selectivity vs the D2R were comparable to the methyl derivative <b>67a</b>, while the affinity at the hERG channel increased by one log unit.</div><div class="NLM_p">When derivative <b>70a</b> (cyano derivative) was prepared to probe the other position close to the pyridine nitrogen, an increase in affinity at the D3R was observed and a good selectivity vs the D2R was maintained. The affinity at the hERG channel, however, also increased, and it was decided not to progress this compound along the screening cascade. The transformation of the nitrile into a primary amide (<b>71a</b>) produced a one log unit increase in the affinity at the D3R, and the selectivity vs the secondary targets was greater than 1600-fold. The in vitro PK profile (<a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>) of <b>71a</b> revealed IC<sub>50</sub> values greater than 17 μM on all CYP P450 isoforms, and it was therefore decided to investigate its profile further. The reduction in lipophilicity (cLogP = 3.4) had a beneficial effect on free fraction (Fu<sub>br</sub> and Fu<sub>bl</sub> = 2.2 and 18.4%, respectively) and on intrinsic clearance (hCl<sub>i</sub> and rCl<sub>i</sub> were 106 and 29 μL/min/mg protein, respectively). Compound <b>71a</b> behaved as a competitive antagonist at the D3R (<a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>) with no activity at the D1R and the D4R, and a 550-fold selectivity over muscarinic M1 and M3 receptors (fp<i>K</i><sub>i</sub> = 6.65 ± 0.05 and 6.72 ± 0.04 on M1 and M3, respectively, <a class="ref internalNav" href="#tbl11" aria-label="Table 11">Table 11</a>); this compound was therefore tested in vivo<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22, 23)</a> to validate correlation with in vitro PK data. Compound <b>71a</b> had a relatively good fraction absorbed (<i>F</i><sub>a</sub> = 40%, <a class="ref internalNav" href="#tbl10" aria-label="Table 10">Table 10</a>) with moderate distribution volume (<i>V</i><sub>ss</sub> = 3.2 L/kg) and relatively low clearance in blood as expected from the in vitro parameters (Cl<sub>b</sub> = 19.4 mL/min/kg), leading to moderate half-life (<i>T</i><sub>1/2</sub> = 2.2 h) and relatively good bioavailability (<i>F</i>% = 40%) given low hepatic extraction (<i>E</i><sub>H</sub> = 0.35). As expected from increased PSA values, the B/B ratio was slightly lower than the previously tested compounds (B/B = 0.3). The overall parameters for this compound constituted an excellent starting point for further exploration in the lead optimization phase.</div><div class="NLM_p">The exploration of the substitution of the 3 pyridine ring continued with the introduction of two methyl groups (<b>72a</b>, <a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>) to assess whether the contemporary substitution of the two positions alpha to the nitrogen was tolerated. No major differences with the mono substituted methyl derivative <b>67a</b> were observed, suggesting that the second methyl was relatively “neutral” to the overall molecule in terms of affinity. The slight increase in lipophilicity had no major impact on the CYP P450 profile (all isoforms had IC<sub>50</sub> values greater than 9 μM) or on free fraction (Fu<sub>br</sub> and Fu<sub>bl</sub> = 4.5 and 12.6%, respectively) but affected Cl<sub>i</sub> with increased values (hCl<sub>i</sub> and rCl<sub>i</sub> were 299 and 98 μL/min/mg protein, respectively, <a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>).</div><div class="NLM_p">Replacing one of the two methyls with the primary amide of <b>71a</b> led to the synthesis of compound <b>73a</b> (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>). This derivative showed a balanced affinity profile, with 1.2 nM affinity at the D3R and 900-fold and 2150-fold vs the D2R and the hERG channel, respectively. Moreover, the absolute affinity at the hERG channel was about 3 μM. The compound had a “clean” CYP P450 profile, showing IC<sub>50</sub> values greater than 10 μM on all isoforms tested. The rCl<sub>i</sub> was greatly reduced with respect to <b>72a</b> (rCl<sub>i</sub> = 38 μL/min/mg protein; <a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>), while hCl<sub>i</sub> values remained unexpectedly high (hCl<sub>i</sub> = 299 μL/min/mg protein). High free fraction values were also observed (Fu<sub>br</sub> and Fu<sub>bl</sub> = 7.8 and 19%, respectively).</div><div class="NLM_p">The compound was further profiled in vitro for selectivity, and functional activity was also assessed. The compound was an antagonist with 0.7 nM affinity at the D3R, thus increasing the selectivity vs the hERG channel to more than 4000-fold; no activity at the D1R and the D4R was observed, and a 270-fold selectivity was achieved over the muscarinic M1 and M3 receptors (fp<i>K</i><sub>i</sub> = 6.29 ± 0.08 and 6.73 ± 0.08 on M1 and M3, respectively, <a class="ref internalNav" href="#tbl11" aria-label="Table 11">Table 11</a>). Given the balanced profile of <b>73a</b>, its PK profile was assessed in vivo.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22, 23)</a> Compound <b>73a</b> showed a good fraction absorbed (<i>F</i><sub>a</sub> = 54%, <a class="ref internalNav" href="#tbl10" aria-label="Table 10">Table 10</a>) with moderate-high distribution volume (<i>V</i><sub>ss</sub> = 5.5 L/kg) and low clearance in blood as expected from the in vitro parameters (Cl<sub>b</sub> = 21.1 mL/min/kg). Given the low hepatic extraction (<i>E</i><sub>H</sub> = 0.28), the parameters showed a good half-life (<i>T</i><sub>1/2</sub> = 4.3 h) and relatively good bioavailability (<i>F</i>% = 53%). Brain penetration was also assessed, showing a B/B ratio equal to 1.5. Similar to 7<b>1a,</b> this compound was selected as a starting point for further characterization.</div><div class="NLM_p">Finally, the transformation of the primary amide into the corresponding carboxylic acid (<b>74a</b>, <a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>) caused an overall reduction in affinity across all targets, and IC<sub>50</sub> on all CYP P450 isoforms tested greater than 38 μM. Quite unexpectedly for a free carboxylic acid, very high free fractions were observed (Fu<sub>br</sub> and Fu<sub>bl</sub> = 22.9 and 29.9%, respectively, <a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>) and Cl<sub>i</sub> values were both very low (r, hCl<sub>i</sub> <11 μL/min/mg protein). Further exploration of this compound during the lead optimization phase of this project was deemed worthy based on its overall selectivity that was greater than 100-fold vs both the D2R and hERG channel (10 μM affinity).</div><div class="NLM_p">The exploration of the pyridine moieties <b>50</b> and <b>52</b> was continued (<a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>) with the introduction of a hydroxyl substituent on the ring, leading to the pyridone derivatives <b>75</b> and <b>76</b>. Only data on the most active <i>cis</i> stereoisomers are reported. Both compounds were originally prepared as racemates showing promising selectivity profiles and nanomolar affinity at the D3R. Because <b>75</b> had an IC<sub>50</sub> at the hERG channel greater than 10 μM, its most active enantiomer (<b>75a</b>) was further profiled, showing selectivity greater than 4300-fold vs the hERG channel. No affinity at the D1R and the D4R was observed, whereas selectivity vs muscarinic M1 and M3 receptors was higher than 200-fold (fp<i>K</i><sub>i</sub> = 6.31 ± 0.01 and 6.01 ± 0.16 on M1 and M3, respectively, <a class="ref internalNav" href="#tbl11" aria-label="Table 11">Table 11</a>). Given its IC<sub>50</sub> greater than 5 μM on all CYP P450 isoforms tested, its high free fraction (Fu<sub>br</sub> and Fu<sub>bl</sub> = 8.0 and 19.1%, respectively), and low Cl<sub>i</sub> both in rat and human (hCl<sub>i</sub> = 28 and rCl<sub>i</sub> = 14 μL/min/mg protein, respectively), it was decided to assess its PK profile in vivo.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22, 23)</a> Unfortunately, because of its low membrane permeability (assessed in Caco-2 cell lines),<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> the fraction absorbed of <b>75a</b> was negligible (<i>F</i><sub>a</sub> ≈ 1%, <a class="ref internalNav" href="#tbl10" aria-label="Table 10">Table 10</a>); this problem, associated with high Clb (48 mL/min/kg) and very high distribution volume (<i>V</i><sub>ss</sub> = 9.2 L/kg), led to negligible bioavailability (<i>F</i>% ≈ 2%). For this reason, compound <b>76</b> (<a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>) was not even separated in its enantiomers and the <i>N</i>-methylated version of both compounds was prepared (<b>77a</b>, <b>78a</b>). Both derivatives showed an excellent affinity and selectivity profile, especially at the hERG channel, showing 1800- and 1300-fold hERG selectivity, respectively. Their overall in vitro properties were similar: IC<sub>50</sub> values on all CYP P450 isoforms tested was greater than 9 μM and greater than 17 μM for <b>77a</b> and <b>78a</b>, respectively. High free fraction was observed for both compounds (Fu<sub>br</sub> and Fu<sub>bl</sub> = 10.8 and 25%, respectively, for <b>77a</b>, and Fu<sub>br</sub> and Fu<sub>bl</sub> = 14.6 and 16.7%, respectively, for <b>78a</b>, <a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>). Both compounds also showed comparable Cl<sub>i</sub> profiles (see <a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>). The medium permeability in the Caco-2 assay for both <b>77a</b> and <b>78a</b> was slightly better than <b>75a</b>. The latter observation was critical considering the in vivo<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22, 23)</a> PK profile. As expected by the in vitro results reported in <a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>, Clb values (reported with the full profiles in <a class="ref internalNav" href="#tbl10" aria-label="Table 10">Table 10</a>) were almost identical, with moderate clearance in rat as well as <i>V</i><sub>ss</sub> and <i>T</i><sub>1/2</sub>. The difference in bioavailability of the two compounds (10% vs 30%) perfectly reflected and matched their difference in fraction absorbed (17% vs 40%). Functional data for <b>78a</b> are available in <a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Affinity Data at the D3R and D2R and Potency Data at the hERG Ion Channel for Derivatives <b>75</b>–<b>85a</b><a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0013.gif" alt="" id="fx7" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char="±">hD3 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="±">hD2 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center">hERG pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">cLogP</th><th class="colsep0 rowsep0" align="center" char="±">PSA</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>75</b></td><td class="colsep0 rowsep0" align="left">[1,2-dihydropyridin-2-one]-5-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.28 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">6.12 ± 0.03</td><td class="colsep0 rowsep0" align="center"><5.0</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char="±">88.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>75a</b></td><td class="colsep0 rowsep0" align="left">[1,2-dihydropyridin-2-one]-5-yl (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">8.66 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">6.47 ± 0.04</td><td class="colsep0 rowsep0" align="center">5.02 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char="±">88.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>76</b></td><td class="colsep0 rowsep0" align="left">[1,2-dihydropyridin-2-one]-4-yl (<i>cis</i>, rac)</td><td class="colsep0 rowsep0" align="char" char="±">8.34 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">5.84 ± 0.07</td><td class="colsep0 rowsep0" align="center">5.46 ± 0.08</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char="±">88.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>77a</b></td><td class="colsep0 rowsep0" align="left">1-Me-[1,2-dihydropyridin-2-one]-5-yl (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">8.76 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">6.06 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.50 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char="±">79.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>78a</b></td><td class="colsep0 rowsep0" align="left">1-Me-[1,2-dihydropyridin-2-one]-4-yl (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">8.23 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">5.65 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.10 ± 0.08</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char="±">79.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>79a</b></td><td class="colsep0 rowsep0" align="left">4-piperidinyl-2-one (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">8.06 ± 0.19</td><td class="colsep0 rowsep0" align="char" char="±">5.87 ± 0.01</td><td class="colsep0 rowsep0" align="center"><5.0</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char="±">88.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>80a</b></td><td class="colsep0 rowsep0" align="left">1-Me-[4-piperidinyl-2-one] (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">7.94 ± 0.15</td><td class="colsep0 rowsep0" align="char" char="±">5.77 ± 0.01</td><td class="colsep0 rowsep0" align="center"><5.0</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char="±">79.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>81a</b></td><td class="colsep0 rowsep0" align="left">[piperidin-4-yl]ethan-1-one (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">8.13 ± 0.21</td><td class="colsep0 rowsep0" align="char" char="±">6.00 ± 0.04</td><td class="colsep0 rowsep0" align="center">5.48 ± 0.07</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char="±">79.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>82a</b></td><td class="colsep0 rowsep0" align="left">4-cyclohexyl-yl-1-acetamide (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">8.56 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="±">6.34 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.16 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">3.9</td><td class="colsep0 rowsep0" align="char" char="±">88.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>83a</b></td><td class="colsep0 rowsep0" align="left">tetrahydropyran-4-yl (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">8.78 ± 0.15</td><td class="colsep0 rowsep0" align="char" char="±">6.09 ± 0.02</td><td class="colsep0 rowsep0" align="center">5.53 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">3.9</td><td class="colsep0 rowsep0" align="char" char="±">68.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>84a</b></td><td class="colsep0 rowsep0" align="left">8-oxabicyclo[3.2.1]octan-3-yl (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">8.49 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">6.13 ± 0.06</td><td class="colsep0 rowsep0" align="center">5.61 ± 0.07</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char="±">68.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>85a</b></td><td class="colsep0 rowsep0" align="left">cyclohexyl (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char="±">9.56 ± 0.17</td><td class="colsep0 rowsep0" align="char" char="±">6.23 ± 0.15</td><td class="colsep0 rowsep0" align="center">6.53 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">5.6</td><td class="colsep0 rowsep0" align="char" char="±">72.1</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup>a</sup><p class="last">se 1= single enantiomer 1; rac = racemate; values shown are the average of three replicates and standard deviation.</p></div></div></div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Affinity Data at the D3R and D2R and Functional Data for Selected Derivatives<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center" char="±">hD3 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="±">hD2 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="±">hD3 fp<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="±">hD2 fp<i>K</i><sub>i</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8b</b></td><td class="colsep0 rowsep0" align="char" char="±">8.26 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">6.10 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">8.18 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">6.17 ± 0.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10a</b></td><td class="colsep0 rowsep0" align="char" char="±">9.20 ± 0.12</td><td class="colsep0 rowsep0" align="char" char="±">6.67 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">9.14 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">7.13 ± 0.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="char" char="±">7.02 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">6.28 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">7.27 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">6.30 ± 0.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char="±">7.34 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">6.01 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">7.67 ± 0.22</td><td class="colsep0 rowsep0" align="char" char="±">5.87 ± 0.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17b</b></td><td class="colsep0 rowsep0" align="char" char="±">9.04 ± 0.12</td><td class="colsep0 rowsep0" align="char" char="±">6.62 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">9.28 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">7.24 ± 0.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19b</b></td><td class="colsep0 rowsep0" align="char" char="±">9.09 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">6.59 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">9.48 ± 0.17</td><td class="colsep0 rowsep0" align="char" char="±">7.18 ± 0.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20b</b></td><td class="colsep0 rowsep0" align="char" char="±">8.21 ± 0.22</td><td class="colsep0 rowsep0" align="char" char="±">6.01 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">8.18 ± 0.24</td><td class="colsep0 rowsep0" align="char" char="±">6.10 ± 0.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21b</b></td><td class="colsep0 rowsep0" align="char" char="±">9.38 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">6.65 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">9.24 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">7.59 ± 0.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="char" char="±">8.50 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">6.89 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">8.84 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="±">6.21 ± 0.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="char" char="±">9.24 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">6.48 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">9.27 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">6.79 ± 0.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40a</b></td><td class="colsep0 rowsep0" align="char" char="±">9.80 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">6.45 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">9.41 ± 0.24</td><td class="colsep0 rowsep0" align="char" char="±">6.37 ± 0.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41a</b></td><td class="colsep0 rowsep0" align="char" char="±">9.33 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">6.20 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">9.25 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">6.47 ± 0.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>47a</b></td><td class="colsep0 rowsep0" align="char" char="±">10.17 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">6.44 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">9.72 ± 0.12</td><td class="colsep0 rowsep0" align="char" char="±">6.17 ± 0.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48a</b></td><td class="colsep0 rowsep0" align="char" char="±">9.74 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">6.30 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">9.44 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">5.93 ± 0.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>54</b></td><td class="colsep0 rowsep0" align="char" char="±">9.10 ± 0.19</td><td class="colsep0 rowsep0" align="char" char="±">6.28 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">9.10 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">6.97 ± 0.14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>65b</b></td><td class="colsep0 rowsep0" align="char" char="±">8.89 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">6.28 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">8.83 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">6.41 ± 0.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>71a</b></td><td class="colsep0 rowsep0" align="char" char="±">9.47 ± 0.18</td><td class="colsep0 rowsep0" align="char" char="±">6.26 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">9.28 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">6.32 ± 0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>73a</b></td><td class="colsep0 rowsep0" align="char" char="±">8.90 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">5.94 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">9.17 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">5.68 ± 0.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>78a</b></td><td class="colsep0 rowsep0" align="char" char="±">8.23 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">5.65 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">8.39 ± 0.18</td><td class="colsep0 rowsep0" align="char" char="±">5.73 ± 0.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>81a</b></td><td class="colsep0 rowsep0" align="char" char="±">8.13 ± 0.21</td><td class="colsep0 rowsep0" align="char" char="±">6.00 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">8.58 ± 0.31</td><td class="colsep0 rowsep0" align="char" char="±">5.93 ± 0.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>83a</b></td><td class="colsep0 rowsep0" align="char" char="±">8.78 ± 0.15</td><td class="colsep0 rowsep0" align="char" char="±">6.09 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">8.83 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">6.16 ± 0.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>84a</b></td><td class="colsep0 rowsep0" align="char" char="±">8.49 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">6.13 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">8.68 ± 0.17</td><td class="colsep0 rowsep0" align="char" char="±">6.27 ± 0.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>85a</b></td><td class="colsep0 rowsep0" align="char" char="±">9.56 ± 0.17</td><td class="colsep0 rowsep0" align="char" char="±">6.23 ± 0.15</td><td class="colsep0 rowsep0" align="char" char="±">9.45 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">6.91 ± 0.08</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup>a</sup><p class="last">Values shown are the average of three replicates and standard deviation.</p></div></div></div><div class="NLM_p">The full removal of the aromaticity from the pyridine ring led to the piperidinyl derivatives <b>79a</b> and <b>80a</b> (<a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>). The compounds both showed affinity at the D3R of about 10 nM with 100-fold selectivity vs the D2R and a 1000-fold selectivity vs the hERG channel (10 μM). The introduction of a lactam ring, especially on derivative <b>79a</b>, reduced cLogP and PSA values, which translated into the CYP P450 profile (IC<sub>50</sub> of <b>79a</b> greater than 21 μM on all isoforms tested). The free fraction (Fu<sub>br</sub> and Fu<sub>bl</sub> = 17.8 and 23%, respectively) and Cl<sub>i</sub> (hCl<sub>i</sub> = 25 and rCl<sub>i</sub> < 11 μL/min/mg protein; <a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>) had benefited from this reduction. Given its low permeability, the compound was not progressed in that phase of the project.</div><div class="NLM_table-wrap" id="tbl9"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 9. Selected in Vitro PK Parameters</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center" char=".">rCl<sub>i</sub><a class="ref internalNav" href="#t9fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">hCl<sub>i</sub><a class="ref internalNav" href="#t9fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">Fu<sub>br</sub> (%)</th><th class="colsep0 rowsep0" align="center" char=".">Fu<sub>bl</sub> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9a</b></td><td class="colsep0 rowsep0" align="char" char=".">161</td><td class="colsep0 rowsep0" align="char" char=".">22</td><td class="colsep0 rowsep0" align="char" char=".">4.8</td><td class="colsep0 rowsep0" align="char" char=".">12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10a</b></td><td class="colsep0 rowsep0" align="char" char=".">83</td><td class="colsep0 rowsep0" align="char" char=".">86</td><td class="colsep0 rowsep0" align="char" char=".">9.5</td><td class="colsep0 rowsep0" align="char" char=".">10.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17b</b></td><td class="colsep0 rowsep0" align="char" char=".">73</td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19b</b></td><td class="colsep0 rowsep0" align="char" char=".">161</td><td class="colsep0 rowsep0" align="char" char=".">53</td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20b</b></td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td><td class="colsep0 rowsep0" align="char" char=".">7.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21b</b></td><td class="colsep0 rowsep0" align="char" char=".">59</td><td class="colsep0 rowsep0" align="char" char=".">107</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td><td class="colsep0 rowsep0" align="char" char=".">5.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22a</b></td><td class="colsep0 rowsep0" align="char" char=".">51</td><td class="colsep0 rowsep0" align="char" char=".">65</td><td class="colsep0 rowsep0" align="char" char=".">8.9</td><td class="colsep0 rowsep0" align="char" char=".">12.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23b</b></td><td class="colsep0 rowsep0" align="char" char=".">151</td><td class="colsep0 rowsep0" align="char" char=".">320</td><td class="colsep0 rowsep0" align="char" char=".">8.6</td><td class="colsep0 rowsep0" align="char" char=".">13.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40a</b></td><td class="colsep0 rowsep0" align="char" char=".">41</td><td class="colsep0 rowsep0" align="char" char=".">155</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td><td class="colsep0 rowsep0" align="char" char=".">14.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41a</b></td><td class="colsep0 rowsep0" align="char" char=".">92</td><td class="colsep0 rowsep0" align="char" char=".">187</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">6.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46a</b></td><td class="colsep0 rowsep0" align="char" char=".">22</td><td class="colsep0 rowsep0" align="char" char=".">58</td><td class="colsep0 rowsep0" align="char" char=".">5.2</td><td class="colsep0 rowsep0" align="char" char=".">14.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>47a</b></td><td class="colsep0 rowsep0" align="char" char=".">276</td><td class="colsep0 rowsep0" align="char" char=".">352</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48a</b></td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">447</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>64b</b></td><td class="colsep0 rowsep0" align="char" char=".">288</td><td class="colsep0 rowsep0" align="char" char=".">485</td><td class="colsep0 rowsep0" align="char" char=".">7.7</td><td class="colsep0 rowsep0" align="char" char=".">16.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>65b</b></td><td class="colsep0 rowsep0" align="char" char=".">253</td><td class="colsep0 rowsep0" align="char" char=".">257</td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>66b</b></td><td class="colsep0 rowsep0" align="char" char=".">75</td><td class="colsep0 rowsep0" align="char" char=".">91</td><td class="colsep0 rowsep0" align="char" char=".">26</td><td class="colsep0 rowsep0" align="char" char=".">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>67a</b></td><td class="colsep0 rowsep0" align="char" char=".">103</td><td class="colsep0 rowsep0" align="char" char=".">80</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td><td class="colsep0 rowsep0" align="char" char=".">18.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>68a</b></td><td class="colsep0 rowsep0" align="char" char=".">114</td><td class="colsep0 rowsep0" align="char" char=".">302</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">7.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>71a</b></td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">106</td><td class="colsep0 rowsep0" align="char" char=".">5.2</td><td class="colsep0 rowsep0" align="char" char=".">18.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>72a</b></td><td class="colsep0 rowsep0" align="char" char=".">98</td><td class="colsep0 rowsep0" align="char" char=".">299</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">12.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>73a</b></td><td class="colsep0 rowsep0" align="char" char=".">38</td><td class="colsep0 rowsep0" align="char" char=".">299</td><td class="colsep0 rowsep0" align="char" char=".">7.8</td><td class="colsep0 rowsep0" align="char" char=".">19.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>74a</b></td><td class="colsep0 rowsep0" align="char" char="."><11</td><td class="colsep0 rowsep0" align="char" char="."><11</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">29.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>75a</b></td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">19.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>77a</b></td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">41</td><td class="colsep0 rowsep0" align="char" char=".">10.8</td><td class="colsep0 rowsep0" align="char" char=".">25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>78a</b></td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">40</td><td class="colsep0 rowsep0" align="char" char=".">14.6</td><td class="colsep0 rowsep0" align="char" char=".">16.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>79a</b></td><td class="colsep0 rowsep0" align="char" char="."><11</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">17.8</td><td class="colsep0 rowsep0" align="char" char=".">23.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>81a</b></td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">61</td><td class="colsep0 rowsep0" align="char" char=".">11.8</td><td class="colsep0 rowsep0" align="char" char=".">20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>83a</b></td><td class="colsep0 rowsep0" align="char" char=".">59</td><td class="colsep0 rowsep0" align="char" char=".">34</td><td class="colsep0 rowsep0" align="char" char=".">9.7</td><td class="colsep0 rowsep0" align="char" char=".">18.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>84a</b></td><td class="colsep0 rowsep0" align="char" char=".">21.3</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char=".">5.6</td><td class="colsep0 rowsep0" align="char" char=".">12.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>85a</b></td><td class="colsep0 rowsep0" align="char" char=".">201</td><td class="colsep0 rowsep0" align="char" char=".">339</td><td class="colsep0 rowsep0" align="char" char=".">0.6</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t9fn1"><div class="footnote" id="t9fn1"><sup>a</sup><p class="last">μL/min/mg protein.</p></div></div></div><div class="NLM_p">To pursue the exploration, the carbonyl group of the lactams (<b>79a</b>, <b>80a</b>, <a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>) was made exocyclic by preparing derivative <b>81a</b>; the system was then further elongated with the cyclohexyl amide <b>82a</b>. Neither of the two corresponding “free” amines were prepared and tested because previous explorations on related compounds<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> and literature data<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> demonstrated that dibasic compounds, especially on lipophilic scaffolds, usually suffer from very high volumes of distribution, long half-life, and high total plasma clearance.</div><div class="NLM_p">Compound <b>81a</b> (<a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>) showed a 7 nM affinity at the D3R, with more than 100-fold selectivity vs the D2R and about 400-fold selectivity vs the hERG channel (3 μM). <b>81a</b> had IC<sub>50</sub> values greater than 17 μM on all CYP P450 isoforms tested, with high free fraction both in brain and blood (Fu<sub>br</sub> and Fu<sub>bl</sub> = 11.8 and 20.0%, respectively); intrinsic clearance values were moderate to low (hCl<sub>i</sub> = 61 and rCl<sub>i</sub> = 17 μL/min/mg protein, respectively, <a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>). A change in the nature of the thiotriazole substituent did not change the functional profile of the compound (<a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>). On the basis of medium cLogP and PSA and medium permeability in the Caco-2 assay, this compound was selected to further assess its PK properties in vivo.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22, 23)</a><b>81a</b> showed good fraction absorbed (<i>F</i><sub>a</sub> = 28%, <a class="ref internalNav" href="#tbl10" aria-label="Table 10">Table 10</a>) with moderate–high distribution volume (<i>V</i><sub>ss</sub> = 6.0 L/kg) and moderate clearance in blood as expected from the in vitro parameters (Cl<sub>b</sub> = 39.8 mL/min/kg). Hepatic extraction was unexpectedly low (<i>E</i><sub>H</sub> = 0.0), leading to moderate half-life (<i>T</i><sub>1/2</sub> = 2.5 h) and a relatively good bioavailability (<i>F</i>% = 33%). Brain penetration was also assessed with a B/B ratio equal to 0.5.</div><div class="NLM_table-wrap" id="tbl10"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 10. Selected in Vivo PK Parameters<a class="ref internalNav" href="#t10fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center"><i>F</i><sub>a</sub> (%)</th><th class="colsep0 rowsep0" align="center"><i>E</i><sub>H</sub></th><th class="colsep0 rowsep0" align="center">Cl<sub>b</sub>(mL/min/kg)</th><th class="colsep0 rowsep0" align="center"><i>V</i><sub>ss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>F</i> (%)</th><th class="colsep0 rowsep0" align="center" char=".">AUC BB ratio</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20b</b></td><td class="colsep0 rowsep0" align="center">70</td><td class="colsep0 rowsep0" align="center">0.31</td><td class="colsep0 rowsep0" align="center">32.9</td><td class="colsep0 rowsep0" align="center">4.9</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">49</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21b</b></td><td class="colsep0 rowsep0" align="center">9</td><td class="colsep0 rowsep0" align="center">0.56</td><td class="colsep0 rowsep0" align="center">15.1</td><td class="colsep0 rowsep0" align="center">0.49</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>71a</b></td><td class="colsep0 rowsep0" align="center">40</td><td class="colsep0 rowsep0" align="center">0.35</td><td class="colsep0 rowsep0" align="center">19.4</td><td class="colsep0 rowsep0" align="center">3.2</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">26</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>73a</b></td><td class="colsep0 rowsep0" align="center">54</td><td class="colsep0 rowsep0" align="center">0.28</td><td class="colsep0 rowsep0" align="center">21.1</td><td class="colsep0 rowsep0" align="center">5.5</td><td class="colsep0 rowsep0" align="char" char=".">4.3</td><td class="colsep0 rowsep0" align="char" char=".">39</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>75a</b></td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">NC</td><td class="colsep0 rowsep0" align="center">48.0</td><td class="colsep0 rowsep0" align="center">9.1</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">NC</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>77a</b></td><td class="colsep0 rowsep0" align="center">17</td><td class="colsep0 rowsep0" align="center">0.4</td><td class="colsep0 rowsep0" align="center">30.2</td><td class="colsep0 rowsep0" align="center">4.0</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>78a</b></td><td class="colsep0 rowsep0" align="center">41</td><td class="colsep0 rowsep0" align="center">0.3</td><td class="colsep0 rowsep0" align="center">34.6</td><td class="colsep0 rowsep0" align="center">3.5</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>81a</b></td><td class="colsep0 rowsep0" align="center">28</td><td class="colsep0 rowsep0" align="center">0.0</td><td class="colsep0 rowsep0" align="center">39.8</td><td class="colsep0 rowsep0" align="center">6.0</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">33</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>83a</b></td><td class="colsep0 rowsep0" align="center">57</td><td class="colsep0 rowsep0" align="center">0.8</td><td class="colsep0 rowsep0" align="center">87.3</td><td class="colsep0 rowsep0" align="center">3.7</td><td class="colsep0 rowsep0" align="char" char=".">0.6</td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>84a</b></td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">80.0</td><td class="colsep0 rowsep0" align="center">2.6</td><td class="colsep0 rowsep0" align="char" char=".">NT</td><td class="colsep0 rowsep0" align="char" char=".">NT</td><td class="colsep0 rowsep0" align="char" char=".">NT</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t10fn1"><div class="footnote" id="t10fn1"><sup>a</sup><p class="last">NT = not tested; NC = not calculated.</p></div></div></div><div class="NLM_p">The acetamide <b>82a</b> (<a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>) was well tolerated by the D3R, showing an affinity of about 3 nM at the D3R and a very good selectivity vs the hERG channel (2500-fold). Although lipophilicity was not significantly increased, IC<sub>50</sub> values on the CYP P450 2D6 decreased with respect to the previous compounds (IC<sub>50</sub> = 2 μM; all other isoforms >6 μM).</div><div class="NLM_p">Following the observation that hydrophilicity was well tolerated in this region of the receptor, the piperidine moiety was replaced by a tetrahydropyrane template (<b>83a</b>). This compound showed an excellent affinity at the D3R (1.7 nM), 480-fold selectivity vs the D2R, and 1700-fold selectivity vs the hERG channel. Marked reduction in PSA and high permeability in Caco-2 cell lines were observed. The affinity of <b>83a</b> was also confirmed functionally (<a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>), with no activity at both the D1R and the D4R. A good selectivity over muscarinic M1 and M3 receptors (>100-fold) was also achieved (fp<i>K</i><sub>i</sub> = 6.60 ± 0.09 and 6.05 ± 0.06 on M1 and M3, respectively, <a class="ref internalNav" href="#tbl11" aria-label="Table 11">Table 11</a>). The CYP P450 profile was excellent, with IC<sub>50</sub> values greater than 10 μM on all isoforms tested; free fraction was high (Fu<sub>br</sub> and Fu<sub>bl</sub> = 9.7 and 18.8%, respectively, <a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>). Intrinsic clearance values in rat were higher than the ones in humans (hCl<sub>i</sub> = 34 and rCl<sub>i</sub> = 59 μL/min/mg protein, respectively). Therefore, the compound was selected to profile its PK properties in vivo.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22, 23)</a><b>83a</b> had good fraction absorbed (<i>F</i><sub>a</sub> = 57%, <a class="ref internalNav" href="#tbl10" aria-label="Table 10">Table 10</a>), with moderate distribution volume (<i>V</i><sub>ss</sub> = 3.7 L/kg) and high clearance in blood as expected from the in vitro parameters (Cl<sub>b</sub> = 87.3 mL/min/kg). Hepatic extraction was high (<i>E</i><sub>H</sub> = 0.8), leading to short half-life (<i>T</i><sub>1/2</sub> = 0.6 h) and low bioavailability (<i>F</i>% = 11%). Brain penetration was also assessed with B/B ratio equal to 0.3, potentially dictated by decreased PSA.</div><div class="NLM_table-wrap" id="tbl11"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 11. Affinity Data at the D3R and D2R, Muscarinic M1 and M3 Receptors, and hERG Channel for Selected Derivatives<a class="ref internalNav" href="#t11fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="center" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center" char="±">hD3 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="±">hD2 p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center">hERG</th><th class="colsep0 rowsep0" align="center" char="±">M1 fp<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center" char="±">M3 fp<i>K</i><sub>i</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8b</b></td><td class="colsep0 rowsep0" align="char" char="±">8.26 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">6.10 ± 0.11</td><td class="colsep0 rowsep0" align="center">5.33 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">6.13 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">5.69 ± 0.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9a</b></td><td class="colsep0 rowsep0" align="char" char="±">8.40 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">6.05 ± 0.08</td><td class="colsep0 rowsep0" align="center">6.72 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="±">6.32 ± 0.15</td><td class="colsep0 rowsep0" align="char" char="±">5.94 ± 0.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10a</b></td><td class="colsep0 rowsep0" align="char" char="±">9.20 ± 0.12</td><td class="colsep0 rowsep0" align="char" char="±">6.67 ± 0.04</td><td class="colsep0 rowsep0" align="center">6.39 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">6.51 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">6.56 ± 0.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20b</b></td><td class="colsep0 rowsep0" align="char" char="±">8.21 ± 0.22</td><td class="colsep0 rowsep0" align="char" char="±"><i>6.10 ± 0.10</i></td><td class="colsep0 rowsep0" align="center">6.35 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">6.86 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">6.26 ± 0.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21b</b></td><td class="colsep0 rowsep0" align="char" char="±">9.38 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">6.65 ± 0.04</td><td class="colsep0 rowsep0" align="center">6.65 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">6.49 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">6.37 ± 0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22a</b></td><td class="colsep0 rowsep0" align="char" char="±">8.98 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="±">6.78 ± 0.02</td><td class="colsep0 rowsep0" align="center">5.92 ± 0.19</td><td class="colsep0 rowsep0" align="char" char="±">6.25 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">5.68 ± 0.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23b</b></td><td class="colsep0 rowsep0" align="char" char="±">8.17 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">5.89 ± 0.03</td><td class="colsep0 rowsep0" align="center">5.57 ± 0.24</td><td class="colsep0 rowsep0" align="char" char="±">6.41 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">5.83 ± 0.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="char" char="±">8.50 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">6.89 ± 0.14</td><td class="colsep0 rowsep0" align="center"><5.0</td><td class="colsep0 rowsep0" align="char" char="±">6.77 ± 0.18</td><td class="colsep0 rowsep0" align="char" char="±">6.56 ± 0.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40a</b></td><td class="colsep0 rowsep0" align="char" char="±">9.80 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">6.45 ± 0.01</td><td class="colsep0 rowsep0" align="center">6.09 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">6.34 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">6.37 ± 0.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41a</b></td><td class="colsep0 rowsep0" align="char" char="±">9.33 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">6.20 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.55 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">6.58 ± 0.16</td><td class="colsep0 rowsep0" align="char" char="±">6.42 ± 0.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>47a</b></td><td class="colsep0 rowsep0" align="char" char="±">10.17 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">6.44 ± 0.01</td><td class="colsep0 rowsep0" align="center">6.82 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">6.17 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">6.52 ± 0.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>67a</b></td><td class="colsep0 rowsep0" align="char" char="±">8.48 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="±">5.83 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.20 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">5.78 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">5.91 ± 0.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>71a</b></td><td class="colsep0 rowsep0" align="char" char="±">9.47 ± 0.18</td><td class="colsep0 rowsep0" align="char" char="±">6.26 ± 0.03</td><td class="colsep0 rowsep0" align="center">6.18 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="±">6.65 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">6.72 ± 0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>73a</b></td><td class="colsep0 rowsep0" align="char" char="±">8.90 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">5.94 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.56 ± 0.17</td><td class="colsep0 rowsep0" align="char" char="±">6.29 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">6.73 ± 0.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>75a</b></td><td class="colsep0 rowsep0" align="char" char="±">8.66 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">6.47 ± 0.04</td><td class="colsep0 rowsep0" align="center">5.02 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">6.31 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">6.20 ± 0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>76</b></td><td class="colsep0 rowsep0" align="char" char="±">8.34 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">5.84 ± 0.07</td><td class="colsep0 rowsep0" align="center">5.46 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">6.43 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">6.17 ± 0.22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>78a</b></td><td class="colsep0 rowsep0" align="char" char="±">8.23 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">5.65 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.10 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">6.52 ± 0.18</td><td class="colsep0 rowsep0" align="char" char="±">6.45 ± 0.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>81a</b></td><td class="colsep0 rowsep0" align="char" char="±">8.13 ± 0.21</td><td class="colsep0 rowsep0" align="char" char="±">6.00 ± 0.04</td><td class="colsep0 rowsep0" align="center">5.48 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">7.18 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">6.17 ± 0.22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>82a</b></td><td class="colsep0 rowsep0" align="char" char="±">8.56 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="±">6.34 ± 0.01</td><td class="colsep0 rowsep0" align="center">5.16 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">6.76 ± 0.18</td><td class="colsep0 rowsep0" align="char" char="±">6.64 ± 0.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>83a</b></td><td class="colsep0 rowsep0" align="char" char="±">8.78 ± 0.15</td><td class="colsep0 rowsep0" align="char" char="±">6.09 ± 0.02</td><td class="colsep0 rowsep0" align="center">5.53 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">6.60 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">6.05 ± 0.06</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t11fn1"><div class="footnote" id="t11fn1"><sup>a</sup><p class="last">Values shown are the average of three replicates and standard deviation.</p></div></div></div><div class="NLM_p">In an attempt to reduce Clb values masking the carbons alpha to the oxygen to prevent their oxidation, the bridged derivative <b>84a</b> (<a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>) was prepared. No major differences were observed in the affinity/selectivity profile, while permeability was slightly lower. The free fraction was also slightly reduced, and the rCl<sub>i</sub> was reduced as expected (from 59 to 21 μL/min/mg protein, <a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>). To quickly test the working hypothesis, the compound was administered intravenously (iv) in vivo<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22, 23)</a> to assess blood clearance in rat. Unfortunately, the difference in Cl<sub>i</sub> was not reflected in a difference in Clb and the values for the two compounds were comparable (<a class="ref internalNav" href="#tbl10" aria-label="Table 10">Table 10</a>) and relatively high.</div><div class="NLM_p">Finally, to complete the exploration, the oxygen of <b>83a</b> was replaced by a carbon atom, leading to the cyclohexyl derivative <b>85a</b> (<a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>). A substantial increase in cLogP was calculated and translated into an increase in affinity at the DA D3 receptor. As expected, such increase was also observed on the off-targets even if an excellent level of selectivity was still maintained (more than 2000-fold vs the D2R and 1000-fold vs hERG). The high lipophilicity led to reduced free fraction and very high Cl<sub>i</sub> (<a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>) as expected for such derivative. For both <b>84a</b> and <b>85a</b>, functional data were generated too (<a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>), confirming the antagonist profile of the series.</div><div class="NLM_p">Some of the compounds reported in <a class="ref internalNav" href="#tbl6" aria-label="Tables 6">Tables 6</a> and <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a> were submitted to an “ExpresSProfile-P1” at CEREP,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> a panel consisting of 55 receptors, ion channels, and transporters at the 1 μM concentration. A great selectivity was observed for the compounds tested on the majority of the off-targets (data not reported) and the previously identified hERG/M1/M3 potential liability points were also highlighted, confirming their correct inclusion in the screening cascade.</div><div class="NLM_p">Finally, to attribute unequivocally the stereochemistry of the compounds and to revise the results of the computational work, an appropriate strategy was set up to determine their absolute configuration. The strategy was divided into two parts and comprised as a first step the preparation of the chiral intermediate, (1<i>R</i>,3<i>S</i>)-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>86</b>), by preparation of (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane and resolution of the racemic mixture by using chiral HPLC procedure as reported in the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>. The assignment of the absolute configuration of the title compound was determined by a single-crystal X-ray structure obtained from a crystal of 5-(4-methylbenzenesulfonyl)-(1<i>R</i>,3<i>S</i>)-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>87</b>) derived from the desired enantiomer and crystallized in EtOH as solvent to obtain a single crystal. For those compounds synthesized from <b>86</b> or from its (1<i>S</i>,3<i>R</i>)-enantiomer (<b>87</b>) (with known absolute stereochemistry based on X-ray structure), a common trend was recognized between absolute configuration of the 5-azaspiro[2.4]heptane moiety and measured binding activity at the D3R for each pair of enantiomers (i.e., the most active enantiomer of each pair of enantiomers generally does have the same absolute configuration). <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a> reports the crystal structures for derivatives <b>88</b> and <b>89</b>.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Left: asymmetric unit of 5-(4-methylbenzenesulfonyl)-(1<i>R</i>,3<i>S</i>)-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>88</b>) comprising four molecular entities (I–IV), right: molecular structure of (<b>88</b>), with thermal ellipsoids drawn at the 50% probability level. (B) Left: asymmetric unit of 5-(4-methylbenzenesulfonyl)-(1<i>S</i>,3<i>R</i>)-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>89</b>) comprising four molecular entities (I–IV). Right: molecular structure of (<b>89</b>), with thermal ellipsoids drawn at the 50% probability level.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00972&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Given the results achieved in this early part of the exploration and reported in <a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a>–<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a>, some hints about the structure–activity of the new 5-azaspiro[2.4]heptane moiety were identified. On this specific scaffold, and coupled to the thiotriazole system to provide D3R antagonism, the relative configuration of the system played a role in determining the affinity at the primary target. Both stereoisomers were active and selective, and the <i>cis</i> stereoisomer resulted often more active than the <i>trans</i> stereoisomer (e.g., <b>10a</b> vs <b>9a</b>; <b>19b</b> vs <b>18b</b>; <b>21b</b> vs <b>20a</b>). Nonetheless, the steric hindrance of the substituents on the aromatic ring of this amino portion also played an important role on the affinity and was able to reverse this general trend (e.g., <b>22a</b> vs <b>23b</b>; <b>14</b> vs <b>15</b>; <b>25</b> vs <b>26</b>). Even if this was not specifically noticed in the molecular modeling calculations performed to support the medicinal chemists, this fact might suggest that a specific dihedral angle in the <i>cis</i> configuration is required for a more appropriate fitting of the whole molecule within the primary binding site in the D3R pocket.</div><div class="NLM_p">An additional observation was related to the very high level of selectivity achieved for all the derivatives prepared in the different series. Differently from other series reported in the literature, a selectivity greater than 1000-fold over the hERG channel was observed for many of the compounds reported in the manuscript. High cLogP values (e.g., <b>33</b>, <b>44a</b>, <b>45a</b>, <b>47a</b>) led to subnanomolar affinity at the D3R but also caused an increase of potency at the hERG channel. Nonetheless, an optimal range of cLogP to achieve both high D3R affinity and an “ideal” selectivity window with respect to the hERG channel was identified (3 < cLogP < 4.5), also in agreement with previously reported data.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">The introduction of additional H-bond capabilities in the molecule (e.g., <b>40a</b>, <b>44a</b>, <b>47a</b>) led to an increase in the affinity at the D3R, suggesting the possibility of additional interactions within the secondary binding pocket (SBP)<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> of the D3R, potentially in the region facing the extracellular loop 2 (EL2) as it might be hypothesized by the docking of compound <b>40a</b> reported in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>.</div><div class="NLM_p last">The presence in the scaffold of an additional moderately basic center (<b>49</b>–<b>74</b>) was generally well tolerated from the affinity point of view and had no major impact on the distribution volume point of view. This, in addition to the previously mentioned H-bond acceptor presence and to appropriate protection of the molecule from the metabolic point of view, led to a series of derivatives with good CYP P450 profile and excellent developability properties (<a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a> and <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a>).</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70124" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70124" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In summary, a combined medicinal chemistry and computational chemistry strategy led to the identification of a new series of 1,2,4-triazolyl 5-azaspiro[2.4]heptanes with high affinity and selectivity at the D3R.</div><div class="NLM_p">Despite the limited exploration of substituents in this early phase of the project, the newly identified “amino portion” provided a well-defined SAR. The affinity at the D3R reached subnanomolar values (e.g., <b>47a</b>) with a large selectivity window, both on the D2R but especially on the hERG channel (e.g., <b>75a</b>, 5000-fold selective, or <b>46a</b>, 10000-fold selective).</div><div class="NLM_p last">The observed pharmacokinetic properties (both in vitro and in vivo) clearly predict good developability characteristics for these molecules even though additional optimization work together with further in vivo testing is now required. Some selected derivatives, which showed an overall balanced in vitro/in vivo profile (e.g., <b>20b</b>, <b>21b</b>, <b>71a</b>, <b>73a</b>, <b>77a</b>, <b>78a</b>, <b>81a</b>, and <b>83a</b>), were considered for further progression along the screening cascade for subsequent lead optimization, which will be reported in due course.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43802" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43802" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Biological Test Methods: In Vitro Studies</h3><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> [<sup>3</sup>H]-Spiperone Filtration Binding Assay on Membranes from hD3-CHO Cells</h4><div class="NLM_p last">This assay provided in vitro affinities (pIC<sub>50</sub>, p<i>K</i><sub>i</sub>) of compounds at the human D3R expressed in CHO-K1 cells. The filtration binding involved the incubation of hD3-CHO cell membranes with the radioligand [<sup>3</sup>H]-spiperone in the presence or absence of test compounds. Displacement binding was measured by separation of bound from free radioligand using filtration through glass microfiber filters. The assay was performed at room temperature (23 °C). Equilibrium was reached after 80 min and remained stable for at least 120 min. Bound radioactivity was measured using a Top-Count reader (for competition experiments) or β-Counter reader for saturation experiments), while radioligand concentration was determined by liquid scintillation counting.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Expression Plasmids</h4><div class="NLM_p last">The human D3R (DRD3), transcript variant (<a href="https://www.ncbi.nlm.nih.gov/gquery?Term=NM_000796.2">NM_000796.2</a>), was purchased from OriGene Technologies, Inc. (catalogue no. SC124083). The cDNA was cloned into pCMV6-XL4 plasmid vector containing CMV promoter for in vivo expression in mammalian cells. The molecular clone sequence data was matched to the reference identifier above as a point of reference. The complete sequence of the molecular clone may differ from the sequence published for this corresponding reference, e.g., by representing an alternative RNA splicing form or single nucleotide polymorphism (SNP).</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Large Scale Cell Transfection and Cell Pellet Preparation</h4><div class="NLM_p last">CHO-K1 cells were maintained in F12K medium supplemented with 10% dialyzed fetal bovine serum in a humidified incubator at 37 °C with 5% CO<sub>2</sub>. For transfection, cells were plated in Corning Dish 500 cm<sup>2</sup> (20 × 106 cells/dish) and cultured overnight. Cells were then transfected with hDRD3-pCMV6-XL4 plasmid, following the manufacture’s protocol with FuGENE HD with a ratio of transfection complex of 3:1 (300 μL of FuGENE: 100 μg plasmid/dish). After 24 h incubation at 37 °C and 5% CO<sub>2</sub>, the medium was removed and cells were washed with 25 mL of DPBS (no Ca<sup>2+</sup>, no Mg<sup>2+</sup>). Versene was added over the cells and incubated at 37 °C for 5 min. Cells were scraped with a “cell scraper”, and the cell suspension was collected in a Costar tubes prior to being put on ice. Cell suspensions were centrifuged in a Beckman GS6R centrifuge for 10 min, 4 °C, 1200 rpm (330 rcf), the supernatant removed and the cell pellets washed and collected in a single tube by resuspension in PBS and centrifugation cycles. The pellet was weighed and frozen down at −80 °C.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> CHO-hDRD3 Membrane Preparation</h4><div class="NLM_p last">The frozen pellet was thawed and homogenized in 10 volumes (w/v) in Membrane Preparation Buffer 1 using an Ultraturrax (3 times for 10 s each cycle). The homogenate was centrifuged for 20 min, 4 °C, at 18500 rpm (40000 rcf) in a SL-50T Sorvall rotor and the pellet resuspended in 10 volumes w/v in Membrane Preparation Buffer 1 and rehomogenized as before. After centrifugation, the pellet was resuspended in 5 volumes of Membrane Preparation Buffer 2. The resulting suspension was aliquoted and frozen down at −80 °C. Protein concentration was determined according to the instructions provided within the BioRad reagent using a BSA standard curve (Buffer 1 = HEPES 20 mM, EDTA 2 mM pH 7.4, ice cold; Buffer 2 = HEPES 20 mM pH 7.4, NaCl 100 mM, MgCl<sub>2</sub> 10 mM, EDTA 1 mM, ice cold).</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Competition Binding Experiments</h4><div class="NLM_p last">Competition binding experiments were performed in a 96-deep well plate at room temperature (23 °C) with a final assay volume of 500 μL/well according to the following protocol: (a) 300 μL of binding buffer were dispensed into each well of the compound plate; (b) [<sup>3</sup>H]-spiperone stock was diluted in binding buffer solution to obtain the 5× [<sup>3</sup>H]-spiperone solution (1.5 nM); (c) 100 μL of 1.5 nM [<sup>3</sup>H]-spiperone solution were dispensed into each well of the compound plate; (d) the competition reaction was started by adding 100 μL of hDRD3-CHO membrane suspension in binding buffer. The final membrane concentration/well was 3.5 μg, and the final [<sup>3</sup>H]-spiperone concentration was 0.3 nM. The plate was then incubated on a shaker at 23 °C for 90 min. The reaction was terminated by rapid filtration through Unifilter-96 GF/B filter plates presoaked for 1 h in polyethylenimine (PEI) 0.5% (w/v) solution and washed with 1.0 mL of ice-cold 0.9% NaCl before the filtration using a Packard Cell Harvester. The filter plate was washed 4 times with 1.0 mL of ice-cold 0.9% NaCl and then left to dry for at least 1 h at 40 °C. The plate was sealed with a back-seal; 50 μL of Microscint-20 were added to each well, and the plate was sealed with a top-seal. Bound radioactivity was measured using a Microplate TopCount. Radioligand concentration was determined as follows: 100 μL of [<sup>3</sup>H]-spiperone solution (5×) and 3 mL of Filter Count were mixed in the total added vial and read in β-Counter TriCarb 2900.</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Saturation Binding Experiments</h4><div class="NLM_p last">Saturation binding experiments were performed similarly to the competition binding experiments, with the following deviation: [<sup>3</sup>H]-spiperone concentrations were chosen from 0.015 to 4.0 nM in a concentration response curve with 12 points. The reaction was terminated by rapid filtration through GF\B paper filter presoaked for 1 h in polyethylenimine (PEI) 0.5% (w/v) solution and washed with 1.0 mL of ice-cold 0.9% NaCl before the filtration using a Brandel harvester. The filter was washed 4 times with 1.0 mL of ice-cold 0.9% NaCl. The filter was put into a pico-vial (PerkinElmer 600252), and 4 mL of Filter count were added. Bound radioactivity was measured using a β-Counter. Samples of working radioligand solution were taken and measured by traditional liquid scintillation counting in order to determine the actual concentration of radiolabel added.</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> [<sup>3</sup>H]-Spiperone Filtration Binding Assay on Membranes from hD2-CHO Cells</h4><div class="NLM_p last">This assay provided in vitro affinities (pIC<sub>50</sub>, p<i>K</i><sub>i</sub>) of compounds at the human DA D2 receptor expressed in CHO-K1 cells. The filtration binding involved the incubation of hD2-Gα16-CHO cell membranes with the radioligand [<sup>3</sup>H]-spiperone in the presence or absence of test compounds. Displacement binding was measured by separation of bound from free radioligand using filtration through Glass Microfiber Filterplates. The assay was performed at room temperature (23 °C). Equilibrium was reached after 90 min and remained stable for at least 150 min. Bound radioactivity was measured using a Top-Count reader (for competition experiments) or s-Counter reader (for saturation experiments), while radioligand concentration was determined by liquid scintillation counting.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Competition Binding Experiments</h4><div class="NLM_p last">Competition binding experiments were performed in a 96-deep-well plate at room temperature (23 °C) with a final assay volume of 1000 μL/well, according to the following protocol: 800 μL of binding buffer were dispensed into each well of the compound plate. [<sup>3</sup>H]-Spiperone stock was diluted in binding buffer solution to obtain the 10× [<sup>3</sup>H]-spiperone solution (0.8 nM). Then 100 μL of 0.8 nM [3H]-spiperone solution were dispensed into each well of the compound plate. The competition reaction was started by adding 100 μL of hD2-Gα16-CHO membrane suspension in binding buffer. The final membrane concentration/well was 2 μg, and the final [<sup>3</sup>H]-spiperone concentration was 0.08 nM. The plate was then incubated on a shaker at 23 °C for 120 min. The reaction was terminated by rapid filtration through Unifilter-96 GF/B filter plates presoaked for at least 1 h in polyethylenimine (PEI) 0.5% (w/v) solution and washed with 1.0 mL of ice cold 0.9% NaCl before the filtration using a Packard Cell Harvester. The filter plate was washed 4 times with 1.0 mL of ice-cold 0.9% NaCl and then left to dry for at least 1 h at 40 °C. The plate was sealed with a back-seal; 50 μL of Microscint-20 were added to each well, and the plate was sealed with a top-seal. Bound radioactivity was measured using a Microplate Top-Count. Radioligand concentration was determined as follows: 100 μL of [<sup>3</sup>H]-spiperone solution (5×) and 3 mL of Filter Count were mixed in the total added vial and read in β-Counter TriCarb 2900.</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Saturation Binding Experiments</h4><div class="NLM_p last">Saturation binding experiments were performed similarly to the competition binding experiments, with the following deviation: [<sup>3</sup>H]-spiperone concentrations were chosen from 0.011 to 3.0 nM in a concentration–response curve with 12 points. The reaction was terminated by rapid filtration through GF/B paper filter presoaked for 1 h in polyethylenimine (PEI) 0.5% (w/v) solution and washed with 1.0 mL of ice-cold 0.9% NaCl before the filtration using a Brandel harvester. The filter was washed 4 times with 1.0 mL of ice-cold 0.9% NaCl. The filter was put into a pico-vial (PerkinElmer 600252), and 4 mL of Filter Count were added. Bound radioactivity was measured using a β-Counter. Samples of working radioligand solution were taken and measured by traditional liquid scintillation counting in order to determine the actual concentration of radiolabel added.</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Functional Calcium Assay at hD2 Recombinant Receptor</h4><div class="NLM_p last">CHO cells stably expressing human DA receptor type 2, long variant (hD2L), coupled to Gα16 protein (CHO-Gα16-hD2L), were seeded into black-walled clear-base 384-well plates at a density of 8000 cells per well and grown overnight at 37 °C. After washing with the assay buffer (20 mM HEPES, 145 mM NaCl, 5 mM KCl, 5.5 mM glucose, 1 mM MgCl<sub>2</sub>, and 2 mM CaCl<sub>2</sub>, pH 7.4) containing 2.5 mM probenecid, cells were incubated with the cytoplasmic Ca<sup>2+</sup> probe Fluo-4AM at 1 μM (final concentration), 37 °C for 60 min. Plates were washed three times as above and placed into a fluorometric imaging plate reader (FLIPR Tetra, Molecular Devices) to monitor cell fluorescence (ex = 470–495 nm, em = 515–575 nm) before and after the addition of different concentrations of test compounds. Test compounds were dissolved in DMSO and 200-fold diluted with assay buffer plus 0.01% Pluronic F-127. Cells were exposed first to test compounds for 10 min, then to a submaximal concentration of the hD2 receptor agonist dopamine (EC<sub>80</sub>, 50–140 nM). The fluorescence before compound addition (baseline) and before and after addition of agonist challenge was monitored. The peak of Ca<sup>2+</sup> stimulation (baseline subtracted) was plotted versus the concentration of test compound and the curve fitted using a four-parameter logistic equation (XLfit) to assess the agonist/antagonist potency and maximal response.</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Functional Binding Assay in Cell Membranes Expressing hD3 Receptors: [<sup>35</sup>S] GTPγS</h4><div class="NLM_p last">In vitro functional studies were performed according to the following [<sup>35</sup>S]-GTPγS protocol. Test compounds were serially diluted 1:3 or 1:4 in neat DMSO, 11-point curves; the final top concentration of test drugs in the assay was 10 μM, 1% DMSO; 0.5 μL of test compound serial dilution were dispensed into the assay plate (white solid 384-well plate format). A suspension containing 3 mg/mL PS-WGA imaging beads (PerkinElmer, RPNQ0260), 0.01% Pluronic F127, 30 μg/mL saponin, 1 μg/mL GDP, and 3.5 μg/mL membranes (PerkinElmer, cat.no ES-173-M400UA) was prepared. Then 0.5 nM [35S]-GTPγS was added to the mixture just prior to plate dispensing of 50 μL per well. In the case of the antagonist mode, 3 nM of quinelorane (EC80) was also added to the SPA mixture. The assay plate was then sealed and centrifuged for 1 min at 800 rpm. After 90 min incubation at room temperature the plate was counted on a Viewlux reader with 613 nm filter, 6× binning for 10 min. Concentration response curves were analyzed using GraphPad or Xlfit for Excel; for agonist mode the pEC<sub>50</sub> was obtained, while for the antagonist mode the fp<i>K</i><sub>i</sub> values of test drug were calculated from the IC<sub>50</sub> using Cheng and Prusoff equation.</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Measure of the Effect on hERG Channel by Tail Current Recording Using in Vitro Rapid ICE</h4><div class="NLM_p">The potency of the compounds in inhibiting human ERG potassium channel (hERG) tail current was assessed in a recombinant HEK293 cell line stably transfected with hERG cDNA under an inducible promoter, using Rapid ICE (rapid ion channel electrophysiology) assay. Rapid ICE is an automated patch-clamp assay utilizing the QPatch HTX system (Sophion Bioscience A/S). Briefly, inducible HEK hERG cells were cultivated in minimum essential medium supplemented with 10% FBS, 1% nonessential amino acids, 1% sodium pyruvate, 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine, 15 μg/mL blasticidin, and 100 μg/mL hygromycin. hERG channel expression induction was obtained by adding 10 μg/mL tetracycline for 24, 48, or 72 h before recordings.</div><div class="NLM_p last">On the day of the experiment, cells were detached with TrypLE and prepared to be loaded on the instrument. Cells were resuspended in 7 mL of Serum-Free Media containing 25 mM Hepes and soybean trypsin inhibitor and immediately placed in the cell storage tank of the machine. The composition of the extracellular buffer was (mM): NaCl 137, KCl<sub>4</sub>, CaCl<sub>2</sub> 1.8, MgCl<sub>2</sub> 1.0, <span class="smallcaps smallerCapital">d</span>-glucose 10, <i>N</i>-2-hydroxyethylpiperazine-<i>N</i>′-2-ethanesulfonic acid (HEPES) 10, pH 7.4 with 1 M NaOH. The composition of the pipet solution was (mM): KCl 130, MgCl<sub>2</sub> 1.0, ethylene glycol-bis(β-aminoethyl ether)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetraacetic acid (EGTA) 5, MgATP 5, HEPES 10, pH 7.2 with 1 M KOH. The voltage protocol included the following steps: step from −80 to −50 mV for 200 ms, +20 mV for 4.8 s, step to −50 mV for 5 s, then step to the holding potential of −80 mV. Compounds were dissolved in DMSO and diluted in extracellular buffer to achieve final test concentrations (0.1, 1, and 10 μM) in 0.1% DMSO. The voltage protocol was run and recorded continuously during the experiment. The vehicle, corresponding to 0.1% DMSO in extracellular buffer was then applied for 3 min, followed by the test substance in triplicate. The standard combined exposure time was 5 min. The average of tail current amplitude values recorded from four sequential voltage pulses was used to calculate for each cell the effect of the test substance by calculating the residual current (% control) compared with vehicle pretreatment. Data were reported as % inhibition for each concentration tested, and IC<sub>50</sub> values were estimated using QPatch software. At least two cells were tested, and even more if results diverged.</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Muscarinic M1 and M3 Receptor: FLIPR Assay</h4><div class="NLM_p last">The recombinant human jmuscarinic M1 receptor cell line (389 Aptuit CHO h-M1) was generated in-house and pharmacologically validated with reference compounds. Briefly, CHO-KI cells were stably transfected with an expression plasmid encoding for the human muscarinic M1 receptor (GeneBank accession no. <a href="https://www.ncbi.nlm.nih.gov/gquery?Term=NM_000738">NM_000738</a>) following a standard transfection protocol with FuGENE HD (Roche Diagnostic). CHO Muscarinic M1 cells were cultured in a humidified incubator with 5% CO2 in F12K medium containing 10% FBS HI and 450 μg/mL of geneticin (G418). The cells were grown in T175 flasks and split when 90–100% confluence was reached. The recombinant human muscarinic M3 receptor cell line (387 Aptuit CHO h-M3) was generated in-house and pharmacologically validated with reference compounds. Briefly, CHO-KI cells were stably transfected with an expression plasmid encoding for the human muscarinic M3 receptor (GeneBank accession no. <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=NM_000740">NM_000740</a>) following a standard transfection protocol with FuGENE HD (Roche Diagnostic). CHO muscarinic M3 cells were cultured in a humidified incubator with 5% CO<sub>2</sub> in F12K medium containing 10% FBS HI and 450 ìg/mL of geneticin (G418). The cells were grown in T175 flasks and split when 90–100% confluence was reached. This assay provided in vitro agonist (pEC<sub>50</sub>) and antagonist (pIC<sub>50</sub>) potency of compounds at the human M1 and M3 receptor stably expressed in CHO cells by modulating intracellular calcium changes using the calcium-sensitive dye Fluo-4AM and fluorometric imaging plate reader (FLIPR, Molecular Devices) instrument. A dual read-out FLIPR protocol was applied, allowing for agonist and antagonist characterization. In the first part, compounds were characterized in a concentration dependent manner (11 point CRC, 1:3 serial dilutions) for their ability to increase intracellular calcium levels with respect to the agonist standard acetylcholine and a pEC<sub>50</sub> calculated. After 10 min of incubation, a second addition containing the EC<sub>80</sub> of acetylcholine followed. Inhibition of the acetylcholine evoked signal indicated antagonist activity of the compound and enabled the calculation of the compounds pIC<sub>50</sub>. A quality check (QC) was present in each compound plate. The signal window was monitored by performing Z′ calculations and evaluating the pharmacology of internal standards. As agonist reference acetylcholine and carbachol were characterized, while atropine and imipramine served as standards in the antagonist mode. Concentration response curves of compounds were run in duplicate from the same compound stock solution.</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Compound Plate Preparation for First FLIPR Addition</h4><div class="NLM_p last">Serial dilutions 1–3 were performed from a compound stock solution by Biomek FX to generate an 11-point concentration curve (CRC). Then 1 μL copy plates were stamped into V-bottom drug plates at a concentration which was 200-fold the final assay concentration. The copy plates were diluted prior to the experiment with assay buffer containing 0.05% Pluronic F-127 to reach four times the final assay concentration (4×, 2% DMSO). The concentration corresponding to the EC<sub>80</sub> CRCs for acetylcholine-induced intracellular Ca<sup>2+</sup> increase was measured during the first FLIPR addition. The EC<sub>80</sub> corresponded to 4 times the EC<sub>50</sub> if the Hill slope was not different from 1.</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Compound Plate Preparation for Second FLIPR Addition</h4><div class="NLM_p last">Acetylcholine stock in DMSO was diluted in assay buffer containing 0.05% Pluronic F-127 to obtain an agonist stimulus solution (DMSO < 0.02%) corresponding to 5-fold the final EC<sub>80</sub> concentration.</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Intracellular Calcium Response Measurement</h4><div class="NLM_p">The cells were washed in washing buffer using the EMBLA 384 instrument, leaving 20 μL/well of buffer after the final aspiration. Cells were then incubated in washing buffer with the cytoplasmic Ca<sup>2+</sup> indicator Fluo-4AM at 2 μM and 0.02% Pluronic F-127 final concentrations for 1 h in a 37 °C incubator (cell loading). After cells were washed three times in washing buffer using the EMBLA 384 instrument, 30 μL of buffer was left in each well at the end of the last wash. The loaded cell plates were transferred to the FLIPR machine and calcium response monitored during the 2 online addition protocols:<ul class="NLM_list-list_type-bullet"><li><p class="inline">First addition: 10 μL/well of the test compound (4× concentration) or buffer were added to the loaded cells, and Ca<sup>2+</sup> mobilization response was measured.</p></li><li><p class="inline">Second addition: after 10 min, 10 μL/well of acetylcholine EC<sub>80</sub> (5× concentration) were added as agonist challenge, and Ca<sup>2+</sup> mobilization responses were followed on line.</p></li></ul></div><div class="NLM_p last">The final DMSO concentration in the assay was 0.5% w/v after the two FLIPR additions.</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> P450 CYPEX Assay</h3><div class="NLM_p">The described assays measured in vitro inhibitory effects (pIC<sub>50</sub>) of compounds at the human P450 isoforms (1A2, 2C9, 2C19, 2D6, 3A4) expressed in recombinant microsomes. Pro-fluorescent probe substrates were metabolized to fluorescent products by the enzymes. Fluorescence was measured in kinetic mode (1 read/min for 10 min), and the fluorescence rate was calculated. Data were normalized to controls: DMSO represented 0% effect (i.e., no inhibition), while 10 μM miconazole was set to 100% effect (i.e., complete inhibition). Inhibition of the fluorescent signal determined by P450 activation indicated antagonist activity of the compound and allowed for the calculation of the compound pIC<sub>50</sub>.</div><div class="NLM_p last">Inhibition (IC<sub>50</sub>) of human CYP1A2, 2C9, 2C19, 2D6, and 3A4 was determined using Cypex Baculosomes expressing the major human P450s. Test compounds serial dilutions 1–3 were performed from a 10 mM stock solution in DMSO by Biomek FX to generate 10-point CRC of test compound. A reference compound (e.g., miconazole) was included in each compound plate. The reference compound was diluted and stamped together with test compounds. The final concentrations of the 10-point CRCs of test compounds in the assay plate were 5.00 × 10<sup>–5</sup> M, 1.67 × 10<sup>–5</sup> M, 5.56 × 10<sup>–6</sup> M, 1.85 × 10<sup>–6</sup> M, 6.17 × 10<sup>–7</sup> M, 2.06 × 10<sup>–7</sup> M, 6.86 × 10<sup>–8</sup> M, 2.29 × 10<sup>–8</sup> M, 7.62 × 10<sup>–9</sup> M, and 2.54 × 10<sup>–9</sup> M, respectively. Then 10 μL/well of the test compounds were transferred from compound plate to assay plate using Biomek FX with the appropriate protocol. The assay plate was then incubated at 37 °C for 10 min on a shaker to allow for the interaction between compounds and enzymes. After that, 10 μL/well of the cofactor were added to the assay plate using a multidrop pipet before placing the assay plate in the plate reader EnVision to measure fluorescence. The plate was read every minute for 10 min according to the P450 isoform specific protocol. CYP450 isoform substrates used were ethoxyresorufin (ER, 1A2, 0.5 μM), 7-methoxy-4-trifluoromethylcoumarin-3-acetic acid (FCA, 2C9, 50 μM), 3-butyryl-7-methoxycoumarin (BMC, 2C19, 10 μM), 4-methylaminomethyl-7-methoxycoumarin (MMC, 2D6, 10 μM), diethoxyflourescein (DEF, 3A4, 1 μM), and 7-benzyloxyquinoline (7-BQ, 3A4, 25 μM). The test was performed in three replicates</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Intrinsic Clearance (Cl<sub>i</sub>) Assay</h3><div class="NLM_p last">Intrinsic clearance (CL<sub>i</sub>) values were determined in rat and human liver microsomes. Test compounds (0.5 μM) were incubated at 37 °C for 30 min in 50 mM potassium phosphate buffer (pH 7.4) containing 0.5 mg of microsomal protein/mL. The reaction was started by addition of cofactor (NADPH, 8 mg/mL). The final concentration of solvent was 1% of the final volume. At 0, 3, 6, 9, 15, and 30 min, an aliquot (50 μL) was taken, quenched with acetonitrile containing an appropriate internal standard, and analyzed by HPLC-MS/MS. The intrinsic clearance (CL<sub>i</sub>) was determined from the first-order elimination constant by nonlinear regression using Grafit v5 (Erithacus software, UK), corrected for the volume of the incubation and assuming 52.5 mg microsomal protein/g liver for all species. Values for CL<sub>i</sub> were expressed as mL/min/g protein. The lower limit of quantification of clearance was determined to be when <15% of the compound had been metabolized by 30 min, and this corresponded to a CL<sub>i</sub> value of 0.5 mL/min/g protein.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Chemistry</h3><div id="sec5_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> General</h4><div class="NLM_p">Proton magnetic resonance (NMR) spectra were recorded either on Varian instruments at 400 or 500 MHz, or on a Bruker instrument at 400 MHz.</div><div class="NLM_p">Chemical shifts were expressed in parts of million (ppm, δ units). Chemical shifts were reported in ppm downfield (δ) from Me<sub>4</sub>Si, used as internal standard, and were assigned as singlets (s), broad singlets (br s), doublets (d), doublets of doublets (dd), doublets of doublets of doublets (ddd), doublets of triplets (dt), triplets (t), triplets of doublets (td), quartets (q), or multiplets (m).</div><div class="NLM_p">LC-MS was recorded under the following conditions: DAD chromatographic traces, mass chromatograms, and mass spectra were taken on UPLC/PDA/MS Acquity system coupled with Micromass ZQ or Waters SQD single quadrupole mass spectrometer operated in positive and/or negative ES ionization mode. There were two QC methods used, one operated under low pH conditions and another one operated under high pH conditions. Details of the method operated under low pH conditions were: column, Acquity BEH C<sub>18</sub>, 1.7 μm, 2.1 mm × 50 mm, or Acquity CSH C<sub>18</sub>, 1.7 μm, 2.1 mm × 50 mm; the temperature column was 40 °C; mobile phase solvent A was Milli-Q water + 0.1% HCOOH; mobile phase solvent B MeCN + 0.1% HCOOH. The flow rate was 1 mL/min. The gradient table was <i>t</i> = 0 min 97% A–3% B, <i>t</i> = 1.5 min 0.1% A–99.9% B, <i>t</i> = 1.9 min 0.1% A–99.9% B, and <i>t</i> = 2 min 97% A–3% B. The UV detection range was 210–350 nm and the ES<sup>+</sup>/ES<sup>–</sup> range was 100–1000 amu.</div><div class="NLM_p">Details of the method operated under high pH conditions were the same as those listed above for the low pH method apart from column Acquity BEH C<sub>18</sub>, 1.7 μm, 2.1 mm × 50 mm; mobile phase solvent A was 10 mM aqueous solution of NH<sub>4</sub>HCO<sub>3</sub> adjusted to pH = 10 with ammonia, mobile phase solvent B MeCN.</div><div class="NLM_p">Semipreparative mass directed autopurifications (MDAP) were carried out using a Waters Fractionlynx system operated under low or high pH chromatographic conditions. The stationary phases used were: XTerra C<sub>18</sub>, XBridge C<sub>18</sub>, Sunfire C<sub>18</sub>, XSelect C<sub>18</sub>, and Gemini AXIA C<sub>18</sub>. The length of the columns was 5, 10, or 15 cm, while the internal diameter was 19, 21, or 30 mm. The particle size of the stationary phases was 5 or 10 μm. The purifications were carried out using low pH or high pH chromatographic conditions. The mobile phase solvent composition was the same used for QC analysis. The combinations stationary/mobile phases used were: XTerra, XBridge, Sunfire, and XSelect for low pH mobile phases and XTerra, XBridge, and Gemini AXIA for high pH mobile phases. All of the purifications were carried out with the column kept at room temperature. The flow rate used was 17 or 20 mL/min for columns of internal diameter 19 or 21 mm and 40 or 43 mL/min for columns of internal diameter 30 mm. The trigger for the collection of the target species was the presence of the target <i>m</i>/<i>z</i> ratio value in the TIC MS signal. The gradient timetable was customized on the <i>R</i><sub>t</sub> behavior of the target species.</div><div class="NLM_p last">Purification was performed using Biotage Isolera or Biotage SP1 flash chromatography systems; these instruments worked with Biotage KP-SIL cartridges, Biotage KP-NH cartridges, or Biotage KP-C18 cartridges.</div></div><div id="sec5_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Synthetic Procedures</h4><div class="NLM_p">Unless otherwise stated, all reactions were typically performed under inert atmosphere (for example under nitrogen). TLC refers to thin layer chromatography on silica plates, and dried refers to a solution dried over anhydrous sodium sulfate.</div><div class="NLM_p">The 1,2,4-triazolyl 5-azaspiro[2.4]heptanes described here were prepared according to general route provided in <a class="ref internalNav" href="#sch1" aria-label="General Synthetic Scheme 1">General Synthetic Scheme 1</a>. Appropriate protected aryl-5-azaspiro[2.4]heptanes (C) were prepared as follows: The desired aldehydes (1 mmol) were added to a solution of hydrazine hydrate (3 mmol) in EtOH and stirred at room temperature until completion of the reaction. The solution was diluted with water and CH<sub>2</sub>Cl<sub>2</sub>. Phases were separated; the organic one was dried and concentrated under reduced pressure, affording the described hydrazones (A). The protected 3-methylidenepyrrolidine-2,5-diones (B, like the 1-benzyl-3-methylidenepyrrolidine-2,5-dione reported in <a class="ref internalNav" href="#sch1" aria-label="General Synthetic Scheme 1">General Synthetic Scheme 1</a>) were prepared by dissolving 1-benzyl-2,5-dihydro-1<i>H</i>-pyrrole-2,5-dione (1 mmol) in AcOH and adding PPh<sub>3</sub> (1 mmol). The resulting solution was stirred for 1 h at RT and then formaldehyde 37% in water (1.8 mmol) was added. The solution was stirred at RT for 2.5 h. Volatiles were removed under reduced pressure. The residue was partitioned between water (300 mL) and CH<sub>2</sub>Cl<sub>2</sub>. The layers were separated, and the organic portion was dried; the crude material was purified by flash chromatography.</div><div class="NLM_p">To obtain compounds (C), to a solution of A (1 mmol) in dioxane at 10 °C, MnO<sub>2</sub> (10 mmol) was added portionwise. The resulting mixture was stirred at RT for 1 h, then it was filtered over a pad of Celite washing with dioxane. This pale-yellow solution was then added into a solution of (B) (1 mmol) in dioxane. The resulting orange/red solution was left stirring at RT for 40 h. Solvent was removed, and the residue was purified by flash chromatography to provide the <i>cis</i> and <i>trans</i> diastereoisomers of derivatives (C) that were used as such in the next steps.</div><div class="NLM_p">The separated diastereoisomers (C, 1 mmol) were then dissolved in THF, and LiAlH<sub>4</sub> 1 M in THF (2 mmol) was added dropwise. The resulting orange solution was refluxed for 1 h. Then it was cooled with an ice bath and quenched with Na<sub>2</sub>SO<sub>4</sub>·10H<sub>2</sub>O until gas evolution ceased. The mixture was filtered over a pad of Celite washing with EtOAc, and the solution was concentrated to afford the racemic derivatives (D) that were used as such in the next steps. Subsequently, these compounds were dissolved in MeOH under N<sub>2</sub>, and ammonium formate was added. After two cycles of vacuum and nitrogen purging, Pd/C was added. The resulting mixture was stirred at reflux for 1 h. After cooling down to RT, it was filtered over a pad of Celite, the solvent was evaporated, and the residue was loaded on SCX cartridge (eluting with 1N NH<sub>3</sub> in MeOH) to afford, after evaporation, intermediates (E).</div><div class="NLM_p">As reported previously,<a onclick="showRef(event, 'ref9 ref10 ref13'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref13">(9, 10, 13)</a> starting from the carboxylic derivatives (F), it is possible to prepare the desired thiotriazoles (G). In brief, to a solution of the carboxylic acid (F, 1 mmol) in DMF, 4-methyl-3-thiosemicarbazide (1.1 mmol) was added. Di-isopropyl ethyl amine (1.8 mmol) was added dropwise at RT, then the mixture was cooled with an ice bath before adding T3P (50% w/w in EtOAc) (1.5 mmol). The reaction was stirred at RT overnight. NaOH 4 M solution was added (until a resulting pH = 8). The reaction was diluted with EtOAc, and the two phases were separated. An additional 4 M NaOH was added up to pH 11 and the mixture heated to 70 °C for 40 min. The clear rusty-red solution was then cooled down to RT in 3 h, then 37% HCl was slowly added until pH = 5. The clear solution was extracted with CH<sub>2</sub>Cl<sub>2</sub>, and combined organics were dried and concentrated to obtain a brown solid. Crude materials were purified by C<sub>18</sub> cartridge (eluting from H<sub>2</sub>O + 0.1% HCOOH to 20% MeCN + 0.1% HCOOH). Fractions containing the product were collected and concentrated to reduce the volume, then extracted twice with CH<sub>2</sub>Cl<sub>2</sub> to obtain the desired derivatives (G). To a suspension of these derivatives (1 mmol) in a mixture MeOH/acetone at RT, the appropriate alkylating agents (e.g., 1-bromo-3-chloropropane, 1.3 mmol) were added, followed by the addition of K<sub>2</sub>CO<sub>3</sub> (1.4 mmol), and the mixture was stirred at RT for 4.5 h. It was then partitioned between water and EtOAc and phases were separated. The organic phase was washed with brine and then dried and concentrated under reduced pressure. Crude material was purified by flash chromatography to achieve the desired compounds (H).</div><div class="NLM_p">Finally, to achieve the target compounds (I) reported in the <a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a>–<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a>, intermediates (H) and (E) in equimolar quantities were dissolved in DMF or acetonitrile, and Na<sub>2</sub>CO<sub>3</sub> (1.2 mmol) and NaI (1.2 mmol) were added. The suspension was heated at 60 °C overnight. The mixtures were diluted with water and EtOAc and extracted several times with EtOAc. The organic phase was washed with brine, dried, and evaporated. The residues were purified by flash chromatography to achieve the above-mentioned compounds.</div><div class="NLM_p">Whenever necessary, the compounds (solids, oils, or foams) were taken up with CH<sub>2</sub>Cl<sub>2</sub> and a 1.0 M HCl solution in diethyl ether was added; the solvent was removed under reduced pressure to give the desired compounds as hydrochloride salts.</div><div class="NLM_p">The enantiomeric purity of each single enantiomer obtained after preparative chromatography on chiral columns was always verified on the analytical column.</div><div class="NLM_p">All the structures of the new compounds were confirmed by <sup>1</sup>H NMR and MS; additionally, <sup>13</sup>C NMR was used where necessary. All general chemicals were the highest available grade, and the purity of all synthetic compounds was determined by HPLC/MS analysis and was greater than 95%. As reported in the text, X-rays data were generated on selected intermediates.</div><div id="sec5_4_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-phenyl-5-azaspiro[2.4]heptane (<b>7</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>7</b> as a foam. Yield 11%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.26 (s, 1H), 7.33–7.09 (m, 5H), 3.78 (s, 3H), 3.36–3.27 (m, 2H), 2.76–2.47 (m, 6H), 2.41 (s, 3H), 2.15–2.09 (m, 1H), 2.00–1.87 (m, 2H), 1.65–1.37 (m, 2H), 1.17–1.04 (m, 2H). MS <i>m</i>/<i>z</i>: 410.33 [M + H]<sup>+</sup>. <b>7</b> was separated into the single enantiomers by preparative chiral HPLC, using the following details: column, Chiralcel OJ-H (25 cm × 2 cm) 5 μm; mobile phase, <i>n</i>-hexane/(ethanol + 0.1% isopropyl alcohol) 25/75% v/v; flow rate, 14 mL/min; DAD detection, 220 nm; loop, 750 μL, affording <b>7a</b> (enantiomer 1), retention time = 10.5 min and <b>7b</b> (enantiomer 2), retention time = 12.1 min.</div></div><div id="sec5_4_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]-sulfanyl}propyl)-1-phenyl-5-azaspiro[2.4]heptane (<b>8</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>8</b> as a foam. Yield 23%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.26 (s, 1H) 7.33–7.09 (m, 5H) 3.78 (s, 3H) 3.36–3.27 (m, 2H) 2.76–2.47 (m, 6H) 2.41 (s, 3H) 2.15–2.09 (m, 1H) 2.00–1.87 (m, 2H) 1.65–1.37 (m, 2H) 1.17–1.04 (m, 2H). MS <i>m</i>/<i>z</i>: 410.32 [M + H]<sup>+</sup>. <b>8</b> was separated into the single enantiomers by preparative chiral HPLC, using the following details: column, Chiralpak AD-H (25 cm × 2 cm) 5 μm; mobile phase, <i>n</i>-hexane/(ethanol + 0.1% isopropyl alcohol) 55/45% v/v; flow rate, 14 mL/min; DAD detection, 220 nm; loop, 750 μL, affording <b>8a</b> (enantiomer 1) retention time = 7.9 min and <b>8b</b> (enantiomer 2), retention time = 9.3 min.</div></div><div id="sec5_4_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>9</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>9</b> as pale-yellow foam. Yield 37%. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ: 7.95 (s, 1H), 7.54 (d, <i>J</i> = 8.03 Hz, 2H), 7.23 (br s, 2H), 3.73 (s, 3H), 3.37 (m, 2H), 2.94–2.59 (m, 5H), 2.55 (s, 3H), 2.31–1.97 (m, 3H), 1.63 (br s, 4H), 1.36–1.25 (m, 1H), 1.19–1.11 (m, 1H). MS <i>m</i>/<i>z</i>: 478.27 [M + H]<sup>+</sup>. <b>9</b> was separated into the single enantiomers by preparative chiral HPLC, using the following details: column, Chiralpak AD-H (25 cm × 2 cm) 5 μm; mobile phase, <i>n</i>-hexane/ethanol 50/50% v/v; flow rate, 14 mL/min; DAD detection, 220 nm; loop, 875 μL, affording <b>9a</b> (enantiomer 1) retention time = 8.5 min and <b>9b</b> (enantiomer 2), retention time = 10.9 min.</div></div><div id="sec5_4_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>10</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>10</b> as yellow oil. Yield 31%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.28 (s, 1H), 7.63 (d, <i>J</i> = 8.03 Hz, 2H), 7.40 (d, <i>J</i> = 8.28 Hz, 2H), 3.77 (s, 3H), 3.36–3.16 (m, 2H), 2.69- 2.46 (m, 5H), 2.44 (s, 3H), 2.30–2.22 (m, 1H), 2.03–1.79 (m, 5H), 1.35–1.26 (m, 1H), 1.26–1.19 (m, 1H). MS <i>m</i>/<i>z</i>: 478.28 [M + H]<sup>+</sup>. <b>10</b> was separated into the single enantiomers by preparative chiral HPLC, using the following details: column, Chiralcel OJ-H (25 cm × 2 cm) 5 μm; mobile phase, <i>n</i>-hexane/ethanol 50/50% v/v; flow rate, 14 mL/min; DAD detection, 220 nm; loop, 750 μL, affording <b>10a</b> (enantiomer 1) retention time = 7.3 min and <b>10b</b> (enantiomer 2), retention time = 9.7 min.</div></div><div id="sec5_4_2_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-5-(2-{[4-Methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}ethyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>11</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>11</b> as yellow foam. Yield 51%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.27 (s, 1H), 7.62 (d, <i>J</i> = 8.28 Hz, 2H), 7.34 (d, <i>J</i> = 8.28 Hz, 2H), 3.81 (s, 2H), 3.39 (m, 2H), 2.87 (m, 2H), 2.83–2.75 (m, 3H), 2.73–2.61 (m, 3H), 2.43 (s, 2H), 2.29–2.21 (m, 1H), 1.70–1.61 (m, 1H), 1.44 (s, 1H), 1.26 (m, 1H), 1.23–1.17 (m, 1H). MS <i>m</i>/<i>z</i>: 464.25 [M + H]<sup>+</sup>. <b>11</b> was separated into the single enantiomers by preparative chiral HPLC, using the following details: column, Chiralpak AD-H (25 cm × 2 cm) 5 μm; mobile phase, <i>n</i>-hexane/ethanol 55/45% v/v; flow rate, 14 mL/min; DAD detection, 220 nm; loop, 750 μL, affording <b>11a</b> (enantiomer 1) retention time = 11.7 min and <b>11b</b> (enantiomer 2), retention time = 13.1 min.</div></div><div id="sec5_4_2_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(2-{[4-Methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}ethyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>12</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>12</b> as pale-yellow foam. Yield 22%. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ: 7.95 (s, 1H), 7.54 (d, <i>J</i> = 8.03 Hz, 2H), 7.20 (d, <i>J</i> = 8.03 Hz, 2H), 3.68 (s, 3H), 3.44–3.28 (m, 2H), 2.85 (s, 3H), 2.76–2.68 (m, 1H), 2.57–2.51 (m, 3H), 2.50- 2.43 (m, 1H), 2.25–2.19 (m, 1H), 2.18–2.11 (m, 1H), 2.05–1.94 (m, 2H), 1.24–1.15 (m, 2H). MS <i>m</i>/<i>z</i>: 464.26 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(4-{[4-Methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}butyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>13</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>13</b> as pale-yellow foam. Yield 15%. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ: 7.95 (s, 1H), 7.55 (d, <i>J</i> = 8.03 Hz, 2H), 7.22 (d, <i>J</i> = 7.78 Hz, 2H), 3.74–3.65 (m, 3H), 3.33–3.20 (m, 2H), 2.94 (br s, 1H), 2.70 (br s, 1H), 2.54 (s, 6H), 2.27–2.14 (m, 2H), 2.05 (d, <i>J</i> = Hz, 2H), 1.82 (m, 3H), 1.32–1.19 (m, 3H). MS <i>m</i>/<i>z</i>: 492.28 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[2-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane Hydrochloride (<b>14</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>14</b> subsequently transformed in corresponding hydrochloride. Yield 13%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.56–10.13 (m, 1H), 8.58 (s, 1H), 7.77 (d, <i>J</i> = 8.03 Hz, 1H), 7.64 (t, <i>J</i> = 7.40 Hz, 1H), 7.48 (t, <i>J</i> = 7.65 Hz, 1H), 7.36 (d, <i>J</i> = 7.78 Hz, 1H), 3.71 (s, 3H), 3.68–3.49 (m, 2H), 3.28 (m, 4H), 3.22–3.12 (m, 1H), 3.07–2.93 (m, 1H), 2.48 (br s, 2H), 2.39 (s, 3H), 2.10 (br s, 2H), 1.72–1.52 (m, 2H), 1.50–1.32 (m, 2H). MS <i>m</i>/<i>z</i>: 478.35 [M + H]<sup>+</sup>. <b>14</b> was separated into the single enantiomers by preparative chiral HPLC, using the following details: column, Chiralpak AD-H (25 cm × 2 cm) 5 μm; mobile phase, <i>n</i>-hexane/(ethanol + 0.1% isopropylamine) 70/30% v/v; flow rate, 16 mL/min; DAD detection, 220 nm; loop, 500 μL, affording <b>14a</b> (enantiomer 1) retention time = 7.2 min and <b>14b</b> (enantiomer 2), retention time = 8.4 min.</div></div><div id="sec5_4_2_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[2-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>15</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>15</b> as a light-yellow foam. Yield 45%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.29 (s, 1H), 7.78–7.71 (m, 1H), 7.62–7.54 (m, 1H), 7.47–7.38 (m, 1H), 7.33–7.27 (m, 1H), 3.77 (s, 3H), 3.37–3.17 (m, 3H), 3.06–2.83 (m, 2H), 2.60 (br s, 4H), 2.44 (s, 3H), 2.41 (br s, 1H), 2.02–1.80 (m, 4H), 1.60–1.51 (m, 1H), 1.25–1.14 (m, 1H). MS <i>m</i>/<i>z</i>: 477.98 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-1-(4-Fluorophenyl)-5-(3-{[4-methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-5-azaspiro[2.4]heptane (<b>16</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>16</b> as a foam. Yield 38%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.31–8.25 (m, 1H), 7.21–7.11 (m, 2H), 7.11–7.00 (m, 2H), 3.80 (s, 3H), 3.37–3.29 (m, 2H), 2.75–2.66 (m, 1H), 2.66–2.50 (m, 5H), 2.44 (s, 3H), 2.17–2.10 (m, 1H), 2.02–1.90 (m, 2H), 1.63–1.48 (m, 1H), 1.47–1.35 (m, 1H), 1.19–1.11 (m, 1H), 1.08–1.04 (m, 1H). MS <i>m</i>/<i>z</i>: 428.41 [M + H]<sup>+</sup>. <b>16</b> was separated into the single enantiomers by preparative chiral HPLC, using the following details: column, Chiralpak AD-H (25 cm × 2.1 cm) 5 μm; modifier, (methanol + 0.1% isopropylamine) 20%; flow rate, 45 mL/min; DAD detection, 220 nm; loop, 700 μL, affording <b>16a</b> (enantiomer 1) retention time = 11.7 min and <b>16b</b> (enantiomer 2), retention time = 16.3 min.</div></div><div id="sec5_4_2_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-1-(4-Fluorophenyl)-5-(3-{[4-methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-5-azaspiro[2.4]heptane (<b>17</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>17</b> as foam. Yield 46%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.26–8.30 (m, 1H), 7.21–7.16 (m, 2H), 7.08–7.00 (m, 2H), 3.77 (s, 3H), 3.31–3.15 (m, 2H), 2.75–2.62 (m, 2H), 2.50 (t, <i>J</i> = 6.65 Hz 2H), 2.43 (s, 3H), 2.37 (d, <i>J</i> = 8.0 Hz, 1H), 2.13–2.09 (m, 2H), 1.99–1.89 (m, 2H), 1.88–1.79 (m, 2H), 1.14–1.06 (m, 2H). MS <i>m</i>/<i>z</i>: 428.41 [M + H]<sup>+</sup>. <b>17</b> was separated into the single enantiomers by preparative chiral HPLC, using the following details: column, Chiralpak AD-H (25 cm × 2.1 cm) 5 μm; modifier, (methanol + 0.1% isopropylamine) 17%; flow rate, 46 mL/min; DAD detection, 220 nm; loop, 700 μL, affording <b>17a</b> (enantiomer 1) retention time = 12.1 min and <b>17b</b> (enantiomer 2), retention time = 15.0 min.</div></div><div id="sec5_4_2_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-1-(2,4-Difluorophenyl)-5-(3-{[4-methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-5-azaspiro[2.4]heptane (<b>18</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>18</b> as foam. Yield 42%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.28 (s, 1H), 7.20–6.87 (m, 3H), 3.81 (s, 2H), 3.33 (t, <i>J</i> = 7.15 Hz, 2H), 2.73–2.50 (m, 7H), 2.43 (s, 3H), 2.15 (br s, 1H), 2.01–1.91 (m, 2H), 1.60–1.51 (m, 1H), 1.38–1.30 (m, 1H), 1.24–1.12 (m, 2H). MS <i>m</i>/<i>z</i>: 446.40 [M + H]<sup>+</sup>. <b>18</b> was separated into the single enantiomers by preparative chiral HPLC, using the following details: column, Chiralpak AD-H (25 cm × 2 cm) 5 μm; mobile phase, (methanol + 0.1% isopropylamine) 18%; flow rate, 45 mL/min; DAD detection, 220 nm; loop, 700 μL, affording <b>18a</b> (enantiomer 1) retention time = 11.5 min and <b>18b</b> (enantiomer 2), retention time = 15.5 min.</div></div><div id="sec5_4_2_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-1-(2,4-Difluorophenyl)-5-(3-{[4-methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-5-azaspiro[2.4]heptane (<b>19</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>19</b> as foam. Yield 49%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.28 (s, 1H), 7.19–7.09 (m, 1H), 7.04–6.88 (m, 2H), 3.77 (s, 5H), 3.31–3.19 (m, 2H), 2.78–2.73 (m, 1H), 2.65–2.57 (m, 1H), 2.55–2.46 (m, 2H), 2.43 (s, 3H), 2.39–2.34 (m, 1H), 2.14–2.09 (m, 1H), 2.01–1.93 (m, 3H), 1.84 (m, 2H), 1.25–1.18 (m, 1H), 1.16–1.11 (m, 1H). MS <i>m</i>/<i>z</i>: 446.40 [M + H]<sup>+</sup>. <b>19</b> was separated into the single enantiomers by preparative chiral HPLC, using the following details: column, Chiralpak AD-H (25 cm × 2.1 cm) 5 μm; mobile phase, (methanol + 0.1% isopropylamine) 12%; flow rate, 45 mL/min; DAD detection, 220 nm; loop, 700 μL, affording <b>19a</b> (enantiomer 1) retention time = 14.1 min and <b>19b</b> (enantiomer 2), retention time = 18.5 min.</div></div><div id="sec5_4_2_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-1-[2-Fluoro-4-(trifluoromethyl)phenyl]-5-(3-{[4-methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-5-azaspiro[2.4]heptane (<b>20</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>20</b> as foam. Yield 62%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.27 (s, 1H), 7.49 (m, 2H), 7.37–7.29 (m, 1H), 3.80 (s, 3H), 3.38–3.32 (m, 2H), 3.12–2.47 (m, 6H), 2.42 (s, 3H), 2.35 (m, 1H), 1.69 (br s, 1H), 1.42 (br s, 1H), 1.39–1.28 (m, 3H). MS <i>m</i>/<i>z</i>: 496.32 [M + H]<sup>+</sup>. <b>20</b> was separated into the single enantiomers by preparative chiral HPLC, using the following details: column, Chiralpak AD-H (25 cm × 2 cm) 5 μm; mobile phase, <i>n</i>-hexane/(ethanol + 0.1% isopropyl alcohol) 55/45% v/v; flow rate, 14 mL/min; DAD detection, 220 nm; loop, 500 μL, affording <b>20a</b> (enantiomer 1) retention time = 7.7 min and <b>20b</b> (enantiomer 2), retention time = 9.3 min.</div></div><div id="sec5_4_2_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-1-[2-Fluoro-4-(trifluoromethyl)phenyl]-5-(3-{[4-methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-5-azaspiro[2.4]heptane (<b>21</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>21</b> as foam. Yield 41%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.27 (s, 1H), 7.49 (d, <i>J</i> = 9.03 Hz, 2H), 7.43–7.38 (m, 1H), 3.76 (s, 3H), 3.35–3.20 (m, 2H), 3.19–2.87 (m, 3H), 2.42 (s, 3H), 2.39–2.33 (m, 1H), 2.19–2.10 (m, 1H), 2.02–1.92 (m, 2H), 1.50–1.42 (m, 1H), 1.36–1.27 (m, 2H). MS <i>m</i>/<i>z</i>: 496.32 [M + H]<sup>+</sup>. <b>21</b> was separated into the single enantiomers by preparative chiral HPLC, using the following details: column, Chiralpak AD-H (25 cm × 2 cm) 5 μm; mobile phase, <i>n</i>-hexane/(ethanol + 0.1% isopropyl alcohol) 70/30% v/v; flow rate, 14 mL/min; DAD detection, 220 nm; loop, 500 μL, affording <b>21a</b> (enantiomer 1) retention time = 7.6 min and <b>21b</b> (enantiomer 2), retention time = 8.8 min.</div></div><div id="sec5_4_2_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> (1<i>S</i>,3<i>S</i> or 1<i>R</i>,3<i>R</i>)-1-[4-Fluoro-2-(trifluoromethyl)phenyl]-5-(3-{[4-methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-5-azaspiro[2.4] heptane Hydrochloride (<b>22a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using the single <i>trans</i> enantiomer 1 and not the racemic mixture; the crude residue was purified by flash chromatography and the hydrochloride made as reported in the general procedure to give <b>22a</b> as a yellow solid. Yield 47%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.30–9.93 (m, 1H), 8.57 (s, 1H), 7.70–7.61 (m, 1H), 7.55–7.46 (m, 1H), 7.44–7.36 (m, 1H), 3.71–3.64 (m, 4H), 3.62–3.48 (m, 2H), 3.39–3.34 (m, 1H), 3.27–3.11 (m, 4H), 3.08–2.94 (m, 1H), 2.47–2.41 (m, 1H), 2.38 (s, 3H), 2.09 (br s, 2H), 1.70–1.51 (m, 2H), 1.49–1.39 (m, 1H), 1.39–1.30 (m, 1H). MS <i>m</i>/<i>z</i>: 496.36 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> (1<i>R</i>,3<i>R</i> or 1<i>S</i>,3<i>S</i>)-1-[4-Fluoro-2-(trifluoromethyl)phenyl]-5-(3-{[4-methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-5-azaspiro[2.4]heptane Hydrochloride (<b>22b</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using the single <i>trans</i> enantiomer 2 and not the racemic mixture; the crude residue was purified by flash chromatography and the hydrochloride made as reported in the general procedure to give <b>22b</b> as a yellow solid. Yield 57%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.15–9.88 (m, 1H), 8.57 (s, 1H), 7.68–7.62 (m, 1H), 7.54–7.48 (m, 1H), 7.46–7.38 (m, 1H), 3.71–3.64 (m, 4H), 3.63–3.47 (m, 2H), 3.46–3.35 (m, 1H), 3.27–3.11 (m, 4H), 3.08–2.94 (m, 1H), 2.47–2.41 (m, 1H), 2.38 (s, 3H), 2.15–2.02 (m, 2H), 1.68–1.52 (m, 2H), 1.50–1.40 (m, 1H), 1.38–1.28 (m, 1H). MS <i>m</i>/<i>z</i>: 496.36 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> (1<i>R</i>,3<i>S</i> or 1<i>S</i>,3<i>R</i>)-1-[4-Fluoro-2-(trifluoromethyl)phenyl]-5-(3-{[4-methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-5-azaspiro[2.4]heptane Hydrochloride (<b>23a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using the single <i>cis</i> enantiomer 1 and not the racemic mixture; the crude residue was purified by flash chromatography and the hydrochloride made as reported in the general procedure to give <b>23a</b> as a yellow solid. Yield 58%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.12 (br s, 1H), 8.57 (s, 1H), 7.68–7.63 (m, 1H), 7.49 (m., 1H), 7.42–7.33 (m, 1H), 3.84–3.72 (m, 1H), 3.69–3.61 (m, 3H), 3.39–3.25 (m, 6H), 3.00–2.84 (m, 1H), 2.62–2.52 (m, 2H), 2.37 (s, 3H), 2.27–1.91 (m, 4H), 1.75–1.61 (m, 1H), 1.42–1.22 (m, 1H). MS <i>m</i>/<i>z</i>: 496.32 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> (1<i>S</i>,3<i>R</i> or 1<i>R</i>,3<i>S</i>)-1-[4-Fluoro-2-(trifluoromethyl)phenyl]-5-(3-{[4-methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl] sulfanyl}propyl)-5-azaspiro[2.4]heptane Hydrochloride (<b>23b</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using the single <i>cis</i> enantiomer 2 and not the racemic mixture; the crude residue was purified by flash chromatography and the hydrochloride made as reported in the general procedure to give <b>23b</b> as a yellow solid. Yield 58%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.29–9.84 (m, 1H), 8.57 (s, 1H), 7.68–7.63 (m, 1H), 7.54–7.44 (m, 1H), 7.42–7.32 (m, 1H), 3.82–3.71 (m, 1H), 3.70–3.61 (m, 3H), 3.24–3.04 (m, 6H), 2.99–2.85 (m, 1H), 2.62–2.52 (m, 2H), 2.37 (s, 3H), 2.11–2.01 (m, 4H), 1.74–1.61 (m, 1H), 1.26 (m, 1H). MS <i>m</i>/<i>z</i>: 496.36 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> 1-{[4-Methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]¬sulfanyl}-3-{1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]-heptan-5-yl}propan-2-ol Diastereoisomeric Mixture (<b>24</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>24</b> as an oil. Yield 9%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.28 (s, 1H), 7.61 (d, <i>J</i> = 8.03 Hz, 2H), 7.43–7.31 (m, 2H), 3.77 (d, <i>J</i> = 3.76 Hz, 3H), 3.59–3.52 (m, 1H), 3.49–3.41 (m, 1H), 3.24–3.11 (m, 2H), 2.57 (br s, 2H), 2.43 (s, 3H), 2.28–2.20 (m, 2H), 2.00–1.92 (m, 2H), 1.92–1.88 (m, 1H), 1.35–1.26 (m, 2H), 1.24–1.17 (m, 1H). MS <i>m</i>/<i>z</i>: 494.39 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-1-(3,5-Dichlorophenyl)-5-(3-{[4-methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-5-azaspiro[2.4]heptane (<b>25</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>25</b>. Yield 9%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.30 (s, 1H), 7.32 (s, 3H), 3.82 (s, 3H), 3.53–3.37 (m, 3H), 2.84–2.79 (m, 2H), 2.44 (s, 3H), 2.41–2.33 (m, 1H), 2.22–2.13 (m, 2H), 1.90–1.79 (m, 2H), 1.72–1.51 (m, 3H), 1.44–1.28 (m, 3H). MS <i>m</i>/<i>z</i>: 478.27 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-1-(3,5-Dichlorophenyl)-5-(3-{[4-methyl-5-(4-methyl-1,3-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-5-azaspiro[2.4]heptane (<b>26</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>26</b>. Yield 34%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.28 (s, 1H), 7.31–7.27 (m, 1H), 7.22 (s, 2H), 3.78 (s, 3H), 3.36–3.25 (m, 2H), 2.85–2.78 (m, 4H), 2.43 (s, 3H), 2.39–2.24 (m, 2H), 2.08 (br s, 2H), 2.02–1.95 (m, 2H), 1.52–1.22 (m, 3H). MS <i>m</i>/<i>z</i>: 477.91 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(4-methyl-1,3-thiazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>27</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>27</b>. Yield 55%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 9.15 (s, 1H), 7.64 (d, <i>J</i> = 8.03 Hz, 2H), 7.39 (d, <i>J</i> = 8.03 Hz, 2H), 3.62 (s, 3H), 3.36 (s, 2H), 2.72 (br s, 5H), 2.50 (s, 3H), 2.36–2.28 (m, 1H), 2.06–1.97 (m, 3H), 1.79–1.67 (m, 1H), 1.53–1.41 (m, 1H), 1.35–1.23 (m, 3H). MS <i>m</i>/<i>z</i>: 494.43 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> (1R,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(4-methyl-1,3-thiazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>28</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>28</b>. Yield 40%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 9.15 (s, 1H), 7.67–7.60 (m, 2H), 7.42–7.37 (m, 2H), 3.58 (s, 3H), 3.25 (m, 2H), 2.77–2.72 (m, 2H), 2.67–2.60 (m, 1H), 2.56–2.46 (m, 6H), 2.26–2.21 (m, 1H), 2.03–1.95 (m, 2H), 1.92–1.82 (m, 2H), 1.30 (m, 1H), 1.22 (m, 1H). MS <i>m</i>/<i>z</i>: 494.45 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(1,3-thiazol-2-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>29</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>29</b>. Yield 61%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.04 (d, <i>J</i> = 3.26 Hz, 1H), 7.81 (d, <i>J</i> = 3.26 Hz, 1H), 7.63 (d, <i>J</i> = 8.28, Hz, 2H), 7.50 (d, <i>J</i> = 7.53 Hz, 2H), 4.06 (s, 3H), 3.39 (m, 2H), 2.96 (br s, 5H), 2.39–2.26 (m, 1H), 2.0 5- 2.01 (m, 3H), 1.79–1.66 (m, 1H), 1.53–1.42 (m, 1H), 1.37–1.21 (m, 3H). MS <i>m</i>/<i>z</i>: 480.38 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(1,3-thiazol-2-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>30</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>30</b>. Yield 48%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.03 (d, <i>J</i> = 3.26 Hz, 1H), 7.79 (d, <i>J</i> = 3.26 Hz, 1H), 7.61 (d, <i>J</i> = 8.03 Hz, 2H), 7.37 (d, <i>J</i> = 8.03 Hz, 2H), 4.02 (s, 3H), 3.37–3.19 (m, 2H), 2.76–2.70 (m, 2H), 2.66–2.56 (m, 1H), 2.57–2.44 (m, 3H), 2.26–2.19 (m, 1H), 2.02–1.92 (m, 2H), 1.91–1.82 (m, 2H), 1.28 (m, 1H), 1.20 (m, 1H). MS <i>m</i>/<i>z</i>: 480.00 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(1,2,3-thiadiazol-4-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane Hydrochloride (<b>31</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography and the free base transformed into the hydrochloride to give <b>31</b>. Yield 22%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.58 (br s, 1H), 9.79 (s, 1H), 7.67 (m, 2H), 7.46 (m, 2H), 3.90 (s, 3H), 3.70–3.50 (m, 4H), 3.40–3.16 (m, 6H), 3.12–3.00 (m, 1H), 2.48–2.37 (m, 1H), 1.97–1.80 (m, 1H), 1.56–1.29 (m, 3H). MS <i>m</i>/<i>z</i>: 481.36 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-(3-{[4-Methyl-5-(1,2,3-thiadiazol-4-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>32</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>32</b>. Yield 9%. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ: 9.34 (s, 1H), 7.54 (d, <i>J</i> = 8.28 Hz, 2H), 7.22 (d, <i>J</i> = 8.03 Hz, 2H), 4.04 (s, 3H), 3.38–3.25 (m, 2H), 2.90–2.80 (m, 1H), 2.74–2.63 (m, 1H), 2.58 (br s, 2H), 2.49–2.42 (m, 1H), 2.16 (d, <i>J</i> = Hz, 2H), 2.01 (br s, 5H), 1.21 (s, 2H). MS <i>m</i>/<i>z</i>: 481.42 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(thiophen-2-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[ 4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>33</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>33</b>. Yield 50%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 7.69 (dd, <i>J</i> = 1.13, 5.14 Hz, 1H), 7.64–7.58 (m, 3H), 7.37 (d, <i>J</i> = 8.28 Hz, 2H), 7.24 (dd, <i>J</i> = 3.64, 5.14 Hz, 1H), 3.79 (s, 3H), 3.29–3.10 (m, 2H), 2.76–2.69 (m, 2H), 2.65–2.57 (m, 1H), 2.57–2.43 (m, 3H), 2.25–2.19 (m, 1H), 2.03–1.92 (m, 2H), 1.87–1.77 (m, 2H), 1.30–1.25 (m, 1H), 1.23–1.17 (m, 1H). MS <i>m</i>/<i>z</i>: 478.99 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(thiophen-3-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>34</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>34</b>. Yield 45%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 7.96 (dd, <i>J</i> = 1.38, 2.89 Hz, 1H), 7.68 (dd, <i>J</i> = 2.76, 5.02 Hz, 1H), 7.58–7.63 (m, 3H), 7.37 (d, <i>J</i> = 8.03 Hz, 2H)), 3.76 (s, 3H), 3.28–3.10 (m, 2H), 2.76–2.69 (m, 2H), 2.65–2.56 (m, 1H), 2.55–2.42 (m, 3H), 2.26–2.19 (m, 1H), 2.02–1.92 (m, 2H), 1.87–1.77 (m, 2H), 1.28 (m, 1H), 1.20 (m, 1H). MS <i>m</i>/<i>z</i>: 479.01 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[5-(Furan-2-yl)-4-methyl-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>35</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>35</b>. Yield 32%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 7.81 (dd, <i>J</i> = 0.75, 1.76 Hz, 1H), 7.61 (d, <i>J</i> = 8.03 Hz, 2H), 7.37 (d, <i>J</i> = 8.28 Hz, 2H), 7.03 (dd, <i>J</i> = 0.75, 3.51 Hz, 1H), 6.68 (dd, <i>J</i> = 1.88, 3.39 Hz, 1H), 3.79 (s, 3H), 3.28–3.11 (m, 2H), 2.76–2.70 (m, 1H), 2.67–2.57 (m, 1H), 2.56–2.44 (m, 3H), 2.25–2.20 (m, 1H), 2.02–1.89 (m, 2H), 1.87–1.79 (m, 2H), 1.55–1.46 (m, 1H), 1.31–1.25 (m, 1H), 1.24–1.17 (m, 1H). MS <i>m</i>/<i>z</i>: 463.00 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[5-(Furan-3-yl)-4-methyl-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>36</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>36</b>. Yield 45%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 7.69 (dd, <i>J</i> = 1.13, 5.14 Hz, 1H), 7.57–7.64 (m, 3H), 7.37 (d, <i>J</i> = 8.28 Hz, 2H), 7.24 (dd, <i>J</i> = 3.76, 5.27 Hz, 1H), 3.72 (s, 3H), 3.27–3.06 (m, 2H), 2.76–2.70 (m, 2H), 2.65–2.56 (m, 1H), 2.55–2.42 (m, 3H), 2.24–2.19 (m, 1H), 2.00–1.92 (m, 2H), 1.81 (m, 2H), 1.30–1.24 (m, 1H), 1.24–1.17 (m, 1H). MS <i>m</i>/<i>z</i>: 463.00 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(1-methyl-1<i>H</i>-pyrazol-4-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>37</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>37</b>. Yield 54%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.17–8.13 (m, 1H), 7.89–7.86 (m, 1H), 7.65–7.60 (m, 2H), 7.41–7.36 (m, 2H), 4.00 (s, 3H), 3.71 (s, 3H), 3.24–3.08 (m, 2H), 2.76–2.71 (m, 1H), 2.65–2.57 (m, 1H), 2.54–2.43 (m, 3H), 2.26–2.20 (m, 2H), 2.04–1.95 (m, 3H), 1.87–1.78 (m, 2H), 1.32–1.26 (m, 1H), 1.24–1.19 (m, 1H). MS <i>m</i>/<i>z</i>: 477.47 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(1-methyl-1<i>H</i>-pyrazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>38</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>38</b>. Yield 43%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 7.61 (d, <i>J</i> = 2.01 Hz, 1H), 7.54 (d, <i>J</i> = 8.03 Hz, 2H), 7.22 (d, <i>J</i> = 8.28 Hz, 2H), 6.51 (d, <i>J</i> = 2.01 Hz, 1H), 4.15 (s, 3H), 3.60–3.56 (m, 3H), 3.39–3.25 (m, 2H), 2.89 (d, <i>J</i> = Hz, 1H), 2.73 (br s, 1H), 2.62 (br s, 2H), 2.50 (d, <i>J</i> = Hz, 1H), 2.25–2.13 (m, 2H), 2.12–1.91 (m, 4H), 1.27–1.18 (m, 2H). MS <i>m</i>/<i>z</i>: 477.40 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(3-methyl-1,2-oxazol-5-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>39</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>39</b>. Yield 64%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 7.65–7.58 (m, 2H), 7.41–7.37 (m, 2H), 6.91–6.88 (m, 1H), 3.84 (s, 3H), 3.28 (m, 2H), 2.90–2.81 (m, 2H), 2.60 (br s, 3H), 2.38 (s, 3H), 2.30–2.23 (m, 1H), 2.03–1.84 (m, 5H), 1.34–1.29 (m, 1H), 1.23 (m, 1H). MS <i>m</i>/<i>z</i>: 478.11 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> 4-[4-Methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]benzamide (<b>40a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography to give <b>40a</b>. Yield 72%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.12 (d, <i>J</i> = 8.53 Hz, 2H), 7.87 (d, <i>J</i> = 8.53 Hz, 2H), 7.63 (d, <i>J</i> = 8.03 Hz, 2H), 7.39 (d, <i>J</i> = 8.28 Hz, 2H), 6.76 (br s, 1H), 3.73 (s, 3H), 3.32–3.17 (m, 2H), 2.76–2.71 (m, 1H), 2.68–2.59 (m, 1H), 2.58–2.44 (m, 3H), 2.29–2.20 (m, 1H), 2.12–2.08 (m, 1H), 2.04–1.82 (m, 4H), 1.30 (m, 1H), 1.22 (m, 1H). MS <i>m</i>/<i>z</i>: 516.38 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_37" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> 3-[4-Methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]benzamide (<b>41a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography to give <b>41a</b>. Yield 66%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.27 (t, <i>J</i> = 1.63 Hz, 1H), 8.11 (td, <i>J</i> = 1.47, 7.84 Hz, 1H), 7.92 (td, <i>J</i> = 1.38, 8.03 Hz, 1H), 7.53–7.77 (m, 4H), 7.39 (d, <i>J</i> = 8.28 Hz, 2H), 6.76 (br s, 1H), 3.72 (s, 3H), 3.32–3.16 (m, 2H), 2.77–2.70 (m, 1H), 2.68–2.59 (m, 1H), 2.58–2.42 (m, 3H), 2.29–2.21 (m, 1H), 2.07–1.93 (m, 3H), 1.91–1.81 (m, 2H), 1.34–1.26 (m, 1H), 1.25–1.19 (m, 1H). MS <i>m</i>/<i>z</i>: 516.38 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_38" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> 2-[4-Methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]benzamide hydrochloride (<b>42a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography and the hydrochloride made as reported in the general procedure to give <b>42a</b>. Yield 22%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 11.03–10.40 (m, 2H), 8.02–7.30 (m, 7H), 6.87 (br s, 1H), 3.82–3.60 (m, 3H), 3.47–3.05 (m, 5H), 3.05–2.61 (m, 4H), 2.48–1.82 (m, 4H), 1.56–1.22- (m, 2H). MS <i>m</i>/<i>z</i>: 516.39 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_39" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> 4-[4-Methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]benzene-1-sulfonamide (<b>43a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography to give <b>43a</b>. Yield 60%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.05 (s, 2H), 7.97 (s, 2H), 7.66–7.58 (m, 2H), 7.42–7.32 (m, 2H), 6.76–6.70 (m, 1H), 3.74 (s, 3H), 3.26 (m, 2H), 2.79 (br s, 2H), 2.70–2.45 (m, 4H), 2.26–2.21 (m, 1H), 2.02–1.81 (m, 4H), 1.29 (m, 1H), 1.21 (m, 1H). MS <i>m</i>/<i>z</i>: 552.34 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_40" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> 1-{4-[4-Methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]phenyl}ethan-1-one Hydrochloride (<b>44a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography and the hydrochloride made as reported in the general procedure to give <b>44a</b>. Yield 62%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.44–10.14 (m, 1H), 8.12 (d, <i>J</i> = 8.28 Hz, 2H), 7.88 (d, <i>J</i> = 8.53 Hz, 2H), 7.65 (d, <i>J</i> = 8.03 Hz, 2H), 7.48–7.38 (m, 2H), 3.74–3.68 (m, 1H), 3.64 (d, <i>J</i> = 4.02 Hz, 3H), 3.47–3.38 (m, 1H), 3.21 (d, <i>J</i> = 7.28 Hz, 6H), 2.65 (s, 3H), 2.45–2.20 (m, 2H), 2.14–1.92 (m, 3H), 1.53–1.26 (m, 2H). MS <i>m</i>/<i>z</i>: 515.39 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_41" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> 4-[4-Methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]benzonitrile (<b>45a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography to give <b>45a</b>. Yield 78%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.05–7.93- (m, 4H), 7.62 (d, <i>J</i> = 8.03 Hz, 2H), 7.39 (d, <i>J</i> = 8.28 Hz, 2H), 3.76 (s, 3H), 3.34–3.19 (m, 2H), 2.77–2.72 (m, 1H), 2.68–2.60 (m, 1H), 2.57–2.45 (m, 3H), 2.28–2.20 (m, 1H), 2.13–2.08 (m, 1H), 2.04–1.81- (m, 4H), 1.32–1.26 (m, 1H), 1.22 (m, 1H). MS <i>m</i>/<i>z</i>: 498.37 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_42" class="NLM_sec NLM_sec_level_4"><div id="ac_i80" class="anchor-spacer"></div><h5 class="article-section__title" id="_i80"> 2-{4-[4-Methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]phenyl}acetamide (<b>46a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography to give <b>46a</b>. Yield 61%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 7.68–7.58 (m, 4H), 7.53 (br s, 1H), 7.45 (d, <i>J</i> = 8.28 Hz, 2H), 7.33 (d, <i>J</i> = 8.28 Hz, 2H), 6.94 (br s, 1H), 3.58 (s, 3H), 3.48 (s, 2H), 3.16–3.08 (m, 2H), 2.74–2.64 (m, 1H), 2.48–2.37 (m, 4H), 2.25–2.17 (m, 1H), 1.96 (d, 3H), 1.76 (s, 2H), 1.28 (t, <i>J</i> = 5.52 Hz, 1H), 1.18 (m, 1H). MS <i>m</i>/<i>z</i>: 530.38 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_43" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> (1<i>R</i>,3<i>S</i>)-5-[3-({4-Methyl-5-[4-(1,3-oxazol-2-yl)phenyl]-4<i>H</i>-1,2,4-triazol-3-yl}sulfanyl)propyl]-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>47a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography to give <b>47a</b>. Yield 60%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.25–8.19 (m, 2H), 8.11 (d, <i>J</i> = 0.75 Hz, 1H), 7.97–7.92 (m, 2H), 7.62 (d, <i>J</i> = 8.03 Hz, 2H), 7.44–7.37 (m, 3H), 3.76 (s, 3H), 3.34–3.20 (m, 2H), 2.72–2.47 (m, 6H), 2.28 (br s, 1H), 2.02–1.86 (m, 4H), 1.33 (br s, 1H), 1.24 (br s, 1H). MS <i>m</i>/<i>z</i>: 540.4 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_44" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> (1<i>R</i>,3<i>S</i>)-5-[3-({4-Methyl-5-[3-(1,3-oxazol-2-yl)phenyl]-4<i>H</i>-1,2,4-triazol-3-yl}sulfanyl)propyl]-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>48a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography to give <b>48a</b>. Yield 57%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.43 (t, <i>J</i> = 1.63 Hz, 1H), 8.22 (td, <i>J</i> = 1.32, 7.91 Hz, 1H), 8.11 (d, <i>J</i> = 0.5 Hz, 1H), 7.91 (td, <i>J</i> = 1.32, 7.91 Hz, 1H), 7.75 (t, <i>J</i> = 7.78 Hz, 1H), 7.63 (d, <i>J</i> = 8.03 Hz, 2H), 7.43 (d, <i>J</i> = 8.03 Hz, 2H), 7.38 (d, <i>J</i> = 0.5 Hz, 1H), 3.77 (s, 3H), 3.38–3.19 (m, 2H), 3.04–2.64 (m, 6H), 2.37–2.28 (m, 2H), 2.05–1.93 (m, 3H), 1.41–1.35 (m, 1H), 1.30–1.25 (m, 1H). MS <i>m</i>/<i>z</i>: 540.4 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_45" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(pyridin-4-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>49</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>49</b>. Yield 62%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.74–8.80 (m, 2H), 7.75–7.80 (m, 2H), 7.63 (d, <i>J</i> = 8.03 Hz, 2H), 7.36 (d, <i>J</i> = 8.03 Hz, 2H), 3.81 (s, 3H), 3.35 (m, 2H), 2.64 (br s, 6H), 2.24–2.32 (m, 1H), 2.09–2.13 (m, 1H), 1.99 (m, 2H), 1.62–1.75 (m, 1H), 1.38–1.50 (m, 1H), 1.29–1.21 (m, 2H). MS <i>m</i>/<i>z</i>: 474.4 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_46" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(pyridin-4-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane Hydrochloride (<b>50</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography and the hydrochloride made as reported in the general procedure to give <b>50</b>. Yield 30%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 11.35–11.13 (m, 1H), 8.92 (d, <i>J</i> = 6.53 Hz, 2H), 8.07 (d, <i>J</i> = 6.53 Hz, 2H), 7.65 (dd, <i>J</i> = 5.14, 7.91 Hz, 2H), 7.45 (m, 2H), 3.73 (d, <i>J</i> = 4.02 Hz, 3H), 3.70–3.61 (m, 1H), 3.45–3.35 (m, 1H), 3.35–3.23 (m, 3H), 3.22–3.03 (m, 2H), 2.65–2.56 (m, 1H), 2.98–2.56 (m, 1H), 2.49–2.36 (m, 1H), 2.34–1.94 (m, 4H), 1.51–1.25 (m, 2H). MS <i>m</i>/<i>z</i>: 474.40 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_47" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(pyridin-3-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>51</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>51</b>. Yield 59%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.96 (d, <i>J</i> = 2.26 Hz, 1H), 8.73 (dd, <i>J</i> = 1.51, 5.02 Hz, 1H), 8.15 (td, <i>J</i> = 1.98, 7.84 Hz 1H), 7.63 (d, <i>J</i> = 8.03 Hz, 2H), 7.60–7.54 (m, 1H), 7.35 (d, <i>J</i> = 8.03 Hz, 2H), 3.76 (s, 3H), 3.33 (m, 2H), 2.73–2.77 (m, 1H), 2.68–2.54 (m, 6H), 2.31–2.24 (m, 1H), 1.97 (s, 2H), 1.72–1.62 (m, 1H), 1.49–1.38 (m, 1H), 1.27 (m, 1H), 1.24–1.19 (m, 1H). MS <i>m</i>/<i>z</i>: 474.39 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_48" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(pyridin-3-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5azaspiro[2.4]heptane Dihydrochloride (<b>52</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography and the compound transformed into the hydrochloride to give <b>52</b>. Yield 34%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 11.38–11.12 (m, 1H), 8.94 (d, <i>J</i> = 2.01 Hz, 1H), 8.77 (dd, <i>J</i> = 1.51, 5.02 Hz, 1H), 8.21 (td, <i>J</i> = 1.88, 7.78 Hz, 2H), 7.76–7.54 (m, 3H), 7.65 (m, 2H), 7. 7.44 (d, <i>J</i> = 7.78, 2H), 3.72–3.60 (m, 4H), 3.46–2.89 (m, 7H), 2.66–2.56 (m, 1H), 2.48–2.18 (m, 2H), 2.16–1.94 (m, 3H), 1.48 (s, 2H). MS <i>m</i>/<i>z</i>: 474.39 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_49" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(pyridin-2-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>53</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>53</b>. Yield 58%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.71 (d, <i>J</i> = 5.02 Hz, 1H), 8.22 (d, <i>J</i> = 8.03 Hz, 1H), 7.98 (dt, <i>J</i> = 1.63, 7.72 Hz, Hz, 1H), 7.62 (d, <i>J</i> = 9.28 Hz, 2H), 7.51–7.45 (m, 1H), 7.35 (d, <i>J</i> = 8.28 Hz, 2H), 4.05 (s, 3H), 3.39–3.30 (m, 2H), 2.77–2.71 (m, 2H), 2.68–2.54 (m, 5H), 2.31–2.23 (m, 1H), 2.02–1.93 (m, 2H), 1.71–1.61 (m, 1H), 1.47–1.37 (m, 1H), 1.29–1.23 (m, 1H), 1.23–1.18 (m, 1H). MS <i>m</i>/<i>z</i>: 474.40 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_50" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3R)-5-(3-{[4-Methyl-5-(pyridin-2-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane Dihydrochloride (<b>54</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography and the compound transformed into the hydrochloride to give <b>54</b>. Yield 30%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.84–10.54 (m, 1H), 8.73 (d, <i>J</i> = 4.52 Hz, 1H), 8.12 (d, <i>J</i> = 7.78 Hz, 1H), 8.01 (t, <i>J</i> = 7.78 Hz 1H), 7.66 (d, <i>J</i> = 8.03 Hz, 2H), 7.56–7.51 (m, 1H), 7.44 (d, <i>J</i> = 8.03 Hz, 2H), 4.00–3.84 (m, 3H), 3.75–3.65 (m, 1H), 3.46–2.92 (m, 7H), 2.70–2.59 (m, 1H), 2.43–2.18 (m, 2H), 2.16–1.92 (m, 3H), 1.53–1.25 (m, 2H). MS <i>m</i>/<i>z</i>: 474.45 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_51" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(pyrazin-2-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>55</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>55</b>. Yield 25%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 9.38 (d, <i>J</i> = 1.25 Hz, 1H), 8.72 (m, 2H), 7.63 (d, <i>J</i> = 8.03 Hz, 2H), 7.52–7.33 (m, 2H), 4.02 (s, 3H), 3.50–3.40 (m, 2H), 3.15–2.89 (m, 3H), 2.47–2.31 (m, 1H), 2.19–2.11 (m, 1H), 2.02 (br s, 1H), 1.86–1.72 (m, 2H), 1.65–1.43 (m, 2H), 1.41–1.25 (m, 3H). MS <i>m</i>/<i>z</i>: 475.08 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_52" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(pyrazin-2-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>56</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>56</b>. Yield 54%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 9.39 (d, <i>J</i> = 1.51 Hz, 1H), 8.76–8.69 (m, 2H), 7.62 (d, <i>J</i> = 8.03 Hz, 2H), 7.39 (d, <i>J</i> = 8.28 Hz, 2H), 4.00 (s, 3H), 3.38–3.21 (m, 2H), 2.77–2.72 (m, 2H), 2.68–2.60 (m, 1H), 2.57–2.47 (m, 3H), 2.28–2.21 (m, 1H), 2.03–1.93 (m, 2H), 1.93–1.83 (m, 2H), 1.32–1.25 (m, 1H), 1.24–1.19 (m, 1H). MS <i>m</i>/<i>z</i>: 475.08 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_53" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(pyrimidin-4-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>57</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>57</b>. Yield 19%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 9.28 (d, <i>J</i> = 1.51 Hz, 1H), 8.95 (d, <i>J</i> = 5.27 Hz, 1H), 8.23 (dd, <i>J</i> = 1.51, 5.27 Hz 1H), 7.63 (d, <i>J</i> = 8.28 Hz, 2H), 7.45–7.31 (m, 2H), 4.09 (s, 3H), 3.47–3.34 (m, 2H), 2.75–2.56 (m, 4H), 2.39–2.26 (m, 1H), 2.05–1.97 (m, 3H), 1.80–1.63 (m, 1H), 1.55–1.40 (m, 1H), 1.37–1.19 (m, 3H). MS <i>m</i>/<i>z</i>: 475.52 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_54" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(pyrimidin-4-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>58</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>58</b>. Yield 55%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 9.31–9.27 (m, 1H), 8.99–8.92 (m, 1H), 8.26–8.22 (m, 1H), 7.65–7.59 (m, 2H), 7.42–7.36 (m, 2H), 4.07 (s, 3H), 3.39–3.22 (m, 2H), 2.76–2.71 (m, 2H), 2.68–2.59 (m, 1H), 2.56–2.46 (m, 3H), 2.27–2.21 (m, 1H), 2.03–1.83 (m, 4H), 1.32–1.26 (m, 1H), 1.22 (m, 1H). MS <i>m</i>/<i>z</i>: 475.08 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_55" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(pyridazin-3-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>59</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>59</b>. Yield 39%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 9.36–9.27 (m, 1H), 8.40 (dd, <i>J</i> = 1.63, 8.66 Hz, 1H), 7.89 (dd, <i>J</i> = 1.63, 8.66 Hz, 1H), 7.63 (d, <i>J</i> = 8.03 Hz, 2H), 7.36 (d, <i>J</i> = 8.28 Hz, 2H), 4.13 (s, 3H), 3.49–3.33 (m, 2H), 2.79–2.74 (m, 2H), 2.68–2.56 (m, 4H), 2.31–2.24 (m, 1H), 2.13–2.10 (m, 1H), 2.13–2.09 (m, 2H), 2.04–1.95 (m, 2H), 1.71–1.62 (m, 1H), 1.49–1.39 (m, 1H), 1.30–1.25 (m, 1H), 1.24–1.18 (m, 1H). MS <i>m</i>/<i>z</i>: 475.35 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_56" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> 1<i>R</i>,3<i>S</i>/1<i>S</i>,3R)-5-(3-{[4-Methyl-5-(pyridazin-3-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>60</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>60</b>. Yield 62%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 9.34–9.28 (m, 1H), 8.44–8.37 (m, 1H), 7.91–7.84 (m, 1H), 7.66–7.59 (m, 2H), 7.45–7.37 (m, 2H), 4.09 (s, 3H), 3.41–3.25 (m, 2H), 2.95–2.52 (m., 6H), 2.32–2.24 (m, 1H), 2.04–1.87 (m, 4H), 1.38–1.20 (m, 2H). MS <i>m</i>/<i>z</i>: 475.35 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_57" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> (1<i>S</i>,3<i>S</i>/1<i>R</i>,3R)-5-(3-{[4-Methyl-5-(pyridazin-4-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>61</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>61</b>. Yield 47%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 9.65 (m, 1H), 9.40 (m, 1H), 8.05 (m, 1H), 7.64 (d, <i>J</i> = 8.28 Hz, 2H), 7.36 (d, <i>J</i> = 8.28 Hz, 2H), 3.90 (s, 3H), 3.38 (m, 2H), 2.67–2.56 (m, 5H), 2.31–2.25 (m, 1H), 2.13–2.09 (m, 1H), 2.03–1.94 (m, 2H), 1.72–1.63 (m, 1H), 1.48–1.39 (m, 1H), 1.30–1.25 (m, 1H), 1.24–1.20 (m, 1H). MS <i>m</i>/<i>z</i>: 475.44 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_58" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(pyridazin-4-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>62</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>62</b>. Yield 63%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 9.66–9.60 (m, 1H), 9.42–9.37 (m, 1H), 8.08–8.02 (m, 1H), 7.66–7.60 (m, 2H), 7.43–7.36 (m, 2H), 3.86 (s, 3H), 3.36–3.21 (m, 2H), 2.73–2.45 (m, 4H), 2.27–2.21 (m, 1H), 2.15–2.08 (m, 1H), 2.03–1.83 (m, 5H), 1.33–1.19 (m, 2H). MS <i>m</i>/<i>z</i>: 475.36 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_59" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> (1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-5-(3-{[4-Methyl-5-(pyridin-3-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-phenyl-5-azaspiro[2.4]heptane (<b>63</b>)</h5><div class="NLM_p last">Following the general procedure, the crude residue was purified by flash chromatography to give <b>63</b>. Yield 47%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.97 (d, <i>J</i> = 1.76 Hz, 1H), 8.75 (dd, <i>J</i> = 1.63, 4.89 Hz, 1H), 8.17 (td, <i>J</i> = 1.98, 7.84 Hz, 1H),7.62–7.54 (m, 1H), 7.33–7.27 (m, 2H), 7.23–7.14 (m, 3H), 3.74 (s, 3H), 3.36–3.20 (m, 3H), 3.00–2.52 (m, 4H), 2.38–2.26 (m, 1H), 2.17 (m 1H), 2.04–1.87 (m, 3H), 1.27–1.11 (m, 2H), 1.19–1.12 (m, 1H). MS <i>m</i>/<i>z</i>: 406.39 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_60" class="NLM_sec NLM_sec_level_4"><div id="ac_i98" class="anchor-spacer"></div><h5 class="article-section__title" id="_i98"> (1<i>S</i>,3<i>S</i>)-1-[2-Fluoro-4-(trifluoromethyl)phenyl]-5-(3-{[4-methyl-5-(pyridin-3-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-5-azaspiro[2.4]heptane Dihydrochloride (<b>64b</b>)</h5><div class="NLM_p last">Following the general procedure, the compound was prepared to give <b>64</b> (data not reported), which was separated into the single enantiomers by preparative chiral HPLC (SFC) and the compound transformed into the hydrochloride to give <b>64b</b>. Column, Chiralpak AD-H (25 cm × 2.0 cm), 5 μm; mobile phase, (methanol + 0.1% isopropylamine) 22%; flow rate, 45 mL/min; DAD detection, 220 nm; loop, 700 μL of <b>64b</b> (enantiomer 2), retention time = 9.1 min. Yield 49%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.96 (d, <i>J</i> = 1.51 Hz, 1H), 8.79 (dd, <i>J</i> = 1.51, 4.77 Hz, 1H), 8.23 (d, <i>J</i> = 8.03 Hz, 1H), 7.60–7.56 (m, 1H), 7.49 (d, <i>J</i> = 8.28 Hz, 2H), 7.38–7.32 (m, 1H), 3.73 (s, 3H), 3.36–3.17 (m, 2H), 2.71–2.44 (m, 5H), 2.30–2.26 (m, 1H), 2.05–1.96 (m, 3H), 1.90–1.83 (m, 2H), 1.41–1.22 (m, 2H). MS <i>m</i>/<i>z</i>: 492.41 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_61" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> (1<i>S</i>,3<i>S</i>)-1-(2,4-Difluorophenyl)-5-(3-{[4-methyl-5-(pyridin-3-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-5-azaspiro[2.4]heptane (<b>65b</b>)</h5><div class="NLM_p last">Following the general procedure, the compound was prepared to give <b>65</b> (data not reported), which was separated into the single enantiomers by preparative chiral HPLC (SFC). Column, Chiralpak AD-H (25 cm × 2.0 cm) 5 μm; mobile phase, (methanol + 0.1% isopropylamine) 25%; flow rate, 45 mL/min; DAD detection, 220 nm; loop, 700 μL of <b>65b</b> (enantiomer 2), retention time = 11.6 min. Yield 39%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.99–8.94 (m, 1H), 8.77–8.72 (m, 1H), 8.19–8.14 (m, 1H), 7.62–7.55 (m, 1H), 7.20–7.12 (m, 1H), 7.04–6.90 (m, 2H), 3.74 (s, 3H), 3.35–3.20 (m, 2H), 2.78–2.74 (m, 1H), 2.66–2.59 (m, 1H), 2.56–2.47 (m, 2H), 2.38 (d, <i>J</i> = 9.03 Hz, 1H), 2.15–2.09 (m, 1H), 2.02–1.95 (m, 3H), 1.92–1.82 (m, 2H), 1.25–1.20 (m, 1H), 1.19–1.12 (m, 1H). MS <i>m</i>/<i>z</i>: 442.34 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_62" class="NLM_sec NLM_sec_level_4"><div id="ac_i100" class="anchor-spacer"></div><h5 class="article-section__title" id="_i100"> (1<i>R</i>,3<i>S</i>)-1-(4-Fluorophenyl)-5-(3-{[4-methyl-5-(pyridin-3-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-5-azaspiro[2.4]heptane (<b>66b</b>)</h5><div class="NLM_p last">Following the general procedure, the compound was prepared to give <b>66</b> (data not reported), which was separated into the single enantiomers by preparative chiral HPLC (SFC) to give <b>66b</b>. Column, Chiralpak AD-H (25 cm × 2.0 cm) 5 μm; mobile phase, (methanol + 0.1% isopropylamine) 25%; flow rate, 45 mL/min; DAD detection, 220 nm; loop, 700 μL of <b>66b</b> (enantiomer 2), retention time = 15.9 min. Yield 31%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.97 (d, <i>J</i> = 1.76 Hz, 1H), 8.74 (dd, <i>J</i> = 1.38, 4.89 Hz, 1H), 1H), 8.16 (d, <i>J</i> = 7.78 Hz 1H), 7.58 (m, 1H), 7.24–7.17 (m, 2H), 7.09–7.01 (m, 2H), 3.76–3.72 (m, 3H), 3.32–3.18 (m, 2H), 2.78–2.62 (m, 2H), 2.52 (m, 2H), 2.44–2.36 (m, 1H), 2.15–2.09 (m, 2H), 2.02–1.82 (m, 4H), 1.16–1.06 (m, 2H). MS <i>m</i>/<i>z</i>: 424.50 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_63" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> (1<i>R</i>,3<i>S</i>)-5-(3-{[4-Methyl-5-(2-methylpyridin-3-yl)-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane Dihydrochloride (<b>67a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography to give the free base that was transformed into the hydrochloride <b>67a</b>. Yield 18%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.99–10.72 (m, 1H), 8.77–8.70 (m, 1H), 8.14–8.04 (m, 1H), 7.66 (d, <i>J</i> = 8.03 Hz, 2H), 7.62–7.56 (m, 1H), 7.45 (d, <i>J</i> = 7.68 Hz, 2H), 3.73–3.63 (m, 2H), 3.48–3.36 (m, 4H), 3.34–3.07 (m, 6H), 3.01–2.91 (m, 1H), 2.71–2.59 (m, 1H), 2.44–2.17 (m, 2H), 2.09 (br s, 3H), 1.53–1.36 (m, 2H), 1.35–1.21 (m, 1H). MS <i>m</i>/<i>z</i>: 488.39 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_64" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> (1<i>R</i>,3<i>S</i>)-5-[3-({4-Methyl-5-[2-(trifluoromethyl)pyridin-3-yl]-4<i>H</i>-1,2,4-triazol-3-yl}sulfanyl)propyl]-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane Dihydrochloride (<b>68a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography to give the free base that was transformed into the hydrochloride <b>68a</b>. Yield 55%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.57–10.24 (m, 1H), 8.98 (d, <i>J</i> = 4.77 Hz, 1H), 8.25 (d, <i>J</i> = 7.78 Hz, 1H), 7.95 (m, 1H), 7.66 (d,, <i>J</i> = 8.03 Hz, 2H), 7.44 (m, 2H), 3.77–3.65 (m, 1H), 3.48–3.41 (m, 1H), 3.35 (br s, 3H), 3.26–3.19 (m, 6H), 3.04–2.63 (m, 1H), 2.45–2.20 (m, 2H), 2.16–1.92 (m, 3H), 1.53–1.27 (m, 2H). MS <i>m</i>/<i>z</i>: 542.40 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_65" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> (1<i>R</i>,3<i>S</i>)-5-(3-{[5-(2-Methoxypyridin-3-yl)-4-methyl-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane Dihydrochloride (<b>69a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography to give the free base that was transformed into the hydrochloride <b>69a</b>. Yield 52%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.88–10.39 (m, 1H), 8.46–8.34 (m, 1H), 7.87 (d, <i>J</i> = 7.34 Hz, 1H), 7.71–7.61 (m, 2H), 7.44 (d, <i>J</i> = 8.31 Hz, 2H), 7.15–7.24 (m, 1H), 3.97–3.86 (m, 3H), 3.82–2.60 (m, 11H), 2.59–2.32 (m, 1H), 2.31–1.89 (m, 4H), 1.54–1.26 (m, 2H), MS <i>m</i>/<i>z</i>: 504.37 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_66" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> 5-[4-Methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]pyridine-2-carbonitrile Dihydrochloride (<b>70a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography to give the free base that was transformed into the hydrochloride <b>70a</b>. Yield 30%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.70–10.41 (m, 1H), 9.11 (s, 1H), 8.42 (dd, <i>J</i> = 2.01, 8.03 Hz, 1H), 8.27 (d, <i>J</i> = 8.28 Hz, 1H), 7.66 (d, <i>J</i> = 7.78 Hz, 2H), 7.49–7.38 (m, 2H), 3.67 (d, <i>J</i> = 4.02 Hz, 3H), 3.47–3.06 (m, 7H), 3.01–2.61 (m, 1H), 2.50–2.45 (m, 1H), 2.44–2.20 (m, 2H), 2.15–1.92 (m, 3H), 1.53–1.26 (m, 2H). MS <i>m</i>/<i>z</i>: 499.45 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_67" class="NLM_sec NLM_sec_level_4"><div id="ac_i105" class="anchor-spacer"></div><h5 class="article-section__title" id="_i105"> (1<i>R</i>,3<i>S</i>)-5-[4-Methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]pyridine-2-carboxamide (<b>71a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography to give <b>71a</b>. Yield 26%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 9.02–8.97 (m, 1H), 8.36 (dd, <i>J</i> = 2.13, 8.16 Hz, 1H), 8.31–8.25 (m, 1H), 8.07–7.95 (m, 1H), 7.62 (d, <i>J</i> = 8.03 Hz, 2H), 7.39 (d, <i>J</i> = 8.03 Hz, 2H), 6.98–6.85 (m, 1H), 3.81–3.73 (m, 3H), 3.36–3.18 (m, 2H), 2.71–2.39 (m, 5H), 2.29–2.20 (m, 1H), 2.17–2.10 (m, 1H), 2.04–1.79 (m, 4H), 1.30 (br s, 1H), 1.23 (d, <i>J</i> = 7.53 Hz, 1H). MS <i>m</i>/<i>z</i>: 517.43 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_68" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> (1<i>R</i>,3<i>S</i>)-5-(3-{[5-(2,6-Dimethylpyridin-3-yl)-4-methyl-4<i>H</i>-1,2,4-triazol-3-yl]sulfanyl}propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>72a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography to give <b>72a</b>. Yield 71%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 7.70–7.66 (m, 1H), 7.64–7.59 (m, 2H), 7.44–7.38 (m, 2H), 7.26–7.21 (m, 1H), 3.45 (s, 3H), 3.36–3.18 (m, 3H), 2.90–2.82 (m, 1H), 2.74–2.57 (m, 3H), 2.55 (s, 3H), 2.39 (s, 3H), 2.34–2.25 (m, 1H), 2.15–2.09 (m, 1H), 2.05–1.98 (m, 3H), 1.97–1.89 (m, 2H), 1.38–1.21 (m, 2H). MS <i>m</i>/<i>z</i>: 502.43 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_69" class="NLM_sec NLM_sec_level_4"><div id="ac_i107" class="anchor-spacer"></div><h5 class="article-section__title" id="_i107"> 6-Methyl-5-[4-methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]pyridine-2-carboxamide (<b>73a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography to give <b>73a</b>. Yield 51%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 8.13 (br s, 1H), 8.05–8.01 (m, 1H), 8.00–7.96 (m, 1H), 7.76 (br s, 1H), 7.60 (d, <i>J</i> = 8.20 Hz, 2H), 7.32 (d, <i>J</i> = 8.20 Hz, 2H), 3.36 (s, 3H), 3.20–3.10 (m, 2H), 2.74–2.69 (m, 1H), 2.54–2.36 (m, 7H), 2.20 (dd, <i>J</i> = 8.30, 6.38 Hz, 1H), 1.98–1.82 (m, 3H), 1.81–1.71 (m, 2H), 1.27 (t, <i>J</i> = 5.63 Hz, 1H), 1.20–1.14 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO) δ: 165.4, 156.7, 152.9, 150.8, 150.6, 145.0, 139.4, 128.0, 125.8, 124.8, 124.6, 124.5, 119.0, 57.2, 54.0, 53.9, 35.5, 30.9, 30.5, 29.9, 28.7, 125.8, 22.7, 18.9. MS <i>m</i>/<i>z</i>: 531.39 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_70" class="NLM_sec NLM_sec_level_4"><div id="ac_i108" class="anchor-spacer"></div><h5 class="article-section__title" id="_i108"> 6-Methyl-5-[4-methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]pyridine-2-carboxylic Acid Formate (<b>74a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by FC on C<sub>18</sub> cartridge (eluent water + 0.1% FA to 60% water + 0.1% FA 40% MeOH + 0.1%) to give <b>74a</b>. Yield 18%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 8.15 (s, 1H), 8.13–8.10 (m, 1H), 7.61 (s, 2H), 7.43–7.36 (m, 2H), 3.54 (s, 3H), 3.37–3.18 (m, 4H), 2.72 (br s, 1H), 2.64–2.51 (m, 6H), 2.31–2.25 (m, 1H), 2.09 (br s, 2H), 2.02–1.93 (m, 2H), 1.90–1.86 (m, 1H), 1.35–1.28 (m, 1H), 1.26–1.21 (m, 1H). MS <i>m</i>/<i>z</i>: 532.37 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_71" class="NLM_sec NLM_sec_level_4"><div id="ac_i109" class="anchor-spacer"></div><h5 class="article-section__title" id="_i109"> 5-[4-Methyl-5-({3-[(1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]-1,2-dihydropyridin-2-one (<b>75</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, and the crude residue was purified by flash chromatography to give <b>75</b>. Yield 45%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 12.10–11.96 (m, 1H), 7.80–7.68 (m, 2H), 7.64–7.55 (m, 2H), 7.36–7.27 (m, 2H), 6.53–6.44 (m, 1H), 3.52 (s, 3H), 3.15–3.01 (m, 2H), 2.68 (br s, 1H), 2.48–2.33 (m, 4H), 2.26–2.16 (m, 1H), 2.02–1.81 (m, 3H), 1.73 (m, 2H), 1.27 (m, 1H), 1.18 (m, 1H). MS <i>m</i>/<i>z</i>: 490.51 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_72" class="NLM_sec NLM_sec_level_4"><div id="ac_i110" class="anchor-spacer"></div><h5 class="article-section__title" id="_i110"> 5-[4-Methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]-1,2-dihydropyridin-2-one (<b>75a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography to give <b>75a</b>. Yield 31%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 7.88 (d, <i>J</i> = 2.51 Hz, 1H), 7.75 (dd, <i>J</i> = 2.51, 9.54 Hz, 1H), 7.61 (d, <i>J</i> = 8.28 Hz 2H), 7.37 (d, <i>J</i> = 8.28 Hz, 2H), 6.52–6.46 (d, <i>J</i> = 9.29 1H), 3.66 (s, 3H), 3.27–3.10 (m, 3H), 2.78–2.68 (m, 2H), 2.65–2.56 (m, 1H), 2.47 (s, 3H), 2.27–2.18 (m, 1H), 2.02–1.92 (m, 2H), 1.87–1.75 (m, 2H), 1.30–1.25 (m, 1H), 1.22–1.17 (m, 1H). MS <i>m</i>/<i>z</i>: 490.42 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_73" class="NLM_sec NLM_sec_level_4"><div id="ac_i111" class="anchor-spacer"></div><h5 class="article-section__title" id="_i111"> 4-[4-Methyl-5-({3-[(1<i>R</i>,3<i>S</i>/1<i>S</i>,3<i>R</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]-1,2-dihydropyridin-2-one (<b>76</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, and crude residue was purified by flash chromatography to give <b>76</b>. Yield 17%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 7.61 (d, <i>J</i> = 8.28 Hz, 2H), 7.54 (d, <i>J</i> = 6.78 Hz, 1H), 7.32 (d, <i>J</i> = 8.03 Hz, 2H), 6.63 (d, <i>J</i> = 1.51 Hz, 1H), 6.53–6.46 (m, 1H), 3.61 (s, 3H), 3.19–3.07 (m, 2H), 2.74–2.65 (m, 1H), 2.49–2.35 (m, 5H), 2.24–2.17 (m, 1H), 1.98–1.83 (m, 3H), 1.79–1.70 (m, 2H), 1.30–1.24 (m, 1H), 1.21–1.13 (m, 1H). MS <i>m</i>/<i>z</i>: 490.41 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_74" class="NLM_sec NLM_sec_level_4"><div id="ac_i112" class="anchor-spacer"></div><h5 class="article-section__title" id="_i112"> 1-Methyl-5-[4-methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]-1,2-dihydropyridin-2-one Hydrochloride (<b>77a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography and then transformed into the hydrochloride to give <b>77a</b>. Yield 58%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.36 (s, 1H), 10.15–10.14 (m, 1H), 8.17 (br s, 1H), 7.79–7.61 (m, 3H), 7.45 (d, <i>J</i> = 7.70 Hz, 2H), 6.55 (d, <i>J</i> = 9.40 Hz, 1H), 3.71 (br s,2H), 3.48–3.64 (m, 6H), 3.47–3.24 (m, 2H), 3.18 (d, <i>J</i> = 4.6 Hz, 4H), 2.99 (br s, 1H), 2.26 (d, <i>J</i> = 5.00 Hz, 1H), 2.18–1.88 (m, 3H), 1.54–1.25 (m, 2H). MS <i>m</i>/<i>z</i>: 504.48 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_75" class="NLM_sec NLM_sec_level_4"><div id="ac_i113" class="anchor-spacer"></div><h5 class="article-section__title" id="_i113"> 1-Methyl-4-[4-methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]-1,2-dihydropyridin-2-one Hydrochloride (<b>78a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography and then transformed into the hydrochloride to give <b>78a</b>. Yield 82%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 7.77–7.72 (m, 1H), 7.65–7.59 (m, 2H), 7.43–7.36 (m, 2H), 6.73–6.68 (m, 1H), 6.63–6.56 (m, 1H), 3.76 (s, 3H), 3.58–3.51 (m, 3H), 3.35–3.15 (m, 2H), 2.58 (br s, 4H), 2.29–2.23 (m, 1H), 2.17 (br s, 1H), 2.02–1.81 (m, 5H), 1.35–1.29 (m, 1H), 1.26–1.20 (m, 1H). MS <i>m</i>/<i>z</i>: 504.33 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_76" class="NLM_sec NLM_sec_level_4"><div id="ac_i114" class="anchor-spacer"></div><h5 class="article-section__title" id="_i114"> 4-[4-Methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]piperidin-2-one Hydrochloride (Diastereomeric Mixture) (<b>79a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography and then transformed into the hydrochloride to give <b>79a</b>. Yield 36%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.25–9.96 (m, 1H), 7.68 (d, <i>J</i> = 8.03 Hz, 2H), 7.60 (br s, 1H), 7.44 (br s, 2H), 3.76–3.59 (m, 2H), 3.50 (s, 3H), 3.40 (br s, 2H), 3.11 (br s, 5H), 2.48–2.40 (m, 3H), 2.31–2.19 (m, 2H), 2.16–1.87 (m, 4H), 1.85–1.71 (m, 2H), 1.55–1.26 (m, 3H). MS <i>m</i>/<i>z</i>: 494.45 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_77" class="NLM_sec NLM_sec_level_4"><div id="ac_i115" class="anchor-spacer"></div><h5 class="article-section__title" id="_i115"> 1-Methyl-4-[4-methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]piperidin-2-one (Diastereomeric Mixture) (<b>80a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography and then transformed into the hydrochloride to give <b>80a</b>. Yield 55%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 7.63 (s, 2H), 7.46–7.33 (m, 2H), 3.62 (s, 3H), 3.52–3.35 (m, 3H), 3.25–3.04 (m, 2H), 2.91 (s, 3H), 2.74–2.33 (m, 7H), 2.31–2.19 (m, 2H), 2.06–1.69 (m, 6H), 1.36–1.20 (m, 2H). MS <i>m</i>/<i>z</i>: 508.41 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_78" class="NLM_sec NLM_sec_level_4"><div id="ac_i116" class="anchor-spacer"></div><h5 class="article-section__title" id="_i116"> 1-{4-[4-Methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]piperidin-1-yl}ethan-1-one (<b>81a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, but using (1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane, <i>cis</i> enantiomer 1, and not the racemic mixture; the crude residue was purified by flash chromatography on NH cartridge (eluting from cyclohexane to EtOAc 100%, then to MeOH 100%) to give <b>81a</b> as a colorless oil. Yield 79%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 7.60 (d, <i>J</i> = 8.19 Hz, 2H), 7.37 (d, <i>J</i> = 8.20 Hz, 2H), 4.48 (d, <i>J</i> = 13.08 Hz, 1H), 3.99 (d, <i>J</i> = 13.45 Hz, 1H), 3.57 (s, 3H), 3.32–3.21 (m, 1H), 3.20–3.01 (m, 3H), 2.93–2.35 (m, 6H), 2.22 (t, <i>J</i> = 7.15 Hz, 1H), 2.14–1.59 (m, 12H), 1.31–1.23 (m, 1H), 1.23–1.16 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO) δ: 168.0, 158.0, 149.1, 145.2, 128.0, 126.0, 124.8, 124.5, 57.2, 54.0, 53.9, 45.3, 35.6, 31.6, 30.5, 30.0, 30.0, 29.7, 29.4, 28.8, 28.0, 21.2, 19.0. MS <i>m</i>/<i>z</i>: 522.33 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_79" class="NLM_sec NLM_sec_level_4"><div id="ac_i117" class="anchor-spacer"></div><h5 class="article-section__title" id="_i117"> N-{4-[4-Methyl-5-({3-[(1<i>R</i>,3<i>S</i>)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptan-5-yl]propyl}sulfanyl)-4<i>H</i>-1,2,4-triazol-3-yl]cyclohexyl}acetamide (<b>82a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, and crude residue was purified by chiral HPLC to give <b>82a</b>. Column, Chiralcel OJ-H (25 cm × 2.0 cm) 5 μm; mobile phase, <i>n</i>-hexane/(ethanol/methanol + 0.1% isopropylamine) 85/15% v/v; flow rate, 16 mL/min; DAD detection, 220 nm; loop, 350 μL of <b>82a</b> (enantiomer 1), retention time = 8.7 min. Yield 55%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 7.61 (d, <i>J</i> = 8.31 Hz, 2H), 7.36 (d, <i>J</i> = 7.83 Hz, 2H), 7.13 (br s, 1H), 4.04–3.89 (m, 1H), 3.53 (s, 3H), 3.22–3.02 (m, 2H), 2.99–2.90 (m, 1H), 2.75–2.67 (m, 1H), 2.59 (q, <i>J</i> = 7.34 Hz, 1H), 2.52–2.38 (m, 3H), 2.22 (dd, <i>J</i> = 8.31, 6.36 Hz, 1H), 2.09–2.07 (m, 1H), 2.03–1.73 (m, 13H), 1.71–1.62 (m, 2H), 1.27 (t, <i>J</i> = 5.62 Hz, 1H), 1.20 (dd, <i>J</i> = 8.56, 5.14 Hz, 1H), MS <i>m</i>/<i>z</i>: 536.51 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_80" class="NLM_sec NLM_sec_level_4"><div id="ac_i118" class="anchor-spacer"></div><h5 class="article-section__title" id="_i118"> (1<i>R</i>,3<i>S</i>)-5-(3-{[4-Methyl-5-(oxan-4-yl)-4<i>H</i>-1,2,4-triazol-3-yl]-sulfanyl}-propyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>83a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, and crude residue was purified by chiral HPLC to give <b>83a</b>. Column, Chiralcel OJ-H (25 cm × 2.0 cm) 5 μm; mobile phase, <i>n</i>-hexane/ethanol 65/35% v/v; flow rate, 14 mL/min; DAD detection, 220 nm; loop, 1000 μL of <b>83a</b> (enantiomer 1), retention time = 8.4 min. Yield 55%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 7.63 (d, <i>J</i> = 8.03 Hz, 2H), 7.41 (s, 2H), 4.07–3.93 (m, 2H), 3.58 (s, 3H), 3.56–3.47 (m, 2H), 3.35–3.31 (m, 1H), 3.23–3.05 (m, 3H), 2.71–2.42 (m, 5H), 2.30–2.20 (m, 1H), 2.02–1.76 (m, 8H), 1.35–1.28 (m, 1H), 1.26–1.19 (m, 1H). MS <i>m</i>/<i>z</i>: 481.32 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_81" class="NLM_sec NLM_sec_level_4"><div id="ac_i119" class="anchor-spacer"></div><h5 class="article-section__title" id="_i119"> (1<i>R</i>,3<i>S</i>)-5-{3-[(4-Methyl-5-{8-oxabicyclo[3.2.1]octan-3-yl}-4<i>H</i>-1,2,4-triazol-3-yl)sulfanyl]propyl}-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>84a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, and crude residue was purified by chiral HPLC to give <b>84a</b>. Column, Chiralcel OJ-H (25 cm × 2.0 cm) 5 μm; mobile phase, <i>n</i>-hexane/(ethanol + 0.1% isopropylamine) 60/40% v/v; flow rate, 14 mL/min; DAD detection, 220 nm; loop, 2000 μL <b>84a</b> (enantiomer 1), retention time = 6.5 min. Yield 52%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 7.62 (d, <i>J</i> = 8.28 Hz, 2H), 7.39 ((d, <i>J</i> = 8.03 Hz, 2H), 4.48–4.33 (m, 2H), 3.57 (s, 3H), 3.46–3.28 (m, 2H), 3.22–3.03 (m, 2H), 2.72–2.38 (m, 5H), 2.29–2.20 (m, 1H), 1.99–1.93 (m, 7H), 1.86–1.69 (m, 5H), 1.35–1.27 (m, 1H), 1.25–1.17 (m, 1H). MS <i>m</i>/<i>z</i>: 507.46 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_82" class="NLM_sec NLM_sec_level_4"><div id="ac_i120" class="anchor-spacer"></div><h5 class="article-section__title" id="_i120"> (1<i>R</i>,3<i>S</i>)-5-{3-[(5-Cyclohexyl-4-methyl-4<i>H</i>-1,2,4-triazol-3-yl)-sulfanyl]propyl}-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane Hydrochloride (<b>85a</b>)</h5><div class="NLM_p last">The compound was prepared following the general procedure, and crude residue was purified by chiral HPLC to give <b>85a</b>. Column, Chiralcel OJ-H (25 cm × 2.0 cm) 5 μm; mobile phase, <i>n</i>-hexane/(ethanol/methanol 1/1 + 0.1% isopropylamine) 85/15% v/v; flow rate, 18 mL/min; DAD detection, 220 nm; loop, 1000 μL of <b>85a</b> (enantiomer 1), retention time = 6.6 min. Yield 32%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>) δ: 7.63 (d, <i>J</i> = 8.03 Hz, 2H), 7.39 (d, <i>J</i> = 7.28 Hz, 2H), 3.54 (s, 3H), 3.42 (m, 1H), 3.23–3.02 (m, 2H), 2.66–2.37 (m, 4H), 2.67–2.35 (m, 1H), 2.31–2.18 (m, 2H), 1.96 (br s, 4H), 1.83 (br s, 5H), 1.67–1.54 (m, 2H), 1.51–1.27 (m, 4H), 1.25–1.17 (m, 1H). MS <i>m</i>/<i>z</i>: 479.12 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_83" class="NLM_sec NLM_sec_level_4"><div id="ac_i121" class="anchor-spacer"></div><h5 class="article-section__title" id="_i121"> (1<i>R</i>,3<i>S</i>)-1-[4-(Trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>86</b>) and (1<i>S</i>,3<i>R</i>)-1-[4-(Trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>87</b>)</h5><div class="NLM_p last">(1<i>S</i>,3<i>R</i>/1<i>R</i>,3<i>S</i>)-5-Benzyl-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane-4,6-dione (diastereoisomer <i>cis</i>) was dissolved in THF, and LiAlH<sub>4</sub> 1 M in THF was added dropwise at 0 °C. The resulting orange solution was heated at reflux for 1 h. Then it was cooled with an ice bath and quenched with Na<sub>2</sub>SO<sub>4</sub>·10H<sub>2</sub>O until gas evolution ceased. The mixture was filtered over a pad of Celite washing with EtOAc, and the solution was concentrated to afford (1<i>R</i>,3<i>S</i>/1S,3<i>R</i>)-5-benzyl-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<i>cis</i>) as oil that submitted to Chiral Prep HPLC (SFC) to separate the enantiomers. Column, Chiralpak AD-H (25 cm × 2.1 cm) 5 μm; mobile phase, (ethanol + 0.1% isopropylamine) 7%; flow rate, 45 mL/min; DAD detection, 220 nm; loop, 900 μL of <b>86</b> (enantiomer 1), retention time = 7.9 min, 100% ee; <b>87</b> (enantiomer 2) retention time 10.2 min, 100% ee. Yield 52%. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ: 7.54 (d, <i>J</i> = 8.03 Hz, 2H), 7.19 (d, <i>J</i> = 8.28 Hz, 2H), 3.08 (dt, <i>J</i> = 4.39, 6.84 Hz, 2H), 2.70 (d, <i>J</i> = 11.04 Hz, 1H), 2.52 (d, <i>J</i> = 11.29 Hz, 1H), 2.21 (dd, <i>J</i> = 6.15, 8.41 Hz, 1H), 1.93 (dt, <i>J</i> = 2.76, 6.78 Hz, 2H), 1.29 (dd, <i>J</i> = 5.27, 8.53 Hz, 1H), 1.17 (t, <i>J</i> = 5.77 Hz, 1H). MS <i>m</i>/<i>z</i>: 242.0 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2_84" class="NLM_sec NLM_sec_level_4"><div id="ac_i122" class="anchor-spacer"></div><h5 class="article-section__title" id="_i122"> (1<i>R</i>,3<i>S</i>)-5-(4-Methylbenzenesulfonyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>88</b>)</h5><div class="NLM_p last"><b>86</b> in dichloromethane (3 mL) was stirred at 0 °C; triethylamine (0.022 mL, 0.15 mmol) was added, followed by 4-methylbenzenesulfonyl chloride (21 mg, 0.11 mmol), and then the mixture was slowly warmed to room temperature and stirred at the same temperature for 1 h. CH<sub>2</sub>Cl<sub>2</sub> was added, washed with water and brine, and then dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was chromatographed by flash chromatography on silica gel (eluent from cyclohexane to 40% ethyl acetate), affording <b>88</b> (33 mg, <i>y</i> = 83%) as white solid. The latter was suspended in 0.3 mL of EtOH and then heated until dissolution. After slow cooling to RT, crystallization was observed. Crystals were filtered and used for the molecular and crystal structure determination by single-crystal high-resolution X-ray diffraction to determine the absolute stereochemistry.</div></div><div id="sec5_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i123" class="anchor-spacer"></div><h5 class="article-section__title" id="_i123"> X-ray Crystallography</h5><div class="NLM_p">The X-ray data collection was performed on a plate-like crystal of approximate dimensions 0.26 × 0.22 × 0.02 mm<sup>3</sup> mounted on a glass capillary. The X-ray intensities were measures on a Bruker Smart system equipped with an APEXII CCD area detector, using the Mo Kα radiation (λ = 0.71073 Å) at <i>T</i> = 293 K. The collected intensities were corrected for Lorentz and polarization factors and empirically for absorption by using the SADABS program.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Crystals diffracted weakly, and the final collection did not reach high Θ values, nevertheless it was possible to determine unambiguously the molecular structure of the compound. The structure was solved by direct methods using the program Sir2011<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and was refined with the program SHELXL-2014.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Anisotropic thermal displacement parameters were refined for all the non-H atoms, apart from the terminal −Ph-CH<sub>3</sub> and −Ph-CF<sub>3</sub> groups. The trifluoromethyl groups were disordered over two positions. Hydrogen atoms were introduced in calculated positions. Crystal data are reported in <a class="ref internalNav" href="#tbl12" aria-label="Table 12">Table 12</a>.</div><div class="NLM_table-wrap" id="tbl12"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 12. Crystal Data</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">empirical formula</td><td class="colsep0 rowsep0" align="left">C<sub>80</sub>H<sub>80</sub>F<sub>12</sub>N<sub>4</sub>O<sub>8</sub>S<sub>4</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">formula weight</td><td class="colsep0 rowsep0" align="left">1581.72</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">temperature (K)</td><td class="colsep0 rowsep0" align="left">293</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">crystal system</td><td class="colsep0 rowsep0" align="left">orthorhombic</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i> (Å)</td><td class="colsep0 rowsep0" align="left">11.75(5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>b</i> (Å)</td><td class="colsep0 rowsep0" align="left">17.30(7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>c</i> (Å)</td><td class="colsep0 rowsep0" align="left">41.04(9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α (deg)</td><td class="colsep0 rowsep0" align="left">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">β (deg)</td><td class="colsep0 rowsep0" align="left">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">γ (deg)</td><td class="colsep0 rowsep0" align="left">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">volume (Å<sup>3</sup>)</td><td class="colsep0 rowsep0" align="left">8342(52)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>Z</i>, <i>Z</i>′</td><td class="colsep0 rowsep0" align="left">16, 4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ρ<sub>calcd</sub> (g/cm<sup>3</sup>)</td><td class="colsep0 rowsep0" align="left">1.259</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">μ (mm<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">0.194</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>F</i> (000)</td><td class="colsep0 rowsep0" align="left">3296</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">crystal size (mm<sup>3</sup>)</td><td class="colsep0 rowsep0" align="left">0.26 × 0.22 × 0.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">radiation (Å)</td><td class="colsep0 rowsep0" align="left">Mo Kα (λ = 0.71073)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2Θ range for data collection (deg)</td><td class="colsep0 rowsep0" align="left">2.540–34.540</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">index ranges</td><td class="colsep0 rowsep0" align="left">–9 ≤ <i>h</i> ≤ 9, −14 ≤ <i>k</i> ≤ 12, −33 ≤ <i>l</i> ≤ 28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">reflections collected</td><td class="colsep0 rowsep0" align="left">17849</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">independent reflections</td><td class="colsep0 rowsep0" align="left">4938 [<i>R</i><sub>int</sub> = 0.1815, <i>R</i><sub>sigma</sub> = 0.1678]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">data/restraints/parameters</td><td class="colsep0 rowsep0" align="left">4938/74/686</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">goodness-of-fit on <i>F</i><sup>2</sup></td><td class="colsep0 rowsep0" align="left">1.003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">final <i>R</i> indexes [<i>I</i> ≥ 2σ (<i>I</i>)]</td><td class="colsep0 rowsep0" align="left"><i>R</i><sub>1</sub> = 0.0786, <i>wR</i><sub>2</sub> = 0.1598</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">final <i>R</i> indexes [all data]</td><td class="colsep0 rowsep0" align="left"><i>R</i><sub>1</sub> = 0.1766, <i>wR</i><sub>2</sub> = 0.2100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">largest Δ<i>F</i> max/min/e Å<sup>–3</sup></td><td class="colsep0 rowsep0" align="left">0.20/–0.20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Flack parameter</td><td class="colsep0 rowsep0" align="left">0.1(2)</td></tr></tbody></table></div></div><div class="NLM_p">The compound crystallizes in the chiral orthorhombic space group <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>. The asymmetric unit comprises four molecules. The Flack parameter for the present structure is 0.094(322) by classical fit to all intensities and 0.064(210) from 639 selected quotients (Parson’s method) strongly supporting the present absolute structure determination. According to the absolute structure determination, the configuration is 1<i>R</i>,3<i>S</i>.</div><div class="NLM_p last"><sup>1</sup>H NMR (CDCl<sub>3</sub>) δ: 7.50–7.64 (m, 2H), 7.46 (d, <i>J</i> = 8.03 Hz, 2H), 7.22 (d, <i>J</i> = 7.78 Hz, 2H), 6.99 (d, <i>J</i> = 8.53 Hz, 2H), 3.55 (ddd, <i>J</i> = 4.52, 7.47, 9.35 Hz, 1H), 3.39–3.25 (m, 1H), 3.14 (d, <i>J</i> = 10.04 Hz, 1H), 2.74 (d, <i>J</i> = 10.04 Hz, 1H), 2.45 (s, 3H), 2.06 (dd, <i>J</i> = 5.14, 7.65 Hz, 2H), 1.82 (ddd, <i>J</i> = 4.64, 7.09, 12.11 Hz, 1H), 1.19 (dd, <i>J</i> = 5.90, 8.91 Hz, 1H), 1.14–1.00 (m, 1H). MS <i>m</i>/<i>z</i>: 396.4 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i125" class="anchor-spacer"></div><h5 class="article-section__title" id="_i125"> (1<i>S</i>,3<i>R</i>)-5-(4-Methylbenzenesulfonyl)-1-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>89</b>)</h5><div class="NLM_p last"><b>87</b> (3.48 mmol) in dichloromethane (15 mL) was stirred at 0 °C; triethylamine (0.73 mL, 5.22 mmol) was added, followed by 4-methylbenzenesulfonyl chloride (730 mg, 3.83 mmol), and then the reaction mixture was slowly warmed to room temperature and stirred at that temperature for 2 h. CH<sub>2</sub>Cl<sub>2</sub> was added, washed with water and brine, and then dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was chromatographed by flash chromatography on silica gel (eluent from cyclohexane to 40% ethyl acetate), affording <b>89</b> as white solid. Then 100 mg of <b>89</b> were suspended in 1 mL of EtOH and then heated until dissolution. After slow cooling to RT, the solution was left standing at RT for 3 days, after which time crystallization was observed. Crystals were filtered and used for the molecular and crystal structure determination by single-crystal high-resolution X-ray diffraction to determine the absolute stereochemistry.</div></div><div id="sec5_4_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i126" class="anchor-spacer"></div><h5 class="article-section__title" id="_i126"> X-ray Crystallography</h5><div class="NLM_p">The X-ray data collection was performed on a plate-like crystal of approximate dimensions 0.31 × 0.24 × 0.07 mm<sup>3</sup> mounted on a glass capillary. The X-ray intensities were measures on a Bruker Smart system equipped with an APEXII CCD area detector, using the Mo Kα radiation (λ = 0.71073 Å) at <i>T</i> = 293 K. The collected intensities were corrected for Lorentz and polarization factors and empirically for absorption by using the SADABS program.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Crystals diffracted weakly, and the final collection did not reach high Θ values, nevertheless it was possible to determine unambiguously the molecular structure of the compound. The structure was solved by direct methods using the program Sir2011<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and was refined with the program SHELXL-2014.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Anisotropic thermal displacement parameters were refined for all the non-H atoms, apart from mthe terminal −CH<sub>3</sub> and −CF<sub>3</sub> groups. The latter ones were disordered over two positions. Hydrogen atoms were introduced in calculated positions. Crystal data are reported in <a class="ref internalNav" href="#tbl13" aria-label="Table 13">Table 13</a>.</div><div class="NLM_table-wrap" id="tbl13"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 13. Crystal Data</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">empirical formula</td><td class="colsep0 rowsep0" align="left">C<sub>80</sub>H<sub>80</sub>F<sub>12</sub>N<sub>4</sub>O<sub>8</sub>S<sub>4</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">formula weight</td><td class="colsep0 rowsep0" align="left">1581.72</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">temperature (K)</td><td class="colsep0 rowsep0" align="left">293</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">crystal system</td><td class="colsep0 rowsep0" align="left">orthorhombic</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i> (Å)</td><td class="colsep0 rowsep0" align="left">11.567(7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>b</i> (Å)</td><td class="colsep0 rowsep0" align="left">17.155(9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>c</i> (Å)</td><td class="colsep0 rowsep0" align="left">40.16(2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α (deg)</td><td class="colsep0 rowsep0" align="left">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">β (deg)</td><td class="colsep0 rowsep0" align="left">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">γ (deg)</td><td class="colsep0 rowsep0" align="left">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">volume (Å<sup>3</sup>)</td><td class="colsep0 rowsep0" align="left">7970(8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>Z</i>, <i>Z</i>′</td><td class="colsep0 rowsep0" align="left">16, 4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ρ<sub>calcd</sub> (g/cm<sup>3</sup>)</td><td class="colsep0 rowsep0" align="left">1.318</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">μ (mm<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">0.203</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>F</i> (000)</td><td class="colsep0 rowsep0" align="left">760.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">crystal size (mm<sup>3</sup>)</td><td class="colsep0 rowsep0" align="left">0.31 × 0.24 × 0.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">radiation (Å)</td><td class="colsep0 rowsep0" align="left">Mo Kα (λ = 0.71073)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2Θ range for data collection (deg)</td><td class="colsep0 rowsep0" align="left">1.291–38.640</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">index ranges</td><td class="colsep0 rowsep0" align="left">–10 ≤ <i>h</i> ≤ 10, −15 ≤ <i>k</i> ≤ 15, −37 ≤ <i>l</i> ≤ 37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">reflections collected</td><td class="colsep0 rowsep0" align="left">47185</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">independent reflections</td><td class="colsep0 rowsep0" align="left">6706 [<i>R</i><sub>int</sub> = 0.0941, <i>R</i><sub>sigma</sub> = 0.0494]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">data/restraints/parameters</td><td class="colsep0 rowsep0" align="left">6706/56/926</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">goodness-of-fit on <i>F</i><sup>2</sup></td><td class="colsep0 rowsep0" align="left">1.013</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">final <i>R</i> indexes [<i>I</i> ≥ 2σ (<i>I</i>)]</td><td class="colsep0 rowsep0" align="left"><i>R</i><sub>1</sub> = 0.0623, <i>wR</i><sub>2</sub> = 0.1419</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">final <i>R</i> indexes [all data]</td><td class="colsep0 rowsep0" align="left"><i>R</i><sub>1</sub> = 0.0993, <i>wR</i><sub>2</sub> = 0.1665</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">largest Δ<i>F</i> max/min (e Å<sup>–3</sup>)</td><td class="colsep0 rowsep0" align="left">0.26/–0.24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Flack parameter</td><td class="colsep0 rowsep0" align="left">–0.01(6)</td></tr></tbody></table></div></div><div class="NLM_p">The compound crystallizes in the chiral orthorhombic space group <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>. The asymmetric unit comprises four molecules. The Flack parameter for the present structure is 0.000(199) by classical fit to all intensities and −0.005(57) from 1566 selected quotients (Parson’s method) strongly supporting the present absolute structure determination. According to the absolute structure determination, the configuration is 1<i>S</i>,3<i>R</i>.</div><div class="NLM_p"><sup>1</sup>H NMR (CDCl<sub>3</sub>) δ: 7.50–7.64 (m, 2H), 7.46 (d, <i>J</i> = 8.03 Hz, 2H), 7.22 (d, <i>J</i> = 7.78 Hz, 2H), 6.99 (d, <i>J</i> = 8.53 Hz, 2H), 3.55 (ddd, <i>J</i> = 4.52, 7.47, 9.35 Hz, 1H), 3.39–3.25 (m, 1H), 3.14 (d, <i>J</i> = 10.04 Hz, 1H), 2.74 (d, <i>J</i> = 10.04 Hz, 1H), 2.45 (s, 3H), 2.06 (dd, <i>J</i> = 5.14, 7.65 Hz, 2H), 1.82 (ddd, <i>J</i> = 4.64, 7.09, 12.11 Hz, 1H), 1.19 (dd, <i>J</i> = 5.90, 8.91 Hz, 1H), 1.14–1.00 (m, 1H)MS <i>m</i>/<i>z</i>: 396.4 [M + H]<sup>+</sup>.</div><div class="NLM_p last">Crystallographic data (excluding structure factors) for <b>89</b> and <b>87</b> have been deposited with the Cambridge Crystallographic Data Centre as supplementary publications nos. CCDC 1489497–1489498. Copies of the data can be obtained free of charge on application to CCDC, 12 Union Road, Cambridge CB2 1EZ, UK (fax: (+44) 1223–336–033; e-mail: <a href="/cdn-cgi/l/email-protection#3b5f5e4b5448524f7b58585f5815585a56155a58154e50"><span class="__cf_email__" data-cfemail="a2c6c7d2cdd1cbd6e2c1c1c6c18cc1c3cf8cc3c18cd7c9">[email protected]</span></a>).</div></div></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i128" class="anchor-spacer"></div><h3 class="article-section__title" id="_i128"> Computational Modeling</h3><div class="NLM_p">A model of the DA D3 receptor (D3R) X-ray crystal structure (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL">3PBL</a>), pretreated using “Structure Preparation” application within MOE,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> was built as described in ref <a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">13</a>. Compounds here described were docked in the minimized model by a pharmacophore guided induced fit docking protocol, consisting of an H bond donor (to Asp110) and an aromatic center (located into the Ptm23 pocket) features as placement constraints. Ambe12: EHT and “Reaction field” were used as force field and implicit solvent model, respectively. Best poses were further submitted to conformational search using LowModeMD simulation. This docking model and protocol previously described<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> provided good predictive validity in discriminating potent (p<i>K</i><sub>i</sub> > 7.5) from relatively weaker (p<i>K</i><sub>i</sub> < 7.0) DA D3 receptor antagonists, the latter showing no meaningful poses to the D3R model.</div><div class="NLM_p">As far as <b>40a</b>–<b>42a</b> were concerned, the three molecules were docked into the DA D3 receptor model in agreement with the previously reported protocol<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> using the GBVI/WSA dG scoring function as implemented in MOE.<a onclick="showRef(event, 'ref21 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref21 ref30 ref31">(21, 30, 31)</a> The lowest score poses were submitted to conformational search using LowModeMD (MOE). The lowest energy conformations were further minimized, and ligand–protein affinity was recalculated. Results are reported below in <a class="ref internalNav" href="#tbl14" aria-label="Table 14">Table 14</a>.</div><div class="NLM_table-wrap" id="tbl14"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 14. GBVI/WSA: Free Energy of Binding of the Ligand Which Uses GB/VI Solvation Model</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">affinity: GBVI/WSA dg (kcal/mol)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char=".">docking (lowest energy pose)</th><th class="colsep0 rowsep0" align="center" char=".">LowModeMD (lowest energy complex)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40a</b></td><td class="colsep0 rowsep0" align="left">4-benzamide (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char=".">–10.5</td><td class="colsep0 rowsep0" align="char" char=".">–12.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41a</b></td><td class="colsep0 rowsep0" align="left">3- benzamide (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char=".">–10.5</td><td class="colsep0 rowsep0" align="char" char=".">–11.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42a</b></td><td class="colsep0 rowsep0" align="left">2- benzamide (<i>cis</i>, se 1)</td><td class="colsep0 rowsep0" align="char" char=".">–9.7</td><td class="colsep0 rowsep0" align="char" char=".">–10.8</td></tr></tbody></table></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i130"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00972">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50123" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50123" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b00972" class="ext-link">10.1021/acs.jmedchem.6b00972</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Key intermediate preparation and their NMR spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00972/suppl_file/jm6b00972_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00972/suppl_file/jm6b00972_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00972/suppl_file/jm6b00972_si_001.pdf">jm6b00972_si_001.pdf (1.22 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00972/suppl_file/jm6b00972_si_002.csv">jm6b00972_si_002.csv (13.9 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.6b00972" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55724" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55724" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fabrizio Micheli</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#86e0e7e4f4effcefe9a8ebefe5eee3eaefc6e7f6f2f3eff2a8e5e9eb"><span class="__cf_email__" data-cfemail="2147404353485b484e0f4c484249444d48614051555448550f424e4c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alessia Bacchi</span> - <span class="hlFld-Affiliation affiliation">Dipartimento di Chimica, Università
di Parma, Viale delle
Scienze, 17/A, Biopharmanet-tec, Viale delle Scienze, 27/A, Campus, I-43124 Parma, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simone Braggio</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura Castelletti</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Palmina Cavallini</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paolo Cavanni</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Susanna Cremonesi</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michele Dal Cin</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aldo Feriani</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sylvie Gehanne</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mahmud Kajbaf</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luciano Marchió</span> - <span class="hlFld-Affiliation affiliation">Dipartimento di Chimica, Università
di Parma, Viale delle
Scienze, 17/A, Biopharmanet-tec, Viale delle Scienze, 27/A, Campus, I-43124 Parma, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Selena Nola</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Beatrice Oliosi</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Annalisa Pellacani</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elisabetta Perdonà</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anna Sava</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Teresa Semeraro</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luca Tarsi</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Silvia Tomelleri</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Wong</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Filippo Visentini</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura Zonzini</span> - <span class="hlFld-Affiliation affiliation">Aptuit s.r.l., Via Fleming 4, 37135 Verona, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian Heidbreder</span> - <span class="hlFld-Affiliation affiliation">Indivior Inc., The Fairfax Building, 10710 Midlothian
Turnpike, Suite 430, Richmond Virginia 23235, United
States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d126e13756-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i131">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03032" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03032" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We acknowledge Indivior, Inc., for supporting this project as well as all the colleagues of the Aptuit departments who helped in generating the data reported in this manuscript and for the fruitful discussion during the preparation of this manuscript.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i132" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i132"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i133" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i133"> Abbreviations Used</h2><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">human ether-à-go-go related gene</p></td></tr><tr><td class="NLM_term">NCE</td><td class="NLM_def"><p class="first last">novel chemical entity</p></td></tr><tr><td class="NLM_term">fp<i>K</i><sub>i</sub></td><td class="NLM_def"><p class="first last">functional p<i>K</i><sub>i</sub></p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">CYP P450</td><td class="NLM_def"><p class="first last">cytochrome P450</p></td></tr><tr><td class="NLM_term">hCl<sub>i</sub></td><td class="NLM_def"><p class="first last">human intrinsic clearance</p></td></tr><tr><td class="NLM_term">rCl<sub>i</sub></td><td class="NLM_def"><p class="first last">rat intrinsic clearance</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">molecular weight</p></td></tr><tr><td class="NLM_term">cLogP</td><td class="NLM_def"><p class="first last">calculated log <i>P</i></p></td></tr><tr><td class="NLM_term">PSA</td><td class="NLM_def"><p class="first last">polar surface area</p></td></tr><tr><td class="NLM_term"><i>F</i>%</td><td class="NLM_def"><p class="first last">bioavailability</p></td></tr><tr><td class="NLM_term"><i>F</i><sub>a</sub></td><td class="NLM_def"><p class="first last">fraction absorbed</p></td></tr><tr><td class="NLM_term"><i>E</i><sub>h</sub></td><td class="NLM_def"><p class="first last">hepatic extraction</p></td></tr><tr><td class="NLM_term">B/B</td><td class="NLM_def"><p class="first last">brain/blood</p></td></tr><tr><td class="NLM_term">Cl<sub>b</sub></td><td class="NLM_def"><p class="first last">blood clearance</p></td></tr><tr><td class="NLM_term"><i>V</i><sub>ss</sub></td><td class="NLM_def"><p class="first last">distribution. volume at steady state</p></td></tr><tr><td class="NLM_term">DA</td><td class="NLM_def"><p class="first last">dopamine</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">ethoxyresorufin</p></td></tr><tr><td class="NLM_term">FCA</td><td class="NLM_def"><p class="first last">7-methoxy-4-trifluoromethylcoumarin-3-acetic acid</p></td></tr><tr><td class="NLM_term">BMC</td><td class="NLM_def"><p class="first last">3-butyryl-7-methoxycoumarin</p></td></tr><tr><td class="NLM_term">MMC</td><td class="NLM_def"><p class="first last">4-methylaminomethyl-7-methoxycoumarin</p></td></tr><tr><td class="NLM_term">DEF</td><td class="NLM_def"><p class="first last">diethoxyflourescein</p></td></tr><tr><td class="NLM_term">7-BQ</td><td class="NLM_def"><p class="first last">7-benzyloxyquinoline</p></td></tr><tr><td class="NLM_term">T3P</td><td class="NLM_def"><p class="first last">propylphosphonic anhydride</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i134">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08730" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08730" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 31 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Heidbreder, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xi, Z.-X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thanos, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mugnaini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagan, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashby, C. R.,  Jr.</span><span> </span><span class="NLM_article-title">The Role of Central DA D<sub>3</sub> Receptors in Drug Addiction: a Review of Pharmacological Evidence</span> <span class="citation_source-journal">Brain Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">77</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1016/j.brainresrev.2004.12.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1016%2Fj.brainresrev.2004.12.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=15960988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlt1Ghu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2005&pages=77-105&author=C.+A.+Heidbrederauthor=E.+L.+Gardnerauthor=Z.-X.+Xiauthor=P.+K.+Thanosauthor=M.+Mugnainiauthor=J.+J.+Haganauthor=C.+R.+Ashby&title=The+Role+of+Central+DA+D3+Receptors+in+Drug+Addiction%3A+a+Review+of+Pharmacological+Evidence&doi=10.1016%2Fj.brainresrev.2004.12.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The role of central dopamine D3 receptors in drug addiction: a review of pharmacological evidence</span></div><div class="casAuthors">Heidbreder, Christian A.; Gardner, Eliot L.; Xi, Zheng-Xiong; Thanos, Panayotis K.; Mugnaini, Manolo; Hagan, Jim J.; Ashby, Charles R.</div><div class="citationInfo"><span class="NLM_cas:title">Brain Research Reviews</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">77-105</span>CODEN:
                <span class="NLM_cas:coden">BRERD2</span>;
        ISSN:<span class="NLM_cas:issn">0165-0173</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The cDNA for the dopamine D3 receptor was isolated and characterized in 1990.  Subsequent studies have indicated that D3 receptors, as well as D3 receptor mRNA, are primarily localized in limbic regions in mammals.  This finding led to the postulate that D3 receptors may be involved in drug dependence and addiction.  However, this hypothesis has been difficult to test due to the lack of compds. with high selectivity for central D3 receptors.  The interpretation of results from studies using mixed D2/D3 agonists and/or antagonists is problematic because these agents have low selectivity for D3 over D2 receptors and it is likely that their actions are primarily related to D2 receptor antagonism and possibly interaction with other neurotransmitter receptors.  Currently, with the synthesis and characterization of new highly selective D3 receptor antagonists such as SB-277011-A this difficulty has been surmounted.  The purpose of the present article is to review, for the first time, the effects of various putative D3 receptor selective compds. in animal models of drug dependence and addiction.  The results obtained with highly selective D3 receptor antagonists such as SB-277011-A, SB-414796, and NGB-2904 indicate that central D3 receptors may play an important role in drug-induced reward, drug-taking, and cue-, drug-, and stress-induced reinstatement of drug-seeking behavior.  Provided these results can be extrapolated to human drug addicts, they suggest that selective DA D3 receptor antagonists may prove effective as potential pharmacotherapeutic agents to manage drug dependence and addiction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwzEJPprm60rVg90H21EOLACvtfcHk0ljzXyH44iZ6Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlt1Ghu7k%253D&md5=3e5948e0f94515fd240d1e3d18812028</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.brainresrev.2004.12.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.brainresrev.2004.12.033%26sid%3Dliteratum%253Aachs%26aulast%3DHeidbreder%26aufirst%3DC.%2BA.%26aulast%3DGardner%26aufirst%3DE.%2BL.%26aulast%3DXi%26aufirst%3DZ.-X.%26aulast%3DThanos%26aufirst%3DP.%2BK.%26aulast%3DMugnaini%26aufirst%3DM.%26aulast%3DHagan%26aufirst%3DJ.%2BJ.%26aulast%3DAshby%26aufirst%3DC.%2BR.%26atitle%3DThe%2520Role%2520of%2520Central%2520DA%2520D3%2520Receptors%2520in%2520Drug%2520Addiction%253A%2520a%2520Review%2520of%2520Pharmacological%2520Evidence%26jtitle%3DBrain%2520Res.%2520Rev.%26date%3D2005%26volume%3D49%26spage%3D77%26epage%3D105%26doi%3D10.1016%2Fj.brainresrev.2004.12.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Heidbreder, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">Current Perspectives on Selective Dopamine D3 Receptor Antagonists as Pharmacotherapeutics for Addictions and Related Disorders</span> <span class="citation_source-journal">Ann. N. Y. Acad. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1187</span><span class="NLM_x">, </span> <span class="NLM_fpage">4</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1111/j.1749-6632.2009.05149.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1111%2Fj.1749-6632.2009.05149.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1187&publication_year=2010&pages=4-34&author=C.+A.+Heidbrederauthor=A.+H.+Newman&title=Current+Perspectives+on+Selective+Dopamine+D3+Receptor+Antagonists+as+Pharmacotherapeutics+for+Addictions+and+Related+Disorders&doi=10.1111%2Fj.1749-6632.2009.05149.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1111%2Fj.1749-6632.2009.05149.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1749-6632.2009.05149.x%26sid%3Dliteratum%253Aachs%26aulast%3DHeidbreder%26aufirst%3DC.%2BA.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DCurrent%2520Perspectives%2520on%2520Selective%2520Dopamine%2520D3%2520Receptor%2520Antagonists%2520as%2520Pharmacotherapeutics%2520for%2520Addictions%2520and%2520Related%2520Disorders%26jtitle%3DAnn.%2520N.%2520Y.%2520Acad.%2520Sci.%26date%3D2010%26volume%3D1187%26spage%3D4%26epage%3D34%26doi%3D10.1111%2Fj.1749-6632.2009.05149.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Heidbreder, C.</span><span> </span><span class="NLM_article-title">Rationale in Support of the Use of Selective Dopamine D3 Receptor Antagonists for the Pharmacotherapeutic Management of Drug Addiction</span> <span class="citation_source-journal">Naunyn-Schmiedeberg's Arch. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">386</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">176</span><span class="refDoi"> DOI: 10.1007/s00210-012-0803-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1007%2Fs00210-012-0803-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=23104235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ejur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2013&pages=167-176&author=C.+Heidbreder&title=Rationale+in+Support+of+the+Use+of+Selective+Dopamine+D3+Receptor+Antagonists+for+the+Pharmacotherapeutic+Management+of+Drug+Addiction&doi=10.1007%2Fs00210-012-0803-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Rationale in support of the use of selective dopamine D3 receptor antagonists for the pharmacotherapeutic management of substance use disorders</span></div><div class="casAuthors">Heidbreder, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Naunyn-Schmiedeberg's Archives of Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">386</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">167-176</span>CODEN:
                <span class="NLM_cas:coden">NSAPCC</span>;
        ISSN:<span class="NLM_cas:issn">0028-1298</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Growing evidence indicates that dopamine (DA) D3 receptors are involved in the control of drug-seeking behavior and may play an important role in the pathophysiol. of substance use disorders.  First, DA D3 receptors are distributed in strategic areas belonging to the mesolimbic DA system such as the ventral striatum, midbrain, and pallidum, which have been assocd. with behaviors controlled by the presentation of drug-assocd. cues.  Second, repeated exposure to drugs of abuse has been shown to produce neuroadaptations in the DA D3 system.  Third, the synthesis and characterization of highly potent and selective DA D3 receptor antagonists has permitted to further define the role of the DA D3 receptor in drug addiction.  Provided that the available preclin. and preliminary clin. evidence can be translated into clin. proof of concept in human, selective DA D3 receptor antagonists show promise for the treatment of substance use disorders as reflected by their potential to (1) regulate the motivation to self-administered drugs under schedules of reinforcement that require an increase in work demand and (2) disrupt the responsiveness to drug-assocd. stimuli that play a key role in the reinstatement of drug-seeking behavior triggered by re-exposure to the drug itself, re-exposure to environmental cues that had been previously assocd. with drug-taking behavior, or stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJryekGxZ_AbVg90H21EOLACvtfcHk0lggGrfQwJFyRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ejur0%253D&md5=4b196deebddce4d802a3dcef465d1dac</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1007%2Fs00210-012-0803-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00210-012-0803-6%26sid%3Dliteratum%253Aachs%26aulast%3DHeidbreder%26aufirst%3DC.%26atitle%3DRationale%2520in%2520Support%2520of%2520the%2520Use%2520of%2520Selective%2520Dopamine%2520D3%2520Receptor%2520Antagonists%2520for%2520the%2520Pharmacotherapeutic%2520Management%2520of%2520Drug%2520Addiction%26jtitle%3DNaunyn-Schmiedeberg%2527s%2520Arch.%2520Pharmacol.%26date%3D2013%26volume%3D386%26spage%3D167%26epage%3D176%26doi%3D10.1007%2Fs00210-012-0803-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Keck, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">John, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czoty, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nader, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">Identifying Medication Targets for Psychostimulant Addiction: Unraveling the Dopamine D3 Receptor Hypothesis</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5361</span><span class="NLM_x">–</span> <span class="NLM_lpage">5380</span><span class="refDoi"> DOI: 10.1021/jm501512b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501512b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsFGgu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5361-5380&author=T.+M.+Keckauthor=W.+S.+Johnauthor=P.+W.+Czotyauthor=M.+A.+Naderauthor=A.+H.+Newman&title=Identifying+Medication+Targets+for+Psychostimulant+Addiction%3A+Unraveling+the+Dopamine+D3+Receptor+Hypothesis&doi=10.1021%2Fjm501512b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4aR"><div class="casContent"><span class="casTitleNuber">4a</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Medication Targets for Psychostimulant Addiction: Unraveling the Dopamine D3 Receptor Hypothesis</span></div><div class="casAuthors">Keck, Thomas M.; John, William S.; Czoty, Paul W.; Nader, Michael A.; Newman, Amy Hauck</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5361-5380</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dopamine D3 receptor (D3R) is a target for developing medications to treat substance use disorders.  D3R-selective compds. with high affinity and varying efficacies have been discovered, providing crit. research tools for cell-based studies that have been translated to in vivo models of drug abuse.  D3R antagonists and partial agonists have shown esp. promising results in rodent models of relapse-like behavior, including stress-, drug-, and cue-induced reinstatement of drug seeking.  However, to date, translation to human studies has been limited.  Herein, we present an overview and illustrate some of the pitfalls and challenges of developing novel D3R-selective compds. toward clin. utility, esp. for treatment of cocaine abuse.  Future research and development of D3R-selective antagonists and partial agonists for substance abuse remains critically important but will also require further evaluation and development of translational animal models to det. the best time in the addiction cycle to target D3Rs for optimal therapeutic efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKVPGvcTvN87Vg90H21EOLACvtfcHk0ljBJ5A1XisUvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsFGgu74%253D&md5=99f531d840ad7f94d0eeadeb6fd48100</span></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1021%2Fjm501512b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501512b%26sid%3Dliteratum%253Aachs%26aulast%3DKeck%26aufirst%3DT.%2BM.%26aulast%3DJohn%26aufirst%3DW.%2BS.%26aulast%3DCzoty%26aufirst%3DP.%2BW.%26aulast%3DNader%26aufirst%3DM.%2BA.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DIdentifying%2520Medication%2520Targets%2520for%2520Psychostimulant%2520Addiction%253A%2520Unraveling%2520the%2520Dopamine%2520D3%2520Receptor%2520Hypothesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5361%26epage%3D5380%26doi%3D10.1021%2Fjm501512b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit4b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Micheli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidbreder, C.</span><span> </span><span class="NLM_article-title">Dopamine D3 Receptor Antagonists: a Patent Review (2007–2012)</span> <span class="citation_source-journal">Expert Opin. Ther. Pat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">363</span><span class="NLM_x">–</span> <span class="NLM_lpage">381</span><span class="refDoi"> DOI: 10.1517/13543776.2013.757593</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1517%2F13543776.2013.757593" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=23282131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1Sqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=363-381&author=F.+Micheliauthor=C.+Heidbreder&title=Dopamine+D3+Receptor+Antagonists%3A+a+Patent+Review+%282007%E2%80%932012%29&doi=10.1517%2F13543776.2013.757593"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4bR"><div class="casContent"><span class="casTitleNuber">4b</span><div class="casTitle"><span class="NLM_cas:atitle">Dopamine D3 receptor antagonists: a patent review (2007 - 2012)</span></div><div class="casAuthors">Micheli, Fabrizio; Heidbreder, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">363-381</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The synthesis and characterization of new highly potent and selective dopamine (DA) D3 receptor antagonists has permitted to characterize the role of the DA D3 receptor in the control of drug-seeking behavior and in the pathophysiol. of impulse control disorders and schizophrenia.Areas covered: In the present review, the authors will first describe most recent classes of DA D3 receptor antagonists by reviewing about 43 patent applications during the 2007 - 2012 period; they will then outline the biol. rationale in support of the use of selective DA D3 receptor antagonists in the treatment of drug addiction, impulse control disorders and schizophrenia.Expert opinion: The strongest clin. application and potential for selective DA D3 receptor antagonists lies in the redn. of drug-induced incentive motivation, the attenuation of drug's rewarding efficacy and the redn. in reinstatement of drug-seeking behavior triggered either by re-exposure to the drug itself, re-exposure to environmental cues that had been previously assocd. with drug-taking behavior or stress.  The selectivity of these antagonists together with reduced lipophilicity (minimizing unspecific binding), increased brain penetration and improved physico-chem. profile are all key factors for clin. efficacy and safety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoz7CcUwWNKXrVg90H21EOLACvtfcHk0ljBJ5A1XisUvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1Sqsrk%253D&md5=cbf7c42a3a88693f44b1adea64f6caf3</span></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1517%2F13543776.2013.757593&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2013.757593%26sid%3Dliteratum%253Aachs%26aulast%3DMicheli%26aufirst%3DF.%26aulast%3DHeidbreder%26aufirst%3DC.%26atitle%3DDopamine%2520D3%2520Receptor%2520Antagonists%253A%2520a%2520Patent%2520Review%2520%25282007%25E2%2580%25932012%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2013%26volume%3D23%26spage%3D363%26epage%3D381%26doi%3D10.1517%2F13543776.2013.757593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Micheli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arista, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonanomi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaney, F. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braggio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelli, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Checchia, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damiani, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di-Fabio, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fontana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gentile, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffante, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamprecht, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchioro, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mugnaini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratti, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tedesco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarsi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terreni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Worby, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashby, C. R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidbreder, C.</span><span> </span><span class="NLM_article-title">1,2,4-Triazolyl Azabicyclo[3.1.0]-Hexanes: a New Series of Potent and Selective Dopamine D(3) Receptor Antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">374</span><span class="NLM_x">–</span> <span class="NLM_lpage">391</span><span class="refDoi"> DOI: 10.1021/jm901319p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901319p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=374-391&author=F.+Micheliauthor=L.+Aristaauthor=G.+Bonanomiauthor=F.+E.+Blaneyauthor=S.+Braggioauthor=A.+M.+Capelliauthor=A.+Checchiaauthor=F.+Damianiauthor=R.+Di-Fabioauthor=S.+Fontanaauthor=G.+Gentileauthor=C.+Griffanteauthor=D.+Hamprechtauthor=C.+Marchioroauthor=M.+Mugnainiauthor=J.+Pinerauthor=E.+Rattiauthor=G.+Tedescoauthor=L.+Tarsiauthor=S.+Terreniauthor=A.+Worbyauthor=C.+R.+Ashbyauthor=C.+Heidbreder&title=1%2C2%2C4-Triazolyl+Azabicyclo%5B3.1.0%5D-Hexanes%3A+a+New+Series+of+Potent+and+Selective+Dopamine+D%283%29+Receptor+Antagonists&doi=10.1021%2Fjm901319p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm901319p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901319p%26sid%3Dliteratum%253Aachs%26aulast%3DMicheli%26aufirst%3DF.%26aulast%3DArista%26aufirst%3DL.%26aulast%3DBonanomi%26aufirst%3DG.%26aulast%3DBlaney%26aufirst%3DF.%2BE.%26aulast%3DBraggio%26aufirst%3DS.%26aulast%3DCapelli%26aufirst%3DA.%2BM.%26aulast%3DChecchia%26aufirst%3DA.%26aulast%3DDamiani%26aufirst%3DF.%26aulast%3DDi-Fabio%26aufirst%3DR.%26aulast%3DFontana%26aufirst%3DS.%26aulast%3DGentile%26aufirst%3DG.%26aulast%3DGriffante%26aufirst%3DC.%26aulast%3DHamprecht%26aufirst%3DD.%26aulast%3DMarchioro%26aufirst%3DC.%26aulast%3DMugnaini%26aufirst%3DM.%26aulast%3DPiner%26aufirst%3DJ.%26aulast%3DRatti%26aufirst%3DE.%26aulast%3DTedesco%26aufirst%3DG.%26aulast%3DTarsi%26aufirst%3DL.%26aulast%3DTerreni%26aufirst%3DS.%26aulast%3DWorby%26aufirst%3DA.%26aulast%3DAshby%26aufirst%3DC.%2BR.%26aulast%3DHeidbreder%26aufirst%3DC.%26atitle%3D1%252C2%252C4-Triazolyl%2520Azabicyclo%255B3.1.0%255D-Hexanes%253A%2520a%2520New%2520Series%2520of%2520Potent%2520and%2520Selective%2520Dopamine%2520D%25283%2529%2520Receptor%2520Antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D374%26epage%3D391%26doi%3D10.1021%2Fjm901319p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Micheli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonanomi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaney, F. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braggio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelli, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Checchia, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curcuruto, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damiani, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Fabio, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donati, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gentile, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gribble, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamprecht, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tedesco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terreni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarsi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lightfoot, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stemp, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacDonald, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pecoraro, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrone, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perini, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Worby, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilla, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valerio, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffante, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mugnaini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andreoli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lacroix, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwarz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozzi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bifone, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashby, C. R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagan, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidbreder, C.</span><span> </span><span class="NLM_article-title">1,2,4-Triazol-3-yl-thiopropyl-tetrahydrobenzazepines: A Series of Potent and Selective Dopamine D3 Receptor Antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">5076</span><span class="NLM_x">–</span> <span class="NLM_lpage">5089</span><span class="refDoi"> DOI: 10.1021/jm0705612</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0705612" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=5076-5089&author=F.+Micheliauthor=G.+Bonanomiauthor=F.+E.+Blaneyauthor=S.+Braggioauthor=A.+M.+Capelliauthor=A.+Checchiaauthor=O.+Curcurutoauthor=F.+Damianiauthor=R.+Di+Fabioauthor=D.+Donatiauthor=G.+Gentileauthor=A.+Gribbleauthor=D.+Hamprechtauthor=G.+Tedescoauthor=S.+Terreniauthor=L.+Tarsiauthor=A.+Lightfootauthor=G.+Stempauthor=G.+MacDonaldauthor=A.+Smithauthor=M.+Pecoraroauthor=M.+Petroneauthor=O.+Periniauthor=J.+Pinerauthor=T.+Rossiauthor=A.+Worbyauthor=M.+Pillaauthor=E.+Valerioauthor=C.+Griffanteauthor=M.+Mugnainiauthor=M.+Woodauthor=C.+Scottauthor=M.+Andreoliauthor=L.+Lacroixauthor=A.+Schwarzauthor=A.+Gozziauthor=A.+Bifoneauthor=C.+R.+Ashbyauthor=J.+J.+Haganauthor=C.+Heidbreder&title=1%2C2%2C4-Triazol-3-yl-thiopropyl-tetrahydrobenzazepines%3A+A+Series+of+Potent+and+Selective+Dopamine+D3+Receptor+Antagonists&doi=10.1021%2Fjm0705612"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm0705612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0705612%26sid%3Dliteratum%253Aachs%26aulast%3DMicheli%26aufirst%3DF.%26aulast%3DBonanomi%26aufirst%3DG.%26aulast%3DBlaney%26aufirst%3DF.%2BE.%26aulast%3DBraggio%26aufirst%3DS.%26aulast%3DCapelli%26aufirst%3DA.%2BM.%26aulast%3DChecchia%26aufirst%3DA.%26aulast%3DCurcuruto%26aufirst%3DO.%26aulast%3DDamiani%26aufirst%3DF.%26aulast%3DDi%2BFabio%26aufirst%3DR.%26aulast%3DDonati%26aufirst%3DD.%26aulast%3DGentile%26aufirst%3DG.%26aulast%3DGribble%26aufirst%3DA.%26aulast%3DHamprecht%26aufirst%3DD.%26aulast%3DTedesco%26aufirst%3DG.%26aulast%3DTerreni%26aufirst%3DS.%26aulast%3DTarsi%26aufirst%3DL.%26aulast%3DLightfoot%26aufirst%3DA.%26aulast%3DStemp%26aufirst%3DG.%26aulast%3DMacDonald%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DA.%26aulast%3DPecoraro%26aufirst%3DM.%26aulast%3DPetrone%26aufirst%3DM.%26aulast%3DPerini%26aufirst%3DO.%26aulast%3DPiner%26aufirst%3DJ.%26aulast%3DRossi%26aufirst%3DT.%26aulast%3DWorby%26aufirst%3DA.%26aulast%3DPilla%26aufirst%3DM.%26aulast%3DValerio%26aufirst%3DE.%26aulast%3DGriffante%26aufirst%3DC.%26aulast%3DMugnaini%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DM.%26aulast%3DScott%26aufirst%3DC.%26aulast%3DAndreoli%26aufirst%3DM.%26aulast%3DLacroix%26aufirst%3DL.%26aulast%3DSchwarz%26aufirst%3DA.%26aulast%3DGozzi%26aufirst%3DA.%26aulast%3DBifone%26aufirst%3DA.%26aulast%3DAshby%26aufirst%3DC.%2BR.%26aulast%3DHagan%26aufirst%3DJ.%2BJ.%26aulast%3DHeidbreder%26aufirst%3DC.%26atitle%3D1%252C2%252C4-Triazol-3-yl-thiopropyl-tetrahydrobenzazepines%253A%2520A%2520Series%2520of%2520Potent%2520and%2520Selective%2520Dopamine%2520D3%2520Receptor%2520Antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D5076%26epage%3D5089%26doi%3D10.1021%2Fjm0705612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Micheli, F.</span><span> </span><span class="NLM_article-title">Recent Advances in the Development of Dopamine D3 Receptor Antagonists: a Medicinal Chemistry Perspective</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1152</span><span class="NLM_x">–</span> <span class="NLM_lpage">1162</span><span class="refDoi"> DOI: 10.1002/cmdc.201000538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1002%2Fcmdc.201000538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=21425240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotVCntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=1152-1162&author=F.+Micheli&title=Recent+Advances+in+the+Development+of+Dopamine+D3+Receptor+Antagonists%3A+a+Medicinal+Chemistry+Perspective&doi=10.1002%2Fcmdc.201000538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Advances in the Development of Dopamine D3 Receptor Antagonists: a Medicinal Chemistry Perspective</span></div><div class="casAuthors">Micheli, Fabrizio</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1152-1162</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Dopamine (DA) D3 receptor antagonism might play a significant role in different therapeutic areas.  A high no. of preclin. studies on DA D3 receptor antagonists have shown efficacy in animal models of Parkinson's disease, schizophrenia and drug dependence.  This Review covers the activities of medicinal chemists in this field over the last ten years towards the identification of truly selective compds.  Both primary and patent literature is reviewed here.  Since the original discoveries, a clear trend towards the optimization of the developability properties of the new scaffold has clearly emerged with time, from both academic and industrial researchers.  Examples of advanced leads from academia and industry are described.  The latest potential therapeutic applications are reported too.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofhYCeq4b9EbVg90H21EOLACvtfcHk0lizKfBqwSIguA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotVCntbk%253D&md5=61c2b60b5e69df3517421e922b7aa73a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201000538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201000538%26sid%3Dliteratum%253Aachs%26aulast%3DMicheli%26aufirst%3DF.%26atitle%3DRecent%2520Advances%2520in%2520the%2520Development%2520of%2520Dopamine%2520D3%2520Receptor%2520Antagonists%253A%2520a%2520Medicinal%2520Chemistry%2520Perspective%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D1152%26epage%3D1162%26doi%3D10.1002%2Fcmdc.201000538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Micheli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cremonesi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semeraro, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarsi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomelleri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavanni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliosi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perdonà, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sava, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zonzini, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feriani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braggio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidbreder, C.</span><span> </span><span class="NLM_article-title">Novel Morpholine Scaffolds as Selective Dopamine (DA) D3 Receptor Antagonists</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1329</span><span class="NLM_x">–</span> <span class="NLM_lpage">1332</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2015.12.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1016%2Fj.bmcl.2015.12.081" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=1329-1332&author=F.+Micheliauthor=S.+Cremonesiauthor=T.+Semeraroauthor=L.+Tarsiauthor=S.+Tomelleriauthor=P.+Cavanniauthor=B.+Oliosiauthor=E.+Perdon%C3%A0author=A.+Savaauthor=L.+Zonziniauthor=A.+Ferianiauthor=S.+Braggioauthor=C.+Heidbreder&title=Novel+Morpholine+Scaffolds+as+Selective+Dopamine+%28DA%29+D3+Receptor+Antagonists&doi=10.1016%2Fj.bmcl.2015.12.081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.12.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.12.081%26sid%3Dliteratum%253Aachs%26aulast%3DMicheli%26aufirst%3DF.%26aulast%3DCremonesi%26aufirst%3DS.%26aulast%3DSemeraro%26aufirst%3DT.%26aulast%3DTarsi%26aufirst%3DL.%26aulast%3DTomelleri%26aufirst%3DS.%26aulast%3DCavanni%26aufirst%3DP.%26aulast%3DOliosi%26aufirst%3DB.%26aulast%3DPerdon%25C3%25A0%26aufirst%3DE.%26aulast%3DSava%26aufirst%3DA.%26aulast%3DZonzini%26aufirst%3DL.%26aulast%3DFeriani%26aufirst%3DA.%26aulast%3DBraggio%26aufirst%3DS.%26aulast%3DHeidbreder%26aufirst%3DC.%26atitle%3DNovel%2520Morpholine%2520Scaffolds%2520as%2520Selective%2520Dopamine%2520%2528DA%2529%2520D3%2520Receptor%2520Antagonists%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D1329%26epage%3D1332%26doi%3D10.1016%2Fj.bmcl.2015.12.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Micheli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernardelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bianchi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braggio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castelletti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavallini, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavanni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cremonesi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dal Cin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feriani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliosi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semeraro, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarsi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomelleri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Visentini, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zonzini, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidbreder, C.</span><span> </span><span class="NLM_article-title">1,2,4-Triazolyl octahydropyrrolo[2,3-b]pyrroles: A New Series of Potent and Selective Dopamine D3 Receptor Antagonists</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">1619</span><span class="NLM_x">–</span> <span class="NLM_lpage">1636</span><span class="refDoi"> DOI: 10.1016/j.bmc.2016.02.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1016%2Fj.bmc.2016.02.031" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=1619-1636&author=F.+Micheliauthor=A.+Bernardelliauthor=F.+Bianchiauthor=S.+Braggioauthor=L.+Castellettiauthor=P.+Cavalliniauthor=P.+Cavanniauthor=S.+Cremonesiauthor=M.+Dal+Cinauthor=A.+Ferianiauthor=B.+Oliosiauthor=T.+Semeraroauthor=L.+Tarsiauthor=S.+Tomelleriauthor=A.+Wongauthor=F.+Visentiniauthor=L.+Zonziniauthor=C.+Heidbreder&title=1%2C2%2C4-Triazolyl+octahydropyrrolo%5B2%2C3-b%5Dpyrroles%3A+A+New+Series+of+Potent+and+Selective+Dopamine+D3+Receptor+Antagonists&doi=10.1016%2Fj.bmc.2016.02.031"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.02.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.02.031%26sid%3Dliteratum%253Aachs%26aulast%3DMicheli%26aufirst%3DF.%26aulast%3DBernardelli%26aufirst%3DA.%26aulast%3DBianchi%26aufirst%3DF.%26aulast%3DBraggio%26aufirst%3DS.%26aulast%3DCastelletti%26aufirst%3DL.%26aulast%3DCavallini%26aufirst%3DP.%26aulast%3DCavanni%26aufirst%3DP.%26aulast%3DCremonesi%26aufirst%3DS.%26aulast%3DDal%2BCin%26aufirst%3DM.%26aulast%3DFeriani%26aufirst%3DA.%26aulast%3DOliosi%26aufirst%3DB.%26aulast%3DSemeraro%26aufirst%3DT.%26aulast%3DTarsi%26aufirst%3DL.%26aulast%3DTomelleri%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DA.%26aulast%3DVisentini%26aufirst%3DF.%26aulast%3DZonzini%26aufirst%3DL.%26aulast%3DHeidbreder%26aufirst%3DC.%26atitle%3D1%252C2%252C4-Triazolyl%2520octahydropyrrolo%255B2%252C3-b%255Dpyrroles%253A%2520A%2520New%2520Series%2520of%2520Potent%2520and%2520Selective%2520Dopamine%2520D3%2520Receptor%2520Antagonists%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D1619%26epage%3D1636%26doi%3D10.1016%2Fj.bmc.2016.02.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Stemp, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashmeade, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branch, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hadley, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nash, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thewlis, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vong, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, N. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avenell, K. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyfield, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagan, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Middlemiss, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reavill, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riley, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Routledge, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, M.</span><span> </span><span class="NLM_article-title">Design and Synthesis of Trans-N-[4-[2-(6-cyano-1,2,3, 4-tetrahydroisoquinolin-2- yl)ethyl]cyclohexyl]-4-quinolinecarboxamide (SB-277011): a Potent and Selective Dopamine D(3) Receptor Antagonist with High Oral Bioavailability and CNS Penetration in the Rat</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1878</span><span class="NLM_x">–</span> <span class="NLM_lpage">1885</span><span class="refDoi"> DOI: 10.1021/jm000090i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm000090i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=1878-1885&author=G.+Stempauthor=T.+Ashmeadeauthor=C.+L.+Branchauthor=M.+S.+Hadleyauthor=A.+J.+Hunterauthor=C.+N.+Johnsonauthor=D.+J.+Nashauthor=K.+M.+Thewlisauthor=A.+K.+Vongauthor=N.+E.+Austinauthor=P.+Jeffreyauthor=K.+Y.+Avenellauthor=I.+Boyfieldauthor=J.+J.+Haganauthor=D.+N.+Middlemissauthor=C.+Reavillauthor=G.+J.+Rileyauthor=C.+Routledgeauthor=M.+Wood&title=Design+and+Synthesis+of+Trans-N-%5B4-%5B2-%286-cyano-1%2C2%2C3%2C+4-tetrahydroisoquinolin-2-+yl%29ethyl%5Dcyclohexyl%5D-4-quinolinecarboxamide+%28SB-277011%29%3A+a+Potent+and+Selective+Dopamine+D%283%29+Receptor+Antagonist+with+High+Oral+Bioavailability+and+CNS+Penetration+in+the+Rat&doi=10.1021%2Fjm000090i"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm000090i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000090i%26sid%3Dliteratum%253Aachs%26aulast%3DStemp%26aufirst%3DG.%26aulast%3DAshmeade%26aufirst%3DT.%26aulast%3DBranch%26aufirst%3DC.%2BL.%26aulast%3DHadley%26aufirst%3DM.%2BS.%26aulast%3DHunter%26aufirst%3DA.%2BJ.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DNash%26aufirst%3DD.%2BJ.%26aulast%3DThewlis%26aufirst%3DK.%2BM.%26aulast%3DVong%26aufirst%3DA.%2BK.%26aulast%3DAustin%26aufirst%3DN.%2BE.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DAvenell%26aufirst%3DK.%2BY.%26aulast%3DBoyfield%26aufirst%3DI.%26aulast%3DHagan%26aufirst%3DJ.%2BJ.%26aulast%3DMiddlemiss%26aufirst%3DD.%2BN.%26aulast%3DReavill%26aufirst%3DC.%26aulast%3DRiley%26aufirst%3DG.%2BJ.%26aulast%3DRoutledge%26aufirst%3DC.%26aulast%3DWood%26aufirst%3DM.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Trans-N-%255B4-%255B2-%25286-cyano-1%252C2%252C3%252C%25204-tetrahydroisoquinolin-2-%2520yl%2529ethyl%255Dcyclohexyl%255D-4-quinolinecarboxamide%2520%2528SB-277011%2529%253A%2520a%2520Potent%2520and%2520Selective%2520Dopamine%2520D%25283%2529%2520Receptor%2520Antagonist%2520with%2520High%2520Oral%2520Bioavailability%2520and%2520CNS%2520Penetration%2520in%2520the%2520Rat%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D1878%26epage%3D1885%26doi%3D10.1021%2Fjm000090i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Mugnaini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iavarone, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavallini, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffante, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliosi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savoia, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaver, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabiner, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Micheli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidbreder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andorn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merlo Pich, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bani, M.</span><span> </span><span class="NLM_article-title">Occupancy of Brain Dopamine D3 Receptors and Drug Craving: a Translational Approach</span> <span class="citation_source-journal">Neuropsychopharmacology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">302</span><span class="NLM_x">–</span> <span class="NLM_lpage">312</span><span class="refDoi"> DOI: 10.1038/npp.2012.171</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1038%2Fnpp.2012.171" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2013&pages=302-312&author=M.+Mugnainiauthor=L.+Iavaroneauthor=P.+Cavalliniauthor=C.+Griffanteauthor=B.+Oliosiauthor=C.+Savoiaauthor=J.+Beaverauthor=E.+A.+Rabinerauthor=F.+Micheliauthor=C.+Heidbrederauthor=A.+Andornauthor=E.+Merlo+Pichauthor=M.+Bani&title=Occupancy+of+Brain+Dopamine+D3+Receptors+and+Drug+Craving%3A+a+Translational+Approach&doi=10.1038%2Fnpp.2012.171"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnpp.2012.171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnpp.2012.171%26sid%3Dliteratum%253Aachs%26aulast%3DMugnaini%26aufirst%3DM.%26aulast%3DIavarone%26aufirst%3DL.%26aulast%3DCavallini%26aufirst%3DP.%26aulast%3DGriffante%26aufirst%3DC.%26aulast%3DOliosi%26aufirst%3DB.%26aulast%3DSavoia%26aufirst%3DC.%26aulast%3DBeaver%26aufirst%3DJ.%26aulast%3DRabiner%26aufirst%3DE.%2BA.%26aulast%3DMicheli%26aufirst%3DF.%26aulast%3DHeidbreder%26aufirst%3DC.%26aulast%3DAndorn%26aufirst%3DA.%26aulast%3DMerlo%2BPich%26aufirst%3DE.%26aulast%3DBani%26aufirst%3DM.%26atitle%3DOccupancy%2520of%2520Brain%2520Dopamine%2520D3%2520Receptors%2520and%2520Drug%2520Craving%253A%2520a%2520Translational%2520Approach%26jtitle%3DNeuropsychopharmacology%26date%3D2013%26volume%3D38%26spage%3D302%26epage%3D312%26doi%3D10.1038%2Fnpp.2012.171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Grundt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McElveen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luedtke, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">Novel Heterocyclic Trans Olefin Analogues of N-{4-[4-(2,3-dichlorophenyl)piperazin-1-yl]butyl}- arylcarboxamides as Selective Probes with High Affinity for the Dopamine D3 Receptor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">839</span><span class="NLM_x">–</span> <span class="NLM_lpage">848</span><span class="refDoi"> DOI: 10.1021/jm049465g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049465g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=839-848&author=P.+Grundtauthor=E.+E.+Carlsonauthor=J.+Caoauthor=C.+J.+Bennettauthor=E.+McElveenauthor=M.+Taylorauthor=R.+R.+Luedtkeauthor=A.+H.+Newman&title=Novel+Heterocyclic+Trans+Olefin+Analogues+of+N-%7B4-%5B4-%282%2C3-dichlorophenyl%29piperazin-1-yl%5Dbutyl%7D-+arylcarboxamides+as+Selective+Probes+with+High+Affinity+for+the+Dopamine+D3+Receptor&doi=10.1021%2Fjm049465g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12a&amp;dbid=16384&amp;doi=10.1021%2Fjm049465g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049465g%26sid%3Dliteratum%253Aachs%26aulast%3DGrundt%26aufirst%3DP.%26aulast%3DCarlson%26aufirst%3DE.%2BE.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DC.%2BJ.%26aulast%3DMcElveen%26aufirst%3DE.%26aulast%3DTaylor%26aufirst%3DM.%26aulast%3DLuedtke%26aufirst%3DR.%2BR.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DNovel%2520Heterocyclic%2520Trans%2520Olefin%2520Analogues%2520of%2520N-%257B4-%255B4-%25282%252C3-dichlorophenyl%2529piperazin-1-yl%255Dbutyl%257D-%2520arylcarboxamides%2520as%2520Selective%2520Probes%2520with%2520High%2520Affinity%2520for%2520the%2520Dopamine%2520D3%2520Receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D839%26epage%3D848%26doi%3D10.1021%2Fjm049465g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Newman, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grundt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cyriac, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deschamps, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luedtke, R. R.</span><span> </span><span class="NLM_article-title"><i>N</i>-(4-(4-(2,3-Dichloro- or 2-methoxyphenyl)piperazin-1-yl)butyl)heterobiarylcarboxamides with Functionalized Linking Chains as High Affinity and Enantioselective D3 Receptor Antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2559</span><span class="NLM_x">–</span> <span class="NLM_lpage">2570</span><span class="refDoi"> DOI: 10.1021/jm900095y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900095y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktFGrt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2559-2570&author=A.+H.+Newmanauthor=P.+Grundtauthor=G.+Cyriacauthor=J.+R.+Deschampsauthor=M.+Taylorauthor=R.+Kumarauthor=D.+Hoauthor=R.+R.+Luedtke&title=N-%284-%284-%282%2C3-Dichloro-+or+2-methoxyphenyl%29piperazin-1-yl%29butyl%29heterobiarylcarboxamides+with+Functionalized+Linking+Chains+as+High+Affinity+and+Enantioselective+D3+Receptor+Antagonists&doi=10.1021%2Fjm900095y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12bR"><div class="casContent"><span class="casTitleNuber">12b</span><div class="casTitle"><span class="NLM_cas:atitle">N-(4-(4-(2,3-Dichloro- or 2-methoxyphenyl)piperazin-1-yl)butyl)heterobiarylcarboxamides with Functionalized Linking Chains as High Affinity and Enantioselective D3 Receptor Antagonists</span></div><div class="casAuthors">Newman, Amy Hauck; Grundt, Peter; Cyriac, George; Deschamps, Jeffrey R.; Taylor, Michelle; Kumar, Rakesh; Ho, David; Luedtke, Robert R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2559-2570</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the present report, the D3 receptor pharmacophore is modified in the 2,3-diCl- and 2-OCH3-phenylpiperazine class of compds. with the goal to improve D3 receptor affinity and selectivity.  This extension of structure-activity relationships (SAR) has resulted in the identification of the first enantioselective D3 antagonists (R- and S-22) to be reported, wherein enantioselectivity is more pronounced at D3 than at D2, and that a binding region on the second extracellular loop (E2) may play a role in both enantioselectivity and D3 receptor selectivity.  Moreover, we have discovered some of the most D3-selective compds. reported to date that show high affinity (Ki = 1 nM) for D3 and ∼400-fold selectivity over the D2 receptor subtype.  Several of these analogs showed exquisite selectivity for D3 receptors over >60 other receptors, further underscoring their value as in vivo research tools.  These lead compds. also have appropriate phys. characteristics for in vivo exploration and therefore will be useful in detg. how intrinsic activity at D3 receptors tested in vitro is related to behaviors in animal models of addiction and other neuropsychiatric disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprbxvbF2dNX7Vg90H21EOLACvtfcHk0ljEt-qE537SLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktFGrt7s%253D&md5=665731381e379cec45ef058cd6c45d0e</span></div><a href="/servlet/linkout?suffix=cit12b&amp;dbid=16384&amp;doi=10.1021%2Fjm900095y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900095y%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DGrundt%26aufirst%3DP.%26aulast%3DCyriac%26aufirst%3DG.%26aulast%3DDeschamps%26aufirst%3DJ.%2BR.%26aulast%3DTaylor%26aufirst%3DM.%26aulast%3DKumar%26aufirst%3DR.%26aulast%3DHo%26aufirst%3DD.%26aulast%3DLuedtke%26aufirst%3DR.%2BR.%26atitle%3DN-%25284-%25284-%25282%252C3-Dichloro-%2520or%25202-methoxyphenyl%2529piperazin-1-yl%2529butyl%2529heterobiarylcarboxamides%2520with%2520Functionalized%2520Linking%2520Chains%2520as%2520High%2520Affinity%2520and%2520Enantioselective%2520D3%2520Receptor%2520Antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D2559%26epage%3D2570%26doi%3D10.1021%2Fjm900095y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levant, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keck, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Tranylcypromine Substituted cis-Hydroxycyclobutylnaphthamides as Potent and Selective Dopamine D3 Receptor Antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">4962</span><span class="NLM_x">–</span> <span class="NLM_lpage">4968</span><span class="refDoi"> DOI: 10.1021/jm401798r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401798r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC2cXosVOnurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4962-4968&author=J.+Chenauthor=B.+Levantauthor=C.+Jiangauthor=T.+M.+Keckauthor=A.+H.+Newmanauthor=S.+Wang&title=Tranylcypromine+Substituted+cis-Hydroxycyclobutylnaphthamides+as+Potent+and+Selective+Dopamine+D3+Receptor+Antagonists&doi=10.1021%2Fjm401798r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Tranylcypromine Substituted cis-Hydroxycyclobutylnaphthamides as Potent and Selective Dopamine D3 Receptor Antagonists</span></div><div class="casAuthors">Chen, Jianyong; Levant, Beth; Jiang, Cheng; Keck, Thomas M.; Newman, Amy Hauck; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4962-4968</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report a class of potent and selective dopamine D3 receptor antagonists based upon tranylcypromine.  Although tranylcypromine has a low affinity for the rat D3 receptor (Ki = 12.8 μM), our efforts have yielded (1R,2S)-11 (CJ-1882), which has Ki values of 2.7 and 2.8 nM at the rat and human dopamine D3 receptors, resp., and displays resp. selectivities of >10000-fold and 223-fold over the rat and human D2 receptors.  Evaluation in a β-arrestin functional assay showed that (1R,2S)-11 is a potent and competitive antagonist at the human D3 receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowWudWdAfWgLVg90H21EOLACvtfcHk0lhl5Voo9ZRN-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXosVOnurg%253D&md5=e2ddf0711d00103be281cb043b19ec3a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm401798r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401798r%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLevant%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DC.%26aulast%3DKeck%26aufirst%3DT.%2BM.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DTranylcypromine%2520Substituted%2520cis-Hydroxycyclobutylnaphthamides%2520as%2520Potent%2520and%2520Selective%2520Dopamine%2520D3%2520Receptor%2520Antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D4962%26epage%3D4968%26doi%3D10.1021%2Fjm401798r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Hu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span> </span><span class="NLM_article-title">The Dopamine D3 Receptor Antagonist YQA14 that Inhibits the Expression and Drug-primed Reactivation of Morphine-induced Conditioned Place Preference in Rats</span> <span class="citation_source-journal">Eur. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">720</span><span class="NLM_x">, </span> <span class="NLM_fpage">212</span><span class="NLM_x">–</span> <span class="NLM_lpage">217</span><span class="refDoi"> DOI: 10.1016/j.ejphar.2013.10.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1016%2Fj.ejphar.2013.10.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=24436985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslKnurnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=720&publication_year=2013&pages=212-217&author=R.+Huauthor=R.+Songauthor=R.+Yangauthor=R.+Suauthor=J.+Li&title=The+Dopamine+D3+Receptor+Antagonist+YQA14+that+Inhibits+the+Expression+and+Drug-primed+Reactivation+of+Morphine-induced+Conditioned+Place+Preference+in+Rats&doi=10.1016%2Fj.ejphar.2013.10.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The dopamine D3 receptor antagonist YQA14 that inhibits the expression and drug-primed reactivation of morphine-induced conditioned place preference in rats</span></div><div class="casAuthors">Hu, Rongrong; Song, Rui; Yang, Rifang; Su, Ruibin; Li, Jin</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">720</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">212-217</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Increasing evidence suggests that the mesolimbic dopamine system plays a crit. role in opioid addiction.  However, there is currently no std. drug treatment for opioid addiction.  Growing preclin. evidence indicates that the dopamine D3 receptor antagonists are the potential anti-addiction pharmacotherapeutic agents based on in animal models of multiple drug addiction.  In this study, we investigated the inhibitory effects of YQA14, a novel dopamine D3 receptor antagonist with a high affinity and selectivity for dopamine D3 receptor, using morphine-induced conditioned place preference (CPP) in rats.  The results suggested that YQA14 (6.25-25 mg/kg; i.p., i.p.) decreased the expression of morphine (10 mg/kg, s.c.)-induced CPP in a dose-related manner but did not influence the acquisition of morphine-induced CPP.  At a 25 mg/kg dose of YQA14, it also notably inhibited the reactivation of morphine-priming CPP.  These findings suggest that YQA14 is a potential agent for anti-opioid addiction which warrants further study and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTXMIGCwk0X7Vg90H21EOLACvtfcHk0lhl5Voo9ZRN-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslKnurnP&md5=05d05b2def79779c975b853df1ee6365</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.ejphar.2013.10.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejphar.2013.10.026%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DR.%26aulast%3DSong%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DSu%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DThe%2520Dopamine%2520D3%2520Receptor%2520Antagonist%2520YQA14%2520that%2520Inhibits%2520the%2520Expression%2520and%2520Drug-primed%2520Reactivation%2520of%2520Morphine-induced%2520Conditioned%2520Place%2520Preference%2520in%2520Rats%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D2013%26volume%3D720%26spage%3D212%26epage%3D217%26doi%3D10.1016%2Fj.ejphar.2013.10.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Hackling, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, H.</span><span> </span><span class="NLM_article-title">Dopamine D3 Receptor Ligands with Antagonist Properties</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">946</span><span class="NLM_x">–</span> <span class="NLM_lpage">961</span><span class="refDoi"> DOI: 10.1002/1439-7633(20021004)3:10<946::AID-CBIC946>3.0.CO;2-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1002%2F1439-7633%2820021004%293%3A10%3C946%3A%3AAID-CBIC946%3E3.0.CO%3B2-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=12362359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BD38XnvFGgu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=946-961&author=A.+E.+Hacklingauthor=H.+Stark&title=Dopamine+D3+Receptor+Ligands+with+Antagonist+Properties&doi=10.1002%2F1439-7633%2820021004%293%3A10%3C946%3A%3AAID-CBIC946%3E3.0.CO%3B2-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Dopamine D3 receptor ligands with antagonist properties</span></div><div class="casAuthors">Hackling, Anneke E.; Stark, Holger</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">946-961</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The dopamine D3 receptor was recognized to play an important role in the mol. mechanisms of various neuropsychiatric disorders.  The development of new dopamine D3 receptor selective antagonists is premised on the potentially improved therapeutic treatment of psychosis like schizophrenia.  Partial agonists at dopamine D3 receptors are supposed to be beneficial when administered to drug abusers or in Parkinson's disease.  The structural basis for most compds. is at least a basic, aryl-substituted alkanamine part with an alkyl moiety, which in many compds. forms a spacer to another aryl residue.  Structural variety among the amine moiety includes aminotetralins, tetrahydroisoquinolines, isoindoles, benzazepines, and amino-indans, as well as pyrrolidines, pyrroles, and 4-phenylpiperazines.  Various ways for lead optimization are shown in different classes of compds.  Promising ligands with high D3 receptor affinity often lack sufficient selectivity or display deficits in the required in vivo parameters.  Structure-activity relationships for dopamine D3 receptor antagonists and partial agonists are discussed here, along with the outlook for their potential therapeutic application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7h56q6SPZN7Vg90H21EOLACvtfcHk0lh5XywXQEAFaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnvFGgu7c%253D&md5=ba9cd2302ced89619ae5d1d82e7d08e4</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2F1439-7633%2820021004%293%3A10%3C946%3A%3AAID-CBIC946%3E3.0.CO%3B2-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1439-7633%252820021004%25293%253A10%253C946%253A%253AAID-CBIC946%253E3.0.CO%253B2-5%26sid%3Dliteratum%253Aachs%26aulast%3DHackling%26aufirst%3DA.%2BE.%26aulast%3DStark%26aufirst%3DH.%26atitle%3DDopamine%2520D3%2520Receptor%2520Ligands%2520with%2520Antagonist%2520Properties%26jtitle%3DChemBioChem%26date%3D2002%26volume%3D3%26spage%3D946%26epage%3D961%26doi%3D10.1002%2F1439-7633%2820021004%293%3A10%3C946%3A%3AAID-CBIC946%3E3.0.CO%3B2-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Newman, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beuming, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banala, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donthamsetti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pongetti, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaBounty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luedtke, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Javitch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span> </span><span class="NLM_article-title">Molecular Determinants of Selectivity and Efficacy at the Dopamine D3 Receptor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">6689</span><span class="NLM_x">–</span> <span class="NLM_lpage">6699</span><span class="refDoi"> DOI: 10.1021/jm300482h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300482h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsFCkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6689-6699&author=A.+H.+Newmanauthor=T.+Beumingauthor=A.+K.+Banalaauthor=P.+Donthamsettiauthor=K.+Pongettiauthor=A.+LaBountyauthor=B.+Levyauthor=J.+Caoauthor=M.+Michinoauthor=R.+R.+Luedtkeauthor=A.+Javitchauthor=L.+Shi&title=Molecular+Determinants+of+Selectivity+and+Efficacy+at+the+Dopamine+D3+Receptor&doi=10.1021%2Fjm300482h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Determinants of Selectivity and Efficacy at the Dopamine D3 Receptor</span></div><div class="casAuthors">Newman, Amy Hauck; Beuming, Thijs; Banala, Ashwini K.; Donthamsetti, Prashant; Pongetti, Katherine; LaBounty, Alex; Levy, Benjamin; Cao, Jianjing; Michino, Mayako; Luedtke, Robert R.; Javitch, Jonathan A.; Shi, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6689-6699</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dopamine D3 receptor (D3R) has been implicated in substance abuse and other neuropsychiatric disorders.  The high sequence homol. between the D3R and D2R, esp. within the orthosteric binding site (OBS) that binds dopamine, has made the development of D3R-selective compds. challenging.  Here, we deconstruct into pharmacophoric elements a series of D3R-selective substituted-4-phenylpiperazine compds. and use computational simulations and binding and activation studies to dissect the structural bases for D3R selectivity and efficacy.  We find that selectivity arises from divergent interactions within a second binding pocket (SBP) sep. from the OBS, whereas efficacy depends on the binding mode in the OBS.  Our findings reveal structural features of the receptor that are crit. to selectivity and efficacy that can be used to design highly D3R-selective ligands with targeted efficacies.  These findings are generalizable to other GPCRs in which the SBP can be targeted by bitopic or allosteric ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmBJU7uyvVg7Vg90H21EOLACvtfcHk0lh5XywXQEAFaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsFCkt7c%253D&md5=033dd49282723650828a3ec0a31211e5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm300482h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300482h%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DBanala%26aufirst%3DA.%2BK.%26aulast%3DDonthamsetti%26aufirst%3DP.%26aulast%3DPongetti%26aufirst%3DK.%26aulast%3DLaBounty%26aufirst%3DA.%26aulast%3DLevy%26aufirst%3DB.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DMichino%26aufirst%3DM.%26aulast%3DLuedtke%26aufirst%3DR.%2BR.%26aulast%3DJavitch%26aufirst%3DA.%26aulast%3DShi%26aufirst%3DL.%26atitle%3DMolecular%2520Determinants%2520of%2520Selectivity%2520and%2520Efficacy%2520at%2520the%2520Dopamine%2520D3%2520Receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6689%26epage%3D6699%26doi%3D10.1021%2Fjm300482h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Hughes, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalindjian, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, K. L.</span><span> </span><span class="NLM_article-title">Principles of Early Drug Discovery</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">162</span><span class="NLM_x">, </span> <span class="NLM_fpage">1239</span><span class="NLM_x">–</span> <span class="NLM_lpage">1249</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.2010.01127.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1111%2Fj.1476-5381.2010.01127.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=21091654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivF2lt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2011&pages=1239-1249&author=J.+P.+Hughesauthor=S.+Reesauthor=S.+B.+Kalindjianauthor=K.+L.+Philpott&title=Principles+of+Early+Drug+Discovery&doi=10.1111%2Fj.1476-5381.2010.01127.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17aR"><div class="casContent"><span class="casTitleNuber">17a</span><div class="casTitle"><span class="NLM_cas:atitle">Principles of early drug discovery</span></div><div class="casAuthors">Hughes, J. P.; Rees, S.; Kalindjian, S. B.; Philpott, K. L.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1239-1249</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Developing a new drug from original idea to the launch of a finished product is a complex process which can take 12-15 years and cost in excess of $1 billion.  The idea for a target can come from a variety of sources including academic and clin. research and from the com. sector.  It may take many years to build up a body of supporting evidence before selecting a target for a costly drug discovery program.  Once a target was chosen, the pharmaceutical industry and more recently some academic centers have streamlined a no. of early processes to identify mols. which possess suitable characteristics to make acceptable drugs.  This review will look at key preclin. stages of the drug discovery process, from initial target identification and validation, through assay development, high throughput screening, hit identification, lead optimization and finally the selection of a candidate mol. for clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooCc74_F_90rVg90H21EOLACvtfcHk0lgptaozEYrvvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivF2lt7o%253D&md5=05a0016f45b155f9fb5822f5c7bcc35d</span></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2010.01127.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2010.01127.x%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DJ.%2BP.%26aulast%3DRees%26aufirst%3DS.%26aulast%3DKalindjian%26aufirst%3DS.%2BB.%26aulast%3DPhilpott%26aufirst%3DK.%2BL.%26atitle%3DPrinciples%2520of%2520Early%2520Drug%2520Discovery%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2011%26volume%3D162%26spage%3D1239%26epage%3D1249%26doi%3D10.1111%2Fj.1476-5381.2010.01127.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit17b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Saxena, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panicucci, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joshi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garad, S.</span><span> </span><span class="NLM_article-title">Developability Assessment in Pharmaceutical Industry: An Integrated Group Approach for Selecting Developable Candidates</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">1962</span><span class="NLM_x">–</span> <span class="NLM_lpage">1979</span><span class="refDoi"> DOI: 10.1002/jps.21592</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1002%2Fjps.21592" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2009&pages=1962-1979&author=V.+Saxenaauthor=R.+Panicucciauthor=Y.+Joshiauthor=S.+Garad&title=Developability+Assessment+in+Pharmaceutical+Industry%3A+An+Integrated+Group+Approach+for+Selecting+Developable+Candidates&doi=10.1002%2Fjps.21592"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17b&amp;dbid=16384&amp;doi=10.1002%2Fjps.21592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.21592%26sid%3Dliteratum%253Aachs%26aulast%3DSaxena%26aufirst%3DV.%26aulast%3DPanicucci%26aufirst%3DR.%26aulast%3DJoshi%26aufirst%3DY.%26aulast%3DGarad%26aufirst%3DS.%26atitle%3DDevelopability%2520Assessment%2520in%2520Pharmaceutical%2520Industry%253A%2520An%2520Integrated%2520Group%2520Approach%2520for%2520Selecting%2520Developable%2520Candidates%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2009%26volume%3D98%26spage%3D1962%26epage%3D1979%26doi%3D10.1002%2Fjps.21592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit17c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Han, C.; Wang, B.</span><span> </span><span class="NLM_article-title">Factors that impact the developability of drug candidates: an overview</span>. In  <span class="citation_source-book">Drug Delivery: Principles and Applications</span>; <span class="NLM_contrib-group">Wang, B.; Siahaan, T.; Soltero, R.</span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">; </span>Chapter 1.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1002%2F0471475734.ch1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=C.+Han&author=B.+Wangauthor=B.+Wang&author=T.+Siahaan&author=R.+Soltero&title=Drug+Delivery%3A+Principles+and+Applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17c&amp;dbid=16384&amp;doi=10.1002%2F0471475734.ch1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F0471475734.ch1%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DC.%26atitle%3DFactors%2520that%2520impact%2520the%2520developability%2520of%2520drug%2520candidates%253A%2520an%2520overview%26btitle%3DDrug%2520Delivery%253A%2520Principles%2520and%2520Applications%26aulast%3DWang%26aufirst%3DB.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit17d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Keck, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">John, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czoty, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nader, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">Identifying Medication Targets for Psychostimulant Addiction: Unraveling the Dopamine D3 Receptor Hypothesis</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5361</span><span class="NLM_x">–</span> <span class="NLM_lpage">5380</span><span class="refDoi"> DOI: 10.1021/jm501512b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501512b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsFGgu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5361-5380&author=T.+M.+Keckauthor=W.+S.+Johnauthor=P.+W.+Czotyauthor=M.+A.+Naderauthor=A.+H.+Newman&title=Identifying+Medication+Targets+for+Psychostimulant+Addiction%3A+Unraveling+the+Dopamine+D3+Receptor+Hypothesis&doi=10.1021%2Fjm501512b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17dR"><div class="casContent"><span class="casTitleNuber">17d</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Medication Targets for Psychostimulant Addiction: Unraveling the Dopamine D3 Receptor Hypothesis</span></div><div class="casAuthors">Keck, Thomas M.; John, William S.; Czoty, Paul W.; Nader, Michael A.; Newman, Amy Hauck</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5361-5380</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dopamine D3 receptor (D3R) is a target for developing medications to treat substance use disorders.  D3R-selective compds. with high affinity and varying efficacies have been discovered, providing crit. research tools for cell-based studies that have been translated to in vivo models of drug abuse.  D3R antagonists and partial agonists have shown esp. promising results in rodent models of relapse-like behavior, including stress-, drug-, and cue-induced reinstatement of drug seeking.  However, to date, translation to human studies has been limited.  Herein, we present an overview and illustrate some of the pitfalls and challenges of developing novel D3R-selective compds. toward clin. utility, esp. for treatment of cocaine abuse.  Future research and development of D3R-selective antagonists and partial agonists for substance abuse remains critically important but will also require further evaluation and development of translational animal models to det. the best time in the addiction cycle to target D3Rs for optimal therapeutic efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKVPGvcTvN87Vg90H21EOLACvtfcHk0ljYUkmSBEuJNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsFGgu74%253D&md5=99f531d840ad7f94d0eeadeb6fd48100</span></div><a href="/servlet/linkout?suffix=cit17d&amp;dbid=16384&amp;doi=10.1021%2Fjm501512b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501512b%26sid%3Dliteratum%253Aachs%26aulast%3DKeck%26aufirst%3DT.%2BM.%26aulast%3DJohn%26aufirst%3DW.%2BS.%26aulast%3DCzoty%26aufirst%3DP.%2BW.%26aulast%3DNader%26aufirst%3DM.%2BA.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DIdentifying%2520Medication%2520Targets%2520for%2520Psychostimulant%2520Addiction%253A%2520Unraveling%2520the%2520Dopamine%2520D3%2520Receptor%2520Hypothesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5361%26epage%3D5380%26doi%3D10.1021%2Fjm501512b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit17e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Watson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viggers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheetham, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porter, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, K. D.</span><span> </span><span class="NLM_article-title">Receptor Occupancy and Brain Free Fraction</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">753</span><span class="NLM_x">–</span> <span class="NLM_lpage">760</span><span class="refDoi"> DOI: 10.1124/dmd.108.022814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1124%2Fdmd.108.022814" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=753-760&author=J.+Watsonauthor=S.+Wrightauthor=A.+Lucasauthor=K.+L.+Clarkeauthor=J.+Viggersauthor=S.+Cheethamauthor=P.+Jeffreyauthor=R.+Porterauthor=K.+D.+Read&title=Receptor+Occupancy+and+Brain+Free+Fraction&doi=10.1124%2Fdmd.108.022814"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17e&amp;dbid=16384&amp;doi=10.1124%2Fdmd.108.022814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.108.022814%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DJ.%26aulast%3DWright%26aufirst%3DS.%26aulast%3DLucas%26aufirst%3DA.%26aulast%3DClarke%26aufirst%3DK.%2BL.%26aulast%3DViggers%26aufirst%3DJ.%26aulast%3DCheetham%26aufirst%3DS.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DPorter%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26atitle%3DReceptor%2520Occupancy%2520and%2520Brain%2520Free%2520Fraction%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2009%26volume%3D37%26spage%3D753%26epage%3D760%26doi%3D10.1124%2Fdmd.108.022814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Keck, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burzynski, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">Beyond Small-molecule SAR: Using the Dopamine D3 Receptor Crystal Structure to Guide Drug Design</span> <span class="citation_source-journal">Adv. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">267</span><span class="NLM_x">–</span> <span class="NLM_lpage">300</span><span class="refDoi"> DOI: 10.1016/B978-0-12-420118-7.00007-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1016%2FB978-0-12-420118-7.00007-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=24484980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsFCitb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2014&pages=267-300&author=T.+M.+Keckauthor=C.+Burzynskiauthor=L.+Shiauthor=A.+H.+Newman&title=Beyond+Small-molecule+SAR%3A+Using+the+Dopamine+D3+Receptor+Crystal+Structure+to+Guide+Drug+Design&doi=10.1016%2FB978-0-12-420118-7.00007-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18aR"><div class="casContent"><span class="casTitleNuber">18a</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond small-molecule SAR: using the dopamine D3 receptor crystal structure to guide drug design</span></div><div class="casAuthors">Keck, Thomas M.; Burzynski, Caitlin; Shi, Lei; Newman, Amy Hauck</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Pharmacology (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">Emerging Targets & Therapeutics in the Treatment of Psychostimulant Abuse</span>),
    <span class="NLM_cas:pages">267-300</span>CODEN:
                <span class="NLM_cas:coden">ADPHEL</span>;
        ISSN:<span class="NLM_cas:issn">1054-3589</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The dopamine D3 receptor is a target of pharmacotherapeutic interest in a variety of neurol. disorders including schizophrenia, restless leg syndrome, and drug addiction.  The high protein sequence homol. between the D3 and D2 receptors has posed a challenge to developing D3 receptor-selective ligands whose behavioral actions can be attributed to D3 receptor engagement, in vivo.  However, through primarily small-mol. structure-activity relationship (SAR) studies, a variety of chem. scaffolds have been discovered over the past two decades that have resulted in several D3 receptor-selective ligands with high affinity and in vivo activity.  Nevertheless, viable clin. candidates remain limited.  The recent detn. of the high-resoln. crystal structure of the D3 receptor has invigorated structure-based drug design, providing refinements to the mol. dynamic models and testable predictions about receptor-ligand interactions.  This chapter will highlight recent preclin. and clin. studies demonstrating potential utility of D3 receptor-selective ligands in the treatment of addiction.  In addn., new structure-based rational drug design strategies for D3 receptor-selective ligands that complement traditional small-mol. SAR to improve the selectivity and directed efficacy profiles are examd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbH7VLMIagF7Vg90H21EOLACvtfcHk0ljYUkmSBEuJNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsFCitb8%253D&md5=d266814a068e4528c70e200a860fab64</span></div><a href="/servlet/linkout?suffix=cit18a&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-420118-7.00007-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-420118-7.00007-X%26sid%3Dliteratum%253Aachs%26aulast%3DKeck%26aufirst%3DT.%2BM.%26aulast%3DBurzynski%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DBeyond%2520Small-molecule%2520SAR%253A%2520Using%2520the%2520Dopamine%2520D3%2520Receptor%2520Crystal%2520Structure%2520to%2520Guide%2520Drug%2520Design%26jtitle%3DAdv.%2520Pharmacol.%26date%3D2014%26volume%3D69%26spage%3D267%26epage%3D300%26doi%3D10.1016%2FB978-0-12-420118-7.00007-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit18b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Ligand Discovery from a Dopamine D3 Receptor Homology Model and Crystal Structure</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">769</span><span class="NLM_x">–</span> <span class="NLM_lpage">778</span><span class="refDoi"> DOI: 10.1038/nchembio.662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1038%2Fnchembio.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=21926995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=769-778&author=J.+Carlssonauthor=R.+G.+Colemanauthor=V.+Setolaauthor=J.+J.+Irwinauthor=H.+Fanauthor=A.+Schlessingerauthor=A.+Saliauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Ligand+Discovery+from+a+Dopamine+D3+Receptor+Homology+Model+and+Crystal+Structure&doi=10.1038%2Fnchembio.662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18bR"><div class="casContent"><span class="casTitleNuber">18b</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span></div><div class="casAuthors">Carlsson, Jens; Coleman, Ryan G.; Setola, Vincent; Irwin, John J.; Fan, Hao; Schlessinger, Avner; Sali, Andrej; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">769-778</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are intensely studied as drug targets and for their role in signaling.  With the detn. of the first crystal structures, interest in structure-based ligand discovery increased.  Unfortunately, for most GPCRs no exptl. structures are available.  The detn. of the D3 receptor structure and the challenge to the community to predict it enabled a fully prospective comparison of ligand discovery from a modeled structure vs. that of the subsequently released crystal structure.  Over 3.3 million mols. were docked against a homol. model, and 26 of the highest ranking were tested for binding.  Six had affinities ranging from 0.2 to 3.1 μM.  Subsequently, the crystal structure was released and the docking screen repeated.  Of the 25 compds. selected, five had affinities ranging from 0.3 to 3.0 μM.  One of the new ligands from the homol. model screen was optimized for affinity to 81 nM.  The feasibility of docking screens against modeled GPCRs more generally is considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6gBx3Olbn7Vg90H21EOLACvtfcHk0lgORBwOHuv10Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP&md5=7242aa512079a348678b309d352ecdb9</span></div><a href="/servlet/linkout?suffix=cit18b&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.662%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520Discovery%2520from%2520a%2520Dopamine%2520D3%2520Receptor%2520Homology%2520Model%2520and%2520Crystal%2520Structure%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D769%26epage%3D778%26doi%3D10.1038%2Fnchembio.662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit18c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Vass, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agai-Csongor, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span> </span><span class="NLM_article-title">Multiple Fragment Docking and Linking in Primary and Secondary Pockets of Dopamine Receptors</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1010</span><span class="NLM_x">–</span> <span class="NLM_lpage">1014</span><span class="refDoi"> DOI: 10.1021/ml500201u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500201u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFeltbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1010-1014&author=M.+Vassauthor=E.+Agai-Csongorauthor=F.+Hortiauthor=G.+M.+Keseru&title=Multiple+Fragment+Docking+and+Linking+in+Primary+and+Secondary+Pockets+of+Dopamine+Receptors&doi=10.1021%2Fml500201u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18cR"><div class="casContent"><span class="casTitleNuber">18c</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple Fragment Docking and Linking in Primary and Secondary Pockets of Dopamine Receptors</span></div><div class="casAuthors">Vass, Marton; Agai-Csongor, Eva; Horti, Ferenc; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1010-1014</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A sequential docking methodol. was applied to computationally predict starting points for fragment linking using the human dopamine D3 receptor crystal structure and a human dopamine D2 receptor homol. model.  Two focused fragment libraries were docked in the primary and secondary binding sites, and best fragment combinations were enumerated.  Similar top scoring fragments were found for the primary site, while secondary site fragments were predicted to convey selectivity.  Three linked compds. were synthesized that had 9-, 39-, and 55-fold selectivity in favor of D3 and the subtype selectivity of the compds. was assessed on a structural basis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIkfBkGmhbwrVg90H21EOLACvtfcHk0lgORBwOHuv10Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFeltbbO&md5=7531b04aabc8b12a4b975f6dc970134d</span></div><a href="/servlet/linkout?suffix=cit18c&amp;dbid=16384&amp;doi=10.1021%2Fml500201u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500201u%26sid%3Dliteratum%253Aachs%26aulast%3DVass%26aufirst%3DM.%26aulast%3DAgai-Csongor%26aufirst%3DE.%26aulast%3DHorti%26aufirst%3DF.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DMultiple%2520Fragment%2520Docking%2520and%2520Linking%2520in%2520Primary%2520and%2520Secondary%2520Pockets%2520of%2520Dopamine%2520Receptors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D1010%26epage%3D1014%26doi%3D10.1021%2Fml500201u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Böhm, H.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flohr, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">Scaffold Hopping</span> <span class="citation_source-journal">Drug Discovery Today: Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">217</span><span class="NLM_x">–</span> <span class="NLM_lpage">224</span><span class="refDoi"> DOI: 10.1016/j.ddtec.2004.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1016%2Fj.ddtec.2004.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=24981488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A280%3ADC%252BC2cfosVaktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=217-224&author=H.-J.+B%C3%B6hmauthor=A.+Flohrauthor=M.+Stahl&title=Scaffold+Hopping&doi=10.1016%2Fj.ddtec.2004.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Scaffold hopping</span></div><div class="casAuthors">Bohm Hans-Joachim; Flohr Alexander; Stahl Martin</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">217-24</span>
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    </div><div class="casAbstract">The aim of scaffold hopping is to discover structurally novel compounds starting from known active compounds by modifying the central core structure of the molecule.  Scaffold hopping is a central task of modern medicinal chemistry requiring a multitude of techniques, which are discussed in this article.  Their application has led to several molecules with chemically completely different core structures, and yet binding to the same receptor.  Computational approaches for scaffold hopping highlight the challenges of the field that are still unsolved.:</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSkmF4guWxLWsKL0FlyGHDofW6udTcc2eY7zA3jNjicJrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cfosVaktQ%253D%253D&md5=9a90f37dd2accf66bfd379f9ec602e03</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2004.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2004.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6hm%26aufirst%3DH.-J.%26aulast%3DFlohr%26aufirst%3DA.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DScaffold%2520Hopping%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2004%26volume%3D1%26spage%3D217%26epage%3D224%26doi%3D10.1016%2Fj.ddtec.2004.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span>MarvinSketch 6.1.6, (build id: 6.1.6_b695); <span class="NLM_publisher-name">Chemaxon</span>: <span class="NLM_publisher-loc">Budapest</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">; </span>(<a href="http://www.chemaxon.com" class="extLink">www.chemaxon.com</a>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MarvinSketch+6.1.6%2C+%28build+id%3A+6.1.6_b695%29%3B+Chemaxon%3A+Budapest%2C+2014%3B+%28www.chemaxon.com%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMarvinSketch%25206.1.6%252C%2520%2528build%2520id%253A%25206.1.6_b695%2529%26pub%3DChemaxon%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span>Molecular Operating Environment (MOE), 2013.08; <span class="NLM_publisher-name">Chemical Computing Group Inc.</span>: <span class="NLM_publisher-loc">1010 Sherbooke St. West, Suite #910, Montreal, QC, Canada, H3A 2R7</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Molecular+Operating+Environment+%28MOE%29%2C+2013.08%3B+Chemical+Computing+Group+Inc.%3A+1010+Sherbooke+St.+West%2C+Suite+%23910%2C+Montreal%2C+QC%2C+Canada%2C+H3A+2R7%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%252C%25202013.08%26pub%3DChemical%2520Computing%2520Group%2520Inc%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="note"><p class="first last">The research complied with national legislation and with company policy on the Care and Use of Animals and with related codes of practice.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Matsuda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satsukawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takimoto, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morisaki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, S.</span><span> </span><span class="NLM_article-title">Assessment of Intestinal Availability of Various Drugs in the Oral Absorption Process Using Portal Vein-cannulated Rats</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">2231</span><span class="NLM_x">–</span> <span class="NLM_lpage">2238</span><span class="refDoi"> DOI: 10.1124/dmd.112.048223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1124%2Fdmd.112.048223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=22930277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslyjsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=2231-2238&author=Y.+Matsudaauthor=Y.+Konnoauthor=M.+Satsukawaauthor=T.+Kobayashiauthor=Y.+Takimotoauthor=K.+Morisakiauthor=S.+Yamashita&title=Assessment+of+Intestinal+Availability+of+Various+Drugs+in+the+Oral+Absorption+Process+Using+Portal+Vein-cannulated+Rats&doi=10.1124%2Fdmd.112.048223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of intestinal availability of various drugs in the oral absorption process using portal vein-cannulated rats</span></div><div class="casAuthors">Matsuda, Yoshiki; Konno, Yoshihiro; Satsukawa, Masahiro; Kobayashi, Taro; Takimoto, Yu; Morisaki, Kunihiko; Yamashita, Shinji</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2231-2238</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">To understand the rate-limiting process of oral drug absorption, not only total bioavailability (F) but also intestinal (Fa · Fg) and hepatic (Fh) availability after oral administration should be evaluated.  Usually, Fa · Fg of drug is calcd. from pharmacokinetic parameters after i.v. and oral administration.  This approach is influenced markedly by the estd. value of Fh, which varies with the hepatic blood flow used in the calcns.  In this study, portal vein-cannulated rats were used to calc. the Fa · Fg of drugs from a single oral dosing expt. without data from i.v. injection.  Portal vein-cannulated rats were prepd. by a new operative method that enables stable portal vein blood flow.  This surgery had no effects on hepatic blood flow and metabolic activity.  Our method for calcg. Fa · Fg was validated by detg. both portal and systemic plasma concn. profiles of various drugs possessing different pharmacokinetic properties after oral administration to the portal vein-cannulated rats.  Simulation of portal and systemic plasma concns. by physiol. based pharmacokinetic modeling indicated that the balance of the absorption rate const. (ka) and elimination rate const. (ke) resulted in different patterns in portal and systemic plasma concn.-time profiles.  This study is expected to provide a new exptl. animal model that enables identification of the factors that limit oral bioavailability and to provide pharmacokinetic information on the oral absorption process of drugs during drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF2bFgGtdpXrVg90H21EOLACvtfcHk0lhJpysQgqdLGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslyjsrjF&md5=b184c93cf08fb29b8f36388683e7d582</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1124%2Fdmd.112.048223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.112.048223%26sid%3Dliteratum%253Aachs%26aulast%3DMatsuda%26aufirst%3DY.%26aulast%3DKonno%26aufirst%3DY.%26aulast%3DSatsukawa%26aufirst%3DM.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DTakimoto%26aufirst%3DY.%26aulast%3DMorisaki%26aufirst%3DK.%26aulast%3DYamashita%26aufirst%3DS.%26atitle%3DAssessment%2520of%2520Intestinal%2520Availability%2520of%2520Various%2520Drugs%2520in%2520the%2520Oral%2520Absorption%2520Process%2520Using%2520Portal%2520Vein-cannulated%2520Rats%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D40%26spage%3D2231%26epage%3D2238%26doi%3D10.1124%2Fdmd.112.048223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">van Breemen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, B.</span><span> </span><span class="NLM_article-title">Cado-2 cell permeability assays to measure drug absorption</span> <span class="citation_source-journal">Expert Opin. Drug Metab. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">185</span><span class="refDoi"> DOI: 10.1517/17425255.1.2.175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1517%2F17425255.1.2.175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=16922635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKltLbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=175-185&author=R.+van+Breemenauthor=B.+Li&title=Cado-2+cell+permeability+assays+to+measure+drug+absorption&doi=10.1517%2F17425255.1.2.175"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Caco-2 cell permeability assays to measure drug absorption</span></div><div class="casAuthors">van Breemen, Richard B.; Li, Yongmei</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">175-185</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Ashley Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Caco-2 cells are a human colon epithelial cancer cell line used as a model of human intestinal absorption of drugs and other compds.  When cultured as a monolayer, Caco-2 cells differentiate to form tight junctions between cells to serve as a model of paracellular movement of compds. across the mono-layer.  In addn., Caco-2 cells express transporter proteins, efflux proteins, and Phase II conjugation enzymes to model a variety of transcellular pathways as well as metabolic transformation of test substances.  In many respects, the Caco-2 cell monolayer mimics the human intestinal epithelium.  One of the functional differences between normal cells and Caco-2 cells is the lack of expression of the cytochrome P 450 isoenzymes and in particular, CYP3A4, which is normally expressed at high levels in the intestine.  However, Caco-2 cells may be induced to express higher levels of CYP3A4 by treatment with vitamin D3.  Caco-2 cell monolayers are usually cultured on semipermeable plastic supports that may be fitted into the wells of multi-well culture plates.  Test compds. are then added to either the apical or basolateral sides of the monolayer.  After incubation for various lengths of time, aliquots of the buffer in opposite chambers are removed for the detn. of the concn. of test compds. and the computation of the rates of permeability for each compd. (called the apparent permeability coeffs.).  Although radiolabeled compds. were used in the original Caco-2 cells monolayer assays, radio-labeled compds. have been replaced in most labs. by the use of liq. chromatog.-mass spectrometry (LC-MS) and LC-tandem mass spectrometry (LC-MS-MS).  Mass spectrometry not only eliminates the need for radiolabeled compds., but permits the simultaneous measurement of multiple compds.  The measurement of multiple compds. per assay reduces the no. of incubations that need to be carried out, thereby increasing the throughput of the expts.  Furthermore, LC-MS and LC-MS-MS add another dimension to Caco-2 assays by facilitating the investigation of the metab. of compds. by Caco-2 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcV-AZvy78ILVg90H21EOLACvtfcHk0lihNvwgKFbc3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKltLbK&md5=7bd0b875ffa9c840f904bab6513b7e3f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1517%2F17425255.1.2.175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.1.2.175%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BBreemen%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DB.%26atitle%3DCado-2%2520cell%2520permeability%2520assays%2520to%2520measure%2520drug%2520absorption%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2005%26volume%3D1%26spage%3D175%26epage%3D185%26doi%3D10.1517%2F17425255.1.2.175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Hay, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaumont, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kemp, M.</span><span> </span><span class="NLM_article-title">Modulation of the Partition Coefficient between Octanol and Buffer at pH 7.4 and pKa to Achieve the Optimum Balance of Blood Clearance and Volume of Distribution for a Series of Tetrahydropyran Histamine Type 3 Receptor Antagonists</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">1864</span><span class="NLM_x">–</span> <span class="NLM_lpage">1870</span><span class="refDoi"> DOI: 10.1124/dmd.109.027888</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1124%2Fdmd.109.027888" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=1864-1870&author=T.+Hayauthor=R.+Jonesauthor=K.+Beaumontauthor=M.+Kemp&title=Modulation+of+the+Partition+Coefficient+between+Octanol+and+Buffer+at+pH+7.4+and+pKa+to+Achieve+the+Optimum+Balance+of+Blood+Clearance+and+Volume+of+Distribution+for+a+Series+of+Tetrahydropyran+Histamine+Type+3+Receptor+Antagonists&doi=10.1124%2Fdmd.109.027888"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25a&amp;dbid=16384&amp;doi=10.1124%2Fdmd.109.027888&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.109.027888%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DT.%26aulast%3DJones%26aufirst%3DR.%26aulast%3DBeaumont%26aufirst%3DK.%26aulast%3DKemp%26aufirst%3DM.%26atitle%3DModulation%2520of%2520the%2520Partition%2520Coefficient%2520between%2520Octanol%2520and%2520Buffer%2520at%2520pH%25207.4%2520and%2520pKa%2520to%2520Achieve%2520the%2520Optimum%2520Balance%2520of%2520Blood%2520Clearance%2520and%2520Volume%2520of%2520Distribution%2520for%2520a%2520Series%2520of%2520Tetrahydropyran%2520Histamine%2520Type%25203%2520Receptor%2520Antagonists%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2009%26volume%3D37%26spage%3D1864%26epage%3D1870%26doi%3D10.1124%2Fdmd.109.027888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit25b"><span><span class="NLM_label">(b) </span> <span class="citation_source-book">Lead Generation: Methods and Strategies</span>; <span class="NLM_contrib-group">Holenz, J.; Mannhold, R.; Kubinyi, H.; Folkers, G.</span>, Eds.; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">67</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1002%2F9783527677047" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=J.+Holenz&author=R.+Mannhold&author=H.+Kubinyi&author=G.+Folkers&title=Lead+Generation%3A+Methods+and+Strategies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25b&amp;dbid=16384&amp;doi=10.1002%2F9783527677047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527677047%26sid%3Dliteratum%253Aachs%26btitle%3DLead%2520Generation%253A%2520Methods%2520and%2520Strategies%26aulast%3DHolenz%26aufirst%3DJ.%26pub%3DWiley%26date%3D2016%26volume%3D67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit25c"><span><span class="NLM_label">(c) </span> <span class="citation_source-book">Pharmacokinetics and Metabolism in Drug Design</span>, <span class="NLM_edition">3rd</span> ed.; <span class="NLM_contrib-group">Smith, D. A.; Allerton, C.; Kalgutkar, A. S.; van de Waterbeemd, H.; Walker, D. K.</span>, Eds.; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">; </span> <span class="NLM_volume">Vol. 51</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1002%2F9783527645763" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=D.+A.+Smith&author=C.+Allerton&author=A.+S.+Kalgutkar&author=H.+van+de+Waterbeemd&author=D.+K.+Walker&title=Pharmacokinetics+and+Metabolism+in+Drug+Design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25c&amp;dbid=16384&amp;doi=10.1002%2F9783527645763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527645763%26sid%3Dliteratum%253Aachs%26btitle%3DPharmacokinetics%2520and%2520Metabolism%2520in%2520Drug%2520Design%26aulast%3DSmith%26aufirst%3DD.%2BA.%26pub%3DWiley%26date%3D2012%26volume%3DVol.%252051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span>Cerep; <a href="http://www.cerep.fr" class="extLink">www.cerep.fr</a> (accessed August 17, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cerep%3B+www.cerep.fr+%28accessed+August+17%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DCerep" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span>SAINT: SAX, Area Detector Integration, Siemens Analytical instruments INC., Madison, Wisconsin, USA; SADABS: Siemens Area Detector Absorption Correction Software; <span class="NLM_contrib-group">Sheldrick, G.</span>, Ed.; <span class="NLM_publisher-name">University of Göttingen</span>, <span class="NLM_publisher-loc">Göttingen, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">1996</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=G.+Sheldrick&title=SAINT%3A+SAX%2C+Area+Detector+Integration%2C+Siemens+Analytical+instruments+INC.%2C+Madison%2C+Wisconsin%2C+USA%3B+SADABS%3A+Siemens+Area+Detector+Absorption+Correction+Software"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSAINT%253A%2520SAX%252C%2520Area%2520Detector%2520Integration%252C%2520Siemens%2520Analytical%2520instruments%2520INC.%252C%2520Madison%252C%2520Wisconsin%252C%2520USA%253B%2520SADABS%253A%2520Siemens%2520Area%2520Detector%2520Absorption%2520Correction%2520Software%26aulast%3DSheldrick%26aufirst%3DG.%26pub%3DUniversity%2520of%2520G%25C3%25B6ttingen%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Burla, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caliandro, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camalli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrozzini, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cascarano, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacovazzo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mallamo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazzone, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polidori, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spagna, R.</span><span> </span><span class="NLM_article-title">SIR2011: A New Package for Crystal Structure Determination and Refinement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">357</span><span class="NLM_x">–</span> <span class="NLM_lpage">361</span><span class="refDoi"> DOI: 10.1107/S0021889812001124</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1107%2FS0021889812001124" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFekt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2012&pages=357-361&author=M.+C.+Burlaauthor=R.+Caliandroauthor=M.+Camalliauthor=B.+Carrozziniauthor=G.+L.+Cascaranoauthor=C.+Giacovazzoauthor=M.+Mallamoauthor=A.+Mazzoneauthor=G.+Polidoriauthor=R.+Spagna&title=SIR2011%3A+A+New+Package+for+Crystal+Structure+Determination+and+Refinement&doi=10.1107%2FS0021889812001124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">SIR2011: a new package for crystal structure determination and refinement</span></div><div class="casAuthors">Burla, Maria Cristina; Caliandro, Rocco; Camalli, Mercedes; Carrozzini, Benedetta; Cascarano, Giovanni Luca; Giacovazzo, Carmelo; Mallamo, Mariarosaria; Mazzone, Annamaria; Polidori, Giampiero; Spagna, Riccardo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">357-361</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">SIR2011, the successor of SIR2004, is the latest program of the SIR suite.  It can solve ab initio crystal structures of small- and medium-size mols., as well as protein structures, using X-ray or electron diffraction data.  With respect to the predecessor the program has several new abilities: e.g. a new phasing method (VLD) has been implemented, it is able to exploit prior knowledge of the mol. geometry via simulated annealing techniques, it can use mol. replacement methods for solving proteins, it includes new tools like free lunch and new approaches for electron diffraction data, and it visualizes three-dimensional electron d. maps.  The graphical interface has been further improved and allows the straightforward use of the program even in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpo4B72KAqF9bVg90H21EOLACvtfcHk0lijG3dNR2tXIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFekt7k%253D&md5=13442a23f3edd07c5e92f4d6d7c71a9d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0021889812001124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889812001124%26sid%3Dliteratum%253Aachs%26aulast%3DBurla%26aufirst%3DM.%2BC.%26aulast%3DCaliandro%26aufirst%3DR.%26aulast%3DCamalli%26aufirst%3DM.%26aulast%3DCarrozzini%26aufirst%3DB.%26aulast%3DCascarano%26aufirst%3DG.%2BL.%26aulast%3DGiacovazzo%26aufirst%3DC.%26aulast%3DMallamo%26aufirst%3DM.%26aulast%3DMazzone%26aufirst%3DA.%26aulast%3DPolidori%26aufirst%3DG.%26aulast%3DSpagna%26aufirst%3DR.%26atitle%3DSIR2011%253A%2520A%2520New%2520Package%2520for%2520Crystal%2520Structure%2520Determination%2520and%2520Refinement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2012%26volume%3D45%26spage%3D357%26epage%3D361%26doi%3D10.1107%2FS0021889812001124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Sheldrick, G. M.</span><span> </span><span class="NLM_article-title">A short history of SHELX</span> <span class="citation_source-journal">Acta Crystallogr., Sect. A: Found. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">112</span><span class="NLM_x">–</span> <span class="NLM_lpage">122</span><span class="refDoi"> DOI: 10.1107/S0108767307043930</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1107%2FS0108767307043930" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=18156677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVGhurzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2008&pages=112-122&author=G.+M.+Sheldrick&title=A+short+history+of+SHELX&doi=10.1107%2FS0108767307043930"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A short history of SHELX</span></div><div class="casAuthors">Sheldrick, George M.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section A: Foundations of Crystallography</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-122</span>CODEN:
                <span class="NLM_cas:coden">ACACEQ</span>;
        ISSN:<span class="NLM_cas:issn">0108-7673</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">An account is given of the development of the SHELX system of computer programs from SHELX-76 to the present day.  In addn. to identifying useful innovations that have come into general use through their implementation in SHELX, a crit. anal. is presented of the less-successful features, missed opportunities and desirable improvements for future releases of the software.  An attempt is made to understand how a program originally designed for photog. intensity data, punched cards and computers over 10000 times slower than an av. modern personal computer has managed to survive for so long.  SHELXL is the most widely used program for small-mol. refinement and SHELXS and SHELXD are often employed for structure soln. despite the availability of objectively superior programs.  SHELXL also finds a niche for the refinement of macromols. against high-resoln. or twinned data; SHELXPRO acts as an interface for macromol. applications.  SHELXC, SHELXD and SHELXE are proving useful for the exptl. phasing of macromols., esp. because they are fast and robust and so are often employed in pipelines for high-throughput phasing.  This paper could serve as a general literature citation when one or more of the open-source SHELX programs (and the Bruker AXS version SHELXTL) are employed in the course of a crystal-structure detn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfLPYyJyJ2RbVg90H21EOLACvtfcHk0lijG3dNR2tXIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVGhurzO&md5=8f91f10be4a9df74b2a9dad522c71e6e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0108767307043930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108767307043930%26sid%3Dliteratum%253Aachs%26aulast%3DSheldrick%26aufirst%3DG.%2BM.%26atitle%3DA%2520short%2520history%2520of%2520SHELX%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%253A%2520Found.%2520Crystallogr.%26date%3D2008%26volume%3D64%26spage%3D112%26epage%3D122%26doi%3D10.1107%2FS0108767307043930" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Naïm, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rankin, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dennis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siddiqi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drabik, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sulea, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakalian, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purisima, E. O.</span><span> </span><span class="NLM_article-title">Solvated Interaction Energy (SIE) for Scoring Protein-ligand Binding Affinities. 1. Exploring the Parameter Space</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">122</span><span class="NLM_x">–</span> <span class="NLM_lpage">133</span><span class="refDoi"> DOI: 10.1021/ci600406v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600406v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A280%3ADC%252BD2s7ntVanug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=122-133&author=M.+Na%C3%AFmauthor=S.+Bhatauthor=K.+N.+Rankinauthor=S.+Dennisauthor=S.+F.+Chowdhuryauthor=I.+Siddiqiauthor=P.+Drabikauthor=T.+Suleaauthor=C.+I.+Baylyauthor=A.+Jakalianauthor=E.+O.+Purisima&title=Solvated+Interaction+Energy+%28SIE%29+for+Scoring+Protein-ligand+Binding+Affinities.+1.+Exploring+the+Parameter+Space&doi=10.1021%2Fci600406v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Solvated interaction energy (SIE) for scoring protein-ligand binding affinities. 1. Exploring the parameter space</span></div><div class="casAuthors">Naim Marwen; Bhat Sathesh; Rankin Kathryn N; Dennis Sheldon; Chowdhury Shafinaz F; Siddiqi Imran; Drabik Piotr; Sulea Traian; Bayly Christopher I; Jakalian Araz; Purisima Enrico O</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-33</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">We present a binding free energy function that consists of force field terms supplemented by solvation terms.  We used this function to calibrate the solvation model along with the binding interaction terms in a self-consistent manner.  The motivation for this approach was that the solute dielectric-constant dependence of calculated hydration gas-to-water transfer free energies is markedly different from that of binding free energies (J.  Comput.  Chem. 2003, 24, 954).  Hence, we sought to calibrate directly the solvation terms in the context of a binding calculation.  The five parameters of the model were systematically scanned to best reproduce the absolute binding free energies for a set of 99 protein-ligand complexes.  We obtained a mean unsigned error of 1.29 kcal/mol for the predicted absolute binding affinity in a parameter space that was fairly shallow near the optimum.  The lowest errors were obtained with solute dielectric values of Din = 20 or higher and scaling of the intermolecular van der Waals interaction energy by factors ranging from 0.03 to 0.15.  The high apparent Din and strong van der Waals scaling may reflect the anticorrelation of the change in solvated potential energy and configurational entropy, that is, enthalpy-entropy compensation in ligand binding (Biophys.  J. 2004, 87, 3035-3049).  Five variations of preparing the protein-ligand data set were explored in order to examine the effect of energy refinement and the presence of bound water on the calculated results.  We find that retaining water in the final protein structure used for calculating the binding free energy is not necessary to obtain good results; that is the continuum solvation model is sufficient.  Virtual screening enrichment studies on estrogen receptor and thymidine kinase showed a good ability of the binding free energy function to recover true hits in a collection of decoys.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR0C1HMXVybNqNc3HjJRoFAfW6udTcc2eakPKUPvWOg0Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s7ntVanug%253D%253D&md5=a5409b8aea519c6413f0fe77dbce4f57</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci600406v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600406v%26sid%3Dliteratum%253Aachs%26aulast%3DNa%25C3%25AFm%26aufirst%3DM.%26aulast%3DBhat%26aufirst%3DS.%26aulast%3DRankin%26aufirst%3DK.%2BN.%26aulast%3DDennis%26aufirst%3DS.%26aulast%3DChowdhury%26aufirst%3DS.%2BF.%26aulast%3DSiddiqi%26aufirst%3DI.%26aulast%3DDrabik%26aufirst%3DP.%26aulast%3DSulea%26aufirst%3DT.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DPurisima%26aufirst%3DE.%2BO.%26atitle%3DSolvated%2520Interaction%2520Energy%2520%2528SIE%2529%2520for%2520Scoring%2520Protein-ligand%2520Binding%2520Affinities.%25201.%2520Exploring%2520the%2520Parameter%2520Space%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D122%26epage%3D133%26doi%3D10.1021%2Fci600406v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Labute, P.</span><span> </span><span class="NLM_article-title">The Generalized Born/Volume Integral (GB/VI) Implicit Solvent Model: Estimation of the Free Energy of Hydration Using London Dispersion Instead of Atomic Surface Area</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1693</span><span class="NLM_x">–</span> <span class="NLM_lpage">1698</span><span class="refDoi"> DOI: 10.1002/jcc.20933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1002%2Fjcc.20933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=18307169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFCgu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=1693-1698&author=P.+Labute&title=The+Generalized+Born%2FVolume+Integral+%28GB%2FVI%29+Implicit+Solvent+Model%3A+Estimation+of+the+Free+Energy+of+Hydration+Using+London+Dispersion+Instead+of+Atomic+Surface+Area&doi=10.1002%2Fjcc.20933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The generalized Born/volume integral implicit solvent model: estimation of the free energy of hydration using London dispersion instead of atomic surface area</span></div><div class="casAuthors">Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1693-1698</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A new generalized Born model for estg. the free energy of hydration is presented.  The new generalized Born/vol. integral (GB/VI) ests. the free energy of hydration as a classical electrostatic energy plus a cavitation energy that is not based upon at. surface area (SA) used in GB/SA hydration models but on a VI London dispersion energy estd. from quantities already calcd. in the classical electrostatic energy.  The (relatively few) GB/VI model parameters are fitted to exptl. data, and parameterizations for two different at. partial charge models are presented.  Comparison of the calcd. and exptl. free energies of hydration for 560 small mols. (both neutral and charged) shows good agreement (r2 = 0.94).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCY3g7kWWRo7Vg90H21EOLACvtfcHk0likAPIgekaN5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFCgu7g%253D&md5=59420cf1dbe689836873df8e802d06cd</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20933%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DThe%2520Generalized%2520Born%252FVolume%2520Integral%2520%2528GB%252FVI%2529%2520Implicit%2520Solvent%2520Model%253A%2520Estimation%2520of%2520the%2520Free%2520Energy%2520of%2520Hydration%2520Using%2520London%2520Dispersion%2520Instead%2520of%2520Atomic%2520Surface%2520Area%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2008%26volume%3D29%26spage%3D1693%26epage%3D1698%26doi%3D10.1002%2Fjcc.20933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4a','cit4b'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12a','cit12b'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17a','cit17b','cit17c','cit17d','cit17e'],'ref18':['cit18a','cit18b','cit18c'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':[],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25a','cit25b','cit25c'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 11 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Liang Tan, Qingtong Zhou, Wenzhong Yan, Jian Sun, Alan P. Kozikowski, Suwen Zhao, Xi-Ping Huang, <span class="NLM_string-name hlFld-ContribAuthor">Jianjun Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Synthesis of Bitopic 2-Phenylcyclopropylmethylamine (PCPMA) Derivatives as Selective Dopamine D3 Receptor Ligands. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (9)
                                     , 4579-4602. <a href="https://doi.org/10.1021/acs.jmedchem.9b01835" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01835</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01835&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01835%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252BSynthesis%252Bof%252BBitopic%252B2-Phenylcyclopropylmethylamine%252B%252528PCPMA%252529%252BDerivatives%252Bas%252BSelective%252BDopamine%252BD3%252BReceptor%252BLigands%26aulast%3DTan%26aufirst%3DLiang%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D18112019%26date%3D23042020%26date%3D13042020%26volume%3D63%26issue%3D9%26spage%3D4579%26epage%3D4602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Anver
Basha Shaik, Vivek Kumar, Alessandro Bonifazi, Adrian M. Guerrero, Sophie L. Cemaj, Alexandra Gadiano, Jenny Lam, Zheng-Xiong Xi, Rana Rais, Barbara S. Slusher, <span class="NLM_string-name hlFld-ContribAuthor">Amy Hauck Newman</span>. </span><span class="cited-content_cbyCitation_article-title">Investigation of Novel Primary and Secondary Pharmacophores and 3-Substitution in the Linking Chain of a Series of Highly Selective and Bitopic Dopamine D3 Receptor Antagonists and Partial Agonists. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (20)
                                     , 9061-9077. <a href="https://doi.org/10.1021/acs.jmedchem.9b00607" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00607</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00607%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DInvestigation%252Bof%252BNovel%252BPrimary%252Band%252BSecondary%252BPharmacophores%252Band%252B3-Substitution%252Bin%252Bthe%252BLinking%252BChain%252Bof%252Ba%252BSeries%252Bof%252BHighly%252BSelective%252Band%252BBitopic%252BDopamine%252BD3%252BReceptor%252BAntagonists%252Band%252BPartial%252BAgonists%26aulast%3DShaik%26aufirst%3DAnver%2BBasha%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D08042019%26date%3D15102019%26date%3D17092019%26volume%3D62%26issue%3D20%26spage%3D9061%26epage%3D9077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sean W. Reilly, Aladdin A. Riad, Chia-Ju Hsieh, Kristoffer Sahlholm, Daniel A. Jacome, Suzy Griffin, Michelle Taylor, Chi-Chang Weng, Kuiying Xu, Nathan Kirschner, Robert R. Luedtke, Christopher Parry, Shipra Malhotra, John Karanicolas, <span class="NLM_string-name hlFld-ContribAuthor">Robert H. Mach</span>. </span><span class="cited-content_cbyCitation_article-title">Leveraging a Low-Affinity Diazaspiro Orthosteric Fragment to Reduce Dopamine D3 Receptor (D3R) Ligand Promiscuity across Highly Conserved Aminergic G-Protein-Coupled Receptors (GPCRs). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (10)
                                     , 5132-5147. <a href="https://doi.org/10.1021/acs.jmedchem.9b00412" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00412</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00412%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DLeveraging%252Ba%252BLow-Affinity%252BDiazaspiro%252BOrthosteric%252BFragment%252Bto%252BReduce%252BDopamine%252BD3%252BReceptor%252B%252528D3R%252529%252BLigand%252BPromiscuity%252Bacross%252BHighly%252BConserved%252BAminergic%252BG-Protein-Coupled%252BReceptors%252B%252528GPCRs%252529%26aulast%3DReilly%26aufirst%3DSean%2BW.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D12032019%26date%3D06052019%26date%3D25042019%26volume%3D62%26issue%3D10%26spage%3D5132%26epage%3D5147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maria
Beatrice Morelli, Consuelo Amantini, Massimo Nabissi, Giorgio Santoni, Bernhard Wünsch, Dirk Schepmann, Cristina Cimarelli, Maura Pellei, Carlo Santini, Stefano Fontana, Valerio Mammoli, Wilma Quaglia, Alessandro Bonifazi, Mario Giannella, Gianfabio Giorgioni, Alessandro Piergentili, <span class="NLM_string-name hlFld-ContribAuthor">Fabio Del Bello</span>. </span><span class="cited-content_cbyCitation_article-title">Role of the NMDA Receptor in the Antitumor Activity of Chiral 1,4-Dioxane Ligands in MCF-7 and SKBR3 Breast Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (4)
                                     , 511-516. <a href="https://doi.org/10.1021/acsmedchemlett.8b00536" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.8b00536</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.8b00536&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.8b00536%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DRole%252Bof%252Bthe%252BNMDA%252BReceptor%252Bin%252Bthe%252BAntitumor%252BActivity%252Bof%252BChiral%252B1%25252C4-Dioxane%252BLigands%252Bin%252BMCF-7%252Band%252BSKBR3%252BBreast%252BCancer%252BCells%26aulast%3DMorelli%26aufirst%3DMaria%2BBeatrice%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D07112018%26date%3D28012019%26date%3D31012019%26date%3D28012019%26volume%3D10%26issue%3D4%26spage%3D511%26epage%3D516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fabio Del Bello, Alessandro Bonifazi, Gianfabio Giorgioni, Carlo Cifani, Maria Vittoria Micioni Di Bonaventura, Riccardo Petrelli, Alessandro Piergentili, Stefano Fontana, Valerio Mammoli, Hideaki Yano, Rosanna Matucci, Giulio Vistoli, <span class="NLM_string-name hlFld-ContribAuthor">Wilma Quaglia</span>. </span><span class="cited-content_cbyCitation_article-title">1-[3-(4-Butylpiperidin-1-yl)propyl]-1,2,3,4-tetrahydroquinolin-2-one (77-LH-28-1) as a Model for the Rational Design of a Novel Class of Brain Penetrant Ligands with High Affinity and Selectivity for Dopamine D4 Receptor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (8)
                                     , 3712-3725. <a href="https://doi.org/10.1021/acs.jmedchem.8b00265" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00265</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00265%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3D1-%25255B3-%2525284-Butylpiperidin-1-yl%252529propyl%25255D-1%25252C2%25252C3%25252C4-tetrahydroquinolin-2-one%252B%25252877-LH-28-1%252529%252Bas%252Ba%252BModel%252Bfor%252Bthe%252BRational%252BDesign%252Bof%252Ba%252BNovel%252BClass%252Bof%252BBrain%252BPenetrant%252BLigands%252Bwith%252BHigh%252BAffinity%252Band%252BSelectivity%252Bfor%252BDopamine%252BD4%252BReceptor%26aulast%3DDel%2BBello%26aufirst%3DFabio%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D16022018%26date%3D06042018%26date%3D28032018%26volume%3D61%26issue%3D8%26spage%3D3712%26epage%3D3725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sean W.  Reilly</span>, <span class="hlFld-ContribAuthor ">Suzy  Griffin</span>, <span class="hlFld-ContribAuthor ">Michelle  Taylor</span>, <span class="hlFld-ContribAuthor ">Kristoffer  Sahlholm</span>, <span class="hlFld-ContribAuthor ">Chi-Chang  Weng</span>, <span class="hlFld-ContribAuthor ">Kuiying  Xu</span>, <span class="hlFld-ContribAuthor ">Daniel A.  Jacome</span>, <span class="hlFld-ContribAuthor ">Robert R.  Luedtke</span>, and <span class="hlFld-ContribAuthor ">Robert H.  Mach</span>  . </span><span class="cited-content_cbyCitation_article-title">Highly Selective Dopamine D3 Receptor Antagonists with Arylated Diazaspiro Alkane Cores. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (23)
                                     , 9905-9910. <a href="https://doi.org/10.1021/acs.jmedchem.7b01248" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01248</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01248%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHighly%252BSelective%252BDopamine%252BD3%252BReceptor%252BAntagonists%252Bwith%252BArylated%252BDiazaspiro%252BAlkane%252BCores%26aulast%3DReilly%26aufirst%3DSean%2BW.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D23082017%26date%3D21112017%26date%3D14122017%26date%3D10112017%26volume%3D60%26issue%3D23%26spage%3D9905%26epage%3D9910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomas  Opsomer</span>, <span class="hlFld-ContribAuthor ">Wim  Dehaen</span>. </span><span class="cited-content_cbyCitation_article-title">1,2,4-Triazoles. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,<a href="https://doi.org/10.1016/B978-0-12-409547-2.14854-1" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.14854-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.14854-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.14854-1%26sid%3Dliteratum%253Aachs%26atitle%3D1%25252C2%25252C4-Triazoles%26aulast%3DOpsomer%26aufirst%3DTomas%26date%3D2020%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BChemistry%25252C%252BMolecular%252BSciences%252Band%252BChemical%252BEngineering%26date%3D2020%26volume%3Dvol.%25205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert K.  Doot</span>, <span class="hlFld-ContribAuthor ">Jacob G.  Dubroff</span>, <span class="hlFld-ContribAuthor ">Kyle J.  Labban</span>, <span class="hlFld-ContribAuthor ">Robert H.  Mach</span>. </span><span class="cited-content_cbyCitation_article-title">Selectivity of probes for PET imaging of dopamine D3 receptors. </span><span class="cited-content_cbyCitation_journal-name">Neuroscience Letters</span><span> <strong>2019,</strong> <em>691 </em>, 18-25. <a href="https://doi.org/10.1016/j.neulet.2018.03.006" title="DOI URL">https://doi.org/10.1016/j.neulet.2018.03.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.neulet.2018.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.neulet.2018.03.006%26sid%3Dliteratum%253Aachs%26jtitle%3DNeuroscience%2520Letters%26atitle%3DSelectivity%252Bof%252Bprobes%252Bfor%252BPET%252Bimaging%252Bof%252Bdopamine%252BD3%252Breceptors%26aulast%3DDoot%26aufirst%3DRobert%2BK.%26date%3D2019%26volume%3D691%26spage%3D18%26epage%3D25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Zhang</span>, <span class="hlFld-ContribAuthor ">Hui  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">3D-QSAR studies on 1,2,4-triazolyl 5-azaspiro [2.4]-heptanes as D 3 R antagonists. </span><span class="cited-content_cbyCitation_journal-name">Chemical Physics Letters</span><span> <strong>2018,</strong> <em>704 </em>, 11-20. <a href="https://doi.org/10.1016/j.cplett.2018.05.022" title="DOI URL">https://doi.org/10.1016/j.cplett.2018.05.022</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cplett.2018.05.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cplett.2018.05.022%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Physics%2520Letters%26atitle%3D3D-QSAR%252Bstudies%252Bon%252B1%25252C2%25252C4-triazolyl%252B5-azaspiro%252B%25255B2.4%25255D-heptanes%252Bas%252BD%252B3%252BR%252Bantagonists%26aulast%3DZhang%26aufirst%3DXin%26date%3D2018%26volume%3D704%26spage%3D11%26epage%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuri  Shafran</span>, <span class="hlFld-ContribAuthor ">Tatiana  Glukhareva</span>, <span class="hlFld-ContribAuthor ">Wim  Dehaen</span>, <span class="hlFld-ContribAuthor ">Vasiliy  Bakulev</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Developments in the Chemistry of 1,2,3-Thiadiazoles. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 109-172. <a href="https://doi.org/10.1016/bs.aihch.2017.12.001" title="DOI URL">https://doi.org/10.1016/bs.aihch.2017.12.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.aihch.2017.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.aihch.2017.12.001%26sid%3Dliteratum%253Aachs%26atitle%3DRecent%252BDevelopments%252Bin%252Bthe%252BChemistry%252Bof%252B1%25252C2%25252C3-Thiadiazoles%26aulast%3DShafran%26aufirst%3DYuri%26date%3D2018%26spage%3D109%26epage%3D172%26pub%3DElsevier%26date%3D2018%26volume%3D126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fabrizio  Micheli</span>. </span><span class="cited-content_cbyCitation_article-title">Novel, Selective, and Developable Dopamine D
              3
              Antagonists with a Modified “Amino” Region. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2017,</strong> <em>12 </em>
                                    (16)
                                     , 1254-1260. <a href="https://doi.org/10.1002/cmdc.201700148" title="DOI URL">https://doi.org/10.1002/cmdc.201700148</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700148%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DNovel%25252C%252BSelective%25252C%252Band%252BDevelopable%252BDopamine%252BD%252B3%252BAntagonists%252Bwith%252Ba%252BModified%252B%2525E2%252580%25259CAmino%2525E2%252580%25259D%252BRegion%26aulast%3DMicheli%26aufirst%3DFabrizio%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D16%26spage%3D1254%26epage%3D1260" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00972&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of known and new D3R antagonists.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00972&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Schematic model of the D3R pharmacophore described in the early 2000 aligned with derivative <b>1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00972&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0006.jpeg" id="rightTab-GRAPHIC-d126e531-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Scheme<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00972&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(i) MnO<sub>2</sub>, dioxane, RT; (ii) 1M LiAlH<sub>4</sub> THF, reflux; (iii) N<sub>2</sub>, NH<sub>4</sub>COOH, Pd/C, MeOH, reflux; (iv) T3P in AcOEt, 4-methyl-3-thiosemicarbazide, NaOH; (v) K<sub>2</sub>CO<sub>3</sub>/acetone/MeOH, RT, alkylating agent; (vi) Na<sub>2</sub>CO<sub>3</sub>, NaI, DMF or CH<sub>3</sub>CN, 60 °C.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Relevant interactions of derivative <b>2</b> (yellow) and <b>10a</b> (red) in docking to the D3R model. Portions of the backbone ribbon have been removed for clarity. The interaction of the basic nitrogen of the amine portion of the two compounds with Asp<sup>3,32</sup> is clearly visible. The region of the thiotriazole and of the oxazole portion clearly points toward the EL. The new 5-azaspiro[2.4]heptanes seem to direct the −CF<sub>3</sub> moiety of derivative <b>10a</b> in a similar hydrophobic region in common with compound <b>2</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00972&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Derivatives <b>40a</b> (cyan), <b>41a</b> (mustard), and <b>42a</b> (red) docking to the D3R model. Portions of the backbone ribbon have been removed for clarity. The interaction of the basic nitrogen of the amine portion of the two compounds with Asp<sup>3,32</sup> is clearly visible. For derivative <b>40a</b>, the potential interaction with Cys<sup>3</sup><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> is also highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00972&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/medium/jm-2016-00972c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Left: asymmetric unit of 5-(4-methylbenzenesulfonyl)-(1<i>R</i>,3<i>S</i>)-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>88</b>) comprising four molecular entities (I–IV), right: molecular structure of (<b>88</b>), with thermal ellipsoids drawn at the 50% probability level. (B) Left: asymmetric unit of 5-(4-methylbenzenesulfonyl)-(1<i>S</i>,3<i>R</i>)-[4-(trifluoromethyl)phenyl]-5-azaspiro[2.4]heptane (<b>89</b>) comprising four molecular entities (I–IV). Right: molecular structure of (<b>89</b>), with thermal ellipsoids drawn at the 50% probability level.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-18/acs.jmedchem.6b00972/20160916/images/large/jm-2016-00972c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00972&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i134">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40864" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40864" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 31 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Heidbreder, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xi, Z.-X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thanos, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mugnaini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagan, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashby, C. R.,  Jr.</span><span> </span><span class="NLM_article-title">The Role of Central DA D<sub>3</sub> Receptors in Drug Addiction: a Review of Pharmacological Evidence</span> <span class="citation_source-journal">Brain Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">77</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1016/j.brainresrev.2004.12.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1016%2Fj.brainresrev.2004.12.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=15960988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlt1Ghu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2005&pages=77-105&author=C.+A.+Heidbrederauthor=E.+L.+Gardnerauthor=Z.-X.+Xiauthor=P.+K.+Thanosauthor=M.+Mugnainiauthor=J.+J.+Haganauthor=C.+R.+Ashby&title=The+Role+of+Central+DA+D3+Receptors+in+Drug+Addiction%3A+a+Review+of+Pharmacological+Evidence&doi=10.1016%2Fj.brainresrev.2004.12.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The role of central dopamine D3 receptors in drug addiction: a review of pharmacological evidence</span></div><div class="casAuthors">Heidbreder, Christian A.; Gardner, Eliot L.; Xi, Zheng-Xiong; Thanos, Panayotis K.; Mugnaini, Manolo; Hagan, Jim J.; Ashby, Charles R.</div><div class="citationInfo"><span class="NLM_cas:title">Brain Research Reviews</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">77-105</span>CODEN:
                <span class="NLM_cas:coden">BRERD2</span>;
        ISSN:<span class="NLM_cas:issn">0165-0173</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The cDNA for the dopamine D3 receptor was isolated and characterized in 1990.  Subsequent studies have indicated that D3 receptors, as well as D3 receptor mRNA, are primarily localized in limbic regions in mammals.  This finding led to the postulate that D3 receptors may be involved in drug dependence and addiction.  However, this hypothesis has been difficult to test due to the lack of compds. with high selectivity for central D3 receptors.  The interpretation of results from studies using mixed D2/D3 agonists and/or antagonists is problematic because these agents have low selectivity for D3 over D2 receptors and it is likely that their actions are primarily related to D2 receptor antagonism and possibly interaction with other neurotransmitter receptors.  Currently, with the synthesis and characterization of new highly selective D3 receptor antagonists such as SB-277011-A this difficulty has been surmounted.  The purpose of the present article is to review, for the first time, the effects of various putative D3 receptor selective compds. in animal models of drug dependence and addiction.  The results obtained with highly selective D3 receptor antagonists such as SB-277011-A, SB-414796, and NGB-2904 indicate that central D3 receptors may play an important role in drug-induced reward, drug-taking, and cue-, drug-, and stress-induced reinstatement of drug-seeking behavior.  Provided these results can be extrapolated to human drug addicts, they suggest that selective DA D3 receptor antagonists may prove effective as potential pharmacotherapeutic agents to manage drug dependence and addiction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwzEJPprm60rVg90H21EOLACvtfcHk0liSWuMVlk36nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlt1Ghu7k%253D&md5=3e5948e0f94515fd240d1e3d18812028</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.brainresrev.2004.12.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.brainresrev.2004.12.033%26sid%3Dliteratum%253Aachs%26aulast%3DHeidbreder%26aufirst%3DC.%2BA.%26aulast%3DGardner%26aufirst%3DE.%2BL.%26aulast%3DXi%26aufirst%3DZ.-X.%26aulast%3DThanos%26aufirst%3DP.%2BK.%26aulast%3DMugnaini%26aufirst%3DM.%26aulast%3DHagan%26aufirst%3DJ.%2BJ.%26aulast%3DAshby%26aufirst%3DC.%2BR.%26atitle%3DThe%2520Role%2520of%2520Central%2520DA%2520D3%2520Receptors%2520in%2520Drug%2520Addiction%253A%2520a%2520Review%2520of%2520Pharmacological%2520Evidence%26jtitle%3DBrain%2520Res.%2520Rev.%26date%3D2005%26volume%3D49%26spage%3D77%26epage%3D105%26doi%3D10.1016%2Fj.brainresrev.2004.12.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Heidbreder, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">Current Perspectives on Selective Dopamine D3 Receptor Antagonists as Pharmacotherapeutics for Addictions and Related Disorders</span> <span class="citation_source-journal">Ann. N. Y. Acad. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1187</span><span class="NLM_x">, </span> <span class="NLM_fpage">4</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1111/j.1749-6632.2009.05149.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1111%2Fj.1749-6632.2009.05149.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1187&publication_year=2010&pages=4-34&author=C.+A.+Heidbrederauthor=A.+H.+Newman&title=Current+Perspectives+on+Selective+Dopamine+D3+Receptor+Antagonists+as+Pharmacotherapeutics+for+Addictions+and+Related+Disorders&doi=10.1111%2Fj.1749-6632.2009.05149.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1111%2Fj.1749-6632.2009.05149.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1749-6632.2009.05149.x%26sid%3Dliteratum%253Aachs%26aulast%3DHeidbreder%26aufirst%3DC.%2BA.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DCurrent%2520Perspectives%2520on%2520Selective%2520Dopamine%2520D3%2520Receptor%2520Antagonists%2520as%2520Pharmacotherapeutics%2520for%2520Addictions%2520and%2520Related%2520Disorders%26jtitle%3DAnn.%2520N.%2520Y.%2520Acad.%2520Sci.%26date%3D2010%26volume%3D1187%26spage%3D4%26epage%3D34%26doi%3D10.1111%2Fj.1749-6632.2009.05149.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Heidbreder, C.</span><span> </span><span class="NLM_article-title">Rationale in Support of the Use of Selective Dopamine D3 Receptor Antagonists for the Pharmacotherapeutic Management of Drug Addiction</span> <span class="citation_source-journal">Naunyn-Schmiedeberg's Arch. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">386</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">176</span><span class="refDoi"> DOI: 10.1007/s00210-012-0803-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1007%2Fs00210-012-0803-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=23104235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ejur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2013&pages=167-176&author=C.+Heidbreder&title=Rationale+in+Support+of+the+Use+of+Selective+Dopamine+D3+Receptor+Antagonists+for+the+Pharmacotherapeutic+Management+of+Drug+Addiction&doi=10.1007%2Fs00210-012-0803-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Rationale in support of the use of selective dopamine D3 receptor antagonists for the pharmacotherapeutic management of substance use disorders</span></div><div class="casAuthors">Heidbreder, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Naunyn-Schmiedeberg's Archives of Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">386</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">167-176</span>CODEN:
                <span class="NLM_cas:coden">NSAPCC</span>;
        ISSN:<span class="NLM_cas:issn">0028-1298</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Growing evidence indicates that dopamine (DA) D3 receptors are involved in the control of drug-seeking behavior and may play an important role in the pathophysiol. of substance use disorders.  First, DA D3 receptors are distributed in strategic areas belonging to the mesolimbic DA system such as the ventral striatum, midbrain, and pallidum, which have been assocd. with behaviors controlled by the presentation of drug-assocd. cues.  Second, repeated exposure to drugs of abuse has been shown to produce neuroadaptations in the DA D3 system.  Third, the synthesis and characterization of highly potent and selective DA D3 receptor antagonists has permitted to further define the role of the DA D3 receptor in drug addiction.  Provided that the available preclin. and preliminary clin. evidence can be translated into clin. proof of concept in human, selective DA D3 receptor antagonists show promise for the treatment of substance use disorders as reflected by their potential to (1) regulate the motivation to self-administered drugs under schedules of reinforcement that require an increase in work demand and (2) disrupt the responsiveness to drug-assocd. stimuli that play a key role in the reinstatement of drug-seeking behavior triggered by re-exposure to the drug itself, re-exposure to environmental cues that had been previously assocd. with drug-taking behavior, or stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJryekGxZ_AbVg90H21EOLACvtfcHk0lhAVb_F6cqfPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ejur0%253D&md5=4b196deebddce4d802a3dcef465d1dac</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1007%2Fs00210-012-0803-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00210-012-0803-6%26sid%3Dliteratum%253Aachs%26aulast%3DHeidbreder%26aufirst%3DC.%26atitle%3DRationale%2520in%2520Support%2520of%2520the%2520Use%2520of%2520Selective%2520Dopamine%2520D3%2520Receptor%2520Antagonists%2520for%2520the%2520Pharmacotherapeutic%2520Management%2520of%2520Drug%2520Addiction%26jtitle%3DNaunyn-Schmiedeberg%2527s%2520Arch.%2520Pharmacol.%26date%3D2013%26volume%3D386%26spage%3D167%26epage%3D176%26doi%3D10.1007%2Fs00210-012-0803-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Keck, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">John, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czoty, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nader, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">Identifying Medication Targets for Psychostimulant Addiction: Unraveling the Dopamine D3 Receptor Hypothesis</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5361</span><span class="NLM_x">–</span> <span class="NLM_lpage">5380</span><span class="refDoi"> DOI: 10.1021/jm501512b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501512b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsFGgu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5361-5380&author=T.+M.+Keckauthor=W.+S.+Johnauthor=P.+W.+Czotyauthor=M.+A.+Naderauthor=A.+H.+Newman&title=Identifying+Medication+Targets+for+Psychostimulant+Addiction%3A+Unraveling+the+Dopamine+D3+Receptor+Hypothesis&doi=10.1021%2Fjm501512b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4aR"><div class="casContent"><span class="casTitleNuber">4a</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Medication Targets for Psychostimulant Addiction: Unraveling the Dopamine D3 Receptor Hypothesis</span></div><div class="casAuthors">Keck, Thomas M.; John, William S.; Czoty, Paul W.; Nader, Michael A.; Newman, Amy Hauck</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5361-5380</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dopamine D3 receptor (D3R) is a target for developing medications to treat substance use disorders.  D3R-selective compds. with high affinity and varying efficacies have been discovered, providing crit. research tools for cell-based studies that have been translated to in vivo models of drug abuse.  D3R antagonists and partial agonists have shown esp. promising results in rodent models of relapse-like behavior, including stress-, drug-, and cue-induced reinstatement of drug seeking.  However, to date, translation to human studies has been limited.  Herein, we present an overview and illustrate some of the pitfalls and challenges of developing novel D3R-selective compds. toward clin. utility, esp. for treatment of cocaine abuse.  Future research and development of D3R-selective antagonists and partial agonists for substance abuse remains critically important but will also require further evaluation and development of translational animal models to det. the best time in the addiction cycle to target D3Rs for optimal therapeutic efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKVPGvcTvN87Vg90H21EOLACvtfcHk0lhAVb_F6cqfPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsFGgu74%253D&md5=99f531d840ad7f94d0eeadeb6fd48100</span></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1021%2Fjm501512b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501512b%26sid%3Dliteratum%253Aachs%26aulast%3DKeck%26aufirst%3DT.%2BM.%26aulast%3DJohn%26aufirst%3DW.%2BS.%26aulast%3DCzoty%26aufirst%3DP.%2BW.%26aulast%3DNader%26aufirst%3DM.%2BA.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DIdentifying%2520Medication%2520Targets%2520for%2520Psychostimulant%2520Addiction%253A%2520Unraveling%2520the%2520Dopamine%2520D3%2520Receptor%2520Hypothesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5361%26epage%3D5380%26doi%3D10.1021%2Fjm501512b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit4b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Micheli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidbreder, C.</span><span> </span><span class="NLM_article-title">Dopamine D3 Receptor Antagonists: a Patent Review (2007–2012)</span> <span class="citation_source-journal">Expert Opin. Ther. Pat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">363</span><span class="NLM_x">–</span> <span class="NLM_lpage">381</span><span class="refDoi"> DOI: 10.1517/13543776.2013.757593</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1517%2F13543776.2013.757593" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=23282131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1Sqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=363-381&author=F.+Micheliauthor=C.+Heidbreder&title=Dopamine+D3+Receptor+Antagonists%3A+a+Patent+Review+%282007%E2%80%932012%29&doi=10.1517%2F13543776.2013.757593"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4bR"><div class="casContent"><span class="casTitleNuber">4b</span><div class="casTitle"><span class="NLM_cas:atitle">Dopamine D3 receptor antagonists: a patent review (2007 - 2012)</span></div><div class="casAuthors">Micheli, Fabrizio; Heidbreder, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">363-381</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The synthesis and characterization of new highly potent and selective dopamine (DA) D3 receptor antagonists has permitted to characterize the role of the DA D3 receptor in the control of drug-seeking behavior and in the pathophysiol. of impulse control disorders and schizophrenia.Areas covered: In the present review, the authors will first describe most recent classes of DA D3 receptor antagonists by reviewing about 43 patent applications during the 2007 - 2012 period; they will then outline the biol. rationale in support of the use of selective DA D3 receptor antagonists in the treatment of drug addiction, impulse control disorders and schizophrenia.Expert opinion: The strongest clin. application and potential for selective DA D3 receptor antagonists lies in the redn. of drug-induced incentive motivation, the attenuation of drug's rewarding efficacy and the redn. in reinstatement of drug-seeking behavior triggered either by re-exposure to the drug itself, re-exposure to environmental cues that had been previously assocd. with drug-taking behavior or stress.  The selectivity of these antagonists together with reduced lipophilicity (minimizing unspecific binding), increased brain penetration and improved physico-chem. profile are all key factors for clin. efficacy and safety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoz7CcUwWNKXrVg90H21EOLACvtfcHk0lhNzyF59zVxjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1Sqsrk%253D&md5=cbf7c42a3a88693f44b1adea64f6caf3</span></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1517%2F13543776.2013.757593&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2013.757593%26sid%3Dliteratum%253Aachs%26aulast%3DMicheli%26aufirst%3DF.%26aulast%3DHeidbreder%26aufirst%3DC.%26atitle%3DDopamine%2520D3%2520Receptor%2520Antagonists%253A%2520a%2520Patent%2520Review%2520%25282007%25E2%2580%25932012%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2013%26volume%3D23%26spage%3D363%26epage%3D381%26doi%3D10.1517%2F13543776.2013.757593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Micheli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arista, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonanomi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaney, F. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braggio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelli, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Checchia, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damiani, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di-Fabio, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fontana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gentile, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffante, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamprecht, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchioro, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mugnaini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratti, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tedesco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarsi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terreni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Worby, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashby, C. R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidbreder, C.</span><span> </span><span class="NLM_article-title">1,2,4-Triazolyl Azabicyclo[3.1.0]-Hexanes: a New Series of Potent and Selective Dopamine D(3) Receptor Antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">374</span><span class="NLM_x">–</span> <span class="NLM_lpage">391</span><span class="refDoi"> DOI: 10.1021/jm901319p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901319p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=374-391&author=F.+Micheliauthor=L.+Aristaauthor=G.+Bonanomiauthor=F.+E.+Blaneyauthor=S.+Braggioauthor=A.+M.+Capelliauthor=A.+Checchiaauthor=F.+Damianiauthor=R.+Di-Fabioauthor=S.+Fontanaauthor=G.+Gentileauthor=C.+Griffanteauthor=D.+Hamprechtauthor=C.+Marchioroauthor=M.+Mugnainiauthor=J.+Pinerauthor=E.+Rattiauthor=G.+Tedescoauthor=L.+Tarsiauthor=S.+Terreniauthor=A.+Worbyauthor=C.+R.+Ashbyauthor=C.+Heidbreder&title=1%2C2%2C4-Triazolyl+Azabicyclo%5B3.1.0%5D-Hexanes%3A+a+New+Series+of+Potent+and+Selective+Dopamine+D%283%29+Receptor+Antagonists&doi=10.1021%2Fjm901319p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm901319p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901319p%26sid%3Dliteratum%253Aachs%26aulast%3DMicheli%26aufirst%3DF.%26aulast%3DArista%26aufirst%3DL.%26aulast%3DBonanomi%26aufirst%3DG.%26aulast%3DBlaney%26aufirst%3DF.%2BE.%26aulast%3DBraggio%26aufirst%3DS.%26aulast%3DCapelli%26aufirst%3DA.%2BM.%26aulast%3DChecchia%26aufirst%3DA.%26aulast%3DDamiani%26aufirst%3DF.%26aulast%3DDi-Fabio%26aufirst%3DR.%26aulast%3DFontana%26aufirst%3DS.%26aulast%3DGentile%26aufirst%3DG.%26aulast%3DGriffante%26aufirst%3DC.%26aulast%3DHamprecht%26aufirst%3DD.%26aulast%3DMarchioro%26aufirst%3DC.%26aulast%3DMugnaini%26aufirst%3DM.%26aulast%3DPiner%26aufirst%3DJ.%26aulast%3DRatti%26aufirst%3DE.%26aulast%3DTedesco%26aufirst%3DG.%26aulast%3DTarsi%26aufirst%3DL.%26aulast%3DTerreni%26aufirst%3DS.%26aulast%3DWorby%26aufirst%3DA.%26aulast%3DAshby%26aufirst%3DC.%2BR.%26aulast%3DHeidbreder%26aufirst%3DC.%26atitle%3D1%252C2%252C4-Triazolyl%2520Azabicyclo%255B3.1.0%255D-Hexanes%253A%2520a%2520New%2520Series%2520of%2520Potent%2520and%2520Selective%2520Dopamine%2520D%25283%2529%2520Receptor%2520Antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D374%26epage%3D391%26doi%3D10.1021%2Fjm901319p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Micheli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonanomi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaney, F. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braggio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelli, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Checchia, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curcuruto, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damiani, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Fabio, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donati, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gentile, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gribble, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamprecht, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tedesco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terreni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarsi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lightfoot, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stemp, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacDonald, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pecoraro, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrone, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perini, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Worby, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilla, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valerio, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffante, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mugnaini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andreoli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lacroix, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwarz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozzi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bifone, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashby, C. R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagan, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidbreder, C.</span><span> </span><span class="NLM_article-title">1,2,4-Triazol-3-yl-thiopropyl-tetrahydrobenzazepines: A Series of Potent and Selective Dopamine D3 Receptor Antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">5076</span><span class="NLM_x">–</span> <span class="NLM_lpage">5089</span><span class="refDoi"> DOI: 10.1021/jm0705612</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0705612" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=5076-5089&author=F.+Micheliauthor=G.+Bonanomiauthor=F.+E.+Blaneyauthor=S.+Braggioauthor=A.+M.+Capelliauthor=A.+Checchiaauthor=O.+Curcurutoauthor=F.+Damianiauthor=R.+Di+Fabioauthor=D.+Donatiauthor=G.+Gentileauthor=A.+Gribbleauthor=D.+Hamprechtauthor=G.+Tedescoauthor=S.+Terreniauthor=L.+Tarsiauthor=A.+Lightfootauthor=G.+Stempauthor=G.+MacDonaldauthor=A.+Smithauthor=M.+Pecoraroauthor=M.+Petroneauthor=O.+Periniauthor=J.+Pinerauthor=T.+Rossiauthor=A.+Worbyauthor=M.+Pillaauthor=E.+Valerioauthor=C.+Griffanteauthor=M.+Mugnainiauthor=M.+Woodauthor=C.+Scottauthor=M.+Andreoliauthor=L.+Lacroixauthor=A.+Schwarzauthor=A.+Gozziauthor=A.+Bifoneauthor=C.+R.+Ashbyauthor=J.+J.+Haganauthor=C.+Heidbreder&title=1%2C2%2C4-Triazol-3-yl-thiopropyl-tetrahydrobenzazepines%3A+A+Series+of+Potent+and+Selective+Dopamine+D3+Receptor+Antagonists&doi=10.1021%2Fjm0705612"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm0705612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0705612%26sid%3Dliteratum%253Aachs%26aulast%3DMicheli%26aufirst%3DF.%26aulast%3DBonanomi%26aufirst%3DG.%26aulast%3DBlaney%26aufirst%3DF.%2BE.%26aulast%3DBraggio%26aufirst%3DS.%26aulast%3DCapelli%26aufirst%3DA.%2BM.%26aulast%3DChecchia%26aufirst%3DA.%26aulast%3DCurcuruto%26aufirst%3DO.%26aulast%3DDamiani%26aufirst%3DF.%26aulast%3DDi%2BFabio%26aufirst%3DR.%26aulast%3DDonati%26aufirst%3DD.%26aulast%3DGentile%26aufirst%3DG.%26aulast%3DGribble%26aufirst%3DA.%26aulast%3DHamprecht%26aufirst%3DD.%26aulast%3DTedesco%26aufirst%3DG.%26aulast%3DTerreni%26aufirst%3DS.%26aulast%3DTarsi%26aufirst%3DL.%26aulast%3DLightfoot%26aufirst%3DA.%26aulast%3DStemp%26aufirst%3DG.%26aulast%3DMacDonald%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DA.%26aulast%3DPecoraro%26aufirst%3DM.%26aulast%3DPetrone%26aufirst%3DM.%26aulast%3DPerini%26aufirst%3DO.%26aulast%3DPiner%26aufirst%3DJ.%26aulast%3DRossi%26aufirst%3DT.%26aulast%3DWorby%26aufirst%3DA.%26aulast%3DPilla%26aufirst%3DM.%26aulast%3DValerio%26aufirst%3DE.%26aulast%3DGriffante%26aufirst%3DC.%26aulast%3DMugnaini%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DM.%26aulast%3DScott%26aufirst%3DC.%26aulast%3DAndreoli%26aufirst%3DM.%26aulast%3DLacroix%26aufirst%3DL.%26aulast%3DSchwarz%26aufirst%3DA.%26aulast%3DGozzi%26aufirst%3DA.%26aulast%3DBifone%26aufirst%3DA.%26aulast%3DAshby%26aufirst%3DC.%2BR.%26aulast%3DHagan%26aufirst%3DJ.%2BJ.%26aulast%3DHeidbreder%26aufirst%3DC.%26atitle%3D1%252C2%252C4-Triazol-3-yl-thiopropyl-tetrahydrobenzazepines%253A%2520A%2520Series%2520of%2520Potent%2520and%2520Selective%2520Dopamine%2520D3%2520Receptor%2520Antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D5076%26epage%3D5089%26doi%3D10.1021%2Fjm0705612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Micheli, F.</span><span> </span><span class="NLM_article-title">Recent Advances in the Development of Dopamine D3 Receptor Antagonists: a Medicinal Chemistry Perspective</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1152</span><span class="NLM_x">–</span> <span class="NLM_lpage">1162</span><span class="refDoi"> DOI: 10.1002/cmdc.201000538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1002%2Fcmdc.201000538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=21425240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotVCntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=1152-1162&author=F.+Micheli&title=Recent+Advances+in+the+Development+of+Dopamine+D3+Receptor+Antagonists%3A+a+Medicinal+Chemistry+Perspective&doi=10.1002%2Fcmdc.201000538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Advances in the Development of Dopamine D3 Receptor Antagonists: a Medicinal Chemistry Perspective</span></div><div class="casAuthors">Micheli, Fabrizio</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1152-1162</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Dopamine (DA) D3 receptor antagonism might play a significant role in different therapeutic areas.  A high no. of preclin. studies on DA D3 receptor antagonists have shown efficacy in animal models of Parkinson's disease, schizophrenia and drug dependence.  This Review covers the activities of medicinal chemists in this field over the last ten years towards the identification of truly selective compds.  Both primary and patent literature is reviewed here.  Since the original discoveries, a clear trend towards the optimization of the developability properties of the new scaffold has clearly emerged with time, from both academic and industrial researchers.  Examples of advanced leads from academia and industry are described.  The latest potential therapeutic applications are reported too.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofhYCeq4b9EbVg90H21EOLACvtfcHk0lhpSxdpEPSS1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotVCntbk%253D&md5=61c2b60b5e69df3517421e922b7aa73a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201000538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201000538%26sid%3Dliteratum%253Aachs%26aulast%3DMicheli%26aufirst%3DF.%26atitle%3DRecent%2520Advances%2520in%2520the%2520Development%2520of%2520Dopamine%2520D3%2520Receptor%2520Antagonists%253A%2520a%2520Medicinal%2520Chemistry%2520Perspective%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D1152%26epage%3D1162%26doi%3D10.1002%2Fcmdc.201000538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Micheli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cremonesi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semeraro, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarsi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomelleri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavanni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliosi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perdonà, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sava, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zonzini, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feriani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braggio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidbreder, C.</span><span> </span><span class="NLM_article-title">Novel Morpholine Scaffolds as Selective Dopamine (DA) D3 Receptor Antagonists</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1329</span><span class="NLM_x">–</span> <span class="NLM_lpage">1332</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2015.12.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1016%2Fj.bmcl.2015.12.081" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=1329-1332&author=F.+Micheliauthor=S.+Cremonesiauthor=T.+Semeraroauthor=L.+Tarsiauthor=S.+Tomelleriauthor=P.+Cavanniauthor=B.+Oliosiauthor=E.+Perdon%C3%A0author=A.+Savaauthor=L.+Zonziniauthor=A.+Ferianiauthor=S.+Braggioauthor=C.+Heidbreder&title=Novel+Morpholine+Scaffolds+as+Selective+Dopamine+%28DA%29+D3+Receptor+Antagonists&doi=10.1016%2Fj.bmcl.2015.12.081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.12.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.12.081%26sid%3Dliteratum%253Aachs%26aulast%3DMicheli%26aufirst%3DF.%26aulast%3DCremonesi%26aufirst%3DS.%26aulast%3DSemeraro%26aufirst%3DT.%26aulast%3DTarsi%26aufirst%3DL.%26aulast%3DTomelleri%26aufirst%3DS.%26aulast%3DCavanni%26aufirst%3DP.%26aulast%3DOliosi%26aufirst%3DB.%26aulast%3DPerdon%25C3%25A0%26aufirst%3DE.%26aulast%3DSava%26aufirst%3DA.%26aulast%3DZonzini%26aufirst%3DL.%26aulast%3DFeriani%26aufirst%3DA.%26aulast%3DBraggio%26aufirst%3DS.%26aulast%3DHeidbreder%26aufirst%3DC.%26atitle%3DNovel%2520Morpholine%2520Scaffolds%2520as%2520Selective%2520Dopamine%2520%2528DA%2529%2520D3%2520Receptor%2520Antagonists%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D1329%26epage%3D1332%26doi%3D10.1016%2Fj.bmcl.2015.12.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Micheli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernardelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bianchi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braggio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castelletti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavallini, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavanni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cremonesi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dal Cin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feriani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliosi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semeraro, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarsi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomelleri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Visentini, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zonzini, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidbreder, C.</span><span> </span><span class="NLM_article-title">1,2,4-Triazolyl octahydropyrrolo[2,3-b]pyrroles: A New Series of Potent and Selective Dopamine D3 Receptor Antagonists</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">1619</span><span class="NLM_x">–</span> <span class="NLM_lpage">1636</span><span class="refDoi"> DOI: 10.1016/j.bmc.2016.02.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1016%2Fj.bmc.2016.02.031" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=1619-1636&author=F.+Micheliauthor=A.+Bernardelliauthor=F.+Bianchiauthor=S.+Braggioauthor=L.+Castellettiauthor=P.+Cavalliniauthor=P.+Cavanniauthor=S.+Cremonesiauthor=M.+Dal+Cinauthor=A.+Ferianiauthor=B.+Oliosiauthor=T.+Semeraroauthor=L.+Tarsiauthor=S.+Tomelleriauthor=A.+Wongauthor=F.+Visentiniauthor=L.+Zonziniauthor=C.+Heidbreder&title=1%2C2%2C4-Triazolyl+octahydropyrrolo%5B2%2C3-b%5Dpyrroles%3A+A+New+Series+of+Potent+and+Selective+Dopamine+D3+Receptor+Antagonists&doi=10.1016%2Fj.bmc.2016.02.031"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.02.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.02.031%26sid%3Dliteratum%253Aachs%26aulast%3DMicheli%26aufirst%3DF.%26aulast%3DBernardelli%26aufirst%3DA.%26aulast%3DBianchi%26aufirst%3DF.%26aulast%3DBraggio%26aufirst%3DS.%26aulast%3DCastelletti%26aufirst%3DL.%26aulast%3DCavallini%26aufirst%3DP.%26aulast%3DCavanni%26aufirst%3DP.%26aulast%3DCremonesi%26aufirst%3DS.%26aulast%3DDal%2BCin%26aufirst%3DM.%26aulast%3DFeriani%26aufirst%3DA.%26aulast%3DOliosi%26aufirst%3DB.%26aulast%3DSemeraro%26aufirst%3DT.%26aulast%3DTarsi%26aufirst%3DL.%26aulast%3DTomelleri%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DA.%26aulast%3DVisentini%26aufirst%3DF.%26aulast%3DZonzini%26aufirst%3DL.%26aulast%3DHeidbreder%26aufirst%3DC.%26atitle%3D1%252C2%252C4-Triazolyl%2520octahydropyrrolo%255B2%252C3-b%255Dpyrroles%253A%2520A%2520New%2520Series%2520of%2520Potent%2520and%2520Selective%2520Dopamine%2520D3%2520Receptor%2520Antagonists%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D1619%26epage%3D1636%26doi%3D10.1016%2Fj.bmc.2016.02.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Stemp, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashmeade, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branch, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hadley, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nash, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thewlis, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vong, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, N. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avenell, K. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyfield, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagan, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Middlemiss, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reavill, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riley, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Routledge, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, M.</span><span> </span><span class="NLM_article-title">Design and Synthesis of Trans-N-[4-[2-(6-cyano-1,2,3, 4-tetrahydroisoquinolin-2- yl)ethyl]cyclohexyl]-4-quinolinecarboxamide (SB-277011): a Potent and Selective Dopamine D(3) Receptor Antagonist with High Oral Bioavailability and CNS Penetration in the Rat</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1878</span><span class="NLM_x">–</span> <span class="NLM_lpage">1885</span><span class="refDoi"> DOI: 10.1021/jm000090i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm000090i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=1878-1885&author=G.+Stempauthor=T.+Ashmeadeauthor=C.+L.+Branchauthor=M.+S.+Hadleyauthor=A.+J.+Hunterauthor=C.+N.+Johnsonauthor=D.+J.+Nashauthor=K.+M.+Thewlisauthor=A.+K.+Vongauthor=N.+E.+Austinauthor=P.+Jeffreyauthor=K.+Y.+Avenellauthor=I.+Boyfieldauthor=J.+J.+Haganauthor=D.+N.+Middlemissauthor=C.+Reavillauthor=G.+J.+Rileyauthor=C.+Routledgeauthor=M.+Wood&title=Design+and+Synthesis+of+Trans-N-%5B4-%5B2-%286-cyano-1%2C2%2C3%2C+4-tetrahydroisoquinolin-2-+yl%29ethyl%5Dcyclohexyl%5D-4-quinolinecarboxamide+%28SB-277011%29%3A+a+Potent+and+Selective+Dopamine+D%283%29+Receptor+Antagonist+with+High+Oral+Bioavailability+and+CNS+Penetration+in+the+Rat&doi=10.1021%2Fjm000090i"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm000090i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000090i%26sid%3Dliteratum%253Aachs%26aulast%3DStemp%26aufirst%3DG.%26aulast%3DAshmeade%26aufirst%3DT.%26aulast%3DBranch%26aufirst%3DC.%2BL.%26aulast%3DHadley%26aufirst%3DM.%2BS.%26aulast%3DHunter%26aufirst%3DA.%2BJ.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DNash%26aufirst%3DD.%2BJ.%26aulast%3DThewlis%26aufirst%3DK.%2BM.%26aulast%3DVong%26aufirst%3DA.%2BK.%26aulast%3DAustin%26aufirst%3DN.%2BE.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DAvenell%26aufirst%3DK.%2BY.%26aulast%3DBoyfield%26aufirst%3DI.%26aulast%3DHagan%26aufirst%3DJ.%2BJ.%26aulast%3DMiddlemiss%26aufirst%3DD.%2BN.%26aulast%3DReavill%26aufirst%3DC.%26aulast%3DRiley%26aufirst%3DG.%2BJ.%26aulast%3DRoutledge%26aufirst%3DC.%26aulast%3DWood%26aufirst%3DM.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Trans-N-%255B4-%255B2-%25286-cyano-1%252C2%252C3%252C%25204-tetrahydroisoquinolin-2-%2520yl%2529ethyl%255Dcyclohexyl%255D-4-quinolinecarboxamide%2520%2528SB-277011%2529%253A%2520a%2520Potent%2520and%2520Selective%2520Dopamine%2520D%25283%2529%2520Receptor%2520Antagonist%2520with%2520High%2520Oral%2520Bioavailability%2520and%2520CNS%2520Penetration%2520in%2520the%2520Rat%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D1878%26epage%3D1885%26doi%3D10.1021%2Fjm000090i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Mugnaini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iavarone, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavallini, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffante, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliosi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savoia, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaver, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabiner, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Micheli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidbreder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andorn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merlo Pich, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bani, M.</span><span> </span><span class="NLM_article-title">Occupancy of Brain Dopamine D3 Receptors and Drug Craving: a Translational Approach</span> <span class="citation_source-journal">Neuropsychopharmacology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">302</span><span class="NLM_x">–</span> <span class="NLM_lpage">312</span><span class="refDoi"> DOI: 10.1038/npp.2012.171</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1038%2Fnpp.2012.171" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2013&pages=302-312&author=M.+Mugnainiauthor=L.+Iavaroneauthor=P.+Cavalliniauthor=C.+Griffanteauthor=B.+Oliosiauthor=C.+Savoiaauthor=J.+Beaverauthor=E.+A.+Rabinerauthor=F.+Micheliauthor=C.+Heidbrederauthor=A.+Andornauthor=E.+Merlo+Pichauthor=M.+Bani&title=Occupancy+of+Brain+Dopamine+D3+Receptors+and+Drug+Craving%3A+a+Translational+Approach&doi=10.1038%2Fnpp.2012.171"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnpp.2012.171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnpp.2012.171%26sid%3Dliteratum%253Aachs%26aulast%3DMugnaini%26aufirst%3DM.%26aulast%3DIavarone%26aufirst%3DL.%26aulast%3DCavallini%26aufirst%3DP.%26aulast%3DGriffante%26aufirst%3DC.%26aulast%3DOliosi%26aufirst%3DB.%26aulast%3DSavoia%26aufirst%3DC.%26aulast%3DBeaver%26aufirst%3DJ.%26aulast%3DRabiner%26aufirst%3DE.%2BA.%26aulast%3DMicheli%26aufirst%3DF.%26aulast%3DHeidbreder%26aufirst%3DC.%26aulast%3DAndorn%26aufirst%3DA.%26aulast%3DMerlo%2BPich%26aufirst%3DE.%26aulast%3DBani%26aufirst%3DM.%26atitle%3DOccupancy%2520of%2520Brain%2520Dopamine%2520D3%2520Receptors%2520and%2520Drug%2520Craving%253A%2520a%2520Translational%2520Approach%26jtitle%3DNeuropsychopharmacology%26date%3D2013%26volume%3D38%26spage%3D302%26epage%3D312%26doi%3D10.1038%2Fnpp.2012.171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Grundt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McElveen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luedtke, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">Novel Heterocyclic Trans Olefin Analogues of N-{4-[4-(2,3-dichlorophenyl)piperazin-1-yl]butyl}- arylcarboxamides as Selective Probes with High Affinity for the Dopamine D3 Receptor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">839</span><span class="NLM_x">–</span> <span class="NLM_lpage">848</span><span class="refDoi"> DOI: 10.1021/jm049465g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049465g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=839-848&author=P.+Grundtauthor=E.+E.+Carlsonauthor=J.+Caoauthor=C.+J.+Bennettauthor=E.+McElveenauthor=M.+Taylorauthor=R.+R.+Luedtkeauthor=A.+H.+Newman&title=Novel+Heterocyclic+Trans+Olefin+Analogues+of+N-%7B4-%5B4-%282%2C3-dichlorophenyl%29piperazin-1-yl%5Dbutyl%7D-+arylcarboxamides+as+Selective+Probes+with+High+Affinity+for+the+Dopamine+D3+Receptor&doi=10.1021%2Fjm049465g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12a&amp;dbid=16384&amp;doi=10.1021%2Fjm049465g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049465g%26sid%3Dliteratum%253Aachs%26aulast%3DGrundt%26aufirst%3DP.%26aulast%3DCarlson%26aufirst%3DE.%2BE.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DC.%2BJ.%26aulast%3DMcElveen%26aufirst%3DE.%26aulast%3DTaylor%26aufirst%3DM.%26aulast%3DLuedtke%26aufirst%3DR.%2BR.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DNovel%2520Heterocyclic%2520Trans%2520Olefin%2520Analogues%2520of%2520N-%257B4-%255B4-%25282%252C3-dichlorophenyl%2529piperazin-1-yl%255Dbutyl%257D-%2520arylcarboxamides%2520as%2520Selective%2520Probes%2520with%2520High%2520Affinity%2520for%2520the%2520Dopamine%2520D3%2520Receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D839%26epage%3D848%26doi%3D10.1021%2Fjm049465g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Newman, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grundt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cyriac, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deschamps, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luedtke, R. R.</span><span> </span><span class="NLM_article-title"><i>N</i>-(4-(4-(2,3-Dichloro- or 2-methoxyphenyl)piperazin-1-yl)butyl)heterobiarylcarboxamides with Functionalized Linking Chains as High Affinity and Enantioselective D3 Receptor Antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2559</span><span class="NLM_x">–</span> <span class="NLM_lpage">2570</span><span class="refDoi"> DOI: 10.1021/jm900095y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900095y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktFGrt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2559-2570&author=A.+H.+Newmanauthor=P.+Grundtauthor=G.+Cyriacauthor=J.+R.+Deschampsauthor=M.+Taylorauthor=R.+Kumarauthor=D.+Hoauthor=R.+R.+Luedtke&title=N-%284-%284-%282%2C3-Dichloro-+or+2-methoxyphenyl%29piperazin-1-yl%29butyl%29heterobiarylcarboxamides+with+Functionalized+Linking+Chains+as+High+Affinity+and+Enantioselective+D3+Receptor+Antagonists&doi=10.1021%2Fjm900095y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12bR"><div class="casContent"><span class="casTitleNuber">12b</span><div class="casTitle"><span class="NLM_cas:atitle">N-(4-(4-(2,3-Dichloro- or 2-methoxyphenyl)piperazin-1-yl)butyl)heterobiarylcarboxamides with Functionalized Linking Chains as High Affinity and Enantioselective D3 Receptor Antagonists</span></div><div class="casAuthors">Newman, Amy Hauck; Grundt, Peter; Cyriac, George; Deschamps, Jeffrey R.; Taylor, Michelle; Kumar, Rakesh; Ho, David; Luedtke, Robert R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2559-2570</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the present report, the D3 receptor pharmacophore is modified in the 2,3-diCl- and 2-OCH3-phenylpiperazine class of compds. with the goal to improve D3 receptor affinity and selectivity.  This extension of structure-activity relationships (SAR) has resulted in the identification of the first enantioselective D3 antagonists (R- and S-22) to be reported, wherein enantioselectivity is more pronounced at D3 than at D2, and that a binding region on the second extracellular loop (E2) may play a role in both enantioselectivity and D3 receptor selectivity.  Moreover, we have discovered some of the most D3-selective compds. reported to date that show high affinity (Ki = 1 nM) for D3 and ∼400-fold selectivity over the D2 receptor subtype.  Several of these analogs showed exquisite selectivity for D3 receptors over >60 other receptors, further underscoring their value as in vivo research tools.  These lead compds. also have appropriate phys. characteristics for in vivo exploration and therefore will be useful in detg. how intrinsic activity at D3 receptors tested in vitro is related to behaviors in animal models of addiction and other neuropsychiatric disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprbxvbF2dNX7Vg90H21EOLACvtfcHk0lhyoK4ai-abPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktFGrt7s%253D&md5=665731381e379cec45ef058cd6c45d0e</span></div><a href="/servlet/linkout?suffix=cit12b&amp;dbid=16384&amp;doi=10.1021%2Fjm900095y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900095y%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DGrundt%26aufirst%3DP.%26aulast%3DCyriac%26aufirst%3DG.%26aulast%3DDeschamps%26aufirst%3DJ.%2BR.%26aulast%3DTaylor%26aufirst%3DM.%26aulast%3DKumar%26aufirst%3DR.%26aulast%3DHo%26aufirst%3DD.%26aulast%3DLuedtke%26aufirst%3DR.%2BR.%26atitle%3DN-%25284-%25284-%25282%252C3-Dichloro-%2520or%25202-methoxyphenyl%2529piperazin-1-yl%2529butyl%2529heterobiarylcarboxamides%2520with%2520Functionalized%2520Linking%2520Chains%2520as%2520High%2520Affinity%2520and%2520Enantioselective%2520D3%2520Receptor%2520Antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D2559%26epage%3D2570%26doi%3D10.1021%2Fjm900095y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levant, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keck, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Tranylcypromine Substituted cis-Hydroxycyclobutylnaphthamides as Potent and Selective Dopamine D3 Receptor Antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">4962</span><span class="NLM_x">–</span> <span class="NLM_lpage">4968</span><span class="refDoi"> DOI: 10.1021/jm401798r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401798r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC2cXosVOnurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4962-4968&author=J.+Chenauthor=B.+Levantauthor=C.+Jiangauthor=T.+M.+Keckauthor=A.+H.+Newmanauthor=S.+Wang&title=Tranylcypromine+Substituted+cis-Hydroxycyclobutylnaphthamides+as+Potent+and+Selective+Dopamine+D3+Receptor+Antagonists&doi=10.1021%2Fjm401798r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Tranylcypromine Substituted cis-Hydroxycyclobutylnaphthamides as Potent and Selective Dopamine D3 Receptor Antagonists</span></div><div class="casAuthors">Chen, Jianyong; Levant, Beth; Jiang, Cheng; Keck, Thomas M.; Newman, Amy Hauck; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4962-4968</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report a class of potent and selective dopamine D3 receptor antagonists based upon tranylcypromine.  Although tranylcypromine has a low affinity for the rat D3 receptor (Ki = 12.8 μM), our efforts have yielded (1R,2S)-11 (CJ-1882), which has Ki values of 2.7 and 2.8 nM at the rat and human dopamine D3 receptors, resp., and displays resp. selectivities of >10000-fold and 223-fold over the rat and human D2 receptors.  Evaluation in a β-arrestin functional assay showed that (1R,2S)-11 is a potent and competitive antagonist at the human D3 receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowWudWdAfWgLVg90H21EOLACvtfcHk0lhyoK4ai-abPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXosVOnurg%253D&md5=e2ddf0711d00103be281cb043b19ec3a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm401798r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401798r%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLevant%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DC.%26aulast%3DKeck%26aufirst%3DT.%2BM.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DTranylcypromine%2520Substituted%2520cis-Hydroxycyclobutylnaphthamides%2520as%2520Potent%2520and%2520Selective%2520Dopamine%2520D3%2520Receptor%2520Antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D4962%26epage%3D4968%26doi%3D10.1021%2Fjm401798r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Hu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span> </span><span class="NLM_article-title">The Dopamine D3 Receptor Antagonist YQA14 that Inhibits the Expression and Drug-primed Reactivation of Morphine-induced Conditioned Place Preference in Rats</span> <span class="citation_source-journal">Eur. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">720</span><span class="NLM_x">, </span> <span class="NLM_fpage">212</span><span class="NLM_x">–</span> <span class="NLM_lpage">217</span><span class="refDoi"> DOI: 10.1016/j.ejphar.2013.10.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1016%2Fj.ejphar.2013.10.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=24436985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslKnurnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=720&publication_year=2013&pages=212-217&author=R.+Huauthor=R.+Songauthor=R.+Yangauthor=R.+Suauthor=J.+Li&title=The+Dopamine+D3+Receptor+Antagonist+YQA14+that+Inhibits+the+Expression+and+Drug-primed+Reactivation+of+Morphine-induced+Conditioned+Place+Preference+in+Rats&doi=10.1016%2Fj.ejphar.2013.10.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The dopamine D3 receptor antagonist YQA14 that inhibits the expression and drug-primed reactivation of morphine-induced conditioned place preference in rats</span></div><div class="casAuthors">Hu, Rongrong; Song, Rui; Yang, Rifang; Su, Ruibin; Li, Jin</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">720</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">212-217</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Increasing evidence suggests that the mesolimbic dopamine system plays a crit. role in opioid addiction.  However, there is currently no std. drug treatment for opioid addiction.  Growing preclin. evidence indicates that the dopamine D3 receptor antagonists are the potential anti-addiction pharmacotherapeutic agents based on in animal models of multiple drug addiction.  In this study, we investigated the inhibitory effects of YQA14, a novel dopamine D3 receptor antagonist with a high affinity and selectivity for dopamine D3 receptor, using morphine-induced conditioned place preference (CPP) in rats.  The results suggested that YQA14 (6.25-25 mg/kg; i.p., i.p.) decreased the expression of morphine (10 mg/kg, s.c.)-induced CPP in a dose-related manner but did not influence the acquisition of morphine-induced CPP.  At a 25 mg/kg dose of YQA14, it also notably inhibited the reactivation of morphine-priming CPP.  These findings suggest that YQA14 is a potential agent for anti-opioid addiction which warrants further study and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTXMIGCwk0X7Vg90H21EOLACvtfcHk0liLyq_vdD0oKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslKnurnP&md5=05d05b2def79779c975b853df1ee6365</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.ejphar.2013.10.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejphar.2013.10.026%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DR.%26aulast%3DSong%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DSu%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DThe%2520Dopamine%2520D3%2520Receptor%2520Antagonist%2520YQA14%2520that%2520Inhibits%2520the%2520Expression%2520and%2520Drug-primed%2520Reactivation%2520of%2520Morphine-induced%2520Conditioned%2520Place%2520Preference%2520in%2520Rats%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D2013%26volume%3D720%26spage%3D212%26epage%3D217%26doi%3D10.1016%2Fj.ejphar.2013.10.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Hackling, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, H.</span><span> </span><span class="NLM_article-title">Dopamine D3 Receptor Ligands with Antagonist Properties</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">946</span><span class="NLM_x">–</span> <span class="NLM_lpage">961</span><span class="refDoi"> DOI: 10.1002/1439-7633(20021004)3:10<946::AID-CBIC946>3.0.CO;2-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1002%2F1439-7633%2820021004%293%3A10%3C946%3A%3AAID-CBIC946%3E3.0.CO%3B2-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=12362359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BD38XnvFGgu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=946-961&author=A.+E.+Hacklingauthor=H.+Stark&title=Dopamine+D3+Receptor+Ligands+with+Antagonist+Properties&doi=10.1002%2F1439-7633%2820021004%293%3A10%3C946%3A%3AAID-CBIC946%3E3.0.CO%3B2-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Dopamine D3 receptor ligands with antagonist properties</span></div><div class="casAuthors">Hackling, Anneke E.; Stark, Holger</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">946-961</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The dopamine D3 receptor was recognized to play an important role in the mol. mechanisms of various neuropsychiatric disorders.  The development of new dopamine D3 receptor selective antagonists is premised on the potentially improved therapeutic treatment of psychosis like schizophrenia.  Partial agonists at dopamine D3 receptors are supposed to be beneficial when administered to drug abusers or in Parkinson's disease.  The structural basis for most compds. is at least a basic, aryl-substituted alkanamine part with an alkyl moiety, which in many compds. forms a spacer to another aryl residue.  Structural variety among the amine moiety includes aminotetralins, tetrahydroisoquinolines, isoindoles, benzazepines, and amino-indans, as well as pyrrolidines, pyrroles, and 4-phenylpiperazines.  Various ways for lead optimization are shown in different classes of compds.  Promising ligands with high D3 receptor affinity often lack sufficient selectivity or display deficits in the required in vivo parameters.  Structure-activity relationships for dopamine D3 receptor antagonists and partial agonists are discussed here, along with the outlook for their potential therapeutic application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7h56q6SPZN7Vg90H21EOLACvtfcHk0liLyq_vdD0oKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnvFGgu7c%253D&md5=ba9cd2302ced89619ae5d1d82e7d08e4</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2F1439-7633%2820021004%293%3A10%3C946%3A%3AAID-CBIC946%3E3.0.CO%3B2-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1439-7633%252820021004%25293%253A10%253C946%253A%253AAID-CBIC946%253E3.0.CO%253B2-5%26sid%3Dliteratum%253Aachs%26aulast%3DHackling%26aufirst%3DA.%2BE.%26aulast%3DStark%26aufirst%3DH.%26atitle%3DDopamine%2520D3%2520Receptor%2520Ligands%2520with%2520Antagonist%2520Properties%26jtitle%3DChemBioChem%26date%3D2002%26volume%3D3%26spage%3D946%26epage%3D961%26doi%3D10.1002%2F1439-7633%2820021004%293%3A10%3C946%3A%3AAID-CBIC946%3E3.0.CO%3B2-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Newman, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beuming, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banala, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donthamsetti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pongetti, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaBounty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luedtke, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Javitch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span> </span><span class="NLM_article-title">Molecular Determinants of Selectivity and Efficacy at the Dopamine D3 Receptor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">6689</span><span class="NLM_x">–</span> <span class="NLM_lpage">6699</span><span class="refDoi"> DOI: 10.1021/jm300482h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300482h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsFCkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6689-6699&author=A.+H.+Newmanauthor=T.+Beumingauthor=A.+K.+Banalaauthor=P.+Donthamsettiauthor=K.+Pongettiauthor=A.+LaBountyauthor=B.+Levyauthor=J.+Caoauthor=M.+Michinoauthor=R.+R.+Luedtkeauthor=A.+Javitchauthor=L.+Shi&title=Molecular+Determinants+of+Selectivity+and+Efficacy+at+the+Dopamine+D3+Receptor&doi=10.1021%2Fjm300482h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Determinants of Selectivity and Efficacy at the Dopamine D3 Receptor</span></div><div class="casAuthors">Newman, Amy Hauck; Beuming, Thijs; Banala, Ashwini K.; Donthamsetti, Prashant; Pongetti, Katherine; LaBounty, Alex; Levy, Benjamin; Cao, Jianjing; Michino, Mayako; Luedtke, Robert R.; Javitch, Jonathan A.; Shi, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6689-6699</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dopamine D3 receptor (D3R) has been implicated in substance abuse and other neuropsychiatric disorders.  The high sequence homol. between the D3R and D2R, esp. within the orthosteric binding site (OBS) that binds dopamine, has made the development of D3R-selective compds. challenging.  Here, we deconstruct into pharmacophoric elements a series of D3R-selective substituted-4-phenylpiperazine compds. and use computational simulations and binding and activation studies to dissect the structural bases for D3R selectivity and efficacy.  We find that selectivity arises from divergent interactions within a second binding pocket (SBP) sep. from the OBS, whereas efficacy depends on the binding mode in the OBS.  Our findings reveal structural features of the receptor that are crit. to selectivity and efficacy that can be used to design highly D3R-selective ligands with targeted efficacies.  These findings are generalizable to other GPCRs in which the SBP can be targeted by bitopic or allosteric ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmBJU7uyvVg7Vg90H21EOLACvtfcHk0ljNpoENrcSioA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsFCkt7c%253D&md5=033dd49282723650828a3ec0a31211e5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm300482h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300482h%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DBanala%26aufirst%3DA.%2BK.%26aulast%3DDonthamsetti%26aufirst%3DP.%26aulast%3DPongetti%26aufirst%3DK.%26aulast%3DLaBounty%26aufirst%3DA.%26aulast%3DLevy%26aufirst%3DB.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DMichino%26aufirst%3DM.%26aulast%3DLuedtke%26aufirst%3DR.%2BR.%26aulast%3DJavitch%26aufirst%3DA.%26aulast%3DShi%26aufirst%3DL.%26atitle%3DMolecular%2520Determinants%2520of%2520Selectivity%2520and%2520Efficacy%2520at%2520the%2520Dopamine%2520D3%2520Receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6689%26epage%3D6699%26doi%3D10.1021%2Fjm300482h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Hughes, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalindjian, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, K. L.</span><span> </span><span class="NLM_article-title">Principles of Early Drug Discovery</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">162</span><span class="NLM_x">, </span> <span class="NLM_fpage">1239</span><span class="NLM_x">–</span> <span class="NLM_lpage">1249</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.2010.01127.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1111%2Fj.1476-5381.2010.01127.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=21091654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivF2lt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2011&pages=1239-1249&author=J.+P.+Hughesauthor=S.+Reesauthor=S.+B.+Kalindjianauthor=K.+L.+Philpott&title=Principles+of+Early+Drug+Discovery&doi=10.1111%2Fj.1476-5381.2010.01127.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17aR"><div class="casContent"><span class="casTitleNuber">17a</span><div class="casTitle"><span class="NLM_cas:atitle">Principles of early drug discovery</span></div><div class="casAuthors">Hughes, J. P.; Rees, S.; Kalindjian, S. B.; Philpott, K. L.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1239-1249</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Developing a new drug from original idea to the launch of a finished product is a complex process which can take 12-15 years and cost in excess of $1 billion.  The idea for a target can come from a variety of sources including academic and clin. research and from the com. sector.  It may take many years to build up a body of supporting evidence before selecting a target for a costly drug discovery program.  Once a target was chosen, the pharmaceutical industry and more recently some academic centers have streamlined a no. of early processes to identify mols. which possess suitable characteristics to make acceptable drugs.  This review will look at key preclin. stages of the drug discovery process, from initial target identification and validation, through assay development, high throughput screening, hit identification, lead optimization and finally the selection of a candidate mol. for clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooCc74_F_90rVg90H21EOLACvtfcHk0ljNpoENrcSioA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivF2lt7o%253D&md5=05a0016f45b155f9fb5822f5c7bcc35d</span></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2010.01127.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2010.01127.x%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DJ.%2BP.%26aulast%3DRees%26aufirst%3DS.%26aulast%3DKalindjian%26aufirst%3DS.%2BB.%26aulast%3DPhilpott%26aufirst%3DK.%2BL.%26atitle%3DPrinciples%2520of%2520Early%2520Drug%2520Discovery%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2011%26volume%3D162%26spage%3D1239%26epage%3D1249%26doi%3D10.1111%2Fj.1476-5381.2010.01127.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit17b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Saxena, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panicucci, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joshi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garad, S.</span><span> </span><span class="NLM_article-title">Developability Assessment in Pharmaceutical Industry: An Integrated Group Approach for Selecting Developable Candidates</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">1962</span><span class="NLM_x">–</span> <span class="NLM_lpage">1979</span><span class="refDoi"> DOI: 10.1002/jps.21592</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1002%2Fjps.21592" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2009&pages=1962-1979&author=V.+Saxenaauthor=R.+Panicucciauthor=Y.+Joshiauthor=S.+Garad&title=Developability+Assessment+in+Pharmaceutical+Industry%3A+An+Integrated+Group+Approach+for+Selecting+Developable+Candidates&doi=10.1002%2Fjps.21592"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17b&amp;dbid=16384&amp;doi=10.1002%2Fjps.21592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.21592%26sid%3Dliteratum%253Aachs%26aulast%3DSaxena%26aufirst%3DV.%26aulast%3DPanicucci%26aufirst%3DR.%26aulast%3DJoshi%26aufirst%3DY.%26aulast%3DGarad%26aufirst%3DS.%26atitle%3DDevelopability%2520Assessment%2520in%2520Pharmaceutical%2520Industry%253A%2520An%2520Integrated%2520Group%2520Approach%2520for%2520Selecting%2520Developable%2520Candidates%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2009%26volume%3D98%26spage%3D1962%26epage%3D1979%26doi%3D10.1002%2Fjps.21592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit17c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Han, C.; Wang, B.</span><span> </span><span class="NLM_article-title">Factors that impact the developability of drug candidates: an overview</span>. In  <span class="citation_source-book">Drug Delivery: Principles and Applications</span>; <span class="NLM_contrib-group">Wang, B.; Siahaan, T.; Soltero, R.</span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">; </span>Chapter 1.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1002%2F0471475734.ch1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=C.+Han&author=B.+Wangauthor=B.+Wang&author=T.+Siahaan&author=R.+Soltero&title=Drug+Delivery%3A+Principles+and+Applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17c&amp;dbid=16384&amp;doi=10.1002%2F0471475734.ch1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F0471475734.ch1%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DC.%26atitle%3DFactors%2520that%2520impact%2520the%2520developability%2520of%2520drug%2520candidates%253A%2520an%2520overview%26btitle%3DDrug%2520Delivery%253A%2520Principles%2520and%2520Applications%26aulast%3DWang%26aufirst%3DB.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit17d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Keck, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">John, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czoty, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nader, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">Identifying Medication Targets for Psychostimulant Addiction: Unraveling the Dopamine D3 Receptor Hypothesis</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5361</span><span class="NLM_x">–</span> <span class="NLM_lpage">5380</span><span class="refDoi"> DOI: 10.1021/jm501512b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501512b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsFGgu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5361-5380&author=T.+M.+Keckauthor=W.+S.+Johnauthor=P.+W.+Czotyauthor=M.+A.+Naderauthor=A.+H.+Newman&title=Identifying+Medication+Targets+for+Psychostimulant+Addiction%3A+Unraveling+the+Dopamine+D3+Receptor+Hypothesis&doi=10.1021%2Fjm501512b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17dR"><div class="casContent"><span class="casTitleNuber">17d</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Medication Targets for Psychostimulant Addiction: Unraveling the Dopamine D3 Receptor Hypothesis</span></div><div class="casAuthors">Keck, Thomas M.; John, William S.; Czoty, Paul W.; Nader, Michael A.; Newman, Amy Hauck</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5361-5380</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dopamine D3 receptor (D3R) is a target for developing medications to treat substance use disorders.  D3R-selective compds. with high affinity and varying efficacies have been discovered, providing crit. research tools for cell-based studies that have been translated to in vivo models of drug abuse.  D3R antagonists and partial agonists have shown esp. promising results in rodent models of relapse-like behavior, including stress-, drug-, and cue-induced reinstatement of drug seeking.  However, to date, translation to human studies has been limited.  Herein, we present an overview and illustrate some of the pitfalls and challenges of developing novel D3R-selective compds. toward clin. utility, esp. for treatment of cocaine abuse.  Future research and development of D3R-selective antagonists and partial agonists for substance abuse remains critically important but will also require further evaluation and development of translational animal models to det. the best time in the addiction cycle to target D3Rs for optimal therapeutic efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKVPGvcTvN87Vg90H21EOLACvtfcHk0ljlXjjxtwCPQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsFGgu74%253D&md5=99f531d840ad7f94d0eeadeb6fd48100</span></div><a href="/servlet/linkout?suffix=cit17d&amp;dbid=16384&amp;doi=10.1021%2Fjm501512b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501512b%26sid%3Dliteratum%253Aachs%26aulast%3DKeck%26aufirst%3DT.%2BM.%26aulast%3DJohn%26aufirst%3DW.%2BS.%26aulast%3DCzoty%26aufirst%3DP.%2BW.%26aulast%3DNader%26aufirst%3DM.%2BA.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DIdentifying%2520Medication%2520Targets%2520for%2520Psychostimulant%2520Addiction%253A%2520Unraveling%2520the%2520Dopamine%2520D3%2520Receptor%2520Hypothesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5361%26epage%3D5380%26doi%3D10.1021%2Fjm501512b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit17e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Watson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viggers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheetham, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porter, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, K. D.</span><span> </span><span class="NLM_article-title">Receptor Occupancy and Brain Free Fraction</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">753</span><span class="NLM_x">–</span> <span class="NLM_lpage">760</span><span class="refDoi"> DOI: 10.1124/dmd.108.022814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1124%2Fdmd.108.022814" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=753-760&author=J.+Watsonauthor=S.+Wrightauthor=A.+Lucasauthor=K.+L.+Clarkeauthor=J.+Viggersauthor=S.+Cheethamauthor=P.+Jeffreyauthor=R.+Porterauthor=K.+D.+Read&title=Receptor+Occupancy+and+Brain+Free+Fraction&doi=10.1124%2Fdmd.108.022814"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17e&amp;dbid=16384&amp;doi=10.1124%2Fdmd.108.022814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.108.022814%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DJ.%26aulast%3DWright%26aufirst%3DS.%26aulast%3DLucas%26aufirst%3DA.%26aulast%3DClarke%26aufirst%3DK.%2BL.%26aulast%3DViggers%26aufirst%3DJ.%26aulast%3DCheetham%26aufirst%3DS.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DPorter%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26atitle%3DReceptor%2520Occupancy%2520and%2520Brain%2520Free%2520Fraction%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2009%26volume%3D37%26spage%3D753%26epage%3D760%26doi%3D10.1124%2Fdmd.108.022814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Keck, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burzynski, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">Beyond Small-molecule SAR: Using the Dopamine D3 Receptor Crystal Structure to Guide Drug Design</span> <span class="citation_source-journal">Adv. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">267</span><span class="NLM_x">–</span> <span class="NLM_lpage">300</span><span class="refDoi"> DOI: 10.1016/B978-0-12-420118-7.00007-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1016%2FB978-0-12-420118-7.00007-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=24484980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsFCitb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2014&pages=267-300&author=T.+M.+Keckauthor=C.+Burzynskiauthor=L.+Shiauthor=A.+H.+Newman&title=Beyond+Small-molecule+SAR%3A+Using+the+Dopamine+D3+Receptor+Crystal+Structure+to+Guide+Drug+Design&doi=10.1016%2FB978-0-12-420118-7.00007-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18aR"><div class="casContent"><span class="casTitleNuber">18a</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond small-molecule SAR: using the dopamine D3 receptor crystal structure to guide drug design</span></div><div class="casAuthors">Keck, Thomas M.; Burzynski, Caitlin; Shi, Lei; Newman, Amy Hauck</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Pharmacology (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">Emerging Targets & Therapeutics in the Treatment of Psychostimulant Abuse</span>),
    <span class="NLM_cas:pages">267-300</span>CODEN:
                <span class="NLM_cas:coden">ADPHEL</span>;
        ISSN:<span class="NLM_cas:issn">1054-3589</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The dopamine D3 receptor is a target of pharmacotherapeutic interest in a variety of neurol. disorders including schizophrenia, restless leg syndrome, and drug addiction.  The high protein sequence homol. between the D3 and D2 receptors has posed a challenge to developing D3 receptor-selective ligands whose behavioral actions can be attributed to D3 receptor engagement, in vivo.  However, through primarily small-mol. structure-activity relationship (SAR) studies, a variety of chem. scaffolds have been discovered over the past two decades that have resulted in several D3 receptor-selective ligands with high affinity and in vivo activity.  Nevertheless, viable clin. candidates remain limited.  The recent detn. of the high-resoln. crystal structure of the D3 receptor has invigorated structure-based drug design, providing refinements to the mol. dynamic models and testable predictions about receptor-ligand interactions.  This chapter will highlight recent preclin. and clin. studies demonstrating potential utility of D3 receptor-selective ligands in the treatment of addiction.  In addn., new structure-based rational drug design strategies for D3 receptor-selective ligands that complement traditional small-mol. SAR to improve the selectivity and directed efficacy profiles are examd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbH7VLMIagF7Vg90H21EOLACvtfcHk0ljlXjjxtwCPQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsFCitb8%253D&md5=d266814a068e4528c70e200a860fab64</span></div><a href="/servlet/linkout?suffix=cit18a&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-420118-7.00007-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-420118-7.00007-X%26sid%3Dliteratum%253Aachs%26aulast%3DKeck%26aufirst%3DT.%2BM.%26aulast%3DBurzynski%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DBeyond%2520Small-molecule%2520SAR%253A%2520Using%2520the%2520Dopamine%2520D3%2520Receptor%2520Crystal%2520Structure%2520to%2520Guide%2520Drug%2520Design%26jtitle%3DAdv.%2520Pharmacol.%26date%3D2014%26volume%3D69%26spage%3D267%26epage%3D300%26doi%3D10.1016%2FB978-0-12-420118-7.00007-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit18b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Ligand Discovery from a Dopamine D3 Receptor Homology Model and Crystal Structure</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">769</span><span class="NLM_x">–</span> <span class="NLM_lpage">778</span><span class="refDoi"> DOI: 10.1038/nchembio.662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1038%2Fnchembio.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=21926995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=769-778&author=J.+Carlssonauthor=R.+G.+Colemanauthor=V.+Setolaauthor=J.+J.+Irwinauthor=H.+Fanauthor=A.+Schlessingerauthor=A.+Saliauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Ligand+Discovery+from+a+Dopamine+D3+Receptor+Homology+Model+and+Crystal+Structure&doi=10.1038%2Fnchembio.662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18bR"><div class="casContent"><span class="casTitleNuber">18b</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span></div><div class="casAuthors">Carlsson, Jens; Coleman, Ryan G.; Setola, Vincent; Irwin, John J.; Fan, Hao; Schlessinger, Avner; Sali, Andrej; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">769-778</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are intensely studied as drug targets and for their role in signaling.  With the detn. of the first crystal structures, interest in structure-based ligand discovery increased.  Unfortunately, for most GPCRs no exptl. structures are available.  The detn. of the D3 receptor structure and the challenge to the community to predict it enabled a fully prospective comparison of ligand discovery from a modeled structure vs. that of the subsequently released crystal structure.  Over 3.3 million mols. were docked against a homol. model, and 26 of the highest ranking were tested for binding.  Six had affinities ranging from 0.2 to 3.1 μM.  Subsequently, the crystal structure was released and the docking screen repeated.  Of the 25 compds. selected, five had affinities ranging from 0.3 to 3.0 μM.  One of the new ligands from the homol. model screen was optimized for affinity to 81 nM.  The feasibility of docking screens against modeled GPCRs more generally is considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6gBx3Olbn7Vg90H21EOLACvtfcHk0lhveu1GXceG3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP&md5=7242aa512079a348678b309d352ecdb9</span></div><a href="/servlet/linkout?suffix=cit18b&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.662%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520Discovery%2520from%2520a%2520Dopamine%2520D3%2520Receptor%2520Homology%2520Model%2520and%2520Crystal%2520Structure%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D769%26epage%3D778%26doi%3D10.1038%2Fnchembio.662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit18c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Vass, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agai-Csongor, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span> </span><span class="NLM_article-title">Multiple Fragment Docking and Linking in Primary and Secondary Pockets of Dopamine Receptors</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1010</span><span class="NLM_x">–</span> <span class="NLM_lpage">1014</span><span class="refDoi"> DOI: 10.1021/ml500201u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500201u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFeltbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1010-1014&author=M.+Vassauthor=E.+Agai-Csongorauthor=F.+Hortiauthor=G.+M.+Keseru&title=Multiple+Fragment+Docking+and+Linking+in+Primary+and+Secondary+Pockets+of+Dopamine+Receptors&doi=10.1021%2Fml500201u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18cR"><div class="casContent"><span class="casTitleNuber">18c</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple Fragment Docking and Linking in Primary and Secondary Pockets of Dopamine Receptors</span></div><div class="casAuthors">Vass, Marton; Agai-Csongor, Eva; Horti, Ferenc; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1010-1014</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A sequential docking methodol. was applied to computationally predict starting points for fragment linking using the human dopamine D3 receptor crystal structure and a human dopamine D2 receptor homol. model.  Two focused fragment libraries were docked in the primary and secondary binding sites, and best fragment combinations were enumerated.  Similar top scoring fragments were found for the primary site, while secondary site fragments were predicted to convey selectivity.  Three linked compds. were synthesized that had 9-, 39-, and 55-fold selectivity in favor of D3 and the subtype selectivity of the compds. was assessed on a structural basis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIkfBkGmhbwrVg90H21EOLACvtfcHk0lhveu1GXceG3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFeltbbO&md5=7531b04aabc8b12a4b975f6dc970134d</span></div><a href="/servlet/linkout?suffix=cit18c&amp;dbid=16384&amp;doi=10.1021%2Fml500201u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500201u%26sid%3Dliteratum%253Aachs%26aulast%3DVass%26aufirst%3DM.%26aulast%3DAgai-Csongor%26aufirst%3DE.%26aulast%3DHorti%26aufirst%3DF.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DMultiple%2520Fragment%2520Docking%2520and%2520Linking%2520in%2520Primary%2520and%2520Secondary%2520Pockets%2520of%2520Dopamine%2520Receptors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D1010%26epage%3D1014%26doi%3D10.1021%2Fml500201u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Böhm, H.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flohr, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">Scaffold Hopping</span> <span class="citation_source-journal">Drug Discovery Today: Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">217</span><span class="NLM_x">–</span> <span class="NLM_lpage">224</span><span class="refDoi"> DOI: 10.1016/j.ddtec.2004.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1016%2Fj.ddtec.2004.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=24981488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A280%3ADC%252BC2cfosVaktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=217-224&author=H.-J.+B%C3%B6hmauthor=A.+Flohrauthor=M.+Stahl&title=Scaffold+Hopping&doi=10.1016%2Fj.ddtec.2004.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Scaffold hopping</span></div><div class="casAuthors">Bohm Hans-Joachim; Flohr Alexander; Stahl Martin</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">217-24</span>
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    </div><div class="casAbstract">The aim of scaffold hopping is to discover structurally novel compounds starting from known active compounds by modifying the central core structure of the molecule.  Scaffold hopping is a central task of modern medicinal chemistry requiring a multitude of techniques, which are discussed in this article.  Their application has led to several molecules with chemically completely different core structures, and yet binding to the same receptor.  Computational approaches for scaffold hopping highlight the challenges of the field that are still unsolved.:</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSkmF4guWxLWsKL0FlyGHDofW6udTcc2ebeiu7pi9R-a7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cfosVaktQ%253D%253D&md5=9a90f37dd2accf66bfd379f9ec602e03</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2004.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2004.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6hm%26aufirst%3DH.-J.%26aulast%3DFlohr%26aufirst%3DA.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DScaffold%2520Hopping%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2004%26volume%3D1%26spage%3D217%26epage%3D224%26doi%3D10.1016%2Fj.ddtec.2004.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span>MarvinSketch 6.1.6, (build id: 6.1.6_b695); <span class="NLM_publisher-name">Chemaxon</span>: <span class="NLM_publisher-loc">Budapest</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">; </span>(<a href="http://www.chemaxon.com" class="extLink">www.chemaxon.com</a>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MarvinSketch+6.1.6%2C+%28build+id%3A+6.1.6_b695%29%3B+Chemaxon%3A+Budapest%2C+2014%3B+%28www.chemaxon.com%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMarvinSketch%25206.1.6%252C%2520%2528build%2520id%253A%25206.1.6_b695%2529%26pub%3DChemaxon%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span>Molecular Operating Environment (MOE), 2013.08; <span class="NLM_publisher-name">Chemical Computing Group Inc.</span>: <span class="NLM_publisher-loc">1010 Sherbooke St. West, Suite #910, Montreal, QC, Canada, H3A 2R7</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Molecular+Operating+Environment+%28MOE%29%2C+2013.08%3B+Chemical+Computing+Group+Inc.%3A+1010+Sherbooke+St.+West%2C+Suite+%23910%2C+Montreal%2C+QC%2C+Canada%2C+H3A+2R7%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%252C%25202013.08%26pub%3DChemical%2520Computing%2520Group%2520Inc%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="note"><p class="first last">The research complied with national legislation and with company policy on the Care and Use of Animals and with related codes of practice.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Matsuda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satsukawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takimoto, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morisaki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, S.</span><span> </span><span class="NLM_article-title">Assessment of Intestinal Availability of Various Drugs in the Oral Absorption Process Using Portal Vein-cannulated Rats</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">2231</span><span class="NLM_x">–</span> <span class="NLM_lpage">2238</span><span class="refDoi"> DOI: 10.1124/dmd.112.048223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1124%2Fdmd.112.048223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=22930277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslyjsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=2231-2238&author=Y.+Matsudaauthor=Y.+Konnoauthor=M.+Satsukawaauthor=T.+Kobayashiauthor=Y.+Takimotoauthor=K.+Morisakiauthor=S.+Yamashita&title=Assessment+of+Intestinal+Availability+of+Various+Drugs+in+the+Oral+Absorption+Process+Using+Portal+Vein-cannulated+Rats&doi=10.1124%2Fdmd.112.048223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of intestinal availability of various drugs in the oral absorption process using portal vein-cannulated rats</span></div><div class="casAuthors">Matsuda, Yoshiki; Konno, Yoshihiro; Satsukawa, Masahiro; Kobayashi, Taro; Takimoto, Yu; Morisaki, Kunihiko; Yamashita, Shinji</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2231-2238</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">To understand the rate-limiting process of oral drug absorption, not only total bioavailability (F) but also intestinal (Fa · Fg) and hepatic (Fh) availability after oral administration should be evaluated.  Usually, Fa · Fg of drug is calcd. from pharmacokinetic parameters after i.v. and oral administration.  This approach is influenced markedly by the estd. value of Fh, which varies with the hepatic blood flow used in the calcns.  In this study, portal vein-cannulated rats were used to calc. the Fa · Fg of drugs from a single oral dosing expt. without data from i.v. injection.  Portal vein-cannulated rats were prepd. by a new operative method that enables stable portal vein blood flow.  This surgery had no effects on hepatic blood flow and metabolic activity.  Our method for calcg. Fa · Fg was validated by detg. both portal and systemic plasma concn. profiles of various drugs possessing different pharmacokinetic properties after oral administration to the portal vein-cannulated rats.  Simulation of portal and systemic plasma concns. by physiol. based pharmacokinetic modeling indicated that the balance of the absorption rate const. (ka) and elimination rate const. (ke) resulted in different patterns in portal and systemic plasma concn.-time profiles.  This study is expected to provide a new exptl. animal model that enables identification of the factors that limit oral bioavailability and to provide pharmacokinetic information on the oral absorption process of drugs during drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF2bFgGtdpXrVg90H21EOLACvtfcHk0li3nSQSZ43IMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslyjsrjF&md5=b184c93cf08fb29b8f36388683e7d582</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1124%2Fdmd.112.048223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.112.048223%26sid%3Dliteratum%253Aachs%26aulast%3DMatsuda%26aufirst%3DY.%26aulast%3DKonno%26aufirst%3DY.%26aulast%3DSatsukawa%26aufirst%3DM.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DTakimoto%26aufirst%3DY.%26aulast%3DMorisaki%26aufirst%3DK.%26aulast%3DYamashita%26aufirst%3DS.%26atitle%3DAssessment%2520of%2520Intestinal%2520Availability%2520of%2520Various%2520Drugs%2520in%2520the%2520Oral%2520Absorption%2520Process%2520Using%2520Portal%2520Vein-cannulated%2520Rats%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D40%26spage%3D2231%26epage%3D2238%26doi%3D10.1124%2Fdmd.112.048223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">van Breemen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, B.</span><span> </span><span class="NLM_article-title">Cado-2 cell permeability assays to measure drug absorption</span> <span class="citation_source-journal">Expert Opin. Drug Metab. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">185</span><span class="refDoi"> DOI: 10.1517/17425255.1.2.175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1517%2F17425255.1.2.175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=16922635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKltLbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=175-185&author=R.+van+Breemenauthor=B.+Li&title=Cado-2+cell+permeability+assays+to+measure+drug+absorption&doi=10.1517%2F17425255.1.2.175"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Caco-2 cell permeability assays to measure drug absorption</span></div><div class="casAuthors">van Breemen, Richard B.; Li, Yongmei</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">175-185</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Ashley Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Caco-2 cells are a human colon epithelial cancer cell line used as a model of human intestinal absorption of drugs and other compds.  When cultured as a monolayer, Caco-2 cells differentiate to form tight junctions between cells to serve as a model of paracellular movement of compds. across the mono-layer.  In addn., Caco-2 cells express transporter proteins, efflux proteins, and Phase II conjugation enzymes to model a variety of transcellular pathways as well as metabolic transformation of test substances.  In many respects, the Caco-2 cell monolayer mimics the human intestinal epithelium.  One of the functional differences between normal cells and Caco-2 cells is the lack of expression of the cytochrome P 450 isoenzymes and in particular, CYP3A4, which is normally expressed at high levels in the intestine.  However, Caco-2 cells may be induced to express higher levels of CYP3A4 by treatment with vitamin D3.  Caco-2 cell monolayers are usually cultured on semipermeable plastic supports that may be fitted into the wells of multi-well culture plates.  Test compds. are then added to either the apical or basolateral sides of the monolayer.  After incubation for various lengths of time, aliquots of the buffer in opposite chambers are removed for the detn. of the concn. of test compds. and the computation of the rates of permeability for each compd. (called the apparent permeability coeffs.).  Although radiolabeled compds. were used in the original Caco-2 cells monolayer assays, radio-labeled compds. have been replaced in most labs. by the use of liq. chromatog.-mass spectrometry (LC-MS) and LC-tandem mass spectrometry (LC-MS-MS).  Mass spectrometry not only eliminates the need for radiolabeled compds., but permits the simultaneous measurement of multiple compds.  The measurement of multiple compds. per assay reduces the no. of incubations that need to be carried out, thereby increasing the throughput of the expts.  Furthermore, LC-MS and LC-MS-MS add another dimension to Caco-2 assays by facilitating the investigation of the metab. of compds. by Caco-2 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcV-AZvy78ILVg90H21EOLACvtfcHk0lh5R2Gem3J2bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKltLbK&md5=7bd0b875ffa9c840f904bab6513b7e3f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1517%2F17425255.1.2.175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.1.2.175%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BBreemen%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DB.%26atitle%3DCado-2%2520cell%2520permeability%2520assays%2520to%2520measure%2520drug%2520absorption%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2005%26volume%3D1%26spage%3D175%26epage%3D185%26doi%3D10.1517%2F17425255.1.2.175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Hay, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaumont, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kemp, M.</span><span> </span><span class="NLM_article-title">Modulation of the Partition Coefficient between Octanol and Buffer at pH 7.4 and pKa to Achieve the Optimum Balance of Blood Clearance and Volume of Distribution for a Series of Tetrahydropyran Histamine Type 3 Receptor Antagonists</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">1864</span><span class="NLM_x">–</span> <span class="NLM_lpage">1870</span><span class="refDoi"> DOI: 10.1124/dmd.109.027888</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1124%2Fdmd.109.027888" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=1864-1870&author=T.+Hayauthor=R.+Jonesauthor=K.+Beaumontauthor=M.+Kemp&title=Modulation+of+the+Partition+Coefficient+between+Octanol+and+Buffer+at+pH+7.4+and+pKa+to+Achieve+the+Optimum+Balance+of+Blood+Clearance+and+Volume+of+Distribution+for+a+Series+of+Tetrahydropyran+Histamine+Type+3+Receptor+Antagonists&doi=10.1124%2Fdmd.109.027888"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25a&amp;dbid=16384&amp;doi=10.1124%2Fdmd.109.027888&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.109.027888%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DT.%26aulast%3DJones%26aufirst%3DR.%26aulast%3DBeaumont%26aufirst%3DK.%26aulast%3DKemp%26aufirst%3DM.%26atitle%3DModulation%2520of%2520the%2520Partition%2520Coefficient%2520between%2520Octanol%2520and%2520Buffer%2520at%2520pH%25207.4%2520and%2520pKa%2520to%2520Achieve%2520the%2520Optimum%2520Balance%2520of%2520Blood%2520Clearance%2520and%2520Volume%2520of%2520Distribution%2520for%2520a%2520Series%2520of%2520Tetrahydropyran%2520Histamine%2520Type%25203%2520Receptor%2520Antagonists%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2009%26volume%3D37%26spage%3D1864%26epage%3D1870%26doi%3D10.1124%2Fdmd.109.027888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit25b"><span><span class="NLM_label">(b) </span> <span class="citation_source-book">Lead Generation: Methods and Strategies</span>; <span class="NLM_contrib-group">Holenz, J.; Mannhold, R.; Kubinyi, H.; Folkers, G.</span>, Eds.; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">67</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1002%2F9783527677047" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=J.+Holenz&author=R.+Mannhold&author=H.+Kubinyi&author=G.+Folkers&title=Lead+Generation%3A+Methods+and+Strategies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25b&amp;dbid=16384&amp;doi=10.1002%2F9783527677047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527677047%26sid%3Dliteratum%253Aachs%26btitle%3DLead%2520Generation%253A%2520Methods%2520and%2520Strategies%26aulast%3DHolenz%26aufirst%3DJ.%26pub%3DWiley%26date%3D2016%26volume%3D67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit25c"><span><span class="NLM_label">(c) </span> <span class="citation_source-book">Pharmacokinetics and Metabolism in Drug Design</span>, <span class="NLM_edition">3rd</span> ed.; <span class="NLM_contrib-group">Smith, D. A.; Allerton, C.; Kalgutkar, A. S.; van de Waterbeemd, H.; Walker, D. K.</span>, Eds.; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">; </span> <span class="NLM_volume">Vol. 51</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1002%2F9783527645763" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=D.+A.+Smith&author=C.+Allerton&author=A.+S.+Kalgutkar&author=H.+van+de+Waterbeemd&author=D.+K.+Walker&title=Pharmacokinetics+and+Metabolism+in+Drug+Design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25c&amp;dbid=16384&amp;doi=10.1002%2F9783527645763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527645763%26sid%3Dliteratum%253Aachs%26btitle%3DPharmacokinetics%2520and%2520Metabolism%2520in%2520Drug%2520Design%26aulast%3DSmith%26aufirst%3DD.%2BA.%26pub%3DWiley%26date%3D2012%26volume%3DVol.%252051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span>Cerep; <a href="http://www.cerep.fr" class="extLink">www.cerep.fr</a> (accessed August 17, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cerep%3B+www.cerep.fr+%28accessed+August+17%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DCerep" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span>SAINT: SAX, Area Detector Integration, Siemens Analytical instruments INC., Madison, Wisconsin, USA; SADABS: Siemens Area Detector Absorption Correction Software; <span class="NLM_contrib-group">Sheldrick, G.</span>, Ed.; <span class="NLM_publisher-name">University of Göttingen</span>, <span class="NLM_publisher-loc">Göttingen, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">1996</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=G.+Sheldrick&title=SAINT%3A+SAX%2C+Area+Detector+Integration%2C+Siemens+Analytical+instruments+INC.%2C+Madison%2C+Wisconsin%2C+USA%3B+SADABS%3A+Siemens+Area+Detector+Absorption+Correction+Software"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSAINT%253A%2520SAX%252C%2520Area%2520Detector%2520Integration%252C%2520Siemens%2520Analytical%2520instruments%2520INC.%252C%2520Madison%252C%2520Wisconsin%252C%2520USA%253B%2520SADABS%253A%2520Siemens%2520Area%2520Detector%2520Absorption%2520Correction%2520Software%26aulast%3DSheldrick%26aufirst%3DG.%26pub%3DUniversity%2520of%2520G%25C3%25B6ttingen%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Burla, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caliandro, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camalli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrozzini, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cascarano, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giacovazzo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mallamo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazzone, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polidori, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spagna, R.</span><span> </span><span class="NLM_article-title">SIR2011: A New Package for Crystal Structure Determination and Refinement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">357</span><span class="NLM_x">–</span> <span class="NLM_lpage">361</span><span class="refDoi"> DOI: 10.1107/S0021889812001124</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1107%2FS0021889812001124" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFekt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2012&pages=357-361&author=M.+C.+Burlaauthor=R.+Caliandroauthor=M.+Camalliauthor=B.+Carrozziniauthor=G.+L.+Cascaranoauthor=C.+Giacovazzoauthor=M.+Mallamoauthor=A.+Mazzoneauthor=G.+Polidoriauthor=R.+Spagna&title=SIR2011%3A+A+New+Package+for+Crystal+Structure+Determination+and+Refinement&doi=10.1107%2FS0021889812001124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">SIR2011: a new package for crystal structure determination and refinement</span></div><div class="casAuthors">Burla, Maria Cristina; Caliandro, Rocco; Camalli, Mercedes; Carrozzini, Benedetta; Cascarano, Giovanni Luca; Giacovazzo, Carmelo; Mallamo, Mariarosaria; Mazzone, Annamaria; Polidori, Giampiero; Spagna, Riccardo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">357-361</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">SIR2011, the successor of SIR2004, is the latest program of the SIR suite.  It can solve ab initio crystal structures of small- and medium-size mols., as well as protein structures, using X-ray or electron diffraction data.  With respect to the predecessor the program has several new abilities: e.g. a new phasing method (VLD) has been implemented, it is able to exploit prior knowledge of the mol. geometry via simulated annealing techniques, it can use mol. replacement methods for solving proteins, it includes new tools like free lunch and new approaches for electron diffraction data, and it visualizes three-dimensional electron d. maps.  The graphical interface has been further improved and allows the straightforward use of the program even in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpo4B72KAqF9bVg90H21EOLACvtfcHk0lg8dbytKLIy_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFekt7k%253D&md5=13442a23f3edd07c5e92f4d6d7c71a9d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0021889812001124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889812001124%26sid%3Dliteratum%253Aachs%26aulast%3DBurla%26aufirst%3DM.%2BC.%26aulast%3DCaliandro%26aufirst%3DR.%26aulast%3DCamalli%26aufirst%3DM.%26aulast%3DCarrozzini%26aufirst%3DB.%26aulast%3DCascarano%26aufirst%3DG.%2BL.%26aulast%3DGiacovazzo%26aufirst%3DC.%26aulast%3DMallamo%26aufirst%3DM.%26aulast%3DMazzone%26aufirst%3DA.%26aulast%3DPolidori%26aufirst%3DG.%26aulast%3DSpagna%26aufirst%3DR.%26atitle%3DSIR2011%253A%2520A%2520New%2520Package%2520for%2520Crystal%2520Structure%2520Determination%2520and%2520Refinement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2012%26volume%3D45%26spage%3D357%26epage%3D361%26doi%3D10.1107%2FS0021889812001124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Sheldrick, G. M.</span><span> </span><span class="NLM_article-title">A short history of SHELX</span> <span class="citation_source-journal">Acta Crystallogr., Sect. A: Found. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">112</span><span class="NLM_x">–</span> <span class="NLM_lpage">122</span><span class="refDoi"> DOI: 10.1107/S0108767307043930</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1107%2FS0108767307043930" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=18156677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVGhurzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2008&pages=112-122&author=G.+M.+Sheldrick&title=A+short+history+of+SHELX&doi=10.1107%2FS0108767307043930"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A short history of SHELX</span></div><div class="casAuthors">Sheldrick, George M.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section A: Foundations of Crystallography</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-122</span>CODEN:
                <span class="NLM_cas:coden">ACACEQ</span>;
        ISSN:<span class="NLM_cas:issn">0108-7673</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">An account is given of the development of the SHELX system of computer programs from SHELX-76 to the present day.  In addn. to identifying useful innovations that have come into general use through their implementation in SHELX, a crit. anal. is presented of the less-successful features, missed opportunities and desirable improvements for future releases of the software.  An attempt is made to understand how a program originally designed for photog. intensity data, punched cards and computers over 10000 times slower than an av. modern personal computer has managed to survive for so long.  SHELXL is the most widely used program for small-mol. refinement and SHELXS and SHELXD are often employed for structure soln. despite the availability of objectively superior programs.  SHELXL also finds a niche for the refinement of macromols. against high-resoln. or twinned data; SHELXPRO acts as an interface for macromol. applications.  SHELXC, SHELXD and SHELXE are proving useful for the exptl. phasing of macromols., esp. because they are fast and robust and so are often employed in pipelines for high-throughput phasing.  This paper could serve as a general literature citation when one or more of the open-source SHELX programs (and the Bruker AXS version SHELXTL) are employed in the course of a crystal-structure detn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfLPYyJyJ2RbVg90H21EOLACvtfcHk0lg8dbytKLIy_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVGhurzO&md5=8f91f10be4a9df74b2a9dad522c71e6e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0108767307043930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108767307043930%26sid%3Dliteratum%253Aachs%26aulast%3DSheldrick%26aufirst%3DG.%2BM.%26atitle%3DA%2520short%2520history%2520of%2520SHELX%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%253A%2520Found.%2520Crystallogr.%26date%3D2008%26volume%3D64%26spage%3D112%26epage%3D122%26doi%3D10.1107%2FS0108767307043930" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Naïm, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rankin, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dennis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siddiqi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drabik, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sulea, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakalian, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purisima, E. O.</span><span> </span><span class="NLM_article-title">Solvated Interaction Energy (SIE) for Scoring Protein-ligand Binding Affinities. 1. Exploring the Parameter Space</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">122</span><span class="NLM_x">–</span> <span class="NLM_lpage">133</span><span class="refDoi"> DOI: 10.1021/ci600406v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600406v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A280%3ADC%252BD2s7ntVanug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=122-133&author=M.+Na%C3%AFmauthor=S.+Bhatauthor=K.+N.+Rankinauthor=S.+Dennisauthor=S.+F.+Chowdhuryauthor=I.+Siddiqiauthor=P.+Drabikauthor=T.+Suleaauthor=C.+I.+Baylyauthor=A.+Jakalianauthor=E.+O.+Purisima&title=Solvated+Interaction+Energy+%28SIE%29+for+Scoring+Protein-ligand+Binding+Affinities.+1.+Exploring+the+Parameter+Space&doi=10.1021%2Fci600406v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Solvated interaction energy (SIE) for scoring protein-ligand binding affinities. 1. Exploring the parameter space</span></div><div class="casAuthors">Naim Marwen; Bhat Sathesh; Rankin Kathryn N; Dennis Sheldon; Chowdhury Shafinaz F; Siddiqi Imran; Drabik Piotr; Sulea Traian; Bayly Christopher I; Jakalian Araz; Purisima Enrico O</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-33</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">We present a binding free energy function that consists of force field terms supplemented by solvation terms.  We used this function to calibrate the solvation model along with the binding interaction terms in a self-consistent manner.  The motivation for this approach was that the solute dielectric-constant dependence of calculated hydration gas-to-water transfer free energies is markedly different from that of binding free energies (J.  Comput.  Chem. 2003, 24, 954).  Hence, we sought to calibrate directly the solvation terms in the context of a binding calculation.  The five parameters of the model were systematically scanned to best reproduce the absolute binding free energies for a set of 99 protein-ligand complexes.  We obtained a mean unsigned error of 1.29 kcal/mol for the predicted absolute binding affinity in a parameter space that was fairly shallow near the optimum.  The lowest errors were obtained with solute dielectric values of Din = 20 or higher and scaling of the intermolecular van der Waals interaction energy by factors ranging from 0.03 to 0.15.  The high apparent Din and strong van der Waals scaling may reflect the anticorrelation of the change in solvated potential energy and configurational entropy, that is, enthalpy-entropy compensation in ligand binding (Biophys.  J. 2004, 87, 3035-3049).  Five variations of preparing the protein-ligand data set were explored in order to examine the effect of energy refinement and the presence of bound water on the calculated results.  We find that retaining water in the final protein structure used for calculating the binding free energy is not necessary to obtain good results; that is the continuum solvation model is sufficient.  Virtual screening enrichment studies on estrogen receptor and thymidine kinase showed a good ability of the binding free energy function to recover true hits in a collection of decoys.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR0C1HMXVybNqNc3HjJRoFAfW6udTcc2ebMUDlBw0xEPrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s7ntVanug%253D%253D&md5=a5409b8aea519c6413f0fe77dbce4f57</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci600406v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600406v%26sid%3Dliteratum%253Aachs%26aulast%3DNa%25C3%25AFm%26aufirst%3DM.%26aulast%3DBhat%26aufirst%3DS.%26aulast%3DRankin%26aufirst%3DK.%2BN.%26aulast%3DDennis%26aufirst%3DS.%26aulast%3DChowdhury%26aufirst%3DS.%2BF.%26aulast%3DSiddiqi%26aufirst%3DI.%26aulast%3DDrabik%26aufirst%3DP.%26aulast%3DSulea%26aufirst%3DT.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DPurisima%26aufirst%3DE.%2BO.%26atitle%3DSolvated%2520Interaction%2520Energy%2520%2528SIE%2529%2520for%2520Scoring%2520Protein-ligand%2520Binding%2520Affinities.%25201.%2520Exploring%2520the%2520Parameter%2520Space%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D122%26epage%3D133%26doi%3D10.1021%2Fci600406v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Labute, P.</span><span> </span><span class="NLM_article-title">The Generalized Born/Volume Integral (GB/VI) Implicit Solvent Model: Estimation of the Free Energy of Hydration Using London Dispersion Instead of Atomic Surface Area</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1693</span><span class="NLM_x">–</span> <span class="NLM_lpage">1698</span><span class="refDoi"> DOI: 10.1002/jcc.20933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=10.1002%2Fjcc.20933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=18307169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFCgu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=1693-1698&author=P.+Labute&title=The+Generalized+Born%2FVolume+Integral+%28GB%2FVI%29+Implicit+Solvent+Model%3A+Estimation+of+the+Free+Energy+of+Hydration+Using+London+Dispersion+Instead+of+Atomic+Surface+Area&doi=10.1002%2Fjcc.20933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The generalized Born/volume integral implicit solvent model: estimation of the free energy of hydration using London dispersion instead of atomic surface area</span></div><div class="casAuthors">Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1693-1698</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A new generalized Born model for estg. the free energy of hydration is presented.  The new generalized Born/vol. integral (GB/VI) ests. the free energy of hydration as a classical electrostatic energy plus a cavitation energy that is not based upon at. surface area (SA) used in GB/SA hydration models but on a VI London dispersion energy estd. from quantities already calcd. in the classical electrostatic energy.  The (relatively few) GB/VI model parameters are fitted to exptl. data, and parameterizations for two different at. partial charge models are presented.  Comparison of the calcd. and exptl. free energies of hydration for 560 small mols. (both neutral and charged) shows good agreement (r2 = 0.94).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCY3g7kWWRo7Vg90H21EOLACvtfcHk0lgK64pC0i94kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFCgu7g%253D&md5=59420cf1dbe689836873df8e802d06cd</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20933%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DThe%2520Generalized%2520Born%252FVolume%2520Integral%2520%2528GB%252FVI%2529%2520Implicit%2520Solvent%2520Model%253A%2520Estimation%2520of%2520the%2520Free%2520Energy%2520of%2520Hydration%2520Using%2520London%2520Dispersion%2520Instead%2520of%2520Atomic%2520Surface%2520Area%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2008%26volume%3D29%26spage%3D1693%26epage%3D1698%26doi%3D10.1002%2Fjcc.20933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=NM_000740" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=NM_000740','PDB','NM_000740'); return false;">PDB: NM_000740</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL','PDB','3PBL'); return false;">PDB: 3PBL</a></li><li><a href="http://www.ncbi.nlm.nih.gov/gquery?Term=NM_000796.2" onclick="trackOutboundLink('http://www.ncbi.nlm.nih.gov/gquery?Term=NM_000796.2','Genbank','NM_000796.2'); return false;">Genbank: NM_000796.2</a></li><li><a href="http://www.ncbi.nlm.nih.gov/gquery?Term=NM_000738" onclick="trackOutboundLink('http://www.ncbi.nlm.nih.gov/gquery?Term=NM_000738','Genbank','NM_000738'); return false;">Genbank: NM_000738</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i130"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00972">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_35010"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b00972">10.1021/acs.jmedchem.6b00972</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Key intermediate preparation and their NMR spectra (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00972/suppl_file/jm6b00972_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00972/suppl_file/jm6b00972_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00972/suppl_file/jm6b00972_si_001.pdf">jm6b00972_si_001.pdf (1.22 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00972/suppl_file/jm6b00972_si_002.csv">jm6b00972_si_002.csv (13.9 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b00972&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b00972%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-18" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b00972" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799d007086b3dbe","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
