
<html lang="en"     class="pb-page"  data-request-id="05096693-9372-4cf0-a01b-bf87eeb9e6b2"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2019.62.issue-14;website:website:acspubs;article:article:10.1021/acs.jmedchem.9b00260;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of a Thiadiazole–Pyridazine-Based Allosteric Glutaminase 1 Inhibitor Series That Demonstrates Oral Bioavailability and Activity in Tumor Xenograft Models" /></meta><meta name="dc.Creator" content="M. Raymond V.  Finlay" /></meta><meta name="dc.Creator" content="Mark  Anderton" /></meta><meta name="dc.Creator" content="Andrew  Bailey" /></meta><meta name="dc.Creator" content="Scott  Boyd" /></meta><meta name="dc.Creator" content="Joanna  Brookfield" /></meta><meta name="dc.Creator" content="Ceri  Cairnduff" /></meta><meta name="dc.Creator" content="Mark  Charles" /></meta><meta name="dc.Creator" content="Anne  Cheasty" /></meta><meta name="dc.Creator" content="Susan E.  Critchlow" /></meta><meta name="dc.Creator" content="Janet  Culshaw" /></meta><meta name="dc.Creator" content="Tennyson  Ekwuru" /></meta><meta name="dc.Creator" content="Ian  Hollingsworth" /></meta><meta name="dc.Creator" content="Neil  Jones" /></meta><meta name="dc.Creator" content="Fred  Leroux" /></meta><meta name="dc.Creator" content="Mairi  Littleson" /></meta><meta name="dc.Creator" content="Hollie  McCarron" /></meta><meta name="dc.Creator" content="Jennifer  McKelvie" /></meta><meta name="dc.Creator" content="Lorraine  Mooney" /></meta><meta name="dc.Creator" content="J. Willem M.  Nissink" /></meta><meta name="dc.Creator" content="David  Perkins" /></meta><meta name="dc.Creator" content="Steve  Powell" /></meta><meta name="dc.Creator" content="Mar Jimenez  Quesada" /></meta><meta name="dc.Creator" content="Piotr  Raubo" /></meta><meta name="dc.Creator" content="Verity  Sabin" /></meta><meta name="dc.Creator" content="James  Smith" /></meta><meta name="dc.Creator" content="Peter D.  Smith" /></meta><meta name="dc.Creator" content="Andrew  Stark" /></meta><meta name="dc.Creator" content="Attilla  Ting" /></meta><meta name="dc.Creator" content="Peng  Wang" /></meta><meta name="dc.Creator" content="Zena  Wilson" /></meta><meta name="dc.Creator" content="Jon J.  Winter-Holt" /></meta><meta name="dc.Creator" content="J. Matthew  Wood" /></meta><meta name="dc.Creator" content="Gail L.  Wrigley" /></meta><meta name="dc.Creator" content="Guoqing  Yu" /></meta><meta name="dc.Creator" content="Peng  Zhang" /></meta><meta name="dc.Description" content="Tumors have evolved a variety of methods to reprogram conventional metabolic pathways to favor their own nutritional needs, including glutaminolysis, the first step of which is the hydrolysis of gl..." /></meta><meta name="Description" content="Tumors have evolved a variety of methods to reprogram conventional metabolic pathways to favor their own nutritional needs, including glutaminolysis, the first step of which is the hydrolysis of gl..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 14, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00260" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00260" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00260" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00260" /></link>
        
    
    

<title>Discovery of a Thiadiazole–Pyridazine-Based Allosteric Glutaminase 1 Inhibitor Series That Demonstrates Oral Bioavailability and Activity in Tumor Xenograft Models | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00260" /></meta><meta property="og:title" content="Discovery of a Thiadiazole–Pyridazine-Based Allosteric Glutaminase 1 Inhibitor Series That Demonstrates Oral Bioavailability and Activity in Tumor Xenograft Models" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0023.jpeg" /></meta><meta property="og:description" content="Tumors have evolved a variety of methods to reprogram conventional metabolic pathways to favor their own nutritional needs, including glutaminolysis, the first step of which is the hydrolysis of glutamine to glutamate by the amidohydrolase glutaminase 1 (GLS1). A GLS1 inhibitor could potentially target certain cancers by blocking the tumor cell’s ability to produce glutamine-derived nutrients. Starting from the known GLS1 inhibitor bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide, we describe the medicinal chemistry evolution of a series from lipophilic inhibitors with suboptimal physicochemical and pharmacokinetic properties to cell potent examples with reduced molecular weight and lipophilicity, leading to compounds with greatly improved oral exposure that demonstrate in vivo target engagement accompanied by activity in relevant disease models." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00260"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00260">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00260&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00260&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00260&amp;href=/doi/10.1021/acs.jmedchem.9b00260" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 6540-6560</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00171" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00329" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a Thiadiazole–Pyridazine-Based Allosteric Glutaminase 1 Inhibitor Series That Demonstrates Oral Bioavailability and Activity in Tumor Xenograft Models</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">M. Raymond V. Finlay</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">M. Raymond V. Finlay</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#c2b0a3bbeca4abacaea3bb82a3b1b6b0a3b8a7aca7a1a3eca1adaf"><span class="__cf_email__" data-cfemail="1a687b63347c7374767b635a7b696e687b607f747f797b34797577">[email protected]</span></a> (M.R.V.F.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=M.+Raymond+V.++Finlay">M. Raymond V. Finlay</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1402-193X" title="Orcid link">http://orcid.org/0000-0003-1402-193X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mark Anderton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mark Anderton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mark++Anderton">Mark Anderton</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrew Bailey</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew Bailey</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew++Bailey">Andrew Bailey</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Scott Boyd</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Scott Boyd</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Scott++Boyd">Scott Boyd</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joanna Brookfield</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joanna Brookfield</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research Campus, Cambridge CB22 3AT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joanna++Brookfield">Joanna Brookfield</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ceri Cairnduff</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ceri Cairnduff</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research Campus, Cambridge CB22 3AT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ceri++Cairnduff">Ceri Cairnduff</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mark Charles</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mark Charles</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research Campus, Cambridge CB22 3AT, United Kingdom</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#327f5340591c715a53405e57417251535c5157401c5d40551c4759"><span class="__cf_email__" data-cfemail="703d11021b5e331811021c15033013111e1315025e1f02175e051b">[email protected]</span></a> (M.C.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mark++Charles">Mark Charles</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anne Cheasty</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anne Cheasty</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research Campus, Cambridge CB22 3AT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anne++Cheasty">Anne Cheasty</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Susan E. Critchlow</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Susan E. Critchlow</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Susan+E.++Critchlow">Susan E. Critchlow</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Janet Culshaw</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Janet Culshaw</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Janet++Culshaw">Janet Culshaw</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tennyson Ekwuru</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tennyson Ekwuru</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research Campus, Cambridge CB22 3AT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tennyson++Ekwuru">Tennyson Ekwuru</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ian Hollingsworth</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ian Hollingsworth</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ian++Hollingsworth">Ian Hollingsworth</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Neil Jones</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Neil Jones</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research Campus, Cambridge CB22 3AT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Neil++Jones">Neil Jones</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fred Leroux</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fred Leroux</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research Campus, Cambridge CB22 3AT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fred++Leroux">Fred Leroux</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mairi Littleson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mairi Littleson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mairi++Littleson">Mairi Littleson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6909-5608" title="Orcid link">http://orcid.org/0000-0001-6909-5608</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hollie McCarron</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hollie McCarron</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research Campus, Cambridge CB22 3AT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hollie++McCarron">Hollie McCarron</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jennifer McKelvie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jennifer McKelvie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research Campus, Cambridge CB22 3AT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jennifer++McKelvie">Jennifer McKelvie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lorraine Mooney</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lorraine Mooney</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lorraine++Mooney">Lorraine Mooney</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">J. Willem M. Nissink</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">J. Willem M. Nissink</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J.+Willem+M.++Nissink">J. Willem M. Nissink</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2572-9140" title="Orcid link">http://orcid.org/0000-0003-2572-9140</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David Perkins</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Perkins</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Perkins">David Perkins</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Steve Powell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Steve Powell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Steve++Powell">Steve Powell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mar Jimenez Quesada</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mar Jimenez Quesada</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research Campus, Cambridge CB22 3AT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mar+Jimenez++Quesada">Mar Jimenez Quesada</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Piotr Raubo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Piotr Raubo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Piotr++Raubo">Piotr Raubo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Verity Sabin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Verity Sabin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research Campus, Cambridge CB22 3AT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Verity++Sabin">Verity Sabin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James Smith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James Smith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research Campus, Cambridge CB22 3AT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James++Smith">James Smith</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter D. Smith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter D. Smith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+D.++Smith">Peter D. Smith</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5312-9893" title="Orcid link">http://orcid.org/0000-0001-5312-9893</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrew Stark</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew Stark</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew++Stark">Andrew Stark</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Attilla Ting</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Attilla Ting</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Attilla++Ting">Attilla Ting</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peng Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peng Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaron Beijing Co., Ltd., 6 Taihe Road, BDA, Beijing 100176, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peng++Wang">Peng Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zena Wilson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zena Wilson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zena++Wilson">Zena Wilson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jon J. Winter-Holt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jon J. Winter-Holt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jon+J.++Winter-Holt">Jon J. Winter-Holt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">J. Matthew Wood</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">J. Matthew Wood</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J.+Matthew++Wood">J. Matthew Wood</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gail L. Wrigley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gail L. Wrigley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology, IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton Road, Cambridge CB4 0FZ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gail+L.++Wrigley">Gail L. Wrigley</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guoqing Yu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guoqing Yu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaron Beijing Co., Ltd., 6 Taihe Road, BDA, Beijing 100176, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guoqing++Yu">Guoqing Yu</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Peng Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peng Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaron Beijing Co., Ltd., 6 Taihe Road, BDA, Beijing 100176, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peng++Zhang">Peng Zhang</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00260&amp;href=/doi/10.1021%2Facs.jmedchem.9b00260" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 6540–6560</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 14, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 February 2019</li><li><span class="item_label"><b>Published</b> online</span>14 June 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 July 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00260" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00260</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6540%26pageCount%3D21%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DM.%2BRaymond%2BV.%2BFinlay%252C%2BMark%2BAnderton%252C%2BAndrew%2BBailey%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D14%26contentID%3Dacs.jmedchem.9b00260%26title%3DDiscovery%2Bof%2Ba%2BThiadiazole%25E2%2580%2593Pyridazine-Based%2BAllosteric%2BGlutaminase%2B1%2BInhibitor%2BSeries%2BThat%2BDemonstrates%2BOral%2BBioavailability%2Band%2BActivity%2Bin%2BTumor%2BXenograft%2BModels%26numPages%3D21%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6560%26publicationDate%3DJuly%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00260"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1839</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">9</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00260" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a Thiadiazole–Pyridazine-Based Allosteric Glutaminase 1 Inhibitor Series That Demonstrates Oral Bioavailability and Activity in Tumor Xenograft Models&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;M.&quot;,&quot;last_name&quot;:&quot;Raymond V. Finlay&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;Anderton&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;Bailey&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;Boyd&quot;},{&quot;first_name&quot;:&quot;Joanna&quot;,&quot;last_name&quot;:&quot;Brookfield&quot;},{&quot;first_name&quot;:&quot;Ceri&quot;,&quot;last_name&quot;:&quot;Cairnduff&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;Charles&quot;},{&quot;first_name&quot;:&quot;Anne&quot;,&quot;last_name&quot;:&quot;Cheasty&quot;},{&quot;first_name&quot;:&quot;Susan&quot;,&quot;last_name&quot;:&quot;E. Critchlow&quot;},{&quot;first_name&quot;:&quot;Janet&quot;,&quot;last_name&quot;:&quot;Culshaw&quot;},{&quot;first_name&quot;:&quot;Tennyson&quot;,&quot;last_name&quot;:&quot;Ekwuru&quot;},{&quot;first_name&quot;:&quot;Ian&quot;,&quot;last_name&quot;:&quot;Hollingsworth&quot;},{&quot;first_name&quot;:&quot;Neil&quot;,&quot;last_name&quot;:&quot;Jones&quot;},{&quot;first_name&quot;:&quot;Fred&quot;,&quot;last_name&quot;:&quot;Leroux&quot;},{&quot;first_name&quot;:&quot;Mairi&quot;,&quot;last_name&quot;:&quot;Littleson&quot;},{&quot;first_name&quot;:&quot;Hollie&quot;,&quot;last_name&quot;:&quot;McCarron&quot;},{&quot;first_name&quot;:&quot;Jennifer&quot;,&quot;last_name&quot;:&quot;McKelvie&quot;},{&quot;first_name&quot;:&quot;Lorraine&quot;,&quot;last_name&quot;:&quot;Mooney&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Willem M. Nissink&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Perkins&quot;},{&quot;first_name&quot;:&quot;Steve&quot;,&quot;last_name&quot;:&quot;Powell&quot;},{&quot;first_name&quot;:&quot;Mar&quot;,&quot;last_name&quot;:&quot;Jimenez Quesada&quot;},{&quot;first_name&quot;:&quot;Piotr&quot;,&quot;last_name&quot;:&quot;Raubo&quot;},{&quot;first_name&quot;:&quot;Verity&quot;,&quot;last_name&quot;:&quot;Sabin&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;Smith&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;D. Smith&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;Stark&quot;},{&quot;first_name&quot;:&quot;Attilla&quot;,&quot;last_name&quot;:&quot;Ting&quot;},{&quot;first_name&quot;:&quot;Peng&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Zena&quot;,&quot;last_name&quot;:&quot;Wilson&quot;},{&quot;first_name&quot;:&quot;Jon&quot;,&quot;last_name&quot;:&quot;J. Winter-Holt&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Matthew Wood&quot;},{&quot;first_name&quot;:&quot;Gail&quot;,&quot;last_name&quot;:&quot;L. Wrigley&quot;},{&quot;first_name&quot;:&quot;Guoqing&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Peng&quot;,&quot;last_name&quot;:&quot;Zhang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;14&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;6540-6560&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00260&quot;},&quot;abstract&quot;:&quot;Tumors have evolved a variety of methods to reprogram conventional metabolic pathways to favor their own nutritional needs, including glutaminolysis, the first step of which is the hydrolysis of glutamine to glutamate by the amidohydrolase glutaminase 1 (GLS1). A GLS1 inhibitor could potentially target certain cancers by blocking the tumor cell’s ability to produce glutamine-derived nutrients. Starting from the known GLS1 inhibitor bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide, we describe the medicinal chemistry evolution of a series from lipophilic inhibitors with suboptimal physicochemical and pharmacokinetic properties to cell potent examples with reduced molecular weight and lipophilicity, leading to compounds with greatly improved oral exposure that demonstrate in vivo target engagement accompanied by activity in relevant disease models.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00260&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00260" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00260&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00260" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00260&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00260" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00260&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00260&amp;href=/doi/10.1021/acs.jmedchem.9b00260" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00260" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00260" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00260&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00260%26sid%3Dliteratum%253Aachs%26pmid%3D31199640%26genre%3Darticle%26aulast%3DFinlay%26date%3D2019%26atitle%3DDiscovery%2Bof%2Ba%2BThiadiazole%25E2%2580%2593Pyridazine-Based%2BAllosteric%2BGlutaminase%2B1%2BInhibitor%2BSeries%2BThat%2BDemonstrates%2BOral%2BBioavailability%2Band%2BActivity%2Bin%2BTumor%2BXenograft%2BModels%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D14%26spage%3D6540%26epage%3D6560%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290658" title="Amides">Amides</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292504" title="Permeability">Permeability</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/jmcmar.2019.62.issue-14/20190725/jmcmar.2019.62.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0023.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Tumors have evolved a variety of methods to reprogram conventional metabolic pathways to favor their own nutritional needs, including glutaminolysis, the first step of which is the hydrolysis of glutamine to glutamate by the amidohydrolase glutaminase 1 (GLS1). A GLS1 inhibitor could potentially target certain cancers by blocking the tumor cell’s ability to produce glutamine-derived nutrients. Starting from the known GLS1 inhibitor bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide, we describe the medicinal chemistry evolution of a series from lipophilic inhibitors with suboptimal physicochemical and pharmacokinetic properties to cell potent examples with reduced molecular weight and lipophilicity, leading to compounds with greatly improved oral exposure that demonstrate in vivo target engagement accompanied by activity in relevant disease models.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11045" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11045" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Alteration of metabolic pathways in tumor cells and how these changes might be exploited for therapeutic benefit has become the focus of intense interest over the past decade.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> While the concept of the Warburg effect<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> has been known for some time, more recent activity has focused on its application (and indeed the modulation of tumor-specific metabolic pathways as a whole) to drive tumor growth inhibition (TGI).</div><div class="NLM_p">Glutamine is an essential nutrient for cancer cells and is used to support cell proliferation<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> by its hydrolysis to glutamate and ammonia by the amidohydrolase enzyme glutaminase (GLS). The glutamate produced in this process can be converted into α-ketoglutarate, a metabolic intermediate within the tricarboxylic acid (TCA) cycle (also known as the Krebs/citric acid cycle), and thereby serves as a precursor for the biosynthesis of adenosine 5′-triphosphate, reduced nicotinamide adenine dinucleotide phosphate, fatty acids, glutathione, nucleic acids, and amino acids.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> In addition, glutaminolysis is essential for reactive oxygen species homeostasis<a onclick="showRef(event, 'cit5a'); return false;" href="javascript:void(0);" class="ref cit5a">(5a)</a> through synthesis of glutathione. Glutathione is a tripeptide (Glu–Cys–Gly) that regulates redox stress by neutralizing peroxide free radicals. Furthermore, glutamate is required to maintain flux of cystine via the SLC7A11 transporter. Cystine is converted into cysteine within cells and is also a critical precursor of glutathione.<a onclick="showRef(event, 'cit5b'); return false;" href="javascript:void(0);" class="ref cit5b">(5b)</a></div><div class="NLM_p">Many tumors show increased expression levels of GLS1 as well as an increased uptake of and dependence upon glutamine to supply the cell with intermediates to feed the TCA cycle.<a onclick="showRef(event, 'cit5a'); return false;" href="javascript:void(0);" class="ref cit5a">(5a)</a> As a consequence of the dependency on, and addiction to glutamine metabolism observed in a variety of tumors, a GLS1 inhibitor could potentially target multiple essential roles for the maintenance, survival, and progression of cancers by blocking the tumor cell’s ability to produce nutrients derived from glutamate and to regulate redox stress. Accordingly, and as recently demonstrated, cancer cell lines with an increased glutamine dependence and high expression of GLS1 should be susceptible to glutaminase inhibition.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">The identification of a GLS1 inhibitor presents a number of medicinal chemistry challenges, most of which are due to the highly polar nature of the enzyme substrate and the consequent need for a corresponding level of polarity in the enzyme active site. Inhibitors binding at this locus, while potentially feasible, would likely need to share the high polarity of glutamine, leading to compounds with suboptimal pharmacokinetic (PK) and cell permeability profiles. For example, the published GLS1 inhibitors 6-diazo-5-oxo-<span class="smallcaps smallerCapital">l</span>-norleucine<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> (DON, <b>1</b>) and acivicin<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> (<b>2</b>) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) are thought to covalently bind to GLS1 in the active site and display high polarity. Nevertheless, a seminal publication in this area disclosed that other modes of inhibition of GLS1 were achievable.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Employing the compound bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide (BPTES, <b>3</b>) the authors were able to show that <b>3</b> is a unique and potent inhibitor of rat GLS1 and elucidated a novel, allosteric mechanism of inactivation.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0001.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of GLS1 inhibitors DON (<b>1</b>), acivicin (<b>2</b>), BPTES (<b>3</b>), 968 (<b>4</b>), and CB-839 (<b>5</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Rather than binding in the enzyme active site, X-ray crystallography has shown that BPTES binds between the dimer interfaces of two GLS1 subunits occupying equivalent space on each protein (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0002.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) BPTES (<b>3</b>) binding mode to GLS1(PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGAA">3KGAA</a>). Two molecules of inhibitor are associated with four units of protein. BPTES sits between the interfaces of two GLS1 subunits occupying equivalent space on each protein and appears to hold the protein in a nonfunctional tetrameric form. (b) Binding mode of BPTES. Each half of the BPTES molecule interacts with one protein chain of the dimer unit and forms hydrogen-bonding contacts with its thiadiazole amide. The interactions are shown for one thiadiazole carboxamide substructure in (c). Four symmetry-related lipophilic residues (Phe322, Tyr394, visible top middle) form a hydrophobic “cap” centered over the lipophilic diethylsulfide linker of BPTES. (c) The backbone atoms of Phe322 and Leu323 form three hydrogen bonds to the thiadiazole carboxamide.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">While <b>3</b> is a useful tool compound, its physicochemical properties prevent it from being used clinically. Nevertheless, a number of groups have employed BPTES as a start point for medicinal chemistry efforts with the goal of delivering a more druglike molecule. Among these, the Cerione Lab has reported compound 968,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> which is believed to bind at an alternative allosteric site to BPTES, while Pfizer,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Agios,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and Johns Hopkins University<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> have detailed elegant approaches to improve BPTES like analogues and, indeed, researchers at Calithera Biosciences have recently advanced GLS inhibitor CB-839<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> (<b>5</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) into clinical trials in cancer patients.</div><div class="NLM_p last">We also employed BPTES as a starting point for our efforts to develop inhibitors binding at the GLS dimer interface. Our main concerns at the outset of the current studies were that it would be challenging to deliver inhibitors with good physicochemical properties, given the molecular size required to interact between the GLS1 monomers. Our design strategy focused on two main elements: first, BPTES requires lipophilic phenyl rings on both ends of the molecule to retain potency. It should be noted that at the outset of this work, all of the reported examples in the literature that retain high levels of GLS1 potency had an analogous structure to BPTES with phenyl rings on either end of the molecule. The role of these phenyl rings was not clear from examination of the crystal structure, so we postulated that it might be possible to replace or remove them completely. Second, we reasoned that it should be possible to conformationally restrict the central linker to improve potency. The compounds herein demonstrate that despite the symmetrical nature of the GLS1 allosteric binding site, inhibitor molecular weight and symmetry can be dramatically reduced, resulting in the discovery of <b>27</b>, a potent, water-soluble, orally (PO) available GLS1 inhibitor, which displays activity in relevant in vivo disease models.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47344" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47344" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In general, the synthesis of the disclosed compounds was based on nucleophilic aromatic substitution reaction (S<sub>N</sub>Ar) between diamines and the corresponding halo-heteroaromatic reagents. Compounds with symmetrical diaminoethyl linkers were obtained in a simple two-step procedure involving nucleophilic aromatic substitution of the bromide in 5-bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) with 0.5 equiv of the corresponding 1,2-diamine, followed by acylation of both heteroaromatic NH<sub>2</sub> groups with 2-phenylacetyl chloride to provide <b>9</b>–<b>11</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<named-content content-type="anchor" rid="tbl1" type="simple"></named-content><named-content content-type="anchor" rid="tbl2" type="simple"></named-content></div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>9</b>–<b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1,2-diamine, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), MeCN or dimethylformamide (DMF), microwave (μW), 80–130 °C; (b) PhCH<sub>2</sub>COCl, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, room temperature (rt), 4–22%.</p></p></figure><div class="NLM_p">For unsymmetrical chiral linkers, mono-<i>N</i>-<i>tert</i>-butoxycarbonyl (<i>N</i>-Boc)-protected chiral diamines were used. Thus, (<i>R</i>)-<i>tert</i>-butyl 3-aminopyrrolidine-1-carboxylate (<b>31</b>) underwent reaction with <b>30</b>, which was followed by acylation of the heterocyclic amine with 2-phenylacetyl chloride and <i>N</i>-Boc deprotection with trifluoroacetic acid in dichloromethane (DCM) to produce the monosubstituted intermediate pyrrolidine <b>32</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Subsequent S<sub>N</sub>Ar reaction with <b>30</b> delivered the intermediate aniline <b>18</b>, which was subjected to amide formation reactions with acid chlorides or carboxylic acids to yield the corresponding unsymmetrical bisamides <b>12</b> and <b>14</b>–<b>17</b>. S<sub>N</sub>Ar reaction between <b>32</b> and 2-bromo-1,3,4-thiadiazole or 2-chloropyridazine delivered monoamides <b>19</b> and <b>23</b>, respectively. The bisamide <b>13</b> was prepared in a similar fashion from (<i>R</i>)-<i>tert</i>-butyl 3-(aminomethyl)pyrrolidine-1-carboxylate (<b>33</b>).</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0009.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>12</b>–<b>19</b> and <b>23</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>30</b>, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C, 88%; (b) PhCH<sub>2</sub>COCl, K<sub>2</sub>CO<sub>3</sub>, tetrahydrofuran (THF), rt, 100%; (c) trifluoroacetic acid (TFA), CH<sub>2</sub>Cl<sub>2</sub>, rt, 98%; (d) <b>30</b>, K<sub>2</sub>CO<sub>3</sub>, MeCN, rt, 73%; (e) R1COCl, DIPEA, DMF, rt, 7–43%; (f) R1COOH, 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (HATU), DIPEA, dimethylacetamide (DMA), rt, 32–39%; (g) 2-bromo-1,3,4-thiadiazole, 4-dimethylaminopyridine (DMAP), DIPEA, <i>N</i>-methyl-2-pyrrolidone (NMP), 106 °C, 6%; (h) 2-chloropyridazine, DMAP, DIPEA, NMP, 106 °C, 26%; (i) <b>30</b>, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, μW, 60 °C; (j) PhCH<sub>2</sub>COCl, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>; (k) TFA, CH<sub>2</sub>Cl<sub>2</sub>, 100% (three steps); (l) <b>30</b>, DIPEA, DMF, 100 °C; (m) PhCH<sub>2</sub>COCl, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 43% (two steps).</p></p></figure><div class="NLM_p">The diamino-pyridazine analogue <b>20</b> was synthesized as shown in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. The Boc-protected 3-aminopyrrolidine <b>34</b> was converted to the 3-amino-1-thiadiazoyl pyrrolidine <b>35</b> on treatment with 2-bromo-1,3,4-thiadiazole in the presence of Hunig’s base, followed by Boc deprotection with trifluoroacetic acid (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Acylation of a commercially available 6-iodopyridazin-3-amine (<b>36</b>) with 2-phenylacetyl chloride gave the iodide <b>37</b>, which was subjected to copper-catalyzed coupling reaction<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> with <b>35</b> in the presence of (2<i>S</i>,4<i>R</i>)-4-hydroxypyrrolidine-2-carboxylic acid to afford <b>20</b>.</div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0010.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>20</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2-bromo-1,3,4-thiadiazole, DIPEA, <i>n</i>-BuOH, 50 °C, 70%; (b) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 98%; (c) PhCH<sub>2</sub>COCl, DIPEA, DMF, rt, 25%; (d) CuI, (2<i>S</i>,4<i>R</i>)-4-hydroxypyrrolidine-2-carboxylic acid, K<sub>2</sub>CO<sub>3</sub>, dimethyl sulfoxide (DMSO), 80 °C, 8.4%.</p></p></figure><div class="NLM_p">Aminothiadiazole analogues <b>21</b> and <b>22</b> were obtained using similar chemical transformations, as described in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, starting from halo aminoheterocycles <b>30</b> and <b>38</b> and the corresponding Boc-protected aminopyrrolidine <b>34</b>, as depicted in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>.</div><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0011.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>21</b> and <b>22</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>34</b>, Et<sub>3</sub>N, <i>n</i>-BuOH, 160–180 °C, 100%; (b) PhCH<sub>2</sub>COCl, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 25%; (c) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 71%; (d) <b>30</b>, DIPEA, MeCN, 80 °C, 17%; (e) <b>34</b>, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C, quant; (f) PhCH<sub>2</sub>COCl, K<sub>2</sub>CO<sub>3</sub>, THF, rt, quant; (g) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 62%; (h) <b>30</b>, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C, 55%.</p></p></figure><div class="NLM_p">For the triazine analogue <b>24</b>, 3-(methylthio)-1,2,4-triazine (<b>39</b>) was oxidized with <i>m</i>-chloroperbenzoic acid (<i>m</i>CPBA) and a subsequent S<sub>N</sub>Ar with <b>34</b> in the presence of Hunig’s base followed by Boc deprotection with TFA, then afforded the amine <b>40</b> in excellent yield (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>). A second S<sub>N</sub>Ar reaction between <b>40</b> and bromo aminothiadiazole <b>30</b> followed by amide formation with phenylacetic acid gave <b>24</b>.</div><figure id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0012.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compound <b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions:  (a) <i>m</i>CPBA, CH<sub>2</sub>Cl<sub>2</sub>, rt; (b) <b>34</b>, DIPEA, isopropyl alcohol (IPA), 50 °C, 98%; (c) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 96%; (d) <b>30</b>, DIPEA, DMF, 100 °C; (e) PhCH<sub>2</sub>COOH, HATU, DIPEA, DMF, rt, 46% (two steps).</p></p></figure><div class="NLM_p">Amides <b>25</b> and <b>26</b> were prepared starting from ethyl 5-bromo-1,3,4-thiadiazole-2-carboxylate (<b>41</b>) (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>).</div><figure id="sch6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0013.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compounds <b>25</b> and <b>26</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) BnNH<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>, rt to 60 °C, 48%; (b) <b>31</b>, K<sub>2</sub>CO<sub>3</sub>, DMA, MeCN 80–100 °C, 46%; (c) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 69%; (d) 2-chloropyridazine, DMAP, DIPEA, NMP, 110 °C, 13%; (e) <b>40</b>, DIPEA, DMF, 100 °C, 53%.</p></p></figure><div class="NLM_p">The ester <b>41</b> was converted to the amide <b>42</b> on treatment with benzylamine. Aromatic nucleophilic substitution of bromine in <b>42</b> with the 3-aminopyrrolidine <b>40</b> delivered the expected triazine <b>26</b>. Similarly, S<sub>N</sub>Ar between <b>42</b> and <b>31</b> followed by Boc group removal to release the free NH pyrrolidine and subsequent S<sub>N</sub>Ar with 2-chloropyridazine afforded the pyridazine analogue <b>25</b>.</div><div class="NLM_p">Compound <b>34</b> was also used as a starting material in the preparation of chiral amides <b>27</b> and <b>28</b> (<a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>).</div><figure id="sch7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0014.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Compounds <b>27</b>–<b>29</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions:  (a) 3,6-dichloropyridazine, DIPEA, MeCN, reflux, 69%; (b) Pd/C, HCOONH<sub>4</sub>, EtOH, reflux, 98%; (c) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 93%; (d) <b>30</b>, DIPEA, MeOH, 45 °C, 83%; (e) for R1 = OMe, (<i>S</i>)-2-methoxy-2-phenylacetic acid, HATU, 1,4,6-trimethylpyridine, DMF, NMP, 0–20 °C, 86%; for R1 = Me; (2<i>S</i>)-phenylpropanoic acid, HATU, DIPEA, DMF, rt, 16%; (f) for A = N; <b>30</b>, DIPEA, DMF, 100 °C, then (<i>S</i>)-2-methoxy-2-phenylacetic acid, HATU, DIPEA, rt, 47%.</p></p></figure><div class="NLM_p last">Initially, <b>34</b> was converted to the aminopyrrolidine <b>43</b> in a three-step reaction sequence, starting with N-arylation of the pyrrolidine nitrogen with 3,6-dichloropyridazine. This was followed by palladium-catalyzed removal of the pyridazine chlorine under hydrogen transfer conditions and deprotection of the pendant amino group. Subsequent S<sub>N</sub>Ar with the bromo amino thiazole <b>30</b> followed by amide coupling with (<i>S</i>)-2-methoxy-2-phenylacetic acid or (<i>S</i>)-phenylpropanoic acid provided expected chiral amides <b>27</b> and <b>28</b>, respectively. In a similar fashion, the aminopyrrolidine <b>40</b> was converted to the corresponding triazine amide <b>29</b>.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85080" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85080" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Although <b>3</b> does offer potency in both biochemical and cell (mode of action and phenotypic) settings, as described above, its high lipophilicity compromises its physicochemical properties. At the outset of this work, little was known of any structure–activity relationships (SARs) around the scaffold and we thus elected to explore some close analogues focusing on the central linker uniting the two symmetrical halves of the molecule. Our initial exploration in this area focused on both replacement of the sulfur atom in this chain and contracting its length. This led to compounds such as those shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. The example where the sulfur was replaced by carbon (<b>6</b>) was somewhat less potent in the PC-3 cell assay than <b>3</b>, but interestingly, truncation of the linker by an additional carbon gave <b>7</b> with improved cell potency relative to <b>3</b>. Further modification of <b>7</b> by the removal of a further carbon atom in the linker chain however led to <b>8</b> with reduced enzyme potency, but an improvement in cellular activity. Other workers in the field had also considered these modifications, and indeed these compounds were subsequently disclosed in the literature by several different groups.<a onclick="showRef(event, 'ref13 ref17'); return false;" href="javascript:void(0);" class="ref ref13 ref17">(13,17)</a> It should be noted that while the biochemical potency of the compounds tested in this study is higher than the corresponding cell activity, the enzyme/cell correlation is very good (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00260/suppl_file/jm9b00260_si_001.pdf" class="ext-link">Supporting Information</a>) and the readout of the biochemical assay is a strong indicator of cell potency. In addition, none of the examples described in this current study showed any appreciable GLS2 activity in our hands.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Physicochemical, Biochemical, and Cell Potencies for Compounds <b>3</b> and <b>6</b>–<b>8</b><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0015.gif" alt="" id="fx22" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>3</b> (BPTES)</th><th class="colsep0 rowsep0" align="center"><b>6</b></th><th class="colsep0 rowsep0" align="center"><b>7</b></th><th class="colsep0 rowsep0" align="center"><b>8</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS1 enzyme IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.135</td><td class="colsep0 rowsep0" align="left">0.38</td><td class="colsep0 rowsep0" align="left">0.051</td><td class="colsep0 rowsep0" align="left">0.202</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS2 enzyme IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">>40</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">>40</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS PC-3 cell IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.028</td><td class="colsep0 rowsep0" align="left">0.086</td><td class="colsep0 rowsep0" align="left">0.004</td><td class="colsep0 rowsep0" align="left">0.029</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NCI-H1703 GI<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.794</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aq solubility (pH = 7.4, μM)</td><td class="colsep0 rowsep0" align="left"><0.6</td><td class="colsep0 rowsep0" align="left"><0.6</td><td class="colsep0 rowsep0" align="left"><0.4</td><td class="colsep0 rowsep0" align="left"><0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Log <i>D</i><sub>7.4</sub></td><td class="colsep0 rowsep0" align="left">>4</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">3.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand efficiency (LE)</td><td class="colsep0 rowsep0" align="left">0.22</td><td class="colsep0 rowsep0" align="left">0.2</td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="left">0.23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">lipophilic ligand efficiency</td><td class="colsep0 rowsep0" align="left"><3.6</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">3.9</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The biochemical assay employs the KGA GLS1 splice variant. The PC-3 mode of action cell assay detects cellular glutamate following cell lysis by means of the same glutamate oxidase/Amplex Red coupled enzyme assay used for biochemical GLS1 inhibition testing. The NCI-H1703 cell assay uses SYTOX Green to measure cell number. The assay employs the Acumen to give a 12-point duplicate dose–response curve using cryopreserved cells seeded at 400 cell/well and runs for 5 days post compound dosing. All cell IC<sub>50</sub> measurements reported are the geometric mean of at least two independent measurements unless otherwise indicated. Aqueous solubility measurement was performed under thermodynamic conditions using solid material obtained by evaporation from DMSO solution unless otherwise stated. Ligand efficiency (LE)<a onclick="showRef(event, 'cit18b'); return false;" href="javascript:void(0);" class="ref cit18b">(18b)</a> and lipophilic ligand efficiency (LLE) were calculated using PC-3 cell IC<sub>50</sub>. LLE<a onclick="showRef(event, 'cit18a'); return false;" href="javascript:void(0);" class="ref cit18a">(18a)</a> is calculated from cell pIC<sub>50</sub> – Log <i>D</i><sub>7.4</sub>.</p></div></div></div><div class="NLM_p">Although the truncated linker in <b>7</b> did offer options for further SAR exploration, the construction of the compounds in question proved relatively complex. With the goal of trying to simplify the chemistry used to prepare novel analogues, we next targeted examples bearing a diamine linker between the two thiazole groups (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Physicochemical, Biochemical, and Cell Mode of Action Potencies for Compounds <b>9</b>–<b>13</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0016.gif" alt="" id="fx23" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>9</b></th><th class="colsep0 rowsep0" align="center"><b>10</b></th><th class="colsep0 rowsep0" align="center"><b>11</b></th><th class="colsep0 rowsep0" align="center"><b>12</b></th><th class="colsep0 rowsep0" align="center"><b>13</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS1 enzyme IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.016</td><td class="colsep0 rowsep0" align="left">0.024</td><td class="colsep0 rowsep0" align="left">0.008</td><td class="colsep0 rowsep0" align="left">0.014</td><td class="colsep0 rowsep0" align="left">0.014</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS2 enzyme IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">>40</td><td class="colsep0 rowsep0" align="left">>40</td><td class="colsep0 rowsep0" align="left">>40</td><td class="colsep0 rowsep0" align="left">>40</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS PC-3 cell IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.011</td><td class="colsep0 rowsep0" align="left">0.002</td><td class="colsep0 rowsep0" align="left">0.001</td><td class="colsep0 rowsep0" align="left">0.003</td><td class="colsep0 rowsep0" align="left">0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aq solubility (pH = 7.4, μM)</td><td class="colsep0 rowsep0" align="left"><3</td><td class="colsep0 rowsep0" align="left"><1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><0.8</td><td class="colsep0 rowsep0" align="left"><0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Log <i>D</i><sub>7.4</sub></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">3.5</td><td class="colsep0 rowsep0" align="left">4.3</td><td class="colsep0 rowsep0" align="left">3.8</td><td class="colsep0 rowsep0" align="left">>4.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand efficiency</td><td class="colsep0 rowsep0" align="left">0.23</td><td class="colsep0 rowsep0" align="left">0.24</td><td class="colsep0 rowsep0" align="left">0.24</td><td class="colsep0 rowsep0" align="left">0.24</td><td class="colsep0 rowsep0" align="left">0.24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">lipophilic ligand efficiency</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">5.2</td><td class="colsep0 rowsep0" align="left">4.6</td><td class="colsep0 rowsep0" align="left">4.7</td><td class="colsep0 rowsep0" align="left">4.7</td></tr></tbody></table></div></div><div class="NLM_p">Our rationale here was that the heteroatoms would allow the use of much more facile S<sub>N</sub>Ar chemistry rather than the more involved coupling methods needed for compounds such as <b>7</b>, with the chiral pool also offering a number of diamine options. In addition, we hypothesized that this approach might also offer an opportunity to reduce the high Log <i>D</i> of the compounds, due to the more polar nature of the diamine linker compared to the corresponding carbon chain examples. As a consequence of this anticipated Log <i>D</i> reduction, we were optimistic that this might also improve both the physicochemical properties and the pharmacokinetic profiles of the compounds. Mindful of the improved biochemical potency observed with <b>7</b> bearing a four-atom spacer, we initially employed an ethylene diamine linker to give <b>9</b>. Pleasingly, the diaminoethyl linker in <b>9</b> did possess slightly increased potency compared to <b>3</b>, showing that the diamine linker was well tolerated while also validating the greater simplicity of the synthetic approach. We subsequently turned our attention to examine ways to rigidify and desymmetrize the linker motif of our GLS1 inhibitors. Restricting the conformational freedom of an inhibitor compound can lead to reduced loss of entropy when binding to a target and consequently improved potency.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Amine methylation (<b>10</b>) drove cell potency to a very high level, possibly due to reduced conformational freedom or fewer hydrogen-bond donors giving improved cell permeability, and in addition, cyclization of the linker system to a cyclopentane diamine delivered <b>11</b> with excellent cell potency. However, lipophilicity was still high, and we sought ways to maintain the beneficial aspects of the cyclic linker but with reduced Log <i>D</i>. One such solution could be to incorporate one of the nitrogen atoms of <b>11</b> into the ring to form a 3-aminopyrrolidine, thereby removing one methylene as well as one hydrogen-bond donor, an approach also adopted by McDermott et al.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Compound <b>12</b> did indeed deliver reduced Log <i>D</i> relative to cyclopentyl analogue <b>11</b>, although its PC-3 cell potency was also slightly reduced. It should be noted that the compound with opposite stereochemistry at the pyrrolidine center was less active (data not shown), and its asymmetry apparently fits well in only one way with the inversion point symmetry of the binding site. Interestingly, homologation of the linker of <b>12</b> by one methylene unit did now deliver <b>13</b> with very high potency, although this was at the expense of Log <i>D</i>, which was found to be at an unacceptably high level. Once more the stereochemistry of the pyrrolidine center was important, with the isomer shown being more active than the opposite isomer. In contrast, aminopyrrolidine <b>12</b> did seem to offer a good balance of potency, lipophilicity, and synthetic tractability, and we elected to focus our further optimization efforts on this template.</div><div class="NLM_p">A potential handle for the modulation of the physicochemical properties of <b>12</b> presented itself in the form of the two phenylacetic acid amide moieties, and we postulated that more polar functionality here might allow us to reduce the high lipophilicity of the compound.</div><div class="NLM_p">Replacement of one of the phenyl rings by a base (<b>14</b>) or more polar neutral groups (<b>15</b> and <b>16</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>)<named-content content-type="anchor" rid="tbl3" type="simple"></named-content> did indeed afford compounds with reduced Log <i>D</i> values. Solubility was enhanced by these changes, with the charged base in <b>14</b> having the greatest effect. The tetrahydropyran in <b>15</b> showed more modest Log <i>D</i> reduction and consequently the solubility increase was not as marked as for this compound. The acetamido piperidine example <b>16</b> did show a good reduction in Log <i>D</i>, and its more modest solubility improvement could perhaps be explained by an increased hydrogen-bonding network introduced by the additional amide, and thus greater crystal lattice energy offsetting the gains achieved by the lower Log <i>D</i>. Despite the progress made in improving the aqueous (aq) solubility of the series by incorporating these changes, an additional challenge now presented itself. While the intrinsic CACO2 permeability of BPTES was modest, the more polar functionality present in <b>14</b> and <b>15</b> appeared to exacerbate this issue, with very low levels of passive permeability being apparent. An alternative strategy to try and improve permeability and the physicochemical properties was to consider the molecular weight of the inhibitor compounds. It has been shown that increasing molecular weight correlates with a reduction in passive permeability, and truncation of compounds such as aminopyrrolidine <b>12</b> might offer improved permeability and the additional benefit of reduced Log <i>D</i>. We adopted a stepwise approach of molecular weight reduction and initially targeted compound <b>17</b>, where one of the terminal phenylacetamide groups was replaced by the simple acetamide. Although <b>17</b> did display slightly reduced potency in the PC-3 cell assay, both its ligand efficiency (LE) and lipophilic ligand efficiency (LLE) were comparable to <b>12</b>, driven by reduced Log <i>D</i> and lower molecular weight. The compound’s aqueous solubility was only slightly improved relative to <b>12</b>, however possibly indicating the key contribution to this parameter of hydrogen-bonding interactions in the solid state as well as lipophilicity. However, given the overall improved profile imparted by this change and the fact that good potency had been maintained, we elected to further reduce the size of the scaffold.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Physicochemical, Biochemical, and Cell Mode of Action Potencies for Compounds <b>14</b>–<b>16</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0017.gif" alt="" id="fx24" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>14</b></th><th class="colsep0 rowsep0" align="center"><b>15</b></th><th class="colsep0 rowsep0" align="center"><b>16</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS1 enzyme IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.244</td><td class="colsep0 rowsep0" align="left">0.007</td><td class="colsep0 rowsep0" align="left">0.099</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS2 enzyme IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS PC-3 cell IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.033</td><td class="colsep0 rowsep0" align="left">0.001</td><td class="colsep0 rowsep0" align="left">0.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aq solubility (pH = 7.4, μM)</td><td class="colsep0 rowsep0" align="left">150</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Log <i>D</i><sub>7.4</sub></td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="left">3.2</td><td class="colsep0 rowsep0" align="left">2.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CACO2 (A–B/B–A)</td><td class="colsep0 rowsep0" align="left">0.11/0.74</td><td class="colsep0 rowsep0" align="left"><0.04/18.4</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand efficiency</td><td class="colsep0 rowsep0" align="left">0.20</td><td class="colsep0 rowsep0" align="left">0.23</td><td class="colsep0 rowsep0" align="left">0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">lipophilic ligand efficiency</td><td class="colsep0 rowsep0" align="left">6.2</td><td class="colsep0 rowsep0" align="left">5.0</td><td class="colsep0 rowsep0" align="left">5.3</td></tr></tbody></table></div></div><div class="NLM_p">We again targeted the amide functionality of <b>17</b>, this time removing the carbonyl (and therefore a hydrogen-bond acceptor) to give aminothiadiazole <b>18</b>, or thiadiazole <b>19</b>. The binding mode of the truncated inhibitor <b>18</b> is remarkably similar to that of <b>12</b>, which has a benzyl substituent at both ends of the molecule (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). The terminal benzyl group of <b>18</b> exhibits a different rotamer, which can be explained by the difference in crystal symmetry and relative lack of contacts in this region steering the orientation of this group. The thiadiazole-amide groups bind in the usual way, but the aminopyrrolidine and terminal aminothiadiazole of <b>18</b> show a slightly different orientation, while maintaining the key hydrogen-bonding contacts to the backbone.</div><figure id="fig3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0003.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding mode of <b>12</b> (green, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KGAC">1KGAC</a>) compared to that of truncated inhibitor <b>18</b> (pink, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6KGAB">6KGAB</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Although once more we did observe a loss of biochemical potency, in both cases, cell potency was maintained, and both ligand efficiency and aqueous solubility were much improved relative to <b>17</b>. While we had made good initial progress in optimizing our GLS1 inhibitor template, efflux and permeability were perceived as key challenges to address, as can be seen for the data presented for <b>17</b>–<b>19</b> in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Physicochemical, Biochemical, and Cell Mode of Action Potencies for Compounds <b>17</b>–<b>19</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0018.gif" alt="" id="fx25" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>17</b></th><th class="colsep0 rowsep0" align="center"><b>18</b></th><th class="colsep0 rowsep0" align="center"><b>19</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS1 enzyme IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.07</td><td class="colsep0 rowsep0" align="left">0.347</td><td class="colsep0 rowsep0" align="left">0.873</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS2 enzyme IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">>40</td><td class="colsep0 rowsep0" align="left">>40</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS PC-3 cell IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.117</td><td class="colsep0 rowsep0" align="left">0.112</td><td class="colsep0 rowsep0" align="left">0.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aq solubility (pH = 7.4, μM)</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">95</td><td class="colsep0 rowsep0" align="left">130</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Log <i>D</i><sub>7.4</sub></td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">2.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CACO2 (A–B/B–A)</td><td class="colsep0 rowsep0" align="left"><0.07/13</td><td class="colsep0 rowsep0" align="left">0.53/13</td><td class="colsep0 rowsep0" align="left">0.4/26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand efficiency</td><td class="colsep0 rowsep0" align="left">0.23</td><td class="colsep0 rowsep0" align="left">0.26</td><td class="colsep0 rowsep0" align="left">0.28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">lipophilic ligand efficiency</td><td class="colsep0 rowsep0" align="left">4.4</td><td class="colsep0 rowsep0" align="left">5.0</td><td class="colsep0 rowsep0" align="left">5.2</td></tr></tbody></table></div></div><div class="NLM_p">At this stage, we had examined SAR around the linker and terminal regions, but we had not looked to investigate changes to the thiadiazole motifs, primarily as we were aware of the importance of the function of the two heterocyclic nitrogens as key hydrogen-bond acceptors in the binding of the compound to the GLS1 protein. Nevertheless, we postulated that a pyridazine ring might be able to bind to the target in a similar fashion and we next undertook the synthesis of compound <b>20</b> (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>)</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Physicochemical, Biochemical, and Cell Mode of Action Potencies for Compounds <b>20</b>–<b>22</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0019.gif" alt="" id="fx26" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."><b>20</b></th><th class="colsep0 rowsep0" align="center"><b>21</b></th><th class="colsep0 rowsep0" align="center"><b>22</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS1 enzyme IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="char" char=".">6.05</td><td class="colsep0 rowsep0" align="left">0.065</td><td class="colsep0 rowsep0" align="left">1.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS2 enzyme IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">>40</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS PC-3 cell IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="char" char=".">0.896</td><td class="colsep0 rowsep0" align="left">0.009</td><td class="colsep0 rowsep0" align="left">1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aq. solubility (pH = 7.4, μM)</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Log <i>D</i><sub>7.4</sub></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">0.85</td><td class="colsep0 rowsep0" align="left">1.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CACO2 (A–B/B–A)</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">0.7/5.3</td><td class="colsep0 rowsep0" align="left">0.09/9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand efficiency</td><td class="colsep0 rowsep0" align="char" char=".">0.22</td><td class="colsep0 rowsep0" align="left">0.29</td><td class="colsep0 rowsep0" align="left">0.22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">lipophilic ligand efficiency</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">7.2</td><td class="colsep0 rowsep0" align="left">4.1</td></tr></tbody></table></div></div><div class="NLM_p">Cell potency was rather poor for this compound, but this did prompt us to consider in more depth the nature of the aminopyrrolidine linker in <b>20</b>. By switching the orientation of the linker such that the tertiary nitrogen now joined the pyrazine, much greater cell potency was now achieved (<b>21</b>), and with a pronounced preference for the R pyrrolidine isomer over the S (data not shown).</div><div class="NLM_p">Intrigued by this finding, we also examined the same regiochemical switch in the bisthiadiazole series, leading us to prepare the pair of compounds <b>18</b> and <b>22</b>. In marked contrast to the pyridazine example <b>21</b>, the bisthiadiazole compounds showed a distinct potency preference to be linked, as shown in example <b>18</b>.</div><div class="NLM_p">We were encouraged by these initial pyridazine data, which showed that isosteric replacement of the thiadaizole with a pyridazine could lead to compounds with significant gains in LLE (i.e., compare <b>18</b> and <b>21</b>). Consequently, we then sought to incorporate the pyridazine motif as a thiadiazole isostere into compounds such as <b>18</b> and <b>19</b>, with the pyridazine now distal to the terminal phenylacetic acid, giving example <b>23</b>. Compound <b>23</b> proved to be a key breakthrough in potency, with excellent activity in cells being evident. In addition, for an essentially isolipophilic change relative to the bisthiadiazole example <b>22</b>, the incorporation of the pyridazine now led to a compound with both excellent ligand efficiency and improved LLE. In addition, the compound also displayed good activity in a phenotypic antiproliferative cell assay using NCI-H1703 cells, a line known to be highly dependent on glutamine for proliferation<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>).</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Physicochemical, Biochemical, and Cell Potencies for Compound <b>23</b><a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0020.gif" alt="" id="fx27" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>23</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS1 enzyme IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.017</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS2 enzyme IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS PC-3 cell IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NCI-H1703 GI<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.023</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aq solubility (pH = 7.4, (μM))</td><td class="colsep0 rowsep0" align="left">14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Log <i>D</i><sub>7.4</sub></td><td class="colsep0 rowsep0" align="left">2.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">% free (human)</td><td class="colsep0 rowsep0" align="left">18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human microsomes Cl<sub>int</sub> (μL/(min mg))</td><td class="colsep0 rowsep0" align="left">3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human hepatocytes Cl<sub>int</sub> (μL/(min 10<sup>6</sup> cells))</td><td class="colsep0 rowsep0" align="left"><1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat hepatocytes Cl<sub>int</sub> (μL/(min 10<sup>6</sup> cells))</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CACO2 (A–B/B–A)</td><td class="colsep0 rowsep0" align="left">0.5/23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand efficiency</td><td class="colsep0 rowsep0" align="left">0.33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">lipophilic ligand efficiency</td><td class="colsep0 rowsep0" align="left">6.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Plasma protein binding was assessed by equilibrium dialysis in the appropriate species at 37 °C in 10% plasma and extrapolated to 100% with a single-site binding model. Free and bound concentrations were quantified by liquid chromatography–mass spectrometry (LC–MS).</p></div></div></div><div class="NLM_p">With this encouraging in vitro profile in hand, we next elected to profile the pharmacokinetic parameters of <b>23</b> in vivo. Oral/intravenous (IV) dosing to Han Wistar rats showed high clearance (50 mL/(min kg)), modest volume of distribution (1.6 L/kg), and low oral bioavailability (2%). Although the modest oral exposure was not unexpected due to the low intrinsic permeability of the compound, the high clearance did not correlate with the low values observed in vitro. A similar study in the severe combined immunodeficient (SCID) mouse, but now employing a high oral dose of 100 mg/kg did in contrast deliver significant oral exposure, resulting in cover over the cell GI<sub>50</sub> for approximately 10 h (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) with the duration of cover limited by high plasma clearance (101 mL/(min kg) in the CD1 mouse).</div><figure id="fig4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0004.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Fold cover over cell GI<sub>50</sub> following 100 mg/kg oral dose of <b>23</b> to SCID mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Despite the encouraging progress made in optimizing the profile of <b>23</b>, challenges clearly remained around both in vivo clearance, solubility and permeability. To try and improve these aspects of <b>23</b>, we identified a number of structural changes that we postulated might prove beneficial toward this goal, with the amide group and terminal pyridazine seeming to offer likely points for variation. Moving from the pyridazine of <b>23</b> to the 1,2,4-triazine of <b>24</b> did have interesting effects. Although cell potency was negatively impacted by this change, permeability did notably improve (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). In vitro clearance seemed largely unaltered compared to <b>23</b>, but upon moving to oral dosing in rat, the oral exposure was much higher, despite plasma clearance remaining high. Reversal of the amide functionality of <b>23</b> gave <b>25</b>, which showed a further incremental improvement in permeability relative to <b>23</b> with a loss of potency, possibly due to differences in the hydrogen-bonding pattern. Again the fraction absorbed here was good with oral bioavailability being limited once more by high clearance. Of particular note is the additive effect of these changes on permeability, with the combination of triazine and reversed amide giving <b>26</b> with excellent permeability and little sign of efflux. However, although these changes did deliver progress against our permeability challenge and useful structure–activity/structure–property learning, the loss in potency observed meant that the compounds themselves would not realistically offer a way forward.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Physicochemical, Biochemical, and Cell Potencies for Compounds <b>24</b>–<b>26</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0021.gif" alt="" id="fx28" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>24</b></th><th class="colsep0 rowsep0" align="center"><b>25</b></th><th class="colsep0 rowsep0" align="center"><b>26</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS1 enzyme IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.207</td><td class="colsep0 rowsep0" align="left">0.114</td><td class="colsep0 rowsep0" align="left">0.59</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS2 enzyme IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">>40</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS PC-3 cell IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.011</td><td class="colsep0 rowsep0" align="left">0.032</td><td class="colsep0 rowsep0" align="left">0.238</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NCI-H1703 GI<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.46</td><td class="colsep0 rowsep0" align="left">0.43</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aq solubility (pH = 7.4, (μM))</td><td class="colsep0 rowsep0" align="left">94</td><td class="colsep0 rowsep0" align="left">>559</td><td class="colsep0 rowsep0" align="left">>228</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Log <i>D</i><sub>7.4</sub></td><td class="colsep0 rowsep0" align="left">2.06</td><td class="colsep0 rowsep0" align="left">2.24</td><td class="colsep0 rowsep0" align="left">2.20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">% free (human)</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human microsomes Cl<sub>int</sub> (μL/(min mg))</td><td class="colsep0 rowsep0" align="left"><3</td><td class="colsep0 rowsep0" align="left"><3</td><td class="colsep0 rowsep0" align="left">3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat hepatocytes Cl<sub>int</sub> (μL/(min 10<sup>6</sup> cells))</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">31</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CACO2 (A–B/B–A)</td><td class="colsep0 rowsep0" align="left">2.8/27</td><td class="colsep0 rowsep0" align="left">5/27</td><td class="colsep0 rowsep0" align="left">21/25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Han Wistar rat pharmacokinetics (Cl/<i>V</i><sub>dss</sub>/%<i>F</i>)</td><td class="colsep0 rowsep0" align="left">54/2/83 (2 mg/kg PO, 0.5 mg/kg IV)</td><td class="colsep0 rowsep0" align="left">100/2.7/12 (1 mg/kg PO, 0.5 mg/kg IV)</td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div></div><div class="NLM_p">We decided to employ library chemistry to explore the SAR scope of the amide functionality in <b>23</b>. From a library of approximately 40 compounds, two emerged as particularly interesting. Compound <b>27</b> (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>) with a benzylic methoxy group displayed excellent potency and much improved aqueous solubility compared to <b>23</b>. In addition, testing in CACO2 showed improved permeability, which in contrast to <b>26</b> did not come at the expense of reduced potency. A close analogue <b>28</b>, in which the benzylic position was substituted by a methyl rather than methoxy, was also of interest. Potency in both cell assays was good, although not quite at the same scale as <b>27</b>. Permeability was also improved relative to <b>23</b>, as was the case for <b>27</b>, but in a similar fashion to potency, the increase in the permeability was not as significant as for the latter compound. The stereochemistry was shown to be important for both compounds, with the opposite benzylic diastereoisomer showing reduced activity (data not shown).</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Physicochemical, Biochemical, and Cell Potencies for Compounds <b>27</b>–<b>29</b><a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0022.gif" alt="" id="fx29" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>27</b></th><th class="colsep0 rowsep0" align="center"><b>28</b></th><th class="colsep0 rowsep0" align="center"><b>29</b></th><th class="colsep0 rowsep0" align="center"><b>5</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS1 enzyme IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.021</td><td class="colsep0 rowsep0" align="left">0.01</td><td class="colsep0 rowsep0" align="left">0.063</td><td class="colsep0 rowsep0" align="left">0.007</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS1 enzyme IC<sub>50</sub> (GAC splice variant, μM)</td><td class="colsep0 rowsep0" align="left">0.051</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS PC-3 cell IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.0003</td><td class="colsep0 rowsep0" align="left">0.0007</td><td class="colsep0 rowsep0" align="left">0.004</td><td class="colsep0 rowsep0" align="left">0.0002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NCI-H1703 GI<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.011</td><td class="colsep0 rowsep0" align="left">0.015</td><td class="colsep0 rowsep0" align="left">0.054</td><td class="colsep0 rowsep0" align="left">0.008</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aq solubility (pH = 7.4, (μM))</td><td class="colsep0 rowsep0" align="left">78 (crystalline)</td><td class="colsep0 rowsep0" align="left">329 (amorphous)</td><td class="colsep0 rowsep0" align="left">516 (amorphous)</td><td class="colsep0 rowsep0" align="left">0.32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Log <i>D</i><sub>7.4</sub></td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">2.6</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">3.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">% free (human)</td><td class="colsep0 rowsep0" align="left">9.4</td><td class="colsep0 rowsep0" align="left">13.4</td><td class="colsep0 rowsep0" align="left">6.6</td><td class="colsep0 rowsep0" align="left">0.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human microsomes Cl<sub>int</sub> (μL/(min mg))</td><td class="colsep0 rowsep0" align="left"><3</td><td class="colsep0 rowsep0" align="left"><3</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human hepatocytes Cl<sub>int</sub> (μL/(min 10<sup>6</sup> cells))</td><td class="colsep0 rowsep0" align="left"><1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat hepatocytes Cl<sub>int</sub> (μL/(min 10<sup>6</sup> cells))</td><td class="colsep0 rowsep0" align="left">4.6</td><td class="colsep0 rowsep0" align="left">8.7</td><td class="colsep0 rowsep0" align="left">4.1</td><td class="colsep0 rowsep0" align="left">30.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CACO2 (A–B/B–A)</td><td class="colsep0 rowsep0" align="left">2.2/24</td><td class="colsep0 rowsep0" align="left">1.1/34</td><td class="colsep0 rowsep0" align="left">8/25</td><td class="colsep0 rowsep0" align="left">0.2/4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat pharmacokinetics (Cl/<i>V</i><sub>dss</sub>/%<i>F</i>)</td><td class="colsep0 rowsep0" align="left">35/2.4/47</td><td class="colsep0 rowsep0" align="left">55/1.9/0</td><td class="colsep0 rowsep0" align="left">12/0.6/41</td><td class="colsep0 rowsep0" align="left">14/3.0/32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dog pharmacokinetics (Cl/<i>V</i><sub>dss</sub>/%<i>F</i>)</td><td class="colsep0 rowsep0" align="left">4/0.3/60 (1 mg/kg PO, 0.5 mg/kg IV)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1/0.15/90 (1 mg/kg PO, 0.5 mg/kg IV)</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hERG IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">>33.3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">>33.3</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand efficiency</td><td class="colsep0 rowsep0" align="left">0.33</td><td class="colsep0 rowsep0" align="left">0.33</td><td class="colsep0 rowsep0" align="left">0.29</td><td class="colsep0 rowsep0" align="left">0.24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">lipophilic ligand efficiency</td><td class="colsep0 rowsep0" align="left">7.6</td><td class="colsep0 rowsep0" align="left">6.6</td><td class="colsep0 rowsep0" align="left">6.6</td><td class="colsep0 rowsep0" align="left">5.8</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">The hERG assay was measured in human Chinese hamster ovary cells using IonWorks HT technology platform.</p></div></div></div><div class="NLM_p">On dosing to Han Wistar rats, <b>27</b> showed reduced in vivo clearance relative to <b>23</b> and improved bioavailability, presumably as a consequence of increased fraction absorbed combined with reduced plasma clearance. In contrast, <b>28</b> did not give any oral exposure in the rat, possibly as a consequence of high clearance combined with modest fraction absorbed. Nevertheless, both compounds were examined at high dose orally in SCID mouse (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>), with <b>27</b> achieving 24 h cover over the free proliferation GI<sub>50</sub>. In contrast, <b>28</b> achieved cover to approximately 15 h, with the possibility of 24 h cover potentially being delivered by a twice a day (BID) dosing schedule.</div><figure id="fig5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0005.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Fold free cover over cell GI<sub>50</sub> following 100 mg/kg oral dose of <b>27</b> and <b>28</b> to SCID mice. Compound <b>29</b> was dosed to nude mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Given the permeability additivity observed in our earlier work (vide supra), we were keen to see if this would be applicable in the case of <b>27</b>. Moving from the pyridazine in <b>27</b> to a 1,2,4-triazine gave <b>29</b> (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>). This compound was less potent in cell than <b>27</b>, but the combined impact of the benzylic methoxy and the triazine was evident in the CACO2 A–B data. Rat PK studies also revealed that a combination of low clearance, high permeability, and low efflux afforded an excellent oral profile that was recapitulated when the compound was dosed in mouse, and as was the case for <b>27</b>, <b>29</b> was seen to deliver excellent cover over the free proliferation GI<sub>50</sub>, almost to the 24 h time point (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Given the attractive pharmacokinetic profiles of <b>27</b> and <b>29</b> in rodents, we elected to further understand their properties by moving to studies in dog. Again, excellent oral exposure was observed, with low plasma clearance, although the volume of distribution observed in each case was relatively low. The profile of compound <b>5</b> is also included in <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a> for comparative purposes. It can be seen that in both biochemical and cellular assays, <b>5</b> shows a slightly higher potency relative to <b>27</b>. However, the higher Log <i>D</i> of the former leads to significantly higher plasma protein binding. As a consequence, much higher levels of <b>5</b> will be required in vivo to achieve cover over the cell end points. In addition, this high level of lipophilicity also leads to modest aqueous solubility for <b>5</b>, which can have an impact on the maximum absorbable dose when moving to the clinic. Compound <b>5</b> exhibits generally much higher intrinsic clearance (Cl<sub>int</sub>) than <b>27</b> and related examples, and this difference will be compounded further when considering unbound clearance, again due to the large differences in free fraction between <b>5</b> and <b>27</b>. When comparing the low-dose rat pharmacokinetic profiles of <b>5</b> and <b>27</b>, it can be seen that the oral bioavailability is modest for both compounds, with <b>27</b> offering the higher figure. Finally, when examining ligand efficiency and ligand lipophilicity efficiency, the lower Log <i>D</i> and smaller molecular weight of <b>27</b> mean that it appears superior to <b>5</b> when using these measures, despite the compound’s similar potencies. The crystal structure of <b>27</b> shows again a tetramer, and compound <b>27</b> can bind in two symmetry-related binding modes in the pocket formed by two of the four protein chains (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a). Overlaying the crystal structures of <b>3</b> and <b>27</b> bound to GLS1 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b), it can be seen that the symmetric binding modes of <b>27</b> occupy the same space as occupied by <b>3</b>, and the spatial orientation of key heteroatoms involved in hydrogen bonding to the protein is maintained, but in the case of <b>27</b>, this is achieved by using a more rigid and less entropically costly scaffold. Slight differences in positioning can be observed in the benzyl and α-methoxy benzyl termini of <b>3</b> and <b>27</b>. The chiral terminus of the latter is sterically more constrained than the benzyl, offering an entropic advantage. Additionally, <b>27</b> achieves a more rigid binding mode by employing the aminopyrrolidine scaffold, which is much less flexible than the diethylthioether linker of <b>3</b>, again gaining an entropic advantage over <b>3</b>.</div><figure id="fig6" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0006.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Overlay of the X-ray crystal structures of <b>3</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGAA">3KGAA</a>) and <b>27</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KGAD">7KGAD</a>) bound to GLS1. (a) Two symmetry-related binding modes (orange and cyan) of <b>27</b>. Symmetry-related chains shown in gray and yellow. (b) Overlay of binding modes of one molecule of <b>27</b> (cyan) and <b>3</b> (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">With a variety of potent and orally well-exposed compounds in hand, we next turned our attention to in vivo studies and how we might employ these inhibitors to investigate target engagement and its effect on tumor growth inhibition. Compound <b>27</b> was dosed once a day by oral gavage for 21 days at 100 mg/kg to SCID mice implanted with NCI-H1703 non-small cell lung cancer cells and tumor volumes were monitored. In addition, on day 21 of the study, the amino acid pharmacodynamic biomarkers glutamine, glutamate, and aspartate were measured in the tumors over a 24 h period following oral dosing of compound <b>27</b> at 100 mg/kg. Following inhibition of GLS1, levels of glutamine would be expected to rise as the enzyme’s substrate accumulates. Equally, levels of the GLS1-catalyzed hydrolysis product glutamate would be expected to fall, as should also be the case for the more distal biomarker aspartate, which results from aspartate transaminase-catalyzed conversion of glutamate and oxaloacetate to aspartate and α-ketoglutarate. Taxotere was also included in the studies to act as a positive control, with a once weekly intravenous dose of 7.5 mg/kg being employed. The results of these studies are shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>. As before, significant exposure of <b>27</b> was achieved (measured on day 21), with continuous free cover over the free proliferation IC<sub>50</sub> being achieved out to the 24 h time point (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>c). Turning first to the end-of-study pharmacodynamic data, it can be seen that levels of tumor glutamine were profoundly affected by oral dosing of <b>27</b> and, as would be expected from the compounds mode of action, glutamine levels were seen to rise while both glutamate and aspartate levels were reduced. At both 4 and 24 h following oral administration, tumor glutamine levels were doubled relative to control, with taxotere having minimal impact. In the case of glutamate, tumor levels are reduced following dosing of <b>27</b>, with the compound having a greater impact than taxotere at 7.5 mg/kg. Finally, levels of aspartate in tumor were again seen to be impacted by administration of <b>27</b>, with levels being reduced relative to control, and once more with a greater effect than use of taxotere, particularly at the 24 h time point. In the tumor growth inhibition study, <b>27</b> was seen to be efficacious when dosed orally once a day at 100 mg/kg, with 37% TGI visible by day 21. The maximum tumor growth inhibition of 73% (<i>P</i> < 0.001) was observed on day 12 of the study. Taxotere dosed once weekly IV at 7.5 mg/kg was also efficacious, delivering tumor stasis by day 21.</div><figure id="fig7" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0007.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (a) Pharmacodynamic data for compound <b>27</b> showing effects on tumor amino acid levels. (b) Tumor growth inhibition study using oral dosing of <b>27</b> and taxotere. See text for further details. (c) Study exposure data and free cover over cell GI<sub>50</sub> calculations. Compound <b>27</b> was measured as 16.3% free in SCID mouse plasma protein. Statistical analysis performed by one-tailed, unequal variance <i>t</i> test vs vehicle control at day 21. ** <i>P</i> < 0.01, *** <i>P</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To further understand the broader selectivity profile of <b>27</b> we also undertook wider screening in several secondary pharmacology panels. In particular, when tested at a compound concentration of 1 μM in a panel of 125 kinases, very little activity was observed with 29% inhibition against MET being the most potent hit. Testing in a secondary pharmacology panel of 157 assays also identified little off-target activity, with modest activity seen against the norepinephrine transporter (IC<sub>50</sub> = 5.3 μM), nicotinic acetylcholine receptor (IC<sub>50</sub> = 18 μM), 5-hydroxytryptamine receptor 2B (13 μM), opioid receptor Mu1 (IC<sub>50</sub> = 60 μM), neurokinin receptor 1 (IC<sub>50</sub> = 71 μM), and dopamine transporter (IC<sub>50</sub> = 61 μM). All other results in the panel were >30 or >100 μM. In addition, we also studied the profile of <b>27</b> in the GAC GLS1 splice variant with the compound showing a very similar level of potency compared to the KGA variant (51 and 21 nM, respectively).</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23505" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23505" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Herein, we have described the evolution of an allosteric glutaminase 1 inhibitor series, starting from a cell-active but lipophilic literature lead with poor pharmacokinetic properties and culminating in the discovery of a cell potent, soluble, and orally exposed inhibitor that engages the target in vivo and displays tumor growth activity in clinically relevant disease models. Key challenges addressed en route to <b>27</b> included potency improvements, overcoming efflux and permeability challenges while also reducing in vivo clearance.</div><div class="NLM_p last">Despite continued interest in GLS1 as an oncology target and several groups reporting inhibitors displaying high levels of phenotypic antiproliferative potency, very few single-agent in vivo studies culminating in high levels of tumor growth inhibition have been disclosed in the literature. As can be seen in the current study, this has also been our experience when studying GLS1 inhibitors in vivo. It remains unclear whether this is due to compensatory effects in the tumor or indeed if the most relevant in vivo models remain to be identified. Nevertheless, we remain optimistic that GLS1 inhibition has a role in the treatment of cancer and further studies on <b>27</b> and related analogues will be disclosed in due course.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84454" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84454" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> General</h3><div class="NLM_p last">All solvents and chemicals used were of reagent grade. Purity and characterization of compounds were established by a combination of LC–MS and NMR techniques. LC–MS spectra were obtained using a Waters liquid chromatography–mass spectrometry system, where purity was determined by UV absorption at a wavelength of 254 nm, and the mass ion was determined by electrospray ionization (ESI, Micromass Instrument). All test compounds were of >95% purity, as assessed by LC–MS and <sup>1</sup>H NMR spectroscopies. <sup>1</sup>H NMR spectra were recorded using a Bruker Avance 400 FT spectrometer or via Flow NMR process using an Avance 500 FT spectrometer, and using DMSO-<i>d</i><sub>6</sub> or CDCl<sub>3</sub> with the data expressed as chemical shifts in parts per million from internal standard tetramethylsilane. Preparative high-performance liquid chromatography (HPLC) was performed on a Waters or Phenomenex column using decreasingly polar mixtures of water (containing 1% formic acid or 1% aqueous NH<sub>4</sub>OH) and acetonitrile. Purification by flash column chromatography (FCC) was typically performed using silica gel (Merck 7734 grade), and solvent mixtures and gradients are recorded herein. Water present in compounds was quantified by Karl Fischer analysis. All reactions were performed under nitrogen at room temperature (rt) unless otherwise stated.</div></div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> <i>N</i>,<i>N</i>′-(5,5′-(Ethane-1,2-diylbis(azanediyl))bis(1,3,4-thiadiazole-5,2-diyl))bis(2-phenylacetamide) (<b>9</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0024.gif" alt="" id="fx1" /></img><div id="sec5_1_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> <i>tert</i>-Butyl (2-((5-(2-Phenylacetamido)-1,3,4-thiadiazol-2-yl)amino)ethyl)carbamate</h4><div class="NLM_p last">5-Bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (1.0 g, 5.55 mmol), <i>tert</i>-butyl (2-aminoethyl)carbamate (0.934 g, 5.83 mmol), and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (1.261 mL, 7.22 mmol) were combined in acetonitrile (15 mL) and the mixture was stirred at 75 °C for 2 h. The reaction was then allowed to cool to rt. Solvent was removed in vacuo. The residue was suspended in a mixture of DCM (20 mL) and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (1.5 mL) before 2-phenylacetyl chloride (1 mL, 7.57 mmol) was added dropwise. The mixture was stirred for 3 h. A solid was filtered off to provide the title compound (1.76 g, 84%). The filtrate was concentrated in vacuo and the solid was filtered off to give a second batch of the title compound (0.36 g, 17%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.36 (s, 9H), 3.10 (q, <i>J</i> = 6.2 Hz, 2H), 3.25 (q, <i>J</i> = 6.3 Hz, 2H), 3.70 (s, 2H), 6.87 (t, <i>J</i> = 5.4 Hz, 1H), 7.20–7.38 (m, 6H), 12.15 (s, 1H); MS (ESI) [M – H]<sup>−</sup><i>m</i>/<i>z</i> 376.</div></div><div id="sec5_1_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> <i>N</i>-(5-((2-Aminoethyl)amino)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide</h4><div class="NLM_p last">TFA (5 mL) was added to <i>tert</i>-butyl (2-((5-(2-phenylacetamido)-1,3,4-thiadiazol-2-yl)amino)ethyl)carbamate (1.75 g, 4.64 mmol) in DCM (20 mL) and was stirred at rt for 1 h. The mixture was concentrated in vacuo, and the residue was purified by ion-exchange chromatography, using an SCX-2 column. The desired product was eluted from the column using a 1:1 mixture of 7 M NH<sub>3</sub>/methanol:DCM, and pure fractions were evaporated to dryness to afford the title compound (1.18 g, 92%) as a solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.71 (2H, t), 3.19–3.25 (2H, m), 3.70 (2H, s), 4.99 (2H, s), 7.15–7.48 (6H, m); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 278.</div></div><div id="sec5_1_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>N</i>-(5-((2-((5-Amino-1,3,4-thiadiazol-2-yl)amino)ethyl)amino)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide</h4><div class="NLM_p last"><i>N</i>-(5-((2-Aminoethyl)amino)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide (215 mg, 0.776 mmol), 5-bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (168 mg, 0.93 mmol), and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.54 mL, 3.10 mmol) were suspended in acetonitrile (4 mL). The reaction was heated to 130 °C for 1 h in a microwave reactor, then cooled to rt. The resulting suspension was filtered off and the solid was washed with acetonitrile. The solid was treated with methanol (3 mL) and heated to 100 °C for 10 min in a microwave reactor. After cooling, a solid was removed by filtration and washed. All of the filtrates were combined and concentrated in vacuo. The residue was purified by flash silica chromatography, elution gradient 0–10% 7 M NH<sub>3</sub>/methanol in DCM. Pure fractions were evaporated to dryness to afford the title compound (80 mg, 27%) as a solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.41–3.45 (2H, m), 3.47 (2H, m), 3.77 (2H, s), 6.66 (1H, s), 6.81 (2H, s), 7.30–7.41 (5H, m), 7.45 (1H, t, <i>J</i> = 5.3 Hz), 12.19 (1H, s); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 377.</div></div><div id="sec5_1_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>N</i>,<i>N</i>′-(5,5′-(Ethane-1,2-diylbis(azanediyl))bis(1,3,4-thiadiazole-5,2-diyl))bis(2-phenylacetamide) (<b>9</b>)</h4><div class="NLM_p last">2-Phenylacetyl chloride (0.051 mL, 0.38 mmol) was added dropwise to a stirred mixture of <i>N</i>-(5-((2-((5-amino-1,3,4-thiadiazol-2-yl)amino)ethyl)amino)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide (80 mg, 0.21 mmol) and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.11 mL, 0.64 mmol) in DCM (3 mL). The mixture was stirred for 30 min at rt, then quenched with methanol and concentrated in vacuo. The residue was triturated under methanol and a solid was filtered off. The solid was transferred to a microwave vial and methanol (1 mL) was added. The mixture was heated at 100 °C for 15 min in a microwave reactor. This was then allowed to cool to rt and the solid was filtered off. This recrystallization procedure was repeated twice to provide the title compound (15 mg, 14%) as a tan solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.42–3.47 (4H, m), 3.71 (4H, s), 7.22–7.37 (10H, m), 7.43 (2H, s), 12.17 (2H, s); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 41.41, 42.91, 126.74, 128.33, 129.16, 134.90, 148.64, 164.28, 168.37; HRMS (ES<sup>+</sup>) for C<sub>22</sub>H<sub>23</sub>N<sub>8</sub>O<sub>2</sub>S<sub>2</sub> [M + H]<sup>+</sup>: calcd, 495.1385; found, 495.1403.</div></div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> <i>N</i>,<i>N</i>′-(5,5′-(Ethane-1,2-diylbis(methylazanediyl))bis(1,3,4-thiadiazole-5,2-diyl))bis(2-phenylacetamide) (<b>10</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0025.gif" alt="" id="fx2" /></img><div class="NLM_p last"><i>N</i>1,<i>N</i>2-Dimethylethane-1,2-diamine (45 mg, 0.51 mmol), 5-bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (184 mg, 1.02 mmol), and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.27 mL, 1.53 mmol) in acetonitrile (4 mL) were heated to 100 °C for 1 h in a microwave reactor. The reaction was then cooled to rt. The reaction mixture was diluted with methanol and the mixture was concentrated in vacuo. The mixture was then azeotroped with DCM. The residue was redissolved in a mixture of DMF (1 mL), DCM (4 mL), and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.4 mL). 2-Phenylacetyl chloride (0.182 mL, 1.38 mmol) was added dropwise to this mixture and stirred for 15 min at rt. This was then diluted with methanol, then concentrated in vacuo. The residue was triturated under methanol. The resulting solid was filtered off and washed with methanol to afford the title compound (60 mg, 22.5%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.01 (6H, s), 3.66 (4H, s), 3.71 (4H, s), 7.20–7.36 (10H, m), 12.15 (2H, s); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 39.73, 41.37, 50.05, 126.75, 128.33, 129.12, 134.86, 148.80, 166.51, 168.35; HRMS (ES<sup>+</sup>) for C<sub>24</sub>H<sub>27</sub>N<sub>8</sub>O<sub>2</sub>S<sub>2</sub> [M + H]<sup>+</sup>: calcd, 523.1698; found, 523.1717.</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> <i>N</i>,<i>N</i>′-((((1<i>R</i>,2<i>R</i>)-Cyclopentane-1,2-diyl)bis(azanediyl))bis(1,3,4-thiadiazole-5,2-diyl))bis(2-phenylacetamide) (<b>11</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0026.gif" alt="" id="fx3" /></img><div class="NLM_p last">5-Bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (62.4 mg, 0.35 mmol) was added to a stirred solution of (1<i>R</i>,2<i>R</i>)-cyclopentane-1,2-diamine dihydrochloride (30 mg, 0.17 mmol) and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.3 mL, 1.72 mmol) in DMF (1 mL). The resulting mixture was stirred at 80 °C for 100 min. After cooling to rt, 2-phenylacetyl chloride (0.046 mL, 0.35 mmol) was added dropwise. The mixture was stirred for 15 min at rt. The reaction mixture was quenched with methanol. The mixture was then concentrated in vacuo and the residue was purified by preparative HPLC (Waters XSelect CSH C18 column, 30 × 100 mm<sup>2</sup> id, 5 μm particle size), using decreasingly polar mixtures of water (containing 0.1% formic acid) and acetonitrile as eluents. Fractions containing the desired compound were evaporated to dryness to afford the title compound (10.7 mg, 12%) as a solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.59 (2H, dd, <i>J</i> = 6.7, 13.2 Hz), 1.72 (2H, p, <i>J</i> = 7.2 Hz), 2.14 (2H, dd, <i>J</i> = 6.6, 12.7 Hz), 3.73 (4H, s), 3.95 (2H, s), 7.18 (2H, s), 7.22–7.43 (10H, m), 11.72 (2H, br); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 20.72, 30.14, 41.41, 60.40, 126.73, 128.33, 129.15, 134.92, 148.62, 163.98, 168.37; HRMS (ES<sup>+</sup>) for C<sub>25</sub>H<sub>27</sub>N<sub>8</sub>O<sub>2</sub>S<sub>2</sub> [M + H]<sup>+</sup>: calcd, 535.1698; found, 535.1703.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> (<i>R</i>)-2-Phenyl-<i>N</i>-(5-(3-((5-(2-phenylacetamido)-1,3,4-thiadiazol-2-yl)amino)pyrrolidin-1-yl)-1,3,4-thiadiazol-2-yl)acetamide (<b>12</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0027.gif" alt="" id="fx4" /></img><div id="sec5_1_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (<i>R</i>)-<i>tert</i>-Butyl 3-((5-Amino-1,3,4-thiadiazol-2-yl)amino)pyrrolidine-1-carboxylate</h4><div class="NLM_p last">Potassium carbonate (2.67 g, 19.33 mmol) was added to 5-bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (2.90 g, 16.11 mmol) and (<i>R</i>)-<i>tert</i>-butyl 3-aminopyrrolidine-1-carboxylate (<b>31</b>) (3.00 g, 16.11 mmol) in acetonitrile (50 mL) at rt. The resulting mixture was stirred at 80 °C for 6 h. The reaction mixture was allowed to cool to rt. This was then diluted with ethyl acetate and water and the solid was filtered off. The organic layer was separated, dried over MgSO<sub>4</sub>, filtered, and evaporated to afford crude product. This was combined with the solid isolated before and was washed with diethyl ether and dried under vacuum to afford the title compound (4.05 g, 88%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.40 (9H, s), 1.73–1.93 (1H, br), 1.97–2.16 (1H, br), 3.13–3.23 (1H, br), 3.24–3.39 (2H, m), 3.40–3.58 (1H, br), 3.98–4.12 (1H, br), 6.24 (2H, s), 6.96 (1H, s); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 286.</div></div><div id="sec5_1_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (<i>R</i>)-<i>tert</i>-Butyl 3-((5-(2-Phenylacetamido)-1,3,4-thiadiazol-2-yl)amino)pyrrolidine-1-carboxylate</h4><div class="NLM_p last">2-Phenylacetyl chloride (2.29 mL, 17.32 mmol) was added dropwise to a stirred mixture of (<i>R</i>)-<i>tert</i>-butyl 3-((5-amino-1,3,4-thiadiazol-2-yl)amino)pyrrolidine-1-carboxylate (4.94 g, 17.32 mmol) and potassium carbonate (1.81 g, 13.09 mmol) in THF (100 mL). The resulting mixture was stirred at rt for 18 h. Water (60 mL) was added and the product was extracted with ethyl acetate (3 × 70 mL). The combined organic extracts were washed with brine (10 mL), dried over MgSO<sub>4</sub>, and filtered. The solvent was then removed in vacuo to afford the title compound (7.16 g, quantitative) as a pale yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.40 (9H, s), 1.85–1.98 (1H, br), 2.07–2.23 (1H, br), 3.19–3.41 (3H, br), 3.47–3.60 (1H, br), 3.78 (2H, s), 4.17–4.32 (1H, br), 7.21–7.38 (5H, m), 8.66–9.21 (1H, br), 12.38–12.72 (1H, br); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 304.</div></div><div id="sec5_1_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (<i>R</i>)-2-Phenyl-<i>N</i>-(5-(pyrrolidin-3-ylamino)-1,3,4-thiadiazol-2-yl)acetamide (<b>32</b>)</h4><div class="NLM_p last">TFA (6 mL) was added to a solution of (<i>R</i>)-<i>tert</i>-butyl 3-((5-(2-phenylacetamido)-1,3,4-thiadiazol-2-yl)amino)pyrrolidine-1-carboxylate (2.3 g, 5.70 mmol) in DCM (30 mL) and the mixture was stirred at rt for 1 h. The reaction mixture was evaporated to dryness and the residue was purified by ion-exchange chromatography, using an SCX-2 column. The desired product was eluted from the column using 7 M NH<sub>3</sub>/methanol to afford the title compound (1.7 g, 98%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.51–1.63 (1H, m), 1.91–2.03 (1H, m), 2.64 (1H, dd, <i>J</i> = 4.1, 11.3 Hz), 2.74 (1H, ddd, <i>J</i> = 5.6, 8.2, 10.8 Hz), 2.86 (1H, dt, <i>J</i> = 7.3, 10.7 Hz), 2.98 (1H, dd, <i>J</i> = 6.2, 11.3 Hz), 3.71 (2H, s), 3.95–4.06 (1H, m), 7.23–7.37 (6H, m); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 404.</div></div><div id="sec5_1_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (<i>R</i>)-2-Phenyl-<i>N</i>-(5-(3-((5-(2-phenylacetamido)-1,3,4-thiadiazol-2-yl)amino)pyrrolidin-1-yl)-1,3,4-thiadiazol-2-yl)acetamide (<b>12</b>)</h4><div class="NLM_p last">(<i>R</i>)-2-Phenyl-<i>N</i>-(5-(pyrrolidin-3-ylamino)-1,3,4-thiadiazol-2-yl)acetamide (<b>32</b>) (482 mg, 1.59 mmol), 5-bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (286 mg, 1.59 mmol), and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.83 mL, 4.77 mmol) were dissolved in DMF (5 mL) and stirred at 100 °C for 30 min, then allowed to cooled to rt. The reaction mixture (2 mL) was removed, and to the remaining reaction mixture, 2-phenylacetyl chloride (0.178 mL, 1.35 mmol) was added dropwise. The mixture was stirred for 15 min at rt. The reaction mixture was quenched with methanol. The solid was filtered off and washed with methanol and diethyl ether to afford the title compound (195 mg, 39%) as a solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.05 (1H, d), 2.24–2.39 (1H, m), 3.39 (1H, dd), 3.42–3.57 (2H, m), 3.69 (1H, d), 3.72 (2H, s), 3.73 (2H, s), 4.35 (1H, d), 7.21–7.39 (10H, m), 7.62 (1H, d), 12.16 (1H, s), 12.20 (1H, s); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 30.82, 41.42, 47.97, 53.51, 55.11, 126.75, 126.76, 128.34, 128.35, 129.12, 129.18, 134.87, 148.44, 149.08, 163.35, 163.79, 168.45, 168.50; HRMS (ES<sup>+</sup>) for C<sub>24</sub>H<sub>25</sub>N<sub>8</sub>O<sub>2</sub>S<sub>2</sub> [M + H]<sup>+</sup>: calcd, 570.2070; found, 570.2067.</div></div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> (<i>R</i>)-2-Phenyl-<i>N</i>-(5-(3-(((5-(2-phenylacetamido)-1,3,4-thiadiazol-2-yl)amino)methyl)pyrrolidin-1-yl)-1,3,4-thiadiazol-2-yl)acetamide (<b>13</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0028.gif" alt="" id="fx5" /></img><div id="sec5_1_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (<i>S</i>)-2-Phenyl-<i>N</i>-(5-((pyrrolidin-3-ylmethyl)amino)-1,3,4-thiadiazol-2-yl)acetamide</h4><div class="NLM_p last">5-Bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (449 mg, 2.5 mmol) was added to a solution of (<i>R</i>)-<i>tert</i>-butyl 3-(aminomethyl)pyrrolidine-1-carboxylate (<b>33</b>) (500 mg, 2.5 mmol) and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (1.31 mL, 7.5 mmol) in DCM (15 mL). The reaction mixture was heated to 60 °C for 2 h in a microwave reactor, then allowed to cool to rt. 2-Phenylacetyl chloride (0.412 mL, 3.12 mmol) was added dropwise to the stirred reaction mixture, which was then stirred for 30 min at rt. TFA (5 mL) was added to the mixture, which was then stirred for 1 h. The mixture was then concentrated in vacuo. The residue was purified by ion-exchange chromatography, using an SCX-2 column. The desired product was eluted from the column using 7 M NH<sub>3</sub>/methanol, and pure fractions were evaporated to dryness to afford the title compound (855 mg, 108%) as a foam. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.32–1.44 (1H, m), 1.77–1.90 (1H, m), 2.31–2.41 (2H, m), 2.56 (1H, dd, <i>J</i> = 6.2, 10.8 Hz), 2.79 (1H, dt, <i>J</i> = 7.5, 10.8 Hz), 2.84–2.96 (2H, m), 3.14–3.23 (2H, m), 3.71 (2H, s), 7.15–7.39 (6H, m); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 318.</div></div><div id="sec5_1_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (<i>R</i>)-2-Phenyl-<i>N</i>-(5-(3-(((5-(2-phenylacetamido)-1,3,4-thiadiazol-2-yl)amino)methyl)pyrrolidin-1-yl)-1,3,4-thiadiazol-2-yl)acetamide (<b>13</b>)</h4><div class="NLM_p last">(<i>S</i>)-2-Phenyl-<i>N</i>-(5-((pyrrolidin-3-ylmethyl)amino)-1,3,4-thiadiazol-2-yl)acetamide (105 mg, 0.33 mmol), 5-bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (60 mg, 0.33 mmol), and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.175 mL, 1.00 mmol) were dissolved in DMF (1 mL). The reaction was then stirred at 100 °C for 30 min. This was then allowed to cool to rt. 2-Phenylacetyl chloride (0.148 mL, 1.12 mmol) was added dropwise and the mixture was stirred for 15 min at rt. The reaction was then quenched with methanol. The mixture was concentrated in vacuo and the residue was purified by flash silica chromatography, elution gradient 0–10% methanol in DCM to afford the crude product, which was recrystalized from methanol to afford the title compound (76 mg, 43%) as a solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.25 (1H, s), 1.77 (1H, m), 2.03–2.15 (1H, m), 2.61–2.73 (1H, m), 3.11–3.19 (1H, m), 3.33–3.54 (4H, m), 3.70 (4H, d, <i>J</i> = 5.4 Hz), 7.20–7.37 (10H, m), 7.45 (1H, t, <i>J</i> = 5.6 Hz), 12.13 (1H, s), 12.19 (1H, s); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 535.</div></div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> (<i>R</i>)-2-(1-Methylpiperidin-4-yl)-<i>N</i>-(5-(3-((5-(2-phenylacetamido)-1,3,4-thiadiazol-2-yl)amino)pyrrolidin-1-yl)-1,3,4-thiadiazol-2-yl)acetamide (<b>14</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0029.gif" alt="" id="fx6" /></img><div class="NLM_p last">1-[Bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (482 mg, 1.27 mmol) was added portionwise to (<i>R</i>)-<i>N</i>-(5-((1-(5-amino-1,3,4-thiadiazol-2-yl)pyrrolidin-3-yl)amino)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide (<b>18</b>) (425 mg, 1.06 mmol), 2-(1-methylpiperidin-4-yl)acetic acid hydrochloride (225 mg, 1.16 mmol), and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.736 mL, 4.22 mmol) in DMA (10 mL) at rt. The resulting solution was stirred at rt for 16 h. The mixture was purified by ion-exchange chromatography, using an SCX-2 column. The desired product was eluted from the column using 7 M NH<sub>3</sub>/methanol to afford a crude product, which was purified by flash silica chromatography, elution gradient 0–10% 2 M NH<sub>3</sub>/methanol in DCM to afford the title compound (224 mg, 39%) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.25 (2H, qd, <i>J</i> = 3.6, 12.2 Hz), 1.62 (2H, d, <i>J</i> = 12.2 Hz), 1.68–1.78 (1H, m), 1.82–1.90 (2H, m), 2.08–2.15 (1H, m), 2.18 (3H, s), 2.32–2.40 (3H, m), 2.74–2.79 (2H, m), 3.46 (1H, dd, <i>J</i> = 3.3, 10.4 Hz), 3.50–3.62 (2H. m), 3.77 (1H, dd, <i>J</i> = 5.7, 10.5 Hz), 3.78 (2H, s), 4.37–4.45 (1H, m), 7.29–7.42 (5H, m), 7.74 (1H, d, <i>J</i> = 5.9 Hz), 12.06 (1H, s), 12.30 (1H, s); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 30.83, 31.39, 32.31, 41.42, 41.44, 45.98, 47.95, 53.52, 55.02, 55.12, 126.75, 128.34, 129.18, 134.87, 148.36, 149.10, 163.35, 163.70, 168.50, 169.51; HRMS (ES<sup>+</sup>) for C<sub>24</sub>H<sub>32</sub>N<sub>9</sub>O<sub>2</sub>S<sub>2</sub> [M + H]<sup>+</sup>: calcd, 542.2120; found, 542.2117.</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> (<i>R</i>)-2-Phenyl-<i>N</i>-(5-((1-(5-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)acetamido)-1,3,4-thiadiazol-2-yl)pyrrolidin-3-yl)amino)-1,3,4-thiadiazol-2-yl)acetamide (<b>15</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0030.gif" alt="" id="fx7" /></img><div class="NLM_p last">A mixture of (<i>R</i>)-2-phenyl-<i>N</i>-(5-(pyrrolidin-3-ylamino)-1,3,4-thiadiazol-2-yl)acetamide (<b>32</b>) (150 mg, 0.49 mmol), 5-bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (89 mg, 0.49 mmol), and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.259 mL, 1.48 mmol) in DMF (1 mL) was stirred at 100 °C for 30 min. After cooling to rt, 2-(tetrahydro-2H-pyran-4-yl)acetyl chloride (80 mg, 0.49 mmol) was added dropwise. The mixture was stirred for 15 min and the crude product was purified by preparative HPLC (Waters SunFire column, 5 μm silica, 50 mm diameter, 100 mm length), using decreasingly polar mixtures of water (containing 0.1% formic acid) and acetonitrile as eluents and repurified by preparative HPLC (Waters XBridge Prep C18 OBD column, 5 μm silica, 30 mm diameter, 100 mm length), using decreasingly polar mixtures of water (containing 1% NH<sub>3</sub>) and acetonitrile as eluents to afford the title compound (19 mg, 7%) as a solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.21 (2H, qd, <i>J</i> = 4.4, 11.9 Hz), 1.48–1.57 (2H, m), 1.91–2.01 (1H, m), 2.02–2.09 (1H, m), 2.26–2.32 (1H, m), 2.34 (2H, d, <i>J</i> = 7.1 Hz), 3.28 (2H, td, <i>J</i> = 1.9, 11.7 Hz), 3.40 (1H, dd, <i>J</i> = 3.3, 10.4 Hz), 3.44–3.56 (2H, m), 3.71 (1H, dd, <i>J</i> = 4.8, 10.4 Hz), 3.72 (2H, s), 3.81 (2H, dd, <i>J</i> = 2.7, 11.4 Hz), 4.35 (1H, dq, <i>J</i> = 5.8, 9.9 Hz), 7.22–7.36 (5H, m), 7.68 (1H, d, <i>J</i> = 5.9 Hz), 12.02 (1H, s), 12.25 (1H, s); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 30.83, 31.94, 32.19, 41.41, 41.72, 47.95, 53.51, 55.12, 66.74, 126.75, 128.34, 129.18, 134.87, 148.33, 149.10, 163.35, 163.71, 168.51, 169.26; HRMS (ES<sup>+</sup>) for C<sub>23</sub>H<sub>29</sub>N<sub>8</sub>O<sub>3</sub>S<sub>2</sub> [M + H]<sup>+</sup>: calcd, 529.1804; found, 529.1823.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 2-(1-Acetylpiperidin-2-yl)-<i>N</i>-(5-((<i>R</i>)-3-((5-(2-phenylacetamido)-1,3,4-thiadiazol-2-yl)amino)pyrrolidin-1-yl)-1,3,4-thiadiazol-2-yl)acetamide (<b>16</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0031.gif" alt="" id="fx8" /></img><div class="NLM_p last">1-[Bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (106 mg, 0.28 mmol) was added to (<i>R</i>)-<i>N</i>-(5-((1-(5-amino-1,3,4-thiadiazol-2-yl)pyrrolidin-3-yl)amino)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide (<b>18</b>) (101 mg, 0.25 mmol), 2-(1-acetylpiperidin-2-yl)acetic acid (52.3 mg, 0.28 mmol), and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.131 mL, 0.75 mmol) in DMA (5 mL). The resulting solution was stirred at rt for 20 h. The crude product was loaded onto an SCX-2 column and eluted with 7 M NH<sub>3</sub>/methanol. The crude product was purified by preparative HPLC (Waters XBridge Prep C18 OBD column, 5 μm silica, 30 mm diameter, 100 mm length), using decreasingly polar mixtures of water (containing 1% NH<sub>3</sub>) and acetonitrile as eluents to afford the title compound (46 mg, 32%) as a mixture of two diastereoisomers. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.19 (1H, s), 1.55 (5H, d), 1.89–2.09 (5H, m), 2.24–2.35 (1H, m), 2.52–2.6 (1H, m), 2.58–2.73 (1H, m), 2.90–3.00 (1H, m), 3.40 (1H, s), 3.42–3.56 (2H, m), 3.70 (2H, d), 4.23–4.47 (2H, m), 5.01 (1H, s), 7.22–7.35 (5H, m), 7.61 (1H, d), 12.08 (2H, s); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 18.41, 18.73, 21.13, 21.65, 25.05, 25.49, 27.73, 28.97, 30.84, 35.33, 35.59, 36.07, 40.43, 41.42, 44.96, 47.96, 50.51, 53.52, 55.12, 126.75, 128.33, 129.17, 134.87, 148.18, 148.40, 149.09, 163.34, 163.75, 163.81, 168.06, 168.29, 168.49; HRMS (ES<sup>+</sup>) for C<sub>25</sub>H<sub>32</sub>N<sub>9</sub>O<sub>3</sub>S<sub>2</sub> [M + H]<sup>+</sup>: calcd, 570.2070; found, 570.2067.</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> (<i>R</i>)-<i>N</i>-(5-((1-(5-Acetamido-1,3,4-thiadiazol-2-yl)pyrrolidin-3-yl)amino)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide (<b>17</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0032.gif" alt="" id="fx9" /></img><div class="NLM_p last">Acetyl chloride (0.018 mL, 0.26 mmol) was added dropwise to a solution of (<i>R</i>)-<i>N</i>-(5-((1-(5-amino-1,3,4-thiadiazol-2-yl)pyrrolidin-3-yl)amino)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide (<b>18</b>) (52 mg, 0.13 mmol) and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.102 mL, 0.58 mmol) in DMF (1 mL). The mixture was stirred for 5 min before being quenched with methanol. The crude sample was then purified by preparative HPLC (Waters XBridge Prep C18 OBD column, 5 μm silica, 50 mm diameter, 100 mm length), using decreasingly polar mixtures of water (containing 1% NH<sub>3</sub>) and acetonitrile as eluents to afford the title compound (11 mg, 19%) as a solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.02–2.09 (1H, m), 2.10 (3H, s), 2.26–2.35 (1H. m), 3.40 (1H, dd, <i>J</i> = 3.4, 10.4 Hz), 3.44–3.56 (2H, m), 3.71 (1H, dd, <i>J</i> = 5.7, 10.3 Hz), 3.72 (2H, s), 4.36 (1H, m), 7.23–7.36 (5H, m), 7.66 (1H, d, <i>J</i> = 5.9 Hz), 12.04 (2H, s); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 22.66, 31.34, 41.92, 48.45, 54.02, 55.61, 127.26, 128.85, 129.69, 135.39, 149.01, 149.61, 163.86, 164.18, 168.08, 169.02; HRMS (ES<sup>+</sup>) for C<sub>18</sub>H<sub>21</sub>N<sub>8</sub>O<sub>2</sub>S<sub>2</sub> [M + H]<sup>+</sup>: calcd, 445.1229; found, 445.1219.</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> (<i>R</i>)-<i>N</i>-(5-((1-(5-Amino-1,3,4-thiadiazol-2-yl)pyrrolidin-3-yl)amino)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide (<b>18</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0033.gif" alt="" id="fx10" /></img><div class="NLM_p last">A mixture of (<i>R</i>)-2-phenyl-<i>N</i>-(5-(pyrrolidin-3-ylamino)-1,3,4-thiadiazol-2-yl)acetamide (<b>32</b>) (401 mg, 1.32 mmol), 5-bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (248 mg, 1.38 mmol), and potassium carbonate (219 mg, 1.59 mmol) in acetonitrile (40 mL) was stirred at rt for 64 h, then at 80 °C for 1 h. The reaction mixture was allowed to cool to rt. It was then quenched with water (40 mL) and the product was extracted with ethyl acetate (4 × 50 mL). The organics were combined, dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The residue was purified by flash silica chromatography, elution gradient 0–10% 2 M NH<sub>3</sub>/methanol in DCM to afford the title compound (387 mg, 73%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.93–2.03 (1H, m), 2.25 (1H, dq), 3.25 (1H, d), 3.31–3.47 (2H, m), 3.61 (1H, dd), 3.71 (2H, s), 4.30 (1H, h), 6.26 (2H, s), 7.21–7.35 (5H, m), 7.59 (1H, d), 12.14 (1H, s); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 31.35, 41.92, 48.65, 53.97, 55.95, 127.27, 128.85, 129.68, 135.38, 149.56, 159.78, 160.56, 163.90, 169.01; HRMS (ES<sup>+</sup>) for C<sub>16</sub>H<sub>19</sub>N<sub>8</sub>OS<sub>2</sub> [M + H]<sup>+</sup>: calcd, 403.1123; found, 403.1143.</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> (<i>R</i>)-<i>N</i>-(5-((1-(1,3,4-Thiadiazol-2-yl)pyrrolidin-3-yl)amino)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide (<b>19</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0034.gif" alt="" id="fx11" /></img><div class="NLM_p last">A mixture of (<i>R</i>)-2-phenyl-<i>N</i>-(5-(pyrrolidin-3-ylamino)-1,3,4-thiadiazol-2-yl)acetamide (<b>32</b>) (109 mg, 0.36 mmol), 2-bromo-1,3,4-thiadiazole (125 mg, 0.76 mmol), DMAP (27 mg, 0.22 mmol), and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.25 mL, 1.43 mmol) in NMP (1 mL) was stirred at 106 °C for 20 h. An additional portion of DMAP (25 mg) and 2-bromo-1,3,4-thiadiazole (109 mg) was added, and the mixture was stirred at 106 °C for a further 24 h. The mixture was diluted with DCM and flushed through a silica column (5 g), elution gradient 0–10% methanol in DCM to afford the crude product, which was purified by preparative HPLC (Waters SunFire column, 5 μm silica, 50 mm diameter, 100 mm length), using decreasingly polar mixtures of water (containing 0.1% formic acid) and acetonitrile as eluents to afford the title compound (9 mg, 7%) as a solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.05–2.14 (1H, m), 2.30–2.37 (1H, m), 3.44 (1H, dd, <i>J</i> = 3.4, 10.5 Hz), 3.49–3.62 (2H, m), 3.73 (2H, s), 3.77 (1H, dd, <i>J</i> = 5.8, 10.5 Hz), 4.36–4.42 (1H, m), 7.24–7.36 (5H, m), 7.66 (1H, d, <i>J</i> = 5.9 Hz), 8.70 (1H, s), 12.15 (1H, s); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 30.86, 41.40, 48.75, 53.58, 55.95, 126.76, 128.34, 129.17, 134.86, 142.08, 149.11, 163.33, 167.64, 168.51; HRMS (ES<sup>+</sup>) for C<sub>16</sub>H<sub>18</sub>N<sub>7</sub>OS<sub>2</sub> [M + H]<sup>+</sup>: calcd, 388.1014; found, 388.1028.</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> (<i>R</i>)-<i>N</i>-(6-((1-(1,3,4-Thiadiazol-2-yl)pyrrolidin-3-yl)amino)pyridazin-3-yl)-2-phenylacetamide (<b>20</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0035.gif" alt="" id="fx12" /></img><div id="sec5_1_12_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (<i>R</i>)-<i>tert</i>-Butyl (1-(1,3,4-Thiadiazol-2-yl)pyrrolidin-3-yl)carbamate</h4><div class="NLM_p last">A mixture of 2-bromo-1,3,4-thiadiazole (1.329 g, 8.05 mmol), (<i>R</i>)-<i>tert</i>-butyl pyrrolidin-3-yl-carbamate (<b>34</b>) (1.5 g, 8.05 mmol), and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (3.52 mL, 20.13 mmol) in <i>n</i>-butanol (2 mL) was stirred at 50 °C for 3 h, then concentrated in vacuo. The residue was dissolved in DMF and the crude product was purified by preparative HPLC (Waters XBridge Prep C18 OBD column, 5 μm silica, 50 mm diameter, 100 mm length), using decreasingly polar mixtures of water (containing 1% NH<sub>3</sub>) and acetonitrile as eluents. Fractions containing the desired compound were evaporated to dryness to afford the title compound (1.52 g, 70%) as a gum. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.40 (9H. s), 1.88–1.99 (1H, m), 2.14–2.25 (1H, m), 3.24–3.29 (1H, m), 3.43 (1H, ddd, <i>J</i> = 5.6, 8.0, 9.7 Hz), 3.54 (1H, dt, <i>J</i> = 7.3, 9.6 Hz), 3.63 (1H, dd, <i>J</i> = 6.3, 10.3 Hz), 4.11–4.21 (1H, m), 7.24 (1H, s), 8.68 (1H, s); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 271.</div></div><div id="sec5_1_12_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (<i>R</i>)-1-(1,3,4-Thiadiazol-2-yl)pyrrolidin-3-amine (<b>35</b>)</h4><div class="NLM_p last">TFA (10 mL) was added to (<i>R</i>)-<i>tert</i>-butyl (1-(1,3,4-thiadiazol-2-yl)pyrrolidin-3-yl)carbamate (1.52 g, 5.62 mmol) in DCM (20 mL) at rt. The resulting solution was stirred at rt for 3 h. The crude product was purified by ion-exchange chromatography, using an SCX-2 column. The desired product was eluted from the column using 2 M NH<sub>3</sub>/methanol and pure fractions were evaporated to dryness to afford the title compound (0.940 g, 98%) as a brown gum. MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 171.</div></div><div id="sec5_1_12_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>N</i>-(6-Iodopyridazin-3-yl)-2-phenylacetamide (<b>37</b>)</h4><div class="NLM_p last">2-Phenylacetyl chloride (1.513 mL, 11.45 mmol) was added to 6-iodopyridazin-3-amine (<b>36</b>) (2.3 g, 10.41 mmol) and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (4.0 mL, 22.90 mmol) in DMF (20 mL) at 5 °C. The resulting solution was stirred at rt for 3 h. The reaction mixture was diluted with ethyl acetate and washed twice with water. The organic layer was dried over MgSO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was triturated with DCM and the solid was filtered off. The filtrate was evaporated to dryness and the solid residue was triturated under ether, filtered, and the solids were combined, triturated again with DCM to give the title compound (0.884 g, 25%) as a pale yellow solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.78 (2H, s), 7.22–7.27 (1H, m), 7.28–7.37 (4H, m), 8.03–8.08 (2H, m), 11.43 (1H, s); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 340.</div></div><div id="sec5_1_12_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (<i>R</i>)-<i>N</i>-(6-((1-(1,3,4-Thiadiazol-2-yl)pyrrolidin-3-yl)amino)pyridazin-3-yl)-2-phenylacetamide (<b>20</b>)</h4><div class="NLM_p last">(<i>R</i>)-1-(1,3,4-Thiadiazol-2-yl)pyrrolidin-3-amine (<b>35</b>) (35 mg, 0.21 mmol), <i>N</i>-(6-iodopyridazin-3-yl)-2-phenylacetamide (<b>37</b>) (70 mg, 0.21 mmol), copper(I) iodide (7.86 mg, 0.04 mmol), (2<i>S</i>,4<i>R</i>)-4-hydroxypyrrolidine-2-carboxylic acid (10.83 mg, 0.08 mmol), and potassium carbonate (86 mg, 0.62 mmol) were combined in degassed DMSO (2 mL). The reaction mixture was heated at 80 °C for 4 h. The crude product was purified by ion-exchange chromatography, using an SCX-2 column. The desired product was eluted from the column using 7 M NH<sub>3</sub>/methanol to afford crude product. The crude product was purified by preparative HPLC (Waters SunFire column, 5 μm silica, 50 mm diameter, 100 mm length), using decreasingly polar mixtures of water (containing 0.1% formic acid) and acetonitrile as eluents. Fractions containing the desired compound were evaporated to dryness to afford the title compound (7 mg, 8%) as a gum. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.08–2.16 (1H, m), 2.38–2.45 (1H, m), 3.43 (1H, dd, <i>J</i> = 3.9, 10.4 Hz), 3.57–3.63 (1H, m), 3.64–3.70 (1H, m), 3.76 (2H, s), 3.88 (1H, dd, <i>J</i> = 6.0, 10.3 Hz), 4.60–4.68 (1H, m), 6.92 (1H, d, <i>J</i> = 9.6 Hz), 7.11 (1H, d, <i>J</i> = 6.2 Hz), 7.28–7.33 (1H, m), 7.36–7.43 (4H. m), 7.96 (1H, d, <i>J</i> = 9.5 Hz), 8.75 (1H, s), 10.80 (1H, s); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 30.99, 42.68, 48.92, 50.79, 56.17, 116.79, 121.32, 126.50, 128.24, 129.10, 135.79, 141.94, 148.55, 156.44, 167.69, 169.73; HRMS (ES<sup>+</sup>) for C<sub>18</sub>H<sub>20</sub>N<sub>7</sub>OS [M + H]<sup>+</sup>: calcd, 382.1450; found, 382.1451.</div></div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> <i>N</i>-[6-[(3<i>R</i>)-3-[(5-Amino-1,3,4-thiadiazol-2-yl)amino]pyrrolidin-1-yl]pyridazin-3-yl]-2-phenyl-acetamide (<b>21</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0036.gif" alt="" id="fx13" /></img><div id="sec5_1_13_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> [(<i>R</i>)-1-(6-Amino-pyridazin-3-yl)-pyrrolidin-3-yl]-carbamic Acid <i>tert</i>-Butyl Ester</h4><div class="NLM_p last">Triethylamine (0.70 mL, 5 mmol) was added to 6-chloro-pyridazin-3-ylamine (<b>38</b>) (0.32 g, 2.5 mmol) and (<i>R</i>)-<i>tert</i>-butyl pyrrolidin-3-yl-carbamate (<b>34</b>) (1.39 g, 7.5 mmol) in <i>n</i>-butanol (2 mL). The reaction mixture was heated in a microwave reactor for 30 min at 160 °C, then heating was continued for 90 min at 180 °C. The mixture was concentrated in vacuo and the residue was dissolved in DCM, washed with water, and then saturated ammonium chloride solution. The organic phase was dried with MgSO<sub>4</sub>, filtered, and evaporated under reduced pressure to yield the title compound (0.70 g, 100%) as a brown gum. MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 280.</div></div><div id="sec5_1_13_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> [(<i>R</i>)-1-(6-Phenylacetylamino-pyridazin-3-yl)-pyrrolidin-3-yl]-carbamic Acid <i>tert</i>-Butyl Ester</h4><div class="NLM_p last">Phenylacetyl chloride (0.32 mL, 2.5 mmol) was added to [(<i>R</i>)-1-(6-amino-pyridazin-3-yl)-pyrrolidin-3-yl]-carbamic acid <i>tert</i>-butyl ester (0.7 g, 2.5 mmol) and triethylamine (0.7 mL, 5.0 mmol) in DCM (10 mL), and the mixture was cooled to 0 °C. This was then allowed to warm to rt and stirred for 15 h. Saturated sodium bicarbonate solution (20 mL) was then added. The organic phase was separated, dried with MgSO<sub>4</sub>, filtered, and evaporated under reduced pressure to give a residue that was purified by flash column chromatography eluting with 2:3 ethyl acetate/cyclohexane increasing to 100% ethyl acetate to yield the title compound (0.25 g, 25%). MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 398.</div></div><div id="sec5_1_13_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> <i>N</i>-[6-((<i>R</i>)-3-Amino-pyrrolidin-1-yl)-pyridazin-3-yl]-2-phenyl-acetamide</h4><div class="NLM_p last">TFA (0.5 mL) was added to [(<i>R</i>)-1-(6-phenylacetylamino-pyridazin-3-yl)-pyrrolidin-3-yl]-carbamic acid <i>tert</i>-butyl ester (95 mg, 0.23 mmol) in DCM (2 mL). The reaction mixture was stirred at rt for 4 h. The solvent was evaporated and the residue was redissolved in methanol and added to an SCX-2 column. The column was washed with DCM and methanol and then the product was eluted with 2 M NH<sub>3</sub>/methanol to provide the title compound (50 mg, 71%) as a light brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.42–1.54 (1H, m), 1.76–1.87 (1H, m), 2.81–2.89 (1H, m), 3.12–3.22 (1H, m), 3.24–3.39 (3H, m), 3.45 (2H, s), 6.64 (1H, d, <i>J</i> = 9.7 Hz), 6.96–7.05 (2H, m), 7.05–7.15 (5H, m), 7.71 (1H, d, <i>J</i> = 9.7 Hz), 10.55 (1H, s); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 298.</div></div><div id="sec5_1_13_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> <i>N</i>-[6-[(3<i>R</i>)-3-[(5-Amino-1,3,4-thiadiazol-2-yl)amino]pyrrolidin-1-yl]pyridazin-3-yl]-2-phenyl-acetamide (<b>21</b>)</h4><div class="NLM_p last"><i>N</i>-Ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.05 mL, 0.258 mmol) was added to a suspension of <i>N</i>-[6-((<i>R</i>)-3-amino-pyrrolidin-1-yl)-pyridazin-3-yl]-2-phenyl-acetamide (40 mg, 0.14 mmol) and 5-bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (25 mg, 0.14 mmol) in acetonitrile (1 mL). The reaction mixture was heated at 80 °C for 2 h and allowed to cool to rt, and a solid precipitate was filtered off to provide crude product. This was purified by mass directed preparative LC–MS using acetonitrile and water with 0.1% formic acid as eluent. The fractions were evaporated and 1.25 M HCl in methanol (1 mL) was added to dissolve the residue, and this was added to an SCX-2 column that was washed with methanol and then eluted with 2 M NH<sub>3</sub>/methanol. The solvent was evaporated under reduced pressure to give the title compound (10 mg, 17%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.92–2.09 (1H, m), 2.15–2.31 (1H, m), 3.39–3.61 (3H, m), 3.63–3.77 (3H, m), 4.17–4.33 (1H, m), 6.32 (2H, s), 6.95 (1H, d, <i>J</i> = 9.7 Hz), 7.10 (1H, d, <i>J</i> = 5.9 Hz), 7.20–7.28 (1H, m), 7.28–7.40 (4H, m), 7.97 (1H, d, <i>J</i> = 9.7 Hz), 10.83 (1H, s); HRMS (ES<sup>+</sup>) for C<sub>18</sub>H<sub>21</sub>N<sub>8</sub>OS [M + H]<sup>+</sup>: calcd, 397.15535; found, 397.15674.</div></div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> (<i>R</i>)-<i>N</i>-(5-(3-((5-Amino-1,3,4-thiadiazol-2-yl)amino)pyrrolidin-1-yl)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide (<b>22</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0037.gif" alt="" id="fx14" /></img><div id="sec5_1_14_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> (<i>R</i>)-<i>tert</i>-Butyl (1-(5-Amino-1,3,4-thiadiazol-2-yl)pyrrolidin-3-yl)carbamate</h4><div class="NLM_p last">(<i>R</i>)-<i>tert</i>-Butyl pyrrolidin-3-yl-carbamate (<b>34</b>) (402 mg, 2.16 mmol), 5-bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (395 mg, 2.19 mmol), and potassium carbonate (361 mg, 2.61 mmol) were dissolved in acetonitrile (30 mL). The solution was stirred at 80 °C under nitrogen for 18 h. The mixture was allowed to cool to rt and concentrated in vacuo. Water was then added to the residue and the solid was filtered off. This was then washed with water. The solid was then redissolved in methanol and the solvent was removed by evaporation to afford the crude title product (1.015 g, >quant) as a solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.40 (9H, s), 1.80–1.90 (1H, m), 2.07–2.17 (1H, m), 3.11 (1H, dd, <i>J</i> = 4.8, 9.9 Hz), 3.29 (1H, s), 3.36–3.45 (1H, m), 3.49 (1H, dd, <i>J</i> = 6.4, 9.9 Hz), 4.01–4.13 (1H, m), 6.25 (2H, s), 7.17 (1H, s); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 286.</div></div><div id="sec5_1_14_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> (<i>R</i>)-<i>tert</i>-Butyl (1-(5-(2-Phenylacetamido)-1,3,4-thiadiazol-2-yl)pyrrolidin-3-yl)carbamate</h4><div class="NLM_p last">2-Phenylacetyl chloride (0.306 mL, 2.31 mmol) was added to (<i>R</i>)-<i>tert</i>-butyl (1-(5-amino-1,3,4-thiadiazol-2-yl)pyrrolidin-3-yl)carbamate (1.015 g, 2.31 mmol) and potassium carbonate (386 mg, 2.79 mmol) in THF (30 mL). The resulting pale orange suspension was stirred at rt for 2 h. The reaction was quenched with water (30 mL) and extracted with ethyl acetate (3 × 50 mL). The organic phases were combined and evaporated to afford a residue that was then dissolved in methanol and evaporated to afford the crude title compound (1.076 g, >quant) as a solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.40 (9H, s), 1.84–1.94 (1H, m), 2.10–2.22 (1H, m), 3.21 (1H, dd, <i>J</i> = 4.6, 10.1 Hz), 3.37 (1H, ddd, <i>J</i> = 5.4, 9.4, 14.6 Hz), 3.45–3.53 (1H, m), 3.57 (1H, dd, <i>J</i> = 6.3, 10.2 Hz), 3.73 (2H, s), 4.13 (1H, s), 7.22 (1H, s), 7.24–7.38 (5H, m), 12.19 (1H, s); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 404.</div></div><div id="sec5_1_14_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> (<i>R</i>)-<i>N</i>-(5-(3-Aminopyrrolidin-1-yl)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide</h4><div class="NLM_p last">TFA (6.56 mL) was added to a suspension of (<i>R</i>)-<i>tert</i>-butyl (1-(5-(2-phenylacetamido)-1,3,4-thiadiazol-2-yl)pyrrolidin-3-yl)carbamate (1.076 g, 2.67 mmol) in DCM (25 mL). The resulting solution was stirred at rt for 90 min. The crude reaction mixture was loaded onto an SCX-2 column and the product was eluted with 2 M NH<sub>3</sub>/methanol. The solvent was then removed by rotary evaporation to afford the title compound (0.506 g, 63%) as a solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.68–1.77 (1H, m), 2.02–2.12 (1H, m), 3.05 (1H, dd, <i>J</i> = 4.3, 9.7 Hz), 3.36 (1H, ddd, <i>J</i> = 5.6, 7.9, 9.5 Hz), 3.45–3.54 (2H, m), 3.59 (1H, p, <i>J</i> = 5.5 Hz), 3.72 (2H, s), 7.21–7.38 (5H, m); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 304.</div></div><div id="sec5_1_14_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> (<i>R</i>)-<i>N</i>-(5-(3-((5-Amino-1,3,4-thiadiazol-2-yl)amino)pyrrolidin-1-yl)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide (<b>22</b>)</h4><div class="NLM_p last">A mixture of (<i>R</i>)-<i>N</i>-(5-(3-aminopyrrolidin-1-yl)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide (506 mg, 1.67 mmol), 5-bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (305 mg, 1.69 mmol), and potassium carbonate (277 mg, 2.00 mmol) in acetonitrile (20 mL) was stirred at 80 °C for 6 h. The reaction mixture was allowed to cool to rt and concentrated in vacuo. A solid was filtered off and washed with water. This solid was dissolved in methanol, then concentrated in vacuo to afford the title compound (369 mg, 55%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.96–2.07 (1H, m), 2.21–2.31 (1H, m), 3.36 (1H, dd, <i>J</i> = 3.6, 10.4 Hz), 3.39–3.55 (2H, m), 3.66 (1H, dd, <i>J</i> = 5.8, 10.4 Hz), 3.72 (2H, s), 4.2–4.3 (1H, m), 6.26 (2H, s), 7.07 (1H, d, <i>J</i> = 5.9 Hz), 7.23–7.28 (1H, m), 7.30–7.36 (4H, m), 12.11 (1H, s); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 30.88, 41.51, 47.98, 53.49, 55.21, 126.74, 128.35, 129.12, 134.96, 148.62, 159.01, 159.46, 163.78, 168.52; HRMS (ES<sup>+</sup>) for C<sub>16</sub>H<sub>19</sub>N<sub>8</sub>OS<sub>2</sub> [M + H]<sup>+</sup>: calcd, 403.1123; found, 403.1104.</div></div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> (<i>R</i>)-2-Phenyl-<i>N</i>-(5-((1-(pyridazin-3-yl)pyrrolidin-3-yl)amino)-1,3,4-thiadiazol-2-yl)acetamide (<b>23</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0038.gif" alt="" id="fx15" /></img><div class="NLM_p last">A mixture of (<i>R</i>)-2-phenyl-<i>N</i>-(5-(pyrrolidin-3-ylamino)-1,3,4-thiadiazol-2-yl)acetamide (<b>32</b>) (272 mg, 0.90 mmol), 3-chloropyridazine hydrochloride (190 mg, 1.26 mmol), <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.626 mL, 3.59 mmol), and DMAP (65.7 mg, 0.54 mmol) in NMP (2 mL) was stirred at 106 °C for 24 h. After cooling to rt, the mixture was diluted with DCM and passed through a silica column (5 g), elution gradient 0–10% methanol in DCM to afford a crude product that was purified by preparative HPLC (Waters SunFire column, 5 μm silica, 50 mm diameter, 100 mm length), using decreasingly polar mixtures of water (containing 0.1% formic acid) and acetonitrile as eluent. Fractions containing the desired compound were evaporated to dryness to afford the title compound (89 mg, 26%) as a solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.03–2.12 (1H, m), 2.26–2.34 (1H, m), 3.50 (1H, dd, <i>J</i> = 3.4, 10.9 Hz), 3.52–3.61 (2H, m), 3.72 (2H, s), 3.76 (1H, dd, <i>J</i> = 5.8, 11.0 Hz), 4.39 (1H, dq, <i>J</i> = 5.9, 10.1 Hz), 6.87 (1H, dd, <i>J</i> = 1.3, 9.2 Hz), 7.24–7.29 (1H, m), 7.29–7.36 (5H, m), 7.62 (1H, d, <i>J</i> = 6.1 Hz), 8.48 (1H, dd, <i>J</i> = 1.2, 4.5 Hz), 12.18 (1H, s); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 30.53, 41.42, 44.49, 51.91, 53.31, 111.88, 126.75, 127.16, 128.34, 129.17, 134.88, 142.05, 148.98, 157.67, 163.47, 168.47; HRMS (ES<sup>+</sup>) for C<sub>18</sub>H<sub>20</sub>N<sub>7</sub>OS [M + H]<sup>+</sup>: calcd, 382.1450; found, 382.1451.</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> (<i>R</i>)-<i>N</i>-(5-((1-(1,2,4-Triazin-3-yl)pyrrolidin-3-yl)amino)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide (<b>24</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0039.gif" alt="" id="fx16" /></img><div id="sec5_1_16_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> (<i>R</i>)-<i>tert</i>-Butyl (1-(1,2,4-Triazin-3-yl)pyrrolidin-3-yl)carbamate</h4><div class="NLM_p last">A solution of 3-chlorobenzoperoxoic acid (485 mg, 1.97 mmol) in DCM (8 mL) was dried over sodium sulfate, then filtered and the solution was added over 10 min to a stirred solution of 3-(methylthio)-1,2,4-triazine (<b>39</b>) (250 mg, 1.97 mmol) in dry DCM (5 mL) at 0 °C. The mixture was stirred at 0 °C for 30 min. To this solution was added a solution of (<i>R</i>)-<i>tert</i>-butyl pyrrolidin-3-yl-carbamate (<b>34</b>) (403 mg, 2.16 mmol) and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.687 mL, 3.93 mmol) in isopropyl alcohol (5 mL). The mixture was then heated at 50 °C for 6 h and then allowed to cool to rt and stirred for 16 h. The reaction mixture was evaporated to dryness and the residue was partitioned between ethyl acetate and saturated NaHCO<sub>3</sub> solution. The aqueous phase was reextracted twice with ethyl acetate, and the combined organic phases were dried, filtered, and evaporated to give a residue that was purified by flash silica chromatography, elution gradient 20–90% ethyl acetate in heptane to yield the title compound (510 mg, 98%) as a yellow solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 100 °C) δ 1.42 (9H, s), 1.96 (1H, dt, <i>J</i> = 6.7, 13.2 Hz), 2.19 (1H, dq, <i>J</i> = 6.5, 14.1 Hz), 3.44 (1H, dd, <i>J</i> = 4.9, 11.7 Hz), 3.54–3.61 (1H, m), 3.66–3.73 (1H, m), 3.78 (1H, dd, <i>J</i> = 6.5, 11.6 Hz), 4.12–4.20 (1H, m), 6.73 (1H, s), 8.23 (1H, d, <i>J</i> = 2.2 Hz), 8.55 (1H, d, <i>J</i> = 2.2 Hz); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 266.</div></div><div id="sec5_1_16_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> (<i>R</i>)-1-(1,2,4-Triazin-3-yl)pyrrolidin-3-amine (<b>40</b>)</h4><div class="NLM_p last">TFA (6.33 mL, 82.16 mmol) was added to (<i>R</i>)-<i>tert</i>-butyl (1-(1,2,4-triazin-3-yl)pyrrolidin-3-yl)carbamate (2.18 g, 8.22 mmol) in DCM (20 mL) and cooled to 0 °C. The resulting solution was stirred at rt for 16 h. The reaction mixture was evaporated to dryness, redissolved in toluene (50 mL), and evaporated again. The residue was dissolved in methanol (50 mL) and purified by ion-exchange chromatography, using an SCX-2 column. The desired product was eluted from the column using 2 M NH<sub>3</sub>/methanol and pure fractions were evaporated to dryness to afford the title compound (1.3 g, 96%) as a red solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 100 °C) δ 1.58 (2H, s), 1.70–1.81 (1H, m), 2.03–2.17 (1H, m), 3.26 (1H, dd, <i>J</i> = 4.6, 11.2 Hz), 3.51–3.77 (4H, m), 8.22 (1H, s), 8.52 (1H, s); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 166.</div></div><div id="sec5_1_16_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> (<i>R</i>)-<i>N</i>-(5-((1-(1,2,4-Triazin-3-yl)pyrrolidin-3-yl)amino)-1,3,4-thiadiazol-2-yl)-2-phenylacetamide (<b>24</b>)</h4><div class="NLM_p last"><i>N</i>-Ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.423 mL, 2.42 mmol) was added to (<i>R</i>)-1-(1,2,4-triazin-3-yl)pyrrolidin-3-amine (200 mg, 1.21 mmol) and 5-bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (218 mg, 1.21 mmol) in DMF (2 mL). The resulting solution was stirred at 100 °C for 45 min. More 5-bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (50 mg) was added and the mixture was stirred for a further 30 min at 100 °C. After cooling to rt, 2-phenylacetic acid (198 mg, 1.45 mmol) and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.21 mL) were added, followed by 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (552 mg, 1.45 mmol). The mixture was stirred at rt under nitrogen for 2 h. The reaction mixture was evaporated to dryness and the residue was dissolved in methanol and purified by ion-exchange chromatography, using an SCX-2 column. The desired product was eluted from the column using 7 M NH<sub>3</sub>/methanol and pure fractions were evaporated to dryness to afford crude product, which was purified by flash silica chromatography, elution gradient 0–20% 3 M NH<sub>3</sub> in methanol in DCM to afford the title compound (213 mg, 46%) as a solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 100 °C) δ 2.06–2.17 (1H, m), 2.29–2.37 (1H, m), 3.59–3.79 (5H, m), 3.89 (1H, dd, <i>J</i> = 6.1, 11.7 Hz), 4.36–4.43 (1H, m), 7.21–7.37 (6H, m), 8.24 (1H, d, <i>J</i> = 2.3 Hz), 8.56 (1H, d, <i>J</i> = 2.3 Hz), 11.71 (1H, s); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 30.41, 41.42, 44.37, 51.65, 53.22, 126.75, 128.34, 129.17, 134.87, 139.88, 149.02, 149.46, 159.46, 163.42, 168.48; HRMS (ES<sup>+</sup>) for C<sub>17</sub>H<sub>19</sub>N<sub>8</sub>OS [M + H]<sup>+</sup>: calcd, 383.1403; found, 383.1422.</div></div></div><div id="sec5_1_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> (<i>R</i>)-<i>N</i>-Benzyl-5-((1-(pyridazin-3-yl)pyrrolidin-3-yl)amino)-1,3,4-thiadiazole-2-carboxamide (<b>25</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0040.gif" alt="" id="fx17" /></img><div id="sec5_1_17_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>N</i>-Benzyl-5-bromo-1,3,4-thiadiazole-2-carboxamide (<b>42</b>)</h4><div class="NLM_p last">Benzylamine (1.06 mL, 9.7 mmol) was added to ethyl 5-bromo-1,3,4-thiadiazole-2-carboxylate (<b>41</b>) (2 g, 8.44 mmol) in DCM (50 mL) at rt. The resulting solution was stirred at rt for 2 days, then heated at 60 °C for 2 h. After cooling to rt, the crude mixture was purified by flash silica chromatography, elution gradient 0–1% methanol in DCM to afford the title compound (1.2 g, 48%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 4.47 (2H, d, <i>J</i> = 6.3 Hz), 7.20–7.38 (5H, m), 9.86 (1H, t, <i>J</i> = 6.1 Hz); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 298/300.</div></div><div id="sec5_1_17_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> (<i>R</i>)-<i>tert</i>-Butyl 3-((5-(Benzylcarbamoyl)-1,3,4-thiadiazol-2-yl)amino)pyrrolidine-1-carboxylate</h4><div class="NLM_p last">Potassium carbonate (417 mg, 3.02 mmol) was added to <i>tert</i>-butyl (3<i>R</i>)-3-aminopyrrolidine-1-carboxylate (<b>31</b>) (469 mg, 2.52 mmol) and <i>N</i>-benzyl-5-bromo-1,3,4-thiadiazole-2-carboxamide (<b>42</b>) (750 mg, 2.52 mmol) in a mixture of DMA (10 mL) and acetonitrile (10 mL) at rt. The resulting mixture was stirred at 80 °C for 16 h. An additional portion of <i>tert</i>-butyl (3<i>R</i>)-3-aminopyrrolidine-1-carboxylate (<b>31</b>) (469 mg, 2.52 mmol) and potassium carbonate (417 mg, 3.02 mmol) was added and the mixture was stirred at 100 °C for a further 5 h. The reaction mixture was cooled and left overnight at rt. The reaction mixture was diluted with EtOAc (200 mL) and washed with water (2 × 100 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product, which was purified by flash silica chromatography to afford the title compound (470 mg, 46%) as a brown/orange solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.41 (9H, s), 1.92 (1H, br), 2.15 (1H, br), 3.21–3.29 (1H, m), 3.31–3.41 (2H, m), 3.56 (1H, br), 4.28 (1H, br), 4.42 (2H, d, <i>J</i> = 6.4 Hz), 7.14–7.38 (5H, m), 8.48 (1H, d, <i>J</i> = 6.2 Hz), 9.29 (1H, t, <i>J</i> = 6.3 Hz); MS (ESI) [M – H]<sup>−</sup><i>m</i>/<i>z</i> 402.</div></div><div id="sec5_1_17_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (<i>R</i>)-<i>N</i>-Benzyl-5-(pyrrolidin-3-ylamino)-1,3,4-thiadiazole-2-carboxamide</h4><div class="NLM_p last">TFA (3 mL) was added to a solution of (<i>R</i>)-<i>tert</i>-butyl 3-((5-(benzylcarbamoyl)-1,3,4-thiadiazol-2-yl)amino)pyrrolidine-1-carboxylate (460 mg, 1.14 mmol) in DCM (15 mL) and the reaction mixture was allowed to stir at rt for 1 h. The crude product was purified by ion-exchange chromatography, using an SCX-2 column. The desired product was eluted from the column using 7 M NH<sub>3</sub>/methanol, and pure fractions were evaporated to dryness to afford the title compound (240 mg, 69%) as an orange gum. MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 304.</div></div><div id="sec5_1_17_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> (<i>R</i>)-<i>N</i>-Benzyl-5-((1-(pyridazin-3-yl)pyrrolidin-3-yl)amino)-1,3,4-thiadiazole-2-carboxamide (<b>25</b>)</h4><div class="NLM_p last"><i>N</i>-Ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (276 μL, 1.58 mmol) was added to (<i>R</i>)-<i>N</i>-benzyl-5-(pyrrolidin-3-ylamino)-1,3,4-thiadiazole-2-carboxamide (120 mg, 0.40 mmol), DMAP (24.16 mg, 0.20 mmol), and 3-chloropyridazine (45.3 mg, 0.40 mmol) in NMP at rt. The resulting mixture was stirred at 110 °C for 4 h. The crude product was purified by preparative HPLC (Waters SunFire column, 5 μm silica, 50 mm diameter, 100 mm length), using decreasingly polar mixtures of water (containing 0.1% formic acid) and acetonitrile as eluents to the title compound (20 mg, 13%) as a gum. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.14–2.23 (1H, m), 2.36–2.42 (1H, m), 3.58–3.67 (2H, m), 3.85 (1H, dd, <i>J</i> = 5.9, 11.1 Hz), 4.48 (2H, d, <i>J</i> = 6.3 Hz), 4.50–4.58 (1H, m), 6.95 (1H, dd, <i>J</i> = 1.3, 9.2 Hz), 7.25–7.34 (1H, m), 7.33–7.43 (5H, m), 8.22 (1H, s), 8.55 (1H, dd, <i>J</i> = 1.2, 4.5 Hz), 8.65 (1H, d, <i>J</i> = 6.3 Hz), 9.38 (1H, t, <i>J</i> = 6.4 Hz); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 30.39, 42.37, 44.46, 51.67, 54.16, 111.96, 126.80, 127.22, 127.27, 128.23, 139.08, 142.17, 153.83, 157.66, 158.42, 170.40; HRMS (ES<sup>+</sup>) for C<sub>18</sub>H<sub>20</sub>N<sub>7</sub>OS [M + H]<sup>+</sup>: calcd, 382.1450; found, 382.1451.</div></div></div><div id="sec5_1_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> (<i>R</i>)-5-((1-(1,2,4-Triazin-3-yl)pyrrolidin-3-yl)amino)-<i>N</i>-benzyl-1,3,4-thiadiazole-2-carboxamide (<b>26</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0041.gif" alt="" id="fx18" /></img><div class="NLM_p last">(<i>R</i>)-1-(1,2,4-Triazin-3-yl)pyrrolidin-3-amine (<b>40</b>) (111 mg, 0.67 mmol), <i>N</i>-benzyl-5-bromo-1,3,4-thiadiazole-2-carboxamide (<b>42</b>) (200 mg, 0.67 mmol), and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.14 mL, 0.80 mmol) in DMF (2 mL) were heated to 100 °C for 5 h in a microwave reactor, then cooled to rt. The reaction mixture was evaporated to dryness and the residue dissolved in methanol (5 mL). This was purified by ion-exchange chromatography, using an SCX-2 (20 g) column. The desired product was eluted from the column using 7 M NH<sub>3</sub>/methanol and pure fractions were evaporated to dryness to afford a residue that was purified by flash silica chromatography, elution gradient 0–6% 3 M NH<sub>3</sub>/methanol in DCM. Pure fractions were evaporated to dryness to afford the title compound (135 mg, 53%) as a solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.07–2.19 (1H, m), 2.27–2.39 (1H, m), 3.68 (3H, br), 3.85 (1H, br), 4.41 (2H, d, <i>J</i> = 6.4 Hz), 4.43–4.51 (1H, m), 7.23 (1H, ddt, <i>J</i> = 2.3, 5.3, 7.2 Hz), 7.27–7.35 (4H, m), 8.32 (1H, d, <i>J</i> = 2.3 Hz), 8.59 (1H, d, <i>J</i> = 6.1 Hz), 8.62 (1H, d, <i>J</i> = 2.3 Hz), 9.33 (1H, t, <i>J</i> = 6.3 Hz); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 30.30, 42.38, 44.33, 51.41, 54.09, 126.80, 127.27, 128.23, 139.06, 140.00, 149.50, 153.93, 158.39, 159.46, 170.36; HRMS (ES<sup>+</sup>) for C<sub>17</sub>H<sub>19</sub>N<sub>8</sub>OS [M + H]<sup>+</sup>: calcd, 383.1403; found, 383.1422.</div></div><div id="sec5_1_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> (<i>S</i>)-2-Methoxy-2-phenyl-<i>N</i>-(5-(((<i>R</i>)-1-(pyridazin-3-yl)pyrrolidin-3-yl)amino)-1,3,4-thiadiazol-2-yl)acetamide (<b>27</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0042.gif" alt="" id="fx19" /></img><div id="sec5_1_19_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> (<i>R</i>)-<i>tert</i>-Butyl (1-(6-Chloropyridazin-3-yl)pyrrolidin-3-yl)carbamate</h4><div class="NLM_p last">Acetonitrile (1100 mL) and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (69.4 mL, 397 mmol) were added to a mixture of (<i>R</i>)-<i>tert</i>-butyl pyrrolidin-3-yl-carbamate (74.0 g, 397 mmol) and 3,6-dichloropyridazine (59.2 g, 397 mmol). The reaction mixture was heated to reflux for 16 h. After cooling to rt, the reaction mixture was diluted with water (1600 mL). The resulting thick white slurry was stirred for 1 h. The solid was collected by filtration under vacuum and the resulting cream filter cake was washed under vacuum with a further portion of water (500 mL) followed by diethyl ether (500 mL). The resulting solid was dried under vacuum in an oven overnight to afford the title compound (81.5 g, 69%) as a cream solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.45 (9H, s), 1.97–2.07 (1H, m), 2.26–2.36 (1H, m), 3.41 (1H, dd, <i>J</i> = 4.3, 10.9 Hz), 3.54–3.69 (2H, m), 3.78 (1H, dd, <i>J</i> = 6.1, 10.9 Hz), 4.36 (1H, br), 4.75 (1H, br), 6.61 (1H, d, <i>J</i> = 9.4 Hz), 7.18 (1H, d, <i>J</i> = 9.4 Hz); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 299.</div></div><div id="sec5_1_19_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> (<i>R</i>)-<i>tert</i>-Butyl (1-(Pyridazin-3-yl)pyrrolidin-3-yl)carbamate</h4><div class="NLM_p last">Ammonium formate (96 g, 1.53 mol) was added to a mixture of palladium on carbon 10% (4.87 g, 4.58 mmol) and (<i>R</i>)-<i>tert</i>-butyl (1-(6-chloropyridazin-3-yl)pyrrolidin-3-yl)carbamate (45.6 g, 152.63 mmol) in ethanol (1500 mL). The reaction was heated at reflux for 50 min and then cooled to rt. The catalyst was filtered off and the filtrate was concentrated in vacuo to give the crude product that was triturated under DCM (500 mL) and left to stand for 1 h. A solid was removed by filtration and the resulting yellow filtrate was concentrated in vacuo to give the title compound (39.4 g, 98%) as a pale yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.46 (9H. s), 1.95–2.08 (1H, m), 2.25–2.37 (1H, m), 3.43 (1H, dd, <i>J</i> = 4.3, 10.8 Hz), 3.53–3.73 (2H, m), 3.79 (1H, dd, <i>J</i> = 6.1, 10.8 Hz), 4.37 (1H, br), 4.80 (1H, d, <i>J</i> = 6.5 Hz), 6.63 (1H, dd, <i>J</i> = 1.3, 9.1 Hz), 7.19 (1H, dd, <i>J</i> = 4.5, 9.1 Hz), 8.54 (1H, dd, <i>J</i> = 1.3, 4.5 Hz); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 265.</div></div><div id="sec5_1_19_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> (<i>R</i>)-1-(Pyridazin-3-yl)pyrrolidin-3-amine (<b>43</b>)</h4><div class="NLM_p last">(<i>R</i>)-<i>tert</i>-Butyl (1-(pyridazin-3-yl)pyrrolidin-3-yl)carbamate (2.4 g, 9.08 mmol) was added to a stirred mixture of DCM (40 mL) and TFA (20 mL) at rt. The resulting solution was stirred at rt for 3 h. The crude product was purified by ion-exchange chromatography, using an SCX-2 column. The desired product was eluted from the column using 2 M NH<sub>3</sub>/methanol to afford the title compound (1.38 g, 93%) as a gum, which partially solidified on standing. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.69–1.81 (1H, m), 2.03–2.15 (1H, m), 3.11–3.19 (1H, m), 3.44 (1H, ddd, <i>J</i> = 5.8, 7.8, 10.3 Hz), 3.51–3.68 (3H, m), 6.79 (1H, dd, <i>J</i> = 1.3, 9.2 Hz), 7.30 (1H, dd, <i>J</i> = 4.4, 9.1 Hz), 8.45 (1H, dd, <i>J</i> = 1.3, 4.4 Hz); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 165.</div></div><div id="sec5_1_19_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> (<i>R</i>)-<i>N</i>2-(1-(Pyridazin-3-yl)pyrrolidin-3-yl)-1,3,4-thiadiazole-2,5-diamine</h4><div class="NLM_p last">(<i>R</i>)-1-(Pyridazin-3-yl)pyrrolidin-3-amine (<b>43</b>) (25.5 g, 150.63 mmol), 5-bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (29.8 g, 165.70 mmol), and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (39.4 mL, 225.95 mmol) were agitated as a slurry in methanol (200 mL) at 45 °C for 4 h. The slurry was cooled to 20 °C and the solid was isolated by vacuum filtration, washed with methanol (50 mL), and dried overnight in a vacuum oven at 40 °C to afford the title compound (32.9 g, 83%) as a beige powder. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.99–2.09 (1H, m), 2.20–2.32 (1H, m), 3.42–3.63 (3H, m), 3.72 (1H, dd, <i>J</i> = 5.9, 11.0 Hz), 4.23–4.33 (1H, m), 6.26 (2H, s), 6.85 (1H, dd, <i>J</i> = 1.3, 9.2 Hz), 7.07 (1H, d, <i>J</i> = 6.0 Hz), 7.32 (1H, dd, <i>J</i> = 4.5, 9.1 Hz), 8.47 (1H, dd, <i>J</i> = 1.3, 4.5 Hz); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 264.</div></div><div id="sec5_1_19_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> (<i>S</i>)-2-Methoxy-2-phenyl-<i>N</i>-(5-(((<i>R</i>)-1-(pyridazin-3-yl)pyrrolidin-3-yl)amino)-1,3,4-thiadiazol-2-yl)acetamide (<b>27</b>)</h4><div class="NLM_p last">1-[Bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (237 mg, 0.62 mmol) was added to (<i>S</i>)-2-methoxy-2-phenylacetic acid (86 mg, 0.52 mmol), (<i>R</i>)-<i>N</i>2-(1-(pyridazin-3-yl)pyrrolidin-3-yl)-1,3,4-thiadiazole-2,5-diamine (137 mg, 0.52 mmol), and 2,4,6-trimethylpyridine (189 mg, 1.56 mmol) in a mixture of DMF (3 mL) and NMP (1 mL) at 0 °C. The resulting solution was stirred at 0 °C for 1 h and then at rt for 18 h. The reaction mixture was diluted with methanol (5 mL) and purified by ion-exchange chromatography, using an SCX-2 column. The desired product was eluted from the column using 1 M NH<sub>3</sub>/methanol to afford crude product, which was purified by flash silica chromatography, elution gradient 0–8% methanol in DCM to afford the title compound (184 mg, 86%) as a gum. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.02–2.13 (1H, m), 2.24–2.36 (1H, m), 3.29 (3H, s), 3.45–3.63 (3H, m), 3.75 (1H, dd, <i>J</i> = 5.8, 11.1 Hz), 4.33–4.44 (1H, m), 4.99 (1H, s), 6.86 (1H, dd, <i>J</i> = 1.3, 9.2 Hz), 7.29–7.42 (4H, m), 7.47 (2H, dd, <i>J</i> = 1.4, 8.1 Hz), 7.67 (1H, d, <i>J</i> = 6.1 Hz), 8.48 (1H, dd, <i>J</i> = 1.3, 4.5 Hz), 12.17 (1H, s); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 31.02, 44.99, 52.39, 53.84, 57.34, 82.55, 112.39, 127.60, 127.67, 128.89, 128.96, 137.29, 142.56, 148.95, 158.18, 164.21, 168.77; HRMS (ES<sup>+</sup>) for C<sub>19</sub>H<sub>22</sub>N<sub>7</sub>O<sub>2</sub>S [M + H]<sup>+</sup>: calcd, 412.1556; found, 412.1576.</div></div></div><div id="sec5_1_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> (2<i>S</i>)-2-Phenyl-<i>N</i>-[5-[[(3<i>R</i>)-1-pyridazin-3-ylpyrrolidin-3-yl]amino]-1,3,4-thiadiazol-2-yl]propanamide (<b>28</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0043.gif" alt="" id="fx20" /></img><div class="NLM_p last">(<i>R</i>)-<i>N</i>2-(1-(Pyridazin-3-yl)pyrrolidin-3-yl)-1,3,4-thiadiazole-2,5-diamine (1.0 g, 3.80 mmol), (2<i>S</i>)-2-phenylpropanoic acid (0.57 g, 3.80 mmol), 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (2.8 g), and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (1.2 g) in DMF (15 mL) were stirred for 2 h at rt. The reaction mixture was diluted with water (30 mL). The aqueous layer was then extracted with ethyl acetate (3 × 30 mL) and the organic phases were combined and concentrated in vacuo to give a crude product that was purified by prep-HPLC A/H<sub>2</sub>O (0.05% HCOOH), B: methanol; A/B: 5–60% to afford the title compound (263 mg, 16%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.43 (3H, d), 2.00–2.17 (1H, m), 2.21– 2.36 (1H, m), 3.48–3.70 (3H, m), 3.77–3.82 (1H, m), 3.90–4.12 (1H, m), 4.35–4.46 (1H, m), 7.11 (1H, d), 7.20–7.38 (5H, m), 7.52 (1H, t), 7.71 (1H, d), 8.52 (1H, d), 12.28 (1H, s); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 18.19, 30.40, 44.51, 44.95, 52.21, 53.25, 114.50, 126.97, 127.32, 128.46, 140.80, 141.83, 149.07, 156.29, 162.94, 163.42, 171.60; HRMS (ES<sup>+</sup>) for C<sub>19</sub>H<sub>22</sub>N<sub>7</sub>OS [M + H]<sup>+</sup>: calcd, 396.1607; found, 396.1616.</div></div><div id="sec5_1_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> (<i>S</i>)-<i>N</i>-(5-(((<i>R</i>)-1-(1,2,4-Triazin-3-yl)pyrrolidin-3-yl)amino)-1,3,4-thiadiazol-2-yl)-2-methoxy-2-phenylacetamide (<b>29</b>)</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0044.gif" alt="" id="fx21" /></img><div class="NLM_p last">5-Bromo-1,3,4-thiadiazol-2-amine (<b>30</b>) (115 mg, 0.63 mmol), (<i>R</i>)-1-(1,2,4-triazin-3-yl)pyrrolidin-3-amine (<b>40</b>) (100 mg, 0.61 mmol), and <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.126 mL, 0.72 mmol) were dissolved in DMF (2.5 mL). The reaction mixture was heated to 100 °C for 90 min in a microwave reactor. After cooling to 0 °C, (<i>S</i>)-2-methoxy-2-phenylacetic acid (106 mg, 0.64 mmol), <i>N</i>-ethyl-<i>N</i>-(propan-2-yl)propan-2-amine (0.316 mL, 1.82 mmol), and 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (276 mg, 0.73 mmol) were added. The resulting solution was then stirred at rt for 2 h. The reaction mixture was partitioned between 2-methyltetrahydrofuran and brine. The organic layer was separated, dried, filtered and evaporated under reduced pressure to give a crude product that was purified by preparative HPLC (Waters XBridge Prep C18 OBD column, 5 μm silica, 50 mm diameter, 100 mm length), using decreasingly polar mixtures of water (containing 1% NH<sub>3</sub>) and acetonitrile as eluents to afford the title compound (118 mg, 47%) as a solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.09–2.20 (1H, m), 2.30–2.42 (1H, m), 3.37 (3H, s), 3.71 (2H, br), 3.87 (1H, br), 4.12 (1H, q, <i>J</i> = 5.2 Hz), 4.37–4.49 (1H, m), 5.04 (1H, s), 7.37–7.47 (3H, m), 7.49–7.55 (2H, m), 7.74 (1H, d, <i>J</i> = 5.9 Hz), 8.36 (1H, d, <i>J</i> = 2.2 Hz), 8.66 (1H, d, <i>J</i> = 2.4 Hz), 12.14 (1H, br); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 30.39, 44.36, 51.61, 53.23, 56.83, 82.04, 127.09, 128.38, 128.45, 136.78, 139.89, 148.46, 149.46, 159.46, 163.64, 168.24; HRMS (ES<sup>+</sup>) for C<sub>18</sub>H<sub>21</sub>N<sub>8</sub>O<sub>2</sub>S [M + H]<sup>+</sup>: calcd, 413.1508; found, 413.1515.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> Cellular Assays</h3><div class="NLM_p last">Cellular mode of action and proliferation assays were performed as reported previously by ourselves.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> All IC<sub>50</sub> data are quoted as geometric means.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Enzyme Assay</h3><div class="NLM_p last">The GLS1 biochemical assay was performed as reported previously by ourselves.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> All IC<sub>50</sub> data are quoted as geometric means.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> In Vivo Antitumor Efficacy Studies</h3><div class="NLM_p">All animal studies were conducted in accordance with U.K. Home Office legislation, the Animal Scientific Procedures Act 1986, as well as the AstraZeneca Global Bioethics policy.</div><div class="NLM_p">NCI-H1703 xenografts were established by subcutaneous implantation of 1 × 10<sup>7</sup> cells per animal, in 100 μL of cell suspension including 50% Matrigel, into the dorsal left flank of male SCID mice (Charles River Laboratories).</div><div class="NLM_p">Randomization of animals onto study was based on initial tumor volumes to ensure equal distribution across groups. A power analysis was performed, whereby group sizes were calculated to enable statistically robust detection of tumor growth inhibition and pharmacodynamic endpoint.</div><div class="NLM_p">Animals were randomized into groups of 10–12 when tumors reached a volume of ca. 200–300 mm<sup>3</sup> and treatment commenced. Treatment groups received compound <b>27</b> dosed at 100 mg/kg once daily by oral gavage or taxotere dosed at 7.5 mg/kg once weekly by intravenous route. Compound <b>27</b> was formulated in 1% polysorbate 80 and pH was adjusted with 1 M HCl to a final pH concentration of pH 3.5 and taxotere (Sanofi) was formulated in a physiological saline.</div><div class="NLM_p">In the subcutaneous xenograft model, tumors were measured three times weekly by caliper and volume was calculated using elliptical formula (π/6 × width × width × length). Tumor growth inhibition (%TGI) from the start of treatment was assessed by comparison of the geometric mean change in tumor volume for the control and treated groups. Statistical significance was evaluated using a one-tailed <i>t</i> test.</div><div class="NLM_p last">Following the final dose of compound <b>27</b>, tumors were excised postmortem from the tumor growth inhibition study at specified time points and frozen for analysis of pharmacodynamic end points. In addition, terminal plasma samples were taken to allow determination of exposure.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i81"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00260" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08357" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08357" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00260" class="ext-link">10.1021/acs.jmedchem.9b00260</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Enzyme/cell correlation plots (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00260/suppl_file/jm9b00260_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00260/suppl_file/jm9b00260_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00260/suppl_file/jm9b00260_si_001.pdf">jm9b00260_si_001.pdf (311.81 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00260/suppl_file/jm9b00260_si_002.csv">jm9b00260_si_002.csv (2.18 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last"><b>3</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGAA">3KGAA</a>), <b>12</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KGAC">1KGAC</a>), <b>18</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6KGAB">6KGAB</a>), and <b>27</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KGAD">7KGAD</a>). The authors will release the atomic coordinates upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00260" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53663" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53663" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">M. Raymond V. Finlay</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1402-193X" title="Orcid link">http://orcid.org/0000-0003-1402-193X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#85f7e4fcabe3ecebe9e4fcc5e4f6f1f7e4ffe0ebe0e6e4abe6eae8"><span class="__cf_email__" data-cfemail="493b2830672f202725283009283a3d3b28332c272c2a28672a2624">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark Charles</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research
Campus, Cambridge CB22
3AT, United Kingdom</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#733e1201185d301b12011f16003310121d1016015d1c01145d0618"><span class="__cf_email__" data-cfemail="511c30233a7f123930233d34221132303f3234237f3e23367f243a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark Anderton</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew Bailey</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott Boyd</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joanna Brookfield</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research
Campus, Cambridge CB22
3AT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ceri Cairnduff</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research
Campus, Cambridge CB22
3AT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anne Cheasty</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research
Campus, Cambridge CB22
3AT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Susan E. Critchlow</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Janet Culshaw</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tennyson Ekwuru</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research
Campus, Cambridge CB22
3AT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ian Hollingsworth</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Neil Jones</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research
Campus, Cambridge CB22
3AT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fred Leroux</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research
Campus, Cambridge CB22
3AT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mairi Littleson</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6909-5608" title="Orcid link">http://orcid.org/0000-0001-6909-5608</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hollie McCarron</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research
Campus, Cambridge CB22
3AT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jennifer McKelvie</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research
Campus, Cambridge CB22
3AT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lorraine Mooney</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span>; 
    <span>Present Address:
                        Alderley Park Limited, Preclinical Services, Alderley Park, Macclesfield SK10 4TG, United Kingdom (L.M.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Willem M. Nissink</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2572-9140" title="Orcid link">http://orcid.org/0000-0003-2572-9140</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Perkins</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steve Powell</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mar Jimenez Quesada</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research
Campus, Cambridge CB22
3AT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Piotr Raubo</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Verity Sabin</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research
Campus, Cambridge CB22
3AT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James Smith</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK, Therapeutic Discovery Laboratories, Jonas Webb Building, Babraham Research
Campus, Cambridge CB22
3AT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter D. Smith</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5312-9893" title="Orcid link">http://orcid.org/0000-0001-5312-9893</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew Stark</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Attilla Ting</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peng Wang</span> - <span class="hlFld-Affiliation affiliation">Pharmaron
Beijing Co., Ltd., 6
Taihe Road, BDA, Beijing 100176, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zena Wilson</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jon J. Winter-Holt</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Matthew Wood</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gail L. Wrigley</span> - <span class="hlFld-Affiliation affiliation">Oncology,
IMED Biotech Unit, AstraZeneca, 310, Cambridge Science Park, Milton
Road, Cambridge CB4 0FZ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guoqing Yu</span> - <span class="hlFld-Affiliation affiliation">Pharmaron
Beijing Co., Ltd., 6
Taihe Road, BDA, Beijing 100176, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peng Zhang</span> - <span class="hlFld-Affiliation affiliation">Pharmaron
Beijing Co., Ltd., 6
Taihe Road, BDA, Beijing 100176, P. R. China</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e6819-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i84">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20115" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20115" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Natalie Stratton for biological assay support, Joe Patel and Judit Debreczeni for crystallographic support, and the members of the Oncology Purification Team (Neil Sumner, Michael Wiggins, and Caroline McMillan) for their technical expertise in delivering this work. They are also grateful to Thorsten Nowak, Andrew Campbell, Teyrnon Jones, Nicola Colclough, and Teresa Klinowska for helpful discussions.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">Cl<sub>int</sub></td><td class="NLM_def"><p class="first last">intrinsic clearance</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N,N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate</p></td></tr><tr><td class="NLM_term">SCID</td><td class="NLM_def"><p class="first last">severe combined immunodeficiency</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i86">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38267" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38267" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 22 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gui, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Targeting metabolism for cancer therapy</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1161</span>– <span class="NLM_lpage">1180</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.08.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1016%2Fj.chembiol.2017.08.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=28938091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1161-1180&author=A.+Luengoauthor=D.+Y.+Guiauthor=M.+G.+Vander%0AHeiden&title=Targeting+metabolism+for+cancer+therapy&doi=10.1016%2Fj.chembiol.2017.08.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Metabolism for Cancer Therapy</span></div><div class="casAuthors">Luengo, Alba; Gui, Dan Y.; Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1161-1180</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Metabolic reprogramming contributes to tumor development and introduces metabolic liabilities that can be exploited to treat cancer.  Chemotherapies targeting metab. have been effective cancer treatments for decades, and the success of these therapies demonstrates that a therapeutic window exists to target malignant metab.  New insights into the differential metabolic dependencies of tumors have provided novel therapeutic strategies to exploit altered metab., some of which are being evaluated in preclin. models or clin. trials.  Here, we review our current understanding of cancer metab. and discuss how this might guide treatments targeting the metabolic requirements of tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUOppBTjvuc7Vg90H21EOLACvtfcHk0liEZNzzwv1q7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurfO&md5=c702238d8642e06df9accd5414b64936</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.08.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.08.028%26sid%3Dliteratum%253Aachs%26aulast%3DLuengo%26aufirst%3DA.%26aulast%3DGui%26aufirst%3DD.%2BY.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DTargeting%2520metabolism%2520for%2520cancer%2520therapy%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1161%26epage%3D1180%26doi%3D10.1016%2Fj.chembiol.2017.08.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Alfarouk, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verduzco, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muddathir, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adil, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elhassan, G. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David
Polo Orozco, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardone, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reshkin, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harguindey, S.</span></span> <span> </span><span class="NLM_article-title">Glycolysis, tumor metabolism, cancer growth and dissemination. A new pH-based etiopathogenic perspective and therapeutic approach to an old cancer question</span>. <i>Oncoscience</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">777</span>– <span class="NLM_lpage">802</span>, <span class="refDoi"> DOI: 10.18632/oncoscience.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.18632%2Foncoscience.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=25621294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A280%3ADC%252BC2MvpvFOmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2014&pages=777-802&author=K.+O.+Alfaroukauthor=D.+Verduzcoauthor=C.+Rauchauthor=A.+K.+Muddathirauthor=H.+H.+Adilauthor=G.+O.+Elhassanauthor=M.+E.+Ibrahimauthor=J.+David%0APolo+Orozcoauthor=R.+A.+Cardoneauthor=S.+J.+Reshkinauthor=S.+Harguindey&title=Glycolysis%2C+tumor+metabolism%2C+cancer+growth+and+dissemination.+A+new+pH-based+etiopathogenic+perspective+and+therapeutic+approach+to+an+old+cancer+question&doi=10.18632%2Foncoscience.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">Glycolysis, tumor metabolism, cancer growth and dissemination. A new pH-based etiopathogenic perspective and therapeutic approach to an old cancer question</span></div><div class="casAuthors">Alfarouk Khalid O; Muddathir Abdel Khalig; Adil H H Bashir; Ibrahim Muntaser E; Verduzco Daniel; Rauch Cyril; Elhassan Gamal O; David Polo Orozco Julian; Harguindey Salvador; Cardone Rosa Angela; Reshkin Stephan J</div><div class="citationInfo"><span class="NLM_cas:title">Oncoscience</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">777-802</span>
        ISSN:<span class="NLM_cas:issn">2331-4737</span>.
    </div><div class="casAbstract">Cancer cells acquire an unusual glycolytic behavior relative, to a large extent, to their intracellular alkaline pH (pHi).  This effect is part of the metabolic alterations found in most, if not all, cancer cells to deal with unfavorable conditions, mainly hypoxia and low nutrient supply, in order to preserve its evolutionary trajectory with the production of lactate after ten steps of glycolysis.  Thus, cancer cells reprogram their cellular metabolism in a way that gives them their evolutionary and thermodynamic advantage.  Tumors exist within a highly heterogeneous microenvironment and cancer cells survive within any of the different habitats that lie within tumors thanks to the overexpression of different membrane-bound proton transporters.  This creates a highly abnormal and selective proton reversal in cancer cells and tissues that is involved in local cancer growth and in the metastatic process.  Because of this environmental heterogeneity, cancer cells within one part of the tumor may have a different genotype and phenotype than within another part.  This phenomenon has frustrated the potential of single-target therapy of this type of reductionist therapeutic approach over the last decades.  Here, we present a detailed biochemical framework on every step of tumor glycolysis and then proposea new paradigm and therapeutic strategy based upon the dynamics of the hydrogen ion in cancer cells and tissues in order to overcome the old paradigm of one enzyme-one target approach to cancer treatment.  Finally, a new and integral explanation of the Warburg effect is advanced.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREdAEVGEC34exsqfvU0ziafW6udTcc2eZlVn0cPmWUiLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MvpvFOmsw%253D%253D&md5=c5014218b1ad904214a0f04a9318f03a</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.18632%2Foncoscience.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncoscience.109%26sid%3Dliteratum%253Aachs%26aulast%3DAlfarouk%26aufirst%3DK.%2BO.%26aulast%3DVerduzco%26aufirst%3DD.%26aulast%3DRauch%26aufirst%3DC.%26aulast%3DMuddathir%26aufirst%3DA.%2BK.%26aulast%3DAdil%26aufirst%3DH.%2BH.%26aulast%3DElhassan%26aufirst%3DG.%2BO.%26aulast%3DIbrahim%26aufirst%3DM.%2BE.%26aulast%3DDavid%2BPolo%2BOrozco%26aufirst%3DJ.%26aulast%3DCardone%26aufirst%3DR.%2BA.%26aulast%3DReshkin%26aufirst%3DS.%2BJ.%26aulast%3DHarguindey%26aufirst%3DS.%26atitle%3DGlycolysis%252C%2520tumor%2520metabolism%252C%2520cancer%2520growth%2520and%2520dissemination.%2520A%2520new%2520pH-based%2520etiopathogenic%2520perspective%2520and%2520therapeutic%2520approach%2520to%2520an%2520old%2520cancer%2520question%26jtitle%3DOncoscience%26date%3D2014%26volume%3D1%26spage%3D777%26epage%3D802%26doi%3D10.18632%2Foncoscience.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit2b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Alfarouk, K. O.</span></span> <span> </span><span class="NLM_article-title">Tumor metabolism, cancer cell transporters, and microenvironmental resistance</span>. <i>J. Enzyme Inhib. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">866</span>, <span class="refDoi"> DOI: 10.3109/14756366.2016.1140753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.3109%2F14756366.2016.1140753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=26864256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC28XisFGrsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2016&pages=859-866&author=K.+O.+Alfarouk&title=Tumor+metabolism%2C+cancer+cell+transporters%2C+and+microenvironmental+resistance&doi=10.3109%2F14756366.2016.1140753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor metabolism, cancer cell transporters, and microenvironmental resistance</span></div><div class="casAuthors">Alfarouk, Khalid O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Enzyme Inhibition and Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">JEIMAZ</span>;
        ISSN:<span class="NLM_cas:issn">1475-6366</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Cancer cells reprogram their metabolic machineries to enter into permanent glycolytic pathways.  The full reason for such reprogramming takes place is unclear.  However, this metabolic switch is not made in vain for the lactate that is generated and exported outside cells is reused by other cells.  This results in the generation of a pH gradient between the low extracellular pH that is acidic (pHe) and the higher cytosolic alk. or near neutral pH (pHi) environments that are tightly regulated by the overexpression of several pumps and ion channels (e.g. NHE-1, MCT-1, V-ATPase, CA9, and CA12).  The generation of this unique pH gradient serves as a detg. factor in defining "tumor fitness".  Tumor fitness is the capacity of the tumor to invade and metastasize due to its ability to reduce the efficiency of the immune system and confer resistance to chemotherapy.  In this article, we highlight the importance of tumor microenvironment in mediating the failure of chemotherapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAjPfA3PdelLVg90H21EOLACvtfcHk0li8rCsUFzkQsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisFGrsbw%253D&md5=881bbff08a460d2cbe01ee667271fa6a</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.3109%2F14756366.2016.1140753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F14756366.2016.1140753%26sid%3Dliteratum%253Aachs%26aulast%3DAlfarouk%26aufirst%3DK.%2BO.%26atitle%3DTumor%2520metabolism%252C%2520cancer%2520cell%2520transporters%252C%2520and%2520microenvironmental%2520resistance%26jtitle%3DJ.%2520Enzyme%2520Inhib.%2520Med.%2520Chem.%26date%3D2016%26volume%3D31%26spage%3D859%26epage%3D866%26doi%3D10.3109%2F14756366.2016.1140753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hensley, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasti, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeBerardinis, R. J.</span></span> <span> </span><span class="NLM_article-title">Glutamine and cancer: cell biology, physiology, and clinical opportunities</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">3678</span>– <span class="NLM_lpage">3684</span>, <span class="refDoi"> DOI: 10.1172/JCI69600</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1172%2FJCI69600" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=23999442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVenu7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2013&pages=3678-3684&author=C.+T.+Hensleyauthor=A.+T.+Wastiauthor=R.+J.+DeBerardinis&title=Glutamine+and+cancer%3A+cell+biology%2C+physiology%2C+and+clinical+opportunities&doi=10.1172%2FJCI69600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamine and cancer: cell biology, physiology, and clinical opportunities</span></div><div class="casAuthors">Hensley, Christopher T.; Wasti, Ajla T.; DeBerardinis, Ralph J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3678-3684</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">A review.  Glutamine is an abundant and versatile nutrient that participates in energy formation, redox homeostasis, macromol. synthesis, and signaling in cancer cells.  These characteristics make glutamine metab. an appealing target for new clin. strategies to detect, monitor, and treat cancer.  Here the authors review the metabolic functions of glutamine as a super nutrient and the surprising roles of glutamine in supporting the biol. hallmarks of malignancy.  The authors also review recent efforts in imaging and therapeutics to exploit tumor cell glutamine dependence, discuss some of the challenges in this arena, and suggest a disease-focused paradigm to deploy these emerging approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocNnK_GlTtRrVg90H21EOLACvtfcHk0lg6HZoN-v9cWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVenu7bL&md5=fb704a7773710a20add6edbf0da08616</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1172%2FJCI69600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI69600%26sid%3Dliteratum%253Aachs%26aulast%3DHensley%26aufirst%3DC.%2BT.%26aulast%3DWasti%26aufirst%3DA.%2BT.%26aulast%3DDeBerardinis%26aufirst%3DR.%2BJ.%26atitle%3DGlutamine%2520and%2520cancer%253A%2520cell%2520biology%252C%2520physiology%252C%2520and%2520clinical%2520opportunities%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2013%26volume%3D123%26spage%3D3678%26epage%3D3684%26doi%3D10.1172%2FJCI69600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">DeBerardinis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancuso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daikhin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissim, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yudkoff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wehrli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span> <span> </span><span class="NLM_article-title">Beyond aerobic glycolysis: Transformed cells can engage in glutamine metabolism that exceeds the requirement for protein and nucleotide synthesis</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">19345</span>– <span class="NLM_lpage">19350</span>, <span class="refDoi"> DOI: 10.1073/pnas.0709747104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1073%2Fpnas.0709747104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=18032601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisVOjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=19345-19350&author=R.+J.+DeBerardinisauthor=A.+Mancusoauthor=E.+Daikhinauthor=I.+Nissimauthor=M.+Yudkoffauthor=S.+Wehrliauthor=C.+B.+Thompson&title=Beyond+aerobic+glycolysis%3A+Transformed+cells+can+engage+in+glutamine+metabolism+that+exceeds+the+requirement+for+protein+and+nucleotide+synthesis&doi=10.1073%2Fpnas.0709747104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4aR"><div class="casContent"><span class="casTitleNuber">4a</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond aerobic glycolysis: transformed cells can engage in glutamine metabolism that exceeds the requirement for protein and nucleotide synthesis</span></div><div class="casAuthors">DeBerardinis, Ralph J.; Mancuso, Anthony; Daikhin, Evgueni; Nissim, Ilana; Yudkoff, Marc; Wehrli, Suzanne; Thompson, Craig B.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19345-19350</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Tumor cell proliferation requires rapid synthesis of macromols. including lipids, proteins, and nucleotides.  Many tumor cells exhibit rapid glucose consumption, with most of the glucose-derived carbon being secreted as lactate despite abundant oxygen availability (the Warburg effect).  Here, we used 13C NMR spectroscopy to examine the metab. of glioblastoma cells exhibiting aerobic glycolysis.  In these cells, the tricarboxylic acid (TCA) cycle was active but was characterized by an efflux of substrates for use in biosynthetic pathways, particularly fatty acid synthesis.  The success of this synthetic activity depends on activation of pathways to generate reductive power (NADPH) and to restore oxaloacetate for continued TCA cycle function (anaplerosis).  Surprisingly, both these needs were met by a high rate of glutamine metab.  First, conversion of glutamine to lactate (glutaminolysis) was rapid enough to produce sufficient NADPH to support fatty acid synthesis.  Second, despite substantial mitochondrial pyruvate metab., pyruvate carboxylation was suppressed, and anaplerotic oxaloacetate was derived from glutamine.  Glutamine catabolism was accompanied by secretion of alanine and ammonia, such that most of the amino groups from glutamine were lost from the cell rather than incorporated into other mols.  These data demonstrate that transformed cells exhibit a high rate of glutamine consumption that cannot be explained by the nitrogen demand imposed by nucleotide synthesis or maintenance of nonessential amino acid pools.  Rather, glutamine metab. provides a carbon source that facilitates the cell's ability to use glucose-derived carbon and TCA cycle intermediates as biosynthetic precursors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoggsgqrqrWmrVg90H21EOLACvtfcHk0lg6HZoN-v9cWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisVOjtQ%253D%253D&md5=786a1c001dd9c477ea50d9227ebfece0</span></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0709747104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0709747104%26sid%3Dliteratum%253Aachs%26aulast%3DDeBerardinis%26aufirst%3DR.%2BJ.%26aulast%3DMancuso%26aufirst%3DA.%26aulast%3DDaikhin%26aufirst%3DE.%26aulast%3DNissim%26aufirst%3DI.%26aulast%3DYudkoff%26aufirst%3DM.%26aulast%3DWehrli%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DBeyond%2520aerobic%2520glycolysis%253A%2520Transformed%2520cells%2520can%2520engage%2520in%2520glutamine%2520metabolism%2520that%2520exceeds%2520the%2520requirement%2520for%2520protein%2520and%2520nucleotide%2520synthesis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D19345%26epage%3D19350%26doi%3D10.1073%2Fpnas.0709747104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit4b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wise, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span> <span> </span><span class="NLM_article-title">Glutamine addiction: a new therapeutic target in cancer</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">427</span>– <span class="NLM_lpage">433</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2010.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1016%2Fj.tibs.2010.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=20570523" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvFKqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2010&pages=427-433&author=D.+R.+Wiseauthor=C.+B.+Thompson&title=Glutamine+addiction%3A+a+new+therapeutic+target+in+cancer&doi=10.1016%2Fj.tibs.2010.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4bR"><div class="casContent"><span class="casTitleNuber">4b</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamine addiction: A new therapeutic target in cancer</span></div><div class="casAuthors">Wise, David R.; Thompson, Craig B.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">427-433</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Most cancers depend on a high rate of aerobic glycolysis for their continued growth and survival.  Paradoxically, some cancer cell lines also display addiction to glutamine despite the fact that glutamine is a nonessential amino acid that can be synthesized from glucose.  The high rate of glutamine uptake exhibited by glutamine-dependent cells does not appear to result solely from its role as a nitrogen donor in nucleotide and amino acid biosynthesis.  Instead, glutamine plays a required role in the uptake of essential amino acids and in maintaining activation of TOR (target of rapamycin) kinase.  Moreover, in many cancer cells, glutamine is the primary mitochondrial substrate and is required for maintenance of mitochondrial membrane potential and integrity and for support of the NADPH prodn. needed for redox control and macromol. synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4Gv_LsNdulLVg90H21EOLACvtfcHk0lg6HZoN-v9cWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvFKqs7s%253D&md5=915fa51996bc1c9b3ec3970b2dae7163</span></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2010.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2010.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DWise%26aufirst%3DD.%2BR.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DGlutamine%2520addiction%253A%2520a%2520new%2520therapeutic%2520target%2520in%2520cancer%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2010%26volume%3D35%26spage%3D427%26epage%3D433%26doi%3D10.1016%2Fj.tibs.2010.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit4c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span> <span> </span><span class="NLM_article-title">Glutaminolysis: supplying carbon or nitrogen or both for cancer cells?</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3884</span>– <span class="NLM_lpage">3886</span>, <span class="refDoi"> DOI: 10.4161/cc.9.19.13302</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.4161%2Fcc.9.19.13302" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=20948290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVentrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=3884-3886&author=C.+V.+Dang&title=Glutaminolysis%3A+supplying+carbon+or+nitrogen+or+both+for+cancer+cells%3F&doi=10.4161%2Fcc.9.19.13302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4cR"><div class="casContent"><span class="casTitleNuber">4c</span><div class="casTitle"><span class="NLM_cas:atitle">Glutaminolysis: supplying carbon or nitrogen or both for cancer cells?</span></div><div class="casAuthors">Dang, Chi V.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3884-3886</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  A cancer cell comprised largely of C, H, O, P, N and S requires not only glucose, which is avidly transported and converted to lactate by aerobic glycolysis or the Warburg effect, but also glutamine as a major substrate.  Glutamine and essential amino acids, such as Met, provide energy through the TCA cycle as well as N, S and C skeletons for growing and proliferating cancer cells.  The interplay between utilization of glutamine and glucose is likely to depend on the genetic make-up of a cancer cell.  While the MYC oncogene induces both aerobic glycolysis and glutaminolysis, activated β-catenin induces glutamine synthesis in hepatocellular carcinoma.  Cancer cells that have elevated glutamine synthetase can use glutamate and ammonia to synthesize glutamine and are hence not addicted to glutamine.  As such, cancer cells have many degrees of freedom for re-programming cell metab., which with better understanding will result in novel therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolGtCajTcmv7Vg90H21EOLACvtfcHk0liCIftZpmVZKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVentrY%253D&md5=a823f452698f7f082eb90873bc9bb2a1</span></div><a href="/servlet/linkout?suffix=cit4c&amp;dbid=16384&amp;doi=10.4161%2Fcc.9.19.13302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.9.19.13302%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26atitle%3DGlutaminolysis%253A%2520supplying%2520carbon%2520or%2520nitrogen%2520or%2520both%2520for%2520cancer%2520cells%253F%26jtitle%3DCell%2520Cycle%26date%3D2010%26volume%3D9%26spage%3D3884%26epage%3D3886%26doi%3D10.4161%2Fcc.9.19.13302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Altman, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stine, Z. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span> <span> </span><span class="NLM_article-title">From Krebs to clinic: glutamine metabolism to cancer therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">619</span>– <span class="NLM_lpage">634</span>, <span class="refDoi"> DOI: 10.1038/nrc.2016.71</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1038%2Fnrc.2016.71" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=27492215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12jurbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=619-634&author=B.+J.+Altmanauthor=Z.+E.+Stineauthor=C.+V.+Dang&title=From+Krebs+to+clinic%3A+glutamine+metabolism+to+cancer+therapy&doi=10.1038%2Fnrc.2016.71"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5aR"><div class="casContent"><span class="casTitleNuber">5a</span><div class="casTitle"><span class="NLM_cas:atitle">From Krebs to clinic: glutamine metabolism to cancer therapy</span></div><div class="casAuthors">Altman, Brian J.; Stine, Zachary E.; Dang, Chi V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">619-634</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The resurgence of research into cancer metab. has recently broadened interests beyond glucose and the Warburg effect to other nutrients, including glutamine.  Because oncogenic alterations of metab. render cancer cells addicted to nutrients, pathways involved in glycolysis or glutaminolysis could be exploited for therapeutic purposes.  In this Review, we provide an updated overview of glutamine metab. and its involvement in tumorigenesis in vitro and in vivo, and explore the recent potential applications of basic science discoveries in the clin. setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2GDHcwHXcnLVg90H21EOLACvtfcHk0liCIftZpmVZKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12jurbI&md5=00b34261d90a91151a3e833605c4aaec</span></div><a href="/servlet/linkout?suffix=cit5a&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2016.71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2016.71%26sid%3Dliteratum%253Aachs%26aulast%3DAltman%26aufirst%3DB.%2BJ.%26aulast%3DStine%26aufirst%3DZ.%2BE.%26aulast%3DDang%26aufirst%3DC.%2BV.%26atitle%3DFrom%2520Krebs%2520to%2520clinic%253A%2520glutamine%2520metabolism%2520to%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2016%26volume%3D16%26spage%3D619%26epage%3D634%26doi%3D10.1038%2Fnrc.2016.71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Koppula, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, B.</span></span> <span> </span><span class="NLM_article-title">Amino acid transporter SLC7A11/xCT at the crossroads of regulating redox homeostasis and nutrient dependency of cancer</span>. <i>Cancer Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1186/s40880-018-0288-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1186%2Fs40880-018-0288-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A280%3ADC%252BC1MfjsFOgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&pages=12-25&author=P.+Koppulaauthor=Y.+Zhangauthor=L.+Zhuangauthor=B.+Gan&title=Amino+acid+transporter+SLC7A11%2FxCT+at+the+crossroads+of+regulating+redox+homeostasis+and+nutrient+dependency+of+cancer&doi=10.1186%2Fs40880-018-0288-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5bR"><div class="casContent"><span class="casTitleNuber">5b</span><div class="casTitle"><span class="NLM_cas:atitle">Amino acid transporter SLC7A11/xCT at the crossroads of regulating redox homeostasis and nutrient dependency of cancer</span></div><div class="casAuthors">Koppula Pranavi; Zhang Yilei; Zhuang Li; Gan Boyi; Koppula Pranavi; Gan Boyi; Gan Boyi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer communications (London, England)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer cells often upregulate nutrient transporters to fulfill their increased biosynthetic and bioenergetic needs, and to maintain redox homeostasis.  One nutrient transporter frequently overexpressed in human cancers is the cystine/glutamate antiporter solute carrier family 7 member 11 (SLC7A11; also known as xCT).  SLC7A11 promotes cystine uptake and glutathione biosynthesis, resulting in protection from oxidative stress and ferroptotic cell death.  Recent studies have unexpectedly revealed that SLC7A11 also plays critical roles in glutamine metabolism and regulates the glucose and glutamine dependency of cancer cells.  This review discusses the roles of SLC7A11 in regulating the antioxidant response and nutrient dependency of cancer cells, explores our current understanding of SLC7A11 regulation in cancer metabolism, and highlights key open questions for future studies in this emerging research area.  A deeper understanding of SLC7A11 in cancer metabolism may identify new therapeutic opportunities to target this important amino acid transporter for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSc0x1bKxSR5v69nzq6467DfW6udTcc2eY_XEBXwTSRnbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MfjsFOgug%253D%253D&md5=22acaa68350c28038eb70f6974151c9f</span></div><a href="/servlet/linkout?suffix=cit5b&amp;dbid=16384&amp;doi=10.1186%2Fs40880-018-0288-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs40880-018-0288-x%26sid%3Dliteratum%253Aachs%26aulast%3DKoppula%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhuang%26aufirst%3DL.%26aulast%3DGan%26aufirst%3DB.%26atitle%3DAmino%2520acid%2520transporter%2520SLC7A11%252FxCT%2520at%2520the%2520crossroads%2520of%2520regulating%2520redox%2520homeostasis%2520and%2520nutrient%2520dependency%2520of%2520cancer%26jtitle%3DCancer%2520Commun.%26date%3D2018%26volume%3D38%26spage%3D12%26epage%3D25%26doi%3D10.1186%2Fs40880-018-0288-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Daemen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nannini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liederer, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Cruz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangaraju, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaochico, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunsaker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latham, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatzivassiliou, G.</span></span> <span> </span><span class="NLM_article-title">Pan-cancer metabolic signature predicts co-dependency on glutaminase and de novo glutathione synthesis linked to a high-mesenchymal cell state</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>, <span class="NLM_elocation-id">383.e9</span> <span class="refDoi"> DOI: 10.1016/j.cmet.2018.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1016%2Fj.cmet.2018.06.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&author=A.+Daemenauthor=B.+Liuauthor=K.+Songauthor=K.+Songauthor=M.+Gaoauthor=R.+Hongauthor=M.+Nanniniauthor=D.+Petersonauthor=B.+M.+Liedererauthor=C.+de+la+Cruzauthor=D.+Sangarajuauthor=A.+Jaochicoauthor=X.+Zhaoauthor=W.+Sandovalauthor=T.+Hunsakerauthor=R.+Firesteinauthor=S.+Lathamauthor=D.+Sampathauthor=D.+Sampathauthor=G.+Hatzivassiliou&title=Pan-cancer+metabolic+signature+predicts+co-dependency+on+glutaminase+and+de+novo+glutathione+synthesis+linked+to+a+high-mesenchymal+cell+state&doi=10.1016%2Fj.cmet.2018.06.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2018.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2018.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DDaemen%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DHong%26aufirst%3DR.%26aulast%3DNannini%26aufirst%3DM.%26aulast%3DPeterson%26aufirst%3DD.%26aulast%3DLiederer%26aufirst%3DB.%2BM.%26aulast%3Dde%2Bla%2BCruz%26aufirst%3DC.%26aulast%3DSangaraju%26aufirst%3DD.%26aulast%3DJaochico%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DSandoval%26aufirst%3DW.%26aulast%3DHunsaker%26aufirst%3DT.%26aulast%3DFirestein%26aufirst%3DR.%26aulast%3DLatham%26aufirst%3DS.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DHatzivassiliou%26aufirst%3DG.%26atitle%3DPan-cancer%2520metabolic%2520signature%2520predicts%2520co-dependency%2520on%2520glutaminase%2520and%2520de%2520novo%2520glutathione%2520synthesis%2520linked%2520to%2520a%2520high-mesenchymal%2520cell%2520state%26jtitle%3DCell%2520Metab.%26date%3D2018%26volume%3D28%26doi%3D10.1016%2Fj.cmet.2018.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit6b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayin, V. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeBoeuf, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karakousi, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhutkar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Rivera, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subbaraj, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prigge, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goparaju, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolgalev, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heguy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allaj, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreira, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudin, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pass, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacks, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papagiannakopoulos, T.</span></span> <span> </span><span class="NLM_article-title">Keap1 loss promotes Kras-driven lung cancer and results in dependence on glutaminolysis</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1362</span>– <span class="NLM_lpage">1368</span>, <span class="refDoi"> DOI: 10.1038/nm.4407</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1038%2Fnm.4407" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=28967920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFylurrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=1362-1368&author=R.+Romeroauthor=V.+I.+Sayinauthor=S.+M.+Davidsonauthor=M.+R.+Bauerauthor=S.+X.+Singhauthor=S.+E.+LeBoeufauthor=T.+R.+Karakousiauthor=D.+C.+Ellisauthor=A.+Bhutkarauthor=S.+J.+S%C3%A1nchez-Riveraauthor=L.+Subbarajauthor=B.+Martinezauthor=R.+T.+Bronsonauthor=J.+R.+Priggeauthor=E.+E.+Schmidtauthor=C.+J.+Thomasauthor=C.+Goparajuauthor=A.+Daviesauthor=I.+Dolgalevauthor=A.+Heguyauthor=V.+Allajauthor=J.+T.+Poirierauthor=A.+L.+Moreiraauthor=C.+M.+Rudinauthor=H.+I.+Passauthor=M.+G.+Vander%0AHeidenauthor=T.+Jacksauthor=T.+Papagiannakopoulos&title=Keap1+loss+promotes+Kras-driven+lung+cancer+and+results+in+dependence+on+glutaminolysis&doi=10.1038%2Fnm.4407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 loss promotes Kras-driven lung cancer and results in dependence on glutaminolysis</span></div><div class="casAuthors">Romero, Rodrigo; Sayin, Volkan I.; Davidson, Shawn M.; Bauer, Matthew R.; Singh, Simranjit X.; Le Boeuf, Sarah E.; Karakousi, Triantafyllia R.; Ellis, Donald C.; Bhutkar, Arjun; Sanchez-Rivera, Francisco J.; Subbaraj, Lakshmipriya; Martinez, Britney; Bronson, Roderick T.; Prigge, Justin R.; Schmidt, Edward E.; Thomas, Craig J.; Goparaju, Chandra; Davies, Angela; Dolgalev, Igor; Heguy, Adriana; Allaj, Viola; Poirier, John T.; Moreira, Andre L.; Rudin, Charles M.; Pass, Harvey I.; Vander Heiden, Matthew G.; Jacks, Tyler; Papagiannakopoulos, Thales</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1362-1368</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Treating KRAS-mutant lung adenocarcinoma (LUAD) remains a major challenge in cancer treatment given the difficulties assocd. with directly inhibiting the KRAS oncoprotein.  One approach to addressing this challenge is to define mutations that frequently co-occur with those in KRAS, which themselves may lead to therapeutic vulnerabilities in tumors.  Approx. 20% of KRAS-mutant LUAD tumors carry loss-of-function mutations in the KEAP1 gene encoding Kelch-like ECH-assocd. protein 1 (refs. 2, 3, 4), a neg. regulator of nuclear factor erythroid 2-like 2 (NFE2L2; hereafter NRF2), which is the master transcriptional regulator of the endogenous antioxidant response.  The high frequency of mutations in KEAP1 suggests an important role for the oxidative stress response in lung tumorigenesis.  Using a CRISPR-Cas9-based approach in a mouse model of KRAS-driven LUAD, we examd. the effects of Keap1 loss in lung cancer progression.  We show that loss of Keap1 hyperactivates NRF2 and promotes KRAS-driven LUAD in mice.  Through a combination of CRISPR-Cas9-based genetic screening and metabolomic analyses, we show that Keap1- or Nrf2-mutant cancers are dependent on increased glutaminolysis, and this property can be therapeutically exploited through the pharmacol. inhibition of glutaminase.  Finally, we provide a rationale for stratification of human patients with lung cancer harboring KRAS/KEAP1- or KRAS/NRF2-mutant lung tumors as likely to respond to glutaminase inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofWgHF0gdsW7Vg90H21EOLACvtfcHk0lgnYZyvnd2BAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFylurrK&md5=2bc4122a4c757cc3a5fdab3eb62b26e3</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1038%2Fnm.4407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4407%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DR.%26aulast%3DSayin%26aufirst%3DV.%2BI.%26aulast%3DDavidson%26aufirst%3DS.%2BM.%26aulast%3DBauer%26aufirst%3DM.%2BR.%26aulast%3DSingh%26aufirst%3DS.%2BX.%26aulast%3DLeBoeuf%26aufirst%3DS.%2BE.%26aulast%3DKarakousi%26aufirst%3DT.%2BR.%26aulast%3DEllis%26aufirst%3DD.%2BC.%26aulast%3DBhutkar%26aufirst%3DA.%26aulast%3DS%25C3%25A1nchez-Rivera%26aufirst%3DS.%2BJ.%26aulast%3DSubbaraj%26aufirst%3DL.%26aulast%3DMartinez%26aufirst%3DB.%26aulast%3DBronson%26aufirst%3DR.%2BT.%26aulast%3DPrigge%26aufirst%3DJ.%2BR.%26aulast%3DSchmidt%26aufirst%3DE.%2BE.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DGoparaju%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DA.%26aulast%3DDolgalev%26aufirst%3DI.%26aulast%3DHeguy%26aufirst%3DA.%26aulast%3DAllaj%26aufirst%3DV.%26aulast%3DPoirier%26aufirst%3DJ.%2BT.%26aulast%3DMoreira%26aufirst%3DA.%2BL.%26aulast%3DRudin%26aufirst%3DC.%2BM.%26aulast%3DPass%26aufirst%3DH.%2BI.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DJacks%26aufirst%3DT.%26aulast%3DPapagiannakopoulos%26aufirst%3DT.%26atitle%3DKeap1%2520loss%2520promotes%2520Kras-driven%2520lung%2520cancer%2520and%2520results%2520in%2520dependence%2520on%2520glutaminolysis%26jtitle%3DNat.%2520Med.%26date%3D2017%26volume%3D23%26spage%3D1362%26epage%3D1368%26doi%3D10.1038%2Fnm.4407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willis, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seegmiller, J. E.</span></span> <span> </span><span class="NLM_article-title">The inhibition by 6-diazo-5-oxo-<span class="smallcaps smallerCapital">l</span>-norleucine of glutamine catabolism of the cultured human lymphoblast</span>. <i>J. Cell Physiol.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">382</span>, <span class="refDoi"> DOI: 10.1002/jcp.1040930308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1002%2Fjcp.1040930308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=22551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADyaE1cXkvVShtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1977&pages=375-382&author=R.+C.+Willisauthor=J.+E.+Seegmiller&title=The+inhibition+by+6-diazo-5-oxo-l-norleucine+of+glutamine+catabolism+of+the+cultured+human+lymphoblast&doi=10.1002%2Fjcp.1040930308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The inhibition by 6-diazo-5-oxo-L-norleucine of glutamine catabolism of the cultured human lymphoblast</span></div><div class="casAuthors">Willis, Randall C.; Seegmiller, J. Edwin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular Physiology</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">375-82</span>CODEN:
                <span class="NLM_cas:coden">JCLLAX</span>;
        ISSN:<span class="NLM_cas:issn">0021-9541</span>.
    </div><div class="casAbstract">The rapid catabolism of glutamine by the cultured human lymphoblast line WI-L2 was inhibited by >95% during incubation of cell suspensions with 6-diazo-5-oxo-L-norleucine (I).  The inhibition persisted for ≥4 h after the removal of I from the cell suspension.  The exposure of cells to I inhibited >95% of the glutaminase activity measured in lysates in the presence of either phosphate or maleate.  Similarly, γ-glutamyl transpeptidase, assayed with γ-glutamyl-p-nitroanilide as substrate and glycylglycine as acceptor, was inhibited >90%.  I-treated and control cells accumulated radioactive material from suspensions contg. L-glutamine-14C at similar initial rates.  The radioactive material accumulated by the I-treated cells was all recoverable as glutamine, whereas the radioactive material accumulated by untreated cells was principally recovered as glutamate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooEluRc1A7arVg90H21EOLACvtfcHk0lgnYZyvnd2BAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXkvVShtQ%253D%253D&md5=75a871601964879332b313079c9d61a2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fjcp.1040930308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcp.1040930308%26sid%3Dliteratum%253Aachs%26aulast%3DWillis%26aufirst%3DR.%2BC.%26aulast%3DSeegmiller%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520inhibition%2520by%25206-diazo-5-oxo-l-norleucine%2520of%2520glutamine%2520catabolism%2520of%2520the%2520cultured%2520human%2520lymphoblast%26jtitle%3DJ.%2520Cell%2520Physiol.%26date%3D1977%26volume%3D93%26spage%3D375%26epage%3D382%26doi%3D10.1002%2Fjcp.1040930308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Steckel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philips, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, T. C.</span></span> <span> </span><span class="NLM_article-title">Kinetic properties and inhibition of Acinetobacter glutaminase-asparaginase</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">971</span>– <span class="NLM_lpage">977</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(83)90613-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1016%2F0006-2952%2883%2990613-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=6838661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADyaL3sXitFajtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1983&pages=971-977&author=J.+Steckelauthor=J.+Robertsauthor=F.+S.+Philipsauthor=T.+C.+Chou&title=Kinetic+properties+and+inhibition+of+Acinetobacter+glutaminase-asparaginase&doi=10.1016%2F0006-2952%2883%2990613-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8aR"><div class="casContent"><span class="casTitleNuber">8a</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic properties and inhibition of Acinetobacter glutaminase-asparaginase</span></div><div class="casAuthors">Steckel, Joph; Roberts, Joseph; Philips, Frederick S.; Chou, Ting Chao</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">971-7</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">Kinetics parameters, substrate specificity, and exclusivity of ligands at binding sites of L-glutaminase-L-asparaginase (I) purified from A. glutaminasificans were studied to gain knowledge about the dual activities of this enzyme and its inhibition by structural analogs.  Both L-glutamine and L-asparagine, which showed similar Km (4-7 × 10-5M) and Vmax (mol. activity 1.0 min-1) values, were competitive with each other for the substrate-binding site.  The products, L-glutamic acid and L-aspartic acid, showed competitive inhibition with respect to either L-glutamine or L-asparagine as substrates.  Multiple inhibition of the glutaminase activity by L-glutamic acid and L-aspartic acid indicated that these ligands are mutually exclusive at the product-releasing site.  The initial rates of both of the activities were competitively inhibited by the following inhibitors (in decreasing order of activity): 6-diazo-5-oxo-L-norleucine (DON), L-methionine sulfoximine, azaserine, and Acivicin.  DON and azaserine inhibited both activities in a time-dependent and irreversible manner.  The kinetic data suggest an ordered mechanism with glutamine or asparagine as the 1st substrate and glutamic acid or aspartic acid, resp., as the last product.  A single mechanism and a single set of binding sites are suggested as responsible for catalyzing both of the activities.  The data also showed that succinylated enzyme, which has a 10-fold increase of plasma half-life in animals and humans and, thus, has benefit as a cancer chemotherapeutic agent, retained its catalytic activity and maintained Km and Vmax values similar to native I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWgzvqrWgjn7Vg90H21EOLACvtfcHk0lhVcRNdkxz7jQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXitFajtLo%253D&md5=a78f0c944956e2b1ac0389ee3cca2596</span></div><a href="/servlet/linkout?suffix=cit8a&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2883%2990613-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252883%252990613-5%26sid%3Dliteratum%253Aachs%26aulast%3DSteckel%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%26aulast%3DPhilips%26aufirst%3DF.%2BS.%26aulast%3DChou%26aufirst%3DT.%2BC.%26atitle%3DKinetic%2520properties%2520and%2520inhibition%2520of%2520Acinetobacter%2520glutaminase-asparaginase%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1983%26volume%3D32%26spage%3D971%26epage%3D977%26doi%3D10.1016%2F0006-2952%2883%2990613-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit8b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proudfoot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skarina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dementieva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joachimiak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yakunin, A. F.</span></span> <span> </span><span class="NLM_article-title">Functional and structural characterization of four glutaminases from <i>Escherichia coli</i> and <i>Bacillus subtilis</i></span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">5724</span>– <span class="NLM_lpage">5735</span>, <span class="refDoi"> DOI: 10.1021/bi800097h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi800097h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsVKmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=5724-5735&author=G.+Brownauthor=A.+Singerauthor=M.+Proudfootauthor=T.+Skarinaauthor=Y.+Kimauthor=C.+Changauthor=I.+Dementievaauthor=E.+Kuznetsovaauthor=C.+F.+Gonzalezauthor=A.+Joachimiakauthor=A.+Savchenkoauthor=A.+F.+Yakunin&title=Functional+and+structural+characterization+of+four+glutaminases+from+Escherichia+coli+and+Bacillus+subtilis&doi=10.1021%2Fbi800097h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8bR"><div class="casContent"><span class="casTitleNuber">8b</span><div class="casTitle"><span class="NLM_cas:atitle">Functional and Structural Characterization of Four Glutaminases from Escherichia coli and Bacillus subtilis</span></div><div class="casAuthors">Brown, Greg; Singer, Alex; Proudfoot, Michael; Skarina, Tatiana; Kim, Youngchang; Chang, Changsoo; Dementieva, Irina; Kuznetsova, Ekaterina; Gonzalez, Claudio F.; Joachimiak, Andrzej; Savchenko, Alexei; Yakunin, Alexander F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5724-5735</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Glutaminases belong to the large superfamily of serine-dependent β-lactamases and penicillin-binding proteins, and they catalyze the hydrolytic deamidation of L-glutamine to L-glutamate.  In this work, we purified and biochem. characterized four predicted glutaminases from Escherichia coli (YbaS and YneH) and Bacillus subtilis (YlaM and YbgJ).  The proteins demonstrated strict specificity to L-glutamine and did not hydrolyze D-glutamine or L-asparagine.  In each organism, one glutaminase showed higher affinity to glutamine (E. coli YbaS and B. subtilis YlaM; Km 7.3 and 7.6 mM, resp.) than the second glutaminase (E. coli YneH and B. subtilis YbgJ; Km 27.6 and 30.6 mM, resp.).  The crystal structures of the E. coli YbaS and the B. subtilis YbgJ revealed the presence of a classical β-lactamase-like fold and conservation of several key catalytic residues of β-lactamases (Ser74, Lys77, Asn126, Lys268, and Ser269 in YbgJ).  Alanine replacement mutagenesis demonstrated that most of the conserved residues located in the putative glutaminase catalytic site are essential for activity.  The crystal structure of the YbgJ complex with the glutaminase inhibitor 6-diazo-5-oxo-L-norleucine revealed the presence of a covalent bond between the inhibitor and the hydroxyl oxygen of Ser74, providing evidence that Ser74 is the primary catalytic nucleophile and that the glutaminase reaction proceeds through formation of an enzyme-glutamyl intermediate.  Growth expts. with the E. coli glutaminase deletion strains revealed that YneH is involved in the assimilation of L-glutamine as a sole source of carbon and nitrogen and suggested that both glutaminases (YbaS and YneH) also contribute to acid resistance in E. coli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6EpjjxHqthLVg90H21EOLACvtfcHk0lhVcRNdkxz7jQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsVKmtrs%253D&md5=5c903849dd40ce74958dac2453b55173</span></div><a href="/servlet/linkout?suffix=cit8b&amp;dbid=16384&amp;doi=10.1021%2Fbi800097h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi800097h%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DG.%26aulast%3DSinger%26aufirst%3DA.%26aulast%3DProudfoot%26aufirst%3DM.%26aulast%3DSkarina%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DChang%26aufirst%3DC.%26aulast%3DDementieva%26aufirst%3DI.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DGonzalez%26aufirst%3DC.%2BF.%26aulast%3DJoachimiak%26aufirst%3DA.%26aulast%3DSavchenko%26aufirst%3DA.%26aulast%3DYakunin%26aufirst%3DA.%2BF.%26atitle%3DFunctional%2520and%2520structural%2520characterization%2520of%2520four%2520glutaminases%2520from%2520Escherichia%2520coli%2520and%2520Bacillus%2520subtilis%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D5724%26epage%3D5735%26doi%3D10.1021%2Fbi800097h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McBryant, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraris, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curthoys, N. P.</span></span> <span> </span><span class="NLM_article-title">Novel mechanism of inhibition of rat kidney-type glutaminase by bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide (BPTES)</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>406</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">414</span>, <span class="refDoi"> DOI: 10.1042/BJ20070039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1042%2FBJ20070039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=17581113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BD2sXps1Wgsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=406&publication_year=2007&pages=407-414&author=M.+M.+Robinsonauthor=S.+J.+McBryantauthor=T.+Tsukamotoauthor=C.+Rojasauthor=D.+V.+Ferrarisauthor=S.+K.+Hamiltonauthor=J.+C.+Hansenauthor=N.+P.+Curthoys&title=Novel+mechanism+of+inhibition+of+rat+kidney-type+glutaminase+by+bis-2-%285-phenylacetamido-1%2C2%2C4-thiadiazol-2-yl%29ethyl+sulfide+%28BPTES%29&doi=10.1042%2FBJ20070039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Novel mechanism of inhibition of rat kidney-type glutaminase by bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide (BPTES)</span></div><div class="casAuthors">Robinson, Mary M.; McBryant, Steven J.; Tsukamoto, Takashi; Rojas, Camilo; Ferraris, Dana V.; Hamilton, Sean K.; Hansen, Jeffrey C.; Curthoys, Norman P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">407-414</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The release of GA (mitochondrial glutaminase) from neurons following acute ischemia or during chronic neurodegenerative diseases may contribute to the propagation of glutamate excitotoxicity.  Thus an inhibitor that selectively inactivates the released GA may limit the accumulation of excess glutamate and minimize the loss of neurol. function that accompanies brain injury.  The present study examines the mechanism of inactivation of rat KGA (kidney GA isoform) by the small-mol. inhibitor BPTES [bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide].  BPTES is a potent inhibitor of KGA, but not of the liver GA isoform, glutamate dehydrogenase or γ-glutamyl transpeptidase.  Kinetic studies indicate that, with respect to glutamine, BPTES has a Ki of approx. 3 μM.  Moreover, these studies suggest that BPTES inhibits the allosteric activation caused by phosphate binding and promotes the formation of an inactive complex.  Gel-filtration chromatog. and sedimentation-velocity anal. were used to examine the effect of BPTES on the phosphate-dependent oligomerization of KGA.  This established that BPTES prevents the formation of large phosphate-induced oligomers and instead promotes the formation of a single oligomeric species with distinct phys. properties.  Sedimentation-equil. studies detd. that the oligomer produced by BPTES is a stable tetramer.  Taken together, the present work indicates that BPTES is a unique and potent inhibitor of rat KGA and elucidates a novel mechanism of inactivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR5Ncyq__sELVg90H21EOLACvtfcHk0ljLip2CeCtLpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXps1Wgsb4%253D&md5=cde9754c0dedd000423226e2289b4690</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1042%2FBJ20070039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20070039%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DM.%2BM.%26aulast%3DMcBryant%26aufirst%3DS.%2BJ.%26aulast%3DTsukamoto%26aufirst%3DT.%26aulast%3DRojas%26aufirst%3DC.%26aulast%3DFerraris%26aufirst%3DD.%2BV.%26aulast%3DHamilton%26aufirst%3DS.%2BK.%26aulast%3DHansen%26aufirst%3DJ.%2BC.%26aulast%3DCurthoys%26aufirst%3DN.%2BP.%26atitle%3DNovel%2520mechanism%2520of%2520inhibition%2520of%2520rat%2520kidney-type%2520glutaminase%2520by%2520bis-2-%25285-phenylacetamido-1%252C2%252C4-thiadiazol-2-yl%2529ethyl%2520sulfide%2520%2528BPTES%2529%26jtitle%3DBiochem.%2520J.%26date%3D2007%26volume%3D406%26spage%3D407%26epage%3D414%26doi%3D10.1042%2FBJ20070039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeLaBarre, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J,J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, J. B.</span></span> <span> </span><span class="NLM_article-title">Full-length human glutaminase in complex with an allosteric inhibitor</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">10764</span>– <span class="NLM_lpage">10770</span>, <span class="refDoi"> DOI: 10.1021/bi201613d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi201613d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVSks73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=10764-10770&author=B.+DeLaBarreauthor=S.+Grossauthor=C.+Fangauthor=Y.+Gaoauthor=A.+Jhaauthor=F.+Jiangauthor=J%2CJ.+Songauthor=W.+Weiauthor=J.+B.+Hurov&title=Full-length+human+glutaminase+in+complex+with+an+allosteric+inhibitor&doi=10.1021%2Fbi201613d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Full-Length Human Glutaminase in Complex with an Allosteric Inhibitor</span></div><div class="casAuthors">DeLaBarre, Byron; Gross, Stefan; Fang, Cheng; Gao, Yi; Jha, Abhishek; Jiang, Fan; Song J., Juanhua; Wei, Wentao; Hurov, Jonathan B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">10764-10770</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Glutaminase (GLS1/2) catalyzes the conversion of L-glutamine to L-glutamate and ammonia.  The level of a splice variant of GLS1 (GAC) is elevated in certain cancers, and GAC is specifically inhibited by bis-2-(5-phenylacetimido-1,2,4,thiadiazol-2-yl)ethyl sulfide (BPTES).  We report here the first full-length crystal structure of GAC in the presence and absence of BPTES mols.  Two BPTES mols. bind at an interface region of the GAC tetramer in a manner that appears to lock the GAC tetramer into a nonproductive conformation.  The importance of these loops with regard to overall enzymic activity of the tetramer was revealed by a series of GAC point mutants designed to create a BPTES resistant GAC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrj21-Ko0HGLVg90H21EOLACvtfcHk0ljLip2CeCtLpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVSks73K&md5=bd94996062f54c4f51a138c8e9f5f99a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fbi201613d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi201613d%26sid%3Dliteratum%253Aachs%26aulast%3DDeLaBarre%26aufirst%3DB.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DC.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DJha%26aufirst%3DA.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DSong%26aufirst%3DJ%252CJ.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DHurov%26aufirst%3DJ.%2BB.%26atitle%3DFull-length%2520human%2520glutaminase%2520in%2520complex%2520with%2520an%2520allosteric%2520inhibitor%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D10764%26epage%3D10770%26doi%3D10.1021%2Fbi201613d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuji, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramachandran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinavahi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrosio, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerione, R. A.</span></span> <span> </span><span class="NLM_article-title">Targeting mitochondrial glutaminase activity inhibits oncogenic transformation</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">219</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2010.08.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1016%2Fj.ccr.2010.08.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=20832749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFGht7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=207-219&author=J.+B.+Wangauthor=J.+W.+Ericksonauthor=R.+Fujiauthor=S.+Ramachandranauthor=P.+Gaoauthor=R.+Dinavahiauthor=K.+F.+Wilsonauthor=A.+L.+Ambrosioauthor=S.+M.+Diasauthor=C.+V.+Dangauthor=R.+A.+Cerione&title=Targeting+mitochondrial+glutaminase+activity+inhibits+oncogenic+transformation&doi=10.1016%2Fj.ccr.2010.08.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Mitochondrial Glutaminase Activity Inhibits Oncogenic Transformation</span></div><div class="casAuthors">Wang, Jian-Bin; Erickson, Jon W.; Fuji, Reina; Ramachandran, Sekar; Gao, Ping; Dinavahi, Ramani; Wilson, Kristin F.; Ambrosio, Andre L. B.; Dias, Sandra M. G.; Dang, Chi-V.; Cerione, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">207-219</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Rho GTPases impact a no. of activities important for oncogenesis.  We describe a small mol. inhibitor that blocks oncogenic transformation induced by various Rho GTPases in fibroblasts, and the growth of human breast cancer and B lymphoma cells, without affecting normal cells.  We identify the target of this inhibitor to be the metabolic enzyme glutaminase, which catalyzes the hydrolysis of glutamine to glutamate.  We show that transformed fibroblasts and breast cancer cells exhibit elevated glutaminase activity that is dependent on Rho GTPases and NF-κB activity, and is blocked by the small mol. inhibitor.  These findings highlight a previously unappreciated connection between Rho GTPase activation and cellular metab. and demonstrate that targeting glutaminase activity can inhibit oncogenic transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTP6YsyzRd1LVg90H21EOLACvtfcHk0lgTT33ePRiEgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFGht7bO&md5=67b64e99c992bbe6636ecfc04b83aac0</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2010.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2010.08.009%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%2BB.%26aulast%3DErickson%26aufirst%3DJ.%2BW.%26aulast%3DFuji%26aufirst%3DR.%26aulast%3DRamachandran%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DP.%26aulast%3DDinavahi%26aufirst%3DR.%26aulast%3DWilson%26aufirst%3DK.%2BF.%26aulast%3DAmbrosio%26aufirst%3DA.%2BL.%26aulast%3DDias%26aufirst%3DS.%2BM.%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DCerione%26aufirst%3DR.%2BA.%26atitle%3DTargeting%2520mitochondrial%2520glutaminase%2520activity%2520inhibits%2520oncogenic%2520transformation%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D18%26spage%3D207%26epage%3D219%26doi%3D10.1016%2Fj.ccr.2010.08.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burns, A. C.</span>; <span class="NLM_string-name">Collins, M. R.</span>; <span class="NLM_string-name">Greasley, S. E.</span>; <span class="NLM_string-name">Hoffman, R. L.</span>; <span class="NLM_string-name">Huang, Q.</span>; <span class="NLM_string-name">Kania, R.
S.</span>; <span class="NLM_string-name">Kung, P.-P.</span>; <span class="NLM_string-name">Linton, M. A.</span>; <span class="NLM_string-name">Narasimhan, L. S.</span>; <span class="NLM_string-name">Richardson, P. F.</span>; <span class="NLM_string-name">Richter, D. T.</span>; <span class="NLM_string-name">Smith, G.</span></span> <span> </span><span class="NLM_article-title">Cycloalkyl-Linked Diheterocycle Derivatives</span>. International Patent Application Number <span class="NLM_patent">WO2015/166373 A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=A.+C.+Burns&author=M.+R.+Collins&author=S.+E.+Greasley&author=R.+L.+Hoffman&author=Q.+Huang&author=R.%0AS.+Kania&author=P.-P.+Kung&author=M.+A.+Linton&author=L.+S.+Narasimhan&author=P.+F.+Richardson&author=D.+T.+Richter&author=G.+Smith&title=Cycloalkyl-Linked+Diheterocycle+Derivatives"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBurns%26aufirst%3DA.%2BC.%26atitle%3DCycloalkyl-Linked%2520Diheterocycle%2520Derivatives%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, R. M.</span>; <span class="NLM_string-name">Popovici-Muller, J.</span>; <span class="NLM_string-name">Salituro, F. G.</span>; <span class="NLM_string-name">Saunders, J. O.</span>; <span class="NLM_string-name">Travins, J.</span>; <span class="NLM_string-name">Yan, S.</span></span> <span> </span><span class="NLM_article-title">Glutaminase Inhibitors and Method of Use</span>. International Patent Application Number <span class="NLM_patent">WO2014/081925 A1</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=R.+M.+Lemieux&author=J.+Popovici-Muller&author=F.+G.+Salituro&author=J.+O.+Saunders&author=J.+Travins&author=S.+Yan&title=Glutaminase+Inhibitors+and+Method+of+Use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLemieux%26aufirst%3DR.%2BM.%26atitle%3DGlutaminase%2520Inhibitors%2520and%2520Method%2520of%2520Use%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraris, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvall, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delahanty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rais, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slusher, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukamoto, T.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and pharmacological evaluation of bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide 3 (BPTES) analogs as glutaminase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">10551</span>– <span class="NLM_lpage">10563</span>, <span class="refDoi"> DOI: 10.1021/jm301191p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301191p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Ohs7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=10551-10563&author=K.+Shuklaauthor=D.+V.+Ferrarisauthor=A.+G.+Thomasauthor=M.+Stathisauthor=B.+Duvallauthor=G.+Delahantyauthor=J.+Altauthor=R.+Raisauthor=C.+Rojasauthor=P.+Gaoauthor=Y.+Xiangauthor=C.+V.+Dangauthor=B.+S.+Slusherauthor=T.+Tsukamoto&title=Design%2C+synthesis%2C+and+pharmacological+evaluation+of+bis-2-%285-phenylacetamido-1%2C2%2C4-thiadiazol-2-yl%29ethyl+sulfide+3+%28BPTES%29+analogs+as+glutaminase+inhibitors&doi=10.1021%2Fjm301191p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14aR"><div class="casContent"><span class="casTitleNuber">14a</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Pharmacological Evaluation of Bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl Sulfide 3 (BPTES) Analogs as Glutaminase Inhibitors</span></div><div class="casAuthors">Shukla, Krupa; Ferraris, Dana V.; Thomas, Ajit G.; Stathis, Marigo; Duvall, Bridget; Delahanty, Greg; Alt, Jesse; Rais, Rana; Rojas, Camilo; Gao, Ping; Xiang, Yan; Dang, Chi V.; Slusher, Barbara S.; Tsukamoto, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10551-10563</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide (BPTES) is a potent and selective allosteric inhibitor of kidney-type glutaminase (GLS) that has served as a mol. probe to det. the therapeutic potential of GLS inhibition.  In an attempt to identify more potent GLS inhibitors with improved drug-like mol. properties, a series of BPTES analogs were synthesized and evaluated.  The structure-activity relationship (SAR) studies revealed that some truncated analogs retained the potency of BPTES, presenting an opportunity to improve its aq. soly.  The analog I exhibited similar potency and better soly. relative to BPTES and attenuated the growth of P493 human lymphoma B cells in vitro as well as in a mouse xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0N1SUSOaTCbVg90H21EOLACvtfcHk0lhiPU0Za9CueA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Ohs7zM&md5=7a54f0c8de3c211a65d5c9bbb4a6db62</span></div><a href="/servlet/linkout?suffix=cit14a&amp;dbid=16384&amp;doi=10.1021%2Fjm301191p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301191p%26sid%3Dliteratum%253Aachs%26aulast%3DShukla%26aufirst%3DK.%26aulast%3DFerraris%26aufirst%3DD.%2BV.%26aulast%3DThomas%26aufirst%3DA.%2BG.%26aulast%3DStathis%26aufirst%3DM.%26aulast%3DDuvall%26aufirst%3DB.%26aulast%3DDelahanty%26aufirst%3DG.%26aulast%3DAlt%26aufirst%3DJ.%26aulast%3DRais%26aufirst%3DR.%26aulast%3DRojas%26aufirst%3DC.%26aulast%3DGao%26aufirst%3DP.%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DSlusher%26aufirst%3DB.%2BS.%26aulast%3DTsukamoto%26aufirst%3DT.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520pharmacological%2520evaluation%2520of%2520bis-2-%25285-phenylacetamido-1%252C2%252C4-thiadiazol-2-yl%2529ethyl%2520sulfide%25203%2520%2528BPTES%2529%2520analogs%2520as%2520glutaminase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D10551%26epage%3D10563%26doi%3D10.1021%2Fjm301191p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit14b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poore, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slusher, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukamoto, T.</span></span> <span> </span><span class="NLM_article-title">Allosteric glutaminase inhibitors based on a 1,4-di(5-amino-1,3,4-thiadiazol-2-yl)butane scaffold</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">520</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFejsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=520-524&author=S.+C.+Zimmermannauthor=E.+F.+Wolfauthor=A.+Luuauthor=A.+G.+Thomasauthor=M.+Stathisauthor=B.+Pooreauthor=C.+Nguyenauthor=A.+Leauthor=C.+Rojasauthor=B.+S.+Slusherauthor=T.+Tsukamoto&title=Allosteric+glutaminase+inhibitors+based+on+a+1%2C4-di%285-amino-1%2C3%2C4-thiadiazol-2-yl%29butane+scaffold&doi=10.1021%2Facsmedchemlett.6b00060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14bR"><div class="casContent"><span class="casTitleNuber">14b</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Glutaminase Inhibitors Based on a 1,4-Di(5-amino-1,3,4-thiadiazol-2-yl)butane Scaffold</span></div><div class="casAuthors">Zimmermann, Sarah C.; Wolf, Emily F.; Luu, Andrew; Thomas, Ajit G.; Stathis, Marigo; Poore, Brad; Nguyen, Christopher; Le, Anne; Rojas, Camilo; Slusher, Barbara S.; Tsukamoto, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">520-524</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of allosteric kidney-type glutaminase (GLS) inhibitors were designed and synthesized using 1,4-di(5-amino-1,3,4-thiadiazol-2-yl)butane as a core scaffold.  A variety of modified phenylacetyl groups were incorporated into the 5-amino group of the two thiadiazole rings in an attempt to facilitate addnl. binding interactions with the allosteric binding site of GLS.  Among the newly synthesized compds., 4-hydroxy-N-[5-[4-[5-[(2-phenylacetyl)amino]-1,3,4-thiadiazol-2-yl]butyl]-1,3,4-thiadiazol-2-yl]-benzeneacetamide, 2m, potently inhibited GLS with an IC50 value of 70 nM, although it did not exhibit time-dependency as seen with CB-839.  Antiproliferative effects of 2m on human breast cancer lines will be also presented in comparison with those obsd. with CB-839.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvSab9P2v_OLVg90H21EOLACvtfcHk0lhiPU0Za9CueA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFejsrY%253D&md5=a0e0dc6fcc0f4d343f94f02a50440adf</span></div><a href="/servlet/linkout?suffix=cit14b&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00060%26sid%3Dliteratum%253Aachs%26aulast%3DZimmermann%26aufirst%3DS.%2BC.%26aulast%3DWolf%26aufirst%3DE.%2BF.%26aulast%3DLuu%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DA.%2BG.%26aulast%3DStathis%26aufirst%3DM.%26aulast%3DPoore%26aufirst%3DB.%26aulast%3DNguyen%26aufirst%3DC.%26aulast%3DLe%26aufirst%3DA.%26aulast%3DRojas%26aufirst%3DC.%26aulast%3DSlusher%26aufirst%3DB.%2BS.%26aulast%3DTsukamoto%26aufirst%3DT.%26atitle%3DAllosteric%2520glutaminase%2520inhibitors%2520based%2520on%2520a%25201%252C4-di%25285-amino-1%252C3%252C4-thiadiazol-2-yl%2529butane%2520scaffold%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D520%26epage%3D524%26doi%3D10.1021%2Facsmedchemlett.6b00060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demo, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennison, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chernov-Rogan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goyal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janes, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laidig, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackinnon, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parlati, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shwonek, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sjogren, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanton, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, M. K.</span></span> <span> </span><span class="NLM_article-title">Antitumor activity of the glutaminase inhibitor CB-839 in triple-negative breast cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">890</span>– <span class="NLM_lpage">901</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-13-0870</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1158%2F1535-7163.MCT-13-0870" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=24523301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Gjurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=890-901&author=M.+I.+Grossauthor=S.+D.+Demoauthor=J.+B.+Dennisonauthor=L.+Chenauthor=T.+Chernov-Roganauthor=B.+Goyalauthor=J.+R.+Janesauthor=G.+J.+Laidigauthor=E.+R.+Lewisauthor=J.+Liauthor=A.+L.+Mackinnonauthor=F.+Parlatiauthor=M.+L.+Rodriguezauthor=P.+J.+Shwonekauthor=E.+B.+Sjogrenauthor=T.+F.+Stantonauthor=T.+Wangauthor=J.+Yangauthor=F.+Zhaoauthor=M.+K.+Bennett&title=Antitumor+activity+of+the+glutaminase+inhibitor+CB-839+in+triple-negative+breast+cancer&doi=10.1158%2F1535-7163.MCT-13-0870"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15aR"><div class="casContent"><span class="casTitleNuber">15a</span><div class="casTitle"><span class="NLM_cas:atitle">Antitumor Activity of the Glutaminase Inhibitor CB-839 in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Gross, Matt I.; Demo, Susan D.; Dennison, Jennifer B.; Chen, Lijing; Chernov-Rogan, Tania; Goyal, Bindu; Janes, Julie R.; Laidig, Guy J.; Lewis, Evan R.; Li, Jim; MacKinnon, Andrew L.; Parlati, Francesco; Rodriguez, Mirna L. M.; Shwonek, Peter J.; Sjogren, Eric B.; Stanton, Timothy F.; Wang, Taotao; Yang, Jinfu; Zhao, Frances; Bennett, Mark K.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">890-901</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Glutamine serves as an important source of energy and building blocks for many tumor cells.  The first step in glutamine utilization is its conversion to glutamate by the mitochondrial enzyme glutaminase.  CB-839 is a potent, selective, and orally bioavailable inhibitor of both splice variants of glutaminase (KGA and GAC).  CB-839 had antiproliferative activity in a triple-neg. breast cancer (TNBC) cell line, HCC-1806, that was assocd. with a marked decrease in glutamine consumption, glutamate prodn., oxygen consumption, and the steady-state levels of glutathione and several tricarboxylic acid cycle intermediates.  In contrast, no antiproliferative activity was obsd. in an estrogen receptor-pos. cell line, T47D, and only modest effects on glutamine consumption and downstream metabolites were obsd.  Across a panel of breast cancer cell lines, GAC protein expression and glutaminase activity were elevated in the majority of TNBC cell lines relative to receptor pos. cells.  Furthermore, the TNBC subtype displayed the greatest sensitivity to CB-839 treatment and this sensitivity was correlated with (i) dependence on extracellular glutamine for growth, (ii) intracellular glutamate and glutamine levels, and (iii) GAC (but not KGA) expression, a potential biomarker for sensitivity.  CB-839 displayed significant antitumor activity in two xenograft models: as a single agent in a patient-derived TNBC model and in a basal like HER2+ cell line model, JIMT-1, both as a single agent and in combination with paclitaxel.  Together, these data provide a strong rationale for the clin. investigation of CB-839 as a targeted therapeutic in patients with TNBC and other glutamine-dependent tumors.  Mol Cancer Ther; 13(4); 890-901.2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDfUGWU-VZObVg90H21EOLACvtfcHk0lh7H0Tma339mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Gjurg%253D&md5=5b9ddb35c56409b58ce2b889bbbbc33f</span></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-13-0870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-13-0870%26sid%3Dliteratum%253Aachs%26aulast%3DGross%26aufirst%3DM.%2BI.%26aulast%3DDemo%26aufirst%3DS.%2BD.%26aulast%3DDennison%26aufirst%3DJ.%2BB.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DChernov-Rogan%26aufirst%3DT.%26aulast%3DGoyal%26aufirst%3DB.%26aulast%3DJanes%26aufirst%3DJ.%2BR.%26aulast%3DLaidig%26aufirst%3DG.%2BJ.%26aulast%3DLewis%26aufirst%3DE.%2BR.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMackinnon%26aufirst%3DA.%2BL.%26aulast%3DParlati%26aufirst%3DF.%26aulast%3DRodriguez%26aufirst%3DM.%2BL.%26aulast%3DShwonek%26aufirst%3DP.%2BJ.%26aulast%3DSjogren%26aufirst%3DE.%2BB.%26aulast%3DStanton%26aufirst%3DT.%2BF.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DBennett%26aufirst%3DM.%2BK.%26atitle%3DAntitumor%2520activity%2520of%2520the%2520glutaminase%2520inhibitor%2520CB-839%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D890%26epage%3D901%26doi%3D10.1158%2F1535-7163.MCT-13-0870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit15b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Garber, K.</span></span> <span> </span><span class="NLM_article-title">Cancer anabolic metabolism inhibitors move into clinic</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">794</span>– <span class="NLM_lpage">795</span>, <span class="refDoi"> DOI: 10.1038/nbt0816-794</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1038%2Fnbt0816-794" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=27504761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlaitLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2016&pages=794-795&author=K.+Garber&title=Cancer+anabolic+metabolism+inhibitors+move+into+clinic&doi=10.1038%2Fnbt0816-794"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15bR"><div class="casContent"><span class="casTitleNuber">15b</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer anabolic metabolism inhibitors move into clinic</span></div><div class="casAuthors">Garber, Ken</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">794-795</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqORON4nNSW6LVg90H21EOLACvtfcHk0lh7H0Tma339mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlaitLfN&md5=c61077fafc878c60f67c7a563724d6fb</span></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1038%2Fnbt0816-794&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt0816-794%26sid%3Dliteratum%253Aachs%26aulast%3DGarber%26aufirst%3DK.%26atitle%3DCancer%2520anabolic%2520metabolism%2520inhibitors%2520move%2520into%2520clinic%26jtitle%3DNat.%2520Biotechnol.%26date%3D2016%26volume%3D34%26spage%3D794%26epage%3D795%26doi%3D10.1038%2Fnbt0816-794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.</span></span> <span> </span><span class="NLM_article-title">Assembly of <i>N</i>,<i>N</i>-disubstituted hydrazines and 1-aryl-1<i>H</i>-indazoles via copper-catalyzed coupling reactions</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2552</span>– <span class="NLM_lpage">2555</span>, <span class="refDoi"> DOI: 10.1021/ol300847v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol300847v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1Oqtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=2552-2555&author=X.+Xiongauthor=Y.+Jiangauthor=D.+Ma&title=Assembly+of+N%2CN-disubstituted+hydrazines+and+1-aryl-1H-indazoles+via+copper-catalyzed+coupling+reactions&doi=10.1021%2Fol300847v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Assembly of N,N-Disubstituted Hydrazines and 1-Aryl-1H-indazoles via Copper-Catalyzed Coupling Reactions</span></div><div class="casAuthors">Xiong, Xiaodong; Jiang, Yongwen; Ma, Dawei</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2552-2555</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">CuI-catalyzed coupling of N-acyl-N'-substituted hydrazines with aryl iodides takes place at 60-90 °C to afford N-acyl-N',N'-disubstituted hydrazines regioselectively and thereby gives a facile method for assembling N,N-diaryl hydrazines.  N-Acyl-N'-substituted hydrazines can also react with 2-bromoarylcarbonylic compds. at 60-125 °C under the catalysis of CuI/4-hydroxy-L-proline to provide 1-aryl-1H-indazoles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr54dWQR6YAILVg90H21EOLACvtfcHk0lhfCpW34v3G2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1Oqtbg%253D&md5=cbca5c2ac9b7edf43ea5148a03bbebe4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fol300847v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol300847v%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DD.%26atitle%3DAssembly%2520of%2520N%252CN-disubstituted%2520hydrazines%2520and%25201-aryl-1H-indazoles%2520via%2520copper-catalyzed%2520coupling%2520reactions%26jtitle%3DOrg.%2520Lett.%26date%3D2012%26volume%3D14%26spage%3D2552%26epage%3D2555%26doi%3D10.1021%2Fol300847v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span>; <span class="NLM_string-name">Chen, L.</span>; <span class="NLM_string-name">Goyal, B.</span>; <span class="NLM_string-name">Laidig, G.</span>; <span class="NLM_string-name">Stanton, T. F.</span>; <span class="NLM_string-name">Sjogren, E. B.</span></span> <span> </span><span class="NLM_article-title">Heterocyclic Inhibitors of Glutaminase</span>. International Patent Application Number <span class="NLM_patent">WO2013/078123 A1</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=J.+Li&author=L.+Chen&author=B.+Goyal&author=G.+Laidig&author=T.+F.+Stanton&author=E.+B.+Sjogren&title=Heterocyclic+Inhibitors+of+Glutaminase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DHeterocyclic%2520Inhibitors%2520of%2520Glutaminase%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springthorpe, B.</span></span> <span> </span><span class="NLM_article-title">The influence of drug-like concepts on decision-making in medicinal chemistry</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">881</span>– <span class="NLM_lpage">890</span>, <span class="refDoi"> DOI: 10.1038/nrd2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1038%2Fnrd2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=17971784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=881-890&author=P.+D.+Leesonauthor=B.+Springthorpe&title=The+influence+of+drug-like+concepts+on+decision-making+in+medicinal+chemistry&doi=10.1038%2Fnrd2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18aR"><div class="casContent"><span class="casTitleNuber">18a</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of drug-like concepts on decision-making in medicinal chemistry</span></div><div class="casAuthors">Leeson, Paul D.; Springthorpe, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">881-890</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the wide acceptance of drug-like principles such as the 'rule of five', this anal. of mols. currently being synthesized in leading pharmaceutical companies reveals that their phys. properties differ significantly from those of recently discovered oral drugs.  The marked increase in lipophilicity in particular could increase the likelihood of attrition in drug development.  The application of guidelines linked to the concept of drug-likeness, such as the 'rule of five', has gained wide acceptance as an approach to reduce attrition in drug discovery and development.  However, despite this acceptance, anal. of recent trends reveals that the phys. properties of mols. that are currently being synthesized in leading drug discovery companies differ significantly from those of recently discovered oral drugs and compds. in clin. development.  The consequences of the marked increase in lipophilicity - the most important drug-like phys. property - include a greater likelihood of lack of selectivity and attrition in drug development.  Tackling the threat of compd.-related toxicol. attrition needs to move to the mainstream of medicinal chem. decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTUeonLD6iebVg90H21EOLACvtfcHk0lhfCpW34v3G2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL&md5=5665a9e2dc266a1ee096e20625757ef7</span></div><a href="/servlet/linkout?suffix=cit18a&amp;dbid=16384&amp;doi=10.1038%2Fnrd2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2445%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSpringthorpe%26aufirst%3DB.%26atitle%3DThe%2520influence%2520of%2520drug-like%2520concepts%2520on%2520decision-making%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D881%26epage%3D890%26doi%3D10.1038%2Fnrd2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit18b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand efficiency: A useful metric for lead selection</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">431</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+A+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18bR"><div class="casContent"><span class="casTitleNuber">18b</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2ebcCeSJpI4kw7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit18b&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520A%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lafont, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amzel, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span> <span> </span><span class="NLM_article-title">Compensating enthalpic and entropic changes hinder binding affinity optimization</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2007.00519.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1111%2Fj.1747-0285.2007.00519.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=17581235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotV2js7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2007&pages=413-422&author=V.+Lafontauthor=A.+A.+Armstrongauthor=H.+Ohtakaauthor=Y.+Kisoauthor=L.+M.+Amzelauthor=E.+Freire&title=Compensating+enthalpic+and+entropic+changes+hinder+binding+affinity+optimization&doi=10.1111%2Fj.1747-0285.2007.00519.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Compensating enthalpic and entropic changes hinder binding affinity optimization</span></div><div class="casAuthors">Lafont, Virginie; Armstrong, Anthony A.; Ohtaka, Hiroyasu; Kiso, Yoshiaki; Amzel, L. Mario; Freire, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">413-422</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A common strategy to improve the potency of drug candidates is to introduce chem. functionalities, like hydrogen bond donors or acceptors, at positions where they are able to establish strong interactions with the target.  However, it is often obsd. that the added functionalities do not necessarily improve potency even if they form strong hydrogen bonds.  Here, we explore the thermodn. and structural basis for those observations.  KNI-10033 is a potent exptl. HIV-1 protease inhibitor with picomolar affinity against the wild-type enzyme (Kd = 13 pM).  The potency of the inhibitor is the result of favorable enthalpic (ΔH = -8.2 kcal/mol) and entropic (-TΔS = -6.7 kcal/mol) interactions.  The replacement of the thioether group in KNI-10033 by a sulfonyl group (KNI-10075) results in a strong hydrogen bond with the amide of Asp 30B of the HIV-1 protease.  This addnl. hydrogen bond improves the binding enthalpy by 3.9 kcal/mol; however, the enthalpy gain is completely compensated by an entropy loss, resulting in no affinity change.  Crystallog. and thermodn. anal. of the inhibitor/protease complexes indicates that the entropy losses are due to a combination of conformational and solvation effects.  These results provide a set of practical guidelines aimed at overcoming enthalpy/entropy compensation and improve binding potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMsDV9Pvk5Y7Vg90H21EOLACvtfcHk0lguJLkZot_mnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotV2js7o%253D&md5=163b3b46eba827d6a174284e8e3fb308</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00519.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00519.x%26sid%3Dliteratum%253Aachs%26aulast%3DLafont%26aufirst%3DV.%26aulast%3DArmstrong%26aufirst%3DA.%2BA.%26aulast%3DOhtaka%26aufirst%3DH.%26aulast%3DKiso%26aufirst%3DY.%26aulast%3DAmzel%26aufirst%3DL.%2BM.%26aulast%3DFreire%26aufirst%3DE.%26atitle%3DCompensating%2520enthalpic%2520and%2520entropic%2520changes%2520hinder%2520binding%2520affinity%2520optimization%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2007%26volume%3D69%26spage%3D413%26epage%3D422%26doi%3D10.1111%2Fj.1747-0285.2007.00519.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernetti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boby, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fioravanti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neill, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drakes, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katt, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerione, R.</span></span> <span> </span><span class="NLM_article-title">Design and evaluation of novel glutaminase inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1819</span>– <span class="NLM_lpage">1839</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.03.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1016%2Fj.bmc.2016.03.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=26988803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC28Xkt1Wmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=1819-1839&author=L.+A.+McDermottauthor=P.+Lyerauthor=L.+Vernettiauthor=S.+Rimerauthor=J.+Sunauthor=M.+Bobyauthor=T.+Yangauthor=M.+Fioravantiauthor=J.+O%E2%80%99Neillauthor=L.+Wangauthor=D.+Drakesauthor=W.+Kattauthor=Q.+Huangauthor=R.+Cerione&title=Design+and+evaluation+of+novel+glutaminase+inhibitors&doi=10.1016%2Fj.bmc.2016.03.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Design and evaluation of novel glutaminase inhibitors</span></div><div class="casAuthors">McDermott, Lee A.; Iyer, Prema; Vernetti, Larry; Rimer, Shawn; Sun, Jingran; Boby, Melissa; Yang, Tianyi; Fioravanti, Michael; O'Neill, Jason; Wang, Liwei; Drakes, Dylan; Katt, William; Huang, Qingqiu; Cerione, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1819-1839</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel set of GAC (kidney glutaminase isoform C) inhibitors able to inhibit the enzymic activity of GAC and the growth of the triple neg. MDA-MB-231 breast cancer cells with low nanomolar potency is described.  Compds. in this series have a reduced no. of rotatable bonds, improved C log Ps, microsomal stability and ligand efficiency when compared to the leading GAC inhibitors BPTES and CB-839.  Property improvements were achieved by the replacement of the flexible n-diethylthio or the Bu moiety present in the leading inhibitors by heteroatom substituted heterocycloalkanes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGfzNc9NMuhbVg90H21EOLACvtfcHk0lguJLkZot_mnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xkt1Wmsbg%253D&md5=f335b5659f76e351b73c96954153127d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.03.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.03.009%26sid%3Dliteratum%253Aachs%26aulast%3DMcDermott%26aufirst%3DL.%2BA.%26aulast%3DLyer%26aufirst%3DP.%26aulast%3DVernetti%26aufirst%3DL.%26aulast%3DRimer%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DBoby%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DFioravanti%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DDrakes%26aufirst%3DD.%26aulast%3DKatt%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DQ.%26aulast%3DCerione%26aufirst%3DR.%26atitle%3DDesign%2520and%2520evaluation%2520of%2520novel%2520glutaminase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D1819%26epage%3D1839%26doi%3D10.1016%2Fj.bmc.2016.03.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
den Heuvel, A. P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wooster, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachman, K. E.</span></span> <span> </span><span class="NLM_article-title">Analysis of glutamine dependency in non-small cell lung cancer GLS1 splice variant GAC is essential for cancer cell growth</span>. <i>Cancer Biol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1185</span>– <span class="NLM_lpage">1194</span>, <span class="refDoi"> DOI: 10.4161/cbt.21348</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.4161%2Fcbt.21348" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=22892846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC3sXisF2nu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=1185-1194&author=A.+P.+J.+van%0Aden+Heuvelauthor=J.+Jingauthor=R.+F.+Woosterauthor=K.+E.+Bachman&title=Analysis+of+glutamine+dependency+in+non-small+cell+lung+cancer+GLS1+splice+variant+GAC+is+essential+for+cancer+cell+growth&doi=10.4161%2Fcbt.21348"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of glutamine dependency in non-small cell lung cancer: GLS1 splice variant GAC is essential for cancer cell growth</span></div><div class="casAuthors">van den Heuvel, A. Pieter J.; Jing, Junping; Wooster, Richard F.; Bachman, Kurtis E.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Biology & Therapy</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1185-1194</span>CODEN:
                <span class="NLM_cas:coden">CBTAAO</span>;
        ISSN:<span class="NLM_cas:issn">1538-4047</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">One of the hallmarks of cancer is metabolic deregulation.  Many tumors display increased glucose uptake and breakdown through the process of aerobic glycolysis, also known as the Warburg effect.  Less studied in cancer development and progression is the importance of the glutamine (Gln) pathway, which provides cells with a variety of essential products to sustain cell proliferation, such as ATP and macromols. for biosynthesis.  To this end Gln dependency was assessed in a panel of non-small cell lung cancer lines (NSCLC).  Gln was found to be essential for the growth of cells with high rates of glutaminolysis, and after exploring multiple genes in the Gln pathway, GLS1 was found to be the key enzyme assocd. with this dependence.  This dependence was confirmed by observing the rescue of decreased growth by exogenous addn. of downstream metabolites of glutaminolysis.  Expression of the GLS1 splice variant KGA was found to be decreased in tumors compared with normal lung tissue.  Transient knock down of GLS1 splice variants indicated that loss of GAC had the most detrimental effect on cancer cell growth.  In conclusion, NSCLC cell lines depend on Gln for glutaminolysis to a varying degree, in which the GLS1 splice variant GAC plays an essential role and is a potential target for cancer metab.-directed therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnWgR5NZWykLVg90H21EOLACvtfcHk0lg6Ws-4rU82hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXisF2nu78%253D&md5=521cd23a9f2a3a7f31c7024603b3a0ca</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.4161%2Fcbt.21348&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcbt.21348%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bden%2BHeuvel%26aufirst%3DA.%2BP.%2BJ.%26aulast%3DJing%26aufirst%3DJ.%26aulast%3DWooster%26aufirst%3DR.%2BF.%26aulast%3DBachman%26aufirst%3DK.%2BE.%26atitle%3DAnalysis%2520of%2520glutamine%2520dependency%2520in%2520non-small%2520cell%2520lung%2520cancer%2520GLS1%2520splice%2520variant%2520GAC%2520is%2520essential%2520for%2520cancer%2520cell%2520growth%26jtitle%3DCancer%2520Biol.%2520Ther.%26date%3D2012%26volume%3D13%26spage%3D1185%26epage%3D1194%26doi%3D10.4161%2Fcbt.21348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finlay, M. R. V.</span>; <span class="NLM_string-name">Ekwuru, C. T.</span>; <span class="NLM_string-name">Charles, M. D.</span>; <span class="NLM_string-name">Raubo, P. A.</span>; <span class="NLM_string-name">Winter, J. J. G.</span>; <span class="NLM_string-name">Nissink, J. W. M.</span></span> <span> </span><span class="NLM_article-title">1,3,4-Thiadiazole Compounds and Their Use in Treating Cancer</span>. International Patent Application Number <span class="NLM_patent">WO2015/181539 A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=M.+R.+V.+Finlay&author=C.+T.+Ekwuru&author=M.+D.+Charles&author=P.+A.+Raubo&author=J.+J.+G.+Winter&author=J.+W.+M.+Nissink&title=1%2C3%2C4-Thiadiazole+Compounds+and+Their+Use+in+Treating+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DFinlay%26aufirst%3DM.%2BR.%2BV.%26atitle%3D1%252C3%252C4-Thiadiazole%2520Compounds%2520and%2520Their%2520Use%2520in%2520Treating%2520Cancer%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2a','cit2b'],'ref3':['cit3'],'ref4':['cit4a','cit4b','cit4c'],'ref5':['cit5a','cit5b'],'ref6':['cit6a','cit6b'],'ref7':['cit7'],'ref8':['cit8a','cit8b'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14a','cit14b'],'ref15':['cit15a','cit15b'],'ref16':['cit16'],'ref17':['cit17a'],'ref18':['cit18a','cit18b'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 9 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xi Xu, Jubo Wang, Min Wang, Xinyu Yuan, Lei Li, Chao Zhang, Huidan Huang, Tian Jing, Chenchen Wang, Chao Tong, Liwen Zhou, Ying Meng, Pengfei Xu, Junping Kou, Zhixia Qiu, Zhiyu Li, <span class="NLM_string-name hlFld-ContribAuthor">Jinlei Bian</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Enabled Discovery of Novel Macrocyclic Inhibitors Targeting Glutaminase 1 Allosteric Binding Site. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 4588-4611. <a href="https://doi.org/10.1021/acs.jmedchem.0c02044" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02044</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02044%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Enabled%252BDiscovery%252Bof%252BNovel%252BMacrocyclic%252BInhibitors%252BTargeting%252BGlutaminase%252B1%252BAllosteric%252BBinding%252BSite%26aulast%3DXu%26aufirst%3DXi%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D25112020%26date%3D01042021%26volume%3D64%26issue%3D8%26spage%3D4588%26epage%3D4611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Michael J. Soth, Kang Le, Maria Emilia Di Francesco, Matthew M. Hamilton, Gang Liu, Jason P. Burke, Chris L. Carroll, Jeffrey J. Kovacs, Jennifer P. Bardenhagen, Christopher A. Bristow, Mario Cardozo, Barbara Czako, Elisa de Stanchina, Ningping Feng, Jill R. Garvey, Jason P. Gay, Mary K. Geck Do, Jennifer Greer, Michelle Han, Angela Harris, Zachary Herrera, Sha Huang, Virginia Giuliani, Yongying Jiang, Sarah B. Johnson, Troy A. Johnson, Zhijun Kang, Paul G. Leonard, Zhen Liu, Timothy McAfoos, Meredith Miller, Pietro Morlacchi, Robert A. Mullinax, Wylie S. Palmer, Jihai Pang, Norma Rogers, Charles M. Rudin, Hannah E. Shepard, Nakia D. Spencer, Jay Theroff, Qi Wu, Alan Xu, Ju Anne Yau, Giulio Draetta, Carlo Toniatti, Timothy P. Heffernan, <span class="NLM_string-name hlFld-ContribAuthor">Philip Jones</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of IPN60090, a Clinical Stage Selective Glutaminase-1 (GLS-1) Inhibitor with Excellent Pharmacokinetic and Physicochemical Properties. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (21)
                                     , 12957-12977. <a href="https://doi.org/10.1021/acs.jmedchem.0c01398" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01398</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01398%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BIPN60090%25252C%252Ba%252BClinical%252BStage%252BSelective%252BGlutaminase-1%252B%252528GLS-1%252529%252BInhibitor%252Bwith%252BExcellent%252BPharmacokinetic%252Band%252BPhysicochemical%252BProperties%26aulast%3DSoth%26aufirst%3DMichael%2BJ.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D09082020%26date%3D29102020%26volume%3D63%26issue%3D21%26spage%3D12957%26epage%3D12977" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marie-Aude  Hiebel</span>, <span class="hlFld-ContribAuthor ">Franck  Suzenet</span>. </span><span class="cited-content_cbyCitation_article-title">Triazines, tetrazines, and fused ring polyaza systems. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 467-499. <a href="https://doi.org/10.1016/B978-0-323-89812-6.00014-6" title="DOI URL">https://doi.org/10.1016/B978-0-323-89812-6.00014-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-323-89812-6.00014-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-323-89812-6.00014-6%26sid%3Dliteratum%253Aachs%26atitle%3DTriazines%25252C%252Btetrazines%25252C%252Band%252Bfused%252Bring%252Bpolyaza%252Bsystems%26aulast%3DHiebel%26aufirst%3DMarie-Aude%26date%3D2021%26spage%3D467%26epage%3D499%26pub%3DElsevier%26date%3D2021%26volume%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhang-Xu  He</span>, <span class="hlFld-ContribAuthor ">Yun-Peng  Gong</span>, <span class="hlFld-ContribAuthor ">Xin  Zhang</span>, <span class="hlFld-ContribAuthor ">Li-Ying  Ma</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Pyridazine as a privileged structure: An updated review on anticancer activity of pyridazine containing bioactive molecules. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>209 </em>, 112946. <a href="https://doi.org/10.1016/j.ejmech.2020.112946" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112946</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112946%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPyridazine%252Bas%252Ba%252Bprivileged%252Bstructure%25253A%252BAn%252Bupdated%252Breview%252Bon%252Banticancer%252Bactivity%252Bof%252Bpyridazine%252Bcontaining%252Bbioactive%252Bmolecules%26aulast%3DHe%26aufirst%3DZhang-Xu%26date%3D2021%26volume%3D209%26spage%3D112946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wu-Xi  Zhou</span>, <span class="hlFld-ContribAuthor ">Chen  Chen</span>, <span class="hlFld-ContribAuthor ">Xiao-Qin  Liu</span>, <span class="hlFld-ContribAuthor ">Ying  Li</span>, <span class="hlFld-ContribAuthor ">Yao-Lan  Lin</span>, <span class="hlFld-ContribAuthor ">Xiu-Tao  Wu</span>, <span class="hlFld-ContribAuthor ">Ling-Yi  Kong</span>, <span class="hlFld-ContribAuthor ">Jian-Guang  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and optimization of withangulatin A derivatives as novel glutaminase 1 inhibitors for the treatment of triple-negative breast cancer. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112980. <a href="https://doi.org/10.1016/j.ejmech.2020.112980" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112980</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112980%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252Boptimization%252Bof%252Bwithangulatin%252BA%252Bderivatives%252Bas%252Bnovel%252Bglutaminase%252B1%252Binhibitors%252Bfor%252Bthe%252Btreatment%252Bof%252Btriple-negative%252Bbreast%252Bcancer%26aulast%3DZhou%26aufirst%3DWu-Xi%26date%3D2021%26volume%3D210%26spage%3D112980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">, <span class="hlFld-ContribAuthor ">Arafath K.  Najumudeen</span>, <span class="hlFld-ContribAuthor ">Fatih  Ceteci</span>, <span class="hlFld-ContribAuthor ">Sigrid K.  Fey</span>, <span class="hlFld-ContribAuthor ">Gregory  Hamm</span>, <span class="hlFld-ContribAuthor ">Rory T.  Steven</span>, <span class="hlFld-ContribAuthor ">Holly  Hall</span>, <span class="hlFld-ContribAuthor ">Chelsea J.  Nikula</span>, <span class="hlFld-ContribAuthor ">Alex  Dexter</span>, <span class="hlFld-ContribAuthor ">Teresa  Murta</span>, <span class="hlFld-ContribAuthor ">Alan M.  Race</span>, <span class="hlFld-ContribAuthor ">David  Sumpton</span>, <span class="hlFld-ContribAuthor ">Nikola  Vlahov</span>, <span class="hlFld-ContribAuthor ">David M.  Gay</span>, <span class="hlFld-ContribAuthor ">John R. P.  Knight</span>, <span class="hlFld-ContribAuthor ">Rene  Jackstadt</span>, <span class="hlFld-ContribAuthor ">Joshua D. G.  Leach</span>, <span class="hlFld-ContribAuthor ">Rachel A.  Ridgway</span>, <span class="hlFld-ContribAuthor ">Emma R.  Johnson</span>, <span class="hlFld-ContribAuthor ">Colin  Nixon</span>, <span class="hlFld-ContribAuthor ">Ann  Hedley</span>, <span class="hlFld-ContribAuthor ">Kathryn  Gilroy</span>, <span class="hlFld-ContribAuthor ">William  Clark</span>, <span class="hlFld-ContribAuthor ">Sudhir B.  Malla</span>, <span class="hlFld-ContribAuthor ">Philip D.  Dunne</span>, <span class="hlFld-ContribAuthor ">Giovanny  Rodriguez-Blanco</span>, <span class="hlFld-ContribAuthor ">Susan E.  Critchlow</span>, <span class="hlFld-ContribAuthor ">Agata  Mrowinska</span>, <span class="hlFld-ContribAuthor ">Gaurav  Malviya</span>, <span class="hlFld-ContribAuthor ">Dmitry  Solovyev</span>, <span class="hlFld-ContribAuthor ">Gavin  Brown</span>, <span class="hlFld-ContribAuthor ">David Y.  Lewis</span>, <span class="hlFld-ContribAuthor ">Gillian M.  Mackay</span>, <span class="hlFld-ContribAuthor ">Douglas  Strathdee</span>, <span class="hlFld-ContribAuthor ">Saverio  Tardito</span>, <span class="hlFld-ContribAuthor ">Eyal  Gottlieb</span>, <span class="hlFld-ContribAuthor ">Zoltan  Takats</span>, <span class="hlFld-ContribAuthor ">Simon T.  Barry</span>, <span class="hlFld-ContribAuthor ">Richard J. A.  Goodwin</span>, <span class="hlFld-ContribAuthor ">Josephine  Bunch</span>, <span class="hlFld-ContribAuthor ">Martin  Bushell</span>, <span class="hlFld-ContribAuthor ">Andrew D.  Campbell</span>, <span class="hlFld-ContribAuthor ">Owen J.  Sansom</span>. </span><span class="cited-content_cbyCitation_article-title">The amino acid transporter SLC7A5 is required for efficient growth of KRAS-mutant colorectal cancer. </span><span class="cited-content_cbyCitation_journal-name">Nature Genetics</span><span> <strong>2021,</strong> <em>53 </em>
                                    (1)
                                     , 16-26. <a href="https://doi.org/10.1038/s41588-020-00753-3" title="DOI URL">https://doi.org/10.1038/s41588-020-00753-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41588-020-00753-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41588-020-00753-3%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Genetics%26atitle%3DThe%252Bamino%252Bacid%252Btransporter%252BSLC7A5%252Bis%252Brequired%252Bfor%252Befficient%252Bgrowth%252Bof%252BKRAS-mutant%252Bcolorectal%252Bcancer%26aulast%3DNajumudeen%26aufirst%3DArafath%2BK.%26date%3D2021%26date%3D2021%26volume%3D53%26issue%3D1%26spage%3D16%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhefang  Wang</span>, <span class="hlFld-ContribAuthor ">Fanyu  Liu</span>, <span class="hlFld-ContribAuthor ">Ningbo  Fan</span>, <span class="hlFld-ContribAuthor ">Chenghui  Zhou</span>, <span class="hlFld-ContribAuthor ">Dai  Li</span>, <span class="hlFld-ContribAuthor ">Thomas  Macvicar</span>, <span class="hlFld-ContribAuthor ">Qiongzhu  Dong</span>, <span class="hlFld-ContribAuthor ">Christiane J.  Bruns</span>, <span class="hlFld-ContribAuthor ">Yue  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Glutaminolysis: New Perspectives to Understand Cancer Development and Novel Strategies for Potential Target Therapies. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Oncology</span><span> <strong>2020,</strong> <em>10 </em><a href="https://doi.org/10.3389/fonc.2020.589508" title="DOI URL">https://doi.org/10.3389/fonc.2020.589508</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fonc.2020.589508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffonc.2020.589508%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Oncology%26atitle%3DTargeting%252BGlutaminolysis%25253A%252BNew%252BPerspectives%252Bto%252BUnderstand%252BCancer%252BDevelopment%252Band%252BNovel%252BStrategies%252Bfor%252BPotential%252BTarget%252BTherapies%26aulast%3DWang%26aufirst%3DZhefang%26date%3D2020%26date%3D2020%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liancheng  Duan</span>, <span class="hlFld-ContribAuthor ">Xin  Wang</span>, <span class="hlFld-ContribAuthor ">Yucheng  Gu</span>, <span class="hlFld-ContribAuthor ">Yunlei  Hou</span>, <span class="hlFld-ContribAuthor ">Ping  Gong</span>. </span><span class="cited-content_cbyCitation_article-title">Regioselective construction of pyridazine and tetrahydrocinnoline derivatives
              via
              [4 + 2] cycloaddition–elimination with α-halogeno hydrazones and enaminones. </span><span class="cited-content_cbyCitation_journal-name">Organic Chemistry Frontiers</span><span> <strong>2020,</strong> <em>7 </em>
                                    (16)
                                     , 2307-2312. <a href="https://doi.org/10.1039/D0QO00555J" title="DOI URL">https://doi.org/10.1039/D0QO00555J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0QO00555J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0QO00555J%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Chemistry%2520Frontiers%26atitle%3DRegioselective%252Bconstruction%252Bof%252Bpyridazine%252Band%252Btetrahydrocinnoline%252Bderivatives%252Bvia%252B%25255B4%252B%25252B%252B2%25255D%252Bcycloaddition%2525E2%252580%252593elimination%252Bwith%252B%2525CE%2525B1-halogeno%252Bhydrazones%252Band%252Benaminones%26aulast%3DDuan%26aufirst%3DLiancheng%26date%3D2020%26date%3D2020%26volume%3D7%26issue%3D16%26spage%3D2307%26epage%3D2312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrés  Méndez-Lucas</span>, <span class="hlFld-ContribAuthor ">Wei  Lin</span>, <span class="hlFld-ContribAuthor ">Paul C.  Driscoll</span>, <span class="hlFld-ContribAuthor ">Nathalie  Legrave</span>, <span class="hlFld-ContribAuthor ">Laura  Novellasdemunt</span>, <span class="hlFld-ContribAuthor ">Chencheng  Xie</span>, <span class="hlFld-ContribAuthor ">Mark  Charles</span>, <span class="hlFld-ContribAuthor ">Zena  Wilson</span>, <span class="hlFld-ContribAuthor ">Neil P.  Jones</span>, <span class="hlFld-ContribAuthor ">Stephen  Rayport</span>, <span class="hlFld-ContribAuthor ">Manuel  Rodríguez-Justo</span>, <span class="hlFld-ContribAuthor ">Vivian  Li</span>, <span class="hlFld-ContribAuthor ">James I.  MacRae</span>, <span class="hlFld-ContribAuthor ">Nissim  Hay</span>, <span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Mariia  Yuneva</span>. </span><span class="cited-content_cbyCitation_article-title">Identifying strategies to target the metabolic flexibility of tumours. </span><span class="cited-content_cbyCitation_journal-name">Nature Metabolism</span><span> <strong>2020,</strong> <em>2 </em>
                                    (4)
                                     , 335-350. <a href="https://doi.org/10.1038/s42255-020-0195-8" title="DOI URL">https://doi.org/10.1038/s42255-020-0195-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s42255-020-0195-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs42255-020-0195-8%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Metabolism%26atitle%3DIdentifying%252Bstrategies%252Bto%252Btarget%252Bthe%252Bmetabolic%252Bflexibility%252Bof%252Btumours%26aulast%3DM%25C3%25A9ndez-Lucas%26aufirst%3DAndr%25C3%25A9s%26date%3D2020%26date%3D2020%26volume%3D2%26issue%3D4%26spage%3D335%26epage%3D350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0023.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0001.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of GLS1 inhibitors DON (<b>1</b>), acivicin (<b>2</b>), BPTES (<b>3</b>), 968 (<b>4</b>), and CB-839 (<b>5</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0002.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) BPTES (<b>3</b>) binding mode to GLS1(PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGAA">3KGAA</a>). Two molecules of inhibitor are associated with four units of protein. BPTES sits between the interfaces of two GLS1 subunits occupying equivalent space on each protein and appears to hold the protein in a nonfunctional tetrameric form. (b) Binding mode of BPTES. Each half of the BPTES molecule interacts with one protein chain of the dimer unit and forms hydrogen-bonding contacts with its thiadiazole amide. The interactions are shown for one thiadiazole carboxamide substructure in (c). Four symmetry-related lipophilic residues (Phe322, Tyr394, visible top middle) form a hydrophobic “cap” centered over the lipophilic diethylsulfide linker of BPTES. (c) The backbone atoms of Phe322 and Leu323 form three hydrogen bonds to the thiadiazole carboxamide.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>9</b>–<b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1,2-diamine, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), MeCN or dimethylformamide (DMF), microwave (μW), 80–130 °C; (b) PhCH<sub>2</sub>COCl, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, room temperature (rt), 4–22%.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0009.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>12</b>–<b>19</b> and <b>23</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>30</b>, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C, 88%; (b) PhCH<sub>2</sub>COCl, K<sub>2</sub>CO<sub>3</sub>, tetrahydrofuran (THF), rt, 100%; (c) trifluoroacetic acid (TFA), CH<sub>2</sub>Cl<sub>2</sub>, rt, 98%; (d) <b>30</b>, K<sub>2</sub>CO<sub>3</sub>, MeCN, rt, 73%; (e) R1COCl, DIPEA, DMF, rt, 7–43%; (f) R1COOH, 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (HATU), DIPEA, dimethylacetamide (DMA), rt, 32–39%; (g) 2-bromo-1,3,4-thiadiazole, 4-dimethylaminopyridine (DMAP), DIPEA, <i>N</i>-methyl-2-pyrrolidone (NMP), 106 °C, 6%; (h) 2-chloropyridazine, DMAP, DIPEA, NMP, 106 °C, 26%; (i) <b>30</b>, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, μW, 60 °C; (j) PhCH<sub>2</sub>COCl, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>; (k) TFA, CH<sub>2</sub>Cl<sub>2</sub>, 100% (three steps); (l) <b>30</b>, DIPEA, DMF, 100 °C; (m) PhCH<sub>2</sub>COCl, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 43% (two steps).</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0010.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>20</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2-bromo-1,3,4-thiadiazole, DIPEA, <i>n</i>-BuOH, 50 °C, 70%; (b) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 98%; (c) PhCH<sub>2</sub>COCl, DIPEA, DMF, rt, 25%; (d) CuI, (2<i>S</i>,4<i>R</i>)-4-hydroxypyrrolidine-2-carboxylic acid, K<sub>2</sub>CO<sub>3</sub>, dimethyl sulfoxide (DMSO), 80 °C, 8.4%.</p></p></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0011.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>21</b> and <b>22</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>34</b>, Et<sub>3</sub>N, <i>n</i>-BuOH, 160–180 °C, 100%; (b) PhCH<sub>2</sub>COCl, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 25%; (c) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 71%; (d) <b>30</b>, DIPEA, MeCN, 80 °C, 17%; (e) <b>34</b>, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C, quant; (f) PhCH<sub>2</sub>COCl, K<sub>2</sub>CO<sub>3</sub>, THF, rt, quant; (g) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 62%; (h) <b>30</b>, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C, 55%.</p></p></figure><figure data-id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0012.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compound <b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions:  (a) <i>m</i>CPBA, CH<sub>2</sub>Cl<sub>2</sub>, rt; (b) <b>34</b>, DIPEA, isopropyl alcohol (IPA), 50 °C, 98%; (c) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 96%; (d) <b>30</b>, DIPEA, DMF, 100 °C; (e) PhCH<sub>2</sub>COOH, HATU, DIPEA, DMF, rt, 46% (two steps).</p></p></figure><figure data-id="sch6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0013.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compounds <b>25</b> and <b>26</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) BnNH<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>, rt to 60 °C, 48%; (b) <b>31</b>, K<sub>2</sub>CO<sub>3</sub>, DMA, MeCN 80–100 °C, 46%; (c) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 69%; (d) 2-chloropyridazine, DMAP, DIPEA, NMP, 110 °C, 13%; (e) <b>40</b>, DIPEA, DMF, 100 °C, 53%.</p></p></figure><figure data-id="sch7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0014.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Compounds <b>27</b>–<b>29</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions:  (a) 3,6-dichloropyridazine, DIPEA, MeCN, reflux, 69%; (b) Pd/C, HCOONH<sub>4</sub>, EtOH, reflux, 98%; (c) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 93%; (d) <b>30</b>, DIPEA, MeOH, 45 °C, 83%; (e) for R1 = OMe, (<i>S</i>)-2-methoxy-2-phenylacetic acid, HATU, 1,4,6-trimethylpyridine, DMF, NMP, 0–20 °C, 86%; for R1 = Me; (2<i>S</i>)-phenylpropanoic acid, HATU, DIPEA, DMF, rt, 16%; (f) for A = N; <b>30</b>, DIPEA, DMF, 100 °C, then (<i>S</i>)-2-methoxy-2-phenylacetic acid, HATU, DIPEA, rt, 47%.</p></p></figure><figure data-id="fig3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0003.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding mode of <b>12</b> (green, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KGAC">1KGAC</a>) compared to that of truncated inhibitor <b>18</b> (pink, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6KGAB">6KGAB</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0004.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Fold cover over cell GI<sub>50</sub> following 100 mg/kg oral dose of <b>23</b> to SCID mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0005.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Fold free cover over cell GI<sub>50</sub> following 100 mg/kg oral dose of <b>27</b> and <b>28</b> to SCID mice. Compound <b>29</b> was dosed to nude mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0006.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Overlay of the X-ray crystal structures of <b>3</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGAA">3KGAA</a>) and <b>27</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KGAD">7KGAD</a>) bound to GLS1. (a) Two symmetry-related binding modes (orange and cyan) of <b>27</b>. Symmetry-related chains shown in gray and yellow. (b) Overlay of binding modes of one molecule of <b>27</b> (cyan) and <b>3</b> (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/medium/jm-2019-00260j_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0007.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (a) Pharmacodynamic data for compound <b>27</b> showing effects on tumor amino acid levels. (b) Tumor growth inhibition study using oral dosing of <b>27</b> and taxotere. See text for further details. (c) Study exposure data and free cover over cell GI<sub>50</sub> calculations. Compound <b>27</b> was measured as 16.3% free in SCID mouse plasma protein. Statistical analysis performed by one-tailed, unequal variance <i>t</i> test vs vehicle control at day 21. ** <i>P</i> < 0.01, *** <i>P</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.9b00260/20190719/images/large/jm-2019-00260j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00260&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i86">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67357" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67357" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 22 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gui, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Targeting metabolism for cancer therapy</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1161</span>– <span class="NLM_lpage">1180</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.08.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1016%2Fj.chembiol.2017.08.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=28938091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1161-1180&author=A.+Luengoauthor=D.+Y.+Guiauthor=M.+G.+Vander%0AHeiden&title=Targeting+metabolism+for+cancer+therapy&doi=10.1016%2Fj.chembiol.2017.08.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Metabolism for Cancer Therapy</span></div><div class="casAuthors">Luengo, Alba; Gui, Dan Y.; Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1161-1180</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Metabolic reprogramming contributes to tumor development and introduces metabolic liabilities that can be exploited to treat cancer.  Chemotherapies targeting metab. have been effective cancer treatments for decades, and the success of these therapies demonstrates that a therapeutic window exists to target malignant metab.  New insights into the differential metabolic dependencies of tumors have provided novel therapeutic strategies to exploit altered metab., some of which are being evaluated in preclin. models or clin. trials.  Here, we review our current understanding of cancer metab. and discuss how this might guide treatments targeting the metabolic requirements of tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUOppBTjvuc7Vg90H21EOLACvtfcHk0liKCrLUgZROWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurfO&md5=c702238d8642e06df9accd5414b64936</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.08.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.08.028%26sid%3Dliteratum%253Aachs%26aulast%3DLuengo%26aufirst%3DA.%26aulast%3DGui%26aufirst%3DD.%2BY.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DTargeting%2520metabolism%2520for%2520cancer%2520therapy%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1161%26epage%3D1180%26doi%3D10.1016%2Fj.chembiol.2017.08.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Alfarouk, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verduzco, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muddathir, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adil, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elhassan, G. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David
Polo Orozco, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardone, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reshkin, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harguindey, S.</span></span> <span> </span><span class="NLM_article-title">Glycolysis, tumor metabolism, cancer growth and dissemination. A new pH-based etiopathogenic perspective and therapeutic approach to an old cancer question</span>. <i>Oncoscience</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">777</span>– <span class="NLM_lpage">802</span>, <span class="refDoi"> DOI: 10.18632/oncoscience.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.18632%2Foncoscience.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=25621294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A280%3ADC%252BC2MvpvFOmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2014&pages=777-802&author=K.+O.+Alfaroukauthor=D.+Verduzcoauthor=C.+Rauchauthor=A.+K.+Muddathirauthor=H.+H.+Adilauthor=G.+O.+Elhassanauthor=M.+E.+Ibrahimauthor=J.+David%0APolo+Orozcoauthor=R.+A.+Cardoneauthor=S.+J.+Reshkinauthor=S.+Harguindey&title=Glycolysis%2C+tumor+metabolism%2C+cancer+growth+and+dissemination.+A+new+pH-based+etiopathogenic+perspective+and+therapeutic+approach+to+an+old+cancer+question&doi=10.18632%2Foncoscience.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">Glycolysis, tumor metabolism, cancer growth and dissemination. A new pH-based etiopathogenic perspective and therapeutic approach to an old cancer question</span></div><div class="casAuthors">Alfarouk Khalid O; Muddathir Abdel Khalig; Adil H H Bashir; Ibrahim Muntaser E; Verduzco Daniel; Rauch Cyril; Elhassan Gamal O; David Polo Orozco Julian; Harguindey Salvador; Cardone Rosa Angela; Reshkin Stephan J</div><div class="citationInfo"><span class="NLM_cas:title">Oncoscience</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">777-802</span>
        ISSN:<span class="NLM_cas:issn">2331-4737</span>.
    </div><div class="casAbstract">Cancer cells acquire an unusual glycolytic behavior relative, to a large extent, to their intracellular alkaline pH (pHi).  This effect is part of the metabolic alterations found in most, if not all, cancer cells to deal with unfavorable conditions, mainly hypoxia and low nutrient supply, in order to preserve its evolutionary trajectory with the production of lactate after ten steps of glycolysis.  Thus, cancer cells reprogram their cellular metabolism in a way that gives them their evolutionary and thermodynamic advantage.  Tumors exist within a highly heterogeneous microenvironment and cancer cells survive within any of the different habitats that lie within tumors thanks to the overexpression of different membrane-bound proton transporters.  This creates a highly abnormal and selective proton reversal in cancer cells and tissues that is involved in local cancer growth and in the metastatic process.  Because of this environmental heterogeneity, cancer cells within one part of the tumor may have a different genotype and phenotype than within another part.  This phenomenon has frustrated the potential of single-target therapy of this type of reductionist therapeutic approach over the last decades.  Here, we present a detailed biochemical framework on every step of tumor glycolysis and then proposea new paradigm and therapeutic strategy based upon the dynamics of the hydrogen ion in cancer cells and tissues in order to overcome the old paradigm of one enzyme-one target approach to cancer treatment.  Finally, a new and integral explanation of the Warburg effect is advanced.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREdAEVGEC34exsqfvU0ziafW6udTcc2eaiahoo4p4eF7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MvpvFOmsw%253D%253D&md5=c5014218b1ad904214a0f04a9318f03a</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.18632%2Foncoscience.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncoscience.109%26sid%3Dliteratum%253Aachs%26aulast%3DAlfarouk%26aufirst%3DK.%2BO.%26aulast%3DVerduzco%26aufirst%3DD.%26aulast%3DRauch%26aufirst%3DC.%26aulast%3DMuddathir%26aufirst%3DA.%2BK.%26aulast%3DAdil%26aufirst%3DH.%2BH.%26aulast%3DElhassan%26aufirst%3DG.%2BO.%26aulast%3DIbrahim%26aufirst%3DM.%2BE.%26aulast%3DDavid%2BPolo%2BOrozco%26aufirst%3DJ.%26aulast%3DCardone%26aufirst%3DR.%2BA.%26aulast%3DReshkin%26aufirst%3DS.%2BJ.%26aulast%3DHarguindey%26aufirst%3DS.%26atitle%3DGlycolysis%252C%2520tumor%2520metabolism%252C%2520cancer%2520growth%2520and%2520dissemination.%2520A%2520new%2520pH-based%2520etiopathogenic%2520perspective%2520and%2520therapeutic%2520approach%2520to%2520an%2520old%2520cancer%2520question%26jtitle%3DOncoscience%26date%3D2014%26volume%3D1%26spage%3D777%26epage%3D802%26doi%3D10.18632%2Foncoscience.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit2b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Alfarouk, K. O.</span></span> <span> </span><span class="NLM_article-title">Tumor metabolism, cancer cell transporters, and microenvironmental resistance</span>. <i>J. Enzyme Inhib. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">866</span>, <span class="refDoi"> DOI: 10.3109/14756366.2016.1140753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.3109%2F14756366.2016.1140753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=26864256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC28XisFGrsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2016&pages=859-866&author=K.+O.+Alfarouk&title=Tumor+metabolism%2C+cancer+cell+transporters%2C+and+microenvironmental+resistance&doi=10.3109%2F14756366.2016.1140753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor metabolism, cancer cell transporters, and microenvironmental resistance</span></div><div class="casAuthors">Alfarouk, Khalid O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Enzyme Inhibition and Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">JEIMAZ</span>;
        ISSN:<span class="NLM_cas:issn">1475-6366</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Cancer cells reprogram their metabolic machineries to enter into permanent glycolytic pathways.  The full reason for such reprogramming takes place is unclear.  However, this metabolic switch is not made in vain for the lactate that is generated and exported outside cells is reused by other cells.  This results in the generation of a pH gradient between the low extracellular pH that is acidic (pHe) and the higher cytosolic alk. or near neutral pH (pHi) environments that are tightly regulated by the overexpression of several pumps and ion channels (e.g. NHE-1, MCT-1, V-ATPase, CA9, and CA12).  The generation of this unique pH gradient serves as a detg. factor in defining "tumor fitness".  Tumor fitness is the capacity of the tumor to invade and metastasize due to its ability to reduce the efficiency of the immune system and confer resistance to chemotherapy.  In this article, we highlight the importance of tumor microenvironment in mediating the failure of chemotherapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAjPfA3PdelLVg90H21EOLACvtfcHk0li7OB7ICvqCkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisFGrsbw%253D&md5=881bbff08a460d2cbe01ee667271fa6a</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.3109%2F14756366.2016.1140753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F14756366.2016.1140753%26sid%3Dliteratum%253Aachs%26aulast%3DAlfarouk%26aufirst%3DK.%2BO.%26atitle%3DTumor%2520metabolism%252C%2520cancer%2520cell%2520transporters%252C%2520and%2520microenvironmental%2520resistance%26jtitle%3DJ.%2520Enzyme%2520Inhib.%2520Med.%2520Chem.%26date%3D2016%26volume%3D31%26spage%3D859%26epage%3D866%26doi%3D10.3109%2F14756366.2016.1140753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hensley, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasti, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeBerardinis, R. J.</span></span> <span> </span><span class="NLM_article-title">Glutamine and cancer: cell biology, physiology, and clinical opportunities</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">3678</span>– <span class="NLM_lpage">3684</span>, <span class="refDoi"> DOI: 10.1172/JCI69600</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1172%2FJCI69600" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=23999442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVenu7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2013&pages=3678-3684&author=C.+T.+Hensleyauthor=A.+T.+Wastiauthor=R.+J.+DeBerardinis&title=Glutamine+and+cancer%3A+cell+biology%2C+physiology%2C+and+clinical+opportunities&doi=10.1172%2FJCI69600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamine and cancer: cell biology, physiology, and clinical opportunities</span></div><div class="casAuthors">Hensley, Christopher T.; Wasti, Ajla T.; DeBerardinis, Ralph J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3678-3684</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">A review.  Glutamine is an abundant and versatile nutrient that participates in energy formation, redox homeostasis, macromol. synthesis, and signaling in cancer cells.  These characteristics make glutamine metab. an appealing target for new clin. strategies to detect, monitor, and treat cancer.  Here the authors review the metabolic functions of glutamine as a super nutrient and the surprising roles of glutamine in supporting the biol. hallmarks of malignancy.  The authors also review recent efforts in imaging and therapeutics to exploit tumor cell glutamine dependence, discuss some of the challenges in this arena, and suggest a disease-focused paradigm to deploy these emerging approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocNnK_GlTtRrVg90H21EOLACvtfcHk0li7OB7ICvqCkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVenu7bL&md5=fb704a7773710a20add6edbf0da08616</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1172%2FJCI69600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI69600%26sid%3Dliteratum%253Aachs%26aulast%3DHensley%26aufirst%3DC.%2BT.%26aulast%3DWasti%26aufirst%3DA.%2BT.%26aulast%3DDeBerardinis%26aufirst%3DR.%2BJ.%26atitle%3DGlutamine%2520and%2520cancer%253A%2520cell%2520biology%252C%2520physiology%252C%2520and%2520clinical%2520opportunities%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2013%26volume%3D123%26spage%3D3678%26epage%3D3684%26doi%3D10.1172%2FJCI69600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">DeBerardinis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancuso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daikhin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissim, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yudkoff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wehrli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span> <span> </span><span class="NLM_article-title">Beyond aerobic glycolysis: Transformed cells can engage in glutamine metabolism that exceeds the requirement for protein and nucleotide synthesis</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">19345</span>– <span class="NLM_lpage">19350</span>, <span class="refDoi"> DOI: 10.1073/pnas.0709747104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1073%2Fpnas.0709747104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=18032601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisVOjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=19345-19350&author=R.+J.+DeBerardinisauthor=A.+Mancusoauthor=E.+Daikhinauthor=I.+Nissimauthor=M.+Yudkoffauthor=S.+Wehrliauthor=C.+B.+Thompson&title=Beyond+aerobic+glycolysis%3A+Transformed+cells+can+engage+in+glutamine+metabolism+that+exceeds+the+requirement+for+protein+and+nucleotide+synthesis&doi=10.1073%2Fpnas.0709747104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4aR"><div class="casContent"><span class="casTitleNuber">4a</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond aerobic glycolysis: transformed cells can engage in glutamine metabolism that exceeds the requirement for protein and nucleotide synthesis</span></div><div class="casAuthors">DeBerardinis, Ralph J.; Mancuso, Anthony; Daikhin, Evgueni; Nissim, Ilana; Yudkoff, Marc; Wehrli, Suzanne; Thompson, Craig B.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19345-19350</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Tumor cell proliferation requires rapid synthesis of macromols. including lipids, proteins, and nucleotides.  Many tumor cells exhibit rapid glucose consumption, with most of the glucose-derived carbon being secreted as lactate despite abundant oxygen availability (the Warburg effect).  Here, we used 13C NMR spectroscopy to examine the metab. of glioblastoma cells exhibiting aerobic glycolysis.  In these cells, the tricarboxylic acid (TCA) cycle was active but was characterized by an efflux of substrates for use in biosynthetic pathways, particularly fatty acid synthesis.  The success of this synthetic activity depends on activation of pathways to generate reductive power (NADPH) and to restore oxaloacetate for continued TCA cycle function (anaplerosis).  Surprisingly, both these needs were met by a high rate of glutamine metab.  First, conversion of glutamine to lactate (glutaminolysis) was rapid enough to produce sufficient NADPH to support fatty acid synthesis.  Second, despite substantial mitochondrial pyruvate metab., pyruvate carboxylation was suppressed, and anaplerotic oxaloacetate was derived from glutamine.  Glutamine catabolism was accompanied by secretion of alanine and ammonia, such that most of the amino groups from glutamine were lost from the cell rather than incorporated into other mols.  These data demonstrate that transformed cells exhibit a high rate of glutamine consumption that cannot be explained by the nitrogen demand imposed by nucleotide synthesis or maintenance of nonessential amino acid pools.  Rather, glutamine metab. provides a carbon source that facilitates the cell's ability to use glucose-derived carbon and TCA cycle intermediates as biosynthetic precursors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoggsgqrqrWmrVg90H21EOLACvtfcHk0ljAIhs7BmR6dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisVOjtQ%253D%253D&md5=786a1c001dd9c477ea50d9227ebfece0</span></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0709747104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0709747104%26sid%3Dliteratum%253Aachs%26aulast%3DDeBerardinis%26aufirst%3DR.%2BJ.%26aulast%3DMancuso%26aufirst%3DA.%26aulast%3DDaikhin%26aufirst%3DE.%26aulast%3DNissim%26aufirst%3DI.%26aulast%3DYudkoff%26aufirst%3DM.%26aulast%3DWehrli%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DBeyond%2520aerobic%2520glycolysis%253A%2520Transformed%2520cells%2520can%2520engage%2520in%2520glutamine%2520metabolism%2520that%2520exceeds%2520the%2520requirement%2520for%2520protein%2520and%2520nucleotide%2520synthesis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D19345%26epage%3D19350%26doi%3D10.1073%2Fpnas.0709747104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit4b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wise, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span> <span> </span><span class="NLM_article-title">Glutamine addiction: a new therapeutic target in cancer</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">427</span>– <span class="NLM_lpage">433</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2010.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1016%2Fj.tibs.2010.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=20570523" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvFKqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2010&pages=427-433&author=D.+R.+Wiseauthor=C.+B.+Thompson&title=Glutamine+addiction%3A+a+new+therapeutic+target+in+cancer&doi=10.1016%2Fj.tibs.2010.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4bR"><div class="casContent"><span class="casTitleNuber">4b</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamine addiction: A new therapeutic target in cancer</span></div><div class="casAuthors">Wise, David R.; Thompson, Craig B.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">427-433</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Most cancers depend on a high rate of aerobic glycolysis for their continued growth and survival.  Paradoxically, some cancer cell lines also display addiction to glutamine despite the fact that glutamine is a nonessential amino acid that can be synthesized from glucose.  The high rate of glutamine uptake exhibited by glutamine-dependent cells does not appear to result solely from its role as a nitrogen donor in nucleotide and amino acid biosynthesis.  Instead, glutamine plays a required role in the uptake of essential amino acids and in maintaining activation of TOR (target of rapamycin) kinase.  Moreover, in many cancer cells, glutamine is the primary mitochondrial substrate and is required for maintenance of mitochondrial membrane potential and integrity and for support of the NADPH prodn. needed for redox control and macromol. synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4Gv_LsNdulLVg90H21EOLACvtfcHk0ljAIhs7BmR6dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvFKqs7s%253D&md5=915fa51996bc1c9b3ec3970b2dae7163</span></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2010.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2010.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DWise%26aufirst%3DD.%2BR.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DGlutamine%2520addiction%253A%2520a%2520new%2520therapeutic%2520target%2520in%2520cancer%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2010%26volume%3D35%26spage%3D427%26epage%3D433%26doi%3D10.1016%2Fj.tibs.2010.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit4c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span> <span> </span><span class="NLM_article-title">Glutaminolysis: supplying carbon or nitrogen or both for cancer cells?</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3884</span>– <span class="NLM_lpage">3886</span>, <span class="refDoi"> DOI: 10.4161/cc.9.19.13302</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.4161%2Fcc.9.19.13302" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=20948290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVentrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=3884-3886&author=C.+V.+Dang&title=Glutaminolysis%3A+supplying+carbon+or+nitrogen+or+both+for+cancer+cells%3F&doi=10.4161%2Fcc.9.19.13302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4cR"><div class="casContent"><span class="casTitleNuber">4c</span><div class="casTitle"><span class="NLM_cas:atitle">Glutaminolysis: supplying carbon or nitrogen or both for cancer cells?</span></div><div class="casAuthors">Dang, Chi V.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3884-3886</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  A cancer cell comprised largely of C, H, O, P, N and S requires not only glucose, which is avidly transported and converted to lactate by aerobic glycolysis or the Warburg effect, but also glutamine as a major substrate.  Glutamine and essential amino acids, such as Met, provide energy through the TCA cycle as well as N, S and C skeletons for growing and proliferating cancer cells.  The interplay between utilization of glutamine and glucose is likely to depend on the genetic make-up of a cancer cell.  While the MYC oncogene induces both aerobic glycolysis and glutaminolysis, activated β-catenin induces glutamine synthesis in hepatocellular carcinoma.  Cancer cells that have elevated glutamine synthetase can use glutamate and ammonia to synthesize glutamine and are hence not addicted to glutamine.  As such, cancer cells have many degrees of freedom for re-programming cell metab., which with better understanding will result in novel therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolGtCajTcmv7Vg90H21EOLACvtfcHk0ljAIhs7BmR6dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVentrY%253D&md5=a823f452698f7f082eb90873bc9bb2a1</span></div><a href="/servlet/linkout?suffix=cit4c&amp;dbid=16384&amp;doi=10.4161%2Fcc.9.19.13302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.9.19.13302%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26atitle%3DGlutaminolysis%253A%2520supplying%2520carbon%2520or%2520nitrogen%2520or%2520both%2520for%2520cancer%2520cells%253F%26jtitle%3DCell%2520Cycle%26date%3D2010%26volume%3D9%26spage%3D3884%26epage%3D3886%26doi%3D10.4161%2Fcc.9.19.13302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Altman, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stine, Z. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span> <span> </span><span class="NLM_article-title">From Krebs to clinic: glutamine metabolism to cancer therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">619</span>– <span class="NLM_lpage">634</span>, <span class="refDoi"> DOI: 10.1038/nrc.2016.71</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1038%2Fnrc.2016.71" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=27492215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12jurbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=619-634&author=B.+J.+Altmanauthor=Z.+E.+Stineauthor=C.+V.+Dang&title=From+Krebs+to+clinic%3A+glutamine+metabolism+to+cancer+therapy&doi=10.1038%2Fnrc.2016.71"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5aR"><div class="casContent"><span class="casTitleNuber">5a</span><div class="casTitle"><span class="NLM_cas:atitle">From Krebs to clinic: glutamine metabolism to cancer therapy</span></div><div class="casAuthors">Altman, Brian J.; Stine, Zachary E.; Dang, Chi V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">619-634</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The resurgence of research into cancer metab. has recently broadened interests beyond glucose and the Warburg effect to other nutrients, including glutamine.  Because oncogenic alterations of metab. render cancer cells addicted to nutrients, pathways involved in glycolysis or glutaminolysis could be exploited for therapeutic purposes.  In this Review, we provide an updated overview of glutamine metab. and its involvement in tumorigenesis in vitro and in vivo, and explore the recent potential applications of basic science discoveries in the clin. setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2GDHcwHXcnLVg90H21EOLACvtfcHk0lhOBIxnuLp1FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12jurbI&md5=00b34261d90a91151a3e833605c4aaec</span></div><a href="/servlet/linkout?suffix=cit5a&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2016.71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2016.71%26sid%3Dliteratum%253Aachs%26aulast%3DAltman%26aufirst%3DB.%2BJ.%26aulast%3DStine%26aufirst%3DZ.%2BE.%26aulast%3DDang%26aufirst%3DC.%2BV.%26atitle%3DFrom%2520Krebs%2520to%2520clinic%253A%2520glutamine%2520metabolism%2520to%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2016%26volume%3D16%26spage%3D619%26epage%3D634%26doi%3D10.1038%2Fnrc.2016.71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Koppula, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, B.</span></span> <span> </span><span class="NLM_article-title">Amino acid transporter SLC7A11/xCT at the crossroads of regulating redox homeostasis and nutrient dependency of cancer</span>. <i>Cancer Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1186/s40880-018-0288-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1186%2Fs40880-018-0288-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A280%3ADC%252BC1MfjsFOgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&pages=12-25&author=P.+Koppulaauthor=Y.+Zhangauthor=L.+Zhuangauthor=B.+Gan&title=Amino+acid+transporter+SLC7A11%2FxCT+at+the+crossroads+of+regulating+redox+homeostasis+and+nutrient+dependency+of+cancer&doi=10.1186%2Fs40880-018-0288-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5bR"><div class="casContent"><span class="casTitleNuber">5b</span><div class="casTitle"><span class="NLM_cas:atitle">Amino acid transporter SLC7A11/xCT at the crossroads of regulating redox homeostasis and nutrient dependency of cancer</span></div><div class="casAuthors">Koppula Pranavi; Zhang Yilei; Zhuang Li; Gan Boyi; Koppula Pranavi; Gan Boyi; Gan Boyi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer communications (London, England)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer cells often upregulate nutrient transporters to fulfill their increased biosynthetic and bioenergetic needs, and to maintain redox homeostasis.  One nutrient transporter frequently overexpressed in human cancers is the cystine/glutamate antiporter solute carrier family 7 member 11 (SLC7A11; also known as xCT).  SLC7A11 promotes cystine uptake and glutathione biosynthesis, resulting in protection from oxidative stress and ferroptotic cell death.  Recent studies have unexpectedly revealed that SLC7A11 also plays critical roles in glutamine metabolism and regulates the glucose and glutamine dependency of cancer cells.  This review discusses the roles of SLC7A11 in regulating the antioxidant response and nutrient dependency of cancer cells, explores our current understanding of SLC7A11 regulation in cancer metabolism, and highlights key open questions for future studies in this emerging research area.  A deeper understanding of SLC7A11 in cancer metabolism may identify new therapeutic opportunities to target this important amino acid transporter for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSc0x1bKxSR5v69nzq6467DfW6udTcc2eYNSMEB4XePnrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MfjsFOgug%253D%253D&md5=22acaa68350c28038eb70f6974151c9f</span></div><a href="/servlet/linkout?suffix=cit5b&amp;dbid=16384&amp;doi=10.1186%2Fs40880-018-0288-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs40880-018-0288-x%26sid%3Dliteratum%253Aachs%26aulast%3DKoppula%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhuang%26aufirst%3DL.%26aulast%3DGan%26aufirst%3DB.%26atitle%3DAmino%2520acid%2520transporter%2520SLC7A11%252FxCT%2520at%2520the%2520crossroads%2520of%2520regulating%2520redox%2520homeostasis%2520and%2520nutrient%2520dependency%2520of%2520cancer%26jtitle%3DCancer%2520Commun.%26date%3D2018%26volume%3D38%26spage%3D12%26epage%3D25%26doi%3D10.1186%2Fs40880-018-0288-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Daemen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nannini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liederer, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Cruz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangaraju, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaochico, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunsaker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latham, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatzivassiliou, G.</span></span> <span> </span><span class="NLM_article-title">Pan-cancer metabolic signature predicts co-dependency on glutaminase and de novo glutathione synthesis linked to a high-mesenchymal cell state</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>, <span class="NLM_elocation-id">383.e9</span> <span class="refDoi"> DOI: 10.1016/j.cmet.2018.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1016%2Fj.cmet.2018.06.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&author=A.+Daemenauthor=B.+Liuauthor=K.+Songauthor=K.+Songauthor=M.+Gaoauthor=R.+Hongauthor=M.+Nanniniauthor=D.+Petersonauthor=B.+M.+Liedererauthor=C.+de+la+Cruzauthor=D.+Sangarajuauthor=A.+Jaochicoauthor=X.+Zhaoauthor=W.+Sandovalauthor=T.+Hunsakerauthor=R.+Firesteinauthor=S.+Lathamauthor=D.+Sampathauthor=D.+Sampathauthor=G.+Hatzivassiliou&title=Pan-cancer+metabolic+signature+predicts+co-dependency+on+glutaminase+and+de+novo+glutathione+synthesis+linked+to+a+high-mesenchymal+cell+state&doi=10.1016%2Fj.cmet.2018.06.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2018.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2018.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DDaemen%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DHong%26aufirst%3DR.%26aulast%3DNannini%26aufirst%3DM.%26aulast%3DPeterson%26aufirst%3DD.%26aulast%3DLiederer%26aufirst%3DB.%2BM.%26aulast%3Dde%2Bla%2BCruz%26aufirst%3DC.%26aulast%3DSangaraju%26aufirst%3DD.%26aulast%3DJaochico%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DSandoval%26aufirst%3DW.%26aulast%3DHunsaker%26aufirst%3DT.%26aulast%3DFirestein%26aufirst%3DR.%26aulast%3DLatham%26aufirst%3DS.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DHatzivassiliou%26aufirst%3DG.%26atitle%3DPan-cancer%2520metabolic%2520signature%2520predicts%2520co-dependency%2520on%2520glutaminase%2520and%2520de%2520novo%2520glutathione%2520synthesis%2520linked%2520to%2520a%2520high-mesenchymal%2520cell%2520state%26jtitle%3DCell%2520Metab.%26date%3D2018%26volume%3D28%26doi%3D10.1016%2Fj.cmet.2018.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit6b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayin, V. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeBoeuf, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karakousi, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhutkar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Rivera, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subbaraj, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prigge, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goparaju, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolgalev, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heguy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allaj, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreira, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudin, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pass, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacks, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papagiannakopoulos, T.</span></span> <span> </span><span class="NLM_article-title">Keap1 loss promotes Kras-driven lung cancer and results in dependence on glutaminolysis</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1362</span>– <span class="NLM_lpage">1368</span>, <span class="refDoi"> DOI: 10.1038/nm.4407</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1038%2Fnm.4407" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=28967920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFylurrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=1362-1368&author=R.+Romeroauthor=V.+I.+Sayinauthor=S.+M.+Davidsonauthor=M.+R.+Bauerauthor=S.+X.+Singhauthor=S.+E.+LeBoeufauthor=T.+R.+Karakousiauthor=D.+C.+Ellisauthor=A.+Bhutkarauthor=S.+J.+S%C3%A1nchez-Riveraauthor=L.+Subbarajauthor=B.+Martinezauthor=R.+T.+Bronsonauthor=J.+R.+Priggeauthor=E.+E.+Schmidtauthor=C.+J.+Thomasauthor=C.+Goparajuauthor=A.+Daviesauthor=I.+Dolgalevauthor=A.+Heguyauthor=V.+Allajauthor=J.+T.+Poirierauthor=A.+L.+Moreiraauthor=C.+M.+Rudinauthor=H.+I.+Passauthor=M.+G.+Vander%0AHeidenauthor=T.+Jacksauthor=T.+Papagiannakopoulos&title=Keap1+loss+promotes+Kras-driven+lung+cancer+and+results+in+dependence+on+glutaminolysis&doi=10.1038%2Fnm.4407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 loss promotes Kras-driven lung cancer and results in dependence on glutaminolysis</span></div><div class="casAuthors">Romero, Rodrigo; Sayin, Volkan I.; Davidson, Shawn M.; Bauer, Matthew R.; Singh, Simranjit X.; Le Boeuf, Sarah E.; Karakousi, Triantafyllia R.; Ellis, Donald C.; Bhutkar, Arjun; Sanchez-Rivera, Francisco J.; Subbaraj, Lakshmipriya; Martinez, Britney; Bronson, Roderick T.; Prigge, Justin R.; Schmidt, Edward E.; Thomas, Craig J.; Goparaju, Chandra; Davies, Angela; Dolgalev, Igor; Heguy, Adriana; Allaj, Viola; Poirier, John T.; Moreira, Andre L.; Rudin, Charles M.; Pass, Harvey I.; Vander Heiden, Matthew G.; Jacks, Tyler; Papagiannakopoulos, Thales</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1362-1368</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Treating KRAS-mutant lung adenocarcinoma (LUAD) remains a major challenge in cancer treatment given the difficulties assocd. with directly inhibiting the KRAS oncoprotein.  One approach to addressing this challenge is to define mutations that frequently co-occur with those in KRAS, which themselves may lead to therapeutic vulnerabilities in tumors.  Approx. 20% of KRAS-mutant LUAD tumors carry loss-of-function mutations in the KEAP1 gene encoding Kelch-like ECH-assocd. protein 1 (refs. 2, 3, 4), a neg. regulator of nuclear factor erythroid 2-like 2 (NFE2L2; hereafter NRF2), which is the master transcriptional regulator of the endogenous antioxidant response.  The high frequency of mutations in KEAP1 suggests an important role for the oxidative stress response in lung tumorigenesis.  Using a CRISPR-Cas9-based approach in a mouse model of KRAS-driven LUAD, we examd. the effects of Keap1 loss in lung cancer progression.  We show that loss of Keap1 hyperactivates NRF2 and promotes KRAS-driven LUAD in mice.  Through a combination of CRISPR-Cas9-based genetic screening and metabolomic analyses, we show that Keap1- or Nrf2-mutant cancers are dependent on increased glutaminolysis, and this property can be therapeutically exploited through the pharmacol. inhibition of glutaminase.  Finally, we provide a rationale for stratification of human patients with lung cancer harboring KRAS/KEAP1- or KRAS/NRF2-mutant lung tumors as likely to respond to glutaminase inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofWgHF0gdsW7Vg90H21EOLACvtfcHk0lhhbVtJoiqZsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFylurrK&md5=2bc4122a4c757cc3a5fdab3eb62b26e3</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1038%2Fnm.4407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4407%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DR.%26aulast%3DSayin%26aufirst%3DV.%2BI.%26aulast%3DDavidson%26aufirst%3DS.%2BM.%26aulast%3DBauer%26aufirst%3DM.%2BR.%26aulast%3DSingh%26aufirst%3DS.%2BX.%26aulast%3DLeBoeuf%26aufirst%3DS.%2BE.%26aulast%3DKarakousi%26aufirst%3DT.%2BR.%26aulast%3DEllis%26aufirst%3DD.%2BC.%26aulast%3DBhutkar%26aufirst%3DA.%26aulast%3DS%25C3%25A1nchez-Rivera%26aufirst%3DS.%2BJ.%26aulast%3DSubbaraj%26aufirst%3DL.%26aulast%3DMartinez%26aufirst%3DB.%26aulast%3DBronson%26aufirst%3DR.%2BT.%26aulast%3DPrigge%26aufirst%3DJ.%2BR.%26aulast%3DSchmidt%26aufirst%3DE.%2BE.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DGoparaju%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DA.%26aulast%3DDolgalev%26aufirst%3DI.%26aulast%3DHeguy%26aufirst%3DA.%26aulast%3DAllaj%26aufirst%3DV.%26aulast%3DPoirier%26aufirst%3DJ.%2BT.%26aulast%3DMoreira%26aufirst%3DA.%2BL.%26aulast%3DRudin%26aufirst%3DC.%2BM.%26aulast%3DPass%26aufirst%3DH.%2BI.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DJacks%26aufirst%3DT.%26aulast%3DPapagiannakopoulos%26aufirst%3DT.%26atitle%3DKeap1%2520loss%2520promotes%2520Kras-driven%2520lung%2520cancer%2520and%2520results%2520in%2520dependence%2520on%2520glutaminolysis%26jtitle%3DNat.%2520Med.%26date%3D2017%26volume%3D23%26spage%3D1362%26epage%3D1368%26doi%3D10.1038%2Fnm.4407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willis, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seegmiller, J. E.</span></span> <span> </span><span class="NLM_article-title">The inhibition by 6-diazo-5-oxo-<span class="smallcaps smallerCapital">l</span>-norleucine of glutamine catabolism of the cultured human lymphoblast</span>. <i>J. Cell Physiol.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">382</span>, <span class="refDoi"> DOI: 10.1002/jcp.1040930308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1002%2Fjcp.1040930308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=22551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADyaE1cXkvVShtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1977&pages=375-382&author=R.+C.+Willisauthor=J.+E.+Seegmiller&title=The+inhibition+by+6-diazo-5-oxo-l-norleucine+of+glutamine+catabolism+of+the+cultured+human+lymphoblast&doi=10.1002%2Fjcp.1040930308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The inhibition by 6-diazo-5-oxo-L-norleucine of glutamine catabolism of the cultured human lymphoblast</span></div><div class="casAuthors">Willis, Randall C.; Seegmiller, J. Edwin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular Physiology</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">375-82</span>CODEN:
                <span class="NLM_cas:coden">JCLLAX</span>;
        ISSN:<span class="NLM_cas:issn">0021-9541</span>.
    </div><div class="casAbstract">The rapid catabolism of glutamine by the cultured human lymphoblast line WI-L2 was inhibited by >95% during incubation of cell suspensions with 6-diazo-5-oxo-L-norleucine (I).  The inhibition persisted for ≥4 h after the removal of I from the cell suspension.  The exposure of cells to I inhibited >95% of the glutaminase activity measured in lysates in the presence of either phosphate or maleate.  Similarly, γ-glutamyl transpeptidase, assayed with γ-glutamyl-p-nitroanilide as substrate and glycylglycine as acceptor, was inhibited >90%.  I-treated and control cells accumulated radioactive material from suspensions contg. L-glutamine-14C at similar initial rates.  The radioactive material accumulated by the I-treated cells was all recoverable as glutamine, whereas the radioactive material accumulated by untreated cells was principally recovered as glutamate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooEluRc1A7arVg90H21EOLACvtfcHk0lhhbVtJoiqZsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXkvVShtQ%253D%253D&md5=75a871601964879332b313079c9d61a2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fjcp.1040930308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcp.1040930308%26sid%3Dliteratum%253Aachs%26aulast%3DWillis%26aufirst%3DR.%2BC.%26aulast%3DSeegmiller%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520inhibition%2520by%25206-diazo-5-oxo-l-norleucine%2520of%2520glutamine%2520catabolism%2520of%2520the%2520cultured%2520human%2520lymphoblast%26jtitle%3DJ.%2520Cell%2520Physiol.%26date%3D1977%26volume%3D93%26spage%3D375%26epage%3D382%26doi%3D10.1002%2Fjcp.1040930308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Steckel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philips, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, T. C.</span></span> <span> </span><span class="NLM_article-title">Kinetic properties and inhibition of Acinetobacter glutaminase-asparaginase</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">971</span>– <span class="NLM_lpage">977</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(83)90613-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1016%2F0006-2952%2883%2990613-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=6838661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADyaL3sXitFajtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1983&pages=971-977&author=J.+Steckelauthor=J.+Robertsauthor=F.+S.+Philipsauthor=T.+C.+Chou&title=Kinetic+properties+and+inhibition+of+Acinetobacter+glutaminase-asparaginase&doi=10.1016%2F0006-2952%2883%2990613-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8aR"><div class="casContent"><span class="casTitleNuber">8a</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic properties and inhibition of Acinetobacter glutaminase-asparaginase</span></div><div class="casAuthors">Steckel, Joph; Roberts, Joseph; Philips, Frederick S.; Chou, Ting Chao</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">971-7</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">Kinetics parameters, substrate specificity, and exclusivity of ligands at binding sites of L-glutaminase-L-asparaginase (I) purified from A. glutaminasificans were studied to gain knowledge about the dual activities of this enzyme and its inhibition by structural analogs.  Both L-glutamine and L-asparagine, which showed similar Km (4-7 × 10-5M) and Vmax (mol. activity 1.0 min-1) values, were competitive with each other for the substrate-binding site.  The products, L-glutamic acid and L-aspartic acid, showed competitive inhibition with respect to either L-glutamine or L-asparagine as substrates.  Multiple inhibition of the glutaminase activity by L-glutamic acid and L-aspartic acid indicated that these ligands are mutually exclusive at the product-releasing site.  The initial rates of both of the activities were competitively inhibited by the following inhibitors (in decreasing order of activity): 6-diazo-5-oxo-L-norleucine (DON), L-methionine sulfoximine, azaserine, and Acivicin.  DON and azaserine inhibited both activities in a time-dependent and irreversible manner.  The kinetic data suggest an ordered mechanism with glutamine or asparagine as the 1st substrate and glutamic acid or aspartic acid, resp., as the last product.  A single mechanism and a single set of binding sites are suggested as responsible for catalyzing both of the activities.  The data also showed that succinylated enzyme, which has a 10-fold increase of plasma half-life in animals and humans and, thus, has benefit as a cancer chemotherapeutic agent, retained its catalytic activity and maintained Km and Vmax values similar to native I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWgzvqrWgjn7Vg90H21EOLACvtfcHk0liHBd_qveRSzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXitFajtLo%253D&md5=a78f0c944956e2b1ac0389ee3cca2596</span></div><a href="/servlet/linkout?suffix=cit8a&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2883%2990613-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252883%252990613-5%26sid%3Dliteratum%253Aachs%26aulast%3DSteckel%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%26aulast%3DPhilips%26aufirst%3DF.%2BS.%26aulast%3DChou%26aufirst%3DT.%2BC.%26atitle%3DKinetic%2520properties%2520and%2520inhibition%2520of%2520Acinetobacter%2520glutaminase-asparaginase%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1983%26volume%3D32%26spage%3D971%26epage%3D977%26doi%3D10.1016%2F0006-2952%2883%2990613-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit8b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proudfoot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skarina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dementieva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joachimiak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yakunin, A. F.</span></span> <span> </span><span class="NLM_article-title">Functional and structural characterization of four glutaminases from <i>Escherichia coli</i> and <i>Bacillus subtilis</i></span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">5724</span>– <span class="NLM_lpage">5735</span>, <span class="refDoi"> DOI: 10.1021/bi800097h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi800097h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsVKmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=5724-5735&author=G.+Brownauthor=A.+Singerauthor=M.+Proudfootauthor=T.+Skarinaauthor=Y.+Kimauthor=C.+Changauthor=I.+Dementievaauthor=E.+Kuznetsovaauthor=C.+F.+Gonzalezauthor=A.+Joachimiakauthor=A.+Savchenkoauthor=A.+F.+Yakunin&title=Functional+and+structural+characterization+of+four+glutaminases+from+Escherichia+coli+and+Bacillus+subtilis&doi=10.1021%2Fbi800097h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8bR"><div class="casContent"><span class="casTitleNuber">8b</span><div class="casTitle"><span class="NLM_cas:atitle">Functional and Structural Characterization of Four Glutaminases from Escherichia coli and Bacillus subtilis</span></div><div class="casAuthors">Brown, Greg; Singer, Alex; Proudfoot, Michael; Skarina, Tatiana; Kim, Youngchang; Chang, Changsoo; Dementieva, Irina; Kuznetsova, Ekaterina; Gonzalez, Claudio F.; Joachimiak, Andrzej; Savchenko, Alexei; Yakunin, Alexander F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5724-5735</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Glutaminases belong to the large superfamily of serine-dependent β-lactamases and penicillin-binding proteins, and they catalyze the hydrolytic deamidation of L-glutamine to L-glutamate.  In this work, we purified and biochem. characterized four predicted glutaminases from Escherichia coli (YbaS and YneH) and Bacillus subtilis (YlaM and YbgJ).  The proteins demonstrated strict specificity to L-glutamine and did not hydrolyze D-glutamine or L-asparagine.  In each organism, one glutaminase showed higher affinity to glutamine (E. coli YbaS and B. subtilis YlaM; Km 7.3 and 7.6 mM, resp.) than the second glutaminase (E. coli YneH and B. subtilis YbgJ; Km 27.6 and 30.6 mM, resp.).  The crystal structures of the E. coli YbaS and the B. subtilis YbgJ revealed the presence of a classical β-lactamase-like fold and conservation of several key catalytic residues of β-lactamases (Ser74, Lys77, Asn126, Lys268, and Ser269 in YbgJ).  Alanine replacement mutagenesis demonstrated that most of the conserved residues located in the putative glutaminase catalytic site are essential for activity.  The crystal structure of the YbgJ complex with the glutaminase inhibitor 6-diazo-5-oxo-L-norleucine revealed the presence of a covalent bond between the inhibitor and the hydroxyl oxygen of Ser74, providing evidence that Ser74 is the primary catalytic nucleophile and that the glutaminase reaction proceeds through formation of an enzyme-glutamyl intermediate.  Growth expts. with the E. coli glutaminase deletion strains revealed that YneH is involved in the assimilation of L-glutamine as a sole source of carbon and nitrogen and suggested that both glutaminases (YbaS and YneH) also contribute to acid resistance in E. coli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6EpjjxHqthLVg90H21EOLACvtfcHk0liHBd_qveRSzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsVKmtrs%253D&md5=5c903849dd40ce74958dac2453b55173</span></div><a href="/servlet/linkout?suffix=cit8b&amp;dbid=16384&amp;doi=10.1021%2Fbi800097h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi800097h%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DG.%26aulast%3DSinger%26aufirst%3DA.%26aulast%3DProudfoot%26aufirst%3DM.%26aulast%3DSkarina%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DChang%26aufirst%3DC.%26aulast%3DDementieva%26aufirst%3DI.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DGonzalez%26aufirst%3DC.%2BF.%26aulast%3DJoachimiak%26aufirst%3DA.%26aulast%3DSavchenko%26aufirst%3DA.%26aulast%3DYakunin%26aufirst%3DA.%2BF.%26atitle%3DFunctional%2520and%2520structural%2520characterization%2520of%2520four%2520glutaminases%2520from%2520Escherichia%2520coli%2520and%2520Bacillus%2520subtilis%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D5724%26epage%3D5735%26doi%3D10.1021%2Fbi800097h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McBryant, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraris, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curthoys, N. P.</span></span> <span> </span><span class="NLM_article-title">Novel mechanism of inhibition of rat kidney-type glutaminase by bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide (BPTES)</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>406</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">414</span>, <span class="refDoi"> DOI: 10.1042/BJ20070039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1042%2FBJ20070039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=17581113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BD2sXps1Wgsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=406&publication_year=2007&pages=407-414&author=M.+M.+Robinsonauthor=S.+J.+McBryantauthor=T.+Tsukamotoauthor=C.+Rojasauthor=D.+V.+Ferrarisauthor=S.+K.+Hamiltonauthor=J.+C.+Hansenauthor=N.+P.+Curthoys&title=Novel+mechanism+of+inhibition+of+rat+kidney-type+glutaminase+by+bis-2-%285-phenylacetamido-1%2C2%2C4-thiadiazol-2-yl%29ethyl+sulfide+%28BPTES%29&doi=10.1042%2FBJ20070039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Novel mechanism of inhibition of rat kidney-type glutaminase by bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide (BPTES)</span></div><div class="casAuthors">Robinson, Mary M.; McBryant, Steven J.; Tsukamoto, Takashi; Rojas, Camilo; Ferraris, Dana V.; Hamilton, Sean K.; Hansen, Jeffrey C.; Curthoys, Norman P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">407-414</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The release of GA (mitochondrial glutaminase) from neurons following acute ischemia or during chronic neurodegenerative diseases may contribute to the propagation of glutamate excitotoxicity.  Thus an inhibitor that selectively inactivates the released GA may limit the accumulation of excess glutamate and minimize the loss of neurol. function that accompanies brain injury.  The present study examines the mechanism of inactivation of rat KGA (kidney GA isoform) by the small-mol. inhibitor BPTES [bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide].  BPTES is a potent inhibitor of KGA, but not of the liver GA isoform, glutamate dehydrogenase or γ-glutamyl transpeptidase.  Kinetic studies indicate that, with respect to glutamine, BPTES has a Ki of approx. 3 μM.  Moreover, these studies suggest that BPTES inhibits the allosteric activation caused by phosphate binding and promotes the formation of an inactive complex.  Gel-filtration chromatog. and sedimentation-velocity anal. were used to examine the effect of BPTES on the phosphate-dependent oligomerization of KGA.  This established that BPTES prevents the formation of large phosphate-induced oligomers and instead promotes the formation of a single oligomeric species with distinct phys. properties.  Sedimentation-equil. studies detd. that the oligomer produced by BPTES is a stable tetramer.  Taken together, the present work indicates that BPTES is a unique and potent inhibitor of rat KGA and elucidates a novel mechanism of inactivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR5Ncyq__sELVg90H21EOLACvtfcHk0liHBd_qveRSzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXps1Wgsb4%253D&md5=cde9754c0dedd000423226e2289b4690</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1042%2FBJ20070039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20070039%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DM.%2BM.%26aulast%3DMcBryant%26aufirst%3DS.%2BJ.%26aulast%3DTsukamoto%26aufirst%3DT.%26aulast%3DRojas%26aufirst%3DC.%26aulast%3DFerraris%26aufirst%3DD.%2BV.%26aulast%3DHamilton%26aufirst%3DS.%2BK.%26aulast%3DHansen%26aufirst%3DJ.%2BC.%26aulast%3DCurthoys%26aufirst%3DN.%2BP.%26atitle%3DNovel%2520mechanism%2520of%2520inhibition%2520of%2520rat%2520kidney-type%2520glutaminase%2520by%2520bis-2-%25285-phenylacetamido-1%252C2%252C4-thiadiazol-2-yl%2529ethyl%2520sulfide%2520%2528BPTES%2529%26jtitle%3DBiochem.%2520J.%26date%3D2007%26volume%3D406%26spage%3D407%26epage%3D414%26doi%3D10.1042%2FBJ20070039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeLaBarre, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J,J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, J. B.</span></span> <span> </span><span class="NLM_article-title">Full-length human glutaminase in complex with an allosteric inhibitor</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">10764</span>– <span class="NLM_lpage">10770</span>, <span class="refDoi"> DOI: 10.1021/bi201613d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi201613d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVSks73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=10764-10770&author=B.+DeLaBarreauthor=S.+Grossauthor=C.+Fangauthor=Y.+Gaoauthor=A.+Jhaauthor=F.+Jiangauthor=J%2CJ.+Songauthor=W.+Weiauthor=J.+B.+Hurov&title=Full-length+human+glutaminase+in+complex+with+an+allosteric+inhibitor&doi=10.1021%2Fbi201613d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Full-Length Human Glutaminase in Complex with an Allosteric Inhibitor</span></div><div class="casAuthors">DeLaBarre, Byron; Gross, Stefan; Fang, Cheng; Gao, Yi; Jha, Abhishek; Jiang, Fan; Song J., Juanhua; Wei, Wentao; Hurov, Jonathan B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">10764-10770</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Glutaminase (GLS1/2) catalyzes the conversion of L-glutamine to L-glutamate and ammonia.  The level of a splice variant of GLS1 (GAC) is elevated in certain cancers, and GAC is specifically inhibited by bis-2-(5-phenylacetimido-1,2,4,thiadiazol-2-yl)ethyl sulfide (BPTES).  We report here the first full-length crystal structure of GAC in the presence and absence of BPTES mols.  Two BPTES mols. bind at an interface region of the GAC tetramer in a manner that appears to lock the GAC tetramer into a nonproductive conformation.  The importance of these loops with regard to overall enzymic activity of the tetramer was revealed by a series of GAC point mutants designed to create a BPTES resistant GAC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrj21-Ko0HGLVg90H21EOLACvtfcHk0linRLWw0g9wew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVSks73K&md5=bd94996062f54c4f51a138c8e9f5f99a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fbi201613d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi201613d%26sid%3Dliteratum%253Aachs%26aulast%3DDeLaBarre%26aufirst%3DB.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DC.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DJha%26aufirst%3DA.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DSong%26aufirst%3DJ%252CJ.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DHurov%26aufirst%3DJ.%2BB.%26atitle%3DFull-length%2520human%2520glutaminase%2520in%2520complex%2520with%2520an%2520allosteric%2520inhibitor%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D10764%26epage%3D10770%26doi%3D10.1021%2Fbi201613d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuji, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramachandran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinavahi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrosio, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerione, R. A.</span></span> <span> </span><span class="NLM_article-title">Targeting mitochondrial glutaminase activity inhibits oncogenic transformation</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">219</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2010.08.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1016%2Fj.ccr.2010.08.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=20832749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFGht7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=207-219&author=J.+B.+Wangauthor=J.+W.+Ericksonauthor=R.+Fujiauthor=S.+Ramachandranauthor=P.+Gaoauthor=R.+Dinavahiauthor=K.+F.+Wilsonauthor=A.+L.+Ambrosioauthor=S.+M.+Diasauthor=C.+V.+Dangauthor=R.+A.+Cerione&title=Targeting+mitochondrial+glutaminase+activity+inhibits+oncogenic+transformation&doi=10.1016%2Fj.ccr.2010.08.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Mitochondrial Glutaminase Activity Inhibits Oncogenic Transformation</span></div><div class="casAuthors">Wang, Jian-Bin; Erickson, Jon W.; Fuji, Reina; Ramachandran, Sekar; Gao, Ping; Dinavahi, Ramani; Wilson, Kristin F.; Ambrosio, Andre L. B.; Dias, Sandra M. G.; Dang, Chi-V.; Cerione, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">207-219</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Rho GTPases impact a no. of activities important for oncogenesis.  We describe a small mol. inhibitor that blocks oncogenic transformation induced by various Rho GTPases in fibroblasts, and the growth of human breast cancer and B lymphoma cells, without affecting normal cells.  We identify the target of this inhibitor to be the metabolic enzyme glutaminase, which catalyzes the hydrolysis of glutamine to glutamate.  We show that transformed fibroblasts and breast cancer cells exhibit elevated glutaminase activity that is dependent on Rho GTPases and NF-κB activity, and is blocked by the small mol. inhibitor.  These findings highlight a previously unappreciated connection between Rho GTPase activation and cellular metab. and demonstrate that targeting glutaminase activity can inhibit oncogenic transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTP6YsyzRd1LVg90H21EOLACvtfcHk0linRLWw0g9wew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFGht7bO&md5=67b64e99c992bbe6636ecfc04b83aac0</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2010.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2010.08.009%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%2BB.%26aulast%3DErickson%26aufirst%3DJ.%2BW.%26aulast%3DFuji%26aufirst%3DR.%26aulast%3DRamachandran%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DP.%26aulast%3DDinavahi%26aufirst%3DR.%26aulast%3DWilson%26aufirst%3DK.%2BF.%26aulast%3DAmbrosio%26aufirst%3DA.%2BL.%26aulast%3DDias%26aufirst%3DS.%2BM.%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DCerione%26aufirst%3DR.%2BA.%26atitle%3DTargeting%2520mitochondrial%2520glutaminase%2520activity%2520inhibits%2520oncogenic%2520transformation%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D18%26spage%3D207%26epage%3D219%26doi%3D10.1016%2Fj.ccr.2010.08.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burns, A. C.</span>; <span class="NLM_string-name">Collins, M. R.</span>; <span class="NLM_string-name">Greasley, S. E.</span>; <span class="NLM_string-name">Hoffman, R. L.</span>; <span class="NLM_string-name">Huang, Q.</span>; <span class="NLM_string-name">Kania, R.
S.</span>; <span class="NLM_string-name">Kung, P.-P.</span>; <span class="NLM_string-name">Linton, M. A.</span>; <span class="NLM_string-name">Narasimhan, L. S.</span>; <span class="NLM_string-name">Richardson, P. F.</span>; <span class="NLM_string-name">Richter, D. T.</span>; <span class="NLM_string-name">Smith, G.</span></span> <span> </span><span class="NLM_article-title">Cycloalkyl-Linked Diheterocycle Derivatives</span>. International Patent Application Number <span class="NLM_patent">WO2015/166373 A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=A.+C.+Burns&author=M.+R.+Collins&author=S.+E.+Greasley&author=R.+L.+Hoffman&author=Q.+Huang&author=R.%0AS.+Kania&author=P.-P.+Kung&author=M.+A.+Linton&author=L.+S.+Narasimhan&author=P.+F.+Richardson&author=D.+T.+Richter&author=G.+Smith&title=Cycloalkyl-Linked+Diheterocycle+Derivatives"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBurns%26aufirst%3DA.%2BC.%26atitle%3DCycloalkyl-Linked%2520Diheterocycle%2520Derivatives%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, R. M.</span>; <span class="NLM_string-name">Popovici-Muller, J.</span>; <span class="NLM_string-name">Salituro, F. G.</span>; <span class="NLM_string-name">Saunders, J. O.</span>; <span class="NLM_string-name">Travins, J.</span>; <span class="NLM_string-name">Yan, S.</span></span> <span> </span><span class="NLM_article-title">Glutaminase Inhibitors and Method of Use</span>. International Patent Application Number <span class="NLM_patent">WO2014/081925 A1</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=R.+M.+Lemieux&author=J.+Popovici-Muller&author=F.+G.+Salituro&author=J.+O.+Saunders&author=J.+Travins&author=S.+Yan&title=Glutaminase+Inhibitors+and+Method+of+Use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLemieux%26aufirst%3DR.%2BM.%26atitle%3DGlutaminase%2520Inhibitors%2520and%2520Method%2520of%2520Use%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraris, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvall, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delahanty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rais, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slusher, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukamoto, T.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and pharmacological evaluation of bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide 3 (BPTES) analogs as glutaminase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">10551</span>– <span class="NLM_lpage">10563</span>, <span class="refDoi"> DOI: 10.1021/jm301191p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301191p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Ohs7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=10551-10563&author=K.+Shuklaauthor=D.+V.+Ferrarisauthor=A.+G.+Thomasauthor=M.+Stathisauthor=B.+Duvallauthor=G.+Delahantyauthor=J.+Altauthor=R.+Raisauthor=C.+Rojasauthor=P.+Gaoauthor=Y.+Xiangauthor=C.+V.+Dangauthor=B.+S.+Slusherauthor=T.+Tsukamoto&title=Design%2C+synthesis%2C+and+pharmacological+evaluation+of+bis-2-%285-phenylacetamido-1%2C2%2C4-thiadiazol-2-yl%29ethyl+sulfide+3+%28BPTES%29+analogs+as+glutaminase+inhibitors&doi=10.1021%2Fjm301191p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14aR"><div class="casContent"><span class="casTitleNuber">14a</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Pharmacological Evaluation of Bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl Sulfide 3 (BPTES) Analogs as Glutaminase Inhibitors</span></div><div class="casAuthors">Shukla, Krupa; Ferraris, Dana V.; Thomas, Ajit G.; Stathis, Marigo; Duvall, Bridget; Delahanty, Greg; Alt, Jesse; Rais, Rana; Rojas, Camilo; Gao, Ping; Xiang, Yan; Dang, Chi V.; Slusher, Barbara S.; Tsukamoto, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10551-10563</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bis-2-(5-phenylacetamido-1,2,4-thiadiazol-2-yl)ethyl sulfide (BPTES) is a potent and selective allosteric inhibitor of kidney-type glutaminase (GLS) that has served as a mol. probe to det. the therapeutic potential of GLS inhibition.  In an attempt to identify more potent GLS inhibitors with improved drug-like mol. properties, a series of BPTES analogs were synthesized and evaluated.  The structure-activity relationship (SAR) studies revealed that some truncated analogs retained the potency of BPTES, presenting an opportunity to improve its aq. soly.  The analog I exhibited similar potency and better soly. relative to BPTES and attenuated the growth of P493 human lymphoma B cells in vitro as well as in a mouse xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0N1SUSOaTCbVg90H21EOLACvtfcHk0lhwiruHCytUIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Ohs7zM&md5=7a54f0c8de3c211a65d5c9bbb4a6db62</span></div><a href="/servlet/linkout?suffix=cit14a&amp;dbid=16384&amp;doi=10.1021%2Fjm301191p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301191p%26sid%3Dliteratum%253Aachs%26aulast%3DShukla%26aufirst%3DK.%26aulast%3DFerraris%26aufirst%3DD.%2BV.%26aulast%3DThomas%26aufirst%3DA.%2BG.%26aulast%3DStathis%26aufirst%3DM.%26aulast%3DDuvall%26aufirst%3DB.%26aulast%3DDelahanty%26aufirst%3DG.%26aulast%3DAlt%26aufirst%3DJ.%26aulast%3DRais%26aufirst%3DR.%26aulast%3DRojas%26aufirst%3DC.%26aulast%3DGao%26aufirst%3DP.%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DSlusher%26aufirst%3DB.%2BS.%26aulast%3DTsukamoto%26aufirst%3DT.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520pharmacological%2520evaluation%2520of%2520bis-2-%25285-phenylacetamido-1%252C2%252C4-thiadiazol-2-yl%2529ethyl%2520sulfide%25203%2520%2528BPTES%2529%2520analogs%2520as%2520glutaminase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D10551%26epage%3D10563%26doi%3D10.1021%2Fjm301191p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit14b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poore, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slusher, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukamoto, T.</span></span> <span> </span><span class="NLM_article-title">Allosteric glutaminase inhibitors based on a 1,4-di(5-amino-1,3,4-thiadiazol-2-yl)butane scaffold</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">520</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFejsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=520-524&author=S.+C.+Zimmermannauthor=E.+F.+Wolfauthor=A.+Luuauthor=A.+G.+Thomasauthor=M.+Stathisauthor=B.+Pooreauthor=C.+Nguyenauthor=A.+Leauthor=C.+Rojasauthor=B.+S.+Slusherauthor=T.+Tsukamoto&title=Allosteric+glutaminase+inhibitors+based+on+a+1%2C4-di%285-amino-1%2C3%2C4-thiadiazol-2-yl%29butane+scaffold&doi=10.1021%2Facsmedchemlett.6b00060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14bR"><div class="casContent"><span class="casTitleNuber">14b</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Glutaminase Inhibitors Based on a 1,4-Di(5-amino-1,3,4-thiadiazol-2-yl)butane Scaffold</span></div><div class="casAuthors">Zimmermann, Sarah C.; Wolf, Emily F.; Luu, Andrew; Thomas, Ajit G.; Stathis, Marigo; Poore, Brad; Nguyen, Christopher; Le, Anne; Rojas, Camilo; Slusher, Barbara S.; Tsukamoto, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">520-524</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of allosteric kidney-type glutaminase (GLS) inhibitors were designed and synthesized using 1,4-di(5-amino-1,3,4-thiadiazol-2-yl)butane as a core scaffold.  A variety of modified phenylacetyl groups were incorporated into the 5-amino group of the two thiadiazole rings in an attempt to facilitate addnl. binding interactions with the allosteric binding site of GLS.  Among the newly synthesized compds., 4-hydroxy-N-[5-[4-[5-[(2-phenylacetyl)amino]-1,3,4-thiadiazol-2-yl]butyl]-1,3,4-thiadiazol-2-yl]-benzeneacetamide, 2m, potently inhibited GLS with an IC50 value of 70 nM, although it did not exhibit time-dependency as seen with CB-839.  Antiproliferative effects of 2m on human breast cancer lines will be also presented in comparison with those obsd. with CB-839.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvSab9P2v_OLVg90H21EOLACvtfcHk0lhwiruHCytUIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFejsrY%253D&md5=a0e0dc6fcc0f4d343f94f02a50440adf</span></div><a href="/servlet/linkout?suffix=cit14b&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00060%26sid%3Dliteratum%253Aachs%26aulast%3DZimmermann%26aufirst%3DS.%2BC.%26aulast%3DWolf%26aufirst%3DE.%2BF.%26aulast%3DLuu%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DA.%2BG.%26aulast%3DStathis%26aufirst%3DM.%26aulast%3DPoore%26aufirst%3DB.%26aulast%3DNguyen%26aufirst%3DC.%26aulast%3DLe%26aufirst%3DA.%26aulast%3DRojas%26aufirst%3DC.%26aulast%3DSlusher%26aufirst%3DB.%2BS.%26aulast%3DTsukamoto%26aufirst%3DT.%26atitle%3DAllosteric%2520glutaminase%2520inhibitors%2520based%2520on%2520a%25201%252C4-di%25285-amino-1%252C3%252C4-thiadiazol-2-yl%2529butane%2520scaffold%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D520%26epage%3D524%26doi%3D10.1021%2Facsmedchemlett.6b00060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demo, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennison, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chernov-Rogan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goyal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janes, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laidig, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackinnon, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parlati, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shwonek, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sjogren, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanton, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, M. K.</span></span> <span> </span><span class="NLM_article-title">Antitumor activity of the glutaminase inhibitor CB-839 in triple-negative breast cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">890</span>– <span class="NLM_lpage">901</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-13-0870</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1158%2F1535-7163.MCT-13-0870" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=24523301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Gjurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=890-901&author=M.+I.+Grossauthor=S.+D.+Demoauthor=J.+B.+Dennisonauthor=L.+Chenauthor=T.+Chernov-Roganauthor=B.+Goyalauthor=J.+R.+Janesauthor=G.+J.+Laidigauthor=E.+R.+Lewisauthor=J.+Liauthor=A.+L.+Mackinnonauthor=F.+Parlatiauthor=M.+L.+Rodriguezauthor=P.+J.+Shwonekauthor=E.+B.+Sjogrenauthor=T.+F.+Stantonauthor=T.+Wangauthor=J.+Yangauthor=F.+Zhaoauthor=M.+K.+Bennett&title=Antitumor+activity+of+the+glutaminase+inhibitor+CB-839+in+triple-negative+breast+cancer&doi=10.1158%2F1535-7163.MCT-13-0870"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15aR"><div class="casContent"><span class="casTitleNuber">15a</span><div class="casTitle"><span class="NLM_cas:atitle">Antitumor Activity of the Glutaminase Inhibitor CB-839 in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Gross, Matt I.; Demo, Susan D.; Dennison, Jennifer B.; Chen, Lijing; Chernov-Rogan, Tania; Goyal, Bindu; Janes, Julie R.; Laidig, Guy J.; Lewis, Evan R.; Li, Jim; MacKinnon, Andrew L.; Parlati, Francesco; Rodriguez, Mirna L. M.; Shwonek, Peter J.; Sjogren, Eric B.; Stanton, Timothy F.; Wang, Taotao; Yang, Jinfu; Zhao, Frances; Bennett, Mark K.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">890-901</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Glutamine serves as an important source of energy and building blocks for many tumor cells.  The first step in glutamine utilization is its conversion to glutamate by the mitochondrial enzyme glutaminase.  CB-839 is a potent, selective, and orally bioavailable inhibitor of both splice variants of glutaminase (KGA and GAC).  CB-839 had antiproliferative activity in a triple-neg. breast cancer (TNBC) cell line, HCC-1806, that was assocd. with a marked decrease in glutamine consumption, glutamate prodn., oxygen consumption, and the steady-state levels of glutathione and several tricarboxylic acid cycle intermediates.  In contrast, no antiproliferative activity was obsd. in an estrogen receptor-pos. cell line, T47D, and only modest effects on glutamine consumption and downstream metabolites were obsd.  Across a panel of breast cancer cell lines, GAC protein expression and glutaminase activity were elevated in the majority of TNBC cell lines relative to receptor pos. cells.  Furthermore, the TNBC subtype displayed the greatest sensitivity to CB-839 treatment and this sensitivity was correlated with (i) dependence on extracellular glutamine for growth, (ii) intracellular glutamate and glutamine levels, and (iii) GAC (but not KGA) expression, a potential biomarker for sensitivity.  CB-839 displayed significant antitumor activity in two xenograft models: as a single agent in a patient-derived TNBC model and in a basal like HER2+ cell line model, JIMT-1, both as a single agent and in combination with paclitaxel.  Together, these data provide a strong rationale for the clin. investigation of CB-839 as a targeted therapeutic in patients with TNBC and other glutamine-dependent tumors.  Mol Cancer Ther; 13(4); 890-901.2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDfUGWU-VZObVg90H21EOLACvtfcHk0lg-eyccCISwPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Gjurg%253D&md5=5b9ddb35c56409b58ce2b889bbbbc33f</span></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-13-0870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-13-0870%26sid%3Dliteratum%253Aachs%26aulast%3DGross%26aufirst%3DM.%2BI.%26aulast%3DDemo%26aufirst%3DS.%2BD.%26aulast%3DDennison%26aufirst%3DJ.%2BB.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DChernov-Rogan%26aufirst%3DT.%26aulast%3DGoyal%26aufirst%3DB.%26aulast%3DJanes%26aufirst%3DJ.%2BR.%26aulast%3DLaidig%26aufirst%3DG.%2BJ.%26aulast%3DLewis%26aufirst%3DE.%2BR.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMackinnon%26aufirst%3DA.%2BL.%26aulast%3DParlati%26aufirst%3DF.%26aulast%3DRodriguez%26aufirst%3DM.%2BL.%26aulast%3DShwonek%26aufirst%3DP.%2BJ.%26aulast%3DSjogren%26aufirst%3DE.%2BB.%26aulast%3DStanton%26aufirst%3DT.%2BF.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DBennett%26aufirst%3DM.%2BK.%26atitle%3DAntitumor%2520activity%2520of%2520the%2520glutaminase%2520inhibitor%2520CB-839%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D890%26epage%3D901%26doi%3D10.1158%2F1535-7163.MCT-13-0870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit15b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Garber, K.</span></span> <span> </span><span class="NLM_article-title">Cancer anabolic metabolism inhibitors move into clinic</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">794</span>– <span class="NLM_lpage">795</span>, <span class="refDoi"> DOI: 10.1038/nbt0816-794</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1038%2Fnbt0816-794" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=27504761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlaitLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2016&pages=794-795&author=K.+Garber&title=Cancer+anabolic+metabolism+inhibitors+move+into+clinic&doi=10.1038%2Fnbt0816-794"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15bR"><div class="casContent"><span class="casTitleNuber">15b</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer anabolic metabolism inhibitors move into clinic</span></div><div class="casAuthors">Garber, Ken</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">794-795</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqORON4nNSW6LVg90H21EOLACvtfcHk0lg-eyccCISwPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlaitLfN&md5=c61077fafc878c60f67c7a563724d6fb</span></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1038%2Fnbt0816-794&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt0816-794%26sid%3Dliteratum%253Aachs%26aulast%3DGarber%26aufirst%3DK.%26atitle%3DCancer%2520anabolic%2520metabolism%2520inhibitors%2520move%2520into%2520clinic%26jtitle%3DNat.%2520Biotechnol.%26date%3D2016%26volume%3D34%26spage%3D794%26epage%3D795%26doi%3D10.1038%2Fnbt0816-794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.</span></span> <span> </span><span class="NLM_article-title">Assembly of <i>N</i>,<i>N</i>-disubstituted hydrazines and 1-aryl-1<i>H</i>-indazoles via copper-catalyzed coupling reactions</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2552</span>– <span class="NLM_lpage">2555</span>, <span class="refDoi"> DOI: 10.1021/ol300847v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol300847v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1Oqtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=2552-2555&author=X.+Xiongauthor=Y.+Jiangauthor=D.+Ma&title=Assembly+of+N%2CN-disubstituted+hydrazines+and+1-aryl-1H-indazoles+via+copper-catalyzed+coupling+reactions&doi=10.1021%2Fol300847v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Assembly of N,N-Disubstituted Hydrazines and 1-Aryl-1H-indazoles via Copper-Catalyzed Coupling Reactions</span></div><div class="casAuthors">Xiong, Xiaodong; Jiang, Yongwen; Ma, Dawei</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2552-2555</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">CuI-catalyzed coupling of N-acyl-N'-substituted hydrazines with aryl iodides takes place at 60-90 °C to afford N-acyl-N',N'-disubstituted hydrazines regioselectively and thereby gives a facile method for assembling N,N-diaryl hydrazines.  N-Acyl-N'-substituted hydrazines can also react with 2-bromoarylcarbonylic compds. at 60-125 °C under the catalysis of CuI/4-hydroxy-L-proline to provide 1-aryl-1H-indazoles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr54dWQR6YAILVg90H21EOLACvtfcHk0lg-eyccCISwPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1Oqtbg%253D&md5=cbca5c2ac9b7edf43ea5148a03bbebe4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fol300847v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol300847v%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DD.%26atitle%3DAssembly%2520of%2520N%252CN-disubstituted%2520hydrazines%2520and%25201-aryl-1H-indazoles%2520via%2520copper-catalyzed%2520coupling%2520reactions%26jtitle%3DOrg.%2520Lett.%26date%3D2012%26volume%3D14%26spage%3D2552%26epage%3D2555%26doi%3D10.1021%2Fol300847v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span>; <span class="NLM_string-name">Chen, L.</span>; <span class="NLM_string-name">Goyal, B.</span>; <span class="NLM_string-name">Laidig, G.</span>; <span class="NLM_string-name">Stanton, T. F.</span>; <span class="NLM_string-name">Sjogren, E. B.</span></span> <span> </span><span class="NLM_article-title">Heterocyclic Inhibitors of Glutaminase</span>. International Patent Application Number <span class="NLM_patent">WO2013/078123 A1</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=J.+Li&author=L.+Chen&author=B.+Goyal&author=G.+Laidig&author=T.+F.+Stanton&author=E.+B.+Sjogren&title=Heterocyclic+Inhibitors+of+Glutaminase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DHeterocyclic%2520Inhibitors%2520of%2520Glutaminase%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springthorpe, B.</span></span> <span> </span><span class="NLM_article-title">The influence of drug-like concepts on decision-making in medicinal chemistry</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">881</span>– <span class="NLM_lpage">890</span>, <span class="refDoi"> DOI: 10.1038/nrd2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1038%2Fnrd2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=17971784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=881-890&author=P.+D.+Leesonauthor=B.+Springthorpe&title=The+influence+of+drug-like+concepts+on+decision-making+in+medicinal+chemistry&doi=10.1038%2Fnrd2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18aR"><div class="casContent"><span class="casTitleNuber">18a</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of drug-like concepts on decision-making in medicinal chemistry</span></div><div class="casAuthors">Leeson, Paul D.; Springthorpe, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">881-890</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the wide acceptance of drug-like principles such as the 'rule of five', this anal. of mols. currently being synthesized in leading pharmaceutical companies reveals that their phys. properties differ significantly from those of recently discovered oral drugs.  The marked increase in lipophilicity in particular could increase the likelihood of attrition in drug development.  The application of guidelines linked to the concept of drug-likeness, such as the 'rule of five', has gained wide acceptance as an approach to reduce attrition in drug discovery and development.  However, despite this acceptance, anal. of recent trends reveals that the phys. properties of mols. that are currently being synthesized in leading drug discovery companies differ significantly from those of recently discovered oral drugs and compds. in clin. development.  The consequences of the marked increase in lipophilicity - the most important drug-like phys. property - include a greater likelihood of lack of selectivity and attrition in drug development.  Tackling the threat of compd.-related toxicol. attrition needs to move to the mainstream of medicinal chem. decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTUeonLD6iebVg90H21EOLACvtfcHk0ljRe1exUcJHBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL&md5=5665a9e2dc266a1ee096e20625757ef7</span></div><a href="/servlet/linkout?suffix=cit18a&amp;dbid=16384&amp;doi=10.1038%2Fnrd2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2445%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSpringthorpe%26aufirst%3DB.%26atitle%3DThe%2520influence%2520of%2520drug-like%2520concepts%2520on%2520decision-making%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D881%26epage%3D890%26doi%3D10.1038%2Fnrd2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit18b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand efficiency: A useful metric for lead selection</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">431</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+A+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18bR"><div class="casContent"><span class="casTitleNuber">18b</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eYIJHw63T9Eybntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit18b&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520A%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lafont, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amzel, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span> <span> </span><span class="NLM_article-title">Compensating enthalpic and entropic changes hinder binding affinity optimization</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2007.00519.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1111%2Fj.1747-0285.2007.00519.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=17581235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotV2js7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2007&pages=413-422&author=V.+Lafontauthor=A.+A.+Armstrongauthor=H.+Ohtakaauthor=Y.+Kisoauthor=L.+M.+Amzelauthor=E.+Freire&title=Compensating+enthalpic+and+entropic+changes+hinder+binding+affinity+optimization&doi=10.1111%2Fj.1747-0285.2007.00519.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Compensating enthalpic and entropic changes hinder binding affinity optimization</span></div><div class="casAuthors">Lafont, Virginie; Armstrong, Anthony A.; Ohtaka, Hiroyasu; Kiso, Yoshiaki; Amzel, L. Mario; Freire, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">413-422</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A common strategy to improve the potency of drug candidates is to introduce chem. functionalities, like hydrogen bond donors or acceptors, at positions where they are able to establish strong interactions with the target.  However, it is often obsd. that the added functionalities do not necessarily improve potency even if they form strong hydrogen bonds.  Here, we explore the thermodn. and structural basis for those observations.  KNI-10033 is a potent exptl. HIV-1 protease inhibitor with picomolar affinity against the wild-type enzyme (Kd = 13 pM).  The potency of the inhibitor is the result of favorable enthalpic (ΔH = -8.2 kcal/mol) and entropic (-TΔS = -6.7 kcal/mol) interactions.  The replacement of the thioether group in KNI-10033 by a sulfonyl group (KNI-10075) results in a strong hydrogen bond with the amide of Asp 30B of the HIV-1 protease.  This addnl. hydrogen bond improves the binding enthalpy by 3.9 kcal/mol; however, the enthalpy gain is completely compensated by an entropy loss, resulting in no affinity change.  Crystallog. and thermodn. anal. of the inhibitor/protease complexes indicates that the entropy losses are due to a combination of conformational and solvation effects.  These results provide a set of practical guidelines aimed at overcoming enthalpy/entropy compensation and improve binding potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMsDV9Pvk5Y7Vg90H21EOLACvtfcHk0lg0OJa3aTjViQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotV2js7o%253D&md5=163b3b46eba827d6a174284e8e3fb308</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00519.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00519.x%26sid%3Dliteratum%253Aachs%26aulast%3DLafont%26aufirst%3DV.%26aulast%3DArmstrong%26aufirst%3DA.%2BA.%26aulast%3DOhtaka%26aufirst%3DH.%26aulast%3DKiso%26aufirst%3DY.%26aulast%3DAmzel%26aufirst%3DL.%2BM.%26aulast%3DFreire%26aufirst%3DE.%26atitle%3DCompensating%2520enthalpic%2520and%2520entropic%2520changes%2520hinder%2520binding%2520affinity%2520optimization%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2007%26volume%3D69%26spage%3D413%26epage%3D422%26doi%3D10.1111%2Fj.1747-0285.2007.00519.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernetti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boby, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fioravanti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neill, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drakes, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katt, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerione, R.</span></span> <span> </span><span class="NLM_article-title">Design and evaluation of novel glutaminase inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1819</span>– <span class="NLM_lpage">1839</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.03.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.1016%2Fj.bmc.2016.03.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=26988803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC28Xkt1Wmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=1819-1839&author=L.+A.+McDermottauthor=P.+Lyerauthor=L.+Vernettiauthor=S.+Rimerauthor=J.+Sunauthor=M.+Bobyauthor=T.+Yangauthor=M.+Fioravantiauthor=J.+O%E2%80%99Neillauthor=L.+Wangauthor=D.+Drakesauthor=W.+Kattauthor=Q.+Huangauthor=R.+Cerione&title=Design+and+evaluation+of+novel+glutaminase+inhibitors&doi=10.1016%2Fj.bmc.2016.03.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Design and evaluation of novel glutaminase inhibitors</span></div><div class="casAuthors">McDermott, Lee A.; Iyer, Prema; Vernetti, Larry; Rimer, Shawn; Sun, Jingran; Boby, Melissa; Yang, Tianyi; Fioravanti, Michael; O'Neill, Jason; Wang, Liwei; Drakes, Dylan; Katt, William; Huang, Qingqiu; Cerione, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1819-1839</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel set of GAC (kidney glutaminase isoform C) inhibitors able to inhibit the enzymic activity of GAC and the growth of the triple neg. MDA-MB-231 breast cancer cells with low nanomolar potency is described.  Compds. in this series have a reduced no. of rotatable bonds, improved C log Ps, microsomal stability and ligand efficiency when compared to the leading GAC inhibitors BPTES and CB-839.  Property improvements were achieved by the replacement of the flexible n-diethylthio or the Bu moiety present in the leading inhibitors by heteroatom substituted heterocycloalkanes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGfzNc9NMuhbVg90H21EOLACvtfcHk0lg0OJa3aTjViQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xkt1Wmsbg%253D&md5=f335b5659f76e351b73c96954153127d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.03.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.03.009%26sid%3Dliteratum%253Aachs%26aulast%3DMcDermott%26aufirst%3DL.%2BA.%26aulast%3DLyer%26aufirst%3DP.%26aulast%3DVernetti%26aufirst%3DL.%26aulast%3DRimer%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DBoby%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DFioravanti%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DDrakes%26aufirst%3DD.%26aulast%3DKatt%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DQ.%26aulast%3DCerione%26aufirst%3DR.%26atitle%3DDesign%2520and%2520evaluation%2520of%2520novel%2520glutaminase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D1819%26epage%3D1839%26doi%3D10.1016%2Fj.bmc.2016.03.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
den Heuvel, A. P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wooster, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachman, K. E.</span></span> <span> </span><span class="NLM_article-title">Analysis of glutamine dependency in non-small cell lung cancer GLS1 splice variant GAC is essential for cancer cell growth</span>. <i>Cancer Biol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1185</span>– <span class="NLM_lpage">1194</span>, <span class="refDoi"> DOI: 10.4161/cbt.21348</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=10.4161%2Fcbt.21348" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=22892846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;key=1%3ACAS%3A528%3ADC%252BC3sXisF2nu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=1185-1194&author=A.+P.+J.+van%0Aden+Heuvelauthor=J.+Jingauthor=R.+F.+Woosterauthor=K.+E.+Bachman&title=Analysis+of+glutamine+dependency+in+non-small+cell+lung+cancer+GLS1+splice+variant+GAC+is+essential+for+cancer+cell+growth&doi=10.4161%2Fcbt.21348"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of glutamine dependency in non-small cell lung cancer: GLS1 splice variant GAC is essential for cancer cell growth</span></div><div class="casAuthors">van den Heuvel, A. Pieter J.; Jing, Junping; Wooster, Richard F.; Bachman, Kurtis E.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Biology & Therapy</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1185-1194</span>CODEN:
                <span class="NLM_cas:coden">CBTAAO</span>;
        ISSN:<span class="NLM_cas:issn">1538-4047</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">One of the hallmarks of cancer is metabolic deregulation.  Many tumors display increased glucose uptake and breakdown through the process of aerobic glycolysis, also known as the Warburg effect.  Less studied in cancer development and progression is the importance of the glutamine (Gln) pathway, which provides cells with a variety of essential products to sustain cell proliferation, such as ATP and macromols. for biosynthesis.  To this end Gln dependency was assessed in a panel of non-small cell lung cancer lines (NSCLC).  Gln was found to be essential for the growth of cells with high rates of glutaminolysis, and after exploring multiple genes in the Gln pathway, GLS1 was found to be the key enzyme assocd. with this dependence.  This dependence was confirmed by observing the rescue of decreased growth by exogenous addn. of downstream metabolites of glutaminolysis.  Expression of the GLS1 splice variant KGA was found to be decreased in tumors compared with normal lung tissue.  Transient knock down of GLS1 splice variants indicated that loss of GAC had the most detrimental effect on cancer cell growth.  In conclusion, NSCLC cell lines depend on Gln for glutaminolysis to a varying degree, in which the GLS1 splice variant GAC plays an essential role and is a potential target for cancer metab.-directed therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnWgR5NZWykLVg90H21EOLACvtfcHk0liiogC-8rDNSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXisF2nu78%253D&md5=521cd23a9f2a3a7f31c7024603b3a0ca</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.4161%2Fcbt.21348&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcbt.21348%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bden%2BHeuvel%26aufirst%3DA.%2BP.%2BJ.%26aulast%3DJing%26aufirst%3DJ.%26aulast%3DWooster%26aufirst%3DR.%2BF.%26aulast%3DBachman%26aufirst%3DK.%2BE.%26atitle%3DAnalysis%2520of%2520glutamine%2520dependency%2520in%2520non-small%2520cell%2520lung%2520cancer%2520GLS1%2520splice%2520variant%2520GAC%2520is%2520essential%2520for%2520cancer%2520cell%2520growth%26jtitle%3DCancer%2520Biol.%2520Ther.%26date%3D2012%26volume%3D13%26spage%3D1185%26epage%3D1194%26doi%3D10.4161%2Fcbt.21348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finlay, M. R. V.</span>; <span class="NLM_string-name">Ekwuru, C. T.</span>; <span class="NLM_string-name">Charles, M. D.</span>; <span class="NLM_string-name">Raubo, P. A.</span>; <span class="NLM_string-name">Winter, J. J. G.</span>; <span class="NLM_string-name">Nissink, J. W. M.</span></span> <span> </span><span class="NLM_article-title">1,3,4-Thiadiazole Compounds and Their Use in Treating Cancer</span>. International Patent Application Number <span class="NLM_patent">WO2015/181539 A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=M.+R.+V.+Finlay&author=C.+T.+Ekwuru&author=M.+D.+Charles&author=P.+A.+Raubo&author=J.+J.+G.+Winter&author=J.+W.+M.+Nissink&title=1%2C3%2C4-Thiadiazole+Compounds+and+Their+Use+in+Treating+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DFinlay%26aufirst%3DM.%2BR.%2BV.%26atitle%3D1%252C3%252C4-Thiadiazole%2520Compounds%2520and%2520Their%2520Use%2520in%2520Treating%2520Cancer%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGAA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGAA','PDB','3KGAA'); return false;">PDB: 3KGAA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KGAC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KGAC','PDB','1KGAC'); return false;">PDB: 1KGAC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6KGAB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6KGAB','PDB','6KGAB'); return false;">PDB: 6KGAB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KGAD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KGAD','PDB','7KGAD'); return false;">PDB: 7KGAD</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i81"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00260">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_01730"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00260">10.1021/acs.jmedchem.9b00260</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Enzyme/cell correlation plots (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00260/suppl_file/jm9b00260_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00260/suppl_file/jm9b00260_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00260/suppl_file/jm9b00260_si_001.pdf">jm9b00260_si_001.pdf (311.81 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00260/suppl_file/jm9b00260_si_002.csv">jm9b00260_si_002.csv (2.18 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last"><b>3</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGAA">3KGAA</a>), <b>12</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KGAC">1KGAC</a>), <b>18</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6KGAB">6KGAB</a>), and <b>27</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KGAD">7KGAD</a>). The authors will release the atomic coordinates upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00260&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-14%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00260%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00260" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679925fd69dad96e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
