
<html lang="en"     class="pb-page"  data-request-id="706a8e8b-a339-4818-865f-29cdceebc193"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.7b01220;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2018.61.issue-1;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="From the Promiscuous Asenapine to Potent Fluorescent Ligands Acting at a Series of Aminergic G-Protein-Coupled Receptors" /></meta><meta name="dc.Creator" content="Candide  Hounsou" /></meta><meta name="dc.Creator" content="Corinne  Baehr" /></meta><meta name="dc.Creator" content="Vincent  Gasparik" /></meta><meta name="dc.Creator" content="Doria  Alili" /></meta><meta name="dc.Creator" content="Abderazak  Belhocine" /></meta><meta name="dc.Creator" content="Thiéric  Rodriguez" /></meta><meta name="dc.Creator" content="Elodie  Dupuis" /></meta><meta name="dc.Creator" content="Thomas  Roux" /></meta><meta name="dc.Creator" content="André  Mann" /></meta><meta name="dc.Creator" content="Denis  Heissler" /></meta><meta name="dc.Creator" content="Jean-Philippe  Pin" /></meta><meta name="dc.Creator" content="Thierry  Durroux" /></meta><meta name="dc.Creator" content="Dominique  Bonnet" /></meta><meta name="dc.Creator" content="Marcel  Hibert" /></meta><meta name="dc.Description" content="Monoamine neurotransmitters such as serotonin, dopamine, histamine, and noradrenaline have important and varied physiological functions and similar chemical structures. Representing important pharm..." /></meta><meta name="Description" content="Monoamine neurotransmitters such as serotonin, dopamine, histamine, and noradrenaline have important and varied physiological functions and similar chemical structures. Representing important pharm..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 22, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01220" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01220" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01220" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01220" /></link>
        
    
    

<title>From the Promiscuous Asenapine to Potent Fluorescent Ligands Acting at a Series of Aminergic G-Protein-Coupled Receptors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01220" /></meta><meta property="og:title" content="From the Promiscuous Asenapine to Potent Fluorescent Ligands Acting at a Series of Aminergic G-Protein-Coupled Receptors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0008.jpeg" /></meta><meta property="og:description" content="Monoamine neurotransmitters such as serotonin, dopamine, histamine, and noradrenaline have important and varied physiological functions and similar chemical structures. Representing important pharmaceutical drug targets, the corresponding G-protein-coupled receptors (termed aminergic GPCRs) belong to the class of cell membrane receptors and share many levels of similarity as well. Given their pharmacological and structural closeness, one could hypothesize the possibility to derivatize a ubiquitous ligand to afford rapidly fluorescent probes for a large set of GPCRs to be used for instance in FRET-based binding assays. Here we report fluorescent derivatives of the nonselective agent asenapine which were designed, synthesized, and evaluated as ligands of 34 serotonin, dopamine, histamine, melatonin, acetylcholine, and adrenergic receptors. It appears that this strategy led rapidly to the discovery and development of nanomolar affinity fluorescent probes for 14 aminergic GPCRs. Selected probes were tested in competition binding assays with unlabeled competitors in order to demonstrate their suitability for drug discovery purposes." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01220"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01220">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01220&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01220&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01220&amp;href=/doi/10.1021/acs.jmedchem.7b01220" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 174-188</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01138" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b01245" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">From the Promiscuous Asenapine to Potent Fluorescent Ligands Acting at a Series of Aminergic G-Protein-Coupled Receptors</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Candide++Hounsou">Candide Hounsou</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Corinne++Baehr">Corinne Baehr</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Vincent++Gasparik">Vincent Gasparik</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Doria++Alili">Doria Alili</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Abderazak++Belhocine">Abderazak Belhocine</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Thi%C3%A9ric++Rodriguez">Thiéric Rodriguez</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elodie++Dupuis">Elodie Dupuis</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Thomas++Roux">Thomas Roux</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andr%C3%A9++Mann">André Mann</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Denis++Heissler">Denis Heissler</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jean-Philippe++Pin">Jean-Philippe Pin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Thierry++Durroux">Thierry Durroux</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dominique++Bonnet">Dominique Bonnet</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-8252-9199" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marcel++Hibert">Marcel Hibert</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-7786-7276" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Institut de Génomique Fonctionnelle, CNRS UMR5203, INSERM U661, Université de Montpellier (IFR3), 141 Rue de la Cardonille, F-34094 Montpellier Cedex 5, France</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Laboratoire d’Innovation Thérapeutique, Faculté de Pharmacie, UMR7200 CNRS, Université de Strasbourg, 74 Route du Rhin, 67412 Illkirch, France</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Cisbio Bioassays, Parc Marcel Boiteux, BP84175, 30200 Codolet, France</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">LabEx Medalis, Université de Strasbourg, 67000 Strasbourg, France</span></div><div class="corresp-info"><strong>*</strong>T.D.: e-mail, <a href="/cdn-cgi/l/email-protection#b6c2d2c3c4c4d9c3cef6dfd1d098d5d8c4c598d0c4"><span class="__cf_email__" data-cfemail="1d6979686f6f7268655d747a7b337e736f6e337b6f">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>M.H.: phone, +33 (0)3 68 85 42 32; e-mail, <a href="/cdn-cgi/l/email-protection#e28f8a8b80879096a2978c8b91969083cc8490"><span class="__cf_email__" data-cfemail="fd9095949f988f89bd8893948e898f9cd39b8f">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01220&amp;href=/doi/10.1021%2Facs.jmedchem.7b01220" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 174–188</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 8, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>18 August 2017</li><li><span class="item_label"><b>Published</b> online</span>22 December 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 January 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01220" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01220</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D174%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DCandide%2BHounsou%252C%2BCorinne%2BBaehr%252C%2BVincent%2BGasparik%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D1%26contentID%3Dacs.jmedchem.7b01220%26title%3DFrom%2Bthe%2BPromiscuous%2BAsenapine%2Bto%2BPotent%2BFluorescent%2BLigands%2BActing%2Bat%2Ba%2BSeries%2Bof%2BAminergic%2BG-Protein-Coupled%2BReceptors%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D188%26publicationDate%3DJanuary%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01220"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1145</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">9</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01220" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;From the Promiscuous Asenapine to Potent Fluorescent Ligands Acting at a Series of Aminergic G-Protein-Coupled Receptors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Candide&quot;,&quot;last_name&quot;:&quot;Hounsou&quot;},{&quot;first_name&quot;:&quot;Corinne&quot;,&quot;last_name&quot;:&quot;Baehr&quot;},{&quot;first_name&quot;:&quot;Vincent&quot;,&quot;last_name&quot;:&quot;Gasparik&quot;},{&quot;first_name&quot;:&quot;Doria&quot;,&quot;last_name&quot;:&quot;Alili&quot;},{&quot;first_name&quot;:&quot;Abderazak&quot;,&quot;last_name&quot;:&quot;Belhocine&quot;},{&quot;first_name&quot;:&quot;Thiéric&quot;,&quot;last_name&quot;:&quot;Rodriguez&quot;},{&quot;first_name&quot;:&quot;Elodie&quot;,&quot;last_name&quot;:&quot;Dupuis&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;Roux&quot;},{&quot;first_name&quot;:&quot;André&quot;,&quot;last_name&quot;:&quot;Mann&quot;},{&quot;first_name&quot;:&quot;Denis&quot;,&quot;last_name&quot;:&quot;Heissler&quot;},{&quot;first_name&quot;:&quot;Jean-Philippe&quot;,&quot;last_name&quot;:&quot;Pin&quot;},{&quot;first_name&quot;:&quot;Thierry&quot;,&quot;last_name&quot;:&quot;Durroux&quot;},{&quot;first_name&quot;:&quot;Dominique&quot;,&quot;last_name&quot;:&quot;Bonnet&quot;},{&quot;first_name&quot;:&quot;Marcel&quot;,&quot;last_name&quot;:&quot;Hibert&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;174-188&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01220&quot;},&quot;abstract&quot;:&quot;Monoamine neurotransmitters such as serotonin, dopamine, histamine, and noradrenaline have important and varied physiological functions and similar chemical structures. Representing important pharmaceutical drug targets, the corresponding G-protein-coupled receptors (termed aminergic GPCRs) belong to the class of cell membrane receptors and share many levels of similarity as well. Given their pharmacological and structural closeness, one could hypothesize the possibility to derivatize a ubiquitous ligand to afford rapidly fluorescent probes for a large set of GPCRs to be used for instance in FRET-based binding assays. Here we report fluorescent derivatives of the nonselective agent asenapine which were designed, synthesized, and evaluated as ligands of 34 serotonin, dopamine, histamine, melatonin, acetylcholine, and adrenergic receptors. It appears that this strategy led rapidly to the discovery and development of nanomolar affinity fluorescent probes for 14 aminergic GPCRs. Selected probes were tested in com&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01220&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01220" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01220&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01220" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01220&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01220" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01220&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01220&amp;href=/doi/10.1021/acs.jmedchem.7b01220" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01220" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01220" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01220%26sid%3Dliteratum%253Aachs%26pmid%3D29219316%26genre%3Darticle%26aulast%3DHounsou%26date%3D2018%26atitle%3DFrom%2Bthe%2BPromiscuous%2BAsenapine%2Bto%2BPotent%2BFluorescent%2BLigands%2BActing%2Bat%2Ba%2BSeries%2Bof%2BAminergic%2BG-Protein-Coupled%2BReceptors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D1%26spage%3D174%26epage%3D188%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291181" title="Amines">Amines</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291912" title="Fluorescence">Fluorescence</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/jmcmar.2018.61.issue-1/20180111/jmcmar.2018.61.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Monoamine neurotransmitters such as serotonin, dopamine, histamine, and noradrenaline have important and varied physiological functions and similar chemical structures. Representing important pharmaceutical drug targets, the corresponding G-protein-coupled receptors (termed aminergic GPCRs) belong to the class of cell membrane receptors and share many levels of similarity as well. Given their pharmacological and structural closeness, one could hypothesize the possibility to derivatize a ubiquitous ligand to afford rapidly fluorescent probes for a large set of GPCRs to be used for instance in FRET-based binding assays. Here we report fluorescent derivatives of the nonselective agent asenapine which were designed, synthesized, and evaluated as ligands of 34 serotonin, dopamine, histamine, melatonin, acetylcholine, and adrenergic receptors. It appears that this strategy led rapidly to the discovery and development of nanomolar affinity fluorescent probes for 14 aminergic GPCRs. Selected probes were tested in competition binding assays with unlabeled competitors in order to demonstrate their suitability for drug discovery purposes.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37237" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37237" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">With seven hydrophobic transmembrane (TM) α-helical segments as a signature feature, G-protein-coupled receptors (GPCRs) represent the largest class of membrane proteins in the human genome.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Because they are signaling transducers (transmission belts) involved in many physiological processes,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> GPCRs are associated with various human diseases,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> have many therapeutic applications, and represent targets of great importance for the pharmaceutical industry.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">As such, targets from the GPCR superfamily have generated around 30% of the currently marketed drugs.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Actually, most of these targets belong to the family of class A (or rhodopsin-like) GPCRs<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and more precisely to the subfamily of aminergic GPCRs which account for only 10% of the entire family.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_p">This pharmaceutical bias toward aminergic GPCRs lies on two main reasons. A historical one at first, since aminergic neurotransmitters were among first to be isolated and identified from biological fluids and tissues. Furthermore, the aminergic β<sub>2</sub>-adrenergic GPCR was the first GPCR ever sequenced after the atypical rhodopsine and many homologous aminergic receptors were then sequenced as well.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> As a consequence, aminergic GPCRs have been extensively investigated along with the development of the GPCR research field and drug discovery since the second half of the previous century. They are considered as the best understood family of GPCRs. For the multilevel similarity inside the subfamily, (a) the endogenous ligands are biogenic amines neurotransmitters and derivatives (acetylcholine, serotonin, dopamine, histamine, noradrenaline, melatonin, etc.) with strikingly close chemical structures; (b) in sequence alignment evaluations, orthosteric binding pockets keep similar ligand binding determinants (key ligand binding residues);<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> (c) tertiary structures are well conserved despite amino acid diversity as shown by early modeling studies.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10, 11)</a> Indeed, orthosteric binding pockets consist of cavities similarly located between extracellular loops and the upper side of the transmembrane region. The position is conserved, but there is diversity in its shape, its depth, and its amino acid composition.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Therefore, aminergic GPCRs display high levels of pharmacological similarity. As a consequence, aminergic GPCRs have also numerous small molecule ligands with polypharmacology and high chemical resemblance.</div><div class="NLM_p">Indeed, in the recent years, many high profile studies regarding class A GPCRs have emerged in the literature. Paolini et al. performed a mapping of pharmacological target space and illustrated the high target promiscuity of aminergic GPCRs.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Keiser et al. quantitatively grouped and related proteins based on the chemical similarity of their ligands.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Later, off-target effects were experimentally predicted and demonstrated for FDA-approved and investigational drugs.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Other similar publications illustrating cross-interactions of GPCR ligands are also available.<a onclick="showRef(event, 'ref16 ref17 ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18 ref19 ref20 ref21">(16-21)</a> In a functional signaling perspective, Wichard et al. identified amino acid positions governing antagonistic and agonistic effects for class A GPCRs using the mutual information analysis.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Using comparative sequence-based and ligand-based classifications for aminergic GPCRs, van der Horst et al. reached nearly identical images.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> This work was later confirmed by Lin et al. and even included non-GPCR targets.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> More recently, Kooistra et al. studied the molecular determinants of ligand binding to aminergic GPCRs with histamine receptors as reference receptors.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><div class="NLM_p">In this flourishing context, it is crystal clear that accurate pharmacological data and further, high quality pharmacological assays are critical for chemogenomics studies.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> As such, ligand binding assays represent an important frontline.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> In recent years, traditional radioligand binding assays have been slowly replaced with fluorescence-based ligand binding assays.<a onclick="showRef(event, 'ref27 ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29 ref30 ref31">(27-31)</a> On the basis of small-molecule (peptidic and nonpeptidic) fluorescent conjugates, a growing toolbox is taking shape, especially regarding class A GPCRs, which has been successfully applied in techniques such as flow cytometry, fluorescence correlation spectroscopy, fluorescence microscopy, fluorescence polarization, fluorescence resonance energy transfer, and scanning confocal microscopy.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Among these, time-resolved fluorescence resonance energy ransfer (TR-FRET) is particularly gaining momentum.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Rare-earth lanthanides with long emission half-lives as donor fluorophores in combination with suitable acceptor probes and time-resolved readouts have enabled significant improvements in signal-to-noise ratios, fastness, sensitivity, stability, reliability, robustness, and compatibility with high throughput screening (HTS) assays on GPCRs, in comparison with other fluorescence readouts.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> This approach based on homogeneous time-resolved fluorescence (separation is no longer required for bound and free fluorescent probes) is being strengthened by recent successes.<a onclick="showRef(event, 'ref29 ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref29 ref35 ref36 ref37">(29, 35-37)</a></div><div class="NLM_p">In order to investigate GPCR binding properties and oligomerization, fluorescent ligands have been reported for targets such as oxytocin and vasopressin receptors,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> the parathyroid hormone receptor,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> the growth hormone secretagogue receptor type 1a,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> the protease activated receptor 2,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> the human complement 5a receptor,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and chemokine receptors.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> However, the number and profile diversity of GPCR fluorescent ligands need to be extended.</div><div class="NLM_p last">As such, the development and the evaluation of FRET-based binding assays for a large number of GPCRs remain of great interest for understanding the fundamental mechanisms of action of GPCRs such as homo- and heterodimerization or allostery and for the drug discovery field. As for drug design and development, it also represents quite a difficult challenge if one attempts to develop hundreds of fluorescent ligands to target <i>specifically</i> their hundreds of GPCRs. However, we reasoned that in order to obtain a specific FRET-based binding assay, the fluorescent ligand does not need to be specific, the signal does, and the specificity of the signal will arise from the specific expression of the tagged, donor target receptor in the cell assay system. Therefore, the original concept we developed was to identify and tag a promiscuous GPCR ligand (frequent GPCR hitter) to attempt turning it into a limited series of fluorescent ligands binding to a large number of GPCRs and thus providing specific time-resolved FRET-based binding assays and molecular probes to investigate ligand kinetic parameters for applications in basic and applied research.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78735" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78735" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Design</h3><div class="NLM_p">Intending to double-down on their multilevel similarity, we hypothesized that, as similar receptors bind similar ligands, nonselective “universal” fluorescent probes should be possible to find for aminergic GPCRs. Asenapine, a psychopharmacologic agent currently in clinical use for the treatment of acute schizophrenia and bipolar disorders (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>), was suitable in many aspects.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Indeed asenapine has subnanomolar and nanomolar affinities (<i>K</i><sub>i</sub> ≈ 5 nM) for diverse and numerous subtypes of aminergic GPCRs (serotonin, norepinephrine, dopamine, and histamine receptors).<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> In contrast, it was described with micromolar affinities for muscarinic acetylcholine receptors and melatonin receptors in addition to no detectable affinity for histamine H<sub>3</sub> and H<sub>4</sub> receptors which could all serve as negative controls. In addition, asenapine proved to have clear antagonistic properties in functional assays for all its targets, a stable behavior highly appreciable for a platform. Asenapine displayed the classical pharmacophore of biogenic amine receptors: a protonable nitrogen, likely to make a salt bridge with the conserved aspartate residue on TM 3, and aromatic rings at a 5–6 Å distance, probably embedded against TMs 5 and 6. Such ligands can usually be branched on the nitrogen atom with different substituents in retaining most of the time some level of biological activity. However, a minimum length of approximately 15 atoms is usually required from the linker to maintain the hindering fluorophore at the mouth of the binding cleft in order to prevent any detrimental interaction with the target receptor.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> For example, as shown in previous works,<a onclick="showRef(event, 'ref28 ref29 ref30 ref31 ref35 ref36 ref37 ref38 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30 ref31 ref35 ref36 ref37 ref38 ref39 ref40">(28-31, 35-40)</a> fluorescent groups can be grafted on linkers of varied lengths and chemical nature. Herein we decided to remove the <i>N</i>-methyl group of asenapine (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) and to replace it with a linker carrying the fluorophore (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of Norasenapine <b>17</b> from Asenapinium Maleate <b>14</b> (Racemate)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (i) PL-TBD resin, CH<sub>2</sub>Cl<sub>2</sub>/MeOH, overnight, rt; (ii) ClCO<sub>2</sub>CHClCH<sub>3</sub>, 1,2-dichloroethane, 100 °C, 2 h; (iii) CH<sub>3</sub>OH, 100 °C, 2 h. Global yield (3 steps): 79%</p></p></figure><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. General strategy for the labeling of asenapine and affinity (<i>K</i><sub>i</sub> in nM) of the parent compound for a set of GPCRs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Four different linkers and two fluorophores were combined. Linkers with different length and hydrophobicity included a 3-atom aminoethyl spacer (“short linker”, SL), a 16-atom polymethylenic monoamide spacer (“long linker”, LL), a corresponding 15-atom pegylated spacer (“pegylated linker”, PL), and a 16-atom polymethylenic diamide spacer (“very long linker”, VLL; <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). These linkers were connected to norasenapine (<a class="ref internalNav" href="#sch3" aria-label="Schemes 3">Schemes 3</a> and <a class="ref internalNav" href="#sch4" aria-label="4">4</a>) and coupled via an amide bond to two structurally different fluorophores that were already known to be compatible with time-resolved FRET: the cyanine derivative 2-((1<i>E</i>,3<i>E</i>)-5-((<i>Z</i>)-3-(4-((2,5-dioxopyrrolidin-1-yl)oxy)-4-oxobutyl)-1-ethyl-3-methyl-5-sulfoindolin-2-ylidene)penta-1,3-dien-1-yl)-1-ethyl-3,3-dimethyl-3<i>H</i>-indol-1-ium-5-sulfonate (DY647; D) and the fluorescein derivative, 6-(fluorescein-5-carboxamido)hexanoic acid (5-SFX; S, <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Fluorescein itself (FLUO; F) was combined with the diamido spacer only. Nine fluorescent derivatives of asenapine have thus been prepared and evaluated in TR-FRET ligand binding assays regarding serotonin, dopamine, histamine, melatonin, acetylcholine, and adrenergic receptors.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Linkers <b>4</b>, <b>7</b>, and <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (i) TsCl, KI, Ag<sub>2</sub>O, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C, 15 min, 85%; (ii) NaN<sub>3</sub>, MeCN, reflux, 12 h, 91%; (iii) MsCl, NEt<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, rt, 2 h, 77%; (iv) HOCH<sub>2</sub>CHOHCH<sub>2</sub>NH<sub>2</sub>, PyBOP, <i>i</i>-Pr<sub>2</sub>EtN, DMF/CH<sub>2</sub>Cl<sub>2</sub>, rt, 12 h, 93%; (v) NaIO<sub>4</sub>, THF/H<sub>2</sub>O, rt, 1 h, 100%; (vi) <i>i</i>-Pr<sub>2</sub>EtN, DMF, rt, 4 h, 82%; (vii) aqueous LiOH, THF, rt, 12 h, 94%.</p></p></figure><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Amines <b>19</b>, <b>21</b>, <b>23</b>, and <b>25</b> from Norasenapine <b>17</b> (Racemates)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (i) BocNHCH<sub>2</sub>-CHO, NaBH<sub>3</sub>CN, MeOH, rt, 2 h, 96%; (ii) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 2 h, 65–98%; (iii) <b>7</b>, NaBH<sub>3</sub>CN, MeOH, rt, 2 h, 79%; (iv) <b>4</b>, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, rt, 12 h, then 50 °C, sonication, 2 h, 47%; (v) PS-PPh<sub>3</sub>, THF/H<sub>2</sub>O, rt, 12 h, 95%, (vi) <b>13</b>, NaBH<sub>3</sub>CN, MeOH, rt, 2 h, 64%.</p></p></figure><figure id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Fluorescent Compounds <b>26</b>–<b>34</b> (Racemates) for TR-FRET Applications<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (i) dye succinimidyl ester, <i>i</i>-Pr<sub>2</sub>EtN, DMSO, rt, 4 h, 30–40%.</p></p></figure></div><div id="sec2-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Chemistry</h3><div id="sec2-2-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> Norasenapine</h4><div class="NLM_p last">Supply of norasenapine <b>17</b> was secured from commercially available asenapinium maleate <b>14</b> (racemate) in 71% overall yield. Treatment of <b>14</b> with polymer linked 1,5,7-triazabicyclo[4.4.0]dec-5-ene (PL-TBD) led to free asenapine <b>15</b> which could be converted into norasenapine <b>17</b> by reaction with 1-chloroethyl chloroformate in 1,2-dichloroethane followed by treatment with methanol according to Olofson and Senet (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></div></div><div id="sec2-2-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> Linkers</h4><div class="NLM_p">Preparation of mesylate <b>4</b> started by selective monotosylation of diol <b>1</b> with tosyl chloride, silver(I) oxide, and potassium iodide.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> It was completed by reaction of tosylate <b>2</b> with sodium azide to give <b>3</b> and by mesylation to provide compound <b>4</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>; 60% overall yield).</div><div class="NLM_p last">The synthesis of aldehyde <b>7</b> was first attempted by treatment of commercially available acid <b>5</b> with allylamine followed by oxidation with RuCl<sub>3</sub>–NaIO<sub>4</sub>. However, the reaction was sluggish and <b>7</b> was only obtained in moderate yield. As an alternative, acid <b>5</b> was converted into amide <b>6</b> with 3-aminopropane-1,2-diol under benzotriazol-1-yloxytripyrrolidinophosphonium hexafluorophosphate (PyBOP) activation and then into <b>7</b> by oxidative cleavage with sodium periodate in 93% overall yield (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). The same reagents also allowed the preparation of aldehyde <b>13</b> from acid <b>11</b>. The latter compound was obtained by reaction of methyl 5-aminopentanoate <b>8</b> with protected <i>N</i>-succinimidyl 6-aminohexanoate <b>9</b> followed by saponification of the methyl ester <b>10</b>. The yield for the conversion of <b>8</b> into <b>13</b> was 71% (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div></div><div id="sec2-2-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Fluorescent Asenapine Derivatives</h4><div class="NLM_p last">The nine ligands <b>26</b>–<b>34</b> have been obtained by connecting norasenapine <b>17</b> to the dyes DY467, 5-SFX, or fluorescein with proper linkers (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). To this end, we have prepared amines <b>19</b>, <b>21</b>, and <b>25</b> from <i>N</i>-Boc-2-aminoacetaldehyde, aldehyde <b>7</b>, and aldehyde <b>13</b>, respectively, by reductive amination with norasenapine <b>17</b> followed by amine deprotection with trifluoroacetic acid. We have also prepared amine <b>23</b> by reaction of norasenapine <b>17</b> with mesylate <b>4</b> followed by a Staudinger reduction of azide <b>22</b> with polymer supported triphenylphosphine (PS-PPh<sub>3</sub>) in THF–water. Reaction of amines <b>19</b>, <b>21</b>, <b>23</b>, and <b>25</b> with the <i>N</i>-succinimidyl esters of DY647 and 5-SFX led to the ligands <b>26</b>–<b>33</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). Ligand <b>34</b> was prepared in the same way by labeling amine <b>25</b> with the activated ester of fluorescein. The fluorescence properties of the ligands <b>26</b>–<b>34</b> are reported in the experimental part and in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b01220/suppl_file/jm7b01220_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div></div><div id="sec2-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Saturation Binding Experiments</h3><div class="NLM_p">TR-FRET receptor binding assays were performed to evaluate the binding affinities of the nine fluorescent probes <b>26</b>–<b>34</b> on 12 serotonin, 5 dopamine, 4 histamine, 5 acetylcholine, and 2 melatonin receptor subtypes. As already described,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> the transient expression of the target proteins as SNAP-tagged constructs at the surface of HEK293T cells was achieved. Reactions with SNAP substrates resulted in SNAP-tagged constructs wearing time-resolved FRET donors of fluorescence (Tb cryptate). Hence, saturation experiments were performed with increasing concentrations of probes <b>26</b>–<b>34</b>. Nonspecific signals were obtained with excess of unlabeled compound (10 μM asenapine), and specific signals were then plotted as functions of fluorescent probes concentrations in order to give dissociation constants (<i>K</i><sub>d</sub>). All results are presented below with a threshold for <i>K</i><sub>d</sub> values at 1 μM (see <a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a>–<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>) and illustrated with compound <b>31</b> in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>. All results are also presented with the corresponding signal-to-noise (<i>S</i>/<i>N</i>) ratios calculated in the binding assay as the ratio of the total signal on the nonspecific signal at the <i>K</i><sub>d</sub> value. The threshold for the <i>S</i>/<i>N</i> ratios was set at 2 meaning that specific signal should amount to one-half of the total signal. The results from all ligand binding assays are summarized in <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a> where the number of hit aminergic GPCRs is indicated for each fluorescent ligand. The binding profile of the different probes is illustrated in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Determination of the Binding Affinities of Fluorescent Probes for Serotonin Receptors Using TR-FRET Binding Assays<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="2" align="center"> </th><th class="rowsep1 colsep0" colspan="24" align="center">ligand binding TR-FRET assay</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="24" align="center">serotonin receptors</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">5-HT<sub>1a</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">5-HT<sub>1b</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">5-HT<sub>1d</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">5-HT<sub>1e</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">5-HT<sub>1f</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">5-HT<sub>2a</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">5-HT<sub>2b</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">5-HT<sub>2c</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">5-HT<sub>4</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">5-HT<sub>5a</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">5-HT<sub>6</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">5-HT<sub>7</sub></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">code<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">SL-S</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i><a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">169 ± 67</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">656 ± 157</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">20 ± 4</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">49 ± 28</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">SL-D</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i><i>n</i></i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">LL-S</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">45 ± 6</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">174 ± 36</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">999 ± 349</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">LL-D</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">63 ± 37</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">50 ± 3</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">40 ± 6</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">36 ± 21</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">122 ± 55</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">47 ± 4</td><td class="colsep0 rowsep0" align="left">3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">PL-S</td><td class="colsep0 rowsep0" align="left">31 ± 7</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">50 ± 15</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">88 ± 10</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">27 ± 16</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">134 ± 24</td><td class="colsep0 rowsep0" align="left">37</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">666 ± 309</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">PL-D</td><td class="colsep0 rowsep0" align="left">32 ± 8</td><td class="colsep0 rowsep0" align="left">9.5</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">3.5</td><td class="colsep0 rowsep0" align="left">17 ± 3</td><td class="colsep0 rowsep0" align="left">9.5</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">17 ± 7</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">8 ± 3</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">53 ± 16</td><td class="colsep0 rowsep0" align="left">65</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">36 ± 9</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">VLL-S</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">121 ± 45</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">119 ± 44</td><td class="colsep0 rowsep0" align="left">5.5</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">VLL-D</td><td class="colsep0 rowsep0" align="left">3 ± 2</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">106 ± 40</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">53 ± 20</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">93 ± 44</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">55 ± 35</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">VLL-F</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">104 ± 38</td><td class="colsep0 rowsep0" align="left">3.5</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Values represent the mean ± SD from at least three independent experiments.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last"><i>K</i><sub>d</sub> values in nM units.</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last"><i>S</i>/<i>N</i> represents the signal-to-noise ratio calculated in the binding assay as the ratio total signal/nonspecific signal at the <i>K</i><sub>d</sub> value. <i>S</i>/<i>N</i> was calculated at 1000 nM for <i>K</i><sub>d</sub> values over 1000 nM.</p></div><div class="footnote" id="t1fn4"><sup>d</sup><p class="last"><i>n</i> indicates S/<i>N</i> < 2.</p></div><div class="footnote" id="t1fn5"><sup>e</sup><p class="last">SL-S = small linker-SFX; SL-D = small linker-DY647; LL-S = long linker-SFX; LL-D = long linker-DY647; PL-S = pegylated linker-SFX; PL-D = pegylated linker-DY647; VLL-S = very long linker-SFX; VLL-D = very long linker-DY647; VLL-F = very long linker-fluorescein.</p></div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Determination of the Binding Affinities of Fluorescent Probes for Dopamine and Histamine Receptors Using TR-FRET Binding Assays<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="18" align="center">ligand binding TR-FRET assay</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="10" align="center">dopamine receptors</th><th class="rowsep1 colsep0" colspan="8" align="center">histamine receptors</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">D<sub>1</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">D<sub>2</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">D<sub>3</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">D<sub>4</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">D<sub>5</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">H<sub>1</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">H<sub>2</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">H<sub>3</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">H<sub>4</sub></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">code<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i><a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">SL-S</td><td class="colsep0 rowsep0" align="left">60 ± 3</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">6 ± 1</td><td class="colsep0 rowsep0" align="left">4.5</td><td class="colsep0 rowsep0" align="left">20 ± 3</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">40 ± 4</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i><a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">42 ± 3</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">395 ± 23</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">SL-D</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">105 ± 21</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">LL-S</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">3.5</td><td class="colsep0 rowsep0" align="left">146 ± 35</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">255 ± 60</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">122 ± 17</td><td class="colsep0 rowsep0" align="left">23</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">LL-D</td><td class="colsep0 rowsep0" align="left">892 ± 383</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">76 ± 23</td><td class="colsep0 rowsep0" align="left">11.5</td><td class="colsep0 rowsep0" align="left">201 ± 58</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">204 ± 51</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">104 ± 19</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">PL-S</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">3.5</td><td class="colsep0 rowsep0" align="left">46 ± 6</td><td class="colsep0 rowsep0" align="left">62</td><td class="colsep0 rowsep0" align="left">73 ± 19</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">560 ± 184</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">82 ± 18</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">PL-D</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">13 ± 2</td><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="left">24 ± 5</td><td class="colsep0 rowsep0" align="left">125</td><td class="colsep0 rowsep0" align="left">56 ± 12</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">36 ± 6</td><td class="colsep0 rowsep0" align="left">29</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">VLL-S</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">65 ± 29</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">316 ± 114</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">310 ± 122</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">VLL-D</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">42 ± 13</td><td class="colsep0 rowsep0" align="left">4.5</td><td class="colsep0 rowsep0" align="left">15 ± 5</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">20 ± 7.5</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">VLL-F</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">105 ± 62</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">118 ± 23</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Values represent the mean ± SD from at least three independent experiments.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last"><i>K</i><sub>d</sub> values in nM units.</p></div><div class="footnote" id="t2fn3"><sup>c</sup><p class="last"><i>S</i>/<i>N</i> represents the signal-to-noise ratio calculated in the binding assay as the ratio total signal/nonspecific signal at the <i>K</i><sub>d</sub> value. <i>S</i>/<i>N</i> was calculated at 1000 nM for <i>K</i><sub>d</sub> values over 1000 nM.</p></div><div class="footnote" id="t2fn4"><sup>d</sup><p class="last"><i>n</i> indicates S/<i>N</i> < 2.</p></div><div class="footnote" id="t2fn5"><sup>e</sup><p class="last">SL-S = small linker-DSX; SL-D = small linker-DY647; LL-S = long linker-SFX; LL-D = long linker-DY647; PL-S = pegylated linker-SFX; PL-D = pegylated linker-DY647; VLL-S = very long linker-SFX; VLL-D = very long linker-DY647; VLL-F = very long linker-fluorescein.</p></div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Determination of the Binding Affinities of Fluorescent Probes for Adrenergic Receptors Using TR-FRET Binding Assays<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="12" align="center">ligand binding TR-FRET assay</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="12" align="center">adrenergic receptors</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">α<sub>1A</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">α<sub>1B</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">α<sub>1D</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">α<sub>2A</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">α<sub>2B</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">α<sub>2C</sub></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">code<a class="ref internalNav" href="#t3fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i><a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">SL-S</td><td class="colsep0 rowsep0" align="left">308 ± 148</td><td class="colsep0 rowsep0" align="left">4,5</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i><a class="ref internalNav" href="#t3fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">396 ± 118</td><td class="colsep0 rowsep0" align="left">7,5</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">SL-D</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">LL-S</td><td class="colsep0 rowsep0" align="left">174 ± 29</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">567 ± 180</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">LL-D</td><td class="colsep0 rowsep0" align="left">121 ± 51</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">70 ± 35</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">160 ± 76</td><td class="colsep0 rowsep0" align="left">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">PL-S</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">223 ± 60</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">770 ± 218</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">PL-D</td><td class="colsep0 rowsep0" align="left">247 ± 44</td><td class="colsep0 rowsep0" align="left">8.5</td><td class="colsep0 rowsep0" align="left">2.4 ± 1</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">127 ± 41</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">VLL-S</td><td class="colsep0 rowsep0" align="left">131 ± 44</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">432 ± 191</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">VLL-D</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">72 ± 30</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">VLL-F</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">180 ± 65</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Values represent the mean ± SD from at least three independent experiments.</p></div><div class="footnote" id="t3fn2"><sup>b</sup><p class="last"><i>K</i><sub>d</sub> values in nM units.</p></div><div class="footnote" id="t3fn3"><sup>c</sup><p class="last"><i>S</i>/<i>N</i> represents the signal-to-noise ratio calculated in the binding assay as the ratio total signal/nonspecific signal at the <i>K</i><sub>d</sub> value. <i>S</i>/<i>N</i> was calculated at 1000 nM for <i>K</i><sub>d</sub> values over 1000 nM.</p></div><div class="footnote" id="t3fn4"><sup>d</sup><p class="last"><i>n</i> indicates S/<i>N</i> < 2.</p></div><div class="footnote" id="t3fn5"><sup>e</sup><p class="last">SL-S = small linker-SFX; SL-D = small linker-DY647; LL-S = long linker-SFX; LL-D = long linker-DY647; PL-S = pegylated linker-SFX; PL-D = pegylated linker-DY647; VLL-S = very long linker-SFX; VLL-D = very long linker-DY647; VLL-F = very long linker-fluorescein.</p></div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Determination of the Binding Affinities of Fluorescent Probes for Acetylcholine and Melatonin Receptors Using TR-FRET Binding Assays<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="14" align="center">ligand binding TR-FRET assay</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="10" align="center">acetylcholine receptors</th><th class="rowsep1 colsep0" colspan="4" align="center">melatonin receptors</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">M<sub>1</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">M<sub>2</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">M<sub>3</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">M<sub>4</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">M<sub>5</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">MTN<sub>1A</sub></th><th class="rowsep1 colsep0" colspan="2" align="center">MTN<sub>1B</sub></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">code<a class="ref internalNav" href="#t4fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i><a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center"><i>S</i>/<i>N</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">SL-S</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i><a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">SL-D</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">LL-S</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">LL-D</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">PL-S</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">PL-D</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">VLL-S</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">VLL-D</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">VLL-F</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left"><i>n</i></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">Values represent the mean ± SD from at least three independent experiments.</p></div><div class="footnote" id="t4fn2"><sup>b</sup><p class="last"><i>K</i><sub>d</sub> values in nM units.</p></div><div class="footnote" id="t4fn3"><sup>c</sup><p class="last"><i>S</i>/<i>N</i> represents the signal-to-noise ratio calculated in the binding assay as the ratio total signal/nonspecific signal at the <i>K</i><sub>d</sub> value. <i>S</i>/<i>N</i> was calculated at 1000 nM for <i>K</i><sub>d</sub> values over 1000 nM.</p></div><div class="footnote" id="t4fn4"><sup>d</sup><p class="last"><i>n</i> indicates S/<i>N</i> < 2.</p></div><div class="footnote" id="t4fn5"><sup>e</sup><p class="last">SL-S = small linker-SFX; SL-D = small linker-DY647; LL-S = long linker-SFX; LL-D = long linker-DY647; PL-S = pegylated linker-SFX; PL-D = pegylated linker-DY647; VLL-S = very long linker-SFX; VLL-D = very long linker-DY647; VLL-F = very long linker-fluorescein.</p></div></div></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Number of Receptors Bound by the Different Fluorescent Probes at Submicromolar Concentrations</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="6" align="center">number of hit aminergic GPCRs<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">code<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">serotonin receptors</th><th class="colsep0 rowsep0" align="center">dopamine receptors</th><th class="colsep0 rowsep0" align="center">histamine receptors</th><th class="colsep0 rowsep0" align="center">adrenergic receptors</th><th class="colsep0 rowsep0" align="center">acetylcholine receptors</th><th class="colsep0 rowsep0" align="center">melatonin receptors</th><th class="colsep0 rowsep0" align="center">total score<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">SL-S</td><td class="colsep0 rowsep0" align="left">4/12</td><td class="colsep0 rowsep0" align="left">4/5</td><td class="colsep0 rowsep0" align="left">2/4</td><td class="colsep0 rowsep0" align="left">2/6</td><td class="colsep0 rowsep0" align="left">0/5</td><td class="colsep0 rowsep0" align="left">0/2</td><td class="colsep0 rowsep0" align="left">12/34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">SL-D</td><td class="colsep0 rowsep0" align="left">0/12</td><td class="colsep0 rowsep0" align="left">0/5</td><td class="colsep0 rowsep0" align="left">1/4</td><td class="colsep0 rowsep0" align="left">0/6</td><td class="colsep0 rowsep0" align="left">0/5</td><td class="colsep0 rowsep0" align="left">0/2</td><td class="colsep0 rowsep0" align="left">1/34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">LL-S</td><td class="colsep0 rowsep0" align="left">3/12</td><td class="colsep0 rowsep0" align="left">2/5</td><td class="colsep0 rowsep0" align="left">1/4</td><td class="colsep0 rowsep0" align="left">2/6</td><td class="colsep0 rowsep0" align="left">0/5</td><td class="colsep0 rowsep0" align="left">0/2</td><td class="colsep0 rowsep0" align="left">8/34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">LL-D</td><td class="colsep0 rowsep0" align="left">6/12</td><td class="colsep0 rowsep0" align="left">4/5</td><td class="colsep0 rowsep0" align="left">1/4</td><td class="colsep0 rowsep0" align="left">3/6</td><td class="colsep0 rowsep0" align="left">0/5</td><td class="colsep0 rowsep0" align="left">0/2</td><td class="colsep0 rowsep0" align="left">14/34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">PL-S</td><td class="colsep0 rowsep0" align="left">6/12</td><td class="colsep0 rowsep0" align="left">3/5</td><td class="colsep0 rowsep0" align="left">1/4</td><td class="colsep0 rowsep0" align="left">2/6</td><td class="colsep0 rowsep0" align="left">0/5</td><td class="colsep0 rowsep0" align="left">0/2</td><td class="colsep0 rowsep0" align="left">12/34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">PL-D</td><td class="colsep0 rowsep0" align="left">6/12</td><td class="colsep0 rowsep0" align="left">3/5</td><td class="colsep0 rowsep0" align="left">1/4</td><td class="colsep0 rowsep0" align="left">3/6</td><td class="colsep0 rowsep0" align="left">0/5</td><td class="colsep0 rowsep0" align="left">0/2</td><td class="colsep0 rowsep0" align="left">13/34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">VLL-S</td><td class="colsep0 rowsep0" align="left">2/12</td><td class="colsep0 rowsep0" align="left">1/5</td><td class="colsep0 rowsep0" align="left">2/4</td><td class="colsep0 rowsep0" align="left">2/6</td><td class="colsep0 rowsep0" align="left">0/5</td><td class="colsep0 rowsep0" align="left">0/2</td><td class="colsep0 rowsep0" align="left">7/34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">VLL-D</td><td class="colsep0 rowsep0" align="left">5/12</td><td class="colsep0 rowsep0" align="left">2/5</td><td class="colsep0 rowsep0" align="left">1/4</td><td class="colsep0 rowsep0" align="left">1/6</td><td class="colsep0 rowsep0" align="left">0/5</td><td class="colsep0 rowsep0" align="left">0/2</td><td class="colsep0 rowsep0" align="left">9/34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">VLL-F</td><td class="colsep0 rowsep0" align="left">1/12</td><td class="colsep0 rowsep0" align="left">1/5</td><td class="colsep0 rowsep0" align="left">1/4</td><td class="colsep0 rowsep0" align="left">1/6</td><td class="colsep0 rowsep0" align="left">0/5</td><td class="colsep0 rowsep0" align="left">0/2</td><td class="colsep0 rowsep0" align="left">4/34</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">Hit aminergic GPCRs represent GPCRs for which a fluorescent ligand shows a <i>K</i><sub>d</sub> value under 1000 nM.</p></div><div class="footnote" id="t5fn2"><sup>b</sup><p class="last">Total score represents the addition of all hit aminergic GPCRs found during the ligand binding study for the designated fluorescent ligand.</p></div><div class="footnote" id="t5fn3"><sup>c</sup><p class="last">SL-S = small linker-SFX; SL-D = small linker-DY647; LL-S = long linker-SFX; LL-D = long linker-DY647; PL-S = pegylated linker-SFX; PL-D = pegylated linker-DY647; VLL-S = very long linker-SFX; VLL-D = very long linker-DY647; VLL-F = very long linker-fluorescein.</p></div></div></div><figure id="fig2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding experiments with <b>31</b>: (A) binding saturation of <b>31</b> on dopamine D<sub>3a</sub> receptor; (B–D) specific binding of <b>31</b> on various receptors belonging to the dopamine and the histamine (B), the serotonin (C), and the adrenergic (D) receptor families; (E) competition binding on 5-HT<sub>2b</sub> receptor. <b>31</b> was used at 17 nM. Competitors were added at a concentration ranging from 10<sup>–12</sup> M to 10<sup>–4</sup> M. All curves result from pooled data obtained in at least three independent experiments. Data were fitted with Prism using the one binding site subroutine in the “binding-competitive” procedure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Web of affinity. Schematic representation of the affinity profiles (<i>K</i><sub>d</sub> in nM) of the different fluorescent probes for a set of 34 class A GPCRs. Compounds <b>26</b>–<b>34</b> ligands are also identified in a coded manner as follows: SL-S = small linker-SFX; SL-D = small linker-DY647; LL-S = long linker-SFX; LL-D = long linker-DY647; PL-S = pegylated linker-SFX; PL-D = pegylated linker-DY647; VLL-S = very long linker-SFX; VLL-D = very long linker-DY647; VLL-F = very long linker-fluorescein.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Overall, as expected, it appears that the fluorescent tracers bound the same subtypes of aminergic GPCRs as the parent asenapine with nanomolar range affinities (see <a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a>–<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>).</div><div id="sec2-3-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Results with Serotonin Receptors (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>)</h4><div class="NLM_p">On 5HT<sub>1a</sub> receptors, compound <b>33</b> shows an affinity (<i>K</i><sub>d</sub> = 3 nM) comparable to asenapine (<i>K</i><sub>d</sub> = 2.5 nM) while compounds <b>30</b> and <b>31</b> display a 10-fold lower affinity. All other asenapine derivatives are inactive suggesting that the combination of the polymethylenic diamide spacer and DY647 fluorophore is ideal and unique in providing a high affinity fluorescent probe for the 5HT<sub>1a</sub> receptor. The pegylated spacer is also accepted by the receptor, the fluorophore making no difference in this case (see compounds <b>30</b> and <b>31</b>; <i>K</i><sub>d</sub> = 31 nM and <i>K</i><sub>d</sub> = 32 nM, respectively).</div><div class="NLM_p">On the 5HT<sub>1b</sub> receptor, the results are quite different. The 5-SFX short chain compound <b>26</b> shows some significant affinity (<i>K</i><sub>d</sub> = 169 nM), while the DY647 analogue is inactive. The reverse is observed for the long chain polymethylenic monoamide derivatives <b>28</b> and <b>29</b>: the DY647 compound (<b>29</b>; <i>K</i><sub>d</sub> = 63 nM) is the most potent fluorescent ligand of this receptor, whereas its 5-SFX analogue <b>28</b> is inactive. Interestingly, the pegylated homologues <b>30</b> and <b>31</b> are inactive while an intermediate affinity is recovered in the diamide linker series (<b>32</b>, <b>33</b>; <i>K</i><sub>d</sub> = 121 nM and 106 nM, respectively) with a much less pronounced discriminating effect of these two fluorophores and a detrimental effect of fluorescein (<b>34</b>; <i>K</i><sub>d</sub> > 1000 nM).</div><div class="NLM_p">On the 5HT<sub>1d</sub> receptor, the short spacer derivatives <b>26</b> and <b>27</b> are inactive. Asenapine derivatives with longer linkers show a significant affinity ranging from 17 nM (compound <b>31</b>) to the 50 nM range (<b>28</b>, <b>29</b>, <b>30</b>, <b>33)</b>. The choice of the fluorophore is discriminant again but only in the diamide series since <b>32</b> and <b>34</b> are completely inactive.</div><div class="NLM_p">On the 5HT<sub>1e</sub> and 5HT<sub>1f</sub> receptors, no significantly active fluorescent probe was identified with the exception of compound <b>26</b> which displays affinity in the low micromolar range for 5HT<sub>1f</sub> (<i>K</i><sub>d</sub> = 656 nM). To our knowledge, no affinity data for asenapine have been reported on these receptors.</div><div class="NLM_p">On the 5HT<sub>2a</sub> receptor, affinity is restricted to probes with the polymethylenic monoamide spacer and a preference for the DY467 fluorophore (<b>30</b>, <i>K</i><sub>d</sub> = 88 nM; <b>31</b>, <i>K</i><sub>d</sub> = 17 nM).</div><div class="NLM_p">On the 5HT<sub>2b</sub>, the activity profile of the series of probes is again different. The short linker series is inactive. In the long linker series, only the DY647 derivative <b>29</b> shows affinity (<i>K</i><sub>d</sub> = 40 nM) in the monoamide series, while all compounds of the diamide series (<b>32</b>–<b>34</b>) are inactive. Both molecules carrying the pegylated linker provide high affinity probes with a preference toward the DY647 fluorophore since compound <b>31</b> alone reaches an 8 nM affinity.</div><div class="NLM_p">Binding to the 5HT<sub>2c</sub> receptor seems less stringent. All probes bind at submicromolar concentrations with the exception of the short-chain DY647 derivative <b>27</b> which is inactive while its 5-SFX analogue <b>26</b> is very potent with a 20 nM affinity. Thus, nanomolar fluorescent probes could be identified for the 5HT<sub>2a-c</sub> receptors but with a 1–2 log decrease in affinity compared to the parent compound, asenapine.</div><div class="NLM_p last">Interestingly, none of the probes display affinity for the 5HT<sub>4</sub> and 5HT<sub>5a</sub> receptors. In contrast, for the 5HT<sub>6</sub> receptor, probes <b>26</b>, <b>29</b>, and <b>33</b> show a reasonably high affinity with <i>K</i><sub>d</sub> = 49 nM, 122 nM, and 55 nM, respectively, while all other probes are inactive. Again, the 5-SFX fluorophore was preferred on short chain derivative <b>26</b> whereas DY647 provides more active compounds on the polymethylenic long chain derivatives <b>29</b> and <b>33</b>. The pegylated spacer is unfavorable to activity on this receptor (<b>30</b>, <b>31</b>). In contrast, on the 5HT<sub>7</sub> receptor subtype, the pegylated compound <b>31</b> is the most potent (<i>K</i><sub>d</sub> = 36 nM) while compound <b>29</b> was the only other active one with an affinity in the same range (<i>K</i><sub>d</sub> = 47 nM).</div></div><div id="sec2-3-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Results with Dopamine Receptors (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>)</h4><div class="NLM_p">The affinity of the nine probes was determined on the five dopamine receptor subtypes. On dopamine D<sub>1</sub> receptor, only compound <b>26</b> carrying the short spacer and the 5-SFX fluorophore displays a significant affinity for the receptor (<i>K</i><sub>d</sub> = 60 nM). The same compound has a noticeable 6 nM affinity for the D<sub>2</sub> receptor. If one excludes its DY647 analogue <b>27</b>, all other probes are active in the 13–146 nM range with a predominant efficacy of the DY647 analogues <b>31</b> and <b>33</b>. The contribution to efficacy of the two fluorophores is opposite in the short chain and the long chain series.</div><div class="NLM_p">On the D<sub>3</sub> receptor, a similar pattern is observed. The most active probes are again <b>26</b>, <b>31</b>, and <b>33</b> with <i>K</i><sub>d</sub> = 20 nM, 24 nM, and 15 nM, respectively. The difference in affinity compared to homologues with other fluorophores is more significant than on D<sub>2</sub> receptor subtype.</div><div class="NLM_p last">On the D<sub>4</sub> receptor, two compounds with different spacer lengths and fluorophores, namely, <b>26</b> and <b>31</b>, are the only ones showing a significant affinity, with <i>K</i><sub>d</sub> = 40 nM and 56 nM, respectively. Finally, none of the nine probes display affinity for the D<sub>5</sub> receptor subtype.</div></div><div id="sec2-3-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Results with Histamine Receptors (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>)</h4><div class="NLM_p last">Results on the histamine receptors are clear-cut. All probes show a good affinity for the H<sub>1</sub> receptor subtype with <i>K</i><sub>d</sub> ranging from 20 nM to 316 nM. As for several other receptors, the 5-SFX fluorophore is the most favorable on the short spacer while the DY647 is preferred on the long spacer. In deep contrast, all probes are weakly active or inactive on the H<sub>2</sub>, H<sub>3</sub>, and H<sub>4</sub> receptor subtypes.</div></div><div id="sec2-3-4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Results with the Adrenergic Receptors (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>)</h4><div class="NLM_p last">On the α<sub>1a</sub> receptor, five probes show a significant affinity with <i>K</i><sub>d</sub> ranging from 121 to 308 nM. However, the interpretation of the results is quite challenging: the 5-SFX probe <b>26</b> is active in the short spacer series (<i>K</i><sub>d</sub> = 308 nM), while its DY647 analogue <b>27</b> is inactive; both fluorophores provide activity in the polymethylenic monoamide series (compounds <b>28</b> and <b>29</b>; <i>K</i><sub>d</sub> = 174 nM and <i>K</i><sub>d</sub> = 121 nM, respectively); the DY647 derivative <b>31</b> only is active in the pegylated series (<i>K</i><sub>d</sub> = 247 nM); and only the 5-SFX compound <b>32</b> is active in the polymethylenic diamide series (<i>K</i><sub>d</sub> = 131 nM). On the α<sub>1b</sub> receptor, the situation is more clear-cut: two DY647 derivatives with long spacers of different nature, namely, <b>29</b> and <b>31</b>, show high potency (<i>K</i><sub>d</sub> = 70 nM and <i>K</i><sub>d</sub> = 2.4 nM, respectively). None of the nine probes bind to the α<sub>1d</sub> and α<sub>2a</sub> receptor subtypes. Seven probes present some affinity for the α<sub>2b</sub> subtype, the best one reaching a 72 nM affinity (compound <b>33</b>). Noteworthy, this receptor is the only one in the series accepting a fluorescein tagged ligand with a significant affinity (compound <b>34</b>; <i>K</i><sub>d</sub> = 180 nM). Finally, the α<sub>2c</sub> receptor subtype binds to only one probe (<b>29</b>; <i>K</i><sub>d</sub> = 160 nM) with a long polymethylenic monoamide spacer and the DY647 fluorophore. To our knowledge, the affinity of the parent compound, asenapine, for these adrenergic receptors had never been published before. High to moderate affinity fluorescent probes derived from asenapine could be identified for four adrenergic receptor subtypes out of the six studied here.</div></div><div id="sec2-3-5" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Results with the Muscarinic and Melatonin Receptors (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>)</h4><div class="NLM_p last">Asenapine has no affinity for muscarinic receptors.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> None of the nine fluorescent probes bind to any of the five muscarinic receptor subtypes. None of them bind to the MTN<sub>1A</sub> or MTN<sub>1B</sub> melatonin receptors.</div></div></div><div id="sec2-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Competition Binding Experiments</h3><div class="NLM_p">As proof of concept, four probes were used to develop novel TR-FRET binding assays for five different receptors, namely, 5HT<sub>2b</sub>, 5HT<sub>1d</sub>, 5HT<sub>2c</sub>, 5HT<sub>6</sub>, and D<sub>4</sub> receptors. As already described,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> the target proteins were transiently expressed as SNAP-tagged constructs at the surface of HEK293T cells. Second, reactions with SNAP fluorescent substrates resulted in SNAP-tagged constructs wearing TR-FRET donors of fluorescence (Tb cryptate). The fluorescent probes <b>26</b>, <b>29</b>, <b>30</b>, and <b>31</b> were added at fixed concentration at their <i>K</i><sub>d</sub> values. Increasing concentrations of reference competitors were then used to afford inhibition constants that were compared to literature values measured in classical radioligand assays.<a onclick="showRef(event, 'ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55">(48-55)</a> Results are listed in <a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a> and exemplified in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a> for compound <b>31</b>. There is generally a good agreement between both data sets with however a significant shift observed in the particular case of ligand <b>26</b> at the D<sub>4</sub> dopamine receptor, when roxindole was used as competitor, and at the 5HT<sub>6</sub> receptor. It is noteworthy that all the competitions with the reference ligands have led to a total displacement of the tracer and all data can easily be fitted when considering a slope of 1. This suggests the absence of positive or negative binding of the reference ligands when using the fluorescent ligands as tracers.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Inhibition Constants Determined by Competition Assay with Fluorescent Probes as Tracers and Reference Compounds as Competitors</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" colspan="8" align="center">Competition Binding, p<i>K</i><sub>i</sub> (nM)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="8" align="center">5HT<sub>2b</sub> receptor</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">5-HT</th><th class="rowsep1 colsep0" colspan="2" align="center">methysergide</th><th class="rowsep1 colsep0" colspan="2" align="center">5-CT</th><th class="rowsep1 colsep0" colspan="2" align="center">asenapine</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">probe</th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48, 49)</a></th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">9.07 ± 0.05</td><td class="colsep0 rowsep0" align="left">7.87 ± 0.04</td><td class="colsep0 rowsep0" align="left">9.41 ± 0.09</td><td class="colsep0 rowsep0" align="left">8.2 ± ?<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></td><td class="colsep0 rowsep0" align="left">7.92 ± 0.18</td><td class="colsep0 rowsep0" align="left">6.73 ± 0.09</td><td class="colsep0 rowsep0" align="left">9.73 ± 0.14</td><td class="colsep0 rowsep0" align="left">9.75 ± 0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">8.81 ± 0.04</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">9.04 ± 0.11</td><td class="colsep0 rowsep0" align="left">9.44 ± 0.5<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></td><td class="colsep0 rowsep0" align="left">7.62 ± 0.12</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">9.4 ± 0.04</td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="8" align="center">5HT<sub>1d</sub> receptor</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">donitriptan</th><th class="rowsep1 colsep0" colspan="2" align="center">methysergide</th><th class="rowsep1 colsep0" colspan="2" align="center">5-HT</th><th class="rowsep1 colsep0" colspan="2" align="center">asenapine</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">probe</th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51, 52)</a></th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">9.46 ± 0.22</td><td class="colsep0 rowsep0" align="left">9.32 ± 0.09</td><td class="colsep0 rowsep0" align="left">8.50 ± 0.11</td><td class="colsep0 rowsep0" align="left">8.08 ± 0.10</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">8.23 ± 0.08</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">9.23 ± 0.01</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">8.25 ± 0.12</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">8.17 ± 0.06</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">8.08 ± 0.12</td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">5-HT<sub>2c</sub> receptor</th><th class="rowsep1 colsep0" colspan="4" align="center">5-HT<sub>6</sub> receptor</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">methylergonovine</th><th class="rowsep1 colsep0" colspan="2" align="center">methysergide</th><th class="rowsep1 colsep0" colspan="2" align="center">lisuride</th><th class="rowsep1 colsep0" colspan="2" align="center">SB 271046</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">probe</th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">7.82 ± 0.05</td><td class="colsep0 rowsep0" align="left">8.34 ± 0.06</td><td class="colsep0 rowsep0" align="left">8.60 ± 0.12</td><td class="colsep0 rowsep0" align="left">8.60 ± 0.05</td><td class="colsep0 rowsep0" align="left">7.71 ± 0.11</td><td class="colsep0 rowsep0" align="left">10.9 ± 1.8</td><td class="colsep0 rowsep0" align="left">8.06 ± 0.22</td><td class="colsep0 rowsep0" align="left">8.92 ± ?</td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="8" align="center">D<sub>4</sub> receptor</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">bromocriptine</th><th class="rowsep1 colsep0" colspan="2" align="center">quinpirole</th><th class="rowsep1 colsep0" colspan="2" align="center">roxindole</th><th class="rowsep1 colsep0" colspan="2" align="center">asenapine</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">probe</th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a><sup>,</sup><a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a><sup>,</sup><a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a><sup>,</sup><a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">found</th><th class="colsep0 rowsep0" align="center">lit.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">6.18 ± 0.12</td><td class="colsep0 rowsep0" align="left">6.43 ± 0.11</td><td class="colsep0 rowsep0" align="left">6.41 ± 0.04</td><td class="colsep0 rowsep0" align="left">7.47 ± 0.05</td><td class="colsep0 rowsep0" align="left">6.13 ± 0.09</td><td class="colsep0 rowsep0" align="left">8.23 ± 0.06</td><td class="colsep0 rowsep0" align="left">8.06 ± 0.28</td><td class="colsep0 rowsep0" align="left">8.95 ± 0.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">6.54 ± 0.16</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">7.25 ± 0.22</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">6.87 ± 0.10</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">9.15 ± 0.11</td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup>a</sup><p class="last">Values represent the mean ± SD from at least three independent experiments. Fluorescent ligands were used at the following concentrations: <b>30</b> and <b>31</b> were respectively used at 70 nM and 10 nM on 5HT<sub>2b</sub> receptor; <b>29</b> and <b>31</b>were respectively used at 4 nM and 17 nM on 5HT<sub>1d</sub> receptor; <b>26</b> was used at 10 nM on 5HT<sub>2c</sub>, 5HT<sub>6</sub>, and D<sub>4</sub> receptors; <b>31</b> was used at 78 nM on D<sub>4</sub> receptor.</p></div><div class="footnote" id="t6fn2"><sup>b</sup><p class="last">h4.4. isoform of the D<sub>4</sub> receptor. Lit.: p<i>K</i><sub>i</sub> reported in the literature.<a onclick="showRef(event, 'ref45 ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref45 ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55">(45, 48-55)</a> nd = not done.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55247" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55247" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Most fluorescent probes derived from asenapine retained the fluorescence properties of the dye building blocks. They showed high binding affinities for one or several GPCRs. In many cases, the large structural modification introduced on the amino group of asenapine in replacement of the methyl group remarkably allowed retaining of most affinity values. Hence, moderate to high affinity fluorescent ligands were discovered for the 5HT<sub>1a</sub>, 5HT<sub>1b</sub>, 5HT<sub>1d</sub>, 5HT<sub>2a</sub>, 5HT<sub>2b</sub>, 5HT<sub>2c</sub>, 5HT<sub>6</sub>, 5HT<sub>7</sub> serotonin receptors (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>); D<sub>1</sub>, D<sub>2</sub>, D<sub>3</sub>, and D<sub>4</sub> dopamine receptors (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>); H<sub>1</sub> and H<sub>2</sub> histamine receptors (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>); α<sub>1A</sub>, α<sub>1B</sub>, α<sub>2B</sub>, and α<sub>2C</sub> adrenergic receptors (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). Since asenapine had no affinity for histamine H<sub>3</sub> and H<sub>4</sub>, muscarinic acetylcholine, and melatonin receptors, it is not surprising to find that asenapine-derived fluorescent probes have no significant affinity for this subset of GPCRs (<i>K</i><sub>d</sub> values of >1 μM). Further, no tracers were found for receptors with no published affinity data for asenapine (5HT<sub>1e</sub>, 5HT<sub>1f</sub>, 5HT<sub>4</sub>, 5HT<sub>5a</sub>, D<sub>5</sub>, α<sub>1D</sub>, and α<sub>2A</sub>) as their <i>K</i><sub>d</sub> values were all over 600 nM. These results illustrate how relevant were the choice of asenapine as platform and the choice of the tagging position on asenapine since fluorescent derivatives, altogether, reproduced the binding profile of the initial chemical platform. Noteworthy, the fluorescence properties of the different ligands are similar to those of the dyes (see <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.7b01220/suppl_file/jm7b01220_si_001.pdf" class="ext-link">Supporting Information</a>) and were not significantly affected by the shorter and longer linkers.</div><div class="NLM_p">From the ligand point of view, compound <b>26</b> is excellent (<i>K</i><sub>d</sub> < 100 nM) for all dopamine receptors except the D<sub>5</sub> subtype. It also shows good affinity for the H<sub>1</sub> histamine (<i>K</i><sub>d</sub> = 42 nM), 5HT<sub>2C</sub> (<i>K</i><sub>d</sub> = 20 nM), and 5HT<sub>6</sub> (<i>K</i><sub>d</sub> = 49 nM) serotonin receptors but marginal or no affinity for other GPCRs. Interestingly, tracer <b>27</b> is very specific since it does not bind any of the 33 GPCR tested except the H<sub>1</sub>R (<i>K</i><sub>d</sub> = 105 nM). Compound <b>34</b> is also peculiar in that it binds significantly only one 5-HT receptor subtype out of 12 (<i>K</i><sub>d</sub> = 104 nM, 5-HT<sub>2C</sub> receptor), one dopamine receptor subtype out of five (<i>K</i><sub>d</sub> = 105 nM, D<sub>2</sub> receptor), one histamine receptor subtype out of four (<i>K</i><sub>d</sub> = 118 nM, H<sub>1</sub> receptor), and one adrenaline receptor subtype out of six (<i>K</i><sub>d</sub> = 180 nM, α<sub>2B</sub> receptor). All other compounds are more ubiquitous, hitting 7–14 receptors out of 27 if one exclude muscarinic and melatonin receptors for which no ligands were found.</div><div class="NLM_p">Few unexpected results are obtained. It is noteworthy that (a) tracers with nanomolar affinities (2.4–70 nM) are found for 5HT<sub>1d</sub> and α<sub>1B</sub>, two receptors with no published affinity data for asenapine, and (b) no tracers are found for GPCRs such as 5-HT<sub>5A</sub> and α<sub>2A</sub> with reported high affinities for asenapine.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></div><div class="NLM_p">The length of the linkers and the nature of the linkers and of the fluorophores have clearly a strong impact on the affinity and selectivity of the probes. However, it appears very difficult to rationalize these effects. The comparison of probes <b>26</b> (12 GPCR hits) and <b>27</b> (1 GPCR hit) indicates that the 5-SFX dye performs better than the DY-647 dye when short linkers are used (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). However, longer linkers as in <b>29</b>, <b>31</b>, and <b>32</b> provide broad spectrum, potent fluorescent probes as well. Pegylated spacers seem to be slightly more favorable than polymethylene spacers but not dramatically and not systematically (see <b>28</b>, <b>30</b>, <b>32</b>). Finally, the nature of the fluorophore is clearly important to retain affinity. On short linkers as in <b>26</b> and <b>27</b>, the 5-SFX fluorescein moiety is clearly preferred to the cyanine fluorophore of DY647. However, the reverse is observed on longer chains such as in <b>28</b> and <b>29</b> while there is no clear difference between <b>30</b> and <b>31</b> or <b>32</b> and <b>33</b>. Thus, the choice of the fluorophore and of the spacer makes a huge impact on the ability of the ligands to interact with the receptors. One may hypothesize that their asenapine fragment binds in the same orthosteric pocket as asenapine itself while the fluorophore may interact with the loops at the entrance of the binding cleft, in allosteric sites more or less well structured, as already proposed.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> This would probably affect several properties of the ligands such as binding kinetics and functional efficacy. This may explain the shift in competition assays as seen with compound <b>26</b> at the D<sub>4</sub> and 5HT<sub>6</sub> receptors (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>). In addition, the impact on functionality of the derivatization of asenapine with linkers and fluorophores has not yet been studied. Though unlikely, one may not exclude shifts from antagonism to partial agonism or inverse agonism. Such modifications would not alter the interest of the new probes for competition binding experiments in search for receptor ligands, but it should be studied in detail if one want to use them for instance for receptor trafficking or receptor binding kinetics studies. Considering the subtlety of our results, the flexibility of the linkers, and the limited knowledge of the receptor dynamics in the extracellular loops regions, we considered that the rationalization of our binding data by docking studies could only be achieved by an extensive modeling study coupled to experimental validation by cocrystallization, site directed mutagenesis, kinetic and functional studies, or other approaches. This work will be undertaken and reported elsewhere.</div><div class="NLM_p last">In terms of FRET efficacy, <i>S</i>/<i>N</i> ratios significantly vary according to the fluorescent ligand/receptor pairs. These ratios are dependent (i) on the nature of the fluorophore acceptor (fluorescein vs DY647), the fluorophore donor (the terbium cryptate) being always the same, and (ii) on the distance between the two fluorophores when the fluorescent ligand is bound to the receptor. It is noteworthy that in time-resolved FRET, the FRET efficacy is weakly dependent on the relative orientation of the fluorophores. The maximum of <i>S</i>/<i>N</i> equal to 125 has been obtained for probe <b>24</b> on dopamine D<sub>3</sub> receptor. <i>S</i>/<i>N</i> ratios greater than 3 are often considered as relevant for time-resolved FRET experiments. Here, in the saturation or competition binding experiments, the assays are particularly sensitive and we observed that a <i>S</i>/<i>N</i> ratio of 2 is relevant to get reliable data. This has been firmly established in using some of them in competition assays with reference compounds for a subset of studied receptors (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>).</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07335" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07335" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Asenapine proved to be indeed a very relevant platform to produce high affinity fluorescent probes for a large set of GPCRs. Starting from this single broad spectrum, nanomolar ligand of a large set of GPCRs, we were able to design a small series of fluorescent probes that hit at least one target GPCR. Some of them hit up to 14 receptors out of 34 tested, whereas other probes happened to be rather specific of one of them. Without surprise, the interaction between such probes and a GPCR is very subtle. As known, adding a “linker” or a fluorescent “tag” on a ligand must not be considered as neutral or trivial. The length of the linker, its chemical and structural properties, the structure and properties of the fluorophore clearly affect the binding process of the probe in a way that will be difficult to rationalize and to anticipate as indicated by our results.</div><div class="NLM_p last">Although no “universal” fluorescent probe has been identified, our concept of derivatizing a broad spectrum, high affinity GPCR ligand was quite efficient since a series of 9 probes tagging 14 different class A GPCRs have been rapidly produced. Some of them may represent specific imaging tools (e.g., compound <b>27</b> for H1 receptors), and all of them are likely to give access to specific time-resolved FRET binding assays with applications in basic research<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> or in drug development.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48537" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48537" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5-1-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Chemistry. General Methods</h3><div class="NLM_p">Reagents were obtained from commercial sources and used without further purification. DY647-NHS, 5-SDX-NHS, and fluorescein-NHS were purchased from Dyomics GmbH (Jena, Germany). Titrisol buffer was purchased from Merck. Thin-layer chromatography (TLC) was performed on silica gel 60 F254 plates. <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded on a Bruker (500 MHz/125 MHz and 400 MHz/100 MHz) spectrometer. Conditions are specified for each spectrum (temperature 25 °C unless specified). Chemical shifts are reported in parts per million (ppm) relative to residual solvent, and coupling constants (<i>J</i>) are reported in hertz (Hz). Signals are described as s (singlet), d (doublet), t (triplet), q (quartet), m (multiplet), and bs (broad singlet). Deuterated solvents were purchased from Euriso-top.</div><div class="NLM_p last">Melting points were determined on a Büchi melting point B-540 apparatus in open capillary tubes. Semipreparative reverse-phase high performance liquid chromatography (RP-HPLC) separations were performed on a Waters C18 SunFire Prep OBD column (5 μm, 19 mm × 150 mm) using a linear gradient (from 5% to 100% of solvent B in solvent A over 25 min; solvent A of water/0.1% trifluoroacetic acid (TFA); solvent B of acetonitrile/0.1% TFA; flow rate of 20 mL·min<sup>–1</sup>; detection at 220 and 254 nm). Analytical RP-HPLC was performed on final products on a Supelco C18 Ascentis Express column (2.7 μm, 4.6 mm × 75 mm) using a linear gradient (from 5% to 100% of solvent B in solvent A over 7.5 min; solvent A of water/0.1% TFA; solvent B of acetonitrile/0.1% TFA; flow rate of 1.6 mL.min-1; detection at 220 and 254 nm). Purified compounds eluted as single and symmetrical peaks (thereby confirming a purity higher than 95%) at retention times (<i>t</i><sub>R</sub>) given below. Their molecular masses was determined by high resolution mass spectrometry (HRMS) on an Agilent Technology 6520 QTof LC/MS mass spectrometer. To establish excitation and emsission spectra, DMSO stock solutions of ligands have been diluted in water and fluorescence properties were measured on a Fluoromax 4 spectrometer (Horiba France, Les Ulis, France).</div></div><div id="sec5-1-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> <i>tert</i>-Butyl 11-(2,3-Dihydroxypropylamino)-11-oxoundecylcarbamate (<b>6</b>)</h3><div class="NLM_p last">PyBOP (828 mg, 1.59 mmol) was added to a stirred solution of 11-<b>(</b><i>tert-</i>butoxycarbonylamino)undecanoic acid <b>5</b> (400 mg, 1.32 mmol) in DMF/CH<sub>2</sub>Cl<sub>2</sub> (1/2, v/v, 30 mL). After 5 min at room temperature, 3-amino-1,2-propanediol (145 mg, 1.59 mmol) and <i>i</i>-Pr<sub>2</sub>EtN (438 μL, 2.65 mmol) were added and the mixture was stirred overnight at room temperature. It was then concentrated in vacuo, dissolved in EtOAc, and washed sequentially with aqueous 2 M NaHSO<sub>4</sub>, saturated aqueous NaHCO<sub>3</sub>, and aqueous NaCl (3 times). After drying over Na<sub>2</sub>SO<sub>4</sub>, the solution was concentrated under reduced pressure and purified by chromatography on a silica gel column (CH<sub>2</sub>Cl<sub>2</sub>/MeOH, 10/0 to 9/1, v/v) to provide compound <b>6</b> as a white solid (456 mg, 92%); <i>R</i><sub><i>f</i></sub> = 0.21 (CH<sub>2</sub>Cl<sub>2</sub>/MeOH 96/4); mp 82–83 °C. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 6.59 (bs, 1 H), 4.61 (bs, 1 H), 3.76–3.72 (m, 3 H), 4.50 (d, <i>J</i> = 4.9 Hz, 2 H), 3.39–3.30 (m, 2 H), 3.12–3.02 (m, 3 H), 2.18 (t, <i>J</i> = 7.4 Hz, 2 H), 1.81–1.78 (m, 1 H), 1.57 (m, 2 H), 1.43–1.39 (m, 2 H), 1.40 (s, 9 H), 1.22 (m, 10 H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz): δ 175.3, 156.1, 79.1, 78.8, 71.1, 63.6, 46.5, 46.4, 42.2, 40.6, 36.5, 30.0, 29.3, 29.3, 29.2, 29.7, 26.4, 26.4, 25.7. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>38</sub>N<sub>2</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 374.2780; found, 374.2773.</div></div><div id="sec5-1-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> <i>tert</i>-Butyl 11-Oxo-11-(2-oxoethylamino)undecylcarbamate (<b>7</b>)</h3><div class="NLM_p last">NaIO<sub>4</sub> (126 mg, 0.592 mmol) was added to a solution of <b>6</b> (148 mg, 0.395 mmol) in THF/water (1/1, v/v, 7 mL). The reaction was stirred for 1 h at room temperature, concentrated in vacuo, dissolved in CH<sub>2</sub>Cl<sub>2</sub>, washed with water, and dried over Na<sub>2</sub>SO<sub>4</sub>. Concentration of the solution under reduced pressure gave <b>7</b> as a pale yellow oil (135 mg, 100%) which was used without further purification; <i>R</i><sub><i>f</i></sub> = 0.32 (CH<sub>2</sub>Cl<sub>2</sub>/MeOH 9/1, v/v). <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 9.60 (m, 1 H), 6.45 (bs, 1 H), 4.55 (bs, 1 H), 4.13 (m, 2 H), 3.10–3.03 (m, 3 H), 2.21 (t, <i>J</i> = 7.9 Hz, 2 H), 1.77 (m, 1 H), 1.58 (m, 2 H), 1.38 (s, 9 H), 1.22 (m, 12 H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz): δ 196.9, 173.6, 156.0, 78.9, 50.2, 46.3, 40.6, 36.2, 30.0, 29.4, 29.3, 29.2, 29.2, 28.4, 26.7, 26.4, 26.4, 25.5. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>34</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup>,<sub>,</sub> 342.2518; found, 342.2507.</div></div><div id="sec5-1-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Methyl 5-(6-(<i>tert-</i>Butoxycarbonylamino)hexanamido)pentanoate (<b>10</b>)</h3><div class="NLM_p last"><i>N</i>-Succinimidyl 6-(<i>tert-</i>butoxycarbonylamino)hexanoate <b>9</b> (2.00 g, 6.09 mmol) and <i>i</i>-Pr<sub>2</sub>EtN (10 mL, 60 mmol) were added to a solution of methyl 5-aminopentanoate <b>8</b> (958 mg, 7.03 mmol) in DMF (2 mL). The reaction was stirred for 4 h at room temperature, concentrated in vacuo, dissolved in EtOAc, washed with aqueous NaCl (3 times). After drying over Na<sub>2</sub>SO<sub>4</sub>, the solution was concentrated under reduced pressure and purified by chromatography on a silica gel column (CH<sub>2</sub>Cl<sub>2</sub>/MeOH, 10/0 to 9/1, v/v) to provide compound <b>10</b> as a colorless oil (1.71 g, 82%); <i>R</i><sub><i>f</i></sub> = 0.34 (CH<sub>2</sub>Cl<sub>2</sub>/MeOH 95/5). <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 5.54 (bs, 1 H), 4.54 (bs, 1 H), 3.64 (s, 3 H), 3.23 (q, <i>J</i> = 7 Hz, 13 Hz, 2 H), 3.08 (m, 2 H), 2.32 (t, <i>J</i> = 7.2 Hz, 2 H), 2.13 (t, <i>J</i> = 7.5 Hz, 2 H), 1.65–1.60 (m, 4 H), 1.53–1.45 (m, 4 H), 1.41 (s, 9 H), 1.35–1.29 (m, 2 H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz): δ 174.1, 173.0, 156.2, 77.4, 76.3, 53.8, 51.8, 40.5, 39.1, 36.8, 33.7, 30.0, 29.2, 28.6, 26.6, 25.5, 22.3. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>32</sub>N<sub>2</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 344.2311; found, 344.2302.</div></div><div id="sec5-1-5" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 5-(6-(<i>tert</i>-Butoxycarbonylamino)hexanamido)pentanoic Acid (<b>11</b>)</h3><div class="NLM_p last">LiOH (1.00 g, 43.5 mmol) in water (10 mL) was added to a solution of <b>10</b> (1.50 g, 4.35 mmol) in THF (30 mL). The reaction was stirred overnight at room temperature, concentrated in vacuo, and purified by chromatography on a silica gel column (CH<sub>2</sub>Cl<sub>2</sub>/MeOH 10/0 to 8/2, v/v) to provide compound <b>11</b> as a white pasty solid (1.35 g, 94%); <i>R</i><sub><i>f</i></sub> = 0.3 (CH<sub>2</sub>Cl<sub>2</sub>/MeOH 9/1 v/v). <sup>1</sup>H NMR (DMSO, 400 MHz): δ 7.73 (m, 1 H), 6.74 (m, 1 H), 3.00 (q, <i>J</i> = 7 Hz, 13 Hz, 2 H), 2.87 (q, <i>J</i> = 7 Hz, 13 Hz, 2 H), 2.18 (t, <i>J</i> = 7.1 Hz, 2 H), 2.01 (t, <i>J</i> = 7.3 Hz, 2 H), 1.47–1.34 (m, 17 H), 1.23–1.16 (m, 2 H). HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>30</sub>N<sub>2</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 330.2156; found, 330.2148.</div></div><div id="sec5-1-6" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> <i>tert</i>-Butyl 6-(5-(2,3-Dihydroxypropylamino)-5-oxopentylamino)-6-oxohexylcarbamate (<b>12</b>)</h3><div class="NLM_p last">PYBOP (756 mg, 1.45 mmol) was added to a stirred solution of <b>11</b> (400 mg, 1.21 mmol) in DMF (10 mL). After 5 min at room temperature, 3-amino-1,2-propanediol (132 mg, 1.45 mmol) and <i>i</i>-Pr<sub>2</sub>EtN (782 μL, 4.84 mmol) were added and the mixture was allowed to react overnight at room temperature. It was then concentrated in vacuo and purified by chromatography over a silica gel column (CH<sub>2</sub>Cl<sub>2</sub>/MeOH, from 10/0 to 8/2, v/v) to provide compound <b>12</b> as a white pasty solid (458 mg, 94%); <i>R</i><sub><i>f</i></sub> = 0.32 (CH<sub>2</sub>Cl<sub>2</sub>/MeOH 9/1). <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 7.95 (bs, 1H), 3.73–3.67 (m, 1H), 3.54–3.46 (m, 2H), 3.38–3.41 (m, 1H), 3.22–3.16 (m, 4H), 3.2 (t, <i>J</i> = 7.0 Hz, 2H), 2.24 (t, <i>J</i> = 7.4 Hz, 2H), 2.18 (t, <i>J</i> = 2.18 Hz, 2H), 1.66–1.57 (m, 4H), 1.55–1.45 (m, 4H), 1.43 (s, 12H), 1.36–1.28 (m, 2H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz): δ 176.6, 176.2, 158.6, 79.9, 72.2, 65.1, 43.5 (2C), 41.3, 40.0 (2C), 37.1, 36.6, 30.8, 30.0, 28.9, 27.6, 26.9, 24.3. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>37</sub>N<sub>3</sub>O<sub>6</sub> [M + H]<sup>+</sup>, 403.2682; found, 403.2686.</div></div><div id="sec5-1-7" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> <i>tert</i>-Butyl 6-Oxo-6-(5-oxo-5-(2-oxoethylamino)pentylamino)hexylcarbamate (<b>13</b>)</h3><div class="NLM_p last">NaIO<sub>4</sub> (159 mg, 0.743 mmol) was added to a stirred solution of <b>12</b> (200 mg, 0.495 mmol) in a mixture of THF/water (1/1, v/v, 7 mL). The reaction was stirred for 1 h at room temperature, concentrated in vacuo, dissolved in CH<sub>2</sub>Cl<sub>2</sub>, washed with water, and dried over Na<sub>2</sub>SO<sub>4</sub>. Concentration of the solution under reduced pressure gave <b>13</b> as a yellow oil (184 mg, 100%) which was used without further purification <i>R</i><sub><i>f</i></sub> = 0.4 (CH<sub>2</sub>Cl<sub>2</sub>/MeOH 9/1, v/v). <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 7.95 (m, 1H), 4.55 (t, <i>J</i> = 5.3 Hz, 1H), 3.24–3.15 (m, 6H), 3.02 (t, <i>J</i> = 7.0 Hz, 2H), 2.22 (t, <i>J</i> = 7.5 Hz, 2H), 2.18 (t, <i>J</i> = 7.5 Hz, 2H), 1.65–1.57 (m, 4H), 1.54–1.40 (m, 4H), 1.43 (s, 9H), 1.36–1.29 (m, 2H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz): δ 176.2, 176.1, 176.0, 158.5, 97.3, 79.8, 55.0, 45.5, 41.2, 39.9, 37.1, 36.4, 30.7, 29.8, 28.9, 27.5, 26.7, 24.2. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>33</sub>N<sub>3</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 371.2420; found, 371.2418.</div></div><div id="sec5-1-8" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> <i>trans</i>-5-Chloro-2,3,3a,12b-tetrahydro-1<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrole (<b>17</b>) (Norasenapine)</h3><div class="NLM_p last">Asenapinium maleate <b>14</b> (50 mg, 0.124 mmol) in CH<sub>2</sub>Cl<sub>2</sub>/MeOH (1/2, v/v, 2.8 mL) was added to PL-TBD resin (0.37 mmol) preswollen in CH<sub>2</sub>Cl<sub>2</sub>/MeOH (1/1, v/v, 9 mL). The reaction was shaken overnight at room temperature and filtered. The filtrate was concentrated in vacuo. The free base (35.56 mg, 0.124 mmol) was dissolved in 1,2-dichloroethane (2 mL) and allowed to react overnight with 1-chloroethyl chloroformate (134 μL, 1.24 mmol) at 100 °C. The mixture was then concentrated in vacuo, dissolved in methanol, stirred for 2 h at reflux, concentrated in vacuo, and purified by chromatography over a silica gel column (CH<sub>2</sub>Cl<sub>2</sub>/MeOH 10/0 to 9/1, v/v) to provide compound <b>17</b> as a white solid (26.8 mg, 79%); <i>R</i><sub><i>f</i></sub> = 0.25 (CH<sub>2</sub>Cl<sub>2</sub>/MeOH 9/1, v/v); mp 236 °C. <sup>1</sup>H NMR (DMSO, 400 MHz): δ 7.27–7.08 (m, 7 H), 3.47–3.42 (m, 2 H), 3.31–3.23 (m, 2 H), 3.10–3.03 (m, 2 H). <sup>13</sup>C NMR (DMSO, 100 MHz): δ 154.8, 153.9, 134.3, 131.8, 128.1, 127.6, 127.5, 127.3, 127.2, 124.5, 122.5, 120.6, 50.2, 50.1, 45.1, 44.7. RP-HPLC purity: >95%, <i>t</i><sub>R</sub> = 3.66. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>14</sub>ClNO [M + H]<sup>+</sup>, 271.0764; found, 271.0763.</div></div><div id="sec5-1-9" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> <i>tert</i>-Butyl 2-(<i>trans</i>-5-Chloro-1,3,3a,12b-tetrahydro-2<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2-yl)ethylcarbamate (<b>18</b>)</h3><div class="NLM_p last"><i>N</i>-Boc-2-aminoacetaldehyde (7 mg, 0.044 mmol), NEt<sub>3</sub> (4 μL, 0.029 mmol), and NaBH<sub>3</sub>CN (2.78 mg, 0.044 mmol) were added to a solution of compound <b>17</b> trifluoroacetate (11.4 mg, 0.029 mmol) in methanol (0.34 mL). The reaction was stirred for 2 h at 25 °C, concentrated in vacuo, and purified by chromatography over a silica gel column (CH<sub>2</sub>Cl<sub>2</sub>/MeOH 100/0 to 99/1, v/v) to provide compound <b>18</b> as a colorless oil (11.8 mg, 96%); <i>R</i><sub><i>f</i></sub> = 0.05 (CH<sub>2</sub>Cl<sub>2</sub> 100%). <sup>1</sup>H NMR (DMSO, 500 MHz): δ 7.27–7.10 (m, 7 H), 6.76 (t, <i>J</i> = 5.1 Hz, 1 H), 3.45–3.40 (m, 2 H), 3.23 (m, 2 H), 3.12 (m, 2 H), 3.04 (m, 2 H), 2.75–2.70 (m, 1 H), 2.62–2.58 (m, 1 H), 1.38 (s, 9 H). <sup>13</sup>C NMR (DMSO, 125 MHz): δ 155.6, 154.7, 153.8, 134.3, 131.8, 128.0, 127.7, 127.4, 127.2, 126.9, 124.4, 122.5, 120.6, 77.5, 56.7, 56.4, 55.7, 43.8, 43.5, 43.4, 43.1, 42.6, 28.2. RP-HPLC purity: >95%, <i>t</i><sub>R</sub> = 4.52. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>27</sub>ClN<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 414.1710; found, 414.1720.</div></div><div id="sec5-1-10" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 2-(<i>trans</i>-5-Chloro-1,3,3a,12b-tetrahydro-2<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2-yl)ethan-1-amine, Trifluoroacetate (<b>19</b>)</h3><div class="NLM_p last">TFA (0.5 mL) was added to a solution of <b>18</b> (10.5 mg, 0.025 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL). The reaction was stirred for 2 h at room temperature, concentrated in vacuo, and purified by RP-HPLC to provide compound <b>19</b> (trifluoroacetate) as a yellow solid (7 mg, 65%); <i>R</i><sub><i>f</i></sub> = 0.27 (CH<sub>2</sub>Cl<sub>2</sub>/NH<sub>4</sub>OH in MeOH 7 M, 95/5); mp 108 °C. <sup>1</sup>H NMR (DMSO, 400 MHz): δ 8.25 (bs, 2H), 7.39–7.18 (m, 7H), 4.10–3.57 (m, 8H), 3.32–3.29 (m, 2H). <sup>13</sup>C NMR (DMSO, 100 MHz): δ 154.6, 153.8, 130.9, 128.7, 128.5, 128.4, 127.5, 127.3, 124.8, 122.9, 121.0, 56.4, 56.1, 52.4, 42.0, 41.7, 34.9. RP-HPLC purity: >95%, <i>t</i><sub>R</sub> = 3.28. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>19</sub>ClN<sub>2</sub>O [M + H]<sup>+</sup>, 314.1186; found, 314.1185.</div></div><div id="sec5-1-11" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> <i>tert</i>-Butyl 11-(2-(<i>trans</i>-5-Chloro-1,3,3a,12b-tetrahydro-2<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2-yl)ethylamino)-11-oxoundecylcarbamate (<b>20</b>)</h3><div class="NLM_p last">Aldehyde <b>7</b> (20 mg, 0.059 mmol) and NaBH<sub>3</sub>CN (2.8 mg, 0.044 mmol) were added to a solution of <b>17</b> (8 mg, 0.03 mmol) in MeOH (1 mL). The reaction was stirred 2 h at room temperature, concentrated in vacuo, and purified by chromatography over a silica gel column (CH<sub>2</sub>Cl<sub>2</sub>/MeOH 100/0 to 95/5, v/v) to provide compound <b>20</b> as a yellow oil (14 mg, 79%); <i>R</i><sub><i>f</i></sub> = 0.4 (CH<sub>2</sub>Cl<sub>2</sub>/MeOH 97.5/2.5). <sup>1</sup>H NMR (DMSO, 500 MHz): δ 7.80 (m, 1H), 7.30–7.11 (m, 7H), 6.74 (m, 1H), 3.51–3.47 (m, 2H), 3.31–3.25 (m, 4H), 3.11 (m, 2H), 2.87 (m, 2H), 2.78 (m, 1H), 2.68 (m, 1H), 2.07 (t, <i>J</i> = 7.2 Hz, 2H), 1.49 (m, 2H), 1.37 (s, 9H), 1.33 (m, 2H), 1.23–1.19 (m, 12H). <sup>13</sup>C NMR (DMSO, 125 MHz) δ 172.1, 155.5, 154.7, 153.8, 134.2, 131.7, 128.0, 127.8, 127.4, 127.2, 126.9, 124.4, 122.5, 120.6, 77.2, 56.7, 56.5, 55.4, 43.7, 43.4, 37.6, 36.6, 35.4, 29.4, 29.0, 28.9, 28.8, 28.7, 28.6, 28.2, 26.2, 25.3. RP-HPLC purity: >95%, <i>t</i><sub>R</sub> = 5.61. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>34</sub>H<sub>48</sub>ClN<sub>3</sub>O<sub>4</sub> [M + H]<sup>+</sup>, 597.3333; found, 597.3338.</div></div><div id="sec5-1-12" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 11-((2-(<i>trans</i>-5-Chloro-1,3,3a,12b-tetrahydro-2<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2-yl)ethyl)amino)-11-oxoundecan-1-amine (<b>21</b>)</h3><div class="NLM_p last">TFA (0.5 mL) was added to a solution of <b>20</b> (14 mg, 0.023 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL). The reaction was stirred for 2 h at room temperature, concentrated in vacuo, and purified by RP-HPLC to provide compound <b>21</b> as a yellow oil (free base: 11.5 mg, 98%); <i>R</i><sub><i>f</i></sub> = 0.27 (CH<sub>2</sub>Cl<sub>2</sub>/NH<sub>3</sub>–MeOH 7 M 95/5). <i>R</i><sub><i>f</i></sub> = 0.44 (CH<sub>2</sub>Cl<sub>2</sub>/NH<sub>3</sub>–MeOH 7 M, 9/1). <sup>1</sup>H NMR (DMSO, 400 MHz): δ 7.79 (t, <i>J</i> = 5.8 Hz, 1 H), 7.28–7.09 (m, 7 H), 3.51–3.43 (m, 2 H), 3.26–3.12 (m, 6 H), 3.08–3.03 (m, 2 H), 2.77–2.71 (m, 1 H), 2.65–2.59 (m, 1 H), 2.07 (t, <i>J</i> = 7.3 Hz, 2 H) 1.52–1.45 (m, 2 H), 1.32–1.19 (m, 16 H). <sup>13</sup>C NMR (DMSO, 100 MHz): δ 172.0, 154.7, 153.8, 134.3, 131.8, 128.0, 127.6, 127.3, 127.1, 126.9, 124.3, 122.4, 120.5, 56.7, 56.5, 55.4, 43.8, 43.5, 41.3, 37.8, 35.4, 32.7, 29.0, 28.9, 28.8, 28.6, 26.3, 25.3. RP-HPLC purity: > 95%, <i>t</i><sub>R</sub> = 3.85. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>29</sub>H<sub>40</sub>ClN<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 497.2809; found, 497.2808.</div></div><div id="sec5-1-13" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> <i>trans</i>-2-(14-Azido-3,6,9,12-tetraoxatetradecyl)-5-chloro-2,3,3a,12b-tetrahydro-1<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrole (<b>22</b>)</h3><div class="NLM_p last">Compound <b>4</b> (7.5 mg, 0.022 mmol) and K<sub>2</sub>CO<sub>3</sub> (6 mg, 0.045 mmol) were added to a solution of compound <b>17</b> (7 mg, 0.018 mmol) in acetonitrile (200 μL). The reaction was stirred overnight at room temperature, sonicated for 2 h at 50 °C, filtered, and purified by column chromatography to provide compound <b>22</b> (4.3 mg, 47%). <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 7.31–7.06 (m, 7 H), 4.47–4.14 (m, 2 H), 3.98–3.80 (m, 6H), 3.71–3.46 (m, 16 H), 3.35 (t, <i>J</i> = 5.1 Hz, 2H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz): δ 155.4, 154.3, 129.3, 128.2, 127.9, 127.0, 126.9, 124.5, 122.6, 121.2, 70.7, 70.6, 70.6, 70.0, 57.6, 57.4, 56.1, 50.7, 43.6, 43.5. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>26</sub>H<sub>33</sub>ClN<sub>4</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 516.2135; found, 516.2139.</div></div><div id="sec5-1-14" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 14-(<i>trans</i>-5-Chloro-1,3,3a,12b-tetrahydro-2<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2-yl)-3,6,9,12-tetraoxatetradecan-1-amine (<b>23</b>)</h3><div class="NLM_p last">Compound <b>22</b> (25 mg, 0.046 mmol) in THF (1.5 mL) and water (0.4 mL) were added to PS-PPh3 resin (46 mmol) preswollen in THF (1.5 mL) for 25 min. The reaction was shaken overnight at room temperature and filtered. The filtrate was concentrated in vacuo and purified by RP-HPLC to provide compound <b>23</b> (free base; 21.5 mg, 95%). <sup>1</sup>H NMR (MeOD, 400 MHz): δ 8.82–8.68 (m, 7H), 5.78–5.57 (m, 4 H), 5.42–5.39 (m, 6 H), 5.24–5.22 (m, 2 H), 5.20–5.17 (m, 3 H), 5.16–5.13 (m, 4 H), 5.11–5.07 (m, 4 H), 5.05–5.02 (m, 3 H), 4.99–4.95 (m, 2 H). <sup>13</sup>C NMR (MeOD, 100 MHz): δ 156.1 131.0, 130.3, 130.1, 129.0, 127.7, 126.3, 124.3, 122.6, 73.8, 71.6, 71.5, 66.4, 62.3, 58.5, 58.2, 57.5, 44.0, 43.5. RP-HPLC purity: >95%, <i>t</i><sub>R</sub> = 3.42. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>26</sub>H<sub>35</sub>ClN<sub>2</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 490.2234; found, 490.2231.</div></div><div id="sec5-1-15" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> <i>tert</i>-Butyl 6-(5-(2-(<i>trans</i>-5-Chloro-1,3,3a,12b-tetrahydro-2<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2-yl)ethylamino)-5-oxopentylamino)-6-oxohexylcarbamate (<b>24</b>)</h3><div class="NLM_p last">Aldehyde <b>13</b> (20.5 mg, 0.055 mmol) and NaBH<sub>3</sub>CN (3.4 mg, 0.055 mmol) were added to a solution of <b>17</b> (10 mg, 0.037 mmol) in MeOH (1 mL). The reaction was stirred 2 h at room temperature, concentrated in vacuo, and purified by chromatography over a silica gel column (CH<sub>2</sub>Cl<sub>2</sub>/MeOH 100/0 to 80/20, v/v) to provide compound <b>24</b> as a yellow oil (14.8 mg, 64%); <i>R</i><sub><i>f</i></sub> = 0.25 (CH<sub>2</sub>Cl<sub>2</sub>/MeOH 95/5 v/v). <sup>1</sup>H NMR (DMSO, 500 MHz): δ 7.79 (t, <i>J</i> = 5.8 Hz, 1 H), 7.71 (t, <i>J</i> = 5.6 Hz, 1 H), 7.27–7.09 (m, 7 H), 6.74 (t, <i>J</i> = 5.4 Hz, 1 H), 3.51–3.41 (m, 2 H), 3.25–3.21 (m, 4 H), 3.05 (m, 2 H), 3.01 (m, 2 H), 2.87 (m, 2 H), 2.76–2.71 (m, 1 H), 2.64–2.59 (m, 1 H), 2.08 (m, 2 H), 2.01 (m, 2 H), 1.52–1.41 (m, 4 H), 1.39–1.30 (m, 4 H), 1.35 (s, 9 H), 1.21–1.14 (m, 2 H). <sup>13</sup>C NMR (DMSO, 125 MHz) δ 171.9, 171.8, 155.5, 154.7, 153.8, 134.3, 131.9, 128.0, 127.7, 127.4, 127.2, 126.9, 124.4, 122.5, 120.6, 77.2, 56.7, 56.6, 55.6, 54.8, 43.8, 43.5, 38.1, 37.8, 35.4, 35.0, 29.3, 28.8, 28.2, 26.0, 25.0, 22.8. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>34</sub>H<sub>47</sub>ClN<sub>4</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 626.3235; found, 626.3247.</div></div><div id="sec5-1-16" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 6-((5-((2-(<i>trans</i>-5-Chloro-1,3,3a,12b-tetrahydro-2<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2-yl)ethyl)amino)-5-oxopentyl)amino)-6-oxohexan-1-amine, Trifluoroacetate (<b>25</b>)</h3><div class="NLM_p last">TFA (0.5 mL) was added to a solution of <b>24</b> (14 mg, 0.022 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL). The reaction was stirred for 2 h at room temperature, concentrated in vacuo, and purified by RP-HPLC to provide compound <b>25</b> as a colorless oil (10.5 mg, 89%); <i>R</i><sub><i>f</i></sub> = 0.42 (CH<sub>2</sub>Cl<sub>2</sub>/MeOH-NH<sub>4</sub> 7 M 85/15 v/v). <sup>1</sup>H NMR (DMSO, 400 MHz): δ 7.82 (t, <i>J</i> = 6.0 Hz, 1H), 7.76 (t, <i>J</i> = 5.3 Hz, 1H), 7.28–7.11 (m, 7H), 5.1–4.2 (bs, 2H), 3.47–3.44 (m, 2H), 3.26–3.21 (m, 4H), 3.07–2.99 (m, 4H), 2.74–2.60 (m, 4H), 2.08 (m, 2H), 2.03 (m, 2H), 1.50–1.43 (m, 6H), 1.38–1.35 (m, 2H), 1.26–1.22 (m, 2H). <sup>13</sup>C NMR (DMSO, 100 MHz): δ 171.9, 171.7, 154.7, 153.8, 134.3, 131.9, 129.1, 128.0, 127.7, 127.4, 127.2, 127.0, 124.4, 122.5, 120.6, 56.7, 56.6, 55.6, 43.8, 43.5, 38.1, 37.8, 35.2, 35.0, 28.9, 28.8, 25.7, 24.9, 22.8. RP-HPLC purity: > 95%, <i>t</i><sub>R</sub> = 3.41. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>29</sub>H<sub>39</sub>ClN<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 526.2711; found, 526.2707.</div></div><div id="sec5-1-17" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> General Procedure for the Labeling of Amines <b>19</b>, <b>21</b>, <b>23</b>, and <b>25</b> with Fluorophores</h3><div class="NLM_p last"><i>i</i>-Pr<sub>2</sub>EtN (10 equiv) and 6-(fluorescein-5-carboxamido)hexanoic acid, succinimidyl ester (5-SFX-NHS), 2-((1<i>E</i>,3<i>E</i>)-5-((<i>Z</i>)-3-(4-((2,5-dioxopyrrolidin-1-yl)oxy)-4-oxobutyl)-1-ethyl-3-methyl-5-sulfoindolin-2-ylidene)penta-1,3-dien-1-yl)-1-ethyl-3,3-dimethyl-3<i>H</i>-indol-1-ium-5-sulfonate (DY647-NHS) or fluorescein-5-<i>N</i>-hydroxysuccinimide ester (FLUO-NHS) (1 equiv) were added to a solution of amine (1 equiv) in DMSO. The reaction was stirred for 2 h at room temperature and was monitored by analytical RP-HPLC. The products were purified by RP-HPLC.</div></div><div id="sec5-1-18" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 5-(6-(2-(<i>trans</i>-5-Chloro-1,3,3a,12b-tetrahydro-2<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2-yl)ethylamino)-6-oxohexylcarbamoyl)-2-(6-hydroxy-3-oxo-3<i>H</i>-xanthen-9-yl)benzoic Acid (<b>26</b>)</h3><div class="NLM_p last">Yellow powder. RP-HPLC purity: >95%, <i>t</i><sub>R</sub> = 4.38. UV (Titrisol buffer): λ<sub>max</sub> excitation 501 nm, λ<sub>max</sub> emission 527 nm. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>45</sub>H<sub>40</sub>ClN<sub>3</sub>O [M + H]<sup>+</sup>, 785.2504; found, 785.2484.</div></div><div id="sec5-1-19" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 2-((1<i>E</i>,3<i>E</i>,5<i>E</i>)-5-(3-(4-((2-(5-Chloro-3,3a-dihydro-1<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2(12bH)-yl)ethyl)amino)-4-oxobutyl)-1-ethyl-3-methyl-5-sulfoindolin-2-ylidene)penta-1,3-dien-1-yl)-1-ethyl-3,3-dimethyl-5-sulfo-3<i>H</i>-indol-1-ium (<b>27</b>)</h3><div class="NLM_p last">Blue powder. RP-HPLC purity: >95%, <i>t</i><sub>R</sub> = 3.87. UV (Titrisol buffer): λ<sub>max</sub> excitation 651 nm, λ<sub>max</sub> emission 665 nm. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>50</sub>H<sub>55</sub>ClN<sub>4</sub>O<sub>8</sub>S<sub>2</sub> [M + H]<sup>+</sup>, 938.3149; found, 938.3150.</div></div><div id="sec5-1-20" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 5-((6-((11-((2-(5-Chloro-3,3a-dihydro-1<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2(12bH)-yl)ethyl)amino)-11-oxoundecyl)amino)-6-oxohexyl)carbamoyl)-2-(6-hydroxy-3-oxo-3<i>H</i>-xanthen-9-yl)benzoic Acid (<b>28</b>)</h3><div class="NLM_p last">Yellow powder. RP-HPLC purity: >95%, <i>t</i><sub>R</sub> = 4.80. UV (Titrisol buffer): λ<sub>max</sub> excitation 494 nm, λ<sub>max</sub> emission 525 nm. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>56</sub>H<sub>61</sub>ClN<sub>4</sub>O<sub>9</sub> [M + H]<sup>+</sup>, 968.4127; found, 968.4122.</div></div><div id="sec5-1-21" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 2-((1<i>E</i>,3<i>E</i>,5<i>E</i>)-5-(3-(4-((11-((2-(5-Chloro-3,3a-dihydro-1<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2(12bH)-yl)ethyl)amino)-11-oxoundecyl)amino)-4-oxobutyl)-1-ethyl-3-methyl-5-sulfoindolin-2-ylidene)penta-1,3-dien-1-yl)-1-ethyl-3,3-dimethyl-5-sulfo-3<i>H</i>-indol-1-ium (<b>29</b>)</h3><div class="NLM_p last">Blue powder. RP-HPLC purity: >95%, <i>t</i><sub>R</sub> = 4.36. UV (Titrisol buffer): λ<sub>max</sub> excitation 653 nm, λ<sub>max</sub> emission 665 nm. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>61</sub>H<sub>77</sub>ClN<sub>5</sub>O<sub>9</sub>S<sub>2</sub> [M + H]<sup>+</sup>, 1121.4773; found, 1121.4757.</div></div><div id="sec5-1-22" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 5-((1-(5-Chloro-3,3a-dihydro-1<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2(12bH)-yl)-16-oxo-3,6,9,12-tetraoxa-15-azahenicosan-21-yl)carbamoyl)-2-(6-hydroxy-3-oxo-3<i>H</i>-xanthen-9-yl)benzoic Acid (<b>30</b>)</h3><div class="NLM_p last">Yellow powder. RP-HPLC purity: >95%, <i>t</i><sub>R</sub> = 4.35. UV (Titrisol buffer): λ<sub>max</sub> excitation 502 nm, λ<sub>max</sub> emission<sub>.</sub> 522 nm. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>53</sub>H<sub>56</sub>ClN<sub>3</sub>O<sub>12</sub> [M + H]<sup>+</sup>, 961.3553; found, 961.3535.</div></div><div id="sec5-1-23" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 2-((1<i>E</i>,3<i>E</i>,5<i>E</i>)-5-(3-(1-(5-Chloro-3,3a-dihydro-1<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2(12bH)-yl)-16-oxo-3,6,9,12-tetraoxa-15-azanonadecan-19-yl)-1-ethyl-3-methyl-5-sulfoindolin-2-ylidene)penta-1,3-dien-1-yl)-1-ethyl-3,3-dimethyl-5-sulfo-3<i>H</i>-indol-1-ium (<b>31</b>)</h3><div class="NLM_p last">Blue powder. RP-HPLC purity: >95%, <i>t</i><sub>R</sub> = 3.92. UV (Titrisol buffer): λ<sub>max</sub> excitation 660 nm, λ<sub>max</sub> emission 668 nm. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>58</sub>H<sub>71</sub>ClN<sub>4</sub>O<sub>12</sub>S<sub>2</sub> [M + H]<sup>+</sup>, 1114.4198; found, 1114.4199.</div></div><div id="sec5-1-24" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 5-((6-((6-((5-((2-(5-Chloro-3,3a-dihydro-1<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2(12bH)-yl)ethyl)amino)-5-oxopentyl)amino)-6-oxohexyl)amino)-6-oxohexyl)carbamoyl)-2-(6-hydroxy-3-oxo-3<i>H</i>-xanthen-9-yl)benzoic Acid (<b>32</b>)</h3><div class="NLM_p last">Yellow powder (143 nmol, 14%). RP-HPLC purity: >95%, <i>t</i><sub>R</sub> = 4.22. UV (Titrisol buffer): λ<sub>max</sub> excitation 500 nm, λ<sub>max</sub> emission 526 nm. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>56</sub>H<sub>60</sub>ClN<sub>5</sub>O<sub>10</sub> [M + H]<sup>+</sup>, 997.4028; found, 997.4010.</div></div><div id="sec5-1-25" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 2-((1<i>E</i>,3<i>E</i>,5<i>E</i>)-5-(3-(4-((6-((5-((2-(5-Chloro-3,3a-dihydro-1<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2(12bH)-yl)ethyl)amino)-5-oxopentyl)amino)-6-oxohexyl)amino)-4-oxobutyl)-1-ethyl-3-methyl-5-sulfoindolin-2-ylidene)penta-1,3-dien-1-yl)-1-ethyl-3,3-dimethyl-3<i>H</i>-indol-1-ium-5-sulfonate (<b>33</b>)</h3><div class="NLM_p last">Blue powder (304 nmol, 30%). UV (MeOH): λ<sub>max</sub> 649 nm. RP-HPLC purity: >95%, <i>t</i><sub>R</sub> = 3.90. UV (Titrisol buffer): λ<sub>max</sub> excitation 655 nm, λ<sub>max</sub> emission<sub>.</sub> 665 nm. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>61</sub>H<sub>77</sub>ClN<sub>6</sub>O<sub>10</sub>S<sub>2</sub> [M+2H]<sup>+2</sup>, 1152.4831; found, 1152.4799.</div></div><div id="sec5-1-26" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 5-((6-((6-((5-((2-(5-Chloro-3,3a-dihydro-1<i>H</i>-dibenzo[2,3:6,7]oxepino[4,5-<i>c</i>]pyrrol-2(12bH)-yl)ethyl)amino)-5-oxopentyl)amino)-6-oxohexyl)amino)-6-oxohexyl)carbamoyl)-2-(6-hydroxy-3-oxo-3<i>H</i>-xanthen-9-yl)benzoic Acid (<b>34</b>)</h3><div class="NLM_p last">Yellow powder (340 nmol, 34%). UV (Titrisol buffer): λ<sub>max</sub> 498 nm. RP-HPLC purity: >95%, <i>t</i><sub>R</sub> = 4.27. UV (Titrisol buffer): λ<sub>max</sub> excitation 495 nm, λ<sub>max</sub> emission 524 nm. HRMS (ESI-TOF) <i>m</i>/<i>z</i> calcd for C<sub>50</sub>H<sub>49</sub>ClN<sub>4</sub>O<sub>9</sub> [M + H]<sup>+</sup>, 884.3188; found, 884.3153.</div></div><div id="sec5-2-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Biology. Reagents</h3><div class="NLM_p last">The 96-well plates (ref 655086) were purchased from Greiner Bio-One (<a href="http://www.gbo.com" class="extLink">www.gbo.com</a>). Substrate SNAP-Lumi4-Tb (ref SSNPTBX) was provided by Cisbio Bioassays. All unlabeled ligands were purchased from Tocris (Bristol, U.K.). All SNAP-GPCR plasmids were provided by Cisbio Bioassays. Details can be found at <a href="http://www.cisbio.com/drug-discovery/receptor-binding-assays" class="extLink">http://www.cisbio.com/drug-discovery/receptor-binding-assays</a>. Tag-lite labeling medium (ref LABMED) was provided by Cisbio Bioassays.</div></div><div id="sec5-2-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Cell Culture</h3><div class="NLM_p last">HEK293T wild-type cells were maintained in Dulbecco’s modified Eagle’s medium (DMEM) glutaMAX (1966-021; Invitrogen, Carlsbad, CA) supplemented with 10% fetal calf serum.</div></div><div id="sec5-2-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Transfection Procedures</h3><div class="NLM_p last">Reverse transient transfections were performed on adherent HEK293T cells in 96-well plates using cell density from 50 000 to 100 000 cells per well. Prior to cell plating, wells were precoated with 50 μL of poly-<span class="smallcaps smallerCapital">l</span>-ornithine for 30 min at 37 °C. For all assays, transfection mixes were prepared using 100 ng of the GPCR plasmid, 0.5 μL of Lipofectamine 2000 (Life Technologies), and 50 μL of OptiMEM, Glutamax culture medium per well. Before addition in plates, transfection mixes were preincubated 20 min at room temperature. Then, an amount of 100 μL of HEK293T cells was added in each well. Plates were incubated at 37 °C under 5% CO<sub>2</sub> for 24 h before ligand binding assays.</div></div><div id="sec5-2-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Time-Resolved FRET Binding Assays</h3><div class="NLM_p last">At first, HEK293T cells transiently expressing SNAP-tagged GPCRs were treated with substrate SNAP-Lumi4-Tb. Cell culture medium was removed from the 96-well plates and 100 nM of reagent, previously diluted in Tag-lite labeling medium, was added (100 μL per well) before an incubation of 1 h at 37 °C under 5% CO<sub>2</sub>. The excess of reagent was removed by 4 washes with 100 μL of Tag-lite labeling medium. Ligand binding experiments were then performed with plates containing 50 μL of Tag-lite labeling medium. 25 μL of unlabeled compound or Tag-lite labeling medium was added, followed by the addition of 25 μL of fluorescent ligand. Plates were then incubated at rt in the dark for 1 h before signal detection. Binding affinities were determined by incubating the cells at rt with increasing concentrations of the desired fluorescent ligand. For each fluorescent ligand concentration, the nonspecific binding was determined in the presence of an excess of unlabeled compound (10 μM asenapine). Both fluorescent ligands and unlabeled compounds were diluted in the Tag-lite labeling medium. For competition binding assays, fluorescent ligands at fixed concentration (<b>30</b> and <b>31</b> were respectively used at 70 nM and 10 nM on 5HT<sub>2b</sub> receptor; <b>29</b> and <b>31</b> were respectively used at 4 nM and 17 nM on 5HT<sub>1d</sub> receptor; <b>26</b> was used at 10 nM on 5HT<sub>2c</sub>, 5HT<sub>6</sub>, and D<sub>4</sub> receptors; <b>31</b> was used at 78 nM on D<sub>4</sub> receptor) are used in the presence of increasing concentrations of unlabeled ligands. Plates are incubated 1 h in the dark at room temperature before signal detection.</div></div><div id="sec5-2-5" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Data Analysis</h3><div class="NLM_p last">Signal detection was performed on Infinite F500 (Tecan, Männedorf, Switzerland). Time-resolved FRET readouts were recorded and analyzed as described previously (<contrib-group>Zwier, J. M.</contrib-group> <cite>J. Biomol. Screening</cite> <span class="NLM_year">2010</span>, <em>15</em> (10), 1248−1259). All binding data were analyzed with GraphPad Prism (GraphPad Software, Inc., San Diego, CA). <i>K</i><sub>d</sub> values of the fluorescent ligands were obtained from saturation curves of the specific binding. Specific binding was determined by subtracting the nonspecific homogeneous time-resolved fluorescence (HTRF) ratio from the total HTRF ratio. Competition data were analyzed according to a sigmoid model by nonlinear regression. <i>K</i><sub>i</sub> values of unlabeled compounds were calculated from IC<sub>50</sub> of binding competition experiments according to the Cheng and Prusoff equation: <i>K</i><sub>i</sub> = IC<sub>50</sub>/(1 + ([L]/<i>K</i><sub>d</sub>)), where IC<sub>50</sub> is the concentration of unlabeled analogue leading to half-maximal inhibition of specific binding, <i>K</i><sub>d</sub> is its affinity for the receptor studied, and [L] is the concentration of the fluorescent probe present in the assay. All results are expressed as the mean ± SEM of at least three independent experiments.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i59"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01220">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51833" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51833" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01220" class="ext-link">10.1021/acs.jmedchem.7b01220</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Fluorescence properties of ligands <b>26</b>–<b>34</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01220/suppl_file/jm7b01220_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01220/suppl_file/jm7b01220_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01220/suppl_file/jm7b01220_si_001.pdf">jm7b01220_si_001.pdf (34.64 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01220/suppl_file/jm7b01220_si_002.csv">jm7b01220_si_002.csv (3.36 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b01220" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77666" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77666" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thierry Durroux</span> - <span class="hlFld-Affiliation affiliation">Institut de Génomique
Fonctionnelle, CNRS UMR5203, INSERM U661, Université de Montpellier
(IFR3), 141 Rue de la Cardonille, F-34094 Montpellier Cedex 5, France</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5c2838292e2e3329241c353b3a723f322e2f723a2e"><span class="__cf_email__" data-cfemail="6511011017170a101d250c02034b060b17164b0317">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marcel Hibert</span> - <span class="hlFld-Affiliation affiliation">Laboratoire
d’Innovation Thérapeutique, Faculté de Pharmacie, UMR7200 CNRS, Université de Strasbourg, 74 Route du Rhin, 67412 Illkirch, France</span>; 
    <span class="hlFld-Affiliation affiliation">LabEx Medalis, Université de Strasbourg, 67000 Strasbourg, France</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7786-7276" title="Orcid link">http://orcid.org/0000-0001-7786-7276</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a6cbcecfc4c3d4d2e6d3c8cfd5d2d4c788c0d4"><span class="__cf_email__" data-cfemail="c6abaeafa4a3b4b286b3a8afb5b2b4a7e8a0b4">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Candide Hounsou</span> - <span class="hlFld-Affiliation affiliation">Institut de Génomique
Fonctionnelle, CNRS UMR5203, INSERM U661, Université de Montpellier
(IFR3), 141 Rue de la Cardonille, F-34094 Montpellier Cedex 5, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Corinne Baehr</span> - <span class="hlFld-Affiliation affiliation">Laboratoire
d’Innovation Thérapeutique, Faculté de Pharmacie, UMR7200 CNRS, Université de Strasbourg, 74 Route du Rhin, 67412 Illkirch, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vincent Gasparik</span> - <span class="hlFld-Affiliation affiliation">Laboratoire
d’Innovation Thérapeutique, Faculté de Pharmacie, UMR7200 CNRS, Université de Strasbourg, 74 Route du Rhin, 67412 Illkirch, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Doria Alili</span> - <span class="hlFld-Affiliation affiliation">Institut de Génomique
Fonctionnelle, CNRS UMR5203, INSERM U661, Université de Montpellier
(IFR3), 141 Rue de la Cardonille, F-34094 Montpellier Cedex 5, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Abderazak Belhocine</span> - <span class="hlFld-Affiliation affiliation">Institut de Génomique
Fonctionnelle, CNRS UMR5203, INSERM U661, Université de Montpellier
(IFR3), 141 Rue de la Cardonille, F-34094 Montpellier Cedex 5, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thiéric Rodriguez</span> - <span class="hlFld-Affiliation affiliation">Institut de Génomique
Fonctionnelle, CNRS UMR5203, INSERM U661, Université de Montpellier
(IFR3), 141 Rue de la Cardonille, F-34094 Montpellier Cedex 5, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elodie Dupuis</span> - <span class="hlFld-Affiliation affiliation">Cisbio Bioassays, Parc Marcel Boiteux, BP84175, 30200 Codolet, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas Roux</span> - <span class="hlFld-Affiliation affiliation">Cisbio Bioassays, Parc Marcel Boiteux, BP84175, 30200 Codolet, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">André Mann</span> - <span class="hlFld-Affiliation affiliation">Laboratoire
d’Innovation Thérapeutique, Faculté de Pharmacie, UMR7200 CNRS, Université de Strasbourg, 74 Route du Rhin, 67412 Illkirch, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Denis Heissler</span> - <span class="hlFld-Affiliation affiliation">Laboratoire
d’Innovation Thérapeutique, Faculté de Pharmacie, UMR7200 CNRS, Université de Strasbourg, 74 Route du Rhin, 67412 Illkirch, France</span>; 
    <span class="hlFld-Affiliation affiliation">LabEx Medalis, Université de Strasbourg, 67000 Strasbourg, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jean-Philippe Pin</span> - <span class="hlFld-Affiliation affiliation">Institut de Génomique
Fonctionnelle, CNRS UMR5203, INSERM U661, Université de Montpellier
(IFR3), 141 Rue de la Cardonille, F-34094 Montpellier Cedex 5, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dominique Bonnet</span> - <span class="hlFld-Affiliation affiliation">Laboratoire
d’Innovation Thérapeutique, Faculté de Pharmacie, UMR7200 CNRS, Université de Strasbourg, 74 Route du Rhin, 67412 Illkirch, France</span>; 
    <span class="hlFld-Affiliation affiliation">LabEx Medalis, Université de Strasbourg, 67000 Strasbourg, France</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8252-9199" title="Orcid link">http://orcid.org/0000-0002-8252-9199</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>C.H. and C.B. contributed equally to the work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i61">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82237" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82237" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Thanks are due to Drs. Eric Trinquet, Gerard Mathis, and Laurent Lamarque for helpful discussions. This work was supported by research grants from the Centre National de la Recherche Scientifique (CNRS), and the Institut National de la Santé et de la Recherche Medicale (INSERM), by the Université de Strasbourg and the LABEX Medalis (Grant ANR-10-LABX-0034), by a grant from the Région Languedoc-Roussillon (Cell2Lead project), the Fonds Unique Interministériel, the Fonds Européen de Développement Régional and Oseo. This work was performed because of the Plateforme Arpège of Montpellier. HTRF and Tag-lite are registered trademarks of Cisbio Bioassays. Lumi4 is a registered trademark of Lumiphore, Inc. SNAP-tag is a trademark of New England Biolabs, Inc. We are grateful to Cyril Antheaume, Barbara Schaeffer, and Justine Vieville for NMR experiments and to Pascale Buisine and Patrick Wehrung for mass spectrometry (PACSI platform GDS3670).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i62" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i62"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i63" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i63"> Abbreviation Used</h2><tr><td class="NLM_term"><sup>13</sup>C NMR</td><td class="NLM_def"><p class="first last">carbon-13 nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle’s medium</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethylsulfoxide</p></td></tr><tr><td class="NLM_term">DY647</td><td class="NLM_def"><p class="first last">3-[5-[(2<i>E</i>)-1,3-dimethyl-2-[(2<i>E</i>,4<i>E</i>)-5-(1,3,3-trimethylindol-1-ium-2-yl)penta-2,4-dienylidene]indol-3-yl]pentoxy[di(propan-2-yl)amino]phosphanyl]oxypropanenitrile, 4-methylbenzenesulfonate</p></td></tr><tr><td class="NLM_term">EL</td><td class="NLM_def"><p class="first last">extracellular loops</p></td></tr><tr><td class="NLM_term">ESI-TOF</td><td class="NLM_def"><p class="first last">electron spray ionization-time-of-flight</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G-protein-coupled receptor</p></td></tr><tr><td class="NLM_term">FDA</td><td class="NLM_def"><p class="first last">Food and Drug Administration</p></td></tr><tr><td class="NLM_term">FRET</td><td class="NLM_def"><p class="first last">fluorescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high throughput screening</p></td></tr><tr><td class="NLM_term">FLUO</td><td class="NLM_def"><p class="first last">fluorescein</p></td></tr><tr><td class="NLM_term"><sup>1</sup>H NMR</td><td class="NLM_def"><p class="first last">proton nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high resolution mass spectrometry</p></td></tr><tr><td class="NLM_term"><i>J</i></td><td class="NLM_def"><p class="first last">coupling constant</p></td></tr><tr><td class="NLM_term">PL-TBD</td><td class="NLM_def"><p class="first last">polymer linked 1,5,7-triazabicyclo[4.4.0]dec-5-ene</p></td></tr><tr><td class="NLM_term">ppm</td><td class="NLM_def"><p class="first last">parts per million</p></td></tr><tr><td class="NLM_term">PS-PPh<sub>3</sub></td><td class="NLM_def"><p class="first last">diphenylphosphinopolystyrene</p></td></tr><tr><td class="NLM_term">PyBoP</td><td class="NLM_def"><p class="first last">benzotriazol-1-yl-oxytripyrrolidinophosphonium hexafluorophosphate</p></td></tr><tr><td class="NLM_term"><i>R</i><sub><i>f</i></sub></td><td class="NLM_def"><p class="first last">retention factor</p></td></tr><tr><td class="NLM_term">RP-HPLC</td><td class="NLM_def"><p class="first last">reverse-phase high performance liquid chromatography</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of the mean</p></td></tr><tr><td class="NLM_term">5-SFX</td><td class="NLM_def"><p class="first last">6-(fluorescein-5-carboxamido)hexanoic acid, succinimidyl ester</p></td></tr><tr><td class="NLM_term"><i>S</i>/<i>N</i></td><td class="NLM_def"><p class="first last">signal to noise</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin layer chromatography</p></td></tr><tr><td class="NLM_term">TM</td><td class="NLM_def"><p class="first last">transmembrane domain</p></td></tr><tr><td class="NLM_term"><i>t</i><sub>R</sub></td><td class="NLM_def"><p class="first last">HPLC retention time</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence resonance energy transfer.</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i64">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06023" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06023" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 55 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Bockaert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pin, J.-P.</span><span> </span><span class="NLM_article-title">Molecular tinkering of G protein-coupled receptors: an evolutionary success</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1723</span><span class="NLM_x">–</span> <span class="NLM_lpage">1729</span><span class="refDoi"> DOI: 10.1093/emboj/18.7.1723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1093%2Femboj%2F18.7.1723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10202136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADyaK1MXivVylu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=1723-1729&author=J.+Bockaertauthor=J.-P.+Pin&title=Molecular+tinkering+of+G+protein-coupled+receptors%3A+an+evolutionary+success&doi=10.1093%2Femboj%2F18.7.1723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular tinkering of G protein-coupled receptors: an evolutionary success</span></div><div class="casAuthors">Bockaert, Joel; Pin, Jean Philippe</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1723-1729</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review, with 56 refs.  Among membrane-bound receptors, the G protein-coupled receptors (GPCRs) are certainly the most diverse.  They have been very successful during evolution, being capable of transducing messages as different as photons, org. odorants, nucleotides, nucleosides, peptides, lipids and proteins.  Indirect studies, as well as two-dimensional crystn. of rhodopsin, have led to a useful model of a common "central core", composed of seven transmembrane helical domains, and its structural modifications during activation.  There are at least six families of GPCRs showing no sequence similarity.  They use an amazing no. of different domains both to bind their ligands and to activate G proteins.  The fine-tuning of their coupling to G proteins is regulated by splicing, RNA editing and phosphorylation.  Some GPCRs have been found to form either homo- or heterodimers with a structurally different GPCR, but also with membrane-bound proteins having one transmembrane domain such as nina-A, odr-4 or RAMP, the latter being involved in their targeting, function and pharmacol.  Finally, some GPCRs are unfaithful to G proteins and interact directly, via their C-terminal domain, with proteins contg. PDZ and Enabled/VASP homol. (EVH)-like domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_j_RcAF1YIrVg90H21EOLACvtfcHk0liYKR16IVjeBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXivVylu7o%253D&md5=4759ee7df2733d92c073e1cad1cda9a7</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F18.7.1723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F18.7.1723%26sid%3Dliteratum%253Aachs%26aulast%3DBockaert%26aufirst%3DJ.%26aulast%3DPin%26aufirst%3DJ.-P.%26atitle%3DMolecular%2520tinkering%2520of%2520G%2520protein-coupled%2520receptors%253A%2520an%2520evolutionary%2520success%26jtitle%3DEMBO%2520J.%26date%3D1999%26volume%3D18%26spage%3D1723%26epage%3D1729%26doi%3D10.1093%2Femboj%2F18.7.1723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Gudermann, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schöneberg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, G.</span><span> </span><span class="NLM_article-title">Functional and structural complexity of signal transduction via G-protein-coupled receptors</span> <span class="citation_source-journal">Annu. Rev. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">399</span><span class="NLM_x">–</span> <span class="NLM_lpage">427</span><span class="refDoi"> DOI: 10.1146/annurev.neuro.20.1.399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1146%2Fannurev.neuro.20.1.399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=9056720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADyaK2sXhslSjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1997&pages=399-427&author=T.+Gudermannauthor=T.+Sch%C3%B6nebergauthor=G.+Schultz&title=Functional+and+structural+complexity+of+signal+transduction+via+G-protein-coupled+receptors&doi=10.1146%2Fannurev.neuro.20.1.399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Functional and structural complexity of signal transduction via G-protein-coupled receptors</span></div><div class="casAuthors">Gudermann, Thomas; Schoeneberg, Torsten; Schultz, Guenter</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Neuroscience</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">399-427</span>CODEN:
                <span class="NLM_cas:coden">ARNSD5</span>;
        ISSN:<span class="NLM_cas:issn">0147-006X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review with 181 refs.  A prerequisite for the maintenance of homeostasis in a living organism is fine-tuned communication between different cells.  The majority of extracellular signaling mols., such as hormones and neurotransmitters, interact with a three-protein transmembrane signaling system consisting of a receptor, a G protein, and an effector.  These single components interact sequentially and reversibly.  Considering that hundreds of G-protein-coupled receptors interact with a limited repertoire of G proteins, the question of coupling specificity is worth considering.  G-protein-mediated signal transduction is a complex signaling network with diverging and converging transduction steps at each coupling interface.  The recent realization that classical signaling pathways are intimately intertwined with growth-factor-signaling cascades adds another level of complexity.  Elaborate studies have significantly enhanced our knowledge of the functional anatomy of G-protein-coupled receptors, and the concept has emerged that receptor function can be modulated with high specificity by coexpressed receptor fragments.  These results may have significant clin. impact in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonpBt3YtTc9bVg90H21EOLACvtfcHk0lijN43-013wSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhslSjt7Y%253D&md5=e26edfe9e07244131d8d22d34cb68c2d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1146%2Fannurev.neuro.20.1.399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.neuro.20.1.399%26sid%3Dliteratum%253Aachs%26aulast%3DGudermann%26aufirst%3DT.%26aulast%3DSch%25C3%25B6neberg%26aufirst%3DT.%26aulast%3DSchultz%26aufirst%3DG.%26atitle%3DFunctional%2520and%2520structural%2520complexity%2520of%2520signal%2520transduction%2520via%2520G-protein-coupled%2520receptors%26jtitle%3DAnnu.%2520Rev.%2520Neurosci.%26date%3D1997%26volume%3D20%26spage%3D399%26epage%3D427%26doi%3D10.1146%2Fannurev.neuro.20.1.399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Heng, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aubel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fussenegger, M.</span><span> </span><span class="NLM_article-title">An overview of the diverse roles of G-protein coupled receptors (GPCRs) in the pathophysiology of various human diseases</span> <span class="citation_source-journal">Biotechnol. Adv.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">1676</span><span class="NLM_x">–</span> <span class="NLM_lpage">1694</span><span class="refDoi"> DOI: 10.1016/j.biotechadv.2013.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2Fj.biotechadv.2013.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=23999358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVKgu7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=1676-1694&author=B.+C.+Hengauthor=D.+Aubelauthor=M.+Fussenegger&title=An+overview+of+the+diverse+roles+of+G-protein+coupled+receptors+%28GPCRs%29+in+the+pathophysiology+of+various+human+diseases&doi=10.1016%2Fj.biotechadv.2013.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">An overview of the diverse roles of G-protein coupled receptors (GPCRs) in the pathophysiology of various human diseases</span></div><div class="casAuthors">Heng, Boon Chin; Aubel, Dominique; Fussenegger, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Advances</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1676-1694</span>CODEN:
                <span class="NLM_cas:coden">BIADDD</span>;
        ISSN:<span class="NLM_cas:issn">0734-9750</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  G-protein coupled receptors (GPCRs) modulate diverse cellular responses to the majority of neurotransmitters and hormones within the human body.  They exhibit much structural and functional diversity, and are responsive to a plethora of endogenous (biogenic amines, cations, lipids, peptides, and glycoproteins) and exogenous (therapeutic drugs, photons, tastants, and odorants) ligands and stimuli.  Due to the key roles of GPCRs in tissue/cell physiol. and homeostasis, signaling pathways assocd. with GPCRs are implicated in the pathophysiol. of various diseases, ranging from metabolic, immunol., and neurodegenerative disorders, to cancer and infectious diseases.  Approx. 40% of clin. approved drugs mediate their effects by modulating GPCR signaling pathways, which makes them attractive targets for drug screening and discovery.  The pace of discovery of new GPCR-based drugs has recently accelerated due to rapid advancements in high-resoln. structure detn., high-throughput screening technol. and in silico computational modeling of GPCR binding interaction with potential drug mols.  This review aims to provide an overview of the diverse roles of GPCRs in the pathophysiol. of various diseases that are the major focus of biopharmaceutical research as potential drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraGad80-LA-bVg90H21EOLACvtfcHk0lijN43-013wSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVKgu7%252FF&md5=ab10a667170b8fe34dee4bee63a8b73a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.biotechadv.2013.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biotechadv.2013.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DHeng%26aufirst%3DB.%2BC.%26aulast%3DAubel%26aufirst%3DD.%26aulast%3DFussenegger%26aufirst%3DM.%26atitle%3DAn%2520overview%2520of%2520the%2520diverse%2520roles%2520of%2520G-protein%2520coupled%2520receptors%2520%2528GPCRs%2529%2520in%2520the%2520pathophysiology%2520of%2520various%2520human%2520diseases%26jtitle%3DBiotechnol.%2520Adv.%26date%3D2013%26volume%3D31%26spage%3D1676%26epage%3D1694%26doi%3D10.1016%2Fj.biotechadv.2013.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Drews, J.</span><span> </span><span class="NLM_article-title">Drug discovery: a historical perspective</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">1960</span><span class="NLM_x">–</span> <span class="NLM_lpage">1964</span><span class="refDoi"> DOI: 10.1126/science.287.5460.1960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1126%2Fscience.287.5460.1960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10720314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvFyrtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2000&pages=1960-1964&author=J.+Drews&title=Drug+discovery%3A+a+historical+perspective&doi=10.1126%2Fscience.287.5460.1960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery: A historical perspective</span></div><div class="casAuthors">Drews, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">5460</span>),
    <span class="NLM_cas:pages">1960-1963</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review with over 60 refs.  Driven by chem. but increasingly guided by pharmacol. and the clin. sciences, drug research has contributed more to the progress of medicine during the past century than any other scientific factor.  The advent of mol. biol. and, in particular, of genomic sciences is having a deep impact on drug discovery.  Recombinant proteins and monoclonal antibodies have greatly enriched our therapeutic armamentarium.  Genome sciences, combined with bioinformatic tools, allow us to dissect the genetic basis of multifactorial diseases and to det. the most suitable points of attack for future medicines, thereby increasing the no. of treatment options.  The dramatic increase in the complexity of drug research is enforcing changes in the institutional basis of this interdisciplinary endeavor.  The biotech industry is establishing itself as the discovery arm of the pharmaceutical industry.  In bridging the gap between academia and large pharmaceutical companies, the biotech firms have been effective instruments of technol. transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdT9qeMTdSTrVg90H21EOLACvtfcHk0lijN43-013wSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvFyrtbc%253D&md5=33804292ba1e2fa5fd8012729d137dff</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.287.5460.1960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.287.5460.1960%26sid%3Dliteratum%253Aachs%26aulast%3DDrews%26aufirst%3DJ.%26atitle%3DDrug%2520discovery%253A%2520a%2520historical%2520perspective%26jtitle%3DScience%26date%3D2000%26volume%3D287%26spage%3D1960%26epage%3D1964%26doi%3D10.1126%2Fscience.287.5460.1960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span> </span><span class="NLM_article-title">The druggable genome</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">730</span><span class="refDoi"> DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-730&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+druggable+genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0lhkfMtW19m1AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520druggable%2520genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D730%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Santos, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ursu, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donadi, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bologa, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlsson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">A comprehensive map of molecular drug targets</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">19</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1038/nrd.2016.230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnrd.2016.230" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=27910877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVSms7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=19-34&author=R.+Santosauthor=O.+Ursuauthor=A.+Gaultonauthor=A.+P.+Bentoauthor=R.+S.+Donadiauthor=C.+G.+Bologaauthor=A.+Karlssonauthor=B.+Al-Lazikaniauthor=A.+Herseyauthor=T.+I.+Opreaauthor=J.+P.+Overington&title=A+comprehensive+map+of+molecular+drug+targets&doi=10.1038%2Fnrd.2016.230"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A comprehensive map of molecular drug targets</span></div><div class="casAuthors">Santos, Rita; Ursu, Oleg; Gaulton, Anna; Bento, A. Patricia; Donadi, Ramesh S.; Bologa, Cristian G.; Karlsson, Anneli; Al-Lazikani, Bissan; Hersey, Anne; Oprea, Tudor I.; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-34</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The success of mechanism-based drug discovery depends on the definition of the drug target.  This definition becomes even more important as we try to link drug response to genetic variation, understand stratified clin. efficacy and safety, rationalize the differences between drugs in the same therapeutic class and predict drug utility in patient subgroups.  However, drug targets are often poorly defined in the literature, both for launched drugs and for potential therapeutic agents in discovery and development.  Here, we present an updated comprehensive map of mol. targets of approved drugs.  We curate a total of 893 human and pathogen-derived biomols. through which 1,578 US FDA-approved drugs act.  These biomols. include 667 human-genome-derived proteins targeted by drugs for human disease.  Anal. of these drug targets indicates the continued dominance of privileged target families across disease areas, but also the growth of novel first-in-class mechanisms, particularly in oncol.  We explore the relationships between bioactivity class and clin. success, as well as the presence of orthologues between human and animal models and between pathogen and human genomes.  Through the collaboration of three independent teams, we highlight some of the ongoing challenges in accurately defining the targets of mol. therapeutics and present conventions for deconvoluting the complexities of mol. pharmacol. and drug efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTfWwWkoZnbrVg90H21EOLACvtfcHk0lhkfMtW19m1AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVSms7rK&md5=7437d4b678ac75dac0a5e3248df98761</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.230%26sid%3Dliteratum%253Aachs%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DUrsu%26aufirst%3DO.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DDonadi%26aufirst%3DR.%2BS.%26aulast%3DBologa%26aufirst%3DC.%2BG.%26aulast%3DKarlsson%26aufirst%3DA.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DA%2520comprehensive%2520map%2520of%2520molecular%2520drug%2520targets%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D19%26epage%3D34%26doi%3D10.1038%2Fnrd.2016.230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Garland, S. L.</span><span> </span><span class="NLM_article-title">Are GPCRs still a source of new targets?</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">947</span><span class="NLM_x">–</span> <span class="NLM_lpage">966</span><span class="refDoi"> DOI: 10.1177/1087057113498418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1177%2F1087057113498418" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=23945874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWitrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=947-966&author=S.+L.+Garland&title=Are+GPCRs+still+a+source+of+new+targets%3F&doi=10.1177%2F1087057113498418"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Are GPCRs still a source of new targets?</span></div><div class="casAuthors">Garland, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">947-966</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  G-protein-coupled receptors (GPCRs) still offer enormous scope for new therapeutic targets.  Currently marketed agents are dominated by those with activity at aminergic receptors and yet they account for only ∼10% of the family.  Progress up until now with other subfamilies, notably orphans, Family A/peptide, Family A/lipid, Family B, Family C, and Family F, has been, at best, patchy.  This may be attributable to the heterogeneous nature of GPCRs, their endogenous ligands, and consequently their binding sites.  Our appreciation of receptor similarity has arguably been too simplistic, and screening collections have not necessarily been well suited to identifying leads in new areas.  Despite the relative shortage of highquality tool mols. in a no. of cases, there is an emerging, and increasingly substantial, body of evidence assocg. many as yet "undrugged" receptors with a very wide range of diseases.  Significant advances in our understanding of receptor pharmacol. and tech. advances in screening, protein X-ray crystallog., and ligand design methods are paving the way for new successes in the area.  Exploitation of allosteric mechanisms; alternative signaling pathways such as G12/13, Gβγ, and β-arrestin; the discovery of "biased" ligands; and the emergence of GPCR-protein complexes as potential drug targets offer scope for new and much improved drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjdjNgoJdJ5LVg90H21EOLACvtfcHk0lhkfMtW19m1AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWitrzK&md5=5bd5fd56a4af9ff828854aa8128f7e1a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1177%2F1087057113498418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113498418%26sid%3Dliteratum%253Aachs%26aulast%3DGarland%26aufirst%3DS.%2BL.%26atitle%3DAre%2520GPCRs%2520still%2520a%2520source%2520of%2520new%2520targets%253F%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26spage%3D947%26epage%3D966%26doi%3D10.1177%2F1087057113498418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Dixon, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strader, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benovic, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohlman, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frielle, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolanowski, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rands, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diehl, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mumford, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slater, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigal, I. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caron, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lefkowitz, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strader, C. D.</span><span> </span><span class="NLM_article-title">Cloning of the gene and cDNA for mammalian beta-adrenergic receptor and homology with rhodopsin</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">321</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">79</span><span class="refDoi"> DOI: 10.1038/321075a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2F321075a0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=1986&pages=75-79&author=R.+A.+Dixonauthor=B.+K.+Kobilkaauthor=D.+J.+Straderauthor=J.+L.+Benovicauthor=H.+G.+Dohlmanauthor=T.+Frielleauthor=M.+A.+Bolanowskiauthor=C.+D.+Bennettauthor=E.+Randsauthor=R.+E.+Diehlauthor=R.+A.+Mumfordauthor=E.+E.+Slaterauthor=I.+S.+Sigalauthor=M.+G.+Caronauthor=R.+J.+Lefkowitzauthor=C.+D.+Strader&title=Cloning+of+the+gene+and+cDNA+for+mammalian+beta-adrenergic+receptor+and+homology+with+rhodopsin&doi=10.1038%2F321075a0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2F321075a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F321075a0%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DR.%2BA.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DStrader%26aufirst%3DD.%2BJ.%26aulast%3DBenovic%26aufirst%3DJ.%2BL.%26aulast%3DDohlman%26aufirst%3DH.%2BG.%26aulast%3DFrielle%26aufirst%3DT.%26aulast%3DBolanowski%26aufirst%3DM.%2BA.%26aulast%3DBennett%26aufirst%3DC.%2BD.%26aulast%3DRands%26aufirst%3DE.%26aulast%3DDiehl%26aufirst%3DR.%2BE.%26aulast%3DMumford%26aufirst%3DR.%2BA.%26aulast%3DSlater%26aufirst%3DE.%2BE.%26aulast%3DSigal%26aufirst%3DI.%2BS.%26aulast%3DCaron%26aufirst%3DM.%2BG.%26aulast%3DLefkowitz%26aufirst%3DR.%2BJ.%26aulast%3DStrader%26aufirst%3DC.%2BD.%26atitle%3DCloning%2520of%2520the%2520gene%2520and%2520cDNA%2520for%2520mammalian%2520beta-adrenergic%2520receptor%2520and%2520homology%2520with%2520rhodopsin%26jtitle%3DNature%26date%3D1986%26volume%3D321%26spage%3D75%26epage%3D79%26doi%3D10.1038%2F321075a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Garland, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gloriam, D. E.</span><span> </span><span class="NLM_article-title">A ligand’s view of target similarity: chemogenomic binding site-directed techniques for drug discovery</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1872</span><span class="NLM_x">–</span> <span class="NLM_lpage">1881</span><span class="refDoi"> DOI: 10.2174/156802611796391276</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.2174%2F156802611796391276" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=21470171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVSmsL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=1872-1881&author=S.+L.+Garlandauthor=D.+E.+Gloriam&title=A+ligand%E2%80%99s+view+of+target+similarity%3A+chemogenomic+binding+site-directed+techniques+for+drug+discovery&doi=10.2174%2F156802611796391276"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A ligand's view of target similarity: chemogenomic binding site-directed techniques for drug discovery</span></div><div class="casAuthors">Garland, Stephen L.; Gloriam, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1872-1881</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  GPCR binding site-directed techniques are rapidly evolving into powerful tools for modern drug discovery.  Many of these approaches bridge chem. and biol., which are inseparable concepts in nature but are often treated as sep. worlds in drug discovery and science in general.  This review shows with several examples how focusing on the binding site(s) has a clear advantage when it comes to establishing sequence-correlated pharmacol. profiles.  By organizing and comparing sequence and structural data it is possible to "borrow" SAR from similar targets to increase the speed of lead-finding and, potentially, to produce ligands for previously intractable receptors.  Sequence motifs correlated with ligands can be applied in the design of target-specific focused libraries that are both efficient and cost-effective and should provide increased hit-rates over diversity screening.  Furthermore, in the optimization phase, the binding motif approach offers the possibility to identify quickly the most likely off-target candidates to be chosen for selectivity screening, as well as potentially characterizing those pockets which may best be exploited for improved selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9ppSbGEytS7Vg90H21EOLACvtfcHk0ljQUMFj1CM61g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVSmsL%252FL&md5=c7c9f29801172934eabc8cbf9e294574</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2174%2F156802611796391276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611796391276%26sid%3Dliteratum%253Aachs%26aulast%3DGarland%26aufirst%3DS.%2BL.%26aulast%3DGloriam%26aufirst%3DD.%2BE.%26atitle%3DA%2520ligand%25E2%2580%2599s%2520view%2520of%2520target%2520similarity%253A%2520chemogenomic%2520binding%2520site-directed%2520techniques%2520for%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26spage%3D1872%26epage%3D1881%26doi%3D10.2174%2F156802611796391276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Hibert, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trumpp-Kallmeyer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruinvels, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoflack, J.</span><span> </span><span class="NLM_article-title">Three-dimensional models of neurotransmitter G-binding protein-coupled receptors</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">8</span><span class="NLM_x">–</span> <span class="NLM_lpage">15</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1991&pages=8-15&author=M.+F.+Hibertauthor=S.+Trumpp-Kallmeyerauthor=A.+Bruinvelsauthor=J.+Hoflack&title=Three-dimensional+models+of+neurotransmitter+G-binding+protein-coupled+receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHibert%26aufirst%3DM.%2BF.%26aulast%3DTrumpp-Kallmeyer%26aufirst%3DS.%26aulast%3DBruinvels%26aufirst%3DA.%26aulast%3DHoflack%26aufirst%3DJ.%26atitle%3DThree-dimensional%2520models%2520of%2520neurotransmitter%2520G-binding%2520protein-coupled%2520receptors%26jtitle%3DMol.%2520Pharmacol.%26date%3D1991%26volume%3D40%26spage%3D8%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Trumpp-Kallmeyer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoflack, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruinvels, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hibert, M.</span><span> </span><span class="NLM_article-title">Modeling of G-protein-coupled receptors: application to dopamine, adrenaline, serotonin, acetylcholine, and mammalian opsin receptors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">3448</span><span class="NLM_x">–</span> <span class="NLM_lpage">3462</span><span class="refDoi"> DOI: 10.1021/jm00097a002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00097a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1992&pages=3448-3462&author=S.+Trumpp-Kallmeyerauthor=J.+Hoflackauthor=A.+Bruinvelsauthor=M.+Hibert&title=Modeling+of+G-protein-coupled+receptors%3A+application+to+dopamine%2C+adrenaline%2C+serotonin%2C+acetylcholine%2C+and+mammalian+opsin+receptors&doi=10.1021%2Fjm00097a002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm00097a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00097a002%26sid%3Dliteratum%253Aachs%26aulast%3DTrumpp-Kallmeyer%26aufirst%3DS.%26aulast%3DHoflack%26aufirst%3DJ.%26aulast%3DBruinvels%26aufirst%3DA.%26aulast%3DHibert%26aufirst%3DM.%26atitle%3DModeling%2520of%2520G-protein-coupled%2520receptors%253A%2520application%2520to%2520dopamine%252C%2520adrenaline%252C%2520serotonin%252C%2520acetylcholine%252C%2520and%2520mammalian%2520opsin%2520receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1992%26volume%3D35%26spage%3D3448%26epage%3D3462%26doi%3D10.1021%2Fjm00097a002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Klabunde, T.</span><span> </span><span class="NLM_article-title">Chemogenomic approaches to drug discovery: similar receptors bind similar ligands</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">152</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0707308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fsj.bjp.0707308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=17533415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFOjur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2007&pages=5-7&author=T.+Klabunde&title=Chemogenomic+approaches+to+drug+discovery%3A+similar+receptors+bind+similar+ligands&doi=10.1038%2Fsj.bjp.0707308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic approaches to drug discovery: similar receptors bind similar ligands</span></div><div class="casAuthors">Klabunde, T.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-7</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Within recent years, a paradigm shift from traditional receptor-specific studies to a cross-receptor view has taken place within pharmaceutical research to increase the efficiency of modern drug discovery.  Receptors are no longer viewed as single entities but grouped into sets of related proteins or receptor families that are explored in a systematic manner.  This interdisciplinary approach attempting to derive predictive links between the chem. structures of bioactive mols. and the receptors with which these mols. interact is referred to as chemogenomics.  Insights from chemogenomics are used for the rational compilation of screening sets and for the rational design and synthesis of directed chem. libraries to accelerate drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQeX3iyp_Z47Vg90H21EOLACvtfcHk0lihiVvtO7sqyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFOjur0%253D&md5=a103cdd189d2bad6d70e799aad869290</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0707308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0707308%26sid%3Dliteratum%253Aachs%26aulast%3DKlabunde%26aufirst%3DT.%26atitle%3DChemogenomic%2520approaches%2520to%2520drug%2520discovery%253A%2520similar%2520receptors%2520bind%2520similar%2520ligands%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2007%26volume%3D152%26spage%3D5%26epage%3D7%26doi%3D10.1038%2Fsj.bjp.0707308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Paolini, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapland, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Hoorn, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Global mapping of pharmacological space</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">805</span><span class="NLM_x">–</span> <span class="NLM_lpage">815</span><span class="refDoi"> DOI: 10.1038/nbt1228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnbt1228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=16841068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=805-815&author=G.+V.+Paoliniauthor=R.+H.+Shaplandauthor=W.+P.+van+Hoornauthor=J.+S.+Masonauthor=A.+L.+Hopkins&title=Global+mapping+of+pharmacological+space&doi=10.1038%2Fnbt1228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Global mapping of pharmacological space</span></div><div class="casAuthors">Paolini, Gaia V.; Shapland, Richard H. B.; van Hoorn, Willem P.; Mason, Jonathan S.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">805-815</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We present the global mapping of pharmacol. space by the integration of several vast sources of medicinal chem. structure-activity relationships (SAR) data.  Our comprehensive mapping of pharmacol. space enables us to identify confidently the human targets for which chem. tools and drugs have been discovered to date.  The integration of SAR data from diverse sources by unique canonical chem. structure, protein sequence and disease indication enables the construction of a ligand-target matrix to explore the global relationships between chem. structure and biol. targets.  Using the data matrix, we are able to catalog the links between proteins in chem. space as a polypharmacol. interaction network.  We demonstrate that probabilistic models can be used to predict pharmacol. from a large knowledge base.  The relationships between proteins, chem. structures and drug-like properties provide a framework for developing a probabilistic approach to drug discovery that can be exploited to increase research productivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV_CE47zZq8LVg90H21EOLACvtfcHk0lihiVvtO7sqyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D&md5=f559b34692cc903a1b503deb07030c5d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnbt1228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1228%26sid%3Dliteratum%253Aachs%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DShapland%26aufirst%3DR.%2BH.%26aulast%3Dvan%2BHoorn%26aufirst%3DW.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DGlobal%2520mapping%2520of%2520pharmacological%2520space%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26spage%3D805%26epage%3D815%26doi%3D10.1038%2Fnbt1228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armbruster, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernsberger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">206</span><span class="refDoi"> DOI: 10.1038/nbt1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197-206&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry&doi=10.1038%2Fnbt1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, α2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0lihiVvtO7sqyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D197%26epage%3D206%26doi%3D10.1038%2Fnbt1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laggner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbas, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hufeisen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuijer, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matos, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whaley, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glennon, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span> </span><span class="NLM_article-title">Predicting new molecular targets for known drugs</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">462</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">181</span><span class="refDoi"> DOI: 10.1038/nature08506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnature08506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=19881490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=175-181&author=M.+J.+Keiserauthor=V.+Setolaauthor=J.+J.+Irwinauthor=C.+Laggnerauthor=A.+I.+Abbasauthor=S.+J.+Hufeisenauthor=N.+H.+Jensenauthor=M.+B.+Kuijerauthor=R.+C.+Matosauthor=T.+B.+Tranauthor=R.+Whaleyauthor=R.+A.+Glennonauthor=J.+Hertauthor=K.+L.+Thomasauthor=D.+D.+Edwardsauthor=B.+K.+Shoichetauthor=B.+L.+Roth&title=Predicting+new+molecular+targets+for+known+drugs&doi=10.1038%2Fnature08506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting new molecular targets for known drugs</span></div><div class="casAuthors">Keiser, Michael J.; Setola, Vincent; Irwin, John J.; Laggner, Christian; Abbas, Atheir I.; Hufeisen, Sandra J.; Jensen, Niels H.; Kuijer, Michael B.; Matos, Roberto C.; Tran, Thuy B.; Whaley, Ryan; Glennon, Richard A.; Hert, Jerome; Thomas, Kelan L. H.; Edwards, Douglas D.; Shoichet, Brian K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7270</span>),
    <span class="NLM_cas:pages">175-181</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although drugs are intended to be selective, at least some bind to several physiol. targets, explaining side effects and efficacy.  Because many drug-target combinations exist, it would be useful to explore possible interactions computationally.  Here we compared 3665 US Food and Drug Administration (FDA)-approved and investigational drugs against hundreds of targets, defining each target by its ligands.  Chem. similarities between drugs and ligand sets predicted thousands of unanticipated assocns.  Thirty were tested exptl., including the antagonism of the β1 receptor by the transporter inhibitor Prozac, the inhibition of the 5-hydroxytryptamine (5-HT) transporter by the ion channel drug Vadilex, and antagonism of the histamine H4 receptor by the enzyme inhibitor Rescriptor.  Overall, 23 new drug-target assocns. were confirmed, five of which were potent (<100 nM).  The physiol. relevance of one, the drug N,N-dimethyltryptamine (DMT) on serotonergic receptors, was confirmed in a knockout mouse.  The chem. similarity approach is systematic and comprehensive, and may suggest side-effects and new indications for many drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozd-8cCXUskrVg90H21EOLACvtfcHk0lj6Vj9YxN6JZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO&md5=974e65f5d5b4dabde1a463c429215546</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature08506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08506%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DLaggner%26aufirst%3DC.%26aulast%3DAbbas%26aufirst%3DA.%2BI.%26aulast%3DHufeisen%26aufirst%3DS.%2BJ.%26aulast%3DJensen%26aufirst%3DN.%2BH.%26aulast%3DKuijer%26aufirst%3DM.%2BB.%26aulast%3DMatos%26aufirst%3DR.%2BC.%26aulast%3DTran%26aufirst%3DT.%2BB.%26aulast%3DWhaley%26aufirst%3DR.%26aulast%3DGlennon%26aufirst%3DR.%2BA.%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DK.%2BL.%26aulast%3DEdwards%26aufirst%3DD.%2BD.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DPredicting%2520new%2520molecular%2520targets%2520for%2520known%2520drugs%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D175%26epage%3D181%26doi%3D10.1038%2Fnature08506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Gregori-Puigjané, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestres, J.</span><span> </span><span class="NLM_article-title">A ligand-based approach to mining the chemogenomic space of drugs</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">676</span><span class="refDoi"> DOI: 10.2174/138620708785739952</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.2174%2F138620708785739952" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=18795886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivFentbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2008&pages=669-676&author=E.+Gregori-Puigjan%C3%A9author=J.+Mestres&title=A+ligand-based+approach+to+mining+the+chemogenomic+space+of+drugs&doi=10.2174%2F138620708785739952"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A ligand-based approach to mining the chemogenomic space of drugs</span></div><div class="casAuthors">Gregori-Puigjane, Elisabet; Mestres, Jordi</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">669-676</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The practical implementation and validation of a ligand-based approach to mining the chemogenomic space of drugs is presented and applied to the in silico target profiling of 767 drugs against 684 targets of therapeutic relevance.  The results reveal that drugs targeting aminergic G protein-coupled receptors (GPCRs) show the most promiscuous pharmacol. profiles.  The detection of cross-pharmacologies between aminergic GPCRs and the opioid, sigma, NMDA, and 5-HT3 receptors aggravate the potential promiscuity of those drugs, predominantly including analgesics, antidepressants, and antipsychotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3IqM_DIwJhbVg90H21EOLACvtfcHk0lj6Vj9YxN6JZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivFentbc%253D&md5=02d2db8cfa6711fb16ef362c99c5dbd0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.2174%2F138620708785739952&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620708785739952%26sid%3Dliteratum%253Aachs%26aulast%3DGregori-Puigjan%25C3%25A9%26aufirst%3DE.%26aulast%3DMestres%26aufirst%3DJ.%26atitle%3DA%2520ligand-based%2520approach%2520to%2520mining%2520the%2520chemogenomic%2520space%2520of%2520drugs%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2008%26volume%3D11%26spage%3D669%26epage%3D676%26doi%3D10.2174%2F138620708785739952" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Faure, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubus, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijjaali, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morlière, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barberan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petitet, F.</span><span> </span><span class="NLM_article-title">Knowledge-based analysis of multi-potent G-protein coupled receptors ligands</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">5708</span><span class="NLM_x">–</span> <span class="NLM_lpage">5717</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2010.09.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2Fj.ejmech.2010.09.027" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2010&pages=5708-5717&author=P.+Faureauthor=E.+Dubusauthor=I.+Ijjaaliauthor=C.+Morli%C3%A8reauthor=O.+Barberanauthor=F.+Petitet&title=Knowledge-based+analysis+of+multi-potent+G-protein+coupled+receptors+ligands&doi=10.1016%2Fj.ejmech.2010.09.027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2010.09.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2010.09.027%26sid%3Dliteratum%253Aachs%26aulast%3DFaure%26aufirst%3DP.%26aulast%3DDubus%26aufirst%3DE.%26aulast%3DIjjaali%26aufirst%3DI.%26aulast%3DMorli%25C3%25A8re%26aufirst%3DC.%26aulast%3DBarberan%26aufirst%3DO.%26aulast%3DPetitet%26aufirst%3DF.%26atitle%3DKnowledge-based%2520analysis%2520of%2520multi-potent%2520G-protein%2520coupled%2520receptors%2520ligands%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2010%26volume%3D45%26spage%3D5708%26epage%3D5717%26doi%3D10.1016%2Fj.ejmech.2010.09.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Briansó, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrascosa, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestres, J.</span><span> </span><span class="NLM_article-title">Cross-pharmacology analysis of G protein-coupled receptors</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1956</span><span class="NLM_x">–</span> <span class="NLM_lpage">1963</span><span class="refDoi"> DOI: 10.2174/156802611796391285</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.2174%2F156802611796391285" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=21851335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVSmsLzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=1956-1963&author=F.+Brians%C3%B3author=M.+C.+Carrascosaauthor=T.+I.+Opreaauthor=J.+Mestres&title=Cross-pharmacology+analysis+of+G+protein-coupled+receptors&doi=10.2174%2F156802611796391285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Cross-pharmacology analysis of G protein-coupled receptors</span></div><div class="casAuthors">Brianso, Ferran; Carrascosa, Maria C.; Oprea, Tudor I.; Mestres, Jordi</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1956-1963</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The degree of applicability of chemogenomic approaches to protein families depends on the accuracy and completeness of pharmacol. data and the corresponding level of pharmacol. similarity obsd. among their protein members.  The recent public domain availability of pharmacol. data for thousands of small mols. on 204 G protein-coupled receptors (GPCRs) provides a firm basis for an in-depth cross-pharmacol. anal. of this superfamily.  The no. of protein targets included in the cross-pharmacol. profile of the different GPCRs changes significantly upon varying the ligand similarity and binding affinity criteria.  However, with the exception of muscarinic receptors, aminergic GPCRs distinguish themselves from the rest of the members in the family by their remarkably high levels of pharmacol. similarity among them.  Clusters of non-GPCR targets related by cross-pharmacol. with particular GPCRs are identified and the implications for unwanted side-effects, as well as for repurposing opportunities, discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcxMaSMU_YxLVg90H21EOLACvtfcHk0lg9vZw1YlWJjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVSmsLzO&md5=7805626c0844e84d16b56ed953823486</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.2174%2F156802611796391285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611796391285%26sid%3Dliteratum%253Aachs%26aulast%3DBrians%25C3%25B3%26aufirst%3DF.%26aulast%3DCarrascosa%26aufirst%3DM.%2BC.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DMestres%26aufirst%3DJ.%26atitle%3DCross-pharmacology%2520analysis%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26spage%3D1956%26epage%3D1963%26doi%3D10.2174%2F156802611796391285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Weill, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">J Development and validation of a novel protein-ligand fingerprint to mine chemogenomic space: application to G protein-coupled receptors and their ligands</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1049</span><span class="NLM_x">–</span> <span class="NLM_lpage">1062</span><span class="refDoi"> DOI: 10.1021/ci800447g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800447g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsVKktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=1049-1062&author=N.+Weillauthor=D.+Rognan&title=J+Development+and+validation+of+a+novel+protein-ligand+fingerprint+to+mine+chemogenomic+space%3A+application+to+G+protein-coupled+receptors+and+their+ligands&doi=10.1021%2Fci800447g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Validation of a Novel Protein-Ligand Fingerprint To Mine Chemogenomic Space: Application to G Protein-Coupled Receptors and Their Ligands</span></div><div class="casAuthors">Weill, Nathanael; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1049-1062</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The present study introduces a novel low-dimensionality fingerprint encoding both ligand and target properties which is suitable to mine protein-ligand chemogenomic space.  Whereas ligand properties have been represented by std. descriptors, protein cavities are encoded by a fixed length bit string describing pharmacophoric properties of a definite no. of binding site residues.  In order to simplify the cavity fingerprint, the concept was applied here to a unique family of targets (G protein-coupled receptors) with a homogeneous cavity description.  Particular attention was given to set up data sets of really diverse protein-ligand pairs covering as exhaustively as possible both ligand and target spaces.  Several machine learning classification algorithms were trained on two sets of roughly 200,000 receptor-ligand fingerprints with a different definition of inactive decoys.  Cross-validated models show excellent precision (>0.9) in distinguishing true from false pairs with a particular preference for support vector machine classifiers.  When applied to two external test sets of GPCR ligands, the most predictive models were not those performing the best in the previous cross-validation.  The ability to recover true GPCR ligands (ligand prediction mode) or true GPCRs (receptor prediction mode) depends on multiple parameters: the mol. complexity of the ligands, the chem. space from which ligand decoys are selected to generate false protein-ligand pairs, and the target space under consideration.  In most cases, predicting ligands is easier than predicting receptors.  Although receptor profiling is possible, it probably requires a more detailed description of the ligand-binding site.  Noteworthy, protein-ligand fingerprints outperform the corresponding ligand fingerprints in mining the GPCR-ligand space.  Since they can be applied to a much larger no. of receptors than ligand-based fingerprints, protein-ligand fingerprints represent a novel and promising way to directly screen protein-ligand pairs in chemogenomic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHYTjaMcCFi7Vg90H21EOLACvtfcHk0lg9vZw1YlWJjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsVKktLs%253D&md5=afd7e1e8beb8d28b68ea7fe574e4d32e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci800447g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800447g%26sid%3Dliteratum%253Aachs%26aulast%3DWeill%26aufirst%3DN.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DJ%2520Development%2520and%2520validation%2520of%2520a%2520novel%2520protein-ligand%2520fingerprint%2520to%2520mine%2520chemogenomic%2520space%253A%2520application%2520to%2520G%2520protein-coupled%2520receptors%2520and%2520their%2520ligands%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D1049%26epage%3D1062%26doi%3D10.1021%2Fci800447g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhne, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">A structural chemogenomics analysis of aminergic GPCRs: lessons for histamine receptor ligand design.</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">170</span><span class="NLM_x">, </span> <span class="NLM_fpage">101</span><span class="NLM_x">–</span> <span class="NLM_lpage">126</span><span class="refDoi"> DOI: 10.1111/bph.12248</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1111%2Fbph.12248" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=23713847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlSgtbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2013&pages=101-126&author=A.+J.+Kooistraauthor=S.+Kuhneauthor=I.+J.+de+Eschauthor=R.+Leursauthor=C.+de+Graaf&title=A+structural+chemogenomics+analysis+of+aminergic+GPCRs%3A+lessons+for+histamine+receptor+ligand+design.&doi=10.1111%2Fbph.12248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A structural chemogenomics analysis of aminergic GPCRs: lessons for histamine receptor ligand design</span></div><div class="casAuthors">Kooistra, A. J.; Kuhne, S.; de Esch, I. J. P.; Leurs, R.; de Graaf, C.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">101-126</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Background and Purpose Chemogenomics focuses on the discovery of new connections between chem. and biol. space leading to the discovery of new protein targets and biol. active mols.  G-protein coupled receptors (GPCRs) are a particularly interesting protein family for chemogenomics studies because there is an overwhelming amt. of ligand binding affinity data available.  The increasing no. of aminergic GPCR crystal structures now for the first time allows the integration of chemogenomics studies with high-resoln. structural analyses of GPCR-ligand complexes.  Exptl. Approach In this study, we have combined ligand affinity data, receptor mutagenesis studies, and amino acid sequence analyses to high-resoln. structural analyses of (hist)aminergic GPCR-ligand interactions.  This integrated structural chemogenomics anal. is used to more accurately describe the mol. and structural determinants of ligand affinity and selectivity in different key binding regions of the crystd. aminergic GPCRs, and histamine receptors in particular.  Key Results Our investigations highlight interesting correlations and differences between ligand similarity and ligand binding site similarity of different aminergic receptors.  Apparent discrepancies can be explained by combining detailed anal. of crystd. or predicted protein-ligand binding modes, receptor mutation studies, and ligand structure-selectivity relationships that identify local differences in essential pharmacophore features in the ligand binding sites of different receptors.  Conclusions and Implications We have performed structural chemogenomics studies that identify links between (hist)aminergic receptor ligands and their binding sites and binding modes.  This knowledge can be used to identify structure-selectivity relationships that increase our understanding of ligand binding to (hist)aminergic receptors and hence can be used in future GPCR ligand discovery and design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxr6kq48O11rVg90H21EOLACvtfcHk0ljCLZiERZqGXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlSgtbnE&md5=08ae55b9565885165e76d98e2db1befa</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1111%2Fbph.12248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.12248%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DKuhne%26aufirst%3DS.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DA%2520structural%2520chemogenomics%2520analysis%2520of%2520aminergic%2520GPCRs%253A%2520lessons%2520for%2520histamine%2520receptor%2520ligand%2520design.%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2013%26volume%3D170%26spage%3D101%26epage%3D126%26doi%3D10.1111%2Fbph.12248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Heilker, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tautermann, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bieler, M.</span><span> </span><span class="NLM_article-title">G-protein-coupled receptor-focused drug discovery using a target class platform approach</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">240</span><span class="refDoi"> DOI: 10.1016/j.drudis.2008.11.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2Fj.drudis.2008.11.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=19121411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjt1Shsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=231-240&author=R.+Heilkerauthor=M.+Wolffauthor=C.+S.+Tautermannauthor=M.+Bieler&title=G-protein-coupled+receptor-focused+drug+discovery+using+a+target+class+platform+approach&doi=10.1016%2Fj.drudis.2008.11.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">G-protein-coupled receptor-focused drug discovery using a target class platform approach</span></div><div class="casAuthors">Heilker, Ralf; Wolff, Michael; Tautermann, Christofer S.; Bieler, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5/6</span>),
    <span class="NLM_cas:pages">231-240</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In recent years, several large pharmaceutical companies have taken a novel approach to drug discovery biol. and chem. in that they channel their efforts with respect to particular target classes, such as G-protein-coupled receptors (GPCRs), toward dedicated, specialized teams.  Benefits of such an organizational structure are the prospects of establishing several target-family-specific exptl. techniques and skill sets, thereby enabling a comprehensive functional profiling of drug candidates in different pharmacol. respects.  In this context, the recently increased no. of reports on GPCR ligand-biased signaling has further spurred the efforts in the pharmaceutical industry toward broader biol. characterization of the test compds., for example employing high-content screening to analyze different GPCR ligand-induced signaling pathways.  The knowledge of the disease-relevant functional properties of the small mol. GPCR ligands enables target-specific chem. optimization and GPCR-subclass-directed library design.  In the case of GPCRs, where little - although at present slowly expanding - structural information on the targets is available, the modeling of GPCR structures crucially depends on biol. validation (typically supported by site-directed mutagenesis of the GPCR ligand binding site).  In this review, we aim to recapitulate efforts in the pharmaceutical industry to address GPCR-directed drug discovery in a target-class-directed platform approach: establishing GPCR-specific biol. assay panels and creating computational chem. methods for finding and optimizing small mols. modulating the activity of GPCRs.  GPCR-target-class-directed platform approach: establishment of GPCR-specific biol. assay panels and creation of computational chem. methods for finding and optimizing small mols. modulating the activity of GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOQSxaIoD2G7Vg90H21EOLACvtfcHk0ljCLZiERZqGXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjt1Shsbk%253D&md5=3943a8e246fef8aac1191e4d6c2b3788</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2008.11.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2008.11.011%26sid%3Dliteratum%253Aachs%26aulast%3DHeilker%26aufirst%3DR.%26aulast%3DWolff%26aufirst%3DM.%26aulast%3DTautermann%26aufirst%3DC.%2BS.%26aulast%3DBieler%26aufirst%3DM.%26atitle%3DG-protein-coupled%2520receptor-focused%2520drug%2520discovery%2520using%2520a%2520target%2520class%2520platform%2520approach%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D231%26epage%3D240%26doi%3D10.1016%2Fj.drudis.2008.11.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Wichard, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ter Laak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krause, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinrich, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kühne, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleinau, G.</span><span> </span><span class="NLM_article-title">Chemogenomic analysis of G-protein coupled receptors and their ligands deciphers locks and keys governing diverse aspects of signalling.</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e16811</span><span class="refDoi"> DOI: 10.1371/journal.pone.0016811</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1371%2Fjournal.pone.0016811" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e16811&author=J.+D.+Wichardauthor=A.+Ter+Laakauthor=G.+Krauseauthor=N.+Heinrichauthor=R.+K%C3%BChneauthor=G.+Kleinau&title=Chemogenomic+analysis+of+G-protein+coupled+receptors+and+their+ligands+deciphers+locks+and+keys+governing+diverse+aspects+of+signalling.&doi=10.1371%2Fjournal.pone.0016811"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0016811&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0016811%26sid%3Dliteratum%253Aachs%26aulast%3DWichard%26aufirst%3DJ.%2BD.%26aulast%3DTer%2BLaak%26aufirst%3DA.%26aulast%3DKrause%26aufirst%3DG.%26aulast%3DHeinrich%26aufirst%3DN.%26aulast%3DK%25C3%25BChne%26aufirst%3DR.%26aulast%3DKleinau%26aufirst%3DG.%26atitle%3DChemogenomic%2520analysis%2520of%2520G-protein%2520coupled%2520receptors%2520and%2520their%2520ligands%2520deciphers%2520locks%2520and%2520keys%2520governing%2520diverse%2520aspects%2520of%2520signalling.%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De16811%26doi%3D10.1371%2Fjournal.pone.0016811" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Van der Horst, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peironcely, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beukers, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Vlijmen, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emmerich, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okuno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span> </span><span class="NLM_article-title">A novel chemogenomics analysis of G protein-coupled receptors (GPCRs) and their ligands: a potential strategy for receptor de-orphanization</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">316</span><span class="refDoi"> DOI: 10.1186/1471-2105-11-316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1186%2F1471-2105-11-316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=20537162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A280%3ADC%252BC3cnltVSgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=316&author=E.+Van+der+Horstauthor=J.+E.+Peironcelyauthor=A.+P.+Ijzermanauthor=M.+W.+Beukersauthor=J.+R.+Laneauthor=H.+W.+van+Vlijmenauthor=M.+T.+Emmerichauthor=Y.+Okunoauthor=A.+Bender&title=A+novel+chemogenomics+analysis+of+G+protein-coupled+receptors+%28GPCRs%29+and+their+ligands%3A+a+potential+strategy+for+receptor+de-orphanization&doi=10.1186%2F1471-2105-11-316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A novel chemogenomics analysis of G protein-coupled receptors (GPCRs) and their ligands: a potential strategy for receptor de-orphanization</span></div><div class="casAuthors">van der Horst Eelke; Peironcely Julio E; Ijzerman Adriaan P; Beukers Margot W; Lane Jonathan R; van Vlijmen Herman W T; Emmerich Michael T M; Okuno Yasushi; Bender Andreas</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">316</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  G protein-coupled receptors (GPCRs) represent a family of well-characterized drug targets with significant therapeutic value.  Phylogenetic classifications may help to understand the characteristics of individual GPCRs and their subtypes.  Previous phylogenetic classifications were all based on the sequences of receptors, adding only minor information about the ligand binding properties of the receptors.  In this work, we compare a sequence-based classification of receptors to a ligand-based classification of the same group of receptors, and evaluate the potential to use sequence relatedness as a predictor for ligand interactions thus aiding the quest for ligands of orphan receptors.  RESULTS:  We present a classification of GPCRs that is purely based on their ligands, complementing sequence-based phylogenetic classifications of these receptors.  Targets were hierarchically classified into phylogenetic trees, for both sequence space and ligand (substructure) space.  The overall organization of the sequence-based tree and substructure-based tree was similar; in particular, the adenosine receptors cluster together as well as most peptide receptor subtypes (e.g. opioid, somatostatin) and adrenoceptor subtypes.  In ligand space, the prostanoid and cannabinoid receptors are more distant from the other targets, whereas the tachykinin receptors, the oxytocin receptor, and serotonin receptors are closer to the other targets, which is indicative for ligand promiscuity.  In 93% of the receptors studied, de-orphanization of a simulated orphan receptor using the ligands of related receptors performed better than random (AUC > 0.5) and for 35% of receptors de-orphanization performance was good (AUC > 0.7).  CONCLUSIONS:  We constructed a phylogenetic classification of GPCRs that is solely based on the ligands of these receptors.  The similarities and differences with traditional sequence-based classifications were investigated: our ligand-based classification uncovers relationships among GPCRs that are not apparent from the sequence-based classification.  This will shed light on potential cross-reactivity of GPCR ligands and will aid the design of new ligands with the desired activity profiles.  In addition, we linked the ligand-based classification with a ligand-focused sequence-based classification described in literature and proved the potential of this method for de-orphanization of GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTvnKpagfv6Gkg6wdEZmT2rfW6udTcc2eY1yodx5ZgRmLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cnltVSgsA%253D%253D&md5=28e6a470f88c58f2f94ae170bd45f243</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-11-316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-11-316%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2Bder%2BHorst%26aufirst%3DE.%26aulast%3DPeironcely%26aufirst%3DJ.%2BE.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DBeukers%26aufirst%3DM.%2BW.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%2BW.%26aulast%3DEmmerich%26aufirst%3DM.%2BT.%26aulast%3DOkuno%26aufirst%3DY.%26aulast%3DBender%26aufirst%3DA.%26atitle%3DA%2520novel%2520chemogenomics%2520analysis%2520of%2520G%2520protein-coupled%2520receptors%2520%2528GPCRs%2529%2520and%2520their%2520ligands%253A%2520a%2520potential%2520strategy%2520for%2520receptor%2520de-orphanization%26jtitle%3DBMC%2520Bioinf.%26date%3D2010%26volume%3D11%26spage%3D316%26doi%3D10.1186%2F1471-2105-11-316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Lin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sassano, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">A pharmacological organization of G protein-coupled receptors</span> <span class="citation_source-journal">Nat. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">140</span><span class="NLM_x">–</span> <span class="NLM_lpage">146</span><span class="refDoi"> DOI: 10.1038/nmeth.2324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnmeth.2324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=23291723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjslelsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=140-146&author=H.+Linauthor=M.+F.+Sassanoauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=A+pharmacological+organization+of+G+protein-coupled+receptors&doi=10.1038%2Fnmeth.2324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A pharmacological organization of G protein-coupled receptors</span></div><div class="casAuthors">Lin, Henry; Sassano, Maria F.; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">140-146</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein classification typically uses structural, sequence or functional similarity.  Here we introduce an orthogonal method that organizes proteins by ligand similarity, focusing on the class A G-protein-coupled receptor (GPCR) protein family.  Comparing a ligand-based dendrogram to a sequence-based one, we identified GPCRs that were distantly linked by sequence but were neighbors by ligand similarity.  Exptl. testing of the ligands predicted to link three of these new pairs confirmed the predicted assocn., with potencies ranging from low nanomolar to low micromolar.  We also predicted hundreds of non-GPCRs closely related to GPCRs by ligand similarity and confirmed several cases exptl.  Ligand similarities among these targets may reflect the conservation of identical ligands among unrelated receptors, which signal in different time domains.  Our method integrates these apparently disparate receptors into chem. coherent circuits and suggests which of these receptors may be targeted by individual ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqx4gVKRHxC8bVg90H21EOLACvtfcHk0liWdVysyEl4YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjslelsQ%253D%253D&md5=eaec5942489a3337effbbc419a69f17e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.2324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.2324%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520pharmacological%2520organization%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DNat.%2520Methods%26date%3D2013%26volume%3D10%26spage%3D140%26epage%3D146%26doi%3D10.1038%2Fnmeth.2324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Mestres, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregori-Puigjané, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valverde, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solé, R. V.</span><span> </span><span class="NLM_article-title">Data completeness-the Achilles heel of drug-target networks</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">983</span><span class="NLM_x">–</span> <span class="NLM_lpage">984</span><span class="refDoi"> DOI: 10.1038/nbt0908-983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnbt0908-983" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=983-984&author=J.+Mestresauthor=E.+Gregori-Puigjan%C3%A9author=S.+Valverdeauthor=R.+V.+Sol%C3%A9&title=Data+completeness-the+Achilles+heel+of+drug-target+networks&doi=10.1038%2Fnbt0908-983"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnbt0908-983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt0908-983%26sid%3Dliteratum%253Aachs%26aulast%3DMestres%26aufirst%3DJ.%26aulast%3DGregori-Puigjan%25C3%25A9%26aufirst%3DE.%26aulast%3DValverde%26aufirst%3DS.%26aulast%3DSol%25C3%25A9%26aufirst%3DR.%2BV.%26atitle%3DData%2520completeness-the%2520Achilles%2520heel%2520of%2520drug-target%2520networks%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D983%26epage%3D984%26doi%3D10.1038%2Fnbt0908-983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Langelaan, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngweniform, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rainey, J. K.</span><span> </span><span class="NLM_article-title">Biophysical characterization of G-protein coupled receptor-peptide ligand binding</span> <span class="citation_source-journal">Biochem. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">89</span><span class="NLM_x">, </span> <span class="NLM_fpage">98</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1139/O10-142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1139%2FO10-142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=21455262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvFeksrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2011&pages=98-105&author=D.+N.+Langelaanauthor=P.+Ngweniformauthor=J.+K.+Rainey&title=Biophysical+characterization+of+G-protein+coupled+receptor-peptide+ligand+binding&doi=10.1139%2FO10-142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Biophysical characterization of G-protein coupled receptor-peptide ligand binding</span></div><div class="casAuthors">Langelaan, David N.; Ngweniform, Pascaline; Rainey, Jan K.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry and Cell Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">98-105</span>CODEN:
                <span class="NLM_cas:coden">BCBIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0829-8211</span>.
    
            (<span class="NLM_cas:orgname">NRC Research Press</span>)
        </div><div class="casAbstract">A review.  G-protein coupled receptors (GPCRs) are ubiquitous membrane proteins allowing intracellular responses to extracellular factors that range from photons of light to small mols. to proteins.  Despite extensive exploitation of GPCRs as therapeutic targets, biophys. characterization of GPCR-ligand interactions remains challenging.  In this minireview, the authors focus on techniques that have been successfully used for structural and biophys. characterization of peptide ligands binding to their cognate GPCRs.  The techniques reviewed include soln.-state NMR spectroscopy, solid-state NMR, x-ray diffraction, fluorescence spectroscopy and single-mol. fluorescence methods, flow cytometry, surface plasmon resonance, isothermal titrn. calorimetry, and at. force microscopy.  The goal herein is to provide a cohesive starting point to allow selection of techniques appropriate to the elucidation of a given GPCR-peptide interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOuaZINHw3v7Vg90H21EOLACvtfcHk0liWdVysyEl4YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjvFeksrc%253D&md5=bab689a93ab74217c5ccdcb8bc143707</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1139%2FO10-142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1139%252FO10-142%26sid%3Dliteratum%253Aachs%26aulast%3DLangelaan%26aufirst%3DD.%2BN.%26aulast%3DNgweniform%26aufirst%3DP.%26aulast%3DRainey%26aufirst%3DJ.%2BK.%26atitle%3DBiophysical%2520characterization%2520of%2520G-protein%2520coupled%2520receptor-peptide%2520ligand%2520binding%26jtitle%3DBiochem.%2520Cell%2520Biol.%26date%3D2011%26volume%3D89%26spage%3D98%26epage%3D105%26doi%3D10.1139%2FO10-142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Lohse, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuber, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffmann, C.</span><span> </span><span class="NLM_article-title">Fluorescence/bioluminescence resonance energy transfer techniques to study G-protein-coupled receptor activation and signaling</span> <span class="citation_source-journal">Pharmacol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">299</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1124/pr.110.004309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1124%2Fpr.110.004309" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=22407612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC38XmslKktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2012&pages=299-336&author=M.+J.+Lohseauthor=S.+Nuberauthor=C.+Hoffmann&title=Fluorescence%2Fbioluminescence+resonance+energy+transfer+techniques+to+study+G-protein-coupled+receptor+activation+and+signaling&doi=10.1124%2Fpr.110.004309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence/bioluminescence resonance energy transfer techniques to study G-protein-coupled receptor activation and signaling</span></div><div class="casAuthors">Lohse, Martin J.; Nuber, Susanne; Hoffmann, Carsten</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">299-336</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Fluorescence and bioluminescence resonance energy transfer (FRET and BRETT) techniques allow the sensitive monitoring of distances between two labels at the nanometer scale.  Depending on the placement of the labels, this permits the anal. of conformational changes within a single protein (for example of a receptor) or the monitoring of protein-protein interactions (for example, between receptors and G-protein subunits).  Over the past decade, numerous such techniques have been developed to monitor the activation and signaling of G-protein-coupled receptors (GPCRs) in both the purified, reconstituted state and in intact cells.  These techniques span the entire spectrum from ligand binding to the receptors down to intracellular second messengers.  They allow the detn. and the visualization of signaling processes with high temporal and spatial resoln.  With these techniques, it has been demonstrated that GPCR signals may show spatial and temporal patterning.  In particular, evidence has been provided for spatial compartmentalization of GPCRs and their signals in intact cells and for distinct physiol. consequences of such spatial patterning.  We review here the FRET and BRET technologies that have been developed for G-protein-coupled receptors and their signaling proteins (G-proteins, effectors) and the concepts that result from such expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzZxuO8ZQ_D7Vg90H21EOLACvtfcHk0liBT-aqrd9aWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmslKktr8%253D&md5=76146f70e0e0776aa1d10dae884462aa</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1124%2Fpr.110.004309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.110.004309%26sid%3Dliteratum%253Aachs%26aulast%3DLohse%26aufirst%3DM.%2BJ.%26aulast%3DNuber%26aufirst%3DS.%26aulast%3DHoffmann%26aufirst%3DC.%26atitle%3DFluorescence%252Fbioluminescence%2520resonance%2520energy%2520transfer%2520techniques%2520to%2520study%2520G-protein-coupled%2520receptor%2520activation%2520and%2520signaling%26jtitle%3DPharmacol.%2520Rev.%26date%3D2012%26volume%3D64%26spage%3D299%26epage%3D336%26doi%3D10.1124%2Fpr.110.004309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Margathe, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iturrioz, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvear-Perez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsol, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riché, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chabane, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tounsi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhry, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heissler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hibert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Llorens-Cortes, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, D.</span><span> </span><span class="NLM_article-title">Structure -activity relationship studies toward the discovery of selective apelin receptor agonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">2908</span><span class="NLM_x">–</span> <span class="NLM_lpage">2919</span><span class="refDoi"> DOI: 10.1021/jm401789v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401789v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=2908-2919&author=J.+F.+Margatheauthor=X.+Iturriozauthor=R.+Alvear-Perezauthor=C.+Marsolauthor=S.+Rich%C3%A9author=H.+Chabaneauthor=N.+Tounsiauthor=M.+Kuhryauthor=D.+Heisslerauthor=M.+Hibertauthor=C.+Llorens-Cortesauthor=D.+Bonnet&title=Structure+-activity+relationship+studies+toward+the+discovery+of+selective+apelin+receptor+agonists&doi=10.1021%2Fjm401789v"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm401789v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401789v%26sid%3Dliteratum%253Aachs%26aulast%3DMargathe%26aufirst%3DJ.%2BF.%26aulast%3DIturrioz%26aufirst%3DX.%26aulast%3DAlvear-Perez%26aufirst%3DR.%26aulast%3DMarsol%26aufirst%3DC.%26aulast%3DRich%25C3%25A9%26aufirst%3DS.%26aulast%3DChabane%26aufirst%3DH.%26aulast%3DTounsi%26aufirst%3DN.%26aulast%3DKuhry%26aufirst%3DM.%26aulast%3DHeissler%26aufirst%3DD.%26aulast%3DHibert%26aufirst%3DM.%26aulast%3DLlorens-Cortes%26aufirst%3DC.%26aulast%3DBonnet%26aufirst%3DD.%26atitle%3DStructure%2520-activity%2520relationship%2520studies%2520toward%2520the%2520discovery%2520of%2520selective%2520apelin%2520receptor%2520agonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D2908%26epage%3D2919%26doi%3D10.1021%2Fjm401789v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Loison, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cottet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orcel, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adihou, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahmeh, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamarque, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hibert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durroux, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouillac, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, D.</span><span> </span><span class="NLM_article-title">Selective fluorescent nonpeptidic antagonists for vasopressin V1A GPCR: application to ligand screening and oligomerization assays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">8588</span><span class="NLM_x">–</span> <span class="NLM_lpage">8602</span><span class="refDoi"> DOI: 10.1021/jm3006146</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3006146" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=8588-8602&author=S.+Loisonauthor=M.+Cottetauthor=H.+Orcelauthor=H.+Adihouauthor=R.+Rahmehauthor=L.+Lamarqueauthor=E.+Trinquetauthor=E.+Kellenbergerauthor=M.+Hibertauthor=T.+Durrouxauthor=B.+Mouillacauthor=D.+Bonnet&title=Selective+fluorescent+nonpeptidic+antagonists+for+vasopressin+V1A+GPCR%3A+application+to+ligand+screening+and+oligomerization+assays&doi=10.1021%2Fjm3006146"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm3006146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3006146%26sid%3Dliteratum%253Aachs%26aulast%3DLoison%26aufirst%3DS.%26aulast%3DCottet%26aufirst%3DM.%26aulast%3DOrcel%26aufirst%3DH.%26aulast%3DAdihou%26aufirst%3DH.%26aulast%3DRahmeh%26aufirst%3DR.%26aulast%3DLamarque%26aufirst%3DL.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DHibert%26aufirst%3DM.%26aulast%3DDurroux%26aufirst%3DT.%26aulast%3DMouillac%26aufirst%3DB.%26aulast%3DBonnet%26aufirst%3DD.%26atitle%3DSelective%2520fluorescent%2520nonpeptidic%2520antagonists%2520for%2520vasopressin%2520V1A%2520GPCR%253A%2520application%2520to%2520ligand%2520screening%2520and%2520oligomerization%2520assays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D8588%26epage%3D8602%26doi%3D10.1021%2Fjm3006146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Daval, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hibert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galzi, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ilien, B.</span><span> </span><span class="NLM_article-title">Fluorescent derivatives of AC-42 to probe bitopic orthosteric/allosteric binding mechanisms on muscarinic M1 receptors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">2125</span><span class="NLM_x">–</span> <span class="NLM_lpage">2143</span><span class="refDoi"> DOI: 10.1021/jm201348t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201348t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC38XitF2nu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=2125-2143&author=S.+B.+Davalauthor=C.+Valantauthor=D.+Bonnetauthor=E.+Kellenbergerauthor=M.+Hibertauthor=J.+L.+Galziauthor=B.+Ilien&title=Fluorescent+derivatives+of+AC-42+to+probe+bitopic+orthosteric%2Fallosteric+binding+mechanisms+on+muscarinic+M1+receptors&doi=10.1021%2Fjm201348t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescent Derivatives of AC-42 To Probe Bitopic Orthosteric/Allosteric Binding Mechanisms on Muscarinic M1 Receptors</span></div><div class="casAuthors">Daval, Sandrine B.; Valant, Celine; Bonnet, Dominique; Kellenberger, Esther; Hibert, Marcel; Galzi, Jean-Luc; Ilien, Brigitte</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2125-2143</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two fluorescent derivs. of the M1 muscarinic selective agonist AC-42 were synthesized by coupling the lissamine rhodamine B fluorophore (in ortho and para positions) to AC42-NH2.  This precursor, prepd. according to an original seven-step procedure, was included in the study together with the LRB fluorophore (alone or linked to an alkyl chain).  All these compds. are antagonists, but examn. of their ability to inhibit or modulate orthosteric [3H]NMS binding revealed that para-LRB-AC42 shared several properties with AC-42.  Carefully designed expts. allowed para-LRB-AC42 to be used as a FRET tracer on EGFP-fused M1 receptors.  Under equil. binding conditions, orthosteric ligands, AC-42, and the allosteric modulator gallamine behaved as competitors of para-LRB-AC42 binding whereas other allosteric compds. such as WIN 51,708 and N-desmethylclozapine were noncompetitive inhibitors.  Finally, mol. modeling studies focused on putative orthosteric/allosteric bitopic poses for AC-42 and para-LRB-AC42 in a 3D model of the human M1 receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOQzkjHcfvTLVg90H21EOLACvtfcHk0liBT-aqrd9aWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitF2nu7c%253D&md5=7883a194d0cc72fde6ad1d867655f8e0</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm201348t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201348t%26sid%3Dliteratum%253Aachs%26aulast%3DDaval%26aufirst%3DS.%2BB.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DBonnet%26aufirst%3DD.%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DHibert%26aufirst%3DM.%26aulast%3DGalzi%26aufirst%3DJ.%2BL.%26aulast%3DIlien%26aufirst%3DB.%26atitle%3DFluorescent%2520derivatives%2520of%2520AC-42%2520to%2520probe%2520bitopic%2520orthosteric%252Fallosteric%2520binding%2520mechanisms%2520on%2520muscarinic%2520M1%2520receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D2125%26epage%3D2143%26doi%3D10.1021%2Fjm201348t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Iturrioz, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvear-Perez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Mota, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franchet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guillier, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leroux, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dabire, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Jouan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chabane, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerbier, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdeaux, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maigret, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galzi, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hibert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Llorens-Cortes, C.</span><span> </span><span class="NLM_article-title">Identification and pharmacological properties of E339–3D6, the first nonpeptidic apelin receptor agonist</span> <span class="citation_source-journal">FASEB J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">1506</span><span class="NLM_x">–</span> <span class="NLM_lpage">1517</span><span class="refDoi"> DOI: 10.1096/fj.09-140715</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1096%2Ffj.09-140715" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=1506-1517&author=X.+Iturriozauthor=R.+Alvear-Perezauthor=N.+De+Motaauthor=C.+Franchetauthor=F.+Guillierauthor=V.+Lerouxauthor=H.+Dabireauthor=M.+Le+Jouanauthor=H.+Chabaneauthor=R.+Gerbierauthor=D.+Bonnetauthor=A.+Berdeauxauthor=B.+Maigretauthor=J.+L.+Galziauthor=M.+Hibertauthor=C.+Llorens-Cortes&title=Identification+and+pharmacological+properties+of+E339%E2%80%933D6%2C+the+first+nonpeptidic+apelin+receptor+agonist&doi=10.1096%2Ffj.09-140715"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1096%2Ffj.09-140715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.09-140715%26sid%3Dliteratum%253Aachs%26aulast%3DIturrioz%26aufirst%3DX.%26aulast%3DAlvear-Perez%26aufirst%3DR.%26aulast%3DDe%2BMota%26aufirst%3DN.%26aulast%3DFranchet%26aufirst%3DC.%26aulast%3DGuillier%26aufirst%3DF.%26aulast%3DLeroux%26aufirst%3DV.%26aulast%3DDabire%26aufirst%3DH.%26aulast%3DLe%2BJouan%26aufirst%3DM.%26aulast%3DChabane%26aufirst%3DH.%26aulast%3DGerbier%26aufirst%3DR.%26aulast%3DBonnet%26aufirst%3DD.%26aulast%3DBerdeaux%26aufirst%3DA.%26aulast%3DMaigret%26aufirst%3DB.%26aulast%3DGalzi%26aufirst%3DJ.%2BL.%26aulast%3DHibert%26aufirst%3DM.%26aulast%3DLlorens-Cortes%26aufirst%3DC.%26atitle%3DIdentification%2520and%2520pharmacological%2520properties%2520of%2520E339%25E2%2580%25933D6%252C%2520the%2520first%2520nonpeptidic%2520apelin%2520receptor%2520agonist%26jtitle%3DFASEB%2520J.%26date%3D2010%26volume%3D24%26spage%3D1506%26epage%3D1517%26doi%3D10.1096%2Ffj.09-140715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Vernall, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellam, B.</span><span> </span><span class="NLM_article-title">The evolving small-molecule fluorescent-conjugate toolbox for Class A GPCRs</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">171</span><span class="NLM_x">, </span> <span class="NLM_fpage">1073</span><span class="NLM_x">–</span> <span class="NLM_lpage">1084</span><span class="refDoi"> DOI: 10.1111/bph.12265</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1111%2Fbph.12265" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=23734587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1Sht7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2014&pages=1073-1084&author=A.+J.+Vernallauthor=S.+J.+Hillauthor=B.+Kellam&title=The+evolving+small-molecule+fluorescent-conjugate+toolbox+for+Class+A+GPCRs&doi=10.1111%2Fbph.12265"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The evolving small-molecule fluorescent-conjugate toolbox for Class A GPCRs</span></div><div class="casAuthors">Vernall, Andrea J.; Hill, Stephen J.; Kellam, Barrie</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1073-1084</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The past decade has witnessed fluorescently tagged drug mols. gaining significant attraction in their use as pharmacol. tools with which to visualize and interrogate receptor targets at the single-cell level.  Addnl., one can generate detailed pharmacol. information, such as affinity measurements, down to almost single-mol. detection limits.  The now accepted utilization of fluorescence-based readouts in high-throughput/high-content screening provides further evidence that fluorescent mols. offer a safer and more adaptable substitute to radioligands in mol. pharmacol. and drug discovery.  One such drug-target family that has received considerable attention are the GPCRs; this review therefore summarizes the most recent developments in the area of fluorescent ligand design for this important drug target.  We assess recently reported fluorescent conjugates by adopting a receptor-family-based approach, highlighting some of the strengths and weaknesses of the individual mols. and their subsequent use.  This review adds further strength to the arguments that fluorescent ligand design and synthesis requires careful planning and execution; providing examples illustrating that selection of the correct fluorescent dye, linker length/compn. and geog. attachment point to the drug scaffold can all influence the ultimate selectivity and potency of the final conjugate when compared with its unlabeled precursor.  When optimized appropriately, the resultant fluorescent conjugates have been successfully employed in an array of assay formats, including flow cytometry, fluorescence microscopy, FRET and scanning confocal microscopy.  It is clear that fluorescently labeled GPCR ligands remain a developing and dynamic research arena.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIxr-W-h3chLVg90H21EOLACvtfcHk0lhd3MyxtZv8GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1Sht7w%253D&md5=5e8768a623f257368a5d9d0278192987</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1111%2Fbph.12265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.12265%26sid%3Dliteratum%253Aachs%26aulast%3DVernall%26aufirst%3DA.%2BJ.%26aulast%3DHill%26aufirst%3DS.%2BJ.%26aulast%3DKellam%26aufirst%3DB.%26atitle%3DThe%2520evolving%2520small-molecule%2520fluorescent-conjugate%2520toolbox%2520for%2520Class%2520A%2520GPCRs%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2014%26volume%3D171%26spage%3D1073%26epage%3D1084%26doi%3D10.1111%2Fbph.12265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Ward, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milligan, G.</span><span> </span><span class="NLM_article-title">Structural and biophysical characterisation of G protein-coupled receptor ligand binding using resonance energy transfer and fluorescent labelling techniques</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Biomembr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1838</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">14</span><span class="refDoi"> DOI: 10.1016/j.bbamem.2013.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2Fj.bbamem.2013.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=23590995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmsVGlt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1838&publication_year=2014&pages=3-14&author=R.+J.+Wardauthor=G.+Milligan&title=Structural+and+biophysical+characterisation+of+G+protein-coupled+receptor+ligand+binding+using+resonance+energy+transfer+and+fluorescent+labelling+techniques&doi=10.1016%2Fj.bbamem.2013.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and biophysical characterisation of G protein-coupled receptor ligand binding using resonance energy transfer and fluorescent labeling techniques</span></div><div class="casAuthors">Ward, Richard J.; Milligan, Graeme</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1838</span>
        (<span class="NLM_cas:issue">1PA</span>),
    <span class="NLM_cas:pages">3-14</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The interaction between ligands and the G protein-coupled receptors (GPCRs) to which they bind has long been the focus of intensive investigation.  The signaling cascades triggered by receptor activation, due in most cases to ligand binding, are of great physiol. and medical importance; indeed, GPCRs are targeted by in excess of 30% of small mol. therapeutic medicines.  Attempts to identify further pharmacol. useful GPCR ligands, for receptors with known and unknown endogenous ligands, continue apace.  In earlier days direct assessment of such interactions was restricted largely to the use of ligands incorporating radioactive isotope labels as this allowed detection of the ligand and monitoring its interaction with the GPCR.  This use of such markers has continued with the development of ligands labeled with fluorophores and their application to the study of receptor-ligand interactions using both light microscopy and resonance energy transfer techniques, including homogenous time-resolved fluorescence resonance energy transfer.  Details of ligand-receptor interactions via x-ray crystallog. are advancing rapidly as methods suitable for routine prodn. of substantial amts. and stabilized forms of GPCRs have been developed and there is hope that this may become as routine as the co-crystn. of serine/threonine kinases with ligands, an approach that has facilitated widespread use of rapid structure-based ligand design.  Conformational changes involved in the activation of GPCRs, widely predicted by biochem. and biophys. means, have inspired the development of intramol. FRET-based sensor forms of GPCRs designed to investigate the events following ligand binding and resulting in a signal propagation across the cell membrane.  Finally, a no. of techniques are emerging in which ligand-GPCR binding can be studied in ways that, while indirect, are able to monitor its results in an unbiased and integrated manner.  This article is part of a Special Issue entitled: Structural and biophys. characterization of membrane protein-ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4R2XZxXKK2LVg90H21EOLACvtfcHk0lhd3MyxtZv8GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmsVGlt7g%253D&md5=b4a4f540ccc85b50f2322536a24a505c</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2013.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2013.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BJ.%26aulast%3DMilligan%26aufirst%3DG.%26atitle%3DStructural%2520and%2520biophysical%2520characterisation%2520of%2520G%2520protein-coupled%2520receptor%2520ligand%2520binding%2520using%2520resonance%2520energy%2520transfer%2520and%2520fluorescent%2520labelling%2520techniques%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Biomembr.%26date%3D2014%26volume%3D1838%26spage%3D3%26epage%3D14%26doi%3D10.1016%2Fj.bbamem.2013.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Bazin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathis, G.</span><span> </span><span class="NLM_article-title">Time resolved amplification of cryptate emission: a versatile technology to trace biomolecular interactions</span> <span class="citation_source-journal">Rev. Mol. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">233</span><span class="NLM_x">–</span> <span class="NLM_lpage">250</span><span class="refDoi"> DOI: 10.1016/S1389-0352(01)00040-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2FS1389-0352%2801%2900040-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD38XhsVSmt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2002&pages=233-250&author=H.+Bazinauthor=E.+Trinquetauthor=G.+Mathis&title=Time+resolved+amplification+of+cryptate+emission%3A+a+versatile+technology+to+trace+biomolecular+interactions&doi=10.1016%2FS1389-0352%2801%2900040-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Time resolved amplification of cryptate emission: a versatile technology to trace biomolecular interactions</span></div><div class="casAuthors">Bazin, H.; Trinquet, E.; Mathis, G.</div><div class="citationInfo"><span class="NLM_cas:title">Reviews in Molecular Biotechnology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">233-250</span>CODEN:
                <span class="NLM_cas:coden">RMBIFZ</span>;
        ISSN:<span class="NLM_cas:issn">1389-0352</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Fluorescence resonance energy transfer (FRET) in assocn. with a time-resolved fluorescence mode of detection was used to design a new homogeneous technol. suitable to monitor biomol. interactions.  A lanthanide cryptate characterized by a long lived fluorescence emission was used as donor and a cross-linked allophycocyanine was used as acceptor.  This new donor/acceptor pair displayed an exceptionally large Forster radius of 9 nm.  This allowed to build up a set of labeling strategies to probe the interactions between biomols. with an emphasis on fully indirect cassette formats particularly suitable for high throughput screening applications.  Herein we describe the basics of the technol., review the latest applications to the study of mol. interactions involved in cells and new oligonucleotides based assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-QBHgf98sd7Vg90H21EOLACvtfcHk0lhd3MyxtZv8GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhsVSmt7s%253D&md5=a8172b56da7349c622662bcb618cc86a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS1389-0352%2801%2900040-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1389-0352%252801%252900040-X%26sid%3Dliteratum%253Aachs%26aulast%3DBazin%26aufirst%3DH.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DMathis%26aufirst%3DG.%26atitle%3DTime%2520resolved%2520amplification%2520of%2520cryptate%2520emission%253A%2520a%2520versatile%2520technology%2520to%2520trace%2520biomolecular%2520interactions%26jtitle%3DRev.%2520Mol.%2520Biotechnol.%26date%3D2002%26volume%3D82%26spage%3D233%26epage%3D250%26doi%3D10.1016%2FS1389-0352%2801%2900040-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Zwier, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cottet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durroux, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douzon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bdioui, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregor, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourrier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oueslati, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicolas, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tinel, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boisseau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yverneau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charrier-Savournin, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fink, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span> </span><span class="NLM_article-title">A fluorescent ligand-binding alternative using Tag-lite® technology</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1248</span><span class="NLM_x">–</span> <span class="NLM_lpage">1259</span><span class="refDoi"> DOI: 10.1177/1087057110384611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1177%2F1087057110384611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=20974902" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVWgtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=1248-1259&author=J.+M.+Zwierauthor=T.+Rouxauthor=M.+Cottetauthor=T.+Durrouxauthor=S.+Douzonauthor=S.+Bdiouiauthor=N.+Gregorauthor=E.+Bourrierauthor=N.+Oueslatiauthor=L.+Nicolasauthor=N.+Tinelauthor=C.+Boisseauauthor=P.+Yverneauauthor=F.+Charrier-Savourninauthor=M.+Finkauthor=E.+Trinquet&title=A+fluorescent+ligand-binding+alternative+using+Tag-lite%C2%AE+technology&doi=10.1177%2F1087057110384611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorescent ligand-binding alternative using Tag-lite technology</span></div><div class="casAuthors">Zwier, Jurriaan M.; Roux, Thomas; Cottet, Martin; Durroux, Thierry; Douzon, Stephanie; Bdioui, Sara; Gregor, Nathalie; Bourrier, Emmanuel; Oueslati, Nadia; Nicolas, Ludovic; Tinel, Norbert; Boisseau, Chloe; Yverneau, Paul; Charrier-Savournin, Fabienne; Fink, Michel; Trinquet, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1248-1259</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are crucial cell surface receptors that transmit signals from a wide range of extracellular ligands.  Indeed, 40% to 50% of all marketed drugs are thought to modulate GPCR activity, making them the major class of targets in the drug discovery process.  Binding assays are widely used to identify high-affinity, selective, and potent GPCR drugs.  In this field, the use of radiolabeled ligands has remained so far the gold-std. method.  Here the authors report a less hazardous alternative for high-throughput screening (HTS) applications by the setup of a nonradioactive fluorescence-based technol. named Tag-lite.  Selective binding of various fluorescent ligands, either peptidic or not, covering a large panel of GPCRs from different classes is illustrated, particularly for chemokine (CXCR4), opioid (δ, μ, and κ), and cholecystokinin (CCK1 and CCK2) receptors.  Affinity consts. of well-known pharmacol. agents of numerous GPCRs are in line with values published in the literature.  The authors clearly demonstrate that the Tag-lite binding assay format can be successfully and reproducibly applied by using different cellular materials such as transient or stable recombinant cells lines expressing SNAP-tagged GPCR.  Such fluorescent-based binding assays can be performed with adherent cells or cells in suspension, in 96- or 384-well plates.  Altogether, this new technol. offers great advantages in terms of flexibility, rapidity, and user-friendliness; allows easy miniaturization; and makes it completely suitable for HTS applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ4y-qYZVkzbVg90H21EOLACvtfcHk0lgvxnF49FAPPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVWgtLY%253D&md5=ebb276667f42394eb16e8f0b64b365da</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1177%2F1087057110384611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057110384611%26sid%3Dliteratum%253Aachs%26aulast%3DZwier%26aufirst%3DJ.%2BM.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DCottet%26aufirst%3DM.%26aulast%3DDurroux%26aufirst%3DT.%26aulast%3DDouzon%26aufirst%3DS.%26aulast%3DBdioui%26aufirst%3DS.%26aulast%3DGregor%26aufirst%3DN.%26aulast%3DBourrier%26aufirst%3DE.%26aulast%3DOueslati%26aufirst%3DN.%26aulast%3DNicolas%26aufirst%3DL.%26aulast%3DTinel%26aufirst%3DN.%26aulast%3DBoisseau%26aufirst%3DC.%26aulast%3DYverneau%26aufirst%3DP.%26aulast%3DCharrier-Savournin%26aufirst%3DF.%26aulast%3DFink%26aufirst%3DM.%26aulast%3DTrinquet%26aufirst%3DE.%26atitle%3DA%2520fluorescent%2520ligand-binding%2520alternative%2520using%2520Tag-lite%25C2%25AE%2520technology%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2010%26volume%3D15%26spage%3D1248%26epage%3D1259%26doi%3D10.1177%2F1087057110384611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Karpenko, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margathe, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pflimlin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dupuis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hibert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durroux, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, D.</span><span> </span><span class="NLM_article-title">Selective nonpeptidic fluorescent ligands for oxytocin receptor: design, synthesis, and application to time-resolved FRET binding assay</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">2547</span><span class="NLM_x">–</span> <span class="NLM_lpage">2552</span><span class="refDoi"> DOI: 10.1021/jm501395b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501395b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvV2ju74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2547-2552&author=I.+A.+Karpenkoauthor=J.+F.+Margatheauthor=T.+Rodriguezauthor=E.+Pflimlinauthor=E.+Dupuisauthor=M.+Hibertauthor=T.+Durrouxauthor=D.+Bonnet&title=Selective+nonpeptidic+fluorescent+ligands+for+oxytocin+receptor%3A+design%2C+synthesis%2C+and+application+to+time-resolved+FRET+binding+assay&doi=10.1021%2Fjm501395b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Nonpeptidic Fluorescent Ligands for Oxytocin Receptor: Design, Synthesis, and Application to Time-Resolved FRET Binding Assay</span></div><div class="casAuthors">Karpenko, Iuliia A.; Margathe, Jean-Francois; Rodriguez, Thieric; Pflimlin, Elsa; Dupuis, Elodie; Hibert, Marcel; Durroux, Thierry; Bonnet, Dominique</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2547-2552</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design and the synthesis of the first high-affinity fluorescent ligands for oxytocin receptor (OTR) are described.  These compds. enabled the development of a TR-FRET based assay for OTR, readily amenable to high throughput screening.  The validation of the assay was achieved by competition expts. with both peptide and nonpeptide OTR ligands as competitors.  These probes represent the first selective fluorescent ligands for the oxytocin G protein-coupled receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYix67Nh90z7Vg90H21EOLACvtfcHk0lgvxnF49FAPPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvV2ju74%253D&md5=7ba9dddc66ed6bafb60b83cbc2efc9ff</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm501395b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501395b%26sid%3Dliteratum%253Aachs%26aulast%3DKarpenko%26aufirst%3DI.%2BA.%26aulast%3DMargathe%26aufirst%3DJ.%2BF.%26aulast%3DRodriguez%26aufirst%3DT.%26aulast%3DPflimlin%26aufirst%3DE.%26aulast%3DDupuis%26aufirst%3DE.%26aulast%3DHibert%26aufirst%3DM.%26aulast%3DDurroux%26aufirst%3DT.%26aulast%3DBonnet%26aufirst%3DD.%26atitle%3DSelective%2520nonpeptidic%2520fluorescent%2520ligands%2520for%2520oxytocin%2520receptor%253A%2520design%252C%2520synthesis%252C%2520and%2520application%2520to%2520time-resolved%2520FRET%2520binding%2520assay%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2547%26epage%3D2552%26doi%3D10.1021%2Fjm501395b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Hounsou, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margathe, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oueslati, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belhocine, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dupuis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ilien, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hibert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pin, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durroux, T.</span><span> </span><span class="NLM_article-title">Time-resolved FRET binding assay to investigate hetero-oligomer binding properties: proof of concept with dopamine D1/D3 heterodimer</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">466</span><span class="NLM_x">–</span> <span class="NLM_lpage">474</span><span class="refDoi"> DOI: 10.1021/cb5007568</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5007568" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=466-474&author=C.+Hounsouauthor=J.+F.+Margatheauthor=N.+Oueslatiauthor=A.+Belhocineauthor=E.+Dupuisauthor=C.+Thomasauthor=A.+Mannauthor=B.+Ilienauthor=D.+Rognanauthor=E.+Trinquetauthor=M.+Hibertauthor=J.+P.+Pinauthor=D.+Bonnetauthor=T.+Durroux&title=Time-resolved+FRET+binding+assay+to+investigate+hetero-oligomer+binding+properties%3A+proof+of+concept+with+dopamine+D1%2FD3+heterodimer&doi=10.1021%2Fcb5007568"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fcb5007568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5007568%26sid%3Dliteratum%253Aachs%26aulast%3DHounsou%26aufirst%3DC.%26aulast%3DMargathe%26aufirst%3DJ.%2BF.%26aulast%3DOueslati%26aufirst%3DN.%26aulast%3DBelhocine%26aufirst%3DA.%26aulast%3DDupuis%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DC.%26aulast%3DMann%26aufirst%3DA.%26aulast%3DIlien%26aufirst%3DB.%26aulast%3DRognan%26aufirst%3DD.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DHibert%26aufirst%3DM.%26aulast%3DPin%26aufirst%3DJ.%2BP.%26aulast%3DBonnet%26aufirst%3DD.%26aulast%3DDurroux%26aufirst%3DT.%26atitle%3DTime-resolved%2520FRET%2520binding%2520assay%2520to%2520investigate%2520hetero-oligomer%2520binding%2520properties%253A%2520proof%2520of%2520concept%2520with%2520dopamine%2520D1%252FD3%2520heterodimer%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D466%26epage%3D474%26doi%3D10.1021%2Fcb5007568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Cottet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faklaris, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Falco, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pin, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouillac, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durroux, T.</span><span> </span><span class="NLM_article-title">Fluorescent ligands to investigate GPCR binding properties and oligomerization</span> <span class="citation_source-journal">Biochem. Soc. Trans.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">148</span><span class="NLM_x">–</span> <span class="NLM_lpage">153</span><span class="refDoi"> DOI: 10.1042/BST20120237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1042%2FBST20120237" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=23356275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1eitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=148-153&author=M.+Cottetauthor=O.+Faklarisauthor=A.+Falcoauthor=E.+Trinquetauthor=J.+P.+Pinauthor=B.+Mouillacauthor=T.+Durroux&title=Fluorescent+ligands+to+investigate+GPCR+binding+properties+and+oligomerization&doi=10.1042%2FBST20120237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescent ligands to investigate GPCR binding properties and oligomerization</span></div><div class="casAuthors">Cottet, Martin; Faklaris, Orestis; Falco, Amadine; Trinquet, Eric; Pin, Jean-Philippe; Mouillac, Bernard; Durroux, Thierry</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Society Transactions</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">148-153</span>CODEN:
                <span class="NLM_cas:coden">BCSTB5</span>;
        ISSN:<span class="NLM_cas:issn">0300-5127</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  Fluorescent ligands for GPCRs (G-protein-coupled receptors) have been synthesized for a long time but their use was usually restricted to receptor localization in the cell by fluorescent imaging microscopy.  During the last two decades, the emergence of new fluorescence-based strategies and the concomitant development of fluorescent measurement app. have dramatically widened the use of fluorescent ligands.  Among the various strategies, TR (time-resolved)-FRET (fluorescence resonance energy transfer) approaches exhibit an interesting potential to study GPCR interactions with various partners.  We have derived various sets of ligands that target different GPCRs with fluorophores, which are compatible with TR-FRET strategies.  Fluorescent ligands labeled either with a fluorescent donor (such as europium or terbium cryptate) or with a fluorescent acceptor (such as fluorescein, dy647 or Alexa Fluor 647), for example, kept high affinities for their cognate receptors.  These ligands turn out to be interesting tools to develop FRET-based binding assays.  We also used these fluorescent ligands to analyze GPCR oligomerization by measuring FRET between ligands bound to receptor dimers.  In contrast with FRET strategies, on the basis of receptor labeling, the ligand-based approach we developed is fully compatible with the study of wild-type receptors and therefore with receptors expressed in native tissues.  Therefore, by using fluorescent analogs of oxytocin, we demonstrated the existence of oxytocin receptor dimers in the mammary gland of lactating rats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqikaTbyJ7bCbVg90H21EOLACvtfcHk0li-gm2HeIND4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1eitb4%253D&md5=fadfa1cbc5204e3f5d864f0a9740ca58</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1042%2FBST20120237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBST20120237%26sid%3Dliteratum%253Aachs%26aulast%3DCottet%26aufirst%3DM.%26aulast%3DFaklaris%26aufirst%3DO.%26aulast%3DFalco%26aufirst%3DA.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DPin%26aufirst%3DJ.%2BP.%26aulast%3DMouillac%26aufirst%3DB.%26aulast%3DDurroux%26aufirst%3DT.%26atitle%3DFluorescent%2520ligands%2520to%2520investigate%2520GPCR%2520binding%2520properties%2520and%2520oligomerization%26jtitle%3DBiochem.%2520Soc.%2520Trans.%26date%3D2013%26volume%3D41%26spage%3D148%26epage%3D153%26doi%3D10.1042%2FBST20120237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Emami-Nemini, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leblay, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourrier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamarque, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohse, M. J.</span><span> </span><span class="NLM_article-title">Time-resolved fluorescence ligand binding for G protein-coupled receptors</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1307</span><span class="NLM_x">–</span> <span class="NLM_lpage">1320</span><span class="refDoi"> DOI: 10.1038/nprot.2013.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnprot.2013.073" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1307-1320&author=A.+Emami-Neminiauthor=T.+Rouxauthor=M.+Leblayauthor=E.+Bourrierauthor=L.+Lamarqueauthor=E.+Trinquetauthor=M.+J.+Lohse&title=Time-resolved+fluorescence+ligand+binding+for+G+protein-coupled+receptors&doi=10.1038%2Fnprot.2013.073"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2013.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2013.073%26sid%3Dliteratum%253Aachs%26aulast%3DEmami-Nemini%26aufirst%3DA.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DLeblay%26aufirst%3DM.%26aulast%3DBourrier%26aufirst%3DE.%26aulast%3DLamarque%26aufirst%3DL.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DLohse%26aufirst%3DM.%2BJ.%26atitle%3DTime-resolved%2520fluorescence%2520ligand%2520binding%2520for%2520G%2520protein-coupled%2520receptors%26jtitle%3DNat.%2520Protoc.%26date%3D2013%26volume%3D8%26spage%3D1307%26epage%3D1320%26doi%3D10.1038%2Fnprot.2013.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Leyris, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdié, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fehrentz, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oueslati, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douzon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourrier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamarque, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagne, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galleyrand, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">M’kadmi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mary, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banères, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marie, J.</span><span> </span><span class="NLM_article-title">Homogeneous time-resolved fluorescence-based assay to screen for ligands targeting the growth hormone secretagogue receptor type 1a</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">408</span><span class="NLM_x">, </span> <span class="NLM_fpage">253</span><span class="NLM_x">–</span> <span class="NLM_lpage">262</span><span class="refDoi"> DOI: 10.1016/j.ab.2010.09.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2Fj.ab.2010.09.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=20937574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVagsLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=408&publication_year=2011&pages=253-262&author=J.+P.+Leyrisauthor=T.+Rouxauthor=E.+Trinquetauthor=P.+Verdi%C3%A9author=J.+A.+Fehrentzauthor=N.+Oueslatiauthor=S.+Douzonauthor=E.+Bourrierauthor=L.+Lamarqueauthor=D.+Gagneauthor=J.+C.+Galleyrandauthor=C.+M%E2%80%99kadmiauthor=J.+Martinezauthor=S.+Maryauthor=J.+L.+Ban%C3%A8resauthor=J.+Marie&title=Homogeneous+time-resolved+fluorescence-based+assay+to+screen+for+ligands+targeting+the+growth+hormone+secretagogue+receptor+type+1a&doi=10.1016%2Fj.ab.2010.09.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Homogeneous time-resolved fluorescence-based assay to screen for ligands targeting the growth hormone secretagogue receptor type 1a</span></div><div class="casAuthors">Leyris, Jean-Philippe; Roux, Thomas; Trinquet, Eric; Verdie, Pascal; Fehrentz, Jean-Alain; Oueslati, Nadia; Douzon, Stephanie; Bourrier, Emmanuel; Lamarque, Laurent; Gagne, Didier; Galleyrand, Jean-Claude; M'kadmi, Celine; Martinez, Jean; Mary, Sophie; Baneres, Jean-Louis; Marie, Jacky</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">408</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">253-262</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The growth hormone secretagogue receptor type 1a (GHS-R1a) belongs to class A G-protein-coupled receptors (GPCR).  This receptor mediates pleiotropic effects of ghrelin and represents a promising target for dysfunctions of growth hormone secretion and energy homeostasis including obesity.  Identification of new compds. which bind GHS-R1a is traditionally achieved using radioactive binding assays.  Here we propose a new fluorescence-based assay, called Tag-lite binding assay, based on a fluorescence resonance energy transfer (FRET) process between a terbium cryptate covalently attached to a SNAP-tag fused GHS-R1a (SNAP-GHS-R1a) and a high-affinity red fluorescent ghrelin ligand.  The long fluorescence lifetime of the terbium cryptate allows a time-resolved detection of the FRET signal.  The assay was made compatible with high-throughput screening by using prelabeled cells in suspension under a 384-well plate format.  Ki values for a panel of 14 compds. displaying agonist, antagonist, or inverse agonist properties were detd. using both the radioactive and the Tag-lite binding assays performed on the same batches of GHS-R1a-expressing cells.  Compd. potencies obtained in the two assays were nicely correlated.  This study is the first description of a sensitive and reliable nonradioactive binding assay for GHS-R1a in a format amenable to high-throughput screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5Df7hm2kmlbVg90H21EOLACvtfcHk0li-gm2HeIND4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVagsLrI&md5=cf1476bd78ebb24d460abd92a70d2a08</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2010.09.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2010.09.030%26sid%3Dliteratum%253Aachs%26aulast%3DLeyris%26aufirst%3DJ.%2BP.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DVerdi%25C3%25A9%26aufirst%3DP.%26aulast%3DFehrentz%26aufirst%3DJ.%2BA.%26aulast%3DOueslati%26aufirst%3DN.%26aulast%3DDouzon%26aufirst%3DS.%26aulast%3DBourrier%26aufirst%3DE.%26aulast%3DLamarque%26aufirst%3DL.%26aulast%3DGagne%26aufirst%3DD.%26aulast%3DGalleyrand%26aufirst%3DJ.%2BC.%26aulast%3DM%25E2%2580%2599kadmi%26aufirst%3DC.%26aulast%3DMartinez%26aufirst%3DJ.%26aulast%3DMary%26aufirst%3DS.%26aulast%3DBan%25C3%25A8res%26aufirst%3DJ.%2BL.%26aulast%3DMarie%26aufirst%3DJ.%26atitle%3DHomogeneous%2520time-resolved%2520fluorescence-based%2520assay%2520to%2520screen%2520for%2520ligands%2520targeting%2520the%2520growth%2520hormone%2520secretagogue%2520receptor%2520type%25201a%26jtitle%3DAnal.%2520Biochem.%26date%3D2011%26volume%3D408%26spage%3D253%26epage%3D262%26doi%3D10.1016%2Fj.ab.2010.09.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Hoffman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flynn, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tillu, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patek, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boitano, S.</span><span> </span><span class="NLM_article-title">Lanthanide labeling of a potent protease activated receptor-2 agonist for time-resolved fluorescence analysis</span> <span class="citation_source-journal">Bioconjugate Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">2098</span><span class="NLM_x">–</span> <span class="NLM_lpage">2104</span><span class="refDoi"> DOI: 10.1021/bc300300q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bc300300q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlOmur7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2012&pages=2098-2104&author=J.+Hoffmanauthor=A.+N.+Flynnauthor=D.+V.+Tilluauthor=Z.+Zhangauthor=R.+Patekauthor=T.+J.+Priceauthor=J.+Vagnerauthor=S.+Boitano&title=Lanthanide+labeling+of+a+potent+protease+activated+receptor-2+agonist+for+time-resolved+fluorescence+analysis&doi=10.1021%2Fbc300300q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Lanthanide Labeling of a Potent Protease Activated Receptor-2 Agonist for Time-Resolved Fluorescence Analysis</span></div><div class="casAuthors">Hoffman, Justin; Flynn, Andrea N.; Tillu, Dipti V.; Zhang, Zhenyu; Patek, Renata; Price, Theodore J.; Vagner, Josef; Boitano, Scott</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2098-2104</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protease activated receptor-2 (PAR2) is one of four G-protein coupled receptors (GPCRs) that can be activated by exogenous or endogenous proteases, which cleave the extracellular amino-terminus to expose a tethered ligand and subsequent G-protein signaling.  Alternatively, PAR2 can be activated by peptide or peptidomimetic ligands derived from the sequence of the natural tethered ligand.  Screening of novel ligands that directly bind to PAR2 to agonize or antagonize the receptor has been hindered by the lack of a sensitive, high-throughput, affinity binding assay.  The authors describe the synthesis and use of a modified PAR2 peptidomimetic agonist, 2-furoyl-LIGRLO-(diethylenetriaminepentaacetic acid)-NH2 (2-f-LIGRLO-dtpa, I), designed for lanthanide-based time-resolved fluorescence screening.  The authors first demonstrate that 2-f-LIGRLO-dtpa is a potent and specific PAR2 agonist across a full spectrum of in vitro assays.  The authors then show that 2-f-LIGRLO-dtpa can be used in an affinity binding assay to evaluate the ligand-receptor interactions between known high potency peptidomimetic agonists (2-furoyl-LIGRLO-NH2, 2-f-LIGRLO; 2-aminothiazol-4-yl-LIGRL-NH2, 2-at-LIGRL; 6-aminonicotinyl-LIGRL-NH2, 6-an-LIGRL) and PAR2.  A sep. N-terminal peptidomimetic modification (3-indoleacetyl-LIGRL-NH2, 3-ia-LIGRL) that does not activate PAR2 signaling was used as a neg. control.  All three peptidomimetic agonists demonstrated sigmoidal competitive binding curves, with the more potent agonists (2-f-LIGRLO and 2-at-LIGRL) displaying increased competition.  In contrast, the control peptide (3-ia-LIGRL) displayed limited competition for PAR2 binding.  In summary, the authors have developed a europium-contg. PAR2 agonist that can be used in a highly sensitive affinity binding assay to screen novel PAR2 ligands in a high-throughput format.  This ligand can serve as a crit. tool in the screening and development of PAR2 ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1zdFIiqcNz7Vg90H21EOLACvtfcHk0ljuanrmzDKJRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlOmur7M&md5=7e269f0e06b7687dee47dd69c1aa2240</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fbc300300q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbc300300q%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DJ.%26aulast%3DFlynn%26aufirst%3DA.%2BN.%26aulast%3DTillu%26aufirst%3DD.%2BV.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DPatek%26aufirst%3DR.%26aulast%3DPrice%26aufirst%3DT.%2BJ.%26aulast%3DVagner%26aufirst%3DJ.%26aulast%3DBoitano%26aufirst%3DS.%26atitle%3DLanthanide%2520labeling%2520of%2520a%2520potent%2520protease%2520activated%2520receptor-2%2520agonist%2520for%2520time-resolved%2520fluorescence%2520analysis%26jtitle%3DBioconjugate%2520Chem.%26date%3D2012%26volume%3D23%26spage%3D2098%26epage%3D2104%26doi%3D10.1021%2Fbc300300q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Hu, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamman, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span> </span><span class="NLM_article-title">A homogeneous G protein-coupled receptor ligand binding assay based on time-resolved fluorescence resonance energy transfer</span> <span class="citation_source-journal">Assay Drug Dev. Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">543</span><span class="NLM_x">–</span> <span class="NLM_lpage">550</span><span class="refDoi"> DOI: 10.1089/adt.2008.152</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1089%2Fadt.2008.152" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=18699727" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFWltL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2008&pages=543-550&author=L.+A.+Huauthor=T.+Zhouauthor=B.+D.+Hammanauthor=Q.+Liu&title=A+homogeneous+G+protein-coupled+receptor+ligand+binding+assay+based+on+time-resolved+fluorescence+resonance+energy+transfer&doi=10.1089%2Fadt.2008.152"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A Homogeneous G Protein-Coupled Receptor Ligand Binding Assay Based on Time-Resolved Fluorescence Resonance Energy Transfer</span></div><div class="casAuthors">Hu, Liaoyuan A.; Zhou, Tian; Hamman, Brian D.; Liu, Qingyun</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">543-550</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Fluorescence resonance energy transfer (FRET) has emerged as a powerful tool to the study of protein-protein interactions, such as receptor-ligand binding.  However, the application of FRET to the study of G protein-coupled receptors (GPCRs) has been limited by the method of labeling receptor with fluorescence probes.  Here the authors described a novel time-resolved (TR)-FRET method to study GPCR-ligand binding by using human complement 5a (C5a) receptor (C5aR) as a model system.  Human C5aR was expressed in human embryonic kidney 293 cells with a hemagglutinin (HA) epitope at the N-terminus.  Purified human C5a was labeled with terbium chelate and used as the fluorescence donor.  Monoclonal anti-HA antibody conjugated with Alexa Fluor 488 was used as the fluorescence acceptor.  Robust FRET signal was obsd. when the labeled ligand and C5aR membrane were mixed in the presence of the conjugated anti-HA antibody.  This FRET signal was specific and saturable.  C5a binding affinity to C5aR measured by the FRET assay was consistent with the data as detd. by competition binding anal. using radiolabeled C5a.  The FRET assay was also used to det. affinity of C5aR antagonists by competition binding anal., and the data are similar to those from radioligand binding studies.  Compared to the commonly used radioligand binding assay, this TR-FRET-based assay provides a nonradioactive, faster, and sensitive homogeneous assay format that could be easily adapted to high-throughput screening.  The principle of this assay should also be applicable to other GPCRs, esp. to those receptors with peptide or protein ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMRxTAXC_4dbVg90H21EOLACvtfcHk0ljuanrmzDKJRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFWltL3J&md5=60084dad6b902bc19ddffb5c22483667</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1089%2Fadt.2008.152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2008.152%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DL.%2BA.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DLiu%26aufirst%3DQ.%26atitle%3DA%2520homogeneous%2520G%2520protein-coupled%2520receptor%2520ligand%2520binding%2520assay%2520based%2520on%2520time-resolved%2520fluorescence%2520resonance%2520energy%2520transfer%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2008%26volume%3D6%26spage%3D543%26epage%3D550%26doi%3D10.1089%2Fadt.2008.152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samama, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burbaum, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroke, I. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appell, K. C.</span><span> </span><span class="NLM_article-title">Chemokine receptor-ligand interactions measured using time-resolved fluorescence</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">2372</span><span class="NLM_x">–</span> <span class="NLM_lpage">2377</span><span class="refDoi"> DOI: 10.1021/bi972161u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi972161u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=2372-2377&author=J.+Ingleseauthor=P.+Samamaauthor=S.+Patelauthor=J.+Burbaumauthor=I.+L.+Strokeauthor=K.+C.+Appell&title=Chemokine+receptor-ligand+interactions+measured+using+time-resolved+fluorescence&doi=10.1021%2Fbi972161u"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fbi972161u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi972161u%26sid%3Dliteratum%253Aachs%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DSamama%26aufirst%3DP.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DBurbaum%26aufirst%3DJ.%26aulast%3DStroke%26aufirst%3DI.%2BL.%26aulast%3DAppell%26aufirst%3DK.%2BC.%26atitle%3DChemokine%2520receptor-ligand%2520interactions%2520measured%2520using%2520time-resolved%2520fluorescence%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D2372%26epage%3D2377%26doi%3D10.1021%2Fbi972161u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Stoner, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pace, H.</span><span> </span><span class="NLM_article-title">A.Asenapine: a clinical review of a second-generation antipsychotic</span> <span class="citation_source-journal">Clin. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">1023</span><span class="NLM_x">–</span> <span class="NLM_lpage">1040</span><span class="refDoi"> DOI: 10.1016/j.clinthera.2012.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2Fj.clinthera.2012.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=22494521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1ajsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2012&pages=1023-1040&author=S.+C.+Stonerauthor=H.+Pace&title=A.Asenapine%3A+a+clinical+review+of+a+second-generation+antipsychotic&doi=10.1016%2Fj.clinthera.2012.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Asenapine: A Clinical Review of a Second-Generation Antipsychotic</span></div><div class="casAuthors">Stoner, Steven C.; Pace, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1023-1040</span>CODEN:
                <span class="NLM_cas:coden">CLTHDG</span>;
        ISSN:<span class="NLM_cas:issn">0149-2918</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Background: Schizophrenia and bipolar disorder are both prevalent types of psychiatric illness in the United States.  As second-generation antipsychotics have become a more viable first-line treatment option, their use has been assocd. with a new era of adverse events (AEs), most notably metabolic and cardiovascular concerns.  Although treatment options for schizophrenia and bipolar disorder have arguably improved, there continues to be a need for medications that achieve and maintain desired efficacy with minimal AEs.  Objectives: This article serves as a comprehensive review of the pharmacol. profile of the second-generation antipsychotic asenapine, as well as a review of its efficacy and safety profiles based on the findings from clin. trials in schizophrenia and bipolar disorder.  Methods: Searches of Ovid MEDLINE, EMBASE, and IDIS were conducted (Jan. 1996 to Nov. 2011) to identify clin. studies and other primary literature sources with the following search terms: asenapine, bipolar disorder, antipsychotic, psychosis, dopamine, and schizophrenia.  Only studies of asenapine and placebo and/or active-comparator arms were included.  Results: The literature search yielded 67 unique articles, including review articles, which were excluded.  The efficacy of asenapine was reported in 3 clin. studies in patients with schizophrenia, 1 each in acute and long-term settings, measured as significant changes in Pos. and Neg. Syndrome Scale scores over 6 and 52 wk.  Asenapine also had reported efficacy in the prevention of relapse in schizophrenia during a 26-wk extension study.  In addn., efficacy of asenapine was reported in 2 studies in acute mania as well as extension phases of both 9 and 40 wk, as detd. by significant changes in Young Mania Rating Scale scores.  The most commonly reported AEs in these studies were somnolence (13%-24%), extrapyramidal symptoms (EPS) (7%-12%), and dizziness (11%).  Conclusions: The findings from multiple studies have suggested that asenapine is efficacious in the acute treatment of schizophrenia.  Asenapine has reported long-term efficacy for this indication and the potential to reduce the incidence of relapse.  Asenapine efficacy was also reported in the treatment of acute manic or mixed states assocd. with bipolar I disorder.  Asenapine had an acceptable safety profile across the different disease states studied, although it was not devoid of metabolic and EPS-related AEs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNbjsE9Lqut7Vg90H21EOLACvtfcHk0ljuanrmzDKJRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1ajsbo%253D&md5=997c53bd27066251cf80f02d950b2139</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.clinthera.2012.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.clinthera.2012.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DStoner%26aufirst%3DS.%2BC.%26aulast%3DPace%26aufirst%3DH.%26atitle%3DA.Asenapine%253A%2520a%2520clinical%2520review%2520of%2520a%2520second-generation%2520antipsychotic%26jtitle%3DClin.%2520Ther.%26date%3D2012%26volume%3D34%26spage%3D1023%26epage%3D1040%26doi%3D10.1016%2Fj.clinthera.2012.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Shahid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zorn, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, E. H.</span><span> </span><span class="NLM_article-title">Asenapine: a novel psychopharmacologic agent with a unique human receptor signature</span> <span class="citation_source-journal">J. Psychopharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">65</span><span class="NLM_x">–</span> <span class="NLM_lpage">73</span><span class="refDoi"> DOI: 10.1177/0269881107082944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1177%2F0269881107082944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=18308814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVClu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=65-73&author=M.+Shahidauthor=G.+B.+Walkerauthor=S.+H.+Zornauthor=E.+H.+Wong&title=Asenapine%3A+a+novel+psychopharmacologic+agent+with+a+unique+human+receptor+signature&doi=10.1177%2F0269881107082944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Asenapine: a novel psychopharmacologic agent with a unique human receptor signature</span></div><div class="casAuthors">Shahid, M.; Walker, G. B.; Zorn, S. H.; Wong, E. H. F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Psychopharmacology (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-73</span>CODEN:
                <span class="NLM_cas:coden">JOPSEQ</span>;
        ISSN:<span class="NLM_cas:issn">0269-8811</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications Ltd.</span>)
        </div><div class="casAbstract">Asenapine is a novel psychopharmacol. agent under development for the treatment of schizophrenia and bipolar disorder.  We detd. and compared the human receptor binding affinities and functional characteristics of asenapine and several antipsychotic drugs.  Compds. were tested under comparable assay conditions using cloned human receptors.  In comparison with the antipsychotics, asenapine showed high affinity and a different rank order of binding affinities (pKi) for serotonin receptors (5-HT1A [8.6], 5-HT1B [8.4], 5-HT2A [10.2], 5-HT2B [9.8], 5-HT2C [10.5], 5-HT5 [8.8], 5-HT6 [9.6] and 5-HT7 [9.9]), adrenoceptors (α1 [8.9], α2A [8.9], α2B [9.5] and α2C [8.9]), dopamine receptors (D1 [8.9], D2 [8.9], D3 [9.4] and D4 [9.0]) and histamine receptors (H1 [9.0] and H2 [8.2]).  It had much lower affinity (pKi ≤ 5) for muscarinic receptors and was the only agent with affinity for H2 receptors.  Relative to its D2 receptor affinity, asenapine had a higher affinity for 5-HT2C, 5-HT2A, 5-HT2B, 5-HT7, 5-HT6, α2B and D3 receptors, suggesting stronger engagement of these targets at therapeutic doses.  Asenapine behaved as a potent antagonist (pKB) at 5-HT1A (7.4), 5-HT1B (8.1), 5-HT2A (9.0), 5-HT2B (9.3), 5-HT2C (9.0), 5-HT6 (8.0), 5-HT7 (8.5), D2 (9.1), D3 (9.1), α2A (7.3), α2B (8.3), α2C (6.8) and H1 (8.4) receptors.  These functional effects differed from those of risperidone (pKB < 5 for 5-HT6) and olanzapine (pKB < 5 for 5-HT1A and α2).  Our results indicate that asenapine has a unique human receptor signature, with binding affinity and antagonistic properties that differ appreciably from those of antipsychotic drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjDUIlfl2wyLVg90H21EOLACvtfcHk0liaqf5kMild4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVClu7Y%253D&md5=4b89d79dd6fcbf958a02198312569ae6</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1177%2F0269881107082944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0269881107082944%26sid%3Dliteratum%253Aachs%26aulast%3DShahid%26aufirst%3DM.%26aulast%3DWalker%26aufirst%3DG.%2BB.%26aulast%3DZorn%26aufirst%3DS.%2BH.%26aulast%3DWong%26aufirst%3DE.%2BH.%26atitle%3DAsenapine%253A%2520a%2520novel%2520psychopharmacologic%2520agent%2520with%2520a%2520unique%2520human%2520receptor%2520signature%26jtitle%3DJ.%2520Psychopharmacol.%26date%3D2009%26volume%3D23%26spage%3D65%26epage%3D73%26doi%3D10.1177%2F0269881107082944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Olofson, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martz, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senet, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piteau, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malfroot, T.</span><span> </span><span class="NLM_article-title">A new reagent for the selective, high-yield N-dealkylation of tertiary amines: improved syntheses of naltrexone and nalbuphine</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1984</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">2081</span><span class="NLM_x">–</span> <span class="NLM_lpage">2082</span><span class="refDoi"> DOI: 10.1021/jo00185a072</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00185a072" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADyaL2cXktlyks7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1984&pages=2081-2082&author=R.+A.+Olofsonauthor=J.+T.+Martzauthor=J.-P.+Senetauthor=M.+Piteauauthor=T.+Malfroot&title=A+new+reagent+for+the+selective%2C+high-yield+N-dealkylation+of+tertiary+amines%3A+improved+syntheses+of+naltrexone+and+nalbuphine&doi=10.1021%2Fjo00185a072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A new reagent for the selective, high-yield N-dealkylation of tertiary amines:  improved syntheses of naltrexone and nalbuphine</span></div><div class="casAuthors">Olofson, R. A.; Martz, Jonathan T.; Senet, Jean Pierre; Piteau, Marc; Malfroot, Thierry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2081-2</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Reaction of tertiary amines with ClCO2CHClMe in ClCH2CH2Cl at reflux affords the intermediates, R2NCO2CHClMe (R = alkyl, R2 = cycloalkyl), which are converted to the desired salts, R2NH·HCl, just by heating in MeOH.  The N-dealkylation reagent is made in 96% yield by mixing MeCHO with COCl2 in the presence of an R4N+ Cl- catalyst.  The N-dealkylations are highly selective, mild, occur in high yield, and permit the presence of a variety of functionalities.  The process is exemplified by the conversion of glaucine (I) to the phenanthrene (II) in 89% yield and by the N-demethylations of arecoline, O-acetyltropine, and 6-acetylcodeine in yields of 95-97%.  The methodol. was used as part of a process for the transformation of oxycodone to the com. narcotic antagonist, naltrexone (58% overall yield) and to the prescription analgesic, nalbuphine (III) (69% overall yield).  Arom. tertiary amines also are N-dealkylated, although under more vigorous conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcSuAJiJApjLVg90H21EOLACvtfcHk0liaqf5kMild4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXktlyks7k%253D&md5=9402bfb15be7a0d057bfe7bf95bde7b6</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjo00185a072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00185a072%26sid%3Dliteratum%253Aachs%26aulast%3DOlofson%26aufirst%3DR.%2BA.%26aulast%3DMartz%26aufirst%3DJ.%2BT.%26aulast%3DSenet%26aufirst%3DJ.-P.%26aulast%3DPiteau%26aufirst%3DM.%26aulast%3DMalfroot%26aufirst%3DT.%26atitle%3DA%2520new%2520reagent%2520for%2520the%2520selective%252C%2520high-yield%2520N-dealkylation%2520of%2520tertiary%2520amines%253A%2520improved%2520syntheses%2520of%2520naltrexone%2520and%2520nalbuphine%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1984%26volume%3D49%26spage%3D2081%26epage%3D2082%26doi%3D10.1021%2Fjo00185a072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Bouzide, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauvé, G.</span><span> </span><span class="NLM_article-title">Silver(I) oxide mediated highly selective monotosylation of symmetrical diols. Application to the synthesis of polysubstituted cyclic ethers</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">2329</span><span class="NLM_x">–</span> <span class="NLM_lpage">2332</span><span class="refDoi"> DOI: 10.1021/ol020071y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol020071y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD38Xktl2mt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2002&pages=2329-2332&author=A.+Bouzideauthor=G.+Sauv%C3%A9&title=Silver%28I%29+oxide+mediated+highly+selective+monotosylation+of+symmetrical+diols.+Application+to+the+synthesis+of+polysubstituted+cyclic+ethers&doi=10.1021%2Fol020071y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Silver(I) Oxide Mediated Highly Selective Monotosylation of Symmetrical Diols. Application to the Synthesis of Polysubstituted Cyclic Ethers</span></div><div class="casAuthors">Bouzide, Abderrahim; Sauve, Gilles</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2329-2332</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tosylation of sym. diols and oligo(ethylene glycol)s, e.g. I, with a stoichiometric amt. of 4-MeC6H4SO2Cl in the presence of Ag2O and a catalytic amt. of KI led selectively to the corresponding monotosylates, e.g. II, in high yields.  Polysubstituted cyclic ethers, e.g. III, were obtained readily upon treatment of the corresponding diols, e.g. I, with an excess of silver oxide.  The high selectivity was explained on the basis of the difference in acidity between the two hydroxy groups, which are involved in intramol. hydrogen bonding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou01kIoVIUV7Vg90H21EOLACvtfcHk0liaqf5kMild4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xktl2mt7s%253D&md5=769cf574899c1c8fa167fe556bb42dcc</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fol020071y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol020071y%26sid%3Dliteratum%253Aachs%26aulast%3DBouzide%26aufirst%3DA.%26aulast%3DSauv%25C3%25A9%26aufirst%3DG.%26atitle%3DSilver%2528I%2529%2520oxide%2520mediated%2520highly%2520selective%2520monotosylation%2520of%2520symmetrical%2520diols.%2520Application%2520to%2520the%2520synthesis%2520of%2520polysubstituted%2520cyclic%2520ethers%26jtitle%3DOrg.%2520Lett.%26date%3D2002%26volume%3D4%26spage%3D2329%26epage%3D2332%26doi%3D10.1021%2Fol020071y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Knight, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Misra, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quirk, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benwell, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Revell, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennett, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bickerdike, M.</span><span> </span><span class="NLM_article-title">Pharmacological characterisation of the agonist radioligand binding site of 5-HT(2A), 5-HT(2B) and 5-HT(2C) receptors</span> <span class="citation_source-journal">Naunyn-Schmiedeberg's Arch. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">370</span><span class="NLM_x">, </span> <span class="NLM_fpage">114</span><span class="NLM_x">–</span> <span class="NLM_lpage">123</span><span class="refDoi"> DOI: 10.1007/s00210-004-0951-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1007%2Fs00210-004-0951-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=15322733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD2cXoslyrtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=370&publication_year=2004&pages=114-123&author=A.+R.+Knightauthor=A.+Misraauthor=K.+Quirkauthor=K.+Benwellauthor=D.+Revellauthor=G.+Kennettauthor=M.+Bickerdike&title=Pharmacological+characterisation+of+the+agonist+radioligand+binding+site+of+5-HT%282A%29%2C+5-HT%282B%29+and+5-HT%282C%29+receptors&doi=10.1007%2Fs00210-004-0951-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological characterisation of the agonist radioligand binding site of 5-HT2A, 5-HT2B and 5-HT2C receptors</span></div><div class="casAuthors">Knight, Antony R.; Misra, Anil; Quirk, Kathleen; Benwell, Karen; Revell, Dean; Kennett, Guy; Bickerdike, Mike</div><div class="citationInfo"><span class="NLM_cas:title">Naunyn-Schmiedeberg's Archives of Pharmacology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">370</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">114-123</span>CODEN:
                <span class="NLM_cas:coden">NSAPCC</span>;
        ISSN:<span class="NLM_cas:issn">0028-1298</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">In the present study we compared the affinity of various drugs for the high affinity "agonist-preferring" binding site of human recombinant 5-HT2A, 5-HT2B and 5-HT2C receptors stably expressed in monoclonal mammalian cell lines.  To ensure that the "agonist-preferring" conformation of the receptor was preferentially labeled in competition binding expts., satn. anal. was conducted using antagonist and agonist radiolabels at each receptor.  Antagonist radiolabels ([3H]-ketanserin for 5-HT2A receptor and [3H]-mesulergine for 5-HT2B and 5-HT2C receptor) bound to a larger population of receptors in each prepn. than the corresponding agonist radiolabel ([125I]-DOI for 5-HT2A receptor binding and [3H]-5-HT for 5-HT2B and 5-HT2C receptor binding).  Competition expts. were subsequently conducted against appropriate concns. of the agonist radiolabels bound to the "agonist-preferring" subset of receptors in each prepn.  These studies confirmed that there are a no. of highly selective antagonists available to investigate 5-HT2 receptor subtype function (for example, MDL 100907, RS-127445 and RS-102221 for 5-HT2A, 5-HT2B and 5-HT2C receptors resp.).  There remains, however, a lack of highly selective agonists.  (-)DOI is potent and moderately selective for 5-HT2A receptors.  BW723C86 has poor selectivity for human 5-HT2B receptors, while Org 37684 and VER-3323 display some selectivity for the 5-HT2C receptor.  We report for the first time in a single study, the selectivity of numerous serotonergic drugs for 5-HT2 receptors from the same species, in mammalian cell lines and using, exclusively, agonist radiolabels.  The results indicate the importance of defining the selectivity of pharmacol. tools, which may have been over-estd. in the past, and highlights the need to find more selective agonists to investigate 5-HT2 receptor pharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDwPm2ckoq17Vg90H21EOLACvtfcHk0ljsuO2jatf91Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXoslyrtrk%253D&md5=c8c65b650fe62433da2ba0d77dcbe4fd</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2Fs00210-004-0951-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00210-004-0951-4%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DA.%2BR.%26aulast%3DMisra%26aufirst%3DA.%26aulast%3DQuirk%26aufirst%3DK.%26aulast%3DBenwell%26aufirst%3DK.%26aulast%3DRevell%26aufirst%3DD.%26aulast%3DKennett%26aufirst%3DG.%26aulast%3DBickerdike%26aufirst%3DM.%26atitle%3DPharmacological%2520characterisation%2520of%2520the%2520agonist%2520radioligand%2520binding%2520site%2520of%25205-HT%25282A%2529%252C%25205-HT%25282B%2529%2520and%25205-HT%25282C%2529%2520receptors%26jtitle%3DNaunyn-Schmiedeberg%2527s%2520Arch.%2520Pharmacol.%26date%3D2004%26volume%3D370%26spage%3D114%26epage%3D123%26doi%3D10.1007%2Fs00210-004-0951-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Wainscott, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schenck, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nissen, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baez, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kursar, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucaites, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, D. L.</span><span> </span><span class="NLM_article-title">Pharmacological characteristics of the newly cloned rat 5-hydroxytryptamine2F receptor</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">419</span><span class="NLM_x">–</span> <span class="NLM_lpage">426</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=1993&pages=419-426&author=D.+B.+Wainscottauthor=M.+L.+Cohenauthor=K.+W.+Schenckauthor=J.+E.+Audiaauthor=J.+S.+Nissenauthor=M.+Baezauthor=J.+D.+Kursarauthor=V.+L.+Lucaitesauthor=D.+L.+Nelson&title=Pharmacological+characteristics+of+the+newly+cloned+rat+5-hydroxytryptamine2F+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWainscott%26aufirst%3DD.%2BB.%26aulast%3DCohen%26aufirst%3DM.%2BL.%26aulast%3DSchenck%26aufirst%3DK.%2BW.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DNissen%26aufirst%3DJ.%2BS.%26aulast%3DBaez%26aufirst%3DM.%26aulast%3DKursar%26aufirst%3DJ.%2BD.%26aulast%3DLucaites%26aufirst%3DV.%2BL.%26aulast%3DNelson%26aufirst%3DD.%2BL.%26atitle%3DPharmacological%2520characteristics%2520of%2520the%2520newly%2520cloned%2520rat%25205-hydroxytryptamine2F%2520receptor%26jtitle%3DMol.%2520Pharmacol.%26date%3D1993%26volume%3D43%26spage%3D419%26epage%3D426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Millan, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maiofiss, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cussac, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audinot, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boutin, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman-Tancredi, A.</span><span> </span><span class="NLM_article-title">Differential actions of antiparkinson agents at multiple classes of monoaminergic receptor. I. A multivariate analysis of the binding profiles of 14 drugs at 21 native and cloned human receptor subtypes</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">791</span><span class="NLM_x">–</span> <span class="NLM_lpage">804</span><span class="refDoi"> DOI: 10.1124/jpet.102.039867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1124%2Fjpet.102.039867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=12388666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD38XotlCnsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2002&pages=791-804&author=M.+J.+Millanauthor=L.+Maiofissauthor=D.+Cussacauthor=V.+Audinotauthor=J.+A.+Boutinauthor=A.+Newman-Tancredi&title=Differential+actions+of+antiparkinson+agents+at+multiple+classes+of+monoaminergic+receptor.+I.+A+multivariate+analysis+of+the+binding+profiles+of+14+drugs+at+21+native+and+cloned+human+receptor+subtypes&doi=10.1124%2Fjpet.102.039867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Differential actions of antiparkinson agents at multiple classes of monoaminergic receptor. I. A multivariate analysis of the binding profiles of 14 drugs at 21 native and cloned human receptor subtypes</span></div><div class="casAuthors">Millan, Mark J.; Maiofiss, Lisa; Cussac, Didier; Audinot, Valerie; Boutin, Jean-A.; Newman-Tancredi, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">791-804</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Because little comparative information is available concerning receptor profiles of antiparkinson drugs, affinities of 14 agents were detd. at diverse receptors implicated in the etiol. and/or treatment of Parkinson's disease: human (h)D1, hD2S, hD2L, hD3, hD4, and hD5 receptors; human 5-hydroxytryptamine (5-HT)1A, h5-HT1B, h5-HT1D, h5-HT2A, h5-HT2B, and h5-HT2C receptors; hα1A-, hα1B-, hα1D-, hα2A-, hα2B-, hα2C-, rat α2D-, hβ1-, and hβ2-adrenoceptors (ARs); and native histamine1 receptors.  A correlation matrix (294 pKi values) demonstrated substantial "co-variance".  Correspondingly, principal components anal. revealed that axis 1, which accounted for 76% variance, was assocd. with the majority of receptor types: drugs displaying overall high vs. modest affinities migrated at opposite extremities.  Axis 2 (7% of variance) differentiated drugs with high affinity for hD4 and H1 receptors vs. hα1-AR subtypes.  Five percent of variance was attributable to axis 3, which distinguished drugs with marked affinity for hβ1- and hβ2-ARs vs. hD5 and 5-HT2A receptors.  Hierarchical (cluster) anal. of global homol. generated a dendrogram differentiating two major groups possessing low vs. high affinity, resp., for multiple serotonergic and hD5 receptors.  Within the first group, quinpirole, quinerolane, ropinirole, and pramipexole interacted principally with hD2, hD3, and hD4 receptors, whereas piribedil and talipexole recognized dopaminergic receptors and hα2-ARs.  Within the second group, lisuride and terguride manifested high affinities for all sites, with roxindole/bromocriptine, cabergoline/pergolide, and 6,7-dihydroxy-N,N-dimethyl-2-ammotetralin (TL99)/apomorphine comprising three addnl. subclusters of closely related ligands.  In conclusion, an innovative multivariate anal. revealed marked heterogeneity in binding profiles of antiparkinson agents.  Actions at sites other than hD2 receptors likely participate in their (contrasting) functional profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1qrMpB3-4sbVg90H21EOLACvtfcHk0ljsuO2jatf91Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotlCnsL8%253D&md5=20be25d95375cdcd3308cf0590aed337</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1124%2Fjpet.102.039867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.102.039867%26sid%3Dliteratum%253Aachs%26aulast%3DMillan%26aufirst%3DM.%2BJ.%26aulast%3DMaiofiss%26aufirst%3DL.%26aulast%3DCussac%26aufirst%3DD.%26aulast%3DAudinot%26aufirst%3DV.%26aulast%3DBoutin%26aufirst%3DJ.%2BA.%26aulast%3DNewman-Tancredi%26aufirst%3DA.%26atitle%3DDifferential%2520actions%2520of%2520antiparkinson%2520agents%2520at%2520multiple%2520classes%2520of%2520monoaminergic%2520receptor.%2520I.%2520A%2520multivariate%2520analysis%2520of%2520the%2520binding%2520profiles%2520of%252014%2520drugs%2520at%252021%2520native%2520and%2520cloned%2520human%2520receptor%2520subtypes%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2002%26volume%3D303%26spage%3D791%26epage%3D804%26doi%3D10.1124%2Fjpet.102.039867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Napier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melrose, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHarg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallis, R.</span><span> </span><span class="NLM_article-title">Characterisation of the 5-HT receptor binding profile of eletriptan and kinetics of [3H]eletriptan binding at human 5-HT1B and 5-HT1D receptors</span> <span class="citation_source-journal">Eur. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">368</span><span class="NLM_x">, </span> <span class="NLM_fpage">259</span><span class="NLM_x">–</span> <span class="NLM_lpage">268</span><span class="refDoi"> DOI: 10.1016/S0014-2999(99)00026-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2FS0014-2999%2899%2900026-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10193663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADyaK1MXhs1OitLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=1999&pages=259-268&author=C.+Napierauthor=M.+Stewartauthor=H.+Melroseauthor=B.+Hopkinsauthor=A.+McHargauthor=R.+Wallis&title=Characterisation+of+the+5-HT+receptor+binding+profile+of+eletriptan+and+kinetics+of+%5B3H%5Deletriptan+binding+at+human+5-HT1B+and+5-HT1D+receptors&doi=10.1016%2FS0014-2999%2899%2900026-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the 5-HT receptor binding profile of eletriptan and kinetics of [3H]eletriptan binding at human 5-HT1B and 5-HT1D receptors</span></div><div class="casAuthors">Napier, Carolyn; Stewart, Michael; Melrose, Heather; Hopkins, Brian; McHarg, Aileen; Wallis, Rob</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">2/3</span>),
    <span class="NLM_cas:pages">259-268</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">The affinity of eletriptan ((R)-3-(1-methyl-2-pyrrolidinylmethyl)-5-[2-(phenylsulfonyl)ethyl]-1H-indole) for a range of 5-HT receptors was compared to values obtained for other 5-HT1B/1D receptor agonists known to be effective in the treatment of migraine.  Eletriptan, like sumatriptan, zolmitriptan, naratriptan and rizatriptan had highest affinity for the human 5-HT1B, 5-HT1D and putative 5-HT1f receptor.  Kinetic studies comparing the binding of [3H]eletriptan and [3H]sumatriptan to the human recombinant 5-HT1B and 5-HT1D receptors expressed in HeLa cells revealed that both radioligands bound with high specificity (>90%) and reached equil. within 10-15 min.  However, [3H]eletriptan had over 6-fold higher affinity than [3H]sumatriptan at the 5-HT1D receptor (KD: 0.92 and 6.58 nM, resp.) and over 3-fold higher affinity than [3H]sumatriptan at the 5-HT1B receptor (KD: 3.14 and 11.07 nM, resp.).  Assocn. and dissocn. rates for both radioligands could only be accurately detd. at the 5-HT1D receptor and then only at 4°.  At this temp., [3H]eletriptan had a significantly faster assocn. rate (Kon 0.249 min-1 nM-1) than [3H]sumatriptan (Kon 0.024 min-1 nM-1) and a significantly slower off-rate (Koff 0.027 min-1 compared to 0.037 min-1 for [3H]sumatriptan).  These data indicate that eletriptan is a potent ligand at the human 5-HT1B, 5-HT1D and 5-HT1f receptors and are consistent with its potent vasoconstrictor activity and use as a drug for the acute treatment of migraine headache.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoikpaa981kkbVg90H21EOLACvtfcHk0lgBG5GGrFEGhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhs1OitLs%253D&md5=9c99c3028504aaaab34ec8059b9486ed</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2FS0014-2999%2899%2900026-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-2999%252899%252900026-6%26sid%3Dliteratum%253Aachs%26aulast%3DNapier%26aufirst%3DC.%26aulast%3DStewart%26aufirst%3DM.%26aulast%3DMelrose%26aufirst%3DH.%26aulast%3DHopkins%26aufirst%3DB.%26aulast%3DMcHarg%26aufirst%3DA.%26aulast%3DWallis%26aufirst%3DR.%26atitle%3DCharacterisation%2520of%2520the%25205-HT%2520receptor%2520binding%2520profile%2520of%2520eletriptan%2520and%2520kinetics%2520of%2520%255B3H%255Deletriptan%2520binding%2520at%2520human%25205-HT1B%2520and%25205-HT1D%2520receptors%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D1999%26volume%3D368%26spage%3D259%26epage%3D268%26doi%3D10.1016%2FS0014-2999%2899%2900026-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Leysen, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gommeren, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heylen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luyten, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van de Weyer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanhoenacker, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haegeman, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schotte, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Gompel, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wouters, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lesage, A. S.</span><span> </span><span class="NLM_article-title">Alniditan, a new 5-hydroxytryptamine1D agonist and migraine-abortive agent: ligand-binding properties of human 5-hydroxytryptamine1D alpha, human 5-hydroxytryptamine1D beta, and calf 5-hydroxytryptamine1D receptors investigated with [3H]5-hydroxytryptamine and [3H]alniditan</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1567</span><span class="NLM_x">–</span> <span class="NLM_lpage">1580</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=8967979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADyaK2sXntlyi" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1996&pages=1567-1580&author=J.+E.+Leysenauthor=W.+Gommerenauthor=L.+Heylenauthor=W.+H.+Luytenauthor=I.+Van+de+Weyerauthor=P.+Vanhoenackerauthor=G.+Haegemanauthor=A.+Schotteauthor=P.+Van+Gompelauthor=R.+Woutersauthor=A.+S.+Lesage&title=Alniditan%2C+a+new+5-hydroxytryptamine1D+agonist+and+migraine-abortive+agent%3A+ligand-binding+properties+of+human+5-hydroxytryptamine1D+alpha%2C+human+5-hydroxytryptamine1D+beta%2C+and+calf+5-hydroxytryptamine1D+receptors+investigated+with+%5B3H%5D5-hydroxytryptamine+and+%5B3H%5Dalniditan"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Alniditan, a new 5-hydroxytryptamine1D agonist and migraine-abortive agent: ligand-binding properties of human 5-hydroxytryptamine1Dα, human 5-hydroxytryptamine1Dβ, and calf 5-hydroxyptryptamineaD receptors investigated with [3H]5-hydroxytryptamine and [3H]alniditan</span></div><div class="casAuthors">Leysen, Jose E.; Gommeren, Walter; Heylen, Lieve; Luyten, Walter H. M. L.; van de Weyer, Inez; Vanhoenacker, Peter; Haegeman, Guy; Schotte, Alain; van Gompel, Paul; Wouters, Ria; Lesage, Anne S.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1567-1580</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">Williams & Wilkins</span>)
        </div><div class="casAbstract">Alniditan is a new migraine-abortive agent.  It is a benzopyran deriv. and therefore structurally unrelated to sumatriptan and other indole-derivs. and to ergoline derivs.  The action of sumatriptan is thought to be mediated by 5-hydroxytryptamine (5-HT)1D-type receptors.  We investigated the receptor-binding profile in vitro of alniditan compared with sumatriptan and dihydroergotamine for 28 neurotransmitter receptor subtypes, several receptors for peptides and lipid-derived factors, ion channel-binding sites, and monoamine transporters.  Alniditan revealed nanomolar affinity for calf substantia nigra 5-HT1D and for cloned h5-HT1Dα, h-HT1Dβ, and h5-HT1A receptors (Ki = 0.8, 0.4, 1.1, and 3.8 nM, resp.).  Alniditan was more potent than sumatriptan at 5-HT1D-type and 5-HT1A receptors.  Alniditan showed moderate-to-low or no affinity for other investigated receptors; sumatriptan showed addnl. binding to 5-HT1F receptors.  Dihydroergotamine had a much broader profile with high affinity for several 5-HT, adrenergic and dopaminergic receptors in signal transduction assays using cells expressing recombinant h5-HT1Dα, h5-HT1Dβ, or h5-HT1A receptors, alniditan (like 5-HT) was a full agonist for inhibition of stimulated adenylyl cyclase (IC50 = 1.1, 1.3, and 74 nM, resp., for alniditan).  Therefore, in functional assays, the potency of alniditan was much higher at 5-HT1D receptors than at 5-HT1A receptors.  We further compared the properties of [3H]alniditan, as a new radioligand for 5-HT1D-type receptors, with those of [3H]5-HT in membrane prepns. of calf substantia nigra, C6 glioma cells expressing h5-HT1Dα, and L929 cells expressing h5-HT1Dβ receptors.  [3H]Alniditan revealed very rapid assocn. and dissocn. binding kinetics and showed slightly higher affinity (Kd = 1-2 nM) than [3H]5-HT.  We investigated 25 compds. for inhibition of [3H]alniditan and [3H]5-HT binding in the three membrane prepns.; Ki values of the radioligands were largely similar, although some subtle differences appeared.  Most compds. did not differentiate between 5-HT1Dα and 5-HT1Dβ receptors, except methylsergide, ritanserin, ocaperidone, risperidone, and ketanserin, which showed 10-60-fold higher affinity for the 5-HT1Dα receptor.  The Ki values of the compds. obtained with 5-HT1D receptors in calf substantia nigra indicated that these receptors are of the 5-HT1Dβ-type.  We demonstrated that alniditan is a potent agonist at h5-HT1Dα and h5-HT1Dβ receptors; its properties probably underlie its cranial vasoconstrictive and antimigraine properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoooU3fQwv0i7Vg90H21EOLACvtfcHk0lgBG5GGrFEGhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXntlyi&md5=22a4fefe7cf41846f7451461c67fd336</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLeysen%26aufirst%3DJ.%2BE.%26aulast%3DGommeren%26aufirst%3DW.%26aulast%3DHeylen%26aufirst%3DL.%26aulast%3DLuyten%26aufirst%3DW.%2BH.%26aulast%3DVan%2Bde%2BWeyer%26aufirst%3DI.%26aulast%3DVanhoenacker%26aufirst%3DP.%26aulast%3DHaegeman%26aufirst%3DG.%26aulast%3DSchotte%26aufirst%3DA.%26aulast%3DVan%2BGompel%26aufirst%3DP.%26aulast%3DWouters%26aufirst%3DR.%26aulast%3DLesage%26aufirst%3DA.%2BS.%26atitle%3DAlniditan%252C%2520a%2520new%25205-hydroxytryptamine1D%2520agonist%2520and%2520migraine-abortive%2520agent%253A%2520ligand-binding%2520properties%2520of%2520human%25205-hydroxytryptamine1D%2520alpha%252C%2520human%25205-hydroxytryptamine1D%2520beta%252C%2520and%2520calf%25205-hydroxytryptamine1D%2520receptors%2520investigated%2520with%2520%255B3H%255D5-hydroxytryptamine%2520and%2520%255B3H%255Dalniditan%26jtitle%3DMol.%2520Pharmacol.%26date%3D1996%26volume%3D50%26spage%3D1567%26epage%3D1580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">John, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pauwels, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halazy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Grand, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verscheure, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valentin, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wurch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chopin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marien, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleven, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koek, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Assie, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carilla-Durand, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarayre, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colpaert, F. C.</span><span> </span><span class="NLM_article-title">F 11356, a novel 5-hydroxytryptamine (5-HT) derivative with potent, selective, and unique high intrinsic activity at 5-HT1B/1D receptors in models relevant to migraine</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10381763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADyaK1MXktFKisrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=1999&pages=83-95&author=G.+W.+Johnauthor=P.+J.+Pauwelsauthor=M.+Perezauthor=S.+Halazyauthor=B.+Le+Grandauthor=Y.+Verscheureauthor=J.+P.+Valentinauthor=C.+Palmierauthor=T.+Wurchauthor=P.+Chopinauthor=M.+Marienauthor=M.+S.+Klevenauthor=W.+Koekauthor=M.+B.+Assieauthor=E.+Carilla-Durandauthor=J.+P.+Tarayreauthor=F.+C.+Colpaert&title=F+11356%2C+a+novel+5-hydroxytryptamine+%285-HT%29+derivative+with+potent%2C+selective%2C+and+unique+high+intrinsic+activity+at+5-HT1B%2F1D+receptors+in+models+relevant+to+migraine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">F 11356, a novel 5-hydroxytryptamine (5-HT) derivative with potent, selective, and unique high intrinsic activity at 5-HT1B/1D receptors in models relevant to migraine</span></div><div class="casAuthors">John, Gareth W.; Pauwels, Petrus J.; Perez, Michel; Halazy, Serge; Grand, Bruno Le; Verscheure, Yvan; Valentin, Jean-Pierre; Palmier, Christiane; Wurch, Thierry; Chopin, Philippe; Marien, Marc; Kleven, Mark S.; Koek, Wouter; Assie, Marie-Bernadette; Carilla-Durand, Elisabeth; Tarayre, Jean-Pierre; Colpaert, Francis C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">83-95</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">F 11356 (4-[4-[2-(2-aminoethyl)-1H-indol-5-yloxyl]acetylpiperazinyl-1-yl]benzonitrile) was designed to take advantage of the superior potency and efficacy characteristics of 5-hydroxytryptamine (5-HT) compared with tryptamine at 5-HT1B/1D receptors.  F 11356 has subnanomolar affinity for cloned human and nonhuman 5-HT1B and 5-HT1D receptors, and its affinity for 5-HT1A and other 5-HT receptors, including the 5-HT1F subtype, is 50-fold lower and micromolar, resp.  In C6 cells expressing human 5-HT1B or human 5-HT1D receptors, F 11356 was the most potent compd. in inhibiting forskolin-induced cAMP formation (pD2 = 8.9 and 9.6), and in contrast to tryptamine and derivs., it produced maximal enhancement of [35S]guanosine-5'-O-(3-thio)triphosphate-specific binding equiv. to 5-HT.  F 11356 was equipotent to 5-HT (pD2 = 7.1 vs. 7.2) and more potent than tryptamine derivs. in contracting rabbit isolated saphenous vein.  In isolated guinea pig trigeminal ganglion neurons, F 11356 was more potent (pD2 = 7.3 vs. 6.7) and induced greater increases in outward hyperpolarizing Ca2+-dependent K+ current than sumatriptan.  In anesthetized pigs, F 11356 elicited highly cranioselective, more potent (from 0.16 μg/kg i.v.) and greater carotid vasoconstriction than tryptamine derivs.  Decreases in carotid blood flow were obsd. in conscious dogs from 0.63 mg/kg oral F 11356 in the absence of changes in heart rate or behavior.  Oral activity was confirmed when hypothermic responses were elicited in guinea pigs (ED50 = 1.6 mg/kg), suggesting that F 11356 also accesses the brain.  F 11356 thus is a selective, high-potency agonist at 5-HT1B/1D receptors, which distinguishes itself from tryptamine and derivs. in exerting high intrinsic activity at these receptors in vascular and neuronal models relevant to migraine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq8SCtDP2atbVg90H21EOLACvtfcHk0lgBG5GGrFEGhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktFKisrc%253D&md5=492ea2af29bf4a9ccf5527cdc73a2d3c</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJohn%26aufirst%3DG.%2BW.%26aulast%3DPauwels%26aufirst%3DP.%2BJ.%26aulast%3DPerez%26aufirst%3DM.%26aulast%3DHalazy%26aufirst%3DS.%26aulast%3DLe%2BGrand%26aufirst%3DB.%26aulast%3DVerscheure%26aufirst%3DY.%26aulast%3DValentin%26aufirst%3DJ.%2BP.%26aulast%3DPalmier%26aufirst%3DC.%26aulast%3DWurch%26aufirst%3DT.%26aulast%3DChopin%26aufirst%3DP.%26aulast%3DMarien%26aufirst%3DM.%26aulast%3DKleven%26aufirst%3DM.%2BS.%26aulast%3DKoek%26aufirst%3DW.%26aulast%3DAssie%26aufirst%3DM.%2BB.%26aulast%3DCarilla-Durand%26aufirst%3DE.%26aulast%3DTarayre%26aufirst%3DJ.%2BP.%26aulast%3DColpaert%26aufirst%3DF.%2BC.%26atitle%3DF%252011356%252C%2520a%2520novel%25205-hydroxytryptamine%2520%25285-HT%2529%2520derivative%2520with%2520potent%252C%2520selective%252C%2520and%2520unique%2520high%2520intrinsic%2520activity%2520at%25205-HT1B%252F1D%2520receptors%2520in%2520models%2520relevant%2520to%2520migraine%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1999%26volume%3D290%26spage%3D83%26epage%3D95" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Boess, F. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monsma, F. J.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwingelstein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sleight, A. J.</span><span> </span><span class="NLM_article-title">Interaction of tryptamine and ergoline compounds with threonine 196 in the ligand binding site of the 5-hydroxytryptamine6 receptor</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">515</span><span class="NLM_x">–</span> <span class="NLM_lpage">523</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1997&pages=515-523&author=F.+G.+Boessauthor=F.+J.+Monsmaauthor=V.+Meyerauthor=C.+Zwingelsteinauthor=A.+J.+Sleight&title=Interaction+of+tryptamine+and+ergoline+compounds+with+threonine+196+in+the+ligand+binding+site+of+the+5-hydroxytryptamine6+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBoess%26aufirst%3DF.%2BG.%26aulast%3DMonsma%26aufirst%3DF.%2BJ.%26aulast%3DMeyer%26aufirst%3DV.%26aulast%3DZwingelstein%26aufirst%3DC.%26aulast%3DSleight%26aufirst%3DA.%2BJ.%26atitle%3DInteraction%2520of%2520tryptamine%2520and%2520ergoline%2520compounds%2520with%2520threonine%2520196%2520in%2520the%2520ligand%2520binding%2520site%2520of%2520the%25205-hydroxytryptamine6%2520receptor%26jtitle%3DMol.%2520Pharmacol.%26date%3D1997%26volume%3D52%26spage%3D515%26epage%3D523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Routledge, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bromidge, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moss, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirst, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riley, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gager, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stean, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Upton, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Middlemiss, D. N.</span><span> </span><span class="NLM_article-title">Characterization of SB-271046: a potent, selective and orally active 5-HT(6) receptor antagonist</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">1606</span><span class="NLM_x">–</span> <span class="NLM_lpage">1612</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0703457</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fsj.bjp.0703457" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10928964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlslygsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2000&pages=1606-1612&author=C.+Routledgeauthor=S.+M.+Bromidgeauthor=S.+F.+Mossauthor=G.+W.+Priceauthor=W.+Hirstauthor=H.+Newmanauthor=G.+Rileyauthor=T.+Gagerauthor=T.+Steanauthor=N.+Uptonauthor=S.+E.+Clarkeauthor=A.+M.+Brownauthor=D.+N.+Middlemiss&title=Characterization+of+SB-271046%3A+a+potent%2C+selective+and+orally+active+5-HT%286%29+receptor+antagonist&doi=10.1038%2Fsj.bjp.0703457"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of SB-271046: a potent, selective and orally active 5-HT6 receptor antagonist</span></div><div class="casAuthors">Routledge, Carol; Bromidge, Steven M.; Moss, Stephen F.; Price, Gary W.; Hirst, Warren; Newman, Helen; Riley, Graham; Gager, Tracey; Stean, Tania; Upton, Neil; Clarke, Stephen E.; Brown, Anthony M.; Middlemiss, Derek N.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1606-1612</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">1 SB-271046, potently displaced [3H]-LSD and [125I]-SB-258585 from human 5-HT6 receptors recombinantly expressed in HeLa cells in vitro (pKi 8.92 and 9.09 resp.).  SB-271046 also displaced [125I]-SB-258585 from human caudate putamen and rat and pig striatum membranes (pKi 8.81, 9.02 and 8.55 resp.).  2 SB-271046 was over 200 fold selective for the 5-HT6 receptor vs 55 other receptors, binding sites and ion channels.  3 In functional studies on human 5-HT6 receptors SB-271046 competitively antagonized 5-HT-induced stimulation of adenylyl cyclase activity with a pA2 of 8.71.  4 SB-271046 produced an increase in seizure threshold over a wide-dose range in the rat maximal electroshock seizure threshold (MEST) test, with a min. ED of ≤ 0.1 mg kg-1 p.o. and max. effect at 4 h post-dose.  The level of anticonvulsant activity achieved correlated well with the blood concns. of SB-271046 (EC50 of 0.16 μM) and brain concns. of 0.01 - 0.04 μM at Cmax.  5 These data, together with the obsd. anticonvulsant activity of other selective 5-HT6 receptor antagonists, SB-258510 (10 mg kg-1, 2-6 h pre-test) and Ro 04-6790 (1-30 mg kg-1, 1 h pre-test), in the rat MEST test, suggest that the anticonvulsant properties of SB-271046 are likely to be mediated by 5-HT6 receptors.  6 Overall, these studies demonstrate that SB-271046 is a potent and selective 5-HT6 receptor antagonist and is orally active in the rat MEST test.  SB-271046 represents a valuable tool for evaluating the in vivo central function of 5-HT6 receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofj-FKPcThe7Vg90H21EOLACvtfcHk0ljDeuyG5lJbVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlslygsro%253D&md5=b382383bcd06a3154186b22e9e9582b6</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0703457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0703457%26sid%3Dliteratum%253Aachs%26aulast%3DRoutledge%26aufirst%3DC.%26aulast%3DBromidge%26aufirst%3DS.%2BM.%26aulast%3DMoss%26aufirst%3DS.%2BF.%26aulast%3DPrice%26aufirst%3DG.%2BW.%26aulast%3DHirst%26aufirst%3DW.%26aulast%3DNewman%26aufirst%3DH.%26aulast%3DRiley%26aufirst%3DG.%26aulast%3DGager%26aufirst%3DT.%26aulast%3DStean%26aufirst%3DT.%26aulast%3DUpton%26aufirst%3DN.%26aulast%3DClarke%26aufirst%3DS.%2BE.%26aulast%3DBrown%26aufirst%3DA.%2BM.%26aulast%3DMiddlemiss%26aufirst%3DD.%2BN.%26atitle%3DCharacterization%2520of%2520SB-271046%253A%2520a%2520potent%252C%2520selective%2520and%2520orally%2520active%25205-HT%25286%2529%2520receptor%2520antagonist%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2000%26volume%3D130%26spage%3D1606%26epage%3D1612%26doi%3D10.1038%2Fsj.bjp.0703457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 9 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sanjay  Yadav</span>, <span class="hlFld-ContribAuthor ">Surisetti  Suresh</span>. </span><span class="cited-content_cbyCitation_article-title">N
              ‐Heterocyclic Carbene (NHC)‐Catalyzed Intramolecular Stetter Reaction to Access Dibenzo‐fused Seven‐membered Heterocycles. </span><span class="cited-content_cbyCitation_journal-name">Asian Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>10 </em>
                                    (6)
                                     , 1406-1409. <a href="https://doi.org/10.1002/ajoc.202100158" title="DOI URL">https://doi.org/10.1002/ajoc.202100158</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ajoc.202100158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fajoc.202100158%26sid%3Dliteratum%253Aachs%26jtitle%3DAsian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DN%252B%2525E2%252580%252590Heterocyclic%252BCarbene%252B%252528NHC%252529%2525E2%252580%252590Catalyzed%252BIntramolecular%252BStetter%252BReaction%252Bto%252BAccess%252BDibenzo%2525E2%252580%252590fused%252BSeven%2525E2%252580%252590membered%252BHeterocycles%26aulast%3DYadav%26aufirst%3DSanjay%26date%3D2021%26date%3D2021%26volume%3D10%26issue%3D6%26spage%3D1406%26epage%3D1409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael P.  Killoran</span>, <span class="hlFld-ContribAuthor ">Sergiy  Levin</span>, <span class="hlFld-ContribAuthor ">Michelle E.  Boursier</span>, <span class="hlFld-ContribAuthor ">Kristopher  Zimmerman</span>, <span class="hlFld-ContribAuthor ">Robin  Hurst</span>, <span class="hlFld-ContribAuthor ">Mary P.  Hall</span>, <span class="hlFld-ContribAuthor ">Thomas  Machleidt</span>, <span class="hlFld-ContribAuthor ">Thomas A.  Kirkland</span>, <span class="hlFld-ContribAuthor ">Rachel  Friedman Ohana</span>. </span><span class="cited-content_cbyCitation_article-title">An Integrated Approach toward NanoBRET Tracers for Analysis of GPCR Ligand Engagement. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (10)
                                     , 2857. <a href="https://doi.org/10.3390/molecules26102857" title="DOI URL">https://doi.org/10.3390/molecules26102857</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26102857&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26102857%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DAn%252BIntegrated%252BApproach%252Btoward%252BNanoBRET%252BTracers%252Bfor%252BAnalysis%252Bof%252BGPCR%252BLigand%252BEngagement%26aulast%3DKilloran%26aufirst%3DMichael%2BP.%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D10%26spage%3D2857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joshua W  Conner</span>, <span class="hlFld-ContribAuthor ">Daniel P  Poole</span>, <span class="hlFld-ContribAuthor ">Manuela  Jörg</span>, <span class="hlFld-ContribAuthor ">Nicholas A  Veldhuis</span>. </span><span class="cited-content_cbyCitation_article-title">New small molecule fluorescent probes for G protein-coupled receptors: valuable tools for drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2021,</strong> <em>13 </em>
                                    (1)
                                     , 63-90. <a href="https://doi.org/10.4155/fmc-2019-0327" title="DOI URL">https://doi.org/10.4155/fmc-2019-0327</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2019-0327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2019-0327%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DNew%252Bsmall%252Bmolecule%252Bfluorescent%252Bprobes%252Bfor%252BG%252Bprotein-coupled%252Breceptors%25253A%252Bvaluable%252Btools%252Bfor%252Bdrug%252Bdiscovery%26aulast%3DConner%26aufirst%3DJoshua%2BW%26date%3D2021%26volume%3D13%26issue%3D1%26spage%3D63%26epage%3D90" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samantha  Alvarez-Herrera</span>, <span class="hlFld-ContribAuthor ">Raúl  Escamilla</span>, <span class="hlFld-ContribAuthor ">Oscar  Medina-Contreras</span>, <span class="hlFld-ContribAuthor ">Ricardo  Saracco</span>, <span class="hlFld-ContribAuthor ">Yvonne  Flores</span>, <span class="hlFld-ContribAuthor ">Gabriela  Hurtado-Alvarado</span>, <span class="hlFld-ContribAuthor ">José Luis  Maldonado-García</span>, <span class="hlFld-ContribAuthor ">Enrique  Becerril-Villanueva</span>, <span class="hlFld-ContribAuthor ">Gilberto  Pérez-Sánchez</span>, <span class="hlFld-ContribAuthor ">Lenin  Pavón</span>. </span><span class="cited-content_cbyCitation_article-title">Immunoendocrine Peripheral Effects Induced by Atypical Antipsychotics. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Endocrinology</span><span> <strong>2020,</strong> <em>11 </em><a href="https://doi.org/10.3389/fendo.2020.00195" title="DOI URL">https://doi.org/10.3389/fendo.2020.00195</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fendo.2020.00195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffendo.2020.00195%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Endocrinology%26atitle%3DImmunoendocrine%252BPeripheral%252BEffects%252BInduced%252Bby%252BAtypical%252BAntipsychotics%26aulast%3DAlvarez-Herrera%26aufirst%3DSamantha%26date%3D2020%26date%3D2020%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natasha C.  Dale</span>, <span class="hlFld-ContribAuthor ">Elizabeth K. M.  Johnstone</span>, <span class="hlFld-ContribAuthor ">Carl W.  White</span>, <span class="hlFld-ContribAuthor ">Kevin D. G.  Pfleger</span>. </span><span class="cited-content_cbyCitation_article-title">NanoBRET: The Bright Future of Proximity-Based Assays. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Bioengineering and Biotechnology</span><span> <strong>2019,</strong> <em>7 </em><a href="https://doi.org/10.3389/fbioe.2019.00056" title="DOI URL">https://doi.org/10.3389/fbioe.2019.00056</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fbioe.2019.00056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffbioe.2019.00056%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Bioengineering%2520and%2520Biotechnology%26atitle%3DNanoBRET%25253A%252BThe%252BBright%252BFuture%252Bof%252BProximity-Based%252BAssays%26aulast%3DDale%26aufirst%3DNatasha%2BC.%26date%3D2019%26date%3D2019%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Oliver  Plettenburg</span>. </span><span class="cited-content_cbyCitation_article-title">The Impact of Chemical Biology on Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Israel Journal of Chemistry</span><span> <strong>2019,</strong> <em>59 </em>
                                    (1-2)
                                     , 29-36. <a href="https://doi.org/10.1002/ijch.201900007" title="DOI URL">https://doi.org/10.1002/ijch.201900007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ijch.201900007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fijch.201900007%26sid%3Dliteratum%253Aachs%26jtitle%3DIsrael%2520Journal%2520of%2520Chemistry%26atitle%3DThe%252BImpact%252Bof%252BChemical%252BBiology%252Bon%252BDrug%252BDiscovery%26aulast%3DPlettenburg%26aufirst%3DOliver%26date%3D2019%26date%3D2019%26volume%3D59%26issue%3D1-2%26spage%3D29%26epage%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carl W.  White</span>, <span class="hlFld-ContribAuthor ">Elizabeth K.M.  Johnstone</span>, <span class="hlFld-ContribAuthor ">Heng B.  See</span>, <span class="hlFld-ContribAuthor ">Kevin D.G.  Pfleger</span>. </span><span class="cited-content_cbyCitation_article-title">NanoBRET ligand binding at a GPCR under endogenous promotion facilitated by CRISPR/Cas9 genome editing. </span><span class="cited-content_cbyCitation_journal-name">Cellular Signalling</span><span> <strong>2019,</strong> <em>54 </em>, 27-34. <a href="https://doi.org/10.1016/j.cellsig.2018.11.018" title="DOI URL">https://doi.org/10.1016/j.cellsig.2018.11.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cellsig.2018.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cellsig.2018.11.018%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520Signalling%26atitle%3DNanoBRET%252Bligand%252Bbinding%252Bat%252Ba%252BGPCR%252Bunder%252Bendogenous%252Bpromotion%252Bfacilitated%252Bby%252BCRISPR%25252FCas9%252Bgenome%252Bediting%26aulast%3DWhite%26aufirst%3DCarl%2BW.%26date%3D2019%26volume%3D54%26spage%3D27%26epage%3D34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joyce  Heuninck</span>, <span class="hlFld-ContribAuthor ">Candide  Hounsou</span>, <span class="hlFld-ContribAuthor ">Elodie  Dupuis</span>, <span class="hlFld-ContribAuthor ">Eric  Trinquet</span>, <span class="hlFld-ContribAuthor ">Bernard  Mouillac</span>, <span class="hlFld-ContribAuthor ">Jean-Philippe  Pin</span>, <span class="hlFld-ContribAuthor ">Dominique  Bonnet</span>, <span class="hlFld-ContribAuthor ">Thierry  Durroux</span>. </span><span class="cited-content_cbyCitation_article-title">Time-Resolved FRET-Based Assays to Characterize G Protein-Coupled Receptor Hetero-oligomer Pharmacology. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 151-168. <a href="https://doi.org/10.1007/978-1-4939-9121-1_8" title="DOI URL">https://doi.org/10.1007/978-1-4939-9121-1_8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-9121-1_8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-9121-1_8%26sid%3Dliteratum%253Aachs%26atitle%3DTime-Resolved%252BFRET-Based%252BAssays%252Bto%252BCharacterize%252BG%252BProtein-Coupled%252BReceptor%252BHetero-oligomer%252BPharmacology%26aulast%3DHeuninck%26aufirst%3DJoyce%26date%3D2019%26date%3D2019%26spage%3D151%26epage%3D168%26pub%3DSpringer%2520New%2520York%26atitle%3DG%252BProtein-Coupled%252BReceptor%252BSignaling%26aulast%3DTiberi%26aufirst%3DMario%26date%3D2019%26volume%3D1947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christos  Iliopoulos-Tsoutsouvas</span>, <span class="hlFld-ContribAuthor ">Rohit N.  Kulkarni</span>, <span class="hlFld-ContribAuthor ">Alexandros  Makriyannis</span>, <span class="hlFld-ContribAuthor ">Spyros P.  Nikas</span>. </span><span class="cited-content_cbyCitation_article-title">Fluorescent probes for G-protein-coupled receptor drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2018,</strong> <em>13 </em>
                                    (10)
                                     , 933-947. <a href="https://doi.org/10.1080/17460441.2018.1518975" title="DOI URL">https://doi.org/10.1080/17460441.2018.1518975</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2018.1518975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2018.1518975%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DFluorescent%252Bprobes%252Bfor%252BG-protein-coupled%252Breceptor%252Bdrug%252Bdiscovery%26aulast%3DIliopoulos-Tsoutsouvas%26aufirst%3DChristos%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D10%26spage%3D933%26epage%3D947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of Norasenapine <b>17</b> from Asenapinium Maleate <b>14</b> (Racemate)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (i) PL-TBD resin, CH<sub>2</sub>Cl<sub>2</sub>/MeOH, overnight, rt; (ii) ClCO<sub>2</sub>CHClCH<sub>3</sub>, 1,2-dichloroethane, 100 °C, 2 h; (iii) CH<sub>3</sub>OH, 100 °C, 2 h. Global yield (3 steps): 79%</p></p></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. General strategy for the labeling of asenapine and affinity (<i>K</i><sub>i</sub> in nM) of the parent compound for a set of GPCRs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Linkers <b>4</b>, <b>7</b>, and <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (i) TsCl, KI, Ag<sub>2</sub>O, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C, 15 min, 85%; (ii) NaN<sub>3</sub>, MeCN, reflux, 12 h, 91%; (iii) MsCl, NEt<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, rt, 2 h, 77%; (iv) HOCH<sub>2</sub>CHOHCH<sub>2</sub>NH<sub>2</sub>, PyBOP, <i>i</i>-Pr<sub>2</sub>EtN, DMF/CH<sub>2</sub>Cl<sub>2</sub>, rt, 12 h, 93%; (v) NaIO<sub>4</sub>, THF/H<sub>2</sub>O, rt, 1 h, 100%; (vi) <i>i</i>-Pr<sub>2</sub>EtN, DMF, rt, 4 h, 82%; (vii) aqueous LiOH, THF, rt, 12 h, 94%.</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Amines <b>19</b>, <b>21</b>, <b>23</b>, and <b>25</b> from Norasenapine <b>17</b> (Racemates)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (i) BocNHCH<sub>2</sub>-CHO, NaBH<sub>3</sub>CN, MeOH, rt, 2 h, 96%; (ii) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 2 h, 65–98%; (iii) <b>7</b>, NaBH<sub>3</sub>CN, MeOH, rt, 2 h, 79%; (iv) <b>4</b>, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, rt, 12 h, then 50 °C, sonication, 2 h, 47%; (v) PS-PPh<sub>3</sub>, THF/H<sub>2</sub>O, rt, 12 h, 95%, (vi) <b>13</b>, NaBH<sub>3</sub>CN, MeOH, rt, 2 h, 64%.</p></p></figure><figure data-id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Fluorescent Compounds <b>26</b>–<b>34</b> (Racemates) for TR-FRET Applications<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (i) dye succinimidyl ester, <i>i</i>-Pr<sub>2</sub>EtN, DMSO, rt, 4 h, 30–40%.</p></p></figure><figure data-id="fig2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding experiments with <b>31</b>: (A) binding saturation of <b>31</b> on dopamine D<sub>3a</sub> receptor; (B–D) specific binding of <b>31</b> on various receptors belonging to the dopamine and the histamine (B), the serotonin (C), and the adrenergic (D) receptor families; (E) competition binding on 5-HT<sub>2b</sub> receptor. <b>31</b> was used at 17 nM. Competitors were added at a concentration ranging from 10<sup>–12</sup> M to 10<sup>–4</sup> M. All curves result from pooled data obtained in at least three independent experiments. Data were fitted with Prism using the one binding site subroutine in the “binding-competitive” procedure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/medium/jm-2017-01220g_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Web of affinity. Schematic representation of the affinity profiles (<i>K</i><sub>d</sub> in nM) of the different fluorescent probes for a set of 34 class A GPCRs. Compounds <b>26</b>–<b>34</b> ligands are also identified in a coded manner as follows: SL-S = small linker-SFX; SL-D = small linker-DY647; LL-S = long linker-SFX; LL-D = long linker-DY647; PL-S = pegylated linker-SFX; PL-D = pegylated linker-DY647; VLL-S = very long linker-SFX; VLL-D = very long linker-DY647; VLL-F = very long linker-fluorescein.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01220/20180110/images/large/jm-2017-01220g_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01220&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i64">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63557" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63557" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 55 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Bockaert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pin, J.-P.</span><span> </span><span class="NLM_article-title">Molecular tinkering of G protein-coupled receptors: an evolutionary success</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1723</span><span class="NLM_x">–</span> <span class="NLM_lpage">1729</span><span class="refDoi"> DOI: 10.1093/emboj/18.7.1723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1093%2Femboj%2F18.7.1723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10202136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADyaK1MXivVylu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=1723-1729&author=J.+Bockaertauthor=J.-P.+Pin&title=Molecular+tinkering+of+G+protein-coupled+receptors%3A+an+evolutionary+success&doi=10.1093%2Femboj%2F18.7.1723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular tinkering of G protein-coupled receptors: an evolutionary success</span></div><div class="casAuthors">Bockaert, Joel; Pin, Jean Philippe</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1723-1729</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review, with 56 refs.  Among membrane-bound receptors, the G protein-coupled receptors (GPCRs) are certainly the most diverse.  They have been very successful during evolution, being capable of transducing messages as different as photons, org. odorants, nucleotides, nucleosides, peptides, lipids and proteins.  Indirect studies, as well as two-dimensional crystn. of rhodopsin, have led to a useful model of a common "central core", composed of seven transmembrane helical domains, and its structural modifications during activation.  There are at least six families of GPCRs showing no sequence similarity.  They use an amazing no. of different domains both to bind their ligands and to activate G proteins.  The fine-tuning of their coupling to G proteins is regulated by splicing, RNA editing and phosphorylation.  Some GPCRs have been found to form either homo- or heterodimers with a structurally different GPCR, but also with membrane-bound proteins having one transmembrane domain such as nina-A, odr-4 or RAMP, the latter being involved in their targeting, function and pharmacol.  Finally, some GPCRs are unfaithful to G proteins and interact directly, via their C-terminal domain, with proteins contg. PDZ and Enabled/VASP homol. (EVH)-like domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_j_RcAF1YIrVg90H21EOLACvtfcHk0lhzTLjDl6F-7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXivVylu7o%253D&md5=4759ee7df2733d92c073e1cad1cda9a7</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F18.7.1723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F18.7.1723%26sid%3Dliteratum%253Aachs%26aulast%3DBockaert%26aufirst%3DJ.%26aulast%3DPin%26aufirst%3DJ.-P.%26atitle%3DMolecular%2520tinkering%2520of%2520G%2520protein-coupled%2520receptors%253A%2520an%2520evolutionary%2520success%26jtitle%3DEMBO%2520J.%26date%3D1999%26volume%3D18%26spage%3D1723%26epage%3D1729%26doi%3D10.1093%2Femboj%2F18.7.1723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Gudermann, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schöneberg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, G.</span><span> </span><span class="NLM_article-title">Functional and structural complexity of signal transduction via G-protein-coupled receptors</span> <span class="citation_source-journal">Annu. Rev. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">399</span><span class="NLM_x">–</span> <span class="NLM_lpage">427</span><span class="refDoi"> DOI: 10.1146/annurev.neuro.20.1.399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1146%2Fannurev.neuro.20.1.399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=9056720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADyaK2sXhslSjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1997&pages=399-427&author=T.+Gudermannauthor=T.+Sch%C3%B6nebergauthor=G.+Schultz&title=Functional+and+structural+complexity+of+signal+transduction+via+G-protein-coupled+receptors&doi=10.1146%2Fannurev.neuro.20.1.399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Functional and structural complexity of signal transduction via G-protein-coupled receptors</span></div><div class="casAuthors">Gudermann, Thomas; Schoeneberg, Torsten; Schultz, Guenter</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Neuroscience</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">399-427</span>CODEN:
                <span class="NLM_cas:coden">ARNSD5</span>;
        ISSN:<span class="NLM_cas:issn">0147-006X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review with 181 refs.  A prerequisite for the maintenance of homeostasis in a living organism is fine-tuned communication between different cells.  The majority of extracellular signaling mols., such as hormones and neurotransmitters, interact with a three-protein transmembrane signaling system consisting of a receptor, a G protein, and an effector.  These single components interact sequentially and reversibly.  Considering that hundreds of G-protein-coupled receptors interact with a limited repertoire of G proteins, the question of coupling specificity is worth considering.  G-protein-mediated signal transduction is a complex signaling network with diverging and converging transduction steps at each coupling interface.  The recent realization that classical signaling pathways are intimately intertwined with growth-factor-signaling cascades adds another level of complexity.  Elaborate studies have significantly enhanced our knowledge of the functional anatomy of G-protein-coupled receptors, and the concept has emerged that receptor function can be modulated with high specificity by coexpressed receptor fragments.  These results may have significant clin. impact in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonpBt3YtTc9bVg90H21EOLACvtfcHk0lhzTLjDl6F-7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhslSjt7Y%253D&md5=e26edfe9e07244131d8d22d34cb68c2d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1146%2Fannurev.neuro.20.1.399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.neuro.20.1.399%26sid%3Dliteratum%253Aachs%26aulast%3DGudermann%26aufirst%3DT.%26aulast%3DSch%25C3%25B6neberg%26aufirst%3DT.%26aulast%3DSchultz%26aufirst%3DG.%26atitle%3DFunctional%2520and%2520structural%2520complexity%2520of%2520signal%2520transduction%2520via%2520G-protein-coupled%2520receptors%26jtitle%3DAnnu.%2520Rev.%2520Neurosci.%26date%3D1997%26volume%3D20%26spage%3D399%26epage%3D427%26doi%3D10.1146%2Fannurev.neuro.20.1.399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Heng, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aubel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fussenegger, M.</span><span> </span><span class="NLM_article-title">An overview of the diverse roles of G-protein coupled receptors (GPCRs) in the pathophysiology of various human diseases</span> <span class="citation_source-journal">Biotechnol. Adv.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">1676</span><span class="NLM_x">–</span> <span class="NLM_lpage">1694</span><span class="refDoi"> DOI: 10.1016/j.biotechadv.2013.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2Fj.biotechadv.2013.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=23999358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVKgu7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=1676-1694&author=B.+C.+Hengauthor=D.+Aubelauthor=M.+Fussenegger&title=An+overview+of+the+diverse+roles+of+G-protein+coupled+receptors+%28GPCRs%29+in+the+pathophysiology+of+various+human+diseases&doi=10.1016%2Fj.biotechadv.2013.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">An overview of the diverse roles of G-protein coupled receptors (GPCRs) in the pathophysiology of various human diseases</span></div><div class="casAuthors">Heng, Boon Chin; Aubel, Dominique; Fussenegger, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Advances</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1676-1694</span>CODEN:
                <span class="NLM_cas:coden">BIADDD</span>;
        ISSN:<span class="NLM_cas:issn">0734-9750</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  G-protein coupled receptors (GPCRs) modulate diverse cellular responses to the majority of neurotransmitters and hormones within the human body.  They exhibit much structural and functional diversity, and are responsive to a plethora of endogenous (biogenic amines, cations, lipids, peptides, and glycoproteins) and exogenous (therapeutic drugs, photons, tastants, and odorants) ligands and stimuli.  Due to the key roles of GPCRs in tissue/cell physiol. and homeostasis, signaling pathways assocd. with GPCRs are implicated in the pathophysiol. of various diseases, ranging from metabolic, immunol., and neurodegenerative disorders, to cancer and infectious diseases.  Approx. 40% of clin. approved drugs mediate their effects by modulating GPCR signaling pathways, which makes them attractive targets for drug screening and discovery.  The pace of discovery of new GPCR-based drugs has recently accelerated due to rapid advancements in high-resoln. structure detn., high-throughput screening technol. and in silico computational modeling of GPCR binding interaction with potential drug mols.  This review aims to provide an overview of the diverse roles of GPCRs in the pathophysiol. of various diseases that are the major focus of biopharmaceutical research as potential drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraGad80-LA-bVg90H21EOLACvtfcHk0ljpRxNbMbOBkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVKgu7%252FF&md5=ab10a667170b8fe34dee4bee63a8b73a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.biotechadv.2013.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biotechadv.2013.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DHeng%26aufirst%3DB.%2BC.%26aulast%3DAubel%26aufirst%3DD.%26aulast%3DFussenegger%26aufirst%3DM.%26atitle%3DAn%2520overview%2520of%2520the%2520diverse%2520roles%2520of%2520G-protein%2520coupled%2520receptors%2520%2528GPCRs%2529%2520in%2520the%2520pathophysiology%2520of%2520various%2520human%2520diseases%26jtitle%3DBiotechnol.%2520Adv.%26date%3D2013%26volume%3D31%26spage%3D1676%26epage%3D1694%26doi%3D10.1016%2Fj.biotechadv.2013.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Drews, J.</span><span> </span><span class="NLM_article-title">Drug discovery: a historical perspective</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">1960</span><span class="NLM_x">–</span> <span class="NLM_lpage">1964</span><span class="refDoi"> DOI: 10.1126/science.287.5460.1960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1126%2Fscience.287.5460.1960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10720314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvFyrtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2000&pages=1960-1964&author=J.+Drews&title=Drug+discovery%3A+a+historical+perspective&doi=10.1126%2Fscience.287.5460.1960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery: A historical perspective</span></div><div class="casAuthors">Drews, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">5460</span>),
    <span class="NLM_cas:pages">1960-1963</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review with over 60 refs.  Driven by chem. but increasingly guided by pharmacol. and the clin. sciences, drug research has contributed more to the progress of medicine during the past century than any other scientific factor.  The advent of mol. biol. and, in particular, of genomic sciences is having a deep impact on drug discovery.  Recombinant proteins and monoclonal antibodies have greatly enriched our therapeutic armamentarium.  Genome sciences, combined with bioinformatic tools, allow us to dissect the genetic basis of multifactorial diseases and to det. the most suitable points of attack for future medicines, thereby increasing the no. of treatment options.  The dramatic increase in the complexity of drug research is enforcing changes in the institutional basis of this interdisciplinary endeavor.  The biotech industry is establishing itself as the discovery arm of the pharmaceutical industry.  In bridging the gap between academia and large pharmaceutical companies, the biotech firms have been effective instruments of technol. transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdT9qeMTdSTrVg90H21EOLACvtfcHk0ljpRxNbMbOBkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvFyrtbc%253D&md5=33804292ba1e2fa5fd8012729d137dff</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.287.5460.1960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.287.5460.1960%26sid%3Dliteratum%253Aachs%26aulast%3DDrews%26aufirst%3DJ.%26atitle%3DDrug%2520discovery%253A%2520a%2520historical%2520perspective%26jtitle%3DScience%26date%3D2000%26volume%3D287%26spage%3D1960%26epage%3D1964%26doi%3D10.1126%2Fscience.287.5460.1960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span> </span><span class="NLM_article-title">The druggable genome</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">730</span><span class="refDoi"> DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-730&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+druggable+genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0ljpRxNbMbOBkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520druggable%2520genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D730%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Santos, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ursu, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donadi, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bologa, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlsson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">A comprehensive map of molecular drug targets</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">19</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1038/nrd.2016.230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnrd.2016.230" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=27910877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVSms7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=19-34&author=R.+Santosauthor=O.+Ursuauthor=A.+Gaultonauthor=A.+P.+Bentoauthor=R.+S.+Donadiauthor=C.+G.+Bologaauthor=A.+Karlssonauthor=B.+Al-Lazikaniauthor=A.+Herseyauthor=T.+I.+Opreaauthor=J.+P.+Overington&title=A+comprehensive+map+of+molecular+drug+targets&doi=10.1038%2Fnrd.2016.230"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A comprehensive map of molecular drug targets</span></div><div class="casAuthors">Santos, Rita; Ursu, Oleg; Gaulton, Anna; Bento, A. Patricia; Donadi, Ramesh S.; Bologa, Cristian G.; Karlsson, Anneli; Al-Lazikani, Bissan; Hersey, Anne; Oprea, Tudor I.; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-34</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The success of mechanism-based drug discovery depends on the definition of the drug target.  This definition becomes even more important as we try to link drug response to genetic variation, understand stratified clin. efficacy and safety, rationalize the differences between drugs in the same therapeutic class and predict drug utility in patient subgroups.  However, drug targets are often poorly defined in the literature, both for launched drugs and for potential therapeutic agents in discovery and development.  Here, we present an updated comprehensive map of mol. targets of approved drugs.  We curate a total of 893 human and pathogen-derived biomols. through which 1,578 US FDA-approved drugs act.  These biomols. include 667 human-genome-derived proteins targeted by drugs for human disease.  Anal. of these drug targets indicates the continued dominance of privileged target families across disease areas, but also the growth of novel first-in-class mechanisms, particularly in oncol.  We explore the relationships between bioactivity class and clin. success, as well as the presence of orthologues between human and animal models and between pathogen and human genomes.  Through the collaboration of three independent teams, we highlight some of the ongoing challenges in accurately defining the targets of mol. therapeutics and present conventions for deconvoluting the complexities of mol. pharmacol. and drug efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTfWwWkoZnbrVg90H21EOLACvtfcHk0lgZscCgkS6XHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVSms7rK&md5=7437d4b678ac75dac0a5e3248df98761</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.230%26sid%3Dliteratum%253Aachs%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DUrsu%26aufirst%3DO.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DDonadi%26aufirst%3DR.%2BS.%26aulast%3DBologa%26aufirst%3DC.%2BG.%26aulast%3DKarlsson%26aufirst%3DA.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DA%2520comprehensive%2520map%2520of%2520molecular%2520drug%2520targets%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D19%26epage%3D34%26doi%3D10.1038%2Fnrd.2016.230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Garland, S. L.</span><span> </span><span class="NLM_article-title">Are GPCRs still a source of new targets?</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">947</span><span class="NLM_x">–</span> <span class="NLM_lpage">966</span><span class="refDoi"> DOI: 10.1177/1087057113498418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1177%2F1087057113498418" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=23945874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWitrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=947-966&author=S.+L.+Garland&title=Are+GPCRs+still+a+source+of+new+targets%3F&doi=10.1177%2F1087057113498418"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Are GPCRs still a source of new targets?</span></div><div class="casAuthors">Garland, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">947-966</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  G-protein-coupled receptors (GPCRs) still offer enormous scope for new therapeutic targets.  Currently marketed agents are dominated by those with activity at aminergic receptors and yet they account for only ∼10% of the family.  Progress up until now with other subfamilies, notably orphans, Family A/peptide, Family A/lipid, Family B, Family C, and Family F, has been, at best, patchy.  This may be attributable to the heterogeneous nature of GPCRs, their endogenous ligands, and consequently their binding sites.  Our appreciation of receptor similarity has arguably been too simplistic, and screening collections have not necessarily been well suited to identifying leads in new areas.  Despite the relative shortage of highquality tool mols. in a no. of cases, there is an emerging, and increasingly substantial, body of evidence assocg. many as yet "undrugged" receptors with a very wide range of diseases.  Significant advances in our understanding of receptor pharmacol. and tech. advances in screening, protein X-ray crystallog., and ligand design methods are paving the way for new successes in the area.  Exploitation of allosteric mechanisms; alternative signaling pathways such as G12/13, Gβγ, and β-arrestin; the discovery of "biased" ligands; and the emergence of GPCR-protein complexes as potential drug targets offer scope for new and much improved drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjdjNgoJdJ5LVg90H21EOLACvtfcHk0liqNyXdyDJHtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWitrzK&md5=5bd5fd56a4af9ff828854aa8128f7e1a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1177%2F1087057113498418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113498418%26sid%3Dliteratum%253Aachs%26aulast%3DGarland%26aufirst%3DS.%2BL.%26atitle%3DAre%2520GPCRs%2520still%2520a%2520source%2520of%2520new%2520targets%253F%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26spage%3D947%26epage%3D966%26doi%3D10.1177%2F1087057113498418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Dixon, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strader, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benovic, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohlman, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frielle, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolanowski, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rands, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diehl, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mumford, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slater, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigal, I. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caron, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lefkowitz, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strader, C. D.</span><span> </span><span class="NLM_article-title">Cloning of the gene and cDNA for mammalian beta-adrenergic receptor and homology with rhodopsin</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">321</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">79</span><span class="refDoi"> DOI: 10.1038/321075a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2F321075a0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=1986&pages=75-79&author=R.+A.+Dixonauthor=B.+K.+Kobilkaauthor=D.+J.+Straderauthor=J.+L.+Benovicauthor=H.+G.+Dohlmanauthor=T.+Frielleauthor=M.+A.+Bolanowskiauthor=C.+D.+Bennettauthor=E.+Randsauthor=R.+E.+Diehlauthor=R.+A.+Mumfordauthor=E.+E.+Slaterauthor=I.+S.+Sigalauthor=M.+G.+Caronauthor=R.+J.+Lefkowitzauthor=C.+D.+Strader&title=Cloning+of+the+gene+and+cDNA+for+mammalian+beta-adrenergic+receptor+and+homology+with+rhodopsin&doi=10.1038%2F321075a0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2F321075a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F321075a0%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DR.%2BA.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DStrader%26aufirst%3DD.%2BJ.%26aulast%3DBenovic%26aufirst%3DJ.%2BL.%26aulast%3DDohlman%26aufirst%3DH.%2BG.%26aulast%3DFrielle%26aufirst%3DT.%26aulast%3DBolanowski%26aufirst%3DM.%2BA.%26aulast%3DBennett%26aufirst%3DC.%2BD.%26aulast%3DRands%26aufirst%3DE.%26aulast%3DDiehl%26aufirst%3DR.%2BE.%26aulast%3DMumford%26aufirst%3DR.%2BA.%26aulast%3DSlater%26aufirst%3DE.%2BE.%26aulast%3DSigal%26aufirst%3DI.%2BS.%26aulast%3DCaron%26aufirst%3DM.%2BG.%26aulast%3DLefkowitz%26aufirst%3DR.%2BJ.%26aulast%3DStrader%26aufirst%3DC.%2BD.%26atitle%3DCloning%2520of%2520the%2520gene%2520and%2520cDNA%2520for%2520mammalian%2520beta-adrenergic%2520receptor%2520and%2520homology%2520with%2520rhodopsin%26jtitle%3DNature%26date%3D1986%26volume%3D321%26spage%3D75%26epage%3D79%26doi%3D10.1038%2F321075a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Garland, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gloriam, D. E.</span><span> </span><span class="NLM_article-title">A ligand’s view of target similarity: chemogenomic binding site-directed techniques for drug discovery</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1872</span><span class="NLM_x">–</span> <span class="NLM_lpage">1881</span><span class="refDoi"> DOI: 10.2174/156802611796391276</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.2174%2F156802611796391276" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=21470171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVSmsL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=1872-1881&author=S.+L.+Garlandauthor=D.+E.+Gloriam&title=A+ligand%E2%80%99s+view+of+target+similarity%3A+chemogenomic+binding+site-directed+techniques+for+drug+discovery&doi=10.2174%2F156802611796391276"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A ligand's view of target similarity: chemogenomic binding site-directed techniques for drug discovery</span></div><div class="casAuthors">Garland, Stephen L.; Gloriam, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1872-1881</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  GPCR binding site-directed techniques are rapidly evolving into powerful tools for modern drug discovery.  Many of these approaches bridge chem. and biol., which are inseparable concepts in nature but are often treated as sep. worlds in drug discovery and science in general.  This review shows with several examples how focusing on the binding site(s) has a clear advantage when it comes to establishing sequence-correlated pharmacol. profiles.  By organizing and comparing sequence and structural data it is possible to "borrow" SAR from similar targets to increase the speed of lead-finding and, potentially, to produce ligands for previously intractable receptors.  Sequence motifs correlated with ligands can be applied in the design of target-specific focused libraries that are both efficient and cost-effective and should provide increased hit-rates over diversity screening.  Furthermore, in the optimization phase, the binding motif approach offers the possibility to identify quickly the most likely off-target candidates to be chosen for selectivity screening, as well as potentially characterizing those pockets which may best be exploited for improved selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9ppSbGEytS7Vg90H21EOLACvtfcHk0liqNyXdyDJHtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVSmsL%252FL&md5=c7c9f29801172934eabc8cbf9e294574</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2174%2F156802611796391276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611796391276%26sid%3Dliteratum%253Aachs%26aulast%3DGarland%26aufirst%3DS.%2BL.%26aulast%3DGloriam%26aufirst%3DD.%2BE.%26atitle%3DA%2520ligand%25E2%2580%2599s%2520view%2520of%2520target%2520similarity%253A%2520chemogenomic%2520binding%2520site-directed%2520techniques%2520for%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26spage%3D1872%26epage%3D1881%26doi%3D10.2174%2F156802611796391276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Hibert, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trumpp-Kallmeyer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruinvels, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoflack, J.</span><span> </span><span class="NLM_article-title">Three-dimensional models of neurotransmitter G-binding protein-coupled receptors</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">8</span><span class="NLM_x">–</span> <span class="NLM_lpage">15</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1991&pages=8-15&author=M.+F.+Hibertauthor=S.+Trumpp-Kallmeyerauthor=A.+Bruinvelsauthor=J.+Hoflack&title=Three-dimensional+models+of+neurotransmitter+G-binding+protein-coupled+receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHibert%26aufirst%3DM.%2BF.%26aulast%3DTrumpp-Kallmeyer%26aufirst%3DS.%26aulast%3DBruinvels%26aufirst%3DA.%26aulast%3DHoflack%26aufirst%3DJ.%26atitle%3DThree-dimensional%2520models%2520of%2520neurotransmitter%2520G-binding%2520protein-coupled%2520receptors%26jtitle%3DMol.%2520Pharmacol.%26date%3D1991%26volume%3D40%26spage%3D8%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Trumpp-Kallmeyer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoflack, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruinvels, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hibert, M.</span><span> </span><span class="NLM_article-title">Modeling of G-protein-coupled receptors: application to dopamine, adrenaline, serotonin, acetylcholine, and mammalian opsin receptors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">3448</span><span class="NLM_x">–</span> <span class="NLM_lpage">3462</span><span class="refDoi"> DOI: 10.1021/jm00097a002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00097a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1992&pages=3448-3462&author=S.+Trumpp-Kallmeyerauthor=J.+Hoflackauthor=A.+Bruinvelsauthor=M.+Hibert&title=Modeling+of+G-protein-coupled+receptors%3A+application+to+dopamine%2C+adrenaline%2C+serotonin%2C+acetylcholine%2C+and+mammalian+opsin+receptors&doi=10.1021%2Fjm00097a002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm00097a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00097a002%26sid%3Dliteratum%253Aachs%26aulast%3DTrumpp-Kallmeyer%26aufirst%3DS.%26aulast%3DHoflack%26aufirst%3DJ.%26aulast%3DBruinvels%26aufirst%3DA.%26aulast%3DHibert%26aufirst%3DM.%26atitle%3DModeling%2520of%2520G-protein-coupled%2520receptors%253A%2520application%2520to%2520dopamine%252C%2520adrenaline%252C%2520serotonin%252C%2520acetylcholine%252C%2520and%2520mammalian%2520opsin%2520receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1992%26volume%3D35%26spage%3D3448%26epage%3D3462%26doi%3D10.1021%2Fjm00097a002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Klabunde, T.</span><span> </span><span class="NLM_article-title">Chemogenomic approaches to drug discovery: similar receptors bind similar ligands</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">152</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0707308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fsj.bjp.0707308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=17533415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFOjur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2007&pages=5-7&author=T.+Klabunde&title=Chemogenomic+approaches+to+drug+discovery%3A+similar+receptors+bind+similar+ligands&doi=10.1038%2Fsj.bjp.0707308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic approaches to drug discovery: similar receptors bind similar ligands</span></div><div class="casAuthors">Klabunde, T.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-7</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Within recent years, a paradigm shift from traditional receptor-specific studies to a cross-receptor view has taken place within pharmaceutical research to increase the efficiency of modern drug discovery.  Receptors are no longer viewed as single entities but grouped into sets of related proteins or receptor families that are explored in a systematic manner.  This interdisciplinary approach attempting to derive predictive links between the chem. structures of bioactive mols. and the receptors with which these mols. interact is referred to as chemogenomics.  Insights from chemogenomics are used for the rational compilation of screening sets and for the rational design and synthesis of directed chem. libraries to accelerate drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQeX3iyp_Z47Vg90H21EOLACvtfcHk0ljsqjc3tk40hA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFOjur0%253D&md5=a103cdd189d2bad6d70e799aad869290</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0707308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0707308%26sid%3Dliteratum%253Aachs%26aulast%3DKlabunde%26aufirst%3DT.%26atitle%3DChemogenomic%2520approaches%2520to%2520drug%2520discovery%253A%2520similar%2520receptors%2520bind%2520similar%2520ligands%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2007%26volume%3D152%26spage%3D5%26epage%3D7%26doi%3D10.1038%2Fsj.bjp.0707308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Paolini, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapland, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Hoorn, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Global mapping of pharmacological space</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">805</span><span class="NLM_x">–</span> <span class="NLM_lpage">815</span><span class="refDoi"> DOI: 10.1038/nbt1228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnbt1228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=16841068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=805-815&author=G.+V.+Paoliniauthor=R.+H.+Shaplandauthor=W.+P.+van+Hoornauthor=J.+S.+Masonauthor=A.+L.+Hopkins&title=Global+mapping+of+pharmacological+space&doi=10.1038%2Fnbt1228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Global mapping of pharmacological space</span></div><div class="casAuthors">Paolini, Gaia V.; Shapland, Richard H. B.; van Hoorn, Willem P.; Mason, Jonathan S.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">805-815</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We present the global mapping of pharmacol. space by the integration of several vast sources of medicinal chem. structure-activity relationships (SAR) data.  Our comprehensive mapping of pharmacol. space enables us to identify confidently the human targets for which chem. tools and drugs have been discovered to date.  The integration of SAR data from diverse sources by unique canonical chem. structure, protein sequence and disease indication enables the construction of a ligand-target matrix to explore the global relationships between chem. structure and biol. targets.  Using the data matrix, we are able to catalog the links between proteins in chem. space as a polypharmacol. interaction network.  We demonstrate that probabilistic models can be used to predict pharmacol. from a large knowledge base.  The relationships between proteins, chem. structures and drug-like properties provide a framework for developing a probabilistic approach to drug discovery that can be exploited to increase research productivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV_CE47zZq8LVg90H21EOLACvtfcHk0ljsqjc3tk40hA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D&md5=f559b34692cc903a1b503deb07030c5d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnbt1228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1228%26sid%3Dliteratum%253Aachs%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DShapland%26aufirst%3DR.%2BH.%26aulast%3Dvan%2BHoorn%26aufirst%3DW.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DGlobal%2520mapping%2520of%2520pharmacological%2520space%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26spage%3D805%26epage%3D815%26doi%3D10.1038%2Fnbt1228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armbruster, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernsberger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">206</span><span class="refDoi"> DOI: 10.1038/nbt1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197-206&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry&doi=10.1038%2Fnbt1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, α2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0lgUCpeYOssd7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D197%26epage%3D206%26doi%3D10.1038%2Fnbt1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laggner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbas, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hufeisen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuijer, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matos, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whaley, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glennon, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span> </span><span class="NLM_article-title">Predicting new molecular targets for known drugs</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">462</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">181</span><span class="refDoi"> DOI: 10.1038/nature08506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnature08506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=19881490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=175-181&author=M.+J.+Keiserauthor=V.+Setolaauthor=J.+J.+Irwinauthor=C.+Laggnerauthor=A.+I.+Abbasauthor=S.+J.+Hufeisenauthor=N.+H.+Jensenauthor=M.+B.+Kuijerauthor=R.+C.+Matosauthor=T.+B.+Tranauthor=R.+Whaleyauthor=R.+A.+Glennonauthor=J.+Hertauthor=K.+L.+Thomasauthor=D.+D.+Edwardsauthor=B.+K.+Shoichetauthor=B.+L.+Roth&title=Predicting+new+molecular+targets+for+known+drugs&doi=10.1038%2Fnature08506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting new molecular targets for known drugs</span></div><div class="casAuthors">Keiser, Michael J.; Setola, Vincent; Irwin, John J.; Laggner, Christian; Abbas, Atheir I.; Hufeisen, Sandra J.; Jensen, Niels H.; Kuijer, Michael B.; Matos, Roberto C.; Tran, Thuy B.; Whaley, Ryan; Glennon, Richard A.; Hert, Jerome; Thomas, Kelan L. H.; Edwards, Douglas D.; Shoichet, Brian K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7270</span>),
    <span class="NLM_cas:pages">175-181</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although drugs are intended to be selective, at least some bind to several physiol. targets, explaining side effects and efficacy.  Because many drug-target combinations exist, it would be useful to explore possible interactions computationally.  Here we compared 3665 US Food and Drug Administration (FDA)-approved and investigational drugs against hundreds of targets, defining each target by its ligands.  Chem. similarities between drugs and ligand sets predicted thousands of unanticipated assocns.  Thirty were tested exptl., including the antagonism of the β1 receptor by the transporter inhibitor Prozac, the inhibition of the 5-hydroxytryptamine (5-HT) transporter by the ion channel drug Vadilex, and antagonism of the histamine H4 receptor by the enzyme inhibitor Rescriptor.  Overall, 23 new drug-target assocns. were confirmed, five of which were potent (<100 nM).  The physiol. relevance of one, the drug N,N-dimethyltryptamine (DMT) on serotonergic receptors, was confirmed in a knockout mouse.  The chem. similarity approach is systematic and comprehensive, and may suggest side-effects and new indications for many drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozd-8cCXUskrVg90H21EOLACvtfcHk0lgUCpeYOssd7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO&md5=974e65f5d5b4dabde1a463c429215546</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature08506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08506%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DLaggner%26aufirst%3DC.%26aulast%3DAbbas%26aufirst%3DA.%2BI.%26aulast%3DHufeisen%26aufirst%3DS.%2BJ.%26aulast%3DJensen%26aufirst%3DN.%2BH.%26aulast%3DKuijer%26aufirst%3DM.%2BB.%26aulast%3DMatos%26aufirst%3DR.%2BC.%26aulast%3DTran%26aufirst%3DT.%2BB.%26aulast%3DWhaley%26aufirst%3DR.%26aulast%3DGlennon%26aufirst%3DR.%2BA.%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DK.%2BL.%26aulast%3DEdwards%26aufirst%3DD.%2BD.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DPredicting%2520new%2520molecular%2520targets%2520for%2520known%2520drugs%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D175%26epage%3D181%26doi%3D10.1038%2Fnature08506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Gregori-Puigjané, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestres, J.</span><span> </span><span class="NLM_article-title">A ligand-based approach to mining the chemogenomic space of drugs</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">676</span><span class="refDoi"> DOI: 10.2174/138620708785739952</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.2174%2F138620708785739952" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=18795886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivFentbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2008&pages=669-676&author=E.+Gregori-Puigjan%C3%A9author=J.+Mestres&title=A+ligand-based+approach+to+mining+the+chemogenomic+space+of+drugs&doi=10.2174%2F138620708785739952"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A ligand-based approach to mining the chemogenomic space of drugs</span></div><div class="casAuthors">Gregori-Puigjane, Elisabet; Mestres, Jordi</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">669-676</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The practical implementation and validation of a ligand-based approach to mining the chemogenomic space of drugs is presented and applied to the in silico target profiling of 767 drugs against 684 targets of therapeutic relevance.  The results reveal that drugs targeting aminergic G protein-coupled receptors (GPCRs) show the most promiscuous pharmacol. profiles.  The detection of cross-pharmacologies between aminergic GPCRs and the opioid, sigma, NMDA, and 5-HT3 receptors aggravate the potential promiscuity of those drugs, predominantly including analgesics, antidepressants, and antipsychotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3IqM_DIwJhbVg90H21EOLACvtfcHk0lgUCpeYOssd7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivFentbc%253D&md5=02d2db8cfa6711fb16ef362c99c5dbd0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.2174%2F138620708785739952&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620708785739952%26sid%3Dliteratum%253Aachs%26aulast%3DGregori-Puigjan%25C3%25A9%26aufirst%3DE.%26aulast%3DMestres%26aufirst%3DJ.%26atitle%3DA%2520ligand-based%2520approach%2520to%2520mining%2520the%2520chemogenomic%2520space%2520of%2520drugs%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2008%26volume%3D11%26spage%3D669%26epage%3D676%26doi%3D10.2174%2F138620708785739952" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Faure, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubus, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijjaali, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morlière, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barberan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petitet, F.</span><span> </span><span class="NLM_article-title">Knowledge-based analysis of multi-potent G-protein coupled receptors ligands</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">5708</span><span class="NLM_x">–</span> <span class="NLM_lpage">5717</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2010.09.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2Fj.ejmech.2010.09.027" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2010&pages=5708-5717&author=P.+Faureauthor=E.+Dubusauthor=I.+Ijjaaliauthor=C.+Morli%C3%A8reauthor=O.+Barberanauthor=F.+Petitet&title=Knowledge-based+analysis+of+multi-potent+G-protein+coupled+receptors+ligands&doi=10.1016%2Fj.ejmech.2010.09.027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2010.09.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2010.09.027%26sid%3Dliteratum%253Aachs%26aulast%3DFaure%26aufirst%3DP.%26aulast%3DDubus%26aufirst%3DE.%26aulast%3DIjjaali%26aufirst%3DI.%26aulast%3DMorli%25C3%25A8re%26aufirst%3DC.%26aulast%3DBarberan%26aufirst%3DO.%26aulast%3DPetitet%26aufirst%3DF.%26atitle%3DKnowledge-based%2520analysis%2520of%2520multi-potent%2520G-protein%2520coupled%2520receptors%2520ligands%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2010%26volume%3D45%26spage%3D5708%26epage%3D5717%26doi%3D10.1016%2Fj.ejmech.2010.09.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Briansó, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrascosa, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestres, J.</span><span> </span><span class="NLM_article-title">Cross-pharmacology analysis of G protein-coupled receptors</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1956</span><span class="NLM_x">–</span> <span class="NLM_lpage">1963</span><span class="refDoi"> DOI: 10.2174/156802611796391285</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.2174%2F156802611796391285" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=21851335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVSmsLzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=1956-1963&author=F.+Brians%C3%B3author=M.+C.+Carrascosaauthor=T.+I.+Opreaauthor=J.+Mestres&title=Cross-pharmacology+analysis+of+G+protein-coupled+receptors&doi=10.2174%2F156802611796391285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Cross-pharmacology analysis of G protein-coupled receptors</span></div><div class="casAuthors">Brianso, Ferran; Carrascosa, Maria C.; Oprea, Tudor I.; Mestres, Jordi</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1956-1963</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The degree of applicability of chemogenomic approaches to protein families depends on the accuracy and completeness of pharmacol. data and the corresponding level of pharmacol. similarity obsd. among their protein members.  The recent public domain availability of pharmacol. data for thousands of small mols. on 204 G protein-coupled receptors (GPCRs) provides a firm basis for an in-depth cross-pharmacol. anal. of this superfamily.  The no. of protein targets included in the cross-pharmacol. profile of the different GPCRs changes significantly upon varying the ligand similarity and binding affinity criteria.  However, with the exception of muscarinic receptors, aminergic GPCRs distinguish themselves from the rest of the members in the family by their remarkably high levels of pharmacol. similarity among them.  Clusters of non-GPCR targets related by cross-pharmacol. with particular GPCRs are identified and the implications for unwanted side-effects, as well as for repurposing opportunities, discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcxMaSMU_YxLVg90H21EOLACvtfcHk0ljenXDR8sWkoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVSmsLzO&md5=7805626c0844e84d16b56ed953823486</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.2174%2F156802611796391285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611796391285%26sid%3Dliteratum%253Aachs%26aulast%3DBrians%25C3%25B3%26aufirst%3DF.%26aulast%3DCarrascosa%26aufirst%3DM.%2BC.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DMestres%26aufirst%3DJ.%26atitle%3DCross-pharmacology%2520analysis%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26spage%3D1956%26epage%3D1963%26doi%3D10.2174%2F156802611796391285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Weill, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">J Development and validation of a novel protein-ligand fingerprint to mine chemogenomic space: application to G protein-coupled receptors and their ligands</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1049</span><span class="NLM_x">–</span> <span class="NLM_lpage">1062</span><span class="refDoi"> DOI: 10.1021/ci800447g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800447g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsVKktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=1049-1062&author=N.+Weillauthor=D.+Rognan&title=J+Development+and+validation+of+a+novel+protein-ligand+fingerprint+to+mine+chemogenomic+space%3A+application+to+G+protein-coupled+receptors+and+their+ligands&doi=10.1021%2Fci800447g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Validation of a Novel Protein-Ligand Fingerprint To Mine Chemogenomic Space: Application to G Protein-Coupled Receptors and Their Ligands</span></div><div class="casAuthors">Weill, Nathanael; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1049-1062</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The present study introduces a novel low-dimensionality fingerprint encoding both ligand and target properties which is suitable to mine protein-ligand chemogenomic space.  Whereas ligand properties have been represented by std. descriptors, protein cavities are encoded by a fixed length bit string describing pharmacophoric properties of a definite no. of binding site residues.  In order to simplify the cavity fingerprint, the concept was applied here to a unique family of targets (G protein-coupled receptors) with a homogeneous cavity description.  Particular attention was given to set up data sets of really diverse protein-ligand pairs covering as exhaustively as possible both ligand and target spaces.  Several machine learning classification algorithms were trained on two sets of roughly 200,000 receptor-ligand fingerprints with a different definition of inactive decoys.  Cross-validated models show excellent precision (>0.9) in distinguishing true from false pairs with a particular preference for support vector machine classifiers.  When applied to two external test sets of GPCR ligands, the most predictive models were not those performing the best in the previous cross-validation.  The ability to recover true GPCR ligands (ligand prediction mode) or true GPCRs (receptor prediction mode) depends on multiple parameters: the mol. complexity of the ligands, the chem. space from which ligand decoys are selected to generate false protein-ligand pairs, and the target space under consideration.  In most cases, predicting ligands is easier than predicting receptors.  Although receptor profiling is possible, it probably requires a more detailed description of the ligand-binding site.  Noteworthy, protein-ligand fingerprints outperform the corresponding ligand fingerprints in mining the GPCR-ligand space.  Since they can be applied to a much larger no. of receptors than ligand-based fingerprints, protein-ligand fingerprints represent a novel and promising way to directly screen protein-ligand pairs in chemogenomic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHYTjaMcCFi7Vg90H21EOLACvtfcHk0ljenXDR8sWkoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsVKktLs%253D&md5=afd7e1e8beb8d28b68ea7fe574e4d32e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci800447g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800447g%26sid%3Dliteratum%253Aachs%26aulast%3DWeill%26aufirst%3DN.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DJ%2520Development%2520and%2520validation%2520of%2520a%2520novel%2520protein-ligand%2520fingerprint%2520to%2520mine%2520chemogenomic%2520space%253A%2520application%2520to%2520G%2520protein-coupled%2520receptors%2520and%2520their%2520ligands%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D1049%26epage%3D1062%26doi%3D10.1021%2Fci800447g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhne, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">A structural chemogenomics analysis of aminergic GPCRs: lessons for histamine receptor ligand design.</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">170</span><span class="NLM_x">, </span> <span class="NLM_fpage">101</span><span class="NLM_x">–</span> <span class="NLM_lpage">126</span><span class="refDoi"> DOI: 10.1111/bph.12248</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1111%2Fbph.12248" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=23713847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlSgtbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2013&pages=101-126&author=A.+J.+Kooistraauthor=S.+Kuhneauthor=I.+J.+de+Eschauthor=R.+Leursauthor=C.+de+Graaf&title=A+structural+chemogenomics+analysis+of+aminergic+GPCRs%3A+lessons+for+histamine+receptor+ligand+design.&doi=10.1111%2Fbph.12248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A structural chemogenomics analysis of aminergic GPCRs: lessons for histamine receptor ligand design</span></div><div class="casAuthors">Kooistra, A. J.; Kuhne, S.; de Esch, I. J. P.; Leurs, R.; de Graaf, C.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">101-126</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Background and Purpose Chemogenomics focuses on the discovery of new connections between chem. and biol. space leading to the discovery of new protein targets and biol. active mols.  G-protein coupled receptors (GPCRs) are a particularly interesting protein family for chemogenomics studies because there is an overwhelming amt. of ligand binding affinity data available.  The increasing no. of aminergic GPCR crystal structures now for the first time allows the integration of chemogenomics studies with high-resoln. structural analyses of GPCR-ligand complexes.  Exptl. Approach In this study, we have combined ligand affinity data, receptor mutagenesis studies, and amino acid sequence analyses to high-resoln. structural analyses of (hist)aminergic GPCR-ligand interactions.  This integrated structural chemogenomics anal. is used to more accurately describe the mol. and structural determinants of ligand affinity and selectivity in different key binding regions of the crystd. aminergic GPCRs, and histamine receptors in particular.  Key Results Our investigations highlight interesting correlations and differences between ligand similarity and ligand binding site similarity of different aminergic receptors.  Apparent discrepancies can be explained by combining detailed anal. of crystd. or predicted protein-ligand binding modes, receptor mutation studies, and ligand structure-selectivity relationships that identify local differences in essential pharmacophore features in the ligand binding sites of different receptors.  Conclusions and Implications We have performed structural chemogenomics studies that identify links between (hist)aminergic receptor ligands and their binding sites and binding modes.  This knowledge can be used to identify structure-selectivity relationships that increase our understanding of ligand binding to (hist)aminergic receptors and hence can be used in future GPCR ligand discovery and design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxr6kq48O11rVg90H21EOLACvtfcHk0ljenXDR8sWkoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlSgtbnE&md5=08ae55b9565885165e76d98e2db1befa</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1111%2Fbph.12248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.12248%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DKuhne%26aufirst%3DS.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DA%2520structural%2520chemogenomics%2520analysis%2520of%2520aminergic%2520GPCRs%253A%2520lessons%2520for%2520histamine%2520receptor%2520ligand%2520design.%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2013%26volume%3D170%26spage%3D101%26epage%3D126%26doi%3D10.1111%2Fbph.12248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Heilker, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tautermann, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bieler, M.</span><span> </span><span class="NLM_article-title">G-protein-coupled receptor-focused drug discovery using a target class platform approach</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">240</span><span class="refDoi"> DOI: 10.1016/j.drudis.2008.11.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2Fj.drudis.2008.11.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=19121411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjt1Shsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=231-240&author=R.+Heilkerauthor=M.+Wolffauthor=C.+S.+Tautermannauthor=M.+Bieler&title=G-protein-coupled+receptor-focused+drug+discovery+using+a+target+class+platform+approach&doi=10.1016%2Fj.drudis.2008.11.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">G-protein-coupled receptor-focused drug discovery using a target class platform approach</span></div><div class="casAuthors">Heilker, Ralf; Wolff, Michael; Tautermann, Christofer S.; Bieler, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5/6</span>),
    <span class="NLM_cas:pages">231-240</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In recent years, several large pharmaceutical companies have taken a novel approach to drug discovery biol. and chem. in that they channel their efforts with respect to particular target classes, such as G-protein-coupled receptors (GPCRs), toward dedicated, specialized teams.  Benefits of such an organizational structure are the prospects of establishing several target-family-specific exptl. techniques and skill sets, thereby enabling a comprehensive functional profiling of drug candidates in different pharmacol. respects.  In this context, the recently increased no. of reports on GPCR ligand-biased signaling has further spurred the efforts in the pharmaceutical industry toward broader biol. characterization of the test compds., for example employing high-content screening to analyze different GPCR ligand-induced signaling pathways.  The knowledge of the disease-relevant functional properties of the small mol. GPCR ligands enables target-specific chem. optimization and GPCR-subclass-directed library design.  In the case of GPCRs, where little - although at present slowly expanding - structural information on the targets is available, the modeling of GPCR structures crucially depends on biol. validation (typically supported by site-directed mutagenesis of the GPCR ligand binding site).  In this review, we aim to recapitulate efforts in the pharmaceutical industry to address GPCR-directed drug discovery in a target-class-directed platform approach: establishing GPCR-specific biol. assay panels and creating computational chem. methods for finding and optimizing small mols. modulating the activity of GPCRs.  GPCR-target-class-directed platform approach: establishment of GPCR-specific biol. assay panels and creation of computational chem. methods for finding and optimizing small mols. modulating the activity of GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOQSxaIoD2G7Vg90H21EOLACvtfcHk0lhqEKIA0KYEjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjt1Shsbk%253D&md5=3943a8e246fef8aac1191e4d6c2b3788</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2008.11.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2008.11.011%26sid%3Dliteratum%253Aachs%26aulast%3DHeilker%26aufirst%3DR.%26aulast%3DWolff%26aufirst%3DM.%26aulast%3DTautermann%26aufirst%3DC.%2BS.%26aulast%3DBieler%26aufirst%3DM.%26atitle%3DG-protein-coupled%2520receptor-focused%2520drug%2520discovery%2520using%2520a%2520target%2520class%2520platform%2520approach%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D231%26epage%3D240%26doi%3D10.1016%2Fj.drudis.2008.11.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Wichard, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ter Laak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krause, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinrich, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kühne, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleinau, G.</span><span> </span><span class="NLM_article-title">Chemogenomic analysis of G-protein coupled receptors and their ligands deciphers locks and keys governing diverse aspects of signalling.</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e16811</span><span class="refDoi"> DOI: 10.1371/journal.pone.0016811</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1371%2Fjournal.pone.0016811" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e16811&author=J.+D.+Wichardauthor=A.+Ter+Laakauthor=G.+Krauseauthor=N.+Heinrichauthor=R.+K%C3%BChneauthor=G.+Kleinau&title=Chemogenomic+analysis+of+G-protein+coupled+receptors+and+their+ligands+deciphers+locks+and+keys+governing+diverse+aspects+of+signalling.&doi=10.1371%2Fjournal.pone.0016811"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0016811&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0016811%26sid%3Dliteratum%253Aachs%26aulast%3DWichard%26aufirst%3DJ.%2BD.%26aulast%3DTer%2BLaak%26aufirst%3DA.%26aulast%3DKrause%26aufirst%3DG.%26aulast%3DHeinrich%26aufirst%3DN.%26aulast%3DK%25C3%25BChne%26aufirst%3DR.%26aulast%3DKleinau%26aufirst%3DG.%26atitle%3DChemogenomic%2520analysis%2520of%2520G-protein%2520coupled%2520receptors%2520and%2520their%2520ligands%2520deciphers%2520locks%2520and%2520keys%2520governing%2520diverse%2520aspects%2520of%2520signalling.%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De16811%26doi%3D10.1371%2Fjournal.pone.0016811" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Van der Horst, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peironcely, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beukers, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Vlijmen, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emmerich, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okuno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span> </span><span class="NLM_article-title">A novel chemogenomics analysis of G protein-coupled receptors (GPCRs) and their ligands: a potential strategy for receptor de-orphanization</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">316</span><span class="refDoi"> DOI: 10.1186/1471-2105-11-316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1186%2F1471-2105-11-316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=20537162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A280%3ADC%252BC3cnltVSgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=316&author=E.+Van+der+Horstauthor=J.+E.+Peironcelyauthor=A.+P.+Ijzermanauthor=M.+W.+Beukersauthor=J.+R.+Laneauthor=H.+W.+van+Vlijmenauthor=M.+T.+Emmerichauthor=Y.+Okunoauthor=A.+Bender&title=A+novel+chemogenomics+analysis+of+G+protein-coupled+receptors+%28GPCRs%29+and+their+ligands%3A+a+potential+strategy+for+receptor+de-orphanization&doi=10.1186%2F1471-2105-11-316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A novel chemogenomics analysis of G protein-coupled receptors (GPCRs) and their ligands: a potential strategy for receptor de-orphanization</span></div><div class="casAuthors">van der Horst Eelke; Peironcely Julio E; Ijzerman Adriaan P; Beukers Margot W; Lane Jonathan R; van Vlijmen Herman W T; Emmerich Michael T M; Okuno Yasushi; Bender Andreas</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">316</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  G protein-coupled receptors (GPCRs) represent a family of well-characterized drug targets with significant therapeutic value.  Phylogenetic classifications may help to understand the characteristics of individual GPCRs and their subtypes.  Previous phylogenetic classifications were all based on the sequences of receptors, adding only minor information about the ligand binding properties of the receptors.  In this work, we compare a sequence-based classification of receptors to a ligand-based classification of the same group of receptors, and evaluate the potential to use sequence relatedness as a predictor for ligand interactions thus aiding the quest for ligands of orphan receptors.  RESULTS:  We present a classification of GPCRs that is purely based on their ligands, complementing sequence-based phylogenetic classifications of these receptors.  Targets were hierarchically classified into phylogenetic trees, for both sequence space and ligand (substructure) space.  The overall organization of the sequence-based tree and substructure-based tree was similar; in particular, the adenosine receptors cluster together as well as most peptide receptor subtypes (e.g. opioid, somatostatin) and adrenoceptor subtypes.  In ligand space, the prostanoid and cannabinoid receptors are more distant from the other targets, whereas the tachykinin receptors, the oxytocin receptor, and serotonin receptors are closer to the other targets, which is indicative for ligand promiscuity.  In 93% of the receptors studied, de-orphanization of a simulated orphan receptor using the ligands of related receptors performed better than random (AUC > 0.5) and for 35% of receptors de-orphanization performance was good (AUC > 0.7).  CONCLUSIONS:  We constructed a phylogenetic classification of GPCRs that is solely based on the ligands of these receptors.  The similarities and differences with traditional sequence-based classifications were investigated: our ligand-based classification uncovers relationships among GPCRs that are not apparent from the sequence-based classification.  This will shed light on potential cross-reactivity of GPCR ligands and will aid the design of new ligands with the desired activity profiles.  In addition, we linked the ligand-based classification with a ligand-focused sequence-based classification described in literature and proved the potential of this method for de-orphanization of GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTvnKpagfv6Gkg6wdEZmT2rfW6udTcc2eaa5ICENL0-abntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cnltVSgsA%253D%253D&md5=28e6a470f88c58f2f94ae170bd45f243</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-11-316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-11-316%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2Bder%2BHorst%26aufirst%3DE.%26aulast%3DPeironcely%26aufirst%3DJ.%2BE.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DBeukers%26aufirst%3DM.%2BW.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%2BW.%26aulast%3DEmmerich%26aufirst%3DM.%2BT.%26aulast%3DOkuno%26aufirst%3DY.%26aulast%3DBender%26aufirst%3DA.%26atitle%3DA%2520novel%2520chemogenomics%2520analysis%2520of%2520G%2520protein-coupled%2520receptors%2520%2528GPCRs%2529%2520and%2520their%2520ligands%253A%2520a%2520potential%2520strategy%2520for%2520receptor%2520de-orphanization%26jtitle%3DBMC%2520Bioinf.%26date%3D2010%26volume%3D11%26spage%3D316%26doi%3D10.1186%2F1471-2105-11-316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Lin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sassano, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">A pharmacological organization of G protein-coupled receptors</span> <span class="citation_source-journal">Nat. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">140</span><span class="NLM_x">–</span> <span class="NLM_lpage">146</span><span class="refDoi"> DOI: 10.1038/nmeth.2324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnmeth.2324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=23291723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjslelsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=140-146&author=H.+Linauthor=M.+F.+Sassanoauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=A+pharmacological+organization+of+G+protein-coupled+receptors&doi=10.1038%2Fnmeth.2324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A pharmacological organization of G protein-coupled receptors</span></div><div class="casAuthors">Lin, Henry; Sassano, Maria F.; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">140-146</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein classification typically uses structural, sequence or functional similarity.  Here we introduce an orthogonal method that organizes proteins by ligand similarity, focusing on the class A G-protein-coupled receptor (GPCR) protein family.  Comparing a ligand-based dendrogram to a sequence-based one, we identified GPCRs that were distantly linked by sequence but were neighbors by ligand similarity.  Exptl. testing of the ligands predicted to link three of these new pairs confirmed the predicted assocn., with potencies ranging from low nanomolar to low micromolar.  We also predicted hundreds of non-GPCRs closely related to GPCRs by ligand similarity and confirmed several cases exptl.  Ligand similarities among these targets may reflect the conservation of identical ligands among unrelated receptors, which signal in different time domains.  Our method integrates these apparently disparate receptors into chem. coherent circuits and suggests which of these receptors may be targeted by individual ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqx4gVKRHxC8bVg90H21EOLACvtfcHk0li6XrmCQLw8XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjslelsQ%253D%253D&md5=eaec5942489a3337effbbc419a69f17e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.2324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.2324%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520pharmacological%2520organization%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DNat.%2520Methods%26date%3D2013%26volume%3D10%26spage%3D140%26epage%3D146%26doi%3D10.1038%2Fnmeth.2324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Mestres, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregori-Puigjané, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valverde, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solé, R. V.</span><span> </span><span class="NLM_article-title">Data completeness-the Achilles heel of drug-target networks</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">983</span><span class="NLM_x">–</span> <span class="NLM_lpage">984</span><span class="refDoi"> DOI: 10.1038/nbt0908-983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnbt0908-983" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=983-984&author=J.+Mestresauthor=E.+Gregori-Puigjan%C3%A9author=S.+Valverdeauthor=R.+V.+Sol%C3%A9&title=Data+completeness-the+Achilles+heel+of+drug-target+networks&doi=10.1038%2Fnbt0908-983"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnbt0908-983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt0908-983%26sid%3Dliteratum%253Aachs%26aulast%3DMestres%26aufirst%3DJ.%26aulast%3DGregori-Puigjan%25C3%25A9%26aufirst%3DE.%26aulast%3DValverde%26aufirst%3DS.%26aulast%3DSol%25C3%25A9%26aufirst%3DR.%2BV.%26atitle%3DData%2520completeness-the%2520Achilles%2520heel%2520of%2520drug-target%2520networks%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D983%26epage%3D984%26doi%3D10.1038%2Fnbt0908-983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Langelaan, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngweniform, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rainey, J. K.</span><span> </span><span class="NLM_article-title">Biophysical characterization of G-protein coupled receptor-peptide ligand binding</span> <span class="citation_source-journal">Biochem. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">89</span><span class="NLM_x">, </span> <span class="NLM_fpage">98</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1139/O10-142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1139%2FO10-142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=21455262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvFeksrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2011&pages=98-105&author=D.+N.+Langelaanauthor=P.+Ngweniformauthor=J.+K.+Rainey&title=Biophysical+characterization+of+G-protein+coupled+receptor-peptide+ligand+binding&doi=10.1139%2FO10-142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Biophysical characterization of G-protein coupled receptor-peptide ligand binding</span></div><div class="casAuthors">Langelaan, David N.; Ngweniform, Pascaline; Rainey, Jan K.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry and Cell Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">98-105</span>CODEN:
                <span class="NLM_cas:coden">BCBIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0829-8211</span>.
    
            (<span class="NLM_cas:orgname">NRC Research Press</span>)
        </div><div class="casAbstract">A review.  G-protein coupled receptors (GPCRs) are ubiquitous membrane proteins allowing intracellular responses to extracellular factors that range from photons of light to small mols. to proteins.  Despite extensive exploitation of GPCRs as therapeutic targets, biophys. characterization of GPCR-ligand interactions remains challenging.  In this minireview, the authors focus on techniques that have been successfully used for structural and biophys. characterization of peptide ligands binding to their cognate GPCRs.  The techniques reviewed include soln.-state NMR spectroscopy, solid-state NMR, x-ray diffraction, fluorescence spectroscopy and single-mol. fluorescence methods, flow cytometry, surface plasmon resonance, isothermal titrn. calorimetry, and at. force microscopy.  The goal herein is to provide a cohesive starting point to allow selection of techniques appropriate to the elucidation of a given GPCR-peptide interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOuaZINHw3v7Vg90H21EOLACvtfcHk0li6XrmCQLw8XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjvFeksrc%253D&md5=bab689a93ab74217c5ccdcb8bc143707</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1139%2FO10-142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1139%252FO10-142%26sid%3Dliteratum%253Aachs%26aulast%3DLangelaan%26aufirst%3DD.%2BN.%26aulast%3DNgweniform%26aufirst%3DP.%26aulast%3DRainey%26aufirst%3DJ.%2BK.%26atitle%3DBiophysical%2520characterization%2520of%2520G-protein%2520coupled%2520receptor-peptide%2520ligand%2520binding%26jtitle%3DBiochem.%2520Cell%2520Biol.%26date%3D2011%26volume%3D89%26spage%3D98%26epage%3D105%26doi%3D10.1139%2FO10-142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Lohse, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuber, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffmann, C.</span><span> </span><span class="NLM_article-title">Fluorescence/bioluminescence resonance energy transfer techniques to study G-protein-coupled receptor activation and signaling</span> <span class="citation_source-journal">Pharmacol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">299</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1124/pr.110.004309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1124%2Fpr.110.004309" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=22407612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC38XmslKktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2012&pages=299-336&author=M.+J.+Lohseauthor=S.+Nuberauthor=C.+Hoffmann&title=Fluorescence%2Fbioluminescence+resonance+energy+transfer+techniques+to+study+G-protein-coupled+receptor+activation+and+signaling&doi=10.1124%2Fpr.110.004309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence/bioluminescence resonance energy transfer techniques to study G-protein-coupled receptor activation and signaling</span></div><div class="casAuthors">Lohse, Martin J.; Nuber, Susanne; Hoffmann, Carsten</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">299-336</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Fluorescence and bioluminescence resonance energy transfer (FRET and BRETT) techniques allow the sensitive monitoring of distances between two labels at the nanometer scale.  Depending on the placement of the labels, this permits the anal. of conformational changes within a single protein (for example of a receptor) or the monitoring of protein-protein interactions (for example, between receptors and G-protein subunits).  Over the past decade, numerous such techniques have been developed to monitor the activation and signaling of G-protein-coupled receptors (GPCRs) in both the purified, reconstituted state and in intact cells.  These techniques span the entire spectrum from ligand binding to the receptors down to intracellular second messengers.  They allow the detn. and the visualization of signaling processes with high temporal and spatial resoln.  With these techniques, it has been demonstrated that GPCR signals may show spatial and temporal patterning.  In particular, evidence has been provided for spatial compartmentalization of GPCRs and their signals in intact cells and for distinct physiol. consequences of such spatial patterning.  We review here the FRET and BRET technologies that have been developed for G-protein-coupled receptors and their signaling proteins (G-proteins, effectors) and the concepts that result from such expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzZxuO8ZQ_D7Vg90H21EOLACvtfcHk0lhoa2vemxGeiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmslKktr8%253D&md5=76146f70e0e0776aa1d10dae884462aa</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1124%2Fpr.110.004309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.110.004309%26sid%3Dliteratum%253Aachs%26aulast%3DLohse%26aufirst%3DM.%2BJ.%26aulast%3DNuber%26aufirst%3DS.%26aulast%3DHoffmann%26aufirst%3DC.%26atitle%3DFluorescence%252Fbioluminescence%2520resonance%2520energy%2520transfer%2520techniques%2520to%2520study%2520G-protein-coupled%2520receptor%2520activation%2520and%2520signaling%26jtitle%3DPharmacol.%2520Rev.%26date%3D2012%26volume%3D64%26spage%3D299%26epage%3D336%26doi%3D10.1124%2Fpr.110.004309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Margathe, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iturrioz, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvear-Perez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsol, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riché, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chabane, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tounsi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhry, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heissler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hibert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Llorens-Cortes, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, D.</span><span> </span><span class="NLM_article-title">Structure -activity relationship studies toward the discovery of selective apelin receptor agonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">2908</span><span class="NLM_x">–</span> <span class="NLM_lpage">2919</span><span class="refDoi"> DOI: 10.1021/jm401789v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401789v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=2908-2919&author=J.+F.+Margatheauthor=X.+Iturriozauthor=R.+Alvear-Perezauthor=C.+Marsolauthor=S.+Rich%C3%A9author=H.+Chabaneauthor=N.+Tounsiauthor=M.+Kuhryauthor=D.+Heisslerauthor=M.+Hibertauthor=C.+Llorens-Cortesauthor=D.+Bonnet&title=Structure+-activity+relationship+studies+toward+the+discovery+of+selective+apelin+receptor+agonists&doi=10.1021%2Fjm401789v"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm401789v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401789v%26sid%3Dliteratum%253Aachs%26aulast%3DMargathe%26aufirst%3DJ.%2BF.%26aulast%3DIturrioz%26aufirst%3DX.%26aulast%3DAlvear-Perez%26aufirst%3DR.%26aulast%3DMarsol%26aufirst%3DC.%26aulast%3DRich%25C3%25A9%26aufirst%3DS.%26aulast%3DChabane%26aufirst%3DH.%26aulast%3DTounsi%26aufirst%3DN.%26aulast%3DKuhry%26aufirst%3DM.%26aulast%3DHeissler%26aufirst%3DD.%26aulast%3DHibert%26aufirst%3DM.%26aulast%3DLlorens-Cortes%26aufirst%3DC.%26aulast%3DBonnet%26aufirst%3DD.%26atitle%3DStructure%2520-activity%2520relationship%2520studies%2520toward%2520the%2520discovery%2520of%2520selective%2520apelin%2520receptor%2520agonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D2908%26epage%3D2919%26doi%3D10.1021%2Fjm401789v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Loison, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cottet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orcel, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adihou, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahmeh, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamarque, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hibert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durroux, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouillac, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, D.</span><span> </span><span class="NLM_article-title">Selective fluorescent nonpeptidic antagonists for vasopressin V1A GPCR: application to ligand screening and oligomerization assays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">8588</span><span class="NLM_x">–</span> <span class="NLM_lpage">8602</span><span class="refDoi"> DOI: 10.1021/jm3006146</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3006146" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=8588-8602&author=S.+Loisonauthor=M.+Cottetauthor=H.+Orcelauthor=H.+Adihouauthor=R.+Rahmehauthor=L.+Lamarqueauthor=E.+Trinquetauthor=E.+Kellenbergerauthor=M.+Hibertauthor=T.+Durrouxauthor=B.+Mouillacauthor=D.+Bonnet&title=Selective+fluorescent+nonpeptidic+antagonists+for+vasopressin+V1A+GPCR%3A+application+to+ligand+screening+and+oligomerization+assays&doi=10.1021%2Fjm3006146"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm3006146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3006146%26sid%3Dliteratum%253Aachs%26aulast%3DLoison%26aufirst%3DS.%26aulast%3DCottet%26aufirst%3DM.%26aulast%3DOrcel%26aufirst%3DH.%26aulast%3DAdihou%26aufirst%3DH.%26aulast%3DRahmeh%26aufirst%3DR.%26aulast%3DLamarque%26aufirst%3DL.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DHibert%26aufirst%3DM.%26aulast%3DDurroux%26aufirst%3DT.%26aulast%3DMouillac%26aufirst%3DB.%26aulast%3DBonnet%26aufirst%3DD.%26atitle%3DSelective%2520fluorescent%2520nonpeptidic%2520antagonists%2520for%2520vasopressin%2520V1A%2520GPCR%253A%2520application%2520to%2520ligand%2520screening%2520and%2520oligomerization%2520assays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D8588%26epage%3D8602%26doi%3D10.1021%2Fjm3006146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Daval, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hibert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galzi, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ilien, B.</span><span> </span><span class="NLM_article-title">Fluorescent derivatives of AC-42 to probe bitopic orthosteric/allosteric binding mechanisms on muscarinic M1 receptors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">2125</span><span class="NLM_x">–</span> <span class="NLM_lpage">2143</span><span class="refDoi"> DOI: 10.1021/jm201348t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201348t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC38XitF2nu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=2125-2143&author=S.+B.+Davalauthor=C.+Valantauthor=D.+Bonnetauthor=E.+Kellenbergerauthor=M.+Hibertauthor=J.+L.+Galziauthor=B.+Ilien&title=Fluorescent+derivatives+of+AC-42+to+probe+bitopic+orthosteric%2Fallosteric+binding+mechanisms+on+muscarinic+M1+receptors&doi=10.1021%2Fjm201348t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescent Derivatives of AC-42 To Probe Bitopic Orthosteric/Allosteric Binding Mechanisms on Muscarinic M1 Receptors</span></div><div class="casAuthors">Daval, Sandrine B.; Valant, Celine; Bonnet, Dominique; Kellenberger, Esther; Hibert, Marcel; Galzi, Jean-Luc; Ilien, Brigitte</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2125-2143</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two fluorescent derivs. of the M1 muscarinic selective agonist AC-42 were synthesized by coupling the lissamine rhodamine B fluorophore (in ortho and para positions) to AC42-NH2.  This precursor, prepd. according to an original seven-step procedure, was included in the study together with the LRB fluorophore (alone or linked to an alkyl chain).  All these compds. are antagonists, but examn. of their ability to inhibit or modulate orthosteric [3H]NMS binding revealed that para-LRB-AC42 shared several properties with AC-42.  Carefully designed expts. allowed para-LRB-AC42 to be used as a FRET tracer on EGFP-fused M1 receptors.  Under equil. binding conditions, orthosteric ligands, AC-42, and the allosteric modulator gallamine behaved as competitors of para-LRB-AC42 binding whereas other allosteric compds. such as WIN 51,708 and N-desmethylclozapine were noncompetitive inhibitors.  Finally, mol. modeling studies focused on putative orthosteric/allosteric bitopic poses for AC-42 and para-LRB-AC42 in a 3D model of the human M1 receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOQzkjHcfvTLVg90H21EOLACvtfcHk0lhoa2vemxGeiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitF2nu7c%253D&md5=7883a194d0cc72fde6ad1d867655f8e0</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm201348t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201348t%26sid%3Dliteratum%253Aachs%26aulast%3DDaval%26aufirst%3DS.%2BB.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DBonnet%26aufirst%3DD.%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DHibert%26aufirst%3DM.%26aulast%3DGalzi%26aufirst%3DJ.%2BL.%26aulast%3DIlien%26aufirst%3DB.%26atitle%3DFluorescent%2520derivatives%2520of%2520AC-42%2520to%2520probe%2520bitopic%2520orthosteric%252Fallosteric%2520binding%2520mechanisms%2520on%2520muscarinic%2520M1%2520receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D2125%26epage%3D2143%26doi%3D10.1021%2Fjm201348t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Iturrioz, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvear-Perez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Mota, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franchet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guillier, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leroux, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dabire, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Jouan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chabane, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerbier, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdeaux, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maigret, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galzi, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hibert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Llorens-Cortes, C.</span><span> </span><span class="NLM_article-title">Identification and pharmacological properties of E339–3D6, the first nonpeptidic apelin receptor agonist</span> <span class="citation_source-journal">FASEB J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">1506</span><span class="NLM_x">–</span> <span class="NLM_lpage">1517</span><span class="refDoi"> DOI: 10.1096/fj.09-140715</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1096%2Ffj.09-140715" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=1506-1517&author=X.+Iturriozauthor=R.+Alvear-Perezauthor=N.+De+Motaauthor=C.+Franchetauthor=F.+Guillierauthor=V.+Lerouxauthor=H.+Dabireauthor=M.+Le+Jouanauthor=H.+Chabaneauthor=R.+Gerbierauthor=D.+Bonnetauthor=A.+Berdeauxauthor=B.+Maigretauthor=J.+L.+Galziauthor=M.+Hibertauthor=C.+Llorens-Cortes&title=Identification+and+pharmacological+properties+of+E339%E2%80%933D6%2C+the+first+nonpeptidic+apelin+receptor+agonist&doi=10.1096%2Ffj.09-140715"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1096%2Ffj.09-140715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.09-140715%26sid%3Dliteratum%253Aachs%26aulast%3DIturrioz%26aufirst%3DX.%26aulast%3DAlvear-Perez%26aufirst%3DR.%26aulast%3DDe%2BMota%26aufirst%3DN.%26aulast%3DFranchet%26aufirst%3DC.%26aulast%3DGuillier%26aufirst%3DF.%26aulast%3DLeroux%26aufirst%3DV.%26aulast%3DDabire%26aufirst%3DH.%26aulast%3DLe%2BJouan%26aufirst%3DM.%26aulast%3DChabane%26aufirst%3DH.%26aulast%3DGerbier%26aufirst%3DR.%26aulast%3DBonnet%26aufirst%3DD.%26aulast%3DBerdeaux%26aufirst%3DA.%26aulast%3DMaigret%26aufirst%3DB.%26aulast%3DGalzi%26aufirst%3DJ.%2BL.%26aulast%3DHibert%26aufirst%3DM.%26aulast%3DLlorens-Cortes%26aufirst%3DC.%26atitle%3DIdentification%2520and%2520pharmacological%2520properties%2520of%2520E339%25E2%2580%25933D6%252C%2520the%2520first%2520nonpeptidic%2520apelin%2520receptor%2520agonist%26jtitle%3DFASEB%2520J.%26date%3D2010%26volume%3D24%26spage%3D1506%26epage%3D1517%26doi%3D10.1096%2Ffj.09-140715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Vernall, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellam, B.</span><span> </span><span class="NLM_article-title">The evolving small-molecule fluorescent-conjugate toolbox for Class A GPCRs</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">171</span><span class="NLM_x">, </span> <span class="NLM_fpage">1073</span><span class="NLM_x">–</span> <span class="NLM_lpage">1084</span><span class="refDoi"> DOI: 10.1111/bph.12265</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1111%2Fbph.12265" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=23734587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1Sht7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2014&pages=1073-1084&author=A.+J.+Vernallauthor=S.+J.+Hillauthor=B.+Kellam&title=The+evolving+small-molecule+fluorescent-conjugate+toolbox+for+Class+A+GPCRs&doi=10.1111%2Fbph.12265"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The evolving small-molecule fluorescent-conjugate toolbox for Class A GPCRs</span></div><div class="casAuthors">Vernall, Andrea J.; Hill, Stephen J.; Kellam, Barrie</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1073-1084</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The past decade has witnessed fluorescently tagged drug mols. gaining significant attraction in their use as pharmacol. tools with which to visualize and interrogate receptor targets at the single-cell level.  Addnl., one can generate detailed pharmacol. information, such as affinity measurements, down to almost single-mol. detection limits.  The now accepted utilization of fluorescence-based readouts in high-throughput/high-content screening provides further evidence that fluorescent mols. offer a safer and more adaptable substitute to radioligands in mol. pharmacol. and drug discovery.  One such drug-target family that has received considerable attention are the GPCRs; this review therefore summarizes the most recent developments in the area of fluorescent ligand design for this important drug target.  We assess recently reported fluorescent conjugates by adopting a receptor-family-based approach, highlighting some of the strengths and weaknesses of the individual mols. and their subsequent use.  This review adds further strength to the arguments that fluorescent ligand design and synthesis requires careful planning and execution; providing examples illustrating that selection of the correct fluorescent dye, linker length/compn. and geog. attachment point to the drug scaffold can all influence the ultimate selectivity and potency of the final conjugate when compared with its unlabeled precursor.  When optimized appropriately, the resultant fluorescent conjugates have been successfully employed in an array of assay formats, including flow cytometry, fluorescence microscopy, FRET and scanning confocal microscopy.  It is clear that fluorescently labeled GPCR ligands remain a developing and dynamic research arena.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIxr-W-h3chLVg90H21EOLACvtfcHk0lhvCam2Efby0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1Sht7w%253D&md5=5e8768a623f257368a5d9d0278192987</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1111%2Fbph.12265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.12265%26sid%3Dliteratum%253Aachs%26aulast%3DVernall%26aufirst%3DA.%2BJ.%26aulast%3DHill%26aufirst%3DS.%2BJ.%26aulast%3DKellam%26aufirst%3DB.%26atitle%3DThe%2520evolving%2520small-molecule%2520fluorescent-conjugate%2520toolbox%2520for%2520Class%2520A%2520GPCRs%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2014%26volume%3D171%26spage%3D1073%26epage%3D1084%26doi%3D10.1111%2Fbph.12265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Ward, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milligan, G.</span><span> </span><span class="NLM_article-title">Structural and biophysical characterisation of G protein-coupled receptor ligand binding using resonance energy transfer and fluorescent labelling techniques</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Biomembr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1838</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">14</span><span class="refDoi"> DOI: 10.1016/j.bbamem.2013.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2Fj.bbamem.2013.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=23590995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmsVGlt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1838&publication_year=2014&pages=3-14&author=R.+J.+Wardauthor=G.+Milligan&title=Structural+and+biophysical+characterisation+of+G+protein-coupled+receptor+ligand+binding+using+resonance+energy+transfer+and+fluorescent+labelling+techniques&doi=10.1016%2Fj.bbamem.2013.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and biophysical characterisation of G protein-coupled receptor ligand binding using resonance energy transfer and fluorescent labeling techniques</span></div><div class="casAuthors">Ward, Richard J.; Milligan, Graeme</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1838</span>
        (<span class="NLM_cas:issue">1PA</span>),
    <span class="NLM_cas:pages">3-14</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The interaction between ligands and the G protein-coupled receptors (GPCRs) to which they bind has long been the focus of intensive investigation.  The signaling cascades triggered by receptor activation, due in most cases to ligand binding, are of great physiol. and medical importance; indeed, GPCRs are targeted by in excess of 30% of small mol. therapeutic medicines.  Attempts to identify further pharmacol. useful GPCR ligands, for receptors with known and unknown endogenous ligands, continue apace.  In earlier days direct assessment of such interactions was restricted largely to the use of ligands incorporating radioactive isotope labels as this allowed detection of the ligand and monitoring its interaction with the GPCR.  This use of such markers has continued with the development of ligands labeled with fluorophores and their application to the study of receptor-ligand interactions using both light microscopy and resonance energy transfer techniques, including homogenous time-resolved fluorescence resonance energy transfer.  Details of ligand-receptor interactions via x-ray crystallog. are advancing rapidly as methods suitable for routine prodn. of substantial amts. and stabilized forms of GPCRs have been developed and there is hope that this may become as routine as the co-crystn. of serine/threonine kinases with ligands, an approach that has facilitated widespread use of rapid structure-based ligand design.  Conformational changes involved in the activation of GPCRs, widely predicted by biochem. and biophys. means, have inspired the development of intramol. FRET-based sensor forms of GPCRs designed to investigate the events following ligand binding and resulting in a signal propagation across the cell membrane.  Finally, a no. of techniques are emerging in which ligand-GPCR binding can be studied in ways that, while indirect, are able to monitor its results in an unbiased and integrated manner.  This article is part of a Special Issue entitled: Structural and biophys. characterization of membrane protein-ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4R2XZxXKK2LVg90H21EOLACvtfcHk0lhvCam2Efby0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmsVGlt7g%253D&md5=b4a4f540ccc85b50f2322536a24a505c</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2013.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2013.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BJ.%26aulast%3DMilligan%26aufirst%3DG.%26atitle%3DStructural%2520and%2520biophysical%2520characterisation%2520of%2520G%2520protein-coupled%2520receptor%2520ligand%2520binding%2520using%2520resonance%2520energy%2520transfer%2520and%2520fluorescent%2520labelling%2520techniques%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Biomembr.%26date%3D2014%26volume%3D1838%26spage%3D3%26epage%3D14%26doi%3D10.1016%2Fj.bbamem.2013.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Bazin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathis, G.</span><span> </span><span class="NLM_article-title">Time resolved amplification of cryptate emission: a versatile technology to trace biomolecular interactions</span> <span class="citation_source-journal">Rev. Mol. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">233</span><span class="NLM_x">–</span> <span class="NLM_lpage">250</span><span class="refDoi"> DOI: 10.1016/S1389-0352(01)00040-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2FS1389-0352%2801%2900040-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD38XhsVSmt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2002&pages=233-250&author=H.+Bazinauthor=E.+Trinquetauthor=G.+Mathis&title=Time+resolved+amplification+of+cryptate+emission%3A+a+versatile+technology+to+trace+biomolecular+interactions&doi=10.1016%2FS1389-0352%2801%2900040-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Time resolved amplification of cryptate emission: a versatile technology to trace biomolecular interactions</span></div><div class="casAuthors">Bazin, H.; Trinquet, E.; Mathis, G.</div><div class="citationInfo"><span class="NLM_cas:title">Reviews in Molecular Biotechnology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">233-250</span>CODEN:
                <span class="NLM_cas:coden">RMBIFZ</span>;
        ISSN:<span class="NLM_cas:issn">1389-0352</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Fluorescence resonance energy transfer (FRET) in assocn. with a time-resolved fluorescence mode of detection was used to design a new homogeneous technol. suitable to monitor biomol. interactions.  A lanthanide cryptate characterized by a long lived fluorescence emission was used as donor and a cross-linked allophycocyanine was used as acceptor.  This new donor/acceptor pair displayed an exceptionally large Forster radius of 9 nm.  This allowed to build up a set of labeling strategies to probe the interactions between biomols. with an emphasis on fully indirect cassette formats particularly suitable for high throughput screening applications.  Herein we describe the basics of the technol., review the latest applications to the study of mol. interactions involved in cells and new oligonucleotides based assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-QBHgf98sd7Vg90H21EOLACvtfcHk0lgLhJ2SvKaJDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhsVSmt7s%253D&md5=a8172b56da7349c622662bcb618cc86a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS1389-0352%2801%2900040-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1389-0352%252801%252900040-X%26sid%3Dliteratum%253Aachs%26aulast%3DBazin%26aufirst%3DH.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DMathis%26aufirst%3DG.%26atitle%3DTime%2520resolved%2520amplification%2520of%2520cryptate%2520emission%253A%2520a%2520versatile%2520technology%2520to%2520trace%2520biomolecular%2520interactions%26jtitle%3DRev.%2520Mol.%2520Biotechnol.%26date%3D2002%26volume%3D82%26spage%3D233%26epage%3D250%26doi%3D10.1016%2FS1389-0352%2801%2900040-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Zwier, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cottet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durroux, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douzon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bdioui, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregor, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourrier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oueslati, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicolas, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tinel, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boisseau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yverneau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charrier-Savournin, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fink, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span> </span><span class="NLM_article-title">A fluorescent ligand-binding alternative using Tag-lite® technology</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1248</span><span class="NLM_x">–</span> <span class="NLM_lpage">1259</span><span class="refDoi"> DOI: 10.1177/1087057110384611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1177%2F1087057110384611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=20974902" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVWgtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=1248-1259&author=J.+M.+Zwierauthor=T.+Rouxauthor=M.+Cottetauthor=T.+Durrouxauthor=S.+Douzonauthor=S.+Bdiouiauthor=N.+Gregorauthor=E.+Bourrierauthor=N.+Oueslatiauthor=L.+Nicolasauthor=N.+Tinelauthor=C.+Boisseauauthor=P.+Yverneauauthor=F.+Charrier-Savourninauthor=M.+Finkauthor=E.+Trinquet&title=A+fluorescent+ligand-binding+alternative+using+Tag-lite%C2%AE+technology&doi=10.1177%2F1087057110384611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorescent ligand-binding alternative using Tag-lite technology</span></div><div class="casAuthors">Zwier, Jurriaan M.; Roux, Thomas; Cottet, Martin; Durroux, Thierry; Douzon, Stephanie; Bdioui, Sara; Gregor, Nathalie; Bourrier, Emmanuel; Oueslati, Nadia; Nicolas, Ludovic; Tinel, Norbert; Boisseau, Chloe; Yverneau, Paul; Charrier-Savournin, Fabienne; Fink, Michel; Trinquet, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1248-1259</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are crucial cell surface receptors that transmit signals from a wide range of extracellular ligands.  Indeed, 40% to 50% of all marketed drugs are thought to modulate GPCR activity, making them the major class of targets in the drug discovery process.  Binding assays are widely used to identify high-affinity, selective, and potent GPCR drugs.  In this field, the use of radiolabeled ligands has remained so far the gold-std. method.  Here the authors report a less hazardous alternative for high-throughput screening (HTS) applications by the setup of a nonradioactive fluorescence-based technol. named Tag-lite.  Selective binding of various fluorescent ligands, either peptidic or not, covering a large panel of GPCRs from different classes is illustrated, particularly for chemokine (CXCR4), opioid (δ, μ, and κ), and cholecystokinin (CCK1 and CCK2) receptors.  Affinity consts. of well-known pharmacol. agents of numerous GPCRs are in line with values published in the literature.  The authors clearly demonstrate that the Tag-lite binding assay format can be successfully and reproducibly applied by using different cellular materials such as transient or stable recombinant cells lines expressing SNAP-tagged GPCR.  Such fluorescent-based binding assays can be performed with adherent cells or cells in suspension, in 96- or 384-well plates.  Altogether, this new technol. offers great advantages in terms of flexibility, rapidity, and user-friendliness; allows easy miniaturization; and makes it completely suitable for HTS applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ4y-qYZVkzbVg90H21EOLACvtfcHk0lgLhJ2SvKaJDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVWgtLY%253D&md5=ebb276667f42394eb16e8f0b64b365da</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1177%2F1087057110384611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057110384611%26sid%3Dliteratum%253Aachs%26aulast%3DZwier%26aufirst%3DJ.%2BM.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DCottet%26aufirst%3DM.%26aulast%3DDurroux%26aufirst%3DT.%26aulast%3DDouzon%26aufirst%3DS.%26aulast%3DBdioui%26aufirst%3DS.%26aulast%3DGregor%26aufirst%3DN.%26aulast%3DBourrier%26aufirst%3DE.%26aulast%3DOueslati%26aufirst%3DN.%26aulast%3DNicolas%26aufirst%3DL.%26aulast%3DTinel%26aufirst%3DN.%26aulast%3DBoisseau%26aufirst%3DC.%26aulast%3DYverneau%26aufirst%3DP.%26aulast%3DCharrier-Savournin%26aufirst%3DF.%26aulast%3DFink%26aufirst%3DM.%26aulast%3DTrinquet%26aufirst%3DE.%26atitle%3DA%2520fluorescent%2520ligand-binding%2520alternative%2520using%2520Tag-lite%25C2%25AE%2520technology%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2010%26volume%3D15%26spage%3D1248%26epage%3D1259%26doi%3D10.1177%2F1087057110384611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Karpenko, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margathe, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pflimlin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dupuis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hibert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durroux, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, D.</span><span> </span><span class="NLM_article-title">Selective nonpeptidic fluorescent ligands for oxytocin receptor: design, synthesis, and application to time-resolved FRET binding assay</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">2547</span><span class="NLM_x">–</span> <span class="NLM_lpage">2552</span><span class="refDoi"> DOI: 10.1021/jm501395b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501395b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvV2ju74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2547-2552&author=I.+A.+Karpenkoauthor=J.+F.+Margatheauthor=T.+Rodriguezauthor=E.+Pflimlinauthor=E.+Dupuisauthor=M.+Hibertauthor=T.+Durrouxauthor=D.+Bonnet&title=Selective+nonpeptidic+fluorescent+ligands+for+oxytocin+receptor%3A+design%2C+synthesis%2C+and+application+to+time-resolved+FRET+binding+assay&doi=10.1021%2Fjm501395b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Nonpeptidic Fluorescent Ligands for Oxytocin Receptor: Design, Synthesis, and Application to Time-Resolved FRET Binding Assay</span></div><div class="casAuthors">Karpenko, Iuliia A.; Margathe, Jean-Francois; Rodriguez, Thieric; Pflimlin, Elsa; Dupuis, Elodie; Hibert, Marcel; Durroux, Thierry; Bonnet, Dominique</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2547-2552</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design and the synthesis of the first high-affinity fluorescent ligands for oxytocin receptor (OTR) are described.  These compds. enabled the development of a TR-FRET based assay for OTR, readily amenable to high throughput screening.  The validation of the assay was achieved by competition expts. with both peptide and nonpeptide OTR ligands as competitors.  These probes represent the first selective fluorescent ligands for the oxytocin G protein-coupled receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYix67Nh90z7Vg90H21EOLACvtfcHk0lgLhJ2SvKaJDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvV2ju74%253D&md5=7ba9dddc66ed6bafb60b83cbc2efc9ff</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm501395b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501395b%26sid%3Dliteratum%253Aachs%26aulast%3DKarpenko%26aufirst%3DI.%2BA.%26aulast%3DMargathe%26aufirst%3DJ.%2BF.%26aulast%3DRodriguez%26aufirst%3DT.%26aulast%3DPflimlin%26aufirst%3DE.%26aulast%3DDupuis%26aufirst%3DE.%26aulast%3DHibert%26aufirst%3DM.%26aulast%3DDurroux%26aufirst%3DT.%26aulast%3DBonnet%26aufirst%3DD.%26atitle%3DSelective%2520nonpeptidic%2520fluorescent%2520ligands%2520for%2520oxytocin%2520receptor%253A%2520design%252C%2520synthesis%252C%2520and%2520application%2520to%2520time-resolved%2520FRET%2520binding%2520assay%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2547%26epage%3D2552%26doi%3D10.1021%2Fjm501395b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Hounsou, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margathe, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oueslati, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belhocine, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dupuis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ilien, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hibert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pin, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durroux, T.</span><span> </span><span class="NLM_article-title">Time-resolved FRET binding assay to investigate hetero-oligomer binding properties: proof of concept with dopamine D1/D3 heterodimer</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">466</span><span class="NLM_x">–</span> <span class="NLM_lpage">474</span><span class="refDoi"> DOI: 10.1021/cb5007568</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5007568" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=466-474&author=C.+Hounsouauthor=J.+F.+Margatheauthor=N.+Oueslatiauthor=A.+Belhocineauthor=E.+Dupuisauthor=C.+Thomasauthor=A.+Mannauthor=B.+Ilienauthor=D.+Rognanauthor=E.+Trinquetauthor=M.+Hibertauthor=J.+P.+Pinauthor=D.+Bonnetauthor=T.+Durroux&title=Time-resolved+FRET+binding+assay+to+investigate+hetero-oligomer+binding+properties%3A+proof+of+concept+with+dopamine+D1%2FD3+heterodimer&doi=10.1021%2Fcb5007568"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fcb5007568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5007568%26sid%3Dliteratum%253Aachs%26aulast%3DHounsou%26aufirst%3DC.%26aulast%3DMargathe%26aufirst%3DJ.%2BF.%26aulast%3DOueslati%26aufirst%3DN.%26aulast%3DBelhocine%26aufirst%3DA.%26aulast%3DDupuis%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DC.%26aulast%3DMann%26aufirst%3DA.%26aulast%3DIlien%26aufirst%3DB.%26aulast%3DRognan%26aufirst%3DD.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DHibert%26aufirst%3DM.%26aulast%3DPin%26aufirst%3DJ.%2BP.%26aulast%3DBonnet%26aufirst%3DD.%26aulast%3DDurroux%26aufirst%3DT.%26atitle%3DTime-resolved%2520FRET%2520binding%2520assay%2520to%2520investigate%2520hetero-oligomer%2520binding%2520properties%253A%2520proof%2520of%2520concept%2520with%2520dopamine%2520D1%252FD3%2520heterodimer%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D466%26epage%3D474%26doi%3D10.1021%2Fcb5007568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Cottet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faklaris, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Falco, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pin, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouillac, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durroux, T.</span><span> </span><span class="NLM_article-title">Fluorescent ligands to investigate GPCR binding properties and oligomerization</span> <span class="citation_source-journal">Biochem. Soc. Trans.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">148</span><span class="NLM_x">–</span> <span class="NLM_lpage">153</span><span class="refDoi"> DOI: 10.1042/BST20120237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1042%2FBST20120237" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=23356275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1eitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=148-153&author=M.+Cottetauthor=O.+Faklarisauthor=A.+Falcoauthor=E.+Trinquetauthor=J.+P.+Pinauthor=B.+Mouillacauthor=T.+Durroux&title=Fluorescent+ligands+to+investigate+GPCR+binding+properties+and+oligomerization&doi=10.1042%2FBST20120237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescent ligands to investigate GPCR binding properties and oligomerization</span></div><div class="casAuthors">Cottet, Martin; Faklaris, Orestis; Falco, Amadine; Trinquet, Eric; Pin, Jean-Philippe; Mouillac, Bernard; Durroux, Thierry</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Society Transactions</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">148-153</span>CODEN:
                <span class="NLM_cas:coden">BCSTB5</span>;
        ISSN:<span class="NLM_cas:issn">0300-5127</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  Fluorescent ligands for GPCRs (G-protein-coupled receptors) have been synthesized for a long time but their use was usually restricted to receptor localization in the cell by fluorescent imaging microscopy.  During the last two decades, the emergence of new fluorescence-based strategies and the concomitant development of fluorescent measurement app. have dramatically widened the use of fluorescent ligands.  Among the various strategies, TR (time-resolved)-FRET (fluorescence resonance energy transfer) approaches exhibit an interesting potential to study GPCR interactions with various partners.  We have derived various sets of ligands that target different GPCRs with fluorophores, which are compatible with TR-FRET strategies.  Fluorescent ligands labeled either with a fluorescent donor (such as europium or terbium cryptate) or with a fluorescent acceptor (such as fluorescein, dy647 or Alexa Fluor 647), for example, kept high affinities for their cognate receptors.  These ligands turn out to be interesting tools to develop FRET-based binding assays.  We also used these fluorescent ligands to analyze GPCR oligomerization by measuring FRET between ligands bound to receptor dimers.  In contrast with FRET strategies, on the basis of receptor labeling, the ligand-based approach we developed is fully compatible with the study of wild-type receptors and therefore with receptors expressed in native tissues.  Therefore, by using fluorescent analogs of oxytocin, we demonstrated the existence of oxytocin receptor dimers in the mammary gland of lactating rats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqikaTbyJ7bCbVg90H21EOLACvtfcHk0liM4ClQC2xfPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1eitb4%253D&md5=fadfa1cbc5204e3f5d864f0a9740ca58</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1042%2FBST20120237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBST20120237%26sid%3Dliteratum%253Aachs%26aulast%3DCottet%26aufirst%3DM.%26aulast%3DFaklaris%26aufirst%3DO.%26aulast%3DFalco%26aufirst%3DA.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DPin%26aufirst%3DJ.%2BP.%26aulast%3DMouillac%26aufirst%3DB.%26aulast%3DDurroux%26aufirst%3DT.%26atitle%3DFluorescent%2520ligands%2520to%2520investigate%2520GPCR%2520binding%2520properties%2520and%2520oligomerization%26jtitle%3DBiochem.%2520Soc.%2520Trans.%26date%3D2013%26volume%3D41%26spage%3D148%26epage%3D153%26doi%3D10.1042%2FBST20120237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Emami-Nemini, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leblay, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourrier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamarque, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohse, M. J.</span><span> </span><span class="NLM_article-title">Time-resolved fluorescence ligand binding for G protein-coupled receptors</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1307</span><span class="NLM_x">–</span> <span class="NLM_lpage">1320</span><span class="refDoi"> DOI: 10.1038/nprot.2013.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fnprot.2013.073" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1307-1320&author=A.+Emami-Neminiauthor=T.+Rouxauthor=M.+Leblayauthor=E.+Bourrierauthor=L.+Lamarqueauthor=E.+Trinquetauthor=M.+J.+Lohse&title=Time-resolved+fluorescence+ligand+binding+for+G+protein-coupled+receptors&doi=10.1038%2Fnprot.2013.073"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2013.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2013.073%26sid%3Dliteratum%253Aachs%26aulast%3DEmami-Nemini%26aufirst%3DA.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DLeblay%26aufirst%3DM.%26aulast%3DBourrier%26aufirst%3DE.%26aulast%3DLamarque%26aufirst%3DL.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DLohse%26aufirst%3DM.%2BJ.%26atitle%3DTime-resolved%2520fluorescence%2520ligand%2520binding%2520for%2520G%2520protein-coupled%2520receptors%26jtitle%3DNat.%2520Protoc.%26date%3D2013%26volume%3D8%26spage%3D1307%26epage%3D1320%26doi%3D10.1038%2Fnprot.2013.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Leyris, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdié, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fehrentz, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oueslati, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douzon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourrier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamarque, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagne, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galleyrand, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">M’kadmi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mary, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banères, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marie, J.</span><span> </span><span class="NLM_article-title">Homogeneous time-resolved fluorescence-based assay to screen for ligands targeting the growth hormone secretagogue receptor type 1a</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">408</span><span class="NLM_x">, </span> <span class="NLM_fpage">253</span><span class="NLM_x">–</span> <span class="NLM_lpage">262</span><span class="refDoi"> DOI: 10.1016/j.ab.2010.09.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2Fj.ab.2010.09.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=20937574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVagsLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=408&publication_year=2011&pages=253-262&author=J.+P.+Leyrisauthor=T.+Rouxauthor=E.+Trinquetauthor=P.+Verdi%C3%A9author=J.+A.+Fehrentzauthor=N.+Oueslatiauthor=S.+Douzonauthor=E.+Bourrierauthor=L.+Lamarqueauthor=D.+Gagneauthor=J.+C.+Galleyrandauthor=C.+M%E2%80%99kadmiauthor=J.+Martinezauthor=S.+Maryauthor=J.+L.+Ban%C3%A8resauthor=J.+Marie&title=Homogeneous+time-resolved+fluorescence-based+assay+to+screen+for+ligands+targeting+the+growth+hormone+secretagogue+receptor+type+1a&doi=10.1016%2Fj.ab.2010.09.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Homogeneous time-resolved fluorescence-based assay to screen for ligands targeting the growth hormone secretagogue receptor type 1a</span></div><div class="casAuthors">Leyris, Jean-Philippe; Roux, Thomas; Trinquet, Eric; Verdie, Pascal; Fehrentz, Jean-Alain; Oueslati, Nadia; Douzon, Stephanie; Bourrier, Emmanuel; Lamarque, Laurent; Gagne, Didier; Galleyrand, Jean-Claude; M'kadmi, Celine; Martinez, Jean; Mary, Sophie; Baneres, Jean-Louis; Marie, Jacky</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">408</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">253-262</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The growth hormone secretagogue receptor type 1a (GHS-R1a) belongs to class A G-protein-coupled receptors (GPCR).  This receptor mediates pleiotropic effects of ghrelin and represents a promising target for dysfunctions of growth hormone secretion and energy homeostasis including obesity.  Identification of new compds. which bind GHS-R1a is traditionally achieved using radioactive binding assays.  Here we propose a new fluorescence-based assay, called Tag-lite binding assay, based on a fluorescence resonance energy transfer (FRET) process between a terbium cryptate covalently attached to a SNAP-tag fused GHS-R1a (SNAP-GHS-R1a) and a high-affinity red fluorescent ghrelin ligand.  The long fluorescence lifetime of the terbium cryptate allows a time-resolved detection of the FRET signal.  The assay was made compatible with high-throughput screening by using prelabeled cells in suspension under a 384-well plate format.  Ki values for a panel of 14 compds. displaying agonist, antagonist, or inverse agonist properties were detd. using both the radioactive and the Tag-lite binding assays performed on the same batches of GHS-R1a-expressing cells.  Compd. potencies obtained in the two assays were nicely correlated.  This study is the first description of a sensitive and reliable nonradioactive binding assay for GHS-R1a in a format amenable to high-throughput screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5Df7hm2kmlbVg90H21EOLACvtfcHk0lg8aqgEhy7Qlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVagsLrI&md5=cf1476bd78ebb24d460abd92a70d2a08</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2010.09.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2010.09.030%26sid%3Dliteratum%253Aachs%26aulast%3DLeyris%26aufirst%3DJ.%2BP.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DVerdi%25C3%25A9%26aufirst%3DP.%26aulast%3DFehrentz%26aufirst%3DJ.%2BA.%26aulast%3DOueslati%26aufirst%3DN.%26aulast%3DDouzon%26aufirst%3DS.%26aulast%3DBourrier%26aufirst%3DE.%26aulast%3DLamarque%26aufirst%3DL.%26aulast%3DGagne%26aufirst%3DD.%26aulast%3DGalleyrand%26aufirst%3DJ.%2BC.%26aulast%3DM%25E2%2580%2599kadmi%26aufirst%3DC.%26aulast%3DMartinez%26aufirst%3DJ.%26aulast%3DMary%26aufirst%3DS.%26aulast%3DBan%25C3%25A8res%26aufirst%3DJ.%2BL.%26aulast%3DMarie%26aufirst%3DJ.%26atitle%3DHomogeneous%2520time-resolved%2520fluorescence-based%2520assay%2520to%2520screen%2520for%2520ligands%2520targeting%2520the%2520growth%2520hormone%2520secretagogue%2520receptor%2520type%25201a%26jtitle%3DAnal.%2520Biochem.%26date%3D2011%26volume%3D408%26spage%3D253%26epage%3D262%26doi%3D10.1016%2Fj.ab.2010.09.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Hoffman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flynn, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tillu, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patek, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boitano, S.</span><span> </span><span class="NLM_article-title">Lanthanide labeling of a potent protease activated receptor-2 agonist for time-resolved fluorescence analysis</span> <span class="citation_source-journal">Bioconjugate Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">2098</span><span class="NLM_x">–</span> <span class="NLM_lpage">2104</span><span class="refDoi"> DOI: 10.1021/bc300300q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bc300300q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlOmur7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2012&pages=2098-2104&author=J.+Hoffmanauthor=A.+N.+Flynnauthor=D.+V.+Tilluauthor=Z.+Zhangauthor=R.+Patekauthor=T.+J.+Priceauthor=J.+Vagnerauthor=S.+Boitano&title=Lanthanide+labeling+of+a+potent+protease+activated+receptor-2+agonist+for+time-resolved+fluorescence+analysis&doi=10.1021%2Fbc300300q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Lanthanide Labeling of a Potent Protease Activated Receptor-2 Agonist for Time-Resolved Fluorescence Analysis</span></div><div class="casAuthors">Hoffman, Justin; Flynn, Andrea N.; Tillu, Dipti V.; Zhang, Zhenyu; Patek, Renata; Price, Theodore J.; Vagner, Josef; Boitano, Scott</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2098-2104</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protease activated receptor-2 (PAR2) is one of four G-protein coupled receptors (GPCRs) that can be activated by exogenous or endogenous proteases, which cleave the extracellular amino-terminus to expose a tethered ligand and subsequent G-protein signaling.  Alternatively, PAR2 can be activated by peptide or peptidomimetic ligands derived from the sequence of the natural tethered ligand.  Screening of novel ligands that directly bind to PAR2 to agonize or antagonize the receptor has been hindered by the lack of a sensitive, high-throughput, affinity binding assay.  The authors describe the synthesis and use of a modified PAR2 peptidomimetic agonist, 2-furoyl-LIGRLO-(diethylenetriaminepentaacetic acid)-NH2 (2-f-LIGRLO-dtpa, I), designed for lanthanide-based time-resolved fluorescence screening.  The authors first demonstrate that 2-f-LIGRLO-dtpa is a potent and specific PAR2 agonist across a full spectrum of in vitro assays.  The authors then show that 2-f-LIGRLO-dtpa can be used in an affinity binding assay to evaluate the ligand-receptor interactions between known high potency peptidomimetic agonists (2-furoyl-LIGRLO-NH2, 2-f-LIGRLO; 2-aminothiazol-4-yl-LIGRL-NH2, 2-at-LIGRL; 6-aminonicotinyl-LIGRL-NH2, 6-an-LIGRL) and PAR2.  A sep. N-terminal peptidomimetic modification (3-indoleacetyl-LIGRL-NH2, 3-ia-LIGRL) that does not activate PAR2 signaling was used as a neg. control.  All three peptidomimetic agonists demonstrated sigmoidal competitive binding curves, with the more potent agonists (2-f-LIGRLO and 2-at-LIGRL) displaying increased competition.  In contrast, the control peptide (3-ia-LIGRL) displayed limited competition for PAR2 binding.  In summary, the authors have developed a europium-contg. PAR2 agonist that can be used in a highly sensitive affinity binding assay to screen novel PAR2 ligands in a high-throughput format.  This ligand can serve as a crit. tool in the screening and development of PAR2 ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1zdFIiqcNz7Vg90H21EOLACvtfcHk0lg8aqgEhy7Qlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlOmur7M&md5=7e269f0e06b7687dee47dd69c1aa2240</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fbc300300q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbc300300q%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DJ.%26aulast%3DFlynn%26aufirst%3DA.%2BN.%26aulast%3DTillu%26aufirst%3DD.%2BV.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DPatek%26aufirst%3DR.%26aulast%3DPrice%26aufirst%3DT.%2BJ.%26aulast%3DVagner%26aufirst%3DJ.%26aulast%3DBoitano%26aufirst%3DS.%26atitle%3DLanthanide%2520labeling%2520of%2520a%2520potent%2520protease%2520activated%2520receptor-2%2520agonist%2520for%2520time-resolved%2520fluorescence%2520analysis%26jtitle%3DBioconjugate%2520Chem.%26date%3D2012%26volume%3D23%26spage%3D2098%26epage%3D2104%26doi%3D10.1021%2Fbc300300q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Hu, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamman, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span> </span><span class="NLM_article-title">A homogeneous G protein-coupled receptor ligand binding assay based on time-resolved fluorescence resonance energy transfer</span> <span class="citation_source-journal">Assay Drug Dev. Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">543</span><span class="NLM_x">–</span> <span class="NLM_lpage">550</span><span class="refDoi"> DOI: 10.1089/adt.2008.152</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1089%2Fadt.2008.152" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=18699727" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFWltL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2008&pages=543-550&author=L.+A.+Huauthor=T.+Zhouauthor=B.+D.+Hammanauthor=Q.+Liu&title=A+homogeneous+G+protein-coupled+receptor+ligand+binding+assay+based+on+time-resolved+fluorescence+resonance+energy+transfer&doi=10.1089%2Fadt.2008.152"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A Homogeneous G Protein-Coupled Receptor Ligand Binding Assay Based on Time-Resolved Fluorescence Resonance Energy Transfer</span></div><div class="casAuthors">Hu, Liaoyuan A.; Zhou, Tian; Hamman, Brian D.; Liu, Qingyun</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">543-550</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Fluorescence resonance energy transfer (FRET) has emerged as a powerful tool to the study of protein-protein interactions, such as receptor-ligand binding.  However, the application of FRET to the study of G protein-coupled receptors (GPCRs) has been limited by the method of labeling receptor with fluorescence probes.  Here the authors described a novel time-resolved (TR)-FRET method to study GPCR-ligand binding by using human complement 5a (C5a) receptor (C5aR) as a model system.  Human C5aR was expressed in human embryonic kidney 293 cells with a hemagglutinin (HA) epitope at the N-terminus.  Purified human C5a was labeled with terbium chelate and used as the fluorescence donor.  Monoclonal anti-HA antibody conjugated with Alexa Fluor 488 was used as the fluorescence acceptor.  Robust FRET signal was obsd. when the labeled ligand and C5aR membrane were mixed in the presence of the conjugated anti-HA antibody.  This FRET signal was specific and saturable.  C5a binding affinity to C5aR measured by the FRET assay was consistent with the data as detd. by competition binding anal. using radiolabeled C5a.  The FRET assay was also used to det. affinity of C5aR antagonists by competition binding anal., and the data are similar to those from radioligand binding studies.  Compared to the commonly used radioligand binding assay, this TR-FRET-based assay provides a nonradioactive, faster, and sensitive homogeneous assay format that could be easily adapted to high-throughput screening.  The principle of this assay should also be applicable to other GPCRs, esp. to those receptors with peptide or protein ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMRxTAXC_4dbVg90H21EOLACvtfcHk0li9zISrU91KxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFWltL3J&md5=60084dad6b902bc19ddffb5c22483667</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1089%2Fadt.2008.152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2008.152%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DL.%2BA.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DLiu%26aufirst%3DQ.%26atitle%3DA%2520homogeneous%2520G%2520protein-coupled%2520receptor%2520ligand%2520binding%2520assay%2520based%2520on%2520time-resolved%2520fluorescence%2520resonance%2520energy%2520transfer%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2008%26volume%3D6%26spage%3D543%26epage%3D550%26doi%3D10.1089%2Fadt.2008.152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samama, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burbaum, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroke, I. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appell, K. C.</span><span> </span><span class="NLM_article-title">Chemokine receptor-ligand interactions measured using time-resolved fluorescence</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">2372</span><span class="NLM_x">–</span> <span class="NLM_lpage">2377</span><span class="refDoi"> DOI: 10.1021/bi972161u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi972161u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=2372-2377&author=J.+Ingleseauthor=P.+Samamaauthor=S.+Patelauthor=J.+Burbaumauthor=I.+L.+Strokeauthor=K.+C.+Appell&title=Chemokine+receptor-ligand+interactions+measured+using+time-resolved+fluorescence&doi=10.1021%2Fbi972161u"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fbi972161u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi972161u%26sid%3Dliteratum%253Aachs%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DSamama%26aufirst%3DP.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DBurbaum%26aufirst%3DJ.%26aulast%3DStroke%26aufirst%3DI.%2BL.%26aulast%3DAppell%26aufirst%3DK.%2BC.%26atitle%3DChemokine%2520receptor-ligand%2520interactions%2520measured%2520using%2520time-resolved%2520fluorescence%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D2372%26epage%3D2377%26doi%3D10.1021%2Fbi972161u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Stoner, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pace, H.</span><span> </span><span class="NLM_article-title">A.Asenapine: a clinical review of a second-generation antipsychotic</span> <span class="citation_source-journal">Clin. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">1023</span><span class="NLM_x">–</span> <span class="NLM_lpage">1040</span><span class="refDoi"> DOI: 10.1016/j.clinthera.2012.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2Fj.clinthera.2012.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=22494521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1ajsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2012&pages=1023-1040&author=S.+C.+Stonerauthor=H.+Pace&title=A.Asenapine%3A+a+clinical+review+of+a+second-generation+antipsychotic&doi=10.1016%2Fj.clinthera.2012.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Asenapine: A Clinical Review of a Second-Generation Antipsychotic</span></div><div class="casAuthors">Stoner, Steven C.; Pace, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1023-1040</span>CODEN:
                <span class="NLM_cas:coden">CLTHDG</span>;
        ISSN:<span class="NLM_cas:issn">0149-2918</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Background: Schizophrenia and bipolar disorder are both prevalent types of psychiatric illness in the United States.  As second-generation antipsychotics have become a more viable first-line treatment option, their use has been assocd. with a new era of adverse events (AEs), most notably metabolic and cardiovascular concerns.  Although treatment options for schizophrenia and bipolar disorder have arguably improved, there continues to be a need for medications that achieve and maintain desired efficacy with minimal AEs.  Objectives: This article serves as a comprehensive review of the pharmacol. profile of the second-generation antipsychotic asenapine, as well as a review of its efficacy and safety profiles based on the findings from clin. trials in schizophrenia and bipolar disorder.  Methods: Searches of Ovid MEDLINE, EMBASE, and IDIS were conducted (Jan. 1996 to Nov. 2011) to identify clin. studies and other primary literature sources with the following search terms: asenapine, bipolar disorder, antipsychotic, psychosis, dopamine, and schizophrenia.  Only studies of asenapine and placebo and/or active-comparator arms were included.  Results: The literature search yielded 67 unique articles, including review articles, which were excluded.  The efficacy of asenapine was reported in 3 clin. studies in patients with schizophrenia, 1 each in acute and long-term settings, measured as significant changes in Pos. and Neg. Syndrome Scale scores over 6 and 52 wk.  Asenapine also had reported efficacy in the prevention of relapse in schizophrenia during a 26-wk extension study.  In addn., efficacy of asenapine was reported in 2 studies in acute mania as well as extension phases of both 9 and 40 wk, as detd. by significant changes in Young Mania Rating Scale scores.  The most commonly reported AEs in these studies were somnolence (13%-24%), extrapyramidal symptoms (EPS) (7%-12%), and dizziness (11%).  Conclusions: The findings from multiple studies have suggested that asenapine is efficacious in the acute treatment of schizophrenia.  Asenapine has reported long-term efficacy for this indication and the potential to reduce the incidence of relapse.  Asenapine efficacy was also reported in the treatment of acute manic or mixed states assocd. with bipolar I disorder.  Asenapine had an acceptable safety profile across the different disease states studied, although it was not devoid of metabolic and EPS-related AEs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNbjsE9Lqut7Vg90H21EOLACvtfcHk0li9zISrU91KxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1ajsbo%253D&md5=997c53bd27066251cf80f02d950b2139</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.clinthera.2012.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.clinthera.2012.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DStoner%26aufirst%3DS.%2BC.%26aulast%3DPace%26aufirst%3DH.%26atitle%3DA.Asenapine%253A%2520a%2520clinical%2520review%2520of%2520a%2520second-generation%2520antipsychotic%26jtitle%3DClin.%2520Ther.%26date%3D2012%26volume%3D34%26spage%3D1023%26epage%3D1040%26doi%3D10.1016%2Fj.clinthera.2012.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Shahid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zorn, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, E. H.</span><span> </span><span class="NLM_article-title">Asenapine: a novel psychopharmacologic agent with a unique human receptor signature</span> <span class="citation_source-journal">J. Psychopharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">65</span><span class="NLM_x">–</span> <span class="NLM_lpage">73</span><span class="refDoi"> DOI: 10.1177/0269881107082944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1177%2F0269881107082944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=18308814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVClu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=65-73&author=M.+Shahidauthor=G.+B.+Walkerauthor=S.+H.+Zornauthor=E.+H.+Wong&title=Asenapine%3A+a+novel+psychopharmacologic+agent+with+a+unique+human+receptor+signature&doi=10.1177%2F0269881107082944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Asenapine: a novel psychopharmacologic agent with a unique human receptor signature</span></div><div class="casAuthors">Shahid, M.; Walker, G. B.; Zorn, S. H.; Wong, E. H. F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Psychopharmacology (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-73</span>CODEN:
                <span class="NLM_cas:coden">JOPSEQ</span>;
        ISSN:<span class="NLM_cas:issn">0269-8811</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications Ltd.</span>)
        </div><div class="casAbstract">Asenapine is a novel psychopharmacol. agent under development for the treatment of schizophrenia and bipolar disorder.  We detd. and compared the human receptor binding affinities and functional characteristics of asenapine and several antipsychotic drugs.  Compds. were tested under comparable assay conditions using cloned human receptors.  In comparison with the antipsychotics, asenapine showed high affinity and a different rank order of binding affinities (pKi) for serotonin receptors (5-HT1A [8.6], 5-HT1B [8.4], 5-HT2A [10.2], 5-HT2B [9.8], 5-HT2C [10.5], 5-HT5 [8.8], 5-HT6 [9.6] and 5-HT7 [9.9]), adrenoceptors (α1 [8.9], α2A [8.9], α2B [9.5] and α2C [8.9]), dopamine receptors (D1 [8.9], D2 [8.9], D3 [9.4] and D4 [9.0]) and histamine receptors (H1 [9.0] and H2 [8.2]).  It had much lower affinity (pKi ≤ 5) for muscarinic receptors and was the only agent with affinity for H2 receptors.  Relative to its D2 receptor affinity, asenapine had a higher affinity for 5-HT2C, 5-HT2A, 5-HT2B, 5-HT7, 5-HT6, α2B and D3 receptors, suggesting stronger engagement of these targets at therapeutic doses.  Asenapine behaved as a potent antagonist (pKB) at 5-HT1A (7.4), 5-HT1B (8.1), 5-HT2A (9.0), 5-HT2B (9.3), 5-HT2C (9.0), 5-HT6 (8.0), 5-HT7 (8.5), D2 (9.1), D3 (9.1), α2A (7.3), α2B (8.3), α2C (6.8) and H1 (8.4) receptors.  These functional effects differed from those of risperidone (pKB < 5 for 5-HT6) and olanzapine (pKB < 5 for 5-HT1A and α2).  Our results indicate that asenapine has a unique human receptor signature, with binding affinity and antagonistic properties that differ appreciably from those of antipsychotic drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjDUIlfl2wyLVg90H21EOLACvtfcHk0li9zISrU91KxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVClu7Y%253D&md5=4b89d79dd6fcbf958a02198312569ae6</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1177%2F0269881107082944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0269881107082944%26sid%3Dliteratum%253Aachs%26aulast%3DShahid%26aufirst%3DM.%26aulast%3DWalker%26aufirst%3DG.%2BB.%26aulast%3DZorn%26aufirst%3DS.%2BH.%26aulast%3DWong%26aufirst%3DE.%2BH.%26atitle%3DAsenapine%253A%2520a%2520novel%2520psychopharmacologic%2520agent%2520with%2520a%2520unique%2520human%2520receptor%2520signature%26jtitle%3DJ.%2520Psychopharmacol.%26date%3D2009%26volume%3D23%26spage%3D65%26epage%3D73%26doi%3D10.1177%2F0269881107082944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Olofson, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martz, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senet, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piteau, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malfroot, T.</span><span> </span><span class="NLM_article-title">A new reagent for the selective, high-yield N-dealkylation of tertiary amines: improved syntheses of naltrexone and nalbuphine</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1984</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">2081</span><span class="NLM_x">–</span> <span class="NLM_lpage">2082</span><span class="refDoi"> DOI: 10.1021/jo00185a072</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00185a072" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADyaL2cXktlyks7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1984&pages=2081-2082&author=R.+A.+Olofsonauthor=J.+T.+Martzauthor=J.-P.+Senetauthor=M.+Piteauauthor=T.+Malfroot&title=A+new+reagent+for+the+selective%2C+high-yield+N-dealkylation+of+tertiary+amines%3A+improved+syntheses+of+naltrexone+and+nalbuphine&doi=10.1021%2Fjo00185a072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A new reagent for the selective, high-yield N-dealkylation of tertiary amines:  improved syntheses of naltrexone and nalbuphine</span></div><div class="casAuthors">Olofson, R. A.; Martz, Jonathan T.; Senet, Jean Pierre; Piteau, Marc; Malfroot, Thierry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2081-2</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Reaction of tertiary amines with ClCO2CHClMe in ClCH2CH2Cl at reflux affords the intermediates, R2NCO2CHClMe (R = alkyl, R2 = cycloalkyl), which are converted to the desired salts, R2NH·HCl, just by heating in MeOH.  The N-dealkylation reagent is made in 96% yield by mixing MeCHO with COCl2 in the presence of an R4N+ Cl- catalyst.  The N-dealkylations are highly selective, mild, occur in high yield, and permit the presence of a variety of functionalities.  The process is exemplified by the conversion of glaucine (I) to the phenanthrene (II) in 89% yield and by the N-demethylations of arecoline, O-acetyltropine, and 6-acetylcodeine in yields of 95-97%.  The methodol. was used as part of a process for the transformation of oxycodone to the com. narcotic antagonist, naltrexone (58% overall yield) and to the prescription analgesic, nalbuphine (III) (69% overall yield).  Arom. tertiary amines also are N-dealkylated, although under more vigorous conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcSuAJiJApjLVg90H21EOLACvtfcHk0lhnfwMbiHhXbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXktlyks7k%253D&md5=9402bfb15be7a0d057bfe7bf95bde7b6</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjo00185a072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00185a072%26sid%3Dliteratum%253Aachs%26aulast%3DOlofson%26aufirst%3DR.%2BA.%26aulast%3DMartz%26aufirst%3DJ.%2BT.%26aulast%3DSenet%26aufirst%3DJ.-P.%26aulast%3DPiteau%26aufirst%3DM.%26aulast%3DMalfroot%26aufirst%3DT.%26atitle%3DA%2520new%2520reagent%2520for%2520the%2520selective%252C%2520high-yield%2520N-dealkylation%2520of%2520tertiary%2520amines%253A%2520improved%2520syntheses%2520of%2520naltrexone%2520and%2520nalbuphine%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1984%26volume%3D49%26spage%3D2081%26epage%3D2082%26doi%3D10.1021%2Fjo00185a072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Bouzide, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauvé, G.</span><span> </span><span class="NLM_article-title">Silver(I) oxide mediated highly selective monotosylation of symmetrical diols. Application to the synthesis of polysubstituted cyclic ethers</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">2329</span><span class="NLM_x">–</span> <span class="NLM_lpage">2332</span><span class="refDoi"> DOI: 10.1021/ol020071y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol020071y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD38Xktl2mt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2002&pages=2329-2332&author=A.+Bouzideauthor=G.+Sauv%C3%A9&title=Silver%28I%29+oxide+mediated+highly+selective+monotosylation+of+symmetrical+diols.+Application+to+the+synthesis+of+polysubstituted+cyclic+ethers&doi=10.1021%2Fol020071y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Silver(I) Oxide Mediated Highly Selective Monotosylation of Symmetrical Diols. Application to the Synthesis of Polysubstituted Cyclic Ethers</span></div><div class="casAuthors">Bouzide, Abderrahim; Sauve, Gilles</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2329-2332</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tosylation of sym. diols and oligo(ethylene glycol)s, e.g. I, with a stoichiometric amt. of 4-MeC6H4SO2Cl in the presence of Ag2O and a catalytic amt. of KI led selectively to the corresponding monotosylates, e.g. II, in high yields.  Polysubstituted cyclic ethers, e.g. III, were obtained readily upon treatment of the corresponding diols, e.g. I, with an excess of silver oxide.  The high selectivity was explained on the basis of the difference in acidity between the two hydroxy groups, which are involved in intramol. hydrogen bonding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou01kIoVIUV7Vg90H21EOLACvtfcHk0lhnfwMbiHhXbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xktl2mt7s%253D&md5=769cf574899c1c8fa167fe556bb42dcc</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fol020071y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol020071y%26sid%3Dliteratum%253Aachs%26aulast%3DBouzide%26aufirst%3DA.%26aulast%3DSauv%25C3%25A9%26aufirst%3DG.%26atitle%3DSilver%2528I%2529%2520oxide%2520mediated%2520highly%2520selective%2520monotosylation%2520of%2520symmetrical%2520diols.%2520Application%2520to%2520the%2520synthesis%2520of%2520polysubstituted%2520cyclic%2520ethers%26jtitle%3DOrg.%2520Lett.%26date%3D2002%26volume%3D4%26spage%3D2329%26epage%3D2332%26doi%3D10.1021%2Fol020071y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Knight, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Misra, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quirk, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benwell, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Revell, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennett, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bickerdike, M.</span><span> </span><span class="NLM_article-title">Pharmacological characterisation of the agonist radioligand binding site of 5-HT(2A), 5-HT(2B) and 5-HT(2C) receptors</span> <span class="citation_source-journal">Naunyn-Schmiedeberg's Arch. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">370</span><span class="NLM_x">, </span> <span class="NLM_fpage">114</span><span class="NLM_x">–</span> <span class="NLM_lpage">123</span><span class="refDoi"> DOI: 10.1007/s00210-004-0951-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1007%2Fs00210-004-0951-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=15322733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD2cXoslyrtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=370&publication_year=2004&pages=114-123&author=A.+R.+Knightauthor=A.+Misraauthor=K.+Quirkauthor=K.+Benwellauthor=D.+Revellauthor=G.+Kennettauthor=M.+Bickerdike&title=Pharmacological+characterisation+of+the+agonist+radioligand+binding+site+of+5-HT%282A%29%2C+5-HT%282B%29+and+5-HT%282C%29+receptors&doi=10.1007%2Fs00210-004-0951-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological characterisation of the agonist radioligand binding site of 5-HT2A, 5-HT2B and 5-HT2C receptors</span></div><div class="casAuthors">Knight, Antony R.; Misra, Anil; Quirk, Kathleen; Benwell, Karen; Revell, Dean; Kennett, Guy; Bickerdike, Mike</div><div class="citationInfo"><span class="NLM_cas:title">Naunyn-Schmiedeberg's Archives of Pharmacology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">370</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">114-123</span>CODEN:
                <span class="NLM_cas:coden">NSAPCC</span>;
        ISSN:<span class="NLM_cas:issn">0028-1298</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">In the present study we compared the affinity of various drugs for the high affinity "agonist-preferring" binding site of human recombinant 5-HT2A, 5-HT2B and 5-HT2C receptors stably expressed in monoclonal mammalian cell lines.  To ensure that the "agonist-preferring" conformation of the receptor was preferentially labeled in competition binding expts., satn. anal. was conducted using antagonist and agonist radiolabels at each receptor.  Antagonist radiolabels ([3H]-ketanserin for 5-HT2A receptor and [3H]-mesulergine for 5-HT2B and 5-HT2C receptor) bound to a larger population of receptors in each prepn. than the corresponding agonist radiolabel ([125I]-DOI for 5-HT2A receptor binding and [3H]-5-HT for 5-HT2B and 5-HT2C receptor binding).  Competition expts. were subsequently conducted against appropriate concns. of the agonist radiolabels bound to the "agonist-preferring" subset of receptors in each prepn.  These studies confirmed that there are a no. of highly selective antagonists available to investigate 5-HT2 receptor subtype function (for example, MDL 100907, RS-127445 and RS-102221 for 5-HT2A, 5-HT2B and 5-HT2C receptors resp.).  There remains, however, a lack of highly selective agonists.  (-)DOI is potent and moderately selective for 5-HT2A receptors.  BW723C86 has poor selectivity for human 5-HT2B receptors, while Org 37684 and VER-3323 display some selectivity for the 5-HT2C receptor.  We report for the first time in a single study, the selectivity of numerous serotonergic drugs for 5-HT2 receptors from the same species, in mammalian cell lines and using, exclusively, agonist radiolabels.  The results indicate the importance of defining the selectivity of pharmacol. tools, which may have been over-estd. in the past, and highlights the need to find more selective agonists to investigate 5-HT2 receptor pharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDwPm2ckoq17Vg90H21EOLACvtfcHk0lhnfwMbiHhXbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXoslyrtrk%253D&md5=c8c65b650fe62433da2ba0d77dcbe4fd</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2Fs00210-004-0951-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00210-004-0951-4%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DA.%2BR.%26aulast%3DMisra%26aufirst%3DA.%26aulast%3DQuirk%26aufirst%3DK.%26aulast%3DBenwell%26aufirst%3DK.%26aulast%3DRevell%26aufirst%3DD.%26aulast%3DKennett%26aufirst%3DG.%26aulast%3DBickerdike%26aufirst%3DM.%26atitle%3DPharmacological%2520characterisation%2520of%2520the%2520agonist%2520radioligand%2520binding%2520site%2520of%25205-HT%25282A%2529%252C%25205-HT%25282B%2529%2520and%25205-HT%25282C%2529%2520receptors%26jtitle%3DNaunyn-Schmiedeberg%2527s%2520Arch.%2520Pharmacol.%26date%3D2004%26volume%3D370%26spage%3D114%26epage%3D123%26doi%3D10.1007%2Fs00210-004-0951-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Wainscott, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schenck, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nissen, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baez, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kursar, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucaites, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, D. L.</span><span> </span><span class="NLM_article-title">Pharmacological characteristics of the newly cloned rat 5-hydroxytryptamine2F receptor</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">419</span><span class="NLM_x">–</span> <span class="NLM_lpage">426</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=1993&pages=419-426&author=D.+B.+Wainscottauthor=M.+L.+Cohenauthor=K.+W.+Schenckauthor=J.+E.+Audiaauthor=J.+S.+Nissenauthor=M.+Baezauthor=J.+D.+Kursarauthor=V.+L.+Lucaitesauthor=D.+L.+Nelson&title=Pharmacological+characteristics+of+the+newly+cloned+rat+5-hydroxytryptamine2F+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWainscott%26aufirst%3DD.%2BB.%26aulast%3DCohen%26aufirst%3DM.%2BL.%26aulast%3DSchenck%26aufirst%3DK.%2BW.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DNissen%26aufirst%3DJ.%2BS.%26aulast%3DBaez%26aufirst%3DM.%26aulast%3DKursar%26aufirst%3DJ.%2BD.%26aulast%3DLucaites%26aufirst%3DV.%2BL.%26aulast%3DNelson%26aufirst%3DD.%2BL.%26atitle%3DPharmacological%2520characteristics%2520of%2520the%2520newly%2520cloned%2520rat%25205-hydroxytryptamine2F%2520receptor%26jtitle%3DMol.%2520Pharmacol.%26date%3D1993%26volume%3D43%26spage%3D419%26epage%3D426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Millan, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maiofiss, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cussac, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audinot, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boutin, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman-Tancredi, A.</span><span> </span><span class="NLM_article-title">Differential actions of antiparkinson agents at multiple classes of monoaminergic receptor. I. A multivariate analysis of the binding profiles of 14 drugs at 21 native and cloned human receptor subtypes</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">791</span><span class="NLM_x">–</span> <span class="NLM_lpage">804</span><span class="refDoi"> DOI: 10.1124/jpet.102.039867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1124%2Fjpet.102.039867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=12388666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD38XotlCnsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2002&pages=791-804&author=M.+J.+Millanauthor=L.+Maiofissauthor=D.+Cussacauthor=V.+Audinotauthor=J.+A.+Boutinauthor=A.+Newman-Tancredi&title=Differential+actions+of+antiparkinson+agents+at+multiple+classes+of+monoaminergic+receptor.+I.+A+multivariate+analysis+of+the+binding+profiles+of+14+drugs+at+21+native+and+cloned+human+receptor+subtypes&doi=10.1124%2Fjpet.102.039867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Differential actions of antiparkinson agents at multiple classes of monoaminergic receptor. I. A multivariate analysis of the binding profiles of 14 drugs at 21 native and cloned human receptor subtypes</span></div><div class="casAuthors">Millan, Mark J.; Maiofiss, Lisa; Cussac, Didier; Audinot, Valerie; Boutin, Jean-A.; Newman-Tancredi, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">791-804</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Because little comparative information is available concerning receptor profiles of antiparkinson drugs, affinities of 14 agents were detd. at diverse receptors implicated in the etiol. and/or treatment of Parkinson's disease: human (h)D1, hD2S, hD2L, hD3, hD4, and hD5 receptors; human 5-hydroxytryptamine (5-HT)1A, h5-HT1B, h5-HT1D, h5-HT2A, h5-HT2B, and h5-HT2C receptors; hα1A-, hα1B-, hα1D-, hα2A-, hα2B-, hα2C-, rat α2D-, hβ1-, and hβ2-adrenoceptors (ARs); and native histamine1 receptors.  A correlation matrix (294 pKi values) demonstrated substantial "co-variance".  Correspondingly, principal components anal. revealed that axis 1, which accounted for 76% variance, was assocd. with the majority of receptor types: drugs displaying overall high vs. modest affinities migrated at opposite extremities.  Axis 2 (7% of variance) differentiated drugs with high affinity for hD4 and H1 receptors vs. hα1-AR subtypes.  Five percent of variance was attributable to axis 3, which distinguished drugs with marked affinity for hβ1- and hβ2-ARs vs. hD5 and 5-HT2A receptors.  Hierarchical (cluster) anal. of global homol. generated a dendrogram differentiating two major groups possessing low vs. high affinity, resp., for multiple serotonergic and hD5 receptors.  Within the first group, quinpirole, quinerolane, ropinirole, and pramipexole interacted principally with hD2, hD3, and hD4 receptors, whereas piribedil and talipexole recognized dopaminergic receptors and hα2-ARs.  Within the second group, lisuride and terguride manifested high affinities for all sites, with roxindole/bromocriptine, cabergoline/pergolide, and 6,7-dihydroxy-N,N-dimethyl-2-ammotetralin (TL99)/apomorphine comprising three addnl. subclusters of closely related ligands.  In conclusion, an innovative multivariate anal. revealed marked heterogeneity in binding profiles of antiparkinson agents.  Actions at sites other than hD2 receptors likely participate in their (contrasting) functional profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1qrMpB3-4sbVg90H21EOLACvtfcHk0linFj0_HlEiEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotlCnsL8%253D&md5=20be25d95375cdcd3308cf0590aed337</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1124%2Fjpet.102.039867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.102.039867%26sid%3Dliteratum%253Aachs%26aulast%3DMillan%26aufirst%3DM.%2BJ.%26aulast%3DMaiofiss%26aufirst%3DL.%26aulast%3DCussac%26aufirst%3DD.%26aulast%3DAudinot%26aufirst%3DV.%26aulast%3DBoutin%26aufirst%3DJ.%2BA.%26aulast%3DNewman-Tancredi%26aufirst%3DA.%26atitle%3DDifferential%2520actions%2520of%2520antiparkinson%2520agents%2520at%2520multiple%2520classes%2520of%2520monoaminergic%2520receptor.%2520I.%2520A%2520multivariate%2520analysis%2520of%2520the%2520binding%2520profiles%2520of%252014%2520drugs%2520at%252021%2520native%2520and%2520cloned%2520human%2520receptor%2520subtypes%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2002%26volume%3D303%26spage%3D791%26epage%3D804%26doi%3D10.1124%2Fjpet.102.039867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Napier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melrose, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHarg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallis, R.</span><span> </span><span class="NLM_article-title">Characterisation of the 5-HT receptor binding profile of eletriptan and kinetics of [3H]eletriptan binding at human 5-HT1B and 5-HT1D receptors</span> <span class="citation_source-journal">Eur. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">368</span><span class="NLM_x">, </span> <span class="NLM_fpage">259</span><span class="NLM_x">–</span> <span class="NLM_lpage">268</span><span class="refDoi"> DOI: 10.1016/S0014-2999(99)00026-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1016%2FS0014-2999%2899%2900026-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10193663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADyaK1MXhs1OitLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=1999&pages=259-268&author=C.+Napierauthor=M.+Stewartauthor=H.+Melroseauthor=B.+Hopkinsauthor=A.+McHargauthor=R.+Wallis&title=Characterisation+of+the+5-HT+receptor+binding+profile+of+eletriptan+and+kinetics+of+%5B3H%5Deletriptan+binding+at+human+5-HT1B+and+5-HT1D+receptors&doi=10.1016%2FS0014-2999%2899%2900026-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the 5-HT receptor binding profile of eletriptan and kinetics of [3H]eletriptan binding at human 5-HT1B and 5-HT1D receptors</span></div><div class="casAuthors">Napier, Carolyn; Stewart, Michael; Melrose, Heather; Hopkins, Brian; McHarg, Aileen; Wallis, Rob</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">2/3</span>),
    <span class="NLM_cas:pages">259-268</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">The affinity of eletriptan ((R)-3-(1-methyl-2-pyrrolidinylmethyl)-5-[2-(phenylsulfonyl)ethyl]-1H-indole) for a range of 5-HT receptors was compared to values obtained for other 5-HT1B/1D receptor agonists known to be effective in the treatment of migraine.  Eletriptan, like sumatriptan, zolmitriptan, naratriptan and rizatriptan had highest affinity for the human 5-HT1B, 5-HT1D and putative 5-HT1f receptor.  Kinetic studies comparing the binding of [3H]eletriptan and [3H]sumatriptan to the human recombinant 5-HT1B and 5-HT1D receptors expressed in HeLa cells revealed that both radioligands bound with high specificity (>90%) and reached equil. within 10-15 min.  However, [3H]eletriptan had over 6-fold higher affinity than [3H]sumatriptan at the 5-HT1D receptor (KD: 0.92 and 6.58 nM, resp.) and over 3-fold higher affinity than [3H]sumatriptan at the 5-HT1B receptor (KD: 3.14 and 11.07 nM, resp.).  Assocn. and dissocn. rates for both radioligands could only be accurately detd. at the 5-HT1D receptor and then only at 4°.  At this temp., [3H]eletriptan had a significantly faster assocn. rate (Kon 0.249 min-1 nM-1) than [3H]sumatriptan (Kon 0.024 min-1 nM-1) and a significantly slower off-rate (Koff 0.027 min-1 compared to 0.037 min-1 for [3H]sumatriptan).  These data indicate that eletriptan is a potent ligand at the human 5-HT1B, 5-HT1D and 5-HT1f receptors and are consistent with its potent vasoconstrictor activity and use as a drug for the acute treatment of migraine headache.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoikpaa981kkbVg90H21EOLACvtfcHk0linFj0_HlEiEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhs1OitLs%253D&md5=9c99c3028504aaaab34ec8059b9486ed</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2FS0014-2999%2899%2900026-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-2999%252899%252900026-6%26sid%3Dliteratum%253Aachs%26aulast%3DNapier%26aufirst%3DC.%26aulast%3DStewart%26aufirst%3DM.%26aulast%3DMelrose%26aufirst%3DH.%26aulast%3DHopkins%26aufirst%3DB.%26aulast%3DMcHarg%26aufirst%3DA.%26aulast%3DWallis%26aufirst%3DR.%26atitle%3DCharacterisation%2520of%2520the%25205-HT%2520receptor%2520binding%2520profile%2520of%2520eletriptan%2520and%2520kinetics%2520of%2520%255B3H%255Deletriptan%2520binding%2520at%2520human%25205-HT1B%2520and%25205-HT1D%2520receptors%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D1999%26volume%3D368%26spage%3D259%26epage%3D268%26doi%3D10.1016%2FS0014-2999%2899%2900026-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Leysen, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gommeren, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heylen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luyten, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van de Weyer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanhoenacker, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haegeman, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schotte, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Gompel, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wouters, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lesage, A. S.</span><span> </span><span class="NLM_article-title">Alniditan, a new 5-hydroxytryptamine1D agonist and migraine-abortive agent: ligand-binding properties of human 5-hydroxytryptamine1D alpha, human 5-hydroxytryptamine1D beta, and calf 5-hydroxytryptamine1D receptors investigated with [3H]5-hydroxytryptamine and [3H]alniditan</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1567</span><span class="NLM_x">–</span> <span class="NLM_lpage">1580</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=8967979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADyaK2sXntlyi" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1996&pages=1567-1580&author=J.+E.+Leysenauthor=W.+Gommerenauthor=L.+Heylenauthor=W.+H.+Luytenauthor=I.+Van+de+Weyerauthor=P.+Vanhoenackerauthor=G.+Haegemanauthor=A.+Schotteauthor=P.+Van+Gompelauthor=R.+Woutersauthor=A.+S.+Lesage&title=Alniditan%2C+a+new+5-hydroxytryptamine1D+agonist+and+migraine-abortive+agent%3A+ligand-binding+properties+of+human+5-hydroxytryptamine1D+alpha%2C+human+5-hydroxytryptamine1D+beta%2C+and+calf+5-hydroxytryptamine1D+receptors+investigated+with+%5B3H%5D5-hydroxytryptamine+and+%5B3H%5Dalniditan"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Alniditan, a new 5-hydroxytryptamine1D agonist and migraine-abortive agent: ligand-binding properties of human 5-hydroxytryptamine1Dα, human 5-hydroxytryptamine1Dβ, and calf 5-hydroxyptryptamineaD receptors investigated with [3H]5-hydroxytryptamine and [3H]alniditan</span></div><div class="casAuthors">Leysen, Jose E.; Gommeren, Walter; Heylen, Lieve; Luyten, Walter H. M. L.; van de Weyer, Inez; Vanhoenacker, Peter; Haegeman, Guy; Schotte, Alain; van Gompel, Paul; Wouters, Ria; Lesage, Anne S.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1567-1580</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">Williams & Wilkins</span>)
        </div><div class="casAbstract">Alniditan is a new migraine-abortive agent.  It is a benzopyran deriv. and therefore structurally unrelated to sumatriptan and other indole-derivs. and to ergoline derivs.  The action of sumatriptan is thought to be mediated by 5-hydroxytryptamine (5-HT)1D-type receptors.  We investigated the receptor-binding profile in vitro of alniditan compared with sumatriptan and dihydroergotamine for 28 neurotransmitter receptor subtypes, several receptors for peptides and lipid-derived factors, ion channel-binding sites, and monoamine transporters.  Alniditan revealed nanomolar affinity for calf substantia nigra 5-HT1D and for cloned h5-HT1Dα, h-HT1Dβ, and h5-HT1A receptors (Ki = 0.8, 0.4, 1.1, and 3.8 nM, resp.).  Alniditan was more potent than sumatriptan at 5-HT1D-type and 5-HT1A receptors.  Alniditan showed moderate-to-low or no affinity for other investigated receptors; sumatriptan showed addnl. binding to 5-HT1F receptors.  Dihydroergotamine had a much broader profile with high affinity for several 5-HT, adrenergic and dopaminergic receptors in signal transduction assays using cells expressing recombinant h5-HT1Dα, h5-HT1Dβ, or h5-HT1A receptors, alniditan (like 5-HT) was a full agonist for inhibition of stimulated adenylyl cyclase (IC50 = 1.1, 1.3, and 74 nM, resp., for alniditan).  Therefore, in functional assays, the potency of alniditan was much higher at 5-HT1D receptors than at 5-HT1A receptors.  We further compared the properties of [3H]alniditan, as a new radioligand for 5-HT1D-type receptors, with those of [3H]5-HT in membrane prepns. of calf substantia nigra, C6 glioma cells expressing h5-HT1Dα, and L929 cells expressing h5-HT1Dβ receptors.  [3H]Alniditan revealed very rapid assocn. and dissocn. binding kinetics and showed slightly higher affinity (Kd = 1-2 nM) than [3H]5-HT.  We investigated 25 compds. for inhibition of [3H]alniditan and [3H]5-HT binding in the three membrane prepns.; Ki values of the radioligands were largely similar, although some subtle differences appeared.  Most compds. did not differentiate between 5-HT1Dα and 5-HT1Dβ receptors, except methylsergide, ritanserin, ocaperidone, risperidone, and ketanserin, which showed 10-60-fold higher affinity for the 5-HT1Dα receptor.  The Ki values of the compds. obtained with 5-HT1D receptors in calf substantia nigra indicated that these receptors are of the 5-HT1Dβ-type.  We demonstrated that alniditan is a potent agonist at h5-HT1Dα and h5-HT1Dβ receptors; its properties probably underlie its cranial vasoconstrictive and antimigraine properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoooU3fQwv0i7Vg90H21EOLACvtfcHk0ljKkmRYeDVD2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXntlyi&md5=22a4fefe7cf41846f7451461c67fd336</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLeysen%26aufirst%3DJ.%2BE.%26aulast%3DGommeren%26aufirst%3DW.%26aulast%3DHeylen%26aufirst%3DL.%26aulast%3DLuyten%26aufirst%3DW.%2BH.%26aulast%3DVan%2Bde%2BWeyer%26aufirst%3DI.%26aulast%3DVanhoenacker%26aufirst%3DP.%26aulast%3DHaegeman%26aufirst%3DG.%26aulast%3DSchotte%26aufirst%3DA.%26aulast%3DVan%2BGompel%26aufirst%3DP.%26aulast%3DWouters%26aufirst%3DR.%26aulast%3DLesage%26aufirst%3DA.%2BS.%26atitle%3DAlniditan%252C%2520a%2520new%25205-hydroxytryptamine1D%2520agonist%2520and%2520migraine-abortive%2520agent%253A%2520ligand-binding%2520properties%2520of%2520human%25205-hydroxytryptamine1D%2520alpha%252C%2520human%25205-hydroxytryptamine1D%2520beta%252C%2520and%2520calf%25205-hydroxytryptamine1D%2520receptors%2520investigated%2520with%2520%255B3H%255D5-hydroxytryptamine%2520and%2520%255B3H%255Dalniditan%26jtitle%3DMol.%2520Pharmacol.%26date%3D1996%26volume%3D50%26spage%3D1567%26epage%3D1580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">John, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pauwels, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halazy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Grand, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verscheure, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valentin, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wurch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chopin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marien, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleven, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koek, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Assie, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carilla-Durand, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarayre, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colpaert, F. C.</span><span> </span><span class="NLM_article-title">F 11356, a novel 5-hydroxytryptamine (5-HT) derivative with potent, selective, and unique high intrinsic activity at 5-HT1B/1D receptors in models relevant to migraine</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10381763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADyaK1MXktFKisrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=1999&pages=83-95&author=G.+W.+Johnauthor=P.+J.+Pauwelsauthor=M.+Perezauthor=S.+Halazyauthor=B.+Le+Grandauthor=Y.+Verscheureauthor=J.+P.+Valentinauthor=C.+Palmierauthor=T.+Wurchauthor=P.+Chopinauthor=M.+Marienauthor=M.+S.+Klevenauthor=W.+Koekauthor=M.+B.+Assieauthor=E.+Carilla-Durandauthor=J.+P.+Tarayreauthor=F.+C.+Colpaert&title=F+11356%2C+a+novel+5-hydroxytryptamine+%285-HT%29+derivative+with+potent%2C+selective%2C+and+unique+high+intrinsic+activity+at+5-HT1B%2F1D+receptors+in+models+relevant+to+migraine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">F 11356, a novel 5-hydroxytryptamine (5-HT) derivative with potent, selective, and unique high intrinsic activity at 5-HT1B/1D receptors in models relevant to migraine</span></div><div class="casAuthors">John, Gareth W.; Pauwels, Petrus J.; Perez, Michel; Halazy, Serge; Grand, Bruno Le; Verscheure, Yvan; Valentin, Jean-Pierre; Palmier, Christiane; Wurch, Thierry; Chopin, Philippe; Marien, Marc; Kleven, Mark S.; Koek, Wouter; Assie, Marie-Bernadette; Carilla-Durand, Elisabeth; Tarayre, Jean-Pierre; Colpaert, Francis C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">83-95</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">F 11356 (4-[4-[2-(2-aminoethyl)-1H-indol-5-yloxyl]acetylpiperazinyl-1-yl]benzonitrile) was designed to take advantage of the superior potency and efficacy characteristics of 5-hydroxytryptamine (5-HT) compared with tryptamine at 5-HT1B/1D receptors.  F 11356 has subnanomolar affinity for cloned human and nonhuman 5-HT1B and 5-HT1D receptors, and its affinity for 5-HT1A and other 5-HT receptors, including the 5-HT1F subtype, is 50-fold lower and micromolar, resp.  In C6 cells expressing human 5-HT1B or human 5-HT1D receptors, F 11356 was the most potent compd. in inhibiting forskolin-induced cAMP formation (pD2 = 8.9 and 9.6), and in contrast to tryptamine and derivs., it produced maximal enhancement of [35S]guanosine-5'-O-(3-thio)triphosphate-specific binding equiv. to 5-HT.  F 11356 was equipotent to 5-HT (pD2 = 7.1 vs. 7.2) and more potent than tryptamine derivs. in contracting rabbit isolated saphenous vein.  In isolated guinea pig trigeminal ganglion neurons, F 11356 was more potent (pD2 = 7.3 vs. 6.7) and induced greater increases in outward hyperpolarizing Ca2+-dependent K+ current than sumatriptan.  In anesthetized pigs, F 11356 elicited highly cranioselective, more potent (from 0.16 μg/kg i.v.) and greater carotid vasoconstriction than tryptamine derivs.  Decreases in carotid blood flow were obsd. in conscious dogs from 0.63 mg/kg oral F 11356 in the absence of changes in heart rate or behavior.  Oral activity was confirmed when hypothermic responses were elicited in guinea pigs (ED50 = 1.6 mg/kg), suggesting that F 11356 also accesses the brain.  F 11356 thus is a selective, high-potency agonist at 5-HT1B/1D receptors, which distinguishes itself from tryptamine and derivs. in exerting high intrinsic activity at these receptors in vascular and neuronal models relevant to migraine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq8SCtDP2atbVg90H21EOLACvtfcHk0ljKkmRYeDVD2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktFKisrc%253D&md5=492ea2af29bf4a9ccf5527cdc73a2d3c</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJohn%26aufirst%3DG.%2BW.%26aulast%3DPauwels%26aufirst%3DP.%2BJ.%26aulast%3DPerez%26aufirst%3DM.%26aulast%3DHalazy%26aufirst%3DS.%26aulast%3DLe%2BGrand%26aufirst%3DB.%26aulast%3DVerscheure%26aufirst%3DY.%26aulast%3DValentin%26aufirst%3DJ.%2BP.%26aulast%3DPalmier%26aufirst%3DC.%26aulast%3DWurch%26aufirst%3DT.%26aulast%3DChopin%26aufirst%3DP.%26aulast%3DMarien%26aufirst%3DM.%26aulast%3DKleven%26aufirst%3DM.%2BS.%26aulast%3DKoek%26aufirst%3DW.%26aulast%3DAssie%26aufirst%3DM.%2BB.%26aulast%3DCarilla-Durand%26aufirst%3DE.%26aulast%3DTarayre%26aufirst%3DJ.%2BP.%26aulast%3DColpaert%26aufirst%3DF.%2BC.%26atitle%3DF%252011356%252C%2520a%2520novel%25205-hydroxytryptamine%2520%25285-HT%2529%2520derivative%2520with%2520potent%252C%2520selective%252C%2520and%2520unique%2520high%2520intrinsic%2520activity%2520at%25205-HT1B%252F1D%2520receptors%2520in%2520models%2520relevant%2520to%2520migraine%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1999%26volume%3D290%26spage%3D83%26epage%3D95" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Boess, F. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monsma, F. J.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwingelstein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sleight, A. J.</span><span> </span><span class="NLM_article-title">Interaction of tryptamine and ergoline compounds with threonine 196 in the ligand binding site of the 5-hydroxytryptamine6 receptor</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">515</span><span class="NLM_x">–</span> <span class="NLM_lpage">523</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1997&pages=515-523&author=F.+G.+Boessauthor=F.+J.+Monsmaauthor=V.+Meyerauthor=C.+Zwingelsteinauthor=A.+J.+Sleight&title=Interaction+of+tryptamine+and+ergoline+compounds+with+threonine+196+in+the+ligand+binding+site+of+the+5-hydroxytryptamine6+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBoess%26aufirst%3DF.%2BG.%26aulast%3DMonsma%26aufirst%3DF.%2BJ.%26aulast%3DMeyer%26aufirst%3DV.%26aulast%3DZwingelstein%26aufirst%3DC.%26aulast%3DSleight%26aufirst%3DA.%2BJ.%26atitle%3DInteraction%2520of%2520tryptamine%2520and%2520ergoline%2520compounds%2520with%2520threonine%2520196%2520in%2520the%2520ligand%2520binding%2520site%2520of%2520the%25205-hydroxytryptamine6%2520receptor%26jtitle%3DMol.%2520Pharmacol.%26date%3D1997%26volume%3D52%26spage%3D515%26epage%3D523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Routledge, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bromidge, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moss, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirst, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riley, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gager, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stean, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Upton, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Middlemiss, D. N.</span><span> </span><span class="NLM_article-title">Characterization of SB-271046: a potent, selective and orally active 5-HT(6) receptor antagonist</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">1606</span><span class="NLM_x">–</span> <span class="NLM_lpage">1612</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0703457</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10.1038%2Fsj.bjp.0703457" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=10928964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlslygsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2000&pages=1606-1612&author=C.+Routledgeauthor=S.+M.+Bromidgeauthor=S.+F.+Mossauthor=G.+W.+Priceauthor=W.+Hirstauthor=H.+Newmanauthor=G.+Rileyauthor=T.+Gagerauthor=T.+Steanauthor=N.+Uptonauthor=S.+E.+Clarkeauthor=A.+M.+Brownauthor=D.+N.+Middlemiss&title=Characterization+of+SB-271046%3A+a+potent%2C+selective+and+orally+active+5-HT%286%29+receptor+antagonist&doi=10.1038%2Fsj.bjp.0703457"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of SB-271046: a potent, selective and orally active 5-HT6 receptor antagonist</span></div><div class="casAuthors">Routledge, Carol; Bromidge, Steven M.; Moss, Stephen F.; Price, Gary W.; Hirst, Warren; Newman, Helen; Riley, Graham; Gager, Tracey; Stean, Tania; Upton, Neil; Clarke, Stephen E.; Brown, Anthony M.; Middlemiss, Derek N.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1606-1612</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">1 SB-271046, potently displaced [3H]-LSD and [125I]-SB-258585 from human 5-HT6 receptors recombinantly expressed in HeLa cells in vitro (pKi 8.92 and 9.09 resp.).  SB-271046 also displaced [125I]-SB-258585 from human caudate putamen and rat and pig striatum membranes (pKi 8.81, 9.02 and 8.55 resp.).  2 SB-271046 was over 200 fold selective for the 5-HT6 receptor vs 55 other receptors, binding sites and ion channels.  3 In functional studies on human 5-HT6 receptors SB-271046 competitively antagonized 5-HT-induced stimulation of adenylyl cyclase activity with a pA2 of 8.71.  4 SB-271046 produced an increase in seizure threshold over a wide-dose range in the rat maximal electroshock seizure threshold (MEST) test, with a min. ED of ≤ 0.1 mg kg-1 p.o. and max. effect at 4 h post-dose.  The level of anticonvulsant activity achieved correlated well with the blood concns. of SB-271046 (EC50 of 0.16 μM) and brain concns. of 0.01 - 0.04 μM at Cmax.  5 These data, together with the obsd. anticonvulsant activity of other selective 5-HT6 receptor antagonists, SB-258510 (10 mg kg-1, 2-6 h pre-test) and Ro 04-6790 (1-30 mg kg-1, 1 h pre-test), in the rat MEST test, suggest that the anticonvulsant properties of SB-271046 are likely to be mediated by 5-HT6 receptors.  6 Overall, these studies demonstrate that SB-271046 is a potent and selective 5-HT6 receptor antagonist and is orally active in the rat MEST test.  SB-271046 represents a valuable tool for evaluating the in vivo central function of 5-HT6 receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofj-FKPcThe7Vg90H21EOLACvtfcHk0li2oHS5jeyvJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlslygsro%253D&md5=b382383bcd06a3154186b22e9e9582b6</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0703457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0703457%26sid%3Dliteratum%253Aachs%26aulast%3DRoutledge%26aufirst%3DC.%26aulast%3DBromidge%26aufirst%3DS.%2BM.%26aulast%3DMoss%26aufirst%3DS.%2BF.%26aulast%3DPrice%26aufirst%3DG.%2BW.%26aulast%3DHirst%26aufirst%3DW.%26aulast%3DNewman%26aufirst%3DH.%26aulast%3DRiley%26aufirst%3DG.%26aulast%3DGager%26aufirst%3DT.%26aulast%3DStean%26aufirst%3DT.%26aulast%3DUpton%26aufirst%3DN.%26aulast%3DClarke%26aufirst%3DS.%2BE.%26aulast%3DBrown%26aufirst%3DA.%2BM.%26aulast%3DMiddlemiss%26aufirst%3DD.%2BN.%26atitle%3DCharacterization%2520of%2520SB-271046%253A%2520a%2520potent%252C%2520selective%2520and%2520orally%2520active%25205-HT%25286%2529%2520receptor%2520antagonist%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2000%26volume%3D130%26spage%3D1606%26epage%3D1612%26doi%3D10.1038%2Fsj.bjp.0703457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i59"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01220">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_42482"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01220">10.1021/acs.jmedchem.7b01220</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Fluorescence properties of ligands <b>26</b>–<b>34</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01220/suppl_file/jm7b01220_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01220/suppl_file/jm7b01220_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01220/suppl_file/jm7b01220_si_001.pdf">jm7b01220_si_001.pdf (34.64 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01220/suppl_file/jm7b01220_si_002.csv">jm7b01220_si_002.csv (3.36 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01220&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01220%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-1%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01220" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67999d392afa3c64","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
