
<html lang="en"     class="pb-page"  data-request-id="60395a84-cc17-40dc-a91d-6592de303774"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2020.63.issue-15;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c00799;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Characterization of TnmH as an O-Methyltransferase Revealing Insights into Tiancimycin Biosynthesis and Enabling a Biocatalytic Strategy To Prepare Antibody–Tiancimycin Conjugates" /></meta><meta name="dc.Creator" content="Ajeeth  Adhikari" /></meta><meta name="dc.Creator" content="Christiana N.  Teijaro" /></meta><meta name="dc.Creator" content="Xiaohui  Yan" /></meta><meta name="dc.Creator" content="Chin-Yuan  Chang" /></meta><meta name="dc.Creator" content="Chun  Gui" /></meta><meta name="dc.Creator" content="Yu-Chen  Liu" /></meta><meta name="dc.Creator" content="Ivana  Crnovcic" /></meta><meta name="dc.Creator" content="Dong  Yang" /></meta><meta name="dc.Creator" content="Thibault  Annaval" /></meta><meta name="dc.Creator" content="Christoph  Rader" /></meta><meta name="dc.Creator" content="Ben  Shen" /></meta><meta name="dc.Description" content="The enediynes are among the most cytotoxic molecules known, and their use as anticancer drugs has been successfully demonstrated by targeted delivery. Clinical advancement of the anthraquinone-fuse..." /></meta><meta name="Description" content="The enediynes are among the most cytotoxic molecules known, and their use as anticancer drugs has been successfully demonstrated by targeted delivery. Clinical advancement of the anthraquinone-fuse..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 13, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00799" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00799" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00799" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00799" /></link>
        
    
    

<title>Characterization of TnmH as an O-Methyltransferase Revealing Insights into Tiancimycin Biosynthesis and Enabling a Biocatalytic Strategy To Prepare Antibody–Tiancimycin Conjugates | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00799" /></meta><meta property="og:title" content="Characterization of TnmH as an O-Methyltransferase Revealing Insights into Tiancimycin Biosynthesis and Enabling a Biocatalytic Strategy To Prepare Antibody–Tiancimycin Conjugates" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0008.jpeg" /></meta><meta property="og:description" content="The enediynes are among the most cytotoxic molecules known, and their use as anticancer drugs has been successfully demonstrated by targeted delivery. Clinical advancement of the anthraquinone-fused enediynes has been hindered by their low titers and lack of functional groups to enable the preparation of antibody–drug conjugates (ADCs). Here we report biochemical and structural characterization of TnmH from the tiancimycin (TNM) biosynthetic pathway, revealing that (i) TnmH catalyzes regiospecific methylation at the C-7 hydroxyl group, (ii) TnmH exhibits broad substrate promiscuity toward hydroxyanthraquinones and S-alkylated SAM analogues and catalyzes efficient installation of reactive alkyl handles, (iii) the X-ray crystal structure of TnmH provides the molecular basis to account for its broad substrate promiscuity, and (iv) TnmH as a biocatalyst enables the development of novel conjugation strategies to prepare antibody–TNM conjugates. These findings should greatly facilitate the construction and evaluation of antibody–TNM conjugates as next-generation ADCs for targeted chemotherapy." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00799"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00799">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00799&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00799&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00799&amp;href=/doi/10.1021/acs.jmedchem.0c00799" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 8432-8441</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00770" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00803" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Characterization of TnmH as an <i>O</i>-Methyltransferase Revealing Insights into Tiancimycin Biosynthesis and Enabling a Biocatalytic Strategy To Prepare Antibody–Tiancimycin Conjugates</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Ajeeth Adhikari</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ajeeth Adhikari</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry,  Department of Immunology and Microbiology, The Scripps Research Institute, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ajeeth++Adhikari">Ajeeth Adhikari</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christiana N. Teijaro</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christiana N. Teijaro</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christiana+N.++Teijaro">Christiana N. Teijaro</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaohui Yan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaohui Yan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaohui++Yan">Xiaohui Yan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7295-8902" title="Orcid link">http://orcid.org/0000-0001-7295-8902</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chin-Yuan Chang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chin-Yuan Chang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chin-Yuan++Chang">Chin-Yuan Chang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chun Gui</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chun Gui</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chun++Gui">Chun Gui</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yu-Chen Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yu-Chen Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yu-Chen++Liu">Yu-Chen Liu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7232-144X" title="Orcid link">http://orcid.org/0000-0002-7232-144X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ivana Crnovcic</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ivana Crnovcic</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ivana++Crnovcic">Ivana Crnovcic</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dong Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dong Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dong++Yang">Dong Yang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2917-0663" title="Orcid link">http://orcid.org/0000-0003-2917-0663</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thibault Annaval</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thibault Annaval</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thibault++Annaval">Thibault Annaval</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christoph Rader</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christoph Rader</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Immunology and Microbiology, The Scripps Research Institute, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christoph++Rader">Christoph Rader</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9955-3454" title="Orcid link">http://orcid.org/0000-0001-9955-3454</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Ben Shen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ben Shen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry,  Department of Molecular Medicine,  Natural Products Discovery Center at Scripps Research, The Scripps Research Institute, Jupiter, Florida 33458, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#3f4c575a515d7f4c5c4d564f4f4c115a5b4a"><span class="__cf_email__" data-cfemail="8efde6ebe0eccefdedfce7fefefda0ebeafb">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ben++Shen">Ben Shen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9750-5982" title="Orcid link">http://orcid.org/0000-0002-9750-5982</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00799&amp;href=/doi/10.1021%2Facs.jmedchem.0c00799" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 8432–8441</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 13, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 May 2020</li><li><span class="item_label"><b>Published</b> online</span>13 July 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 August 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00799" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00799</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8432%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAjeeth%2BAdhikari%252C%2BChristiana%2BN.%2BTeijaro%252C%2BXiaohui%2BYan%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D15%26contentID%3Dacs.jmedchem.0c00799%26title%3DCharacterization%2Bof%2BTnmH%2Bas%2Ban%2BO-Methyltransferase%2BRevealing%2BInsights%2Binto%2BTiancimycin%2BBiosynthesis%2Band%2BEnabling%2Ba%2BBiocatalytic%2BStrategy%2BTo%2BPrepare%2BAntibody%25E2%2580%2593Tiancimycin%2BConjugates%26numPages%3D10%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8441%26publicationDate%3DAugust%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00799"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">947</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00799" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Characterization of TnmH as an O-Methyltransferase Revealing Insights into Tiancimycin Biosynthesis and Enabling a Biocatalytic Strategy To Prepare Antibody–Tiancimycin Conjugates&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Ajeeth&quot;,&quot;last_name&quot;:&quot;Adhikari&quot;},{&quot;first_name&quot;:&quot;Christiana&quot;,&quot;last_name&quot;:&quot;N. Teijaro&quot;},{&quot;first_name&quot;:&quot;Xiaohui&quot;,&quot;last_name&quot;:&quot;Yan&quot;},{&quot;first_name&quot;:&quot;Chin-Yuan&quot;,&quot;last_name&quot;:&quot;Chang&quot;},{&quot;first_name&quot;:&quot;Chun&quot;,&quot;last_name&quot;:&quot;Gui&quot;},{&quot;first_name&quot;:&quot;Yu-Chen&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Ivana&quot;,&quot;last_name&quot;:&quot;Crnovcic&quot;},{&quot;first_name&quot;:&quot;Dong&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Thibault&quot;,&quot;last_name&quot;:&quot;Annaval&quot;},{&quot;first_name&quot;:&quot;Christoph&quot;,&quot;last_name&quot;:&quot;Rader&quot;},{&quot;first_name&quot;:&quot;Ben&quot;,&quot;last_name&quot;:&quot;Shen&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;13&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;8432-8441&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00799&quot;},&quot;abstract&quot;:&quot;The enediynes are among the most cytotoxic molecules known, and their use as anticancer drugs has been successfully demonstrated by targeted delivery. Clinical advancement of the anthraquinone-fused enediynes has been hindered by their low titers and lack of functional groups to enable the preparation of antibody–drug conjugates (ADCs). Here we report biochemical and structural characterization of TnmH from the tiancimycin (TNM) biosynthetic pathway, revealing that (i) TnmH catalyzes regiospecific methylation at the C-7 hydroxyl group, (ii) TnmH exhibits broad substrate promiscuity toward hydroxyanthraquinones and S-alkylated SAM analogues and catalyzes efficient installation of reactive alkyl handles, (iii) the X-ray crystal structure of TnmH provides the molecular basis to account for its broad substrate promiscuity, and (iv) TnmH as a biocatalyst enables the development of novel conjugation strategies to prepare antibody–TNM conjugates. These findings should greatly facilitate the construction and evaluati&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00799&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00799" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00799&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00799" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00799&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00799" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00799&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00799&amp;href=/doi/10.1021/acs.jmedchem.0c00799" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00799" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00799" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00799%26sid%3Dliteratum%253Aachs%26pmid%3D32658465%26genre%3Darticle%26aulast%3DAdhikari%26date%3D2020%26atitle%3DCharacterization%2Bof%2BTnmH%2Bas%2Ban%2BO-Methyltransferase%2BRevealing%2BInsights%2Binto%2BTiancimycin%2BBiosynthesis%2Band%2BEnabling%2Ba%2BBiocatalytic%2BStrategy%2BTo%2BPrepare%2BAntibody%25E2%2580%2593Tiancimycin%2BConjugates%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D15%26spage%3D8432%26epage%3D8441%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291750" title="Hydroxyls">Hydroxyls</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=290644" title="Molecular properties">Molecular properties</a>,</li><li><a href="/action/doSearch?ConceptID=292349" title="Alkyls">Alkyls</a>,</li><li><a href="/action/doSearch?ConceptID=291290" title="Conjugate acid-base pairs">Conjugate acid-base pairs</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/jmcmar.2020.63.issue-15/20200813/jmcmar.2020.63.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/medium/jm0c00799_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00799&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The enediynes are among the most cytotoxic molecules known, and their use as anticancer drugs has been successfully demonstrated by targeted delivery. Clinical advancement of the anthraquinone-fused enediynes has been hindered by their low titers and lack of functional groups to enable the preparation of antibody–drug conjugates (ADCs). Here we report biochemical and structural characterization of TnmH from the tiancimycin (TNM) biosynthetic pathway, revealing that (i) TnmH catalyzes regiospecific methylation at the C-7 hydroxyl group, (ii) TnmH exhibits broad substrate promiscuity toward hydroxyanthraquinones and S-alkylated SAM analogues and catalyzes efficient installation of reactive alkyl handles, (iii) the X-ray crystal structure of TnmH provides the molecular basis to account for its broad substrate promiscuity, and (iv) TnmH as a biocatalyst enables the development of novel conjugation strategies to prepare antibody–TNM conjugates. These findings should greatly facilitate the construction and evaluation of antibody–TNM conjugates as next-generation ADCs for targeted chemotherapy.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30785" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30785" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The enediyne antitumor antibiotics are some of the most potent cytotoxic agents found in nature. The biological activity of the enediynes is driven by their shared mechanism of action, electronic rearrangement of the macrocyclic enediyne core to produce a transient benzenoid diradical capable of generating DNA lesions.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Due to their indiscriminate cytotoxicity, enediynes require a targeting system to be successfully utilized for therapeutic purposes. This has been successfully exemplified by the clinical utilization of neocarzinostatin (NCS, <b>1</b>) as a poly(styrene-<i>co</i>-maleic acid) conjugate for the treatment of hepatocellular carcinoma and a hydrazide-derivative of calicheamicin (CAL-DMH, <b>2</b>) as antibody–drug conjugates (ADCs) targeting CD33 (gemtuzumab ozogamicin) and CD22 (inotuzumab ozogamicin) for the treatment of acute myeloid leukemia and acute lymphoblastic leukemia, respectively (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3−5)</a> The shared mechanism of action of the enediynes and the remarkable clinical success support the wisdom of evaluating additional enediyne scaffolds to identify promising new payloads for targeted chemotherapy.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/medium/jm0c00799_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of enediynes used clinically for targeted delivery and selected anthraquinone-fused enediynes: (A) neocarzinostatin chromophore (NCS, <b>1</b>) and a hydrazide-derivative of calicheamicin (CAL-DMH, <b>2</b>) used in the clinic for targeted delivery; (B) selected anthraquinone-fused enediynes tiancimycin A (<b>3</b>), uncialamycin (<b>4</b>) and amino-UCM (<b>5</b>), yangpumicin A (<b>6</b>), and dynemicin A (<b>7</b>), highlighting the varying substitutions on the A-ring. Active handles introduced by medicinal chemistry or total synthesis to facilitate antibody–drug conjugation, as exemplified by <b>2</b> or <b>5</b>, respectively, are shown in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00799&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The enediyne natural products are classified based on two structural features: the size of the macrocyclic enediyne cores (9- vs 10-membered) and the pendant peripheral moieties.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The 9-membered enediynes, such as NCS, are chromoproteins isolated with their cognate apoproteins stabilizing the labile 9-membered enediyne chromophores. The 10-membered enediynes are discrete small molecules with stable 10-membered enediyne cores and can be further divided into two subfamilies based on the peripheral moieties: the CAL-like and the anthraquinone- fused enediynes. The anthraquinone-fused subfamily consists of tiancimycin A (TNM A, <b>3</b>),<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> uncialamycin (UCM, <b>4</b>),<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> yangpumicin A (YPM A, <b>6</b>),<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> and dynemicin A (DYN A, <b>7</b>)<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). Currently, clinical use of the anthraquinone-fused enediynes has not been realized.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a></div><div class="NLM_p">Advancement of the anthraquinone-fused enediynes into the clinic has been hindered by their low titers from fermenting the native producers and their lack of functional groups to install appropriate linkers for preparing ADCs. DYN A and YPM A are produced in low titers, at 1.2–1.5 mg/L<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and 0.065 mg/L,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> respectively, and both are produced by members of the <i>Micromonospora</i> genus that are known to be recalcitrant to common genetic manipulations.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> Both UCM and TNM A are produced by members of the <i>Streptomyces</i> genus, which are known to be genetically amenable.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> However, submerged fermentation of wild-type <i>Streptomyces uncialis</i> for UCM production is yet to be realized, and UCM titers from fermenting <i>S. uncialis</i> on solid media remain extremely low at ∼0.019 mg/L;<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> wild-type <i>Streptomyces</i> sp. CB03234 produces TNM A at approximately 1–2 mg/L.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Furthermore, advancement of the anthraquinone-fused enediynes as payload candidates is limited by the inability to selectively functionalize the isolated natural products for antibody conjugation. Tremendous efforts have been devoted to overcome some of these challenges, as exemplified by the recent total synthesis of TNM A<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and several UCM analogues, including the designer amino-UCM (<b>5</b>) with a judiciously installed amino group at C-8 of the A ring (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B),<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> to enable regiospecific conjugation, thereby evaluating UCM as an ADC payload candidate in preclinical studies.<a onclick="showRef(event, 'ref11 ref12 ref18'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref18">(11,12,18)</a></div><div class="NLM_p">Significant advancements have been made to facilitate the reliable supply of anthraquinone-fused enediynes by microbial fermentation. Strain improvement of wild-type <i>S.</i> sp. CB03234 has afforded engineered strains, upon subsequent medium optimization, with improved TNM titers exceeding 20 mg/L.<a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19−21)</a> Comparative analysis of DYN, TNM, UCM, and YPM biosynthesis has enabled formulation of a unified biosynthetic pathway for the anthraquinone-fused enediynes, setting the stage to develop <i>S</i>. sp. CB03234 as a biotechnology platform for production of the natural enediynes and engineering of designer analogues (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Specifically, the Δ<i>tnmL</i> mutant strain <i>S</i>. sp. SB20020 accumulated TNM B (<b>8</b>) and TNM E (<b>9</b>), whose lack of hydroxyl groups at both C-6 and C-7 would support the predicted function of TnmL as a P450 hydroxylase.<a onclick="showRef(event, 'ref6 ref22'); return false;" href="javascript:void(0);" class="ref ref6 ref22">(6,22)</a> The Δ<i>tnmH</i> mutant strain <i>S</i>. sp. SB20002 afforded TNM F (<b>10</b>) and TNM C (<b>11</b>), both of which are characterized with a free hydroxyl group at C-7, a structural feature that would be consistent with the predicted function of TnmH as an <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (SAM)-dependent <i>O</i>-methyltransferase.<a onclick="showRef(event, 'ref6 ref22'); return false;" href="javascript:void(0);" class="ref ref6 ref22">(6,22)</a> Together with the isolation of TNM D (<b>13</b>), as well as shunt metabolites supporting the intermediacy of TNM G (<b>12</b>) in TNM biosynthesis, from the wild-type <i>S</i>. sp. CB03234 strain, these findings suggested substrate promiscuity of TnmH, TnmL, or both but fall short of defining their extact timing in the post-enediyne polyketide synthase (PKS) tailoring steps for TNM A biosynthesis as depicted in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> (path I vs path II).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/medium/jm0c00799_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Unified biosynthetic pathway for the anthraquinone-fused enediynes and the post-enediyne polyketide synthase tailoring steps for TNM A (<b>3</b>) biosynthesis as revealed by the isolation of TNM B (<b>8</b>) and TNM E (<b>9</b>), from the ΔtnmL mutant strain S. sp. SB20020, TNM F (<b>10</b>) and TNM C (<b>11</b>), from the ΔtnmH mutant strain <i>S</i>. sp. SB20002, and TNM D (<b>13</b>), as well as shunt metabolites supporting the intermediacy of TNM G (<b>12</b>), from wild-type <i>S</i>. sp. CB03234. Paths I and II highlight different timing of TnmL in installing the hydroxyl groups at C-6 and C-7 and of TnmH in installing the methyl group at the hydroxyl group of C-7, respectively, and in vitro characterization of TnmH in the current study supports path I.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00799&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">TnmH as a SAM-dependent methyltransferase presents a unique opportunity to functionalize the A-ring of TNMs (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figures S1 and S2</a>). Inspired by literature precedent that methyltransferases could accept S-alkylated SAM analogues containing reactive alkyl handles,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> we sought to characterize TnmH in vitro to determine its timing in the post-enediyne PKS tailoring steps for TNM A biosynthesis, investigate its substrate specificity and promiscuity, and explore its utility as a biocatalyst to functionalize the A-ring of the TNM scaffold for andibody–TNM conjugation. Here we report the biochemical and structural characterization of TnmH, revealing that (i) TnmH catalyzes regiospecific methylation at the C-7 hydroxyl group, with <b>11</b> as the preferred substrate to <b>10</b>, (ii) TnmH exhibits a broad substrate promiscuity toward both hydroxyanthraquinones and S-alkylated SAM analogues and catalyzes efficient installation of reactive alkyl handles regiospecificially at C-7 of the TNM scaffold, (iii) the X-ray crystal structure of TnmH, which is a homodimer in solution, provides the molecular basis to account for its broad substrate promiscuity, and (iv) TnmH as a biocatalyst enables the development of novel conjugation strategies to prepare antibody–TNM conjugates. These findings should greatly facilitate the construction and evaluation of antibody–TNM conjugates as next-generation ADCs for targeted cancer chemotherapy.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71201" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71201" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Construction of the Second Generation Δ<i>tnmH</i> Mutant Strain <i>S</i>. sp. SB20024 Enabling a Reliable Supply of TNM C (<b>11</b>)</h3><div class="NLM_p">The original Δ<i>tnmH</i> mutant strain <i>S</i>. sp. SB20002, from which <b>11</b> was first isolated (0.7 mg from 6 L) and characterized, was constructed in the wild-type <i>S</i>. sp. CB03234 strain.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Subsequent medium optimization improved TNM production, leading to the isolation of <b>3</b> (7.2 mg from 14 L), together with <b>13</b> (2.3 mg from 14 L) and shunt metabolites supporting the intermediacy of <b>12</b> in TNM A biosynthesis, from <i>S</i>. sp. CB03234<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and <b>11</b> (4.1 mg from 14 L), together with <b>10</b> (1.2 mg from 14 L), from <i>S</i>. sp. CB20002,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> respectively. While isolation of <b>10</b>, <b>11</b>, <b>13</b>, together with the proposed intermediacy of <b>12</b>, supported the predicted function of TnmH as a SAM-dependent <i>O</i>-methyltransferase (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figures S1 and S2</a>), it also raised the question for the timing of TnmH to catalyze methylation of the C-7 hydroxyl group in TNM A biosynthesis (i.e., path I vs path II, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> It was also noticed during medium optimization that both <i>S</i>. sp. CB03234 and <i>S</i>. sp. SB20002 co-produced a family of metabolites in high titers that interfered with the isolation of TNMs. We have since characterized this family of metabolites as tiancilactones and cloned and characterized its biosynthetic gene cluster (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S3</a>).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Progress has also been made in improving TNM production by subjecting the wild-type <i>S</i>. sp. CB03234 to multiple rounds of strain improvement. Most significantly, two variants, <i>S</i>. sp. CB03234-R, with titer of <b>3</b> exceeding 22 mg/L,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and <i>S</i>. sp. CB03234-S, with a combinted titer of <b>3</b> and <b>13</b> exceeding 33 mg/L,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> have been isolated. Fortuitously the <i>S</i>. sp. CB03234-S strain lost a large portion of its genome, resulting in loss of its ability to produce the tiancilactone family of metabolites as well as other unknown metabolites and affording a significantly cleaner background for TNM isolation (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/medium/jm0c00799_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Second generation Δ<i>tnmH</i> mutant strain <i>S</i>. sp. SB20024 provides a reliable supply of TNM C (<b>11</b>). HPLC analysis of (A) the original Δ<i>tnmH</i> mutant strain <i>S</i>. sp. SB20002 producing both TNM F (<b>10</b>) and TNM C (<b>11</b>) (at a combined isolated yield of ∼0.4 mg/L) co-produced with the tiancilactones and (B) the second generation Δ<i>tnmH</i> mutant strain <i>S</i>. sp. SB20024 producing TNM C (<b>11</b>) as the sole metabolite (at a titer of ∼4 mg/L) free from the tiancilactones. While UV–vis at 254 nm detects both the TNMs and tiancilactones, UV–vis at 539 nm detects the TNMs only. TNM F (<b>10</b>) (●); TNM C (<b>11</b>) (◆); tiancilactone A (▽); tiancilactone B (■).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00799&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The improved overall titer of TNMs, together with its simplified metabolite profile, motivated us to construct the second generation Δ<i>tnmH</i> mutant in the <i>S</i>. sp. CB03234-S strain to facilitate the production and isolation of <b>10</b> and <b>11</b>. We inactivated the <i>tnmH</i> gene in <i>S</i>. sp. CB03234-S by replacing it with a kanamycin resistance cassette, and gene replacement was carried out by following previously described methods (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Supporting Information Materials and Methods</a>).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The resultant second generation Δ<i>tnmH</i> mutant strain <i>S</i>. sp. SB20024 genotype was confirmed by PCR and Southern blot analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S4</a>). Under the optimized conditions for TMN production,<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a><i>S</i>. sp. SB20024 produces <b>11</b> at a titer of ∼4 mg/L, free from the inference of the tiancilactones, enabling a reliable supply of <b>11</b>. Intriguingly, in contrast to <i>S</i>. sp. SB20002, which produces both <b>10</b> and <b>11</b> in a 1:3.4 ratio,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a><i>S</i>. sp. SB20024 produces <b>11</b>, with little or undetectable amount of <b>10</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S3C</a>). This further facilitates the isolation and purification of <b>11</b> for in vitro characterization of TnmH (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) and the utilization of TnmH as a biocatalyst enabling the preparation of antibody–TNM conjugates (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/medium/jm0c00799_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. TnmH as a SAM-dependent <i>O</i>-methyltransferase catalyzes the regiospecific methylation of the hydroxyl group at C-7 of TNM F (<b>10</b>) and TNM C (<b>11</b>) in vitro. (A) HPLC analyses of TnmH-catalyzed reaction in the presence of SAM with TNM F (<b>10</b>) as a substrate: (I) an authentic standard of <b>10</b>; (II) complete reaction with boiled TnmH; (III) complete reaction at 28 <i>°</i>C for 20 min; (IV) an authentic standard of <b>12</b>. (B) HPLC analyses of TnmH-catalyzed reaction in the presence of SAM with TNM C (<b>11</b>) as a substrate: (I) an authentic standard of <b>11</b>; (II) complete reaction with boiled TnmH; (III) complete reaction at 28 <i>°</i>C for 5 min; (IV) an authentic standard of <b>13</b>. TNM F (<b>10</b>) (●); TNM G (<b>12</b>) (○); TNM C (<b>11</b>) (◆); TNM D (<b>13</b>) (◇).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00799&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. (A) TnmH-Catalyzed O-Methylation Exhibiting Broad Substrate Promiscuity toward Hydroxyanthraquinones<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a> and (B) TnmH-Catalyzed O-Alkylation of Hydroxyanthraquinone, As Exemplified by <b>11</b>, Exhibiting Broad Substrate Promiscuity toward S-Alkylated SAM Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/medium/jm0c00799_0007.gif" alt="" id="GRAPHIC-d7e844-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">hydroxyanthraquinone</th><th class="colsep0 rowsep0" align="center">cofactor</th><th class="colsep0 rowsep0" align="center">product</th><th class="colsep0 rowsep0" align="center">relative activity (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">SAM</td><td class="colsep0 rowsep0" align="left"><b>13</b><a class="ref internalNav" href="#t1fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">SAM</td><td class="colsep0 rowsep0" align="left"><b>19</b><a class="ref internalNav" href="#t1fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">5.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">SAM</td><td class="colsep0 rowsep0" align="left"><b>20</b><a class="ref internalNav" href="#t1fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">SAM</td><td class="colsep0 rowsep0" align="left"><b>21</b><a class="ref internalNav" href="#t1fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">SAM</td><td class="colsep0 rowsep0" align="left"><b>22</b><a class="ref internalNav" href="#t1fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">3.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">SAM</td><td class="colsep0 rowsep0" align="left"><b>23</b><a class="ref internalNav" href="#t1fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">SAM</td><td class="colsep0 rowsep0" align="left"><b>13</b><a class="ref internalNav" href="#t1fn2" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left"><b>27</b><a class="ref internalNav" href="#t1fn2" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left"><b>28</b><a class="ref internalNav" href="#t1fn2" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">6.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left"><b>29</b><a class="ref internalNav" href="#t1fn2" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">26</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup><sup>a</sup></sup><p class="last">The A-ring of <b>14–18</b> following the same numbering as <b>11</b>.</p></div><div class="footnote" id="t1fn1"><sup><sup>b</sup></sup><p class="last">Conditions: 5 μM TnmH, 100 μM hydroxyanthraquinone, 1.0 mM cofactor, pH 7.5, 15 min.</p></div><div class="footnote" id="t1fn2"><sup><sup>c</sup></sup><p class="last">Conditions: 1 μM TnmH, 250 μM hydroxyanthraquinone, 1.0 mM cofactor, pH 7.5, 30 min.</p></div></div></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/medium/jm0c00799_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystal structures of TnmH (PDB entries <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CLW">6CLW</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CLX">6CLX</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BBX">6BBX</a>). (A) Overall structure of TnmH homodimer in complex with SAM (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CLX">6CLX</a>). (B) Model of TNM C (<b>11</b>) docked into the X-ray structure of holo-TnmH in complex with SAM (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BBX">6BBX</a>), revealing a large active site pocket. SAM and the surface of <b>11</b>, shown in purple, and the active site residues His246 and Asp247 are labeled. (C) Close-up view of the <i>S</i>-methyl group of SAM seen in the holo-TnmH structure, with proximal residues labeled and colored by b-factor according to the color key shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00799&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/medium/jm0c00799_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Leveraging the TnmH-installed reactive alkyl handles at C-7 of TNMs to develop novel conjugation strategies to prepare antibody–TNM conjugates as exemplified by the synthesis of <b>27</b>, <b>28</b>, <b>29</b> from <b>11</b> and subsequent preparation of <b>31</b> from <b>27</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00799&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> In Vitro Characterization of TnmH as a SAM-Dependent <i>O</i>-Methyltransferase Catalyzing Regiospecific Methylation at the C-7 Hydroxyl Group, with TNM C (<b>11</b>) as the Preferred Substrate to TNM F (<b>10</b>)</h3><div class="NLM_p">To characterize TnmH in vitro, we cloned the <i>tnmH</i> gene from <i>S</i>. sp. CB03234 and expressed it heterologously in <i>E. coli</i> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Supporting Information Materials and Methods</a>). The overproduced TnmH protein was purified to homogeneity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S5</a>) and found to homodimerize in solution on the basis of size exclusion chromatography (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S6</a>). In the presence of SAM, TnmH catalyzed time-dependent conversion of <b>10</b> or <b>11</b> into a new product, respectively, and these products were absent in the negative controls using boiled TnmH (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). To establish their identity, the new products were subjected to high-resolution electrospray mass spectrometry (HR-ESI-MS) analysis, revealing their molecular weights to be 14 Da higher than the corresponding substrates, consistent with that of methylated products (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S7</a>). Since regiospecifc methylation at the C-7 hydroxyl group of <b>11</b> would afford <b>13</b>, which has been isolated and structurally characterized from the wild-type <i>S</i>. sp. CB03234 strain previously,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> TnmH-catalyzed O-methylation of the C-7 hydroxyl group of <b>11</b> was established by HPLC and HR-ESI-MS analysis by co-injection with an authentic standard of <b>13</b>. Regiospecific methylation of the C-7 hydroxyl group of <b>10</b> would afford <b>12</b>. Although the intermediacy of <b>12</b> in TNM biosynthesis has been proposed based on the shunt metabolites isolated from the wild-type <i>S</i>. sp. CB03234 strain,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a><b>12</b> has neither been isolated nor structurally characterized to date. Thus, we scaled up the reaction of TnmH-catalyzed methylation of <b>10</b>, in the presence of SAM, and prepared and isolated the resultant product for full structural characterization (<a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). The identify of the new product <b>12</b> was fully established by 1D and 2D NMR spectroscopic analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Table S4</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figures S21–S27</a>). On the basis of the NMR data, the only difference between <b>10</b> and <b>12</b> was the presence of a CH<sub>3</sub>O group (δ<sub>C</sub> 56.5, CH<sub>3</sub> and δ<sub>H</sub> 4.03, s) at C-7 (δ<sub>C</sub> 165.9, C). Specifically, the HMBC correlation between CH<sub>3</sub>O at C-7 (δ<sub>H</sub> 4.03) and C-7 (δ<sub>C</sub> 165.9) and the ROESY correlation between CH<sub>3</sub>O at C-7 (δ<sub>H</sub> 4.03) and H-8 (δ<sub>H</sub> 7.39) supported the assignment of a CH<sub>3</sub>O group at C-7 of <b>12</b>, unambiguously establishing TnmH-catalyzed regiospecific O-methylation of the C-7 hydroxyl group of <b>10</b> to yield <b>12</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><div class="NLM_p last">To establish the conditions for kinetic analysis of the TnmH-catalyzed O-methylation of <b>10</b> and <b>11</b>, we used alizarin (<b>14</b>) as a substrate surrogate to determine optimal pH and metal ion dependence (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S8</a>) and, following reaction optimization, carried out a time course to determine the linear range of conversion rate for TnmH-catalyzed O-methylation of <b>11</b> to <b>13</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S9</a>). The kinetic parameters for the TnmH-catalyzed O-methylation of <b>10</b> or <b>11</b> were first determined with saturating SAM (at 1 mM) and varying concentrations of <b>10</b> or <b>11</b> (from 7.8 to 500 μM) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Supporting Information Materials and Methods</a>). TnmH-catalyzed conversion of <b>10</b> and <b>11</b> to <b>12</b> and <b>13</b> followed Michaelis–Menten kinetics with a <i>K</i><sub>m</sub> value of 11.7 ± 1.2 μM and a <i>k</i><sub>cat</sub> value of 0.34 ± 0.076 min<sup>–1</sup> for <b>10</b> and a <i>K</i><sub>m</sub> value of 31.1 ± 4.1 μM and a <i>k</i><sub>cat</sub> value of 2.69 ± 0.09 min<sup>–1</sup> for <b>11</b>, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S10</a>). These experiments were repeated with saturating <b>11</b> (at 500 μM) and varying concentration of SAM (from 7.8 μM to 1.0 mM), and the results displayed similar Michaelis–Menten kinetics with a <i>K</i><sub>m</sub> value for SAM of 62 ± 4.8 μM and a <i>k</i><sub>cat</sub> value of 2.46 ± 0.1 min<sup>–1</sup> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S10</a>). The <i>k</i><sub>cat</sub> value determined with saturating SAM is in good agreement with that obtained with saturating <b>11</b>. Steady-state kinetics of TnmH revealed 3-fold higher catalytic efficiency (<i>k</i><sub>cat</sub>/<i>K</i><sub>m</sub>) for <b>11</b> over <b>10</b>, revealing <b>11</b> as the preferred substrate of TnmH. These results are consistent with <b>11</b> as the major metabolite accumulated in the Δ<i>tnmH</i> mutants (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), supporting that sequential hydroxylation of <b>9</b> and <b>10</b> by TnmL to afford <b>11</b> occurs prior to TnmH-catalyzed methylation of <b>11</b> to afford <b>13</b> in TNM biosynthesis (path I, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> TnmH Exhibits a Broad Substrate Promiscuity toward Both Hydroxyanthraquinones and S-Alkylated SAM Analogues</h3><div class="NLM_p">Substrate promiscuity, a hallmark feature of natural product biosynthetic pathways, has been extensively exploited to generate natural product structural diversity by combinatorial biosynthetic strategies.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Motivated by the fact that TnmH can catalyze efficient O-methylation of both <b>10</b> and <b>11</b>, we set out to investigate if TnmH could also accept other hydroxyanthraquinones as potential substrates (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Supporting Information Materials and Methods</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S11 and S12</a>). A panel of five additional hydroxyanthraquinones (<b>14</b>–<b>18</b>) were selected (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>A). Compounds <b>15</b> and <b>16</b>, featuring the same A-ring hydroxyl substitutions as <b>10</b> and <b>11</b>, respectively, are cycloaromatized products of <b>10</b> and <b>11</b> isolated previously from <i>S</i>. sp. SB20002.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Compounds <b>14</b>, <b>17</b>, and <b>18</b> are alizarin analogues mimicking the anthraquinone moiety of the anthraquinone-fused subfamily of enediynes with varying hydroxyl substitutions on the A-ring as <b>10</b> and <b>11</b>, as well as <b>6</b> and <b>7</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Under the identical condition used to assay <b>10</b> and <b>11</b> in the presence of SAM, TnmH recognized all five hydroxyanthraquinones as substrates and catalyzed efficient formation of the O-methylated products (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>A), the identities of which were confirmed by HPLC, co-injected with authentic standards, HR-ESI-MS analysis, or both (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S13</a>). Similar to <b>10</b> and <b>11</b>, TnmH regiospecifically methylated the C-7 hydroxyl group of <b>15</b>, exhibiting a higher relative activity for <b>15</b> compared to <b>16</b>. In contrast, while the same regiospecific methylation of the C-7 hydroxyl group of <b>14</b> was observed, TnmH displayed similar activities toward <b>14</b> and <b>17</b>. In fact, <b>18</b>, which features a C-6 hydroxy group, was also O-methylated with a lower relative activity in comparison to <b>14</b> and <b>17</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>A). These findings unveil the intrinsic substrate promiscuity of TnmH toward the hydroxyanthraquinone scaffold and will surely inspire future efforts to engineer new members of the anthraquinone-fused subfamily of enediynes by manipulating the post-enediyne PKS steps in their biosynthesis (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div><div class="NLM_p last">Inspired by the substrate promiscuity toward hydroxyanthraquinones, we decided to investigate if TnmH could use S-alkylated SAM analogues to regiospecifically alkylate the C-7 hydroxyl group, thereby installing reactive alkyl handles to the TNM scaffold. Methyltransferase-mediated exploitation of S-alkylated SAM analogues to install reactive alkyl groups for targeted natural product functionalization is well precedented in the literature.<a onclick="showRef(event, 'ref23 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref23 ref26 ref27">(23,26,27)</a> Three S-alkylated SAM analogues, i.e., <i>S</i>-propargyl (<b>24</b>), <i>S</i>-2-oxopropyl (<b>25</b>), and <i>S</i>-allyl (<b>26</b>), were selected, the choice of which was motivated by the potential of further functionalization of the resultant alkylated TNM products to develop novel conjugation chemistry<a onclick="showRef(event, 'ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30 ref31">(28−31)</a> (see <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">S20</a>). The three S-alkylated SAM analogues were synthesized according to literature procedures (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Supporting Information Materials and Methods</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S14</a>).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Under the identical conditions used to assay <b>11</b>, but substituting SAM with the diasteromeric preparations of <b>24</b>, <b>25</b>, or <b>26</b>, TnmH utilized all three S-alkylated SAM analogues and catalyzed regiospecific alkylation of <b>11</b> to afford the corresponding C-7 O-alkylated product <b>27</b>, <b>28</b>, or <b>29</b>, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>B), the identities of which were confirmed by HPLC, HR-ESI-MS, or 1D and 2D NMR spectroscopic analysis (<a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figures S15 and S28–S42</a>). As is well-known in the literature for other methyltransferases,<a onclick="showRef(event, 'ref23 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref23 ref33 ref34">(23,33,34)</a> TnmH also suffered from reduced activities toward the three <i>S</i>-alkyl-SAM analogues relative to SAM. Nonetheless, these findings validated our initial hypothesis and warrant further optimization of TnmH as a biocatalyst to enable the development of novel conjugation strategies toward antibody–TNM conjugates.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Crystal Structures of TnmH Reveal a Large Active Site Pocket and Provide a Molecular Basis for the Observed Substrate Promiscuity</h3><div class="NLM_p">Given the observed unusually broad substrate promiscuity, we set out to characterize the TnmH structure for a deeper understanding of the molecular determinants of substrate promiscuity toward both hydroxyanthraquinones and S-alkylated SAM analogues. TnmH crystals were obtained by using the hanging drop vapor-diffusion method.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> The X-ray crystal structure of TnmH (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CLW">6CLW</a>) was determined by the molecular replacement method,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> using the structure of the NcsB1 <i>O</i>-methyltransferase (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3I53">3I53</a>)<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> as a search model, and solved at a resolution of 2.74 Å. Crystals of TnmH in complex with SAM were obtained by soaking TnmH crystals with SAM. The holo-TnmH X-ray crystal structure in complex with SAM (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CLX">6CLX</a>) was determined using the X-ray crystal structure of TnmH as a search model and solved at a resolution of 2.73 Å. A summary of the crystallographic data and refinement statistics is given in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Table S5</a>. To generate the ternary structure model of TnmH in complex with both SAM and <b>11</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>), the 3D model of <b>11</b> was extracted from the TnmS3 and <b>11</b> complex structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BBX">6BBX</a>) we solved previously,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> docked into the holo-TnmH structure in complex with SAM, and energetically minimized (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Supporting Information Materials and Methods</a>).</div><div class="NLM_p">Consistent with the oligomeric state in solution, as determined by gel filtration chromatography (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S6</a>), X-ray crystallography revealed that TnmH is homodimeric and displays an overall architecture similar to the large family of SAM-dependent methyltransferases (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Each monomeric subunit is made up of two domains totaling 20 α-helices and nine β-sheets. The dimeric interface primary consists of hydrophobic residues, and the C-terminal domain exhibits a Rossman-like fold conserved in structural homologues,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> which comprises most of the SAM-binding site. While the methyl group of SAM is relatively solvent accessible, the nearest residues impacting SAM binding include Asp247 (3.5 Å), Arg153 (3.4 Å), and Met151 (3.1 Å) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B,C). Representation of residues proximal to SAM binding by their flexibility (b-factor) revealed flexibility among residues in the active site, most notably Arg153 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C), which may contribute to the observed substrtate promiscuity toward the S-alkylated SAM analogues.</div><div class="NLM_p last">The active site of TnmH reveals a large cavity capable of accommodating diverse substrates. A docking model of the ternary complex generated using the TnmH-SAM holo structure revealed the distance separating the donor methyl group of SAM and the C-7 and C-6 hydroxyl groups of <b>11</b> to be 2.7 and 3.8 Å, respectively, accounting for the observed regiospecificity (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). Moreover, the distal side of the molecule (consisting of the enediyne core and additional side chain at C-26) is bound in a large, solvent- accessible cavity, providing a rationale for the observed methylation of simple hydroxyanthraquinones and cycloaromatized enediyne congeners that differ substantially in their structural features. This large cavity, with numerous flexible residues at the interface of <b>11</b> and SAM, may enable rotations for productive methylation at the C-6 hydroxyl group when simplified substrates that lack the C-7 hydroxyl group, such as <b>18</b>, are presented. Catalytic roles for residues in the active site may include activation of the phenol for nucleophilic attack via general acid–base catalysis, as implicated previously for phenolic <i>O</i>-methyltransferases.<a onclick="showRef(event, 'ref37 ref40'); return false;" href="javascript:void(0);" class="ref ref37 ref40">(37,40)</a> In TnmH, two residues in the proximity of the donor methyl group of SAM may play this role; His246 and Asp247 may function to activate the C-7 hydroxyl group of <b>11</b>, as well as other hydroxyanthraquinones, for nucleophilic attack and/or participate in proton shuttling, as shown by null mutagenesis of each residue independently demonstrating their contribution to catalysis, and abolished activity seen in the double mutant (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S16</a>).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> TnmH as a Biocatalyst Enables the Development of Novel Conjugation Strategies To Prepare Antibody–TNM Conjugates</h3><div class="NLM_p">The broad substrate promiscuity observed for TnmH led us to explore the utility of TnmH as a biocatalyst to install reactive alkyl handles to the C-7 hydroxyl group of <b>11</b> and develop novel strategies to prepare antibody–TNM conjugates. In an attempt to better accommodate the S-alkylated SAM analogues, four TnmH mutants, Asn243Ala, Asn243Gly, Val244Ala, and Arg153Ala, were constructed to expand the SAM binding site (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Supporting Information Materials and Methods</a>, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). The β-carbons of Asn243 and Val244 and the η<sup>2</sup>-nitrogen of Arg153 were found at distances of 4.6, 5.4, and 3.4 Å, respectively, from the donor methyl group of SAM and are hypothesized to sterically clash with or disrupt binding of S-alkylated SAM analogue <b>24</b> or <b>25</b>. The four TnmH mutants were similarly overproduced in <i>E. coli</i>, purified to homogeneity, and subjected to an extra step of activated charcoal chromatography to remove any endogeneously bound SAM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S5D</a>). When assayed under the identical conditions as wild-type TnmH, using <b>11</b> and <b>24</b> or <b>25</b> as substrates, the four TnmH mutants were all catalytically competent, none of which, however, gained any apparent enhancement of <b>28</b> or <b>29</b> formation relative to wild-type TnmH (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S17</a>). Moreover, the ratio of turnover of <b>24</b> verse <b>25</b> remained relatively constant for all assayed mutants, suggesting there is no preference for either cosubstrates with the engineered mutants. Wild-type TnmH was therefore used as the preferred biocatalyst to further optimize the enzymatic reaction to obtain both <b>28</b> and <b>29</b>.</div><div class="NLM_p">Exploitation of methyltransferases as biocatalysts is often complicated by <i>S</i>-adenosylhomocysteine (SAH), the co-product of SAM-dependent methyltransferase that is known to exhibit product inhibition, and this is particularly problematic when using S-alkylated SAM analogues as substrates due to their decreased activity relative to SAM.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> To overcome this problem, we cloned the <i>mtnN</i> gene from <i>E. coli</i>, which encodes a SAH nucleosidase,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and expressed it in <i>E. coli</i>; the overproduced SAH nucleosidase was purified to homogeneity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Supporting Information Materials and Methods</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S5B</a>). SAH nucleosidase is known to alleviate the inhibitory effect of SAH on methyltransferases by degrading SAH into <i>S</i>-ribosylhomocysteine and adenine.<a onclick="showRef(event, 'ref41 ref43'); return false;" href="javascript:void(0);" class="ref ref41 ref43">(41,43)</a> Under the optimized conditions and in the presence of the recombinant SAH nucleosidase, TnmH-catalyzed reactions of <b>11</b> with <b>24</b> or <b>25</b>, on 10 mg scale, afforded <b>27</b> or <b>28</b>, respectively, with 50–80% yields (<a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div><div class="NLM_p">Next, we subjected <b>27</b> and <b>28</b>, together with <b>12</b> and <b>13</b>, in comparison with <b>10</b> and <b>11</b>, as well as <b>3</b>, to determine the effect of the varying alkylations of the C-7 hydroxyl group on their cytotoxicity (<a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S18</a>). As noted previously,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> methylation at the C-7 hydroxyl group enhanced cytotoxic activity across all cell lines examined. This was seen with the increased potency of <b>12</b> and <b>13</b> relative to <b>10</b> and <b>11</b>, and <b>27</b> and <b>28</b> exhibited comparable or enhanced potency relative to <b>13</b>, with <b>28</b> being as potent as <b>3</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The observed relative cytotoxicity for each of these TNM analogues correlated well with their DNA cleavage activity as seen with the plasmid relaxation assays (<a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S19</a>). Taken together, these findings support the wisdom of exploiting the reactive alkyl handles installed at C-7 by TnmH to develop novel conjugation strategies to prepare antibody–TNM conjugates.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Cytotoxicity (EC<sub>50</sub> in nM) of <b>27</b>, <b>28</b>, and Other TNM Analogues with Varying C-7 Substitutions (<b>10</b>, <b>11</b>, <b>12</b>, <b>13</b>), in Comparison with TNM A (<b>3</b>), against Four Selected Human Cancer Cell Lines</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center">SF-295</th><th class="colsep0 rowsep0" align="center">SKMEL-5</th><th class="colsep0 rowsep0" align="center">MDA-MB-231</th><th class="colsep0 rowsep0" align="center">NCI-H226</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">0.45 ± 0.14</td><td class="colsep0 rowsep0" align="left">0.74 ± 0.2</td><td class="colsep0 rowsep0" align="left">1.1 ± 0.3</td><td class="colsep0 rowsep0" align="left">4.2 ± 1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">51 ± 3.0</td><td class="colsep0 rowsep0" align="left">36 ± 16</td><td class="colsep0 rowsep0" align="left">37 ± 3.0</td><td class="colsep0 rowsep0" align="left">75 ± 9.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">45 ± 12</td><td class="colsep0 rowsep0" align="left">32 ± 13</td><td class="colsep0 rowsep0" align="left">34 ± 6.0</td><td class="colsep0 rowsep0" align="left">66 ± 14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">21 ± 14</td><td class="colsep0 rowsep0" align="left">19 ± 6.5</td><td class="colsep0 rowsep0" align="left">14 ± 1.0</td><td class="colsep0 rowsep0" align="left">68 ± 10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">4.0 ± 1.6</td><td class="colsep0 rowsep0" align="left">2.8 ± 0.6</td><td class="colsep0 rowsep0" align="left">5.0 ± 0.3</td><td class="colsep0 rowsep0" align="left">32 ± 6.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">2.8 ± 0.7</td><td class="colsep0 rowsep0" align="left">2.8 ± 0.7</td><td class="colsep0 rowsep0" align="left">3.7 ± 2.1</td><td class="colsep0 rowsep0" align="left">26 ± 4.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">1.1 ± 0.4</td><td class="colsep0 rowsep0" align="left">0.8 ± 0.3</td><td class="colsep0 rowsep0" align="left">0.9 ± 0.5</td><td class="colsep0 rowsep0" align="left">3.7 ± 1.0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last"><i>N</i> = 2.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last"><i>N</i> = 3.</p></div></div></div><div class="NLM_p last">Finally, as a proof-of-concept, we assessed the reactivity of <b>27</b> under conditions for copper-catalyzed azide–alkyne cycloaddition (CuAAC) to demonstrate the functional utility of the installed propargyl handle in preparing antibody–TNM conjugates. Terminal alkynes are common functional handles for chemoselective bioconjugation due to their reactivity with organic azides in the presence of a Cu(I) catalyst,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> but the compatibility of this reaction with enediynes (sensitive to reducing and acidic conditions) was previously unknown. We subjected <b>27</b> to conventional CuAAC conditions with a heterobifunctional linker <b>30</b> we have reported previously that contained an azide and β-lactam functionality used for site-specific conjugation to uniquely reactive lysine residues of engineered antibodies (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Supporting Information Materials and Methods</a>).<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> The cycloaddition product <b>31</b> was readily observed by HPLC analysis, and upon reaction optimization, we reliably prepared <b>30</b> with a >80% yield based on recovered substrate 27 (<a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). The identity of <b>31</b> was fully characterized by HR-ESI-MS and 1D and 2D NMR spectroscopic analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figures S43–S51</a>). The efficient synthesis of <b>31</b> demonstrates the utility of the reactive alkyl handles installed at C-7 by TnmH in enabling downstream bioconjugation. Similarly, the 2-oxopropyl handle of <b>28</b> could be functionalized with a hydrazine as has been exemplified by the CAL-based ADCs.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Allyl-group-enabled bioconjugation, via tetrazine ligation<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> or photoclick chemistry,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> has also been demonstrated and should be applicable to 29 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S20</a>).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02221" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02221" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The anthraquinone-fused subfamily of enediynes are promising anticancer drug candidates for targeted delivery. Advancement of the anthraquinone-fused enediynes into the clinic, however, has been hindered by their low titers from fermenting the native producers and lack of functional groups that enable ADC preparation. We have now constructed the second generation Δ<i>tnmH</i> mutant strain <i>S</i>. sp. SB20024 in the TNM overproducing <i>S</i>. sp. CB03234-S strain, which reliably produces <b>11</b> with a titer of ∼4 mg/L (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). In vitro biochemical characterization of TnmH as a SAM-dependent methyltransferase not only confirmed its precise role in TNM biosynthesis, catalyzing regiospecific O-methylation at the C-7 hydroxyl group of the TNM scaffold with <b>11</b> as the preferred substrate (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>), but also unveiled its broad substrate promiscuity toward both other hydroxyanthraquinones and S-alkylated SAM analogues (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Structural characterization of TnmH provided a molecular basis to account for the observed broad substrate promiscuity (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Significantly, we demonstrated the feasibility of installing active alkyl handles to the C-7 hydroxyl group of the TNM scaffold by exploiting TnmH’s ability to utilize selected S-alkylated SAM analogues. The resultant TNM analogues retain full cytotoxicity, supporting the C-7 hydroxyl group as an excellent site for functionalization and antibody–TNM conjugation (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Finally, we showcased TnmH as a biocatalyst enabling the development of novel conjugation strategies by attaching a β-lactam linker using CuAAC chemistry to the C-7 O-propargylated TNM (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). Taken together, these findings set the stage to prepare antibody–TNM conjugates and should greatly facilitate the advancement of the anthraquinone-fused subfamily of enediynes as next-generation ADCs for targeted chemotherapy.</div><div class="NLM_p last">While the current study highlighted TnmH-catalyzed regiospecific installation of reactive alkyl groups, thereby functionalizing the TNM scaffold to enable antibody conjugation, this strategy should be readily applicable to the other members of the anthraquinone-fused subfamily of enediynes on the basis of their unified biosynthetic pathway and the demonstrated feasibility to engineer designer analogues in <i>S</i>. sp. CB03234 and its engineered overproducers as platform strains by combinatorial biosynthesis strategies (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). It should also inspire exploitation of other enzymes of the TNM biosynthetic pathway or enzymes of any other natural product biosynthetic pathways in general as biocatalysts to overcome difficulties in functionalizing natural products, further enriching the toolbox<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> for antibody–drug conjugation.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54887" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54887" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> General Experimental Procedures</h3><div class="NLM_p last">All reactions containing water- or air-sensitive reagents were performed using flame-dried glassware under nitrogen or argon. Tetrahydrofuran and dichloromethane were passed through two columns of neutral alumina. Compounds <b>3</b>, <b>8</b>–<b>11</b>, <b>15</b>, and <b>16</b> were isolated following known protocols.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> Compounds <b>19</b>, <b>22</b>–<b>26</b>, and <b>30</b> were prepared according to literature procedures.<a onclick="showRef(event, 'ref32 ref46 ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref32 ref46 ref48 ref49">(32,46,48,49)</a> All other chemicals were purchased from commercial sources and used without further purification. All <sup>1</sup>H, <sup>13</sup>C, and 2D NMR (<sup>1</sup>H–<sup>1</sup>H COSY, <sup>1</sup>H–<sup>13</sup>C HSQC, <sup>1</sup>H–<sup>13</sup>C HMBC, <sup>1</sup>H–<sup>1</sup>H ROESY) experiments were run on a Bruker Avance III Ultrashield 700 with QCI cryoprobe at 700 MHz for <sup>1</sup>H and 175 MHz for <sup>13</sup>C nuclei, a Bruker Avance NEO 600 with TCI cryoprobe at 600 MHz for <sup>1</sup>H and 150 MHz for <sup>13</sup>C nuclei, or a Bruker Avance 400 at 400 MHz for <sup>1</sup>H nuclei and 100 MHz for <sup>13</sup>C nuclei. MPLC separation was conducted on a Biotage Isolera One equipped with a Biotage SNAP cartridge KP-C18-HS column (30 g). Semipreparative HPLC was performed on a Varian liquid chromatography system equipped with a YMC-pack ODS-A column (250 mm × 10 mm, 5 μm), unless otherwise specified. LC–MS was performed on an Agilent 1260 Infinity LC coupled to a 6230 TOF (HRESI) equipped with an Agilent Poroshell 120 EC-C18 column (50 mm × 4.6 mm, 2.7 μm). Optical rotations were obtained using an AUTOPOL IV automatic polarimeter (Rudolph Research Analytical). UV was measured with a NanoDrop 2000C spectrophotometer (Thermo Scientific). All compounds tested were >95% pure.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Enzymatic Synthesis and Structural Elucidation of <b>12</b>, <b>27</b>, and <b>28</b></h3><div class="NLM_p last">For enzymatic synthesis of <b>12</b>, <b>27</b>, or <b>28</b>, a typical 20 mL batch of the reaction mixture contained 100 μM TnmH, 200 μM SAH nucleosidase, 2.5 mM SAM or SAM analog (<b>24</b> or <b>25</b>), and 500 μM <b>10</b> or <b>11</b>, respectively, in 100 mM Na<sub>2</sub>HPO<sub>4</sub> buffer (pH 7.5) with a final DMSO concentration of 5%. The reactions were incubated at 28 °C overnight with occasional shaking and terminated by the addition of methanol to a final concentration of 50%. The resulting mixtures were centrifuged to pellet the precipitated proteins, and the pellets were washed twice with MeOH (5 mL). The supernatants and MeOH washes were combined and concentrated to afford the residues that were subsequently dissolved in MeOH and subjected to HPLC purification to afford pure products of <b>12</b>, <b>27</b>, and <b>28</b>. Preparative HPLC was performed on an Agilent 1260 Prep Infinity LC with a MWD detector equipped with an Agilent Eclipse Plus phenyl-hexyl column (250 mm × 9.4 mm, 5 μm). Reaction products using <b>10</b> and SAM were purified using CH<sub>3</sub>OH/H<sub>2</sub>O (65/35) as the mobile phase to afford <b>12</b> (2.0 mg). Reaction products containing using <b>11</b> and <b>24</b> or <b>25</b> were purified using CH<sub>3</sub>OH/H<sub>2</sub>O (65/35) to afford <b>27</b> (7.2 mg) and <b>28</b> (11.0 mg), respectively.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> TNM G (<b>12</b>)</h3><div class="NLM_p last">Isolated as a purple film; [α]<sub>D</sub><sup>25</sup> +1587 (<i>c</i> = 0.00001, CH<sub>3</sub>OH); UV (CH<sub>3</sub>OH) λ<sub>max</sub> (log ε) 209 (0.69), 215 (0.93), 252 (0.73), 270 (0.82), 285 (0.79), 300 (1.1), 374 (0.34), 539 (1.4) nm; <sup>1</sup>H and <sup>13</sup>C NMR data, see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Table S4</a>; HR-ESI-MS (positive ion) affording the [M + H]<sup>+</sup> ion at <i>m</i>/<i>z</i> 558.1369 (calcd [M + H]<sup>+</sup> ion for molecular formula C<sub>30</sub>H<sub>23</sub>NO<sub>10</sub> at <i>m</i>/<i>z</i> 558.1395).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Propargyl-TNM C (<b>27</b>)</h3><div class="NLM_p last">Isolated as a purple film; [α]<sub>D</sub><sup>25</sup> +1977 (<i>c</i> = 0.00001, CH<sub>3</sub>OH); UV (CH<sub>3</sub>OH) λ<sub>max</sub> (log ε) 209 (1.2), 219 (1.2), 242 (1.6), 260 (1.5), 417 (0.21), 544 (0.79), 576 (0.68) nm; <sup>1</sup>H and <sup>13</sup>C NMR data, see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Table S4</a>; HR-ESI-MS (positive ion) affording the [M + H]<sup>+</sup> ion at <i>m</i>/<i>z</i> 598.1338 (calcd [M + H]<sup>+</sup> ion for molecular formula C<sub>32</sub>H<sub>23</sub>NO<sub>11</sub> at <i>m</i>/<i>z</i> 598.1344).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Keto-TNM C (<b>28</b>)</h3><div class="NLM_p last">Isolated as a purple film; [α]<sub>D</sub><sup>25</sup> +3,333 (<i>c</i> = 0.00001, CH<sub>3</sub>OH); UV (CH<sub>3</sub>OH) λ<sub>max</sub> (log ε) 209 (1.3), 215 (1.2), 245 (2.0), 260 (1.9), 408 (0.20),544 (1.0), 578 (0.83) nm; <sup>1</sup>H and <sup>13</sup>C NMR data, see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Table S4</a>; HR-ESI-MS (positive ion) affording the [M + H]<sup>+</sup> ion at <i>m</i>/<i>z</i> 616.1461 (calcd [M + H]<sup>+</sup> ion for molecular formula C<sub>32</sub>H<sub>25</sub>NO<sub>12</sub> at <i>m</i>/<i>z</i> 616.1450).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Functionalization of Propargyl-TNM C (<b>27</b>)</h3><div class="NLM_p last">As a proof-of-concept for further functionalization and downstream bioconjugation, propargyl-TNM C (<b>27</b>) was subjected to copper-catalyzed azide–alkyne cycloaddition (CuAAC)<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> to attach a β-lactam functional group (<b>30</b>) that can serve as an attachment handle for antibodies.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> β-Lactam TNM C (<b>31</b>)</h3><div class="NLM_p last">To a solution of <b>30</b> (4.3 mg, 8.0 μmol) and <b>27</b> (4.0 mg, 6.7 μmol) in THF (250 μL) were added tris(3-hydroxypropyltriazolylmethyl)amine (THPTA, 0.73 μg, 1.7 μmol), copper sulfate (0.42 μg,1.7 μmol), and sodium ascorbate (0.66 μg, 3.4 μmol). The reaction was stirred at room temperature for 16 h and purified by reverse phase HPLC (10–80% CH<sub>3</sub>CN/H<sub>2</sub>O) resulting in a purple film (3.6 mg, 83% BRSM). [α]<sub>D</sub><sup>25</sup> +348.0 (<i>c</i> = 0.00008, CH<sub>3</sub>CN); UV (CH<sub>3</sub>CN) λ<sub>max</sub> (log ε) 205 (0.36), 215 (0.10), 225 (0.27), 243 (0.23), 277 (0.23), 547 (0.41), 585 (0.36) nm; <sup>1</sup>H and <sup>13</sup>C NMR data, see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Table S6</a>; HR-ESI-MS (positive ion) affording the [M + H]<sup>+</sup> ion at <i>m</i>/<i>z</i> 1132.3778 (calcd [M + H]<sup>+</sup> ion for molecular formula C<sub>32</sub>H<sub>23</sub>NO<sub>11</sub> at <i>m</i>/<i>z</i> 1132.3782).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Cell Viability Assay</h3><div class="NLM_p last">Compounds <b>3</b>, <b>10</b>−<b>13</b>, <b>27</b>, and <b>28</b> were tested for cell viability against four human cancer cell lines including melanoma (SK-MEL-5), breast (MDA-MB-231), central nervous system (SF-295), and non-small-cell lung cancer (NCI-H226) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S18</a>). Suspended cultures of cells were diluted to a concentration of 5 × 10<sup>4</sup> cells per mL in either RPMI 1640 or DMEM medium supplemented with 10% fetal bovine serum, 100 μg per mL of streptomycin, and 100 unit per mL of penicillin. The suspended cultures were dispensed into 96-well plates (100 μL per well), and the plates were incubated for 24 h at 37 °C in an atmosphere of 5% CO<sub>2</sub>, 95% air, and 100% humidity. The original medium was then removed, and 100 μL of fresh medium was added, followed by addition of serial dilutions of drugs (1 μL in DMSO with final concentrations ranging from 0 to 1000 nM). Plates were incubated under the above conditions for 72 h. Finally, 20 μL of CellTiter 96 AQueous One Solution Reagent (Promega) was added to the plates and incubation continued at 37 °C in a humidified, 5% CO<sub>2</sub> atmosphere for 30–60 min. The absorbance at 490 nm was recorded using a SpectraMax M5 plate reader. Each point represents the mean ± SD of three replicates, and the EC<sub>50</sub> was determined by computerized curve fitting using GraphPad Prism. The same software was used to determine statistical significance using an extra sum-of-squares <i>F</i>-test. All new compounds in this study (<b>12</b>, <b>27</b>, and <b>28</b>) were assayed three times independently against all cell lines examined.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Plasmid Relaxation Assay</h3><div class="NLM_p last">Compounds <b>3</b>, <b>10</b>–<b>13</b>, <b>27</b>, <b>28</b> were assayed for their efficacy to relax supercoiled plasmid DNA by generating single- and double-strand DNA breaks, as previously described.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Briefly, assays were performed in 20 μL (total volume) of Dulbecco’s phosphate buffered saline (pH 7.4) with 5% DMSO, containing 0.8 μg (25 nM plasmid, 120 μM base pair) of pUC19 (New England Biolabs), and varying concentrations of compounds. The assays were performed in triplicate in the presence and absence of 1 mM reduced glutathione. The reactions were incubated at 37 °C for 24 h, after which 5 μL of loading dye was added and samples were analyzed by electrophoresis on a 1% agarose gel containing SYBR Green I nucleic acid gel stain (Thermo Fisher). Gel electrophoresis was carried out in TAE buffer at 100 V for 1 h. The relative activity was determined by quantifying the intensity of bands corresponding to supercoiled plasmid (form I, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S19B</a>) normalized to DMSO-treated controls. Band intensity was quantified using ImageJ software.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The quantified band intensities were plotted using GraphPad Prism (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">Figure S19C</a>).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00799" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31758" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31758" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00799?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00799</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Addition materials and methods; strains, plasmids, and primers used in this study (Tables S1–S3); summary of NMR data for <b>12</b>, <b>27</b>, and <b>28</b> (Table S4); crystal data and structural refinement for apo-TnmH and TnmH-SAM complex (Table S5); summary of NMR data for <b>31</b> (Table S6); anthraquinone-fused enediyne BGCs (Figure S1); sequence alignment for TnmH and selected phenolic <i>O</i>-methyltransferases (Figure S2); <i>tnl</i> BGC and encoded metabolites (Figure S3); construction and confirmation of Δ<i>tnmH</i> mutant strain <i>S</i>. sp. SB20024 (Figure S4); SDS–PAGE analysis of recombinant proteins TnmH, SAH nucleosidase, and TnmH mutants (Figure S5); size exclusion chromatography of TnmH (Figure S6); HR-ESI-MS analysis for <b>12</b> and <b>13</b> (Figure S7); pH and metal-ion dependence for TnmH O-methylation of <b>14</b> (Figure S8); time course of TnmH O-methylation of <b>11</b> (Figure S9); steady state kinetics for TnmH-catalyzed O-methylation of <b>10</b> and <b>11</b> (Figure S10); TnmH-catalyzed O-methylation of hydroxyanthraquinones <b>14</b>, <b>17</b>, and <b>18</b> (Figure S11); TnmH-catalyzed O-methylation of cycloaromatized enediyne congeners <b>15</b> and <b>16</b> (Figure S12); HR-ESI-MS analysis of O-methylated hydroxyanthraquinone products <b>19</b>–<b>23</b> (Figure S13); HR-ESI-MS analysis of the S-alkylated SAM analogues <b>24</b>–<b>26</b> (Figure S14); TnmH catalyzed preparation of <b>29</b> (Figure S15); end point assays of active site mutants of TnmH with <b>11</b> (Figure S16); end point assays of TnmH mutants with expanded SAM binding site with S-alkylated SAM analogues <b>24</b> and <b>25</b> (Figure S17); cell viability curves and DNA plasmid relaxation assays of selected TNMs (Figures S18 and S19); functionalization strategies for conjugation of <b>27</b>, <b>28</b>, <b>29</b> (Figure S20); HR-ESI-MS analysis, HPLC analysis for purity, and NMR spectra of 12, 27, 28, 31 (Figure S21–S51); HPLC analysis of purity of <b>3</b>, <b>10</b>, <b>11</b>, and <b>13</b> (Figures S52–S55) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings with associated data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf">jm0c00799_si_001.pdf (4.86 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_002.csv">jm0c00799_si_002.csv (2.41 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00799" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81277" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81277" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ben Shen</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, 
    Department of Molecular Medicine, 
    Natural Products Discovery Center
at Scripps Research, 
    , , , and , The Scripps Research Institute, Jupiter, Florida 33458, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9750-5982" title="Orcid link">http://orcid.org/0000-0002-9750-5982</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b0c3d8d5ded2f0c3d3c2d9c0c0c39ed5d4c5"><span class="__cf_email__" data-cfemail="74071c111a16340717061d0404075a111001">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ajeeth Adhikari</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, 
    Department of Immunology and Microbiology, 
    , , , and , The Scripps Research Institute, Jupiter, Florida 33458, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christiana N. Teijaro</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, , , and , The Scripps Research Institute, Jupiter, Florida 33458, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaohui Yan</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, , , and , The Scripps Research Institute, Jupiter, Florida 33458, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7295-8902" title="Orcid link">http://orcid.org/0000-0001-7295-8902</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chin-Yuan Chang</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, , , and , The Scripps Research Institute, Jupiter, Florida 33458, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chun Gui</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, , , and , The Scripps Research Institute, Jupiter, Florida 33458, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu-Chen Liu</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, , , and , The Scripps Research Institute, Jupiter, Florida 33458, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7232-144X" title="Orcid link">http://orcid.org/0000-0002-7232-144X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ivana Crnovcic</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, , , and , The Scripps Research Institute, Jupiter, Florida 33458, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dong Yang</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, , , and , The Scripps Research Institute, Jupiter, Florida 33458, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2917-0663" title="Orcid link">http://orcid.org/0000-0003-2917-0663</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thibault Annaval</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, , , and , The Scripps Research Institute, Jupiter, Florida 33458, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christoph Rader</span> - <span class="hlFld-Affiliation affiliation">Department of Immunology and Microbiology, , , and , The Scripps Research Institute, Jupiter, Florida 33458, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9955-3454" title="Orcid link">http://orcid.org/0000-0001-9955-3454</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>A.A. and C.N.T. contributed equally.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was supported in part by National Institutes of Health Grants GM115575 (B.S.) and CA204484 (B.S. and C.R.). C.N.T. is a recipient of a NIH Postdoctoral Fellowship GM128345. I.C. was supported in part by postdoctoral fellowships from the German Research Foundation. This is manuscript no. 29979 from The Scripps Research Institute.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17728" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17728" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Y. Duan, Xiangya International Academy of Translational Medicine, Central South University, Changsha, China, for the <i>Streptomyces</i> sp. CB03234 and CB03234-S strains that produce TNM A and TNM D, and Dr. HaeJeung Park of the X-ray Crystallography Core Facility and Dr. Xiangming Kong of the NMR Core Facility at The Scripps Research Institute, Jupiter, Florida, for assistance with X-ray and NMR analysis.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ADC</td><td class="NLM_def"><p class="first last">antibody–drug conjugate</p></td></tr><tr><td class="NLM_term">BGC</td><td class="NLM_def"><p class="first last">biosynthetic gene cluster</p></td></tr><tr><td class="NLM_term">CAL</td><td class="NLM_def"><p class="first last">calicheamicin</p></td></tr><tr><td class="NLM_term">CuAAC</td><td class="NLM_def"><p class="first last">copper-catalyzed azide–alkyne cycloaddition</p></td></tr><tr><td class="NLM_term">DYN</td><td class="NLM_def"><p class="first last">dynemicin</p></td></tr><tr><td class="NLM_term">HR-ESI-MS</td><td class="NLM_def"><p class="first last">high-resolution electrospray ionization mass spectrometry</p></td></tr><tr><td class="NLM_term">NCS</td><td class="NLM_def"><p class="first last">neocarzinostatin</p></td></tr><tr><td class="NLM_term">PKS</td><td class="NLM_def"><p class="first last">polyketide synthase</p></td></tr><tr><td class="NLM_term">SAH</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosylhomocysteine</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term">TNM</td><td class="NLM_def"><p class="first last">tiancimycin</p></td></tr><tr><td class="NLM_term">UCM</td><td class="NLM_def"><p class="first last">uncialamycin</p></td></tr><tr><td class="NLM_term">YPM</td><td class="NLM_def"><p class="first last">yangpumicin</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70611" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70611" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 51 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, A.</span>; <span class="NLM_string-name">Teijaro, C. N.</span>; <span class="NLM_string-name">Townsend, C. A.</span>; <span class="NLM_string-name">Shen, B.</span></span> <span> </span><span class="NLM_article-title">Biosynthesis of Enediyne Natural Products</span>. In  <i>Comprehensive Natural Products III</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. W.</span>, <span class="NLM_string-name">Begley, T.</span></span>, Eds.; <span class="NLM_publisher-name">Elsevier</span>, <span class="NLM_year">2020</span>; Vol.  <span class="NLM_volume">3</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-409547-2.14669-4</span> , in press.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2FB978-0-12-409547-2.14669-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=A.+Adhikari&author=C.+N.+Teijaro&author=C.+A.+Townsend&author=B.+Shenauthor=H.+W.+Liu&author=T.+Begley&title=Comprehensive+Natural+Products+III"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-409547-2.14669-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-409547-2.14669-4%26sid%3Dliteratum%253Aachs%26aulast%3DAdhikari%26aufirst%3DA.%26atitle%3DBiosynthesis%2520of%2520Enediyne%2520Natural%2520Products%26btitle%3DComprehensive%2520Natural%2520Products%2520III%26aulast%3DLiu%26aufirst%3DH.%2BW.%26pub%3DElsevier%26date%3D2020%26volume%3D3%26doi%3D10.1016%2FB978-0-12-409547-2.14669-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W. M.</span></span> <span> </span><span class="NLM_article-title">Chemistry and biology of the enediyne anticancer antibiotics</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1387</span>– <span class="NLM_lpage">1416</span>, <span class="refDoi"> DOI: 10.1002/anie.199113873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fanie.199113873" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1991&pages=1387-1416&author=K.+C.+Nicolaouauthor=W.+M.+Dai&title=Chemistry+and+biology+of+the+enediyne+anticancer+antibiotics&doi=10.1002%2Fanie.199113873"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fanie.199113873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.199113873%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26aulast%3DDai%26aufirst%3DW.%2BM.%26atitle%3DChemistry%2520and%2520biology%2520of%2520the%2520enediyne%2520anticancer%2520antibiotics%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D1991%26volume%3D30%26spage%3D1387%26epage%3D1416%26doi%3D10.1002%2Fanie.199113873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, H.</span></span> <span> </span><span class="NLM_article-title">SMANCS and polymer-conjugated macromolecular drugs: advantages in cancer chemotherapy</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(00)00134-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2FS0169-409X%2800%2900134-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=11259839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=169-185&author=H.+Maeda&title=SMANCS+and+polymer-conjugated+macromolecular+drugs%3A+advantages+in+cancer+chemotherapy&doi=10.1016%2FS0169-409X%2800%2900134-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">SMANCS and polymer-conjugated macromolecular drugs: advantages in cancer chemotherapy</span></div><div class="casAuthors">Maeda, H.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">169-185</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">This review, with 66 refs., discusses the development and therapeutic potential of prototype macromol. drugs for use in cancer chemotherapy, in particular the development and use of SMANCS, a conjugate of neocarzinostatin (NCS) and poly(styrene-co-maleic acid).  The various topics covered include a brief description of the chem. and polymer conjugation, the binding of the conjugate to albumin and the biol. behavior in vitro and in vivo after arterial injection in animals, including plasma half-life, and the lipid soly. of SMANCS in medium chain triglycerides and Lipiodol, a lipid contrast medium suitable for use in x-ray-computed tomog.  The biol. response-modifying effects and the tumor-targeting mechanism of SMANCS and other macromol. drugs are also discussed.  The latter mechanism is accounted for in terms of a tumor 'enhanced permeability and retention' (or EPR) effect.  A principal advantage in the use of SMANCS or other macromol. drugs is the potential for a redn. or elimination of toxicity.  Macromol. drugs such as a pyran copolymer-NCS conjugate show a marked redn. in bone marrow toxicity normally assocd. with the use of NCS.  This is believed to be due to a hypothetical blood-bone marrow 'barrier' which, relative to NCS, restricts or limits access of the macromol. drug to the bone marrow.  In addn., the clin. possibilities for SMANCS are discussed, including the suggestion that angiotensin II-induced hypertension has clin. potential in improving the selective delivery of macromol. drugs (i.e. SMANCS) to tumors.  Aq. SMANCS formulations have been tested in pilot studies in patients with solid tumors of the ovary, esophagus, lung, stomach, adrenal gland and in the brain.  Formulations based on SMANCS/Lipiodol have been shown to be effective both as a diagnostic tool and for therapeutic use in solid tumors where the formulations are given arterially via a catheter.  In a pilot study in primary unresectable hepatoma, an objective redn. in tumor size was obsd. for about 90% of cases when an adequate amt. of the macromol. drug was administered.  A patient receiving such treatment with no active liver cirrhosis and tumor nodules/lesion confined within one liver segment might expect to have a 90% chance of survival after treatment for at least 5 yr.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqodKBCU3HvbVg90H21EOLACvtfcHk0lhhYlpMucGc4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhsL0%253D&md5=c0cf1d27fa22dc32aa56d258b681206e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900134-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900134-4%26sid%3Dliteratum%253Aachs%26aulast%3DMaeda%26aufirst%3DH.%26atitle%3DSMANCS%2520and%2520polymer-conjugated%2520macromolecular%2520drugs%253A%2520advantages%2520in%2520cancer%2520chemotherapy%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D169%26epage%3D185%26doi%3D10.1016%2FS0169-409X%2800%2900134-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinman, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollander, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beyer, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holcomb, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallett, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsou, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upeslacis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shochat, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mountain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flowers, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, I.</span></span> <span> </span><span class="NLM_article-title">Gemtuzumab ozogamicin, a potent and selective anti-CD33 antibody-calicheamicin conjugate for treatment of acute myeloid leukemia</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1021/bc010021y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bc010021y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD3MXptFanurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2002&pages=47-58&author=P.+R.+Hamannauthor=L.+M.+Hinmanauthor=I.+Hollanderauthor=C.+F.+Beyerauthor=D.+Lindhauthor=R.+Holcombauthor=W.+Hallettauthor=H.+R.+Tsouauthor=J.+Upeslacisauthor=D.+Shochatauthor=A.+Mountainauthor=D.+A.+Flowersauthor=I.+Bernstein&title=Gemtuzumab+ozogamicin%2C+a+potent+and+selective+anti-CD33+antibody-calicheamicin+conjugate+for+treatment+of+acute+myeloid+leukemia&doi=10.1021%2Fbc010021y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Gemtuzumab Ozogamicin, A Potent and Selective Anti-CD33 Antibody-Calicheamicin Conjugate for Treatment of Acute Myeloid Leukemia</span></div><div class="casAuthors">Hamann, Philip R.; Hinman, Lois M.; Hollander, Irwin; Beyer, Carl F.; Lindh, Delores; Holcomb, Ryan; Hallett, William; Tsou, Hwei-Ru; Upeslacis, Janis; Shochat, Dan; Mountain, Andrew; Flowers, David A.; Bernstein, Irwin</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-58</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">CD33 is expressed by acute myeloid leukemia (AML) cells in >80% of patients but not by normal hematopoietic stem cells, suggesting that elimination of CD33+ cells may be therapeutically beneficial.  A conjugate of a calicheamicin hydrazide deriv. attached via hydrazone formation to the oxidized carbohydrates of the anti-CD33 murine antibody P67.6 had been chosen for use in AML prior to humanization of this antibody.  However, the CDR-grafted humanized P67.6 could not be used to make the carbohydrate conjugate because of the unexpected sensitivity of this antibody to periodate oxidn.  Exploration of a series of bifunctional linkers resulted in a new class of calicheamicin conjugates, termed the hybrid conjugates, that allows for the attachment of the calicheamicin to lysines but incorporates the site of hydrolytic release, a hydrazone, previously shown to be required for activity.  The optimized conjugate chosen for clin. trials, gemtuzumab ozogamicin ("gem-ozo", Mylotarg, formerly designated CMA-676), was significantly more potent and selective than the carbohydrate conjugate it replaced.  It was selectively cytotoxic to HL-60 leukemia cells in tissue culture with an IC50 in the low to sub-pg cal/mL range (cal = calicheamicin equiv.).  Doses of gem-ozo as low as 50 μg cal/kg given three times to mice bearing HL-60 xenografts routinely resulted in long-term, tumor-free survivors, while a nonbinding control conjugate was relatively inactive.  Gem-ozo at a concn. of 2 to 10 ng cal/mL selectively inhibited leukemia colony formation by marrow cells from a significant proportion of AML patients.  Gem-ozo has also shown significant activity against AML in Phase II trials and is the first antibody-targeted chemotherapeutic agent approved by the FDA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUzPmqKMIOx7Vg90H21EOLACvtfcHk0lhhYlpMucGc4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXptFanurg%253D&md5=56f273f4975fa6ebfa81e28f23111e8a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fbc010021y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbc010021y%26sid%3Dliteratum%253Aachs%26aulast%3DHamann%26aufirst%3DP.%2BR.%26aulast%3DHinman%26aufirst%3DL.%2BM.%26aulast%3DHollander%26aufirst%3DI.%26aulast%3DBeyer%26aufirst%3DC.%2BF.%26aulast%3DLindh%26aufirst%3DD.%26aulast%3DHolcomb%26aufirst%3DR.%26aulast%3DHallett%26aufirst%3DW.%26aulast%3DTsou%26aufirst%3DH.%2BR.%26aulast%3DUpeslacis%26aufirst%3DJ.%26aulast%3DShochat%26aufirst%3DD.%26aulast%3DMountain%26aufirst%3DA.%26aulast%3DFlowers%26aufirst%3DD.%2BA.%26aulast%3DBernstein%26aufirst%3DI.%26atitle%3DGemtuzumab%2520ozogamicin%252C%2520a%2520potent%2520and%2520selective%2520anti-CD33%2520antibody-calicheamicin%2520conjugate%2520for%2520treatment%2520of%2520acute%2520myeloid%2520leukemia%26jtitle%3DBioconjugate%2520Chem.%26date%3D2002%26volume%3D13%26spage%3D47%26epage%3D58%26doi%3D10.1021%2Fbc010021y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, Y. N.</span></span> <span> </span><span class="NLM_article-title">Inotuzumab ozogamicin: first global approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">1603</span>– <span class="NLM_lpage">1610</span>, <span class="refDoi"> DOI: 10.1007/s40265-017-0802-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1007%2Fs40265-017-0802-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=28819740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlGktL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=1603-1610&author=Y.+N.+Lamb&title=Inotuzumab+ozogamicin%3A+first+global+approval&doi=10.1007%2Fs40265-017-0802-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inotuzumab Ozogamicin: First Global Approval</span></div><div class="casAuthors">Lamb, Yvette N.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1603-1610</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">I.v. inotuzumab ozogamicin (Besponsa; Pfizer) is an anti-CD22 monoclonal antibody-calicheamicin conjugate that binds to CD22-expressing tumor cells.  Upon binding, the complex is internalized and the cytotoxic calicheamicin deriv. is released inside the cell, inducing double-strand DNA breakage and subsequent cell death.  In June 2017, the EMA granted inotuzumab ozogamicin approval as monotherapy for the treatment of adults with relapsed or refractory CD22-pos. B-cell precursor acute lymphoblastic leukemia (ALL).  The use of inotuzumab ozogamicin in adult patients with Philadelphia chromosome-pos., relapsed or refractory CD22-pos. B-cell precursor ALL is restricted to those who have failed treatment with at least one tyrosine kinase inhibitor.  Inotuzumab ozogamicin was granted priority review for the treatment of relapsed or refractory B-cell precursor ALL by the US FDA in Feb. 2017.  In the USA, a phase III trial evaluating inotuzumab ozogamicin in combination with frontline chemotherapy in adults with newly diagnosed B-cell ALL has recently been initiated and inotuzumab ozogamicin is under phase II evaluation in childhood CD22-pos. B-cell ALL.  Inotuzumab ozogamicin combination therapies are also being evaluated in the phase I/II or II setting in ALL and chronic myeloid leukemia and in the phase I setting in Burkitt's lymphoma.  This article summarizes the milestones in the development of inotuzumab ozogamicin leading to this first approval for ALL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSHnoFMpPaqbVg90H21EOLACvtfcHk0lhWeTo_pWGFTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlGktL3P&md5=76c6dbf41181e769901bbc313aabdb20</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1007%2Fs40265-017-0802-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-017-0802-5%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DY.%2BN.%26atitle%3DInotuzumab%2520ozogamicin%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2017%26volume%3D77%26spage%3D1603%26epage%3D1610%26doi%3D10.1007%2Fs40265-017-0802-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hindra</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crnovcic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gansemans, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Nieuwerburgh, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rader, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span> <span> </span><span class="NLM_article-title">Strain prioritization and genome mining for enediyne natural products</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e02104-16</span> <span class="refDoi"> DOI: 10.1128/mBio.02104-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1128%2FmBio.02104-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=27999165" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&author=X.+Yanauthor=H.+Geauthor=T.+Huangauthor=+Hindraauthor=D.+Yangauthor=Q.+Tengauthor=I.+Crnovcicauthor=X.+Liauthor=J.+D.+Rudolfauthor=J.+R.+Lohmanauthor=Y.+Gansemansauthor=X.+Zhuauthor=Y.+Huangauthor=L.+X.+Zhaoauthor=Y.+Jiangauthor=F.+Van+Nieuwerburghauthor=C.+Raderauthor=Y.+Duanauthor=B.+Shen&title=Strain+prioritization+and+genome+mining+for+enediyne+natural+products&doi=10.1128%2FmBio.02104-16"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1128%2FmBio.02104-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmBio.02104-16%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DX.%26aulast%3DGe%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DT.%26aulast%3DHindra%26aulast%3DYang%26aufirst%3DD.%26aulast%3DTeng%26aufirst%3DQ.%26aulast%3DCrnovcic%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DRudolf%26aufirst%3DJ.%2BD.%26aulast%3DLohman%26aufirst%3DJ.%2BR.%26aulast%3DGansemans%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DL.%2BX.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DVan%2BNieuwerburgh%26aufirst%3DF.%26aulast%3DRader%26aufirst%3DC.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DB.%26atitle%3DStrain%2520prioritization%2520and%2520genome%2520mining%2520for%2520enediyne%2520natural%2520products%26jtitle%3DmBio%26date%3D2016%26volume%3D7%26doi%3D10.1128%2FmBio.02104-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warabi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, R. J.</span></span> <span> </span><span class="NLM_article-title">Uncialamycin, a new enediyne antibiotic</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">5233</span>– <span class="NLM_lpage">5236</span>, <span class="refDoi"> DOI: 10.1021/ol052081f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol052081f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtV2nu73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=5233-5236&author=J.+Daviesauthor=H.+Wangauthor=T.+Taylorauthor=K.+Warabiauthor=X.+H.+Huangauthor=R.+J.+Andersen&title=Uncialamycin%2C+a+new+enediyne+antibiotic&doi=10.1021%2Fol052081f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Uncialamycin, a new enediyne antibiotic</span></div><div class="casAuthors">Davies, Julian; Wang, Hao; Taylor, Terry; Warabi, Kaoru; Huang, Xin-Hui; Andersen, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5233-5236</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lab. cultures of an undescribed streptomycete obtained from the surface of a British Columbia lichen produce uncialamycin (I), a new enediyne antibiotic.  The structure of I was elucidated by anal. of spectroscopic data.  I exhibits potent in vitro antibacterial activity against Gram-pos. and Gram-neg. human pathogens, including Burkholderia cepacia, a major cause of morbidity and mortality in patients with cystic fibrosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGcJkghP1-j7Vg90H21EOLACvtfcHk0lhWeTo_pWGFTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtV2nu73E&md5=97531c57ebc3393de4b82cd871368a3f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fol052081f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol052081f%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DTaylor%26aufirst%3DT.%26aulast%3DWarabi%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DX.%2BH.%26aulast%3DAndersen%26aufirst%3DR.%2BJ.%26atitle%3DUncialamycin%252C%2520a%2520new%2520enediyne%2520antibiotic%26jtitle%3DOrg.%2520Lett.%26date%3D2005%26volume%3D7%26spage%3D5233%26epage%3D5236%26doi%3D10.1021%2Fol052081f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crnovcic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rader, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span> <span> </span><span class="NLM_article-title">Genome mining of <i>Micromonospora yangpuensis</i> DSM 45577 as a producer of an anthraquinone-fused enediyne</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">6192</span>– <span class="NLM_lpage">6195</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.7b03120</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.7b03120" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslemu7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=6192-6195&author=X.+Yanauthor=J.+J.+Chenauthor=A.+Adhikariauthor=D.+Yangauthor=I.+Crnovcicauthor=N.+Wangauthor=C.+Y.+Changauthor=C.+Raderauthor=B.+Shen&title=Genome+mining+of+Micromonospora+yangpuensis+DSM+45577+as+a+producer+of+an+anthraquinone-fused+enediyne&doi=10.1021%2Facs.orglett.7b03120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Genome Mining of Micromonospora yangpuensis DSM 45577 as a Producer of an Anthraquinone-Fused Enediyne</span></div><div class="casAuthors">Yan, Xiaohui; Chen, Jian-Jun; Adhikari, Ajeeth; Yang, Dong; Crnovcic, Ivana; Wang, Nan; Chang, Chin-Yuan; Rader, Christoph; Shen, Ben</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6192-6195</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new anthraquinone-fused enediyne, yangpumicin A (YPM A, 1), along with four Bergman cyclization congeners (YPM B-E, 2-5), was isolated from Micromonospora yangpuensis DSM 45577 after mining enediyne biosynthetic gene clusters from public actinobacterial genome databases and prioritizing the hits by an enediyne genome neighborhood network anal. for discovery.  YPM A is potent against a broad spectrum of human cancer cell lines.  The discovery of 1 provides new opportunities for the functionalization of enediynes to develop new conjugation chemistries for antibody-drug conjugates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOQFuX9dkpvLVg90H21EOLACvtfcHk0liW1RByMqNnWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslemu7%252FE&md5=0f2dacc312aeb2d50370f816b5f201d2</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.7b03120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.7b03120%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%2BJ.%26aulast%3DAdhikari%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DCrnovcic%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DChang%26aufirst%3DC.%2BY.%26aulast%3DRader%26aufirst%3DC.%26aulast%3DShen%26aufirst%3DB.%26atitle%3DGenome%2520mining%2520of%2520Micromonospora%2520yangpuensis%2520DSM%252045577%2520as%2520a%2520producer%2520of%2520an%2520anthraquinone-fused%2520enediyne%26jtitle%3DOrg.%2520Lett.%26date%3D2017%26volume%3D19%26spage%3D6192%26epage%3D6195%26doi%3D10.1021%2Facs.orglett.7b03120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span> <span> </span><span class="NLM_article-title">Yangpumicins F and G, enediyne congeners from <i>Micromonospora yangpuensis</i> DSM 45577</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">2483</span>– <span class="NLM_lpage">2488</span>, <span class="refDoi"> DOI: 10.1021/acs.jnatprod.9b00229</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.9b00229" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslWmsbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2019&pages=2483-2488&author=Z.+Wangauthor=Z.+Wenauthor=L.+Liuauthor=X.+Zhuauthor=B.+Shenauthor=X.+Yanauthor=Y.+Duanauthor=Y.+Huang&title=Yangpumicins+F+and+G%2C+enediyne+congeners+from+Micromonospora+yangpuensis+DSM+45577&doi=10.1021%2Facs.jnatprod.9b00229"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Yangpumicins F and G, enediyne congeners from Micromonospora yangpuensis DSM 45577</span></div><div class="casAuthors">Wang, Zilong; Wen, Zhongqing; Liu, Ling; Zhu, Xiangcheng; Shen, Ben; Yan, Xiaohui; Duan, Yanwen; Huang, Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2483-2488</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">Enediyne natural products are among the most cytotoxic small mols. and thus excellent payload candidates for the development of antibody-drug conjugates (ADCs).  Here we report the isolation and structural elucidation of two new 10-membered anthraquinone-fused enediynes, yangpumicins (YPM) F (6) and G (7), together with five known congeners, YPM A-E (1-5), from Micromonospora yangpuensis DSM 45577.  YPM F (6) and G (7) showed strong cytotoxicity against the tested human cancer cell lines, as well as activity against several Gram-pos. and Gram-neg. pathogens.  The 1,2-diols in 6 and 7 promise to enable new linker chem. for the development of YPM-based ADCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0bBiQ9l9e27Vg90H21EOLACvtfcHk0liW1RByMqNnWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslWmsbzN&md5=9a0cbcb8ad1411f84c26e09ca21dd121</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.9b00229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.9b00229%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWen%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DY.%26atitle%3DYangpumicins%2520F%2520and%2520G%252C%2520enediyne%2520congeners%2520from%2520Micromonospora%2520yangpuensis%2520DSM%252045577%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2019%26volume%3D82%26spage%3D2483%26epage%3D2488%26doi%3D10.1021%2Facs.jnatprod.9b00229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konishi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohkuma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanDuyne, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clardy, J.</span></span> <span> </span><span class="NLM_article-title">Dynemicin A, a novel antibiotic with the anthraquinone and 1,5-diyn-3-ene subunit</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1449</span>– <span class="NLM_lpage">1452</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.42.1449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.7164%2Fantibiotics.42.1449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=2793600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADyaK3cXjtlSgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1989&pages=1449-1452&author=M.+Konishiauthor=H.+Ohkumaauthor=K.+Matsumotoauthor=T.+Tsunoauthor=H.+Kameiauthor=T.+Miyakiauthor=T.+Okiauthor=H.+Kawaguchiauthor=G.+D.+VanDuyneauthor=J.+Clardy&title=Dynemicin+A%2C+a+novel+antibiotic+with+the+anthraquinone+and+1%2C5-diyn-3-ene+subunit&doi=10.7164%2Fantibiotics.42.1449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Dynemicin A, a novel antibiotic with the anthraquinone and 1,5-diyn-3-ene subunit</span></div><div class="casAuthors">Konishi, Masataka; Ohkuma, Hiroaki; Matsumoto, Kiyoshi; Tsuno, Takashi; Kamei, Hideo; Miyaki, Takeo; Oki, Toshikazu; Kawaguchi, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1449-52</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Dynemicin A (I) was isolated from the culture broth of Micromonospora species and its structure was detd.  It was esp. active against gram-pos. bacteria and prolonged the life span of mice inoculated with P388 leukemia.  Treating I with acetic acid anhydride yielded a triacetyl deriv. of I which was 2-5-fold as potent as I against all microbes tested.  In addn. both compds. displayed significant activity against P388 leukemia and 316 melanoma in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6sU283EH0HbVg90H21EOLACvtfcHk0liW1RByMqNnWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXjtlSgsw%253D%253D&md5=48845123f3f9273cb43fb87bb3f55604</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.42.1449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.42.1449%26sid%3Dliteratum%253Aachs%26aulast%3DKonishi%26aufirst%3DM.%26aulast%3DOhkuma%26aufirst%3DH.%26aulast%3DMatsumoto%26aufirst%3DK.%26aulast%3DTsuno%26aufirst%3DT.%26aulast%3DKamei%26aufirst%3DH.%26aulast%3DMiyaki%26aufirst%3DT.%26aulast%3DOki%26aufirst%3DT.%26aulast%3DKawaguchi%26aufirst%3DH.%26aulast%3DVanDuyne%26aufirst%3DG.%2BD.%26aulast%3DClardy%26aufirst%3DJ.%26atitle%3DDynemicin%2520A%252C%2520a%2520novel%2520antibiotic%2520with%2520the%2520anthraquinone%2520and%25201%252C5-diyn-3-ene%2520subunit%26jtitle%3DJ.%2520Antibiot.%26date%3D1989%26volume%3D42%26spage%3D1449%26epage%3D1452%26doi%3D10.7164%2Fantibiotics.42.1449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chowdari, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langley, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaprakasam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sufi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derwin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passmore, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rangan, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardarelli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vite, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangwar, S.</span></span> <span> </span><span class="NLM_article-title">Uncialamycin as a novel payload for antibody drug conjugate (ADC) based targeted cancer therapy</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">466</span>– <span class="NLM_lpage">470</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.12.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2Fj.bmcl.2018.12.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=30579797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFygu7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=466-470&author=N.+S.+Chowdariauthor=C.+Panauthor=C.+Raoauthor=D.+R.+Langleyauthor=P.+Sivaprakasamauthor=B.+Sufiauthor=D.+Derwinauthor=Y.+C.+Wangauthor=E.+Kwokauthor=D.+Passmoreauthor=V.+S.+Ranganauthor=S.+Deshpandeauthor=P.+Cardarelliauthor=G.+Viteauthor=S.+Gangwar&title=Uncialamycin+as+a+novel+payload+for+antibody+drug+conjugate+%28ADC%29+based+targeted+cancer+therapy&doi=10.1016%2Fj.bmcl.2018.12.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Uncialamycin as a novel payload for antibody drug conjugate (ADC) based targeted cancer therapy</span></div><div class="casAuthors">Chowdari, Naidu S.; Pan, Chin; Rao, Chetana; Langley, David R.; Sivaprakasam, Prasanna; Sufi, Bilal; Derwin, Daniel; Wang, Yichong; Kwok, Eilene; Passmore, David; Rangan, Vangipuram S.; Deshpande, Shrikant; Cardarelli, Pina; Vite, Gregory; Gangwar, Sanjeev</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">466-470</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Uncialamycin analogs were evaluated as potential cytotoxic agents in an antibody-drug conjugate (ADC) approach to treating human cancer.  These analogs were synthesized using Hauser annulations of substituted phthalides as a key step.  A highly potent uncialamycin analog 3c with a valine-citrulline dipeptide linker was conjugated to an anti-mesothelin monoclonal antibody (mAb) through lysines to generate a meso-13 conjugate.  This conjugate demonstrated subnanomolar potency (IC50 = 0.88 nM, H226 cell line) in in vitro cytotoxicity expts. with good immunol. specificity to mesothelin-pos. lung cancer cell lines.  The potency and mechanism of action of this uncialamycin class of enediyne antitumor antibiotics make them attractive payloads in ADC-based cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqBfRZz2Cfk7Vg90H21EOLACvtfcHk0ljggiLoTsSGHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFygu7rJ&md5=371237eb33fc1f2dd6230a8649d75233</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.12.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.12.021%26sid%3Dliteratum%253Aachs%26aulast%3DChowdari%26aufirst%3DN.%2BS.%26aulast%3DPan%26aufirst%3DC.%26aulast%3DRao%26aufirst%3DC.%26aulast%3DLangley%26aufirst%3DD.%2BR.%26aulast%3DSivaprakasam%26aufirst%3DP.%26aulast%3DSufi%26aufirst%3DB.%26aulast%3DDerwin%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%2BC.%26aulast%3DKwok%26aufirst%3DE.%26aulast%3DPassmore%26aufirst%3DD.%26aulast%3DRangan%26aufirst%3DV.%2BS.%26aulast%3DDeshpande%26aufirst%3DS.%26aulast%3DCardarelli%26aufirst%3DP.%26aulast%3DVite%26aufirst%3DG.%26aulast%3DGangwar%26aufirst%3DS.%26atitle%3DUncialamycin%2520as%2520a%2520novel%2520payload%2520for%2520antibody%2520drug%2520conjugate%2520%2528ADC%2529%2520based%2520targeted%2520cancer%2520therapy%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D466%26epage%3D470%26doi%3D10.1016%2Fj.bmcl.2018.12.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poudel, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotapati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thevanayagam, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardarelli, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chowdari, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaspady, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samikannu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuppusamy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saravanakumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arunachalam, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rangan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rampulla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathur, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vite, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangwar, S.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of phenol-linked uncialamycin antibody-drug conjugates</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">126782</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.126782</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2Fj.bmcl.2019.126782" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=31767265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1eks73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=126782&author=Y.+B.+Poudelauthor=C.+Raoauthor=S.+Kotapatiauthor=M.+Deshpandeauthor=L.+Thevanayagamauthor=C.+Panauthor=J.+Cardarelliauthor=N.+Chowdariauthor=M.+Kaspadyauthor=R.+Samikannuauthor=P.+Kuppusamyauthor=P.+Saravanakumarauthor=P.+N.+Arunachalamauthor=S.+Deshpandeauthor=V.+Ranganauthor=R.+Rampullaauthor=A.+Mathurauthor=G.+D.+Viteauthor=S.+Gangwar&title=Design%2C+synthesis+and+biological+evaluation+of+phenol-linked+uncialamycin+antibody-drug+conjugates&doi=10.1016%2Fj.bmcl.2019.126782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of phenol-linked uncialamycin antibody-drug conjugates</span></div><div class="casAuthors">Poudel, Yam B.; Rao, Chetana; Kotapati, Srikanth; Deshpande, Madhura; Thevanayagam, Lourdes; Pan, Chin; Cardarelli, Josephine; Chowdari, Naidu; Kaspady, Mahammed; Samikannu, Ramesh; Kuppusamy, Prakasam; Saravanakumar, Pon; Arunachalam, Pirama N.; Deshpande, Shrikant; Rangan, Vangipuram; Rampulla, Richard; Mathur, Arvind; Vite, Gregory D.; Gangwar, Sanjeev</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">126782</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Uncialamycin is one of the structurally simpler and newer members of enediyne family of natural products.  It exhibits highly potent activity against several types of bacteria and cancer cells.  Described herein is a strategy for the targeted delivery of this cytotoxic agent to tumors using an antibody-drug conjugate (ADC) approach.  Central to the design of ADC were the generation of potent and chem. stable uncialamycin analogs and attachment of protease cleavable linkers to newly realized phenolic handles to prep. linker-payloads.  Conjugation of the linker-payloads to tumor targeting antibody, in vitro activity and in vivo evaluation are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8pthCP5tbf7Vg90H21EOLACvtfcHk0ljggiLoTsSGHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1eks73N&md5=5c112ff7f2ec60c67c8f095df6d3fbca</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126782%26sid%3Dliteratum%253Aachs%26aulast%3DPoudel%26aufirst%3DY.%2BB.%26aulast%3DRao%26aufirst%3DC.%26aulast%3DKotapati%26aufirst%3DS.%26aulast%3DDeshpande%26aufirst%3DM.%26aulast%3DThevanayagam%26aufirst%3DL.%26aulast%3DPan%26aufirst%3DC.%26aulast%3DCardarelli%26aufirst%3DJ.%26aulast%3DChowdari%26aufirst%3DN.%26aulast%3DKaspady%26aufirst%3DM.%26aulast%3DSamikannu%26aufirst%3DR.%26aulast%3DKuppusamy%26aufirst%3DP.%26aulast%3DSaravanakumar%26aufirst%3DP.%26aulast%3DArunachalam%26aufirst%3DP.%2BN.%26aulast%3DDeshpande%26aufirst%3DS.%26aulast%3DRangan%26aufirst%3DV.%26aulast%3DRampulla%26aufirst%3DR.%26aulast%3DMathur%26aufirst%3DA.%26aulast%3DVite%26aufirst%3DG.%2BD.%26aulast%3DGangwar%26aufirst%3DS.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520phenol-linked%2520uncialamycin%2520antibody-drug%2520conjugates%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D126782%26doi%3D10.1016%2Fj.bmcl.2019.126782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lam, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Titus, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dabrah, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimball, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veitch, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustavson, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Compton, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forenza, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beutler, J.</span></span> <span> </span><span class="NLM_article-title">Improved processes for the production and isolation of dynemicin A and large-scale fermentation in a 10000-liter fermentor</span>. <i>J. Ind. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.1007/BF01583725</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1007%2FBF01583725" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1369016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADyaK3sXhs1Onu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1992&pages=7-12&author=K.+S.+Lamauthor=J.+A.+Titusauthor=T.+T.+Dabrahauthor=D.+L.+Kimballauthor=J.+M.+Veitchauthor=D.+R.+Gustavsonauthor=B.+J.+Comptonauthor=J.+A.+Matsonauthor=S.+Forenzaauthor=J.+Rossauthor=D.+Millerauthor=J.+Roachauthor=J.+Beutler&title=Improved+processes+for+the+production+and+isolation+of+dynemicin+A+and+large-scale+fermentation+in+a+10000-liter+fermentor&doi=10.1007%2FBF01583725"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Improved processes for the production and isolation of dynemicin A and large-scale fermentation in a 10000-liter fermentor</span></div><div class="casAuthors">Lam, Kin Sing; Titus, Jeffrey A.; Dabrah, Thomas T.; Kimball, Dennis L.; Veitch, Jacqueline M.; Gustavson, Donald R.; Compton, Bruce J.; Matson, James A.; Forenza, Salvatore; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Industrial Microbiology</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-12</span>CODEN:
                <span class="NLM_cas:coden">JIMIE7</span>;
        ISSN:<span class="NLM_cas:issn">0169-4146</span>.
    </div><div class="casAbstract">Supplementing the culture of Micromonospora chersina sp. nov. No. M956-1 with NaI (0.5 mg/L) enhanced the prodn. of dynemicin A by 35-fold in shake flask culture.  Homogeneous dynemicin A was obtained from the whole broth ext. by Dicalite chromatog., Sephadex LH-20 chromatog. and vacuum liq. chromatog.  Gram quantities of dynemicin A were obtained from the fermn. of M. chersina sp. nov. No. M956-1 in a 10,000-L fermentor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLTP3GkoqE-bVg90H21EOLACvtfcHk0lhXSbgZcCLXQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXhs1Onu7s%253D&md5=5bbc80dca0bb50e6daacc971009cc3fc</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2FBF01583725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01583725%26sid%3Dliteratum%253Aachs%26aulast%3DLam%26aufirst%3DK.%2BS.%26aulast%3DTitus%26aufirst%3DJ.%2BA.%26aulast%3DDabrah%26aufirst%3DT.%2BT.%26aulast%3DKimball%26aufirst%3DD.%2BL.%26aulast%3DVeitch%26aufirst%3DJ.%2BM.%26aulast%3DGustavson%26aufirst%3DD.%2BR.%26aulast%3DCompton%26aufirst%3DB.%2BJ.%26aulast%3DMatson%26aufirst%3DJ.%2BA.%26aulast%3DForenza%26aufirst%3DS.%26aulast%3DRoss%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DD.%26aulast%3DRoach%26aufirst%3DJ.%26aulast%3DBeutler%26aufirst%3DJ.%26atitle%3DImproved%2520processes%2520for%2520the%2520production%2520and%2520isolation%2520of%2520dynemicin%2520A%2520and%2520large-scale%2520fermentation%2520in%2520a%252010000-liter%2520fermentor%26jtitle%3DJ.%2520Ind.%2520Microbiol.%26date%3D1992%26volume%3D11%26spage%3D7%26epage%3D12%26doi%3D10.1007%2FBF01583725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valdés, M.</span></span> <span> </span><span class="NLM_article-title"><i>Micromonospora</i>: An important microbe for biomedicine and potentially for biocontrol and biofuels</span>. <i>Soil Biol. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1016/j.soilbio.2009.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2Fj.soilbio.2009.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC3cXis1SgsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2010&pages=536-542&author=A.+M.+Hirschauthor=M.+Vald%C3%A9s&title=Micromonospora%3A+An+important+microbe+for+biomedicine+and+potentially+for+biocontrol+and+biofuels&doi=10.1016%2Fj.soilbio.2009.11.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Micromonospora: An important microbe for biomedicine and potentially for biocontrol and biofuels</span></div><div class="casAuthors">Hirsch, Ann M.; Valdes, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Soil Biology & Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">536-542</span>CODEN:
                <span class="NLM_cas:coden">SBIOAH</span>;
        ISSN:<span class="NLM_cas:issn">0038-0717</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Micromonospora species have long been recognized as important sources of antibiotics and also for their unusual spores.  However, their involvement in plant-microbe assocns. is poorly understood although several studies demonstrate that Micromonospora species function in biocontrol, plant growth promotion, root ecol., and in the breakdown of plant cell wall material.  Our knowledge of this generally understudied group of actinomycetes has been greatly advanced by the increasing no. of reports of their assocns. with plants, by the deployment of DNA cloning and mol. systematics techniques, and by the recent application of whole genome sequencing.  Efforts to annotate the genomes of several Micromonospora species are underway.  This information will greatly augment our knowledge of these versatile microorganisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF5Tw8YxW-ALVg90H21EOLACvtfcHk0lhXSbgZcCLXQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXis1SgsLg%253D&md5=cc420b1cbb5d4c38afe1da3f93d833c1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.soilbio.2009.11.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.soilbio.2009.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DHirsch%26aufirst%3DA.%2BM.%26aulast%3DVald%25C3%25A9s%26aufirst%3DM.%26atitle%3DMicromonospora%253A%2520An%2520important%2520microbe%2520for%2520biomedicine%2520and%2520potentially%2520for%2520biocontrol%2520and%2520biofuels%26jtitle%3DSoil%2520Biol.%2520Biochem.%26date%3D2010%26volume%3D42%26spage%3D536%26epage%3D542%26doi%3D10.1016%2Fj.soilbio.2009.11.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dairi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, E.</span></span> <span> </span><span class="NLM_article-title">A novel, highly efficient gene-cloning system for <i>Micromonospora</i> strains</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">7004</span>– <span class="NLM_lpage">7011</span>, <span class="refDoi"> DOI: 10.1128/JB.173.21.7004-7011.1991</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1128%2FJB.173.21.7004-7011.1991" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1938902" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADyaK38Xhs1Cisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=1991&pages=7004-7011&author=M.+Hasegawaauthor=T.+Dairiauthor=T.+Ohtaauthor=E.+Hashimoto&title=A+novel%2C+highly+efficient+gene-cloning+system+for+Micromonospora+strains&doi=10.1128%2FJB.173.21.7004-7011.1991"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A novel, highly efficient gene-cloning system for Micromonospora strains</span></div><div class="casAuthors">Hasegawa, Mamoru; Dairi, Tohru; Ohta, Toshio; Hashimoto, Eri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7004-11</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    </div><div class="casAbstract">A highly efficient gene-cloning system for M. olivasterospora, a producer of the antibiotic fortimicin a (astromicin), suited to shotgun cloning had been developed.  The system is supported by 2 new advancements accomplished in this study.  One is the construction of novel plasmid vectors pMO116, pMO126, pMO133, pMO136, and pMO217, all consisting of replicons from newly found Micromonospora plasmids and selectable markers cloned from a neomycin-producing Micromonospora strain.  The other advancement is the establishment of a new protocol for bacterial protoplasting in which some kinds of sugar alcs. are added in precultures.  Such sugar alcs. were found to sensitize a wide taxonomical range of bacteria to lysozyme.  The system is reproducible and reliable and has a high efficiency of more than 106 CFU/μg of DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq89hnnZDY9rrVg90H21EOLACvtfcHk0lhXSbgZcCLXQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xhs1Cisw%253D%253D&md5=c902d33b7ed20dab3cb223101771ef3b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FJB.173.21.7004-7011.1991&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.173.21.7004-7011.1991%26sid%3Dliteratum%253Aachs%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DDairi%26aufirst%3DT.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DHashimoto%26aufirst%3DE.%26atitle%3DA%2520novel%252C%2520highly%2520efficient%2520gene-cloning%2520system%2520for%2520Micromonospora%2520strains%26jtitle%3DJ.%2520Bacteriol.%26date%3D1991%26volume%3D173%26spage%3D7004%26epage%3D7011%26doi%3D10.1128%2FJB.173.21.7004-7011.1991" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopwood, D. A.</span></span> <span> </span><span class="NLM_article-title">Highlights of <i>Streptomyces</i> genetics</span>. <i>Heredity</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1038/s41437-019-0196-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1038%2Fs41437-019-0196-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=31189905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A280%3ADC%252BB3M3lvVCgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2019&pages=23-32&author=D.+A.+Hopwood&title=Highlights+of+Streptomyces+genetics&doi=10.1038%2Fs41437-019-0196-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Highlights of Streptomyces genetics</span></div><div class="casAuthors">Hopwood David A</div><div class="citationInfo"><span class="NLM_cas:title">Heredity</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-32</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Sixty years ago, the actinomycetes, which include members of the genus Streptomyces, with their bacterial cellular dimensions but a mycelial growth habit like fungi, were generally regarded as a possible intermediate group, and virtually nothing was known about their genetics.  We now know that they are bacteria, but with many original features.  Their genome is linear with a unique mode of replication, not circular like those of nearly all other bacteria.  They transfer their chromosome from donor to recipient by a conjugation mechanism, but this is radically different from the E. coli paradigm.  They have twice as many genes as a typical rod-shaped bacterium like Escherichia coli or Bacillus subtilis, and the genome typically carries 20 or more gene clusters encoding the biosynthesis of antibiotics and other specialised metabolites, only a small proportion of which are expressed under typical laboratory screening conditions.  This means that there is a vast number of potentially valuable compounds to be discovered when these 'sleeping' genes are activated.  Streptomyces genetics has revolutionised natural product chemistry by facilitating the analysis of novel biosynthetic steps and has led to the ability to engineer novel biosynthetic pathways and hence 'unnatural natural products', with potential to generate lead compounds for use in the struggle to combat the rise of antimicrobial resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS-X8YfK5A6lSjiVMCqkKxVfW6udTcc2ea080NCcihaNrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3lvVCgug%253D%253D&md5=a04d8335352df921a5720a340a1f7def</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fs41437-019-0196-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41437-019-0196-0%26sid%3Dliteratum%253Aachs%26aulast%3DHopwood%26aufirst%3DD.%2BA.%26atitle%3DHighlights%2520of%2520Streptomyces%2520genetics%26jtitle%3DHeredity%26date%3D2019%26volume%3D123%26spage%3D23%26epage%3D32%26doi%3D10.1038%2Fs41437-019-0196-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rout, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitsinos, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyssikatos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schammel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aujay, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavrilyuk, J.</span></span> <span> </span><span class="NLM_article-title">Total synthesis and biological evaluation of tiancimycins A and B, yangpumicin A, and related anthraquinone-fused enediyne antitumor antibiotics</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">2549</span>– <span class="NLM_lpage">2561</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b12522</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b12522" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFWms7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=2549-2561&author=K.+C.+Nicolaouauthor=D.+Dasauthor=Y.+Luauthor=S.+Routauthor=E.+N.+Pitsinosauthor=J.+Lyssikatosauthor=A.+Schammelauthor=J.+Sandovalauthor=M.+Hammondauthor=M.+Aujayauthor=J.+Gavrilyuk&title=Total+synthesis+and+biological+evaluation+of+tiancimycins+A+and+B%2C+yangpumicin+A%2C+and+related+anthraquinone-fused+enediyne+antitumor+antibiotics&doi=10.1021%2Fjacs.9b12522"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Total Synthesis and Biological Evaluation of Tiancimycins A and B, Yangpumicin A, and Related Anthraquinone-Fused Enediyne Antitumor Antibiotics</span></div><div class="casAuthors">Nicolaou, K. C.; Das, Dipendu; Lu, Yong; Rout, Subhrajit; Pitsinos, Emmanuel N.; Lyssikatos, Joseph; Schammel, Alexander; Sandoval, Joseph; Hammond, Mikhail; Aujay, Monette; Gavrilyuk, Julia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2549-2561</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The family of anthraquinone-fused enediyne antitumor antibiotics was established by the discovery of dynemicin A and deoxy-dynemicin A.  It was then expanded, first by the isolation of uncialamycin, and then by the addn. to the family of tiancimycins A-F and yangpumicin A.  This family of natural products provides opportunities in total synthesis, biol., and medicine due to their novel and challenging mol. structures, intriguing biol. properties and mechanism of action, and potential in targeted cancer therapies.  Herein, the total syntheses of tiancimycins A and B, yangpumicin A, and a no. of related anthraquinone-fused enediynes are described.  Biol. evaluation of the synthesized compds. revealed extremely potent cytotoxicities against a no. of cell lines, thus enriching the structure-activity relationships within this class of compds.  The findings of these studies may facilitate future investigations directed toward antibody-drug conjugates for targeted cancer therapies and provide inspiration for further advances in total synthesis and chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1BrlmzoSwJLVg90H21EOLACvtfcHk0lievANP44FGyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFWms7s%253D&md5=6fb43e13dabe5299b97a8cceb63be319</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b12522&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b12522%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26aulast%3DDas%26aufirst%3DD.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DRout%26aufirst%3DS.%26aulast%3DPitsinos%26aufirst%3DE.%2BN.%26aulast%3DLyssikatos%26aufirst%3DJ.%26aulast%3DSchammel%26aufirst%3DA.%26aulast%3DSandoval%26aufirst%3DJ.%26aulast%3DHammond%26aufirst%3DM.%26aulast%3DAujay%26aufirst%3DM.%26aulast%3DGavrilyuk%26aufirst%3DJ.%26atitle%3DTotal%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520tiancimycins%2520A%2520and%2520B%252C%2520yangpumicin%2520A%252C%2520and%2520related%2520anthraquinone-fused%2520enediyne%2520antitumor%2520antibiotics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26spage%3D2549%26epage%3D2561%26doi%3D10.1021%2Fjacs.9b12522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattanayak, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasunoori, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poudel, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chowdari, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nazeer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangwar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vite, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitsinos, E. N.</span></span> <span> </span><span class="NLM_article-title">Streamlined total synthesis of uncialamycin and its application to the synthesis of designed analogues for biological investigations</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">8235</span>– <span class="NLM_lpage">8246</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b04339</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b04339" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC28XpsVClsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=8235-8246&author=K.+C.+Nicolaouauthor=Y.+Wangauthor=M.+Luauthor=D.+Mandalauthor=M.+R.+Pattanayakauthor=R.+Yuauthor=A.+A.+Shahauthor=J.+S.+Chenauthor=H.+Zhangauthor=J.+J.+Crawfordauthor=L.+Pasunooriauthor=Y.+B.+Poudelauthor=N.+S.+Chowdariauthor=C.+Panauthor=A.+Nazeerauthor=S.+Gangwarauthor=G.+Viteauthor=E.+N.+Pitsinos&title=Streamlined+total+synthesis+of+uncialamycin+and+its+application+to+the+synthesis+of+designed+analogues+for+biological+investigations&doi=10.1021%2Fjacs.6b04339"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Streamlined Total Synthesis of Uncialamycin and Its Application to the Synthesis of Designed Analogues for Biological Investigations</span></div><div class="casAuthors">Nicolaou, K. C.; Wang, Yanping; Lu, Min; Mandal, Debashis; Pattanayak, Manas R.; Yu, Ruocheng; Shah, Akshay A.; Chen, Jason S.; Zhang, Hongjun; Crawford, James J.; Pasunoori, Laxman; Poudel, Yam B.; Chowdari, Naidu S.; Pan, Chin; Nazeer, Ayesha; Gangwar, Sanjeev; Vite, Gregory; Pitsinos, Emmanuel N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">8235-8246</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">From the enediyne class of antitumor antibiotics, uncialamycin is among the rarest and most potent, yet one of the structurally simpler, making it attractive for chem. synthesis and potential applications in biol. and medicine.  In this article we describe a streamlined and practical enantioselective total synthesis of uncialamycin that is amenable to the synthesis of novel analogs and renders the natural product readily available for biol. and drug development studies.  Starting from hydroxy- or methoxyisatin, the synthesis features a Noyori enantioselective redn., a Yamaguchi acetylide-pyridinium coupling, a stereoselective acetylide-aldehyde cyclization, and a newly developed annulation reaction that allows efficient coupling of a cyanophthalide and a p-methoxy semiquinone aminal to forge the anthraquinone moiety of the mol. [e.g., I + II →→ III].  Overall, the developed streamlined synthesis proceeds in 22 linear steps (14 chromatog. sepns.) and 11% overall yield.  The developed synthetic strategies and technologies were applied to the synthesis of a series of designed uncialamycin analogs equipped with suitable functional groups for conjugation to antibodies and other delivery systems.  Biol. evaluation of a select no. of these analogs led to the identification of compds. with low picomolar potencies against certain cancer cell lines.  These compds. and others like them may serve as powerful payloads for the development of antibody drug conjugates (ADCs) intended for personalized targeted cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJad8noSFDWrVg90H21EOLACvtfcHk0lievANP44FGyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpsVClsbc%253D&md5=c30cad508810a5f0b06cbf8da0c96871</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b04339&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b04339%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DM.%26aulast%3DMandal%26aufirst%3DD.%26aulast%3DPattanayak%26aufirst%3DM.%2BR.%26aulast%3DYu%26aufirst%3DR.%26aulast%3DShah%26aufirst%3DA.%2BA.%26aulast%3DChen%26aufirst%3DJ.%2BS.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26aulast%3DPasunoori%26aufirst%3DL.%26aulast%3DPoudel%26aufirst%3DY.%2BB.%26aulast%3DChowdari%26aufirst%3DN.%2BS.%26aulast%3DPan%26aufirst%3DC.%26aulast%3DNazeer%26aufirst%3DA.%26aulast%3DGangwar%26aufirst%3DS.%26aulast%3DVite%26aufirst%3DG.%26aulast%3DPitsinos%26aufirst%3DE.%2BN.%26atitle%3DStreamlined%2520total%2520synthesis%2520of%2520uncialamycin%2520and%2520its%2520application%2520to%2520the%2520synthesis%2520of%2520designed%2520analogues%2520for%2520biological%2520investigations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D8235%26epage%3D8246%26doi%3D10.1021%2Fjacs.6b04339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span> <span> </span><span class="NLM_article-title">Ribosome engineering and fermentation optimization leads to overproduction of tiancimycin A, a new enediyne natural product from <i>Streptomyces</i> sp. CB03234</span>. <i>J. Ind. Microbiol. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1007/s10295-018-2014-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1007%2Fs10295-018-2014-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=29396746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit12hs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2018&pages=141-151&author=L.+Liuauthor=J.+Panauthor=Z.+Wangauthor=X.+Yanauthor=D.+Yangauthor=X.+Zhuauthor=B.+Shenauthor=Y.+Duanauthor=Y.+Huang&title=Ribosome+engineering+and+fermentation+optimization+leads+to+overproduction+of+tiancimycin+A%2C+a+new+enediyne+natural+product+from+Streptomyces+sp.+CB03234&doi=10.1007%2Fs10295-018-2014-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Ribosome engineering and fermentation optimization leads to overproduction of tiancimycin A, a new enediyne natural product from Streptomyces sp. CB03234</span></div><div class="casAuthors">Liu, Ling; Pan, Jian; Wang, Zilong; Yan, Xiaohui; Yang, Dong; Zhu, Xiangcheng; Shen, Ben; Duan, Yanwen; Huang, Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Industrial Microbiology & Biotechnology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">141-151</span>CODEN:
                <span class="NLM_cas:coden">JIMBFL</span>;
        ISSN:<span class="NLM_cas:issn">1367-5435</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Tiancimycin (TNM) A, a recently discovered enediyne natural product from Streptomyces sp.  CB03234, showed rapid and complete killing of cancer cells and could be used as a payload in antibody drug conjugates.  The low yield of TNM A in the wild-type strain promoted us to use ribosome engineering and fermn. optimization for its yield improvement.  The Streptomyces sp.  CB03234-R-16 mutant strain with a L422P mutation in RpoB, the RNA polymerase β-subunit, was obtained from the rifamycin-resistant screening.  After fermn. optimization, the titers of TNM A in Streptomyces sp.  CB03234-R-16 reached to 22.5 ± 3.1 mg L-1 in shaking flasks, and 13 ± 1 mg L-1 in 15 L fermentors, which were at least 40-fold higher than that in the wild-type strain (∼ 0.3 mg L-1).  Quant. real-time RT-PCR revealed markedly enhanced expression of key genes encoding TNM A biosynthetic enzymes and regulators in Streptomyces sp.  CB03234-R-16.  Our study should greatly facilitate the future efforts to develop TNM A into a clin. anticancer drug.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4tFVDNUS8W7Vg90H21EOLACvtfcHk0lievANP44FGyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit12hs7k%253D&md5=a50063bdb64df4afb255310d2d950611</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs10295-018-2014-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10295-018-2014-8%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DPan%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DY.%26atitle%3DRibosome%2520engineering%2520and%2520fermentation%2520optimization%2520leads%2520to%2520overproduction%2520of%2520tiancimycin%2520A%252C%2520a%2520new%2520enediyne%2520natural%2520product%2520from%2520Streptomyces%2520sp.%2520CB03234%26jtitle%3DJ.%2520Ind.%2520Microbiol.%2520Biotechnol.%26date%3D2018%26volume%3D45%26spage%3D141%26epage%3D151%26doi%3D10.1007%2Fs10295-018-2014-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span> <span> </span><span class="NLM_article-title">Genome shuffling based on different types of ribosome engineering mutants for enhanced production of 10-membered enediyne tiancimycin-A</span>. <i>Appl. Microbiol. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">4359</span>– <span class="NLM_lpage">4369</span>, <span class="refDoi"> DOI: 10.1007/s00253-020-10583-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1007%2Fs00253-020-10583-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=32236679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmtVKlt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2020&pages=4359-4369&author=H.+Liuauthor=C.+Jiangauthor=J.+Linauthor=Z.+Zhuangauthor=W.+Kongauthor=L.+Liuauthor=Y.+Huangauthor=Y.+Duanauthor=X.+Zhu&title=Genome+shuffling+based+on+different+types+of+ribosome+engineering+mutants+for+enhanced+production+of+10-membered+enediyne+tiancimycin-A&doi=10.1007%2Fs00253-020-10583-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Genome shuffling based on different types of ribosome engineering mutants for enhanced production of 10-membered enediyne tiancimycin-A</span></div><div class="casAuthors">Liu, Huiming; Jiang, Chengzhou; Lin, Jing; Zhuang, Zhoukang; Kong, Wenping; Liu, Ling; Huang, Yong; Duan, Yanwen; Zhu, Xiangcheng</div><div class="citationInfo"><span class="NLM_cas:title">Applied Microbiology and Biotechnology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4359-4369</span>CODEN:
                <span class="NLM_cas:coden">AMBIDG</span>;
        ISSN:<span class="NLM_cas:issn">0175-7598</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Abstr.: Tiancimycin-A (TNM-A) is an anthraquinone-fused ten-membered enediyne produced by Streptomyces sp.  CB03234, which is very promising for the development of anticancer antibody-drug conjugates (ADCs).  To improve the titer of TNM-A, we have generated high-producing mutants CB03234-S and CB03234-R through ribosome engineering, but still not sufficient for pilot prodn. of TNM-A.  As the follow-up work, gentamycin-induced ribosome engineering was further adopted here to generate the mutant CB03234-G, which produced similar level of TNM-A as in CB03234-S and CB03234-R.  Benefiting from the distinct antibiotic resistances of three ribosome engineering mutants, genome shuffling between any two of them was resp. carried out, and finally obtained the recombinant CB03234-GS26.  Under optimal conditions, CB03234-GS26 produced 40.6 ± 1.0 mg/L TNM-A in shaking flasks and 20.8 ± 0.4 mg/L in a scaled-up 30-L fermentor.  Comparing with the parental high-producing mutants, the over 1.6-fold titer improvement of CB03234-GS26 in fermentor was more promising for pilot prodn. of TNM-A.  Besides the distinctive morphol. features, genetic characterization revealed that CB03234-GS26 possessed 1.8 kb rsmG related deletion just the same as CB03234-S, but no mutation was found in rpsL.  Subsequent knockouts proved that rsmG was unrelated to titer improvement of TNM-A, which implied other genomic variations and mechanisms rather than ribosome engineering to enhance the biosynthesis of TNM-A.  Therefore, CB03234-GS26 provided a basis to locate potential novel genetic targets, and explore the interactions between complex metabolic network and TNM biosynthetic pathway, which shall promote future construction of high-yielding systems for TNM-A and other anthraquinone-fused enediynes.  Key Points •United genome shuffling and ribosome engineering help further strain improvement. •CB03234-GS26 with improved titer is practical for the pilot prodn. of TNM-A. •Enhanced TNM-A prodn. should attribute to novel genetic features/mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowT-Voi_xWA7Vg90H21EOLACvtfcHk0lhVOAwygYhA3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmtVKlt7w%253D&md5=38015dfef5ae8a80ab386ffa805cf7f4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs00253-020-10583-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00253-020-10583-2%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DZhuang%26aufirst%3DZ.%26aulast%3DKong%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DX.%26atitle%3DGenome%2520shuffling%2520based%2520on%2520different%2520types%2520of%2520ribosome%2520engineering%2520mutants%2520for%2520enhanced%2520production%2520of%252010-membered%2520enediyne%2520tiancimycin-A%26jtitle%3DAppl.%2520Microbiol.%2520Biotechnol.%26date%3D2020%26volume%3D104%26spage%3D4359%26epage%3D4369%26doi%3D10.1007%2Fs00253-020-10583-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span> <span> </span><span class="NLM_article-title">Streptomycin-induced ribosome engineering complemented with fermentation optimization for enhanced production of 10-membered enediynes tiancimycin-A and tiancimycin-D</span>. <i>Biotechnol. Bioeng.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">1304</span>– <span class="NLM_lpage">1314</span>, <span class="refDoi"> DOI: 10.1002/bit.26944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fbit.26944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=30712262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVyitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=1304-1314&author=Z.+Zhuangauthor=C.+Jiangauthor=F.+Zhangauthor=R.+Huangauthor=L.+Yiauthor=Y.+Huangauthor=X.+Yanauthor=Y.+Duanauthor=X.+Zhu&title=Streptomycin-induced+ribosome+engineering+complemented+with+fermentation+optimization+for+enhanced+production+of+10-membered+enediynes+tiancimycin-A+and+tiancimycin-D&doi=10.1002%2Fbit.26944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Streptomycin-induced ribosome engineering complemented with fermentation optimization for enhanced production of 10-membered enediynes tiancimycin-A and tiancimycin-D</span></div><div class="casAuthors">Zhuang, Zhoukang; Jiang, Chengzhou; Zhang, Fan; Huang, Rong; Yi, Liwei; Huang, Yong; Yan, Xiaohui; Duan, Yanwen; Zhu, Xiangcheng</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology and Bioengineering</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1304-1314</span>CODEN:
                <span class="NLM_cas:coden">BIBIAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3592</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Tiancimycins (TNMs) are a group of 10-membered anthraquinone-fused enediynes, newly discovered from Streptomyces sp.  CB03234. Among them, TNM-A and TNM-D have exhibited excellent antitumor performances and could be exploited as very promising warheads for the development of anticancer antibody-drug conjugates (ADCs).  However, their low titers, esp. TNM-D, have severely limited following progress.  Therefore, the streptomycin-induced ribosome engineering was adopted in this work for strain improvement of CB03234, and a TNMs high producer S. sp.  CB03234-S with the K43N mutation at 30S ribosomal protein S12 was successfully screened out.  Subsequent media optimization revealed the essential effects of iodide and copper ion on the prodn. of TNMs, while the substitution of nitrogen source could evidently promote the accumulation of TNM-D, and the ratio of produced TNM-A and TNM-D was responsive to the change of carbon and nitrogen ratio in the medium.  Further amelioration of the pH control in scaled up 25 L fermn. increased the av. titers of TNM-A and TNM-D up to 13.7 ± 0.3 and 19.2 ± 0.4 mg/L, resp.  The achieved over 45-fold titer improvement of TNM-A, and 109-fold total titer improvement of TNM-A and TNM-D enabled the efficient purifn. of over 200 mg of each target mol. from 25 L fermn.  Our efforts have demonstrated a practical strategy for titer improvement of anthraquinone-fused enediynes and set up a solid base for the pilot scale prodn. and preclin. studies of TNMs to expedite the future development of anticancer ADC drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd_0ApYytIDbVg90H21EOLACvtfcHk0lhVOAwygYhA3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVyitr8%253D&md5=1c75f4bed197e064cf200e41770e1ba0</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fbit.26944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbit.26944%26sid%3Dliteratum%253Aachs%26aulast%3DZhuang%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DYi%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DX.%26atitle%3DStreptomycin-induced%2520ribosome%2520engineering%2520complemented%2520with%2520fermentation%2520optimization%2520for%2520enhanced%2520production%2520of%252010-membered%2520enediynes%2520tiancimycin-A%2520and%2520tiancimycin-D%26jtitle%3DBiotechnol.%2520Bioeng.%26date%3D2019%26volume%3D116%26spage%3D1304%26epage%3D1314%26doi%3D10.1002%2Fbit.26944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teijaro, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crnovcic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annaval, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rader, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span> <span> </span><span class="NLM_article-title">Comparative studies of the biosynthetic gene clusters for anthraquinone-fused enediynes shedding light into the tailoring steps of tiancimycin biosynthesis</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5918</span>– <span class="NLM_lpage">5921</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.8b02584</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.8b02584" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWitb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=5918-5921&author=X.+Yanauthor=J.+J.+Chenauthor=A.+Adhikariauthor=C.+N.+Teijaroauthor=H.+Geauthor=I.+Crnovcicauthor=C.+Y.+Changauthor=T.+Annavalauthor=D.+Yangauthor=C.+Raderauthor=B.+Shen&title=Comparative+studies+of+the+biosynthetic+gene+clusters+for+anthraquinone-fused+enediynes+shedding+light+into+the+tailoring+steps+of+tiancimycin+biosynthesis&doi=10.1021%2Facs.orglett.8b02584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative studies of the biosynthetic gene clusters for anthraquinone-fused enediynes shedding light into the tailoring steps of tiancimycin biosynthesis</span></div><div class="casAuthors">Yan, Xiaohui; Chen, Jian-Jun; Adhikari, Ajeeth; Teijaro, Christiana N.; Ge, Huiming; Crnovcic, Ivana; Chang, Chin-Yuan; Annaval, Thibault; Yang, Dong; Rader, Christoph; Shen, Ben</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5918-5921</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Comparative analyses of the four known anthraquinone-fused enediynes biosynthetic gene clusters identified four genes, tnmE6, tnmH, tnmL, and tnmQ, unique to the tnm gene cluster.  Larger scale fermn. of both the Streptomyces sp. CB03234 wild-type and the ΔtnmH and ΔtnmL mutant strains resulted in the characterization of 20 new tiancimycin (TNM) congeners, including five enediynes.  These findings enabled a proposal for the late stage of TNM biosynthesis featuring an intermediate possibly common for all anthraquinone-fused enediynes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooxPD2YCUwK7Vg90H21EOLACvtfcHk0lhVOAwygYhA3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWitb7K&md5=8eb622f3fd0eaa003938aac8548619c7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.8b02584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.8b02584%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%2BJ.%26aulast%3DAdhikari%26aufirst%3DA.%26aulast%3DTeijaro%26aufirst%3DC.%2BN.%26aulast%3DGe%26aufirst%3DH.%26aulast%3DCrnovcic%26aufirst%3DI.%26aulast%3DChang%26aufirst%3DC.%2BY.%26aulast%3DAnnaval%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DRader%26aufirst%3DC.%26aulast%3DShen%26aufirst%3DB.%26atitle%3DComparative%2520studies%2520of%2520the%2520biosynthetic%2520gene%2520clusters%2520for%2520anthraquinone-fused%2520enediynes%2520shedding%2520light%2520into%2520the%2520tailoring%2520steps%2520of%2520tiancimycin%2520biosynthesis%26jtitle%3DOrg.%2520Lett.%26date%3D2018%26volume%3D20%26spage%3D5918%26epage%3D5921%26doi%3D10.1021%2Facs.orglett.8b02584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huber, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorson, J. S.</span></span> <span> </span><span class="NLM_article-title">AdoMet analog synthesis and utilization: current state of the art</span>. <i>Curr. Opin. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">189</span>– <span class="NLM_lpage">197</span>, <span class="refDoi"> DOI: 10.1016/j.copbio.2016.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2Fj.copbio.2016.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=27506965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Kls7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2016&pages=189-197&author=T.+D.+Huberauthor=B.+R.+Johnsonauthor=J.+Zhangauthor=J.+S.+Thorson&title=AdoMet+analog+synthesis+and+utilization%3A+current+state+of+the+art&doi=10.1016%2Fj.copbio.2016.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">AdoMet analog synthesis and utilization: current state of the art</span></div><div class="casAuthors">Huber, Tyler D.; Johnson, Brooke R.; Zhang, Jianjun; Thorson, Jon S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Biotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">189-197</span>CODEN:
                <span class="NLM_cas:coden">CUOBE3</span>;
        ISSN:<span class="NLM_cas:issn">0958-1669</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  S-Adenosyl-L-methionine (AdoMet) is an essential enzyme cosubstrate in fundamental biol. with an expanding range of biocatalytic and therapeutic applications.  In recent years, technologies enabling the synthesis and utilization of novel functional AdoMet surrogates have rapidly advanced.  Developments highlighted within this brief review include improved syntheses of AdoMet analogs, unique S-adenosyl-L-methionine isosteres with enhanced stability, and corresponding applications in epigenetics, proteomics and natural product/small mol. diversification ('alkylrandomization').</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdXtVaKlqZLLVg90H21EOLACvtfcHk0lgqWDDdZ2NleQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Kls7vM&md5=c2b40e0b3242276b6246998be3f9ba95</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.copbio.2016.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.copbio.2016.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DT.%2BD.%26aulast%3DJohnson%26aufirst%3DB.%2BR.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DThorson%26aufirst%3DJ.%2BS.%26atitle%3DAdoMet%2520analog%2520synthesis%2520and%2520utilization%253A%2520current%2520state%2520of%2520the%2520art%26jtitle%3DCurr.%2520Opin.%2520Biotechnol.%26date%3D2016%26volume%3D42%26spage%3D189%26epage%3D197%26doi%3D10.1016%2Fj.copbio.2016.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, L.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, M.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of the tiancilactone antibiotics by genome mining of atypical bacterial type II diterpene synthases</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1727</span>– <span class="NLM_lpage">1733</span>, <span class="refDoi"> DOI: 10.1002/cbic.201800285</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fcbic.201800285" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ans7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=1727-1733&author=L.-B.+Dongauthor=J.+D.+Rudolfauthor=M.-R.+Dengauthor=X.+Yanauthor=B.+Shen&title=Discovery+of+the+tiancilactone+antibiotics+by+genome+mining+of+atypical+bacterial+type+II+diterpene+synthases&doi=10.1002%2Fcbic.201800285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of the Tiancilactone Antibiotics by Genome Mining of Atypical Bacterial Type II Diterpene Synthases</span></div><div class="casAuthors">Dong, Liao-Bin; Rudolf, Jeffrey D.; Deng, Ming-Rong; Yan, Xiaohui; Shen, Ben</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1727-1733</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Although genome mining has advanced the identification, discovery, and study of microbial natural products, the discovery of bacterial diterpenoids continues to lag behind.  Herein, we report the identification of 66 putative producers of novel bacterial diterpenoids, and the discovery of the tiancilactone (TNL) family of antibiotics, by genome mining of type II diterpene synthases that do not possess the canonical DXDD motif.  The TNLs, which are broad-spectrum antibiotics with moderate activities, are produced by both Streptomyces sp.  CB03234 and Streptomyces sp.  CB03238 and feature a highly functionalized diterpenoid skeleton that is further decorated with chloroanthranilate and γ-butyrolactone moieties.  Genetic manipulation of the tnl gene cluster resulted in TNL congeners, which provided insights into their biosynthesis and structure-activity relationships.  This work highlights the biosynthetic potential that bacteria possess to produce diterpenoids and should inspire continued efforts to discover terpenoid natural products from bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY188wjn4457Vg90H21EOLACvtfcHk0lgqWDDdZ2NleQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ans7%252FJ&md5=33bc28a93cb14b3a4fa327a0849a5289</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201800285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201800285%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DL.-B.%26aulast%3DRudolf%26aufirst%3DJ.%2BD.%26aulast%3DDeng%26aufirst%3DM.-R.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520the%2520tiancilactone%2520antibiotics%2520by%2520genome%2520mining%2520of%2520atypical%2520bacterial%2520type%2520II%2520diterpene%2520synthases%26jtitle%3DChemBioChem%26date%3D2018%26volume%3D19%26spage%3D1727%26epage%3D1733%26doi%3D10.1002%2Fcbic.201800285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, J. D.</span>; <span class="NLM_string-name">Crnovcic, I.</span>; <span class="NLM_string-name">Shen, B.</span>; <span class="NLM_string-name">Newman, D. J.</span>; <span class="NLM_string-name">Cragg, G. M.</span>; <span class="NLM_string-name">Grothaus, P. G.</span></span> <span> </span><span class="NLM_article-title">The Role of Combinatorial Biosynthesis in Natural Products Discovery</span>. In  <i>Chemical Biology of Natural Products</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, D. J.</span>, <span class="NLM_string-name">Cragg, G. M.</span>, <span class="NLM_string-name">Grothaus, P. G.</span></span>, Eds.; <span class="NLM_publisher-name">CRC Press</span>, <span class="NLM_year">2017</span>; pp  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">125</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=87-125&author=J.+D.+Rudolf&author=I.+Crnovcic&author=B.+Shen&author=D.+J.+Newman&author=G.+M.+Cragg&author=P.+G.+Grothausauthor=D.+J.+Newman&author=G.+M.+Cragg&author=P.+G.+Grothaus&title=Chemical+Biology+of+Natural+Products"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DRudolf%26aufirst%3DJ.%2BD.%26atitle%3DThe%2520Role%2520of%2520Combinatorial%2520Biosynthesis%2520in%2520Natural%2520Products%2520Discovery%26btitle%3DChemical%2520Biology%2520of%2520Natural%2520Products%26aulast%3DNewman%26aufirst%3DD.%2BJ.%26pub%3DCRC%2520Press%26date%3D2017%26spage%3D87%26epage%3D125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stecher, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tengg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueberbacher, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwab, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griengl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber-Khadjawi, M.</span></span> <span> </span><span class="NLM_article-title">Biocatalytic friedel-crafts alkylation using non-natural cofactors</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">9546</span>– <span class="NLM_lpage">9548</span>, <span class="refDoi"> DOI: 10.1002/anie.200905095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fanie.200905095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFars7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=9546-9548&author=H.+Stecherauthor=M.+Tenggauthor=B.+J.+Ueberbacherauthor=P.+Remlerauthor=H.+Schwabauthor=H.+Grienglauthor=M.+Gruber-Khadjawi&title=Biocatalytic+friedel-crafts+alkylation+using+non-natural+cofactors&doi=10.1002%2Fanie.200905095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Biocatalytic Friedel-Crafts Alkylation Using Non-natural Cofactors</span></div><div class="casAuthors">Stecher, Harald; Tengg, Martin; Ueberbacher, Bernhard J.; Remler, Peter; Schwab, Helmut; Griengl, Herfried; Gruber-Khadjawi, Mandana</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">9546-9548, S9546/1-S9546/11</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A biocatalytic equiv. for the Friedel-Crafts alkylation is described.  The S-adenosyl-L-methionine-dependent methyltransferases NovO and COuO from Streptomyces sphaeroides were employed to alkylate amninocoumarins with alkylated derivs. of S-adenosyl-L-methionine.  These enzymes were shown to accept these modified cofactors and catalyzed the synthesis of not only methylated, but also allyl, propargyl, and benzylated arenes in moderate to high yields with excellent regioselectivity.  The enzymes also demonstrated acceptance of a wide variety of substrates and were used to methylate several naphthalenediols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEGC437uLgLbVg90H21EOLACvtfcHk0lgqWDDdZ2NleQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFars7fE&md5=4698c1cf0058a9a357811e786d401142</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fanie.200905095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200905095%26sid%3Dliteratum%253Aachs%26aulast%3DStecher%26aufirst%3DH.%26aulast%3DTengg%26aufirst%3DM.%26aulast%3DUeberbacher%26aufirst%3DB.%2BJ.%26aulast%3DRemler%26aufirst%3DP.%26aulast%3DSchwab%26aufirst%3DH.%26aulast%3DGriengl%26aufirst%3DH.%26aulast%3DGruber-Khadjawi%26aufirst%3DM.%26atitle%3DBiocatalytic%2520friedel-crafts%2520alkylation%2520using%2520non-natural%2520cofactors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26spage%3D9546%26epage%3D9548%26doi%3D10.1002%2Fanie.200905095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, B. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struck, A.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Micklefield, J.</span></span> <span> </span><span class="NLM_article-title">Site-specific bioalkylation of rapamycin by the RapM 16-<i>O</i>-methyltransferase</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">2885</span>– <span class="NLM_lpage">2892</span>, <span class="refDoi"> DOI: 10.1039/C5SC00164A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1039%2FC5SC00164A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=29403635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsF2qt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=2885-2892&author=B.+J.+C.+Lawauthor=A.-W.+Struckauthor=M.+R.+Bennettauthor=B.+Wilkinsonauthor=J.+Micklefield&title=Site-specific+bioalkylation+of+rapamycin+by+the+RapM+16-O-methyltransferase&doi=10.1039%2FC5SC00164A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Site-specific bioalkylation of rapamycin by the RapM 16-O-methyltransferase</span></div><div class="casAuthors">Law, Brian J. C.; Struck, Anna-Winona; Bennett, Matthew R.; Wilkinson, Barrie; Micklefield, Jason</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2885-2892</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The methylation of natural products by S-adenosyl methionine (AdoMet, also known as SAM)-dependent methyltransferase enzymes is a common tailoring step in many biosynthetic pathways.  The introduction of Me substituents can affect the biol. and physicochem. properties of the secondary metabolites produced.  Recently it has become apparent that some AdoMet-dependent methyltransferases exhibit promiscuity and will accept AdoMet analogs enabling the transfer of alternative alkyl groups.  In this study we have characterized a methyltransferase, RapM, which is involved in the biosynthesis of the potent immunosuppressive agent rapamycin.  We have shown that recombinant RapM regioselectively methylates the C16 hydroxyl group of desmethyl rapamycin precursors in vitro and is promiscuous in accepting alternative co-factors in addn. to AdoMet.  A coupled enzyme system was developed, including a mutant human enzyme methionine adenosyl transferase (MAT), along with RapM, which was used to prep. alkylated rapamycin derivs. (rapalogs) with alternative Et and allyl ether groups, derived from simple S-Et or S-allyl methionine analogs.  There are two other methyltransferases RapI and RapQ which provide Me substituents of rapamycin.  Consequently, using the enzymic approach described here, it should be possible to generate a diverse array of alkylated rapalogs, with altered properties, that would be difficult to obtain by traditional synthetic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdFgiDKgQ8U7Vg90H21EOLACvtfcHk0lhRBLM03a5aaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsF2qt74%253D&md5=7ed40aa2654771cb440486828f28aef2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC5SC00164A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5SC00164A%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DB.%2BJ.%2BC.%26aulast%3DStruck%26aufirst%3DA.-W.%26aulast%3DBennett%26aufirst%3DM.%2BR.%26aulast%3DWilkinson%26aufirst%3DB.%26aulast%3DMicklefield%26aufirst%3DJ.%26atitle%3DSite-specific%2520bioalkylation%2520of%2520rapamycin%2520by%2520the%2520RapM%252016-O-methyltransferase%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D2885%26epage%3D2892%26doi%3D10.1039%2FC5SC00164A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharpless, K. B.</span></span> <span> </span><span class="NLM_article-title">Click chemistry: diverse chemical function from a few good reactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">2004</span>– <span class="NLM_lpage">2021</span>, <span class="refDoi"> DOI: 10.1002/1521-3773(20010601)40:11<2004::AID-ANIE2004>3.0.CO;2-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=11433435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD3MXksVOis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=2004-2021&author=H.+C.+Kolbauthor=M.+G.+Finnauthor=K.+B.+Sharpless&title=Click+chemistry%3A+diverse+chemical+function+from+a+few+good+reactions&doi=10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Click chemistry: diverse chemical function from a few good reactions</span></div><div class="casAuthors">Kolb, Hartmuth C.; Finn, M. G.; Sharpless, K. Barry</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2004-2021</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Review with > 88 refs.  Examn. of nature's favorite mols. reveals a striking preference for making carbon-heteroatom bonds over carbon-carbon bonds - surely no surprise given that carbon dioxide is nature's starting material and that most reactions are performed in water.  Nucleic acids, proteins, and polysaccharides are condensation polymers of small subunits stitched together by carbon-heteroatom bonds.  Even the 35 or so building blocks from which these crucial mols. are made each contain, at most, six contiguous C-C bonds, except for the three arom. amino acids.  Taking a cue from nature's approach, the development of a set of powerful, highly reliable, and selective reactions for the rapid synthesis of useful new compds. and combinatorial libraries through heteroatom links (C-X-C), an approach called "click chem." is addressed.  Click chem. is at once defined, enabled, and constrained by a handful of nearly perfect "spring-loaded" reactions.  The stringent criteria for a process to earn click chem. status are described along with examples of the mol. frameworks that are easily made using this spartan, but powerful, synthetic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptrjo9nxXNRrVg90H21EOLACvtfcHk0lhRBLM03a5aaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXksVOis78%253D&md5=61c28b416c42c87821bc8a5bc2fdec36</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3773%252820010601%252940%253A11%253C2004%253A%253AAID-ANIE2004%253E3.0.CO%253B2-5%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DH.%2BC.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26aulast%3DSharpless%26aufirst%3DK.%2BB.%26atitle%3DClick%2520chemistry%253A%2520diverse%2520chemical%2520function%2520from%2520a%2520few%2520good%2520reactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2001%26volume%3D40%26spage%3D2004%26epage%3D2021%26doi%3D10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolmel, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kool, E. T.</span></span> <span> </span><span class="NLM_article-title">Oximes and hydrazones in bioconjugation: mechanism and catalysis</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">10358</span>– <span class="NLM_lpage">10376</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00090</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00090" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVaqsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=10358-10376&author=D.+K.+Kolmelauthor=E.+T.+Kool&title=Oximes+and+hydrazones+in+bioconjugation%3A+mechanism+and+catalysis&doi=10.1021%2Facs.chemrev.7b00090"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Oximes and Hydrazones in Bioconjugation: Mechanism and Catalysis</span></div><div class="casAuthors">Kolmel, Dominik K.; Kool, Eric T.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">10358-10376</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The formation of oximes and hydrazones is employed in numerous scientific fields as a simple and versatile conjugation strategy.  This imine-forming reaction is applied in fields as diverse as polymer chem., biomaterials and hydrogels, dynamic combinatorial chem., org. synthesis, and chem. biol.  Here we outline chem. developments in this field, with special focus on the past ∼10 years of developments.  Recent strategies for installing reactive carbonyl groups and α-nucleophiles into biomols. are described.  The basic chem. properties of reactants and products in this reaction are then reviewed, with an eye to understanding the reaction's mechanism and how reactant structure controls rates and equil. in the process.  Recent work that has uncovered structural features and new mechanisms for speeding the reaction, sometimes by orders of magnitude, is discussed.  We describe recent studies that have identified esp. fast reacting aldehyde/ketone substrates and structural effects that lead to rapid-reacting α-nucleophiles as well.  Among the most effective new strategies has been the development of substituents near the reactive aldehyde group that either transfer protons at the transition state or trap the initially formed tetrahedral intermediates.  In addn., the recent development of efficient nucleophilic catalysts for the reaction is outlined, improving greatly upon aniline, the classical catalyst for imine formation.  A no. of uses of such second- and third-generation catalysts in bioconjugation and in cellular applications are highlighted.  While formation of hydrazone and oxime has been traditionally regarded as being limited by slow rates, developments in the past 5 years have resulted in completely overturning this limitation; indeed, the reaction is now one of the fastest and most versatile reactions available for conjugations of biomols. and biomaterials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnTRIa9Q3ZtbVg90H21EOLACvtfcHk0lhRBLM03a5aaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVaqsbzP&md5=e8c44eea0d1343e0bb0b17dc460640eb</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00090%26sid%3Dliteratum%253Aachs%26aulast%3DKolmel%26aufirst%3DD.%2BK.%26aulast%3DKool%26aufirst%3DE.%2BT.%26atitle%3DOximes%2520and%2520hydrazones%2520in%2520bioconjugation%253A%2520mechanism%2520and%2520catalysis%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D10358%26epage%3D10376%26doi%3D10.1021%2Facs.chemrev.7b00090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutureira, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerreiro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimenez-Oses, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardes, G. J.</span></span> <span> </span><span class="NLM_article-title">A minimal, unstrained <i>S</i>-allyl handle for pre-targeting diels-alder bioorthogonal labeling in live cells</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">14683</span>– <span class="NLM_lpage">14687</span>, <span class="refDoi"> DOI: 10.1002/anie.201608438</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fanie.201608438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslagt7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=14683-14687&author=B.+L.+Oliveiraauthor=Z.+Guoauthor=O.+Boutureiraauthor=A.+Guerreiroauthor=G.+Jimenez-Osesauthor=G.+J.+Bernardes&title=A+minimal%2C+unstrained+S-allyl+handle+for+pre-targeting+diels-alder+bioorthogonal+labeling+in+live+cells&doi=10.1002%2Fanie.201608438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A Minimal, Unstrained S-Allyl Handle for Pre-Targeting Diels-Alder Bioorthogonal Labeling in Live Cells</span></div><div class="casAuthors">Oliveira, Bruno L.; Guo, Zijian; Boutureira, Omar; Guerreiro, Ana; Jimenez-Oses, Gonzalo; Bernardes, Goncalo J. L.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">14683-14687</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The unstrained S-allyl cysteine amino acid was site-specifically installed on apoptosis protein biomarkers and was further used as a chem. handle and ligation partner for 1,2,4,5-tetrazines by means of an inverse-electron-demand Diels-Alder reaction.  We demonstrate the utility of this minimal handle for the efficient labeling of apoptotic cells using a fluorogenic tetrazine dye in a pre-targeting approach.  The small size, easy chem. installation, and selective reactivity of the S-allyl handle towards tetrazines should be readily extendable to other proteins and biomols., which could facilitate their labeling within live cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkCcPrl66Fd7Vg90H21EOLACvtfcHk0lgBdWXuAD3jwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslagt7jK&md5=94ec320e3e90a3af8b7a24828ce623b1</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201608438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201608438%26sid%3Dliteratum%253Aachs%26aulast%3DOliveira%26aufirst%3DB.%2BL.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DBoutureira%26aufirst%3DO.%26aulast%3DGuerreiro%26aufirst%3DA.%26aulast%3DJimenez-Oses%26aufirst%3DG.%26aulast%3DBernardes%26aufirst%3DG.%2BJ.%26atitle%3DA%2520minimal%252C%2520unstrained%2520S-allyl%2520handle%2520for%2520pre-targeting%2520diels-alder%2520bioorthogonal%2520labeling%2520in%2520live%2520cells%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D14683%26epage%3D14687%26doi%3D10.1002%2Fanie.201608438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tasdelen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yagci, Y.</span></span> <span> </span><span class="NLM_article-title">Light-induced click reactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">5930</span>– <span class="NLM_lpage">5938</span>, <span class="refDoi"> DOI: 10.1002/anie.201208741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fanie.201208741" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntF2lsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=5930-5938&author=M.+A.+Tasdelenauthor=Y.+Yagci&title=Light-induced+click+reactions&doi=10.1002%2Fanie.201208741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Light-Induced Click Reactions</span></div><div class="casAuthors">Tasdelen, Mehmet Atilla; Yagci, Yusuf</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5930-5938</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Spatial and temporal control over chem. and biol. processes, both in terms of "tuning" products and providing site-specific control, is one of the most exciting and rapidly developing areas of modern science.  For synthetic chemists, the challenge is to discover and develop selective and efficient reactions capable of generating useful mols. in a variety of matrixes.  In recent studies, light has been recognized as a valuable method for detg. where, when, and to what extent a process is started or stopped.  Accordingly, this Minireview will present the fundamental aspects of light-induced click reactions, highlight the applications of these reactions to diverse fields of study, and discuss the potential for this methodol. to be applied to the study of biomol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOIfssFxuR9rVg90H21EOLACvtfcHk0lgBdWXuAD3jwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntF2lsLY%253D&md5=19bac16930cc09dee72fcc1f0307befe</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.201208741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201208741%26sid%3Dliteratum%253Aachs%26aulast%3DTasdelen%26aufirst%3DM.%2BA.%26aulast%3DYagci%26aufirst%3DY.%26atitle%3DLight-induced%2520click%2520reactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D5930%26epage%3D5938%26doi%3D10.1002%2Fanie.201208741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bothwell, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Islam, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M. K.</span></span> <span> </span><span class="NLM_article-title">Se-Adenosyl-L-selenomethionine cofactor analogue as a reporter of protein methylation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">14905</span>– <span class="NLM_lpage">14912</span>, <span class="refDoi"> DOI: 10.1021/ja304782r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja304782r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1GhtbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=14905-14912&author=I.+R.+Bothwellauthor=K.+Islamauthor=Y.+L.+Chenauthor=W.+H.+Zhengauthor=G.+Blumauthor=H.+T.+Dengauthor=M.+K.+Luo&title=Se-Adenosyl-L-selenomethionine+cofactor+analogue+as+a+reporter+of+protein+methylation&doi=10.1021%2Fja304782r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Se-Adenosyl-L-selenomethionine Cofactor Analogue as a Reporter of Protein Methylation</span></div><div class="casAuthors">Bothwell, Ian R.; Islam, Kabirul; Chen, Yuling; Zheng, Weihong; Blum, Gil; Deng, Haiteng; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14905-14912</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Post-translational methylation by S-adenosyl-L-methionine (SAM)-dependent methyltransferases plays essential roles in modulating protein function in both normal and disease states.  As such, there is a growing need to develop chem. reporters to examine the physiol. and pathol. roles of protein methyltransferases.  Several sterically bulky SAM analogs have previously been used to label substrates of specific protein methyltransferases.  However, broad application of these compds. has been limited by their general incompatibility with native enzymes.  Here we report a SAM surrogate, ProSeAM (propargylic Se-adenosyl-L-selenomethionine), as a reporter of methyltransferases.  ProSeAM can be processed by multiple protein methyltransferases for substrate labeling.  In contrast, sulfur-based propargylic SAM undergoes rapid decompn. at physiol. pH, likely via an allene intermediate.  In conjunction with fluorescent/affinity-based azide probes, copper-catalyzed azide-alkyne cycloaddn. chem., in-gel fluorescence visualization and proteomic anal., we further demonstrated ProSeAM's utility to profile substrates of endogenous methyltransferases in diverse cellular contexts.  These results thus feature ProSeAM as a convenient probe to study the activities of endogenous protein methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrulCEDr0aPsrVg90H21EOLACvtfcHk0lgBdWXuAD3jwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1GhtbbL&md5=d3b3cdab8c5bd12ffb9340e94e21242d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja304782r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja304782r%26sid%3Dliteratum%253Aachs%26aulast%3DBothwell%26aufirst%3DI.%2BR.%26aulast%3DIslam%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DY.%2BL.%26aulast%3DZheng%26aufirst%3DW.%2BH.%26aulast%3DBlum%26aufirst%3DG.%26aulast%3DDeng%26aufirst%3DH.%2BT.%26aulast%3DLuo%26aufirst%3DM.%2BK.%26atitle%3DSe-Adenosyl-L-selenomethionine%2520cofactor%2520analogue%2520as%2520a%2520reporter%2520of%2520protein%2520methylation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D14905%26epage%3D14912%26doi%3D10.1021%2Fja304782r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Islam, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span> <span> </span><span class="NLM_article-title">Expanding cofactor repertoire of protein lysine methyltransferase for substrate labeling</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">679</span>– <span class="NLM_lpage">684</span>, <span class="refDoi"> DOI: 10.1021/cb2000567</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb2000567" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltVyjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=679-684&author=K.+Islamauthor=W.+Zhengauthor=H.+Yuauthor=H.+Dengauthor=M.+Luo&title=Expanding+cofactor+repertoire+of+protein+lysine+methyltransferase+for+substrate+labeling&doi=10.1021%2Fcb2000567"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Expanding Cofactor Repertoire of Protein Lysine Methyltransferase for Substrate Labeling</span></div><div class="casAuthors">Islam, Kabirul; Zheng, Weihong; Yu, Haiqiang; Deng, Haiteng; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">679-684</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases (PKMTs) play crucial roles in normal physiol. and disease processes.  Profiling PKMT targets is an important but challenging task.  With cancer-relevant G9a as a target, we have demonstrated success in developing S-adenosyl-L-methionine (SAM) analogs, particularly (E)-hex-2-en-5-ynyl SAM (Hey-SAM), as cofactors for engineered G9a.  Hey-SAM analog in combination with G9a Y1154A mutant modifies the same set of substrates as their native counterparts with remarkable efficiency.  (E)-Hex-2-en-5-ynylated substrates undergo smooth click reaction with an azide-based probe.  This approach is thus suitable for substrate characterization of G9a and expected to further serve as a starting point to evolve other PKMTs to utilize a similar set of cofactors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZw9uQ2VEeIrVg90H21EOLACvtfcHk0ljx9YFyQtwapQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltVyjtr8%253D&md5=2401404608fddd56a7b07a27d9ab1647</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fcb2000567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb2000567%26sid%3Dliteratum%253Aachs%26aulast%3DIslam%26aufirst%3DK.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DExpanding%2520cofactor%2520repertoire%2520of%2520protein%2520lysine%2520methyltransferase%2520for%2520substrate%2520labeling%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2011%26volume%3D6%26spage%3D679%26epage%3D684%26doi%3D10.1021%2Fcb2000567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willnow, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscher, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinhold, E.</span></span> <span> </span><span class="NLM_article-title">A selenium-based click AdoMet analogue for versatile substrate labeling with wild-type protein methyltransferases</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1167</span>– <span class="NLM_lpage">1173</span>, <span class="refDoi"> DOI: 10.1002/cbic.201100781</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fcbic.201100781" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=22549896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt12hu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=1167-1173&author=S.+Willnowauthor=M.+Martinauthor=B.+Luscherauthor=E.+Weinhold&title=A+selenium-based+click+AdoMet+analogue+for+versatile+substrate+labeling+with+wild-type+protein+methyltransferases&doi=10.1002%2Fcbic.201100781"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A Selenium-Based Click AdoMet Analogue for Versatile Substrate Labeling with Wild-Type Protein Methyltransferases</span></div><div class="casAuthors">Willnow, Sophie; Martin, Michael; Luescher, Bernhard; Weinhold, Elmar</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1167-1173</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Protein methylation is catalyzed by S-adenosyl-L-methionine-dependent protein methyltransferases (MTases), and this posttranslational modification serves diverse cellular functions.  Some MTases seem to exhibit broad substrate specificities and comprehensive methods for target profiling are needed.  Here we report the synthesis of a new AdoMet analog for enzymic transfer of a small propargyl group and labeling of modified proteins through copper-catalyzed azide-alkyne cycloaddn. (CuAAC).  Replacement of sulfur by selenium strongly enhanced the stability of the progargylic cofactor, leading, in combination with better activation by the selenonium center, to higher enzymic reactivity.  A broad spectrum of wild-type protein MTases acting on lysine, arginine, and glutamine residues accept this cofactor and modified substrates can be efficiently labeled by CuAAC click chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1u6lpXCDHprVg90H21EOLACvtfcHk0ljx9YFyQtwapQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt12hu7Y%253D&md5=e5fb071a350e100a863403cb429c9651</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201100781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201100781%26sid%3Dliteratum%253Aachs%26aulast%3DWillnow%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DM.%26aulast%3DLuscher%26aufirst%3DB.%26aulast%3DWeinhold%26aufirst%3DE.%26atitle%3DA%2520selenium-based%2520click%2520AdoMet%2520analogue%2520for%2520versatile%2520substrate%2520labeling%2520with%2520wild-type%2520protein%2520methyltransferases%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26spage%3D1167%26epage%3D1173%26doi%3D10.1002%2Fcbic.201100781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ilari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savino, C.</span></span> <span> </span><span class="NLM_article-title">A practical approach to protein crystallography</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1525</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-6622-6_3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1007%2F978-1-4939-6622-6_3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=27896717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVCjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1525&publication_year=2017&pages=47-78&author=A.+Ilariauthor=C.+Savino&title=A+practical+approach+to+protein+crystallography&doi=10.1007%2F978-1-4939-6622-6_3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A practical approach to protein crystallography</span></div><div class="casAuthors">Ilari, Andrea; Savino, Carmelinda</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1525</span>
        (<span class="NLM_cas:issue">Bioinformatics, Volume 1: Data, Sequence Analysis, and Evolution</span>),
    <span class="NLM_cas:pages">47-78</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Macromol. crystallog. is a powerful tool for structural biol.  The resoln. of a protein crystal structure is becoming much easier than in the past, thanks to developments in computing, automation of crystn. techniques and high-flux synchrotron sources to collect diffraction datasets.  The aim of this chapter is to provide practical procedures to det. a protein crystal structure, illustrating the new techniques, exptl. methods, and software that have made protein crystallog. a tool accessible to a larger scientific community.  It is impossible to give more than a taste of what the X-ray crystallog. technique entails in one brief chapter and there are different ways to solve a protein structure.  Since the no. of structures available in the Protein Data Bank (PDB) is becoming ever larger (the protein data bank now contains more than 100,000 entries) and therefore the probability to find a good model to solve the structure is ever increasing, we focus our attention on the Mol. Replacement method.  Indeed, whenever applicable, this method allows the resoln. of macromol. structures starting from a single data set and a search model downloaded from the PDB, with the aid only of computer work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbAoxlcHP5a7Vg90H21EOLACvtfcHk0ljx9YFyQtwapQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVCjsbs%253D&md5=12a678a111723c52365ff3150c7db06e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-6622-6_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-6622-6_3%26sid%3Dliteratum%253Aachs%26aulast%3DIlari%26aufirst%3DA.%26aulast%3DSavino%26aufirst%3DC.%26atitle%3DA%2520practical%2520approach%2520to%2520protein%2520crystallography%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2017%26volume%3D1525%26spage%3D47%26epage%3D78%26doi%3D10.1007%2F978-1-4939-6622-6_3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">Molecular replacement with MOLREP</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1107%2FS0907444909042589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=20057045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=22-25&author=A.+Vaginauthor=A.+Teplyakov&title=Molecular+replacement+with+MOLREP&doi=10.1107%2FS0907444909042589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular replacement with MOLREP</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-25</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement that utilizes a no. of original approaches to rotational and translational search and data prepn.  Since the first publication describing the program, MOLREP has acquired a variety of features that include weighting of the X-ray data and search models, multi-copy search, fitting the model into electron d., structural superposition of two models and rigid-body refinement.  The program can run in a fully automatic mode using optimized parameters calcd. from the input data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzj5SzYvyu_7Vg90H21EOLACvtfcHk0ljx9YFyQtwapQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D&md5=820d114719aca209994ffb0403e3b20d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042589%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMolecular%2520replacement%2520with%2520MOLREP%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D22%26epage%3D25%26doi%3D10.1107%2FS0907444909042589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guenther, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruner, S. D.</span></span> <span> </span><span class="NLM_article-title">Molecular basis of substrate promiscuity for the SAM-dependent <i>O</i>-methyltransferase NcsB1, involved in the biosynthesis of the enediyne antitumor antibiotic neocarzinostatin</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">9590</span>– <span class="NLM_lpage">9598</span>, <span class="refDoi"> DOI: 10.1021/bi901257q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi901257q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFemu7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=9590-9598&author=H.+A.+Cookeauthor=E.+L.+Guentherauthor=Y.+Luoauthor=B.+Shenauthor=S.+D.+Bruner&title=Molecular+basis+of+substrate+promiscuity+for+the+SAM-dependent+O-methyltransferase+NcsB1%2C+involved+in+the+biosynthesis+of+the+enediyne+antitumor+antibiotic+neocarzinostatin&doi=10.1021%2Fbi901257q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis of Substrate Promiscuity for the SAM-Dependent O-Methyltransferase NcsB1, Involved in the Biosynthesis of the Enediyne Antitumor Antibiotic Neocarzinostatin</span></div><div class="casAuthors">Cooke, Heather A.; Guenther, Elizabeth L.; Luo, Yinggang; Shen, Ben; Bruner, Steven D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">9590-9598</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The small mol. component of chromoprotein enediyne antitumor antibiotics is biosynthesized through a convergent route, incorporating amino acid, polyketide, and carbohydrate building blocks around a central enediyne hydrocarbon core.  The naphthoic acid moiety of the enediyne neocarzinostatin plays key roles in the biol. activity of the natural product by interacting with both the carrier protein and duplex DNA at the site of action.  The in vitro characterization of an S-adenosylmethionine-dependent O-methyltransferase (NcsB1) in the neocarzinostatin biosynthetic pathway has previously been described [Y. Luo et al. (2008)].  Here, a structural basis for NcsB1 activity is provided, illustrating that the enzyme shares an overall architecture with a large family of S-adenosylmethionine-dependent proteins.  In addn., NcsB1 represents the first enzyme to be structurally characterized in the biosynthetic pathway of neocarzinostatin.  By cocrystg. the enzyme with various combinations of the cofactor and substrate analogs, details of the active site structure have been established.  Changes in subdomain orientation were obsd. via comparison of structures in the presence and absence of substrate, suggesting that reorientation of the enzyme is involved in binding of the substrate.  In addn., residues important for substrate discrimination were predicted and probed through site-directed mutagenesis and in vitro biochem. characterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGricMGpJqkBz7Vg90H21EOLACvtfcHk0lgBePHS3T2ZWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFemu7jE&md5=997f7ed8daf7a9bd97ad252516017ee0</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fbi901257q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi901257q%26sid%3Dliteratum%253Aachs%26aulast%3DCooke%26aufirst%3DH.%2BA.%26aulast%3DGuenther%26aufirst%3DE.%2BL.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DBruner%26aufirst%3DS.%2BD.%26atitle%3DMolecular%2520basis%2520of%2520substrate%2520promiscuity%2520for%2520the%2520SAM-dependent%2520O-methyltransferase%2520NcsB1%252C%2520involved%2520in%2520the%2520biosynthesis%2520of%2520the%2520enediyne%2520antitumor%2520antibiotic%2520neocarzinostatin%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D9590%26epage%3D9598%26doi%3D10.1021%2Fbi901257q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crnovcic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annaval, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nocek, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hindra</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babnigg, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joachimiak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. N.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span> <span> </span><span class="NLM_article-title">Resistance to enediyne antitumor antibiotics by sequestration</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1075</span>– <span class="NLM_lpage">1085</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.05.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2Fj.chembiol.2018.05.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=29937405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFyms7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=1075-1085&author=C.+Y.+Changauthor=X.+Yanauthor=I.+Crnovcicauthor=T.+Annavalauthor=C.+Changauthor=B.+Nocekauthor=J.+D.+Rudolfauthor=D.+Yangauthor=+Hindraauthor=G.+Babniggauthor=A.+Joachimiakauthor=G.+N.+Phillipsauthor=B.+Shen&title=Resistance+to+enediyne+antitumor+antibiotics+by+sequestration&doi=10.1016%2Fj.chembiol.2018.05.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to Enediyne Antitumor Antibiotics by Sequestration</span></div><div class="casAuthors">Chang, Chin-Yuan; Yan, Xiaohui; Crnovcic, Ivana; Annaval, Thibault; Chang, Changsoo; Nocek, Boguslaw; Rudolf, Jeffrey D.; Yang, Dong; Hindra; Babnigg, Gyorgy; Joachimiak, Andrzej; Phillips, George N. Jr.; Shen, Ben</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1075-1085.e4</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The enediynes, microbial natural products with extraordinary cytotoxicities, have been translated into clin. drugs.  Two self-resistance mechanisms are known in the enediyne producers-apoproteins for the nine-membered enediynes and self-sacrifice proteins for the ten-membered enediyne calicheamicin.  Here we show that: (1) tnmS1, tnmS2, and tnmS3 encode tiancimycin (TNM) resistance in its producer Streptomyces sp.  CB03234, (2) tnmS1, tnmS2, and tnmS3 homologs are found in all anthraquinone-fused enediyne producers, (3) TnmS1, TnmS2, and TnmS3 share a similar β barrel-like structure, bind TNMs with nanomolar KD values, and confer resistance by sequestration, and (4) TnmS1, TnmS2, and TnmS3 homologs are widespread in nature, including in the human microbiome.  These findings unveil an unprecedented resistance mechanism for the enediynes.  Mechanisms of self-resistance in producers serve as models to predict and combat future drug resistance in clin. settings.  Enediyne-based chemotherapies should now consider the fact that the human microbiome harbors genes encoding enediyne resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBh2PaCvbYc7Vg90H21EOLACvtfcHk0lgBePHS3T2ZWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFyms7%252FF&md5=087aa51c9d8b8d27662f1c0c0a122e24</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.05.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.05.012%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DC.%2BY.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DCrnovcic%26aufirst%3DI.%26aulast%3DAnnaval%26aufirst%3DT.%26aulast%3DChang%26aufirst%3DC.%26aulast%3DNocek%26aufirst%3DB.%26aulast%3DRudolf%26aufirst%3DJ.%2BD.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DHindra%26aulast%3DBabnigg%26aufirst%3DG.%26aulast%3DJoachimiak%26aufirst%3DA.%26aulast%3DPhillips%26aufirst%3DG.%2BN.%26aulast%3DShen%26aufirst%3DB.%26atitle%3DResistance%2520to%2520enediyne%2520antitumor%2520antibiotics%2520by%2520sequestration%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D1075%26epage%3D1085%26doi%3D10.1016%2Fj.chembiol.2018.05.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liscombe, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louie, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noel, J. P.</span></span> <span> </span><span class="NLM_article-title">Architectures, mechanisms and molecular evolution of natural product methyltransferases</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1238</span>– <span class="NLM_lpage">1250</span>, <span class="refDoi"> DOI: 10.1039/c2np20029e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1039%2Fc2np20029e" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=22850796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlCjsbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2012&pages=1238-1250&author=D.+K.+Liscombeauthor=G.+V.+Louieauthor=J.+P.+Noel&title=Architectures%2C+mechanisms+and+molecular+evolution+of+natural+product+methyltransferases&doi=10.1039%2Fc2np20029e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Architectures, mechanisms and molecular evolution of natural product methyltransferases</span></div><div class="casAuthors">Liscombe, David K.; Louie, Gordon V.; Noel, Joseph P.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1238-1250</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The addn. of a Me moiety to a small chem. is a common transformation in the biosynthesis of natural products across all 3 domains of life.  These methylation reactions are most often catalyzed by S-adenosyl-L-methionine (SAM)-dependent methyltransferases (MTs).  MTs are categorized based on the electron-rich, Me accepting atom, usually O, N, C, or S.  SAM-dependent natural product MTs (NPMTs) are responsible for the modification of a wide array of structurally distinct substrates, including signaling and host defense compds., pigments, prosthetic groups, cofactors, cell membrane and cell wall components, and xenobiotics.  Most notably, methylation modulates the bioavailability, bioactivity, and reactivity of acceptor mols., and thus exerts a central role on the functional output of many metabolic pathways.  The current understanding of the structural enzymol. of NPMTs groups these phylogenetically diverse enzymes into 2 MT-superfamily fold classes (class I and class III).  Structural biol. has also shed light on the catalytic mechanisms and mol. bases for substrate specificity for >50 NPMTs.  These biophys.-based approaches have contributed to an understanding of NPMT evolution, demonstrating how a widespread protein fold evolved to accommodate chem. diverse Me acceptors and to catalyze disparate mechanisms suited to the physiochem. properties of the target substrates.  This evolutionary diversity suggests that NPMTs may serve as starting points for generating new biocatalysts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4FfgX1BfZj7Vg90H21EOLACvtfcHk0lgBePHS3T2ZWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlCjsbjE&md5=3a753fff151c8b6e1df0b2472ddeea16</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1039%2Fc2np20029e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2np20029e%26sid%3Dliteratum%253Aachs%26aulast%3DLiscombe%26aufirst%3DD.%2BK.%26aulast%3DLouie%26aufirst%3DG.%2BV.%26aulast%3DNoel%26aufirst%3DJ.%2BP.%26atitle%3DArchitectures%252C%2520mechanisms%2520and%2520molecular%2520evolution%2520of%2520natural%2520product%2520methyltransferases%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2012%26volume%3D29%26spage%3D1238%26epage%3D1250%26doi%3D10.1039%2Fc2np20029e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruner, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span> <span> </span><span class="NLM_article-title">Regiospecific <i>O</i>-methylation of naphthoic acids catalyzed by NcsB1, an <i>O</i>-methyltransferase involved in the biosynthesis of the enediyne antitumor antibiotic neocarzinostatin</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">14694</span>– <span class="NLM_lpage">14702</span>, <span class="refDoi"> DOI: 10.1074/jbc.M802206200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1074%2Fjbc.M802206200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=18387946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvFyjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=14694-14702&author=Y.+Luoauthor=S.+Linauthor=J.+Zhangauthor=H.+A.+Cookeauthor=S.+D.+Brunerauthor=B.+Shen&title=Regiospecific+O-methylation+of+naphthoic+acids+catalyzed+by+NcsB1%2C+an+O-methyltransferase+involved+in+the+biosynthesis+of+the+enediyne+antitumor+antibiotic+neocarzinostatin&doi=10.1074%2Fjbc.M802206200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Regiospecific O-Methylation of Naphthoic Acids Catalyzed by NcsB1, an O-Methyltransferase Involved in the Biosynthesis of the Enediyne Antitumor Antibiotic Neocarzinostatin</span></div><div class="casAuthors">Luo, Yinggang; Lin, Shuangjun; Zhang, Jian; Cooke, Heather A.; Bruner, Steven D.; Shen, Ben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">14694-14702</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Neocarzinostatin, a clin. anticancer drug, is the archetypal member of the chromoprotein family of enediyne antitumor antibiotics that are composed of a nonprotein chromophore and an apoprotein.  The neocarzinostatin chromophore consists of a nine-membered enediyne core, a deoxyaminosugar, and a naphthoic acid moiety.  We have previously cloned and sequenced the neocarzinostatin biosynthetic gene cluster and proposed that the biosynthesis of the naphthoic acid moiety and its incorporation into the neocarzinostatin chromophore are catalyzed by five enzymes NcsB, NcsB1, NcsB2, NcsB3, and NcsB4.  Here we report the biochem. characterization of NcsB1, unveiling that: (i) NcsB1 is an S-adenosyl-L-methionine-dependent O-methyltransferase; (ii) NcsB1 catalyzes regiospecific methylation at the 7-hydroxy group of its native substrate, 2,7-dihydroxy-5-methyl-1-naphthoic acid; (iii) NcsB1 also recognizes other dihydroxynaphthoic acids as substrates and catalyzes regiospecific O-methylation; and (iv) the carboxylate and its ortho-hydroxy groups of the substrate appear to be crucial for NcsB1 substrate recognition and binding, and O-methylation takes place only at the free hydroxy group of these dihydroxynaphthoic acids.  These findings establish that NcsB1 catalyzes the third step in the biosynthesis of the naphthoic acid moiety of the neocarzinostatin chromophore and further support the early proposal for the biosynthesis of the naphthoic acid and its incorporation into the neocarzinostatin chromophore with free naphthoic acids serving as intermediates.  NcsB1 represents another opportunity that can now be exploited to produce novel neocarzinostatin analogs by engineering neocarzinostatin biosynthesis or applying directed biosynthesis strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb6utg1AhukbVg90H21EOLACvtfcHk0ljhdaFVtr3tKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvFyjtr0%253D&md5=2afb1f4afeea179b502ece557118c4a1</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M802206200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M802206200%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DCooke%26aufirst%3DH.%2BA.%26aulast%3DBruner%26aufirst%3DS.%2BD.%26aulast%3DShen%26aufirst%3DB.%26atitle%3DRegiospecific%2520O-methylation%2520of%2520naphthoic%2520acids%2520catalyzed%2520by%2520NcsB1%252C%2520an%2520O-methyltransferase%2520involved%2520in%2520the%2520biosynthesis%2520of%2520the%2520enediyne%2520antitumor%2520antibiotic%2520neocarzinostatin%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D14694%26epage%3D14702%26doi%3D10.1074%2Fjbc.M802206200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hendricks, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichersky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noel, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z. S.</span></span> <span> </span><span class="NLM_article-title">An enzyme-coupled colorimetric assay for <i>S</i>-adenosylmethionine-dependent methyltransferases</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>326</i></span>,  <span class="NLM_fpage">100</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2003.11.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2Fj.ab.2003.11.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=14769341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtV2htr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=326&publication_year=2004&pages=100-105&author=C.+L.+Hendricksauthor=J.+R.+Rossauthor=E.+Picherskyauthor=J.+P.+Noelauthor=Z.+S.+Zhou&title=An+enzyme-coupled+colorimetric+assay+for+S-adenosylmethionine-dependent+methyltransferases&doi=10.1016%2Fj.ab.2003.11.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">An enzyme-coupled colorimetric assay for S-adenosylmethionine-dependent methyltransferases</span></div><div class="casAuthors">Hendricks, Cheryl L.; Ross, Jeannine R.; Pichersky, Eran; Noel, Joseph P.; Zhou, Zhaohui Sunny</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">326</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">100-105</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">An enzyme-coupled colorimetric assay for salicylate O-methyltransferase (SAMT), which utilizes S-adenosyl-L-methionine (AdoMet) as the Me donor, is reported.  In this assay, S-adenosyl-L-homocysteine (AdoHcy), a common product of AdoMet-dependent transmethylation reactions, is 1st hydrolyzed by recombinant adenosylhomocysteine nucleosidase (EC 3.2.2.9) into adenine and S-ribosylhomocysteine.  Recombinant LuxS ribosylhomocysteinase (EC 3.2.1.148) then cleaves the latter compd. to form homocysteine.  Finally, homocysteine is quantified using Ellman's reagent and the accompanying absorption change at 412 nm through recording using a microplate format.  Notably, SAMT and most AdoMet-dependent methyltransferases undergo marked AdoHcy-mediated product inhibition.  As such, an addnl. advantage of this assay which includes adenosylhomocysteine nucleosidase is the destruction of AdoHcy, thus alleviating product inhibition.  Under the authors' assay conditions, complete substrate conversion was obsd. and precise kinetic parameters could be detd. in a facile and quant. manner.  This assay should be generally applicable to other AdoMet-dependent methyltransferases.  Moreover, the procedure is easily amendable to batch assay and high-throughput screening approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWrYV3XlPCCrVg90H21EOLACvtfcHk0ljhdaFVtr3tKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtV2htr0%253D&md5=4a260e9377063acf07ac392164b2fb69</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2003.11.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2003.11.014%26sid%3Dliteratum%253Aachs%26aulast%3DHendricks%26aufirst%3DC.%2BL.%26aulast%3DRoss%26aufirst%3DJ.%2BR.%26aulast%3DPichersky%26aufirst%3DE.%26aulast%3DNoel%26aufirst%3DJ.%2BP.%26aulast%3DZhou%26aufirst%3DZ.%2BS.%26atitle%3DAn%2520enzyme-coupled%2520colorimetric%2520assay%2520for%2520S-adenosylmethionine-dependent%2520methyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D326%26spage%3D100%26epage%3D105%26doi%3D10.1016%2Fj.ab.2003.11.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cornell, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swarts, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barry, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riscoe, M. K.</span></span> <span> </span><span class="NLM_article-title">Characterization of recombinant <i>Escherichia coli</i> 5′-methylthioadenosine/<i>S</i>-adenosylhomocysteine nucleosidase: Analysis of enzymatic activity and substrate specificity</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>228</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">732</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1996.1723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1006%2Fbbrc.1996.1723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=8941345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADyaK28XntVaitr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=228&publication_year=1996&pages=724-732&author=K.+A.+Cornellauthor=W.+E.+Swartsauthor=R.+D.+Barryauthor=M.+K.+Riscoe&title=Characterization+of+recombinant+Escherichia+coli+5%E2%80%B2-methylthioadenosine%2FS-adenosylhomocysteine+nucleosidase%3A+Analysis+of+enzymatic+activity+and+substrate+specificity&doi=10.1006%2Fbbrc.1996.1723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of recombinant Escherichia coli 5'-methylthioadenosine/S-adenosylhomocysteine nucleosidase: analysis of enzymic activity and substrate specificity</span></div><div class="casAuthors">Cornell, Kenneth A.; Swarts, William E.; Barry, Ronald D.; Riscoe, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">228</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">724-732</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Recombinant E. coli 5'-methylthioadenosine (MTA)/S-adenosylhomocysteine nucleosidase (EC 3.2.2.9) (I) was used to study the potential for this enzyme to serve as a target for chemotherapeutic intervention.  An examn. of the parameters required for enzymic activity indicated that I functions over a broad range of pH and temp., with acidic conditions and temps. of 37-45° being optimal.  Analogs of MTA and adenosine were assessed as potential enzyme inhibitors and to provide details regarding substrate specificity and reaction mechanism.  The 5'-arylthio analog, 5'-(p-nitrophenyl)thioadenosine, was the most potent enzyme inhibitor studied, with a Ki of 20 nM.  A mutant of I lacking the 1st 8 amino acids was engineered to det. the contribution of these conserved residues toward enzyme specificity.  Truncated I exhibited a Km[MTA] of 1.43 μM, ∼3-fold higher than the Km reported for full-length I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-cVJrNdqwkrVg90H21EOLACvtfcHk0ljhdaFVtr3tKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XntVaitr4%253D&md5=8bedc21db3990089434f199bfb600594</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1996.1723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1996.1723%26sid%3Dliteratum%253Aachs%26aulast%3DCornell%26aufirst%3DK.%2BA.%26aulast%3DSwarts%26aufirst%3DW.%2BE.%26aulast%3DBarry%26aufirst%3DR.%2BD.%26aulast%3DRiscoe%26aufirst%3DM.%2BK.%26atitle%3DCharacterization%2520of%2520recombinant%2520Escherichia%2520coli%25205%25E2%2580%25B2-methylthioadenosine%252FS-adenosylhomocysteine%2520nucleosidase%253A%2520Analysis%2520of%2520enzymatic%2520activity%2520and%2520substrate%2520specificity%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1996%26volume%3D228%26spage%3D724%26epage%3D732%26doi%3D10.1006%2Fbbrc.1996.1723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunkara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goff, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Lanen, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorson, J. S.</span></span> <span> </span><span class="NLM_article-title">Facile chemoenzymatic strategies for the synthesis and utilization of <i>S</i>-adenosyl-L-methionine analogues</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">3965</span>– <span class="NLM_lpage">3969</span>, <span class="refDoi"> DOI: 10.1002/anie.201308272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fanie.201308272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkt1Sqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=3965-3969&author=S.+Singhauthor=J.+Zhangauthor=T.+D.+Huberauthor=M.+Sunkaraauthor=K.+Hurleyauthor=R.+D.+Goffauthor=G.+Wangauthor=W.+Zhangauthor=C.+Liuauthor=J.+Rohrauthor=S.+G.+Van+Lanenauthor=A.+J.+Morrisauthor=J.+S.+Thorson&title=Facile+chemoenzymatic+strategies+for+the+synthesis+and+utilization+of+S-adenosyl-L-methionine+analogues&doi=10.1002%2Fanie.201308272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Facile Chemoenzymatic Strategies for the Synthesis and Utilization of S-Adenosyl-L-Methionine Analogues</span></div><div class="casAuthors">Singh, Shanteri; Zhang, Jianjun; Huber, Tyler D.; Sunkara, Manjula; Hurley, Katherine; Goff, Randal D.; Wang, Guojun; Zhang, Wen; Liu, Chunming; Rohr, Juergen; Van Lanen, Steven G.; Morris, Andrew J.; Thorson, Jon S.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3965-3969</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A chemoenzymic platform for the synthesis of S-adenosyl-L-methionine (SAM) analogs compatible with downstream SAM-utilizing enzymes is reported.  Forty-four non-native S/Se-alkylated Met analogs were synthesized and applied to probing the substrate specificity of five diverse methionine adenosyltransferases (MATs).  Human MAT II was among the most permissive of the MATs analyzed and enabled the chemoenzymic synthesis of 29 non-native SAM analogs.  As a proof of concept for the feasibility of natural product "alkylrandomization", a small set of differentially-alkylated indolocarbazole analogs was generated by using a coupled hMAT2-RebM system (RebM is the sugar C4'-O-methyltransferase that is involved in rebeccamycin biosynthesis).  The ability to couple SAM synthesis and utilization in a single vessel circumvents issues assocd. with the rapid decompn. of SAM analogs and thereby opens the door for the further interrogation of a wide range of SAM utilizing enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6Qa-txIq8pbVg90H21EOLACvtfcHk0liRzIgg2rglcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkt1Sqtr0%253D&md5=25c1dd4656cb37918790d35e588a91bf</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fanie.201308272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201308272%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DHuber%26aufirst%3DT.%2BD.%26aulast%3DSunkara%26aufirst%3DM.%26aulast%3DHurley%26aufirst%3DK.%26aulast%3DGoff%26aufirst%3DR.%2BD.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DRohr%26aufirst%3DJ.%26aulast%3DVan%2BLanen%26aufirst%3DS.%2BG.%26aulast%3DMorris%26aufirst%3DA.%2BJ.%26aulast%3DThorson%26aufirst%3DJ.%2BS.%26atitle%3DFacile%2520chemoenzymatic%2520strategies%2520for%2520the%2520synthesis%2520and%2520utilization%2520of%2520S-adenosyl-L-methionine%2520analogues%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D3965%26epage%3D3969%26doi%3D10.1002%2Fanie.201308272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Presolski, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, M. G.</span></span> <span> </span><span class="NLM_article-title">Copper-catalyzed azide-alkyne click chemistry for bioconjugation</span>. <i>Curr. Protoc. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">153</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1002/9780470559277.ch110148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2F9780470559277.ch110148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=22844652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A280%3ADC%252BC2srmslKltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=153-162&author=S.+I.+Presolskiauthor=V.+P.+Hongauthor=M.+G.+Finn&title=Copper-catalyzed+azide-alkyne+click+chemistry+for+bioconjugation&doi=10.1002%2F9780470559277.ch110148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Copper-Catalyzed Azide-Alkyne Click Chemistry for Bioconjugation</span></div><div class="casAuthors">Presolski Stanislav I; Hong Vu Phong; Finn M G</div><div class="citationInfo"><span class="NLM_cas:title">Current protocols in chemical biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">153-162</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The copper-catalyzed azide-alkyne cycloaddition reaction is widely used for the connection of molecular entities of all sizes.  A protocol is provided here for the process with biomolecules.  Ascorbate is used as reducing agent to maintain the required cuprous oxidation state.  Since these convenient conditions produce reactive oxygen species, five equivalents of a copper-binding ligand is used with respect to metal.  The ligand both accelerates the reaction and serves as a sacrificial reductant, protecting the biomolecules from oxidation.  A procedure is also described for testing the efficiency of the reaction under desired conditions for purposes of optimization, before expensive biological reagents are used.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR8A2KT2nosgEg6wdEZmT2rfW6udTcc2eYANqCzQcUZsLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srmslKltg%253D%253D&md5=cc622ec6ed0e16170ef2e6ebd1541d0a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2F9780470559277.ch110148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9780470559277.ch110148%26sid%3Dliteratum%253Aachs%26aulast%3DPresolski%26aufirst%3DS.%2BI.%26aulast%3DHong%26aufirst%3DV.%2BP.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26atitle%3DCopper-catalyzed%2520azide-alkyne%2520click%2520chemistry%2520for%2520bioconjugation%26jtitle%3DCurr.%2520Protoc.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D3%26spage%3D153%26epage%3D162%26doi%3D10.1002%2F9780470559277.ch110148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nanna, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walseng, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedzisa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goydel, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hymel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, T. R.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roush, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rader, C.</span></span> <span> </span><span class="NLM_article-title">Harnessing a catalytic lysine residue for the one-step preparation of homogeneous antibody-drug conjugates</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1112</span>– <span class="NLM_lpage">1121</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01257-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1038%2Fs41467-017-01257-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=29062027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A280%3ADC%252BC1M7js12hsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1112-1121&author=A.+R.+Nannaauthor=X.+Liauthor=E.+Walsengauthor=L.+Pedzisaauthor=R.+S.+Goydelauthor=D.+Hymelauthor=T.+R.+Burkeauthor=W.+R.+Roushauthor=C.+Rader&title=Harnessing+a+catalytic+lysine+residue+for+the+one-step+preparation+of+homogeneous+antibody-drug+conjugates&doi=10.1038%2Fs41467-017-01257-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing a catalytic lysine residue for the one-step preparation of homogeneous antibody-drug conjugates</span></div><div class="casAuthors">Nanna Alex R; Li Xiuling; Walseng Even; Goydel Rebecca S; Rader Christoph; Nanna Alex R; Pedzisa Lee; Roush William R; Hymel David; Burke Terrence R Jr</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1112</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Current strategies to produce homogeneous antibody-drug conjugates (ADCs) rely on mutations or inefficient conjugation chemistries.  Here we present a strategy to produce site-specific ADCs using a highly reactive natural buried lysine embedded in a dual variable domain (DVD) format.  This approach is mutation free and drug conjugation proceeds rapidly at neutral pH in a single step without removing any charges.  The conjugation chemistry is highly robust, enabling the use of crude DVD for ADC preparation.  In addition, this strategy affords the ability to precisely monitor the efficiency of drug conjugation with a catalytic assay.  ADCs targeting HER2 were prepared and demonstrated to be highly potent and specific in vitro and in vivo.  Furthermore, the modular DVD platform was used to prepare potent and specific ADCs targeting CD138 and CD79B, two clinically established targets overexpressed in multiple myeloma and non-Hodgkin lymphoma, respectively.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9zxYb5ouxkFVwdslNBF-tfW6udTcc2eYANqCzQcUZsLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7js12hsA%253D%253D&md5=a9f2f5bc6d4a7fbb9595ca06d784868a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01257-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01257-1%26sid%3Dliteratum%253Aachs%26aulast%3DNanna%26aufirst%3DA.%2BR.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWalseng%26aufirst%3DE.%26aulast%3DPedzisa%26aufirst%3DL.%26aulast%3DGoydel%26aufirst%3DR.%2BS.%26aulast%3DHymel%26aufirst%3DD.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26aulast%3DRoush%26aufirst%3DW.%2BR.%26aulast%3DRader%26aufirst%3DC.%26atitle%3DHarnessing%2520a%2520catalytic%2520lysine%2520residue%2520for%2520the%2520one-step%2520preparation%2520of%2520homogeneous%2520antibody-drug%2520conjugates%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1112%26epage%3D1121%26doi%3D10.1038%2Fs41467-017-01257-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilchan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanna, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roush, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rader, C.</span></span> <span> </span><span class="NLM_article-title">Site-selective antibody functionalization via orthogonally reactive arginine and lysine residues</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1229</span>– <span class="NLM_lpage">1239</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2Fj.chembiol.2019.05.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=31231031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1WqtL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=1229-1239&author=D.+Hwangauthor=N.+Nilchanauthor=A.+R.+Nannaauthor=X.+Liauthor=M.+D.+Cameronauthor=W.+R.+Roushauthor=H.+Parkauthor=C.+Rader&title=Site-selective+antibody+functionalization+via+orthogonally+reactive+arginine+and+lysine+residues&doi=10.1016%2Fj.chembiol.2019.05.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Site-selective antibody functionalization via orthogonally reactive arginine and lysine residues</span></div><div class="casAuthors">Hwang, Dobeen; Nilchan, Napon; Nanna, Alex R.; Li, Xiaohai; Cameron, Michael D.; Roush, William R.; Park, HaJeung; Rader, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1229-1239.e9</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Homogeneous antibody-drug conjugates (ADCs) that use a highly reactive buried lysine (Lys) residue embedded in a dual variable domain (DVD)-IgG1 format can be assembled with high precision and efficiency under mild conditions.  Here we show that replacing the Lys with an arginine (Arg) residue affords an orthogonal ADC assembly that is site-selective and stable.  X-ray crystallog. confirmed the location of the reactive Arg residue at the bottom of a deep pocket.  As the Lys-to-Arg mutation is confined to a single residue in the heavy chain of the DVD-IgG1, heterodimeric assemblies that combine a buried Lys in one arm, a buried Arg in the other arm, and identical light chains, are readily assembled.  Furthermore, the orthogonal conjugation chem. enables the loading of heterodimeric DVD-IgG1s with two different cargos in a one-pot reaction and thus affords a convenient platform for dual-warhead ADCs and other multifaceted antibody conjugates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7QYOXv5kWkrVg90H21EOLACvtfcHk0lglpBR0VGNgoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1WqtL%252FO&md5=fc7b7f5ab440e03d220b224ad305a612</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DHwang%26aufirst%3DD.%26aulast%3DNilchan%26aufirst%3DN.%26aulast%3DNanna%26aufirst%3DA.%2BR.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DRoush%26aufirst%3DW.%2BR.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DRader%26aufirst%3DC.%26atitle%3DSite-selective%2520antibody%2520functionalization%2520via%2520orthogonally%2520reactive%2520arginine%2520and%2520lysine%2520residues%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D1229%26epage%3D1239%26doi%3D10.1016%2Fj.chembiol.2019.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetsch, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumontet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corvaïa, N.</span></span> <span> </span><span class="NLM_article-title">Strategies and challenges for the next generation of antibody-drug conjugates</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">315</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.268</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1038%2Fnrd.2016.268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=28303026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1Onsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=315-337&author=A.+Beckauthor=L.+Goetschauthor=C.+Dumontetauthor=N.+Corva%C3%AFa&title=Strategies+and+challenges+for+the+next+generation+of+antibody-drug+conjugates&doi=10.1038%2Fnrd.2016.268"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies and challenges for the next generation of antibody-drug conjugates</span></div><div class="casAuthors">Beck, Alain; Goetsch, Liliane; Dumontet, Charles; Corvaia, Nathalie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">315-337</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Antibody-drug conjugates (ADCs) are one of the fastest growing classes of oncol. therapeutics.  After half a century of research, the approvals of brentuximab vedotin (in 2011) and trastuzumab emtansine (in 2013) have paved the way for ongoing clin. trials that are evaluating more than 60 further ADC candidates.  The limited success of first-generation ADCs (developed in the early 2000s) informed strategies to bring second-generation ADCs to the market, which have higher levels of cytotoxic drug conjugation, lower levels of naked antibodies and more-stable linkers between the drug and the antibody.  Furthermore, lessons learned during the past decade are now being used in the development of third-generation ADCs.  In this Review, we discuss strategies to select the best target antigens as well as suitable cytotoxic drugs; the design of optimized linkers; the discovery of bioorthogonal conjugation chemistries; and toxicity issues.  The selection and engineering of antibodies for site-specific drug conjugation, which will result in higher homogeneity and increased stability, as well as the quest for new conjugation chemistries and mechanisms of action, are priorities in ADC research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP7lV63Ws8rbVg90H21EOLACvtfcHk0lglpBR0VGNgoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1Onsbc%253D&md5=7358b6080dd1aed1d21e98c078cd9ef1</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.268%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DA.%26aulast%3DGoetsch%26aufirst%3DL.%26aulast%3DDumontet%26aufirst%3DC.%26aulast%3DCorva%25C3%25AFa%26aufirst%3DN.%26atitle%3DStrategies%2520and%2520challenges%2520for%2520the%2520next%2520generation%2520of%2520antibody-drug%2520conjugates%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D315%26epage%3D337%26doi%3D10.1038%2Fnrd.2016.268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suchand, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satyanarayana, G.</span></span> <span> </span><span class="NLM_article-title">Palladium-catalyzed direct acylation: one-pot relay synthesis of anthraquinones</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">779</span>, <span class="refDoi"> DOI: 10.1055/s-0037-1610296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1055%2Fs-0037-1610296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFSrur3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=769-779&author=B.+Suchandauthor=G.+Satyanarayana&title=Palladium-catalyzed+direct+acylation%3A+one-pot+relay+synthesis+of+anthraquinones&doi=10.1055%2Fs-0037-1610296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium-Catalyzed Direct Acylation: One-Pot Relay Synthesis of Anthraquinones</span></div><div class="casAuthors">Suchand, Basuli; Satyanarayana, Gedu</div><div class="citationInfo"><span class="NLM_cas:title">Synthesis</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">769-779</span>CODEN:
                <span class="NLM_cas:coden">SYNTBF</span>;
        ISSN:<span class="NLM_cas:issn">1437-210X</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">A bis-acylation strategy to access functionalized anthraquinones via one-pot relay process, is presented.  The first acylation was feasible under [Pd]-catalyzed intermol. direct acylation reaction, while, the second acylation was accomplished by using intramol. Friedel-Crafts acylation.  Notably, benchtop aldehydes were used as non-toxic acylation agents in the key [Pd]-catalyzed acylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMDvJyQj-cCLVg90H21EOLACvtfcHk0lglpBR0VGNgoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFSrur3L&md5=dcf159946c59a384a9b5d6be0b734965</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1055%2Fs-0037-1610296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0037-1610296%26sid%3Dliteratum%253Aachs%26aulast%3DSuchand%26aufirst%3DB.%26aulast%3DSatyanarayana%26aufirst%3DG.%26atitle%3DPalladium-catalyzed%2520direct%2520acylation%253A%2520one-pot%2520relay%2520synthesis%2520of%2520anthraquinones%26jtitle%3DSynthesis%26date%3D2019%26volume%3D51%26spage%3D769%26epage%3D779%26doi%3D10.1055%2Fs-0037-1610296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Somai
Magar, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. R.</span></span> <span> </span><span class="NLM_article-title">Organocatalyzed benzannulation for the construction of diverse anthraquinones and tetracenediones</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">8592</span>– <span class="NLM_lpage">8595</span>, <span class="refDoi"> DOI: 10.1039/C5CC00623F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1039%2FC5CC00623F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=25858160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsVWkt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=8592-8595&author=K.+B.+Somai%0AMagarauthor=L.+Xiaauthor=Y.+R.+Lee&title=Organocatalyzed+benzannulation+for+the+construction+of+diverse+anthraquinones+and+tetracenediones&doi=10.1039%2FC5CC00623F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Organocatalyzed benzannulation for the construction of diverse anthraquinones and tetracenediones</span></div><div class="casAuthors">Somai Magar, Krishna Bahadur; Xia, Likai; Lee, Yong Rok</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">8592-8595</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">An efficient one-pot synthesis of anthraquinones and tetracenediones was achieved by an L-proline catalyzed [4+2]cycloaddn. of in-situ generated azadiene from α,β-unsatd. aldehydes and 1,4-naphthoquinones or 1,4-anthracenedione in good to excellent yield.  This protocol constitutes an unprecedented tandem benzannulation that allows one-pot construction of diverse anthraquinones and tetracenediones in the presence of organocatalysts.  This methodol. was applied successfully to the synthesis of naturally occurring mols. and photochem. interesting phenanthrenequinone derivs.  The synthesis of the target compds. was achieved by a reaction of 1,4-naphthalenedione, 1,4-anthracenedione derivs. with crotonaldehyde, trans-2-hexenal, trans-2-pentenal.  The title compds. thus formed included 9,10-anthracenedione derivs. (anthraquinones), 5,12-naphthacenedione derivs. (naphthacenequinones).  9,10-Phenanthrenedione and 1,2-naphthalenedione were also prepd. from 1,2-naphthalenedione and 2,5-cyclohexadiene-1,4-dione.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmteJ58eVWJ7Vg90H21EOLACvtfcHk0lglpBR0VGNgoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsVWkt7Y%253D&md5=a319ce64bd8a9bfeb8b06f5774bd26cf</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1039%2FC5CC00623F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC00623F%26sid%3Dliteratum%253Aachs%26aulast%3DSomai%2BMagar%26aufirst%3DK.%2BB.%26aulast%3DXia%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DY.%2BR.%26atitle%3DOrganocatalyzed%2520benzannulation%2520for%2520the%2520construction%2520of%2520diverse%2520anthraquinones%2520and%2520tetracenediones%26jtitle%3DChem.%2520Commun.%26date%3D2015%26volume%3D51%26spage%3D8592%26epage%3D8595%26doi%3D10.1039%2FC5CC00623F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montero, A.</span></span> <span> </span><span class="NLM_article-title">Asymmetric synthesis and biological properties of uncialamycin and 26-epi-uncialamycin</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">189</span>, <span class="refDoi"> DOI: 10.1002/anie.200704577</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fanie.200704577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVShsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=185-189&author=K.+C.+Nicolaouauthor=J.+S.+Chenauthor=H.+Zhangauthor=A.+Montero&title=Asymmetric+synthesis+and+biological+properties+of+uncialamycin+and+26-epi-uncialamycin&doi=10.1002%2Fanie.200704577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Asymmetric synthesis and biological properties of uncialamycin and 26-epi-uncialamycin</span></div><div class="casAuthors">Nicolaou, K. C.; Chen, Jason S.; Zhang, Hongjun; Montero, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">185-189</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The highly potent DNA-cleaving mol. uncialamycin (I) was prepd. in an asym. total synthesis featuring an enantioselective Noyori redn.  Compd. I and its C26 epimer exhibit impressive broad-spectrum antibacterial properties and highly potent antitumor activities against a variety of cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVaG-7C5oaD7Vg90H21EOLACvtfcHk0licjjMsptgrkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVShsw%253D%253D&md5=7e1d386d727427ceb611065c06d21d26</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fanie.200704577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200704577%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26aulast%3DChen%26aufirst%3DJ.%2BS.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DMontero%26aufirst%3DA.%26atitle%3DAsymmetric%2520synthesis%2520and%2520biological%2520properties%2520of%2520uncialamycin%2520and%252026-epi-uncialamycin%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2008%26volume%3D47%26spage%3D185%26epage%3D189%26doi%3D10.1002%2Fanie.200704577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasband, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliceiri, K. W.</span></span> <span> </span><span class="NLM_article-title">NIH Image to ImageJ: 25 years of image analysis</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">671</span>– <span class="NLM_lpage">675</span>, <span class="refDoi"> DOI: 10.1038/nmeth.2089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1038%2Fnmeth.2089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=22930834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVKntb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=671-675&author=C.+A.+Schneiderauthor=W.+S.+Rasbandauthor=K.+W.+Eliceiri&title=NIH+Image+to+ImageJ%3A+25+years+of+image+analysis&doi=10.1038%2Fnmeth.2089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">NIH Image to ImageJ: 25 years of image analysis</span></div><div class="casAuthors">Schneider, Caroline A.; Rasband, Wayne S.; Eliceiri, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7_part1</span>),
    <span class="NLM_cas:pages">671-675</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">For the past 25 years NIH Image and ImageJ software have been pioneers as open tools for the anal. of scientific images.  We discuss the origins, challenges and solns. of these two programs, and how their history can serve to advise and inform other software projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofzHXrdLwDn7Vg90H21EOLACvtfcHk0licjjMsptgrkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVKntb7P&md5=85ab928cd79f1e2f2351c834c0c600f0</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.2089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.2089%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DC.%2BA.%26aulast%3DRasband%26aufirst%3DW.%2BS.%26aulast%3DEliceiri%26aufirst%3DK.%2BW.%26atitle%3DNIH%2520Image%2520to%2520ImageJ%253A%252025%2520years%2520of%2520image%2520analysis%26jtitle%3DNat.%2520Methods%26date%3D2012%26volume%3D9%26spage%3D671%26epage%3D675%26doi%3D10.1038%2Fnmeth.2089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Guang-Lei Ma, Hoa Thi Tran, Zhen Jie Low, Hartono Candra, Li Mei Pang, Qing Wei Cheang, Mingliang Fang, <span class="NLM_string-name hlFld-ContribAuthor">Zhao-Xun Liang</span>. </span><span class="cited-content_cbyCitation_article-title">Pathway Retrofitting Yields Insights into the Biosynthesis of Anthraquinone-Fused Enediynes. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2021,</strong> <em>143 </em>
                                    (30)
                                     , 11500-11509. <a href="https://doi.org/10.1021/jacs.1c03911" title="DOI URL">https://doi.org/10.1021/jacs.1c03911</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.1c03911&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.1c03911%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DPathway%252BRetrofitting%252BYields%252BInsights%252Binto%252Bthe%252BBiosynthesis%252Bof%252BAnthraquinone-Fused%252BEnediynes%26aulast%3DMa%26aufirst%3DGuang-Lei%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D14042021%26date%3D23072021%26volume%3D143%26issue%3D30%26spage%3D11500%26epage%3D11509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Thibault Annaval, Christiana N. Teijaro, Ajeeth Adhikari, Xiaohui Yan, Jian-Jun Chen, Ivana Crnovcic, Dong Yang, <span class="NLM_string-name hlFld-ContribAuthor">Ben Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Cytochrome P450 Hydroxylase TnmL Catalyzing Sequential Hydroxylation with an Additional Proofreading Activity in Tiancimycin Biosynthesis. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (7)
                                     , 1172-1178. <a href="https://doi.org/10.1021/acschembio.1c00365" title="DOI URL">https://doi.org/10.1021/acschembio.1c00365</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.1c00365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.1c00365%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DCytochrome%252BP450%252BHydroxylase%252BTnmL%252BCatalyzing%252BSequential%252BHydroxylation%252Bwith%252Ban%252BAdditional%252BProofreading%252BActivity%252Bin%252BTiancimycin%252BBiosynthesis%26aulast%3DAnnaval%26aufirst%3DThibault%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D17052021%26date%3D15062021%26date%3D17062021%26volume%3D16%26issue%3D7%26spage%3D1172%26epage%3D1178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hindra</span>, <span class="hlFld-ContribAuthor ">Dong  Yang</span>, <span class="hlFld-ContribAuthor ">Jun  Luo</span>, <span class="hlFld-ContribAuthor ">Tingting  Huang</span>, <span class="hlFld-ContribAuthor ">Xiaohui  Yan</span>, <span class="hlFld-ContribAuthor ">Ajeeth  Adhikari</span>, <span class="hlFld-ContribAuthor ">Christiana N  Teijaro</span>, <span class="hlFld-ContribAuthor ">Huiming  Ge</span>, <span class="hlFld-ContribAuthor ">Ben  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Submerged fermentation of
              Streptomyces uncialis
              providing a biotechnology platform for uncialamycin biosynthesis, engineering, and production. </span><span class="cited-content_cbyCitation_journal-name">Journal of Industrial Microbiology and Biotechnology</span><span> <strong>2021,</strong> <em>48 </em>
                                    (3-4)
                                     <a href="https://doi.org/10.1093/jimb/kuab025" title="DOI URL">https://doi.org/10.1093/jimb/kuab025</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/jimb/kuab025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fjimb%2Fkuab025%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Industrial%2520Microbiology%2520and%2520Biotechnology%26atitle%3DSubmerged%252Bfermentation%252Bof%252BStreptomyces%252Buncialis%252Bproviding%252Ba%252Bbiotechnology%252Bplatform%252Bfor%252Buncialamycin%252Bbiosynthesis%25252C%252Bengineering%25252C%252Band%252Bproduction%26aulast%3DHindra%26date%3D2021%26date%3D2021%26volume%3D48%26issue%3D3-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elnaz  Jalali</span>, <span class="hlFld-ContribAuthor ">Jon S  Thorson</span>. </span><span class="cited-content_cbyCitation_article-title">Enzyme-mediated bioorthogonal technologies: catalysts, chemoselective reactions and recent methyltransferase applications. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Biotechnology</span><span> <strong>2021,</strong> <em>69 </em>, 290-298. <a href="https://doi.org/10.1016/j.copbio.2021.02.010" title="DOI URL">https://doi.org/10.1016/j.copbio.2021.02.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.copbio.2021.02.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.copbio.2021.02.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Biotechnology%26atitle%3DEnzyme-mediated%252Bbioorthogonal%252Btechnologies%25253A%252Bcatalysts%25252C%252Bchemoselective%252Breactions%252Band%252Brecent%252Bmethyltransferase%252Bapplications%26aulast%3DJalali%26aufirst%3DElnaz%26date%3D2021%26volume%3D69%26spage%3D290%26epage%3D298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ajeeth  Adhikari</span>, <span class="hlFld-ContribAuthor ">Ben  Shen</span>, <span class="hlFld-ContribAuthor ">Christoph  Rader</span>. </span><span class="cited-content_cbyCitation_article-title">Challenges and opportunities to develop enediyne natural products as payloads for antibody-drug conjugates. </span><span class="cited-content_cbyCitation_journal-name">Antibody Therapeutics</span><span> <strong>2021,</strong> <em>4 </em>
                                    (1)
                                     , 1-15. <a href="https://doi.org/10.1093/abt/tbab001" title="DOI URL">https://doi.org/10.1093/abt/tbab001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/abt/tbab001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fabt%2Ftbab001%26sid%3Dliteratum%253Aachs%26jtitle%3DAntibody%2520Therapeutics%26atitle%3DChallenges%252Band%252Bopportunities%252Bto%252Bdevelop%252Benediyne%252Bnatural%252Bproducts%252Bas%252Bpayloads%252Bfor%252Bantibody-drug%252Bconjugates%26aulast%3DAdhikari%26aufirst%3DAjeeth%26date%3D2021%26date%3D2021%26volume%3D4%26issue%3D1%26spage%3D1%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/medium/jm0c00799_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00799&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/medium/jm0c00799_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of enediynes used clinically for targeted delivery and selected anthraquinone-fused enediynes: (A) neocarzinostatin chromophore (NCS, <b>1</b>) and a hydrazide-derivative of calicheamicin (CAL-DMH, <b>2</b>) used in the clinic for targeted delivery; (B) selected anthraquinone-fused enediynes tiancimycin A (<b>3</b>), uncialamycin (<b>4</b>) and amino-UCM (<b>5</b>), yangpumicin A (<b>6</b>), and dynemicin A (<b>7</b>), highlighting the varying substitutions on the A-ring. Active handles introduced by medicinal chemistry or total synthesis to facilitate antibody–drug conjugation, as exemplified by <b>2</b> or <b>5</b>, respectively, are shown in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00799&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/medium/jm0c00799_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Unified biosynthetic pathway for the anthraquinone-fused enediynes and the post-enediyne polyketide synthase tailoring steps for TNM A (<b>3</b>) biosynthesis as revealed by the isolation of TNM B (<b>8</b>) and TNM E (<b>9</b>), from the ΔtnmL mutant strain S. sp. SB20020, TNM F (<b>10</b>) and TNM C (<b>11</b>), from the ΔtnmH mutant strain <i>S</i>. sp. SB20002, and TNM D (<b>13</b>), as well as shunt metabolites supporting the intermediacy of TNM G (<b>12</b>), from wild-type <i>S</i>. sp. CB03234. Paths I and II highlight different timing of TnmL in installing the hydroxyl groups at C-6 and C-7 and of TnmH in installing the methyl group at the hydroxyl group of C-7, respectively, and in vitro characterization of TnmH in the current study supports path I.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00799&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/medium/jm0c00799_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Second generation Δ<i>tnmH</i> mutant strain <i>S</i>. sp. SB20024 provides a reliable supply of TNM C (<b>11</b>). HPLC analysis of (A) the original Δ<i>tnmH</i> mutant strain <i>S</i>. sp. SB20002 producing both TNM F (<b>10</b>) and TNM C (<b>11</b>) (at a combined isolated yield of ∼0.4 mg/L) co-produced with the tiancilactones and (B) the second generation Δ<i>tnmH</i> mutant strain <i>S</i>. sp. SB20024 producing TNM C (<b>11</b>) as the sole metabolite (at a titer of ∼4 mg/L) free from the tiancilactones. While UV–vis at 254 nm detects both the TNMs and tiancilactones, UV–vis at 539 nm detects the TNMs only. TNM F (<b>10</b>) (●); TNM C (<b>11</b>) (◆); tiancilactone A (▽); tiancilactone B (■).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00799&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/medium/jm0c00799_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. TnmH as a SAM-dependent <i>O</i>-methyltransferase catalyzes the regiospecific methylation of the hydroxyl group at C-7 of TNM F (<b>10</b>) and TNM C (<b>11</b>) in vitro. (A) HPLC analyses of TnmH-catalyzed reaction in the presence of SAM with TNM F (<b>10</b>) as a substrate: (I) an authentic standard of <b>10</b>; (II) complete reaction with boiled TnmH; (III) complete reaction at 28 <i>°</i>C for 20 min; (IV) an authentic standard of <b>12</b>. (B) HPLC analyses of TnmH-catalyzed reaction in the presence of SAM with TNM C (<b>11</b>) as a substrate: (I) an authentic standard of <b>11</b>; (II) complete reaction with boiled TnmH; (III) complete reaction at 28 <i>°</i>C for 5 min; (IV) an authentic standard of <b>13</b>. TNM F (<b>10</b>) (●); TNM G (<b>12</b>) (○); TNM C (<b>11</b>) (◆); TNM D (<b>13</b>) (◇).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00799&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/medium/jm0c00799_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystal structures of TnmH (PDB entries <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CLW">6CLW</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CLX">6CLX</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BBX">6BBX</a>). (A) Overall structure of TnmH homodimer in complex with SAM (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CLX">6CLX</a>). (B) Model of TNM C (<b>11</b>) docked into the X-ray structure of holo-TnmH in complex with SAM (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BBX">6BBX</a>), revealing a large active site pocket. SAM and the surface of <b>11</b>, shown in purple, and the active site residues His246 and Asp247 are labeled. (C) Close-up view of the <i>S</i>-methyl group of SAM seen in the holo-TnmH structure, with proximal residues labeled and colored by b-factor according to the color key shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00799&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/medium/jm0c00799_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Leveraging the TnmH-installed reactive alkyl handles at C-7 of TNMs to develop novel conjugation strategies to prepare antibody–TNM conjugates as exemplified by the synthesis of <b>27</b>, <b>28</b>, <b>29</b> from <b>11</b> and subsequent preparation of <b>31</b> from <b>27</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00799/20200806/images/large/jm0c00799_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00799&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i26">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01486" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01486" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 51 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, A.</span>; <span class="NLM_string-name">Teijaro, C. N.</span>; <span class="NLM_string-name">Townsend, C. A.</span>; <span class="NLM_string-name">Shen, B.</span></span> <span> </span><span class="NLM_article-title">Biosynthesis of Enediyne Natural Products</span>. In  <i>Comprehensive Natural Products III</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. W.</span>, <span class="NLM_string-name">Begley, T.</span></span>, Eds.; <span class="NLM_publisher-name">Elsevier</span>, <span class="NLM_year">2020</span>; Vol.  <span class="NLM_volume">3</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-409547-2.14669-4</span> , in press.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2FB978-0-12-409547-2.14669-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=A.+Adhikari&author=C.+N.+Teijaro&author=C.+A.+Townsend&author=B.+Shenauthor=H.+W.+Liu&author=T.+Begley&title=Comprehensive+Natural+Products+III"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-409547-2.14669-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-409547-2.14669-4%26sid%3Dliteratum%253Aachs%26aulast%3DAdhikari%26aufirst%3DA.%26atitle%3DBiosynthesis%2520of%2520Enediyne%2520Natural%2520Products%26btitle%3DComprehensive%2520Natural%2520Products%2520III%26aulast%3DLiu%26aufirst%3DH.%2BW.%26pub%3DElsevier%26date%3D2020%26volume%3D3%26doi%3D10.1016%2FB978-0-12-409547-2.14669-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W. M.</span></span> <span> </span><span class="NLM_article-title">Chemistry and biology of the enediyne anticancer antibiotics</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1387</span>– <span class="NLM_lpage">1416</span>, <span class="refDoi"> DOI: 10.1002/anie.199113873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fanie.199113873" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1991&pages=1387-1416&author=K.+C.+Nicolaouauthor=W.+M.+Dai&title=Chemistry+and+biology+of+the+enediyne+anticancer+antibiotics&doi=10.1002%2Fanie.199113873"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fanie.199113873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.199113873%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26aulast%3DDai%26aufirst%3DW.%2BM.%26atitle%3DChemistry%2520and%2520biology%2520of%2520the%2520enediyne%2520anticancer%2520antibiotics%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D1991%26volume%3D30%26spage%3D1387%26epage%3D1416%26doi%3D10.1002%2Fanie.199113873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, H.</span></span> <span> </span><span class="NLM_article-title">SMANCS and polymer-conjugated macromolecular drugs: advantages in cancer chemotherapy</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(00)00134-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2FS0169-409X%2800%2900134-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=11259839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=169-185&author=H.+Maeda&title=SMANCS+and+polymer-conjugated+macromolecular+drugs%3A+advantages+in+cancer+chemotherapy&doi=10.1016%2FS0169-409X%2800%2900134-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">SMANCS and polymer-conjugated macromolecular drugs: advantages in cancer chemotherapy</span></div><div class="casAuthors">Maeda, H.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">169-185</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">This review, with 66 refs., discusses the development and therapeutic potential of prototype macromol. drugs for use in cancer chemotherapy, in particular the development and use of SMANCS, a conjugate of neocarzinostatin (NCS) and poly(styrene-co-maleic acid).  The various topics covered include a brief description of the chem. and polymer conjugation, the binding of the conjugate to albumin and the biol. behavior in vitro and in vivo after arterial injection in animals, including plasma half-life, and the lipid soly. of SMANCS in medium chain triglycerides and Lipiodol, a lipid contrast medium suitable for use in x-ray-computed tomog.  The biol. response-modifying effects and the tumor-targeting mechanism of SMANCS and other macromol. drugs are also discussed.  The latter mechanism is accounted for in terms of a tumor 'enhanced permeability and retention' (or EPR) effect.  A principal advantage in the use of SMANCS or other macromol. drugs is the potential for a redn. or elimination of toxicity.  Macromol. drugs such as a pyran copolymer-NCS conjugate show a marked redn. in bone marrow toxicity normally assocd. with the use of NCS.  This is believed to be due to a hypothetical blood-bone marrow 'barrier' which, relative to NCS, restricts or limits access of the macromol. drug to the bone marrow.  In addn., the clin. possibilities for SMANCS are discussed, including the suggestion that angiotensin II-induced hypertension has clin. potential in improving the selective delivery of macromol. drugs (i.e. SMANCS) to tumors.  Aq. SMANCS formulations have been tested in pilot studies in patients with solid tumors of the ovary, esophagus, lung, stomach, adrenal gland and in the brain.  Formulations based on SMANCS/Lipiodol have been shown to be effective both as a diagnostic tool and for therapeutic use in solid tumors where the formulations are given arterially via a catheter.  In a pilot study in primary unresectable hepatoma, an objective redn. in tumor size was obsd. for about 90% of cases when an adequate amt. of the macromol. drug was administered.  A patient receiving such treatment with no active liver cirrhosis and tumor nodules/lesion confined within one liver segment might expect to have a 90% chance of survival after treatment for at least 5 yr.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqodKBCU3HvbVg90H21EOLACvtfcHk0lh2F0ao4Le_2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhsL0%253D&md5=c0cf1d27fa22dc32aa56d258b681206e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900134-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900134-4%26sid%3Dliteratum%253Aachs%26aulast%3DMaeda%26aufirst%3DH.%26atitle%3DSMANCS%2520and%2520polymer-conjugated%2520macromolecular%2520drugs%253A%2520advantages%2520in%2520cancer%2520chemotherapy%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D169%26epage%3D185%26doi%3D10.1016%2FS0169-409X%2800%2900134-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinman, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollander, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beyer, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holcomb, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallett, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsou, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upeslacis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shochat, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mountain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flowers, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, I.</span></span> <span> </span><span class="NLM_article-title">Gemtuzumab ozogamicin, a potent and selective anti-CD33 antibody-calicheamicin conjugate for treatment of acute myeloid leukemia</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1021/bc010021y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bc010021y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD3MXptFanurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2002&pages=47-58&author=P.+R.+Hamannauthor=L.+M.+Hinmanauthor=I.+Hollanderauthor=C.+F.+Beyerauthor=D.+Lindhauthor=R.+Holcombauthor=W.+Hallettauthor=H.+R.+Tsouauthor=J.+Upeslacisauthor=D.+Shochatauthor=A.+Mountainauthor=D.+A.+Flowersauthor=I.+Bernstein&title=Gemtuzumab+ozogamicin%2C+a+potent+and+selective+anti-CD33+antibody-calicheamicin+conjugate+for+treatment+of+acute+myeloid+leukemia&doi=10.1021%2Fbc010021y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Gemtuzumab Ozogamicin, A Potent and Selective Anti-CD33 Antibody-Calicheamicin Conjugate for Treatment of Acute Myeloid Leukemia</span></div><div class="casAuthors">Hamann, Philip R.; Hinman, Lois M.; Hollander, Irwin; Beyer, Carl F.; Lindh, Delores; Holcomb, Ryan; Hallett, William; Tsou, Hwei-Ru; Upeslacis, Janis; Shochat, Dan; Mountain, Andrew; Flowers, David A.; Bernstein, Irwin</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-58</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">CD33 is expressed by acute myeloid leukemia (AML) cells in >80% of patients but not by normal hematopoietic stem cells, suggesting that elimination of CD33+ cells may be therapeutically beneficial.  A conjugate of a calicheamicin hydrazide deriv. attached via hydrazone formation to the oxidized carbohydrates of the anti-CD33 murine antibody P67.6 had been chosen for use in AML prior to humanization of this antibody.  However, the CDR-grafted humanized P67.6 could not be used to make the carbohydrate conjugate because of the unexpected sensitivity of this antibody to periodate oxidn.  Exploration of a series of bifunctional linkers resulted in a new class of calicheamicin conjugates, termed the hybrid conjugates, that allows for the attachment of the calicheamicin to lysines but incorporates the site of hydrolytic release, a hydrazone, previously shown to be required for activity.  The optimized conjugate chosen for clin. trials, gemtuzumab ozogamicin ("gem-ozo", Mylotarg, formerly designated CMA-676), was significantly more potent and selective than the carbohydrate conjugate it replaced.  It was selectively cytotoxic to HL-60 leukemia cells in tissue culture with an IC50 in the low to sub-pg cal/mL range (cal = calicheamicin equiv.).  Doses of gem-ozo as low as 50 μg cal/kg given three times to mice bearing HL-60 xenografts routinely resulted in long-term, tumor-free survivors, while a nonbinding control conjugate was relatively inactive.  Gem-ozo at a concn. of 2 to 10 ng cal/mL selectively inhibited leukemia colony formation by marrow cells from a significant proportion of AML patients.  Gem-ozo has also shown significant activity against AML in Phase II trials and is the first antibody-targeted chemotherapeutic agent approved by the FDA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUzPmqKMIOx7Vg90H21EOLACvtfcHk0lh2F0ao4Le_2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXptFanurg%253D&md5=56f273f4975fa6ebfa81e28f23111e8a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fbc010021y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbc010021y%26sid%3Dliteratum%253Aachs%26aulast%3DHamann%26aufirst%3DP.%2BR.%26aulast%3DHinman%26aufirst%3DL.%2BM.%26aulast%3DHollander%26aufirst%3DI.%26aulast%3DBeyer%26aufirst%3DC.%2BF.%26aulast%3DLindh%26aufirst%3DD.%26aulast%3DHolcomb%26aufirst%3DR.%26aulast%3DHallett%26aufirst%3DW.%26aulast%3DTsou%26aufirst%3DH.%2BR.%26aulast%3DUpeslacis%26aufirst%3DJ.%26aulast%3DShochat%26aufirst%3DD.%26aulast%3DMountain%26aufirst%3DA.%26aulast%3DFlowers%26aufirst%3DD.%2BA.%26aulast%3DBernstein%26aufirst%3DI.%26atitle%3DGemtuzumab%2520ozogamicin%252C%2520a%2520potent%2520and%2520selective%2520anti-CD33%2520antibody-calicheamicin%2520conjugate%2520for%2520treatment%2520of%2520acute%2520myeloid%2520leukemia%26jtitle%3DBioconjugate%2520Chem.%26date%3D2002%26volume%3D13%26spage%3D47%26epage%3D58%26doi%3D10.1021%2Fbc010021y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, Y. N.</span></span> <span> </span><span class="NLM_article-title">Inotuzumab ozogamicin: first global approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">1603</span>– <span class="NLM_lpage">1610</span>, <span class="refDoi"> DOI: 10.1007/s40265-017-0802-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1007%2Fs40265-017-0802-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=28819740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlGktL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=1603-1610&author=Y.+N.+Lamb&title=Inotuzumab+ozogamicin%3A+first+global+approval&doi=10.1007%2Fs40265-017-0802-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inotuzumab Ozogamicin: First Global Approval</span></div><div class="casAuthors">Lamb, Yvette N.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1603-1610</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">I.v. inotuzumab ozogamicin (Besponsa; Pfizer) is an anti-CD22 monoclonal antibody-calicheamicin conjugate that binds to CD22-expressing tumor cells.  Upon binding, the complex is internalized and the cytotoxic calicheamicin deriv. is released inside the cell, inducing double-strand DNA breakage and subsequent cell death.  In June 2017, the EMA granted inotuzumab ozogamicin approval as monotherapy for the treatment of adults with relapsed or refractory CD22-pos. B-cell precursor acute lymphoblastic leukemia (ALL).  The use of inotuzumab ozogamicin in adult patients with Philadelphia chromosome-pos., relapsed or refractory CD22-pos. B-cell precursor ALL is restricted to those who have failed treatment with at least one tyrosine kinase inhibitor.  Inotuzumab ozogamicin was granted priority review for the treatment of relapsed or refractory B-cell precursor ALL by the US FDA in Feb. 2017.  In the USA, a phase III trial evaluating inotuzumab ozogamicin in combination with frontline chemotherapy in adults with newly diagnosed B-cell ALL has recently been initiated and inotuzumab ozogamicin is under phase II evaluation in childhood CD22-pos. B-cell ALL.  Inotuzumab ozogamicin combination therapies are also being evaluated in the phase I/II or II setting in ALL and chronic myeloid leukemia and in the phase I setting in Burkitt's lymphoma.  This article summarizes the milestones in the development of inotuzumab ozogamicin leading to this first approval for ALL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSHnoFMpPaqbVg90H21EOLACvtfcHk0lhpRnQSLvWdJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlGktL3P&md5=76c6dbf41181e769901bbc313aabdb20</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1007%2Fs40265-017-0802-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-017-0802-5%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DY.%2BN.%26atitle%3DInotuzumab%2520ozogamicin%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2017%26volume%3D77%26spage%3D1603%26epage%3D1610%26doi%3D10.1007%2Fs40265-017-0802-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hindra</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crnovcic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gansemans, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Nieuwerburgh, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rader, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span> <span> </span><span class="NLM_article-title">Strain prioritization and genome mining for enediyne natural products</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e02104-16</span> <span class="refDoi"> DOI: 10.1128/mBio.02104-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1128%2FmBio.02104-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=27999165" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&author=X.+Yanauthor=H.+Geauthor=T.+Huangauthor=+Hindraauthor=D.+Yangauthor=Q.+Tengauthor=I.+Crnovcicauthor=X.+Liauthor=J.+D.+Rudolfauthor=J.+R.+Lohmanauthor=Y.+Gansemansauthor=X.+Zhuauthor=Y.+Huangauthor=L.+X.+Zhaoauthor=Y.+Jiangauthor=F.+Van+Nieuwerburghauthor=C.+Raderauthor=Y.+Duanauthor=B.+Shen&title=Strain+prioritization+and+genome+mining+for+enediyne+natural+products&doi=10.1128%2FmBio.02104-16"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1128%2FmBio.02104-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmBio.02104-16%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DX.%26aulast%3DGe%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DT.%26aulast%3DHindra%26aulast%3DYang%26aufirst%3DD.%26aulast%3DTeng%26aufirst%3DQ.%26aulast%3DCrnovcic%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DRudolf%26aufirst%3DJ.%2BD.%26aulast%3DLohman%26aufirst%3DJ.%2BR.%26aulast%3DGansemans%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DL.%2BX.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DVan%2BNieuwerburgh%26aufirst%3DF.%26aulast%3DRader%26aufirst%3DC.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DB.%26atitle%3DStrain%2520prioritization%2520and%2520genome%2520mining%2520for%2520enediyne%2520natural%2520products%26jtitle%3DmBio%26date%3D2016%26volume%3D7%26doi%3D10.1128%2FmBio.02104-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warabi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, R. J.</span></span> <span> </span><span class="NLM_article-title">Uncialamycin, a new enediyne antibiotic</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">5233</span>– <span class="NLM_lpage">5236</span>, <span class="refDoi"> DOI: 10.1021/ol052081f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol052081f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtV2nu73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=5233-5236&author=J.+Daviesauthor=H.+Wangauthor=T.+Taylorauthor=K.+Warabiauthor=X.+H.+Huangauthor=R.+J.+Andersen&title=Uncialamycin%2C+a+new+enediyne+antibiotic&doi=10.1021%2Fol052081f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Uncialamycin, a new enediyne antibiotic</span></div><div class="casAuthors">Davies, Julian; Wang, Hao; Taylor, Terry; Warabi, Kaoru; Huang, Xin-Hui; Andersen, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5233-5236</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lab. cultures of an undescribed streptomycete obtained from the surface of a British Columbia lichen produce uncialamycin (I), a new enediyne antibiotic.  The structure of I was elucidated by anal. of spectroscopic data.  I exhibits potent in vitro antibacterial activity against Gram-pos. and Gram-neg. human pathogens, including Burkholderia cepacia, a major cause of morbidity and mortality in patients with cystic fibrosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGcJkghP1-j7Vg90H21EOLACvtfcHk0lhpRnQSLvWdJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtV2nu73E&md5=97531c57ebc3393de4b82cd871368a3f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fol052081f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol052081f%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DTaylor%26aufirst%3DT.%26aulast%3DWarabi%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DX.%2BH.%26aulast%3DAndersen%26aufirst%3DR.%2BJ.%26atitle%3DUncialamycin%252C%2520a%2520new%2520enediyne%2520antibiotic%26jtitle%3DOrg.%2520Lett.%26date%3D2005%26volume%3D7%26spage%3D5233%26epage%3D5236%26doi%3D10.1021%2Fol052081f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crnovcic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rader, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span> <span> </span><span class="NLM_article-title">Genome mining of <i>Micromonospora yangpuensis</i> DSM 45577 as a producer of an anthraquinone-fused enediyne</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">6192</span>– <span class="NLM_lpage">6195</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.7b03120</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.7b03120" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslemu7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=6192-6195&author=X.+Yanauthor=J.+J.+Chenauthor=A.+Adhikariauthor=D.+Yangauthor=I.+Crnovcicauthor=N.+Wangauthor=C.+Y.+Changauthor=C.+Raderauthor=B.+Shen&title=Genome+mining+of+Micromonospora+yangpuensis+DSM+45577+as+a+producer+of+an+anthraquinone-fused+enediyne&doi=10.1021%2Facs.orglett.7b03120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Genome Mining of Micromonospora yangpuensis DSM 45577 as a Producer of an Anthraquinone-Fused Enediyne</span></div><div class="casAuthors">Yan, Xiaohui; Chen, Jian-Jun; Adhikari, Ajeeth; Yang, Dong; Crnovcic, Ivana; Wang, Nan; Chang, Chin-Yuan; Rader, Christoph; Shen, Ben</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6192-6195</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new anthraquinone-fused enediyne, yangpumicin A (YPM A, 1), along with four Bergman cyclization congeners (YPM B-E, 2-5), was isolated from Micromonospora yangpuensis DSM 45577 after mining enediyne biosynthetic gene clusters from public actinobacterial genome databases and prioritizing the hits by an enediyne genome neighborhood network anal. for discovery.  YPM A is potent against a broad spectrum of human cancer cell lines.  The discovery of 1 provides new opportunities for the functionalization of enediynes to develop new conjugation chemistries for antibody-drug conjugates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOQFuX9dkpvLVg90H21EOLACvtfcHk0ljgqJvyx_iWZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslemu7%252FE&md5=0f2dacc312aeb2d50370f816b5f201d2</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.7b03120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.7b03120%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%2BJ.%26aulast%3DAdhikari%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DCrnovcic%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DChang%26aufirst%3DC.%2BY.%26aulast%3DRader%26aufirst%3DC.%26aulast%3DShen%26aufirst%3DB.%26atitle%3DGenome%2520mining%2520of%2520Micromonospora%2520yangpuensis%2520DSM%252045577%2520as%2520a%2520producer%2520of%2520an%2520anthraquinone-fused%2520enediyne%26jtitle%3DOrg.%2520Lett.%26date%3D2017%26volume%3D19%26spage%3D6192%26epage%3D6195%26doi%3D10.1021%2Facs.orglett.7b03120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span> <span> </span><span class="NLM_article-title">Yangpumicins F and G, enediyne congeners from <i>Micromonospora yangpuensis</i> DSM 45577</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">2483</span>– <span class="NLM_lpage">2488</span>, <span class="refDoi"> DOI: 10.1021/acs.jnatprod.9b00229</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.9b00229" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslWmsbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2019&pages=2483-2488&author=Z.+Wangauthor=Z.+Wenauthor=L.+Liuauthor=X.+Zhuauthor=B.+Shenauthor=X.+Yanauthor=Y.+Duanauthor=Y.+Huang&title=Yangpumicins+F+and+G%2C+enediyne+congeners+from+Micromonospora+yangpuensis+DSM+45577&doi=10.1021%2Facs.jnatprod.9b00229"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Yangpumicins F and G, enediyne congeners from Micromonospora yangpuensis DSM 45577</span></div><div class="casAuthors">Wang, Zilong; Wen, Zhongqing; Liu, Ling; Zhu, Xiangcheng; Shen, Ben; Yan, Xiaohui; Duan, Yanwen; Huang, Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2483-2488</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">Enediyne natural products are among the most cytotoxic small mols. and thus excellent payload candidates for the development of antibody-drug conjugates (ADCs).  Here we report the isolation and structural elucidation of two new 10-membered anthraquinone-fused enediynes, yangpumicins (YPM) F (6) and G (7), together with five known congeners, YPM A-E (1-5), from Micromonospora yangpuensis DSM 45577.  YPM F (6) and G (7) showed strong cytotoxicity against the tested human cancer cell lines, as well as activity against several Gram-pos. and Gram-neg. pathogens.  The 1,2-diols in 6 and 7 promise to enable new linker chem. for the development of YPM-based ADCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0bBiQ9l9e27Vg90H21EOLACvtfcHk0ljgqJvyx_iWZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslWmsbzN&md5=9a0cbcb8ad1411f84c26e09ca21dd121</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.9b00229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.9b00229%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWen%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DY.%26atitle%3DYangpumicins%2520F%2520and%2520G%252C%2520enediyne%2520congeners%2520from%2520Micromonospora%2520yangpuensis%2520DSM%252045577%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2019%26volume%3D82%26spage%3D2483%26epage%3D2488%26doi%3D10.1021%2Facs.jnatprod.9b00229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konishi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohkuma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanDuyne, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clardy, J.</span></span> <span> </span><span class="NLM_article-title">Dynemicin A, a novel antibiotic with the anthraquinone and 1,5-diyn-3-ene subunit</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1449</span>– <span class="NLM_lpage">1452</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.42.1449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.7164%2Fantibiotics.42.1449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=2793600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADyaK3cXjtlSgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1989&pages=1449-1452&author=M.+Konishiauthor=H.+Ohkumaauthor=K.+Matsumotoauthor=T.+Tsunoauthor=H.+Kameiauthor=T.+Miyakiauthor=T.+Okiauthor=H.+Kawaguchiauthor=G.+D.+VanDuyneauthor=J.+Clardy&title=Dynemicin+A%2C+a+novel+antibiotic+with+the+anthraquinone+and+1%2C5-diyn-3-ene+subunit&doi=10.7164%2Fantibiotics.42.1449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Dynemicin A, a novel antibiotic with the anthraquinone and 1,5-diyn-3-ene subunit</span></div><div class="casAuthors">Konishi, Masataka; Ohkuma, Hiroaki; Matsumoto, Kiyoshi; Tsuno, Takashi; Kamei, Hideo; Miyaki, Takeo; Oki, Toshikazu; Kawaguchi, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1449-52</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Dynemicin A (I) was isolated from the culture broth of Micromonospora species and its structure was detd.  It was esp. active against gram-pos. bacteria and prolonged the life span of mice inoculated with P388 leukemia.  Treating I with acetic acid anhydride yielded a triacetyl deriv. of I which was 2-5-fold as potent as I against all microbes tested.  In addn. both compds. displayed significant activity against P388 leukemia and 316 melanoma in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6sU283EH0HbVg90H21EOLACvtfcHk0ljgqJvyx_iWZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXjtlSgsw%253D%253D&md5=48845123f3f9273cb43fb87bb3f55604</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.42.1449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.42.1449%26sid%3Dliteratum%253Aachs%26aulast%3DKonishi%26aufirst%3DM.%26aulast%3DOhkuma%26aufirst%3DH.%26aulast%3DMatsumoto%26aufirst%3DK.%26aulast%3DTsuno%26aufirst%3DT.%26aulast%3DKamei%26aufirst%3DH.%26aulast%3DMiyaki%26aufirst%3DT.%26aulast%3DOki%26aufirst%3DT.%26aulast%3DKawaguchi%26aufirst%3DH.%26aulast%3DVanDuyne%26aufirst%3DG.%2BD.%26aulast%3DClardy%26aufirst%3DJ.%26atitle%3DDynemicin%2520A%252C%2520a%2520novel%2520antibiotic%2520with%2520the%2520anthraquinone%2520and%25201%252C5-diyn-3-ene%2520subunit%26jtitle%3DJ.%2520Antibiot.%26date%3D1989%26volume%3D42%26spage%3D1449%26epage%3D1452%26doi%3D10.7164%2Fantibiotics.42.1449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chowdari, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langley, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaprakasam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sufi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derwin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passmore, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rangan, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardarelli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vite, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangwar, S.</span></span> <span> </span><span class="NLM_article-title">Uncialamycin as a novel payload for antibody drug conjugate (ADC) based targeted cancer therapy</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">466</span>– <span class="NLM_lpage">470</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.12.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2Fj.bmcl.2018.12.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=30579797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFygu7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=466-470&author=N.+S.+Chowdariauthor=C.+Panauthor=C.+Raoauthor=D.+R.+Langleyauthor=P.+Sivaprakasamauthor=B.+Sufiauthor=D.+Derwinauthor=Y.+C.+Wangauthor=E.+Kwokauthor=D.+Passmoreauthor=V.+S.+Ranganauthor=S.+Deshpandeauthor=P.+Cardarelliauthor=G.+Viteauthor=S.+Gangwar&title=Uncialamycin+as+a+novel+payload+for+antibody+drug+conjugate+%28ADC%29+based+targeted+cancer+therapy&doi=10.1016%2Fj.bmcl.2018.12.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Uncialamycin as a novel payload for antibody drug conjugate (ADC) based targeted cancer therapy</span></div><div class="casAuthors">Chowdari, Naidu S.; Pan, Chin; Rao, Chetana; Langley, David R.; Sivaprakasam, Prasanna; Sufi, Bilal; Derwin, Daniel; Wang, Yichong; Kwok, Eilene; Passmore, David; Rangan, Vangipuram S.; Deshpande, Shrikant; Cardarelli, Pina; Vite, Gregory; Gangwar, Sanjeev</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">466-470</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Uncialamycin analogs were evaluated as potential cytotoxic agents in an antibody-drug conjugate (ADC) approach to treating human cancer.  These analogs were synthesized using Hauser annulations of substituted phthalides as a key step.  A highly potent uncialamycin analog 3c with a valine-citrulline dipeptide linker was conjugated to an anti-mesothelin monoclonal antibody (mAb) through lysines to generate a meso-13 conjugate.  This conjugate demonstrated subnanomolar potency (IC50 = 0.88 nM, H226 cell line) in in vitro cytotoxicity expts. with good immunol. specificity to mesothelin-pos. lung cancer cell lines.  The potency and mechanism of action of this uncialamycin class of enediyne antitumor antibiotics make them attractive payloads in ADC-based cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqBfRZz2Cfk7Vg90H21EOLACvtfcHk0lhtbZRbiFaN1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFygu7rJ&md5=371237eb33fc1f2dd6230a8649d75233</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.12.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.12.021%26sid%3Dliteratum%253Aachs%26aulast%3DChowdari%26aufirst%3DN.%2BS.%26aulast%3DPan%26aufirst%3DC.%26aulast%3DRao%26aufirst%3DC.%26aulast%3DLangley%26aufirst%3DD.%2BR.%26aulast%3DSivaprakasam%26aufirst%3DP.%26aulast%3DSufi%26aufirst%3DB.%26aulast%3DDerwin%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%2BC.%26aulast%3DKwok%26aufirst%3DE.%26aulast%3DPassmore%26aufirst%3DD.%26aulast%3DRangan%26aufirst%3DV.%2BS.%26aulast%3DDeshpande%26aufirst%3DS.%26aulast%3DCardarelli%26aufirst%3DP.%26aulast%3DVite%26aufirst%3DG.%26aulast%3DGangwar%26aufirst%3DS.%26atitle%3DUncialamycin%2520as%2520a%2520novel%2520payload%2520for%2520antibody%2520drug%2520conjugate%2520%2528ADC%2529%2520based%2520targeted%2520cancer%2520therapy%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D466%26epage%3D470%26doi%3D10.1016%2Fj.bmcl.2018.12.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poudel, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotapati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thevanayagam, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardarelli, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chowdari, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaspady, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samikannu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuppusamy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saravanakumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arunachalam, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rangan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rampulla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathur, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vite, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangwar, S.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of phenol-linked uncialamycin antibody-drug conjugates</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">126782</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.126782</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2Fj.bmcl.2019.126782" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=31767265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1eks73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=126782&author=Y.+B.+Poudelauthor=C.+Raoauthor=S.+Kotapatiauthor=M.+Deshpandeauthor=L.+Thevanayagamauthor=C.+Panauthor=J.+Cardarelliauthor=N.+Chowdariauthor=M.+Kaspadyauthor=R.+Samikannuauthor=P.+Kuppusamyauthor=P.+Saravanakumarauthor=P.+N.+Arunachalamauthor=S.+Deshpandeauthor=V.+Ranganauthor=R.+Rampullaauthor=A.+Mathurauthor=G.+D.+Viteauthor=S.+Gangwar&title=Design%2C+synthesis+and+biological+evaluation+of+phenol-linked+uncialamycin+antibody-drug+conjugates&doi=10.1016%2Fj.bmcl.2019.126782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of phenol-linked uncialamycin antibody-drug conjugates</span></div><div class="casAuthors">Poudel, Yam B.; Rao, Chetana; Kotapati, Srikanth; Deshpande, Madhura; Thevanayagam, Lourdes; Pan, Chin; Cardarelli, Josephine; Chowdari, Naidu; Kaspady, Mahammed; Samikannu, Ramesh; Kuppusamy, Prakasam; Saravanakumar, Pon; Arunachalam, Pirama N.; Deshpande, Shrikant; Rangan, Vangipuram; Rampulla, Richard; Mathur, Arvind; Vite, Gregory D.; Gangwar, Sanjeev</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">126782</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Uncialamycin is one of the structurally simpler and newer members of enediyne family of natural products.  It exhibits highly potent activity against several types of bacteria and cancer cells.  Described herein is a strategy for the targeted delivery of this cytotoxic agent to tumors using an antibody-drug conjugate (ADC) approach.  Central to the design of ADC were the generation of potent and chem. stable uncialamycin analogs and attachment of protease cleavable linkers to newly realized phenolic handles to prep. linker-payloads.  Conjugation of the linker-payloads to tumor targeting antibody, in vitro activity and in vivo evaluation are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8pthCP5tbf7Vg90H21EOLACvtfcHk0lhtbZRbiFaN1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1eks73N&md5=5c112ff7f2ec60c67c8f095df6d3fbca</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126782%26sid%3Dliteratum%253Aachs%26aulast%3DPoudel%26aufirst%3DY.%2BB.%26aulast%3DRao%26aufirst%3DC.%26aulast%3DKotapati%26aufirst%3DS.%26aulast%3DDeshpande%26aufirst%3DM.%26aulast%3DThevanayagam%26aufirst%3DL.%26aulast%3DPan%26aufirst%3DC.%26aulast%3DCardarelli%26aufirst%3DJ.%26aulast%3DChowdari%26aufirst%3DN.%26aulast%3DKaspady%26aufirst%3DM.%26aulast%3DSamikannu%26aufirst%3DR.%26aulast%3DKuppusamy%26aufirst%3DP.%26aulast%3DSaravanakumar%26aufirst%3DP.%26aulast%3DArunachalam%26aufirst%3DP.%2BN.%26aulast%3DDeshpande%26aufirst%3DS.%26aulast%3DRangan%26aufirst%3DV.%26aulast%3DRampulla%26aufirst%3DR.%26aulast%3DMathur%26aufirst%3DA.%26aulast%3DVite%26aufirst%3DG.%2BD.%26aulast%3DGangwar%26aufirst%3DS.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520phenol-linked%2520uncialamycin%2520antibody-drug%2520conjugates%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D126782%26doi%3D10.1016%2Fj.bmcl.2019.126782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lam, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Titus, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dabrah, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimball, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veitch, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustavson, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Compton, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forenza, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beutler, J.</span></span> <span> </span><span class="NLM_article-title">Improved processes for the production and isolation of dynemicin A and large-scale fermentation in a 10000-liter fermentor</span>. <i>J. Ind. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.1007/BF01583725</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1007%2FBF01583725" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1369016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADyaK3sXhs1Onu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1992&pages=7-12&author=K.+S.+Lamauthor=J.+A.+Titusauthor=T.+T.+Dabrahauthor=D.+L.+Kimballauthor=J.+M.+Veitchauthor=D.+R.+Gustavsonauthor=B.+J.+Comptonauthor=J.+A.+Matsonauthor=S.+Forenzaauthor=J.+Rossauthor=D.+Millerauthor=J.+Roachauthor=J.+Beutler&title=Improved+processes+for+the+production+and+isolation+of+dynemicin+A+and+large-scale+fermentation+in+a+10000-liter+fermentor&doi=10.1007%2FBF01583725"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Improved processes for the production and isolation of dynemicin A and large-scale fermentation in a 10000-liter fermentor</span></div><div class="casAuthors">Lam, Kin Sing; Titus, Jeffrey A.; Dabrah, Thomas T.; Kimball, Dennis L.; Veitch, Jacqueline M.; Gustavson, Donald R.; Compton, Bruce J.; Matson, James A.; Forenza, Salvatore; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Industrial Microbiology</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-12</span>CODEN:
                <span class="NLM_cas:coden">JIMIE7</span>;
        ISSN:<span class="NLM_cas:issn">0169-4146</span>.
    </div><div class="casAbstract">Supplementing the culture of Micromonospora chersina sp. nov. No. M956-1 with NaI (0.5 mg/L) enhanced the prodn. of dynemicin A by 35-fold in shake flask culture.  Homogeneous dynemicin A was obtained from the whole broth ext. by Dicalite chromatog., Sephadex LH-20 chromatog. and vacuum liq. chromatog.  Gram quantities of dynemicin A were obtained from the fermn. of M. chersina sp. nov. No. M956-1 in a 10,000-L fermentor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLTP3GkoqE-bVg90H21EOLACvtfcHk0lhtbZRbiFaN1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXhs1Onu7s%253D&md5=5bbc80dca0bb50e6daacc971009cc3fc</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2FBF01583725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01583725%26sid%3Dliteratum%253Aachs%26aulast%3DLam%26aufirst%3DK.%2BS.%26aulast%3DTitus%26aufirst%3DJ.%2BA.%26aulast%3DDabrah%26aufirst%3DT.%2BT.%26aulast%3DKimball%26aufirst%3DD.%2BL.%26aulast%3DVeitch%26aufirst%3DJ.%2BM.%26aulast%3DGustavson%26aufirst%3DD.%2BR.%26aulast%3DCompton%26aufirst%3DB.%2BJ.%26aulast%3DMatson%26aufirst%3DJ.%2BA.%26aulast%3DForenza%26aufirst%3DS.%26aulast%3DRoss%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DD.%26aulast%3DRoach%26aufirst%3DJ.%26aulast%3DBeutler%26aufirst%3DJ.%26atitle%3DImproved%2520processes%2520for%2520the%2520production%2520and%2520isolation%2520of%2520dynemicin%2520A%2520and%2520large-scale%2520fermentation%2520in%2520a%252010000-liter%2520fermentor%26jtitle%3DJ.%2520Ind.%2520Microbiol.%26date%3D1992%26volume%3D11%26spage%3D7%26epage%3D12%26doi%3D10.1007%2FBF01583725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valdés, M.</span></span> <span> </span><span class="NLM_article-title"><i>Micromonospora</i>: An important microbe for biomedicine and potentially for biocontrol and biofuels</span>. <i>Soil Biol. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1016/j.soilbio.2009.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2Fj.soilbio.2009.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC3cXis1SgsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2010&pages=536-542&author=A.+M.+Hirschauthor=M.+Vald%C3%A9s&title=Micromonospora%3A+An+important+microbe+for+biomedicine+and+potentially+for+biocontrol+and+biofuels&doi=10.1016%2Fj.soilbio.2009.11.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Micromonospora: An important microbe for biomedicine and potentially for biocontrol and biofuels</span></div><div class="casAuthors">Hirsch, Ann M.; Valdes, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Soil Biology & Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">536-542</span>CODEN:
                <span class="NLM_cas:coden">SBIOAH</span>;
        ISSN:<span class="NLM_cas:issn">0038-0717</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Micromonospora species have long been recognized as important sources of antibiotics and also for their unusual spores.  However, their involvement in plant-microbe assocns. is poorly understood although several studies demonstrate that Micromonospora species function in biocontrol, plant growth promotion, root ecol., and in the breakdown of plant cell wall material.  Our knowledge of this generally understudied group of actinomycetes has been greatly advanced by the increasing no. of reports of their assocns. with plants, by the deployment of DNA cloning and mol. systematics techniques, and by the recent application of whole genome sequencing.  Efforts to annotate the genomes of several Micromonospora species are underway.  This information will greatly augment our knowledge of these versatile microorganisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF5Tw8YxW-ALVg90H21EOLACvtfcHk0lg92zziHq3JIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXis1SgsLg%253D&md5=cc420b1cbb5d4c38afe1da3f93d833c1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.soilbio.2009.11.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.soilbio.2009.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DHirsch%26aufirst%3DA.%2BM.%26aulast%3DVald%25C3%25A9s%26aufirst%3DM.%26atitle%3DMicromonospora%253A%2520An%2520important%2520microbe%2520for%2520biomedicine%2520and%2520potentially%2520for%2520biocontrol%2520and%2520biofuels%26jtitle%3DSoil%2520Biol.%2520Biochem.%26date%3D2010%26volume%3D42%26spage%3D536%26epage%3D542%26doi%3D10.1016%2Fj.soilbio.2009.11.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dairi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, E.</span></span> <span> </span><span class="NLM_article-title">A novel, highly efficient gene-cloning system for <i>Micromonospora</i> strains</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">7004</span>– <span class="NLM_lpage">7011</span>, <span class="refDoi"> DOI: 10.1128/JB.173.21.7004-7011.1991</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1128%2FJB.173.21.7004-7011.1991" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1938902" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADyaK38Xhs1Cisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=1991&pages=7004-7011&author=M.+Hasegawaauthor=T.+Dairiauthor=T.+Ohtaauthor=E.+Hashimoto&title=A+novel%2C+highly+efficient+gene-cloning+system+for+Micromonospora+strains&doi=10.1128%2FJB.173.21.7004-7011.1991"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A novel, highly efficient gene-cloning system for Micromonospora strains</span></div><div class="casAuthors">Hasegawa, Mamoru; Dairi, Tohru; Ohta, Toshio; Hashimoto, Eri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7004-11</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    </div><div class="casAbstract">A highly efficient gene-cloning system for M. olivasterospora, a producer of the antibiotic fortimicin a (astromicin), suited to shotgun cloning had been developed.  The system is supported by 2 new advancements accomplished in this study.  One is the construction of novel plasmid vectors pMO116, pMO126, pMO133, pMO136, and pMO217, all consisting of replicons from newly found Micromonospora plasmids and selectable markers cloned from a neomycin-producing Micromonospora strain.  The other advancement is the establishment of a new protocol for bacterial protoplasting in which some kinds of sugar alcs. are added in precultures.  Such sugar alcs. were found to sensitize a wide taxonomical range of bacteria to lysozyme.  The system is reproducible and reliable and has a high efficiency of more than 106 CFU/μg of DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq89hnnZDY9rrVg90H21EOLACvtfcHk0lg92zziHq3JIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xhs1Cisw%253D%253D&md5=c902d33b7ed20dab3cb223101771ef3b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FJB.173.21.7004-7011.1991&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.173.21.7004-7011.1991%26sid%3Dliteratum%253Aachs%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DDairi%26aufirst%3DT.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DHashimoto%26aufirst%3DE.%26atitle%3DA%2520novel%252C%2520highly%2520efficient%2520gene-cloning%2520system%2520for%2520Micromonospora%2520strains%26jtitle%3DJ.%2520Bacteriol.%26date%3D1991%26volume%3D173%26spage%3D7004%26epage%3D7011%26doi%3D10.1128%2FJB.173.21.7004-7011.1991" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopwood, D. A.</span></span> <span> </span><span class="NLM_article-title">Highlights of <i>Streptomyces</i> genetics</span>. <i>Heredity</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1038/s41437-019-0196-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1038%2Fs41437-019-0196-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=31189905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A280%3ADC%252BB3M3lvVCgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2019&pages=23-32&author=D.+A.+Hopwood&title=Highlights+of+Streptomyces+genetics&doi=10.1038%2Fs41437-019-0196-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Highlights of Streptomyces genetics</span></div><div class="casAuthors">Hopwood David A</div><div class="citationInfo"><span class="NLM_cas:title">Heredity</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-32</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Sixty years ago, the actinomycetes, which include members of the genus Streptomyces, with their bacterial cellular dimensions but a mycelial growth habit like fungi, were generally regarded as a possible intermediate group, and virtually nothing was known about their genetics.  We now know that they are bacteria, but with many original features.  Their genome is linear with a unique mode of replication, not circular like those of nearly all other bacteria.  They transfer their chromosome from donor to recipient by a conjugation mechanism, but this is radically different from the E. coli paradigm.  They have twice as many genes as a typical rod-shaped bacterium like Escherichia coli or Bacillus subtilis, and the genome typically carries 20 or more gene clusters encoding the biosynthesis of antibiotics and other specialised metabolites, only a small proportion of which are expressed under typical laboratory screening conditions.  This means that there is a vast number of potentially valuable compounds to be discovered when these 'sleeping' genes are activated.  Streptomyces genetics has revolutionised natural product chemistry by facilitating the analysis of novel biosynthetic steps and has led to the ability to engineer novel biosynthetic pathways and hence 'unnatural natural products', with potential to generate lead compounds for use in the struggle to combat the rise of antimicrobial resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS-X8YfK5A6lSjiVMCqkKxVfW6udTcc2eYlvoSo_qAhP7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3lvVCgug%253D%253D&md5=a04d8335352df921a5720a340a1f7def</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fs41437-019-0196-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41437-019-0196-0%26sid%3Dliteratum%253Aachs%26aulast%3DHopwood%26aufirst%3DD.%2BA.%26atitle%3DHighlights%2520of%2520Streptomyces%2520genetics%26jtitle%3DHeredity%26date%3D2019%26volume%3D123%26spage%3D23%26epage%3D32%26doi%3D10.1038%2Fs41437-019-0196-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rout, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitsinos, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyssikatos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schammel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aujay, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavrilyuk, J.</span></span> <span> </span><span class="NLM_article-title">Total synthesis and biological evaluation of tiancimycins A and B, yangpumicin A, and related anthraquinone-fused enediyne antitumor antibiotics</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">2549</span>– <span class="NLM_lpage">2561</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b12522</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b12522" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFWms7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=2549-2561&author=K.+C.+Nicolaouauthor=D.+Dasauthor=Y.+Luauthor=S.+Routauthor=E.+N.+Pitsinosauthor=J.+Lyssikatosauthor=A.+Schammelauthor=J.+Sandovalauthor=M.+Hammondauthor=M.+Aujayauthor=J.+Gavrilyuk&title=Total+synthesis+and+biological+evaluation+of+tiancimycins+A+and+B%2C+yangpumicin+A%2C+and+related+anthraquinone-fused+enediyne+antitumor+antibiotics&doi=10.1021%2Fjacs.9b12522"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Total Synthesis and Biological Evaluation of Tiancimycins A and B, Yangpumicin A, and Related Anthraquinone-Fused Enediyne Antitumor Antibiotics</span></div><div class="casAuthors">Nicolaou, K. C.; Das, Dipendu; Lu, Yong; Rout, Subhrajit; Pitsinos, Emmanuel N.; Lyssikatos, Joseph; Schammel, Alexander; Sandoval, Joseph; Hammond, Mikhail; Aujay, Monette; Gavrilyuk, Julia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2549-2561</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The family of anthraquinone-fused enediyne antitumor antibiotics was established by the discovery of dynemicin A and deoxy-dynemicin A.  It was then expanded, first by the isolation of uncialamycin, and then by the addn. to the family of tiancimycins A-F and yangpumicin A.  This family of natural products provides opportunities in total synthesis, biol., and medicine due to their novel and challenging mol. structures, intriguing biol. properties and mechanism of action, and potential in targeted cancer therapies.  Herein, the total syntheses of tiancimycins A and B, yangpumicin A, and a no. of related anthraquinone-fused enediynes are described.  Biol. evaluation of the synthesized compds. revealed extremely potent cytotoxicities against a no. of cell lines, thus enriching the structure-activity relationships within this class of compds.  The findings of these studies may facilitate future investigations directed toward antibody-drug conjugates for targeted cancer therapies and provide inspiration for further advances in total synthesis and chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1BrlmzoSwJLVg90H21EOLACvtfcHk0lirUv0n5PwXLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFWms7s%253D&md5=6fb43e13dabe5299b97a8cceb63be319</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b12522&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b12522%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26aulast%3DDas%26aufirst%3DD.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DRout%26aufirst%3DS.%26aulast%3DPitsinos%26aufirst%3DE.%2BN.%26aulast%3DLyssikatos%26aufirst%3DJ.%26aulast%3DSchammel%26aufirst%3DA.%26aulast%3DSandoval%26aufirst%3DJ.%26aulast%3DHammond%26aufirst%3DM.%26aulast%3DAujay%26aufirst%3DM.%26aulast%3DGavrilyuk%26aufirst%3DJ.%26atitle%3DTotal%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520tiancimycins%2520A%2520and%2520B%252C%2520yangpumicin%2520A%252C%2520and%2520related%2520anthraquinone-fused%2520enediyne%2520antitumor%2520antibiotics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26spage%3D2549%26epage%3D2561%26doi%3D10.1021%2Fjacs.9b12522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattanayak, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasunoori, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poudel, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chowdari, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nazeer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangwar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vite, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitsinos, E. N.</span></span> <span> </span><span class="NLM_article-title">Streamlined total synthesis of uncialamycin and its application to the synthesis of designed analogues for biological investigations</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">8235</span>– <span class="NLM_lpage">8246</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b04339</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b04339" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC28XpsVClsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=8235-8246&author=K.+C.+Nicolaouauthor=Y.+Wangauthor=M.+Luauthor=D.+Mandalauthor=M.+R.+Pattanayakauthor=R.+Yuauthor=A.+A.+Shahauthor=J.+S.+Chenauthor=H.+Zhangauthor=J.+J.+Crawfordauthor=L.+Pasunooriauthor=Y.+B.+Poudelauthor=N.+S.+Chowdariauthor=C.+Panauthor=A.+Nazeerauthor=S.+Gangwarauthor=G.+Viteauthor=E.+N.+Pitsinos&title=Streamlined+total+synthesis+of+uncialamycin+and+its+application+to+the+synthesis+of+designed+analogues+for+biological+investigations&doi=10.1021%2Fjacs.6b04339"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Streamlined Total Synthesis of Uncialamycin and Its Application to the Synthesis of Designed Analogues for Biological Investigations</span></div><div class="casAuthors">Nicolaou, K. C.; Wang, Yanping; Lu, Min; Mandal, Debashis; Pattanayak, Manas R.; Yu, Ruocheng; Shah, Akshay A.; Chen, Jason S.; Zhang, Hongjun; Crawford, James J.; Pasunoori, Laxman; Poudel, Yam B.; Chowdari, Naidu S.; Pan, Chin; Nazeer, Ayesha; Gangwar, Sanjeev; Vite, Gregory; Pitsinos, Emmanuel N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">8235-8246</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">From the enediyne class of antitumor antibiotics, uncialamycin is among the rarest and most potent, yet one of the structurally simpler, making it attractive for chem. synthesis and potential applications in biol. and medicine.  In this article we describe a streamlined and practical enantioselective total synthesis of uncialamycin that is amenable to the synthesis of novel analogs and renders the natural product readily available for biol. and drug development studies.  Starting from hydroxy- or methoxyisatin, the synthesis features a Noyori enantioselective redn., a Yamaguchi acetylide-pyridinium coupling, a stereoselective acetylide-aldehyde cyclization, and a newly developed annulation reaction that allows efficient coupling of a cyanophthalide and a p-methoxy semiquinone aminal to forge the anthraquinone moiety of the mol. [e.g., I + II →→ III].  Overall, the developed streamlined synthesis proceeds in 22 linear steps (14 chromatog. sepns.) and 11% overall yield.  The developed synthetic strategies and technologies were applied to the synthesis of a series of designed uncialamycin analogs equipped with suitable functional groups for conjugation to antibodies and other delivery systems.  Biol. evaluation of a select no. of these analogs led to the identification of compds. with low picomolar potencies against certain cancer cell lines.  These compds. and others like them may serve as powerful payloads for the development of antibody drug conjugates (ADCs) intended for personalized targeted cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJad8noSFDWrVg90H21EOLACvtfcHk0lirUv0n5PwXLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpsVClsbc%253D&md5=c30cad508810a5f0b06cbf8da0c96871</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b04339&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b04339%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DM.%26aulast%3DMandal%26aufirst%3DD.%26aulast%3DPattanayak%26aufirst%3DM.%2BR.%26aulast%3DYu%26aufirst%3DR.%26aulast%3DShah%26aufirst%3DA.%2BA.%26aulast%3DChen%26aufirst%3DJ.%2BS.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26aulast%3DPasunoori%26aufirst%3DL.%26aulast%3DPoudel%26aufirst%3DY.%2BB.%26aulast%3DChowdari%26aufirst%3DN.%2BS.%26aulast%3DPan%26aufirst%3DC.%26aulast%3DNazeer%26aufirst%3DA.%26aulast%3DGangwar%26aufirst%3DS.%26aulast%3DVite%26aufirst%3DG.%26aulast%3DPitsinos%26aufirst%3DE.%2BN.%26atitle%3DStreamlined%2520total%2520synthesis%2520of%2520uncialamycin%2520and%2520its%2520application%2520to%2520the%2520synthesis%2520of%2520designed%2520analogues%2520for%2520biological%2520investigations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D8235%26epage%3D8246%26doi%3D10.1021%2Fjacs.6b04339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span> <span> </span><span class="NLM_article-title">Ribosome engineering and fermentation optimization leads to overproduction of tiancimycin A, a new enediyne natural product from <i>Streptomyces</i> sp. CB03234</span>. <i>J. Ind. Microbiol. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1007/s10295-018-2014-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1007%2Fs10295-018-2014-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=29396746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit12hs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2018&pages=141-151&author=L.+Liuauthor=J.+Panauthor=Z.+Wangauthor=X.+Yanauthor=D.+Yangauthor=X.+Zhuauthor=B.+Shenauthor=Y.+Duanauthor=Y.+Huang&title=Ribosome+engineering+and+fermentation+optimization+leads+to+overproduction+of+tiancimycin+A%2C+a+new+enediyne+natural+product+from+Streptomyces+sp.+CB03234&doi=10.1007%2Fs10295-018-2014-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Ribosome engineering and fermentation optimization leads to overproduction of tiancimycin A, a new enediyne natural product from Streptomyces sp. CB03234</span></div><div class="casAuthors">Liu, Ling; Pan, Jian; Wang, Zilong; Yan, Xiaohui; Yang, Dong; Zhu, Xiangcheng; Shen, Ben; Duan, Yanwen; Huang, Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Industrial Microbiology & Biotechnology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">141-151</span>CODEN:
                <span class="NLM_cas:coden">JIMBFL</span>;
        ISSN:<span class="NLM_cas:issn">1367-5435</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Tiancimycin (TNM) A, a recently discovered enediyne natural product from Streptomyces sp.  CB03234, showed rapid and complete killing of cancer cells and could be used as a payload in antibody drug conjugates.  The low yield of TNM A in the wild-type strain promoted us to use ribosome engineering and fermn. optimization for its yield improvement.  The Streptomyces sp.  CB03234-R-16 mutant strain with a L422P mutation in RpoB, the RNA polymerase β-subunit, was obtained from the rifamycin-resistant screening.  After fermn. optimization, the titers of TNM A in Streptomyces sp.  CB03234-R-16 reached to 22.5 ± 3.1 mg L-1 in shaking flasks, and 13 ± 1 mg L-1 in 15 L fermentors, which were at least 40-fold higher than that in the wild-type strain (∼ 0.3 mg L-1).  Quant. real-time RT-PCR revealed markedly enhanced expression of key genes encoding TNM A biosynthetic enzymes and regulators in Streptomyces sp.  CB03234-R-16.  Our study should greatly facilitate the future efforts to develop TNM A into a clin. anticancer drug.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4tFVDNUS8W7Vg90H21EOLACvtfcHk0lih_KLRDpQ9bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit12hs7k%253D&md5=a50063bdb64df4afb255310d2d950611</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs10295-018-2014-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10295-018-2014-8%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DPan%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DY.%26atitle%3DRibosome%2520engineering%2520and%2520fermentation%2520optimization%2520leads%2520to%2520overproduction%2520of%2520tiancimycin%2520A%252C%2520a%2520new%2520enediyne%2520natural%2520product%2520from%2520Streptomyces%2520sp.%2520CB03234%26jtitle%3DJ.%2520Ind.%2520Microbiol.%2520Biotechnol.%26date%3D2018%26volume%3D45%26spage%3D141%26epage%3D151%26doi%3D10.1007%2Fs10295-018-2014-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span> <span> </span><span class="NLM_article-title">Genome shuffling based on different types of ribosome engineering mutants for enhanced production of 10-membered enediyne tiancimycin-A</span>. <i>Appl. Microbiol. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">4359</span>– <span class="NLM_lpage">4369</span>, <span class="refDoi"> DOI: 10.1007/s00253-020-10583-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1007%2Fs00253-020-10583-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=32236679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmtVKlt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2020&pages=4359-4369&author=H.+Liuauthor=C.+Jiangauthor=J.+Linauthor=Z.+Zhuangauthor=W.+Kongauthor=L.+Liuauthor=Y.+Huangauthor=Y.+Duanauthor=X.+Zhu&title=Genome+shuffling+based+on+different+types+of+ribosome+engineering+mutants+for+enhanced+production+of+10-membered+enediyne+tiancimycin-A&doi=10.1007%2Fs00253-020-10583-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Genome shuffling based on different types of ribosome engineering mutants for enhanced production of 10-membered enediyne tiancimycin-A</span></div><div class="casAuthors">Liu, Huiming; Jiang, Chengzhou; Lin, Jing; Zhuang, Zhoukang; Kong, Wenping; Liu, Ling; Huang, Yong; Duan, Yanwen; Zhu, Xiangcheng</div><div class="citationInfo"><span class="NLM_cas:title">Applied Microbiology and Biotechnology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4359-4369</span>CODEN:
                <span class="NLM_cas:coden">AMBIDG</span>;
        ISSN:<span class="NLM_cas:issn">0175-7598</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Abstr.: Tiancimycin-A (TNM-A) is an anthraquinone-fused ten-membered enediyne produced by Streptomyces sp.  CB03234, which is very promising for the development of anticancer antibody-drug conjugates (ADCs).  To improve the titer of TNM-A, we have generated high-producing mutants CB03234-S and CB03234-R through ribosome engineering, but still not sufficient for pilot prodn. of TNM-A.  As the follow-up work, gentamycin-induced ribosome engineering was further adopted here to generate the mutant CB03234-G, which produced similar level of TNM-A as in CB03234-S and CB03234-R.  Benefiting from the distinct antibiotic resistances of three ribosome engineering mutants, genome shuffling between any two of them was resp. carried out, and finally obtained the recombinant CB03234-GS26.  Under optimal conditions, CB03234-GS26 produced 40.6 ± 1.0 mg/L TNM-A in shaking flasks and 20.8 ± 0.4 mg/L in a scaled-up 30-L fermentor.  Comparing with the parental high-producing mutants, the over 1.6-fold titer improvement of CB03234-GS26 in fermentor was more promising for pilot prodn. of TNM-A.  Besides the distinctive morphol. features, genetic characterization revealed that CB03234-GS26 possessed 1.8 kb rsmG related deletion just the same as CB03234-S, but no mutation was found in rpsL.  Subsequent knockouts proved that rsmG was unrelated to titer improvement of TNM-A, which implied other genomic variations and mechanisms rather than ribosome engineering to enhance the biosynthesis of TNM-A.  Therefore, CB03234-GS26 provided a basis to locate potential novel genetic targets, and explore the interactions between complex metabolic network and TNM biosynthetic pathway, which shall promote future construction of high-yielding systems for TNM-A and other anthraquinone-fused enediynes.  Key Points •United genome shuffling and ribosome engineering help further strain improvement. •CB03234-GS26 with improved titer is practical for the pilot prodn. of TNM-A. •Enhanced TNM-A prodn. should attribute to novel genetic features/mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowT-Voi_xWA7Vg90H21EOLACvtfcHk0lih_KLRDpQ9bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmtVKlt7w%253D&md5=38015dfef5ae8a80ab386ffa805cf7f4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs00253-020-10583-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00253-020-10583-2%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DZhuang%26aufirst%3DZ.%26aulast%3DKong%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DX.%26atitle%3DGenome%2520shuffling%2520based%2520on%2520different%2520types%2520of%2520ribosome%2520engineering%2520mutants%2520for%2520enhanced%2520production%2520of%252010-membered%2520enediyne%2520tiancimycin-A%26jtitle%3DAppl.%2520Microbiol.%2520Biotechnol.%26date%3D2020%26volume%3D104%26spage%3D4359%26epage%3D4369%26doi%3D10.1007%2Fs00253-020-10583-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span> <span> </span><span class="NLM_article-title">Streptomycin-induced ribosome engineering complemented with fermentation optimization for enhanced production of 10-membered enediynes tiancimycin-A and tiancimycin-D</span>. <i>Biotechnol. Bioeng.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">1304</span>– <span class="NLM_lpage">1314</span>, <span class="refDoi"> DOI: 10.1002/bit.26944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fbit.26944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=30712262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVyitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=1304-1314&author=Z.+Zhuangauthor=C.+Jiangauthor=F.+Zhangauthor=R.+Huangauthor=L.+Yiauthor=Y.+Huangauthor=X.+Yanauthor=Y.+Duanauthor=X.+Zhu&title=Streptomycin-induced+ribosome+engineering+complemented+with+fermentation+optimization+for+enhanced+production+of+10-membered+enediynes+tiancimycin-A+and+tiancimycin-D&doi=10.1002%2Fbit.26944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Streptomycin-induced ribosome engineering complemented with fermentation optimization for enhanced production of 10-membered enediynes tiancimycin-A and tiancimycin-D</span></div><div class="casAuthors">Zhuang, Zhoukang; Jiang, Chengzhou; Zhang, Fan; Huang, Rong; Yi, Liwei; Huang, Yong; Yan, Xiaohui; Duan, Yanwen; Zhu, Xiangcheng</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology and Bioengineering</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1304-1314</span>CODEN:
                <span class="NLM_cas:coden">BIBIAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3592</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Tiancimycins (TNMs) are a group of 10-membered anthraquinone-fused enediynes, newly discovered from Streptomyces sp.  CB03234. Among them, TNM-A and TNM-D have exhibited excellent antitumor performances and could be exploited as very promising warheads for the development of anticancer antibody-drug conjugates (ADCs).  However, their low titers, esp. TNM-D, have severely limited following progress.  Therefore, the streptomycin-induced ribosome engineering was adopted in this work for strain improvement of CB03234, and a TNMs high producer S. sp.  CB03234-S with the K43N mutation at 30S ribosomal protein S12 was successfully screened out.  Subsequent media optimization revealed the essential effects of iodide and copper ion on the prodn. of TNMs, while the substitution of nitrogen source could evidently promote the accumulation of TNM-D, and the ratio of produced TNM-A and TNM-D was responsive to the change of carbon and nitrogen ratio in the medium.  Further amelioration of the pH control in scaled up 25 L fermn. increased the av. titers of TNM-A and TNM-D up to 13.7 ± 0.3 and 19.2 ± 0.4 mg/L, resp.  The achieved over 45-fold titer improvement of TNM-A, and 109-fold total titer improvement of TNM-A and TNM-D enabled the efficient purifn. of over 200 mg of each target mol. from 25 L fermn.  Our efforts have demonstrated a practical strategy for titer improvement of anthraquinone-fused enediynes and set up a solid base for the pilot scale prodn. and preclin. studies of TNMs to expedite the future development of anticancer ADC drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd_0ApYytIDbVg90H21EOLACvtfcHk0lih_KLRDpQ9bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVyitr8%253D&md5=1c75f4bed197e064cf200e41770e1ba0</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fbit.26944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbit.26944%26sid%3Dliteratum%253Aachs%26aulast%3DZhuang%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DYi%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DX.%26atitle%3DStreptomycin-induced%2520ribosome%2520engineering%2520complemented%2520with%2520fermentation%2520optimization%2520for%2520enhanced%2520production%2520of%252010-membered%2520enediynes%2520tiancimycin-A%2520and%2520tiancimycin-D%26jtitle%3DBiotechnol.%2520Bioeng.%26date%3D2019%26volume%3D116%26spage%3D1304%26epage%3D1314%26doi%3D10.1002%2Fbit.26944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teijaro, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crnovcic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annaval, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rader, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span> <span> </span><span class="NLM_article-title">Comparative studies of the biosynthetic gene clusters for anthraquinone-fused enediynes shedding light into the tailoring steps of tiancimycin biosynthesis</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5918</span>– <span class="NLM_lpage">5921</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.8b02584</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.8b02584" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWitb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=5918-5921&author=X.+Yanauthor=J.+J.+Chenauthor=A.+Adhikariauthor=C.+N.+Teijaroauthor=H.+Geauthor=I.+Crnovcicauthor=C.+Y.+Changauthor=T.+Annavalauthor=D.+Yangauthor=C.+Raderauthor=B.+Shen&title=Comparative+studies+of+the+biosynthetic+gene+clusters+for+anthraquinone-fused+enediynes+shedding+light+into+the+tailoring+steps+of+tiancimycin+biosynthesis&doi=10.1021%2Facs.orglett.8b02584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative studies of the biosynthetic gene clusters for anthraquinone-fused enediynes shedding light into the tailoring steps of tiancimycin biosynthesis</span></div><div class="casAuthors">Yan, Xiaohui; Chen, Jian-Jun; Adhikari, Ajeeth; Teijaro, Christiana N.; Ge, Huiming; Crnovcic, Ivana; Chang, Chin-Yuan; Annaval, Thibault; Yang, Dong; Rader, Christoph; Shen, Ben</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5918-5921</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Comparative analyses of the four known anthraquinone-fused enediynes biosynthetic gene clusters identified four genes, tnmE6, tnmH, tnmL, and tnmQ, unique to the tnm gene cluster.  Larger scale fermn. of both the Streptomyces sp. CB03234 wild-type and the ΔtnmH and ΔtnmL mutant strains resulted in the characterization of 20 new tiancimycin (TNM) congeners, including five enediynes.  These findings enabled a proposal for the late stage of TNM biosynthesis featuring an intermediate possibly common for all anthraquinone-fused enediynes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooxPD2YCUwK7Vg90H21EOLACvtfcHk0lgxIJS2mQKQDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWitb7K&md5=8eb622f3fd0eaa003938aac8548619c7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.8b02584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.8b02584%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%2BJ.%26aulast%3DAdhikari%26aufirst%3DA.%26aulast%3DTeijaro%26aufirst%3DC.%2BN.%26aulast%3DGe%26aufirst%3DH.%26aulast%3DCrnovcic%26aufirst%3DI.%26aulast%3DChang%26aufirst%3DC.%2BY.%26aulast%3DAnnaval%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DRader%26aufirst%3DC.%26aulast%3DShen%26aufirst%3DB.%26atitle%3DComparative%2520studies%2520of%2520the%2520biosynthetic%2520gene%2520clusters%2520for%2520anthraquinone-fused%2520enediynes%2520shedding%2520light%2520into%2520the%2520tailoring%2520steps%2520of%2520tiancimycin%2520biosynthesis%26jtitle%3DOrg.%2520Lett.%26date%3D2018%26volume%3D20%26spage%3D5918%26epage%3D5921%26doi%3D10.1021%2Facs.orglett.8b02584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huber, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorson, J. S.</span></span> <span> </span><span class="NLM_article-title">AdoMet analog synthesis and utilization: current state of the art</span>. <i>Curr. Opin. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">189</span>– <span class="NLM_lpage">197</span>, <span class="refDoi"> DOI: 10.1016/j.copbio.2016.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2Fj.copbio.2016.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=27506965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Kls7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2016&pages=189-197&author=T.+D.+Huberauthor=B.+R.+Johnsonauthor=J.+Zhangauthor=J.+S.+Thorson&title=AdoMet+analog+synthesis+and+utilization%3A+current+state+of+the+art&doi=10.1016%2Fj.copbio.2016.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">AdoMet analog synthesis and utilization: current state of the art</span></div><div class="casAuthors">Huber, Tyler D.; Johnson, Brooke R.; Zhang, Jianjun; Thorson, Jon S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Biotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">189-197</span>CODEN:
                <span class="NLM_cas:coden">CUOBE3</span>;
        ISSN:<span class="NLM_cas:issn">0958-1669</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  S-Adenosyl-L-methionine (AdoMet) is an essential enzyme cosubstrate in fundamental biol. with an expanding range of biocatalytic and therapeutic applications.  In recent years, technologies enabling the synthesis and utilization of novel functional AdoMet surrogates have rapidly advanced.  Developments highlighted within this brief review include improved syntheses of AdoMet analogs, unique S-adenosyl-L-methionine isosteres with enhanced stability, and corresponding applications in epigenetics, proteomics and natural product/small mol. diversification ('alkylrandomization').</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdXtVaKlqZLLVg90H21EOLACvtfcHk0lgxIJS2mQKQDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Kls7vM&md5=c2b40e0b3242276b6246998be3f9ba95</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.copbio.2016.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.copbio.2016.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DT.%2BD.%26aulast%3DJohnson%26aufirst%3DB.%2BR.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DThorson%26aufirst%3DJ.%2BS.%26atitle%3DAdoMet%2520analog%2520synthesis%2520and%2520utilization%253A%2520current%2520state%2520of%2520the%2520art%26jtitle%3DCurr.%2520Opin.%2520Biotechnol.%26date%3D2016%26volume%3D42%26spage%3D189%26epage%3D197%26doi%3D10.1016%2Fj.copbio.2016.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, L.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, M.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of the tiancilactone antibiotics by genome mining of atypical bacterial type II diterpene synthases</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1727</span>– <span class="NLM_lpage">1733</span>, <span class="refDoi"> DOI: 10.1002/cbic.201800285</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fcbic.201800285" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ans7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=1727-1733&author=L.-B.+Dongauthor=J.+D.+Rudolfauthor=M.-R.+Dengauthor=X.+Yanauthor=B.+Shen&title=Discovery+of+the+tiancilactone+antibiotics+by+genome+mining+of+atypical+bacterial+type+II+diterpene+synthases&doi=10.1002%2Fcbic.201800285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of the Tiancilactone Antibiotics by Genome Mining of Atypical Bacterial Type II Diterpene Synthases</span></div><div class="casAuthors">Dong, Liao-Bin; Rudolf, Jeffrey D.; Deng, Ming-Rong; Yan, Xiaohui; Shen, Ben</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1727-1733</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Although genome mining has advanced the identification, discovery, and study of microbial natural products, the discovery of bacterial diterpenoids continues to lag behind.  Herein, we report the identification of 66 putative producers of novel bacterial diterpenoids, and the discovery of the tiancilactone (TNL) family of antibiotics, by genome mining of type II diterpene synthases that do not possess the canonical DXDD motif.  The TNLs, which are broad-spectrum antibiotics with moderate activities, are produced by both Streptomyces sp.  CB03234 and Streptomyces sp.  CB03238 and feature a highly functionalized diterpenoid skeleton that is further decorated with chloroanthranilate and γ-butyrolactone moieties.  Genetic manipulation of the tnl gene cluster resulted in TNL congeners, which provided insights into their biosynthesis and structure-activity relationships.  This work highlights the biosynthetic potential that bacteria possess to produce diterpenoids and should inspire continued efforts to discover terpenoid natural products from bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY188wjn4457Vg90H21EOLACvtfcHk0lidbH06OArplg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ans7%252FJ&md5=33bc28a93cb14b3a4fa327a0849a5289</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201800285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201800285%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DL.-B.%26aulast%3DRudolf%26aufirst%3DJ.%2BD.%26aulast%3DDeng%26aufirst%3DM.-R.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520the%2520tiancilactone%2520antibiotics%2520by%2520genome%2520mining%2520of%2520atypical%2520bacterial%2520type%2520II%2520diterpene%2520synthases%26jtitle%3DChemBioChem%26date%3D2018%26volume%3D19%26spage%3D1727%26epage%3D1733%26doi%3D10.1002%2Fcbic.201800285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, J. D.</span>; <span class="NLM_string-name">Crnovcic, I.</span>; <span class="NLM_string-name">Shen, B.</span>; <span class="NLM_string-name">Newman, D. J.</span>; <span class="NLM_string-name">Cragg, G. M.</span>; <span class="NLM_string-name">Grothaus, P. G.</span></span> <span> </span><span class="NLM_article-title">The Role of Combinatorial Biosynthesis in Natural Products Discovery</span>. In  <i>Chemical Biology of Natural Products</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, D. J.</span>, <span class="NLM_string-name">Cragg, G. M.</span>, <span class="NLM_string-name">Grothaus, P. G.</span></span>, Eds.; <span class="NLM_publisher-name">CRC Press</span>, <span class="NLM_year">2017</span>; pp  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">125</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=87-125&author=J.+D.+Rudolf&author=I.+Crnovcic&author=B.+Shen&author=D.+J.+Newman&author=G.+M.+Cragg&author=P.+G.+Grothausauthor=D.+J.+Newman&author=G.+M.+Cragg&author=P.+G.+Grothaus&title=Chemical+Biology+of+Natural+Products"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DRudolf%26aufirst%3DJ.%2BD.%26atitle%3DThe%2520Role%2520of%2520Combinatorial%2520Biosynthesis%2520in%2520Natural%2520Products%2520Discovery%26btitle%3DChemical%2520Biology%2520of%2520Natural%2520Products%26aulast%3DNewman%26aufirst%3DD.%2BJ.%26pub%3DCRC%2520Press%26date%3D2017%26spage%3D87%26epage%3D125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stecher, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tengg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueberbacher, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwab, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griengl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber-Khadjawi, M.</span></span> <span> </span><span class="NLM_article-title">Biocatalytic friedel-crafts alkylation using non-natural cofactors</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">9546</span>– <span class="NLM_lpage">9548</span>, <span class="refDoi"> DOI: 10.1002/anie.200905095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fanie.200905095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFars7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=9546-9548&author=H.+Stecherauthor=M.+Tenggauthor=B.+J.+Ueberbacherauthor=P.+Remlerauthor=H.+Schwabauthor=H.+Grienglauthor=M.+Gruber-Khadjawi&title=Biocatalytic+friedel-crafts+alkylation+using+non-natural+cofactors&doi=10.1002%2Fanie.200905095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Biocatalytic Friedel-Crafts Alkylation Using Non-natural Cofactors</span></div><div class="casAuthors">Stecher, Harald; Tengg, Martin; Ueberbacher, Bernhard J.; Remler, Peter; Schwab, Helmut; Griengl, Herfried; Gruber-Khadjawi, Mandana</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">9546-9548, S9546/1-S9546/11</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A biocatalytic equiv. for the Friedel-Crafts alkylation is described.  The S-adenosyl-L-methionine-dependent methyltransferases NovO and COuO from Streptomyces sphaeroides were employed to alkylate amninocoumarins with alkylated derivs. of S-adenosyl-L-methionine.  These enzymes were shown to accept these modified cofactors and catalyzed the synthesis of not only methylated, but also allyl, propargyl, and benzylated arenes in moderate to high yields with excellent regioselectivity.  The enzymes also demonstrated acceptance of a wide variety of substrates and were used to methylate several naphthalenediols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEGC437uLgLbVg90H21EOLACvtfcHk0lidbH06OArplg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFars7fE&md5=4698c1cf0058a9a357811e786d401142</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fanie.200905095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200905095%26sid%3Dliteratum%253Aachs%26aulast%3DStecher%26aufirst%3DH.%26aulast%3DTengg%26aufirst%3DM.%26aulast%3DUeberbacher%26aufirst%3DB.%2BJ.%26aulast%3DRemler%26aufirst%3DP.%26aulast%3DSchwab%26aufirst%3DH.%26aulast%3DGriengl%26aufirst%3DH.%26aulast%3DGruber-Khadjawi%26aufirst%3DM.%26atitle%3DBiocatalytic%2520friedel-crafts%2520alkylation%2520using%2520non-natural%2520cofactors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26spage%3D9546%26epage%3D9548%26doi%3D10.1002%2Fanie.200905095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, B. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struck, A.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Micklefield, J.</span></span> <span> </span><span class="NLM_article-title">Site-specific bioalkylation of rapamycin by the RapM 16-<i>O</i>-methyltransferase</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">2885</span>– <span class="NLM_lpage">2892</span>, <span class="refDoi"> DOI: 10.1039/C5SC00164A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1039%2FC5SC00164A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=29403635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsF2qt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=2885-2892&author=B.+J.+C.+Lawauthor=A.-W.+Struckauthor=M.+R.+Bennettauthor=B.+Wilkinsonauthor=J.+Micklefield&title=Site-specific+bioalkylation+of+rapamycin+by+the+RapM+16-O-methyltransferase&doi=10.1039%2FC5SC00164A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Site-specific bioalkylation of rapamycin by the RapM 16-O-methyltransferase</span></div><div class="casAuthors">Law, Brian J. C.; Struck, Anna-Winona; Bennett, Matthew R.; Wilkinson, Barrie; Micklefield, Jason</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2885-2892</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The methylation of natural products by S-adenosyl methionine (AdoMet, also known as SAM)-dependent methyltransferase enzymes is a common tailoring step in many biosynthetic pathways.  The introduction of Me substituents can affect the biol. and physicochem. properties of the secondary metabolites produced.  Recently it has become apparent that some AdoMet-dependent methyltransferases exhibit promiscuity and will accept AdoMet analogs enabling the transfer of alternative alkyl groups.  In this study we have characterized a methyltransferase, RapM, which is involved in the biosynthesis of the potent immunosuppressive agent rapamycin.  We have shown that recombinant RapM regioselectively methylates the C16 hydroxyl group of desmethyl rapamycin precursors in vitro and is promiscuous in accepting alternative co-factors in addn. to AdoMet.  A coupled enzyme system was developed, including a mutant human enzyme methionine adenosyl transferase (MAT), along with RapM, which was used to prep. alkylated rapamycin derivs. (rapalogs) with alternative Et and allyl ether groups, derived from simple S-Et or S-allyl methionine analogs.  There are two other methyltransferases RapI and RapQ which provide Me substituents of rapamycin.  Consequently, using the enzymic approach described here, it should be possible to generate a diverse array of alkylated rapalogs, with altered properties, that would be difficult to obtain by traditional synthetic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdFgiDKgQ8U7Vg90H21EOLACvtfcHk0lgjPV1R2eUobQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsF2qt74%253D&md5=7ed40aa2654771cb440486828f28aef2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC5SC00164A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5SC00164A%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DB.%2BJ.%2BC.%26aulast%3DStruck%26aufirst%3DA.-W.%26aulast%3DBennett%26aufirst%3DM.%2BR.%26aulast%3DWilkinson%26aufirst%3DB.%26aulast%3DMicklefield%26aufirst%3DJ.%26atitle%3DSite-specific%2520bioalkylation%2520of%2520rapamycin%2520by%2520the%2520RapM%252016-O-methyltransferase%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D2885%26epage%3D2892%26doi%3D10.1039%2FC5SC00164A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharpless, K. B.</span></span> <span> </span><span class="NLM_article-title">Click chemistry: diverse chemical function from a few good reactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">2004</span>– <span class="NLM_lpage">2021</span>, <span class="refDoi"> DOI: 10.1002/1521-3773(20010601)40:11<2004::AID-ANIE2004>3.0.CO;2-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=11433435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD3MXksVOis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=2004-2021&author=H.+C.+Kolbauthor=M.+G.+Finnauthor=K.+B.+Sharpless&title=Click+chemistry%3A+diverse+chemical+function+from+a+few+good+reactions&doi=10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Click chemistry: diverse chemical function from a few good reactions</span></div><div class="casAuthors">Kolb, Hartmuth C.; Finn, M. G.; Sharpless, K. Barry</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2004-2021</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Review with > 88 refs.  Examn. of nature's favorite mols. reveals a striking preference for making carbon-heteroatom bonds over carbon-carbon bonds - surely no surprise given that carbon dioxide is nature's starting material and that most reactions are performed in water.  Nucleic acids, proteins, and polysaccharides are condensation polymers of small subunits stitched together by carbon-heteroatom bonds.  Even the 35 or so building blocks from which these crucial mols. are made each contain, at most, six contiguous C-C bonds, except for the three arom. amino acids.  Taking a cue from nature's approach, the development of a set of powerful, highly reliable, and selective reactions for the rapid synthesis of useful new compds. and combinatorial libraries through heteroatom links (C-X-C), an approach called "click chem." is addressed.  Click chem. is at once defined, enabled, and constrained by a handful of nearly perfect "spring-loaded" reactions.  The stringent criteria for a process to earn click chem. status are described along with examples of the mol. frameworks that are easily made using this spartan, but powerful, synthetic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptrjo9nxXNRrVg90H21EOLACvtfcHk0lgjPV1R2eUobQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXksVOis78%253D&md5=61c28b416c42c87821bc8a5bc2fdec36</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3773%252820010601%252940%253A11%253C2004%253A%253AAID-ANIE2004%253E3.0.CO%253B2-5%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DH.%2BC.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26aulast%3DSharpless%26aufirst%3DK.%2BB.%26atitle%3DClick%2520chemistry%253A%2520diverse%2520chemical%2520function%2520from%2520a%2520few%2520good%2520reactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2001%26volume%3D40%26spage%3D2004%26epage%3D2021%26doi%3D10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolmel, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kool, E. T.</span></span> <span> </span><span class="NLM_article-title">Oximes and hydrazones in bioconjugation: mechanism and catalysis</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">10358</span>– <span class="NLM_lpage">10376</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00090</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00090" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVaqsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=10358-10376&author=D.+K.+Kolmelauthor=E.+T.+Kool&title=Oximes+and+hydrazones+in+bioconjugation%3A+mechanism+and+catalysis&doi=10.1021%2Facs.chemrev.7b00090"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Oximes and Hydrazones in Bioconjugation: Mechanism and Catalysis</span></div><div class="casAuthors">Kolmel, Dominik K.; Kool, Eric T.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">10358-10376</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The formation of oximes and hydrazones is employed in numerous scientific fields as a simple and versatile conjugation strategy.  This imine-forming reaction is applied in fields as diverse as polymer chem., biomaterials and hydrogels, dynamic combinatorial chem., org. synthesis, and chem. biol.  Here we outline chem. developments in this field, with special focus on the past ∼10 years of developments.  Recent strategies for installing reactive carbonyl groups and α-nucleophiles into biomols. are described.  The basic chem. properties of reactants and products in this reaction are then reviewed, with an eye to understanding the reaction's mechanism and how reactant structure controls rates and equil. in the process.  Recent work that has uncovered structural features and new mechanisms for speeding the reaction, sometimes by orders of magnitude, is discussed.  We describe recent studies that have identified esp. fast reacting aldehyde/ketone substrates and structural effects that lead to rapid-reacting α-nucleophiles as well.  Among the most effective new strategies has been the development of substituents near the reactive aldehyde group that either transfer protons at the transition state or trap the initially formed tetrahedral intermediates.  In addn., the recent development of efficient nucleophilic catalysts for the reaction is outlined, improving greatly upon aniline, the classical catalyst for imine formation.  A no. of uses of such second- and third-generation catalysts in bioconjugation and in cellular applications are highlighted.  While formation of hydrazone and oxime has been traditionally regarded as being limited by slow rates, developments in the past 5 years have resulted in completely overturning this limitation; indeed, the reaction is now one of the fastest and most versatile reactions available for conjugations of biomols. and biomaterials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnTRIa9Q3ZtbVg90H21EOLACvtfcHk0lh8O3_bYaTz2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVaqsbzP&md5=e8c44eea0d1343e0bb0b17dc460640eb</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00090%26sid%3Dliteratum%253Aachs%26aulast%3DKolmel%26aufirst%3DD.%2BK.%26aulast%3DKool%26aufirst%3DE.%2BT.%26atitle%3DOximes%2520and%2520hydrazones%2520in%2520bioconjugation%253A%2520mechanism%2520and%2520catalysis%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D10358%26epage%3D10376%26doi%3D10.1021%2Facs.chemrev.7b00090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutureira, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerreiro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimenez-Oses, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardes, G. J.</span></span> <span> </span><span class="NLM_article-title">A minimal, unstrained <i>S</i>-allyl handle for pre-targeting diels-alder bioorthogonal labeling in live cells</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">14683</span>– <span class="NLM_lpage">14687</span>, <span class="refDoi"> DOI: 10.1002/anie.201608438</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fanie.201608438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslagt7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=14683-14687&author=B.+L.+Oliveiraauthor=Z.+Guoauthor=O.+Boutureiraauthor=A.+Guerreiroauthor=G.+Jimenez-Osesauthor=G.+J.+Bernardes&title=A+minimal%2C+unstrained+S-allyl+handle+for+pre-targeting+diels-alder+bioorthogonal+labeling+in+live+cells&doi=10.1002%2Fanie.201608438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A Minimal, Unstrained S-Allyl Handle for Pre-Targeting Diels-Alder Bioorthogonal Labeling in Live Cells</span></div><div class="casAuthors">Oliveira, Bruno L.; Guo, Zijian; Boutureira, Omar; Guerreiro, Ana; Jimenez-Oses, Gonzalo; Bernardes, Goncalo J. L.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">14683-14687</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The unstrained S-allyl cysteine amino acid was site-specifically installed on apoptosis protein biomarkers and was further used as a chem. handle and ligation partner for 1,2,4,5-tetrazines by means of an inverse-electron-demand Diels-Alder reaction.  We demonstrate the utility of this minimal handle for the efficient labeling of apoptotic cells using a fluorogenic tetrazine dye in a pre-targeting approach.  The small size, easy chem. installation, and selective reactivity of the S-allyl handle towards tetrazines should be readily extendable to other proteins and biomols., which could facilitate their labeling within live cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkCcPrl66Fd7Vg90H21EOLACvtfcHk0lh8O3_bYaTz2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslagt7jK&md5=94ec320e3e90a3af8b7a24828ce623b1</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201608438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201608438%26sid%3Dliteratum%253Aachs%26aulast%3DOliveira%26aufirst%3DB.%2BL.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DBoutureira%26aufirst%3DO.%26aulast%3DGuerreiro%26aufirst%3DA.%26aulast%3DJimenez-Oses%26aufirst%3DG.%26aulast%3DBernardes%26aufirst%3DG.%2BJ.%26atitle%3DA%2520minimal%252C%2520unstrained%2520S-allyl%2520handle%2520for%2520pre-targeting%2520diels-alder%2520bioorthogonal%2520labeling%2520in%2520live%2520cells%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D14683%26epage%3D14687%26doi%3D10.1002%2Fanie.201608438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tasdelen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yagci, Y.</span></span> <span> </span><span class="NLM_article-title">Light-induced click reactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">5930</span>– <span class="NLM_lpage">5938</span>, <span class="refDoi"> DOI: 10.1002/anie.201208741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fanie.201208741" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntF2lsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=5930-5938&author=M.+A.+Tasdelenauthor=Y.+Yagci&title=Light-induced+click+reactions&doi=10.1002%2Fanie.201208741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Light-Induced Click Reactions</span></div><div class="casAuthors">Tasdelen, Mehmet Atilla; Yagci, Yusuf</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5930-5938</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Spatial and temporal control over chem. and biol. processes, both in terms of "tuning" products and providing site-specific control, is one of the most exciting and rapidly developing areas of modern science.  For synthetic chemists, the challenge is to discover and develop selective and efficient reactions capable of generating useful mols. in a variety of matrixes.  In recent studies, light has been recognized as a valuable method for detg. where, when, and to what extent a process is started or stopped.  Accordingly, this Minireview will present the fundamental aspects of light-induced click reactions, highlight the applications of these reactions to diverse fields of study, and discuss the potential for this methodol. to be applied to the study of biomol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOIfssFxuR9rVg90H21EOLACvtfcHk0lh8O3_bYaTz2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntF2lsLY%253D&md5=19bac16930cc09dee72fcc1f0307befe</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.201208741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201208741%26sid%3Dliteratum%253Aachs%26aulast%3DTasdelen%26aufirst%3DM.%2BA.%26aulast%3DYagci%26aufirst%3DY.%26atitle%3DLight-induced%2520click%2520reactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D5930%26epage%3D5938%26doi%3D10.1002%2Fanie.201208741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bothwell, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Islam, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M. K.</span></span> <span> </span><span class="NLM_article-title">Se-Adenosyl-L-selenomethionine cofactor analogue as a reporter of protein methylation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">14905</span>– <span class="NLM_lpage">14912</span>, <span class="refDoi"> DOI: 10.1021/ja304782r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja304782r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1GhtbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=14905-14912&author=I.+R.+Bothwellauthor=K.+Islamauthor=Y.+L.+Chenauthor=W.+H.+Zhengauthor=G.+Blumauthor=H.+T.+Dengauthor=M.+K.+Luo&title=Se-Adenosyl-L-selenomethionine+cofactor+analogue+as+a+reporter+of+protein+methylation&doi=10.1021%2Fja304782r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Se-Adenosyl-L-selenomethionine Cofactor Analogue as a Reporter of Protein Methylation</span></div><div class="casAuthors">Bothwell, Ian R.; Islam, Kabirul; Chen, Yuling; Zheng, Weihong; Blum, Gil; Deng, Haiteng; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14905-14912</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Post-translational methylation by S-adenosyl-L-methionine (SAM)-dependent methyltransferases plays essential roles in modulating protein function in both normal and disease states.  As such, there is a growing need to develop chem. reporters to examine the physiol. and pathol. roles of protein methyltransferases.  Several sterically bulky SAM analogs have previously been used to label substrates of specific protein methyltransferases.  However, broad application of these compds. has been limited by their general incompatibility with native enzymes.  Here we report a SAM surrogate, ProSeAM (propargylic Se-adenosyl-L-selenomethionine), as a reporter of methyltransferases.  ProSeAM can be processed by multiple protein methyltransferases for substrate labeling.  In contrast, sulfur-based propargylic SAM undergoes rapid decompn. at physiol. pH, likely via an allene intermediate.  In conjunction with fluorescent/affinity-based azide probes, copper-catalyzed azide-alkyne cycloaddn. chem., in-gel fluorescence visualization and proteomic anal., we further demonstrated ProSeAM's utility to profile substrates of endogenous methyltransferases in diverse cellular contexts.  These results thus feature ProSeAM as a convenient probe to study the activities of endogenous protein methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrulCEDr0aPsrVg90H21EOLACvtfcHk0lggQcZqOd4_lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1GhtbbL&md5=d3b3cdab8c5bd12ffb9340e94e21242d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja304782r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja304782r%26sid%3Dliteratum%253Aachs%26aulast%3DBothwell%26aufirst%3DI.%2BR.%26aulast%3DIslam%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DY.%2BL.%26aulast%3DZheng%26aufirst%3DW.%2BH.%26aulast%3DBlum%26aufirst%3DG.%26aulast%3DDeng%26aufirst%3DH.%2BT.%26aulast%3DLuo%26aufirst%3DM.%2BK.%26atitle%3DSe-Adenosyl-L-selenomethionine%2520cofactor%2520analogue%2520as%2520a%2520reporter%2520of%2520protein%2520methylation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D14905%26epage%3D14912%26doi%3D10.1021%2Fja304782r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Islam, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span> <span> </span><span class="NLM_article-title">Expanding cofactor repertoire of protein lysine methyltransferase for substrate labeling</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">679</span>– <span class="NLM_lpage">684</span>, <span class="refDoi"> DOI: 10.1021/cb2000567</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb2000567" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltVyjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=679-684&author=K.+Islamauthor=W.+Zhengauthor=H.+Yuauthor=H.+Dengauthor=M.+Luo&title=Expanding+cofactor+repertoire+of+protein+lysine+methyltransferase+for+substrate+labeling&doi=10.1021%2Fcb2000567"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Expanding Cofactor Repertoire of Protein Lysine Methyltransferase for Substrate Labeling</span></div><div class="casAuthors">Islam, Kabirul; Zheng, Weihong; Yu, Haiqiang; Deng, Haiteng; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">679-684</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases (PKMTs) play crucial roles in normal physiol. and disease processes.  Profiling PKMT targets is an important but challenging task.  With cancer-relevant G9a as a target, we have demonstrated success in developing S-adenosyl-L-methionine (SAM) analogs, particularly (E)-hex-2-en-5-ynyl SAM (Hey-SAM), as cofactors for engineered G9a.  Hey-SAM analog in combination with G9a Y1154A mutant modifies the same set of substrates as their native counterparts with remarkable efficiency.  (E)-Hex-2-en-5-ynylated substrates undergo smooth click reaction with an azide-based probe.  This approach is thus suitable for substrate characterization of G9a and expected to further serve as a starting point to evolve other PKMTs to utilize a similar set of cofactors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZw9uQ2VEeIrVg90H21EOLACvtfcHk0lggQcZqOd4_lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltVyjtr8%253D&md5=2401404608fddd56a7b07a27d9ab1647</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fcb2000567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb2000567%26sid%3Dliteratum%253Aachs%26aulast%3DIslam%26aufirst%3DK.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DExpanding%2520cofactor%2520repertoire%2520of%2520protein%2520lysine%2520methyltransferase%2520for%2520substrate%2520labeling%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2011%26volume%3D6%26spage%3D679%26epage%3D684%26doi%3D10.1021%2Fcb2000567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willnow, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscher, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinhold, E.</span></span> <span> </span><span class="NLM_article-title">A selenium-based click AdoMet analogue for versatile substrate labeling with wild-type protein methyltransferases</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1167</span>– <span class="NLM_lpage">1173</span>, <span class="refDoi"> DOI: 10.1002/cbic.201100781</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fcbic.201100781" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=22549896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt12hu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=1167-1173&author=S.+Willnowauthor=M.+Martinauthor=B.+Luscherauthor=E.+Weinhold&title=A+selenium-based+click+AdoMet+analogue+for+versatile+substrate+labeling+with+wild-type+protein+methyltransferases&doi=10.1002%2Fcbic.201100781"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A Selenium-Based Click AdoMet Analogue for Versatile Substrate Labeling with Wild-Type Protein Methyltransferases</span></div><div class="casAuthors">Willnow, Sophie; Martin, Michael; Luescher, Bernhard; Weinhold, Elmar</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1167-1173</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Protein methylation is catalyzed by S-adenosyl-L-methionine-dependent protein methyltransferases (MTases), and this posttranslational modification serves diverse cellular functions.  Some MTases seem to exhibit broad substrate specificities and comprehensive methods for target profiling are needed.  Here we report the synthesis of a new AdoMet analog for enzymic transfer of a small propargyl group and labeling of modified proteins through copper-catalyzed azide-alkyne cycloaddn. (CuAAC).  Replacement of sulfur by selenium strongly enhanced the stability of the progargylic cofactor, leading, in combination with better activation by the selenonium center, to higher enzymic reactivity.  A broad spectrum of wild-type protein MTases acting on lysine, arginine, and glutamine residues accept this cofactor and modified substrates can be efficiently labeled by CuAAC click chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1u6lpXCDHprVg90H21EOLACvtfcHk0lggQcZqOd4_lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt12hu7Y%253D&md5=e5fb071a350e100a863403cb429c9651</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201100781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201100781%26sid%3Dliteratum%253Aachs%26aulast%3DWillnow%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DM.%26aulast%3DLuscher%26aufirst%3DB.%26aulast%3DWeinhold%26aufirst%3DE.%26atitle%3DA%2520selenium-based%2520click%2520AdoMet%2520analogue%2520for%2520versatile%2520substrate%2520labeling%2520with%2520wild-type%2520protein%2520methyltransferases%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26spage%3D1167%26epage%3D1173%26doi%3D10.1002%2Fcbic.201100781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ilari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savino, C.</span></span> <span> </span><span class="NLM_article-title">A practical approach to protein crystallography</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1525</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-6622-6_3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1007%2F978-1-4939-6622-6_3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=27896717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVCjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1525&publication_year=2017&pages=47-78&author=A.+Ilariauthor=C.+Savino&title=A+practical+approach+to+protein+crystallography&doi=10.1007%2F978-1-4939-6622-6_3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A practical approach to protein crystallography</span></div><div class="casAuthors">Ilari, Andrea; Savino, Carmelinda</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1525</span>
        (<span class="NLM_cas:issue">Bioinformatics, Volume 1: Data, Sequence Analysis, and Evolution</span>),
    <span class="NLM_cas:pages">47-78</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Macromol. crystallog. is a powerful tool for structural biol.  The resoln. of a protein crystal structure is becoming much easier than in the past, thanks to developments in computing, automation of crystn. techniques and high-flux synchrotron sources to collect diffraction datasets.  The aim of this chapter is to provide practical procedures to det. a protein crystal structure, illustrating the new techniques, exptl. methods, and software that have made protein crystallog. a tool accessible to a larger scientific community.  It is impossible to give more than a taste of what the X-ray crystallog. technique entails in one brief chapter and there are different ways to solve a protein structure.  Since the no. of structures available in the Protein Data Bank (PDB) is becoming ever larger (the protein data bank now contains more than 100,000 entries) and therefore the probability to find a good model to solve the structure is ever increasing, we focus our attention on the Mol. Replacement method.  Indeed, whenever applicable, this method allows the resoln. of macromol. structures starting from a single data set and a search model downloaded from the PDB, with the aid only of computer work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbAoxlcHP5a7Vg90H21EOLACvtfcHk0lggQcZqOd4_lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVCjsbs%253D&md5=12a678a111723c52365ff3150c7db06e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-6622-6_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-6622-6_3%26sid%3Dliteratum%253Aachs%26aulast%3DIlari%26aufirst%3DA.%26aulast%3DSavino%26aufirst%3DC.%26atitle%3DA%2520practical%2520approach%2520to%2520protein%2520crystallography%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2017%26volume%3D1525%26spage%3D47%26epage%3D78%26doi%3D10.1007%2F978-1-4939-6622-6_3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">Molecular replacement with MOLREP</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1107%2FS0907444909042589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=20057045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=22-25&author=A.+Vaginauthor=A.+Teplyakov&title=Molecular+replacement+with+MOLREP&doi=10.1107%2FS0907444909042589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular replacement with MOLREP</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-25</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement that utilizes a no. of original approaches to rotational and translational search and data prepn.  Since the first publication describing the program, MOLREP has acquired a variety of features that include weighting of the X-ray data and search models, multi-copy search, fitting the model into electron d., structural superposition of two models and rigid-body refinement.  The program can run in a fully automatic mode using optimized parameters calcd. from the input data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzj5SzYvyu_7Vg90H21EOLACvtfcHk0liLk8Fyl5Q3_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D&md5=820d114719aca209994ffb0403e3b20d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042589%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMolecular%2520replacement%2520with%2520MOLREP%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D22%26epage%3D25%26doi%3D10.1107%2FS0907444909042589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guenther, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruner, S. D.</span></span> <span> </span><span class="NLM_article-title">Molecular basis of substrate promiscuity for the SAM-dependent <i>O</i>-methyltransferase NcsB1, involved in the biosynthesis of the enediyne antitumor antibiotic neocarzinostatin</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">9590</span>– <span class="NLM_lpage">9598</span>, <span class="refDoi"> DOI: 10.1021/bi901257q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi901257q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFemu7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=9590-9598&author=H.+A.+Cookeauthor=E.+L.+Guentherauthor=Y.+Luoauthor=B.+Shenauthor=S.+D.+Bruner&title=Molecular+basis+of+substrate+promiscuity+for+the+SAM-dependent+O-methyltransferase+NcsB1%2C+involved+in+the+biosynthesis+of+the+enediyne+antitumor+antibiotic+neocarzinostatin&doi=10.1021%2Fbi901257q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis of Substrate Promiscuity for the SAM-Dependent O-Methyltransferase NcsB1, Involved in the Biosynthesis of the Enediyne Antitumor Antibiotic Neocarzinostatin</span></div><div class="casAuthors">Cooke, Heather A.; Guenther, Elizabeth L.; Luo, Yinggang; Shen, Ben; Bruner, Steven D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">9590-9598</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The small mol. component of chromoprotein enediyne antitumor antibiotics is biosynthesized through a convergent route, incorporating amino acid, polyketide, and carbohydrate building blocks around a central enediyne hydrocarbon core.  The naphthoic acid moiety of the enediyne neocarzinostatin plays key roles in the biol. activity of the natural product by interacting with both the carrier protein and duplex DNA at the site of action.  The in vitro characterization of an S-adenosylmethionine-dependent O-methyltransferase (NcsB1) in the neocarzinostatin biosynthetic pathway has previously been described [Y. Luo et al. (2008)].  Here, a structural basis for NcsB1 activity is provided, illustrating that the enzyme shares an overall architecture with a large family of S-adenosylmethionine-dependent proteins.  In addn., NcsB1 represents the first enzyme to be structurally characterized in the biosynthetic pathway of neocarzinostatin.  By cocrystg. the enzyme with various combinations of the cofactor and substrate analogs, details of the active site structure have been established.  Changes in subdomain orientation were obsd. via comparison of structures in the presence and absence of substrate, suggesting that reorientation of the enzyme is involved in binding of the substrate.  In addn., residues important for substrate discrimination were predicted and probed through site-directed mutagenesis and in vitro biochem. characterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGricMGpJqkBz7Vg90H21EOLACvtfcHk0liLk8Fyl5Q3_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFemu7jE&md5=997f7ed8daf7a9bd97ad252516017ee0</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fbi901257q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi901257q%26sid%3Dliteratum%253Aachs%26aulast%3DCooke%26aufirst%3DH.%2BA.%26aulast%3DGuenther%26aufirst%3DE.%2BL.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DBruner%26aufirst%3DS.%2BD.%26atitle%3DMolecular%2520basis%2520of%2520substrate%2520promiscuity%2520for%2520the%2520SAM-dependent%2520O-methyltransferase%2520NcsB1%252C%2520involved%2520in%2520the%2520biosynthesis%2520of%2520the%2520enediyne%2520antitumor%2520antibiotic%2520neocarzinostatin%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D9590%26epage%3D9598%26doi%3D10.1021%2Fbi901257q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crnovcic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annaval, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nocek, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hindra</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babnigg, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joachimiak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. N.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span> <span> </span><span class="NLM_article-title">Resistance to enediyne antitumor antibiotics by sequestration</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1075</span>– <span class="NLM_lpage">1085</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.05.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2Fj.chembiol.2018.05.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=29937405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFyms7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=1075-1085&author=C.+Y.+Changauthor=X.+Yanauthor=I.+Crnovcicauthor=T.+Annavalauthor=C.+Changauthor=B.+Nocekauthor=J.+D.+Rudolfauthor=D.+Yangauthor=+Hindraauthor=G.+Babniggauthor=A.+Joachimiakauthor=G.+N.+Phillipsauthor=B.+Shen&title=Resistance+to+enediyne+antitumor+antibiotics+by+sequestration&doi=10.1016%2Fj.chembiol.2018.05.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to Enediyne Antitumor Antibiotics by Sequestration</span></div><div class="casAuthors">Chang, Chin-Yuan; Yan, Xiaohui; Crnovcic, Ivana; Annaval, Thibault; Chang, Changsoo; Nocek, Boguslaw; Rudolf, Jeffrey D.; Yang, Dong; Hindra; Babnigg, Gyorgy; Joachimiak, Andrzej; Phillips, George N. Jr.; Shen, Ben</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1075-1085.e4</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The enediynes, microbial natural products with extraordinary cytotoxicities, have been translated into clin. drugs.  Two self-resistance mechanisms are known in the enediyne producers-apoproteins for the nine-membered enediynes and self-sacrifice proteins for the ten-membered enediyne calicheamicin.  Here we show that: (1) tnmS1, tnmS2, and tnmS3 encode tiancimycin (TNM) resistance in its producer Streptomyces sp.  CB03234, (2) tnmS1, tnmS2, and tnmS3 homologs are found in all anthraquinone-fused enediyne producers, (3) TnmS1, TnmS2, and TnmS3 share a similar β barrel-like structure, bind TNMs with nanomolar KD values, and confer resistance by sequestration, and (4) TnmS1, TnmS2, and TnmS3 homologs are widespread in nature, including in the human microbiome.  These findings unveil an unprecedented resistance mechanism for the enediynes.  Mechanisms of self-resistance in producers serve as models to predict and combat future drug resistance in clin. settings.  Enediyne-based chemotherapies should now consider the fact that the human microbiome harbors genes encoding enediyne resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBh2PaCvbYc7Vg90H21EOLACvtfcHk0liLk8Fyl5Q3_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFyms7%252FF&md5=087aa51c9d8b8d27662f1c0c0a122e24</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.05.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.05.012%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DC.%2BY.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DCrnovcic%26aufirst%3DI.%26aulast%3DAnnaval%26aufirst%3DT.%26aulast%3DChang%26aufirst%3DC.%26aulast%3DNocek%26aufirst%3DB.%26aulast%3DRudolf%26aufirst%3DJ.%2BD.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DHindra%26aulast%3DBabnigg%26aufirst%3DG.%26aulast%3DJoachimiak%26aufirst%3DA.%26aulast%3DPhillips%26aufirst%3DG.%2BN.%26aulast%3DShen%26aufirst%3DB.%26atitle%3DResistance%2520to%2520enediyne%2520antitumor%2520antibiotics%2520by%2520sequestration%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D1075%26epage%3D1085%26doi%3D10.1016%2Fj.chembiol.2018.05.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liscombe, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louie, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noel, J. P.</span></span> <span> </span><span class="NLM_article-title">Architectures, mechanisms and molecular evolution of natural product methyltransferases</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1238</span>– <span class="NLM_lpage">1250</span>, <span class="refDoi"> DOI: 10.1039/c2np20029e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1039%2Fc2np20029e" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=22850796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlCjsbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2012&pages=1238-1250&author=D.+K.+Liscombeauthor=G.+V.+Louieauthor=J.+P.+Noel&title=Architectures%2C+mechanisms+and+molecular+evolution+of+natural+product+methyltransferases&doi=10.1039%2Fc2np20029e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Architectures, mechanisms and molecular evolution of natural product methyltransferases</span></div><div class="casAuthors">Liscombe, David K.; Louie, Gordon V.; Noel, Joseph P.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1238-1250</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The addn. of a Me moiety to a small chem. is a common transformation in the biosynthesis of natural products across all 3 domains of life.  These methylation reactions are most often catalyzed by S-adenosyl-L-methionine (SAM)-dependent methyltransferases (MTs).  MTs are categorized based on the electron-rich, Me accepting atom, usually O, N, C, or S.  SAM-dependent natural product MTs (NPMTs) are responsible for the modification of a wide array of structurally distinct substrates, including signaling and host defense compds., pigments, prosthetic groups, cofactors, cell membrane and cell wall components, and xenobiotics.  Most notably, methylation modulates the bioavailability, bioactivity, and reactivity of acceptor mols., and thus exerts a central role on the functional output of many metabolic pathways.  The current understanding of the structural enzymol. of NPMTs groups these phylogenetically diverse enzymes into 2 MT-superfamily fold classes (class I and class III).  Structural biol. has also shed light on the catalytic mechanisms and mol. bases for substrate specificity for >50 NPMTs.  These biophys.-based approaches have contributed to an understanding of NPMT evolution, demonstrating how a widespread protein fold evolved to accommodate chem. diverse Me acceptors and to catalyze disparate mechanisms suited to the physiochem. properties of the target substrates.  This evolutionary diversity suggests that NPMTs may serve as starting points for generating new biocatalysts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4FfgX1BfZj7Vg90H21EOLACvtfcHk0liESm8Gys4QWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlCjsbjE&md5=3a753fff151c8b6e1df0b2472ddeea16</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1039%2Fc2np20029e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2np20029e%26sid%3Dliteratum%253Aachs%26aulast%3DLiscombe%26aufirst%3DD.%2BK.%26aulast%3DLouie%26aufirst%3DG.%2BV.%26aulast%3DNoel%26aufirst%3DJ.%2BP.%26atitle%3DArchitectures%252C%2520mechanisms%2520and%2520molecular%2520evolution%2520of%2520natural%2520product%2520methyltransferases%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2012%26volume%3D29%26spage%3D1238%26epage%3D1250%26doi%3D10.1039%2Fc2np20029e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruner, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span> <span> </span><span class="NLM_article-title">Regiospecific <i>O</i>-methylation of naphthoic acids catalyzed by NcsB1, an <i>O</i>-methyltransferase involved in the biosynthesis of the enediyne antitumor antibiotic neocarzinostatin</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">14694</span>– <span class="NLM_lpage">14702</span>, <span class="refDoi"> DOI: 10.1074/jbc.M802206200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1074%2Fjbc.M802206200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=18387946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvFyjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=14694-14702&author=Y.+Luoauthor=S.+Linauthor=J.+Zhangauthor=H.+A.+Cookeauthor=S.+D.+Brunerauthor=B.+Shen&title=Regiospecific+O-methylation+of+naphthoic+acids+catalyzed+by+NcsB1%2C+an+O-methyltransferase+involved+in+the+biosynthesis+of+the+enediyne+antitumor+antibiotic+neocarzinostatin&doi=10.1074%2Fjbc.M802206200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Regiospecific O-Methylation of Naphthoic Acids Catalyzed by NcsB1, an O-Methyltransferase Involved in the Biosynthesis of the Enediyne Antitumor Antibiotic Neocarzinostatin</span></div><div class="casAuthors">Luo, Yinggang; Lin, Shuangjun; Zhang, Jian; Cooke, Heather A.; Bruner, Steven D.; Shen, Ben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">14694-14702</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Neocarzinostatin, a clin. anticancer drug, is the archetypal member of the chromoprotein family of enediyne antitumor antibiotics that are composed of a nonprotein chromophore and an apoprotein.  The neocarzinostatin chromophore consists of a nine-membered enediyne core, a deoxyaminosugar, and a naphthoic acid moiety.  We have previously cloned and sequenced the neocarzinostatin biosynthetic gene cluster and proposed that the biosynthesis of the naphthoic acid moiety and its incorporation into the neocarzinostatin chromophore are catalyzed by five enzymes NcsB, NcsB1, NcsB2, NcsB3, and NcsB4.  Here we report the biochem. characterization of NcsB1, unveiling that: (i) NcsB1 is an S-adenosyl-L-methionine-dependent O-methyltransferase; (ii) NcsB1 catalyzes regiospecific methylation at the 7-hydroxy group of its native substrate, 2,7-dihydroxy-5-methyl-1-naphthoic acid; (iii) NcsB1 also recognizes other dihydroxynaphthoic acids as substrates and catalyzes regiospecific O-methylation; and (iv) the carboxylate and its ortho-hydroxy groups of the substrate appear to be crucial for NcsB1 substrate recognition and binding, and O-methylation takes place only at the free hydroxy group of these dihydroxynaphthoic acids.  These findings establish that NcsB1 catalyzes the third step in the biosynthesis of the naphthoic acid moiety of the neocarzinostatin chromophore and further support the early proposal for the biosynthesis of the naphthoic acid and its incorporation into the neocarzinostatin chromophore with free naphthoic acids serving as intermediates.  NcsB1 represents another opportunity that can now be exploited to produce novel neocarzinostatin analogs by engineering neocarzinostatin biosynthesis or applying directed biosynthesis strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb6utg1AhukbVg90H21EOLACvtfcHk0liESm8Gys4QWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvFyjtr0%253D&md5=2afb1f4afeea179b502ece557118c4a1</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M802206200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M802206200%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DCooke%26aufirst%3DH.%2BA.%26aulast%3DBruner%26aufirst%3DS.%2BD.%26aulast%3DShen%26aufirst%3DB.%26atitle%3DRegiospecific%2520O-methylation%2520of%2520naphthoic%2520acids%2520catalyzed%2520by%2520NcsB1%252C%2520an%2520O-methyltransferase%2520involved%2520in%2520the%2520biosynthesis%2520of%2520the%2520enediyne%2520antitumor%2520antibiotic%2520neocarzinostatin%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D14694%26epage%3D14702%26doi%3D10.1074%2Fjbc.M802206200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hendricks, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichersky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noel, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z. S.</span></span> <span> </span><span class="NLM_article-title">An enzyme-coupled colorimetric assay for <i>S</i>-adenosylmethionine-dependent methyltransferases</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>326</i></span>,  <span class="NLM_fpage">100</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2003.11.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2Fj.ab.2003.11.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=14769341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtV2htr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=326&publication_year=2004&pages=100-105&author=C.+L.+Hendricksauthor=J.+R.+Rossauthor=E.+Picherskyauthor=J.+P.+Noelauthor=Z.+S.+Zhou&title=An+enzyme-coupled+colorimetric+assay+for+S-adenosylmethionine-dependent+methyltransferases&doi=10.1016%2Fj.ab.2003.11.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">An enzyme-coupled colorimetric assay for S-adenosylmethionine-dependent methyltransferases</span></div><div class="casAuthors">Hendricks, Cheryl L.; Ross, Jeannine R.; Pichersky, Eran; Noel, Joseph P.; Zhou, Zhaohui Sunny</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">326</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">100-105</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">An enzyme-coupled colorimetric assay for salicylate O-methyltransferase (SAMT), which utilizes S-adenosyl-L-methionine (AdoMet) as the Me donor, is reported.  In this assay, S-adenosyl-L-homocysteine (AdoHcy), a common product of AdoMet-dependent transmethylation reactions, is 1st hydrolyzed by recombinant adenosylhomocysteine nucleosidase (EC 3.2.2.9) into adenine and S-ribosylhomocysteine.  Recombinant LuxS ribosylhomocysteinase (EC 3.2.1.148) then cleaves the latter compd. to form homocysteine.  Finally, homocysteine is quantified using Ellman's reagent and the accompanying absorption change at 412 nm through recording using a microplate format.  Notably, SAMT and most AdoMet-dependent methyltransferases undergo marked AdoHcy-mediated product inhibition.  As such, an addnl. advantage of this assay which includes adenosylhomocysteine nucleosidase is the destruction of AdoHcy, thus alleviating product inhibition.  Under the authors' assay conditions, complete substrate conversion was obsd. and precise kinetic parameters could be detd. in a facile and quant. manner.  This assay should be generally applicable to other AdoMet-dependent methyltransferases.  Moreover, the procedure is easily amendable to batch assay and high-throughput screening approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWrYV3XlPCCrVg90H21EOLACvtfcHk0liESm8Gys4QWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtV2htr0%253D&md5=4a260e9377063acf07ac392164b2fb69</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2003.11.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2003.11.014%26sid%3Dliteratum%253Aachs%26aulast%3DHendricks%26aufirst%3DC.%2BL.%26aulast%3DRoss%26aufirst%3DJ.%2BR.%26aulast%3DPichersky%26aufirst%3DE.%26aulast%3DNoel%26aufirst%3DJ.%2BP.%26aulast%3DZhou%26aufirst%3DZ.%2BS.%26atitle%3DAn%2520enzyme-coupled%2520colorimetric%2520assay%2520for%2520S-adenosylmethionine-dependent%2520methyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D326%26spage%3D100%26epage%3D105%26doi%3D10.1016%2Fj.ab.2003.11.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cornell, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swarts, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barry, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riscoe, M. K.</span></span> <span> </span><span class="NLM_article-title">Characterization of recombinant <i>Escherichia coli</i> 5′-methylthioadenosine/<i>S</i>-adenosylhomocysteine nucleosidase: Analysis of enzymatic activity and substrate specificity</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>228</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">732</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1996.1723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1006%2Fbbrc.1996.1723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=8941345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADyaK28XntVaitr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=228&publication_year=1996&pages=724-732&author=K.+A.+Cornellauthor=W.+E.+Swartsauthor=R.+D.+Barryauthor=M.+K.+Riscoe&title=Characterization+of+recombinant+Escherichia+coli+5%E2%80%B2-methylthioadenosine%2FS-adenosylhomocysteine+nucleosidase%3A+Analysis+of+enzymatic+activity+and+substrate+specificity&doi=10.1006%2Fbbrc.1996.1723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of recombinant Escherichia coli 5'-methylthioadenosine/S-adenosylhomocysteine nucleosidase: analysis of enzymic activity and substrate specificity</span></div><div class="casAuthors">Cornell, Kenneth A.; Swarts, William E.; Barry, Ronald D.; Riscoe, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">228</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">724-732</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Recombinant E. coli 5'-methylthioadenosine (MTA)/S-adenosylhomocysteine nucleosidase (EC 3.2.2.9) (I) was used to study the potential for this enzyme to serve as a target for chemotherapeutic intervention.  An examn. of the parameters required for enzymic activity indicated that I functions over a broad range of pH and temp., with acidic conditions and temps. of 37-45° being optimal.  Analogs of MTA and adenosine were assessed as potential enzyme inhibitors and to provide details regarding substrate specificity and reaction mechanism.  The 5'-arylthio analog, 5'-(p-nitrophenyl)thioadenosine, was the most potent enzyme inhibitor studied, with a Ki of 20 nM.  A mutant of I lacking the 1st 8 amino acids was engineered to det. the contribution of these conserved residues toward enzyme specificity.  Truncated I exhibited a Km[MTA] of 1.43 μM, ∼3-fold higher than the Km reported for full-length I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-cVJrNdqwkrVg90H21EOLACvtfcHk0liESm8Gys4QWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XntVaitr4%253D&md5=8bedc21db3990089434f199bfb600594</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1996.1723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1996.1723%26sid%3Dliteratum%253Aachs%26aulast%3DCornell%26aufirst%3DK.%2BA.%26aulast%3DSwarts%26aufirst%3DW.%2BE.%26aulast%3DBarry%26aufirst%3DR.%2BD.%26aulast%3DRiscoe%26aufirst%3DM.%2BK.%26atitle%3DCharacterization%2520of%2520recombinant%2520Escherichia%2520coli%25205%25E2%2580%25B2-methylthioadenosine%252FS-adenosylhomocysteine%2520nucleosidase%253A%2520Analysis%2520of%2520enzymatic%2520activity%2520and%2520substrate%2520specificity%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1996%26volume%3D228%26spage%3D724%26epage%3D732%26doi%3D10.1006%2Fbbrc.1996.1723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunkara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goff, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Lanen, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorson, J. S.</span></span> <span> </span><span class="NLM_article-title">Facile chemoenzymatic strategies for the synthesis and utilization of <i>S</i>-adenosyl-L-methionine analogues</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">3965</span>– <span class="NLM_lpage">3969</span>, <span class="refDoi"> DOI: 10.1002/anie.201308272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fanie.201308272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkt1Sqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=3965-3969&author=S.+Singhauthor=J.+Zhangauthor=T.+D.+Huberauthor=M.+Sunkaraauthor=K.+Hurleyauthor=R.+D.+Goffauthor=G.+Wangauthor=W.+Zhangauthor=C.+Liuauthor=J.+Rohrauthor=S.+G.+Van+Lanenauthor=A.+J.+Morrisauthor=J.+S.+Thorson&title=Facile+chemoenzymatic+strategies+for+the+synthesis+and+utilization+of+S-adenosyl-L-methionine+analogues&doi=10.1002%2Fanie.201308272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Facile Chemoenzymatic Strategies for the Synthesis and Utilization of S-Adenosyl-L-Methionine Analogues</span></div><div class="casAuthors">Singh, Shanteri; Zhang, Jianjun; Huber, Tyler D.; Sunkara, Manjula; Hurley, Katherine; Goff, Randal D.; Wang, Guojun; Zhang, Wen; Liu, Chunming; Rohr, Juergen; Van Lanen, Steven G.; Morris, Andrew J.; Thorson, Jon S.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3965-3969</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A chemoenzymic platform for the synthesis of S-adenosyl-L-methionine (SAM) analogs compatible with downstream SAM-utilizing enzymes is reported.  Forty-four non-native S/Se-alkylated Met analogs were synthesized and applied to probing the substrate specificity of five diverse methionine adenosyltransferases (MATs).  Human MAT II was among the most permissive of the MATs analyzed and enabled the chemoenzymic synthesis of 29 non-native SAM analogs.  As a proof of concept for the feasibility of natural product "alkylrandomization", a small set of differentially-alkylated indolocarbazole analogs was generated by using a coupled hMAT2-RebM system (RebM is the sugar C4'-O-methyltransferase that is involved in rebeccamycin biosynthesis).  The ability to couple SAM synthesis and utilization in a single vessel circumvents issues assocd. with the rapid decompn. of SAM analogs and thereby opens the door for the further interrogation of a wide range of SAM utilizing enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6Qa-txIq8pbVg90H21EOLACvtfcHk0lgIZsq3L2wEaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkt1Sqtr0%253D&md5=25c1dd4656cb37918790d35e588a91bf</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fanie.201308272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201308272%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DHuber%26aufirst%3DT.%2BD.%26aulast%3DSunkara%26aufirst%3DM.%26aulast%3DHurley%26aufirst%3DK.%26aulast%3DGoff%26aufirst%3DR.%2BD.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DRohr%26aufirst%3DJ.%26aulast%3DVan%2BLanen%26aufirst%3DS.%2BG.%26aulast%3DMorris%26aufirst%3DA.%2BJ.%26aulast%3DThorson%26aufirst%3DJ.%2BS.%26atitle%3DFacile%2520chemoenzymatic%2520strategies%2520for%2520the%2520synthesis%2520and%2520utilization%2520of%2520S-adenosyl-L-methionine%2520analogues%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D3965%26epage%3D3969%26doi%3D10.1002%2Fanie.201308272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Presolski, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, M. G.</span></span> <span> </span><span class="NLM_article-title">Copper-catalyzed azide-alkyne click chemistry for bioconjugation</span>. <i>Curr. Protoc. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">153</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1002/9780470559277.ch110148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2F9780470559277.ch110148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=22844652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A280%3ADC%252BC2srmslKltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=153-162&author=S.+I.+Presolskiauthor=V.+P.+Hongauthor=M.+G.+Finn&title=Copper-catalyzed+azide-alkyne+click+chemistry+for+bioconjugation&doi=10.1002%2F9780470559277.ch110148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Copper-Catalyzed Azide-Alkyne Click Chemistry for Bioconjugation</span></div><div class="casAuthors">Presolski Stanislav I; Hong Vu Phong; Finn M G</div><div class="citationInfo"><span class="NLM_cas:title">Current protocols in chemical biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">153-162</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The copper-catalyzed azide-alkyne cycloaddition reaction is widely used for the connection of molecular entities of all sizes.  A protocol is provided here for the process with biomolecules.  Ascorbate is used as reducing agent to maintain the required cuprous oxidation state.  Since these convenient conditions produce reactive oxygen species, five equivalents of a copper-binding ligand is used with respect to metal.  The ligand both accelerates the reaction and serves as a sacrificial reductant, protecting the biomolecules from oxidation.  A procedure is also described for testing the efficiency of the reaction under desired conditions for purposes of optimization, before expensive biological reagents are used.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR8A2KT2nosgEg6wdEZmT2rfW6udTcc2eYigLAzSAaRTrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srmslKltg%253D%253D&md5=cc622ec6ed0e16170ef2e6ebd1541d0a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2F9780470559277.ch110148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9780470559277.ch110148%26sid%3Dliteratum%253Aachs%26aulast%3DPresolski%26aufirst%3DS.%2BI.%26aulast%3DHong%26aufirst%3DV.%2BP.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26atitle%3DCopper-catalyzed%2520azide-alkyne%2520click%2520chemistry%2520for%2520bioconjugation%26jtitle%3DCurr.%2520Protoc.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D3%26spage%3D153%26epage%3D162%26doi%3D10.1002%2F9780470559277.ch110148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nanna, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walseng, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedzisa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goydel, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hymel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, T. R.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roush, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rader, C.</span></span> <span> </span><span class="NLM_article-title">Harnessing a catalytic lysine residue for the one-step preparation of homogeneous antibody-drug conjugates</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1112</span>– <span class="NLM_lpage">1121</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01257-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1038%2Fs41467-017-01257-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=29062027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A280%3ADC%252BC1M7js12hsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1112-1121&author=A.+R.+Nannaauthor=X.+Liauthor=E.+Walsengauthor=L.+Pedzisaauthor=R.+S.+Goydelauthor=D.+Hymelauthor=T.+R.+Burkeauthor=W.+R.+Roushauthor=C.+Rader&title=Harnessing+a+catalytic+lysine+residue+for+the+one-step+preparation+of+homogeneous+antibody-drug+conjugates&doi=10.1038%2Fs41467-017-01257-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing a catalytic lysine residue for the one-step preparation of homogeneous antibody-drug conjugates</span></div><div class="casAuthors">Nanna Alex R; Li Xiuling; Walseng Even; Goydel Rebecca S; Rader Christoph; Nanna Alex R; Pedzisa Lee; Roush William R; Hymel David; Burke Terrence R Jr</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1112</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Current strategies to produce homogeneous antibody-drug conjugates (ADCs) rely on mutations or inefficient conjugation chemistries.  Here we present a strategy to produce site-specific ADCs using a highly reactive natural buried lysine embedded in a dual variable domain (DVD) format.  This approach is mutation free and drug conjugation proceeds rapidly at neutral pH in a single step without removing any charges.  The conjugation chemistry is highly robust, enabling the use of crude DVD for ADC preparation.  In addition, this strategy affords the ability to precisely monitor the efficiency of drug conjugation with a catalytic assay.  ADCs targeting HER2 were prepared and demonstrated to be highly potent and specific in vitro and in vivo.  Furthermore, the modular DVD platform was used to prepare potent and specific ADCs targeting CD138 and CD79B, two clinically established targets overexpressed in multiple myeloma and non-Hodgkin lymphoma, respectively.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9zxYb5ouxkFVwdslNBF-tfW6udTcc2eYigLAzSAaRTrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7js12hsA%253D%253D&md5=a9f2f5bc6d4a7fbb9595ca06d784868a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01257-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01257-1%26sid%3Dliteratum%253Aachs%26aulast%3DNanna%26aufirst%3DA.%2BR.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWalseng%26aufirst%3DE.%26aulast%3DPedzisa%26aufirst%3DL.%26aulast%3DGoydel%26aufirst%3DR.%2BS.%26aulast%3DHymel%26aufirst%3DD.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26aulast%3DRoush%26aufirst%3DW.%2BR.%26aulast%3DRader%26aufirst%3DC.%26atitle%3DHarnessing%2520a%2520catalytic%2520lysine%2520residue%2520for%2520the%2520one-step%2520preparation%2520of%2520homogeneous%2520antibody-drug%2520conjugates%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1112%26epage%3D1121%26doi%3D10.1038%2Fs41467-017-01257-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilchan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanna, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roush, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rader, C.</span></span> <span> </span><span class="NLM_article-title">Site-selective antibody functionalization via orthogonally reactive arginine and lysine residues</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1229</span>– <span class="NLM_lpage">1239</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1016%2Fj.chembiol.2019.05.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=31231031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1WqtL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=1229-1239&author=D.+Hwangauthor=N.+Nilchanauthor=A.+R.+Nannaauthor=X.+Liauthor=M.+D.+Cameronauthor=W.+R.+Roushauthor=H.+Parkauthor=C.+Rader&title=Site-selective+antibody+functionalization+via+orthogonally+reactive+arginine+and+lysine+residues&doi=10.1016%2Fj.chembiol.2019.05.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Site-selective antibody functionalization via orthogonally reactive arginine and lysine residues</span></div><div class="casAuthors">Hwang, Dobeen; Nilchan, Napon; Nanna, Alex R.; Li, Xiaohai; Cameron, Michael D.; Roush, William R.; Park, HaJeung; Rader, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1229-1239.e9</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Homogeneous antibody-drug conjugates (ADCs) that use a highly reactive buried lysine (Lys) residue embedded in a dual variable domain (DVD)-IgG1 format can be assembled with high precision and efficiency under mild conditions.  Here we show that replacing the Lys with an arginine (Arg) residue affords an orthogonal ADC assembly that is site-selective and stable.  X-ray crystallog. confirmed the location of the reactive Arg residue at the bottom of a deep pocket.  As the Lys-to-Arg mutation is confined to a single residue in the heavy chain of the DVD-IgG1, heterodimeric assemblies that combine a buried Lys in one arm, a buried Arg in the other arm, and identical light chains, are readily assembled.  Furthermore, the orthogonal conjugation chem. enables the loading of heterodimeric DVD-IgG1s with two different cargos in a one-pot reaction and thus affords a convenient platform for dual-warhead ADCs and other multifaceted antibody conjugates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7QYOXv5kWkrVg90H21EOLACvtfcHk0lhyeDJ5VZrhAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1WqtL%252FO&md5=fc7b7f5ab440e03d220b224ad305a612</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DHwang%26aufirst%3DD.%26aulast%3DNilchan%26aufirst%3DN.%26aulast%3DNanna%26aufirst%3DA.%2BR.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DRoush%26aufirst%3DW.%2BR.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DRader%26aufirst%3DC.%26atitle%3DSite-selective%2520antibody%2520functionalization%2520via%2520orthogonally%2520reactive%2520arginine%2520and%2520lysine%2520residues%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D1229%26epage%3D1239%26doi%3D10.1016%2Fj.chembiol.2019.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetsch, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumontet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corvaïa, N.</span></span> <span> </span><span class="NLM_article-title">Strategies and challenges for the next generation of antibody-drug conjugates</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">315</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.268</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1038%2Fnrd.2016.268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=28303026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1Onsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=315-337&author=A.+Beckauthor=L.+Goetschauthor=C.+Dumontetauthor=N.+Corva%C3%AFa&title=Strategies+and+challenges+for+the+next+generation+of+antibody-drug+conjugates&doi=10.1038%2Fnrd.2016.268"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies and challenges for the next generation of antibody-drug conjugates</span></div><div class="casAuthors">Beck, Alain; Goetsch, Liliane; Dumontet, Charles; Corvaia, Nathalie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">315-337</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Antibody-drug conjugates (ADCs) are one of the fastest growing classes of oncol. therapeutics.  After half a century of research, the approvals of brentuximab vedotin (in 2011) and trastuzumab emtansine (in 2013) have paved the way for ongoing clin. trials that are evaluating more than 60 further ADC candidates.  The limited success of first-generation ADCs (developed in the early 2000s) informed strategies to bring second-generation ADCs to the market, which have higher levels of cytotoxic drug conjugation, lower levels of naked antibodies and more-stable linkers between the drug and the antibody.  Furthermore, lessons learned during the past decade are now being used in the development of third-generation ADCs.  In this Review, we discuss strategies to select the best target antigens as well as suitable cytotoxic drugs; the design of optimized linkers; the discovery of bioorthogonal conjugation chemistries; and toxicity issues.  The selection and engineering of antibodies for site-specific drug conjugation, which will result in higher homogeneity and increased stability, as well as the quest for new conjugation chemistries and mechanisms of action, are priorities in ADC research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP7lV63Ws8rbVg90H21EOLACvtfcHk0lhyeDJ5VZrhAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1Onsbc%253D&md5=7358b6080dd1aed1d21e98c078cd9ef1</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.268%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DA.%26aulast%3DGoetsch%26aufirst%3DL.%26aulast%3DDumontet%26aufirst%3DC.%26aulast%3DCorva%25C3%25AFa%26aufirst%3DN.%26atitle%3DStrategies%2520and%2520challenges%2520for%2520the%2520next%2520generation%2520of%2520antibody-drug%2520conjugates%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D315%26epage%3D337%26doi%3D10.1038%2Fnrd.2016.268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suchand, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satyanarayana, G.</span></span> <span> </span><span class="NLM_article-title">Palladium-catalyzed direct acylation: one-pot relay synthesis of anthraquinones</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">779</span>, <span class="refDoi"> DOI: 10.1055/s-0037-1610296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1055%2Fs-0037-1610296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFSrur3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=769-779&author=B.+Suchandauthor=G.+Satyanarayana&title=Palladium-catalyzed+direct+acylation%3A+one-pot+relay+synthesis+of+anthraquinones&doi=10.1055%2Fs-0037-1610296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium-Catalyzed Direct Acylation: One-Pot Relay Synthesis of Anthraquinones</span></div><div class="casAuthors">Suchand, Basuli; Satyanarayana, Gedu</div><div class="citationInfo"><span class="NLM_cas:title">Synthesis</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">769-779</span>CODEN:
                <span class="NLM_cas:coden">SYNTBF</span>;
        ISSN:<span class="NLM_cas:issn">1437-210X</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">A bis-acylation strategy to access functionalized anthraquinones via one-pot relay process, is presented.  The first acylation was feasible under [Pd]-catalyzed intermol. direct acylation reaction, while, the second acylation was accomplished by using intramol. Friedel-Crafts acylation.  Notably, benchtop aldehydes were used as non-toxic acylation agents in the key [Pd]-catalyzed acylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMDvJyQj-cCLVg90H21EOLACvtfcHk0lhyeDJ5VZrhAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFSrur3L&md5=dcf159946c59a384a9b5d6be0b734965</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1055%2Fs-0037-1610296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0037-1610296%26sid%3Dliteratum%253Aachs%26aulast%3DSuchand%26aufirst%3DB.%26aulast%3DSatyanarayana%26aufirst%3DG.%26atitle%3DPalladium-catalyzed%2520direct%2520acylation%253A%2520one-pot%2520relay%2520synthesis%2520of%2520anthraquinones%26jtitle%3DSynthesis%26date%3D2019%26volume%3D51%26spage%3D769%26epage%3D779%26doi%3D10.1055%2Fs-0037-1610296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Somai
Magar, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. R.</span></span> <span> </span><span class="NLM_article-title">Organocatalyzed benzannulation for the construction of diverse anthraquinones and tetracenediones</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">8592</span>– <span class="NLM_lpage">8595</span>, <span class="refDoi"> DOI: 10.1039/C5CC00623F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1039%2FC5CC00623F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=25858160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsVWkt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=8592-8595&author=K.+B.+Somai%0AMagarauthor=L.+Xiaauthor=Y.+R.+Lee&title=Organocatalyzed+benzannulation+for+the+construction+of+diverse+anthraquinones+and+tetracenediones&doi=10.1039%2FC5CC00623F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Organocatalyzed benzannulation for the construction of diverse anthraquinones and tetracenediones</span></div><div class="casAuthors">Somai Magar, Krishna Bahadur; Xia, Likai; Lee, Yong Rok</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">8592-8595</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">An efficient one-pot synthesis of anthraquinones and tetracenediones was achieved by an L-proline catalyzed [4+2]cycloaddn. of in-situ generated azadiene from α,β-unsatd. aldehydes and 1,4-naphthoquinones or 1,4-anthracenedione in good to excellent yield.  This protocol constitutes an unprecedented tandem benzannulation that allows one-pot construction of diverse anthraquinones and tetracenediones in the presence of organocatalysts.  This methodol. was applied successfully to the synthesis of naturally occurring mols. and photochem. interesting phenanthrenequinone derivs.  The synthesis of the target compds. was achieved by a reaction of 1,4-naphthalenedione, 1,4-anthracenedione derivs. with crotonaldehyde, trans-2-hexenal, trans-2-pentenal.  The title compds. thus formed included 9,10-anthracenedione derivs. (anthraquinones), 5,12-naphthacenedione derivs. (naphthacenequinones).  9,10-Phenanthrenedione and 1,2-naphthalenedione were also prepd. from 1,2-naphthalenedione and 2,5-cyclohexadiene-1,4-dione.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmteJ58eVWJ7Vg90H21EOLACvtfcHk0lgUxR0y4JAWhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsVWkt7Y%253D&md5=a319ce64bd8a9bfeb8b06f5774bd26cf</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1039%2FC5CC00623F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC00623F%26sid%3Dliteratum%253Aachs%26aulast%3DSomai%2BMagar%26aufirst%3DK.%2BB.%26aulast%3DXia%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DY.%2BR.%26atitle%3DOrganocatalyzed%2520benzannulation%2520for%2520the%2520construction%2520of%2520diverse%2520anthraquinones%2520and%2520tetracenediones%26jtitle%3DChem.%2520Commun.%26date%3D2015%26volume%3D51%26spage%3D8592%26epage%3D8595%26doi%3D10.1039%2FC5CC00623F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montero, A.</span></span> <span> </span><span class="NLM_article-title">Asymmetric synthesis and biological properties of uncialamycin and 26-epi-uncialamycin</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">189</span>, <span class="refDoi"> DOI: 10.1002/anie.200704577</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1002%2Fanie.200704577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVShsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=185-189&author=K.+C.+Nicolaouauthor=J.+S.+Chenauthor=H.+Zhangauthor=A.+Montero&title=Asymmetric+synthesis+and+biological+properties+of+uncialamycin+and+26-epi-uncialamycin&doi=10.1002%2Fanie.200704577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Asymmetric synthesis and biological properties of uncialamycin and 26-epi-uncialamycin</span></div><div class="casAuthors">Nicolaou, K. C.; Chen, Jason S.; Zhang, Hongjun; Montero, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">185-189</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The highly potent DNA-cleaving mol. uncialamycin (I) was prepd. in an asym. total synthesis featuring an enantioselective Noyori redn.  Compd. I and its C26 epimer exhibit impressive broad-spectrum antibacterial properties and highly potent antitumor activities against a variety of cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVaG-7C5oaD7Vg90H21EOLACvtfcHk0lgUxR0y4JAWhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVShsw%253D%253D&md5=7e1d386d727427ceb611065c06d21d26</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fanie.200704577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200704577%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26aulast%3DChen%26aufirst%3DJ.%2BS.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DMontero%26aufirst%3DA.%26atitle%3DAsymmetric%2520synthesis%2520and%2520biological%2520properties%2520of%2520uncialamycin%2520and%252026-epi-uncialamycin%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2008%26volume%3D47%26spage%3D185%26epage%3D189%26doi%3D10.1002%2Fanie.200704577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasband, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliceiri, K. W.</span></span> <span> </span><span class="NLM_article-title">NIH Image to ImageJ: 25 years of image analysis</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">671</span>– <span class="NLM_lpage">675</span>, <span class="refDoi"> DOI: 10.1038/nmeth.2089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=10.1038%2Fnmeth.2089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=22930834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVKntb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=671-675&author=C.+A.+Schneiderauthor=W.+S.+Rasbandauthor=K.+W.+Eliceiri&title=NIH+Image+to+ImageJ%3A+25+years+of+image+analysis&doi=10.1038%2Fnmeth.2089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">NIH Image to ImageJ: 25 years of image analysis</span></div><div class="casAuthors">Schneider, Caroline A.; Rasband, Wayne S.; Eliceiri, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7_part1</span>),
    <span class="NLM_cas:pages">671-675</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">For the past 25 years NIH Image and ImageJ software have been pioneers as open tools for the anal. of scientific images.  We discuss the origins, challenges and solns. of these two programs, and how their history can serve to advise and inform other software projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofzHXrdLwDn7Vg90H21EOLACvtfcHk0lgUxR0y4JAWhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVKntb7P&md5=85ab928cd79f1e2f2351c834c0c600f0</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.2089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.2089%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DC.%2BA.%26aulast%3DRasband%26aufirst%3DW.%2BS.%26aulast%3DEliceiri%26aufirst%3DK.%2BW.%26atitle%3DNIH%2520Image%2520to%2520ImageJ%253A%252025%2520years%2520of%2520image%2520analysis%26jtitle%3DNat.%2520Methods%26date%3D2012%26volume%3D9%26spage%3D671%26epage%3D675%26doi%3D10.1038%2Fnmeth.2089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CLW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CLW','PDB','6CLW'); return false;">PDB: 6CLW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CLX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CLX','PDB','6CLX'); return false;">PDB: 6CLX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BBX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BBX','PDB','6BBX'); return false;">PDB: 6BBX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3I53" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3I53','PDB','3I53'); return false;">PDB: 3I53</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i22"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00799">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_65772"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00799?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00799</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Addition materials and methods; strains, plasmids, and primers used in this study (Tables S1–S3); summary of NMR data for <b>12</b>, <b>27</b>, and <b>28</b> (Table S4); crystal data and structural refinement for apo-TnmH and TnmH-SAM complex (Table S5); summary of NMR data for <b>31</b> (Table S6); anthraquinone-fused enediyne BGCs (Figure S1); sequence alignment for TnmH and selected phenolic <i>O</i>-methyltransferases (Figure S2); <i>tnl</i> BGC and encoded metabolites (Figure S3); construction and confirmation of Δ<i>tnmH</i> mutant strain <i>S</i>. sp. SB20024 (Figure S4); SDS–PAGE analysis of recombinant proteins TnmH, SAH nucleosidase, and TnmH mutants (Figure S5); size exclusion chromatography of TnmH (Figure S6); HR-ESI-MS analysis for <b>12</b> and <b>13</b> (Figure S7); pH and metal-ion dependence for TnmH O-methylation of <b>14</b> (Figure S8); time course of TnmH O-methylation of <b>11</b> (Figure S9); steady state kinetics for TnmH-catalyzed O-methylation of <b>10</b> and <b>11</b> (Figure S10); TnmH-catalyzed O-methylation of hydroxyanthraquinones <b>14</b>, <b>17</b>, and <b>18</b> (Figure S11); TnmH-catalyzed O-methylation of cycloaromatized enediyne congeners <b>15</b> and <b>16</b> (Figure S12); HR-ESI-MS analysis of O-methylated hydroxyanthraquinone products <b>19</b>–<b>23</b> (Figure S13); HR-ESI-MS analysis of the S-alkylated SAM analogues <b>24</b>–<b>26</b> (Figure S14); TnmH catalyzed preparation of <b>29</b> (Figure S15); end point assays of active site mutants of TnmH with <b>11</b> (Figure S16); end point assays of TnmH mutants with expanded SAM binding site with S-alkylated SAM analogues <b>24</b> and <b>25</b> (Figure S17); cell viability curves and DNA plasmid relaxation assays of selected TNMs (Figures S18 and S19); functionalization strategies for conjugation of <b>27</b>, <b>28</b>, <b>29</b> (Figure S20); HR-ESI-MS analysis, HPLC analysis for purity, and NMR spectra of 12, 27, 28, 31 (Figure S21–S51); HPLC analysis of purity of <b>3</b>, <b>10</b>, <b>11</b>, and <b>13</b> (Figures S52–S55) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings with associated data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_001.pdf">jm0c00799_si_001.pdf (4.86 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00799/suppl_file/jm0c00799_si_002.csv">jm0c00799_si_002.csv (2.41 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00799&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-15%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00799%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00799" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67996f8beed9227f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
