
<html lang="en"     class="pb-page"  data-request-id="b9a21a27-3219-4e6e-9669-0631c1ffbd07"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.7b01925;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2018.61.issue-15;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Detailed Exploration around 4-Aminoquinolines Chemical Space to Navigate the Lysine Methyltransferase G9a and DNA Methyltransferase Biological Spaces" /></meta><meta name="dc.Creator" content="Obdulia  Rabal" /></meta><meta name="dc.Creator" content="Juan Antonio  Sánchez-Arias" /></meta><meta name="dc.Creator" content="Edurne  San José-Enériz" /></meta><meta name="dc.Creator" content="Xabier  Agirre" /></meta><meta name="dc.Creator" content="Irene  de Miguel" /></meta><meta name="dc.Creator" content="Leire  Garate" /></meta><meta name="dc.Creator" content="Estibaliz  Miranda" /></meta><meta name="dc.Creator" content="Elena  Sáez" /></meta><meta name="dc.Creator" content="Sergio  Roa" /></meta><meta name="dc.Creator" content="José Angel  Martínez-Climent" /></meta><meta name="dc.Creator" content="Yingying  Liu" /></meta><meta name="dc.Creator" content="Wei  Wu" /></meta><meta name="dc.Creator" content="Musheng  Xu" /></meta><meta name="dc.Creator" content="Felipe  Prosper" /></meta><meta name="dc.Creator" content="Julen  Oyarzabal" /></meta><meta name="dc.Description" content="Epigenetic regulators that exhibit aberrant enzymatic activities or expression profiles are potential therapeutic targets for cancers. Specifically, enzymes responsible for methylation at histone-3..." /></meta><meta name="Description" content="Epigenetic regulators that exhibit aberrant enzymatic activities or expression profiles are potential therapeutic targets for cancers. Specifically, enzymes responsible for methylation at histone-3..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 11, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01925" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01925" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01925" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01925" /></link>
        
    
    

<title>Detailed Exploration around 4-Aminoquinolines Chemical Space to Navigate the Lysine Methyltransferase G9a and DNA Methyltransferase Biological Spaces | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01925" /></meta><meta property="og:title" content="Detailed Exploration around 4-Aminoquinolines Chemical Space to Navigate the Lysine Methyltransferase G9a and DNA Methyltransferase Biological Spaces" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0022.jpeg" /></meta><meta property="og:description" content="Epigenetic regulators that exhibit aberrant enzymatic activities or expression profiles are potential therapeutic targets for cancers. Specifically, enzymes responsible for methylation at histone-3 lysine-9 (like G9a) and aberrant DNA hypermethylation (DNMTs) have been implicated in a number of cancers. Recently, molecules bearing a 4-aminoquinoline scaffold were reported as dual inhibitors of these targets and showed a significant in vivo efficacy in animal models of hematological malignancies. Here, we report a detailed exploration around three growing vectors born by this chemotype. Exploring this chemical space led to the identification of features to navigate G9a and DNMT1 biological spaces: not only their corresponding exclusive areas, selective compounds, but also common spaces. Thus, we identified from selective G9a and first-in-class DNMT1 inhibitors, &gt;1 log unit between their IC50 values, with IC50 &lt; 25 nM (e.g., 43 and 26, respectively) to equipotent inhibitors with IC50 &lt; 50 nM for both targets (e.g., 13). Their ADME/Tox profiling and antiproliferative efficacies, versus some cancer cell lines, are also reported." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01925"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01925">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01925&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01925&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01925&amp;href=/doi/10.1021/acs.jmedchem.7b01925" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 6546-6573</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01926" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00050" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Detailed Exploration around 4-Aminoquinolines Chemical Space to Navigate the Lysine Methyltransferase G9a and DNA Methyltransferase Biological Spaces</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Obdulia Rabal</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Obdulia Rabal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Small Molecule Discovery Platform, Molecular Therapeutics Program, Ciberonc, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Obdulia++Rabal">Obdulia Rabal</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3224-0987" title="Orcid link">http://orcid.org/0000-0002-3224-0987</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Juan Antonio Sánchez-Arias</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Juan Antonio Sánchez-Arias</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Small Molecule Discovery Platform, Molecular Therapeutics Program, Ciberonc, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Juan+Antonio++S%C3%A1nchez-Arias">Juan Antonio Sánchez-Arias</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Edurne San José-Enériz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Edurne San José-Enériz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Area de Hemato-Oncología, IDISNA, Ciberonc, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Edurne++San+Jos%C3%A9-En%C3%A9riz">Edurne San José-Enériz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xabier Agirre</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xabier Agirre</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Area de Hemato-Oncología, IDISNA, Ciberonc, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xabier++Agirre">Xabier Agirre</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Irene de Miguel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Irene de Miguel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Small Molecule Discovery Platform, Molecular Therapeutics Program, Ciberonc, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Irene++de+Miguel">Irene de Miguel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Leire Garate</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Leire Garate</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Area de Hemato-Oncología, IDISNA, Ciberonc, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Leire++Garate">Leire Garate</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Estibaliz Miranda</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Estibaliz Miranda</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Area de Hemato-Oncología, IDISNA, Ciberonc, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Estibaliz++Miranda">Estibaliz Miranda</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elena Sáez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elena Sáez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Small Molecule Discovery Platform, Molecular Therapeutics Program, Ciberonc, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elena++S%C3%A1ez">Elena Sáez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sergio Roa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sergio Roa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Area de Hemato-Oncología, IDISNA, Ciberonc, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sergio++Roa">Sergio Roa</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">José Angel Martínez-Climent</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">José Angel Martínez-Climent</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Area de Hemato-Oncología, IDISNA, Ciberonc, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jos%C3%A9+Angel++Mart%C3%ADnez-Climent">José Angel Martínez-Climent</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yingying Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yingying Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi Apptec (Tianjin) Co. Ltd., TEDA, No. 111 HuangHai Road, Fourth Avenue, Tianjin 300456, PR China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yingying++Liu">Yingying Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wei Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wei Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi Apptec (Tianjin) Co. Ltd., TEDA, No. 111 HuangHai Road, Fourth Avenue, Tianjin 300456, PR China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Wu">Wei Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Musheng Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Musheng Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi Apptec (Tianjin) Co. Ltd., TEDA, No. 111 HuangHai Road, Fourth Avenue, Tianjin 300456, PR China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Musheng++Xu">Musheng Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Felipe Prosper</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Felipe Prosper</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Area de Hemato-Oncología, IDISNA, Ciberonc, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div><div class="loa-info-affiliations-info">Departmento de Hematología, Clinica Universidad de Navarra, University of Navarra, Avenida Pio XII 36, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Felipe++Prosper">Felipe Prosper</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Julen Oyarzabal</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Julen Oyarzabal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Small Molecule Discovery Platform, Molecular Therapeutics Program, Ciberonc, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +34 948 19 47 00, ext 2044. E-mail: <a href="/cdn-cgi/l/email-protection#573d223b3239382e36252d3635363b1722393621793224"><span class="__cf_email__" data-cfemail="d5bfa0b9b0bbbaacb4a7afb4b7b4b995a0bbb4a3fbb0a6">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Julen++Oyarzabal">Julen Oyarzabal</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1941-7255" title="Orcid link">http://orcid.org/0000-0003-1941-7255</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01925&amp;href=/doi/10.1021%2Facs.jmedchem.7b01925" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 6546–6573</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 11, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>30 December 2017</li><li><span class="item_label"><b>Published</b> online</span>11 June 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 August 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01925" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01925</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6546%26pageCount%3D28%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DObdulia%2BRabal%252C%2BJuan%2BAntonio%2BS%25C3%25A1nchez-Arias%252C%2BEdurne%2BSan%2BJos%25C3%25A9-En%25C3%25A9riz%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D15%26contentID%3Dacs.jmedchem.7b01925%26title%3DDetailed%2BExploration%2Baround%2B4-Aminoquinolines%2BChemical%2BSpace%2Bto%2BNavigate%2Bthe%2BLysine%2BMethyltransferase%2BG9a%2Band%2BDNA%2BMethyltransferase%2BBiological%2BSpaces%26numPages%3D28%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6573%26publicationDate%3DAugust%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01925"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1320</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">11</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01925" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Detailed Exploration around 4-Aminoquinolines Chemical Space to Navigate the Lysine Methyltransferase G9a and DNA Methyltransferase Biological Spaces&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Obdulia&quot;,&quot;last_name&quot;:&quot;Rabal&quot;},{&quot;first_name&quot;:&quot;Juan&quot;,&quot;last_name&quot;:&quot;Antonio Sánchez-Arias&quot;},{&quot;first_name&quot;:&quot;Edurne&quot;,&quot;last_name&quot;:&quot;San José-Enériz&quot;},{&quot;first_name&quot;:&quot;Xabier&quot;,&quot;last_name&quot;:&quot;Agirre&quot;},{&quot;first_name&quot;:&quot;Irene&quot;,&quot;last_name&quot;:&quot;de Miguel&quot;},{&quot;first_name&quot;:&quot;Leire&quot;,&quot;last_name&quot;:&quot;Garate&quot;},{&quot;first_name&quot;:&quot;Estibaliz&quot;,&quot;last_name&quot;:&quot;Miranda&quot;},{&quot;first_name&quot;:&quot;Elena&quot;,&quot;last_name&quot;:&quot;Sáez&quot;},{&quot;first_name&quot;:&quot;Sergio&quot;,&quot;last_name&quot;:&quot;Roa&quot;},{&quot;first_name&quot;:&quot;José&quot;,&quot;last_name&quot;:&quot;Angel Martínez-Climent&quot;},{&quot;first_name&quot;:&quot;Yingying&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Musheng&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Felipe&quot;,&quot;last_name&quot;:&quot;Prosper&quot;},{&quot;first_name&quot;:&quot;Julen&quot;,&quot;last_name&quot;:&quot;Oyarzabal&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;11&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;6546-6573&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01925&quot;},&quot;abstract&quot;:&quot;Epigenetic regulators that exhibit aberrant enzymatic activities or expression profiles are potential therapeutic targets for cancers. Specifically, enzymes responsible for methylation at histone-3 lysine-9 (like G9a) and aberrant DNA hypermethylation (DNMTs) have been implicated in a number of cancers. Recently, molecules bearing a 4-aminoquinoline scaffold were reported as dual inhibitors of these targets and showed a significant in vivo efficacy in animal models of hematological malignancies. Here, we report a detailed exploration around three growing vectors born by this chemotype. Exploring this chemical space led to the identification of features to navigate G9a and DNMT1 biological spaces: not only their corresponding exclusive areas, selective compounds, but also common spaces. Thus, we identified from selective G9a and first-in-class DNMT1 inhibitors, &gt;1 log unit between their IC50 values, with IC50 &lt; 25 nM (e.g., 43 and 26, respectively) to equipotent inhibitors with IC50 &lt; 50 nM for both targets (e&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01925&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01925" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01925&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01925" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01925&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01925" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01925&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01925&amp;href=/doi/10.1021/acs.jmedchem.7b01925" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01925" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01925" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01925%26sid%3Dliteratum%253Aachs%26pmid%3D29890830%26genre%3Darticle%26aulast%3DRabal%26date%3D2018%26atitle%3DDetailed%2BExploration%2Baround%2B4-Aminoquinolines%2BChemical%2BSpace%2Bto%2BNavigate%2Bthe%2BLysine%2BMethyltransferase%2BG9a%2Band%2BDNA%2BMethyltransferase%2BBiological%2BSpaces%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D15%26spage%3D6546%26epage%3D6573%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291300" title="Palladium">Palladium</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=290723" title="Solutions">Solutions</a>,</li><li><a href="/action/doSearch?ConceptID=291667" title="Vacuum">Vacuum</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/jmcmar.2018.61.issue-15/20180809/jmcmar.2018.61.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Epigenetic regulators that exhibit aberrant enzymatic activities or expression profiles are potential therapeutic targets for cancers. Specifically, enzymes responsible for methylation at histone-3 lysine-9 (like G9a) and aberrant DNA hypermethylation (DNMTs) have been implicated in a number of cancers. Recently, molecules bearing a 4-aminoquinoline scaffold were reported as dual inhibitors of these targets and showed a significant in vivo efficacy in animal models of hematological malignancies. Here, we report a detailed exploration around three growing vectors born by this chemotype. Exploring this chemical space led to the identification of features to navigate G9a and DNMT1 biological spaces: not only their corresponding exclusive areas, selective compounds, but also common spaces. Thus, we identified from selective G9a and first-in-class DNMT1 inhibitors, >1 log unit between their IC<sub>50</sub> values, with IC<sub>50</sub> < 25 nM (e.g., <b>43</b> and <b>26</b>, respectively) to equipotent inhibitors with IC<sub>50</sub> < 50 nM for both targets (e.g., <b>13</b>). Their ADME/Tox profiling and antiproliferative efficacies, versus some cancer cell lines, are also reported.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68766" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68766" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Methyltransferases are responsible for the methylation of their corresponding substrates (histones, proteins for protein methyltransferases (PMTs); or DNA, for DNA methyltransferases (DNMTs)) using <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (SAM or AdoMet) as cofactor, thus acting as epigenetic writers that control epigenetic gene regulation and carcinogenesis.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> G9a (also known as KMT1C or EHMT2) catalyzes mono- or dimethylation of histone H3 lysine 9 (H3K9) and other protein targets such as p53.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> G9a upregulation and overexpression is present in a variety of cancers.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> DNMT1, a protein catalyzing methylation of C5-cytosine of DNA, binds G9a and both proteins cooperate in promoting transcriptional silencing of target genes.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Moreover, pharmacologic or siRNA-mediated inhibition of G9a synergizes with DNMT1 inhibition in reducing cell proliferation.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> All these observations strongly suggested that dual inhibitors of both targets could be valuable anticancer agents, and we sought to identify such compounds. A number of selective G9a inhibitors have been reported,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> with substrate-competitive inhibitors <b>1</b> (UNC-0638)<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and <b>2</b> (A-366)<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> being extensively used as chemical probes to investigate the biological role of G9a in multiple diseases beyond cancer (<a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>). The development of potent reversible DNMT inhibitors is more appealing as most advanced compounds include nucleoside mimetics such as <b>3</b> (azacytidine) and <b>4</b> (decitabine) that incorporate into DNA and irreversible bind to DNMT1. As an alternative, current reversible non-nucleoside DNMT1 inhibitors lack potency, with IC<sub>50</sub> values in the low micromolar range.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a></div><figure id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Known G9a Inhibitors (<b>1</b>, <b>2</b>), Nucleoside DNMT1 Inhibitors (<b>3</b>, <b>4</b>), and Inhibitor (<b>5</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>G9a IC<sub>50</sub> values of <b>1</b>, <b>3</b>, <b>4</b>, and <b>5</b> and DNMT1 IC<sub>50</sub> values for <b>1</b> and <b>5</b> as determined internally (see footnote in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Alternatively, the reported IC<sub>50</sub> value for compound <b>1</b> (UNC-0638) was <15 nM (G9a, using a SAHH-coupled assay) and 1287 nM (DNMT1, using a radioactive methyl transfer assay).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Discrepancies with initially reported DNMT1 IC<sub>50</sub> values for <b>1</b> (IC<sub>50</sub> = 107000 nM) have been attributed to a different assay format.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> IC<sub>50</sub> values for compound <b>2</b><a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> (G9a, DNMT1), <b>3</b>,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and  <b>4</b><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> (DNMT1, corresponding to the lower concentration at which DNA hypomethylation is observed) have been previously reported.</p></p></figure><div class="NLM_p last">We recently reported the discovery of the quinoline-based lead compound <b>5</b> (CM-272) as an inhibitor of G9a (IC<sub>50</sub> = 8 nM) and DNMT1 (IC<sub>50</sub> = 382 nM) with in vivo efficacy in acute myeloid leukemia (AML), acute lymphoblastic leukemia (ALL) and diffuse large B-cell lymphoma (DLBCL) xenogeneic models.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Competition assays and microscale thermophoresis demonstrated that inhibitors of this chemical series bind to the substrate binding site of both targets.<a onclick="showRef(event, 'ref6 ref13'); return false;" href="javascript:void(0);" class="ref ref6 ref13">(6,13)</a> Our efforts to identify this lead compound utilized a combination of knowledge- and structure-based approaches, starting from the chemical structure of compound <b>1</b>. Initial structure–activity relationship (SAR) studies were focused on determining the optimal substitution pattern at the 2-position of the quinoline not only to achieve G9a and DNMT1 enzymatic inhibition but also a cellular response and a suitable pharmacokinetic (PK) profile for in vivo proof-of-concept.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Additional SAR exploration around the 4-, 6-, and 7-positions of the quinoline scaffold, aimed at further DNMT1 potency optimization and target selectivity modulation, identified key chemical features that confer selectivity toward G9a or DNMT1 (where selectivity involves >1 log unit difference between corresponding IC<sub>50</sub> values). In fact, first-in-class DNMT1 potent (low nanomolar, IC<sub>50</sub> < 25 nM), selective and reversible inhibitors are achieved. In addition, antiproliferative activities in cancer cells and preliminary ADME/Tox of this exploration are discussed. Furthermore, novel R-groups mimicking the lysine side chain of histone substrate (G9a) or the cytosine of DNA (DNMT1) are introduced. Our results and compounds not only represent interesting, and first-in-class, tools for targeting relevant epigenetic enzymes but also constitute valuable alternatives for the design of epigenetic inhibitors targeting the substrate-binding site of other methyltransferases.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81687" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81687" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> SAR in Biochemical G9a and DNMT1 Assays</h3><div class="NLM_p">SAR examination of the 2-position of the quinoline scaffold revealed 5-alkyl-2-furyl groups (5-methyl in <b>5</b> and 5-ethyl in <b>6</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) as the optimal groups for yielding functional cellular activity.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Thus, we decided to alternatively use these substituents as initial reference points for subsequent SAR exploration of the 4-, 6-, and 7-positions of the quinoline scaffold, using a sequential approach (<a class="ref internalNav" href="#cht2" aria-label="Chart 2">Chart 2</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Exploration of the 4-Position</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0014.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0015.gif" alt="" id="fx2" /></img><div></div></div><figure id="cht2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Chart 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 2. Sequential SAR Exploration Strategy for Positions 4, 6, and 7 of the Quinoline Scaffold; Position 2 Is Fixed with a 2-Furyl Ring Substituted at Position 5 (where R<sub>2</sub> is Methyl or Ethyl)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=cht2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The 4-amino moiety was first explored (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). According to the predicted binding mode of <b>5</b><a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and <b>6</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A,B) into both methyltransferases and X-ray structure of <b>1</b> and related quinazoline analogues into G9a,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> this position extends out toward the so-called solvent accessible area, suggesting that chemically diverse groups, exploring the biological space,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> could be well-tolerated. Removal of the basic nitrogen of <b>5</b> (compounds <b>7</b>–<b>11</b>, <b>16</b>) caused a significant drop in activity against both targets, and only the piperidone derivative <b>11</b> retained potent DNMT1 activity (IC<sub>50</sub> < 500 nM). This result is consistent with the predicted binding mode of <b>6</b> with G9a, as the positively charged nitrogen of the piperidine ring interacts with Asp1078 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). With respect to DNMT1, no explicit contacts were predicted by docking between the piperidine ring and the protein (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B), although a negative electrostatic potential is observed in this area (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). Homologation of the piperidine ring of <b>5</b> with a methylene linker (compounds <b>12</b><a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a>–<b>13</b>) greatly improved DNMT1 activity (IC<sub>50</sub> < 50 nM) while causing minor potency loss against G9a (IC<sub>50</sub> ∼ 15 nM). Given the poor pharmacokinetic profile of <b>12</b>,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> the <i>gem</i>-dimethyl analogue <b>15</b> was synthesized as the methylene linker was viewed as a potential metabolic liability. As <b>15</b> was significantly less potent against both targets, we concentrated on alternatives to the piperidine ring by limiting molecular flexibility<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> with spiropiperidines (<b>17</b>, <b>18</b>, <b>20</b>, <b>21</b>), spiropyrrolidines (<b>19</b>, <b>22</b>), and bridged piperazines (<b>23</b>–<b>26</b>). In general, these conformationally constrained analogues share a biochemical profile similar to that of <b>5</b>, especially in terms of selectivity toward G9a over DNMT1. Exceptionally, racemic analogues <b>20</b>–<b>22</b>, whose basic nitrogen predictably positions more distant from that of the piperidine (in green in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D compared to the other docked spiro derivatives in orange and <b>5</b> in violet, that overlap well), were less potent against G9a (IC<sub>50</sub> > 50 nM). DNMT1 SAR was flat, with no general improvements in activity (200–300 nM, pIC<sub>50</sub> ranging between 6.5 and 6.7, with less than 0.3 log units compared to <b>5</b> and <b>6</b>) except for some remarkable analogues such as the homologated 3-azabicyclo[3.2.1]octane of <b>25</b> (IC<sub>50</sub> = 80 nM). As expected,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> loss of the hydrogen bond donor of the 4-amino group of <b>5</b> by methylation (<b>27</b>) or replacement by an oxygen atom (<b>28</b>)<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> was detrimental for G9a activity. Interestingly, similar results were observed against DNMT1 and are in agreement with a predicted contact between this 4-amino group and Ser1233 of DNMT1 (Ser1230 in human DNMT1, hDNMT1, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B), what might explain the decreased DNMT1 activity of <b>27</b> and <b>28</b>. In summary, this exploration of the 4-position either mainly retained the selectivity profile of lead compound <b>5</b> (e.g., with constrained rings) or equally potent compounds were obtained at the cost of losing G9a activity (e.g., <b>11</b>). Only the homologated compounds <b>12</b> and <b>13</b> exhibit both G9a and DNMT1 activities in the low nanomolar range (<50 nM) and can be regarded as nonselective, especially in comparison with their counterparts <b>5</b> and <b>6</b> (absolute pIC<sub>50</sub> differences between both targets of 0.3 for <b>12</b> and <b>13</b> versus 1.7 (<b>5</b>) and 2.1 (<b>6</b>)).</div><figure id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0003.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Predicted complex of <b>6</b> with G9a (A, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW">3RJW</a>) and DNMT1 (B, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DA4">4DA4</a>). (C) Electrostatic potential of DNMT1 (only negative, in red), especially at the regions accommodating both basic centers of compound <b>6</b>. (D) Overlaid conformations of docked spirocyclic and bridged derivatives in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> to compound <b>5</b> (violet), showing that the basic nitrogen of compounds <b>20</b>–<b>22</b> (green, all stereoisomers considered) lies distal to the basic nitrogen of the rest of spirocycles and bridged structures (<b>17</b>–<b>19</b>, <b>23</b>–<b>26</b>, orange). (E) Overlaid conformations of docked compounds <b>5</b>, <b>41</b>, <b>43</b>, and <b>44</b> showing that the basic nitrogen overlays well among them.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For data in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a> and <a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>, all biochemical results are the average of at least two independent replicates performed at different days. If absolute pIC<sub>50</sub> difference was higher than 0.3 log units, additional replicates were performed until satisfying the experimental error (by discarding individual results with values outside 2 MADs of the mean value).</div><div class="NLM_p">Exploration of the 6-position with small groups (<b>29</b>–<b>33</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), designed to modulate ADME properties, produced a notable decrease in activity, especially against G9a, affording equally potent compounds against both targets in the midnanomolar or low-micromolar range. Only replacement of the methoxy group by chlorine (<b>31</b>) had negligible impact on DNMT1 activity (IC<sub>50</sub> = 298 nM) compared to <b>5</b>, so no further exploration around this position was continued. Inspection of the binding cavity for this derivatization position did not reveal any specific contacts between this substitution pattern and both proteins as to rationalize the weaker activities observed against both of them.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Exploration of the 6-Position</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0016.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0017.gif" alt="" id="fx4" /></img><div></div></div><div class="NLM_p">Finally, we modulated the 7-(3-pyrrolidin-1-ylpropoxy) moiety that predictably mimics the lysine side chain (G9a) or the cytosine of DNA (DNMT1) (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The 7-methoxy analogue <b>34</b>, lacking the basic nitrogen of <b>5</b> at the 7-position, was considerably less potent against G9a (IC<sub>50</sub> = 2060 nM, absolute pIC<sub>50</sub> difference of 2.4 log units), likely as a result of missing interactions with Leu1086 (hydrogen-bond) and Tyr1154 (cation−π interaction) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). The impact of this moiety on DNMT1 was less notorious (IC<sub>50</sub> raising from 382 nM for <b>5</b> to 750 nM for <b>34</b>, <0.3 log units), suggesting that the predicted interaction between the basic nitrogen of pyrrolidine and catalytic Glu1269 (Glu1266 in hDNMT1, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B) was not so important for the inhibitory activity of this chemical series. Thus, we speculated that selective DNMT1 inhibitors could be developed by exploring amine analogues of pyrrolidine with different basicity and distance extension from the quinoline core. Compounds in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> were designed and synthesized. G9a potency was attenuated in all cases, with piperidine <b>37</b> being the only replacement retaining G9a IC<sub>50</sub> value below 100 nM. Interestingly, the 4-piperidyl (<b>40</b>), 1-methyl-4-piperidyl (<b>41</b>), 2-methyl-2-azaspiro[3.3]heptan-6-yl (<b>43</b>), and 2-isopropyl-2-azaspiro[3.3]heptan-6-yl (<b>44</b>) groups conferred high DNMT1 potency (<100 nM), yielding compounds selective for DNMT1 over G9a (>1 log units). The reduced basicity of all these groups (estimated p<i>K</i><sub>a</sub> with Pipeline Pilot<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> for the amine nitrogen at position 7 of 6.33, 8.75, and 7.26 for <b>40</b>, <b>41</b> and <b>43</b>, respectively), compared to that of the pyrrolidin ring of <b>5</b> (p<i>K</i><sub>a</sub> of 12.1) and <b>37</b> (p<i>K</i><sub>a</sub> of 11.8) seems to play a major role on target selectivity, as on the other hand the cationic environment overlaps well among all these derivatives (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E). While some of these amines in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> have already been examined as lysine mimetics<a onclick="showRef(event, 'ref10 ref18 ref20'); return false;" href="javascript:void(0);" class="ref ref10 ref18 ref20">(10,18,20)</a> that reduce the lipophilicity of the parent compound <b>5</b> (acyclic amine <b>35</b>, piperidine <b>37</b>, morpholine <b>38</b>, 3-(3-fluoropyrrolidin-1-yl)propyl <b>39</b>), the DNMT1 potency of the spirocyclic derivative <b>43</b> (IC<sub>50</sub> = 21 nM) was considered as a key SAR finding and a novelty in epigenetic drug design. Decreasing its basicity with <i>N</i>-isopropyl (<b>44</b>) and <i>N</i>-cyclopropyl (<b>45</b>) capping groups correlated with loss of potency. Compounds <b>46</b>–<b>50</b> were obtained by combining the identified optimal groups at positions 4 and 7 (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Unfortunately, the excellent DNMT1 potency of <b>43</b> was not retained when this group was combined with the most potent 4-substituents in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> to afford analogues <b>47</b>–<b>50</b>, suggesting nonadditive SAR effects.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Despite lower DNMT1 inhibitory activity (between 187 and 501 nM for <b>47</b>–<b>50</b>), all these compounds displayed DNMT1 selectivity over G9a comparable to that of <b>43</b>.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Exploration of the 7-Position</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0018.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0019.gif" alt="" id="fx6" /></img><div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Combined Exploration of Positions <b>4</b> and <b>7</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0020.gif" alt="" id="fx7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0021.gif" alt="" id="fx8" /></img><div></div></div><div class="NLM_p last">Having identified interesting compounds with different selectivity profiles toward G9a and/or DNMT1 (<b>13</b>, <b>43</b>), we analyzed their selectivity at 10 μM against a panel of epigenetic targets comprising 14 lysine and arginine methyltransferases, DNMTs (DNMT3A, DNMT3B), bromodomains (BRD2, CREBBP, and BAZ2B) and histone demethylases (JMJD2C, JMJD3, JMJD1A) (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information, Table 1</a>). Despite that we initially anticipated potent GLP inhibition (as for <b>12</b><a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a>), given the close similarity among G9a and GLP, compounds <b>13</b> and <b>43</b> displayed high selectivity toward G9a (GLP IC<sub>50</sub> values higher than 20 μM). Reasons for this selectivity are not clear considering the high structural similarity between <b>12</b> and <b>13</b>. Oppositely, in agreement with initial prediction, all three compounds are potent inhibitors of DNMT3A and DNMT3B (>95% inhibition at 10 μM). For the remaining targets, inhibition values were <50%, with the exception of PRMT1 for <b>13</b> (84%) and JMJD3 for <b>43</b> (71%).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Antiproliferative Effect and Preliminary ADME</h3><div class="NLM_p">Following our screening funnel, the antiproliferative response of compounds with potent inhibitory activity (G9a < 100 nM and DNMT1 < 500 nM) was tested against ALL cell lines (CEMO-1) and DLBCL (OCI-Ly3 and OCI-Ly10) cell lines (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). All selected compounds displayed potent antiproliferative effects, with GI<sub>50</sub> values below 1 μM against at least one cell line. However, no evident correlation was found between G9a/DNMT1 activity/selectivity, passive membrane permeability of compounds as measured using the parallel artificial membrane permeation assay (PAMPA), and phenotypic response, this last being greatly dependent on the cell type. For example, compounds <b>12</b> and <b>43</b>, two of the most interesting compounds from the viewpoint of G9a/DNMT1 selectivity, mainly differing in their G9a activity (16 versus 351 nM) and being equally very potent against DNMT1 (32 versus 21 nM), have the same growth inhibitory activity against CEMO-1 (GI<sub>50</sub> ∼ 75 nM) but diverse response against DLBCL cell lines: OCI-Ly10 cell line (<31 nM versus 253 nM for <b>12</b> and <b>43</b>, respectively) and the opposite response against OCI-Ly3 (202 versus 74 nM for <b>12</b> and <b>43</b>, respectively). Low toxicity of compounds, assayed against the nontumoral hepatic cell line THLE-2 (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>), was regarded as a requisite for further compound progression into in vitro preliminary ADME, ideally affording a therapeutic window (absolute difference between pGI<sub>50</sub> and pLC<sub>50</sub>) of a least one log unit against one out of the four tumoral cell lines (mostly CEMO-1, <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Selected compounds in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a> showed low cytochrome P450 inhibition of five major isoforms (<25% at 10 μM) and were inactive against hERG (pIC<sub>50</sub> < 4), thus limiting the potential for cardiotoxicity. As no compound with a novel selectivity profile (e.g., <b>43</b>) improved the metabolic stability of lead compound <b>5</b>, especially in human liver microsomes, compounds were not further progressed into PK studies.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Antiproliferative Response of Selected Compounds against Cancer Cell Lines and Nontumoural Hepatic Cell Line THLE-2</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="center" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Cpd</th><th class="colsep0 rowsep0" align="center" char=".">CEMO-1 GI<sub>50</sub> (nM)<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">OCI-Ly3 GI<sub>50</sub> (nM)<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">OCI-Ly10 GI<sub>50</sub> (nM)<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">THLE-2 LC<sub>50</sub> (nM) 72 h<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">PAMPA <i>P</i><sub>e</sub> (nm/s)<a class="ref internalNav" href="#t5fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center" char=".">G9a pIC<sub>50</sub>–DNMT1 pIC<sub>50</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b><a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></td><td class="colsep0 rowsep0" align="char" char=".">218</td><td class="colsep0 rowsep0" align="center">409</td><td class="colsep0 rowsep0" align="center">455</td><td class="colsep0 rowsep0" align="char" char=".">1780</td><td class="colsep0 rowsep0" align="char" char=".">12.9</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b><a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></td><td class="colsep0 rowsep0" align="char" char=".">56</td><td class="colsep0 rowsep0" align="center">95</td><td class="colsep0 rowsep0" align="center">64</td><td class="colsep0 rowsep0" align="char" char=".">2320</td><td class="colsep0 rowsep0" align="char" char=".">23.9</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="char" char=".">172</td><td class="colsep0 rowsep0" align="center">215</td><td class="colsep0 rowsep0" align="center">170</td><td class="colsep0 rowsep0" align="char" char=".">7950</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b><a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></td><td class="colsep0 rowsep0" align="char" char=".">75</td><td class="colsep0 rowsep0" align="center">202</td><td class="colsep0 rowsep0" align="center"><31</td><td class="colsep0 rowsep0" align="char" char=".">1300</td><td class="colsep0 rowsep0" align="char" char=".">11.0</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">93</td><td class="colsep0 rowsep0" align="center">277</td><td class="colsep0 rowsep0" align="center">133</td><td class="colsep0 rowsep0" align="char" char=".">954</td><td class="colsep0 rowsep0" align="char" char=".">16.2</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="char" char=".">659</td><td class="colsep0 rowsep0" align="center">ND<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="char" char=".">777</td><td class="colsep0 rowsep0" align="char" char=".">19.2</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="char" char=".">235</td><td class="colsep0 rowsep0" align="center">243</td><td class="colsep0 rowsep0" align="center">100</td><td class="colsep0 rowsep0" align="char" char=".">992</td><td class="colsep0 rowsep0" align="char" char=".">10.5</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="char" char=".">426</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char=".">841</td><td class="colsep0 rowsep0" align="char" char=".">19.3</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="char" char=".">768</td><td class="colsep0 rowsep0" align="center">374</td><td class="colsep0 rowsep0" align="center">257</td><td class="colsep0 rowsep0" align="char" char=".">8380</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="char" char=".">424</td><td class="colsep0 rowsep0" align="center">246</td><td class="colsep0 rowsep0" align="center">112</td><td class="colsep0 rowsep0" align="char" char=".">731</td><td class="colsep0 rowsep0" align="char" char=".">15.9</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="char" char=".">186</td><td class="colsep0 rowsep0" align="center">237</td><td class="colsep0 rowsep0" align="center">638</td><td class="colsep0 rowsep0" align="char" char=".">517</td><td class="colsep0 rowsep0" align="char" char=".">37.7</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="char" char=".">252</td><td class="colsep0 rowsep0" align="center">205</td><td class="colsep0 rowsep0" align="center">131</td><td class="colsep0 rowsep0" align="char" char=".">1980</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="char" char=".">156</td><td class="colsep0 rowsep0" align="center">557</td><td class="colsep0 rowsep0" align="center">313</td><td class="colsep0 rowsep0" align="char" char=".">527</td><td class="colsep0 rowsep0" align="char" char=".">33.7</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="char" char=".">115</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="char" char="."><164</td><td class="colsep0 rowsep0" align="char" char=".">36.1</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="char" char=".">671</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="char" char=".">4150</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="char" char=".">782</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="char" char=".">2310</td><td class="colsep0 rowsep0" align="char" char=".">24.5</td><td class="colsep0 rowsep0" align="char" char=".">–0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="char" char=".">370</td><td class="colsep0 rowsep0" align="center">86</td><td class="colsep0 rowsep0" align="center">1100</td><td class="colsep0 rowsep0" align="char" char=".">3520</td><td class="colsep0 rowsep0" align="char" char=".">14.9</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="char" char=".">184</td><td class="colsep0 rowsep0" align="center">230</td><td class="colsep0 rowsep0" align="center">596</td><td class="colsep0 rowsep0" align="char" char=".">2040</td><td class="colsep0 rowsep0" align="char" char=".">36.6</td><td class="colsep0 rowsep0" align="char" char=".">–0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="char" char=".">196</td><td class="colsep0 rowsep0" align="center">271</td><td class="colsep0 rowsep0" align="center">576</td><td class="colsep0 rowsep0" align="char" char=".">1380</td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="char" char=".">1070</td><td class="colsep0 rowsep0" align="center"><63</td><td class="colsep0 rowsep0" align="center">1380</td><td class="colsep0 rowsep0" align="char" char=".">24800</td><td class="colsep0 rowsep0" align="char" char=".">2.9</td><td class="colsep0 rowsep0" align="char" char=".">–1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41</b></td><td class="colsep0 rowsep0" align="char" char=".">272</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="char" char=".">1030</td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">–1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="char" char=".">73</td><td class="colsep0 rowsep0" align="center">74</td><td class="colsep0 rowsep0" align="center">253</td><td class="colsep0 rowsep0" align="char" char=".">698</td><td class="colsep0 rowsep0" align="char" char=".">7.1</td><td class="colsep0 rowsep0" align="char" char=".">–1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46</b></td><td class="colsep0 rowsep0" align="char" char=".">504</td><td class="colsep0 rowsep0" align="center">12</td><td class="colsep0 rowsep0" align="center">186</td><td class="colsep0 rowsep0" align="char" char=".">2530</td><td class="colsep0 rowsep0" align="char" char=".">14.9</td><td class="colsep0 rowsep0" align="char" char=".">–1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>49</b></td><td class="colsep0 rowsep0" align="char" char=".">452</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="char" char=".">4650</td><td class="colsep0 rowsep0" align="char" char=".">11.1</td><td class="colsep0 rowsep0" align="char" char=".">–1.4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">ND = not determined.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">Proliferation assays are the average of three replicates at different days.</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">THLE-2 cytotoxicity results after 72 h of incubation are the average of at least two independent experiments performed at different days. If absolute pLC<sub>50</sub> difference was higher than 1 log unit, additional replicates were performed until satisfying the experimental error (by discarding individual results with values outside 3 MADs of the mean value).</p></div><div class="footnote" id="t5fn4"><sup><sup>d</sup></sup><p class="last">The PAMPA assay was performed in triplicate. Depending on permeability values (<i>P</i><sub>e</sub>, nm/s), compounds can be regarded as poor (<i>P</i><sub>e</sub> < 10 nm/s), moderate (10 < <i>P</i><sub>e</sub> < 30 nm/s), or good (>30 nm/s) permeators.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></p></div></div></div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. ADME Profile of Selected Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Cpd</th><th class="colsep0 rowsep0" align="center" char=".">1A2 (%)<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">2C9 (%)<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">2C19 (%)<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">2D6 (%)<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">3A4 (%)<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">HLM (%)<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">MLM (%)<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">hERG IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b><a onclick="showRef(event, 'ref6 ref13'); return false;" href="javascript:void(0);" class="ref ref6 ref13">(6,13)</a></td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">0.56</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">4.9</td><td class="colsep0 rowsep0" align="char" char=".">0.0</td><td class="colsep0 rowsep0" align="char" char=".">99.7</td><td class="colsep0 rowsep0" align="char" char=".">70.9</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="char" char=".">9.3</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="char" char=".">22.4</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">82.7</td><td class="colsep0 rowsep0" align="char" char=".">71.9</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">7.1</td><td class="colsep0 rowsep0" align="char" char=".">16.8</td><td class="colsep0 rowsep0" align="char" char=".">23.1</td><td class="colsep0 rowsep0" align="char" char=".">19.3</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">69.0</td><td class="colsep0 rowsep0" align="char" char=".">59.1</td><td class="colsep0 rowsep0" align="center">ND<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">68.6</td><td class="colsep0 rowsep0" align="char" char=".">40.6</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">80.4</td><td class="colsep0 rowsep0" align="char" char=".">53.3</td><td class="colsep0 rowsep0" align="center">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">6.3</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">60.3</td><td class="colsep0 rowsep0" align="char" char=".">43.9</td><td class="colsep0 rowsep0" align="center">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="char" char=".">7.2</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">81.7</td><td class="colsep0 rowsep0" align="char" char=".">81.7</td><td class="colsep0 rowsep0" align="center">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="char" char=".">6.1</td><td class="colsep0 rowsep0" align="char" char=".">6.3</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">21.5</td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">49.8</td><td class="colsep0 rowsep0" align="char" char=".">43.3</td><td class="colsep0 rowsep0" align="center">ND</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">ND = not determined.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">Percent inhibition at 10 μM.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">Percent compound remaining after a 20 min incubation in human or mouse liver microsomes (HLM and MLM respectively). All assays were performed in duplicate, with the exception of hERG binding data (<i>n</i> = 1).</p></div></div></div><div class="NLM_p">Finally, the functional cellular potency of selected compounds was assessed by quantifying the global levels of H3K9me2 (Western Blot) and 5-methylcytosine (5mC) (Dot Blot) in the CEMO-1 cell line following 96 h of exposure. As an example, the selected reference compounds <b>13</b> and <b>43</b> were tested. Both compounds reduced the H3K9me2 and 5mC levels in a concentration-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Western Blots were quantized and reported in the Supporting Information (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Figure S1</a>). In summary, these results demonstrate the functional dual effect of compounds <b>13</b> and <b>43</b> against the methyltranferase activity of G9a and DNMTs.</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0004.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. H3K9me2 and 5mC hallmarks in the CEMO-1 cell line after treatment with compounds <b>13</b> and <b>43</b>. (A) H3K9me2 levels after 96 h of treatment with different doses of compounds <b>13</b> and <b>43</b> in CEMO-1 cell line. H3 total was used as loading control. (B) 5mC levels after 96 h of treatment with different doses of compounds <b>13</b> and <b>43</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemistry</h3><div class="NLM_p">Preparation of compounds <b>7</b>–<b>11</b>, <b>13</b>, and <b>16</b> with different substituents at 4-position of the quinoline were prepared as outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> from commercially available 2-methoxy-5-nitro-phenol (<b>51</b>). This alcohol was first converted into compound <b>52</b> through Mitsunobu reaction, and then intermediate <b>54</b> was achieved after hydrogenation and reaction with POCl<sub>3</sub> in malonic acid. Then, Suzuki coupling with boronic esters 2-(5-ethyl-2-furyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane or 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane afforded compounds <b>55a</b> and <b>55b</b>, which were finally converted into desired 4-aminoquinolines <b>7</b>–<b>11</b>, <b>13</b>, and <b>16</b> by Pd-coupling reaction using different amines.</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0005.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>7</b>–<b>11</b>, <b>13</b>, and <b>16</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) 3-pyrrolidin-1-yl-propan-1-ol, PPh<sub>3</sub>, DEAD, THF, rt, 5 h; (ii) Pd/C, H<sub>2</sub> (1 atm), MeOH, rt, 3 h; (iii) POCl<sub>3</sub>, malonic acid, rt, 4 h, then, 90 °C, overnight; (iv) 2-(5-ethyl-2-furyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane or 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane, Na<sub>2</sub>CO<sub>3</sub> or K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane, or 1,4-dioxane/H<sub>2</sub>O (15:1), 110 °C, MW or conventional heating, 4–12 h; (v) corresponding amine, Cs<sub>2</sub>CO<sub>3</sub>, BINAP, Pd<sub>2</sub>(dba)<sub>3</sub>, 1,4-dioxane, 120–130 °C, MW or conventional heating, 3–12 h; (vi) corresponding amine, <i>t</i>-BuONa, xantphos, Pd<sub>2</sub>(dba)<sub>3</sub>, toluene, 100 °C, MW, 2 h.</p></p></figure><div class="NLM_p">Synthesis of isopropyl analogue <b>14</b>, <i>gem</i>-dimethyl analogue <b>15</b>, spiropiperidines <b>17</b>, <b>18</b>, <b>20</b>, and <b>21</b>, spiropyrrolidines <b>19</b> and <b>22</b>, bridged piperazines <b>23</b>–<b>26</b>, and methylated analogue <b>27</b> is described in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Starting from previously described 4-chloroquinolines <b>55a</b> or <b>55b</b>, BOC-protected amines <b>56a</b>–<b>k</b> were prepared by Buchwald–Hartwig amination and tertiary amine <b>56l</b> using <i>tert</i>-butyl 4-(methylamino)piperidine-1-carboxylate and PTSA in <i>t</i>-BuOH. Then, the BOC protecting group was removed under acidic conditions and desired methyl or isopropyl substituent was installed by reductive amination. All analogues with chiral centers and pseudostereocenters in this manuscript (<b>11</b>, <b>19</b>–<b>26</b>, <b>39</b>, <b>50</b>) were prepared as racemic mixtures.</div><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0006.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>14</b>, <b>15</b>, and <b>17</b>–<b>27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) corresponding amine, Cs<sub>2</sub>CO<sub>3</sub>, BINAP, Pd<sub>2</sub>(dba)<sub>3</sub>, 1,4-dioxane, 115–130 °C, MW or conventional heating, 5–48 h; (ii) <i>tert</i>-butyl 4-(methylamino)piperidine-1-carboxylate, PTSA, <i>t</i>-BuOH, 120 °C, 48 h; (iii) HCl/EtOAc (1.0 or 2.0 M) or HCl/MeOH (2.0 or 4.0 M), 16–25 °C, 1–16 h; (iv) acetone, NaBH<sub>3</sub>CN, AcOH, <i>i</i>-PrOH, 50–60 °C, 15–16 h; (v) (HCHO)<sub>n</sub>, HCOOH, NaBH(OAc)<sub>3</sub>, MeOH, rt or 60 °C, 12 h or overnight.</p></p></figure><div class="NLM_p">Compounds with a hydrogen atom, a chlorine atom and a −OCF<sub>3</sub> group at position 6 of the quinoline (<b>29</b>, <b>31</b>, and <b>32</b>, respectively) were synthesized from intermediates <b>60a</b>–<b>c</b> through Mitsunobu reaction and reduction of nitro group (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Then, key intermediates <b>63a</b>–<b>b</b> were achieved after heating with POCl<sub>3</sub> and malonic acid and compound <b>63c</b> by a three-step protocol (reaction with ethyl 3-chloro-3-oxo-propanoate, ester hydrolysis, and reaction with POCl<sub>3</sub>). Subsequent reaction with 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane and Buchwald–Hartwig amination led us to desired compounds <b>29</b>, <b>31</b>, and <b>32</b>.</div><figure id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0007.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>29</b>, <b>31</b>, and <b>32</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) 4,4,5,5-tetramethyl-2-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1,3,2-dioxaborolane, KOAc, Pd(dppf)Cl<sub>2</sub>, 1,4-dioxane, 90 °C, 10 h; (ii) NaOH, H<sub>2</sub>O<sub>2</sub>, THF, 30 min, 22 °C; (iii) 3-pyrrolidin-1-ylpropan-1-ol, PPh<sub>3</sub>, DEAD or DIAD, THF, 0 °C or rt, 5–10 h; (iv) Pd/C, H<sub>2</sub> (40 psi), MeOH, rt, 2–15 h; (v) Fe, NH<sub>4</sub>Cl, EtOH/H<sub>2</sub>O (4:1), rt, 3 h; (vi) POCl<sub>3</sub>, malonic acid, rt, 4 h, then 90 °C, overnight; (vii) DMAP, pyridine, CH<sub>2</sub>Cl<sub>2</sub>, −70 °C, 1 h, then ethyl 3-chloro-3-oxo-propanoate, 20 °C, 15 h; (viii) LiOH·H<sub>2</sub>O, THF/MeOH/H<sub>2</sub>O (4:2:3), 20 °C, 15 h; (ix) POCl<sub>3</sub>, 100 °C, 3 h; (x) 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane, Na<sub>2</sub>CO<sub>3</sub>, or K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane/H<sub>2</sub>O (5:1, 15:1 or 10:1), 80–110 °C, MW or conventional heating, 2–15 h; (xi) 1-methylpiperidin-4-amine, x-Phos, Pd<sub>2</sub>(dba)<sub>3</sub>, <i>t</i>-BuOK, 1,4-dioxane, 130 °C, MW, 4 h; (xii) 1-methylpiperidin-4-amine, BINAP, Pd<sub>2</sub>(dba)<sub>3</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 80–110 °C, 12–15 h.</p></p></figure><div class="NLM_p">Compounds with a hydroxy and a cyano substituent at position 6 of the quinoline (compounds <b>30</b> and <b>33</b>) were also prepared. As illustrated in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>, from previously described <b>5</b>, compound <b>30</b> was afforded using BBr<sub>3</sub> in CH<sub>2</sub>Cl<sub>2</sub>. Then, trifluoromethanesulfonate intermediate <b>65</b>, obtained by reaction with PhN(OTf)<sub>2</sub>, was converted into desired compound <b>33</b> using Zn(CN)<sub>2</sub> and Pd(PPh<sub>3</sub>)<sub>4</sub> in DMF.</div><figure id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0008.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>30</b> and <b>33</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) BBr<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C, 2 h; (ii) PhN(OTf)<sub>2</sub>, DIEA, DMF, 0 °C, 2 h, then 25 °C, 12 h; (iii) Zn(CN)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, DMF, 110 °C, 12 h.</p></p></figure><div class="NLM_p">To continue with exploration at position 7 of the quinoline, compounds <b>35</b>, <b>36</b>, and <b>39</b> were synthesized (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>). Starting from commercially available 5-amino-2-methoxy-phenol (<b>66</b>), intermediate <b>67</b> was prepared by reaction with POCl<sub>3</sub>. Substitution at position 7 was then installed using 1,3-dibromopropane and <i>N</i>-methylmethanamine or 5-azaspiro[2.4]heptane to obtain compounds <b>68a,b</b>. On the other hand, intermediate <b>68c</b> was achieved in only one step by Mitsunobu reaction with 3-(3-fluoropyrrolidin-1-yl)propan-1-ol. Then, dichloroquinolines <b>68a</b>–<b>c</b> were converted into desired compounds <b>35</b>, <b>36</b>, and <b>39</b> by the two step protocol previously described: Suzuki coupling and Buchwlad–Hartwig amination.</div><figure id="sch5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0009.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>35</b>, <b>36</b>, and <b>39</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) POCl<sub>3</sub>, malonic acid, 95 °C, 12 h; (ii) 1,3-dibromopropane, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 60 °C, 88 h; (iii) corresponding amine, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 60 °C, 16 h; (iv) 3-(3-fluoropyrrolidin-1-yl)propan-1-ol, PPh<sub>3</sub>, DEAD, THF, 20 °C 16 h; (v) 2-(5-ethyl-2-furyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane or 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane/H<sub>2</sub>O (1:1 or 10:1), 110 °C, 16 h; (vi) 1-methylpiperidin-4-amine, Cs<sub>2</sub>CO<sub>3</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, 1,4-dioxane, 120 °C, 12–16 h.</p></p></figure><div class="NLM_p">Compounds with a piperidine or morpholine at position 7 were also prepared as outlined in <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>. Conversion of commercially available 2-methoxy-5-nitro-phenol (<b>51</b>) to intermediate <b>69</b> was first achieved by reaction with 1,3-dibromopropane. Then, substitution by piperidine or morpholine and subsequent hydrogenation led us to compounds <b>70a</b>,<b>b</b> which were converted into corresponding 2,4-dichloroanilines <b>71a</b>,<b>b</b> after reaction with ethyl 3-chloro-3-oxo-propanoate, ester hydrolysis and reaction with POCl<sub>3</sub>. Finally, desired 4-aminoquinolines <b>37</b> and <b>38</b> were isolated through Suzuki coupling and amination under Buchwald-Hartwig conditions.</div><figure id="sch6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0010.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compounds <b>37</b> and <b>38</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) 1,3-dibromopropane, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 20 °C, 16 h; (ii) piperidine or morpholine, Cs<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 90 °C, 16 h; (iii) Pd/C, MeOH, H<sub>2</sub> (30 psi), 15 °C, 2 h; (iv) DMAP, pyridine, CH<sub>2</sub>Cl<sub>2</sub>, −78 °C, then ethyl 3-chloro-3-oxo-propanoate, 15 °C, 16 h; (v) LiOH·H<sub>2</sub>O, THF/MeOH/H<sub>2</sub>O (20:20:13 or 3:3:2), 15 °C, 16 h; (vi) POCl<sub>3</sub>, 100 °C, 2 h; (vii) 2-(5-ethyl-2-furyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane or 1,4-dioxane/H<sub>2</sub>O (1:1), 110 °C, 16 h; (viii) 1-methylpiperidin-4-amine, Cs<sub>2</sub>CO<sub>3</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, 1,4-dioxane, 120 °C, 16 h.</p></p></figure><div class="NLM_p">Preparation of compounds <b>34</b>, <b>40</b>–<b>45</b> is shown in <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>. In this case, the synthesis started with commercially available anilines <b>72a</b>,<b>b</b> and <b>66</b>, which were transformed into 2,4-dichloroanilines <b>73a</b>,<b>b</b> and <b>67</b> following methods described above. Then, 5-methyl-2-furyl at position 2 was installed through Suzuki coupling to afford intermediates <b>74a</b>–<b>c</b>. Compound <b>34</b> with a methoxy group at position 7 of the quinoline, was then prepared from intermediate <b>74a</b> by Buchwald–Hartwig amination using 1-methylpiperidin-4-amine as in other cases. 4-Piperidyl derivatives <b>40</b> and <b>41</b> were synthesized from intermediate <b>74b</b> through Mitsunobu reaction with <i>tert</i>-butyl 4-(hydroxymethyl) piperidine-1-carboxylate, coupling with 1-methylpiperidin-4-amine, removal of BOC protecting group under acidic conditions and reductive amination in the case of methylated derivative <b>41</b>. Compound <b>42</b> was prepared from intermediate <b>75</b> (obtained from <b>74c</b> by Suzuki coupling and hydrogenation), after reaction with <i>tert</i>-butyl 4-methylsulfonyloxypiperidine-1-carboxylate and acidic removal of BOC protecting group. Finally, 2-azaspiro[3.3]heptan-6-yl derivatives <b>43</b>–<b>45</b> were isolated after reaction of intermediate <b>75</b> with <i>tert</i>-butyl 6-methylsulfonyloxy-2-azaspiro[3.3]heptane-2-carboxylate and methylation using LiAlH<sub>4</sub> for compound <b>43</b> or deprotection with TFA and reductive amination or cyclopropanation for compounds <b>44</b> and <b>45</b>, respectively.</div><figure id="sch7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0011.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Compounds <b>34</b>, <b>40</b>–<b>45</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) POCl<sub>3</sub>, malonic acid, rt, 4 h, then 90 °C, overnight or 95 °C, 12 h; (ii) Et<sub>3</sub>N, ethyl 3-chloro-3-oxo-propanoate, CH<sub>2</sub>Cl<sub>2</sub>, 25 °C, 12 h; (iii) LiOH·H<sub>2</sub>O, THF/MeOH/H<sub>2</sub>O (3:3:2), 25 °C, 16 h; (iv) POCl<sub>3</sub>, 90 °C, 2 h; (v) 4,4,5,5-tetramethyl-2-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1,3,2-dioxaborolane, Na<sub>2</sub>CO<sub>3</sub>, or K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane, or 1,4-dioxane/H<sub>2</sub>O (15:1), 100–120 °C, MW or conventional heating, 2–16 h; (vi) 1-methylpiperidin-4-amine, Cs<sub>2</sub>CO<sub>3</sub>, BINAP, Pd<sub>2</sub>(dba)<sub>3</sub>, 1,4-dioxane, 110–120 °C, MW or conventional heating, 3–16h; (vii) Pd/C, H<sub>2</sub> (50 psi), MeOH, 25 °C, 16 h; (viii) <i>tert</i>-butyl 4-(hydroxymethyl) piperidine-1-carboxylate, PPh<sub>3</sub>, DIAD, THF, 0 °C, 8 h; (ix) HCl/EtOAc (1.0 or 2.0 M), 25 °C, 2–3 h; (x) (HCHO)<sub><i>n</i></sub>, NaBH(OAc)<sub>3</sub>, HCOOH, 1,4-dioxane, 100 °C, 2 h; (xi) <i>tert</i>-butyl 4-methylsulfonyloxypiperidine-1-carboxylate or <i>tert</i>-butyl 6-methylsulfonyloxy-2-azaspiro[3.3]heptane-2-carboxylate, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 100 °C, 16 h; (xii) LiAlH<sub>4</sub>, THF, 70 °C, 16 h; (xiii) TFA, CH<sub>2</sub>Cl<sub>2</sub>, 18 °C, 2 h; (xiv) acetone or (1-ethoxycyclopropoxy)-trimethyl-silane, NaBH<sub>3</sub>CN, AcOH, <i>i</i>-PrOH or <i>t</i>-BuOH, 60 °C, 16 h.</p></p></figure><div class="NLM_p">Synthesis of analogues <b>46</b>–<b>48</b> and <b>50</b> was performed as described in <a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>. From intermediate <b>74b</b>, compound <b>46</b> was prepared through Mitsunobu reaction, Pd mediated coupling with <i>tert</i>-butyl 2-amino-7-azaspiro[3.5]nonane-7-carboxylate, deprotection in acidic media, and reductive amination. On the other hand, compounds <b>47</b>, <b>48</b>, and <b>50</b> were prepared from <b>74c</b>. In this case, this intermediate was transformed into compounds <b>79a</b>–<b>c</b> after amination with different BOC-protected amines and hydrogenation. Then, substituent at position 7 was installed by reaction with <i>tert</i>-butyl 6-methylsulfonyloxy-2-azaspiro[3.3]heptane-2-carboxylate. Finally, BOC protecting groups were removed using TFA and free secondary amines were methylated through reductive amination.</div><figure id="sch8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0012.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of Compounds <b>46</b>–<b>48</b> and <b>50</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) <i>tert</i>-butyl 4-(hydroxymethyl) piperidine-1-carboxylate, PPh<sub>3</sub>, DIAD, THF, 0 °C, 8 h; (ii) corresponding amine, Cs<sub>2</sub>CO<sub>3</sub>, BINAP, Pd<sub>2</sub>(dba)<sub>3</sub>, or Pd(dba)<sub>2</sub>, 1,4-dioxane, 110–130 °C, 12–16 h; (iii) Pd/C, MeOH, H<sub>2</sub> (50 Psi), 25 °C, 16 h; (iv) <i>tert</i>-butyl 6-methylsulfonyloxy-2-azaspiro[3.3]heptane-2-carboxylate, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 100 °C, 16 h; (v) HCl/EtOAc (2.0 M), 25 °C, 2 h; vi) TFA, CH<sub>2</sub>Cl<sub>2</sub>, 18 °C, 2 h; vii) (HCHO)<sub>n</sub>, NaBH<sub>3</sub>CN or NaBH(OAc)<sub>3</sub>, HCOOH, MeOH, 60–70 °C, 12–16 h.</p></p></figure><div class="NLM_p">Finally, derivative <b>49</b> was synthesized as outlined in <a class="ref internalNav" href="#sch9" aria-label="Scheme 9">Scheme 9</a>. To prepare this compound, BOC protecting group of intermediate <b>78b</b> was removed and then an isopropyl group was installed by reductive amination. Compound <b>83</b> was then achieved after hydrogenation. Subsequent Mitsunobu reaction using <i>tert</i>-butyl 6-methylsulfonyloxy-2-azaspiro[3.3]heptane-2-carboxylate led us to intermediate <b>84</b>, which was finally converted into desired analogue <b>49</b> by acidic deprotection and reductive amination.</div><figure id="sch9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0013.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of Compound <b>49</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) HCl/EtOAc (2.0 M), 18 °C, 2 h; (ii) acetone, NaBH<sub>3</sub>CN, AcOH, <i>i</i>-PrOH, 60 °C, 16 h; (iii) Pd/C, MeOH, H<sub>2</sub> (50 psi), 20 °C, 8 h; (iv) <i>tert</i>-butyl 6-methylsulfonyloxy-2-azaspiro[3.3]heptane-2-carboxylate, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 100 °C, 16 h; (v) TFA, DCM, 18 °C, 2 h; (vi) (HCHO)<sub><i>n</i></sub>, NaBH(OAc)<sub>3</sub>, HCOOH, MeOH, 60 °C, 16 h.</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63773" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63773" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we present a detailed exploration around three main growing vectors born by the 2-(2-furyl)-4-aminoquinoline scaffold. Exploring this chemical space led to the identification of key chemical features (substructures) that guide our navigation to well-defined biological spaces covering the above-mentioned epigenetic targets; thus, we discovered selective G9a inhibitors (<b>5</b>, <b>6</b>, and <b>26</b>), first-in-class potent, and selective DNMT1 inhibitors (<b>43</b>) as well as equipotent dual inhibitors targetting G9a and DNMT1 (<b>12</b> and <b>13</b>). These molecules are valuable tools for comparative cellular studies and to investigate the mechanistic role and function of these targets in different diseases. In this regard, all the above-mentioned compounds showed remarkable antiproliferative effects against different hematological cancer cell lines and an adequate therapeutic window. Thus, these first-in-class substrate competitive and reversible inhibitors, covering different activity profiles, may serve as valuable starting points for drug discovery programs.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17658" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17658" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Chemistry: General Procedure</h3><div class="NLM_p">Unless otherwise noted, all reagents and solvents were of the highest commercial quality and used without further purification. All experiments dealing with moisture sensitive compounds were conducted under N<sub>2</sub>. Flash column chromatography was performed on silica gel, particle size 60 Å, mesh = 230–400 (Merck) under standard techniques. Automated flash column chromatography was performed using ready-to-connect cartridges from Varian, on irregular silica gel, particle size 15–40 μm (normal phase disposable flash columns) on a Biotage SPX flash purification system. Microwave-assisted reactions were performed in a Biotage Smith Synthesis microwave reactor. The NMR spectroscopic data were recorded on a Bruker AV400 or VARIAN 400MR spectrometer with standard pulse sequences, operating at 400 MHz. Chemical shifts (δ) are reported in parts per million (ppm) downfield from tetramethylsilane (TMS), which was used as internal standard. The abbreviations used to explain multiplicities are s = singlet, d = doublet, t = triplet, m = multiplet. Coupling constants (<i>J</i>) are in hertz. HPLC-analysis was performed using a Shimadzu LC-20AB or LC-20AD with a Luna-C18(2), 5 μm, 2.0 mm × 50 mm column at 40 °C and UV detection at 215, 220, and 254 nm. Flow from the column was split to a MS spectrometer. The MS detector (Agilent 1200, 6110MS or Agilent 1200, 6120MS quadropole) was configured with an electrospray source or API/APCI. N<sub>2</sub> was used as the nebulizer gas. The source temperature was maintained at 50 °C. Data acquisition was accomplished with ChemStation LC/MSD quad software. All tested compounds possessed a purity of at least 95% established by HPLC or LCMS unless otherwise noted. Reported yields were not optimized, the emphasis being on purity of product rather than quantity.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>N</i>-Cyclopropyl-6-methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>7</b>)</h4><div class="NLM_p last">To a solution of compound <b>55a</b> (200 mg, 0.5 mmol) in 1,4-dioxane (10 mL) were added Cs<sub>2</sub>CO<sub>3</sub> (325 mg, 1 mmol), BINAP (67 mg, 0.1 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (30 mg, 0.03 mmol), and cyclopropanamine (60 mg, 1 mmol) and the reaction mixture was stirred at 120 °C for 5 h under microwave. Then, the solution was concentrated and extracted with EtOAc. The combined organic phase was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 1 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to obtain pure compound <b>7</b> (35 mg, 17%) as yellow solid; mp 103–104 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.69 (s, 1H), 7.49 (d, <i>J</i> = 2.8 Hz, 1H), 7.47 (s, 1H), 7.31 (s, 1H), 6.43 (d, <i>J</i> = 2.8 Hz, 1H), 4.36–4.33 (m, 2H), 4.01 (s, 3H), 3.85–3.78 (m, 2H), 3.51–3.47 (m, 2H), 3.20–3.10 (m, 2H), 2.91–2.89 (m, 1H), 2.50 (s, 3H), 2.39–2.34 (m, 2H), 2.24–2.17 (m, 2H), 2.09–2.05 (m, 2H), 1.11–1.09 (m, 2H), 0.85–0.82 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 422.3 [M + H]<sup>+</sup> calcd for C<sub>25</sub>H<sub>31</sub>N<sub>3</sub>O<sub>3</sub>.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>N</i>-Cyclopentyl-6-methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>8</b>)</h4><div class="NLM_p last">To a solution of compound <b>55a</b> (90 mg, 0.225 mmol) in toluene (5 mL) was added <i>t</i>-BuONa (43 mg, 0.45 mmol), xantphos (45 mg, 0.07 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (64 mg, 0.07 mmol), and cyclopentanamine (40 mg, 0.45 mmol) and the solution was heated to 100 °C for 2 h under microwave. Then, the solution was concentrated and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 2 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to obtain pure compound <b>8</b> (7.2 mg, 7%) as yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.83 (s, 1H), 7.52–7.51 (m, 1H), 7.46 (s, 1H), 6.99 (s, 1H), 6.43–6.42 (m, 1H), 4.37–4.33 (m, 3H), 4.04 (s, 3H), 3.90–3.75 (m, 2H), 3.55–3.45 (m, 2H), 3.25–3.10 (m, 2H), 2.51 (s, 3H), 2.45–2.35 (m, 2H), 2.35–2.15 (m, 4H), 2.15–2.00 (m, 2H), 2.00–1.70 (m, 6H). ESI-MS <i>m</i>/<i>z</i> 450.3 [M + H]<sup>+</sup> calcd for C<sub>27</sub>H<sub>35</sub>N<sub>3</sub>O<sub>3</sub>.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 2-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]ethanol (<b>9</b>)</h4><div class="NLM_p last">To a solution of compound <b>55a</b> (200 mg, 0.5 mmol) in 1,4-dioxane (10 mL) were added Cs<sub>2</sub>CO<sub>3</sub> (325 mg, 1 mmol), BINAP (67 mg, 0.1 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (30 mg, 0.03 mmol), and 2-aminoethanol (61 mg, 1 mmol) and the reaction mixture was stirred at 120 °C for 3 h under microwave. Then, the solution was concentrated and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 1 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to obtain compound <b>9</b> (60 mg, 28%) as a yellow solid; mp 112–113 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.73 (s, 1H), 7.50–7.49 (m, 2H), 7.06 (s, 1H), 6.42 (d, <i>J</i> = 2.8 Hz, 1H), 4.36–4.33 (m, 2H), 4.06 (s, 3H), 3.93–3.89 (m, 2H), 3.86–3.78 (m, 2H), 3.77–3.74 (m, 2H), 3.52–3.47 (m, 2H), 3.21–3.13 (m, 2H), 2.49 (s, 3H), 2.40–2.36 (m, 2H), 2.25–2.16 (m, 2H), 2.11–2.05 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 426.3 [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>31</sub>N<sub>3</sub>O<sub>4</sub>.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 3-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-<i>N</i>-methyl-propanamide (<b>10</b>)</h4><div class="NLM_p last">To a solution of compound <b>55a</b> (200 mg, 0.5 mmol) in 1,4-dioxane (10 mL) was added Cs<sub>2</sub>CO<sub>3</sub> (325 mg, 1 mmol), BINAP (67 mg, 0.1 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (30 mg, 0.03 mmol), and 3-amino-<i>N</i>-methylpropanamide (204 mg, 1 mmol), and the solution was heated to 130 °C for 5 h under microwave. Then, the solution was concentrated and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 1 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to obtain pure compound <b>10</b> (40 mg, 17%) as yellow solid; mp 105–106 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.68 (s, 1H), 7.55 (d, <i>J</i> = 3.2 Hz, 1H), 7.49 (s, 1H), 7.01 (s, 1H), 6.46 (d, <i>J</i> = 3.2 Hz, 1H), 4.38–4.34 (m, 2H), 4.05 (s, 3H), 3.94–3.80 (m, 4H), 3.54–3.50 (m, 2H), 3.23–3.12 (m, 2H), 2.75–2.65 (m, 5H), 2.53 (s, 3H), 2.43–2.38 (m, 2H), 2.27–2.17 (m, 2H), 2.12–2.05 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 467.3 [M + H]<sup>+</sup> calcd for C<sub>26</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 4-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-1-methyl-piperidin-2-one (<b>11</b>)</h4><div class="NLM_p last">To a solution of compound <b>55a</b> (200 mg, 0.5 mmol) in 1,4-dioxane (10 mL) were added Cs<sub>2</sub>CO<sub>3</sub> (325 mg, 1 mmol), BINAP (67 mg, 0.1 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (30 mg, 0.03 mmol), and 4-amino-1-methylpiperidin-2-one (128 mg, 1 mmol) and the reaction mixture was stirred at 120 °C for 3 h under microwave. Then, the solution was concentrated and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 1 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to obtain pure compound <b>11</b> (20 mg, 8%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.83 (s, 1H), 7.58 (d, <i>J</i> = 3.2 Hz, 1H), 7.51 (s, 1H), 7.11 (s, 1H), 6.44 (d, <i>J</i> = 3.2 Hz, 1H), 4.57–4.51 (m, 1H), 4.40–4.34 (m, 2H), 4.07 (s, 3H), 3.89–3.80 (m, 2H), 3.62–3.48 (m, 4H), 3.21–3.12 (m, 2H), 3.04 (s, 3H), 2.99–2.93 (m, 1H), 2.64–2.53 (m, 4H), 2.44–2.27 (m, 3H), 2.24–2.07 (m, 5H). ESI-MS <i>m</i>/<i>z</i> 493.3 [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>36</sub>N<sub>4</sub>O<sub>4</sub>.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 2-(5-Ethyl-2-furyl)-6-methoxy-<i>N</i>-[(1-methyl-4-piperidyl)methyl]-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>13</b>)</h4><div class="NLM_p last">To a solution of compound <b>55b</b> (207 mg, 0.5 mmol) in 1,4-dioxane (15 mL) was added Cs<sub>2</sub>CO<sub>3</sub> (325 mg, 1 mmol), BINAP (33.75 mg, 0.05 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (30 mg, 0.03 mmol), and (1-methylpiperidin-4-yl)methanamine (128 mg, 1.0 mmol) and the solution was heated to 120 °C for 12 h. Then, the solution was concentrated and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 3 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to obtain pure compound <b>13</b> (110 mg, 43%) as yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.73 (s, 1H), 7.51 (d, <i>J</i> = 3.2 Hz, 1H), 7.48 (s, 1H), 6.99 (s, 1H), 6.46 (d, <i>J</i> = 3.2 Hz, 1H), 4.37–4.31 (m, 3H), 4.04 (s, 3H), 3.86–3.74 (m, 2H), 3.61–3.52 (m, 4H), 3.50–3.42 (m, 2H), 3.23–3.12 (m, 2H), 3.11–2.98 (m, 2H), 2.91–2.83 (m, 5H), 2.42–2.32 (m, 2H), 2.26–2.11 (m, 5H), 2.11–2.02 (m, 2H), 1.65–1.55 (m, 2H), 1.38–1.34 (m, 3H). ESI-MS <i>m</i>/<i>z</i> 507.3 [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>42</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>N</i>-[(1-Isopropyl-4-piperidyl)methyl]-6-methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>14</b>)</h4><div class="NLM_p last">To a mixture of compound <b>57a</b> (125 mg, 0.261 mmol) and acetone (85 mg, 1.46 mmol) in <i>i</i>-PrOH (20 mL) were added NaBH<sub>3</sub>CN (92 mg, 1.46 mmol) and AcOH (88 mg, 1.46 mmol), and the mixture was stirred at 50 °C for 15 h under N<sub>2</sub>. Then, the solution was cooled to 16 °C, filtered, and concentrated in vacuum. The residue was purified by prep-HPLC (method 4 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford the desired compound <b>14</b> (42.0 mg, 31%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.73 (s, 1H), 7.56–7.48 (m, 2H), 6.96 (s, 1H), 6.42 (d, <i>J</i> = 2.4 Hz, 1H), 4.34–4.29 (m, 2H), 4.03 (s, 3H), 3.90–3.78 (m, 2H), 3.62–3.55 (m, 2H), 3.53–3.49 (m, 5H), 3.16 (m, 2H), 3.09–3.03 (m, 2H), 2.50 (s, 3H), 2.42–2.33 (m, 2H), 2.20–2.17 (m, 4H), 2.10–2.01 (m, 2H), 1.72–1.64 (m, 2H), 1.42–1.32 (m, 7H). ESI-MS <i>m</i>/<i>z</i> 521.4 [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>44</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 6-Methoxy-2-(5-methyl-2-furyl)-<i>N</i>-[1-methyl-1-(1-methyl-4-piperidyl)ethyl]-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>15</b>)</h4><div class="NLM_p last">A mixture of <b>57b</b> (15.0 mg, 0.029 mmol), (HCHO)<i><sub>n</sub></i> (5.3 mg, 0.059 mmol), HCOOH (0.28 mg, 5.92 μmol), and NaBH(OAc)<sub>3</sub> (12.55 mg, 0.059 mmol) in MeOH (5.00 mL) was degassed and purged with N<sub>2</sub> for 3 times and then the mixture was stirred at 60 °C for 12 h. Then, the reaction mixture was filtrated, and the filtrate was concentrated under vacuum to give a residue. The residue was purified by prep-HPLC (method 5 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford compound <b>15</b> (4.5 mg, 30%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.74 (s, 1H), 7.56 (d, <i>J</i> = 3.54 Hz, 1H), 7.47 (s, 1H), 7.09 (s, 1H), 6.44–6.42 (m, 1H), 4.35 (t, <i>J</i> = 5.2 Hz, 2H), 4.03 (s, 3H), 3.85–3.76 (m, 2H), 3.58–3.44 (m, 4H), 3.18–3.10 (m, 2H), 3.06–2.96 (m, 2H), 2.83 (s, 3H), 2.62–2.54 (m, 1H), 2.50 (s, 3H), 2.40–2.33 (m, 2H), 2.23–2.16 (m, 2H), 2.10–1.95 (m, 4H), 1.84–1.71 (m, 2H), 1.64 (s, 6H). ESI-MS <i>m</i>/<i>z</i> 521.4 [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>44</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>N</i>-Benzyl-6-methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>16</b>)</h4><div class="NLM_p last">To a solution of compound <b>55a</b> (200 mg, 0.5 mmol) in 1,4-dioxane (10 mL) were added Cs<sub>2</sub>CO<sub>3</sub> (325 mg, 1 mmol), BINAP (67 mg, 0.1 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (30 mg, 0.03 mmol), and phenylmethanamine (107 mg, 1 mmol) and the reaction mixture was stirred at 120 °C for 3 h under microwave. Then, the solution was concentrated and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 1 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to obtain pure compound <b>16</b> (60 mg, 26%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.78 (s, 1H), 7.32 (s, 7H), 6.92 (s, 1H), 6.38 (d, <i>J</i> = 2.8 Hz, 1H), 4.79 (s, 2H), 4.36–4.33 (m, 2H), 4.04 (s, 3H), 3.87–3.78 (m, 2H), 3.52–3.47 (m, 2H), 3.19–3.13 (m, 2H), 2.46 (s, 3H), 2.42–2.35 (m, 2H), 2.26–2.19 (m, 2H), 2.11–2.04 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 472.3 [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>33</sub>N<sub>3</sub>O<sub>3</sub>.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 6-Methoxy-<i>N</i>-(7-methyl-7-azaspiro[3.5]nonan-2-yl)-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>17</b>)</h4><div class="NLM_p last">To a solution of compound <b>57c</b> (50.4 mg, 0.1 mmol) in MeOH (10 mL) was added (HCHO)<i><sub>n</sub></i> (9 mg, 0.3 mmol) and the solution was stirred at room temperature for 1 h. Then, NaBH(OAc)<sub>3</sub> (63 mg, 0.3 mmol) was added, and the mixture was stirred at room temperature overnight. The solution was extracted with EtOAc and the combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 1 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>17</b> (25 mg, 48%) as yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.85 (s, 1H), 7.55 (d, <i>J</i> = 2.8 Hz, 1H), 7.51 (s, 1H), 6.85 (s, 1H), 6.45 (d, <i>J</i> = 2.4 Hz, 1H), 4.55–4.53 (m, 1H), 4.40–4.33 (m, 2H), 4.06 (s, 3H), 3.89–3.82 (m, 2H), 3.53–3.50 (m, 3H), 3.46–3.43 (m, 1H), 3.18–3.15 (m, 3H), 3.05–2.98 (m, 1H), 2.90 (s, 3H), 2.87–2.84 (m, 1H), 2.58–2.54 (m, 1H), 2.53 (s, 3H), 2.40 (s, 2H), 2.28–2.25 (m, 4H), 2.18–2.09 (m, 3H), 1.98–1.95 (m, 3H). ESI-MS <i>m</i>/<i>z</i> 519.3 [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>42</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> <i>N</i>-(7-Isopropyl-7-azaspiro[3.5]nonan-2-yl)-6-methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>18</b>)</h4><div class="NLM_p last">A mixture of <b>57c</b> (200 mg, 0.396 mmol), acetone (128 mg, 2.22 mmol), NaBH<sub>3</sub>CN (139 mg, 2.22 mmol), and AcOH (133 mg, 2.22 mmol) in <i>i</i>-PrOH (5 mL) was degassed and purged with N<sub>2</sub> for 3 times and then the mixture was stirred at 60 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was purified by prep-HPLC (method 6 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>18</b> (32 mg, 15%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.82 (s, 1H), 7.54 (d, <i>J</i> = 3.6 Hz, 1H), 7.48 (s, 1H), 6.81 (s, 1H), 6.43 (d, <i>J</i> =3.2 Hz, 1H), 4.52–4.50 (m, 1H), 4.35–4.34 (m, 2H), 4.04 (s, 3H), 3.83 (br s, 2H), 3.52–3.47 (m, 6H), 3.17–3.14 (m, 4H), 2.98–2.93 (m, 1H), 2.82–2.78 (m, 1H), 2.55–2.50 (m, 4H), 2.38–2.36 (m, 2H), 2.29–2.26 (m, 2H), 2.23–2.20 (m, 2H), 2.10–2.08 (m, 2H), 2.00–1.98 (m, 2H), 1.39–1.37 (m, 6H). ESI-MS <i>m</i>/<i>z</i> 547.5 [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>46</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 6-Methoxy-<i>N</i>-(6-methyl-6-azaspiro[3.4]octan-2-yl)-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>19</b>)</h4><div class="NLM_p last">To a solution of compound <b>57d</b> (80 mg, 0.16 mmol) in MeOH (10 mL) was added (HCHO)<i><sub>n</sub></i> (15 mg, 0.48 mmol) and the solution was stirred at room temperature for 1 h. Then, NaBH(OAc)<sub>3</sub> (170 mg, 0.8 mmol) was added and the reaction mixture was stirred at room temperature overnight. Then, the solution was extracted with EtOAc and the combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 1 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford compound <b>19</b> (15 mg, 19%) as yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.80 (s, 1H), 7.55–7.54 (m, 1H), 7.47 (s, 1H), 6.85–6.83 (m, 1H), 6.44–6.43 (m, 1H), 4.54–4.46 (m, 1H), 4.36–4.33 (m, 2H), 4.04 (s, 3H), 3.83–3.70 (m, 4H), 3.52–3.47 (m, 2H), 3.19–3.16 (m, 4H), 3.00–2.94 (m, 3H), 2.80–2.65 (m, 3H), 2.60–2.50 (m, 4H), 2.39–2.38 (m, 3H), 2.25–2.21 (m, 3H), 2.12–2.08 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 505.3 [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>40</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 6-Methoxy-<i>N</i>-(7-methyl-7-azaspiro[3.5]nonan-3-yl)-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>20</b>)</h4><div class="NLM_p last">To a solution of compound <b>57e</b> (50.5 mg, 0.1 mmol) in MeOH (10 mL) was added (HCHO)<i><sub>n</sub></i> (9 mg, 0.3 mmol) and the solution was stirred at room temperature for 1 h. Then, NaBH(OAc)<sub>3</sub> (63 mg, 0.3 mmol) was added, and the reaction mixture was stirred at room temperature overnight. The solution was extracted with EtOAc and the combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 1 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>20</b> (25 mg, 48%) as yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.90 (s, 1H), 7.51 (s, 2H), 7.01 (s, 1H), 6.47 (s, 1H), 4.63–4.56 (m, 1H), 4.40–4.32 (m, 2H), 4.08 (s, 3H), 3.89–3.81 (m, 2H), 3.55–3.40 (m, 4H), 3.26–3.12 (m, 3H), 3.08–2.96 (m, 1H), 2.83 (s, 3H), 2.61–2.48 (m, 5H), 2.46–2.28 (m, 3H), 2.26–2.03 (m, 7H), 1.94–1.82 (m, 1H), 1.70–1.61 (m, 1H). ESI-MS <i>m</i>/<i>z</i> 519.3 [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>42</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 2-(5-Ethyl-2-furyl)-6-methoxy-<i>N</i>-(7-methyl-7-azaspiro[3.5]nonan-3-yl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>21</b>)</h4><div class="NLM_p last">A mixture of <b>57f</b> (50 mg, 0.096 mmol), (HCHO)<i><sub>n</sub></i> (26.1 mg, 0.289 mmol), NaBH(OAc)<sub>3</sub> (61 mg, 0.289 mmol), and HCOOH (4.6 mg, 0.96 mmol) in MeOH (5 mL) was degassed and purged with N<sub>2</sub> for 3 times and then the mixture was stirred at 60 °C for 16 h. Then, the solution was concentrated under vacuum to give a residue. The residue was purified by prep-HPLC (method 3 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>21</b> (12.5 mg, 24%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.88 (s, 1H), 7.51–7.50 (m, 2H), 6.98 (s, 1H), 6.44 (d, <i>J</i> = 3.2 Hz, 1H), 4.58–4.55 (m, 1H), 4.37–4.34 (m, 2H), 4.05 (s, 3H), 3.85–3.80 (m, 2H), 3.49–3.42 (m, 4H), 3.17–3.14 (m, 4H), 3.03–2.97 (m, 1H), 2.86–2.81 (m, 5H), 2.58–2.52 (m, 2H), 2.40–2.37 (m, 2H), 2.22–2.06 (m, 7H), 1.89–1.86 (m, 1H), 1.68–1.64 (m, 1H), 1.36 (t, <i>J</i> = 14.8 Hz, 3H). ESI-MS <i>m</i>/<i>z</i> 533.5 [M + H]<sup>+</sup> calcd for C<sub>32</sub>H<sub>44</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 6-Methoxy-<i>N</i>-(6-methyl-6-azaspiro[3.4]octan-3-yl)-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>22</b>)</h4><div class="NLM_p last">To a solution of compound <b>57g</b> (49.1 mg, 0.1 mmol) in MeOH (10 mL) was added (HCHO)<i><sub>n</sub></i> (9 mg, 0.3 mmol) and the solution was stirred at room temperature for 1 h. Then NaBH(OAc)<sub>3</sub> (63 mg, 0.3 mmol) was added, and the reaction mixture was stirred at room temperature overnight. The solution was extracted with EtOAc and the combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 1 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>22</b> (15 mg, 30%) as yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.98 (s, 1H), 7.60–7.52 (m, 2H), 7.11–7.08 (m, 1H), 6.48 (s, 1H), 4.41–4.32 (m, 2H), 4.09 (s, 3H), 3.89–3.79 (m, 4H), 3.55–3.46 (m, 3H), 3.24–3.12 (m, 4H), 2.93–2.82 (m, 3H), 2.54–2.35 (m, 7H), 2.26–2.03 (m, 8H). ESI-MS <i>m</i>/<i>z</i> 505.3 [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>40</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 6-Methoxy-<i>N</i>-(3-methyl-3-azabicyclo[3.2.1]octan-8-yl)-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>23</b>)</h4><div class="NLM_p last">A mixture of <b>57h</b> (150 mg, 0.306 mmol), (HCHO)<sub><i>n</i></sub> (82 mg, 0.917 mmol), NaBH(OAc)<sub>3</sub> (194 mg, 0.917 mmol), and HCOOH (14 mg, 0.305 mmol) in MeOH (10 mL) was degassed and purged with N<sub>2</sub> for 3 times and then the mixture was stirred at 60 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was purified by prep-HPLC (method 7 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>23</b> (22.5 mg, 14%) as a white solid; mp 120–121 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.94 (s, 1H), 7.61 (d, <i>J</i> = 3.6 Hz, 1H), 7.54 (s, 1H), 6.99 (s, 1H), 6.44 (d, <i>J</i> = 2.8 Hz, 1H), 4.37–4.35 (m, 2H), 4.10 (s, 3H), 4.04–4.00 (m, 1H), 3.86–3.77 (m, 2H), 3.54–3.49 (m, 4H), 3.40–3.37 (m, 2H), 3.18–3.14 (m, 2H), 2.92–2.89 (m, 5H), 2.51 (s, 3H), 2.41–2.38 (m, 2H), 2.27–2.22 (m, 4H), 2.06–1.99 (m, 4H). ESI-MS <i>m</i>/<i>z</i> 505.4 [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>40</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 6-Methoxy-<i>N</i>-[(3-methyl-3-azabicyclo[3.2.1]octan-8-yl)methyl]-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>24</b>)</h4><div class="NLM_p last">A mixture of <b>57i</b> (300 mg, 0.55 mmol), (HCHO)<i><sub>n</sub></i> (149 mg, 1.66 mmol), NaBH(OAc)<sub>3</sub> (352 mg, 1.66 mmol), and HCOOH (26 mg, 0.554 mmol) in MeOH (10 mL) was degassed and purged with N<sub>2</sub> for 3 times, and then the mixture was stirred at 60 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was purified by prep-HPLC (method 8 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford compound <b>24</b> (47 mg, 16%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.79–7.76 (m, 1H), 7.57–7.56 (m, 1H), 7.50 (s, 1H), 7.08–7.02 (m, 1H), 6.44–6.43 (m, 1H), 4.36–4.34 (m, 2H), 4.04–4.01 (m, 4H), 3.86–3.78 (m, 2H), 3.53–3.41 (m, 6H), 3.16–3.13 (m, 3H), 2.93–2.83 (m, 3H), 2.55–2.37 (m, 8H), 2.22–2.20 (m, 3H), 2.10–2.08 (m, 3H), 1.90–1.89 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 519.4 [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>42</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 2-(5-Ethyl-2-furyl)-6-methoxy-<i>N</i>-[(3-methyl-3-azabicyclo[3.2.1]octan-8-yl)methyl]-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>25</b>)</h4><div class="NLM_p last">A mixture of <b>57j</b> (360 mg, 0.649 mmol), (HCHO)<sub><i>n</i></sub> (175 mg, 1.95 mmol), NaBH(OAc)<sub>3</sub> (412 mg, 1.95 mmol), and HCOOH (31 mg, 0.648 mmol) in MeOH (10 mL) was degassed and purged with N<sub>2</sub> for 3 times and then the mixture was stirred at 60 °C overnight. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was purified by prep-HPLC (method 8 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford compound <b>25</b> (48 mg, 14%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.79–7.75 (m, 1H), 7.59–7.57 (m, 1H), 7.50 (s, 1H), 7.07–7.02 (m, 1H), 6.46–6.44 (m, 1H), 4.36–4.34 (m, 2H), 4.04–4.01 (m, 4H), 3.86–3.77 (m, 2H), 3.53–3.41 (m, 6H), 3.16–3.13 (m, 3H), 2.93–2.83 (m, 5H), 2.55–2.37 (m, 5H), 2.21–2.20 (m, 3H), 2.10–2.07 (m, 3H), 1.90–1.89 (m, 2H), 1.36 (t, <i>J</i> = 14.8 Hz, 3H). ESI-MS <i>m</i>/<i>z</i> 533.4 [M + H]<sup>+</sup> calcd for C<sub>32</sub>H<sub>44</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 6-Methoxy-<i>N</i>-(8-methyl-8-azabicyclo[3.2.1]octan-3-yl)-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>26</b>)</h4><div class="NLM_p last">To a solution of compound <b>57k</b> (200 mg, 0.408 mmol) in MeOH (10 mL) were added HCOOH (18 mg, 0.379 mmol), NaBH(OAc)<sub>3</sub> (241 mg, 1.14 mmol), and (HCHO)<i><sub>n</sub></i> (103 mg, 1.14 mmol) at 20 °C under N<sub>2</sub>, and the mixture was stirred at 60 °C for 16 h. Then, the mixture was concentrated under reduced pressure and the residue was purified by prep-HPLC (method 9 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>26</b> (61.6 mg, 30%) as a yellow solid; mp 120–121 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.80–7.78 (m, 1H), 7.60 (d, <i>J</i> = 3.2 Hz, 1H), 7.50 (s, 1H), 7.08 (s, 1H), 6.42 (d, <i>J</i> = 3.2 Hz, 1H), 4.60–4.55 (m, 1H), 4.33 (t, <i>J</i> = 5.2 Hz, 2H), 4.08–4.04 (m, 5H), 3.87–3.77 (m, 2H), 3.49 (t, <i>J</i> = 7.2 Hz, 2H), 3.17–3.13 (m, 2H), 2.87 (s, 3H), 2.51 (s, 3H), 2.42–2.34 (m, 8H), 2.25–2.15 (m, 3H), 2.15–1.99 (m, 3H). ESI-MS <i>m</i>/<i>z</i> 505.3 [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>40</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 6-Methoxy-<i>N</i>-methyl-2-(5-methyl-2-furyl)-<i>N</i>-(1-methyl-4-piperidyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>27</b>)</h4><div class="NLM_p last">To a solution of compound <b>57l</b> (40 mg, 0.083 mmol) in MeOH (5 mL) were added HCOOH (4 mg, 0.083 mmol), NaBH(OAc)<sub>3</sub> (53 mg, 0.25 mmol), and (HCHO)<sub><i>n</i></sub> (23 mg, 0.25 mmol), and the mixture was stirred at 60 °C for 16 h under N<sub>2</sub>. Then, mixture was concentrated under reduced pressure and the resulting residue was purified by prep-HPLC (method 10 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>27</b> (8.2 mg, 20%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.60 (d, <i>J</i> = 14.8 Hz, 2H), 7.31 (d, <i>J</i> = 9.2 Hz, 2H), 6.46 (s, 1H), 4.46–4.32 (m, 3H), 4.04 (s, 3H), 3.86–3.77 (m, 2H), 3.67 (d, <i>J</i> = 13.6 Hz, 2H), 3.54–3.47 (m, 2H), 3.29–3.10 (m, 7H), 2.91 (s, 3H), 2.54–2.49 (m, 3H), 2.46–2.35 (m, 4H), 2.28–2.13 (m, 4H), 2.12–2.03 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 493.3 [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>40</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 2-(5-Methyl-2-furyl)-<i>N</i>-(1-methyl-4-piperidyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>29</b>)</h4><div class="NLM_p last">To a solution of compound <b>64a</b> (140 mg, 0.38 mmol) in 1,4-dioxane (10 mL) was added <i>t</i>-BuOK (1.0 M, 1.14 mL, 1.14 mmol), x-Phos (54 mg, 0.114 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (69 mg, 0.076 mmol), and 1-methylpiperidin-4-amine (252 mg, 2.2 mmol) and the solution was heated to 130 °C for 4 h under microwave. Then, the mixture was quenched with water and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 2 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>29</b> (18.7 mg, 11%) as a yellow solid; mp 98–99 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.34 (d, <i>J</i> = 9.2 Hz, 1H), 7.64 (s, 1H), 7.46 (s, 1H), 7.30–7.25 (m, 1H), 7.07 (s, 1H), 6.45 (d, <i>J</i> = 2.8 Hz, 1H), 4.32–4.29 (m, 3H), 3.73–3.66 (m, 4H), 3.48–3.44 (m, 2H), 3.26–3.25 (m, 1H), 3.15–3.10 (m, 2H), 2.93 (s, 3H), 2.51 (s, 3H), 2.34–2.31 (m, 4H), 2.18–2.17 (m, 4H), 2.16–2.14 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 449.3 [M + H]<sup>+</sup> calcd for C<sub>27</sub>H<sub>36</sub>N<sub>4</sub>O<sub>2</sub>.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 2-(5-Methyl-2-furyl)-4-[(1-methyl-4-piperidyl)amino]-7-(3-pyrrolidin-1-ylpropoxy)quinolin-6-ol (<b>30</b>)</h4><div class="NLM_p last">To a solution of compound <b>5</b> (50 mg, 0.101 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) was added BBr<sub>3</sub> (254 mg, 1.01 mmol) slowly at 0 °C, and the solution was stirred for 2 h at 0 °C under N<sub>2.</sub> Then, the reaction was quenched with water and concentrated to give the crude product, which was purified by prep-HPLC (method 2 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>30</b> (11 mg, 23%) as yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.77 (s, 1H), 7.58 (d, <i>J</i> = 3.2 Hz, 1H), 7.52 (s, 1H), 7.07 (s, 1H), 6.46 (d, <i>J</i> = 3.2 Hz, 1H), 4.41–4.32 (m, 2H), 4.34–4.24 (m, 2H), 3.87–3.71 (m, 4H), 3.58–3.51 (m, 2H), 3.32–3.25 (m, 2H), 3.20–3.09 (m, 2H), 2.97 (s, 3H), 2.50 (s, 3H), 2.44–2.33 (m, 4H), 2.24–2.04 (m, 6H). ESI-MS <i>m</i>/<i>z</i> 465.3 [M + H]<sup>+</sup> calcd for C<sub>27</sub>H<sub>36</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 6-Chloro-2-(5-methyl-2-furyl)-<i>N</i>-(1-methyl-4-piperidyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>31</b>)</h4><div class="NLM_p last">To a solution of compound <b>64b</b> (100 mg, 0.25 mmol) in 1,4-dioxane (15 mL) was added Cs<sub>2</sub>CO<sub>3</sub> (161 mg, 0.49 mmol), BINAP (31 mg, 0.049 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (45 mg, 0.05 mmol), and 1-methylpiperidin-4-amine (128 mg, 1.12 mmol), and the mixture was stirred at 110 °C for 12 h under N<sub>2</sub>. Then, the mixture was quenched with water and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 2 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>31</b> (10 mg, 8%) as a yellow solid; mp 109–110 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.54–8.47 (m, 1H), 7.54–7.44 (m, 2H), 7.10–7.02 (m, 1H), 6.42–6.35 (m, 1H), 4.43–4.35 (m, 2H), 4.26–4.17 (m, 1H), 3.93–3.82 (m, 2H), 3.60–3.42 (m, 6H), 3.25–3.13 (m, 4H), 2.89 (s, 3H), 2.51 (s, 3H), 2.45–2.30 (m, 3H), 2.20–2.03 (m, 5H). ESI-MS <i>m</i>/<i>z</i> 483.3 [M + H]<sup>+</sup> calcd for C<sub>27</sub>H<sub>35</sub>ClN<sub>4</sub>O<sub>2</sub>.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 2-(5-Methyl-2-furyl)-<i>N</i>-(1-methyl-4-piperidyl)-7-(3-pyrrolidin-1-ylpropoxy)-6-(trifluoromethoxy)quinolin-4-amine (<b>32</b>)</h4><div class="NLM_p last">To a solution of compound <b>64c</b> (195 mg, 0.43 mmol) in 1,4-dioxane (15.00 mL) were successively added 1-methylpiperidin-4-amine (108 mg, 1.03 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (39 mg, 0.04 mmol), BINAP (53 mg, 0.086 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (350 mg, 1.08 mmol) and the resulting mixture was stirred at 80 °C for 15 h under N<sub>2</sub>. Then, the reaction mixture was diluted with water and extracted with EtOAc. The combined organic layer was washed with brine dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to obtain the crude product which was purified by prep-HPLC (method 2 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>32</b> (64.4 mg, 28%) as a yellow solid; mp 195–196 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.52 (s, 1H), 7.75–7.70 (m, 2H), 7.14 (s, 1H), 6.48 (d, <i>J</i> = 2.4 Hz, 1H), 4.42–4.39 (m, 2H), 4.36–4.28 (m, 1H), 3.76–3.68 (m, 4H), 3.47–3.43 (m, 2H), 3.31–3.26 (m, 2H), 3.19–3.10 (m, 2H), 2.95 (s, 3H), 2.53 (s, 3H), 2.42–2.36 (m, 4H), 2.19–2.16 (m, 4H), 2.08–2.07 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 533.2 [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>35</sub>F<sub>3</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 2-(5-Methyl-2-furyl)-4-[(1-methyl-4-piperidyl)amino]-7-(3-pyrrolidin-1-ylpropoxy)quinoline-6-carbonitrile (<b>33</b>)</h4><div class="NLM_p last">To a solution of compound <b>65</b> (100 mg, 0.168 mmol) in DMF (5 mL) was added Zn(CN)<sub>2</sub> (39 mg, 0.336 mmol) and Pd(PPh<sub>3</sub>)<sub>4</sub> (20 mg, 0.017 mmol) and the solution was heated at 110 °C for 12 h. Then, the solution was concentrated and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 3 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>33</b> (58 mg, 73%) as yellow solid; mp 123–124 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.88 (s, 1H), 7.75 (d, <i>J</i> = 3.6 Hz, 1H), 7.62 (s, 1H), 7.15 (s, 1H), 6.51 (d, <i>J</i> = 3.6 Hz, 1H), 4.46–4.42 (m, 2H), 4.38–4.27 (m, 1H), 3.83–3.76 (m, 1H), 3.73–3.64 (m, 2H), 3.54–3.46 (m, 2H), 3.21–3.12 (m, 4H), 2.94 (s, 3H), 2.52 (s, 3H), 2.45–2.33 (m, 4H), 2.21–2.02 (m, 6H). ESI-MS <i>m</i>/<i>z</i> 474.3 [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>35</sub>N<sub>5</sub>O<sub>2</sub>.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 6,7-Dimethoxy-2-(5-methyl-2-furyl)-<i>N</i>-(1-methyl-4-piperidyl)quinolin-4-amine (<b>34</b>)</h4><div class="NLM_p last">To a solution of compound <b>74a</b> (70 mg, 0.23 mmol) in 1,4-dioxane (5 mL) was added Cs<sub>2</sub>CO<sub>3</sub> (226 mg, 0.69 mmol), BINAP (423 mg, 0.69 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (38 mg, 0.041 mmol), and 1-methylpiperidin-4-amine (52 mg, 0.46 mmol), and the mixture was heated to 110 °C for 3 h under Microwave. Then, the mixture was quenched with water and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by the prep-HPLC (method 2 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>34</b> (7.8 mg, 9%) as yellow solid; mp 21–220 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.77 (s, 1H), 7.56–7.54 (d, <i>J</i> = 8 Hz, 1H), 7.47 (s, 1H), 7.08 (s, 1H), 6.45 (s, 1H), 4.04–4.02 (m, 7H), 3.73–3.68 (m, 2H), 3.27–3.20 (m, 2H), 2.96 (s 3H), 2.52 (s, 3H), 2.45–2.30 (m, 2H), 2.20–2.05 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 382.3 [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>27</sub>N<sub>3</sub>O<sub>3</sub>.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 7-[3-(Dimethylamino)propoxy]-2-(5-ethyl-2-furyl)-6-methoxy-<i>N</i>-(1-methyl-4-piperidyl)quinolin-4-amine (<b>35</b>)</h4><div class="NLM_p last">A mixture of <b>68a</b> (150 mg, 0.455 mmol), 2-(5-ethyl-2-furyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (121 mg, 0.546 mmol), K<sub>2</sub>CO<sub>3</sub> (157 mg, 1.14 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (53 mg, 0.045 mmol) in 1,4-dioxane/H<sub>2</sub>O (1:1, 10 mL) was degassed and purged with N<sub>2</sub> for 3 times and then the mixture was stirred at 110 °C for 16 h. Then, the reaction mixture was poured into water and extracted with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give a residue which was purified by prep-TLC (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 10:1) to afford pure intermediate 3-[[4-chloro-2-(5-ethyl-2-furyl)-6-methoxy-7-quinolyl]oxy]-<i>N</i>,<i>N</i>-dimethyl-propan-1-amine (150 mg, 85%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 389.3 [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>25</sub>ClN<sub>2</sub>O<sub>3</sub>. A mixture of this intermediate (150 mg, 0.385 mmol), 1-methylpiperidin-4-amine (88 mg, 0.771 mmol), Cs<sub>2</sub>CO<sub>3</sub> (251 mg, 0.7712 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (35 mg, 0.038 mmol), and BINAP (24 mg, 0.038 mmol) in 1,4-dioxane (5 mL) was degassed and purged with N<sub>2</sub> for 3 times, and the mixture was stirred at 120 °C for 16 h. Then, reaction mixture was concentrated in vacuum to give a residue which was successively purified by prep-TLC (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 7:1) and prep-HPLC (method 3 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>35</b> (6 mg, 3%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.82 (s, 1H), 7.60 (d, <i>J</i> = 3.2 Hz, 1H), 7.52 (s, 1H), 7.09 (s, 1H), 6.46 (d, <i>J</i> = 3.6 Hz, 1H), 4.36–4.34 (m, 3H), 4.05 (s, 3H), 3.72–3.69 (m, 2H), 3.44–3.42 (m, 2H), 3.00 (s, 6H), 2.95 (s, 3H), 2.88 (q, <i>J</i> = 7.6 Hz, 15.2 Hz, 2H), 2.40–2.38 (m, 6H), 2.17–2.14 (m, 2H), 1.36 (t, <i>J</i> = 14.8 Hz, 3H). ESI-MS <i>m</i>/<i>z</i> 467.4 [M + H]<sup>+</sup> calcd for C<sub>27</sub>H<sub>38</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 7-[3-(5-Azaspiro[2.4]heptan-5-yl)propoxy]-2-(5-ethyl-2-furyl)-6-methoxy-<i>N</i>-(1-methyl-4-piperidyl)quinolin-4-amine (<b>36</b>)</h4><div class="NLM_p last">A mixture of <b>68b</b> (100 mg, 0.262 mmol), 2-(5-ethyl-2-furyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (70 mg, 0.314 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (30 mg, 0.026 mmol), and K<sub>2</sub>CO<sub>3</sub> (90 mg, 0.655 mmol) in 1,4-dioxane/H<sub>2</sub>O (1:1, 12 mL) was degassed and purged with N<sub>2</sub> for 3 times, and the mixture was stirred at 110 °C for 16 h under N<sub>2</sub> atmosphere. Then, the reaction mixture was concentrated in vacuum to give a residue which was purified by prep-TLC (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 10:1) to afford intermediate 7-[3-(5-azaspiro[2.4]heptan-5-yl)propoxy]-4-chloro-2-(5-ethyl-2-furyl)-6-methoxy-quinoline (230 mg, containing some impurities) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 441.3 [M + H]<sup>+</sup> calcd for C<sub>25</sub>H<sub>29</sub>ClN<sub>2</sub>O<sub>3</sub>. Then, a mixture of this intermediate (130 mg, 0.294 mmol), 1-methylpiperidin-4-amine (67 mg, 0.589 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (27 mg, 0.029 mmol), BINAP (18 mg, 0.029 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (192 mg, 0.589 mmol) in 1,4-dioxane (5 mL) was degassed and purged with N<sub>2</sub> for 3 times, and the mixture was stirred at 120 °C for 16 h under N<sub>2</sub> atmosphere. Then, the reaction mixture was concentrated and successively purified by prep-TLC (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 10:1) and prep-HPLC (method 3 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>36</b> (10 mg, 6%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.82 (s, 1H), 7.59 (d, <i>J</i> = 3.6 Hz, 1H), 7.51 (s, 1H), 7.09 (s, 1H), 6.47 (d, <i>J</i> = 3.6 Hz, 1H), 4.36–4.31 (m, 3H), 4.05 (s, 3H), 4.04–3.96 (m, 2H), 3.72–3.69 (m, 2H), 3.55–3.52 (m, 3H), 3.43–3.39 (m, 1H), 2.96 (s, 3H), 2.90–2.82 (m, 2H), 2.42–2.38 (m, 4H), 2.19–2.06 (m, 6H), 1.36 (t, <i>J</i> = 14.8 Hz, 3H), 0.83–0.78 (m, 4H). ESI-MS <i>m</i>/<i>z</i> 519.4 [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>42</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 2-(5-Ethyl-2-furyl)-6-methoxy-<i>N</i>-(1-methyl-4-piperidyl)-7-[3-(1-piperidyl)propoxy]quinolin-4-amine (<b>37</b>)</h4><div class="NLM_p last">A mixture of <b>71a</b> (200 mg, 0.541 mmol), 2-(5-ethyl-2-furyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (144 mg, 0.650 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (62 mg, 0.054 mmol), and K<sub>2</sub>CO<sub>3</sub> (187 mg, 1.35 mmol) in 1,4-dioxane (3 mL) was degassed and purged with N<sub>2</sub> for 3 times, and the mixture was stirred at 110 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was purified by prep-TLC (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 10:1) to afford intermediate 4-chloro-2-(5-ethyl-2-furyl)-6-methoxy-7-[3-(1-piperidyl)propoxy]quinoline (220 mg, 94%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 429.3 [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>29</sub>ClN<sub>2</sub>O<sub>3</sub>. Then, a mixture of this intermediate (220 mg, 0.513 mmol), 1-methylpiperidin-4-amine (117 mg, 1.03 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (47 mg, 0.051 mmol), BINAP (32 mg, 0.051 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (334 mg, 1.03 mmol) in 1,4-dioxane (10 mL) was degassed and purged with N<sub>2</sub> for 3 times, and the mixture was stirred at 120 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was successively purified by prep-TLC (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 10:1) and prep-HPLC (method 3 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>37</b> (15.1 mg 6%) as a yellow solid; mp 127–128 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.81 (s, 1H), 7.59 (d, <i>J</i> = 3.6 Hz, 1H), 7.51 (s, 1H), 7.09 (s, 1H), 6.47 (d, <i>J</i> = 3.6 Hz, 1H), 4.35–4.32 (m, 3H), 4.04 (s, 3H), 3.69–3.67 (m, 4H), 3.39–3.37 (m, 3H), 3.31–3.26 (m, 1H), 3.00–2.96 (m, 5H), 2.86 (q, <i>J</i> = 7.6 Hz, 14.8 Hz, 2H), 2.41–2.38 (m, 4H), 2.16–1.80 (m, 7H), 1.59–1.51 (m, 1H), 1.36 (t, <i>J</i> = 14.8 Hz, 3H). ESI-MS <i>m</i>/<i>z</i> 507.4 [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>42</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 2-(5-Ethyl-2-furyl)-6-methoxy-<i>N</i>-(1-methyl-4-piperidyl)-7-(3-morpholinopropoxy)quinolin-4-amine (<b>38</b>)</h4><div class="NLM_p last">A mixture of <b>71b</b> (200 mg, 0.539 mmol), 2-(5-ethyl-2-furyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (143 mg, 0.6465 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (62 mg, 0.054 mmol), and K<sub>2</sub>CO<sub>3</sub> (186 mg, 1.35 mmol) in 1,4-dioxane/H<sub>2</sub>O (1:1, 6 mL) was degassed and purged with N<sub>2</sub> for 3 times, and the mixture was stirred at 110 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was purified by TLC (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 12:1) to give intermediate 4-[3-[[4-chloro-2-(5-ethyl-2-furyl)-6-methoxy-7-quinolyl]oxy]propyl]morpholine (190 mg, 81%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 431.1 [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>27</sub>ClN<sub>2</sub>O<sub>4</sub>. Then, a mixture of this intermediate (190 mg, 0.441 mmol), 1-methylpiperidin-4-amine (100 mg, 0.882 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (40 mg, 0.044 mmol), BINAP (27 mg, 0.044 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (287 mg, 0.882 mmol) in 1,4-dioxane (10 mL) was degassed and purged with N<sub>2</sub> for 3 times, and the mixture was stirred at 120 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was successively purified by TLC (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 10:1) and prep-HPLC (method 3 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>38</b> (25.7 mg 11%) as a yellow solid; mp 121–122 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.81 (s, 1H), 7.59 (d, <i>J</i> = 3.2 Hz, 1H), 7.51 (s, 1H), 7.09 (s, 1H), 6.47 (d, <i>J</i> = 3.6 Hz, 1H), 4.37–4.31 (m, 3H), 4.04 (s, 3H), 3.91–3.82 (m, 3H), 3.72–3.69 (m, 3H), 3.48–3.45 (m, 3H), 3.26–3.13 (m, 3H), 2.96 (s, 3H), 2.88 (q, <i>J</i> = 7.2 Hz, 14.8 Hz, 2H), 2.42–2.38 (m, 6H), 2.16–2.12 (m, 2H), 1.36 (t, <i>J</i> = 14.8 Hz, 3H). ESI-MS <i>m</i>/<i>z</i> 509.4 [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>40</sub>N<sub>4</sub>O<sub>4</sub>.</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 7-[3-(3-Fluoropyrrolidin-1-yl)propoxy]-6-methoxy-2-(5-methyl-2-furyl)-<i>N</i>-(1-methyl-4-piperidyl)quinolin-4-amine (<b>39</b>)</h4><div class="NLM_p last">A mixture of <b>67c</b> (400 mg, 1.07 mmol), 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane (156 mg, 0.75 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (123 mg, 0.107 mmol), and K<sub>2</sub>CO<sub>3</sub> (296 mg, 2.14 mmol) in 1,4-dioxane/H<sub>2</sub>O (10:1, 55 mL) was degassed and purged with N<sub>2</sub> for 3 times, and the mixture was stirred at 110 °C for 16 h. Then, the mixture was concentrated and the residue was extracted with EtOAc. The combined organic phase was washed with brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give intermediate 4-chloro-7-[3-(3-fluoropyrrolidin-1-yl)propoxy]-6-methoxy-2-(5-methyl-2-furyl)quinoline (400 mg, 89%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 419.1 [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>24</sub>ClFN<sub>2</sub>O<sub>3</sub>. A mixture of this intermediate (50 mg, 0.119 mmol), 1-methylpiperidin-4-amine (41 mg, 0.358 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (22 mg, 0.024 mmol), Cs<sub>2</sub>CO<sub>3</sub> (78 mg, 0.239 mmol), and BINAP (15 mg, 0.024 mmol) in 1,4-dioxane (5.00 mL) was degassed and purged with N<sub>2</sub> for 3 times, and the mixture was stirred at 120 °C for 12 h. Then, the reaction mixture was filtrated and the filtrate was concentrated under vacuum to give the crude product. The crude product was purified by prep-HPLC (method 11 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford compound <b>39</b> (5.0 mg, 8%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.83 (s, 1H), 7.61 (d, <i>J</i> = 3.2 Hz, 1H), 7.51 (s, 1H), 7.11 (s, 1H), 6.45 (d, <i>J</i> = 3.2 Hz, 1H), 5.58–5.45 (m, 1H), 4.36 (d, <i>J</i> = 4.8 Hz, 3H), 4.10–4.00 (m, 5H), 3.73–3.67 (m, 2H), 3.60–3.50 (m, 6H), 2.95 (s, 3H), 2.62–2.55 (m, 1H), 2.51 (s, 3H), 2.49–2.26 (m, 5H), 2.26–2.09 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 497.3 [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>37</sub>FN<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 6-Methoxy-2-(5-methyl-2-furyl)-<i>N</i>-(1-methyl-4-piperidyl)-7-(4-piperidylmethoxy)quinolin-4-amine (<b>40</b>)</h4><div class="NLM_p last">To a mixture of <b>74b</b> (650 mg, 2.24 mmol), <i>tert</i>-butyl 4-(hydroxymethyl) piperidine-1-carboxylate (540 mg, 2.51 mmol), and PPh<sub>3</sub> (1.18 g, 4.48 mmol) in THF (50 mL) was added DIAD (907 mg, 4.48 mmol), and the mixture was stirred at 0 °C for 8 h. Then, the mixture was concentrated and extracted with EtOAc. The combined organic phase was washed with saturated brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum. The residue was purified by silica gel column chromatography (PE/EtOAc = 2/1) to afford intermediate <i>tert</i>-butyl 4-[[4-chloro-6-methoxy-2-(5-methyl-2-furyl)-7-quinolyl]oxymethyl]piperidine-1-carboxylate (700 mg, 64%). ESI-MS <i>m</i>/<i>z</i> 487.3 [M + H]<sup>+</sup> calcd for C<sub>26</sub>H<sub>31</sub>ClN<sub>2</sub>O<sub>5</sub>. To a mixture of this intermediate (250 mg, 0.51 mmol) and 1-methylpiperidin-4-amine (120 mg, 1.06 mmol) in 1,4-dioxane (20 mL) was added Cs<sub>2</sub>CO<sub>3</sub> (344 mg, 1.06 mmol) and Pd(dba)<sub>2</sub> (30 mg, 0.052 mmol). The mixture was stirred at 120 °C for 12 h. Then, the solution was concentrated and the residue was purified by silica gel column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10/1) to afford intermediate <i>tert</i>-butyl 4-[[6-methoxy-2-(5-methyl-2-furyl)-4-[(1-methyl-4-piperidyl)amino]-7-quinolyl]oxymethyl]piperidine-1-carboxylate (100 mg, 34%) as yellow solid. ESI-MS <i>m</i>/<i>z</i> 565.2 [M + H]<sup>+</sup> calcd for C<sub>32</sub>H<sub>44</sub>N<sub>4</sub>O<sub>5</sub>. Finally, to a solution of this intermediate (160 mg, 0.283 mmol) in EtOAc (10 mL) was added HCl/EtOAc (10 mL, 2.0 M), and the mixture was stirred at 25 °C for 3 h. Then, the solution was concentrated and the residue was purified by prep-HPLC (method 3 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>40</b> (100 mg, 75%) as yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.79 (s, 1H), 7.58 (d, <i>J</i> = 3.2 Hz, 1H), 7.48 (s, 1H), 7.08 (s, 1H), 6.43 (d, <i>J</i> = 3.2 Hz, 1H), 4.37–4.26 (m, 1H), 4.14–4.10 (m, 2H), 4.01 (s, 3H), 3.74–3.65 (m, 2H), 3.54–3.46 (m, 2H), 3.35–3.27 (m, 2H), 3.16–3.02 (m, 2H), 2.95 (s, 3H), 2.50 (s, 3H), 2.45–2.23 (m, 3H), 2.20–2.08 (m, 4H), 1.79–1.69 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 465.3 [M + H]<sup>+</sup> calcd for C<sub>27</sub>H<sub>36</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 6-Methoxy-2-(5-methyl-2-furyl)-<i>N</i>-(1-methyl-4-piperidyl)-7-[(1-methyl-4-piperidyl)methoxy]quinolin-4-amine (<b>41</b>)</h4><div class="NLM_p last">To a solution of compound <b>40</b> (80 mg, 0.172 mmol) in 1,4-dioxane (5 mL) was added (HCHO)<i><sub>n</sub></i> (46.5 mg, 0.516 mmol), NaBH(OAc)<sub>3</sub> (109 mg, 0.516 mmol), and HCOOH (8.3 mg, 0.172 mmol), and the mixture was stirred at 100 °C for 2 h. Then, the solution was concentrated and the residue was purified by prep-HPLC (method 3 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>41</b> (15.0 mg, 18%) as yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.81 (s, 1H), 7.62 (d, <i>J</i> = 3.6 Hz, 1H), 7.50 (s, 1H), 7.09 (s, 1H), 6.43 (d, <i>J</i> = 2.4 Hz, 1H), 4.39–4.30 (m, 1H), 4.13–4.08 (m, 2H), 4.02 (s, 3H), 3.71–3.63 (m, 2H), 3.62–3.55 (m, 2H), 3.35–3.28 (m, 2H), 3.13–3.04 (m, 2H), 2.95 (s, 3H), 2.90 (s, 3H), 2.50 (s, 3H), 2.38–2.10 (m, 7H), 1.84–1.71 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 479.4 [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>38</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 6-Methoxy-2-(5-methyl-2-furyl)-<i>N</i>-(1-methyl-4-piperidyl)-7-(4-piperidyloxy)quinolin-4-amine (<b>42</b>)</h4><div class="NLM_p last">To a mixture of <b>75</b> (100 mg, 0.272 mmol) and <i>tert</i>-butyl 4-methylsulfonyloxypiperidine-1-carboxylate (91 mg, 0.326 mmol) in DMF (5.00 mL) was added Cs<sub>2</sub>CO<sub>3</sub> (177 mg, 0.544 mmol), and the mixture was stirred at 100 °C for 16 h under N<sub>2</sub>. Then, the mixture was concentrated and purified by prep-TLC (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 10:1) to afford intermediate <i>tert</i>-butyl 4-[[6-methoxy-2-(5-methyl-2-furyl)-4-[(1-methyl-4-piperidyl)amino]-7-quinolyl]oxy]piperidine-1-carboxylate (50 mg, 33%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 551.3 [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>42</sub>N<sub>4</sub>O<sub>5</sub>. Then, a solution of this intermediate (50 mg, 0.091 mmol) in HCl/EtOAc (5.00 mL, 1.0 M) was stirred at 25 °C for 2 h under N<sub>2</sub>. The mixture was concentrated and purified by prep-HPLC (method 3 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>42</b> (10.0 mg, 24%) as a yellow solid; mp 150–151 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.84 (s, 1H), 7.61 (s, 2H), 7.09 (s, 1H), 6.45 (s, 1H), 4.96–4.90 (m, 2H), 4.32 (s, 1H), 4.04 (s, 3H), 3.72–3.65 (m, 2H), 3.48–3.43 (m, 2H), 3.31 (s, 1H), 2.95 (s, 3H), 2.50 (s, 3H), 2.40–2.19 (m, 10H). ESI-MS <i>m</i>/<i>z</i> 451.3 [M + H]<sup>+</sup> calcd for C<sub>26</sub>H<sub>34</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 6-Methoxy-7-[(2-methyl-2-azaspiro[3.3]heptan-6-yl)oxy]-2-(5-methyl-2-furyl)-<i>N</i>-(1-methyl-4-piperidyl)quinolin-4-amine (<b>43</b>)</h4><div class="NLM_p last">To a mixture of compound <b>75</b> (100 mg, 0.272 mmol) and <i>tert</i>-butyl 6-methylsulfonyloxy-2-azaspiro[3.3]heptane-2-carboxylate (95 mg, 0.326 mmol) in DMF (5.0 mL) was added Cs<sub>2</sub>CO<sub>3</sub> (177 mg, 0.544 mmol), and the mixture was stirred at 100 °C for 16 h. Then, the mixture was concentrated and purified by TLC to give intermediate <i>tert</i>-butyl 6-[[6-methoxy-2-(5-methyl-2-furyl)-4-[(1-methyl-4-piperidyl)amino]-7-quinolyl]oxy]-2-azaspiro[3.3]heptane-2-carboxylate (60.0 mg, 39%). ESI-MS <i>m</i>/<i>z</i> 563.3 [M + H]<sup>+</sup> calcd for C<sub>32</sub>H<sub>42</sub>N<sub>4</sub>O<sub>5</sub>. A mixture of this intermediate (100 mg, 0.177 mmol), and LiAlH<sub>4</sub> (34 mg, 0.89 mmol) in dry THF (10.0 mL) was degassed and purged with N<sub>2</sub> for 3 times, and the mixture was stirred at 70 °C for 16 h under N<sub>2</sub> atmosphere. Then, the reaction was quenched with water and concentrated. The residue was purified by prep-HPLC (method 2 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>43</b> (21 mg, 25%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.77 (s, 1H), 7.60 (d, <i>J</i> = 3.6 Hz, 1H), 7.35 (s, 1H), 7.07 (s, 1H), 6.42 (d, <i>J</i> = 4.4 Hz, 1H), 4.46–4.43 (m, 1H), 4.35–4.31 (m, 2H), 4.17–4.10 (m, 2H), 4.00 (s, 3H), 3.70–3.67 (m, 2H), 3.35–3.28 (m, 3H), 3.07–3.01 (m, 2H), 2.95 (s, 3H), 2.92 (s, 3H), 2.57–2.55 (m, 2H), 2.49 (s, 3H), 2.38–2.35 (m, 2H), 2.17–2.14 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 477.4 [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>36</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 7-[(2-Isopropyl-2-azaspiro[3.3]heptan-6-yl)oxy]-6-methoxy-2-(5-methyl-2-furyl)-<i>N</i>-(1-methyl-4-piperidyl)quinolin-4-amine (<b>44</b>)</h4><div class="NLM_p last">To a solution of intermediate <i>tert</i>-butyl 6-[[6-methoxy-2-(5-methyl-2-furyl)-4-[(1-methyl-4-piperidyl)amino]-7-quinolyl]oxy]-2-azaspiro[3.3]heptane-2-carboxylate (described in the synthesis of compound <b>43</b>) (170 mg, 0.302 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (12 mL) was added TFA (2.6 g) at 0 °C, and the mixture was stirred at 18 °C for 2 h. Then, the mixture was concentrated to obtain intermediate 7-(2-azaspiro[3.3]heptan-6-yloxy)-6-methoxy-2-(5-methyl-2-furyl)-<i>N</i>-(1-methyl-4-piperidyl)quinolin-4-amine (500 mg, crude) as yellow oil. ESI-MS <i>m</i>/<i>z</i> 463.3 [M + H]<sup>+</sup> calcd for C<sub>27</sub>H<sub>34</sub>N<sub>4</sub>O<sub>3</sub>. Then, to a solution of this intermediate (150 mg, 0.325 mmol) in <i>i</i>-PrOH (4 mL) were added NaBH<sub>3</sub>CN (98 mg, 1.56 mmol), acetone (91 mg, 1.56 mmol), and AcOH (94 mg, 1.56 mmol) at 18 °C under N<sub>2</sub>, and the mixture was stirred at 60 °C for 16 h. Then, the mixture was concentrated. The residue was purified by prep-HPLC (method 12 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>44</b> (23 mg, 14%) as yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.78 (s, 1H), 7.58 (d, <i>J</i> = 3.53 Hz, 1H), 7.34 (s, 1H), 7.07 (s, 1H), 6.43 (d, <i>J</i> = 3.53 Hz, 1H), 4.31 (br s, 1H), 4.26 (d, <i>J</i> = 19.41 Hz, 2H), 4.20 (s, 2H), 4.01 (s, 3H), 3.69 (d, <i>J</i> = 12.35 Hz, 2H), 3.43–3.39 (m, 1H), 3.32 (br s, 1H), 3.29–3.23 (m, 1H), 3.12–3.07 (m, 1H), 2.99 (br s, 1H), 2.94 (s, 3H), 2.91–2.86 (m, 1H), 2.64–2.49 (m, 5H), 2.38 (d, <i>J</i> =13.23 Hz, 2H), 2.19–2.08 (m, 2H), 1.23 (dd, <i>J</i> = 7.94 Hz, <i>J</i> = 6.62 Hz, 6H). ESI-MS <i>m</i>/<i>z</i> 505.4 [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>40</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 7-[(2-Cyclopropyl-2-azaspiro[3.3]heptan-6-yl)oxy]-6-methoxy-2-(5-methyl-2-furyl)-<i>N</i>-(1-methyl-4-piperidyl)quinolin-4-amine (<b>45</b>)</h4><div class="NLM_p last">To a solution of intermediate 7-(2-azaspiro[3.3]heptan-6-yloxy)-6-methoxy-2-(5-methyl-2-furyl)-<i>N</i>-(1-methyl-4-piperidyl)quinolin-4-amine (described in the synthesis of compound <b>44</b>, 100 mg, 0.216 mmol) and (1-ethoxycyclopropoxy)-trimethyl-silane (226 mg, 1.30 mmol) in <i>t</i>-BuOH (10 mL) were added NaBH<sub>3</sub>CN (81 mg, 1.30 mmol) and AcOH (77 mg, 1.30 mmol), and the mixture was stirred at 60 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was purified by prep-HPLC (method 13 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford compound <b>45</b> (30 mg, 27%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.78 (s, 1H), 7.59 (d, <i>J</i> = 3.6 Hz, 1H), 7.34 (s, 1H), 7.07 (s, 1H), 6.43 (d, <i>J</i> = 3.6 Hz, 1H), 4.35–4.28 (m, 6H), 4.01 (s, 3H), 3.71–3.68 (m, 2H), 3.36–3.29 (m, 2H), 3.04–2.99 (m, 3H), 2.95 (s, 3H), 2.58–2.55 (m, 2H), 2.50 (s, 3H), 2.39–2.36 (m, 2H), 2.16–2.10 (m, 2H), 0.92–0.85 (m, 4H). ESI-MS <i>m</i>/<i>z</i> 503.3 [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>38</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 6-Methoxy-<i>N</i>-(7-methyl-7-azaspiro[3.5]nonan-2-yl)-2-(5-methyl-2-furyl)-7-[(1-methyl-4-piperidyl)methoxy]quinolin-4-amine (<b>46</b>)</h4><div class="NLM_p last">To a solution of <b>80a</b> (30.0 mg, 0.061 mmol) in MeOH (5.0 mL) was added HCOOH (6 mg, 0.13 mmol), NaBH(OAc)<sub>3</sub> (78 mg, 0.367 mmol), and (HCHO)<i><sub>n</sub></i> (33 mg, 0.367 mmol), and the mixture was stirred at 70 °C for 12 h. Then, the mixture was concentrated, and the residue was purified by prep-HPLC (method 3 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>46</b> (8.0 mg, 25%) as yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.80 (s, 1H), 7.54 (s, 1H), 7.46 (s, 1H), 6.81 (s, 1H), 6.43 (s, 1H), 4.53 (d, <i>J</i> = 7.6 Hz 1H), 4.13 (d, <i>J</i> = 5.2 Hz, 2H), 4.02 (s, 3H), 3.64–3.55 (m, 2H), 3.53–3.48 (m, 2H), 3.44–3.36 (m, 2H), 3.16–3.02 (m, 4H), 3.00–2.80 (m, 6H), 2.80–2.71 (m, 1H), 2.51 (s, 3H), 2.31–2.09 (m, 6H), 2.01–1.84 (m, 2H), 1.83–1.73 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 519.4 [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>42</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 6-Methoxy-7-[(2-methyl-2-azaspiro[3.3]heptan-6-yl)oxy]-2-(5-methyl-2-furyl)-<i>N</i>-[(1-methyl-4-piperidyl)methyl]quinolin-4-amine (<b>47</b>)</h4><div class="NLM_p last">To a solution of compound <b>80b</b> (150 mg, 0.325 mmol) in MeOH (10 mL) were added NaBH<sub>3</sub>CN (49 mg, 0.780 mmol), HCOOH (12 mg, 0.260 mmol), and (HCHO)<i><sub>n</sub></i> (70 mg, 0.780 mmol), and the mixture was stirred at 60 °C for 16 h. Then the mixture was concentrated and the residue was purified by prep-HPLC (method 14 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>47</b> (9.5 mg, 6%) as yellow solid; mp 158–160 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.70 (s, 1H), 7.56 (d, <i>J</i> = 3.53 Hz, 1H), 7.33 (s, 1H), 6.98 (s, 1H), 6.45–6.39 (m, 1H), 4.47–4.34 (m, 2H), 4.14 (br s, 2H), 4.01 (s, 3H), 3.64–3.52 (m, 4H), 3.14–2.94 (m, 5H), 2.92 (s, 3H), 2.86 (s, 3H), 2.57 (d, <i>J</i> = 12.35 Hz, 2H), 2.50 (s, 3H), 2.16 (d, <i>J</i> = 13.23 Hz, 3H), 1.70–1.60 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 491.4 [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>38</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 6-Methoxy-7-[(2-methyl-2-azaspiro[3.3]heptan-6-yl)oxy]-<i>N</i>-(7-methyl-7-azaspiro[3.5]nonan-2-yl)-2-(5-methyl-2-furyl)quinolin-4-amine (<b>48</b>)</h4><div class="NLM_p last">To a solution of compound <b>80c</b> (150 mg, 0.249 mmol) in MeOH (10 mL) were added NaBH(OAc)<sub>3</sub> (317 mg, 1.49 mmol), HCOOH (12 mg, 0.249 mmol), and (HCHO)<i><sub>n</sub></i> (135 mg, 1.49 mmol), and the mixture was stirred at 60 °C for 16 h. Then, the mixture was concentrated and the residue was purified by prep-HPLC (method 15 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>48</b> (6.8 mg, 5%) as yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.79 (s, 1H), 7.54 (d, <i>J</i> = 3.2 Hz, 1H), 7.31 (s, 1H), 6.80 (s, 1H), 6.42 (d, <i>J</i> = 2.4 Hz, 1H), 4.52–4.48 (m, 1H), 4.44 (d, <i>J</i> = 15.6 Hz, 1H), 4.36 (d, <i>J</i> = 10.4 Hz, 1H), 4.13 (d, <i>J</i> = 5.6 Hz, 2H) 4.01 (s, 3H), 3.52–3.38 (m, 3H), 3.16–3.11 (m, 1H), 2.97 (d, <i>J</i> = 5.2 Hz, 1H), 2.92 (s, 3H), 2.87 (s, 3H), 2.80–2.75 (m, 1H), 2.60–2.53 (m, 3H), 2.50 (s, 3H), 2.31–2.21 (m, 3H), 2.15–2.10 (m, 1H), 2.00–1.88 (m, 4H). ESI-MS <i>m</i>/<i>z</i> 517.4 [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>40</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> <i>N</i>-(7-Isopropyl-7-azaspiro[3.5]nonan-2-yl)-6-methoxy-7-[(2-methyl-2-azaspiro[3.3]heptan-6-yl)oxy]-2-(5-methyl-2-furyl)quinolin-4-amine (<b>49</b>)</h4><div class="NLM_p last">To a solution of compound <b>85</b> (250 mg, 0.471 mmol) in MeOH (15 mL) were added NaBH(OAc)<sub>3</sub> (300 mg, 1.41 mmol), HCOOH (22 mg, 0.471.08 mmol), and (HCHO)<sub><i>n</i></sub> (127 mg, 1.41 mmol), and the mixture was stirred at 60 °C for 16 h under N<sub>2</sub>. Then, the mixture was concentrated and the residue was purified by prep-HPLC (method 16 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>49</b> (14.4 mg, 5%) as yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.79 (s, 1H), 7.54 (br s, 1H), 7.31 (s, 1H), 6.79 (s, 1H), 6.42 (br s, 1H), 4.52–4.43 (m, 2H), 4.36 (d, <i>J</i> = 10.6 Hz, 1H), 4.17–4.07 (m, 2H), 4.01 (s, 3H), 3.53–3.45 (m, 2H), 3.39 (d, <i>J</i> = 12.4 Hz, 1H), 3.15–3.05 (m, 2H), 2.97 (br s, 2H), 2.92 (s, 3H), 2.79 (br s, 1H), 2.61–2.48 (m, 6H), 2.32–2.23 (m, 2H), 2.14–2.09 (m, 1H), 2.07–1.86 (m, 4H), 1.38 (d, <i>J</i> = 6.6 Hz, 6H). ESI-MS <i>m</i>/<i>z</i> 545.4 [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>44</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 6-Methoxy-<i>N</i>-(3-methyl-3-azabicyclo[3.2.1]octan-8-yl)-7-[(2-methyl-2-azaspiro[3.3]heptan-6-yl)oxy]-2-(5-methyl-2-furyl)quinolin-4-amine (<b>50</b>)</h4><div class="NLM_p last">To a solution of compound <b>80d</b> (150 mg, 0.254 mmol) in MeOH (10 mL) were added NaBH<sub>3</sub>CN (48 mg, 0.764 mmol), HCOOH (12 mg, 0.254 mmol), and (HCHO)<i><sub>n</sub></i> (69 mg, 0.764 mmol), and the mixture was stirred at 60 °C for 16 h. Then, the mixture was concentrated and the residue was purified by prep-HPLC (method 17 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>50</b> (10 mg, 7%) as yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.81 (s, 1H), 7.51 (d, <i>J</i> = 3.53 Hz, 1H), 7.29 (s, 1H), 6.89 (s, 1H), 6.37–6.31 (m, 1H), 4.36 (d, <i>J</i> = 10.58 Hz, 1H), 4.27 (d, <i>J</i> = 10.14 Hz, 1H), 4.08–4.01 (m, 2H), 3.98 (s, 3H), 3.93 (t, <i>J</i> = 4.19 Hz, 1H), 3.43–3.38 (m, 2H), 3.31–3.26 (m, 2H), 2.96 (dd, <i>J</i> = 11.03, 4.85 Hz, 1H), 2.93–2.84 (m, 2H), 2.83 (s, 3H), 2.82–2.77 (m, 5H), 2.52–2.45 (m, 2H), 2.41 (s, 3H), 2.22–2.15 (m, 2H), 1.93–1.86 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 503.4 [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>38</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 1-[3-(2-Methoxy-5-nitro-phenoxy)propyl]pyrrolidine (<b>52</b>)</h4><div class="NLM_p last">To a solution of commercially available 2-methoxy-5-nitro-phenol (<b>51</b>) (39.2 g, 0.24 mol) in THF (1.0 L), PPh<sub>3</sub> (121 g, 0.46 mol), 3-pyrrolidin-1-yl-propan-1-ol (41.08 g, 0.32 mol), and DEAD (79.12 g. 0.46 mmol) were added at 0 °C, and the reaction mixture was stirred at room temperature for 5 h. Then, the reaction mixture was concentrated and extracted with EtOAc. The combined organic layers were washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by silica gel column chromatography to obtain pure compound <b>52</b> (40 g, 60%) as yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 7.92 (d, <i>J</i> = 2.4 Hz, 1H), 7.89 (s, 1H), 6.90 (d, <i>J</i> = 2.4 Hz, 1H), 4.23–4.14 (m, 2H), 3.97 (s, 3H), 2.70–2.60 (m, 2H), 2.58–2.50 (m, 4H), 2.15–2.05 (m, 2H), 1.85–1.75 (m, 4H). ESI-MS <i>m</i>/<i>z</i> 281.2 [M + H]<sup>+</sup> calcd for C<sub>14</sub>H<sub>20</sub>N<sub>2</sub>O<sub>4</sub>.</div></div><div id="sec4_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 4-Methoxy-3-(3-pyrrolidin-1-ylpropoxy)aniline (<b>53</b>)</h4><div class="NLM_p last">To a solution of compound <b>52</b> (28 g, 0.1 mol) in MeOH (500 mL) was added Pd/C (5 g), and the solution was stirred at room temperature for 3 h under H<sub>2</sub> atmosphere (1 atm). Then, the solution was filtered and the filtrate was concentrated to give compound <b>53</b> (21 g, 84%) as yellow oil which was used in the next step without further purification. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 6.70 (d, <i>J</i> = 8.4 Hz, 1H), 6.34 (s, 1H), 6.22 (d, <i>J</i> = 8.4 Hz, 1H), 4.08–4.00 (m, 2H), 3.79 (s, 3H), 2.70–2.60 (m, 2H), 2.59–2.52 (m, 4H), 2.10–2.02 (m, 2H), 1.90–1.70 (m, 4H). ESI-MS <i>m</i>/<i>z</i> 251.2 [M + H]<sup>+</sup> calcd for C<sub>14</sub>H<sub>22</sub>N<sub>2</sub>O<sub>2</sub>.</div></div><div id="sec4_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 2,4-Dichloro-6-methoxy-7-(3-pyrrolidin-1-ylpropoxy)quinoline (<b>54</b>)</h4><div class="NLM_p last">To a solution of compound <b>53</b> (20 g, 0.08 mol) in POCl<sub>3</sub> (130 mL) was added malonic acid (10.3 g, 0.098 mol) at room temperature, and the mixture was stirred for 4 h. Then, the solution was heated to 90 °C and stirred for overnight. POCl<sub>3</sub> was removed by distillation under vacuum and the residue was poured into ice–water. The resulting mixture was extracted with EtOAc, and the combined organic layers were washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give compound <b>54</b> (7 g, 24%) as pale-yellow solid which was used in the next step without further purification. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 7.33–7.28 (m, 2H), 7.19 (s, 1H), 4.20–4.10 (m, 2H), 3.96 (s, 3H), 2.80–2.60 (m, 6H), 2.20–2.10 (m, 2H), 1.95–1.75 (m, 4H). ESI-MS <i>m</i>/<i>z</i> 355.3 [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>2</sub>.</div></div><div id="sec4_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 4-Chloro-6-methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinoline (<b>55a</b>)</h4><div class="NLM_p last">To a solution of compound <b>54</b> (600 mg, 1.7 mmol) in 1,4-dioxane/H<sub>2</sub>O (15:1, 16 mL) were added Na<sub>2</sub>CO<sub>3</sub> (0.54 g, 5.1 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (0.22 g, 0.17 mmol), and 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane (0.39 g, 1.87 mmol), and the mixture was stirred at 110 °C for 4 h under microwave. Then, the mixture was quenched with water and extracted with EtOAc. The combined organic phase was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 1 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to obtain compound <b>55a</b> (400 mg, 59%) as a yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 7.83 (s, 1H), 7.47 (s, 1H), 7.36 (s, 1H), 7.19 (d, <i>J</i> = 3.2 Hz, 1H), 6.31 (d, <i>J</i> = 3.2 Hz, 1H), 4.30–4.22 (m, 2H), 3.96 (s, 3H), 3.65–3.55 (m, 2H), 3.35–3.26 (m, 2H), 3.08–2.98 (m, 2H), 2.39 (s, 3H), 2.23–2.16 (m, 2H), 2.05–1.95 (m, 2H), 1.91–1.82 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 401.2 [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>25</sub>ClN<sub>2</sub>O<sub>3</sub>.</div></div><div id="sec4_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 4-Chloro-2-(5-ethyl-2-furyl)-6-methoxy-7-(3-pyrrolidin-1-ylpropoxy)quinoline (<b>55b</b>)</h4><div class="NLM_p last">To a solution of compound <b>54</b> (354 mg, 1 mmol) and 2-(5-ethyl-2-furyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (222 mg, 1 mmol) in 1,4-dioxane (15 mL) was added K<sub>2</sub>CO<sub>3</sub> (280 mg, 2 mmol) and Pd(PPh<sub>3</sub>)<sub>4</sub> (50 mg, 0.030 mmol), and the solution was heated to 110 °C for 12 h. Then, the mixture was concentrated to give the crude product which was purified by prep-HPLC (method 1 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>55b</b> (150 mg, 36%) as a yellow solid. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.02 (s, 1H), 7.59–7.52 (m, 2H), 7.38 (s, 1H), 6.35–6.40 (m, 1H), 4.40–4.30 (m, 2H), 4.07 (s, 3H), 3.85–3.75 (m, 2H), 3.51–3.45 (m, 2H), 3.18–3.10 (m, 2H), 2.85–2.75 (m, 2H), 2.41–2.32 (m, 2H), 2.25–2.15 (m, 2H), 2.10–2.00 (m, 2H), 1.37–1.31 (m, 3H). ESI-MS <i>m</i>/<i>z</i> 415.2 [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>27</sub>ClN<sub>2</sub>O<sub>3</sub>.</div></div><div id="sec4_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> <i>tert</i>-Butyl 4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]piperidine-1-carboxylate (<b>56a</b>)</h4><div class="NLM_p last">To a solution of compound <b>55a</b> (120 mg, 0.3 mmol) and <i>tert</i>-butyl 4-(aminomethyl)piperidine-1-carboxylate (160 mg, 0.75 mmol) in 1,4-dioxane (15 mL) were successively added Pd<sub>2</sub>(dba)<sub>3</sub> (55 mg, 0.06 mmol), Cs<sub>2</sub>CO<sub>3</sub> (243 mg, 0.75 mmol), and BINAP (75 mg, 0.12 mmol), and the mixture was stirred at 115 °C for 18 h under N<sub>2</sub> and at 125 °C for 10 h. Then, the solution was diluted with water and extracted with EtOAc. The combined organic layer was washed with brine, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give crude product which was purified by prep-TLC (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 5:1) to obtain pure compound <b>56a</b> (125.0 mg, 72%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 579.3 [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>46</sub>N<sub>4</sub>O<sub>5</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> <i>tert</i>-Butyl 4-[1-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-1-methyl–ethyl]piperidine-1-carboxylate (<b>56b</b>)</h4><div class="NLM_p last">A mixture of <b>55a</b> (50 mg, 0.125 mmol), <i>tert</i>-butyl 4-(1-amino-1-methyl–ethyl)piperidine-1-carboxylate (36 mg, 0.149 mmol), Cs<sub>2</sub>CO<sub>3</sub> (81 mg, 0.25 mmol), BINAP (16 mg, 0.024 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (11 mg, 0.012 mmol) in 1,4-dioxane (3.00 mL) was degassed and purged with N<sub>2</sub> for 3 times, and then the mixture was stirred at 120 °C for 24 h under N<sub>2</sub> atmosphere. Then, the reaction mixture was filtrated and the filtrate was concentrated under vacuum to give a residue. The residue was purified by prep-HPLC (method 18 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford compound <b>56b</b> (5.0 mg, 7%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 607.4 [M + H]<sup>+</sup> calcd for C<sub>35</sub>H<sub>50</sub>N<sub>4</sub>O<sub>5</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> <i>tert</i>-Butyl 2-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-7-azaspiro[3.5]nonane-7-carboxylate (<b>56c</b>)</h4><div class="NLM_p last">To a solution of compound <b>55a</b> (400 mg, 1 mmol) in 1,4-dioxane (10 mL) was added Cs<sub>2</sub>CO<sub>3</sub> (650 mg, 2 mmol), BINAP (70 mg, 0.11 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (100 mg, 0.11 mmol), and <i>tert</i>-butyl 2-amino-7-azaspiro[3.5]nonane-7-carboxylate (240 mg, 1 mmol), and the solution was heated to 130 °C for 5 h under microwave. Then, the solution was concentrated and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 1 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford compound <b>56c</b> (200 mg, 33%) as yellow solid. ESI-MS <i>m</i>/<i>z</i> 605.3 [M + H]<sup>+</sup> calcd for C<sub>35</sub>H<sub>48</sub>N<sub>4</sub>O<sub>5</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> <i>tert</i>-Butyl 2-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-6-azaspiro[3.4]octane-6-carboxylate (<b>56d</b>)</h4><div class="NLM_p last">To a solution of compound <b>55a</b> (400 mg, 1 mmol) in 1,4-dioxane (10 mL) was added Cs<sub>2</sub>CO<sub>3</sub> (650 mg, 2 mmol), BINAP (70 mg, 0.11 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (100 mg, 0.11 mmol), and <i>tert</i>-butyl 2-amino-6-azaspiro[3.4]octane-6-carboxylate (226 mg, 1 mmol), and the solution was heated to 130 °C for 5 h under microwave. Then, the solution was concentrated and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 1 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>56d</b> (100 mg, 17%) as yellow solid. ESI-MS <i>m</i>/<i>z</i> 591.3 [M + H]<sup>+</sup> calcd for C<sub>34</sub>H<sub>46</sub>N<sub>4</sub>O<sub>5</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> <i>tert</i>-Butyl 3-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-7-azaspiro[3.5]nonane-7-carboxylate (<b>56e</b>)</h4><div class="NLM_p last">To a solution of compound <b>55a</b> (400 mg, 1 mmol) in 1,4-dioxane (10 mL) was added Cs<sub>2</sub>CO<sub>3</sub> (650 mg, 2 mmol), BINAP (70 mg, 0.11 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (100 mg, 0.11 mmol), and <i>tert</i>-butyl 3-amino-7-azaspiro[3.5]nonane-7-carboxylate (224 mg, 1 mmol), and the solution was heated to 130 °C for 5 h under microwave. Then, the solution was concentrated and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 1 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford compound <b>56e</b> (200 mg, 33%) as yellow solid. ESI-MS <i>m</i>/<i>z</i> 605.3 [M + H]<sup>+</sup> calcd for C<sub>35</sub>H<sub>48</sub>N<sub>4</sub>O<sub>5</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> <i>tert</i>-Butyl 3-[[2-(5-Ethyl-2-furyl)-6-methoxy-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-7-azaspiro[3.5]nonane-7-carboxylate (<b>56f</b>)</h4><div class="NLM_p last">A mixture of <b>55b</b> (550 mg, 1.33 mmol), <i>tert</i>-butyl 3-amino-7-azaspiro[3.5]nonane-7-carboxylate (637 mg, 2.65 mmol), Cs<sub>2</sub>CO<sub>3</sub> (863 mg, 2.65 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (121 mg, 0.133 mmol), and BINAP (83 mg, 0.133 mmol) in 1,4-dioxane (50 mL) was degassed and purged with N<sub>2</sub> for 3 times, and then the mixture was stirred at 120 °C for 16 h. Then, the mixture was concentrated and purified by silica gel column chromatography (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 200:1–1:1) to afford pure compound <b>56f</b> (200 mg, 24%) as a white solid. ESI-MS <i>m</i>/<i>z</i> 619.4 [M + H]<sup>+</sup> calcd for C<sub>36</sub>H<sub>50</sub>N<sub>4</sub>O<sub>5</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> <i>tert</i>-Butyl 3-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-6-azaspiro[3.4]octane-6-carboxylate (<b>56g</b>)</h4><div class="NLM_p last">To a solution of compound <b>55a</b> (400 mg, 1 mmol) in 1,4-dioxane (10 mL) was added Cs<sub>2</sub>CO<sub>3</sub> (650 mg, 2 mmol), BINAP (70 mg, 0.11 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (100 mg, 0.11 mmol), and <i>tert</i>-butyl 1-amino-6-azaspiro[3.4]octane-6-carboxylate (226 mg, 1 mmol), and the solution was heated to 130 °C for 5 h under microwave. Then, the solution was concentrated and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 1 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>56g</b> (150 mg, 25%) as yellow solid. ESI-MS <i>m</i>/<i>z</i> 591.3 [M + H]<sup>+</sup> calcd for C<sub>34</sub>H<sub>46</sub>N<sub>4</sub>O<sub>5</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> <i>tert</i>-Butyl 8-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-3-azabicyclo[3.2.1]octane-3-carboxylate (<b>56h</b>)</h4><div class="NLM_p last">A mixture of <b>55a</b> (200 mg, 0.5 mmol), <i>tert</i>-butyl 8-amino-3-azabicyclo[3.2.1]octane-3-carboxylate (226 mg, 1.0 mmol), Cs<sub>2</sub>CO<sub>3</sub> (325 mg, 1.0 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (45 mg, 0.05 mmol), and BINAP (31 mg, 0.05 mmol) in 1,4-dioxane (10 mL) was degassed and purged with N<sub>2</sub> for 3 times, and then the mixture was stirred at 120 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was purified by silica gel column chromatography (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 5:1) to obtain pure compound <b>56h</b> (200 mg, 67%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 591.4 [M + H]<sup>+</sup> calcd for C<sub>34</sub>H<sub>46</sub>N<sub>4</sub>O<sub>5</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> <i>tert</i>-Butyl 8-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-3-azabicyclo[3.2.1]octane-3-carboxylate (<b>56i</b>)</h4><div class="NLM_p last">A mixture of <b>55a</b> (300 mg, 0.75 mmol), <i>tert</i>-butyl 8-(aminomethyl)-3-azabicyclo[3.2.1]octane-3-carboxylate (359 mg, 1.50 mmol), Cs<sub>2</sub>CO<sub>3</sub> (487 mg, 1.5 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (137 mg, 0.15 mmol), and BINAP (186 mg, 0.3 mmol) in 1,4-dioxane (10 mL) was degassed and purged with N<sub>2</sub> for 3 times, and then the mixture was stirred at 120 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was purified by prep-TLC (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 5:1) to afford compound <b>56i</b> (300 mg, 66%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 605.4 [M + H]<sup>+</sup> calcd for C<sub>35</sub>H<sub>48</sub>N<sub>4</sub>O<sub>5</sub>.</div></div><div id="sec4_1_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> <i>tert</i>-Butyl 8-[[[2-(5-Ethyl-2-furyl)-6-methoxy-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-3-azabicyclo[3.2.1]octane-3-carboxylate (<b>56j</b>)</h4><div class="NLM_p last">A mixture of <b>55b</b> (300 mg, 0.72 mmol), <i>tert</i>-butyl 8-(aminomethyl)-3-azabicyclo[3.2.1]octane-3-carboxylate (347 mg, 1.45 mmol), Cs<sub>2</sub>CO<sub>3</sub> (471 mg, 1.45 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (132 mg, 0.144 mmol), and BINAP (180 mg, 0.289 mmol) in 1,4-dioxane (10 mL) was degassed and purged with N<sub>2</sub> for 3 times, and then the mixture was stirred at 120 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was purified by TLC (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 5:1) to afford pure compound <b>56j</b> (360 mg, 80%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 619.4 [M + H]<sup>+</sup> calcd for C<sub>36</sub>H<sub>50</sub>N<sub>4</sub>O<sub>5</sub>.</div></div><div id="sec4_1_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> <i>tert</i>-Butyl 3-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>56k</b>)</h4><div class="NLM_p last">To a solution of compound <b>55a</b> (200 mg, 0.5 mmol) in 1,4-dioxane (10 mL) were added Pd<sub>2</sub>(dba)<sub>3</sub> (46 mg, 0.05 mmol), Cs<sub>2</sub>CO<sub>3</sub> (325 mg, 1.0 mmol), <i>tert</i>-butyl 3-amino-8-azabicyclo[3.2.1]octane-8-carboxylate (226 mg, 1.0 mmol), and BINAP (31 mg, 0.05 mmol), and the mixture was stirred at 120 °C for 16 h under N<sub>2</sub>. Then, the mixture was concentrated under reduced pressure and the residue was purified by prep-TLC (CH<sub>2</sub>Cl<sub>2</sub>: MeOH = 10:1) to afford pure compound <b>56k</b> (200 mg, 67%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 591.5 [M + H]<sup>+</sup> calcd for C<sub>36</sub>H<sub>46</sub>N<sub>4</sub>O<sub>5</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> <i>tert</i>-Butyl 4-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]-methylamino]piperidine-1-carboxylate (<b>56l</b>)</h4><div class="NLM_p last">A mixture of compound <b>55a</b> (300 mg, 0.75 mmol), <i>tert</i>-butyl 4-(methylamino)piperidine-1-carboxylate (192 mg, 0.897 mmol), and PTSA (155 mg, 0.9 mmol) in <i>t</i>-BuOH (7.5 mL) was degassed and purged with N<sub>2</sub> for 3 times, and then the mixture was stirred at 120 °C for 48 h. Then, the reaction mixture was concentrated in vacuum to give crude product which was purified by prep-HPLC (method 10 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>56l</b> (40 mg 9%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 579.5 [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>46</sub>N<sub>4</sub>O<sub>5</sub>.</div></div><div id="sec4_1_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> 6-Methoxy-2-(5-methyl-2-furyl)-<i>N</i>-(4-piperidylmethyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>57a</b>)</h4><div class="NLM_p last">A solution of compound <b>56a</b> (125.0 mg, 0.21 mmol) in HCl/EtOAc (2.0 M, 30 mL) was stirred at 16 °C for 8 h. Then, the solution was concentrated to give compound <b>57a</b> (125.0 mg, crude) as a yellow solid which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 479.3 [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>38</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> 6-Methoxy-2-(5-methyl-2-furyl)-<i>N</i>-[1-methyl-1-(4-piperidyl)ethyl]-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>57b</b>)</h4><div class="NLM_p last">A solution of <b>56b</b> (15.00 mg, 0.024 mmol) in HCl/MeOH (2.0 M, 5.00 mL) was degassed and purged with N<sub>2</sub> for 3 times, and the mixture was stirred at 25 °C for 12 h. Then, the solution was concentrated under vacuum to give compound <b>57b</b> (10.0 mg, crude) as a yellow solid which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 507.4 [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>42</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> <i>N</i>-(7-Azaspiro[3.5]nonan-2-yl)-6-methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>57c</b>)</h4><div class="NLM_p last">To a solution of compound <b>56c</b> (60.4 mg, 0.1 mmol) in MeOH (10 mL) was added HCl/MeOH (5 mL, 4.0 M), and the solution was stirred at room temperature for 5 h. Then, the mixture was concentrated to give compound <b>57c</b> (49 mg, crude) as yellow solid which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 505.3 [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>40</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> <i>N</i>-(6-Azaspiro[3.4]octan-2-yl)-6-methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>57d</b>)</h4><div class="NLM_p last">To a solution of compound <b>56d</b> (100 mg, 0.17 mmol) in MeOH (5 mL) was added HCl/MeOH (5 mL, 4.0 M), and the solution was stirred at room temperature for 5 h. Then, the solution was concentrated to give compound <b>57d</b> (90 mg, crude) as a yellow solid which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 491.3 [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>38</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> <i>N</i>-(7-Azaspiro[3.5]nonan-3-yl)-6-methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>57e</b>)</h4><div class="NLM_p last">To a solution of compound <b>56e</b> (60.4 mg, 0.1 mmol) in MeOH (10 mL) was added HCl/MeOH (5 mL, 4.0 M), and the solution was stirred at room temperature for 5 h. Then, the solution was concentrated to give compound <b>57e</b> (49 mg, crude) as yellow solid which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 505.3 [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>40</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_65" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> <i>N</i>-(7-Azaspiro[3.5]nonan-3-yl)-2-(5-ethyl-2-furyl)-6-methoxy-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>57f</b>)</h4><div class="NLM_p last">A solution of <b>56f</b> (80 mg, 0.129 mmol) in HCl/EtOAc (5.00 mL, 1.0 M) was stirred at 20 °C for 16 h. Then, the reaction mixture was concentrated to dryness under vacuum to obtain compound <b>57f</b> (60 mg, crude) as a yellow solid which was used in next step without further purification. ESI-MS <i>m</i>/<i>z</i> 519.4 [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>42</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_66" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> <i>N</i>-(6-Azaspiro[3.4]octan-3-yl)-6-methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>57g</b>)</h4><div class="NLM_p last">To a solution of compound <b>56g</b> (150 mg, 0.25 mmol) in MeOH (10 mL) was added HCl/MeOH (5 mL, 4.0 M), and the solution was stirred at room temperature for 5 h. Then, the solution was concentrated to give compound <b>57g</b> (110 mg, crude) as yellow solid which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 491.3 [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>38</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_67" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> <i>N</i>-(3-Azabicyclo[3.2.1]octan-8-yl)-6-methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>57h</b>)</h4><div class="NLM_p last">A solution of <b>56h</b> (200 mg, 0.338 mmol) in HCl/EtOAc (5 mL, 2.0 M) was stirred at 15 °C for 2 h. Then, the reaction mixture was concentrated in vacuum to give compound <b>57h</b> (150 mg, crude) which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 491.4 [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>38</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_68" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> <i>N</i>-(3-Azabicyclo[3.2.1]octan-8-ylmethyl)-6-methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>57i</b>)</h4><div class="NLM_p last">A solution of <b>56i</b> (300 mg, 0.496 mmol) in HCl/EtOAc (5.00 mL, 2.0 M) was stirred at 18 °C for 2 h. Then, the reaction mixture was concentrated in vacuum to give compound <b>57i</b> (300 mg, crude) as a yellow solid which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 505.4 [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>40</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_69" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> <i>N</i>-(3-Azabicyclo[3.2.1]octan-8-ylmethyl)-2-(5-ethyl-2-furyl)-6-methoxy-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>57j</b>)</h4><div class="NLM_p last">A solution of <b>56j</b> (360 mg, 0.581 mmol) in HCl/EtOAc (5 mL, 2.0 M) was stirred at 18 °C for 2 h. Then, the reaction mixture was concentrated in vacuum to give compound <b>57j</b> (360 mg, crude) as a yellow solid which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 519.4 [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>42</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_70" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> <i>N</i>-(8-Azabicyclo[3.2.1]octan-3-yl)-6-methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>57k</b>)</h4><div class="NLM_p last">A solution of compound <b>56k</b> (200 mg, 0.338 mmol) in HCl/EtOAc (10 mL, 2.0 M) was stirred at 20 °C for 2 h. Then, the mixture was concentrated under vacuum to give compound <b>57k</b> (200 mg, crude) as a yellow solid which was used in next step without further purification. ESI-MS <i>m</i>/<i>z</i> 491.4 [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>38</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_71" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> 6-Methoxy-<i>N</i>-methyl-2-(5-methyl-2-furyl)-<i>N</i>-(4-piperidyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>57l</b>)</h4><div class="NLM_p last"><b>A</b> solution of compound <b>56l</b> (40 mg, 0.0692 mmol) in HCl/EtOAc (4 mL, 2.0 M) was stirred at 20 °C for 1 h. Then, the mixture was concentrated under vacuum to give compound <b>57l</b> (40 mg, crude) as a yellow solid which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 479.4 [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>38</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_72" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> 4,4,5,5-Tetramethyl-2-[5-nitro-2-(trifluoromethoxy)phenyl]-1,3,2-dioxaborolane (<b>59</b>)</h4><div class="NLM_p last">A mixture of commercially available 2-bromo-4-nitro-1-(trifluoromethoxy)benzene (<b>58</b>) (12.20 g, 42.66 mmol), 4,4,5,5-tetramethyl-2-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1,3,2-dioxaborolane (13.00 g, 51.19 mmol), KOAc (9.21 g, 93.85 mmol), and Pd(dppf)Cl<sub>2</sub> (3.12 g, 4.27 mmol) in 1,4-dioxane (200 mL) was degassed and heated to 90 °C for 10 h under N<sub>2</sub>. Then, the reaction was quenched with water and extracted with EtOAc. The organic phase was washed with saturated brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give a residue, which was purified by silica gel column chromatography (PE:EtOAc = 100:1–20:1) to afford the compound <b>59</b> (13.00 g, 91%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 334.1 [M + H]<sup>+</sup> calcd for C<sub>13</sub>H<sub>15</sub>BF<sub>3</sub>NO<sub>5</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_73" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> 5-Nitro-2-(trifluoromethoxy)phenol (<b>60c</b>)</h4><div class="NLM_p last">A Schenk flask was charged with compound <b>59</b> (17.10 g, 52.2 mmol) and NaOH (12.53 g, 313.2 mmol), and the mixture was dissolved in THF (200 mL). Then, an aqueous solution of H<sub>2</sub>O<sub>2</sub> (33%, 35.51 g, 313.2 mmol) was added dropwise under N<sub>2</sub>, and the reaction mixture was stirred for 30 min at 22 °C. Then, the mixture was extracted with EtOAc. The combined organic layer was washed with water and brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and filtered. The solvents were removed under reduced pressure, and the crude product was purified by silica gel column chromatography (PE:EtOAc = 9:1–3:1) to afford compound <b>60c</b> (10.40 g, 89%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 224.1 [M + H]<sup>+</sup> calcd for C<sub>7</sub>H<sub>4</sub>F<sub>3</sub>NO<sub>4</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_74" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> 1-[3-(3-Nitrophenoxy)propyl]pyrrolidine (<b>61a</b>)</h4><div class="NLM_p last">To a solution of commercially available 3-nitrophenol (<b>60a</b>) (4.9 g, 0.035 mol) in THF (500 mL) was added PPh<sub>3</sub> (11.1 g, 0.042 mol), 3-pyrrolidin-1-ylpropan-1-ol (5 g, 0.038 mol), and DEAD (7.3 g. 0.042 mol) at 0 °C, and the solution was stirred at room temperature for 10 h. Then, the reaction mixture was concentrated and extracted with EtOAc. The combined organic layers were washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by silica gel column chromatography to afford compound <b>61a</b> (3.3 g, 37%) as yellow solid. ESI-MS <i>m</i>/<i>z</i> 251 [M + H]<sup>+</sup> calcd for C<sub>13</sub>H<sub>18</sub>N<sub>2</sub>O<sub>3</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_75" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> 1-[3-(2-Chloro-5-nitro-phenoxy)propyl]pyrrolidine (<b>61b</b>)</h4><div class="NLM_p last">To a solution of commercially available 2-chloro-5-nitrophenol (<b>60b</b>) (20 g, 0.12 mol) in THF (1000 mL) was added PPh<sub>3</sub> (60.45 g, 0.23 mol), 3-pyrrolidin-1-ylpropan-1-ol (22.33 g, 0.17 mol), and DEAD (40.14 g. 0.23 mmol) at 0 °C, and the solution was stirred at room temperature for 5 h. Then, the reaction mixture was concentrated and extracted with EtOAc. The combined organic layers were washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by silica gel column chromatography to afford pure compound <b>61b</b> (20 g, 58%) as yellow solid. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.90 (d, <i>J</i> = 2.4 Hz, 1H), 7.85 (s, 1H), 7.68 (d, <i>J</i> = 2.4 Hz, 1H), 4.28–4.20 (m, 2H), 2.76–2.70 (m, 2H), 2.61–2.45 (m, 4H), 2.13–2.06 (m, 2H), 1.85–1.75 (m, 4H). ESI-MS <i>m</i>/<i>z</i> 285 [M + H]<sup>+</sup> calcd for C<sub>13</sub>H<sub>17</sub>ClN<sub>2</sub>O<sub>3</sub>.</div></div><div id="sec4_1_76" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> 1-[3-[5-Nitro-2-(trifluoromethoxy)phenoxy]propyl]pyrrolidine (<b>61c</b>)</h4><div class="NLM_p last">To a mixture of compound <b>60c</b> (9.20 g, 41.24 mmol), 3-pyrrolidin-1-ylpropan-1-ol (6.39 g, 49.48 mmol), and PPh<sub>3</sub> (21.63 g, 82.47 mmol) in THF (350 mL) was added DIAD (16.68 g, 82.47 mmol) in one portion at 0 °C under N<sub>2</sub>, and the mixture was stirred at 0 °C for 8 h. Then, the mixture was quenched with water and extracted with EtOAc. The combined organic phase was washed with saturated brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum. The residue was purified by silica gel chromatography (PE:EtOAc = 1:3) to afford pure compound <b>61c</b> (8.60 g, 62%) as a yellow liquid. ESI-MS <i>m</i>/<i>z</i> 335.1 [M + H]<sup>+</sup> calcd for C<sub>14</sub>H<sub>17</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_77" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> 3-(3-Pyrrolidin-1-ylpropoxy)aniline (<b>62a</b>)</h4><div class="NLM_p last">To a solution of compound <b>61a</b> (3.3 g, 0.013 mol) in MeOH (20 mL) was added Pd/C (1.5 g), and the solution was stirred at room temperature for 2 h under H<sub>2</sub> atmosphere (40 psi). Then, the solution was filtered, and the filtrate was concentrated to give compound <b>62a</b> (3 g, 96%) as yellow oil. ESI-MS <i>m</i>/<i>z</i> 221 [M + H]<sup>+</sup> calcd for C<sub>13</sub>H<sub>20</sub>N<sub>2</sub>O. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_78" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> 4-Chloro-3-(3-pyrrolidin-1-ylpropoxy)aniline (<b>62b</b>)</h4><div class="NLM_p last">To a solution of compound <b>61b</b> (5 g, 17.56 mmol) in EtOH/H<sub>2</sub>O (4:1, 250 mL) was added Fe (2.94 g, 56.28 mmol) and NH<sub>4</sub>Cl (2.82 g, 56.28 mmol) slowly, and the mixture was stirred at room temperature for 3 h under N<sub>2</sub> atmosphere. Then, the solution was filtered and the filtrate was concentrated to give compound <b>62b</b> (2 g, 44%) as yellow solid. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.36 (d, <i>J</i> = 8.4 Hz, 1H), 6.88 (d, <i>J</i> = 2.8 Hz, 1H), 6.76–6.73 (m, 1H), 4.22–4.18 (m, 2H), 3.81–3.72 (m, 2H), 3.48–3.42 (m, 2H), 3.19–3.08 (m, 2H), 2.34–2.26 (m, 2H), 2.24–2.13 (m, 2H), 2.14–1.98 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 255 [M + H]<sup>+</sup> calcd for C<sub>13</sub>H<sub>19</sub>ClN<sub>2</sub>O.</div></div><div id="sec4_1_79" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> 3-(3-Pyrrolidin-1-ylpropoxy)-4-(trifluoromethoxy)aniline (<b>62c</b>)</h4><div class="NLM_p last">To a solution of compound <b>61c</b> (8.60 g, 25.73 mmol) in MeOH (100 mL) was added Pd/C (1.50 g), and the mixture was stirred at room temperature for 15 h under H<sub>2</sub> atmosphere (40 Psi). Then, the mixture was filtered, and the filtrate was concentrated to give compound <b>62c</b> (7.80 g, 99%) as a gray liquid which was used for the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 305.0 [M + H]<sup>+</sup> calcd for C<sub>14</sub>H<sub>19</sub>F<sub>3</sub>N<sub>2</sub>O<sub>2</sub>.</div></div><div id="sec4_1_80" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> 2,4-Dichloro-7-(3-pyrrolidin-1-ylpropoxy)quinoline (<b>63a</b>)</h4><div class="NLM_p last">To a solution of compound <b>62a</b> (3 g, 0.013 mol) in POCl<sub>3</sub> (40 mL) was added malonic acid (2.1 g, 0.020 mol), and the mixture was stirred at room temperature for 4 h and then at 90 °C overnight. Then, the solution was concentrated and extracted with EtOAc. The combined organic layers were washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give compound <b>63a</b> (1.5 g, 34%) as pale-yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.09 (d, <i>J</i> = 8.8 Hz, 1H), 7.76 (s, 1H), 7.44–7.40 (m, 2H) 4.29–4.26 (m, 2H), 3.55–3.49 (m, 2H), 3.34–3.25 (m, 2H), 3.10–2.91 (m, 2H), 2.26–2.22 (m, 2H), 2.00–1.89 (m, 4H). ESI-MS <i>m</i>/<i>z</i> 325 [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>18</sub>Cl<sub>2</sub>N<sub>2</sub>O.</div></div><div id="sec4_1_81" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> 2,4,6-Trichloro-7-(3-pyrrolidin-1-ylpropoxy)quinoline (<b>63b</b>)</h4><div class="NLM_p last">To a solution of compound <b>62b</b> (3 g, 11.78 mmol) in POCl<sub>3</sub> (30 mL) was added malonic acid (1.84 g, 17.86 mmol), and the mixture was stirred at room temperature for 4 h and at 90 °C overnight. Then, the solution was concentrated and extracted with EtOAc. The combined organic layers were washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give compound <b>63b</b> (1 g, 24%) as pale-yellow solid. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.29 (s, 1H), 7.66 (s, 1H), 7.52 (s, 1H), 4.48–4.37 (m, 2H), 3.85–3.74 (m, 2H), 3.56–3.45 (m, 2H), 3.26–3.13 (m, 2H), 2.48–2.37 (m, 2H), 2.27–2.17 (m, 2H), 2.16–2.02 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 359 [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>17</sub>Cl<sub>3</sub>N<sub>2</sub>O.</div></div><div id="sec4_1_82" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> 2,4-Dichloro-7-(3-pyrrolidin-1-ylpropoxy)-6-(trifluoromethoxy)quinoline (<b>63c</b>)</h4><div class="NLM_p last">DMAP (129 mg, 1.06 mmol) and pyridine (5.86 g, 74.07 mmol) were added to a stirred suspension of compound <b>62c</b> (9.20 g, 30.16 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (60 mL) at −70 °C over a period of 10 min under nitrogen. The resulting solution was stirred at −70 °C for 1 h, and then ethyl 3-chloro-3-oxo-propanoate (3.35 g, 22.22 mmol) was added over a period of 10 min under nitrogen. The resulting solution was stirred at 20 °C for 15 h. Then, the reaction was quenched with water then extracted with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic phase was washed with saturated brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum. The residue was purified by silica gel column chromatography (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 10:1) to afford pure intermediate ethyl 3-oxo-3-[3-(3-pyrrolidin-1-ylpropoxy)-4-(trifluoromethoxy)anilino]propanoate (5.40 g, 42%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 419.0 [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>25</sub>F<sub>3</sub>N<sub>2</sub>O<sub>5</sub>. To a solution of this intermediate (5.40 g, 12.91 mmol) in THF/MeOH/H<sub>2</sub>O (4:2:3, 45 mL) was added LiOH·H<sub>2</sub>O (1.63 g, 38.73 mmol), and the resulting mixture was stirred at 20 °C for 15 h. Then, the mixture was diluted with water and adjusted pH to 4–5 with 1.0 N HCl. The solution was extracted with EtOAc, and the combined organic layer was washed with brine and concentrated to afford the desired intermediate 3-oxo-3-[3-(3-pyrrolidin-1-ylpropoxy)-4-(trifluoromethoxy)anilino]propanoic acid (4.89 g, 97%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 391.1 [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>21</sub>F<sub>3</sub>N<sub>2</sub>O<sub>5</sub>. A solution of this intermediate (4.80 g, 12.30 mmol) in POCl<sub>3</sub> (60 mL) was then stirred at 100 °C for 3 h. Then, the solution was concentrated to give the crude compound which was purified by silica gel column chromatography (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 10:1) to afford pure compound <b>63c</b> (2.10 g, 41%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 409.1 [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>17</sub>Cl<sub>2</sub>F<sub>3</sub>N<sub>2</sub>O<sub>2</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_83" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> 4-Chloro-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinoline (<b>64a</b>)</h4><div class="NLM_p last">To a solution of compound <b>63a</b> (400 mg, 1.23 mmol) in 1,4-dioxane/H<sub>2</sub>O (5:1, 18 mL) was added Na<sub>2</sub>CO<sub>3</sub> (196 mg, 1.8 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (138 mg, 0.12 mmol), and 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane (288 mg, 1.36 mmol), and the solution was heated at 110 °C for 2 h under microwave. Then, the mixture was concentrated and extracted with EtOAc. The combined organic layers were washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-TLC to afford compound <b>64a</b> (0.4 g, 88%) as pale yellow solid. ESI-MS <i>m</i>/<i>z</i> 371 [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>23</sub>ClN<sub>2</sub>O<sub>2</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_84" class="NLM_sec NLM_sec_level_3"><div id="ac_i110" class="anchor-spacer"></div><h4 class="article-section__title" id="_i110"> 4,6-Dichloro-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)quinoline (<b>64b</b>)</h4><div class="NLM_p last">To a solution of compound <b>63b</b> (180 mg, 0.5 mmol) in 1,4-dioxane/H<sub>2</sub>O (15:1, 16 mL) was added K<sub>2</sub>CO<sub>3</sub> (0.138 g, 1 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (100.1 mg, 0.1 mmol), and 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane (104 mg, 1 mmol) under N<sub>2</sub>, and the mixture was stirred at 110 °C for 12 h. The mixture was quenched with water and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by prep-HPLC (method 2 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford compound <b>64b</b> (100 mg, 49%) as a yellow solid. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.23 (s, 1H), 7.92 (s, 1H), 7.58 (s, 1H), 7.27–7.25 (m, 1H), 6.31–6.29 (m, 1H), 4.43–4.37 (m, 2H), 3.80–3.74 (m, 2H), 3.55–3.49 (m, 2H), 3.23–3.09 (m, 2H), 2.46–2.33 (m, 5H), 2.25–2.16 (m, 2H), 2.13–2.02 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 405 [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>22</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>2</sub>.</div></div><div id="sec4_1_85" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> 4-Chloro-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-6-(trifluoromethoxy)quinoline (<b>64c</b>)</h4><div class="NLM_p last">To a solution of compound <b>63c</b> (1.50 g, 3.67 mmol) in 1,4-dioxane/H<sub>2</sub>O (10:1, 55 mL) were successively added 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane (800 mg, 3.85 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (423 mg, 0.366 mmol), and K<sub>2</sub>CO<sub>3</sub> (1.27 g, 9.16 mmol), and the resulting mixture was stirred at 80 °C for 15 h under N<sub>2</sub>. Then, the reaction mixture was diluted with water and extracted with EtOAc. The organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to obtain the crude product which was purified by silica gel column chromatography (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 10:1) to afford pure compound <b>64c</b> (1.36 g, 81%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 455.2 [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>22</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>3</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_86" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> [2-(5-Methyl-2-furyl)-4-[(1-methyl-4-piperidyl)amino]-7-(3-pyrrolidin-1-ylpropoxy)-6-quinolyl] trifluoromethanesulfonate (<b>65</b>)</h4><div class="NLM_p last">To a solution of compound <b>30</b> (232 mg, 0.5 mmol) in DMF (5 mL) was added DIEA (202 mg, 1.56 mmol) and PhN(OTf)<sub>2</sub> (270 mg, 0.75 mmol) at 0 °C, and the solution was stirred at 0 °C for 2 h and at 25 °C for 12 h. Then, the solution was concentrated to give the crude product which was purified by prep-HPLC (method 3 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>65</b> (180 mg, 60%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 597.3 [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>35</sub>F<sub>3</sub>N<sub>4</sub>O<sub>5</sub>S. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_87" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> 2,4-Dichloro-6-methoxy-quinolin-7-ol (<b>67</b>)</h4><div class="NLM_p last">To a mixture of commercially available 5-amino-2-methoxy-phenol (<b>66</b>) (4.91 g, 35.29 mmol) and malonic acid (7.34 g, 70.57 mmol) was added POCl<sub>3</sub> (70 mL), and the mixture was stirred at 95 °C for 12 h under N<sub>2</sub>. Then, the mixture was concentrated in reduced pressure to remove POCl<sub>3</sub>. The residue was poured into water and stirred for 20 min. The aqueous phase was extracted with EtOAc, and the combined organic phase was washed with saturated brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum. The residue was purified by silica gel column chromatography (PE/EtOAc = 2/1) to afford compound <b>67</b> (2.10 g, 247%). ESI-MS <i>m</i>/<i>z</i> 244.0 [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>27</sub>ClN<sub>2</sub>O<sub>3</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_88" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> 3-[(2,4-Dichloro-6-methoxy-7-quinolyl)oxy]-<i>N</i>,<i>N</i>-dimethyl-propan-1-amine (<b>68a</b>)</h4><div class="NLM_p last">To a solution of <b>67</b> (3 g, 12.29 mmol) and 1,3-dibromopropane (2.98 g, 14.75 mmol) in CH<sub>3</sub>CN (60 mL) was added K<sub>2</sub>CO<sub>3</sub> (4.25 g, 30.73 mmol), and the mixture was stirred at 60 °C for 88 h. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was diluted with water and filtered. The filter cake was dried in vacuum and further purified by silica gel column chromatography (from PE:EtOAc (30:1) to pure MeOH) to afford intermediate 7-(3-bromopropoxy)-2,4-dichloro-6-methoxy-quinoline (1 g, 22%) as a white solid. ESI-MS <i>m</i>/<i>z</i> 364.1 [M + H]<sup>+</sup> calcd for C<sub>13</sub>H<sub>12</sub>BrCl<sub>2</sub>NO<sub>2</sub>. To a solution of this intermediate (200 mg, 0.550 mmol) and <i>N</i>-methylmethanamine hydrochloride (67 mg, 0.821 mmol) in CH<sub>3</sub>CN (20 mL) was added K<sub>2</sub>CO<sub>3</sub> (189 mg, 1.37 mmol), and the mixture was stirred at 60 °C for 16 h. Then, mixture was concentrated in vacuum to give a residue. The residue was diluted with water and extracted with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give compound <b>68a</b> (150 mg, 83%) as a white solid which was used in next step without further purification. ESI-MS <i>m</i>/<i>z</i> 329.2 [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>18</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>2</sub>.</div></div><div id="sec4_1_89" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> 7-[3-(5-Azaspiro[2.4]heptan-5-yl)propoxy]-2,4-dichloro-6-methoxy-quinoline (<b>68b</b>)</h4><div class="NLM_p last">To a solution of 7-(3-bromopropoxy)-2,4-dichloro-6-methoxy-quinoline (intermediate described in the synthesis of <b>68a</b>, 598 mg, 1.64 mmol) and 5-azaspiro[2.4]heptane hydrochloride (328 mg, 2.46 mmol) in CH<sub>3</sub>CN (20 mL) was added K<sub>2</sub>CO<sub>3</sub> (680 mg, 4.92 mmol), and the mixture was stirred at 60 °C for 16 h. Then, the mixture was concentrated in vacuum to give a residue. The residue was dissolved in CH<sub>2</sub>Cl<sub>2</sub> and filtered through a Celite pad. The filtrate was concentrated in vacuum and the residue was purified by prep-HPLC (method 3 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford compound <b>68b</b> (150 mg, 24%) as a purple solid. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.56 (s, 1H), 7.46 (s, 1H), 7.33 (s, 1H), 4.35–4.32 (m, 2H), 4.04 (s, 3H), 3.97–3.95 (m, 1H), 3.54–3.51 (m, 3H), 3.43–3.40 (m, 1H), 3.30–3.29 (m, 1H), 2.40–2.35 (m, 2H), 2.22–2.20 (m, 1H), 2.05–2.02 (m, 1H), 0.86–0.76 (m, 4H). ESI-MS <i>m</i>/<i>z</i> 381.2 [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>22</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>2</sub>.</div></div><div id="sec4_1_90" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> 2,4-Dichloro-7-[3-(3-fluoropyrrolidin-1-yl)propoxy]-6-methoxy-quinoline (<b>68c</b>)</h4><div class="NLM_p last">A mixture of <b>67</b> (829 mg, 3.40 mmol), 3-(3-fluoropyrrolidin-1-yl)propan-1-ol (500 mg, 3.40 mmol), and PPh<sub>3</sub> (1.16 g, 4.42 mmolq) in THF (30.0 mL) was degassed and purged with N<sub>2</sub> for 3 times. Then, DEAD (769 mg, 4.42 mmol) was added at 0 °C under N<sub>2</sub>, and the mixture was stirred at 20 °C for 16 h. Then, reaction was quenched with water, and the mixture was concentrated to give a residue. The residue was extracted with EtOAc, and the combined organic phase was washed with brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give the crude product. The crude product was purified by silica gel column chromatography (PE/EtOAc = 100:1–0:1, CH<sub>2</sub>Cl<sub>2</sub>/methanol = 1/0–10:1) to afford pure compound <b>68c</b> (700 mg, 55%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 373.1 [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>19</sub>Cl<sub>2</sub>FN<sub>2</sub>O<sub>2</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_91" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> 2-(3-Bromopropoxy)-1-methoxy-4-nitro-benzene (<b>69</b>)</h4><div class="NLM_p last">A mixture of commercially available 2-methoxy-5-nitro-phenol (<b>51</b>) (30.63 g, 181.10 mmol), 1,3-dibromopropane (47.53 g, 235.43 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (118.01 g, 362.20 mmol) in DMF (500 mL) was degassed and purged with N<sub>2</sub> for 3 times, and the mixture was stirred at 20 °C for 16 h. Then, the mixture was poured into water and extracted with MTBE. The combined organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give a residue. The residue was purified by silica gel column chromatography (PE:EtOAc = 40:1–5:1) to afford pure compound <b>69</b> (20 g, 38%) as a black solid. ESI-MS <i>m</i>/<i>z</i> 290.1 [M + H]<sup>+</sup> calcd for C<sub>10</sub>H<sub>12</sub>BrNO<sub>4</sub>. This intermediate was used in the next step without further characterization.</div></div><div id="sec4_1_92" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118"> 4-Methoxy-3-[3-(1-piperidyl)propoxy]aniline (<b>70a</b>)</h4><div class="NLM_p last">To a solution of <b>69</b> (10 g, 34.47 mmol) and piperidine (4.4 g, 51.71 mmol) in CH<sub>3</sub>CN (100 mL) was added Cs<sub>2</sub>CO<sub>3</sub> (22.46 g, 68.94 mmol), and the mixture was stirred at 90 °C for 16 h. Then, the reaction mixture was concentrated in vacuum and extracted with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give a residue. The residue was purified by silica gel column chromatography (PE:EtOAc = 20:1, pure EtOAc) to afford pure intermediate 1-[3-(2-methoxy-5-nitro-phenoxy)propyl]piperidine (6.3 g, 62%) as a white solid. ESI-MS <i>m</i>/<i>z</i> 295.3 [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>22</sub>N<sub>2</sub>O<sub>4</sub>. Then, to a solution of this intermediate (6.3 g, 21.40 mmol) in MeOH (150 mL) was added Pd/C (10%, 1.3 g) and the suspension was degassed and purged with H<sub>2</sub> for 3 times. The mixture was stirred under H<sub>2</sub> (30 Psi) at 15 °C for 2 h. Then, the reaction mixture was filtered through a Celite pad, and the filtrate was concentrated in vacuum to give compound <b>70a</b> (4.8 g, 84%) as a brown solid which was used in the next step without further purification. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 6.72–6.68 (m, 1H), 6.33 (d, <i>J</i> = 2.4 Hz, 1H), 6.22 (dd, <i>J</i> = 2.4 Hz, 8.4 Hz, 1H), 4.03–4.00 (m, 2H), 3.78 (s, 3H), 3.40 (br s, 2H), 2.50–2.46 (m, 2H), 2.40 (br s, 4H), 2.03–2.00 (m, 2H), 1.62–1.56 (m, 4H), 1.45–1.42 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 265.3 [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>24</sub>N<sub>2</sub>O<sub>2</sub>.</div></div><div id="sec4_1_93" class="NLM_sec NLM_sec_level_3"><div id="ac_i119" class="anchor-spacer"></div><h4 class="article-section__title" id="_i119"> 4-Methoxy-3-(3-morpholinopropoxy)aniline (<b>70b</b>)</h4><div class="NLM_p last">To a solution of <b>69</b> (10 g, 34.47 mmol) and morpholine (4.5 g, 51.71 mmol) in CH<sub>3</sub>CN (100 mL) was added Cs<sub>2</sub>CO<sub>3</sub> (22.46 g, 68.94 mmol), and the mixture was stirred at 90 °C for 16 h. Then, the reaction mixture was concentrated in vacuum and extracted with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give a residue which was purified by silica gel column chromatography (PE:EtOAc = 20:1, pure EtOAc) to afford intermediate 4-[3-(2-methoxy-5-nitro-phenoxy)propyl]morpholine (5 g, 49%) as a white solid. ESI-MS <i>m</i>/<i>z</i> 297.3 [M + H]<sup>+</sup> calcd for C<sub>14</sub>H<sub>20</sub>N<sub>2</sub>O<sub>5</sub>. Then, to a solution of this intermediate (5.0 g, 16.87 mmol) in MeOH (150 mL) was added Pd/C (10%, 1 g) under N<sub>2</sub> atmosphere. The suspension was degassed and purged with H<sub>2</sub> for 3 times, and then the mixture was stirred under H<sub>2</sub> (30 psi) at 15 °C for 2 h. Then, the reaction mixture was filtered through a Celite pad, and the filtrate was concentrated in vacuum to give compound <b>70b</b> (4.4 g, 98%) as a yellow solid which was used in the next step without further purification. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 6.71 (d, <i>J</i> = 8.4 Hz, 1H), 6.33 (d, <i>J</i> = 2.4 Hz, 1H), 6.23 (dd, <i>J</i> = 2.8 Hz, 8.4 Hz, 1H), 4.05–4.01 (m, 2H), 3.78 (s, 3H), 3.72–3.70 (m, 4H), 3.45 (br s, 2H), 2.54–2.50 (m, 2H), 2.46 (br s, 4H), 2.04–1.99 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 267.3 [M + H]<sup>+</sup> calcd for C<sub>14</sub>H<sub>22</sub>N<sub>2</sub>O<sub>3</sub>.</div></div><div id="sec4_1_94" class="NLM_sec NLM_sec_level_3"><div id="ac_i120" class="anchor-spacer"></div><h4 class="article-section__title" id="_i120"> 2,4-Dichloro-6-methoxy-7-[3-(1-piperidyl)propoxy]quinoline (<b>71a</b>)</h4><div class="NLM_p last">To a solution of <b>70a</b> (4.6 g, 17.40 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (50 mL) were added pyridine (4.82 g, 60.90 mmol) and DMAP (212 mg, 1.74 mmol) at −78 °C. Then ethyl 3-chloro-3-oxo-propanoate (3.14 g, 20.88 mmol) was added dropwise into above solution, and the mixture was stirred at 15 °C for 16 h. Then, the mixture was poured into water and extracted with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give intermediate ethyl 3-[4-methoxy-3-[3-(1-piperidyl)propoxy]anilino]-3-oxo-propanoate (2.3 g, 34%) as a black solid. ESI-MS <i>m</i>/<i>z</i> 379.3 [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>30</sub>N<sub>2</sub>O<sub>5</sub>. To a solution of this intermediate (2.3 g, 6.08 mmol) in THF/MeOH/H<sub>2</sub>O (20:20:13, 53 mL) was added LiOH.H<sub>2</sub>O (510 mg, 12.16 mmol), and the mixture was stirred at 15 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give intermediate 3-[4-methoxy-3-[3-(1-piperidyl)propoxy]anilino]-3-oxo-propanoic acid (2 g, 93%) as a black solid. ESI-MS <i>m</i>/<i>z</i> 351.3 [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>26</sub>N<sub>2</sub>O<sub>5</sub>. Finally, a solution of this intermediate (2 g, 5.71 mmol) in POCl<sub>3</sub> (8.75 g, 57.08 mmol) was stirred at 100 °C for 2 h. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was dissolved in EtOAc and poured slowly into water. The resulting mixture was adjust to pH = 8 with NaOH (2.0 M) and then extracted with EtOAc. The combined organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give a residue which was purified by silica gel column chromatography (PE:EtOAc = 1:1 to CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 10:1) to afford pure compound <b>71a</b> (1.2 g, 56%) as a brown solid. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.57 (s, 1H), 7.48 (s, 1H), 7.35 (s, 1H), 4.34–4.32 (m, 2H), 4.03 (s, 3H), 3.70–3.67 (m, 2H), 3.41–3.37 (m, 2H), 3.04–2.98 (m, 2H), 2.40–2.35 (m, 2H), 2.03–1.99 (m, 2H), 2.87–2.78 (m, 3H), 1.59–1.55 (m, 1H). ESI-MS <i>m</i>/<i>z</i> 369.2 [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>22</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>2</sub>.</div></div><div id="sec4_1_95" class="NLM_sec NLM_sec_level_3"><div id="ac_i121" class="anchor-spacer"></div><h4 class="article-section__title" id="_i121"> 4-[3-[(2,4-Dichloro-6-methoxy-7-quinolyl)oxy]propyl]morpholine (<b>71b</b>)</h4><div class="NLM_p last">To a solution of <b>70b</b> (4.3 g, 16.14 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10.0 mL) were added pyridine (4.47 g, 56.51 mmol) and DMAP (197 mg, 1.61 mmol) at −78 °C, and then ethyl 3-chloro-3-oxo-propanoate (2.92 g, 19.37 mmol) was added dropwise into above solution. The reaction mixture was stirred at 15 °C for 16 h. Then, the mixture was poured into water and extracted with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give intermediate ethyl 3-[4-methoxy-3-(3-morpholinopropoxy)anilino]-3-oxo-propanoate (6 g, 98%) as a brown oil. ESI-MS <i>m</i>/<i>z</i> 381.3 [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>28</sub>N<sub>2</sub>O<sub>6</sub>. To a solution of this intermediate (6 g, 15.77 mmol) in THF/MeOH/H<sub>2</sub>O (3:3:2, 80 mL) was added LiOH·H<sub>2</sub>O (1.32 g, 31.54 mmol), and the mixture was stirred at 15 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give intermediate 3-[4-methoxy-3-(3-morpholinopropoxy)anilino]-3-oxo-propanoic acid (5 g, 90%) as a brown solid. ESI-MS <i>m</i>/<i>z</i> 353.3 [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>24</sub>N<sub>2</sub>O<sub>6</sub>. Finally, a solution of this intermediate (5 g, 14.19 mmol) in POCl<sub>3</sub> (21.76 g, 141.89 mmol) was stirred at 100 °C for 2 h. Then, the reaction mixture was concentrated in vacuum to give a residue which was dissolved in EtOAc and poured into water slowly. The mixture was adjust to pH = 8 with NaOH (2.0 M) and then extracted with EtOAc. The combined organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give a residue which was purified by silica gel column chromatography (PE:EtOAc = 1:1 to CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 10:1) to afford pure compound <b>71b</b> (3.8 g, 72%) as a white solid. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.60 (s, 1H), 7.48 (s, 1H), 7.36 (s, 1H), 4.36–4.35 (m, 2H), 4.13–4.10 (m, 2H), 4.03 (s, 3H), 3.71–3.64 (m, 4H), 3.48–3.46 (m, 2H), 3.26–3.23 (m, 2H), 2.43 (br s, 2H). ESI-MS <i>m</i>/<i>z</i> 371.2 [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>3</sub>.</div></div><div id="sec4_1_96" class="NLM_sec NLM_sec_level_3"><div id="ac_i122" class="anchor-spacer"></div><h4 class="article-section__title" id="_i122"> 2,4-Dichloro-6,7-dimethoxy-quinoline (<b>73a</b>)</h4><div class="NLM_p last">To a solution of commercially available 3,4-dimethoxyaniline (<b>72a</b>) (8 g, 52 mmol) in POCl<sub>3</sub> (100 mL) was added malonic acid (6.69 g, 64 mmol), and the solution was heated for 4 h at room temperature and overnight at 90 °C. Then, the mixture was concentrated and extracted with EtOAc. The combined organic layers were washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by the silica gel column chromatography to afford pure compound <b>73a</b> (8 g, 60%) as a pale-yellow solid. ESI-MS <i>m</i>/<i>z</i> 258 [M + H]<sup>+</sup> calcd for C<sub>11</sub>H<sub>9</sub>Cl<sub>2</sub>NO<sub>2</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_97" class="NLM_sec NLM_sec_level_3"><div id="ac_i123" class="anchor-spacer"></div><h4 class="article-section__title" id="_i123"> 7-Benzyloxy-2,4-dichloro-6-methoxy-quinoline (<b>73b</b>)</h4><div class="NLM_p last">To a mixture of commercially available 3-benzyloxy-4-methoxy-aniline (<b>72b</b>) (35 g, 0.153 mol) and TEA (30.87 g, 0.306 mol) in CH<sub>2</sub>Cl<sub>2</sub> (1 L) was added dropwise ethyl 3-chloro-3-oxo-propanoate (25.25 g, 0.168 mol) at 0 °C, and the mixture was stirred at 25 °C for 12 h. Then, the reaction was poured into water and extracted with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic phase was dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to afford intermediate ethyl 3-(3-benzyloxy-4-methoxy-anilino)-3-oxo-propanoate (40 g, 76%) which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 344.2 [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>21</sub>NO<sub>5</sub>. To a mixture of this intermediate (20.40 g, 59.41 mmol) in THF/MeOH/H<sub>2</sub>O (3:3:2, 267 mL) was added LiOH·H<sub>2</sub>O (3.74 g, 89.12 mmol), and the mixture was stirred at 25 °C for 16 h. Then, the solvent was removed and the residue was poured into ice–water. The resulting slurry was filtered and the filter cake was dried under vacuum to afford intermediate 3-(3-benzyloxy-4-methoxy-anilino)-3-oxo-propanoic acid (19.30 g, crude) as a white solid. ESI-MS <i>m</i>/<i>z</i> 316.2 [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>17</sub>NO<sub>5</sub>. Finally, this intermediate (7.00 g, 22.20 mmol) was suspended in POCl<sub>3</sub> (68.08 g, 444 mmol) in a 500 mL single-necked round-bottom flask, and the mixture was stirred at 90 °C for 2 h under N<sub>2</sub>. The reaction mixture was cooled and concentrated to remove POCl<sub>3</sub>. The residue was further purified by silica gel column chromatography (PE:EtOAc = 50:1–10:1) to give afford pure <b>73c</b> (2.50 g, 33%). ESI-MS <i>m</i>/<i>z</i> 334.2 [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>13</sub>Cl<sub>2</sub>NO<sub>2</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_98" class="NLM_sec NLM_sec_level_3"><div id="ac_i124" class="anchor-spacer"></div><h4 class="article-section__title" id="_i124"> 4-Chloro-6,7-dimethoxy-2-(5-methyl-2-furyl)quinoline (<b>74a</b>)</h4><div class="NLM_p last">To a solution of compound <b>73a</b> (150 mg, 0.58 mmol) in 1,4-dioxane/H<sub>2</sub>O (15:1, 16 mL) was added Na<sub>2</sub>CO<sub>3</sub> (189 mg, 1.75 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (75 mg, 0.064 mmol), and 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane (120 mg, 0.58 mmol), and the mixture was stirred at 110 °C for 2 h under microwave. Then, the mixture was quenched with water and extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the crude product which was purified by the prep-TLC to afford pure compound <b>74a</b> (70 mg, 40%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 304 [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>14</sub>ClNO<sub>3</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_99" class="NLM_sec NLM_sec_level_3"><div id="ac_i125" class="anchor-spacer"></div><h4 class="article-section__title" id="_i125"> 4-Chloro-6-methoxy-2-(5-methyl-2-furyl)quinolin-7-ol (<b>74b</b>)</h4><div class="NLM_p last">To a mixture of <b>67</b> (6.00 g, 24.58 mmol), 4,4,5,5-tetramethyl −2-(5-methyl-2-furyl)-1,3,2-dioxaborolane (5.63 g, 27.04 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (2.86 g, 2.46 mmol) in 1,4-dioxane (90 mL) was added K<sub>2</sub>CO<sub>3</sub> (3.40 g, 24.64 mmol) in H<sub>2</sub>O (30 mL), and the mixture was stirred at 120 °C for 12 h. Then, the mixture was extracted with EtOAc, and the combined organic phase was washed with saturated brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum. The residue was purified by silica gel column chromatography (PE/EtOAc = 2:1) to afford pure compound <b>74b</b> (2.10 g, 30%) as yellow solid. ESI-MS <i>m</i>/<i>z</i> 290.1 [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>12</sub>ClNO<sub>3</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_100" class="NLM_sec NLM_sec_level_3"><div id="ac_i126" class="anchor-spacer"></div><h4 class="article-section__title" id="_i126"> 7-Benzyloxy-4-chloro-6-methoxy-2-(5-methyl-2-furyl)quinoline (<b>74c</b>)</h4><div class="NLM_p last">A mixture of compound <b>73b</b> (900 mg, 2.69 mmol), 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane (588 mg, 2.83 mmol), K<sub>2</sub>CO<sub>3</sub> (558 mg, 4.04 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (311 mg, 0.27 mmol) in 1,4-dioxane (10 mL) was degassed and heated to 100 °C for 16 h under N<sub>2</sub>. Then, the reaction mixture was poured into water and extracted with EtOAc. The combined organic phase was washed with saturated brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give a residue, which was purified by silica gel column chromatography (PE:EtOAc = 30:1–5:1) to afford the pure compound <b>74c</b> (500 mg, 49%). ESI-MS <i>m</i>/<i>z</i> 380.1 [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>18</sub>ClNO<sub>3</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_101" class="NLM_sec NLM_sec_level_3"><div id="ac_i127" class="anchor-spacer"></div><h4 class="article-section__title" id="_i127"> 6-Methoxy-2-(5-methyl-2-furyl)-4-[(1-methyl-4-piperidyl)amino]quinolin-7-ol (<b>75</b>)</h4><div class="NLM_p last">A solution of compound <b>74c</b> (500 mg, 1.32 mmol), 1-methylpiperidin-4-amine (300 mg, 2.63 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (120 mg, 0.13 mmol), BINAP (82 mg, 0.13 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (858 mg, 2.63 mmol) in 1,4-dioxane (10 mL) was stirred at 110 °C for 16 h under N<sub>2</sub>. Then, the reaction mixture was concentrated and purified by silica gel column chromatography (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 10:1) to afford intermediate 7-benzyloxy-6-methoxy-2-(5-methyl-2-furyl)-<i>N</i>-(1-methyl-4-piperidyl)quinolin-4-amine (400 mg, 66%). ESI-MS <i>m</i>/<i>z</i> 458.2 [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>31</sub>N<sub>3</sub>O<sub>3</sub>. Then to a solution of this intermediate (457 mg, 1.0 mmol) in MeOH (50 mL) was added Pd/C (120 mg), and the mixture was stirred under H<sub>2</sub> (50 psi) at 25 °C for 16 h. Then, the reaction mixture was filtered and the filter was concentrated. The crude product was purified by silica gel column chromatography (PE/EtOAC = 5:1) to afford pure compound <b>75</b> (350 mg, 95%) as yellow solid. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.77 (s, 1H), 7.53 (s, 1H), 7.36 (s, 1H), 7.02 (s, 1H), 6.43 (s, 1H), 4.29 (s, 1H), 4.06 (s, 3H), 3.72–3.69 (m, 2H), 2.95 (s, 3H), 2.50 (s, 3H), 2.41–2.37 (m, 3H), 2.15–2.12 (m, 3H). ESI-MS <i>m</i>/<i>z</i> 368.2 [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>25</sub>N<sub>3</sub>O<sub>3</sub>.</div></div><div id="sec4_1_102" class="NLM_sec NLM_sec_level_3"><div id="ac_i128" class="anchor-spacer"></div><h4 class="article-section__title" id="_i128"> <i>tert</i>-Butyl 4-[[4-Chloro-6-methoxy-2-(5-methyl-2-furyl)-7-quinolyl]oxymethyl]piperidine-1-carboxylate (<b>76</b>)</h4><div class="NLM_p last">To a mixture of <b>74b</b> (650 mg, 2.24 mmol), <i>tert</i>-butyl 4-(hydroxymethyl) piperidine-1-carboxylate (540 mg, 2.51 mmol), and PPh<sub>3</sub> (1.18 g, 4.48 mmol) in THF (50 mL) was added DIAD (907 mg, 4.48 mmol) in one portion at 0 °C under N<sub>2</sub>, and the mixture was stirred at 0 °C for 8 h. Then, the mixture was concentrated and extracted with EtOAc. The combined organic phase was washed with saturated brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum. The residue was purified by silica gel column chromatography (PE/EtOAc = 2/1) to afford compound <b>76</b> (700 mg, 64%). <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 7.73 (s, 1H), 7.42 (s, 1H), 7.37 (s, 1H), 7.03 (s, 1H), 6.17 (s, 1H), 5.01–4.95 (m, 1H), 4.19–4.13 (m, 2H), 4.04 (s, 3H), 2.81–2.74 (m, 2H), 2.45 (s, 3H), 2.19–2.13 (m, 2H), 1.93–1.85 (m, 2H), 1.75–1.66 (m, 2H), 1.48 (s, 9H). ESI-MS <i>m</i>/<i>z</i> 487.3 [M + H]<sup>+</sup> calcd for C<sub>26</sub>H<sub>31</sub>ClN<sub>2</sub>O<sub>5</sub>.</div></div><div id="sec4_1_103" class="NLM_sec NLM_sec_level_3"><div id="ac_i129" class="anchor-spacer"></div><h4 class="article-section__title" id="_i129"> <i>tert</i>-Butyl 2-[[7-[(1-<i>tert</i>-Butoxycarbonyl-4-piperidyl)methoxy]-6-methoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]-7-azaspiro[3.5]nonane-7-carboxylate (<b>77a</b>)</h4><div class="NLM_p last">To a mixture of <b>76</b> (100 mg, 0.205 mmol) and <i>tert</i>-butyl 2-amino-7-azaspiro[3.5]nonane-7-carboxylate (99 mg, 0.410 mmol) in 1,4-dioxane (10 mL) was added Pd(dba)<sub>2</sub> (12 mg, 0.021 mmol), BINAP (13 mg, 0.021 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (133 mg, 0.410 mmol), and the mixture was stirred at 110 °C for 12 h. Then, the mixture was concentrated and the residue was purified by prep-TLC (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 15/1) to afford compound <b>77a</b> (80.0 mg, 56%) as yellow solid. ESI-MS <i>m</i>/<i>z</i> 691.4 [M + H]<sup>+</sup> calcd for C<sub>39</sub>H<sub>54</sub>N<sub>4</sub>O<sub>7</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_104" class="NLM_sec NLM_sec_level_3"><div id="ac_i130" class="anchor-spacer"></div><h4 class="article-section__title" id="_i130"> <i>tert</i>-Butyl 6-[[4-[(1-<i>tert</i>-Butoxycarbonyl-4-piperidyl)methylamino]-6-methoxy-2-(5-methyl-2-furyl)-7-quinolyl]oxy]-2-azaspiro[3.3]heptane-2-carboxylate (<b>77b</b>)</h4><div class="NLM_p last">To a solution of compound <b>79a</b> (50 mg, 0.107 mmol) in DMF (10 mL) were added Cs<sub>2</sub>CO<sub>3</sub> (70 mg, 0.214 mmol) and <i>tert</i>-butyl 6-methylsulfonyloxy-2-azaspiro[3.3]heptane-2-carboxylate (47 mg, 0.160 mmol), and the mixture was stirred at 100 °C for 16 h. Then, the mixture was concentrated to give crude compound <b>77b</b> (100 mg, crude) as yellow solid which was used for next step without further purification. ESI-MS <i>m</i>/<i>z</i> 663.4 [M + H]<sup>+</sup> calcd for C<sub>37</sub>H<sub>50</sub>N<sub>4</sub>O<sub>7</sub>.</div></div><div id="sec4_1_105" class="NLM_sec NLM_sec_level_3"><div id="ac_i131" class="anchor-spacer"></div><h4 class="article-section__title" id="_i131"> <i>tert</i>-Butyl 2-[[7-[(2-<i>tert</i>-Butoxycarbonyl-2-azaspiro[3.3]heptan-6-yl)oxy]-6-methoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]-7-azaspiro[3.5]nonane-7-carboxylate (<b>77c</b>)</h4><div class="NLM_p last">To a solution of compound <b>79b</b> (190 mg, 0.385 mmol) in DMF (15 mL) were added Cs<sub>2</sub>CO<sub>3</sub> (251 mg, 0.770 mmol) and <i>tert</i>-butyl 6-methylsulfonyloxy-2-azaspiro[3.3]heptane-2-carboxylate (168 mg, 0.577 mmol), and the mixture was stirred at 100 °C for 16 h under N<sub>2</sub>. Then, the reaction mixture was filtered and the filtrate was concentrated. The residue was purified by prep-TLC (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 20:1) to afford pure compound <b>77c</b> (200 mg 75%) as yellow solid. ESI-MS <i>m</i>/<i>z</i> 689.4 [M + H]<sup>+</sup> calcd for C<sub>39</sub>H<sub>52</sub>N<sub>4</sub>O<sub>7</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_106" class="NLM_sec NLM_sec_level_3"><div id="ac_i132" class="anchor-spacer"></div><h4 class="article-section__title" id="_i132"> <i>tert</i>-Butyl 8-[[7-[(2-<i>tert</i>-Butoxycarbonyl-2-azaspiro[3.3]heptan-6-yl)oxy]-6-methoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]-3-azabicyclo[3.2.1]octane-3-carboxylate (<b>77d</b>)</h4><div class="NLM_p last">To a solution of compound <b>79c</b> (110 mg, 0.229 mmol) in DMF (10 mL) were added Cs<sub>2</sub>CO<sub>3</sub> (149 mg, 0.458 mmol) and <i>tert</i>-butyl 6-methylsulfonyloxy-2-azaspiro[3.3]heptane-2-carboxylate (100 mg, 0.344 mmol), and the mixture was stirred at 100 °C for 16 h. Then, the reaction mixture was filtered and the filtrate was concentrated to afford crude compound <b>77d</b> (160 mg, crude) as yellow solid which was used for next step without further purification. ESI-MS <i>m</i>/<i>z</i> 675.4 [M + H]<sup>+</sup> calcd for C<sub>38</sub>H<sub>50</sub>N<sub>4</sub>O<sub>7</sub>.</div></div><div id="sec4_1_107" class="NLM_sec NLM_sec_level_3"><div id="ac_i133" class="anchor-spacer"></div><h4 class="article-section__title" id="_i133"> <i>tert</i>-Butyl 4-[[[7-Benzyloxy-6-methoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]piperidine-1-carboxylate (<b>78a</b>)</h4><div class="NLM_p last">To a solution of compound <b>74c</b> (300 mg, 0.790 mmol), in 1,4-dioxane (20 mL) were added Cs<sub>2</sub>CO<sub>3</sub> (515 mg, 1.58 mmol), BINAP (49 mg, 0.079 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (72 mg, 0.079 mmol), and <i>tert</i>-butyl 4-(aminomethyl)piperidine-1-carboxylate (338 mg, 1.58 mmol), and the mixture was stirred at 130 °C for 16 h. Then, the mixture was concentrated and extracted with EtOAc. The combined organic phase was washed with brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give the crude product. The residue was purified by silica gel column chromatography (CH<sub>2</sub>Cl<sub>2</sub>:methanol = 50:1 to 20:1) to afford pure compound <b>78a</b> (130 mg 29%) as yellow solid. ESI-MS <i>m</i>/<i>z</i> 558.4 [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>39</sub>N<sub>3</sub>O<sub>5</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_108" class="NLM_sec NLM_sec_level_3"><div id="ac_i134" class="anchor-spacer"></div><h4 class="article-section__title" id="_i134"> <i>tert</i>-Butyl 2-[[7-Benzyloxy-6-methoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]-7-azaspiro[3.5]nonane-7-carboxylate (<b>78b</b>)</h4><div class="NLM_p last">To a solution of compound <b>74c</b> (400 mg, 1.05 mmol) in 1,4-dioxane (10 mL) were added Cs<sub>2</sub>CO<sub>3</sub> (686 mg, 2.11 mmol), BINAP (66 mg, 0.105 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (96 mg, 0.105 mmol), and <i>tert</i>-butyl 2-amino-7-azaspiro[3.5]nonane-7-carboxylate (506 mg, 2.11 mmol), and the mixture was stirred at 120 °C for 16 h. Then, the solution was extracted with EtOAc and the combined organic phase was washed with brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give the crude product. The residue was purified by prep-TLC (SiO<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 20:1) to afford pure compound <b>78b</b> (300 mg, 49%) as yellow solid. ESI-MS <i>m</i>/<i>z</i> 584.3 [M + H]<sup>+</sup> calcd for C<sub>35</sub>H<sub>41</sub>N<sub>3</sub>O<sub>5</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_109" class="NLM_sec NLM_sec_level_3"><div id="ac_i135" class="anchor-spacer"></div><h4 class="article-section__title" id="_i135"> <i>tert</i>-Butyl 8-[[7-Benzyloxy-6-methoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]-3-azabicyclo[3.2.1]octane-3-carboxylate (<b>78c</b>)</h4><div class="NLM_p last">To a solution of compound <b>74c</b> (300 mg, 0.790 mmol) in 1,4-dioxane (20 mL) were added Cs<sub>2</sub>CO<sub>3</sub> (514.7 mg, 1.58 mmol), BINAP (49 mg, 0.079 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (72 mg, 0.079 mmol), and <i>tert</i>-butyl 8-amino-3-azabicyclo[3.2.1]octane-3-carboxylate (357 mg, 1.58 mmol), and the mixture was stirred at 130 °C for 16 h. Then, the mixture was concentrated and extracted with EtOAc. The combined organic phase was washed with brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give the crude product which was purified by silica gel column chromatography (CH<sub>2</sub>Cl<sub>2</sub>:methanol = 30:1 to 10:1) to afford pure compound <b>78c</b> (210 mg, 47%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 570.4 [M + H]<sup>+</sup> calcd for C<sub>34</sub>H<sub>39</sub>N<sub>3</sub>O<sub>5</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_110" class="NLM_sec NLM_sec_level_3"><div id="ac_i136" class="anchor-spacer"></div><h4 class="article-section__title" id="_i136"> <i>tert</i>-Butyl 4-[[[7-Hydroxy-6-methoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]piperidine-1-carboxylate (<b>79a</b>)</h4><div class="NLM_p last">To a solution of compound <b>78a</b> (130 mg, 0.233 mmol) in MeOH (15 mL) was added Pd/C (150 mg, 50%), and the suspension was degassed and purged with H<sub>2</sub> for 3 times. The mixture was stirred under H<sub>2</sub> (50 Psi) at 25 °C for 16 h. Then, the reaction mixture was filtered and the filtrate was concentrated to give compound <b>79a</b> (50 mg, 46%) as yellow solid which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 468.3 [M + H]<sup>+</sup> calcd for C<sub>26</sub>H<sub>33</sub>N<sub>3</sub>O<sub>5</sub>.</div></div><div id="sec4_1_111" class="NLM_sec NLM_sec_level_3"><div id="ac_i137" class="anchor-spacer"></div><h4 class="article-section__title" id="_i137"> <i>tert</i>-Butyl 2-[[7-Hydroxy-6-methoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]-7-azaspiro[3.5]nonane-7-carboxylate (<b>79b</b>)</h4><div class="NLM_p last">To a solution of compound <b>78b</b> (300 mg, 0.514 mmol) in MeOH (30 mL) was added Pd/C (100 mg, 10%), and the suspension was degassed and purged with H<sub>2</sub> for 3 times. The mixture was stirred under H<sub>2</sub> (50 Psi) at 25 °C for 16 h. Then, the reaction mixture was filtered, and the filtrate was concentrated to give the compound <b>79b</b> (190 mg, 75%) as yellow solid which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 494.4 [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>35</sub>N<sub>3</sub>O<sub>5</sub>.</div></div><div id="sec4_1_112" class="NLM_sec NLM_sec_level_3"><div id="ac_i138" class="anchor-spacer"></div><h4 class="article-section__title" id="_i138"> <i>tert</i>-Butyl 8-[[7-Hydroxy-6-methoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]-3-azabicyclo[3.2.1]octane-3-carboxylate (<b>79c</b>)</h4><div class="NLM_p last">To a solution of compound <b>78c</b> (210 mg, 0.368 mmol) in MeOH (15 mL) was added Pd/C (200 mg, 50%), and the suspension was degassed and purged with H<sub>2</sub> for 3 times. The mixture was stirred under H<sub>2</sub> (50 Psi) at 25 °C for 16 h. Then, the reaction mixture was filtered and the filtrate was concentrated to give compound <b>79c</b> (110 mg, 62%) as yellow solid which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 480.3 [M + H]<sup>+</sup> calcd for C<sub>27</sub>H<sub>33</sub>N<sub>3</sub>O<sub>5</sub>.</div></div><div id="sec4_1_113" class="NLM_sec NLM_sec_level_3"><div id="ac_i139" class="anchor-spacer"></div><h4 class="article-section__title" id="_i139"> <i>N</i>-(7-Azaspiro[3.5]nonan-2-yl)-6-methoxy-2-(5-methyl-2-furyl)-7-(4-piperidylmethoxy)quinolin-4-amine (<b>80a</b>)</h4><div class="NLM_p last">To a mixture of <b>77a</b> (50.0 mg, 0.072 mmol) in EtOAc (10.0 mL) was added EtOAc/HCl (2.0 M, 5 mL), and the mixture was stirred at 25 °C for 2 h under N<sub>2</sub>. Then, the mixture was concentrated and purified by prep-HPLC (method 3 described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">Supporting Information</a>) to afford compound <b>80a</b> (15.0 mg, 42%) as yellow solid. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.80 (s, 1H), 7.54 (d, <i>J</i> = 3.6 Hz 1H), 7.46 (s, 1H), 6.81 (s, 1H), 6.44 (d, <i>J</i> = 2.8 Hz 1H), 4.53 (d, <i>J</i> = 8.0 Hz 1H), 4.13 (d, <i>J</i> = 5.6 Hz 2H), 4.02 (s, 3H), 3.53–3.47 (m, 2H), 3.31–3.22 (m, 2H), 3.16–3.07 (m, 4H), 2.73–2.64 (m, 2H), 2.51 (s, 3H), 2.27 (br s, 1H), 2.23–2.11 (m, 4H), 2.09–2.05 (m, 2H), 1.94–1.86 (m, 2H), 1.81–1.68 (m, 2H). ESI-MS <i>m</i>/<i>z</i> 491.4 [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>38</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_114" class="NLM_sec NLM_sec_level_3"><div id="ac_i140" class="anchor-spacer"></div><h4 class="article-section__title" id="_i140"> 7-(2-Azaspiro[3.3]heptan-6-yloxy)-6-methoxy-2-(5-methyl-2-furyl)-<i>N</i>-(4-piperidylmethyl)quinolin-4-amine (<b>80b</b>)</h4><div class="NLM_p last">To a solution of compound <b>77b</b> (100 mg, 0.151 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (8 mL) was added TFA (3.1 g) at 0 °C, and the mixture was stirred at 18 °C for 2 h. Then, the mixture was concentrated to give crude compound <b>80b</b> (150 mg, crude) as yellow oil which was used for next step without further purification. ESI-MS <i>m</i>/<i>z</i> 463.4 [M + H]<sup>+</sup> calcd for C<sub>37</sub>H<sub>50</sub>N<sub>4</sub>O<sub>7</sub>.</div></div><div id="sec4_1_115" class="NLM_sec NLM_sec_level_3"><div id="ac_i141" class="anchor-spacer"></div><h4 class="article-section__title" id="_i141"> 7-(2-Azaspiro[3.3]heptan-6-yloxy)-<i>N</i>-(7-azaspiro[3.5]nonan-2-yl)-6-methoxy-2-(5-methyl-2-furyl)quinolin-4-amine (<b>80c</b>)</h4><div class="NLM_p last">To a solution of compound <b>77c</b> (200 mg, 0.290 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (16 mL) was added TFA (6.12 g) at 0 °C, and the mixture was stirred at 18 °C for 2 h. Then, the mixture was concentrated to give the crude compound <b>80c</b> (180 mg, crude) as yellow oil which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 489.3 [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>36</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_116" class="NLM_sec NLM_sec_level_3"><div id="ac_i142" class="anchor-spacer"></div><h4 class="article-section__title" id="_i142"> <i>N</i>-(3-Azabicyclo[3.2.1]octan-8-yl)-7-(2-azaspiro[3.3]heptan-6-yloxy)-6-methoxy-2-(5-methyl-2-furyl)quinolin-4-amine (<b>80d</b>)</h4><div class="NLM_p last">To a solution of compound <b>77d</b> (110 mg, 0.163 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (8 mL) was added TFA (3.06 g) at 0 °C, and the mixture was stirred at 18 °C for 2 h. Then, the mixture was concentrated to afford crude compound <b>80d</b> (150 mg, crude) as yellow oil which was used for next step without further purification. ESI-MS <i>m</i>/<i>z</i> 475.4 [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>34</sub>N<sub>4</sub>O<sub>3</sub>.</div></div><div id="sec4_1_117" class="NLM_sec NLM_sec_level_3"><div id="ac_i143" class="anchor-spacer"></div><h4 class="article-section__title" id="_i143"> <i>N</i>-(7-Azaspiro[3.5]nonan-2-yl)-7-benzyloxy-6-methoxy-2-(5-methyl-2-furyl)quinolin-4-amine (<b>81</b>)</h4><div class="NLM_p last">Compound <b>78b</b> (650 mg, 1.11 mmol) was dissolved in HCl/EtOAc (2.0 M, 10 mL), and the mixture was stirred at 18 °C for 2 h. Then, the mixture was concentrated to afford crude compound <b>81</b> (650 mg, crude) as a yellow solid which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 484.3 [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>33</sub>N<sub>3</sub>O<sub>3</sub>.</div></div><div id="sec4_1_118" class="NLM_sec NLM_sec_level_3"><div id="ac_i144" class="anchor-spacer"></div><h4 class="article-section__title" id="_i144"> 7-Benzyloxy-<i>N</i>-(7-isopropyl-7-azaspiro[3.5]nonan-2-yl)-6-methoxy-2-(5-methyl-2-furyl)quinolin-4-amine (<b>82</b>)</h4><div class="NLM_p last">To a solution of compound <b>81</b> (650 mg, 1.25 mmol) in <i>i</i>-PrOH (15 mL) were added CH<sub>3</sub>COOH (450 mg, 7.50 mmol), acetone (436 mg, 7.50 mmol), and NaBH<sub>3</sub>CN (471 mg, 7.50 mmol), and the mixture was stirred at 60 °C for 16 h under N<sub>2</sub>. Then, the mixture was concentrated and extracted with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic phase was washed with brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give a residue. The residue was purified by silica gel column chromatography (CH<sub>2</sub>Cl<sub>2</sub>:methanol = 50:1 to 1:1) to afford compound <b>82</b> (300 mg, 46%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i> 526.4 [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>39</sub>N<sub>3</sub>O<sub>3</sub>. This compound was used in the next step without further characterization.</div></div><div id="sec4_1_119" class="NLM_sec NLM_sec_level_3"><div id="ac_i145" class="anchor-spacer"></div><h4 class="article-section__title" id="_i145"> 4-[(7-Isopropyl-7-azaspiro[3.5]nonan-2-yl)amino]-6-methoxy-2-(5-methyl-2-furyl)quinolin-7-ol (<b>83</b>)</h4><div class="NLM_p last">To a solution of compound <b>82</b> (300 mg, 0.571 mmol) in MeOH (50 mL) was added Pd/C (10%, 100 mg), and the suspension was degassed and purged with H<sub>2</sub> for 3 times. The mixture was stirred under H<sub>2</sub> (50 psi) at 20 °C for 8 h. Then, the reaction mixture was filtered and the filtrate was concentrated to give crude compound <b>83</b> (200 mg, 81%) as a yellow solid which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 436.3 [M + H]<sup>+</sup> calcd for C<sub>26</sub>H<sub>33</sub>N<sub>3</sub>O<sub>3</sub>.</div></div><div id="sec4_1_120" class="NLM_sec NLM_sec_level_3"><div id="ac_i146" class="anchor-spacer"></div><h4 class="article-section__title" id="_i146"> <i>tert</i>-Butyl 6-[[4-[(7-Isopropyl-7-azaspiro[3.5]nonan-2-yl)amino]-6-methoxy-2-(5-methyl-2-furyl)-7-quinolyl]oxy]-2-azaspiro[3.3]heptane-2-carboxylate (<b>84</b>)</h4><div class="NLM_p last">To a solution of compound <b>83</b> (150 mg, 0.344 mmol) in DMF (10 mL) were added Cs<sub>2</sub>CO<sub>3</sub> (224 mg, 0.689 mmol) and <i>tert</i>-butyl 6-methylsulfonyloxy-2-azaspiro[3.3]heptane-2-carboxylate (151 mg, 0.516 mmol), and the mixture was stirred at 100 °C for 16 h under N<sub>2</sub>. Then, the reaction mixture was filtered and the filtrate was concentrated to give crude compound <b>84</b> (300 mg, crude) as a yellow solid which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 631.5 [M + H]<sup>+</sup> calcd for C<sub>37</sub>H<sub>50</sub>N<sub>4</sub>O<sub>5</sub>.</div></div><div id="sec4_1_121" class="NLM_sec NLM_sec_level_3"><div id="ac_i147" class="anchor-spacer"></div><h4 class="article-section__title" id="_i147"> 7-(2-Azaspiro[3.3]heptan-6-yloxy)-<i>N</i>-(7-isopropyl-7-azaspiro[3.5]nonan-2-yl)-6-methoxy-2-(5-methyl-2-furyl)quinolin-4-amine (<b>85</b>)</h4><div class="NLM_p last">To a solution of compound <b>84</b> (300 mg, 0.476 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (8 mL) was added TFA (3.08 g, 27.01 mmol) at 0 °C, and the mixture was stirred at 18 °C for 2 h. Then, the mixture was concentrated to afford compound <b>85</b> (300 mg, crude) as yellow oil which was used in the next step without further purification. ESI-MS <i>m</i>/<i>z</i> 531.5 [M + H]<sup>+</sup> calcd for C<sub>32</sub>H<sub>42</sub>N<sub>4</sub>O<sub>3</sub>.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i148" class="anchor-spacer"></div><h3 class="article-section__title" id="_i148"> G9a and DNMT1 Docking</h3><div class="NLM_p">Compound <b>6</b> was superposed to the conformation of UNC0638 in the cocrystal structure of the G9-UNC0638-SAH complex (Protein Data Bank, PDB, entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW">3RJW</a>) with the MOE program (Chemical Computing Group, <a href="http://www.chemcomp.com" class="extLink">http://www.chemcomp.com</a>). Then, the overlaid conformation of the compound was translated into the G9a-UNC0638-SAH crystal in order to analyze the key interactions between the ligand and the methyltransferase.</div><div class="NLM_p last">The GoldSuite 5.2 program (Cambridge Crystallographic Data Centre, <a href="https://www.ccdc.cam.ac.uk/pages/Home.aspx" class="extLink">https://www.ccdc.cam.ac.uk/pages/Home.aspx</a>) was used to carry out docking of compounds to DNMT1 and G9a. For G9a, the PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW">3RJW</a> was used, with the ChemScore function and a cavity of 10 Å around the carboxylate oxygen of Asp1088. The setting was validated after reproducing the binding mode of UNC0638. The crystal structure of mouse DNMT1 bound to hemimethylated CpG DNA (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DA4">4DA4</a>) was chosen. To explore both, different binding pockets and different binding modes, a range of docking set ups where considered with emphasis on keeping adequate volume occupancy of the different binding pockets and considering protein–ligand interactions, especially those involving conserved catalytic residues. The docking configuration was, when adequate, validated by reproducing the crystallographic binding mode of SAH. In the final selected setup, the docking region used was a 20 Å sphere around the carboxylate oxygen of Glu1269. The PLP scoring function was used to rank docking poses, and protein hydrogen bond constraints for binding to carboxylate of Glu-1269 were imposed on the ligand. The top 20 best docked structures out of 100 independent genetic algorithm runs were retrieved and visually inspected. The high-scoring pose was finally chosen as it has a plausible binding mode with key interactions with DNMT1 and a high degree of convergence (rmsd < 2 Å) was observed among the top three ranked poses.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i149" class="anchor-spacer"></div><h3 class="article-section__title" id="_i149"> Calculation of the Electrostatic Potential of DNMT1</h3><div class="NLM_p last">A customized python script using OpenEye Toolkits<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> was programmed to calculate the void volume between DNMT1 cavity bound to compound <b>6</b> and color coded it by electrostatic potential. For the purpose of clarity, only negative potentials are shown in red in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i150" class="anchor-spacer"></div><h3 class="article-section__title" id="_i150"> G9a Enzyme Activity Assay</h3><div class="NLM_p">The biochemical assay to measure G9a enzyme activity relies on time-resolved fluorescence energy transfer (TR-FRET) between europium cryptate (donor) and XL665 (acceptor). TR-FRET is observed when biotinylated histone monomethyl-H3K9 peptide is incubated with cryptate-labeled anti- H3K9me2 antibody (CisBio catalogue no. 61KB2KAE) and streptavidin XL665 (CisBio catalogue no. 610SAXLA), after enzymatic reaction of G9a.</div><div class="NLM_p last">The assay was carried out during 1 h at room temperature, in a final volume of 20 μL, with 0.2 nM human G9a enzyme, 40 nM biotinylated histone monomethyl-H3K9 peptide, 20 μM <i>S</i>-adenosylmethionine (SAM) and different final concentrations of tested compounds in assay buffer (50 mM Tris-HCl, 10 mM NaCl, 4 mM DTT, 0.01% Tween-20 pH 9). The final percentage of DMSO was 0.5%. After incubation time enzyme activity was stopped by adding 150 nM of cryptate-labeled anti-H3K9me2 antibody and 16 μM of streptavidin XL665 beads. After 1 h of incubation at room temperature, fluorescence was measured at 620 and 665 nm. A ratio (665 nm/620 nm) was then calculated in order to minimize medium interferences. Positive control was obtained in the presence of the vehicle of the compounds. Negative control was obtained in the absence of G9a enzyme activity. Calculated IC<sub>50</sub> values were determined using GraphPrism using a 4-parameter inhibition curve. Compounds were tested in duplicate at different days, within an experimental error of 0.3 log units. If absolute pIC<sub>50</sub> difference was higher than 0.3 log units, additional replicates were performed until satisfying the experimental error (by discarding individual results with values outside 2 MADs of the mean value).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i151" class="anchor-spacer"></div><h3 class="article-section__title" id="_i151"> DNMT1 Enzyme Activity Assay</h3><div class="NLM_p">The biochemical assay to measure DNMT1 enzyme activity relies on time-resolved fluorescence energy transfer (TR-FRET) between lumi4-Tb (donor) and d2 (acceptor) using the EPIgeneous methyltransferase assay (CisBio catalogue no. 62SAHPEB). TR-FRET is observed when antibody specific to <i>S</i>-adenosylhomocysteine labeled with Lumi4-Tb is incubated with d2-labeled <i>S</i>-adenosylhomocysteine. TR-FRET signal is inversely proportional to the concentration of SAH, product of DNMT1 enzyme activity, in the sample.</div><div class="NLM_p last">The assay was carried out during 15 min at 37 °C, in a final volume of 20 μL, with 20 nM human DNMT1enzyme, 1 μg/mL polydeoxyinosine polydeoxycytosine (pdI-pdC) DNA, 1 μM <i>S</i>-adenosylmethionine (SAM) and different final concentrations of tested compounds in assay buffer (50 mM Tris-HCl, 1 mM EDTA, 1 mM DTT, 0.1% Triton X-100, 5% glycerol pH 7.5). The final percentage of DMSO was 0.5%. After incubation time enzyme activity was stopped by adding 2 μL of buffer one of the EPIgeneous methyltransferase kit assay. After 10 min at room temperature, it was added 4 μL of antibody specific to <i>S</i>-adenosylhomocysteine labeled with Lumi4-Tb 50× and 4 μL of d2-labeled <i>S</i>-adenosylhomocysteine 31× both diluted in buffer two of the EPIgeneous methyltransferase kit. Fluorescence was measured at 620 and 665 nm 1 h later. A ratio (665 nm/620 nm) was then calculated in order to minimize medium interferences. Positive control was obtained in the presence of the vehicle of the compounds. Negative control was obtained in the absence of DNMT1 enzyme activity. Calculated IC<sub>50</sub> values were determined using GraphPrism using a 4-parameter inhibition curve. Compounds were tested in duplicate, within an experimental error of 0.3 log units. If absolute pIC<sub>50</sub> difference was higher than 0.3 log units, additional replicates were performed until satisfying the experimental error (by discarding individual results with values outside 2 MADs of the mean value).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i152" class="anchor-spacer"></div><h3 class="article-section__title" id="_i152"> Epigenetics Selectivity Panel</h3><div class="NLM_p last">Selectivity of <b>13</b> and <b>43</b> against methyltransferases (MLL1, SET7/9, SUV39H1, SUV39H2, PRMT1, PRMT3, PRMT4, PRMT5, PRMT6, PRMT8, EZH1, EZH2, SETD2), DNMTs (DNMT1, DNMT3A, DNMT3B), bromodomains (BRD2, CREBBP, BAZ2B), and histone demethylases (JMJD2C, JMJD3 and JMJD1A) was performed by BPS Bioscience (<a href="http://www.bpsbioscience.com/index.ph" class="extLink">http://www.bpsbioscience.com/index.ph</a>). Binding experiments were performed in duplicate at each concentration. GLP IC<sub>50</sub> determination for <b>13</b> and <b>43</b> was carried out at Eurofins (<a href="https://www.eurofins.com/" class="extLink">https://www.eurofins.com/</a>) in duplicate.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i153" class="anchor-spacer"></div><h3 class="article-section__title" id="_i153"> Cell Culture</h3><div class="NLM_p last">CEMO-1 cell lines were cultured with RPMI 1640 medium supplemented with 20% fetal bovine serum and OCI-Ly10 and OCI-Ly3 cells with IMDM supplemented with 20% human serum and 55 μM of β-mercaptoethanol. All cell lines were maintained in culture at 37 °C in a humid atmosphere containing 5% CO<sub>2</sub> and were tested for mycoplasma.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i154" class="anchor-spacer"></div><h3 class="article-section__title" id="_i154"> Cell Proliferation Assay: MTS</h3><div class="NLM_p last">Cell proliferation was analyzed after 48 h of in vitro treatment using the CellTiter 96 Aqueous One Solution cell proliferation assay (Promega). This is a colorimetric method for determining the number of viable cells in proliferation. For the assay, cells were cultured by triplicate at a density of 1 × 10<sup>6</sup> cells/mL in 96-well plates (100000 cells/well, 100 μL/well), except for OCI-Ly3 and OCI-Ly10 cell lines which were cultured at a density of 0.5 × 10<sup>6</sup> cells/mL (50000 cells/well, 100 μL/well). In all cases, only the 60 inner wells were used to avoid any border effects. After 48 h of treatment, plates were centrifuged at 800 g for 10 min and medium was removed. Then, cells were incubated with 100 μL/well of medium and 20 μL/well of CellTiter 96 Aqueous One Solution reagent. After 1–3 h of incubation at 37 °C, absorbance was measured at 490 nm in a 96-well plate reader. The background absorbance was measured in wells with only cell line medium and solution reagent. Data was calculated as a percentage of total absorbance of treated cell/absorbance of nontreated cells.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i155" class="anchor-spacer"></div><h3 class="article-section__title" id="_i155"> CYP Inhibition</h3><div class="NLM_p last">The inhibitory effect of the compounds on five human cytochrome P450s (1A2, 2C9, 2C19, 2D6, and 3A4) was evaluated in human liver microsomes at WuXi (<a href="http://www.wuxi.com/" class="extLink">http://www.wuxi.com/</a>). Compounds were prepared at 10 μM, and the corresponding substrates for each P450 isoform (20 μL) were incubated with 140 μL of liver microsomes (0.286 mg/mL; BD Gentest) and NADPH cofactor (20 μL, 1 mM) for 10 min at 37 °C. The reaction was terminated by adding 400 μL of cold stop solution (200 ng/mL tolbutamide in ACN), and samples were centrifuged at 1500<i>g</i> for 20 min. Supernatants were analyzed by LC-MS/MS (Shimadzu LC 20-AD-API 4000) using the peak area ratio of the analyte/internal standard. Compounds and positive controls were tested in duplicate. The percentage of inhibition was calculated as the ratio of substrate metabolite detected in treated and nontreated wells.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i156" class="anchor-spacer"></div><h3 class="article-section__title" id="_i156"> hERG Blockade Assay</h3><div class="NLM_p last">The effect of the compounds on the hERG potassium channel was determined using a PredictorTM hERG fluorescence polarization commercial assay kit (Life Technologies, catalogue no. PV5365). The assay was carried out in black 384-well plates (Corning, catalogue no. 3677 PS), monitoring changes in the fluorescence polarization properties of the labeled hERG ligand between its soluble and bound states. The compounds, which will compete with the fluorescently labeled hERG, were solubilized in 100% DMSO across a 16-point concentration curve and then diluted 1:25 with hERG assay buffer. The assay contained 5 μL/well of studied compound, 10 μL/well of hERG membranes, and 5 μL/well of hERG Tracer Red. The plate was incubated 2 h at rt and protected from light. Fluorescence polarization signals were recorded with an Envision plate reader (PerkinElmer).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i157" class="anchor-spacer"></div><h3 class="article-section__title" id="_i157"> Metabolic Stability</h3><div class="NLM_p last">Test compounds (1 μM, 5% MeOH in potassium phosphate buffer) were incubated with human (catalogue no. 452161 from BD Gentest) and mouse (catalogue no. M1000, Xenotech) liver microsomes at 37 °C for 10 min. Liver microsomes were at a final assay concentration of 0.7 mg protein/mL. The reaction was started by the addition of 90 μL of NADP cofactor solution and stopped by the addition of 300 μL of stop solution (ACN at 4 °C, including 100 ng/mL tolbutamide as an internal standard) after 20 min of incubation. The samples were shaken for 5 min and then centrifuged for 20 min at 1500<i>g</i>. A 100 μL aliquot of the supernatant was transferred to eight new 96-well plates with 300 μL of HPLC water and centrifuged at 1500<i>g</i> for LC-MS/MS analysis (Shimadzu LC 10-AD–API 4000). An injection volume of 10 μL was added to a Phenomenex Synergi C18 column eluting with formic acid in water or ACN at a flow rate of 800 μL/min. The percent loss of parent compound was calculated from the peak area ratio of the analyte/internal standard. Compounds and positive controls were tested in duplicate.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i158" class="anchor-spacer"></div><h3 class="article-section__title" id="_i158"> PAMPA Permeability</h3><div class="NLM_p last">The permeability of compounds was evaluated with the parallel artificial membrane permeation assay (PAMPA) as an in vitro model of passive diffusion. Donor solutions of test compounds (180 μL, 50 μM in PBS/EtOH 70:30) were added to each well of the donor plate, whose PVDF membrane was precoated with 4 μL of a 20 mg × mL<sup>–1</sup> PBL/dodecane mixture. PBS/EtOH (180 μL) was added to each well of the PTFE acceptor plate. The donor and acceptor plates were combined together and incubated for 18 h at 20 °C without shaking. In each plate, compounds and controls were tested in duplicate. Drug concentration in the acceptor, the donor, and the reference wells was determined using the UV plate reader with 130 μL of acceptor and donor samples. Permeability rates (<i>P</i><sub>e</sub> in nm s<sup>–1</sup>) were calculated with <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>. The permeability rate of each compound is the averaged value of three independent measurements.<span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_m001.gif" alt="" id="_i159" /></img><span class="labelSpan">(1)</span></span>where <span class="NLM_inline-formula"><img class="imgWithoutFigViewer inline-formula-styles" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_m002.gif" alt="" id="_i160" /></img></span> , <i>V</i><sub>D</sub> = 0.18 mL, <i>V</i><sub>A</sub> = 0.18 mL, area = 0.32 cm<sup>2</sup>, time = 64800 s, <i>D</i><sub>F</sub> = 180/130, [drug]<sub>equilibrium</sub> = ([drug]<sub>donor</sub> × <i>V</i><sub>D</sub> + [drug]<sub>acceptor</sub> × <i>V</i><sub>A</sub>)/(<i>V</i><sub>D</sub> + <i>V</i><sub>A</sub>), [drug]<sub>donor</sub> = (<i>A</i><sub>a</sub>/<i>A</i><sub>i</sub> × <i>D</i><sub>F</sub>)<sub>donor</sub>, [drug]<sub>acceptor</sub> = (<i>A</i><sub>a</sub>/<i>A</i><sub>i</sub> × <i>D</i><sub>F</sub>)<sub>acceptor</sub>, <i>A</i><sub>a donor</sub> = Abs<sub>donor</sub> – Abs<sub>vehicle</sub>, <i>A</i><sub>a acceptor</sub> = Abs<sub>acceptor</sub> – Abs<sub>vehicle</sub>, <i>A</i><sub>i</sub> = Abs<sub>withoutPBL</sub> – Abs<sub>vehicle</sub>.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i161" class="anchor-spacer"></div><h3 class="article-section__title" id="_i161"> Interference Compound Assessment</h3><div class="NLM_p last">Potential pan assay interference compounds (PAINS) liability of reported compounds was assessed according to the structural filters defined by Baell and Holloway<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> (charts 3 and 4 and tables 1–3 of ref <a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a>) using a customized Pipeline Pilot protocol.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> No compound reported in the manuscript matches any of these substructures.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i162" class="anchor-spacer"></div><h3 class="article-section__title" id="_i162"> Western Blot</h3><div class="NLM_p last">After 96 h of treatment, CEMO-1 cells were washed twice with PBS, being the last centrifugation of 4000 rpm for 10 min at 4 °C. Histone extraction was performed as recommended by Upstate Biotechnology. Briefly, cells were homogenized in 5 volumes of lysis buffer (10 mM HEPES, pH 7.9; 1.5 mM MgCl<sub>2</sub>; 10 mM KCl; 0.5 mM DTT; protease inhibitor cocktail (Complete Mini, catalogue no. 11836153301, Roche), and HCl was added to a final concentration of 0.2 M. After incubation on ice for 30 min, the homogenate was centrifuged at 11000<i>g</i> for 10 min at 4 °C, and the supernatant was first dialyzed twice against 0.1 M glacial acetic acid (1 h each time) and then three times against water for 1 h, 3 h and overnight, respectively. The histone concentration in the extract was measured using the dye-binding assay of Bradford. Then 10 μg of histone was separated on 15% SDS-PAGE gel and transferred to a nitrocellulose membrane. The membrane, after being blocked with Tropix I-block blocking reagent (catalogue no.  AI300, Tropix) in PBS with 0.1% of Tween-20 and 0.02 NaN<sub>3</sub>, was incubated with the primary antibody against H3K9me2 (Mouse monoclonal antibody to Histone H3 dimethyl K9, catalogue no.  ab1220, Abcam) diluted 1:2000 overnight at 4 °C and then with alkaline phosphatase-conjugated secondary antibodies. Bound antibodies were revealed by a chemiluminiscent reagent (Tropix) and detected using HyperfilmTM enhanced chemilumincescence. Total H3 was used as a loading control (diluted 1:50000 overnight at 4 °C or for 1 h at rt) (Anti-Histone H3, CT, pan, rabbit polyclonal, catalogue no. 07-690, Millipore).</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i163" class="anchor-spacer"></div><h3 class="article-section__title" id="_i163"> Dot Blot</h3><div class="NLM_p last">After 96 h of treatment, cells were washed twice with PBS and genomic DNA was extracted using a DNA kit (Nucleo Spin Tissue, catalogue no.  74095250, Macherey-Nagel) following the manufacturer’s instructions. DNA purity and concentration was measured using a NanoDrop spectrophotometer (Thermo Scientific). Then 500 ng of genomic DNA was loaded onto a nitrocellulose membrane (Amersham Hybond_N+, RPN203B, GE Healthcare), prewetted in 6× SSC for 10 min, using the Bio-Dot microfiltration apparatus (catalogue no. 170-6545, BioRad) following the manufacturer’s instructions. Then the membrane was incubated with 2× SSC for 5 min and was cross-linked for 2 h at 80 °C. The membrane, after being blocked with Tropix I-block blocking reagent (catalogue no. AI300, Tropix) in PBS with 0.1% of Tween-20 and 0.02 NaN<sub>3</sub>, was incubated with the primary antibody against 5-methylcytosine (monoclonal antibody 5-methylcytidine, catalogue no. BI-MECY-1000, Eurogentec) diluted 1:4000 overnight at 4 °C and then with alkaline phosphatase-conjugated secondary antibody. Bound antibodies were revealed by a chemiluminiscent reagent (Tropix) and detected using Hyperfilm enhanced chemilumincescence.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i164"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01925" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10767" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10767" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01925" class="ext-link">10.1021/acs.jmedchem.7b01925</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Protocols for preparative HPLC purification methods; method for high resolution mass spectrometry of final compounds; methods for LCMS, analytical HPLC, and UHPLC; HRMS and purities of final compounds; HPLC traces of final compounds; selectivity of compounds <b>13</b> and <b>43</b> against epigenetic targets (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_001.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_001.csv">jm7b01925_si_001.csv (3.56 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf">jm7b01925_si_002.pdf (2.21 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB ID Codes: <b>1</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW">3RJW</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DA4">4DA4</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b01925" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66821" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66821" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julen Oyarzabal</span> - <span class="hlFld-Affiliation affiliation">Small
Molecule Discovery Platform, Molecular Therapeutics Program, , Ciberonc, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1941-7255" title="Orcid link">http://orcid.org/0000-0003-1941-7255</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6a001f060f0405130b18100b080b062a1f040b1c440f19"><span class="__cf_email__" data-cfemail="8ee4fbe2ebe0e1f7effcf4efecefe2cefbe0eff8a0ebfd">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Obdulia Rabal</span> - <span class="hlFld-Affiliation affiliation">Small
Molecule Discovery Platform, Molecular Therapeutics Program, , Ciberonc, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3224-0987" title="Orcid link">http://orcid.org/0000-0002-3224-0987</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Juan Antonio Sánchez-Arias</span> - <span class="hlFld-Affiliation affiliation">Small
Molecule Discovery Platform, Molecular Therapeutics Program, , Ciberonc, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Edurne San José-Enériz</span> - <span class="hlFld-Affiliation affiliation">Area de Hemato-Oncología,
IDISNA, , Ciberonc, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xabier Agirre</span> - <span class="hlFld-Affiliation affiliation">Area de Hemato-Oncología,
IDISNA, , Ciberonc, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Irene de Miguel</span> - <span class="hlFld-Affiliation affiliation">Small
Molecule Discovery Platform, Molecular Therapeutics Program, , Ciberonc, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Leire Garate</span> - <span class="hlFld-Affiliation affiliation">Area de Hemato-Oncología,
IDISNA, , Ciberonc, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Estibaliz Miranda</span> - <span class="hlFld-Affiliation affiliation">Area de Hemato-Oncología,
IDISNA, , Ciberonc, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elena Sáez</span> - <span class="hlFld-Affiliation affiliation">Small
Molecule Discovery Platform, Molecular Therapeutics Program, , Ciberonc, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sergio Roa</span> - <span class="hlFld-Affiliation affiliation">Area de Hemato-Oncología,
IDISNA, , Ciberonc, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">José Angel Martínez-Climent</span> - <span class="hlFld-Affiliation affiliation">Area de Hemato-Oncología,
IDISNA, , Ciberonc, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yingying Liu</span> - <span class="hlFld-Affiliation affiliation">WuXi Apptec (Tianjin) Co. Ltd., TEDA, No. 111 HuangHai Road, Fourth Avenue, Tianjin 300456, PR China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Wu</span> - <span class="hlFld-Affiliation affiliation">WuXi Apptec (Tianjin) Co. Ltd., TEDA, No. 111 HuangHai Road, Fourth Avenue, Tianjin 300456, PR China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Musheng Xu</span> - <span class="hlFld-Affiliation affiliation">WuXi Apptec (Tianjin) Co. Ltd., TEDA, No. 111 HuangHai Road, Fourth Avenue, Tianjin 300456, PR China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Felipe Prosper</span> - <span class="hlFld-Affiliation affiliation">Area de Hemato-Oncología,
IDISNA, 
    , , Ciberonc, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span>; 
    <span class="hlFld-Affiliation affiliation">Departmento de Hematología, Clinica Universidad
de Navarra, University of Navarra, Avenida Pio XII 36, E-31008 Pamplona, Spain</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e11688-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i166">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58431" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58431" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the Foundation for Applied Medical Research (FIMA), University of Navarra (Pamplona, Spain), Asociación de Amigos of the University of Navarra as well as the Fundación Fuentes Dutor for financial support. This work has been partially supported through Ministerio de Economía y Competitividad and grants from Instituto de Salud Carlos III PI10/01691, PI13/00862, PI13/01469, PI14/01867, PI16/02024, RTICC RD12/0036/0068, CIBERONC (CB16/12/00489) (co-finance with FEDER funds), ERA-NET programs TRANSCA<i>N</i>-2 JTC EPICA by the “Torres Quevedo” Subprogram (PTQ-11-04777 and PTQ-14-07320 I.D.M), Fundació La Marató de TV3 (20132130-31-32), Gobierno de Navarra (40/2016, PT053/2016 and PT027/2017), and FSE (Inncorpora-Torres Quevedo grant). Finally, we thank OpenEye Scientific Software for providing us with an academic license for use of its software.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AcOH</td><td class="NLM_def"><p class="first last">acetic acid</p></td></tr><tr><td class="NLM_term">ADME</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, and excretion</p></td></tr><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">BINAP</td><td class="NLM_def"><p class="first last">2,2′-bis(diphenylphosphino)-1,1′-binaphthyl</p></td></tr><tr><td class="NLM_term">BOC</td><td class="NLM_def"><p class="first last"><i>tert</i>-butoxycarbonyl</p></td></tr><tr><td class="NLM_term">Cpd</td><td class="NLM_def"><p class="first last">compound</p></td></tr><tr><td class="NLM_term">dba</td><td class="NLM_def"><p class="first last">dibenzylideneacetone</p></td></tr><tr><td class="NLM_term">DEAD</td><td class="NLM_def"><p class="first last">diisopropyl azodicarboxylate</p></td></tr><tr><td class="NLM_term">DIAD</td><td class="NLM_def"><p class="first last">diisopropyl azodicarboxylate</p></td></tr><tr><td class="NLM_term">DLBCL</td><td class="NLM_def"><p class="first last">diffuse large B-cell lymphoma</p></td></tr><tr><td class="NLM_term">DMAP</td><td class="NLM_def"><p class="first last">4-dimethylaminopyridine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">DNA</td><td class="NLM_def"><p class="first last">deoxyribonucleic acid</p></td></tr><tr><td class="NLM_term">DNMT</td><td class="NLM_def"><p class="first last">DNA methyltransferase</p></td></tr><tr><td class="NLM_term">DTT</td><td class="NLM_def"><p class="first last">dithiothreitol</p></td></tr><tr><td class="NLM_term">EDTA</td><td class="NLM_def"><p class="first last">ethylenediaminetetraacetic acid</p></td></tr><tr><td class="NLM_term">EHMT2</td><td class="NLM_def"><p class="first last">euchromatic histone methyltransferase 2</p></td></tr><tr><td class="NLM_term">ESI-MS</td><td class="NLM_def"><p class="first last">electrospray ionization mass spectrometry</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term"><i>i</i>-PrOH</td><td class="NLM_def"><p class="first last">propan-2-ol</p></td></tr><tr><td class="NLM_term">KMT1C</td><td class="NLM_def"><p class="first last">lysine methyltransferase 1C</p></td></tr><tr><td class="NLM_term">KOAc</td><td class="NLM_def"><p class="first last">potassium acetate</p></td></tr><tr><td class="NLM_term">LCMS</td><td class="NLM_def"><p class="first last">liquid chromatography–mass spectrometry</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">mp</td><td class="NLM_def"><p class="first last">melting point</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">microwave</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">PAMPA</td><td class="NLM_def"><p class="first last">parallel artificial membrane permeability assay</p></td></tr><tr><td class="NLM_term">PBS</td><td class="NLM_def"><p class="first last">phosphate buffered saline</p></td></tr><tr><td class="NLM_term">PE</td><td class="NLM_def"><p class="first last">petroleum eter</p></td></tr><tr><td class="NLM_term">Ph</td><td class="NLM_def"><p class="first last">phenyl</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">PMT</td><td class="NLM_def"><p class="first last">protein methyltransferase</p></td></tr><tr><td class="NLM_term">PTFE</td><td class="NLM_def"><p class="first last">polytetrafluoroethylene</p></td></tr><tr><td class="NLM_term">PTSA</td><td class="NLM_def"><p class="first last"><i>p</i>-toluenesulfonic acid</p></td></tr><tr><td class="NLM_term">PVDF</td><td class="NLM_def"><p class="first last">polyvinylidene difluoride</p></td></tr><tr><td class="NLM_term">rt</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term"><i>R</i><sub>t</sub></td><td class="NLM_def"><p class="first last">retention time</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosyl methionine</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term"><i>t</i>-BuOH</td><td class="NLM_def"><p class="first last"><i>tert</i>-butanol</p></td></tr><tr><td class="NLM_term"><i>t</i>-BuOK</td><td class="NLM_def"><p class="first last">potassium <i>tert</i>-butoxide</p></td></tr><tr><td class="NLM_term"><i>t</i>-BuONa</td><td class="NLM_def"><p class="first last">sodium <i>tert</i>-butoxide</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin layer chromatography</p></td></tr><tr><td class="NLM_term">TMS</td><td class="NLM_def"><p class="first last">tetramethylsilane</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">UPLC</td><td class="NLM_def"><p class="first last">ultra performance liquid chromatography</p></td></tr><tr><td class="NLM_term">UV</td><td class="NLM_def"><p class="first last">ultraviolet</p></td></tr><tr><td class="NLM_term">xantphos</td><td class="NLM_def"><p class="first last">4,5-bis(diphenylphosphino)-9,9-dimethylxanthene</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i168">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48617" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48617" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 24 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Portela, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteller, M.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Modifications and Human Disease</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1057</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1038/nbt.1685</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1038%2Fnbt.1685" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=20944598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yjsLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=1057-1068&issue=10&author=A.+Portelaauthor=M.+Esteller&title=Epigenetic+Modifications+and+Human+Disease&doi=10.1038%2Fnbt.1685"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic modifications and human disease</span></div><div class="casAuthors">Portela, Anna; Esteller, Manel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1057-1068</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetics is one of the most rapidly expanding fields in biol.  The recent characterization of a human DNA methylome at single nucleotide resoln., the discovery of the CpG island shores, the finding of new histone variants and modifications, and the unveiling of genome-wide nucleosome positioning maps highlight the accelerating speed of discovery over the past two years.  Increasing interest in epigenetics has been accompanied by technol. breakthroughs that now make it possible to undertake large-scale epigenomic studies.  These allow the mapping of epigenetic marks, such as DNA methylation, histone modifications and nucleosome positioning, which are crit. for regulating gene and noncoding RNA expression.  In turn, we are learning how aberrant placement of these epigenetic marks and mutations in the epigenetic machinery is involved in disease.  Thus, a comprehensive understanding of epigenetic mechanisms, their interactions and alterations in health and disease, has become a priority in biomedical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmU-oOba01dbVg90H21EOLACvtfcHk0lgvbeH2fd7waw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yjsLzF&md5=02649d0c2ed862eda90d7bd349871c71</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1685%26sid%3Dliteratum%253Aachs%26aulast%3DPortela%26aufirst%3DA.%26aulast%3DEsteller%26aufirst%3DM.%26atitle%3DEpigenetic%2520Modifications%2520and%2520Human%2520Disease%26jtitle%3DNat.%2520Biotechnol.%26date%3D2010%26volume%3D28%26issue%3D10%26spage%3D1057%26epage%3D1068%26doi%3D10.1038%2Fnbt.1685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">Marked for Death: Targeting Epigenetic Changes in Cancer</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">241</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1038%2Fnrd.2016.256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=28280262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1Ontr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=241-263&issue=4&author=S.+X.+Pfisterauthor=A.+Ashworth&title=Marked+for+Death%3A+Targeting+Epigenetic+Changes+in+Cancer&doi=10.1038%2Fnrd.2016.256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Marked for death: targeting epigenetic changes in cancer</span></div><div class="casAuthors">Pfister, Sophia Xiao; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">241-263</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In the past few years, it has become clear that mutations in epigenetic regulatory genes are common in human cancers.  Therapeutic strategies are now being developed to target cancers with mutations in these genes using specific chem. inhibitors.  In addn., a complementary approach based on the concept of synthetic lethality, which allows exploitation of loss-of-function mutations in cancers that are not targetable by conventional methods, has gained traction.  Both of these approaches are now being tested in several clin. trials.  In this Review, we present recent advances in epigenetic drug discovery and development, and suggest possible future avenues of investigation to drive progress in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP-s0n0CRO2rVg90H21EOLACvtfcHk0lgvbeH2fd7waw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1Ontr8%253D&md5=f028b0c2e596b7c358e2158318a1303c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.256%26sid%3Dliteratum%253Aachs%26aulast%3DPfister%26aufirst%3DS.%2BX.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DMarked%2520for%2520Death%253A%2520Targeting%2520Epigenetic%2520Changes%2520in%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26issue%3D4%26spage%3D241%26epage%3D263%26doi%3D10.1038%2Fnrd.2016.256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorsey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuikov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenuwein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. L.</span></span> <span> </span><span class="NLM_article-title">G9a and Glp Methylate Lysine 373 in the Tumor Suppressor p53</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">9636</span>– <span class="NLM_lpage">9641</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.062588</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1074%2Fjbc.M109.062588" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=9636-9641&issue=13&author=J.+Huangauthor=J.+Dorseyauthor=S.+Chuikovauthor=X.+Zhangauthor=T.+Jenuweinauthor=D.+Reinbergauthor=S.+L.+Berger&title=G9a+and+Glp+Methylate+Lysine+373+in+the+Tumor+Suppressor+p53&doi=10.1074%2Fjbc.M109.062588"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.062588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.062588%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDorsey%26aufirst%3DJ.%26aulast%3DChuikov%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DReinberg%26aufirst%3DD.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DG9a%2520and%2520Glp%2520Methylate%2520Lysine%2520373%2520in%2520the%2520Tumor%2520Suppressor%2520p53%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26issue%3D13%26spage%3D9636%26epage%3D9641%26doi%3D10.1074%2Fjbc.M109.062588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Casciello, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Windloch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gannon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. S.</span></span> <span> </span><span class="NLM_article-title">Functional Role of G9a Histone Methyltransferase in Cancer</span>. <i>Front. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">487</span>, <span class="refDoi"> DOI: 10.3389/fimmu.2015.00487</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.3389%2Ffimmu.2015.00487" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=26441991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmt1Glsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=487&author=F.+Cascielloauthor=K.+Windlochauthor=F.+Gannonauthor=J.+S.+Lee&title=Functional+Role+of+G9a+Histone+Methyltransferase+in+Cancer&doi=10.3389%2Ffimmu.2015.00487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Functional role of G9a histone methyltransferase in cancer</span></div><div class="casAuthors">Casciello, Francesco; Windloch, Karolina; Gannon, Frank; Lee, Jason S.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">487/1-487/12</span>CODEN:
                <span class="NLM_cas:coden">FIRMCW</span>;
        ISSN:<span class="NLM_cas:issn">1664-3224</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Post-translational modifications of DNA and histones are epigenetic mechanisms, which affect the chromatin structure, ultimately leading to gene expression changes.  A no. of different epigenetic enzymes are actively involved in the addn. or the removal of various covalent modifications, which include acetylation, methylation, phosphorylation, ubiquitination, and sumoylation.  Deregulation of these processes is a hallmark of can-cer.  For instance, G9a, a histone methyltransferase responsible for histone H3 lysine 9 (H3K9) mono- and dimethylation, has been obsd. to be upregulated in different types of cancer and its overexpression has been assocd. with poor prognosis.  Key roles played by these enzymes in various diseases have led to the hypothesis that these mols. represent valuable targets for future therapies.  Several small mol. inhibi-tors have been developed to specifically block the epigenetic activity of these enzymes, representing promising therapeutic tools in the treatment of human malignancies, such as cancer.  In this review, the role of one of these epigenetic enzymes, G9a, is discussed, focusing on its functional role in regulating gene expression as well as its implications in cancer initiation and progression.  We also discuss important findings from recent studies using epigenetic inhibitors in cell systems in vitro as well as exptl. tumor growth and metastasis assays in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptTH4mE94-t7Vg90H21EOLACvtfcHk0liwW7tURFtxiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmt1Glsbk%253D&md5=74bd529b0e0c55e016075fadedebc7e1</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2015.00487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2015.00487%26sid%3Dliteratum%253Aachs%26aulast%3DCasciello%26aufirst%3DF.%26aulast%3DWindloch%26aufirst%3DK.%26aulast%3DGannon%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DJ.%2BS.%26atitle%3DFunctional%2520Role%2520of%2520G9a%2520Histone%2520Methyltransferase%2520in%2520Cancer%26jtitle%3DFront.%2520Immunol.%26date%3D2015%26volume%3D6%26spage%3D487%26doi%3D10.3389%2Ffimmu.2015.00487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Estève, P.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smallwood, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feehery, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangisetty, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karpf, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carey, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradhan, S.</span></span> <span> </span><span class="NLM_article-title">Direct Interaction between DNMT1 and G9a Coordinates DNA and Histone Methylation during Replication</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">3089</span>– <span class="NLM_lpage">3103</span>, <span class="refDoi"> DOI: 10.1101/gad.1463706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1101%2Fgad.1463706" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2006&pages=3089-3103&issue=22&author=P.-O.+Est%C3%A8veauthor=H.+G.+Chinauthor=A.+Smallwoodauthor=G.+R.+Feeheryauthor=O.+Gangisettyauthor=A.+R.+Karpfauthor=M.+F.+Careyauthor=S.+Pradhan&title=Direct+Interaction+between+DNMT1+and+G9a+Coordinates+DNA+and+Histone+Methylation+during+Replication&doi=10.1101%2Fgad.1463706"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1101%2Fgad.1463706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1463706%26sid%3Dliteratum%253Aachs%26aulast%3DEst%25C3%25A8ve%26aufirst%3DP.-O.%26aulast%3DChin%26aufirst%3DH.%2BG.%26aulast%3DSmallwood%26aufirst%3DA.%26aulast%3DFeehery%26aufirst%3DG.%2BR.%26aulast%3DGangisetty%26aufirst%3DO.%26aulast%3DKarpf%26aufirst%3DA.%2BR.%26aulast%3DCarey%26aufirst%3DM.%2BF.%26aulast%3DPradhan%26aufirst%3DS.%26atitle%3DDirect%2520Interaction%2520between%2520DNMT1%2520and%2520G9a%2520Coordinates%2520DNA%2520and%2520Histone%2520Methylation%2520during%2520Replication%26jtitle%3DGenes%2520Dev.%26date%3D2006%26volume%3D20%26issue%3D22%26spage%3D3089%26epage%3D3103%26doi%3D10.1101%2Fgad.1463706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">San
José-Enériz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agirre, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilas-Zornoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paiva, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estella-Hermoso
de Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casares, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Subero, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogi, F.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soule, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiveri, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos-Olivas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia Fernandez de Barrena, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Madoz, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barchino, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasarte, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avila, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez-Climent, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosper, F.</span></span> <span> </span><span class="NLM_article-title">Discovery of First-in-Class Reversible Dual Small Molecule Inhibitors against G9a and DNMTs with in Vivo Activity in Hematological Malignancies</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">15424</span>, <span class="refDoi"> DOI: 10.1038/ncomms15424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1038%2Fncomms15424" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=28548080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1egs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=15424&author=E.+San%0AJos%C3%A9-En%C3%A9rizauthor=X.+Agirreauthor=O.+Rabalauthor=A.+Vilas-Zornozaauthor=J.+A.+S%C3%A1nchez-Ariasauthor=E.+Mirandaauthor=A.+Ugarteauthor=S.+Roaauthor=B.+Paivaauthor=A.+Estella-Hermoso%0Ade+Mendozaauthor=R.+M.+Alvarezauthor=N.+Casaresauthor=V.+Seguraauthor=J.+I.+Mart%C3%ADn-Suberoauthor=F.-X.+Ogiauthor=P.+Souleauthor=C.+M.+Santiveriauthor=R.+Campos-Olivasauthor=G.+Castellanoauthor=M.+Garcia+Fernandez+de+Barrenaauthor=J.+R.+Rodriguez-Madozauthor=M.+J.+Garc%C3%ADa-Barchinoauthor=J.+J.+Lasarteauthor=M.+A.+Avilaauthor=J.+A.+Martinez-Climentauthor=J.+Oyarzabalauthor=F.+Prosper&title=Discovery+of+First-in-Class+Reversible+Dual+Small+Molecule+Inhibitors+against+G9a+and+DNMTs+with+in+Vivo+Activity+in+Hematological+Malignancies&doi=10.1038%2Fncomms15424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of first-in-class reversible dual small molecule inhibitors against G9a and DNMTs in hematological malignancies</span></div><div class="casAuthors">San Jose-Eneriz, Edurne; Agirre, Xabier; Rabal, Obdulia; Vilas-Zornoza, Amaia; Sanchez-Arias, Juan A.; Miranda, Estibaliz; Ugarte, Ana; Roa, Sergio; Paiva, Bruno; Estella-Hermoso de Mendoza, Ander; Alvarez, Rosa Maria; Casares, Noelia; Segura, Victor; Martin-Subero, Jose I.; Ogi, Francois-Xavier; Soule, Pierre; Santiveri, Clara M.; Campos-Olivas, Ramon; Castellano, Giancarlo; Fernandez de Barrena, Maite Garcia; Rodriguez-Madoz, Juan Roberto; Garcia-Barchino, Maria Jose; Lasarte, Juan Jose; Avila, Matias A.; Martinez-Climent, Jose Angel; Oyarzabal, Julen; Prosper, Felipe</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15424</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The indisputable role of epigenetics in cancer and the fact that epigenetic alterations can be reversed have favored development of epigenetic drugs.  In this study, we design and synthesize potent novel, selective and reversible chem. probes that simultaneously inhibit the G9a and DNMTs methyltransferase activity.  In vitro treatment of haematol. neoplasia (acute myeloid leukemia-AML, acute lymphoblastic leukemia-ALL and diffuse large B-cell lymphoma-DLBCL) with the lead compd. CM-272, inhibits cell proliferation and promotes apoptosis, inducing interferon-stimulated genes and immunogenic cell death.  CM-272 significantly prolongs survival of AML, ALL and DLBCL xenogeneic models.  Our results represent the discovery of first-in-class dual inhibitors of G9a/DNMTs and establish this chem. series as a promising therapeutic tool for unmet needs in haematol. tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpstz1QqrJd4rVg90H21EOLACvtfcHk0lgX6GeNRD683g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1egs7Y%253D&md5=6d37a6433cdb371bfb535b1fd8a7a803</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fncomms15424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms15424%26sid%3Dliteratum%253Aachs%26aulast%3DSan%2BJos%25C3%25A9-En%25C3%25A9riz%26aufirst%3DE.%26aulast%3DAgirre%26aufirst%3DX.%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DVilas-Zornoza%26aufirst%3DA.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DMiranda%26aufirst%3DE.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DRoa%26aufirst%3DS.%26aulast%3DPaiva%26aufirst%3DB.%26aulast%3DEstella-Hermoso%2Bde%2BMendoza%26aufirst%3DA.%26aulast%3DAlvarez%26aufirst%3DR.%2BM.%26aulast%3DCasares%26aufirst%3DN.%26aulast%3DSegura%26aufirst%3DV.%26aulast%3DMart%25C3%25ADn-Subero%26aufirst%3DJ.%2BI.%26aulast%3DOgi%26aufirst%3DF.-X.%26aulast%3DSoule%26aufirst%3DP.%26aulast%3DSantiveri%26aufirst%3DC.%2BM.%26aulast%3DCampos-Olivas%26aufirst%3DR.%26aulast%3DCastellano%26aufirst%3DG.%26aulast%3DGarcia%2BFernandez%2Bde%2BBarrena%26aufirst%3DM.%26aulast%3DRodriguez-Madoz%26aufirst%3DJ.%2BR.%26aulast%3DGarc%25C3%25ADa-Barchino%26aufirst%3DM.%2BJ.%26aulast%3DLasarte%26aufirst%3DJ.%2BJ.%26aulast%3DAvila%26aufirst%3DM.%2BA.%26aulast%3DMartinez-Climent%26aufirst%3DJ.%2BA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26aulast%3DProsper%26aufirst%3DF.%26atitle%3DDiscovery%2520of%2520First-in-Class%2520Reversible%2520Dual%2520Small%2520Molecule%2520Inhibitors%2520against%2520G9a%2520and%2520DNMTs%2520with%2520in%2520Vivo%2520Activity%2520in%2520Hematological%2520Malignancies%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D15424%26doi%3D10.1038%2Fncomms15424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerke, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stallcup, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, G.</span></span> <span> </span><span class="NLM_article-title">Lysine Methyltransferase G9a Is Not Required for DNMT3A/3B Anchoring to Methylated Nucleosomes and Maintenance of DNA Methylation in Somatic Cells</span>. <i>Epigenet. Chromatin</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">3</span>, <span class="refDoi"> DOI: 10.1186/1756-8935-5-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1186%2F1756-8935-5-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=22284370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVaitbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=3&issue=1&author=S.+Sharmaauthor=D.+S.+Gerkeauthor=H.+F.+Hanauthor=S.+Jeongauthor=M.+R.+Stallcupauthor=P.+A.+Jonesauthor=G.+Liang&title=Lysine+Methyltransferase+G9a+Is+Not+Required+for+DNMT3A%2F3B+Anchoring+to+Methylated+Nucleosomes+and+Maintenance+of+DNA+Methylation+in+Somatic+Cells&doi=10.1186%2F1756-8935-5-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine methyltransferase G9a is not required for DNMT3A/3B anchoring to methylated nucleosomes and maintenance of DNA methylation in somatic cells</span></div><div class="casAuthors">Sharma, Shikhar; Gerke, Daniel S.; Han, Han F.; Jeong, Shinwu; Stallcup, Michael R.; Jones, Peter A.; Liang, Gangning</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics & Chromatin</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3</span>CODEN:
                <span class="NLM_cas:coden">ECPHAK</span>;
        ISSN:<span class="NLM_cas:issn">1756-8935</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: DNA methylation, histone modifications and nucleosome occupancy act in concert for regulation of gene expression patterns in mammalian cells.  Recently, G9a, a H3K9 methyltransferase, has been shown to play a role in establishment of DNA methylation at embryonic gene targets in ES cells through recruitment of de novo DNMT3A/3B enzymes.  However, whether G9a plays a similar role in maintenance of DNA methylation in somatic cells is still unclear.  Results: Here we show that G9a is not essential for maintenance of DNA methylation in somatic cells.  Knockdown of G9a has no measurable effect on DNA methylation levels at G9a-target loci.  DNMT3A/3B remain stably anchored to nucleosomes contg. methylated DNA even in the absence of G9a, ensuring faithful propagation of methylated states in cooperation with DNMT1 through somatic divisions.  Moreover, G9a also assocs. with nucleosomes in a DNMT3A/3B and DNA methylation-independent manner.  However, G9a knockdown synergizes with pharmacol. inhibition of DNMTs resulting in increased hypomethylation and inhibition of cell proliferation.  Conclusions: Taken together, these data suggest that G9a is not involved in maintenance of DNA methylation in somatic cells but might play a role in re-initiation of de novo methylation after treatment with hypomethylating drugs, thus serving as a potential target for combinatorial treatments strategies involving DNMTs inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG7H2E7h_WpLVg90H21EOLACvtfcHk0lgX6GeNRD683g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVaitbo%253D&md5=d4c61a81e34cabcb684be6b4a7dab495</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2F1756-8935-5-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8935-5-3%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DGerke%26aufirst%3DD.%2BS.%26aulast%3DHan%26aufirst%3DH.%2BF.%26aulast%3DJeong%26aufirst%3DS.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DLiang%26aufirst%3DG.%26atitle%3DLysine%2520Methyltransferase%2520G9a%2520Is%2520Not%2520Required%2520for%2520DNMT3A%252F3B%2520Anchoring%2520to%2520Methylated%2520Nucleosomes%2520and%2520Maintenance%2520of%2520DNA%2520Methylation%2520in%2520Somatic%2520Cells%26jtitle%3DEpigenet.%2520Chromatin%26date%3D2012%26volume%3D5%26issue%3D1%26spage%3D3%26doi%3D10.1186%2F1756-8935-5-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Protein Methyltransferases and Demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+%C3%9C.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+Protein+Methyltransferases+and+Demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0lguuSOQeaeieQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520Protein%2520Methyltransferases%2520and%2520Demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rival-Gervier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labrie, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimaggio, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasney, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siarheyeva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangano, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattenden, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petronis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">A Chemical Probe Selectively Inhibits G9a and GLP Methyltransferase Activity in Cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">566</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1038/nchembio.599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&issue=8&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+Dimaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.-C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+J.+Manganoauthor=X.-P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+Chemical+Probe+Selectively+Inhibits+G9a+and+GLP+Methyltransferase+Activity+in+Cells&doi=10.1038%2Fnchembio.599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0lj2EM9B-g_9YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDimaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.-C.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520Chemical%2520Probe%2520Selectively%2520Inhibits%2520G9a%2520and%2520GLP%2520Methyltransferase%2520Activity%2520in%2520Cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26issue%3D8%26spage%3D566%26epage%3D574%26doi%3D10.1038%2Fnchembio.599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sweis, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pliushchev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maag, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappano, W. N.</span></span> <span> </span><span class="NLM_article-title">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase g9a</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1021/ml400496h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400496h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=205-209&issue=2&author=R.+F.+Sweisauthor=M.+Pliushchevauthor=P.+J.+Brownauthor=J.+Guoauthor=F.+Liauthor=D.+Maagauthor=A.+M.+Petrosauthor=N.+B.+Soniauthor=C.+Tseauthor=M.+Vedadiauthor=M.+R.+Michaelidesauthor=G.+G.+Chiangauthor=W.+N.+Pappano&title=Discovery+and+Development+of+Potent+and+Selective+Inhibitors+of+Histone+Methyltransferase+g9a&doi=10.1021%2Fml400496h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase G9a</span></div><div class="casAuthors">Sweis, Ramzi F.; Pliushchev, Marina; Brown, Peter J.; Guo, Jun; Li, Fengling; Maag, David; Petros, Andrew M.; Soni, Nirupama B.; Tse, Chris; Vedadi, Masoud; Michaelides, Michael R.; Chiang, Gary G.; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-209</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G9a is a histone lysine methyltransferase responsible for the methylation of histone H3 lysine 9.  The discovery of A-366 arose from a unique diversity screening hit, which was optimized by incorporation of a propyl-pyrrolidine subunit to occupy the enzyme lysine channel.  A-366 (I) is a potent inhibitor of G9a (IC50: 3.3 nM) with greater than 1000-fold selectivity over 21 other methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMvdTiUgx-oLVg90H21EOLACvtfcHk0lj2EM9B-g_9YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D&md5=26affad72f2950510aa9f253e540d575</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fml400496h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400496h%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520and%2520Development%2520of%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520Histone%2520Methyltransferase%2520g9a%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26issue%3D2%26spage%3D205%26epage%3D209%26doi%3D10.1021%2Fml400496h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erdmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halby, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fahy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span> <span> </span><span class="NLM_article-title">Targeting DNA Methylation with Small Molecules: What’s Next?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">2569</span>– <span class="NLM_lpage">2583</span>, <span class="refDoi"> DOI: 10.1021/jm500843d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500843d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2569-2583&issue=6&author=A.+Erdmannauthor=L.+Halbyauthor=J.+Fahyauthor=P.+B.+Arimondo&title=Targeting+DNA+Methylation+with+Small+Molecules%3A+What%E2%80%99s+Next%3F&doi=10.1021%2Fjm500843d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm500843d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500843d%26sid%3Dliteratum%253Aachs%26aulast%3DErdmann%26aufirst%3DA.%26aulast%3DHalby%26aufirst%3DL.%26aulast%3DFahy%26aufirst%3DJ.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26atitle%3DTargeting%2520DNA%2520Methylation%2520with%2520Small%2520Molecules%253A%2520What%25E2%2580%2599s%2520Next%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D6%26spage%3D2569%26epage%3D2583%26doi%3D10.1021%2Fjm500843d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span> <span> </span><span class="NLM_article-title">DNA Methyltransferase Inhibitors: An Updated Patent Review (2012–2015)</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1017</span>– <span class="NLM_lpage">1030</span>, <span class="refDoi"> DOI: 10.1080/13543776.2016.1209488</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1080%2F13543776.2016.1209488" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=27376512" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFyhtLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=1017-1030&issue=9&author=P.+Xuauthor=G.+Huauthor=C.+Luoauthor=Z.+Liang&title=DNA+Methyltransferase+Inhibitors%3A+An+Updated+Patent+Review+%282012%E2%80%932015%29&doi=10.1080%2F13543776.2016.1209488"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">DNA methyltransferase inhibitors: an updated patent review (2012-2015)</span></div><div class="casAuthors">Xu, Pan; Hu, Guang; Luo, Cheng; Liang, Zhongjie</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1017-1030</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  DNA methyltransferases (DNMTs), important enzymes involved in epigenetic regulation of gene expression, represent promising targets in cancer therapy.  DNMT inhibitors (DNMTi), which can modulate the aberrant DNA methylation pattern in a reversible way via inhibiting DNMT activity, have attracted significant attention in recent years.  This review outlines the newly patented inhibitors targeting DNMTs, mainly incorporating small mol. inhibitors and oligonucleotide derivs.  The chem. structures, biol. activity, and the encouraging clin. research in progress are delineated in detail.  Two drugs, azacitidine and decitabine, have evidently shown efficacy in hematol. malignancies, yet do not work well on solid tumors, have low specificity, substantial toxicity, and poor bioavailability.  With the rapid advancement in systems biol., drug combinations, such as DNMTi, in conjugation with histone deacetylase inhibitors (HDACi) or immunotherapy, probably serve as an efficient way of implementing epigenetic therapy.  Meanwhile, the resolved autoinhibitory structures of DNMTs afford a novel strategy for targeting the protein-protein interface involved in the autoinhi-bitory interactions.  The mol. mechanism underlying the conformational transitions would also shed new light on the design of allosteric inhibitors.  Both strategies would produce inhibitors with more selectivity compared to nucleotide derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFxx109NDo-rVg90H21EOLACvtfcHk0liShNooikuddw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFyhtLnP&md5=fdfc78375967b9ab9b5fa55d8519d75f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1080%2F13543776.2016.1209488&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2016.1209488%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DP.%26aulast%3DHu%26aufirst%3DG.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DLiang%26aufirst%3DZ.%26atitle%3DDNA%2520Methyltransferase%2520Inhibitors%253A%2520An%2520Updated%2520Patent%2520Review%2520%25282012%25E2%2580%25932015%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2016%26volume%3D26%26issue%3D9%26spage%3D1017%26epage%3D1030%26doi%3D10.1080%2F13543776.2016.1209488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span>; <span class="NLM_string-name">SanJosé-Enériz, E.</span>; <span class="NLM_string-name">Agirre, X.</span>; <span class="NLM_string-name">Sánchez-Arias, J. A.</span>; <span class="NLM_string-name">Vilas-Zornoza, A.</span>; <span class="NLM_string-name">Ugarte, A.</span>; <span class="NLM_string-name">de Miguel, I.</span>; <span class="NLM_string-name">Miranda, E.</span>; <span class="NLM_string-name">Garate, L.</span>; <span class="NLM_string-name">Fraga, M.</span>; <span class="NLM_string-name">Santamarina, P.</span>; <span class="NLM_string-name">Fernandez Perez, R.</span>; <span class="NLM_string-name">Ordoñez, R.</span>; <span class="NLM_string-name">Sáez, E.</span>; <span class="NLM_string-name">Roa, S.</span>; <span class="NLM_string-name">García-Barchino, M. J.</span>; <span class="NLM_string-name">Martínez-Climent, J. Á.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Wei, W.</span>; <span class="NLM_string-name">Musheng, X.</span>; <span class="NLM_string-name">Prosper, F.</span>; <span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Reversible DNAMethyltransferase and Lysine Methyltransferase G9a Inhibitors withAntitumoral in Vivo Efficacy</span>. In press, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01926</span> .</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Rabal%2C+O.%3B+SanJos%C3%A9-En%C3%A9riz%2C+E.%3B+Agirre%2C+X.%3B+S%C3%A1nchez-Arias%2C+J.+A.%3B+Vilas-Zornoza%2C+A.%3B+Ugarte%2C+A.%3B+de+Miguel%2C+I.%3B+Miranda%2C+E.%3B+Garate%2C+L.%3B+Fraga%2C+M.%3B+Santamarina%2C+P.%3B+Fernandez+Perez%2C+R.%3B+Ordo%C3%B1ez%2C+R.%3B+S%C3%A1ez%2C+E.%3B+Roa%2C+S.%3B+Garc%C3%ADa-Barchino%2C+M.+J.%3B+Mart%C3%ADnez-Climent%2C+J.+%C3%81.%3B+Liu%2C+Y.%3B+Wei%2C+W.%3B+Musheng%2C+X.%3B+Prosper%2C+F.%3B+Oyarzabal%2C+J.+Discovery+of+Reversible+DNAMethyltransferase+and+Lysine+Methyltransferase+G9a+Inhibitors+withAntitumoral+in+Vivo+Efficacy.+In+press%2C+10.1021%2Facs.jmedchem.7b01926."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dunpublished%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01926%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26atitle%3DDiscovery%2520of%2520Reversible%2520DNAMethyltransferase%2520and%2520Lysine%2520Methyltransferase%2520G9a%2520Inhibitors%2520withAntitumoral%2520in%2520Vivo%2520Efficacy%26doi%3D10.1021%2Facs.jmedchem.7b01926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span> <i>UNC0638 Selective Chemical Probe for G9a/GLP Methyltransferases</i>; <span class="NLM_publisher-name">SGC</span>, <span class="NLM_year">2016</span><a href="http://www.thesgc.org/chemical-probes/UNC0638" class="extLink">http://www.thesgc.org/chemical-probes/UNC0638</a> (accessed Dec 16, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+UNC0638+Selective+Chemical+Probe+for+G9a%2FGLP+Methyltransferases%3B+SGC%2C+2016http%3A%2F%2Fwww.thesgc.org%2Fchemical-probes%2FUNC0638+%28accessed+Dec+16%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DUNC0638%2520Selective%2520Chemical%2520Probe%2520for%2520G9a%252FGLP%2520Methyltransferases%26pub%3DSGC%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foulks, J.
M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parnell, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swierczek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrickson, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, K.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCullar, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanner, S. B.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Drug Discovery: Targeting DNA Methyltransferases</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">2</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1177/1087057111421212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1177%2F1087057111421212" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=21965114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC38Xit1Oltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=2-17&issue=1&author=J.%0AM.+Foulksauthor=K.+M.+Parnellauthor=R.+N.+Nixauthor=S.+Chauauthor=K.+Swierczekauthor=M.+Saundersauthor=K.+Wrightauthor=T.+F.+Hendricksonauthor=K.-K.+Hoauthor=M.+V.+McCullarauthor=S.+B.+Kanner&title=Epigenetic+Drug+Discovery%3A+Targeting+DNA+Methyltransferases&doi=10.1177%2F1087057111421212"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic drug discovery: targeting DNA methyltransferases</span></div><div class="casAuthors">Foulks, Jason M.; Parnell, K. Mark; Nix, Rebecca N.; Chau, Suzanna; Swierczek, Krzysztof; Saunders, Michael; Wright, Kevin; Hendrickson, Thomas F.; Ho, Koc-Kan; McCullar, Michael V.; Kanner, Steven B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-17</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Epigenetic modification of DNA leads to changes in gene expression.  DNA methyltransferases (DNMTs) comprise a family of nuclear enzymes that catalyze the methylation of CpG dinucleotides, resulting in an epigenetic methylome distinguished between normal cells and those in disease states such as cancer.  Disrupting gene expression patterns through promoter methylation has been implicated in many malignancies and supports DNMTs as attractive therapeutic targets.  This review focuses on the rationale of targeting DNMTs in cancer, the historical approach to DNMT inhibition, and current marketed hypomethylating therapeutics azacytidine and decitabine.  In addn., we address novel DNMT inhibitory agents emerging in development, including CP-4200 and SGI-110, analogs of azacytidine and decitabine, resp.; the oligonucleotides MG98 and miR29a; and a no. of reversible inhibitors, some of which appear to be selective against particular DNMT isoforms.  Finally, we discuss future opportunities and challenges for next-generation therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZfIt_0O0PXrVg90H21EOLACvtfcHk0ljhLw6KrGoJcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xit1Oltro%253D&md5=e1901374d600c0361e04a3c0e6d004d1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1177%2F1087057111421212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111421212%26sid%3Dliteratum%253Aachs%26aulast%3DFoulks%26aufirst%3DJ.%2BM.%26aulast%3DParnell%26aufirst%3DK.%2BM.%26aulast%3DNix%26aufirst%3DR.%2BN.%26aulast%3DChau%26aufirst%3DS.%26aulast%3DSwierczek%26aufirst%3DK.%26aulast%3DSaunders%26aufirst%3DM.%26aulast%3DWright%26aufirst%3DK.%26aulast%3DHendrickson%26aufirst%3DT.%2BF.%26aulast%3DHo%26aufirst%3DK.-K.%26aulast%3DMcCullar%26aufirst%3DM.%2BV.%26aulast%3DKanner%26aufirst%3DS.%2BB.%26atitle%3DEpigenetic%2520Drug%2520Discovery%253A%2520Targeting%2520DNA%2520Methyltransferases%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26issue%3D1%26spage%3D2%26epage%3D17%26doi%3D10.1177%2F1087057111421212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Biologically Relevant Chemical Space Navigator: From Patent and Structure-Activity Relationship Analysis to Library Acquisition and Design</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3123</span>– <span class="NLM_lpage">3137</span>, <span class="refDoi"> DOI: 10.1021/ci3004539</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3004539" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslWksbfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3123-3137&issue=12&author=O.+Rabalauthor=J.+Oyarzabal&title=Biologically+Relevant+Chemical+Space+Navigator%3A+From+Patent+and+Structure-Activity+Relationship+Analysis+to+Library+Acquisition+and+Design&doi=10.1021%2Fci3004539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Biologically Relevant Chemical Space Navigator: From Patent and Structure-Activity Relationship Analysis to Library Acquisition and Design</span></div><div class="casAuthors">Rabal, Obdulia; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3123-3137</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new and versatile visualization tool, based on a descriptor accounting for ligand-receptor interactions (LiRIf), is introduced for guiding medicinal chemists in analyzing the R-groups from a congeneric series.  Anal. is performed in a ref.-independent scenario where the whole biol. relevant chem. space (BRCS) is represented.  Using a real project-based data set, the authors show the impact of this tool on four key navigation strategies for the drug discovery process.  First, this navigator analyzes competitors' patents, including a comparison of patents coverage and the identification of the most frequent fragments.  Second, the tool analyzes the structure-activity relationship (SAR) leading to the representation of ref.-independent activity landscapes that enable the identification not only of crit. ligand-receptor interactions (LRI) and substructural features but also of activity cliffs.  Third, this navigator enables comparison of libraries, thus selecting com. available mols. that complement unexplored spaces or areas of interest.  Finally, this tool also enables the design of new analogs, which is based on reaction types and the exploration purpose (focused or diverse), selecting the most appropriate reagents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIldm-mo1cdrVg90H21EOLACvtfcHk0ljhLw6KrGoJcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslWksbfP&md5=7ee9cb508b97b607067b519d580cab08</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fci3004539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3004539%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DBiologically%2520Relevant%2520Chemical%2520Space%2520Navigator%253A%2520From%2520Patent%2520and%2520Structure-Activity%2520Relationship%2520Analysis%2520to%2520Library%2520Acquisition%2520and%2520Design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D12%26spage%3D3123%26epage%3D3137%26doi%3D10.1021%2Fci3004539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mann, A.</span>; <span class="NLM_string-name">Camille
Georges, W.</span></span> <span> </span><span class="NLM_article-title">Conformational Restriction And/or Steric Hindrance in Medicinal Chemistry</span>. In  <i>The Practice of Medicinal Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wermuth, C.</span></span>, Ed.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2008</span>; Chapter 17, pp  <span class="NLM_fpage">363</span>– <span class="NLM_lpage">379</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1016%2FB978-0-12-374194-3.00017-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=363-379&author=A.+Mann&author=W.+Camille%0AGeorgesauthor=C.+Wermuth&title=The+Practice+of+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-374194-3.00017-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-374194-3.00017-2%26sid%3Dliteratum%253Aachs%26aulast%3DMann%26aufirst%3DA.%26atitle%3DConformational%2520Restriction%2520And%252For%2520Steric%2520Hindrance%2520in%2520Medicinal%2520Chemistry%26btitle%3DThe%2520Practice%2520of%2520Medicinal%2520Chemistry%26aulast%3DWermuth%26aufirst%3DC.%26pub%3DAcademic%2520Press%26date%3D2008%26spage%3D363%26epage%3D379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasney, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siarheyeva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herold, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Protein Lysine Methyltransferase G9a Inhibitors: Design, Synthesis, and Structure Activity Relationships of 2,4-Diamino-7-Aminoalkoxy-Quinazolines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">5844</span>– <span class="NLM_lpage">5857</span>, <span class="refDoi"> DOI: 10.1021/jm100478y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100478y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5844-5857&issue=15&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=T.+J.+Wigleauthor=G.+A.+Wasneyauthor=A.+Dongauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=W.+P.+Janzenauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Protein+Lysine+Methyltransferase+G9a+Inhibitors%3A+Design%2C+Synthesis%2C+and+Structure+Activity+Relationships+of+2%2C4-Diamino-7-Aminoalkoxy-Quinazolines&doi=10.1021%2Fjm100478y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm100478y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100478y%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProtein%2520Lysine%2520Methyltransferase%2520G9a%2520Inhibitors%253A%2520Design%252C%2520Synthesis%252C%2520and%2520Structure%2520Activity%2520Relationships%2520of%25202%252C4-Diamino-7-Aminoalkoxy-Quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D15%26spage%3D5844%26epage%3D5857%26doi%3D10.1021%2Fjm100478y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span> <i>Pipeline Pilot</i>, <span class="NLM_edition">version 9.5</span>; <span class="NLM_publisher-name">Accelrys Software Inc.</span>; <span class="NLM_publisher-loc">San Diego,
CA</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Pipeline+Pilot%2C+version+9.5%3B+Accelrys+Software+Inc.%3B+San+Diego%2C%0ACA%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPipeline%2520Pilot%26pub%3DAccelrys%2520Software%2520Inc%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herold, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Optimization of Cellular Activity of G9a Inhibitors 7-Aminoalkoxy-Quinazolines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">6139</span>– <span class="NLM_lpage">6150</span>, <span class="refDoi"> DOI: 10.1021/jm200903z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200903z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6139-6150&issue=17&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=A.+Allali-Hassaniauthor=Y.+Heauthor=J.+M.+Heroldauthor=X.+Chenauthor=C.+M.+Yatesauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=J.+Huangauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=Optimization+of+Cellular+Activity+of+G9a+Inhibitors+7-Aminoalkoxy-Quinazolines&doi=10.1021%2Fjm200903z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm200903z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200903z%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYates%26aufirst%3DC.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DOptimization%2520of%2520Cellular%2520Activity%2520of%2520G9a%2520Inhibitors%25207-Aminoalkoxy-Quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26issue%3D17%26spage%3D6139%26epage%3D6150%26doi%3D10.1021%2Fjm200903z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillet, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span> <span> </span><span class="NLM_article-title">Assessment of Additive/nonadditive Effects in Structure-Activity Relationships: Implications for Iterative Drug Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">7552</span>– <span class="NLM_lpage">7562</span>, <span class="refDoi"> DOI: 10.1021/jm801070q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801070q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7552-7562&issue=23&author=Y.+Patelauthor=V.+J.+Gilletauthor=T.+Howeauthor=J.+Pastorauthor=J.+Oyarzabalauthor=P.+Willett&title=Assessment+of+Additive%2Fnonadditive+Effects+in+Structure-Activity+Relationships%3A+Implications+for+Iterative+Drug+Design&doi=10.1021%2Fjm801070q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm801070q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801070q%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DY.%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DPastor%26aufirst%3DJ.%26aulast%3DOyarzabal%26aufirst%3DJ.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DAssessment%2520of%2520Additive%252Fnonadditive%2520Effects%2520in%2520Structure-Activity%2520Relationships%253A%2520Implications%2520for%2520Iterative%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26issue%3D23%26spage%3D7552%26epage%3D7562%26doi%3D10.1021%2Fjm801070q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado-Tejedor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miguel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-González, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barroso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estella-Hermoso
de Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espelosin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ursua, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haizhong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Osta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Biological Evaluation of First-in-Class Dual Acting Histone Deacetylases (HDACs) and Phosphodiesterase 5 (PDE5) Inhibitors for the Treatment of Alzheimer’s Disease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">8967</span>– <span class="NLM_lpage">9004</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00908</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00908" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVyqtL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8967-9004&issue=19&author=O.+Rabalauthor=J.+A.+S%C3%A1nchez-Ariasauthor=M.+Cuadrado-Tejedorauthor=I.+de+Miguelauthor=M.+P%C3%A9rez-Gonz%C3%A1lezauthor=C.+Garc%C3%ADa-Barrosoauthor=A.+Ugarteauthor=A.+Estella-Hermoso%0Ade+Mendozaauthor=E.+S%C3%A1ezauthor=M.+Espelosinauthor=S.+Ursuaauthor=T.+Haizhongauthor=W.+Weiauthor=X.+Mushengauthor=A.+Garcia-Ostaauthor=J.+Oyarzabal&title=Design%2C+Synthesis%2C+and+Biological+Evaluation+of+First-in-Class+Dual+Acting+Histone+Deacetylases+%28HDACs%29+and+Phosphodiesterase+5+%28PDE5%29+Inhibitors+for+the+Treatment+of+Alzheimer%E2%80%99s+Disease&doi=10.1021%2Facs.jmedchem.6b00908"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Biological Evaluation of First-in-Class Dual Acting Histone Deacetylases (HDACs) and Phosphodiesterase 5 (PDE5) Inhibitors for the Treatment of Alzheimer's Disease</span></div><div class="casAuthors">Rabal, Obdulia; Sanchez-Arias, Juan A.; Cuadrado-Tejedor, Mar; de Miguel, Irene; Perez-Gonzalez, Marta; Garcia-Barroso, Carolina; Ugarte, Ana; Estella-Hermoso de Mendoza, Ander; Saez, Elena; Espelosin, Maria; Ursua, Susana; Haizhong, Tan; Wei, Wu; Musheng, Xu; Garcia-Osta, Ana; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8967-9004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Simultaneous inhibition of phosphodiesterase 5 (PDE5) and histone deacetylases (HDAC) has recently been validated as a potentially novel therapeutic approach for Alzheimer's Disease (AD).  To further extend this concept, the authors designed and synthesized the first chem. series of dual acting PDE5 and HDAC inhibitors, and the authors validated this systems therapeutics approach.  Following the implementation of structure- and knowledge-based approaches, initial hits were designed and were shown to validate the hypothesis of dual in vitro inhibition.  Then, an optimization strategy was pursued to obtain a proper tool compd. for in vivo testing in AD models.  Initial hits were translated into mols. with adequate cellular functional responses (histone acetylation and cAMP/cGMP response element-binding (CREB) phosphorylation in the nanomolar range), an acceptable therapeutic window (>1 log unit) and the ability to cross the blood-brain barrier, leading to the identification of 7 as a candidate for in vivo proof-of-concept testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbHBoJlefNEbVg90H21EOLACvtfcHk0lhyKLHawuYSDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVyqtL7O&md5=756a252809f05f5d7fc59e828736cfba</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00908%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DCuadrado-Tejedor%26aufirst%3DM.%26aulast%3Dde%2BMiguel%26aufirst%3DI.%26aulast%3DP%25C3%25A9rez-Gonz%25C3%25A1lez%26aufirst%3DM.%26aulast%3DGarc%25C3%25ADa-Barroso%26aufirst%3DC.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DEstella-Hermoso%2Bde%2BMendoza%26aufirst%3DA.%26aulast%3DS%25C3%25A1ez%26aufirst%3DE.%26aulast%3DEspelosin%26aufirst%3DM.%26aulast%3DUrsua%26aufirst%3DS.%26aulast%3DHaizhong%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DMusheng%26aufirst%3DX.%26aulast%3DGarcia-Osta%26aufirst%3DA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Biological%2520Evaluation%2520of%2520First-in-Class%2520Dual%2520Acting%2520Histone%2520Deacetylases%2520%2528HDACs%2529%2520and%2520Phosphodiesterase%25205%2520%2528PDE5%2529%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D19%26spage%3D8967%26epage%3D9004%26doi%3D10.1021%2Facs.jmedchem.6b00908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span> <i>OpenEye Toolkit</i>; <span class="NLM_publisher-name">OpenEye Scientific</span>, <span class="NLM_year">2016</span>;<a href="http://www.eyesopen.com/toolkits" class="extLink">http://www.eyesopen.com/toolkits</a> (accessed Oct 20, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+OpenEye+Toolkit%3B+OpenEye+Scientific%2C+2016%3Bhttp%3A%2F%2Fwww.eyesopen.com%2Ftoolkits+%28accessed+Oct+20%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DOpenEye%2520Toolkit%26pub%3DOpenEye%2520Scientific%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&issue=7&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0liPKxiemiNS2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D7%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 11 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Obdulia Rabal, Edurne San José-Enériz, Xabier Agirre, Juan Antonio Sánchez-Arias, Irene de Miguel, Raquel Ordoñez, Leire Garate, Estíbaliz Miranda, Elena Sáez, Amaia Vilas-Zornoza, Antonio Pineda-Lucena, Ander Estella, Feifei Zhang, Wei Wu, Musheng Xu, Felipe Prosper, <span class="NLM_string-name hlFld-ContribAuthor">Julen Oyarzabal</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Synthesis of Novel Epigenetic Inhibitors Targeting Histone Deacetylases, DNA Methyltransferase 1, and Lysine Methyltransferase G9a with In Vivo Efficacy in Multiple Myeloma. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (6)
                                     , 3392-3426. <a href="https://doi.org/10.1021/acs.jmedchem.0c02255" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02255</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02255%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252BSynthesis%252Bof%252BNovel%252BEpigenetic%252BInhibitors%252BTargeting%252BHistone%252BDeacetylases%25252C%252BDNA%252BMethyltransferase%252B1%25252C%252Band%252BLysine%252BMethyltransferase%252BG9a%252Bwith%252BIn%252BVivo%252BEfficacy%252Bin%252BMultiple%252BMyeloma%26aulast%3DRabal%26aufirst%3DObdulia%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D29122020%26date%3D04032021%26volume%3D64%26issue%3D6%26spage%3D3392%26epage%3D3426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Obdulia Rabal, Edurne San José-Enériz, Xabier Agirre, Juan Antonio Sánchez-Arias, Amaia Vilas-Zornoza, Ana Ugarte, Irene de Miguel, Estíbaliz Miranda, Leire Garate, Mario Fraga, Pablo Santamarina, Raul Fernandez Perez, Raquel Ordoñez, Elena Sáez, Sergio Roa, María José García-Barchino, José Angel Martínez-Climent, Yingying Liu, Wei Wu, Musheng Xu, Felipe Prosper, <span class="NLM_string-name hlFld-ContribAuthor">Julen Oyarzabal</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Reversible DNA Methyltransferase and Lysine Methyltransferase G9a Inhibitors with Antitumoral in Vivo Efficacy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (15)
                                     , 6518-6545. <a href="https://doi.org/10.1021/acs.jmedchem.7b01926" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01926</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01926%26sid%3Dliteratum%253Aachs%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D15424%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BReversible%252BDNA%252BMethyltransferase%252Band%252BLysine%252BMethyltransferase%252BG9a%252BInhibitors%252Bwith%252BAntitumoral%252Bin%252BVivo%252BEfficacy%26aulast%3DRabal%26aufirst%3DObdulia%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D30122017%26date%3D19072018%26date%3D28062018%26volume%3D61%26issue%3D15%26spage%3D6518%26epage%3D6545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Antonio  Garcia-Gomez</span>, <span class="hlFld-ContribAuthor ">Tianlu  Li</span>, <span class="hlFld-ContribAuthor ">Carlos  de la Calle-Fabregat</span>, <span class="hlFld-ContribAuthor ">Javier  Rodríguez-Ubreva</span>, <span class="hlFld-ContribAuthor ">Laura  Ciudad</span>, <span class="hlFld-ContribAuthor ">Francesc  Català-Moll</span>, <span class="hlFld-ContribAuthor ">Gerard  Godoy-Tena</span>, <span class="hlFld-ContribAuthor ">Montserrat  Martín-Sánchez</span>, <span class="hlFld-ContribAuthor ">Laura  San-Segundo</span>, <span class="hlFld-ContribAuthor ">Sandra  Muntión</span>, <span class="hlFld-ContribAuthor ">Xabier  Morales</span>, <span class="hlFld-ContribAuthor ">Carlos  Ortiz-de-Solórzano</span>, <span class="hlFld-ContribAuthor ">Julen  Oyarzabal</span>, <span class="hlFld-ContribAuthor ">Edurne  San José-Enériz</span>, <span class="hlFld-ContribAuthor ">Manel  Esteller</span>, <span class="hlFld-ContribAuthor ">Xabier  Agirre</span>, <span class="hlFld-ContribAuthor ">Felipe  Prosper</span>, <span class="hlFld-ContribAuthor ">Mercedes  Garayoa</span>, <span class="hlFld-ContribAuthor ">Esteban  Ballestar</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting aberrant DNA methylation in mesenchymal stromal cells as a treatment for myeloma bone disease. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2021,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-020-20715-x" title="DOI URL">https://doi.org/10.1038/s41467-020-20715-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-020-20715-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-020-20715-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DTargeting%252Baberrant%252BDNA%252Bmethylation%252Bin%252Bmesenchymal%252Bstromal%252Bcells%252Bas%252Ba%252Btreatment%252Bfor%252Bmyeloma%252Bbone%252Bdisease%26aulast%3DGarcia-Gomez%26aufirst%3DAntonio%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kimberly T.  To</span>, <span class="hlFld-ContribAuthor ">Lindsey  St. Mary</span>, <span class="hlFld-ContribAuthor ">Allyson H.  Wooley</span>, <span class="hlFld-ContribAuthor ">Mitchell S.  Wilbanks</span>, <span class="hlFld-ContribAuthor ">Anthony J.  Bednar</span>, <span class="hlFld-ContribAuthor ">Edward J.  Perkins</span>, <span class="hlFld-ContribAuthor ">Lisa  Truong</span>, <span class="hlFld-ContribAuthor ">Robyn L.  Tanguay</span>, <span class="hlFld-ContribAuthor ">Natàlia  Garcia-Reyero</span>. </span><span class="cited-content_cbyCitation_article-title">Morphological and Behavioral Effects in Zebrafish Embryos after Exposure to Smoke Dyes. </span><span class="cited-content_cbyCitation_journal-name">Toxics</span><span> <strong>2021,</strong> <em>9 </em>
                                    (1)
                                     , 9. <a href="https://doi.org/10.3390/toxics9010009" title="DOI URL">https://doi.org/10.3390/toxics9010009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/toxics9010009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Ftoxics9010009%26sid%3Dliteratum%253Aachs%26jtitle%3DToxics%26atitle%3DMorphological%252Band%252BBehavioral%252BEffects%252Bin%252BZebrafish%252BEmbryos%252Bafter%252BExposure%252Bto%252BSmoke%252BDyes%26aulast%3DTo%26aufirst%3DKimberly%2BT.%26date%3D2021%26date%3D2021%26volume%3D9%26issue%3D1%26spage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Katsushi  Katayama</span>, <span class="hlFld-ContribAuthor ">Ken  Ishii</span>, <span class="hlFld-ContribAuthor ">Eisuke  Tsuda</span>, <span class="hlFld-ContribAuthor ">Keiichi  Yotsumoto</span>, <span class="hlFld-ContribAuthor ">Kumiko  Hiramoto</span>, <span class="hlFld-ContribAuthor ">Makoto  Suzuki</span>, <span class="hlFld-ContribAuthor ">Isao  Yasumatsu</span>, <span class="hlFld-ContribAuthor ">Wataru  Igarashi</span>, <span class="hlFld-ContribAuthor ">Munefumi  Torihata</span>, <span class="hlFld-ContribAuthor ">Takashi  Ishiyama</span>, <span class="hlFld-ContribAuthor ">Takahiro  Katagiri</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel histone lysine methyltransferase G9a/GLP (EHMT2/1) inhibitors: Design, synthesis, and structure-activity relationships of 2,4-diamino-6-methylpyrimidines. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (20)
                                     , 127475. <a href="https://doi.org/10.1016/j.bmcl.2020.127475" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127475</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127475%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252Bnovel%252Bhistone%252Blysine%252Bmethyltransferase%252BG9a%25252FGLP%252B%252528EHMT2%25252F1%252529%252Binhibitors%25253A%252BDesign%25252C%252Bsynthesis%25252C%252Band%252Bstructure-activity%252Brelationships%252Bof%252B2%25252C4-diamino-6-methylpyrimidines%26aulast%3DKatayama%26aufirst%3DKatsushi%26date%3D2020%26volume%3D30%26issue%3D20%26spage%3D127475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Angel R.  de Lera</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">Two-hit wonders: The expanding universe of multitargeting epigenetic agents. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>57 </em>, 135-154. <a href="https://doi.org/10.1016/j.cbpa.2020.05.009" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.05.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.05.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DTwo-hit%252Bwonders%25253A%252BThe%252Bexpanding%252Buniverse%252Bof%252Bmultitargeting%252Bepigenetic%252Bagents%26aulast%3Dde%2BLera%26aufirst%3DAngel%2BR.%26date%3D2020%26volume%3D57%26spage%3D135%26epage%3D154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edgar  López-López</span>, <span class="hlFld-ContribAuthor ">Obdulia  Rabal</span>, <span class="hlFld-ContribAuthor ">Julen  Oyarzabal</span>, <span class="hlFld-ContribAuthor ">José L.  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Towards the understanding of the activity of G9a inhibitors: an activity landscape and molecular modeling approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2020,</strong> <em>34 </em>
                                    (6)
                                     , 659-669. <a href="https://doi.org/10.1007/s10822-020-00298-x" title="DOI URL">https://doi.org/10.1007/s10822-020-00298-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-020-00298-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-020-00298-x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DTowards%252Bthe%252Bunderstanding%252Bof%252Bthe%252Bactivity%252Bof%252BG9a%252Binhibitors%25253A%252Ban%252Bactivity%252Blandscape%252Band%252Bmolecular%252Bmodeling%252Bapproach%26aulast%3DL%25C3%25B3pez-L%25C3%25B3pez%26aufirst%3DEdgar%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D6%26spage%3D659%26epage%3D669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniela  Tomaselli</span>, <span class="hlFld-ContribAuthor ">Alessia  Lucidi</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic polypharmacology: A new frontier for epi‐drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2020,</strong> <em>40 </em>
                                    (1)
                                     , 190-244. <a href="https://doi.org/10.1002/med.21600" title="DOI URL">https://doi.org/10.1002/med.21600</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21600%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DEpigenetic%252Bpolypharmacology%25253A%252BA%252Bnew%252Bfrontier%252Bfor%252Bepi%2525E2%252580%252590drug%252Bdiscovery%26aulast%3DTomaselli%26aufirst%3DDaniela%26date%3D2020%26date%3D2019%26volume%3D40%26issue%3D1%26spage%3D190%26epage%3D244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kailasam N.  Vennila</span>, <span class="hlFld-ContribAuthor ">Kolandhaivel  Prabha</span>, <span class="hlFld-ContribAuthor ">Deval  Sunny</span>, <span class="hlFld-ContribAuthor ">Subbiah  Madhuri</span>, <span class="hlFld-ContribAuthor ">Kuppannagounder P.  Elango</span>. </span><span class="cited-content_cbyCitation_article-title">Preparation and biological evaluation of quinoline amines as anticancer agents and its molecular docking. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2019,</strong> <em>28 </em>
                                    (8)
                                     , 1298-1307. <a href="https://doi.org/10.1007/s00044-019-02374-w" title="DOI URL">https://doi.org/10.1007/s00044-019-02374-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-019-02374-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-019-02374-w%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DPreparation%252Band%252Bbiological%252Bevaluation%252Bof%252Bquinoline%252Bamines%252Bas%252Banticancer%252Bagents%252Band%252Bits%252Bmolecular%252Bdocking%26aulast%3DVennila%26aufirst%3DKailasam%2BN.%26date%3D2019%26date%3D2019%26volume%3D28%26issue%3D8%26spage%3D1298%26epage%3D1307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Obdulia  Rabal</span>, <span class="hlFld-ContribAuthor ">Andrea  Castellar</span>, <span class="hlFld-ContribAuthor ">Julen  Oyarzabal</span>. </span><span class="cited-content_cbyCitation_article-title">Novel pharmacological maps of protein lysine methyltransferases: key for target deorphanization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2018,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-018-0288-5" title="DOI URL">https://doi.org/10.1186/s13321-018-0288-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-018-0288-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-018-0288-5%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DNovel%252Bpharmacological%252Bmaps%252Bof%252Bprotein%252Blysine%252Bmethyltransferases%25253A%252Bkey%252Bfor%252Btarget%252Bdeorphanization%26aulast%3DRabal%26aufirst%3DObdulia%26date%3D2018%26date%3D2018%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edgar  López-López</span>, <span class="hlFld-ContribAuthor ">Fernando  Prieto-Martínez</span>, <span class="hlFld-ContribAuthor ">José  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Activity Landscape and Molecular Modeling to Explore the SAR of Dual Epigenetic Inhibitors: A Focus on G9a and DNMT1. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (12)
                                     , 3282. <a href="https://doi.org/10.3390/molecules23123282" title="DOI URL">https://doi.org/10.3390/molecules23123282</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23123282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23123282%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DActivity%252BLandscape%252Band%252BMolecular%252BModeling%252Bto%252BExplore%252Bthe%252BSAR%252Bof%252BDual%252BEpigenetic%252BInhibitors%25253A%252BA%252BFocus%252Bon%252BG9a%252Band%252BDNMT1%26aulast%3DL%25C3%25B3pez-L%25C3%25B3pez%26aufirst%3DEdgar%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D12%26spage%3D3282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Known G9a Inhibitors (<b>1</b>, <b>2</b>), Nucleoside DNMT1 Inhibitors (<b>3</b>, <b>4</b>), and Inhibitor (<b>5</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>G9a IC<sub>50</sub> values of <b>1</b>, <b>3</b>, <b>4</b>, and <b>5</b> and DNMT1 IC<sub>50</sub> values for <b>1</b> and <b>5</b> as determined internally (see footnote in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Alternatively, the reported IC<sub>50</sub> value for compound <b>1</b> (UNC-0638) was <15 nM (G9a, using a SAHH-coupled assay) and 1287 nM (DNMT1, using a radioactive methyl transfer assay).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Discrepancies with initially reported DNMT1 IC<sub>50</sub> values for <b>1</b> (IC<sub>50</sub> = 107000 nM) have been attributed to a different assay format.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> IC<sub>50</sub> values for compound <b>2</b><a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> (G9a, DNMT1), <b>3</b>,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and  <b>4</b><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> (DNMT1, corresponding to the lower concentration at which DNA hypomethylation is observed) have been previously reported.</p></p></figure><figure data-id="cht2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Chart 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 2. Sequential SAR Exploration Strategy for Positions 4, 6, and 7 of the Quinoline Scaffold; Position 2 Is Fixed with a 2-Furyl Ring Substituted at Position 5 (where R<sub>2</sub> is Methyl or Ethyl)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=cht2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0003.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Predicted complex of <b>6</b> with G9a (A, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW">3RJW</a>) and DNMT1 (B, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DA4">4DA4</a>). (C) Electrostatic potential of DNMT1 (only negative, in red), especially at the regions accommodating both basic centers of compound <b>6</b>. (D) Overlaid conformations of docked spirocyclic and bridged derivatives in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> to compound <b>5</b> (violet), showing that the basic nitrogen of compounds <b>20</b>–<b>22</b> (green, all stereoisomers considered) lies distal to the basic nitrogen of the rest of spirocycles and bridged structures (<b>17</b>–<b>19</b>, <b>23</b>–<b>26</b>, orange). (E) Overlaid conformations of docked compounds <b>5</b>, <b>41</b>, <b>43</b>, and <b>44</b> showing that the basic nitrogen overlays well among them.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0004.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. H3K9me2 and 5mC hallmarks in the CEMO-1 cell line after treatment with compounds <b>13</b> and <b>43</b>. (A) H3K9me2 levels after 96 h of treatment with different doses of compounds <b>13</b> and <b>43</b> in CEMO-1 cell line. H3 total was used as loading control. (B) 5mC levels after 96 h of treatment with different doses of compounds <b>13</b> and <b>43</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0005.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>7</b>–<b>11</b>, <b>13</b>, and <b>16</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) 3-pyrrolidin-1-yl-propan-1-ol, PPh<sub>3</sub>, DEAD, THF, rt, 5 h; (ii) Pd/C, H<sub>2</sub> (1 atm), MeOH, rt, 3 h; (iii) POCl<sub>3</sub>, malonic acid, rt, 4 h, then, 90 °C, overnight; (iv) 2-(5-ethyl-2-furyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane or 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane, Na<sub>2</sub>CO<sub>3</sub> or K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane, or 1,4-dioxane/H<sub>2</sub>O (15:1), 110 °C, MW or conventional heating, 4–12 h; (v) corresponding amine, Cs<sub>2</sub>CO<sub>3</sub>, BINAP, Pd<sub>2</sub>(dba)<sub>3</sub>, 1,4-dioxane, 120–130 °C, MW or conventional heating, 3–12 h; (vi) corresponding amine, <i>t</i>-BuONa, xantphos, Pd<sub>2</sub>(dba)<sub>3</sub>, toluene, 100 °C, MW, 2 h.</p></p></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0006.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>14</b>, <b>15</b>, and <b>17</b>–<b>27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) corresponding amine, Cs<sub>2</sub>CO<sub>3</sub>, BINAP, Pd<sub>2</sub>(dba)<sub>3</sub>, 1,4-dioxane, 115–130 °C, MW or conventional heating, 5–48 h; (ii) <i>tert</i>-butyl 4-(methylamino)piperidine-1-carboxylate, PTSA, <i>t</i>-BuOH, 120 °C, 48 h; (iii) HCl/EtOAc (1.0 or 2.0 M) or HCl/MeOH (2.0 or 4.0 M), 16–25 °C, 1–16 h; (iv) acetone, NaBH<sub>3</sub>CN, AcOH, <i>i</i>-PrOH, 50–60 °C, 15–16 h; (v) (HCHO)<sub>n</sub>, HCOOH, NaBH(OAc)<sub>3</sub>, MeOH, rt or 60 °C, 12 h or overnight.</p></p></figure><figure data-id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0007.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>29</b>, <b>31</b>, and <b>32</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) 4,4,5,5-tetramethyl-2-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1,3,2-dioxaborolane, KOAc, Pd(dppf)Cl<sub>2</sub>, 1,4-dioxane, 90 °C, 10 h; (ii) NaOH, H<sub>2</sub>O<sub>2</sub>, THF, 30 min, 22 °C; (iii) 3-pyrrolidin-1-ylpropan-1-ol, PPh<sub>3</sub>, DEAD or DIAD, THF, 0 °C or rt, 5–10 h; (iv) Pd/C, H<sub>2</sub> (40 psi), MeOH, rt, 2–15 h; (v) Fe, NH<sub>4</sub>Cl, EtOH/H<sub>2</sub>O (4:1), rt, 3 h; (vi) POCl<sub>3</sub>, malonic acid, rt, 4 h, then 90 °C, overnight; (vii) DMAP, pyridine, CH<sub>2</sub>Cl<sub>2</sub>, −70 °C, 1 h, then ethyl 3-chloro-3-oxo-propanoate, 20 °C, 15 h; (viii) LiOH·H<sub>2</sub>O, THF/MeOH/H<sub>2</sub>O (4:2:3), 20 °C, 15 h; (ix) POCl<sub>3</sub>, 100 °C, 3 h; (x) 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane, Na<sub>2</sub>CO<sub>3</sub>, or K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane/H<sub>2</sub>O (5:1, 15:1 or 10:1), 80–110 °C, MW or conventional heating, 2–15 h; (xi) 1-methylpiperidin-4-amine, x-Phos, Pd<sub>2</sub>(dba)<sub>3</sub>, <i>t</i>-BuOK, 1,4-dioxane, 130 °C, MW, 4 h; (xii) 1-methylpiperidin-4-amine, BINAP, Pd<sub>2</sub>(dba)<sub>3</sub>, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 80–110 °C, 12–15 h.</p></p></figure><figure data-id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0008.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>30</b> and <b>33</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) BBr<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C, 2 h; (ii) PhN(OTf)<sub>2</sub>, DIEA, DMF, 0 °C, 2 h, then 25 °C, 12 h; (iii) Zn(CN)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, DMF, 110 °C, 12 h.</p></p></figure><figure data-id="sch5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0009.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>35</b>, <b>36</b>, and <b>39</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) POCl<sub>3</sub>, malonic acid, 95 °C, 12 h; (ii) 1,3-dibromopropane, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 60 °C, 88 h; (iii) corresponding amine, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 60 °C, 16 h; (iv) 3-(3-fluoropyrrolidin-1-yl)propan-1-ol, PPh<sub>3</sub>, DEAD, THF, 20 °C 16 h; (v) 2-(5-ethyl-2-furyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane or 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane/H<sub>2</sub>O (1:1 or 10:1), 110 °C, 16 h; (vi) 1-methylpiperidin-4-amine, Cs<sub>2</sub>CO<sub>3</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, 1,4-dioxane, 120 °C, 12–16 h.</p></p></figure><figure data-id="sch6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0010.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compounds <b>37</b> and <b>38</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) 1,3-dibromopropane, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 20 °C, 16 h; (ii) piperidine or morpholine, Cs<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 90 °C, 16 h; (iii) Pd/C, MeOH, H<sub>2</sub> (30 psi), 15 °C, 2 h; (iv) DMAP, pyridine, CH<sub>2</sub>Cl<sub>2</sub>, −78 °C, then ethyl 3-chloro-3-oxo-propanoate, 15 °C, 16 h; (v) LiOH·H<sub>2</sub>O, THF/MeOH/H<sub>2</sub>O (20:20:13 or 3:3:2), 15 °C, 16 h; (vi) POCl<sub>3</sub>, 100 °C, 2 h; (vii) 2-(5-ethyl-2-furyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane or 1,4-dioxane/H<sub>2</sub>O (1:1), 110 °C, 16 h; (viii) 1-methylpiperidin-4-amine, Cs<sub>2</sub>CO<sub>3</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, 1,4-dioxane, 120 °C, 16 h.</p></p></figure><figure data-id="sch7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0011.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Compounds <b>34</b>, <b>40</b>–<b>45</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) POCl<sub>3</sub>, malonic acid, rt, 4 h, then 90 °C, overnight or 95 °C, 12 h; (ii) Et<sub>3</sub>N, ethyl 3-chloro-3-oxo-propanoate, CH<sub>2</sub>Cl<sub>2</sub>, 25 °C, 12 h; (iii) LiOH·H<sub>2</sub>O, THF/MeOH/H<sub>2</sub>O (3:3:2), 25 °C, 16 h; (iv) POCl<sub>3</sub>, 90 °C, 2 h; (v) 4,4,5,5-tetramethyl-2-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1,3,2-dioxaborolane, Na<sub>2</sub>CO<sub>3</sub>, or K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane, or 1,4-dioxane/H<sub>2</sub>O (15:1), 100–120 °C, MW or conventional heating, 2–16 h; (vi) 1-methylpiperidin-4-amine, Cs<sub>2</sub>CO<sub>3</sub>, BINAP, Pd<sub>2</sub>(dba)<sub>3</sub>, 1,4-dioxane, 110–120 °C, MW or conventional heating, 3–16h; (vii) Pd/C, H<sub>2</sub> (50 psi), MeOH, 25 °C, 16 h; (viii) <i>tert</i>-butyl 4-(hydroxymethyl) piperidine-1-carboxylate, PPh<sub>3</sub>, DIAD, THF, 0 °C, 8 h; (ix) HCl/EtOAc (1.0 or 2.0 M), 25 °C, 2–3 h; (x) (HCHO)<sub><i>n</i></sub>, NaBH(OAc)<sub>3</sub>, HCOOH, 1,4-dioxane, 100 °C, 2 h; (xi) <i>tert</i>-butyl 4-methylsulfonyloxypiperidine-1-carboxylate or <i>tert</i>-butyl 6-methylsulfonyloxy-2-azaspiro[3.3]heptane-2-carboxylate, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 100 °C, 16 h; (xii) LiAlH<sub>4</sub>, THF, 70 °C, 16 h; (xiii) TFA, CH<sub>2</sub>Cl<sub>2</sub>, 18 °C, 2 h; (xiv) acetone or (1-ethoxycyclopropoxy)-trimethyl-silane, NaBH<sub>3</sub>CN, AcOH, <i>i</i>-PrOH or <i>t</i>-BuOH, 60 °C, 16 h.</p></p></figure><figure data-id="sch8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0012.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of Compounds <b>46</b>–<b>48</b> and <b>50</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) <i>tert</i>-butyl 4-(hydroxymethyl) piperidine-1-carboxylate, PPh<sub>3</sub>, DIAD, THF, 0 °C, 8 h; (ii) corresponding amine, Cs<sub>2</sub>CO<sub>3</sub>, BINAP, Pd<sub>2</sub>(dba)<sub>3</sub>, or Pd(dba)<sub>2</sub>, 1,4-dioxane, 110–130 °C, 12–16 h; (iii) Pd/C, MeOH, H<sub>2</sub> (50 Psi), 25 °C, 16 h; (iv) <i>tert</i>-butyl 6-methylsulfonyloxy-2-azaspiro[3.3]heptane-2-carboxylate, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 100 °C, 16 h; (v) HCl/EtOAc (2.0 M), 25 °C, 2 h; vi) TFA, CH<sub>2</sub>Cl<sub>2</sub>, 18 °C, 2 h; vii) (HCHO)<sub>n</sub>, NaBH<sub>3</sub>CN or NaBH(OAc)<sub>3</sub>, HCOOH, MeOH, 60–70 °C, 12–16 h.</p></p></figure><figure data-id="sch9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/medium/jm-2017-019256_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0013.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of Compound <b>49</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-15/acs.jmedchem.7b01925/20180802/images/large/jm-2017-019256_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01925&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) HCl/EtOAc (2.0 M), 18 °C, 2 h; (ii) acetone, NaBH<sub>3</sub>CN, AcOH, <i>i</i>-PrOH, 60 °C, 16 h; (iii) Pd/C, MeOH, H<sub>2</sub> (50 psi), 20 °C, 8 h; (iv) <i>tert</i>-butyl 6-methylsulfonyloxy-2-azaspiro[3.3]heptane-2-carboxylate, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 100 °C, 16 h; (v) TFA, DCM, 18 °C, 2 h; (vi) (HCHO)<sub><i>n</i></sub>, NaBH(OAc)<sub>3</sub>, HCOOH, MeOH, 60 °C, 16 h.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i168">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07221" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07221" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 24 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Portela, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteller, M.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Modifications and Human Disease</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1057</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1038/nbt.1685</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1038%2Fnbt.1685" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=20944598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yjsLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=1057-1068&issue=10&author=A.+Portelaauthor=M.+Esteller&title=Epigenetic+Modifications+and+Human+Disease&doi=10.1038%2Fnbt.1685"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic modifications and human disease</span></div><div class="casAuthors">Portela, Anna; Esteller, Manel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1057-1068</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetics is one of the most rapidly expanding fields in biol.  The recent characterization of a human DNA methylome at single nucleotide resoln., the discovery of the CpG island shores, the finding of new histone variants and modifications, and the unveiling of genome-wide nucleosome positioning maps highlight the accelerating speed of discovery over the past two years.  Increasing interest in epigenetics has been accompanied by technol. breakthroughs that now make it possible to undertake large-scale epigenomic studies.  These allow the mapping of epigenetic marks, such as DNA methylation, histone modifications and nucleosome positioning, which are crit. for regulating gene and noncoding RNA expression.  In turn, we are learning how aberrant placement of these epigenetic marks and mutations in the epigenetic machinery is involved in disease.  Thus, a comprehensive understanding of epigenetic mechanisms, their interactions and alterations in health and disease, has become a priority in biomedical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmU-oOba01dbVg90H21EOLACvtfcHk0lh8Zt-y1Qo3SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yjsLzF&md5=02649d0c2ed862eda90d7bd349871c71</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1685%26sid%3Dliteratum%253Aachs%26aulast%3DPortela%26aufirst%3DA.%26aulast%3DEsteller%26aufirst%3DM.%26atitle%3DEpigenetic%2520Modifications%2520and%2520Human%2520Disease%26jtitle%3DNat.%2520Biotechnol.%26date%3D2010%26volume%3D28%26issue%3D10%26spage%3D1057%26epage%3D1068%26doi%3D10.1038%2Fnbt.1685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">Marked for Death: Targeting Epigenetic Changes in Cancer</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">241</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1038%2Fnrd.2016.256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=28280262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1Ontr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=241-263&issue=4&author=S.+X.+Pfisterauthor=A.+Ashworth&title=Marked+for+Death%3A+Targeting+Epigenetic+Changes+in+Cancer&doi=10.1038%2Fnrd.2016.256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Marked for death: targeting epigenetic changes in cancer</span></div><div class="casAuthors">Pfister, Sophia Xiao; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">241-263</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In the past few years, it has become clear that mutations in epigenetic regulatory genes are common in human cancers.  Therapeutic strategies are now being developed to target cancers with mutations in these genes using specific chem. inhibitors.  In addn., a complementary approach based on the concept of synthetic lethality, which allows exploitation of loss-of-function mutations in cancers that are not targetable by conventional methods, has gained traction.  Both of these approaches are now being tested in several clin. trials.  In this Review, we present recent advances in epigenetic drug discovery and development, and suggest possible future avenues of investigation to drive progress in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP-s0n0CRO2rVg90H21EOLACvtfcHk0lh8Zt-y1Qo3SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1Ontr8%253D&md5=f028b0c2e596b7c358e2158318a1303c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.256%26sid%3Dliteratum%253Aachs%26aulast%3DPfister%26aufirst%3DS.%2BX.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DMarked%2520for%2520Death%253A%2520Targeting%2520Epigenetic%2520Changes%2520in%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26issue%3D4%26spage%3D241%26epage%3D263%26doi%3D10.1038%2Fnrd.2016.256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorsey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuikov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenuwein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. L.</span></span> <span> </span><span class="NLM_article-title">G9a and Glp Methylate Lysine 373 in the Tumor Suppressor p53</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">9636</span>– <span class="NLM_lpage">9641</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.062588</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1074%2Fjbc.M109.062588" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=9636-9641&issue=13&author=J.+Huangauthor=J.+Dorseyauthor=S.+Chuikovauthor=X.+Zhangauthor=T.+Jenuweinauthor=D.+Reinbergauthor=S.+L.+Berger&title=G9a+and+Glp+Methylate+Lysine+373+in+the+Tumor+Suppressor+p53&doi=10.1074%2Fjbc.M109.062588"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.062588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.062588%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDorsey%26aufirst%3DJ.%26aulast%3DChuikov%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DReinberg%26aufirst%3DD.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DG9a%2520and%2520Glp%2520Methylate%2520Lysine%2520373%2520in%2520the%2520Tumor%2520Suppressor%2520p53%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26issue%3D13%26spage%3D9636%26epage%3D9641%26doi%3D10.1074%2Fjbc.M109.062588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Casciello, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Windloch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gannon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. S.</span></span> <span> </span><span class="NLM_article-title">Functional Role of G9a Histone Methyltransferase in Cancer</span>. <i>Front. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">487</span>, <span class="refDoi"> DOI: 10.3389/fimmu.2015.00487</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.3389%2Ffimmu.2015.00487" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=26441991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmt1Glsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=487&author=F.+Cascielloauthor=K.+Windlochauthor=F.+Gannonauthor=J.+S.+Lee&title=Functional+Role+of+G9a+Histone+Methyltransferase+in+Cancer&doi=10.3389%2Ffimmu.2015.00487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Functional role of G9a histone methyltransferase in cancer</span></div><div class="casAuthors">Casciello, Francesco; Windloch, Karolina; Gannon, Frank; Lee, Jason S.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">487/1-487/12</span>CODEN:
                <span class="NLM_cas:coden">FIRMCW</span>;
        ISSN:<span class="NLM_cas:issn">1664-3224</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Post-translational modifications of DNA and histones are epigenetic mechanisms, which affect the chromatin structure, ultimately leading to gene expression changes.  A no. of different epigenetic enzymes are actively involved in the addn. or the removal of various covalent modifications, which include acetylation, methylation, phosphorylation, ubiquitination, and sumoylation.  Deregulation of these processes is a hallmark of can-cer.  For instance, G9a, a histone methyltransferase responsible for histone H3 lysine 9 (H3K9) mono- and dimethylation, has been obsd. to be upregulated in different types of cancer and its overexpression has been assocd. with poor prognosis.  Key roles played by these enzymes in various diseases have led to the hypothesis that these mols. represent valuable targets for future therapies.  Several small mol. inhibi-tors have been developed to specifically block the epigenetic activity of these enzymes, representing promising therapeutic tools in the treatment of human malignancies, such as cancer.  In this review, the role of one of these epigenetic enzymes, G9a, is discussed, focusing on its functional role in regulating gene expression as well as its implications in cancer initiation and progression.  We also discuss important findings from recent studies using epigenetic inhibitors in cell systems in vitro as well as exptl. tumor growth and metastasis assays in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptTH4mE94-t7Vg90H21EOLACvtfcHk0ljFL9SXYP9vSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmt1Glsbk%253D&md5=74bd529b0e0c55e016075fadedebc7e1</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2015.00487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2015.00487%26sid%3Dliteratum%253Aachs%26aulast%3DCasciello%26aufirst%3DF.%26aulast%3DWindloch%26aufirst%3DK.%26aulast%3DGannon%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DJ.%2BS.%26atitle%3DFunctional%2520Role%2520of%2520G9a%2520Histone%2520Methyltransferase%2520in%2520Cancer%26jtitle%3DFront.%2520Immunol.%26date%3D2015%26volume%3D6%26spage%3D487%26doi%3D10.3389%2Ffimmu.2015.00487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Estève, P.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smallwood, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feehery, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangisetty, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karpf, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carey, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradhan, S.</span></span> <span> </span><span class="NLM_article-title">Direct Interaction between DNMT1 and G9a Coordinates DNA and Histone Methylation during Replication</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">3089</span>– <span class="NLM_lpage">3103</span>, <span class="refDoi"> DOI: 10.1101/gad.1463706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1101%2Fgad.1463706" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2006&pages=3089-3103&issue=22&author=P.-O.+Est%C3%A8veauthor=H.+G.+Chinauthor=A.+Smallwoodauthor=G.+R.+Feeheryauthor=O.+Gangisettyauthor=A.+R.+Karpfauthor=M.+F.+Careyauthor=S.+Pradhan&title=Direct+Interaction+between+DNMT1+and+G9a+Coordinates+DNA+and+Histone+Methylation+during+Replication&doi=10.1101%2Fgad.1463706"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1101%2Fgad.1463706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1463706%26sid%3Dliteratum%253Aachs%26aulast%3DEst%25C3%25A8ve%26aufirst%3DP.-O.%26aulast%3DChin%26aufirst%3DH.%2BG.%26aulast%3DSmallwood%26aufirst%3DA.%26aulast%3DFeehery%26aufirst%3DG.%2BR.%26aulast%3DGangisetty%26aufirst%3DO.%26aulast%3DKarpf%26aufirst%3DA.%2BR.%26aulast%3DCarey%26aufirst%3DM.%2BF.%26aulast%3DPradhan%26aufirst%3DS.%26atitle%3DDirect%2520Interaction%2520between%2520DNMT1%2520and%2520G9a%2520Coordinates%2520DNA%2520and%2520Histone%2520Methylation%2520during%2520Replication%26jtitle%3DGenes%2520Dev.%26date%3D2006%26volume%3D20%26issue%3D22%26spage%3D3089%26epage%3D3103%26doi%3D10.1101%2Fgad.1463706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">San
José-Enériz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agirre, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilas-Zornoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paiva, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estella-Hermoso
de Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casares, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Subero, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogi, F.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soule, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiveri, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos-Olivas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia Fernandez de Barrena, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Madoz, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barchino, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasarte, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avila, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez-Climent, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosper, F.</span></span> <span> </span><span class="NLM_article-title">Discovery of First-in-Class Reversible Dual Small Molecule Inhibitors against G9a and DNMTs with in Vivo Activity in Hematological Malignancies</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">15424</span>, <span class="refDoi"> DOI: 10.1038/ncomms15424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1038%2Fncomms15424" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=28548080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1egs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=15424&author=E.+San%0AJos%C3%A9-En%C3%A9rizauthor=X.+Agirreauthor=O.+Rabalauthor=A.+Vilas-Zornozaauthor=J.+A.+S%C3%A1nchez-Ariasauthor=E.+Mirandaauthor=A.+Ugarteauthor=S.+Roaauthor=B.+Paivaauthor=A.+Estella-Hermoso%0Ade+Mendozaauthor=R.+M.+Alvarezauthor=N.+Casaresauthor=V.+Seguraauthor=J.+I.+Mart%C3%ADn-Suberoauthor=F.-X.+Ogiauthor=P.+Souleauthor=C.+M.+Santiveriauthor=R.+Campos-Olivasauthor=G.+Castellanoauthor=M.+Garcia+Fernandez+de+Barrenaauthor=J.+R.+Rodriguez-Madozauthor=M.+J.+Garc%C3%ADa-Barchinoauthor=J.+J.+Lasarteauthor=M.+A.+Avilaauthor=J.+A.+Martinez-Climentauthor=J.+Oyarzabalauthor=F.+Prosper&title=Discovery+of+First-in-Class+Reversible+Dual+Small+Molecule+Inhibitors+against+G9a+and+DNMTs+with+in+Vivo+Activity+in+Hematological+Malignancies&doi=10.1038%2Fncomms15424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of first-in-class reversible dual small molecule inhibitors against G9a and DNMTs in hematological malignancies</span></div><div class="casAuthors">San Jose-Eneriz, Edurne; Agirre, Xabier; Rabal, Obdulia; Vilas-Zornoza, Amaia; Sanchez-Arias, Juan A.; Miranda, Estibaliz; Ugarte, Ana; Roa, Sergio; Paiva, Bruno; Estella-Hermoso de Mendoza, Ander; Alvarez, Rosa Maria; Casares, Noelia; Segura, Victor; Martin-Subero, Jose I.; Ogi, Francois-Xavier; Soule, Pierre; Santiveri, Clara M.; Campos-Olivas, Ramon; Castellano, Giancarlo; Fernandez de Barrena, Maite Garcia; Rodriguez-Madoz, Juan Roberto; Garcia-Barchino, Maria Jose; Lasarte, Juan Jose; Avila, Matias A.; Martinez-Climent, Jose Angel; Oyarzabal, Julen; Prosper, Felipe</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15424</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The indisputable role of epigenetics in cancer and the fact that epigenetic alterations can be reversed have favored development of epigenetic drugs.  In this study, we design and synthesize potent novel, selective and reversible chem. probes that simultaneously inhibit the G9a and DNMTs methyltransferase activity.  In vitro treatment of haematol. neoplasia (acute myeloid leukemia-AML, acute lymphoblastic leukemia-ALL and diffuse large B-cell lymphoma-DLBCL) with the lead compd. CM-272, inhibits cell proliferation and promotes apoptosis, inducing interferon-stimulated genes and immunogenic cell death.  CM-272 significantly prolongs survival of AML, ALL and DLBCL xenogeneic models.  Our results represent the discovery of first-in-class dual inhibitors of G9a/DNMTs and establish this chem. series as a promising therapeutic tool for unmet needs in haematol. tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpstz1QqrJd4rVg90H21EOLACvtfcHk0lgUC9tTVxyB6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1egs7Y%253D&md5=6d37a6433cdb371bfb535b1fd8a7a803</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fncomms15424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms15424%26sid%3Dliteratum%253Aachs%26aulast%3DSan%2BJos%25C3%25A9-En%25C3%25A9riz%26aufirst%3DE.%26aulast%3DAgirre%26aufirst%3DX.%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DVilas-Zornoza%26aufirst%3DA.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DMiranda%26aufirst%3DE.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DRoa%26aufirst%3DS.%26aulast%3DPaiva%26aufirst%3DB.%26aulast%3DEstella-Hermoso%2Bde%2BMendoza%26aufirst%3DA.%26aulast%3DAlvarez%26aufirst%3DR.%2BM.%26aulast%3DCasares%26aufirst%3DN.%26aulast%3DSegura%26aufirst%3DV.%26aulast%3DMart%25C3%25ADn-Subero%26aufirst%3DJ.%2BI.%26aulast%3DOgi%26aufirst%3DF.-X.%26aulast%3DSoule%26aufirst%3DP.%26aulast%3DSantiveri%26aufirst%3DC.%2BM.%26aulast%3DCampos-Olivas%26aufirst%3DR.%26aulast%3DCastellano%26aufirst%3DG.%26aulast%3DGarcia%2BFernandez%2Bde%2BBarrena%26aufirst%3DM.%26aulast%3DRodriguez-Madoz%26aufirst%3DJ.%2BR.%26aulast%3DGarc%25C3%25ADa-Barchino%26aufirst%3DM.%2BJ.%26aulast%3DLasarte%26aufirst%3DJ.%2BJ.%26aulast%3DAvila%26aufirst%3DM.%2BA.%26aulast%3DMartinez-Climent%26aufirst%3DJ.%2BA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26aulast%3DProsper%26aufirst%3DF.%26atitle%3DDiscovery%2520of%2520First-in-Class%2520Reversible%2520Dual%2520Small%2520Molecule%2520Inhibitors%2520against%2520G9a%2520and%2520DNMTs%2520with%2520in%2520Vivo%2520Activity%2520in%2520Hematological%2520Malignancies%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D15424%26doi%3D10.1038%2Fncomms15424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerke, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stallcup, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, G.</span></span> <span> </span><span class="NLM_article-title">Lysine Methyltransferase G9a Is Not Required for DNMT3A/3B Anchoring to Methylated Nucleosomes and Maintenance of DNA Methylation in Somatic Cells</span>. <i>Epigenet. Chromatin</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">3</span>, <span class="refDoi"> DOI: 10.1186/1756-8935-5-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1186%2F1756-8935-5-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=22284370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVaitbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=3&issue=1&author=S.+Sharmaauthor=D.+S.+Gerkeauthor=H.+F.+Hanauthor=S.+Jeongauthor=M.+R.+Stallcupauthor=P.+A.+Jonesauthor=G.+Liang&title=Lysine+Methyltransferase+G9a+Is+Not+Required+for+DNMT3A%2F3B+Anchoring+to+Methylated+Nucleosomes+and+Maintenance+of+DNA+Methylation+in+Somatic+Cells&doi=10.1186%2F1756-8935-5-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine methyltransferase G9a is not required for DNMT3A/3B anchoring to methylated nucleosomes and maintenance of DNA methylation in somatic cells</span></div><div class="casAuthors">Sharma, Shikhar; Gerke, Daniel S.; Han, Han F.; Jeong, Shinwu; Stallcup, Michael R.; Jones, Peter A.; Liang, Gangning</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics & Chromatin</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3</span>CODEN:
                <span class="NLM_cas:coden">ECPHAK</span>;
        ISSN:<span class="NLM_cas:issn">1756-8935</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: DNA methylation, histone modifications and nucleosome occupancy act in concert for regulation of gene expression patterns in mammalian cells.  Recently, G9a, a H3K9 methyltransferase, has been shown to play a role in establishment of DNA methylation at embryonic gene targets in ES cells through recruitment of de novo DNMT3A/3B enzymes.  However, whether G9a plays a similar role in maintenance of DNA methylation in somatic cells is still unclear.  Results: Here we show that G9a is not essential for maintenance of DNA methylation in somatic cells.  Knockdown of G9a has no measurable effect on DNA methylation levels at G9a-target loci.  DNMT3A/3B remain stably anchored to nucleosomes contg. methylated DNA even in the absence of G9a, ensuring faithful propagation of methylated states in cooperation with DNMT1 through somatic divisions.  Moreover, G9a also assocs. with nucleosomes in a DNMT3A/3B and DNA methylation-independent manner.  However, G9a knockdown synergizes with pharmacol. inhibition of DNMTs resulting in increased hypomethylation and inhibition of cell proliferation.  Conclusions: Taken together, these data suggest that G9a is not involved in maintenance of DNA methylation in somatic cells but might play a role in re-initiation of de novo methylation after treatment with hypomethylating drugs, thus serving as a potential target for combinatorial treatments strategies involving DNMTs inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG7H2E7h_WpLVg90H21EOLACvtfcHk0lgUC9tTVxyB6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVaitbo%253D&md5=d4c61a81e34cabcb684be6b4a7dab495</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2F1756-8935-5-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8935-5-3%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DGerke%26aufirst%3DD.%2BS.%26aulast%3DHan%26aufirst%3DH.%2BF.%26aulast%3DJeong%26aufirst%3DS.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DLiang%26aufirst%3DG.%26atitle%3DLysine%2520Methyltransferase%2520G9a%2520Is%2520Not%2520Required%2520for%2520DNMT3A%252F3B%2520Anchoring%2520to%2520Methylated%2520Nucleosomes%2520and%2520Maintenance%2520of%2520DNA%2520Methylation%2520in%2520Somatic%2520Cells%26jtitle%3DEpigenet.%2520Chromatin%26date%3D2012%26volume%3D5%26issue%3D1%26spage%3D3%26doi%3D10.1186%2F1756-8935-5-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Protein Methyltransferases and Demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+%C3%9C.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+Protein+Methyltransferases+and+Demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0ljrT6uF7H1ZZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520Protein%2520Methyltransferases%2520and%2520Demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rival-Gervier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labrie, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimaggio, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasney, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siarheyeva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangano, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattenden, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petronis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">A Chemical Probe Selectively Inhibits G9a and GLP Methyltransferase Activity in Cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">566</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1038/nchembio.599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&issue=8&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+Dimaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.-C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+J.+Manganoauthor=X.-P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+Chemical+Probe+Selectively+Inhibits+G9a+and+GLP+Methyltransferase+Activity+in+Cells&doi=10.1038%2Fnchembio.599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0ljrT6uF7H1ZZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDimaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.-C.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520Chemical%2520Probe%2520Selectively%2520Inhibits%2520G9a%2520and%2520GLP%2520Methyltransferase%2520Activity%2520in%2520Cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26issue%3D8%26spage%3D566%26epage%3D574%26doi%3D10.1038%2Fnchembio.599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sweis, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pliushchev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maag, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappano, W. N.</span></span> <span> </span><span class="NLM_article-title">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase g9a</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1021/ml400496h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400496h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=205-209&issue=2&author=R.+F.+Sweisauthor=M.+Pliushchevauthor=P.+J.+Brownauthor=J.+Guoauthor=F.+Liauthor=D.+Maagauthor=A.+M.+Petrosauthor=N.+B.+Soniauthor=C.+Tseauthor=M.+Vedadiauthor=M.+R.+Michaelidesauthor=G.+G.+Chiangauthor=W.+N.+Pappano&title=Discovery+and+Development+of+Potent+and+Selective+Inhibitors+of+Histone+Methyltransferase+g9a&doi=10.1021%2Fml400496h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase G9a</span></div><div class="casAuthors">Sweis, Ramzi F.; Pliushchev, Marina; Brown, Peter J.; Guo, Jun; Li, Fengling; Maag, David; Petros, Andrew M.; Soni, Nirupama B.; Tse, Chris; Vedadi, Masoud; Michaelides, Michael R.; Chiang, Gary G.; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-209</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G9a is a histone lysine methyltransferase responsible for the methylation of histone H3 lysine 9.  The discovery of A-366 arose from a unique diversity screening hit, which was optimized by incorporation of a propyl-pyrrolidine subunit to occupy the enzyme lysine channel.  A-366 (I) is a potent inhibitor of G9a (IC50: 3.3 nM) with greater than 1000-fold selectivity over 21 other methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMvdTiUgx-oLVg90H21EOLACvtfcHk0liEBuU98mGH1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D&md5=26affad72f2950510aa9f253e540d575</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fml400496h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400496h%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520and%2520Development%2520of%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520Histone%2520Methyltransferase%2520g9a%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26issue%3D2%26spage%3D205%26epage%3D209%26doi%3D10.1021%2Fml400496h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erdmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halby, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fahy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span> <span> </span><span class="NLM_article-title">Targeting DNA Methylation with Small Molecules: What’s Next?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">2569</span>– <span class="NLM_lpage">2583</span>, <span class="refDoi"> DOI: 10.1021/jm500843d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500843d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2569-2583&issue=6&author=A.+Erdmannauthor=L.+Halbyauthor=J.+Fahyauthor=P.+B.+Arimondo&title=Targeting+DNA+Methylation+with+Small+Molecules%3A+What%E2%80%99s+Next%3F&doi=10.1021%2Fjm500843d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm500843d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500843d%26sid%3Dliteratum%253Aachs%26aulast%3DErdmann%26aufirst%3DA.%26aulast%3DHalby%26aufirst%3DL.%26aulast%3DFahy%26aufirst%3DJ.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26atitle%3DTargeting%2520DNA%2520Methylation%2520with%2520Small%2520Molecules%253A%2520What%25E2%2580%2599s%2520Next%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D6%26spage%3D2569%26epage%3D2583%26doi%3D10.1021%2Fjm500843d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span> <span> </span><span class="NLM_article-title">DNA Methyltransferase Inhibitors: An Updated Patent Review (2012–2015)</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1017</span>– <span class="NLM_lpage">1030</span>, <span class="refDoi"> DOI: 10.1080/13543776.2016.1209488</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1080%2F13543776.2016.1209488" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=27376512" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFyhtLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=1017-1030&issue=9&author=P.+Xuauthor=G.+Huauthor=C.+Luoauthor=Z.+Liang&title=DNA+Methyltransferase+Inhibitors%3A+An+Updated+Patent+Review+%282012%E2%80%932015%29&doi=10.1080%2F13543776.2016.1209488"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">DNA methyltransferase inhibitors: an updated patent review (2012-2015)</span></div><div class="casAuthors">Xu, Pan; Hu, Guang; Luo, Cheng; Liang, Zhongjie</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1017-1030</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  DNA methyltransferases (DNMTs), important enzymes involved in epigenetic regulation of gene expression, represent promising targets in cancer therapy.  DNMT inhibitors (DNMTi), which can modulate the aberrant DNA methylation pattern in a reversible way via inhibiting DNMT activity, have attracted significant attention in recent years.  This review outlines the newly patented inhibitors targeting DNMTs, mainly incorporating small mol. inhibitors and oligonucleotide derivs.  The chem. structures, biol. activity, and the encouraging clin. research in progress are delineated in detail.  Two drugs, azacitidine and decitabine, have evidently shown efficacy in hematol. malignancies, yet do not work well on solid tumors, have low specificity, substantial toxicity, and poor bioavailability.  With the rapid advancement in systems biol., drug combinations, such as DNMTi, in conjugation with histone deacetylase inhibitors (HDACi) or immunotherapy, probably serve as an efficient way of implementing epigenetic therapy.  Meanwhile, the resolved autoinhibitory structures of DNMTs afford a novel strategy for targeting the protein-protein interface involved in the autoinhi-bitory interactions.  The mol. mechanism underlying the conformational transitions would also shed new light on the design of allosteric inhibitors.  Both strategies would produce inhibitors with more selectivity compared to nucleotide derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFxx109NDo-rVg90H21EOLACvtfcHk0liEBuU98mGH1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFyhtLnP&md5=fdfc78375967b9ab9b5fa55d8519d75f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1080%2F13543776.2016.1209488&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2016.1209488%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DP.%26aulast%3DHu%26aufirst%3DG.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DLiang%26aufirst%3DZ.%26atitle%3DDNA%2520Methyltransferase%2520Inhibitors%253A%2520An%2520Updated%2520Patent%2520Review%2520%25282012%25E2%2580%25932015%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2016%26volume%3D26%26issue%3D9%26spage%3D1017%26epage%3D1030%26doi%3D10.1080%2F13543776.2016.1209488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span>; <span class="NLM_string-name">SanJosé-Enériz, E.</span>; <span class="NLM_string-name">Agirre, X.</span>; <span class="NLM_string-name">Sánchez-Arias, J. A.</span>; <span class="NLM_string-name">Vilas-Zornoza, A.</span>; <span class="NLM_string-name">Ugarte, A.</span>; <span class="NLM_string-name">de Miguel, I.</span>; <span class="NLM_string-name">Miranda, E.</span>; <span class="NLM_string-name">Garate, L.</span>; <span class="NLM_string-name">Fraga, M.</span>; <span class="NLM_string-name">Santamarina, P.</span>; <span class="NLM_string-name">Fernandez Perez, R.</span>; <span class="NLM_string-name">Ordoñez, R.</span>; <span class="NLM_string-name">Sáez, E.</span>; <span class="NLM_string-name">Roa, S.</span>; <span class="NLM_string-name">García-Barchino, M. J.</span>; <span class="NLM_string-name">Martínez-Climent, J. Á.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Wei, W.</span>; <span class="NLM_string-name">Musheng, X.</span>; <span class="NLM_string-name">Prosper, F.</span>; <span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Reversible DNAMethyltransferase and Lysine Methyltransferase G9a Inhibitors withAntitumoral in Vivo Efficacy</span>. In press, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01926</span> .</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Rabal%2C+O.%3B+SanJos%C3%A9-En%C3%A9riz%2C+E.%3B+Agirre%2C+X.%3B+S%C3%A1nchez-Arias%2C+J.+A.%3B+Vilas-Zornoza%2C+A.%3B+Ugarte%2C+A.%3B+de+Miguel%2C+I.%3B+Miranda%2C+E.%3B+Garate%2C+L.%3B+Fraga%2C+M.%3B+Santamarina%2C+P.%3B+Fernandez+Perez%2C+R.%3B+Ordo%C3%B1ez%2C+R.%3B+S%C3%A1ez%2C+E.%3B+Roa%2C+S.%3B+Garc%C3%ADa-Barchino%2C+M.+J.%3B+Mart%C3%ADnez-Climent%2C+J.+%C3%81.%3B+Liu%2C+Y.%3B+Wei%2C+W.%3B+Musheng%2C+X.%3B+Prosper%2C+F.%3B+Oyarzabal%2C+J.+Discovery+of+Reversible+DNAMethyltransferase+and+Lysine+Methyltransferase+G9a+Inhibitors+withAntitumoral+in+Vivo+Efficacy.+In+press%2C+10.1021%2Facs.jmedchem.7b01926."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dunpublished%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01926%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26atitle%3DDiscovery%2520of%2520Reversible%2520DNAMethyltransferase%2520and%2520Lysine%2520Methyltransferase%2520G9a%2520Inhibitors%2520withAntitumoral%2520in%2520Vivo%2520Efficacy%26doi%3D10.1021%2Facs.jmedchem.7b01926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span> <i>UNC0638 Selective Chemical Probe for G9a/GLP Methyltransferases</i>; <span class="NLM_publisher-name">SGC</span>, <span class="NLM_year">2016</span><a href="http://www.thesgc.org/chemical-probes/UNC0638" class="extLink">http://www.thesgc.org/chemical-probes/UNC0638</a> (accessed Dec 16, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+UNC0638+Selective+Chemical+Probe+for+G9a%2FGLP+Methyltransferases%3B+SGC%2C+2016http%3A%2F%2Fwww.thesgc.org%2Fchemical-probes%2FUNC0638+%28accessed+Dec+16%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DUNC0638%2520Selective%2520Chemical%2520Probe%2520for%2520G9a%252FGLP%2520Methyltransferases%26pub%3DSGC%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foulks, J.
M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parnell, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swierczek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrickson, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, K.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCullar, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanner, S. B.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Drug Discovery: Targeting DNA Methyltransferases</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">2</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1177/1087057111421212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1177%2F1087057111421212" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=21965114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC38Xit1Oltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=2-17&issue=1&author=J.%0AM.+Foulksauthor=K.+M.+Parnellauthor=R.+N.+Nixauthor=S.+Chauauthor=K.+Swierczekauthor=M.+Saundersauthor=K.+Wrightauthor=T.+F.+Hendricksonauthor=K.-K.+Hoauthor=M.+V.+McCullarauthor=S.+B.+Kanner&title=Epigenetic+Drug+Discovery%3A+Targeting+DNA+Methyltransferases&doi=10.1177%2F1087057111421212"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic drug discovery: targeting DNA methyltransferases</span></div><div class="casAuthors">Foulks, Jason M.; Parnell, K. Mark; Nix, Rebecca N.; Chau, Suzanna; Swierczek, Krzysztof; Saunders, Michael; Wright, Kevin; Hendrickson, Thomas F.; Ho, Koc-Kan; McCullar, Michael V.; Kanner, Steven B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-17</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Epigenetic modification of DNA leads to changes in gene expression.  DNA methyltransferases (DNMTs) comprise a family of nuclear enzymes that catalyze the methylation of CpG dinucleotides, resulting in an epigenetic methylome distinguished between normal cells and those in disease states such as cancer.  Disrupting gene expression patterns through promoter methylation has been implicated in many malignancies and supports DNMTs as attractive therapeutic targets.  This review focuses on the rationale of targeting DNMTs in cancer, the historical approach to DNMT inhibition, and current marketed hypomethylating therapeutics azacytidine and decitabine.  In addn., we address novel DNMT inhibitory agents emerging in development, including CP-4200 and SGI-110, analogs of azacytidine and decitabine, resp.; the oligonucleotides MG98 and miR29a; and a no. of reversible inhibitors, some of which appear to be selective against particular DNMT isoforms.  Finally, we discuss future opportunities and challenges for next-generation therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZfIt_0O0PXrVg90H21EOLACvtfcHk0lheTxO_GtEq7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xit1Oltro%253D&md5=e1901374d600c0361e04a3c0e6d004d1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1177%2F1087057111421212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111421212%26sid%3Dliteratum%253Aachs%26aulast%3DFoulks%26aufirst%3DJ.%2BM.%26aulast%3DParnell%26aufirst%3DK.%2BM.%26aulast%3DNix%26aufirst%3DR.%2BN.%26aulast%3DChau%26aufirst%3DS.%26aulast%3DSwierczek%26aufirst%3DK.%26aulast%3DSaunders%26aufirst%3DM.%26aulast%3DWright%26aufirst%3DK.%26aulast%3DHendrickson%26aufirst%3DT.%2BF.%26aulast%3DHo%26aufirst%3DK.-K.%26aulast%3DMcCullar%26aufirst%3DM.%2BV.%26aulast%3DKanner%26aufirst%3DS.%2BB.%26atitle%3DEpigenetic%2520Drug%2520Discovery%253A%2520Targeting%2520DNA%2520Methyltransferases%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26issue%3D1%26spage%3D2%26epage%3D17%26doi%3D10.1177%2F1087057111421212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Biologically Relevant Chemical Space Navigator: From Patent and Structure-Activity Relationship Analysis to Library Acquisition and Design</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3123</span>– <span class="NLM_lpage">3137</span>, <span class="refDoi"> DOI: 10.1021/ci3004539</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3004539" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslWksbfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3123-3137&issue=12&author=O.+Rabalauthor=J.+Oyarzabal&title=Biologically+Relevant+Chemical+Space+Navigator%3A+From+Patent+and+Structure-Activity+Relationship+Analysis+to+Library+Acquisition+and+Design&doi=10.1021%2Fci3004539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Biologically Relevant Chemical Space Navigator: From Patent and Structure-Activity Relationship Analysis to Library Acquisition and Design</span></div><div class="casAuthors">Rabal, Obdulia; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3123-3137</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new and versatile visualization tool, based on a descriptor accounting for ligand-receptor interactions (LiRIf), is introduced for guiding medicinal chemists in analyzing the R-groups from a congeneric series.  Anal. is performed in a ref.-independent scenario where the whole biol. relevant chem. space (BRCS) is represented.  Using a real project-based data set, the authors show the impact of this tool on four key navigation strategies for the drug discovery process.  First, this navigator analyzes competitors' patents, including a comparison of patents coverage and the identification of the most frequent fragments.  Second, the tool analyzes the structure-activity relationship (SAR) leading to the representation of ref.-independent activity landscapes that enable the identification not only of crit. ligand-receptor interactions (LRI) and substructural features but also of activity cliffs.  Third, this navigator enables comparison of libraries, thus selecting com. available mols. that complement unexplored spaces or areas of interest.  Finally, this tool also enables the design of new analogs, which is based on reaction types and the exploration purpose (focused or diverse), selecting the most appropriate reagents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIldm-mo1cdrVg90H21EOLACvtfcHk0lgbNIVVBIzA1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslWksbfP&md5=7ee9cb508b97b607067b519d580cab08</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fci3004539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3004539%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DBiologically%2520Relevant%2520Chemical%2520Space%2520Navigator%253A%2520From%2520Patent%2520and%2520Structure-Activity%2520Relationship%2520Analysis%2520to%2520Library%2520Acquisition%2520and%2520Design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D12%26spage%3D3123%26epage%3D3137%26doi%3D10.1021%2Fci3004539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mann, A.</span>; <span class="NLM_string-name">Camille
Georges, W.</span></span> <span> </span><span class="NLM_article-title">Conformational Restriction And/or Steric Hindrance in Medicinal Chemistry</span>. In  <i>The Practice of Medicinal Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wermuth, C.</span></span>, Ed.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2008</span>; Chapter 17, pp  <span class="NLM_fpage">363</span>– <span class="NLM_lpage">379</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=10.1016%2FB978-0-12-374194-3.00017-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=363-379&author=A.+Mann&author=W.+Camille%0AGeorgesauthor=C.+Wermuth&title=The+Practice+of+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-374194-3.00017-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-374194-3.00017-2%26sid%3Dliteratum%253Aachs%26aulast%3DMann%26aufirst%3DA.%26atitle%3DConformational%2520Restriction%2520And%252For%2520Steric%2520Hindrance%2520in%2520Medicinal%2520Chemistry%26btitle%3DThe%2520Practice%2520of%2520Medicinal%2520Chemistry%26aulast%3DWermuth%26aufirst%3DC.%26pub%3DAcademic%2520Press%26date%3D2008%26spage%3D363%26epage%3D379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasney, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siarheyeva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herold, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Protein Lysine Methyltransferase G9a Inhibitors: Design, Synthesis, and Structure Activity Relationships of 2,4-Diamino-7-Aminoalkoxy-Quinazolines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">5844</span>– <span class="NLM_lpage">5857</span>, <span class="refDoi"> DOI: 10.1021/jm100478y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100478y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5844-5857&issue=15&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=T.+J.+Wigleauthor=G.+A.+Wasneyauthor=A.+Dongauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=W.+P.+Janzenauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Protein+Lysine+Methyltransferase+G9a+Inhibitors%3A+Design%2C+Synthesis%2C+and+Structure+Activity+Relationships+of+2%2C4-Diamino-7-Aminoalkoxy-Quinazolines&doi=10.1021%2Fjm100478y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm100478y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100478y%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProtein%2520Lysine%2520Methyltransferase%2520G9a%2520Inhibitors%253A%2520Design%252C%2520Synthesis%252C%2520and%2520Structure%2520Activity%2520Relationships%2520of%25202%252C4-Diamino-7-Aminoalkoxy-Quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D15%26spage%3D5844%26epage%3D5857%26doi%3D10.1021%2Fjm100478y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span> <i>Pipeline Pilot</i>, <span class="NLM_edition">version 9.5</span>; <span class="NLM_publisher-name">Accelrys Software Inc.</span>; <span class="NLM_publisher-loc">San Diego,
CA</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Pipeline+Pilot%2C+version+9.5%3B+Accelrys+Software+Inc.%3B+San+Diego%2C%0ACA%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPipeline%2520Pilot%26pub%3DAccelrys%2520Software%2520Inc%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herold, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Optimization of Cellular Activity of G9a Inhibitors 7-Aminoalkoxy-Quinazolines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">6139</span>– <span class="NLM_lpage">6150</span>, <span class="refDoi"> DOI: 10.1021/jm200903z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200903z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6139-6150&issue=17&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=A.+Allali-Hassaniauthor=Y.+Heauthor=J.+M.+Heroldauthor=X.+Chenauthor=C.+M.+Yatesauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=J.+Huangauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=Optimization+of+Cellular+Activity+of+G9a+Inhibitors+7-Aminoalkoxy-Quinazolines&doi=10.1021%2Fjm200903z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm200903z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200903z%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYates%26aufirst%3DC.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DOptimization%2520of%2520Cellular%2520Activity%2520of%2520G9a%2520Inhibitors%25207-Aminoalkoxy-Quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26issue%3D17%26spage%3D6139%26epage%3D6150%26doi%3D10.1021%2Fjm200903z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillet, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span> <span> </span><span class="NLM_article-title">Assessment of Additive/nonadditive Effects in Structure-Activity Relationships: Implications for Iterative Drug Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">7552</span>– <span class="NLM_lpage">7562</span>, <span class="refDoi"> DOI: 10.1021/jm801070q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801070q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7552-7562&issue=23&author=Y.+Patelauthor=V.+J.+Gilletauthor=T.+Howeauthor=J.+Pastorauthor=J.+Oyarzabalauthor=P.+Willett&title=Assessment+of+Additive%2Fnonadditive+Effects+in+Structure-Activity+Relationships%3A+Implications+for+Iterative+Drug+Design&doi=10.1021%2Fjm801070q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm801070q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801070q%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DY.%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DPastor%26aufirst%3DJ.%26aulast%3DOyarzabal%26aufirst%3DJ.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DAssessment%2520of%2520Additive%252Fnonadditive%2520Effects%2520in%2520Structure-Activity%2520Relationships%253A%2520Implications%2520for%2520Iterative%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26issue%3D23%26spage%3D7552%26epage%3D7562%26doi%3D10.1021%2Fjm801070q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado-Tejedor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miguel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-González, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barroso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estella-Hermoso
de Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espelosin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ursua, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haizhong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Osta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Biological Evaluation of First-in-Class Dual Acting Histone Deacetylases (HDACs) and Phosphodiesterase 5 (PDE5) Inhibitors for the Treatment of Alzheimer’s Disease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">8967</span>– <span class="NLM_lpage">9004</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00908</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00908" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVyqtL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8967-9004&issue=19&author=O.+Rabalauthor=J.+A.+S%C3%A1nchez-Ariasauthor=M.+Cuadrado-Tejedorauthor=I.+de+Miguelauthor=M.+P%C3%A9rez-Gonz%C3%A1lezauthor=C.+Garc%C3%ADa-Barrosoauthor=A.+Ugarteauthor=A.+Estella-Hermoso%0Ade+Mendozaauthor=E.+S%C3%A1ezauthor=M.+Espelosinauthor=S.+Ursuaauthor=T.+Haizhongauthor=W.+Weiauthor=X.+Mushengauthor=A.+Garcia-Ostaauthor=J.+Oyarzabal&title=Design%2C+Synthesis%2C+and+Biological+Evaluation+of+First-in-Class+Dual+Acting+Histone+Deacetylases+%28HDACs%29+and+Phosphodiesterase+5+%28PDE5%29+Inhibitors+for+the+Treatment+of+Alzheimer%E2%80%99s+Disease&doi=10.1021%2Facs.jmedchem.6b00908"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Biological Evaluation of First-in-Class Dual Acting Histone Deacetylases (HDACs) and Phosphodiesterase 5 (PDE5) Inhibitors for the Treatment of Alzheimer's Disease</span></div><div class="casAuthors">Rabal, Obdulia; Sanchez-Arias, Juan A.; Cuadrado-Tejedor, Mar; de Miguel, Irene; Perez-Gonzalez, Marta; Garcia-Barroso, Carolina; Ugarte, Ana; Estella-Hermoso de Mendoza, Ander; Saez, Elena; Espelosin, Maria; Ursua, Susana; Haizhong, Tan; Wei, Wu; Musheng, Xu; Garcia-Osta, Ana; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8967-9004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Simultaneous inhibition of phosphodiesterase 5 (PDE5) and histone deacetylases (HDAC) has recently been validated as a potentially novel therapeutic approach for Alzheimer's Disease (AD).  To further extend this concept, the authors designed and synthesized the first chem. series of dual acting PDE5 and HDAC inhibitors, and the authors validated this systems therapeutics approach.  Following the implementation of structure- and knowledge-based approaches, initial hits were designed and were shown to validate the hypothesis of dual in vitro inhibition.  Then, an optimization strategy was pursued to obtain a proper tool compd. for in vivo testing in AD models.  Initial hits were translated into mols. with adequate cellular functional responses (histone acetylation and cAMP/cGMP response element-binding (CREB) phosphorylation in the nanomolar range), an acceptable therapeutic window (>1 log unit) and the ability to cross the blood-brain barrier, leading to the identification of 7 as a candidate for in vivo proof-of-concept testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbHBoJlefNEbVg90H21EOLACvtfcHk0lgDLBsPgt8vCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVyqtL7O&md5=756a252809f05f5d7fc59e828736cfba</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00908%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DCuadrado-Tejedor%26aufirst%3DM.%26aulast%3Dde%2BMiguel%26aufirst%3DI.%26aulast%3DP%25C3%25A9rez-Gonz%25C3%25A1lez%26aufirst%3DM.%26aulast%3DGarc%25C3%25ADa-Barroso%26aufirst%3DC.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DEstella-Hermoso%2Bde%2BMendoza%26aufirst%3DA.%26aulast%3DS%25C3%25A1ez%26aufirst%3DE.%26aulast%3DEspelosin%26aufirst%3DM.%26aulast%3DUrsua%26aufirst%3DS.%26aulast%3DHaizhong%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DMusheng%26aufirst%3DX.%26aulast%3DGarcia-Osta%26aufirst%3DA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Biological%2520Evaluation%2520of%2520First-in-Class%2520Dual%2520Acting%2520Histone%2520Deacetylases%2520%2528HDACs%2529%2520and%2520Phosphodiesterase%25205%2520%2528PDE5%2529%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D19%26spage%3D8967%26epage%3D9004%26doi%3D10.1021%2Facs.jmedchem.6b00908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span> <i>OpenEye Toolkit</i>; <span class="NLM_publisher-name">OpenEye Scientific</span>, <span class="NLM_year">2016</span>;<a href="http://www.eyesopen.com/toolkits" class="extLink">http://www.eyesopen.com/toolkits</a> (accessed Oct 20, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+OpenEye+Toolkit%3B+OpenEye+Scientific%2C+2016%3Bhttp%3A%2F%2Fwww.eyesopen.com%2Ftoolkits+%28accessed+Oct+20%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DOpenEye%2520Toolkit%26pub%3DOpenEye%2520Scientific%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&issue=7&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lgDLBsPgt8vCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D7%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW','PDB','3RJW'); return false;">PDB: 3RJW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DA4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DA4','PDB','4DA4'); return false;">PDB: 4DA4</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i164"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01925">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_52051"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01925">10.1021/acs.jmedchem.7b01925</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Protocols for preparative HPLC purification methods; method for high resolution mass spectrometry of final compounds; methods for LCMS, analytical HPLC, and UHPLC; HRMS and purities of final compounds; HPLC traces of final compounds; selectivity of compounds <b>13</b> and <b>43</b> against epigenetic targets (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_001.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_001.csv">jm7b01925_si_001.csv (3.56 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01925/suppl_file/jm7b01925_si_002.pdf">jm7b01925_si_002.pdf (2.21 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB ID Codes: <b>1</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW">3RJW</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DA4">4DA4</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01925%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-15%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01925" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799111aca281969","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
