
<html lang="en"     class="pb-page"  data-request-id="51abcee5-fce7-40e0-afbb-856f44874a34"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c00605;issue:issue:10.1021/jmcmar.2020.63.issue-17;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Design and Synthesis of a Highly Selective and In Vivo-Capable Inhibitor of the Second Bromodomain of the Bromodomain and Extra Terminal Domain Family of Proteins" /></meta><meta name="dc.Creator" content="Alex  Preston" /></meta><meta name="dc.Creator" content="Stephen  Atkinson" /></meta><meta name="dc.Creator" content="Paul  Bamborough" /></meta><meta name="dc.Creator" content="Chun-wa  Chung" /></meta><meta name="dc.Creator" content="Peter D.  Craggs" /></meta><meta name="dc.Creator" content="Laurie  Gordon" /></meta><meta name="dc.Creator" content="Paola  Grandi" /></meta><meta name="dc.Creator" content="James R. J.  Gray" /></meta><meta name="dc.Creator" content="Emma J.  Jones" /></meta><meta name="dc.Creator" content="Matthew  Lindon" /></meta><meta name="dc.Creator" content="Anne-Marie  Michon" /></meta><meta name="dc.Creator" content="Darren J.  Mitchell" /></meta><meta name="dc.Creator" content="Rab K.  Prinjha" /></meta><meta name="dc.Creator" content="Francesco  Rianjongdee" /></meta><meta name="dc.Creator" content="Inmaculada  Rioja" /></meta><meta name="dc.Creator" content="Jonathan  Seal" /></meta><meta name="dc.Creator" content="Simon  Taylor" /></meta><meta name="dc.Creator" content="Ian  Wall" /></meta><meta name="dc.Creator" content="Robert J.  Watson" /></meta><meta name="dc.Creator" content="James  Woolven" /></meta><meta name="dc.Creator" content="Emmanuel H  Demont" /></meta><meta name="dc.Description" content="Pan-bromodomain and extra terminal domain (BET) inhibitors interact equipotently with the eight bromodomains of the BET family of proteins and have shown profound efficacy in a number of in vitro p..." /></meta><meta name="Description" content="Pan-bromodomain and extra terminal domain (BET) inhibitors interact equipotently with the eight bromodomains of the BET family of proteins and have shown profound efficacy in a number of in vitro p..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 21, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00605" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2020 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00605" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00605" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00605" /></link>
        
    
    

<title>Design and Synthesis of a Highly Selective and In Vivo-Capable Inhibitor of the Second Bromodomain of the Bromodomain and Extra Terminal Domain Family of Proteins | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00605" /></meta><meta property="og:title" content="Design and Synthesis of a Highly Selective and In Vivo-Capable Inhibitor of the Second Bromodomain of the Bromodomain and Extra Terminal Domain Family of Proteins" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0021.jpeg" /></meta><meta property="og:description" content="Pan-bromodomain and extra terminal domain (BET) inhibitors interact equipotently with the eight bromodomains of the BET family of proteins and have shown profound efficacy in a number of in vitro phenotypic assays and in vivo pre-clinical models in inflammation or oncology. A number of these inhibitors have progressed to the clinic where pharmacology-driven adverse events have been reported. To better understand the contribution of each domain to their efficacy and improve their safety profile, selective inhibitors are required. This article discloses the profile of GSK046, also known as iBET-BD2, a highly selective inhibitor of the second bromodomains of the BET proteins that has undergone extensive pre-clinical in vitro and in vivo characterization." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00605"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00605">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00605&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00605&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00605&amp;href=/doi/10.1021/acs.jmedchem.0c00605" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 9070-9092</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00614" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00796" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design and Synthesis of a Highly Selective and <i>In Vivo</i>-Capable Inhibitor of the Second Bromodomain of the Bromodomain and Extra Terminal Domain Family of Proteins</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Alex Preston</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alex Preston</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#b1d0ddd4c99fd6c29fc1c3d4c2c5dedff1d6c2da9fd2dedc"><span class="__cf_email__" data-cfemail="0f6e636a7721687c217f7d6a7c7b60614f687c64216c6062">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alex++Preston">Alex Preston</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0334-0679" title="Orcid link">http://orcid.org/0000-0003-0334-0679</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Stephen Atkinson</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephen Atkinson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#daa9aebfaab2bfb4f4bbaeb1b3b4a9b5b4eb9abba9aea8bba0bfb4bfb9bbf4b9b5b7"><span class="__cf_email__" data-cfemail="5a292e3f2a323f34743b2e3133342935346b1a3b292e283b203f343f393b74393537">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephen++Atkinson">Stephen Atkinson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3636-3674" title="Orcid link">http://orcid.org/0000-0003-3636-3674</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Paul Bamborough</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul Bamborough</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul++Bamborough">Paul Bamborough</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9479-2894" title="Orcid link">http://orcid.org/0000-0001-9479-2894</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Chun-wa Chung</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chun-wa Chung</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chun-wa++Chung">Chun-wa Chung</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2480-3110" title="Orcid link">http://orcid.org/0000-0002-2480-3110</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Peter D. Craggs</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter D. Craggs</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+D.++Craggs">Peter D. Craggs</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Laurie Gordon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laurie Gordon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laurie++Gordon">Laurie Gordon</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Paola Grandi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paola Grandi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">IVIVT Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstr. 1, 69117 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Grandi">Paola Grandi</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">James R. J. Gray</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James R. J. Gray</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Quantitative Pharmacology, Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+R.+J.++Gray">James R. J. Gray</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Emma J. Jones</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Emma J. Jones</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Emma+J.++Jones">Emma J. Jones</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Matthew Lindon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew Lindon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew++Lindon">Matthew Lindon</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Anne-Marie Michon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anne-Marie Michon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">IVIVT Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstr. 1, 69117 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anne-Marie++Michon">Anne-Marie Michon</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Darren J. Mitchell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Darren J. Mitchell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Darren+J.++Mitchell">Darren J. Mitchell</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Rab K. Prinjha</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rab K. Prinjha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rab+K.++Prinjha">Rab K. Prinjha</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Francesco Rianjongdee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Francesco Rianjongdee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Francesco++Rianjongdee">Francesco Rianjongdee</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Inmaculada Rioja</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Inmaculada Rioja</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Inmaculada++Rioja">Inmaculada Rioja</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Jonathan Seal</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jonathan Seal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jonathan++Seal">Jonathan Seal</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0148-5487" title="Orcid link">http://orcid.org/0000-0003-0148-5487</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Simon Taylor</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simon Taylor</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Quantitative Pharmacology, Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simon++Taylor">Simon Taylor</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Ian Wall</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ian Wall</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ian++Wall">Ian Wall</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Robert J. Watson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert J. Watson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+J.++Watson">Robert J. Watson</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">James Woolven</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James Woolven</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James++Woolven">James Woolven</a></span></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Emmanuel H Demont</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Emmanuel H Demont</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Emmanuel+H++Demont">Emmanuel H Demont</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7307-3129" title="Orcid link">http://orcid.org/0000-0001-7307-3129</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00605&amp;href=/doi/10.1021%2Facs.jmedchem.0c00605" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 9070â9092</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 21, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 April 2020</li><li><span class="item_label"><b>Published</b> online</span>21 July 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 September 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00605" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00605</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ec-1612971054143.svg" alt="ACS Editorsâ Choice" /></img><span>ACS Editorsâ Choice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00605"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">11255</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">15</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00605" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design and Synthesis of a Highly Selective and In Vivo-Capable Inhibitor of the Second Bromodomain of the Bromodomain and Extra Terminal Domain Family of Proteins&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Alex&quot;,&quot;last_name&quot;:&quot;Preston&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;Atkinson&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;Bamborough&quot;},{&quot;first_name&quot;:&quot;Chun-wa&quot;,&quot;last_name&quot;:&quot;Chung&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;D. Craggs&quot;},{&quot;first_name&quot;:&quot;Laurie&quot;,&quot;last_name&quot;:&quot;Gordon&quot;},{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Grandi&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;R. J. Gray&quot;},{&quot;first_name&quot;:&quot;Emma&quot;,&quot;last_name&quot;:&quot;J. Jones&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;Lindon&quot;},{&quot;first_name&quot;:&quot;Anne-Marie&quot;,&quot;last_name&quot;:&quot;Michon&quot;},{&quot;first_name&quot;:&quot;Darren&quot;,&quot;last_name&quot;:&quot;J. Mitchell&quot;},{&quot;first_name&quot;:&quot;Rab&quot;,&quot;last_name&quot;:&quot;K. Prinjha&quot;},{&quot;first_name&quot;:&quot;Francesco&quot;,&quot;last_name&quot;:&quot;Rianjongdee&quot;},{&quot;first_name&quot;:&quot;Inmaculada&quot;,&quot;last_name&quot;:&quot;Rioja&quot;},{&quot;first_name&quot;:&quot;Jonathan&quot;,&quot;last_name&quot;:&quot;Seal&quot;},{&quot;first_name&quot;:&quot;Simon&quot;,&quot;last_name&quot;:&quot;Taylor&quot;},{&quot;first_name&quot;:&quot;Ian&quot;,&quot;last_name&quot;:&quot;Wall&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;J. Watson&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;Woolven&quot;},{&quot;first_name&quot;:&quot;Emmanuel&quot;,&quot;last_name&quot;:&quot;H Demont&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;21&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;9070-9092&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00605&quot;},&quot;abstract&quot;:&quot;Pan-bromodomain and extra terminal domain (BET) inhibitors interact equipotently with the eight bromodomains of the BET family of proteins and have shown profound efficacy in a number of in vitro phenotypic assays and in vivo pre-clinical models in inflammation or oncology. A number of these inhibitors have progressed to the clinic where pharmacology-driven adverse events have been reported. To better understand the contribution of each domain to their efficacy and improve their safety profile, selective inhibitors are required. This article discloses the profile of GSK046, also known as iBET-BD2, a highly selective inhibitor of the second bromodomains of the BET proteins that has undergone extensive pre-clinical in vitro and in vivo characterization.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00605&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00605" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00605&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00605" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00605&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00605" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00605&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00605&amp;href=/doi/10.1021/acs.jmedchem.0c00605" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00605" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00605" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00605%26sid%3Dliteratum%253Aachs%26pmid%3D32691591%26genre%3Darticle%26aulast%3DPreston%26date%3D2020%26atitle%3DDesign%2Band%2BSynthesis%2Bof%2Ba%2BHighly%2BSelective%2Band%2BIn%2BVivo-Capable%2BInhibitor%2Bof%2Bthe%2BSecond%2BBromodomain%2Bof%2Bthe%2BBromodomain%2Band%2BExtra%2BTerminal%2BDomain%2BFamily%2Bof%2BProteins%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D17%26spage%3D9070%26epage%3D9092%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=290658" title="Amides">Amides</a>,</li><li><a href="/action/doSearch?ConceptID=292349" title="Alkyls">Alkyls</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/jmcmar.2020.63.issue-17/20200910/jmcmar.2020.63.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0021.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Pan-bromodomain and extra terminal domain (BET) inhibitors interact equipotently with the eight bromodomains of the BET family of proteins and have shown profound efficacy in a number of <i>in vitro</i> phenotypic assays and <i>in vivo</i> pre-clinical models in inflammation or oncology. A number of these inhibitors have progressed to the clinic where pharmacology-driven adverse events have been reported. To better understand the contribution of each domain to their efficacy and improve their safety profile, selective inhibitors are required. This article discloses the profile of GSK046, also known as iBET-BD2, a highly selective inhibitor of the second bromodomains of the BET proteins that has undergone extensive pre-clinical <i>in vitro</i> and <i>in vivo</i> characterization.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24307" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24307" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The bromodomain and extra terminal domain (BET) family of proteins consists of four isoforms: BRD2, BRD3, BRD4, and BRDT. Each of these BET proteins contain two bromodomains, known as bromodomain 1 (N-terminus, BD1) and bromodomain 2 (C-terminus, BD2). The profound anti-proliferative and anti-inflammatory effects of pan-BET inhibitors, acting at both BD1 and BD2 of all four BET proteins, have now been well documented.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22">(1â22)</a> Indeed, a number of pan-BET inhibitors are progressing through phase I to II clinical trials for oncology indications.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> However, associated pharmacology-driven toxicities with pan-BET inhibitors have also been reported.<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24â26)</a> Our strategy was therefore to target the rational design of selective BET inhibitors in order to elucidate the functional contribution of each domain to the phenotype observed with pan-BET inhibitors with the ambition of improving on the adverse event profile observed in humans. Due to the high homology across the four bromodomain-containing proteins, it was envisaged that obtaining a single isoform-selective inhibitor would be complex. However, the homology between the BD2 and the BD1 domains is lower with key residues close to the acetylated lysine (AcK) recognition pocket altered (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00605/suppl_file/jm0c00605_si_001.pdf" class="ext-link">Figure S2</a>), which, it was hypothesized, would give the opportunity to deliver pan BD2 selectivity.<a onclick="showRef(event, 'ref27 ref28 ref29 ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29 ref30 ref31 ref32">(27â32)</a> Herein, we describe our approach to identify a tool molecule with second bromodomain (BD2) pIC<sub>50</sub> > 7 at BRD2, 3, 4, and T combined with selectivity over the first domain (BD1) of at least 100-fold. In addition, we wanted to explore the opportunity of using BD2-selective molecules as tools to understand the <i>in vivo</i> capabilities and wider developability of our series.</div><div class="NLM_p">At the time this work was carried out, there were no highly potent BD2-selective inhibitors known. The best-studied molecule reported to show bias in favor of BD2 was RVX-208 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), which has a <i>K</i><sub>d</sub> of 8.9 Î¼M against BD1 and 300 nM against BD2 by ITC, a selectivity of approximately 30-fold.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Independent ITC measurements showed greater potency but lower selectivity (<i>K</i><sub>d</sub> of 1.1 Î¼M for BRD4 BD1 and 135 nM for BRD4 BD2).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> We have subsequently reported a series of quinoxaline-based inhibitors such as GSK340, which show greater potency and selectivity (<i>K</i><sub>d</sub> in the Discoverx BROMOscan assays of 1.5 Î¼M against BRD4 BD1 and 7 nM against BRD4 BD2).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Very recently, ABBV-744 was reported as a nanomolar BD2 inhibitor and clinical candidate with >290-fold selectivity over the BD1 domains of BRD2/3/4.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Literature BD2-selective inhibitors RVX-208, GSK340, and ABBV-744.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In order to identify hits that showed selectivity for the BD2 domain over the BD1 domain, we initiated a high-throughput screen (HTS) of over 2 M compounds. The compounds were screened initially at a single concentration of 10 Î¼M versus the BRD4 dual domain protein construct using a TR-FRET assay. In this construct, in order to evaluate binding at the BD2 domain only, the conserved tyrosine of the BD1 acetyl-lysine binding site was mutated to alanine (Y97A) to inactivate the binding site leaving just the BD2 domain able to bind the fluoro-ligand. A corresponding Y390A construct was used to assess BD1 binding. The BRD4 isoform was considered to be representative of all the BET isoforms. A thorough triage was undertaken, including an orthogonal BD2 biochemical assay, full-curve follow-up against BRD4 BD2 and BD1, and SPR biophysical characterization to provide robust and progressible hits. Acetamide <b>1</b> was identified as a hit compound from this screening process (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The difficulties in obtaining highly domain-selective compounds were well known to us so we were attracted to <b>1</b> due to the good selectivity of this hit, being over 10-fold-selective for BRD4 BD2 over BD1. Notably, this translated to similar selectivity for BRD2, 3, and T. The molecule was also in good physicochemical space with a <i>c</i> log <i>P</i> of 1.1, chrom log <i>D</i> of 2.2,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and property forecast index (PFI) value of 5.2. The later value is a metric used originally as a prediction of solubility with a value of <6 seen as optimal.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Pleasingly, <b>1</b> also had good kinetic solubility, as measured by chemiluminescent nitrogen detection (CLND),<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> of 125 Î¼g/mL. Encouragingly for an unoptimized hit, the microsomal clearance in both rats and humans was moderate, giving us confidence that there were no major metabolic vulnerabilities with the template.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Profile of HTS hit <b>1</b>. Some results below the curve-fitting threshold of 4.3 could not be included in the mean. In these cases, the numbers included in the mean/number of times tested are both shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To better understand the selectivity of this molecule, we obtained crystal structures of <b>1</b> in the BD1 domain of BRD4 and the BD2 domain of BRD2. Since the conservation within BD1 and BD2 acetyl-lysine sites is very high (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00605/suppl_file/jm0c00605_si_001.pdf" class="ext-link">Figure S2</a>, Supporting Information), these two structures are very likely to represent binding to all bromodomains of the family. Compound <b>1</b> contains an acetyl-lysine mimetic core closely related to paracetamol, and its binding mode is very similar to the published structure of paracetamol in the BRD2 BD1 bromodomain (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> The <i>N</i>-acetyl group mimics the acetylated lysine of the histone tail with its carbonyl group hydrogen bonding to the conserved asparagine (BRD4 BD1 Asn140) and the conserved network of water molecules at the bottom of the binding site.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Crystal structure of the BRD4 BD1 complex with <b>1</b> (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6z7g">6z7g</a>) superimposed on the BRD2 BD1-bound structure of paracetamol (magenta, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4a9j">4a9j</a>). (B) BRD4 BD1 complex with <b>1</b>. (C) BRD2 BD2 complex with <b>1</b> showing alternative conformations (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6z7f">6z7f</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Moving beyond the core of <b>1</b>, the 3-position benzyloxy substituent turns sharply to position the terminal phenyl ring on the WPF shelf of BRD4 BD1. This is a hydrophobic region created by the side chains of Ile146 and the Trp81âPro82âPhe83 sequence (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). This geometry is encouraged by the formation of an internal hydrogen-bond between the 3-position ether oxygen and the 4-position acetamido NH group. The 1-position amide NH is involved in a water-bridged interaction with Asn140. The imidazole group is visible in the electron density and forms a water-bridged hydrogen bond network with the acid side chain of Asp96.</div><div class="NLM_p">The binding mode of <b>1</b> in BRD2 BD2 is comparable to that in BRD4 BD1 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C), with similar interactions between the inhibitor and the analogous BD2 residues Asn429, Val435, and Trp370âPro371âPhe372. These interactions are common to many pan-BET<a onclick="showRef(event, 'ref39 ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41">(39â41)</a> inhibitors and do not offer on their own an insight into the BD2 bias of <b>1</b>. The main difference from the BD1-bound complex in BD2 lies around the 4-position imidazole substituent where the electron density is complex, suggesting that this group can adopt at least two conformations. The neighboring His433 side chain also exists in two alternative rotameric states. In one <i>trans</i> chi1 rotamer, the histidine side chain points away from the inhibitor and allows two water molecules to enter the interface, while in the other <i>gauche</i>(+) rotamer, the side chain points toward the inhibitor and displaces these waters. This mobility of His433 contrasts with the well-resolved position of the equivalent BRD4 BD1 residue, Asp144, which is constrained by a hydrogen bond to the backbone NH of Lys141.</div><div class="NLM_p last">Embarking on optimization of this template, the crystal structures suggested that the core of the molecule, including both amide groups, should be retained. It was hoped that further optimization of the shelf and amide vectors would be possible and also that the methylene linker of the benzyl shelf group could offer a vector from which to explore the ZA channel. This is a narrow cleft present in both BD2 and BD1 domains, flanked by Leu381 and Trp370 in BRD2 BD2. This region has been accessed by a number of pan-BET inhibitors leading to higher affinity.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15277" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15277" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Initial SAR investigations focused on the amide-imidazole vector and attempted to improve the artificial membrane permeability (AMP), which is correlated with oral bioavailability, as well as ligand efficiency (LE),<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42,43)</a> BD2 potency, and domain selectivity (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). It was reasoned that the basic imidazole group could be responsible for limiting the permeability, while reoptimization of this substituent could also lead to LE improvements.</div><div class="NLM_p">In order to understand the importance of the amide group to potency, we removed this group altogether to give the fragment <b>2</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), which had a poor BD2 pIC<sub>50</sub> of 4.6 although interestingly an improved LE versus <b>1</b>, suggesting that the core was inherently an efficient BD2 binder. It should also be noted that deletion of the amide causes a significant and undesired increase in lipophilicity. Truncating the amide back to the methyl amide <b>3</b> saw potency returned and LE maintained versus <b>2</b>, albeit with poor selectivity, indicating that the amide was important for potency and the imidazole-linked moiety was offering little additional binding to BD2. The importance of the amide NH suggested by the crystal structure of <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) was confirmed by reduced potency of the tertiary amide <b>4</b> (BRD4 BD2 pIC<sub>50</sub> = 4.5). Homologating the methyl amide <b>3</b> to ethyl amide <b>5</b> brought selectivity back, but BD2 potency and LE were reduced. Introduction of a hydroxyl to give ethyl alcohol <b>6</b> maintained potency and selectivity but was detrimental to permeability. This OH was replaced with an NH<sub>2</sub> group <b>7</b> with no real effect on potency/selectivity observed, but a further reduced AMP (<3 nm/s) resulted, likely due to the highly polar nature of the protonated amine group. Further extending the OH to propanol <b>9</b> again showed no real benefit in terms of potency. Replacing this OH with an NH<sub>2</sub> group <b>8</b> resulted in an increase in potency (BRD4 BD2 pIC<sub>50</sub> = 5.8), and selectivity was also increased (20-fold), but reduced AMP was observed. In order to address the reduced permeability seen with <b>6</b>, <b>7</b>, and <b>8</b>, examples were made where the heteroatom was constrained into a ring system such as tetrahydropyran (THP) <b>10</b> where a boost in BD2 potency was observed. Pushing this ring out further toward solvent (<b>11</b> and <b>13</b>) had little impact on both potency and selectivity, but an increase in chrom log <i>D</i> was observed. We also looked at replacing the THP with piperidine <b>12</b>; here, an increase in potency and selectivity was seen, but introduction of a basic center once again had a negative impact on permeability as measured by the AMP assay. Finally, modification of THP <b>10</b> to <i>trans</i>-cyclohexanol <b>14</b> provided a compound with a good balance of potency, selectivity, solubility, and permeability, albeit the additional H-bond donor versus <b>10</b> was likely responsible for the moderate permeability measured.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Optimization of the Amide Vector of <b>1</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0016.gif" alt="" id="gr15" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0017.gif" alt="" id="gr16" /></img><div></div></div><div class="NLM_p">In parallel to the amide exploration, the shelf region of the molecule was investigated, which is accessed from the phenolic position of the central ring. We decided to do this SAR exploration with the propylamine amide group as this had been identified early in our investigations as a group with good potency, LE, and selectivity (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, <b>8</b>). After conducting an array, it was determined that aromaticity was important in this region with substituents such as THP <b>15</b> being inactive. Introduction of a heteroatom into the ring also gave reduced potency (compounds <b>16</b>â<b>18</b>). Electron-withdrawing substituents such as chloride were tolerated in all three positions (compounds <b>19</b>â<b>21</b>) but offered no advantage to inhibitor <b>8</b>. Replacing the <i>para</i>-Cl with a <i>para</i>-Me (<b>22</b>) was equally well tolerated, and expanding this group to a <i>para</i>-iPr (<b>23</b>) gave a slight increase in potency of 0.1 log units but was accompanied with a 1.5 log unit increase in chrom log <i>D</i>. Introduction of the OMe at the <i>para</i> position (<b>24</b>) again had a similar profile to that of the <i>para</i>-Cl (<b>19</b>); however, more polar groups at this position such as the para-CONH<sub>2</sub> (<b>25</b>) were not well tolerated. A key breakthrough arrived with the introduction of an Î±-methyl group at the benzylic position to provide <b>26</b>. This gave a 0.5 log increase in potency, maintained LE, and also improved BD2 selectivity to 40-fold.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Optimization of the Benzylic Group of <b>8</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0018.gif" alt="" id="gr17" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0019.gif" alt="" id="gr18" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Some results below the curve-fitting threshold of 4.3 could not be included in the mean and SD. In these cases, the numbers included in the mean/number of times tested are both shown.</p></div></div><div></div></div><div class="NLM_p">To establish which enantiomer of the Î±-methyl benzyl was the more potent, both the <i>R</i> and <i>S</i> enantiomers <b>27</b> and <b>28</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) were prepared using S<sub>N</sub>Ar chemistry with the corresponding chiral alcohols (<a class="ref internalNav" href="#sch4" aria-label="scheme 4">scheme 4</a>). The <i>S</i> enantiomer was found to be the more potent of the two enantiomers. This can be rationalized using the BRD2 BD2 crystal structure of <b>1</b>. Substituents with <i>S</i> stereochemistry can grow freely toward the ZA channel of the site, whereas even a methyl group with <i>R</i> stereochemistry would bump with the phenyl ring of the core as well as the benzylic ring (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00605/suppl_file/jm0c00605_si_001.pdf" class="ext-link">Figure S6</a>, Supporting Information). The torsional changes needed to relax this bump would prevent the benzylic ring from occupying the WPF shelf.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Profile of lead molecules <b>28</b> and <b>29</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Combining our favored cyclohexanol amide groups (cf. <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <b>14</b>) with the (<i>S</i>)-Î±-methyl benzyl shelf group gave <b>29</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). <b>29</b> in particular showed a highly encouraging overall profile with good levels of BD2 potency, nearing 100 nM, coupled with exquisite selectivity for BD2 over BD1 of 250-fold. This compared favorably with <b>28</b>, which had a BD2 potency of 250 nM and selectivity of 125-fold. Importantly, the good physicochemical properties of the initial hit are maintained in both with a PFI of <6. The compounds are permeable and soluble as measured by AMP and CLND, respectively, while the LE has been improved from <b>1</b> and is now >0.3. <b>29</b> was preferred over <b>28</b> due to its better overall profile.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Chemistry</h3><div class="NLM_p">Preparation of the amide variants <b>3</b>â<b>14</b> was achieved using the synthetic route outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. A key late stage acid intermediate <b>33</b> was prepared by acetylation of the commercially available methyl 4-amino-3-hydroxybenzoate (<b>30</b>) followed by alkylation of the phenol with benzyl bromide to give <b>32</b>; hydrolysis of the methyl amide using lithium hydroxide gave <b>33</b> in a good yield. This acid intermediate was then used to vary the amide R<sub>2</sub> using standard amide coupling conditions to give examples <b>3</b>â<b>14</b>.</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route for Variance of the Amide Group<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) acetyl chloride, THF/xylene (1:2), 70 Â°C, 16 h (92%). (ii) BnBr, K<sub>2</sub>CO<sub>3</sub>, DMF, 70 Â°C, 16 h (71%). (iii) LiOH, THF/MeOH/H<sub>2</sub>O (2:2:1), rt, 16 h (83%). (iv) R<sub>1</sub>N(R<sub>2</sub>)H, HATU or T<sub>3</sub>P, DIPEA, DMF, 16 h, rt (4â67%). (v) HCl or TFA, 1,4-dioxane or CH<sub>2</sub>Cl<sub>2</sub>, 0 Â°C, 2 h (21â40%).</p></p></figure><div class="NLM_p">Similar chemistries could be utilized to access the shelf variants <b>15</b>â<b>26</b> as detailed in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Here, the strategy of synthesizing a late-stage intermediate, which allowed variation of the shelf vector R<sub>3</sub> using standard alkylation chemistry, was employed. Acetylation of 4-amino-3-hydroxybenzoic acid <b>34</b> to give <b>35</b> was followed by amide coupling of a suitably protected (boc, trifluoroacetamide) diaminopropyl group to yield <b>36</b> or <b>37</b>. Alkylation of the phenol with commercially available benzyl bromides and subsequent deprotection of the amide side chain gave <b>15</b>â<b>26</b>.</div><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Route for Variance of Shelf Group R<sub>3</sub><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) acetyl chloride, THF/xylene (1:2), 70 Â°C, 5 h (75%). (ii) Amine, EDC, HOBt, DMF 16 h, rt (56%). (iii) EDC, HOBt, TEA, DMF, rt (44%). (iv) R<sub>3</sub>Br, K<sub>2</sub>CO<sub>3</sub>, DMF, 60â70 Â°C, 2â3 h then HCl, 1,4-dioxane, rt, 3â16 h (24â69%). (v) K<sub>2</sub>CO<sub>3</sub>, DMF, 70 Â°C, 3 h then K<sub>2</sub>CO<sub>3</sub>, MeOH, 12 h, rt (40%).</p></p></figure><div class="NLM_p">A modified synthetic strategy (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>) was used when accessing the chirally pure example <b>29</b>. Again, a late-stage intermediate <b>42</b> was prepared, which would then allow variation of the amide as the final step. An S<sub>N</sub>Ar reaction was performed with the commercially available 3-fluoro-4-nitrobenzoic acid <b>38</b> and (<i>S</i>)-1-phenylethanol followed by addition of MeI to the reaction and gave the methyl ester <b>39</b>. Reduction of the nitro group using Pt/C as a catalyst followed by acetylation of the resulting amine <b>40</b> gave <b>41</b>. The ester of <b>41</b> could then be hydrolyzed to give the acid <b>42</b>. Coupling of this acid with the appropriate amine using standard amide coupling conditions gave the desired compound <b>29</b> in acceptable yields.</div><figure id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Route to <b>29</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) NaH, MeI, (<i>S</i>)-1-phenylethanol, DMF/THF (2:1), 0â25 Â°C, 22 h (74%). (ii) Pt/C, H<sub>2</sub>, EtOH, rt, 1 atm (97%). (iii) Ac<sub>2</sub>O, H<sub>2</sub>O, rt, 2 h, (81%). (iv) LiOH, THF/H<sub>2</sub>O (1:1), 50 Â°C 3 h, rt, 16 h (81%). (v) <i>trans</i>-4-aminocyclohexanol, HATU, DIPEA, DMF, rt, 18 h (55%).</p></p></figure><div class="NLM_p">The synthetic steps in this sequence were reordered in order to access <b>27</b> and <b>28</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). Here, the first step was an amide coupling between the acid <b>38</b> and tetrahydropyran amine using standard amide coupling conditions to give <b>43</b>. An S<sub>N</sub>Ar reaction was then performed with the commercially available (<i>S</i>)-chiral alcohol to give <b>44</b>, and this was then followed by reduction of the nitro <b>45</b> and subsequent acetylation as seen in the previous scheme to give <b>28</b>. The enantiomer <b>27</b> was accessed using similar chemistry but with the (<i>R</i>)-chiral alcohol in step ii.</div><figure id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0014.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthetic Route to <b>27</b> and <b>28</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) CDI, DMF, tetrahydro-2<i>H</i>-pyran-4-amine HCl, rt, 18 h (92%). (ii) NaH, DMF, (<i>S</i>)-1-phenylethanol, rt, 2 h, (72%) for <b>44b</b> or 1 M LiHMDS in THF, THF, (<i>R</i>)-1-phenylethanol, 100 Â°C, 30 min (81%) for <b>44</b>. (iii) Sn(II)Cl, EtOH, 60 Â°C, 16 h (37% for <b>45b</b> and 39% for <b>45</b>). (iv) Ac<sub>2</sub>O, H<sub>2</sub>O, 50 Â°C, 16 h, (51% for <b>28</b> and 76% for <b>27</b>).</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> AMES Liability</h3><div class="NLM_p">Despite the promising profile of <b>29</b>, this molecule contains an embedded aniline, which brings with it the risk of a potential genotoxic liability by virtue of the possibility of forming a mutagenic nitrenium ion <i>in vivo</i> following deacetylation.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Therefore, we wanted to gain a better understanding of this risk before further progressing the molecule. An <i>in vitro</i> metabolic identification study on a closely related analogue (<b>28</b>) in human and rat liver S9 fractions showed deacetylation to aniline as the main metabolite (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00605/suppl_file/jm0c00605_si_001.pdf" class="ext-link">Figure S4</a>, Supporting Information). Assuming the metabolic process would be similar across the series aniline <b>46</b> would be the main metabolite of compounds <b>28</b> and <b>29</b>. <i>In silico</i> analysis of <b>28</b> using meteor (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00605/suppl_file/jm0c00605_si_001.pdf" class="ext-link">Figure S3</a>, Supporting Information) suggested that both amides would be metabolically vulnerable. Debenzylation was also predicted. A rudimentary stability study of <b>29</b> was performed by treatment with a 2 M HCl aqueous solution for 24 h. Aniline <b>46</b> was identified as a potential degradant. Removal of the benzyl group to give the phenol was also observed. Certain anilines are known to have mutagenic properties as they are oxidized by the cytochrome P450 enzymes <i>in vivo</i> to form a hydroxylamine, which can then break down to a potentially mutagenic nitrenium species, which reacts with DNA. The gold standard test for mutagenicity is the Ames test.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> However, in order to predict the mutagenic liability of <b>46</b>, together with another potential degradant/predicted metabolite, phenol <b>47</b>, an <i>in silico</i> semi-empirical calculation, the eHOMO (electronic highest occupied molecular orbital) model, was used<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). A HOMO energy of â0.320 atomic units (a.u.) or higher represents an elevated risk of Ames positive test results. eHOMO results for both <b>46</b> and <b>47</b> were similar and borderline with <b>46</b> considered as having a reduced risk while aniline <b>47</b> considered as having a high risk of generating an Ames positive result. Interestingly, acetamide <b>29</b> was predicted to be negative. Given the eHOMO findings and a finite likelihood of generating <b>46</b> and <b>47</b><i>in vivo</i>, we decided to progress them into the Ames assay to confirm these findings. Aniline <b>46</b> was found to be negative in the Ames assay against strains TA1535, TA1537, TA98, TA100, and <i>Escherichia coli</i> WP2uvrA +/âS9. Unfortunately, the potential degradant/predicted metabolite <b>47</b> was indeed positive in the Ames assay against strain TA98 in the presence of S9 in agreement with the eHOMO prediction. Therefore, the potential for <b>29</b> to degrade/metabolize to <b>47</b> was considered a significant risk if we were to consider developing this series toward a clinical candidate.</div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Investigating and mitigating the mutagenic liabilities of potential degradants of <b>29</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Strategies were investigated to mitigate the potential mutagenicity risk inherent in <b>29</b>. One option was to reduce the electron density of the aniline through the introduction of electron-withdrawing groups to the aromatic core. This was investigated by substituting with fluorine at the 2, 5, or 6 positions to provide <b>49</b>, <b>51</b>, and <b>53</b>, respectively. These potential degradants/metabolites all had reduced HOMO energies and were therefore predicted to be negative in the AMES test (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). As a result, examples of these F-substituted compounds were prepared (compounds <b>48</b>, <b>50</b>, and <b>52</b>). For synthetic ease, the optimal cyclohexanol amide was replaced with an ethyl group, while the Î±-methyl group was not introduced on these examples initially. Introduction of fluorine at the 2-position (<b>48</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) gave a reduction in potency at BD2 when compared with the unsubstituted parent <b>5</b>. However, the 5-substituted fluoro example <b>50</b> showed an increase in potency, but this was accompanied with an unacceptable increase in chrom log <i>D</i>. The 6-F-substituted compound <b>52</b> was preferable, demonstrating an increase in BD2 potency of 5-fold and also a slight decrease in chrom log <i>D</i>.</div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Mitigating the genotoxic risk of aniline <b>47</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Potency and Selectivity of Fluorinated Core Derivatives of <b>5</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="(" /></col><col align="char" char="(" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="/" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>compound no.</b></td><td class="colsep0 rowsep0" align="center" char="("><b>BRD4 BD1 pIC<sub>50</sub> (<i>n</i>)</b></td><td class="colsep0 rowsep0" align="center" char="("><b>BRD4 BD2 pIC<sub>50</sub> (<i>n</i>)</b></td><td class="colsep0 rowsep0" align="center" char="."><b>BRD4 BD2 sel (fold)</b></td><td class="colsep0 rowsep0" align="center" char="."><b>BRD4 BD2 LE</b></td><td class="colsep0 rowsep0" align="center" char="."><b>chrom log <i>D</i></b></td><td class="colsep0 rowsep0" align="center" char="/"><b>AMP (nm/sec)/cLND (Î¼g/mL)</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char="("><4.3 (4)</td><td class="colsep0 rowsep0" align="char" char="(">5.3 (3)</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">0.31</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td><td class="colsep0 rowsep0" align="char" char="/">320/143</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48</b></td><td class="colsep0 rowsep0" align="char" char="("><4.3 (2)</td><td class="colsep0 rowsep0" align="char" char="(">4.7 (2)</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">0.27</td><td class="colsep0 rowsep0" align="char" char=".">3.9</td><td class="colsep0 rowsep0" align="char" char="/">460/119</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>50</b></td><td class="colsep0 rowsep0" align="char" char="("><4.3 (2)</td><td class="colsep0 rowsep0" align="char" char="(">5.8 (2)</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">0.33</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td><td class="colsep0 rowsep0" align="char" char="/">670/159</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>52</b></td><td class="colsep0 rowsep0" align="char" char="("><4.3 (2)</td><td class="colsep0 rowsep0" align="char" char="(">6.0 (2)</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char=".">0.34</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char="/">110/18</td></tr></tbody></table></div></div><div class="NLM_p">Given these exciting data, it was decided to combine the 6-F substituent with the favored shelf and amide groups of <b>29</b>. This gave acetamide <b>59</b>, which can be prepared using the synthetic route defined in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. An S<sub>N</sub>Ar reaction was performed on 5-bromo-1,3-difluoro-2-nitrobenzene <b>54</b> with (<i>S</i>)-phenylethanol to give <b>55</b>, and reduction of the nitro and subsequent acetylation of the resulting amine <b>56</b> gave <b>57</b> in good overall yields. A carbonylation reaction with <b>57</b> using Xantphos and Pd(OAc)<sub>2</sub> under a CO atmosphere could be used to access the methyl ester <b>58</b>. Hydrolysis of <b>58</b> followed by amide coupling under standard conditions was used to give <b>59</b>. Interestingly, the late-stage intermediate <b>57</b> (where R = Bn) could be used to synthesize <b>52</b> directly using an amidocarbonylation reaction with Xantphos, Pd(OAc)<sub>2</sub>, ethylamine, Co<sub>2</sub>CO<sub>8</sub>, and microwave irradiation.</div><figure id="sch5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0015.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of 5-F-Substituted Analogues Including <b>59</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) ROH, LiHMDS, THF, rt, 30 min (99%), (ii) Fe(0), NH<sub>4</sub>Cl, EtOH, 80 Â°C, 1 h (82%). (iii) Ac<sub>2</sub>O, rt, 20 h (81%). (iv) Xantphos, Pd(OAc)<sub>2</sub>, DMAP, Co<sub>2</sub>CO<sub>8</sub>, 1,4-dioxane, ethylamine, 90 Â°C, 20 min (66%). (v) Xantphos, Pd(OAc)<sub>2</sub>, Et<sub>3</sub>N, CO, MeOH, DMF, 0 Â°C, 1 h (73%). (vi) LiOH, THF/H<sub>2</sub>O 1:1, 50 Â°C, 2 h (93%). (vii) <i>trans</i>-4-aminocyclohexanol, HATU, DIPEA, DCM, rt, 1 h (89%).</p></p></figure><div class="NLM_p"><b>59</b> was tested in the Brd4 assay and pleasingly resulted in a 1000 fold BD2 selective inhibitor (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Regrettably, while <b>59</b> itself was Ames negative, when we tested potential metabolite/degradant <b>53</b>, it tested Ames positive against strain WP2 uvrA in the presence and absence of S9. Therefore, while the eHOMO model is in general a very useful <i>in silico</i> predictor, on this occasion, it predicted the mutagenicity risk of <b>53</b> incorrectly. Despite the positive Ames result with the potential metabolite, <b>59</b> had the potential to be an excellent tool molecule and so was fully profiled for its potency, selectivity, and pharmacokinetic attributes. Indeed, the high level of potency and selectivity observed in the BRD4 FRET assay was recapitulated against the other BET bromodomains (BRD2, 3, and T), while <b>59</b> was also sent to DiscoverX and profiled in their BROMOscan platform (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00605/suppl_file/jm0c00605_si_001.pdf" class="ext-link">Table S2</a>, Supporting Information). <b>59</b> shows an excellent level of selectivity for the BET family with no measurable activity for non-BET bromodomains.</div><figure id="fig7" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. BROMOscan profile of <b>59</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Profile of pan-BD2 Selective Probe <b>59</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0020.gif" alt="" id="gr19" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="center" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>BRD4 BD1/BD2 pIC<sub>50</sub></b></td><td class="colsep0 rowsep0" align="center"><b>4.2Â +/âÂ 0.44Â (3/13)/7.3Â +/âÂ 0.13Â (14)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>selectivity (fold)</b></td><td class="colsep0 rowsep0" align="center">>1000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>BD2 LE</b></td><td class="colsep0 rowsep0" align="center">0.33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b><i>C</i> log <i>p</i>/MW</b></td><td class="colsep0 rowsep0" align="center">1.3/414</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>chrom log <i>D</i><sub>7.4</sub>/PFI</b></td><td class="colsep0 rowsep0" align="center">2.8/4.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>CLNDÂ solubilityÂ (Î¼g/mL)/FaSSIFÂ (Î¼g/mL)</b></td><td class="colsep0 rowsep0" align="center">91/996</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>AMP<sub>7.4</sub> permeability (nm/s)</b></td><td class="colsep0 rowsep0" align="center">16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>hERG pIC<sub>50</sub>/HSA binding (%)</b></td><td class="colsep0 rowsep0" align="center"><4.3/47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>CYP3A4 (MDI)</b></td><td class="colsep0 rowsep0" align="center"><4.4</td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="/" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>59</b></td><td class="colsep0 rowsep0" align="center"><b>BD1</b></td><td class="colsep0 rowsep0" align="center" char="/"><b>BD2</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4</td><td class="colsep0 rowsep0" align="left">4.2Â +/âÂ 0.44Â (3/13)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char="/">7.3Â +/âÂ 0.13Â (12)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD2</td><td class="colsep0 rowsep0" align="left">5.0 (1/11)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char="/">6.6 +/â 0.2 (9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD3</td><td class="colsep0 rowsep0" align="left">4.4 +/â 0.01 (2/8)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char="/">7.0 +/â 0.08 (8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRDT</td><td class="colsep0 rowsep0" align="left"><4.3 (8)</td><td class="colsep0 rowsep0" align="char" char="/">6.7 +/â 0.25 (8)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Some results below the curve-fitting threshold of 4.3 could not be included in the mean and SD. In these cases, the numbers included in the mean/number of times tested are both shown.</p></div></div></div><div class="NLM_p">Given the profound efficacy associated with pan-BET inhibitors in models of inflammation and oncology, having such high selectivity makes delineating the respective roles of the BD2 domain of proteins BRD2, 3, 4, and T more facile. <b>59</b> was therefore tested in an LPS-stimulated peripheral blood mononuclear cell (PBMC) cellular assay. After stimulation, these immune cells release a range of cytokines and chemokines, including monocyte chemoattractant protein 1 (MCP-1/CCL2).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> MCP-1 is a key chemokine that regulates migration and infiltration of macrophages and as such is involved in a range of immuno-inflammatory diseases. It has been shown previously that pan-BET and BD2-biased inhibitors are capable of inhibiting an MCP-1 response.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The BD2-selective inhibitor <b>59</b> also inhibits MCP-1 production (pIC<sub>50</sub> = 7.5) with a minimal drop-off from the biochemical BRD4 BD2 potency observed. Not only does this confirm cellular penetration of <b>59</b>, but this also strongly suggests that pan-BD2 inhibitors are capable of maintaining an anti-inflammatory phenotype with minimal BD1 engagement. Pleasingly, this inhibition translated into submicromolar inhibition in human whole blood. A greater drop-off would be expected in this assay as a proportion of the inhibitor will be bound to blood proteins (human blood <i>F</i><sub>ub</sub> = 32%). <b>59</b> also has an encouraging developability profile with a chrom log <i>D</i> of 2.8 and a PFI in the desired range of 4.8. This translates into good kinetic solubility (CLND) and perhaps, more importantly, excellent thermodynamic solubility measured in fasted-state simulated intestinal fluid (FaSSIF) of 996 Î¼g/mL. The predicted permeability using the AMP assay was low; however, we had already shown that this assay was not predictive of cellular penetration using the PBMC assay, so this result was not overtly concerning. In search of an <i>in vivo</i>-capable probe, the pharmacokinetics of <b>59</b> was evaluated (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). Utilizing an external panel of isolated rat, dog, and human hepatocytes, we were able to show that <b>59</b> was not metabolized at the level of detection possible in these <i>in vitro</i> systems, which therefore was predictive of a low/moderate clearance in vivo. To explore this prediction, <b>59</b> was characterized <i>in vivo</i> in the rat and dog by both intravenous and oral routes of administration. Pleasingly, <b>59</b> showed moderate clearance in the rat and dog of 46% and 36% liver blood flow (LBF), respectively, when dosed i.v. After oral administration, <b>59</b> had a bioavailability in the rat of 35% and dog of 66%. These data show that <b>59</b> is a suitable probe for use in <i>in vivo</i> animal models. Indeed, we have recently described further studies with <b>59</b> (also known as iBET-BD2), which show that selective BD2 inhibition is efficacious in a T-cell-dependent immunization model.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div><figure id="fig8" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. <i>In vitro</i> and <i>in vivo</i> pharmacokinetic profile of <b>59</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To understand the origins of the high levels of BD2 selectivity seen with <b>59</b>, crystal structures were obtained in BRD2 BD2 and BRD4 BD1. The binding mode of the molecule to both bromodomains is similar to that of <b>1</b> with the 4-acetamido group occupying the acetyl-lysine pocket and hydrogen-bonding to the conserved W1 water. The 3-benzyloxy group still occupies the WPF shelf region of the site as shown for BRD2 BD2 in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>a, including Val435 and Trp370. A comparison to the BRD2 bound complex of <b>1</b> reveals that <b>59</b> binds with a rotation in the site relative to <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>b) with a less planar conformation of the phenyl-4-acetamido torsion (about 50Â° in the case of <b>59</b> vs 14Â° for <b>1</b>). This is probably due to electrostatic repulsion between the 5-fluoro substituent and the 5-acetamide carbonyl, but maybe the steric constraint exerted upon the 3-position benzylic methyl group of <b>59</b> by the narrow ZA channel region between Trp370 and Leu381 could also contribute. One consequence of the movement is to bring the 3-position benzyl group closer to the side chain of His433, which unambiguously lies in a single <i>gauche</i>(+) conformation with the two rings making an edge-to-face contact. Another is to bring the 1-position amide NH significantly closer to the Asn429 carbonyl oxygen than in the complex with <b>1</b> into direct hydrogen-bonding distance. The cyclohexanol 4-position group is clearly visible and makes predominantly hydrophobic contacts with Pro430 and His433.</div><figure id="fig9" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. X-ray structures of (A) <b>59</b> (cyan, PDB 6swp) and <b>1</b> (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6z7f">6z7f</a>; mobile imidazole not shown for clarity) in BRD2 BD2. (B) <b>59</b> (orange, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6swq">6swq</a>) in BRD4 BD1. (C) <b>59</b> (cyan) with the BRD2 BD2 surface shown in blue. The surface of His433 is shown in yellow and its contacts with <b>59</b> are shown as red dashed lines. (D) <b>59</b> (orange) with the Asp144 surface shown in yellow. (E) Superimposed crystal structures of BRD2 BD2 bound to <b>59</b> (cyan, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6swp">6swp</a>) and ABBV-744 (magenta, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6e6j">6e6j</a><a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The complex of <b>59</b> with BRD4 BD1 is similar to the BRD2 BD2 structure (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>c). The extra potency and selectivity for BD2 over BD1 most likely comes from the significantly greater interactions with His433 in BD2, which include the edgeâface arylâaryl contact with the shelf benzyl substituent as well as the cyclohexanol ring and with Pro430 (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>c). These stabilize the His433 <i>gauche</i>(+) conformation resulting in excellent shape complementarity. In BD1, the His433 equivalent residue Asp144 points away from the inhibitor and makes essentially no direct interaction, resulting in a much smaller contact surface (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>d) and explaining the lower potency. A comparison of the BRD2 BD2 bound structure of <b>59</b> with the BRD4 BD2 bound complex with ABBV-744 shows similar positioning of the amide substituents in each, reinforcing the importance of this group to the BD2 selectivity of both inhibitors (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>e).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38263" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38263" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we have detailed the optimization of a relatively weak BD2-selective HTS hit into <b>59</b>, a potent and highly selective pan-BD2 inhibitor. A screening hit with weak BD2 bias containing a simple AcK mimetic and WPF shelf group was extended to exploit a key residue difference between BD1 and BD2 to generate highly potent and domain-selective BD2 inhibitors. <b>59</b> has suitable physicochemical and pharmacokinetic properties to be utilized <i>in vivo</i>, and it has also been demonstrated that a highly selective BD2 inhibitor retains the ability to potently inhibit MCP-1 cytokine release in a cellular and whole blood context. <b>59</b> proved useful for further biological profiling. Indeed, we have recently published a comparison between the <i>in vitro</i> and <i>in vivo</i> phenotypes of selective pan-BD1<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> and pan-BD2 inhibitors.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> These data show that <b>59</b> is efficacious in a broad range of inflammatory pathologies and selective BD2 inhibition may be a useful new therapeutic strategy for immunoinflammatory diseases.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52272" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52272" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Experimental Procedure</h3><div class="NLM_p last">Unless otherwise stated, all reactions were carried out under an atmosphere of nitrogen in heat or oven-dried glassware and anhydrous solvent. Solvents and reagents were purchased from commercial suppliers and used as received. Reactions were monitored by thin-layer chromatography (TLC) or liquid chromatographyâmass spectrometry (LCâMS). TLC was carried out on glass or aluminum-backed 60 silica plates coated with a UV<sub>254</sub> fluorescent indicator. Spots were visualized using UV light (254 or 365 nm) or alkaline KMnO<sub>4</sub> solution followed by gentle heating. LCMS analysis was carried out on a Waters Acquity UPLC instrument equipped with a CSH C18 column (50 mm Ã 2.1 mm, 1.7 Î¼m in packing diameter) and a Waters micromass ZQ MS using an alternate-scan positive and negative electrospray. Analytes were detected as a summed UV wavelength of 210â350 nm. Two liquid-phase methods were used: formic - 40 Â°C, 1 mL/min flow rate. Gradient elution with the mobile phases was achieved with (A) H<sub>2</sub>O containing 0.1% volume/volume (v/v) formic acid and (B) acetonitrile containing 0.1% (v/v) formic acid. High pH: 40 Â°C, 1 mL/min flow rate. Gradient elution with the mobile phases was also achieved with (A) 10 mM aqueous ammonium bicarbonate solution, adjusted to pH 10 with 0.88 M aqueous ammonia, and (B) acetonitrile. Flash column chromatography was carried out using a Biotage SP4 or Isolera One apparatus with SNAP silica cartridges. Mass-directed automatic purification (MDAP) was carried out using a Waters ZQ MS using an alternate-scan positive and negative electrospray and a summed UV wavelength of 210â350 nm. Two liquid-phase methods were used: formic - Sunfire C18 column (100 mm Ã 19 mm, 5 Î¼m in packing diameter, 20 mL/min flow rate) or Sunfire C18 column (150 mm Ã 30 mm, 5 Î¼m in packing diameter, 40 mL/min flow rate). Gradient elution at ambient temperature with the mobile phases was achieved with (A) H<sub>2</sub>O containing 0.1% volume/volume (v/v) formic acid and (B) acetonitrile containing 0.1% (v/v) formic acid. High pH - Xbridge C18 column (100 mm Ã 19 mm, 5 Î¼m in packing diameter, 20 mL/min flow rate) or Xbridge C18 column (150 mm Ã 30 mm, 5 Î¼m in packing diameter, 40 mL/min flow rate). Gradient elution at ambient temperature with the mobile phases was also achieved with (A) 10 mM aqueous ammonium bicarbonate solution, adjusted to pH 10 with 0.88 M aqueous ammonia, and (B) acetonitrile. NMR spectra were recorded at ambient temperature (unless otherwise stated) using standard pulse methods on any of the following spectrometers and signal frequencies: Bruker AV-400 (<sup>1</sup>H = 400 MHz, <sup>13</sup>C = 101 MHz), Bruker AV-600 (<sup>1</sup>H = 600 MHz, <sup>13</sup>C = 150 MHz), or Bruker AV4 700 MHz spectrometer (<sup>1</sup>H = 700 MHz, <sup>13</sup>C = 176 MHz). Chemical shifts are referenced to trimethylsilane (TMS) or the residual solvent peak and reported in parts per million. Coupling constants are quoted to the nearest 0.1 Hz, and multiplicities are given by the following abbreviations and combinations thereof: s (singlet), Î´ (doublet), t (triplet), q (quartet), quin (quintet), sxt (sextet), m (multiplet), br (broad). The purity of synthesized compounds was determined by LCMS analysis. All compounds for biological testing were >95% pure.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Synthetic Procedures</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Methyl 4-Acetamido-3-hydroxybenzoate (<b>31</b>)</h4><div class="NLM_p last">To a solution of methyl 4-amino-3-hydroxybenzoate (22 g, 132 mmol, commercially available from a supplier such as Fluorochem) in <i>p</i>-xylene (440 mL) and THF (220 mL) was added acetyl chloride (9.39 mL, 132 mmol) at rt, and the reaction was stirred at 70 Â°C for 16 h. The reaction was cooled to rt and partitioned between H<sub>2</sub>O and EtOAc, and the organics were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated <i>in vacuo</i> to afford crude methyl 4-acetamido-3-hydroxybenzoate (24 g, 106 mmol, 92% yield) as a brown solid. LCMS (formic modifier) retention time of 3.46 min, [M + H]<sup>+</sup> = 209.9. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 10.10â10.50 (m, 1H), 9.34 (s, 1H), 8.09 (br d, 1H, <i>J</i> = 8.6 Hz), 7.30â7.50 (m, 2H), 3.80 (s, 3H), 2.14 (s, 3H).</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Methyl 4-Acetamido-3-(benzyloxy)benzoate (<b>32</b>)</h4><div class="NLM_p">To a solution of methyl 4-acetamido-3-hydroxybenzoate (24 g, 115 mmol) in DMF (240 mL) was added K<sub>2</sub>CO<sub>3</sub> (23.78 g, 172 mmol) and stirred at rt for 20 min, and to this, benzyl bromide (17.74 mL, 149 mmol) in DMF (20 mL) was added, and the reaction was stirred at 70 Â°C for 16 h. The reaction mass was cooled to rt and poured on crushed ice and extracted with EtOAc. The organic phase was washed with cold H<sub>2</sub>O, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i> to afford a brown sticky solid. This solid was triturated with DCM/pentane to afford methyl 4-acetamido-3-(benzyloxy)benzoate (25 g, 82 mmol, 71%) as a brown solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 2.26 min, [M + H]<sup>+</sup> = 300.0. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) Î´ 9.32 (s, 1H), 8.16 (d, 1H, <i>J</i> = 8.1 Hz), 7.55â7.59 (m, 1H), 7.50â7.55 (m, 2H), 7.40â7.45 (m, 2H), 7.30â7.35 (m, 1H), 5.30 (s, 2H), 3.81 (s, 3H), 2.15 (s, 3H).</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 4-Acetamido-3-(benzyloxy)benzoic Acid (<b>33</b>)</h4><div class="NLM_p">To a solution of methyl 4-acetamido-3-(benzyloxy)benzoate (15 g, 50.1 mmol) in THF (60 mL)/MeOH (60 mL) was added a solution of LiOH (4.21 g, 100 mmol) in H<sub>2</sub>O (30 mL) at 0 Â°C, and the reaction was stirred at rt for 16 h. The reaction was concentrated <i>in vacuo</i> and diluted with H<sub>2</sub>O and extracted with EtOAc. The aqueous phase was cooled to 0 Â°C and taken to pH = 3 with 1 N HCl (aq) and extracted with EtOAc. The organic phase was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i> to afford crude 4-acetamido-3-(benzyloxy)benzoic acid (12 g, 41.5 mmol, 83%) as an off-white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 1.01 min, [M + H]<sup>+</sup> = 286.0. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) Î´ 12.55â12.86 (m, 1H), 9.29 (s, 1H), 8.11 (d, 1H, <i>J</i> = 8.3 Hz), 7.55â7.65 (m, 3H), 7.40 (t, 2H, <i>J</i> = 7.5 Hz), 7.30â7.35 (m, 1H), 5.29 (s, 2H), 2.14 (s, 3H).</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> <i>N</i>-(2-(1<i>H</i>-Imidazol-4-yl)ethyl)-4-acetamido-3-(benzyloxy)benzamide (<b>1</b>)</h4><div class="NLM_p">To a solution of 4-acetamido-3-(benzyloxy)benzoic acid (300 mg, 1.052 mmol) in DMF (2 mL) stirred under N<sub>2</sub> at rt was added DIPEA (0.367 mL, 2.103 mmol) and HATU (600 mg, 1.577 mmol), and the reaction was stirred for 10 min. To the resulting solution, 2-(1<i>H</i>-imidazol-5-yl)ethanamine (140 mg, 1.262 mmol) was added, and the reaction was stirred at rt for 16 h. The reaction was poured onto ice water and extracted with EtOAc. The organic phase was washed with H<sub>2</sub>O, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated to give a light brown solid. This solid was purified using prep HPLC, column: X Bridge C18 (250 mm Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 0â100%, to afford <i>N</i>-(2-(1<i>H</i>-imidazol-5-yl)ethyl)-4-acetamido-3-(benzyloxy)benzamide (190 mg, 0.499 mmol, 47.4% yield) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 0.56 min, [M + H]<sup>+</sup> = 379.3. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) Î´ 9.20 (s, 1H), 8.45 (br t, 1H, <i>J</i> = 5.4 Hz), 8.01 (br d, 1H, <i>J</i> = 7.9 Hz), 7.54â7.56 (m, 3H), 7.38â7.43 (m, 3H), 7.30â7.35 (m, 1H), 5.25â5.27 (m, 2H), 3.42â3.50 (m, 2H), 2.70â2.77 (m, 2H), 2.11â2.13 (m, 3H). Similarly prepared were the following.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 4-Acetamido-3-(benzyloxy)-<i>N</i>,<i>N</i>-dimethylbenzamide (<b>4</b>)</h4><div class="NLM_p">Purification by prep HPLC. Column: X Bridge C18 (250 mm Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 0â100% to give 4-acetamido-3-(benzyloxy)-<i>N</i>,<i>N</i>-dimethylbenzamide (147 mg, 0.468 mmol, 66.7% yield) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 0.77 min, [M + H]<sup>+</sup> = 313.3. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) Î´ 9.20 (br s, 1H), 7.95 (br d, 1H, <i>J</i> = 8.1 Hz), 7.45â7.50 (m, 2H), 7.35â7.39 (m, 2H), 7.31 (s, 1H), 7.06 (d, 1H, <i>J</i> = 1.5 Hz), 6.93 (dd, 1H, <i>J</i> = 1.4, 8.2 Hz), 5.25 (s, 2H), 2.78â2.94 (m, 6H), 2.11 (s, 3H).</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 4-Acetamido-3-(benzyloxy)-<i>N</i>-(3-hydroxypropyl)benzamide (<b>9</b>)</h4><div class="NLM_p">Purification by prep HPLC. Column: X Bridge C18 (250 mm Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 0â100% to give 4-acetamido-3-(benzyloxy)-<i>N</i>-(3-hydroxypropyl)benzamide (133 mg, 0.387 mmol, 55.2 yield) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 0.70 min, [M + H]<sup>+</sup> = 343.4. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) Î´ 9.20 (s, 1H), 8.32 (br s, 1H), 8.00 (br d, 1H, <i>J</i> = 8.1 Hz), 7.45â7.54 (m, 3H), 7.40 (t, 3H, <i>J</i> = 7.8 Hz), 7.30â7.36 (m, 1H), 5.26 (s, 2H), 4.44 (t, 1H, <i>J =</i> 5.2 Hz), 3.45â3.51 (m, 2H), 2.97 (br t, <i>J =</i> 6.25 Hz 2H), 2.11 (s, 3H), 1.65â1.70 (m, 2H).</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 4-Acetamido-3-(benzyloxy)-<i>N</i>-((1<i>R</i>,4<i>S</i>)-4-hydroxycyclohexyl)benzamide (<b>14</b>)</h4><div class="NLM_p">To a solution of 4-acetamido-3-(benzyloxy)benzoic acid (200 mg, 0.701 mmol), HATU (400 mg, 1.052 mmol), and DIPEA (0.245 mL, 1.402 mmol) in DMF (5 mL) stirred under nitrogen at 0 Â°C was added <i>trans</i>-4-aminocyclohexanol (97 mg, 0.841 mmol). The reaction mixture was stirred at rt for 18 h. The reaction was partitioned between water and EtOAc, and the organic layer was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i> to afford a light brown solid. Purification by trituration with 10% EtOAc and Et<sub>2</sub>O to give 4-acetamido-3-(benzyloxy)-<i>N</i>-((1<i>R</i>,4<i>S</i>)-4-hydroxycyclohexyl)benzamide (154 mg, 0.400 mmol, 57.1% yield) as an off-white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 1.77 min, [M + H]<sup>+</sup> = 383.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.19 (s, 1H), 7.95â8.04 (m, 2H), 7.51 (d, <i>J</i> = 8.1 Hz, 3H), 7.37â7.44 (m, 3H), 7.30â7.35 (m, 1H), 5.25 (s, 2H), 4.52 (d, <i>J =</i> 4.4 Hz, 1H), 3.63â3.75 (m, 1H), 3.33â3.43 (m, 1H), 2.11 (s, 3H), 1.75â1.88 (m, 4H), 1.14â1.43 (m, 4H).</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 4-Acetamido-3-(benzyloxy)-<i>N</i>-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)benzamide (<b>13</b>)</h4><div class="NLM_p">To a solution 4-acetamido-3-(benzyloxy)benzoic acid (35 mg, 0.123 mmol) and HATU (46.6 mg, 0.123 mmol) in DMF (0.6 mL) was added DIPEA (0.064 mL, 0.368 mmol) 2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethanamine (21 mg, 0.159 mmol) and stirred for 18 h at rt. The reaction was purified by mass-directed AutoPrep on Waters CSH C18 30 Ã 150 mm, 5 Î¼m, using acetonitrile water with an ammonium carbonate modifier to give 4-acetamido-3-(benzyloxy)-<i>N</i>-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)benzamide (2.2 mg, 0.005 mmol), 4% yield.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 0.93 min, [M + H]<sup>+</sup> = 397.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.24 (s, 1H), 8.33 (t, <i>J</i> = 5.3 Hz, 1H), 8.01 (d, <i>J</i> = 7.6 Hz, 1H), 7.49â7.56 (m, 3H), 7.37â7.43 (m, 3H), 7.30â7.35 (m, 1H), 5.27 (s, 2H), 3.82 (dd, <i>J</i> = 11.1, 3.2 Hz, 2H), 3.22â3.31 (m, 4H), 2.12 (s, 3H), 1.60 (d, <i>J</i> = 12.8 Hz, 2H), 1.52 (dd, <i>J</i> = 7.0, 3.6 Hz, 1H), 1.45 (q, <i>J</i> = 6.8 Hz, 2H), 1.10â1.22 (m, 2H).</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 4-Acetamido-3-(benzyloxy)-<i>N</i>-methylbenzamide (<b>3</b>)</h4><div class="NLM_p">To a solution of 4-acetamido-3-(benzyloxy)benzoic acid (200 mg, 0.701 mmol) in DMF (2 mL) stirred under N<sub>2</sub> at rt were added DIPEA (0.245 mL, 1.402 mmol) and T<sub>3</sub>P (0.626 mL, 1.052 mmol), and the reaction was stirred for 10 min. To this solution, methanamine hydrochloride (56.8 mg, 0.841 mmol) was added, and the reaction was stirred at rt for 16 h. Further, T<sub>3</sub>P (0.104 mL, 0.351 mmol) was added and the reaction was stirred for 18 h. The reaction was poured onto ice and extracted with EtOAc. The organic phase was washed with water, brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i> to afford a pale yellow solid. The crude was purified by prep HPLC. Column: X Bridge C18 (150 mm Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 15â100% to give 4-acetamido-3-(benzyloxy)-<i>N</i>-methylbenzamide (63 mg, 0.209 mmol, 29.9% yield) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 1.75 min, [M + H]<sup>+</sup> = 299.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.23 (s, 1H), 8.33 (d, <i>J</i> = 4.38 Hz, 1H), 8.03 (d, <i>J</i> = 8.11 Hz, 1H), 7.51â7.58 (m, 3H), 7.39â7.46 (m, 3H), 7.36 (d, <i>J</i> = 7.23 Hz, 1H), 5.28 (s, 2H), 2.79 (d, <i>J</i> = 4.60 Hz, 3H), 2.14 (s, 3H). Similarly prepared were the following.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 4-Acetamido-3-(benzyloxy)-<i>N</i>-ethylbenzamide (<b>5</b>)</h4><div class="NLM_p">Second addition of T<sub>3</sub>P is not required. Purification by prep HPLC. Column: phenyl hexyl (250 mm Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 20â100% to give 4-acetamido-3-(benzyloxy)-<i>N</i>-ethylbenzamide (104 mg, 0.333 mmol, 47.4% yield) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 1.88 min, [M + H]<sup>+</sup> = 313.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.22 (s, 1H), 8.36 (t, <i>J</i> = 5.5 Hz, 1H), 8.03 (d, <i>J</i> = 8.3 Hz, 1H), 7.50â7.60 (m, 3H), 7.40â7.47 (m, 3H), 7.32â7.38 (m, 1H), 5.29 (s, 2H), 3.28 (q, <i>J</i> = 1.0 Hz, 2H), 2.14 (s, 3H), 1.14 (t, <i>J</i> = 7.2 Hz, 3H).</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 4-Acetamido-3-(benzyloxy)-<i>N</i>-(2-hydroxyethyl)benzamide (<b>6</b>)</h4><div class="NLM_p">Second addition of T<sub>3</sub>P is not required. Purification by prep HPLC. Column: X Bridge C18 (150 mm Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 0â100% to give 4-acetamido-3-(benzyloxy)-<i>N</i>-(2-hydroxyethyl)benzamide (110 mg, 0.334 mmol, 47.7% yield) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 0.67 min, [M + H]<sup>+</sup> = 329.2. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) Î´ 9.19 (s, 1H), 8.32 (br t, 1H, <i>J</i> = 5.3 Hz), 8.01 (br d, 1H, <i>J</i> = 7.7 Hz), 7.57â7.59 (m, 1H), 7.54â7.51 (m, 2H), 7.37â7.46 (m, 3H), 7.33â7.36 (m, 1H), 5.26 (s, 2H), 4.68 (t, 1H, <i>J =</i> 5.5 Hz), 3.45â3.51 (m, 2H), 3.28â3.31 (m, 2H), 2.07â2.15 (m, 3H).</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 4-Acetamido-3-(benzyloxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (<b>10</b>)</h4><div class="NLM_p">Second addition of T<sub>3</sub>P is not required. Purification by prep HPLC. Column: X TERRA RP18 (250 mm Ã 19 mm Ã 10 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 17 mL/min, and gradient: 0â100% to give 4-acetamido-3-(benzyloxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (116 mg, 0.308 mmol, 44.0% yield) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 1.85 min, [M + H]<sup>+</sup> = 369.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.23 (s, 1H), 8.18 (d, <i>J</i> = 7.7 Hz, 1H), 8.03 (d, <i>J</i> = 8.1 Hz, 1H), 7.51â7.58 (m, 3H), 7.39â7.49 (m, 3H), 7.31â7.38 (m, 1H), 5.29 (s, 2H), 3.94â4.06 (m, 1H), 3.90 (dd, <i>J =</i> 11.7, 2.1 Hz, 2H), 3.40 (td, <i>J</i> = 11.7, 2.1 Hz, 2H), 2.14 (s, 3H), 1.76 (dd, <i>J =</i> 12.6, 2.3 Hz, 2H), 1.59 (qd, <i>J =</i> 12.0, 4.4 Hz, 2H).</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 4-Acetamido-3-(benzyloxy)-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)benzamide (<b>11</b>)</h4><div class="NLM_p">Second addition of T<sub>3</sub>P is not required. Purification by prep HPLC. Column: X Bridge C18 (150 mm Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and isocratic gradient: 60% to give 4-acetamido-3-(benzyloxy)-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)benzamide (55 mg, 0.143 mmol, 20.43% yield) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 1.87 min, [M + H]<sup>+</sup> = 383.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.23 (s, 1H), 8.38 (t, <i>J</i> = 5.7 Hz, 1H), 8.03 (d, <i>J =</i> 8.3 Hz, 1H), 7.51â7.58 (m, 3H), 7.39â7.47 (m, 3H), 7.36 (d, <i>J</i> = 7.2 Hz, 1H), 5.29 (s, 2H), 3.87 (dd, <i>J</i> = 11.4, 2.4 Hz, 2H), 3.23â3.34 (m, 4 H), 3.16 (t, <i>J</i> = 6.4 Hz, 2H), 2.14 (s, 3H), 1.80 (br s, 1H), 1.59 (d, <i>J</i> = 12.7 Hz, 2H), 1.21 (qd, <i>J</i> = 12.2, 4.0 Hz, 2H).</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 4-Acetamido-<i>N</i>-(2-aminoethyl)-3-(benzyloxy)benzamide (<b>7</b>)</h4><div class="NLM_p">To a solution of 4-acetamido-3-(benzyloxy)benzoic acid (200 mg, 0.701 mmol) in DMF (2 mL) stirred under N<sub>2</sub> at rt were added DIPEA (0.245 mL, 1.402 mmol) and HATU (400 mg, 1.052 mmol), and the reaction was stirred for 10 min. To this solution, <i>tert</i>-butyl (2-aminoethyl)carbamate (0.133 mL, 0.841 mmol) was added, and the reaction was stirred at rt for 16 h. The reaction was poured onto ice water and extracted with EtOAc. The organic phase was washed with water, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i> to afford an off-white sticky solid. This solid was purified by silica gel column chromatogrpahy eluting with an isocratic gradient of 10% MeOH in DCM to afford the boc-protected title compound as a white solid. This solid was stirred with 1,4-dioxane at 0 Â°C and treated with 4 M HCl in 1,4-dioxane (0.5 mL, 2.00 mmol) and stirred at rt for 2 h. The reaction was concentrated <i>in vacuo</i> and washed successively with DCM/pentane (1:1) and Et<sub>2</sub>O and dried to give 4-acetamido-<i>N</i>-(2-aminoethyl)-3-(benzyloxy)benzamide dihydrochloride (20 mg, 0.05 mmol, 21%) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 0.54 min, [M + H]<sup>+</sup> = 328.2. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) Î´ 9.2â9.3 (m, 1H), 8.57 (m, 1H), 8.04â8.07 (m, 1H), 7.77â7.80 (m, 2H), 7.60â7.61 (m, 1H), 7.50â7.54 (m, 2H), 7.45â7.48 (m, 1H), 7.37â7.43 (m, 2H), 7.30â7.36 (m, 1H), 5.27â5.28 (m, 2H), 3.45â3.52 (m, 2H), 2.97 (br t, 2H, <i>J</i> = 6.2 Hz), 2.11â2.13 (m, 3H).</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 4-Acetamido-<i>N</i>-(3-aminopropyl)-3-(benzyloxy)benzamide 2,2,2-trifluoroacetate (<b>8</b>)</h4><div class="NLM_p">To a solution of 4-acetamido-3-(benzyloxy)benzoic acid (70 mg, 0.245 mmol), <i>tert</i>-butyl 3-aminopropylcarbamate (86 mg, 0.491 mmol) and HATU (187 mg, 0.491 mmol) in DMF (1.5 mL) were added DIPEA (0.129 mL, 0.736 mmol). Then, the solution was stirred at 25 Â°C for 5 h. The reaction was partitioned between EtOAc and H<sub>2</sub>O, and the organic layer was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i> to give a yellow oil. This oil was taken up with DCM (1 mL) and treated with TFA (0.995 mL, 12.91 mmol), and the reaction was stirred at rt overnight. The reaction was concentrated <i>in vacuo</i> and purified by reversed-phase (C18) chromatography eluting with a gradient of 5â35% MeOHâwater (0.05% TFA) to give 4-acetamido-<i>N</i>-(3-aminopropyl)-3-(benzyloxy)benzamide (22 mg, 0.046 mmol, 13.9% yield) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 1.31 min, [M + H]<sup>+</sup> = 342.1. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) Î´ ppm 8.14 (d, <i>J</i> = 8.3 Hz, 1H), 7.59 (d, <i>J</i> = 1.8 Hz, 1H), 7.49 (d, <i>J</i> = 7.0 Hz, 2H), 7.38â7.46 (m, 3H), 7.31â7.37 (m, 1H), 5.27 (s, 2H), 3.49 (t, <i>J</i> = 6.7 Hz, 2H), 2.99 (t, <i>J</i> = 7.3 Hz, 2H), 2.18 (s, 3H), 1.95 (quin, <i>J =</i> 6.9 Hz, 2H).</div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 4-Acetamido-3-(benzyloxy)-<i>N</i>-(piperidin-4-ylmethyl)benzamide (<b>12</b>)</h4><div class="NLM_p">To a solution of 4-acetamido-3-(benzyloxy)benzoic acid (200 mg, 0.701 mmol) in DMF (5 mL) stirred under N<sub>2</sub> at 20 Â°C were added DIPEA (0.367 mL, 2.103 mmol) and HATU (400 mg, 1.052 mmol). The reaction mixture was stirred for 10 min then <i>tert</i>-butyl 4-(aminomethyl)piperidine-1-carboxylate (180 mg, 0.841 mmol) was added, and the reaction mixture was stirred at rt for 12 h. The reaction was poured onto ice and extracted with EtOAc, and the organic layer was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure to give a crude boc-protected product. The crude was purified using silica gel column chromatography eluting with a gradient of 0â10% MeOH in DCM to give an impure boc-protected product, which was again purified using silica gel column chromatography eluting with a gradient of 0â90% EtOAc in hexane to give the boc-protected desired a white solid. This solid was taken up in 1,4-dioxane (1 mL) and treated with 4 M HCl in 1,4-dioxane (0.125 mL, 0.498 mmol). The reaction mixture was stirred at rt for 12 h. The reaction was concentrated and triturated with acetone and Et<sub>2</sub>O to give 4-acetamido-3-(benzyloxy)-<i>N</i>-(piperidin-4-ylmethyl)benzamide hydrochloride (31 mg, 0.073 mmol, 29.2% yield) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 0.57 min, [M + H]<sup>+</sup> = 382.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub> + D<sub>2</sub>O) Î´ ppm 8.50â8.56 (m, 1H), 8.00 (br d, <i>J =</i> 8.11 Hz, 1H), 7.48â7.54 (m, 2H), 7.38â7.45 (m, 3H), 7.30â7.36 (m, 1H), 5.24â5.31 (m, 2H), 3.24â3.32 (m, 2H), 3.16â3.22 (m, 2H), 2.79â2.90 (m, 2H), 2.09â2.16 (m, 3H), 1.77â1.87 (m, 3H), 1.28â1.41 (m, 2H).</div></div><div id="sec4_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 4-Acetamido-3-hydroxybenzoic Acid (<b>35</b>)</h4><div class="NLM_p">Acetyl chloride (11.14 mL, 157 mmol) was added dropwise to a solution of 4-amino-3-hydroxybenzoic acid (20 g, 131 mmol) in <i>p</i>-xylene (300 mL) and THF (150 mL) at rt, and the reaction mixture was stirred at 70 Â°C for 5 h. The reaction was diluted with water and extracted with EtOAc and dried with Na<sub>2</sub>SO<sub>4</sub> and concentrated to give 4-acetamido-3-hydroxybenzoic acid (19 g, 97 mmol, 74.5% yield).</div><div class="NLM_p last">LCMS (formic modifier) retention time of 1.20 min, [M + H]<sup>+</sup> = 196.0. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) Î´ 12.58 (br s, 1H), 10.25 (br s, 1H), 9.34 (s, 1H), 8.02â8.08 (m, 1H), 7.43â7.47 (m, 1H), 7.35â7.40 (m, 1H), 2.13 (s, 3H).</div></div><div id="sec4_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> <i>tert</i>-Butyl (3-(4-Acetamido-3-hydroxybenzamido)propyl)carbamate (<b>36</b>)</h4><div class="NLM_p"><i>tert</i>-Butyl (3-aminopropyl)carbamate (67.0 mg, 0.384 mmol) was added to 4-acetamido-3-hydroxybenzoic acid (50 mg, 0.256 mmol), EDC (49.1 mg, 0.256 mmol), and HOBT (39.2 mg, 0.256 mmol) in DMF (10 mL). The reaction was stirred at rt under N<sub>2</sub> for 16 h. The reaction was diluted with water, extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated to give crude <i>tert</i>-butyl (3-(4-acetamido-3-hydroxybenzamido)propyl)carbamate (65 mg, 0.150 mmol, 56% yield).</div><div class="NLM_p last">LCMS (formic modifier) retention time of 1.96 min, [M + H]<sup>+</sup> = 352.1.</div></div><div id="sec4_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 4-Acetamido-3-hydroxy-<i>N</i>-(3-(2,2,2-trifluoroacetamido)propyl)benzamide (<b>37</b>)</h4><div class="NLM_p">To a solution of 4-acetamido-3-hydroxybenzoic acid (568 mg, 2.91 mmol), <i>N</i>-(3-aminopropyl)-2,2,2-trifluoroacetamide (495 mg, 2.91 mmol), HOBT (490 mg, 3.20 mmol), and EDC (614 mg, 3.20 mmol) in DMF (20 mL) stirred under N<sub>2</sub> at rt were added. TEA (1.217 mL, 8.73 mmol) was added, and the reaction was stirred at rt. The reaction mixture was quenched with water and extracted with EtOAc, and the organic layer was washed with water, dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated under reduced pressure to give 4-acetamido-3-hydroxy-<i>N</i>-(3-(2,2,2-trifluoroacetamido)propyl)benzamide (520 mg, 1.288 mmol, 44.3% yield).</div><div class="NLM_p last">LCMS (formic modifier) retention time of 1.45 min, [M + H]<sup>+</sup> = 348.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub> + D<sub>2</sub>O) Î´ ppm 7.88 (d, <i>J</i> = 8.33 Hz, 1H), 7.35 (d, <i>J</i> = 1.97 Hz, 1H), 7.26 (dd, <i>J</i> = 8.44, 2.08 Hz, 1H), 3.24 (q, <i>J</i> = 6.50 Hz, 4H), 2.11â2.13 (m, 3H), 1.69â1.78 (m, 2H).</div></div><div id="sec4_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 4-Acetamido-<i>N</i>-(3-aminopropyl)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)benzamide (<b>15</b>)</h4><div class="NLM_p">To a solution of <i>tert</i>-butyl (3-(4-acetamido-3-hydroxybenzamido)propyl)carbamate (200 mg, 0.569 mmol) and K<sub>2</sub>CO<sub>3</sub> (118 mg, 0.854 mmol) in DMF (3 mL) was added 4-(bromomethyl)tetrahydro-2<i>H</i>-pyran (112 mg, 0.626 mmol) dropwise over 1 min, and the reaction mixture was stirred at rt for 10 min then heated at 60 Â°C for 2 h. The reaction mixture was diluted with water and extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated <i>in vacuo</i> to give the crude, which was purified by HPLC. Column: X Bridge C18 (150 mm Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 0â10%. The fractions containing the product were concentrated and then diluted with water and extracted with DCM, and the organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to give the boc-protected target compound. This was taken up in 1,4-dioxane and treated with 4 M HCl in dioxane (1 mL, 4.00 mmol), and the reaction was stirred at rt for 3 h. The reaction mixture was concentrated, and the residue was azeotroped with Et<sub>2</sub>O and MeCN and dried <i>in vacuo</i> to give 4-acetamido-<i>N</i>-(3-aminopropyl)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)benzamide hydrochloride (72 mg, 0.186 mmol, 69.6% yield) as a white solid.</div><div class="NLM_p last">LCMS (HpH modifier) retention time 2.37 min, [M + H]<sup>+</sup> = 350.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.02 (s, 1H), 8.67 (t, <i>J</i> = 5.5 Hz, 1H), 8.03 (d, <i>J</i> = 8.3 Hz, 1H), 7.91 (br s, 3H), 7.54 (s, 1H), 7.47 (d, <i>J</i> = 9.1 Hz, 1H), 3.87â4.00 (m, 4H), 3.33â3.42 (m, 4H), 2.85 (br s, 2H), 2.14 (s, 3H), 1.74â1.88 (m, 4H), 1.30â1.45 (m, 2H).</div></div><div id="sec4_2_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 4-Acetamido-<i>N</i>-(3-aminopropyl)-3-(pyridin-4-ylmethoxy)benzamide (<b>16</b>)</h4><div class="NLM_p">To a solution of <i>tert</i>-butyl (3-(4-acetamido-3-hydroxybenzamido)propyl)carbamate (200 mg, 0.569 mmol) and K<sub>2</sub>CO<sub>3</sub> (197 mg, 1.423 mmol) in DMF (3 mL) was added 4-(chloromethyl)pyridine hydrochloride (103 mg, 0.626 mmol) dropwise over 1 min, and the reaction mixture was stirred at rt for 10 min then heated at 60 Â°C for 2 h. The reaction mixture was diluted with water and extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated <i>in vacuo</i> to give the crude, which was purified by HPLC. Column: X Bridge C18 (150 mm Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 15â100%. The fractions containing the product were concentrated and then diluted with water and extracted with DCM, and the organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to give the boc-protected target compound. This was taken up in 1,4-dioxane (3 mL) and treated with 4 M HCl in dioxane (0.5 mL, 2.00 mmol), and the reaction was stirred at rt for 6 h. The reaction mixture was concentrated, and the residue was azeotroped with Et<sub>2</sub>O and MeCN and dried <i>in vacuo</i> to give 4-acetamido-<i>N</i>-(3-aminopropyl)-3-(pyridin-4-ylmethoxy)benzamide hydrochloride (82 mg, 0.209 mmol, 93% yield) as a tan-colored solid.</div><div class="NLM_p last">LCMS (HpH modifier) retention time of 2.24 min, [M + H]<sup>+</sup> = 343.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.44 (s, 1H), 8.75 (d, <i>J</i> = 5.5 Hz, 3H), 7.95â8.06 (m, 3H), 7.80 (d, <i>J</i> = 3.7 Hz, 2H), 7.64 (d, <i>J</i> = 1.3 Hz, 1H), 7.52 (dd, <i>J =</i> 8.4, 1.4 Hz, 1H), 5.49 (s, 2H), 3.33 (q, <i>J</i> = 6.3 Hz, 2H), 2.84 (q, <i>J</i> = 6.3 Hz, 2H), 2.18 (s, 3H), 1.83 (quin, <i>J</i> = 6.9 Hz, 2H).</div></div><div id="sec4_2_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 4-Acetamido-<i>N</i>-(3-aminopropyl)-3-(pyridin-3-ylmethoxy)benzamide (<b>17</b>)</h4><div class="NLM_p">To a solution of <i>tert</i>-butyl (3-(4-acetamido-3-hydroxybenzamido)propyl)carbamate (200 mg, 0.569 mmol) and K<sub>2</sub>CO<sub>3</sub> (197 mg, 1.423 mmol) in DMF (5 mL) was added 3-(chloromethyl)pyridine hydrochloride (103 mg, 0.626 mmol) dropwise over 1 min, and the reaction mixture was stirred at rt for 10 min then heated at 60 Â°C for 2 h. The reaction mixture was diluted with water and extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated in vacuo to give the crude, which was purified by HPLC. Column: X Bridge C18 (250 mm Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 20â100%. The fractions containing the product were concentrated and then diluted with water and extracted with DCM, and the organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to give the boc-protected target compound. This was taken up in 1,4-dioxane (3 mL) and treated with 4 M HCl in dioxane (0.5 mL, 2.00 mmol), and the reaction was stirred at rt for 4 h. The reaction mixture was concentrated, and the residue was azeotroped with Et<sub>2</sub>O and MeCN and dried <i>in vacuo</i> to give 4-acetamido-<i>N</i>-(3-aminopropyl)-3-(pyridin-3-ylmethoxy)benzamide hydrochloride (82 mg, 0.211 mmol, 98% yield) as a white solid.</div><div class="NLM_p last">LCMS (HpH modifier) retention time of 3.41 min, [M + H]<sup>+</sup> = 343.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.36 (s, 1H), 9.01 (s, 1H), 8.77 (d, <i>J</i> = 5.0 Hz, 1H), 8.74 (t, <i>J</i> = 5.8 Hz, 1H), 8.41 (d, <i>J</i> = 8.5 Hz, 1H), 8.05 (d, <i>J</i> = 8.1 Hz, 1H), 7.91 (br s, 2H), 7.80â7.86 (m, 1H), 7.72 (d, <i>J</i> = 1.5 Hz, 1H), 7.53 (dd, <i>J</i> = 8.3, 1.3 Hz, 1H), 5.45 (s, 2H), 3.36 (q, <i>J</i> = 6.4 Hz, 2H), 2.86 (q, <i>J</i> = 6.3 Hz, 2H), 2.16 (s, 3H), 1.84 (quin, <i>J</i> = 6.9 Hz, 2H).</div></div><div id="sec4_2_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 4-Acetamido-<i>N</i>-(3-aminopropyl)-3-(pyridin-2-ylmethoxy)benzamide (<b>18</b>)</h4><div class="NLM_p">To a solution of <i>tert</i>-butyl (3-(4-acetamido-3-hydroxybenzamido)propyl)carbamate (200 mg, 0.569 mmol) and K<sub>2</sub>CO<sub>3</sub> (197 mg, 1.423 mmol) in DMF (5 mL) was added 2-(chloromethyl)pyridine hydrochloride (103 mg, 0.626 mmol) dropwise over 1 min, and the reaction mixture was stirred at rt for 10 mi then heated at 60 Â°C for 2 h. The reaction mixture was diluted with water and extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated <i>in vacuo</i> to give the crude, which was azeotroped with Et<sub>2</sub>O and pentane and concentrated to give the boc-protected target compound. This was taken up in 1,4-dioxane (3 mL) and treated with 4 M HCl in dioxane (0.5 mL, 2.00 mmol), and the reaction was stirred at rt for 4 h. The reaction mixture was concentrated, and the residue was azeotroped with Et<sub>2</sub>O and MeCN and dried <i>in vacuo</i> to give: 4-acetamido-<i>N</i>-(3-aminopropyl)-3-(pyridin-2-ylmethoxy)benzamide hydrochloride (28 mg, 0.073 mmol, 35.9% yield) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 0.37 min, [M + H]<sup>+</sup> = 343.4. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub> + D<sub>2</sub>O) Î´ ppm 8.68â8.74 (m, 1H), 8.06â8.15 (m, 2H), 7.76â7.83 (m, 1H), 7.56â7.64 (m, 2H), 7.47â7.53 (m, 1H), 5.40â5.45 (m, 2H), 3.28â3.36 (m, 2H), 2.79â2.87 (m, 2H), 2.14â2.18 (m, 3H), 1.77â1.86 (m, 2H).</div></div><div id="sec4_2_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 4-Acetamido-<i>N</i>-(3-aminopropyl)-3-((4-chlorobenzyl)oxy)benzamide (<b>19</b>)</h4><div class="NLM_p">To a solution of <i>tert</i>-butyl (3-(4-acetamido-3-hydroxybenzamido)propyl)carbamate (200 mg, 0.569 mmol) and K<sub>2</sub>CO<sub>3</sub> (118 mg, 0.854 mmol) in DMF (3 mL) was added 1-(bromomethyl)-4-chlorobenzene (129 mg, 0.626 mmol) dropwise over 1 min, and the reaction mixture was stirred at rt for 10 min then heated at 60 Â°C for 2 h. The reaction mixture was diluted with water and extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated <i>in vacuo</i> to give the crude, which was purified by HPLC. Column: X Bridge C18 (150 Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 0â80%. The fractions containing the product were concentrated and then diluted with water and extracted with DCM, and the organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to give the boc-protected target compound. This was taken up in 1,4-dioxane (2 mL) and treated with 4 M HCl in dioxane (0.018 mL, 0.074 mmol), and the reaction was stirred at rt for 16 h. The reaction mixture was concentrated, and the residue was azeotroped with Et<sub>2</sub>O and dried <i>in vacuo</i> to give: 4-acetamido-<i>N</i>-(3-aminopropyl)-3-((4-chlorobenzyl)oxy)benzamide hydrochloride (26 mg, 0.060 mmol, 82% yield) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 0.63 min, [M + H]<sup>+</sup> = 376.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.28 (s, 1H), 8.63 (t, <i>J</i> = 5.7 Hz, 1H), 8.05 (d, <i>J =</i> 8.3 Hz, 1H), 7.83 (br s, 2H), 7.56â7.61 (m, 3H), 7.46â7.51 (m, 3H), 5.30 (s, 2H), 3.34 (d, <i>J</i> = 5.3 Hz, 2H), 2.85 (q, <i>J</i> = 6.3 Hz, 2H), 2.15 (s, 3H), 1.82 (quin, <i>J</i> = 7.0 Hz, 2H).</div></div><div id="sec4_2_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 4-Acetamido-<i>N</i>-(3-aminopropyl)-3-((3-chlorobenzyl)oxy)benzamide (<b>20</b>)</h4><div class="NLM_p">To a solution of <i>tert</i>-butyl (3-(4-acetamido-3-hydroxybenzamido)propyl)carbamate (300 mg, 0.512 mmol, 60 wt %) and K<sub>2</sub>CO<sub>3</sub> (106 mg, 0.768 mmol) in DMF (5 mL) was added 1-(bromomethyl)-3-chlorobenzene (116 mg, 0.563 mmol) dropwise over 1 min, and the reaction mixture was stirred at rt for 10 min then heated at 60 Â°C for 2 h. The reaction mixture was diluted with water and extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated <i>in vacuo</i> to give the crude, which was purified by HPLC. Column: X Bridge C18 (150 mm Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mm ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 10â95%. The fractions containing the product were concentrated and then diluted with water and extracted with DCM, and the organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to give the boc-protected target compound. This was taken up in 1,4-dioxane (3 mL) and treated with 4 M HCl in dioxane (0.500 mL, 2.000 mmol), and the reaction was stirred at rt for 4 h. The reaction mixture was concentrated, and the residue was azeotroped with Et<sub>2</sub>O and MeCN and dried <i>in vacuo</i> to give 4-acetamido-<i>N</i>-(3-aminopropyl)-3-((3-chlorobenzyl)oxy)benzamide hydrochloride (61 mg, 0.148 mmol, 88% yield) as an off-white solid.</div><div class="NLM_p">LCMS (formic modifier) retention time of 1.59 min, [M + H]<sup>+</sup> = 376.2.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.34 (s, 1H), 8.66 (t, <i>J</i> = 5.7 Hz, 1H), 8.03 (d, <i>J =</i> 8.3 Hz, 1H), 7.86 (br s, 2H), 7.65 (s, 1H), 7.61 (d, <i>J</i> = 1.8 Hz, 1H), 7.37â7.53 (m, 4H), 5.31 (s, 2H), 3.31â3.37 (m, 2H), 2.85 (q, <i>J =</i> 6.3 Hz, 2H), 2.15 (s, 3H), 1.82 (quin, <i>J =</i> 6.9 Hz, 2H).</div></div><div id="sec4_2_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 4-Acetamido-<i>N</i>-(3-aminopropyl)-3-((2-chlorobenzyl)oxy)benzamide (<b>21</b>)</h4><div class="NLM_p">To a solution of <i>tert</i>-butyl (3-(4-acetamido-3-hydroxybenzamido)propyl)carbamate (200 mg, 0.569 mmol) and K<sub>2</sub>CO<sub>3</sub> (118 mg, 0.854 mmol) in DMF (3 mL) was added 1-(bromomethyl)-2-chlorobenzene (129 mg, 0.626 mmol) dropwise over 1 min, and the reaction mixture was stirred at rt for 10 min then heated at 60 Â°C for 2 h. The reaction mixture was diluted with water and extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated <i>in vacuo</i> to give the crude, which was purified by HPLC. Column: X Bridge C18 (150 mm Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 30â100%. The fractions containing the product were concentrated and then diluted with water and extracted with DCM, and the organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to give the boc-protected target compound. This was taken up in 1,4-dioxane (3 mL) and treated with 4 M HCl in dioxane (0.500 mL, 2.000 mmol), and the reaction was stirred at rt for 4 h. The reaction mixture was concentrated, and the residue was azeotroped with Et<sub>2</sub>O and MeCN and dried <i>in vacuo</i> to give 4-acetamido-<i>N</i>-(3-aminopropyl)-3-((2-chlorobenzyl)oxy)benzamide hydrochloride (43 mg, 0.104 mmol, 41% yield) as an off-white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 1.55 min, [M + H]<sup>+</sup> = 376.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.25 (s, 1H), 8.68 (t, <i>J</i> = 5.6 Hz, 1H), 8.06 (d, <i>J</i> = 8.3 Hz, 1H), 7.89 (br s, 2H), 7.69 (dd, <i>J</i> = 5.4, 4.1 Hz, 1H), 7.63 (d, <i>J</i> = 1.5 Hz, 1H), 7.49â7.55 (m, 2H), 7.39â7.45 (m, 2H), 5.34 (s, 2H), 3.30â3.39 (m, 2H), 2.85 (t, <i>J</i> = 7.3 Hz, 2H), 2.13 (s, 3H), 1.78â1.88 (m, 2H).</div></div><div id="sec4_2_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 4-Acetamido-<i>N</i>-(3-aminopropyl)-3-((4-methylbenzyl)oxy)benzamide (<b>22</b>)</h4><div class="NLM_p">To a solution of <i>tert</i>-butyl (3-(4-acetamido-3-hydroxybenzamido)propyl)carbamate (300 mg, 0.854 mmol) in DMF (1.5 mL) stirred under N<sub>2</sub> at 20 Â°C was added K<sub>2</sub>CO<sub>3</sub> (354 mg, 2.56 mmol) followed by the addition of 1-(bromomethyl)-4-methylbenzene (237 mg, 1.281 mmol) dropwise over 1 min. The reaction mixture was stirred at 70 Â°C for 3 h. The reaction was poured onto ice and extracted with EtOAc, and the organic layer was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure to afford a crude residue. This residue was triturated with <i>n</i>-pentane and dried to give the boc-protected title compound, which was taken up in 1,4-dioxane (1 mL) stirred and treated with HCl (0.012 mL, 0.410 mmol). The reaction mixture was stirred for 25 Â°C at 3 h. The reaction was concentrated and azeotroped with acetone and diethyl ether and dried under reduced pressure to get the crude product. This crude was purified by HPLC. Column: X Bridge C18 (250 mm Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 35â90% to give 4-acetamido-<i>N</i>-(3-aminopropyl)-3-((4-methylbenzyl)oxy)benzamide (40 mg, 0.099 mmol, 26.6% yield) as a white solid.</div><div class="NLM_p last">LCMS (HpH method) retention time of 3.17 min, [M + H]<sup>+</sup> = 356.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub> + D<sub>2</sub>O) Î´ ppm 8.71 (br d, <i>J</i> = 5.26 Hz, 1H), 7.95â8.18 (m, 2H), 7.79 (br d, <i>J</i> = 8.11 Hz, 1H), 7.54â7.66 (m, 2H), 7.49 (dd, <i>J</i> = 8.44, 1.86 Hz, 1H), 5.43 (s, 2H), 3.59â3.59 (m, 3H), 3.32 (brt, <i>J</i> = 6.69 Hz, 2H), 2.83 (brt, <i>J =</i> 7.45 Hz, 2H), 2.16 (s, 3H), 1.81 (dt, <i>J =</i> 14.52, 7.10 Hz, 2H).</div></div><div id="sec4_2_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 4-Acetamido-<i>N</i>-(3-aminopropyl)-3-((4-isopropylbenzyl)oxy)benzamide (<b>23</b>)</h4><div class="NLM_p">To a solution of <i>tert</i>-butyl (3-(3-hydroxy-4-(2-oxopropyl)benzamido)propyl)carbamate (300 mg, 0.856 mmol) in DMF (2 mL) stirred under N<sub>2</sub> at 20 Â°C were added K<sub>2</sub>CO<sub>3</sub> (355 mg, 2.57 mmol) and 1-(bromomethyl)-4-isopropylbenzene (0.215 mL, 1.284 mmol). The reaction was stirred at 70 Â°C for 3 h. The reaction was poured onto ice and extracted with EtOAc, and the organic layer was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure to afford the crude, which was purified by HPLC. Column: X Bridge C18 (150 mm Ã 19 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 0â100% to give the boc-protected title compound. This was taken up in 1,4-dioxane (1 mL) and stirred under N<sub>2</sub> at 20 Â°C; HCl (7.67 Î¼L, 0.252 mmol) was added, and the reaction was stirred at 25 Â°C for 3 h. The reaction was concentrated and azeotroped with Et<sub>2</sub>O and then triturated with Et<sub>2</sub>O to give 4-acetamido-<i>N</i>-(3-aminopropyl)-3-((4-isopropylbenzyl)oxy)benzamide hydrochloride (54 mg, 0.121 mmol, 47.8% yield) as an ash solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 0.74 min, [M + H]<sup>+</sup> = 384.4. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.22 (s, 1H), 8.65 (t, <i>J =</i> 5.7 Hz, 1H), 8.03 (d, <i>J</i> = 7.9 Hz, 1H), 7.85 (br s, 2H), 7.62 (d, <i>J</i> = 1.8 Hz, 1H), 7.42â7.47 (m, 3H), 7.27 (d, <i>J</i> = 7.9 Hz, 2H), 5.23 (s, 2H), 2.86â2.94 (m, 1H), 2.79â2.86 (m, 2H), 2.12 (s, 3H), 1.75â1.85 (m, 2H), 1.20 (d, <i>J</i> = 6.8 Hz, 6H).</div></div><div id="sec4_2_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 4-Acetamido-<i>N</i>-(3-aminopropyl)-3-((4-methoxybenzyl)oxy)benzamide (<b>24</b>)</h4><div class="NLM_p">To a solution of 4-acetamido-3-hydroxy-<i>N</i>-(3-(2,2,2-trifluoroacetamido)propyl)benzamide (200 mg, 0.576 mmol) in DMF (1.5 mL) stirred under N<sub>2</sub> at 20 Â°C was added K<sub>2</sub>CO<sub>3</sub> (239 mg, 1.728 mmol) and 1-(bromomethyl)-4-methoxybenzene (0.126 mL, 0.864 mmol). The reaction was stirred at 70 Â°C for 3 h. The reaction was poured onto ice and extracted with EtOAc, and the organic layer was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure to afford a crude residue, which was triturated with <i>n</i>-pentane and dried <i>in vacuo</i> to give the protected title compound. This was taken up in MeOH and stirred under N<sub>2</sub> at 20 Â°C; K<sub>2</sub>CO<sub>3</sub> (86 mg, 0.625 mmol) was added, and the reaction was stirred at 25 Â°C for 12 h. The reaction was diluted with DCM and filtered through celite washing with MeOH, and the filtrate was concentrated under reduced pressure to the give crude product, which was purified by HPLC. Column: X Bridge C18 (250 mm Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 10â100% to give 4-acetamido-<i>N</i>-(3-aminopropyl)-3-((4-methoxybenzyl)oxy)benzamide (47 mg, 0.124 mmol, 39.7% yield) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 1.44 min, [M + H]<sup>+</sup> = 372.3. <sup>1</sup>H NMR (chloroform-<i>d</i>, 400 MHz) Î´ 8.42 (br d, 1H, <i>J</i> = 8.3 Hz), 7.82 (br s, 1H), 7.7â7.8 (m, 1H), 7.62 (d, 1H, <i>J</i> = 1.8 Hz), 7.35 (d, 1H, <i>J</i> = 8.6 Hz), 6.6â7.0 (m, 2H), 5.10 (s, 2H), 3.83 (s, 3H), 3.5â3.7 (m, 2H), 2.9â3.0 (m, 2H), 2.15 (s, 3H), 1.74 (td, 2H, <i>J</i> = 6.2, 12.3 Hz).</div></div><div id="sec4_2_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 4-Acetamido-<i>N</i>-(3-aminopropyl)-3-((4-carbamoylbenzyl)oxy)benzamide (<b>25</b>)</h4><div class="NLM_p">To a solution of <i>tert</i>-butyl (3-(4-acetamido-3-hydroxybenzamido)propyl)carbamate (200 mg, 0.569 mmol) in DMF (1.5 mL) stirred under N<sub>2</sub> at 20 Â°C was added K<sub>2</sub>CO<sub>3</sub> (236 mg, 1.707 mmol) and 4-(bromomethyl)benzamide (146 mg, 0.683 mmol). The reaction was stirred at 70 Â°C for 3 h. The reaction was poured onto ice and extracted with EtOAc, and the organic layer was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure to afford the crude, which was purified by HPLC: column: X Bridge C18 (250 mm Ã 30 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and isocratic gradient: 35% to give the protected title compound. This was taken up in 1,4-dioxane (2 mL), and a solution of HCl (0.031 mL, 0.124 mmol) in 1,4-dioxane was added. The reaction was stirred at 25 Â°C for 3 h. The reaction was concentrated under reduced pressure and azeotroped with Et<sub>2</sub>O and triturated with Et<sub>2</sub>O to give 4-acetamido-<i>N</i>-(3-aminopropyl)-3-((4-carbamoylbenzyl)oxy)benzamide hydrochloride (50 mg, 0.118 mmol, 95% yield) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 0.39 min, [M + H]<sup>+</sup> = 385.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.30 (s, 1H), 8.62 (t, <i>J</i> = 5.7 Hz, 1H), 8.03 (d, <i>J</i> = 8.1 Hz, 1H), 7.97 (br s, 1H), 7.89 (d, <i>J</i> = 8.3 Hz, 2H), 7.82 (br s, 2H), 7.56â7.62 (m, 3H), 7.45 (dd, <i>J</i> = 8.3, 1.8 Hz, 1H), 7.36 (br s, 1H), 5.34 (s, 2H), 3.31 (q, <i>J</i> = 6.4 Hz, 2H), 2.82 (sxt, <i>J =</i> 6.4 Hz, 2H), 2.13 (s, 3H), 1.79 (quin, <i>J</i> = 7.0 Hz, 2H).</div></div><div id="sec4_2_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (+/â) Acetamido-<i>N</i>-(3-aminopropyl)-3-(1-phenylethoxy)benzamide (<b>26</b>)</h4><div class="NLM_p">To a solution of 4-acetamido-3-hydroxy-<i>N</i>-(3-(2,2,2-trifluoroacetamido)propyl)benzamide (300 mg, 0.864 mmol) and K<sub>2</sub>CO<sub>3</sub> (239 mg, 1.728 mmol) in DMF (3 mL) was added (1-bromoethyl)benzene (176 mg, 0.950 mmol), and the reaction was stirred at 60 Â°C for 3 h. The reaction was quenched with water and extracted with EtOAc; the organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to give the crude protected title compound. This crude was taken up in MeOH and treated with K<sub>2</sub>CO<sub>3</sub> (327 mg, 2.366 mmol), and the reaction was stirred at rt for 16 h. The reaction mixture was diluted with DCM and filtered through celite and washed with 15% MeOH in DCM. The filtrate was evaporated <i>in vacuo</i> to get the crude compound, which was purified using HPLC. Column: X Bridge C18 (150 mm Ã 19 mm Ã 5 Î¼m), mobile phase A: 10 mM ammonium bicarbonate (aq), mobile phase B: acetonitrile, flow: 30 mL/min, and gradient: 10â35% to give 4-acetamido-<i>N</i>-(3-aminopropyl)-3-(1-phenylethoxy)benzamide (45 mg, 0.126 mmol, 15.98% yield) as a white solid.</div><div class="NLM_p last">LCMS (formic modifier) retention time of 0.60 min, [M + H]<sup>+</sup> = 356.4. <sup>1</sup>H NMR (chloroform-<i>d</i>, 400 MHz) Î´ 8.3â8.5 (m, 1H), 7.9â8.0 (m, 1H), 7.6â7.7 (m, 1H), 7.42 (d, 1H, <i>J</i> = 1.8 Hz), 7.36 (d, 2H, <i>J</i> = 4.2 Hz), 7.30 (br d, 1H, <i>J</i> = 4.6 Hz), 7.2â7.2 (m, 1H), 5.4â5.5 (m, 1H), 3.5â3.6 (m, 2H), 2.8â2.9 (m, 2H), 2.2â2.2 (m, 3H), 1.6â1.8 (m, 5H).</div></div><div id="sec4_2_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (<i>S</i>)-Methyl 4-Nitro-3-(1-phenylethoxy)benzoate (<b>39</b>)</h4><div class="NLM_p">(<i>S</i>)-1-Phenylethanol (0.9 g, 7.37 mmol) was dissolved in DMF (10 mL) and THF (5 mL) and cooled in an ice bath under N<sub>2</sub> then NaH (0.5 g, 12.5 mmol, 60% dispersion in oil) was added, and the mixture was stirred for 20 min. 3-Fluoro-4-nitrobenzoic acid (1 g, 5.4 mmol) was added, and the mixture was stirred for 4 h at 0 Â°C then allowed to warm to rt. MeI (0.34 mL) was added, and the mixture was stirred for a further 18 h. The mixture was diluted with water and extracted with EtOAc; the organic layer washed with brine, dried, and evaporated <i>in vacuo</i> to give a yellow oil. This oil was purified using silica gel column chromatography eluting with a gradient of 0â30% EtOAc/cyclohexane to give (<i>S</i>)-methyl 4-nitro-3-(1-phenylethoxy)benzoate as a pale yellow oil (1.20 g, 3.98 mmol, 74% yield)</div><div class="NLM_p last">LCMS (formic) retention time of 1.29 min, [M + H]<sup>+</sup> = poor ionization. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) Î´ ppm 7.77 (d, <i>J</i> = 8.3 Hz, 1H), 7.66 (d, <i>J</i> = 1.5 Hz, 1H), 7.62 (dd, <i>J</i> = 8.3, 1.5 Hz, 1H), 7.42â7.47 (m, 2H), 7.35â7.41 (m, 2H), 7.29â7.34 (m, 1H), 5.57 (q, <i>J</i> = 6.4 Hz, 1H), 3.91 (s, 3H), 1.72 (d, <i>J</i> = 6.4 Hz, 3H).</div></div><div id="sec4_2_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> (<i>S</i>)-Methyl 4-Amino-3-(1-phenylethoxy)benzoate (<b>40</b>)</h4><div class="NLM_p">(<i>S</i>)-Methyl 4-nitro-3-(1-phenylethoxy)benzoate (1.2 g, 3.98 mmol) was dissolved in EtOH (100 mL) and hydrogenated in the H cube at atmospheric pressure and rt over a Pt/C cartridge. The eluant was evaporated <i>in vacuo</i> to give (<i>S</i>)-methyl 4-amino-3-(1-phenylethoxy)benzoate (1.05 g, 3.87 mmol, 97% yield) as a colorless oil.</div><div class="NLM_p last">LCMS (Formic) retention time of 1.15 min, [M + H]<sup>+</sup> = 272.0. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) Î´ ppm 7.50 (dd, <i>J</i> = 8.2, 1.8 Hz, 1H), 7.40â7.44 (m, 3H), 7.34â7.39 (m, 2H), 7.28â7.32 (m, 1H), 6.67 (d, <i>J</i> = 8.1 Hz, 1H), 5.44 (q, <i>J =</i> 6.4 Hz, 1H), 3.82 (s, 3H), 1.69 (d, <i>J =</i> 6.6 Hz, 3H).</div></div><div id="sec4_2_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (<i>S</i>)-Methyl 4-Acetamido-3-(1-phenylethoxy)benzoate (<b>41</b>)</h4><div class="NLM_p">(<i>S</i>)-Methyl 4-amino-3-(1-phenylethoxy)benzoate (1.2 g, 4.42 mmol) was suspended in H<sub>2</sub>O (50 mL), and acetic anhydride (452 mg, 4.42 mmol) was added dropwise. The resulting suspension was stirred at rt for 2 h then extracted with EtOAc, and the organic layer was dried and evaporated to give (<i>S</i>)-methyl 4-acetamido-3-(1-phenylethoxy)benzoate (1.12 g, 3.57 mmol, 81% yield) as a colorless solid.</div><div class="NLM_p last">LCMS (formic) retention time of 1.14 min, [M + H]<sup>+</sup> = 314.0. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) Î´ ppm 8.45 (d, <i>J</i> = 8.6 Hz, 1H), 7.98 (br s, 1H), 7.64 (dd, <i>J</i> = 8.6, 1.7 Hz, 1H), 7.51 (d, <i>J</i> = 1.7 Hz, 1H), 7.36â7.43 (m, 4H), 7.31â7.35 (m, 1H), 5.46 (q, <i>J</i> = 6.5 Hz, 1H), 3.86 (s, 3H), 2.24 (s, 3H), 1.75 (d, <i>J</i> = 6.6 Hz, 3H).</div></div><div id="sec4_2_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (<i>S</i>)-4-Acetamido-3-(1-phenylethoxy)benzoic Acid (<b>42</b>)</h4><div class="NLM_p">LiOH (84 mg, 3.51 mmol) was added to a solution of (<i>S</i>)-methyl 4-acetamido-3-(1-phenylethoxy)benzoate (1.1 g, 3.51 mmol) in THF (10 mL) and H<sub>2</sub>O (10 mL), and the mixture was then heated at 50 Â°C for 3 h then allowed to stand overnight at rt. The mixture was evaporated to half its original volume then diluted with water and acidified with 2 M HCl (15 mL). The resulting suspension was extracted with EtOAc, and the combined organics were dried and evaporated <i>in vacuo</i> to give (<i>S</i>)-4-acetamido-3-(1-phenylethoxy)benzoic acid (850 mg, 2.84 mmol, 81% yield) as a pale yellow solid.</div><div class="NLM_p last">LCMS (Formic) retention time 0.98 min, [M + H]<sup>+</sup> = 300.0. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) Î´ ppm 8.49 (d, <i>J</i> = 8.3 Hz, 1H), 8.03 (s, 1H), 7.72 (dd, <i>J</i> = 8.6, 1.7 Hz, 1H), 7.54 (d, <i>J =</i> 1.7 Hz, 1H), 7.37â7.44 (m, 4H), 7.30â7.36 (m, 1H), 5.47 (q, <i>J</i> = 6.6 Hz, 1H), 2.26 (s, 3H), 1.76 (d, <i>J =</i> 6.4 Hz, 3H).</div></div><div id="sec4_2_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> (<i>S</i>)-(9<i>H</i>-Fluoren-9-yl)methyl(3-(4-acetamido-3-(1-phenylethoxy)benzamido)propyl)carbamate (<b>27</b>, Step 1)</h4><div class="NLM_p">(<i>S</i>)-4-Acetamido-3-(1-phenylethoxy)benzoic acid (100 mg, 0.334 mmol) was taken up in DCM (2 mL). DIPEA (0.233 mL, 1.336 mmol) then T<sub>3</sub>P (0.199 mL, 0.334 mmol) was added, and the reaction was stirred for 10 min at rt. (9<i>H</i>-Fluoren-9-yl)methyl (3-aminopropyl)carbamate hydrobromide (139 mg, 0.367 mmol) was added, and stirring at rt continued for 2 h and 15 min. The reaction was diluted with DCM and washed with sat. NaHCO3 (aq). The aqueous part was re-extracted with DCM, and the combined organics were eluted through a hydrophobic frit then concentrated <i>in vacuo</i> to give a cream solid. The crude product was purified using silica gel column chromatography eluting with a gradient of 0â40% (3:1 EtOAc/EtOH) in cyclohexane/EtOAc to give (<i>S</i>)-(9<i>H</i>-fluoren-9-yl)methyl (3-(4-acetamido-3-(1-phenylethoxy)benzamido)propyl)carbamate (153 mg, 0.252 mmol, 75% yield) as a cream solid.</div><div class="NLM_p last">LCMS (HpH) retention time of 1.29 min, [M + H]<sup>+</sup> = 578.4. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) Î´ ppm 8.45 (d, <i>J</i> = 8.3 Hz, 1H), 7.95 (br s, 1H), 7.78 (d, <i>J</i> = 7.3 Hz, 2H), 7.62 (d, <i>J</i> = 7.6 Hz, 2H), 7.29â7.49 (m, 12H), 5.50 (q, <i>J</i> = 6.1 Hz, 1H), 5.22 (t, <i>J</i> = 6.0 Hz, 1H), 4.48 (d, <i>J</i> = 6.8 Hz, 2H), 4.22 (t, <i>J =</i> 6.6 Hz, 1H), 4.14 (q, <i>J =</i> 7.2 Hz, 1H), 3.40 (dt, <i>J</i> = 13.1, 6.5 Hz, 2H), 3.21â3.31 (m, 2H), 2.23 (s, 3H), 1.73 (d, <i>J</i> = 6.4 Hz, 3H), 1.69 (d, <i>J =</i> 3.4 Hz, 2H).</div></div><div id="sec4_2_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> (<i>S</i>)-4-Acetamido-<i>N</i>-(3-aminopropyl)-3-(1-phenylethoxy)benzamide</h4><div class="NLM_p">(<i>S</i>)-(9<i>H</i>-fluoren-9-yl)methyl(3-(4-acetamido-3-(1-phenylethoxy)benzamido)propyl)carbamate (147.9 mg, 0.256 mmol) was taken up in DMF (5 mL) to give a yellow solution, piperidine (0.051 mL, 0.512 mmol) was added, and the reaction was left to stir at rt for 2 h. The reaction was concentrated <i>in vacuo</i> to give an orange solid, which was purified using silica gel column chromatography eluting with a gradient of 2â20% 2 M NH<sub>3</sub> in MeOH/DCM to give (<i>S</i>)-4-acetamido-<i>N</i>-(3-aminopropyl)-3-(1-phenylethoxy)benzamide (56 mg, 0.150 mmol, 58.5% yield) as a yellow oil.</div><div class="NLM_p last">LCMS (HpH) retention time of 0.81 min, [M + H]<sup>+</sup> = 356.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.12 (s, 1H), 8.34 (t, <i>J =</i> 5.5 Hz, 1H), 8.01 (d, <i>J =</i> 7.8 Hz, 1H), 7.49 (d, <i>J</i> = 7.3 Hz, 1H), 7.39 (d, <i>J</i> = 1.7 Hz, 1H), 7.31â7.37 (m, 2H), 7.22â7.29 (m, 1H), 5.59 (q, <i>J</i> = 6.4 Hz, 1 H), 3.22â3.27 (m, 2H), 2.53â2.57 (m, 2H), 2.17 (s, 3H), 1.64 (d, <i>J</i> = 6.4 Hz, 3H), 1.54 (quin, <i>J =</i> 6.7 Hz, 2H).</div></div><div id="sec4_2_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 4-Acetamido-<i>N</i>-((1<i>R</i>,4<i>S</i>)-4-hydroxycyclohexyl)-3-((<i>S</i>)-1-phenylethoxy)benzamide (<b>29</b>)</h4><div class="NLM_p">To a solution of (<i>S</i>)-4-acetamido-3-(1-phenylethoxy)benzoic acid (50 mg, 0.167 mmol) and HATU (63.5 mg, 0.167 mmol), and DMF (0.8 mL) were added DIPEA (0.088 mL, 0.501 mmol) and <i>trans</i>-aminocyclohexanol (19 mg, 0.167 mmol), and the reaction was stirred at rt for 18 h. The reaction was MDAP (ammonium carbonate modifier) to give 4-acetamido-<i>N</i>-((1<i>R</i>,4<i>S</i>)-4-hydroxycyclohexyl)-3-((<i>S</i>)-1-phenylethoxy)benzamide (40.3 mg, 0.091 mmol, 54.8% yield).</div><div class="NLM_p last">LCMS (formic) retention time of 0.88 min, [M + H]<sup>+</sup> = 397.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.12 (s, 1H), 8.01 (br d, 1H, <i>J =</i> 8.3 Hz), 7.95 (d, 1H, <i>J</i> = 7.8 Hz), 7.43â7.50 (m, 2H), 7.1â7.42 (m, 4H), 7.24â7.40 (m, 2H), 5.59 (q, <i>J</i> = 6.4 Hz, 1H), 4.54 (d, <i>J</i> = 4.4 Hz, 1H), 3.65 (dtd, <i>J</i> = 11.3, 7.4, 7.4, 3.9 Hz, 1H), 3.35â3.44 (m, 1H), 2.17 (s, 3H), 1.80â1.88 (m, 2H), 1.72â1.79 (m, 2H), 1.63 (d, <i>J</i> = 6.4 Hz, 3H), 1.28â1.41 (m, 2H), 1.14â1.27 (m, 2H).</div></div><div id="sec4_2_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 3-Fluoro-4-nitro-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (<b>43</b>)</h4><div class="NLM_p">To a solution of 3-fluoro-4-nitrobenzoic acid (2.65 g, 14.32 mmol) in DMF(50 mL) was added CDI (3.48 g, 21.47 mmol), and the reaction was stirred for 1 h and 15 min before tetrahydro-2<i>H</i>-pyran-4-amine hydrochloride (2.95 g, 21.47 mmol) was added, and the reaction was stirred for 16 h at rt. The reaction was partitioned between 10% citric acid (aq) and EtOAc. The aqueous layer was further extracted with EtOAc, and the organic layers were combined and washed with 10% LiCl (aq). The organics were then dried over a hydrophobic frit and concentrated to give 3-fluoro-4-nitro-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (3.539 g, 13.19 mmol, 92% yield) as a pale yellow solid.</div><div class="NLM_p last">LCMS (formic) retention time of 0.79 min, [M + H]<sup>+</sup> = 269. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) Î´ 8.67 (br d, 1H, <i>J</i> = 7.3 Hz), 8.26 (dd, 1H, <i>J</i> = 7.6, 8.6 Hz), 7.98 (dd, 1H, <i>J</i> = 2.0, 12.2 Hz), 7.88 (td, 1H, <i>J</i> = 1.0, 8.2 Hz), 3.94â4.07 (m, 1H), 3.82â3.92 (m, 2H), 3.36â3.47 (m, 2H), 1.74â1.84 (m, 2H), 1.50â1.67 (m, 2H).</div></div><div id="sec4_2_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> (<i>R</i>)-4-Nitro-3-(1-phenylethoxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (<b>44</b>)</h4><div class="NLM_p">(<i>R</i>)-1-Phenylethanol (0.090 mL, 0.743 mmol), 3-fluoro-4-nitro-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (100 mg, 0.371 mmol), THF (2 mL), and 1 M LiHMDS in THF (0.743 mL, 0.743 mmol) were placed in a microwaveable vial and irradiated at 100 Â°C for 30 min. The reaction was diluted with water and extracted with EtOAc, and the combined organics were washed with brine, dried using a hydrophobic frit, and concentrated to an orange oil. This oil was purified by silica gel column chromatography eluting with a gradient of 0â60% EtOAc/cyclohexane to give 3-fluoro-4-nitro-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (112 mg, 0.302 mmol, 81% yield) as a yellow oil.</div><div class="NLM_p last">LCMS (formic) retention time of 1.13 min, [M + H]<sup>+</sup> = 371.</div></div><div id="sec4_2_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> (<i>R</i>)-4-Amino-3-(1-phenylethoxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (<b>45</b>)</h4><div class="NLM_p">(<i>R</i>)-4-Nitro-3-(1-phenylethoxy)-<i>N</i>-(tetrahydro-2H-pyran-4-yl)benzamide (112 mg, 0.302 mmol), tin(II) chloride (172 mg, 0.907 mmol), and EtOH (5 mL) were stirred at 60 Â°C for 16 h. The reactions were diluted with 1 M NaOH (aq) and extracted with EtOAc; the organic layers were washed with 1% DBU (aq) and brine, dried using a hydrophobic frit, and concentrated to an oil. This oil was purified using silica gel column chromatography eluting with a gradient of 0â5% 2 M NH<sub>3</sub> in MeOH/DCM to give (<i>R</i>)-4-amino-3-(1-phenylethoxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (40 mg, 0.118 mmol, 38.9% yield) as a pale yellow oil.</div><div class="NLM_p last">LCMS (formic) retention time of 0.91 min, [M + H]<sup>+</sup> = 341.</div></div><div id="sec4_2_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> (<i>R</i>)-4-Acetamido-3-(1-phenylethoxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (<b>27</b>)</h4><div class="NLM_p">(<i>R</i>)-4-Amino-3-(1-phenylethoxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (40 mg, 0.118 mmol) was placed in a round-bottom flask and treated with water (2 mL) and acetic anhydride (400 Î¼L, 4.24 mmol) and stirred at rt for 1 h. The reaction was diluted with NaHCO<sub>3</sub> (aq) and extracted with DCM, and the organic layers were dried using a hydrophobic frit and concentrated to give an off-white solid, which was purified using silica gel column chromatography eluting with a gradient of 0â4% 2 M NH<sub>3</sub> in MeOH/DCM to give (<i>R</i>)-4-acetamido-3-(1-phenylethoxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (34 mg, 0.089 mmol, 76% yield).</div><div class="NLM_p last">LCMS (formic) retention time of 0.95 min, [M + H]<sup>+</sup> = 383.1. <sup>1</sup>H NMR (methanol-<i>d</i><sub>4</sub>, 400 MHz) Î´ 8.07 (br d, 1H, <i>J</i> = 8.3 Hz), 7.43â7.48 (m, 2H), 7.40 (br s, 1H), 7.32â7.38 (m, 3H), 7.22â7.30 (m, 1H), 5.51â5.58 (m, 1H), 4.01â4.10 (m, 1H), 3.95â4.00 (m, 2H), 3.46â3.56 (m, 2H), 2.21â2.26 (m, 3H), 1.85â1.89 (m, 2H), 1.71â1.75 (m, 3H), 1.58â1.70 (m, 2H).</div></div><div id="sec4_2_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> (<i>S</i>)-4-Nitro-3-(1-phenylethoxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (<b>44b</b>)</h4><div class="NLM_p">To a solution of (<i>S</i>)-1-phenylethanol (0.630 mL, 5.22 mmol) in DMF (7 mL) was added NaH 60% dispersion in mineral oil (209 mg, 5.22 mmol), and the reaction was stirred for 15 min before 3-fluoro-4-nitro-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (700 mg, 2.61 mmol) was added, and the reaction was stirred for 2 h. The reaction was diluted with water and extracted with EtOAc. The organics were washed with 10% LiCl (aq) and dried over a hydrophobic frit and concentrated under reduced pressure; the resulting crude was purified using silica gel column chromatography eluting with a gradient of 18â90% EtOAc-cyclohexane to give (<i>S</i>)-4-nitro-3-(1-phenylethoxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (700 mg, 1.890 mmol, 72.4% yield) as a yellow solid.</div><div class="NLM_p last">LCMS (formic) retention time of 1.04 min, [M + H]<sup>+</sup> = 371. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) Î´ 8.44â8.46 (m, 1H), 7.92 (d, 1H, <i>J =</i> 8.3 Hz), 7.66â7.68 (m, 1H), 7.47â7.54 (m, 1H), 7.42â7.46 (m, 2H), 7.36â7.40 (m, 2H), 7.25â7.32 (m, 1H), 5.79â5.89 (m, 1H), 3.92â4.00 (m, 1H), 3.84â3.91 (m, 2H), 3.29â3.42 (m, 4H), 1.99 (s, 3H), 1.74 (br dd, 4H, <i>J</i> = 2.4, 11.7 Hz), 1.48â1.56 (m, 5H).</div></div><div id="sec4_2_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> (<i>S</i>)-4-Amino-3-(1-phenylethoxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (<b>45b</b>)</h4><div class="NLM_p">A solution of (<i>S</i>)-4-nitro-3-(1-phenylethoxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (700 mg, 1.890 mmol) and tin(II) chloride (1075 mg, 5.67 mmol) in EtOH (10 mL) was stirred at 60 Â°C for 16 h. The reaction was diluted with 1 M NaOH (aq) and extracted with EtOAc. The organics were washed with dilute DBU solution (aq) and brine and then dried over a hydrophobic frit and concentrated. The residue was purified using silica gel column chromatography eluting with a gradient of 20â100% EtOAc/cyclohexane to give (<i>S</i>)-4-amino-3-(1-phenylethoxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (240 mg, 0.705 mmol, 37.3% yield) as a pale yellow solid.</div><div class="NLM_p last">LCMS (formic) retention time of 0.85 min, [M + H]<sup>+</sup> = 341. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) Î´ 7.74 (d, 1H, <i>J</i> = 7.34 Hz), 7.42â7.49 (m, 2H), 7.31â7.37 (m, 2H), 7.20â7.28 (m, 3H), 6.56â6.62 (m, 1H), 5.46â5.54 (m, 1H), 3.99â4.07 (m, 3H), 3.29â3.39 (m, 5H), 1.65â1.73 (m, 2H), 1.55â1.59 (m, 5H).</div></div><div id="sec4_2_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> (<i>S</i>)-4-Acetamido-3-(1-phenylethoxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (<b>28</b>)</h4><div class="NLM_p">A solution of (<i>S</i>)-4-amino-3-(1-phenylethoxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (240 mg, 0.705 mmol) and acetic anhydride (1 mL, 10.60 mmol) in water (3 mL) was stirred at 50 Â°C for 16 h. The reaction was cooled to rt; sat. NaHCO<sub>3</sub> (aq) was added, and the reaction was extracted with EtOAc. The organics were dried over a hydrophobic frit and concentrated. The residue was purified using silica gel column chromatography eluting with a gradient of 10â65% 3:1 EtOAc/EtOH/cyclohexane to give (<i>S</i>)-4-acetamido-3-(1-phenylethoxy)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)benzamide (137.5 mg, 0.360 mmol, 51.0% yield) as an off-white solid.</div><div class="NLM_p last">LCMS (formic) retention time of 0.91 min, [M + H]<sup>+</sup> = 383.1. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) Î´ 9.12 (s, 1H), 8.06â8.12 (m, 1H), 8.00â8.04 (m, 1H), 7.46â7.52 (m, 2H), 7.31â7.41 (m, 1H), 7.32â7.37 (m, 3H), 7.23â7.28 (m, 1H), 5.56â5.64 (m, 1H), 3.90â3.99 (m, 1H), 3.84â3.89 (m, 2H), 3.30â3.40 (m, 2H), 2.16â2.20 (m, 3H), 1.68â1.76 (m, 2H), 1.62â1.66 (m, 3H), 1.46â1.61 (m, 2H).</div></div><div id="sec4_2_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 3-Fluoro-<i>N</i>-((1<i>R</i>,4<i>S</i>)-4-hydroxycyclohexyl)-4-nitrobenzamide</h4><div class="NLM_p">To a solution of 3-fluoro-4-nitrobenzoic acid (75 mg, 0.405 mmol) and HATU (185 mg, 0.486 mmol) in DMF (2 mL) stirred at rt were added <i>trans</i>-4-aminocyclohexanol (70 mg, 0.608 mmol) and DIPEA (0.071 mL, 0.405 mmol). The reaction mixture was stirred at rt for 1.5 h. The reaction was purified directly by MDAP (formic modifier) to give 3-fluoro-<i>N</i>-((1<i>R</i>,4<i>S</i>)-4-hydroxycyclohexyl)-4-nitrobenzamide (92 mg, 0.326 mmol, 80% yield).</div><div class="NLM_p last">LCMS (formic) retention time of 0.73 min, [M + H]<sup>+</sup> = 283.0. <sup>1</sup>H NMR (400 MHz, Chloroform-<i>d</i>) Î´ ppm 8.13 (dd, <i>J</i> = 8.4, 7.2 Hz, 1H), 7.71 (dd, <i>J</i> = 11.0, 1.7 Hz, 1H), 7.61â7.66 (m, 1H), 5.88â5.93 (m, 1H), 3.94â4.04 (m, 1H), 3.69 (s, 1H), 2.16 (d, <i>J</i> = 11.2 Hz, 2H), 2.08 (d, <i>J</i> = 12.5 Hz, 2H), 1.45â1.53 (m, 2H), 1.30â1.41 (m, 2H).</div></div><div id="sec4_2_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>N</i>-((1<i>R</i>,4<i>S</i>)-4-Hydroxycyclohexyl)-4-nitro-3-((<i>S</i>)-1-phenylethoxy)benzamide</h4><div class="NLM_p">The (<i>S</i>)-1-phenylethanol (0.115 mL, 0.957 mmol) was dissolved in DMF (1 mL) and THF (0.500 mL) and cooled in an ice bath under N<sub>2</sub>; NaH (60% in mineral oil, 63.8 mg, 1.594 mmol) was added, and the mixture was stirred for 20 min. 3-Fluoro-<i>N</i>-((1<i>R</i>,4<i>S</i>)-4-hydroxycyclohexyl)-4-nitrobenzamide (90 mg, 0.319 mmol) was added as a solution in DMF (2 mL) dropwise, and the mixture was stirred for 4 h at 0 Â°C then allowed to warm to rt. The reaction was quenched with water and extracted with EtOAc, and the organic phase was washed with brine and evaporated to afford the crude product as a yellow solid. The crude was purified using silica gel column chromatography eluting with a gradient of 30â100% 3:1 ethyl acetate/EtOH/cyclohexane. The appropriate fractions were combined and evaporated <i>in vacuo</i> and then azeotroped with toluene and dried to give <i>N</i>-((1<i>R</i>,4<i>S</i>)-4-hydroxycyclohexyl)-4-nitro-3-((<i>S</i>)-1-phenylethoxy)benzamide (102 mg, 0.265 mmol, 83% yield) as a yellow solid.</div><div class="NLM_p last">LCMS (formic) retention time of 1.03 min, [M + H]<sup>+</sup> = 385.1. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) Î´ ppm 7.79 (d, <i>J</i> = 8.3 Hz, 1H), 7.42â7.47 (m, 3H), 7.39 (t, <i>J</i> = 7.5 Hz, 2H), 7.24â7.35 (m, 1H), 7.13â7.22 (m, 2H), 5.69 (d, <i>J</i> = 7.6 Hz, 1H), 5.55 (q, <i>J</i> = 6.5 Hz, 1H), 3.82â3.96 (m, 1H), 3.67 (br s, 1H), 1.97â2.15 (m, 4H), 1.73 (d, <i>J</i> = 6.3 Hz, 3H), 1.41â1.53 (m, 2H), 1.20â1.34 (m, 2H).</div></div><div id="sec4_2_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 4-Amino-<i>N</i>-((1<i>R</i>,4<i>S</i>)-4-hydroxycyclohexyl)-3-((<i>S</i>)-1-phenylethoxy)benzamide (<b>46</b>)</h4><div class="NLM_p">To a solution of <i>N</i>-((1<i>R</i>,4<i>S</i>)-4-hydroxycyclohexyl)-4-nitro-3-((<i>S</i>)-1-phenylethoxy)benzamide (71 mg, 0.185 mmol) in EtOH (3 mL) under N<sub>2</sub> was added tin(II) chloride (105 mg, 0.554 mmol), and the reaction mixture was heated to 50 Â°C for 16 h. The reaction was quenched with water and extracted with EtOAc. The organic phase was washed with brine and dried through a hydrophobic frit, and the solvent was evaporated <i>in vacuo.</i> The resulting residue was purified using silica gel column chromatography eluting with a gradient of 15â60% 3:1 EtOAc/EtOH/cyclohexane to give 4-amino-<i>N</i>-((1<i>R</i>,4<i>S</i>)-4-hydroxycyclohexyl)-3-((<i>S</i>)-1-phenylethoxy)benzamide (32.2 mg, 0.091 mmol, 49.2% yield).</div><div class="NLM_p last">LCMS (formic) retention time of 0.86 min, [M + H]<sup>+</sup> = 355.1. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) Î´ ppm 7.43 (d, <i>J</i> = 7.3 Hz, 2H), 7.33 (t, <i>J</i> = 7.6 Hz, 2H), 7.19â7.27 (m, 3H), 6.71 (d, <i>J</i> = 8.1 Hz, 1H), 5.48 (q, <i>J</i> = 6.3 Hz, 1H), 3.77 (br s, 1H), 3.49â3.61 (m, 1H), 1.89â2.02 (m, 4H), 1.65 (d, <i>J</i> = 6.4 Hz, 3H), 1.33â1.45 (m, 4H).</div></div><div id="sec4_2_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (<i>S</i>)-5-Bromo-1-fluoro-2-nitro-3-(1-phenylethoxy)benzene (<b>55</b>)</h4><div class="NLM_p last">This reaction was performed in five 10 g batches. To a solution of 5-bromo-1,3-difluoro-2-nitrobenzene (50.0 g, 210 mmol) and (<i>S</i>)-1-phenylethanol (28.2 g, 231 mmol) in THF (500 mL) stirred under N<sub>2</sub> (g) at rt was added a solution of LiHMDS (210 mL, 210 mmol) dropwise. The reaction mixture was stirred at 100 Â°C for 1 h in a sealed tube. The combined reaction mixtures were quenched with water and extracted with EtOAc. The organic phase was washed with sat. brine (aq) (500 mL), dried over sodium sulfate, and evaporated <i>in vacuo</i> to give the crude product as a black liquid. This crude was purified using silica gel column chromatography eluting with 2% EtOAc in pet ether to afford (<i>S</i>)-5-bromo-1-fluoro-2-nitro-3-(1-phenylethoxy)benzene (44 g, 129 mmol, 61.6% yield) as an orange liquid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 7.32â7.26 (m, 2H), 7.28â7.24 (m, 2H), 7.25â7.20 (m, 1H), 6.85 (dd, <i>J</i> = 8.5, 1.5 Hz, 1H), 6.78â6.76 (m, 1H), 5.31 (q, <i>J</i> = 6.5 Hz, 1H), 1.57 (d, <i>J</i> = 6.5 Hz, 3H).</div></div><div id="sec4_2_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> (<i>S</i>)-4-Bromo-2-fluoro-6-(1-phenylethoxy)aniline (<b>56</b>)</h4><div class="NLM_p">A solution of (<i>S</i>)-5-bromo-1-fluoro-2-nitro-3-(1-phenylethoxy)benzene (44.8 g, 132 mmol) and NH<sub>4</sub>Cl (35.2 g, 659 mmol) in EtOH (440 mL) and H<sub>2</sub>O (147 mL) at rt was treated with Fe(0) (22.07 g, 395 mmol), and the resulting mixture was refluxed for 1 h then was cooled to rt and partitioned between water (500 mL) and EtOAc (500 mL). The mixture was filtered through celite, and the layers were separated. The aqueous phase was extracted with EtOAc (2Ã 500 mL) and the combined organics were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i> to give (<i>S</i>)-4-bromo-2-fluoro-6-(1-phenylethoxy)aniline (34.5 g, 82%) as a colorless liquid. This was used in the next step without further purification.</div><div class="NLM_p last">LCMS (method high pH) retention time of 1.34 min, [M + H]<sup>+</sup> = 312.1 (1 Br). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 7.36â7.34 (m, 2H), 7.35â7.33 (m, 2H), 7.31â7.26 (m, 1H), 6.78 (dd, <i>J</i> = 9.8, 2.1 Hz, 1H), 6.59 (t, <i>J</i> = 1.8 Hz, 1H), 5.28 (q, <i>J</i> = 6.4 Hz, 1H), 1.65 (d, <i>J</i> = 6.5 Hz, 3H).</div></div><div id="sec4_2_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> (<i>S</i>)-<i>N</i>-(4-Bromo-2-fluoro-6-(1-phenylethoxy)phenyl)acetamide (<b>57</b>)</h4><div class="NLM_p">To neat (<i>S</i>)-4-bromo-2-fluoro-6-(1-phenylethoxy)aniline (34.0 g, 106 mmol) at rt was added neat acetic anhydride (523 mL, 5541 mmol) in one charge, and the resulting mixture was stirred at this temperature for 16 h then concentrated <i>in vacuo</i>. The residue obtained was triturated with <i>n</i>-pentane (2Ã 100 mL). The solid obtained was filtered through a Buchner funnel, rinsed with <i>n</i>-pentane, and dried under reduced pressure to give (<i>S</i>)-<i>N</i>-(4-bromo-2-fluoro-6-(1-phenylethoxy)phenyl)acetamide (30.8 g, 81%) as a white solid, which was used in the next step without further purification.</div><div class="NLM_p last">LCMS (method high pH) retention time of 1.14 min, [M + H]<sup>+</sup> = 352.0 (1 Br). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm, 9.28 (br s, 1H), 7.46â7.41 (m, 2H) 7.38â7.32 (m, 2H), 7.31â7.24 (m, 1H), 7.09 (dd, <i>J</i> = 9.0, 1.6 Hz, 1H), 6.99â6.91 (m, 1H), 5.59 (q, <i>J</i> = 6.2 Hz, 1H), 2.08 (br s, 3H), 1.52 (d, <i>J</i> = 6.2 Hz, 3H).</div></div><div id="sec4_2_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 4-Acetamido-3-(benzyloxy)-<i>N</i>-ethyl-5-fluorobenzamide (<b>52</b>)</h4><div class="NLM_p">To an oven-dried microwave vial, <i>N</i>-(2-(benzyloxy)-4-bromo-6-fluorophenyl)acetamide (100 mg, 0.296 mmol), diacetoxypalladium (26.6 mg, 0.118 mmol), Xantphos (68.4 mg, 0.118 mmol), cobalt carbonyl (50.6 mg, 0.148 mmol), and DMAP (50.6 mg, 0.414 mmol) were dissolved in 1,4-dioxane (10 mL). Ethanamine (2 M) in THF (0.503 mL, 1.005 mmol) was added, and the vial was sealed immediately. The reaction was irradiated at 90 Â°C for 20 min in a microwave reactor. The reaction mixture from a previous reaction (0.059 mmol scale reaction) was combined, and the solution was partitioned between water and EtOAc and extracted into EtOAc, including a brine wash. The organic fractions were concentrated <i>in vacuo</i>, and the crude product was purified by silica gel column chromatography eluting with a gradient of 0â60% EtOAc/cyclohexane followed by 0â100% 20% MeOH in DCM/DCM to give the crude title compound. This crude product was purified with MDAP (formic modifier) to give 4-acetamido-3-(benzyloxy)-<i>N</i>-ethyl-5-fluorobenzamide (64 mg, 0.194 mmol, 54.4% yield).</div><div class="NLM_p last">LCMS (formic) retention time of 0.82 min, [M + H]<sup>+</sup> = 331.0. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) Î´ ppm 8.57 (s, 1H), 7.49 (d, <i>J</i> = 7.3 Hz, 1H), 7.44 (s, 1H), 7.37â7.43 (m, 2H), 7.35 (d, <i>J</i> = 7.1 Hz, 1H), 7.29 (d, <i>J</i> = 10.0 Hz, 1H), 5.22 (s, 2H), 3.37â3.47 (m, 2H), 2.16 (s, 3H), 1.24 (t, <i>J</i> = 7.2 Hz, 3H).</div></div><div id="sec4_2_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> (<i>S</i>)-Methyl 4-Acetamido-3-fluoro-5-(1-phenylethoxy)benzoate (<b>58</b>)</h4><div class="NLM_p">A flask was charged with (<i>S</i>)-<i>N</i>-(4-bromo-2-fluoro-6-(1-phenylethoxy)phenyl)acetamide (5.0 g, 14.2 mmol), Pd(OAc)<sub>2</sub> (0.319 g, 1.42 mmol), and Xantphos (0.821 g, 1.420 mmol) and purged with N<sub>2</sub> then filled with DMF (40 mL) and MeOH (20 mL), and the resulting mixture was treated with NEt<sub>3</sub> (6.00 mL, 43.0 mmol) before being purged with carbon monoxide from a balloon (approximately 1 L volume). A fresh balloon was fitted, and the mixture was stirred at 70 Â°C for 4 h then cooled to rt and diluted with EtOAc (100 mL). The organic phase was washed with a 10% w/w LiCl aqueous solution (2Ã 100 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (100 g column, gradient: 10â100% EtOAc in cyclohexane) gave (<i>S</i>)-methyl-4-acetamido-3-fluoro-5-(1-phenylethoxy)benzoate (4.5 g, 96%) as a pale yellow foam.</div><div class="NLM_p last">LCMS (method: formic) retention time of 1.00 min, [M + H]<sup>+</sup> = 332.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.50 (s, 1H), 7.49â7.44 (m, 2H), 7.38â7.33 (m, 2H), 7.32â7.28 (m, 1H), 7.29â7.28 (m, 1H), 7.28â7.24 (m, 1H), 5.61 (q, <i>J</i> = 6.4 Hz, 1H), 3.79 (s, 3H), 2.11 (s, 3H), 1.56 (d, <i>J</i> = 6.4 Hz, 3H).</div></div><div id="sec4_2_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> (<i>S</i>)-4-Acetamido-3-fluoro-5-(1-phenylethoxy)benzoic Acid (<b>59</b>, Step 1)</h4><div class="NLM_p">A solution of methyl (<i>S</i>)-4-acetamido-3-fluoro-5-(1-phenylethoxy)benzoate (1.657 g, 5 mmol) in THF (7 mL) and ethanol (7 mL) at rt was treated with 2 M NaOH (aq) (7.50 mL, 15.00 mmol), and the resulting mixture was stirred at this temperature for 2 h then concentrated <i>in vacuo</i>. The residue was partitioned between H<sub>2</sub>O and Et<sub>2</sub>O and the layers were separated. The aqueous layer was treated with 2 N HCl (aq) (10 mL); a white precipitate formed, which was extracted with EtOAc. The combined organics were dried over MgSO<sub>4</sub> and concentrated <i>in vacuo</i> to give (<i>S</i>)-4-acetamido-3-fluoro-5-(1-phenylethoxy)benzoic acid (1.3 g, 4.10 mmol, 82% yield) as a white foam.</div><div class="NLM_p last">LCMS (method: formic) retention time of 0.87 min, [M â H]<sup>â</sup> = 316.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 13.08 (br d, <i>J</i>= 2.4 Hz, 1H), 9.46 (br s, 1H), 7.51â7.42 (m, 2H), 7.39â7.32 (m, 2H), 7.27â7.25 (m, 1H), 7.30â7.25 (m, 1H), 7.29â7.24 (m, 1H), 5.59 (q, <i>J</i> = 6.4 Hz, 1H), 2.10 (s, 3H), 1.55 (d, <i>J</i> = 5.9 Hz, 3H).</div></div><div id="sec4_2_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 4-Acetamido-3-fluoro-<i>N</i>-((1<i>R</i>,4<i>S</i>)-4-hydroxycyclohexyl)-5-((<i>S</i>)-1-phenylethoxy)benzamide (<b>59</b>)</h4><div class="NLM_p">A solution of (S)-4-acetamido-3-fluoro-5-(1-phenylethoxy)benzoic acid (10.0 g, 31.5 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (200 mL) at rt was treated with DIPEA (16.5 mL, 95.0 mmol) then HATU (18.0 g, 47.3 mmol), and the reaction was stirred at this temperature for 20 min. <i>trans</i>-4-Aminocyclohexanol (3.99 g, 34.7 mmol) was added, and stirring at rt was continued. After 30 min, CH<sub>2</sub>Cl<sub>2</sub> (100 mL) was added to aid in stirring the thick suspension formed. After an overall 1 h, the reaction mixture was diluted with 10% MeOH in CH<sub>2</sub>Cl<sub>2</sub> (1500 mL) and washed successively with a 0.5 N HCl aqueous solution (500 mL), a saturated NaHCO<sub>3</sub> aqueous solution (1000 mL), and brine (1000 mL) then dried over MgSO<sub>4</sub> and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (340 g column, gradient 10â70% (3:1 EtOAc/EtOH) in cyclohexane) gave 4-acetamido-3-fluoro-<i>N</i>-((1<i>R</i>,4<i>S</i>)-4-hydroxycyclohexyl)-5-((<i>S</i>)-1-phenylethoxy)benzamide (12.2 g, 89%) as a cream solid.</div><div class="NLM_p last">LCMS (method: high pH) retention time of 0.83 min, [M â H]<sup>â</sup> = 413.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 13.08 (br d, <i>J</i> = 2.4 Hz, 1H), 9.46 (br s, 1H), 7.51â7.42 (m, 2H), 7.39â7.32 (m, 2H), 7.27â7.25 (m, 1H), 7.30â7.25 (m, 1H), 7.29â7.24 (m, 1H), 5.59 (q, <i>J</i> = 6.4 Hz, 1H), 2.10 (s, 3H), 1.55 (d, <i>J</i> = 5.9 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 168.6 (br s, 1C), 166.4 (s, 1C), 158.0 (d, <i>J</i> = 247.2 Hz, 1C), 154.0 (br d, <i>J</i> = 5.1 Hz, 1C), 142.8 (s, 1C), 130.0 (br d, <i>J</i> = 9.5 Hz, 1C), 129.0 (s, 2C), 128.1 (s, 1C), 126.0 (s, 2C), 120.4 (d, <i>J</i> = 15.4 Hz, 1C), 111.2 (br s, 1C), 109.1 (br d, <i>J</i> = 22.7 Hz, 1C), 76.7 (s, 1C), 24.5 (s, 1C), 23.0 (br s, 1C).</div></div><div id="sec4_2_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 4-Amino-3-fluoro-5-hydroxybenzoic Acid (<b>53</b>)</h4><div class="NLM_p">To a solution of 4-amino-3-(benzyloxy)-5-fluorobenzoic acid (1.2 g, 4.20 mmol) in EtOH (10 mL) was added Pd/C (0.447 g, 4.20 mmol) and stirred under H<sub>2</sub> (g) at rt for 2 h. The reaction mixture was filtered through celite and washed with EtOAc, and the eluent was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to give a gray solid. This solid was purified using silica column chromatography, eluting with a gradient of 50% EtOAc/hexane to afford the desired compound 4-amino-3-fluoro-5-hydroxybenzoic acid (351 mg, 1.980 mmol, 47.1% yield) as a light gray solid.</div><div class="NLM_p last">LCMS (method: formic) retention time of 1.18 min, [M + H]<sup>â</sup> = 172.0.</div></div><div id="sec4_2_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 4-Amino-3-(benzyloxy)-5-fluorobenzoic Acid</h4><div class="NLM_p">A solution of 1-(benzyloxy)-5-bromo-3-fluoro-2-nitrobenzene (1 g, 2.70 mmol), sodium acetate (1.107 g, 13.49 mmol), triphenylphosphine (0.283 g, 1.079 mmol), and palladium(II) acetate (0.061 g, 0.270 mmol) in DMF (5 mL) was stirred under an atmosphere of CO (g) at 110 Â°C for 16 h. The reaction mixture was filtered through celite washing with EtOAc, and the eluent was partitioned between EtOAc and water. The organic phase was washed with sat. brine, dried over sodium sulfate, and concentrated to give the crude. The crude was purified using silica gel column chromatography eluting with an isocratic gradient of 70% ETOAc/hexane to afford 4-amino-3-(benzyloxy)-5-fluorobenzoic acid (275 mg, 0.963 mmol, 35.7% yield) as a light gray solid.</div><div class="NLM_p last">LCMS (method: formic) retention time of 4.71 min, [M + H]<sup>â</sup> = 260.1. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) Î´ 12.33â12.52 (m, 1H), 7.49â7.51 (m, 2H), 7.39 (t, 3H, <i>J</i> = 7.3 Hz), 7.30â7.35 (m, 2H), 7.23â7.27 (m, 1H), 5.17â5.22 (m, 2H).</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> eHOMO Calculations</h3><div class="NLM_p last">For each uncharged compound, a single three-dimensional conformation was generated using Babel v 2.3.1.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Semi-empirical AM1 calculations of the highest-occupied molecular orbital were then carried out using Gaussian 03 and energies reported.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Energies > â0.320, and atomic units were flagged as having an elevated risk of a positive result in an Ames test.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> BRD4 Mutant TR-FRET ASSAY</h3><div class="NLM_p">Tandem bromodomains of 6His-Thr-BRD4(1â477) were expressed with an appropriate mutation in BD2 (Y390A) to monitor compound binding to BD1 or in BD1 (97A) to monitor compound binding to BD2. Analogous Y â A mutants were used to measure binding to the other BET bromodomains: 6His-Thr-BRD2 (1â473 Y386A or Y113A), 6His-Thr-BRD3 (1â435 Y348A or Y73A), and 6His-FLAG-Tev-BRDT (1â397 Y309A or Y66A). The AlexaFluor 647-labeled BET bromodomain ligand was prepared as follows: to a solution of AlexaFluor 647 hydroxysuccinimide ester in DMF was added a 1.8-fold excess of <i>N</i>-(5-aminopentyl)-2-((4<i>S</i>)-6-(4-chlorophenyl)-8-methoxy-1-methyl-4<i>H</i>-benzo[<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]-diazepin-4-yl)acetamide, also in DMF, and when thoroughly mixed, the solution was basified by the addition of a 3-fold excess of diisopropylethylamine. The reaction progress was followed by electrospray LC/MS, and when judged complete, the product was isolated and purified by reversed-phase C18 HPLC. The final compound was characterized by mass spectroscopy and analytical reversed-phase HPLC.</div><div class="NLM_p last">Compounds were titrated from 10 mM in 100% DMSO, and 50 nL was transferred to a low-volume black 384-well microtiter plate using a Labcyte Echo 555. A Thermo Scientific Multidrop Combi was used to dispense 5 Î¼L of 20 nM protein in an assay buffer of 50 mM HEPES, 150 mM NaCl, 5% glycerol, 1 mM DTT, and 1 mM CHAPS, pH 7.4, and in the presence of 100 nM fluorescent ligand (â¼<i>K</i><sub>d</sub> concentration for the interaction between BRD4 BD1 and the ligand). After equilibrating for 30 min in the dark at rt, the bromodomain protein/fluorescent ligand interaction was detected using TR-FRET following a 5 Î¼L addition of 3 nM europium chelate labeled anti-6His antibody (PerkinElmer, W1024, AD0111) in assay buffer. Time-resolved fluorescence (TRF) was then detected on a TRF laser-equipped PerkinElmer Envision multimode plate reader (excitation = 337 nm; emission 1 = 615 nm; emission 2 = 665 nm; dual wavelength bias dichroic = 400 nm, 630 nm). The TR-FRET ratio was calculated using the following equation: ratio = ((acceptor fluorescence at 665 nm)/(donor fluorescence at 615 nm)) Ã 1000. TR-FRET ratio data were normalized to high (DMSO) and low (compound control derivative of I-BET762) controls, and IC<sub>50</sub> values were determined for each of the compounds tested by fitting the fluorescence ratio data to a four-parameter model: <i>y</i> = <i>a</i> + ((<i>b</i> â <i>a</i>)/(1 + (10<i><sup>x</sup></i>/10<i><sup>c</sup></i>)<i><sup>d</sup></i>) where â<i>a</i>â is the minimum, â<i>b</i>â is the Hill slope, â<i>c</i>â is the IC<sub>50</sub>, and â<i>d</i>â is the maximum.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> <i>In Vivo</i> DMPK Studies</h3><div class="NLM_p last">All animal studies were ethically reviewed and carried out in accordance with the Animals (Scientific Procedures) Act 1986 and the GSK Policy on the Care, Welfare, and Treatment of Laboratory Animals. For all <i>in vivo</i> studies, the temperature and humidity were nominally maintained at 21 Â± 2 Â°C and 55 Â± 10%, respectively. The diet for rodents was a 5LF2 Eurodent diet (14%) (PMI Labdiet, Richmond, IN) and for dogs was Harlan Teklad 2021C (HarlanTeklad, Madison, WI). There were no known contaminants in the diet or water at concentrations that could interfere with the outcome of the studies.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Meteor Metabolite Prediction</h3><div class="NLM_p last"><i>In silico</i> metabolite predictions were performed with Meteor Nexus (Lhasa Limited) with knowledge base Meteor KB 2018 1.0.0 using mammalian species, site-of-metabolism scoring (with molecular mass variance), a molecular mass similarity threshold of 70, and a score threshold of 70.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> <i>In Vitro</i> Metabolite Identification</h3><div class="NLM_p last">Rat liver and human liver S9 (20 mg/mL) cryopreserved stocks were prepared with addition of cofactors. Compound <b>28</b> was incubated in duplicate at a concentration of 10 Î¼M for 120 min with no-compound control, no-matrix control, and positive control incubations run in parallel for both rat and human S9 samples. Met ID analysis was conducted with use of an LTQ-Orbitrap mass spectrometer using positive-ion FTMS from which comparison of the relevant extracted ion chromatograms was made.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> Physicochemical Properties</h3><div class="NLM_p last">Permeability across a lipid membrane, chromatographic (chrom) log <i>D</i> at pH 7.4, and CLND solubility by precipitation into saline were measured using published protocols.<a onclick="showRef(event, 'ref51 ref52 ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref51 ref52 ref53 ref54">(51â54)</a></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> FaSSIF Solubility</h3><div class="NLM_p last">Compounds were dissolved in DMSO at 2.5 mg/mL and then diluted in fast-state simulated intestinal fluid (FaSSIF, pH 6.5) at 125 Î¼g/mL (final DMSO concentration is 5%). After 16 h of incubation at 25 Â°C, the suspension was filtered. The concentration of the compound was determined by a fast HPLC gradient. The ratio of the peak areas was obtained from the standards, and the sample filtrate was used to calculate the solubility of the compound.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> hWB MCP-1 Assay</h3><div class="NLM_p last">Compounds to be tested were diluted in 100% DMSO to give a range of appropriate concentrations at 140Ã; the required final assay concentration of which, 1 Î¼L, was added to a 96-well tissue culture plate. A volume of 130 Î¼L of human whole blood, collected into a sodium heparin anticoagulant, (1 unit/mL final) was added to each well, and plates were incubated at 37 Â°C (5% CO<sub>2</sub>) for 30 min before the addition of 10 Î¼L of 2.8 Î¼g/mL LPS (<i>Salmonella typhosa</i>) diluted in complete RPMI 1640 (final concentration of 200 ng/mL) to give a total volume of 140 Î¼L/well. After further incubation for 24 h at 37 Â°C, 140 Î¼L of PBS was added to each well. The plates were sealed, shaken for 10 min, and then centrifuged (2500 rpm Ã 10 min). A volume of 100 Î¼L of the supernatant was removed, and MCP-1 levels were assayed immediately by immunoassay (MesoScale Discovery technology).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> BROMOscan Bromodomain Profiling</h3><div class="NLM_p last">BROMOscan bromodomain profiling was provided by Eurofins DiscoverX Corp. (Fremont, CA, U.S.A., <a href="http://www.discoverx.com" class="extLink">http://www.discoverx.com</a>). Determination of the KD between test compounds and DNA-tagged bromodomains was achieved through binding competition against a proprietary reference immobilized ligand.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> Animal Welfare Statement</h3><div class="NLM_p last">All animal studies were ethically reviewed and carried out in accordance with the Animals (Scientific Procedures) Act 1986 and the GSK Policy on the Care, Welfare, and Treatment of Animals.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i86"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00605" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15252" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15252" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00605?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00605</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">S1: assay protocols, S2: crystallization methods, S3: Table S1 - data collection and refinement statistics for BRD2-BD2 X-ray structures, Figure S1: electron density maps for BRD2 BD2 X-ray structures, Figure S2: sequence alignment of BET proteins, Table S2: DiscoverX BROMOscan data for <b>59</b>, Table S3: cross screening data of liability panel for <b>59</b>, S4: DMPK methods, S5: .meteor <i>in silico</i> metabolite predictions of <b>28</b> and <b>59</b>, Figure S3: meteor <i>in silico</i> metabolite predictions of <b>28</b> and <b>59</b>, S6: <i>in vitro</i> metabolism of <b>28</b> in human and rat S9 fractions, Figure S4: <i>in vitro</i> metabolism of <b>28</b> in human and rat liver S9 fractions, S7: selected LCMS traces, S8: <sup>1</sup>H and <sup>13</sup>C NMR traces of compound <b>59</b>, Figure S5: <sup>1</sup>H and <sup>13</sup>C NMR traces of compound <b>59</b>, S9: BRD2 BD2 crystal structure: rational for favored stereochemistry, Figure S6: BRD2 BD2 crystal structure: rational for favored stereochemistry (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00605/suppl_file/jm0c00605_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00605/suppl_file/jm0c00605_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00605/suppl_file/jm0c00605_si_001.pdf">jm0c00605_si_001.pdf (2.78 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00605/suppl_file/jm0c00605_si_002.csv">jm0c00605_si_002.csv (2.63 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Authors will release the atomic coordinates upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00605" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80881" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80881" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alex Preston</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0334-0679" title="Orcid link">http://orcid.org/0000-0003-0334-0679</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c2a3aea7baeca5b1ecb2b0a7b1b6adac82a5b1a9eca1adaf"><span class="__cf_email__" data-cfemail="8cede0e9f4a2ebffa2fcfee9fff8e3e2ccebffe7a2efe3e1">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen Atkinson</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3636-3674" title="Orcid link">http://orcid.org/0000-0003-3636-3674</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#55262130253d303b7b34213e3c3b263a3b641534262127342f303b3036347b363a38"><span class="__cf_email__" data-cfemail="f7848392879f9299d996839c9e99849899c6b796848385968d9299929496d994989a">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul Bamborough</span> - <span class="hlFld-Affiliation affiliation">Platform
Technology and Science, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9479-2894" title="Orcid link">http://orcid.org/0000-0001-9479-2894</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chun-wa Chung</span> - <span class="hlFld-Affiliation affiliation">Platform
Technology and Science, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2480-3110" title="Orcid link">http://orcid.org/0000-0002-2480-3110</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter D. Craggs</span> - <span class="hlFld-Affiliation affiliation">Platform
Technology and Science, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laurie Gordon</span> - <span class="hlFld-Affiliation affiliation">Platform
Technology and Science, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Grandi</span> - <span class="hlFld-Affiliation affiliation">IVIVT
Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstr.
1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James R. J. Gray</span> - <span class="hlFld-Affiliation affiliation">Quantitative
Pharmacology, Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emma J. Jones</span> - <span class="hlFld-Affiliation affiliation">Platform
Technology and Science, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew Lindon</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span>Present Address:
                        Research and Early Development, RIA, Biopharmaceuticals R&D, AstraZeneca, SE-431 83, MÃ¶lndal, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anne-Marie Michon</span> - <span class="hlFld-Affiliation affiliation">IVIVT
Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstr.
1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darren J. Mitchell</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rab K. Prinjha</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Francesco Rianjongdee</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Inmaculada Rioja</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jonathan Seal</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0148-5487" title="Orcid link">http://orcid.org/0000-0003-0148-5487</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simon Taylor</span> - <span class="hlFld-Affiliation affiliation">Quantitative
Pharmacology, Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span>Present Address:
                        Drug Discovery Services Europe, Pharmaron, Hertford Road, Hoddesdon, EN11 9BU, U.K</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ian Wall</span> - <span class="hlFld-Affiliation affiliation">Platform
Technology and Science, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert J. Watson</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James Woolven</span> - <span class="hlFld-Affiliation affiliation">Platform
Technology and Science, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emmanuel H Demont</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, GlaxoSmithKline,
Medicines Research Centre, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7307-3129" title="Orcid link">http://orcid.org/0000-0001-7307-3129</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>All authors were GlaxoSmithKline full-time employees when this study was performed.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): All authors were GlaxoSmithKline full-time employees when this study was performed.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i91">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78167" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78167" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the members of Platform Technology Sciences for protein reagent generation, assay, and crystallization support, Dr. Richard Upton for NMR support, and Nigel Deeks for generation of the <i>in vitro</i> metabolite data.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AMP</td><td class="NLM_def"><p class="first last">artificial membrane permeability</p></td></tr><tr><td class="NLM_term">BD1</td><td class="NLM_def"><p class="first last">bromodomain 1 (N-terminal bromodomain)</p></td></tr><tr><td class="NLM_term">BD2</td><td class="NLM_def"><p class="first last">bromodomain 2 (C-terminal bromodomain)</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromo and extra terminal domain</p></td></tr><tr><td class="NLM_term">BRD2, 3, 4, and T</td><td class="NLM_def"><p class="first last">bromodomain containing proteins 2, 3, 4, and T</p></td></tr><tr><td class="NLM_term">BSEP</td><td class="NLM_def"><p class="first last">bile salt export pump</p></td></tr><tr><td class="NLM_term">CDI</td><td class="NLM_def"><p class="first last">carbonyldiimidazole</p></td></tr><tr><td class="NLM_term">CL<sub>b</sub></td><td class="NLM_def"><p class="first last">blood clearance</p></td></tr><tr><td class="NLM_term">CL<sub>int</sub></td><td class="NLM_def"><p class="first last">intrinsic clearance</p></td></tr><tr><td class="NLM_term">CLND</td><td class="NLM_def"><p class="first last">chemiluminescent nitrogen detection</p></td></tr><tr><td class="NLM_term"><i>D</i></td><td class="NLM_def"><p class="first last">dose</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last"><i>N</i>-ethyl-<i>N</i>-(3-dimethylaminopropyl)carbodiimide</p></td></tr><tr><td class="NLM_term">GSH</td><td class="NLM_def"><p class="first last">glutathione</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate;)</p></td></tr><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">euman ether-a-go-go-related gene</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">hWB</td><td class="NLM_def"><p class="first last">human whole blood</p></td></tr><tr><td class="NLM_term"><i>k</i></td><td class="NLM_def"><p class="first last">elimination rate constant</p></td></tr><tr><td class="NLM_term">KAc</td><td class="NLM_def"><p class="first last">acetylated lysine</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">LLE</td><td class="NLM_def"><p class="first last">lipophilic ligand efficiency</p></td></tr><tr><td class="NLM_term">LLE<sub>at</sub></td><td class="NLM_def"><p class="first last">Astex lipophilic ligand efficiency</p></td></tr><tr><td class="NLM_term">MCP-1/CCL2</td><td class="NLM_def"><p class="first last">monocyte chemoattractant protein 1</p></td></tr><tr><td class="NLM_term">MDAP</td><td class="NLM_def"><p class="first last">mass-directed auto preparation</p></td></tr><tr><td class="NLM_term">MDI</td><td class="NLM_def"><p class="first last">metabolism dependent inhibition</p></td></tr><tr><td class="NLM_term">T<sub>3</sub>P</td><td class="NLM_def"><p class="first last">propanephosphonic acid anhydride</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term"><i>V</i></td><td class="NLM_def"><p class="first last">incubation volume</p></td></tr><tr><td class="NLM_term"><i>V</i><sub>ss</sub></td><td class="NLM_def"><p class="first last">volume of distribution at the steady state</p></td></tr><tr><td class="NLM_term">WPF</td><td class="NLM_def"><p class="first last">tryptophanâprolineâphenylalanine</p></td></tr><tr><td class="NLM_term">Xantphos</td><td class="NLM_def"><p class="first last">4,5-bis(diphenylphosphino)-9,9-dimethylxanthene</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i93">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14452" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14452" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 54 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>â <span class="NLM_lpage">1073</span>, <span class="refDoi">Â DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0ljleLRXfJ384A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappaport, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sison, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chicas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogan, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">524</span>â <span class="NLM_lpage">528</span>, <span class="refDoi">Â DOI: 10.1038/nature10334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1038%2Fnature10334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=21814200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=524-528&author=J.+Zuberauthor=J.+Shiauthor=E.+Wangauthor=A.+R.+Rappaportauthor=H.+Herrmannauthor=E.+A.+Sisonauthor=D.+Magoonauthor=J.+Qiauthor=K.+Blattauthor=M.+Wunderlichauthor=M.+J.+Taylorauthor=C.+Johnsauthor=A.+Chicasauthor=J.+C.+Mulloyauthor=S.+C.+Koganauthor=P.+Brownauthor=P.+Valentauthor=J.+E.+Bradnerauthor=S.+W.+Loweauthor=C.+R.+Vakoc&title=RNAi+screen+identifies+Brd4+as+a+therapeutic+target+in+acute+myeloid+leukaemia&doi=10.1038%2Fnature10334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span></div><div class="casAuthors">Zuber, Johannes; Shi, Junwei; Wang, Eric; Rappaport, Amy R.; Herrmann, Harald; Sison, Edward A.; Magoon, Daniel; Qi, Jun; Blatt, Katharina; Wunderlich, Mark; Taylor, Meredith J.; Johns, Christopher; Chicas, Agustin; Mulloy, James C.; Kogan, Scott C.; Brown, Patrick; Valent, Peter; Bradner, James E.; Lowe, Scott W.; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic pathways can regulate gene expression by controlling and interpreting chromatin modifications.  Cancer cells are characterized by altered epigenetic landscapes, and commonly exploit the chromatin regulatory machinery to enforce oncogenic gene expression programs.  Although chromatin alterations are, in principle, reversible and often amenable to drug intervention, the promise of targeting such pathways therapeutically has been limited by an incomplete understanding of cancer-specific dependencies on epigenetic regulators.  Here the authors describe a non-biased approach to probe epigenetic vulnerabilities in acute myeloid leukemia (AML), an aggressive hematopoietic malignancy that is often assocd. with aberrant chromatin states.  By screening a custom library of small hairpin RNAs (shRNAs) targeting known chromatin regulators in a genetically defined AML mouse model, the authors identify the protein bromodomain-contg. 4 (Brd4) as being critically required for disease maintenance.  Suppression of Brd4 using shRNAs or the small-mol. inhibitor JQ1 led to robust antileukemic effects in vitro and in vivo, accompanied by terminal myeloid differentiation and elimination of leukemia stem cells.  Similar sensitivities were obsd. in a variety of human AML cell lines and primary patient samples, revealing that JQ1 has broad activity in diverse AML subtypes.  The effects of Brd4 suppression are, at least in part, due to its role in sustaining Myc expression to promote aberrant self-renewal, which implicates JQ1 as a pharmacol. means to suppress MYC in cancer.  The authors' results establish small-mol. inhibition of Brd4 as a promising therapeutic strategy in AML and, potentially, other cancers, and highlight the utility of RNA interference (RNAi) screening for revealing epigenetic vulnerabilities that can be exploited for direct pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVyWFIqFkUVrVg90H21EOLACvtfcHk0lhtfzyohhB2tw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP&md5=618793a766f037bff5909524743fedb7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature10334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10334%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DHerrmann%26aufirst%3DH.%26aulast%3DSison%26aufirst%3DE.%2BA.%26aulast%3DMagoon%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DK.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DJohns%26aufirst%3DC.%26aulast%3DChicas%26aufirst%3DA.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DRNAi%2520screen%2520identifies%2520Brd4%2520as%2520a%2520therapeutic%2520target%2520in%2520acute%2520myeloid%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D524%26epage%3D528%26doi%3D10.1038%2Fnature10334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span> <span> </span><span class="NLM_article-title">Place your BETs: the therapeutic potential of bromodomains</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">146</span>â <span class="NLM_lpage">153</span>, <span class="refDoi">Â DOI: 10.1016/j.tips.2011.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.tips.2011.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=22277300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=146-153&author=R.+K.+Prinjhaauthor=J.+Witheringtonauthor=K.+Lee&title=Place+your+BETs%3A+the+therapeutic+potential+of+bromodomains&doi=10.1016%2Fj.tips.2011.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Place your BETs: the therapeutic potential of bromodomains</span></div><div class="casAuthors">Prinjha, R. K.; Witherington, J.; Lee, K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-153</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Therapeutic targeting of the processes that regulate histone modification is a growing area of scientific exploration.  Although most interest has concd. on the various families of enzymes that contribute to these processes, this review focuses on emerging data demonstrating the chem. tractability and therapeutic potential of a hitherto underexplored family of proteins, namely the bromodomain (BRD) family of reader proteins.  These proteins perform a crucial role in translating histone modifications with powerful transcriptional consequences.  We review current knowledge of the biol. of this emergent target class and highlight recent breakthroughs that now make the BRD family of reader proteins attractive for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_N9p7ru787rVg90H21EOLACvtfcHk0lhtfzyohhB2tw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D&md5=ea1e9e5740b1131239aa0407748d549b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPlace%2520your%2520BETs%253A%2520the%2520therapeutic%2520potential%2520of%2520bromodomains%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D146%26epage%3D153%26doi%3D10.1016%2Fj.tips.2011.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET Bromodomain Targets Genetically Diverse Glioblastoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1748</span>â <span class="NLM_lpage">1759</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-12-3066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1158%2F1078-0432.CCR-12-3066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=23403638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVCmsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=1748-1759&author=Z.+Chengauthor=Y.+Gongauthor=Y.+Maauthor=K.+Luauthor=X.+Luauthor=L.+A.+Pierceauthor=R.+C.+Thompsonauthor=S.+Mullerauthor=S.+Knappauthor=J.+Wang&title=Inhibition+of+BET+Bromodomain+Targets+Genetically+Diverse+Glioblastoma&doi=10.1158%2F1078-0432.CCR-12-3066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET Bromodomain Targets Genetically Diverse Glioblastoma</span></div><div class="casAuthors">Cheng, Zhixiang; Gong, Yuanying; Ma, Yufang; Lu, Kaihua; Lu, Xiang; Pierce, Larry A.; Thompson, Reid C.; Muller, Susanne; Knapp, Stefan; Wang, Jialiang</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1748-1759</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Glioblastoma is refractory to conventional therapies.  The bromodomain and extraterminal domain (BET) proteins are epigenetic readers that selectively bind to acetylated lysine residues on histone tails.  These proteins recently emerged as important therapeutic targets in NUT midline carcinoma and several types of hematopoietic cancers.  In this study, the therapeutic potential of a novel BET bromodomain inhibitor, JQ1, was assessed in a panel of genetically heterogeneous glioblastoma samples.  Exptl. Design: The antineoplastic effects of JQ1 were shown using ex vivo cultures derived from primary glioblastoma xenograft lines and surgical specimens of different genetic background.  The in vivo efficacy was assessed in orthotopic glioblastoma tumors.  Results: We showed that JQ1 induced marked G1 cell-cycle arrest and apoptosis, which was phenocopied by knockdown of individual BET family members.  JQ1 treatment resulted in significant changes in expression of genes that play important roles in glioblastoma such as c-Myc, p21CIP1/WAF1, hTERT, Bcl-2, and Bcl-xL.  Unlike the observations in some hematopoietic cancer cell lines, exogenous c-Myc did not significantly protect glioblastoma cells against JQ1.  In contrast, ectopically expressed Bcl-xL partially rescued cells from JQ1-induced apoptosis, and knockdown of p21CIP1/WAF1 attenuated JQ1-induced cell-cycle arrest.  Cells genetically engineered for Akt hyperactivation or p53/Rb inactivation did not compromise JQ1 efficacy, suggesting that these frequently mutated signaling pathways may not confer resistance to JQ1.  Furthermore, JQ1 significantly repressed growth of orthotopic glioblastoma tumors.  Conclusion: Our results suggest potentially broad therapeutic use of BET bromodomain inhibitors for treating genetically diverse glioblastoma tumors.  Clin Cancer Res; 19(7); 1748-59. Â©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7OsFKqlwyDbVg90H21EOLACvtfcHk0liNMvT2KFFPag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVCmsLo%253D&md5=1722189e5d950da233b785558f2c9f58</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-3066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-3066%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DGong%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DPierce%26aufirst%3DL.%2BA.%26aulast%3DThompson%26aufirst%3DR.%2BC.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520BET%2520Bromodomain%2520Targets%2520Genetically%2520Diverse%2520Glioblastoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2013%26volume%3D19%26spage%3D1748%26epage%3D1759%26doi%3D10.1158%2F1078-0432.CCR-12-3066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Costa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanasopoulou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniere, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâSullivan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stankovic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">PFI-1, a Highly Selective Protein Interaction Inhibitor, Targeting BET Bromodomains</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">3336</span>â <span class="NLM_lpage">3346</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-12-3292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1158%2F0008-5472.CAN-12-3292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=23576556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=3336-3346&author=S.+Picaudauthor=D.+Da+Costaauthor=A.+Thanasopoulouauthor=P.+Filippakopoulosauthor=P.+V.+Fishauthor=M.+Philpottauthor=O.+Fedorovauthor=P.+Brennanauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=J.+E.+Bradnerauthor=P.+Taniereauthor=B.+O%E2%80%99Sullivanauthor=S.+M%C3%BCllerauthor=J.+Schwallerauthor=T.+Stankovicauthor=S.+Knapp&title=PFI-1%2C+a+Highly+Selective+Protein+Interaction+Inhibitor%2C+Targeting+BET+Bromodomains&doi=10.1158%2F0008-5472.CAN-12-3292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">PFI-1, a Highly Selective Protein Interaction Inhibitor, Targeting BET Bromodomains</span></div><div class="casAuthors">Picaud, Sarah; Da Costa, David; Thanasopoulou, Angeliki; Filippakopoulos, Panagis; Fish, Paul V.; Philpott, Martin; Fedorov, Oleg; Brennan, Paul; Bunnage, Mark E.; Owen, Dafydd R.; Bradner, James E.; Taniere, Philippe; O'Sullivan, Brendan; Mueller, Susanne; Schwaller, Juerg; Stankovic, Tatjana; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3336-3346</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bromo and extra terminal (BET) proteins (BRD2, BRD3, BRD4, and BRDT) are transcriptional regulators required for efficient expression of several growth promoting and antiapoptotic genes as well as for cell-cycle progression.  BET proteins are recruited on transcriptionally active chromatin via their two N-terminal bromo-domains (BRD), a protein interaction module that specifically recognizes acetylated lysine residues in histones H3 and H4.  Inhibition of the BET-histone interaction results in transcriptional downregulation of a no. of oncogenes, providing a novel pharmacol. strategy for the treatment of cancer.  Here, we present a potent and highly selective dihydroquinazoline-2-one inhibitor, PFFI-1, which efficiently blocks the interaction of BET BRDs with acetylated histone tails.  Cocrystal structures showed that PFI-1 acts as an acetyl-lysine (Kac) mimetic inhibitor efficiently occupying the Kac binding site in BRD4 and BRD2.  PFI-1 has antiproliferative effects on leukemic cell lines and efficiently abrogates their clonogenic growth.  Exposure of sensitive cell lines with PFI-I results in G1 cell-cycle arrest, downregulation of MYC expression, as well as induction of apoptosis and induces differentiation of primary leukemic blasts.  Intriguingly, cells exposed to PFI-1 showed significant downregulation of Aurora B kinase, thus attenuating phosphorylation of the Aurora substrate H3S10, providing an alternative strategy for the specific inhibition of this well-established oncol. target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFGSqZBi1UELVg90H21EOLACvtfcHk0liNMvT2KFFPag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D&md5=4d35b56dc6e0cdbc0f3e6ccdabefca94</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-3292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-3292%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DDa%2BCosta%26aufirst%3DD.%26aulast%3DThanasopoulou%26aufirst%3DA.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTaniere%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DB.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DStankovic%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DPFI-1%252C%2520a%2520Highly%2520Selective%2520Protein%2520Interaction%2520Inhibitor%252C%2520Targeting%2520BET%2520Bromodomains%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D3336%26epage%3D3346%26doi%3D10.1158%2F0008-5472.CAN-12-3292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segura, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontanals-Cirera, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaziel-Sovran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guijarro, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanniford, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GonzÃ¡lez-Gomez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morante, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubierre, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darvishian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernando, E.</span></span> <span> </span><span class="NLM_article-title">BRD4 Sustains Melanoma Proliferation and Represents a New Target for Epigenetic Therapy</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">6264</span>â <span class="NLM_lpage">6276</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-13-0122-T</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1158%2F0008-5472.CAN-13-0122-T" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=23950209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6264-6276&author=M.+F.+Seguraauthor=B.+Fontanals-Cireraauthor=A.+Gaziel-Sovranauthor=M.+V.+Guijarroauthor=D.+Hannifordauthor=G.+Zhangauthor=P.+Gonz%C3%A1lez-Gomezauthor=M.+Moranteauthor=L.+Jubierreauthor=W.+Zhangauthor=F.+Darvishianauthor=M.+Ohlmeyerauthor=I.+Osmanauthor=M.-M.+Zhouauthor=E.+Hernando&title=BRD4+Sustains+Melanoma+Proliferation+and+Represents+a+New+Target+for+Epigenetic+Therapy&doi=10.1158%2F0008-5472.CAN-13-0122-T"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Sustains Melanoma Proliferation and Represents a New Target for Epigenetic Therapy</span></div><div class="casAuthors">Segura, Miguel F.; Fontanals-Cirera, Barbara; Gaziel-Sovran, Avital; Guijarro, Maria V.; Hanniford, Doug; Zhang, Guangtao; Gonzalez-Gomez, Pilar; Morante, Marta; Jubierre, Luz; Zhang, Weijia; Darvishian, Farbod; Ohlmeyer, Michael; Osman, Iman; Zhou, Ming-Ming; Hernando, Eva</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6264-6276</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Metastatic melanoma remains a mostly incurable disease.  Although newly approved targeted therapies are efficacious in a subset of patients, resistance and relapse rapidly ensue.  Alternative therapeutic strategies to manipulate epigenetic regulators and disrupt the transcriptional program that maintains tumor cell identity are emerging.  Bromodomain and extraterminal domain (BET) proteins are epigenome readers known to exert key roles at the interface between chromatin remodeling and transcriptional regulation.  Here, we report that BRD4, a BET family member, is significantly upregulated in primary and metastatic melanoma tissues compared with melanocytes and nevi.  Treatment with BET inhibitors impaired melanoma cell proliferation in vitro and tumor growth and metastatic behavior in vivo, effects that were mostly recapitulated by individual silencing of BRD4.  RNA sequencing of BET inhibitor-treated cells followed by Gene Ontol. anal. showed a striking impact on transcriptional programs controlling cell growth, proliferation, cell-cycle regulation, and differentiation.  In particular, we found that, rapidly after BET displacement, key cell-cycle genes (SKP2, ERK1, and c-MYC) were downregulated concomitantly with the accumulation of cyclin-dependent kinase (CDK) inhibitors (p21 and p27), followed by cell-cycle arrest.  Importantly, BET inhibitor efficacy was not influenced by BRAF or NRAS mutational status, opening the possibility of using these small-mol. compds. to treat patients for whom no effective targeted therapy exists.  Collectively, our study reveals a crit. role for BRD4 in melanoma tumor maintenance and renders it a legitimate and novel target for epigenetic therapy directed against the core transcriptional program of melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq80TZH2ueQFbVg90H21EOLACvtfcHk0ljsOvytNGdXCA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7N&md5=a4a77f188de01c4c84f7293a32478fc7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0122-T&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0122-T%26sid%3Dliteratum%253Aachs%26aulast%3DSegura%26aufirst%3DM.%2BF.%26aulast%3DFontanals-Cirera%26aufirst%3DB.%26aulast%3DGaziel-Sovran%26aufirst%3DA.%26aulast%3DGuijarro%26aufirst%3DM.%2BV.%26aulast%3DHanniford%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DGonz%25C3%25A1lez-Gomez%26aufirst%3DP.%26aulast%3DMorante%26aufirst%3DM.%26aulast%3DJubierre%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DDarvishian%26aufirst%3DF.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DOsman%26aufirst%3DI.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DHernando%26aufirst%3DE.%26atitle%3DBRD4%2520Sustains%2520Melanoma%2520Proliferation%2520and%2520Represents%2520a%2520New%2520Target%2520for%2520Epigenetic%2520Therapy%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D6264%26epage%3D6276%26doi%3D10.1158%2F0008-5472.CAN-13-0122-T" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaidos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouvedenou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marigo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhry, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotolo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Mahdi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahemtulla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleijnen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karadimitris, A.</span></span> <span> </span><span class="NLM_article-title">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">697</span>â <span class="NLM_lpage">705</span>, <span class="refDoi">Â DOI: 10.1182/blood-2013-01-478420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1182%2Fblood-2013-01-478420" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=24335499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2014&pages=697-705&author=A.+Chaidosauthor=V.+Caputoauthor=K.+Gouvedenouauthor=B.+Liuauthor=I.+Marigoauthor=M.+S.+Chaudhryauthor=A.+Rotoloauthor=D.+F.+Toughauthor=N.+N.+Smithersauthor=A.+K.+Bassilauthor=T.+D.+Chapmanauthor=N.+R.+Harkerauthor=O.+Barbashauthor=P.+Tumminoauthor=N.+Al-Mahdiauthor=A.+C.+Haynesauthor=L.+Cutlerauthor=B.+Leauthor=A.+Rahemtullaauthor=I.+Robertsauthor=M.+Kleijnenauthor=J.+J.+Witheringtonauthor=N.+J.+Parrauthor=R.+K.+Prinjhaauthor=A.+Karadimitris&title=Potent+antimyeloma+activity+of+the+novel+bromodomain+inhibitors+I-BET151+and+I-BET762&doi=10.1182%2Fblood-2013-01-478420"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span></div><div class="casAuthors">Chaidos, Aristeidis; Caputo, Valentina; Gouvedenou, Katerina; Liu, Binbin; Marigo, Ilaria; Chaudhry, Mohammed Suhail; Rotolo, Antonia; Tough, David F.; Smithers, Nicholas N.; Bassil, Anna K.; Chapman, Trevor D.; Harker, Nicola R.; Barbash, Olena; Tummino, Peter; Al-Mahdi, Niam; Haynes, Andrea C.; Cutler, Leanne; Le, Bao Chau; Rahemtulla, Amin; Roberts, Irene; Kleijnen, Maurits; Witherington, Jason J.; Parr, Nigel J.; Prinjha, Rab K.; Karadimitris, Anastasios</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">697-705</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD2-4 inhibitors hold therapeutic promise in preclin. models of hematol. malignancies.  However, translation of these data to mols. suitable for clin. development has yet to be accomplished.  Herein we expand the mechanistic understanding of BET inhibitors in multiple myeloma by using the chem. probe mol. I-BET151.  I-BET151 induces apoptosis and exerts strong antiproliferative effect in vitro and in vivo.  This is assocd. with contrasting effects on oncogenic MYC and HEXIM1, an inhibitor of the transcriptional activator P-TEFb.  I-BET151 causes transcriptional repression of MYC and MYC-dependent programs by abrogating recruitment to the chromatin of the P-TEFb component CDK9 in a BRD2-4-dependent manner.  In contrast, transcriptional upregulation of HEXIM1 is BRD2-4 independent.  Finally, preclin. studies show that I-BET762 has a favorable pharmacol. profile as an oral agent and that it inhibits myeloma cell proliferation, resulting in survival advantage in a systemic myeloma xenograft model.  These data provide a strong rationale for extending the clin. testing of the novel antimyeloma agent I-BET762 and reveal insights into biol. pathways required for myeloma cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0bNtpJQBXsrVg90H21EOLACvtfcHk0ljfH5y0zjrU7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D&md5=f8f734f84a07d3b86ebdc264f6741b22</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-01-478420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-01-478420%26sid%3Dliteratum%253Aachs%26aulast%3DChaidos%26aufirst%3DA.%26aulast%3DCaputo%26aufirst%3DV.%26aulast%3DGouvedenou%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DMarigo%26aufirst%3DI.%26aulast%3DChaudhry%26aufirst%3DM.%2BS.%26aulast%3DRotolo%26aufirst%3DA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DTummino%26aufirst%3DP.%26aulast%3DAl-Mahdi%26aufirst%3DN.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLe%26aufirst%3DB.%26aulast%3DRahemtulla%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DI.%26aulast%3DKleijnen%26aufirst%3DM.%26aulast%3DWitherington%26aufirst%3DJ.%2BJ.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DKaradimitris%26aufirst%3DA.%26atitle%3DPotent%2520antimyeloma%2520activity%2520of%2520the%2520novel%2520bromodomain%2520inhibitors%2520I-BET151%2520and%2520I-BET762%26jtitle%3DBlood%26date%3D2014%26volume%3D123%26spage%3D697%26epage%3D705%26doi%3D10.1182%2Fblood-2013-01-478420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasconi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarantelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascione, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponzoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mensah, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stussi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inghirami, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">The BET Bromodomain Inhibitor OTX015 Affects Pathogenetic Pathways in Preclinical B-cell Tumor Models and Synergizes with Targeted Drugs</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1628</span>â <span class="NLM_lpage">1638</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-14-1561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1158%2F1078-0432.CCR-14-1561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=25623213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1628-1638&author=M.+Boiauthor=E.+Gaudioauthor=P.+Bonettiauthor=I.+Kweeauthor=E.+Bernasconiauthor=C.+Tarantelliauthor=A.+Rinaldiauthor=M.+Testoniauthor=L.+Cascioneauthor=M.+Ponzoniauthor=A.+A.+Mensahauthor=A.+Stathisauthor=G.+Stussiauthor=M.+E.+Riveiroauthor=P.+Heraitauthor=G.+Inghiramiauthor=E.+Cvitkovicauthor=E.+Zuccaauthor=F.+Bertoni&title=The+BET+Bromodomain+Inhibitor+OTX015+Affects+Pathogenetic+Pathways+in+Preclinical+B-cell+Tumor+Models+and+Synergizes+with+Targeted+Drugs&doi=10.1158%2F1078-0432.CCR-14-1561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The BET Bromodomain Inhibitor OTX015 Affects Pathogenetic Pathways in Preclinical B-cell Tumor Models and Synergizes with Targeted Drugs</span></div><div class="casAuthors">Boi, Michela; Gaudio, Eugenio; Bonetti, Paola; Kwee, Ivo; Bernasconi, Elena; Tarantelli, Chiara; Rinaldi, Andrea; Testoni, Monica; Cascione, Luciano; Ponzoni, Maurilio; Mensah, Afua Adjeiwaa; Stathis, Anastasios; Stussi, Georg; Riveiro, Maria Eugenia; Herait, Patrice; Inghirami, Giorgio; Cvitkovic, Esteban; Zucca, Emanuele; Bertoni, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1628-1638</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">In cancer cells, the epigenome is often deregulated, and inhibition of the bromodomain and extra-terminal (BET) family of bromodomain-contg. proteins is a novel epigenetic therapeutic approach.  Preliminary results of an ongoing phase I trial have reported promising activity and tolerability with the new BET bromodomain inhibitor OTX015.  We assessed the preclin. activity of OTX015 as single agent and in combination in mature B-cell lymphoma models and performed in vitro and in vivo expts. to identify the mechanism of action and the genetic features assocd. with sensitivity to the compd.  OTX015 showed antiproliferative activity in a large panel of cell lines derived from mature B-cell lymphoid tumors with median IC50 of 240 nmol/L, without significant differences among the different histotypes.  In vitro and in vivo expts. showed that OTX015 targeted NFKB/TLR/JAK/STAT signaling pathways, MYC- and E2F1-regulated genes, cell-cycle regulation, and chromatin structure.  OTX015 presented in vitro synergism with several anticancer agents, esp. with mTOR and BTK inhibitors.  Gene expression signatures assocd. with different degrees of sensitivity to OTX015 were identified.  Although OTX015 was mostly cytostatic, the compd. induced apoptosis in a genetically defined subgroup of cells, derived from activated B-cell-like diffuse large B-cell lymphoma, bearing wtTP53, mutations in MYD88, and CD79B or CARD11.  Together with the data coming from the ongoing phase I study, the in vitro and in vivo data presented here provide the basis for further clin. investigation of OTX015 as single agent and in combination therapies.  Clin Cancer Res; 21(7); 1628-38. Â©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJcy3O1ElDJ7Vg90H21EOLACvtfcHk0ljfH5y0zjrU7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D&md5=5742b4d5884261d546588a339f3c2ea0</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-1561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-1561%26sid%3Dliteratum%253Aachs%26aulast%3DBoi%26aufirst%3DM.%26aulast%3DGaudio%26aufirst%3DE.%26aulast%3DBonetti%26aufirst%3DP.%26aulast%3DKwee%26aufirst%3DI.%26aulast%3DBernasconi%26aufirst%3DE.%26aulast%3DTarantelli%26aufirst%3DC.%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DTestoni%26aufirst%3DM.%26aulast%3DCascione%26aufirst%3DL.%26aulast%3DPonzoni%26aufirst%3DM.%26aulast%3DMensah%26aufirst%3DA.%2BA.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DStussi%26aufirst%3DG.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DInghirami%26aufirst%3DG.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DThe%2520BET%2520Bromodomain%2520Inhibitor%2520OTX015%2520Affects%2520Pathogenetic%2520Pathways%2520in%2520Preclinical%2520B-cell%2520Tumor%2520Models%2520and%2520Synergizes%2520with%2520Targeted%2520Drugs%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D1628%26epage%3D1638%26doi%3D10.1158%2F1078-0432.CCR-14-1561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">da
Motta, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ledaki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purshouse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bastiani, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baban, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morotti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigfield, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klamt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, A.</span></span> <span> </span><span class="NLM_article-title">The BET inhibitor JQ1 selectively impairs tumour response to hypoxia and downregulates CA9 and angiogenesis in triple negative breast cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">122</span>â <span class="NLM_lpage">132</span>, <span class="refDoi">Â DOI: 10.1038/onc.2016.184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1038%2Fonc.2016.184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=27292261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC28Xps1ejtr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=122-132&author=L.+L.+da%0AMottaauthor=I.+Ledakiauthor=K.+Purshouseauthor=S.+Haiderauthor=M.+A.+De+Bastianiauthor=D.+Babanauthor=M.+Morottiauthor=G.+Steersauthor=S.+Wigfieldauthor=E.+Bridgesauthor=J.+L.+Liauthor=S.+Knappauthor=D.+Ebnerauthor=F.+Klamtauthor=A.+L.+Harrisauthor=A.+McIntyre&title=The+BET+inhibitor+JQ1+selectively+impairs+tumour+response+to+hypoxia+and+downregulates+CA9+and+angiogenesis+in+triple+negative+breast+cancer&doi=10.1038%2Fonc.2016.184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The BET inhibitor JQ1 selectively impairs tumour response to hypoxia and downregulates CA9 and angiogenesis in triple negative breast cancer</span></div><div class="casAuthors">da Motta, L. L.; Ledaki, I.; Purshouse, K.; Haider, S.; De Bastiani, M. A.; Baban, D.; Morotti, M.; Steers, G.; Wigfield, S.; Bridges, E.; Li, J.-L.; Knapp, S.; Ebner, D.; Klamt, F.; Harris, A. L.; McIntyre, A.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-132</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The availability of bromodomain and extra-terminal inhibitors (BETi) has enabled translational epigenetic studies in cancer.  BET proteins regulate transcription by selectively recognizing acetylated lysine residues on chromatin.  BETi compete with this process leading to both downregulation and upregulation of gene expression.  Hypoxia enables progression of triple neg. breast cancer (TNBC), the most aggressive form of breast cancer, partly by driving metabolic adaptation, angiogenesis and metastasis through upregulation of hypoxia-regulated genes (for example, carbonic anhydrase 9 (CA9) and vascular endothelial growth factor A (VEGF-A).  Responses to hypoxia can be mediated epigenetically, thus we investigated whether BETi JQ1 could impair the TNBC response induced by hypoxia and exert anti-tumor effects.  JQ1 significantly modulated 44% of hypoxia-induced genes, of which two-thirds were downregulated including CA9 and VEGF-A.  JQ1 prevented HIF binding to the hypoxia response element in CA9 promoter, but did not alter HIF expression or activity, suggesting some HIF targets are BET-dependent.  JQ1 reduced TNBC growth in vitro and in vivo and inhibited xenograft vascularization.  These findings identify that BETi dually targets angiogenesis and the hypoxic response, an effective combination at reducing tumor growth in preclin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq79-F0o2tp6bVg90H21EOLACvtfcHk0ljfH5y0zjrU7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xps1ejtr4%253D&md5=9f1592548db8e6306703bdab7e1e1c90</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.184%26sid%3Dliteratum%253Aachs%26aulast%3Dda%2BMotta%26aufirst%3DL.%2BL.%26aulast%3DLedaki%26aufirst%3DI.%26aulast%3DPurshouse%26aufirst%3DK.%26aulast%3DHaider%26aufirst%3DS.%26aulast%3DDe%2BBastiani%26aufirst%3DM.%2BA.%26aulast%3DBaban%26aufirst%3DD.%26aulast%3DMorotti%26aufirst%3DM.%26aulast%3DSteers%26aufirst%3DG.%26aulast%3DWigfield%26aufirst%3DS.%26aulast%3DBridges%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DJ.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DEbner%26aufirst%3DD.%26aulast%3DKlamt%26aufirst%3DF.%26aulast%3DHarris%26aufirst%3DA.%2BL.%26aulast%3DMcIntyre%26aufirst%3DA.%26atitle%3DThe%2520BET%2520inhibitor%2520JQ1%2520selectively%2520impairs%2520tumour%2520response%2520to%2520hypoxia%2520and%2520downregulates%2520CA9%2520and%2520angiogenesis%2520in%2520triple%2520negative%2520breast%2520cancer%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D122%26epage%3D132%26doi%3D10.1038%2Fonc.2016.184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rhyasen, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dulak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boiko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castriotta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahdesmaki, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâConnor, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dry, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">AZD5153: A Novel Bivalent BET Bromodomain Inhibitor Highly Active against Hematologic Malignancies</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2563</span>â <span class="NLM_lpage">2574</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.MCT-16-0141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1158%2F1535-7163.MCT-16-0141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=27573426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2563-2574&author=G.+W.+Rhyasenauthor=M.+M.+Hattersleyauthor=Y.+Yaoauthor=A.+Dulakauthor=W.+Wangauthor=P.+Petterutiauthor=I.+L.+Daleauthor=S.+Boikoauthor=T.+Cheungauthor=J.+Zhangauthor=S.+Wenauthor=L.+Castriottaauthor=D.+Lawsonauthor=M.+Collinsauthor=L.+Baoauthor=M.+J.+Ahdesmakiauthor=G.+Walkerauthor=G.+O%E2%80%99Connorauthor=T.+C.+Yehauthor=A.+A.+Rabowauthor=J.+R.+Dryauthor=C.+Reimerauthor=P.+Lyneauthor=G.+B.+Millsauthor=S.+E.+Fawellauthor=M.+J.+Waringauthor=M.+Zindaauthor=E.+Clarkauthor=H.+Chen&title=AZD5153%3A+A+Novel+Bivalent+BET+Bromodomain+Inhibitor+Highly+Active+against+Hematologic+Malignancies&doi=10.1158%2F1535-7163.MCT-16-0141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">AZD5153: a novel bivalent BET bromodomain inhibitor highly active against hematologic malignancies</span></div><div class="casAuthors">Rhyasen, Garrett W.; Hattersley, Maureen M.; Yao, Yi; Dulak, Austin; Wang, Wenxian; Petteruti, Philip; Dale, Ian L.; Boiko, Scott; Cheung, Tony; Zhang, Jingwen; Wen, Shenghua; Castriotta, Lillian; Lawson, Deborah; Collins, Michael; Bao, Larry; Ahdesmaki, Miika J.; Walker, Graeme; O'Connor, Greg; Yeh, Tammie C.; Rabow, Alfred A.; Dry, Jonathan R.; Reimer, Corinne; Lyne, Paul; Mills, Gordon B.; Fawell, Stephen E.; Waring, Michael J.; Zinda, Michael; Clark, Edwin; Chen, Huawei</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2563-2574</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD4 regulates gene expression via recruitment of transcriptional regulatory complexes to acetylated chromatin.  Pharmacol. targeting of BRD4 bromodomains by small mol. inhibitors has proven to be an effective means to disrupt aberrant transcriptional programs crit. for tumor growth and/or survival.  Herein, we report AZD5153, a potent, selective, and orally available BET/BRD4 bromodomain inhibitor possessing a bivalent binding mode.  Unlike previously described monovalent inhibitors, AZD5153 ligates two bromodomains in BRD4 simultaneously.  The enhanced avidity afforded through bivalent binding translates into increased cellular and antitumor activity in preclin. hematol. tumor models.  In vivo administration of AZD5153 led to tumor stasis or regression in multiple xenograft models of acute myeloid leukemia, multiple myeloma, and diffuse large B-cell lymphoma.  The relationship between AZD5153 exposure and efficacy suggests that prolonged BRD4 target coverage is a primary efficacy driver.  AZD5153 treatment markedly affects transcriptional programs of MYC, E2F, and mTOR.  Of note, mTOR pathway modulation is assocd. with cell line sensitivity to AZD5153.  Transcriptional modulation of MYC and HEXIM1 was confirmed in AZD5153-treated human whole blood, thus supporting their use as clin. pharmacodynamic biomarkers.  This study establishes AZD5153 as a highly potent, orally available BET/BRD4 inhibitor and provides a rationale for clin. development in hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAiKYOFhro1bVg90H21EOLACvtfcHk0liuXXNZE4HW3A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D&md5=8d0ac899d081705aaa12d49fd90165a3</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0141%26sid%3Dliteratum%253Aachs%26aulast%3DRhyasen%26aufirst%3DG.%2BW.%26aulast%3DHattersley%26aufirst%3DM.%2BM.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DDulak%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DDale%26aufirst%3DI.%2BL.%26aulast%3DBoiko%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DS.%26aulast%3DCastriotta%26aufirst%3DL.%26aulast%3DLawson%26aufirst%3DD.%26aulast%3DCollins%26aufirst%3DM.%26aulast%3DBao%26aufirst%3DL.%26aulast%3DAhdesmaki%26aufirst%3DM.%2BJ.%26aulast%3DWalker%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DG.%26aulast%3DYeh%26aufirst%3DT.%2BC.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DDry%26aufirst%3DJ.%2BR.%26aulast%3DReimer%26aufirst%3DC.%26aulast%3DLyne%26aufirst%3DP.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DFawell%26aufirst%3DS.%2BE.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DAZD5153%253A%2520A%2520Novel%2520Bivalent%2520BET%2520Bromodomain%2520Inhibitor%2520Highly%2520Active%2520against%2520Hematologic%2520Malignancies%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2563%26epage%3D2574%26doi%3D10.1158%2F1535-7163.MCT-16-0141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Epigenetic drug discovery: breaking through the immune barrier</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">835</span>â <span class="NLM_lpage">853</span>, <span class="refDoi">Â DOI: 10.1038/nrd.2016.185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1038%2Fnrd.2016.185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=27765940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWhsLnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=835-853&author=D.+F.+Toughauthor=P.+P.+Takauthor=A.+Tarakhovskyauthor=R.+K.+Prinjha&title=Epigenetic+drug+discovery%3A+breaking+through+the+immune+barrier&doi=10.1038%2Fnrd.2016.185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic drug discovery: breaking through the immune barrier</span></div><div class="casAuthors">Tough, David F.; Tak, Paul P.; Tarakhovsky, Alexander; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">835-853</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Immune-mediated diseases are clin. heterogeneous but they share genetic and pathogenic mechanisms.  These diseases may develop from the interplay of genetic factors and environmental or lifestyle factors.  Exposure to such factors, including infectious agents, is assocd. with coordinated changes in gene transcription owing to epigenetic alterations.  A growing understanding of how epigenetic mechanisms control gene expression patterns and cell function has been aided by the development of small-mol. inhibitors that target these processes.  These chem. tools have helped to reveal the importance of epigenetics in guiding cell fate decisions during immune responses and have also highlighted the potential for targeting epigenetic mechanisms for the treatment of inflammation and immune-mediated diseases.  In this Review, we discuss the most advanced areas of epigenetic drug development for autoimmune and inflammatory diseases and summarize the promising preclin. data in this exciting and evolving field.  These agents will inevitably begin to move into clin. trials for use in patients with immune-mediated diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplOyEVMzBi37Vg90H21EOLACvtfcHk0li3GsRylVOJgg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWhsLnE&md5=f939bcbbe3d74ce24718eef706aae3d1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.185%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DEpigenetic%2520drug%2520discovery%253A%2520breaking%2520through%2520the%2520immune%2520barrier%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D835%26epage%3D853%26doi%3D10.1038%2Fnrd.2016.185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>â <span class="NLM_lpage">1123</span>, <span class="refDoi">Â DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-w.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0li3GsRylVOJgg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bandukwala, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamperti, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molesworth, A. M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">14532</span>â <span class="NLM_lpage">14537</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1212264109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1073%2Fpnas.1212264109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=22912406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14532-14537&author=H.+S.+Bandukwalaauthor=J.+Gagnonauthor=S.+Togherauthor=J.+A.+Greenbaumauthor=E.+D.+Lampertiauthor=N.+J.+Parrauthor=A.+M.+H.+Molesworthauthor=N.+Smithersauthor=K.+Leeauthor=J.+Witheringtonauthor=D.+F.+Toughauthor=R.+K.+Prinjhaauthor=B.+Petersauthor=A.+Rao&title=Selective+inhibition+of+CD4%2B+T-cell+cytokine+production+and+autoimmunity+by+BET+protein+and+c-Myc+inhibitors&doi=10.1073%2Fpnas.1212264109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span></div><div class="casAuthors">Bandukwala, Hozefa S.; Gagnon, John; Togher, Susan; Greenbaum, Jason A.; Lamperti, Edward D.; Parr, Nigel J.; Molesworth, Amy M. H.; Smithers, Nicholas; Lee, Kevin; Witherington, Jason; Tough, David F.; Prinjha, Rab K.; Peters, Bjoern; Rao, Anjana</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14532-14537, S14532/1-S14532/21</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins bind acetylated lysine residues on histone tails and are involved in the recruitment of addnl. factors that mediate histone modifications and enable transcription.  A compd., I-BET-762, that inhibits binding of an acetylated histone peptide to proteins of the bromodomain and extra-terminal domain (BET) family, was previously shown to suppress the prodn. of proinflammatory proteins by macrophages and block acute inflammation in mice.  Here, we investigated the effect of short-term treatment with I-BET-762 on T-cell function.  Treatment of naive CD4+ T cells with I-BET-762 during the first 2 d of differentiation had long-lasting effects on subsequent gene expression and cytokine prodn.  Gene expression anal. revealed up-regulated expression of several antiinflammatory gene products, including IL-10, Lag3, and Egr2, and down-regulated expression of several proinflammatory cytokines including GM-CSF and IL-17.  The short 2-d treatment with I-BET-762 inhibited the ability of antigen-specific T cells, differentiated under Th1 but not Th17 conditions in vitro, to induce pathogenesis in an adoptive transfer model of exptl. autoimmune encephalomyelitis.  The suppressive effects of I-BET-762 on T-cell mediated inflammation in vivo were accompanied by decreased recruitment of macrophages, consistent with decreased GM-CSF prodn. by CNS-infiltrating T cells.  These effects were mimicked by an inhibitor of c-myc function, implicating reduced expression of c-myc and GM-CSF as one avenue by which I-BET-762 suppresses the inflammatory functions of T cells.  Our study demonstrates that inhibiting the functions of BET-family proteins during early T-cell differentiation causes long-lasting suppression of the proinflammatory functions of Th1 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5akF95GW7K7Vg90H21EOLACvtfcHk0li3GsRylVOJgg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO&md5=110cd83c79e452cb99d01ad902506a5b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1212264109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1212264109%26sid%3Dliteratum%253Aachs%26aulast%3DBandukwala%26aufirst%3DH.%2BS.%26aulast%3DGagnon%26aufirst%3DJ.%26aulast%3DTogher%26aufirst%3DS.%26aulast%3DGreenbaum%26aufirst%3DJ.%2BA.%26aulast%3DLamperti%26aufirst%3DE.%2BD.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DMolesworth%26aufirst%3DA.%2BM.%2BH.%26aulast%3DSmithers%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPeters%26aufirst%3DB.%26aulast%3DRao%26aufirst%3DA.%26atitle%3DSelective%2520inhibition%2520of%2520CD4%252B%2520T-cell%2520cytokine%2520production%2520and%2520autoimmunity%2520by%2520BET%2520protein%2520and%2520c-Myc%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D14532%26epage%3D14537%26doi%3D10.1073%2Fpnas.1212264109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolajczyk, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>190</i></span>,  <span class="NLM_fpage">3670</span>â <span class="NLM_lpage">3678</span>, <span class="refDoi">Â DOI: 10.4049/jimmunol.1202838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.4049%2Fjimmunol.1202838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=23420887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=190&publication_year=2013&pages=3670-3678&author=A.+C.+Belkinaauthor=B.+S.+Nikolajczykauthor=G.+V.+Denis&title=BET+Protein+Function+Is+Required+for+Inflammation%3A+Brd2+Genetic+Disruption+and+BET+Inhibitor+JQ1+Impair+Mouse+Macrophage+Inflammatory+Responses&doi=10.4049%2Fjimmunol.1202838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span></div><div class="casAuthors">Belkina, Anna C.; Nikolajczyk, Barbara S.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">190</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3670-3678</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Histone acetylation regulates activation and repression of multiple inflammatory genes known to play crit. roles in chronic inflammatory diseases.  However, proteins responsible for translating the histone acetylation code into an orchestrated proinflammatory cytokine response remain poorly characterized.  Bromodomain and extraterminal (BET) proteins are "readers" of histone acetylation marks, with demonstrated roles in gene transcription, but the ability of BET proteins to coordinate the response of inflammatory cytokine genes through translation of histone marks is unknown.  We hypothesize that members of the BET family of dual bromodomain-contg. transcriptional regulators directly control inflammatory genes.  We examd. the genetic model of brd2 lo mice, a BET protein hypomorph, to show that Brd2 is essential for proinflammatory cytokine prodn. in macrophages.  Studies that use small interfering RNA knockdown and a small-mol. inhibitor of BET protein binding, JQ1, independently demonstrate BET proteins are crit. for macrophage inflammatory responses.  Furthermore, we show that Brd2 and Brd4 phys. assoc. with the promoters of inflammatory cytokine genes in macrophages.  This assocn. is absent in the presence of BET inhibition by JQ1.  Finally, we demonstrate that JQ1 ablates cytokine prodn. in vitro and blunts the "cytokine storm" in endotoxemic mice by reducing levels of IL-6 and TNF-Î± while rescuing mice from LPS-induced death.  We propose that targeting BET proteins with small-mol. inhibitors will benefit hyperinflammatory conditions assocd. with high levels of cytokine prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrbebEe0G277Vg90H21EOLACvtfcHk0lgHlji_-VV0cQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D&md5=8c4f69177b9ed814a7db339a8e2ab7ed</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1202838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1202838%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DNikolajczyk%26aufirst%3DB.%2BS.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520Protein%2520Function%2520Is%2520Required%2520for%2520Inflammation%253A%2520Brd2%2520Genetic%2520Disruption%2520and%2520BET%2520Inhibitor%2520JQ1%2520Impair%2520Mouse%2520Macrophage%2520Inflammatory%2520Responses%26jtitle%3DJ.%2520Immunol.%26date%3D2013%26volume%3D190%26spage%3D3670%26epage%3D3678%26doi%3D10.4049%2Fjimmunol.1202838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mele, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmeron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition suppresses TH17-mediated pathology</span>. <i>J. Exp. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>210</i></span>,  <span class="NLM_fpage">2181</span>â <span class="NLM_lpage">2190</span>, <span class="refDoi">Â DOI: 10.1084/jem.20130376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1084%2Fjem.20130376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=24101376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ygsLbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=210&publication_year=2013&pages=2181-2190&author=D.+A.+Meleauthor=A.+Salmeronauthor=S.+Ghoshauthor=H.-R.+Huangauthor=B.+M.+Bryantauthor=J.+M.+Lora&title=BET+bromodomain+inhibition+suppresses+TH17-mediated+pathology&doi=10.1084%2Fjem.20130376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition suppresses TH17-mediated pathology</span></div><div class="casAuthors">Mele, Deanna A.; Salmeron, Andres; Ghosh, Srimoyee; Huang, Hon-Ren; Bryant, Barbara M.; Lora, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">210</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2181-2190</span>CODEN:
                <span class="NLM_cas:coden">JEMEAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-1007</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Interleukin (IL) 17-producing T helper (TH17) cells have been selected through evolution for their ability to control fungal and bacterial infections.  It is also firmly established that their aberrant generation and activation results in autoimmune conditions.  Using a characterized potent and selective small mol. inhibitor, we show that the bromodomain and extra-terminal domain (BET) family of chromatin adaptors plays fundamental and selective roles in human and murine TH17 differentiation from naive CD4+ T cells, as well as in the activation of previously differentiated TH17 cells.  We provide evidence that BET controls TH17 differentiation in a bromodomain-dependent manner through a mechanism that includes the direct regulation of multiple effector TH17-assocd. cytokines, including IL17, IL21, and GMCSF.  We also demonstrate that BET family members Brd2 and Brd4 assoc. with the Il17 locus in TH17 cells, and that this assocn. requires bromodomains.  We recapitulate the crit. role of BET bromodomains in TH17 differentiation in vivo and show that therapeutic dosing of the BET inhibitor is efficacious in mouse models of autoimmunity.  Our results identify the BET family of proteins as a fundamental link between chromatin signaling and TH17 biol., and support the notion of BET inhibition as a point of therapeutic intervention in autoimmune conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBqVtggp2Fa7Vg90H21EOLACvtfcHk0lgHlji_-VV0cQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ygsLbF&md5=e6822f456cf42cdead9779afa92c05e0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1084%2Fjem.20130376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20130376%26sid%3Dliteratum%253Aachs%26aulast%3DMele%26aufirst%3DD.%2BA.%26aulast%3DSalmeron%26aufirst%3DA.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26atitle%3DBET%2520bromodomain%2520inhibition%2520suppresses%2520TH17-mediated%2520pathology%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2013%26volume%3D210%26spage%3D2181%26epage%3D2190%26doi%3D10.1084%2Fjem.20130376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">BET Inhibitor JQ1 Blocks Inflammation and Bone Destruction</span>. <i>J. Dent. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">657</span>â <span class="NLM_lpage">662</span>, <span class="refDoi">Â DOI: 10.1177/0022034514534261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1177%2F0022034514534261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=24799421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWhsL%252FI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2014&pages=657-662&author=S.+Mengauthor=L.+Zhangauthor=Y.+Tangauthor=Q.+Tuauthor=L.+Zhengauthor=L.+Yuauthor=D.+Murrayauthor=J.+Chengauthor=S.+H.+Kimauthor=X.+Zhouauthor=J.+Chen&title=BET+Inhibitor+JQ1+Blocks+Inflammation+and+Bone+Destruction&doi=10.1177%2F0022034514534261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitor JQ1 blocks inflammation and bone destruction</span></div><div class="casAuthors">Meng, S.; Zhang, L.; Tang, Y.; Tu, Q.; Zheng, L.; Yu, L.; Murray, D.; Cheng, J.; Kim, S. H.; Zhou, X.; Chen, J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Dental Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">657-662</span>CODEN:
                <span class="NLM_cas:coden">JDREAF</span>;
        ISSN:<span class="NLM_cas:issn">0022-0345</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">BET proteins are a group of epigenetic regulators controlling transcription through reading acetylated histone tails and recruiting transcription complexes.  They are considered as potential therapeutic targets in many distinct diseases.  A novel synthetic bromodomain and extraterminal domain (BET) inhibitor, JQ1, was proved to suppress oncogene transcription and inflammatory responses.  The present study was aimed to investigate the effects of JQ1 on inflammatory response and bone destruction in exptl. periodontitis.  We found that JQ1 significantly suppressed lipopolysaccharide (LPS)-stimulated inflammatory cytokine transcription, including interleukin (IL)-1Î², IL-6, and tumor necrosis factor alpha (TNF-Î±), as well as receptor activator of nuclear factor kappa- B ligand (RANKL)-induced osteoclast markers, such as c-Fos, nuclear factor of activated T-cells, cytoplasmic, calcineurin- dependent 1 (NFATc1), tartrate-resistant acid phosphatase (TRAP) and cathepsin K in vitro.  JQ1 also inhibited toll-like receptors 2/4 (TLR2/4) expression and nuclear factor kappa-light-chain-enhancer of activated B cells (NF-ÎºB) phosphorylation and nuclear translocation.  Chromatin immunopptn. and quant. polymerase chain reaction (ChIP-qPCR) revealed that JQ1 neutralized BRD4 enrichment at several gene promoter regions, including NF-ÎºB, TNF-Î±, c-Fos, and NFATc1.  In a murine periodontitis model, systemic administration of JQ1 significantly inhibited inflammatory cytokine expression in diseased gingival tissues.  Alveolar bone loss was alleviated in JQ1-treated mice because of reduced osteoclasts in periodontal tissues.  These unprecedented results suggest the BET inhibitor JQ1 as a prospective new approach for treating periodontitis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtMKKZI5MhYLVg90H21EOLACvtfcHk0lgHlji_-VV0cQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWhsL%252FI&md5=7e65e561c2141b78954deb6b779ed9b5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1177%2F0022034514534261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0022034514534261%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTu%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DMurray%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DBET%2520Inhibitor%2520JQ1%2520Blocks%2520Inflammation%2520and%2520Bone%2520Destruction%26jtitle%3DJ.%2520Dent.%2520Res.%26date%3D2014%26volume%3D93%26spage%3D657%26epage%3D662%26doi%3D10.1177%2F0022034514534261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarilina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivashkiv, L. B.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition suppresses transcriptional responses to cytokine-Jak-STAT signaling in a gene-specific manner in human monocytes</span>. <i>Eur. J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">287</span>â <span class="NLM_lpage">297</span>, <span class="refDoi">Â DOI: 10.1002/eji.201444862</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1002%2Feji.201444862" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=25345375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2jurvP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2015&pages=287-297&author=C.+H.+Chanauthor=C.+Fangauthor=Y.+Qiaoauthor=A.+Yarilinaauthor=R.+K.+Prinjhaauthor=L.+B.+Ivashkiv&title=BET+bromodomain+inhibition+suppresses+transcriptional+responses+to+cytokine-Jak-STAT+signaling+in+a+gene-specific+manner+in+human+monocytes&doi=10.1002%2Feji.201444862"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition suppresses transcriptional responses to cytokine-Jak-STAT signaling in a gene-specific manner in human monocytes</span></div><div class="casAuthors">Chan, Chun Hin; Fang, Celestia; Qiao, Yu; Yarilina, Anna; Prinjha, Rab K.; Ivashkiv, Lionel B.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">287-297</span>CODEN:
                <span class="NLM_cas:coden">EJIMAF</span>;
        ISSN:<span class="NLM_cas:issn">0014-2980</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Disruption of the interaction of bromo and extraterminal (BET) proteins with acetylated histones using small mol. inhibitors suppresses Myc-driven cancers and TLR-induced inflammation in mouse models.  The predominant mechanism of BET inhibitor action is to suppress BET-mediated recruitment of pos. transcription elongation factor b and, thus, transcription elongation.  We investigated the effects of BET inhibitor I-BET151 on transcriptional responses to TLR4 and TNF in primary human monocytes and also on responses to cytokines IFN-Î², IFN-Î³, IL-4, and IL-10, which activate the JAK-STAT signaling pathway and are important for monocyte polarization and inflammatory diseases.  I-BET151 suppressed TLR4- and TNF-induced IFN responses by diminishing both autocrine IFN-Î² expression and transcriptional responses to IFN-Î².  I-BET151 inhibited cytokine-induced transcription of STAT targets in a gene-specific manner without affecting STAT activation or recruitment.  This inhibition was independent of Myc or other upstream activators.  IFN-stimulated gene transcription is regulated primarily at the level of transcription initiation.  Accordingly, we found that I-BET151 suppressed the recruitment of transcriptional machinery to the CXCL10 promoter and an upstream enhancer.  Our findings suggest that BET inhibition reduces inflammation partially through suppressing cytokine activity and expands the understanding of the inhibitory and potentially selective immunosuppressive effects of inhibiting BET proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbE0f71lXpp7Vg90H21EOLACvtfcHk0liWXboAMbZ5PA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2jurvP&md5=efa081bc6c79db9233ae1a99ebf49945</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Feji.201444862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Feji.201444862%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DC.%2BH.%26aulast%3DFang%26aufirst%3DC.%26aulast%3DQiao%26aufirst%3DY.%26aulast%3DYarilina%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DIvashkiv%26aufirst%3DL.%2BB.%26atitle%3DBET%2520bromodomain%2520inhibition%2520suppresses%2520transcriptional%2520responses%2520to%2520cytokine-Jak-STAT%2520signaling%2520in%2520a%2520gene-specific%2520manner%2520in%2520human%2520monocytes%26jtitle%3DEur.%2520J.%2520Immunol.%26date%3D2015%26volume%3D45%26spage%3D287%26epage%3D297%26doi%3D10.1002%2Feji.201444862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nadeem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Sherbeeny, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoheir, K. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attia, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hosaini, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Sharary, S. D.</span></span> <span> </span><span class="NLM_article-title">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">248</span>â <span class="NLM_lpage">257</span>, <span class="refDoi">Â DOI: 10.1016/j.phrs.2015.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.phrs.2015.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=26149470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2015&pages=248-257&author=A.+Nadeemauthor=N.+O.+Al-Harbiauthor=M.+M.+Al-Harbiauthor=A.+M.+El-Sherbeenyauthor=S.+F.+Ahmadauthor=N.+Siddiquiauthor=M.+A.+Ansariauthor=K.+M.+A.+Zoheirauthor=S.+M.+Attiaauthor=K.+A.+Al-Hosainiauthor=S.+D.+Al-Sharary&title=Imiquimod-induced+psoriasis-like+skin+inflammation+is+suppressed+by+BET+bromodomain+inhibitor+in+mice+through+RORC%2FIL-17A+pathway+modulation&doi=10.1016%2Fj.phrs.2015.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span></div><div class="casAuthors">Nadeem, Ahmed; Al-Harbi, Naif O.; Al-Harbi, Mohamed M.; El-Sherbeeny, Ahmed M.; Ahmad, Sheikh F.; Siddiqui, Nahid; Ansari, Mushtaq A.; Zoheir, Khairy M. A.; Attia, Sabry M.; Al-Hosaini, Khaled A.; Al-Sharary, Shakir D.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">248-257</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Psoriasis is one of the most common skin disorders characterized by erythematous plaques that result from hyperproliferative keratinocytes and infiltration of inflammatory leukocytes into dermis and epidermis.  Recent studies suggest that IL-23/IL-17A/IL-22 cytokine axis plays an important role in the pathogenesis of psoriasis.  The small mol. bromodomain and extraterminal domain (BET) inhibitors, that disrupt interaction of BET proteins with acetylated histones have recently demonstrated efficacy in various models of inflammation through suppression of several pathways, one of them being synthesis of IL-17A/IL-22 which primarily depends on transcription factor, retinoic acid receptor-related orphan receptor C (RORC).  However, the efficacy and mechanistic aspect of a BET inhibitor in mouse model of skin inflammation has not been explored previously.  Therefore, this study investigated the role of BET inhibitor, JQ-1 in mouse model of psoriasis-like inflammation.  Mice were topically applied imiquimod (IMQ) to develop psoriasis-like inflammation on the shaved back and ear followed by assessment of skin inflammation (myeloperoxidase activity, ear thickness, and histopathol.), RORC and its signature cytokines (IL-17A/IL-22).  JQ-1 suppressed IMQ-induced skin inflammation as reflected by a decrease in ear thickness/myeloperoxidase activity, and RORC/IL-17A/IL-22 expression.  Addnl., a RORÎ±/Î³ agonist SR1078 was utilized to investigate the role of RORC in BET-mediated skin inflammation.  SR1078 reversed the protective effect of JQ-1 on skin inflammation at both histol. and mol. levels in the IMQ model.  The current study suggests that BET bromodomains are involved in psoriasis-like inflammation through induction of RORC/IL-17A pathway.  Therefore, inhibition of BET bromodomains may provide a new therapy against skin inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcOnC_sPUMSLVg90H21EOLACvtfcHk0liWXboAMbZ5PA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I&md5=3edb10526e30c18bce037dc0aca611a4</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DNadeem%26aufirst%3DA.%26aulast%3DAl-Harbi%26aufirst%3DN.%2BO.%26aulast%3DAl-Harbi%26aufirst%3DM.%2BM.%26aulast%3DEl-Sherbeeny%26aufirst%3DA.%2BM.%26aulast%3DAhmad%26aufirst%3DS.%2BF.%26aulast%3DSiddiqui%26aufirst%3DN.%26aulast%3DAnsari%26aufirst%3DM.%2BA.%26aulast%3DZoheir%26aufirst%3DK.%2BM.%2BA.%26aulast%3DAttia%26aufirst%3DS.%2BM.%26aulast%3DAl-Hosaini%26aufirst%3DK.%2BA.%26aulast%3DAl-Sharary%26aufirst%3DS.%2BD.%26atitle%3DImiquimod-induced%2520psoriasis-like%2520skin%2520inflammation%2520is%2520suppressed%2520by%2520BET%2520bromodomain%2520inhibitor%2520in%2520mice%2520through%2520RORC%252FIL-17A%2520pathway%2520modulation%26jtitle%3DPharmacol.%2520Res.%26date%3D2015%26volume%3D99%26spage%3D248%26epage%3D257%26doi%3D10.1016%2Fj.phrs.2015.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.-C.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomain-containing protein 4 (BRD4) benefits rheumatoid arthritis</span>. <i>Immunol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>166</i></span>,  <span class="NLM_fpage">103</span>â <span class="NLM_lpage">108</span>, <span class="refDoi">Â DOI: 10.1016/j.imlet.2015.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.imlet.2015.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=26093279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVKhsbbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2015&pages=103-108&author=Q.-G.+Zhangauthor=J.+Qianauthor=Y.-C.+Zhu&title=Targeting+bromodomain-containing+protein+4+%28BRD4%29+benefits+rheumatoid+arthritis&doi=10.1016%2Fj.imlet.2015.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomain-containing protein 4 (BRD4) benefits rheumatoid arthritis</span></div><div class="casAuthors">Zhang, Qing-gang; Qian, Jing; Zhu, Yu-chang</div><div class="citationInfo"><span class="NLM_cas:title">Immunology Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">103-108</span>CODEN:
                <span class="NLM_cas:coden">IMLED6</span>;
        ISSN:<span class="NLM_cas:issn">0165-2478</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We aimed to explore the effects of bromodomain-contg. protein 4 (BRD4) inhibition on tumor necrosis factor (TNF)-Î±-stimulated human rheumatoid arthritis fibroblast-like synoviocytes (RA-FLS) behavior and the therapeutic implications using BRD4 inhibitor JQ1 were explored in vivo.  The levels of interleukin (IL)-1Î², IL-6, IL-17 and IL-18 in cultural supernatants from TNFÎ±-stimulated RA-FLS were measured by ELISA.  RA-FLS migration and invasion in vitro were investigated using wound healing and Matrigel assay.  Expression of signaling pathway proteins was measured by Western blot.  The in vivo effects of BRD4 inhibitor JQ1 were elucidated using collagen-induced arthritis (CIA) mice.  We found BRD4 silencing reduced the secretion of IL-1Î², IL-6, IL-17 and IL-18 from TNFÎ±-stimulated human RA-FLS.  Downregulation of BRD4 inhibited FBS-induced migration and invasion of human RA-FLS.  BRD4 silencing decreased the phosphorylation of c-Jun and activation of NFÎºB in TNFÎ±-stimulated RA-FLS.  In vivo, BRD4 inhibitor JQ1 reduced the inflammatory response, autoantibody prodn. and joint damage of CIA model.  Our data suggest for the first time that BRD4 inhibition has anti-inflammatory property in RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-RTAy64CzFbVg90H21EOLACvtfcHk0liXGOubOnimuA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVKhsbbM&md5=395fce2cd419dd86aebd9fd93ce80c0f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.imlet.2015.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.imlet.2015.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DQ.-G.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DY.-C.%26atitle%3DTargeting%2520bromodomain-containing%2520protein%25204%2520%2528BRD4%2529%2520benefits%2520rheumatoid%2520arthritis%26jtitle%3DImmunol.%2520Lett.%26date%3D2015%26volume%3D166%26spage%3D103%26epage%3D108%26doi%3D10.1016%2Fj.imlet.2015.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grabiec, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ospelt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span> <span> </span><span class="NLM_article-title">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span>. <i>Ann. Rheum. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">422</span>â <span class="NLM_lpage">429</span>, <span class="refDoi">Â DOI: 10.1136/annrheumdis-2014-205809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1136%2Fannrheumdis-2014-205809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=25467295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2016&pages=422-429&author=K.+Kleinauthor=P.+A.+Kabalaauthor=A.+M.+Grabiecauthor=R.+E.+Gayauthor=C.+Kollingauthor=L.-L.+Linauthor=S.+Gayauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=C.+Ospeltauthor=K.+A.+Reedquist&title=The+bromodomain+protein+inhibitor+I-BET151+suppresses+expression+of+inflammatory+genes+and+matrix+degrading+enzymes+in+rheumatoid+arthritis+synovial+fibroblasts&doi=10.1136%2Fannrheumdis-2014-205809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span></div><div class="casAuthors">Klein, Kerstin; Kabala, Pawel A.; Grabiec, Aleksander M.; Gay, Renate E.; Kolling, Christoph; Lin, Lih-Ling; Gay, Steffen; Tak, Paul P.; Prinjha, Rab K.; Ospelt, Caroline; Reedquist, Kris A.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">422-429</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Objective To investigate the effects of BET bromodomain protein inhibition on inflammatory activation and functional properties of rheumatoid arthritis synovial fibroblasts (RASF).  The expression of the BET bromodomain proteins BRD2, BRD3 and BRD4 was analyzed in synovial tissue by immunohistochem.  RASF were stimulated with tumor necrosis factor (TNF)- Î±, interleukin (IL)-1Î² and toll-like receptor (TLR) ligands (Pam3, pIC and lipopolysaccharide (LPS)) in the presence or absence of the BET inhibitor I-BET151, or siRNA targeting BRD2, BRD3 and BRD4.  RASF expression of inflammatory mediators, including MMP1, MMP3, IL-6 and IL-8, was measured by q-PCR, q-PCR array and ELISA.  Cellular viability, apoptosis, proliferation and chemoattractive properties of RASF were investigated using MTT, cell apoptosis ELISA, BrdU-based proliferation and transwell migration assays.  BRD2, BRD3 and BRD4 proteins were detected in rheumatoid arthritis (RA) synovial tissue, expressed in both RASF and macrophages.  I-BET151 suppressed cytokine and TLR ligand-induced secretion of MMP1, MMP3, IL-6 and IL-8, and mRNA expression of more than 70% of genes induced by TNF- Î± and IL-1Î² .  Combined silencing of BRD2, BRD3 and BRD4 significantly reduced cytokine and TLR ligand-induced expression of a subset of gene products targeted by I-BET151, including MMP1, CXCL10 and CXCL11.  I-BET151 treatment of RASF reduced RASF proliferation, and the chemotactic potential for peripheral blood leukocytes of RASF conditioned medium.  Conclusions Inhibition of BET family proteins suppresses the inflammatory, matrix-degrading, proliferative and chemoattractive properties of RASF and suggests a therapeutic potential in the targeting of epigenetic reader proteins in RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpELw7WcQzbLLVg90H21EOLACvtfcHk0liXGOubOnimuA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO&md5=97f611195571a9772ff7668d06cdb335</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2014-205809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2014-205809%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DK.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DGrabiec%26aufirst%3DA.%2BM.%26aulast%3DGay%26aufirst%3DR.%2BE.%26aulast%3DKolling%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DL.-L.%26aulast%3DGay%26aufirst%3DS.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DOspelt%26aufirst%3DC.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26atitle%3DThe%2520bromodomain%2520protein%2520inhibitor%2520I-BET151%2520suppresses%2520expression%2520of%2520inflammatory%2520genes%2520and%2520matrix%2520degrading%2520enzymes%2520in%2520rheumatoid%2520arthritis%2520synovial%2520fibroblasts%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2016%26volume%3D75%26spage%3D422%26epage%3D429%26doi%3D10.1136%2Fannrheumdis-2014-205809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schilderink, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reginato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilbers, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jonge, W. J.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span>. <i>Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">66</span>â <span class="NLM_lpage">76</span>, <span class="refDoi">Â DOI: 10.1016/j.molimm.2016.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.molimm.2016.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=27710838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2016&pages=66-76&author=R.+Schilderinkauthor=M.+Bellauthor=E.+Reginatoauthor=C.+Pattenauthor=I.+Riojaauthor=F.+W.+Hilbersauthor=P.+A.+Kabalaauthor=K.+A.+Reedquistauthor=D.+F.+Toughauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=W.+J.+de+Jonge&title=BET+bromodomain+inhibition+reduces+maturation+and+enhances+tolerogenic+properties+of+human+and+mouse+dendritic+cells&doi=10.1016%2Fj.molimm.2016.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span></div><div class="casAuthors">Schilderink, Ronald; Bell, Matthew; Reginato, Eleonora; Patten, Chris; Rioja, Inmaculada; Hilbers, Francisca W.; Kabala, Pawel A.; Reedquist, Kris A.; Tough, David F.; Tak, Paul Peter; Prinjha, Rab K.; de Jonge, Wouter J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">66-76</span>CODEN:
                <span class="NLM_cas:coden">MOIMD5</span>;
        ISSN:<span class="NLM_cas:issn">0161-5890</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Transcription of inflammatory genes is tightly regulated by acetylation and deacetylation of histone tails.  An inhibitor of the acetylated-lysine reader bromodomain and extra-terminal domain (BET) proteins, I-BET151, is known to counteract the induction of expression of inflammatory genes in macrophages.  We have investigated the effects of I-BET151 on dendritic cell function, including expression of co-stimulatory mols. and cytokines, and capacity for T cell activation.  Treatment of mouse bone marrow derived dendritic cells (BMDC) and human monocyte derived DCs (mdDC) with I-BET151 reduced LPS-induced expression of co-stimulatory mols., as well as the prodn. of multiple cytokines and chemokines.  Most strikingly, secretion of IL-6, IL-12 and IL-10 was significantly reduced to 89.7%, 99.9% and 98.6% resp. of that produced by control cells.  I-BET151-treated mdDC showed a reduced ability to stimulate proliferation of autologous Revaxis-specific T cells.  Moreover, while I-BET151 treatment of BMDC did not affect their ability to polarize ovalbumin specific CD4+ CD62L+ naive T cells towards Th1, Th2, or Th17 phenotypes, an increase in Foxp3 expressing Tregs secreting higher IL-10 levels was obsd.  Suppression assays demonstrated that Tregs generated in response to I-BET151-treated BMDC displayed anti-proliferative capacity.  Finally, evidence that I-BET151 treatment can ameliorate inflammation in vivo in a T cell dependent colitis model is shown.  Overall, these results demonstrate marked effects of BET inhibition on DC maturation, reducing their capacity for pro-inflammatory cytokine secretion and T cell activation and enhancing the potential of DC to induce Foxp3 expressing Treg with suppressive properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIuoARRI77BrVg90H21EOLACvtfcHk0liXGOubOnimuA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I&md5=6a0da0ba320112023cb09df03c4802dc</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.molimm.2016.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molimm.2016.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DSchilderink%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DM.%26aulast%3DReginato%26aufirst%3DE.%26aulast%3DPatten%26aufirst%3DC.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DHilbers%26aufirst%3DF.%2BW.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3Dde%2BJonge%26aufirst%3DW.%2BJ.%26atitle%3DBET%2520bromodomain%2520inhibition%2520reduces%2520maturation%2520and%2520enhances%2520tolerogenic%2520properties%2520of%2520human%2520and%2520mouse%2520dendritic%2520cells%26jtitle%3DMol.%2520Immunol.%26date%3D2016%26volume%3D79%26spage%3D66%26epage%3D76%26doi%3D10.1016%2Fj.molimm.2016.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">573</span>â <span class="NLM_lpage">584</span>, <span class="refDoi">Â DOI: 10.2217/epi-2016-0144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.2217%2Fepi-2016-0144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=27925476" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2016&pages=573-584&author=D.+F.+Toughauthor=R.+K.+Prinjha&title=Immune+disease-associated+variants+in+gene+enhancers+point+to+BET+epigenetic+mechanisms+for+therapeutic+intervention&doi=10.2217%2Fepi-2016-0144"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.2217%2Fepi-2016-0144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi-2016-0144%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DImmune%2520disease-associated%2520variants%2520in%2520gene%2520enhancers%2520point%2520to%2520BET%2520epigenetic%2520mechanisms%2520for%2520therapeutic%2520intervention%26jtitle%3DEpigenomics%26date%3D2016%26volume%3D9%26spage%3D573%26epage%3D584%26doi%3D10.2217%2Fepi-2016-0144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P. M.</span></span> <span> </span><span class="NLM_article-title">BET inhibitors: a novel epigenetic approach</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1776</span>â <span class="NLM_lpage">1787</span>, <span class="refDoi">Â DOI: 10.1093/annonc/mdx157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1093%2Fannonc%2Fmdx157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=28838216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=1776-1787&author=D.+B.+Doroshowauthor=J.+P.+Ederauthor=P.+M.+LoRusso&title=BET+inhibitors%3A+a+novel+epigenetic+approach&doi=10.1093%2Fannonc%2Fmdx157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors: a novel epigenetic approach</span></div><div class="casAuthors">Doroshow D B; Eder J P; LoRusso P M</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1776-1787</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetics has been defined as 'the structural adaptation of chromosomal regions so as to register, signal or perpetuate altered activity states.' Currently, several classes of anticancer drugs function at the epigenetic level, including inhibitors of DNA methyltransferase, histone deacetylase (HDAC), lysine-specific demethylase 1, zeste homolog 2, and bromodomain and extra-terminal motif (BET) proteins.BET proteins have multiple functions, including the initiation and elongation of transcription and cell cycle regulation.  In recent years, inhibitors of BET proteins have been developed as anticancer agents.  These inhibitors exhibit selectivity for tumor cells by preferentially binding to superenhancers, noncoding regions of DNA critical for the transcription of genes that determine a cell's identity.  Preclinical research on BET inhibitors has identified them as a potential means of targeting MYC.Early clinical trials with BET inhibitors have had mixed results, with few responses in both hematologic and solid tumors that tend to be short-lived.  Toxicities have included severe, thrombocytopenia, fatigue, nausea, vomiting, and diarrhea; GI side-effects, fatigue, and low-grade dysgeusia have limited compliance.  However, preclinical data suggest that BET inhibitors may have a promising future in combination with other agents.  They appear to be able to overcome resistance to targeted agents and have strong synergy with immune checkpoint inhibitors as well as with multiple epigenetic agents, particularly HDAC inhibitors.  In many instances, BET and HDAC inhibitors were synergistic at reduced doses, suggesting a potential means of avoiding the overlapping toxicities of the two drug classes.BET inhibitors provide a novel approach to epigenetic anticancer therapy.  However, to date they appear to have limited efficacy as single agents.  A focus on BET inhibitors in combination with other drugs such as targeted and/or as other epigenetic agents is warranted, due to limited monotherapy activity, including pharmacodynamic correlatives differential activity amongst select drug combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSttah97bhI2UPI-fCgwosgfW6udTcc2ean50xgK-JMlrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D&md5=580ac90cf48f18db8313471ed61af173</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdx157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdx157%26sid%3Dliteratum%253Aachs%26aulast%3DDoroshow%26aufirst%3DD.%2BB.%26aulast%3DEder%26aufirst%3DJ.%2BP.%26aulast%3DLoRusso%26aufirst%3DP.%2BM.%26atitle%3DBET%2520inhibitors%253A%2520a%2520novel%2520epigenetic%2520approach%26jtitle%3DAnn.%2520Oncol.%26date%3D2017%26volume%3D28%26spage%3D1776%26epage%3D1787%26doi%3D10.1093%2Fannonc%2Fmdx157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery Targeting Bromodomain-Containing Protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>â <span class="NLM_lpage">4558</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+Discovery+Targeting+Bromodomain-Containing+Protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0lh9wX1h72Qb7A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520Discovery%2520Targeting%2520Bromodomain-Containing%2520Protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Postel-Vinay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbschleb, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodcock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewerton, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poelman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleton, M.</span></span> <span> </span><span class="NLM_article-title">First-in-human phase I study of the bromodomain and extraterminal motif inhibitor BAY 1238097: emerging pharmacokinetic/pharmacodynamic relationship and early termination due to unexpected toxicity</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">103</span>â <span class="NLM_lpage">110</span>, <span class="refDoi">Â DOI: 10.1016/j.ejca.2018.12.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.ejca.2018.12.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=30711772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslCltLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2019&pages=103-110&author=S.+Postel-Vinayauthor=K.+Herbschlebauthor=C.+Massardauthor=V.+Woodcockauthor=J.-C.+Soriaauthor=A.+O.+Walterauthor=F.+Ewertonauthor=M.+Poelmanauthor=N.+Bensonauthor=M.+Ockerauthor=G.+Wilkinsonauthor=M.+Middleton&title=First-in-human+phase+I+study+of+the+bromodomain+and+extraterminal+motif+inhibitor+BAY+1238097%3A+emerging+pharmacokinetic%2Fpharmacodynamic+relationship+and+early+termination+due+to+unexpected+toxicity&doi=10.1016%2Fj.ejca.2018.12.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">First-in-human phase I study of the bromodomain and extraterminal motif inhibitor BAY 1238097: emerging pharmacokinetic/pharmacodynamic relationship and early termination due to unexpected toxicity</span></div><div class="casAuthors">Postel-Vinay, Sophie; Herbschleb, Karin; Massard, Christophe; Woodcock, Victoria; Soria, Jean-Charles; Walter, Annette O.; Ewerton, Flavio; Poelman, Martine; Benson, Neil; Ocker, Matthias; Wilkinson, Gary; Middleton, Mark</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">103-110</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Bromodomain and extraterminal motif (BET) protein inhibition is a promising cancer treatment strategy, notably for targeting MYC- or BRD4-driven diseases.  A first-in-human study investigated the safety, pharmacokinetics, max. tolerated dose and recommended phase II dose of the BET inhibitor BAY 1238097 in patients with advanced malignancies.In this phase I, open-label, non-randomised, multicentre study, patients with cytol. or histol. confirmed advanced refractory malignancies received oral BAY 1238097 twice weekly in 21-day cycles using an adaptive dose-escalation design at a starting dose of 10 mg/wk.  Model-based dose-response anal. was performed to guide dose escalation.  Safety, pharmacokinetics, pharmacodynamics and tumor response were evaluated.Eight patients were enrolled at three dose levels (10 mg/wk, n = 3; 40 mg/wk, n = 3; 80 mg/wk, n = 2).  Both patients receiving 80 mg/wk had dose-limiting toxicities (DLTs) (grade 3 vomiting, grade 3 headache and grade 2/3 back pain).  The most common adverse events were nausea, vomiting, headache, back pain and fatigue.  Pharmacokinetic anal. indicated a linear dose response with increasing dose.  Two patients displayed prolonged stable disease; no responses were obsd.  Biomarker evaluation of MYC and HEXIM1 expression demonstrated an emerging pharmacokinetic/pharmacodynamic relationship, with a trend towards decreased MYC and increased HEXIM1 expression in response to treatment.The study was prematurely terminated because of the occurrence of DLTs at a dose below targeted drug exposure.  Pharmacokinetic modeling indicated that an alternate dosing schedule whereby DLTs could be avoided while reaching efficacious exposure was not feasible.  Registration no.: NCT02369029.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9snvNdTKTtrVg90H21EOLACvtfcHk0lgOnahrHZgODg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslCltLo%253D&md5=c5969b7827810fd1146338b84c8f8621</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2018.12.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2018.12.020%26sid%3Dliteratum%253Aachs%26aulast%3DPostel-Vinay%26aufirst%3DS.%26aulast%3DHerbschleb%26aufirst%3DK.%26aulast%3DMassard%26aufirst%3DC.%26aulast%3DWoodcock%26aufirst%3DV.%26aulast%3DSoria%26aufirst%3DJ.-C.%26aulast%3DWalter%26aufirst%3DA.%2BO.%26aulast%3DEwerton%26aufirst%3DF.%26aulast%3DPoelman%26aufirst%3DM.%26aulast%3DBenson%26aufirst%3DN.%26aulast%3DOcker%26aufirst%3DM.%26aulast%3DWilkinson%26aufirst%3DG.%26aulast%3DMiddleton%26aufirst%3DM.%26atitle%3DFirst-in-human%2520phase%2520I%2520study%2520of%2520the%2520bromodomain%2520and%2520extraterminal%2520motif%2520inhibitor%2520BAY%25201238097%253A%2520emerging%2520pharmacokinetic%252Fpharmacodynamic%2520relationship%2520and%2520early%2520termination%2520due%2520to%2520unexpected%2520toxicity%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2019%26volume%3D109%26spage%3D103%26epage%3D110%26doi%3D10.1016%2Fj.ejca.2018.12.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Odenike, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolff, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borthakur, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldoss, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzieri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prebet, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freise, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonas, B. A.</span></span> <span> </span><span class="NLM_article-title">Results from the first-in-human study of mivebresib (ABBV-075), a pan-inhibitor of bromodomain and extra terminal proteins, in patients with relapsed/refractory acute myeloid leukemia</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">7030</span>â <span class="NLM_lpage">7030</span>, <span class="refDoi">Â DOI: 10.1200/JCO.2019.37.15_suppl.7030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1200%2FJCO.2019.37.15_suppl.7030" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=7030-7030&author=O.+Odenikeauthor=J.+E.+Wolffauthor=G.+Borthakurauthor=I.+T.+Aldossauthor=D.+Rizzieriauthor=T.+Prebetauthor=B.+Huauthor=M.+Dinhauthor=X.+Chenauthor=D.+Modiauthor=K.+J.+Freiseauthor=B.+A.+Jonas&title=Results+from+the+first-in-human+study+of+mivebresib+%28ABBV-075%29%2C+a+pan-inhibitor+of+bromodomain+and+extra+terminal+proteins%2C+in+patients+with+relapsed%2Frefractory+acute+myeloid+leukemia&doi=10.1200%2FJCO.2019.37.15_suppl.7030"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1200%2FJCO.2019.37.15_suppl.7030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2019.37.15_suppl.7030%26sid%3Dliteratum%253Aachs%26aulast%3DOdenike%26aufirst%3DO.%26aulast%3DWolff%26aufirst%3DJ.%2BE.%26aulast%3DBorthakur%26aufirst%3DG.%26aulast%3DAldoss%26aufirst%3DI.%2BT.%26aulast%3DRizzieri%26aufirst%3DD.%26aulast%3DPrebet%26aufirst%3DT.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DDinh%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DModi%26aufirst%3DD.%26aulast%3DFreise%26aufirst%3DK.%2BJ.%26aulast%3DJonas%26aufirst%3DB.%2BA.%26atitle%3DResults%2520from%2520the%2520first-in-human%2520study%2520of%2520mivebresib%2520%2528ABBV-075%2529%252C%2520a%2520pan-inhibitor%2520of%2520bromodomain%2520and%2520extra%2520terminal%2520proteins%252C%2520in%2520patients%2520with%2520relapsed%252Frefractory%2520acute%2520myeloid%2520leukemia%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D37%26spage%3D7030%26epage%3D7030%26doi%3D10.1200%2FJCO.2019.37.15_suppl.7030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">19754</span>â <span class="NLM_lpage">19759</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+M%C3%BCllerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0lgOnahrHZgODg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Shiao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">New Synthetic Routes to Triazolo-benzodiazepine Analogues: Expanding the Scope of the Bump-and-Hole Approach for Selective Bromo and Extra-Terminal (BET) Bromodomain Inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1492</span>â <span class="NLM_lpage">1500</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01135</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01135" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyksbnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1492-1500&author=M.+G.+J.+Baudauthor=E.+Lin-Shiaoauthor=M.+Zengerleauthor=C.+Tallantauthor=A.+Ciulli&title=New+Synthetic+Routes+to+Triazolo-benzodiazepine+Analogues%3A+Expanding+the+Scope+of+the+Bump-and-Hole+Approach+for+Selective+Bromo+and+Extra-Terminal+%28BET%29+Bromodomain+Inhibition&doi=10.1021%2Facs.jmedchem.5b01135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">New Synthetic Routes to Triazolo-benzodiazepine Analogues: Expanding the Scope of the Bump-and-Hole Approach for Selective Bromo and Extra-Terminal (BET) Bromodomain Inhibition</span></div><div class="casAuthors">Baud, Matthias G. J.; Lin-Shiao, Enrique; Zengerle, Michael; Tallant, Cynthia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1492-1500</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe new synthetic routes developed toward a range of substituted analogs of bromo and extra-terminal (BET) bromodomain inhibitors I-BET762/JQ1 based on the triazolo-benzodiazepine scaffold.  These new routes allow for the derivatization of the methoxyphenyl and chlorophenyl rings, in addn. to the diazepine ternary center and the side chain methylene moiety.  Substitution at the level of the side chain methylene afforded compds. targeting specifically and potently engineered BET bromodomains designed as part of a bump and hole approach.  We further demonstrate that marked selectivity for the second over the first bromodomain can be achieved with an indole deriv. that exploits differential interaction with an aspartate/histidine conservative substitution on the BC loop of BET bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUpYVun8F94bVg90H21EOLACvtfcHk0ljQmct3_75o2g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyksbnE&md5=65dd39e3f7e0bc52169ffb1466218c43</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01135%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DLin-Shiao%26aufirst%3DE.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DNew%2520Synthetic%2520Routes%2520to%2520Triazolo-benzodiazepine%2520Analogues%253A%2520Expanding%2520the%2520Scope%2520of%2520the%2520Bump-and-Hole%2520Approach%2520for%2520Selective%2520Bromo%2520and%2520Extra-Terminal%2520%2528BET%2529%2520Bromodomain%2520Inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1492%26epage%3D1500%26doi%3D10.1021%2Facs.jmedchem.5b01135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, A. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4317</span>â <span class="NLM_lpage">4334</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4317-4334&author=R.+P.+Lawauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.-W.+Chungauthor=E.+H.+Demontauthor=L.+J.+Gordonauthor=M.+Lindonauthor=R.+K.+Prinjhaauthor=A.+J.+B.+Watsonauthor=D.+J.+Hirst&title=Discovery+of+Tetrahydroquinoxalines+as+Bromodomain+and+Extra-Terminal+Domain+%28BET%29+Inhibitors+with+Selectivity+for+the+Second+Bromodomain&doi=10.1021%2Facs.jmedchem.7b01666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Law, Robert P.; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Gordon, Laurie J.; Lindon, Matthew; Prinjha, Rab K.; Watson, Allan J. B.; Hirst, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4317-4334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins bind acetylated lysine residues on histone proteins.  The four BET bromodomains-BRD2, BRD3, BRD4, and BRDT-each contain two bromodomain modules.  BET bromodomain inhibition is a potential therapy for various cancers and immunoinflammatory diseases, but few reported inhibitors show selectivity within the BET family.  Inhibitors with selectivity for the first or second bromodomain are desired to aid investigation of the biol. function of these domains.  Focused library screening identified a series of tetrahydroquinoxalines with selectivity for the second bromodomains of the BET family (BD2).  Structure-guided optimization of the template improved potency, selectivity, and physicochem. properties, culminating in potent BET inhibitors with BD2 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobBKYwu7NyTbVg90H21EOLACvtfcHk0ljQmct3_75o2g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D&md5=078db91b35feaf1958fa8c33d0544387</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01666%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BP.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWatson%26aufirst%3DA.%2BJ.%2BB.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520Tetrahydroquinoxalines%2520as%2520Bromodomain%2520and%2520Extra-Terminal%2520Domain%2520%2528BET%2529%2520Inhibitors%2520with%2520Selectivity%2520for%2520the%2520Second%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4317%26epage%3D4334%26doi%3D10.1021%2Facs.jmedchem.7b01666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gesner, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span> <span> </span><span class="NLM_article-title">RVX-297- a novel BD2 selective inhibitor of BET bromodomains</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>477</i></span>,  <span class="NLM_fpage">62</span>â <span class="NLM_lpage">67</span>, <span class="refDoi">Â DOI: 10.1016/j.bbrc.2016.06.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.bbrc.2016.06.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=27282480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC28Xpslajtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2016&pages=62-67&author=O.+A.+Kharenkoauthor=E.+M.+Gesnerauthor=R.+G.+Patelauthor=K.+Norekauthor=A.+Whiteauthor=E.+Fontanoauthor=R.+K.+Sutoauthor=P.+R.+Youngauthor=K.+G.+McLureauthor=H.+C.+Hansen&title=RVX-297-+a+novel+BD2+selective+inhibitor+of+BET+bromodomains&doi=10.1016%2Fj.bbrc.2016.06.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-297- a novel BD2 selective inhibitor of BET bromodomains</span></div><div class="casAuthors">Kharenko, Olesya A.; Gesner, Emily M.; Patel, Reena G.; Norek, Karen; White, Andre; Fontano, Eric; Suto, Robert K.; Young, Peter R.; McLure, Kevin G.; Hansen, Henrik C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">62-67</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic readers that specifically bind to the acetyl lysine residues of histones and transcription factors.  Small mol. BET bromodomain inhibitors can disrupt this interaction which leads to potential modulation of several disease states.  Here we describe the binding properties of a novel BET inhibitor RVX-297 that is structurally related to the clin. compd. RVX-208, currently undergoing phase III clin. trials for the treatment of cardiovascular diseases, but is distinctly different in its biol. and pharmacokinetic profiles.  We report that RVX-297 preferentially binds to the BD2 domains of the BET bromodomain and Extra Terminal (BET) family of protein.  We demonstrate the differential binding modes of RVX-297 in BD1 and BD2 domains of BRD4 and BRD2 using X-ray crystallog., and describe the structural differences driving the BD2 selective binding of RVX-297.  The isothermal titrn. calorimetry (ITC) data illustrate the related differential thermodn. of binding of RVX-297 to single as well as dual BET bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDLUg96I7HObVg90H21EOLACvtfcHk0ljHdmtMnOk6Lw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xpslajtrw%253D&md5=a3d064e2245bf14ae525eac2490808cb</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2016.06.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2016.06.021%26sid%3Dliteratum%253Aachs%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DPatel%26aufirst%3DR.%2BG.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DFontano%26aufirst%3DE.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26atitle%3DRVX-297-%2520a%2520novel%2520BD2%2520selective%2520inhibitor%2520of%2520BET%2520bromodomains%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2016%26volume%3D477%26spage%3D62%26epage%3D67%26doi%3D10.1016%2Fj.bbrc.2016.06.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jahagirdar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marusic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendele, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenoy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulikowski, E.</span></span> <span> </span><span class="NLM_article-title">RVX-297, a BET Bromodomain Inhibitor, Has Therapeutic Effects in Preclinical Models of Acute Inflammation and Autoimmune Disease</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">694</span>â <span class="NLM_lpage">706</span>, <span class="refDoi">Â DOI: 10.1124/mol.117.110379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1124%2Fmol.117.110379" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=28974538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGgurbK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2017&pages=694-706&author=R.+Jahagirdarauthor=S.+Attwellauthor=S.+Marusicauthor=A.+Bendeleauthor=N.+Shenoyauthor=K.+G.+McLureauthor=D.+Gilhamauthor=K.+Norekauthor=H.+C.+Hansenauthor=R.+Yuauthor=J.+Tobinauthor=G.+S.+Wagnerauthor=P.+R.+Youngauthor=N.+C.+W.+Wongauthor=E.+Kulikowski&title=RVX-297%2C+a+BET+Bromodomain+Inhibitor%2C+Has+Therapeutic+Effects+in+Preclinical+Models+of+Acute+Inflammation+and+Autoimmune+Disease&doi=10.1124%2Fmol.117.110379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-297, a BET bromodomain inhibitor, has therapeutic effects in preclinical models of acute inflammation and autoimmune disease</span></div><div class="casAuthors">Jahagirdar, Ravi; Attwell, Sarah; Marusic, Suzana; Bendele, Alison; Shenoy, Narmada; McLure, Kevin G.; Gilham, Dean; Norek, Karen; Hansen, Henrik C.; Yu, Raymond; Tobin, Jennifer; Wagner, Gregory S.; Young, Peter R.; Wong, Norman C. W.; Kulikowski, Ewelina</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">694-706</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Bromodomain (BD) and extra-terminal domain contg. proteins (BET) are chromatin adapters that bind acetylated histone marks via two tandem BDs, BD1 and BD2, to regulate gene transcription.  BET proteins are involved in transcriptional reprogramming in response to inflammatory stimuli.  BET BD inhibitors (BETis) that are nonselective for BD1 or BD2 have recognized anti-inflammatory properties in vitro and counter pathol. in models of inflammation or autoimmune disease.  Although both BD1 and BD2 bind acetylated histone residues, they may independently regulate the expression of BET-sensitive genes.  Here we characterized the ability of RVX-297, a novel orally active BETi with selectivity for BD2, to modulate inflammatory processes in vitro, in vivo, and ex vivo.  RVX-297 suppressed inflammatory gene expression in multiple immune cell types in culture.  Mechanistically, RVX-297 displaced BET proteins from the promoters of sensitive genes and disrupted recruitment of active RNA polymerase II, a property shared with pan-BETis that nonselectively bind BET BDs.  In the lipopolysaccharide model of inflammation, RVX-297 reduced proinflammatory mediators assessed in splenic gene expression and serum proteins.  RVX-297 also countered pathol. in three rodent models of polyarthritis: rat and mouse collagen-induced arthritis, and mouse collagen antibody-induced arthritis.  Further, RVX-297 prevented murine exptl. autoimmune encephalomyelitis (a model of human multiple sclerosis) disease development when administered prophylactically and reduced hallmarks of pathol. when administered therapeutically.  We show for the first time that a BD2-selective BETi maintains anti-inflammatory properties and is effective in preclin. models of acute inflammation and autoimmunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7r1JpLemFA7Vg90H21EOLACvtfcHk0ljHdmtMnOk6Lw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGgurbK&md5=3f1a1b86b17315c413dd1f13d3a7736a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1124%2Fmol.117.110379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.117.110379%26sid%3Dliteratum%253Aachs%26aulast%3DJahagirdar%26aufirst%3DR.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DMarusic%26aufirst%3DS.%26aulast%3DBendele%26aufirst%3DA.%26aulast%3DShenoy%26aufirst%3DN.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DGilham%26aufirst%3DD.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DYu%26aufirst%3DR.%26aulast%3DTobin%26aufirst%3DJ.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DKulikowski%26aufirst%3DE.%26atitle%3DRVX-297%252C%2520a%2520BET%2520Bromodomain%2520Inhibitor%252C%2520Has%2520Therapeutic%2520Effects%2520in%2520Preclinical%2520Models%2520of%2520Acute%2520Inflammation%2520and%2520Autoimmune%2520Disease%26jtitle%3DMol.%2520Pharmacol.%26date%3D2017%26volume%3D92%26spage%3D694%26epage%3D706%26doi%3D10.1124%2Fmol.117.110379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, S. A. J.</span>; <span class="NLM_string-name">Atkinson, S. J.</span>; <span class="NLM_string-name">Aylott, H.</span>; <span class="NLM_string-name">Bamborough, P.</span>; <span class="NLM_string-name">Chung, C.-w.</span>; <span class="NLM_string-name">Copley, R. C. B.</span>; <span class="NLM_string-name">Gordon, L. J.</span></span> ; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span>; <span class="NLM_string-name">Gray, J. R. J.</span>; <span class="NLM_string-name">Harrison, L. A.</span>; <span class="NLM_string-name">Hayhow, T. G.</span>; <span class="NLM_string-name">Messenger, C.</span>; <span class="NLM_string-name">Michon, A.-M.</span>; <span class="NLM_string-name">Mitchell, D. J.</span>; <span class="NLM_string-name">Preston, A.</span>; <span class="NLM_string-name">Prinjha, R. K.</span>; <span class="NLM_string-name">Rioja, I.</span>; <span class="NLM_string-name">Taylor, S.</span>; <span class="NLM_string-name">Lindon, M.</span>; <span class="NLM_string-name">Wall, I.</span>; <span class="NLM_string-name">Watson, R. R.</span>; <span class="NLM_string-name">Woolven, J. M.</span>; <span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">The Optimisation of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor</span>,  <i>J . Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00796</span> .</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=S.+A.+J.+Seal&author=S.+J.+Atkinson&author=H.+Aylott&author=P.+Bamborough&author=C.-w.+Chung&author=R.+C.+B.+Copley&author=L.+J.+Gordonauthor=P.+Grandi&author=J.+R.+J.+Gray&author=L.+A.+Harrison&author=T.+G.+Hayhow&author=C.+Messenger&author=A.-M.+Michon&author=D.+J.+Mitchell&author=A.+Preston&author=R.+K.+Prinjha&author=I.+Rioja&author=S.+Taylor&author=M.+Lindon&author=I.+Wall&author=R.+R.+Watson&author=J.+M.+Woolven&author=E.+H.+Demont&title=The+Optimisation+of+a+Novel%2C+Weak+Bromo+and+Extra+Terminal+Domain+%28BET%29+Bromodomain+Fragment+Ligand+to+a+Potent+and+Selective+Second+Bromodomain+%28BD2%29+Inhibitor&doi=10.1021%2Facs.jmedchem.0c00796"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00796%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DS.%2BA.%2BJ.%26aulast%3DGrandi%26aufirst%3DP.%26atitle%3DThe%2520Optimisation%2520of%2520a%2520Novel%252C%2520Weak%2520Bromo%2520and%2520Extra%2520Terminal%2520Domain%2520%2528BET%2529%2520Bromodomain%2520Fragment%2520Ligand%2520to%2520a%2520Potent%2520and%2520Selective%2520Second%2520Bromodomain%2520%2528BD2%2529%2520Inhibitor%26jtitle%3DJ%2520.%2520Med.%2520Chem.%26date%3D2020%26doi%3D10.1021%2Facs.jmedchem.0c00796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gesner, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujikawa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campeau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasiak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an Inducer of ApoA-I in Humans, Is a BET Bromodomain Antagonist</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e83190</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0083190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1371%2Fjournal.pone.0083190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=24391744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=K.+G.+McLureauthor=E.+M.+Gesnerauthor=L.+Tsujikawaauthor=O.+A.+Kharenkoauthor=S.+Attwellauthor=E.+Campeauauthor=S.+Wasiakauthor=A.+Steinauthor=A.+Whiteauthor=E.+Fontanoauthor=R.+K.+Sutoauthor=N.+C.+W.+Wongauthor=G.+S.+Wagnerauthor=H.+C.+Hansenauthor=P.+R.+Young&title=RVX-208%2C+an+Inducer+of+ApoA-I+in+Humans%2C+Is+a+BET+Bromodomain+Antagonist&doi=10.1371%2Fjournal.pone.0083190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inducer of ApoA-I in humans, is a BET bromodomain antagonist</span></div><div class="casAuthors">McLure, Kevin G.; Gesner, Emily M.; Tsujikawa, Laura; Kharenko, Olesya A.; Attwell, Sarah; Campeau, Eric; Wasiak, Sylwia; Stein, Adam; White, Andre; Fontano, Eric; Suto, Robert K.; Wong, Norman C. W.; Wagner, Gregory S.; Hansen, Henrik C.; Young, Peter R.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e83190/1-e83190/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Increased synthesis of Apolipoprotein A-I (ApoA-I) and HDL is believed to provide a new approach to treating atherosclerosis through the stimulation of reverse cholesterol transport.  RVX-208 increases the prodn. of ApoA-I in hepatocytes in vitro and in vivo in monkeys and humans, which results in increased HDL-C, but the mol. target was not previously reported.  Using binding assays and X-ray crystallog., we now show that RVX-208 selectively binds to bromodomains of the BET (Bromodomain and Extra Terminal) family, competing for a site bound by the endogenous ligand, acetylated lysine and that this accounts for its pharmacol. activity. siRNA expts. further suggest that induction of ApoA-I mRNA is mediated by BET family member BRD4.  These data indicate that RVX-208 increases ApoA-I prodn. through an epigenetic mechanism and suggests that BET inhibition may be a promising new approach to the treatment of atherosclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLHPlAY0x6NLVg90H21EOLACvtfcHk0lj54hoGJkT59A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D&md5=519ef7ed6d06f97ef8a1359832c11d92</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0083190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0083190%26sid%3Dliteratum%253Aachs%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DTsujikawa%26aufirst%3DL.%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DCampeau%26aufirst%3DE.%26aulast%3DWasiak%26aufirst%3DS.%26aulast%3DStein%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DFontano%26aufirst%3DE.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26atitle%3DRVX-208%252C%2520an%2520Inducer%2520of%2520ApoA-I%2520in%2520Humans%252C%2520Is%2520a%2520BET%2520Bromodomain%2520Antagonist%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0083190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantena, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnik, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolette, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigelow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>578</i></span>,  <span class="NLM_fpage">306</span>â <span class="NLM_lpage">310</span>, <span class="refDoi">Â DOI: 10.1038/s41586-020-1930-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1038%2Fs41586-020-1930-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=31969702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=578&publication_year=2020&pages=306-310&author=E.+J.+Faivreauthor=K.+F.+McDanielauthor=D.+H.+Albertauthor=S.+R.+Mantenaauthor=J.+P.+Plotnikauthor=D.+Wilcoxauthor=L.+Zhangauthor=M.+H.+Buiauthor=G.+S.+Sheppardauthor=L.+Wangauthor=V.+Sehgalauthor=X.+Linauthor=X.+Huangauthor=X.+Luauthor=T.+Uzielauthor=P.+Hesslerauthor=L.+T.+Lamauthor=R.+J.+Bellinauthor=G.+Mehtaauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=C.+Sunauthor=S.+C.+Panchalauthor=J.+J.+Nicoletteauthor=S.+L.+Fosseyauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=L.+Bigelowauthor=M.+Torrentauthor=S.+H.+Rosenbergauthor=W.+M.+Katiauthor=Y.+Shen&title=Selective+inhibition+of+the+BD2+bromodomain+of+BET+proteins+in+prostate+cancer&doi=10.1038%2Fs41586-020-1930-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span></div><div class="casAuthors">Faivre, Emily J.; McDaniel, Keith F.; Albert, Daniel H.; Mantena, Srinivasa R.; Plotnik, Joshua P.; Wilcox, Denise; Zhang, Lu; Bui, Mai H.; Sheppard, George S.; Wang, Le; Sehgal, Vasudha; Lin, Xiaoyu; Huang, Xiaoli; Lu, Xin; Uziel, Tamar; Hessler, Paul; Lam, Lloyd T.; Bellin, Richard J.; Mehta, Gaurav; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa A.; Sun, Chaohong; Panchal, Sanjay C.; Nicolette, John J.; Fossey, Stacey L.; Park, Chang H.; Longenecker, Kenton; Bigelow, Lance; Torrent, Maricel; Rosenberg, Saul H.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">578</span>
        (<span class="NLM_cas:issue">7794</span>),
    <span class="NLM_cas:pages">306-310</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Proteins of the bromodomain and extra-terminal (BET) domain family are epigenetic readers that bind acetylated histones through their bromodomains to regulate gene transcription.  Dual-bromodomain BET inhibitors (DbBi) that bind with similar affinities to the first (BD1) and second (BD2) bromodomains of BRD2, BRD3, BRD4 and BRDt have displayed modest clin. activity in monotherapy cancer trials.  A reduced no. of thrombocytes in the blood (thrombocytopenia) as well as symptoms of gastrointestinal toxicity are dose-limiting adverse events for some types of DbBi1-5.  Given that similar haematol. and gastrointestinal defects were obsd. after genetic silencing of Brd4 in mice6, the platelet and gastrointestinal toxicities may represent on-target activities assocd. with BET inhibition.  The two individual bromodomains in BET family proteins may have distinct functions7-9 and different cellular phenotypes after pharmacol. inhibition of one or both bromodomains have been reported10,11, suggesting that selectively targeting one of the bromodomains may result in a different efficacy and tolerability profile compared with DbBi.  Available compds. that are selective to individual domains lack sufficient potency and the pharmacokinetics properties that are required for in vivo efficacy and tolerability assessment10-13.  Here we carried out a medicinal chem. campaign that led to the discovery of ABBV-744, a highly potent and selective inhibitor of the BD2 domain of BET family proteins with drug-like properties.  In contrast to the broad range of cell growth inhibition induced by DbBi, the antiproliferative activity of ABBV-744 was largely, but not exclusively, restricted to cell lines of acute myeloid leukemia and prostate cancer that expressed the full-length androgen receptor (AR).  ABBV-744 retained robust activity in prostate cancer xenografts, and showed fewer platelet and gastrointestinal toxicities than the DbBi ABBV-07514.  Analyses of RNA expression and chromatin immunopptn. followed by sequencing revealed that ABBV-744 displaced BRD4 from AR-contg. super-enhancers and inhibited AR-dependent transcription, with less impact on global transcription compared with ABBV-075.  These results underscore the potential value of selectively targeting the BD2 domain of BET family proteins for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF98kER1y9pLVg90H21EOLACvtfcHk0lj54hoGJkT59A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D&md5=800d9106c33084390fd8cbf1ac7f1766</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-1930-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-1930-8%26sid%3Dliteratum%253Aachs%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMantena%26aufirst%3DS.%2BR.%26aulast%3DPlotnik%26aufirst%3DJ.%2BP.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSehgal%26aufirst%3DV.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DBellin%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DNicolette%26aufirst%3DJ.%2BJ.%26aulast%3DFossey%26aufirst%3DS.%2BL.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DBigelow%26aufirst%3DL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520the%2520BD2%2520bromodomain%2520of%2520BET%2520proteins%2520in%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2020%26volume%3D578%26spage%3D306%26epage%3D310%26doi%3D10.1038%2Fs41586-020-1930-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5585</span>â <span class="NLM_lpage">5623</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00628</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00628" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5585-5623&author=G.+S.+Sheppardauthor=L.+Wangauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=J.+K.+Prattauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=W.+Qiuauthor=M.+Torrentauthor=P.+J.+Kovarauthor=M.+Buiauthor=E.+Faivreauthor=X.+Huangauthor=X.+Linauthor=D.+Wilcoxauthor=L.+Zhangauthor=Y.+Shenauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=W.+M.+Katiauthor=K.+F.+McDaniel&title=Discovery+of+N-Ethyl-4-%5B2-%284-fluoro-2%2C6-dimethyl-phenoxy%29-5-%281-hydroxy-1-methyl-ethyl%29phenyl%5D-6-methyl-7-oxo-1H-pyrrolo%5B2%2C3-c%5Dpyridine-2-carboxamide+%28ABBV-744%29%2C+a+BET+Bromodomain+Inhibitor+with+Selectivity+for+the+Second+Bromodomain&doi=10.1021%2Facs.jmedchem.0c00628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Sheppard, George S.; Wang, Le; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Pratt, John K.; Park, Chang H.; Longenecker, Kenton; Qiu, Wei; Torrent, Maricel; Kovar, Peter J.; Bui, Mai; Faivre, Emily; Huang, Xiaoli; Lin, Xiaoyu; Wilcox, Denise; Zhang, Lu; Shen, Yu; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5585-5623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The BET family of proteins consists of BRD2, BRD3, BRD4, and BRDt.  Each protein contains two distinct bromodomains (BD1 and BD2).  BET family bromodomain inhibitors under clin. development for oncol. bind to each of the eight bromodomains with similar affinities.  We hypothesized that it may be possible to achieve an improved therapeutic index by selectively targeting subsets of the BET bromodomains.  Both BD1 and BD2 are highly conserved across family members (>70% identity), whereas BD1 and BD2 from the same protein exhibit a larger degree of divergence (~ 40% identity), suggesting selectivity between BD1 and BD2 of all family members would be more straightforward to achieve.  Exploiting the Asp144/His437 and Ile146/Val439 sequence differences (BRD4 BD1/BD2 numbering) allowed the identification of compd. 27 demonstrating greater than 100-fold selectivity for BRD4 BD2 over BRD4 BD1.  Further optimization to improve BD2 selectivity and oral bioavailability resulted in the clin. development compd. 46 (ABBV-744).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJtZIcWro0xLVg90H21EOLACvtfcHk0lh8DEq5FFDbNA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D&md5=675827659c3c7b6099ccd667f4c60f95</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00628%26sid%3Dliteratum%253Aachs%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DKovar%26aufirst%3DP.%2BJ.%26aulast%3DBui%26aufirst%3DM.%26aulast%3DFaivre%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26atitle%3DDiscovery%2520of%2520N-Ethyl-4-%255B2-%25284-fluoro-2%252C6-dimethyl-phenoxy%2529-5-%25281-hydroxy-1-methyl-ethyl%2529phenyl%255D-6-methyl-7-oxo-1H-pyrrolo%255B2%252C3-c%255Dpyridine-2-carboxamide%2520%2528ABBV-744%2529%252C%2520a%2520BET%2520Bromodomain%2520Inhibitor%2520with%2520Selectivity%2520for%2520the%2520Second%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5585%26epage%3D5623%26doi%3D10.1021%2Facs.jmedchem.0c00628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">648</span>â <span class="NLM_lpage">655</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2010.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.drudis.2010.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=20570751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=648-655&author=A.+P.+Hillauthor=R.+J.+Young&title=Getting+physical+in+drug+discovery%3A+a+contemporary+perspective+on+solubility+and+hydrophobicity&doi=10.1016%2Fj.drudis.2010.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span></div><div class="casAuthors">Hill, Alan P.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15/16</span>),
    <span class="NLM_cas:pages">648-655</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Suboptimal phys. properties have been identified as a particular shortcoming of compds. in contemporary drug discovery, contributing to high attrition levels.  An anal. of the relationship between hydrophobicity (calcd. and measured) and â¼100 k measured kinetic soly. values has been undertaken.  In line with the General Soly. Equation, ests. of hydrophobicity, particularly ACD c log D pH7.4, give a useful indication of the likely soly. classification of particular mols.  Taking ACD c log D pH7.4 values together with the no. of arom. rings in a given mol. provides enhanced prediction.  The Soly. Forecast Index' (SFI = c log D pH7.4 + #Ar) is proposed as a simple, yet effective, guide to predicting soly.  Moreover, anal. of measured distribution/partition coeff. values highlighted statistically significant shortcomings in the applicability of octanol/water as a model system for hydrophobicity detn. with poorly sol. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdtQyfNjbFUrVg90H21EOLACvtfcHk0lh8DEq5FFDbNA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P&md5=7b201081c5586a6ea4b7af00d507d462</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%253A%2520a%2520contemporary%2520perspective%2520on%2520solubility%2520and%2520hydrophobicity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2010%26volume%3D15%26spage%3D648%26epage%3D655%26doi%3D10.1016%2Fj.drudis.2010.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscombe, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span> <span> </span><span class="NLM_article-title">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">822</span>â <span class="NLM_lpage">830</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2011.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.drudis.2011.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=21704184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=822-830&author=R.+J.+Youngauthor=D.+V.+S.+Greenauthor=C.+N.+Luscombeauthor=A.+P.+Hill&title=Getting+physical+in+drug+discovery+II%3A+the+impact+of+chromatographic+hydrophobicity+measurements+and+aromaticity&doi=10.1016%2Fj.drudis.2011.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span></div><div class="casAuthors">Young, Robert J.; Green, Darren V. S.; Luscombe, Christopher N.; Hill, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17/18</span>),
    <span class="NLM_cas:pages">822-830</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here, we review the performance of chromatog. hydrophobicity measurements in a data set of 100 000 GlaxoSmithKline compds., demonstrating the advantages of the method over octanol-water partitioning and highlighting new insights for drug discovery.  The value of chromatog. measurements, vs. other hydrophobicity ests., was supported by improved relationships with soly., permeation, cytochrome P450s, intrinsic clearance, hERG binding and promiscuity.  We also obsd. marked differentiation of the relative influence of intrinsic and effective hydrophobicity.  The summing of hydrophobicity values plus arom. ring count [log D pH7.4 (or log P) + #Ar], indicated a wide relevance for simplistic property forecast indexes' in developability assays, clearly enhanced by chromatog. values; therefore establishing new foundations for enriching property-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmIJyWi-DBW7Vg90H21EOLACvtfcHk0lh4fGLmIZN53w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF&md5=22788603a45aa77537e8666433473d14</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DHill%26aufirst%3DA.%2BP.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%2520II%253A%2520the%2520impact%2520of%2520chromatographic%2520hydrophobicity%2520measurements%2520and%2520aromaticity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26spage%3D822%26epage%3D830%26doi%3D10.1016%2Fj.drudis.2011.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Discovery of Bromodomain Inhibitors Part 1: Inhibitor Binding Modes and Implications for Lead Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">576</span>â <span class="NLM_lpage">586</span>, <span class="refDoi">Â DOI: 10.1021/jm201320w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201320w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=576-586&author=C.-w.+Chungauthor=A.+W.+Deanauthor=J.+M.+Woolvenauthor=P.+Bamborough&title=Fragment-Based+Discovery+of+Bromodomain+Inhibitors+Part+1%3A+Inhibitor+Binding+Modes+and+Implications+for+Lead+Discovery&doi=10.1021%2Fjm201320w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Bromodomain Inhibitors Part 1: Inhibitor Binding Modes and Implications for Lead Discovery</span></div><div class="casAuthors">Chung, Chun-wa; Dean, Anthony W.; Woolven, James M.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">576-586</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are key epigenetic regulators of gene transcription and readers of the histone code.  However, the therapeutic benefits of modulating this target class are largely unexplored due to the lack of suitable chem. probes.  This article describes the generation of lead mols. for the BET bromodomains through screening a fragment set chosen using structural insights and computational approaches.  Anal. of 40 BRD2/fragment x-ray complexes highlights both shared and disparate interaction features that may be exploited for affinity and selectivity.  Six representative crystal structures are then exemplified in detail.  Two of the fragments are completely new bromodomain chemotypes, and three have never before been crystd. in a bromodomain, so our results significantly extend the limited public knowledge-base of crystallog. small mol./bromodomain interactions.  Certain fragments (including paracetamol) bind in a consistent mode to different bromodomains such as CREBBP, suggesting their potential to act as generic bromodomain templates.  An important implication is that the bromodomains are not only a phylogenetic family but also a system in which chem. and structural knowledge of one bromodomain gives insights transferable to others.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonNb7GnZnS_rVg90H21EOLACvtfcHk0lh4fGLmIZN53w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN&md5=5b7d7239d0de57266d94beef1ea080ae</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm201320w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201320w%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDean%26aufirst%3DA.%2BW.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-Based%2520Discovery%2520of%2520Bromodomain%2520Inhibitors%2520Part%25201%253A%2520Inhibitor%2520Binding%2520Modes%2520and%2520Implications%2520for%2520Lead%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D576%26epage%3D586%26doi%3D10.1021%2Fjm201320w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delves, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ClÃ©ment, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Small Molecule Inhibitors of the BET Family Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3827</span>â <span class="NLM_lpage">3838</span>, <span class="refDoi">Â DOI: 10.1021/jm200108t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200108t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3827-3838&author=C.-w.+Chungauthor=H.+Costeauthor=J.+H.+Whiteauthor=O.+Mirguetauthor=J.+Wildeauthor=R.+L.+Gosminiauthor=C.+Delvesauthor=S.+M.+Magnyauthor=R.+Woodwardauthor=S.+A.+Hughesauthor=E.+V.+Boursierauthor=H.+Flynnauthor=A.+M.+Bouillotauthor=P.+Bamboroughauthor=J.-M.+G.+Brusqauthor=F.+J.+Gellibertauthor=E.+J.+Jonesauthor=A.+M.+Riouauthor=P.+Homesauthor=S.+L.+Martinauthor=I.+J.+Uingsauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=A.-B.+Boullayauthor=R.+L.+Grimleyauthor=F.+M.+Blandelauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=Discovery+and+Characterization+of+Small+Molecule+Inhibitors+of+the+BET+Family+Bromodomains&doi=10.1021%2Fjm200108t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span></div><div class="casAuthors">Chung, Chun-wa; Coste, Herve; White, Julia H.; Mirguet, Olivier; Wilde, Jonathan; Gosmini, Romain L.; Delves, Chris; Magny, Sylvie M.; Woodward, Robert; Hughes, Stephen A.; Boursier, Eric V.; Flynn, Helen; Bouillot, Anne M.; Bamborough, Paul; Brusq, Jean-Marie G.; Gellibert, Francoise J.; Jones, Emma J.; Riou, Alizon M.; Homes, Paul; Martin, Sandrine L.; Uings, Iain J.; Toum, Jerome; Clement, Catherine A.; Boullay, Anne-Benedicte; Grimley, Rachel L.; Blandel, Florence M.; Prinjha, Rab K.; Lee, Kevin; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3827-3838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic mechanisms of gene regulation have a profound role in normal development and disease processes.  An integral part of this mechanism occurs through lysine acetylation of histone tails which are recognized by bromodomains.  While the biol. and structural characterization of many bromodomain contg. proteins has advanced considerably, the therapeutic tractability of this protein family is only now becoming understood.  This paper describes the discovery and mol. characterization of potent (nM) small mol. inhibitors that disrupt the function of the BET family of bromodomains (Brd2, Brd3, and Brd4).  By using a combination of phenotypic screening, chemoproteomics, and biophys. studies, we have discovered that the protein-protein interactions between bromodomains and acetylated histones can be antagonized by selective small mols. that bind at the acetylated lysine recognition pocket.  X-ray crystal structures of compds. bound into bromodomains of Brd2 and Brd4 elucidate the mol. interactions of binding and explain the precisely defined stereochem. required for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VauuYkT5YLVg90H21EOLACvtfcHk0lh4fGLmIZN53w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D&md5=ebc0f6062321dece877259d775f1c19b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm200108t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200108t%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%2BH.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DJ.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DDelves%26aufirst%3DC.%26aulast%3DMagny%26aufirst%3DS.%2BM.%26aulast%3DWoodward%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DS.%2BA.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DBouillot%26aufirst%3DA.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBrusq%26aufirst%3DJ.-M.%2BG.%26aulast%3DGellibert%26aufirst%3DF.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DRiou%26aufirst%3DA.%2BM.%26aulast%3DHomes%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BL.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBoullay%26aufirst%3DA.-B.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBlandel%26aufirst%3DF.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520the%2520BET%2520Family%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3827%26epage%3D3838%26doi%3D10.1021%2Fjm200108t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>â <span class="NLM_lpage">356</span>, <span class="refDoi">Â DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0liP55oo_c8h1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>â <span class="NLM_lpage">231</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+Recognition+and+Large-Scale+Structural+Analysis+of+the+Human+Bromodomain+Family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize Îµ-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0liP55oo_c8h1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520Recognition%2520and%2520Large-Scale%2520Structural%2520Analysis%2520of%2520the%2520Human%2520Bromodomain%2520Family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">430</span>â <span class="NLM_lpage">431</span>, <span class="refDoi">Â DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eZAkRiomPKhVLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span> <span> </span><span class="NLM_article-title">The maximal affinity of ligands</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">9997</span>â <span class="NLM_lpage">10002</span>, <span class="refDoi">Â DOI: 10.1073/pnas.96.18.9997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1073%2Fpnas.96.18.9997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10468550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=9997-10002&author=I.+D.+Kuntzauthor=K.+Chenauthor=K.+A.+Sharpauthor=P.+A.+Kollman&title=The+maximal+affinity+of+ligands&doi=10.1073%2Fpnas.96.18.9997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The maximal affinity of ligands</span></div><div class="casAuthors">Kuntz, I. D.; Chen, K.; Sharp, K. A.; Kollman, P. A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">9997-10002</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We explore the question of what are the best ligands for macromol. targets.  A survey of exptl. data on a large no. of the strongest-binding ligands indicates that the free energy of binding increases with the no. of nonhydrogen atoms with an initial slope of â-1.5 kcal/mol (1 cal = 4.18 J) per atom.  For ligands that contain more than 15 nonhydrogen atoms, the free energy of binding increases very little with relative mol. mass.  This nonlinearity is largely ascribed to nonthermodynamic factors.  An anal. of the dominant interactions suggests that van der Waals interactions and hydrophobic effects provide a reasonable basis for understanding binding affinities across the entire set of ligands.  Interesting outliers that bind unusually strongly on a per atom basis include metal ions, covalently attached ligands, and a few well known complexes such as biotin-avidin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0L6KZhR0TxbVg90H21EOLACvtfcHk0ljk5Juz91USTw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D&md5=46fe894b016f3041831a2f0b71f812b1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.18.9997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.18.9997%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DSharp%26aufirst%3DK.%2BA.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DThe%2520maximal%2520affinity%2520of%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26spage%3D9997%26epage%3D10002%26doi%3D10.1073%2Fpnas.96.18.9997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skipper, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wogan, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tannenbaum, S. R.</span></span> <span> </span><span class="NLM_article-title">Monocyclic aromatic amines as potential human carcinogens: old is new again</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">50</span>â <span class="NLM_lpage">58</span>, <span class="refDoi">Â DOI: 10.1093/carcin/bgp267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1093%2Fcarcin%2Fbgp267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=19887514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktlCnsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=50-58&author=P.+L.+Skipperauthor=M.+Y.+Kimauthor=H.+L.+P.+Sunauthor=G.+N.+Woganauthor=S.+R.+Tannenbaum&title=Monocyclic+aromatic+amines+as+potential+human+carcinogens%3A+old+is+new+again&doi=10.1093%2Fcarcin%2Fbgp267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Monocyclic aromatic amines as potential human carcinogens: old is new again</span></div><div class="casAuthors">Skipper, Paul L.; Kim, Min Young; Sun, H.-L. Patty; Wogan, Gerald N.; Tannenbaum, Steven R.</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-58</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Alkylanilines are a group of chems. whose ubiquitous presence in the environment is a result of the multitude of sources from which they originate.  Exposure assessments indicate that most individuals experience lifelong exposure to these compds.  Many alkylanilines have biol. activity similar to that of the carcinogenic multi-ring arom. amines.  This review provides an overview of human exposure and biol. effects.  It also describes recent investigations into the biochem. mechanisms of action that lead to the assessment that they are most probably more complex than those of the more extensively investigated multi-ring arom. amines.  Not only is nitrenium ion chem. implicated in DNA damage by alkylanilines but also reactions involving quinone imines and perhaps reactive oxygen species.  Recent results described here indicate that alkylanilines can be potent genotoxins for cultured mammalian cells when activated by exogenous or endogenous phase I and phase II xenobiotic-metabolizing enzymes.  The nature of specific DNA damage products responsible for mutagenicity remains to be identified but evidence to date supports mechanisms of activation through obligatory N-hydroxylation as well as subsequent conjugation by sulfation and/or acetylation.  A fuller understanding of the mechanisms of alkylaniline genotoxicity is expected to provide important insights into the environmental and genetic origins of one or more human cancers and may reveal a substantial role for this group of compds. as potential human chem. carcinogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu8jtJ_-C4s7Vg90H21EOLACvtfcHk0ljk5Juz91USTw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktlCnsw%253D%253D&md5=37e5a874a1ee9edf96acb3ad73ec2925</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp267%26sid%3Dliteratum%253Aachs%26aulast%3DSkipper%26aufirst%3DP.%2BL.%26aulast%3DKim%26aufirst%3DM.%2BY.%26aulast%3DSun%26aufirst%3DH.%2BL.%2BP.%26aulast%3DWogan%26aufirst%3DG.%2BN.%26aulast%3DTannenbaum%26aufirst%3DS.%2BR.%26atitle%3DMonocyclic%2520aromatic%2520amines%2520as%2520potential%2520human%2520carcinogens%253A%2520old%2520is%2520new%2520again%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D50%26epage%3D58%26doi%3D10.1093%2Fcarcin%2Fbgp267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ames, B. N.</span></span> <span> </span><span class="NLM_article-title">Identifying environmental chemicals causing mutations and cancer</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>204</i></span>,  <span class="NLM_fpage">587</span>â <span class="NLM_lpage">593</span>, <span class="refDoi">Â DOI: 10.1126/science.373122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1126%2Fscience.373122" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=373122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADyaE1MXktFClt70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=204&publication_year=1979&pages=587-593&author=B.+N.+Ames&title=Identifying+environmental+chemicals+causing+mutations+and+cancer&doi=10.1126%2Fscience.373122"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying environmental chemicals causing mutations and cancer</span></div><div class="casAuthors">Ames, Bruce N.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">204</span>
        (<span class="NLM_cas:issue">4393</span>),
    <span class="NLM_cas:pages">587-93</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A review with 57 refs. on environmental chem. induced carcinogenesis and mutagenesis and their relation to DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2AzWl_6PJ4bVg90H21EOLACvtfcHk0ljk5Juz91USTw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1MXktFClt70%253D&md5=9ac98bad1482e9cda849723a71f32a9d</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fscience.373122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.373122%26sid%3Dliteratum%253Aachs%26aulast%3DAmes%26aufirst%3DB.%2BN.%26atitle%3DIdentifying%2520environmental%2520chemicals%2520causing%2520mutations%2520and%2520cancer%26jtitle%3DScience%26date%3D1979%26volume%3D204%26spage%3D587%26epage%3D593%26doi%3D10.1126%2Fscience.373122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frisch, M. J. G. W. T.</span>; <span class="NLM_string-name">Schlegel, H. B.</span>; <span class="NLM_string-name">Scuseria, G. E.</span>; <span class="NLM_string-name">Robb, M. A.</span>; <span class="NLM_string-name">Cheeseman, J. R.</span>; <span class="NLM_string-name">Montgomery,  Jr., J. A.</span>; <span class="NLM_string-name">Vreven, T.</span>; <span class="NLM_string-name">Kudin, K. N.</span>; <span class="NLM_string-name">Burant, J. C.</span>; <span class="NLM_string-name">Millam, J. M.</span>; <span class="NLM_string-name">Iyengar, S. S.</span>; <span class="NLM_string-name">Tomasi, J.</span>; <span class="NLM_string-name">Barone, V.</span>; <span class="NLM_string-name">Mennucci, B.</span>; <span class="NLM_string-name">Cossi, M.</span>; <span class="NLM_string-name">Scalmani, G.</span>; <span class="NLM_string-name">Rega, N.</span>; <span class="NLM_string-name">Petersson, G. A.</span>; <span class="NLM_string-name">Nakatsuji, H.</span>; <span class="NLM_string-name">Hada, M.</span>; <span class="NLM_string-name">Ehara, M.</span>; <span class="NLM_string-name">Toyota, K.</span>; <span class="NLM_string-name">Fukuda, R.</span>; <span class="NLM_string-name">Hasegawa, J.</span>; <span class="NLM_string-name">Ishida, M.</span>; <span class="NLM_string-name">Nakajima, T.</span>; <span class="NLM_string-name">Honda, Y.</span>; <span class="NLM_string-name">Kitao, O.</span>; <span class="NLM_string-name">Nakai, H.</span>; <span class="NLM_string-name">Klene, M.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Knox, J. E.</span>; <span class="NLM_string-name">Hratchian, H. P.</span>; <span class="NLM_string-name">Cross, J. B.</span>; <span class="NLM_string-name">Bakken, V.</span>; <span class="NLM_string-name">Adamo, C.</span>; <span class="NLM_string-name">Jaramillo, J.</span>; <span class="NLM_string-name">Gomperts, R.</span>; <span class="NLM_string-name">Stratmann, R. E.</span>; <span class="NLM_string-name">Yazyev, O.</span>; <span class="NLM_string-name">Austin, A. J.</span>; <span class="NLM_string-name">Cammi, R.</span>; <span class="NLM_string-name">Pomelli, C.</span>; <span class="NLM_string-name">Ochterski, J. W.</span>; <span class="NLM_string-name">Ayala, P. Y.</span>; <span class="NLM_string-name">Morokuma, K.</span>; <span class="NLM_string-name">Voth, G. A.</span>; <span class="NLM_string-name">Salvador, P.</span>; <span class="NLM_string-name">Dannenberg, J. J.</span>; <span class="NLM_string-name">Zakrzewski, V. G.</span>; <span class="NLM_string-name">Dapprich, S.</span>; <span class="NLM_string-name">Daniels, A. D.</span>; <span class="NLM_string-name">Strain, M. C.</span>; <span class="NLM_string-name">Farkas, O.</span>; <span class="NLM_string-name">Malick, D. K.</span>; <span class="NLM_string-name">Rabuck, A. D.</span>; <span class="NLM_string-name">Raghavachari, K.</span>; <span class="NLM_string-name">Foresman, J. B.</span>; <span class="NLM_string-name">Ortiz, J. V.</span>; <span class="NLM_string-name">Cui, Q.</span>; <span class="NLM_string-name">Baboul, A. G.</span>; <span class="NLM_string-name">Clifford, S.</span>; <span class="NLM_string-name">Cioslowski, J.</span>; <span class="NLM_string-name">Stefanov, B. B.</span>; <span class="NLM_string-name">Liu, G.</span>; <span class="NLM_string-name">Liashenko, A.</span>; <span class="NLM_string-name">Piskorz, P.</span>; <span class="NLM_string-name">Komaromi, I.</span>; <span class="NLM_string-name">Martin, R. L.</span>; <span class="NLM_string-name">Fox, D. J.</span>; <span class="NLM_string-name">Keith, T.</span>; <span class="NLM_string-name">Al-Laham, M. A.</span>; <span class="NLM_string-name">Peng, C. Y.</span>; <span class="NLM_string-name">Nanayakkara, A.</span>; <span class="NLM_string-name">Challacombe, M.</span>; <span class="NLM_string-name">Gill, P. M. W.</span>; <span class="NLM_string-name">Johnson, B.</span>; <span class="NLM_string-name">Chen, W.</span>; <span class="NLM_string-name">Wong, M. W.</span>; <span class="NLM_string-name">Gonzalez, C.</span>; <span class="NLM_string-name">Pople, J. A.</span></span> <i>Gaussian 03</i>; Revision C.02 <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=M.+J.+G.+W.+T.+Frisch&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=J.+A.+Montgomery&author=T.+Vreven&author=K.+N.+Kudin&author=J.+C.+Burant&author=J.+M.+Millam&author=S.+S.+Iyengar&author=J.+Tomasi&author=V.+Barone&author=B.+Mennucci&author=M.+Cossi&author=G.+Scalmani&author=N.+Rega&author=G.+A.+Petersson&author=H.+Nakatsuji&author=M.+Hada&author=M.+Ehara&author=K.+Toyota&author=R.+Fukuda&author=J.+Hasegawa&author=M.+Ishida&author=T.+Nakajima&author=Y.+Honda&author=O.+Kitao&author=H.+Nakai&author=M.+Klene&author=X.+Li&author=J.+E.+Knox&author=H.+P.+Hratchian&author=J.+B.+Cross&author=V.+Bakken&author=C.+Adamo&author=J.+Jaramillo&author=R.+Gomperts&author=R.+E.+Stratmann&author=O.+Yazyev&author=A.+J.+Austin&author=R.+Cammi&author=C.+Pomelli&author=J.+W.+Ochterski&author=P.+Y.+Ayala&author=K.+Morokuma&author=G.+A.+Voth&author=P.+Salvador&author=J.+J.+Dannenberg&author=V.+G.+Zakrzewski&author=S.+Dapprich&author=A.+D.+Daniels&author=M.+C.+Strain&author=O.+Farkas&author=D.+K.+Malick&author=A.+D.+Rabuck&author=K.+Raghavachari&author=J.+B.+Foresman&author=J.+V.+Ortiz&author=Q.+Cui&author=A.+G.+Baboul&author=S.+Clifford&author=J.+Cioslowski&author=B.+B.+Stefanov&author=G.+Liu&author=A.+Liashenko&author=P.+Piskorz&author=I.+Komaromi&author=R.+L.+Martin&author=D.+J.+Fox&author=T.+Keith&author=M.+A.+Al-Laham&author=C.+Y.+Peng&author=A.+Nanayakkara&author=M.+Challacombe&author=P.+M.+W.+Gill&author=B.+Johnson&author=W.+Chen&author=M.+W.+Wong&author=C.+Gonzalez&author=J.+A.+Pople&title=Gaussian+03"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%2BG.%2BW.%2BT.%26jtitle%3DGaussian%252003%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deshmane, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kremlev, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawaya, B. E.</span></span> <span> </span><span class="NLM_article-title">Monocyte Chemoattractant Protein-1 (MCP-1): An Overview</span>. <i>J. Interferon Cytokine Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">313</span>â <span class="NLM_lpage">326</span>, <span class="refDoi">Â DOI: 10.1089/jir.2008.0027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1089%2Fjir.2008.0027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=19441883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvF2rs7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=313-326&author=S.+L.+Deshmaneauthor=S.+Kremlevauthor=S.+Aminiauthor=B.+E.+Sawaya&title=Monocyte+Chemoattractant+Protein-1+%28MCP-1%29%3A+An+Overview&doi=10.1089%2Fjir.2008.0027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Monocyte Chemoattractant Protein-1 (MCP-1): An Overview</span></div><div class="casAuthors">Deshmane, Satish L.; Kremlev, Sergey; Amini, Shohreh; Sawaya, Bassel E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Interferon & Cytokine Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">313-326</span>CODEN:
                <span class="NLM_cas:coden">JICRFJ</span>;
        ISSN:<span class="NLM_cas:issn">1079-9907</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Chemokines constitute a family of chemoattractant cytokines and are subdivided into four families on the basis of the no. and spacing of the conserved cysteine residues in the N-terminus of the protein.  Chemokines play a major role in selectively recruiting monocytes, neutrophils, and lymphocytes, as well as in inducing chemotaxis through the activation of G-protein-coupled receptors.  Monocyte chemoattractant protein-1 (MCP-1/CCL2) is one of the key chemokines that regulate migration and infiltration of monocytes/macrophages.  Both CCL2 and its receptor CCR2 have been demonstrated to be induced and involved in various diseases.  Migration of monocytes from the blood stream across the vascular endothelium is required for routine immunol. surveillance of tissues, as well as in response to inflammation.  This review will discuss these biol. processes and the structure and function of CCL2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSmnT35tYDZLVg90H21EOLACvtfcHk0lgJsVAx69S8FA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvF2rs7s%253D&md5=d58ea0c1925c2eff93631916f93edad3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1089%2Fjir.2008.0027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fjir.2008.0027%26sid%3Dliteratum%253Aachs%26aulast%3DDeshmane%26aufirst%3DS.%2BL.%26aulast%3DKremlev%26aufirst%3DS.%26aulast%3DAmini%26aufirst%3DS.%26aulast%3DSawaya%26aufirst%3DB.%2BE.%26atitle%3DMonocyte%2520Chemoattractant%2520Protein-1%2520%2528MCP-1%2529%253A%2520An%2520Overview%26jtitle%3DJ.%2520Interferon%2520Cytokine%2520Res.%26date%3D2009%26volume%3D29%26spage%3D313%26epage%3D326%26doi%3D10.1089%2Fjir.2008.0027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E. Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliadis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burr, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A. G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbetti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vappiani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">Selective targeting of BD1 and BD2 of the BET proteins in cancer and immuno-inflammation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>386</i></span> (<span class="NLM_issue">6489</span>),  <span class="NLM_fpage">387</span>â <span class="NLM_lpage">394</span>, <span class="refDoi">Â DOI: 10.1126/science.aaz8455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1126%2Fscience.aaz8455" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2020&pages=387-394&issue=6489&author=O.+Gilanauthor=I.+Riojaauthor=K.+Knezevicauthor=M.+J.+Bellauthor=M.+M.+Yeungauthor=N.+R.+Harkerauthor=E.+Y.+N.+Lamauthor=C.-w.+Chungauthor=P.+Bamboroughauthor=M.+Petretichauthor=M.+Urhauthor=S.+J.+Atkinsonauthor=A.+K.+Bassilauthor=E.+J.+Robertsauthor=D.+Vassiliadisauthor=M.+L.+Burrauthor=A.+G.+S.+Prestonauthor=C.+Wellawayauthor=T.+Wernerauthor=J.+R.+Grayauthor=A.-M.+Michonauthor=T.+Gobbettiauthor=V.+Kumarauthor=P.+E.+Sodenauthor=A.+Haynesauthor=J.+Vappianiauthor=D.+F.+Toughauthor=S.+Taylorauthor=S.-J.+Dawsonauthor=M.+Bantscheffauthor=M.+Lindonauthor=G.+Drewesauthor=E.+H.+Demontauthor=D.+L.+Danielsauthor=P.+Grandiauthor=R.+K.+Prinjhaauthor=M.+A.+Dawson&title=Selective+targeting+of+BD1+and+BD2+of+the+BET+proteins+in+cancer+and+immuno-inflammation&doi=10.1126%2Fscience.aaz8455"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaz8455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaz8455%26sid%3Dliteratum%253Aachs%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DKnezevic%26aufirst%3DK.%26aulast%3DBell%26aufirst%3DM.%2BJ.%26aulast%3DYeung%26aufirst%3DM.%2BM.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DLam%26aufirst%3DE.%2BY.%2BN.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DVassiliadis%26aufirst%3DD.%26aulast%3DBurr%26aufirst%3DM.%2BL.%26aulast%3DPreston%26aufirst%3DA.%2BG.%2BS.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DJ.%2BR.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DGobbetti%26aufirst%3DT.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DHaynes%26aufirst%3DA.%26aulast%3DVappiani%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DDawson%26aufirst%3DS.-J.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DSelective%2520targeting%2520of%2520BD1%2520and%2520BD2%2520of%2520the%2520BET%2520proteins%2520in%2520cancer%2520and%2520immuno-inflammation%26jtitle%3DScience%26date%3D2020%26volume%3D386%26issue%3D6489%26spage%3D387%26epage%3D394%26doi%3D10.1126%2Fscience.aaz8455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, P. B.</span>; <span class="NLM_string-name">Bernard, S.</span>; <span class="NLM_string-name">Chung, C.-w.</span>; <span class="NLM_string-name">Craggs, P. D.</span>; <span class="NLM_string-name">Cutler, L.</span>; <span class="NLM_string-name">Demont, E. H.</span>; <span class="NLM_string-name">Evans, J.P.</span>; <span class="NLM_string-name">Gordon, L.</span>; <span class="NLM_string-name">Karamshi, B.</span>; <span class="NLM_string-name">Lewis, A. J.</span>; <span class="NLM_string-name">Lindon, M. J.</span>; <span class="NLM_string-name">Mitchell, D. J.</span>; <span class="NLM_string-name">Rioja, I.</span>; <span class="NLM_string-name">Soden, P. E.</span>; <span class="NLM_string-name">Taylor, S.</span>; <span class="NLM_string-name">Watson, R. J.</span>; <span class="NLM_string-name">Willis, R.</span>; <span class="NLM_string-name">Woolven, J. M.</span>; <span class="NLM_string-name">Wyspianska, B. S.</span>; <span class="NLM_string-name">Kerr, W. J.</span>; <span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based Design of a Bromodomain and Extraterminal Domain (BET) Inhibitor Selective for the N-terminal Bromodomains that Retains an Anti-inflammatory and Anti-proliferative Phenotype</span>.  <i>J. Med. Chem.</i> [Online early access]. <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00566</span>  Published Online: August 3, <span class="NLM_year" style="font-weight: bold;">2020</span>.</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=P.+B.+Wellaway&author=S.+Bernard&author=C.-w.+Chung&author=P.+D.+Craggs&author=L.+Cutler&author=E.+H.+Demont&author=J.P.+Evans&author=L.+Gordon&author=B.+Karamshi&author=A.+J.+Lewis&author=M.+J.+Lindon&author=D.+J.+Mitchell&author=I.+Rioja&author=P.+E.+Soden&author=S.+Taylor&author=R.+J.+Watson&author=R.+Willis&author=J.+M.+Woolven&author=B.+S.+Wyspianska&author=W.+J.+Kerr&author=R.+K.+Prinjha&title=Structure-based+Design+of+a+Bromodomain+and+Extraterminal+Domain+%28BET%29+Inhibitor+Selective+for+the+N-terminal+Bromodomains+that+Retains+an+Anti-inflammatory+and+Anti-proliferative+Phenotype&doi=10.1021%2Facs.jmedchem.0c00566"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00566%26sid%3Dliteratum%253Aachs%26aulast%3DWellaway%26aufirst%3DP.%2BB.%26atitle%3DStructure-based%2520Design%2520of%2520a%2520Bromodomain%2520and%2520Extraterminal%2520Domain%2520%2528BET%2529%2520Inhibitor%2520Selective%2520for%2520the%2520N-terminal%2520Bromodomains%2520that%2520Retains%2520an%2520Anti-inflammatory%2520and%2520Anti-proliferative%2520Phenotype%26jtitle%3DJ.%2520Med.%2520Chem.%26doi%3D10.1021%2Facs.jmedchem.0c00566%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">OâBoyle, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banck, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandermeersch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchison, G. R.</span></span> <span> </span><span class="NLM_article-title">Open Babel: An open chemical toolbox</span>. <i>Aust. J. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">33</span>, <span class="refDoi">Â DOI: 10.1186/1758-2946-3-33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1186%2F1758-2946-3-33" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=33&author=N.+M.+O%E2%80%99Boyleauthor=M.+Banckauthor=C.+A.+Jamesauthor=C.+Morleyauthor=T.+Vandermeerschauthor=G.+R.+Hutchison&title=Open+Babel%3A+An+open+chemical+toolbox&doi=10.1186%2F1758-2946-3-33"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-3-33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-3-33%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%2BM.%26aulast%3DBanck%26aufirst%3DM.%26aulast%3DJames%26aufirst%3DC.%2BA.%26aulast%3DMorley%26aufirst%3DC.%26aulast%3DVandermeersch%26aufirst%3DT.%26aulast%3DHutchison%26aufirst%3DG.%2BR.%26atitle%3DOpen%2520Babel%253A%2520An%2520open%2520chemical%2520toolbox%26jtitle%3DAust.%2520J.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D33%26doi%3D10.1186%2F1758-2946-3-33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camurri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaramella, A.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Liquid Chromatography/Mass Spectrometry Method for the Determination of the Chromatographic Hydrophobicity Index</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">3716</span>â <span class="NLM_lpage">3722</span>, <span class="refDoi">Â DOI: 10.1021/ac001388j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac001388j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkvVCgsbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2001&pages=3716-3722&author=G.+Camurriauthor=A.+Zaramella&title=High-Throughput+Liquid+Chromatography%2FMass+Spectrometry+Method+for+the+Determination+of+the+Chromatographic+Hydrophobicity+Index&doi=10.1021%2Fac001388j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Liquid Chromatography/Mass Spectrometry Method for the Determination of the Chromatographic Hydrophobicity Index</span></div><div class="casAuthors">Camurri, Giulio; Zaramella, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3716-3722</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A fast gradient reversed-phase liq. chromatog. (LC) method, using an acetonitrile gradient was developed to det. the chromatog. hydrophobicity index (CHI), as reported by Valco et al. (Anal. Chem. 1997, 69, 2022-2029).The anal. method provides retention times, based on UV detection at two different wavelengths, which then are converted into CHI values after calibration with a set of test compds.  The CHI of each compd. is measured at three different pH values, 2.0, 7.4, and 10.5; so using an 8-min gradient at each pH value one compd. can be analyzed in â¼24 min.  The aim of this work is to improve the throughput of the CHI screening using a LC/MS approach, so the application of the LC/MS technique is an extension of the LC/UV approach previously reported by Valco et al.  This approach allows contemporary injection of N compds. into the LC/MS system, the retention time of each compd. can be then extd. from the selected ion recording chromatograms.  The throughput of the existing screening method could be increased by N times, where N is the no. of compds. injected, so only three runs are needed to det. the CHI at three different pH values for a set of N compds.  The highest value of N depends on the total no. of channels that can be monitored simultaneously; 32 channels were used.  This LC/MS method was tested for a no. of com. products analyzed as mixts., and data obtained were compared with those coming from the classical LC/UV approach.  In the same way, the method was tested for a no. of compds. assocd. with two GlaxoWellcome projects in the antibacterial area.  The LC/MS method can be successfully applied for analyzing compds. in mixts. and for compds. with poor UV absorption, which cannot be analyzed with the std. LC/UV method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnpqYuIDpy5rVg90H21EOLACvtfcHk0ljkSJ2ecDu7Iw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkvVCgsbY%253D&md5=8773c87e08ba7ddce3c2d679360a6220</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fac001388j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac001388j%26sid%3Dliteratum%253Aachs%26aulast%3DCamurri%26aufirst%3DG.%26aulast%3DZaramella%26aufirst%3DA.%26atitle%3DHigh-Throughput%2520Liquid%2520Chromatography%252FMass%2520Spectrometry%2520Method%2520for%2520the%2520Determination%2520of%2520the%2520Chromatographic%2520Hydrophobicity%2520Index%26jtitle%3DAnal.%2520Chem.%26date%3D2001%26volume%3D73%26spage%3D3716%26epage%3D3722%26doi%3D10.1021%2Fac001388j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunhuck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, D. P.</span></span> <span> </span><span class="NLM_article-title">Fast gradient HPLC method to determine compounds binding to human serum albumin. Relationships with octanol/water and immobilized artificial membrane lipophilicity</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">2236</span>â <span class="NLM_lpage">2248</span>, <span class="refDoi">Â DOI: 10.1002/jps.10494</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1002%2Fjps.10494" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=14603509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovFCrs7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2003&pages=2236-2248&author=K.+Valkoauthor=S.+Nunhuckauthor=C.+Bevanauthor=M.+H.+Abrahamauthor=D.+P.+Reynolds&title=Fast+gradient+HPLC+method+to+determine+compounds+binding+to+human+serum+albumin.+Relationships+with+octanol%2Fwater+and+immobilized+artificial+membrane+lipophilicity&doi=10.1002%2Fjps.10494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Fast gradient HPLC method to determine compounds binding to human serum albumin. Relationships with octanol/water and immobilized artificial membrane lipophilicity</span></div><div class="casAuthors">Valko, Klara; Nunhuck, Shenaz; Bevan, Chris; Abraham, Michael H.; Reynolds, Derek P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2236-2248</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A fast gradient HPLC method (cycle time 15 min) has been developed to det. Human Serum Albumin (HSA) binding of discovery compds. using chem. bonded protein stationary phases.  The HSA binding values were derived from the gradient retention times that were converted to the logarithm of the equil. consts. (logK HSA) using data from a calibration set of mols.  The method has been validated using literature plasma protein binding data of 68 known drug mols.  The method is fully automated, and has been used for lead optimization in more than 20 company projects.  The HSA binding data obtained for more than 4000 compds. were suitable to set up global and project specific quant. structure binding relationships that helped compd. design in early drug discovery.  The obtained HSA binding of known drug mols. were compared to the Immobilizd Artificial Membrane binding data (CHI IAM) obtained by our previously described HPLC-based method.  The solvation equation approach has been used to characterize the normal binding ability of HSA, and this relationship shows that compd. lipophilicity is a significant factor.  It was found that the selectivity of the "baseline" lipophilicity governing HSA binding, membrane interaction, and octanol/water partition are very similar.  However, the effect of the presence of pos. or neg. charges have very different effects.  It was found that neg. charged compds. bind more strongly to HSA than it would be expected from the lipophilicity of the ionized species at pH 7.4.  Several compds. showed stronger HSA binding than can be expected from their lipophilicity alone, and comparison between predicted and exptl. binding affinity allows the identification of compds. that have good complementarities with any of the known binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr1LUBLdv-t7Vg90H21EOLACvtfcHk0li4e-79bRGWmQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovFCrs7Y%253D&md5=14eb1e0f0ad616846a2b5d1428133fa9</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fjps.10494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.10494%26sid%3Dliteratum%253Aachs%26aulast%3DValko%26aufirst%3DK.%26aulast%3DNunhuck%26aufirst%3DS.%26aulast%3DBevan%26aufirst%3DC.%26aulast%3DAbraham%26aufirst%3DM.%2BH.%26aulast%3DReynolds%26aufirst%3DD.%2BP.%26atitle%3DFast%2520gradient%2520HPLC%2520method%2520to%2520determine%2520compounds%2520binding%2520to%2520human%2520serum%2520albumin.%2520Relationships%2520with%2520octanol%252Fwater%2520and%2520immobilized%2520artificial%2520membrane%2520lipophilicity%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2003%26volume%3D92%26spage%3D2236%26epage%3D2248%26doi%3D10.1002%2Fjps.10494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span> <span> </span><span class="NLM_article-title">A Chemical Probe for the ATAD2 Bromodomain</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">11382</span>â <span class="NLM_lpage">11386</span>, <span class="refDoi">Â DOI: 10.1002/anie.201603928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1002%2Fanie.201603928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=11382-11386&author=P.+Bamboroughauthor=C.-w.+Chungauthor=E.+H.+Demontauthor=R.+C.+Furzeauthor=A.+J.+Bannisterauthor=K.+H.+Cheauthor=H.+Dialloauthor=C.+Douaultauthor=P.+Grandiauthor=T.+Kouzaridesauthor=A.-M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=S.+Robsonauthor=R.+J.+Sheppardauthor=R.+Uptonauthor=R.+J.+Watson&title=A+Chemical+Probe+for+the+ATAD2+Bromodomain&doi=10.1002%2Fanie.201603928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Probe for the ATAD2 Bromodomain</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Furze, Rebecca C.; Bannister, Andrew J.; Che, Ka Hing; Diallo, Hawa; Douault, Clement; Grandi, Paola; Kouzarides, Tony; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Robson, Samuel; Sheppard, Robert J.; Upton, Richard; Watson, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11382-11386</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">ATAD2 is a cancer-assocd. protein whose bromodomain has been described as among the least druggable of that target class.  Starting from a potent lead, permeability and selectivity were improved through a dual approach: (1) using CF2 as a sulfone bio-isostere to exploit the unique properties of fluorine, and (2) using 1,3-interactions to control the conformation of a piperidine ring.  This resulted in the first reported low-nanomolar, selective and cell permeable chem. probe for ATAD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqId2vaqTEeyLVg90H21EOLACvtfcHk0li4e-79bRGWmQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL&md5=7eef64dfd3c75b9c17cd7f43d368df04</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fanie.201603928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201603928%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DRobson%26aufirst%3DS.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DUpton%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3DA%2520Chemical%2520Probe%2520for%2520the%2520ATAD2%2520Bromodomain%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D11382%26epage%3D11386%26doi%3D10.1002%2Fanie.201603928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6151</span>â <span class="NLM_lpage">6178</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00773</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00773" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6151-6178&author=P.+Bamboroughauthor=C.-w.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.-M.+Michonauthor=R.+J.+Sheppardauthor=H.+Barnettauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=R.+J.+Watsonauthor=T.+Wernerauthor=E.+H.+Demont&title=Structure-Based+Optimization+of+Naphthyridones+into+Potent+ATAD2+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b00773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Sheppard, Robert J.; Barnett, Heather; Diallo, Hawa; Dixon, David P.; Douault, Clement; Jones, Emma J.; Karamshi, Bhumika; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Watson, Robert J.; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6151-6178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 is a bromodomain-contg. protein whose overexpression is linked to poor outcomes in a no. of different cancer types.  To date, no potent and selective inhibitors of the bromodomain have been reported.  This article describes the structure-based optimization of a series of naphthyridones from micromolar leads with no selectivity over the BET bromodomains to inhibitors with sub-100 nM ATAD2 potency and 100-fold BET selectivity, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcCY9tFp5dGbVg90H21EOLACvtfcHk0lifKNtMG5EE_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK&md5=c7dcd72fdf4ab1c8557be45085cc68fa</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00773%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DStructure-Based%2520Optimization%2520of%2520Naphthyridones%2520into%2520Potent%2520ATAD2%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6151%26epage%3D6178%26doi%3D10.1021%2Facs.jmedchem.5b00773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 15 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Francesco Rianjongdee, Stephen J. Atkinson, Chun-wa Chung, Paola Grandi, James R. J. Gray, Laura J. Kaushansky, Patricia Medeiros, Cassie Messenger, Alex Phillipou, Alex Preston, Rab K. Prinjha, Inmaculada Rioja, Alexander L. Satz, Simon Taylor, Ian D. Wall, Robert J. Watson, Gang Yao, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Highly Selective BET BD2 Inhibitor from a DNA-Encoded Library Technology Screening Hit. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00412%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BHighly%252BSelective%252BBET%252BBD2%252BInhibitor%252Bfrom%252Ba%252BDNA-Encoded%252BLibrary%252BTechnology%252BScreening%252BHit%26aulast%3DRianjongdee%26aufirst%3DFrancesco%26date%3D2021%26date%3D2021%26date%3D08032021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yi Zhang, Deqin Rong, Bingbing Li, <span class="NLM_string-name hlFld-ContribAuthor">Yuanxiang Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 7900-7925. <a href="https://doi.org/10.1021/acs.jmedchem.0c02055" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02055%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BEpigenetic%252BRegulators%252Bwith%252BCovalent%252BSmall-Molecule%252BInhibitors%26aulast%3DZhang%26aufirst%3DYi%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26112020%26date%3D18022021%26volume%3D64%26issue%3D12%26spage%3D7900%26epage%3D7925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Adam I. Green, <span class="NLM_string-name hlFld-ContribAuthor">George M. Burslem</span>. </span><span class="cited-content_cbyCitation_article-title">Focused Libraries for Epigenetic Drug Discovery: The Importance of Isosteres. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (11)
                                     , 7231-7240. <a href="https://doi.org/10.1021/acs.jmedchem.1c00592" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00592</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00592%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFocused%252BLibraries%252Bfor%252BEpigenetic%252BDrug%252BDiscovery%25253A%252BThe%252BImportance%252Bof%252BIsosteres%26aulast%3DGreen%26aufirst%3DAdam%2BI.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D31032021%26date%3D27052021%26volume%3D64%26issue%3D11%26spage%3D7231%26epage%3D7240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Helen E. Aylott, Stephen J. Atkinson, Paul Bamborough, Anna Bassil, Chun-wa Chung, Laurie Gordon, Paola Grandi, James R. J. Gray, Lee A. Harrison, Thomas G. Hayhow, Cassie Messenger, Darren Mitchell, Alexander Phillipou, Alex Preston, Rab K. Prinjha, Francesco Rianjongdee, Inmaculada Rioja, Jonathan T. Seal, Ian D. Wall, Robert J. Watson, James M. Woolven, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Template-Hopping Approach Leads to Potent, Selective, and Highly Soluble Bromo and Extraterminal Domain (BET) Second Bromodomain (BD2) Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (6)
                                     , 3249-3281. <a href="https://doi.org/10.1021/acs.jmedchem.0c02156" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02156</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02156%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTemplate-Hopping%252BApproach%252BLeads%252Bto%252BPotent%25252C%252BSelective%25252C%252Band%252BHighly%252BSoluble%252BBromo%252Band%252BExtraterminal%252BDomain%252B%252528BET%252529%252BSecond%252BBromodomain%252B%252528BD2%252529%252BInhibitors%26aulast%3DAylott%26aufirst%3DHelen%2BE.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D14122020%26date%3D04032021%26volume%3D64%26issue%3D6%26spage%3D3249%26epage%3D3281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Pan Tang, Jifa Zhang, Jie Liu, Cheng-Ming Chiang, <span class="NLM_string-name hlFld-ContribAuthor">Liang Ouyang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Bromodomain and Extraterminal Proteins for Drug Discovery: From Current Progress to Technological Development. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2419-2435. <a href="https://doi.org/10.1021/acs.jmedchem.0c01487" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01487</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01487%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BBromodomain%252Band%252BExtraterminal%252BProteins%252Bfor%252BDrug%252BDiscovery%25253A%252BFrom%252BCurrent%252BProgress%252Bto%252BTechnological%252BDevelopment%26aulast%3DTang%26aufirst%3DPan%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26082020%26date%3D22022021%26volume%3D64%26issue%3D5%26spage%3D2419%26epage%3D2435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peter S. Dragovich, Thomas H. Pillow, Robert A. Blake, Jack D. Sadowsky, Emel Adaligil, Pragya Adhikari, Sunil Bhakta, Nicole Blaquiere, Jinhua Chen, Josefa dela Cruz-Chuh, Karen E. Gascoigne, Steven J. Hartman, Mingtao He, Susan Kaufman, Tracy Kleinheinz, Katherine R. Kozak, Liang Liu, Liling Liu, Qi Liu, Ying Lu, Fanwei Meng, Melinda M. Mulvihill, Aimee OâDonohue, Rebecca K. Rowntree, Leanna R. Staben, Steven T. Staben, John Wai, Jian Wang, BinQing Wei, Catherine Wilson, Jianfeng Xin, Zijin Xu, Hui Yao, Donglu Zhang, Hongyan Zhang, Hao Zhou, <span class="NLM_string-name hlFld-ContribAuthor">Xiaoyu Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Antibody-Mediated Delivery of Chimeric BRD4 Degraders. Part 1: Exploration of Antibody Linker, Payload Loading, and Payload Molecular Properties. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2534-2575. <a href="https://doi.org/10.1021/acs.jmedchem.0c01845" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01845</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01845%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAntibody-Mediated%252BDelivery%252Bof%252BChimeric%252BBRD4%252BDegraders.%252BPart%252B1%25253A%252BExploration%252Bof%252BAntibody%252BLinker%25252C%252BPayload%252BLoading%25252C%252Band%252BPayload%252BMolecular%252BProperties%26aulast%3DDragovich%26aufirst%3DPeter%2BS.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D21102020%26date%3D17022021%26volume%3D64%26issue%3D5%26spage%3D2534%26epage%3D2575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peter S. Dragovich, Thomas H. Pillow, Robert A. Blake, Jack D. Sadowsky, Emel Adaligil, Pragya Adhikari, Jinhua Chen, Nicholas Corr, Josefa dela Cruz-Chuh, Geoffrey Del Rosario, Aaron Fullerton, Steven J. Hartman, Fan Jiang, Susan Kaufman, Tracy Kleinheinz, Katherine R. Kozak, Liling Liu, Ying Lu, Melinda M. Mulvihill, Jeremy M. Murray, Aimee OâDonohue, Rebecca K. Rowntree, William S. Sawyer, Leanna R. Staben, John Wai, Jian Wang, BinQing Wei, Wentao Wei, Zijin Xu, Hui Yao, Shang-Fan Yu, Donglu Zhang, Hongyan Zhang, Shenhua Zhang, Yongxin Zhao, Hao Zhou, <span class="NLM_string-name hlFld-ContribAuthor">Xiaoyu Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Antibody-Mediated Delivery of Chimeric BRD4 Degraders. Part 2: Improvement of In Vitro Antiproliferation Activity and In Vivo Antitumor Efficacy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2576-2607. <a href="https://doi.org/10.1021/acs.jmedchem.0c01846" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01846</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01846%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAntibody-Mediated%252BDelivery%252Bof%252BChimeric%252BBRD4%252BDegraders.%252BPart%252B2%25253A%252BImprovement%252Bof%252BIn%252BVitro%252BAntiproliferation%252BActivity%252Band%252BIn%252BVivo%252BAntitumor%252BEfficacy%26aulast%3DDragovich%26aufirst%3DPeter%2BS.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D21102020%26date%3D17022021%26volume%3D64%26issue%3D5%26spage%3D2576%26epage%3D2607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Stuart J. Conway,  (<span class="NLM_role">Associate Editor, <i>Journal of Medicinal Chemistry</i></span>)Paola Arimondo,  (<span class="NLM_role">Guest Editor, Institut Pasteur Paris</span>)Cheryl Arrowsmith,  (<span class="NLM_role">Guest Editor, University of Toronto</span>)Jian Jin,  (<span class="NLM_role">Guest Editor, Icahn School of Medicine at Mount Sinai</span>)Cheng Luo,  (<span class="NLM_role">Guest Editor, Shanghai Institute of Materia Medica</span>)Nicholas Meanwell,  (<span class="NLM_role">Perspectives Editor, <i>Journal of Medicinal Chemistry</i></span>)Wendy Young,  (<span class="NLM_role">Drug Annotations Editor, <i>Journal of Medicinal Chemistry</i></span>)Gunda Georg,  (<span class="NLM_role">Editor-in-Chief, <i>Journal of Medicinal Chemistry</i></span>)<span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span> (<span class="NLM_role">Editor-in-Chief, <i>Journal of Medicinal Chemistry</i></span>). </span><span class="cited-content_cbyCitation_article-title">Epigenetics 2.0: Special Issue on EpigeneticsâCall for Papers. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (21)
                                     , 12129-12130. <a href="https://doi.org/10.1021/acs.jmedchem.0c01748" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01748</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01748%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEpigenetics%252B2.0%25253A%252BSpecial%252BIssue%252Bon%252BEpigenetics%2525E2%252580%252594Call%252Bfor%252BPapers%26aulast%3DConway%26aufirst%3DStuart%2BJ.%26date%3D2020%26date%3D2020%26date%3D22102020%26volume%3D63%26issue%3D21%26spage%3D12129%26epage%3D12130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher R. Wellaway, Paul Bamborough, Sharon G. Bernard, Chun-wa Chung, Peter D. Craggs, Leanne Cutler, Emmanuel H. Demont, John P. Evans, Laurie Gordon, Bhumika Karamshi, Antonia J. Lewis, Matthew J. Lindon, Darren J. Mitchell, Inmaculada Rioja, Peter E. Soden, Simon Taylor, Robert J. Watson, Rob Willis, James M. Woolven, Beata S. WyspiaÅska, William J. Kerr, <span class="NLM_string-name hlFld-ContribAuthor">Rab K. Prinjha</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of a Bromodomain and Extraterminal Domain (BET) Inhibitor Selective for the N-Terminal Bromodomains That Retains an Anti-inflammatory and Antiproliferative Phenotype. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9020-9044. <a href="https://doi.org/10.1021/acs.jmedchem.0c00566" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00566</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00566%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDesign%252Bof%252Ba%252BBromodomain%252Band%252BExtraterminal%252BDomain%252B%252528BET%252529%252BInhibitor%252BSelective%252Bfor%252Bthe%252BN-Terminal%252BBromodomains%252BThat%252BRetains%252Ban%252BAnti-inflammatory%252Band%252BAntiproliferative%252BPhenotype%26aulast%3DWellaway%26aufirst%3DChristopher%2BR.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06042020%26date%3D03082020%26volume%3D63%26issue%3D17%26spage%3D9020%26epage%3D9044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Robert J. Watson, Paul Bamborough, Heather Barnett, Chun-wa Chung, Rob Davis, Laurie Gordon, Paola Grandi, Massimo Petretich, Alex Phillipou, Rab K. Prinjha, Inmaculada Rioja, Peter Soden, Thilo Werner, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">GSK789: A Selective Inhibitor of the First Bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9045-9069. <a href="https://doi.org/10.1021/acs.jmedchem.0c00614" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00614</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00614%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DGSK789%25253A%252BA%252BSelective%252BInhibitor%252Bof%252Bthe%252BFirst%252BBromodomains%252B%252528BD1%252529%252Bof%252Bthe%252BBromo%252Band%252BExtra%252BTerminal%252BDomain%252B%252528BET%252529%252BProteins%26aulast%3DWatson%26aufirst%3DRobert%2BJ.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14042020%26date%3D04082020%26date%3D21072020%26volume%3D63%26issue%3D17%26spage%3D9045%26epage%3D9069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jonathan T. Seal, Stephen J. Atkinson, Helen Aylott, Paul Bamborough, Chun-wa Chung, Royston C. B. Copley, Laurie Gordon, Paola Grandi, James R. J. Gray, Lee A. Harrison, Thomas G. Hayhow, Matthew Lindon, Cassie Messenger, Anne-Marie Michon, Darren Mitchell, Alex Preston, Rab K. Prinjha, Inmaculada Rioja, Simon Taylor, Ian D. Wall, Robert J. Watson, James M. Woolven, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9093-9126. <a href="https://doi.org/10.1021/acs.jmedchem.0c00796" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00796</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00796%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252BOptimization%252Bof%252Ba%252BNovel%25252C%252BWeak%252BBromo%252Band%252BExtra%252BTerminal%252BDomain%252B%252528BET%252529%252BBromodomain%252BFragment%252BLigand%252Bto%252Ba%252BPotent%252Band%252BSelective%252BSecond%252BBromodomain%252B%252528BD2%252529%252BInhibitor%26aulast%3DSeal%26aufirst%3DJonathan%2BT.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D11052020%26date%3D30082020%26date%3D23072020%26volume%3D63%26issue%3D17%26spage%3D9093%26epage%3D9126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alex Preston, Stephen J. Atkinson, Paul Bamborough, Chun-wa Chung, Laurie J. Gordon, Paola Grandi, James R. J. Gray, Lee A. Harrison, Antonia J. Lewis, David Lugo, Cassie Messenger, Anne-Marie Michon, Darren J. Mitchell, Rab K. Prinjha, Inmaculada Rioja, Jon Seal, Simon Taylor, Pierre Thesmar, Ian D. Wall, Robert J. Watson, James M. Woolven, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">GSK973 Is an Inhibitor of the Second Bromodomains (BD2s) of the Bromodomain and Extra-Terminal (BET) Family. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (8)
                                     , 1581-1587. <a href="https://doi.org/10.1021/acsmedchemlett.0c00247" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00247</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00247%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DGSK973%252BIs%252Ban%252BInhibitor%252Bof%252Bthe%252BSecond%252BBromodomains%252B%252528BD2s%252529%252Bof%252Bthe%252BBromodomain%252Band%252BExtra-Terminal%252B%252528BET%252529%252BFamily%26aulast%3DPreston%26aufirst%3DAlex%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D11052020%26date%3D06072020%26date%3D09072020%26date%3D06072020%26volume%3D11%26issue%3D8%26spage%3D1581%26epage%3D1587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isabelle A.  Engelberg</span>, <span class="hlFld-ContribAuthor ">Caroline A.  Foley</span>, <span class="hlFld-ContribAuthor ">Lindsey I.  James</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>. </span><span class="cited-content_cbyCitation_article-title">Improved methods for targeting epigenetic reader domains of acetylated and methylated lysine. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>63 </em>, 132-144. <a href="https://doi.org/10.1016/j.cbpa.2021.03.002" title="DOI URL">https://doi.org/10.1016/j.cbpa.2021.03.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2021.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2021.03.002%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DImproved%252Bmethods%252Bfor%252Btargeting%252Bepigenetic%252Breader%252Bdomains%252Bof%252Bacetylated%252Band%252Bmethylated%252Blysine%26aulast%3DEngelberg%26aufirst%3DIsabelle%2BA.%26date%3D2021%26volume%3D63%26spage%3D132%26epage%3D144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tingting  Liu</span>, <span class="hlFld-ContribAuthor ">Shubin  Song</span>, <span class="hlFld-ContribAuthor ">Xu  Wang</span>, <span class="hlFld-ContribAuthor ">Jifu  Hao</span>. </span><span class="cited-content_cbyCitation_article-title">Small-molecule inhibitors of breast cancer-related targets: Potential therapeutic agents for breast cancer. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112954. <a href="https://doi.org/10.1016/j.ejmech.2020.112954" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112954</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112954%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall-molecule%252Binhibitors%252Bof%252Bbreast%252Bcancer-related%252Btargets%25253A%252BPotential%252Btherapeutic%252Bagents%252Bfor%252Bbreast%252Bcancer%26aulast%3DLiu%26aufirst%3DTingting%26date%3D2021%26volume%3D210%26spage%3D112954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Massimo  Petretich</span>, <span class="hlFld-ContribAuthor ">Emmanuel H.  Demont</span>, <span class="hlFld-ContribAuthor ">Paola  Grandi</span>. </span><span class="cited-content_cbyCitation_article-title">Domain-selective targeting of BET proteins in cancer and immunological diseases. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>57 </em>, 184-193. <a href="https://doi.org/10.1016/j.cbpa.2020.02.003" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.02.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.02.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DDomain-selective%252Btargeting%252Bof%252BBET%252Bproteins%252Bin%252Bcancer%252Band%252Bimmunological%252Bdiseases%26aulast%3DPetretich%26aufirst%3DMassimo%26date%3D2020%26volume%3D57%26spage%3D184%26epage%3D193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0021.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Literature BD2-selective inhibitors RVX-208, GSK340, and ABBV-744.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Profile of HTS hit <b>1</b>. Some results below the curve-fitting threshold of 4.3 could not be included in the mean. In these cases, the numbers included in the mean/number of times tested are both shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Crystal structure of the BRD4 BD1 complex with <b>1</b> (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6z7g">6z7g</a>) superimposed on the BRD2 BD1-bound structure of paracetamol (magenta, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4a9j">4a9j</a>). (B) BRD4 BD1 complex with <b>1</b>. (C) BRD2 BD2 complex with <b>1</b> showing alternative conformations (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6z7f">6z7f</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Profile of lead molecules <b>28</b> and <b>29</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route for Variance of the Amide Group<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) acetyl chloride, THF/xylene (1:2), 70 Â°C, 16 h (92%). (ii) BnBr, K<sub>2</sub>CO<sub>3</sub>, DMF, 70 Â°C, 16 h (71%). (iii) LiOH, THF/MeOH/H<sub>2</sub>O (2:2:1), rt, 16 h (83%). (iv) R<sub>1</sub>N(R<sub>2</sub>)H, HATU or T<sub>3</sub>P, DIPEA, DMF, 16 h, rt (4â67%). (v) HCl or TFA, 1,4-dioxane or CH<sub>2</sub>Cl<sub>2</sub>, 0 Â°C, 2 h (21â40%).</p></p></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Route for Variance of Shelf Group R<sub>3</sub><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) acetyl chloride, THF/xylene (1:2), 70 Â°C, 5 h (75%). (ii) Amine, EDC, HOBt, DMF 16 h, rt (56%). (iii) EDC, HOBt, TEA, DMF, rt (44%). (iv) R<sub>3</sub>Br, K<sub>2</sub>CO<sub>3</sub>, DMF, 60â70 Â°C, 2â3 h then HCl, 1,4-dioxane, rt, 3â16 h (24â69%). (v) K<sub>2</sub>CO<sub>3</sub>, DMF, 70 Â°C, 3 h then K<sub>2</sub>CO<sub>3</sub>, MeOH, 12 h, rt (40%).</p></p></figure><figure data-id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Route to <b>29</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) NaH, MeI, (<i>S</i>)-1-phenylethanol, DMF/THF (2:1), 0â25 Â°C, 22 h (74%). (ii) Pt/C, H<sub>2</sub>, EtOH, rt, 1 atm (97%). (iii) Ac<sub>2</sub>O, H<sub>2</sub>O, rt, 2 h, (81%). (iv) LiOH, THF/H<sub>2</sub>O (1:1), 50 Â°C 3 h, rt, 16 h (81%). (v) <i>trans</i>-4-aminocyclohexanol, HATU, DIPEA, DMF, rt, 18 h (55%).</p></p></figure><figure data-id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0014.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthetic Route to <b>27</b> and <b>28</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) CDI, DMF, tetrahydro-2<i>H</i>-pyran-4-amine HCl, rt, 18 h (92%). (ii) NaH, DMF, (<i>S</i>)-1-phenylethanol, rt, 2 h, (72%) for <b>44b</b> or 1 M LiHMDS in THF, THF, (<i>R</i>)-1-phenylethanol, 100 Â°C, 30 min (81%) for <b>44</b>. (iii) Sn(II)Cl, EtOH, 60 Â°C, 16 h (37% for <b>45b</b> and 39% for <b>45</b>). (iv) Ac<sub>2</sub>O, H<sub>2</sub>O, 50 Â°C, 16 h, (51% for <b>28</b> and 76% for <b>27</b>).</p></p></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Investigating and mitigating the mutagenic liabilities of potential degradants of <b>29</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Mitigating the genotoxic risk of aniline <b>47</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0015.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of 5-F-Substituted Analogues Including <b>59</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) ROH, LiHMDS, THF, rt, 30 min (99%), (ii) Fe(0), NH<sub>4</sub>Cl, EtOH, 80 Â°C, 1 h (82%). (iii) Ac<sub>2</sub>O, rt, 20 h (81%). (iv) Xantphos, Pd(OAc)<sub>2</sub>, DMAP, Co<sub>2</sub>CO<sub>8</sub>, 1,4-dioxane, ethylamine, 90 Â°C, 20 min (66%). (v) Xantphos, Pd(OAc)<sub>2</sub>, Et<sub>3</sub>N, CO, MeOH, DMF, 0 Â°C, 1 h (73%). (vi) LiOH, THF/H<sub>2</sub>O 1:1, 50 Â°C, 2 h (93%). (vii) <i>trans</i>-4-aminocyclohexanol, HATU, DIPEA, DCM, rt, 1 h (89%).</p></p></figure><figure data-id="fig7" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. BROMOscan profile of <b>59</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. <i>In vitro</i> and <i>in vivo</i> pharmacokinetic profile of <b>59</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/medium/jm0c00605_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. X-ray structures of (A) <b>59</b> (cyan, PDB 6swp) and <b>1</b> (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6z7f">6z7f</a>; mobile imidazole not shown for clarity) in BRD2 BD2. (B) <b>59</b> (orange, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6swq">6swq</a>) in BRD4 BD1. (C) <b>59</b> (cyan) with the BRD2 BD2 surface shown in blue. The surface of His433 is shown in yellow and its contacts with <b>59</b> are shown as red dashed lines. (D) <b>59</b> (orange) with the Asp144 surface shown in yellow. (E) Superimposed crystal structures of BRD2 BD2 bound to <b>59</b> (cyan, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6swp">6swp</a>) and ABBV-744 (magenta, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6e6j">6e6j</a><a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00605/20200903/images/large/jm0c00605_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00605&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i93">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24742" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24742" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 54 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>â <span class="NLM_lpage">1073</span>, <span class="refDoi">Â DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0liU2gLdgZ3EZw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappaport, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sison, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chicas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogan, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">524</span>â <span class="NLM_lpage">528</span>, <span class="refDoi">Â DOI: 10.1038/nature10334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1038%2Fnature10334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=21814200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=524-528&author=J.+Zuberauthor=J.+Shiauthor=E.+Wangauthor=A.+R.+Rappaportauthor=H.+Herrmannauthor=E.+A.+Sisonauthor=D.+Magoonauthor=J.+Qiauthor=K.+Blattauthor=M.+Wunderlichauthor=M.+J.+Taylorauthor=C.+Johnsauthor=A.+Chicasauthor=J.+C.+Mulloyauthor=S.+C.+Koganauthor=P.+Brownauthor=P.+Valentauthor=J.+E.+Bradnerauthor=S.+W.+Loweauthor=C.+R.+Vakoc&title=RNAi+screen+identifies+Brd4+as+a+therapeutic+target+in+acute+myeloid+leukaemia&doi=10.1038%2Fnature10334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span></div><div class="casAuthors">Zuber, Johannes; Shi, Junwei; Wang, Eric; Rappaport, Amy R.; Herrmann, Harald; Sison, Edward A.; Magoon, Daniel; Qi, Jun; Blatt, Katharina; Wunderlich, Mark; Taylor, Meredith J.; Johns, Christopher; Chicas, Agustin; Mulloy, James C.; Kogan, Scott C.; Brown, Patrick; Valent, Peter; Bradner, James E.; Lowe, Scott W.; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic pathways can regulate gene expression by controlling and interpreting chromatin modifications.  Cancer cells are characterized by altered epigenetic landscapes, and commonly exploit the chromatin regulatory machinery to enforce oncogenic gene expression programs.  Although chromatin alterations are, in principle, reversible and often amenable to drug intervention, the promise of targeting such pathways therapeutically has been limited by an incomplete understanding of cancer-specific dependencies on epigenetic regulators.  Here the authors describe a non-biased approach to probe epigenetic vulnerabilities in acute myeloid leukemia (AML), an aggressive hematopoietic malignancy that is often assocd. with aberrant chromatin states.  By screening a custom library of small hairpin RNAs (shRNAs) targeting known chromatin regulators in a genetically defined AML mouse model, the authors identify the protein bromodomain-contg. 4 (Brd4) as being critically required for disease maintenance.  Suppression of Brd4 using shRNAs or the small-mol. inhibitor JQ1 led to robust antileukemic effects in vitro and in vivo, accompanied by terminal myeloid differentiation and elimination of leukemia stem cells.  Similar sensitivities were obsd. in a variety of human AML cell lines and primary patient samples, revealing that JQ1 has broad activity in diverse AML subtypes.  The effects of Brd4 suppression are, at least in part, due to its role in sustaining Myc expression to promote aberrant self-renewal, which implicates JQ1 as a pharmacol. means to suppress MYC in cancer.  The authors' results establish small-mol. inhibition of Brd4 as a promising therapeutic strategy in AML and, potentially, other cancers, and highlight the utility of RNA interference (RNAi) screening for revealing epigenetic vulnerabilities that can be exploited for direct pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVyWFIqFkUVrVg90H21EOLACvtfcHk0lhIr-m_N0Jv8w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP&md5=618793a766f037bff5909524743fedb7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature10334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10334%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DHerrmann%26aufirst%3DH.%26aulast%3DSison%26aufirst%3DE.%2BA.%26aulast%3DMagoon%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DK.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DJohns%26aufirst%3DC.%26aulast%3DChicas%26aufirst%3DA.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DRNAi%2520screen%2520identifies%2520Brd4%2520as%2520a%2520therapeutic%2520target%2520in%2520acute%2520myeloid%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D524%26epage%3D528%26doi%3D10.1038%2Fnature10334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span> <span> </span><span class="NLM_article-title">Place your BETs: the therapeutic potential of bromodomains</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">146</span>â <span class="NLM_lpage">153</span>, <span class="refDoi">Â DOI: 10.1016/j.tips.2011.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.tips.2011.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=22277300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=146-153&author=R.+K.+Prinjhaauthor=J.+Witheringtonauthor=K.+Lee&title=Place+your+BETs%3A+the+therapeutic+potential+of+bromodomains&doi=10.1016%2Fj.tips.2011.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Place your BETs: the therapeutic potential of bromodomains</span></div><div class="casAuthors">Prinjha, R. K.; Witherington, J.; Lee, K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-153</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Therapeutic targeting of the processes that regulate histone modification is a growing area of scientific exploration.  Although most interest has concd. on the various families of enzymes that contribute to these processes, this review focuses on emerging data demonstrating the chem. tractability and therapeutic potential of a hitherto underexplored family of proteins, namely the bromodomain (BRD) family of reader proteins.  These proteins perform a crucial role in translating histone modifications with powerful transcriptional consequences.  We review current knowledge of the biol. of this emergent target class and highlight recent breakthroughs that now make the BRD family of reader proteins attractive for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_N9p7ru787rVg90H21EOLACvtfcHk0lhIr-m_N0Jv8w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D&md5=ea1e9e5740b1131239aa0407748d549b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPlace%2520your%2520BETs%253A%2520the%2520therapeutic%2520potential%2520of%2520bromodomains%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D146%26epage%3D153%26doi%3D10.1016%2Fj.tips.2011.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET Bromodomain Targets Genetically Diverse Glioblastoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1748</span>â <span class="NLM_lpage">1759</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-12-3066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1158%2F1078-0432.CCR-12-3066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=23403638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVCmsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=1748-1759&author=Z.+Chengauthor=Y.+Gongauthor=Y.+Maauthor=K.+Luauthor=X.+Luauthor=L.+A.+Pierceauthor=R.+C.+Thompsonauthor=S.+Mullerauthor=S.+Knappauthor=J.+Wang&title=Inhibition+of+BET+Bromodomain+Targets+Genetically+Diverse+Glioblastoma&doi=10.1158%2F1078-0432.CCR-12-3066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET Bromodomain Targets Genetically Diverse Glioblastoma</span></div><div class="casAuthors">Cheng, Zhixiang; Gong, Yuanying; Ma, Yufang; Lu, Kaihua; Lu, Xiang; Pierce, Larry A.; Thompson, Reid C.; Muller, Susanne; Knapp, Stefan; Wang, Jialiang</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1748-1759</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Glioblastoma is refractory to conventional therapies.  The bromodomain and extraterminal domain (BET) proteins are epigenetic readers that selectively bind to acetylated lysine residues on histone tails.  These proteins recently emerged as important therapeutic targets in NUT midline carcinoma and several types of hematopoietic cancers.  In this study, the therapeutic potential of a novel BET bromodomain inhibitor, JQ1, was assessed in a panel of genetically heterogeneous glioblastoma samples.  Exptl. Design: The antineoplastic effects of JQ1 were shown using ex vivo cultures derived from primary glioblastoma xenograft lines and surgical specimens of different genetic background.  The in vivo efficacy was assessed in orthotopic glioblastoma tumors.  Results: We showed that JQ1 induced marked G1 cell-cycle arrest and apoptosis, which was phenocopied by knockdown of individual BET family members.  JQ1 treatment resulted in significant changes in expression of genes that play important roles in glioblastoma such as c-Myc, p21CIP1/WAF1, hTERT, Bcl-2, and Bcl-xL.  Unlike the observations in some hematopoietic cancer cell lines, exogenous c-Myc did not significantly protect glioblastoma cells against JQ1.  In contrast, ectopically expressed Bcl-xL partially rescued cells from JQ1-induced apoptosis, and knockdown of p21CIP1/WAF1 attenuated JQ1-induced cell-cycle arrest.  Cells genetically engineered for Akt hyperactivation or p53/Rb inactivation did not compromise JQ1 efficacy, suggesting that these frequently mutated signaling pathways may not confer resistance to JQ1.  Furthermore, JQ1 significantly repressed growth of orthotopic glioblastoma tumors.  Conclusion: Our results suggest potentially broad therapeutic use of BET bromodomain inhibitors for treating genetically diverse glioblastoma tumors.  Clin Cancer Res; 19(7); 1748-59. Â©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7OsFKqlwyDbVg90H21EOLACvtfcHk0lg-OJPS-rnXIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVCmsLo%253D&md5=1722189e5d950da233b785558f2c9f58</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-3066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-3066%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DGong%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DPierce%26aufirst%3DL.%2BA.%26aulast%3DThompson%26aufirst%3DR.%2BC.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520BET%2520Bromodomain%2520Targets%2520Genetically%2520Diverse%2520Glioblastoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2013%26volume%3D19%26spage%3D1748%26epage%3D1759%26doi%3D10.1158%2F1078-0432.CCR-12-3066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Costa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanasopoulou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniere, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâSullivan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stankovic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">PFI-1, a Highly Selective Protein Interaction Inhibitor, Targeting BET Bromodomains</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">3336</span>â <span class="NLM_lpage">3346</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-12-3292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1158%2F0008-5472.CAN-12-3292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=23576556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=3336-3346&author=S.+Picaudauthor=D.+Da+Costaauthor=A.+Thanasopoulouauthor=P.+Filippakopoulosauthor=P.+V.+Fishauthor=M.+Philpottauthor=O.+Fedorovauthor=P.+Brennanauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=J.+E.+Bradnerauthor=P.+Taniereauthor=B.+O%E2%80%99Sullivanauthor=S.+M%C3%BCllerauthor=J.+Schwallerauthor=T.+Stankovicauthor=S.+Knapp&title=PFI-1%2C+a+Highly+Selective+Protein+Interaction+Inhibitor%2C+Targeting+BET+Bromodomains&doi=10.1158%2F0008-5472.CAN-12-3292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">PFI-1, a Highly Selective Protein Interaction Inhibitor, Targeting BET Bromodomains</span></div><div class="casAuthors">Picaud, Sarah; Da Costa, David; Thanasopoulou, Angeliki; Filippakopoulos, Panagis; Fish, Paul V.; Philpott, Martin; Fedorov, Oleg; Brennan, Paul; Bunnage, Mark E.; Owen, Dafydd R.; Bradner, James E.; Taniere, Philippe; O'Sullivan, Brendan; Mueller, Susanne; Schwaller, Juerg; Stankovic, Tatjana; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3336-3346</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bromo and extra terminal (BET) proteins (BRD2, BRD3, BRD4, and BRDT) are transcriptional regulators required for efficient expression of several growth promoting and antiapoptotic genes as well as for cell-cycle progression.  BET proteins are recruited on transcriptionally active chromatin via their two N-terminal bromo-domains (BRD), a protein interaction module that specifically recognizes acetylated lysine residues in histones H3 and H4.  Inhibition of the BET-histone interaction results in transcriptional downregulation of a no. of oncogenes, providing a novel pharmacol. strategy for the treatment of cancer.  Here, we present a potent and highly selective dihydroquinazoline-2-one inhibitor, PFFI-1, which efficiently blocks the interaction of BET BRDs with acetylated histone tails.  Cocrystal structures showed that PFI-1 acts as an acetyl-lysine (Kac) mimetic inhibitor efficiently occupying the Kac binding site in BRD4 and BRD2.  PFI-1 has antiproliferative effects on leukemic cell lines and efficiently abrogates their clonogenic growth.  Exposure of sensitive cell lines with PFI-I results in G1 cell-cycle arrest, downregulation of MYC expression, as well as induction of apoptosis and induces differentiation of primary leukemic blasts.  Intriguingly, cells exposed to PFI-1 showed significant downregulation of Aurora B kinase, thus attenuating phosphorylation of the Aurora substrate H3S10, providing an alternative strategy for the specific inhibition of this well-established oncol. target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFGSqZBi1UELVg90H21EOLACvtfcHk0lg-OJPS-rnXIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D&md5=4d35b56dc6e0cdbc0f3e6ccdabefca94</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-3292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-3292%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DDa%2BCosta%26aufirst%3DD.%26aulast%3DThanasopoulou%26aufirst%3DA.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTaniere%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DB.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DStankovic%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DPFI-1%252C%2520a%2520Highly%2520Selective%2520Protein%2520Interaction%2520Inhibitor%252C%2520Targeting%2520BET%2520Bromodomains%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D3336%26epage%3D3346%26doi%3D10.1158%2F0008-5472.CAN-12-3292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segura, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontanals-Cirera, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaziel-Sovran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guijarro, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanniford, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GonzÃ¡lez-Gomez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morante, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubierre, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darvishian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernando, E.</span></span> <span> </span><span class="NLM_article-title">BRD4 Sustains Melanoma Proliferation and Represents a New Target for Epigenetic Therapy</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">6264</span>â <span class="NLM_lpage">6276</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-13-0122-T</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1158%2F0008-5472.CAN-13-0122-T" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=23950209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6264-6276&author=M.+F.+Seguraauthor=B.+Fontanals-Cireraauthor=A.+Gaziel-Sovranauthor=M.+V.+Guijarroauthor=D.+Hannifordauthor=G.+Zhangauthor=P.+Gonz%C3%A1lez-Gomezauthor=M.+Moranteauthor=L.+Jubierreauthor=W.+Zhangauthor=F.+Darvishianauthor=M.+Ohlmeyerauthor=I.+Osmanauthor=M.-M.+Zhouauthor=E.+Hernando&title=BRD4+Sustains+Melanoma+Proliferation+and+Represents+a+New+Target+for+Epigenetic+Therapy&doi=10.1158%2F0008-5472.CAN-13-0122-T"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Sustains Melanoma Proliferation and Represents a New Target for Epigenetic Therapy</span></div><div class="casAuthors">Segura, Miguel F.; Fontanals-Cirera, Barbara; Gaziel-Sovran, Avital; Guijarro, Maria V.; Hanniford, Doug; Zhang, Guangtao; Gonzalez-Gomez, Pilar; Morante, Marta; Jubierre, Luz; Zhang, Weijia; Darvishian, Farbod; Ohlmeyer, Michael; Osman, Iman; Zhou, Ming-Ming; Hernando, Eva</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6264-6276</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Metastatic melanoma remains a mostly incurable disease.  Although newly approved targeted therapies are efficacious in a subset of patients, resistance and relapse rapidly ensue.  Alternative therapeutic strategies to manipulate epigenetic regulators and disrupt the transcriptional program that maintains tumor cell identity are emerging.  Bromodomain and extraterminal domain (BET) proteins are epigenome readers known to exert key roles at the interface between chromatin remodeling and transcriptional regulation.  Here, we report that BRD4, a BET family member, is significantly upregulated in primary and metastatic melanoma tissues compared with melanocytes and nevi.  Treatment with BET inhibitors impaired melanoma cell proliferation in vitro and tumor growth and metastatic behavior in vivo, effects that were mostly recapitulated by individual silencing of BRD4.  RNA sequencing of BET inhibitor-treated cells followed by Gene Ontol. anal. showed a striking impact on transcriptional programs controlling cell growth, proliferation, cell-cycle regulation, and differentiation.  In particular, we found that, rapidly after BET displacement, key cell-cycle genes (SKP2, ERK1, and c-MYC) were downregulated concomitantly with the accumulation of cyclin-dependent kinase (CDK) inhibitors (p21 and p27), followed by cell-cycle arrest.  Importantly, BET inhibitor efficacy was not influenced by BRAF or NRAS mutational status, opening the possibility of using these small-mol. compds. to treat patients for whom no effective targeted therapy exists.  Collectively, our study reveals a crit. role for BRD4 in melanoma tumor maintenance and renders it a legitimate and novel target for epigenetic therapy directed against the core transcriptional program of melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq80TZH2ueQFbVg90H21EOLACvtfcHk0lg-OJPS-rnXIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7N&md5=a4a77f188de01c4c84f7293a32478fc7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0122-T&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0122-T%26sid%3Dliteratum%253Aachs%26aulast%3DSegura%26aufirst%3DM.%2BF.%26aulast%3DFontanals-Cirera%26aufirst%3DB.%26aulast%3DGaziel-Sovran%26aufirst%3DA.%26aulast%3DGuijarro%26aufirst%3DM.%2BV.%26aulast%3DHanniford%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DGonz%25C3%25A1lez-Gomez%26aufirst%3DP.%26aulast%3DMorante%26aufirst%3DM.%26aulast%3DJubierre%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DDarvishian%26aufirst%3DF.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DOsman%26aufirst%3DI.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DHernando%26aufirst%3DE.%26atitle%3DBRD4%2520Sustains%2520Melanoma%2520Proliferation%2520and%2520Represents%2520a%2520New%2520Target%2520for%2520Epigenetic%2520Therapy%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D6264%26epage%3D6276%26doi%3D10.1158%2F0008-5472.CAN-13-0122-T" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaidos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouvedenou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marigo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhry, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotolo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Mahdi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahemtulla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleijnen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karadimitris, A.</span></span> <span> </span><span class="NLM_article-title">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">697</span>â <span class="NLM_lpage">705</span>, <span class="refDoi">Â DOI: 10.1182/blood-2013-01-478420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1182%2Fblood-2013-01-478420" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=24335499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2014&pages=697-705&author=A.+Chaidosauthor=V.+Caputoauthor=K.+Gouvedenouauthor=B.+Liuauthor=I.+Marigoauthor=M.+S.+Chaudhryauthor=A.+Rotoloauthor=D.+F.+Toughauthor=N.+N.+Smithersauthor=A.+K.+Bassilauthor=T.+D.+Chapmanauthor=N.+R.+Harkerauthor=O.+Barbashauthor=P.+Tumminoauthor=N.+Al-Mahdiauthor=A.+C.+Haynesauthor=L.+Cutlerauthor=B.+Leauthor=A.+Rahemtullaauthor=I.+Robertsauthor=M.+Kleijnenauthor=J.+J.+Witheringtonauthor=N.+J.+Parrauthor=R.+K.+Prinjhaauthor=A.+Karadimitris&title=Potent+antimyeloma+activity+of+the+novel+bromodomain+inhibitors+I-BET151+and+I-BET762&doi=10.1182%2Fblood-2013-01-478420"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span></div><div class="casAuthors">Chaidos, Aristeidis; Caputo, Valentina; Gouvedenou, Katerina; Liu, Binbin; Marigo, Ilaria; Chaudhry, Mohammed Suhail; Rotolo, Antonia; Tough, David F.; Smithers, Nicholas N.; Bassil, Anna K.; Chapman, Trevor D.; Harker, Nicola R.; Barbash, Olena; Tummino, Peter; Al-Mahdi, Niam; Haynes, Andrea C.; Cutler, Leanne; Le, Bao Chau; Rahemtulla, Amin; Roberts, Irene; Kleijnen, Maurits; Witherington, Jason J.; Parr, Nigel J.; Prinjha, Rab K.; Karadimitris, Anastasios</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">697-705</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD2-4 inhibitors hold therapeutic promise in preclin. models of hematol. malignancies.  However, translation of these data to mols. suitable for clin. development has yet to be accomplished.  Herein we expand the mechanistic understanding of BET inhibitors in multiple myeloma by using the chem. probe mol. I-BET151.  I-BET151 induces apoptosis and exerts strong antiproliferative effect in vitro and in vivo.  This is assocd. with contrasting effects on oncogenic MYC and HEXIM1, an inhibitor of the transcriptional activator P-TEFb.  I-BET151 causes transcriptional repression of MYC and MYC-dependent programs by abrogating recruitment to the chromatin of the P-TEFb component CDK9 in a BRD2-4-dependent manner.  In contrast, transcriptional upregulation of HEXIM1 is BRD2-4 independent.  Finally, preclin. studies show that I-BET762 has a favorable pharmacol. profile as an oral agent and that it inhibits myeloma cell proliferation, resulting in survival advantage in a systemic myeloma xenograft model.  These data provide a strong rationale for extending the clin. testing of the novel antimyeloma agent I-BET762 and reveal insights into biol. pathways required for myeloma cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0bNtpJQBXsrVg90H21EOLACvtfcHk0lgziztNk6C3_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D&md5=f8f734f84a07d3b86ebdc264f6741b22</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-01-478420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-01-478420%26sid%3Dliteratum%253Aachs%26aulast%3DChaidos%26aufirst%3DA.%26aulast%3DCaputo%26aufirst%3DV.%26aulast%3DGouvedenou%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DMarigo%26aufirst%3DI.%26aulast%3DChaudhry%26aufirst%3DM.%2BS.%26aulast%3DRotolo%26aufirst%3DA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DTummino%26aufirst%3DP.%26aulast%3DAl-Mahdi%26aufirst%3DN.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLe%26aufirst%3DB.%26aulast%3DRahemtulla%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DI.%26aulast%3DKleijnen%26aufirst%3DM.%26aulast%3DWitherington%26aufirst%3DJ.%2BJ.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DKaradimitris%26aufirst%3DA.%26atitle%3DPotent%2520antimyeloma%2520activity%2520of%2520the%2520novel%2520bromodomain%2520inhibitors%2520I-BET151%2520and%2520I-BET762%26jtitle%3DBlood%26date%3D2014%26volume%3D123%26spage%3D697%26epage%3D705%26doi%3D10.1182%2Fblood-2013-01-478420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasconi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarantelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascione, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponzoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mensah, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stussi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inghirami, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">The BET Bromodomain Inhibitor OTX015 Affects Pathogenetic Pathways in Preclinical B-cell Tumor Models and Synergizes with Targeted Drugs</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1628</span>â <span class="NLM_lpage">1638</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-14-1561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1158%2F1078-0432.CCR-14-1561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=25623213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1628-1638&author=M.+Boiauthor=E.+Gaudioauthor=P.+Bonettiauthor=I.+Kweeauthor=E.+Bernasconiauthor=C.+Tarantelliauthor=A.+Rinaldiauthor=M.+Testoniauthor=L.+Cascioneauthor=M.+Ponzoniauthor=A.+A.+Mensahauthor=A.+Stathisauthor=G.+Stussiauthor=M.+E.+Riveiroauthor=P.+Heraitauthor=G.+Inghiramiauthor=E.+Cvitkovicauthor=E.+Zuccaauthor=F.+Bertoni&title=The+BET+Bromodomain+Inhibitor+OTX015+Affects+Pathogenetic+Pathways+in+Preclinical+B-cell+Tumor+Models+and+Synergizes+with+Targeted+Drugs&doi=10.1158%2F1078-0432.CCR-14-1561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The BET Bromodomain Inhibitor OTX015 Affects Pathogenetic Pathways in Preclinical B-cell Tumor Models and Synergizes with Targeted Drugs</span></div><div class="casAuthors">Boi, Michela; Gaudio, Eugenio; Bonetti, Paola; Kwee, Ivo; Bernasconi, Elena; Tarantelli, Chiara; Rinaldi, Andrea; Testoni, Monica; Cascione, Luciano; Ponzoni, Maurilio; Mensah, Afua Adjeiwaa; Stathis, Anastasios; Stussi, Georg; Riveiro, Maria Eugenia; Herait, Patrice; Inghirami, Giorgio; Cvitkovic, Esteban; Zucca, Emanuele; Bertoni, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1628-1638</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">In cancer cells, the epigenome is often deregulated, and inhibition of the bromodomain and extra-terminal (BET) family of bromodomain-contg. proteins is a novel epigenetic therapeutic approach.  Preliminary results of an ongoing phase I trial have reported promising activity and tolerability with the new BET bromodomain inhibitor OTX015.  We assessed the preclin. activity of OTX015 as single agent and in combination in mature B-cell lymphoma models and performed in vitro and in vivo expts. to identify the mechanism of action and the genetic features assocd. with sensitivity to the compd.  OTX015 showed antiproliferative activity in a large panel of cell lines derived from mature B-cell lymphoid tumors with median IC50 of 240 nmol/L, without significant differences among the different histotypes.  In vitro and in vivo expts. showed that OTX015 targeted NFKB/TLR/JAK/STAT signaling pathways, MYC- and E2F1-regulated genes, cell-cycle regulation, and chromatin structure.  OTX015 presented in vitro synergism with several anticancer agents, esp. with mTOR and BTK inhibitors.  Gene expression signatures assocd. with different degrees of sensitivity to OTX015 were identified.  Although OTX015 was mostly cytostatic, the compd. induced apoptosis in a genetically defined subgroup of cells, derived from activated B-cell-like diffuse large B-cell lymphoma, bearing wtTP53, mutations in MYD88, and CD79B or CARD11.  Together with the data coming from the ongoing phase I study, the in vitro and in vivo data presented here provide the basis for further clin. investigation of OTX015 as single agent and in combination therapies.  Clin Cancer Res; 21(7); 1628-38. Â©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJcy3O1ElDJ7Vg90H21EOLACvtfcHk0lgziztNk6C3_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D&md5=5742b4d5884261d546588a339f3c2ea0</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-1561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-1561%26sid%3Dliteratum%253Aachs%26aulast%3DBoi%26aufirst%3DM.%26aulast%3DGaudio%26aufirst%3DE.%26aulast%3DBonetti%26aufirst%3DP.%26aulast%3DKwee%26aufirst%3DI.%26aulast%3DBernasconi%26aufirst%3DE.%26aulast%3DTarantelli%26aufirst%3DC.%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DTestoni%26aufirst%3DM.%26aulast%3DCascione%26aufirst%3DL.%26aulast%3DPonzoni%26aufirst%3DM.%26aulast%3DMensah%26aufirst%3DA.%2BA.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DStussi%26aufirst%3DG.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DInghirami%26aufirst%3DG.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DThe%2520BET%2520Bromodomain%2520Inhibitor%2520OTX015%2520Affects%2520Pathogenetic%2520Pathways%2520in%2520Preclinical%2520B-cell%2520Tumor%2520Models%2520and%2520Synergizes%2520with%2520Targeted%2520Drugs%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D1628%26epage%3D1638%26doi%3D10.1158%2F1078-0432.CCR-14-1561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">da
Motta, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ledaki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purshouse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bastiani, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baban, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morotti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigfield, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klamt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, A.</span></span> <span> </span><span class="NLM_article-title">The BET inhibitor JQ1 selectively impairs tumour response to hypoxia and downregulates CA9 and angiogenesis in triple negative breast cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">122</span>â <span class="NLM_lpage">132</span>, <span class="refDoi">Â DOI: 10.1038/onc.2016.184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1038%2Fonc.2016.184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=27292261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC28Xps1ejtr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=122-132&author=L.+L.+da%0AMottaauthor=I.+Ledakiauthor=K.+Purshouseauthor=S.+Haiderauthor=M.+A.+De+Bastianiauthor=D.+Babanauthor=M.+Morottiauthor=G.+Steersauthor=S.+Wigfieldauthor=E.+Bridgesauthor=J.+L.+Liauthor=S.+Knappauthor=D.+Ebnerauthor=F.+Klamtauthor=A.+L.+Harrisauthor=A.+McIntyre&title=The+BET+inhibitor+JQ1+selectively+impairs+tumour+response+to+hypoxia+and+downregulates+CA9+and+angiogenesis+in+triple+negative+breast+cancer&doi=10.1038%2Fonc.2016.184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The BET inhibitor JQ1 selectively impairs tumour response to hypoxia and downregulates CA9 and angiogenesis in triple negative breast cancer</span></div><div class="casAuthors">da Motta, L. L.; Ledaki, I.; Purshouse, K.; Haider, S.; De Bastiani, M. A.; Baban, D.; Morotti, M.; Steers, G.; Wigfield, S.; Bridges, E.; Li, J.-L.; Knapp, S.; Ebner, D.; Klamt, F.; Harris, A. L.; McIntyre, A.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-132</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The availability of bromodomain and extra-terminal inhibitors (BETi) has enabled translational epigenetic studies in cancer.  BET proteins regulate transcription by selectively recognizing acetylated lysine residues on chromatin.  BETi compete with this process leading to both downregulation and upregulation of gene expression.  Hypoxia enables progression of triple neg. breast cancer (TNBC), the most aggressive form of breast cancer, partly by driving metabolic adaptation, angiogenesis and metastasis through upregulation of hypoxia-regulated genes (for example, carbonic anhydrase 9 (CA9) and vascular endothelial growth factor A (VEGF-A).  Responses to hypoxia can be mediated epigenetically, thus we investigated whether BETi JQ1 could impair the TNBC response induced by hypoxia and exert anti-tumor effects.  JQ1 significantly modulated 44% of hypoxia-induced genes, of which two-thirds were downregulated including CA9 and VEGF-A.  JQ1 prevented HIF binding to the hypoxia response element in CA9 promoter, but did not alter HIF expression or activity, suggesting some HIF targets are BET-dependent.  JQ1 reduced TNBC growth in vitro and in vivo and inhibited xenograft vascularization.  These findings identify that BETi dually targets angiogenesis and the hypoxic response, an effective combination at reducing tumor growth in preclin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq79-F0o2tp6bVg90H21EOLACvtfcHk0ljV-FNN1vU6gw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xps1ejtr4%253D&md5=9f1592548db8e6306703bdab7e1e1c90</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.184%26sid%3Dliteratum%253Aachs%26aulast%3Dda%2BMotta%26aufirst%3DL.%2BL.%26aulast%3DLedaki%26aufirst%3DI.%26aulast%3DPurshouse%26aufirst%3DK.%26aulast%3DHaider%26aufirst%3DS.%26aulast%3DDe%2BBastiani%26aufirst%3DM.%2BA.%26aulast%3DBaban%26aufirst%3DD.%26aulast%3DMorotti%26aufirst%3DM.%26aulast%3DSteers%26aufirst%3DG.%26aulast%3DWigfield%26aufirst%3DS.%26aulast%3DBridges%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DJ.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DEbner%26aufirst%3DD.%26aulast%3DKlamt%26aufirst%3DF.%26aulast%3DHarris%26aufirst%3DA.%2BL.%26aulast%3DMcIntyre%26aufirst%3DA.%26atitle%3DThe%2520BET%2520inhibitor%2520JQ1%2520selectively%2520impairs%2520tumour%2520response%2520to%2520hypoxia%2520and%2520downregulates%2520CA9%2520and%2520angiogenesis%2520in%2520triple%2520negative%2520breast%2520cancer%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D122%26epage%3D132%26doi%3D10.1038%2Fonc.2016.184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rhyasen, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dulak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boiko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castriotta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahdesmaki, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâConnor, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dry, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">AZD5153: A Novel Bivalent BET Bromodomain Inhibitor Highly Active against Hematologic Malignancies</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2563</span>â <span class="NLM_lpage">2574</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.MCT-16-0141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1158%2F1535-7163.MCT-16-0141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=27573426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2563-2574&author=G.+W.+Rhyasenauthor=M.+M.+Hattersleyauthor=Y.+Yaoauthor=A.+Dulakauthor=W.+Wangauthor=P.+Petterutiauthor=I.+L.+Daleauthor=S.+Boikoauthor=T.+Cheungauthor=J.+Zhangauthor=S.+Wenauthor=L.+Castriottaauthor=D.+Lawsonauthor=M.+Collinsauthor=L.+Baoauthor=M.+J.+Ahdesmakiauthor=G.+Walkerauthor=G.+O%E2%80%99Connorauthor=T.+C.+Yehauthor=A.+A.+Rabowauthor=J.+R.+Dryauthor=C.+Reimerauthor=P.+Lyneauthor=G.+B.+Millsauthor=S.+E.+Fawellauthor=M.+J.+Waringauthor=M.+Zindaauthor=E.+Clarkauthor=H.+Chen&title=AZD5153%3A+A+Novel+Bivalent+BET+Bromodomain+Inhibitor+Highly+Active+against+Hematologic+Malignancies&doi=10.1158%2F1535-7163.MCT-16-0141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">AZD5153: a novel bivalent BET bromodomain inhibitor highly active against hematologic malignancies</span></div><div class="casAuthors">Rhyasen, Garrett W.; Hattersley, Maureen M.; Yao, Yi; Dulak, Austin; Wang, Wenxian; Petteruti, Philip; Dale, Ian L.; Boiko, Scott; Cheung, Tony; Zhang, Jingwen; Wen, Shenghua; Castriotta, Lillian; Lawson, Deborah; Collins, Michael; Bao, Larry; Ahdesmaki, Miika J.; Walker, Graeme; O'Connor, Greg; Yeh, Tammie C.; Rabow, Alfred A.; Dry, Jonathan R.; Reimer, Corinne; Lyne, Paul; Mills, Gordon B.; Fawell, Stephen E.; Waring, Michael J.; Zinda, Michael; Clark, Edwin; Chen, Huawei</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2563-2574</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD4 regulates gene expression via recruitment of transcriptional regulatory complexes to acetylated chromatin.  Pharmacol. targeting of BRD4 bromodomains by small mol. inhibitors has proven to be an effective means to disrupt aberrant transcriptional programs crit. for tumor growth and/or survival.  Herein, we report AZD5153, a potent, selective, and orally available BET/BRD4 bromodomain inhibitor possessing a bivalent binding mode.  Unlike previously described monovalent inhibitors, AZD5153 ligates two bromodomains in BRD4 simultaneously.  The enhanced avidity afforded through bivalent binding translates into increased cellular and antitumor activity in preclin. hematol. tumor models.  In vivo administration of AZD5153 led to tumor stasis or regression in multiple xenograft models of acute myeloid leukemia, multiple myeloma, and diffuse large B-cell lymphoma.  The relationship between AZD5153 exposure and efficacy suggests that prolonged BRD4 target coverage is a primary efficacy driver.  AZD5153 treatment markedly affects transcriptional programs of MYC, E2F, and mTOR.  Of note, mTOR pathway modulation is assocd. with cell line sensitivity to AZD5153.  Transcriptional modulation of MYC and HEXIM1 was confirmed in AZD5153-treated human whole blood, thus supporting their use as clin. pharmacodynamic biomarkers.  This study establishes AZD5153 as a highly potent, orally available BET/BRD4 inhibitor and provides a rationale for clin. development in hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAiKYOFhro1bVg90H21EOLACvtfcHk0ljV-FNN1vU6gw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D&md5=8d0ac899d081705aaa12d49fd90165a3</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0141%26sid%3Dliteratum%253Aachs%26aulast%3DRhyasen%26aufirst%3DG.%2BW.%26aulast%3DHattersley%26aufirst%3DM.%2BM.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DDulak%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DDale%26aufirst%3DI.%2BL.%26aulast%3DBoiko%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DS.%26aulast%3DCastriotta%26aufirst%3DL.%26aulast%3DLawson%26aufirst%3DD.%26aulast%3DCollins%26aufirst%3DM.%26aulast%3DBao%26aufirst%3DL.%26aulast%3DAhdesmaki%26aufirst%3DM.%2BJ.%26aulast%3DWalker%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DG.%26aulast%3DYeh%26aufirst%3DT.%2BC.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DDry%26aufirst%3DJ.%2BR.%26aulast%3DReimer%26aufirst%3DC.%26aulast%3DLyne%26aufirst%3DP.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DFawell%26aufirst%3DS.%2BE.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DAZD5153%253A%2520A%2520Novel%2520Bivalent%2520BET%2520Bromodomain%2520Inhibitor%2520Highly%2520Active%2520against%2520Hematologic%2520Malignancies%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2563%26epage%3D2574%26doi%3D10.1158%2F1535-7163.MCT-16-0141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Epigenetic drug discovery: breaking through the immune barrier</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">835</span>â <span class="NLM_lpage">853</span>, <span class="refDoi">Â DOI: 10.1038/nrd.2016.185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1038%2Fnrd.2016.185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=27765940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWhsLnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=835-853&author=D.+F.+Toughauthor=P.+P.+Takauthor=A.+Tarakhovskyauthor=R.+K.+Prinjha&title=Epigenetic+drug+discovery%3A+breaking+through+the+immune+barrier&doi=10.1038%2Fnrd.2016.185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic drug discovery: breaking through the immune barrier</span></div><div class="casAuthors">Tough, David F.; Tak, Paul P.; Tarakhovsky, Alexander; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">835-853</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Immune-mediated diseases are clin. heterogeneous but they share genetic and pathogenic mechanisms.  These diseases may develop from the interplay of genetic factors and environmental or lifestyle factors.  Exposure to such factors, including infectious agents, is assocd. with coordinated changes in gene transcription owing to epigenetic alterations.  A growing understanding of how epigenetic mechanisms control gene expression patterns and cell function has been aided by the development of small-mol. inhibitors that target these processes.  These chem. tools have helped to reveal the importance of epigenetics in guiding cell fate decisions during immune responses and have also highlighted the potential for targeting epigenetic mechanisms for the treatment of inflammation and immune-mediated diseases.  In this Review, we discuss the most advanced areas of epigenetic drug development for autoimmune and inflammatory diseases and summarize the promising preclin. data in this exciting and evolving field.  These agents will inevitably begin to move into clin. trials for use in patients with immune-mediated diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplOyEVMzBi37Vg90H21EOLACvtfcHk0lj_bHe8Q6uw1Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWhsLnE&md5=f939bcbbe3d74ce24718eef706aae3d1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.185%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DEpigenetic%2520drug%2520discovery%253A%2520breaking%2520through%2520the%2520immune%2520barrier%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D835%26epage%3D853%26doi%3D10.1038%2Fnrd.2016.185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>â <span class="NLM_lpage">1123</span>, <span class="refDoi">Â DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-w.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lj_bHe8Q6uw1Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bandukwala, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamperti, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molesworth, A. M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">14532</span>â <span class="NLM_lpage">14537</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1212264109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1073%2Fpnas.1212264109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=22912406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14532-14537&author=H.+S.+Bandukwalaauthor=J.+Gagnonauthor=S.+Togherauthor=J.+A.+Greenbaumauthor=E.+D.+Lampertiauthor=N.+J.+Parrauthor=A.+M.+H.+Molesworthauthor=N.+Smithersauthor=K.+Leeauthor=J.+Witheringtonauthor=D.+F.+Toughauthor=R.+K.+Prinjhaauthor=B.+Petersauthor=A.+Rao&title=Selective+inhibition+of+CD4%2B+T-cell+cytokine+production+and+autoimmunity+by+BET+protein+and+c-Myc+inhibitors&doi=10.1073%2Fpnas.1212264109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span></div><div class="casAuthors">Bandukwala, Hozefa S.; Gagnon, John; Togher, Susan; Greenbaum, Jason A.; Lamperti, Edward D.; Parr, Nigel J.; Molesworth, Amy M. H.; Smithers, Nicholas; Lee, Kevin; Witherington, Jason; Tough, David F.; Prinjha, Rab K.; Peters, Bjoern; Rao, Anjana</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14532-14537, S14532/1-S14532/21</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins bind acetylated lysine residues on histone tails and are involved in the recruitment of addnl. factors that mediate histone modifications and enable transcription.  A compd., I-BET-762, that inhibits binding of an acetylated histone peptide to proteins of the bromodomain and extra-terminal domain (BET) family, was previously shown to suppress the prodn. of proinflammatory proteins by macrophages and block acute inflammation in mice.  Here, we investigated the effect of short-term treatment with I-BET-762 on T-cell function.  Treatment of naive CD4+ T cells with I-BET-762 during the first 2 d of differentiation had long-lasting effects on subsequent gene expression and cytokine prodn.  Gene expression anal. revealed up-regulated expression of several antiinflammatory gene products, including IL-10, Lag3, and Egr2, and down-regulated expression of several proinflammatory cytokines including GM-CSF and IL-17.  The short 2-d treatment with I-BET-762 inhibited the ability of antigen-specific T cells, differentiated under Th1 but not Th17 conditions in vitro, to induce pathogenesis in an adoptive transfer model of exptl. autoimmune encephalomyelitis.  The suppressive effects of I-BET-762 on T-cell mediated inflammation in vivo were accompanied by decreased recruitment of macrophages, consistent with decreased GM-CSF prodn. by CNS-infiltrating T cells.  These effects were mimicked by an inhibitor of c-myc function, implicating reduced expression of c-myc and GM-CSF as one avenue by which I-BET-762 suppresses the inflammatory functions of T cells.  Our study demonstrates that inhibiting the functions of BET-family proteins during early T-cell differentiation causes long-lasting suppression of the proinflammatory functions of Th1 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5akF95GW7K7Vg90H21EOLACvtfcHk0lj_bHe8Q6uw1Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO&md5=110cd83c79e452cb99d01ad902506a5b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1212264109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1212264109%26sid%3Dliteratum%253Aachs%26aulast%3DBandukwala%26aufirst%3DH.%2BS.%26aulast%3DGagnon%26aufirst%3DJ.%26aulast%3DTogher%26aufirst%3DS.%26aulast%3DGreenbaum%26aufirst%3DJ.%2BA.%26aulast%3DLamperti%26aufirst%3DE.%2BD.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DMolesworth%26aufirst%3DA.%2BM.%2BH.%26aulast%3DSmithers%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPeters%26aufirst%3DB.%26aulast%3DRao%26aufirst%3DA.%26atitle%3DSelective%2520inhibition%2520of%2520CD4%252B%2520T-cell%2520cytokine%2520production%2520and%2520autoimmunity%2520by%2520BET%2520protein%2520and%2520c-Myc%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D14532%26epage%3D14537%26doi%3D10.1073%2Fpnas.1212264109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolajczyk, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>190</i></span>,  <span class="NLM_fpage">3670</span>â <span class="NLM_lpage">3678</span>, <span class="refDoi">Â DOI: 10.4049/jimmunol.1202838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.4049%2Fjimmunol.1202838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=23420887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=190&publication_year=2013&pages=3670-3678&author=A.+C.+Belkinaauthor=B.+S.+Nikolajczykauthor=G.+V.+Denis&title=BET+Protein+Function+Is+Required+for+Inflammation%3A+Brd2+Genetic+Disruption+and+BET+Inhibitor+JQ1+Impair+Mouse+Macrophage+Inflammatory+Responses&doi=10.4049%2Fjimmunol.1202838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span></div><div class="casAuthors">Belkina, Anna C.; Nikolajczyk, Barbara S.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">190</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3670-3678</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Histone acetylation regulates activation and repression of multiple inflammatory genes known to play crit. roles in chronic inflammatory diseases.  However, proteins responsible for translating the histone acetylation code into an orchestrated proinflammatory cytokine response remain poorly characterized.  Bromodomain and extraterminal (BET) proteins are "readers" of histone acetylation marks, with demonstrated roles in gene transcription, but the ability of BET proteins to coordinate the response of inflammatory cytokine genes through translation of histone marks is unknown.  We hypothesize that members of the BET family of dual bromodomain-contg. transcriptional regulators directly control inflammatory genes.  We examd. the genetic model of brd2 lo mice, a BET protein hypomorph, to show that Brd2 is essential for proinflammatory cytokine prodn. in macrophages.  Studies that use small interfering RNA knockdown and a small-mol. inhibitor of BET protein binding, JQ1, independently demonstrate BET proteins are crit. for macrophage inflammatory responses.  Furthermore, we show that Brd2 and Brd4 phys. assoc. with the promoters of inflammatory cytokine genes in macrophages.  This assocn. is absent in the presence of BET inhibition by JQ1.  Finally, we demonstrate that JQ1 ablates cytokine prodn. in vitro and blunts the "cytokine storm" in endotoxemic mice by reducing levels of IL-6 and TNF-Î± while rescuing mice from LPS-induced death.  We propose that targeting BET proteins with small-mol. inhibitors will benefit hyperinflammatory conditions assocd. with high levels of cytokine prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrbebEe0G277Vg90H21EOLACvtfcHk0liB07tdupm-lA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D&md5=8c4f69177b9ed814a7db339a8e2ab7ed</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1202838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1202838%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DNikolajczyk%26aufirst%3DB.%2BS.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520Protein%2520Function%2520Is%2520Required%2520for%2520Inflammation%253A%2520Brd2%2520Genetic%2520Disruption%2520and%2520BET%2520Inhibitor%2520JQ1%2520Impair%2520Mouse%2520Macrophage%2520Inflammatory%2520Responses%26jtitle%3DJ.%2520Immunol.%26date%3D2013%26volume%3D190%26spage%3D3670%26epage%3D3678%26doi%3D10.4049%2Fjimmunol.1202838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mele, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmeron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition suppresses TH17-mediated pathology</span>. <i>J. Exp. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>210</i></span>,  <span class="NLM_fpage">2181</span>â <span class="NLM_lpage">2190</span>, <span class="refDoi">Â DOI: 10.1084/jem.20130376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1084%2Fjem.20130376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=24101376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ygsLbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=210&publication_year=2013&pages=2181-2190&author=D.+A.+Meleauthor=A.+Salmeronauthor=S.+Ghoshauthor=H.-R.+Huangauthor=B.+M.+Bryantauthor=J.+M.+Lora&title=BET+bromodomain+inhibition+suppresses+TH17-mediated+pathology&doi=10.1084%2Fjem.20130376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition suppresses TH17-mediated pathology</span></div><div class="casAuthors">Mele, Deanna A.; Salmeron, Andres; Ghosh, Srimoyee; Huang, Hon-Ren; Bryant, Barbara M.; Lora, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">210</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2181-2190</span>CODEN:
                <span class="NLM_cas:coden">JEMEAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-1007</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Interleukin (IL) 17-producing T helper (TH17) cells have been selected through evolution for their ability to control fungal and bacterial infections.  It is also firmly established that their aberrant generation and activation results in autoimmune conditions.  Using a characterized potent and selective small mol. inhibitor, we show that the bromodomain and extra-terminal domain (BET) family of chromatin adaptors plays fundamental and selective roles in human and murine TH17 differentiation from naive CD4+ T cells, as well as in the activation of previously differentiated TH17 cells.  We provide evidence that BET controls TH17 differentiation in a bromodomain-dependent manner through a mechanism that includes the direct regulation of multiple effector TH17-assocd. cytokines, including IL17, IL21, and GMCSF.  We also demonstrate that BET family members Brd2 and Brd4 assoc. with the Il17 locus in TH17 cells, and that this assocn. requires bromodomains.  We recapitulate the crit. role of BET bromodomains in TH17 differentiation in vivo and show that therapeutic dosing of the BET inhibitor is efficacious in mouse models of autoimmunity.  Our results identify the BET family of proteins as a fundamental link between chromatin signaling and TH17 biol., and support the notion of BET inhibition as a point of therapeutic intervention in autoimmune conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBqVtggp2Fa7Vg90H21EOLACvtfcHk0liB07tdupm-lA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ygsLbF&md5=e6822f456cf42cdead9779afa92c05e0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1084%2Fjem.20130376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20130376%26sid%3Dliteratum%253Aachs%26aulast%3DMele%26aufirst%3DD.%2BA.%26aulast%3DSalmeron%26aufirst%3DA.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26atitle%3DBET%2520bromodomain%2520inhibition%2520suppresses%2520TH17-mediated%2520pathology%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2013%26volume%3D210%26spage%3D2181%26epage%3D2190%26doi%3D10.1084%2Fjem.20130376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">BET Inhibitor JQ1 Blocks Inflammation and Bone Destruction</span>. <i>J. Dent. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">657</span>â <span class="NLM_lpage">662</span>, <span class="refDoi">Â DOI: 10.1177/0022034514534261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1177%2F0022034514534261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=24799421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWhsL%252FI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2014&pages=657-662&author=S.+Mengauthor=L.+Zhangauthor=Y.+Tangauthor=Q.+Tuauthor=L.+Zhengauthor=L.+Yuauthor=D.+Murrayauthor=J.+Chengauthor=S.+H.+Kimauthor=X.+Zhouauthor=J.+Chen&title=BET+Inhibitor+JQ1+Blocks+Inflammation+and+Bone+Destruction&doi=10.1177%2F0022034514534261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitor JQ1 blocks inflammation and bone destruction</span></div><div class="casAuthors">Meng, S.; Zhang, L.; Tang, Y.; Tu, Q.; Zheng, L.; Yu, L.; Murray, D.; Cheng, J.; Kim, S. H.; Zhou, X.; Chen, J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Dental Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">657-662</span>CODEN:
                <span class="NLM_cas:coden">JDREAF</span>;
        ISSN:<span class="NLM_cas:issn">0022-0345</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">BET proteins are a group of epigenetic regulators controlling transcription through reading acetylated histone tails and recruiting transcription complexes.  They are considered as potential therapeutic targets in many distinct diseases.  A novel synthetic bromodomain and extraterminal domain (BET) inhibitor, JQ1, was proved to suppress oncogene transcription and inflammatory responses.  The present study was aimed to investigate the effects of JQ1 on inflammatory response and bone destruction in exptl. periodontitis.  We found that JQ1 significantly suppressed lipopolysaccharide (LPS)-stimulated inflammatory cytokine transcription, including interleukin (IL)-1Î², IL-6, and tumor necrosis factor alpha (TNF-Î±), as well as receptor activator of nuclear factor kappa- B ligand (RANKL)-induced osteoclast markers, such as c-Fos, nuclear factor of activated T-cells, cytoplasmic, calcineurin- dependent 1 (NFATc1), tartrate-resistant acid phosphatase (TRAP) and cathepsin K in vitro.  JQ1 also inhibited toll-like receptors 2/4 (TLR2/4) expression and nuclear factor kappa-light-chain-enhancer of activated B cells (NF-ÎºB) phosphorylation and nuclear translocation.  Chromatin immunopptn. and quant. polymerase chain reaction (ChIP-qPCR) revealed that JQ1 neutralized BRD4 enrichment at several gene promoter regions, including NF-ÎºB, TNF-Î±, c-Fos, and NFATc1.  In a murine periodontitis model, systemic administration of JQ1 significantly inhibited inflammatory cytokine expression in diseased gingival tissues.  Alveolar bone loss was alleviated in JQ1-treated mice because of reduced osteoclasts in periodontal tissues.  These unprecedented results suggest the BET inhibitor JQ1 as a prospective new approach for treating periodontitis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtMKKZI5MhYLVg90H21EOLACvtfcHk0ljNr-WM-XWTZA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWhsL%252FI&md5=7e65e561c2141b78954deb6b779ed9b5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1177%2F0022034514534261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0022034514534261%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTu%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DMurray%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DBET%2520Inhibitor%2520JQ1%2520Blocks%2520Inflammation%2520and%2520Bone%2520Destruction%26jtitle%3DJ.%2520Dent.%2520Res.%26date%3D2014%26volume%3D93%26spage%3D657%26epage%3D662%26doi%3D10.1177%2F0022034514534261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarilina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivashkiv, L. B.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition suppresses transcriptional responses to cytokine-Jak-STAT signaling in a gene-specific manner in human monocytes</span>. <i>Eur. J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">287</span>â <span class="NLM_lpage">297</span>, <span class="refDoi">Â DOI: 10.1002/eji.201444862</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1002%2Feji.201444862" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=25345375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2jurvP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2015&pages=287-297&author=C.+H.+Chanauthor=C.+Fangauthor=Y.+Qiaoauthor=A.+Yarilinaauthor=R.+K.+Prinjhaauthor=L.+B.+Ivashkiv&title=BET+bromodomain+inhibition+suppresses+transcriptional+responses+to+cytokine-Jak-STAT+signaling+in+a+gene-specific+manner+in+human+monocytes&doi=10.1002%2Feji.201444862"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition suppresses transcriptional responses to cytokine-Jak-STAT signaling in a gene-specific manner in human monocytes</span></div><div class="casAuthors">Chan, Chun Hin; Fang, Celestia; Qiao, Yu; Yarilina, Anna; Prinjha, Rab K.; Ivashkiv, Lionel B.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">287-297</span>CODEN:
                <span class="NLM_cas:coden">EJIMAF</span>;
        ISSN:<span class="NLM_cas:issn">0014-2980</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Disruption of the interaction of bromo and extraterminal (BET) proteins with acetylated histones using small mol. inhibitors suppresses Myc-driven cancers and TLR-induced inflammation in mouse models.  The predominant mechanism of BET inhibitor action is to suppress BET-mediated recruitment of pos. transcription elongation factor b and, thus, transcription elongation.  We investigated the effects of BET inhibitor I-BET151 on transcriptional responses to TLR4 and TNF in primary human monocytes and also on responses to cytokines IFN-Î², IFN-Î³, IL-4, and IL-10, which activate the JAK-STAT signaling pathway and are important for monocyte polarization and inflammatory diseases.  I-BET151 suppressed TLR4- and TNF-induced IFN responses by diminishing both autocrine IFN-Î² expression and transcriptional responses to IFN-Î².  I-BET151 inhibited cytokine-induced transcription of STAT targets in a gene-specific manner without affecting STAT activation or recruitment.  This inhibition was independent of Myc or other upstream activators.  IFN-stimulated gene transcription is regulated primarily at the level of transcription initiation.  Accordingly, we found that I-BET151 suppressed the recruitment of transcriptional machinery to the CXCL10 promoter and an upstream enhancer.  Our findings suggest that BET inhibition reduces inflammation partially through suppressing cytokine activity and expands the understanding of the inhibitory and potentially selective immunosuppressive effects of inhibiting BET proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbE0f71lXpp7Vg90H21EOLACvtfcHk0ljNr-WM-XWTZA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2jurvP&md5=efa081bc6c79db9233ae1a99ebf49945</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Feji.201444862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Feji.201444862%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DC.%2BH.%26aulast%3DFang%26aufirst%3DC.%26aulast%3DQiao%26aufirst%3DY.%26aulast%3DYarilina%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DIvashkiv%26aufirst%3DL.%2BB.%26atitle%3DBET%2520bromodomain%2520inhibition%2520suppresses%2520transcriptional%2520responses%2520to%2520cytokine-Jak-STAT%2520signaling%2520in%2520a%2520gene-specific%2520manner%2520in%2520human%2520monocytes%26jtitle%3DEur.%2520J.%2520Immunol.%26date%3D2015%26volume%3D45%26spage%3D287%26epage%3D297%26doi%3D10.1002%2Feji.201444862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nadeem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Sherbeeny, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoheir, K. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attia, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hosaini, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Sharary, S. D.</span></span> <span> </span><span class="NLM_article-title">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">248</span>â <span class="NLM_lpage">257</span>, <span class="refDoi">Â DOI: 10.1016/j.phrs.2015.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.phrs.2015.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=26149470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2015&pages=248-257&author=A.+Nadeemauthor=N.+O.+Al-Harbiauthor=M.+M.+Al-Harbiauthor=A.+M.+El-Sherbeenyauthor=S.+F.+Ahmadauthor=N.+Siddiquiauthor=M.+A.+Ansariauthor=K.+M.+A.+Zoheirauthor=S.+M.+Attiaauthor=K.+A.+Al-Hosainiauthor=S.+D.+Al-Sharary&title=Imiquimod-induced+psoriasis-like+skin+inflammation+is+suppressed+by+BET+bromodomain+inhibitor+in+mice+through+RORC%2FIL-17A+pathway+modulation&doi=10.1016%2Fj.phrs.2015.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span></div><div class="casAuthors">Nadeem, Ahmed; Al-Harbi, Naif O.; Al-Harbi, Mohamed M.; El-Sherbeeny, Ahmed M.; Ahmad, Sheikh F.; Siddiqui, Nahid; Ansari, Mushtaq A.; Zoheir, Khairy M. A.; Attia, Sabry M.; Al-Hosaini, Khaled A.; Al-Sharary, Shakir D.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">248-257</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Psoriasis is one of the most common skin disorders characterized by erythematous plaques that result from hyperproliferative keratinocytes and infiltration of inflammatory leukocytes into dermis and epidermis.  Recent studies suggest that IL-23/IL-17A/IL-22 cytokine axis plays an important role in the pathogenesis of psoriasis.  The small mol. bromodomain and extraterminal domain (BET) inhibitors, that disrupt interaction of BET proteins with acetylated histones have recently demonstrated efficacy in various models of inflammation through suppression of several pathways, one of them being synthesis of IL-17A/IL-22 which primarily depends on transcription factor, retinoic acid receptor-related orphan receptor C (RORC).  However, the efficacy and mechanistic aspect of a BET inhibitor in mouse model of skin inflammation has not been explored previously.  Therefore, this study investigated the role of BET inhibitor, JQ-1 in mouse model of psoriasis-like inflammation.  Mice were topically applied imiquimod (IMQ) to develop psoriasis-like inflammation on the shaved back and ear followed by assessment of skin inflammation (myeloperoxidase activity, ear thickness, and histopathol.), RORC and its signature cytokines (IL-17A/IL-22).  JQ-1 suppressed IMQ-induced skin inflammation as reflected by a decrease in ear thickness/myeloperoxidase activity, and RORC/IL-17A/IL-22 expression.  Addnl., a RORÎ±/Î³ agonist SR1078 was utilized to investigate the role of RORC in BET-mediated skin inflammation.  SR1078 reversed the protective effect of JQ-1 on skin inflammation at both histol. and mol. levels in the IMQ model.  The current study suggests that BET bromodomains are involved in psoriasis-like inflammation through induction of RORC/IL-17A pathway.  Therefore, inhibition of BET bromodomains may provide a new therapy against skin inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcOnC_sPUMSLVg90H21EOLACvtfcHk0ljNr-WM-XWTZA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I&md5=3edb10526e30c18bce037dc0aca611a4</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DNadeem%26aufirst%3DA.%26aulast%3DAl-Harbi%26aufirst%3DN.%2BO.%26aulast%3DAl-Harbi%26aufirst%3DM.%2BM.%26aulast%3DEl-Sherbeeny%26aufirst%3DA.%2BM.%26aulast%3DAhmad%26aufirst%3DS.%2BF.%26aulast%3DSiddiqui%26aufirst%3DN.%26aulast%3DAnsari%26aufirst%3DM.%2BA.%26aulast%3DZoheir%26aufirst%3DK.%2BM.%2BA.%26aulast%3DAttia%26aufirst%3DS.%2BM.%26aulast%3DAl-Hosaini%26aufirst%3DK.%2BA.%26aulast%3DAl-Sharary%26aufirst%3DS.%2BD.%26atitle%3DImiquimod-induced%2520psoriasis-like%2520skin%2520inflammation%2520is%2520suppressed%2520by%2520BET%2520bromodomain%2520inhibitor%2520in%2520mice%2520through%2520RORC%252FIL-17A%2520pathway%2520modulation%26jtitle%3DPharmacol.%2520Res.%26date%3D2015%26volume%3D99%26spage%3D248%26epage%3D257%26doi%3D10.1016%2Fj.phrs.2015.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.-C.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomain-containing protein 4 (BRD4) benefits rheumatoid arthritis</span>. <i>Immunol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>166</i></span>,  <span class="NLM_fpage">103</span>â <span class="NLM_lpage">108</span>, <span class="refDoi">Â DOI: 10.1016/j.imlet.2015.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.imlet.2015.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=26093279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVKhsbbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2015&pages=103-108&author=Q.-G.+Zhangauthor=J.+Qianauthor=Y.-C.+Zhu&title=Targeting+bromodomain-containing+protein+4+%28BRD4%29+benefits+rheumatoid+arthritis&doi=10.1016%2Fj.imlet.2015.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomain-containing protein 4 (BRD4) benefits rheumatoid arthritis</span></div><div class="casAuthors">Zhang, Qing-gang; Qian, Jing; Zhu, Yu-chang</div><div class="citationInfo"><span class="NLM_cas:title">Immunology Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">103-108</span>CODEN:
                <span class="NLM_cas:coden">IMLED6</span>;
        ISSN:<span class="NLM_cas:issn">0165-2478</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We aimed to explore the effects of bromodomain-contg. protein 4 (BRD4) inhibition on tumor necrosis factor (TNF)-Î±-stimulated human rheumatoid arthritis fibroblast-like synoviocytes (RA-FLS) behavior and the therapeutic implications using BRD4 inhibitor JQ1 were explored in vivo.  The levels of interleukin (IL)-1Î², IL-6, IL-17 and IL-18 in cultural supernatants from TNFÎ±-stimulated RA-FLS were measured by ELISA.  RA-FLS migration and invasion in vitro were investigated using wound healing and Matrigel assay.  Expression of signaling pathway proteins was measured by Western blot.  The in vivo effects of BRD4 inhibitor JQ1 were elucidated using collagen-induced arthritis (CIA) mice.  We found BRD4 silencing reduced the secretion of IL-1Î², IL-6, IL-17 and IL-18 from TNFÎ±-stimulated human RA-FLS.  Downregulation of BRD4 inhibited FBS-induced migration and invasion of human RA-FLS.  BRD4 silencing decreased the phosphorylation of c-Jun and activation of NFÎºB in TNFÎ±-stimulated RA-FLS.  In vivo, BRD4 inhibitor JQ1 reduced the inflammatory response, autoantibody prodn. and joint damage of CIA model.  Our data suggest for the first time that BRD4 inhibition has anti-inflammatory property in RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-RTAy64CzFbVg90H21EOLACvtfcHk0lhtmwP31tUahg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVKhsbbM&md5=395fce2cd419dd86aebd9fd93ce80c0f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.imlet.2015.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.imlet.2015.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DQ.-G.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DY.-C.%26atitle%3DTargeting%2520bromodomain-containing%2520protein%25204%2520%2528BRD4%2529%2520benefits%2520rheumatoid%2520arthritis%26jtitle%3DImmunol.%2520Lett.%26date%3D2015%26volume%3D166%26spage%3D103%26epage%3D108%26doi%3D10.1016%2Fj.imlet.2015.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grabiec, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ospelt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span> <span> </span><span class="NLM_article-title">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span>. <i>Ann. Rheum. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">422</span>â <span class="NLM_lpage">429</span>, <span class="refDoi">Â DOI: 10.1136/annrheumdis-2014-205809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1136%2Fannrheumdis-2014-205809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=25467295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2016&pages=422-429&author=K.+Kleinauthor=P.+A.+Kabalaauthor=A.+M.+Grabiecauthor=R.+E.+Gayauthor=C.+Kollingauthor=L.-L.+Linauthor=S.+Gayauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=C.+Ospeltauthor=K.+A.+Reedquist&title=The+bromodomain+protein+inhibitor+I-BET151+suppresses+expression+of+inflammatory+genes+and+matrix+degrading+enzymes+in+rheumatoid+arthritis+synovial+fibroblasts&doi=10.1136%2Fannrheumdis-2014-205809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span></div><div class="casAuthors">Klein, Kerstin; Kabala, Pawel A.; Grabiec, Aleksander M.; Gay, Renate E.; Kolling, Christoph; Lin, Lih-Ling; Gay, Steffen; Tak, Paul P.; Prinjha, Rab K.; Ospelt, Caroline; Reedquist, Kris A.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">422-429</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Objective To investigate the effects of BET bromodomain protein inhibition on inflammatory activation and functional properties of rheumatoid arthritis synovial fibroblasts (RASF).  The expression of the BET bromodomain proteins BRD2, BRD3 and BRD4 was analyzed in synovial tissue by immunohistochem.  RASF were stimulated with tumor necrosis factor (TNF)- Î±, interleukin (IL)-1Î² and toll-like receptor (TLR) ligands (Pam3, pIC and lipopolysaccharide (LPS)) in the presence or absence of the BET inhibitor I-BET151, or siRNA targeting BRD2, BRD3 and BRD4.  RASF expression of inflammatory mediators, including MMP1, MMP3, IL-6 and IL-8, was measured by q-PCR, q-PCR array and ELISA.  Cellular viability, apoptosis, proliferation and chemoattractive properties of RASF were investigated using MTT, cell apoptosis ELISA, BrdU-based proliferation and transwell migration assays.  BRD2, BRD3 and BRD4 proteins were detected in rheumatoid arthritis (RA) synovial tissue, expressed in both RASF and macrophages.  I-BET151 suppressed cytokine and TLR ligand-induced secretion of MMP1, MMP3, IL-6 and IL-8, and mRNA expression of more than 70% of genes induced by TNF- Î± and IL-1Î² .  Combined silencing of BRD2, BRD3 and BRD4 significantly reduced cytokine and TLR ligand-induced expression of a subset of gene products targeted by I-BET151, including MMP1, CXCL10 and CXCL11.  I-BET151 treatment of RASF reduced RASF proliferation, and the chemotactic potential for peripheral blood leukocytes of RASF conditioned medium.  Conclusions Inhibition of BET family proteins suppresses the inflammatory, matrix-degrading, proliferative and chemoattractive properties of RASF and suggests a therapeutic potential in the targeting of epigenetic reader proteins in RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpELw7WcQzbLLVg90H21EOLACvtfcHk0lhtmwP31tUahg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO&md5=97f611195571a9772ff7668d06cdb335</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2014-205809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2014-205809%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DK.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DGrabiec%26aufirst%3DA.%2BM.%26aulast%3DGay%26aufirst%3DR.%2BE.%26aulast%3DKolling%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DL.-L.%26aulast%3DGay%26aufirst%3DS.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DOspelt%26aufirst%3DC.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26atitle%3DThe%2520bromodomain%2520protein%2520inhibitor%2520I-BET151%2520suppresses%2520expression%2520of%2520inflammatory%2520genes%2520and%2520matrix%2520degrading%2520enzymes%2520in%2520rheumatoid%2520arthritis%2520synovial%2520fibroblasts%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2016%26volume%3D75%26spage%3D422%26epage%3D429%26doi%3D10.1136%2Fannrheumdis-2014-205809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schilderink, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reginato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilbers, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jonge, W. J.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span>. <i>Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">66</span>â <span class="NLM_lpage">76</span>, <span class="refDoi">Â DOI: 10.1016/j.molimm.2016.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.molimm.2016.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=27710838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2016&pages=66-76&author=R.+Schilderinkauthor=M.+Bellauthor=E.+Reginatoauthor=C.+Pattenauthor=I.+Riojaauthor=F.+W.+Hilbersauthor=P.+A.+Kabalaauthor=K.+A.+Reedquistauthor=D.+F.+Toughauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=W.+J.+de+Jonge&title=BET+bromodomain+inhibition+reduces+maturation+and+enhances+tolerogenic+properties+of+human+and+mouse+dendritic+cells&doi=10.1016%2Fj.molimm.2016.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span></div><div class="casAuthors">Schilderink, Ronald; Bell, Matthew; Reginato, Eleonora; Patten, Chris; Rioja, Inmaculada; Hilbers, Francisca W.; Kabala, Pawel A.; Reedquist, Kris A.; Tough, David F.; Tak, Paul Peter; Prinjha, Rab K.; de Jonge, Wouter J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">66-76</span>CODEN:
                <span class="NLM_cas:coden">MOIMD5</span>;
        ISSN:<span class="NLM_cas:issn">0161-5890</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Transcription of inflammatory genes is tightly regulated by acetylation and deacetylation of histone tails.  An inhibitor of the acetylated-lysine reader bromodomain and extra-terminal domain (BET) proteins, I-BET151, is known to counteract the induction of expression of inflammatory genes in macrophages.  We have investigated the effects of I-BET151 on dendritic cell function, including expression of co-stimulatory mols. and cytokines, and capacity for T cell activation.  Treatment of mouse bone marrow derived dendritic cells (BMDC) and human monocyte derived DCs (mdDC) with I-BET151 reduced LPS-induced expression of co-stimulatory mols., as well as the prodn. of multiple cytokines and chemokines.  Most strikingly, secretion of IL-6, IL-12 and IL-10 was significantly reduced to 89.7%, 99.9% and 98.6% resp. of that produced by control cells.  I-BET151-treated mdDC showed a reduced ability to stimulate proliferation of autologous Revaxis-specific T cells.  Moreover, while I-BET151 treatment of BMDC did not affect their ability to polarize ovalbumin specific CD4+ CD62L+ naive T cells towards Th1, Th2, or Th17 phenotypes, an increase in Foxp3 expressing Tregs secreting higher IL-10 levels was obsd.  Suppression assays demonstrated that Tregs generated in response to I-BET151-treated BMDC displayed anti-proliferative capacity.  Finally, evidence that I-BET151 treatment can ameliorate inflammation in vivo in a T cell dependent colitis model is shown.  Overall, these results demonstrate marked effects of BET inhibition on DC maturation, reducing their capacity for pro-inflammatory cytokine secretion and T cell activation and enhancing the potential of DC to induce Foxp3 expressing Treg with suppressive properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIuoARRI77BrVg90H21EOLACvtfcHk0ljkvto0Ue5PWA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I&md5=6a0da0ba320112023cb09df03c4802dc</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.molimm.2016.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molimm.2016.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DSchilderink%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DM.%26aulast%3DReginato%26aufirst%3DE.%26aulast%3DPatten%26aufirst%3DC.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DHilbers%26aufirst%3DF.%2BW.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3Dde%2BJonge%26aufirst%3DW.%2BJ.%26atitle%3DBET%2520bromodomain%2520inhibition%2520reduces%2520maturation%2520and%2520enhances%2520tolerogenic%2520properties%2520of%2520human%2520and%2520mouse%2520dendritic%2520cells%26jtitle%3DMol.%2520Immunol.%26date%3D2016%26volume%3D79%26spage%3D66%26epage%3D76%26doi%3D10.1016%2Fj.molimm.2016.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">573</span>â <span class="NLM_lpage">584</span>, <span class="refDoi">Â DOI: 10.2217/epi-2016-0144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.2217%2Fepi-2016-0144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=27925476" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2016&pages=573-584&author=D.+F.+Toughauthor=R.+K.+Prinjha&title=Immune+disease-associated+variants+in+gene+enhancers+point+to+BET+epigenetic+mechanisms+for+therapeutic+intervention&doi=10.2217%2Fepi-2016-0144"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.2217%2Fepi-2016-0144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi-2016-0144%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DImmune%2520disease-associated%2520variants%2520in%2520gene%2520enhancers%2520point%2520to%2520BET%2520epigenetic%2520mechanisms%2520for%2520therapeutic%2520intervention%26jtitle%3DEpigenomics%26date%3D2016%26volume%3D9%26spage%3D573%26epage%3D584%26doi%3D10.2217%2Fepi-2016-0144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P. M.</span></span> <span> </span><span class="NLM_article-title">BET inhibitors: a novel epigenetic approach</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1776</span>â <span class="NLM_lpage">1787</span>, <span class="refDoi">Â DOI: 10.1093/annonc/mdx157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1093%2Fannonc%2Fmdx157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=28838216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=1776-1787&author=D.+B.+Doroshowauthor=J.+P.+Ederauthor=P.+M.+LoRusso&title=BET+inhibitors%3A+a+novel+epigenetic+approach&doi=10.1093%2Fannonc%2Fmdx157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors: a novel epigenetic approach</span></div><div class="casAuthors">Doroshow D B; Eder J P; LoRusso P M</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1776-1787</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetics has been defined as 'the structural adaptation of chromosomal regions so as to register, signal or perpetuate altered activity states.' Currently, several classes of anticancer drugs function at the epigenetic level, including inhibitors of DNA methyltransferase, histone deacetylase (HDAC), lysine-specific demethylase 1, zeste homolog 2, and bromodomain and extra-terminal motif (BET) proteins.BET proteins have multiple functions, including the initiation and elongation of transcription and cell cycle regulation.  In recent years, inhibitors of BET proteins have been developed as anticancer agents.  These inhibitors exhibit selectivity for tumor cells by preferentially binding to superenhancers, noncoding regions of DNA critical for the transcription of genes that determine a cell's identity.  Preclinical research on BET inhibitors has identified them as a potential means of targeting MYC.Early clinical trials with BET inhibitors have had mixed results, with few responses in both hematologic and solid tumors that tend to be short-lived.  Toxicities have included severe, thrombocytopenia, fatigue, nausea, vomiting, and diarrhea; GI side-effects, fatigue, and low-grade dysgeusia have limited compliance.  However, preclinical data suggest that BET inhibitors may have a promising future in combination with other agents.  They appear to be able to overcome resistance to targeted agents and have strong synergy with immune checkpoint inhibitors as well as with multiple epigenetic agents, particularly HDAC inhibitors.  In many instances, BET and HDAC inhibitors were synergistic at reduced doses, suggesting a potential means of avoiding the overlapping toxicities of the two drug classes.BET inhibitors provide a novel approach to epigenetic anticancer therapy.  However, to date they appear to have limited efficacy as single agents.  A focus on BET inhibitors in combination with other drugs such as targeted and/or as other epigenetic agents is warranted, due to limited monotherapy activity, including pharmacodynamic correlatives differential activity amongst select drug combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSttah97bhI2UPI-fCgwosgfW6udTcc2eaOS3o21tALirntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D&md5=580ac90cf48f18db8313471ed61af173</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdx157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdx157%26sid%3Dliteratum%253Aachs%26aulast%3DDoroshow%26aufirst%3DD.%2BB.%26aulast%3DEder%26aufirst%3DJ.%2BP.%26aulast%3DLoRusso%26aufirst%3DP.%2BM.%26atitle%3DBET%2520inhibitors%253A%2520a%2520novel%2520epigenetic%2520approach%26jtitle%3DAnn.%2520Oncol.%26date%3D2017%26volume%3D28%26spage%3D1776%26epage%3D1787%26doi%3D10.1093%2Fannonc%2Fmdx157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery Targeting Bromodomain-Containing Protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>â <span class="NLM_lpage">4558</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+Discovery+Targeting+Bromodomain-Containing+Protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0ljDKZqswH9O2Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520Discovery%2520Targeting%2520Bromodomain-Containing%2520Protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Postel-Vinay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbschleb, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodcock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewerton, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poelman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleton, M.</span></span> <span> </span><span class="NLM_article-title">First-in-human phase I study of the bromodomain and extraterminal motif inhibitor BAY 1238097: emerging pharmacokinetic/pharmacodynamic relationship and early termination due to unexpected toxicity</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">103</span>â <span class="NLM_lpage">110</span>, <span class="refDoi">Â DOI: 10.1016/j.ejca.2018.12.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.ejca.2018.12.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=30711772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslCltLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2019&pages=103-110&author=S.+Postel-Vinayauthor=K.+Herbschlebauthor=C.+Massardauthor=V.+Woodcockauthor=J.-C.+Soriaauthor=A.+O.+Walterauthor=F.+Ewertonauthor=M.+Poelmanauthor=N.+Bensonauthor=M.+Ockerauthor=G.+Wilkinsonauthor=M.+Middleton&title=First-in-human+phase+I+study+of+the+bromodomain+and+extraterminal+motif+inhibitor+BAY+1238097%3A+emerging+pharmacokinetic%2Fpharmacodynamic+relationship+and+early+termination+due+to+unexpected+toxicity&doi=10.1016%2Fj.ejca.2018.12.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">First-in-human phase I study of the bromodomain and extraterminal motif inhibitor BAY 1238097: emerging pharmacokinetic/pharmacodynamic relationship and early termination due to unexpected toxicity</span></div><div class="casAuthors">Postel-Vinay, Sophie; Herbschleb, Karin; Massard, Christophe; Woodcock, Victoria; Soria, Jean-Charles; Walter, Annette O.; Ewerton, Flavio; Poelman, Martine; Benson, Neil; Ocker, Matthias; Wilkinson, Gary; Middleton, Mark</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">103-110</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Bromodomain and extraterminal motif (BET) protein inhibition is a promising cancer treatment strategy, notably for targeting MYC- or BRD4-driven diseases.  A first-in-human study investigated the safety, pharmacokinetics, max. tolerated dose and recommended phase II dose of the BET inhibitor BAY 1238097 in patients with advanced malignancies.In this phase I, open-label, non-randomised, multicentre study, patients with cytol. or histol. confirmed advanced refractory malignancies received oral BAY 1238097 twice weekly in 21-day cycles using an adaptive dose-escalation design at a starting dose of 10 mg/wk.  Model-based dose-response anal. was performed to guide dose escalation.  Safety, pharmacokinetics, pharmacodynamics and tumor response were evaluated.Eight patients were enrolled at three dose levels (10 mg/wk, n = 3; 40 mg/wk, n = 3; 80 mg/wk, n = 2).  Both patients receiving 80 mg/wk had dose-limiting toxicities (DLTs) (grade 3 vomiting, grade 3 headache and grade 2/3 back pain).  The most common adverse events were nausea, vomiting, headache, back pain and fatigue.  Pharmacokinetic anal. indicated a linear dose response with increasing dose.  Two patients displayed prolonged stable disease; no responses were obsd.  Biomarker evaluation of MYC and HEXIM1 expression demonstrated an emerging pharmacokinetic/pharmacodynamic relationship, with a trend towards decreased MYC and increased HEXIM1 expression in response to treatment.The study was prematurely terminated because of the occurrence of DLTs at a dose below targeted drug exposure.  Pharmacokinetic modeling indicated that an alternate dosing schedule whereby DLTs could be avoided while reaching efficacious exposure was not feasible.  Registration no.: NCT02369029.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9snvNdTKTtrVg90H21EOLACvtfcHk0ljDKZqswH9O2Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslCltLo%253D&md5=c5969b7827810fd1146338b84c8f8621</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2018.12.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2018.12.020%26sid%3Dliteratum%253Aachs%26aulast%3DPostel-Vinay%26aufirst%3DS.%26aulast%3DHerbschleb%26aufirst%3DK.%26aulast%3DMassard%26aufirst%3DC.%26aulast%3DWoodcock%26aufirst%3DV.%26aulast%3DSoria%26aufirst%3DJ.-C.%26aulast%3DWalter%26aufirst%3DA.%2BO.%26aulast%3DEwerton%26aufirst%3DF.%26aulast%3DPoelman%26aufirst%3DM.%26aulast%3DBenson%26aufirst%3DN.%26aulast%3DOcker%26aufirst%3DM.%26aulast%3DWilkinson%26aufirst%3DG.%26aulast%3DMiddleton%26aufirst%3DM.%26atitle%3DFirst-in-human%2520phase%2520I%2520study%2520of%2520the%2520bromodomain%2520and%2520extraterminal%2520motif%2520inhibitor%2520BAY%25201238097%253A%2520emerging%2520pharmacokinetic%252Fpharmacodynamic%2520relationship%2520and%2520early%2520termination%2520due%2520to%2520unexpected%2520toxicity%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2019%26volume%3D109%26spage%3D103%26epage%3D110%26doi%3D10.1016%2Fj.ejca.2018.12.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Odenike, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolff, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borthakur, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldoss, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzieri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prebet, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freise, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonas, B. A.</span></span> <span> </span><span class="NLM_article-title">Results from the first-in-human study of mivebresib (ABBV-075), a pan-inhibitor of bromodomain and extra terminal proteins, in patients with relapsed/refractory acute myeloid leukemia</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">7030</span>â <span class="NLM_lpage">7030</span>, <span class="refDoi">Â DOI: 10.1200/JCO.2019.37.15_suppl.7030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1200%2FJCO.2019.37.15_suppl.7030" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=7030-7030&author=O.+Odenikeauthor=J.+E.+Wolffauthor=G.+Borthakurauthor=I.+T.+Aldossauthor=D.+Rizzieriauthor=T.+Prebetauthor=B.+Huauthor=M.+Dinhauthor=X.+Chenauthor=D.+Modiauthor=K.+J.+Freiseauthor=B.+A.+Jonas&title=Results+from+the+first-in-human+study+of+mivebresib+%28ABBV-075%29%2C+a+pan-inhibitor+of+bromodomain+and+extra+terminal+proteins%2C+in+patients+with+relapsed%2Frefractory+acute+myeloid+leukemia&doi=10.1200%2FJCO.2019.37.15_suppl.7030"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1200%2FJCO.2019.37.15_suppl.7030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2019.37.15_suppl.7030%26sid%3Dliteratum%253Aachs%26aulast%3DOdenike%26aufirst%3DO.%26aulast%3DWolff%26aufirst%3DJ.%2BE.%26aulast%3DBorthakur%26aufirst%3DG.%26aulast%3DAldoss%26aufirst%3DI.%2BT.%26aulast%3DRizzieri%26aufirst%3DD.%26aulast%3DPrebet%26aufirst%3DT.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DDinh%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DModi%26aufirst%3DD.%26aulast%3DFreise%26aufirst%3DK.%2BJ.%26aulast%3DJonas%26aufirst%3DB.%2BA.%26atitle%3DResults%2520from%2520the%2520first-in-human%2520study%2520of%2520mivebresib%2520%2528ABBV-075%2529%252C%2520a%2520pan-inhibitor%2520of%2520bromodomain%2520and%2520extra%2520terminal%2520proteins%252C%2520in%2520patients%2520with%2520relapsed%252Frefractory%2520acute%2520myeloid%2520leukemia%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D37%26spage%3D7030%26epage%3D7030%26doi%3D10.1200%2FJCO.2019.37.15_suppl.7030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">19754</span>â <span class="NLM_lpage">19759</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+M%C3%BCllerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0liK_zuauVzbdg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Shiao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">New Synthetic Routes to Triazolo-benzodiazepine Analogues: Expanding the Scope of the Bump-and-Hole Approach for Selective Bromo and Extra-Terminal (BET) Bromodomain Inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1492</span>â <span class="NLM_lpage">1500</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01135</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01135" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyksbnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1492-1500&author=M.+G.+J.+Baudauthor=E.+Lin-Shiaoauthor=M.+Zengerleauthor=C.+Tallantauthor=A.+Ciulli&title=New+Synthetic+Routes+to+Triazolo-benzodiazepine+Analogues%3A+Expanding+the+Scope+of+the+Bump-and-Hole+Approach+for+Selective+Bromo+and+Extra-Terminal+%28BET%29+Bromodomain+Inhibition&doi=10.1021%2Facs.jmedchem.5b01135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">New Synthetic Routes to Triazolo-benzodiazepine Analogues: Expanding the Scope of the Bump-and-Hole Approach for Selective Bromo and Extra-Terminal (BET) Bromodomain Inhibition</span></div><div class="casAuthors">Baud, Matthias G. J.; Lin-Shiao, Enrique; Zengerle, Michael; Tallant, Cynthia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1492-1500</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe new synthetic routes developed toward a range of substituted analogs of bromo and extra-terminal (BET) bromodomain inhibitors I-BET762/JQ1 based on the triazolo-benzodiazepine scaffold.  These new routes allow for the derivatization of the methoxyphenyl and chlorophenyl rings, in addn. to the diazepine ternary center and the side chain methylene moiety.  Substitution at the level of the side chain methylene afforded compds. targeting specifically and potently engineered BET bromodomains designed as part of a bump and hole approach.  We further demonstrate that marked selectivity for the second over the first bromodomain can be achieved with an indole deriv. that exploits differential interaction with an aspartate/histidine conservative substitution on the BC loop of BET bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUpYVun8F94bVg90H21EOLACvtfcHk0liK_zuauVzbdg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyksbnE&md5=65dd39e3f7e0bc52169ffb1466218c43</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01135%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DLin-Shiao%26aufirst%3DE.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DNew%2520Synthetic%2520Routes%2520to%2520Triazolo-benzodiazepine%2520Analogues%253A%2520Expanding%2520the%2520Scope%2520of%2520the%2520Bump-and-Hole%2520Approach%2520for%2520Selective%2520Bromo%2520and%2520Extra-Terminal%2520%2528BET%2529%2520Bromodomain%2520Inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1492%26epage%3D1500%26doi%3D10.1021%2Facs.jmedchem.5b01135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, A. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4317</span>â <span class="NLM_lpage">4334</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4317-4334&author=R.+P.+Lawauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.-W.+Chungauthor=E.+H.+Demontauthor=L.+J.+Gordonauthor=M.+Lindonauthor=R.+K.+Prinjhaauthor=A.+J.+B.+Watsonauthor=D.+J.+Hirst&title=Discovery+of+Tetrahydroquinoxalines+as+Bromodomain+and+Extra-Terminal+Domain+%28BET%29+Inhibitors+with+Selectivity+for+the+Second+Bromodomain&doi=10.1021%2Facs.jmedchem.7b01666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Law, Robert P.; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Gordon, Laurie J.; Lindon, Matthew; Prinjha, Rab K.; Watson, Allan J. B.; Hirst, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4317-4334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins bind acetylated lysine residues on histone proteins.  The four BET bromodomains-BRD2, BRD3, BRD4, and BRDT-each contain two bromodomain modules.  BET bromodomain inhibition is a potential therapy for various cancers and immunoinflammatory diseases, but few reported inhibitors show selectivity within the BET family.  Inhibitors with selectivity for the first or second bromodomain are desired to aid investigation of the biol. function of these domains.  Focused library screening identified a series of tetrahydroquinoxalines with selectivity for the second bromodomains of the BET family (BD2).  Structure-guided optimization of the template improved potency, selectivity, and physicochem. properties, culminating in potent BET inhibitors with BD2 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobBKYwu7NyTbVg90H21EOLACvtfcHk0liBfEmT_HeAow"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D&md5=078db91b35feaf1958fa8c33d0544387</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01666%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BP.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWatson%26aufirst%3DA.%2BJ.%2BB.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520Tetrahydroquinoxalines%2520as%2520Bromodomain%2520and%2520Extra-Terminal%2520Domain%2520%2528BET%2529%2520Inhibitors%2520with%2520Selectivity%2520for%2520the%2520Second%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4317%26epage%3D4334%26doi%3D10.1021%2Facs.jmedchem.7b01666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gesner, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span> <span> </span><span class="NLM_article-title">RVX-297- a novel BD2 selective inhibitor of BET bromodomains</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>477</i></span>,  <span class="NLM_fpage">62</span>â <span class="NLM_lpage">67</span>, <span class="refDoi">Â DOI: 10.1016/j.bbrc.2016.06.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.bbrc.2016.06.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=27282480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC28Xpslajtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2016&pages=62-67&author=O.+A.+Kharenkoauthor=E.+M.+Gesnerauthor=R.+G.+Patelauthor=K.+Norekauthor=A.+Whiteauthor=E.+Fontanoauthor=R.+K.+Sutoauthor=P.+R.+Youngauthor=K.+G.+McLureauthor=H.+C.+Hansen&title=RVX-297-+a+novel+BD2+selective+inhibitor+of+BET+bromodomains&doi=10.1016%2Fj.bbrc.2016.06.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-297- a novel BD2 selective inhibitor of BET bromodomains</span></div><div class="casAuthors">Kharenko, Olesya A.; Gesner, Emily M.; Patel, Reena G.; Norek, Karen; White, Andre; Fontano, Eric; Suto, Robert K.; Young, Peter R.; McLure, Kevin G.; Hansen, Henrik C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">62-67</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic readers that specifically bind to the acetyl lysine residues of histones and transcription factors.  Small mol. BET bromodomain inhibitors can disrupt this interaction which leads to potential modulation of several disease states.  Here we describe the binding properties of a novel BET inhibitor RVX-297 that is structurally related to the clin. compd. RVX-208, currently undergoing phase III clin. trials for the treatment of cardiovascular diseases, but is distinctly different in its biol. and pharmacokinetic profiles.  We report that RVX-297 preferentially binds to the BD2 domains of the BET bromodomain and Extra Terminal (BET) family of protein.  We demonstrate the differential binding modes of RVX-297 in BD1 and BD2 domains of BRD4 and BRD2 using X-ray crystallog., and describe the structural differences driving the BD2 selective binding of RVX-297.  The isothermal titrn. calorimetry (ITC) data illustrate the related differential thermodn. of binding of RVX-297 to single as well as dual BET bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDLUg96I7HObVg90H21EOLACvtfcHk0liBfEmT_HeAow"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xpslajtrw%253D&md5=a3d064e2245bf14ae525eac2490808cb</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2016.06.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2016.06.021%26sid%3Dliteratum%253Aachs%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DPatel%26aufirst%3DR.%2BG.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DFontano%26aufirst%3DE.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26atitle%3DRVX-297-%2520a%2520novel%2520BD2%2520selective%2520inhibitor%2520of%2520BET%2520bromodomains%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2016%26volume%3D477%26spage%3D62%26epage%3D67%26doi%3D10.1016%2Fj.bbrc.2016.06.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jahagirdar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marusic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendele, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenoy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulikowski, E.</span></span> <span> </span><span class="NLM_article-title">RVX-297, a BET Bromodomain Inhibitor, Has Therapeutic Effects in Preclinical Models of Acute Inflammation and Autoimmune Disease</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">694</span>â <span class="NLM_lpage">706</span>, <span class="refDoi">Â DOI: 10.1124/mol.117.110379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1124%2Fmol.117.110379" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=28974538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGgurbK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2017&pages=694-706&author=R.+Jahagirdarauthor=S.+Attwellauthor=S.+Marusicauthor=A.+Bendeleauthor=N.+Shenoyauthor=K.+G.+McLureauthor=D.+Gilhamauthor=K.+Norekauthor=H.+C.+Hansenauthor=R.+Yuauthor=J.+Tobinauthor=G.+S.+Wagnerauthor=P.+R.+Youngauthor=N.+C.+W.+Wongauthor=E.+Kulikowski&title=RVX-297%2C+a+BET+Bromodomain+Inhibitor%2C+Has+Therapeutic+Effects+in+Preclinical+Models+of+Acute+Inflammation+and+Autoimmune+Disease&doi=10.1124%2Fmol.117.110379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-297, a BET bromodomain inhibitor, has therapeutic effects in preclinical models of acute inflammation and autoimmune disease</span></div><div class="casAuthors">Jahagirdar, Ravi; Attwell, Sarah; Marusic, Suzana; Bendele, Alison; Shenoy, Narmada; McLure, Kevin G.; Gilham, Dean; Norek, Karen; Hansen, Henrik C.; Yu, Raymond; Tobin, Jennifer; Wagner, Gregory S.; Young, Peter R.; Wong, Norman C. W.; Kulikowski, Ewelina</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">694-706</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Bromodomain (BD) and extra-terminal domain contg. proteins (BET) are chromatin adapters that bind acetylated histone marks via two tandem BDs, BD1 and BD2, to regulate gene transcription.  BET proteins are involved in transcriptional reprogramming in response to inflammatory stimuli.  BET BD inhibitors (BETis) that are nonselective for BD1 or BD2 have recognized anti-inflammatory properties in vitro and counter pathol. in models of inflammation or autoimmune disease.  Although both BD1 and BD2 bind acetylated histone residues, they may independently regulate the expression of BET-sensitive genes.  Here we characterized the ability of RVX-297, a novel orally active BETi with selectivity for BD2, to modulate inflammatory processes in vitro, in vivo, and ex vivo.  RVX-297 suppressed inflammatory gene expression in multiple immune cell types in culture.  Mechanistically, RVX-297 displaced BET proteins from the promoters of sensitive genes and disrupted recruitment of active RNA polymerase II, a property shared with pan-BETis that nonselectively bind BET BDs.  In the lipopolysaccharide model of inflammation, RVX-297 reduced proinflammatory mediators assessed in splenic gene expression and serum proteins.  RVX-297 also countered pathol. in three rodent models of polyarthritis: rat and mouse collagen-induced arthritis, and mouse collagen antibody-induced arthritis.  Further, RVX-297 prevented murine exptl. autoimmune encephalomyelitis (a model of human multiple sclerosis) disease development when administered prophylactically and reduced hallmarks of pathol. when administered therapeutically.  We show for the first time that a BD2-selective BETi maintains anti-inflammatory properties and is effective in preclin. models of acute inflammation and autoimmunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7r1JpLemFA7Vg90H21EOLACvtfcHk0liBfEmT_HeAow"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGgurbK&md5=3f1a1b86b17315c413dd1f13d3a7736a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1124%2Fmol.117.110379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.117.110379%26sid%3Dliteratum%253Aachs%26aulast%3DJahagirdar%26aufirst%3DR.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DMarusic%26aufirst%3DS.%26aulast%3DBendele%26aufirst%3DA.%26aulast%3DShenoy%26aufirst%3DN.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DGilham%26aufirst%3DD.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DYu%26aufirst%3DR.%26aulast%3DTobin%26aufirst%3DJ.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DKulikowski%26aufirst%3DE.%26atitle%3DRVX-297%252C%2520a%2520BET%2520Bromodomain%2520Inhibitor%252C%2520Has%2520Therapeutic%2520Effects%2520in%2520Preclinical%2520Models%2520of%2520Acute%2520Inflammation%2520and%2520Autoimmune%2520Disease%26jtitle%3DMol.%2520Pharmacol.%26date%3D2017%26volume%3D92%26spage%3D694%26epage%3D706%26doi%3D10.1124%2Fmol.117.110379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, S. A. J.</span>; <span class="NLM_string-name">Atkinson, S. J.</span>; <span class="NLM_string-name">Aylott, H.</span>; <span class="NLM_string-name">Bamborough, P.</span>; <span class="NLM_string-name">Chung, C.-w.</span>; <span class="NLM_string-name">Copley, R. C. B.</span>; <span class="NLM_string-name">Gordon, L. J.</span></span> ; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span>; <span class="NLM_string-name">Gray, J. R. J.</span>; <span class="NLM_string-name">Harrison, L. A.</span>; <span class="NLM_string-name">Hayhow, T. G.</span>; <span class="NLM_string-name">Messenger, C.</span>; <span class="NLM_string-name">Michon, A.-M.</span>; <span class="NLM_string-name">Mitchell, D. J.</span>; <span class="NLM_string-name">Preston, A.</span>; <span class="NLM_string-name">Prinjha, R. K.</span>; <span class="NLM_string-name">Rioja, I.</span>; <span class="NLM_string-name">Taylor, S.</span>; <span class="NLM_string-name">Lindon, M.</span>; <span class="NLM_string-name">Wall, I.</span>; <span class="NLM_string-name">Watson, R. R.</span>; <span class="NLM_string-name">Woolven, J. M.</span>; <span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">The Optimisation of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor</span>,  <i>J . Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00796</span> .</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=S.+A.+J.+Seal&author=S.+J.+Atkinson&author=H.+Aylott&author=P.+Bamborough&author=C.-w.+Chung&author=R.+C.+B.+Copley&author=L.+J.+Gordonauthor=P.+Grandi&author=J.+R.+J.+Gray&author=L.+A.+Harrison&author=T.+G.+Hayhow&author=C.+Messenger&author=A.-M.+Michon&author=D.+J.+Mitchell&author=A.+Preston&author=R.+K.+Prinjha&author=I.+Rioja&author=S.+Taylor&author=M.+Lindon&author=I.+Wall&author=R.+R.+Watson&author=J.+M.+Woolven&author=E.+H.+Demont&title=The+Optimisation+of+a+Novel%2C+Weak+Bromo+and+Extra+Terminal+Domain+%28BET%29+Bromodomain+Fragment+Ligand+to+a+Potent+and+Selective+Second+Bromodomain+%28BD2%29+Inhibitor&doi=10.1021%2Facs.jmedchem.0c00796"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00796%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DS.%2BA.%2BJ.%26aulast%3DGrandi%26aufirst%3DP.%26atitle%3DThe%2520Optimisation%2520of%2520a%2520Novel%252C%2520Weak%2520Bromo%2520and%2520Extra%2520Terminal%2520Domain%2520%2528BET%2529%2520Bromodomain%2520Fragment%2520Ligand%2520to%2520a%2520Potent%2520and%2520Selective%2520Second%2520Bromodomain%2520%2528BD2%2529%2520Inhibitor%26jtitle%3DJ%2520.%2520Med.%2520Chem.%26date%3D2020%26doi%3D10.1021%2Facs.jmedchem.0c00796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gesner, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujikawa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campeau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasiak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an Inducer of ApoA-I in Humans, Is a BET Bromodomain Antagonist</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e83190</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0083190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1371%2Fjournal.pone.0083190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=24391744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=K.+G.+McLureauthor=E.+M.+Gesnerauthor=L.+Tsujikawaauthor=O.+A.+Kharenkoauthor=S.+Attwellauthor=E.+Campeauauthor=S.+Wasiakauthor=A.+Steinauthor=A.+Whiteauthor=E.+Fontanoauthor=R.+K.+Sutoauthor=N.+C.+W.+Wongauthor=G.+S.+Wagnerauthor=H.+C.+Hansenauthor=P.+R.+Young&title=RVX-208%2C+an+Inducer+of+ApoA-I+in+Humans%2C+Is+a+BET+Bromodomain+Antagonist&doi=10.1371%2Fjournal.pone.0083190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inducer of ApoA-I in humans, is a BET bromodomain antagonist</span></div><div class="casAuthors">McLure, Kevin G.; Gesner, Emily M.; Tsujikawa, Laura; Kharenko, Olesya A.; Attwell, Sarah; Campeau, Eric; Wasiak, Sylwia; Stein, Adam; White, Andre; Fontano, Eric; Suto, Robert K.; Wong, Norman C. W.; Wagner, Gregory S.; Hansen, Henrik C.; Young, Peter R.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e83190/1-e83190/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Increased synthesis of Apolipoprotein A-I (ApoA-I) and HDL is believed to provide a new approach to treating atherosclerosis through the stimulation of reverse cholesterol transport.  RVX-208 increases the prodn. of ApoA-I in hepatocytes in vitro and in vivo in monkeys and humans, which results in increased HDL-C, but the mol. target was not previously reported.  Using binding assays and X-ray crystallog., we now show that RVX-208 selectively binds to bromodomains of the BET (Bromodomain and Extra Terminal) family, competing for a site bound by the endogenous ligand, acetylated lysine and that this accounts for its pharmacol. activity. siRNA expts. further suggest that induction of ApoA-I mRNA is mediated by BET family member BRD4.  These data indicate that RVX-208 increases ApoA-I prodn. through an epigenetic mechanism and suggests that BET inhibition may be a promising new approach to the treatment of atherosclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLHPlAY0x6NLVg90H21EOLACvtfcHk0ljscjzuEavI0w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D&md5=519ef7ed6d06f97ef8a1359832c11d92</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0083190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0083190%26sid%3Dliteratum%253Aachs%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DTsujikawa%26aufirst%3DL.%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DCampeau%26aufirst%3DE.%26aulast%3DWasiak%26aufirst%3DS.%26aulast%3DStein%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DFontano%26aufirst%3DE.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26atitle%3DRVX-208%252C%2520an%2520Inducer%2520of%2520ApoA-I%2520in%2520Humans%252C%2520Is%2520a%2520BET%2520Bromodomain%2520Antagonist%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0083190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantena, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnik, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolette, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigelow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>578</i></span>,  <span class="NLM_fpage">306</span>â <span class="NLM_lpage">310</span>, <span class="refDoi">Â DOI: 10.1038/s41586-020-1930-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1038%2Fs41586-020-1930-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=31969702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=578&publication_year=2020&pages=306-310&author=E.+J.+Faivreauthor=K.+F.+McDanielauthor=D.+H.+Albertauthor=S.+R.+Mantenaauthor=J.+P.+Plotnikauthor=D.+Wilcoxauthor=L.+Zhangauthor=M.+H.+Buiauthor=G.+S.+Sheppardauthor=L.+Wangauthor=V.+Sehgalauthor=X.+Linauthor=X.+Huangauthor=X.+Luauthor=T.+Uzielauthor=P.+Hesslerauthor=L.+T.+Lamauthor=R.+J.+Bellinauthor=G.+Mehtaauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=C.+Sunauthor=S.+C.+Panchalauthor=J.+J.+Nicoletteauthor=S.+L.+Fosseyauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=L.+Bigelowauthor=M.+Torrentauthor=S.+H.+Rosenbergauthor=W.+M.+Katiauthor=Y.+Shen&title=Selective+inhibition+of+the+BD2+bromodomain+of+BET+proteins+in+prostate+cancer&doi=10.1038%2Fs41586-020-1930-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span></div><div class="casAuthors">Faivre, Emily J.; McDaniel, Keith F.; Albert, Daniel H.; Mantena, Srinivasa R.; Plotnik, Joshua P.; Wilcox, Denise; Zhang, Lu; Bui, Mai H.; Sheppard, George S.; Wang, Le; Sehgal, Vasudha; Lin, Xiaoyu; Huang, Xiaoli; Lu, Xin; Uziel, Tamar; Hessler, Paul; Lam, Lloyd T.; Bellin, Richard J.; Mehta, Gaurav; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa A.; Sun, Chaohong; Panchal, Sanjay C.; Nicolette, John J.; Fossey, Stacey L.; Park, Chang H.; Longenecker, Kenton; Bigelow, Lance; Torrent, Maricel; Rosenberg, Saul H.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">578</span>
        (<span class="NLM_cas:issue">7794</span>),
    <span class="NLM_cas:pages">306-310</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Proteins of the bromodomain and extra-terminal (BET) domain family are epigenetic readers that bind acetylated histones through their bromodomains to regulate gene transcription.  Dual-bromodomain BET inhibitors (DbBi) that bind with similar affinities to the first (BD1) and second (BD2) bromodomains of BRD2, BRD3, BRD4 and BRDt have displayed modest clin. activity in monotherapy cancer trials.  A reduced no. of thrombocytes in the blood (thrombocytopenia) as well as symptoms of gastrointestinal toxicity are dose-limiting adverse events for some types of DbBi1-5.  Given that similar haematol. and gastrointestinal defects were obsd. after genetic silencing of Brd4 in mice6, the platelet and gastrointestinal toxicities may represent on-target activities assocd. with BET inhibition.  The two individual bromodomains in BET family proteins may have distinct functions7-9 and different cellular phenotypes after pharmacol. inhibition of one or both bromodomains have been reported10,11, suggesting that selectively targeting one of the bromodomains may result in a different efficacy and tolerability profile compared with DbBi.  Available compds. that are selective to individual domains lack sufficient potency and the pharmacokinetics properties that are required for in vivo efficacy and tolerability assessment10-13.  Here we carried out a medicinal chem. campaign that led to the discovery of ABBV-744, a highly potent and selective inhibitor of the BD2 domain of BET family proteins with drug-like properties.  In contrast to the broad range of cell growth inhibition induced by DbBi, the antiproliferative activity of ABBV-744 was largely, but not exclusively, restricted to cell lines of acute myeloid leukemia and prostate cancer that expressed the full-length androgen receptor (AR).  ABBV-744 retained robust activity in prostate cancer xenografts, and showed fewer platelet and gastrointestinal toxicities than the DbBi ABBV-07514.  Analyses of RNA expression and chromatin immunopptn. followed by sequencing revealed that ABBV-744 displaced BRD4 from AR-contg. super-enhancers and inhibited AR-dependent transcription, with less impact on global transcription compared with ABBV-075.  These results underscore the potential value of selectively targeting the BD2 domain of BET family proteins for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF98kER1y9pLVg90H21EOLACvtfcHk0ljS99oPKLi-vg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D&md5=800d9106c33084390fd8cbf1ac7f1766</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-1930-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-1930-8%26sid%3Dliteratum%253Aachs%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMantena%26aufirst%3DS.%2BR.%26aulast%3DPlotnik%26aufirst%3DJ.%2BP.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSehgal%26aufirst%3DV.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DBellin%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DNicolette%26aufirst%3DJ.%2BJ.%26aulast%3DFossey%26aufirst%3DS.%2BL.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DBigelow%26aufirst%3DL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520the%2520BD2%2520bromodomain%2520of%2520BET%2520proteins%2520in%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2020%26volume%3D578%26spage%3D306%26epage%3D310%26doi%3D10.1038%2Fs41586-020-1930-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5585</span>â <span class="NLM_lpage">5623</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00628</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00628" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5585-5623&author=G.+S.+Sheppardauthor=L.+Wangauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=J.+K.+Prattauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=W.+Qiuauthor=M.+Torrentauthor=P.+J.+Kovarauthor=M.+Buiauthor=E.+Faivreauthor=X.+Huangauthor=X.+Linauthor=D.+Wilcoxauthor=L.+Zhangauthor=Y.+Shenauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=W.+M.+Katiauthor=K.+F.+McDaniel&title=Discovery+of+N-Ethyl-4-%5B2-%284-fluoro-2%2C6-dimethyl-phenoxy%29-5-%281-hydroxy-1-methyl-ethyl%29phenyl%5D-6-methyl-7-oxo-1H-pyrrolo%5B2%2C3-c%5Dpyridine-2-carboxamide+%28ABBV-744%29%2C+a+BET+Bromodomain+Inhibitor+with+Selectivity+for+the+Second+Bromodomain&doi=10.1021%2Facs.jmedchem.0c00628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Sheppard, George S.; Wang, Le; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Pratt, John K.; Park, Chang H.; Longenecker, Kenton; Qiu, Wei; Torrent, Maricel; Kovar, Peter J.; Bui, Mai; Faivre, Emily; Huang, Xiaoli; Lin, Xiaoyu; Wilcox, Denise; Zhang, Lu; Shen, Yu; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5585-5623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The BET family of proteins consists of BRD2, BRD3, BRD4, and BRDt.  Each protein contains two distinct bromodomains (BD1 and BD2).  BET family bromodomain inhibitors under clin. development for oncol. bind to each of the eight bromodomains with similar affinities.  We hypothesized that it may be possible to achieve an improved therapeutic index by selectively targeting subsets of the BET bromodomains.  Both BD1 and BD2 are highly conserved across family members (>70% identity), whereas BD1 and BD2 from the same protein exhibit a larger degree of divergence (~ 40% identity), suggesting selectivity between BD1 and BD2 of all family members would be more straightforward to achieve.  Exploiting the Asp144/His437 and Ile146/Val439 sequence differences (BRD4 BD1/BD2 numbering) allowed the identification of compd. 27 demonstrating greater than 100-fold selectivity for BRD4 BD2 over BRD4 BD1.  Further optimization to improve BD2 selectivity and oral bioavailability resulted in the clin. development compd. 46 (ABBV-744).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJtZIcWro0xLVg90H21EOLACvtfcHk0ljS99oPKLi-vg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D&md5=675827659c3c7b6099ccd667f4c60f95</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00628%26sid%3Dliteratum%253Aachs%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DKovar%26aufirst%3DP.%2BJ.%26aulast%3DBui%26aufirst%3DM.%26aulast%3DFaivre%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26atitle%3DDiscovery%2520of%2520N-Ethyl-4-%255B2-%25284-fluoro-2%252C6-dimethyl-phenoxy%2529-5-%25281-hydroxy-1-methyl-ethyl%2529phenyl%255D-6-methyl-7-oxo-1H-pyrrolo%255B2%252C3-c%255Dpyridine-2-carboxamide%2520%2528ABBV-744%2529%252C%2520a%2520BET%2520Bromodomain%2520Inhibitor%2520with%2520Selectivity%2520for%2520the%2520Second%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5585%26epage%3D5623%26doi%3D10.1021%2Facs.jmedchem.0c00628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">648</span>â <span class="NLM_lpage">655</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2010.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.drudis.2010.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=20570751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=648-655&author=A.+P.+Hillauthor=R.+J.+Young&title=Getting+physical+in+drug+discovery%3A+a+contemporary+perspective+on+solubility+and+hydrophobicity&doi=10.1016%2Fj.drudis.2010.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span></div><div class="casAuthors">Hill, Alan P.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15/16</span>),
    <span class="NLM_cas:pages">648-655</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Suboptimal phys. properties have been identified as a particular shortcoming of compds. in contemporary drug discovery, contributing to high attrition levels.  An anal. of the relationship between hydrophobicity (calcd. and measured) and â¼100 k measured kinetic soly. values has been undertaken.  In line with the General Soly. Equation, ests. of hydrophobicity, particularly ACD c log D pH7.4, give a useful indication of the likely soly. classification of particular mols.  Taking ACD c log D pH7.4 values together with the no. of arom. rings in a given mol. provides enhanced prediction.  The Soly. Forecast Index' (SFI = c log D pH7.4 + #Ar) is proposed as a simple, yet effective, guide to predicting soly.  Moreover, anal. of measured distribution/partition coeff. values highlighted statistically significant shortcomings in the applicability of octanol/water as a model system for hydrophobicity detn. with poorly sol. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdtQyfNjbFUrVg90H21EOLACvtfcHk0lgTE2pGSh7Ueg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P&md5=7b201081c5586a6ea4b7af00d507d462</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%253A%2520a%2520contemporary%2520perspective%2520on%2520solubility%2520and%2520hydrophobicity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2010%26volume%3D15%26spage%3D648%26epage%3D655%26doi%3D10.1016%2Fj.drudis.2010.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscombe, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span> <span> </span><span class="NLM_article-title">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">822</span>â <span class="NLM_lpage">830</span>, <span class="refDoi">Â DOI: 10.1016/j.drudis.2011.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.drudis.2011.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=21704184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=822-830&author=R.+J.+Youngauthor=D.+V.+S.+Greenauthor=C.+N.+Luscombeauthor=A.+P.+Hill&title=Getting+physical+in+drug+discovery+II%3A+the+impact+of+chromatographic+hydrophobicity+measurements+and+aromaticity&doi=10.1016%2Fj.drudis.2011.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span></div><div class="casAuthors">Young, Robert J.; Green, Darren V. S.; Luscombe, Christopher N.; Hill, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17/18</span>),
    <span class="NLM_cas:pages">822-830</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here, we review the performance of chromatog. hydrophobicity measurements in a data set of 100 000 GlaxoSmithKline compds., demonstrating the advantages of the method over octanol-water partitioning and highlighting new insights for drug discovery.  The value of chromatog. measurements, vs. other hydrophobicity ests., was supported by improved relationships with soly., permeation, cytochrome P450s, intrinsic clearance, hERG binding and promiscuity.  We also obsd. marked differentiation of the relative influence of intrinsic and effective hydrophobicity.  The summing of hydrophobicity values plus arom. ring count [log D pH7.4 (or log P) + #Ar], indicated a wide relevance for simplistic property forecast indexes' in developability assays, clearly enhanced by chromatog. values; therefore establishing new foundations for enriching property-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmIJyWi-DBW7Vg90H21EOLACvtfcHk0lgTE2pGSh7Ueg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF&md5=22788603a45aa77537e8666433473d14</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DHill%26aufirst%3DA.%2BP.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%2520II%253A%2520the%2520impact%2520of%2520chromatographic%2520hydrophobicity%2520measurements%2520and%2520aromaticity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26spage%3D822%26epage%3D830%26doi%3D10.1016%2Fj.drudis.2011.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Discovery of Bromodomain Inhibitors Part 1: Inhibitor Binding Modes and Implications for Lead Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">576</span>â <span class="NLM_lpage">586</span>, <span class="refDoi">Â DOI: 10.1021/jm201320w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201320w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=576-586&author=C.-w.+Chungauthor=A.+W.+Deanauthor=J.+M.+Woolvenauthor=P.+Bamborough&title=Fragment-Based+Discovery+of+Bromodomain+Inhibitors+Part+1%3A+Inhibitor+Binding+Modes+and+Implications+for+Lead+Discovery&doi=10.1021%2Fjm201320w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Bromodomain Inhibitors Part 1: Inhibitor Binding Modes and Implications for Lead Discovery</span></div><div class="casAuthors">Chung, Chun-wa; Dean, Anthony W.; Woolven, James M.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">576-586</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are key epigenetic regulators of gene transcription and readers of the histone code.  However, the therapeutic benefits of modulating this target class are largely unexplored due to the lack of suitable chem. probes.  This article describes the generation of lead mols. for the BET bromodomains through screening a fragment set chosen using structural insights and computational approaches.  Anal. of 40 BRD2/fragment x-ray complexes highlights both shared and disparate interaction features that may be exploited for affinity and selectivity.  Six representative crystal structures are then exemplified in detail.  Two of the fragments are completely new bromodomain chemotypes, and three have never before been crystd. in a bromodomain, so our results significantly extend the limited public knowledge-base of crystallog. small mol./bromodomain interactions.  Certain fragments (including paracetamol) bind in a consistent mode to different bromodomains such as CREBBP, suggesting their potential to act as generic bromodomain templates.  An important implication is that the bromodomains are not only a phylogenetic family but also a system in which chem. and structural knowledge of one bromodomain gives insights transferable to others.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonNb7GnZnS_rVg90H21EOLACvtfcHk0lgTE2pGSh7Ueg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN&md5=5b7d7239d0de57266d94beef1ea080ae</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm201320w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201320w%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDean%26aufirst%3DA.%2BW.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-Based%2520Discovery%2520of%2520Bromodomain%2520Inhibitors%2520Part%25201%253A%2520Inhibitor%2520Binding%2520Modes%2520and%2520Implications%2520for%2520Lead%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D576%26epage%3D586%26doi%3D10.1021%2Fjm201320w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delves, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ClÃ©ment, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Small Molecule Inhibitors of the BET Family Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3827</span>â <span class="NLM_lpage">3838</span>, <span class="refDoi">Â DOI: 10.1021/jm200108t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200108t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3827-3838&author=C.-w.+Chungauthor=H.+Costeauthor=J.+H.+Whiteauthor=O.+Mirguetauthor=J.+Wildeauthor=R.+L.+Gosminiauthor=C.+Delvesauthor=S.+M.+Magnyauthor=R.+Woodwardauthor=S.+A.+Hughesauthor=E.+V.+Boursierauthor=H.+Flynnauthor=A.+M.+Bouillotauthor=P.+Bamboroughauthor=J.-M.+G.+Brusqauthor=F.+J.+Gellibertauthor=E.+J.+Jonesauthor=A.+M.+Riouauthor=P.+Homesauthor=S.+L.+Martinauthor=I.+J.+Uingsauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=A.-B.+Boullayauthor=R.+L.+Grimleyauthor=F.+M.+Blandelauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=Discovery+and+Characterization+of+Small+Molecule+Inhibitors+of+the+BET+Family+Bromodomains&doi=10.1021%2Fjm200108t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span></div><div class="casAuthors">Chung, Chun-wa; Coste, Herve; White, Julia H.; Mirguet, Olivier; Wilde, Jonathan; Gosmini, Romain L.; Delves, Chris; Magny, Sylvie M.; Woodward, Robert; Hughes, Stephen A.; Boursier, Eric V.; Flynn, Helen; Bouillot, Anne M.; Bamborough, Paul; Brusq, Jean-Marie G.; Gellibert, Francoise J.; Jones, Emma J.; Riou, Alizon M.; Homes, Paul; Martin, Sandrine L.; Uings, Iain J.; Toum, Jerome; Clement, Catherine A.; Boullay, Anne-Benedicte; Grimley, Rachel L.; Blandel, Florence M.; Prinjha, Rab K.; Lee, Kevin; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3827-3838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic mechanisms of gene regulation have a profound role in normal development and disease processes.  An integral part of this mechanism occurs through lysine acetylation of histone tails which are recognized by bromodomains.  While the biol. and structural characterization of many bromodomain contg. proteins has advanced considerably, the therapeutic tractability of this protein family is only now becoming understood.  This paper describes the discovery and mol. characterization of potent (nM) small mol. inhibitors that disrupt the function of the BET family of bromodomains (Brd2, Brd3, and Brd4).  By using a combination of phenotypic screening, chemoproteomics, and biophys. studies, we have discovered that the protein-protein interactions between bromodomains and acetylated histones can be antagonized by selective small mols. that bind at the acetylated lysine recognition pocket.  X-ray crystal structures of compds. bound into bromodomains of Brd2 and Brd4 elucidate the mol. interactions of binding and explain the precisely defined stereochem. required for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VauuYkT5YLVg90H21EOLACvtfcHk0liXSZEfRhsX9A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D&md5=ebc0f6062321dece877259d775f1c19b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm200108t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200108t%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%2BH.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DJ.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DDelves%26aufirst%3DC.%26aulast%3DMagny%26aufirst%3DS.%2BM.%26aulast%3DWoodward%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DS.%2BA.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DBouillot%26aufirst%3DA.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBrusq%26aufirst%3DJ.-M.%2BG.%26aulast%3DGellibert%26aufirst%3DF.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DRiou%26aufirst%3DA.%2BM.%26aulast%3DHomes%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BL.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBoullay%26aufirst%3DA.-B.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBlandel%26aufirst%3DF.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520the%2520BET%2520Family%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3827%26epage%3D3838%26doi%3D10.1021%2Fjm200108t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>â <span class="NLM_lpage">356</span>, <span class="refDoi">Â DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0liXSZEfRhsX9A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>â <span class="NLM_lpage">231</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+Recognition+and+Large-Scale+Structural+Analysis+of+the+Human+Bromodomain+Family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize Îµ-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0liXSZEfRhsX9A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520Recognition%2520and%2520Large-Scale%2520Structural%2520Analysis%2520of%2520the%2520Human%2520Bromodomain%2520Family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">430</span>â <span class="NLM_lpage">431</span>, <span class="refDoi">Â DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eYGtq_-c5Rdrbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span> <span> </span><span class="NLM_article-title">The maximal affinity of ligands</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">9997</span>â <span class="NLM_lpage">10002</span>, <span class="refDoi">Â DOI: 10.1073/pnas.96.18.9997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1073%2Fpnas.96.18.9997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10468550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=9997-10002&author=I.+D.+Kuntzauthor=K.+Chenauthor=K.+A.+Sharpauthor=P.+A.+Kollman&title=The+maximal+affinity+of+ligands&doi=10.1073%2Fpnas.96.18.9997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The maximal affinity of ligands</span></div><div class="casAuthors">Kuntz, I. D.; Chen, K.; Sharp, K. A.; Kollman, P. A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">9997-10002</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We explore the question of what are the best ligands for macromol. targets.  A survey of exptl. data on a large no. of the strongest-binding ligands indicates that the free energy of binding increases with the no. of nonhydrogen atoms with an initial slope of â-1.5 kcal/mol (1 cal = 4.18 J) per atom.  For ligands that contain more than 15 nonhydrogen atoms, the free energy of binding increases very little with relative mol. mass.  This nonlinearity is largely ascribed to nonthermodynamic factors.  An anal. of the dominant interactions suggests that van der Waals interactions and hydrophobic effects provide a reasonable basis for understanding binding affinities across the entire set of ligands.  Interesting outliers that bind unusually strongly on a per atom basis include metal ions, covalently attached ligands, and a few well known complexes such as biotin-avidin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0L6KZhR0TxbVg90H21EOLACvtfcHk0lgA8joW3kFRyA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D&md5=46fe894b016f3041831a2f0b71f812b1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.18.9997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.18.9997%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DSharp%26aufirst%3DK.%2BA.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DThe%2520maximal%2520affinity%2520of%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26spage%3D9997%26epage%3D10002%26doi%3D10.1073%2Fpnas.96.18.9997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skipper, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wogan, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tannenbaum, S. R.</span></span> <span> </span><span class="NLM_article-title">Monocyclic aromatic amines as potential human carcinogens: old is new again</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">50</span>â <span class="NLM_lpage">58</span>, <span class="refDoi">Â DOI: 10.1093/carcin/bgp267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1093%2Fcarcin%2Fbgp267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=19887514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktlCnsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=50-58&author=P.+L.+Skipperauthor=M.+Y.+Kimauthor=H.+L.+P.+Sunauthor=G.+N.+Woganauthor=S.+R.+Tannenbaum&title=Monocyclic+aromatic+amines+as+potential+human+carcinogens%3A+old+is+new+again&doi=10.1093%2Fcarcin%2Fbgp267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Monocyclic aromatic amines as potential human carcinogens: old is new again</span></div><div class="casAuthors">Skipper, Paul L.; Kim, Min Young; Sun, H.-L. Patty; Wogan, Gerald N.; Tannenbaum, Steven R.</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-58</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Alkylanilines are a group of chems. whose ubiquitous presence in the environment is a result of the multitude of sources from which they originate.  Exposure assessments indicate that most individuals experience lifelong exposure to these compds.  Many alkylanilines have biol. activity similar to that of the carcinogenic multi-ring arom. amines.  This review provides an overview of human exposure and biol. effects.  It also describes recent investigations into the biochem. mechanisms of action that lead to the assessment that they are most probably more complex than those of the more extensively investigated multi-ring arom. amines.  Not only is nitrenium ion chem. implicated in DNA damage by alkylanilines but also reactions involving quinone imines and perhaps reactive oxygen species.  Recent results described here indicate that alkylanilines can be potent genotoxins for cultured mammalian cells when activated by exogenous or endogenous phase I and phase II xenobiotic-metabolizing enzymes.  The nature of specific DNA damage products responsible for mutagenicity remains to be identified but evidence to date supports mechanisms of activation through obligatory N-hydroxylation as well as subsequent conjugation by sulfation and/or acetylation.  A fuller understanding of the mechanisms of alkylaniline genotoxicity is expected to provide important insights into the environmental and genetic origins of one or more human cancers and may reveal a substantial role for this group of compds. as potential human chem. carcinogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu8jtJ_-C4s7Vg90H21EOLACvtfcHk0lgA8joW3kFRyA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktlCnsw%253D%253D&md5=37e5a874a1ee9edf96acb3ad73ec2925</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp267%26sid%3Dliteratum%253Aachs%26aulast%3DSkipper%26aufirst%3DP.%2BL.%26aulast%3DKim%26aufirst%3DM.%2BY.%26aulast%3DSun%26aufirst%3DH.%2BL.%2BP.%26aulast%3DWogan%26aufirst%3DG.%2BN.%26aulast%3DTannenbaum%26aufirst%3DS.%2BR.%26atitle%3DMonocyclic%2520aromatic%2520amines%2520as%2520potential%2520human%2520carcinogens%253A%2520old%2520is%2520new%2520again%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D50%26epage%3D58%26doi%3D10.1093%2Fcarcin%2Fbgp267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ames, B. N.</span></span> <span> </span><span class="NLM_article-title">Identifying environmental chemicals causing mutations and cancer</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>204</i></span>,  <span class="NLM_fpage">587</span>â <span class="NLM_lpage">593</span>, <span class="refDoi">Â DOI: 10.1126/science.373122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1126%2Fscience.373122" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=373122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADyaE1MXktFClt70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=204&publication_year=1979&pages=587-593&author=B.+N.+Ames&title=Identifying+environmental+chemicals+causing+mutations+and+cancer&doi=10.1126%2Fscience.373122"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying environmental chemicals causing mutations and cancer</span></div><div class="casAuthors">Ames, Bruce N.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">204</span>
        (<span class="NLM_cas:issue">4393</span>),
    <span class="NLM_cas:pages">587-93</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A review with 57 refs. on environmental chem. induced carcinogenesis and mutagenesis and their relation to DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2AzWl_6PJ4bVg90H21EOLACvtfcHk0lgezgBNrygMGw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1MXktFClt70%253D&md5=9ac98bad1482e9cda849723a71f32a9d</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fscience.373122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.373122%26sid%3Dliteratum%253Aachs%26aulast%3DAmes%26aufirst%3DB.%2BN.%26atitle%3DIdentifying%2520environmental%2520chemicals%2520causing%2520mutations%2520and%2520cancer%26jtitle%3DScience%26date%3D1979%26volume%3D204%26spage%3D587%26epage%3D593%26doi%3D10.1126%2Fscience.373122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frisch, M. J. G. W. T.</span>; <span class="NLM_string-name">Schlegel, H. B.</span>; <span class="NLM_string-name">Scuseria, G. E.</span>; <span class="NLM_string-name">Robb, M. A.</span>; <span class="NLM_string-name">Cheeseman, J. R.</span>; <span class="NLM_string-name">Montgomery,  Jr., J. A.</span>; <span class="NLM_string-name">Vreven, T.</span>; <span class="NLM_string-name">Kudin, K. N.</span>; <span class="NLM_string-name">Burant, J. C.</span>; <span class="NLM_string-name">Millam, J. M.</span>; <span class="NLM_string-name">Iyengar, S. S.</span>; <span class="NLM_string-name">Tomasi, J.</span>; <span class="NLM_string-name">Barone, V.</span>; <span class="NLM_string-name">Mennucci, B.</span>; <span class="NLM_string-name">Cossi, M.</span>; <span class="NLM_string-name">Scalmani, G.</span>; <span class="NLM_string-name">Rega, N.</span>; <span class="NLM_string-name">Petersson, G. A.</span>; <span class="NLM_string-name">Nakatsuji, H.</span>; <span class="NLM_string-name">Hada, M.</span>; <span class="NLM_string-name">Ehara, M.</span>; <span class="NLM_string-name">Toyota, K.</span>; <span class="NLM_string-name">Fukuda, R.</span>; <span class="NLM_string-name">Hasegawa, J.</span>; <span class="NLM_string-name">Ishida, M.</span>; <span class="NLM_string-name">Nakajima, T.</span>; <span class="NLM_string-name">Honda, Y.</span>; <span class="NLM_string-name">Kitao, O.</span>; <span class="NLM_string-name">Nakai, H.</span>; <span class="NLM_string-name">Klene, M.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Knox, J. E.</span>; <span class="NLM_string-name">Hratchian, H. P.</span>; <span class="NLM_string-name">Cross, J. B.</span>; <span class="NLM_string-name">Bakken, V.</span>; <span class="NLM_string-name">Adamo, C.</span>; <span class="NLM_string-name">Jaramillo, J.</span>; <span class="NLM_string-name">Gomperts, R.</span>; <span class="NLM_string-name">Stratmann, R. E.</span>; <span class="NLM_string-name">Yazyev, O.</span>; <span class="NLM_string-name">Austin, A. J.</span>; <span class="NLM_string-name">Cammi, R.</span>; <span class="NLM_string-name">Pomelli, C.</span>; <span class="NLM_string-name">Ochterski, J. W.</span>; <span class="NLM_string-name">Ayala, P. Y.</span>; <span class="NLM_string-name">Morokuma, K.</span>; <span class="NLM_string-name">Voth, G. A.</span>; <span class="NLM_string-name">Salvador, P.</span>; <span class="NLM_string-name">Dannenberg, J. J.</span>; <span class="NLM_string-name">Zakrzewski, V. G.</span>; <span class="NLM_string-name">Dapprich, S.</span>; <span class="NLM_string-name">Daniels, A. D.</span>; <span class="NLM_string-name">Strain, M. C.</span>; <span class="NLM_string-name">Farkas, O.</span>; <span class="NLM_string-name">Malick, D. K.</span>; <span class="NLM_string-name">Rabuck, A. D.</span>; <span class="NLM_string-name">Raghavachari, K.</span>; <span class="NLM_string-name">Foresman, J. B.</span>; <span class="NLM_string-name">Ortiz, J. V.</span>; <span class="NLM_string-name">Cui, Q.</span>; <span class="NLM_string-name">Baboul, A. G.</span>; <span class="NLM_string-name">Clifford, S.</span>; <span class="NLM_string-name">Cioslowski, J.</span>; <span class="NLM_string-name">Stefanov, B. B.</span>; <span class="NLM_string-name">Liu, G.</span>; <span class="NLM_string-name">Liashenko, A.</span>; <span class="NLM_string-name">Piskorz, P.</span>; <span class="NLM_string-name">Komaromi, I.</span>; <span class="NLM_string-name">Martin, R. L.</span>; <span class="NLM_string-name">Fox, D. J.</span>; <span class="NLM_string-name">Keith, T.</span>; <span class="NLM_string-name">Al-Laham, M. A.</span>; <span class="NLM_string-name">Peng, C. Y.</span>; <span class="NLM_string-name">Nanayakkara, A.</span>; <span class="NLM_string-name">Challacombe, M.</span>; <span class="NLM_string-name">Gill, P. M. W.</span>; <span class="NLM_string-name">Johnson, B.</span>; <span class="NLM_string-name">Chen, W.</span>; <span class="NLM_string-name">Wong, M. W.</span>; <span class="NLM_string-name">Gonzalez, C.</span>; <span class="NLM_string-name">Pople, J. A.</span></span> <i>Gaussian 03</i>; Revision C.02 <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=M.+J.+G.+W.+T.+Frisch&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=J.+A.+Montgomery&author=T.+Vreven&author=K.+N.+Kudin&author=J.+C.+Burant&author=J.+M.+Millam&author=S.+S.+Iyengar&author=J.+Tomasi&author=V.+Barone&author=B.+Mennucci&author=M.+Cossi&author=G.+Scalmani&author=N.+Rega&author=G.+A.+Petersson&author=H.+Nakatsuji&author=M.+Hada&author=M.+Ehara&author=K.+Toyota&author=R.+Fukuda&author=J.+Hasegawa&author=M.+Ishida&author=T.+Nakajima&author=Y.+Honda&author=O.+Kitao&author=H.+Nakai&author=M.+Klene&author=X.+Li&author=J.+E.+Knox&author=H.+P.+Hratchian&author=J.+B.+Cross&author=V.+Bakken&author=C.+Adamo&author=J.+Jaramillo&author=R.+Gomperts&author=R.+E.+Stratmann&author=O.+Yazyev&author=A.+J.+Austin&author=R.+Cammi&author=C.+Pomelli&author=J.+W.+Ochterski&author=P.+Y.+Ayala&author=K.+Morokuma&author=G.+A.+Voth&author=P.+Salvador&author=J.+J.+Dannenberg&author=V.+G.+Zakrzewski&author=S.+Dapprich&author=A.+D.+Daniels&author=M.+C.+Strain&author=O.+Farkas&author=D.+K.+Malick&author=A.+D.+Rabuck&author=K.+Raghavachari&author=J.+B.+Foresman&author=J.+V.+Ortiz&author=Q.+Cui&author=A.+G.+Baboul&author=S.+Clifford&author=J.+Cioslowski&author=B.+B.+Stefanov&author=G.+Liu&author=A.+Liashenko&author=P.+Piskorz&author=I.+Komaromi&author=R.+L.+Martin&author=D.+J.+Fox&author=T.+Keith&author=M.+A.+Al-Laham&author=C.+Y.+Peng&author=A.+Nanayakkara&author=M.+Challacombe&author=P.+M.+W.+Gill&author=B.+Johnson&author=W.+Chen&author=M.+W.+Wong&author=C.+Gonzalez&author=J.+A.+Pople&title=Gaussian+03"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%2BG.%2BW.%2BT.%26jtitle%3DGaussian%252003%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deshmane, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kremlev, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawaya, B. E.</span></span> <span> </span><span class="NLM_article-title">Monocyte Chemoattractant Protein-1 (MCP-1): An Overview</span>. <i>J. Interferon Cytokine Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">313</span>â <span class="NLM_lpage">326</span>, <span class="refDoi">Â DOI: 10.1089/jir.2008.0027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1089%2Fjir.2008.0027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=19441883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvF2rs7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=313-326&author=S.+L.+Deshmaneauthor=S.+Kremlevauthor=S.+Aminiauthor=B.+E.+Sawaya&title=Monocyte+Chemoattractant+Protein-1+%28MCP-1%29%3A+An+Overview&doi=10.1089%2Fjir.2008.0027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Monocyte Chemoattractant Protein-1 (MCP-1): An Overview</span></div><div class="casAuthors">Deshmane, Satish L.; Kremlev, Sergey; Amini, Shohreh; Sawaya, Bassel E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Interferon & Cytokine Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">313-326</span>CODEN:
                <span class="NLM_cas:coden">JICRFJ</span>;
        ISSN:<span class="NLM_cas:issn">1079-9907</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Chemokines constitute a family of chemoattractant cytokines and are subdivided into four families on the basis of the no. and spacing of the conserved cysteine residues in the N-terminus of the protein.  Chemokines play a major role in selectively recruiting monocytes, neutrophils, and lymphocytes, as well as in inducing chemotaxis through the activation of G-protein-coupled receptors.  Monocyte chemoattractant protein-1 (MCP-1/CCL2) is one of the key chemokines that regulate migration and infiltration of monocytes/macrophages.  Both CCL2 and its receptor CCR2 have been demonstrated to be induced and involved in various diseases.  Migration of monocytes from the blood stream across the vascular endothelium is required for routine immunol. surveillance of tissues, as well as in response to inflammation.  This review will discuss these biol. processes and the structure and function of CCL2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSmnT35tYDZLVg90H21EOLACvtfcHk0lgezgBNrygMGw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvF2rs7s%253D&md5=d58ea0c1925c2eff93631916f93edad3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1089%2Fjir.2008.0027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fjir.2008.0027%26sid%3Dliteratum%253Aachs%26aulast%3DDeshmane%26aufirst%3DS.%2BL.%26aulast%3DKremlev%26aufirst%3DS.%26aulast%3DAmini%26aufirst%3DS.%26aulast%3DSawaya%26aufirst%3DB.%2BE.%26atitle%3DMonocyte%2520Chemoattractant%2520Protein-1%2520%2528MCP-1%2529%253A%2520An%2520Overview%26jtitle%3DJ.%2520Interferon%2520Cytokine%2520Res.%26date%3D2009%26volume%3D29%26spage%3D313%26epage%3D326%26doi%3D10.1089%2Fjir.2008.0027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E. Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliadis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burr, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A. G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbetti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vappiani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">Selective targeting of BD1 and BD2 of the BET proteins in cancer and immuno-inflammation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>386</i></span> (<span class="NLM_issue">6489</span>),  <span class="NLM_fpage">387</span>â <span class="NLM_lpage">394</span>, <span class="refDoi">Â DOI: 10.1126/science.aaz8455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1126%2Fscience.aaz8455" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2020&pages=387-394&issue=6489&author=O.+Gilanauthor=I.+Riojaauthor=K.+Knezevicauthor=M.+J.+Bellauthor=M.+M.+Yeungauthor=N.+R.+Harkerauthor=E.+Y.+N.+Lamauthor=C.-w.+Chungauthor=P.+Bamboroughauthor=M.+Petretichauthor=M.+Urhauthor=S.+J.+Atkinsonauthor=A.+K.+Bassilauthor=E.+J.+Robertsauthor=D.+Vassiliadisauthor=M.+L.+Burrauthor=A.+G.+S.+Prestonauthor=C.+Wellawayauthor=T.+Wernerauthor=J.+R.+Grayauthor=A.-M.+Michonauthor=T.+Gobbettiauthor=V.+Kumarauthor=P.+E.+Sodenauthor=A.+Haynesauthor=J.+Vappianiauthor=D.+F.+Toughauthor=S.+Taylorauthor=S.-J.+Dawsonauthor=M.+Bantscheffauthor=M.+Lindonauthor=G.+Drewesauthor=E.+H.+Demontauthor=D.+L.+Danielsauthor=P.+Grandiauthor=R.+K.+Prinjhaauthor=M.+A.+Dawson&title=Selective+targeting+of+BD1+and+BD2+of+the+BET+proteins+in+cancer+and+immuno-inflammation&doi=10.1126%2Fscience.aaz8455"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaz8455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaz8455%26sid%3Dliteratum%253Aachs%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DKnezevic%26aufirst%3DK.%26aulast%3DBell%26aufirst%3DM.%2BJ.%26aulast%3DYeung%26aufirst%3DM.%2BM.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DLam%26aufirst%3DE.%2BY.%2BN.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DVassiliadis%26aufirst%3DD.%26aulast%3DBurr%26aufirst%3DM.%2BL.%26aulast%3DPreston%26aufirst%3DA.%2BG.%2BS.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DJ.%2BR.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DGobbetti%26aufirst%3DT.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DHaynes%26aufirst%3DA.%26aulast%3DVappiani%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DDawson%26aufirst%3DS.-J.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DSelective%2520targeting%2520of%2520BD1%2520and%2520BD2%2520of%2520the%2520BET%2520proteins%2520in%2520cancer%2520and%2520immuno-inflammation%26jtitle%3DScience%26date%3D2020%26volume%3D386%26issue%3D6489%26spage%3D387%26epage%3D394%26doi%3D10.1126%2Fscience.aaz8455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, P. B.</span>; <span class="NLM_string-name">Bernard, S.</span>; <span class="NLM_string-name">Chung, C.-w.</span>; <span class="NLM_string-name">Craggs, P. D.</span>; <span class="NLM_string-name">Cutler, L.</span>; <span class="NLM_string-name">Demont, E. H.</span>; <span class="NLM_string-name">Evans, J.P.</span>; <span class="NLM_string-name">Gordon, L.</span>; <span class="NLM_string-name">Karamshi, B.</span>; <span class="NLM_string-name">Lewis, A. J.</span>; <span class="NLM_string-name">Lindon, M. J.</span>; <span class="NLM_string-name">Mitchell, D. J.</span>; <span class="NLM_string-name">Rioja, I.</span>; <span class="NLM_string-name">Soden, P. E.</span>; <span class="NLM_string-name">Taylor, S.</span>; <span class="NLM_string-name">Watson, R. J.</span>; <span class="NLM_string-name">Willis, R.</span>; <span class="NLM_string-name">Woolven, J. M.</span>; <span class="NLM_string-name">Wyspianska, B. S.</span>; <span class="NLM_string-name">Kerr, W. J.</span>; <span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based Design of a Bromodomain and Extraterminal Domain (BET) Inhibitor Selective for the N-terminal Bromodomains that Retains an Anti-inflammatory and Anti-proliferative Phenotype</span>.  <i>J. Med. Chem.</i> [Online early access]. <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00566</span>  Published Online: August 3, <span class="NLM_year" style="font-weight: bold;">2020</span>.</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=P.+B.+Wellaway&author=S.+Bernard&author=C.-w.+Chung&author=P.+D.+Craggs&author=L.+Cutler&author=E.+H.+Demont&author=J.P.+Evans&author=L.+Gordon&author=B.+Karamshi&author=A.+J.+Lewis&author=M.+J.+Lindon&author=D.+J.+Mitchell&author=I.+Rioja&author=P.+E.+Soden&author=S.+Taylor&author=R.+J.+Watson&author=R.+Willis&author=J.+M.+Woolven&author=B.+S.+Wyspianska&author=W.+J.+Kerr&author=R.+K.+Prinjha&title=Structure-based+Design+of+a+Bromodomain+and+Extraterminal+Domain+%28BET%29+Inhibitor+Selective+for+the+N-terminal+Bromodomains+that+Retains+an+Anti-inflammatory+and+Anti-proliferative+Phenotype&doi=10.1021%2Facs.jmedchem.0c00566"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00566%26sid%3Dliteratum%253Aachs%26aulast%3DWellaway%26aufirst%3DP.%2BB.%26atitle%3DStructure-based%2520Design%2520of%2520a%2520Bromodomain%2520and%2520Extraterminal%2520Domain%2520%2528BET%2529%2520Inhibitor%2520Selective%2520for%2520the%2520N-terminal%2520Bromodomains%2520that%2520Retains%2520an%2520Anti-inflammatory%2520and%2520Anti-proliferative%2520Phenotype%26jtitle%3DJ.%2520Med.%2520Chem.%26doi%3D10.1021%2Facs.jmedchem.0c00566%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">OâBoyle, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banck, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandermeersch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchison, G. R.</span></span> <span> </span><span class="NLM_article-title">Open Babel: An open chemical toolbox</span>. <i>Aust. J. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">33</span>, <span class="refDoi">Â DOI: 10.1186/1758-2946-3-33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1186%2F1758-2946-3-33" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=33&author=N.+M.+O%E2%80%99Boyleauthor=M.+Banckauthor=C.+A.+Jamesauthor=C.+Morleyauthor=T.+Vandermeerschauthor=G.+R.+Hutchison&title=Open+Babel%3A+An+open+chemical+toolbox&doi=10.1186%2F1758-2946-3-33"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-3-33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-3-33%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%2BM.%26aulast%3DBanck%26aufirst%3DM.%26aulast%3DJames%26aufirst%3DC.%2BA.%26aulast%3DMorley%26aufirst%3DC.%26aulast%3DVandermeersch%26aufirst%3DT.%26aulast%3DHutchison%26aufirst%3DG.%2BR.%26atitle%3DOpen%2520Babel%253A%2520An%2520open%2520chemical%2520toolbox%26jtitle%3DAust.%2520J.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D33%26doi%3D10.1186%2F1758-2946-3-33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camurri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaramella, A.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Liquid Chromatography/Mass Spectrometry Method for the Determination of the Chromatographic Hydrophobicity Index</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">3716</span>â <span class="NLM_lpage">3722</span>, <span class="refDoi">Â DOI: 10.1021/ac001388j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac001388j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkvVCgsbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2001&pages=3716-3722&author=G.+Camurriauthor=A.+Zaramella&title=High-Throughput+Liquid+Chromatography%2FMass+Spectrometry+Method+for+the+Determination+of+the+Chromatographic+Hydrophobicity+Index&doi=10.1021%2Fac001388j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Liquid Chromatography/Mass Spectrometry Method for the Determination of the Chromatographic Hydrophobicity Index</span></div><div class="casAuthors">Camurri, Giulio; Zaramella, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3716-3722</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A fast gradient reversed-phase liq. chromatog. (LC) method, using an acetonitrile gradient was developed to det. the chromatog. hydrophobicity index (CHI), as reported by Valco et al. (Anal. Chem. 1997, 69, 2022-2029).The anal. method provides retention times, based on UV detection at two different wavelengths, which then are converted into CHI values after calibration with a set of test compds.  The CHI of each compd. is measured at three different pH values, 2.0, 7.4, and 10.5; so using an 8-min gradient at each pH value one compd. can be analyzed in â¼24 min.  The aim of this work is to improve the throughput of the CHI screening using a LC/MS approach, so the application of the LC/MS technique is an extension of the LC/UV approach previously reported by Valco et al.  This approach allows contemporary injection of N compds. into the LC/MS system, the retention time of each compd. can be then extd. from the selected ion recording chromatograms.  The throughput of the existing screening method could be increased by N times, where N is the no. of compds. injected, so only three runs are needed to det. the CHI at three different pH values for a set of N compds.  The highest value of N depends on the total no. of channels that can be monitored simultaneously; 32 channels were used.  This LC/MS method was tested for a no. of com. products analyzed as mixts., and data obtained were compared with those coming from the classical LC/UV approach.  In the same way, the method was tested for a no. of compds. assocd. with two GlaxoWellcome projects in the antibacterial area.  The LC/MS method can be successfully applied for analyzing compds. in mixts. and for compds. with poor UV absorption, which cannot be analyzed with the std. LC/UV method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnpqYuIDpy5rVg90H21EOLACvtfcHk0ljIUfY9HCHSrw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkvVCgsbY%253D&md5=8773c87e08ba7ddce3c2d679360a6220</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fac001388j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac001388j%26sid%3Dliteratum%253Aachs%26aulast%3DCamurri%26aufirst%3DG.%26aulast%3DZaramella%26aufirst%3DA.%26atitle%3DHigh-Throughput%2520Liquid%2520Chromatography%252FMass%2520Spectrometry%2520Method%2520for%2520the%2520Determination%2520of%2520the%2520Chromatographic%2520Hydrophobicity%2520Index%26jtitle%3DAnal.%2520Chem.%26date%3D2001%26volume%3D73%26spage%3D3716%26epage%3D3722%26doi%3D10.1021%2Fac001388j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunhuck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, D. P.</span></span> <span> </span><span class="NLM_article-title">Fast gradient HPLC method to determine compounds binding to human serum albumin. Relationships with octanol/water and immobilized artificial membrane lipophilicity</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">2236</span>â <span class="NLM_lpage">2248</span>, <span class="refDoi">Â DOI: 10.1002/jps.10494</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1002%2Fjps.10494" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=14603509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovFCrs7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2003&pages=2236-2248&author=K.+Valkoauthor=S.+Nunhuckauthor=C.+Bevanauthor=M.+H.+Abrahamauthor=D.+P.+Reynolds&title=Fast+gradient+HPLC+method+to+determine+compounds+binding+to+human+serum+albumin.+Relationships+with+octanol%2Fwater+and+immobilized+artificial+membrane+lipophilicity&doi=10.1002%2Fjps.10494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Fast gradient HPLC method to determine compounds binding to human serum albumin. Relationships with octanol/water and immobilized artificial membrane lipophilicity</span></div><div class="casAuthors">Valko, Klara; Nunhuck, Shenaz; Bevan, Chris; Abraham, Michael H.; Reynolds, Derek P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2236-2248</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A fast gradient HPLC method (cycle time 15 min) has been developed to det. Human Serum Albumin (HSA) binding of discovery compds. using chem. bonded protein stationary phases.  The HSA binding values were derived from the gradient retention times that were converted to the logarithm of the equil. consts. (logK HSA) using data from a calibration set of mols.  The method has been validated using literature plasma protein binding data of 68 known drug mols.  The method is fully automated, and has been used for lead optimization in more than 20 company projects.  The HSA binding data obtained for more than 4000 compds. were suitable to set up global and project specific quant. structure binding relationships that helped compd. design in early drug discovery.  The obtained HSA binding of known drug mols. were compared to the Immobilizd Artificial Membrane binding data (CHI IAM) obtained by our previously described HPLC-based method.  The solvation equation approach has been used to characterize the normal binding ability of HSA, and this relationship shows that compd. lipophilicity is a significant factor.  It was found that the selectivity of the "baseline" lipophilicity governing HSA binding, membrane interaction, and octanol/water partition are very similar.  However, the effect of the presence of pos. or neg. charges have very different effects.  It was found that neg. charged compds. bind more strongly to HSA than it would be expected from the lipophilicity of the ionized species at pH 7.4.  Several compds. showed stronger HSA binding than can be expected from their lipophilicity alone, and comparison between predicted and exptl. binding affinity allows the identification of compds. that have good complementarities with any of the known binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr1LUBLdv-t7Vg90H21EOLACvtfcHk0lhAQ2YSB-YZIg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovFCrs7Y%253D&md5=14eb1e0f0ad616846a2b5d1428133fa9</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fjps.10494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.10494%26sid%3Dliteratum%253Aachs%26aulast%3DValko%26aufirst%3DK.%26aulast%3DNunhuck%26aufirst%3DS.%26aulast%3DBevan%26aufirst%3DC.%26aulast%3DAbraham%26aufirst%3DM.%2BH.%26aulast%3DReynolds%26aufirst%3DD.%2BP.%26atitle%3DFast%2520gradient%2520HPLC%2520method%2520to%2520determine%2520compounds%2520binding%2520to%2520human%2520serum%2520albumin.%2520Relationships%2520with%2520octanol%252Fwater%2520and%2520immobilized%2520artificial%2520membrane%2520lipophilicity%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2003%26volume%3D92%26spage%3D2236%26epage%3D2248%26doi%3D10.1002%2Fjps.10494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span> <span> </span><span class="NLM_article-title">A Chemical Probe for the ATAD2 Bromodomain</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">11382</span>â <span class="NLM_lpage">11386</span>, <span class="refDoi">Â DOI: 10.1002/anie.201603928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=10.1002%2Fanie.201603928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=11382-11386&author=P.+Bamboroughauthor=C.-w.+Chungauthor=E.+H.+Demontauthor=R.+C.+Furzeauthor=A.+J.+Bannisterauthor=K.+H.+Cheauthor=H.+Dialloauthor=C.+Douaultauthor=P.+Grandiauthor=T.+Kouzaridesauthor=A.-M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=S.+Robsonauthor=R.+J.+Sheppardauthor=R.+Uptonauthor=R.+J.+Watson&title=A+Chemical+Probe+for+the+ATAD2+Bromodomain&doi=10.1002%2Fanie.201603928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Probe for the ATAD2 Bromodomain</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Furze, Rebecca C.; Bannister, Andrew J.; Che, Ka Hing; Diallo, Hawa; Douault, Clement; Grandi, Paola; Kouzarides, Tony; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Robson, Samuel; Sheppard, Robert J.; Upton, Richard; Watson, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11382-11386</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">ATAD2 is a cancer-assocd. protein whose bromodomain has been described as among the least druggable of that target class.  Starting from a potent lead, permeability and selectivity were improved through a dual approach: (1) using CF2 as a sulfone bio-isostere to exploit the unique properties of fluorine, and (2) using 1,3-interactions to control the conformation of a piperidine ring.  This resulted in the first reported low-nanomolar, selective and cell permeable chem. probe for ATAD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqId2vaqTEeyLVg90H21EOLACvtfcHk0lhAQ2YSB-YZIg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL&md5=7eef64dfd3c75b9c17cd7f43d368df04</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fanie.201603928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201603928%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DRobson%26aufirst%3DS.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DUpton%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3DA%2520Chemical%2520Probe%2520for%2520the%2520ATAD2%2520Bromodomain%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D11382%26epage%3D11386%26doi%3D10.1002%2Fanie.201603928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6151</span>â <span class="NLM_lpage">6178</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00773</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00773" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6151-6178&author=P.+Bamboroughauthor=C.-w.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.-M.+Michonauthor=R.+J.+Sheppardauthor=H.+Barnettauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=R.+J.+Watsonauthor=T.+Wernerauthor=E.+H.+Demont&title=Structure-Based+Optimization+of+Naphthyridones+into+Potent+ATAD2+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b00773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Sheppard, Robert J.; Barnett, Heather; Diallo, Hawa; Dixon, David P.; Douault, Clement; Jones, Emma J.; Karamshi, Bhumika; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Watson, Robert J.; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6151-6178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 is a bromodomain-contg. protein whose overexpression is linked to poor outcomes in a no. of different cancer types.  To date, no potent and selective inhibitors of the bromodomain have been reported.  This article describes the structure-based optimization of a series of naphthyridones from micromolar leads with no selectivity over the BET bromodomains to inhibitors with sub-100 nM ATAD2 potency and 100-fold BET selectivity, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcCY9tFp5dGbVg90H21EOLACvtfcHk0lj28NbF6JUb4A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK&md5=c7dcd72fdf4ab1c8557be45085cc68fa</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00773%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DStructure-Based%2520Optimization%2520of%2520Naphthyridones%2520into%2520Potent%2520ATAD2%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6151%26epage%3D6178%26doi%3D10.1021%2Facs.jmedchem.5b00773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6z7g" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6z7g','PDB','6z7g'); return false;">PDB: 6z7g</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4a9j" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4a9j','PDB','4a9j'); return false;">PDB: 4a9j</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6z7f" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6z7f','PDB','6z7f'); return false;">PDB: 6z7f</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6swq" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6swq','PDB','6swq'); return false;">PDB: 6swq</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6swp" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6swp','PDB','6swp'); return false;">PDB: 6swp</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6e6j" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6e6j','PDB','6e6j'); return false;">PDB: 6e6j</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i86"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00605">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_67768"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00605?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00605</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">S1: assay protocols, S2: crystallization methods, S3: Table S1 - data collection and refinement statistics for BRD2-BD2 X-ray structures, Figure S1: electron density maps for BRD2 BD2 X-ray structures, Figure S2: sequence alignment of BET proteins, Table S2: DiscoverX BROMOscan data for <b>59</b>, Table S3: cross screening data of liability panel for <b>59</b>, S4: DMPK methods, S5: .meteor <i>in silico</i> metabolite predictions of <b>28</b> and <b>59</b>, Figure S3: meteor <i>in silico</i> metabolite predictions of <b>28</b> and <b>59</b>, S6: <i>in vitro</i> metabolism of <b>28</b> in human and rat S9 fractions, Figure S4: <i>in vitro</i> metabolism of <b>28</b> in human and rat liver S9 fractions, S7: selected LCMS traces, S8: <sup>1</sup>H and <sup>13</sup>C NMR traces of compound <b>59</b>, Figure S5: <sup>1</sup>H and <sup>13</sup>C NMR traces of compound <b>59</b>, S9: BRD2 BD2 crystal structure: rational for favored stereochemistry, Figure S6: BRD2 BD2 crystal structure: rational for favored stereochemistry (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00605/suppl_file/jm0c00605_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00605/suppl_file/jm0c00605_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00605/suppl_file/jm0c00605_si_001.pdf">jm0c00605_si_001.pdf (2.78 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00605/suppl_file/jm0c00605_si_002.csv">jm0c00605_si_002.csv (2.63 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Authors will release the atomic coordinates upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00605%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-17%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00605" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67991f990e382367","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
