
<html lang="en"     class="pb-page"  data-request-id="d6916ece-5605-4e03-9edb-f72facb34a6c"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.9b00394;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-16;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Virtual Screening Approach and Investigation of Structure–Activity Relationships To Discover Novel Bacterial Topoisomerase Inhibitors Targeting Gram-Positive and Gram-Negative Pathogens" /></meta><meta name="dc.Creator" content="Gabriele  Magarò" /></meta><meta name="dc.Creator" content="Federica  Prati" /></meta><meta name="dc.Creator" content="Barbara  Garofalo" /></meta><meta name="dc.Creator" content="Gaia  Corso" /></meta><meta name="dc.Creator" content="Guido  Furlotti" /></meta><meta name="dc.Creator" content="Claudia  Apicella" /></meta><meta name="dc.Creator" content="Giorgina  Mangano" /></meta><meta name="dc.Creator" content="Noemi  D’Atanasio" /></meta><meta name="dc.Creator" content="Daniel  Robinson" /></meta><meta name="dc.Creator" content="Francesco Paolo  Di Giorgio" /></meta><meta name="dc.Creator" content="Rosella  Ombrato" /></meta><meta name="dc.Description" content="Bacterial resistance is increasing rapidly, requiring urgent identification of new antibacterial drugs that are effective against multidrug-resistant pathogens. Novel bacterial topoisomerase inhibi..." /></meta><meta name="Description" content="Bacterial resistance is increasing rapidly, requiring urgent identification of new antibacterial drugs that are effective against multidrug-resistant pathogens. Novel bacterial topoisomerase inhibi..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 5, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00394" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00394" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00394" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00394" /></link>
        
    
    

<title>Virtual Screening Approach and Investigation of Structure–Activity Relationships To Discover Novel Bacterial Topoisomerase Inhibitors Targeting Gram-Positive and Gram-Negative Pathogens | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00394" /></meta><meta property="og:title" content="Virtual Screening Approach and Investigation of Structure–Activity Relationships To Discover Novel Bacterial Topoisomerase Inhibitors Targeting Gram-Positive and Gram-Negative Pathogens" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0001.jpeg" /></meta><meta property="og:description" content="Bacterial resistance is increasing rapidly, requiring urgent identification of new antibacterial drugs that are effective against multidrug-resistant pathogens. Novel bacterial topoisomerase inhibitors (NBTIs) provide a new strategy for investigating the well-validated DNA gyrase and topoisomerase IV targets while preventing cross-resistance issues. On this basis, starting from a virtual screening campaign and subsequent structure-based hit optimization guided by X-ray studies, a novel class of piperazine-like NBTIs with outstanding enzymatic activity against Staphylococcus aureus and Escherichia coli DNA gyrase and topoisomerase IV was identified. Notably, compounds (±)-33, (±)-35, and (±)-36 with potent and balanced multitarget enzymatic profiles exhibited excellent efficacy against selected Gram-positive and Gram-negative pathogens, as well as clinically relevant resistant strains. Overall, the new NBTI chemotype described herein, owing to the broad-spectrum antibacterial activity and favorable in vitro safety profile, might serve as a basis for the development of novel treatments against serious infections." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00394"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00394">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00394&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00394&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00394&amp;href=/doi/10.1021/acs.jmedchem.9b00394" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 16</span><span class="cit-fg-pageRange">, 7445-7472</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/16" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00365" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00510" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Virtual Screening Approach and Investigation of Structure–Activity Relationships To Discover Novel Bacterial Topoisomerase Inhibitors Targeting Gram-Positive and Gram-Negative Pathogens</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Gabriele Magarò</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gabriele Magarò</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Angelini RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gabriele++Magar%C3%B2">Gabriele Magarò</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Federica Prati</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Federica Prati</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Angelini RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Federica++Prati">Federica Prati</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8483-7646" title="Orcid link">http://orcid.org/0000-0002-8483-7646</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Barbara Garofalo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Barbara Garofalo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Angelini RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Barbara++Garofalo">Barbara Garofalo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gaia Corso</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gaia Corso</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Angelini RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gaia++Corso">Gaia Corso</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guido Furlotti</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guido Furlotti</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Angelini RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guido++Furlotti">Guido Furlotti</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6715-9919" title="Orcid link">http://orcid.org/0000-0002-6715-9919</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Claudia Apicella</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Claudia Apicella</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Angelini RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Claudia++Apicella">Claudia Apicella</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Giorgina Mangano</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Giorgina Mangano</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Angelini RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Giorgina++Mangano">Giorgina Mangano</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Noemi D’Atanasio</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Noemi D’Atanasio</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Angelini RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Noemi++D%E2%80%99Atanasio">Noemi D’Atanasio</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniel Robinson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel Robinson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Schrodinger, 120 West 45th Street, New York, New York 10036, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel++Robinson">Daniel Robinson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Francesco Paolo Di Giorgio</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Francesco Paolo Di Giorgio</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Angelini RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Francesco+Paolo++Di+Giorgio">Francesco Paolo Di Giorgio</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Rosella Ombrato</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rosella Ombrato</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Angelini RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#94e6fbe7f1f8f8f5bafbf9f6e6f5e0fbd4f5faf3f1f8fdfafde4fcf5e6f9f5baf7fbf9"><span class="__cf_email__" data-cfemail="0c7e637f6960606d2263616e7e6d78634c6d626b69606562657c646d7e616d226f6361">[email protected]</span></a>. Phone: 0039 06 91045375. Fax: 0039 06 78332434.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rosella++Ombrato">Rosella Ombrato</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8625-599X" title="Orcid link">http://orcid.org/0000-0002-8625-599X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00394&amp;href=/doi/10.1021%2Facs.jmedchem.9b00394" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 16</span><span class="cit-pageRange">, 7445–7472</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 5, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>5 March 2019</li><li><span class="item_label"><b>Published</b> online</span>5 July 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 August 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00394" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00394</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7445%26pageCount%3D28%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DGabriele%2BMagar%25C3%25B2%252C%2BFederica%2BPrati%252C%2BBarbara%2BGarofalo%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D16%26contentID%3Dacs.jmedchem.9b00394%26title%3DVirtual%2BScreening%2BApproach%2Band%2BInvestigation%2Bof%2BStructure%25E2%2580%2593Activity%2BRelationships%2BTo%2BDiscover%2BNovel%2BBacterial%2BTopoisomerase%2BInhibitors%2BTargeting%2BGram-Positive%2Band%2BGram-Negative%2BPathogens%26numPages%3D28%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7472%26publicationDate%3DAugust%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00394"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1884</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00394" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Virtual Screening Approach and Investigation of Structure–Activity Relationships To Discover Novel Bacterial Topoisomerase Inhibitors Targeting Gram-Positive and Gram-Negative Pathogens&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Gabriele&quot;,&quot;last_name&quot;:&quot;Magarò&quot;},{&quot;first_name&quot;:&quot;Federica&quot;,&quot;last_name&quot;:&quot;Prati&quot;},{&quot;first_name&quot;:&quot;Barbara&quot;,&quot;last_name&quot;:&quot;Garofalo&quot;},{&quot;first_name&quot;:&quot;Gaia&quot;,&quot;last_name&quot;:&quot;Corso&quot;},{&quot;first_name&quot;:&quot;Guido&quot;,&quot;last_name&quot;:&quot;Furlotti&quot;},{&quot;first_name&quot;:&quot;Claudia&quot;,&quot;last_name&quot;:&quot;Apicella&quot;},{&quot;first_name&quot;:&quot;Giorgina&quot;,&quot;last_name&quot;:&quot;Mangano&quot;},{&quot;first_name&quot;:&quot;Noemi&quot;,&quot;last_name&quot;:&quot;D’Atanasio&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;Robinson&quot;},{&quot;first_name&quot;:&quot;Francesco&quot;,&quot;last_name&quot;:&quot;Paolo Di Giorgio&quot;},{&quot;first_name&quot;:&quot;Rosella&quot;,&quot;last_name&quot;:&quot;Ombrato&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;16&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;7445-7472&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00394&quot;},&quot;abstract&quot;:&quot;Bacterial resistance is increasing rapidly, requiring urgent identification of new antibacterial drugs that are effective against multidrug-resistant pathogens. Novel bacterial topoisomerase inhibitors (NBTIs) provide a new strategy for investigating the well-validated DNA gyrase and topoisomerase IV targets while preventing cross-resistance issues. On this basis, starting from a virtual screening campaign and subsequent structure-based hit optimization guided by X-ray studies, a novel class of piperazine-like NBTIs with outstanding enzymatic activity against Staphylococcus aureus and Escherichia coli DNA gyrase and topoisomerase IV was identified. Notably, compounds (±)-33, (±)-35, and (±)-36 with potent and balanced multitarget enzymatic profiles exhibited excellent efficacy against selected Gram-positive and Gram-negative pathogens, as well as clinically relevant resistant strains. Overall, the new NBTI chemotype described herein, owing to the broad-spectrum antibacterial activity and favorable in vitro sa&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00394&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00394" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00394&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00394" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00394&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00394" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00394&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00394&amp;href=/doi/10.1021/acs.jmedchem.9b00394" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00394" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00394" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (10 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00394%26sid%3Dliteratum%253Aachs%26pmid%3D31276392%26genre%3Darticle%26aulast%3DMagar%25C3%25B2%26date%3D2019%26atitle%3DVirtual%2BScreening%2BApproach%2Band%2BInvestigation%2Bof%2BStructure%25E2%2580%2593Activity%2BRelationships%2BTo%2BDiscover%2BNovel%2BBacterial%2BTopoisomerase%2BInhibitors%2BTargeting%2BGram-Positive%2Band%2BGram-Negative%2BPathogens%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D16%26spage%3D7445%26epage%3D7472%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (5)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=292202" title="Drug discovery">Drug discovery</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/16" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/jmcmar.2019.62.issue-16/20190822/jmcmar.2019.62.issue-16.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Bacterial resistance is increasing rapidly, requiring urgent identification of new antibacterial drugs that are effective against multidrug-resistant pathogens. Novel bacterial topoisomerase inhibitors (NBTIs) provide a new strategy for investigating the well-validated DNA gyrase and topoisomerase IV targets while preventing cross-resistance issues. On this basis, starting from a virtual screening campaign and subsequent structure-based hit optimization guided by X-ray studies, a novel class of piperazine-like NBTIs with outstanding enzymatic activity against <i>Staphylococcus aureus</i> and <i>Escherichia coli</i> DNA gyrase and topoisomerase IV was identified. Notably, compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> with potent and balanced multitarget enzymatic profiles exhibited excellent efficacy against selected Gram-positive and Gram-negative pathogens, as well as clinically relevant resistant strains. Overall, the new NBTI chemotype described herein, owing to the broad-spectrum antibacterial activity and favorable in vitro safety profile, might serve as a basis for the development of novel treatments against serious infections.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03243" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03243" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The increasing prevalence of resistant bacteria, such as methicillin-resistant <i>Staphylococcus aureus</i> (MRSA), vancomycin-resistant enterococci (VRE), and fluoroquinolone-resistant pathogens, represents a global health burden and poses the need for novel and effective antibiotics.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5">(1−5)</a> In this respect, the discovery of antibacterial compounds with new mechanisms of action might be desirable to reduce the likelihood of the emergence of resistance, improve activity against persistent infections, and provide improved safety.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Nevertheless, considerable research efforts remain dedicated to the development of new antibiotics that act on well-established and clinically validated targets. In particular, DNA gyrase and topoisomerase IV, which are members of the bacterial type IIA topoisomerase family, continue to attract significant attention from the scientific community. Both enzymes, which are known to regulate DNA topology during replication, bind to double-stranded DNA and cut the phosphate backbone, enabling the strand to be untangled or unwound. DNA gyrase introduces negative supercoils into DNA,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> while topoisomerase IV is largely responsible for the decatenation of the sister chromatids at the end of the replication process.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Both enzymes are heterotetramers; DNA gyrase is composed of GyrA<sub>2</sub>/GyrB<sub>2</sub> subunits, and topoisomerase IV consists of ParC<sub>2</sub>/ParE<sub>2</sub> subunits.</div><div class="NLM_p">Bacterial type IIA topoisomerases are well-validated targets as fluoroquinolones interact with the GyrA and ParC subunits of DNA gyrase and topoisomerase IV, respectively. Specifically, fluoroquinolones bind to the cleavage-ligation site located in the GyrA and ParC subunits, stabilize the double-stranded breaks in DNA, block the replication process, and finally induce an SOS response that leads to bacterial cell death.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Although fluoroquinolones have been used as successful frontline drugs for more than 50 years to treat broad-spectrum bacterial infections, resistance to these therapeutics has recently emerged due to mutations that alter the accumulation of these drugs in bacterial cells and affect molecular target recognition.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Therefore, new drug discovery approaches and technologies have been envisaged to identify novel topoisomerase inhibitors that can circumnavigate these resistance mechanisms and maintain clinical efficacy.</div><div class="NLM_p">In recent years, several pharmaceutical companies have independently discovered new classes of novel (nonfluoroquinolone) bacterial type IIA topoisomerase inhibitors (NBTIs), and the results of this research have gradually entered the public domain (compounds A–G in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26">(11−26)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0028.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0028.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. NBTI structures reported in the literature for (A) NXL101, (B) GSK2140944, (C) NBTI5463, (D) ACT-051, (E) AM-8722, (F) REDX06213, and (G) GSK299423 and most relevant compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> of the present study.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Importantly, several NBTIs showed lack of cross-resistance with fluoroquinolones and potent broad-spectrum antibacterial activity against the most problematic Gram-positive and Gram-negative pathogens, including efficacy in mouse models of infections. Notwithstanding the well-established cornerstones in the rational development of NBTIs, hERG liability still remains a pivotal encumbrance for their progression into clinical trials.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> In this respect, Novexel discontinued the development of NXL101 (compound A) due to unacceptable QT interval prolongation in healthy subjects during the early clinical phase.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> GlaxoSmithKline made more progress, having disclosed GSK2140944 (gepotidacin, compound B). Recently, GSK2140944 completed phase II clinical evaluation for the treatment of uncomplicated urogenital gonorrhea caused by <i>Neisseria gonorrhoeae</i> and for acute bacterial skin and skin structure infections caused by <i>S. aureus</i>.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> Additionally, GSK2140944 is in preclinical development for anthrax, plague, and tularemia infections.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><div class="NLM_p">Overall, the use of NBTIs is considered a suitable strategy for targeting clinically validated topoisomerase enzymes while restoring therapeutic efficacy. The mechanism of action of NBTIs was first disclosed in 2010,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> when Bax et al. reported the crystal structure of GSK299423 (compound G in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) in complex with <i>S. aureus</i> DNA gyrase and DNA (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>). The X-ray model unambiguously revealed the NBTI binding mode, confirming a binding site different from the one occupied by fluoroquinolones and highlighting key interactions of the ligand with both the DNA strand and GyrA subunits.</div><div class="NLM_p">To the best of our knowledge, the binding mode of NBTIs to DNA gyrases of other bacterial species, as well as topoisomerase IV enzymes, is unknown. However, comparison of the whole amino acid sequence of <i>S. aureus</i> DNA gyrase with that of <i>S. aureus</i> topoisomerase IV shows that these enzymes share a high degree of identity (35%). Similar levels of homology are also observed across species. Indeed, <i>E. coli</i> DNA gyrase and topoisomerase IV exhibit 47 and 35% identity with <i>S. aureus</i> DNA gyrase, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Figure S1</a>). Furthermore, a closer analysis revealed that highly conserved amino acids are present in the NBTI-binding regions of the DNA gyrase and topoisomerase IV enzymes of both <i>S. aureus</i> and <i>E. coli</i>, demonstrating that the key residues involved in the NBTI interaction are retained (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0027.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0027.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Amino acid sequence alignment of the binding sites of the DNA gyrase and topoisomerase IV enzymes in <i>S. aureus</i> and <i>E. coli</i>. The color scheme for the residues is as follows: blue for hydrophobic residues; red for acidic residues; green-yellow for basic residues; orange for other residues. The “Weight Colours by Alignment Quality” option of Schrödinger Release 2017-1: Prime, Schrödinger, LLC, New York, NY, was used for sequence alignment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Indeed, the first shells of interacting residues in the four enzymes differ at one position only (a conservative change of Met75 in <i>S. aureus</i> DNA gyrase). In particular, Met75 in <i>S. aureus</i> DNA gyrase is replaced by Ile in <i>E. coli</i> DNA gyrase and Leu in <i>E. coli</i> topoisomerase IV. This finding strongly suggests a conserved NBTI-binding region in all the considered type IIA topoisomerases, regardless of bacterial origin. Therefore, it is highly conceivable that small molecules that can simultaneously bind to DNA gyrase and topoisomerase IV enzymes from different microbial species could be identified, providing broad-spectrum antibacterial agents with a multitarget mechanism of action. Indeed, multitarget ligands represent an important prospect in antibacterial research as, although they will not overcome the issue of resistance in a long term, they might delay its onset and guarantee much improved longevity over single targeted agents. In fact, to develop high-level resistance to a multitargeted drug, bacteria would need sequential mutations in more than one essential target. Thus, if the inhibitory activity is well balanced against multiple targets, the likelihood of single-step resistance would be decreased. Importantly, successful systemic monotherapeutic antibacterials, such as fluoroquinolones, have been retrospectively discovered to inhibit DNA gyrase and topoisomerase IV simultaneously.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Therefore, the prospective design of multitarget ligands is an inviting opportunity for the development of effective monotherapeutic drugs that might even be beneficial over combination therapy in terms of simplification of the therapeutic regimen and pharmacokinetic profile, as well as obviation of possible drug–drug interactions.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">On these premises, we embarked on a drug discovery program (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) aimed at identifying a novel class of NBTIs. Specifically, a computer-aided drug design strategy using (i) virtual screening, (ii) shape screening, and (iii) pharmacophore search was applied.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0026.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0026.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Multistep approach of the Angelini NBTI drug discovery program.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The resulting virtual hits, independently selected from the three steps described above, were visually inspected and cherry-picked to be tested for inhibitory activity in DNA gyrase supercoiling and topoisomerase IV decatenation assays. From these initial investigations, an original class of piperazine derivatives was identified for further structure-based (SB) optimization. This campaign exploited an X-ray crystallographic study that retrospectively confirmed the predicted binding mode. Extensive medicinal chemistry research, resulting in compounds <b>1</b>–<b>36</b>, provided consistent structure–activity relationships (SARs) and led to the identification of potent NBTIs with excellent enzymatic and whole-cell activity against different bacterial strains.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65216" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65216" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Identification of Hit Molecules by a Virtual Screening Campaign</h3><div class="NLM_p">At the start of the computer-aided drug design investigation, the only relevant 3D cocrystal structure available for NBTI-like compounds was that of GSK299423 complexed with <i>S. aureus</i> DNA gyrase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Nevertheless, the chemical structures of several NBTIs were already published in the public domain, thus providing an opportunity for integrated structure-based virtual screening (SBVS) and ligand/pharmacophore-based virtual screening (LBVS) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><div class="NLM_p">To evaluate the suitability of the GSK299423-<i>S. aureus</i> DNA gyrase 3D structure<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> for virtual screening, the Schrödinger’s Glide program was used.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The first step was to ensure that simple redocking correctly reproduced the information contained within the crystal structure, and the cognate ligand scored suitably high (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Figure S2</a>). Simultaneously, a set of 44 known active compounds from the NBTI-related literature (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Table S1</a>) were docked, and the poses of these compounds were inspected for basic concordance with the NBTI structural outline. The shape generated by docking (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Figure S3</a>) confirmed the conserved binding mode of the 44 NBTIs and the overall similarity to the GSK299423 pose, which was faithfully reproduced and received a very favorable “docking score” (−11.7 kcal/mol).</div><div class="NLM_p">The ability of this structural model to enrich for NBTIs was then tested by combining ligands from the literature with a decoy dataset. The Glide decoy set (available from the Schrödinger Inc. website <a href="https://www.schrodinger.com/glide" class="extLink">https://www.schrodinger.com/glide</a>) was downloaded and filtered to extract only those decoys that had physicochemical properties that were similar to those of known NBTIs (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Figure S4</a>). This complete database was then run through a “mock” SBVS. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, the enrichment factor (EF) of the 44 compounds suggests a favorable performance in the early recognition with Glide using default settings (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for further details).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0025.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0025.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Graph showing enrichment factors for standard precision (blue line) docking of the dataset (44 compounds) into the <i>S. aureus</i> DNA gyrase model based on PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>. The red and green lines represent perfect and random ligand retrieval, respectively. (B) Enrichment factor data in tabular form.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Given the expected performance of the SBVS protocol, we subsequently set up the main screening. To explore a large area of chemical space and increase the chemical diversity in the virtual screening campaign, a collection of over 30,000 compounds was prepared by including both corporate and commercial libraries, the latter being selected as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. The 3D ligand structures were subsequently docked into the <i>S. aureus</i> DNA gyrase model (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The best docking results (docking score ≤ −7.00 kcal/mol) were then visually inspected for key interactions with the binding site, with particular emphasis on the presence of a central amino group. This procedure resulted in approximately 500 virtual hits from different chemical classes.</div><div class="NLM_p">With only a single 3D structure available (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>)<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> at the time of investigation, there was the chance of a significant bias entering the results. Consequently, we augmented the SBVS with a pharmacophore-based model derived solely from the structures of known NBTIs. In this regard, eight compounds from the literature (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Table S2</a>),<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> including GSK299423, were carefully evaluated on the basis of chemical diversity, and a conformationally constrained subset was generated. As a class, NBTIs consist of three structural motifs: (i) an unsaturated heterocyclic left-hand side (LHS), (ii) a mono- or bicyclic right-hand side (RHS), and (iii) a largely aliphatic linker, which joins the LHS and RHS portions in an appropriate conformation (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0024.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Three structural motifs of NBTI compounds (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BUL">4BUL</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The Phase pharmacophore modeling package<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> was used to identify the low-energy conformations that maximized the overlapping of the three portions of the NBTI structures. From this alignment, several different pharmacophore hypotheses were generated, combining default settings and custom features in Phase using either the vector or the projected point description for the acceptor groups. A minimum of 75% of the literature molecules had to match with the hypothesis. The two best models had five features: two hydrogen bond acceptors (A) and one ring (R) modeling the interactions with DNA, one hydrogen bond donor (D) or positive group (P) describing the interactions with Asp83, and finally, one aromatic/hydrophobic ring (R) representing the hydrophobic group that is nestled within the protein. According to the specific combination of the molecular features, the two selected pharmacophores were indicated as follows: (i) AADRR and (ii) AAPRR (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Figure S5</a>). Following the standard practice in virtual screening, the pharmacophore models were first assessed for their ability to recognize the bioactive conformation of GSK299423 and then, in terms of EF, using the 44 compounds and decoy dataset (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). All five sites matched well with the dataset compounds. These two pharmacophore models were then used to screen the 30,000 compound library discussed earlier, specifying that the compound must match all five features to be considered as a hit. Approximately 200 compounds were selected using this approach.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0023.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Graph showing the enrichment factors of the selected pharmacophore models AADRR (purple) and AAPRR (turquoise). The red and green lines represent perfect and random ligand retrieval, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">All NBTIs have a fairly distinctive shape, and given the speed of shape-based virtual screening,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> it seemed appropriate that an additional shape-based screen was conducted. The literature and decoy ligands were used to validate the shape model. This procedure retrieved a marked number of active compounds in the top 2% of the ranked dataset and full recovery in the top 5% (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Table S3</a>). In light of these results, searching the 30,000 compound library using identical settings allowed us to identify approximately 200 additional compounds.</div><div class="NLM_p last">In total, the computational screens aggregated approximately 900 virtual hits. Moreover, an additional set of filters and clustering was applied to remove redundancy in the selected compounds (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for further details). Eventually, 162 compounds (<b>ID-9</b> to <b>ID-162</b>) were selected and taken forward for experimental screening.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Biological Testing</h3><div class="NLM_p">Enzyme inhibition, expressed as an IC<sub>50</sub> value, was assessed against DNA gyrase and topoisomerase IV from both <i>S. aureus</i> and <i>E. coli</i> in supercoiling and decatenation assays, respectively.</div><div class="NLM_p">Among the 162 tested compounds, 12 (selected according to the methods and ranking reported in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Table S4</a>) were confirmed to inhibit one or both enzymes in the supercoiling and decatenation assays, and the IC<sub>50</sub> values are reported in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Results of the Biological Assay of Identified Compounds with Inhibitory Activity against DNA Gyrase and Topoisomerase IV of <i>E. coli</i> and <i>S. aureus</i></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center"><i>E. coli</i><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="rowsep1 colsep0" colspan="2" align="center"><i>S. aureus</i><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">DNA gyrase<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">topoisomerase IV<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center">DNA gyrase<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">topoisomerase IV<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>ID-9</b></td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">63</td><td class="colsep0 rowsep0" align="left">>800</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>ID-14</b></td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">265</td><td class="colsep0 rowsep0" align="left">>800</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>ID-46</b></td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">4.1</td><td class="colsep0 rowsep0" align="left">27.8</td><td class="colsep0 rowsep0" align="left">100–200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>ID-62</b></td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">378.2</td><td class="colsep0 rowsep0" align="left">109.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>ID-64</b></td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">301</td><td class="colsep0 rowsep0" align="left">>400<a class="ref internalNav" href="#t1fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">>800</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>ID-72</b></td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">50.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>ID-81</b></td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">42.1</td><td class="colsep0 rowsep0" align="left">500–800</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>ID-96</b></td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">115.7</td><td class="colsep0 rowsep0" align="left">104.9</td><td class="colsep0 rowsep0" align="left">>400<a class="ref internalNav" href="#t1fn6" aria-label="f">f</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>ID-110</b></td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">55</td><td class="colsep0 rowsep0" align="left">365.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>ID-120</b></td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">104</td><td class="colsep0 rowsep0" align="left">>800</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>ID-126</b></td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">>800</td><td class="colsep0 rowsep0" align="left">145.8</td><td class="colsep0 rowsep0" align="left">513</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>ID-130</b></td><td class="colsep0 rowsep0" align="left">112</td><td class="colsep0 rowsep0" align="left">130</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">489</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were calculated on concentration–response curves using the global curve fit nonlinear regression tool. Compounds were assayed at eight concentrations in duplicate with twofold or half-log dilutions starting from 800 μM.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last"><i>E. coli</i> ATCC 25922.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last"><i>S. aureus</i> ATCC 29213.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Compounds were tested in DNA gyrase supercoiling assay kit (Norwich, U.K.).</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Compounds were tested in topoisomerase IV decatenation assay kit (Norwich, U.K.).</p></div><div class="footnote" id="t1fn6"><sup><sup>f</sup></sup><p class="last">Compounds were tested starting from 400 μM concentration.</p></div></div></div><div class="NLM_p">Four out of the 12 active compounds (<b>ID-46</b>, <b>ID-62</b>, <b>ID-110</b>, and <b>ID-130</b>) exhibited IC<sub>50</sub> values lower than 400 μM for at least one DNA gyrase and one topoisomerase IV enzyme, resulting in dual inhibitors. Two of the most promising hits, piperazine <b>ID-130</b> and aminopiperidine <b>ID-46</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>), showed potency toward <i>S. aureus</i> DNA gyrase and topoisomerase IV in the double-digit and high-micromolar ranges, respectively. Importantly, compound <b>ID-130</b>, with promising inhibitory activity also against both <i>E. coli</i> enzymes, was tested for antibacterial potency (expressed as minimum inhibitory concentration or MIC) toward a small but representative panel of Gram-negative and Gram-positive strains comprising <i>E. coli</i> ATCC 25922, <i>S. aureus</i> ATCC 29213, and <i>S. aureus</i> BAA-1720 (known as MRSA, a fluoroquinolone-resistant strain). MICs were determined according to the guidelines of the Clinical and Laboratory Standards Institute (CLSI).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0022.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Design strategy for compound <b>1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Notably, despite being less potent than the reference compound ciprofloxacin in Gram-negative <i>E. coli</i> (MIC = 256 μg/mL vs 0.016 μg/mL) and Gram-positive <i>S. aureus</i> (MIC = 16 μg/mL vs 0.5 μg/mL), <b>ID-130</b> demonstrated more pronounced antibacterial activity against fluoroquinolone-resistant Gram-positive strains, such as <i>S. aureus</i> BAA-1720 (MIC = 32 μg/mL vs >64 μg/mL) (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Enzymatic and Antibacterial Activity of the Selected Hit Compound (<b>ID-130</b>)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="rowsep1 colsep0" colspan="3" align="center">MIC (μg/mL)<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">DNA gyrase<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th><th class="rowsep1 colsep0" colspan="2" align="center">topoisomerase IV<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><i>E. coli</i><a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center"><i>S. aureus</i><a class="ref internalNav" href="#t2fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center"><i>E. coli</i><a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center"><i>S. aureus</i><a class="ref internalNav" href="#t2fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center"><i>E. coli</i><a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center"><i>S. aureus</i><a class="ref internalNav" href="#t2fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center"><i>S. aureus</i> BAA-1720<a class="ref internalNav" href="#t2fn7" aria-label="g">g</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ciprofloxacin</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">187</td><td class="colsep0 rowsep0" align="left">51</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">0.016</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">>64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>ID-130</b></td><td class="colsep0 rowsep0" align="left">112</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">130</td><td class="colsep0 rowsep0" align="left">489</td><td class="colsep0 rowsep0" align="left">256</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">32</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were calculated on concentration–response curves using the global curve fit nonlinear regression tool. Compounds were assayed at eight concentrations in duplicate with twofold or half-log dilutions starting from 800 μM.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Minimum inhibitory concentration were determined by the broth microdilution method according to guidelines of the CLSI.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Compounds were tested in DNA gyrase supercoiling assay kit (Norwich, U.K.).</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">Compounds were tested in topoisomerase IV decatenation assay kit (Norwich, U.K.).</p></div><div class="footnote" id="t2fn5"><sup><sup>e</sup></sup><p class="last"><i>E. coli</i> ATCC 25922.</p></div><div class="footnote" id="t2fn6"><sup><sup>f</sup></sup><p class="last"><i>S. aureus</i> ATCC 29213.</p></div><div class="footnote" id="t2fn7"><sup><sup>g</sup></sup><p class="last">Methicillin-resistant <i>S. aureus</i> (MRSA).</p></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> X-ray Crystallographic Structure Determination</h3><div class="NLM_p">Hit compound <b>ID-130</b> was submitted for X-ray crystallography to confirm the putative binding mode proposed by computational studies. Initial attempts to cocrystallize the ternary system composed of <b>ID-130</b>, <i>S. aureus</i> DNA gyrase, and a short strand of DNA were not successful. Therefore, docking models based on the 2XCS structure<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> were used as references for much of the SB optimization process.</div><div class="NLM_p">Nonetheless, a cocrystal structure of <b>ID-130</b> in <i>S. aureus</i> DNA gyrase truncated gyrB27-A56 (specifically GyrB27-A56 (GKdel/Tyr123Phe)) fusion construct and 20 base-pair synthetic DNA binding site fragment was finally obtained (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FM4">6FM4</a>) at a 2.7 Å resolution (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>) (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Supporting Information</a> for the experimental details).</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0021.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. X-ray crystal structure of <b>ID-130</b> (represented as a yellow tube) in the <i>S. aureus</i> DNA gyrase site (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FM4">6FM4</a>). The DNA strand is shown as a green and blue tube structure; the subunits of the protein are shown as green and blue ribbons. Hydrogen bonds are represented as yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The comparatively low resolution of this structure made detailed modeling of the complex difficult, in particular, the precise conformation of some of the most flexible sections of the ligand. However, the result was fairly consistent with similar structures in the public domain; for example, the protein and DNA chains could be convincingly superimposed onto the 2XCS structure, yielding an RMSD of 0.35 Å. Additionally, the 3D structure was even consistent with the previous modeling work carried out on <b>ID-130</b>. The isoquinoline ring (LHS) was bound between the two central base pairs of the stretched DNA. The hydrogen of the amidic group at the bottom of the linker was engaged in H-bonding interactions with the carboxylate of Asp83′, and the phenyl moiety (RHS) pointed toward the hydrophobic pocket between the two DNA gyrase subunits.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Chemistry</h3><div class="NLM_p">After identification of compounds <b>ID-130</b> and <b>ID-46</b>, a hit optimization campaign was started to improve the enzymatic and antibacterial activities of these compounds via iterative cycles of SB drug design and synthesis based on biochemical and whole-cell screening. This effort generated a novel series of <i>N</i>-linked piperazines (<b>1</b>–<b>36</b> in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR of the Left-Hand Side (LHS)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0016.gif" alt="" id="gr21" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0010.gif" alt="" id="gr22" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were calculated on concentration–response curves using the global curve fit nonlinear regression tool. Compounds were assayed at eight concentrations in duplicate with twofold or half-log dilutions starting from 100 μM.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Compounds were tested in DNA gyrase supercoiling assay kit (Norwich, U.K.).</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Compounds were tested in topoisomerase IV decatenation assay kit (Norwich, U.K.).</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last"><i>E. coli</i> ATCC 25922.</p></div><div class="footnote" id="t3fn5"><sup><sup>e</sup></sup><p class="last"><i>S. aureus</i> ATCC 29213.</p></div></div><div></div></div><div class="NLM_p">Compounds <b>1</b>–<b>36</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>) were synthesized by different synthetic approaches, as depicted in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>–<a class="ref internalNav" href="#sch8" aria-label="8">8</a>. Compounds <b>1</b>–<b>4</b>, <b>6</b>, <b>11</b>, <b>12</b>, <b>14</b>–<b>23</b>, <b>27</b>, <b>28</b>, and <b>30</b> were obtained via a multistep synthetic route, starting from the proper commercially available 1-chloroisoquinoline reagent (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0002.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>1</b>–<b>4</b>, <b>6</b>, <b>11</b>, <b>12</b>, <b>14</b>–<b>23</b>, <b>27</b>, <b>28</b>, and <b>30</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert-</i>butyl piperazine-1-carboxylate, K<sub>2</sub>CO<sub>3</sub>, DMSO, 110 °C; (b) TFA, DCM, rt; (c) <i>tert-</i>butyl (3-bromopropyl)carbamate (for intermediates <b>IIIa</b>–<b>IIIg</b>) or <i>tert-</i>butyl (2-bromoethyl)carbamate (for intermediate <b>IIIh</b>), KI, K<sub>2</sub>CO<sub>3</sub>, DMF, rt; or <i>tert</i>-butyl (3-oxocyclobutyl)carbamate, NaBH(OAc)<sub>3</sub>, DCM, rt, for intermediates <b>IIIi</b> and <b>IIIj</b>; (d) NaH, CH<sub>3</sub>I, DMF, rt; (e) triflic anhydride, TEA, DCM, −10 °C and then rt; (f) TEA, CH<sub>3</sub>CN, 70 °C, followed by Boc deprotection under standard conditions for compound <b>19</b>.</p></p></figure><div class="NLM_p"><i>tert</i>-Butyl 4-(isoquinolin-1-yl)piperazine-1-carboxylates <b>Ia</b>–<b>Ih</b> were obtained by an S<sub>N</sub>Ar reaction of the proper 1-chloroisoquinoline with <i>tert-</i>butyl piperazine-1-carboxylate. Subsequent deprotection of <b>Ia</b>–<b>Ih</b> under standard conditions afforded the corresponding piperazines <b>IIa</b>–<b>IIh</b>. N-protected amines <b>IIIa</b>–<b>IIIj</b> were obtained either by (i) alkylation of <b>IIa</b>–<b>IIg</b> using <i>tert-</i>butyl (3-bromopropyl)carbamate (for intermediates <b>IIIa</b>–<b>IIIg</b>) or <i>tert-</i>butyl (2-bromoethyl)carbamate (for intermediate <b>IIIh</b>) or (ii) reduction of the iminium intermediate generated in situ by reaction of amine <b>IIa</b> with <i>tert</i>-butyl (3-oxocyclobutyl)carbamate (for <i>cis</i> and <i>trans</i> isomers <b>IIIi</b> and <b>IIIj</b>, respectively). N-methylation of <i>tert-</i>butyl {3-[4-(isoquinolin-1-yl)piperazin-1-yl]propyl}carbamate <b>IIIa</b> led to intermediate <b>IVa</b>. Subsequently, Boc-protected intermediates <b>IIIa</b>–<b>IIIj</b> and <b>IVa</b> underwent deprotection under standard conditions to afford primary amines <b>Va</b>–<b>Vk</b>. Finally, the reaction of amines <b>Va</b>–<b>Vk</b> with the proper pyran-2-ones <b>VIa</b>–<b>VIj</b>, prepared by triflation of the corresponding hydroxyl precursors, provided the final compounds <b>1</b>–<b>4</b>, <b>6</b>, <b>11</b>, <b>12</b>, <b>14</b>–<b>23</b>, <b>27</b>, <b>28</b>, and <b>30</b>.</div><div class="NLM_p">The synthetic route for compounds <b>5</b>, <b>7</b>–<b>10</b>, and <b>13</b> involved the preparation of the key Boc-protected common intermediate <b>VIIa</b>, starting from <b>VIa</b> and the commercial <i>tert</i>-butyl 4-(3-aminopropyl)piperazine-1-carboxylate. Subsequent deprotection of <b>VIIa</b> afforded amines <b>VIIIa</b>, which was coupled in a parallel fashion with the proper chloroisoquinolines or chloronaphthyridines to afford target compounds <b>5</b>, <b>7</b>–<b>10</b>, and <b>13</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch2" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0003.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>5</b>, <b>7</b>–<b>10</b>, and <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert</i>-butyl 4-(3-aminopropyl)piperazine-1-carboxylate, TEA, CH<sub>3</sub>CN, 70 °C; (b) TFA, DCM, rt; (c) K<sub>2</sub>CO<sub>3</sub>, DMSO, 110 °C.</p></p></figure><div class="NLM_p">For synthesis of the <i>O</i>-derivative <b>24</b>, the hydroxyl intermediate <b>IXa</b> was prepared via a MW-assisted S<sub>N</sub>Ar of 1-(3-hydroxy-1-propyl)piperazine on 1-chloroisoquinoline. Conversion of the hydroxyl derivative <b>IXa</b> into mesylate <b>Xa</b> followed by reaction with 4-hydroxy-2<i>H</i>-1-benzopyran-2-one under standard coupling conditions provided the target compound (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>).</div><figure id="sch3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0004.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1-(3-hydroxy-1-propyl)piperazine, K<sub>2</sub>CO<sub>3</sub>, DMSO, 120 °C, MW; (b) CH<sub>3</sub>SO<sub>2</sub>Cl, TEA, DCM, 0 °C to rt; (c) 4-hydroxy-2<i>H</i>-1-benzopyran-2-one, NaH, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 80 °C.</p></p></figure><div class="NLM_p">Amide <b>25</b> was prepared using a similar strategy as described for compound <b>24</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). Amide <b>XIa</b>, prepared by the reaction of 1-chloro-7-fluoroisoquinoline with 3-(piperazin-1-yl)propenamide, was coupled by Buchwald cross-coupling with coumarin <b>VIa</b> in the presence of Pd(dba)<sub>2</sub> and Xantphos as the catalyst/ligand system.</div><figure id="sch4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0005.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compound <b>25</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 3-(piperazin-1-yl)propanamide, K<sub>2</sub>CO<sub>3</sub>, DMSO, 110 °C; (b) <b>VIa</b>, K<sub>3</sub>PO<sub>4</sub>, Pd(dba)<sub>2</sub>, Xantphos, dry dioxane, 100 °C.</p></p></figure><div class="NLM_p">Compound <b>26</b> was synthesized as depicted in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. Intermediate <b>XIIa</b>, prepared by S<sub>N</sub>Ar of piperazin-2-one on 1-chloroisoquinoline, was alkylated with <i>N</i>-Boc-3-bromopropylamine to give the protected amine <b>XIIIa</b>. Subsequent deprotection of <b>XIIIa</b> to obtain <b>XIVa</b> and reaction with <b>VIa</b> provided the target compound.</div><figure id="sch5" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0006.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compound <b>26</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) piperazin-2-one, K<sub>2</sub>CO<sub>3</sub>, DMSO, 110 °C; (b) <i>tert</i>-butyl <i>N</i>-(3-bromopropyl)carbamate, NaH, DMF, rt; (c) TFA, DCM, rt; (d) <b>VIa</b>, TEA, CH<sub>3</sub>CN, 70 °C.</p></p></figure><div class="NLM_p">According to <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>, the synthesis of the hydroxyl derivatives <b>(<i>R</i>)-29</b>, <b>(<i>S</i>)-29</b>, <b>(<i>R</i>)-34</b>, and racemic <b>(±)-35</b> and <b>(±)-36</b> involved the preparation of intermediates <b>XVa</b>–<b>XVd</b>.</div><figure id="sch6" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0007.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compounds <b>(<i>S</i>)-29</b>, <b>(<i>R</i>)-29</b>, <b>(<i>R</i>)-34</b>, <b>(±)-35</b>, and <b>(±)-36</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (2<i>R</i>)-<i>N</i>-(2,3-epoxypropyl)phthalimide or (2<i>S</i>)-<i>N</i>-(2,3-epoxypropyl)phthalimide, CH<sub>3</sub>CN, 70 °C; (b) MeNH<sub>2</sub>, EtOH, 40 °C; (c) epichlorohydrin, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), EtOH, rt; (d) NaN<sub>3</sub>, DMF, 85 °C; (e) H<sub>2</sub>, 10% Pd/C, EtOH, rt; (f) <b>VIa</b> or <b>XVIa</b>, TEA, CH<sub>3</sub>CN, 70 °C; (g) <b>XVIb</b>, DIPEA, CH<sub>3</sub>CN, 0 °C to rt; (h) PCl<sub>5</sub>, DCE, 0 °C to reflux. Compounds <b>(<i>S</i>)-29</b>, <b>(<i>R</i>)-29</b>, and <b>(<i>R</i>)-34</b> were obtained following steps (a), (b), and (f); racemic compounds <b>(±)-35</b> and <b>(±)-36</b> were obtained following steps (c)–(e) and (g).</p></p></figure><div class="NLM_p">Compounds <b>(<i>R</i>)-XVa</b> and <b>(<i>S</i>)-XVb</b> were prepared via the ring-opening reaction of (2<i>R</i>)-<i>N</i>-(2,3-epoxypropyl)phthalimide and (2<i>S</i>)-<i>N</i>-(2,3-epoxypropyl)phthalimide by piperazine <b>IIb</b>, respectively, followed by treatment with methylamine. Intermediates <b>(±)-XVc</b> and <b>(±)-XVd</b> were obtained by the reaction of racemic epichlorohydrine with piperazines <b>IIb</b> and <b>IIh</b>, respectively, followed by conversion of the resulting chloride into azide and its catalytic reduction to the corresponding amine. To afford the target compounds, the resulting amines <b>XVa</b>–<b>XVd</b> were then reacted with triflate <b>VIa</b> or chloride <b>XVIa</b> and <b>XVIb</b>, prepared via chlorination of the corresponding 4-hydroxy-coumarin or 4-hydroxy-2<i>H</i>-1,3-benzoxazin-2-one with phosphorus pentachloride.</div><div class="NLM_p">Synthesis of compound <b>(±)-31</b> involved the preparation of chloride <b>XVIIa</b> from 4-methylcoumarin and 3-chloropropanoyl chloride using LiHDMS. The S<sub>N</sub>2 reaction of <b>XVIIa</b> with 7-fluoro-1-(piperazin-1-yl)isoquinoline <b>IIb</b> led to ketone <b>XVIIIa</b>, which underwent subsequent reduction of the carbonyl group by sodium borohydride to yield the target compound (<a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>).</div><figure id="sch7" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0008.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Compound <b>(±)-31</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) LiHDMS, 3-chloropropanoyl chloride, dry THF, −78 °C to rt; (b) 7-fluoro-1-(piperazin-1-yl)isoquinoline (<b>IIb</b>), K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 45 °C; (c) NaBH<sub>4</sub>, MeOH, rt.</p></p></figure><div class="NLM_p">The synthesis of compounds <b>(±)-32</b> and <b>(±)-33</b> is illustrated in <a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>.</div><figure id="sch8" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0009.jpeg" id="gr20" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of Compounds <b>(±)-32</b> and <b>(±)-33</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TEA, CH<sub>3</sub>CN, 70 °C; (b) HCl in dioxane, MeOH, rt; (c) 5-chloro-1,6-naphthyridine, K<sub>2</sub>CO<sub>3</sub>, DMSO, 110 °C for <b>(±)-32</b>; or 1-chloro-3-fluoro-7-methoxyisoquinoline, DIPEA, DMSO, 100 °C for <b>(±)-33</b>.</p></p></figure><div class="NLM_p last">Reaction of triflate V<b>Ib</b> with (±)-<i>tert</i>-butyl 4-(3-amino-2-hydroxypropyl)piperazine-1-carboxylate provided the Boc-protected intermediate <b>(±)-XIXa</b>. Subsequent amine deprotection of <b>(±)-XIXa</b> provided amine <b>(±)-XXa</b>, which was used in the final S<sub>N</sub>Ar reaction with 5-chloro-1,6-naphthyridine and 1-chloro-3-fluoro-7-methoxyisoquinoline to afford <b>(±)-32</b> and <b>(±)-33</b>, respectively.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Merging and Modeling Approaches</h3><div class="NLM_p">Starting from the X-ray structural data and docking model (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Figure S6</a>) generated for <b>ID-130</b> and <b>ID-46</b>, respectively, we envisaged a merging approach to combine the isoquinoline moiety of <b>ID-130</b> (LHS) and the 4-aminocoumarin portion of <b>ID-46</b> (RHS) to form a new chemical entity (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). This strategy resulted in compound <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>), which retained the same binding pose as the parent hits and showed improved enzymatic activity.</div><div class="NLM_p">Compound <b>1</b> inhibited <i>E. coli</i> DNA gyrase and topoisomerase IV and <i>S. aureus</i> DNA gyrase with IC<sub>50</sub> values ranging from 0.5 to 14 μM, whereas the inhibitory activity toward <i>S. aureus</i> topoisomerase IV was 90 μM, showing a favorable multitarget profile and confirming the value of the applied merging strategy.</div><div class="NLM_p">To help rationalize the obtained enzymatic data and assess the solvent contribution to the target binding affinity of compound <b>1</b>, Schrödinger’s WaterMap application<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> was used. WaterMap is a molecular dynamics (MD)-based computational method that uses statistical mechanics to describe the thermodynamic properties (entropy, enthalpy, and free energy) of water molecules (hydration sites) in proteins and protein–ligand complexes, thus revealing the energetic effects of solvation and desolvation on binding and ultimately guiding hit/lead optimization. WaterMap analysis of compound <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a> and <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a>) modeled into the <i>S. aureus</i> DNA gyrase crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>)<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> showed that the LHS and RHS moieties replaced unstable hydration sites within the DNA-binding region and the hydrophobic pocket at the interface between the GyrA chains, respectively. These findings supported the promising inhibitory activity of compound <b>1</b>, which served as the basis for further SB drug design efforts.</div><figure id="fig9" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0020.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Compound <b>1</b> was modeled into the <i>S. aureus</i> DNA gyrase crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>). The LHS portion of the ligand is represented in a ball-and-stick model. The red surface shows the free-energy density of the solvent calculated by WaterMap, together with the WaterMap-generated hydration sites (red and green spheres). Red spheres indicate unstable water molecules with positive Δ<i>G</i> values, and green spheres indicate stable water molecules with negative Δ<i>G</i> values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig10" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0019.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Compound <b>1</b> was modeled into the <i>S. aureus</i> DNA gyrase crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>). The RHS portion of the ligand is represented in a ball-and-stick model. The red surface shows the free-energy density of the solvent calculated by WaterMap, together with the WaterMap-generated hydration sites (red and green spheres). The gray surface represents the extent of the protein from the 2XCS structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Starting from compound <b>1</b>, an optimization strategy was applied, providing novel analogues by modifying the three identifiable structural motifs: LHS (isoquinoline), RHS (coumarin), and linker (aminopropyl-piperazine).</div><div id="sec2_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> LHS SAR</h4><div class="NLM_p">LHS modifications were centered on the introduction of different substituents at various positions on the bicyclic ring of compound <b>1</b>, as well as replacement of the isoquinoline moiety by other heterocycles, affording compounds <b>2</b>–<b>13</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div class="NLM_p">As anticipated above, WaterMap studies indicated that the nitrogen of the isoquinoline of <b>1</b> was relevant for binding because this moiety replaced an unstable hydration site and allowed the ligand to better fit into the natural structure of the solvent in and around the DNA strand (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). This result was consistent with the loss of activity in the supercoiling and decatenation assays for the equivalent napthyl-bearing ligand (data not shown).</div><div class="NLM_p">Furthermore, collateral “high-energy” hydration sites, close to the isoquinoline moiety and accessible from the 7-position of the bicyclic system, were identified. Therefore, the introduction of different 7-substituents on the isoquinoline ring was predicted to possibly displace these unstable water molecules and increase binding affinity. The resulting compounds <b>2</b>–<b>7</b> showed an enhanced and more balanced multitarget profile with respect to the parent hit <b>1</b>, validating the rational design. Specifically, compounds <b>2</b> (7-F) and <b>3</b> (7-Cl) retained submicromolar potency against <i>E. coli</i> DNA gyrase (IC<sub>50</sub> = 0.8 and 0.9 μM, respectively), and all derivatives <b>2</b>–<b>7</b> showed ameliorated inhibitory capacity against the remaining tested topoisomerases, especially <i>S. aureus</i> topoisomerase IV.</div><div class="NLM_p">From the modeling and WaterMap results, compound <b>7</b>, harboring a large 7-OCH<sub>3</sub> group, was predicted to displace the high-energy hydration sites from this region to a greater extent. This ability was likely reflected in the overall profile of compound <b>7</b>, with IC<sub>50</sub> values of 1–7 μM for all four enzymes.</div><div class="NLM_p">Modeling suggested that there was a limited scope for modification of any other positions on the isoquinoline ring. A few substitutions of the 3-position were considered, which generally confirmed this view (compounds <b>8</b>–<b>12</b>). Small groups, such as 3-F (<b>8</b>), 3-Cl (<b>9</b>), and 3-CH<sub>3</sub> (<b>11</b>), retained or improved the general inhibitory profile with respect to <b>1</b>. In particular, compounds <b>8</b>, <b>9</b>, and <b>11</b> showed significantly enhanced inhibition of <i>S. aureus</i> topoisomerase IV (IC<sub>50</sub> = 7.1, 10.2, and 7.5 μM, respectively). Conversely, larger groups, such as 3-CN (<b>10</b>) and 3-OCH<sub>3</sub> (<b>12</b>), showed poor inhibitory profiles overall.</div><div class="NLM_p">These findings are entirely consistent with data reported in the literature for other NBTIs (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Indeed, the vast majority of NBTIs has small substituents similarly placed in the 3- and 7-positions of the isoquinoline ring, as discovered in the current work.</div><div class="NLM_p last">Finally, the replacement of the isoquinoline ring with a 1,6-naphthyridine moiety (<b>13</b>) resulted in weakening of inhibitory potency.</div></div><div id="sec2_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> RHS SAR</h4><div class="NLM_p">RHS modifications were centered on the introduction of different substituents at various positions of the coumarin rings of <b>1</b> and <b>2</b>, as well as replacement of the bicyclic system by other heterocycles, affording compounds <b>14</b>–<b>22</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR of the Right-Hand Side (RHS)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0015.gif" alt="" id="gr23" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0014.gif" alt="" id="gr24" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were calculated on concentration–response curves using the global curve fit nonlinear regression tool. Compounds were assayed at eight concentrations in duplicate with twofold or half-log dilutions starting from 100 μM.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Compounds were tested in DNA gyrase supercoiling assay kit (Norwich, U.K.).</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">Compounds were tested in topoisomerase IV decatenation assay kit (Norwich, U.K.).</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last"><i>E. coli</i> ATCC 25922.</p></div><div class="footnote" id="t4fn5"><sup><sup>e</sup></sup><p class="last"><i>S. aureus</i> ATCC 29213.</p></div></div><div></div></div><div class="NLM_p">Models of compound <b>1</b> in the <i>S. aureus</i> DNA gyrase crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>)<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> showed the coumarin ring buried in a small, highly hydrophobic depression at the surface of the DNA gyrase (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>), with the carbonyl moiety water-bridging to Arg122. Initial static modeling indicated a fairly limited scope for substitution of the coumarin, although dynamics-based calculations did indicate plasticity.</div><div class="NLM_p">The observed SAR was mostly consistent with these theoretical predictions, albeit with some exceptions (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Notably, hydrophobic analogues of <b>1</b>, bearing a 6-Cl (<b>14</b>) or 6-F (<b>15</b>) coumarin, exhibited IC<sub>50</sub> values in the range of 0.6–5.5 μM on the four enzymes, with considerable enhancement of the activity toward <i>S. aureus</i> topoisomerase IV compared with that of <b>1</b>. Conversely, close analogues of <b>2</b> harboring a 3-CH<sub>3</sub> (<b>16</b>) or 5-CH<sub>3</sub> (<b>17</b>) coumarin resulted in the complete loss of activity against <i>S. aureus</i> topoisomerase IV. Replacement of the coumarin ring with a 5,6,7,8-tetrahydro-2<i>H</i>-chromen-2-one moiety (<b>18</b>) led to increased inhibition of <i>S. aureus</i> DNA gyrase and topoisomerase IV but reduced the activity toward <i>E. coli</i> DNA gyrase, with respect to <b>1</b>. Additional unsaturated derivatives of <b>2</b> were also evaluated. The fusion of the 2<i>H</i>-pyran-2-one ring with piperidine (<b>19</b>), tetrahydropyran (<b>20</b>), and tetrahydro-2<i>H</i>-thiopyran dioxide (<b>21</b>) resulted in relatively poor profiles toward <i>S. aureus</i> topoisomerases. This effect can likely be attributed to the high desolvation energy required for these polar moieties to pass from the solvent media to the hydrophobic binding pocket. Only the tetrahydrothiopyran ring of compound <b>22</b> provided a more balanced profile against the <i>S. aureus</i> enzymes.</div><div class="NLM_p last">Overall, the SAR described herein highlight the 6-Cl and 6-F coumarin rings as suitable RHS moieties to be applied for the optimization process. Interestingly, these moieties represent a valuable RHS alternative to the heterocycles displayed by other NBTIs, most often a pyridine fused to a nonaromatic ring (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div></div><div id="sec2_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Linker SAR</h4><div class="NLM_p">To investigate the importance of the aminopropyl-piperazine moiety of compounds <b>1</b> and <b>2</b>, various linker modifications were explored, affording derivatives <b>23</b>–<b>31</b> (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. SAR of the Linker</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0013.gif" alt="" id="gr25" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0012.gif" alt="" id="gr26" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were calculated on concentration–response curves using the global curve fit nonlinear regression tool. Compounds were assayed at eight concentrations in duplicate with twofold or half-log dilutions starting from 100 μM.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">Compounds were tested in DNA gyrase supercoiling assay kit (Norwich, U.K.).</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">Compounds were tested in topoisomerase IV decatenation assay kit (Norwich, U.K.).</p></div><div class="footnote" id="t5fn4"><sup><sup>d</sup></sup><p class="last"><i>E. coli</i> ATCC 25922.</p></div><div class="footnote" id="t5fn5"><sup><sup>e</sup></sup><p class="last"><i>S. aureus</i> ATCC 29213.</p></div></div><div></div></div><div class="NLM_p">Understanding the SAR around the linker is quite complicated from a modeling perspective. While the LHS and RHS moieties of the ligand have well-defined binding sites, the linker environment is ambiguous, being determined by the “void” between the DNA strand and protein. This open environment enables multiple transient interactions with polar residues while retaining high solvent exposure (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>).</div><figure id="fig11" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0018.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. (A) Representative snapshot from the simulation of compound <b>2</b> in complex with <i>S. aureus</i> DNA gyrase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>). The DNA strand is shown as a red “ladder” structure, and the protein is shown as a blue ribbon. Hydrogen bonds are represented as dashed lines. (B) Simulation interaction diagram summarizing the protein–ligand interactions across the entire simulation of compound <b>2</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Unfortunately, technologies such as docking, which generally use a rigid protein model, do not appropriately take into account the inherent mobility of this region. Additionally, naı̈ve molecular mechanics-based approaches, which either neglect the effect of the solvent entirely or approximate it via some type of continuum model, are unable to capture the contribution of discrete waters. Indeed, specific bridges may be formed between groups only if their geometric arrangements are favorable. Consequently, a simple MD-based approach, a technique that inherently includes full flexibility and explicit solvation, was applied to investigate linker modification. Focusing on the linker moiety, MD simulations on compound <b>2</b> showed that, in the most representative binding pose of the ligand, the 1<i>N</i>-piperazine was protonated and indirectly interacted with Asp83′ via a water molecule, and the exocyclic amino group established a direct contact with Asp83′ (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>). On this basis, different linker modifications were explored to tentatively engage the target in strong and direct interactions.</div><div class="NLM_p">N-methylation (<b>23</b>) of the exocyclic amino group in the linker of compound <b>1</b> or replacement of this group with an oxygen atom (<b>24</b>) resulted in generally poor inhibitory potencies toward the four enzymes due to the lost interaction with Asp83′. Replacement of the exocyclic amino group of <b>2</b> with an amide (<b>25</b>) was well tolerated by DNA gyrase enzymes, although loss of activity toward topoisomerases IV was observed.</div><div class="NLM_p">Modifications around the piperazine moiety were also explored. The piperazine ring of <b>1</b> was replaced with a piperazinone moiety (<b>26</b>), resulting in substantial loss of activity across all enzymes in both bacterial species. In fact, the <i>N</i>-amide of <b>26</b> is not prone to protonation, resulting in loss of the positively charged feature within the linker, which represents a key trait of NBTI target recognition.</div><div class="NLM_p">Modifications of the propyl chain afforded compounds <b>27</b>–<b>31</b>. Replacement of the linear propyl chain of <b>1</b> with <i>cis</i>-(<b>27</b>) and <i>trans</i>-(<b>28</b>) 1,3-cyclobutyl rings showed that, while the overall inhibitory profile was retained or even improved with respect to <b>1</b>, only the <i>trans</i> isomer exhibited activity toward <i>S. aureus</i> topoisomerase IV.</div><div class="NLM_p">Notably, introduction of a 2-OH substituent in the propyl chain of <b>2</b> resulted in the enantiomers <b>(<i>S</i>)-29</b> and <b>(<i>R</i>)-29</b>, with a well-balanced multitarget profile in the single-digit micromolar range against the four enzymes. The symmetric conformation of the binding site, together with the high solvent exposure of the site, might explain the observed insensitivity of the system to the chirality of the ligand.</div><div class="NLM_p">The analogue of <b>1</b> bearing a shorter aminoethyl chain (<b>30</b>) presented decreased activity, suggesting that the reduced compound size did not allow spanning of the required distance from the DNA strand down to the hydrophobic depression in the protein. Replacement of the aminopropyl chain in <b>2</b> with a 2-hydroxybutyl chain (<b>(±)-31</b>) provided improved potency against <i>S. aureus</i> topoisomerase IV and retention of the activity toward the other enzymes.</div><div class="NLM_p">In aggregate, our findings support what is already seen for other NBTIs. Indeed, the majority of reported NBTIs contain LHS and RHS heterocycles typically linked by an eight-atom bridge, which, although generally quite flexible, can incorporate one or more aliphatic rings (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><div class="NLM_p last">Importantly, the 1-(piperazin-1-yl)propan-2-ol and 1-(piperazin-1-yl)propan-3-ol linkers explored herein differentiate the developed compounds against the NBTI scenario.</div></div><div id="sec2_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Mix and Match SAR</h4><div class="NLM_p">In an effort to combine the best modifications discussed above for the LHS, RHS, and linker and to further investigate the chemical space, compounds <b>32</b>–<b>36</b> were synthesized (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>).</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. SAR Modification around the LHS, RHS, and Linker</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0011.gif" alt="" id="gr27" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were calculated on concentration–response curves using the global curve fit nonlinear regression tool. Compounds were assayed at eight concentrations in duplicate with twofold or half-log dilutions.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">Compounds were tested in DNA gyrase supercoiling assay kit (Norwich, U.K.).</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">Compounds were tested in topoisomerase IV decatenation assay kit (Norwich, U.K.).</p></div><div class="footnote" id="t6fn4"><sup><sup>d</sup></sup><p class="last"><i>E. coli</i> ATCC 25922.</p></div><div class="footnote" id="t6fn5"><sup><sup>e</sup></sup><p class="last"><i>S. aureus</i> ATCC 29213.</p></div></div><div></div></div><div class="NLM_p">All compounds contained the 1-amino-3-(piperazin-1-yl)propan-2-ol linker combined with different LHS and RHS rings. Compound <b>(±)-32</b>, despite the non-optimal 1,6-napthyridine LHS moiety, showed a very promising and balanced inhibition profile, with an IC<sub>50</sub> of 1.1 μM toward <i>S. aureus</i> DNA gyrase and IC<sub>50</sub> values in the range of 0.15–0.3 μM against the remaining enzymes. Compound <b>(±)-33</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), bearing a disubstituted 3-fluoro-7-methoxyisoquinoline in the LHS region, was obtained by merging the LHS moieties of <b>7</b> and <b>8</b>. Notably, compound <b>(±)-33</b> showed a very high potency, with IC<sub>50</sub> values of ≤0.1 μM for all tested enzymes.</div><div class="NLM_p">Modeling studies suggested that introduction of a H-bond acceptor at the 3-position on the coumarin ring might have been beneficial for binding affinity. Therefore, replacement of the RHS of <b>(<i>R</i>)-29</b> with a 2<i>H</i>-1,3-benzoxazin-2-one moiety was envisaged, affording compound <b>(<i>R</i>)-34</b> with a slightly improved profile. The introduction of a 6-fluoro substituent on the 2<i>H</i>-1,3-benzoxazin-2-one ring was also explored. In particular, combination of the 6-fluoro-2<i>H</i>-1,3-benzoxazin-2-one RHS with the 7-fluoroisoquinoline and 3,7-di-fluoroisoquinoline LHS moieties provided compounds <b>(±)-35</b> and <b>(±)-36</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), respectively. Remarkably, compounds <b>(±)-35</b> and <b>(±)-36</b> turned out to be extremely potent and balanced inhibitors, with excellent IC<sub>50</sub> values (20–50 nM) against all four enzymes, resulting in greater effectiveness than that of the reference drug ciprofloxacin. Additionally, considering the literature data for the most advanced NBTI compound GSK2140944, derivatives <b>(±)-35</b> and <b>(±)-36</b> were equipotent against <i>S. aureus</i> DNA gyrase and approximately onefold more potent against <i>S. aureus</i> topoisomerase IV.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The potency of compounds <b>(±)-35</b> and <b>(±)-36</b> could be qualitatively rationalized by modeling using a dynamics protocol similar to that employed for compound <b>2</b>. MD showed rapid evolution of the initial docked pose of <b>(±)-35</b> into the <i>S. aureus</i> DNA gyrase crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>)<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> toward the binding mode shown in <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>.</div><figure id="fig12" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0017.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. (A) Representative snapshot from the simulation of compound <b>35</b> in complex with <i>S. aureus</i> DNA gyrase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>). The DNA strand is shown as a red ladder structure, and the protein is shown as a blue ribbon. Hydrogen bonds are represented as dashed lines. (B) Simulation interaction diagram summarizing the protein–ligand interactions across the entire simulation of compound <b>35</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Notably, <b>(±)-35</b> established a far more extensive interaction network with the protein target than the parent compound <b>2</b>. The linker of <b>(±)-35</b> formed a bidentate interaction with Asp83′, while the nitrogen of the 2<i>H</i>-1,3-benzoxazin-2-one was water-bridged to Asp83.</div><div class="NLM_p last">Remarkably, compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> were the most potent compounds identified in this investigation and effectively targeted both DNA gyrase and topoisomerase IV in the tested bacterial strains. Conversely, fluoroquinolones preferentially target DNA gyrase in Gram-negative pathogens and topoisomerase IV in Gram-positive pathogens.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Taken together, these results indicate that compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> have a distinct mechanism of action and more balanced activity than fluoroquinolones.</div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Antibacterial Activity</h3><div class="NLM_p">Parent compound <b>1</b> and compounds with single-digit micromolar potency toward the four enzymes were tested for in vitro antibacterial activity against a representative panel of Gram-negative and Gram-positive strains, including <i>E. coli</i> ATCC 25922, <i>Pseudomonas aeruginosa</i> ATCC 27853, <i>S. aureus</i> ATCC 29213, and <i>S. aureus</i> BAA-1720 (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>).</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. MICs of Selected Compounds and Reference Compound Ciprofloxacin against Representative Gram-Positive and Gram-Negative Strains with and without Human Plasma, Calculated PPB Levels</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">MIC (μg/mL)<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><i>E. coli</i> ATCC 25922<a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>P. aeruginosa</i> ATCC 27853<a class="ref internalNav" href="#t7fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center"><i>S. aureus</i> ATCC 29213<a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>S. aureus</i> BAA-1720<a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">PPB<a class="ref internalNav" href="#t7fn4" aria-label="d">d</a> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ciprofloxacin</td><td class="colsep0 rowsep0" align="left">0.004/0.004</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.25/0.25</td><td class="colsep0 rowsep0" align="left">>4/>4</td><td class="colsep0 rowsep0" align="char" char=".">29.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">>32</td><td class="colsep0 rowsep0" align="left">>32</td><td class="colsep0 rowsep0" align="left">>32</td><td class="colsep0 rowsep0" align="char" char=".">97.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">95.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">>8</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="char" char=".">93.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">95.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">96.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="char" char=".">95.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">96.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(<i>R</i>)-29</b></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">90.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(<i>S</i>)-29</b></td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="char" char=".">90.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(±)-31</b></td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">>8</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="char" char=".">93.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(±)-32</b></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="char" char=".">90.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(±)-33</b></td><td class="colsep0 rowsep0" align="left">0.25/>8</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">0.03/8</td><td class="colsep0 rowsep0" align="left">0.03/4</td><td class="colsep0 rowsep0" align="char" char=".">94.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(<i>R</i>)-34</b></td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">>8</td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">97.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(±)-35</b></td><td class="colsep0 rowsep0" align="left">0.25/>8</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">0.03/>8</td><td class="colsep0 rowsep0" align="left">0.25/>8</td><td class="colsep0 rowsep0" align="char" char=".">95.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(±)-36</b></td><td class="colsep0 rowsep0" align="left">0.12/>8</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">0.03/4</td><td class="colsep0 rowsep0" align="left">0.03/4</td><td class="colsep0 rowsep0" align="char" char=".">96.2</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">Minimum inhibitory concentrations were determined according to guidelines of the CLSI.</p></div><div class="footnote" id="t7fn2"><sup><sup>b</sup></sup><p class="last">MIC in Mueller-Hinton broth/MIC in Mueller-Hinton broth with 50% human serum.</p></div><div class="footnote" id="t7fn3"><sup><sup>c</sup></sup><p class="last">MIC in Mueller-Hinton agar.</p></div><div class="footnote" id="t7fn4"><sup><sup>d</sup></sup><p class="last">PPB values were calculated using ACD/Percepta 2017.2.</p></div></div></div><div class="NLM_p">Overall, meaningful correlation between enzymatic inhibition and cellular potency was observed. Interestingly, the dual targeting of DNA gyrase as well as topoisomerase IV was reflected in good MICs. Indeed, while compound <b>1</b> with poor activity against <i>S. aureus</i> topoisomerase IV became inactive in <i>S. aureus</i> whole-cell assay, compounds with a more balanced dual target profile (i.e., <b>3</b>, <b>4</b>, <b>7</b>, <b>8</b>, <b>15</b>, <b>(<i>R</i>)-29</b>, <b>(<i>S</i>)-29</b>, and <b>31</b>–<b>36</b>) showed more favorable MIC values. Importantly, the improved antimicrobial activity against <i>S. aureus</i> seems to be correlated to the balanced dual-targeting profile of our molecules rather than to the enhanced <i>S. aureus</i> topoisomerase IV inhibitory activity. Indeed, compounds <b>11</b> and <b>20</b> with single-digit micromolar potency against <i>S. aureus</i> topoisomerase IV but poorer activity toward <i>S. aureus</i> DNA gyrase became inactive in <i>S. aureus</i> whole-cell assay (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Supporting Information</a>), confirming the potential of the multitarget approach. Having said that, the generation of spontaneous resistant mutants against tested compounds would ultimately confirm the putative dual inhibition.</div><div class="NLM_p">In the whole dataset, compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> have been identified as the most promising in terms of antibacterial effect. Notably, the aforementioned compounds were more potent against <i>S. aureus</i> ATCC 29213 (MIC = 0.03 μg/mL) and the fluoroquinolone-resistant <i>S. aureus</i> BAA-1720 strain (MIC values 0.03–0.25 μg/mL) than the reference drug ciprofloxacin (<i>S. aureus</i> ATCC 29213 (MIC = 0.25 μg/mL) and <i>S. aureus</i> BAA-1720 (MIC > 4 μg/mL)). Importantly, the antibacterial activity against <i>E. coli</i> was also retained.</div><div class="NLM_p">The antimicrobial profiles of compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> were further evaluated against other ESKAPE (<i>Enterococcus faecium</i>, <i>S. aureus</i>, <i>Klebsiella pneumoniae</i>, <i>Acinetobacter baumannii</i>, <i>P. aeruginosa</i>, and <i>Enterobacter</i> spp.) pathogens,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> comprising Gram-positive enterococci as well as the Gram-negative species <i>Haemophilus influenzae</i>, <i>N. gonorrhoeae</i>, and <i>K. pneumoniae</i>.</div><div class="NLM_p">Compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> showed potency toward vancomycin-sensitive (Van-S) and vancomycin-resistant <i>Enterococcus faecalis</i> and <i>E. faecium</i>, which are important etiological agents in nosocomial infections (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>, ciprofloxacin and vancomycin included as reference compounds).</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Antibacterial Activity of Compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> and Reference Compounds Ciprofloxacin and Vancomycin against Selected Gram-Positive Pathogens</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="12" align="center">MIC (μg/mL)<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><i>E. faecium</i> APV 00420 (Van-B)</th><th class="colsep0 rowsep0" align="center"><i>E. faecalis</i> ATCC 29212</th><th class="colsep0 rowsep0" align="center"><i>E. faecalis</i> APV 00404 (Van-S)</th><th class="colsep0 rowsep0" align="center"><i>E. faecalis</i> APV 00406 (Van-S)</th><th class="colsep0 rowsep0" align="center"><i>E. faecalis</i> APV 00408 (Van-A)</th><th class="colsep0 rowsep0" align="center"><i>E. faecalis</i> APV 00409 (Van-A)</th><th class="colsep0 rowsep0" align="center"><i>E. faecium</i> APV 00422 (Van-A)</th><th class="colsep0 rowsep0" align="center"><i>E. faecium</i> APV 00424 (Van-A)</th><th class="colsep0 rowsep0" align="center"><i>E. faecium</i> APV 00553 (Van-B)</th><th class="colsep0 rowsep0" align="center"><i>E. faecalis</i> APV 00623 (Van-S)</th><th class="colsep0 rowsep0" align="center"><i>E. faecium</i> APV 00626</th><th class="colsep0 rowsep0" align="center"><i>E. faecium</i> APV 00627</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ciprofloxacin</td><td class="colsep0 rowsep0" align="left">>32</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">64</td><td class="colsep0 rowsep0" align="left">64</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">vancomycin</td><td class="colsep0 rowsep0" align="left">>8</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(±)-33</b></td><td class="colsep0 rowsep0" align="left">0.12</td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">n.d.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(±)-35</b></td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="left">0.12</td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="left">0.12</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(±)-36</b></td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="left">0.12</td><td class="colsep0 rowsep0" align="left">0.12</td><td class="colsep0 rowsep0" align="left">0.06</td><td class="colsep0 rowsep0" align="left">0.06</td><td class="colsep0 rowsep0" align="left">0.06</td><td class="colsep0 rowsep0" align="left">0.12</td><td class="colsep0 rowsep0" align="left">0.06</td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="left">0.12</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">0.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">Minimum inhibitory concentrations were determined according to guidelines of the CLSI. Van-A: bacterial strain resistant to vancomycin and teicoplanin. Van-B: bacterial strain resistant to vancomycin but sensitive to teicoplanin. Van-S: vancomycin-sensitive strain. n.d.: not determined.</p></div></div></div><div class="NLM_p">Moreover, the three compounds were active against fastidious Gram-negative <i>H. influenzae</i> (H128 and ATCC 49766) and <i>N. gonorrhoeae</i> (ATCC 19424) and moderately active against the nonfastidious <i>K. pneumoniae</i> (ATCC 700603) (<a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">Table <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">9</a></a>, ciprofloxacin and vancomycin included as reference compounds).</div><div class="NLM_table-wrap" id="tbl9"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 9. Antibacterial Activity of Compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> and Reference Compounds Ciprofloxacin and Vancomycin against Selected Gram-Negative Pathogens</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" colspan="4" align="center">MIC (μg/mL)<a class="ref internalNav" href="#t9fn1" aria-label="a">a</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><i>H. influenzae</i> H128 (quinolone-S)</th><th class="colsep0 rowsep0" align="center"><i>H. influenzae</i> ATCC 49766 (quinolone-S)</th><th class="colsep0 rowsep0" align="center" char="."><i>N. gonorrhoeae</i> ATCC 19424</th><th class="colsep0 rowsep0" align="center"><i>K. pneumoniae</i> ATCC 700603 (ESBL; quinolone-S)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ciprofloxacin</td><td class="colsep0 rowsep0" align="left">0.008</td><td class="colsep0 rowsep0" align="left">0.015</td><td class="colsep0 rowsep0" align="char" char=".">≤0.06</td><td class="colsep0 rowsep0" align="left">0.125</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">vancomycin</td><td class="colsep0 rowsep0" align="left">>8</td><td class="colsep0 rowsep0" align="left">>8</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">>8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(±)-33</b></td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="char" char=".">≤0.06</td><td class="colsep0 rowsep0" align="left">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(±)-35</b></td><td class="colsep0 rowsep0" align="left">0.12</td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="char" char=".">≤0.06</td><td class="colsep0 rowsep0" align="left">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(±)-36</b></td><td class="colsep0 rowsep0" align="left">0.06</td><td class="colsep0 rowsep0" align="left">0.12</td><td class="colsep0 rowsep0" align="char" char=".">≤0.06</td><td class="colsep0 rowsep0" align="left">1</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t9fn1"><div class="footnote" id="t9fn1"><sup><sup>a</sup></sup><p class="last">Minimum inhibitory concentrations were determined according to guidelines of the CLSI. Quinolone-S: quinolone-sensitive. ESBL: extended spectrum β-lactamases.</p></div></div></div><div class="NLM_p">Remarkably, the in vitro activities of compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> were approximately comparable or even more potent against tested <i>E. coli</i>, <i>S. aureus</i>, <i>H. influenzae</i>, and <i>N. gonorrhoeae</i> strains than reported for GSK2140944.<a onclick="showRef(event, 'ref14 ref38'); return false;" href="javascript:void(0);" class="ref ref14 ref38">(14,38)</a></div><div class="NLM_p">Overall, compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> showed a promising broad antibacterial spectrum against both wild-type and fluoroquinolone-resistant strains.</div><div class="NLM_p last">The effect of human serum on the in vitro activity of compounds <b>(±)-33</b>, <b>(±)-35</b>, <b>(±)-36</b>, and ciprofloxacin against <i>E. coli</i> ATCC 25922, <i>S. aureus</i> ATCC 29213, and <i>S. aureus</i> BAA-1720 was also investigated. The antimicrobial potency of compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> was affected by the presence of human serum (MIC elevations of 32-fold to >267-fold), whereas the MICs of ciprofloxacin remained unchanged (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). These results correlated with the predicted high plasma binding levels (94–96%) for compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). Indeed, for antibiotics, elevated plasma protein binding (PPB) is often, although not always, accompanied with a certain loss in microbiological activity. Having said that, it has been widely recognized that PPB may not be associated with diminished efficacy in vivo for the exact reasons stated below. Indeed, serum shift in vitro does not include metabolism and efflux that, in a living system, remove unbound drug and affect the average unbound plasma concentration. Additionally, several lead optimization strategies focused on reducing PPB levels to improve pharmacological activity were unsuccessful. The misconception was that increasing the fraction unbound would increase the average unbound plasma concentration responsible for the pharmacodynamic effects. However, in vivo unbound drug concentration is usually unchanged by PPB. Indeed, lower binding leads to higher in vivo clearance because more free drug molecules are available for metabolism and elimination, resulting in a zero net increase in average unbound plasma concentration. Therefore, serum shift and elevated in vitro PPB should not discourage drug candidate selection.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Cytotoxicity</h3><div class="NLM_p">Compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> were tested on five human cell lines to evaluate their cytotoxic effect (<a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">Table <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a></a>).</div><div class="NLM_table-wrap" id="tbl10"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 10. MIC Values of Compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> with Respect to <i>E. coli</i> ATCC 25922 and <i>S. aureus</i> ATCC 29213 and Their Cytotoxic Profiles on Human Liver Hepatocellular Carcinoma (HepG2), Human Embryonic Kidney 293 (HEK293), Human Colorectal Adenocarcinoma (HT-29), Human Corneal Epithelial (HCE-2), and Human Keratinocyte (HaCaT) Cell Lines</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" colspan="2" align="center" char=".">MIC (μM)<a class="ref internalNav" href="#t10fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" colspan="5" align="center">IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t10fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="."><i>E. coli</i><a class="ref internalNav" href="#t10fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char="."><i>S. aureus</i><a class="ref internalNav" href="#t10fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">HepG2<a class="ref internalNav" href="#t10fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center">HEK293<a class="ref internalNav" href="#t10fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center">HT-29<a class="ref internalNav" href="#t10fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center">HCE-2<a class="ref internalNav" href="#t10fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center">HaCaT<a class="ref internalNav" href="#t10fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center">SI<a class="ref internalNav" href="#t10fn7" aria-label="g">g</a></th><th class="colsep0 rowsep0" align="center">SI<a class="ref internalNav" href="#t10fn8" aria-label="h">h</a></th><th class="colsep0 rowsep0" align="center">S<i>I</i><a class="ref internalNav" href="#t10fn9" aria-label="i">i</a></th><th class="colsep0 rowsep0" align="center">SI<a class="ref internalNav" href="#t10fn10" aria-label="j">j</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(±)-33</b></td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char="."><0.12</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">66</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>150</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">>208</td><td class="colsep0 rowsep0" align="left">66</td><td class="colsep0 rowsep0" align="left">>5500</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(±)-35</b></td><td class="colsep0 rowsep0" align="char" char=".">2.14</td><td class="colsep0 rowsep0" align="char" char=".">0.21</td><td class="colsep0 rowsep0" align="left">55</td><td class="colsep0 rowsep0" align="left">80</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">262</td><td class="colsep0 rowsep0" align="left">37</td><td class="colsep0 rowsep0" align="left">381</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(±)-36</b></td><td class="colsep0 rowsep0" align="char" char=".">1.02</td><td class="colsep0 rowsep0" align="char" char="."><0.12</td><td class="colsep0 rowsep0" align="left">31</td><td class="colsep0 rowsep0" align="left">52</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" align="left">>258</td><td class="colsep0 rowsep0" align="left">51</td><td class="colsep0 rowsep0" align="left">>433</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t10fn1"><div class="footnote" id="t10fn1"><sup><sup>a</sup></sup><p class="last">Minimum inhibitory concentrations were determined according to guidelines of the CLSI.</p></div><div class="footnote" id="t10fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values were calculated on concentration–response curves using the global curve fit nonlinear regression tool.</p></div><div class="footnote" id="t10fn3"><sup><sup>c</sup></sup><p class="last"><i>E. coli</i> ATCC 25922.</p></div><div class="footnote" id="t10fn4"><sup><sup>d</sup></sup><p class="last"><i>S. aureus</i> ATCC 29213.</p></div><div class="footnote" id="t10fn5"><sup><sup>e</sup></sup><p class="last">Concentration–response curves were performed for each compound by testing eight concentrations in triplicate with threefold dilutions starting from 100 to 0.030 μM after incubation for 48 h at 37 °C.</p></div><div class="footnote" id="t10fn6"><sup><sup>f</sup></sup><p class="last">Concentration–response curves were performed for each compound by testing four/six concentrations in triplicate starting from 150 or 100 μM after incubation for 24 h at 37 °C.</p></div><div class="footnote" id="t10fn7"><sup><sup>g</sup></sup><p class="last">SI = IC<sub>50</sub> HepG2/MIC <i>E. coli</i><i>.</i></p></div><div class="footnote" id="t10fn8"><sup><sup>h</sup></sup><p class="last">SI = IC<sub>50</sub> HepG2/MIC <i>S. aureus</i>.</p></div><div class="footnote" id="t10fn9"><sup><sup>i</sup></sup><p class="last">SI = IC<sub>50</sub> HEK293/MIC <i>E. coli</i><i>.</i></p></div><div class="footnote" id="t10fn10"><sup><sup>j</sup></sup><p class="last">SI = IC<sub>50</sub> HEK293/MIC <i>S. aureus</i><i>.</i></p></div></div></div><div class="NLM_p last">The tested compounds showed IC<sub>50</sub> values ranging from 25 to 55 μM in the human liver hepatocellular carcinoma (HepG2) cell line, whereas the IC<sub>50</sub> values were in the 52–80 μM range in human embryonic kidney 293 (HEK293) cells after incubation for 48 h. The selectivity index (SI) was calculated by comparing compound IC<sub>50</sub> values in the HepG2 and HEK293 cell lines against the MIC of the same compound in <i>E. coli</i> ATCC 25922 and <i>S. aureus</i> ATCC 29213. Compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> exhibited moderate SI values for <i>E. coli</i> ATCC 25922 (25- to 66-fold), whereas a notable safety window (SI > 200) was observed for <i>S. aureus</i> ATCC 29213. Nonetheless, it must be specified that SI values were calculated as the ratio of cytotoxic data measured using 10% of fetal bovine serum over MICs without serum. Therefore, what was presented might reflect the best-case scenario. Indeed, considering the previously commented shift in MIC values with 50% of human serum, the calculated safety window will be overestimated. In terms of early toxicity signals, compound <b>(±)-33</b> was also assessed in human colorectal adenocarcinoma (HT-29) and human corneal epithelial (HCE-2) cell lines, while compound <b>(±)-36</b> was tested in human keratinocytes (HaCaT). Importantly, both compounds exhibited no sign of cytotoxicity against the selected lines at up to 100 μM after incubation for 24 h.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> hERG Inhibition</h3><div class="NLM_p last">Despite the excellent antibacterial properties, NBTI-induced hERG toxicity still remains a main obstacle for their advancement into clinical trials.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> In this respect, compounds <b>(±)-35</b> and <b>(±)-36</b> were screened in automated electrophysiological patch clamp assay to assess hERG channel effects (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Supporting Information</a> for the experimental details). Compound <b>(±)-35</b> showed moderate hERG activity (67%) expressed as percent inhibition at 10 μM concentration, whereas compound <b>(±)-36</b> displayed reduced hERG liability (24% inhibition at 10 μM). Based on the data, the hERG margin of safety will need to be improved for an eventual drug candidate. Indeed, the IC<sub>50</sub> value for hERG binding with the reference NBTI compound GSK2140944 was 1.4 mM,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> which is considerably higher than values registered by other discontinued NBTIs.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25622" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25622" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have shown herein how an initial set of SBVS and pharmacophore- and shape-based models can be used to screen a library of 30,000 compounds, including Angelini proprietary compounds, and ultimately identify a couple of distinct chemical series suitable for optimization. The validity of our model was presented by resolution of the cocrystal structure of one of the initial hits (<b>ID-130</b>) bound to <i>S. aureus</i> DNA gyrase. Importantly, this X-ray investigation revealed a mechanism of action for compound <b>ID-130</b> that was distinct from that of fluoroquinolones and in agreement with the NBTI binding mode. This finding represented the basis for embracing a hit-to-lead campaign toward the development of novel antibacterial agents. Considering the promising antimicrobial profile of hit <b>ID-130</b>, we exploited the physical relevance of our computational models to support the optimization strategy. Our efforts led to the discovery of new derivatives <b>1</b>–<b>36</b> containing typical NBTI molecular features. The combination of novel and unique LHS and RHS moieties resulted in an original NBTI series. Notably, compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> showed balanced multitarget activity against <i>E. coli</i> and <i>S. aureus</i> DNA gyrase and topoisomerase IV enzymes. Furthermore, compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> exhibited excellent efficacy against a representative panel of Gram-negative and Gram-positive pathogens, as well as clinically relevant resistant strains, that was competitive with that of established antibiotics (i.e., ciprofloxacin and vancomycin) and known NBTIs previously reported in the literature.<a onclick="showRef(event, 'ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26">(11−26)</a> Despite the good antibacterial profiles, the MIC values of compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> were affected by the presence of human serum, likely due to high PPB levels. Nevertheless, there is evidence in the literature that serum shift and elevated PPB might not correlate with in vivo average unbound plasma concentration and should not discourage drug candidate selection.</div><div class="NLM_p">Despite the fact that preliminary cytotoxicity investigation revealed the favorable safety profiles of these compounds, hERG liability was identified as a potential criticism as typically observed for the NBTI class. The fine-tuning of NBTI physicochemical properties could offer a promising solution toward diminishing hERG-related issues while retaining antibacterial potency.</div><div class="NLM_p last">Overall, the piperazine derivatives reported herein represent a new chemotype in the NBTI landscape that has further potential for optimization and might serve as a basis for the future development of suitable candidates to be tested in an in vivo model of bacterial infection and pursued for the treatment of serious infections.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i35">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21811" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21811" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Some of the images included in the present work were obtained using the software PyMOL.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Chemistry: General Information and Synthetic Procedures</h3><div class="NLM_p">Reagents and starting materials were purchased from commercial sources and used as received.</div><div class="NLM_p">Microwave-assisted reactions were carried out in a Biotage Initiator+ microwave synthesizer. Reaction progress was monitored by thin-layer chromatography (TLC, Merck silica gel 60 F254 plates, 0.04–0.063 mm) or UPLC-QTOF chromatography (Waters SYNAPT MS-ACQUITY UPLC system).</div><div class="NLM_p">Final compounds were purified by semipreparative HPLC or standard flash chromatography on silica gel, when appropriate. Semipreparative HPLC purification was run using an X-Bridge Prep C18 5 μm column (Waters) through a Waters 2767 Sample manager, Waters 2478 dual λ absorbance detector, and Waters Micromass ZQ single quadrupole mass spectrometer with an electrospray ionization (ESI) source. Samples were dissolved in DMSO/CH<sub>3</sub>CN (1:1; 50 mg/mL). Purification was accomplished using a proper gradient of phase A (CH<sub>3</sub>CN + 0.1% formic acid) and phase B (H<sub>2</sub>O + 0.1% formic acid) and a 40 mL/min flow rate at 25 °C. Fraction collection was managed by MassLynx software (v. 4.1).</div><div class="NLM_p">Compound purity and molecular mass were determined by high-pressure liquid chromatography (HPLC) and high-resolution mass spectrometry (UPLC-QTOF). HPLC analysis was carried out through a pump/autosampler (Waters Alliance 2695), UV photo diode array detector (Waters 2996), and Waters system data management (Empower 2) using a Suplex pKb-100 (250 × 4.6 mm, 5 μm) column. For UPLC-QTOF, an Acquity BEH C18 (2.1 × 50 mm, 1.7 μm) column was used, and a positive ion mode in the “VOptics” configuration was employed. Leucine-enkephalin (200 pg/μL) was used as the lock mass to provide authenticated exact mass measurement in MS and MS/MS modes within 5 ppm RMS mass accuracy. NMR values were obtained using a Bruker Avance system (300, 400, and 500 MHz). All resonance bands were referenced to tetramethylsilane (internal standard). For <sup>1</sup>H-NMR spectroscopy: s = singlet; d = doublet; t = triplet; q = quartet; quin = quintet; dd = double doublet; dt = double triplet; ddd = double double doublet; dtd = double triple doublet; m = multiplet; br = broad; <i>J</i> = coupling constant (Hz); and δ = chemical shift (ppm).</div><div class="NLM_p"><sup>1</sup>H-NMR and <sup>13</sup>C-NMR spectra of the final compounds were acquired on the corresponding hydrochloride salts, unless otherwise specified. The salts were prepared by dissolving the product in DCM and the minimum required amount of MeOH and by adding HCl (1 M solution in Et<sub>2</sub>O), causing precipitation. The solvents were evaporated under reduced pressure, and the residue was triturated with Et<sub>2</sub>O. The resulting solid was dried to give the corresponding hydrochloride salt. All compounds tested possessed a purity of ≥95%.</div><div class="NLM_p last">General procedures A–G and experimental details of intermediates with general formulas <b>I</b>–<b>XX</b> are reported in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> General Procedure H: Synthesis of Compounds <b>1</b>–<b>4</b>, <b>6</b>, <b>11</b>, <b>12</b>, <b>14</b>–<b>23</b>, <b>26</b>–<b>28</b>, and <b>30</b></h3><div class="NLM_p last">A solution of the proper amines <b>Va</b>–<b>Vk</b> or <b>XIVa</b> (1.1 equiv), TEA (1.5 equiv), and pyran-2-ones <b>VIa</b>–<b>VIj</b> (1.0 equiv) in CH<sub>3</sub>CN was heated at 70 °C for 1 h. After cooling to room temperature, the resulting precipitate was filtered off and washed with CH<sub>3</sub>CN (2 × 1 mL). The resulting solid was further purified by semipreparative HPLC on silica gel to give the title compounds.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> General Procedure I: Synthesis of Compounds <b>5</b>, <b>7</b>–<b>10</b>, <b>13</b>, and <b>(±)-32</b></h3><div class="NLM_p last">A mixture of the proper 1-chloroisoquinoline or 5-chloro-1,6-napthyridine (1.0 equiv), K<sub>2</sub>CO<sub>3</sub> (1.5–2.0 equiv), and 4-{[3-(piperazin-1-yl)propyl]amino}-2<i>H</i>-1-benzopyran-2-one <b>VIIIa</b> or 6-fluoro-4-{[2-hydroxy-3-(piperazin-1-yl)propyl]amino}-2<i>H</i>-1-benzopyran-2-one <b>(±)-XXa</b> (1.1–10 equiv) was submitted to three cycles of vacuum/N<sub>2</sub> in a flask. DMSO was added, and the suspension was stirred at 110 °C for 5 h. The reaction was then allowed to cool to room temperature, diluted with water, and extracted with EtOAc. The organic phase was separated, washed with brine, dried over sodium sulfate, and evaporated under reduced pressure. The crude material was purified by flash chromatography on silica gel to afford the title compounds.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> General Procedure J: Synthesis of Compounds <b>(<i>R</i>)-29</b>, <b>(<i>S</i>)-29</b>, and <b>(<i>R</i>)-34</b></h3><div class="NLM_p last">A solution of the proper amines X<b>Va</b> and <b>XVb</b> (1.1 equiv), TEA (1.5 equiv), and intermediate <b>VIa</b> (1.0 equiv) or X<b>VIa</b> (2.0 equiv) in CH<sub>3</sub>CN (2 mL) was stirred at 70 °C for 1 h, during which time a precipitate was formed. The reaction mixture was cooled, filtered, and washed with CH<sub>3</sub>CN. The crude material was purified by flash chromatography on silica gel to afford the title compounds.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> General Procedure K: Synthesis of Compounds <b>(±)-35</b> and <b>(±)<i>-</i>36</b></h3><div class="NLM_p last">To a suspension of chloride <b>XVIb</b> (3.0 equiv) in CH<sub>3</sub>CN, DIPEA (3.0 equiv) and proper amines <b>(±)-XVc</b> and <b>(±)-XVd</b> (1.0 equiv) were added at 0 °C. The mixture was stirred at room temperature for 15 h, and then the resulted solid was filtered, washed with water, and dried. The crude material was purified by flash chromatography on silica gel, affording the final compounds <b>(±)-35</b> and <b>(±)-36</b> as racemates.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 4-({3-[4-(Isoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2<i>H</i>-chromen-2-one (<b>1</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 3-[4-(isoquinolin-1-yl)piperazin-1-yl]propan-1-amine <b>Va</b> (141 mg, 0.52 mmol), TEA (107 μL, 0.78 mmol), 2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIa</b> (153 mg, 0.52 mmol), and CH<sub>3</sub>CN (5 mL). Yield = 45%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 415.2; HRMS (ESI): [M + H]<sup>+</sup>, 415.2128; found, 415.2136; HPLC purity: 100%, <i>R</i><sub>t</sub> = 14.12 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.07 (br s, 1H), 8.19 (d, <i>J =</i> 8.5 Hz, 1H), 8.15 (d, <i>J =</i> 8.3 Hz, 1H), 8.08 (d, <i>J =</i> 6.0 Hz, 1H), 8.00 (d, <i>J =</i> 8.3 Hz, 1H), 7.95 (t, <i>J =</i> 5.5 Hz, 1H), 7.85 (t, <i>J =</i> 7.5 Hz, 1H), 7.73–7.67 (m, 1H), 7.63–7.58 (m, 1H), 7.57 (d, <i>J =</i> 6.0 Hz, 1H), 7.39–7.28 (m, 2H), 5.27 (s, 1H), 4.00 (d, <i>J =</i> 13.3 Hz, 2H), 3.65 (d, <i>J =</i> 10.5 Hz, 4H), 3.50–3.34 (m, 4H), 3.30 (td, <i>J =</i> 4.8, 9.4 Hz, 2H), 2.14 ppm (quin, <i>J =</i> 7.3 Hz, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.5, 157.6, 153.1, 153.0, 138.2, 132.1, 131.9, 127.7, 127.6, 126.2, 123.2, 122.7, 120.4, 116.9, 116.4, 114.5, 81.6, 53.4, 50.6, 47.9, 21.8 ppm.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 4-({3-[4-(7-Fluoroisoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2<i>H-</i>chromen-2-one (<b>2</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 3-[4-(7-fluorosoquinolin-1-yl)piperazin-1-yl]propan-1-amine <b>Vb</b> (115 mg, 0.4 mmol), TEA (82 μL, 0.6 mmol), 2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIa</b> (118 mg, 0.4 mmol), and CH<sub>3</sub>CN (4 mL). Yield = 42%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 433.3; HRMS (ESI): [M + H]<sup>+</sup>, 433.2034; found, 433.2037; HPLC purity: 98.2%, <i>R</i><sub>t</sub> = 15.55 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.27 (br s, 1H), 8.16 (d, <i>J =</i> 5.8 Hz, 1H), 8.11–8.03 (m, 2H), 7.88–7.80 (m, 2H), 7.70 (dt, <i>J =</i> 2.5, 8.8 Hz, 1H), 7.64–7.58 (m, 1H), 7.56 (d, <i>J =</i> 5.8 Hz, 1H), 7.36–7.32 (m, 2H), 5.29 (s, 1H), 3.81 (d, <i>J =</i> 12.0 Hz, 2H), 3.65–3.62 (m, 2H), 3.45–3.36 (m, 6H), 3.33–3.27 (m, 2H), 2.15–2.08 ppm (m, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.5, 160.1 (d, <i>J =</i> 245.0 Hz), 158.6, 153.0, 152.9, 138.5, 134.9, 131.8, 130.5 (d, <i>J =</i> 8.3 Hz), 123.2, 122.6, 121.6 (d, <i>J =</i> 8.3 Hz), 120.9 (d, <i>J =</i> 24.0 Hz), 116.8, 116.4, 114.4, 109.1 (d, <i>J =</i> 22.1 Hz), 81.6, 53.4, 50.7, 47.6, 21.8 ppm.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 4-({3-[4-(7-Chloroisoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2<i>H</i>-chromen-2-one (<b>3</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 3-[4-(7-chlorosoquinolin-1-yl)piperazin-1-yl]propan-1-amine <b>Vc</b> (114 mg, 0.374 mmol), TEA (70 μL, 0.51 mmol), 2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIa</b> (100 mg, 0.34 mmol), and CH<sub>3</sub>CN (4 mL). Yield = 39%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 449.3; HRMS (ESI): [M + H]<sup>+</sup>, 449.1739; found, 449.1752; HPLC purity: 97.6%, <i>R</i><sub>t</sub> = 11.87 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.72 (br s, 1H), 8.18 (d, <i>J =</i> 5.5 Hz, 1H), 8.15–8.09 (m, 2H), 8.01 (d, <i>J =</i> 8.8 Hz, 1H), 7.89 (t, <i>J =</i> 5.9 Hz, 1H), 7.79 (dd, <i>J =</i> 1.9, 8.7 Hz, 1H), 7.63–7.57 (m, 1H), 7.55 (d, <i>J =</i> 5.5 Hz, 1H), 7.38–7.28 (m, 2H), 5.27 (s, 1H), 3.81 (d, <i>J =</i> 9.0 Hz, 2H), 3.62 (d, <i>J =</i> 10.0 Hz, 2H), 3.53–3.34 (m, 6H), 3.33–3.22 (m, 2H), 2.14 ppm (quin, <i>J =</i> 7.0 Hz, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.5, 158.1, 153.1, 153.0, 138.9, 136.4, 131.9, 131.7, 131.6, 129.8, 124.2, 123.2, 122.7, 121.5, 116.9, 116.3, 114.5, 81.7, 53.5, 50.7, 47.8, 21.8 ppm.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 1-(4-{3-[(2-Oxo-2<i>H</i>-chromen-4-yl)amino]propyl}piperazin-1-yl)isoquinoline-7-carbonitrile (<b>4</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 1-[4-(3-aminopropyl)piperazin-1-yl]isoquinoline-7-carbonitrile <b>Vd</b> (110 mg, 0.34 mmol), TEA (70 μL, 0.51 mmol), 2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIa</b> (100 mg, 0.34 mmol), and CH<sub>3</sub>CN (2 mL). Yield = 41%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 440.3; HRMS (ESI): [M + H]<sup>+</sup>, 440.2081; found, 440.2072; HPLC purity: 96.0%, <i>R</i><sub>t</sub> = 12.71 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.73 (br s1H), 8.65 (s, 1H), 8.30 (d, <i>J =</i> 5.8 Hz, 1H), 8.16–8.07 (m, 2H), 8.03 (dd, <i>J =</i> 1.5, 8.5 Hz, 1H), 7.89 (t, <i>J =</i> 5.1 Hz, 1H), 7.69–7.52 (m, 2H), 7.41–7.25 (m, 2H), 5.28 (s, 1H), 3.89 (d, <i>J =</i> 11.5 Hz, 2H), 3.62 (d, <i>J =</i> 9.3 Hz, 2H), 3.55–3.36 (m, 6H), 3.35–3.17 (m, 2H), 2.14 ppm (quin, <i>J =</i> 7.0 Hz, 2H).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 4-({3-[4-(7-Methylisoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2<i>H</i>-chromen-2-one (<b>5</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure I using 4-{[3-(piperazin-1-yl)propyl]amino}-2<i>H</i>-1-benzopyran-2-one <b>VIIIa</b> (40 mg, 0.13 mmol), 1-chloro-7-methylisoquinoline (19 mg, 0.10 mmol), K<sub>2</sub>CO<sub>3</sub> (23 mg, 0.17 mmol), and DMSO (1 mL). Yield = 11%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 429.4; HRMS (ESI): [M + H]<sup>+</sup>, 429.2285; found, 418.2289; HPLC purity: 98.5%, <i>R</i><sub>t</sub> = 16.61 min; <sup>1</sup>H-NMR of the free base (300 MHz, CDCl<sub>3</sub>): δ 8.14 (d, <i>J =</i> 5.5 Hz, 1H), 7.93 (br s, 2H), 7.84 (s, 1H), 7.71 (d, <i>J =</i> 8.5 Hz, 1H), 7.55–7.44 (m, 2H), 7.34–7.28 (m, 3H), 5.25 (s, 1H), 3.70 (br s, 4H), 3.49 (br s, 2H), 3.04 (br s, 4H), 2.92 (br s, 2H), 2.56 (s, 3H), 2.18–2.04 ppm (m, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.5, 160.0, 153.1, 153.0, 139.5, 136.0, 135.7, 131.8, 131.8, 127.0, 123.8, 123.1, 122.3, 120.9, 116.9, 115.5, 114.4, 81.2, 55.3, 52.6, 50.7, 40.4, 21.5, 20.9 ppm.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 4-({3-[4-(7-Dimethylamino-isoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2<i>H</i>-chromen-2-one (<b>6</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 1-[4-(3-aminopropyl)piperazin-1-yl]-<i>N</i>,<i>N</i>-dimethylisoquinolin-7-amine <b>Ve</b> (222 mg, 0.70 mmol), TEA (1.05 mmol, 0.146 mL), 2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIa</b> (197 mg, 0.67 mmol), and CH<sub>3</sub>CN (6.1 mL). Yield = 23%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 458.4; HRMS (ESI): [M + H]<sup>+</sup>, 458.2551; found, 458.2559; HPLC purity: 96.7%, <i>R</i><sub>t</sub> = 14.17 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.12 (br s, 1H), 8.14 (d, <i>J =</i> 7.8 Hz, 1H), 7.97–7.94 (m, 2H), 7.78 (d, <i>J =</i> 6.3 Hz, 1H), 7.68–7.49 (m, 3H), 7.38–7.27 (m, 2H), 6.96 (br s, 1H), 5.28 (s, 1H), 4.13 (d, <i>J =</i> 12.0 Hz, 2H), 3.79 (t, <i>J =</i> 9.0 Hz, 2H), 3.69 (d, <i>J =</i> 12.3 Hz, 2H), 3.47–3.35 (m, 4H), 3.33–3.28 (m, 2H), 3.11 (s, 6H), 2.23–2.08 ppm (m, 2H).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 4-({3-[4-(7-Methoxyisoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2<i>H</i>-chromen-2-one (<b>7</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure I using 4-{[3-(piperazin-1-yl)propyl]amino}-2<i>H</i>-1-benzopyran-2-one <b>VIIIa</b> (100 mg, 0.34 mmol), 1-chloro-7-methoxyisoquinoline (56 mg, 0.29 mmol), K<sub>2</sub>CO<sub>3</sub> (60 mg, 0.43 mmol), and DMSO (3 mL). Yield = 42%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 445.4; HRMS (ESI): [M + H]<sup>+</sup>, 445.2234; found, 445.2234; HPLC purity: 96.2%, <i>R</i><sub>t</sub> = 16.00 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.07 (br s, 1H), 8.16 (d, <i>J =</i> 7.8 Hz, 1H), 8.00 (d, <i>J =</i> 6.0 Hz, 1H), 7.97–7.89 (m, 2H), 7.65–7.56 (m, 1H), 7.56–7.45 (m, 2H), 7.40–7.27 (m, 3H), 5.27 (s, 1H), 4.03–3.84 (m, 5H), 3.73–3.52 (m, 4H), 3.50–3.35 (m, 4H), 3.30 (br s, 2H), 2.16 ppm (quin, <i>J =</i> 7.2 Hz, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.5, 158.2, 156.6, 153.0, 153.0, 133.2, 131.8, 129.4, 123.8, 123.2, 122.7, 121.8, 116.8, 116.5, 114.4, 104.7, 81.6, 55.5, 53.5, 50.6, 47.5, 21.8 ppm.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 4-({3-[4-(3-Fluoroisoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2<i>H</i>-chromen-2-one (<b>8</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure I using 4-{[3-(piperazin-1-yl)propyl]amino}-2<i>H</i>-1-benzopyran-2-one <b>VIIIa</b> (126 mg, 0.44 mmol), 1-chloro-3-fluoroisoquinoline (100 mg, 0.44 mmol), K<sub>2</sub>CO<sub>3</sub> (152 mg, 1.1 mmol), and DMSO (3 mL). Yield = 89%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 433.3; HRMS (ESI): [M + H]<sup>+</sup>, 433.2034; found, 433.2041; HPLC purity: 97.2%, <i>R</i><sub>t</sub> = 15.26 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.52 (br s, 1H), 8.14–8.07 (m, 2H), 7.93 (d, <i>J =</i> 8.3 Hz, 1H), 7.86 (t, <i>J =</i> 5.5 Hz, 1H), 7.74 (t, <i>J =</i> 7.5 Hz, 1H), 7.64–7.58 (m, 1H), 7.58–7.52 (m, 1H), 7.40–7.28 (m, 2H), 7.15 (s, 1H), 5.28 (s, 1H), 3.96 (d, <i>J =</i> 13.3 Hz, 2H), 3.68 (br s, 2H), 3.63 (d, <i>J =</i> 11.5 Hz, 2H), 3.52–3.24 (m, 8H), 2.17–2.04 ppm (m, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.5, 159.2, 159.0, 158.4 (d, <i>J =</i> 231.2 Hz), 153.0 (d, <i>J =</i> 8.4 Hz), 141.1 (d, <i>J =</i> 9.2 Hz), 131.8, 131.0, 127.1, 125.5 (d, <i>J =</i> 13.4 Hz), 123.2, 122.6, 118.5, 116.8, 114.4, 96.1 (d, <i>J =</i> 35.4 Hz), 81.6, 53.4, 50.6, 47.4, 21.8 ppm.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 4-({3-[4-(3-Chloroisoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2<i>H</i>-chromen-2-one (<b>9</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure I using 4-{[3-(piperazin-1-yl)propyl]amino}-2<i>H</i>-1-benzopyran-2-one <b>VIIIa</b> (100 mg, 0.34 mmol), 1,3-dichloro-isoquinoline (57 mg, 0.29 mmol), K<sub>2</sub>CO<sub>3</sub> (60 mg, 0.43 mmol), and DMSO (2 mL). Yield = 61%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 449.3; HRMS (ESI): [M + H]<sup>+</sup>, 449.1739; found, 449.1744; HPLC purity: 98.7%, <i>R</i><sub>t</sub> = 13.53 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.99 (br s, 1H), 8.11 (d, <i>J =</i> 8.3 Hz, 1H), 8.05 (d, <i>J =</i> 8.3 Hz, 1H), 7.91 (d, <i>J =</i> 7.8 Hz, 1H), 7.85–7.72 (m, 2H), 7.69–7.56 (m, 3H), 7.41–7.28 (m, 2H), 5.29 (s, 1H), 3.96 (d, <i>J =</i> 9.3 Hz, 2H), 3.64 (d, <i>J =</i> 6.5 Hz, 2H), 3.48–3.34 (m, 8H), 2.16–2.01 ppm (m, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.5, 159.5, 153.0, 152.9, 141.7, 139.5, 131.8, 131.1, 127.0, 126.7, 125.3, 123.2, 122.6, 119.0, 116.8, 114.4, 114.4, 81.6, 53.3, 50.6, 47.4, 21.8 ppm.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 1-(4-{3-[(2-Oxo-2<i>H</i>-chromen-4-yl)amino]propyl}piperazin-1-yl)isoquinoline-3-carbonitrile (<b>10</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure I using 4-{[3-(piperazin-1-yl)propyl]amino}-2<i>H</i>-1-benzopyran-2-one <b>VIIIa</b> (141 mg, 0.49 mmol), 1-chloroisoquinoline-3-carbonitrile (77 mg, 0.41 mmol), K<sub>2</sub>CO<sub>3</sub> (85 mg, 0.61 mmol), and DMSO (3 mL). Yield = 48%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 440.3; HRMS (ESI): [M + H]<sup>+</sup>, 440.2081; found, 440.2079; HPLC purity: 98.5%, <i>R</i><sub>t</sub> = 14.51 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.57 (br s, 1H), 8.24 (s, 1H), 8.21 (d, <i>J =</i> 8.3 Hz, 1H), 8.09 (dd, <i>J =</i> 8.0, 13.1 Hz, 2H), 7.94–7.88 (m, 1H), 7.88–7.80 (m, 2H), 7.65–7.57 (m, 1H), 7.38–7.28 (m, 2H), 5.28 (s, 1H), 3.98 (d, <i>J =</i> 13.3 Hz, 2H), 3.63 (d, <i>J =</i> 11.0 Hz, 2H), 3.55–3.24 (m, 8H), 2.12 ppm (quin, <i>J =</i> 7.1 Hz, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.5, 160.1, 153.0, 153.0, 136.6, 131.8, 131.6, 130.0, 128.1, 125.5, 123.8, 123.2, 122.9, 122.6, 121.8, 118.3, 116.8, 114.4, 81.6, 53.4, 50.5, 47.3, 21.8 ppm.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 4-({3-[4-(3-Methylisoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2<i>H</i>-chromen-2-one (<b>11</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 3-[4-(3-methylisoquinolin-1-yl)piperazin-1-yl]propan-1-amine <b>Vf</b> (130 mg, 0.45 mmol), TEA (0.065 mL, 0.47 mmol), 2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIa</b> (120 mg, 0.4 mmol), and CH<sub>3</sub>CN (6 mL). Yield = 17%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 429.4 HRMS (ESI): [M + H]<sup>+</sup>, 429.2285; found, 429.2295; HPLC purity: 97.8%, <i>R</i><sub>t</sub> = 16.18 min; <sup>1</sup>H-NMR of the hydrochloride salt (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.3 (br s, 2H), 8.19–8.17 (m, 1H), 8.11 (d, <i>J =</i> 8.4 Hz, 1H), 7.97 (t, <i>J =</i> 5.4 Hz, 1H), 7.86 (d, <i>J =</i> 8.1 Hz, 1H), 7.77 (t, <i>J =</i> 7.5 Hz, 1H), 7.62–7.60 (m, 2H), 7.36–7.31 (m, 3H), 5.27 (s, 1H), 3.97 (d, <i>J =</i> 13.4 Hz, 2H), 3.68–3.63 (m, 4H), 3.45–3.37 (m, 4H), 3.32–3.26 (m, 2H), 2.54 (s, 1H), 2.20–2.14 ppm (m, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.5, 157.9, 153.0, 153.0, 138.7, 131.8, 131.5, 126.8, 126.4, 125.7, 123.2, 122.7, 118.7, 116.8, 114.5, 114.4, 81.6, 53.4, 50.7, 47.8, 21.7 ppm.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 4-({3-[4-(3-Methoxyisoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2<i>H</i>-chromen-2-one (<b>12</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 3-[4-(3-methyoxyisoquinolin-1-yl)piperazin-1-yl]propan-1-amine <b>Vg</b> (165 mg, 0.54 mmol), TEA (58 mg, 0.57 mmol, 0.08 mL), 2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIa</b> (147 mg, 0.49 mmol), and CH<sub>3</sub>CN (5 mL). Yield = 13%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 445.4; HRMS (ESI): [M + H]<sup>+</sup>, 445.2234; found, 445.2240; HPLC purity: 95.7%, <i>R</i><sub>t</sub> = 15.13 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.07 (br s, 1H), 8.17 (d, <i>J =</i> 5.8 Hz, 1H), 7.97 (br s, 2H), 7.76 (d, <i>J =</i> 6.8 Hz, 1H), 7.59 (br s, 2H), 7.32 (br s, 3H), 6.77 (br s, 1H), 5.26 ( s, 1H), 3.90 (br s, 3H), 3.61–3.29 (m, 12H), 2.15 ppm (br s, 2H).</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 4-({3-[4-(1,6-Naphthyridin-5-yl)piperazin-1-yl]propyl}amino)-2<i>H</i>-chromen-2-one (<b>13</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure I using 4-{[3-(piperazin-1-yl)propyl]amino}-2<i>H</i>-1-benzopyran-2-one <b>VIIIa</b> (40 mg, 0.13 mmol), 5-chloro-1,6-naphthyridine (19 mg, 0.11 mmol), K<sub>2</sub>CO<sub>3</sub> (23 mg, 0.17 mmol), and DMSO (1 mL). Yield = 76%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 416.3; HRMS (ESI): [M + H]<sup>+</sup>, 416.2081; found, 416.2092; HPLC purity: 96.9%, <i>R</i><sub>t</sub> = 14.76 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.10 (br s, 1H), 9.06 (dd, <i>J =</i> 1.6, 4.1 Hz, 1H), 8.54 (d, <i>J =</i> 8.3 Hz, 1H), 8.35 (d, <i>J =</i> 6.0 Hz, 1H), 8.07 (d, <i>J =</i> 8.0 Hz, 1H), 7.81 (t, <i>J =</i> 5.5 Hz, 1H), 7.68–7.57 (m, 2H), 7.53 (d, <i>J =</i> 6.0 Hz, 1H), 7.40–7.28 (m, 2H), 5.28 (s, 1H), 3.92 (d, <i>J =</i> 11.5 Hz, 2H), 3.63 (d, <i>J =</i> 10.0 Hz, 2H), 3.37–3.18 (m, 8H), 2.18–2.01 ppm (m, 2H).</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 6-Chloro-4-({3-[4-(isoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2<i>H</i>-chromen-2-one (<b>14</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 3-[4-(isoquinolin-1-yl)piperazin-1-yl]propan-1-amine <b>Va</b> (50 mg, 0.18 mmol), TEA (0.027 mL, 0.194 mmol), 6-chloro-2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIb</b> (55 mg, 0.16 mmol), and CH<sub>3</sub>CN (5 mL). Yield = 33%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 449.3; HRMS (ESI): [M + H]<sup>+</sup>, 449.1739; found, 449.1750; HPLC purity: 97.1%, <i>R</i><sub>t</sub> = 15.76 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.86 (br s, 1H), 8.28 (d, <i>J =</i> 1.8 Hz, 1H), 8.17 (d, <i>J =</i> 8.3 Hz, 1H), 8.11 (d, <i>J =</i> 5.8 Hz, 1H), 7.97 (d, <i>J =</i> 8.0 Hz, 1H), 7.94–7.92 (m, 1H), 7.81 (t, <i>J =</i> 7.4 Hz, 1H), 7.769–7.60 (m, 2H), 7.54 (d, <i>J =</i> 5.8 Hz, 1H), 7.37 (d, <i>J =</i> 9.0 Hz, 1H), 5.32 (s, 1H), 3.94 (d, <i>J =</i> 13.3 Hz, 2H), 3.66–3.53 (m, 4H), 3.43–3.139 (m, 4H), 3.30 (br s, 2H), 2.25–2.03 ppm (m, 2H).</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 6-Fluoro-4-({3-[4-(isoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2<i>H</i>-chromen-2-one (<b>15</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 3-[4-(isoquinolin-1-yl)piperazin-1-yl]propan-1-amine <b>Va</b> (50 mg, 0.18 mmol), TEA (0.027 mL, 0.194 mmol), 6-fluoro-2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIc</b> (50 mg, 0.16 mmol), and CH<sub>3</sub>CN (5 mL). Yield = 20%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 433.3; HRMS (ESI): [M + H]<sup>+</sup>, 433.2034; found, 433.2039; HPLC purity: 96.9%, <i>R</i><sub>t</sub> = 14.62 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.07 (br s, 1H), 8.18 (d, <i>J =</i> 8.3 Hz, 1H), 8.14–8.05 (m, 2H), 7.99 (d, <i>J =</i> 8.3 Hz, 1H), 7.92 (t, <i>J =</i> 5.3 Hz, 1H), 7.83 (t, <i>J =</i> 7.4 Hz, 1H), 7.71–7.67 (m, 1H), 7.55 (d, <i>J =</i> 6.0 Hz, 1H), 7.49 (dt, <i>J =</i> 2.8, 8.5 Hz, 1H), 7.40–7.37 (m, 1H), 5.31 (s, 1H), 3.98 (d, <i>J =</i> 13.3 Hz, 2H), 3.65 (d, <i>J =</i> 11.5 Hz, 4H), 3.48–3.35 (m, 4H), 3.30 (br s, 2H), 2.24–2.06 ppm (m, 2H).</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> 3-Methyl-4-({3-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2<i>H</i>-chromen-2-one (<b>16</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 3-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]propan-1-amine <b>Vb</b> (100 mg, 0.35 mmol), TEA (0.48 mmol, 67 μL), 3-methyl-2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VId</b> (98.6 mg, 0.32 mmol), and CH<sub>3</sub>CN (3 mL). Yield = 38%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 447.3; HRMS (ESI): [M + H]<sup>+</sup>, 447.2191; found, 447.2193; HPLC purity: 97.8%, <i>R</i><sub>t</sub> = 14.49 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.84 (br s, 1H), 8.20–8.11 (m, 2H), 8.07 (dd, <i>J =</i> 5.5, 9.0 Hz, 1H), 7.84 (dd, <i>J =</i> 2.4, 10.2 Hz, 1H), 7.71 (dt, <i>J =</i> 2.5, 8.8 Hz, 1H), 7.61–7.48 (m, 2H), 7.38–7.24 (m, 2H), 3.81 (d, <i>J =</i> 11.8 Hz, 2H), 3.65 (t, <i>J =</i> 6.7 Hz, 2H), 3.59 (d, <i>J =</i> 11.0 Hz, 2H), 3.48–3.32 (m, 4H), 3.25–3.20 (m, 2H), 2.16 (s, 3H), 2.14–2.04 ppm (m, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.8, 160.2 (d, <i>J =</i> 246.4 Hz), 158.2, 151.2, 150.5, 137.2, 135.0, 130.8, 130.6 (d, <i>J =</i> 8.4 Hz), 123.3, 122.7, 121.6 (d, <i>J =</i> 8.4 Hz), 121.3 (d, <i>J =</i> 25.2 Hz), 116.5, 116.4, 116.2, 109.5 (d, <i>J =</i> 20.6 Hz), 94.6, 53.0, 50.5, 47.6, 42.3, 24.5, 13.0 ppm.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> 5-Methyl-4-({3-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2<i>H</i>-chromen-2-one (<b>17</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 3-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]propan-1-amine <b>Vb</b> (100 mg, 0.35 mmol), TEA (0.48 mmol, 67 μL), 5-methyl-2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIe</b> (98.6 mg, 0.32 mmol), and CH<sub>3</sub>CN (3 mL). Yield = 31%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 447.3; HRMS (ESI): [M + H]<sup>+</sup>, 447.2191; found, 447.2198; HPLC purity: 97.4%, <i>R</i><sub>t</sub> = 14.55 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.17 (br s, 1H), 8.13 (d, <i>J =</i> 5.8 Hz, 1H), 8.09 (dd, <i>J =</i> 5.5, 8.8 Hz, 1H), 7.87 (d, <i>J =</i> 9.8 Hz, 1H), 7.74 (t, <i>J =</i> 8.5 Hz, 1H), 7.59 (d, <i>J =</i> 5.8 Hz, 1H), 7.44 (t, <i>J =</i> 7.9 Hz, 1H), 7.16 (d, <i>J =</i> 8.0 Hz, 1H), 7.12 (d, <i>J =</i> 7.5 Hz, 1H), 6.67–6.65 (m, 1H), 5.27 (s, 1H), 3.87 (d, <i>J =</i> 12.8 Hz, 2H), 3.63 (d, <i>J =</i> 11.3 Hz, 2H), 3.55 (t, <i>J =</i> 12.8 Hz, 2H), 3.44–3.38 (m, 4H), 3.28 (br s, 2H), 2.84 (s, 3H), 2.27–2.06 ppm (m, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.0, 160.1 (d, <i>J =</i> 246.4 Hz), 158.6, 155.6, 154.3, 138.3 (d, <i>J =</i> 6.9 Hz), 135.0, 134.9, 130.9, 130.5 (d, <i>J =</i> 9.16 Hz), 127.5, 121.6 (d, <i>J =</i> 7.7 Hz), 120.9 (d, <i>J =</i> 23.6 Hz), 116.4, 115.4, 114.0, 109.2 (d, <i>J =</i> 22.1 Hz), 83.0, 53.3, 50.7, 47.5, 23.2, 21.5 ppm.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> 4-({3-[4-(Isoquinolin-1-yl)piperazin-1-yl]propyl}amino)-5,6,7,8-tetrahydro-2<i>H</i>-chromen-2-one (<b>18</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 3-[4-(isoquinolin-1-yl)piperazin-1-yl]propan-1-amine <b>Va</b> (95 mg, 0.35 mmol), TEA (70 μL, 0.50 mmol), 2-oxo-5,6,7,8-tetrahydro-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIf</b> (100 mg, 0.33 mmol), and CH<sub>3</sub>CN (2 mL). Yield = 52%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 419.4; HRMS (ESI): [M + H]<sup>+</sup>, 419.2442; found, 419.2441; HPLC purity: 97.3%, <i>R</i><sub>t</sub> = 13.68 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.56 (br s, 1H), 8.24 (d, <i>J =</i> 8.3 Hz, 1H), 8.08–8.02 (m, 2H), 7.93–7.90 (m, 1H), 7.76–7.73 (m, 1H), 7.61(d, <i>J =</i> 5.8 Hz, 1H), 6.84 (br s, 1H), 4.92 (s, 1H), 4.12 (d, <i>J =</i> 13.3 Hz, 2H), 3.88–3.84 (m, 2H), 3.62 (d, <i>J =</i> 12.0 Hz, 2H), 3.43–3.41 (m, 2H), 3.26–3.22 (m, 4H), 2.36 (br s, 2H), 2.24 (br s, 2H), 2.04 (br s, 2H), 1.68 ppm (br s, 4H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.7, 157.4, 156.8, 138.2, 132.3, 127.8, 127.6, 126.3, 120.3, 116.4, 105.4, 79.1, 53.2, 50.4, 47.8, 27.0, 21.7, 21.2, 21.1, 20.9 ppm.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> 4-({3-[4-(7-Fluoroisoquinolin-1-yl)piperazin-1-yl]propyl}amino)-5,6,7,8-tetrahydro-2<i>H</i>-pyrano[3,2-<i>c</i>]pyridin-2-one (<b>19</b>)</h3><div class="NLM_p last">Intermediate <i>tert</i>-butyl 4-({3-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2-oxo-7,8-dihydro-2<i>H</i>-pyrano[3,2-<i>c</i>]pyridine-6(5<i>H</i>)-carboxylate was obtained according to general procedure H using 3-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]propan-1-amine <b>Vb</b> (80 mg, 0.27 mmol), TEA (75 μL, 0.54 mmol), <i>tert</i>-butyl 2-oxo-4-(trifluoromethanesulfonyloxy)-2<i>H</i>,5<i>H</i>,6<i>H</i>,7<i>H</i>,8<i>H</i>-pyrano[3,2-<i>c</i>]pyridine-6-carboxylate <b>VIg</b> (113 mg, 0.28 mmol), and CH<sub>3</sub>CN (2 mL). Yield = 17%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 538.4. The <i>N</i>-protected intermediate was deprotected according to general procedure B (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Supporting Information</a>). Yield = 72%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 438.2; HRMS (ESI): [M + H]<sup>+</sup>, 438.2300; found, 438.2301; HPLC purity: 97.6%, <i>R</i><sub>t</sub> = 11.17 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.74 (br s, 1H), 9.61 (br s, 1H), 8.16 (d, <i>J =</i> 5.8 Hz, 1H), 8.07 (dd, <i>J =</i> 5.8, 9.0 Hz, 1H), 7.84 (dd, <i>J =</i> 2.5, 10.3 Hz, 1H), 7.71 (dt, <i>J =</i> 2.5, 8.8 Hz, 1H), 7.57 (d, <i>J =</i> 5.8 Hz, 1H), 7.16 (t, <i>J =</i> 5.3 Hz, 1H), 5.06 (s, 1H), 3.94–3.90 (m, 4H), 3.81 (d, <i>J =</i> 10.8 Hz, 2H), 3.59 (d, <i>J =</i> 9.0 Hz, 2H), 3.46–3.35 (m, 6H), 3.328–3.21 (m, 4H), 2.73 (t, <i>J =</i> 5.8 Hz, 2H), 2.07–1.99 ppm (m, 2H).</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> 4-({3-[4-(7-Fluoroisoquinolin-1-yl)piperazin-1-yl]propyl}amino)-7,8-dihydro-2<i>H</i>,5<i>H</i>-pyrano[4,3-<i>b</i>]pyran-2-one (<b>20</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 3-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]propan-1-amine <b>Vb</b> (150 mg, 0.52 mmol), TEA (145 μL, 1.04 mmol), 2-oxo-7,8-dihydro-2<i>H</i>,5<i>H</i>-pyrano[4,3-<i>b</i>]pyran-4-yl trifluoromethanesulfonate <b>VIh</b> (164 mg, 0.54 mmol), and CH<sub>3</sub>CN (4 mL). Yield = 25%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 439.3; HRMS (ESI): [M + H]<sup>+</sup>, 439.2140; found, 439.2143; HPLC purity: 99.2%, <i>R</i><sub>t</sub> = 15.21 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.07 (br s, 1H), 8.17 (d, <i>J =</i> 5.5 Hz, 1H), 8.06 (dd, <i>J =</i> 5.8, 9.0 Hz, 1H), 7.84 (dd, <i>J =</i> 2.4, 10.4 Hz, 1H), 7.70 (dt, <i>J =</i> 2.5, 8.8 Hz, 1H), 7.56 (d, <i>J =</i> 5.8 Hz, 1H), 6.75 (t, <i>J =</i> 5.4 Hz, 1H), 4.99 (s, 1H), 4.35 (s, 2H), 3.987–3.80 (m, 5H), 3.60 (br s, 2H) 3.43–3.30 (m, 5H), 3.25–3.20 (m, 4H), 2.02–1.94 ppm (m, 2H).</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 4-({3-[4-(7-Fluoroisoquinolin-1-yl)piperazin-1-yl]propyl}amino)-2,6,6-trioxo-5,6,7,8-tetrahydro-2<i>H</i>-6ƛ6-thiopyrano[4,3-<i>b</i>]pyrano (<b>21</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 3-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]propan-1-amine <b>Vb</b> (207 mg, 0.71 mmol), TEA (197 μL, 1.42 mmol), 2,6,6-trioxo-5,6,7,8-tetrahydro-2<i>H</i>-6ƛ6-thiopyrano[4,3-<i>b</i>]pyran-4-yl trifluoromethanesulfonate <b>VIi</b> (274 mg, 0.79 mmol), and CH<sub>3</sub>CN (5 mL). Yield = 16%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 487.3; HRMS (ESI): [M + H]<sup>+</sup>, 487.1810; found, 487.1825; HPLC purity: 97.6%, <i>R</i><sub>t</sub> = 15.03 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.49 (br s, 1H), 8.16 (d, <i>J =</i> 5.8 Hz, 1H), 8.07 (dd, <i>J =</i> 5.6, 8.9 Hz, 1H), 7.85 (dd, <i>J =</i> 2.1, 10.2 Hz, 1H), 7.71 (dt, <i>J =</i> 2.5, 8.8 Hz, 1H), 7.57 (d, <i>J =</i> 5.8 Hz, 1H), 6.92 (t, <i>J =</i> 5.4 Hz, 1H), 5.06 (s, 1H), 4.07 (s, 2H), 3.82 (d, <i>J =</i> 9.3 Hz, 2H), 3.60 (d, <i>J =</i> 5.8 Hz, 2H), 3.51 (t, <i>J =</i> 6.4 Hz, 2H), 3.46–3.32 (m, 4H), 3.25–3.22 (m, 4H), 3.02 (t, <i>J =</i> 6.1 Hz, 2H), 2.04–1.98 ppm (m, 2H).</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> 4-({3-[4-(7-Fluoroisoquinolin-1-yl)piperazin-1-yl]propyl}amino)-7,8-dihydro-2<i>H</i>,5<i>H</i>-thiopyrano[4,3-<i>b</i>]pyran-2-one (<b>22</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 3-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]propan-1-amine <b>Vb</b> (207 mg, 0.71 mmol), TEA (197 μL, 1.42 mmol), 2-oxo-7,8-dihydro-2<i>H</i>,5<i>H</i>-thiopyrano[4,3-<i>b</i>]pyran-4-yl trifluoromethanesulfonate <b>VIj</b> (250 mg, 0.79 mmol), and CH<sub>3</sub>CN (2 mL). Yield = 26%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 455.3; HRMS (ESI): [M + H]<sup>+</sup>, 455.1912; found, 455.1920; HPLC purity: 98.4%, <i>R</i><sub>t</sub> = 17.92 min; <sup>1</sup>H-NMR of the hydrochloride salt (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.27 (br s, 1H), 8.13–8.09 (m, 2H), 7.89 (dd, <i>J =</i> 10.1, 2.0 Hz, 1H), 7.77 (td, <i>J =</i> 8.8, 2.4 Hz, 1H), 7.60 (d, <i>J =</i> 5.5 Hz, 1H), 7.01 (t, <i>J =</i> 5.0 Hz, 1H), 4.98 (s, 1H), 3.90 (d, <i>J =</i> 10.0 Hz, 2H), 3.64–3.59 (m, 2H), 3.43–3.37 (m, 4H), 3.28–3.21 (m, 4H), 2.85 (t, <i>J =</i> 5.5 Hz, 2H), 2.67 (t, <i>J =</i> 5.7 Hz, 2H), 2.06 ppm (quin, <i>J =</i> 7.1 Hz, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.0, 160.2 (d, <i>J =</i> 246.4 Hz), 158.1, 157.8, 155.9, 137.1, 135.0, 130.6 (d, <i>J =</i> 9.2 Hz), 121.6 (d, <i>J =</i> 8.4 Hz), 121.3 (d, <i>J =</i> 24.4 Hz), 116.3, 109.6 (d, <i>J =</i> 22.1 Hz), 105.0, 79.5, 53.2, 50.6, 47.6, 29.0, 23.3, 21.9, 21.6 ppm.</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> 4-({3-[4-(Isoquinolin-1-yl)piperazin-1-yl]propyl}(methyl)amino)-2<i>H</i>-chromen-2-one (<b>23</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 3-[4-(isoquinolin-1-yl)piperazin-1-yl]-<i>N</i>-methylpropan-1-amine <b>Vk</b> (56 mg, 0.197 mmol), TEA (0.269 mmol, 37 μL), 2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIa</b> (53 mg, 0.179 mmol), and CH<sub>3</sub>CN (2 mL). Yield = 35%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 429.2; HRMS (ESI): [M + H]<sup>+</sup>, 429.2285; found, 429.2264; HPLC purity: 97.1%, <i>R</i><sub>t</sub> = 15.77 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.12 (br s, 1H), 8.17 (d, <i>J =</i> 8.3 Hz, 1H), 8.11 (d, <i>J =</i> 5.8 Hz, 1H), 7.99 (d, <i>J =</i> 8.0 Hz, 1H), 7.84–7.81 (m, 2H), 7.69 (t, <i>J =</i> 7.5 Hz, 1H), 7.61 (t, <i>J =</i> 7.5 Hz, 1H), 7.55 (d, <i>J =</i> 5.8 Hz, 1H), 7.39–7.34 (m, 2H), 6.27 (br s, 1H), 5.64 (s, 1H), 3.96 (d, <i>J =</i> 13.8 Hz, 2H), 3.67–3.55 (m, 4H), 3.47–3.38 (m, 4H), 3.24–3.19 (m, 2H), 3.06 (s, 3H), 2.22 ppm (quin, <i>J =</i> 7.0 Hz, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.0, 159.9, 157.7, 157.7, 153.6, 138.1, 132.0, 131.6, 127.6, 127.6, 126.1, 125.8, 123.4, 120.4, 117.2, 116.4, 115.9, 93.9, 53.0, 50.8, 50.5, 47.8, 20.9 ppm.</div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> 4-{3-[4-(Isoquinolin-1-yl)piperazin-1-yl]propoxy}-2<i>H</i>-chromen-2-one (<b>24</b>)</h3><div class="NLM_p last">NaH (60% dispersion in mineral oil, 15 mg, 0.37 mmol, 1.0 equiv) was added to a stirred solution of 4-hydroxy-2<i>H</i>-1-benzopyran-2-one (60 mg, 0.37 mmol, 1.0 equiv) in CH<sub>3</sub>CN (4 mL). The suspension was stirred at room temperature for 30 min and then added to a solution of 3-[4-(isoquinolin-1-yl)piperazin-1-yl]propyl methanesulfonate <b>Xa</b> (0.37 mmol, 1.0 equiv) in CH<sub>3</sub>CN (4 mL). K<sub>2</sub>CO<sub>3</sub> (76 mg, 0.55 mmol, 1.5 equiv) was added, and the mixture was stirred at 80 °C for 2 h. The volatiles were removed under reduced pressure, and the residue was dissolved in DCM and washed with water and brine. The organic phase was dried over sodium sulfate, filtered, and concentrated in vacuo. The residue was chromatographed on silica gel eluted with a gradient of 0–10% MeOH/NH<sub>4</sub>OH (95/5) in DCM to give the title compound. Yield = 51%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 416.3; HRMS (ESI): [M + H]<sup>+</sup>, 416.1969; found, 416.1975; HPLC purity: 98.1%, <i>R</i><sub>t</sub> = 17.12 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.42 (br s, 1H), 8.20 (d, <i>J =</i> 8.5 Hz, 1H), 8.10 (d, <i>J =</i> 6.0 Hz, 1H), 8.00 (d, <i>J =</i> 8.3 Hz, 1H), 7.95 (dd, <i>J =</i> 1.4, 7.9 Hz, 1H), 7.85 (t, <i>J =</i> 7.4 Hz, 1H), 7.73–7.66 (m, 2H), 7.57 (d, <i>J =</i> 6.0 Hz, 1H), 7.43–7.37 (m, 2H), 5.95 (s, 1H), 4.38 (t, <i>J =</i> 5.6 Hz, 2H), 4.01 (d, <i>J =</i> 13.1 Hz, 2H), 3.73–3.65 (m, 4H), 3.47–3.343 (m, 4H), 2.44–2.37 ppm (m, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.6, 161.5, 157.3, 152.7, 138.2, 132.7, 132.4, 127.8, 127.6, 126.3, 124.0, 123.1, 120.4, 116.4, 116.3, 115.0, 90.7, 66.7, 52.7, 50.5, 47.9, 22.6 ppm.</div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> 3-[4-(7-Fluoroisoquinolin-1-yl)piperazin-1-yl]-<i>N</i>-(2-oxo-2<i>H</i>-chromen-4-yl)propanamide (<b>25</b>)</h3><div class="NLM_p last">A mixture of 2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIa</b> (149 mg, 0.51 mmol, 1.3 equiv), 3-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]propanamide <b>XIa</b> (117 mg, 0.39 mmol, 1.0 equiv), K<sub>3</sub>PO<sub>4</sub> (165 mg, 0.78 mmol, 2.0 equiv), Pd(dba)<sub>2</sub> (11 mg, 0.019 mmol, 0.05 equiv), and Xantphos (11 mg, 0.019 mmol, 0.05 equiv) was loaded into a Schlenk tube, which was filled with N<sub>2</sub>. Dry dioxane (4 mL) was added, and the resulting mixture was stirred at 100 °C for 30 min. The organic phase was washed with H<sub>2</sub>O, filtered through a hydrophobic frit (Phase Separator), and evaporated under reduced pressure. The residue was chromatographed on silica gel (SNAP 10) eluted with a gradient of 50–100% EtOAc in cyclohexane to obtain the title compound. Yield = 29%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 447.3; HRMS (ESI): [M + H]<sup>+</sup>, 447.1827; found, 447.1857; HPLC purity: 95.5%, <i>R</i><sub>t</sub> = 14.03 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.81 (br s, 1H), 10.44 (s, 1H), 8.34 (dd, <i>J =</i> 1.4, 8.4 Hz, 1H), 8.16 (d, <i>J =</i> 5.8 Hz, 1H), 8.08 (dd, <i>J =</i> 5.8, 9.0 Hz, 1H), 7.86 (dd, <i>J =</i> 2.5, 10.3 Hz, 1H), 7.74–7.68 (m, 2H), 7.58 (d, <i>J =</i> 5.8 Hz, 1H), 7.47–7.43 (m, 2H), 7.24 (s, 1H), 3.85 (d, <i>J =</i> 10.5 Hz, 2H), 3.63–3.61 (m, 2H), 3.58–3.53 (m, 2H), 3.50–3.41 (m, 4H), 3.31 ppm (t, <i>J =</i> 7.0 Hz, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.8, 161.0, 160.1 (d, <i>J =</i> 245.8 Hz), 158.3, 153.1, 146.5, 137.7, 135.0, 132.5, 130.6 (d, <i>J =</i> 8.5 Hz), 123.8, 123.6, 121.6 (d, <i>J =</i> 8.5 Hz), 121.2 (d, <i>J =</i> 24.6 Hz), 117.0, 116.4, 114.0, 109.4 (d, <i>J =</i> 23.8 Hz), 99.9, 50.9, 47.6, 31.1 ppm.</div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> 4-(Isoquinolin-1-yl)-1-{3-[(2-oxo-2<i>H</i>-chromen-4-yl)amino]propyl}piperazin-2-one (<b>26</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 1-(3-aminopropyl)-4-(isoquinolin-1-yl)piperazin-2-one <b>XIVa</b> (85 mg, 0.30 mmol), TEA (57 μL, 0.41 mmol), 2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIa</b> (80 mg, 0.27 mmol), and CH<sub>3</sub>CN (4 mL). Yield = 80%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 429.3; HRMS (ESI): [M + H]<sup>+</sup>, 429.1921; found, 429.1920; HPLC purity: 99.6%, <i>R</i><sub>t</sub> = 15.59 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.23 (d, <i>J =</i> 8.0 Hz, 1H), 8.05 (d, <i>J =</i> 8.0 Hz, 1H), 8.00–7.97 (m, 2H), 7.87 (t, <i>J =</i> 6.9 Hz, 1H), 7.72–7.68 (m, 2H), 7.659–7.55 (m, 1H), 7.50 (d, <i>J =</i> 6.3 Hz, 1H), 7.32–7.28 (m, 2H), 5.19 (s, 1H), 4.24 (br s, 2H), 3.92 (br s, 2H), 3.60 (br s, 2H), 3.51 (t, <i>J =</i> 7.0 Hz, 2H), 3.30 (q, <i>J =</i> 7.0 Hz, 2H), 1.93 ppm (quin, <i>J =</i> 7.0 Hz, 2H); <sup>13</sup>C-NMR (125 MHz; DMSO-<i>d</i><sub>6</sub>): δ 164.4, 161.5, 155.7, 153.0, 153.0, 138.3, 132.8, 131.8, 127.7, 127.6, 126.8, 123.2, 122.4, 120.0, 116.8, 115.2, 114.4, 81.3, 53.7, 48.5, 44.9, 43.9, 25.0 ppm.</div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> <i>cis</i>-4-({3-[4-(Isoquinolin-1-yl)piperazin-1-yl]cyclobutyl}amino)-2<i>H</i>-chromen-2-one (<b>27</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using <i>cis</i>-3-[4-(isoquinolin-1-yl)piperazin-1-yl]cyclobutan-1-amine <b>Vi</b> (55 mg, 0.177 mmol), TEA (37 μL, 0.266 mmol), 2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIa</b> (52 mg, 0.195), and CH<sub>3</sub>CN (2 mL). Yield = 43%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 427.4; HRMS (ESI): [M + H]<sup>+</sup>, 427.2129; found, 427.2137; HPLC purity: 98.4%, <i>R</i><sub>t</sub> = 12.90 min; <sup>1</sup>H-NMR of the hydrochloride salt (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.03 (br s, 1H), 8.24 (t, <i>J =</i> 9.3 Hz, 2H), 8.08 (d, <i>J =</i> 6.4 Hz, 1H), 8.04–8.01 (m, 2H), 7.89 (t, <i>J =</i> 7.3 Hz, 1H), 7.73 (t, <i>J =</i> 7.6 Hz, 1H), 7.62–7.59 (m, 2H), 7.36–7.31 (m, 2H), 5.11 (s, 1H), 4.08 (d, <i>J =</i> 12.2 Hz, 2H), 3.96–3.89 (m, 1H), 3.76 (br s, 2H), 3.67–3.63 (m, 1H), 3.57 (d, <i>J =</i> 11.7 Hz, 2H), 3.38–3.23 (m, 2H), 2.92–2.86 (m, 2H), 2.71–2.66 ppm (m, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.3, 157.5, 153.0, 152.1, 138.2, 132.3, 131.9, 127.8, 127.6, 126.3, 123.2, 122.9, 120.4, 116.8, 116.4, 114.1, 82.7, 53.2, 47.9, 47.7, 40.3, 31.8 ppm.</div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> <i>trans</i>-4-({3-[4-(Isoquinolin-1-yl)piperazin-1-yl]cyclobutyl}amino)-2<i>H</i>-chromen-2-one (<b>28</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using <i>trans</i>-3-[4-(isoquinolin-1-yl)piperazin-1-yl]cyclobutan-1-amine <b>Vj</b> (55 mg, 0.177 mmol), TEA (37 μL, 0.266 mmol), 2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIa</b> (52 mg, 0.195), and CH<sub>3</sub>CN (2 mL). Yield = 35%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 427.4; HRMS (ESI): [M + H]<sup>+</sup>, 427.2129; found, 427.2122; HPLC purity: 99.5%, <i>R</i><sub>t</sub> = 14.87 min; <sup>1</sup>H-NMR of the hydrochloride salt (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.21 (br s, 1H), 8.29–8.23 (m, 2H), 8.07–8.01 (m, 3H), 7.91 (t, <i>J =</i> 7.3 Hz, 1H), 7.75 (t, <i>J =</i> 7.7 Hz, 1H), 7.63–7.60 (m, 2H), 7.36–7.32 (m, 2H), 4.98 (s, 1H), 4.35–4.33 (m, 1H), 4.14–4.12 (m, 2H),4.01–3.98 (m, 1H), 3.85–3.83 (m, 2H), 3.69–3.66 (d, <i>J =</i> 11.4 Hz, 2H), 3.33–3.10 (m, 2H), 3.03–2.98 (m, 2H), 2.65–2.615 ppm (m, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.2, 157.4, 153.0, 152.4, 138.3, 132.6, 131.9, 127.8, 127.6, 126.4, 123.2, 123.0, 120.4, 116.8, 116.4, 114.2, 82.8, 56.5, 48.0, 47.8, 43.0, 29.7 ppm.</div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> 4-({(2<i>R</i>)-3-[4-(7-Fluoroisoquinolin-1-yl)piperazin-1-yl]-2-hydroxypropyl}amino)-2<i>H-</i>1-benzopyran-2-one (<b>(<i>R</i>)-29</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure J using (2<i>R</i>)-1-amino-3-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]propan-2-ol <b>(<i>R</i>)-XVa</b> (80 mg, 0.26 mmol), 2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIa</b> (70 mg, 0.24 mmol), TEA (50 μL, 0.36 mmol), and CH<sub>3</sub>CN (3 mL). Yield = 34%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 449.1; HRMS (ESI): [M + H]<sup>+</sup>, 449.1983; found, 449.1970; HPLC purity: 96.7%, <i>R</i><sub>t</sub> = 15.42 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.16 (br s, 1H), 8.19–8.14 (m, 2H), 8.07 (dd, <i>J =</i> 5.6, 8.9 Hz, 1H), 7.93–7.91 (m, 1H), 7.84 (d, <i>J =</i> 10.3 Hz, 1H), 7.71 (dt, <i>J =</i> 2.3, 8.8 Hz, 1H), 7.63–7.59 (m, 1H), 7.56 (d, <i>J =</i> 6.0 Hz, 1 H), 7.36–7.32 (m, 2H), 5.37 (s, 1H), 4.38–4.34 (m, 2H), 3.82 (t, <i>J =</i> 13.0 Hz, 2H), 3.71 (d, <i>J =</i> 10.8 Hz, 1H), 3.61 (d, <i>J =</i> 10.8 Hz, 1H), 3.55–3.122 ppm (m, 8H).</div></div><div id="sec4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> 4-({(2<i>S</i>)-3-[4-(7-Fluoroisoquinolin-1-yl)piperazin-1-yl]-2-hydroxypropyl}amino)-2<i>H</i>-1-benzopyran-2-one (<b>(<i>S</i>)-29</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure J using (2<i>S</i>)-1-amino-3-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]propan-2-ol <b>(<i>S</i>)-XVb</b> (53 mg, 0.17 mmol), 2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIa</b> (45 mg, 0.15 mmol), TEA (36 μL, 0.26 mmol), and CH<sub>3</sub>CN (2 mL). Yield = 45%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 449.1; HRMS (ESI): [M + H]<sup>+</sup>, 449.1983; found, 449.1992; HPLC purity: 95.1%, <i>R</i><sub>t</sub> = 15.44 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.24 (br s, 1H), 8.19 (d, <i>J =</i> 8.0 Hz, 1H), 8.14 (d, <i>J =</i> 5.8 Hz, 1H), 8.07 (dd, <i>J =</i> 5.8, 9.0 Hz, 1H), 7.94 (t, <i>J =</i> 5.5 Hz, 1H), 7.85 (dd, <i>J =</i> 2.3, 10.3 Hz, 1H), 7.72 (dt, <i>J =</i> 2.5, 8.8 Hz, 1H), 7.63–7.56 (m, 2H), 7.36–7.32 (m, 2H), 5.36 (s, 1H), 4.39–4.36 (m, 2H), 3.86–3.22 ppm (m, 12H).</div></div><div id="sec4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> 4-({2-[4-(Isoquinolin-1-yl)piperazin-1-yl]ethyl}amino)chromen-2-one Dihydrochloride (<b>30</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure H using 2-[4-(isoquinolin-1-yl)piperazin-1-yl]ethan-1-amine <b>Vh</b> (60 mg, 0.23 mmol), TEA (34 μL, 0.24 mmol), 2-oxo-2<i>H</i>-1-benzopyran-4-yl trifluoromethanesulfonate <b>VIa</b> (62 mg, 0.21 mmol), and CH<sub>3</sub>CN (5 mL). Yield = 38%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 401.3; HRMS (ESI): [M + H]<sup>+</sup>, 401.1972; found, 401.1973; HPLC purity: 97.9%, <i>R</i><sub>t</sub> = 16.20 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.28 (br s, 1H), 8.23 (d, <i>J =</i> 7.5 Hz, 1H), 8.19 (d, <i>J =</i> 8.3 Hz, 1H), 8.11 (d, <i>J =</i> 5.8 Hz, 1H), 8.04 (t, <i>J =</i> 5.3 Hz, 1H), 7.99 (d, <i>J =</i> 8.0 Hz, 1H), 7.83 (t, <i>J =</i> 7.4 Hz, 1H), 7.73–7.66 (m, 1H), 7.63–7.59 (m, 1H), 7.56 (d, <i>J =</i> 5.8 Hz, 1H), 7.36–7.32 (m, 2H), 5.39 (s, 1H), 3.99 (d, <i>J =</i> 11.8 Hz, 2H), 3.87–3.73 (m, 4H), 3.64 (t, <i>J =</i> 11.4 Hz, 2H), 3.51 ppm (br s, 4H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.4, 157.5, 153.0, 152.9, 138.2, 132.3, 131.9, 127.8, 127.6, 126.2, 123.3, 123.0, 120.4, 116.8, 116.5, 114.4, 82.5, 53.4, 50.7, 47.9, 36.6 ppm.</div></div><div id="sec4_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> (±)-4-{4-[4-(7-Fluoroisoquinolin-1-yl)piperazin-1-yl]-2-hydroxybutyl}2<i>H</i>-chromen-2-one (<b>(±)-31</b>)</h3><div class="NLM_p last">To a solution of 4-{4-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]-2-oxobutyl}-2<i>H</i>-chromen-2-one <b>XVIIIa</b> (100 mg, 0.22 mmol, 1.0 equiv) in MeOH (8 mL), NaBH<sub>4</sub> (18 mg, 1.1 mmol, 5.0 equiv) was added, and the mixture was stirred at room temperature overnight. The volatiles were evaporated under vacuum, and the crude product purified by flash chromatography was eluted with a gradient of 0–10% MeOH/NH<sub>3</sub> (90/10) in DCM to give the title compound. Yield = 46%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 448.4; HRMS (ESI): [M + H]<sup>+</sup>, 448.2031; found, 448.2030; HPLC purity: 99.1%, <i>R</i><sub>t</sub> = 17.10 min; <sup>1</sup>H-NMR of the formate salt (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.20 (s, 1H), 8.12 (d, <i>J =</i> 5.4 Hz, 1H), 8.01 (dd, <i>J =</i> 8.8, 5.87 Hz, 1H), 7.95 (d, <i>J =</i> 7.8 Hz, 1H), 7.72–7.69 (m, 1H), 7.66–7.61 (m, 2H), 7.45 (d, <i>J =</i> 5.9 Hz, 1H), 7.41–7.36 (m, 2H), 6.39 (s, 1H), 3.96–3.91 (m, 1H), 3.31 (br s, 4H), 3.08 (dd, <i>J =</i> 13.4, 3.2 Hz, 1H), 2.878–2.659 (m, 6H), 2.59–2.54 (m, 1H), 1.81–1.71 ppm (m, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.5, 160.3, 159.7, 159.7 (d, <i>J =</i> 245.7 Hz), 154.2, 153.2, 140.0, 134.7, 131.8, 130.3 (d, <i>J =</i> 8.4 Hz), 125.6, 124.1, 121.7 (d, <i>J =</i> 7.6 Hz), 120.1 (d, <i>J =</i> 25.2 Hz), 119.3, 116.6, 115.4, 115.3, 108.7 (d, <i>J =</i> 22.1 Hz), 67.9, 54.7, 52.7, 50.8, 33.8 ppm.</div></div><div id="sec4_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> (±)-6-Fluoro-4-({2-hydroxy-3-[4-(1,6-naphthyridin-5-yl)piperazin-1-yl]propyl}amino)-2<i>H</i>-1-benzopyran-2-one (<b>(±)-32</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure I using 5-chloro-1,6-naphthyridine (430 mg, 2.61 mmol), K<sub>2</sub>CO<sub>3</sub> (721 mg, 5.22 mmol), (±)-6-fluoro-4-{[2-hydroxy-3-(piperazin-1-yl)propyl]amino}-2<i>H</i>-1-benzopyran-2-one <b>(±)-XXa</b> (924 mg, 2.87 mmol), and DMSO (10 mL). Yield = 80%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 450.0; HRMS (ESI): [M + H]<sup>+</sup>, 450.1936; found, 450.1931; HPLC purity: 98.1%, <i>R</i><sub>t</sub> = 12.93 min; <sup>1</sup>H-NMR of the free base (300 MHz, CDCl<sub>3</sub>): δ 9.01 (dd, <i>J</i> = 1.2, 4.3 Hz, 1H), 8.40–8.34 (m, 2H), 7.53 (d, <i>J</i> = 5.9 Hz, 1H), 7.45 (dd, <i>J</i> = 4.3, 8.4 Hz, 1H), 7.36–7.19 (m, 3H), 5.78 (t, <i>J</i> = 4.3 Hz, 1H), 5.36 (s, 1H), 4.21–4.14 (m, 1H), 3.56–3.43 (m, 6H), 3.26–3.18 (m, 1H), 3.02–2.95 (m, 2H), 2.79–2.72 (m, 2H), 2.62 ppm (d, <i>J</i> = 6.8 Hz, 2H); <sup>13</sup>C-NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.4, 161.2, 157.8 (d, <i>J =</i> 239.3 Hz), 153.9, 152.7, 152.3, 149.4, 144.1, 133.9, 121.5, 119.2, 118.8 (d, <i>J =</i> 9.0 Hz), 116.0 (d, <i>J =</i> 21.4 Hz), 115.9, 115.6 (d, <i>J =</i> 7.5 Hz), 108.6 (d, <i>J =</i> 26.3 Hz), 81.9, 65.7, 62.2, 53.5, 51.1, 47.7 ppm.</div></div><div id="sec4_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> (±)<i>-</i>6-Fluoro-4-({3-[4-(3-fluoro-7-methoxyisoquinolin-1-yl)piperazin-1-yl]-2-hydroxypropyl}amino)-2<i>H</i>-1-benzopyran-2-one (<b>(±)-33</b>)</h3><div class="NLM_p last">DIPEA (1.7 mL, 9.57 mmol, 5 equiv) was added to a stirred suspension of 1-chloro-3-fluoro-7-methoxyisoquinoline (404 mg, 1.91 mmol, 1.0 equiv) and (±)-6-fluoro-4-{[2-hydroxy-3-(piperazin-1-yl)propyl]amino}-2<i>H</i>-1-benzopyran-2-one <b>(±)-XXa</b> (754 mg, 1.91 mmol, 1.0 equiv) in DMSO (15 mL). The resulting mixture was stirred at 100 °C for 18 h and then cooled, poured into water (25 mL), and extracted with Et<sub>2</sub>O (3 × 25 mL) and with DCM (20 mL). The combined organic layers were dried over magnesium sulfate, filtered, and concentrated. The crude material was purified by flash chromatography eluted with a gradient of 0–10% MeOH/NH<sub>3</sub> (90/10) in DCM to give the title compound. Yield = 39%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 497.1; HRMS (ESI): [M + H]<sup>+</sup>, 497.1995; found, 497.2002; HPLC purity: 97.3%, <i>R</i><sub>t</sub> = 14.44 min; <sup>1</sup>H-NMR of the free base (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.02 (dd, <i>J =</i> 10.0, 2.9 Hz, 1H), 7.82 (d, <i>J =</i> 9.0 Hz, 1H), 7.769 (t, <i>J =</i> 5.5 Hz, 1H), 7.49–7.45 (m, 1H), 7.420–7.37 (m, 2H), 7.28 (d, <i>J =</i> 2.4 Hz, 1H), 7.01 (s, 1H), 5.34 (s, 1H), 4.99 (d, <i>J =</i> 4.8 Hz, 1H), 4.00–3.97 (m, 1H), 3.89 (s, 3H), 3.48–3.35 (m, 5H), 3.26–3.20 (m, 1H), 2.79–2.72 (m, 2H) 2.55–2.46 ppm (m, 3H).</div></div><div id="sec4_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> 4-({(2<i>R</i>)-3-[4-(7-Fluoroisoquinolin-1-yl)piperazin-1-yl]-2-hydroxypropyl}amino)-2<i>H</i>-1,3-benzoxazin-2-one (<b>(<i>R</i>)-34</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure J using (2<i>R</i>)-1-amino-3-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]propan-2-ol <b>(<i>R</i>)</b>-<b>XVa</b> (65 mg, 0.21 mmol), 4-chloro-2<i>H</i>-1,3-benzoxazin-2-one <b>XVIa</b> (76 mg, 0.42 mmol), TEA (44 μL, 0.31 mmol), and CH<sub>3</sub>CN (2 mL). Yield = 24%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 450.3; HRMS (ESI): [M + H]<sup>+</sup>, 450.1936; found, 450.1942; HPLC purity: 98.7%, <i>R</i><sub>t</sub> = 14.17 min; <sup>1</sup>H-NMR of the hydrochloride salt (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.95 (br s, 1H), 9.439–9.37 (m, 1H), 8.25 (d, <i>J =</i> 6.0 Hz, 1H), 8.15 (d, <i>J =</i> 5.8 Hz, 1H), 8.05 (dd, <i>J =</i> 5.6, 8.7 Hz, 1H), 7.82 (d, <i>J =</i> 10.3 Hz, 1H), 7.77–7.67 (m, 2H), 7.55 (d, <i>J =</i> 5.8 Hz, 1H), 7.39–7.32 (m, 2H), 4.43–4.37 (m, 2H), 3.84–3.33 (m, 11H), 3.28–3.21 ppm (m, 1H).</div></div><div id="sec4_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> (±)-6-Fluoro-4-({3-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]-2-hydroxypropyl}amino)-2<i>H</i>-1,3-benzoxazin-2-one (<b>(±)-35</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure K using 4-chloro-6-fluoro-2<i>H</i>-1,3-benzoxazin-2-one <b>XVIb</b> (235 mg, 1.2 mmol), DIPEA (402 mg, 3.1 mmol), (±)-1-amino-3-[4-(7-fluoroisoquinolin-1-yl)piperazin-1-yl]propan-2-ol <b>(±)-XVc</b> (120 mg, 0.39 mmol), and CH<sub>3</sub>CN (5 mL). Yield = 74%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 468.2; HRMS (ESI): [M + H]<sup>+</sup>, 468.1842; found, 468.1855; HPLC purity: 96.5%, <i>R</i><sub>t</sub> = 14.52 min; <sup>1</sup>H-NMR of the free base (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.12 (t, <i>J =</i> 5.4, 1H), 8.13–8.10 (m, 2H), 8.00 (dd, <i>J =</i> 9.0, 5.7 Hz, 1H), 7.70 (dd, <i>J =</i> 10.3, 2.6 Hz, 1H), 7.63 (qd, <i>J =</i> 8.9, 2.9 Hz, 2H), 7.44 (d, <i>J =</i> 5.6 Hz, 1H), 7.39–7.36 (m, 1H), 4.99 (d, <i>J =</i> 4.5 Hz, 1H), 4.11–4.03 (m, 1H), 3.77–3.71 (m, 1H), 3.47–3.40 (m, 1H), 3.29–3.26 (m, 4H), 2.81–2.70 (m, 4H), 2.55–2.45 ppm (m, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO<i>-d</i><sub>6</sub>): δ 160.3, 159.9 (d, <i>J =</i> 297.5 Hz), 159.4 (d, <i>J =</i> 301.2 Hz), 156.3, 151.7, 151.5, 140.0, 134.7, 130.2 (d, <i>J =</i> 8.3 Hz), 122.4 (d, <i>J =</i> 23.9 Hz), 121.6 (d, <i>J =</i> 7.3 Hz), 120.0 (d, <i>J =</i> 24.8 Hz), 118.6 (d, <i>J =</i> 8.3 Hz), 115.3, 110.6 (d, <i>J =</i> 8.3 Hz), 109.7 (d, <i>J =</i> 25.7 Hz), 108.7 (d, <i>J =</i> 21.1 Hz), 65.8, 62.4, 53.3, 50.8, 46.2 ppm.</div></div><div id="sec4_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> (±)-4-({3-[4-(3,7-Difluoroisoquinolin-1-yl)piperazin-1-yl]-2-hydroxypropyl}amino)-6-fluoro-2<i>H</i>-1,3-benzoxazin-2-one (<b>(±)-36</b>)</h3><div class="NLM_p last">The title compound was obtained according to general procedure K using 4-chloro-6-fluoro-2<i>H</i>-1,3-benzoxazin-2-one <b>XVIb</b> (83 mg, 0.42 mmol), DIPEA (0.11 mg, 0.85 mmol), and (±)-1-amino-3-[4-(3,7-difluoroisoquinolin-1-yl)piperazin-1-yl]propan-2-ol <b>(±)-XVd</b> (46 mg, 0.14 mmol), and CH<sub>3</sub>CN (3 mL). Yield = 23%. LCMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup>, 486.2; HRMS (ESI): [M + H]<sup>+</sup>, 486.1748; found, 486.1737; HPLC purity: 99.2%, <i>R</i><sub>t</sub> = 14.20 min; <sup>1</sup>H-NMR of the free base (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.13 (s, 1H), 8.10 (dd, <i>J =</i> 9.4, 3.0 Hz, 1H), 7.99 7.72–7.60 (m, 3H), 7.38 (dd, <i>J =</i> 9.0, 4.6 Hz, 1H), 7.10 (s, 1H), 5.01 (br d, <i>J =</i> 4.8, 1H), 4.10–4.04 (m, 1H), 3.74 (dd, <i>J =</i> 12.8, 4.4, 1H), 3.37–3.41 (m, 5H), 2.80–2.70 (m, 4H), 2.41 ppm (m, 2H); <sup>13</sup>C-NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.2, 159.3, 158.8 (d, <i>J =</i> 243.7 Hz), 158.6 (d, <i>J =</i> 229.8 Hz), 157.3 (d, <i>J =</i> 240.8 Hz), 151.8, 151.5, 138.2 (d, <i>J =</i> 9.2 Hz), 129.8 (t, <i>J =</i> 7.5 Hz), 122.5 (d, <i>J =</i> 24.8 Hz), 121.1 (d, <i>J =</i> 24.8 Hz), 119.3 (d, <i>J =</i> 8.3 Hz), 118.6 (d, <i>J =</i> 8.3 Hz), 110.6 (d, <i>J =</i> 8.3 Hz), 109.8 (d, <i>J =</i> 24.8 Hz), 109.3 (d, <i>J =</i> 22.1 Hz), 94.9 (d, <i>J =</i> 37.7 Hz), 65.8, 62.3, 53.1, 50.7, 46.2 ppm.</div></div><div id="sec4_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> Computational Methods</h3><div id="sec4_43_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> Dataset Preparation</h4><div class="NLM_p">KNIME<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> workflows were used, integrating the Schrödinger KNIME extensions<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and custom Python<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> scripts to speed up the calculations and manage large amounts of data.</div><div class="NLM_p">The 44 unique chemical compounds used for the docking model validation were collected from the literature, and the IC<sub>50</sub> values of these compounds are provided (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Table S1</a>).<a onclick="showRef(event, 'ref31 ref44 ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref31 ref44 ref45 ref46 ref47">(31,44−47)</a> The 2D structures of approximately 3 million compounds were collected from different worldwide vendors, and custom filters based on physicochemical, chemical, and toxicological properties were applied.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The well-documented rapid elimination of swill (REOS), pan-assay interference compounds (PAINS), and toxicophore filters (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Table S5</a>) were applied, leading to a significant reduction in the number of compounds that were deemed worthy of further consideration. Following two-step diversity subset generation, the number of compounds was further reduced to approximately 25,000. An additional 5000 compounds were selected from Angelini’s proprietary collection. The combination of these two sets provided the 30,000-molecule set used in the computational screens.</div><div class="NLM_p last">The eight compounds used for generation of the pharmacophore model are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Table S2</a> (Supporting Information).</div></div><div id="sec4_43_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> Ligand and Protein Preparation</h4><div class="NLM_p">All of the small-molecule structures, including the set of known NBTIs (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Tables S1 and S2</a>), were prepared using the LigPrep tool from Schrödinger.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The LigPrep tool runs a number of calculations to convert raw ligand information to full 3D structures that are suitable for subsequent modeling operations. These steps include (i) addition of all relevant hydrogen atoms, including an exploration of the energetically plausible protonation and tautomerization states at pH = 7.0 ± 2.0 using Schrödinger’s Epik module,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> and (ii) generation of plausible low-energy 3D coordinates for the ligand including, where necessary, enumeration of unspecified stereo-centers.</div><div class="NLM_p last">The protein structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>)<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> was made suitable for further modeling using the Protein Preparation Wizard tools supplied with the Schrödinger modeling suite.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The Protein Preparation Wizard converts raw crystal structures, primarily obtained from the PDB,<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> into fully atom-typed structures usable for high-level calculations. This procedure involves several stages, including (i) assignment of bond order and addition of hydrogens to complete valencies, (ii) enumeration of ionization states and tautomers of both the protein and ligand, (iii) adjustment of side chains and waters to maximize hydrogen bonding networks, and (iv) restrained relaxation of the crystallographic coordinates to make them consistent with the optimized potential for liquid simulation (OPLS_2005/OPLS2.1) force field used for subsequent modeling.</div></div><div id="sec4_43_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> Docking Calculations</h4><div class="NLM_p last">Calculations for docking into the prepared <i>S. aureus</i> DNA gyrase structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>)<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> were performed using the Glide docking program.<a onclick="showRef(event, 'ref30 ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref30 ref53 ref54 ref55">(30,53−55)</a> Glide functions by evaluating the energies of a large number of potential conformations of each ligand within the binding site using the proprietary “GlideScore” scoring function. To accommodate various numbers of ligands and computational resources, Glide provides several different modes: (i) high-throughput virtual screening for coarse filtering of large numbers of compounds, (ii) standard precision (SP) for general production for intermediate numbers of compounds, and (iii) extra precision (XP) for detailed calculations with considerably increased computational complexity. For this work, SP with default settings was considered to provide the correct balance of accuracy and throughput.</div></div><div id="sec4_43_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> EF Calculation</h4><div class="NLM_p last">The EFs were calculated at 2, 5, 10, and 20% of the entire screened database using the following formula<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_m001.gif" alt="" id="_i83" /></img></span>where hits<sub>sampled</sub> is the number of active compounds retrieved in the first <i>n</i> positions of the screened database, and hits<sub>total</sub> is the total number of active compounds included in the database of <i>N</i><sub>total</sub> compounds. <i>N</i><sub>sampled</sub> refers to the compounds for which the model found a binding pose. Here, we report the EF in graphical and tabular forms and present accumulation curves that show how the fraction of active compounds recovered varies with the percentage of the database screened (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div></div><div id="sec4_43_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> Pharmacophore-Based Model</h4><div class="NLM_p last">Pharmacophore modeling was carried out using the Schrödinger’s Phase program.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Phase assigns pharmacophoric features to the underlying ligands using a set of feature definitions, which can be customized by the user. The program then attempts to generate common <i>k</i>-point pharmacophores based on these definitions by searching through an ensemble of low-energy conformations of the input compounds; in this case, the input compounds were the eight known NBTIs selected from the literature (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Table S2</a>). In many instances, this search will locate several potential pharmacophore models, which need further evaluation. For pharmacophore model building, we have chosen as input structures a focused set of eight compounds that are derived from the literature search. The Phase program automatically finds common pharmacophore features between the structures of the active molecules. In our case, we selected two of the possible pharmacophoric models based on their ability to recognize the bioactive conformation of GSK299423 and their anticipated ability to enrich for active NBTIs.</div></div><div id="sec4_43_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> Shape-Based Model</h4><div class="NLM_p last">Shape screening was performed with the phase_shape component of the main Phase program.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The phase_shape utility aligns compounds to the user-specified query, returning a score derived from the overlap of the two compounds. The overlap can be calculated using several metrics, ranging from pure shape, where all atoms are considered to be equivalent volumes, to more discriminatory functions, where the overlap considers only matching pharmacophoric features.</div></div><div id="sec4_43_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> Cluster Analysis for Compound Selection</h4><div class="NLM_p last">The 900 compounds located after the SBVS and LBVS stages were clustered by considering their Tanimoto distances derived from Canvas fingerprints.<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(56,57)</a> The highest diversity was obtained by combining radial-triplet fingerprints (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">Figure S7</a>). Thereafter, the 900 compounds were duly considered, and final selection of the virtual hits, considering the most diverse hits from each set (docking, shape, and pharmacophore models), was performed. In addition, to include the highest number of true positives as an additional parameter, consensus analysis was carried out within the three sets.</div></div><div id="sec4_43_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> MD Simulations</h4><div class="NLM_p">MD simulations were run using the graphics processing unit (GPU) tool of the Desmond MD package,<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> as supplied with the Schrödinger 2014-4 suite.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a></div><div class="NLM_p">Prior to simulation, all protein–ligand complexes (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>) were again prepared using the Schrödinger Protein Preparation Wizard.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> For the most part, default settings were applied during protein preparation. However, care was taken to preserve any crystallographically determined water molecules from the original structure, thus easing the burden of solvent equilibration in critical areas.</div><div class="NLM_p">The prepared complex was converted to a fully solvated system ready for MD simulation using the Desmond System Builder included in the Schrödinger software suite.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> The System Builder was instructed to solvate the system using simple point charge (SPC) waters, providing an orthorhombic solvation box with a 10 Å buffer around the original complex. Electrostatic neutrality was ensured with the addition of Na<sup>+</sup>/Cl<sup>–</sup> ions, and an additional ionic atmosphere of 0.15 M NaCl was included. Typical systems consisted of approximately 160,000 atoms.</div><div class="NLM_p">Force field parameters were supplied by the OPLS-2.1 force field.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Default parameters were assigned to the protein. Ligand torsions were assigned with default values from the OPLS-2.1 force field when a sufficiently good match between the torsion and default parameter was found. However, some ligand torsions were custom-fitted using the Force Field Builder, a component of the OPLS-2.1 force field supplied by Schrödinger. The nucleic acid parameters were assigned from a modified version of the OPLS-2.1 force field that was specifically fit to remove some of the deficiencies observed with the default terms.</div><div class="NLM_p">The overall simulation protocol followed the default relaxation schedule implemented by Schrödinger, which includes the following:<table class="listgroup" border="0" width="95%" list-type="label"><tr class="li1"><td valign="top">1.</td><td colspan="5" valign="top" class="paragraph"><p class="last">100 ps, constant-volume ensemble (NVT) (<i>T</i> = 10 K) Brownian dynamics; 50 kcal/mol/Å<sup>2</sup> restraints on the solute heavy atoms.</p></td></tr><tr class="li1"><td valign="top">2.</td><td colspan="5" valign="top" class="paragraph"><p class="last">12 ps, NVT (<i>T</i> = 10 K) dynamics; 50 kcal/mol/Å<sup>2</sup> restraints on the solute heavy atoms.</p></td></tr><tr class="li1"><td valign="top">3.</td><td colspan="5" valign="top" class="paragraph"><p class="last">12 ps, constant-pressure ensemble (NPT) (<i>T</i> = 10 K, <i>P</i> = 1 atm) dynamics; 50 kcal/mol/Å<sup>2</sup> restraints on the solute heavy atoms.</p></td></tr><tr class="li1"><td valign="top">4.</td><td colspan="5" valign="top" class="paragraph"><p class="last">12 ps, NPT (<i>T</i> = 300 K, <i>P</i> = 1 atm) dynamics; 50 kcal/mol/Å<sup>2</sup> restraints on the solute heavy atoms.</p></td></tr><tr class="li1"><td valign="top">5.</td><td colspan="5" valign="top" class="paragraph"><p class="last">24 ps, NPT (<i>T</i> = 300 K, <i>P</i> = 1 atm) dynamics; unrestrained.</p></td></tr></table></div><div class="NLM_p last">This relaxation schedule was followed by the main production run of NPT (<i>T</i> = 300 K, <i>P</i> = 1 atm) unrestrained dynamics. All production simulations were between 15 and 20 ns in duration. The length of the simulation was chosen to allow reasonable sampling of solvent and local motions without requiring unreasonable amounts of GPU time.</div></div><div id="sec4_43_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> Basic Simulation Analysis</h4><div class="NLM_p last">Basic analysis of the MD trajectories was performed within the Maestro environment of the Schrödinger suite using the supplied Simulation Interaction Diagram tool.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> This tool allowed easy calculation of the root-mean-square deviation (RMSD) and root-mean-square fluctuation (RMSF) of both the protein and ligand, enabling very rapid testing of the basic health of the system as the simulation progressed as well as providing useful 2D summaries of the interactions between the protein and ligand.</div></div><div id="sec4_43_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> WaterMap Calculations</h4><div class="NLM_p last">WaterMap calculations were run and analyzed using the tool chain supplied with the Schrödinger 2014-4 suite.<a onclick="showRef(event, 'ref35 ref36 ref61'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref61">(35,36,61)</a> The relevant protein–ligand structures based on the PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> were prepared using the same approach as that adopted for the MD simulations.</div></div></div><div id="sec4_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Pharmacology: DNA Gyrase and Topoisomerase IV Inhibition Assays</h3><div class="NLM_p last"><i>S. aureus</i> and <i>E. coli</i> gyrase supercoiling assay kits and topoisomerase IV decatenation assay kits were from Inspiralis (Norwich, U.K.). For gyrase supercoiling gel assays, 1 U of DNA gyrase was incubated with 0.5 μg of relaxed pBR322 DNA. In topoisomerase IV decatenation gel assays, 1 U of topoisomerase IV was incubated with 200 ng of kDNA in a 30 μL reaction at 37 °C for 30 min. Concentration–response curves were performed for each compound by testing eight concentrations in duplicate with twofold or half-log dilutions starting from 800 or 300 μM. Final DMSO concentration in the assays was 1% (v/v). Each reaction was stopped by the addition of 30 μL of chloroform/iso-amyl alcohol (26:1) and 30 μL of Stop Dye (40% sucrose, 100 mM Tris–HCl (pH 7.5), 1 mM EDTA, 0.5 μg/mL bromophenol blue) before being loaded on a 1.0% TAE (0.04 mM Tris acetate and 0.002 mM EDTA) gel run at 80 V for 2 h. Bands were visualized by ethidium bromide staining for 10 min in water, analyzed by gel documentation equipment (Syngene, Cambridge, U.K.), and quantitated using Syngene Gene Tools software. Raw gel data (fluorescent band volumes) were converted by the software to a percentage of the control (the fully supercoiled or decatenated DNA band). The data were then analyzed using SigmaPlot version 12.3 (2013). The global curve fit nonlinear regression tool was used to calculate the IC<sub>50</sub> values. Ciprofloxacin was used as a positive reference compound.</div></div><div id="sec4_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Microbiology: MIC Testing</h3><div class="NLM_p last">The bacterial strains <i>E. coli</i> ATCC 25922, <i>S. aureus</i> ATCC 29213, <i>S. aureus</i> BAA-1720, <i>E. faecalis</i> ATCC 29212, <i>N. gonorrhoeae</i> ATCC 19424, <i>H. influenzae</i> ATCC 49766, and <i>K. pneumoniae</i> ATCC 700603 were purchased from ATCC via LGC Standards (Middlesex, U.K.). The <i>E. faecalis</i> strain resistant to vancomycin and teicoplanin (Van-A) and <i>E. faecalis</i> strain resistant to vancomycin but sensitive to teicoplanin (Van-B), as well as the <i>E. faecium</i> Van-A and <i>E. faecium</i> Van-B strains, were provided by Aptuit (Verona, Italy). MICs were determined by the broth microdilution method (the agar dilution method was used for <i>N. gonorrhoeae</i> only) according to the guidelines of the CLSI.<a onclick="showRef(event, 'ref62 ref63'); return false;" href="javascript:void(0);" class="ref ref62 ref63">(62,63)</a> The MIC values of antibiotics used in clinical practice were interpreted according to the susceptibility interpretive criteria reported in the appropriate CLSI guidelines.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Moreover, for <i>E. coli</i> ATCC 25922, <i>S. aureus</i> ATCC 29213, and <i>S. aureus</i> BAA-1720, MIC values were also detected in the presence of 50% human serum. Compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> were tested at concentrations ranging from 8 to 0.015 μg/mL, and the reference compound ciprofloxacin was tested at concentrations ranging from 4 to 0.002 μg/mL. After incubation for 16–24 h at 37 °C in ambient air, the MIC was determined as the lowest concentration of the drug that led to no visible growth. The MIC values were determined by visual inspection.</div></div><div id="sec4_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> ADME-Tox</h3><div id="sec4_46_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> Cytotoxicity Assay</h4><div class="NLM_p">Compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> were tested in cell viability assays with HepG2 and HEK293 cells. The viability of cultured cells was evaluated via alamarBlue staining, as described by Nociari et al.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Concentration–response curves were prepared for each compound by testing eight concentrations in triplicate with threefold dilutions starting from 100 to 0.030 μM. After incubation for 48 h at 37 °C, the percentage of control activity was calculated by comparing the fluorescence reading in the presence of the test compound to that in the absence of a test compound (control). IC<sub>50</sub> values were determined by nonlinear regression analysis of the concentration–response curves. These parameters were obtained by Hill equation curve fitting. Chlorpromazine was used as a reference compound. Additionally, cell viability was also assessed in the HT-29, HaCaT, and HCE cell lines. The HT-29 human colorectal adenocarcinoma cell line was obtained from ATCC (LGC Standards, U.K.). Human immortalized keratinocyte HaCaT cells (T0020001; lot. 0000123) were obtained from AddexBio Technologies (San Diego, USA). The experimental assay was performed in HT-29 and HaCaT cells following 24 h of incubation at 37 °C in a humidified atmosphere with 5% CO<sub>2</sub>. Compounds <b>(±)-33</b> and <b>(±)-36</b> were tested by preparing a dose–response curve at four to six increasing concentrations. Staurosporine was used as a positive reference control in both cell models. Rifaximin was used as a reference control only in HT-29, and etoposide was used as a reference control only in HaCaT.</div><div class="NLM_p">The CellTiter-Blue cell viability assay provides a homogeneous fluorimetric method for estimation of the number of viable cells based on the metabolic ability of live cells to convert a redox dye (resazurin) to a highly fluorescent end product, resofurin.</div><div class="NLM_p">After 24 h of incubation, following plate centrifugation, supernatants were discarded, and 100 μL of 0.5% FBS DMEM and 20 μL of CellTiter-Blue cell viability assay reagent were added. The plate was then incubated at 37 °C for 1.5 h, and the fluorescence was measured at 560 nm (excitation)/590 nm (emission) using a spectrofluorometer (Victor III, PerkinElmer). Absorbance values for the vehicle control DMSO were averaged and arbitrarily set to 100% viability. The absolute values of absorbance for each treatment were then normalized with respect to the vehicle control and expressed as a percentage. Viability inhibition of >20% compared to the positive control was considered significant.</div><div class="NLM_p last">HCE cells were seeded on fibronectin/gelatin-coated 96-well plates and cultured in a culture medium for 24 h. The medium was then replaced with a culture medium containing or lacking (controls) the test compounds, the positive controls (benzalkonium chloride and H<sub>2</sub>O<sub>2</sub>), or the solvent control (DMSO), and the cells were incubated for 2 h. All experiments were performed in triplicate (<i>n</i> = 3), except for the controls (<i>n</i> = 6). After treatment, the cells were incubated with methylthialazole tetrazolium (MTT), which was reduced to blue formazan crystals by the mitochondrial enzyme succinate dehydrogenase. This transformation is proportional to the enzyme activity. After cell dissociation and formazan crystal solubilization using DMSO, the optical density of the extracts at 540 nm, proportional to the number of live cells and their metabolic activity, was recorded with a microplate reader (VERSAmax, Molecular Devices). The absolute values of absorbance for each treatment were then normalized with respect to the absorbance of the vehicle control and expressed as a percentage.</div></div><div id="sec4_46_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> hERG Assay</h4><div class="NLM_p last">Compounds were tested on voltage-gated ion channels using the FastPatch (QPatch) technology, as described by Mathes.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i95"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00394" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84730" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84730" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00394" class="ext-link">10.1021/acs.jmedchem.9b00394</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Crystal structure of <i>S. aureus</i> DNA gyrase in complex with <b>ID-130</b>; hERG assay: hERG-CHO automated patch-clamp; synthesis of intermediates with general formulas <b>I</b>–<b>XX</b>; <sup>1</sup>H-NMR and <sup>13</sup>C-NMR spectra of final compounds; HPLC analysis data of final compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and associated biochemical and biological data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">3D PDB file for <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a> and <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">3D PDB file for <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">3D PDB file for <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_005.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf">jm9b00394_si_001.pdf (6.47 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_002.csv">jm9b00394_si_002.csv (5.03 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_003.pdb">jm9b00394_si_003.pdb (1.9 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_004.pdb">jm9b00394_si_004.pdb (1.9 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_005.pdb">jm9b00394_si_005.pdb (1.85 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The PDB code for the structure of <i>S. aureus</i> DNA gyrase in complex with <b>ID-130</b> is 6FM4. Authors will release the atomic coordinate data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00394" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68468" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68468" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rosella Ombrato</span> - <span class="hlFld-Affiliation affiliation">Angelini
RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della
Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8625-599X" title="Orcid link">http://orcid.org/0000-0002-8625-599X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#26544955434a4a4708494b445447524966474841434a4f484f564e47544b470845494b"><span class="__cf_email__" data-cfemail="76041905131a1a1758191b140417021936171811131a1f181f061e17041b175815191b">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gabriele Magarò</span> - <span class="hlFld-Affiliation affiliation">Angelini
RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della
Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Federica Prati</span> - <span class="hlFld-Affiliation affiliation">Angelini
RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della
Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8483-7646" title="Orcid link">http://orcid.org/0000-0002-8483-7646</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Barbara Garofalo</span> - <span class="hlFld-Affiliation affiliation">Angelini
RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della
Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gaia Corso</span> - <span class="hlFld-Affiliation affiliation">Angelini
RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della
Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guido Furlotti</span> - <span class="hlFld-Affiliation affiliation">Angelini
RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della
Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6715-9919" title="Orcid link">http://orcid.org/0000-0002-6715-9919</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Claudia Apicella</span> - <span class="hlFld-Affiliation affiliation">Angelini
RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della
Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giorgina Mangano</span> - <span class="hlFld-Affiliation affiliation">Angelini
RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della
Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Noemi D’Atanasio</span> - <span class="hlFld-Affiliation affiliation">Angelini
RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della
Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel Robinson</span> - <span class="hlFld-Affiliation affiliation">Schrodinger, 120 West 45th Street, New York, New York 10036, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Francesco Paolo Di Giorgio</span> - <span class="hlFld-Affiliation affiliation">Angelini
RR&D (Research, Regulatory & Development), Angelini S.p.A., Piazzale della
Stazione SNC, S. Palomba-Pomezia, Rome 00071, Italy</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i98">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13240" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13240" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We acknowledge Benedetta Marchetti for aiding with data collection and synthetic procedures, Rosa Buonfiglio for contributing to the revision of the manuscript, and Daniele Frollano for providing analytical data.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">CLSI</td><td class="NLM_def"><p class="first last">Clinical and Laboratory Standards Institute</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term"><i>E. coli</i></td><td class="NLM_def"><p class="first last"><i>Escherichia coli</i></p></td></tr><tr><td class="NLM_term"><i>E. faecalis</i></td><td class="NLM_def"><p class="first last"><i>Enterococcus faecalis</i></p></td></tr><tr><td class="NLM_term"><i>E. faecium</i></td><td class="NLM_def"><p class="first last"><i>Enterococcus faecium</i></p></td></tr><tr><td class="NLM_term">EF</td><td class="NLM_def"><p class="first last">enrichment factor</p></td></tr><tr><td class="NLM_term">ESKAPE</td><td class="NLM_def"><p class="first last"><i>Enterococcus faecium</i>, <i>Staphylococcus aureus</i>, <i>Klebsiella pneumoniae</i>, <i>Acinetobacter baumannii</i>, <i>Pseudomonas aeruginosa</i>, <i>Enterobacter</i> spp.</p></td></tr><tr><td class="NLM_term">GPU</td><td class="NLM_def"><p class="first last">graphics processing unit</p></td></tr><tr><td class="NLM_term"><i>H. influenzae</i></td><td class="NLM_def"><p class="first last"><i>Haemophilus influenzae</i></p></td></tr><tr><td class="NLM_term">HaCaT</td><td class="NLM_def"><p class="first last">human keratinocytes</p></td></tr><tr><td class="NLM_term">HCE-2</td><td class="NLM_def"><p class="first last">human corneal epithelial</p></td></tr><tr><td class="NLM_term">HEK293</td><td class="NLM_def"><p class="first last">human embryonic kidney 293</p></td></tr><tr><td class="NLM_term">HepG2</td><td class="NLM_def"><p class="first last">human liver hepatocellular carcinoma</p></td></tr><tr><td class="NLM_term">HT-29</td><td class="NLM_def"><p class="first last">human colorectal adenocarcinoma</p></td></tr><tr><td class="NLM_term"><i>K. pneumoniae</i></td><td class="NLM_def"><p class="first last"><i>Klebsiella pneumoniae</i></p></td></tr><tr><td class="NLM_term">LBVS</td><td class="NLM_def"><p class="first last">ligand-based virtual screening</p></td></tr><tr><td class="NLM_term">LHS</td><td class="NLM_def"><p class="first last">left-hand side</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">molecular dynamics</p></td></tr><tr><td class="NLM_term">MIC</td><td class="NLM_def"><p class="first last">minimal inhibitory concentration</p></td></tr><tr><td class="NLM_term">MRSA</td><td class="NLM_def"><p class="first last">methicillin-resistant <i>Staphylococcus aureus</i></p></td></tr><tr><td class="NLM_term">MTT</td><td class="NLM_def"><p class="first last">methylthiazole tetrazolium</p></td></tr><tr><td class="NLM_term"><i>N. gonorrhoeae</i></td><td class="NLM_def"><p class="first last"><i>Neisseria gonorrhoeae</i></p></td></tr><tr><td class="NLM_term">NBTI</td><td class="NLM_def"><p class="first last">novel bacterial topoisomerase inhibitor</p></td></tr><tr><td class="NLM_term">NPT</td><td class="NLM_def"><p class="first last">constant-pressure ensemble</p></td></tr><tr><td class="NLM_term">NVT</td><td class="NLM_def"><p class="first last">constant-volume ensemble</p></td></tr><tr><td class="NLM_term">OPLS</td><td class="NLM_def"><p class="first last">optimized potential for liquid simulations</p></td></tr><tr><td class="NLM_term"><i>P. aeruginosa</i></td><td class="NLM_def"><p class="first last"><i>Pseudomonas aeruginosa</i></p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan-assay interference compounds</p></td></tr><tr><td class="NLM_term">PPB</td><td class="NLM_def"><p class="first last">plasma protein binding</p></td></tr><tr><td class="NLM_term">REOS</td><td class="NLM_def"><p class="first last">rapid elimination of swill</p></td></tr><tr><td class="NLM_term">RHS</td><td class="NLM_def"><p class="first last">right-hand side</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">RMSF</td><td class="NLM_def"><p class="first last">root-mean-square fluctuation</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term"><i>S. aureus</i></td><td class="NLM_def"><p class="first last"><i>Staphylococcus aureus</i></p></td></tr><tr><td class="NLM_term">SB</td><td class="NLM_def"><p class="first last">structure-based</p></td></tr><tr><td class="NLM_term">SBVS</td><td class="NLM_def"><p class="first last">structure-based virtual screening</p></td></tr><tr><td class="NLM_term">SI</td><td class="NLM_def"><p class="first last">selectivity index</p></td></tr><tr><td class="NLM_term">SP</td><td class="NLM_def"><p class="first last">standard precision</p></td></tr><tr><td class="NLM_term">SPC</td><td class="NLM_def"><p class="first last">simple point charge</p></td></tr><tr><td class="NLM_term"><i>S. pneumoniae</i></td><td class="NLM_def"><p class="first last"><i>Streptococcus pneumoniae</i></p></td></tr><tr><td class="NLM_term">SSRF</td><td class="NLM_def"><p class="first last">Shanghai Synchrotron Radiation Facility</p></td></tr><tr><td class="NLM_term">Van-A</td><td class="NLM_def"><p class="first last">bacterial strain resistant to vancomycin and teicoplanin</p></td></tr><tr><td class="NLM_term">Van-B</td><td class="NLM_def"><p class="first last">bacterial strain resistant to vancomycin but sensitive to teicoplanin</p></td></tr><tr><td class="NLM_term">Van-S</td><td class="NLM_def"><p class="first last">vancomycin-sensitive strain</p></td></tr><tr><td class="NLM_term">VRE</td><td class="NLM_def"><p class="first last">vancomycin-resistant enterococci</p></td></tr><tr><td class="NLM_term">XP</td><td class="NLM_def"><p class="first last">extra precision</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i100">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86870" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86870" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 65 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">WHO</span>.  <i>Global Priority List of Antibiotic-Resistant Bacteria to
Guide Research, Discovery, and Development of New Antibiotics</i>. <span class="NLM_publisher-name">WHO</span>: <span class="NLM_publisher-loc">Geneva, Switzerland</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=WHO&title=Global+Priority+List+of+Antibiotic-Resistant+Bacteria+to%0AGuide+Research%2C+Discovery%2C+and+Development+of+New+Antibiotics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DGlobal%2520Priority%2520List%2520of%2520Antibiotic-Resistant%2520Bacteria%2520to%250AGuide%2520Research%252C%2520Discovery%252C%2520and%2520Development%2520of%2520New%2520Antibiotics%26pub%3DWHO%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhanel, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeCorby, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvey, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCracken, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagacé-Wiens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nichol, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wierzbowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baudry, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tailor, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlowsky, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walkty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweizer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoban, D. J.</span></span>; <span class="NLM_contrib-group">Canadian Antimicrobial Resistance Alliance</span> <span> </span><span class="NLM_article-title">Prevalence of Antimicrobial-Resistant Pathogens in Canadian Hospitals: Results of the Canadian Ward Surveillance Study (Canward 2008)</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">4684</span>– <span class="NLM_lpage">4693</span>, <span class="refDoi"> DOI: 10.1128/AAC.00469-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.00469-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=20805395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVCitrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2010&pages=4684-4693&author=G.+G.+Zhanelauthor=M.+DeCorbyauthor=H.+Adamauthor=M.+R.+Mulveyauthor=M.+McCrackenauthor=P.+Lagac%C3%A9-Wiensauthor=K.+A.+Nicholauthor=A.+Wierzbowskiauthor=P.+J.+Baudryauthor=F.+Tailorauthor=J.+A.+Karlowskyauthor=A.+Walktyauthor=F.+Schweizerauthor=J.+Johnsonauthor=D.+J.+Hobanauthor=Canadian+Antimicrobial+Resistance+Alliance&title=Prevalence+of+Antimicrobial-Resistant+Pathogens+in+Canadian+Hospitals%3A+Results+of+the+Canadian+Ward+Surveillance+Study+%28Canward+2008%29&doi=10.1128%2FAAC.00469-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence of antimicrobial-resistant pathogens in Canadian hospitals: results of the Canadian ward surveillance study (CANWARD 2008)</span></div><div class="casAuthors">Zhanel, George G.; DeCorby, Melanie; Adam, Heather; Mulvey, Michael R.; McCracken, Melissa; Lagace-Wiens, Philippe; Nichol, Kimberly A.; Wierzbowski, Aleksandra; Baudry, Patricia J.; Tailor, Franil; Karlowsky, James A.; Walkty, Andrew; Schweizer, Frank; Johnson, Jack; Hoban, Daryl J.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4684-4693</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A total of 5,282 bacterial isolates obtained between 1 Jan. and 31 Dec. 31 2008, inclusive, from patients in 10 hospitals across Canada as part of the Canadian Ward Surveillance Study (CANWARD 2008) underwent susceptibility testing.  The 10 most common organisms, representing 78.8% of all clin. specimens, were as follows: Escherichia coli (21.4%), methicillin-susceptible Staphylococcus aureus (MSSA; 13.9%), Streptococcus pneumoniae (10.3%), Pseudomonas aeruginosa (7.1%), Klebsiella pneumoniae (6.0%), coagulase-neg. staphylococci/Staphylococcus epidermidis (5.4%), methicillin-resistant S. aureus (MRSA; 5.1%), Haemophilus influenzae (4.1%), Enterococcus spp. (3.3%), Enterobacter cloacae (2.2%).  MRSA comprised 27.0% (272/1,007) of all S. aureus isolates (genotypically, 68.8% of MRSA were health care assocd. [HA-MRSA] and 27.6% were community assocd. [CA-MRSA]).  Extended-spectrum β-lactamase (ESBL)-producing E. coli occurred in 4.9% of E. coli isolates.  The CTX-M type was the predominant ESBL, with CTX-M-15 the most prevalent genotype.  MRSA demonstrated no resistance to ceftobiprole, daptomycin, linezolid, telavancin, tigecycline, or vancomycin (0.4% intermediate intermediate resistance).  E. coli demonstrated no resistance to ertapenem, meropenem, or tigecycline.  Resistance rates with P. aeruginosa were as follows: colistin (polymyxin E), 0.8%; amikacin, 3.5%; cefepime, 7.2%; gentamicin, 12.3%; fluoroquinolones, 19.0 to 24.1%; meropenem, 5.6%; piperacillin-tazobactam, 8.0%.  A multidrug-resistant (MDR) phenotype occurred frequently in P. aeruginosa (5.9%) but uncommonly in E. coli (1.2%) and K. pneumoniae (0.9%).  In conclusion, E. coli, S. aureus (MSSA and MRSA), P. aeruginosa, S. pneumoniae, K. pneumoniae, H. influenzae, and Enterococcus spp. are the most common isolates recovered from clin. specimens in Canadian hospitals.  The prevalence of MRSA was 27.0% (of which genotypically 27.6% were CA-MRSA), while ESBL-producing E. coli occurred in 4.9% of isolates.  An MDR phenotype was common in P. aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzoXX--1oar7Vg90H21EOLACvtfcHk0ljkIvWeRR4g9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVCitrzF&md5=2cfe5fe769b68c8b375bf07a4a26a2d8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1128%2FAAC.00469-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00469-10%26sid%3Dliteratum%253Aachs%26aulast%3DZhanel%26aufirst%3DG.%2BG.%26aulast%3DDeCorby%26aufirst%3DM.%26aulast%3DAdam%26aufirst%3DH.%26aulast%3DMulvey%26aufirst%3DM.%2BR.%26aulast%3DMcCracken%26aufirst%3DM.%26aulast%3DLagac%25C3%25A9-Wiens%26aufirst%3DP.%26aulast%3DNichol%26aufirst%3DK.%2BA.%26aulast%3DWierzbowski%26aufirst%3DA.%26aulast%3DBaudry%26aufirst%3DP.%2BJ.%26aulast%3DTailor%26aufirst%3DF.%26aulast%3DKarlowsky%26aufirst%3DJ.%2BA.%26aulast%3DWalkty%26aufirst%3DA.%26aulast%3DSchweizer%26aufirst%3DF.%26aulast%3DJohnson%26aufirst%3DJ.%26aulast%3DHoban%26aufirst%3DD.%2BJ.%26aulast%3D%26atitle%3DPrevalence%2520of%2520Antimicrobial-Resistant%2520Pathogens%2520in%2520Canadian%2520Hospitals%253A%2520Results%2520of%2520the%2520Canadian%2520Ward%2520Surveillance%2520Study%2520%2528Canward%25202008%2529%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2010%26volume%3D54%26spage%3D4684%26epage%3D4693%26doi%3D10.1128%2FAAC.00469-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laxminarayan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wattal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaidi, A. K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertheim, H. F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sumpradit, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vlieghe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gould, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greko, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">So, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigdeli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodhouse, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ombaka, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peralta, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qamar, F. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mir, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kariuki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhutta, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coates, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergstrom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cars, O.</span></span> <span> </span><span class="NLM_article-title">Antibiotic Resistance—The Need for Global Solutions</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1057</span>– <span class="NLM_lpage">1098</span>, <span class="refDoi"> DOI: 10.1016/S1473-3099(13)70318-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2FS1473-3099%2813%2970318-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=24252483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A280%3ADC%252BC2c7os1Gntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=1057-1098&author=R.+Laxminarayanauthor=A.+Duseauthor=C.+Wattalauthor=A.+K.+M.+Zaidiauthor=H.+F.+L.+Wertheimauthor=N.+Sumpraditauthor=E.+Vliegheauthor=G.+L.+Haraauthor=I.+M.+Gouldauthor=H.+Goossensauthor=C.+Grekoauthor=A.+D.+Soauthor=M.+Bigdeliauthor=G.+Tomsonauthor=W.+Woodhouseauthor=E.+Ombakaauthor=A.+Q.+Peraltaauthor=F.+N.+Qamarauthor=F.+Mirauthor=S.+Kariukiauthor=Z.+A.+Bhuttaauthor=A.+Coatesauthor=R.+Bergstromauthor=G.+D.+Wrightauthor=E.+D.+Brownauthor=O.+Cars&title=Antibiotic+Resistance%E2%80%94The+Need+for+Global+Solutions&doi=10.1016%2FS1473-3099%2813%2970318-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic resistance-the need for global solutions</span></div><div class="casAuthors">Laxminarayan Ramanan; Duse Adriano; Wattal Chand; Zaidi Anita K M; Wertheim Heiman F L; Sumpradit Nithima; Vlieghe Erika; Hara Gabriel Levy; Gould Ian M; Goossens Herman; Greko Christina; So Anthony D; Bigdeli Maryam; Tomson Goran; Woodhouse Will; Ombaka Eva; Peralta Arturo Quizhpe; Qamar Farah Naz; Mir Fatima; Kariuki Sam; Bhutta Zulfiqar A; Coates Anthony; Bergstrom Richard; Wright Gerard D; Brown Eric D; Cars Otto</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Infectious diseases</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1057-98</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The causes of antibiotic resistance are complex and include human behaviour at many levels of society; the consequences affect everybody in the world.  Similarities with climate change are evident.  Many efforts have been made to describe the many different facets of antibiotic resistance and the interventions needed to meet the challenge.  However, coordinated action is largely absent, especially at the political level, both nationally and internationally.  Antibiotics paved the way for unprecedented medical and societal developments, and are today indispensible in all health systems.  Achievements in modern medicine, such as major surgery, organ transplantation, treatment of preterm babies, and cancer chemotherapy, which we today take for granted, would not be possible without access to effective treatment for bacterial infections.  Within just a few years, we might be faced with dire setbacks, medically, socially, and economically, unless real and unprecedented global coordinated actions are immediately taken.  Here, we describe the global situation of antibiotic resistance, its major causes and consequences, and identify key areas in which action is urgently needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRve4U0u-sYhWx8TEN9K18QfW6udTcc2eZIno8xpjUQ_bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c7os1Gntg%253D%253D&md5=dc5cd314f3d81483c4b767f090af1482</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2813%2970318-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252813%252970318-9%26sid%3Dliteratum%253Aachs%26aulast%3DLaxminarayan%26aufirst%3DR.%26aulast%3DDuse%26aufirst%3DA.%26aulast%3DWattal%26aufirst%3DC.%26aulast%3DZaidi%26aufirst%3DA.%2BK.%2BM.%26aulast%3DWertheim%26aufirst%3DH.%2BF.%2BL.%26aulast%3DSumpradit%26aufirst%3DN.%26aulast%3DVlieghe%26aufirst%3DE.%26aulast%3DHara%26aufirst%3DG.%2BL.%26aulast%3DGould%26aufirst%3DI.%2BM.%26aulast%3DGoossens%26aufirst%3DH.%26aulast%3DGreko%26aufirst%3DC.%26aulast%3DSo%26aufirst%3DA.%2BD.%26aulast%3DBigdeli%26aufirst%3DM.%26aulast%3DTomson%26aufirst%3DG.%26aulast%3DWoodhouse%26aufirst%3DW.%26aulast%3DOmbaka%26aufirst%3DE.%26aulast%3DPeralta%26aufirst%3DA.%2BQ.%26aulast%3DQamar%26aufirst%3DF.%2BN.%26aulast%3DMir%26aufirst%3DF.%26aulast%3DKariuki%26aufirst%3DS.%26aulast%3DBhutta%26aufirst%3DZ.%2BA.%26aulast%3DCoates%26aufirst%3DA.%26aulast%3DBergstrom%26aufirst%3DR.%26aulast%3DWright%26aufirst%3DG.%2BD.%26aulast%3DBrown%26aufirst%3DE.%2BD.%26aulast%3DCars%26aufirst%3DO.%26atitle%3DAntibiotic%2520Resistance%25E2%2580%2594The%2520Need%2520for%2520Global%2520Solutions%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2013%26volume%3D13%26spage%3D1057%26epage%3D1098%26doi%3D10.1016%2FS1473-3099%2813%2970318-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laxminarayan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsoso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brower, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Røttingen, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klugman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, S.</span></span> <span> </span><span class="NLM_article-title">Access to Effective Antimicrobials: A Worldwide Challenge</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">168</span>– <span class="NLM_lpage">175</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(15)00474-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2FS0140-6736%2815%2900474-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=26603918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A280%3ADC%252BC28vltVGrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=2016&pages=168-175&author=R.+Laxminarayanauthor=P.+Matsosoauthor=S.+Pantauthor=C.+Browerauthor=J.+A.+R%C3%B8ttingenauthor=K.+Klugmanauthor=S.+Davies&title=Access+to+Effective+Antimicrobials%3A+A+Worldwide+Challenge&doi=10.1016%2FS0140-6736%2815%2900474-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Access to effective antimicrobials: a worldwide challenge</span></div><div class="casAuthors">Laxminarayan Ramanan; Matsoso Precious; Pant Suraj; Brower Charles; Rottingen John-Arne; Klugman Keith; Davies Sally</div><div class="citationInfo"><span class="NLM_cas:title">Lancet (London, England)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">387</span>
        (<span class="NLM_cas:issue">10014</span>),
    <span class="NLM_cas:pages">168-75</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Recent years have seen substantial improvements in life expectancy and access to antimicrobials, especially in low-income and lower-middle-income countries, but increasing pathogen resistance to antimicrobials threatens to roll back this progress.  Resistant organisms in health-care and community settings pose a threat to survival rates from serious infections, including neonatal sepsis and health-care-associated infections, and limit the potential health benefits from surgeries, transplants, and cancer treatment.  The challenge of simultaneously expanding appropriate access to antimicrobials, while restricting inappropriate access, particularly to expensive, newer generation antimicrobials, is unique in global health and requires new approaches to financing and delivering health care and a one-health perspective on the connections between pathogen transmission in animals and humans.  Here, we describe the importance of effective antimicrobials.  We assess the disease burden caused by limited access to antimicrobials, attributable to resistance to antimicrobials, and the potential effect of vaccines in restricting the need for antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSzqsAQa5w3CcULxIEhxUjPfW6udTcc2eYwZ0c-pyQWc7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28vltVGrug%253D%253D&md5=ff397045340f81521e75bfee7f933244</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2815%2900474-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252815%252900474-2%26sid%3Dliteratum%253Aachs%26aulast%3DLaxminarayan%26aufirst%3DR.%26aulast%3DMatsoso%26aufirst%3DP.%26aulast%3DPant%26aufirst%3DS.%26aulast%3DBrower%26aufirst%3DC.%26aulast%3DR%25C3%25B8ttingen%26aufirst%3DJ.%2BA.%26aulast%3DKlugman%26aufirst%3DK.%26aulast%3DDavies%26aufirst%3DS.%26atitle%3DAccess%2520to%2520Effective%2520Antimicrobials%253A%2520A%2520Worldwide%2520Challenge%26jtitle%3DLancet%26date%3D2016%26volume%3D387%26spage%3D168%26epage%3D175%26doi%3D10.1016%2FS0140-6736%2815%2900474-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huttner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbarth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosgrove, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarlier, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittet, D.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial Resistance: A Global View from the 2013 World Healthcare-Associated Infections Forum</span>. <i>Antimicrob. Resist. Infect. Control</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">31</span>, <span class="refDoi"> DOI: 10.1186/2047-2994-2-31</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1186%2F2047-2994-2-31" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=24237856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A280%3ADC%252BC2c7ns1Chug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=31&author=A.+Huttnerauthor=S.+Harbarthauthor=J.+Carletauthor=S.+Cosgroveauthor=H.+Goossensauthor=A.+Holmesauthor=V.+Jarlierauthor=A.+Vossauthor=D.+Pittet&title=Antimicrobial+Resistance%3A+A+Global+View+from+the+2013+World+Healthcare-Associated+Infections+Forum&doi=10.1186%2F2047-2994-2-31"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial resistance: a global view from the 2013 World Healthcare-Associated Infections Forum</span></div><div class="casAuthors">Huttner Angela; Harbarth Stephan; Pittet Didier; Carlet Jean; Cosgrove Sara; Goossens Herman; Holmes Alison; Jarlier Vincent; Voss Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial resistance and infection control</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">31</span>
        ISSN:<span class="NLM_cas:issn">2047-2994</span>.
    </div><div class="casAbstract">Antimicrobial resistance (AMR) is now a global threat.  Its emergence rests on antimicrobial overuse in humans and food-producing animals; globalization and suboptimal infection control facilitate its spread.  While aggressive measures in some countries have led to the containment of some resistant gram-positive organisms, extensively resistant gram-negative organisms such as carbapenem-resistant enterobacteriaceae and pan-resistant Acinetobacter spp. continue their rapid spread.  Antimicrobial conservation/stewardship programs have seen some measure of success in reducing antimicrobial overuse in humans, but their reach is limited to acute-care settings in high-income countries.  Outside the European Union, there is scant or no oversight of antimicrobial administration to food-producing animals, while evidence mounts that this administration leads directly to resistant human infections.  Both horizontal and vertical infection control measures can interrupt transmission among humans, but many of these are costly and essentially limited to high-income countries as well.  Novel antimicrobials are urgently needed; in recent decades pharmaceutical companies have largely abandoned antimicrobial discovery and development given their high costs and low yield.  Against this backdrop, international and cross-disciplinary collaboration appears to be taking root in earnest, although specific strategies still need defining.  Educational programs targeting both antimicrobial prescribers and consumers must be further developed and supported.  The general public must continue to be made aware of the current scale of AMR's threat, and must perceive antimicrobials as they are: a non-renewable and endangered resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ0kHV8NqxvhiO2kE7krGOBfW6udTcc2eYwZ0c-pyQWc7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c7ns1Chug%253D%253D&md5=2c045c0cd094702cea5b5ed5c2012f61</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1186%2F2047-2994-2-31&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F2047-2994-2-31%26sid%3Dliteratum%253Aachs%26aulast%3DHuttner%26aufirst%3DA.%26aulast%3DHarbarth%26aufirst%3DS.%26aulast%3DCarlet%26aufirst%3DJ.%26aulast%3DCosgrove%26aufirst%3DS.%26aulast%3DGoossens%26aufirst%3DH.%26aulast%3DHolmes%26aufirst%3DA.%26aulast%3DJarlier%26aufirst%3DV.%26aulast%3DVoss%26aufirst%3DA.%26aulast%3DPittet%26aufirst%3DD.%26atitle%3DAntimicrobial%2520Resistance%253A%2520A%2520Global%2520View%2520from%2520the%25202013%2520World%2520Healthcare-Associated%2520Infections%2520Forum%26jtitle%3DAntimicrob.%2520Resist.%2520Infect.%2520Control%26date%3D2013%26volume%3D2%26spage%3D31%26doi%3D10.1186%2F2047-2994-2-31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">East, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span> <span> </span><span class="NLM_article-title">Multitarget Ligands in Antibacterial Research: Progress and Opportunities</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">156</span>, <span class="refDoi"> DOI: 10.1517/17460441.2013.743991</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1517%2F17460441.2013.743991" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=23252414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Cku74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=143-156&author=S.+P.+Eastauthor=L.+L.+Silver&title=Multitarget+Ligands+in+Antibacterial+Research%3A+Progress+and+Opportunities&doi=10.1517%2F17460441.2013.743991"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Multitarget ligands in antibacterial research: progress and opportunities</span></div><div class="casAuthors">East, Stephen P.; Silver, Lynn L.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">143-156</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: Resistance to current antibacterial therapies is an inevitability that represents a significant global health concern.  Bacteria have the capacity to render all current drug treatments ineffective, which places a demand on the drug discovery community to constantly develop new antibacterial agents.  Compds. that inhibit multiple biol. targets, often referred to as multitarget ligands, are an inviting prospect in antibacterial research because, although they will not solve the issue of resistance, they might help to delay the onset. Areas covered: This review covers some of the recent progress in identifying new ligands that deliberately interact with more than one essential biol. target in bacteria.  The two principal areas covered are inhibitors of DNA replication and cell wall biosynthesis. Expert opinion: Antibacterial programs for the design of multitarget ligands present an important opportunity for prodn. of antibacterial agents.  Their longevity, due to slow development of resistance, is comparable to that seen with other successful agents - but is much improved over single-targeted agents for which resistance can appear in vitro overnight.  The preclin. development of these agents will have to overcome the std. problems of antibacterial discovery.  Such problems include optimization of characteristics favoring cell entry and particularly the demonstration of selectivity of inhibition of the desired multiple targets without inhibition of other bacterial or any mammalian functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbhpbYgIA8HrVg90H21EOLACvtfcHk0liuhwDkjrCvDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Cku74%253D&md5=2b335d54e579d210ab9a1220ea2d6a57</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1517%2F17460441.2013.743991&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2013.743991%26sid%3Dliteratum%253Aachs%26aulast%3DEast%26aufirst%3DS.%2BP.%26aulast%3DSilver%26aufirst%3DL.%2BL.%26atitle%3DMultitarget%2520Ligands%2520in%2520Antibacterial%2520Research%253A%2520Progress%2520and%2520Opportunities%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2013%26volume%3D8%26spage%3D143%26epage%3D156%26doi%3D10.1517%2F17460441.2013.743991" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gellert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizuuchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Dea, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, H. A.</span></span> <span> </span><span class="NLM_article-title">DNA Gyrase: An Enzyme That Introduces Superhelical Turns into DNA</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">3872</span>– <span class="NLM_lpage">3876</span>, <span class="refDoi"> DOI: 10.1073/pnas.73.11.3872</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1073%2Fpnas.73.11.3872" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=186775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADyaE2sXjvFWmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=1976&pages=3872-3876&author=M.+Gellertauthor=K.+Mizuuchiauthor=M.+H.+O%E2%80%99Deaauthor=H.+A.+Nash&title=DNA+Gyrase%3A+An+Enzyme+That+Introduces+Superhelical+Turns+into+DNA&doi=10.1073%2Fpnas.73.11.3872"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">DNA gyrase:  An enzyme that introduces superhelical turns into DNA</span></div><div class="casAuthors">Gellert, Martin; Mizuuchi, Kiyoshi; O'Dea, Mary H.; Nash, Howard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3872-6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Relaxed closed-circular DNA was converted to neg. supercoiled DNA by DNA gyrase.  This enzyme was purified from Escherichia coli cells.  The reaction required ATP and Mg2+ and was stimulated by spermidine.  The enzyme acted equally well on relaxed closed-circular colicin E1, bacteriophage λ, and simian virus 40 DNA.  The final superhelix d. of the DNA could be considerably greater than that found in intracellularly supercoiled DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWknjnsm_VqrVg90H21EOLACvtfcHk0liOB9DAS3hT5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXjvFWmsQ%253D%253D&md5=adc7a9bfe9d48cba5143727e982e7b81</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.73.11.3872&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.73.11.3872%26sid%3Dliteratum%253Aachs%26aulast%3DGellert%26aufirst%3DM.%26aulast%3DMizuuchi%26aufirst%3DK.%26aulast%3DO%25E2%2580%2599Dea%26aufirst%3DM.%2BH.%26aulast%3DNash%26aufirst%3DH.%2BA.%26atitle%3DDNA%2520Gyrase%253A%2520An%2520Enzyme%2520That%2520Introduces%2520Superhelical%2520Turns%2520into%2520DNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1976%26volume%3D73%26spage%3D3872%26epage%3D3876%26doi%3D10.1073%2Fpnas.73.11.3872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kato, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiraga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, H.</span></span> <span> </span><span class="NLM_article-title">New Topoisomerase Essential for Chromosome Segregation in E. Coli</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">393</span>– <span class="NLM_lpage">404</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(90)90172-B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2F0092-8674%2890%2990172-B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=2170028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADyaK3MXhsFOrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1990&pages=393-404&author=J.+Katoauthor=Y.+Nishimuraauthor=R.+Imamuraauthor=H.+Nikiauthor=S.+Hiragaauthor=H.+Suzuki&title=New+Topoisomerase+Essential+for+Chromosome+Segregation+in+E.+Coli&doi=10.1016%2F0092-8674%2890%2990172-B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">New topoisomerase essential for chromosome segregation in E. coli</span></div><div class="casAuthors">Kato, Junichi; Nishimura, Yukinobu; Imamura, Ryu; Niki, Hironori; Hiraga, Sota; Suzuki, Hideho</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">393-404</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">The nucleotide sequence of the parC gene essential for chromosome partition in Escherichia coli was detd.  The deduced amino acid sequence was homologous to that of the A subunit of gyrase.  A new gene encoding a ∼70 kd protein was found.  The gene was sequenced, and the deduced amino acid sequence revealed that the gene product was homologous to the gyrase B subunit.  Mutants of this gene were isolated and showed the typical Par phenotype at nonpermissive temp.; thus, the gene was named parE.  Enhanced relaxation activity of supercoiled plasmid mols. was detected in the combined crude cell lysates prepd. from the ParC and ParE overproducers.  A topA mutation defective in topoisomerase I could be compensated by increasing both the parC and the parE gene dosage.  It is suggested that the parC and parE genes code for the subunits of a new topoisomerase, named topo IV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgpWpdKUf52LVg90H21EOLACvtfcHk0liOB9DAS3hT5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXhsFOrtLY%253D&md5=6a0ec298af3872e372b39f710ec57093</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2890%2990172-B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252890%252990172-B%26sid%3Dliteratum%253Aachs%26aulast%3DKato%26aufirst%3DJ.%26aulast%3DNishimura%26aufirst%3DY.%26aulast%3DImamura%26aufirst%3DR.%26aulast%3DNiki%26aufirst%3DH.%26aulast%3DHiraga%26aufirst%3DS.%26aulast%3DSuzuki%26aufirst%3DH.%26atitle%3DNew%2520Topoisomerase%2520Essential%2520for%2520Chromosome%2520Segregation%2520in%2520E.%2520Coli%26jtitle%3DCell%26date%3D1990%26volume%3D63%26spage%3D393%26epage%3D404%26doi%3D10.1016%2F0092-8674%2890%2990172-B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drlica, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span> <span> </span><span class="NLM_article-title">Quinolone-Mediated Bacterial Death</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">385</span>– <span class="NLM_lpage">392</span>, <span class="refDoi"> DOI: 10.1128/AAC.01617-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.01617-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=17724149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlSqtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2008&pages=385-392&author=K.+Drlicaauthor=M.+Malikauthor=R.+J.+Kernsauthor=X.+Zhao&title=Quinolone-Mediated+Bacterial+Death&doi=10.1128%2FAAC.01617-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Quinolone-mediated bacterial death</span></div><div class="casAuthors">Drlica, Karl; Malik, Muhammad; Kerns, Robert J.; Zhao, Xilin</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">385-392</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  In the present review we consider cell death through a two-part "poison" hypothesis in which quinolones form reversible drug-topoisomerase-DNA complexes that subsequently lead to several types of irreversible (lethal) damage.  Topics discussed include: quinolone-topoisomerase-DNA complexes; inhibition of DNA replication; SOS response and cell filamentation; toxin-antitoxin modules and programmed cell death; chromosome fragmentation; and destabilization of cleaved complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowBOqyJ-TV4bVg90H21EOLACvtfcHk0liOB9DAS3hT5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlSqtLc%253D&md5=eef420d64b9f28336fb0cc5f6c42a803</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FAAC.01617-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01617-06%26sid%3Dliteratum%253Aachs%26aulast%3DDrlica%26aufirst%3DK.%26aulast%3DMalik%26aufirst%3DM.%26aulast%3DKerns%26aufirst%3DR.%2BJ.%26aulast%3DZhao%26aufirst%3DX.%26atitle%3DQuinolone-Mediated%2520Bacterial%2520Death%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2008%26volume%3D52%26spage%3D385%26epage%3D392%26doi%3D10.1128%2FAAC.01617-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Redgrave, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piddock, L. J.</span></span> <span> </span><span class="NLM_article-title">Fluoroquinolone Resistance: Mechanisms, Impact on Bacteria, and Role in Evolutionary Success</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">445</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2014.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.tim.2014.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=24842194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotlOqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=438-445&author=L.+S.+Redgraveauthor=S.+B.+Suttonauthor=M.+A.+Webberauthor=L.+J.+Piddock&title=Fluoroquinolone+Resistance%3A+Mechanisms%2C+Impact+on+Bacteria%2C+and+Role+in+Evolutionary+Success&doi=10.1016%2Fj.tim.2014.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Fluoroquinolone resistance: mechanisms, impact on bacteria, and role in evolutionary success</span></div><div class="casAuthors">Redgrave, Liam S.; Sutton, Sam B.; Webber, Mark A.; Piddock, Laura J. V.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">438-445</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Quinolone and fluoroquinolone antibiotics are potent, broad-spectrum agents commonly used to treat a range of infections.  Resistance to these agents is multifactorial and can be via one or a combination of target-site gene mutations, increased prodn. of multidrug-resistance (MDR) efflux pumps, modifying enzymes, and/or target-protection proteins.  Fluoroquinolone-resistant clin. isolates of bacteria have emerged readily and recent data have shown that resistance to this class of antibiotics can have diverse, species-dependent impacts on host-strain fitness.  Here we outline the impacts of quinolone-resistance mutations in relation to the fitness and evolutionary success of mutant strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJrla4tEwdbrVg90H21EOLACvtfcHk0lh8UZpwY5uW-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotlOqsLc%253D&md5=060fb6501e12ed1243e4f30190cb4917</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2014.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2014.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DRedgrave%26aufirst%3DL.%2BS.%26aulast%3DSutton%26aufirst%3DS.%2BB.%26aulast%3DWebber%26aufirst%3DM.%2BA.%26aulast%3DPiddock%26aufirst%3DL.%2BJ.%26atitle%3DFluoroquinolone%2520Resistance%253A%2520Mechanisms%252C%2520Impact%2520on%2520Bacteria%252C%2520and%2520Role%2520in%2520Evolutionary%2520Success%26jtitle%3DTrends%2520Microbiol.%26date%3D2014%26volume%3D22%26spage%3D438%26epage%3D445%26doi%3D10.1016%2Fj.tim.2014.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Black, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stachyra, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girard, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claudon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruneau, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miossec, C.</span></span> <span> </span><span class="NLM_article-title">Mechanism of Action of the Antibiotic Nxl101, a Novel Nonfluoroquinolone Inhibitor of Bacterial Type Ii Topoisomerases</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3339</span>– <span class="NLM_lpage">3349</span>, <span class="refDoi"> DOI: 10.1128/AAC.00496-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.00496-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=18625781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtF2gt77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2008&pages=3339-3349&author=M.+T.+Blackauthor=T.+Stachyraauthor=D.+Platelauthor=A.+M.+Girardauthor=M.+Claudonauthor=J.+M.+Bruneauauthor=C.+Miossec&title=Mechanism+of+Action+of+the+Antibiotic+Nxl101%2C+a+Novel+Nonfluoroquinolone+Inhibitor+of+Bacterial+Type+Ii+Topoisomerases&doi=10.1128%2FAAC.00496-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of action of the antibiotic NXL101, a novel nonfluoroquinolone inhibitor of bacterial type II topoisomerases</span></div><div class="casAuthors">Black, Michael T.; Stachyra, Therese; Platel, Denis; Girard, Anne-Marie; Claudon, Monique; Bruneau, Jean-Michel; Miossec, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3339-3349</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">NXL101 is one of a new class of quinoline antibacterial DNA gyrase and topoisomerase IV inhibitors showing potent activity against gram-pos. bacteria, including methicillin- and fluoroquinolone-resistant strains.  NXL101 inhibited topoisomerase IV more effectively than gyrase from Escherichia coli, whereas the converse is true of enzymes from Staphylococcus aureus.  This apparent target preference is opposite to that which is assocd. with most fluoroquinolone antibiotics.  In vitro isolation of S. aureus mutants resistant to NXL101 followed by cloning and sequencing of the genes encoding gyrase and topoisomerase IV led to the identification of several different point mutations within, or close to, the quinolone resistance-detg. region (QRDR) of GyrA.  However, the mutations were not those that are most frequently assocd. with decreased sensitivity to quinolones.  A fluoroquinolone-resistant mutant variant of gyrase generated in vitro was highly resistant to inhibition by the fluoroquinolones ciprofloxacin and moxifloxacin but remained fully susceptible to inhibition by NXL101.  Two mutant gyrases constructed in vitro, with mutations in gyrA engineered according to those most frequently found in S. aureus strains resistant to NXL101, were insensitive to inhibition by NXL101 and had a diminished sensitivity to ciprofloxacin and moxifloxacin.  Certain combinations of mutations giving rise to NXL101 resistance and those giving rise to fluoroquinolone resistance may be mutually exclusive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPRmIzSCazIbVg90H21EOLACvtfcHk0lh8UZpwY5uW-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtF2gt77L&md5=a39fe88e84c958369d86f4614c9f3e06</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2FAAC.00496-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00496-08%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DM.%2BT.%26aulast%3DStachyra%26aufirst%3DT.%26aulast%3DPlatel%26aufirst%3DD.%26aulast%3DGirard%26aufirst%3DA.%2BM.%26aulast%3DClaudon%26aufirst%3DM.%26aulast%3DBruneau%26aufirst%3DJ.%2BM.%26aulast%3DMiossec%26aufirst%3DC.%26atitle%3DMechanism%2520of%2520Action%2520of%2520the%2520Antibiotic%2520Nxl101%252C%2520a%2520Novel%2520Nonfluoroquinolone%2520Inhibitor%2520of%2520Bacterial%2520Type%2520Ii%2520Topoisomerases%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2008%26volume%3D52%26spage%3D3339%26epage%3D3349%26doi%3D10.1128%2FAAC.00496-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Black, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span> <span> </span><span class="NLM_article-title">New Inhibitors of Bacterial Topoisomerase Gyra/Parc Subunits</span>. <i>Curr. Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">804</span>– <span class="NLM_lpage">810</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=19649925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnt1KhsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=804-810&author=M.+T.+Blackauthor=K.+Coleman&title=New+Inhibitors+of+Bacterial+Topoisomerase+Gyra%2FParc+Subunits"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">New inhibitors of bacterial topoisomerase GyrA/ParC subunits</span></div><div class="casAuthors">Black, Michael T.; Coleman, Kenneth</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Investigational Drugs (BioMed Central)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">804-810</span>CODEN:
                <span class="NLM_cas:coden">COIDAZ</span>;
        ISSN:<span class="NLM_cas:issn">2040-3429</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Inhibitors of the GyrA and ParC subunits of the bacterial type IIA topoisomerases DNA gyrase and topoisomerase IV, resp., belong to classes of antibiotics that are among the most successful of the past 50 years, both in a therapeutic and a com. sense.  The initial promise of progressive modification of an antibiotic scaffold to extend the antibacterial spectrum and to modulate pharmacokinetic/pharmacodynamic parameters of existing antibiotics in an advantageous manner has been tempered by the frequent concomitant appearance of serious toxicity.  The merits of the most prominent novel antibiotics that have reached, or are expected to reach the clin. phase of drug development are summarized in this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTnu8CeuJntLVg90H21EOLACvtfcHk0lh8UZpwY5uW-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnt1KhsLg%253D&md5=c1bc05041d4144fda13f691acbef4758</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DM.%2BT.%26aulast%3DColeman%26aufirst%3DK.%26atitle%3DNew%2520Inhibitors%2520of%2520Bacterial%2520Topoisomerase%2520Gyra%252FParc%2520Subunits%26jtitle%3DCurr.%2520Opin.%2520Invest.%2520Drugs%26date%3D2009%26volume%3D10%26spage%3D804%26epage%3D810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ross, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scangarella-Oman, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flamm, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. N.</span></span> <span> </span><span class="NLM_article-title">Determination of Disk Diffusion and Mic Quality Control Guidelines for Gsk2140944, a Novel Bacterial Type Ii Topoisomerase Inhibitor Antimicrobial Agent</span>. <i>J. Clin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2629</span>– <span class="NLM_lpage">2632</span>, <span class="refDoi"> DOI: 10.1128/JCM.00656-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FJCM.00656-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=24759716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1yqu7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2014&pages=2629-2632&author=J.+E.+Rossauthor=N.+E.+Scangarella-Omanauthor=R.+K.+Flammauthor=R.+N.+Jones&title=Determination+of+Disk+Diffusion+and+Mic+Quality+Control+Guidelines+for+Gsk2140944%2C+a+Novel+Bacterial+Type+Ii+Topoisomerase+Inhibitor+Antimicrobial+Agent&doi=10.1128%2FJCM.00656-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of disk diffusion and MIC quality control guidelines for GSK2140944, a novel bacterial type II topoisomerase inhibitor antimicrobial agent</span></div><div class="casAuthors">Ross, James E.; Scangarella-Oman, Nicole E.; Flamm, Robert K.; Jones, Ronald N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2629-2632, 5 pp.</span>CODEN:
                <span class="NLM_cas:coden">JCMIDW</span>;
        ISSN:<span class="NLM_cas:issn">1098-660X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">GSK2140944 is a novel bacterial type II topoisomerase inhibitor in development for the treatment of conventional and biothreat pathogens, including Gram-pos. pathogens and methicillin-resistant Staphylococcus aureus.  This quality control study was performed to establish ranges for selected control strains: S. aureus ATCC 29213 and ATCC 25923, Escherichia coli ATCC 25922, Haemophilus influenzae ATCC 49247, and Streptococcus pneumoniae ATCC 49619.  The control ranges will be crucial for the accurate evaluation of GSK2140944 potency as it progresses through clin. trial development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6LkQjobYlJ7Vg90H21EOLACvtfcHk0lh8UZpwY5uW-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1yqu7rN&md5=b8c019eee78124d5588db240d2659d5a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FJCM.00656-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJCM.00656-14%26sid%3Dliteratum%253Aachs%26aulast%3DRoss%26aufirst%3DJ.%2BE.%26aulast%3DScangarella-Oman%26aufirst%3DN.%2BE.%26aulast%3DFlamm%26aufirst%3DR.%2BK.%26aulast%3DJones%26aufirst%3DR.%2BN.%26atitle%3DDetermination%2520of%2520Disk%2520Diffusion%2520and%2520Mic%2520Quality%2520Control%2520Guidelines%2520for%2520Gsk2140944%252C%2520a%2520Novel%2520Bacterial%2520Type%2520Ii%2520Topoisomerase%2520Inhibitor%2520Antimicrobial%2520Agent%26jtitle%3DJ.%2520Clin.%2520Microbiol.%26date%3D2014%26volume%3D52%26spage%3D2629%26epage%3D2632%26doi%3D10.1128%2FJCM.00656-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sader, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhomberg, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scangarella-Oman, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flamm, R. K.</span></span> <span> </span><span class="NLM_article-title"><i>In Vitro</i> Activity of Gepotidacin (Gsk2140944) against <i>Neisseria Gonorrhoeae</i></span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>, <span class="NLM_elocation-id">e02047</span> <span class="refDoi"> DOI: 10.1128/AAC.02047-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.02047-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=28069643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFektrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&author=D.+J.+Farrellauthor=H.+S.+Saderauthor=P.+R.+Rhombergauthor=N.+E.+Scangarella-Omanauthor=R.+K.+Flamm&title=In+Vitro+Activity+of+Gepotidacin+%28Gsk2140944%29+against+Neisseria+Gonorrhoeae&doi=10.1128%2FAAC.02047-16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro activity of gepotidacin (GSK2140944) against Neisseria gonorrhoeae</span></div><div class="casAuthors">Farrell, D. J.; Sader, H. S.; Rhomberg, P. R.; Scangarella-Oman, N. E.; Flamm, R. K.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e02047-16/1-e02047-16/7</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Gepotidacin (formerly GSK2140944) is a novel, first-in-class, triazaacenaphthylene antibacterial that inhibits bacterial DNA gyrase and topoisomerase IV via a unique mechanism and has demonstrated in vitro activity against Neisseria gonorrhoeae, including drug-resistant strains, and also targets pathogens assocd. with other conventional and biothreat infections.  Broth microdilution was used to evaluate the MIC and min. bactericidal concn. (MBC) activity of gepotidacin and comparators against 25 N. gonorrhoeae strains (including five ciprofloxacin-nonsusceptible strains).  Gepotidacin activity was also evaluated against three N. gonorrhoeae strains (including a ciprofloxacin-nonsusceptible strain) for resistance development, against three N. gonorrhoeae strains (including two tetracycline- and azithromycin-nonsusceptible strains) using time-kill kinetics and checkerboard methods, and against two N. gonorrhoeae strains for the investigation of postantibiotic (PAE) and subinhibitory (PAE-SME) effects.  The MIC50 and MIC90 for gepotidacin against the 25 N. gonorrhoeae isolates tested were 0.12 and 0.25 μg/mL, resp.  The MBC50 and MBC90 for gepotidacin were 0.25 and 0.5 μg/mL, resp.  Gepotidacin was bactericidal, and single-step resistance selection studies did not recover any mutants, indicating a low rate of spontaneous single-step resistance.  For combinations of gepotidacin and comparators tested using checkerboard methods, there were no instances where antagonism occurred and only one instance of synergy (with moxifloxacin; fractional inhibitory concn., 0.375).  This was not confirmed by in vitro time-kill studies.  The PAE for gepotidacin against the wild-type strain ranged from 0.5 to >2.5 h, and the PAE-SME was >2.5 h.  These in vitro data indicate that further study of gepotidacin is warranted for potential use in treating infections caused by N. gonorrhoeae.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO4ROH3tiEkbVg90H21EOLACvtfcHk0lgAt_InrXMTbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFektrbO&md5=289e4d3f32d31105582ac3c4320fd0f3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1128%2FAAC.02047-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02047-16%26sid%3Dliteratum%253Aachs%26aulast%3DFarrell%26aufirst%3DD.%2BJ.%26aulast%3DSader%26aufirst%3DH.%2BS.%26aulast%3DRhomberg%26aufirst%3DP.%2BR.%26aulast%3DScangarella-Oman%26aufirst%3DN.%2BE.%26aulast%3DFlamm%26aufirst%3DR.%2BK.%26atitle%3DIn%2520Vitro%2520Activity%2520of%2520Gepotidacin%2520%2528Gsk2140944%2529%2520against%2520Neisseria%2520Gonorrhoeae%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26doi%3D10.1128%2FAAC.02047-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osheroff, N.</span></span> <span> </span><span class="NLM_article-title">Mechanistic and Structural Basis for the Actions of the Antibacterial Gepotidacin against Staphylococcus aureus Gyrase</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">570</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.8b00315</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.8b00315" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivVOgt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=570&author=E.+G.+Gibsonauthor=B.+Baxauthor=P.+F.+Chanauthor=N.+Osheroff&title=Mechanistic+and+Structural+Basis+for+the+Actions+of+the+Antibacterial+Gepotidacin+against+Staphylococcus+aureus+Gyrase&doi=10.1021%2Facsinfecdis.8b00315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic and Structural Basis for the Actions of the Antibacterial Gepotidacin against Staphylococcus aureus Gyrase</span></div><div class="casAuthors">Gibson, Elizabeth G.; Bax, Ben; Chan, Pan F.; Osheroff, Neil</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">570-581</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Gepotidacin is a first-in-class triazaacenaphthylene novel bacterial topoisomerase inhibitor (NBTI).  The compd. has successfully completed phase II trials for the treatment of acute bacterial skin/skin structure infections and for the treatment of uncomplicated urogenital gonorrhea.  It also displays robust in vitro activity against a range of wild-type and fluoroquinolone-resistant bacteria.  Due to the clin. promise of gepotidacin, a detailed understanding of its interactions with its antibacterial targets is essential.  Thus, we characterized the mechanism of action of gepotidacin against Staphylococcus aureus gyrase.  Gepotidacin was a potent inhibitor of gyrase-catalyzed DNA supercoiling (IC50 ≈ 0.047 μM) and relaxation of pos. supercoiled substrates (IC50 ≈ 0.6 μM).  Unlike fluoroquinolones, which induce primarily double-stranded DNA breaks, gepotidacin induced high levels of gyrase-mediated single-stranded breaks.  No double-stranded breaks were obsd. even at high gepotidacin concn., long cleavage times, or in the presence of ATP.  Moreover, gepotidacin suppressed the formation of double-stranded breaks.  Gepotidacin formed gyrase-DNA cleavage complexes that were stable for >4 h.  In vitro competition suggests that gyrase binding by gepotidacin and fluoroquinolones are mutually exclusive.  Finally, we detd. crystal structures of gepotidacin with the S. aureus gyrase core fusion truncate with nicked (2.31 Å resoln.) or intact (uncleaved) DNA (2.37 Å resoln.).  In both cases, a single gepotidacin mol. was bound midway between the two scissile DNA bonds and in a pocket between the two GyrA subunits.  A comparison of the two structures demonstrates conformational flexibility within the central linker of gepotidacin, which may contribute to the activity of the compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZKsv0roHKe7Vg90H21EOLACvtfcHk0lgAt_InrXMTbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivVOgt7s%253D&md5=b13ebf930786a12be3d868bf575e6fb3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.8b00315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.8b00315%26sid%3Dliteratum%253Aachs%26aulast%3DGibson%26aufirst%3DE.%2BG.%26aulast%3DBax%26aufirst%3DB.%26aulast%3DChan%26aufirst%3DP.%2BF.%26aulast%3DOsheroff%26aufirst%3DN.%26atitle%3DMechanistic%2520and%2520Structural%2520Basis%2520for%2520the%2520Actions%2520of%2520the%2520Antibacterial%2520Gepotidacin%2520against%2520Staphylococcus%2520aureus%2520Gyrase%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2019%26spage%3D570%26doi%3D10.1021%2Facsinfecdis.8b00315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Riordan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiffany, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scangarella-Oman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hossain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumont, E.</span></span> <span> </span><span class="NLM_article-title">Efficacy, Safety, and Tolerability of Gepotidacin (Gsk2140944) in the Treatment of Patients with Suspected or Confirmed Gram-Positive Acute Bacterial Skin and Skin Structure Infections</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>, <span class="NLM_elocation-id">e02095</span> <span class="refDoi"> DOI: 10.1128/AAC.02095-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.02095-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=28373199" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&author=W.+O%E2%80%99Riordanauthor=C.+Tiffanyauthor=N.+Scangarella-Omanauthor=C.+Perryauthor=M.+Hossainauthor=T.+Ashtonauthor=E.+Dumont&title=Efficacy%2C+Safety%2C+and+Tolerability+of+Gepotidacin+%28Gsk2140944%29+in+the+Treatment+of+Patients+with+Suspected+or+Confirmed+Gram-Positive+Acute+Bacterial+Skin+and+Skin+Structure+Infections&doi=10.1128%2FAAC.02095-16"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1128%2FAAC.02095-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02095-16%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Riordan%26aufirst%3DW.%26aulast%3DTiffany%26aufirst%3DC.%26aulast%3DScangarella-Oman%26aufirst%3DN.%26aulast%3DPerry%26aufirst%3DC.%26aulast%3DHossain%26aufirst%3DM.%26aulast%3DAshton%26aufirst%3DT.%26aulast%3DDumont%26aufirst%3DE.%26atitle%3DEfficacy%252C%2520Safety%252C%2520and%2520Tolerability%2520of%2520Gepotidacin%2520%2528Gsk2140944%2529%2520in%2520the%2520Treatment%2520of%2520Patients%2520with%2520Suspected%2520or%2520Confirmed%2520Gram-Positive%2520Acute%2520Bacterial%2520Skin%2520and%2520Skin%2520Structure%2520Infections%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26doi%3D10.1128%2FAAC.02095-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumont, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raychaudhuri, A.</span></span> <span> </span><span class="NLM_article-title">O05.3 a Phase Ii, Randomised, Study in Adult Subjects Evaluating the Efficacy, Safety, and Tolerability of Single Doses of Gepotidacin (Gsk2140944) for Treatment of Uncomplicated Urogenital Gonorrhoea</span>. <i>Sex. Transm. Infect.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">A11</span>, <span class="refDoi"> DOI: 10.1136/sextrans-2017-053264.26</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1136%2Fsextrans-2017-053264.26" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=27356549" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2017&pages=A11&author=C.+Perryauthor=E.+Dumontauthor=C.+Perryauthor=A.+Raychaudhuri&title=O05.3+a+Phase+Ii%2C+Randomised%2C+Study+in+Adult+Subjects+Evaluating+the+Efficacy%2C+Safety%2C+and+Tolerability+of+Single+Doses+of+Gepotidacin+%28Gsk2140944%29+for+Treatment+of+Uncomplicated+Urogenital+Gonorrhoea&doi=10.1136%2Fsextrans-2017-053264.26"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1136%2Fsextrans-2017-053264.26&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fsextrans-2017-053264.26%26sid%3Dliteratum%253Aachs%26aulast%3DPerry%26aufirst%3DC.%26aulast%3DDumont%26aufirst%3DE.%26aulast%3DPerry%26aufirst%3DC.%26aulast%3DRaychaudhuri%26aufirst%3DA.%26atitle%3DO05.3%2520a%2520Phase%2520Ii%252C%2520Randomised%252C%2520Study%2520in%2520Adult%2520Subjects%2520Evaluating%2520the%2520Efficacy%252C%2520Safety%252C%2520and%2520Tolerability%2520of%2520Single%2520Doses%2520of%2520Gepotidacin%2520%2528Gsk2140944%2529%2520for%2520Treatment%2520of%2520Uncomplicated%2520Urogenital%2520Gonorrhoea%26jtitle%3DSex.%2520Transm.%2520Infect.%26date%3D2017%26volume%3D93%26spage%3DA11%26doi%3D10.1136%2Fsextrans-2017-053264.26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span> <i>Search of: GSK2140944 - List Results - ClinicalTrials.gov</i>. <a href="https://clinicaltrials.gov/ct2/results?cond=&amp;term=GSK2140944&amp;cntry1=&amp;state1=&amp;recrs" class="extLink">https://clinicaltrials.gov/ct2/results?cond=&term=GSK2140944&cntry1=&state1=&recrs</a> (accessed 7 November <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Search+of%3A+GSK2140944+-+List+Results+-+ClinicalTrials.gov.+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fresults%3Fcond%3D%26term%3DGSK2140944%26cntry1%3D%26state1%3D%26recrs+%28accessed+7+November+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DSearch%2520of%253A%2520GSK2140944%2520-%2520List%2520Results%2520-%2520ClinicalTrials.gov%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nayar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reck, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehmann, D. E.</span></span> <span> </span><span class="NLM_article-title">Nbti 5463 Is a Novel Bacterial Type Ii Topoisomerase Inhibitor with Activity against Gram-Negative Bacteria and <i>in Vivo</i> Efficacy</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2657</span>– <span class="NLM_lpage">2664</span>, <span class="refDoi"> DOI: 10.1128/AAC.02778-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.02778-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=24566174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC2cXovFOks7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=2657-2664&author=T.+J.+Doughertyauthor=A.+Nayarauthor=J.+V.+Newmanauthor=S.+Hopkinsauthor=G.+G.+Stoneauthor=M.+Johnstoneauthor=A.+B.+Shapiroauthor=M.+Croninauthor=F.+Reckauthor=D.+E.+Ehmann&title=Nbti+5463+Is+a+Novel+Bacterial+Type+Ii+Topoisomerase+Inhibitor+with+Activity+against+Gram-Negative+Bacteria+and+in+Vivo+Efficacy&doi=10.1128%2FAAC.02778-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">NBTI 5463 is a novel bacterial type II topoisomerase inhibitor with activity against Gram-negative bacteria and in vivo efficacy</span></div><div class="casAuthors">Dougherty, Thomas J.; Nayar, Asha; Newman, Joseph V.; Hopkins, Sussie; Stone, Gregory G.; Johnstone, Michele; Shapiro, Adam B.; Cronin, Mark; Reck, Folkert; Ehmann, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2657-2664, 9 pp.</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The need for new antibiotics that address serious Gram-neg. infections is well recognized.  Our efforts with a series of novel bacterial type II topoisomerase inhibitors (NBTIs) led to the discovery of NBTI 5463, an agent with improved activity over other NBTIs against Gram-neg. bacteria, in particular against Pseudomonas aeruginosa.  In the present work, NBTI 5463 demonstrated promising activity against a broad range of Gram-neg. pathogens.  In contrast to fluoroquinolones, the compd. did not form a double-strand DNA cleavable complex with Escherichia coli DNA gyrase and DNA, but it was a potent inhibitor of both DNA gyrase and E. coli topoisomerase IV catalytic activities.  In studies with P. aeruginosa, NBTI 5463 was bactericidal.  Resistant mutants arose at a low rate and the mutations were found exclusively in the nfxB gene, a regulator of the MexCD-OprJ efflux system.  Levofloxacin-selected resistance mutations in GyrA did not result in decreased susceptibility to NBTI 5463.  Animal infection studies demonstrated that NBTI 5463 was efficacious in mouse models of lung, thigh and ascending urinary tract infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDlCffNen7R7Vg90H21EOLACvtfcHk0lgwPoWO6s-13A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXovFOks7o%253D&md5=b60c0469b036e6698aaeeed604dd0145</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FAAC.02778-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02778-13%26sid%3Dliteratum%253Aachs%26aulast%3DDougherty%26aufirst%3DT.%2BJ.%26aulast%3DNayar%26aufirst%3DA.%26aulast%3DNewman%26aufirst%3DJ.%2BV.%26aulast%3DHopkins%26aufirst%3DS.%26aulast%3DStone%26aufirst%3DG.%2BG.%26aulast%3DJohnstone%26aufirst%3DM.%26aulast%3DShapiro%26aufirst%3DA.%2BB.%26aulast%3DCronin%26aufirst%3DM.%26aulast%3DReck%26aufirst%3DF.%26aulast%3DEhmann%26aufirst%3DD.%2BE.%26atitle%3DNbti%25205463%2520Is%2520a%2520Novel%2520Bacterial%2520Type%2520Ii%2520Topoisomerase%2520Inhibitor%2520with%2520Activity%2520against%2520Gram-Negative%2520Bacteria%2520and%2520in%2520Vivo%2520Efficacy%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2014%26volume%3D58%26spage%3D2657%26epage%3D2664%26doi%3D10.1128%2FAAC.02778-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubschwerlen, C.</span>; <span class="NLM_string-name">Ritz, D.</span>; <span class="NLM_string-name">Rueedi, G.</span>; <span class="NLM_string-name">Surivet, J.-P.</span>; <span class="NLM_string-name">Acklin, C. Z.</span></span> <span> </span><span class="NLM_article-title">Tricyclic Oxazolidinone Compounds as Antibiotics and Their Preparation and Use in the Treatment of Bacterial Infection</span>. <span class="NLM_patent">WO 2,010,041,194 A1</span>, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=C.+Hubschwerlen&author=D.+Ritz&author=G.+Rueedi&author=J.-P.+Surivet&author=C.+Z.+Acklin&title=Tricyclic+Oxazolidinone+Compounds+as+Antibiotics+and+Their+Preparation+and+Use+in+the+Treatment+of+Bacterial+Infection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHubschwerlen%26aufirst%3DC.%26atitle%3DTricyclic%2520Oxazolidinone%2520Compounds%2520as%2520Antibiotics%2520and%2520Their%2520Preparation%2520and%2520Use%2520in%2520the%2520Treatment%2520of%2520Bacterial%2520Infection%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parr, T.</span>; <span class="NLM_string-name">Hammond, J.</span>; <span class="NLM_string-name">Tateda, K.</span>; <span class="NLM_string-name">Reck, F.</span>; <span class="NLM_string-name">Rueedi, G.</span>; <span class="NLM_string-name">MAsuda, N.</span>; <span class="NLM_string-name">Rahman, k.</span>; <span class="NLM_string-name">Katvars, L.</span>; <span class="NLM_string-name">Sullivan, S.</span></span> In  <i>New Agents Discovery Summary Session: Early New Antimicrobial Agents, ASM-Microbe 2017</i>, New Orleans, June 1–5; <span class="NLM_publisher-name">ASM Microbe</span>;  <span class="NLM_fpage">Sa261</span>– <span class="NLM_lpage">Sa265</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=Sa261-Sa265&author=T.+Parr&author=J.+Hammond&author=K.+Tateda&author=F.+Reck&author=G.+Rueedi&author=N.+MAsuda&author=k.+Rahman&author=L.+Katvars&author=S.+Sullivan&title=New+Agents+Discovery+Summary+Session%3A+Early+New+Antimicrobial+Agents%2C+ASM-Microbe+2017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DParr%26aufirst%3DT.%26btitle%3DNew%2520Agents%2520Discovery%2520Summary%2520Session%253A%2520Early%2520New%2520Antimicrobial%2520Agents%252C%2520ASM-Microbe%25202017%26pub%3DASM%2520Microbe%26spage%3DSa261%26epage%3DSa265%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toussaint, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchiya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlisi, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinke, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miesel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagrutta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishii, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oohata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span> <span> </span><span class="NLM_article-title"><i>In Vitro</i> and <i>in Vivo</i> Characterization of the Novel Oxabicyclooctane-Linked Bacterial Topoisomerase Inhibitor Am-8722, a Selective, Potent Inhibitor of Bacterial DNA Gyrase</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4830</span>– <span class="NLM_lpage">4839</span>, <span class="refDoi"> DOI: 10.1128/AAC.00619-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.00619-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=27246784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFSis7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2016&pages=4830-4839&author=C.+M.+Tanauthor=C.+J.+Gillauthor=J.+Wuauthor=N.+Toussaintauthor=J.+Yinauthor=T.+Tsuchiyaauthor=C.+G.+Garlisiauthor=D.+Kaelinauthor=P.+T.+Meinkeauthor=L.+Mieselauthor=D.+B.+Olsenauthor=A.+Lagruttaauthor=H.+Fukudaauthor=R.+Kishiiauthor=M.+Takeiauthor=K.+Oohataauthor=T.+Takeuchiauthor=T.+Shibueauthor=H.+Takanoauthor=A.+Nishimuraauthor=Y.+Fukudaauthor=S.+B.+Singh&title=In+Vitro+and+in+Vivo+Characterization+of+the+Novel+Oxabicyclooctane-Linked+Bacterial+Topoisomerase+Inhibitor+Am-8722%2C+a+Selective%2C+Potent+Inhibitor+of+Bacterial+DNA+Gyrase&doi=10.1128%2FAAC.00619-16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro and in vivo characterization of the novel oxabicyclooctane-linked bacterial topoisomerase inhibitor AM-8722, a selective, potent inhibitor of bacterial DNA gyrase</span></div><div class="casAuthors">Tan, Christopher M.; Gill, Charles J.; Wu, Jin; Toussaint, Nathalie; Yin, Jingjun; Tsuchiya, Takayuki; Garlisi, Charles G.; Kaelin, David; Meinke, Peter T.; Miesel, Lynn; Olsen, David B.; Lagrutta, Armando; Fukuda, Hideyuki; Kishii, Ryuta; Takei, Masaya; Oohata, Kouhei; Takeuchi, Tomoko; Shibue, Taku; Takano, Hisashi; Nishimura, Akinori; Fukuda, Yasumichi; Singh, Sheo B.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4830-4839</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Oxabicyclooctane-linked novel bacterial topoisomerase inhibitors (NBTIs) represent a new class of recently described antibacterial agents with broad-spectrum activity.  NBTIs dually inhibit the clin. validated bacterial targets DNA gyrase and topoisomerase IV and have been shown to bind distinctly from known classes of antibacterial agents directed against these targets.  Herein we report the mol., cellular, and in vivo characterization of AM-8722 as a representative N-alkylated-1,5-naphthyridone left-hand-side-substituted NBTI.  Consistent with its mode of action, macromol. labeling studies revealed a specific effect of AM-8722 to dose dependently inhibit bacterial DNA synthesis.  AM-8722 displayed greater intrinsic enzymic potency than levofloxacin vs. both DNA gyrase and topoisomerase IV from Staphylococcus aureus and Escherichia coli and displayed selectivity against human topoisomerase II.  AM-8722 was rapidly bactericidal and exhibited whole-cell activity vs. a range of Gram-neg. and Gram-pos. organisms, with no whole-cell potency shift due to the presence of DNA or human serum.  Frequency-of-resistance studies demonstrated an acceptable rate of resistance emergence in vitro at concns. 16- to 32- fold the MIC.  AM-8722 displayed acceptable pharmacokinetic properties and was shown to be efficacious in mouse models of bacterial septicemia.  Overall, AM-8722 is a selective and potent NBTI that displays broad-spectrum antimicrobial activity in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQRBIdegoVg7Vg90H21EOLACvtfcHk0lgj2RxybHH5AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFSis7rP&md5=406686030d49a9ce465d6fcaa25a9770</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1128%2FAAC.00619-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00619-16%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DC.%2BM.%26aulast%3DGill%26aufirst%3DC.%2BJ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DToussaint%26aufirst%3DN.%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DTsuchiya%26aufirst%3DT.%26aulast%3DGarlisi%26aufirst%3DC.%2BG.%26aulast%3DKaelin%26aufirst%3DD.%26aulast%3DMeinke%26aufirst%3DP.%2BT.%26aulast%3DMiesel%26aufirst%3DL.%26aulast%3DOlsen%26aufirst%3DD.%2BB.%26aulast%3DLagrutta%26aufirst%3DA.%26aulast%3DFukuda%26aufirst%3DH.%26aulast%3DKishii%26aufirst%3DR.%26aulast%3DTakei%26aufirst%3DM.%26aulast%3DOohata%26aufirst%3DK.%26aulast%3DTakeuchi%26aufirst%3DT.%26aulast%3DShibue%26aufirst%3DT.%26aulast%3DTakano%26aufirst%3DH.%26aulast%3DNishimura%26aufirst%3DA.%26aulast%3DFukuda%26aufirst%3DY.%26aulast%3DSingh%26aufirst%3DS.%2BB.%26atitle%3DIn%2520Vitro%2520and%2520in%2520Vivo%2520Characterization%2520of%2520the%2520Novel%2520Oxabicyclooctane-Linked%2520Bacterial%2520Topoisomerase%2520Inhibitor%2520Am-8722%252C%2520a%2520Selective%252C%2520Potent%2520Inhibitor%2520of%2520Bacterial%2520DNA%2520Gyrase%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2016%26volume%3D60%26spage%3D4830%26epage%3D4839%26doi%3D10.1128%2FAAC.00619-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Charrier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salisbury, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savage, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaffer-Malam, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ooi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGarry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichowicz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigerson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craighead, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratcliffe, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Best, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, N. R.</span></span> <span> </span><span class="NLM_article-title">Novel Bacterial Topoisomerase Inhibitors with Potent Broad-Spectrum Activity against Drug-Resistant Bacteria</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>, <span class="NLM_elocation-id">e02100</span> <span class="refDoi"> DOI: 10.1128/AAC.02100-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.02100-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=28223393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFeks7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&author=C.+Charrierauthor=A.+M.+Salisburyauthor=V.+J.+Savageauthor=T.+Duffyauthor=E.+Moyoauthor=N.+Chaffer-Malamauthor=N.+Ooiauthor=R.+Newmanauthor=J.+Cheungauthor=R.+Metzgerauthor=D.+McGarryauthor=M.+Pichowiczauthor=R.+Sigersonauthor=I.+R.+Cooperauthor=G.+Nelsonauthor=H.+S.+Butlerauthor=M.+Craigheadauthor=A.+J.+Ratcliffeauthor=S.+A.+Bestauthor=N.+R.+Stokes&title=Novel+Bacterial+Topoisomerase+Inhibitors+with+Potent+Broad-Spectrum+Activity+against+Drug-Resistant+Bacteria&doi=10.1128%2FAAC.02100-16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Novel bacterial topoisomerase inhibitors with potent broad-spectrum activity against drug-resistant bacteria</span></div><div class="casAuthors">Charrier, Cedric; Salisbury, Anne-Marie; Savage, Victoria J.; Duffy, Thomas; Moyo, Emmanuel; Chaffer-Malam, Nathan; Ooi, Nicola; Newman, Rebecca; Cheung, Jonathan; Metzger, Richard; McGarry, David; Pichowicz, Mark; Sigerson, Ralph; Cooper, Ian R.; Nelson, Gary; Butler, Hayley S.; Craighead, Mark; Ratcliffe, Andrew J.; Best, Stuart A.; Stokes, Neil R.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e02100-16/1-e02100-16/13</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The novel bacterial topoisomerase inhibitor class is an investigational type of antibacterial inhibitor of DNA gyrase and topoisomerase IV that does not have cross-resistance with the quinolones.  Here, we report the evaluation of the in vitro properties of a new series of this type of small mol.  Exemplar compds. selectively and potently inhibited the catalytic activities of Escherichia coli DNA gyrase and topoisomerase IV but did not block the DNA breakage-reunion step.  Compds. showed broad-spectrum inhibitory activity against a wide range of Gram-pos. and Gram-neg. pathogens, including biodefence microorganisms and Mycobacterium tuberculosis.  No cross-resistance with fluoroquinolone-resistant Staphylococcus aureus and E. coli isolates was obsd.  Measured MIC90 values were 4 and 8 μg/mL against a panel of contemporary multidrug-resistant isolates of Acinetobacter baumannii and E. coli, resp.  In addn., representative compds. exhibited greater antibacterial potency than the quinolones against obligate anaerobic species.  Spontaneous mutation rates were low, with frequencies of resistance typically <10-8 against E. coli and A. baumannii at concns. equiv. to 4-fold the MIC.  Compd.-resistant E. coli mutants that were isolated following serial passage were characterized by whole-genome sequencing and carried a single Arg38Leu amino acid substitution in the GyrA subunit of DNA gyrase.  Preliminary in vitro safety data indicate that the series shows a promising therapeutic index and potential for low human ether-a-go-go-related gene (hERG) inhibition (50% inhibitory concn. [IC50], >100 μM).  In summary, the compds.' distinct mechanism of action relative to the fluoroquinolones, whole-cell potency, low potential for resistance development, and favorable in vitro safety profile warrant their continued investigation as potential broad-spectrum antibacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT7wMoQx56jLVg90H21EOLACvtfcHk0lgj2RxybHH5AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFeks7jJ&md5=5bb1d8f7b51c869245bfe6b5dd316855</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2FAAC.02100-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02100-16%26sid%3Dliteratum%253Aachs%26aulast%3DCharrier%26aufirst%3DC.%26aulast%3DSalisbury%26aufirst%3DA.%2BM.%26aulast%3DSavage%26aufirst%3DV.%2BJ.%26aulast%3DDuffy%26aufirst%3DT.%26aulast%3DMoyo%26aufirst%3DE.%26aulast%3DChaffer-Malam%26aufirst%3DN.%26aulast%3DOoi%26aufirst%3DN.%26aulast%3DNewman%26aufirst%3DR.%26aulast%3DCheung%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DR.%26aulast%3DMcGarry%26aufirst%3DD.%26aulast%3DPichowicz%26aufirst%3DM.%26aulast%3DSigerson%26aufirst%3DR.%26aulast%3DCooper%26aufirst%3DI.%2BR.%26aulast%3DNelson%26aufirst%3DG.%26aulast%3DButler%26aufirst%3DH.%2BS.%26aulast%3DCraighead%26aufirst%3DM.%26aulast%3DRatcliffe%26aufirst%3DA.%2BJ.%26aulast%3DBest%26aufirst%3DS.%2BA.%26aulast%3DStokes%26aufirst%3DN.%2BR.%26atitle%3DNovel%2520Bacterial%2520Topoisomerase%2520Inhibitors%2520with%2520Potent%2520Broad-Spectrum%2520Activity%2520against%2520Drug-Resistant%2520Bacteria%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26doi%3D10.1128%2FAAC.02100-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitton-Fry, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brickner, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamel, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casavant, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finegan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardink, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huband, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marfat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCurdy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanyam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisialowski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaniewski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zook, C.</span></span> <span> </span><span class="NLM_article-title">Novel 3-Fluoro-6-Methoxyquinoline Derivatives as Inhibitors of Bacterial DNA Gyrase and Topoisomerase Iv</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3353</span>– <span class="NLM_lpage">3358</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2017.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=28610977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVSjtLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=3353-3358&author=M.+J.+Mitton-Fryauthor=S.+J.+Bricknerauthor=J.+C.+Hamelauthor=R.+Barhamauthor=L.+Brennanauthor=J.+M.+Casavantauthor=X.+Dingauthor=S.+Fineganauthor=J.+Hardinkauthor=T.+Hoangauthor=M.+D.+Hubandauthor=M.+Maloneyauthor=A.+Marfatauthor=S.+P.+McCurdyauthor=D.+McLeodauthor=C.+Subramanyamauthor=M.+Plotkinauthor=U.+Reillyauthor=J.+Schaferauthor=G.+G.+Stoneauthor=D.+P.+Uccelloauthor=T.+Wisialowskiauthor=K.+Yoonauthor=R.+Zaniewskiauthor=C.+Zook&title=Novel+3-Fluoro-6-Methoxyquinoline+Derivatives+as+Inhibitors+of+Bacterial+DNA+Gyrase+and+Topoisomerase+Iv&doi=10.1016%2Fj.bmcl.2017.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Novel 3-fluoro-6-methoxyquinoline derivatives as inhibitors of bacterial DNA gyrase and topoisomerase IV</span></div><div class="casAuthors">Mitton-Fry, Mark J.; Brickner, Steven J.; Hamel, Judith C.; Barham, Rose; Brennan, Lori; Casavant, Jeffrey M.; Ding, Xiaoyuan; Finegan, Steven; Hardink, Joel; Hoang, Thuy; Huband, Michael D.; Maloney, Meghan; Marfat, Anthony; McCurdy, Sandra P.; McLeod, Dale; Subramanyam, Chakrapani; Plotkin, Michael; Reilly, Usa; Schafer, John; Stone, Gregory G.; Uccello, Daniel P.; Wisialowski, Todd; Yoon, Kwansik; Zaniewski, Richard; Zook, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3353-3358</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Novel (non-fluoroquinolone) inhibitors of bacterial type II topoisomerases (NBTIs) are an emerging class of antibacterial agents.  The authors report an optimized series of cyclobutylaryl-substituted NBTIs.  Compd. 14 (I) demonstrated excellent activity both in vitro (S. aureus MIC90 = 0.125 μg/mL) and in vivo (systemic and tissue infections).  Enhanced inhibition of Topoisomerase IV correlated with improved activity in S. aureus strains with mutations conferring resistance to NBTIs.  Compd. 14 also displayed an improved hERG IC50 of 85.9 μM and a favorable profile in the anesthetized guinea pig model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXkMNn5y1uArVg90H21EOLACvtfcHk0lgj2RxybHH5AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVSjtLfN&md5=5a675cd783671f0591cb1d394e9f166e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DMitton-Fry%26aufirst%3DM.%2BJ.%26aulast%3DBrickner%26aufirst%3DS.%2BJ.%26aulast%3DHamel%26aufirst%3DJ.%2BC.%26aulast%3DBarham%26aufirst%3DR.%26aulast%3DBrennan%26aufirst%3DL.%26aulast%3DCasavant%26aufirst%3DJ.%2BM.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DFinegan%26aufirst%3DS.%26aulast%3DHardink%26aufirst%3DJ.%26aulast%3DHoang%26aufirst%3DT.%26aulast%3DHuband%26aufirst%3DM.%2BD.%26aulast%3DMaloney%26aufirst%3DM.%26aulast%3DMarfat%26aufirst%3DA.%26aulast%3DMcCurdy%26aufirst%3DS.%2BP.%26aulast%3DMcLeod%26aufirst%3DD.%26aulast%3DSubramanyam%26aufirst%3DC.%26aulast%3DPlotkin%26aufirst%3DM.%26aulast%3DReilly%26aufirst%3DU.%26aulast%3DSchafer%26aufirst%3DJ.%26aulast%3DStone%26aufirst%3DG.%2BG.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DWisialowski%26aufirst%3DT.%26aulast%3DYoon%26aufirst%3DK.%26aulast%3DZaniewski%26aufirst%3DR.%26aulast%3DZook%26aufirst%3DC.%26atitle%3DNovel%25203-Fluoro-6-Methoxyquinoline%2520Derivatives%2520as%2520Inhibitors%2520of%2520Bacterial%2520DNA%2520Gyrase%2520and%2520Topoisomerase%2520Iv%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D3353%26epage%3D3358%26doi%3D10.1016%2Fj.bmcl.2017.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitton-Fry, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brickner, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamel, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casavant, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driscoll, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardink, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huband, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marfat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCurdy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepard, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanyam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaniewski, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zook, C.</span></span> <span> </span><span class="NLM_article-title">Novel Quinoline Derivatives as Inhibitors of Bacterial DNA Gyrase and Topoisomerase Iv</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2955</span>– <span class="NLM_lpage">2961</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.03.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2013.03.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=23566517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlslSqtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=2955-2961&author=M.+J.+Mitton-Fryauthor=S.+J.+Bricknerauthor=J.+C.+Hamelauthor=L.+Brennanauthor=J.+M.+Casavantauthor=M.+Chenauthor=T.+Chenauthor=X.+Dingauthor=J.+Driscollauthor=J.+Hardinkauthor=T.+Hoangauthor=E.+Huaauthor=M.+D.+Hubandauthor=M.+Maloneyauthor=A.+Marfatauthor=S.+P.+McCurdyauthor=D.+McLeodauthor=M.+Plotkinauthor=U.+Reillyauthor=S.+Robinsonauthor=J.+Schaferauthor=R.+M.+Shepardauthor=J.+F.+Smithauthor=G.+G.+Stoneauthor=C.+Subramanyamauthor=K.+Yoonauthor=W.+Yuanauthor=R.+P.+Zaniewskiauthor=C.+Zook&title=Novel+Quinoline+Derivatives+as+Inhibitors+of+Bacterial+DNA+Gyrase+and+Topoisomerase+Iv&doi=10.1016%2Fj.bmcl.2013.03.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Novel quinoline derivatives as inhibitors of bacterial DNA gyrase and topoisomerase IV</span></div><div class="casAuthors">Mitton-Fry, Mark J.; Brickner, Steven J.; Hamel, Judith C.; Brennan, Lori; Casavant, Jeffrey M.; Chen, Michael; Chen, Tao; Ding, Xiaoyuan; Driscoll, James; Hardink, Joel; Hoang, Thuy; Hua, Erbing; Huband, Michael D.; Maloney, Meghan; Marfat, Anthony; McCurdy, Sandra P.; McLeod, Dale; Plotkin, Michael; Reilly, Usa; Robinson, Shaughn; Schafer, John; Shepard, Richard M.; Smith, James F.; Stone, Gregory G.; Subramanyam, Chakrapani; Yoon, Kwansik; Yuan, Wei; Zaniewski, Richard P.; Zook, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2955-2961</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A structurally novel set of inhibitors of bacterial type II topoisomerases with potent in vitro and in vivo antibacterial activity was developed.  Dual-targeting ability, hERG inhibition, and pharmacokinetic properties were also assessed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAbdQVIkCXC7Vg90H21EOLACvtfcHk0lh6aQynixVbew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlslSqtrg%253D&md5=5dea2df13eb5fb785805ea8cba229fc0</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.03.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.03.047%26sid%3Dliteratum%253Aachs%26aulast%3DMitton-Fry%26aufirst%3DM.%2BJ.%26aulast%3DBrickner%26aufirst%3DS.%2BJ.%26aulast%3DHamel%26aufirst%3DJ.%2BC.%26aulast%3DBrennan%26aufirst%3DL.%26aulast%3DCasavant%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DDriscoll%26aufirst%3DJ.%26aulast%3DHardink%26aufirst%3DJ.%26aulast%3DHoang%26aufirst%3DT.%26aulast%3DHua%26aufirst%3DE.%26aulast%3DHuband%26aufirst%3DM.%2BD.%26aulast%3DMaloney%26aufirst%3DM.%26aulast%3DMarfat%26aufirst%3DA.%26aulast%3DMcCurdy%26aufirst%3DS.%2BP.%26aulast%3DMcLeod%26aufirst%3DD.%26aulast%3DPlotkin%26aufirst%3DM.%26aulast%3DReilly%26aufirst%3DU.%26aulast%3DRobinson%26aufirst%3DS.%26aulast%3DSchafer%26aufirst%3DJ.%26aulast%3DShepard%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BF.%26aulast%3DStone%26aufirst%3DG.%2BG.%26aulast%3DSubramanyam%26aufirst%3DC.%26aulast%3DYoon%26aufirst%3DK.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DZaniewski%26aufirst%3DR.%2BP.%26aulast%3DZook%26aufirst%3DC.%26atitle%3DNovel%2520Quinoline%2520Derivatives%2520as%2520Inhibitors%2520of%2520Bacterial%2520DNA%2520Gyrase%2520and%2520Topoisomerase%2520Iv%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D2955%26epage%3D2961%26doi%3D10.1016%2Fj.bmcl.2013.03.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ehmann, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahiri, S. D.</span></span> <span> </span><span class="NLM_article-title">Novel Compounds Targeting Bacterial DNA Topoisomerase/DNA Gyrase</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">83</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2014.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.coph.2014.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=25271174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFylurjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2014&pages=76-83&author=D.+E.+Ehmannauthor=S.+D.+Lahiri&title=Novel+Compounds+Targeting+Bacterial+DNA+Topoisomerase%2FDNA+Gyrase&doi=10.1016%2Fj.coph.2014.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Novel compounds targeting bacterial DNA topoisomerase/DNA gyrase</span></div><div class="casAuthors">Ehmann, David E.; Lahiri, Sushmita D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">76-83</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Among the targets for the development of new antibacterial agents, bacterial topoisomerases remain a vibrant area of discovery.  A structurally diverse set of inhibitors that bind to the ATP (ATP) site of type II topoisomerases have been disclosed recently.  Seven compds. with this mechanism are highlighted, focusing on antibacterial potency and spectrum, as well as examples of in vivo efficacy against pathogens including Staphylococcus aureus and Mycobacterium tuberculosis.  Five compds. from two structural classes are exemplified that are inhibitors that bind to the catalytic site of DNA gyrase and topoisomerase IV.  The pharmacokinetic and pharmacodynamic properties of these mols., derived from in vivo efficacy against Gram-pos. and Gram-neg. pathogens, define the potential for these agents with broad-spectrum and targeted-spectrum clin. utilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtyDfk3On92LVg90H21EOLACvtfcHk0lh6aQynixVbew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFylurjP&md5=d9d7ddb464477c921e073f8076e219e7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2014.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2014.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DEhmann%26aufirst%3DD.%2BE.%26aulast%3DLahiri%26aufirst%3DS.%2BD.%26atitle%3DNovel%2520Compounds%2520Targeting%2520Bacterial%2520DNA%2520Topoisomerase%252FDNA%2520Gyrase%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2014%26volume%3D18%26spage%3D76%26epage%3D83%26doi%3D10.1016%2Fj.coph.2014.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolarič, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minovski, N.</span></span> <span> </span><span class="NLM_article-title">Novel Bacterial Topoisomerase Inhibitors: Challenges and Perspectives in Reducing Herg Toxicity</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2241</span>– <span class="NLM_lpage">2244</span>, <span class="refDoi"> DOI: 10.4155/fmc-2018-0272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.4155%2Ffmc-2018-0272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=30215281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVaqtb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=2241-2244&author=A.+Kolari%C4%8Dauthor=N.+Minovski&title=Novel+Bacterial+Topoisomerase+Inhibitors%3A+Challenges+and+Perspectives+in+Reducing+Herg+Toxicity&doi=10.4155%2Ffmc-2018-0272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Novel bacterial topoisomerase inhibitors: challenges and perspectives in reducing hERG toxicity</span></div><div class="casAuthors">Kolaric, Anja; Minovski, Nikola</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2241-2244</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA1NIj4g1xlrVg90H21EOLACvtfcHk0ljMgbscJiAx8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVaqtb3K&md5=fd3a6e97393867ff99f72f003c0182c1</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2018-0272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2018-0272%26sid%3Dliteratum%253Aachs%26aulast%3DKolari%25C4%258D%26aufirst%3DA.%26aulast%3DMinovski%26aufirst%3DN.%26atitle%3DNovel%2520Bacterial%2520Topoisomerase%2520Inhibitors%253A%2520Challenges%2520and%2520Perspectives%2520in%2520Reducing%2520Herg%2520Toxicity%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2018%26volume%3D10%26spage%3D2241%26epage%3D2244%26doi%3D10.4155%2Ffmc-2018-0272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bax, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggleston, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fosberry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gentry, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorrec, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennessy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hibbs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spitzfaden, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shillings, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theobald, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wohlkonig, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwynn, M. N.</span></span> <span> </span><span class="NLM_article-title">Type Iia Topoisomerase Inhibition by a New Class of Antibacterial Agents</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>466</i></span>,  <span class="NLM_fpage">935</span>– <span class="NLM_lpage">940</span>, <span class="refDoi"> DOI: 10.1038/nature09197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1038%2Fnature09197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=20686482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A280%3ADC%252BC3cjmtFaktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2010&pages=935-940&author=B.+D.+Baxauthor=P.+F.+Chanauthor=D.+S.+Egglestonauthor=A.+Fosberryauthor=D.+R.+Gentryauthor=F.+Gorrecauthor=I.+Giordanoauthor=M.+M.+Hannauthor=A.+Hennessyauthor=M.+Hibbsauthor=J.+Huangauthor=E.+Jonesauthor=J.+Jonesauthor=K.+K.+Brownauthor=C.+J.+Lewisauthor=E.+W.+Mayauthor=M.+R.+Saundersauthor=O.+Singhauthor=C.+E.+Spitzfadenauthor=C.+Shenauthor=A.+Shillingsauthor=A.+J.+Theobaldauthor=A.+Wohlkonigauthor=N.+D.+Pearsonauthor=M.+N.+Gwynn&title=Type+Iia+Topoisomerase+Inhibition+by+a+New+Class+of+Antibacterial+Agents&doi=10.1038%2Fnature09197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Type IIA topoisomerase inhibition by a new class of antibacterial agents</span></div><div class="casAuthors">Bax Benjamin D; Chan Pan F; Eggleston Drake S; Fosberry Andrew; Gentry Daniel R; Gorrec Fabrice; Giordano Ilaria; Hann Michael M; Hennessy Alan; Hibbs Martin; Huang Jianzhong; Jones Emma; Jones Jo; Brown Kristin Koretke; Lewis Ceri J; May Earl W; Saunders Martin R; Singh Onkar; Spitzfaden Claus E; Shen Carol; Shillings Anthony; Theobald Andrew J; Wohlkonig Alexandre; Pearson Neil D; Gwynn Michael N</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">7309</span>),
    <span class="NLM_cas:pages">935-40</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Despite the success of genomics in identifying new essential bacterial genes, there is a lack of sustainable leads in antibacterial drug discovery to address increasing multidrug resistance.  Type IIA topoisomerases cleave and religate DNA to regulate DNA topology and are a major class of antibacterial and anticancer drug targets, yet there is no well developed structural basis for understanding drug action.  Here we report the 2.1 A crystal structure of a potent, new class, broad-spectrum antibacterial agent in complex with Staphylococcus aureus DNA gyrase and DNA, showing a new mode of inhibition that circumvents fluoroquinolone resistance in this clinically important drug target.  The inhibitor 'bridges' the DNA and a transient non-catalytic pocket on the two-fold axis at the GyrA dimer interface, and is close to the active sites and fluoroquinolone binding sites.  In the inhibitor complex the active site seems poised to cleave the DNA, with a single metal ion observed between the TOPRIM (topoisomerase/primase) domain and the scissile phosphate.  This work provides new insights into the mechanism of topoisomerase action and a platform for structure-based drug design of a new class of antibacterial agents against a clinically proven, but conformationally flexible, enzyme class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTzBPk2FYUX-kBuFdjgVNKSfW6udTcc2eY902-HDWbrfrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cjmtFaktA%253D%253D&md5=006fef200c7a44f0671829b2a5fbddef</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnature09197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09197%26sid%3Dliteratum%253Aachs%26aulast%3DBax%26aufirst%3DB.%2BD.%26aulast%3DChan%26aufirst%3DP.%2BF.%26aulast%3DEggleston%26aufirst%3DD.%2BS.%26aulast%3DFosberry%26aufirst%3DA.%26aulast%3DGentry%26aufirst%3DD.%2BR.%26aulast%3DGorrec%26aufirst%3DF.%26aulast%3DGiordano%26aufirst%3DI.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DHennessy%26aufirst%3DA.%26aulast%3DHibbs%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DE.%26aulast%3DJones%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DK.%2BK.%26aulast%3DLewis%26aufirst%3DC.%2BJ.%26aulast%3DMay%26aufirst%3DE.%2BW.%26aulast%3DSaunders%26aufirst%3DM.%2BR.%26aulast%3DSingh%26aufirst%3DO.%26aulast%3DSpitzfaden%26aufirst%3DC.%2BE.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DShillings%26aufirst%3DA.%26aulast%3DTheobald%26aufirst%3DA.%2BJ.%26aulast%3DWohlkonig%26aufirst%3DA.%26aulast%3DPearson%26aufirst%3DN.%2BD.%26aulast%3DGwynn%26aufirst%3DM.%2BN.%26atitle%3DType%2520Iia%2520Topoisomerase%2520Inhibition%2520by%2520a%2520New%2520Class%2520of%2520Antibacterial%2520Agents%26jtitle%3DNature%26date%3D2010%26volume%3D466%26spage%3D935%26epage%3D940%26doi%3D10.1038%2Fnature09197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drlica, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiasa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustaev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span> <span> </span><span class="NLM_article-title">Quinolones: Action and Resistance Updated</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">981</span>– <span class="NLM_lpage">998</span>, <span class="refDoi"> DOI: 10.2174/156802609789630947</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.2174%2F156802609789630947" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=19747119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFWisb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=981-998&author=K.+Drlicaauthor=H.+Hiasaauthor=R.+Kernsauthor=M.+Malikauthor=A.+Mustaevauthor=X.+Zhao&title=Quinolones%3A+Action+and+Resistance+Updated&doi=10.2174%2F156802609789630947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Quinolones: action and resistance updated</span></div><div class="casAuthors">Drlica, Karl; Hiasa, Hiroshi; Kerns, Robert; Malik, Muhammad; Mustaev, Arkady; Zhao, Xilin</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">981-998</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The quinolones trap DNA gyrase and DNA topoisomerase IV on DNA as complexes in which the DNA is broken but constrained by protein.  Early studies suggested that drug binding occurs largely along helix-4 of the GyrA (gyrase) and ParC (topoisomerase IV) proteins.  However, recent X-ray crystallog. shows drug intercalating between the -1 and +1 nucleotides of cut DNA, with only one end of the drug extending to helix-4.  These two models may reflect distinct structural steps in complex formation.  A consequence of drug-enzyme-DNA complex formation is reversible inhibition of DNA replication; cell death arises from subsequent events in which bacterial chromosomes are fragmented through two poorly understood pathways.  In one pathway, chromosome fragmentation stimulates excessive accumulation of highly toxic reactive oxygen species that are responsible for cell death.  Quinolone resistance arises stepwise through selective amplification of mutants when drug concns. are above the MIC and below the MPC, as obsd. with static agar plate assays, dynamic in vitro systems, and exptl. infection of rabbits.  The gap between MIC and MPC can be narrowed by compd. design that should restrict the emergence of resistance.  Resistance is likely to become increasingly important, since three types of plasmid-borne resistance have been reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwqmNnQ2e72rVg90H21EOLACvtfcHk0ljMgbscJiAx8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFWisb%252FF&md5=e8f8b87460d826d75acd11efceb1a7f4</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2174%2F156802609789630947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802609789630947%26sid%3Dliteratum%253Aachs%26aulast%3DDrlica%26aufirst%3DK.%26aulast%3DHiasa%26aufirst%3DH.%26aulast%3DKerns%26aufirst%3DR.%26aulast%3DMalik%26aufirst%3DM.%26aulast%3DMustaev%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DX.%26atitle%3DQuinolones%253A%2520Action%2520and%2520Resistance%2520Updated%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2009%26volume%3D9%26spage%3D981%26epage%3D998%26doi%3D10.2174%2F156802609789630947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Schrödinger
LLC</span>.  <i>Schrödinger Release 2013-1:
Glide</i>. <span class="NLM_publisher-name">Schrödinger LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=Schr%C3%B6dinger%0ALLC&title=Schr%C3%B6dinger+Release+2013-1%3A%0AGlide"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DSchr%25C3%25B6dinger%2520Release%25202013-1%253A%250AGlide%26pub%3DSchr%25C3%25B6dinger%2520LLC%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reck, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alm, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassil, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dejonge, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyermann, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breault, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comita-Prevoir, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galullo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grebe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morningstar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayter, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, S.</span></span> <span> </span><span class="NLM_article-title">Novel N-Linked Aminopiperidine Inhibitors of Bacterial Topoisomerase Type Ii: Broad-Spectrum Antibacterial Agents with Reduced Herg Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7834</span>– <span class="NLM_lpage">7847</span>, <span class="refDoi"> DOI: 10.1021/jm2008826</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2008826" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlWrs7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7834-7847&author=F.+Reckauthor=R.+Almauthor=P.+Brassilauthor=J.+Newmanauthor=B.+Dejongeauthor=C.+J.+Eyermannauthor=G.+Breaultauthor=J.+Breenauthor=J.+Comita-Prevoirauthor=M.+Croninauthor=H.+Davisauthor=D.+Ehmannauthor=V.+Galulloauthor=B.+Gengauthor=T.+Grebeauthor=M.+Morningstarauthor=P.+Walkerauthor=B.+Hayterauthor=S.+Fisher&title=Novel+N-Linked+Aminopiperidine+Inhibitors+of+Bacterial+Topoisomerase+Type+Ii%3A+Broad-Spectrum+Antibacterial+Agents+with+Reduced+Herg+Activity&doi=10.1021%2Fjm2008826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Novel N-Linked Aminopiperidine Inhibitors of Bacterial Topoisomerase Type II: Broad-Spectrum Antibacterial Agents with Reduced hERG Activity</span></div><div class="casAuthors">Reck, Folkert; Alm, Richard; Brassil, Patrick; Newman, Joseph; DeJonge, Boudewijn; Eyermann, Charles J.; Breault, Gloria; Breen, John; Comita-Prevoir, Janelle; Cronin, Mark; Davis, Hajnalka; Ehmann, David; Galullo, Vincent; Geng, Bolin; Grebe, Tyler; Morningstar, Marshall; Walker, Phil; Hayter, Barry; Fisher, Stewart</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7834-7847</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Novel non-fluoroquinolone inhibitors of bacterial type II topoisomerases (DNA gyrase and topoisomerase IV) are of interest for the development of new antibacterial agents that are not impacted by target-mediated cross-resistance with fluoroquinolones.  Aminopiperidines that have a bicyclic arom. moiety linked through a carbon to an Et bridge, such as I, generally show potent broad-spectrum antibacterial activity, including quinolone-resistant isolates, but suffer from potent hERG inhibition (IC50= 3 μM for 1).  We now disclose the finding that new analogs of I with an N-linked cyclic amide moiety attached to the Et bridge, such as II, retain the broad-spectrum antibacterial activity of I but show significantly less hERG inhibition (IC50= 31 μM for II) and higher free fraction than 1.  One optimized analog showed moderate clearance in the dog and promising efficacy against Staphylococcus aureus in a mouse thigh infection model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohmQMvZgbqsbVg90H21EOLACvtfcHk0lica4FKlTySVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlWrs7fM&md5=ec76eb59cded28eb4c80627166f3e280</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm2008826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2008826%26sid%3Dliteratum%253Aachs%26aulast%3DReck%26aufirst%3DF.%26aulast%3DAlm%26aufirst%3DR.%26aulast%3DBrassil%26aufirst%3DP.%26aulast%3DNewman%26aufirst%3DJ.%26aulast%3DDejonge%26aufirst%3DB.%26aulast%3DEyermann%26aufirst%3DC.%2BJ.%26aulast%3DBreault%26aufirst%3DG.%26aulast%3DBreen%26aufirst%3DJ.%26aulast%3DComita-Prevoir%26aufirst%3DJ.%26aulast%3DCronin%26aufirst%3DM.%26aulast%3DDavis%26aufirst%3DH.%26aulast%3DEhmann%26aufirst%3DD.%26aulast%3DGalullo%26aufirst%3DV.%26aulast%3DGeng%26aufirst%3DB.%26aulast%3DGrebe%26aufirst%3DT.%26aulast%3DMorningstar%26aufirst%3DM.%26aulast%3DWalker%26aufirst%3DP.%26aulast%3DHayter%26aufirst%3DB.%26aulast%3DFisher%26aufirst%3DS.%26atitle%3DNovel%2520N-Linked%2520Aminopiperidine%2520Inhibitors%2520of%2520Bacterial%2520Topoisomerase%2520Type%2520Ii%253A%2520Broad-Spectrum%2520Antibacterial%2520Agents%2520with%2520Reduced%2520Herg%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7834%26epage%3D7847%26doi%3D10.1021%2Fjm2008826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miles, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennessy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cailleau, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dabbs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esken, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoover, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukmar, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spitzfaden, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markwell, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minthorn, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rittenhouse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwynn, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, N. D.</span></span> <span> </span><span class="NLM_article-title">Novel Hydroxyl Tricyclics (E.G., Gsk966587) as Potent Inhibitors of Bacterial Type Iia Topoisomerases</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5437</span>– <span class="NLM_lpage">5441</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2013.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=23968823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlShtrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=5437-5441&author=T.+J.+Milesauthor=A.+J.+Hennessyauthor=B.+Baxauthor=G.+Brooksauthor=B.+S.+Brownauthor=P.+Brownauthor=N.+Cailleauauthor=D.+Chenauthor=S.+Dabbsauthor=D.+T.+Daviesauthor=J.+M.+Eskenauthor=I.+Giordanoauthor=J.+L.+Hooverauthor=J.+Huangauthor=G.+E.+Jonesauthor=S.+K.+Sukmarauthor=C.+Spitzfadenauthor=R.+E.+Markwellauthor=E.+A.+Minthornauthor=S.+Rittenhouseauthor=M.+N.+Gwynnauthor=N.+D.+Pearson&title=Novel+Hydroxyl+Tricyclics+%28E.G.%2C+Gsk966587%29+as+Potent+Inhibitors+of+Bacterial+Type+Iia+Topoisomerases&doi=10.1016%2Fj.bmcl.2013.07.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Novel hydroxyl tricyclics (e.g., GSK966587) as potent inhibitors of bacterial type IIA topoisomerases</span></div><div class="casAuthors">Miles, Timothy J.; Hennessy, Alan J.; Bax, Ben; Brooks, Gerald; Brown, Barry S.; Brown, Pamela; Cailleau, Nathalie; Chen, Dongzhao; Dabbs, Steven; Davies, David T.; Esken, Joel M.; Giordano, Ilaria; Hoover, Jennifer L.; Huang, Jianzhong; Jones, Graham E.; Kusalakumari Sukmar, Senthill K.; Spitzfaden, Claus; Markwell, Roger E.; Minthorn, Elisabeth A.; Rittenhouse, Steve; Gwynn, Michael N.; Pearson, Neil D.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5437-5441</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">During the course of the authors' research to find novel mode of action antibacterials, the authors discovered a series of hydroxyl tricyclic compds. that showed good potency against Gram-pos. and Gram-neg. pathogens.  These compds. inhibit bacterial type IIA topoisomerases.  Herein the authors will discuss structure-activity relationships in this series and report advanced studies on compd. 1 (GSK966587) which demonstrates good PK and in vivo efficacy properties.  X-ray crystallog. studies were used to provide insight into the structural basis for the difference in antibacterial potency between enantiomers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz_XwqrKpCrLVg90H21EOLACvtfcHk0lica4FKlTySVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlShtrjN&md5=976c6c41b726dc03f0e5b543406f04b4</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DMiles%26aufirst%3DT.%2BJ.%26aulast%3DHennessy%26aufirst%3DA.%2BJ.%26aulast%3DBax%26aufirst%3DB.%26aulast%3DBrooks%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DB.%2BS.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DCailleau%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDabbs%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DD.%2BT.%26aulast%3DEsken%26aufirst%3DJ.%2BM.%26aulast%3DGiordano%26aufirst%3DI.%26aulast%3DHoover%26aufirst%3DJ.%2BL.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DG.%2BE.%26aulast%3DSukmar%26aufirst%3DS.%2BK.%26aulast%3DSpitzfaden%26aufirst%3DC.%26aulast%3DMarkwell%26aufirst%3DR.%2BE.%26aulast%3DMinthorn%26aufirst%3DE.%2BA.%26aulast%3DRittenhouse%26aufirst%3DS.%26aulast%3DGwynn%26aufirst%3DM.%2BN.%26aulast%3DPearson%26aufirst%3DN.%2BD.%26atitle%3DNovel%2520Hydroxyl%2520Tricyclics%2520%2528E.G.%252C%2520Gsk966587%2529%2520as%2520Potent%2520Inhibitors%2520of%2520Bacterial%2520Type%2520Iia%2520Topoisomerases%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D5437%26epage%3D5441%26doi%3D10.1016%2Fj.bmcl.2013.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Schrödinger
LLC</span>.  <i>Schrödinger Release 2013-1:
Phase</i>. <span class="NLM_publisher-name">Schrödinger LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=Schr%C3%B6dinger%0ALLC&title=Schr%C3%B6dinger+Release+2013-1%3A%0APhase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DSchr%25C3%25B6dinger%2520Release%25202013-1%253A%250APhase%26pub%3DSchr%25C3%25B6dinger%2520LLC%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sastry, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Rapid Shape-Based Ligand Alignment and Virtual Screening Method Based on Atom/Feature-Pair Similarities and Volume Overlap Scoring</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2455</span>– <span class="NLM_lpage">2466</span>, <span class="refDoi"> DOI: 10.1021/ci2002704</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci2002704" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKnt73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2455-2466&author=G.+M.+Sastryauthor=S.+L.+Dixonauthor=W.+Sherman&title=Rapid+Shape-Based+Ligand+Alignment+and+Virtual+Screening+Method+Based+on+Atom%2FFeature-Pair+Similarities+and+Volume+Overlap+Scoring&doi=10.1021%2Fci2002704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Shape-Based Ligand Alignment and Virtual Screening Method Based on Atom/Feature-Pair Similarities and Volume Overlap Scoring</span></div><div class="casAuthors">Sastry, G. Madhavi; Dixon, Steven L.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2455-2466</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Shape-based methods for aligning and scoring ligands have proven to be valuable in the field of computer-aided drug design.  Here, we describe a new shape-based flexible ligand superposition and virtual screening method, Phase Shape, which is shown to rapidly produce accurate 3D ligand alignments and efficiently enrich actives in virtual screening.  We describe the methodol., which is based on the principle of atom distribution triplets to rapidly define trial alignments, followed by refinement of top alignments to maximize the vol. overlap.  The method can be run in a shape-only mode or it can include atom types or pharmacophore feature encoding, the latter consistently producing the best results for database screening.  We apply Phase Shape to flexibly align mols. that bind to the same target and show that the method consistently produces correct alignments when compared with crystal structures.  We then illustrate the effectiveness of the method for identifying active compds. in virtual screening of eleven diverse targets.  Multiple parameters are explored, including atom typing, query structure conformation, and the database conformer generation protocol.  We show that Phase Shape performs well in database screening calcns. when compared with other shape-based methods using a common set of actives and decoys from the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM-u23DvDpQ7Vg90H21EOLACvtfcHk0lica4FKlTySVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKnt73K&md5=4234abb2dd7e6364c377e92047b65aba</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci2002704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2002704%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DG.%2BM.%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DRapid%2520Shape-Based%2520Ligand%2520Alignment%2520and%2520Virtual%2520Screening%2520Method%2520Based%2520on%2520Atom%252FFeature-Pair%2520Similarities%2520and%2520Volume%2520Overlap%2520Scoring%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2455%26epage%3D2466%26doi%3D10.1021%2Fci2002704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Motifs for Molecular Recognition Exploiting Hydrophobic Enclosure in Protein-Ligand Binding</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">808</span>– <span class="NLM_lpage">813</span>, <span class="refDoi"> DOI: 10.1073/pnas.0610202104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1073%2Fpnas.0610202104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=17204562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=808-813&author=T.+Youngauthor=R.+Abelauthor=B.+Kimauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Motifs+for+Molecular+Recognition+Exploiting+Hydrophobic+Enclosure+in+Protein-Ligand+Binding&doi=10.1073%2Fpnas.0610202104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Motifs for molecular recognition exploiting hydrophobic enclosure in protein-ligand binding</span></div><div class="casAuthors">Young, Tom; Abel, Robert; Kim, Byungchan; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">808-813</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The thermodn. properties and phase behavior of water in confined regions can vary significantly from that obsd. in the bulk.  This is particularly true for systems in which the confinement is on the mol.-length scale.  In this study, we use mol. dynamics simulations and a powerful solvent anal. technique based on inhomogeneous solvation theory to investigate the properties of water mols. that solvate the confined regions of protein active sites.  Our simulations and anal. indicate that the solvation of protein active sites that are characterized by hydrophobic enclosure and correlated hydrogen bonds induce atypical entropic and enthalpic penalties of hydration.  These penalties apparently stabilize the protein-ligand complex with respect to the independently solvated ligand and protein, which leads to enhanced binding affinities.  Our anal. elucidates several challenging cases, including the super affinity of the streptavidin-biotin system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWumQ61YTqILVg90H21EOLACvtfcHk0liDuPDScViTDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D&md5=1109e3f48670cd910885f7fd527eb880</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0610202104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0610202104%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DMotifs%2520for%2520Molecular%2520Recognition%2520Exploiting%2520Hydrophobic%2520Enclosure%2520in%2520Protein-Ligand%2520Binding%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D808%26epage%3D813%26doi%3D10.1073%2Fpnas.0610202104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">2817</span>– <span class="NLM_lpage">2831</span>, <span class="refDoi"> DOI: 10.1021/ja0771033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0771033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=2817-2831&author=R.+Abelauthor=T.+Youngauthor=R.+Faridauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Role+of+the+Active-Site+Solvent+in+the+Thermodynamics+of+Factor+Xa+Ligand+Binding&doi=10.1021%2Fja0771033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span></div><div class="casAuthors">Abel, Robert; Young, Tom; Farid, Ramy; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2817-2831</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding the underlying physics of the binding of small-mol. ligands to protein active sites is a key objective of computational chem. and biol.  It is widely believed that displacement of water mols. from the active site by the ligand is a principal (if not the dominant) source of binding free energy.  Although continuum theories of hydration are routinely used to describe the contributions of the solvent to the binding affinity of the complex, it is still an unsettled question as to whether or not these continuum solvation theories describe the underlying mol. physics with sufficient accuracy to reliably rank the binding affinities of a set of ligands for a given protein.  Here we develop a novel, computationally efficient descriptor of the contribution of the solvent to the binding free energy of a small mol. and its assocd. receptor that captures the effects of the ligand displacing the solvent from the protein active site with at. detail.  This descriptor quant. predicts (R2 = 0.81) the binding free energy differences between congeneric ligand pairs for the test system factor Xa, elucidates phys. properties of the active site solvent that appear to be missing in most continuum theories of hydration, and identifies several features of the hydration of the factor Xa active site relevant to the structure-activity relationship of its inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS42r59nNXH7Vg90H21EOLACvtfcHk0liDuPDScViTDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D&md5=8eefe76547a82bccaea38f2c84e6cd5f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fja0771033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0771033%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DRole%2520of%2520the%2520Active-Site%2520Solvent%2520in%2520the%2520Thermodynamics%2520of%2520Factor%2520Xa%2520Ligand%2520Binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D2817%26epage%3D2831%26doi%3D10.1021%2Fja0771033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pendleton, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorman, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmore, B. F.</span></span> <span> </span><span class="NLM_article-title">Clinical Relevance of the Eskape Pathogens</span>. <i>Expert Rev. Anti-infect. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1586/eri.13.12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1586%2Feri.13.12" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2014&pages=297-308&author=J.+N.+Pendletonauthor=S.+P.+Gormanauthor=B.+F.+Gilmore&title=Clinical+Relevance+of+the+Eskape+Pathogens&doi=10.1586%2Feri.13.12"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1586%2Feri.13.12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1586%252Feri.13.12%26sid%3Dliteratum%253Aachs%26aulast%3DPendleton%26aufirst%3DJ.%2BN.%26aulast%3DGorman%26aufirst%3DS.%2BP.%26aulast%3DGilmore%26aufirst%3DB.%2BF.%26atitle%3DClinical%2520Relevance%2520of%2520the%2520Eskape%2520Pathogens%26jtitle%3DExpert%2520Rev.%2520Anti-infect.%2520Ther.%26date%3D2014%26volume%3D11%26spage%3D297%26epage%3D308%26doi%3D10.1586%2Feri.13.12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biedenbach, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouchillon, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hackel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scangarella-Oman, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakielaszek, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahm, D. F.</span></span> <span> </span><span class="NLM_article-title">In Vitro Activity of Gepotidacin, a Novel Triazaacenaphthylene Bacterial Topoisomerase Inhibitor, against a Broad Spectrum of Bacterial Pathogens</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1918</span>– <span class="NLM_lpage">1923</span>, <span class="refDoi"> DOI: 10.1128/AAC.02820-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.02820-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=26729499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsF2ntbbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2016&pages=1918-1923&author=D.+J.+Biedenbachauthor=S.+K.+Bouchillonauthor=M.+Hackelauthor=L.+A.+Millerauthor=N.+E.+Scangarella-Omanauthor=C.+Jakielaszekauthor=D.+F.+Sahm&title=In+Vitro+Activity+of+Gepotidacin%2C+a+Novel+Triazaacenaphthylene+Bacterial+Topoisomerase+Inhibitor%2C+against+a+Broad+Spectrum+of+Bacterial+Pathogens&doi=10.1128%2FAAC.02820-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro activity of gepotidacin, a novel triazaacenaphthylene bacterial topoisomerase inhibitor, against a broad spectrum of bacterial pathogens</span></div><div class="casAuthors">Biedenbach, D. J.; Bouchillon, S. K.; Hackel, M.; Miller, L. A.; Scangarella-Oman, N. E.; Jakielaszek, C.; Sahm, D. F.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1918-1923</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Gepotidacin inhibits bacterial DNA replication through a mode different from that of fluoroquinolones.  Gepotidacin and comparators were tested by broth and agar diln. against clin. isolates.  The in vitro activities of gepotidacin were comparable against methicillin-susceptible and -resistant Staphylococcus aureus (MSSA and MRSA, resp.) isolates (MIC90, 0.5 μg/mL).  The gepotidacin MIC90s were as follows (in micrograms per mL) for the indicated bacteria: Streptococcus pyogenes, 0.25; Escherichia coli, 2; Moraxella catarrhalis, ≤0.06; Streptococcus pneumoniae (0.25), Haemophilus influenzae, 1; Clostridium perfringens, 0.5; and Shigella spp., 1, including levofloxacin-resistant subsets.  Gepotidacin warrants further investigation for clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFHvWFX6zKO7Vg90H21EOLACvtfcHk0liDuPDScViTDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsF2ntbbE&md5=80b9eca78be2d6f2fae33a97bdabef68</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1128%2FAAC.02820-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02820-15%26sid%3Dliteratum%253Aachs%26aulast%3DBiedenbach%26aufirst%3DD.%2BJ.%26aulast%3DBouchillon%26aufirst%3DS.%2BK.%26aulast%3DHackel%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DL.%2BA.%26aulast%3DScangarella-Oman%26aufirst%3DN.%2BE.%26aulast%3DJakielaszek%26aufirst%3DC.%26aulast%3DSahm%26aufirst%3DD.%2BF.%26atitle%3DIn%2520Vitro%2520Activity%2520of%2520Gepotidacin%252C%2520a%2520Novel%2520Triazaacenaphthylene%2520Bacterial%2520Topoisomerase%2520Inhibitor%252C%2520against%2520a%2520Broad%2520Spectrum%2520of%2520Bacterial%2520Pathogens%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2016%26volume%3D60%26spage%3D1918%26epage%3D1923%26doi%3D10.1128%2FAAC.02820-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span>; <span class="NLM_string-name">Di, L.</span></span> <i>Drug-Like Properties: Concepts, Structure, Design, and Methods</i>. <span class="NLM_edition">2nd</span> ed.; <span class="NLM_publisher-name">Elsevier</span>: <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=E.+Kerns&author=L.+Di&title=Drug-Like+Properties%3A+Concepts%2C+Structure%2C+Design%2C+and+Methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKerns%26aufirst%3DE.%26btitle%3DDrug-Like%2520Properties%253A%2520Concepts%252C%2520Structure%252C%2520Design%252C%2520and%2520Methods%26pub%3DElsevier%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Schrödinger
LLC</span>.  <i>The Pymol Molecular Graphics
System, Version 1.8</i><i>.</i> <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=Schr%C3%B6dinger%0ALLC&title=The+Pymol+Molecular+Graphics%0ASystem%2C+Version+1.8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DThe%2520Pymol%2520Molecular%2520Graphics%250ASystem%252C%2520Version%25201.8%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berthold, M. R.</span>; <span class="NLM_string-name">Cebron, N.</span>; <span class="NLM_string-name">Dill, F.</span>; <span class="NLM_string-name">Gabriel, T. R.</span>; <span class="NLM_string-name">Kötter, T.</span>; <span class="NLM_string-name">Meinl, T.</span>; <span class="NLM_string-name">Ohl, P.</span>; <span class="NLM_string-name">Sieb, C.</span>; <span class="NLM_string-name">Thiel, K.</span>; <span class="NLM_string-name">Wiswedel, B.</span></span> <span> </span><span class="NLM_article-title">Knime: The Konstanz Information Miner</span>. In  <i>Data Analysis, Machine Learning and Applications: Studies in Classification, Data Analysis, and Knowledge Organization</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preisach, C.</span>; <span class="NLM_string-name">Burkhardt, H.</span>; <span class="NLM_string-name">Schmidt-Thieme, L.</span>; <span class="NLM_string-name">Decker, R.</span></span>, Eds. <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Berlin, Heidelberg</span>, <span class="NLM_year">2008</span>; pp  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1007%2F978-3-540-78246-9_38" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=319-326&author=M.+R.+Berthold&author=N.+Cebron&author=F.+Dill&author=T.+R.+Gabriel&author=T.+K%C3%B6tter&author=T.+Meinl&author=P.+Ohl&author=C.+Sieb&author=K.+Thiel&author=B.+Wiswedelauthor=C.+Preisach&author=H.+Burkhardt&author=L.+Schmidt-Thieme&author=R.+Decker&title=Data+Analysis%2C+Machine+Learning+and+Applications%3A+Studies+in+Classification%2C+Data+Analysis%2C+and+Knowledge+Organization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2F978-3-540-78246-9_38&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-540-78246-9_38%26sid%3Dliteratum%253Aachs%26aulast%3DBerthold%26aufirst%3DM.%2BR.%26atitle%3DKnime%253A%2520The%2520Konstanz%2520Information%2520Miner%26btitle%3DData%2520Analysis%252C%2520Machine%2520Learning%2520and%2520Applications%253A%2520Studies%2520in%2520Classification%252C%2520Data%2520Analysis%252C%2520and%2520Knowledge%2520Organization%26aulast%3DPreisach%26aufirst%3DC.%26pub%3DSpringer%26date%3D2008%26spage%3D319%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Schrödinger
LLC</span>.  <i>Schrödinger Release 2013–1:
Schrödinger Knime Extensions</i>; <span class="NLM_publisher-name">Schrödinger
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=Schr%C3%B6dinger%0ALLC&title=Schr%C3%B6dinger+Release+2013%E2%80%931%3A%0ASchr%C3%B6dinger+Knime+Extensions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DSchr%25C3%25B6dinger%2520Release%25202013%25E2%2580%25931%253A%250ASchr%25C3%25B6dinger%2520Knime%2520Extensions%26pub%3DSchr%25C3%25B6dinger%250ALLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Python Software Foundation</span>.  <i>Python Language Reference</i>, Version
2.7. <a href="http://www.python.org" class="extLink">www.python.org</a></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Python+Software+Foundation.+Python+Language+Reference%2C+Version%0A2.7.+www.python.org"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DPython%2520Language%2520Reference" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiener, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santillán, A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allison, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinde, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hack, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motley, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldschmidt, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grice, C. A.</span></span> <span> </span><span class="NLM_article-title">Tetrahydroindazole Inhibitors of Bacterial Type Ii Topoisomerases. Part 2: Sar Development and Potency against Multidrug-Resistant Strains</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2718</span>– <span class="NLM_lpage">2722</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2007.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2007.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=17382544" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslOqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=2718-2722&author=J.+J.+Wienerauthor=L.+Gomezauthor=H.+Venkatesanauthor=A.+Santill%C3%A1nauthor=B.+D.+Allisonauthor=K.+L.+Schwarzauthor=S.+Shindeauthor=L.+Tangauthor=M.+D.+Hackauthor=B.+J.+Morrowauthor=S.+T.+Motleyauthor=R.+M.+Goldschmidtauthor=K.+J.+Shawauthor=T.+K.+Jonesauthor=C.+A.+Grice&title=Tetrahydroindazole+Inhibitors+of+Bacterial+Type+Ii+Topoisomerases.+Part+2%3A+Sar+Development+and+Potency+against+Multidrug-Resistant+Strains&doi=10.1016%2Fj.bmcl.2007.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Tetrahydroindazole inhibitors of bacterial type II topoisomerases. Part 2: SAR development and potency against multidrug-resistant strains</span></div><div class="casAuthors">Wiener, John J. M.; Gomez, Laurent; Venkatesan, Hariharan; Santillan, Alejandro, Jr.; Allison, Brett D.; Schwarz, Kimberly L.; Shinde, Shirin; Tang, Liu; Hack, Michael D.; Morrow, Brian J.; Motley, S. Timothy; Goldschmidt, Raul M.; Shaw, Karen Joy; Jones, Todd K.; Grice, Cheryl A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2718-2722</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">We have previously reported a novel class of tetrahydroindazoles that display potency against a variety of Gram-pos. and Gram-neg. bacteria, potentially via interaction with type II bacterial topoisomerases.  Herein are reported SAR investigations of this new series.  Several compds. possessing broad-spectrum potency were prepd.  Further, these compds. exhibit activity against multidrug-resistant Gram-pos. microorganisms equiv. to that against susceptible strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKlC4VAkQDZLVg90H21EOLACvtfcHk0lgmeqra0usx4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslOqtbY%253D&md5=9bc16d091987be0ec1ced5dd2d910d5e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DWiener%26aufirst%3DJ.%2BJ.%26aulast%3DGomez%26aufirst%3DL.%26aulast%3DVenkatesan%26aufirst%3DH.%26aulast%3DSantill%25C3%25A1n%26aufirst%3DA.%26aulast%3DAllison%26aufirst%3DB.%2BD.%26aulast%3DSchwarz%26aufirst%3DK.%2BL.%26aulast%3DShinde%26aufirst%3DS.%26aulast%3DTang%26aufirst%3DL.%26aulast%3DHack%26aufirst%3DM.%2BD.%26aulast%3DMorrow%26aufirst%3DB.%2BJ.%26aulast%3DMotley%26aufirst%3DS.%2BT.%26aulast%3DGoldschmidt%26aufirst%3DR.%2BM.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DJones%26aufirst%3DT.%2BK.%26aulast%3DGrice%26aufirst%3DC.%2BA.%26atitle%3DTetrahydroindazole%2520Inhibitors%2520of%2520Bacterial%2520Type%2520Ii%2520Topoisomerases.%2520Part%25202%253A%2520Sar%2520Development%2520and%2520Potency%2520against%2520Multidrug-Resistant%2520Strains%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D2718%26epage%3D2722%26doi%3D10.1016%2Fj.bmcl.2007.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hack, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiener, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santillán, A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pippel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motley, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grice, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. K.</span></span> <span> </span><span class="NLM_article-title">Novel Pyrazole Derivatives as Potent Inhibitors of Type Ii Topoisomerases. Part 1: Synthesis and Preliminary Sar Analysis</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2723</span>– <span class="NLM_lpage">2727</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2007.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2007.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=17368897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslOqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=2723-2727&author=L.+Gomezauthor=M.+D.+Hackauthor=J.+Wuauthor=J.+J.+Wienerauthor=H.+Venkatesanauthor=A.+Santill%C3%A1nauthor=D.+J.+Pippelauthor=N.+Maniauthor=B.+J.+Morrowauthor=S.+T.+Motleyauthor=K.+J.+Shawauthor=R.+Wolinauthor=C.+A.+Griceauthor=T.+K.+Jones&title=Novel+Pyrazole+Derivatives+as+Potent+Inhibitors+of+Type+Ii+Topoisomerases.+Part+1%3A+Synthesis+and+Preliminary+Sar+Analysis&doi=10.1016%2Fj.bmcl.2007.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Novel pyrazole derivatives as potent inhibitors of type II topoisomerases. Part 1: Synthesis and preliminary SAR analysis</span></div><div class="casAuthors">Gomez, Laurent; Hack, Michael D.; Wu, Jiejun; Wiener, John J. M.; Venkatesan, Hari; Santillan, Alejandro; Pippel, Daniel J.; Mani, Neelakandha; Morrow, Brian J.; Motley, S. Timothy; Shaw, Karen Joy; Wolin, Ronald; Grice, Cheryl A.; Jones, Todd K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2723-2727</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In an attempt to search for a new class of antibacterial agents, a series of pyrazole analogs that possess good antibacterial activity for Gram-pos. and Gram-neg. organisms via inhibition of type II bacterial topoisomerases was discovered.  The structure-activity relationships of this series, with an emphasis on the length and conformation of the linker was also investigated.  Several tetrahydroindazole analogs was identified.  Compd. I was shown to be the most potent class of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp54rXZYBq1jbVg90H21EOLACvtfcHk0liPME2hT8_BtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslOqtbc%253D&md5=bfb82255564581660b9b7002fc079950</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DGomez%26aufirst%3DL.%26aulast%3DHack%26aufirst%3DM.%2BD.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DWiener%26aufirst%3DJ.%2BJ.%26aulast%3DVenkatesan%26aufirst%3DH.%26aulast%3DSantill%25C3%25A1n%26aufirst%3DA.%26aulast%3DPippel%26aufirst%3DD.%2BJ.%26aulast%3DMani%26aufirst%3DN.%26aulast%3DMorrow%26aufirst%3DB.%2BJ.%26aulast%3DMotley%26aufirst%3DS.%2BT.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DWolin%26aufirst%3DR.%26aulast%3DGrice%26aufirst%3DC.%2BA.%26aulast%3DJones%26aufirst%3DT.%2BK.%26atitle%3DNovel%2520Pyrazole%2520Derivatives%2520as%2520Potent%2520Inhibitors%2520of%2520Type%2520Ii%2520Topoisomerases.%2520Part%25201%253A%2520Synthesis%2520and%2520Preliminary%2520Sar%2520Analysis%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D2723%26epage%3D2727%26doi%3D10.1016%2Fj.bmcl.2007.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miles, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barfoot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dabbs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Downie, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyrisch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwynn, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennessy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoover, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markwell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rittenhouse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, N.</span></span> <span> </span><span class="NLM_article-title">Novel Cyclohexyl-Amides as Potent Antibacterials Targeting Bacterial Type Iia Topoisomerases</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">7483</span>– <span class="NLM_lpage">7488</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.09.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2011.09.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=22030032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFSqtb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=7483-7488&author=T.+J.+Milesauthor=C.+Barfootauthor=G.+Brooksauthor=P.+Brownauthor=D.+Chenauthor=S.+Dabbsauthor=D.+T.+Daviesauthor=D.+L.+Downieauthor=S.+Eyrischauthor=I.+Giordanoauthor=M.+N.+Gwynnauthor=A.+Hennessyauthor=J.+Hooverauthor=J.+Huangauthor=G.+Jonesauthor=R.+Markwellauthor=S.+Rittenhouseauthor=H.+Xiangauthor=N.+Pearson&title=Novel+Cyclohexyl-Amides+as+Potent+Antibacterials+Targeting+Bacterial+Type+Iia+Topoisomerases&doi=10.1016%2Fj.bmcl.2011.09.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Novel cyclohexyl-amides as potent antibacterials targeting bacterial type IIA topoisomerases</span></div><div class="casAuthors">Miles, Timothy J.; Barfoot, Christopher; Brooks, Gerald; Brown, Pamela; Chen, Dongzhao; Dabbs, Steven; Davies, David T.; Downie, David L.; Eyrisch, Susanne; Giordano, Ilaria; Gwynn, Michael N.; Hennessy, Alan; Hoover, Jennifer; Huang, Jianzhong; Jones, Graham; Markwell, Roger; Rittenhouse, Stephen; Xiang, Hong; Pearson, Neil</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7483-7488</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">As part of our wider efforts to exploit novel mode of action antibacterials, we have discovered a series of cyclohexyl-amide compds. that has good Gram pos. and Gram neg. potency.  The mechanism of action is via inhibition of bacterial topoisomerases II and IV.  We have investigated various subunits in this series and report advanced studies on compd. 7 which demonstrates good PK and in vivo efficacy properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOaL1uO_CXkLVg90H21EOLACvtfcHk0liPME2hT8_BtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFSqtb7E&md5=a4404e4f4cd9c8ea602316b3593d68b7</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.09.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.09.114%26sid%3Dliteratum%253Aachs%26aulast%3DMiles%26aufirst%3DT.%2BJ.%26aulast%3DBarfoot%26aufirst%3DC.%26aulast%3DBrooks%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDabbs%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DD.%2BT.%26aulast%3DDownie%26aufirst%3DD.%2BL.%26aulast%3DEyrisch%26aufirst%3DS.%26aulast%3DGiordano%26aufirst%3DI.%26aulast%3DGwynn%26aufirst%3DM.%2BN.%26aulast%3DHennessy%26aufirst%3DA.%26aulast%3DHoover%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DG.%26aulast%3DMarkwell%26aufirst%3DR.%26aulast%3DRittenhouse%26aufirst%3DS.%26aulast%3DXiang%26aufirst%3DH.%26aulast%3DPearson%26aufirst%3DN.%26atitle%3DNovel%2520Cyclohexyl-Amides%2520as%2520Potent%2520Antibacterials%2520Targeting%2520Bacterial%2520Type%2520Iia%2520Topoisomerases%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D7483%26epage%3D7488%26doi%3D10.1016%2Fj.bmcl.2011.09.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miles, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axten, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barfoot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dabbs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Downie, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyrisch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwynn, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennessy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoover, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markwell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minthorn, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rittenhouse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seefeld, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, N.</span></span> <span> </span><span class="NLM_article-title">Novel Amino-Piperidines as Potent Antibacterials Targeting Bacterial Type Iia Topoisomerases</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">7489</span>– <span class="NLM_lpage">7495</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.09.117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2011.09.117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=22047689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFSqtb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=7489-7495&author=T.+J.+Milesauthor=J.+M.+Axtenauthor=C.+Barfootauthor=G.+Brooksauthor=P.+Brownauthor=D.+Chenauthor=S.+Dabbsauthor=D.+T.+Daviesauthor=D.+L.+Downieauthor=S.+Eyrischauthor=T.+Gallagherauthor=I.+Giordanoauthor=M.+N.+Gwynnauthor=A.+Hennessyauthor=J.+Hooverauthor=J.+Huangauthor=G.+Jonesauthor=R.+Markwellauthor=W.+H.+Millerauthor=E.+A.+Minthornauthor=S.+Rittenhouseauthor=M.+Seefeldauthor=N.+Pearson&title=Novel+Amino-Piperidines+as+Potent+Antibacterials+Targeting+Bacterial+Type+Iia+Topoisomerases&doi=10.1016%2Fj.bmcl.2011.09.117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Novel amino-piperidines as potent antibacterials targeting bacterial type IIA topoisomerases</span></div><div class="casAuthors">Miles, Timothy J.; Axten, Jeffrey M.; Barfoot, Christopher; Brooks, Gerald; Brown, Pamela; Chen, Dongzhao; Dabbs, Steven; Davies, David T.; Downie, David L.; Eyrisch, Susanne; Gallagher, Timothy; Giordano, Ilaria; Gwynn, Michael N.; Hennessy, Alan; Hoover, Jennifer; Huang, Jianzhong; Jones, Graham; Markwell, Roger; Miller, William H.; Minthorn, Elizabeth A.; Rittenhouse, Stephen; Seefeld, Mark; Pearson, Neil</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7489-7495</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We have identified a series of amino-piperidine antibacterials with a good broad spectrum potency.  We report the investigation of various subunits in this series and advanced studies on compd. 8.  Compd. 8 possesses good pharmacokinetics, broad spectrum antibacterial activity and demonstrates oral efficacy in a rat lung infection model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpe_9NHsx5_Y7Vg90H21EOLACvtfcHk0liPME2hT8_BtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFSqtb%252FN&md5=ddc20f19a1e6f608066586ff739f0af8</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.09.117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.09.117%26sid%3Dliteratum%253Aachs%26aulast%3DMiles%26aufirst%3DT.%2BJ.%26aulast%3DAxten%26aufirst%3DJ.%2BM.%26aulast%3DBarfoot%26aufirst%3DC.%26aulast%3DBrooks%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDabbs%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DD.%2BT.%26aulast%3DDownie%26aufirst%3DD.%2BL.%26aulast%3DEyrisch%26aufirst%3DS.%26aulast%3DGallagher%26aufirst%3DT.%26aulast%3DGiordano%26aufirst%3DI.%26aulast%3DGwynn%26aufirst%3DM.%2BN.%26aulast%3DHennessy%26aufirst%3DA.%26aulast%3DHoover%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DG.%26aulast%3DMarkwell%26aufirst%3DR.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DMinthorn%26aufirst%3DE.%2BA.%26aulast%3DRittenhouse%26aufirst%3DS.%26aulast%3DSeefeld%26aufirst%3DM.%26aulast%3DPearson%26aufirst%3DN.%26atitle%3DNovel%2520Amino-Piperidines%2520as%2520Potent%2520Antibacterials%2520Targeting%2520Bacterial%2520Type%2520Iia%2520Topoisomerases%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D7489%26epage%3D7495%26doi%3D10.1016%2Fj.bmcl.2011.09.117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namchuk, M.</span></span> <span> </span><span class="NLM_article-title">Designing Screens: How to Make Your Hits a Hit</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">259</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1038/nrd1063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1038%2Fnrd1063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=12669025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD3sXisFWktrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=259-266&author=W.+P.+Waltersauthor=M.+Namchuk&title=Designing+Screens%3A+How+to+Make+Your+Hits+a+Hit&doi=10.1038%2Fnrd1063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A guide to drug discovery: Designing screens: how to make your hits a hit</span></div><div class="casAuthors">Walters, W. Patrick; Namchuk, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">259-266</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The basic goal of small-mol. screening is the identification of chem. interesting starting points for elaboration towards a drug.  A no. of innovative approaches for pursuing this goal have evolved, and the right approach is dictated by the target class being pursued and the capabilities of the organization involved.hood.  A recent trend in high-throughput screening has been to place less emphasis on the no. of data points that can be produced, and to focus instead on the quality of the data obtained.  Several computational and technol. advances have aided in the selection of compds. for screening and widened the variety of assay formats available for screening.  The effect on the efficiency of the screening process is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ5E12ik7BrbVg90H21EOLACvtfcHk0lhdGgLkTKDW7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXisFWktrk%253D&md5=6bec4547b8dce69ab84ce226cde28f8a</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnrd1063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1063%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DNamchuk%26aufirst%3DM.%26atitle%3DDesigning%2520Screens%253A%2520How%2520to%2520Make%2520Your%2520Hits%2520a%2520Hit%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D259%26epage%3D266%26doi%3D10.1038%2Fnrd1063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Schrödinger
LLC</span>.  <i>Schrödinger Release 2014-3:
Ligprep, Version 3.1</i>. <span class="NLM_publisher-name">Schrödinger
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=Schr%C3%B6dinger%0ALLC&title=Schr%C3%B6dinger+Release+2014-3%3A%0ALigprep%2C+Version+3.1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DSchr%25C3%25B6dinger%2520Release%25202014-3%253A%250ALigprep%252C%2520Version%25203.1%26pub%3DSchr%25C3%25B6dinger%250ALLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schrödinger
LLC</span></span>.  <i>Schrödinger Release 2014-3: Epik</i>. <span class="NLM_publisher-name">Schrödinger LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=+Schr%C3%B6dinger%0ALLC&title=Schr%C3%B6dinger+Release+2014-3%3A+Epik"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSchr%25C3%25B6dinger%2BLLC%26jtitle%3DSchr%25C3%25B6dinger%2520Release%25202014-3%253A%2520Epik%26pub%3DSchr%25C3%25B6dinger%2520LLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hundsdörfer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemmerling, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totzke, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednarski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Borgne, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jose, J.</span></span> <span> </span><span class="NLM_article-title">Indeno[1,2-B]Indole Derivatives as a Novel Class of Potent Human Protein Kinase Ck2 Inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">2282</span>– <span class="NLM_lpage">2289</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2012.02.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmc.2012.02.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=22377675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A280%3ADC%252BC38zgvFCqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=2282-2289&author=C.+Hundsd%C3%B6rferauthor=H.+J.+Hemmerlingauthor=C.+G%C3%B6tzauthor=F.+Totzkeauthor=P.+Bednarskiauthor=M.+Le+Borgneauthor=J.+Jose&title=Indeno%5B1%2C2-B%5DIndole+Derivatives+as+a+Novel+Class+of+Potent+Human+Protein+Kinase+Ck2+Inhibitors&doi=10.1016%2Fj.bmc.2012.02.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Indeno[1,2-b]indole derivatives as a novel class of potent human protein kinase CK2 inhibitors</span></div><div class="casAuthors">Hundsdorfer Claas; Hemmerling Hans-Jorg; Gotz Claudia; Totzke Frank; Bednarski Patrick; Le Borgne Marc; Jose Joachim</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & medicinal chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2282-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Herein we describe the synthesis and properties of indeno[1,2-b]indole derivatives as a novel class of potent inhibitors of the human protein kinase CK2.  A set of 19 compounds was obtained using a convenient and straightforward synthesis protocol.  The compounds were tested for inhibition of human protein kinase CK2, which was recombinantly expressed in Escherichia coli.  New inhibitors with IC(50) in the micro- and sub-micromolar range were identified.  Compound 4b (5-isopropyl-7,8-dihydroindeno[1,2-b]indole-9,10(5H,6H)-dione) inhibited human CK2 with an IC(50) of 0.11 μM and did not significantly inhibit 22 other human protein kinases, suggesting selectivity towards CK2.  ATP-competitive inhibition by compound 4b was shown and a K(i) of 0.06 μM was determined.  Our findings indicate that indeno[1,2-b]indoles are a promising starting point for further development and optimization of human protein kinase CK2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRFnjW4s3gmBiXXDYhVZBdlfW6udTcc2eYoEWc-TKLm6Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38zgvFCqtQ%253D%253D&md5=e32d29d7c2c80cbc2a16f1dc56743da6</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.02.017%26sid%3Dliteratum%253Aachs%26aulast%3DHundsd%25C3%25B6rfer%26aufirst%3DC.%26aulast%3DHemmerling%26aufirst%3DH.%2BJ.%26aulast%3DG%25C3%25B6tz%26aufirst%3DC.%26aulast%3DTotzke%26aufirst%3DF.%26aulast%3DBednarski%26aufirst%3DP.%26aulast%3DLe%2BBorgne%26aufirst%3DM.%26aulast%3DJose%26aufirst%3DJ.%26atitle%3DIndeno%255B1%252C2-B%255DIndole%2520Derivatives%2520as%2520a%2520Novel%2520Class%2520of%2520Potent%2520Human%2520Protein%2520Kinase%2520Ck2%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2012%26volume%3D20%26spage%3D2282%26epage%3D2289%26doi%3D10.1016%2Fj.bmc.2012.02.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zardecki, C.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=P.+E.+Bourneauthor=J.+Westbrookauthor=C.+Zardecki&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lhdGgLkTKDW7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DZardecki%26aufirst%3DC.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span> <span> </span><span class="NLM_article-title">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6177</span>– <span class="NLM_lpage">6196</span>, <span class="refDoi"> DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+Precision+Glide%3A+Docking+and+Scoring+Incorporating+a+Model+of+Hydrophobic+Enclosure+for+Protein-Ligand+Complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0lj-nRVcMVaLAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520Precision%2520Glide%253A%2520Docking%2520and%2520Scoring%2520Incorporating%2520a%2520Model%2520of%2520Hydrophobic%2520Enclosure%2520for%2520Protein-Ligand%2520Complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+2.+Enrichment+Factors+in+Database+Screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lj-nRVcMVaLAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25202.%2520Enrichment%2520Factors%2520in%2520Database%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R.
A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.%0AA.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lj-nRVcMVaLAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sastry, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowrie, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Systematic Analysis of 2d Fingerprint Methods and Parameters to Improve Virtual Screening Enrichments</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">771</span>– <span class="NLM_lpage">784</span>, <span class="refDoi"> DOI: 10.1021/ci100062n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100062n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlslynu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=771-784&author=M.+Sastryauthor=J.+F.+Lowrieauthor=S.+L.+Dixonauthor=W.+Sherman&title=Large-Scale+Systematic+Analysis+of+2d+Fingerprint+Methods+and+Parameters+to+Improve+Virtual+Screening+Enrichments&doi=10.1021%2Fci100062n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Large-Scale Systematic Analysis of 2D Fingerprint Methods and Parameters to Improve Virtual Screening Enrichments</span></div><div class="casAuthors">Sastry, Madhavi; Lowrie, Jeffrey F.; Dixon, Steven L.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">771-784</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A systematic virtual screening study on 11 pharmaceutically relevant targets has been conducted to investigate the interrelation between 8 two-dimensional (2D) fingerprinting methods, 13 atom-typing schemes, 13 bit scaling rules, and 12 similarity metrics using the new cheminformatics package Canvas.  In total, 157 872 virtual screens were performed to assess the ability of each combination of parameters to identify actives in a database screen.  In general, fingerprint methods, such as MOLPRINT2D, Radial, and Dendritic that encode information about local environment beyond simple linear paths outperformed other fingerprint methods.  Atom-typing schemes with more specific information, such as Daylight, Mol2, and Carhart were generally superior to more generic atom-typing schemes.  Enrichment factors across all targets were improved considerably with the best settings, although no single set of parameters performed optimally on all targets.  The size of the addressable bit space for the fingerprints was also explored, and it was found to have a substantial impact on enrichments.  Small bit spaces, such as 1024, resulted in many collisions and in a significant degrdn. in enrichments compared to larger bit spaces that avoid collisions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH6_ENmYgPPrVg90H21EOLACvtfcHk0ljmnDX4O2sK6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlslynu7k%253D&md5=54e3d9f7e1d493e3b3ee2f153f1af4b2</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fci100062n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100062n%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DM.%26aulast%3DLowrie%26aufirst%3DJ.%2BF.%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DLarge-Scale%2520Systematic%2520Analysis%2520of%25202d%2520Fingerprint%2520Methods%2520and%2520Parameters%2520to%2520Improve%2520Virtual%2520Screening%2520Enrichments%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D771%26epage%3D784%26doi%3D10.1021%2Fci100062n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowrie, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Analysis and Comparison of 2d Fingerprints: Insights into Database Screening Performance Using Eight Fingerprint Methods</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2010.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.jmgm.2010.05.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=20579912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFamtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=157-170&author=J.+Duanauthor=S.+L.+Dixonauthor=J.+F.+Lowrieauthor=W.+Sherman&title=Analysis+and+Comparison+of+2d+Fingerprints%3A+Insights+into+Database+Screening+Performance+Using+Eight+Fingerprint+Methods&doi=10.1016%2Fj.jmgm.2010.05.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis and comparison of 2D fingerprints: Insights into database screening performance using eight fingerprint methods</span></div><div class="casAuthors">Duan, Jian-Xin; Dixon, Steven L.; Lowrie, Jeffrey F.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-170</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Virtual screening is a widely used strategy in modern drug discovery and 2D fingerprint similarity is an important tool that has been successfully applied to retrieve active compds. from large datasets.  However, it is not always straightforward to select an appropriate fingerprint method and assocd. settings for a given problem.  Here, we applied eight different fingerprint methods, as implemented in the new cheminformatics package Canvas, on a well-validated dataset covering five targets.  The fingerprint methods include Linear, Dendritic, Radial, MACCS, MOLPRINT2D, Pairwise, Triplet, and Torsion.  We find that most fingerprints have similar retrieval rates on av.; however, each has special characteristics that distinguish its performance on different query mols. and ligand sets.  For example, some fingerprints exhibit a significant ligand size dependency whereas others are more robust with respect to variations in the query or active compds.  In cases where little information is known about the active ligands, MOLPRINT2D fingerprints produce the highest av. retrieval actives.  When multiple queries are available, we find that a fingerprint averaged over all query mols. is generally superior to fingerprints derived from single queries.  Finally, a complementarity metric is proposed to det. which fingerprint methods can be combined to improve screening results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWCcy4qE10YbVg90H21EOLACvtfcHk0ljmnDX4O2sK6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFamtLrM&md5=f72d91018346b1e5ef97fc83f648bd86</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2010.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2010.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DJ.%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DLowrie%26aufirst%3DJ.%2BF.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DAnalysis%2520and%2520Comparison%2520of%25202d%2520Fingerprints%253A%2520Insights%2520into%2520Database%2520Screening%2520Performance%2520Using%2520Eight%2520Fingerprint%2520Methods%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2010%26volume%3D29%26spage%3D157%26epage%3D170%26doi%3D10.1016%2Fj.jmgm.2010.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, K. J.</span>; <span class="NLM_string-name">Chow, D. E.</span>; <span class="NLM_string-name">Xu, H.</span>; <span class="NLM_string-name">Dror, R. O.</span>; <span class="NLM_string-name">Eastwood, M. P.</span>; <span class="NLM_string-name">Gregersen, B. A.</span>; <span class="NLM_string-name">Klepeis, J. L.</span>; <span class="NLM_string-name">Kolossvary, I.</span>; <span class="NLM_string-name">Moraes, M. A.</span>; <span class="NLM_string-name">Sacerdoti, F. D.</span>; <span class="NLM_string-name">Salmon, J. K.</span>; <span class="NLM_string-name">Shan, Y.</span>; <span class="NLM_string-name">Shaw, D. E.</span></span> <span> </span><span class="NLM_article-title">Scalable Algorithms for Molecular Dynamics Simulations on Commodity Clusters</span>. In  <i>Sc ’06: Proceedings of the 2006 Acm/Ieee Conference on Supercomputing</i>; <span class="NLM_publisher-name">IEEE</span>: <span class="NLM_publisher-loc">Tampa, FL</span>, <span class="NLM_year">2006</span>; p  <span class="NLM_fpage">746</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1109%2FSC.2006.54" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=746&author=K.+J.+Bowers&author=D.+E.+Chow&author=H.+Xu&author=R.+O.+Dror&author=M.+P.+Eastwood&author=B.+A.+Gregersen&author=J.+L.+Klepeis&author=I.+Kolossvary&author=M.+A.+Moraes&author=F.+D.+Sacerdoti&author=J.+K.+Salmon&author=Y.+Shan&author=D.+E.+Shaw&title=Sc+%E2%80%9906%3A+Proceedings+of+the+2006+Acm%2FIeee+Conference+on+Supercomputing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1109%2FSC.2006.54&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1109%252FSC.2006.54%26sid%3Dliteratum%253Aachs%26aulast%3DBowers%26aufirst%3DK.%2BJ.%26atitle%3DScalable%2520Algorithms%2520for%2520Molecular%2520Dynamics%2520Simulations%2520on%2520Commodity%2520Clusters%26btitle%3DSc%2520%25E2%2580%259906%253A%2520Proceedings%2520of%2520the%25202006%2520Acm%252FIeee%2520Conference%2520on%2520Supercomputing%26pub%3DIEEE%26date%3D2006%26spage%3D746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haddach, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierre, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regan, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borsan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerdoncuff, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaebe, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui-Jain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macalino, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drygin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryckman, D. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Sar of Inhibitors of Protein Kinase Ck2: Novel Tricyclic Quinoline Analogs</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.11.087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2011.11.087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=22169261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC38XivVymsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=45-48&author=M.+Haddachauthor=F.+Pierreauthor=C.+F.+Reganauthor=C.+Borsanauthor=J.+Michauxauthor=E.+Stefanauthor=P.+Kerdoncuffauthor=M.+K.+Schwaebeauthor=P.+C.+Chuaauthor=A.+Siddiqui-Jainauthor=D.+Macalinoauthor=D.+Dryginauthor=S.+E.+O%E2%80%99Brienauthor=W.+G.+Riceauthor=D.+M.+Ryckman&title=Synthesis+and+Sar+of+Inhibitors+of+Protein+Kinase+Ck2%3A+Novel+Tricyclic+Quinoline+Analogs&doi=10.1016%2Fj.bmcl.2011.11.087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and SAR of inhibitors of protein kinase CK2: Novel tricyclic quinoline analogs</span></div><div class="casAuthors">Haddach, Mustapha; Pierre, Fabrice; Regan, Collin F.; Borsan, Cosmin; Michaux, Jerome; Stefan, Eric; Kerdoncuff, Pauline; Schwaebe, Michael K.; Chua, Peter C.; Siddiqui-Jain, Adam; Macalino, Diwata; Drygin, Denis; O'Brien, Sean E.; Rice, William G.; Ryckman, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-48</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein kinase CK2 is a potential drug target for many diseases including cancer and inflammation disorders.  The crystal structure of clin. candidate CX-4945 with CK2 revealed an indirect interaction with the protein through hydrogen bonding between the NH of the 3-chlorophenylamine and a water mol.  The relevance of this hydrogen bond is investigated by prepn. of several novel tricyclic derivs. lacking an NH moiety at the same position.  This SAR study allowed the discovery of 5-arylbenzo[c][2,6]naphthyridine-8-carboxylic acids as highly potent CK2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXez4R82fuOLVg90H21EOLACvtfcHk0lg676_J3fYegQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivVymsw%253D%253D&md5=f329788c09ed192ac49b8c1a02342806</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.11.087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.11.087%26sid%3Dliteratum%253Aachs%26aulast%3DHaddach%26aufirst%3DM.%26aulast%3DPierre%26aufirst%3DF.%26aulast%3DRegan%26aufirst%3DC.%2BF.%26aulast%3DBorsan%26aufirst%3DC.%26aulast%3DMichaux%26aufirst%3DJ.%26aulast%3DStefan%26aufirst%3DE.%26aulast%3DKerdoncuff%26aufirst%3DP.%26aulast%3DSchwaebe%26aufirst%3DM.%2BK.%26aulast%3DChua%26aufirst%3DP.%2BC.%26aulast%3DSiddiqui-Jain%26aufirst%3DA.%26aulast%3DMacalino%26aufirst%3DD.%26aulast%3DDrygin%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DS.%2BE.%26aulast%3DRice%26aufirst%3DW.%2BG.%26aulast%3DRyckman%26aufirst%3DD.%2BM.%26atitle%3DSynthesis%2520and%2520Sar%2520of%2520Inhibitors%2520of%2520Protein%2520Kinase%2520Ck2%253A%2520Novel%2520Tricyclic%2520Quinoline%2520Analogs%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D45%26epage%3D48%26doi%3D10.1016%2Fj.bmcl.2011.11.087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shivakumar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Prediction of Absolute Solvation Free Energies Using Molecular Dynamics Free Energy Perturbation and the Opls Force Field</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1509</span>– <span class="NLM_lpage">1519</span>, <span class="refDoi"> DOI: 10.1021/ct900587b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct900587b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkslKhu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=1509-1519&author=D.+Shivakumarauthor=J.+Williamsauthor=Y.+Wuauthor=W.+Dammauthor=J.+Shelleyauthor=W.+Sherman&title=Prediction+of+Absolute+Solvation+Free+Energies+Using+Molecular+Dynamics+Free+Energy+Perturbation+and+the+Opls+Force+Field&doi=10.1021%2Fct900587b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Absolute Solvation Free Energies using Molecular Dynamics Free Energy Perturbation and the OPLS Force Field</span></div><div class="casAuthors">Shivakumar, Devleena; Williams, Joshua; Wu, Yujie; Damm, Wolfgang; Shelley, John; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1509-1519</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accurate prediction of protein-ligand binding free energies is a primary objective in computer-aided drug design.  The solvation free energy of a small mol. provides a surrogate to the desolvation of the ligand in the thermodn. process of protein-ligand binding.  Here, we use explicit solvent mol. dynamics free energy perturbation to predict the abs. solvation free energies of a set of 239 small mols., spanning diverse chem. functional groups commonly found in drugs and drug-like mols.  We also compare the performance of abs. solvation free energies obtained using the OPLS_2005 force field with two other commonly used small mol. force fields - general AMBER force field (GAFF) with AM1-BCC charges and CHARMm-MSI with CHelpG charges.  Using the OPLS_2005 force field, we obtain high correlation with exptl. solvation free energies (R2 = 0.94) and low av. unsigned errors for a majority of the functional groups compared to AM1-BCC/GAFF or CHelpG/CHARMm-MSI.  However, OPLS_2005 has errors of over 1.3 kcal/mol for certain classes of polar compds.  We show that predictions on these compd. classes can be improved by using a semiempirical charge assignment method with an implicit bond charge correction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon_fkbSHqg3bVg90H21EOLACvtfcHk0lg676_J3fYegQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkslKhu7g%253D&md5=25d464fb9238e3235e881945c0b50a76</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fct900587b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct900587b%26sid%3Dliteratum%253Aachs%26aulast%3DShivakumar%26aufirst%3DD.%26aulast%3DWilliams%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DShelley%26aufirst%3DJ.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DPrediction%2520of%2520Absolute%2520Solvation%2520Free%2520Energies%2520Using%2520Molecular%2520Dynamics%2520Free%2520Energy%2520Perturbation%2520and%2520the%2520Opls%2520Force%2520Field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D1509%26epage%3D1519%26doi%3D10.1021%2Fct900587b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Schrödinger
LLC</span>.  <i>Schrödinger Release 2014-4:
Watermap</i>; <span class="NLM_publisher-name">Schrödinger LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=Schr%C3%B6dinger%0ALLC&title=Schr%C3%B6dinger+Release+2014-4%3A%0AWatermap"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DSchr%25C3%25B6dinger%2520Release%25202014-4%253A%250AWatermap%26pub%3DSchr%25C3%25B6dinger%2520LLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Clinical
and
Laboratory Standards Institute</span>.  <i>M07-A10:
Methods for Dilution Antimicrobial Susceptibility Tests for Bacteria
That Grow Aerobically; Approved Standard</i>; <span class="NLM_publisher-name">Clinical and Laboratory Standards Institute</span>: <span class="NLM_publisher-loc">Wayne, PA</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=Clinical%0Aand%0ALaboratory+Standards+Institute&title=M07-A10%3A%0AMethods+for+Dilution+Antimicrobial+Susceptibility+Tests+for+Bacteria%0AThat+Grow+Aerobically%3B+Approved+Standard"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DM07-A10%253A%250AMethods%2520for%2520Dilution%2520Antimicrobial%2520Susceptibility%2520Tests%2520for%2520Bacteria%250AThat%2520Grow%2520Aerobically%253B%2520Approved%2520Standard%26pub%3DClinical%2520and%2520Laboratory%2520Standards%2520Institute%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Clinical and Laboratory Standards
Institute</span>.  <i>M11-A8: Methods for Antimicrobial
Susceptibility Testing of Anaerobic Bacteria; Approved Standard</i>; <span class="NLM_publisher-name">Clinical and Laboratory Standards Institute</span>: <span class="NLM_publisher-loc">Wayne, PA</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=Clinical+and+Laboratory+Standards%0AInstitute&title=M11-A8%3A+Methods+for+Antimicrobial%0ASusceptibility+Testing+of+Anaerobic+Bacteria%3B+Approved+Standard"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DM11-A8%253A%2520Methods%2520for%2520Antimicrobial%250ASusceptibility%2520Testing%2520of%2520Anaerobic%2520Bacteria%253B%2520Approved%2520Standard%26pub%3DClinical%2520and%2520Laboratory%2520Standards%2520Institute%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nociari, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shalev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benias, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, C.</span></span> <span> </span><span class="NLM_article-title">A Novel One-Step, Highly Sensitive Fluorometric Assay to Evaluate Cell-Mediated Cytotoxicity</span>. <i>J. Immunol. Methods</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>213</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1016/S0022-1759(98)00028-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2FS0022-1759%2898%2900028-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=9692848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADyaK1cXjvVSksb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=213&publication_year=1998&pages=157-167&author=M.+M.+Nociariauthor=A.+Shalevauthor=P.+Beniasauthor=C.+Russo&title=A+Novel+One-Step%2C+Highly+Sensitive+Fluorometric+Assay+to+Evaluate+Cell-Mediated+Cytotoxicity&doi=10.1016%2FS0022-1759%2898%2900028-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">A novel one-step, highly sensitive fluorometric assay to evaluate cell-mediated cytotoxicity</span></div><div class="casAuthors">Nociari, Marcelo M.; Shalev, Assaf; Benias, Petros; Russo, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunological Methods</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">213</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-167</span>CODEN:
                <span class="NLM_cas:coden">JIMMBG</span>;
        ISSN:<span class="NLM_cas:issn">0022-1759</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">In this study, a fluorometric method using alamarBlue has been developed for detecting cell-mediated cytotoxicity in vitro.  AlamarBlue is a non-toxic metabolic indicator of viable cells that becomes fluorescent upon mitochondrial redn.  Specific lysis of targets by effector cells is quantified by comparing the total no. of viable cells in wells contg. effector and targets together, vs. with wells where target and effector cells were sep. seeded.  Cell-mediated cytotoxic activity by alloreactive T cells and natural killer cells has been detected using a novel application of the alamarBlue technique.  The assay that we have developed to detect cell-mediated cytotoxicity is extremely sensitive and specific and requires a significant lower no. of effector cells than the std. 51Cr assay.  Since alamarBlue reagent is non-toxic to cells and the assay can be performed under sterile conditions, effector cells may be recovered at the end for further anal. or cell expansion, if desired.  Direct comparison of cell-mediated cytotoxicity measured by the alamarBlue method with the std. 51Cr release assay revealed that the former method is as specific and more sensitive than the conventional assay.  Moreover, very small inter and intra-assay variations have been obsd. for alamarBlue cytotoxicity assays.  In conclusion, this study shows that the alamarBlue assay is an extremely sensitive, economical, simple and non-toxic procedure to evaluate cell-mediated cytotoxicity that yields accurate results using a limited no. of effector cells.  Furthermore, since this assay is a one-step procedure, and does not involve any risk for the personnel, it may be useful to analyze automatically cell-mediated cytotoxicity in a large no. of samples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq03rcpA2J96bVg90H21EOLACvtfcHk0lir3mF6jVEluA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjvVSksb4%253D&md5=9bce12d254e546d7a41ec2a152abd4cb</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2FS0022-1759%2898%2900028-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-1759%252898%252900028-3%26sid%3Dliteratum%253Aachs%26aulast%3DNociari%26aufirst%3DM.%2BM.%26aulast%3DShalev%26aufirst%3DA.%26aulast%3DBenias%26aufirst%3DP.%26aulast%3DRusso%26aufirst%3DC.%26atitle%3DA%2520Novel%2520One-Step%252C%2520Highly%2520Sensitive%2520Fluorometric%2520Assay%2520to%2520Evaluate%2520Cell-Mediated%2520Cytotoxicity%26jtitle%3DJ.%2520Immunol.%2520Methods%26date%3D1998%26volume%3D213%26spage%3D157%26epage%3D167%26doi%3D10.1016%2FS0022-1759%2898%2900028-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mathes, C.</span></span> <span> </span><span class="NLM_article-title">Qpatch: The Past, Present and Future of Automated Patch Clamp</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1517/14728222.10.2.319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1517%2F14728222.10.2.319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=16548779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD28XivVOmt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2006&pages=319-327&author=C.+Mathes&title=Qpatch%3A+The+Past%2C+Present+and+Future+of+Automated+Patch+Clamp&doi=10.1517%2F14728222.10.2.319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">QPatch: the past, present and future of automated patch clamp</span></div><div class="casAuthors">Mathes, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">319-327</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Ashley Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  The QPatch 16 significantly increases throughput for gigaseal patch clamp expts., making direct measurements in ion channel drug discovery and safety testing feasible.  Released to the market in the Autumn of 2004 by Sophion Bioscience, the QPatch originated from work done at NeuroSearch (Denmark) in the early days of automated patch clamp.  Today, the QPatch provides many unique features.  For example, only the QPatch includes an automated cell prepn. station making several hours of unattended operation possible.  The 16-channel electrode array, called the QPlate, includes glass-coated microfluidic channels for less compd. absorption and, hence, more accurate IC50 values.  The microfluidic pathways also allow for very small amts. of compd. used for each expt. (∼ 5 μl per addn.).  Only the QPatch has four independent pipetting heads for more efficient liq. handling (esp. for ligand-gated ion channel expts.).  Patch clamp recordings with the QPatch match the high quality of conventional patch clamp and in some cases the results are even better.  For example, only the QPatch includes 100% series resistance compensation for the elimination of false positives due to voltage errors.  Finally, the modular QPatch 16 was designed with more channels in mind.  The upgrade pathway to 48-channels (the QPatch HT) will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqywPZMP8PNwrVg90H21EOLACvtfcHk0lir3mF6jVEluA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivVOmt7k%253D&md5=7dcd02a69652955e261f6928fbb76b81</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1517%2F14728222.10.2.319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.10.2.319%26sid%3Dliteratum%253Aachs%26aulast%3DMathes%26aufirst%3DC.%26atitle%3DQpatch%253A%2520The%2520Past%252C%2520Present%2520and%2520Future%2520of%2520Automated%2520Patch%2520Clamp%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2006%26volume%3D10%26spage%3D319%26epage%3D327%26doi%3D10.1517%2F14728222.10.2.319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yanran Lu, Jonathan L. Papa, Sheri Nolan, Anthony English, Justin T. Seffernick, Nicholas Shkolnikov, Josh Powell, Steffen Lindert, Daniel J. Wozniak, Jack Yalowich, <span class="NLM_string-name hlFld-ContribAuthor">Mark J. Mitton-Fry</span>. </span><span class="cited-content_cbyCitation_article-title">Dioxane-Linked Amide Derivatives as Novel Bacterial Topoisomerase Inhibitors against Gram-Positive Staphylococcus aureus. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (12)
                                     , 2446-2454. <a href="https://doi.org/10.1021/acsmedchemlett.0c00428" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00428</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00428%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDioxane-Linked%252BAmide%252BDerivatives%252Bas%252BNovel%252BBacterial%252BTopoisomerase%252BInhibitors%252Bagainst%252BGram-Positive%252BStaphylococcus%252Baureus%26aulast%3DLu%26aufirst%3DYanran%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D04082020%26date%3D14102020%26date%3D19102020%26volume%3D11%26issue%3D12%26spage%3D2446%26epage%3D2454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Elizabeth
G. Gibson, Alexandria A. Oviatt, Monica Cacho, Keir C. Neuman, Pan F. Chan, <span class="NLM_string-name hlFld-ContribAuthor">Neil Osheroff</span>. </span><span class="cited-content_cbyCitation_article-title">Bimodal Actions of a Naphthyridone/Aminopiperidine-Based Antibacterial That Targets Gyrase and Topoisomerase IV. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2019,</strong> <em>58 </em>
                                    (44)
                                     , 4447-4455. <a href="https://doi.org/10.1021/acs.biochem.9b00805" title="DOI URL">https://doi.org/10.1021/acs.biochem.9b00805</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.9b00805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.9b00805%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DBimodal%252BActions%252Bof%252Ba%252BNaphthyridone%25252FAminopiperidine-Based%252BAntibacterial%252BThat%252BTargets%252BGyrase%252Band%252BTopoisomerase%252BIV%26aulast%3DGibson%26aufirst%3DElizabeth%2BG.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D02092019%26date%3D10102019%26date%3D28102019%26date%3D16102019%26volume%3D58%26issue%3D44%26spage%3D4447%26epage%3D4455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rajkumar  Kommera</span>, <span class="hlFld-ContribAuthor ">China Raju  Bhimapaka</span>. </span><span class="cited-content_cbyCitation_article-title">A simple and efficient approach for the preparation of dihydroxanthyletin, xanthyletin, decursinol and marmesin. </span><span class="cited-content_cbyCitation_journal-name">Synthetic Communications</span><span> <strong>2020,</strong> <em>50 </em>
                                    (21)
                                     , 3204-3211. <a href="https://doi.org/10.1080/00397911.2020.1797812" title="DOI URL">https://doi.org/10.1080/00397911.2020.1797812</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/00397911.2020.1797812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F00397911.2020.1797812%26sid%3Dliteratum%253Aachs%26jtitle%3DSynthetic%2520Communications%26atitle%3DA%252Bsimple%252Band%252Befficient%252Bapproach%252Bfor%252Bthe%252Bpreparation%252Bof%252Bdihydroxanthyletin%25252C%252Bxanthyletin%25252C%252Bdecursinol%252Band%252Bmarmesin%26aulast%3DKommera%26aufirst%3DRajkumar%26date%3D2020%26date%3D2020%26volume%3D50%26issue%3D21%26spage%3D3204%26epage%3D3211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0028.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0028.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. NBTI structures reported in the literature for (A) NXL101, (B) GSK2140944, (C) NBTI5463, (D) ACT-051, (E) AM-8722, (F) REDX06213, and (G) GSK299423 and most relevant compounds <b>(±)-33</b>, <b>(±)-35</b>, and <b>(±)-36</b> of the present study.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0027.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0027.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Amino acid sequence alignment of the binding sites of the DNA gyrase and topoisomerase IV enzymes in <i>S. aureus</i> and <i>E. coli</i>. The color scheme for the residues is as follows: blue for hydrophobic residues; red for acidic residues; green-yellow for basic residues; orange for other residues. The “Weight Colours by Alignment Quality” option of Schrödinger Release 2017-1: Prime, Schrödinger, LLC, New York, NY, was used for sequence alignment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0026.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0026.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Multistep approach of the Angelini NBTI drug discovery program.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0025.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0025.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Graph showing enrichment factors for standard precision (blue line) docking of the dataset (44 compounds) into the <i>S. aureus</i> DNA gyrase model based on PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>. The red and green lines represent perfect and random ligand retrieval, respectively. (B) Enrichment factor data in tabular form.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0024.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Three structural motifs of NBTI compounds (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BUL">4BUL</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0023.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Graph showing the enrichment factors of the selected pharmacophore models AADRR (purple) and AAPRR (turquoise). The red and green lines represent perfect and random ligand retrieval, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0022.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Design strategy for compound <b>1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0021.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. X-ray crystal structure of <b>ID-130</b> (represented as a yellow tube) in the <i>S. aureus</i> DNA gyrase site (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FM4">6FM4</a>). The DNA strand is shown as a green and blue tube structure; the subunits of the protein are shown as green and blue ribbons. Hydrogen bonds are represented as yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0002.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>1</b>–<b>4</b>, <b>6</b>, <b>11</b>, <b>12</b>, <b>14</b>–<b>23</b>, <b>27</b>, <b>28</b>, and <b>30</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert-</i>butyl piperazine-1-carboxylate, K<sub>2</sub>CO<sub>3</sub>, DMSO, 110 °C; (b) TFA, DCM, rt; (c) <i>tert-</i>butyl (3-bromopropyl)carbamate (for intermediates <b>IIIa</b>–<b>IIIg</b>) or <i>tert-</i>butyl (2-bromoethyl)carbamate (for intermediate <b>IIIh</b>), KI, K<sub>2</sub>CO<sub>3</sub>, DMF, rt; or <i>tert</i>-butyl (3-oxocyclobutyl)carbamate, NaBH(OAc)<sub>3</sub>, DCM, rt, for intermediates <b>IIIi</b> and <b>IIIj</b>; (d) NaH, CH<sub>3</sub>I, DMF, rt; (e) triflic anhydride, TEA, DCM, −10 °C and then rt; (f) TEA, CH<sub>3</sub>CN, 70 °C, followed by Boc deprotection under standard conditions for compound <b>19</b>.</p></p></figure><figure data-id="sch2" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0003.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>5</b>, <b>7</b>–<b>10</b>, and <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert</i>-butyl 4-(3-aminopropyl)piperazine-1-carboxylate, TEA, CH<sub>3</sub>CN, 70 °C; (b) TFA, DCM, rt; (c) K<sub>2</sub>CO<sub>3</sub>, DMSO, 110 °C.</p></p></figure><figure data-id="sch3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0004.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1-(3-hydroxy-1-propyl)piperazine, K<sub>2</sub>CO<sub>3</sub>, DMSO, 120 °C, MW; (b) CH<sub>3</sub>SO<sub>2</sub>Cl, TEA, DCM, 0 °C to rt; (c) 4-hydroxy-2<i>H</i>-1-benzopyran-2-one, NaH, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 80 °C.</p></p></figure><figure data-id="sch4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0005.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compound <b>25</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 3-(piperazin-1-yl)propanamide, K<sub>2</sub>CO<sub>3</sub>, DMSO, 110 °C; (b) <b>VIa</b>, K<sub>3</sub>PO<sub>4</sub>, Pd(dba)<sub>2</sub>, Xantphos, dry dioxane, 100 °C.</p></p></figure><figure data-id="sch5" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0006.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compound <b>26</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) piperazin-2-one, K<sub>2</sub>CO<sub>3</sub>, DMSO, 110 °C; (b) <i>tert</i>-butyl <i>N</i>-(3-bromopropyl)carbamate, NaH, DMF, rt; (c) TFA, DCM, rt; (d) <b>VIa</b>, TEA, CH<sub>3</sub>CN, 70 °C.</p></p></figure><figure data-id="sch6" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0007.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compounds <b>(<i>S</i>)-29</b>, <b>(<i>R</i>)-29</b>, <b>(<i>R</i>)-34</b>, <b>(±)-35</b>, and <b>(±)-36</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (2<i>R</i>)-<i>N</i>-(2,3-epoxypropyl)phthalimide or (2<i>S</i>)-<i>N</i>-(2,3-epoxypropyl)phthalimide, CH<sub>3</sub>CN, 70 °C; (b) MeNH<sub>2</sub>, EtOH, 40 °C; (c) epichlorohydrin, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), EtOH, rt; (d) NaN<sub>3</sub>, DMF, 85 °C; (e) H<sub>2</sub>, 10% Pd/C, EtOH, rt; (f) <b>VIa</b> or <b>XVIa</b>, TEA, CH<sub>3</sub>CN, 70 °C; (g) <b>XVIb</b>, DIPEA, CH<sub>3</sub>CN, 0 °C to rt; (h) PCl<sub>5</sub>, DCE, 0 °C to reflux. Compounds <b>(<i>S</i>)-29</b>, <b>(<i>R</i>)-29</b>, and <b>(<i>R</i>)-34</b> were obtained following steps (a), (b), and (f); racemic compounds <b>(±)-35</b> and <b>(±)-36</b> were obtained following steps (c)–(e) and (g).</p></p></figure><figure data-id="sch7" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0008.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Compound <b>(±)-31</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) LiHDMS, 3-chloropropanoyl chloride, dry THF, −78 °C to rt; (b) 7-fluoro-1-(piperazin-1-yl)isoquinoline (<b>IIb</b>), K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 45 °C; (c) NaBH<sub>4</sub>, MeOH, rt.</p></p></figure><figure data-id="sch8" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0009.jpeg" id="rightTab-gr20" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of Compounds <b>(±)-32</b> and <b>(±)-33</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TEA, CH<sub>3</sub>CN, 70 °C; (b) HCl in dioxane, MeOH, rt; (c) 5-chloro-1,6-naphthyridine, K<sub>2</sub>CO<sub>3</sub>, DMSO, 110 °C for <b>(±)-32</b>; or 1-chloro-3-fluoro-7-methoxyisoquinoline, DIPEA, DMSO, 100 °C for <b>(±)-33</b>.</p></p></figure><figure data-id="fig9" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0020.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Compound <b>1</b> was modeled into the <i>S. aureus</i> DNA gyrase crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>). The LHS portion of the ligand is represented in a ball-and-stick model. The red surface shows the free-energy density of the solvent calculated by WaterMap, together with the WaterMap-generated hydration sites (red and green spheres). Red spheres indicate unstable water molecules with positive Δ<i>G</i> values, and green spheres indicate stable water molecules with negative Δ<i>G</i> values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0019.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Compound <b>1</b> was modeled into the <i>S. aureus</i> DNA gyrase crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>). The RHS portion of the ligand is represented in a ball-and-stick model. The red surface shows the free-energy density of the solvent calculated by WaterMap, together with the WaterMap-generated hydration sites (red and green spheres). The gray surface represents the extent of the protein from the 2XCS structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0018.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. (A) Representative snapshot from the simulation of compound <b>2</b> in complex with <i>S. aureus</i> DNA gyrase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>). The DNA strand is shown as a red “ladder” structure, and the protein is shown as a blue ribbon. Hydrogen bonds are represented as dashed lines. (B) Simulation interaction diagram summarizing the protein–ligand interactions across the entire simulation of compound <b>2</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/medium/jm-2019-00394m_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0017.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. (A) Representative snapshot from the simulation of compound <b>35</b> in complex with <i>S. aureus</i> DNA gyrase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS">2XCS</a>). The DNA strand is shown as a red ladder structure, and the protein is shown as a blue ribbon. Hydrogen bonds are represented as dashed lines. (B) Simulation interaction diagram summarizing the protein–ligand interactions across the entire simulation of compound <b>35</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00394/20190815/images/large/jm-2019-00394m_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00394&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i100">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68420" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68420" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 65 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">WHO</span>.  <i>Global Priority List of Antibiotic-Resistant Bacteria to
Guide Research, Discovery, and Development of New Antibiotics</i>. <span class="NLM_publisher-name">WHO</span>: <span class="NLM_publisher-loc">Geneva, Switzerland</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=WHO&title=Global+Priority+List+of+Antibiotic-Resistant+Bacteria+to%0AGuide+Research%2C+Discovery%2C+and+Development+of+New+Antibiotics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DGlobal%2520Priority%2520List%2520of%2520Antibiotic-Resistant%2520Bacteria%2520to%250AGuide%2520Research%252C%2520Discovery%252C%2520and%2520Development%2520of%2520New%2520Antibiotics%26pub%3DWHO%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhanel, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeCorby, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvey, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCracken, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagacé-Wiens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nichol, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wierzbowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baudry, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tailor, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlowsky, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walkty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweizer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoban, D. J.</span></span>; <span class="NLM_contrib-group">Canadian Antimicrobial Resistance Alliance</span> <span> </span><span class="NLM_article-title">Prevalence of Antimicrobial-Resistant Pathogens in Canadian Hospitals: Results of the Canadian Ward Surveillance Study (Canward 2008)</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">4684</span>– <span class="NLM_lpage">4693</span>, <span class="refDoi"> DOI: 10.1128/AAC.00469-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.00469-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=20805395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVCitrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2010&pages=4684-4693&author=G.+G.+Zhanelauthor=M.+DeCorbyauthor=H.+Adamauthor=M.+R.+Mulveyauthor=M.+McCrackenauthor=P.+Lagac%C3%A9-Wiensauthor=K.+A.+Nicholauthor=A.+Wierzbowskiauthor=P.+J.+Baudryauthor=F.+Tailorauthor=J.+A.+Karlowskyauthor=A.+Walktyauthor=F.+Schweizerauthor=J.+Johnsonauthor=D.+J.+Hobanauthor=Canadian+Antimicrobial+Resistance+Alliance&title=Prevalence+of+Antimicrobial-Resistant+Pathogens+in+Canadian+Hospitals%3A+Results+of+the+Canadian+Ward+Surveillance+Study+%28Canward+2008%29&doi=10.1128%2FAAC.00469-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence of antimicrobial-resistant pathogens in Canadian hospitals: results of the Canadian ward surveillance study (CANWARD 2008)</span></div><div class="casAuthors">Zhanel, George G.; DeCorby, Melanie; Adam, Heather; Mulvey, Michael R.; McCracken, Melissa; Lagace-Wiens, Philippe; Nichol, Kimberly A.; Wierzbowski, Aleksandra; Baudry, Patricia J.; Tailor, Franil; Karlowsky, James A.; Walkty, Andrew; Schweizer, Frank; Johnson, Jack; Hoban, Daryl J.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4684-4693</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A total of 5,282 bacterial isolates obtained between 1 Jan. and 31 Dec. 31 2008, inclusive, from patients in 10 hospitals across Canada as part of the Canadian Ward Surveillance Study (CANWARD 2008) underwent susceptibility testing.  The 10 most common organisms, representing 78.8% of all clin. specimens, were as follows: Escherichia coli (21.4%), methicillin-susceptible Staphylococcus aureus (MSSA; 13.9%), Streptococcus pneumoniae (10.3%), Pseudomonas aeruginosa (7.1%), Klebsiella pneumoniae (6.0%), coagulase-neg. staphylococci/Staphylococcus epidermidis (5.4%), methicillin-resistant S. aureus (MRSA; 5.1%), Haemophilus influenzae (4.1%), Enterococcus spp. (3.3%), Enterobacter cloacae (2.2%).  MRSA comprised 27.0% (272/1,007) of all S. aureus isolates (genotypically, 68.8% of MRSA were health care assocd. [HA-MRSA] and 27.6% were community assocd. [CA-MRSA]).  Extended-spectrum β-lactamase (ESBL)-producing E. coli occurred in 4.9% of E. coli isolates.  The CTX-M type was the predominant ESBL, with CTX-M-15 the most prevalent genotype.  MRSA demonstrated no resistance to ceftobiprole, daptomycin, linezolid, telavancin, tigecycline, or vancomycin (0.4% intermediate intermediate resistance).  E. coli demonstrated no resistance to ertapenem, meropenem, or tigecycline.  Resistance rates with P. aeruginosa were as follows: colistin (polymyxin E), 0.8%; amikacin, 3.5%; cefepime, 7.2%; gentamicin, 12.3%; fluoroquinolones, 19.0 to 24.1%; meropenem, 5.6%; piperacillin-tazobactam, 8.0%.  A multidrug-resistant (MDR) phenotype occurred frequently in P. aeruginosa (5.9%) but uncommonly in E. coli (1.2%) and K. pneumoniae (0.9%).  In conclusion, E. coli, S. aureus (MSSA and MRSA), P. aeruginosa, S. pneumoniae, K. pneumoniae, H. influenzae, and Enterococcus spp. are the most common isolates recovered from clin. specimens in Canadian hospitals.  The prevalence of MRSA was 27.0% (of which genotypically 27.6% were CA-MRSA), while ESBL-producing E. coli occurred in 4.9% of isolates.  An MDR phenotype was common in P. aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzoXX--1oar7Vg90H21EOLACvtfcHk0lgcWwrGmwQ7Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVCitrzF&md5=2cfe5fe769b68c8b375bf07a4a26a2d8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1128%2FAAC.00469-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00469-10%26sid%3Dliteratum%253Aachs%26aulast%3DZhanel%26aufirst%3DG.%2BG.%26aulast%3DDeCorby%26aufirst%3DM.%26aulast%3DAdam%26aufirst%3DH.%26aulast%3DMulvey%26aufirst%3DM.%2BR.%26aulast%3DMcCracken%26aufirst%3DM.%26aulast%3DLagac%25C3%25A9-Wiens%26aufirst%3DP.%26aulast%3DNichol%26aufirst%3DK.%2BA.%26aulast%3DWierzbowski%26aufirst%3DA.%26aulast%3DBaudry%26aufirst%3DP.%2BJ.%26aulast%3DTailor%26aufirst%3DF.%26aulast%3DKarlowsky%26aufirst%3DJ.%2BA.%26aulast%3DWalkty%26aufirst%3DA.%26aulast%3DSchweizer%26aufirst%3DF.%26aulast%3DJohnson%26aufirst%3DJ.%26aulast%3DHoban%26aufirst%3DD.%2BJ.%26aulast%3D%26atitle%3DPrevalence%2520of%2520Antimicrobial-Resistant%2520Pathogens%2520in%2520Canadian%2520Hospitals%253A%2520Results%2520of%2520the%2520Canadian%2520Ward%2520Surveillance%2520Study%2520%2528Canward%25202008%2529%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2010%26volume%3D54%26spage%3D4684%26epage%3D4693%26doi%3D10.1128%2FAAC.00469-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laxminarayan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wattal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaidi, A. K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertheim, H. F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sumpradit, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vlieghe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gould, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greko, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">So, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigdeli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodhouse, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ombaka, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peralta, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qamar, F. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mir, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kariuki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhutta, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coates, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergstrom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cars, O.</span></span> <span> </span><span class="NLM_article-title">Antibiotic Resistance—The Need for Global Solutions</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1057</span>– <span class="NLM_lpage">1098</span>, <span class="refDoi"> DOI: 10.1016/S1473-3099(13)70318-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2FS1473-3099%2813%2970318-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=24252483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A280%3ADC%252BC2c7os1Gntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=1057-1098&author=R.+Laxminarayanauthor=A.+Duseauthor=C.+Wattalauthor=A.+K.+M.+Zaidiauthor=H.+F.+L.+Wertheimauthor=N.+Sumpraditauthor=E.+Vliegheauthor=G.+L.+Haraauthor=I.+M.+Gouldauthor=H.+Goossensauthor=C.+Grekoauthor=A.+D.+Soauthor=M.+Bigdeliauthor=G.+Tomsonauthor=W.+Woodhouseauthor=E.+Ombakaauthor=A.+Q.+Peraltaauthor=F.+N.+Qamarauthor=F.+Mirauthor=S.+Kariukiauthor=Z.+A.+Bhuttaauthor=A.+Coatesauthor=R.+Bergstromauthor=G.+D.+Wrightauthor=E.+D.+Brownauthor=O.+Cars&title=Antibiotic+Resistance%E2%80%94The+Need+for+Global+Solutions&doi=10.1016%2FS1473-3099%2813%2970318-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic resistance-the need for global solutions</span></div><div class="casAuthors">Laxminarayan Ramanan; Duse Adriano; Wattal Chand; Zaidi Anita K M; Wertheim Heiman F L; Sumpradit Nithima; Vlieghe Erika; Hara Gabriel Levy; Gould Ian M; Goossens Herman; Greko Christina; So Anthony D; Bigdeli Maryam; Tomson Goran; Woodhouse Will; Ombaka Eva; Peralta Arturo Quizhpe; Qamar Farah Naz; Mir Fatima; Kariuki Sam; Bhutta Zulfiqar A; Coates Anthony; Bergstrom Richard; Wright Gerard D; Brown Eric D; Cars Otto</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Infectious diseases</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1057-98</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The causes of antibiotic resistance are complex and include human behaviour at many levels of society; the consequences affect everybody in the world.  Similarities with climate change are evident.  Many efforts have been made to describe the many different facets of antibiotic resistance and the interventions needed to meet the challenge.  However, coordinated action is largely absent, especially at the political level, both nationally and internationally.  Antibiotics paved the way for unprecedented medical and societal developments, and are today indispensible in all health systems.  Achievements in modern medicine, such as major surgery, organ transplantation, treatment of preterm babies, and cancer chemotherapy, which we today take for granted, would not be possible without access to effective treatment for bacterial infections.  Within just a few years, we might be faced with dire setbacks, medically, socially, and economically, unless real and unprecedented global coordinated actions are immediately taken.  Here, we describe the global situation of antibiotic resistance, its major causes and consequences, and identify key areas in which action is urgently needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRve4U0u-sYhWx8TEN9K18QfW6udTcc2eZH8LcghZA8Zrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c7os1Gntg%253D%253D&md5=dc5cd314f3d81483c4b767f090af1482</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2813%2970318-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252813%252970318-9%26sid%3Dliteratum%253Aachs%26aulast%3DLaxminarayan%26aufirst%3DR.%26aulast%3DDuse%26aufirst%3DA.%26aulast%3DWattal%26aufirst%3DC.%26aulast%3DZaidi%26aufirst%3DA.%2BK.%2BM.%26aulast%3DWertheim%26aufirst%3DH.%2BF.%2BL.%26aulast%3DSumpradit%26aufirst%3DN.%26aulast%3DVlieghe%26aufirst%3DE.%26aulast%3DHara%26aufirst%3DG.%2BL.%26aulast%3DGould%26aufirst%3DI.%2BM.%26aulast%3DGoossens%26aufirst%3DH.%26aulast%3DGreko%26aufirst%3DC.%26aulast%3DSo%26aufirst%3DA.%2BD.%26aulast%3DBigdeli%26aufirst%3DM.%26aulast%3DTomson%26aufirst%3DG.%26aulast%3DWoodhouse%26aufirst%3DW.%26aulast%3DOmbaka%26aufirst%3DE.%26aulast%3DPeralta%26aufirst%3DA.%2BQ.%26aulast%3DQamar%26aufirst%3DF.%2BN.%26aulast%3DMir%26aufirst%3DF.%26aulast%3DKariuki%26aufirst%3DS.%26aulast%3DBhutta%26aufirst%3DZ.%2BA.%26aulast%3DCoates%26aufirst%3DA.%26aulast%3DBergstrom%26aufirst%3DR.%26aulast%3DWright%26aufirst%3DG.%2BD.%26aulast%3DBrown%26aufirst%3DE.%2BD.%26aulast%3DCars%26aufirst%3DO.%26atitle%3DAntibiotic%2520Resistance%25E2%2580%2594The%2520Need%2520for%2520Global%2520Solutions%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2013%26volume%3D13%26spage%3D1057%26epage%3D1098%26doi%3D10.1016%2FS1473-3099%2813%2970318-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laxminarayan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsoso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brower, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Røttingen, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klugman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, S.</span></span> <span> </span><span class="NLM_article-title">Access to Effective Antimicrobials: A Worldwide Challenge</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">168</span>– <span class="NLM_lpage">175</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(15)00474-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2FS0140-6736%2815%2900474-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=26603918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A280%3ADC%252BC28vltVGrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=2016&pages=168-175&author=R.+Laxminarayanauthor=P.+Matsosoauthor=S.+Pantauthor=C.+Browerauthor=J.+A.+R%C3%B8ttingenauthor=K.+Klugmanauthor=S.+Davies&title=Access+to+Effective+Antimicrobials%3A+A+Worldwide+Challenge&doi=10.1016%2FS0140-6736%2815%2900474-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Access to effective antimicrobials: a worldwide challenge</span></div><div class="casAuthors">Laxminarayan Ramanan; Matsoso Precious; Pant Suraj; Brower Charles; Rottingen John-Arne; Klugman Keith; Davies Sally</div><div class="citationInfo"><span class="NLM_cas:title">Lancet (London, England)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">387</span>
        (<span class="NLM_cas:issue">10014</span>),
    <span class="NLM_cas:pages">168-75</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Recent years have seen substantial improvements in life expectancy and access to antimicrobials, especially in low-income and lower-middle-income countries, but increasing pathogen resistance to antimicrobials threatens to roll back this progress.  Resistant organisms in health-care and community settings pose a threat to survival rates from serious infections, including neonatal sepsis and health-care-associated infections, and limit the potential health benefits from surgeries, transplants, and cancer treatment.  The challenge of simultaneously expanding appropriate access to antimicrobials, while restricting inappropriate access, particularly to expensive, newer generation antimicrobials, is unique in global health and requires new approaches to financing and delivering health care and a one-health perspective on the connections between pathogen transmission in animals and humans.  Here, we describe the importance of effective antimicrobials.  We assess the disease burden caused by limited access to antimicrobials, attributable to resistance to antimicrobials, and the potential effect of vaccines in restricting the need for antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSzqsAQa5w3CcULxIEhxUjPfW6udTcc2eZH8LcghZA8Zrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28vltVGrug%253D%253D&md5=ff397045340f81521e75bfee7f933244</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2815%2900474-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252815%252900474-2%26sid%3Dliteratum%253Aachs%26aulast%3DLaxminarayan%26aufirst%3DR.%26aulast%3DMatsoso%26aufirst%3DP.%26aulast%3DPant%26aufirst%3DS.%26aulast%3DBrower%26aufirst%3DC.%26aulast%3DR%25C3%25B8ttingen%26aufirst%3DJ.%2BA.%26aulast%3DKlugman%26aufirst%3DK.%26aulast%3DDavies%26aufirst%3DS.%26atitle%3DAccess%2520to%2520Effective%2520Antimicrobials%253A%2520A%2520Worldwide%2520Challenge%26jtitle%3DLancet%26date%3D2016%26volume%3D387%26spage%3D168%26epage%3D175%26doi%3D10.1016%2FS0140-6736%2815%2900474-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huttner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbarth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosgrove, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarlier, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittet, D.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial Resistance: A Global View from the 2013 World Healthcare-Associated Infections Forum</span>. <i>Antimicrob. Resist. Infect. Control</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">31</span>, <span class="refDoi"> DOI: 10.1186/2047-2994-2-31</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1186%2F2047-2994-2-31" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=24237856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A280%3ADC%252BC2c7ns1Chug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=31&author=A.+Huttnerauthor=S.+Harbarthauthor=J.+Carletauthor=S.+Cosgroveauthor=H.+Goossensauthor=A.+Holmesauthor=V.+Jarlierauthor=A.+Vossauthor=D.+Pittet&title=Antimicrobial+Resistance%3A+A+Global+View+from+the+2013+World+Healthcare-Associated+Infections+Forum&doi=10.1186%2F2047-2994-2-31"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial resistance: a global view from the 2013 World Healthcare-Associated Infections Forum</span></div><div class="casAuthors">Huttner Angela; Harbarth Stephan; Pittet Didier; Carlet Jean; Cosgrove Sara; Goossens Herman; Holmes Alison; Jarlier Vincent; Voss Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial resistance and infection control</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">31</span>
        ISSN:<span class="NLM_cas:issn">2047-2994</span>.
    </div><div class="casAbstract">Antimicrobial resistance (AMR) is now a global threat.  Its emergence rests on antimicrobial overuse in humans and food-producing animals; globalization and suboptimal infection control facilitate its spread.  While aggressive measures in some countries have led to the containment of some resistant gram-positive organisms, extensively resistant gram-negative organisms such as carbapenem-resistant enterobacteriaceae and pan-resistant Acinetobacter spp. continue their rapid spread.  Antimicrobial conservation/stewardship programs have seen some measure of success in reducing antimicrobial overuse in humans, but their reach is limited to acute-care settings in high-income countries.  Outside the European Union, there is scant or no oversight of antimicrobial administration to food-producing animals, while evidence mounts that this administration leads directly to resistant human infections.  Both horizontal and vertical infection control measures can interrupt transmission among humans, but many of these are costly and essentially limited to high-income countries as well.  Novel antimicrobials are urgently needed; in recent decades pharmaceutical companies have largely abandoned antimicrobial discovery and development given their high costs and low yield.  Against this backdrop, international and cross-disciplinary collaboration appears to be taking root in earnest, although specific strategies still need defining.  Educational programs targeting both antimicrobial prescribers and consumers must be further developed and supported.  The general public must continue to be made aware of the current scale of AMR's threat, and must perceive antimicrobials as they are: a non-renewable and endangered resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ0kHV8NqxvhiO2kE7krGOBfW6udTcc2eZH8LcghZA8Zrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c7ns1Chug%253D%253D&md5=2c045c0cd094702cea5b5ed5c2012f61</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1186%2F2047-2994-2-31&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F2047-2994-2-31%26sid%3Dliteratum%253Aachs%26aulast%3DHuttner%26aufirst%3DA.%26aulast%3DHarbarth%26aufirst%3DS.%26aulast%3DCarlet%26aufirst%3DJ.%26aulast%3DCosgrove%26aufirst%3DS.%26aulast%3DGoossens%26aufirst%3DH.%26aulast%3DHolmes%26aufirst%3DA.%26aulast%3DJarlier%26aufirst%3DV.%26aulast%3DVoss%26aufirst%3DA.%26aulast%3DPittet%26aufirst%3DD.%26atitle%3DAntimicrobial%2520Resistance%253A%2520A%2520Global%2520View%2520from%2520the%25202013%2520World%2520Healthcare-Associated%2520Infections%2520Forum%26jtitle%3DAntimicrob.%2520Resist.%2520Infect.%2520Control%26date%3D2013%26volume%3D2%26spage%3D31%26doi%3D10.1186%2F2047-2994-2-31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">East, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span> <span> </span><span class="NLM_article-title">Multitarget Ligands in Antibacterial Research: Progress and Opportunities</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">156</span>, <span class="refDoi"> DOI: 10.1517/17460441.2013.743991</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1517%2F17460441.2013.743991" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=23252414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Cku74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=143-156&author=S.+P.+Eastauthor=L.+L.+Silver&title=Multitarget+Ligands+in+Antibacterial+Research%3A+Progress+and+Opportunities&doi=10.1517%2F17460441.2013.743991"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Multitarget ligands in antibacterial research: progress and opportunities</span></div><div class="casAuthors">East, Stephen P.; Silver, Lynn L.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">143-156</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: Resistance to current antibacterial therapies is an inevitability that represents a significant global health concern.  Bacteria have the capacity to render all current drug treatments ineffective, which places a demand on the drug discovery community to constantly develop new antibacterial agents.  Compds. that inhibit multiple biol. targets, often referred to as multitarget ligands, are an inviting prospect in antibacterial research because, although they will not solve the issue of resistance, they might help to delay the onset. Areas covered: This review covers some of the recent progress in identifying new ligands that deliberately interact with more than one essential biol. target in bacteria.  The two principal areas covered are inhibitors of DNA replication and cell wall biosynthesis. Expert opinion: Antibacterial programs for the design of multitarget ligands present an important opportunity for prodn. of antibacterial agents.  Their longevity, due to slow development of resistance, is comparable to that seen with other successful agents - but is much improved over single-targeted agents for which resistance can appear in vitro overnight.  The preclin. development of these agents will have to overcome the std. problems of antibacterial discovery.  Such problems include optimization of characteristics favoring cell entry and particularly the demonstration of selectivity of inhibition of the desired multiple targets without inhibition of other bacterial or any mammalian functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbhpbYgIA8HrVg90H21EOLACvtfcHk0liqxLgem4i8Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Cku74%253D&md5=2b335d54e579d210ab9a1220ea2d6a57</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1517%2F17460441.2013.743991&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2013.743991%26sid%3Dliteratum%253Aachs%26aulast%3DEast%26aufirst%3DS.%2BP.%26aulast%3DSilver%26aufirst%3DL.%2BL.%26atitle%3DMultitarget%2520Ligands%2520in%2520Antibacterial%2520Research%253A%2520Progress%2520and%2520Opportunities%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2013%26volume%3D8%26spage%3D143%26epage%3D156%26doi%3D10.1517%2F17460441.2013.743991" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gellert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizuuchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Dea, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, H. A.</span></span> <span> </span><span class="NLM_article-title">DNA Gyrase: An Enzyme That Introduces Superhelical Turns into DNA</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">3872</span>– <span class="NLM_lpage">3876</span>, <span class="refDoi"> DOI: 10.1073/pnas.73.11.3872</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1073%2Fpnas.73.11.3872" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=186775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADyaE2sXjvFWmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=1976&pages=3872-3876&author=M.+Gellertauthor=K.+Mizuuchiauthor=M.+H.+O%E2%80%99Deaauthor=H.+A.+Nash&title=DNA+Gyrase%3A+An+Enzyme+That+Introduces+Superhelical+Turns+into+DNA&doi=10.1073%2Fpnas.73.11.3872"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">DNA gyrase:  An enzyme that introduces superhelical turns into DNA</span></div><div class="casAuthors">Gellert, Martin; Mizuuchi, Kiyoshi; O'Dea, Mary H.; Nash, Howard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3872-6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Relaxed closed-circular DNA was converted to neg. supercoiled DNA by DNA gyrase.  This enzyme was purified from Escherichia coli cells.  The reaction required ATP and Mg2+ and was stimulated by spermidine.  The enzyme acted equally well on relaxed closed-circular colicin E1, bacteriophage λ, and simian virus 40 DNA.  The final superhelix d. of the DNA could be considerably greater than that found in intracellularly supercoiled DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWknjnsm_VqrVg90H21EOLACvtfcHk0liqxLgem4i8Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXjvFWmsQ%253D%253D&md5=adc7a9bfe9d48cba5143727e982e7b81</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.73.11.3872&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.73.11.3872%26sid%3Dliteratum%253Aachs%26aulast%3DGellert%26aufirst%3DM.%26aulast%3DMizuuchi%26aufirst%3DK.%26aulast%3DO%25E2%2580%2599Dea%26aufirst%3DM.%2BH.%26aulast%3DNash%26aufirst%3DH.%2BA.%26atitle%3DDNA%2520Gyrase%253A%2520An%2520Enzyme%2520That%2520Introduces%2520Superhelical%2520Turns%2520into%2520DNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1976%26volume%3D73%26spage%3D3872%26epage%3D3876%26doi%3D10.1073%2Fpnas.73.11.3872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kato, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiraga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, H.</span></span> <span> </span><span class="NLM_article-title">New Topoisomerase Essential for Chromosome Segregation in E. Coli</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">393</span>– <span class="NLM_lpage">404</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(90)90172-B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2F0092-8674%2890%2990172-B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=2170028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADyaK3MXhsFOrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1990&pages=393-404&author=J.+Katoauthor=Y.+Nishimuraauthor=R.+Imamuraauthor=H.+Nikiauthor=S.+Hiragaauthor=H.+Suzuki&title=New+Topoisomerase+Essential+for+Chromosome+Segregation+in+E.+Coli&doi=10.1016%2F0092-8674%2890%2990172-B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">New topoisomerase essential for chromosome segregation in E. coli</span></div><div class="casAuthors">Kato, Junichi; Nishimura, Yukinobu; Imamura, Ryu; Niki, Hironori; Hiraga, Sota; Suzuki, Hideho</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">393-404</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">The nucleotide sequence of the parC gene essential for chromosome partition in Escherichia coli was detd.  The deduced amino acid sequence was homologous to that of the A subunit of gyrase.  A new gene encoding a ∼70 kd protein was found.  The gene was sequenced, and the deduced amino acid sequence revealed that the gene product was homologous to the gyrase B subunit.  Mutants of this gene were isolated and showed the typical Par phenotype at nonpermissive temp.; thus, the gene was named parE.  Enhanced relaxation activity of supercoiled plasmid mols. was detected in the combined crude cell lysates prepd. from the ParC and ParE overproducers.  A topA mutation defective in topoisomerase I could be compensated by increasing both the parC and the parE gene dosage.  It is suggested that the parC and parE genes code for the subunits of a new topoisomerase, named topo IV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgpWpdKUf52LVg90H21EOLACvtfcHk0liqxLgem4i8Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXhsFOrtLY%253D&md5=6a0ec298af3872e372b39f710ec57093</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2890%2990172-B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252890%252990172-B%26sid%3Dliteratum%253Aachs%26aulast%3DKato%26aufirst%3DJ.%26aulast%3DNishimura%26aufirst%3DY.%26aulast%3DImamura%26aufirst%3DR.%26aulast%3DNiki%26aufirst%3DH.%26aulast%3DHiraga%26aufirst%3DS.%26aulast%3DSuzuki%26aufirst%3DH.%26atitle%3DNew%2520Topoisomerase%2520Essential%2520for%2520Chromosome%2520Segregation%2520in%2520E.%2520Coli%26jtitle%3DCell%26date%3D1990%26volume%3D63%26spage%3D393%26epage%3D404%26doi%3D10.1016%2F0092-8674%2890%2990172-B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drlica, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span> <span> </span><span class="NLM_article-title">Quinolone-Mediated Bacterial Death</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">385</span>– <span class="NLM_lpage">392</span>, <span class="refDoi"> DOI: 10.1128/AAC.01617-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.01617-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=17724149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlSqtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2008&pages=385-392&author=K.+Drlicaauthor=M.+Malikauthor=R.+J.+Kernsauthor=X.+Zhao&title=Quinolone-Mediated+Bacterial+Death&doi=10.1128%2FAAC.01617-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Quinolone-mediated bacterial death</span></div><div class="casAuthors">Drlica, Karl; Malik, Muhammad; Kerns, Robert J.; Zhao, Xilin</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">385-392</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  In the present review we consider cell death through a two-part "poison" hypothesis in which quinolones form reversible drug-topoisomerase-DNA complexes that subsequently lead to several types of irreversible (lethal) damage.  Topics discussed include: quinolone-topoisomerase-DNA complexes; inhibition of DNA replication; SOS response and cell filamentation; toxin-antitoxin modules and programmed cell death; chromosome fragmentation; and destabilization of cleaved complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowBOqyJ-TV4bVg90H21EOLACvtfcHk0lhyvW4f7pw_NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlSqtLc%253D&md5=eef420d64b9f28336fb0cc5f6c42a803</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FAAC.01617-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01617-06%26sid%3Dliteratum%253Aachs%26aulast%3DDrlica%26aufirst%3DK.%26aulast%3DMalik%26aufirst%3DM.%26aulast%3DKerns%26aufirst%3DR.%2BJ.%26aulast%3DZhao%26aufirst%3DX.%26atitle%3DQuinolone-Mediated%2520Bacterial%2520Death%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2008%26volume%3D52%26spage%3D385%26epage%3D392%26doi%3D10.1128%2FAAC.01617-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Redgrave, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piddock, L. J.</span></span> <span> </span><span class="NLM_article-title">Fluoroquinolone Resistance: Mechanisms, Impact on Bacteria, and Role in Evolutionary Success</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">445</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2014.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.tim.2014.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=24842194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotlOqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=438-445&author=L.+S.+Redgraveauthor=S.+B.+Suttonauthor=M.+A.+Webberauthor=L.+J.+Piddock&title=Fluoroquinolone+Resistance%3A+Mechanisms%2C+Impact+on+Bacteria%2C+and+Role+in+Evolutionary+Success&doi=10.1016%2Fj.tim.2014.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Fluoroquinolone resistance: mechanisms, impact on bacteria, and role in evolutionary success</span></div><div class="casAuthors">Redgrave, Liam S.; Sutton, Sam B.; Webber, Mark A.; Piddock, Laura J. V.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">438-445</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Quinolone and fluoroquinolone antibiotics are potent, broad-spectrum agents commonly used to treat a range of infections.  Resistance to these agents is multifactorial and can be via one or a combination of target-site gene mutations, increased prodn. of multidrug-resistance (MDR) efflux pumps, modifying enzymes, and/or target-protection proteins.  Fluoroquinolone-resistant clin. isolates of bacteria have emerged readily and recent data have shown that resistance to this class of antibiotics can have diverse, species-dependent impacts on host-strain fitness.  Here we outline the impacts of quinolone-resistance mutations in relation to the fitness and evolutionary success of mutant strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJrla4tEwdbrVg90H21EOLACvtfcHk0lhyvW4f7pw_NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotlOqsLc%253D&md5=060fb6501e12ed1243e4f30190cb4917</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2014.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2014.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DRedgrave%26aufirst%3DL.%2BS.%26aulast%3DSutton%26aufirst%3DS.%2BB.%26aulast%3DWebber%26aufirst%3DM.%2BA.%26aulast%3DPiddock%26aufirst%3DL.%2BJ.%26atitle%3DFluoroquinolone%2520Resistance%253A%2520Mechanisms%252C%2520Impact%2520on%2520Bacteria%252C%2520and%2520Role%2520in%2520Evolutionary%2520Success%26jtitle%3DTrends%2520Microbiol.%26date%3D2014%26volume%3D22%26spage%3D438%26epage%3D445%26doi%3D10.1016%2Fj.tim.2014.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Black, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stachyra, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girard, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claudon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruneau, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miossec, C.</span></span> <span> </span><span class="NLM_article-title">Mechanism of Action of the Antibiotic Nxl101, a Novel Nonfluoroquinolone Inhibitor of Bacterial Type Ii Topoisomerases</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3339</span>– <span class="NLM_lpage">3349</span>, <span class="refDoi"> DOI: 10.1128/AAC.00496-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.00496-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=18625781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtF2gt77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2008&pages=3339-3349&author=M.+T.+Blackauthor=T.+Stachyraauthor=D.+Platelauthor=A.+M.+Girardauthor=M.+Claudonauthor=J.+M.+Bruneauauthor=C.+Miossec&title=Mechanism+of+Action+of+the+Antibiotic+Nxl101%2C+a+Novel+Nonfluoroquinolone+Inhibitor+of+Bacterial+Type+Ii+Topoisomerases&doi=10.1128%2FAAC.00496-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of action of the antibiotic NXL101, a novel nonfluoroquinolone inhibitor of bacterial type II topoisomerases</span></div><div class="casAuthors">Black, Michael T.; Stachyra, Therese; Platel, Denis; Girard, Anne-Marie; Claudon, Monique; Bruneau, Jean-Michel; Miossec, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3339-3349</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">NXL101 is one of a new class of quinoline antibacterial DNA gyrase and topoisomerase IV inhibitors showing potent activity against gram-pos. bacteria, including methicillin- and fluoroquinolone-resistant strains.  NXL101 inhibited topoisomerase IV more effectively than gyrase from Escherichia coli, whereas the converse is true of enzymes from Staphylococcus aureus.  This apparent target preference is opposite to that which is assocd. with most fluoroquinolone antibiotics.  In vitro isolation of S. aureus mutants resistant to NXL101 followed by cloning and sequencing of the genes encoding gyrase and topoisomerase IV led to the identification of several different point mutations within, or close to, the quinolone resistance-detg. region (QRDR) of GyrA.  However, the mutations were not those that are most frequently assocd. with decreased sensitivity to quinolones.  A fluoroquinolone-resistant mutant variant of gyrase generated in vitro was highly resistant to inhibition by the fluoroquinolones ciprofloxacin and moxifloxacin but remained fully susceptible to inhibition by NXL101.  Two mutant gyrases constructed in vitro, with mutations in gyrA engineered according to those most frequently found in S. aureus strains resistant to NXL101, were insensitive to inhibition by NXL101 and had a diminished sensitivity to ciprofloxacin and moxifloxacin.  Certain combinations of mutations giving rise to NXL101 resistance and those giving rise to fluoroquinolone resistance may be mutually exclusive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPRmIzSCazIbVg90H21EOLACvtfcHk0lhyvW4f7pw_NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtF2gt77L&md5=a39fe88e84c958369d86f4614c9f3e06</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2FAAC.00496-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00496-08%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DM.%2BT.%26aulast%3DStachyra%26aufirst%3DT.%26aulast%3DPlatel%26aufirst%3DD.%26aulast%3DGirard%26aufirst%3DA.%2BM.%26aulast%3DClaudon%26aufirst%3DM.%26aulast%3DBruneau%26aufirst%3DJ.%2BM.%26aulast%3DMiossec%26aufirst%3DC.%26atitle%3DMechanism%2520of%2520Action%2520of%2520the%2520Antibiotic%2520Nxl101%252C%2520a%2520Novel%2520Nonfluoroquinolone%2520Inhibitor%2520of%2520Bacterial%2520Type%2520Ii%2520Topoisomerases%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2008%26volume%3D52%26spage%3D3339%26epage%3D3349%26doi%3D10.1128%2FAAC.00496-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Black, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span> <span> </span><span class="NLM_article-title">New Inhibitors of Bacterial Topoisomerase Gyra/Parc Subunits</span>. <i>Curr. Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">804</span>– <span class="NLM_lpage">810</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=19649925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnt1KhsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=804-810&author=M.+T.+Blackauthor=K.+Coleman&title=New+Inhibitors+of+Bacterial+Topoisomerase+Gyra%2FParc+Subunits"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">New inhibitors of bacterial topoisomerase GyrA/ParC subunits</span></div><div class="casAuthors">Black, Michael T.; Coleman, Kenneth</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Investigational Drugs (BioMed Central)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">804-810</span>CODEN:
                <span class="NLM_cas:coden">COIDAZ</span>;
        ISSN:<span class="NLM_cas:issn">2040-3429</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Inhibitors of the GyrA and ParC subunits of the bacterial type IIA topoisomerases DNA gyrase and topoisomerase IV, resp., belong to classes of antibiotics that are among the most successful of the past 50 years, both in a therapeutic and a com. sense.  The initial promise of progressive modification of an antibiotic scaffold to extend the antibacterial spectrum and to modulate pharmacokinetic/pharmacodynamic parameters of existing antibiotics in an advantageous manner has been tempered by the frequent concomitant appearance of serious toxicity.  The merits of the most prominent novel antibiotics that have reached, or are expected to reach the clin. phase of drug development are summarized in this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTnu8CeuJntLVg90H21EOLACvtfcHk0lhiSuSusy4vMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnt1KhsLg%253D&md5=c1bc05041d4144fda13f691acbef4758</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DM.%2BT.%26aulast%3DColeman%26aufirst%3DK.%26atitle%3DNew%2520Inhibitors%2520of%2520Bacterial%2520Topoisomerase%2520Gyra%252FParc%2520Subunits%26jtitle%3DCurr.%2520Opin.%2520Invest.%2520Drugs%26date%3D2009%26volume%3D10%26spage%3D804%26epage%3D810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ross, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scangarella-Oman, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flamm, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. N.</span></span> <span> </span><span class="NLM_article-title">Determination of Disk Diffusion and Mic Quality Control Guidelines for Gsk2140944, a Novel Bacterial Type Ii Topoisomerase Inhibitor Antimicrobial Agent</span>. <i>J. Clin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2629</span>– <span class="NLM_lpage">2632</span>, <span class="refDoi"> DOI: 10.1128/JCM.00656-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FJCM.00656-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=24759716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1yqu7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2014&pages=2629-2632&author=J.+E.+Rossauthor=N.+E.+Scangarella-Omanauthor=R.+K.+Flammauthor=R.+N.+Jones&title=Determination+of+Disk+Diffusion+and+Mic+Quality+Control+Guidelines+for+Gsk2140944%2C+a+Novel+Bacterial+Type+Ii+Topoisomerase+Inhibitor+Antimicrobial+Agent&doi=10.1128%2FJCM.00656-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of disk diffusion and MIC quality control guidelines for GSK2140944, a novel bacterial type II topoisomerase inhibitor antimicrobial agent</span></div><div class="casAuthors">Ross, James E.; Scangarella-Oman, Nicole E.; Flamm, Robert K.; Jones, Ronald N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2629-2632, 5 pp.</span>CODEN:
                <span class="NLM_cas:coden">JCMIDW</span>;
        ISSN:<span class="NLM_cas:issn">1098-660X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">GSK2140944 is a novel bacterial type II topoisomerase inhibitor in development for the treatment of conventional and biothreat pathogens, including Gram-pos. pathogens and methicillin-resistant Staphylococcus aureus.  This quality control study was performed to establish ranges for selected control strains: S. aureus ATCC 29213 and ATCC 25923, Escherichia coli ATCC 25922, Haemophilus influenzae ATCC 49247, and Streptococcus pneumoniae ATCC 49619.  The control ranges will be crucial for the accurate evaluation of GSK2140944 potency as it progresses through clin. trial development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6LkQjobYlJ7Vg90H21EOLACvtfcHk0lhiSuSusy4vMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1yqu7rN&md5=b8c019eee78124d5588db240d2659d5a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FJCM.00656-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJCM.00656-14%26sid%3Dliteratum%253Aachs%26aulast%3DRoss%26aufirst%3DJ.%2BE.%26aulast%3DScangarella-Oman%26aufirst%3DN.%2BE.%26aulast%3DFlamm%26aufirst%3DR.%2BK.%26aulast%3DJones%26aufirst%3DR.%2BN.%26atitle%3DDetermination%2520of%2520Disk%2520Diffusion%2520and%2520Mic%2520Quality%2520Control%2520Guidelines%2520for%2520Gsk2140944%252C%2520a%2520Novel%2520Bacterial%2520Type%2520Ii%2520Topoisomerase%2520Inhibitor%2520Antimicrobial%2520Agent%26jtitle%3DJ.%2520Clin.%2520Microbiol.%26date%3D2014%26volume%3D52%26spage%3D2629%26epage%3D2632%26doi%3D10.1128%2FJCM.00656-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sader, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhomberg, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scangarella-Oman, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flamm, R. K.</span></span> <span> </span><span class="NLM_article-title"><i>In Vitro</i> Activity of Gepotidacin (Gsk2140944) against <i>Neisseria Gonorrhoeae</i></span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>, <span class="NLM_elocation-id">e02047</span> <span class="refDoi"> DOI: 10.1128/AAC.02047-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.02047-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=28069643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFektrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&author=D.+J.+Farrellauthor=H.+S.+Saderauthor=P.+R.+Rhombergauthor=N.+E.+Scangarella-Omanauthor=R.+K.+Flamm&title=In+Vitro+Activity+of+Gepotidacin+%28Gsk2140944%29+against+Neisseria+Gonorrhoeae&doi=10.1128%2FAAC.02047-16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro activity of gepotidacin (GSK2140944) against Neisseria gonorrhoeae</span></div><div class="casAuthors">Farrell, D. J.; Sader, H. S.; Rhomberg, P. R.; Scangarella-Oman, N. E.; Flamm, R. K.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e02047-16/1-e02047-16/7</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Gepotidacin (formerly GSK2140944) is a novel, first-in-class, triazaacenaphthylene antibacterial that inhibits bacterial DNA gyrase and topoisomerase IV via a unique mechanism and has demonstrated in vitro activity against Neisseria gonorrhoeae, including drug-resistant strains, and also targets pathogens assocd. with other conventional and biothreat infections.  Broth microdilution was used to evaluate the MIC and min. bactericidal concn. (MBC) activity of gepotidacin and comparators against 25 N. gonorrhoeae strains (including five ciprofloxacin-nonsusceptible strains).  Gepotidacin activity was also evaluated against three N. gonorrhoeae strains (including a ciprofloxacin-nonsusceptible strain) for resistance development, against three N. gonorrhoeae strains (including two tetracycline- and azithromycin-nonsusceptible strains) using time-kill kinetics and checkerboard methods, and against two N. gonorrhoeae strains for the investigation of postantibiotic (PAE) and subinhibitory (PAE-SME) effects.  The MIC50 and MIC90 for gepotidacin against the 25 N. gonorrhoeae isolates tested were 0.12 and 0.25 μg/mL, resp.  The MBC50 and MBC90 for gepotidacin were 0.25 and 0.5 μg/mL, resp.  Gepotidacin was bactericidal, and single-step resistance selection studies did not recover any mutants, indicating a low rate of spontaneous single-step resistance.  For combinations of gepotidacin and comparators tested using checkerboard methods, there were no instances where antagonism occurred and only one instance of synergy (with moxifloxacin; fractional inhibitory concn., 0.375).  This was not confirmed by in vitro time-kill studies.  The PAE for gepotidacin against the wild-type strain ranged from 0.5 to >2.5 h, and the PAE-SME was >2.5 h.  These in vitro data indicate that further study of gepotidacin is warranted for potential use in treating infections caused by N. gonorrhoeae.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO4ROH3tiEkbVg90H21EOLACvtfcHk0lhiSuSusy4vMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFektrbO&md5=289e4d3f32d31105582ac3c4320fd0f3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1128%2FAAC.02047-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02047-16%26sid%3Dliteratum%253Aachs%26aulast%3DFarrell%26aufirst%3DD.%2BJ.%26aulast%3DSader%26aufirst%3DH.%2BS.%26aulast%3DRhomberg%26aufirst%3DP.%2BR.%26aulast%3DScangarella-Oman%26aufirst%3DN.%2BE.%26aulast%3DFlamm%26aufirst%3DR.%2BK.%26atitle%3DIn%2520Vitro%2520Activity%2520of%2520Gepotidacin%2520%2528Gsk2140944%2529%2520against%2520Neisseria%2520Gonorrhoeae%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26doi%3D10.1128%2FAAC.02047-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osheroff, N.</span></span> <span> </span><span class="NLM_article-title">Mechanistic and Structural Basis for the Actions of the Antibacterial Gepotidacin against Staphylococcus aureus Gyrase</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">570</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.8b00315</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.8b00315" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivVOgt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=570&author=E.+G.+Gibsonauthor=B.+Baxauthor=P.+F.+Chanauthor=N.+Osheroff&title=Mechanistic+and+Structural+Basis+for+the+Actions+of+the+Antibacterial+Gepotidacin+against+Staphylococcus+aureus+Gyrase&doi=10.1021%2Facsinfecdis.8b00315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic and Structural Basis for the Actions of the Antibacterial Gepotidacin against Staphylococcus aureus Gyrase</span></div><div class="casAuthors">Gibson, Elizabeth G.; Bax, Ben; Chan, Pan F.; Osheroff, Neil</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">570-581</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Gepotidacin is a first-in-class triazaacenaphthylene novel bacterial topoisomerase inhibitor (NBTI).  The compd. has successfully completed phase II trials for the treatment of acute bacterial skin/skin structure infections and for the treatment of uncomplicated urogenital gonorrhea.  It also displays robust in vitro activity against a range of wild-type and fluoroquinolone-resistant bacteria.  Due to the clin. promise of gepotidacin, a detailed understanding of its interactions with its antibacterial targets is essential.  Thus, we characterized the mechanism of action of gepotidacin against Staphylococcus aureus gyrase.  Gepotidacin was a potent inhibitor of gyrase-catalyzed DNA supercoiling (IC50 ≈ 0.047 μM) and relaxation of pos. supercoiled substrates (IC50 ≈ 0.6 μM).  Unlike fluoroquinolones, which induce primarily double-stranded DNA breaks, gepotidacin induced high levels of gyrase-mediated single-stranded breaks.  No double-stranded breaks were obsd. even at high gepotidacin concn., long cleavage times, or in the presence of ATP.  Moreover, gepotidacin suppressed the formation of double-stranded breaks.  Gepotidacin formed gyrase-DNA cleavage complexes that were stable for >4 h.  In vitro competition suggests that gyrase binding by gepotidacin and fluoroquinolones are mutually exclusive.  Finally, we detd. crystal structures of gepotidacin with the S. aureus gyrase core fusion truncate with nicked (2.31 Å resoln.) or intact (uncleaved) DNA (2.37 Å resoln.).  In both cases, a single gepotidacin mol. was bound midway between the two scissile DNA bonds and in a pocket between the two GyrA subunits.  A comparison of the two structures demonstrates conformational flexibility within the central linker of gepotidacin, which may contribute to the activity of the compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZKsv0roHKe7Vg90H21EOLACvtfcHk0lh8SSuEelCHpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivVOgt7s%253D&md5=b13ebf930786a12be3d868bf575e6fb3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.8b00315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.8b00315%26sid%3Dliteratum%253Aachs%26aulast%3DGibson%26aufirst%3DE.%2BG.%26aulast%3DBax%26aufirst%3DB.%26aulast%3DChan%26aufirst%3DP.%2BF.%26aulast%3DOsheroff%26aufirst%3DN.%26atitle%3DMechanistic%2520and%2520Structural%2520Basis%2520for%2520the%2520Actions%2520of%2520the%2520Antibacterial%2520Gepotidacin%2520against%2520Staphylococcus%2520aureus%2520Gyrase%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2019%26spage%3D570%26doi%3D10.1021%2Facsinfecdis.8b00315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Riordan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiffany, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scangarella-Oman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hossain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumont, E.</span></span> <span> </span><span class="NLM_article-title">Efficacy, Safety, and Tolerability of Gepotidacin (Gsk2140944) in the Treatment of Patients with Suspected or Confirmed Gram-Positive Acute Bacterial Skin and Skin Structure Infections</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>, <span class="NLM_elocation-id">e02095</span> <span class="refDoi"> DOI: 10.1128/AAC.02095-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.02095-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=28373199" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&author=W.+O%E2%80%99Riordanauthor=C.+Tiffanyauthor=N.+Scangarella-Omanauthor=C.+Perryauthor=M.+Hossainauthor=T.+Ashtonauthor=E.+Dumont&title=Efficacy%2C+Safety%2C+and+Tolerability+of+Gepotidacin+%28Gsk2140944%29+in+the+Treatment+of+Patients+with+Suspected+or+Confirmed+Gram-Positive+Acute+Bacterial+Skin+and+Skin+Structure+Infections&doi=10.1128%2FAAC.02095-16"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1128%2FAAC.02095-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02095-16%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Riordan%26aufirst%3DW.%26aulast%3DTiffany%26aufirst%3DC.%26aulast%3DScangarella-Oman%26aufirst%3DN.%26aulast%3DPerry%26aufirst%3DC.%26aulast%3DHossain%26aufirst%3DM.%26aulast%3DAshton%26aufirst%3DT.%26aulast%3DDumont%26aufirst%3DE.%26atitle%3DEfficacy%252C%2520Safety%252C%2520and%2520Tolerability%2520of%2520Gepotidacin%2520%2528Gsk2140944%2529%2520in%2520the%2520Treatment%2520of%2520Patients%2520with%2520Suspected%2520or%2520Confirmed%2520Gram-Positive%2520Acute%2520Bacterial%2520Skin%2520and%2520Skin%2520Structure%2520Infections%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26doi%3D10.1128%2FAAC.02095-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumont, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raychaudhuri, A.</span></span> <span> </span><span class="NLM_article-title">O05.3 a Phase Ii, Randomised, Study in Adult Subjects Evaluating the Efficacy, Safety, and Tolerability of Single Doses of Gepotidacin (Gsk2140944) for Treatment of Uncomplicated Urogenital Gonorrhoea</span>. <i>Sex. Transm. Infect.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">A11</span>, <span class="refDoi"> DOI: 10.1136/sextrans-2017-053264.26</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1136%2Fsextrans-2017-053264.26" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=27356549" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2017&pages=A11&author=C.+Perryauthor=E.+Dumontauthor=C.+Perryauthor=A.+Raychaudhuri&title=O05.3+a+Phase+Ii%2C+Randomised%2C+Study+in+Adult+Subjects+Evaluating+the+Efficacy%2C+Safety%2C+and+Tolerability+of+Single+Doses+of+Gepotidacin+%28Gsk2140944%29+for+Treatment+of+Uncomplicated+Urogenital+Gonorrhoea&doi=10.1136%2Fsextrans-2017-053264.26"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1136%2Fsextrans-2017-053264.26&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fsextrans-2017-053264.26%26sid%3Dliteratum%253Aachs%26aulast%3DPerry%26aufirst%3DC.%26aulast%3DDumont%26aufirst%3DE.%26aulast%3DPerry%26aufirst%3DC.%26aulast%3DRaychaudhuri%26aufirst%3DA.%26atitle%3DO05.3%2520a%2520Phase%2520Ii%252C%2520Randomised%252C%2520Study%2520in%2520Adult%2520Subjects%2520Evaluating%2520the%2520Efficacy%252C%2520Safety%252C%2520and%2520Tolerability%2520of%2520Single%2520Doses%2520of%2520Gepotidacin%2520%2528Gsk2140944%2529%2520for%2520Treatment%2520of%2520Uncomplicated%2520Urogenital%2520Gonorrhoea%26jtitle%3DSex.%2520Transm.%2520Infect.%26date%3D2017%26volume%3D93%26spage%3DA11%26doi%3D10.1136%2Fsextrans-2017-053264.26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span> <i>Search of: GSK2140944 - List Results - ClinicalTrials.gov</i>. <a href="https://clinicaltrials.gov/ct2/results?cond=&amp;term=GSK2140944&amp;cntry1=&amp;state1=&amp;recrs" class="extLink">https://clinicaltrials.gov/ct2/results?cond=&term=GSK2140944&cntry1=&state1=&recrs</a> (accessed 7 November <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Search+of%3A+GSK2140944+-+List+Results+-+ClinicalTrials.gov.+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fresults%3Fcond%3D%26term%3DGSK2140944%26cntry1%3D%26state1%3D%26recrs+%28accessed+7+November+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DSearch%2520of%253A%2520GSK2140944%2520-%2520List%2520Results%2520-%2520ClinicalTrials.gov%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nayar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reck, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehmann, D. E.</span></span> <span> </span><span class="NLM_article-title">Nbti 5463 Is a Novel Bacterial Type Ii Topoisomerase Inhibitor with Activity against Gram-Negative Bacteria and <i>in Vivo</i> Efficacy</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2657</span>– <span class="NLM_lpage">2664</span>, <span class="refDoi"> DOI: 10.1128/AAC.02778-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.02778-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=24566174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC2cXovFOks7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=2657-2664&author=T.+J.+Doughertyauthor=A.+Nayarauthor=J.+V.+Newmanauthor=S.+Hopkinsauthor=G.+G.+Stoneauthor=M.+Johnstoneauthor=A.+B.+Shapiroauthor=M.+Croninauthor=F.+Reckauthor=D.+E.+Ehmann&title=Nbti+5463+Is+a+Novel+Bacterial+Type+Ii+Topoisomerase+Inhibitor+with+Activity+against+Gram-Negative+Bacteria+and+in+Vivo+Efficacy&doi=10.1128%2FAAC.02778-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">NBTI 5463 is a novel bacterial type II topoisomerase inhibitor with activity against Gram-negative bacteria and in vivo efficacy</span></div><div class="casAuthors">Dougherty, Thomas J.; Nayar, Asha; Newman, Joseph V.; Hopkins, Sussie; Stone, Gregory G.; Johnstone, Michele; Shapiro, Adam B.; Cronin, Mark; Reck, Folkert; Ehmann, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2657-2664, 9 pp.</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The need for new antibiotics that address serious Gram-neg. infections is well recognized.  Our efforts with a series of novel bacterial type II topoisomerase inhibitors (NBTIs) led to the discovery of NBTI 5463, an agent with improved activity over other NBTIs against Gram-neg. bacteria, in particular against Pseudomonas aeruginosa.  In the present work, NBTI 5463 demonstrated promising activity against a broad range of Gram-neg. pathogens.  In contrast to fluoroquinolones, the compd. did not form a double-strand DNA cleavable complex with Escherichia coli DNA gyrase and DNA, but it was a potent inhibitor of both DNA gyrase and E. coli topoisomerase IV catalytic activities.  In studies with P. aeruginosa, NBTI 5463 was bactericidal.  Resistant mutants arose at a low rate and the mutations were found exclusively in the nfxB gene, a regulator of the MexCD-OprJ efflux system.  Levofloxacin-selected resistance mutations in GyrA did not result in decreased susceptibility to NBTI 5463.  Animal infection studies demonstrated that NBTI 5463 was efficacious in mouse models of lung, thigh and ascending urinary tract infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDlCffNen7R7Vg90H21EOLACvtfcHk0lh8SSuEelCHpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXovFOks7o%253D&md5=b60c0469b036e6698aaeeed604dd0145</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FAAC.02778-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02778-13%26sid%3Dliteratum%253Aachs%26aulast%3DDougherty%26aufirst%3DT.%2BJ.%26aulast%3DNayar%26aufirst%3DA.%26aulast%3DNewman%26aufirst%3DJ.%2BV.%26aulast%3DHopkins%26aufirst%3DS.%26aulast%3DStone%26aufirst%3DG.%2BG.%26aulast%3DJohnstone%26aufirst%3DM.%26aulast%3DShapiro%26aufirst%3DA.%2BB.%26aulast%3DCronin%26aufirst%3DM.%26aulast%3DReck%26aufirst%3DF.%26aulast%3DEhmann%26aufirst%3DD.%2BE.%26atitle%3DNbti%25205463%2520Is%2520a%2520Novel%2520Bacterial%2520Type%2520Ii%2520Topoisomerase%2520Inhibitor%2520with%2520Activity%2520against%2520Gram-Negative%2520Bacteria%2520and%2520in%2520Vivo%2520Efficacy%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2014%26volume%3D58%26spage%3D2657%26epage%3D2664%26doi%3D10.1128%2FAAC.02778-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubschwerlen, C.</span>; <span class="NLM_string-name">Ritz, D.</span>; <span class="NLM_string-name">Rueedi, G.</span>; <span class="NLM_string-name">Surivet, J.-P.</span>; <span class="NLM_string-name">Acklin, C. Z.</span></span> <span> </span><span class="NLM_article-title">Tricyclic Oxazolidinone Compounds as Antibiotics and Their Preparation and Use in the Treatment of Bacterial Infection</span>. <span class="NLM_patent">WO 2,010,041,194 A1</span>, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=C.+Hubschwerlen&author=D.+Ritz&author=G.+Rueedi&author=J.-P.+Surivet&author=C.+Z.+Acklin&title=Tricyclic+Oxazolidinone+Compounds+as+Antibiotics+and+Their+Preparation+and+Use+in+the+Treatment+of+Bacterial+Infection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHubschwerlen%26aufirst%3DC.%26atitle%3DTricyclic%2520Oxazolidinone%2520Compounds%2520as%2520Antibiotics%2520and%2520Their%2520Preparation%2520and%2520Use%2520in%2520the%2520Treatment%2520of%2520Bacterial%2520Infection%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parr, T.</span>; <span class="NLM_string-name">Hammond, J.</span>; <span class="NLM_string-name">Tateda, K.</span>; <span class="NLM_string-name">Reck, F.</span>; <span class="NLM_string-name">Rueedi, G.</span>; <span class="NLM_string-name">MAsuda, N.</span>; <span class="NLM_string-name">Rahman, k.</span>; <span class="NLM_string-name">Katvars, L.</span>; <span class="NLM_string-name">Sullivan, S.</span></span> In  <i>New Agents Discovery Summary Session: Early New Antimicrobial Agents, ASM-Microbe 2017</i>, New Orleans, June 1–5; <span class="NLM_publisher-name">ASM Microbe</span>;  <span class="NLM_fpage">Sa261</span>– <span class="NLM_lpage">Sa265</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=Sa261-Sa265&author=T.+Parr&author=J.+Hammond&author=K.+Tateda&author=F.+Reck&author=G.+Rueedi&author=N.+MAsuda&author=k.+Rahman&author=L.+Katvars&author=S.+Sullivan&title=New+Agents+Discovery+Summary+Session%3A+Early+New+Antimicrobial+Agents%2C+ASM-Microbe+2017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DParr%26aufirst%3DT.%26btitle%3DNew%2520Agents%2520Discovery%2520Summary%2520Session%253A%2520Early%2520New%2520Antimicrobial%2520Agents%252C%2520ASM-Microbe%25202017%26pub%3DASM%2520Microbe%26spage%3DSa261%26epage%3DSa265%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toussaint, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchiya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlisi, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinke, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miesel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagrutta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishii, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oohata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span> <span> </span><span class="NLM_article-title"><i>In Vitro</i> and <i>in Vivo</i> Characterization of the Novel Oxabicyclooctane-Linked Bacterial Topoisomerase Inhibitor Am-8722, a Selective, Potent Inhibitor of Bacterial DNA Gyrase</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4830</span>– <span class="NLM_lpage">4839</span>, <span class="refDoi"> DOI: 10.1128/AAC.00619-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.00619-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=27246784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFSis7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2016&pages=4830-4839&author=C.+M.+Tanauthor=C.+J.+Gillauthor=J.+Wuauthor=N.+Toussaintauthor=J.+Yinauthor=T.+Tsuchiyaauthor=C.+G.+Garlisiauthor=D.+Kaelinauthor=P.+T.+Meinkeauthor=L.+Mieselauthor=D.+B.+Olsenauthor=A.+Lagruttaauthor=H.+Fukudaauthor=R.+Kishiiauthor=M.+Takeiauthor=K.+Oohataauthor=T.+Takeuchiauthor=T.+Shibueauthor=H.+Takanoauthor=A.+Nishimuraauthor=Y.+Fukudaauthor=S.+B.+Singh&title=In+Vitro+and+in+Vivo+Characterization+of+the+Novel+Oxabicyclooctane-Linked+Bacterial+Topoisomerase+Inhibitor+Am-8722%2C+a+Selective%2C+Potent+Inhibitor+of+Bacterial+DNA+Gyrase&doi=10.1128%2FAAC.00619-16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro and in vivo characterization of the novel oxabicyclooctane-linked bacterial topoisomerase inhibitor AM-8722, a selective, potent inhibitor of bacterial DNA gyrase</span></div><div class="casAuthors">Tan, Christopher M.; Gill, Charles J.; Wu, Jin; Toussaint, Nathalie; Yin, Jingjun; Tsuchiya, Takayuki; Garlisi, Charles G.; Kaelin, David; Meinke, Peter T.; Miesel, Lynn; Olsen, David B.; Lagrutta, Armando; Fukuda, Hideyuki; Kishii, Ryuta; Takei, Masaya; Oohata, Kouhei; Takeuchi, Tomoko; Shibue, Taku; Takano, Hisashi; Nishimura, Akinori; Fukuda, Yasumichi; Singh, Sheo B.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4830-4839</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Oxabicyclooctane-linked novel bacterial topoisomerase inhibitors (NBTIs) represent a new class of recently described antibacterial agents with broad-spectrum activity.  NBTIs dually inhibit the clin. validated bacterial targets DNA gyrase and topoisomerase IV and have been shown to bind distinctly from known classes of antibacterial agents directed against these targets.  Herein we report the mol., cellular, and in vivo characterization of AM-8722 as a representative N-alkylated-1,5-naphthyridone left-hand-side-substituted NBTI.  Consistent with its mode of action, macromol. labeling studies revealed a specific effect of AM-8722 to dose dependently inhibit bacterial DNA synthesis.  AM-8722 displayed greater intrinsic enzymic potency than levofloxacin vs. both DNA gyrase and topoisomerase IV from Staphylococcus aureus and Escherichia coli and displayed selectivity against human topoisomerase II.  AM-8722 was rapidly bactericidal and exhibited whole-cell activity vs. a range of Gram-neg. and Gram-pos. organisms, with no whole-cell potency shift due to the presence of DNA or human serum.  Frequency-of-resistance studies demonstrated an acceptable rate of resistance emergence in vitro at concns. 16- to 32- fold the MIC.  AM-8722 displayed acceptable pharmacokinetic properties and was shown to be efficacious in mouse models of bacterial septicemia.  Overall, AM-8722 is a selective and potent NBTI that displays broad-spectrum antimicrobial activity in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQRBIdegoVg7Vg90H21EOLACvtfcHk0lgruHY_LusSmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFSis7rP&md5=406686030d49a9ce465d6fcaa25a9770</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1128%2FAAC.00619-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00619-16%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DC.%2BM.%26aulast%3DGill%26aufirst%3DC.%2BJ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DToussaint%26aufirst%3DN.%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DTsuchiya%26aufirst%3DT.%26aulast%3DGarlisi%26aufirst%3DC.%2BG.%26aulast%3DKaelin%26aufirst%3DD.%26aulast%3DMeinke%26aufirst%3DP.%2BT.%26aulast%3DMiesel%26aufirst%3DL.%26aulast%3DOlsen%26aufirst%3DD.%2BB.%26aulast%3DLagrutta%26aufirst%3DA.%26aulast%3DFukuda%26aufirst%3DH.%26aulast%3DKishii%26aufirst%3DR.%26aulast%3DTakei%26aufirst%3DM.%26aulast%3DOohata%26aufirst%3DK.%26aulast%3DTakeuchi%26aufirst%3DT.%26aulast%3DShibue%26aufirst%3DT.%26aulast%3DTakano%26aufirst%3DH.%26aulast%3DNishimura%26aufirst%3DA.%26aulast%3DFukuda%26aufirst%3DY.%26aulast%3DSingh%26aufirst%3DS.%2BB.%26atitle%3DIn%2520Vitro%2520and%2520in%2520Vivo%2520Characterization%2520of%2520the%2520Novel%2520Oxabicyclooctane-Linked%2520Bacterial%2520Topoisomerase%2520Inhibitor%2520Am-8722%252C%2520a%2520Selective%252C%2520Potent%2520Inhibitor%2520of%2520Bacterial%2520DNA%2520Gyrase%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2016%26volume%3D60%26spage%3D4830%26epage%3D4839%26doi%3D10.1128%2FAAC.00619-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Charrier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salisbury, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savage, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaffer-Malam, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ooi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGarry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichowicz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigerson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craighead, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratcliffe, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Best, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, N. R.</span></span> <span> </span><span class="NLM_article-title">Novel Bacterial Topoisomerase Inhibitors with Potent Broad-Spectrum Activity against Drug-Resistant Bacteria</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>, <span class="NLM_elocation-id">e02100</span> <span class="refDoi"> DOI: 10.1128/AAC.02100-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.02100-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=28223393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFeks7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&author=C.+Charrierauthor=A.+M.+Salisburyauthor=V.+J.+Savageauthor=T.+Duffyauthor=E.+Moyoauthor=N.+Chaffer-Malamauthor=N.+Ooiauthor=R.+Newmanauthor=J.+Cheungauthor=R.+Metzgerauthor=D.+McGarryauthor=M.+Pichowiczauthor=R.+Sigersonauthor=I.+R.+Cooperauthor=G.+Nelsonauthor=H.+S.+Butlerauthor=M.+Craigheadauthor=A.+J.+Ratcliffeauthor=S.+A.+Bestauthor=N.+R.+Stokes&title=Novel+Bacterial+Topoisomerase+Inhibitors+with+Potent+Broad-Spectrum+Activity+against+Drug-Resistant+Bacteria&doi=10.1128%2FAAC.02100-16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Novel bacterial topoisomerase inhibitors with potent broad-spectrum activity against drug-resistant bacteria</span></div><div class="casAuthors">Charrier, Cedric; Salisbury, Anne-Marie; Savage, Victoria J.; Duffy, Thomas; Moyo, Emmanuel; Chaffer-Malam, Nathan; Ooi, Nicola; Newman, Rebecca; Cheung, Jonathan; Metzger, Richard; McGarry, David; Pichowicz, Mark; Sigerson, Ralph; Cooper, Ian R.; Nelson, Gary; Butler, Hayley S.; Craighead, Mark; Ratcliffe, Andrew J.; Best, Stuart A.; Stokes, Neil R.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e02100-16/1-e02100-16/13</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The novel bacterial topoisomerase inhibitor class is an investigational type of antibacterial inhibitor of DNA gyrase and topoisomerase IV that does not have cross-resistance with the quinolones.  Here, we report the evaluation of the in vitro properties of a new series of this type of small mol.  Exemplar compds. selectively and potently inhibited the catalytic activities of Escherichia coli DNA gyrase and topoisomerase IV but did not block the DNA breakage-reunion step.  Compds. showed broad-spectrum inhibitory activity against a wide range of Gram-pos. and Gram-neg. pathogens, including biodefence microorganisms and Mycobacterium tuberculosis.  No cross-resistance with fluoroquinolone-resistant Staphylococcus aureus and E. coli isolates was obsd.  Measured MIC90 values were 4 and 8 μg/mL against a panel of contemporary multidrug-resistant isolates of Acinetobacter baumannii and E. coli, resp.  In addn., representative compds. exhibited greater antibacterial potency than the quinolones against obligate anaerobic species.  Spontaneous mutation rates were low, with frequencies of resistance typically <10-8 against E. coli and A. baumannii at concns. equiv. to 4-fold the MIC.  Compd.-resistant E. coli mutants that were isolated following serial passage were characterized by whole-genome sequencing and carried a single Arg38Leu amino acid substitution in the GyrA subunit of DNA gyrase.  Preliminary in vitro safety data indicate that the series shows a promising therapeutic index and potential for low human ether-a-go-go-related gene (hERG) inhibition (50% inhibitory concn. [IC50], >100 μM).  In summary, the compds.' distinct mechanism of action relative to the fluoroquinolones, whole-cell potency, low potential for resistance development, and favorable in vitro safety profile warrant their continued investigation as potential broad-spectrum antibacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT7wMoQx56jLVg90H21EOLACvtfcHk0lgruHY_LusSmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFeks7jJ&md5=5bb1d8f7b51c869245bfe6b5dd316855</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2FAAC.02100-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02100-16%26sid%3Dliteratum%253Aachs%26aulast%3DCharrier%26aufirst%3DC.%26aulast%3DSalisbury%26aufirst%3DA.%2BM.%26aulast%3DSavage%26aufirst%3DV.%2BJ.%26aulast%3DDuffy%26aufirst%3DT.%26aulast%3DMoyo%26aufirst%3DE.%26aulast%3DChaffer-Malam%26aufirst%3DN.%26aulast%3DOoi%26aufirst%3DN.%26aulast%3DNewman%26aufirst%3DR.%26aulast%3DCheung%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DR.%26aulast%3DMcGarry%26aufirst%3DD.%26aulast%3DPichowicz%26aufirst%3DM.%26aulast%3DSigerson%26aufirst%3DR.%26aulast%3DCooper%26aufirst%3DI.%2BR.%26aulast%3DNelson%26aufirst%3DG.%26aulast%3DButler%26aufirst%3DH.%2BS.%26aulast%3DCraighead%26aufirst%3DM.%26aulast%3DRatcliffe%26aufirst%3DA.%2BJ.%26aulast%3DBest%26aufirst%3DS.%2BA.%26aulast%3DStokes%26aufirst%3DN.%2BR.%26atitle%3DNovel%2520Bacterial%2520Topoisomerase%2520Inhibitors%2520with%2520Potent%2520Broad-Spectrum%2520Activity%2520against%2520Drug-Resistant%2520Bacteria%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26doi%3D10.1128%2FAAC.02100-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitton-Fry, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brickner, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamel, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casavant, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finegan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardink, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huband, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marfat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCurdy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanyam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisialowski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaniewski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zook, C.</span></span> <span> </span><span class="NLM_article-title">Novel 3-Fluoro-6-Methoxyquinoline Derivatives as Inhibitors of Bacterial DNA Gyrase and Topoisomerase Iv</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3353</span>– <span class="NLM_lpage">3358</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2017.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=28610977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVSjtLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=3353-3358&author=M.+J.+Mitton-Fryauthor=S.+J.+Bricknerauthor=J.+C.+Hamelauthor=R.+Barhamauthor=L.+Brennanauthor=J.+M.+Casavantauthor=X.+Dingauthor=S.+Fineganauthor=J.+Hardinkauthor=T.+Hoangauthor=M.+D.+Hubandauthor=M.+Maloneyauthor=A.+Marfatauthor=S.+P.+McCurdyauthor=D.+McLeodauthor=C.+Subramanyamauthor=M.+Plotkinauthor=U.+Reillyauthor=J.+Schaferauthor=G.+G.+Stoneauthor=D.+P.+Uccelloauthor=T.+Wisialowskiauthor=K.+Yoonauthor=R.+Zaniewskiauthor=C.+Zook&title=Novel+3-Fluoro-6-Methoxyquinoline+Derivatives+as+Inhibitors+of+Bacterial+DNA+Gyrase+and+Topoisomerase+Iv&doi=10.1016%2Fj.bmcl.2017.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Novel 3-fluoro-6-methoxyquinoline derivatives as inhibitors of bacterial DNA gyrase and topoisomerase IV</span></div><div class="casAuthors">Mitton-Fry, Mark J.; Brickner, Steven J.; Hamel, Judith C.; Barham, Rose; Brennan, Lori; Casavant, Jeffrey M.; Ding, Xiaoyuan; Finegan, Steven; Hardink, Joel; Hoang, Thuy; Huband, Michael D.; Maloney, Meghan; Marfat, Anthony; McCurdy, Sandra P.; McLeod, Dale; Subramanyam, Chakrapani; Plotkin, Michael; Reilly, Usa; Schafer, John; Stone, Gregory G.; Uccello, Daniel P.; Wisialowski, Todd; Yoon, Kwansik; Zaniewski, Richard; Zook, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3353-3358</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Novel (non-fluoroquinolone) inhibitors of bacterial type II topoisomerases (NBTIs) are an emerging class of antibacterial agents.  The authors report an optimized series of cyclobutylaryl-substituted NBTIs.  Compd. 14 (I) demonstrated excellent activity both in vitro (S. aureus MIC90 = 0.125 μg/mL) and in vivo (systemic and tissue infections).  Enhanced inhibition of Topoisomerase IV correlated with improved activity in S. aureus strains with mutations conferring resistance to NBTIs.  Compd. 14 also displayed an improved hERG IC50 of 85.9 μM and a favorable profile in the anesthetized guinea pig model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXkMNn5y1uArVg90H21EOLACvtfcHk0lh0plyJ5fxUGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVSjtLfN&md5=5a675cd783671f0591cb1d394e9f166e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DMitton-Fry%26aufirst%3DM.%2BJ.%26aulast%3DBrickner%26aufirst%3DS.%2BJ.%26aulast%3DHamel%26aufirst%3DJ.%2BC.%26aulast%3DBarham%26aufirst%3DR.%26aulast%3DBrennan%26aufirst%3DL.%26aulast%3DCasavant%26aufirst%3DJ.%2BM.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DFinegan%26aufirst%3DS.%26aulast%3DHardink%26aufirst%3DJ.%26aulast%3DHoang%26aufirst%3DT.%26aulast%3DHuband%26aufirst%3DM.%2BD.%26aulast%3DMaloney%26aufirst%3DM.%26aulast%3DMarfat%26aufirst%3DA.%26aulast%3DMcCurdy%26aufirst%3DS.%2BP.%26aulast%3DMcLeod%26aufirst%3DD.%26aulast%3DSubramanyam%26aufirst%3DC.%26aulast%3DPlotkin%26aufirst%3DM.%26aulast%3DReilly%26aufirst%3DU.%26aulast%3DSchafer%26aufirst%3DJ.%26aulast%3DStone%26aufirst%3DG.%2BG.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DWisialowski%26aufirst%3DT.%26aulast%3DYoon%26aufirst%3DK.%26aulast%3DZaniewski%26aufirst%3DR.%26aulast%3DZook%26aufirst%3DC.%26atitle%3DNovel%25203-Fluoro-6-Methoxyquinoline%2520Derivatives%2520as%2520Inhibitors%2520of%2520Bacterial%2520DNA%2520Gyrase%2520and%2520Topoisomerase%2520Iv%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D3353%26epage%3D3358%26doi%3D10.1016%2Fj.bmcl.2017.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitton-Fry, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brickner, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamel, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casavant, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driscoll, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardink, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huband, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marfat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCurdy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepard, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanyam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaniewski, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zook, C.</span></span> <span> </span><span class="NLM_article-title">Novel Quinoline Derivatives as Inhibitors of Bacterial DNA Gyrase and Topoisomerase Iv</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2955</span>– <span class="NLM_lpage">2961</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.03.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2013.03.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=23566517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlslSqtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=2955-2961&author=M.+J.+Mitton-Fryauthor=S.+J.+Bricknerauthor=J.+C.+Hamelauthor=L.+Brennanauthor=J.+M.+Casavantauthor=M.+Chenauthor=T.+Chenauthor=X.+Dingauthor=J.+Driscollauthor=J.+Hardinkauthor=T.+Hoangauthor=E.+Huaauthor=M.+D.+Hubandauthor=M.+Maloneyauthor=A.+Marfatauthor=S.+P.+McCurdyauthor=D.+McLeodauthor=M.+Plotkinauthor=U.+Reillyauthor=S.+Robinsonauthor=J.+Schaferauthor=R.+M.+Shepardauthor=J.+F.+Smithauthor=G.+G.+Stoneauthor=C.+Subramanyamauthor=K.+Yoonauthor=W.+Yuanauthor=R.+P.+Zaniewskiauthor=C.+Zook&title=Novel+Quinoline+Derivatives+as+Inhibitors+of+Bacterial+DNA+Gyrase+and+Topoisomerase+Iv&doi=10.1016%2Fj.bmcl.2013.03.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Novel quinoline derivatives as inhibitors of bacterial DNA gyrase and topoisomerase IV</span></div><div class="casAuthors">Mitton-Fry, Mark J.; Brickner, Steven J.; Hamel, Judith C.; Brennan, Lori; Casavant, Jeffrey M.; Chen, Michael; Chen, Tao; Ding, Xiaoyuan; Driscoll, James; Hardink, Joel; Hoang, Thuy; Hua, Erbing; Huband, Michael D.; Maloney, Meghan; Marfat, Anthony; McCurdy, Sandra P.; McLeod, Dale; Plotkin, Michael; Reilly, Usa; Robinson, Shaughn; Schafer, John; Shepard, Richard M.; Smith, James F.; Stone, Gregory G.; Subramanyam, Chakrapani; Yoon, Kwansik; Yuan, Wei; Zaniewski, Richard P.; Zook, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2955-2961</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A structurally novel set of inhibitors of bacterial type II topoisomerases with potent in vitro and in vivo antibacterial activity was developed.  Dual-targeting ability, hERG inhibition, and pharmacokinetic properties were also assessed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAbdQVIkCXC7Vg90H21EOLACvtfcHk0lh0plyJ5fxUGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlslSqtrg%253D&md5=5dea2df13eb5fb785805ea8cba229fc0</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.03.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.03.047%26sid%3Dliteratum%253Aachs%26aulast%3DMitton-Fry%26aufirst%3DM.%2BJ.%26aulast%3DBrickner%26aufirst%3DS.%2BJ.%26aulast%3DHamel%26aufirst%3DJ.%2BC.%26aulast%3DBrennan%26aufirst%3DL.%26aulast%3DCasavant%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DDriscoll%26aufirst%3DJ.%26aulast%3DHardink%26aufirst%3DJ.%26aulast%3DHoang%26aufirst%3DT.%26aulast%3DHua%26aufirst%3DE.%26aulast%3DHuband%26aufirst%3DM.%2BD.%26aulast%3DMaloney%26aufirst%3DM.%26aulast%3DMarfat%26aufirst%3DA.%26aulast%3DMcCurdy%26aufirst%3DS.%2BP.%26aulast%3DMcLeod%26aufirst%3DD.%26aulast%3DPlotkin%26aufirst%3DM.%26aulast%3DReilly%26aufirst%3DU.%26aulast%3DRobinson%26aufirst%3DS.%26aulast%3DSchafer%26aufirst%3DJ.%26aulast%3DShepard%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BF.%26aulast%3DStone%26aufirst%3DG.%2BG.%26aulast%3DSubramanyam%26aufirst%3DC.%26aulast%3DYoon%26aufirst%3DK.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DZaniewski%26aufirst%3DR.%2BP.%26aulast%3DZook%26aufirst%3DC.%26atitle%3DNovel%2520Quinoline%2520Derivatives%2520as%2520Inhibitors%2520of%2520Bacterial%2520DNA%2520Gyrase%2520and%2520Topoisomerase%2520Iv%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D2955%26epage%3D2961%26doi%3D10.1016%2Fj.bmcl.2013.03.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ehmann, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahiri, S. D.</span></span> <span> </span><span class="NLM_article-title">Novel Compounds Targeting Bacterial DNA Topoisomerase/DNA Gyrase</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">83</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2014.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.coph.2014.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=25271174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFylurjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2014&pages=76-83&author=D.+E.+Ehmannauthor=S.+D.+Lahiri&title=Novel+Compounds+Targeting+Bacterial+DNA+Topoisomerase%2FDNA+Gyrase&doi=10.1016%2Fj.coph.2014.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Novel compounds targeting bacterial DNA topoisomerase/DNA gyrase</span></div><div class="casAuthors">Ehmann, David E.; Lahiri, Sushmita D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">76-83</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Among the targets for the development of new antibacterial agents, bacterial topoisomerases remain a vibrant area of discovery.  A structurally diverse set of inhibitors that bind to the ATP (ATP) site of type II topoisomerases have been disclosed recently.  Seven compds. with this mechanism are highlighted, focusing on antibacterial potency and spectrum, as well as examples of in vivo efficacy against pathogens including Staphylococcus aureus and Mycobacterium tuberculosis.  Five compds. from two structural classes are exemplified that are inhibitors that bind to the catalytic site of DNA gyrase and topoisomerase IV.  The pharmacokinetic and pharmacodynamic properties of these mols., derived from in vivo efficacy against Gram-pos. and Gram-neg. pathogens, define the potential for these agents with broad-spectrum and targeted-spectrum clin. utilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtyDfk3On92LVg90H21EOLACvtfcHk0lh0plyJ5fxUGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFylurjP&md5=d9d7ddb464477c921e073f8076e219e7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2014.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2014.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DEhmann%26aufirst%3DD.%2BE.%26aulast%3DLahiri%26aufirst%3DS.%2BD.%26atitle%3DNovel%2520Compounds%2520Targeting%2520Bacterial%2520DNA%2520Topoisomerase%252FDNA%2520Gyrase%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2014%26volume%3D18%26spage%3D76%26epage%3D83%26doi%3D10.1016%2Fj.coph.2014.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolarič, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minovski, N.</span></span> <span> </span><span class="NLM_article-title">Novel Bacterial Topoisomerase Inhibitors: Challenges and Perspectives in Reducing Herg Toxicity</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2241</span>– <span class="NLM_lpage">2244</span>, <span class="refDoi"> DOI: 10.4155/fmc-2018-0272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.4155%2Ffmc-2018-0272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=30215281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVaqtb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=2241-2244&author=A.+Kolari%C4%8Dauthor=N.+Minovski&title=Novel+Bacterial+Topoisomerase+Inhibitors%3A+Challenges+and+Perspectives+in+Reducing+Herg+Toxicity&doi=10.4155%2Ffmc-2018-0272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Novel bacterial topoisomerase inhibitors: challenges and perspectives in reducing hERG toxicity</span></div><div class="casAuthors">Kolaric, Anja; Minovski, Nikola</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2241-2244</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA1NIj4g1xlrVg90H21EOLACvtfcHk0lglu2DMW519Sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVaqtb3K&md5=fd3a6e97393867ff99f72f003c0182c1</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2018-0272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2018-0272%26sid%3Dliteratum%253Aachs%26aulast%3DKolari%25C4%258D%26aufirst%3DA.%26aulast%3DMinovski%26aufirst%3DN.%26atitle%3DNovel%2520Bacterial%2520Topoisomerase%2520Inhibitors%253A%2520Challenges%2520and%2520Perspectives%2520in%2520Reducing%2520Herg%2520Toxicity%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2018%26volume%3D10%26spage%3D2241%26epage%3D2244%26doi%3D10.4155%2Ffmc-2018-0272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bax, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggleston, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fosberry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gentry, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorrec, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennessy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hibbs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spitzfaden, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shillings, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theobald, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wohlkonig, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwynn, M. N.</span></span> <span> </span><span class="NLM_article-title">Type Iia Topoisomerase Inhibition by a New Class of Antibacterial Agents</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>466</i></span>,  <span class="NLM_fpage">935</span>– <span class="NLM_lpage">940</span>, <span class="refDoi"> DOI: 10.1038/nature09197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1038%2Fnature09197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=20686482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A280%3ADC%252BC3cjmtFaktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2010&pages=935-940&author=B.+D.+Baxauthor=P.+F.+Chanauthor=D.+S.+Egglestonauthor=A.+Fosberryauthor=D.+R.+Gentryauthor=F.+Gorrecauthor=I.+Giordanoauthor=M.+M.+Hannauthor=A.+Hennessyauthor=M.+Hibbsauthor=J.+Huangauthor=E.+Jonesauthor=J.+Jonesauthor=K.+K.+Brownauthor=C.+J.+Lewisauthor=E.+W.+Mayauthor=M.+R.+Saundersauthor=O.+Singhauthor=C.+E.+Spitzfadenauthor=C.+Shenauthor=A.+Shillingsauthor=A.+J.+Theobaldauthor=A.+Wohlkonigauthor=N.+D.+Pearsonauthor=M.+N.+Gwynn&title=Type+Iia+Topoisomerase+Inhibition+by+a+New+Class+of+Antibacterial+Agents&doi=10.1038%2Fnature09197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Type IIA topoisomerase inhibition by a new class of antibacterial agents</span></div><div class="casAuthors">Bax Benjamin D; Chan Pan F; Eggleston Drake S; Fosberry Andrew; Gentry Daniel R; Gorrec Fabrice; Giordano Ilaria; Hann Michael M; Hennessy Alan; Hibbs Martin; Huang Jianzhong; Jones Emma; Jones Jo; Brown Kristin Koretke; Lewis Ceri J; May Earl W; Saunders Martin R; Singh Onkar; Spitzfaden Claus E; Shen Carol; Shillings Anthony; Theobald Andrew J; Wohlkonig Alexandre; Pearson Neil D; Gwynn Michael N</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">7309</span>),
    <span class="NLM_cas:pages">935-40</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Despite the success of genomics in identifying new essential bacterial genes, there is a lack of sustainable leads in antibacterial drug discovery to address increasing multidrug resistance.  Type IIA topoisomerases cleave and religate DNA to regulate DNA topology and are a major class of antibacterial and anticancer drug targets, yet there is no well developed structural basis for understanding drug action.  Here we report the 2.1 A crystal structure of a potent, new class, broad-spectrum antibacterial agent in complex with Staphylococcus aureus DNA gyrase and DNA, showing a new mode of inhibition that circumvents fluoroquinolone resistance in this clinically important drug target.  The inhibitor 'bridges' the DNA and a transient non-catalytic pocket on the two-fold axis at the GyrA dimer interface, and is close to the active sites and fluoroquinolone binding sites.  In the inhibitor complex the active site seems poised to cleave the DNA, with a single metal ion observed between the TOPRIM (topoisomerase/primase) domain and the scissile phosphate.  This work provides new insights into the mechanism of topoisomerase action and a platform for structure-based drug design of a new class of antibacterial agents against a clinically proven, but conformationally flexible, enzyme class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTzBPk2FYUX-kBuFdjgVNKSfW6udTcc2ebVG8ZUUAVFCbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cjmtFaktA%253D%253D&md5=006fef200c7a44f0671829b2a5fbddef</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnature09197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09197%26sid%3Dliteratum%253Aachs%26aulast%3DBax%26aufirst%3DB.%2BD.%26aulast%3DChan%26aufirst%3DP.%2BF.%26aulast%3DEggleston%26aufirst%3DD.%2BS.%26aulast%3DFosberry%26aufirst%3DA.%26aulast%3DGentry%26aufirst%3DD.%2BR.%26aulast%3DGorrec%26aufirst%3DF.%26aulast%3DGiordano%26aufirst%3DI.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DHennessy%26aufirst%3DA.%26aulast%3DHibbs%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DE.%26aulast%3DJones%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DK.%2BK.%26aulast%3DLewis%26aufirst%3DC.%2BJ.%26aulast%3DMay%26aufirst%3DE.%2BW.%26aulast%3DSaunders%26aufirst%3DM.%2BR.%26aulast%3DSingh%26aufirst%3DO.%26aulast%3DSpitzfaden%26aufirst%3DC.%2BE.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DShillings%26aufirst%3DA.%26aulast%3DTheobald%26aufirst%3DA.%2BJ.%26aulast%3DWohlkonig%26aufirst%3DA.%26aulast%3DPearson%26aufirst%3DN.%2BD.%26aulast%3DGwynn%26aufirst%3DM.%2BN.%26atitle%3DType%2520Iia%2520Topoisomerase%2520Inhibition%2520by%2520a%2520New%2520Class%2520of%2520Antibacterial%2520Agents%26jtitle%3DNature%26date%3D2010%26volume%3D466%26spage%3D935%26epage%3D940%26doi%3D10.1038%2Fnature09197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drlica, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiasa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustaev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span> <span> </span><span class="NLM_article-title">Quinolones: Action and Resistance Updated</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">981</span>– <span class="NLM_lpage">998</span>, <span class="refDoi"> DOI: 10.2174/156802609789630947</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.2174%2F156802609789630947" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=19747119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFWisb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=981-998&author=K.+Drlicaauthor=H.+Hiasaauthor=R.+Kernsauthor=M.+Malikauthor=A.+Mustaevauthor=X.+Zhao&title=Quinolones%3A+Action+and+Resistance+Updated&doi=10.2174%2F156802609789630947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Quinolones: action and resistance updated</span></div><div class="casAuthors">Drlica, Karl; Hiasa, Hiroshi; Kerns, Robert; Malik, Muhammad; Mustaev, Arkady; Zhao, Xilin</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">981-998</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The quinolones trap DNA gyrase and DNA topoisomerase IV on DNA as complexes in which the DNA is broken but constrained by protein.  Early studies suggested that drug binding occurs largely along helix-4 of the GyrA (gyrase) and ParC (topoisomerase IV) proteins.  However, recent X-ray crystallog. shows drug intercalating between the -1 and +1 nucleotides of cut DNA, with only one end of the drug extending to helix-4.  These two models may reflect distinct structural steps in complex formation.  A consequence of drug-enzyme-DNA complex formation is reversible inhibition of DNA replication; cell death arises from subsequent events in which bacterial chromosomes are fragmented through two poorly understood pathways.  In one pathway, chromosome fragmentation stimulates excessive accumulation of highly toxic reactive oxygen species that are responsible for cell death.  Quinolone resistance arises stepwise through selective amplification of mutants when drug concns. are above the MIC and below the MPC, as obsd. with static agar plate assays, dynamic in vitro systems, and exptl. infection of rabbits.  The gap between MIC and MPC can be narrowed by compd. design that should restrict the emergence of resistance.  Resistance is likely to become increasingly important, since three types of plasmid-borne resistance have been reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwqmNnQ2e72rVg90H21EOLACvtfcHk0lglu2DMW519Sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFWisb%252FF&md5=e8f8b87460d826d75acd11efceb1a7f4</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2174%2F156802609789630947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802609789630947%26sid%3Dliteratum%253Aachs%26aulast%3DDrlica%26aufirst%3DK.%26aulast%3DHiasa%26aufirst%3DH.%26aulast%3DKerns%26aufirst%3DR.%26aulast%3DMalik%26aufirst%3DM.%26aulast%3DMustaev%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DX.%26atitle%3DQuinolones%253A%2520Action%2520and%2520Resistance%2520Updated%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2009%26volume%3D9%26spage%3D981%26epage%3D998%26doi%3D10.2174%2F156802609789630947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Schrödinger
LLC</span>.  <i>Schrödinger Release 2013-1:
Glide</i>. <span class="NLM_publisher-name">Schrödinger LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=Schr%C3%B6dinger%0ALLC&title=Schr%C3%B6dinger+Release+2013-1%3A%0AGlide"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DSchr%25C3%25B6dinger%2520Release%25202013-1%253A%250AGlide%26pub%3DSchr%25C3%25B6dinger%2520LLC%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reck, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alm, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassil, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dejonge, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyermann, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breault, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comita-Prevoir, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galullo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grebe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morningstar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayter, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, S.</span></span> <span> </span><span class="NLM_article-title">Novel N-Linked Aminopiperidine Inhibitors of Bacterial Topoisomerase Type Ii: Broad-Spectrum Antibacterial Agents with Reduced Herg Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7834</span>– <span class="NLM_lpage">7847</span>, <span class="refDoi"> DOI: 10.1021/jm2008826</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2008826" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlWrs7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7834-7847&author=F.+Reckauthor=R.+Almauthor=P.+Brassilauthor=J.+Newmanauthor=B.+Dejongeauthor=C.+J.+Eyermannauthor=G.+Breaultauthor=J.+Breenauthor=J.+Comita-Prevoirauthor=M.+Croninauthor=H.+Davisauthor=D.+Ehmannauthor=V.+Galulloauthor=B.+Gengauthor=T.+Grebeauthor=M.+Morningstarauthor=P.+Walkerauthor=B.+Hayterauthor=S.+Fisher&title=Novel+N-Linked+Aminopiperidine+Inhibitors+of+Bacterial+Topoisomerase+Type+Ii%3A+Broad-Spectrum+Antibacterial+Agents+with+Reduced+Herg+Activity&doi=10.1021%2Fjm2008826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Novel N-Linked Aminopiperidine Inhibitors of Bacterial Topoisomerase Type II: Broad-Spectrum Antibacterial Agents with Reduced hERG Activity</span></div><div class="casAuthors">Reck, Folkert; Alm, Richard; Brassil, Patrick; Newman, Joseph; DeJonge, Boudewijn; Eyermann, Charles J.; Breault, Gloria; Breen, John; Comita-Prevoir, Janelle; Cronin, Mark; Davis, Hajnalka; Ehmann, David; Galullo, Vincent; Geng, Bolin; Grebe, Tyler; Morningstar, Marshall; Walker, Phil; Hayter, Barry; Fisher, Stewart</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7834-7847</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Novel non-fluoroquinolone inhibitors of bacterial type II topoisomerases (DNA gyrase and topoisomerase IV) are of interest for the development of new antibacterial agents that are not impacted by target-mediated cross-resistance with fluoroquinolones.  Aminopiperidines that have a bicyclic arom. moiety linked through a carbon to an Et bridge, such as I, generally show potent broad-spectrum antibacterial activity, including quinolone-resistant isolates, but suffer from potent hERG inhibition (IC50= 3 μM for 1).  We now disclose the finding that new analogs of I with an N-linked cyclic amide moiety attached to the Et bridge, such as II, retain the broad-spectrum antibacterial activity of I but show significantly less hERG inhibition (IC50= 31 μM for II) and higher free fraction than 1.  One optimized analog showed moderate clearance in the dog and promising efficacy against Staphylococcus aureus in a mouse thigh infection model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohmQMvZgbqsbVg90H21EOLACvtfcHk0ljKkYFimI2YXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlWrs7fM&md5=ec76eb59cded28eb4c80627166f3e280</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm2008826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2008826%26sid%3Dliteratum%253Aachs%26aulast%3DReck%26aufirst%3DF.%26aulast%3DAlm%26aufirst%3DR.%26aulast%3DBrassil%26aufirst%3DP.%26aulast%3DNewman%26aufirst%3DJ.%26aulast%3DDejonge%26aufirst%3DB.%26aulast%3DEyermann%26aufirst%3DC.%2BJ.%26aulast%3DBreault%26aufirst%3DG.%26aulast%3DBreen%26aufirst%3DJ.%26aulast%3DComita-Prevoir%26aufirst%3DJ.%26aulast%3DCronin%26aufirst%3DM.%26aulast%3DDavis%26aufirst%3DH.%26aulast%3DEhmann%26aufirst%3DD.%26aulast%3DGalullo%26aufirst%3DV.%26aulast%3DGeng%26aufirst%3DB.%26aulast%3DGrebe%26aufirst%3DT.%26aulast%3DMorningstar%26aufirst%3DM.%26aulast%3DWalker%26aufirst%3DP.%26aulast%3DHayter%26aufirst%3DB.%26aulast%3DFisher%26aufirst%3DS.%26atitle%3DNovel%2520N-Linked%2520Aminopiperidine%2520Inhibitors%2520of%2520Bacterial%2520Topoisomerase%2520Type%2520Ii%253A%2520Broad-Spectrum%2520Antibacterial%2520Agents%2520with%2520Reduced%2520Herg%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7834%26epage%3D7847%26doi%3D10.1021%2Fjm2008826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miles, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennessy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cailleau, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dabbs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esken, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoover, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukmar, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spitzfaden, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markwell, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minthorn, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rittenhouse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwynn, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, N. D.</span></span> <span> </span><span class="NLM_article-title">Novel Hydroxyl Tricyclics (E.G., Gsk966587) as Potent Inhibitors of Bacterial Type Iia Topoisomerases</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5437</span>– <span class="NLM_lpage">5441</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2013.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=23968823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlShtrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=5437-5441&author=T.+J.+Milesauthor=A.+J.+Hennessyauthor=B.+Baxauthor=G.+Brooksauthor=B.+S.+Brownauthor=P.+Brownauthor=N.+Cailleauauthor=D.+Chenauthor=S.+Dabbsauthor=D.+T.+Daviesauthor=J.+M.+Eskenauthor=I.+Giordanoauthor=J.+L.+Hooverauthor=J.+Huangauthor=G.+E.+Jonesauthor=S.+K.+Sukmarauthor=C.+Spitzfadenauthor=R.+E.+Markwellauthor=E.+A.+Minthornauthor=S.+Rittenhouseauthor=M.+N.+Gwynnauthor=N.+D.+Pearson&title=Novel+Hydroxyl+Tricyclics+%28E.G.%2C+Gsk966587%29+as+Potent+Inhibitors+of+Bacterial+Type+Iia+Topoisomerases&doi=10.1016%2Fj.bmcl.2013.07.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Novel hydroxyl tricyclics (e.g., GSK966587) as potent inhibitors of bacterial type IIA topoisomerases</span></div><div class="casAuthors">Miles, Timothy J.; Hennessy, Alan J.; Bax, Ben; Brooks, Gerald; Brown, Barry S.; Brown, Pamela; Cailleau, Nathalie; Chen, Dongzhao; Dabbs, Steven; Davies, David T.; Esken, Joel M.; Giordano, Ilaria; Hoover, Jennifer L.; Huang, Jianzhong; Jones, Graham E.; Kusalakumari Sukmar, Senthill K.; Spitzfaden, Claus; Markwell, Roger E.; Minthorn, Elisabeth A.; Rittenhouse, Steve; Gwynn, Michael N.; Pearson, Neil D.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5437-5441</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">During the course of the authors' research to find novel mode of action antibacterials, the authors discovered a series of hydroxyl tricyclic compds. that showed good potency against Gram-pos. and Gram-neg. pathogens.  These compds. inhibit bacterial type IIA topoisomerases.  Herein the authors will discuss structure-activity relationships in this series and report advanced studies on compd. 1 (GSK966587) which demonstrates good PK and in vivo efficacy properties.  X-ray crystallog. studies were used to provide insight into the structural basis for the difference in antibacterial potency between enantiomers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz_XwqrKpCrLVg90H21EOLACvtfcHk0ljKkYFimI2YXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlShtrjN&md5=976c6c41b726dc03f0e5b543406f04b4</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DMiles%26aufirst%3DT.%2BJ.%26aulast%3DHennessy%26aufirst%3DA.%2BJ.%26aulast%3DBax%26aufirst%3DB.%26aulast%3DBrooks%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DB.%2BS.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DCailleau%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDabbs%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DD.%2BT.%26aulast%3DEsken%26aufirst%3DJ.%2BM.%26aulast%3DGiordano%26aufirst%3DI.%26aulast%3DHoover%26aufirst%3DJ.%2BL.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DG.%2BE.%26aulast%3DSukmar%26aufirst%3DS.%2BK.%26aulast%3DSpitzfaden%26aufirst%3DC.%26aulast%3DMarkwell%26aufirst%3DR.%2BE.%26aulast%3DMinthorn%26aufirst%3DE.%2BA.%26aulast%3DRittenhouse%26aufirst%3DS.%26aulast%3DGwynn%26aufirst%3DM.%2BN.%26aulast%3DPearson%26aufirst%3DN.%2BD.%26atitle%3DNovel%2520Hydroxyl%2520Tricyclics%2520%2528E.G.%252C%2520Gsk966587%2529%2520as%2520Potent%2520Inhibitors%2520of%2520Bacterial%2520Type%2520Iia%2520Topoisomerases%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D5437%26epage%3D5441%26doi%3D10.1016%2Fj.bmcl.2013.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Schrödinger
LLC</span>.  <i>Schrödinger Release 2013-1:
Phase</i>. <span class="NLM_publisher-name">Schrödinger LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=Schr%C3%B6dinger%0ALLC&title=Schr%C3%B6dinger+Release+2013-1%3A%0APhase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DSchr%25C3%25B6dinger%2520Release%25202013-1%253A%250APhase%26pub%3DSchr%25C3%25B6dinger%2520LLC%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sastry, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Rapid Shape-Based Ligand Alignment and Virtual Screening Method Based on Atom/Feature-Pair Similarities and Volume Overlap Scoring</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2455</span>– <span class="NLM_lpage">2466</span>, <span class="refDoi"> DOI: 10.1021/ci2002704</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci2002704" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKnt73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2455-2466&author=G.+M.+Sastryauthor=S.+L.+Dixonauthor=W.+Sherman&title=Rapid+Shape-Based+Ligand+Alignment+and+Virtual+Screening+Method+Based+on+Atom%2FFeature-Pair+Similarities+and+Volume+Overlap+Scoring&doi=10.1021%2Fci2002704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Shape-Based Ligand Alignment and Virtual Screening Method Based on Atom/Feature-Pair Similarities and Volume Overlap Scoring</span></div><div class="casAuthors">Sastry, G. Madhavi; Dixon, Steven L.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2455-2466</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Shape-based methods for aligning and scoring ligands have proven to be valuable in the field of computer-aided drug design.  Here, we describe a new shape-based flexible ligand superposition and virtual screening method, Phase Shape, which is shown to rapidly produce accurate 3D ligand alignments and efficiently enrich actives in virtual screening.  We describe the methodol., which is based on the principle of atom distribution triplets to rapidly define trial alignments, followed by refinement of top alignments to maximize the vol. overlap.  The method can be run in a shape-only mode or it can include atom types or pharmacophore feature encoding, the latter consistently producing the best results for database screening.  We apply Phase Shape to flexibly align mols. that bind to the same target and show that the method consistently produces correct alignments when compared with crystal structures.  We then illustrate the effectiveness of the method for identifying active compds. in virtual screening of eleven diverse targets.  Multiple parameters are explored, including atom typing, query structure conformation, and the database conformer generation protocol.  We show that Phase Shape performs well in database screening calcns. when compared with other shape-based methods using a common set of actives and decoys from the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM-u23DvDpQ7Vg90H21EOLACvtfcHk0lhp6Wfsz_bhEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKnt73K&md5=4234abb2dd7e6364c377e92047b65aba</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci2002704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2002704%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DG.%2BM.%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DRapid%2520Shape-Based%2520Ligand%2520Alignment%2520and%2520Virtual%2520Screening%2520Method%2520Based%2520on%2520Atom%252FFeature-Pair%2520Similarities%2520and%2520Volume%2520Overlap%2520Scoring%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2455%26epage%3D2466%26doi%3D10.1021%2Fci2002704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Motifs for Molecular Recognition Exploiting Hydrophobic Enclosure in Protein-Ligand Binding</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">808</span>– <span class="NLM_lpage">813</span>, <span class="refDoi"> DOI: 10.1073/pnas.0610202104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1073%2Fpnas.0610202104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=17204562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=808-813&author=T.+Youngauthor=R.+Abelauthor=B.+Kimauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Motifs+for+Molecular+Recognition+Exploiting+Hydrophobic+Enclosure+in+Protein-Ligand+Binding&doi=10.1073%2Fpnas.0610202104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Motifs for molecular recognition exploiting hydrophobic enclosure in protein-ligand binding</span></div><div class="casAuthors">Young, Tom; Abel, Robert; Kim, Byungchan; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">808-813</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The thermodn. properties and phase behavior of water in confined regions can vary significantly from that obsd. in the bulk.  This is particularly true for systems in which the confinement is on the mol.-length scale.  In this study, we use mol. dynamics simulations and a powerful solvent anal. technique based on inhomogeneous solvation theory to investigate the properties of water mols. that solvate the confined regions of protein active sites.  Our simulations and anal. indicate that the solvation of protein active sites that are characterized by hydrophobic enclosure and correlated hydrogen bonds induce atypical entropic and enthalpic penalties of hydration.  These penalties apparently stabilize the protein-ligand complex with respect to the independently solvated ligand and protein, which leads to enhanced binding affinities.  Our anal. elucidates several challenging cases, including the super affinity of the streptavidin-biotin system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWumQ61YTqILVg90H21EOLACvtfcHk0lhp6Wfsz_bhEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D&md5=1109e3f48670cd910885f7fd527eb880</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0610202104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0610202104%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DMotifs%2520for%2520Molecular%2520Recognition%2520Exploiting%2520Hydrophobic%2520Enclosure%2520in%2520Protein-Ligand%2520Binding%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D808%26epage%3D813%26doi%3D10.1073%2Fpnas.0610202104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">2817</span>– <span class="NLM_lpage">2831</span>, <span class="refDoi"> DOI: 10.1021/ja0771033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0771033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=2817-2831&author=R.+Abelauthor=T.+Youngauthor=R.+Faridauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Role+of+the+Active-Site+Solvent+in+the+Thermodynamics+of+Factor+Xa+Ligand+Binding&doi=10.1021%2Fja0771033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span></div><div class="casAuthors">Abel, Robert; Young, Tom; Farid, Ramy; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2817-2831</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding the underlying physics of the binding of small-mol. ligands to protein active sites is a key objective of computational chem. and biol.  It is widely believed that displacement of water mols. from the active site by the ligand is a principal (if not the dominant) source of binding free energy.  Although continuum theories of hydration are routinely used to describe the contributions of the solvent to the binding affinity of the complex, it is still an unsettled question as to whether or not these continuum solvation theories describe the underlying mol. physics with sufficient accuracy to reliably rank the binding affinities of a set of ligands for a given protein.  Here we develop a novel, computationally efficient descriptor of the contribution of the solvent to the binding free energy of a small mol. and its assocd. receptor that captures the effects of the ligand displacing the solvent from the protein active site with at. detail.  This descriptor quant. predicts (R2 = 0.81) the binding free energy differences between congeneric ligand pairs for the test system factor Xa, elucidates phys. properties of the active site solvent that appear to be missing in most continuum theories of hydration, and identifies several features of the hydration of the factor Xa active site relevant to the structure-activity relationship of its inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS42r59nNXH7Vg90H21EOLACvtfcHk0lhp6Wfsz_bhEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D&md5=8eefe76547a82bccaea38f2c84e6cd5f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fja0771033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0771033%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DRole%2520of%2520the%2520Active-Site%2520Solvent%2520in%2520the%2520Thermodynamics%2520of%2520Factor%2520Xa%2520Ligand%2520Binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D2817%26epage%3D2831%26doi%3D10.1021%2Fja0771033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pendleton, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorman, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmore, B. F.</span></span> <span> </span><span class="NLM_article-title">Clinical Relevance of the Eskape Pathogens</span>. <i>Expert Rev. Anti-infect. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1586/eri.13.12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1586%2Feri.13.12" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2014&pages=297-308&author=J.+N.+Pendletonauthor=S.+P.+Gormanauthor=B.+F.+Gilmore&title=Clinical+Relevance+of+the+Eskape+Pathogens&doi=10.1586%2Feri.13.12"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1586%2Feri.13.12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1586%252Feri.13.12%26sid%3Dliteratum%253Aachs%26aulast%3DPendleton%26aufirst%3DJ.%2BN.%26aulast%3DGorman%26aufirst%3DS.%2BP.%26aulast%3DGilmore%26aufirst%3DB.%2BF.%26atitle%3DClinical%2520Relevance%2520of%2520the%2520Eskape%2520Pathogens%26jtitle%3DExpert%2520Rev.%2520Anti-infect.%2520Ther.%26date%3D2014%26volume%3D11%26spage%3D297%26epage%3D308%26doi%3D10.1586%2Feri.13.12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biedenbach, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouchillon, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hackel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scangarella-Oman, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakielaszek, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahm, D. F.</span></span> <span> </span><span class="NLM_article-title">In Vitro Activity of Gepotidacin, a Novel Triazaacenaphthylene Bacterial Topoisomerase Inhibitor, against a Broad Spectrum of Bacterial Pathogens</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1918</span>– <span class="NLM_lpage">1923</span>, <span class="refDoi"> DOI: 10.1128/AAC.02820-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1128%2FAAC.02820-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=26729499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsF2ntbbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2016&pages=1918-1923&author=D.+J.+Biedenbachauthor=S.+K.+Bouchillonauthor=M.+Hackelauthor=L.+A.+Millerauthor=N.+E.+Scangarella-Omanauthor=C.+Jakielaszekauthor=D.+F.+Sahm&title=In+Vitro+Activity+of+Gepotidacin%2C+a+Novel+Triazaacenaphthylene+Bacterial+Topoisomerase+Inhibitor%2C+against+a+Broad+Spectrum+of+Bacterial+Pathogens&doi=10.1128%2FAAC.02820-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro activity of gepotidacin, a novel triazaacenaphthylene bacterial topoisomerase inhibitor, against a broad spectrum of bacterial pathogens</span></div><div class="casAuthors">Biedenbach, D. J.; Bouchillon, S. K.; Hackel, M.; Miller, L. A.; Scangarella-Oman, N. E.; Jakielaszek, C.; Sahm, D. F.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1918-1923</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Gepotidacin inhibits bacterial DNA replication through a mode different from that of fluoroquinolones.  Gepotidacin and comparators were tested by broth and agar diln. against clin. isolates.  The in vitro activities of gepotidacin were comparable against methicillin-susceptible and -resistant Staphylococcus aureus (MSSA and MRSA, resp.) isolates (MIC90, 0.5 μg/mL).  The gepotidacin MIC90s were as follows (in micrograms per mL) for the indicated bacteria: Streptococcus pyogenes, 0.25; Escherichia coli, 2; Moraxella catarrhalis, ≤0.06; Streptococcus pneumoniae (0.25), Haemophilus influenzae, 1; Clostridium perfringens, 0.5; and Shigella spp., 1, including levofloxacin-resistant subsets.  Gepotidacin warrants further investigation for clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFHvWFX6zKO7Vg90H21EOLACvtfcHk0lj0cZxbWUJbvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsF2ntbbE&md5=80b9eca78be2d6f2fae33a97bdabef68</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1128%2FAAC.02820-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02820-15%26sid%3Dliteratum%253Aachs%26aulast%3DBiedenbach%26aufirst%3DD.%2BJ.%26aulast%3DBouchillon%26aufirst%3DS.%2BK.%26aulast%3DHackel%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DL.%2BA.%26aulast%3DScangarella-Oman%26aufirst%3DN.%2BE.%26aulast%3DJakielaszek%26aufirst%3DC.%26aulast%3DSahm%26aufirst%3DD.%2BF.%26atitle%3DIn%2520Vitro%2520Activity%2520of%2520Gepotidacin%252C%2520a%2520Novel%2520Triazaacenaphthylene%2520Bacterial%2520Topoisomerase%2520Inhibitor%252C%2520against%2520a%2520Broad%2520Spectrum%2520of%2520Bacterial%2520Pathogens%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2016%26volume%3D60%26spage%3D1918%26epage%3D1923%26doi%3D10.1128%2FAAC.02820-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span>; <span class="NLM_string-name">Di, L.</span></span> <i>Drug-Like Properties: Concepts, Structure, Design, and Methods</i>. <span class="NLM_edition">2nd</span> ed.; <span class="NLM_publisher-name">Elsevier</span>: <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=E.+Kerns&author=L.+Di&title=Drug-Like+Properties%3A+Concepts%2C+Structure%2C+Design%2C+and+Methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKerns%26aufirst%3DE.%26btitle%3DDrug-Like%2520Properties%253A%2520Concepts%252C%2520Structure%252C%2520Design%252C%2520and%2520Methods%26pub%3DElsevier%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Schrödinger
LLC</span>.  <i>The Pymol Molecular Graphics
System, Version 1.8</i><i>.</i> <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=Schr%C3%B6dinger%0ALLC&title=The+Pymol+Molecular+Graphics%0ASystem%2C+Version+1.8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DThe%2520Pymol%2520Molecular%2520Graphics%250ASystem%252C%2520Version%25201.8%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berthold, M. R.</span>; <span class="NLM_string-name">Cebron, N.</span>; <span class="NLM_string-name">Dill, F.</span>; <span class="NLM_string-name">Gabriel, T. R.</span>; <span class="NLM_string-name">Kötter, T.</span>; <span class="NLM_string-name">Meinl, T.</span>; <span class="NLM_string-name">Ohl, P.</span>; <span class="NLM_string-name">Sieb, C.</span>; <span class="NLM_string-name">Thiel, K.</span>; <span class="NLM_string-name">Wiswedel, B.</span></span> <span> </span><span class="NLM_article-title">Knime: The Konstanz Information Miner</span>. In  <i>Data Analysis, Machine Learning and Applications: Studies in Classification, Data Analysis, and Knowledge Organization</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preisach, C.</span>; <span class="NLM_string-name">Burkhardt, H.</span>; <span class="NLM_string-name">Schmidt-Thieme, L.</span>; <span class="NLM_string-name">Decker, R.</span></span>, Eds. <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Berlin, Heidelberg</span>, <span class="NLM_year">2008</span>; pp  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1007%2F978-3-540-78246-9_38" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=319-326&author=M.+R.+Berthold&author=N.+Cebron&author=F.+Dill&author=T.+R.+Gabriel&author=T.+K%C3%B6tter&author=T.+Meinl&author=P.+Ohl&author=C.+Sieb&author=K.+Thiel&author=B.+Wiswedelauthor=C.+Preisach&author=H.+Burkhardt&author=L.+Schmidt-Thieme&author=R.+Decker&title=Data+Analysis%2C+Machine+Learning+and+Applications%3A+Studies+in+Classification%2C+Data+Analysis%2C+and+Knowledge+Organization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2F978-3-540-78246-9_38&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-540-78246-9_38%26sid%3Dliteratum%253Aachs%26aulast%3DBerthold%26aufirst%3DM.%2BR.%26atitle%3DKnime%253A%2520The%2520Konstanz%2520Information%2520Miner%26btitle%3DData%2520Analysis%252C%2520Machine%2520Learning%2520and%2520Applications%253A%2520Studies%2520in%2520Classification%252C%2520Data%2520Analysis%252C%2520and%2520Knowledge%2520Organization%26aulast%3DPreisach%26aufirst%3DC.%26pub%3DSpringer%26date%3D2008%26spage%3D319%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Schrödinger
LLC</span>.  <i>Schrödinger Release 2013–1:
Schrödinger Knime Extensions</i>; <span class="NLM_publisher-name">Schrödinger
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=Schr%C3%B6dinger%0ALLC&title=Schr%C3%B6dinger+Release+2013%E2%80%931%3A%0ASchr%C3%B6dinger+Knime+Extensions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DSchr%25C3%25B6dinger%2520Release%25202013%25E2%2580%25931%253A%250ASchr%25C3%25B6dinger%2520Knime%2520Extensions%26pub%3DSchr%25C3%25B6dinger%250ALLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Python Software Foundation</span>.  <i>Python Language Reference</i>, Version
2.7. <a href="http://www.python.org" class="extLink">www.python.org</a></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Python+Software+Foundation.+Python+Language+Reference%2C+Version%0A2.7.+www.python.org"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DPython%2520Language%2520Reference" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiener, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santillán, A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allison, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinde, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hack, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motley, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldschmidt, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grice, C. A.</span></span> <span> </span><span class="NLM_article-title">Tetrahydroindazole Inhibitors of Bacterial Type Ii Topoisomerases. Part 2: Sar Development and Potency against Multidrug-Resistant Strains</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2718</span>– <span class="NLM_lpage">2722</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2007.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2007.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=17382544" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslOqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=2718-2722&author=J.+J.+Wienerauthor=L.+Gomezauthor=H.+Venkatesanauthor=A.+Santill%C3%A1nauthor=B.+D.+Allisonauthor=K.+L.+Schwarzauthor=S.+Shindeauthor=L.+Tangauthor=M.+D.+Hackauthor=B.+J.+Morrowauthor=S.+T.+Motleyauthor=R.+M.+Goldschmidtauthor=K.+J.+Shawauthor=T.+K.+Jonesauthor=C.+A.+Grice&title=Tetrahydroindazole+Inhibitors+of+Bacterial+Type+Ii+Topoisomerases.+Part+2%3A+Sar+Development+and+Potency+against+Multidrug-Resistant+Strains&doi=10.1016%2Fj.bmcl.2007.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Tetrahydroindazole inhibitors of bacterial type II topoisomerases. Part 2: SAR development and potency against multidrug-resistant strains</span></div><div class="casAuthors">Wiener, John J. M.; Gomez, Laurent; Venkatesan, Hariharan; Santillan, Alejandro, Jr.; Allison, Brett D.; Schwarz, Kimberly L.; Shinde, Shirin; Tang, Liu; Hack, Michael D.; Morrow, Brian J.; Motley, S. Timothy; Goldschmidt, Raul M.; Shaw, Karen Joy; Jones, Todd K.; Grice, Cheryl A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2718-2722</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">We have previously reported a novel class of tetrahydroindazoles that display potency against a variety of Gram-pos. and Gram-neg. bacteria, potentially via interaction with type II bacterial topoisomerases.  Herein are reported SAR investigations of this new series.  Several compds. possessing broad-spectrum potency were prepd.  Further, these compds. exhibit activity against multidrug-resistant Gram-pos. microorganisms equiv. to that against susceptible strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKlC4VAkQDZLVg90H21EOLACvtfcHk0lj0cZxbWUJbvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslOqtbY%253D&md5=9bc16d091987be0ec1ced5dd2d910d5e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DWiener%26aufirst%3DJ.%2BJ.%26aulast%3DGomez%26aufirst%3DL.%26aulast%3DVenkatesan%26aufirst%3DH.%26aulast%3DSantill%25C3%25A1n%26aufirst%3DA.%26aulast%3DAllison%26aufirst%3DB.%2BD.%26aulast%3DSchwarz%26aufirst%3DK.%2BL.%26aulast%3DShinde%26aufirst%3DS.%26aulast%3DTang%26aufirst%3DL.%26aulast%3DHack%26aufirst%3DM.%2BD.%26aulast%3DMorrow%26aufirst%3DB.%2BJ.%26aulast%3DMotley%26aufirst%3DS.%2BT.%26aulast%3DGoldschmidt%26aufirst%3DR.%2BM.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DJones%26aufirst%3DT.%2BK.%26aulast%3DGrice%26aufirst%3DC.%2BA.%26atitle%3DTetrahydroindazole%2520Inhibitors%2520of%2520Bacterial%2520Type%2520Ii%2520Topoisomerases.%2520Part%25202%253A%2520Sar%2520Development%2520and%2520Potency%2520against%2520Multidrug-Resistant%2520Strains%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D2718%26epage%3D2722%26doi%3D10.1016%2Fj.bmcl.2007.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hack, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiener, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santillán, A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pippel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motley, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grice, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. K.</span></span> <span> </span><span class="NLM_article-title">Novel Pyrazole Derivatives as Potent Inhibitors of Type Ii Topoisomerases. Part 1: Synthesis and Preliminary Sar Analysis</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2723</span>– <span class="NLM_lpage">2727</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2007.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2007.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=17368897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslOqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=2723-2727&author=L.+Gomezauthor=M.+D.+Hackauthor=J.+Wuauthor=J.+J.+Wienerauthor=H.+Venkatesanauthor=A.+Santill%C3%A1nauthor=D.+J.+Pippelauthor=N.+Maniauthor=B.+J.+Morrowauthor=S.+T.+Motleyauthor=K.+J.+Shawauthor=R.+Wolinauthor=C.+A.+Griceauthor=T.+K.+Jones&title=Novel+Pyrazole+Derivatives+as+Potent+Inhibitors+of+Type+Ii+Topoisomerases.+Part+1%3A+Synthesis+and+Preliminary+Sar+Analysis&doi=10.1016%2Fj.bmcl.2007.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Novel pyrazole derivatives as potent inhibitors of type II topoisomerases. Part 1: Synthesis and preliminary SAR analysis</span></div><div class="casAuthors">Gomez, Laurent; Hack, Michael D.; Wu, Jiejun; Wiener, John J. M.; Venkatesan, Hari; Santillan, Alejandro; Pippel, Daniel J.; Mani, Neelakandha; Morrow, Brian J.; Motley, S. Timothy; Shaw, Karen Joy; Wolin, Ronald; Grice, Cheryl A.; Jones, Todd K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2723-2727</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In an attempt to search for a new class of antibacterial agents, a series of pyrazole analogs that possess good antibacterial activity for Gram-pos. and Gram-neg. organisms via inhibition of type II bacterial topoisomerases was discovered.  The structure-activity relationships of this series, with an emphasis on the length and conformation of the linker was also investigated.  Several tetrahydroindazole analogs was identified.  Compd. I was shown to be the most potent class of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp54rXZYBq1jbVg90H21EOLACvtfcHk0ljQoYK5LyX8yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslOqtbc%253D&md5=bfb82255564581660b9b7002fc079950</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DGomez%26aufirst%3DL.%26aulast%3DHack%26aufirst%3DM.%2BD.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DWiener%26aufirst%3DJ.%2BJ.%26aulast%3DVenkatesan%26aufirst%3DH.%26aulast%3DSantill%25C3%25A1n%26aufirst%3DA.%26aulast%3DPippel%26aufirst%3DD.%2BJ.%26aulast%3DMani%26aufirst%3DN.%26aulast%3DMorrow%26aufirst%3DB.%2BJ.%26aulast%3DMotley%26aufirst%3DS.%2BT.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DWolin%26aufirst%3DR.%26aulast%3DGrice%26aufirst%3DC.%2BA.%26aulast%3DJones%26aufirst%3DT.%2BK.%26atitle%3DNovel%2520Pyrazole%2520Derivatives%2520as%2520Potent%2520Inhibitors%2520of%2520Type%2520Ii%2520Topoisomerases.%2520Part%25201%253A%2520Synthesis%2520and%2520Preliminary%2520Sar%2520Analysis%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D2723%26epage%3D2727%26doi%3D10.1016%2Fj.bmcl.2007.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miles, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barfoot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dabbs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Downie, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyrisch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwynn, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennessy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoover, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markwell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rittenhouse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, N.</span></span> <span> </span><span class="NLM_article-title">Novel Cyclohexyl-Amides as Potent Antibacterials Targeting Bacterial Type Iia Topoisomerases</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">7483</span>– <span class="NLM_lpage">7488</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.09.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2011.09.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=22030032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFSqtb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=7483-7488&author=T.+J.+Milesauthor=C.+Barfootauthor=G.+Brooksauthor=P.+Brownauthor=D.+Chenauthor=S.+Dabbsauthor=D.+T.+Daviesauthor=D.+L.+Downieauthor=S.+Eyrischauthor=I.+Giordanoauthor=M.+N.+Gwynnauthor=A.+Hennessyauthor=J.+Hooverauthor=J.+Huangauthor=G.+Jonesauthor=R.+Markwellauthor=S.+Rittenhouseauthor=H.+Xiangauthor=N.+Pearson&title=Novel+Cyclohexyl-Amides+as+Potent+Antibacterials+Targeting+Bacterial+Type+Iia+Topoisomerases&doi=10.1016%2Fj.bmcl.2011.09.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Novel cyclohexyl-amides as potent antibacterials targeting bacterial type IIA topoisomerases</span></div><div class="casAuthors">Miles, Timothy J.; Barfoot, Christopher; Brooks, Gerald; Brown, Pamela; Chen, Dongzhao; Dabbs, Steven; Davies, David T.; Downie, David L.; Eyrisch, Susanne; Giordano, Ilaria; Gwynn, Michael N.; Hennessy, Alan; Hoover, Jennifer; Huang, Jianzhong; Jones, Graham; Markwell, Roger; Rittenhouse, Stephen; Xiang, Hong; Pearson, Neil</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7483-7488</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">As part of our wider efforts to exploit novel mode of action antibacterials, we have discovered a series of cyclohexyl-amide compds. that has good Gram pos. and Gram neg. potency.  The mechanism of action is via inhibition of bacterial topoisomerases II and IV.  We have investigated various subunits in this series and report advanced studies on compd. 7 which demonstrates good PK and in vivo efficacy properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOaL1uO_CXkLVg90H21EOLACvtfcHk0ljQoYK5LyX8yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFSqtb7E&md5=a4404e4f4cd9c8ea602316b3593d68b7</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.09.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.09.114%26sid%3Dliteratum%253Aachs%26aulast%3DMiles%26aufirst%3DT.%2BJ.%26aulast%3DBarfoot%26aufirst%3DC.%26aulast%3DBrooks%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDabbs%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DD.%2BT.%26aulast%3DDownie%26aufirst%3DD.%2BL.%26aulast%3DEyrisch%26aufirst%3DS.%26aulast%3DGiordano%26aufirst%3DI.%26aulast%3DGwynn%26aufirst%3DM.%2BN.%26aulast%3DHennessy%26aufirst%3DA.%26aulast%3DHoover%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DG.%26aulast%3DMarkwell%26aufirst%3DR.%26aulast%3DRittenhouse%26aufirst%3DS.%26aulast%3DXiang%26aufirst%3DH.%26aulast%3DPearson%26aufirst%3DN.%26atitle%3DNovel%2520Cyclohexyl-Amides%2520as%2520Potent%2520Antibacterials%2520Targeting%2520Bacterial%2520Type%2520Iia%2520Topoisomerases%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D7483%26epage%3D7488%26doi%3D10.1016%2Fj.bmcl.2011.09.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miles, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axten, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barfoot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dabbs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Downie, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyrisch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwynn, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennessy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoover, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markwell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minthorn, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rittenhouse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seefeld, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, N.</span></span> <span> </span><span class="NLM_article-title">Novel Amino-Piperidines as Potent Antibacterials Targeting Bacterial Type Iia Topoisomerases</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">7489</span>– <span class="NLM_lpage">7495</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.09.117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2011.09.117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=22047689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFSqtb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=7489-7495&author=T.+J.+Milesauthor=J.+M.+Axtenauthor=C.+Barfootauthor=G.+Brooksauthor=P.+Brownauthor=D.+Chenauthor=S.+Dabbsauthor=D.+T.+Daviesauthor=D.+L.+Downieauthor=S.+Eyrischauthor=T.+Gallagherauthor=I.+Giordanoauthor=M.+N.+Gwynnauthor=A.+Hennessyauthor=J.+Hooverauthor=J.+Huangauthor=G.+Jonesauthor=R.+Markwellauthor=W.+H.+Millerauthor=E.+A.+Minthornauthor=S.+Rittenhouseauthor=M.+Seefeldauthor=N.+Pearson&title=Novel+Amino-Piperidines+as+Potent+Antibacterials+Targeting+Bacterial+Type+Iia+Topoisomerases&doi=10.1016%2Fj.bmcl.2011.09.117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Novel amino-piperidines as potent antibacterials targeting bacterial type IIA topoisomerases</span></div><div class="casAuthors">Miles, Timothy J.; Axten, Jeffrey M.; Barfoot, Christopher; Brooks, Gerald; Brown, Pamela; Chen, Dongzhao; Dabbs, Steven; Davies, David T.; Downie, David L.; Eyrisch, Susanne; Gallagher, Timothy; Giordano, Ilaria; Gwynn, Michael N.; Hennessy, Alan; Hoover, Jennifer; Huang, Jianzhong; Jones, Graham; Markwell, Roger; Miller, William H.; Minthorn, Elizabeth A.; Rittenhouse, Stephen; Seefeld, Mark; Pearson, Neil</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7489-7495</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We have identified a series of amino-piperidine antibacterials with a good broad spectrum potency.  We report the investigation of various subunits in this series and advanced studies on compd. 8.  Compd. 8 possesses good pharmacokinetics, broad spectrum antibacterial activity and demonstrates oral efficacy in a rat lung infection model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpe_9NHsx5_Y7Vg90H21EOLACvtfcHk0ljQoYK5LyX8yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFSqtb%252FN&md5=ddc20f19a1e6f608066586ff739f0af8</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.09.117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.09.117%26sid%3Dliteratum%253Aachs%26aulast%3DMiles%26aufirst%3DT.%2BJ.%26aulast%3DAxten%26aufirst%3DJ.%2BM.%26aulast%3DBarfoot%26aufirst%3DC.%26aulast%3DBrooks%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDabbs%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DD.%2BT.%26aulast%3DDownie%26aufirst%3DD.%2BL.%26aulast%3DEyrisch%26aufirst%3DS.%26aulast%3DGallagher%26aufirst%3DT.%26aulast%3DGiordano%26aufirst%3DI.%26aulast%3DGwynn%26aufirst%3DM.%2BN.%26aulast%3DHennessy%26aufirst%3DA.%26aulast%3DHoover%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DG.%26aulast%3DMarkwell%26aufirst%3DR.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DMinthorn%26aufirst%3DE.%2BA.%26aulast%3DRittenhouse%26aufirst%3DS.%26aulast%3DSeefeld%26aufirst%3DM.%26aulast%3DPearson%26aufirst%3DN.%26atitle%3DNovel%2520Amino-Piperidines%2520as%2520Potent%2520Antibacterials%2520Targeting%2520Bacterial%2520Type%2520Iia%2520Topoisomerases%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D7489%26epage%3D7495%26doi%3D10.1016%2Fj.bmcl.2011.09.117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namchuk, M.</span></span> <span> </span><span class="NLM_article-title">Designing Screens: How to Make Your Hits a Hit</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">259</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1038/nrd1063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1038%2Fnrd1063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=12669025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD3sXisFWktrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=259-266&author=W.+P.+Waltersauthor=M.+Namchuk&title=Designing+Screens%3A+How+to+Make+Your+Hits+a+Hit&doi=10.1038%2Fnrd1063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A guide to drug discovery: Designing screens: how to make your hits a hit</span></div><div class="casAuthors">Walters, W. Patrick; Namchuk, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">259-266</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The basic goal of small-mol. screening is the identification of chem. interesting starting points for elaboration towards a drug.  A no. of innovative approaches for pursuing this goal have evolved, and the right approach is dictated by the target class being pursued and the capabilities of the organization involved.hood.  A recent trend in high-throughput screening has been to place less emphasis on the no. of data points that can be produced, and to focus instead on the quality of the data obtained.  Several computational and technol. advances have aided in the selection of compds. for screening and widened the variety of assay formats available for screening.  The effect on the efficiency of the screening process is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ5E12ik7BrbVg90H21EOLACvtfcHk0lg4o2My7IgOSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXisFWktrk%253D&md5=6bec4547b8dce69ab84ce226cde28f8a</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnrd1063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1063%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DNamchuk%26aufirst%3DM.%26atitle%3DDesigning%2520Screens%253A%2520How%2520to%2520Make%2520Your%2520Hits%2520a%2520Hit%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D259%26epage%3D266%26doi%3D10.1038%2Fnrd1063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Schrödinger
LLC</span>.  <i>Schrödinger Release 2014-3:
Ligprep, Version 3.1</i>. <span class="NLM_publisher-name">Schrödinger
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=Schr%C3%B6dinger%0ALLC&title=Schr%C3%B6dinger+Release+2014-3%3A%0ALigprep%2C+Version+3.1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DSchr%25C3%25B6dinger%2520Release%25202014-3%253A%250ALigprep%252C%2520Version%25203.1%26pub%3DSchr%25C3%25B6dinger%250ALLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schrödinger
LLC</span></span>.  <i>Schrödinger Release 2014-3: Epik</i>. <span class="NLM_publisher-name">Schrödinger LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=+Schr%C3%B6dinger%0ALLC&title=Schr%C3%B6dinger+Release+2014-3%3A+Epik"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSchr%25C3%25B6dinger%2BLLC%26jtitle%3DSchr%25C3%25B6dinger%2520Release%25202014-3%253A%2520Epik%26pub%3DSchr%25C3%25B6dinger%2520LLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hundsdörfer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemmerling, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totzke, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednarski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Borgne, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jose, J.</span></span> <span> </span><span class="NLM_article-title">Indeno[1,2-B]Indole Derivatives as a Novel Class of Potent Human Protein Kinase Ck2 Inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">2282</span>– <span class="NLM_lpage">2289</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2012.02.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmc.2012.02.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=22377675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A280%3ADC%252BC38zgvFCqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=2282-2289&author=C.+Hundsd%C3%B6rferauthor=H.+J.+Hemmerlingauthor=C.+G%C3%B6tzauthor=F.+Totzkeauthor=P.+Bednarskiauthor=M.+Le+Borgneauthor=J.+Jose&title=Indeno%5B1%2C2-B%5DIndole+Derivatives+as+a+Novel+Class+of+Potent+Human+Protein+Kinase+Ck2+Inhibitors&doi=10.1016%2Fj.bmc.2012.02.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Indeno[1,2-b]indole derivatives as a novel class of potent human protein kinase CK2 inhibitors</span></div><div class="casAuthors">Hundsdorfer Claas; Hemmerling Hans-Jorg; Gotz Claudia; Totzke Frank; Bednarski Patrick; Le Borgne Marc; Jose Joachim</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & medicinal chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2282-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Herein we describe the synthesis and properties of indeno[1,2-b]indole derivatives as a novel class of potent inhibitors of the human protein kinase CK2.  A set of 19 compounds was obtained using a convenient and straightforward synthesis protocol.  The compounds were tested for inhibition of human protein kinase CK2, which was recombinantly expressed in Escherichia coli.  New inhibitors with IC(50) in the micro- and sub-micromolar range were identified.  Compound 4b (5-isopropyl-7,8-dihydroindeno[1,2-b]indole-9,10(5H,6H)-dione) inhibited human CK2 with an IC(50) of 0.11 μM and did not significantly inhibit 22 other human protein kinases, suggesting selectivity towards CK2.  ATP-competitive inhibition by compound 4b was shown and a K(i) of 0.06 μM was determined.  Our findings indicate that indeno[1,2-b]indoles are a promising starting point for further development and optimization of human protein kinase CK2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRFnjW4s3gmBiXXDYhVZBdlfW6udTcc2eZjQkYyvztcArntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38zgvFCqtQ%253D%253D&md5=e32d29d7c2c80cbc2a16f1dc56743da6</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.02.017%26sid%3Dliteratum%253Aachs%26aulast%3DHundsd%25C3%25B6rfer%26aufirst%3DC.%26aulast%3DHemmerling%26aufirst%3DH.%2BJ.%26aulast%3DG%25C3%25B6tz%26aufirst%3DC.%26aulast%3DTotzke%26aufirst%3DF.%26aulast%3DBednarski%26aufirst%3DP.%26aulast%3DLe%2BBorgne%26aufirst%3DM.%26aulast%3DJose%26aufirst%3DJ.%26atitle%3DIndeno%255B1%252C2-B%255DIndole%2520Derivatives%2520as%2520a%2520Novel%2520Class%2520of%2520Potent%2520Human%2520Protein%2520Kinase%2520Ck2%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2012%26volume%3D20%26spage%3D2282%26epage%3D2289%26doi%3D10.1016%2Fj.bmc.2012.02.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zardecki, C.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=P.+E.+Bourneauthor=J.+Westbrookauthor=C.+Zardecki&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0ljfJs-5JeUZ7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DZardecki%26aufirst%3DC.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span> <span> </span><span class="NLM_article-title">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6177</span>– <span class="NLM_lpage">6196</span>, <span class="refDoi"> DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+Precision+Glide%3A+Docking+and+Scoring+Incorporating+a+Model+of+Hydrophobic+Enclosure+for+Protein-Ligand+Complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0ljfJs-5JeUZ7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520Precision%2520Glide%253A%2520Docking%2520and%2520Scoring%2520Incorporating%2520a%2520Model%2520of%2520Hydrophobic%2520Enclosure%2520for%2520Protein-Ligand%2520Complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+2.+Enrichment+Factors+in+Database+Screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0ljfJs-5JeUZ7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25202.%2520Enrichment%2520Factors%2520in%2520Database%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R.
A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.%0AA.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lhRFxoBHL6XQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sastry, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowrie, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Systematic Analysis of 2d Fingerprint Methods and Parameters to Improve Virtual Screening Enrichments</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">771</span>– <span class="NLM_lpage">784</span>, <span class="refDoi"> DOI: 10.1021/ci100062n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100062n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlslynu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=771-784&author=M.+Sastryauthor=J.+F.+Lowrieauthor=S.+L.+Dixonauthor=W.+Sherman&title=Large-Scale+Systematic+Analysis+of+2d+Fingerprint+Methods+and+Parameters+to+Improve+Virtual+Screening+Enrichments&doi=10.1021%2Fci100062n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Large-Scale Systematic Analysis of 2D Fingerprint Methods and Parameters to Improve Virtual Screening Enrichments</span></div><div class="casAuthors">Sastry, Madhavi; Lowrie, Jeffrey F.; Dixon, Steven L.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">771-784</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A systematic virtual screening study on 11 pharmaceutically relevant targets has been conducted to investigate the interrelation between 8 two-dimensional (2D) fingerprinting methods, 13 atom-typing schemes, 13 bit scaling rules, and 12 similarity metrics using the new cheminformatics package Canvas.  In total, 157 872 virtual screens were performed to assess the ability of each combination of parameters to identify actives in a database screen.  In general, fingerprint methods, such as MOLPRINT2D, Radial, and Dendritic that encode information about local environment beyond simple linear paths outperformed other fingerprint methods.  Atom-typing schemes with more specific information, such as Daylight, Mol2, and Carhart were generally superior to more generic atom-typing schemes.  Enrichment factors across all targets were improved considerably with the best settings, although no single set of parameters performed optimally on all targets.  The size of the addressable bit space for the fingerprints was also explored, and it was found to have a substantial impact on enrichments.  Small bit spaces, such as 1024, resulted in many collisions and in a significant degrdn. in enrichments compared to larger bit spaces that avoid collisions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH6_ENmYgPPrVg90H21EOLACvtfcHk0lhRFxoBHL6XQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlslynu7k%253D&md5=54e3d9f7e1d493e3b3ee2f153f1af4b2</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fci100062n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100062n%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DM.%26aulast%3DLowrie%26aufirst%3DJ.%2BF.%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DLarge-Scale%2520Systematic%2520Analysis%2520of%25202d%2520Fingerprint%2520Methods%2520and%2520Parameters%2520to%2520Improve%2520Virtual%2520Screening%2520Enrichments%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D771%26epage%3D784%26doi%3D10.1021%2Fci100062n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowrie, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Analysis and Comparison of 2d Fingerprints: Insights into Database Screening Performance Using Eight Fingerprint Methods</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2010.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.jmgm.2010.05.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=20579912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFamtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=157-170&author=J.+Duanauthor=S.+L.+Dixonauthor=J.+F.+Lowrieauthor=W.+Sherman&title=Analysis+and+Comparison+of+2d+Fingerprints%3A+Insights+into+Database+Screening+Performance+Using+Eight+Fingerprint+Methods&doi=10.1016%2Fj.jmgm.2010.05.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis and comparison of 2D fingerprints: Insights into database screening performance using eight fingerprint methods</span></div><div class="casAuthors">Duan, Jian-Xin; Dixon, Steven L.; Lowrie, Jeffrey F.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-170</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Virtual screening is a widely used strategy in modern drug discovery and 2D fingerprint similarity is an important tool that has been successfully applied to retrieve active compds. from large datasets.  However, it is not always straightforward to select an appropriate fingerprint method and assocd. settings for a given problem.  Here, we applied eight different fingerprint methods, as implemented in the new cheminformatics package Canvas, on a well-validated dataset covering five targets.  The fingerprint methods include Linear, Dendritic, Radial, MACCS, MOLPRINT2D, Pairwise, Triplet, and Torsion.  We find that most fingerprints have similar retrieval rates on av.; however, each has special characteristics that distinguish its performance on different query mols. and ligand sets.  For example, some fingerprints exhibit a significant ligand size dependency whereas others are more robust with respect to variations in the query or active compds.  In cases where little information is known about the active ligands, MOLPRINT2D fingerprints produce the highest av. retrieval actives.  When multiple queries are available, we find that a fingerprint averaged over all query mols. is generally superior to fingerprints derived from single queries.  Finally, a complementarity metric is proposed to det. which fingerprint methods can be combined to improve screening results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWCcy4qE10YbVg90H21EOLACvtfcHk0lhRFxoBHL6XQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFamtLrM&md5=f72d91018346b1e5ef97fc83f648bd86</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2010.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2010.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DJ.%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DLowrie%26aufirst%3DJ.%2BF.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DAnalysis%2520and%2520Comparison%2520of%25202d%2520Fingerprints%253A%2520Insights%2520into%2520Database%2520Screening%2520Performance%2520Using%2520Eight%2520Fingerprint%2520Methods%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2010%26volume%3D29%26spage%3D157%26epage%3D170%26doi%3D10.1016%2Fj.jmgm.2010.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, K. J.</span>; <span class="NLM_string-name">Chow, D. E.</span>; <span class="NLM_string-name">Xu, H.</span>; <span class="NLM_string-name">Dror, R. O.</span>; <span class="NLM_string-name">Eastwood, M. P.</span>; <span class="NLM_string-name">Gregersen, B. A.</span>; <span class="NLM_string-name">Klepeis, J. L.</span>; <span class="NLM_string-name">Kolossvary, I.</span>; <span class="NLM_string-name">Moraes, M. A.</span>; <span class="NLM_string-name">Sacerdoti, F. D.</span>; <span class="NLM_string-name">Salmon, J. K.</span>; <span class="NLM_string-name">Shan, Y.</span>; <span class="NLM_string-name">Shaw, D. E.</span></span> <span> </span><span class="NLM_article-title">Scalable Algorithms for Molecular Dynamics Simulations on Commodity Clusters</span>. In  <i>Sc ’06: Proceedings of the 2006 Acm/Ieee Conference on Supercomputing</i>; <span class="NLM_publisher-name">IEEE</span>: <span class="NLM_publisher-loc">Tampa, FL</span>, <span class="NLM_year">2006</span>; p  <span class="NLM_fpage">746</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1109%2FSC.2006.54" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=746&author=K.+J.+Bowers&author=D.+E.+Chow&author=H.+Xu&author=R.+O.+Dror&author=M.+P.+Eastwood&author=B.+A.+Gregersen&author=J.+L.+Klepeis&author=I.+Kolossvary&author=M.+A.+Moraes&author=F.+D.+Sacerdoti&author=J.+K.+Salmon&author=Y.+Shan&author=D.+E.+Shaw&title=Sc+%E2%80%9906%3A+Proceedings+of+the+2006+Acm%2FIeee+Conference+on+Supercomputing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1109%2FSC.2006.54&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1109%252FSC.2006.54%26sid%3Dliteratum%253Aachs%26aulast%3DBowers%26aufirst%3DK.%2BJ.%26atitle%3DScalable%2520Algorithms%2520for%2520Molecular%2520Dynamics%2520Simulations%2520on%2520Commodity%2520Clusters%26btitle%3DSc%2520%25E2%2580%259906%253A%2520Proceedings%2520of%2520the%25202006%2520Acm%252FIeee%2520Conference%2520on%2520Supercomputing%26pub%3DIEEE%26date%3D2006%26spage%3D746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haddach, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierre, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regan, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borsan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerdoncuff, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaebe, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui-Jain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macalino, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drygin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryckman, D. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Sar of Inhibitors of Protein Kinase Ck2: Novel Tricyclic Quinoline Analogs</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.11.087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2Fj.bmcl.2011.11.087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=22169261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC38XivVymsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=45-48&author=M.+Haddachauthor=F.+Pierreauthor=C.+F.+Reganauthor=C.+Borsanauthor=J.+Michauxauthor=E.+Stefanauthor=P.+Kerdoncuffauthor=M.+K.+Schwaebeauthor=P.+C.+Chuaauthor=A.+Siddiqui-Jainauthor=D.+Macalinoauthor=D.+Dryginauthor=S.+E.+O%E2%80%99Brienauthor=W.+G.+Riceauthor=D.+M.+Ryckman&title=Synthesis+and+Sar+of+Inhibitors+of+Protein+Kinase+Ck2%3A+Novel+Tricyclic+Quinoline+Analogs&doi=10.1016%2Fj.bmcl.2011.11.087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and SAR of inhibitors of protein kinase CK2: Novel tricyclic quinoline analogs</span></div><div class="casAuthors">Haddach, Mustapha; Pierre, Fabrice; Regan, Collin F.; Borsan, Cosmin; Michaux, Jerome; Stefan, Eric; Kerdoncuff, Pauline; Schwaebe, Michael K.; Chua, Peter C.; Siddiqui-Jain, Adam; Macalino, Diwata; Drygin, Denis; O'Brien, Sean E.; Rice, William G.; Ryckman, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-48</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein kinase CK2 is a potential drug target for many diseases including cancer and inflammation disorders.  The crystal structure of clin. candidate CX-4945 with CK2 revealed an indirect interaction with the protein through hydrogen bonding between the NH of the 3-chlorophenylamine and a water mol.  The relevance of this hydrogen bond is investigated by prepn. of several novel tricyclic derivs. lacking an NH moiety at the same position.  This SAR study allowed the discovery of 5-arylbenzo[c][2,6]naphthyridine-8-carboxylic acids as highly potent CK2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXez4R82fuOLVg90H21EOLACvtfcHk0ljiepSamQPMeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivVymsw%253D%253D&md5=f329788c09ed192ac49b8c1a02342806</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.11.087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.11.087%26sid%3Dliteratum%253Aachs%26aulast%3DHaddach%26aufirst%3DM.%26aulast%3DPierre%26aufirst%3DF.%26aulast%3DRegan%26aufirst%3DC.%2BF.%26aulast%3DBorsan%26aufirst%3DC.%26aulast%3DMichaux%26aufirst%3DJ.%26aulast%3DStefan%26aufirst%3DE.%26aulast%3DKerdoncuff%26aufirst%3DP.%26aulast%3DSchwaebe%26aufirst%3DM.%2BK.%26aulast%3DChua%26aufirst%3DP.%2BC.%26aulast%3DSiddiqui-Jain%26aufirst%3DA.%26aulast%3DMacalino%26aufirst%3DD.%26aulast%3DDrygin%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DS.%2BE.%26aulast%3DRice%26aufirst%3DW.%2BG.%26aulast%3DRyckman%26aufirst%3DD.%2BM.%26atitle%3DSynthesis%2520and%2520Sar%2520of%2520Inhibitors%2520of%2520Protein%2520Kinase%2520Ck2%253A%2520Novel%2520Tricyclic%2520Quinoline%2520Analogs%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D45%26epage%3D48%26doi%3D10.1016%2Fj.bmcl.2011.11.087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shivakumar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Prediction of Absolute Solvation Free Energies Using Molecular Dynamics Free Energy Perturbation and the Opls Force Field</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1509</span>– <span class="NLM_lpage">1519</span>, <span class="refDoi"> DOI: 10.1021/ct900587b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct900587b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkslKhu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=1509-1519&author=D.+Shivakumarauthor=J.+Williamsauthor=Y.+Wuauthor=W.+Dammauthor=J.+Shelleyauthor=W.+Sherman&title=Prediction+of+Absolute+Solvation+Free+Energies+Using+Molecular+Dynamics+Free+Energy+Perturbation+and+the+Opls+Force+Field&doi=10.1021%2Fct900587b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Absolute Solvation Free Energies using Molecular Dynamics Free Energy Perturbation and the OPLS Force Field</span></div><div class="casAuthors">Shivakumar, Devleena; Williams, Joshua; Wu, Yujie; Damm, Wolfgang; Shelley, John; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1509-1519</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accurate prediction of protein-ligand binding free energies is a primary objective in computer-aided drug design.  The solvation free energy of a small mol. provides a surrogate to the desolvation of the ligand in the thermodn. process of protein-ligand binding.  Here, we use explicit solvent mol. dynamics free energy perturbation to predict the abs. solvation free energies of a set of 239 small mols., spanning diverse chem. functional groups commonly found in drugs and drug-like mols.  We also compare the performance of abs. solvation free energies obtained using the OPLS_2005 force field with two other commonly used small mol. force fields - general AMBER force field (GAFF) with AM1-BCC charges and CHARMm-MSI with CHelpG charges.  Using the OPLS_2005 force field, we obtain high correlation with exptl. solvation free energies (R2 = 0.94) and low av. unsigned errors for a majority of the functional groups compared to AM1-BCC/GAFF or CHelpG/CHARMm-MSI.  However, OPLS_2005 has errors of over 1.3 kcal/mol for certain classes of polar compds.  We show that predictions on these compd. classes can be improved by using a semiempirical charge assignment method with an implicit bond charge correction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon_fkbSHqg3bVg90H21EOLACvtfcHk0ljiepSamQPMeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkslKhu7g%253D&md5=25d464fb9238e3235e881945c0b50a76</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fct900587b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct900587b%26sid%3Dliteratum%253Aachs%26aulast%3DShivakumar%26aufirst%3DD.%26aulast%3DWilliams%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DShelley%26aufirst%3DJ.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DPrediction%2520of%2520Absolute%2520Solvation%2520Free%2520Energies%2520Using%2520Molecular%2520Dynamics%2520Free%2520Energy%2520Perturbation%2520and%2520the%2520Opls%2520Force%2520Field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D1509%26epage%3D1519%26doi%3D10.1021%2Fct900587b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Schrödinger
LLC</span>.  <i>Schrödinger Release 2014-4:
Watermap</i>; <span class="NLM_publisher-name">Schrödinger LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=Schr%C3%B6dinger%0ALLC&title=Schr%C3%B6dinger+Release+2014-4%3A%0AWatermap"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DSchr%25C3%25B6dinger%2520Release%25202014-4%253A%250AWatermap%26pub%3DSchr%25C3%25B6dinger%2520LLC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Clinical
and
Laboratory Standards Institute</span>.  <i>M07-A10:
Methods for Dilution Antimicrobial Susceptibility Tests for Bacteria
That Grow Aerobically; Approved Standard</i>; <span class="NLM_publisher-name">Clinical and Laboratory Standards Institute</span>: <span class="NLM_publisher-loc">Wayne, PA</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=Clinical%0Aand%0ALaboratory+Standards+Institute&title=M07-A10%3A%0AMethods+for+Dilution+Antimicrobial+Susceptibility+Tests+for+Bacteria%0AThat+Grow+Aerobically%3B+Approved+Standard"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DM07-A10%253A%250AMethods%2520for%2520Dilution%2520Antimicrobial%2520Susceptibility%2520Tests%2520for%2520Bacteria%250AThat%2520Grow%2520Aerobically%253B%2520Approved%2520Standard%26pub%3DClinical%2520and%2520Laboratory%2520Standards%2520Institute%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Clinical and Laboratory Standards
Institute</span>.  <i>M11-A8: Methods for Antimicrobial
Susceptibility Testing of Anaerobic Bacteria; Approved Standard</i>; <span class="NLM_publisher-name">Clinical and Laboratory Standards Institute</span>: <span class="NLM_publisher-loc">Wayne, PA</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=Clinical+and+Laboratory+Standards%0AInstitute&title=M11-A8%3A+Methods+for+Antimicrobial%0ASusceptibility+Testing+of+Anaerobic+Bacteria%3B+Approved+Standard"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DM11-A8%253A%2520Methods%2520for%2520Antimicrobial%250ASusceptibility%2520Testing%2520of%2520Anaerobic%2520Bacteria%253B%2520Approved%2520Standard%26pub%3DClinical%2520and%2520Laboratory%2520Standards%2520Institute%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nociari, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shalev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benias, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, C.</span></span> <span> </span><span class="NLM_article-title">A Novel One-Step, Highly Sensitive Fluorometric Assay to Evaluate Cell-Mediated Cytotoxicity</span>. <i>J. Immunol. Methods</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>213</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1016/S0022-1759(98)00028-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1016%2FS0022-1759%2898%2900028-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=9692848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADyaK1cXjvVSksb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=213&publication_year=1998&pages=157-167&author=M.+M.+Nociariauthor=A.+Shalevauthor=P.+Beniasauthor=C.+Russo&title=A+Novel+One-Step%2C+Highly+Sensitive+Fluorometric+Assay+to+Evaluate+Cell-Mediated+Cytotoxicity&doi=10.1016%2FS0022-1759%2898%2900028-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">A novel one-step, highly sensitive fluorometric assay to evaluate cell-mediated cytotoxicity</span></div><div class="casAuthors">Nociari, Marcelo M.; Shalev, Assaf; Benias, Petros; Russo, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunological Methods</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">213</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-167</span>CODEN:
                <span class="NLM_cas:coden">JIMMBG</span>;
        ISSN:<span class="NLM_cas:issn">0022-1759</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">In this study, a fluorometric method using alamarBlue has been developed for detecting cell-mediated cytotoxicity in vitro.  AlamarBlue is a non-toxic metabolic indicator of viable cells that becomes fluorescent upon mitochondrial redn.  Specific lysis of targets by effector cells is quantified by comparing the total no. of viable cells in wells contg. effector and targets together, vs. with wells where target and effector cells were sep. seeded.  Cell-mediated cytotoxic activity by alloreactive T cells and natural killer cells has been detected using a novel application of the alamarBlue technique.  The assay that we have developed to detect cell-mediated cytotoxicity is extremely sensitive and specific and requires a significant lower no. of effector cells than the std. 51Cr assay.  Since alamarBlue reagent is non-toxic to cells and the assay can be performed under sterile conditions, effector cells may be recovered at the end for further anal. or cell expansion, if desired.  Direct comparison of cell-mediated cytotoxicity measured by the alamarBlue method with the std. 51Cr release assay revealed that the former method is as specific and more sensitive than the conventional assay.  Moreover, very small inter and intra-assay variations have been obsd. for alamarBlue cytotoxicity assays.  In conclusion, this study shows that the alamarBlue assay is an extremely sensitive, economical, simple and non-toxic procedure to evaluate cell-mediated cytotoxicity that yields accurate results using a limited no. of effector cells.  Furthermore, since this assay is a one-step procedure, and does not involve any risk for the personnel, it may be useful to analyze automatically cell-mediated cytotoxicity in a large no. of samples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq03rcpA2J96bVg90H21EOLACvtfcHk0ljiepSamQPMeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjvVSksb4%253D&md5=9bce12d254e546d7a41ec2a152abd4cb</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2FS0022-1759%2898%2900028-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-1759%252898%252900028-3%26sid%3Dliteratum%253Aachs%26aulast%3DNociari%26aufirst%3DM.%2BM.%26aulast%3DShalev%26aufirst%3DA.%26aulast%3DBenias%26aufirst%3DP.%26aulast%3DRusso%26aufirst%3DC.%26atitle%3DA%2520Novel%2520One-Step%252C%2520Highly%2520Sensitive%2520Fluorometric%2520Assay%2520to%2520Evaluate%2520Cell-Mediated%2520Cytotoxicity%26jtitle%3DJ.%2520Immunol.%2520Methods%26date%3D1998%26volume%3D213%26spage%3D157%26epage%3D167%26doi%3D10.1016%2FS0022-1759%2898%2900028-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mathes, C.</span></span> <span> </span><span class="NLM_article-title">Qpatch: The Past, Present and Future of Automated Patch Clamp</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1517/14728222.10.2.319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=10.1517%2F14728222.10.2.319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=16548779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;key=1%3ACAS%3A528%3ADC%252BD28XivVOmt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2006&pages=319-327&author=C.+Mathes&title=Qpatch%3A+The+Past%2C+Present+and+Future+of+Automated+Patch+Clamp&doi=10.1517%2F14728222.10.2.319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">QPatch: the past, present and future of automated patch clamp</span></div><div class="casAuthors">Mathes, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">319-327</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Ashley Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  The QPatch 16 significantly increases throughput for gigaseal patch clamp expts., making direct measurements in ion channel drug discovery and safety testing feasible.  Released to the market in the Autumn of 2004 by Sophion Bioscience, the QPatch originated from work done at NeuroSearch (Denmark) in the early days of automated patch clamp.  Today, the QPatch provides many unique features.  For example, only the QPatch includes an automated cell prepn. station making several hours of unattended operation possible.  The 16-channel electrode array, called the QPlate, includes glass-coated microfluidic channels for less compd. absorption and, hence, more accurate IC50 values.  The microfluidic pathways also allow for very small amts. of compd. used for each expt. (∼ 5 μl per addn.).  Only the QPatch has four independent pipetting heads for more efficient liq. handling (esp. for ligand-gated ion channel expts.).  Patch clamp recordings with the QPatch match the high quality of conventional patch clamp and in some cases the results are even better.  For example, only the QPatch includes 100% series resistance compensation for the elimination of false positives due to voltage errors.  Finally, the modular QPatch 16 was designed with more channels in mind.  The upgrade pathway to 48-channels (the QPatch HT) will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqywPZMP8PNwrVg90H21EOLACvtfcHk0lguKX8kGqu33A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivVOmt7k%253D&md5=7dcd02a69652955e261f6928fbb76b81</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1517%2F14728222.10.2.319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.10.2.319%26sid%3Dliteratum%253Aachs%26aulast%3DMathes%26aufirst%3DC.%26atitle%3DQpatch%253A%2520The%2520Past%252C%2520Present%2520and%2520Future%2520of%2520Automated%2520Patch%2520Clamp%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2006%26volume%3D10%26spage%3D319%26epage%3D327%26doi%3D10.1517%2F14728222.10.2.319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XCS','PDB','2XCS'); return false;">PDB: 2XCS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BUL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BUL','PDB','4BUL'); return false;">PDB: 4BUL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FM4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FM4','PDB','6FM4'); return false;">PDB: 6FM4</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i95"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00394">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_82626"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00394">10.1021/acs.jmedchem.9b00394</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Crystal structure of <i>S. aureus</i> DNA gyrase in complex with <b>ID-130</b>; hERG assay: hERG-CHO automated patch-clamp; synthesis of intermediates with general formulas <b>I</b>–<b>XX</b>; <sup>1</sup>H-NMR and <sup>13</sup>C-NMR spectra of final compounds; HPLC analysis data of final compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and associated biochemical and biological data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_002.csv">CSV</a>)</p></li><li><p class="inline">3D PDB file for <a id="rightTab-" class=" internalNav" href="#fig9">Figures <a id="rightTab-" class=" internalNav" href="#fig9">9</a></a> and <a id="rightTab-" class=" internalNav" href="#fig10">10</a> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_003.pdb">PDB</a>)</p></li><li><p class="inline">3D PDB file for <a id="rightTab-" class=" internalNav" href="#fig12">Figure <a id="rightTab-" class=" internalNav" href="#fig12">12</a></a> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_004.pdb">PDB</a>)</p></li><li><p class="inline">3D PDB file for <a id="rightTab-" class=" internalNav" href="#fig11">Figure <a id="rightTab-" class=" internalNav" href="#fig11">11</a></a> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_005.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_001.pdf">jm9b00394_si_001.pdf (6.47 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_002.csv">jm9b00394_si_002.csv (5.03 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_003.pdb">jm9b00394_si_003.pdb (1.9 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_004.pdb">jm9b00394_si_004.pdb (1.9 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00394/suppl_file/jm9b00394_si_005.pdb">jm9b00394_si_005.pdb (1.85 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The PDB code for the structure of <i>S. aureus</i> DNA gyrase in complex with <b>ID-130</b> is 6FM4. Authors will release the atomic coordinate data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00394&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00394%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-16%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00394" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                10MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799c7836a2221a6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
