
<html lang="en"     class="pb-page"  data-request-id="2a6e10eb-88d6-4514-9062-aed0609b670d"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2018.61.issue-11;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.8b00140;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="From Cancer to Pain Target by Automated Selectivity Inversion of a Clinical Candidate" /></meta><meta name="dc.Creator" content="Samo  Turk" /></meta><meta name="dc.Creator" content="Benjamin  Merget" /></meta><meta name="dc.Creator" content="Sameh  Eid" /></meta><meta name="dc.Creator" content="Simone  Fulle" /></meta><meta name="dc.Description" content="Elimination of inadvertent binding is crucial for inhibitor design targeting conserved protein classes like kinases. Compounds in clinical trials provide a rich source for initiating drug design ef..." /></meta><meta name="Description" content="Elimination of inadvertent binding is crucial for inhibitor design targeting conserved protein classes like kinases. Compounds in clinical trials provide a rich source for initiating drug design ef..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 10, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00140" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00140" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00140" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00140" /></link>
        
    
    

<title>From Cancer to Pain Target by Automated Selectivity Inversion of a Clinical Candidate | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00140" /></meta><meta property="og:title" content="From Cancer to Pain Target by Automated Selectivity Inversion of a Clinical Candidate" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0006.jpeg" /></meta><meta property="og:description" content="Elimination of inadvertent binding is crucial for inhibitor design targeting conserved protein classes like kinases. Compounds in clinical trials provide a rich source for initiating drug design efforts by exploiting such secondary binding events. Considering both aspects, we shifted the selectivity of tozasertib, originally developed against AurA as cancer target, toward the pain target TrkA. First, selectivity-determining features in binding pockets were identified by fusing interaction grids of several key and off-target conformations. A focused library was subsequently created and prioritized using a multiobjective selection scheme that filters for selective and highly active compounds based on orthogonal methods grounded in computational chemistry and machine learning. Eighteen high-ranking compounds were synthesized and experimentally tested. The top-ranked compound has 10000-fold improved selectivity versus AurA, nanomolar cellular activity, and is highly selective in a kinase panel. This was achieved in a single round of automated in silico optimization, highlighting the power of recent advances in computer-aided drug design to automate design and selection processes." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00140"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00140">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00140&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00140&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00140&amp;href=/doi/10.1021/acs.jmedchem.8b00140" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 11</span><span class="cit-fg-pageRange">, 4851-4859</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/11" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00071" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00168" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">From Cancer to Pain Target by Automated Selectivity Inversion of a Clinical Candidate</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Samo Turk</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Samo Turk</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">BioMed X Innovation Center, Im Neuenheimer Feld 515, 69120 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Samo++Turk">Samo Turk</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2044-7670" title="Orcid link">http://orcid.org/0000-0003-2044-7670</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Benjamin Merget</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Benjamin Merget</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">BioMed X Innovation Center, Im Neuenheimer Feld 515, 69120 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Benjamin++Merget">Benjamin Merget</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sameh Eid</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sameh Eid</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">BioMed X Innovation Center, Im Neuenheimer Feld 515, 69120 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sameh++Eid">Sameh Eid</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Simone Fulle</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simone Fulle</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">BioMed X Innovation Center, Im Neuenheimer Feld 515, 69120 Heidelberg, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#4620332a2a2306242f29682b3e"><span class="__cf_email__" data-cfemail="f690839a9a93b6949f99d89b8e">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simone++Fulle">Simone Fulle</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7646-5889" title="Orcid link">http://orcid.org/0000-0002-7646-5889</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00140&amp;href=/doi/10.1021%2Facs.jmedchem.8b00140" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 11</span><span class="cit-pageRange">, 4851–4859</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 10, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>27 January 2018</li><li><span class="item_label"><b>Published</b> online</span>10 May 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 June 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00140" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00140</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4851%26pageCount%3D9%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSamo%2BTurk%252C%2BBenjamin%2BMerget%252C%2BSameh%2BEid%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D11%26contentID%3Dacs.jmedchem.8b00140%26title%3DFrom%2BCancer%2Bto%2BPain%2BTarget%2Bby%2BAutomated%2BSelectivity%2BInversion%2Bof%2Ba%2BClinical%2BCandidate%26numPages%3D9%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4859%26publicationDate%3DJune%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00140"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2241</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">7</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00140" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;From Cancer to Pain Target by Automated Selectivity Inversion of a Clinical Candidate&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Samo&quot;,&quot;last_name&quot;:&quot;Turk&quot;},{&quot;first_name&quot;:&quot;Benjamin&quot;,&quot;last_name&quot;:&quot;Merget&quot;},{&quot;first_name&quot;:&quot;Sameh&quot;,&quot;last_name&quot;:&quot;Eid&quot;},{&quot;first_name&quot;:&quot;Simone&quot;,&quot;last_name&quot;:&quot;Fulle&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;10&quot;,&quot;issue&quot;:&quot;11&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;4851-4859&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00140&quot;},&quot;abstract&quot;:&quot;Elimination of inadvertent binding is crucial for inhibitor design targeting conserved protein classes like kinases. Compounds in clinical trials provide a rich source for initiating drug design efforts by exploiting such secondary binding events. Considering both aspects, we shifted the selectivity of tozasertib, originally developed against AurA as cancer target, toward the pain target TrkA. First, selectivity-determining features in binding pockets were identified by fusing interaction grids of several key and off-target conformations. A focused library was subsequently created and prioritized using a multiobjective selection scheme that filters for selective and highly active compounds based on orthogonal methods grounded in computational chemistry and machine learning. Eighteen high-ranking compounds were synthesized and experimentally tested. The top-ranked compound has 10000-fold improved selectivity versus AurA, nanomolar cellular activity, and is highly selective in a kinase panel. This was achieved &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00140&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00140" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00140&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00140" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00140&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00140" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00140&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00140&amp;href=/doi/10.1021/acs.jmedchem.8b00140" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00140" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00140" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00140%26sid%3Dliteratum%253Aachs%26pmid%3D29746776%26genre%3Darticle%26aulast%3DTurk%26date%3D2018%26atitle%3DFrom%2BCancer%2Bto%2BPain%2BTarget%2Bby%2BAutomated%2BSelectivity%2BInversion%2Bof%2Ba%2BClinical%2BCandidate%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D11%26spage%3D4851%26epage%3D4859%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292461" title="Neurophysiology">Neurophysiology</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/11" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/jmcmar.2018.61.issue-11/20180614/jmcmar.2018.61.issue-11.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/medium/jm-2018-001408_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00140&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Elimination of inadvertent binding is crucial for inhibitor design targeting conserved protein classes like kinases. Compounds in clinical trials provide a rich source for initiating drug design efforts by exploiting such secondary binding events. Considering both aspects, we shifted the selectivity of tozasertib, originally developed against AurA as cancer target, toward the pain target TrkA. First, selectivity-determining features in binding pockets were identified by fusing interaction grids of several key and off-target conformations. A focused library was subsequently created and prioritized using a multiobjective selection scheme that filters for selective and highly active compounds based on orthogonal methods grounded in computational chemistry and machine learning. Eighteen high-ranking compounds were synthesized and experimentally tested. The top-ranked compound has 10000-fold improved selectivity versus AurA, nanomolar cellular activity, and is highly selective in a kinase panel. This was achieved in a single round of automated in silico optimization, highlighting the power of recent advances in computer-aided drug design to automate design and selection processes.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41027" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41027" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The focus of R&D efforts on kinases for more than two decades has resulted in many kinase inhibitors with known bioactivity and safety profiles. These compounds provide a great source for initiating new compound design efforts, as for instance many compounds inhibit more than one molecular target and can have a therapeutic impact via so far unknown or ignored mechanisms.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> The latter is particularly true for inhibitors that bind to the highly conserved ATP-binding pocket of kinases and, thus, often bind unintentionally also to secondary targets. Inhibiting multiple targets can result in beneficial synergistic therapeutic effects but also could lead to unwanted side effects.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a></div><div class="NLM_p last">Hence, specifically switching off the activity against undesired targets while maintaining or even improving the affinity to the key target is of great importance, especially when treating nonlife-threatening diseases such as inflammation or pain. Here, we report the automated design of improved inhibitors of the tropomyosin receptor kinase A (TrkA) by jointly employing a de novo design platform and a multiobjective selection scheme that considers selectivity and activity aspects as predicted by novel in silico tools. The selection strategy aimed to shift the selectivity of a particular kinase inhibitor from its original key target toward another validated kinase and takes in addition the selectivity profile against the kinome into account. The initial target–compound pair (i.e., TrkA and tozasertib) was chosen by mining the available kinase profiling data<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> for kinase inhibitors that had entered clinical trials but also inhibit noncancer targets. The corresponding protein binding sites were subsequently analyzed with respect to selectivity-determining features to further prioritize target–compound pairs (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details). The original key target of tozasertib is Aurora kinase A (AurA), whose inhibition has common adverse effects such as neutropenia and hematological toxicities.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In turn, the profiling data revealed that tozasertib also inhibits TrkA, which is a validated drug target for cancer and pain.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Thus, inverting the selectivity of tozasertib from AurA toward TrkA will likely result in a new compound series with reduced side effects and hence is of potential interest for pain treatment.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66511" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66511" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Identification of Selectivity-Determining Features in TrkA</h3><div class="NLM_p">The selectivity hot-spots in the TrkA binding site were initially identified by fusing atom-based interaction energy grids of several TrkA, Aurora A and B structures into one content-rich representation of target specific subpockets (i.e., energy grids representing the conformational flexibility of the key and off-targets were calculated, respectively, and finally fused to one representation via difference rules; see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf" class="ext-link">Supporting Information</a> for details including validation results). The resulting “selectivity grids” highlight three areas of interest for compound optimization: two are favorable hydrophobic subpockets for TrkA selectivity adjacent to the gatekeeper residue and enclosed between the Asp residue of the DFG-motif and a Phe residue of the glycine-rich loop (G-loop), respectively, and one is an unfavorable pocket for TrkA selectivity that overlaps with the cyclopropyl moiety of tozasertib (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/medium/jm-2018-001408_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Selectivity hot-spot areas in TrkA binding site. Predicted binding modes of (a) tozasertib and (b) compound <b>a1</b> to TrkA (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YNE">4YNE</a>). Compound <b>a1</b> occupies the favorable hydrophobic subpockets adjacent to the gatekeeper residue (green) but does not reach into the unfavorable area (red) occupied by the cyclopropyl moiety of tozasertib. The favorable subpockets adjacent to the Phe residue of the G-Loop is occupied by substituents in the <b>B</b> set. Note that Phe521 of the G-loop samples differing conformations in other TrkA structures and that the PDB structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YNE">4YNE</a> was chosen for binding mode prediction and structure–activity analysis because it resulted into the most favorable docking scores (i.e., when bound to compounds of the <b>B</b> series).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00140&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Virtual Compound Library Design</h3><div class="NLM_p">The identified selectivity-determining areas subsequently guided the compound library design where two parts of tozasertib were enumerated using commercially available drug-like fragments and retrosynthetic rules (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The resulting compound set <b>A</b> contains 7404 possible modifications of the amino-5-methylpyrazole that forms hydrogen bonds with the hinge and extends toward the gatekeeper residue, while set <b>B</b> contains 6326 possible modifications of the <i>N</i>-(4-aminothiophene)cyclopropylcarboxamide. It should be noted that in set <b>A</b> the cyclopropylcarboxamide moiety was removed to avoid potential intramolecular steric clashes and that in set <b>B</b> modifications extend toward the DFG motif but can potentially also form interactions with the gatekeeper residue. The resulting library was prioritized using a multiobjective compound selection scheme that filters for selective and highly active compounds (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). All employed prediction tools showed good prediction power in an initial validation on project-specific data (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf" class="ext-link">Supporting Information</a> for details).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/medium/jm-2018-001408_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Summary of computationally designed hit compounds. Affinity values for tozasertib and the top hits are listed for the key target (TrkA; green) and off-target (AurA; red). Modifications compared to the starting compound are highlighted in bold lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00140&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/medium/jm-2018-001408_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Multiobjective compound selection scheme. After initial binding pose generation via docking calculations, the compounds are prioritized via in silico methods that filter first with respect to selectivity criteria (i.e., the ligand-based “kinase profiling platform” approach filters out promiscuous compounds, while the structure-based “selectivity grids” approach filters for TrkA–Aurora selective compounds). Finally, highly active compounds on the key target are prioritized via two complementary machine learning technologies (i.e., using fragment-based matched molecular pair descriptors and based on quantum mechanical calculations, respectively). The final selection does not rely on any manual filtering and includes in the case of the compound set <b>A</b> top ranking compounds from the MMP/ML and QM/ML technologies and in the case of the set <b>B</b> top ranking compounds from MMP/ML.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00140&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Selectivity and Activity Optimization</h3><div class="NLM_p">Initially, binding poses of compounds in sets <b>A</b> and <b>B</b> were generated with Glide SP.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Compounds were removed for reasons of either poor docking scores (> −7.5 kcal/mol), wrong orientation, or lack of key interactions (<a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). In the next step (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), compounds with an unfavorable selectivity profile were filtered out. This was accomplished via machine learning-based activity prediction models<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> that were used (I) to remove promiscuous compounds (i.e., predicted to be active at IC<sub>50</sub> of 500 nM on ≥20 kinases) and (II) those that are predicted to be highly active (IC<sub>50</sub> < 10 nM) on Aurora A, B, or C kinases. The selectivity filtering was complemented by a structure-based procedure employing the TrkA–Aurora “selectivity grids” for rescoring of docking solutions. The remaining compounds in both sets (<b>A</b>, 592; <b>B</b>, 1145) were finally prioritized for highly active compounds using two complementary machine learning (ML) technologies. All compounds were evaluated by an “MMP/ML” approach,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> which is trained on fragment-based matched molecular pairs (MMPs) and quantifies compound activity differences. In addition, compounds in the <b>A</b> set interacting with the Phe gatekeeper were additionally evaluated by a hybrid QM/ML pipeline. This pipeline is trained on high-level quantum mechanical (QM) calculations to quantify ligand–gatekeeper interactions and rescores the top hits in a second step with fragment molecular orbital calculations, taking the entire binding pocket into consideration.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a></div><div class="NLM_p last">The final selection of compounds was obtained in an automated fashion based on the scoring ranks (i.e., without any manual selection beside synthesizability criteria). Eighteen of the top prioritized compounds were synthesized and tested for inhibition on TrkA and AurA (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf" class="ext-link">Tables S1–S2</a>), nine each from sets <b>A</b> (<b>a1</b>–<b>a9</b>) and <b>B</b> (<b>b1</b>–<b>b9</b>). The synthesized compounds were from the top ∼50 compounds from sets <b>A</b> and <b>B</b>, respectively. This yield was achieved by considering synthetic pathway and building block availability in the library design.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Experimental Testing on TrkA and AurA</h3><div class="NLM_p">An initial screening via binding assays revealed that 4 out of 18 tested compounds (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a><named-content content-type="anchor" rid="tbl2" type="simple"></named-content>) have an improved activity/selectivity profile compared to the starting compound tozasertib (i.e., <b>a1</b>, <b>a4</b>, <b>b7</b>, and <b>b8</b>; <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) and originate from both sets <b>A</b> and <b>B</b>. Compound <b>a1</b> shows the best overall improvement compared to tozasertib with slightly higher TrkA affinity and almost complete loss of the AurA activity (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Compound <b>a1</b> was highly scored with both orthogonal ML methods used for the final ranking (ranked 1 with MMP/ML and 3 with QM/ML). Predicted binding modes <b>(</b><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) indicate that, in contrast to tozasertib, <b>a1</b> forms an additional π–π interaction with the gatekeeper residue Phe in TrkA via the fluorophenyl moiety and does not occupy the adjacent unfavorable area (occupied by the cyclopropylcarboxamide group of tozasertib). This trend is consistent with reference <b>2</b> (i.e., tozasertib without cyclopropylcarboxamide), which shows improved TrkA–AurA selectivity (ΔΔp<i>K</i><sub>d</sub> = 0.9) and thus underscores the importance of avoiding this unfavorable area (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Compound <b>a4</b> also has a slightly improved TrkA affinity but retains AurA binding. In contrast to the cyclopentyl of <b>a4</b>, the polarized fluorophenyl of <b>a1</b> seems to make less favorable interactions with the Leu gatekeeper in AurA, possibly explaining the higher selectivity of compound <b>a1</b> compared to <b>a4</b>. On the basis of the primary screening results, it is apparent that in the <b>A</b> set all triazole-containing compounds are inactive on both TrkA and AurA (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf" class="ext-link">Table S1</a>), whereas modifications containing a pyrazole hinge binder in general show activity against TrkA as well as TrkA–AurA selectivity at 100 nM screening concentration. Interestingly, even a small extension toward the gatekeeper, such as the <i>tert</i>-butyl moiety in compound <b>a8</b> (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf" class="ext-link">Table S1</a>), leads to a complete loss of affinity to AurA. This underlines that bulky moieties are less tolerated by the AurA binding pocket and confirms the vicinity of the gatekeeper as selectivity-determining region.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Summary of Compound Sets <b>A</b> and <b>B</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">no. of compounds</th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">no. of experimentally verified hits<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">set</th><th class="colsep0 rowsep0" align="center" char=".">no. calculated</th><th class="colsep0 rowsep0" align="center" char=".">no. synthesized</th><th class="colsep0 rowsep0" align="center" char=".">no. more selective</th><th class="colsep0 rowsep0" align="center" char=".">TrkA < 10 nM</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>A</b></td><td class="colsep0 rowsep0" align="char" char=".">7404</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>B</b></td><td class="colsep0 rowsep0" align="char" char=".">6326</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">2</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Number of compounds which are TrkA-selective vs AurA [and which have in addition TrkA activity <10 nM in primary screen].</p></div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Experimental Affinity Values of Top Hits</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center" char=".">TrkA [p<i>K</i><sub>d</sub>]</th><th class="colsep0 rowsep0" align="center" char=".">AurA [p<i>K</i><sub>d</sub>]</th><th class="colsep0 rowsep0" align="center" char=".">selectivity [Δp<i>K</i><sub>d</sub>]</th><th class="colsep0 rowsep0" align="center" char=".">improved selectivity [ΔΔp<i>K</i><sub>d</sub>]</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tozasertib</td><td class="colsep0 rowsep0" align="char" char=".">8.5</td><td class="colsep0 rowsep0" align="char" char=".">9.3</td><td class="colsep0 rowsep0" align="char" char=".">–0.8</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">reference <b>2</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">7.5</td><td class="colsep0 rowsep0" align="char" char=".">7.4</td><td class="colsep0 rowsep0" align="char" char=".">0.1</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a1</b></td><td class="colsep0 rowsep0" align="char" char=".">8.6</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char=".">4.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a4</b></td><td class="colsep0 rowsep0" align="char" char=".">8.8</td><td class="colsep0 rowsep0" align="char" char=".">7.0</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">2.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>b7</b></td><td class="colsep0 rowsep0" align="char" char=".">9.0</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>b8</b></td><td class="colsep0 rowsep0" align="char" char=".">8.4</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Tozasertib without cyclopropylcarboxamide. Reference <b>2</b> has an improved selectivity versus tozasertib, which is in line with the unoccupied unfavorable TrkA-selectivity area. Hit compounds from sets <b>A</b> and <b>B</b> restore (or even improve) activity on TrkA without restoring AurA activity.</p></div></div></div><div class="NLM_p">The two compounds <b>b7</b> and <b>b8</b> are highly similar and both introduce an additional methylene-linker between a sulfur and a <i>para</i>-substituted phenyl ring (<i>para</i>-fluoro in <b>b7</b> and <i>para</i>-methyl in <b>b8</b>). The phenyl rings in both compounds are predicted to stack with Phe521 from the G-loop, potentially benefiting from a more favorable orientation provided by the additional methylene-linker group compared to tozasertib. The higher activity of <b>b7</b> against TrkA indicates that the electron-withdrawing fluoro-substitution of <b>b7</b> increases the strength of the stacking interaction compared to the electron-donating methyl-substitution of <b>b8</b>. The primary screening results of the <b>B</b> set overall indicate a positive effect of a methylene-linker between the ligand scaffold and the terminal aryl as well as a <i>para</i>-substitution on the phenyl ring for TrkA–AurA selectivity (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf" class="ext-link">Table S2</a>).</div><div class="NLM_p last">The best compound from each set was further characterized in a cell-based functional assay revealing low nanomolar cellular potency for both compounds (<b>a1</b>, IC<sub>50</sub> = 26 nM; <b>b7</b>, IC<sub>50</sub> = 23 nM). This indicates that the compounds are able to reach the intracellular kinase domain and thus permeate through the cellular membrane.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Kinase Profiling</h3><div class="NLM_p">To evaluate the overall selectivity profile of all four hit compounds, profiling on a panel of 92 wild-type kinases and 5 additional variants was conducted at 100 nM screening concentration. The kinase selectivity of compound <b>a1</b> is very high, with only six kinases (excluding TrkA) inhibited, leading to a selectivity score of <i>S</i>(35%) = 0.08 and <i>S</i>(10%) = 0.04 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>; Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf" class="ext-link">Table S3</a>). The remaining three hit compounds show slightly lower kinome selectivity (<i>S</i>(35%) ≈ 0.2; Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf" class="ext-link">Figure S1</a>). At a concentration of 1 μM, <b>a1</b> still inhibits only 19 kinases, which is in line with the initial filtering step via the in silico kinase profiling platform that has excluded promiscuous compounds (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf" class="ext-link">Figure S2</a>). It is also noteworthy that all four hit compounds are inactive on AurB, which was part of the off-target ensemble in the selectivity optimization scoring via the “selectivity grids”. Compound <b>a1</b> displays the most unique binding profile and is even more selective than the starting compound tozasertib.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/medium/jm-2018-001408_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Profiling results. (a) Kinome profiling data of compound <b>a1</b>. With <1% control, compound <b>a1</b> is highly active only against the key target TrkA. Beside TrkA, six further kinases are inhibited at a screening concentration of 100 nM (i.e., TrkB, TrkC, FLT3, KIT, PDGFRb, RET), leading to a selectivity score of <i>S</i>(35%) = 0.08. The compound is inactive on the off-targets AurA and AurB (white triangles). (b) Heatmap of kinase profiles of the hit compounds <b>a1</b>, <b>a4</b>, <b>b7</b>, and <b>b8</b> as well as the starting compound tozasertib. The ABL1 variants include phosphorylated version of the ABL1 mutants E255K and T315I as well as the nonphosphorylated and phosphorylated versions of the ABL1 wild type.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00140&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The kinases inhibited by compound <b>a1</b> include, like for tozasertib, FLT3, and RET (both common cancer targets), but also the two other Trk isoforms TrkB and TrkC, rendering <b>a1</b> a pan-Trk inhibitor with excellent selectivity against other kinases. Compared to <b>a1</b>, <b>b8</b> is less active on TrkB and TrkC and inactive on PDGFRb, pointing to the potential value of fusing both modifications in the future. Finally, hematological toxicities such as thrombocytopenia, anemia, and neutropenia are typically associated with ABL and JAK2 inhibitors.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> Thus, it is encouraging that compared to tozasertib, compound <b>a1</b> is also inactive against all tested ABL1 variants (i.e., two mutants and the (un)phosphorylated wild type) as well as Janus kinases (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Overall, the high kinome selectivity of <b>a1</b>, together with the fact that <b>a1</b> is, compared to tozasertib, inhibiting fewer kinases with target-associated toxicity (i.e., AurA and ABL1), makes <b>a1</b> an interesting candidate for further studies. Many Trk inhibitor scaffolds have been reported in publications and patent applications and are summarized in excellent reviews on TrkA inhibitors.<a onclick="showRef(event, 'ref8 ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref8 ref16 ref17 ref18 ref19">(8,16−19)</a> It is noteworthy that the aminopyrazolyl-pyrimidine scaffold in the presented compound series has been already reported previously in the pan-Trk inhibitor AZ-23.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> However, AZ-23 has a significantly different substitution pattern on the pyrimidine ring (i.e., amino linker at position 2 and unsubstituted position 6).</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Role of TrkA for Pain Treatment and Current Status of TrkA Inhibitor Design</h3><div class="NLM_p last">TrkA is a high affinity receptor for the nerve growth factor (NGF) and widely expressed on peripheral pain sensing neurons. Activation of TrkA can lead to pathogenesis of many difficult to treat human pain conditions such as osteoarthritis and cancer-related types of pain. Loss-of-function TrkA variants are associated with congenital insensitivity to pain.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Furthermore, in vivo TrkA inhibition studies indicate pain suppression effects<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> underlining the promise of TrkA as a pain target. Other selective pan-Trk inhibitors already exist<a onclick="showRef(event, 'ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref27">(24−27)</a> and show promise for the treatment of acute and chronic pain. Studies with an allosteric TrkA inhibitor indicate that TrkA specific inhibitors can be expected to have an improved safety profile because pan-Trk inhibitors have a hyperphagic effect.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Reaching isoform selectivity vs TrkB and TrkC in the ATP-binding site is difficult as only two residues differ among the three isoforms (i.e., two Arg in TrkA are replaced by Lys in TrkB and TrkC)<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> but might be tackled in future improvements with modifications of the so far unexplored <i>N</i>-methylpiperazine moiety that form, e.g., cation−π interactions with one of the arginines.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66446" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66446" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Starting from a compound that reached clinical trials for cancer indications, we have designed a new series of TrkA inhibitors by employing a multiobjective selection scheme that filters for selective and highly active compounds. In agreement with the design objectives, the top compound <b>a1</b> is highly active on TrkA, overall highly selective in a kinase panel, and, most importantly from a design perspective, has a 10000-fold improved selectivity against the selected off-target AurA. Nanomolar cellular potency against TrkA further underlines the potential value of <b>a1</b> as an advanced hit compound for the treatment of acute and chronic pain or other conditions resulting from abnormal TrkA activity such as inflammation and cancer.</div><div class="NLM_p last">Promiscuous interactions between drugs and proteins can cause adverse effects.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Thus, being able to switch off undesired targets in a single round of automated in silico optimization underlines the power of recent advances in computer-aided drug design technologies, especially those employing machine learning techniques such as the virtual kinase profiling platform<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> or the MMP/ML approach.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> It is noteworthy that compound <b>a1</b> was the highest-ranking compound in the filtering pipeline (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf" class="ext-link">Table S1</a>). The initially employed “selectivity grids” pointed to selectivity-determining features in the TrkA structure and thereby guided the subsequent compound library design. Furthermore, they were used to filter for TrkA–Aurora selective compounds. Other successful in silico-driven compound design efforts considering pharmacological profiles include those that identified new targets<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> or guided compound optimization.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The uniqueness of our study is that we navigated through the highly conserved and large target class of kinases instead of seeking predictions across more diverse protein classes. Overall, the discovery platform described herein is widely applicable for compound optimization for target classes with common selectivity issues but can be also generally employed for switching off undesired activities in order to reduce adverse effects.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61243" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61243" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Computational Methods</h3><div class="NLM_p">Details about the employed computational methods are described below and in related validation results in the <a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf" class="ext-link">Supporting Information</a>. Kinome tree figures were created using KinMap.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Selection of Initial Target-Compound Pair</h4><div class="NLM_p last">Mining kinase inhibitors in clinical trials (obtained from the data sources ChEMBL and ClinicalTrials.gov) pointed to TrkA as drug target for pain treatment while mining bioactivity data of a curated ChEMBL v22<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> kinase inhibitor panel and other kinase profiling data sets<a onclick="showRef(event, 'ref3 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref3 ref5 ref6">(3,5,6)</a> pointed to the Aurora inhibitor tozasertib as a promising starting compound.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Identification of Selectivity Determining Features in TrkA</h4><div class="NLM_p">PDB structures of TrkA, AurA, and AurB kinases in the DFG-in state were used for the generation of the selectivity grids (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>; i.e. 4 PDB structures of TrkA were used as the key target set, while 67 AurA and 2 AurB PDB structures constituted the off-target set). All kinase structures were aligned to <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4pmt">4pmt</a> using PyMOL, considering only binding site residues (defined as all residues within 5 Å of any bound ligand). Atom-based energy grids were calculated for the aligned structures using AutoGrid (version 4.2.5),<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> a grid spacing of 0.375 Å, and covering all ligand atoms in the PDB structures. Subsequently, energy grids were fused into single key and off-target grids by taking the minimum (median) interaction energy value at each point across the respective grid ensemble for scoring (for visualization). Extreme energy values from steric clashes were capped at a maximum value corresponding to the absolute value of the most favorable interaction energy. Finally, the key and off-target fused grids were combined using fusion rules.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> An aromatic carbon grid representation (A probe) was used to identify selective hydrophobic subpockets, while the acceptor oxygen (OA), acceptor nitrogen (NA), and donor hydrogen (HD) probes were used to identify polar subpockets.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. PDB Structures Used to Generate “Selectivity Grids”</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">kinase</th><th class="colsep0 rowsep0" align="center">PDB codes</th><th class="colsep0 rowsep0" align="center" char=".">no. of models<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TrkA</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4aoj">4aoj</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4pmt">4pmt</a>,<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4yne">4yne</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4yps">4yps</a></td><td class="colsep0 rowsep0" align="char" char=".">6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AurA</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1mq4">1mq4</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ol5">1ol5</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ol6">1ol6</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ol7">1ol7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2c6d">2c6d</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2dwb">2dwb</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2np8">2np8</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2w1c">2w1c</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2w1d">2w1d</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2w1e">2w1e</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2w1f">2w1f</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2w1g">2w1g</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2wtw">2wtw</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2x6d">2x6d</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2x6e">2x6e</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xne">2xne</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xng">2xng</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xru">2xru</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d14">3d14</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d15">3d15</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d2i">3d2i</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d2k">3d2k</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3dj7">3dj7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3e5a">3e5a</a><a class="ref internalNav" href="#t3fn3" aria-label="c">c</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3efw">3efw</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3fdn">3fdn</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3h0z">3h0z</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ha6">3ha6</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3k5u">3k5u</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3m11">3m11</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3myg">3myg</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3nrm">3nrm</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3o50">3o50</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3p9j">3p9j</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3r21">3r21</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3r22">3r22</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uo4">3uo4</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uo5">3uo5</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uod">3uod</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3up2">3up2</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3up7">3up7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3vap">3vap</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4bn1">4bn1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4byi">4byi</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4c3p">4c3p</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4c3r">4c3r</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ceg">4ceg</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dea">4dea</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4deb">4deb</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ded">4ded</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dee">4dee</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dhf">4dhf</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4j8m">4j8m</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4j8n">4j8n</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jaj">4jaj</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jbo">4jbo</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jbp">4jbp</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4o0s">4o0s</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4o0u">4o0u</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4o0w">4o0w</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4prj">4prj</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4utd">4utd</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5aad">5aad</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5aae">5aae</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5aaf">5aaf</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5aag">5aag</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ew9">5ew9</a></td><td class="colsep0 rowsep0" align="char" char=".">76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AurB</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4af3">4af3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4b8m">4b8m</a></td><td class="colsep0 rowsep0" align="char" char=".">3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Total number of models includes individual chains and alternate models present in the PDB codes.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4pmt">4pmt</a> is used as reference structure for alignment.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3e5a">3e5a</a> is the PDB of cocrystallized tozasertib.</p></div></div></div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Virtual Compound Library Design</h4><div class="NLM_p last">A virtual compound library was designed by modification of tozasertib via systematic replacements of 3-methylpyrazole (compound set <b>A</b>) and <i>N</i>-(4-aminothiophene)cyclopropylcarboxamide (compound set <b>B</b>), using the MolPort building block database as source of fragments. For set <b>A</b>, all primary amines connected to an aromatic N with one further connection were extracted and further narrowed down to fragments with ≤15 heavy atoms and exactly one amino group. For set <b>B</b>, aromatic thiole containing fragments were combined with acyl chloride containing fragments. Here, aromatic thiols were initially filtered to keep only those with ≤15 heavy atoms and one amino group while acyl chlorides were filtered to keep only those with ≤10 heavy atoms and no aromatic rings. The resulting fragments in set <b>A</b> were combined with the remaining part of tozasertib without the cyclopropylcarboxamide moiety to avoid potential intramolecular steric clashes. All virtual compounds were finally filtered using the following cutoffs: molecular weight ≤600 Da, log<i>P</i> ≤ 5, number of hydrogen bond donors ≤5, and acceptors ≤12.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Prediction of Binding Modes and Structure-Based Screening via “Selectivity Grids”</h4><div class="NLM_p last">The four prealigned TrkA PDB structures (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) were processed using the Protein Preparation Wizard in the Schrödinger Suite (release 2016-4, Schrödinger, LLC, New York, NY, 2016) by adjusting protonation, optimizing hydrogens, and restricted geometry optimization using the OPLS3 force field. Docking grids were generated for the four protein models using the Glide program,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> including a hydrogen bond constraint to the backbone NH of the Met592 hinge residue. The ensemble docking run employed the standard precision protocol (Glide SP) with enhanced conformational sampling and increasing the number of initial poses to 50000. For each virtual ligand, only the best pose as judged by the Glide SP score was kept for subsequent analysis along with the corresponding protein configuration. To filter the virtual library further, compounds were excluded if: (1) the top pose had a docking score > −7.5 kcal/mol, (2) it had a wrong binding orientation, e.g., the piperazine moiety is buried rather than being solvent-exposed as in tozasertib, or (3) it lacked key tozasertib–TrkA interactions, namel, Phe589 contact, H-bond to Met592, contact to Asp668 in the DFG motif, and a favorable contact to either Arg654 or Arg673. Finally, the top poses of the remaining compounds were rescored by interpolating the energy values from the corresponding TrkA–Aurora selectivity grids to estimate their selectivity propensity.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Filtering of Selective Compounds via in Silico Kinase Profiling Platform</h4><div class="NLM_p last">The employed platform contains machine learning-based activity prediction models for small molecules which were trained on chemical fingerprints and a large and diverse data set of kinase inhibitor data.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The technology was used in a two-step filtering procedure: (1) to remove promiscuous compounds and (2) to filter with respect to selected off-target activity.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> In the first step, compounds were removed that were predicted to be active on ≥20 kinases with a probability of ≥0.7 in high-quality models (132 models with AUC values ≥0.8; activity threshold, IC<sub>50</sub> = 500 nM). In the second step, prediction models were trained for Aurora kinases A, B, and C employing an activity threshold of IC<sub>50</sub> = 10 nM to remove compounds that are highly active on these off-targets. Compounds were finally removed that were predicted to be active with a probability of ≥0.7 on either of the three Aurora models.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Prioritization of Active Compounds via MMP/ML and a QM/ML Pipeline</h4><div class="NLM_p">Two complementary machine learning technologies were employed for prioritizing active compounds: (1) a MMP/ML approach,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> which is trained on matched molecular pairs (MMPs) as descriptors and considers the entire compound for ranking of compound in sets <b>A</b> and <b>B</b>, and (2) a hybrid QM/ML pipeline, which is trained on quantum mechanical calculations and used to optimize the Phe-gatekeeper interaction of compounds in set <b>A</b>.</div><div class="NLM_p">MMP/ML was trained by obtaining all compounds with reported IC<sub>50</sub>, <i>K</i><sub>i</sub> or <i>K</i><sub>d</sub> measurements against TrkA from ChEMBL.<a onclick="showRef(event, 'ref4 ref11'); return false;" href="javascript:void(0);" class="ref ref4 ref11">(4,11)</a> MMPs were extracted using an in-house implementation based on retrosynthetic rules (BRICS<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>) and encoded as concatenated Morgan fingerprint<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> consisting of the static core that is shared between the two molecules forming an MMP and both fragments F1 and F2 (describing the transformation).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> A regression model was trained on 23000 fragment-based MMPs using a deep neural network to predict the change in activity (ΔpIC<sub>50</sub>) associated with each MMP. Model validation was done using a “new fragments” scenario, where fragments in the individual test sets inside a 5-fold cross validation were new to the machine learning model (validation results, see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf" class="ext-link">Supporting Information</a>). The final prediction of change in activity was estimated by employing the median ΔpIC<sub>50</sub> of all five prediction models.</div><div class="NLM_p last">For the hybrid QM/ML pipeline, high-level QM calculations at the B3LYP-D3/6-31+G** level were used to calculate pair interaction energies (PIE) between the gatekeeper residue and a training set of compounds consisting of: (1) 30 compounds selected from set <b>A</b> and (2) 50 benzyl-derivatives of tozasertib (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf" class="ext-link">Table S4</a>). The benzyl derivatives cover several substituents and heterocycles (e.g., halogens, alkyls, and pyridyls) to model various substituent effects. For each compound, the PIE was calculated from the gatekeeper–compound dimer and the respective monomeric systems: PIE = <i>E</i><sub>dimer</sub> – (<i>E</i><sub>gatekeeper</sub> + <i>E</i><sub>compound</sub>). Binding modes for set <b>A</b> were obtained via docking calculation, while the benzyl derivatives were modeled manually in the binding pocket starting from the tozasertib binding mode and each minimized in the OPLS3 force field using Maestro from the Schrödinger suite.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> QM calculations were conducted using GAMESS-US.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Fast Hartree–Fock calculations at the HF/6-31+G** level were used to derive electronic features (dipole moment, quadrupole moment, total self-consistent field energy) of the 80 ligands in the training set and the compounds in set <b>A</b>. A Random Forest regression model predicting the PIE from electronic ligand features was generated based on the training set using Scikit-Learn<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and employed to prioritize the compounds from set <b>A</b> with respect to PIEs with the gatekeeper residue. Hartree–Fock calculations and PIE predictions were applied on the interacting fragment only, i.e., not the entire ligand. The top 50 compounds were further characterized by considering the entire binding pocket by fragment molecular orbital (FMO)<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> calculations at the MP2/6-31(+)G* level (using docked binding poses and a polarizable continuum solvation water model) and sorted according to the sum of PIEs. FMO calculations were set up using Facio<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and calculated using GAMESS-US.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Chemistry</h3><div class="NLM_p">General synthetic route for tozasertib (a–d) and the compound sets <b>A</b> (a, i–j) and <b>B</b> (a, e–g) is described in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/medium/jm-2018-001408_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0005.jpeg" id="GRAPHIC-d7e1447-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00140&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 3-chloroperoxybenzoic acid, DCM, rt, 3 h, yield 95%; (b) <i>N</i>-(4-mercaptophenyl)cyclopropanecarboxamide, TEA, CH<sub>3</sub>CN, 80 °C, 3–10 h, yield 70%; (c) 5-methyl-1<i>H</i>-pyrazol-3-amine, DIPEA, DMF, 95 °C, 16 h, yield 44%; (d) amine (1-methylpiperazine or morpholine), DMF, DIPEA, 90 °C, 6–12 h, yield 43%; (e) corresponding thiol, TEA, CH<sub>3</sub>CN, 80 °C, 3–10 h, yields 20–85%; (f) 5-methyl-1<i>H</i>-pyrazol-3-amine, DIPEA, dioxane, 95 °C, 3–6 h, yields 50–75%; (g) amine (1-methylpiperazine or morpholine), DMF, DIPEA, 90 °C, 6–12 h, yields 30–70%; (h) thiophenol, TEA, THF, 50 °C, yields 20–85%; (i) corresponding amine, DIPEA, dioxane, 95 °C, 3–6 h, yields 50–75%; (j) amine (1-methylpiperazine or morpholine), DMF, DIPEA, 90 °C, 6–12 h, yields 30–70%.</p></p></figure><div class="NLM_p">Final yields tozasertib: 31%, reference <b>2</b>2, 41%; <b>a1</b>, 49%; <b>a2</b>, 37%; <b>a3</b>, 51%; <b>a4</b>, 31%; <b>a5</b>, 38%; <b>a6</b>, 51%; <b>a7</b>, 64%; <b>a8</b>, 34%; <b>a9</b>, 48%; <b>b1</b>, 58%; <b>b2</b>, 41%; <b>b3</b>, 53%; <b>b4</b>, 48%; <b>b5</b>, 51%; <b>b6</b>, 32%; <b>b7</b>, 39%; <b>b8</b>, 41%; <b>b9</b>, 49%. All compounds were synthesized under contract by Enamine Ltd. Purification was performed using HPLC (H<sub>2</sub>O – MeOH; Agilent 1260 Infinity systems equipped with DAD and mass detectors; Waters Sunfire C18 OBD Prep column, 100 Å, 5 μm, 19 mm × 100 mm with SunFire C18 Prep Guard Cartridge, 100 Å, 10 μm, 19 mm × 10 mm).</div><div class="NLM_p last">Compound characterization was done with <sup>1</sup>H NMR and LC/MS. <sup>1</sup>H NMR was performed using Bruker AVANCE DRX 500 and Varian UNITYplus 400. LC/MS was performed using Agilent 1100 series LC/MSD system with DAD/ELSD and Agilent LC\MSD VL (G1956A), SL (G1956B) mass spectrometer, and Agilent 1200 series LC/MSD system with DAD/ELSD and Agilent LC/MSD SL (G6130A), SL (G6140A) mass spectrometer. Both systems used Zorbax SB-C18 1.8 μm 4.6 mm × 15 mm Rapid Resolution cartridge. Mobile phase: A, acetonitrile, 0.1% formic acid; B, water (0.1% formic acid) with gradient (0 min, 100% B; 0.01 min, 100% B; 1.5 min, 0% B; 1.8 min, 0% B; 1.81 min, 100% B) and flow rate of 3 mL/min. Detection was performed using APCI ionization mode and scan range of <i>m</i>/<i>z</i> 80–1000. The purities of all final products of the two reference compounds (tozasertib, reference <b>2</b>) and of 14 out of 18 reported compounds were found to be >95%. The remaining four compounds (i.e., <b>a2</b>, <b>b2</b>, <b>b4</b>, and <b>b8</b>) have a purity between 90 and 95%. All reported compounds passed common PAINS flags,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> as calculated using RDKit.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Experimental Assays</h3><div class="NLM_p last">The primary screen (TrkA at 10 and 100 nM; AurA at 100 nM and 1 μM concentration), <i>K</i><sub>d</sub> measurements, profiling over a panel of 97 kinases (tozasertib, <b>a1</b>, <b>a4</b>, <b>b7</b>, and <b>b8</b> at 100 nM; <b>a1</b> additionally at 1 μM concentration), and cellular assays (<b>a1</b> and <b>b7</b>) were done using the KINOMEscan<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and PathHunter technologies from DiscoverX, respectively. Results of the primary screen were reported as % of control (% control = (test compound signal – positive control signal)/(DMSO signal – positive control signal). The profiling was done via the scanEDGE assay panel, where KIT(D816 V) and KIT (V559D, T670I) were replaced by TrkB and TrkC. See <a href="http://www.discoverx.com" class="extLink">www.discoverx.com</a> for details on the employed profiling assays. Selectivity scores (number of inhibited kinases divided by the total number of tested wild-type kinases) were calculated at activity cut-offs of 35% and 10% of control, respectively.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24"><a href="/doi/suppl/10.1021/acs.jmedchem.8b00140" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38816" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38816" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b00140" class="ext-link">10.1021/acs.jmedchem.8b00140</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Profiling results, results of primary screens, selectivity scores of top hits, the data set used to train the QM/ML approach as well as additional information about the validation results of the employed prediction tools, about the biological assays and assay results, and analytic data (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Predicted binding pose of <b>a1</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings for all chemical structures mentioned (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf">jm8b00140_si_001.pdf (1.2 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_002.pdb">jm8b00140_si_002.pdb (600.75 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_003.csv">jm8b00140_si_003.csv (1.14 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b00140" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32213" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32213" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simone Fulle</span> - <span class="hlFld-Affiliation affiliation">BioMed X Innovation Center, Im Neuenheimer Feld 515, 69120 Heidelberg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7646-5889" title="Orcid link">http://orcid.org/0000-0002-7646-5889</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3157445d5d547153585e1f5c49"><span class="__cf_email__" data-cfemail="93f5e6fffff6d3f1fafcbdfeeb">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Samo Turk</span> - <span class="hlFld-Affiliation affiliation">BioMed X Innovation Center, Im Neuenheimer Feld 515, 69120 Heidelberg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2044-7670" title="Orcid link">http://orcid.org/0000-0003-2044-7670</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Benjamin Merget</span> - <span class="hlFld-Affiliation affiliation">BioMed X Innovation Center, Im Neuenheimer Feld 515, 69120 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sameh Eid</span> - <span class="hlFld-Affiliation affiliation">BioMed X Innovation Center, Im Neuenheimer Feld 515, 69120 Heidelberg, Germany</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>S.T., B.M., and S.E. contributed equally to this work as a first author. The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e1643-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33234" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33234" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Christian Tidona and Ann De Beuckelaer (BioMed X) for fruitful discussions. The project was supported by the BioMed X Innovation Center, Heidelberg.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">TrkA</td><td class="NLM_def"><p class="first last">tropomyosin receptor kinase A</p></td></tr><tr><td class="NLM_term">AurA</td><td class="NLM_def"><p class="first last">Aurora kinase A</p></td></tr><tr><td class="NLM_term">ML</td><td class="NLM_def"><p class="first last">machine learning</p></td></tr><tr><td class="NLM_term">MMPs</td><td class="NLM_def"><p class="first last">matched molecular pairs</p></td></tr><tr><td class="NLM_term">QM</td><td class="NLM_def"><p class="first last">quantum mechanics</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63325" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63325" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duong-ly, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devarajan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horiuchi, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, J. R.</span></span> <span> </span><span class="NLM_article-title">Kinase Inhibitor Profiling Reveals Unexpected Opportunities to Inhibit Disease-Associated Mutant Kinases</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">772</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2015.12.080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1016%2Fj.celrep.2015.12.080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=26776524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVaqt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=772-781&issue=4&author=K.+C.+Duong-lyauthor=K.+Devarajanauthor=S.+Liangauthor=K.+Y.+Horiuchiauthor=Y.+Wangauthor=H.+Maauthor=J.+R.+Peterson&title=Kinase+Inhibitor+Profiling+Reveals+Unexpected+Opportunities+to+Inhibit+Disease-Associated+Mutant+Kinases&doi=10.1016%2Fj.celrep.2015.12.080"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase Inhibitor Profiling Reveals Unexpected Opportunities to Inhibit Disease-Associated Mutant Kinases</span></div><div class="casAuthors">Duong-Ly, Krisna C.; Devarajan, Karthik; Liang, Shuguang; Horiuchi, Kurumi Y.; Wang, Yuren; Ma, Haiching; Peterson, Jeffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">772-781</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Small-mol. kinase inhibitors have typically been designed to inhibit wild-type kinases rather than the mutant forms that frequently arise in diseases such as cancer.  Mutations can have serious clin. implications by increasing kinase catalytic activity or conferring therapeutic resistance.  To identify opportunities to repurpose inhibitors against disease-assocd. mutant kinases, we conducted a large-scale functional screen of 183 known kinase inhibitors against 76 recombinant mutant kinases.  The results revealed lead compds. with activity against clin. important mutant kinases, including ALK, LRRK2, RET, and EGFR, as well as unexpected opportunities for repurposing FDA-approved kinase inhibitors as leads for addnl. indications.  Furthermore, using T674I PDGFRα as an example, we show how single-dose screening data can provide predictive structure-activity data to guide subsequent inhibitor optimization.  This study provides a resource for the development of inhibitors against numerous disease-assocd. mutant kinases and illustrates the potential of unbiased profiling as an approach to compd.-centric inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbXqJUD9YoWLVg90H21EOLACvtfcHk0lihnN1vHP8UWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVaqt7c%253D&md5=b01e8f6ce2aa441400531399d8cf88fc</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2015.12.080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2015.12.080%26sid%3Dliteratum%253Aachs%26aulast%3DDuong-ly%26aufirst%3DK.%2BC.%26aulast%3DDevarajan%26aufirst%3DK.%26aulast%3DLiang%26aufirst%3DS.%26aulast%3DHoriuchi%26aufirst%3DK.%2BY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26atitle%3DKinase%2520Inhibitor%2520Profiling%2520Reveals%2520Unexpected%2520Opportunities%2520to%2520Inhibit%2520Disease-Associated%2520Mutant%2520Kinases%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D14%26issue%3D4%26spage%3D772%26epage%3D781%26doi%3D10.1016%2Fj.celrep.2015.12.080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">Computational and Practical Aspects of Drug Repositioning</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1089/adt.2015.29011.tiodrrr</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1089%2Fadt.2015.29011.tiodrrr" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=299-306&issue=6&author=T.+I.+Opreaauthor=J.+P.+Overington&title=Computational+and+Practical+Aspects+of+Drug+Repositioning&doi=10.1089%2Fadt.2015.29011.tiodrrr"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1089%2Fadt.2015.29011.tiodrrr&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2015.29011.tiodrrr%26sid%3Dliteratum%253Aachs%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DComputational%2520and%2520Practical%2520Aspects%2520of%2520Drug%2520Repositioning%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2015%26volume%3D13%26issue%3D6%26spage%3D299%26epage%3D306%26doi%3D10.1089%2Fadt.2015.29011.tiodrrr" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metz, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merta, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kifle, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span> <span> </span><span class="NLM_article-title">Navigating the Kinome</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">200</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1038/nchembio.530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1038%2Fnchembio.530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=21336281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=200-202&issue=4&author=J.+T.+Metzauthor=E.+F.+Johnsonauthor=N.+B.+Soniauthor=P.+J.+Mertaauthor=L.+Kifleauthor=P.+J.+Hajduk&title=Navigating+the+Kinome&doi=10.1038%2Fnchembio.530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating the kinome</span></div><div class="casAuthors">Metz, James T.; Johnson, Eric F.; Soni, Niru B.; Merta, Philip J.; Kifle, Lemma; Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">200-202</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although it is increasingly being recognized that drug-target interaction networks can be powerful tools for the interrogation of systems biol. and the rational design of multitargeted drugs, there is no generalized, statistically validated approach to harmonizing sequence-dependent and pharmacol.-dependent networks.  Here we demonstrate the creation of a comprehensive kinome interaction network based not only on sequence comparisons but also on multiple pharmacol. parameters derived from activity profiling data.  The framework described for statistical interpretation of these network connections also enables rigorous investigation of chemotype-specific interaction networks, which is crit. for multitargeted drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOgzF9Q0eH87Vg90H21EOLACvtfcHk0lihnN1vHP8UWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D&md5=09234ec451a4918e5aca6d65e0b65aee</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.530%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DKifle%26aufirst%3DL.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DNavigating%2520the%2520Kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26issue%3D4%26spage%3D200%26epage%3D202%26doi%3D10.1038%2Fnchembio.530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krüger, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Light, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlinchey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadatos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">The ChEMBL Bioactivity Database: An Update</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1083</span>– <span class="NLM_lpage">1090</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt1031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1093%2Fnar%2Fgkt1031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=24214965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslWl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1083-1090&issue=D1&author=A.+P.+Bentoauthor=A.+Gaultonauthor=A.+Herseyauthor=L.+J.+Bellisauthor=J.+Chambersauthor=M.+Daviesauthor=F.+A.+Kr%C3%BCgerauthor=Y.+Lightauthor=L.+Makauthor=S.+McGlincheyauthor=M.+Nowotkaauthor=G.+Papadatosauthor=R.+Santosauthor=J.+P.+Overington&title=The+ChEMBL+Bioactivity+Database%3A+An+Update&doi=10.1093%2Fnar%2Fgkt1031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The ChEMBL bioactivity database: an update</span></div><div class="casAuthors">Bento, A. Patricia; Gaulton, Anna; Hersey, Anne; Bellis, Louisa J.; Chambers, Jon; Davies, Mark; Krueger, Felix A.; Light, Yvonne; Mak, Lora; McGlinchey, Shaun; Nowotka, Michal; Papadatos, George; Santos, Rita; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1083-D1090</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an open large-scale bioactivity database (https://www.ebi.ac.uk/chembl), previously described in the 2012 Nucleic Acids Research Database Issue.  Since then, a variety of new data sources and improvements in functionality have contributed to the growth and utility of the resource.  In particular, more comprehensive tracking of compds. from research stages through clin. development to market is provided through the inclusion of data from United States Adopted Name applications; a new richer data model for representing drug targets has been developed; and a no. of methods have been put in place to allow users to more easily identify reliable data.  Finally, access to ChEMBL is now available via a new Resource Description Framework format, in addn. to the web-based interface, data downloads and web services.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXb5k1IvJNaLVg90H21EOLACvtfcHk0lihnN1vHP8UWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslWl&md5=31b832d03d56ea3065d7aa29618362bc</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1031%26sid%3Dliteratum%253Aachs%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DKr%25C3%25BCger%26aufirst%3DF.%2BA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520ChEMBL%2520Bioactivity%2520Database%253A%2520An%2520Update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26issue%3DD1%26spage%3DD1083%26epage%3D1090%26doi%3D10.1093%2Fnar%2Fgkt1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Analysis of Kinase Inhibitor Selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&issue=11&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+Analysis+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lhn-rBd3rLIiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520Analysis%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26issue%3D11%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anastassiadis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deacon, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devarajan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, J. R.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Assay of Kinase Catalytic Activity Reveals Features of Kinase Inhibitor Selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1039</span>– <span class="NLM_lpage">1045</span>, <span class="refDoi"> DOI: 10.1038/nbt.2017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1038%2Fnbt.2017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=22037377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1039-1045&issue=11&author=T.+Anastassiadisauthor=S.+W.+Deaconauthor=K.+Devarajanauthor=H.+Maauthor=J.+R.+Peterson&title=Comprehensive+Assay+of+Kinase+Catalytic+Activity+Reveals+Features+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt.2017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span></div><div class="casAuthors">Anastassiadis, Theonie; Deacon, Sean W.; Devarajan, Karthik; Ma, Haiching; Peterson, Jeffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1039-1045</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. protein kinase inhibitors are widely used to elucidate cellular signaling pathways and are promising therapeutic agents.  Owing to evolutionary conservation of the ATP-binding site, most kinase inhibitors that target this site promiscuously inhibit multiple kinases.  Interpretation of expts. that use these compds. is confounded by a lack of data on the comprehensive kinase selectivity of most inhibitors.  Here we used functional assays to profile the activity of 178 com. available kinase inhibitors against a panel of 300 recombinant protein kinases.  Quant. anal. revealed complex and often unexpected interactions between protein kinases and kinase inhibitors, with a wide spectrum of promiscuity.  Many off-target interactions occur with seemingly unrelated kinases, revealing how large-scale profiling can identify multitargeted inhibitors of specific, diverse kinases.  The results have implications for drug development and provide a resource for selecting compds. to elucidate kinase function and for interpreting the results of expts. involving kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUljWIhZGPkrVg90H21EOLACvtfcHk0lhn-rBd3rLIiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E&md5=b2f705509b0e09afa46b01d4669f0515</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2017%26sid%3Dliteratum%253Aachs%26aulast%3DAnastassiadis%26aufirst%3DT.%26aulast%3DDeacon%26aufirst%3DS.%2BW.%26aulast%3DDevarajan%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26atitle%3DComprehensive%2520Assay%2520of%2520Kinase%2520Catalytic%2520Activity%2520Reveals%2520Features%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26issue%3D11%26spage%3D1039%26epage%3D1045%26doi%3D10.1038%2Fnbt.2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falchook, G.
S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastida, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurzrock, R.</span></span> <span> </span><span class="NLM_article-title">Aurora Kinase Inhibitors in Oncology Clinical Trials: Current State of the Progress</span>. <i>Semin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">832</span>– <span class="NLM_lpage">848</span>, <span class="refDoi"> DOI: 10.1053/j.seminoncol.2015.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1053%2Fj.seminoncol.2015.09.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=26615129" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWksL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2015&pages=832-848&issue=6&author=G.%0AS.+Falchookauthor=C.+C.+Bastidaauthor=R.+Kurzrock&title=Aurora+Kinase+Inhibitors+in+Oncology+Clinical+Trials%3A+Current+State+of+the+Progress&doi=10.1053%2Fj.seminoncol.2015.09.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Aurora Kinase Inhibitors in Oncology Clinical Trials: Current State of the Progress</span></div><div class="casAuthors">Falchook, Gerald S.; Bastida, Christel C.; Kurzrock, Razelle</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">832-848</span>CODEN:
                <span class="NLM_cas:coden">SOLGAV</span>;
        ISSN:<span class="NLM_cas:issn">0093-7754</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The Aurora kinase family of kinases (Aurora A, B, and C) are involved in multiple mitotic events, and aberrant expression of these kinases is assocd. with tumorigenesis.  Aurora A and Aurora B are validated anticancer targets, and the development of Aurora kinase inhibitors has progressed from preclin. to clin. studies.  A variety of Aurora A, B and pan-Aurora kinase inhibitors have entered the clinic.  The main side effects include febrile neutropenia, stomatitis, gastrointestinal toxicity, hypertension, and fatigue.  Responses including complete remissions have been described in diverse, advanced malignancies, most notably ovarian cancer and acute myelogenous leukemia.  This review highlights the biol. rationale for Aurora kinase as a target, and clin. trials involving Aurora kinase inhibitors, with particular emphasis on published early phase studies, and the obsd. anti-tumor activity of these agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFVqflqPnM6LVg90H21EOLACvtfcHk0lhn-rBd3rLIiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWksL7L&md5=55465822ce60d2cf2dfd19606de09a71</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1053%2Fj.seminoncol.2015.09.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.seminoncol.2015.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DFalchook%26aufirst%3DG.%2BS.%26aulast%3DBastida%26aufirst%3DC.%2BC.%26aulast%3DKurzrock%26aufirst%3DR.%26atitle%3DAurora%2520Kinase%2520Inhibitors%2520in%2520Oncology%2520Clinical%2520Trials%253A%2520Current%2520State%2520of%2520the%2520Progress%26jtitle%3DSemin.%2520Oncol.%26date%3D2015%26volume%3D42%26issue%3D6%26spage%3D832%26epage%3D848%26doi%3D10.1053%2Fj.seminoncol.2015.09.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Norman, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, J. S.</span></span> <span> </span><span class="NLM_article-title">Targeting the Nerve Growth Factor (NGF) Pathway in Drug Discovery. Potential Applications to New Therapies for Chronic Pain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">66</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00964</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00964" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=66-88&issue=1&author=B.+H.+Normanauthor=J.+S.+McDermott&title=Targeting+the+Nerve+Growth+Factor+%28NGF%29+Pathway+in+Drug+Discovery.+Potential+Applications+to+New+Therapies+for+Chronic+Pain&doi=10.1021%2Facs.jmedchem.6b00964"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00964&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00964%26sid%3Dliteratum%253Aachs%26aulast%3DNorman%26aufirst%3DB.%2BH.%26aulast%3DMcDermott%26aufirst%3DJ.%2BS.%26atitle%3DTargeting%2520the%2520Nerve%2520Growth%2520Factor%2520%2528NGF%2529%2520Pathway%2520in%2520Drug%2520Discovery.%2520Potential%2520Applications%2520to%2520New%2520Therapies%2520for%2520Chronic%2520Pain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D1%26spage%3D66%26epage%3D88%26doi%3D10.1021%2Facs.jmedchem.6b00964" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=D.+E.+Shawauthor=M.+Shelleyauthor=J.+K.+Perryauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lhB_-Dg45qsdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merget, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulle, S.</span></span> <span> </span><span class="NLM_article-title">Profiling Prediction of Kinase Inhibitors: Toward the Virtual Assay</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">474</span>– <span class="NLM_lpage">485</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01611</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01611" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=474-485&issue=1&author=B.+Mergetauthor=S.+Turkauthor=S.+Eidauthor=F.+Rippmannauthor=S.+Fulle&title=Profiling+Prediction+of+Kinase+Inhibitors%3A+Toward+the+Virtual+Assay&doi=10.1021%2Facs.jmedchem.6b01611"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01611%26sid%3Dliteratum%253Aachs%26aulast%3DMerget%26aufirst%3DB.%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DEid%26aufirst%3DS.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DFulle%26aufirst%3DS.%26atitle%3DProfiling%2520Prediction%2520of%2520Kinase%2520Inhibitors%253A%2520Toward%2520the%2520Virtual%2520Assay%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D1%26spage%3D474%26epage%3D485%26doi%3D10.1021%2Facs.jmedchem.6b01611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merget, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulle, S.</span></span> <span> </span><span class="NLM_article-title">Coupling Matched Molecular Pairs with Machine Learning for Virtual Compound Optimization</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3079</span>– <span class="NLM_lpage">3085</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00298</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00298" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2gsLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=3079-3085&issue=12&author=S.+Turkauthor=B.+Mergetauthor=F.+Rippmannauthor=S.+Fulle&title=Coupling+Matched+Molecular+Pairs+with+Machine+Learning+for+Virtual+Compound+Optimization&doi=10.1021%2Facs.jcim.7b00298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Coupling Matched Molecular Pairs with Machine Learning for Virtual Compound Optimization</span></div><div class="casAuthors">Turk, Samo; Merget, Benjamin; Rippmann, Friedrich; Fulle, Simone</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3079-3085</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Matched mol. pair (MMP) analyses are widely used in compd. optimization projects to gain insights into structure-activity relationships (SAR).  The anal. is traditionally done via statistical methods but can also be employed together with machine learning (ML) approaches to extrapolate to novel compds.  The here introduced MMP/ML method combines a fragment-based MMP implementation with different machine learning methods to obtain automated SAR decompn. and prediction.  To test the prediction capabilities and model transferability, two different compd. optimization scenarios were designed: (1) "new fragments" which occurs when exploring new fragments for a defined compd. series and (2) "new static core and transformations" which resembles for instance the identification of a new compd. series.  Very good results were achieved by all employed machine learning methods esp. for the new fragments case, but overall deep neural network models performed best, allowing reliable predictions also for the new static core and transformations scenario, where comprehensive SAR knowledge of the compd. series is missing.  Furthermore, the authors show that models trained on all available data have a higher generalizability compared to models trained on focused series and can extend beyond chem. space covered in the training data.  Thus, coupling MMP with deep neural networks provides a promising approach to make high quality predictions on various data sets and in different compd. optimization scenarios.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKw4NG2oltF7Vg90H21EOLACvtfcHk0lhB_-Dg45qsdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2gsLnJ&md5=828c42567a0c5dfc2f189b730055f154</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00298%26sid%3Dliteratum%253Aachs%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DMerget%26aufirst%3DB.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DFulle%26aufirst%3DS.%26atitle%3DCoupling%2520Matched%2520Molecular%2520Pairs%2520with%2520Machine%2520Learning%2520for%2520Virtual%2520Compound%2520Optimization%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D12%26spage%3D3079%26epage%3D3085%26doi%3D10.1021%2Facs.jcim.7b00298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uebayasi, M.</span></span> <span> </span><span class="NLM_article-title">Fragment Molecular Orbital Method: An Approximate Computational Method for Large Molecules</span>. <i>Chem. Phys. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>313</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">701</span>– <span class="NLM_lpage">706</span>, <span class="refDoi"> DOI: 10.1016/S0009-2614(99)00874-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1016%2FS0009-2614%2899%2900874-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=1999&pages=701-706&issue=3&author=K.+Kitauraauthor=E.+Ikeoauthor=T.+Asadaauthor=T.+Nakanoauthor=M.+Uebayasi&title=Fragment+Molecular+Orbital+Method%3A+An+Approximate+Computational+Method+for+Large+Molecules&doi=10.1016%2FS0009-2614%2899%2900874-X"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2FS0009-2614%2899%2900874-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0009-2614%252899%252900874-X%26sid%3Dliteratum%253Aachs%26aulast%3DKitaura%26aufirst%3DK.%26aulast%3DIkeo%26aufirst%3DE.%26aulast%3DAsada%26aufirst%3DT.%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DUebayasi%26aufirst%3DM.%26atitle%3DFragment%2520Molecular%2520Orbital%2520Method%253A%2520An%2520Approximate%2520Computational%2520Method%2520for%2520Large%2520Molecules%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D1999%26volume%3D313%26issue%3D3%26spage%3D701%26epage%3D706%26doi%3D10.1016%2FS0009-2614%2899%2900874-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldridge, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boatz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elbert, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koseki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Windus, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupuis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J. A.,  Jr.</span></span> <span> </span><span class="NLM_article-title">General Atomic and Molecular Electronic Structure System</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1363</span>, <span class="refDoi"> DOI: 10.1002/jcc.540141112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1002%2Fjcc.540141112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADyaK2cXhsFaqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1993&pages=1347-1363&issue=11&author=M.+W.+Schmidtauthor=K.+K.+Baldridgeauthor=J.+A.+Boatzauthor=S.+T.+Elbertauthor=M.+S.+Gordonauthor=J.+H.+Jensenauthor=S.+Kosekiauthor=N.+Matsunagaauthor=K.+A.+Nguyenauthor=S.+Suauthor=T.+L.+Windusauthor=M.+Dupuisauthor=J.+A.+Montgomery&title=General+Atomic+and+Molecular+Electronic+Structure+System&doi=10.1002%2Fjcc.540141112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">General atomic and molecular electronic structure system</span></div><div class="casAuthors">Schmidt, Michael W.; Baldridge, Kim K.; Boatz, Jerry A.; Elbert, Steven T.; Gordon, Mark S.; Jensen, Jan H.; Koseki, Shiro; Matsunaga, Nikita; Nguyen, Kiet A.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1347-63</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">A description of the ab initio quantum chem. package GAMESS is presented.  Chem. systems contg. atoms through Rn can be treated with wave functions ranging from the simplest closed-shell case up to a general MCSCF case, permitting calcns. at the necessary level of sophistication.  Emphasis is given to novel features of the program.  The parallelization strategy used in the RHF, ROHF, UHF, and GVB sections of the program is described, and detailed speedup results are given.  Parallel calcns. can be run on ordinary workstations as well as dedicated parallel machines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pHblAdXU4rVg90H21EOLACvtfcHk0ljzA9pv9Lx_NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhsFaqtQ%253D%253D&md5=4d9e37cd3ec67a92bdfb5b0c0084707c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540141112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540141112%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DM.%2BW.%26aulast%3DBaldridge%26aufirst%3DK.%2BK.%26aulast%3DBoatz%26aufirst%3DJ.%2BA.%26aulast%3DElbert%26aufirst%3DS.%2BT.%26aulast%3DGordon%26aufirst%3DM.%2BS.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26aulast%3DKoseki%26aufirst%3DS.%26aulast%3DMatsunaga%26aufirst%3DN.%26aulast%3DNguyen%26aufirst%3DK.%2BA.%26aulast%3DSu%26aufirst%3DS.%26aulast%3DWindus%26aufirst%3DT.%2BL.%26aulast%3DDupuis%26aufirst%3DM.%26aulast%3DMontgomery%26aufirst%3DJ.%2BA.%26atitle%3DGeneral%2520Atomic%2520and%2520Molecular%2520Electronic%2520Structure%2520System%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1993%26volume%3D14%26issue%3D11%26spage%3D1347%26epage%3D1363%26doi%3D10.1002%2Fjcc.540141112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, M.</span></span> <span> </span><span class="NLM_article-title">Managing Common Toxicities with New Tyrosine Kinase Inhibitors</span> <i>CancerWorld</i> <span class="NLM_year" style="font-weight: bold;">2015</span>, <span class="NLM_issue">November–December</span>,  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">37</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=31-37&issue=November%E2%80%93December&author=M.+Oliveira&title=Managing+Common+Toxicities+with+New+Tyrosine+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOliveira%26aufirst%3DM.%26atitle%3DManaging%2520Common%2520Toxicities%2520with%2520New%2520Tyrosine%2520Kinase%2520Inhibitors%26jtitle%3DCancerWorld%26date%3D2015%26issue%3DNovember%25E2%2580%2593December%26spage%3D31%26epage%3D37" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’shea, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kontzias, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaoka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurence, A.</span></span> <span> </span><span class="NLM_article-title">Janus Kinase Inhibitors in Autoimmune Diseases</span>. <i>Ann. Rheum. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">i111</span>– <span class="NLM_lpage">i115</span>, <span class="refDoi"> DOI: 10.1136/annrheumdis-2012-202576</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1136%2Fannrheumdis-2012-202576" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFyntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2013&pages=i111-i115&author=J.+J.+O%E2%80%99sheaauthor=A.+Kontziasauthor=K.+Yamaokaauthor=Y.+Tanakaauthor=A.+Laurence&title=Janus+Kinase+Inhibitors+in+Autoimmune+Diseases&doi=10.1136%2Fannrheumdis-2012-202576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Janus kinase inhibitors in autoimmune diseases</span></div><div class="casAuthors">O'Shea, John J.; Kontzias, Apostolos; Yamaoka, Kunihiro; Tanaka, Yoshiya; Laurence, Arian</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">Suppl. 2</span>),
    <span class="NLM_cas:pages">ii111-ii115</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">A review.  Biol. therapies directed at proinflammatory cytokines have irrevocably changed the landscape of treatment of rheumatoid arthritis (RA) and other autoimmune diseases.  With the advances in our knowledge in cytokine signalling, the question emerges whether targeting intracellular signalling might also be a safe and efficacious strategy.  Janus kinases or Jaks are crit. for a large family of cytokines and the first Jak inhibitors has been approved by the FDA.  It is therefore timely to consider this new category of drugs and reflect on their potential roles, present and future, in the treatment of RA and related disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWFj0gZ1gj1LVg90H21EOLACvtfcHk0ljzA9pv9Lx_NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFyntbk%253D&md5=2776146c05d128b9659c36ab6b4964fa</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2012-202576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2012-202576%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599shea%26aufirst%3DJ.%2BJ.%26aulast%3DKontzias%26aufirst%3DA.%26aulast%3DYamaoka%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DLaurence%26aufirst%3DA.%26atitle%3DJanus%2520Kinase%2520Inhibitors%2520in%2520Autoimmune%2520Diseases%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2013%26volume%3D72%26spage%3Di111%26epage%3Di115%26doi%3D10.1136%2Fannrheumdis-2012-202576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, M. L.</span></span> <span> </span><span class="NLM_article-title">Trk Kinase Inhibitors as New Treatments for Cancer and Pain</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">319</span>, <span class="refDoi"> DOI: 10.1517/13543770902721261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1517%2F13543770902721261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=19441906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjslWlu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=305-319&issue=3&author=T.+Wangauthor=D.+Yuauthor=M.+L.+Lamb&title=Trk+Kinase+Inhibitors+as+New+Treatments+for+Cancer+and+Pain&doi=10.1517%2F13543770902721261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Trk kinase inhibitors as new treatments for cancer and pain</span></div><div class="casAuthors">Wang, Tao; Yu, Dingwei; Lamb, Michelle L.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">305-319</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Background: Tropomyosin-related kinases (Trks) are a family of receptor tyrosine kinases activated by neurotrophins.  Trks play important roles in pain sensation as well as tumor cell growth and survival signaling.  Thus, inhibitors of Trk receptor kinases might provide targeted treatments for pain and cancer.  Objective: This paper reviews those patent applications since 2002 claiming small-mol. inhibitors of Trk receptor kinases.  Methods: Primary literature and patents were searched with SciFinder and Google Scholar.  Patents were selected based on their relevance to Trks and were evaluated and representative compds. were listed as examples.  Results/conclusion: Several series of Trk inhibitors with excellent in vitro potencies have been reported and a no. of compds. have gone into the clinic.  It should be noted that few of these inhibitors are Trk selective, demonstrating that targeting Trk kinases for treatment of pain and/or cancer offers a promising but also challenging approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvknvXUUz2DbVg90H21EOLACvtfcHk0lgGZa0_Z-2ttw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjslWlu7o%253D&md5=ffc2d3e33102631cac7b89dce7f96a5a</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1517%2F13543770902721261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543770902721261%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26atitle%3DTrk%2520Kinase%2520Inhibitors%2520as%2520New%2520Treatments%2520for%2520Cancer%2520and%2520Pain%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2009%26volume%3D19%26issue%3D3%26spage%3D305%26epage%3D319%26doi%3D10.1517%2F13543770902721261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, E.</span></span> <span> </span><span class="NLM_article-title">Tropomyosin Receptor Kinase Inhibitors: A Patent Update 2009-2013</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">731</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1517/13543776.2014.910195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1517%2F13543776.2014.910195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=24809946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVSqtr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=731-744&author=C.+McCarthyauthor=E.+Walker&title=Tropomyosin+Receptor+Kinase+Inhibitors%3A+A+Patent+Update+2009-2013&doi=10.1517%2F13543776.2014.910195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Tropomyosin receptor kinase inhibitors: a patent update 2009 - 2013</span></div><div class="casAuthors">McCarthy, Clive; Walker, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">731-744</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: Tropomyosin receptor kinases (Trks) are a family of three similar tyrosine kinases activated by peptide hormones of the neurotrophin family.  The nerve growth factor antibody tanezumab has provided clin. proof of concept for inhibition of the TrkA pathway in pain.  As an alternative modality, small-mol. inhibitors of the Trks have been pursued in recent years to probe the role of these neurotrophin pathways in pain, cancer and other indications.  Areas covered: This paper reviews the patent literature between mid-2009 and 2013, claiming inhibitors of Trk family members as the primary biol. targets.  Addnl. patents have been reviewed where Trk is not the main kinase of interest but in which high Trk potency is obsd. and the chem. matter is particularly noteworthy.  Patents pre-dating this period have been reviewed previously.  Scifinder and Google were used to find relevant patents and clin. information using Trk or Tropomyosin as the search term.  Expert opinion: Considerable recent progress has been made in the identification of selective pan Trk inhibitors with pharmacodynamic and pharmacokinetic properties appropriate for clin. evaluation.  Inhibitors of both active and inactive conformations of the Trks as well as peripherally restricted mols. have been identified.  Furthermore, TrkA-selective allosteric inhibitors have recently been disclosed, which enables the biol. of this isoform to be probed.  The recent identification of a TrkA gene fusion in a subset of lung cancer patients will increase further the attraction of Trk inhibition to the pharmaceutical industry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjXwkruGnLXbVg90H21EOLACvtfcHk0lgGZa0_Z-2ttw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVSqtr7L&md5=b924331cd9d69192bc10f53424997550</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1517%2F13543776.2014.910195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2014.910195%26sid%3Dliteratum%253Aachs%26aulast%3DMcCarthy%26aufirst%3DC.%26aulast%3DWalker%26aufirst%3DE.%26atitle%3DTropomyosin%2520Receptor%2520Kinase%2520Inhibitors%253A%2520A%2520Patent%2520Update%25202009-2013%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2014%26volume%3D24%26spage%3D731%26epage%3D744%26doi%3D10.1517%2F13543776.2014.910195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirrmacher, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard-Gauthier, V.</span></span> <span> </span><span class="NLM_article-title">Tropomyosin Receptor Kinase Inhibitors: An Updated Patent Review for 2010–2016 - Part I</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">751</span>, <span class="refDoi"> DOI: 10.1080/13543776.2017.1297796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1080%2F13543776.2017.1297796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=28270010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvFyjtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=733-751&author=J.+Baileyauthor=R.+Schirrmacherauthor=K.+Farrellauthor=V.+Bernard-Gauthier&title=Tropomyosin+Receptor+Kinase+Inhibitors%3A+An+Updated+Patent+Review+for+2010%E2%80%932016+-+Part+I&doi=10.1080%2F13543776.2017.1297796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Tropomyosin receptor kinase inhibitors: an updated patent review for 2010-2016 - Part I</span></div><div class="casAuthors">Bailey, Justin J.; Schirrmacher, Ralf; Farrell, Kristen; Bernard-Gauthier, Vadim</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">733-751</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Tropomyosin receptor kinases (TrkA/B/C) play crucial roles in the development and maintenance of the nervous system, and aberrant expression of Trk has been implicated in neurol. disorders as well as neural and non-neural neoplasms.  Patent activity encompassing Trk inhibitors has grown substantially over the last 6 years, recognized by a rise in the no. of pharmaceutical entrants to the field and the escalation of novel inhibitor chemotypes.  In Part I of this two part review, a biol. and structural overview of Trk is provided in the context of Trk as a therapeutic target for cancer and pain, followed by the report of recent patent literature claiming small mol. inhibitors of Trk family kinases or which describe inhibitors developed for other kinase targets but include noteworthy Trk inhibition/application.  The discussion of the patent literature continues in Part II of this review, which includes an in-depth view of the current clin. applications of Trk inhibitors.  Substantial synthetic efforts in Trk inhibitor development has propagated numerous and diverse inhibitor chemotypes, including TrkA-specific inhibitors.  While many novel Trk inhibitors remain the original progeny of Trk-specific development programs, kinase inhibitors initially developed for other kinases have also been successfully repositioned for Trk.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIHEog1RqjUrVg90H21EOLACvtfcHk0lgGZa0_Z-2ttw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvFyjtb0%253D&md5=e863493e3065568c43e39c0fe5a73751</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1080%2F13543776.2017.1297796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2017.1297796%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DJ.%26aulast%3DSchirrmacher%26aufirst%3DR.%26aulast%3DFarrell%26aufirst%3DK.%26aulast%3DBernard-Gauthier%26aufirst%3DV.%26atitle%3DTropomyosin%2520Receptor%2520Kinase%2520Inhibitors%253A%2520An%2520Updated%2520Patent%2520Review%2520for%25202010%25E2%2580%25932016%2520-%2520Part%2520I%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2017%26volume%3D27%26spage%3D733%26epage%3D751%26doi%3D10.1080%2F13543776.2017.1297796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirrmacher, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard-Gauthier, V.</span></span> <span> </span><span class="NLM_article-title">Tropomyosin Receptor Kinase Inhibitors: An Updated Patent Review for 2010–2016 - Part II</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">831</span>– <span class="NLM_lpage">849</span>, <span class="refDoi"> DOI: 10.1080/13543776.2017.1297797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1080%2F13543776.2017.1297797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=28270021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvFyjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=831-849&author=J.+Baileyauthor=R.+Schirrmacherauthor=K.+Farrellauthor=V.+Bernard-Gauthier&title=Tropomyosin+Receptor+Kinase+Inhibitors%3A+An+Updated+Patent+Review+for+2010%E2%80%932016+-+Part+II&doi=10.1080%2F13543776.2017.1297797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Tropomyosin receptor kinase inhibitors: an updated patent review for 2010-2016 - Part II</span></div><div class="casAuthors">Bailey, Justin J.; Schirrmacher, Ralf; Farrell, Kristen; Bernard-Gauthier, Vadim</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">831-849</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">TrkA/B/C receptor activation supports growth, survival, and differentiation of discrete neuronal populations during development, adult life, and ageing but also plays numerous roles in human disease onset and progression.  Trk-specific inhibitors have therapeutic applications in cancer and pain and thus constitute a growing area of interest in oncol. and neurol.  There has been substantial growth in the no. of structural classes of Trk inhibitors and the no. of industrial entrants to the Trk inhibitor field over the past six years.  In Part II of this two-part review, the discussion of recent patent literature covering Trk family inhibitors is continued from Part I and clin. research with Trk inhibitors is considered.  Trk has been molecularly targeted for over a decade resulting in the progressive evolution of structurally diversified Trk inhibitors arising from scaffold hopping and HTS efforts.  Correspondingly, there have been a growing no. of clin. investigations utilizing Trk inhibitors in recent years, with a particular focus on the treatment of NTRK-fusion pos. cancers and chronic pain.  The obsd. potential of Trk inhibitors to cause adverse CNS side effects however suggests the need for a more rigorous consideration of BBB permeation capabilities during drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG_4h-yrNplrVg90H21EOLACvtfcHk0lgGZa0_Z-2ttw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvFyjtbw%253D&md5=9c0a05bec29ecf9ccf53cc26dda78709</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1080%2F13543776.2017.1297797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2017.1297797%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DJ.%26aulast%3DSchirrmacher%26aufirst%3DR.%26aulast%3DFarrell%26aufirst%3DK.%26aulast%3DBernard-Gauthier%26aufirst%3DV.%26atitle%3DTropomyosin%2520Receptor%2520Kinase%2520Inhibitors%253A%2520An%2520Updated%2520Patent%2520Review%2520for%25202010%25E2%2580%25932016%2520-%2520Part%2520II%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2017%26volume%3D27%26spage%3D831%26epage%3D849%26doi%3D10.1080%2F13543776.2017.1297797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thress, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacIntyre, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitston, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zage, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweidler-McKay, P. A.</span></span> <span> </span><span class="NLM_article-title">Identification and Preclinical Characterization of AZ-23, a Novel, Selective, and Orally Bioavailable Inhibitor of the Trk Kinase Pathway</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1818</span>– <span class="NLM_lpage">1827</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-09-0036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1158%2F1535-7163.MCT-09-0036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=19509272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosVymsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=1818-1827&issue=7&author=K.+Thressauthor=T.+MacIntyreauthor=H.+Wangauthor=D.+Whitstonauthor=Z.-Y.+Liuauthor=E.+Hoffmannauthor=T.+Wangauthor=J.+L.+Brownauthor=K.+Websterauthor=C.+Omerauthor=P.+E.+Zageauthor=L.+Zengauthor=P.+A.+Zweidler-McKay&title=Identification+and+Preclinical+Characterization+of+AZ-23%2C+a+Novel%2C+Selective%2C+and+Orally+Bioavailable+Inhibitor+of+the+Trk+Kinase+Pathway&doi=10.1158%2F1535-7163.MCT-09-0036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and preclinical characterization of AZ-23, a novel, selective, and orally bioavailable inhibitor of the Trk kinase pathway</span></div><div class="casAuthors">Thress, Kenneth; MacIntyre, Terry; Wang, Haiyun; Whitston, Dave; Liu, Zhong-Ying; Hoffmann, Ethan; Wang, Tao; Brown, Jeffrey L.; Webster, Kevin; Omer, Charles; Zage, Peter E.; Zeng, Lizhi; Zweidler-McKay, Patrick A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1818-1827</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Tropomyosin-related kinases (TrkA, TrkB, and TrkC) are receptor tyrosine kinases that, along with their ligands, the neurotrophins, are involved in neuronal cell growth, development, and survival.  The Trk-neurotrophin pathway may also play a role in tumorigenesis through oncogenic fusions, mutations, and autocrine signaling, prompting the development of novel Trk inhibitors as agents for cancer therapy.  This report describes the identification of AZ-23, a novel, potent, and selective Trk kinase inhibitor.  In vitro studies with AZ-23 showed improved selectivity over previous compds. and inhibition of Trk kinase activity in cells at low nanomolar concns.  AZ-23 showed in vivo TrkA kinase inhibition and efficacy in mice following oral administration in a TrkA-driven allograft model and significant tumor growth inhibition in a Trk-expressing xenograft model of neuroblastoma.  AZ-23 represents a potent and selective Trk kinase inhibitor from a novel series with the potential for use as a treatment for cancer. [Mol Cancer Ther 2009;8(7):1818-27].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG2uvCSID5vbVg90H21EOLACvtfcHk0ljHKOd1PV2eAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosVymsro%253D&md5=8501f75147407ee64920fd4f5e733536</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-09-0036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-09-0036%26sid%3Dliteratum%253Aachs%26aulast%3DThress%26aufirst%3DK.%26aulast%3DMacIntyre%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWhitston%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DZ.-Y.%26aulast%3DHoffmann%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DBrown%26aufirst%3DJ.%2BL.%26aulast%3DWebster%26aufirst%3DK.%26aulast%3DOmer%26aufirst%3DC.%26aulast%3DZage%26aufirst%3DP.%2BE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZweidler-McKay%26aufirst%3DP.%2BA.%26atitle%3DIdentification%2520and%2520Preclinical%2520Characterization%2520of%2520AZ-23%252C%2520a%2520Novel%252C%2520Selective%252C%2520and%2520Orally%2520Bioavailable%2520Inhibitor%2520of%2520the%2520Trk%2520Kinase%2520Pathway%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2009%26volume%3D8%26issue%3D7%26spage%3D1818%26epage%3D1827%26doi%3D10.1158%2F1535-7163.MCT-09-0036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bakri, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahbeh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abu Sneina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al Khader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obeidat, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">AlAwwa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ki, C.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masri, A.</span></span> <span> </span><span class="NLM_article-title">Congenital Insensitivity to Pain and Anhydrosis due to a Rare Mutation and That Is Complicated by Inflammatory Bowel Disease and Amyloidosis: A Case Report</span>. <i>Clin. Case Reports</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">997</span>– <span class="NLM_lpage">1000</span>, <span class="refDoi"> DOI: 10.1002/ccr3.689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1002%2Fccr3.689" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=27761255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A280%3ADC%252BC2srktFKhsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2016&pages=997-1000&issue=10&author=F.+G.+Bakriauthor=A.+Wahbehauthor=A.+Abu+Sneinaauthor=A.+Al+Khaderauthor=F.+Obeidatauthor=I.+AlAwwaauthor=M.+Buniauthor=C.-S.+Kiauthor=A.+Masri&title=Congenital+Insensitivity+to+Pain+and+Anhydrosis+due+to+a+Rare+Mutation+and+That+Is+Complicated+by+Inflammatory+Bowel+Disease+and+Amyloidosis%3A+A+Case+Report&doi=10.1002%2Fccr3.689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Congenital insensitivity to pain and anhydrosis due to a rare mutation and that is complicated by inflammatory bowel disease and amyloidosis: a case report</span></div><div class="casAuthors">Bakri Faris G; Wahbeh Ayman; Abu Sneina Awni; AlAwwa Izzat; Al Khader Ali; Obeidat Fatima; Buni Maryam; Ki Chang-Seok; Masri Amira</div><div class="citationInfo"><span class="NLM_cas:title">Clinical case reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">997-1000</span>
        ISSN:<span class="NLM_cas:issn">2050-0904</span>.
    </div><div class="casAbstract">Patients with congenital insensitivity to pain and anhydrosis syndrome are at risk for renal amyloidosis and inflammatory bowel disease.  Physicians caring for such patients should be aware of these complications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdsZ2fFId2JBe4jJGKT5N0fW6udTcc2eb2yX-HGivjaLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srktFKhsQ%253D%253D&md5=513f89ccf382d11aa22326c933b633ad</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fccr3.689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fccr3.689%26sid%3Dliteratum%253Aachs%26aulast%3DBakri%26aufirst%3DF.%2BG.%26aulast%3DWahbeh%26aufirst%3DA.%26aulast%3DAbu%2BSneina%26aufirst%3DA.%26aulast%3DAl%2BKhader%26aufirst%3DA.%26aulast%3DObeidat%26aufirst%3DF.%26aulast%3DAlAwwa%26aufirst%3DI.%26aulast%3DBuni%26aufirst%3DM.%26aulast%3DKi%26aufirst%3DC.-S.%26aulast%3DMasri%26aufirst%3DA.%26atitle%3DCongenital%2520Insensitivity%2520to%2520Pain%2520and%2520Anhydrosis%2520due%2520to%2520a%2520Rare%2520Mutation%2520and%2520That%2520Is%2520Complicated%2520by%2520Inflammatory%2520Bowel%2520Disease%2520and%2520Amyloidosis%253A%2520A%2520Case%2520Report%26jtitle%3DClin.%2520Case%2520Reports%26date%3D2016%26volume%3D4%26issue%3D10%26spage%3D997%26epage%3D1000%26doi%3D10.1002%2Fccr3.689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nwosu, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapp, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, D. A.</span></span> <span> </span><span class="NLM_article-title">Blocking the Tropomyosin Receptor Kinase A (TrkA) Receptor Inhibits Pain Behaviour in Two Rat Models of Osteoarthritis</span>. <i>Ann. Rheum. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">1246</span>– <span class="NLM_lpage">1254</span>, <span class="refDoi"> DOI: 10.1136/annrheumdis-2014-207203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1136%2Fannrheumdis-2014-207203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=26286016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2sXivV2qtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2016&pages=1246-1254&author=L.+N.+Nwosuauthor=P.+I.+Mappauthor=V.+Chapmanauthor=D.+A.+Walsh&title=Blocking+the+Tropomyosin+Receptor+Kinase+A+%28TrkA%29+Receptor+Inhibits+Pain+Behaviour+in+Two+Rat+Models+of+Osteoarthritis&doi=10.1136%2Fannrheumdis-2014-207203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Blocking the tropomyosin receptor kinase A (TrkA) receptor inhibits pain behaviour in two rat models of osteoarthritis</span></div><div class="casAuthors">Nwosu, Lilian N.; Mapp, Paul I.; Chapman, Victoria; Walsh, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1246-1254</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Objectives: Tropomyosin receptor kinase A (TrkA) mediates nociceptor sensitization by nerve growth factor (NGF), but it is unknown whether selective TrkA inhibition will be an effective strategy for treating osteoarthritis (OA) pain.  We detd. the effects of a TrkA inhibitor (AR786) on pain behavior, synovitis and joint pathol. in two rat OA models.  Methods: Knee OA was induced in rats by intraarticular monosodium-iodoacetate (MIA) injection or meniscal transection (MNX) and compared with saline injected or sham-operated controls.  Pain behavior was assessed as wt.-bearing asymmetry and paw withdrawal threshold to punctate stimulation.  Oral doses (30 mg/kg) of AR786 or vehicle were administered twice daily in either preventive (day -1 to -27) or treatment (day 14-28) protocols.  Effect maintenance was evaluated for 2 wk after treatment discontinuation.  Alterations in knee structure (cartilage, subchondral bone and synovium) were examd. by macroscopic visualization of articular surfaces and histopathol.  Results: Preventive AR786 treatment inhibited pain behavior development and therapeutic treatment attenuated established pain behavior.  Wt.-bearing asymmetry increased 1 wk after treatment discontinuation, but remained less than in vehicle-treated arthritic rats, whereas paw withdrawal thresholds returned to levels of untreated rats within 5 days of treatment discontinuation.  AR786 treatment reduced MIA-induced synovitis and did not significantly affect osteochondral pathol. in either model.  Conclusions: Blocking NGF activity by inhibiting TrkA reduced pain behavior in two rat models of OA.  Analgesia was obsd. both using preventive and treatment protocols, and was sustained after treatment discontinuation.  Selective inhibitors of TrkA therefore hold potential for OA pain relief.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWAm0vTz5k3rVg90H21EOLACvtfcHk0ljHKOd1PV2eAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXivV2qtrc%253D&md5=a0159f007c97c40704266844aad93ca1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2014-207203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2014-207203%26sid%3Dliteratum%253Aachs%26aulast%3DNwosu%26aufirst%3DL.%2BN.%26aulast%3DMapp%26aufirst%3DP.%2BI.%26aulast%3DChapman%26aufirst%3DV.%26aulast%3DWalsh%26aufirst%3DD.%2BA.%26atitle%3DBlocking%2520the%2520Tropomyosin%2520Receptor%2520Kinase%2520A%2520%2528TrkA%2529%2520Receptor%2520Inhibits%2520Pain%2520Behaviour%2520in%2520Two%2520Rat%2520Models%2520of%2520Osteoarthritis%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2016%26volume%3D75%26spage%3D1246%26epage%3D1254%26doi%3D10.1136%2Fannrheumdis-2014-207203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, S. W.</span></span> <span> </span><span class="NLM_article-title">Allosteric Small Molecule Inhibitors of the NGF/TrkA Pathway, a New Approach to Treating Inflammatory Pain</span>. In  <i>International Association for the Study of Pain World Congress on Pain</i>; <span class="NLM_publisher-name">Array BioPharma</span>: <span class="NLM_publisher-loc">Boulder, CO</span>, August 29, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=S.+W.+Andrews&title=Allosteric+Small+Molecule+Inhibitors+of+the+NGF%2FTrkA+Pathway%2C+a+New+Approach+to+Treating+Inflammatory+Pain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DS.%2BW.%26atitle%3DAllosteric%2520Small%2520Molecule%2520Inhibitors%2520of%2520the%2520NGF%252FTrkA%2520Pathway%252C%2520a%2520New%2520Approach%2520to%2520Treating%2520Inflammatory%2520Pain%26jtitle%3DInternational%2520Association%2520for%2520the%2520Study%2520of%2520Pain%2520World%2520Congress%2520on%2520Pain%26pub%3DArray%2520BioPharma%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Block, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyne, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaus, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Josey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thress, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacIntyre, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span> <span> </span><span class="NLM_article-title">Identification of 4-Aminopyrazolylpyrimidines as Potent Inhibitors of Trk Kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">4672</span>– <span class="NLM_lpage">4684</span>, <span class="refDoi"> DOI: 10.1021/jm800343j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800343j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=4672-4684&issue=15&author=T.+Wangauthor=M.+L.+Lambauthor=D.+A.+Scottauthor=H.+Wangauthor=M.+H.+Blockauthor=P.+D.+Lyneauthor=J.+W.+Leeauthor=A.+M.+Daviesauthor=H.-J.+Zhangauthor=Y.+Zhuauthor=F.+Guauthor=Y.+Hanauthor=B.+Wangauthor=P.+J.+Mohrauthor=R.+J.+Kausauthor=J.+A.+Joseyauthor=E.+Hoffmannauthor=K.+Thressauthor=T.+MacIntyreauthor=H.+Wangauthor=C.+A.+Omerauthor=D.+Yu&title=Identification+of+4-Aminopyrazolylpyrimidines+as+Potent+Inhibitors+of+Trk+Kinases&doi=10.1021%2Fjm800343j"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm800343j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800343j%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DT.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DScott%26aufirst%3DD.%2BA.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DBlock%26aufirst%3DM.%2BH.%26aulast%3DLyne%26aufirst%3DP.%2BD.%26aulast%3DLee%26aufirst%3DJ.%2BW.%26aulast%3DDavies%26aufirst%3DA.%2BM.%26aulast%3DZhang%26aufirst%3DH.-J.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DF.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DMohr%26aufirst%3DP.%2BJ.%26aulast%3DKaus%26aufirst%3DR.%2BJ.%26aulast%3DJosey%26aufirst%3DJ.%2BA.%26aulast%3DHoffmann%26aufirst%3DE.%26aulast%3DThress%26aufirst%3DK.%26aulast%3DMacIntyre%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DOmer%26aufirst%3DC.%2BA.%26aulast%3DYu%26aufirst%3DD.%26atitle%3DIdentification%2520of%25204-Aminopyrazolylpyrimidines%2520as%2520Potent%2520Inhibitors%2520of%2520Trk%2520Kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26issue%3D15%26spage%3D4672%26epage%3D4684%26doi%3D10.1021%2Fjm800343j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stachel, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henze, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudd, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanda, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egbertson, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manley, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K. L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brnardic, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grobler, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanney, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leitl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, M.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krasowska-Zoladek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuck, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kane, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilodeau, M. T.</span></span> <span> </span><span class="NLM_article-title">Maximizing Diversity from a Kinase Screen: Identification of Novel and Selective Pan-Trk Inhibitors for Chronic Pain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">5800</span>– <span class="NLM_lpage">5816</span>, <span class="refDoi"> DOI: 10.1021/jm5006429</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006429" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=5800-5816&issue=13&author=S.+J.+Stachelauthor=J.+M.+Sandersauthor=D.+A.+Henzeauthor=M.+T.+Ruddauthor=H.-P.+Suauthor=Y.+Liauthor=K.+K.+Nandaauthor=M.+S.+Egbertsonauthor=P.+J.+Manleyauthor=K.+L.+G.+Jonesauthor=E.+J.+Brnardicauthor=A.+Greenauthor=J.+A.+Groblerauthor=B.+Hanneyauthor=M.+Leitlauthor=M.-T.+Laiauthor=V.+Munshiauthor=D.+Murphyauthor=K.+Rickertauthor=D.+Rileyauthor=A.+Krasowska-Zoladekauthor=C.+Daleyauthor=P.+Zuckauthor=S.+A.+Kaneauthor=M.+T.+Bilodeau&title=Maximizing+Diversity+from+a+Kinase+Screen%3A+Identification+of+Novel+and+Selective+Pan-Trk+Inhibitors+for+Chronic+Pain&doi=10.1021%2Fjm5006429"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm5006429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006429%26sid%3Dliteratum%253Aachs%26aulast%3DStachel%26aufirst%3DS.%2BJ.%26aulast%3DSanders%26aufirst%3DJ.%2BM.%26aulast%3DHenze%26aufirst%3DD.%2BA.%26aulast%3DRudd%26aufirst%3DM.%2BT.%26aulast%3DSu%26aufirst%3DH.-P.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DNanda%26aufirst%3DK.%2BK.%26aulast%3DEgbertson%26aufirst%3DM.%2BS.%26aulast%3DManley%26aufirst%3DP.%2BJ.%26aulast%3DJones%26aufirst%3DK.%2BL.%2BG.%26aulast%3DBrnardic%26aufirst%3DE.%2BJ.%26aulast%3DGreen%26aufirst%3DA.%26aulast%3DGrobler%26aufirst%3DJ.%2BA.%26aulast%3DHanney%26aufirst%3DB.%26aulast%3DLeitl%26aufirst%3DM.%26aulast%3DLai%26aufirst%3DM.-T.%26aulast%3DMunshi%26aufirst%3DV.%26aulast%3DMurphy%26aufirst%3DD.%26aulast%3DRickert%26aufirst%3DK.%26aulast%3DRiley%26aufirst%3DD.%26aulast%3DKrasowska-Zoladek%26aufirst%3DA.%26aulast%3DDaley%26aufirst%3DC.%26aulast%3DZuck%26aufirst%3DP.%26aulast%3DKane%26aufirst%3DS.%2BA.%26aulast%3DBilodeau%26aufirst%3DM.%2BT.%26atitle%3DMaximizing%2520Diversity%2520from%2520a%2520Kinase%2520Screen%253A%2520Identification%2520of%2520Novel%2520and%2520Selective%2520Pan-Trk%2520Inhibitors%2520for%2520Chronic%2520Pain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D13%26spage%3D5800%26epage%3D5816%26doi%3D10.1021%2Fjm5006429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burris, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brose, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farago, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doebele, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruickshank, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J. A.</span></span> <span> </span><span class="NLM_article-title">A First-in-Human Study of LOXO-101, a Highly Selective Inhibitor of the Tropomyosin Receptor Kinase (TRK) Family</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">TPS2624</span>, <span class="refDoi"> DOI: 10.1200/jco.2015.33.15_suppl.tps2624</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1200%2Fjco.2015.33.15_suppl.tps2624" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=TPS2624&author=H.+A.+Burrisauthor=M.+S.+Broseauthor=A.+T.+Shawauthor=T.+M.+Bauerauthor=A.+F.+Faragoauthor=R.+C.+Doebeleauthor=S.+Smithauthor=M.+Fernandesauthor=S.+Cruickshankauthor=J.+A.+Low&title=A+First-in-Human+Study+of+LOXO-101%2C+a+Highly+Selective+Inhibitor+of+the+Tropomyosin+Receptor+Kinase+%28TRK%29+Family&doi=10.1200%2Fjco.2015.33.15_suppl.tps2624"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1200%2Fjco.2015.33.15_suppl.tps2624&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252Fjco.2015.33.15_suppl.tps2624%26sid%3Dliteratum%253Aachs%26aulast%3DBurris%26aufirst%3DH.%2BA.%26aulast%3DBrose%26aufirst%3DM.%2BS.%26aulast%3DShaw%26aufirst%3DA.%2BT.%26aulast%3DBauer%26aufirst%3DT.%2BM.%26aulast%3DFarago%26aufirst%3DA.%2BF.%26aulast%3DDoebele%26aufirst%3DR.%2BC.%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DFernandes%26aufirst%3DM.%26aulast%3DCruickshank%26aufirst%3DS.%26aulast%3DLow%26aufirst%3DJ.%2BA.%26atitle%3DA%2520First-in-Human%2520Study%2520of%2520LOXO-101%252C%2520a%2520Highly%2520Selective%2520Inhibitor%2520of%2520the%2520Tropomyosin%2520Receptor%2520Kinase%2520%2528TRK%2529%2520Family%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2015%26volume%3D33%26spage%3DTPS2624%26doi%3D10.1200%2Fjco.2015.33.15_suppl.tps2624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skerratt, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagal, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilsland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bungay, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morao, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nedderman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryckmans, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stupple, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldron, G.</span></span> <span> </span><span class="NLM_article-title">The Discovery of a Potent, Selective, and Peripherally Restricted Pan-Trk Inhibitor (PF-06273340) for the Treatment of Pain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">10084</span>– <span class="NLM_lpage">10099</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00850</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00850" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWjsbfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10084-10099&issue=22&author=S.+E.+Skerrattauthor=M.+Andrewsauthor=S.+K.+Bagalauthor=J.+Bilslandauthor=D.+Brownauthor=P.+J.+Bungayauthor=S.+Coleauthor=K.+R.+Gibsonauthor=R.+Jonesauthor=I.+Moraoauthor=A.+Neddermanauthor=K.+Omotoauthor=C.+Robinsonauthor=T.+Ryckmansauthor=K.+Skinnerauthor=P.+Stuppleauthor=G.+Waldron&title=The+Discovery+of+a+Potent%2C+Selective%2C+and+Peripherally+Restricted+Pan-Trk+Inhibitor+%28PF-06273340%29+for+the+Treatment+of+Pain&doi=10.1021%2Facs.jmedchem.6b00850"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The Discovery of a Potent, Selective, and Peripherally Restricted Pan-Trk Inhibitor (PF-06273340) for the Treatment of Pain</span></div><div class="casAuthors">Skerratt, Sarah E.; Andrews, Mark; Bagal, Sharan K.; Bilsland, James; Brown, David; Bungay, Peter J.; Cole, Susan; Gibson, Karl R.; Jones, Russell; Morao, Inaki; Nedderman, Angus; Omoto, Kiyoyuki; Robinson, Colin; Ryckmans, Thomas; Skinner, Kimberly; Stupple, Paul; Waldron, Gareth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10084-10099</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The neurotrophin family of growth factors, comprised of nerve growth factor (NGF), brain derived neurotrophic factor (BDNF), neurotrophin 3 (NT3) and neurotrophin 4 (NT4), is implicated in the physiol. of chronic pain.  Given the clin. efficacy of anti-NGF monoclonal antibody (mAb) therapies, there is significant interest in the development of small mol. modulators of neurotrophin activity.  Neurotrophins signal through the tropomyosin related kinase (Trk) family of tyrosine kinase receptors, hence Trk kinase inhibition represents a potentially "druggable" point of intervention.  To deliver the safety profile required for chronic, non-life threatening pain indications, highly kinase-selective Trk inhibitors with minimal brain availability are sought.  Herein the authors describe how the use of SBDD, 2D QSAR models and Matched Mol. Pair data in compd. design enabled the delivery of the highly potent, kinase-selective and peripherally restricted clin. candidate PF-06273340.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo67CFmGAkmRrVg90H21EOLACvtfcHk0lgNNLtp9zE7UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWjsbfL&md5=f8c904a92c1716062c217fefe2374ce6</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00850&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00850%26sid%3Dliteratum%253Aachs%26aulast%3DSkerratt%26aufirst%3DS.%2BE.%26aulast%3DAndrews%26aufirst%3DM.%26aulast%3DBagal%26aufirst%3DS.%2BK.%26aulast%3DBilsland%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DD.%26aulast%3DBungay%26aufirst%3DP.%2BJ.%26aulast%3DCole%26aufirst%3DS.%26aulast%3DGibson%26aufirst%3DK.%2BR.%26aulast%3DJones%26aufirst%3DR.%26aulast%3DMorao%26aufirst%3DI.%26aulast%3DNedderman%26aufirst%3DA.%26aulast%3DOmoto%26aufirst%3DK.%26aulast%3DRobinson%26aufirst%3DC.%26aulast%3DRyckmans%26aufirst%3DT.%26aulast%3DSkinner%26aufirst%3DK.%26aulast%3DStupple%26aufirst%3DP.%26aulast%3DWaldron%26aufirst%3DG.%26atitle%3DThe%2520Discovery%2520of%2520a%2520Potent%252C%2520Selective%252C%2520and%2520Peripherally%2520Restricted%2520Pan-Trk%2520Inhibitor%2520%2528PF-06273340%2529%2520for%2520the%2520Treatment%2520of%2520Pain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D22%26spage%3D10084%26epage%3D10099%26doi%3D10.1021%2Facs.jmedchem.6b00850" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bissantz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">A Medicinal Chemist’s Guide to Molecular Interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">5061</span>– <span class="NLM_lpage">5084</span>, <span class="refDoi"> DOI: 10.1021/jm100112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5061-5084&issue=14&author=C.+Bissantzauthor=B.+Kuhnauthor=M.+Stahl&title=A+Medicinal+Chemist%E2%80%99s+Guide+to+Molecular+Interactions&doi=10.1021%2Fjm100112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A Medicinal Chemist's Guide to Molecular Interactions</span></div><div class="casAuthors">Bissantz, Caterina; Kuhn, Bernd; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5061-5084</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraV8HPbwsihbVg90H21EOLACvtfcHk0lgNNLtp9zE7UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D&md5=f39cf8700267cdc311c73de4d433bab0</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm100112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100112j%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520Medicinal%2520Chemist%25E2%2580%2599s%2520Guide%2520to%2520Molecular%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D14%26spage%3D5061%26epage%3D5084%26doi%3D10.1021%2Fjm100112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Besnard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setola, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abecassis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguiz, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norval, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, L. a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeons, F. R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stojanovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidah, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constam, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickerton, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetsel, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Automated Design of Ligands to Polypharmacological Profiles</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>492</i></span> (<span class="NLM_issue">7428</span>),  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1038/nature11691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&issue=7428&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.-P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+a+Websterauthor=F.+R.+C.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+Design+of+Ligands+to+Polypharmacological+Profiles&doi=10.1038%2Fnature11691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0lgNNLtp9zE7UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2Ba%26aulast%3DSimeons%26aufirst%3DF.%2BR.%2BC.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520Design%2520of%2520Ligands%2520to%2520Polypharmacological%2520Profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26issue%3D7428%26spage%3D215%26epage%3D220%26doi%3D10.1038%2Fnature11691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lounkine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keiser, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitebread, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikhailov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, L.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Prediction and Testing of Drug Activity on Side-Effect Targets</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>486</i></span> (<span class="NLM_issue">7403</span>),  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1038/nature11159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1038%2Fnature11159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=22722194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyrtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=486&publication_year=2012&pages=361-367&issue=7403&author=E.+Lounkineauthor=M.+J.+Keiserauthor=S.+Whitebreadauthor=D.+Mikhailovauthor=J.+Hamonauthor=J.+L.+Jenkinsauthor=P.+Lavanauthor=E.+Weberauthor=A.+K.+Doakauthor=S.+C%C3%B4t%C3%A9author=B.+K.+Shoichetauthor=L.+Urban&title=Large-Scale+Prediction+and+Testing+of+Drug+Activity+on+Side-Effect+Targets&doi=10.1038%2Fnature11159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale prediction and testing of drug activity on side-effect targets</span></div><div class="casAuthors">Lounkine, Eugen; Keiser, Michael J.; Whitebread, Steven; Mikhailov, Dmitri; Hamon, Jacques; Jenkins, Jeremy L.; Lavan, Paul; Weber, Eckhard; Doak, Allison K.; Cote, Serge; Shoichet, Brian K.; Urban, Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">486</span>
        (<span class="NLM_cas:issue">7403</span>),
    <span class="NLM_cas:pages">361-367</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Discovering the unintended off-targets' that predict adverse drug reactions is daunting by empirical methods alone.  Drugs can act on several protein targets, some of which can be unrelated by conventional mol. metrics, and hundreds of proteins have been implicated in side effects.  Here we use a computational strategy to predict the activity of 656 marketed drugs on 73 unintended side-effect' targets.  Approx. half of the predictions were confirmed, either from proprietary databases unknown to the method or by new exptl. assays.  Affinities for these new off-targets ranged from 1 nM to 30 μM.  To explore relevance, we developed an assocn. metric to prioritize those new off-targets that explained side effects better than any known target of a given drug, creating a drug-target-adverse drug reaction network.  Among these new assocns. was the prediction that the abdominal pain side effect of the synthetic estrogen chlorotrianisene was mediated through its newly discovered inhibition of the enzyme cyclooxygenase-1.  The clin. relevance of this inhibition was borne out in whole human blood platelet aggregation assays.  This approach may have wide application to de-risking toxicol. liabilities in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5sVjfn4mgLLVg90H21EOLACvtfcHk0ljPp2_ztynxnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyrtb0%253D&md5=9c133b839d9527818a27a5834e4dfe11</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnature11159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11159%26sid%3Dliteratum%253Aachs%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DWhitebread%26aufirst%3DS.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DHamon%26aufirst%3DJ.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DLavan%26aufirst%3DP.%26aulast%3DWeber%26aufirst%3DE.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DS.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DUrban%26aufirst%3DL.%26atitle%3DLarge-Scale%2520Prediction%2520and%2520Testing%2520of%2520Drug%2520Activity%2520on%2520Side-Effect%2520Targets%26jtitle%3DNature%26date%3D2012%26volume%3D486%26issue%3D7403%26spage%3D361%26epage%3D367%26doi%3D10.1038%2Fnature11159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">A Computational Method for Unveiling the Target Promiscuity of Pharmacologically Active Compounds</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">11520</span>– <span class="NLM_lpage">11524</span>, <span class="refDoi"> DOI: 10.1002/anie.201706376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1002%2Fanie.201706376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht12hsrvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=11520-11524&issue=38&author=P.+Schneiderauthor=G.+Schneider&title=A+Computational+Method+for+Unveiling+the+Target+Promiscuity+of+Pharmacologically+Active+Compounds&doi=10.1002%2Fanie.201706376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A Computational Method for Unveiling the Target Promiscuity of Pharmacologically Active Compounds</span></div><div class="casAuthors">Schneider, Petra; Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11520-11524</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Drug discovery is governed by the desire to find ligands with defined modes of action.  It has been realized that even designated selective drugs may have more macromol. targets than is commonly thought.  Consequently, it will be mandatory to consider multitarget activity for the design of future medicines.  Computational models assist medicinal chemists in this effort by helping to eliminate unsuitable lead structures and spot undesired drug effects early in the discovery process.  Here, we present a straightforward computational method to find previously unknown targets of pharmacol. active compds.  Validation expts. revealed hitherto unknown targets of the natural product resveratrol and the nonsteroidal anti-inflammatory drug celecoxib.  The obtained results advocate machine learning for polypharmacol.-based mol. design, drug re-purposing, and the "de-orphaning" of phenotypic drug effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwz7epuJpraLVg90H21EOLACvtfcHk0ljPp2_ztynxnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht12hsrvO&md5=7dc990ab4ede2187f69b8edb41826633</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.201706376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201706376%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DP.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DA%2520Computational%2520Method%2520for%2520Unveiling%2520the%2520Target%2520Promiscuity%2520of%2520Pharmacologically%2520Active%2520Compounds%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26issue%3D38%26spage%3D11520%26epage%3D11524%26doi%3D10.1002%2Fanie.201706376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarolimek, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sridhar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitebread, S.</span></span> <span> </span><span class="NLM_article-title">Reducing Safety-Related Drug Attrition: The Use of in Vitro Pharmacological Profiling</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">909</span>– <span class="NLM_lpage">922</span>, <span class="refDoi"> DOI: 10.1038/nrd3845</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1038%2Fnrd3845" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=23197038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslCnur%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=909-922&issue=12&author=J.+Bowesauthor=A.+J.+Brownauthor=J.+Hamonauthor=W.+Jarolimekauthor=A.+Sridharauthor=G.+Waldronauthor=S.+Whitebread&title=Reducing+Safety-Related+Drug+Attrition%3A+The+Use+of+in+Vitro+Pharmacological+Profiling&doi=10.1038%2Fnrd3845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Reducing safety-related drug attrition: the use of in vitro pharmacological profiling</span></div><div class="casAuthors">Bowes, Joanne; Brown, Andrew J.; Hamon, Jacques; Jarolimek, Wolfgang; Sridhar, Arun; Waldron, Gareth; Whitebread, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">909-922</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In vitro pharmacol. profiling is increasingly being used earlier in the drug discovery process to identify undesirable off-target activity profiles that could hinder or halt the development of candidate drugs or even lead to market withdrawal if discovered after a drug is approved.  Here, for the first time, the rationale, strategies and methodologies for in vitro pharmacol. profiling at four major pharmaceutical companies (AstraZeneca, GlaxoSmithKline, Novartis and Pfizer) are presented and illustrated with examples of their impact on the drug discovery process.  We hope that this will enable other companies and academic institutions to benefit from this knowledge and consider joining us in our collaborative knowledge sharing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomVp3wnTONWrVg90H21EOLACvtfcHk0ljPp2_ztynxnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslCnur%252FJ&md5=7559001563ef3febd610535b8f096a45</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrd3845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3845%26sid%3Dliteratum%253Aachs%26aulast%3DBowes%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DA.%2BJ.%26aulast%3DHamon%26aufirst%3DJ.%26aulast%3DJarolimek%26aufirst%3DW.%26aulast%3DSridhar%26aufirst%3DA.%26aulast%3DWaldron%26aufirst%3DG.%26aulast%3DWhitebread%26aufirst%3DS.%26atitle%3DReducing%2520Safety-Related%2520Drug%2520Attrition%253A%2520The%2520Use%2520of%2520in%2520Vitro%2520Pharmacological%2520Profiling%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26issue%3D12%26spage%3D909%26epage%3D922%26doi%3D10.1038%2Fnrd3845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkamer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulle, S.</span></span> <span> </span><span class="NLM_article-title">KinMap: A Web-Based Tool for Interactive Navigation through Human Kinome Data</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">16</span>, <span class="refDoi"> DOI: 10.1186/s12859-016-1433-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1186%2Fs12859-016-1433-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=28056780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjsFCjtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=16&author=S.+Eidauthor=S.+Turkauthor=A.+Volkamerauthor=F.+Rippmannauthor=S.+Fulle&title=KinMap%3A+A+Web-Based+Tool+for+Interactive+Navigation+through+Human+Kinome+Data&doi=10.1186%2Fs12859-016-1433-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">KinMap: a web-based tool for interactive navigation through human kinome data</span></div><div class="casAuthors">Eid, Sameh; Turk, Samo; Volkamer, Andrea; Rippmann, Friedrich; Fulle, Simone</div><div class="citationInfo"><span class="NLM_cas:title">BMC Bioinformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">16/1-16/6</span>CODEN:
                <span class="NLM_cas:coden">BBMIC4</span>;
        ISSN:<span class="NLM_cas:issn">1471-2105</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Annotations of the phylogenetic tree of the human kinome is an intuitive way to visualize compd. profiling data, structural features of kinases or functional relationships within this important class of proteins.  The increasing vol. and complexity of kinase-related data underlines the need for a tool that enables complex queries pertaining to kinase disease involvement and potential therapeutic uses of kinase inhibitors.  Results: Here, we present KinMap, a user-friendly online tool that facilitates the interactive navigation through kinase knowledge by linking biochem., structural, and disease assocn. data to the human kinome tree.  To this end, preprocessed data from freely-available sources, such as ChEMBL, the Protein Data Bank, and the Center for Therapeutic Target Validation platform are integrated into KinMap and can easily be complemented by proprietary data.  The value of KinMap will be exemplarily demonstrated for uncovering new therapeutic indications of known kinase inhibitors and for prioritizing kinases for drug development efforts.  Conclusions: KinMap represents a new generation of kinome tree viewers which facilitates interactive exploration of the human kinome.  KinMap enables generation of high-quality annotated images of the human kinome tree as well as exchange of kinome-related data in scientific communications.  Furthermore, KinMap supports multiple input and output formats and recognizes alternative kinase names and links them to a unified naming scheme, which makes it a useful tool across different disciplines and applications.  A web-service of KinMap is freely available at http://www.kinhub.org/kinmap/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1M2aa8fuKALVg90H21EOLACvtfcHk0lhbhwTQN0eWzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjsFCjtLc%253D&md5=29737c7cda8d9caf75018e0d29e9a459</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1186%2Fs12859-016-1433-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-016-1433-7%26sid%3Dliteratum%253Aachs%26aulast%3DEid%26aufirst%3DS.%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DFulle%26aufirst%3DS.%26atitle%3DKinMap%253A%2520A%2520Web-Based%2520Tool%2520for%2520Interactive%2520Navigation%2520through%2520Human%2520Kinome%2520Data%26jtitle%3DBMC%2520Bioinf.%26date%3D2017%26volume%3D18%26spage%3D16%26doi%3D10.1186%2Fs12859-016-1433-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindstrom, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanner, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belew, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: Automated Docking with Selective Receptor Flexibility</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">2785</span>– <span class="NLM_lpage">2791</span>, <span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&issue=16&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+Automated+Docking+with+Selective+Receptor+Flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0lhbhwTQN0eWzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520Automated%2520Docking%2520with%2520Selective%2520Receptor%2520Flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26issue%3D16%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wade, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodford, P. J.</span></span> <span> </span><span class="NLM_article-title">Identifying Targets for Bioreductive Agents: Using GRID to Predict Selective Binding Regions of Proteins</span>. <i>J. Mol. Graphics</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">108</span>, <span class="refDoi"> DOI: 10.1016/S0263-7855(89)80013-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1016%2FS0263-7855%2889%2980013-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=2488262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADyaL1MXlvFSqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1989&pages=103-108&issue=2&author=C.+A.+Reynoldsauthor=R.+C.+Wadeauthor=P.+J.+Goodford&title=Identifying+Targets+for+Bioreductive+Agents%3A+Using+GRID+to+Predict+Selective+Binding+Regions+of+Proteins&doi=10.1016%2FS0263-7855%2889%2980013-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying targets for bioreductive agents:  using GRID to predict selective binding regions of proteins</span></div><div class="casAuthors">Reynolds, Christopher A.; Wade, Rebecca C.; Goodford, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">103-8, 100</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    </div><div class="casAbstract">A computational procedure based on the computer program, GRID, is described for investigating potential binding sites of a target macromol. for their ability to bind both a reduced probe mol. and an oxidized probe mol.  The interaction energies are obtained using a mol. mechanics method and can be displayed as 3-dimensional energy contours, indicating regions of the target mol. that may have favorable interactions with the probe mol.  Differences in the interaction energies of the oxidized and reduced probe with the target can also be plotted as contours, indicating regions that are selective for the reduced probe.  These selectivity contours can be used to show whether the macromol. is a potential target for bioreductive agents.  The method has been applied to the chicken liver dihydrofolate reductase enzyme and has indicated new binding regions that may be suitable binding sites for bioreductive agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoVQeHx4QT47Vg90H21EOLACvtfcHk0lhbhwTQN0eWzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXlvFSqs7k%253D&md5=0d536c811584e37b3a40105e08018cc5</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0263-7855%2889%2980013-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0263-7855%252889%252980013-X%26sid%3Dliteratum%253Aachs%26aulast%3DReynolds%26aufirst%3DC.%2BA.%26aulast%3DWade%26aufirst%3DR.%2BC.%26aulast%3DGoodford%26aufirst%3DP.%2BJ.%26atitle%3DIdentifying%2520Targets%2520for%2520Bioreductive%2520Agents%253A%2520Using%2520GRID%2520to%2520Predict%2520Selective%2520Binding%2520Regions%2520of%2520Proteins%26jtitle%3DJ.%2520Mol.%2520Graphics%26date%3D1989%26volume%3D7%26issue%3D2%26spage%3D103%26epage%3D108%26doi%3D10.1016%2FS0263-7855%2889%2980013-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegscheid-Gerlach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaliani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span> <span> </span><span class="NLM_article-title">On the Art of Compiling and Using “Drug-Like” Chemical Fragment Spaces</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1503</span>– <span class="NLM_lpage">1507</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200800178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1002%2Fcmdc.200800178" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1503-1507&issue=10&author=J.+Degenauthor=C.+Wegscheid-Gerlachauthor=A.+Zalianiauthor=M.+Rarey&title=On+the+Art+of+Compiling+and+Using+%E2%80%9CDrug-Like%E2%80%9D+Chemical+Fragment+Spaces&doi=10.1002%2Fcmdc.200800178"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200800178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200800178%26sid%3Dliteratum%253Aachs%26aulast%3DDegen%26aufirst%3DJ.%26aulast%3DWegscheid-Gerlach%26aufirst%3DC.%26aulast%3DZaliani%26aufirst%3DA.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DOn%2520the%2520Art%2520of%2520Compiling%2520and%2520Using%2520%25E2%2580%259CDrug-Like%25E2%2580%259D%2520Chemical%2520Fragment%2520Spaces%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26issue%3D10%26spage%3D1503%26epage%3D1507%26doi%3D10.1002%2Fcmdc.200800178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, M.</span></span> <span> </span><span class="NLM_article-title">Extended-Connectivity Fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">742</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&issue=5&author=D.+Rogersauthor=M.+Hahn&title=Extended-Connectivity+Fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0ljTZbReDLEYTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-Connectivity%2520Fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D5%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span> <i>Schrödinger Release 2017-1: Maestro</i>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schr%C3%B6dinger+Release+2017-1%3A+Maestro%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%2520Release%25202017-1%253A%2520Maestro%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buitinck, L.</span>; <span class="NLM_string-name">Louppe, G.</span>; <span class="NLM_string-name">Blondel, M.</span>; <span class="NLM_string-name">Pedregosa, F.</span>; <span class="NLM_string-name">Mueller, A.</span>; <span class="NLM_string-name">Grisel, O.</span>; <span class="NLM_string-name">Niculae, V.</span>; <span class="NLM_string-name">Prettenhofer, P.</span>; <span class="NLM_string-name">Gramfort, A.</span>; <span class="NLM_string-name">Grobler, J.</span>; <span class="NLM_string-name">Layton, R.</span>; <span class="NLM_string-name">VanderPlas, J.</span>; <span class="NLM_string-name">Joly, A.</span>; <span class="NLM_string-name">Holt, B.</span>; <span class="NLM_string-name">Varoquaux, G.</span></span> <span> </span><span class="NLM_article-title">API Design for Machine Learning Software: Experiences from the Scikit-Learn Project</span>. In  <i>ECML/PKDD Workshop: Languages for Data Mining and Machine Learning, Prague, September 23, 2013</i>; <span class="NLM_publisher-name">Liege Universite</span>: <span class="NLM_publisher-loc">Liege, Belgium</span>, <span class="NLM_year">2013</span>; pp  <span class="NLM_fpage">108</span>– <span class="NLM_lpage">122</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=108-122&author=L.+Buitinck&author=G.+Louppe&author=M.+Blondel&author=F.+Pedregosa&author=A.+Mueller&author=O.+Grisel&author=V.+Niculae&author=P.+Prettenhofer&author=A.+Gramfort&author=J.+Grobler&author=R.+Layton&author=J.+VanderPlas&author=A.+Joly&author=B.+Holt&author=G.+Varoquaux&title=API+Design+for+Machine+Learning+Software%3A+Experiences+from+the+Scikit-Learn+Project"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DBuitinck%26aufirst%3DL.%26atitle%3DAPI%2520Design%2520for%2520Machine%2520Learning%2520Software%253A%2520Experiences%2520from%2520the%2520Scikit-Learn%2520Project%26jtitle%3DECML%252FPKDD%2520Workshop%253A%2520Languages%2520for%2520Data%2520Mining%2520and%2520Machine%2520Learning%252C%2520Prague%252C%2520September%252023%252C%25202013%26pub%3DLiege%2520Universite%26date%3D2013%26spage%3D108%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suenaga, M.</span></span> <span> </span><span class="NLM_article-title">Development of GUI for GAMESS/FMO Calculation</span>. <i>J. Comput. Chem., Jpn.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.2477/jccj.H1920</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.2477%2Fjccj.H1920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1ejsb3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=33-54&issue=1&author=M.+Suenaga&title=Development+of+GUI+for+GAMESS%2FFMO+Calculation&doi=10.2477%2Fjccj.H1920"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Development of GUI for GAMESS/FMO calculation</span></div><div class="casAuthors">Suenaga, Masahiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer Chemistry, Japan</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-54</span>CODEN:
                <span class="NLM_cas:coden">JCCJAG</span>;
        ISSN:<span class="NLM_cas:issn">1347-1767</span>.
    
            (<span class="NLM_cas:orgname">Nippon Konputa Kagakkai</span>)
        </div><div class="casAbstract">A GUI (Graphic User Interface) for GAMESS/FMO (Fragment MO method) calcn. was developed and implemented on Facio, the pre/post-processor for computational chem.  By using this GUI, one can easily make FMO fragments for a given protein or nucleic acid and generate FMO input files.  Besides automatic fragmentation, one can manually define addnl. FMO fractioning points to non-peptide moiety of conjugated protein.  For manual definition, a novel local structure viewer was also developed.  The desired local structure is defined by an atom no. or by a fragment no.  Using this local structure viewer, one can check the structural validity of each FMO fragment and perform manual definition of addnl. fractioning points on the fragment.  Manually defined fractioning points can be saved as "fragment definition file", which can be loaded later to set the predefined fractioning points.  For the optimization of hydrogen atoms automatically added to the PDB file of a protein or nucleic acid which lacks hydrogen atoms, a fragment optimization function was implemented for the GUI.  While a mol. dynamics method or mol. mechanics method is usually used for this purpose, the new optimization method utilizes a MO method, such as PM3 or STO-3G of GAMESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomdWWrcqKIU7Vg90H21EOLACvtfcHk0ljTZbReDLEYTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1ejsb3J&md5=ad5c3e0ff6c1c15cc11dae87292268f7</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.2477%2Fjccj.H1920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2477%252Fjccj.H1920%26sid%3Dliteratum%253Aachs%26aulast%3DSuenaga%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520GUI%2520for%2520GAMESS%252FFMO%2520Calculation%26jtitle%3DJ.%2520Comput.%2520Chem.%252C%2520Jpn.%26date%3D2008%26volume%3D7%26issue%3D1%26spage%3D33%26epage%3D54%26doi%3D10.2477%2Fjccj.H1920" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&issue=7&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lj-Q_ovWty3Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D7%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggs, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azimioara, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lélias, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velasco, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span> <span> </span><span class="NLM_article-title">A Small Molecule-Kinase Interaction Map for Clinical Kinase Inhibitors</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&issue=3&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.-M.+L%C3%A9liasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+Small+Molecule-Kinase+Interaction+Map+for+Clinical+Kinase+Inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lj-Q_ovWty3Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DL%25C3%25A9lias%26aufirst%3DJ.-M.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520Small%2520Molecule-Kinase%2520Interaction%2520Map%2520for%2520Clinical%2520Kinase%2520Inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26issue%3D3%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 7 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Zhouling Xie, Xiaoxiao Yang, Yajun Duan, Jihong Han, <span class="NLM_string-name hlFld-ContribAuthor">Chenzhong Liao</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Kinase Inhibitors for the Treatment of Nononcologic Diseases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (3)
                                     , 1283-1345. <a href="https://doi.org/10.1021/acs.jmedchem.0c01511" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01511</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01511%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall-Molecule%252BKinase%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BNononcologic%252BDiseases%26aulast%3DXie%26aufirst%3DZhouling%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D29082020%26date%3D22012021%26volume%3D64%26issue%3D3%26spage%3D1283%26epage%3D1345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Gaochan Wu, Tong Zhao, Dongwei Kang, Jian Zhang, Yuning Song, Vigneshwaran Namasivayam, Jacob Kongsted, Christophe Pannecouque, Erik De Clercq, Vasanthanathan Poongavanam, Xinyong Liu, <span class="NLM_string-name hlFld-ContribAuthor">Peng Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Overview of Recent Strategic Advances in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9375-9414. <a href="https://doi.org/10.1021/acs.jmedchem.9b00359" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00359%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOverview%252Bof%252BRecent%252BStrategic%252BAdvances%252Bin%252BMedicinal%252BChemistry%26aulast%3DWu%26aufirst%3DGaochan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26022019%26date%3D28052019%26date%3D03052019%26volume%3D62%26issue%3D21%26spage%3D9375%26epage%3D9414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Baowen Qi, Ling Zhong, Jun He, Hongjia Zhang, Fengqiong Li, Ting Wang, Jing Zou, Yao-Xin Lin, Chengchen Zhang, Xiaoqiang Guo, Rui Li, <span class="NLM_string-name hlFld-ContribAuthor">Jianyou Shi</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Inhibitors of Aurora/PLK Targets as Anticancer Agents. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (17)
                                     , 7697-7707. <a href="https://doi.org/10.1021/acs.jmedchem.9b00353" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00353</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00353%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BInhibitors%252Bof%252BAurora%25252FPLK%252BTargets%252Bas%252BAnticancer%252BAgents%26aulast%3DQi%26aufirst%3DBaowen%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26022019%26date%3D21082019%26date%3D05082019%26volume%3D62%26issue%3D17%26spage%3D7697%26epage%3D7707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francis  Giraud</span>, <span class="hlFld-ContribAuthor ">Elisabeth  Pereira</span>, <span class="hlFld-ContribAuthor ">Fabrice  Anizon</span>, <span class="hlFld-ContribAuthor ">Pascale  Moreau</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in Pain Management: Relevant Protein Kinases and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (9)
                                     , 2696. <a href="https://doi.org/10.3390/molecules26092696" title="DOI URL">https://doi.org/10.3390/molecules26092696</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26092696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26092696%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DRecent%252BAdvances%252Bin%252BPain%252BManagement%25253A%252BRelevant%252BProtein%252BKinases%252Band%252BTheir%252BInhibitors%26aulast%3DGiraud%26aufirst%3DFrancis%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D9%26spage%3D2696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sravani  Sana</span>, <span class="hlFld-ContribAuthor ">Velma Ganga  Reddy</span>, <span class="hlFld-ContribAuthor ">T.  Srinivasa Reddy</span>, <span class="hlFld-ContribAuthor ">Ramya  Tokala</span>, <span class="hlFld-ContribAuthor ">Rahul  Kumar</span>, <span class="hlFld-ContribAuthor ">Suresh K.  Bhargava</span>, <span class="hlFld-ContribAuthor ">Nagula  Shankaraiah</span>. </span><span class="cited-content_cbyCitation_article-title">Cinnamide derived pyrimidine-benzimidazole hybrids as tubulin inhibitors: Synthesis, in silico and cell growth inhibition studies. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>110 </em>, 104765. <a href="https://doi.org/10.1016/j.bioorg.2021.104765" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.104765</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.104765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.104765%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DCinnamide%252Bderived%252Bpyrimidine-benzimidazole%252Bhybrids%252Bas%252Btubulin%252Binhibitors%25253A%252BSynthesis%25252C%252Bin%252Bsilico%252Band%252Bcell%252Bgrowth%252Binhibition%252Bstudies%26aulast%3DSana%26aufirst%3DSravani%26date%3D2021%26volume%3D110%26spage%3D104765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Justin J.  Bailey</span>, <span class="hlFld-ContribAuthor ">Carolin  Jaworski</span>, <span class="hlFld-ContribAuthor ">Donovan  Tung</span>, <span class="hlFld-ContribAuthor ">Carmen  Wängler</span>, <span class="hlFld-ContribAuthor ">Björn  Wängler</span>, <span class="hlFld-ContribAuthor ">Ralf  Schirrmacher</span>. </span><span class="cited-content_cbyCitation_article-title">Tropomyosin receptor kinase inhibitors: an updated patent review for 2016–2019. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2020,</strong> <em>30 </em>
                                    (5)
                                     , 325-339. <a href="https://doi.org/10.1080/13543776.2020.1737011" title="DOI URL">https://doi.org/10.1080/13543776.2020.1737011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2020.1737011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2020.1737011%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DTropomyosin%252Breceptor%252Bkinase%252Binhibitors%25253A%252Ban%252Bupdated%252Bpatent%252Breview%252Bfor%252B2016%2525E2%252580%2525932019%26aulast%3DBailey%26aufirst%3DJustin%2BJ.%26date%3D2020%26date%3D2020%26volume%3D30%26issue%3D5%26spage%3D325%26epage%3D339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dmitri G.  Fedorov</span>. </span><span class="cited-content_cbyCitation_article-title">Analyzing Interactions with the Fragment Molecular Orbital Method. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 49-73. <a href="https://doi.org/10.1007/978-1-0716-0282-9_4" title="DOI URL">https://doi.org/10.1007/978-1-0716-0282-9_4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-0716-0282-9_4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-0716-0282-9_4%26sid%3Dliteratum%253Aachs%26atitle%3DAnalyzing%252BInteractions%252Bwith%252Bthe%252BFragment%252BMolecular%252BOrbital%252BMethod%26aulast%3DFedorov%26aufirst%3DDmitri%2BG.%26date%3D2020%26date%3D2020%26spage%3D49%26epage%3D73%26pub%3DSpringer%2520US%26atitle%3DQuantum%252BMechanics%252Bin%252BDrug%252BDiscovery%26aulast%3DHeifetz%26aufirst%3DAlexander%26date%3D2020%26volume%3D2114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/medium/jm-2018-001408_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00140&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/medium/jm-2018-001408_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Selectivity hot-spot areas in TrkA binding site. Predicted binding modes of (a) tozasertib and (b) compound <b>a1</b> to TrkA (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YNE">4YNE</a>). Compound <b>a1</b> occupies the favorable hydrophobic subpockets adjacent to the gatekeeper residue (green) but does not reach into the unfavorable area (red) occupied by the cyclopropyl moiety of tozasertib. The favorable subpockets adjacent to the Phe residue of the G-Loop is occupied by substituents in the <b>B</b> set. Note that Phe521 of the G-loop samples differing conformations in other TrkA structures and that the PDB structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YNE">4YNE</a> was chosen for binding mode prediction and structure–activity analysis because it resulted into the most favorable docking scores (i.e., when bound to compounds of the <b>B</b> series).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00140&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/medium/jm-2018-001408_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Summary of computationally designed hit compounds. Affinity values for tozasertib and the top hits are listed for the key target (TrkA; green) and off-target (AurA; red). Modifications compared to the starting compound are highlighted in bold lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00140&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/medium/jm-2018-001408_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Multiobjective compound selection scheme. After initial binding pose generation via docking calculations, the compounds are prioritized via in silico methods that filter first with respect to selectivity criteria (i.e., the ligand-based “kinase profiling platform” approach filters out promiscuous compounds, while the structure-based “selectivity grids” approach filters for TrkA–Aurora selective compounds). Finally, highly active compounds on the key target are prioritized via two complementary machine learning technologies (i.e., using fragment-based matched molecular pair descriptors and based on quantum mechanical calculations, respectively). The final selection does not rely on any manual filtering and includes in the case of the compound set <b>A</b> top ranking compounds from the MMP/ML and QM/ML technologies and in the case of the set <b>B</b> top ranking compounds from MMP/ML.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00140&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/medium/jm-2018-001408_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Profiling results. (a) Kinome profiling data of compound <b>a1</b>. With <1% control, compound <b>a1</b> is highly active only against the key target TrkA. Beside TrkA, six further kinases are inhibited at a screening concentration of 100 nM (i.e., TrkB, TrkC, FLT3, KIT, PDGFRb, RET), leading to a selectivity score of <i>S</i>(35%) = 0.08. The compound is inactive on the off-targets AurA and AurB (white triangles). (b) Heatmap of kinase profiles of the hit compounds <b>a1</b>, <b>a4</b>, <b>b7</b>, and <b>b8</b> as well as the starting compound tozasertib. The ABL1 variants include phosphorylated version of the ABL1 mutants E255K and T315I as well as the nonphosphorylated and phosphorylated versions of the ABL1 wild type.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00140&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/medium/jm-2018-001408_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0005.jpeg" id="rightTab-GRAPHIC-d7e1447-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00140/20180608/images/large/jm-2018-001408_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00140&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 3-chloroperoxybenzoic acid, DCM, rt, 3 h, yield 95%; (b) <i>N</i>-(4-mercaptophenyl)cyclopropanecarboxamide, TEA, CH<sub>3</sub>CN, 80 °C, 3–10 h, yield 70%; (c) 5-methyl-1<i>H</i>-pyrazol-3-amine, DIPEA, DMF, 95 °C, 16 h, yield 44%; (d) amine (1-methylpiperazine or morpholine), DMF, DIPEA, 90 °C, 6–12 h, yield 43%; (e) corresponding thiol, TEA, CH<sub>3</sub>CN, 80 °C, 3–10 h, yields 20–85%; (f) 5-methyl-1<i>H</i>-pyrazol-3-amine, DIPEA, dioxane, 95 °C, 3–6 h, yields 50–75%; (g) amine (1-methylpiperazine or morpholine), DMF, DIPEA, 90 °C, 6–12 h, yields 30–70%; (h) thiophenol, TEA, THF, 50 °C, yields 20–85%; (i) corresponding amine, DIPEA, dioxane, 95 °C, 3–6 h, yields 50–75%; (j) amine (1-methylpiperazine or morpholine), DMF, DIPEA, 90 °C, 6–12 h, yields 30–70%.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48775" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48775" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duong-ly, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devarajan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horiuchi, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, J. R.</span></span> <span> </span><span class="NLM_article-title">Kinase Inhibitor Profiling Reveals Unexpected Opportunities to Inhibit Disease-Associated Mutant Kinases</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">772</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2015.12.080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1016%2Fj.celrep.2015.12.080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=26776524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVaqt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=772-781&issue=4&author=K.+C.+Duong-lyauthor=K.+Devarajanauthor=S.+Liangauthor=K.+Y.+Horiuchiauthor=Y.+Wangauthor=H.+Maauthor=J.+R.+Peterson&title=Kinase+Inhibitor+Profiling+Reveals+Unexpected+Opportunities+to+Inhibit+Disease-Associated+Mutant+Kinases&doi=10.1016%2Fj.celrep.2015.12.080"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase Inhibitor Profiling Reveals Unexpected Opportunities to Inhibit Disease-Associated Mutant Kinases</span></div><div class="casAuthors">Duong-Ly, Krisna C.; Devarajan, Karthik; Liang, Shuguang; Horiuchi, Kurumi Y.; Wang, Yuren; Ma, Haiching; Peterson, Jeffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">772-781</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Small-mol. kinase inhibitors have typically been designed to inhibit wild-type kinases rather than the mutant forms that frequently arise in diseases such as cancer.  Mutations can have serious clin. implications by increasing kinase catalytic activity or conferring therapeutic resistance.  To identify opportunities to repurpose inhibitors against disease-assocd. mutant kinases, we conducted a large-scale functional screen of 183 known kinase inhibitors against 76 recombinant mutant kinases.  The results revealed lead compds. with activity against clin. important mutant kinases, including ALK, LRRK2, RET, and EGFR, as well as unexpected opportunities for repurposing FDA-approved kinase inhibitors as leads for addnl. indications.  Furthermore, using T674I PDGFRα as an example, we show how single-dose screening data can provide predictive structure-activity data to guide subsequent inhibitor optimization.  This study provides a resource for the development of inhibitors against numerous disease-assocd. mutant kinases and illustrates the potential of unbiased profiling as an approach to compd.-centric inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbXqJUD9YoWLVg90H21EOLACvtfcHk0lipOvY-IM1_nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVaqt7c%253D&md5=b01e8f6ce2aa441400531399d8cf88fc</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2015.12.080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2015.12.080%26sid%3Dliteratum%253Aachs%26aulast%3DDuong-ly%26aufirst%3DK.%2BC.%26aulast%3DDevarajan%26aufirst%3DK.%26aulast%3DLiang%26aufirst%3DS.%26aulast%3DHoriuchi%26aufirst%3DK.%2BY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26atitle%3DKinase%2520Inhibitor%2520Profiling%2520Reveals%2520Unexpected%2520Opportunities%2520to%2520Inhibit%2520Disease-Associated%2520Mutant%2520Kinases%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D14%26issue%3D4%26spage%3D772%26epage%3D781%26doi%3D10.1016%2Fj.celrep.2015.12.080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">Computational and Practical Aspects of Drug Repositioning</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1089/adt.2015.29011.tiodrrr</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1089%2Fadt.2015.29011.tiodrrr" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=299-306&issue=6&author=T.+I.+Opreaauthor=J.+P.+Overington&title=Computational+and+Practical+Aspects+of+Drug+Repositioning&doi=10.1089%2Fadt.2015.29011.tiodrrr"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1089%2Fadt.2015.29011.tiodrrr&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2015.29011.tiodrrr%26sid%3Dliteratum%253Aachs%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DComputational%2520and%2520Practical%2520Aspects%2520of%2520Drug%2520Repositioning%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2015%26volume%3D13%26issue%3D6%26spage%3D299%26epage%3D306%26doi%3D10.1089%2Fadt.2015.29011.tiodrrr" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metz, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merta, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kifle, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span> <span> </span><span class="NLM_article-title">Navigating the Kinome</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">200</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1038/nchembio.530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1038%2Fnchembio.530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=21336281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=200-202&issue=4&author=J.+T.+Metzauthor=E.+F.+Johnsonauthor=N.+B.+Soniauthor=P.+J.+Mertaauthor=L.+Kifleauthor=P.+J.+Hajduk&title=Navigating+the+Kinome&doi=10.1038%2Fnchembio.530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating the kinome</span></div><div class="casAuthors">Metz, James T.; Johnson, Eric F.; Soni, Niru B.; Merta, Philip J.; Kifle, Lemma; Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">200-202</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although it is increasingly being recognized that drug-target interaction networks can be powerful tools for the interrogation of systems biol. and the rational design of multitargeted drugs, there is no generalized, statistically validated approach to harmonizing sequence-dependent and pharmacol.-dependent networks.  Here we demonstrate the creation of a comprehensive kinome interaction network based not only on sequence comparisons but also on multiple pharmacol. parameters derived from activity profiling data.  The framework described for statistical interpretation of these network connections also enables rigorous investigation of chemotype-specific interaction networks, which is crit. for multitargeted drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOgzF9Q0eH87Vg90H21EOLACvtfcHk0lhxYvBYBhtsZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D&md5=09234ec451a4918e5aca6d65e0b65aee</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.530%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DKifle%26aufirst%3DL.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DNavigating%2520the%2520Kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26issue%3D4%26spage%3D200%26epage%3D202%26doi%3D10.1038%2Fnchembio.530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krüger, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Light, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlinchey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadatos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">The ChEMBL Bioactivity Database: An Update</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1083</span>– <span class="NLM_lpage">1090</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt1031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1093%2Fnar%2Fgkt1031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=24214965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslWl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1083-1090&issue=D1&author=A.+P.+Bentoauthor=A.+Gaultonauthor=A.+Herseyauthor=L.+J.+Bellisauthor=J.+Chambersauthor=M.+Daviesauthor=F.+A.+Kr%C3%BCgerauthor=Y.+Lightauthor=L.+Makauthor=S.+McGlincheyauthor=M.+Nowotkaauthor=G.+Papadatosauthor=R.+Santosauthor=J.+P.+Overington&title=The+ChEMBL+Bioactivity+Database%3A+An+Update&doi=10.1093%2Fnar%2Fgkt1031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The ChEMBL bioactivity database: an update</span></div><div class="casAuthors">Bento, A. Patricia; Gaulton, Anna; Hersey, Anne; Bellis, Louisa J.; Chambers, Jon; Davies, Mark; Krueger, Felix A.; Light, Yvonne; Mak, Lora; McGlinchey, Shaun; Nowotka, Michal; Papadatos, George; Santos, Rita; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1083-D1090</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an open large-scale bioactivity database (https://www.ebi.ac.uk/chembl), previously described in the 2012 Nucleic Acids Research Database Issue.  Since then, a variety of new data sources and improvements in functionality have contributed to the growth and utility of the resource.  In particular, more comprehensive tracking of compds. from research stages through clin. development to market is provided through the inclusion of data from United States Adopted Name applications; a new richer data model for representing drug targets has been developed; and a no. of methods have been put in place to allow users to more easily identify reliable data.  Finally, access to ChEMBL is now available via a new Resource Description Framework format, in addn. to the web-based interface, data downloads and web services.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXb5k1IvJNaLVg90H21EOLACvtfcHk0lhxYvBYBhtsZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslWl&md5=31b832d03d56ea3065d7aa29618362bc</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1031%26sid%3Dliteratum%253Aachs%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DKr%25C3%25BCger%26aufirst%3DF.%2BA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520ChEMBL%2520Bioactivity%2520Database%253A%2520An%2520Update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26issue%3DD1%26spage%3DD1083%26epage%3D1090%26doi%3D10.1093%2Fnar%2Fgkt1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Analysis of Kinase Inhibitor Selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&issue=11&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+Analysis+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0liEGOBUxl0-Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520Analysis%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26issue%3D11%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anastassiadis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deacon, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devarajan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, J. R.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Assay of Kinase Catalytic Activity Reveals Features of Kinase Inhibitor Selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1039</span>– <span class="NLM_lpage">1045</span>, <span class="refDoi"> DOI: 10.1038/nbt.2017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1038%2Fnbt.2017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=22037377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1039-1045&issue=11&author=T.+Anastassiadisauthor=S.+W.+Deaconauthor=K.+Devarajanauthor=H.+Maauthor=J.+R.+Peterson&title=Comprehensive+Assay+of+Kinase+Catalytic+Activity+Reveals+Features+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt.2017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span></div><div class="casAuthors">Anastassiadis, Theonie; Deacon, Sean W.; Devarajan, Karthik; Ma, Haiching; Peterson, Jeffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1039-1045</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. protein kinase inhibitors are widely used to elucidate cellular signaling pathways and are promising therapeutic agents.  Owing to evolutionary conservation of the ATP-binding site, most kinase inhibitors that target this site promiscuously inhibit multiple kinases.  Interpretation of expts. that use these compds. is confounded by a lack of data on the comprehensive kinase selectivity of most inhibitors.  Here we used functional assays to profile the activity of 178 com. available kinase inhibitors against a panel of 300 recombinant protein kinases.  Quant. anal. revealed complex and often unexpected interactions between protein kinases and kinase inhibitors, with a wide spectrum of promiscuity.  Many off-target interactions occur with seemingly unrelated kinases, revealing how large-scale profiling can identify multitargeted inhibitors of specific, diverse kinases.  The results have implications for drug development and provide a resource for selecting compds. to elucidate kinase function and for interpreting the results of expts. involving kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUljWIhZGPkrVg90H21EOLACvtfcHk0liEGOBUxl0-Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E&md5=b2f705509b0e09afa46b01d4669f0515</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2017%26sid%3Dliteratum%253Aachs%26aulast%3DAnastassiadis%26aufirst%3DT.%26aulast%3DDeacon%26aufirst%3DS.%2BW.%26aulast%3DDevarajan%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26atitle%3DComprehensive%2520Assay%2520of%2520Kinase%2520Catalytic%2520Activity%2520Reveals%2520Features%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26issue%3D11%26spage%3D1039%26epage%3D1045%26doi%3D10.1038%2Fnbt.2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falchook, G.
S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastida, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurzrock, R.</span></span> <span> </span><span class="NLM_article-title">Aurora Kinase Inhibitors in Oncology Clinical Trials: Current State of the Progress</span>. <i>Semin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">832</span>– <span class="NLM_lpage">848</span>, <span class="refDoi"> DOI: 10.1053/j.seminoncol.2015.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1053%2Fj.seminoncol.2015.09.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=26615129" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWksL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2015&pages=832-848&issue=6&author=G.%0AS.+Falchookauthor=C.+C.+Bastidaauthor=R.+Kurzrock&title=Aurora+Kinase+Inhibitors+in+Oncology+Clinical+Trials%3A+Current+State+of+the+Progress&doi=10.1053%2Fj.seminoncol.2015.09.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Aurora Kinase Inhibitors in Oncology Clinical Trials: Current State of the Progress</span></div><div class="casAuthors">Falchook, Gerald S.; Bastida, Christel C.; Kurzrock, Razelle</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">832-848</span>CODEN:
                <span class="NLM_cas:coden">SOLGAV</span>;
        ISSN:<span class="NLM_cas:issn">0093-7754</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The Aurora kinase family of kinases (Aurora A, B, and C) are involved in multiple mitotic events, and aberrant expression of these kinases is assocd. with tumorigenesis.  Aurora A and Aurora B are validated anticancer targets, and the development of Aurora kinase inhibitors has progressed from preclin. to clin. studies.  A variety of Aurora A, B and pan-Aurora kinase inhibitors have entered the clinic.  The main side effects include febrile neutropenia, stomatitis, gastrointestinal toxicity, hypertension, and fatigue.  Responses including complete remissions have been described in diverse, advanced malignancies, most notably ovarian cancer and acute myelogenous leukemia.  This review highlights the biol. rationale for Aurora kinase as a target, and clin. trials involving Aurora kinase inhibitors, with particular emphasis on published early phase studies, and the obsd. anti-tumor activity of these agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFVqflqPnM6LVg90H21EOLACvtfcHk0lgr-yL0-biVJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWksL7L&md5=55465822ce60d2cf2dfd19606de09a71</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1053%2Fj.seminoncol.2015.09.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.seminoncol.2015.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DFalchook%26aufirst%3DG.%2BS.%26aulast%3DBastida%26aufirst%3DC.%2BC.%26aulast%3DKurzrock%26aufirst%3DR.%26atitle%3DAurora%2520Kinase%2520Inhibitors%2520in%2520Oncology%2520Clinical%2520Trials%253A%2520Current%2520State%2520of%2520the%2520Progress%26jtitle%3DSemin.%2520Oncol.%26date%3D2015%26volume%3D42%26issue%3D6%26spage%3D832%26epage%3D848%26doi%3D10.1053%2Fj.seminoncol.2015.09.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Norman, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, J. S.</span></span> <span> </span><span class="NLM_article-title">Targeting the Nerve Growth Factor (NGF) Pathway in Drug Discovery. Potential Applications to New Therapies for Chronic Pain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">66</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00964</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00964" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=66-88&issue=1&author=B.+H.+Normanauthor=J.+S.+McDermott&title=Targeting+the+Nerve+Growth+Factor+%28NGF%29+Pathway+in+Drug+Discovery.+Potential+Applications+to+New+Therapies+for+Chronic+Pain&doi=10.1021%2Facs.jmedchem.6b00964"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00964&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00964%26sid%3Dliteratum%253Aachs%26aulast%3DNorman%26aufirst%3DB.%2BH.%26aulast%3DMcDermott%26aufirst%3DJ.%2BS.%26atitle%3DTargeting%2520the%2520Nerve%2520Growth%2520Factor%2520%2528NGF%2529%2520Pathway%2520in%2520Drug%2520Discovery.%2520Potential%2520Applications%2520to%2520New%2520Therapies%2520for%2520Chronic%2520Pain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D1%26spage%3D66%26epage%3D88%26doi%3D10.1021%2Facs.jmedchem.6b00964" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=D.+E.+Shawauthor=M.+Shelleyauthor=J.+K.+Perryauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lijM4F9LafZeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merget, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulle, S.</span></span> <span> </span><span class="NLM_article-title">Profiling Prediction of Kinase Inhibitors: Toward the Virtual Assay</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">474</span>– <span class="NLM_lpage">485</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01611</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01611" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=474-485&issue=1&author=B.+Mergetauthor=S.+Turkauthor=S.+Eidauthor=F.+Rippmannauthor=S.+Fulle&title=Profiling+Prediction+of+Kinase+Inhibitors%3A+Toward+the+Virtual+Assay&doi=10.1021%2Facs.jmedchem.6b01611"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01611%26sid%3Dliteratum%253Aachs%26aulast%3DMerget%26aufirst%3DB.%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DEid%26aufirst%3DS.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DFulle%26aufirst%3DS.%26atitle%3DProfiling%2520Prediction%2520of%2520Kinase%2520Inhibitors%253A%2520Toward%2520the%2520Virtual%2520Assay%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D1%26spage%3D474%26epage%3D485%26doi%3D10.1021%2Facs.jmedchem.6b01611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merget, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulle, S.</span></span> <span> </span><span class="NLM_article-title">Coupling Matched Molecular Pairs with Machine Learning for Virtual Compound Optimization</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3079</span>– <span class="NLM_lpage">3085</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00298</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00298" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2gsLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=3079-3085&issue=12&author=S.+Turkauthor=B.+Mergetauthor=F.+Rippmannauthor=S.+Fulle&title=Coupling+Matched+Molecular+Pairs+with+Machine+Learning+for+Virtual+Compound+Optimization&doi=10.1021%2Facs.jcim.7b00298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Coupling Matched Molecular Pairs with Machine Learning for Virtual Compound Optimization</span></div><div class="casAuthors">Turk, Samo; Merget, Benjamin; Rippmann, Friedrich; Fulle, Simone</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3079-3085</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Matched mol. pair (MMP) analyses are widely used in compd. optimization projects to gain insights into structure-activity relationships (SAR).  The anal. is traditionally done via statistical methods but can also be employed together with machine learning (ML) approaches to extrapolate to novel compds.  The here introduced MMP/ML method combines a fragment-based MMP implementation with different machine learning methods to obtain automated SAR decompn. and prediction.  To test the prediction capabilities and model transferability, two different compd. optimization scenarios were designed: (1) "new fragments" which occurs when exploring new fragments for a defined compd. series and (2) "new static core and transformations" which resembles for instance the identification of a new compd. series.  Very good results were achieved by all employed machine learning methods esp. for the new fragments case, but overall deep neural network models performed best, allowing reliable predictions also for the new static core and transformations scenario, where comprehensive SAR knowledge of the compd. series is missing.  Furthermore, the authors show that models trained on all available data have a higher generalizability compared to models trained on focused series and can extend beyond chem. space covered in the training data.  Thus, coupling MMP with deep neural networks provides a promising approach to make high quality predictions on various data sets and in different compd. optimization scenarios.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKw4NG2oltF7Vg90H21EOLACvtfcHk0lijM4F9LafZeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2gsLnJ&md5=828c42567a0c5dfc2f189b730055f154</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00298%26sid%3Dliteratum%253Aachs%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DMerget%26aufirst%3DB.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DFulle%26aufirst%3DS.%26atitle%3DCoupling%2520Matched%2520Molecular%2520Pairs%2520with%2520Machine%2520Learning%2520for%2520Virtual%2520Compound%2520Optimization%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D12%26spage%3D3079%26epage%3D3085%26doi%3D10.1021%2Facs.jcim.7b00298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uebayasi, M.</span></span> <span> </span><span class="NLM_article-title">Fragment Molecular Orbital Method: An Approximate Computational Method for Large Molecules</span>. <i>Chem. Phys. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>313</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">701</span>– <span class="NLM_lpage">706</span>, <span class="refDoi"> DOI: 10.1016/S0009-2614(99)00874-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1016%2FS0009-2614%2899%2900874-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=1999&pages=701-706&issue=3&author=K.+Kitauraauthor=E.+Ikeoauthor=T.+Asadaauthor=T.+Nakanoauthor=M.+Uebayasi&title=Fragment+Molecular+Orbital+Method%3A+An+Approximate+Computational+Method+for+Large+Molecules&doi=10.1016%2FS0009-2614%2899%2900874-X"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2FS0009-2614%2899%2900874-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0009-2614%252899%252900874-X%26sid%3Dliteratum%253Aachs%26aulast%3DKitaura%26aufirst%3DK.%26aulast%3DIkeo%26aufirst%3DE.%26aulast%3DAsada%26aufirst%3DT.%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DUebayasi%26aufirst%3DM.%26atitle%3DFragment%2520Molecular%2520Orbital%2520Method%253A%2520An%2520Approximate%2520Computational%2520Method%2520for%2520Large%2520Molecules%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D1999%26volume%3D313%26issue%3D3%26spage%3D701%26epage%3D706%26doi%3D10.1016%2FS0009-2614%2899%2900874-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldridge, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boatz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elbert, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koseki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Windus, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupuis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J. A.,  Jr.</span></span> <span> </span><span class="NLM_article-title">General Atomic and Molecular Electronic Structure System</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1363</span>, <span class="refDoi"> DOI: 10.1002/jcc.540141112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1002%2Fjcc.540141112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADyaK2cXhsFaqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1993&pages=1347-1363&issue=11&author=M.+W.+Schmidtauthor=K.+K.+Baldridgeauthor=J.+A.+Boatzauthor=S.+T.+Elbertauthor=M.+S.+Gordonauthor=J.+H.+Jensenauthor=S.+Kosekiauthor=N.+Matsunagaauthor=K.+A.+Nguyenauthor=S.+Suauthor=T.+L.+Windusauthor=M.+Dupuisauthor=J.+A.+Montgomery&title=General+Atomic+and+Molecular+Electronic+Structure+System&doi=10.1002%2Fjcc.540141112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">General atomic and molecular electronic structure system</span></div><div class="casAuthors">Schmidt, Michael W.; Baldridge, Kim K.; Boatz, Jerry A.; Elbert, Steven T.; Gordon, Mark S.; Jensen, Jan H.; Koseki, Shiro; Matsunaga, Nikita; Nguyen, Kiet A.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1347-63</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">A description of the ab initio quantum chem. package GAMESS is presented.  Chem. systems contg. atoms through Rn can be treated with wave functions ranging from the simplest closed-shell case up to a general MCSCF case, permitting calcns. at the necessary level of sophistication.  Emphasis is given to novel features of the program.  The parallelization strategy used in the RHF, ROHF, UHF, and GVB sections of the program is described, and detailed speedup results are given.  Parallel calcns. can be run on ordinary workstations as well as dedicated parallel machines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pHblAdXU4rVg90H21EOLACvtfcHk0li5v0fFFzocGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhsFaqtQ%253D%253D&md5=4d9e37cd3ec67a92bdfb5b0c0084707c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540141112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540141112%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DM.%2BW.%26aulast%3DBaldridge%26aufirst%3DK.%2BK.%26aulast%3DBoatz%26aufirst%3DJ.%2BA.%26aulast%3DElbert%26aufirst%3DS.%2BT.%26aulast%3DGordon%26aufirst%3DM.%2BS.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26aulast%3DKoseki%26aufirst%3DS.%26aulast%3DMatsunaga%26aufirst%3DN.%26aulast%3DNguyen%26aufirst%3DK.%2BA.%26aulast%3DSu%26aufirst%3DS.%26aulast%3DWindus%26aufirst%3DT.%2BL.%26aulast%3DDupuis%26aufirst%3DM.%26aulast%3DMontgomery%26aufirst%3DJ.%2BA.%26atitle%3DGeneral%2520Atomic%2520and%2520Molecular%2520Electronic%2520Structure%2520System%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1993%26volume%3D14%26issue%3D11%26spage%3D1347%26epage%3D1363%26doi%3D10.1002%2Fjcc.540141112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, M.</span></span> <span> </span><span class="NLM_article-title">Managing Common Toxicities with New Tyrosine Kinase Inhibitors</span> <i>CancerWorld</i> <span class="NLM_year" style="font-weight: bold;">2015</span>, <span class="NLM_issue">November–December</span>,  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">37</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=31-37&issue=November%E2%80%93December&author=M.+Oliveira&title=Managing+Common+Toxicities+with+New+Tyrosine+Kinase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOliveira%26aufirst%3DM.%26atitle%3DManaging%2520Common%2520Toxicities%2520with%2520New%2520Tyrosine%2520Kinase%2520Inhibitors%26jtitle%3DCancerWorld%26date%3D2015%26issue%3DNovember%25E2%2580%2593December%26spage%3D31%26epage%3D37" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’shea, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kontzias, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaoka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurence, A.</span></span> <span> </span><span class="NLM_article-title">Janus Kinase Inhibitors in Autoimmune Diseases</span>. <i>Ann. Rheum. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">i111</span>– <span class="NLM_lpage">i115</span>, <span class="refDoi"> DOI: 10.1136/annrheumdis-2012-202576</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1136%2Fannrheumdis-2012-202576" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFyntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2013&pages=i111-i115&author=J.+J.+O%E2%80%99sheaauthor=A.+Kontziasauthor=K.+Yamaokaauthor=Y.+Tanakaauthor=A.+Laurence&title=Janus+Kinase+Inhibitors+in+Autoimmune+Diseases&doi=10.1136%2Fannrheumdis-2012-202576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Janus kinase inhibitors in autoimmune diseases</span></div><div class="casAuthors">O'Shea, John J.; Kontzias, Apostolos; Yamaoka, Kunihiro; Tanaka, Yoshiya; Laurence, Arian</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">Suppl. 2</span>),
    <span class="NLM_cas:pages">ii111-ii115</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">A review.  Biol. therapies directed at proinflammatory cytokines have irrevocably changed the landscape of treatment of rheumatoid arthritis (RA) and other autoimmune diseases.  With the advances in our knowledge in cytokine signalling, the question emerges whether targeting intracellular signalling might also be a safe and efficacious strategy.  Janus kinases or Jaks are crit. for a large family of cytokines and the first Jak inhibitors has been approved by the FDA.  It is therefore timely to consider this new category of drugs and reflect on their potential roles, present and future, in the treatment of RA and related disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWFj0gZ1gj1LVg90H21EOLACvtfcHk0lju10IAUvYVXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFyntbk%253D&md5=2776146c05d128b9659c36ab6b4964fa</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2012-202576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2012-202576%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599shea%26aufirst%3DJ.%2BJ.%26aulast%3DKontzias%26aufirst%3DA.%26aulast%3DYamaoka%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DLaurence%26aufirst%3DA.%26atitle%3DJanus%2520Kinase%2520Inhibitors%2520in%2520Autoimmune%2520Diseases%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2013%26volume%3D72%26spage%3Di111%26epage%3Di115%26doi%3D10.1136%2Fannrheumdis-2012-202576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, M. L.</span></span> <span> </span><span class="NLM_article-title">Trk Kinase Inhibitors as New Treatments for Cancer and Pain</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">319</span>, <span class="refDoi"> DOI: 10.1517/13543770902721261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1517%2F13543770902721261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=19441906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjslWlu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=305-319&issue=3&author=T.+Wangauthor=D.+Yuauthor=M.+L.+Lamb&title=Trk+Kinase+Inhibitors+as+New+Treatments+for+Cancer+and+Pain&doi=10.1517%2F13543770902721261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Trk kinase inhibitors as new treatments for cancer and pain</span></div><div class="casAuthors">Wang, Tao; Yu, Dingwei; Lamb, Michelle L.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">305-319</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Background: Tropomyosin-related kinases (Trks) are a family of receptor tyrosine kinases activated by neurotrophins.  Trks play important roles in pain sensation as well as tumor cell growth and survival signaling.  Thus, inhibitors of Trk receptor kinases might provide targeted treatments for pain and cancer.  Objective: This paper reviews those patent applications since 2002 claiming small-mol. inhibitors of Trk receptor kinases.  Methods: Primary literature and patents were searched with SciFinder and Google Scholar.  Patents were selected based on their relevance to Trks and were evaluated and representative compds. were listed as examples.  Results/conclusion: Several series of Trk inhibitors with excellent in vitro potencies have been reported and a no. of compds. have gone into the clinic.  It should be noted that few of these inhibitors are Trk selective, demonstrating that targeting Trk kinases for treatment of pain and/or cancer offers a promising but also challenging approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvknvXUUz2DbVg90H21EOLACvtfcHk0lju10IAUvYVXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjslWlu7o%253D&md5=ffc2d3e33102631cac7b89dce7f96a5a</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1517%2F13543770902721261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543770902721261%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26atitle%3DTrk%2520Kinase%2520Inhibitors%2520as%2520New%2520Treatments%2520for%2520Cancer%2520and%2520Pain%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2009%26volume%3D19%26issue%3D3%26spage%3D305%26epage%3D319%26doi%3D10.1517%2F13543770902721261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, E.</span></span> <span> </span><span class="NLM_article-title">Tropomyosin Receptor Kinase Inhibitors: A Patent Update 2009-2013</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">731</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1517/13543776.2014.910195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1517%2F13543776.2014.910195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=24809946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVSqtr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=731-744&author=C.+McCarthyauthor=E.+Walker&title=Tropomyosin+Receptor+Kinase+Inhibitors%3A+A+Patent+Update+2009-2013&doi=10.1517%2F13543776.2014.910195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Tropomyosin receptor kinase inhibitors: a patent update 2009 - 2013</span></div><div class="casAuthors">McCarthy, Clive; Walker, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">731-744</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: Tropomyosin receptor kinases (Trks) are a family of three similar tyrosine kinases activated by peptide hormones of the neurotrophin family.  The nerve growth factor antibody tanezumab has provided clin. proof of concept for inhibition of the TrkA pathway in pain.  As an alternative modality, small-mol. inhibitors of the Trks have been pursued in recent years to probe the role of these neurotrophin pathways in pain, cancer and other indications.  Areas covered: This paper reviews the patent literature between mid-2009 and 2013, claiming inhibitors of Trk family members as the primary biol. targets.  Addnl. patents have been reviewed where Trk is not the main kinase of interest but in which high Trk potency is obsd. and the chem. matter is particularly noteworthy.  Patents pre-dating this period have been reviewed previously.  Scifinder and Google were used to find relevant patents and clin. information using Trk or Tropomyosin as the search term.  Expert opinion: Considerable recent progress has been made in the identification of selective pan Trk inhibitors with pharmacodynamic and pharmacokinetic properties appropriate for clin. evaluation.  Inhibitors of both active and inactive conformations of the Trks as well as peripherally restricted mols. have been identified.  Furthermore, TrkA-selective allosteric inhibitors have recently been disclosed, which enables the biol. of this isoform to be probed.  The recent identification of a TrkA gene fusion in a subset of lung cancer patients will increase further the attraction of Trk inhibition to the pharmaceutical industry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjXwkruGnLXbVg90H21EOLACvtfcHk0lju10IAUvYVXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVSqtr7L&md5=b924331cd9d69192bc10f53424997550</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1517%2F13543776.2014.910195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2014.910195%26sid%3Dliteratum%253Aachs%26aulast%3DMcCarthy%26aufirst%3DC.%26aulast%3DWalker%26aufirst%3DE.%26atitle%3DTropomyosin%2520Receptor%2520Kinase%2520Inhibitors%253A%2520A%2520Patent%2520Update%25202009-2013%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2014%26volume%3D24%26spage%3D731%26epage%3D744%26doi%3D10.1517%2F13543776.2014.910195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirrmacher, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard-Gauthier, V.</span></span> <span> </span><span class="NLM_article-title">Tropomyosin Receptor Kinase Inhibitors: An Updated Patent Review for 2010–2016 - Part I</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">751</span>, <span class="refDoi"> DOI: 10.1080/13543776.2017.1297796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1080%2F13543776.2017.1297796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=28270010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvFyjtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=733-751&author=J.+Baileyauthor=R.+Schirrmacherauthor=K.+Farrellauthor=V.+Bernard-Gauthier&title=Tropomyosin+Receptor+Kinase+Inhibitors%3A+An+Updated+Patent+Review+for+2010%E2%80%932016+-+Part+I&doi=10.1080%2F13543776.2017.1297796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Tropomyosin receptor kinase inhibitors: an updated patent review for 2010-2016 - Part I</span></div><div class="casAuthors">Bailey, Justin J.; Schirrmacher, Ralf; Farrell, Kristen; Bernard-Gauthier, Vadim</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">733-751</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Tropomyosin receptor kinases (TrkA/B/C) play crucial roles in the development and maintenance of the nervous system, and aberrant expression of Trk has been implicated in neurol. disorders as well as neural and non-neural neoplasms.  Patent activity encompassing Trk inhibitors has grown substantially over the last 6 years, recognized by a rise in the no. of pharmaceutical entrants to the field and the escalation of novel inhibitor chemotypes.  In Part I of this two part review, a biol. and structural overview of Trk is provided in the context of Trk as a therapeutic target for cancer and pain, followed by the report of recent patent literature claiming small mol. inhibitors of Trk family kinases or which describe inhibitors developed for other kinase targets but include noteworthy Trk inhibition/application.  The discussion of the patent literature continues in Part II of this review, which includes an in-depth view of the current clin. applications of Trk inhibitors.  Substantial synthetic efforts in Trk inhibitor development has propagated numerous and diverse inhibitor chemotypes, including TrkA-specific inhibitors.  While many novel Trk inhibitors remain the original progeny of Trk-specific development programs, kinase inhibitors initially developed for other kinases have also been successfully repositioned for Trk.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIHEog1RqjUrVg90H21EOLACvtfcHk0lju10IAUvYVXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvFyjtb0%253D&md5=e863493e3065568c43e39c0fe5a73751</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1080%2F13543776.2017.1297796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2017.1297796%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DJ.%26aulast%3DSchirrmacher%26aufirst%3DR.%26aulast%3DFarrell%26aufirst%3DK.%26aulast%3DBernard-Gauthier%26aufirst%3DV.%26atitle%3DTropomyosin%2520Receptor%2520Kinase%2520Inhibitors%253A%2520An%2520Updated%2520Patent%2520Review%2520for%25202010%25E2%2580%25932016%2520-%2520Part%2520I%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2017%26volume%3D27%26spage%3D733%26epage%3D751%26doi%3D10.1080%2F13543776.2017.1297796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirrmacher, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard-Gauthier, V.</span></span> <span> </span><span class="NLM_article-title">Tropomyosin Receptor Kinase Inhibitors: An Updated Patent Review for 2010–2016 - Part II</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">831</span>– <span class="NLM_lpage">849</span>, <span class="refDoi"> DOI: 10.1080/13543776.2017.1297797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1080%2F13543776.2017.1297797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=28270021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvFyjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=831-849&author=J.+Baileyauthor=R.+Schirrmacherauthor=K.+Farrellauthor=V.+Bernard-Gauthier&title=Tropomyosin+Receptor+Kinase+Inhibitors%3A+An+Updated+Patent+Review+for+2010%E2%80%932016+-+Part+II&doi=10.1080%2F13543776.2017.1297797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Tropomyosin receptor kinase inhibitors: an updated patent review for 2010-2016 - Part II</span></div><div class="casAuthors">Bailey, Justin J.; Schirrmacher, Ralf; Farrell, Kristen; Bernard-Gauthier, Vadim</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">831-849</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">TrkA/B/C receptor activation supports growth, survival, and differentiation of discrete neuronal populations during development, adult life, and ageing but also plays numerous roles in human disease onset and progression.  Trk-specific inhibitors have therapeutic applications in cancer and pain and thus constitute a growing area of interest in oncol. and neurol.  There has been substantial growth in the no. of structural classes of Trk inhibitors and the no. of industrial entrants to the Trk inhibitor field over the past six years.  In Part II of this two-part review, the discussion of recent patent literature covering Trk family inhibitors is continued from Part I and clin. research with Trk inhibitors is considered.  Trk has been molecularly targeted for over a decade resulting in the progressive evolution of structurally diversified Trk inhibitors arising from scaffold hopping and HTS efforts.  Correspondingly, there have been a growing no. of clin. investigations utilizing Trk inhibitors in recent years, with a particular focus on the treatment of NTRK-fusion pos. cancers and chronic pain.  The obsd. potential of Trk inhibitors to cause adverse CNS side effects however suggests the need for a more rigorous consideration of BBB permeation capabilities during drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG_4h-yrNplrVg90H21EOLACvtfcHk0ljZJSEFkRo5AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvFyjtbw%253D&md5=9c0a05bec29ecf9ccf53cc26dda78709</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1080%2F13543776.2017.1297797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2017.1297797%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DJ.%26aulast%3DSchirrmacher%26aufirst%3DR.%26aulast%3DFarrell%26aufirst%3DK.%26aulast%3DBernard-Gauthier%26aufirst%3DV.%26atitle%3DTropomyosin%2520Receptor%2520Kinase%2520Inhibitors%253A%2520An%2520Updated%2520Patent%2520Review%2520for%25202010%25E2%2580%25932016%2520-%2520Part%2520II%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2017%26volume%3D27%26spage%3D831%26epage%3D849%26doi%3D10.1080%2F13543776.2017.1297797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thress, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacIntyre, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitston, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zage, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweidler-McKay, P. A.</span></span> <span> </span><span class="NLM_article-title">Identification and Preclinical Characterization of AZ-23, a Novel, Selective, and Orally Bioavailable Inhibitor of the Trk Kinase Pathway</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1818</span>– <span class="NLM_lpage">1827</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-09-0036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1158%2F1535-7163.MCT-09-0036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=19509272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosVymsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=1818-1827&issue=7&author=K.+Thressauthor=T.+MacIntyreauthor=H.+Wangauthor=D.+Whitstonauthor=Z.-Y.+Liuauthor=E.+Hoffmannauthor=T.+Wangauthor=J.+L.+Brownauthor=K.+Websterauthor=C.+Omerauthor=P.+E.+Zageauthor=L.+Zengauthor=P.+A.+Zweidler-McKay&title=Identification+and+Preclinical+Characterization+of+AZ-23%2C+a+Novel%2C+Selective%2C+and+Orally+Bioavailable+Inhibitor+of+the+Trk+Kinase+Pathway&doi=10.1158%2F1535-7163.MCT-09-0036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and preclinical characterization of AZ-23, a novel, selective, and orally bioavailable inhibitor of the Trk kinase pathway</span></div><div class="casAuthors">Thress, Kenneth; MacIntyre, Terry; Wang, Haiyun; Whitston, Dave; Liu, Zhong-Ying; Hoffmann, Ethan; Wang, Tao; Brown, Jeffrey L.; Webster, Kevin; Omer, Charles; Zage, Peter E.; Zeng, Lizhi; Zweidler-McKay, Patrick A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1818-1827</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Tropomyosin-related kinases (TrkA, TrkB, and TrkC) are receptor tyrosine kinases that, along with their ligands, the neurotrophins, are involved in neuronal cell growth, development, and survival.  The Trk-neurotrophin pathway may also play a role in tumorigenesis through oncogenic fusions, mutations, and autocrine signaling, prompting the development of novel Trk inhibitors as agents for cancer therapy.  This report describes the identification of AZ-23, a novel, potent, and selective Trk kinase inhibitor.  In vitro studies with AZ-23 showed improved selectivity over previous compds. and inhibition of Trk kinase activity in cells at low nanomolar concns.  AZ-23 showed in vivo TrkA kinase inhibition and efficacy in mice following oral administration in a TrkA-driven allograft model and significant tumor growth inhibition in a Trk-expressing xenograft model of neuroblastoma.  AZ-23 represents a potent and selective Trk kinase inhibitor from a novel series with the potential for use as a treatment for cancer. [Mol Cancer Ther 2009;8(7):1818-27].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG2uvCSID5vbVg90H21EOLACvtfcHk0ljZJSEFkRo5AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosVymsro%253D&md5=8501f75147407ee64920fd4f5e733536</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-09-0036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-09-0036%26sid%3Dliteratum%253Aachs%26aulast%3DThress%26aufirst%3DK.%26aulast%3DMacIntyre%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWhitston%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DZ.-Y.%26aulast%3DHoffmann%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DBrown%26aufirst%3DJ.%2BL.%26aulast%3DWebster%26aufirst%3DK.%26aulast%3DOmer%26aufirst%3DC.%26aulast%3DZage%26aufirst%3DP.%2BE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZweidler-McKay%26aufirst%3DP.%2BA.%26atitle%3DIdentification%2520and%2520Preclinical%2520Characterization%2520of%2520AZ-23%252C%2520a%2520Novel%252C%2520Selective%252C%2520and%2520Orally%2520Bioavailable%2520Inhibitor%2520of%2520the%2520Trk%2520Kinase%2520Pathway%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2009%26volume%3D8%26issue%3D7%26spage%3D1818%26epage%3D1827%26doi%3D10.1158%2F1535-7163.MCT-09-0036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bakri, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahbeh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abu Sneina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al Khader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obeidat, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">AlAwwa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ki, C.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masri, A.</span></span> <span> </span><span class="NLM_article-title">Congenital Insensitivity to Pain and Anhydrosis due to a Rare Mutation and That Is Complicated by Inflammatory Bowel Disease and Amyloidosis: A Case Report</span>. <i>Clin. Case Reports</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">997</span>– <span class="NLM_lpage">1000</span>, <span class="refDoi"> DOI: 10.1002/ccr3.689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1002%2Fccr3.689" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=27761255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A280%3ADC%252BC2srktFKhsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2016&pages=997-1000&issue=10&author=F.+G.+Bakriauthor=A.+Wahbehauthor=A.+Abu+Sneinaauthor=A.+Al+Khaderauthor=F.+Obeidatauthor=I.+AlAwwaauthor=M.+Buniauthor=C.-S.+Kiauthor=A.+Masri&title=Congenital+Insensitivity+to+Pain+and+Anhydrosis+due+to+a+Rare+Mutation+and+That+Is+Complicated+by+Inflammatory+Bowel+Disease+and+Amyloidosis%3A+A+Case+Report&doi=10.1002%2Fccr3.689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Congenital insensitivity to pain and anhydrosis due to a rare mutation and that is complicated by inflammatory bowel disease and amyloidosis: a case report</span></div><div class="casAuthors">Bakri Faris G; Wahbeh Ayman; Abu Sneina Awni; AlAwwa Izzat; Al Khader Ali; Obeidat Fatima; Buni Maryam; Ki Chang-Seok; Masri Amira</div><div class="citationInfo"><span class="NLM_cas:title">Clinical case reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">997-1000</span>
        ISSN:<span class="NLM_cas:issn">2050-0904</span>.
    </div><div class="casAbstract">Patients with congenital insensitivity to pain and anhydrosis syndrome are at risk for renal amyloidosis and inflammatory bowel disease.  Physicians caring for such patients should be aware of these complications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdsZ2fFId2JBe4jJGKT5N0fW6udTcc2ea_ehl_xseWerntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srktFKhsQ%253D%253D&md5=513f89ccf382d11aa22326c933b633ad</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fccr3.689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fccr3.689%26sid%3Dliteratum%253Aachs%26aulast%3DBakri%26aufirst%3DF.%2BG.%26aulast%3DWahbeh%26aufirst%3DA.%26aulast%3DAbu%2BSneina%26aufirst%3DA.%26aulast%3DAl%2BKhader%26aufirst%3DA.%26aulast%3DObeidat%26aufirst%3DF.%26aulast%3DAlAwwa%26aufirst%3DI.%26aulast%3DBuni%26aufirst%3DM.%26aulast%3DKi%26aufirst%3DC.-S.%26aulast%3DMasri%26aufirst%3DA.%26atitle%3DCongenital%2520Insensitivity%2520to%2520Pain%2520and%2520Anhydrosis%2520due%2520to%2520a%2520Rare%2520Mutation%2520and%2520That%2520Is%2520Complicated%2520by%2520Inflammatory%2520Bowel%2520Disease%2520and%2520Amyloidosis%253A%2520A%2520Case%2520Report%26jtitle%3DClin.%2520Case%2520Reports%26date%3D2016%26volume%3D4%26issue%3D10%26spage%3D997%26epage%3D1000%26doi%3D10.1002%2Fccr3.689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nwosu, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapp, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, D. A.</span></span> <span> </span><span class="NLM_article-title">Blocking the Tropomyosin Receptor Kinase A (TrkA) Receptor Inhibits Pain Behaviour in Two Rat Models of Osteoarthritis</span>. <i>Ann. Rheum. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">1246</span>– <span class="NLM_lpage">1254</span>, <span class="refDoi"> DOI: 10.1136/annrheumdis-2014-207203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1136%2Fannrheumdis-2014-207203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=26286016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2sXivV2qtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2016&pages=1246-1254&author=L.+N.+Nwosuauthor=P.+I.+Mappauthor=V.+Chapmanauthor=D.+A.+Walsh&title=Blocking+the+Tropomyosin+Receptor+Kinase+A+%28TrkA%29+Receptor+Inhibits+Pain+Behaviour+in+Two+Rat+Models+of+Osteoarthritis&doi=10.1136%2Fannrheumdis-2014-207203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Blocking the tropomyosin receptor kinase A (TrkA) receptor inhibits pain behaviour in two rat models of osteoarthritis</span></div><div class="casAuthors">Nwosu, Lilian N.; Mapp, Paul I.; Chapman, Victoria; Walsh, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1246-1254</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Objectives: Tropomyosin receptor kinase A (TrkA) mediates nociceptor sensitization by nerve growth factor (NGF), but it is unknown whether selective TrkA inhibition will be an effective strategy for treating osteoarthritis (OA) pain.  We detd. the effects of a TrkA inhibitor (AR786) on pain behavior, synovitis and joint pathol. in two rat OA models.  Methods: Knee OA was induced in rats by intraarticular monosodium-iodoacetate (MIA) injection or meniscal transection (MNX) and compared with saline injected or sham-operated controls.  Pain behavior was assessed as wt.-bearing asymmetry and paw withdrawal threshold to punctate stimulation.  Oral doses (30 mg/kg) of AR786 or vehicle were administered twice daily in either preventive (day -1 to -27) or treatment (day 14-28) protocols.  Effect maintenance was evaluated for 2 wk after treatment discontinuation.  Alterations in knee structure (cartilage, subchondral bone and synovium) were examd. by macroscopic visualization of articular surfaces and histopathol.  Results: Preventive AR786 treatment inhibited pain behavior development and therapeutic treatment attenuated established pain behavior.  Wt.-bearing asymmetry increased 1 wk after treatment discontinuation, but remained less than in vehicle-treated arthritic rats, whereas paw withdrawal thresholds returned to levels of untreated rats within 5 days of treatment discontinuation.  AR786 treatment reduced MIA-induced synovitis and did not significantly affect osteochondral pathol. in either model.  Conclusions: Blocking NGF activity by inhibiting TrkA reduced pain behavior in two rat models of OA.  Analgesia was obsd. both using preventive and treatment protocols, and was sustained after treatment discontinuation.  Selective inhibitors of TrkA therefore hold potential for OA pain relief.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWAm0vTz5k3rVg90H21EOLACvtfcHk0li6C2BCs6BiBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXivV2qtrc%253D&md5=a0159f007c97c40704266844aad93ca1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2014-207203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2014-207203%26sid%3Dliteratum%253Aachs%26aulast%3DNwosu%26aufirst%3DL.%2BN.%26aulast%3DMapp%26aufirst%3DP.%2BI.%26aulast%3DChapman%26aufirst%3DV.%26aulast%3DWalsh%26aufirst%3DD.%2BA.%26atitle%3DBlocking%2520the%2520Tropomyosin%2520Receptor%2520Kinase%2520A%2520%2528TrkA%2529%2520Receptor%2520Inhibits%2520Pain%2520Behaviour%2520in%2520Two%2520Rat%2520Models%2520of%2520Osteoarthritis%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2016%26volume%3D75%26spage%3D1246%26epage%3D1254%26doi%3D10.1136%2Fannrheumdis-2014-207203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, S. W.</span></span> <span> </span><span class="NLM_article-title">Allosteric Small Molecule Inhibitors of the NGF/TrkA Pathway, a New Approach to Treating Inflammatory Pain</span>. In  <i>International Association for the Study of Pain World Congress on Pain</i>; <span class="NLM_publisher-name">Array BioPharma</span>: <span class="NLM_publisher-loc">Boulder, CO</span>, August 29, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=S.+W.+Andrews&title=Allosteric+Small+Molecule+Inhibitors+of+the+NGF%2FTrkA+Pathway%2C+a+New+Approach+to+Treating+Inflammatory+Pain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DS.%2BW.%26atitle%3DAllosteric%2520Small%2520Molecule%2520Inhibitors%2520of%2520the%2520NGF%252FTrkA%2520Pathway%252C%2520a%2520New%2520Approach%2520to%2520Treating%2520Inflammatory%2520Pain%26jtitle%3DInternational%2520Association%2520for%2520the%2520Study%2520of%2520Pain%2520World%2520Congress%2520on%2520Pain%26pub%3DArray%2520BioPharma%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Block, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyne, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaus, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Josey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thress, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacIntyre, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span> <span> </span><span class="NLM_article-title">Identification of 4-Aminopyrazolylpyrimidines as Potent Inhibitors of Trk Kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">4672</span>– <span class="NLM_lpage">4684</span>, <span class="refDoi"> DOI: 10.1021/jm800343j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800343j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=4672-4684&issue=15&author=T.+Wangauthor=M.+L.+Lambauthor=D.+A.+Scottauthor=H.+Wangauthor=M.+H.+Blockauthor=P.+D.+Lyneauthor=J.+W.+Leeauthor=A.+M.+Daviesauthor=H.-J.+Zhangauthor=Y.+Zhuauthor=F.+Guauthor=Y.+Hanauthor=B.+Wangauthor=P.+J.+Mohrauthor=R.+J.+Kausauthor=J.+A.+Joseyauthor=E.+Hoffmannauthor=K.+Thressauthor=T.+MacIntyreauthor=H.+Wangauthor=C.+A.+Omerauthor=D.+Yu&title=Identification+of+4-Aminopyrazolylpyrimidines+as+Potent+Inhibitors+of+Trk+Kinases&doi=10.1021%2Fjm800343j"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm800343j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800343j%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DT.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DScott%26aufirst%3DD.%2BA.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DBlock%26aufirst%3DM.%2BH.%26aulast%3DLyne%26aufirst%3DP.%2BD.%26aulast%3DLee%26aufirst%3DJ.%2BW.%26aulast%3DDavies%26aufirst%3DA.%2BM.%26aulast%3DZhang%26aufirst%3DH.-J.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DF.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DMohr%26aufirst%3DP.%2BJ.%26aulast%3DKaus%26aufirst%3DR.%2BJ.%26aulast%3DJosey%26aufirst%3DJ.%2BA.%26aulast%3DHoffmann%26aufirst%3DE.%26aulast%3DThress%26aufirst%3DK.%26aulast%3DMacIntyre%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DOmer%26aufirst%3DC.%2BA.%26aulast%3DYu%26aufirst%3DD.%26atitle%3DIdentification%2520of%25204-Aminopyrazolylpyrimidines%2520as%2520Potent%2520Inhibitors%2520of%2520Trk%2520Kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26issue%3D15%26spage%3D4672%26epage%3D4684%26doi%3D10.1021%2Fjm800343j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stachel, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henze, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudd, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanda, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egbertson, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manley, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K. L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brnardic, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grobler, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanney, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leitl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, M.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krasowska-Zoladek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuck, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kane, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilodeau, M. T.</span></span> <span> </span><span class="NLM_article-title">Maximizing Diversity from a Kinase Screen: Identification of Novel and Selective Pan-Trk Inhibitors for Chronic Pain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">5800</span>– <span class="NLM_lpage">5816</span>, <span class="refDoi"> DOI: 10.1021/jm5006429</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006429" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=5800-5816&issue=13&author=S.+J.+Stachelauthor=J.+M.+Sandersauthor=D.+A.+Henzeauthor=M.+T.+Ruddauthor=H.-P.+Suauthor=Y.+Liauthor=K.+K.+Nandaauthor=M.+S.+Egbertsonauthor=P.+J.+Manleyauthor=K.+L.+G.+Jonesauthor=E.+J.+Brnardicauthor=A.+Greenauthor=J.+A.+Groblerauthor=B.+Hanneyauthor=M.+Leitlauthor=M.-T.+Laiauthor=V.+Munshiauthor=D.+Murphyauthor=K.+Rickertauthor=D.+Rileyauthor=A.+Krasowska-Zoladekauthor=C.+Daleyauthor=P.+Zuckauthor=S.+A.+Kaneauthor=M.+T.+Bilodeau&title=Maximizing+Diversity+from+a+Kinase+Screen%3A+Identification+of+Novel+and+Selective+Pan-Trk+Inhibitors+for+Chronic+Pain&doi=10.1021%2Fjm5006429"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm5006429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006429%26sid%3Dliteratum%253Aachs%26aulast%3DStachel%26aufirst%3DS.%2BJ.%26aulast%3DSanders%26aufirst%3DJ.%2BM.%26aulast%3DHenze%26aufirst%3DD.%2BA.%26aulast%3DRudd%26aufirst%3DM.%2BT.%26aulast%3DSu%26aufirst%3DH.-P.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DNanda%26aufirst%3DK.%2BK.%26aulast%3DEgbertson%26aufirst%3DM.%2BS.%26aulast%3DManley%26aufirst%3DP.%2BJ.%26aulast%3DJones%26aufirst%3DK.%2BL.%2BG.%26aulast%3DBrnardic%26aufirst%3DE.%2BJ.%26aulast%3DGreen%26aufirst%3DA.%26aulast%3DGrobler%26aufirst%3DJ.%2BA.%26aulast%3DHanney%26aufirst%3DB.%26aulast%3DLeitl%26aufirst%3DM.%26aulast%3DLai%26aufirst%3DM.-T.%26aulast%3DMunshi%26aufirst%3DV.%26aulast%3DMurphy%26aufirst%3DD.%26aulast%3DRickert%26aufirst%3DK.%26aulast%3DRiley%26aufirst%3DD.%26aulast%3DKrasowska-Zoladek%26aufirst%3DA.%26aulast%3DDaley%26aufirst%3DC.%26aulast%3DZuck%26aufirst%3DP.%26aulast%3DKane%26aufirst%3DS.%2BA.%26aulast%3DBilodeau%26aufirst%3DM.%2BT.%26atitle%3DMaximizing%2520Diversity%2520from%2520a%2520Kinase%2520Screen%253A%2520Identification%2520of%2520Novel%2520and%2520Selective%2520Pan-Trk%2520Inhibitors%2520for%2520Chronic%2520Pain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D13%26spage%3D5800%26epage%3D5816%26doi%3D10.1021%2Fjm5006429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burris, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brose, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farago, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doebele, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruickshank, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J. A.</span></span> <span> </span><span class="NLM_article-title">A First-in-Human Study of LOXO-101, a Highly Selective Inhibitor of the Tropomyosin Receptor Kinase (TRK) Family</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">TPS2624</span>, <span class="refDoi"> DOI: 10.1200/jco.2015.33.15_suppl.tps2624</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1200%2Fjco.2015.33.15_suppl.tps2624" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=TPS2624&author=H.+A.+Burrisauthor=M.+S.+Broseauthor=A.+T.+Shawauthor=T.+M.+Bauerauthor=A.+F.+Faragoauthor=R.+C.+Doebeleauthor=S.+Smithauthor=M.+Fernandesauthor=S.+Cruickshankauthor=J.+A.+Low&title=A+First-in-Human+Study+of+LOXO-101%2C+a+Highly+Selective+Inhibitor+of+the+Tropomyosin+Receptor+Kinase+%28TRK%29+Family&doi=10.1200%2Fjco.2015.33.15_suppl.tps2624"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1200%2Fjco.2015.33.15_suppl.tps2624&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252Fjco.2015.33.15_suppl.tps2624%26sid%3Dliteratum%253Aachs%26aulast%3DBurris%26aufirst%3DH.%2BA.%26aulast%3DBrose%26aufirst%3DM.%2BS.%26aulast%3DShaw%26aufirst%3DA.%2BT.%26aulast%3DBauer%26aufirst%3DT.%2BM.%26aulast%3DFarago%26aufirst%3DA.%2BF.%26aulast%3DDoebele%26aufirst%3DR.%2BC.%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DFernandes%26aufirst%3DM.%26aulast%3DCruickshank%26aufirst%3DS.%26aulast%3DLow%26aufirst%3DJ.%2BA.%26atitle%3DA%2520First-in-Human%2520Study%2520of%2520LOXO-101%252C%2520a%2520Highly%2520Selective%2520Inhibitor%2520of%2520the%2520Tropomyosin%2520Receptor%2520Kinase%2520%2528TRK%2529%2520Family%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2015%26volume%3D33%26spage%3DTPS2624%26doi%3D10.1200%2Fjco.2015.33.15_suppl.tps2624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skerratt, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagal, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilsland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bungay, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morao, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nedderman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryckmans, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stupple, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldron, G.</span></span> <span> </span><span class="NLM_article-title">The Discovery of a Potent, Selective, and Peripherally Restricted Pan-Trk Inhibitor (PF-06273340) for the Treatment of Pain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">10084</span>– <span class="NLM_lpage">10099</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00850</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00850" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWjsbfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10084-10099&issue=22&author=S.+E.+Skerrattauthor=M.+Andrewsauthor=S.+K.+Bagalauthor=J.+Bilslandauthor=D.+Brownauthor=P.+J.+Bungayauthor=S.+Coleauthor=K.+R.+Gibsonauthor=R.+Jonesauthor=I.+Moraoauthor=A.+Neddermanauthor=K.+Omotoauthor=C.+Robinsonauthor=T.+Ryckmansauthor=K.+Skinnerauthor=P.+Stuppleauthor=G.+Waldron&title=The+Discovery+of+a+Potent%2C+Selective%2C+and+Peripherally+Restricted+Pan-Trk+Inhibitor+%28PF-06273340%29+for+the+Treatment+of+Pain&doi=10.1021%2Facs.jmedchem.6b00850"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The Discovery of a Potent, Selective, and Peripherally Restricted Pan-Trk Inhibitor (PF-06273340) for the Treatment of Pain</span></div><div class="casAuthors">Skerratt, Sarah E.; Andrews, Mark; Bagal, Sharan K.; Bilsland, James; Brown, David; Bungay, Peter J.; Cole, Susan; Gibson, Karl R.; Jones, Russell; Morao, Inaki; Nedderman, Angus; Omoto, Kiyoyuki; Robinson, Colin; Ryckmans, Thomas; Skinner, Kimberly; Stupple, Paul; Waldron, Gareth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10084-10099</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The neurotrophin family of growth factors, comprised of nerve growth factor (NGF), brain derived neurotrophic factor (BDNF), neurotrophin 3 (NT3) and neurotrophin 4 (NT4), is implicated in the physiol. of chronic pain.  Given the clin. efficacy of anti-NGF monoclonal antibody (mAb) therapies, there is significant interest in the development of small mol. modulators of neurotrophin activity.  Neurotrophins signal through the tropomyosin related kinase (Trk) family of tyrosine kinase receptors, hence Trk kinase inhibition represents a potentially "druggable" point of intervention.  To deliver the safety profile required for chronic, non-life threatening pain indications, highly kinase-selective Trk inhibitors with minimal brain availability are sought.  Herein the authors describe how the use of SBDD, 2D QSAR models and Matched Mol. Pair data in compd. design enabled the delivery of the highly potent, kinase-selective and peripherally restricted clin. candidate PF-06273340.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo67CFmGAkmRrVg90H21EOLACvtfcHk0ljfZc-kC_R4nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWjsbfL&md5=f8c904a92c1716062c217fefe2374ce6</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00850&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00850%26sid%3Dliteratum%253Aachs%26aulast%3DSkerratt%26aufirst%3DS.%2BE.%26aulast%3DAndrews%26aufirst%3DM.%26aulast%3DBagal%26aufirst%3DS.%2BK.%26aulast%3DBilsland%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DD.%26aulast%3DBungay%26aufirst%3DP.%2BJ.%26aulast%3DCole%26aufirst%3DS.%26aulast%3DGibson%26aufirst%3DK.%2BR.%26aulast%3DJones%26aufirst%3DR.%26aulast%3DMorao%26aufirst%3DI.%26aulast%3DNedderman%26aufirst%3DA.%26aulast%3DOmoto%26aufirst%3DK.%26aulast%3DRobinson%26aufirst%3DC.%26aulast%3DRyckmans%26aufirst%3DT.%26aulast%3DSkinner%26aufirst%3DK.%26aulast%3DStupple%26aufirst%3DP.%26aulast%3DWaldron%26aufirst%3DG.%26atitle%3DThe%2520Discovery%2520of%2520a%2520Potent%252C%2520Selective%252C%2520and%2520Peripherally%2520Restricted%2520Pan-Trk%2520Inhibitor%2520%2528PF-06273340%2529%2520for%2520the%2520Treatment%2520of%2520Pain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D22%26spage%3D10084%26epage%3D10099%26doi%3D10.1021%2Facs.jmedchem.6b00850" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bissantz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">A Medicinal Chemist’s Guide to Molecular Interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">5061</span>– <span class="NLM_lpage">5084</span>, <span class="refDoi"> DOI: 10.1021/jm100112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5061-5084&issue=14&author=C.+Bissantzauthor=B.+Kuhnauthor=M.+Stahl&title=A+Medicinal+Chemist%E2%80%99s+Guide+to+Molecular+Interactions&doi=10.1021%2Fjm100112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A Medicinal Chemist's Guide to Molecular Interactions</span></div><div class="casAuthors">Bissantz, Caterina; Kuhn, Bernd; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5061-5084</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraV8HPbwsihbVg90H21EOLACvtfcHk0ljfZc-kC_R4nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D&md5=f39cf8700267cdc311c73de4d433bab0</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm100112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100112j%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520Medicinal%2520Chemist%25E2%2580%2599s%2520Guide%2520to%2520Molecular%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D14%26spage%3D5061%26epage%3D5084%26doi%3D10.1021%2Fjm100112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Besnard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setola, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abecassis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguiz, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norval, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, L. a</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeons, F. R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stojanovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidah, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constam, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickerton, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetsel, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Automated Design of Ligands to Polypharmacological Profiles</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>492</i></span> (<span class="NLM_issue">7428</span>),  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1038/nature11691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&issue=7428&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.-P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+a+Websterauthor=F.+R.+C.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+Design+of+Ligands+to+Polypharmacological+Profiles&doi=10.1038%2Fnature11691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0ljfZc-kC_R4nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2Ba%26aulast%3DSimeons%26aufirst%3DF.%2BR.%2BC.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520Design%2520of%2520Ligands%2520to%2520Polypharmacological%2520Profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26issue%3D7428%26spage%3D215%26epage%3D220%26doi%3D10.1038%2Fnature11691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lounkine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keiser, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitebread, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikhailov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, L.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Prediction and Testing of Drug Activity on Side-Effect Targets</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>486</i></span> (<span class="NLM_issue">7403</span>),  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1038/nature11159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1038%2Fnature11159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=22722194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyrtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=486&publication_year=2012&pages=361-367&issue=7403&author=E.+Lounkineauthor=M.+J.+Keiserauthor=S.+Whitebreadauthor=D.+Mikhailovauthor=J.+Hamonauthor=J.+L.+Jenkinsauthor=P.+Lavanauthor=E.+Weberauthor=A.+K.+Doakauthor=S.+C%C3%B4t%C3%A9author=B.+K.+Shoichetauthor=L.+Urban&title=Large-Scale+Prediction+and+Testing+of+Drug+Activity+on+Side-Effect+Targets&doi=10.1038%2Fnature11159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale prediction and testing of drug activity on side-effect targets</span></div><div class="casAuthors">Lounkine, Eugen; Keiser, Michael J.; Whitebread, Steven; Mikhailov, Dmitri; Hamon, Jacques; Jenkins, Jeremy L.; Lavan, Paul; Weber, Eckhard; Doak, Allison K.; Cote, Serge; Shoichet, Brian K.; Urban, Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">486</span>
        (<span class="NLM_cas:issue">7403</span>),
    <span class="NLM_cas:pages">361-367</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Discovering the unintended off-targets' that predict adverse drug reactions is daunting by empirical methods alone.  Drugs can act on several protein targets, some of which can be unrelated by conventional mol. metrics, and hundreds of proteins have been implicated in side effects.  Here we use a computational strategy to predict the activity of 656 marketed drugs on 73 unintended side-effect' targets.  Approx. half of the predictions were confirmed, either from proprietary databases unknown to the method or by new exptl. assays.  Affinities for these new off-targets ranged from 1 nM to 30 μM.  To explore relevance, we developed an assocn. metric to prioritize those new off-targets that explained side effects better than any known target of a given drug, creating a drug-target-adverse drug reaction network.  Among these new assocns. was the prediction that the abdominal pain side effect of the synthetic estrogen chlorotrianisene was mediated through its newly discovered inhibition of the enzyme cyclooxygenase-1.  The clin. relevance of this inhibition was borne out in whole human blood platelet aggregation assays.  This approach may have wide application to de-risking toxicol. liabilities in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5sVjfn4mgLLVg90H21EOLACvtfcHk0ljyJ1TVQKVUcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyrtb0%253D&md5=9c133b839d9527818a27a5834e4dfe11</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnature11159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11159%26sid%3Dliteratum%253Aachs%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DWhitebread%26aufirst%3DS.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DHamon%26aufirst%3DJ.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DLavan%26aufirst%3DP.%26aulast%3DWeber%26aufirst%3DE.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DS.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DUrban%26aufirst%3DL.%26atitle%3DLarge-Scale%2520Prediction%2520and%2520Testing%2520of%2520Drug%2520Activity%2520on%2520Side-Effect%2520Targets%26jtitle%3DNature%26date%3D2012%26volume%3D486%26issue%3D7403%26spage%3D361%26epage%3D367%26doi%3D10.1038%2Fnature11159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">A Computational Method for Unveiling the Target Promiscuity of Pharmacologically Active Compounds</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">11520</span>– <span class="NLM_lpage">11524</span>, <span class="refDoi"> DOI: 10.1002/anie.201706376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1002%2Fanie.201706376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht12hsrvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=11520-11524&issue=38&author=P.+Schneiderauthor=G.+Schneider&title=A+Computational+Method+for+Unveiling+the+Target+Promiscuity+of+Pharmacologically+Active+Compounds&doi=10.1002%2Fanie.201706376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A Computational Method for Unveiling the Target Promiscuity of Pharmacologically Active Compounds</span></div><div class="casAuthors">Schneider, Petra; Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11520-11524</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Drug discovery is governed by the desire to find ligands with defined modes of action.  It has been realized that even designated selective drugs may have more macromol. targets than is commonly thought.  Consequently, it will be mandatory to consider multitarget activity for the design of future medicines.  Computational models assist medicinal chemists in this effort by helping to eliminate unsuitable lead structures and spot undesired drug effects early in the discovery process.  Here, we present a straightforward computational method to find previously unknown targets of pharmacol. active compds.  Validation expts. revealed hitherto unknown targets of the natural product resveratrol and the nonsteroidal anti-inflammatory drug celecoxib.  The obtained results advocate machine learning for polypharmacol.-based mol. design, drug re-purposing, and the "de-orphaning" of phenotypic drug effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwz7epuJpraLVg90H21EOLACvtfcHk0ljyJ1TVQKVUcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht12hsrvO&md5=7dc990ab4ede2187f69b8edb41826633</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.201706376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201706376%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DP.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DA%2520Computational%2520Method%2520for%2520Unveiling%2520the%2520Target%2520Promiscuity%2520of%2520Pharmacologically%2520Active%2520Compounds%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26issue%3D38%26spage%3D11520%26epage%3D11524%26doi%3D10.1002%2Fanie.201706376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarolimek, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sridhar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitebread, S.</span></span> <span> </span><span class="NLM_article-title">Reducing Safety-Related Drug Attrition: The Use of in Vitro Pharmacological Profiling</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">909</span>– <span class="NLM_lpage">922</span>, <span class="refDoi"> DOI: 10.1038/nrd3845</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1038%2Fnrd3845" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=23197038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslCnur%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=909-922&issue=12&author=J.+Bowesauthor=A.+J.+Brownauthor=J.+Hamonauthor=W.+Jarolimekauthor=A.+Sridharauthor=G.+Waldronauthor=S.+Whitebread&title=Reducing+Safety-Related+Drug+Attrition%3A+The+Use+of+in+Vitro+Pharmacological+Profiling&doi=10.1038%2Fnrd3845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Reducing safety-related drug attrition: the use of in vitro pharmacological profiling</span></div><div class="casAuthors">Bowes, Joanne; Brown, Andrew J.; Hamon, Jacques; Jarolimek, Wolfgang; Sridhar, Arun; Waldron, Gareth; Whitebread, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">909-922</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In vitro pharmacol. profiling is increasingly being used earlier in the drug discovery process to identify undesirable off-target activity profiles that could hinder or halt the development of candidate drugs or even lead to market withdrawal if discovered after a drug is approved.  Here, for the first time, the rationale, strategies and methodologies for in vitro pharmacol. profiling at four major pharmaceutical companies (AstraZeneca, GlaxoSmithKline, Novartis and Pfizer) are presented and illustrated with examples of their impact on the drug discovery process.  We hope that this will enable other companies and academic institutions to benefit from this knowledge and consider joining us in our collaborative knowledge sharing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomVp3wnTONWrVg90H21EOLACvtfcHk0ljyJ1TVQKVUcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslCnur%252FJ&md5=7559001563ef3febd610535b8f096a45</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrd3845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3845%26sid%3Dliteratum%253Aachs%26aulast%3DBowes%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DA.%2BJ.%26aulast%3DHamon%26aufirst%3DJ.%26aulast%3DJarolimek%26aufirst%3DW.%26aulast%3DSridhar%26aufirst%3DA.%26aulast%3DWaldron%26aufirst%3DG.%26aulast%3DWhitebread%26aufirst%3DS.%26atitle%3DReducing%2520Safety-Related%2520Drug%2520Attrition%253A%2520The%2520Use%2520of%2520in%2520Vitro%2520Pharmacological%2520Profiling%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26issue%3D12%26spage%3D909%26epage%3D922%26doi%3D10.1038%2Fnrd3845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkamer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulle, S.</span></span> <span> </span><span class="NLM_article-title">KinMap: A Web-Based Tool for Interactive Navigation through Human Kinome Data</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">16</span>, <span class="refDoi"> DOI: 10.1186/s12859-016-1433-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1186%2Fs12859-016-1433-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=28056780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjsFCjtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=16&author=S.+Eidauthor=S.+Turkauthor=A.+Volkamerauthor=F.+Rippmannauthor=S.+Fulle&title=KinMap%3A+A+Web-Based+Tool+for+Interactive+Navigation+through+Human+Kinome+Data&doi=10.1186%2Fs12859-016-1433-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">KinMap: a web-based tool for interactive navigation through human kinome data</span></div><div class="casAuthors">Eid, Sameh; Turk, Samo; Volkamer, Andrea; Rippmann, Friedrich; Fulle, Simone</div><div class="citationInfo"><span class="NLM_cas:title">BMC Bioinformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">16/1-16/6</span>CODEN:
                <span class="NLM_cas:coden">BBMIC4</span>;
        ISSN:<span class="NLM_cas:issn">1471-2105</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Annotations of the phylogenetic tree of the human kinome is an intuitive way to visualize compd. profiling data, structural features of kinases or functional relationships within this important class of proteins.  The increasing vol. and complexity of kinase-related data underlines the need for a tool that enables complex queries pertaining to kinase disease involvement and potential therapeutic uses of kinase inhibitors.  Results: Here, we present KinMap, a user-friendly online tool that facilitates the interactive navigation through kinase knowledge by linking biochem., structural, and disease assocn. data to the human kinome tree.  To this end, preprocessed data from freely-available sources, such as ChEMBL, the Protein Data Bank, and the Center for Therapeutic Target Validation platform are integrated into KinMap and can easily be complemented by proprietary data.  The value of KinMap will be exemplarily demonstrated for uncovering new therapeutic indications of known kinase inhibitors and for prioritizing kinases for drug development efforts.  Conclusions: KinMap represents a new generation of kinome tree viewers which facilitates interactive exploration of the human kinome.  KinMap enables generation of high-quality annotated images of the human kinome tree as well as exchange of kinome-related data in scientific communications.  Furthermore, KinMap supports multiple input and output formats and recognizes alternative kinase names and links them to a unified naming scheme, which makes it a useful tool across different disciplines and applications.  A web-service of KinMap is freely available at http://www.kinhub.org/kinmap/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1M2aa8fuKALVg90H21EOLACvtfcHk0li8bGVHUzyYxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjsFCjtLc%253D&md5=29737c7cda8d9caf75018e0d29e9a459</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1186%2Fs12859-016-1433-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-016-1433-7%26sid%3Dliteratum%253Aachs%26aulast%3DEid%26aufirst%3DS.%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DFulle%26aufirst%3DS.%26atitle%3DKinMap%253A%2520A%2520Web-Based%2520Tool%2520for%2520Interactive%2520Navigation%2520through%2520Human%2520Kinome%2520Data%26jtitle%3DBMC%2520Bioinf.%26date%3D2017%26volume%3D18%26spage%3D16%26doi%3D10.1186%2Fs12859-016-1433-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindstrom, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanner, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belew, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: Automated Docking with Selective Receptor Flexibility</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">2785</span>– <span class="NLM_lpage">2791</span>, <span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&issue=16&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+Automated+Docking+with+Selective+Receptor+Flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0li8bGVHUzyYxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520Automated%2520Docking%2520with%2520Selective%2520Receptor%2520Flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26issue%3D16%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wade, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodford, P. J.</span></span> <span> </span><span class="NLM_article-title">Identifying Targets for Bioreductive Agents: Using GRID to Predict Selective Binding Regions of Proteins</span>. <i>J. Mol. Graphics</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">108</span>, <span class="refDoi"> DOI: 10.1016/S0263-7855(89)80013-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1016%2FS0263-7855%2889%2980013-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=2488262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADyaL1MXlvFSqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1989&pages=103-108&issue=2&author=C.+A.+Reynoldsauthor=R.+C.+Wadeauthor=P.+J.+Goodford&title=Identifying+Targets+for+Bioreductive+Agents%3A+Using+GRID+to+Predict+Selective+Binding+Regions+of+Proteins&doi=10.1016%2FS0263-7855%2889%2980013-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying targets for bioreductive agents:  using GRID to predict selective binding regions of proteins</span></div><div class="casAuthors">Reynolds, Christopher A.; Wade, Rebecca C.; Goodford, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">103-8, 100</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    </div><div class="casAbstract">A computational procedure based on the computer program, GRID, is described for investigating potential binding sites of a target macromol. for their ability to bind both a reduced probe mol. and an oxidized probe mol.  The interaction energies are obtained using a mol. mechanics method and can be displayed as 3-dimensional energy contours, indicating regions of the target mol. that may have favorable interactions with the probe mol.  Differences in the interaction energies of the oxidized and reduced probe with the target can also be plotted as contours, indicating regions that are selective for the reduced probe.  These selectivity contours can be used to show whether the macromol. is a potential target for bioreductive agents.  The method has been applied to the chicken liver dihydrofolate reductase enzyme and has indicated new binding regions that may be suitable binding sites for bioreductive agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoVQeHx4QT47Vg90H21EOLACvtfcHk0li8bGVHUzyYxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXlvFSqs7k%253D&md5=0d536c811584e37b3a40105e08018cc5</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0263-7855%2889%2980013-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0263-7855%252889%252980013-X%26sid%3Dliteratum%253Aachs%26aulast%3DReynolds%26aufirst%3DC.%2BA.%26aulast%3DWade%26aufirst%3DR.%2BC.%26aulast%3DGoodford%26aufirst%3DP.%2BJ.%26atitle%3DIdentifying%2520Targets%2520for%2520Bioreductive%2520Agents%253A%2520Using%2520GRID%2520to%2520Predict%2520Selective%2520Binding%2520Regions%2520of%2520Proteins%26jtitle%3DJ.%2520Mol.%2520Graphics%26date%3D1989%26volume%3D7%26issue%3D2%26spage%3D103%26epage%3D108%26doi%3D10.1016%2FS0263-7855%2889%2980013-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegscheid-Gerlach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaliani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span> <span> </span><span class="NLM_article-title">On the Art of Compiling and Using “Drug-Like” Chemical Fragment Spaces</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1503</span>– <span class="NLM_lpage">1507</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200800178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1002%2Fcmdc.200800178" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1503-1507&issue=10&author=J.+Degenauthor=C.+Wegscheid-Gerlachauthor=A.+Zalianiauthor=M.+Rarey&title=On+the+Art+of+Compiling+and+Using+%E2%80%9CDrug-Like%E2%80%9D+Chemical+Fragment+Spaces&doi=10.1002%2Fcmdc.200800178"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200800178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200800178%26sid%3Dliteratum%253Aachs%26aulast%3DDegen%26aufirst%3DJ.%26aulast%3DWegscheid-Gerlach%26aufirst%3DC.%26aulast%3DZaliani%26aufirst%3DA.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DOn%2520the%2520Art%2520of%2520Compiling%2520and%2520Using%2520%25E2%2580%259CDrug-Like%25E2%2580%259D%2520Chemical%2520Fragment%2520Spaces%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26issue%3D10%26spage%3D1503%26epage%3D1507%26doi%3D10.1002%2Fcmdc.200800178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, M.</span></span> <span> </span><span class="NLM_article-title">Extended-Connectivity Fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">742</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&issue=5&author=D.+Rogersauthor=M.+Hahn&title=Extended-Connectivity+Fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0lhfcv_WhhtF1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-Connectivity%2520Fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26issue%3D5%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span> <i>Schrödinger Release 2017-1: Maestro</i>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schr%C3%B6dinger+Release+2017-1%3A+Maestro%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%2520Release%25202017-1%253A%2520Maestro%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buitinck, L.</span>; <span class="NLM_string-name">Louppe, G.</span>; <span class="NLM_string-name">Blondel, M.</span>; <span class="NLM_string-name">Pedregosa, F.</span>; <span class="NLM_string-name">Mueller, A.</span>; <span class="NLM_string-name">Grisel, O.</span>; <span class="NLM_string-name">Niculae, V.</span>; <span class="NLM_string-name">Prettenhofer, P.</span>; <span class="NLM_string-name">Gramfort, A.</span>; <span class="NLM_string-name">Grobler, J.</span>; <span class="NLM_string-name">Layton, R.</span>; <span class="NLM_string-name">VanderPlas, J.</span>; <span class="NLM_string-name">Joly, A.</span>; <span class="NLM_string-name">Holt, B.</span>; <span class="NLM_string-name">Varoquaux, G.</span></span> <span> </span><span class="NLM_article-title">API Design for Machine Learning Software: Experiences from the Scikit-Learn Project</span>. In  <i>ECML/PKDD Workshop: Languages for Data Mining and Machine Learning, Prague, September 23, 2013</i>; <span class="NLM_publisher-name">Liege Universite</span>: <span class="NLM_publisher-loc">Liege, Belgium</span>, <span class="NLM_year">2013</span>; pp  <span class="NLM_fpage">108</span>– <span class="NLM_lpage">122</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=108-122&author=L.+Buitinck&author=G.+Louppe&author=M.+Blondel&author=F.+Pedregosa&author=A.+Mueller&author=O.+Grisel&author=V.+Niculae&author=P.+Prettenhofer&author=A.+Gramfort&author=J.+Grobler&author=R.+Layton&author=J.+VanderPlas&author=A.+Joly&author=B.+Holt&author=G.+Varoquaux&title=API+Design+for+Machine+Learning+Software%3A+Experiences+from+the+Scikit-Learn+Project"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DBuitinck%26aufirst%3DL.%26atitle%3DAPI%2520Design%2520for%2520Machine%2520Learning%2520Software%253A%2520Experiences%2520from%2520the%2520Scikit-Learn%2520Project%26jtitle%3DECML%252FPKDD%2520Workshop%253A%2520Languages%2520for%2520Data%2520Mining%2520and%2520Machine%2520Learning%252C%2520Prague%252C%2520September%252023%252C%25202013%26pub%3DLiege%2520Universite%26date%3D2013%26spage%3D108%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suenaga, M.</span></span> <span> </span><span class="NLM_article-title">Development of GUI for GAMESS/FMO Calculation</span>. <i>J. Comput. Chem., Jpn.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.2477/jccj.H1920</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.2477%2Fjccj.H1920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1ejsb3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=33-54&issue=1&author=M.+Suenaga&title=Development+of+GUI+for+GAMESS%2FFMO+Calculation&doi=10.2477%2Fjccj.H1920"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Development of GUI for GAMESS/FMO calculation</span></div><div class="casAuthors">Suenaga, Masahiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer Chemistry, Japan</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-54</span>CODEN:
                <span class="NLM_cas:coden">JCCJAG</span>;
        ISSN:<span class="NLM_cas:issn">1347-1767</span>.
    
            (<span class="NLM_cas:orgname">Nippon Konputa Kagakkai</span>)
        </div><div class="casAbstract">A GUI (Graphic User Interface) for GAMESS/FMO (Fragment MO method) calcn. was developed and implemented on Facio, the pre/post-processor for computational chem.  By using this GUI, one can easily make FMO fragments for a given protein or nucleic acid and generate FMO input files.  Besides automatic fragmentation, one can manually define addnl. FMO fractioning points to non-peptide moiety of conjugated protein.  For manual definition, a novel local structure viewer was also developed.  The desired local structure is defined by an atom no. or by a fragment no.  Using this local structure viewer, one can check the structural validity of each FMO fragment and perform manual definition of addnl. fractioning points on the fragment.  Manually defined fractioning points can be saved as "fragment definition file", which can be loaded later to set the predefined fractioning points.  For the optimization of hydrogen atoms automatically added to the PDB file of a protein or nucleic acid which lacks hydrogen atoms, a fragment optimization function was implemented for the GUI.  While a mol. dynamics method or mol. mechanics method is usually used for this purpose, the new optimization method utilizes a MO method, such as PM3 or STO-3G of GAMESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomdWWrcqKIU7Vg90H21EOLACvtfcHk0lhfcv_WhhtF1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1ejsb3J&md5=ad5c3e0ff6c1c15cc11dae87292268f7</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.2477%2Fjccj.H1920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2477%252Fjccj.H1920%26sid%3Dliteratum%253Aachs%26aulast%3DSuenaga%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520GUI%2520for%2520GAMESS%252FFMO%2520Calculation%26jtitle%3DJ.%2520Comput.%2520Chem.%252C%2520Jpn.%26date%3D2008%26volume%3D7%26issue%3D1%26spage%3D33%26epage%3D54%26doi%3D10.2477%2Fjccj.H1920" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&issue=7&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lhfcv_WhhtF1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D7%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggs, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azimioara, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lélias, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velasco, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span> <span> </span><span class="NLM_article-title">A Small Molecule-Kinase Interaction Map for Clinical Kinase Inhibitors</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&issue=3&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.-M.+L%C3%A9liasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+Small+Molecule-Kinase+Interaction+Map+for+Clinical+Kinase+Inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0liS8Shh8N4vGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DL%25C3%25A9lias%26aufirst%3DJ.-M.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520Small%2520Molecule-Kinase%2520Interaction%2520Map%2520for%2520Clinical%2520Kinase%2520Inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26issue%3D3%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YNE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YNE','PDB','4YNE'); return false;">PDB: 4YNE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4pmt" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4pmt','PDB','4pmt'); return false;">PDB: 4pmt</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4aoj" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4aoj','PDB','4aoj'); return false;">PDB: 4aoj</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4yne" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4yne','PDB','4yne'); return false;">PDB: 4yne</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4yps" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4yps','PDB','4yps'); return false;">PDB: 4yps</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1mq4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1mq4','PDB','1mq4'); return false;">PDB: 1mq4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ol5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ol5','PDB','1ol5'); return false;">PDB: 1ol5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ol6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ol6','PDB','1ol6'); return false;">PDB: 1ol6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ol7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ol7','PDB','1ol7'); return false;">PDB: 1ol7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2c6d" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2c6d','PDB','2c6d'); return false;">PDB: 2c6d</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2dwb" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2dwb','PDB','2dwb'); return false;">PDB: 2dwb</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2np8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2np8','PDB','2np8'); return false;">PDB: 2np8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2w1c" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2w1c','PDB','2w1c'); return false;">PDB: 2w1c</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2w1d" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2w1d','PDB','2w1d'); return false;">PDB: 2w1d</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2w1e" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2w1e','PDB','2w1e'); return false;">PDB: 2w1e</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2w1f" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2w1f','PDB','2w1f'); return false;">PDB: 2w1f</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2w1g" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2w1g','PDB','2w1g'); return false;">PDB: 2w1g</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2wtw" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2wtw','PDB','2wtw'); return false;">PDB: 2wtw</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2x6d" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2x6d','PDB','2x6d'); return false;">PDB: 2x6d</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2x6e" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2x6e','PDB','2x6e'); return false;">PDB: 2x6e</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xne" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xne','PDB','2xne'); return false;">PDB: 2xne</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xng" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xng','PDB','2xng'); return false;">PDB: 2xng</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xru" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xru','PDB','2xru'); return false;">PDB: 2xru</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d14" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d14','PDB','3d14'); return false;">PDB: 3d14</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d15" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d15','PDB','3d15'); return false;">PDB: 3d15</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d2i" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d2i','PDB','3d2i'); return false;">PDB: 3d2i</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d2k" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d2k','PDB','3d2k'); return false;">PDB: 3d2k</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3dj7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3dj7','PDB','3dj7'); return false;">PDB: 3dj7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3e5a" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3e5a','PDB','3e5a'); return false;">PDB: 3e5a</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3efw" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3efw','PDB','3efw'); return false;">PDB: 3efw</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3fdn" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3fdn','PDB','3fdn'); return false;">PDB: 3fdn</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3h0z" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3h0z','PDB','3h0z'); return false;">PDB: 3h0z</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ha6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ha6','PDB','3ha6'); return false;">PDB: 3ha6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3k5u" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3k5u','PDB','3k5u'); return false;">PDB: 3k5u</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3m11" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3m11','PDB','3m11'); return false;">PDB: 3m11</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3myg" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3myg','PDB','3myg'); return false;">PDB: 3myg</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3nrm" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3nrm','PDB','3nrm'); return false;">PDB: 3nrm</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3o50" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3o50','PDB','3o50'); return false;">PDB: 3o50</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3p9j" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3p9j','PDB','3p9j'); return false;">PDB: 3p9j</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3r21" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3r21','PDB','3r21'); return false;">PDB: 3r21</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3r22" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3r22','PDB','3r22'); return false;">PDB: 3r22</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uo4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uo4','PDB','3uo4'); return false;">PDB: 3uo4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uo5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uo5','PDB','3uo5'); return false;">PDB: 3uo5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uod" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3uod','PDB','3uod'); return false;">PDB: 3uod</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3up2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3up2','PDB','3up2'); return false;">PDB: 3up2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3up7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3up7','PDB','3up7'); return false;">PDB: 3up7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3vap" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3vap','PDB','3vap'); return false;">PDB: 3vap</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4bn1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4bn1','PDB','4bn1'); return false;">PDB: 4bn1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4byi" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4byi','PDB','4byi'); return false;">PDB: 4byi</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4c3p" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4c3p','PDB','4c3p'); return false;">PDB: 4c3p</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4c3r" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4c3r','PDB','4c3r'); return false;">PDB: 4c3r</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ceg" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ceg','PDB','4ceg'); return false;">PDB: 4ceg</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dea" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dea','PDB','4dea'); return false;">PDB: 4dea</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4deb" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4deb','PDB','4deb'); return false;">PDB: 4deb</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ded" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ded','PDB','4ded'); return false;">PDB: 4ded</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dee" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dee','PDB','4dee'); return false;">PDB: 4dee</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dhf" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4dhf','PDB','4dhf'); return false;">PDB: 4dhf</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4j8m" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4j8m','PDB','4j8m'); return false;">PDB: 4j8m</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4j8n" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4j8n','PDB','4j8n'); return false;">PDB: 4j8n</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jaj" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jaj','PDB','4jaj'); return false;">PDB: 4jaj</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jbo" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jbo','PDB','4jbo'); return false;">PDB: 4jbo</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jbp" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jbp','PDB','4jbp'); return false;">PDB: 4jbp</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4o0s" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4o0s','PDB','4o0s'); return false;">PDB: 4o0s</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4o0u" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4o0u','PDB','4o0u'); return false;">PDB: 4o0u</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4o0w" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4o0w','PDB','4o0w'); return false;">PDB: 4o0w</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4prj" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4prj','PDB','4prj'); return false;">PDB: 4prj</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4utd" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4utd','PDB','4utd'); return false;">PDB: 4utd</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5aad" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5aad','PDB','5aad'); return false;">PDB: 5aad</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5aae" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5aae','PDB','5aae'); return false;">PDB: 5aae</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5aaf" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5aaf','PDB','5aaf'); return false;">PDB: 5aaf</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5aag" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5aag','PDB','5aag'); return false;">PDB: 5aag</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ew9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ew9','PDB','5ew9'); return false;">PDB: 5ew9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4af3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4af3','PDB','4af3'); return false;">PDB: 4af3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4b8m" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4b8m','PDB','4b8m'); return false;">PDB: 4b8m</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i24"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00140">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_64843"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b00140">10.1021/acs.jmedchem.8b00140</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Profiling results, results of primary screens, selectivity scores of top hits, the data set used to train the QM/ML approach as well as additional information about the validation results of the employed prediction tools, about the biological assays and assay results, and analytic data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Predicted binding pose of <b>a1</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_002.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings for all chemical structures mentioned (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_001.pdf">jm8b00140_si_001.pdf (1.2 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_002.pdb">jm8b00140_si_002.pdb (600.75 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00140/suppl_file/jm8b00140_si_003.csv">jm8b00140_si_003.csv (1.14 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00140&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-11%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00140%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00140" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67992ddf6aa3247f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
